{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os, glob, shutil\n",
    "import sys\n",
    "\n",
    "#Import config file. Update config.py according to your environment\n",
    "from config import path_to_data\n",
    "\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "from scipy import stats\n",
    "\n",
    "import requests, json, re\n",
    "\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.service import Service\n",
    "from selenium.webdriver.common.by import By\n",
    "from urllib3.exceptions import ReadTimeoutError\n",
    "from selenium.common.exceptions import TimeoutException, WebDriverException\n",
    "\n",
    "from pdfminer.high_level import extract_text, extract_pages\n",
    "from pdfminer.layout import LTTextContainer, LTChar, LTTextLine, LTTextBox\n",
    "\n",
    "from tqdm import tqdm\n",
    "import time\n",
    "from datetime import datetime\n",
    "import multiprocessing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset_source = 'semanticsscholar' #'arxiv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def load_dataset(dataset_source='semanticsscholar', years=None, data_types=None):\n",
    "    \"\"\"Fetch dataset metadata and text data for given years and data types.\"\"\"\n",
    "    years = (\n",
    "        [str(year) for year in years] if isinstance(years, list) else\n",
    "        [str(years)] if years else\n",
    "        [year for year in os.listdir(os.path.join(path_to_data, 'metadata')) if year.isdigit()]\n",
    "    )\n",
    "    data_types = data_types or ['metadata', 'text']\n",
    "\n",
    "    load_metadata = 'metadata' in data_types\n",
    "    load_textdata = 'text' in data_types\n",
    "\n",
    "    metadata, textdata = [], []\n",
    "    metadata_years, textdata_years = [], []\n",
    "\n",
    "    for year in years:\n",
    "        if load_metadata:\n",
    "            metadata_path = os.path.join(path_to_data, 'metadata', year, f'{dataset_source}_metadata_{year}.parquet')\n",
    "            if os.path.isfile(metadata_path):\n",
    "                metadata.append(pd.read_parquet(metadata_path, engine=\"pyarrow\"))\n",
    "                metadata_years.append(year)\n",
    "        \n",
    "        if load_textdata:\n",
    "            textdata_path = os.path.join(path_to_data, 'text', year, f'{dataset_source}_text_{year}.parquet')\n",
    "            if os.path.isfile(textdata_path):\n",
    "                textdata.append(pd.read_parquet(textdata_path, engine=\"pyarrow\"))\n",
    "                textdata_years.append(year)\n",
    "    \n",
    "    metadata = pd.concat(metadata, axis=0) if metadata else []\n",
    "    textdata = pd.concat(textdata, axis=0) if textdata else []\n",
    "    \n",
    "    msg_parts = []\n",
    "    if load_metadata:\n",
    "        msg_parts.append(f'metadata loaded for years: {metadata_years}')\n",
    "    if load_textdata:\n",
    "        msg_parts.append(f'text data loaded for years: {textdata_years}')\n",
    "    \n",
    "    if msg_parts:\n",
    "        print(\"; \".join(msg_parts))\n",
    "    \n",
    "    if load_metadata and load_textdata and len(metadata) != len(textdata):\n",
    "        raise ValueError(\"Metadata and text data don't have the same length.\")\n",
    "    \n",
    "    output = (data for data in [metadata, textdata] if len(data) > 0)\n",
    "    return output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "def pdf_main_fontsize(pdf_path):\n",
    "    sizes = [\n",
    "        character.size\n",
    "        for page_layout in extract_pages(pdf_path)\n",
    "        for element in page_layout if isinstance(element, LTTextContainer)\n",
    "        for text_line in element if isinstance(text_line, LTTextLine)\n",
    "        for character in text_line if isinstance(character, LTChar)\n",
    "    ]\n",
    "    return stats.mode(sizes, keepdims=True)[0][0] if sizes else 0\n",
    "\n",
    "def extract_pdf_sections(pdf_path, authorlist, paperId, possible_section_headings=None, size_threshold=None):\n",
    "    author_last_names = [name.split()[-1] for name in authorlist.split(',')]\n",
    "    last_section_names = {'acknowledgment', 'acknowledgement', 'acknowlegment', 'reference'}\n",
    "    month_markers = {'january', 'february', 'march', 'april', 'may', 'june', 'july', 'august', 'september', 'november', 'december',\n",
    "                     'jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'nov', 'dec'}\n",
    "    year_markers = {str(y) for y in range(1900, 2030)}\n",
    "    section_list = possible_section_headings or {\n",
    "        'introduction', 'results', 'discussion', 'conclusions', 'methods', 'materials', 'experimental',\n",
    "        'materials and methods', 'experimental procedure', 'related work', 'i.', 'ii.', 'iii.', 'iv.', 'v.', 'vi.'\n",
    "    }\n",
    "\n",
    "    size_threshold = size_threshold if size_threshold is not None else 0.9\n",
    "\n",
    "    page_width, page_height = 595, 842\n",
    "    nword_abstract_th, nword_sections_th = 30, 30\n",
    "\n",
    "    #Determine the most common font size (mode)\n",
    "    size_mode = pdf_main_fontsize(pdf_path)\n",
    "\n",
    "    text_blocks, tag_blocks, nwords_in_blocks = [], [], []\n",
    "    section_to_find, section_heading, tag = 'AUTHORS', 'UNDEFINED', 'UNDEFINED'\n",
    "    reached_end = False\n",
    "    for p, page_layout in enumerate(extract_pages(pdf_path)):\n",
    "        if reached_end:\n",
    "            break\n",
    "\n",
    "        for element in page_layout:\n",
    "            if not isinstance(element, LTTextBox):\n",
    "                continue\n",
    "\n",
    "            x0, y0, x1, y1 = element.bbox\n",
    "            if not (y0 > 0.05*page_height and y1 < 0.95*page_height and x0 > 0.05*page_width and x1 < 0.95*page_width):\n",
    "                continue\n",
    "\n",
    "            filtered_text, sizes = [], []\n",
    "            for text_line in element:\n",
    "                if isinstance(text_line, LTTextLine):\n",
    "                    line_text = [character.get_text() for character in text_line]\n",
    "                    sizes.extend([character.size for character in text_line if isinstance(character, LTChar)])\n",
    "                    line_str = \"\".join(line_text).strip()\n",
    "                    if not re.fullmatch(r\"\\d+\", line_str): # Keeps lines that are NOT just numbers\n",
    "                        filtered_text.append(line_str)\n",
    "                    filtered_text.append(\"\\n\") # Preserve line breaks\n",
    "\n",
    "            #joining characters in a single text while removing weird markers generated by pdfminer\n",
    "            filtered_text = re.sub(r'\\(cid:[^\\)]+\\)', '', \"\".join(filtered_text).strip())\n",
    "            word_list = re.split(r'[\\n\\s]+', filtered_text.lower().strip())\n",
    "            nwords = len(word_list)\n",
    "\n",
    "            if any(end_section in word_list[:3] for end_section in last_section_names):\n",
    "                reached_end = True\n",
    "                continue\n",
    "            \n",
    "            #removing everything before the author block as well as the correspondance fields\n",
    "            if p <= 1:\n",
    "                nauthors_detected = sum(lastname.lower() in ' '.join(word_list) for lastname in author_last_names)\n",
    "                if nauthors_detected >= 0.5 * len(author_last_names) and y0 > 0.3 * page_height:\n",
    "                    text_blocks, tag_blocks, nwords_in_blocks = [], [], []\n",
    "                    section_to_find = 'ABSTRACT'\n",
    "                    filtered_text = []\n",
    "                if nauthors_detected > 0 and '@' in filtered_text:\n",
    "                    filtered_text = []\n",
    "\n",
    "            #removing blocks likely headers with publication date\n",
    "            if any([m in word_list for m in month_markers]) and any([y in word_list for y in year_markers]) and nwords < 10:\n",
    "                continue\n",
    "\n",
    "            #removing figure captions\n",
    "            if any([figname in word_list[0] for figname in ['fig', 'figure', 'table', 'image']]):\n",
    "                continue\n",
    "\n",
    "            #removing previous block if likely a header but not followed by capitalized paragraph\n",
    "            if filtered_text and not filtered_text[0].isupper() and nwords_in_blocks and nwords_in_blocks[-1] <= 3:\n",
    "                text_blocks.pop()\n",
    "                tag_blocks.pop()\n",
    "                nwords_in_blocks.pop()\n",
    "            elif (filtered_text and filtered_text.strip() and filtered_text.strip()[0].isupper() and nwords_in_blocks and \n",
    "                  nwords_in_blocks[-1] <= 3 and text_blocks and any(h in re.sub(r'[\\n\\s]+', ' ', text_blocks[-1].lower()) for h in section_list)):\n",
    "                section_heading = ''.join([w.upper() for w in re.sub(r'[\\d.]', '', text_blocks[-1])])\n",
    "                if nwords > nword_sections_th:\n",
    "                    tag_blocks[-1] = section_heading\n",
    "\n",
    "            if not reached_end and filtered_text and (max(sizes, default=0) >= size_threshold * size_mode or nwords > 50):\n",
    "                if section_to_find == 'ABSTRACT' and nwords > nword_abstract_th:\n",
    "                    tag = 'ABSTRACT'\n",
    "                    if word_list[-1][-1] == '.':\n",
    "                        section_to_find = 'INTRODUCTION'\n",
    "                elif section_to_find == 'INTRODUCTION':\n",
    "                    if nwords > nword_sections_th:\n",
    "                        tag = 'INTRODUCTION'\n",
    "                        section_heading, section_to_find = 'INTRODUCTION', 'NEXTHEADING'\n",
    "                    else:\n",
    "                        tag = 'UNDEFINED'\n",
    "                elif section_to_find == 'NEXTHEADING' and nwords > nword_sections_th:\n",
    "                    tag = section_heading\n",
    "                \n",
    "                text_blocks.append(filtered_text)\n",
    "                tag_blocks.append(tag)\n",
    "                nwords_in_blocks.append(nwords)\n",
    "\n",
    "    sections = {'paperId': paperId,\n",
    "            'pdf_abstract': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'abstract'})),\n",
    "            'pdf_introduction': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'introduction', 'related work', 'i.', 'ii.'})),\n",
    "            'pdf_results': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'results', 'experiment', 'i.', 'ii.'})),\n",
    "            'pdf_discussion': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'discussion', 'conclusion', 'v.', 'vi.'})),\n",
    "            'pdf_methods': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'methods', 'materials', 'experimental', 'materials and methods', 'experimental procedure'})),\n",
    "            'pdf_text': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if not any(s in t.lower() for s in {'undefined'})),\n",
    "            'author_list': authorlist\n",
    "            }\n",
    "    \n",
    "    return sections, text_blocks, tag_blocks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def download_pdfs(inputs):\n",
    "    #you'll need to have ChromeDriver installed\n",
    "    txtdata, downloads_dir = inputs\n",
    "    service = Service(executable_path=\"/usr/bin/chromedriver\")\n",
    "    downloads_dir = downloads_dir or './data/pdf/downloads'\n",
    "    os.makedirs(downloads_dir, exist_ok=True)\n",
    "    output_dir = os.path.dirname(downloads_dir)\n",
    "\n",
    "    txtdata, downloads_dir = inputs\n",
    "    service = Service(executable_path=\"/usr/bin/chromedriver\")\n",
    "    if downloads_dir is None:\n",
    "        downloads_dir = './data/pdf/downloads'\n",
    "\n",
    "    output_dir = '/'.join(downloads_dir.split('/')[:-1])\n",
    "    \n",
    "\n",
    "    timeout_seconds = 10\n",
    "    timeout_startdw_seconds = 5\n",
    "    block_markers = {\"captcha\", \"verify you are human\", \"forbidden\", \"this site can’t be reached\"}\n",
    "\n",
    "    options = webdriver.ChromeOptions()\n",
    "    prefs = {\n",
    "        \"plugins.always_open_pdf_externally\": True,\n",
    "        \"download.prompt_for_download\": False,\n",
    "        \"download.default_directory\": downloads_dir,\n",
    "        \"download.directory_upgrade\": True\n",
    "    }\n",
    "    options.add_experimental_option(\"prefs\", prefs)\n",
    "\n",
    "    driver = webdriver.Chrome(service=service, options=options)\n",
    "    text_pdfs = []\n",
    "    fraction_print = 0\n",
    "    processed_list = []\n",
    "\n",
    "    for k, paper in enumerate(txtdata):\n",
    "        try:\n",
    "            custom_download_dir = os.path.join(downloads_dir, paper['paperId'])\n",
    "            os.makedirs(custom_download_dir, exist_ok=True)\n",
    "\n",
    "            # Set Chrome's download directory dynamically\n",
    "            driver.execute_cdp_cmd(\n",
    "                    \"Page.setDownloadBehavior\", \n",
    "                    {\"behavior\": \"allow\",\n",
    "                    \"downloadPath\": custom_download_dir}\n",
    "                    )\n",
    "            \n",
    "            clean_url = re.sub(r'^http://', 'https://', paper['openAccessPdf'])\n",
    "            filename = []\n",
    "            driver.set_page_load_timeout(timeout_seconds)\n",
    "            driver.get(clean_url)\n",
    "            end_time = time.time() + timeout_seconds\n",
    "            end_time_startdw = time.time() + timeout_startdw_seconds\n",
    "            \n",
    "            while time.time() < end_time:\n",
    "                time.sleep(0.1)\n",
    "                filename = [fname for fname in glob.glob(os.path.join(custom_download_dir, '*.pdf')) if not any(id in fname for id in processed_list)]\n",
    "                if filename:\n",
    "                    processed_list = [id for id in processed_list if not any(id in fname for fname in filename)]\n",
    "                    if len(filename) > 1:\n",
    "                        raise ImportError('more than one download to process', filename, processed_list)\n",
    "                    filename = filename[0]\n",
    "                    break\n",
    "                if any(marker.lower() in driver.page_source.lower() for marker in block_markers):\n",
    "                    filename = None\n",
    "                    break\n",
    "                if time.time() > end_time_startdw and not glob.glob(os.path.join(custom_download_dir, '*.crdownload')):\n",
    "                    print(f\"Paper {k}: Download not started after {timeout_startdw_seconds} seconds for {clean_url}.\")\n",
    "                    filename = None\n",
    "                    break\n",
    "            else:\n",
    "                print(f\"Paper {k}: Download not finished after {timeout_seconds} seconds for {clean_url}.\")\n",
    "                filename = None\n",
    "        \n",
    "            if filename:\n",
    "                filepath = os.path.join(custom_download_dir, filename)\n",
    "                pdf_data,_,_ = extract_pdf_sections(filepath, paper['authorName'], paper['paperId'])\n",
    "                processed_list.append(paper['paperId'])\n",
    "                pdf_data['problem'] = None\n",
    "                shutil.rmtree(custom_download_dir)\n",
    "                # os.remove(filepath)\n",
    "            else:\n",
    "                pdf_data = {'paperId': txtdata[k]['paperId'], 'author_list': paper['authorName'], 'pdf_text': None, 'pdf_abstract': None, 'pdf_introduction': None, \n",
    "                            'pdf_discussion': None, 'pdf_results': None, 'pdf_methods': None, 'problem': 'captcha'}\n",
    "        except (TimeoutException, WebDriverException):\n",
    "            print(f\"Paper {k}: Error loading {clean_url}\")\n",
    "            pdf_data = {'paperId': txtdata[k]['paperId'], 'author_list': paper['authorName'], 'pdf_text': None, 'pdf_abstract': None, 'pdf_introduction': None, \n",
    "                        'pdf_discussion': None, 'pdf_results': None, 'pdf_methods': None, 'problem': 'broken link'}\n",
    "        \n",
    "        text_pdfs.append(pdf_data)\n",
    "        fraction_done = (k + 1) / len(txtdata) * 100\n",
    "        if fraction_done > fraction_print:\n",
    "            print(f\"{fraction_done:.0f}% of papers processed in {downloads_dir}\")\n",
    "            fraction_print = fraction_print + 20\n",
    "\n",
    "    driver.quit()\n",
    "\n",
    "    #Extracting pdfs from downloads that took too long to be processed immediately\n",
    "    if os.path.isdir(downloads_dir):\n",
    "        for dirname in os.listdir(downloads_dir):\n",
    "            dir_path = os.path.join(downloads_dir, dirname)\n",
    "            if os.path.isdir(dir_path) and not glob.glob(os.path.join(dir_path, '*.pdf')):\n",
    "                shutil.rmtree(dir_path)\n",
    "            elif glob.glob(os.path.join(dir_path, '*.pdf')):\n",
    "                filepath = glob.glob(os.path.join(dir_path, '*.pdf'))[0]\n",
    "                paper_data = next((pdf_data for pdf_data in text_pdfs if pdf_data['paperId'] == dirname), None)\n",
    "                if paper_data:\n",
    "                    pdf_data,_,_ = extract_pdf_sections(filepath, paper_data['author_list'], paper_data['paperId'])\n",
    "                    for key in pdf_data.keys():\n",
    "                        paper_data[key] = pdf_data[key]\n",
    "                    paper_data['problem'] = None\n",
    "                \n",
    "        shutil.rmtree(downloads_dir)\n",
    "\n",
    "    return text_pdfs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "metadata loaded for years: ['2005']; text data loaded for years: ['2005']\n"
     ]
    }
   ],
   "source": [
    "metadata, textdata = load_dataset(dataset_source, years=2005)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#implement extraction of cited articles before running the batch\n",
    "start_year = 2020\n",
    "end_year = 2025\n",
    "num_processes = 10\n",
    "download_basedir = '/home/jul/DST/recoRAG/data/pdf'\n",
    "\n",
    "for year in range(start_year, end_year+1):\n",
    "    metadata, textdata = load_dataset(dataset_source, years=year)\n",
    "    N = len(textdata)\n",
    "    batch_size = 100 #len(textdata) // num_processes\n",
    "    data4pdf = pd.merge(textdata[['paperId','openAccessPdf']], metadata[['paperId','authorName']], on='paperId', how='left')\n",
    "    textdata_list = [(data4pdf.iloc[k*batch_size:min(N, (k+1)*batch_size)].to_dict(orient='records'), os.path.join(download_basedir, str(k))) for k in range(N // batch_size + 1)]\n",
    "    textdata_list = [batch for batch in textdata_list if len(batch[0]) > 0]\n",
    "    with multiprocessing.Pool(processes=num_processes) as pool:\n",
    "        results = list(pool.imap(download_pdfs, textdata_list))\n",
    "\n",
    "    pdf_texts = pd.DataFrame([pdf for result in results for pdf in result])\n",
    "    textdata_pdf = pd.merge(left=textdata, right=pdf_texts, on='paperId', how='left')\n",
    "\n",
    "    filepath = os.path.join(path_to_data, 'text', str(year), f'{dataset_source}_textpdf_{year}.parquet')\n",
    "    textdata_pdf.to_parquet(filepath, engine=\"pyarrow\", compression=\"snappy\", index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Merging with text database and saving\n",
    "pdf_texts = pd.DataFrame([pdf for result in results for pdf in result])\n",
    "textdata_pdf = pd.merge(left=textdata, right=pdf_texts, on='paperId', how='left')\n",
    "# textdata_pdf.to_parquet(file_path + '_text_pdf.parquet', engine=\"pyarrow\", compression=\"snappy\", index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "6e2c7927-a286-4418-949a-7fa13c561047",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "Disruption of Fibroblast Growth Factor Signal Pathway Inhibits the Growth of Synovial Sarcomas: Potential Application of Signal Inhibitors to Molecular Target Therapy",
         "Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which are unknown. We investigated the involvement of fibroblast growth factor (FGF) signals in synovial sarcoma and evaluated the therapeutic effect of inhibiting the FGF signal. Experimental Design: The expression of 22 FGF and 4 FGF receptor (FGFR) genes in 18 primary tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR. Effects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein kinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory effects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo. Results: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed in neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma cell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal–regulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the FGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle arrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the FGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not p38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sarcoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2. Conclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibitory molecules will be of potential use for molecular target therapy in synovial sarcoma.",
         "",
         "https://aacrjournals.org/clincancerres/article-pdf/11/7/2702/1962059/2702-2712.pdf"
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "Shuffled iterative decoding",
         "Shuffled versions of iterative decoding of low-density parity-check codes and turbo codes are presented. The proposed schemes have about the same computational complexity as the standard versions, and converge faster. Simulations show that the new schedules offer better performance/complexity tradeoffs, especially when the maximum number of iterations has to remain small.",
         "Low-Density Parity-Check Codes Turbo decoder architectures for low-density parity-check codes Shuffled belief propagation decoding A parallel decoder for low latency decoding of turbo product codes Parallel versus sequential updating for belief propagation decoding High throughput low-density parity-check decoder architectures Good error-correcting codes based on very sparse matrices Near optimum error correcting coding and decoding: turbo-codes Multiple turbo codes for deep-space communications Optimal decoding of linear codes for minimizing symbol error rate (Corresp.) Generic Architecture for LDPC Codes Decoding Optimal decoding of linear codes for minimizing symbol error rate",
         "http://www.merl.com/publications/docs/TR2005-063.pdf"
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "Diphasic effects of Astragalus membranaceus BUNGE (Leguminosae) on vascular tone in rat thoracic aorta.",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus BUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath. In endothelium-intact aortic rings, A. membranaceus extract induced a significant dose-dependent relaxation of the rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(omega)-nitro-arginine methyl ester or methylthioninium chloride. In endothelium-denuded ones, the extract could dose-dependently relax the rings contracted by phenylephrine, not by KCl; and it could also attenuate contractile response to phenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca(2+)-free medium; but it failed to affect the CaCl(2)-induced enhancement of contractile response to phenylephrine in Ca(2+)-free medium. These results indicate that nitric oxide signaling and Ca(2+)-handling pathway are involved in the A. membranaceus extract-induced vasodilatation.",
         "",
         "https://www.jstage.jst.go.jp/article/bpb/28/8/28_8_1450/_pdf"
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "Critical roles of CD146 in zebrafish vascular development",
         "In this report, we use zebrafish as a model system to understand the importance of CD146 in vascular development. Endothelial‐specific expression of CD146 was verified by whole‐mount in situ hybridization. Suppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly developed intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood flow through the ISV region, despite that fluorescence microangiography showed that the ISVs were present. This finding suggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally, remodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of CD146 protein expression resulted in a circulation shunt that caused the circulation to by‐pass part of the caudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino oligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N terminus deleted in an in‐frame manner. This region, therefore, is likely to contain elements critical for CD146 function. This study provides the first in vivo functional assessment of CD146 in embryonic development by showing that knockdown of CD146 protein expression severely hinders vascular development in zebrafish. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley‐Liss, Inc.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/dvdy.20220"
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "Differential Contribution of Troponin I Phosphorylation Sites to the Endothelin-modulated Contractile Response*",
         "Cardiac troponin I is a phosphorylation target for endothelin-activated protein kinase C. Earlier work in cardiac myocytes expressing nonphosphorylatable slow skeletal troponin I provided evidence that protein kinase C-mediated cardiac troponin I phosphorylation accelerates relaxation. However, replacement with the slow skeletal isoform also alters the myofilament pH response and the Ca2+ transient, which could influence endothelin-mediated relaxation. Here, differences in the Ca2+ transient could not explain the divergent relaxation response to endothelin in myocytes expressing cardiac versus slow skeletal troponin I nor could activation of Na+/H+ exchange. Three separate clusters within cardiac troponin I are phosphorylated by protein kinase C, and we set out to determine the contribution of the Thr144 and Ser23/Ser24 clusters to the endothelin-mediated contractile response. Myocyte replacement with a cardiac troponin I containing a Thr144 substituted with the Pro residue found in slow skeletal troponin I resulted in prolonged relaxation in response to acute endothelin compared with control myocytes. Ser23/Ser24 also is a target for protein kinase C phosphorylation of purified cardiac troponin I, and although this cluster was not acutely phosphorylated in intact myocytes, significant phosphorylation developed within 1 h after adding endothelin. Replacement of Ser23/Ser24 with Ala indicated that this cluster contributes significantly to relaxation during more prolonged endothelin stimulation. Overall, results with these mutants provide evidence that Thr144 plays an important role in the acute acceleration of relaxation, whereas Ser23/Ser24 contributes to relaxation during more prolonged activation of protein kinase C by endothelin.",
         "",
         "http://www.jbc.org/article/S0021925820589394/pdf"
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "Programming: factors that influence success",
         "This paper documents a study, carried out in the academic year 2003-2004, on fifteen factors that may influence performance on a first year object-oriented programming module. The factors included prior academic experience, prior computer experience, self-perception of programming performance and comfort level on the module and specific cognitive skills. The study found that a student's perception of their understanding of the module had the strongest correlation with programming performance, r=0.76, p‹0.01. In addition, Leaving Certificate (LC) mathematics and science scores were shown to have a strong correlation with performance. A regression module, based upon a student's perception of their understanding of the module, gender, LC mathematics score and comfort level was able to account for 79% of the variance in programming performance results.",
         "The impact of prior experience in an information technology programming course sequence Predictors of success and failure in a CS1 course A multi-national, multi-institutional study of assessment of programming skills of first-year CS students The effect of student attributes on success in programming Contributing to success in an introductory computer science course: a study of twelve factors Does it help to have some programming experience before beginning a computing degree program? Factors affecting performance in first-year computing The effect of a constructivist learning environment for field-dependent/independent students on achievement in introductory computer programming What best predicts computer proficiency? Predicting student performance in a beginning computer science class Predicting the success of freshmen in a computer science major Predicting student success in an introductory programming course Predicting success in a first programming course Investigating the relationship between the development of abstract reasoning and performance in an introductory programming class Factors affecting course outcomes in introductory programming An Overview of the Integration of Problem Based Learning into an existing Computer Science Programming Module Discovering the likely performance of novice programmers on an introductory programming module Differential Performance by gender and achievement in the Certificate Examination Results 2000/2001: Final Report, Dublin: National Council for Curriculum and Assessment Examining Gender Cognitive skills needed in computer programming speciﬁc cognitive skills: as measured by an in-house cognitive test previous academic and computer experience: as measured by performance in the Irish Leaving Certiﬁcate (LC) examinations in",
         "http://dl.acm.org/ft_gateway.cfm?id=1047480&type=pdf"
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "Intracellular and Surface Distribution of Monocyte Tissue Factor: Application to Intersubject Variability",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced monocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and shedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs). Methods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of TF-positive monocytes in HRs (32.0±3.5%) versus LRs (11.2±1.2%; P≤0.05), along with a stronger mean fluorescence intensity of TF signal in HRs (7.1±0.5 arbitrary units [AU]) compared with LRs (5.4±0.4 AU; P≤0.05). The LPS-treated blood of the HR group contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular accumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs versus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of surface TF, whereas MNCs from LRs only had a minor increase in surface TF levels. Conclusions—The higher availability of surface TF antigen on MNCs from HRs and TF-containing microparticles might make these individuals more susceptible to hypercoagulation.",
         "The production and availability of tissue thromboplastin in cellular populations of whole blood exposed to various concentrations of endotoxin. An assay for detection of endotoxin. Tissue factor activity in whole blood. Nicotinamide inhibits endotoxin‐induced monocyte tissue factor expression Cellular activation responses in blood in relation to lipid pattern: healthy men and women in families with myocardial infarction or cancer Tissue factor activity in human monocytes is regulated by plasma: implications for the high and low responder phenomenon Role of Platelet P-Selectin and CD40 Ligand in the Induction of Monocytic Tissue Factor Expression Eicosanoids and Iso-Eicosanoids: Constitutive, Inducible and Transcellular Biosynthesis in Vascular Disease Cytokine gene polymorphism and heart transplant rejection. Platelet-derived growth factor-BB and monocyte chemotactic protein-1 induce human peripheral blood monocytes to express tissue factor. Association of smooth muscle cell tissue factor with caveolae. Manifestation of cryptic fibroblast tissue factor occurs at detergent concentrations which dissolve the plasma membrane A novel biological effect of platelet factor 4 (PF4): enhancement of LPS‐induced tissue factor activity in monocytes Th1 and Th2 T-helper cells exert opposite regulatory effects on procoagulant activity and tissue factor production by human monocytes. Granulocytes enhance LPS‐induced tissue factor activity in monocytes via an interaction with platelets Cathepsin G-dependent platelet stimulation by activated polymorphonuclear leukocytes and its inhibition by antiproteinases: role of P-selectin-mediated cell-cell adhesion. Relations between factor VIIa binding and expression of factor VIIa/tissue factor catalytic activity on cell surfaces. Platelet activating factor enhancement of lipopolysaccharide‐induced tissue factor activity in monocytes: requirement of platelets and granulocytes Dysregulation of in vitro cytokine production by monocytes during sepsis. The role of arachidonic acid release and lipoxygenase pathway in lipopolysaccharide-induced thromboplastin activity in monocytes. In vitro production of IL 1β, IL 1α, TNF and IL 2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation Functional tissue factor is entirely cell surface expressed on lipopolysaccharide-stimulated human blood monocytes and a constitutively tissue factor-producing neoplastic cell line Localization of tissue factor in the normal vessel wall and in the atherosclerotic plaque. Cytokine-induced procoagulant activity in monocytes and endothelial cells. Further enhancement by cyclosporine. Tumor necrosis factor induces tissue factor-like activity in human leukemia cell line U937 and peripheral blood monocytes. Enhancement of mononuclear procoagulant activity by platelet 12-hydroxyeicosatetraenoic acid. The stimulatory effect of platelets and platelet membranes on the procoagulant activity of leukocytes. The Role of Protein Synthesis on the Generation of Tissue Factor Activity by Leukocytes Monocytes. The high responder phenomenon: enhancement of LPS induced tissue factor activity in monocytes by platelets and granulocytes. Selective cellular expression of tissue factor in human tissues. Implications for disorders of hemostasis and thrombosis. In vitro production of IL 1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation. Mononuclear cell tissue factor: cell of origin and requirements for activation. After blocking with 5% non-fat skimmed milk, membranes were incubated with mouse anti-human TF monoclonal antibodies After Lymphoprep isolation, total RNA was extracted from MNC pellets with an OR) anti-human TF monoclonal antibodies (10 µg/ml) for 30 min on ice. Mouse monoclonal antibodies recognizing extracellular domain of human TF were prepared by the Monoclonal Antibody Facility",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.ATV.0000168413.29874.d7"
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "Potential Clindamycin Resistance in Clindamycin-Susceptible, Erythromycin-Resistant Staphylococcus aureus: Report of a Clinical Failure",
         "ABSTRACT The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin clinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-susceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of methicillin-resistant S. aureus strains were D-test positive.",
         "",
         "https://europepmc.org/articles/pmc549252?pdf=render"
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         "Symmetric and asymmetric Slepian-Wolf codes with systematic and nonsystematic linear codes",
         "We propose a constructive approach for distributed source coding of correlated binary sources using linear channel codes that can achieve any point of the Slepian-Wolf achievable rate region. Our approach is very intuitive and can be used with systematic and nonsystematic linear codes. Moreover, the proposed coding strategy can easily be extended to the case of more than two sources.",
         "Design of Slepian-Wolf codes by channel code partitioning On some new approaches to practical Slepian-Wolf compression inspired by channel coding Distributed code constructions for the entire Slepian-Wolf rate region for arbitrarily correlated sources Compression of binary sources with side information at the decoder using LDPC codes Compression with side information using turbo codes Compression of correlated binary sources using turbo codes The capacity of low-density parity-check codes under message-passing decoding Distributed source coding: symmetric rates and applications to sensor networks Distributed source coding using syndromes (DISCUS): design and construction Noiseless coding of correlated information sources",
         "http://www.commsp.ee.ic.ac.uk/~ng1/pdf/gehrigD_CL_Jan05.pdf"
        ],
        [
         "9",
         "000a2234245d642c786a7355a15623444c33c1dc",
         "Selective Defect in Antigen-Induced TCR Internalization at the Immune Synapse of CD8 T Cells Bearing the ZAP-70(Y292F) Mutation1",
         "Cbl proteins have been implicated in ligand-induced TCR/CD3 down-modulation, but underlying mechanisms are unclear. We analyzed the effect of mutation of a cbl-binding site on ZAP-70 (ZAP-Y292F) on dynamics, internalization, and degradation of the TCR/CD3 complex in response to distinct stimuli. Naive CD8 T cells expressing the P14 transgenic TCR from ZAP-Y292F mice were selectively affected in TCR/CD3 down-modulation in response to antigenic stimulation, whereas neither anti-CD3 Ab-, and PMA-induced TCR down-modulation, nor constitutive receptor endocytosis/cycling were impaired. We further established that the defect in TCR/CD3 down-modulation in response to Ag was paralleled by an impaired TCR/CD3 internalization and CD3ζ degradation. Analysis of T/APC conjugates revealed that delayed redistribution of TCR at the T/APC contact zone was paralleled by a delay in TCR internalization in the synaptic zone in ZAP-Y292F compared with ZAP-wild-type T cells. Cbl recruitment to the synapse was also retarded in ZAP-Y292F T cells, although F-actin and LFA-1 redistribution was similar for both cell types. This study identifies a step involving ZAP-70/cbl interaction that is critical for rapid internalization of the TCR/CD3 complex at the CD8 T cell/APC synapse.",
         "",
         "https://www.jimmunol.org/content/jimmunol/175/5/3140.full.pdf"
        ],
        [
         "10",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "Applying Machine Learning Techniques to Improve Linux Process Scheduling",
         "In this work we use Machine Learning (ML) techniques to learn the CPU time-slice utilization behavior of known programs in a Linux system. Learning is done by an analysis of certain static and dynamic attributes of the processes while they are being run. Our objective was to discover the most important static and dynamic attributes of the processes that can help best in prediction of CPU burst times which minimize the process TaT (Turn-around-Time). In our experimentation we modify the Linux Kernel scheduler (version 2.4.20-8) to allow scheduling with customized time slices. The \"Waikato Environment for Knowledge Analysis\" (Weka), an open source machine-learning tool is used to find the most suitable ML method to characterize our programs. We experimentally find that the C'4.5 Decision Tree algorithm most effectively solved the problem. We find that predictive scheduling could reduce TaT in the range of 1.4% to 5.8%. This was due to a reduction in the number of context switches needed to complete the process execution. We find our result interesting in the context that generally operating systems presently never make use of a program's previous execution history in their scheduling behavior.",
         "Performance Contracts: Predicting and Monitoring Grid Application Behavior The design, implementation and initial evaluation of an advanced knowledge-based process scheduler Understanding the Linux Kernel Predicting Application Run Times Using Historical Information A Historical Application Profiler for Use by Parallel Schedulers Context Switching and IPC Performance Comparison between uClinux and Linux on the ARM9 based Processor Characterizing Process Execution Behaviour Using Machine Learning Techniques The Pearson Education Linux Kernel Development , 1st ed Correlation-based Feature Selection for Machine Learning A Dynamically Adaptive CPU Scheduler Linux Kernel 2.4 Internals Co-rrelation based feature selection for Machine Learning, Master Thesis, Department of Computer Science, University of Waikato Machine Learning , 1st ed, pp: 52-75, 154-183, 230-244, The WEKA: The Waikato Environment for Knowledge Analysis ELF: From The Programmer's Perspective The Design of the UNIX Operating System Build the knowledge base of static and dynamic characteristics of the programs from the run traces obtained in step 1 and train them with the C4.5 decision tree algorithm is a Masters student in the Department of Computer and Informations Sciences We send this STS information to modi(cid:2)ed scheduler through a system call The scheduler instructs the CPU such that CPU allocates STS ticks to ’X’ decision tree will classify ’X’ and output the STS. per second.(cid:2)le Aivazian , Tigran , Linux Kernel 2 . 4 Internals , The Linux Documentation Project , August , 2002",
         "http://www.cs.ucr.edu/%7Ekishore/papers/tencon.pdf"
        ],
        [
         "11",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "Neuronal Leucine-Rich Repeat Protein 4 Functions in Hippocampus-Dependent Long-Lasting Memory",
         "ABSTRACT Neuronal leucine-rich repeat proteins (NLRRs) are type I transmembrane proteins and expressed in neuronal tissues, but their function remains unknown. Here, we describe the identification and characterization of a new member of the NLRR family, NLRR4, and its potential role in long-lasting memory. We generated NLRR4-deficient (NLRR4−/−) mice and found that they showed impaired memory retention. In hippocampus-dependent learning tasks, NLRR4−/− mice were able to learn and maintain the memories for one day but unable to retain the memories for four days after learning. In contrast, in a hippocampus-independent task, NLRR4−/− mice were able to retain the memory normally for at least seven days. These results suggest that NLRR4 plays a key role in hippocampus-dependent long-lasting memory.",
         "",
         "https://europepmc.org/articles/pmc1087730?pdf=render"
        ],
        [
         "12",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "Semantic video analysis for adaptive content delivery and automatic description",
         "We present an encoding framework which exploits semantics for video content delivery. The video content is organized based on the idea of main content message. In the work reported in this paper, the main content message is extracted from the video data through semantic video analysis, an application-dependent process that separates relevant information from non relevant information. We use here semantic analysis and the corresponding content annotation under a new perspective: the results of the analysis are exploited for object-based encoders, such as MPEG-4, as well as for frame-based encoders, such as MPEG-1. Moreover, the use of MPEG-7 content descriptors in conjunction with the video is used for improving content visualization for narrow channels and devices with limited capabilities. Finally, we analyze and evaluate the impact of semantic video analysis in video encoding and show that the use of semantic video analysis prior to encoding sensibly reduces the bandwidth requirements compared to traditional encoders not only for an object-based encoder but also for a frame-based encoder.",
         "Automatic foveation for video compression using a neurobiological model of visual attention Text information extraction in images and video: a survey DAVE: a system for quality driven adaptive video delivery Video transmission adaptation on mobile devices Visual attention for region of interest coding in JPEG 2000 Semantic segmentation and description for video transcoding Resource adaptation based on MPEG-21 usage environment descriptions Video transcoding architectures and techniques: an overview Metadata-driven multimedia access Semantic transcoding for live video server Multiple video object tracking in complex scenes Complexity-quality analysis of transcoding architectures for reduced spatial resolution Instantly indexed multimedia databases of real world events Face detection in color images MPEG-7 description of generic video objects for scene reconstruction A new content-based hybrid video transcoding method Video Processing and Communications Media conversions to support mobile users Object-based transcoding for adaptable video content delivery Overview of fine granularity scalability in MPEG-4 video standard Video object extraction based on adaptive background and statistical change detection Video transcoding for universal multimedia access Heterogeneous Video Transcoding to Lower Spatio-Temporal Resolutions and Different Encoding Formats Scalable multimedia delivery for pervasive computing Adapting Multimedia Internet Content for Universal Access A frequency-domain video transcoder for dynamic bit-rate reduction of MPEG-2 bit streams Multimedia content description in the InfoPyramid Objective methods for assessment of video quality : state of the art On the methods and performances of rational downsizing video transcoding Shadow detection in image sequences Video Transcoding Architectures and Techniques : An Overview Visual radio: should we paint pictures with workds, or pictures? Dave: A system for qual ity driven adaptive video delivery,”Proceedings of the 5th ACM SIGMM international workshop on Multimedia information retriev Processing and Information technology – coding of audio-visual objects – part 2 visual–amendment 2: Streaming video profiles Statistics, 3 ed Continuous speech recognit  io Submitted by Information technology — Generic coding of moving pictures and associated audio information — Part 2: Video SUBJECTIVE VIDEO QUALITY ASSESSMENT METHODS FOR MULTIMEDIA APPLICATIONS",
         "https://infoscience.epfl.ch/record/87259/files/Cavallaro2005_1275.pdf"
        ],
        [
         "13",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "Trends in Drug Resistance, Serotypes, and Molecular Types of Streptococcus pneumoniae Colonizing Preschool-Age Children Attending Day Care Centers in Lisbon, Portugal: a Summary of 4 Years of Annual Surveillance",
         "ABSTRACT Of the nasopharyngeal cultures recovered from 942 day care center (DCC) attendees in Lisbon, Portugal, 591 (62%) yielded Streptococcus pneumoniae during a surveillance performed in February and March of 1999. Forty percent of the isolates were resistant to one or more antimicrobial agents. In particular, 2% were penicillin resistant and 20% had intermediate penicillin resistance. Multidrug resistance to macrolides, lincosamides, and tetracycline was the most frequent antibiotype (17% of all isolates). Serotyping and molecular typing by pulsed-field gel electrophoresis were performed for 202 out of 237 drug-resistant pneumococci (DRPn). The most frequent serotypes were 6B (26%), 14 (22%), 19F (16%), 23F (10%), and nontypeable (12%). The majority (67%) of the DRPn strains were representatives of nine international clones included in the Pneumococcal Molecular Epidemiology Network; eight of them had been detected in previous studies. Fourteen novel clones were identified, corresponding to 26% of the DRPn strains. The remaining 7% of the strains were local clones detected in our previous studies. Comparison with studies conducted since 1996 in Portuguese DCCs identified several trends: (i) the rate of DRPn frequency has fluctuated between 40 and 50%; (ii) the serotypes most frequently recovered have remained the same; (iii) nontypeable strains appear to be increasing in frequency; and (iv) a clone of serotype 33F emerged in 1999. Together, our observations highlight that the nasopharynxes of children in DCCs are a melting pot of successful DRPn clones that are important to study and monitor if we aim to gain a better understanding on the epidemiology of this pathogen.",
         "",
         "https://europepmc.org/articles/pmc1081291?pdf=render"
        ],
        [
         "14",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "Laboratory Exposures to Brucellae and Implications for Bioterrorism",
         "Laboratory exposures to brucellae indicate unfamiliarity with organism.",
         "Laboratory-acquired Brucellosis Acute hepatitis due to brucellosis in a laboratory technician. Prevention of Laboratory-Acquired Brucellosis Analysis of risk factors for laboratory-acquired brucella infections. Use of the Isolator 1.5 Microbial Tube for Detection of Brucella melitensis in Synovial Fluid Bacterial Pathogens as Biological Weapons and Agents of Bioterrorism Unusual Complication of Breast Implants: Brucella Infection Brucellosis in laboratory workers at a Saudi Arabian hospital. Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases Brucella abortus Infection Acquired in Microbiology Laboratories Posttreatment Follow-Up of Brucellosis by PCR Assay Detection of Brucellae in Blood Cultures Rapid diagnosis of human brucellosis by peripheral-blood PCR assay Failure of short antimicrobial treatments for human brucellosis Comparison of BACTEC 9240 Peds Plus medium and isolator 1.5 microbial tube for detection of Brucella melitensis from blood cultures Exposure to Brucella in the laboratory Brucellosis from sniffing bacteriological cultures Grand Rounds—Hammersmith Hospital: A case of laboratory acquired brucellosis Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases Live Brucella spp. fail to induce tumor necrosis factor alpha excretion upon infection of U937-derived phagocytes Outbreak of Brucella melitensis among microbiology laboratory workers Unsuspected brucellosis diagnosed in a child as a result of an outbreak of laboratory-acquired brucellosis. Brucellosis: imported and laboratory-acquired cases, and an overview of treatment trials. Biochemical mis-identification of Brucella melitensis and subsequent laboratory-acquired infections. Laboratory diagnosis of brucella infection: some pitfalls Outbreak of Brucella melitensis among microbiology laboratory workers in a community hospital. An outbreak of Brucella melitensis infection by airborne transmission among laboratory workers. Laboratory-acquired meningitis caused by Brucella abortus strain 19. Past and present hazards of working with infectious agents. From the Center for Disease Control Brucella canis: A cause of undulant fever An unusual outbreak of brucellosis. Brucellosis: I. Laboratory-Acquired Acute Infection Brucellosis in the United States. Brucella species Brucellosis: An occupational hazard for medical laboratory personnel. Report of five cases Beer-Sheva 84101, Israel; fax: 972-8640-3541 Prevention of laboratory-acquired brucellosis. (revised). Geneva: World Health Organization Basic protocols for level A laboratories for the presumptive identification of Brucella species. Washington: American Society for Microbiology Exposure of hospital personnel to Brucella melitensis and occurrence of laboratory-acquired disease in an endemic area. Biosafety in microbiological and biomedical laboratories Laboratory-associated infections: incidence, fatalities, causes, and prevention. Principles and practice of infectious diseases. Vols 1 and 2. Brucellosis in the United States, 1965-1974. Laboratory Infections Due to Brucella",
         "https://wwwnc.cdc.gov/eid/article/11/8/pdfs/04-1197.pdf"
        ],
        [
         "15",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "Safe motion planning in dynamic environments",
         "This paper addresses the problem of motion planning (MP) in dynamic environments. It is first argued that dynamic environments impose a real-time constraint upon MP: it has a limited time only to compute a motion, the time available being a function of the dynamicity of the environment. Now, given the intrinsic complexity of MP, computing a complete motion to the goal within the time available is impossible to achieve in most real situations. Partial motion planning (PMP) is the answer to this problem proposed in this paper. PMP is a motion planning scheme with an anytime flavor: when the time available is over, PMP returns the best partial motion to the goal computed so far. Like reactive navigation scheme, PMP faces a safety issue: what guarantee is there that the system will never end up in a critical situation yielding an inevitable collision? The answer proposed in this paper to this safety issue relies upon the concept of inevitable collision states (ICS). ICS takes into account the dynamics of both the system and the moving obstacles. By computing ICS-free partial motion, the system safety can be guaranteed. Application of PMP to the case of a car-like system in a dynamic environment is presented.",
         "Motion prediction for moving objects: a statistical approach Inevitable collision states. A step towards safer robots? Towards real-time global motion planning in a dynamic environment using the NLVO concept An integrated approach to goal-directed obstacle avoidance under dynamic constraints for dynamic environments Real-time randomized path planning for robot navigation Improved fast replanning for robot navigation in unknown terrain Learning motion patterns of persons for mobile service robots Reactive navigation for non-holonomic robots using the ego-kinematic space Randomized Kinodynamic Motion Planning with Moving Obstacles Real-time motion planning for agile autonomous vehicles Kinodynamic motion planning amidst moving obstacles VFH/sup */: local obstacle avoidance with look-ahead verification Randomized Kinodynamic Planning High-speed navigation using the global dynamic window approach Motion Planning in Dynamic Environments Using Velocity Obstacles The dynamic window approach to collision avoidance Probabilistic roadmaps for path planning in high-dimensional configuration spaces The curvature-velocity method for local obstacle avoidance The Focussed D* Algorithm for Real-Time Replanning Reflexive collision avoidance: a generalized approach Motion planning in the presence of moving obstacles Robot Motion Planning Sensor-Based Motion Control for Mobile Robots Kinodynamic planning in a structured and time-varying 2-D workspace",
         "https://hal.inria.fr/inria-00182046/file/2005-petti-fraichard-iros.pdf"
        ],
        [
         "16",
         "0012aecca19e4d3e41741599b1dbd00fb46d0c22",
         "Glutathione S-transferase T1 polymorphisms are associated with outcome in colorectal cancer.",
         "Colorectal cancer (CRC) remains a significant cause of mortality accounting for approximately 10% of all deaths from malignancy in the western world. Polymorphism in the glutathione S-transferase GSTT1 gene has been associated with CRC risk in some but not all studies. In this study, we examined associations between GSTT1 genotypes and CRC risk, and prognosis in 361 cases and 881 unrelated controls. GSTT1 null was associated with a small but significant increase in risk (P = 0.0006, odds ratio (OR) = 1.65, 95% confidence interval (CI) = 1.22-2.24). GSTT1 null was also associated with a significantly younger age at diagnosis (mean 65.2 years) compared with GSTT1 A (mean 67.6 years, P = 0.031). There were no significant associations between GSTT1 genotypes and clinical factors (e.g. Dukes stage, differentiation and tumour node metastasis classification) in the total case group. However, following stratification by age (<70 versus > or =70 years at diagnosis), in the patients diagnosed <70 years of age, GSTT1 null was more common in Dukes grade A/B tumours (P = 0.046), stage T1/T2 tumours (P = 0.053) and those with a pushing margin (P = 0.066). We also identified associations between GSTT1 null and increased prevalence of host lymphocyte response, particularly in the younger patients (P = 0.036). Furthermore, GSTT1 null was associated with improved survival in younger patients (P = 0.017, hazards ratio (HR) = 0.52, 95% CI = 0.31-0.89) but poorer survival in older patients (P = 0.017, HR = 1.89, 95% CI = 1.12-3.20). We proposed a model based on the dual functionality of GSTT1 to explain these contrasting results. We suggest that the null genotype is associated with improved immune response in younger patients, but poorer detoxification in older patients. These findings may also provide an explanation for the contrasting finding of other studies on the role of this gene in CRC.",
         "",
         "https://academic.oup.com/carcin/article-pdf/26/12/2157/7088641/bgi195.pdf"
        ],
        [
         "17",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "The N‐terminus of B96Bom, a Bombyx mori G‐protein‐coupled receptor, is N‐myristoylated and translocated across the membrane",
         "In eukaryotic cellular proteins, protein N‐myristoylation has been recognized as a protein modification that occurs mainly on cytoplasmic or nucleoplasmic proteins. In this study, to search for a eukaryotic N‐myristoylated transmembrane protein, the susceptibility of the N‐terminus of several G‐protein‐coupled receptors (GPCRs) to protein N‐myristoylation was evaluated by in vitro and in vivo metabolic labeling. It was found that the N‐terminal 10 residues of B96Bom, a Bombyx mori GPCR, efficiently directed the protein N‐myristoylation. Analysis of a tumor necrosis factor (TNF) fusion protein with the N‐terminal 90 residues of B96Bom at its N‐terminus revealed that (a) transmembrane domain 1 of B96Bom functioned as a type I signal anchor sequence, (b) the N‐myristoylated N‐terminal domain (58 residues) was translocated across the membrane, and (c) two N‐glycosylation motifs located in this domain were efficiently N‐glycosylated. In addition, when Ala4 in the N‐myristoylation motif of B96Bom90‐TNF, Met‐Gly‐Gln‐Ala‐Ala‐Thr(1–6), was replaced with Asn to generate a new N‐glycosylation motif, Asn‐Ala‐Thr(4–6), efficient N‐glycosylation was observed on this newly introduced N‐glycosylation site in the expressed protein. These results indicate that the N‐myristoylated N‐terminus of B96Bom is translocated across the membrane and exposed to the extracellular surface. To our knowledge, this is the first report showing that a eukaryotic transmembrane protein can be N‐myristoylated and that the N‐myristoylated N‐terminus of the protein can be translocated across the membrane.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2004.04487.x"
        ],
        [
         "18",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "Uptake of denatured collagen into hepatic stellate cells: evidence for the involvement of urokinase plasminogen activator receptor-associated protein/Endo180.",
         "Tissue remodelling is dependent on the integration of signals that control turnover of ECM (extracellular matrix). Breakdown and endocytosis of collagen, a major component of the ECM, is central to this process. Whereas controlled secretion of matrix-degrading enzymes (such as matrix metalloproteinases) has long been known to mediate ECM breakdown, it is becoming clear that uPARAP/Endo180 (where uPARAP stands for urokinase plasminogen activator receptor-associated protein) serves as a receptor that mediates endocytosis of collagen by several types of cells. In the liver, the stellate cells play a major role in turnover of ECM including collagens. These cells synthesize various collagens and also produce matrix metalloproteinases. In the present study, we investigated the capacity of rat hepatic stellate cells to endocytose and degrade 125I-labelled heat-denatured collagen I. It was found that the collagen is efficiently taken up and degraded by these cells. Degradation was inhibited by inhibitors of lysosomal proteases (leupeptin and E-64d) and the vacuolar proton pump (concanamycin A), indicating that it takes place in lysosomes. Furthermore, endocytosed FITC-labelled collagen was shown to reach late endocytic compartments in which it colocalized with LysoTracker (a marker of late endocytic compartments). Competition experiments showed that uPA and unlabelled collagen are capable of inhibiting binding and uptake of [125I]collagen in a dose-dependent manner. Moreover, Western-blot analysis of cell lysate (using a polyclonal rabbit human-Endo180 antiserum) revealed a single band at 180 kDa. In addition, the antiserum was capable of reducing [125I]collagen binding to the cell surface. Finally, using two primers designed from the human uPARAP/Endo180 mRNA sequence, the expression of uPARAP/Endo180 mRNA was detected by reverse transcriptase-PCR. These results together suggest that uPARAP/Endo180 mediates endocytosis of collagen in rat liver stellate cells.",
         "uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV. Identification and characterization of the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor. A targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen uptake Liver extracellular matrix in health and disease Hepatic stellate cells: unique characteristics in cell biology and phenotype. uPARAP/Endo180 is essential for cellular uptake of collagen and promotes fibroblast collagen adhesion Characterization of Sugar Binding by the Mannose Receptor Family Member, Endo180* The mannose receptor family. The C-type Lectin Receptor Endo180 Displays Internalization and Recycling Properties Distinct from Other Members of the Mannose Receptor Family* Collagen-receptor signaling in health and disease. The Urokinase Plasminogen Activator Receptor–Associated Protein/Endo180 Is Coexpressed with Its Interaction Partners Urokinase Plasminogen Activator Receptor and Matrix Metalloprotease-13 during Osteogenesis Hepatic Blood Flow Regulation by Stellate Cells in Normal and Injured Liver Extracellular Matrix Degradation and the Role of Hepatic Stellate Cells Wortmannin-sensitive trafficking steps in the endocytic pathway in rat liver endothelial cells. Effects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy. Endo180, an endocytic recycling glycoprotein related to the macrophage mannose receptor is expressed on fibroblasts, endothelial cells and macrophages and functions as a lectin receptor. Selective adhesion of macrophages to denatured forms of type I collagen is mediated by scavenger receptors. A Urokinase Receptor-associated Protein with Specific Collagen Binding Properties* The C‐type lectin superfamily in the immune system An orphan receptor tyrosine kinase family whose members serve as nonintegrin collagen receptors. The discoidin domain receptor tyrosine kinases are activated by collagen. Vascular Smooth Muscle αvβ3 Integrin Mediates Arteriolar Vasodilation in Response to RGD Peptides Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis. Cell adhesion receptors for native and denatured type I collagens and fibronectin in rabbit arterial smooth muscle cells in culture. An endothelial cell receptor for plasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of t-PA-dependent plasminogen activation. Differential expression of matrix-metalloproteinase-1 and -2 genes in normal and fibrotic human liver. Transfer of retinol-binding protein from HepG2 human hepatoma cells to cocultured rat stellate cells. Fat-storing cells as liver-specific pericytes. Spatial dynamics of agonist-stimulated intracellular calcium transients. Affinity of integrins for damaged extracellular matrix: αvβ3 binds to denatured collagen type I through RGD sites Identification of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin in collagen. Internalization of retinol-binding protein in parenchymal and stellate cells of rat liver. p180, a novel recycling transmembrane glycoprotein with restricted cell type expression The alpha 2 beta 1 integrin cell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen. Activation of cultured rat hepatic lipocytes by Kupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation of cell proliferation via induction of platelet-derived growth factor receptors. Characterization of desmin‐positive rat liver sinusoidal cells Hepatic lipocytes: the principal collagen-producing cells of normal rat liver. A radioiodinated, intracellularly trapped ligand for determining the sites of plasma protein degradation in vivo. The effect of leupeptin on intracellular digestion of asialofetuin in rat hepatocytes. The structures of the galactose-containing sugar chains of ovalbumin. HIGH-YIELD PREPARATION OF ISOLATED RAT LIVER PARENCHYMAL CELLS Distribution of Endo180 receptor and ligand in developing articular cartilage. Endocytic uptake of advanced glycation end products by mouse liver sinusoidal endothelial cells is mediated by a scavenger receptor distinct from the macrophage scavenger receptor class A. Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-α and TGF-β1 Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1. Vascular smooth muscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD peptides. Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis. Affinity of integrins for damaged extracellular matrix: alpha v beta 3 binds to denatured collagen type I through RGD sites. Expression of the gene of the α-smooth muscle-actin isoform in rat liver and in rat fat-storing (ITO) cells Acidification of the endocytic and exocytic pathways. Collagen production by rat liver fat-storing cells in primary culture. Newly administered [3H]retinol is transferred from hepatocytes to stellate cells in liver for storage. Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes), their related structure in and around the liver sinusoids, and vitamin A-storing cells in extrahepatic organs. Preparation of isolated rat liver cells. Hepatic stellate cells: morphology, function, and regulation Isolation of perisinusoidal stellate cells (vitamin A-storting cells, fat-storting cells) of the rat liver",
         "https://europepmc.org/articles/pmc1134930?pdf=render"
        ],
        [
         "19",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "ISEcp1B-Mediated Transposition of blaCTX-M in Escherichia coli",
         "ABSTRACT Several expanded-spectrum β-lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enterobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional mechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.",
         "",
         "https://europepmc.org/articles/pmc538921?pdf=render"
        ],
        [
         "20",
         "0018ece872bb78b06eb26ea35d5ed27f53702ce6",
         "Protein function prediction using the Protein Link EXplorer (PLEX)",
         "UNLABELLED\nWe introduce the Protein Link EXplorer (PLEX), a web-based environment that allows the construction of a phylogenetic profile for any given amino acid sequence, and its comparison with profiles of approximately 350,000 predicted genes from 89 genomes, as a means of interactively identifying functionally linked genes and predicting protein function. PLEX can be searched iteratively and also enables searches for chromosomal gene neighbors and Rosetta Stone linkages. PLEX search results are accompanied by quantitative estimates of linkage confidence, enabling users to take advantage of coinheritance, operon and gene fusion-based methods for inferring gene function and reconstructing cellular systems and pathways.\n\n\nAVAILABILITY\nhttp://bioinformatics.icmb.utexas.edu/plex",
         "",
         "https://academic.oup.com/bioinformatics/article-pdf/21/10/2558/48972005/bioinformatics_21_10_2558.pdf"
        ],
        [
         "21",
         "001a3ef673ee3e7703c7db597386a2162a2dc402",
         "How and When Selection Experiments Might Actually be Useful1",
         "Abstract Laboratory natural selection and artificial selection are vital tools for addressing specific questions about evolutionary patterns of variation. Laboratory natural selection can illuminate whether a putative selective agent is capable of generating long-term, sustained changes in individual traits and suites of traits. Artificial selection is the essential tool for understanding the general evolvability of traits and the extent to which genetic correlations constrain evolution. We review the contexts in which each type of experiment seems capable of offering key insights into important evolutionary issues. We also discuss theoretical and methodological considerations that play critical roles in designing selection experiments that are relevant to evolutionary patterns of trait variation. In particular, we focus on the critical role of selection intensity and the consequences of experiments with different intensities. While selection experiments are not practical in many cases, sophisticated selection experiments—designed with careful consideration of the theory of selection—should be taken beyond model organisms and used in well-chosen natural systems to understand natural patterns of variation.",
         "",
         "https://academic.oup.com/icb/article-pdf/45/3/391/1674343/i1540-7063-045-03-0391.pdf"
        ],
        [
         "22",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "Phylogenetic relationships of Irkut and West Caucasian bat viruses within the Lyssavirus genus and suggested quantitative criteria based on the N gene sequence for lyssavirus genotype definition.",
         "None",
         "Novel Lyssaviruses Isolated from Bats in Russia Bat lyssaviruses (Aravan and Khujand) from Central Asia: phylogenetic relationships according to N, P and G gene sequences. A comparison of DNA vaccines for the rabies-related virus, Mokola. New Lyssavirus Genotype from the Lesser Mouse-eared Bat (Myotis blythi), Kyrghyzstan A molecular epidemiological study of Australian bat lyssavirus. Competition Commission inquiry into POMs. Characterisation of an Australian bat lyssavirus variant isolated from an insectivorous bat. Phylogenetic comparison of the genus Lyssavirus using distal coding sequences of the glycoprotein and nucleoprotein genes Lyssavirus P gene characterisation provides insights into the phylogeny of the genus and identifies structural similarities and diversity within the encoded phosphoprotein. Increased positive selection pressure in persistent (SSPE) versus acute measles virus infections. European Bat Lyssavirus Infection in Spanish Bat Populations Serologic Evidence of Lyssavirus Infections among Bats, the Philippines Genetic constraints and the adaptive evolution of rabies virus in nature. Expression of the B-cell and T-cell epitopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG and induction of an humoral response in mice. MEGA2: molecular evolutionary genetics analysis software Extensive Attenuation of Rabies Virus by Simultaneously Modifying the Dynein Light Chain Binding Site in the P Protein and Replacing Arg333 in the G Protein Molecular basis for the interaction between rabies virus phosphoprotein P and the dynein light chain LC8: dissociation of dynein-binding properties and transcriptional functionality of P. Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains Host Switching in Lyssavirus History from the Chiroptera to the Carnivora Orders Evidence of Two Lyssavirus Phylogroups with Distinct Pathogenicity and Immunogenicity Antigenic and genetic divergence of rabies viruses from bat species indigenous to Canada. Codon-substitution models for heterogeneous selection pressure at amino acid sites. Sequencing and position analysis of the glycoprotein gene of four Chinese rabies viruses Ecology and evolution of rabies virus in Europe. Phylogeographic patterns exhibited by Ontario rabies virus variants Chimeric Lyssavirus Glycoproteins with Increased Immunological Potential Multiple sequence alignment with Clustal X. Rabies virus quasispecies: implications for pathogenesis. DNA-based immunization for exploring the enlargement of immunological cross-reactivity against the lyssaviruses. PAML: a program package for phylogenetic analysis by maximum likelihood Evolution of European bat lyssaviruses. The complete Mokola virus genome sequence: structure of the RNA-dependent RNA polymerase. A Biographical History of Veterinary Pathology Ten-year survey of British bats for the existence of rabies Encephalitis caused by a Lyssavirus in fruit bats in Australia. Characterization of a unique variant of bat rabies virus responsible for newly emerging human cases in North America. The design of strain-specific polymerase chain reactions for discrimination of the racoon rabies virus strain from indigenous rabies viruses of Ontario. Genetic polymorphism in the rabies virus nucleoprotein gene. Molecular epidemiology of lyssaviruses: focus on the glycoprotein and pseudogenes. Molecular diversity of the Lyssavirus genus. Isolation and characterization of 115 street rabies virus isolates from Ethiopia by using monoclonal antibodies: identification of 2 isolates as Mokola and Lagos bat viruses. [A lyssavirus with an unusual antigenic structure isolated from a bat in southern Kyrgyzstan]. Rapid sequence evolution of street rabies glycoprotein is related to the highly heterogeneous nature of the viral population. PCR technique as an alternative method for diagnosis and molecular epidemiology of rabies virus. T helper cell epitope of rabies virus nucleoprotein defined by tri‐ and tetrapeptides Molecular cloning and complete nucleotide sequence of the attenuated rabies virus SAD B19. Rabies-related Yuli virus; identification with a panel of monoclonal antibodies. Efficacy of rabies vaccines against Duvenhage virus isolated from European house bats (Eptesicus serotinus), classic rabies and rabies-related viruses. Immunity against the European bat rabies (Duvenhage) virus induced by rabies vaccines: an experimental study in mice. Localization and immunological characterization of antigenic domains of the rabies virus internal N and NS proteins. Walking along the rabies genome: is the large G-L intergenic region a remnant gene? HUMAN RABIES OF BAT ORIGIN IN EUROPE Rabies in the Tropics Rabies-related viruses. Rabies virus infection of a flying fox bat, Pteropus policephalus in Chandigarh, Northern India. Giulio Alfredo Maccacaro. Experimental infection of dogs and monkeys with two rabies serogroup viruses, Lagos bat and Mokola (IbAn 27377): clinical, serologic, virologic, and fluorescent-antibody studies. An unusual case of human rabies thought to be of chiropteran origin. Two African Viruses Serologically and Morphologically Related to Rabies Virus Isolation of Rabies Virus from Fruit Bats in Thailand Isolation of a virus from Nigerian fruit bats. New Cases of Mokola Virus Infection in South Africa: A Genotypic Comparison of Southern African Virus Isolates BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT MODELTEST: testing the model of DNA substitution Immunoprotection by rabies virus nucleoprotein. Nucleotide and deduced amino acid sequences of the nominal nonstructural phosphoprotein of the ERA, PM and CVS-11 strains of rabies virus. Rabies and Rabies-related viruses in Zimbabwe historical, virological and ecological aspects. Rabies-Related Viruses Characterization of an antigenic determinant of the glycoprotein that correlates with pathogenicity of rabies virus. Antigenic characterization of the bangui strain (ANCB-672d) of lagos bat virus",
         "https://zenodo.org/record/1259441/files/article.pdf"
        ],
        [
         "23",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         "Neuronal Signals in the Monkey Basolateral Amygdala during Reward Schedules",
         "The amygdala is critical for connecting emotional reactions with environmental events. We recorded neurons from the basolateral complex of two monkeys while they performed visually cued schedules of sequential color discrimination trials, with both valid and random cues. When the cues were valid, the visual cue, which was present throughout each trial, indicated how many trials remained to be successfully completed before a reward. Seventy-six percent of recorded neurons showed response selectivity, with the selectivity depending on some aspects of the current schedule. After a reward, when the monkeys knew that the upcoming cue would be valid, 88 of 246 (36%) neurons responded between schedules, seemingly anticipating the receiving information about the upcoming schedule length. When the cue appeared, 102 of 246 (41%) neurons became selective, at this point encoding information about whether the current trial was the only trial required or how many more trials are needed to obtain a reward. These cue-related responses had a median latency of 120 ms (just between the latencies in inferior temporal visual area TE and perirhinal cortex). When the monkey was releasing a touch bar to complete the trial correctly, 71 of 246 (29%) neurons responded, with responses in the rewarded trials being similar no matter which schedule was ending, thus being sensitive to the reward contingency. Finally, 39 of 246 (16%) neurons responded around the reward. We suggest that basolateral amygdala, by anticipating and then delineating the schedule and representing reward contingency, provide contextual information that is important for adjusting motivational level as a function of immediate behavior goals.",
         "",
         "https://www.jneurosci.org/content/jneuro/25/48/11071.full.pdf"
        ],
        [
         "24",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "The role of microRNA genes in papillary thyroid carcinoma.",
         "Apart from alterations in the RET/PTC-RAS-BRAF pathway, comparatively little is known about the genetics of papillary thyroid carcinoma (PTC). We show that numerous microRNAs (miRNAs) are transcriptionally up-regulated in PTC tumors compared with unaffected thyroid tissue. A set of five miRNAs, including the three most up-regulated ones (miR-221, -222, and -146), distinguished unequivocally between PTC and normal thyroid. Additionally, miR-221 was up-regulated in unaffected thyroid tissue in several PTC patients, presumably an early event in carcinogenesis. Tumors in which the up-regulation (11- to 19-fold) of miR-221, -222, and -146 was strongest showed dramatic loss of KIT transcript and Kit protein. In 5 of 10 such cases, this down expression was associated with germline single-nucleotide changes in the two recognition sequences in KIT for these miRNAs. We conclude that up-regulation of several miRs and regulation of KIT are involved in PTC pathogenesis, and that sequence changes in genes targeted by miRNAs can contribute to their regulation.",
         "MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation. MicroRNA gene expression deregulation in human breast cancer. MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells. miRNAs, Cancer, and Stem Cell Division MicroRNA expression profiles classify human cancers Cancer genomics: Small RNAs with big impacts A microRNA polycistron as a potential human oncogene Small regulatory RNAs in mammals. Combinatorial microRNA target predictions The RET/PTC-RAS-BRAF linear signaling cascade mediates the motile and mitogenic phenotype of thyroid cancer cells. MicroRNAs: a developing story. RAS Is Regulated by the let-7 MicroRNA Family Accumulation of miR-155 and BIC RNA in human B cell lymphomas. A new mechanism of BRAF activation in human thyroid papillary carcinomas. Detection of Overexpressed and Phosphorylated Wild-Type Kit Receptor in Surgical Specimens of Small Cell Lung Cancer Kit as a human oncogenic tyrosine kinase. Oncogenic protein tyrosine kinases Human MicroRNA Targets Fast and effective prediction of microRNA/target duplexes. Genetic analysis of genome-wide variation in human gene expression MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias. Annual report to the nation on the status of cancer, 1975–2001, with a special feature regarding survival An oligonucleotide microchip for genome-wide microRNA profiling in human and mouse tissues. Reduced Expression of the let-7 MicroRNAs in Human Lung Cancers in Association with Shortened Postoperative Survival MicroRNAs Genomics, Biogenesis, Mechanism, and Function Prediction of Mammalian MicroRNA Targets BRAF mutations in thyroid tumors are restricted to papillary carcinomas and anaplastic or poorly differentiated carcinomas arising from papillary carcinomas. Reduced accumulation of specific microRNAs in colorectal neoplasia. Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma BRAF mutation in papillary thyroid carcinoma. High prevalence of BRAF mutations in thyroid cancer: genetic evidence for constitutive activation of the RET/PTC-RAS-BRAF signaling pathway in papillary thyroid carcinoma. Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia Environmental and heritable causes of cancer among 9.6 million individuals in the Swedish family‐cancer database Gene expression in papillary thyroid carcinoma reveals highly consistent profiles Localization of a susceptibility gene for familial nonmedullary thyroid carcinoma to chromosome 2q21. Papillary thyroid carcinoma associated with papillary renal neoplasia: genetic linkage analysis of a distinct heritable tumor syndrome. The biology of stem cell factor and its receptor C-kit. A gene predisposing to familial thyroid tumors with cell oxyphilia maps to chromosome 19p13.2. Paraffin section detection of the c-kit gene product (CD117) in human tissues: value in the diagnosis of mast cell disorders. c-Kit proto-oncogene is more likely to lose expression in differentiated thyroid carcinoma than three thyroid-specific genes: thyroid peroxidase, thyroglobulin, and thyroid stimulating hormone receptor. Transformation of thyroid epithelium is associated with loss of c-kit receptor. Systematic population-based assessment of cancer risk in first-degree relatives of cancer probands. Expression of c-kit receptor in normal and transformed human nonlymphoid tissues. RNA silencing. M  ED  IC  A  L  SC  IE  N  CE Loss of heterozygosity in papillary and follicular thyroid carcinoma: a mini review. de la Chapelle, A. Age at diagnosis Gender Race Diagnosis",
         "https://europepmc.org/articles/pmc1323209?pdf=render"
        ],
        [
         "25",
         "001ce484190e733e0d1dee181f9ee37fd0b15581",
         "Lethality of chlorine, chlorine dioxide, and a commercial fruit and vegetable sanitizer to vegetative cells and spores of Bacillus cereus and spores of Bacillus thuringiensis",
         "None",
         "Sanitizers: Halogens, Surface-Active Agents, and Peroxides Mechanisms of killing of Bacillus subtilis spores by hypochlorite and chlorine dioxide Bacillus anthracis: current knowledge in relation to contamination of food. Peracetic acid as an alternative wastewater disinfectant to chlorine dioxide Mechanisms of killing spores of Bacillus subtilis by acid, alkali and ethanol Differential efficacy of a chlorine dioxide‐containing sanitizer against single species and binary biofilms of a dairy‐associated Bacillus cereus and a Pseudomonas fluorescens isolate Reduction of listeria monocytogenes on green peppers (Capsicum annuum L.) by gaseous and aqueous chlorine dioxide and water washing and its growth at 7 degrees C. A study on the effects of high pressure and heat on Bacillus subtilis spores at low pH. Inactivation of Escherichia coli O157:H7 on surface-uninjured and -injured green pepper (Capsicum annuum L.) by chlorine dioxide gas as demonstrated by confocal laser scanning microscopy Resistance of Bacillus Endospores to Extreme Terrestrial and Extraterrestrial Environments Antiseptics and Disinfectants: Activity, Action, and Resistance Sensitivity of Bacillus coagulans spores to combinations of high hydrostatic pressure, heat, acidity and nisin The effects of various disinfectants againstListeria monocytogeneson fresh-cut vegetables Comparative sporicidal effects of liquid chemical agents Microbiological Control of Cucumber Hydrocooling Water with Chlorine Dioxide †. Interaction of Bacillus subtilis spores with sodium hypochlorite, sodium dichloroisocyanurate and chloramine-T. Bacterial spores and chemical sporicidal agents Bacillus Species Effects of Organic Acids on Thermal Inactivation of Bacillus stearothermophilus and Bacillus coagulans Spores in Frankfurter Emulsion Slurry. Sporicidal activity of mixtures of alcohol and hypochlorite. Ultrastructural and chemical changes in spores of Bacillus cereus after action of disinfectants. A sporulation medium for Bacillus anthracis. Sporicidal properties of some halogens. Kinetics and mechanism of bacterial disinfection by chlorine dioxide. Resistance of bacterial spores. In: Storz G, Hengge-Aronis R (eds) Bacterial stress Principles and practices of disinfection, reservation and sterilization Surface decontamination of fruits and vegetables eaten raw: a review. Food Safety Issues, Food Safety Unit, World Health Organization These include: Evicting an unwelcome tenant: anthrax",
         "https://academic.oup.com/jimb/article-pdf/32/7/301/34739283/jimb0301.pdf"
        ],
        [
         "26",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "Recent advances in understanding the cell death pathways activated by anticancer therapy",
         "Over the past two decades, the role of apoptosis in the cytotoxicity of anticancer drugs has become clear. Apoptosis may occur via a death receptor‐dependent (extrinsic) or independent (intrinsic or mitochondrial) pathway. Mitochondria play a central role in cell death in response to DNA damage, and mediate the interaction(s) of various cytoplasmic organelles, including the endoplasmic reticulum, Golgi apparatus, and lysosomes. The mitochondrial pathway of cell death is mediated by Bcl‐2 family proteins, a group of antiapoptotic and proapoptotic proteins that regulate the passage of small molecules, such as cytochrome c, Smac/Diablo, and apoptosis‐inducing factor, which activates caspase cascades, through the mitochondrial transition pore. In addition, apoptosis can induce autophagic cell death via crosstalk between the two pathways upon treatment with anticancer drugs. The current review focused on recent advances surrounding the mechanism(s) of cell death induced by anticancer agents and discussed potential molecular targets for enhancing the chemotherapeutic effect(s) of anticancer agents. Cancer 2005. © 2005 American Cancer Society.",
         "Neutralization of Smac/Diablo by Inhibitors of Apoptosis (IAPs) Aging as a catabolic malfunction. Apoptosis, autophagy, and more. Dynamic evolution of the adenine nucleotide translocase interactome during chemotherapy-induced apoptosis Bax is an important determinant for radiation sensitivity in esophageal carcinoma cells. Apollon ubiquitinates SMAC and caspase-9, and has an essential cytoprotection function A Phase I Pharmacokinetic and Biological Correlative Study of Oblimersen Sodium (Genasense, G3139), an Antisense Oligonucleotide to the Bcl-2 mRNA, and of Docetaxel in Patients with Hormone-Refractory Prostate Cancer DAP-kinase as a target for drug design in cancer and diseases associated with accelerated cell death. Endogenously released Smac is insufficient to mediate cell death of human lung carcinoma in response to etoposide. Smac induces cytochrome c release and apoptosis independently from Bax/Bcl-xL in a strictly caspase-3-dependent manner in human carcinoma cells Nuclear and mitochondrial conversations in cell death: PARP-1 and AIF signaling. Ceramide-mediated Macroautophagy Involves Inhibition of Protein Kinase B and Up-regulation of Beclin 1* JNK promotes Bax translocation to mitochondria through phosphorylation of 14‐3‐3 proteins Multiple cell death pathways as regulators of tumour initiation and progression Apoptosis defects and chemotherapy resistance: molecular interaction maps and networks Changes in Therapy for Solid Tumors: Potential for Overcoming Drug Resistance In Vivo with Molecular Targeting Agents Molecular therapy for peritoneal dissemination of xenotransplanted human MKN-45 gastric cancer cells with adenovirus mediated Bax gene transfer Mitochondrial protease Omi/HtrA2 enhances caspase activation through multiple pathways Apoptosis pathways in cancer and cancer therapy In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker. Endonuclease G is required for early embryogenesis and normal apoptosis in mice Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor Phosphoinositide 3-kinase accelerates autophagic cell death during glucose deprivation in the rat cardiomyocyte-derived cell line H9c2 Endostatin induces autophagic cell death in EAhy926 human endothelial cells. HSpin1, a transmembrane protein interacting with Bcl-2/Bcl-xL, induces a caspase-independent autophagic cell death Direct Interaction between Survivin and Smac/DIABLO Is Essential for the Anti-apoptotic Activity of Survivin during Taxol-induced Apoptosis* Omi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP) irreversibly inactivates IAPs and facilitates caspase activity in apoptosis. Transformation-selective apoptotic program triggered by farnesyltransferase inhibitors requires Bin1 The CD95 type I/type II model. Amino Acids Interfere with the ERK1/2-dependent Control of Macroautophagy by Controlling the Activation of Raf-1 in Human Colon Cancer HT-29 Cells* Induction of autophagic cell death in malignant glioma cells by arsenic trioxide. Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs. Chemotherapy enhances TNF-related apoptosis-inducing ligand DISC assembly in HT29 human colon cancer cells Molecular response of human glioblastoma multiforme cells to ionizing radiation: cell cycle arrest, modulation of the expression of cyclin-dependent kinase inhibitors, and autophagy. Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death? Current status of the molecular mechanisms of anticancer drug-induced apoptosis Hypoxia and Nitric Oxide Treatment Confer Tolerance to Glucose Starvation in a 5′-AMP-activated Protein Kinase-dependent Manner* Mediation of Poly(ADP-Ribose) Polymerase-1-Dependent Cell Death by Apoptosis-Inducing Factor DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death Bcl-2 and Bcl-xL Inhibit CD95-mediated Apoptosis by Preventing Mitochondrial Release of Smac/DIABLO and Subsequent Inactivation of X-linked Inhibitor-of-Apoptosis Protein* Increased Ras expression and caspase-independent neuroblastoma cell death: possible mechanism of spontaneous neuroblastoma regression. Paclitaxel triggers cell death primarily via caspase-independent routes in the non-small cell lung cancer cell line NCI-H460. The Apaf-1 apoptosome: a large caspase-activating complex. Apoptosis-inducing factor (AIF): a novel caspase-independent death effector released from mitochondria. Identification of Omi/HtrA2 as a Mitochondrial Apoptotic Serine Protease That Disrupts Inhibitor of Apoptosis Protein-Caspase Interaction* The Tumor Suppressor PTEN Positively Regulates Macroautophagy by Inhibiting the Phosphatidylinositol 3-Kinase/Protein Kinase B Pathway* Autophagy delays sulindac sulfide-induced apoptosis in the human intestinal colon cancer cell line HT-29. Bax translocation is crucial for the sensitivity of leukaemic cells to etoposide-induced apoptosis The target of rapamycin (TOR) proteins Proapoptotic BAX and BAK: A Requisite Gateway to Mitochondrial Dysfunction and Death Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function. PTEN controls tumor-induced angiogenesis Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis Bcl-2 down-regulation causes autophagy in a caspase-independent manner in human leukemic HL60 cells Molecular Determinants of the Caspase-promoting Activity of Smac/DIABLO and Its Role in the Death Receptor Pathway* BNIP3 and Genetic Control of Necrosis-Like Cell Death through the Mitochondrial Permeability Transition Pore Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments. Distinct Classes of Phosphatidylinositol 3′-Kinases Are Involved in Signaling Pathways That Control Macroautophagy in HT-29 Cells* Induction of autophagy and inhibition of tumorigenesis by beclin 1 Autophagy Is Activated by Apoptotic Signalling in Sympathetic Neurons: An Alternative Mechanism of Death Execution Cloning and genomic organization of beclin 1, a candidate tumor suppressor gene on chromosome 17q21. Oncogenic Ras triggers cell suicide through the activation of a caspase-independent cell death program in human cancer cells BIN1 is a novel MYC–interacting protein with features of a tumour suppressor Ultrastructural and immunocytochemical characterization of autophagic vacuoles in isolated hepatocytes: effects of vinblastine and asparagine on vacuole distributions. Studies on the mechanisms of autophagy: formation of the autophagic vacuole Vinblastine-induced autophagic vacuoles in mouse liver and Ehrlich ascites tumor cells as assessed by freeze-fracture electron microscopy. The morphology of various types of cell death in prenatal tissues. Inducing cancer cell death by targeting transcription factors. Mitochondrial dysfunction is an essential step for killing of non-small cell lung carcinomas resistant to conventional treatment",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/cncr.20947"
        ],
        [
         "27",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "Biochemical and spectroscopic characterization of the bacterial phytochrome of Pseudomonas aeruginosa",
         "Phytochromes are photochromic biliproteins found in plants as well as in some cyanotrophic, photoautotrophic and heterotrophic bacteria. In many bacteria, their function is largely unknown. Here we describe the biochemical and spectroscopic characterization of recombinant bacterial phytochrome from the opportunistic pathogen Pseudomonas aeruginosa (PaBphP). The recombinant protein displays all the characteristic features of a bonafide phytochrome. In contrast with cyanobacteria and plants, the chromophore of this bacterial phytochrome is biliverdin IXα, which is produced by the heme oxygenase BphO in P. aeruginosa. This chromophore was shown to be covalently attached via its A‐ring endo‐vinyl group to a cysteine residue outside the defined bilin lyase domain of plant and cyanobacterial phytochromes. Site‐directed mutagenesis identified Cys12 and His247 as being important for chromophore binding and photoreversibility, respectively. PaBphP is synthesized in the dark in the red‐light‐absorbing Pr form and immediately converted into a far‐red‐light‐absorbing Pfr‐enriched form. It shows the characteristic red/far‐red‐light‐induced photoreversibility of phytochromes. A chromophore analog that lacks the C15/16 double bond was used to show that this photoreversibility is due to a 15Z/15E isomerization of the biliverdin chromophore. Autophosphorylation of PaBphP was demonstrated, confirming its role as a sensor kinase of a bacterial two‐component signaling system.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2005.04623.x"
        ],
        [
         "28",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "Mycobacteria use their surface-exposed glycolipids to infect human macrophages through a receptor-dependent process Published, JLR Papers in Press, December 1, 2004. DOI 10.1194/jlr.M400308-JLR200",
         "Two subfamilies of the polar glycopeptidolipids (GPLs) located on the surface of Mycobacterium smegmatis, along with unknown phospholipids, were recently shown to participate in the nonopsonic phagocytosis of mycobacteria by human macrophages (Villeneuve, C., G. Etienne, V. Abadie, H. Montrozier, C. Bordier, F. Laval, M. Daffe, I. Maridonneau-Parini, and C. Astarie-Dequeker. 2003. Surface-exposed glycopeptidolipids of Mycobacterium smegmatis specifically inhibit the phagocytosis of mycobacteria by human macrophages. Identification of a novel family of glycopeptidolipids. J. Biol. Chem. 278: 51291–51300). As demonstrated herein, a phospholipid mixture that derived from the methanol-insoluble fraction inhibited the phagocytosis of M. smegmatis. Inhibition was essentially attributable to phosphatidylinositol mannosides (PIMs), namely PIM2 and PIM6, because the purified phosphatidylethanolamine, phosphatidylglycerol, and phosphatidylinositol were inactive. This was further confirmed using purified PIM2 and PIM6 from M. bovis BCG that decreased by half the internalization of M. smegmatis. Both compounds also inhibited the uptake of M. tuberculosis and M. avium but had no effect on the internalization of zymosan used as a control particle of the phagocytic process. When coated on latex beads, PIM2 and polar GPL (GPL III) favored the particle entry through complement receptor 3. GPL III, but not PIM2, also directed particle entry through the mannose receptor. Therefore, surface-exposed mycobacterial PIM and polar GPL participate in the receptor-dependent internalization of mycobacteria in human macrophages.",
         "",
         "http://www.jlr.org/article/S0022227520340323/pdf"
        ],
        [
         "29",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "Insights from the developing world: thrifty genotypes and thrifty phenotypes",
         "Few researchers would dispute that the pandemic of obesity is caused by a profound mismatch between humanity's present environmental circumstances and those that have moulded evolutionary selection. This concept was first articulated when gestational diabetes was described as being the result of a ‘thrifty genotype rendered detrimental by progress’. More recently, this hypothesis has been extended to the concept of a ‘thrifty phenotype’ to describe the metabolic adaptations adopted as a survival strategy by a malnourished fetus; changes that may also be inappropriate to deal with a later life of affluence. Both the thrifty genotype and the thrifty phenotype hypotheses would predict that populations in some areas of the developing world would be at greater risk of obesity and its co-morbidities; a proposition to be explored in the present paper. To date thrifty genes remain little more than a nebulous concept propagated by the intuitive logic that man has been selected to survive episodic famine and seasonal hungry periods. Under such conditions those individuals who could lay down extra energy stores and use them most efficiently would have a survival advantage. The search for candidate thrifty genes needs to cover every aspect of human energy balance from food-seeking behaviour to the coupling efficiency of oxidative phosphorylation. The present paper will describe examples of attempts to find thrifty genes in three selected candidate areas: maternally-transmitted mitochondrial genes; the uncoupling proteins; apoE4, whose geographical distribution has been linked to a possible thrifty role in lipoprotein and cholesterol metabolism.",
         "Fetal origins of insulin resistance and obesity Variation at the insulin gene VNTR (variable number tandem repeat) polymorphism and early growth: studies in a large Finnish birth cohort. Apolipoprotein E polymorphism and fertility: a study in pre-industrial populations. Opposing influences of prenatal and postnatal weight gain on adrenarche in normal boys and girls. Maternal-fetal interactions and birth order influence insulin variable number of tandem repeats allele class associations with head size at birth and childhood weight gain. The human obesity gene map: the 2003 update. Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA Intrauterine factors, adiposity, and hyperinsulinaemia Global haplotype diversity in the human insulin gene region. Birthweight and perinatal mortality: paradoxes, social class, and sibling dependencies. The double puzzle of diabetes The epidemiology of adverse pregnancy outcomes: an overview. Neonatal anthropometry: the thin–fat Indian baby. The Pune Maternal Nutrition Study Adiposity and hyperinsulinemia in Indians are present at birth. The role of uncoupling proteins in pathophysiological states. Racial differences in the relation between uncoupling protein genes and resting energy expenditure. An overview on the nutrition transition and its health implications: the Bellagio meeting An adipose-specific control of thermogenesis in body weight regulation Insulin gene variable number of tandem repeat genotype and the low birth weight, precocious pubarche, and hyperinsulinism sequence. On the importance--and the unimportance--of birthweight. The thrifty phenotype hypothesis. Obesity and undernutrition and cardiovascular risk factors in rural and urban Gambian communities. Glucose, insulin and lipid metabolism in rural Gambians exposed to early malnutrition Uncoupling proteins: functional characteristics and role in the pathogenesis of obesity and Type II diabetes Uncoupling proteins: their roles in adaptive thermogenesis and substrate metabolism reconsidered Fires of life: the struggles of an ancient metabolism in a modern world The nutrition transition and obesity in the developing world. Thrifty Genotypes and Phenotypes in the Pathogenesis of Type 2 Diabetes Mellitus Uncoupling to survive? The role of mitochondrial inefficiency in ageing The Fetal and Childhood Growth of Persons Who Develop Type 2 Diabetes Energy adaptations in human pregnancy: limits and long-term consequences. Interactions of perturbations in intrauterine growth and growth during childhood on the risk of adult-onset disease Insulin resistance syndrome in 8-year-old Indian children: small at birth, big at 8 years, or both? Transmission ratio distortion at the INS-IGF2 VNTR Apolipoprotein E (APOE) allele distribution in the world. Is APOE*4 a ‘thrifty’ allele? The insulin gene VNTR, type 2 diabetes and birth weight The thrifty genotype in type 2 diabetes The INS 5′ Variable Number of Tandem Repeats Is Associated with IGF2 Expression in Humans* Association of the INS VNTR with size at birth The Great Famine: Northern Europe in the Early Fourteenth Century Bergmann's rule and the thrifty genotype. Imprinted and genotype-specific expression of genes at the IDDM2 locus in pancreas and leucocytes. Susceptibility to human type 1 diabetes at IDDM2 is determined by tandem repeat variation at the insulin gene minisatellite locus Biochemical control logic and the metabolism of glutamine. Type 2 (non-insulin-dependent) diabetes mellitus: the thrifty phenotype hypothesis The possible role of glutamine substrate cycles in skeletal muscle. Maternal weight gain, infant birth weight, and diet: causal sequences. Role of triglyceride-fatty acid cycle in controlling fat metabolism in humans during and after exercise. A Polymorphic Locus Near the Human Insulin Gene Is Associated with Insulin-dependent Diabetes Melliitus Poverty and famine: An essay on entitlement and deprivation : , Oxford, New York: Oxford University Press, 1981. xi + 257 pp., index. $17.95 Malnutrition and fertility. Poverty and famines. An essay on entitlement and deprivation [incl. case studies of famines in Bengal, Ethiopia, Sahel, and Bangladesh]. Famine in Tudor and Stuart England. The high prevalence of diabetes mellitus on a Central Pacific island Famine and Human Development: The Dutch Hunger Winter 1944-1945 Faminee of history and of today Diabetes in American Indians and Other Native Populations of the New World The determinants of growth and form. Review lecture - The determinants of growth and form Diabetes mellitus: a \"thrifty\" genotype rendered detrimental by \"progress\"? Biology of Human Starvation A critical evaluation of the fetal origins hypothesis and its implications for developing countries. Lack of support for a role of the insulin gene variable number of tandem repeats minisatellite (INS-VNTR) locus in fetal growth or type 2 diabetes-related intermediate traits in United Kingdom populations. Differential reproductive efficiency associated with common apolipoprotein e alleles in postreproductive-aged subjects. Eating, exercise, and \"thrifty\" genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases. The insulin gene variable number tandem repeat class I/III polymorphism is in linkage disequilibrium with birth weight but not Type 2 diabetes in the Pima population. Floods, Famines and Emperors. London: Pimlico PPARgamma, the ultimate thrifty gene. Fetal origins of adult disease-the hypothesis revisited. Genomic imprinting of fetal growth The Demography of Famine: An Indian Historical Perspective Biochemical control logic and the metabolism of glutamine. Agriculture’s mixed blessings The Third Chimpanzee Determinants of low birth weight: methodological assessment and meta-analysis. Origin of Species",
         "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D4334A1A0F9096B08FAE923BAB0CA36D/S0029665105000194a.pdf/div-class-title-insights-from-the-developing-world-thrifty-genotypes-and-thrifty-phenotypes-div.pdf"
        ],
        [
         "30",
         "00206ca6a32791adcca5effdfed6eecd035075f4",
         "Sparse basis selection: new results and application to adaptive prediction of video source traffic",
         "Real-time prediction of video source traffic is an important step in many network management tasks such as dynamic bandwidth allocation and end-to-end quality-of-service (QoS) control strategies. In this paper, an adaptive prediction model for MPEG-coded traffic is developed. A novel technology is used, first developed in the signal processing community, called sparse basis selection. It is based on selecting a small subset of inputs (basis) from among a large dictionary of possible inputs. A new sparse basis selection algorithm is developed that is based on efficiently updating the input selection adaptively. When a new measurement is received, the proposed algorithm updates the selected inputs in a recursive manner. Thus, adaptability is not only in the weight adjustment, but also in the dynamic update of the inputs. The algorithm is applied to the problem of single-step-ahead prediction of MPEG-coded video source traffic, and the developed method achieves improved results, as compared to the published results in the literature. The present analysis indicates that the adaptive feature of the developed algorithm seems to add significant overall value.",
         "",
         "http://www.vision.caltech.edu/malaa/publications/atiya05sparse.pdf"
        ],
        [
         "31",
         "0020d5bdfc5f13026e7a1aca78e94724e8342b62",
         "A mutation in the Icsbp1 gene causes susceptibility to infection and a chronic myeloid leukemia–like syndrome in BXH-2 mice",
         "BXH-2 mice develop a fatal myeloid leukemia by a two-step mutagenic process. First, a BXH-2–specific recessive mutation causes a myeloproliferative syndrome. Second, retroviral insertions alter oncogenes or tumor suppressors, resulting in clonal expansion of leukemic cells. We have identified a recessive locus on chromosome 8 (Myls) that is responsible for myeloproliferation in BXH-2. This Myls interval has been narrowed down to 2 Mb and found to contain several positional candidates, including the interferon consensus sequence–binding protein 1 gene (Icsbp, also known as interferon regulatory factor 8 [IRF8]). We show that BXH-2 mice carry a mutation (915 C to T) resulting in an arginine-to-cysteine substitution at position 294 within the predicted IRF association domain of the protein. Although expression of Icsbp1 mRNA transcripts is normal in BXH-2 splenocytes, these cells are unable to produce interleukin 12 and interferon-γ in response to activating stimuli, confirming that R294C behaves as a loss-of-function mutation. Myeloproliferation in BXH-2 mice is concomitant to increased susceptibility to Mycobacterium bovis (BCG) despite the presence of resistance alleles at the Nramp1 locus. These results suggest a two-step model for chronic myeloid leukemia in BXH-2, in which inactivation of Icsbp1 predisposes to myeloproliferation and immunodeficiency. This event is required for retroviral replication, and subsequent insertional mutagenesis that causes leukemia in BXH-2 mice.",
         "",
         "https://rupress.org/jem/article-pdf/201/6/881/1153938/jem2016881.pdf"
        ],
        [
         "32",
         "0022e70878c2f2a7c3c679a3ef3160a5cb803e20",
         "Binding of estrogen receptor beta to estrogen response element in situ is independent of estradiol and impaired by its amino terminus.",
         "The functions of 17beta-estradiol (E2) are mediated by estrogen receptor (ER) alpha and beta. ERs display similar DNA- and ligand-binding properties in vitro. However, ERbeta shows lower transcriptional activity than ERalpha from the estrogen response element (ERE)-dependent signaling. We predicted that distinct amino termini contribute to differences in transcription efficacies of ERs by affecting in situ ER-ERE interactions. We used chromatin immunoprecipitation and a novel in situ ERE competition assay, which is based on the ability of ER to compete for ERE binding with a designer activator that constitutively induces transcription from an ERE-driven reporter construct. Interference of activator-mediated transcription by unliganded or liganded ERs was taken as an indication of ER-ERE interaction. Results revealed that ERs interacted with ERE similarly in the absence of E2. However, E2 enhanced the ERE binding of ERalpha but not that of ERbeta. The removal of the amino terminus increased the ERbeta-ERE interaction independent of E2. The ERbeta amino terminus also prevented E2-mediated enhancement of the chimeric ERalpha-ERE interaction. Thus, the amino terminus of ERbeta impairs the binding of ERbeta to ERE. The abrogation of ligand-dependent activation function 2 of the amino-terminally truncated ERbeta resulted in the manifestation of E2 effect on ERbeta-ERE interaction. This implies that E2-mediated enhancement of ERbeta-ERE interaction is masked by the activation function 2, whereas the intact amino terminus is a dominant region that decreases the binding of ERbeta to ERE. Thus, ERbeta-ERE interaction is independent of E2 and is impaired by its amino terminus. These findings provide an additional explanation for differences between ERalpha and ERbeta functions that could differentially affect the physiology and pathophysiology of E2 signaling.",
         "",
         "https://academic.oup.com/mend/article-pdf/19/11/2696/10717141/mend2696.pdf"
        ],
        [
         "33",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "Sensor node localization using mobile acoustic beacons",
         "We present a mobile acoustic beacon based sensor node localization method. Our technique is passive in that the sensor nodes themselves do not need to generate an acoustic signal for ranging. This saves cost, power and provides stealthy operation. Furthermore, the beacon can generate much more acoustic energy than a severely resource constrained sensor node, thereby significantly increasing the range. The acoustic ranging method uses a linear frequency modulated signal that can be accurately detected by matched filtering. This provides longer range and higher accuracy than the current state-of-the-art. The localization algorithm was especially designed to work in such acoustically reverberant environment, as urban terrain. The algorithm presented handles non-Gaussian ranging errors caused by echoes. Node locations are computed centrally by solving a global non-linear optimization problem in an iterative and incremental fashion",
         "Resilient Localization for Sensor Networks in Outdoor Environments Mobile-assisted localization in wireless sensor networks Sensor network-based countersniper system Robust distributed network localization with noisy range measurements The flooding time synchronization protocol Localization of wireless sensor networks with a mobile beacon Smart sensors to network the world. Range-free localization schemes for large scale sensor networks The n-Hop Multilateration Primitive for Node Localization Problems Ad hoc positioning system (APS) using AOA Mica: A Wireless Platform for Deeply Embedded Networks Middleware challenges for wireless sensor networks The bits and flops of the n-hop multilateration primitive for node localization problems Wireless sensor networks for habitat monitoring A survey on position-based routing in mobile ad hoc networks Robust range estimation using acoustic and multimodal sensing Location Systems for Ubiquitous Computing Smart kindergarten: sensor-based wireless networks for smart developmental problem-solving environments Exposure in wireless Ad-Hoc sensor networks Dynamic fine-grained localization in Ad-Hoc networks of sensors The cricket compass for context-aware mobile applications GPS-free Positioning in Mobile Ad Hoc Networks GPS-less low-cost outdoor localization for very small devices Embedding the Internet: introduction Acoustic Ranging in Resource-Constrained Sensor Networks IEEE Computer Graphics & Applications: Projects In VR Pervasive Pose-Aware Applications and Infrastructure VR Pervasive Pose-Aware Applications and Infrastructure. IEEE Distributed Systems Online Range-free localization schemes in large scale sensor network This copyright notice must be included in the reproduced paper. USENIX acknowledges all trademarks herein. Robust Positioning Algorithms for Distributed Ad-Hoc Wireless Sensor Networks The Design of Calamari : an Ad-hoc Localization System for Sensor Networks Complex Behavior at Scale: An Experimental Study of Low-Power Wireless Sensor Networks Convex Optimization Methods for Sensor Node Position Estimation SpotON: An Indoor 3D Location Sensing Technology Based on RF Signal Strength The Cricket location-support system Global Positioning System Overview (last modified 5.1.2000) Numerical Optimization, pages 250–275 Theory and application of digital signal processing If both | µ i | and σ i are large then again location might be affected by echo and we follow the same procedure as in case 2 above If µ i is close to zero but σ i is large then either the range deviations are spread around zero or few large deviations caused σ i to be large If | µ i | is large but σ i is small then all elements of ∆ i are large i.e. the node location is deﬁnitely incorrect",
         "https://ir.vanderbilt.edu/bitstream/1803/13704/1/ms-thesis-manish-kushwaha.pdf"
        ],
        [
         "34",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "Impact of Software Vulnerability Announcements on the Market Value of Software Vendors - an Empirical Investigation",
         "Researchers in the area of information security have mainly been concerned with tools, techniques and policies that firms can use to protect themselves against security breaches. However, information security is as much about security software as it is about secure software. Software is not secure when it has defects or flaws which can be exploited by hackers to cause attacks such as unauthorized intrusion or denial of service attacks. Any public announcement makes about a software defect is termed as 'vulnerability disclosure'. Although research in software economics have studied firms' incentive to improve overall quality, there have been no studies to show that software vendors have an incentive to invest in building more secure software. In this paper, we use the event study methodology to examine software vendors' incentives to build more secure software. We collect data from leading national newspapers and industry sources like CERT by searching for reports on published software vulnerabilities. We show that vulnerability disclosures lead to a negative and significant change in market value for a software vendor. On average, a vendor loses around 0.6% value in stock price when a vulnerability is reported. This is equivalent to a loss in market capitalization values of $0.86 billion per vulnerability announcement. To provide further insight, we use the information content of the disclosure announcement to classify vulnerabilities into various types. We find that the change in stock price is more negative if the vendor fails to provide a patch at the time of disclosure. Moreover, vulnerabilities which cause a confidentiality related breach cause a greater decline in the market value for a vendor than the vulnerabilities which cause non-confidentiality related breaches. Also, more severe flaws have a significantly greater impact than flaws with low or moderate severity. Finally, we find that the markets do not punish a software vendor more severely if a third party discovers a flaw in its product than if the vendor itself discovers the flaw. Our analysis provides many interesting implications for software vendors as well as policy makers.",
         "",
         "http://archive.nyu.edu/bitstream/2451/14996/2/Infosec_Book%2bWattal%2bTelang.pdf"
        ],
        [
         "35",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "Real-Time Polymerase Chain Reaction Quantification of Phytophthora capsici in Different Pepper Genotypes.",
         "ABSTRACT Reliable and sensitive quantification of Phytophthora capsici in pepper plants is of crucial importance in managing the multiple syndromes caused by this pathogen. A real-time polymerase chain reaction (PCR) assay was developed for the determination of P. capsici in pepper tissues. DNA levels of a highly virulent and a less virulent isolate were measured in different pepper genotypes with varying degrees of resistance. Using SYBR Green and specific primers for P. capsici, the minimal amount of pathogen DNA quantified was 10 pg. Pathogen DNA was recorded as early as 8 h postinoculation. Thereafter, the increase was rapid in susceptible cultivars and slower in resistant ones. The amount of pathogen DNA quantified in each pepper genotype correlated with susceptibility to Phytophthora root rot. Likewise, there was a relationship between the virulence of the pathogen and the degree of colonization. Differences also were found in oomycete amount among pepper tissues, with maximal pathogen biomass occurring in stems. The real-time PCR technique developed in this study was sensitive and robust enough to assess both pathogen development and resistance to Phytophthora root rot in different pepper genotypes.",
         "",
         "https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-95-1423"
        ],
        [
         "36",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "Wing-to-Leg Homeosis by Spineless Causes Apoptosis Regulated by Fish-lips, a Novel Leucine-Rich Repeat Transmembrane Protein",
         "ABSTRACT Growth, patterning, and apoptosis are mutually interactive during development. For example, cells that select an abnormal fate in a developing field are frequently removed by apoptosis. An important issue in this process that needs to be resolved is the mechanism used by cells to discern their correct fate from an abnormal fate. In order to examine this issue, we developed an animal model that expresses the dioxin receptor homolog Spineless (Ss) ectopically in the Drosophila wing. The presence of mosaic clones ectopically expressing ss results in a local transformation of organ identity, homeosis, from wing into a leg or antenna. The cells with misspecified fates subsequently activate c-Jun N-terminal kinase to undergo apoptosis in an autonomous or nonautonomous manner depending on their position within the wing, suggesting that a cell-cell interaction is, at least in some cases, involved in the detection of misspecified cells. Similar position dependence is commonly observed when various homeotic genes controlling the body segments are ectopically expressed. The autonomous and nonautonomous apoptosis caused by ss is regulated by a novel leucine-rich repeat family transmembrane protein, Fish-lips (Fili) that interacts with surrounding normal cells. These data support a mechanism in which the lack of some membrane proteins helps to recognize the presence of different cell types and direct these cells to an apoptotic fate in order to exclude them from the normal developing field.",
         "",
         "https://europepmc.org/articles/pmc1069588?pdf=render"
        ],
        [
         "37",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "The sentinel within: exploiting the immune system for cancer biomarkers.",
         "The release of proteins from tumors triggers an immune response in cancer patients. These tumor antigens arise from several mechanisms including tumor-specific alterations in protein expression, mutation, folding, degradation, or intracellular localization. Responses to most tumor antigens are rarely observed in healthy individuals, making the response itself a biomarker that betrays the presence of underlying cancer. Antibody immune responses show promise as clinical biomarkers because antibodies have long half-lives in serum, are easy to measure, and are stable in blood samples. However, our understanding of the specificity and the impact of the immune response in early stages of cancer is limited. The immune response to cancer, whether endogenous or driven by vaccines, involves highly specific T lymphocytes (which target tumor-derived peptides bound to self-MHC proteins) and B lymphocytes (which generate antibodies to tumor-derived proteins). T cell target antigens have been identified either by expression cloning from tumor cDNA libraries, or by prediction based on patterns of antigen expression (\"reverse immunology\"). B cell targets have been similarly identified using the antibodies in patient sera to screen cDNA libraries derived from tumor cell lines. This review focuses on the application of recent advances in proteomics for the identification of tumor antigens. These advances are opening the door for targeted vaccine development, and for using immune response signatures as biomarkers for cancer diagnosis and monitoring.",
         "T Cell–Dependent Antibody Responses against Aberrantly Expressed Cyclin B1 Protein in Patients with Cancer and Premalignant Disease Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast Cancer A phase I trial of DNA vaccination with a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate cancer Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant induces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans. Self-Assembling Protein Microarrays Protein microarray technology Minor Histocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem Cell Transplantation An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in association with HLA-DR. Development of natural protein microarrays for diagnosing cancer based on an antibody response to tumor antigens. NY-ESO-1 Expression and Immunogenicity in Malignant and Benign Breast Tumors Clinical proteomics: revolutionizing disease detection and patient tailoring therapy. The clinical value of intracellular autoantigens B-cell epitopes in systemic rheumatic diseases. Serological cloning of cancer/testis antigens expressed in prostate cancer using cDNA phage surface display Complete response to donor lymphocyte infusion in multiple myeloma is associated with antibody responses to highly expressed antigens. Humoral immune response against melanoma antigens induced by vaccination with cytokine gene‐modified autologous tumor cells Identification of a shared epitope recognized by melanoma-specific, HLA-A3-restricted cytotoxic T lymphocytes. Genome-wide Characterization of a Viral Cytotoxic T Lymphocyte Epitope Repertoire* Microarrays of tumor cell derived proteins uncover a distinct pattern of prostate cancer serum immunoreactivity Protein arrays for autoantibody profiling and fine‐specificity mapping Protein microarrays: Promising tools for proteomic research Recursive partitioning as an approach to selection of immune markers for tumor diagnosis. Antigen-specific immunity in neuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen. Expression of cancer/testis (CT) antigens in lung cancer. Unique conformation of cancer autoantigen B23 in hepatoma: A mechanism for specificity in the autoimmune response NY-ESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial ovarian cancer. Identification of tumor‐associated antigens by screening phage‐displayed human cDNA libraries with sera from tumor patients Tumor-specific immunological recognition of frameshift-mutated peptides in colon cancer with microsatellite instability. CD1d-restricted T cells regulate dendritic cell function and antitumor immunity in a granulocyte–macrophage colony-stimulating factor-dependent fashion Melanoma inhibitor of apoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction The use of HPLC-MS in T-cell epitope identification. Immunomic analysis of human sarcoma Enhancement of antibody detection in cancer using panel of recombinant tumor-associated antigens. Immunoscreening of phage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease. Integrated functional genomics approach for the design of patient-individual antitumor vaccines. Proteomics Approaches to Uncover the Repertoire of Circulating Biomarkers for Breast Cancer CML28 is a broadly immunogenic antigen, which is overexpressed in tumor cells. Towards patient‐specific tumor antigen selection for vaccination Cancer/testis antigens: an expanding family of targets for cancer immunotherapy Serological identification of antigens associated with renal cell carcinoma. Characterisation of tumour-associated antigens in colon cancer Solid-Phase Epitope Recovery: A High Throughput Method for Antigen Identification and Epitope Optimization Enhanced humoral immune response correlates with improved disease-free and overall survival in American Joint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine. Identification of Breast Cancer-Restricted Antigens by Antibody Screening of SKBR3 cDNA Library Using a Preselected Patient's Serum Posttranslational protein modifications: new flavors in the menu of autoantigens ATP6S1 elicits potent humoral responses associated with immune-mediated tumor destruction Identification of cancer/testis genes by database mining and mRNA expression analysis Autoantigen microarrays for multiplex characterization of autoantibody responses Miniaturised multiplexed immunoassays. Identification of Cyclin B1 as a Shared Human Epithelial Tumor-Associated Antigen Recognized by T Cells Trends in biomarker research for cancer detection. Autoimmune diseases: are markers ready for prediction? Post-translational protein modifications in antigen recognition and autoimmunity. Identification of a tissue-specific putative transcription factor in breast tissue by serological screening of a breast cancer library. Profiling the immune response in patients with breast cancer by phage‐displayed cDNA libraries Tumor-associated antigen TA90 immune complex assay predicts recurrence and survival after surgical treatment of stage I-III melanoma. Evidence that specific T lymphocytes may participate in the elimination of chronic myelogenous leukemia Pre-existent immunity to the HER-2/neu oncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer Profiling the immune responses in patient sera with peptide and cDNA display libraries. Reactivity of natural and induced human antibodies to MUC1 mucin with MUC1 peptides and n‐acetylgalactosamine (GalNAc) peptides Rapid degradation of a large fraction of newly synthesized proteins by proteasomes Serological identification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung cancer. p53 Antibodies in the sera of patients with various types of cancer: a review. Expression of cancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a new CT gene, CT9. CT10: A new cancer‐testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational‐difference analysis p53 gene mutation, tumor p53 protein overexpression, and serum p53 autoantibody generation in patients with breast cancer. Cancer-testis antigens and ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-specific ING1 transcripts and a homologue gene. Antigens recognized by autologous antibody in patients with renal‐cell carcinoma SYFPEITHI: database for MHC ligands and peptide motifs Cleavage by Granzyme B Is Strongly Predictive of Autoantigen Status Humoral immune responses of cancer patients against \"Cancer-Testis\" antigen NY-ESO-1: correlation with clinical events. The telomerase catalytic subunit is a widely expressed tumor-associated antigen recognized by cytotoxic T lymphocytes. Isoforms of the human PDZ-73 protein exhibit differential tissue expression. The 72-kDa Component of Signal Recognition Particle Is Cleaved during Apoptosis* The Central Role of CD4+ T Cells in the Antitumor Immune Response Characterization of human colon cancer antigens recognized by autologous antibodies A Survey of the Humoral Immune Response of Cancer Patients to a Panel of Human Tumor Antigens Human lung cancer antigens recognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor gene locus on chromosome 3p21.3. High-titer HER-2/neu protein-specific antibody can be detected in patients with early-stage breast cancer. SSX: A multigene family with several members transcribed in normal testis and human cancer Autoantibodies to p53 in ovarian cancer patients and healthy women: a comparison between whole p53 protein and 18-mer peptides for screening purposes. A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening. Humoral immune response against p53 protein in patients with colorectal carcinoma DNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to detect all known p53 alterations in human malignancies. Anti-p53 antibodies in sera from patients with chronic obstructive pulmonary disease can predate a diagnosis of cancer. Anti-p53 antibodies in sera of workers occupationally exposed to vinyl chloride. Absence of p53 autoantibodies in sera from glioma patients. A motif in human histidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-coil that is essential for enzymatic activity and contains the major autoantigenic epitope. Clinical utility of the immunocytochemical detection of p53 protein in cytological specimens. Role of bone marrow-derived cells in presenting MHC class I-restricted tumor antigens. Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains. Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-macrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor immunity. Development of antibodies against p53 in lung cancer patients appears to be dependent on the type of p53 mutation. HLA-A2 molecules in an antigen-processing mutant cell contain signal sequence-derived peptides Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry. Very long charge runs in systemic lupus erythematosus-associated autoantigens. Characterization of naturally occurring minor histocompatibility peptides including H-4 and H-Y. The fine specificity of anti-La antibodies induced in mice by immunization with recombinant human La autoantigen. Post-translational modifications of proteins: some problems left to solve. CA125 Antigen Levels in Obstetric and Gynecologic Patients Purification of a human prostate specific antigen. Detection of a transformation-related antigen in chemically induced sarcomas and other transformed cells of the mouse. SPECIFIC CARCINOEMBRYONIC ANTIGENS OF THE HUMAN DIGESTIVE SYSTEM Immunotherapy (APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3 randomized placebo-controlled trial Efficient purification of unique antibodies using peptide affinity-matrix columns. Endogenous immune response to early- and intermediate-stage melanoma is correlated with outcomes and is independent of locoregional relapse and standard prognostic factors. Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem cell transplantation and in healthy female donors. Recombinant antigen expression on yeast surface (RAYS) for the detection of serological immune responses in cancer patients. Immunotherapy of hematologic malignancy. p53 autoantibodies can be indicative of the development of breast cancer relapse. The autoantibody repertoire: searching for order MUC1-like tandem repeat proteins are broadly immunogenic in cancer patients. Epitope identification and vaccine design for cancer immunotherapy. Immunotherapeutic Targets Cancer : Identification of Potential Diagnostic and Cancer-related Serological Recognition of Human Colon Updated Identification of tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by serological screening of a testicular library with breast cancer serum. Humoral immunity to human breast cancer: antigen definition and quantitative analysis of mRNA expression. Monitoring of p53 autoantibodies after resection of colorectal cancer: relationship to operative curability. Gure, A. O. et al. CT10: a new cancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-difference analysis. Int. J. Cancer 85, 726-732 MHCPEP, a database of MHC-binding peptides: update 1997 Cancer,Facts and Figures about.",
         "https://europepmc.org/articles/pmc2522321?pdf=render"
        ],
        [
         "38",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "Molecular Mechanisms of Cation Transport by the Renal Na+-K+-Cl- Cotransporter",
         "Two variants of the renal Na+-K+-Cl- cotransporter (NKCC2), called NKCC2A and NKCC2F, display marked differences in Na+, Rb+, and Cl- affinities, yet are identical to one another except for a 23-residue membrane-associated domain that is derived from alternatively spliced exons. The proximal portion of these exons is predicted to encode the second transmembrane domain (tm2) in the form of an α-helix, and the distal portion, part of the following connecting segment (cs1a). In recent studies, we have taken advantage of the A–F differences in kinetic behavior to determine which regions in tm2-cs1a are involved in ion transport. Functional characterizations of chimeras in which tm2 or cs1a were interchanged between the variants showed that both regions are important in specifying ion affinities, but did not allow delineating the contribution of individual residues. Here, we have extended these structure-function analyses by studying additional mutants in which variant residues between A and F were interchanged individually in the tm2-cs1a region (amino acid number 216, 220, 223, 229, or 233 in NKCC2). None of the substitutions were found to affect Km (C1-), suggesting that the affinity difference for anion transport is conveyed by a combination of variant residues in this domain. However, 2 substitutions in the tm2 of F were found to affect cation constants specifically; interestingly, one of these mutations (residue 216) only affected Km (Rb+) while the other (residue 220) only affected Km (Na+). We have thus identified two novel residues in NKCC2 that play a key role in cation transport. Because such residues should be adjacent to one another on the vertical axis of the tm2 α-helix, our results imply, furthermore, that the ion transport sites in NKCC2 could be physically linked.",
         "",
         "http://www.jbc.org/article/S0021925820792379/pdf"
        ],
        [
         "39",
         "00295936451c8a6b8a982d6ba9ab64cb647370ae",
         "Choline deficiency in mice and humans is associated with increased plasma homocysteine concentration after a methionine load.",
         "BACKGROUND\nElevated concentrations of homocysteine in blood may be an independent risk factor for the development of atherosclerosis. Elevated homocysteine concentrations can be caused by decreased methylation of homocysteine to form methionine, as occurs in folate deficiency. A parallel pathway exists for methylation of homocysteine, in which choline, by way of betaine, is the methyl donor.\n\n\nOBJECTIVE\nOur goal was to determine whether choline deficiency results in a decreased capacity to methylate homocysteine.\n\n\nDESIGN\nC57BL/6J mice were fed diets containing 0, 10, or 35 mmol choline/kg diet for 3 wk. We then administered an oral methionine load to the animals and measured plasma homocysteine concentrations. Also, in a pilot study, we examined 8 men who were fed a diet providing 550 mg choline/d per 70 kg body weight for 10 d, followed by a diet providing almost no choline, until the subjects were clinically judged to be choline deficient or for <or=42 d. A methionine load was administered at the end of each dietary phase.\n\n\nRESULTS\nTwo hours after the methionine load, choline-deficient mice had plasma homocysteine concentrations twice those of choline-fed mice. Four hours after the methionine load, clinically choline-depleted men had plasma homocysteine concentrations that were 35% greater than those in men not choline depleted.\n\n\nCONCLUSION\nThese results suggest that choline, like folate, plays an important role in the metabolism of homocysteine in humans and that response to a methionine load may be useful when assessing choline nutriture.",
         "Choline- and betaine-defined diets for use in clinical research and for the management of trimethylaminuria. Elevated serum creatine phosphokinase in choline-deficient humans: mechanistic studies in C2C12 mouse myoblasts. Betaine as a Determinant of Postmethionine Load Total Plasma Homocysteine Before and After B-Vitamin Supplementation Folate status response to controlled folate intake is affected by the methylenetetrahydrofolate reductase 677C-->T polymorphism in young women. Effect of folic acid on fenofibrate-induced elevation of homocysteine and cysteine. Betaine supplementation lowers plasma homocysteine in healthy men and women. Concentrations of choline-containing compounds and betaine in common foods. Phosphatidylethanolamine N-methyltransferase (PEMT) knockout mice have hepatic steatosis and abnormal hepatic choline metabolite concentrations despite ingesting a recommended dietary intake of choline. Effects of cytidine 5'-diphosphocholine on plasma homocysteine levels in rat. Quantitation of choline and its metabolites in tissues and foods by liquid chromatography/electrospray ionization-isotope dilution mass spectrometry. Hyperhomocysteinemia due to methionine synthase deficiency, cblG: structure of the MTR gene, genotype diversity, and recognition of a common mutation, P1173L. Choline deficiency causes reversible hepatic abnormalities in patients receiving parenteral nutrition: proof of a human choline requirement: a placebo-controlled trial. The 1298A-->C polymorphism in methylenetetrahydrofolate reductase (MTHFR): in vitro expression and association with homocysteine. Determinants of plasma total homocysteine concentration in the Framingham Offspring cohort. Betaine-homocysteine methyltransferase expression in porcine and human tissues and chromosomal localization of the human gene. The use of homocysteine and other metabolites in the specific diagnosis of vitamin B-12 deficiency. Choline deficiency: A cause of hepatic steatosis during parenteral nutrition that can be reversed with intravenous choline supplementation Evidence that homocysteine is an independent risk factor for atherosclerosis in hyperlipidemic patients. Vitamin B-6 deficiency vs folate deficiency: comparison of responses to methionine loading in rats. AIN-93 purified diets for laboratory rodents: final report of the American Institute of Nutrition ad hoc writing committee on the reformulation of the AIN-76A rodent diet. Low plasma free choline is prevalent in patients receiving long term parenteral nutrition and is associated with hepatic aminotransferase abnormalities. Accumulation of 1,2-sn-diradylglycerol with increased membrane-associated protein kinase C may be the mechanism for spontaneous hepatocarcinogenesis in choline-deficient rats. Rapid high-performance liquid chromatographic assay for total homocysteine levels in human serum. Choline, an essential nutrient for humans Lactobacillus casei microbiological assay of folic acid derivatives in 96-well microtiter plates. The active synthesis of phosphatidylcholine is required for very low density lipoprotein secretion from rat hepatocytes. Plasma choline concentration in humans fed parenterally. Betaine in the treatment of homocystinuria due to 5,10-methylenetetrahydrofolate reductase deficiency A rapid method of total lipid extraction and purification. Effect of methotrexate on homocysteine and other sulfur compounds in tissues of rats fed a normal or a defined, choline-deficient diet Pathways and Regulation of Homocysteine Metabolism in Mammals Institute of Medicine, National Academy of Sciences USA. Dietary reference intakes for folate, thiamin, riboflavin, niacin, vitamin B-12, panthothenic acid, biotin, and choline The biosynthesis of choline and its relation to phospholipid metabolism.",
         "https://academic.oup.com/ajcn/article-pdf/81/2/440/23875105/440.pdf"
        ],
        [
         "40",
         "002a108f32eb157c0b88aea3c6f29d4b614c3b78",
         "Altered gene expression during rat Wolffian duct development following di(n-butyl) phthalate exposure.",
         "Di(n-butyl) phthalate (DBP) is a common plasticizer and solvent that disrupts androgen-dependent male reproductive development in rats. In utero exposure to 500 mg/kg/day DBP on gestation days (GD) 12 to 21 decreases androgen biosynthetic enzymes, resulting in decreased fetal testicular testosterone levels. One consequence of prenatal DBP exposure is malformed epididymides in adult rats. Reduced fetal testosterone levels may be responsible for the malformation, since testosterone is required for Wolffian duct stabilization and their development into epididymides. Currently, little is understood about the molecular mechanisms of Wolffian duct differentiation. The objective of this study was to identify changes in gene expression associated with altered morphology of the proximal Wolffian duct following in utero exposure to DBP. Pregnant Crl:CD(R) (SD) rats were gavaged with corn oil vehicle or 500 mg/kg/day DBP from GD 12 to GD 19 or 21. There were only small morphological differences between control and DBP-exposed Wolffian ducts on GD 19. On GD 21, 89% of male fetuses in the DBP dose group showed marked underdevelopment of Wolffian ducts, characterized by decreased coiling. RNA was isolated from Wolffian ducts on GD 19 and 21. Together with empirical information, cDNA microarrays were used to help identify candidate genes that could be associated with the morphological changes observed on GD 21. These candidate genes were analyzed by real-time RT-PCR. Changes in mRNA expression were observed in genes within the insulin-like growth factor (IGF) pathway, the matrix metalloproteinase (MMP) family, the extracellular matrix, and in other developmentally conserved signaling pathways. On GD 19, immunolocalization of IGF-1 receptor protein demonstrated an increase in cytoplasmic expression in the mesenchymal and epithelial cells. There was also a variable decrease in androgen receptor protein in ductal epithelial cells on GD 19. This study provides insight into the effects of antiandrogens on the molecular mechanisms involved in Wolffian duct development. The altered morphology and changes in gene expression following DBP exposure are suggestive of altered paracrine interactions between ductal epithelial cells and the surrounding mesenchyme during Wolffian duct differentiation due to lowered testosterone production.",
         "",
         "https://academic.oup.com/toxsci/article-pdf/86/1/161/4657319/kfi172.pdf"
        ],
        [
         "41",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         "Prospective grading of neoplastic change in rat esophagus epithelium using angle-resolved low-coherence interferometry.",
         "Angle-resolved low-coherence interferometry (a/LCI) is used to obtain quantitative, depth-resolved nuclear morphology measurements. We compare the average diameter and texture of cell nuclei in rat esophagus epithelial tissue to grading criteria established in a previous a/LCI study to prospectively grade neoplastic progression. We exploit the depth resolution of a/LCI to exclusively examine the basal layer of the epithelium, approximately 50 to 100 microm beneath the tissue surface, without the need for exogenous contrast agents, tissue sectioning, or fixation. The results of two studies are presented that compare the performance of two a/LCI modalities. Overall, the combined studies show 91% sensitivity and 97% specificity for detecting dysplasia, using histopathology as the standard. In addition, the studies enable the effects of dietary chemopreventive agents, difluoromethylornithine (DFMO) and curcumin, to be assessed by observing modulation in the incidence of neoplastic change. We demonstrate that a/LCI is highly effective for monitoring neoplastic change and can be applied to assessing the efficacy of chemopreventive agents in the rat esophagus.",
         "",
         "https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-10/issue-5/051604/Prospective-grading-of-neoplastic-change-in-rat-esophagus-epithelium-using/10.1117/1.2102767.pdf"
        ],
        [
         "42",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "The role of fear-relevant stimuli in visual search: a comparison of phylogenetic and ontogenetic stimuli.",
         "It has been argued that phylogenetic fear-relevant stimuli elicit preattentive capture of attention. To distinguish between fear relevance and time of appearance in evolutionary history, the authors compare phylogenetic and ontogenetic fear-relevant and fear-irrelevant stimuli in a visual search task. The authors found no evidence for a special role of phylogenetic fear-relevant stimuli; it seems that fear relevance in general is more important than is the evolutionary age. The pattern of results indicates that attention toward threatening stimuli is mainly affected by a late component that prolongs the disengagement of attention.",
         "Attentional control of the processing of neural and emotional stimuli. Searching for threat Emotion drives attention: detecting the snake in the grass. The face in the crowd revisited: a threat advantage with schematic stimuli. Fear and anxiety: animal models and human cognitive psychophysiology. Facial Expressions of Emotion: Are Angry Faces Detected More Efficiently? A subcortical pathway to the right amygdala mediating \"unseen\" fear. It Takes a Confounded Face to Pop Out of a Crowd On the automatic nature of phobic fear: conditioned electrodermal responses to masked fear-relevant stimuli. Orienting and schizophrenia: stimulus significance, attention, and distraction in a signaled reaction time task. A feature-integration theory of attention The premise of equipotentiality in human classical conditioning: conditioned electrodermal responses to potentially phobic stimuli. One-trial learning and superior resistance to extinction of autonomic responses conditioned to potentially phobic stimuli. Phobias and preparedness The cognitive neuroscience of emotion Fears, phobias, and preparedness: toward an evolved module of fear and fear learning. Unconscious emotion : Evolutionary perspectives, psychophysiological data, and neuropsychological mechanisms A threat imminence approach to human fear responding : direction of threath, aversive contexts, and electrodermal responses Preparedness and electrodermal fear-conditioning: ontogenetic vs phylogenetic explanations. Finding the face in the crowd: an anger superiority effect. Feature analysis in early vision: evidence from search asymmetries. The emotional brain. E-Prime 32306-1270), as editor of the Clinician's Research Digest newsletter for",
         "https://access.archive-ouverte.unige.ch/access/metadata/414fbfe1-4481-4240-92c1-e62a3593f0f0/download"
        ],
        [
         "43",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "WBN/Kob-Ht rats spontaneously develop dermatitis under conventional conditions: another possible model for atopic dermatitis.",
         "WBN/Kob-Ht rats (Ht-rats) raised under conventional conditions spontaneously developed dermatitis. In this study, we carried out histopathological analysis to elucidate the pathological features of the dermatitis in Ht-rats. We then tried to detect Staphylococcus species recovered from the skin lesions of Ht-rats. We also measured the serum levels of total IgE, IL-4 and IFN-gamma in these rats. The histopathological data indicated that inflammatory cells had infiltrated the skin lesions. Staphylococcus aureus was recovered from the skin lesions, and the serum levels of total IgE and IL-4 were elevated in Ht-rats with dermatitis. These results suggest that dermatitis in Ht-rats is similar to that in the DS-Nh mice, which has recently been proposed as an animal model for human atopic dermatitis.",
         "Role of the Nh (Non-hair) mutation in the development of dermatitis and hyperproduction of IgE in DS-Nh mice. DS–Nh as an experimental model of atopic dermatitis induced by Staphylococcus aureus producing staphylococcal enterotoxin C Characterization of dermatitis arising spontaneously in DS-Nh mice maintained under conventional conditions: another possible model for atopic dermatitis. Atopic dermatitis: new insights and opportunities for therapeutic intervention. Locus of dominant hairless gene (Ht) causing abnormal hair and keratinization maps to rat chromosome 10. Characteristics of spontaneous erythema appeared in hairless rats. IgE production by normal human lymphocytes is induced by interleukin 4 and suppressed by interferons gamma and alpha and prostaglandin E2.",
         "https://www.jstage.jst.go.jp/article/expanim/54/5/54_5_461/_pdf"
        ],
        [
         "44",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "Structural Analyses Reveal Phosphatidyl Inositols as Ligands for the NR5 Orphan Receptors SF-1 and LRH-1",
         "None",
         "The DEAD-Box Protein DP103 (Ddx20 or Gemin-3) Represses Orphan Nuclear Receptor Activity via SUMO Modification Amplification of the steroidogenic factor 1 gene in childhood adrenocortical tumors. Identification of liver receptor homolog-1 as a novel regulator of apolipoprotein AI gene transcription. Synergy between LRH-1 and beta-catenin induces G1 cyclin-mediated cell proliferation. Quantitative Proteomics of the Thyroid Hormone Receptor-Coregulator Interactions*[boxs] LRH-1: an orphan nuclear receptor involved in development, metabolism and steroidogenesis. Sex determination: a ‘window’ of DAX1 activity Generation and Characterization of Human Steroidogenic Factor 1 LBD Crystals with and without Bound Cofactor Peptide SHIP-2 and PTEN Are Expressed and Active in Vascular Smooth Muscle Cell Nuclei, but Only SHIP-2 Is Associated with Nuclear Speckles* Resurrecting the Ancestral Steroid Receptor: Ancient Origin of Estrogen Signaling MrBayes 3: Bayesian phylogenetic inference under mixed models Activation of nuclear receptors: a perspective from structural genomics. StAR-related Lipid Transfer (START) Proteins: Mediators of Intracellular Lipid Metabolism* The Drosophila Orphan Nuclear Receptor DHR38 Mediates an Atypical Ecdysteroid Signaling Pathway Structural basis for ligand-independent activation of the orphan nuclear receptor LRH-1. Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors Nuclear lipid signalling Nuclear lipids: New functions for old molecules? X-Ray Structure of the hRORα LBD at 1.63 Å Elevation of plasma phospholipid transfer protein in transgenic mice increases VLDL secretion Published, JLR Papers in Press, August 16, 2002. DOI 10.1194/jlr.M200166-JLR200 Crystal Structure of the HNF4α Ligand Binding Domain in Complex with Endogenous Fatty Acid Ligand* Phosphorylation and Intramolecular Stabilization of the Ligand Binding Domain in the Nuclear Receptor Steroidogenic Factor 1 Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids. Inhibition of androgen receptor (AR) function by the reproductive orphan nuclear receptor DAX-1. Control of Cholesterol Turnover in the Mouse* Phosphatidic Acid-Mediated Mitogenic Activation of mTOR Signaling The PITP family of phosphatidylinositol transfer proteins Characterization of a G Protein-activated Phosphoinositide 3-Kinase in Vascular Smooth Muscle Cell Nuclei* Binding of ligands and activation of transcription by nuclear receptors. Structure of a Multifunctional Protein Crystal Structure of the Ligand-binding Domain of the Ultraspiracle Protein USP, the Ortholog of Retinoid X Receptors in Insects* Haploinsufficiency of steroidogenic factor-1 in mice disrupts adrenal development leading to an impaired stress response. Nuclear Lipid Signaling Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors. Ligand binding and nuclear receptor evolution A new method to localize and test the significance of incongruence: detecting domain shuffling in the nuclear receptor superfamily. Inferring the historical patterns of biological evolution Oxysterol Regulation of Steroidogenic Acute Regulatory Protein Gene Expression Structure and specificity of nuclear receptor-coactivator interactions. 25-Hydroxycholesterol is not a ligand for the orphan nuclear receptor steroidogenic factor-1 (SF-1). Wilms' Tumor 1 and Dax-1 Modulate the Orphan Nuclear Receptor SF-1 in Sex-Specific Gene Expression Activation of the orphan nuclear receptor steroidogenic factor 1 by oxysterols. DAX-1 inhibits SF-1-mediated transactivation via a carboxy-terminal domain that is deleted in adrenal hypoplasia congenita A new method of inference of ancestral nucleotide and amino acid sequences. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. The nuclear receptor steroidogenic factor 1 acts at multiple levels of the reproductive axis. A cell-specific nuclear receptor is essential for adrenal and gonadal development and sexual differentiation Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology Repression of orphan nuclear recep - or activity via SUMO - modification and the Dead - Box Protein P 103 ( Ddx 20 , Gemin - 3 ) Nuclear lipids: key signaling effectors in the nervous system and other tissues X-ray structure of the hRORalpha LBD at 1.63 A: structural and functional data that cholesterol or a cholesterol derivative is the natural ligand of RORalpha. Nuclear-receptor ligands and ligand-binding domains. Standard Fmoc protocols. Methods Enzymol",
         "http://www.cell.com/article/S009286740500098X/pdf"
        ],
        [
         "45",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "Rat Strain Effects of AT1 Receptor Activation on D1 Dopamine Receptors in Immortalized Renal Proximal Tubule Cells",
         "The dopaminergic and renin-angiotensin systems regulate blood pressure, in part, by affecting sodium transport in renal proximal tubules (RPTs). We have reported that activation of a D1-like receptor decreases AT1 receptor expression in the mouse kidney and in immortalized RPT cells from Wistar-Kyoto (WKY) rats. The current studies were designed to test the hypothesis that activation of the AT1 receptor can also regulate the D1 receptor in RPT cells, and this regulation is aberrant in spontaneously hypertensive rats (SHRs). Long-term (24 hours) stimulation of RPT cells with angiotensin II, via AT1 receptors increased total cellular D1 receptor protein in a time- and concentration-dependent manner in WKY but not in SHR cells. Short-term stimulation (15 minutes) with angiotensin II did not affect total cellular D1 receptor protein in either rat strain. However, in the short-term experiments, angiotensin II decreased cell surface membrane D1 receptor protein in WKY but not in SHR cells. D1 and AT1 receptors colocalized (confocal microscopy) and their coimmunoprecipitation was greater in WKY than in SHRs. However, AT1/D1 receptor coimmunoprecipitation was decreased by angiotensin II (10−8M/24 hours) to a similar extent in WKY (−22±8%) and SHRs (−22±12%). In summary, these studies show that AT1 and D1 receptors interact differently in RPT cells from WKY and SHRs. It is possible that an angiotensin II-mediated increase in D1 receptors and dissociation of AT1 from D1 receptors serve to counter regulate the long-term action of angiotensin II in WKY rats; different effects are seen in SHRs.",
         "",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.HYP.0000184251.01159.72"
        ],
        [
         "46",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         "The value of c-kit mutational analysis in a cytokeratin positive gastrointestinal stromal tumour",
         "The expression of cytokeratins in gastrointestinal stromal tumours (GISTs) is rare and may lead to diagnostic confusion when it occurs. This report describes a metastatic GIST that stained strongly for cytokeratins, CD117, and CD34 in a patient who was previously diagnosed with gastric epithelioid angiosarcoma. A review of both tumours showed the same histological and immunohistochemical profiles, and c-kit molecular analysis revealed an insertional mutation at codon 558 of exon 11 in both tumours. Thus, pathologists should be aware that GISTs can occasionally express cytokeratins, and that c-kit mutational investigations may have a key diagnostic role and may prevent diagnostic mistakes that could have important clinical implications.",
         "",
         "https://jcp.bmj.com/content/jclinpath/58/9/991.full.pdf"
        ],
        [
         "47",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "Contribution of high p34cdc2 kinase activity to premature chromosome condensation of injected somatic cell nuclei in rat oocytes.",
         "The present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated rat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes were placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-dependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15-45 min rather than 46-120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity transiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-phase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed oocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat oocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent level of aged (120 min) or enucleated (+60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2 kinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were treated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated oocytes. A significantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control oocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus cell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage. These results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for their inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemicals such as MG132 is in part effective for rat oocytes to promote PCC and further development.",
         "",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/129/2/1290171.pdf"
        ],
        [
         "48",
         "002cc4c2e457c3cc0fb0f704d0483a003e8a0575",
         "Comparison of automated microbroth dilution and agar dilution for antimicrobial susceptibility of Campylobacter jejuni isolated from dairy sources.",
         "OBJECTIVES\nTo compare the agreement between microbroth dilution and agar dilution for antimicrobial susceptibility testing of Campylobacter jejuni.\n\n\nMETHODS\nUtilizing commercially prepared antimicrobial panels, microbroth dilution was compared with agar dilution for determining antimicrobial susceptibility in C. jejuni isolates. To assess the performance of both techniques for ampicillin, 190 C. jejuni isolates from dairy cattle were utilized. A group of 172 C. jejuni isolates from dairy sources were used to compare the susceptibility to ciprofloxacin, erythromycin, nalidixic acid and tetracycline.\n\n\nRESULTS\nOur results indicate that microbroth dilution and agar dilution agree within +/-1 log2 dilution for 86.7% of the isolates tested. Ciprofloxacin had the highest level of agreement for isolates tested by both techniques, resulting in a kappa of 0.886 and 97.1% agreement +/-1 log2 dilution. The least agreement was observed in determining the susceptibility of isolates to ampicillin and erythromycin (82.1 and 79.7% agreement +/-1 log2 dilution). However, kappa statistics were considered to have good agreement for these antimicrobials. There were no significant differences in the summary statistics for any of the five antimicrobials evaluated for the isolates analysed by the percentage of resistant isolates, MIC50, MIC75 or MIC90 beyond +/-1 log2 dilution. There was no association in the classification of resistance by the testing methods employed. We also demonstrated that the quality control strain of C. jejuni ATCC 33650 performed in a consistent manner for both agar dilution and microbroth dilution.\n\n\nCONCLUSIONS\nMicrobroth dilution may be an acceptable alternative to agar dilution for determining susceptibility of C. jejuni in research or surveillance where flow of samples, labour efficiency and cost may restrict the use of agar dilution.",
         "",
         "https://academic.oup.com/jac/article-pdf/56/4/686/2308205/dki309.pdf"
        ],
        [
         "49",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "Numerical Periodic Normalization for Codim 1 Bifurcations of Limit Cycles",
         "Explicit computational formulas for the coefficients of the periodic normal forms for all codim 1 bifurcations of limit cycles in generic autonomous ODEs are derived. They include second-order coefficients for the fold (limit point) bifurcation, as well as third-order coefficients for the flip (period-doubling) and Neimark--Sacker (torus) bifurcations. The formulas are independent of the dimension of the phase space and involve solutions of certain boundary-value problems on the interval [0,T], where T is the period of the critical cycle, as well as multilinear functions from the Taylor expansion of the right-hand sides near the cycle. The formulas allow us to distinguish between sub- and supercritical bifurcations, in agreement with earlier asymptotic expansions of the bifurcating solutions. Our formulation makes it possible to use robust numerical boundary-value algorithms based on orthogonal collocation, rather than shooting techniques, which greatly expands its applicability. The actual implementation is described in detail. We include three numerical examples, in which codim 2 singularities are detected along branches of codim 1 bifurcations of limit cycles as zeros of the periodic normal form coefficients.",
         "Numerical Methods for Two-Parameter Local Bifurcation Analysis of Maps Newton's descent method for the determination of invariant tori. A Newton Method for Locating Invariant Tori of Maps Sensitivity Analysis of ODES/DAES Using the Taylor Series Method Numerical Normal Forms for Codim 2 Bifurcations of Fixed Points with at Most Two Critical Eigenvalues Numerical Methods for the Continuation of Invariant Tori MATCONT: A MATLAB package for numerical bifurcation analysis of ODEs Cl_matcont: a continuation toolbox in Matlab Computation of Periodic Solution Bifurcations in ODEs Using Bordered Systems Chapter 4 – Numerical Continuation, and Computation of Normal Forms Computing Periodic Orbits and their Bifurcations with Automatic Differentiation Numerical Normalization Techniques for All Codim 2 Bifurcations of Equilibria in ODE's Topics in bifurcation theory and applications Elements of applied bifurcation theory (2nd ed.) Harmonic balance analysis of period-doubling bifurcations with implications for control of nonlinear dynamics Algorithm 755: ADOL-C: a package for the automatic differentiation of algorithms written in C/C++ Bifurcation and predictability analysis of a low-order atmospheric circulation model Models of complex dynamics in nonlinear systems Bifurcation sequences at 1:4 resonance: an inventory Harmonic balance methods for the analysis of chaotic dynamics in nonlinear systems BIFURCATIONS AND CHAOS IN A PERIODIC PREDATOR-PREY MODEL NUMERICAL ANALYSIS AND CONTROL OF BIFURCATION PROBLEMS (II): BIFURCATION IN INFINITE DIMENSIONS Numerical Calculation of Invariant Tori THE QUASIPERIODIC ROUTE TO CHAOS IN A MODEL OF THE PEROXIDASE-OXIDASE REACTION Global Characterization of the Normal Form for a Vector Field Near a Closed Orbit A simple global characterization for normal forms of singular vector fields Normal form reduction for time-periodically driven differential equations Numerical computation of invariant circles of maps NONLINEAR OSCILLATIONS, DYNAMICAL SYSTEMS, AND BIFURCATIONS OF VECTOR FIELDS (Applied Mathematical Sciences, 42) Irregularity: a fundamental property of the atmosphere* Nonlinear Oscillations, Dynamical Systems, and Bifurcations of Vector Fields Collocation at Gaussian Points Continuation of Quasi-periodic Invariant Tori The fold-flip bifurcation On the computation of characteristic multipliers for predicting limit cycle bifurcations Elements of applied bifurcation theory AUTO 2000 : CONTINUATION AND BIFURCATION SOFTWARE FOR ORDINARY DIFFERENTIAL EQUATIONS (with HomCont) CONTENT: A multiplatform environment for analyzing dynamical systems Normal forms of bifurcating periodic orbits Numerical computation of periodic solution branches and oscillatory dynamics of the stirred tank reactor with A → B → C reactions Geometrical Methods in the Theory of Ordinary Differential Equations Elementary stability and bifurcation theory Bifurcation of maps and applications Topics in bifurcation theory Numerical solution of boundary value problems in ordinary differential equations Numerical Continuation, And Computation Of Normal Forms CAPD: Computer assisted proofs in dynamics",
         "http://arxiv.org/pdf/1111.4445.pdf"
        ]
       ],
       "shape": {
        "columns": 5,
        "rows": 69988
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>referenceTitles</th>\n",
       "      <th>openAccessPdf</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>Disruption of Fibroblast Growth Factor Signal ...</td>\n",
       "      <td>Purpose: Synovial sarcoma is a soft tissue sar...</td>\n",
       "      <td></td>\n",
       "      <td>https://aacrjournals.org/clincancerres/article...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>Shuffled iterative decoding</td>\n",
       "      <td>Shuffled versions of iterative decoding of low...</td>\n",
       "      <td>Low-Density Parity-Check Codes Turbo decoder a...</td>\n",
       "      <td>http://www.merl.com/publications/docs/TR2005-0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>Diphasic effects of Astragalus membranaceus BU...</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.jstage.jst.go.jp/article/bpb/28/8/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>Critical roles of CD146 in zebrafish vascular ...</td>\n",
       "      <td>In this report, we use zebrafish as a model sy...</td>\n",
       "      <td></td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/pdfdirect/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>Differential Contribution of Troponin I Phosph...</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td></td>\n",
       "      <td>http://www.jbc.org/article/S0021925820589394/pdf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69983</th>\n",
       "      <td>fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6</td>\n",
       "      <td>Irregular cycles and steroid hormones in polyc...</td>\n",
       "      <td>BACKGROUND\\nThis cross-sectional study was und...</td>\n",
       "      <td>Predictors for treatment failure after laparos...</td>\n",
       "      <td>https://academic.oup.com/humrep/article-pdf/20...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69984</th>\n",
       "      <td>fffbc9b37f20515de2ff4fe60353d52dc37e65f5</td>\n",
       "      <td>An organizational grid of federated MOSIX clus...</td>\n",
       "      <td>MOSIX is a cluster management system that uses...</td>\n",
       "      <td>From sandbox to playground: dynamic virtual en...</td>\n",
       "      <td>http://www.cs.virginia.edu/~jl3aq/courses/CS85...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69985</th>\n",
       "      <td>fffd2d6d54b4023ece28bfdebffdfb00502a09ec</td>\n",
       "      <td>Mutations in the RNA Polymerase III Subunit Rp...</td>\n",
       "      <td>ABSTRACT Termination by RNA polymerase III (Po...</td>\n",
       "      <td>Nuclear surveillance and degradation of hypomo...</td>\n",
       "      <td>https://europepmc.org/articles/pmc543423?pdf=r...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69986</th>\n",
       "      <td>ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3</td>\n",
       "      <td>The Core Histone N-terminal Tail Domains Funct...</td>\n",
       "      <td>Salt-dependent oligomerization of nucleosomal ...</td>\n",
       "      <td>Histone modifications: combinatorial complexit...</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820789519/pdf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69987</th>\n",
       "      <td>ffffeaedbe7931d161ab532015de3be15f99b180</td>\n",
       "      <td>Of mice and men: should we extrapolate rodent ...</td>\n",
       "      <td>None</td>\n",
       "      <td>Developmental neurotoxicity of ketamine: morph...</td>\n",
       "      <td>https://pubs.asahq.org/anesthesiology/article-...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>69988 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paperId  \\\n",
       "0      0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1      000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2      000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3      0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4      00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "...                                         ...   \n",
       "69983  fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6   \n",
       "69984  fffbc9b37f20515de2ff4fe60353d52dc37e65f5   \n",
       "69985  fffd2d6d54b4023ece28bfdebffdfb00502a09ec   \n",
       "69986  ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3   \n",
       "69987  ffffeaedbe7931d161ab532015de3be15f99b180   \n",
       "\n",
       "                                                   title  \\\n",
       "0      Disruption of Fibroblast Growth Factor Signal ...   \n",
       "1                            Shuffled iterative decoding   \n",
       "2      Diphasic effects of Astragalus membranaceus BU...   \n",
       "3      Critical roles of CD146 in zebrafish vascular ...   \n",
       "4      Differential Contribution of Troponin I Phosph...   \n",
       "...                                                  ...   \n",
       "69983  Irregular cycles and steroid hormones in polyc...   \n",
       "69984  An organizational grid of federated MOSIX clus...   \n",
       "69985  Mutations in the RNA Polymerase III Subunit Rp...   \n",
       "69986  The Core Histone N-terminal Tail Domains Funct...   \n",
       "69987  Of mice and men: should we extrapolate rodent ...   \n",
       "\n",
       "                                                abstract  \\\n",
       "0      Purpose: Synovial sarcoma is a soft tissue sar...   \n",
       "1      Shuffled versions of iterative decoding of low...   \n",
       "2      This study was designed to investigate the eff...   \n",
       "3      In this report, we use zebrafish as a model sy...   \n",
       "4      Cardiac troponin I is a phosphorylation target...   \n",
       "...                                                  ...   \n",
       "69983  BACKGROUND\\nThis cross-sectional study was und...   \n",
       "69984  MOSIX is a cluster management system that uses...   \n",
       "69985  ABSTRACT Termination by RNA polymerase III (Po...   \n",
       "69986  Salt-dependent oligomerization of nucleosomal ...   \n",
       "69987                                               None   \n",
       "\n",
       "                                         referenceTitles  \\\n",
       "0                                                          \n",
       "1      Low-Density Parity-Check Codes Turbo decoder a...   \n",
       "2                                                          \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "69983  Predictors for treatment failure after laparos...   \n",
       "69984  From sandbox to playground: dynamic virtual en...   \n",
       "69985  Nuclear surveillance and degradation of hypomo...   \n",
       "69986  Histone modifications: combinatorial complexit...   \n",
       "69987  Developmental neurotoxicity of ketamine: morph...   \n",
       "\n",
       "                                           openAccessPdf  \n",
       "0      https://aacrjournals.org/clincancerres/article...  \n",
       "1      http://www.merl.com/publications/docs/TR2005-0...  \n",
       "2      https://www.jstage.jst.go.jp/article/bpb/28/8/...  \n",
       "3      https://onlinelibrary.wiley.com/doi/pdfdirect/...  \n",
       "4       http://www.jbc.org/article/S0021925820589394/pdf  \n",
       "...                                                  ...  \n",
       "69983  https://academic.oup.com/humrep/article-pdf/20...  \n",
       "69984  http://www.cs.virginia.edu/~jl3aq/courses/CS85...  \n",
       "69985  https://europepmc.org/articles/pmc543423?pdf=r...  \n",
       "69986   http://www.jbc.org/article/S0021925820789519/pdf  \n",
       "69987  https://pubs.asahq.org/anesthesiology/article-...  \n",
       "\n",
       "[69988 rows x 5 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_abstract",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_introduction",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_results",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_discussion",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_methods",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_text",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "author_list",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "problem",
         "rawType": "object",
         "type": "unknown"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "055befbd-8e0d-43d3-949e-5d933d47c21f",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "Abstract Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which\nare unknown.We investigated the involvement of fibroblast growth factor (FGF) signals in synovial\nsarcoma and evaluated the therapeutic effect of inhibiting the FGF signal.\nExperimental Design:The expression of 22 FGF and 4 FGF receptor (FGFR) genes in18 prima-\nry tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR.\nEffects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein\nkinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory\neffects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo.\nResults: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed\nin neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma\ncell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal ^\nregulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the\nFGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle ar-\nrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the\nFGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not\np38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sar-\ncoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2.\nConclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibi-\ntory molecules will be of potential use for molecular target therapy in synovial sarcoma.",
         "Synovial sarcoma is the most frequent soft-tissue sarcoma\n(STS) among patients in the third to fourth decade of life (1)\nand accounts for about 7% to 10% of all human STSs (2). It\npredominantly affects the lower extremities but can occur in\nany part of the body. Surgical resection with an adequate\nfor\nsurgical margin is the definitive choice of\nprimary tumors and has been shown to control\nlocal\nrecurrence (3, 4). Disseminated distant metastasis is the major\ncause of poor outcome, and several reports describing the\nresults of current therapy showed a 5-year survival rate of\naround 50% to 60% (3, 4). The efficacy of adjuvant\nchemotherapy is still a matter of debate, and the development\nof a new therapeutic approach is required to improve the\nprognosis.\nthe reciprocal\nDespite little progress in clinical treatment during the last\n20 years, cytogenetic and molecular genetic analyses have greatly\nimproved the understanding of this type of tumor, especially\nwith the discovery of\ntranslocation\nt(18;X)(q11;p11) creating an SYT-SSX fusion gene as a synovial\nsarcoma – specific genetic alteration (5, 6). Thus far, three SSX\ngenes (SSX1, SSX2, and SSX4) have been identified as a partner\nof the SYT gene, and >95% of synovial sarcoma tumors carried\none of these fusion genes (7). Although the precise function and\nthe mechanism of oncogenesis are not yet clearly shown, the\nhigh sensitivity and specificity of the SYT-SSX fusion gene in\nsynovial sarcoma have proven to be useful for molecular\ndiagnosis (8). In particular types of tumors with a specific\nreciprocal translocation such as PML-RARa in acute promyelo-\ncytic leukemia (9) and BCR-ABL in chronic myelogenous\nleukemia (10), fusion gene products themselves serve as targets\nfor the therapy. Although immunotherapy using a peptide\nderived from SYT-SSX protein as a specific vaccine has been\ninvestigated (11), no therapeutic approach has been discovered\nfor directly targeting the fusion protein or its function.\nGene expression profiling of tumors has been shown a\npowerful tool with which to isolate a molecular target for\ntherapy (12). We have done a gene expression analysis of\nsynovial sarcoma using a genome-wide cDNA microarray and\nfound that synovial sarcoma shared its molecular signature with\nmalignant peripheral nerve sheath tumor, of which the\nprecursors were neural crest – derived cells, and also identified\na set of genes commonly up-regulated in synovial sarcoma\nincluding the fibroblast growth factor 18 (FGF18) gene (13). The\nFGF signaling pathway seems to play significant roles in tumor\ndevelopment and progression (14 – 16), and recently FGF18 was\nidentified as an autocrine growth factor involved in colon\ncancers (17). Thus far, 22 genes have been identified as members\nof the FGF family, and our cDNA microarray contained 10 FGF\ngenes (FGF2, FGF3, FGF4, FGF7, FGF9, FGF11, FGF12, FGF13,\nFGF18, and FGF19). We found that some FGF genes other\nthan FGF18 were also expressed in synovial sarcoma (data\nshown). Based on these results, we have done an\nnot\nintensive analysis of FGF and its receptor (FGFR) genes in\nsynovial sarcoma and also investigated whether inhibition of\nthe FGF signal is a new therapeutic modality for synovial\nsarcoma.",
         "Results\nSynovial sarcoma cell lines expressed multiple fibroblast growth\nfactor and fibroblast growth factor receptor genes. The mRNA\nexpression of 22 subtypes of FGF genes was analyzed by RT-\nPCR in five synovial sarcoma and five other cell lines (Fig. 1A).\nfive synovial sarcoma cell\nNone of the five synovial sarcoma cell lines expressed the\nFGF5, FGF6, FGF12, FGF14, FGF16, FGF17, FGF22, or FGF23\ngene, whereas the FGF2, FGF8, FGF9, FGF11, and FGF18 genes\nwere expressed in all\nlines.\nExpression of the remaining nine FGF genes varied among\ncell\nlines. The expression profile in NMS-2 (malignant\nperipheral nerve sheath tumor) was similar to that in synovial\nsarcoma, which may relate to the overall similarity of the gene\nexpression profile between synovial sarcoma and malignant\nperipheral nerve sheath tumor (13). HT1080 (fibrosarcoma)\nexpressed a similar set of FGF genes to malignant peripheral\nnerve sheath tumor but at much lower levels, and the\nexpression profile of Saos2 (osteosarcoma) was quite different\nbeing positive only for FGF1 and FGF13. Interestingly, synovial\nsarcoma shared with colon carcinoma cell lines the expression\nof some FGF genes such as FGF3 and FGF18 genes, which were\nnot expressed in mesenchymal cell lines. Secretion of these FGF\nproteins by synovial sarcoma cell lines was confirmed by the\nWestern blotting of culture supernatant. FGF18 proteins were\nclearly detected in culture supernatant of synovial sarcoma cell\nlines, although the level of protein was not completely agreed\nwith the level of RNA expression (Fig. 1C).\nIn contrast with the heterogeneous expression pattern of\nthe expression profile of receptors (FGFR) was\nligands,\nrelatively homogeneous among synovial sarcoma cell lines,\nbeing positive for all subtypes analyzed except FGFR2b, which\nis an epithelia-specific FGFR expressed in colon carcinomas\n(Fig. 1B). In addition to FGFR2b, NMS-2 was negative for\nFGFR3, and HT1080 was negative for FGFR2c. Expression of\nFGFR3 in synovial sarcoma cell lines was confirmed at the\nprotein level by the Western blotting using cell lysates of each\ncell line (Fig. 1D).\nExpression of fibroblast growth factor and fibroblast growth\nfactor receptor genes in tumor tissues.\nThe FGF2, FGF8,\nFGF9, FGF11, and FGF18 genes, which were commonly ex-\npressed in the five synovial sarcoma cell lines, were inves-\ntigated in tumor tissues of 18 synovial sarcomas and 11 STSs\nof other types (Fig. 2A). In accordance with the results in the\ncell lines, almost all 18 synovial sarcoma tumors expressed\nall of these FGF genes (Fig. 2A). Among five FGF genes, the\nFGF2, FGF9, and FGF11 genes were also expressed in some\ntumors other than synovial sarcoma. In contrast, expression of\nFGF8 and FGF18 genes was only weakly detected in other\ntumors. Expression profiles of these five FGF genes were further\nanalyzed by the quantitative RT-PCR (Fig. 2B). The level of\nexpression of FGF8 and FGF18 genes in synovial sarcoma\ntumors were significantly higher than in tumor tissues of other\ntype of tumors, whereas those of FGF2, FGF9, and FGF11 genes\nshowed no difference.\nAs for receptor genes, synovial sarcoma tumors expressed all\ntypes of FGFR genes analyzed including the FGFR2b gene\nGrowth stimulatory effects of\nfibroblast growth factors in\nsynovial sarcoma cell lines. Expression of FGF and FGFR genes\nand proteins suggested the involvement of FGFs as autocrine/\nparacrine growth factors in synovial sarcoma. Among FGF genes\nexpressed in synovial sarcoma both in vitro and in vivo, we\nfocused on the FGF2, FGF8, and FGF18 genes in subsequent\nexperiments. The FGF2 gene was selected as a gene commonly\nup-regulated in STS (Fig. 2A). FGF8 and FGF18 genes were\nselected as synovial sarcoma – specific up-regulated FGF genes\n(Fig. 2A).\nGrowth stimulatory effects of each rhFGF protein were\nserum-starved conditions. Recombinant\nanalyzed under\nhFGF8 showed a dose-dependent growth-promoting effect\nin all synovial sarcoma cell lines (Fig. 3A). On the treatment\nwith rhFGF18, only HS-SY-II\nshowed a dose-dependent\nincrease (Fig. 3B). A heterogeneous response to FGF was\nmore clearly observed in the case of rhFGF2, in which the\ngrowth of HS-SY-II\nincrease,\nwhereas a growth inhibitory effect was observed in the case\nof YaFuSS (Fig. 3C).\nshowed a dose-dependent\nActivation of mitogen-activated protein kinases by fibroblast\ngrowth factor signals. One of major signal pathways located\ndownstream of FGFR is the MAPKs, for which three subtypes are\nknown: ERK, p38 MAPKs, and c-Jun NH2-terminal kinases\n(JNK; ref. 25). To investigate the events downstream of activated\nFGFRs, the phosphorylation of these MAPKs was investigated. In\nall synovial sarcoma cell lines analyzed, ERK1/2 was phosphor-\nylated under serum-free conditions, suggesting the presence of\nendogenous stimulatory signals (Fig. 4A). The amount of\nphosphorylated ERK1/2 (pERK1/2) increased on treatment\nwith FGF8 in a dose-dependent manner in all four cell lines\nanalyzed, with the response of HS-SY-II being the highest\nlevels. A dose-dependent\nFibroblast growth factor receptor – specific inhibitors reduce the\nlines in accordance with\ngrowth of synovial sarcoma cell\ninactivation of fibroblast growth factor receptor. MAPKs in\nsynovial sarcoma are activated by intrinsic pathways, and\nexogenous FGFs can stimulate the growth of synovial sarcoma,\nsuggesting that inhibition of the intrinsic FGF signal pathway\nmay inhibit the growth of synovial sarcoma. SU5402 is known\nto inhibit FGFR autophosphorylation at an IC50 of 10 to\n20 Amol/L (22). To confirm the effect of SU5402 on synovial\nsarcoma cell lines, the phosphorylation status of FGFR3 was\nanalyzed by the immunoprecipitation and Western blotting in\nthree synovial sarcoma cell lines (YaFuSS, SYO-1, and 1273/\n99; Fig. 5A). Phosphorylated FGFR3 was detected in these cell\nlines with serum-starved condition, and the treatment with\nrhFGF18 protein increased the amount of phosphorylated\nFGFR3 (Fig. 5A), which was compatible with the autocrine/\nparacrine model of FGFs in synovial sarcoma cells. Treatment\nwith SU5402 effectively decreased the phosphorylation of\nFGFR3 into almost undetectable level (Fig. 5A), indicating that\nSU5402 can be used as an inhibitor of FGFR in synovial\nsarcoma cells. By the treatment with SU5402, the growth of all\nfive synovial sarcoma cells was inhibited in a dose-dependent\nmanner (Fig. 5B), and the IC50 value ranged from 8.5 to 17.2\nAmol/L (Table 1). The growth inhibitory effect of SU5402 for\ncell lines other than synovial sarcoma was not remarkable. The\nsame experiments were done under low-serum conditions (1%\nFBS, Fig. 5C), resulting in similar growth inhibition in all\nsynovial sarcoma cell lines. The IC50 was almost identical in\nthe three cell lines (YaFuSS, HS-SY-II, and SYO-1), whereas the\nlines (Fuji and 1273/99) required a higher\nother two cell\nconcentration of SU5402 (Table 1). Growth inhibitory effects\nof another FGFR-specific inhibitor, PD166866 (23), was also\ninvestigated under the low-serum conditions. All five synovial\nsarcoma cell lines had an IC50 value (0.08-4.7 Amol/L) much\nlower than those of other cell lines (>10 Amol/L, Table 1).\nSimilar to the data obtained using SU5402, IC50 values in\nYaFuSS, HS-SY-II, and SYO-1 were much lower than those in\nFuji and 1273/99.\n11.5 F 1.8\n15.1 F3.4\n8.5 F 2.5\n17.2 F 1.8\n12.8 F 6.1\n46.4 F 6.9\n65.4 F 2.6\n>80\n>80\n54.4 F 21.5\n11.4 F 3.6\n4.9 F 0.5\n8.6 F 1.6\n25.3 F 3.6\n29.9 F 16.6\n60.2 F 20.6\n63.7 F 27.8\n55.8 F 19.1\n>80\n78.1 F20.3\n0.08 F 0.04\n0.77 F 0.52\n0.18 F 0.11\n4.0 F 1.6\n2.7 F 0.4\n>10\n>10\n>10\n>10\n>10\nGrowth inhibition of fibroblast growth factor receptor inhibitor\nassociated with the reduction of phosphorylation of extracellular\nsignal – regulated kinase 1/2 but not of p38.\nTo investigate\nwhich signals may contribute to the growth of synovial\nsarcoma, the phosphorylation of ERK1/2 and p38 before and\nafter the treatment with SU5402 was investigated. In three\nsynovial sarcoma cell lines (YaFuSS, HS-SY-II, and SYO-1),\nwhich were relatively sensitive to the growth inhibitory effect of\nSU5402, phosphorylation of ERK1/2 under low-serum con-\nditions was completely inhibited by SU5402, whereas the\namount of phosphorylated p38 showed no change (Fig. 6A). A\nsmall amount of pERK1/2 was still observed after SU5402\ntreatment in 1273/99, whereas no significant decrease was\nobserved in Fuji, both of which showed no significant change in\nthe pp38 (Fig. 6A). In cell lines other than synovial sarcoma,\nSU5402 showed no significant changes in the amount of\npERK1/2 or pp38 (data not shown). These results suggested that\nthe signal to induce the phosphorylation of ERK1/2 in synovial\nsarcoma was sent mainly through the FGF and FGFR pathway\nand that although the FGF signal induced the phosphorylation\nof both ERK1/2 and p38, the growth stimulatory effects were\ntransmitted through ERK1/2. To further confirm that the growth\ninhibition of SU5402 was due to the reduction in pERK1/2, the\nfunction of MAPK/ERK kinase 1/2 in synovial sarcoma, which is\na kinase of ERK1/2, was inhibited by U0216. Phosphorylation\nof ERK1/2 was effectively inhibited in all five cell lines, in which\n1273/99 still had a small amount of pERK1/2 at the lower\nconcentration (Fig. 6B). BrdUrd uptake was inhibited in a dose-\ndependent manner in all five cell lines, in which 1273/99 was\nrelatively resistant at the lower concentration, consistent with\nthe status of pERK1/2 (Fig. 6C). These results further indicated\nthat the growth inhibition of SU5402 in synovial sarcoma was\nachieved through the inhibition of signals through ERK1/2.\nGrowth inhibition by fibroblast growth factor receptor inhibitor\nin synovial sarcoma is associated with cell cycle arrest. The\nfraction of each cell cycle phase was analyzed before and after\nthe treatment with SU5402 in YaFuSS cells. SU5402 signifi-\ncantly increased the G1 and subG1 fractions, and decreased the\nS fraction (Fig. 6D). These data suggested that the growth\ninhibition of synovial sarcoma by SU5402 was due to its\ninduction of G1 arrest as previously reported for other tyrosine\nkinase inhibitors (26).\nFibroblast growth factor receptor inhibitors reduced the\nFor in vivo study,\ngrowth of synovial sarcoma cells in vivo.\nwe used SYO-1 because of\nits consistency in developing\ntumor, and PD166866 was used because of its lower IC50\nvalue. To confirm its efficacy in vivo, tumor tissue was taken\n30 minutes after the injection of PD166866. We found that\n0.1 mg of PD166866 was enough to reduce the phosphory-\nlated ERK1/2 in the tumor (Fig. 7A). Based on this result,\nthe growth inhibitory effect of PD166866 (0.1 and 0.5 mg/\nbody) was analyzed in vivo. Although PD166866 was unable\nto stop the growth of the tumor completely, a significant\ninhibition of growth was observed during the period of\nadministration (P < 0.01), and the effect was retained even at\nday 21 at the higher dose (P = 0.02, Fig. 7B). No definite\nreduction of body weight or pathologic changes in vital\norgans such as liver and kidney was observed (data not\nshown). No growth inhibitory effect was observed when\nHT1080 was used instead of SYO-1 even at the higher dose\n(data not shown).\nHistologic examination of tumor tissues showed no signifi-\ncant morphologic changes or focal necrosis. The number of\nPCNA-positive cells, however, was clearly smaller in PD166866-\ntreated tumors (19.3 F 6.6%, Fig. 7D) than control tumors\n(53.0 F 2.0%, Fig. 7C), indicating that PD166866 reduced the\nnumber of cells in S phase. No such difference was observed\nwhen HT1080 was used instead of SYO-1 (data not shown).",
         "Discussion\nFGFs may activate genetic programs which promote cell\ngrowth by at least one of three general mechanisms: first, as\nmitogens\nsecond, by\npromoting angiogenesis to supply a growing tumor; and\nthird, by inhibiting apoptosis and allowing tumor cells to\ncontinue to grow beyond normal constraints. We found that\nsynovial sarcoma cells expressed a number of FGF genes, of\nwhich FGF2 , FGF8 , FGF9 , FGF11 , and FGF18 were\ncommonly expressed. The involvement of FGF2 in malignant\ntumors as an autocrine growth factor and/or a paracrine\nangiogenic factor was first shown in brain tumors (27), and\nsubsequently in non – small cell carcinoma (14). FGF8 was\noriginally identified as an androgen-induced growth factor\nfound in conditioned medium of androgen-dependent mouse\nmammary carcinoma SC-3 cells and reported to have a\ngrowth stimulatory effect (28). The human FGF8 gene was\nexpressed in breast and prostate cancers (15, 16, 29) and\nassociated with anchorage-independent proliferation and\ninvasion (30, 31). FGF9 was originally purified from the\nconditioned medium of\nline NMC-G1 and\ndesignated as glia-activating factor due to its mitogenic\nactivity toward glial cells (32). FGF11 was one of four FGF\nhomology factors, which shared structural homology with\nfailed to activate FGFRs (33). Recently,\nclassic FGFs but\nFGF11 was identified as one of the gene products enriched in\nneuronal precursors during the development of the nervous\nsystem (34). FGF18 is cloned by its homology to FGF8 and\nFGF9 (35, 36). FGF18, in association with FGF8, is expressed\nat various times and places during embryogenesis, especially\nneurogenesis (33, 37). Mitotic activity was shown in colon\ncarcinomas,\nin which the expression of FGF18 was up-\nregulated by WNT signals (17). Overall, we found that\nsynovial sarcoma expressed several FGF genes, particularly\nthose expressed in neural tissues, which may further suggest\nthe neural origin of precursor cells of synovial sarcoma as we\npreviously proposed (13). Mitotic activity of exogeneous FGFs\nwas not remarkable in some cell lines (Fig. 3). The serum-\nfree medium used in this study contained insulin, which is\nknown to potentiate the effect of FGF, and the further\naddition of insulin showed no increase of the BrdUrd uptake\n(data not shown). It is possible that the extent of autocrine\nstimulation may already be saturating so that further FGF\nstimulation does not lead to additional growth.\nThe expression of FGFR1, FGFR2c, FGFR3, and FGFR4, to\nwhich FGF2 (1c, 3c and 4), FGF8 (2c, 3c, 4), and FGF18 (2c,\n3c, 4) preferentially bind (38, 39), was confirmed in both tumor\ntissues and cell lines. The expression of FGFR2b gene, which was\ndetected in tumor tissues but not in cell lines, is intriguing.\nFGFR2b is a receptor for keratinocyte growth factor (FGF7), and\nexpressed in epithelial cells (40). As expected, all biphasic\nsynovial sarcomas expressed the FGFR2b gene, whereas the\nexpression was low in most of the monophasic synovial\nsarcomas and other STSs. It is, however, intriguing that some\nmonophasic synovial sarcomas with no apparent epithelial\nstructures expressed the FGFR2b gene at a level comparable with\nthat in biphasic synovial sarcoma (Fig. 2D). All five synovial\nsarcoma cell lines used in this study including SYO-1, which was\nestablished from a biphasic synovial sarcoma, were negative for\nthe FGFR2b gene. These results suggested that synovial sarcoma\ntumor cells may have an intrinsic mechanism to express FGFR2b\nin vivo, which was somehow inhibited or disappeared in vitro.\nAlternatively, synovial sarcoma cells positive for FGFR2b may\nhave growth disadvantages, and therefore only negative cells\nwere established as cell\nlines. Recently, single nucleotide\npolymorphism in FGFR4 is reported to associate with poor\nprognosis in high-grade soft tissue sarcomas including synovial\nsarcomas (41) and both FGF8 and FGF18 bind to FGFR4.\nAlthough the status of polymorphism was not investigated in our\nsamples, FGF signals through FGFR4 may contribute to the\naggressiveness of synovial sarcoma.\nThe binding of FGF to FGFR leads to receptor dimerization\nand tyrosine autophosphorylation, followed by downstream\nactivation of PLCg, Crk, and SNT-1/FRS2 signaling pathways\namong which the SNT-1/FRS2 is the major pathway for\nmitogenic signals (25). Phosphorylated SNT-1/FRS2 directly\nbinds to the GRB2-SOS complex, and then membrane-\nassociated RAS recruits RAF-1, which in turn activates MAPKs\n(42). MAPKs are composed of three well-characterized sub-\nfamilies, ERK, p38 MAPK, and JNK (25), and we found that the\ngrowth-promoting signal of FGF was transmitted mainly\nthrough ERK1/2. Although ERK1/2 are activated by signals\nthrough other receptor tyrosine kinases (43), phosphorylation\nof ERK1/2 was completely inhibited by FGFR inhibitors, which\nindicated that the major signal to activate ERK1/2 in synovial\nsarcoma is FGF signaling, suggesting the important role of FGF\nin synovial sarcoma.\nWe have no clear explanation for the heterogeneous response\nin synovial sarcoma cell lines to FGF2, which stimulated the\ngrowth of HS-SY-II, but inhibited the growth of YaFuSS. In\ncontrast with many studies describing the role of FGF2 as a\nmitogen, several reports showed that FGF2 acted as a growth\ninhibitor of tumors of Ewing’s sarcoma family, which are\nsupposed to be of neuroectodermal origin (44, 45). Multiple\nFGF signals are involved in the biological process in synovial\nsarcoma, and the response may depend on the cell specificity.\nNevertheless, the disruption of FGF signals was found to cause\ngrowth inhibition in all synovial sarcoma cell lines, suggesting\nthat an inhibitory molecule for FGFR will be a promising tool\nfor molecular target therapy in synovial sarcoma.\nMolecular therapy targeted to FGFR has been investigated\nwith hematologic malignancies carrying a translocation involv-\ning the FGFR gene such as chronic myeloid leukemia with the\nBCR-FGFR1 gene (46) and multiple myeloma with the MMSET-\nFGFR3 fusion gene (47). In both instances, SU5402 was found\nto effectively inhibit the growth of cell lines with an IC50\nequivalent to that in synovial sarcoma. For experiments in vivo,\nwe used PD166866 instead of SU5402 because of its lower IC50\nvalue and found that i.p. given PD166866 efficiently inhibited\ntumor causing an\nthe phosphorylation of ERK1/2 in s.c.\ninhibition of growth without significant side effects. As in the\ncase of other tyrosine kinase inhibitors, PD166866 did not\nstrongly induce apoptosis or necrosis, and therefore no\ncomplete tumor remission was observed. However, cell cycle\narrest caused by the treatment with PD166866 may enhance\nthe cytotoxic effects of anticancer drugs as previously shown in\nsome instances (48, 49), and therefore further in vivo studies\nusing the treatment combined with suitable anticancer drugs\nmay provide a promising approach for clinical application.\nReferences\n1.Weiss SW, Goldblum JR. Enzinger andWeiss’s soft tis-\n5. Turc-Carel C, Dal Cin P, Limon J, et al. Involvement of\nchromosome X in primary cytogenetic change in hu-\nman neoplasia: nonrandom translocation in synovial\nsarcoma. Proc Natl Acad Sci U S A 1987;84:1981 ^ 5.\n6. Clark J, Rocques PJ, Crew AJ, et al. Identifi-\ninvolved in\ncation of novel genes, SYT and SSX,\nthe t(X;18)(p11.2;q11.2) translocation found in human\nsynovial sarcoma. Nat Genet 1994;7:502 ^ 8.\n15. Tanaka A, Miyamoto K, Matsuo H, Matsumoto K,\nYoshida H. Human androgen-induced growth factor in\nprostate and breast cancer cells: its molecular cloning\nand growth properties. FEBS Lett 1995;363:226 ^ 30.\n16. Dorkin TJ, Robinson MC, March C, Bjartell A,\nNeal DE, Leung HY. FGF8 over-expression in pros-\ntate cancer is associated with decreased patient\nsurvival and persists in androgen independent dis-\nease. Oncogene 1999;18:2755 ^ 61.\n27. TakahashiJ, Mori H, Fukumoto M, et al. Gene expres-\nsion of fibroblast growth factors in human gliomas and\nmeningiomas: demonstration of cellular source of basic\nfibroblast growth factor mRNA and peptide in tumor\ntissues. Proc Natl Acad Sci US A1990;87:5710 ^ 4.\n28. Tanaka A, Miyamoto K, Minamino N, et al. Cloning\nand characterization of an androgen-induced growth\nfactor essential for the androgen-dependent growth\nof mouse mammary carcinoma cells. Proc Natl Acad\nSci U S A 1992;89:8928 ^ 32.\n32. Miyamoto M, Naruo K, Seko C, Matsumoto S,\nKondo T, Kurokawa T. Molecular cloning of a novel\ncytokine cDNA encoding the ninth member of the\nfibroblast growth factor family, which has a unique\nsecretion property. Mol Cell Biol 1993;13:4251 ^ 9.\n33. Smallwood PM, Munoz-Sanjuan I,Tong P, et al. Fi-\nbroblast growth factor (FGF) homologous factors:\nnew members of the FGF family implicated in nervous",
         "Materials and Methods\nTumor tissues were obtained at either\nbiopsy or resection surgery and kept at 80jC. Informed consent was\nobtained from each patient, and tumor samples were approved for\nanalysis by the Ethics Committee of the Faculty of Medicine, Kyoto\nUniversity. Five human synovial sarcoma cell lines (YaFuSS, HS-SY-II,\nSYO-1, Fuji, and 1273/99) were used in this study. YaFuSS and HS-SY-II\ncells have the SYT-SSX1 fusion gene and the others have the SYT-SSX2\nfusion gene (data not shown). YaFuSS was established in our laboratory\nfrom a monophasic synovial sarcoma in a 28-year-old male. HS-SY-II\nwas a gift from H. Sonobe (Kochi University, Japan; ref. 18), SYO-1\nfrom A. Kawai (Okayama University, Japan; ref. 19), Fuji from S.\nTanaka (Hokkaido University, Japan; ref. 20), and 1273/99 from\nO. Larsson (Karolinska Institute, Sweden). Among control cell lines,\nNMS-2 (malignant peripheral nerve sheath tumor; ref. 21) was\nprovided by A. Ogose (Niigata University,\nJapan), and Saos2\n(osteosarcoma), HT1080 (fibrosarcoma), COLO205 (colon cancer),\nand SW480 (colon cancer) cells were purchased from American Type\nCulture Collection (Manassas, VA). Cells were maintained in RPMI\n1640 (Sigma-Aldrich, St. Louis, MI) for Fuji, SW480, and COLO205\nwith 10% fetal bovine serum (FBS, HyClone, Road Logan, UT) or\nDMEM (Sigma, St. Louis, MO) for other cells with 10% FBS. OPTI-\nMEM I, which contains insulin and transferrin as protein supplements\n(Invitrogen, Carlsberg, CA), was used in serum-free cultures.\nThe FGFR-specific tyrosine kinase\ninhibitor SU5402 was purchased from Calbiochem (La Jolla, CA;\nref. 22). PD166866 was provided by Pfizer Global Research and\nDevelopment (Groton, CT; ref. 23). The MAPK/extracellular signal –\nregulated kinase (ERK) kinase (MEK) inhibitor U0126 was purchased\nfrom Promega (Madison, WI; ref. 24). Recombinant human FGF18\n(rhFGF18) was purchased from Wako Pure Chemical Industry (Osaka,\nJapan),\nrhFGF8 from PeproTech (London, United Kingdom),\nand rhFGF2 from Oncogene Research Products (San Diego, CA).\nAnti-ERK1 (M12320), anti-phosphorylated-p38 (P39520), anti-pan-\np38 (P19820), anti-phosphorylated-JNK1/SAPK1 (S37220), and anti-\npan-JNK1/SAPK1 (M54920) antibodies were purchased from\nBD Biosciences PharMingen (San Diego, CA); anti-phosphorylated-\nERK1/2 (sc-7383), anti-FGFR3 (sc-123), and peroxidase-conjugated\nanti-mouse IgG (sc-2005) from Santa Cruz Biotechnology (Santa\nCruz, CA); anti-phosphotyrosine (PY20) from Zymed Laboratories\n(South San Francisco, CA); anti-FGF18 (MAB667) from R&D Systems\n(Minneapolis, MN); peroxidase-conjugated anti-rabbitimmunoglobu-\nlin (P0399) from DakoCytomation (Glostrup, Denmark).\nTotal RNA was extracted using TRIzol\nreagent (Invitrogen) following the manufacturer’s instructions. After\ntreatment with DNase I (Nippon Gene, Osaka, Japan), 1 Ag of total\nRNA was reverse transcribed for single-stranded cDNAs using oligo(dT)\nprimer and Superscript II reverse transcriptase (Invitrogen). PCR was\ndone using 1 AL of RT product in a final volume of 25 AL containing\n20 pmol each of the sense and antisense primers, 2.5 mmol/L MgCl2,\n0.2 mmol/L of each deoxynucleotide triphosphate, and 1 unit of rTaq\npolymerase (TOYOBO, Osaka, Japan). All PCRs were done using\nGeneAmp 9700 (PE Applied Biosystems, Foster City, CA). Information\nof the primers are available upon request.\nQuantitative reverse transcription-PCR (RT-PCR) analyses were done\nin selected FGF and FGFR genes with ABI PRISM 7700 Sequence\nDetection System (PE Applied Biosystems). One microliter of RT\nproduct in a final volume of 25 AL containing 12.5 AL of 2 SYBR\nGreen Mastermix (PE Applied Biosystems). Information for primers are\navailable upon requests. The mean of triplicated data was used to\ncalculate the ratio of target gene/b-actin expression. Statistical analysis\nwas done by t test after logarithmic transformation.\nTo detect FGF18 protein in the supernatant,\ncells were cultured up to 80% confluency in 100-mm dish. After\nwashing the dish, cells were further incubated in 5 mL OPTI-MEM I for\n4 days. The supernatant was harvested, mixed with 8 mL ice-cold\nacetone, and kept at 80jC for 1 hour. The mixture was centrifuged at\n10,000  g for 15 minutes, and the precipitate was suspended by lysis\nbuffer (100 AL) containing aprotinin (1 Ag/mL), leupeptin (1 Ag/mL),\npepstatin A (1 Ag/mL), and phenylmethylsulfonyl fluoride (1 mmol/L)\nfollowed by sonication and centrifugation. Twenty microliters of the\nsupernatant were electrophoresed on a 10% SDS-polyacrylamide gel\nand transferred onto a polyvinylidene difluoride membrane (Millipore,\nBedford, MA). After blocking with 3% skim milk, membranes were\nprobed with anti-FGF18 antibody at 1:1,000 dilutions for 1 hour and\nwith peroxidase-conjugated anti-mouse IgG 1:2,000 for 1 hour.\nImmunoreactive bands were detected with Enhanced Chemilumines-\ncence Plus (Amersham Biosciences, Little Chalfont Buckinghamshire,\nUnited Kingdom).\nTo detect FGFR3 protein in cell lysate, cells were harvested by the\nsame lysis buffer mentioned above. The lysate (40 Ag) was electro-\nphoresed using 8% SDS-polyacrylamide gel, blotted, and blocked by\n5% skim milk. The membrane was probed with anti-FGFR3 antibody at\n1:500 dilutions and with peroxidase-conjugated anti-rabbit immuno-\nglobulin at 1:2,000.\nTo evaluate the phosphorylation status\nof FGFR3, serum-starved cells were treated by 20 Amol/L SU5402 for\n30 minutes followed by the treatment with rhFGF18 or vehicle. The\ncells were harvested by 1 mL modified radioimmunoprecipitation assay\nbuffer containing protease inhibitors and sodium orthovanadate,\nsonicated, and centrifuged at 15,000  g for 10 minutes. After\npreclearing by 100 AL protein A agarose (Upstate, Lake Placid, NY),\n1 mg of the protein was added with 4 Ag anti-FGFR3 antibody and\nrotated for 2 hours at 4jC, and the immunocomplex was mixed with\n100 AL protein A agarose and rotated for another 2 hours. The mixture\nwas washed twice by PBS, and the precipitate was boiled for 5 minutes\nwith 60 AL of 2 sample buffer, centrifuged, and the supernatant was\nused for Western blotting using anti-FGFR3 antibody (1:500) or anti-\nphosphotyrosine (1:500).\nTo evaluate the phosphorylation of MAPKs, cells (4  105) were\nseeded on 60-mm dishes and incubated overnight with medium\ncontaining 10% FBS followed by serum starvation for 24 hours. After\ntreatment with each reagent for 15 minutes in the serum-free medium,\ncells were lysed in a buffer (100 AL) containing sodium orthovanadate\n(5 mmol/L) and protease inhibitors, sonicated, and centrifuged.\nProteins (15 Ag) were used for Western blotting.\nCell cycle analysis. Cells (1  105) were incubated overnight in\n60-mm dishes\nfollowed by serum\nstarvation for 24 hours. After treatment with SU5402 (20 Amol/L)\nfor 30 minutes, FBS (1%) was added to the medium and cells were\nfurther incubated for 48 hours. Cells were then harvested, washed\nonce with PBS, and fixed in 70% ethanol overnight. After an\nincubation in PBS containing propidium iodide (10 Ag/mL) and\nRNaseA (10 Ag/mL) for 30 minutes at 37jC, the DNA content of\nthese cells was analyzed using FACScan and CELL Quest Software\nIn vivo growth assay. All animal studies were approved by the\nAnimal Research Committee, Graduate School of Medicine, Kyoto\nUniversity and done according to the Guideline for Animal Experi-\nments of Kyoto University. First,\nthe effect of PD166866 on\nphosphorylated ERK1/2 in vivo was investigated. SYO-1 (5  106)\ncells suspended in 100 AL of PBS were s.c. injected into the hind\nflank region of male BALB/c nu/nu athymic mice at 5 weeks of age\n(Japan SLC, Hamamatsu,\nJapan). PD166866 (0.1 or 0.5 mg)\nsuspended in 50 AL of DMSO was given i.p. when the tumor\nvolume was f1,500 mm3. Tumors were dissected 30 minutes after\nthe injection, and tissue blocks (5  5  5 mm) were prepared from\nthe central and peripheral portion of tumors. Whole cell lysates were\nprepared from each tissue block and used for Western blot. Growth\ninhibition experiments were done using SYO-1 and HT1080. Cells\n(5  106) were inoculated as described above. When the tumor\nvolume reached about 50 mm3 (usually 11-12 days after inocula-\ntion), PD166866 was given i.p., a treatment which was repeated\nthereafter once a day, 6 days a week, for 2 weeks. Tumor size was\nmeasured with vernier calipers, and the volume was calculated as k/\n6  length  width  height.\nImmunohistochemical\nstaining of proliferating cell nuclear antigen (PCNA) was done using\nEPOS anti-PCNA/horseradish peroxidase system (DakoCytomation).\nOn day 13 (the day of the last administration), tumor tissues were\ndissected, fixed in 10% formalin, and embedded in paraffin. Sections\n(5-Am-thick) were probed with anti-PCNA antibody for 1 hour and\ncounterstained with hematoxylin. To quantify the PCNA-positive area,",
         "Abstract Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which\nare unknown.We investigated the involvement of fibroblast growth factor (FGF) signals in synovial\nsarcoma and evaluated the therapeutic effect of inhibiting the FGF signal.\nExperimental Design:The expression of 22 FGF and 4 FGF receptor (FGFR) genes in18 prima-\nry tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR.\nEffects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein\nkinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory\neffects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo.\nResults: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed\nin neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma\ncell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal ^\nregulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the\nFGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle ar-\nrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the\nFGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not\np38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sar-\ncoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2.\nConclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibi-\ntory molecules will be of potential use for molecular target therapy in synovial sarcoma.\nSynovial sarcoma is the most frequent soft-tissue sarcoma\n(STS) among patients in the third to fourth decade of life (1)\nand accounts for about 7% to 10% of all human STSs (2). It\npredominantly affects the lower extremities but can occur in\nany part of the body. Surgical resection with an adequate\nfor\nsurgical margin is the definitive choice of\nprimary tumors and has been shown to control\nlocal\nrecurrence (3, 4). Disseminated distant metastasis is the major\ncause of poor outcome, and several reports describing the\nresults of current therapy showed a 5-year survival rate of\naround 50% to 60% (3, 4). The efficacy of adjuvant\nchemotherapy is still a matter of debate, and the development\nof a new therapeutic approach is required to improve the\nprognosis.\nthe reciprocal\nDespite little progress in clinical treatment during the last\n20 years, cytogenetic and molecular genetic analyses have greatly\nimproved the understanding of this type of tumor, especially\nwith the discovery of\ntranslocation\nt(18;X)(q11;p11) creating an SYT-SSX fusion gene as a synovial\nsarcoma – specific genetic alteration (5, 6). Thus far, three SSX\ngenes (SSX1, SSX2, and SSX4) have been identified as a partner\nof the SYT gene, and >95% of synovial sarcoma tumors carried\none of these fusion genes (7). Although the precise function and\nthe mechanism of oncogenesis are not yet clearly shown, the\nhigh sensitivity and specificity of the SYT-SSX fusion gene in\nsynovial sarcoma have proven to be useful for molecular\ndiagnosis (8). In particular types of tumors with a specific\nreciprocal translocation such as PML-RARa in acute promyelo-\ncytic leukemia (9) and BCR-ABL in chronic myelogenous\nleukemia (10), fusion gene products themselves serve as targets\nfor the therapy. Although immunotherapy using a peptide\nderived from SYT-SSX protein as a specific vaccine has been\ninvestigated (11), no therapeutic approach has been discovered\nfor directly targeting the fusion protein or its function.\nGene expression profiling of tumors has been shown a\npowerful tool with which to isolate a molecular target for\ntherapy (12). We have done a gene expression analysis of\nsynovial sarcoma using a genome-wide cDNA microarray and\nfound that synovial sarcoma shared its molecular signature with\nmalignant peripheral nerve sheath tumor, of which the\nprecursors were neural crest – derived cells, and also identified\na set of genes commonly up-regulated in synovial sarcoma\nincluding the fibroblast growth factor 18 (FGF18) gene (13). The\nFGF signaling pathway seems to play significant roles in tumor\ndevelopment and progression (14 – 16), and recently FGF18 was\nidentified as an autocrine growth factor involved in colon\ncancers (17). Thus far, 22 genes have been identified as members\nof the FGF family, and our cDNA microarray contained 10 FGF\ngenes (FGF2, FGF3, FGF4, FGF7, FGF9, FGF11, FGF12, FGF13,\nFGF18, and FGF19). We found that some FGF genes other\nthan FGF18 were also expressed in synovial sarcoma (data\nshown). Based on these results, we have done an\nnot\nintensive analysis of FGF and its receptor (FGFR) genes in\nsynovial sarcoma and also investigated whether inhibition of\nthe FGF signal is a new therapeutic modality for synovial\nsarcoma.\nMaterials and Methods\nTumor tissues were obtained at either\nbiopsy or resection surgery and kept at 80jC. Informed consent was\nobtained from each patient, and tumor samples were approved for\nanalysis by the Ethics Committee of the Faculty of Medicine, Kyoto\nUniversity. Five human synovial sarcoma cell lines (YaFuSS, HS-SY-II,\nSYO-1, Fuji, and 1273/99) were used in this study. YaFuSS and HS-SY-II\ncells have the SYT-SSX1 fusion gene and the others have the SYT-SSX2\nfusion gene (data not shown). YaFuSS was established in our laboratory\nfrom a monophasic synovial sarcoma in a 28-year-old male. HS-SY-II\nwas a gift from H. Sonobe (Kochi University, Japan; ref. 18), SYO-1\nfrom A. Kawai (Okayama University, Japan; ref. 19), Fuji from S.\nTanaka (Hokkaido University, Japan; ref. 20), and 1273/99 from\nO. Larsson (Karolinska Institute, Sweden). Among control cell lines,\nNMS-2 (malignant peripheral nerve sheath tumor; ref. 21) was\nprovided by A. Ogose (Niigata University,\nJapan), and Saos2\n(osteosarcoma), HT1080 (fibrosarcoma), COLO205 (colon cancer),\nand SW480 (colon cancer) cells were purchased from American Type\nCulture Collection (Manassas, VA). Cells were maintained in RPMI\n1640 (Sigma-Aldrich, St. Louis, MI) for Fuji, SW480, and COLO205\nwith 10% fetal bovine serum (FBS, HyClone, Road Logan, UT) or\nDMEM (Sigma, St. Louis, MO) for other cells with 10% FBS. OPTI-\nMEM I, which contains insulin and transferrin as protein supplements\n(Invitrogen, Carlsberg, CA), was used in serum-free cultures.\nThe FGFR-specific tyrosine kinase\ninhibitor SU5402 was purchased from Calbiochem (La Jolla, CA;\nref. 22). PD166866 was provided by Pfizer Global Research and\nDevelopment (Groton, CT; ref. 23). The MAPK/extracellular signal –\nregulated kinase (ERK) kinase (MEK) inhibitor U0126 was purchased\nfrom Promega (Madison, WI; ref. 24). Recombinant human FGF18\n(rhFGF18) was purchased from Wako Pure Chemical Industry (Osaka,\nJapan),\nrhFGF8 from PeproTech (London, United Kingdom),\nand rhFGF2 from Oncogene Research Products (San Diego, CA).\nAnti-ERK1 (M12320), anti-phosphorylated-p38 (P39520), anti-pan-\np38 (P19820), anti-phosphorylated-JNK1/SAPK1 (S37220), and anti-\npan-JNK1/SAPK1 (M54920) antibodies were purchased from\nBD Biosciences PharMingen (San Diego, CA); anti-phosphorylated-\nERK1/2 (sc-7383), anti-FGFR3 (sc-123), and peroxidase-conjugated\nanti-mouse IgG (sc-2005) from Santa Cruz Biotechnology (Santa\nCruz, CA); anti-phosphotyrosine (PY20) from Zymed Laboratories\n(South San Francisco, CA); anti-FGF18 (MAB667) from R&D Systems\n(Minneapolis, MN); peroxidase-conjugated anti-rabbitimmunoglobu-\nlin (P0399) from DakoCytomation (Glostrup, Denmark).\nTotal RNA was extracted using TRIzol\nreagent (Invitrogen) following the manufacturer’s instructions. After\ntreatment with DNase I (Nippon Gene, Osaka, Japan), 1 Ag of total\nRNA was reverse transcribed for single-stranded cDNAs using oligo(dT)\nprimer and Superscript II reverse transcriptase (Invitrogen). PCR was\ndone using 1 AL of RT product in a final volume of 25 AL containing\n20 pmol each of the sense and antisense primers, 2.5 mmol/L MgCl2,\n0.2 mmol/L of each deoxynucleotide triphosphate, and 1 unit of rTaq\npolymerase (TOYOBO, Osaka, Japan). All PCRs were done using\nGeneAmp 9700 (PE Applied Biosystems, Foster City, CA). Information\nof the primers are available upon request.\nQuantitative reverse transcription-PCR (RT-PCR) analyses were done\nin selected FGF and FGFR genes with ABI PRISM 7700 Sequence\nDetection System (PE Applied Biosystems). One microliter of RT\nproduct in a final volume of 25 AL containing 12.5 AL of 2 SYBR\nGreen Mastermix (PE Applied Biosystems). Information for primers are\navailable upon requests. The mean of triplicated data was used to\ncalculate the ratio of target gene/b-actin expression. Statistical analysis\nwas done by t test after logarithmic transformation.\nTo detect FGF18 protein in the supernatant,\ncells were cultured up to 80% confluency in 100-mm dish. After\nwashing the dish, cells were further incubated in 5 mL OPTI-MEM I for\n4 days. The supernatant was harvested, mixed with 8 mL ice-cold\nacetone, and kept at 80jC for 1 hour. The mixture was centrifuged at\n10,000  g for 15 minutes, and the precipitate was suspended by lysis\nbuffer (100 AL) containing aprotinin (1 Ag/mL), leupeptin (1 Ag/mL),\npepstatin A (1 Ag/mL), and phenylmethylsulfonyl fluoride (1 mmol/L)\nfollowed by sonication and centrifugation. Twenty microliters of the\nsupernatant were electrophoresed on a 10% SDS-polyacrylamide gel\nand transferred onto a polyvinylidene difluoride membrane (Millipore,\nBedford, MA). After blocking with 3% skim milk, membranes were\nprobed with anti-FGF18 antibody at 1:1,000 dilutions for 1 hour and\nwith peroxidase-conjugated anti-mouse IgG 1:2,000 for 1 hour.\nImmunoreactive bands were detected with Enhanced Chemilumines-\ncence Plus (Amersham Biosciences, Little Chalfont Buckinghamshire,\nUnited Kingdom).\nTo detect FGFR3 protein in cell lysate, cells were harvested by the\nsame lysis buffer mentioned above. The lysate (40 Ag) was electro-\nphoresed using 8% SDS-polyacrylamide gel, blotted, and blocked by\n5% skim milk. The membrane was probed with anti-FGFR3 antibody at\n1:500 dilutions and with peroxidase-conjugated anti-rabbit immuno-\nglobulin at 1:2,000.\nTo evaluate the phosphorylation status\nof FGFR3, serum-starved cells were treated by 20 Amol/L SU5402 for\n30 minutes followed by the treatment with rhFGF18 or vehicle. The\ncells were harvested by 1 mL modified radioimmunoprecipitation assay\nbuffer containing protease inhibitors and sodium orthovanadate,\nsonicated, and centrifuged at 15,000  g for 10 minutes. After\npreclearing by 100 AL protein A agarose (Upstate, Lake Placid, NY),\n1 mg of the protein was added with 4 Ag anti-FGFR3 antibody and\nrotated for 2 hours at 4jC, and the immunocomplex was mixed with\n100 AL protein A agarose and rotated for another 2 hours. The mixture\nwas washed twice by PBS, and the precipitate was boiled for 5 minutes\nwith 60 AL of 2 sample buffer, centrifuged, and the supernatant was\nused for Western blotting using anti-FGFR3 antibody (1:500) or anti-\nphosphotyrosine (1:500).\nTo evaluate the phosphorylation of MAPKs, cells (4  105) were\nseeded on 60-mm dishes and incubated overnight with medium\ncontaining 10% FBS followed by serum starvation for 24 hours. After\ntreatment with each reagent for 15 minutes in the serum-free medium,\ncells were lysed in a buffer (100 AL) containing sodium orthovanadate\n(5 mmol/L) and protease inhibitors, sonicated, and centrifuged.\nProteins (15 Ag) were used for Western blotting.\nCell cycle analysis. Cells (1  105) were incubated overnight in\n60-mm dishes\nfollowed by serum\nstarvation for 24 hours. After treatment with SU5402 (20 Amol/L)\nfor 30 minutes, FBS (1%) was added to the medium and cells were\nfurther incubated for 48 hours. Cells were then harvested, washed\nonce with PBS, and fixed in 70% ethanol overnight. After an\nincubation in PBS containing propidium iodide (10 Ag/mL) and\nRNaseA (10 Ag/mL) for 30 minutes at 37jC, the DNA content of\nthese cells was analyzed using FACScan and CELL Quest Software\nIn vivo growth assay. All animal studies were approved by the\nAnimal Research Committee, Graduate School of Medicine, Kyoto\nUniversity and done according to the Guideline for Animal Experi-\nments of Kyoto University. First,\nthe effect of PD166866 on\nphosphorylated ERK1/2 in vivo was investigated. SYO-1 (5  106)\ncells suspended in 100 AL of PBS were s.c. injected into the hind\nflank region of male BALB/c nu/nu athymic mice at 5 weeks of age\n(Japan SLC, Hamamatsu,\nJapan). PD166866 (0.1 or 0.5 mg)\nsuspended in 50 AL of DMSO was given i.p. when the tumor\nvolume was f1,500 mm3. Tumors were dissected 30 minutes after\nthe injection, and tissue blocks (5  5  5 mm) were prepared from\nthe central and peripheral portion of tumors. Whole cell lysates were\nprepared from each tissue block and used for Western blot. Growth\ninhibition experiments were done using SYO-1 and HT1080. Cells\n(5  106) were inoculated as described above. When the tumor\nvolume reached about 50 mm3 (usually 11-12 days after inocula-\ntion), PD166866 was given i.p., a treatment which was repeated\nthereafter once a day, 6 days a week, for 2 weeks. Tumor size was\nmeasured with vernier calipers, and the volume was calculated as k/\n6  length  width  height.\nImmunohistochemical\nstaining of proliferating cell nuclear antigen (PCNA) was done using\nEPOS anti-PCNA/horseradish peroxidase system (DakoCytomation).\nOn day 13 (the day of the last administration), tumor tissues were\ndissected, fixed in 10% formalin, and embedded in paraffin. Sections\n(5-Am-thick) were probed with anti-PCNA antibody for 1 hour and\ncounterstained with hematoxylin. To quantify the PCNA-positive area,\nResults\nSynovial sarcoma cell lines expressed multiple fibroblast growth\nfactor and fibroblast growth factor receptor genes. The mRNA\nexpression of 22 subtypes of FGF genes was analyzed by RT-\nPCR in five synovial sarcoma and five other cell lines (Fig. 1A).\nfive synovial sarcoma cell\nNone of the five synovial sarcoma cell lines expressed the\nFGF5, FGF6, FGF12, FGF14, FGF16, FGF17, FGF22, or FGF23\ngene, whereas the FGF2, FGF8, FGF9, FGF11, and FGF18 genes\nwere expressed in all\nlines.\nExpression of the remaining nine FGF genes varied among\ncell\nlines. The expression profile in NMS-2 (malignant\nperipheral nerve sheath tumor) was similar to that in synovial\nsarcoma, which may relate to the overall similarity of the gene\nexpression profile between synovial sarcoma and malignant\nperipheral nerve sheath tumor (13). HT1080 (fibrosarcoma)\nexpressed a similar set of FGF genes to malignant peripheral\nnerve sheath tumor but at much lower levels, and the\nexpression profile of Saos2 (osteosarcoma) was quite different\nbeing positive only for FGF1 and FGF13. Interestingly, synovial\nsarcoma shared with colon carcinoma cell lines the expression\nof some FGF genes such as FGF3 and FGF18 genes, which were\nnot expressed in mesenchymal cell lines. Secretion of these FGF\nproteins by synovial sarcoma cell lines was confirmed by the\nWestern blotting of culture supernatant. FGF18 proteins were\nclearly detected in culture supernatant of synovial sarcoma cell\nlines, although the level of protein was not completely agreed\nwith the level of RNA expression (Fig. 1C).\nIn contrast with the heterogeneous expression pattern of\nthe expression profile of receptors (FGFR) was\nligands,\nrelatively homogeneous among synovial sarcoma cell lines,\nbeing positive for all subtypes analyzed except FGFR2b, which\nis an epithelia-specific FGFR expressed in colon carcinomas\n(Fig. 1B). In addition to FGFR2b, NMS-2 was negative for\nFGFR3, and HT1080 was negative for FGFR2c. Expression of\nFGFR3 in synovial sarcoma cell lines was confirmed at the\nprotein level by the Western blotting using cell lysates of each\ncell line (Fig. 1D).\nExpression of fibroblast growth factor and fibroblast growth\nfactor receptor genes in tumor tissues.\nThe FGF2, FGF8,\nFGF9, FGF11, and FGF18 genes, which were commonly ex-\npressed in the five synovial sarcoma cell lines, were inves-\ntigated in tumor tissues of 18 synovial sarcomas and 11 STSs\nof other types (Fig. 2A). In accordance with the results in the\ncell lines, almost all 18 synovial sarcoma tumors expressed\nall of these FGF genes (Fig. 2A). Among five FGF genes, the\nFGF2, FGF9, and FGF11 genes were also expressed in some\ntumors other than synovial sarcoma. In contrast, expression of\nFGF8 and FGF18 genes was only weakly detected in other\ntumors. Expression profiles of these five FGF genes were further\nanalyzed by the quantitative RT-PCR (Fig. 2B). The level of\nexpression of FGF8 and FGF18 genes in synovial sarcoma\ntumors were significantly higher than in tumor tissues of other\ntype of tumors, whereas those of FGF2, FGF9, and FGF11 genes\nshowed no difference.\nAs for receptor genes, synovial sarcoma tumors expressed all\ntypes of FGFR genes analyzed including the FGFR2b gene\nGrowth stimulatory effects of\nfibroblast growth factors in\nsynovial sarcoma cell lines. Expression of FGF and FGFR genes\nand proteins suggested the involvement of FGFs as autocrine/\nparacrine growth factors in synovial sarcoma. Among FGF genes\nexpressed in synovial sarcoma both in vitro and in vivo, we\nfocused on the FGF2, FGF8, and FGF18 genes in subsequent\nexperiments. The FGF2 gene was selected as a gene commonly\nup-regulated in STS (Fig. 2A). FGF8 and FGF18 genes were\nselected as synovial sarcoma – specific up-regulated FGF genes\n(Fig. 2A).\nGrowth stimulatory effects of each rhFGF protein were\nserum-starved conditions. Recombinant\nanalyzed under\nhFGF8 showed a dose-dependent growth-promoting effect\nin all synovial sarcoma cell lines (Fig. 3A). On the treatment\nwith rhFGF18, only HS-SY-II\nshowed a dose-dependent\nincrease (Fig. 3B). A heterogeneous response to FGF was\nmore clearly observed in the case of rhFGF2, in which the\ngrowth of HS-SY-II\nincrease,\nwhereas a growth inhibitory effect was observed in the case\nof YaFuSS (Fig. 3C).\nshowed a dose-dependent\nActivation of mitogen-activated protein kinases by fibroblast\ngrowth factor signals. One of major signal pathways located\ndownstream of FGFR is the MAPKs, for which three subtypes are\nknown: ERK, p38 MAPKs, and c-Jun NH2-terminal kinases\n(JNK; ref. 25). To investigate the events downstream of activated\nFGFRs, the phosphorylation of these MAPKs was investigated. In\nall synovial sarcoma cell lines analyzed, ERK1/2 was phosphor-\nylated under serum-free conditions, suggesting the presence of\nendogenous stimulatory signals (Fig. 4A). The amount of\nphosphorylated ERK1/2 (pERK1/2) increased on treatment\nwith FGF8 in a dose-dependent manner in all four cell lines\nanalyzed, with the response of HS-SY-II being the highest\nlevels. A dose-dependent\nFibroblast growth factor receptor – specific inhibitors reduce the\nlines in accordance with\ngrowth of synovial sarcoma cell\ninactivation of fibroblast growth factor receptor. MAPKs in\nsynovial sarcoma are activated by intrinsic pathways, and\nexogenous FGFs can stimulate the growth of synovial sarcoma,\nsuggesting that inhibition of the intrinsic FGF signal pathway\nmay inhibit the growth of synovial sarcoma. SU5402 is known\nto inhibit FGFR autophosphorylation at an IC50 of 10 to\n20 Amol/L (22). To confirm the effect of SU5402 on synovial\nsarcoma cell lines, the phosphorylation status of FGFR3 was\nanalyzed by the immunoprecipitation and Western blotting in\nthree synovial sarcoma cell lines (YaFuSS, SYO-1, and 1273/\n99; Fig. 5A). Phosphorylated FGFR3 was detected in these cell\nlines with serum-starved condition, and the treatment with\nrhFGF18 protein increased the amount of phosphorylated\nFGFR3 (Fig. 5A), which was compatible with the autocrine/\nparacrine model of FGFs in synovial sarcoma cells. Treatment\nwith SU5402 effectively decreased the phosphorylation of\nFGFR3 into almost undetectable level (Fig. 5A), indicating that\nSU5402 can be used as an inhibitor of FGFR in synovial\nsarcoma cells. By the treatment with SU5402, the growth of all\nfive synovial sarcoma cells was inhibited in a dose-dependent\nmanner (Fig. 5B), and the IC50 value ranged from 8.5 to 17.2\nAmol/L (Table 1). The growth inhibitory effect of SU5402 for\ncell lines other than synovial sarcoma was not remarkable. The\nsame experiments were done under low-serum conditions (1%\nFBS, Fig. 5C), resulting in similar growth inhibition in all\nsynovial sarcoma cell lines. The IC50 was almost identical in\nthe three cell lines (YaFuSS, HS-SY-II, and SYO-1), whereas the\nlines (Fuji and 1273/99) required a higher\nother two cell\nconcentration of SU5402 (Table 1). Growth inhibitory effects\nof another FGFR-specific inhibitor, PD166866 (23), was also\ninvestigated under the low-serum conditions. All five synovial\nsarcoma cell lines had an IC50 value (0.08-4.7 Amol/L) much\nlower than those of other cell lines (>10 Amol/L, Table 1).\nSimilar to the data obtained using SU5402, IC50 values in\nYaFuSS, HS-SY-II, and SYO-1 were much lower than those in\nFuji and 1273/99.\n11.5 F 1.8\n15.1 F3.4\n8.5 F 2.5\n17.2 F 1.8\n12.8 F 6.1\n46.4 F 6.9\n65.4 F 2.6\n>80\n>80\n54.4 F 21.5\n11.4 F 3.6\n4.9 F 0.5\n8.6 F 1.6\n25.3 F 3.6\n29.9 F 16.6\n60.2 F 20.6\n63.7 F 27.8\n55.8 F 19.1\n>80\n78.1 F20.3\n0.08 F 0.04\n0.77 F 0.52\n0.18 F 0.11\n4.0 F 1.6\n2.7 F 0.4\n>10\n>10\n>10\n>10\n>10\nGrowth inhibition of fibroblast growth factor receptor inhibitor\nassociated with the reduction of phosphorylation of extracellular\nsignal – regulated kinase 1/2 but not of p38.\nTo investigate\nwhich signals may contribute to the growth of synovial\nsarcoma, the phosphorylation of ERK1/2 and p38 before and\nafter the treatment with SU5402 was investigated. In three\nsynovial sarcoma cell lines (YaFuSS, HS-SY-II, and SYO-1),\nwhich were relatively sensitive to the growth inhibitory effect of\nSU5402, phosphorylation of ERK1/2 under low-serum con-\nditions was completely inhibited by SU5402, whereas the\namount of phosphorylated p38 showed no change (Fig. 6A). A\nsmall amount of pERK1/2 was still observed after SU5402\ntreatment in 1273/99, whereas no significant decrease was\nobserved in Fuji, both of which showed no significant change in\nthe pp38 (Fig. 6A). In cell lines other than synovial sarcoma,\nSU5402 showed no significant changes in the amount of\npERK1/2 or pp38 (data not shown). These results suggested that\nthe signal to induce the phosphorylation of ERK1/2 in synovial\nsarcoma was sent mainly through the FGF and FGFR pathway\nand that although the FGF signal induced the phosphorylation\nof both ERK1/2 and p38, the growth stimulatory effects were\ntransmitted through ERK1/2. To further confirm that the growth\ninhibition of SU5402 was due to the reduction in pERK1/2, the\nfunction of MAPK/ERK kinase 1/2 in synovial sarcoma, which is\na kinase of ERK1/2, was inhibited by U0216. Phosphorylation\nof ERK1/2 was effectively inhibited in all five cell lines, in which\n1273/99 still had a small amount of pERK1/2 at the lower\nconcentration (Fig. 6B). BrdUrd uptake was inhibited in a dose-\ndependent manner in all five cell lines, in which 1273/99 was\nrelatively resistant at the lower concentration, consistent with\nthe status of pERK1/2 (Fig. 6C). These results further indicated\nthat the growth inhibition of SU5402 in synovial sarcoma was\nachieved through the inhibition of signals through ERK1/2.\nGrowth inhibition by fibroblast growth factor receptor inhibitor\nin synovial sarcoma is associated with cell cycle arrest. The\nfraction of each cell cycle phase was analyzed before and after\nthe treatment with SU5402 in YaFuSS cells. SU5402 signifi-\ncantly increased the G1 and subG1 fractions, and decreased the\nS fraction (Fig. 6D). These data suggested that the growth\ninhibition of synovial sarcoma by SU5402 was due to its\ninduction of G1 arrest as previously reported for other tyrosine\nkinase inhibitors (26).\nFibroblast growth factor receptor inhibitors reduced the\nFor in vivo study,\ngrowth of synovial sarcoma cells in vivo.\nwe used SYO-1 because of\nits consistency in developing\ntumor, and PD166866 was used because of its lower IC50\nvalue. To confirm its efficacy in vivo, tumor tissue was taken\n30 minutes after the injection of PD166866. We found that\n0.1 mg of PD166866 was enough to reduce the phosphory-\nlated ERK1/2 in the tumor (Fig. 7A). Based on this result,\nthe growth inhibitory effect of PD166866 (0.1 and 0.5 mg/\nbody) was analyzed in vivo. Although PD166866 was unable\nto stop the growth of the tumor completely, a significant\ninhibition of growth was observed during the period of\nadministration (P < 0.01), and the effect was retained even at\nday 21 at the higher dose (P = 0.02, Fig. 7B). No definite\nreduction of body weight or pathologic changes in vital\norgans such as liver and kidney was observed (data not\nshown). No growth inhibitory effect was observed when\nHT1080 was used instead of SYO-1 even at the higher dose\n(data not shown).\nHistologic examination of tumor tissues showed no signifi-\ncant morphologic changes or focal necrosis. The number of\nPCNA-positive cells, however, was clearly smaller in PD166866-\ntreated tumors (19.3 F 6.6%, Fig. 7D) than control tumors\n(53.0 F 2.0%, Fig. 7C), indicating that PD166866 reduced the\nnumber of cells in S phase. No such difference was observed\nwhen HT1080 was used instead of SYO-1 (data not shown).\nDiscussion\nFGFs may activate genetic programs which promote cell\ngrowth by at least one of three general mechanisms: first, as\nmitogens\nsecond, by\npromoting angiogenesis to supply a growing tumor; and\nthird, by inhibiting apoptosis and allowing tumor cells to\ncontinue to grow beyond normal constraints. We found that\nsynovial sarcoma cells expressed a number of FGF genes, of\nwhich FGF2 , FGF8 , FGF9 , FGF11 , and FGF18 were\ncommonly expressed. The involvement of FGF2 in malignant\ntumors as an autocrine growth factor and/or a paracrine\nangiogenic factor was first shown in brain tumors (27), and\nsubsequently in non – small cell carcinoma (14). FGF8 was\noriginally identified as an androgen-induced growth factor\nfound in conditioned medium of androgen-dependent mouse\nmammary carcinoma SC-3 cells and reported to have a\ngrowth stimulatory effect (28). The human FGF8 gene was\nexpressed in breast and prostate cancers (15, 16, 29) and\nassociated with anchorage-independent proliferation and\ninvasion (30, 31). FGF9 was originally purified from the\nconditioned medium of\nline NMC-G1 and\ndesignated as glia-activating factor due to its mitogenic\nactivity toward glial cells (32). FGF11 was one of four FGF\nhomology factors, which shared structural homology with\nfailed to activate FGFRs (33). Recently,\nclassic FGFs but\nFGF11 was identified as one of the gene products enriched in\nneuronal precursors during the development of the nervous\nsystem (34). FGF18 is cloned by its homology to FGF8 and\nFGF9 (35, 36). FGF18, in association with FGF8, is expressed\nat various times and places during embryogenesis, especially\nneurogenesis (33, 37). Mitotic activity was shown in colon\ncarcinomas,\nin which the expression of FGF18 was up-\nregulated by WNT signals (17). Overall, we found that\nsynovial sarcoma expressed several FGF genes, particularly\nthose expressed in neural tissues, which may further suggest\nthe neural origin of precursor cells of synovial sarcoma as we\npreviously proposed (13). Mitotic activity of exogeneous FGFs\nwas not remarkable in some cell lines (Fig. 3). The serum-\nfree medium used in this study contained insulin, which is\nknown to potentiate the effect of FGF, and the further\naddition of insulin showed no increase of the BrdUrd uptake\n(data not shown). It is possible that the extent of autocrine\nstimulation may already be saturating so that further FGF\nstimulation does not lead to additional growth.\nThe expression of FGFR1, FGFR2c, FGFR3, and FGFR4, to\nwhich FGF2 (1c, 3c and 4), FGF8 (2c, 3c, 4), and FGF18 (2c,\n3c, 4) preferentially bind (38, 39), was confirmed in both tumor\ntissues and cell lines. The expression of FGFR2b gene, which was\ndetected in tumor tissues but not in cell lines, is intriguing.\nFGFR2b is a receptor for keratinocyte growth factor (FGF7), and\nexpressed in epithelial cells (40). As expected, all biphasic\nsynovial sarcomas expressed the FGFR2b gene, whereas the\nexpression was low in most of the monophasic synovial\nsarcomas and other STSs. It is, however, intriguing that some\nmonophasic synovial sarcomas with no apparent epithelial\nstructures expressed the FGFR2b gene at a level comparable with\nthat in biphasic synovial sarcoma (Fig. 2D). All five synovial\nsarcoma cell lines used in this study including SYO-1, which was\nestablished from a biphasic synovial sarcoma, were negative for\nthe FGFR2b gene. These results suggested that synovial sarcoma\ntumor cells may have an intrinsic mechanism to express FGFR2b\nin vivo, which was somehow inhibited or disappeared in vitro.\nAlternatively, synovial sarcoma cells positive for FGFR2b may\nhave growth disadvantages, and therefore only negative cells\nwere established as cell\nlines. Recently, single nucleotide\npolymorphism in FGFR4 is reported to associate with poor\nprognosis in high-grade soft tissue sarcomas including synovial\nsarcomas (41) and both FGF8 and FGF18 bind to FGFR4.\nAlthough the status of polymorphism was not investigated in our\nsamples, FGF signals through FGFR4 may contribute to the\naggressiveness of synovial sarcoma.\nThe binding of FGF to FGFR leads to receptor dimerization\nand tyrosine autophosphorylation, followed by downstream\nactivation of PLCg, Crk, and SNT-1/FRS2 signaling pathways\namong which the SNT-1/FRS2 is the major pathway for\nmitogenic signals (25). Phosphorylated SNT-1/FRS2 directly\nbinds to the GRB2-SOS complex, and then membrane-\nassociated RAS recruits RAF-1, which in turn activates MAPKs\n(42). MAPKs are composed of three well-characterized sub-\nfamilies, ERK, p38 MAPK, and JNK (25), and we found that the\ngrowth-promoting signal of FGF was transmitted mainly\nthrough ERK1/2. Although ERK1/2 are activated by signals\nthrough other receptor tyrosine kinases (43), phosphorylation\nof ERK1/2 was completely inhibited by FGFR inhibitors, which\nindicated that the major signal to activate ERK1/2 in synovial\nsarcoma is FGF signaling, suggesting the important role of FGF\nin synovial sarcoma.\nWe have no clear explanation for the heterogeneous response\nin synovial sarcoma cell lines to FGF2, which stimulated the\ngrowth of HS-SY-II, but inhibited the growth of YaFuSS. In\ncontrast with many studies describing the role of FGF2 as a\nmitogen, several reports showed that FGF2 acted as a growth\ninhibitor of tumors of Ewing’s sarcoma family, which are\nsupposed to be of neuroectodermal origin (44, 45). Multiple\nFGF signals are involved in the biological process in synovial\nsarcoma, and the response may depend on the cell specificity.\nNevertheless, the disruption of FGF signals was found to cause\ngrowth inhibition in all synovial sarcoma cell lines, suggesting\nthat an inhibitory molecule for FGFR will be a promising tool\nfor molecular target therapy in synovial sarcoma.\nMolecular therapy targeted to FGFR has been investigated\nwith hematologic malignancies carrying a translocation involv-\ning the FGFR gene such as chronic myeloid leukemia with the\nBCR-FGFR1 gene (46) and multiple myeloma with the MMSET-\nFGFR3 fusion gene (47). In both instances, SU5402 was found\nto effectively inhibit the growth of cell lines with an IC50\nequivalent to that in synovial sarcoma. For experiments in vivo,\nwe used PD166866 instead of SU5402 because of its lower IC50\nvalue and found that i.p. given PD166866 efficiently inhibited\ntumor causing an\nthe phosphorylation of ERK1/2 in s.c.\ninhibition of growth without significant side effects. As in the\ncase of other tyrosine kinase inhibitors, PD166866 did not\nstrongly induce apoptosis or necrosis, and therefore no\ncomplete tumor remission was observed. However, cell cycle\narrest caused by the treatment with PD166866 may enhance\nthe cytotoxic effects of anticancer drugs as previously shown in\nsome instances (48, 49), and therefore further in vivo studies\nusing the treatment combined with suitable anticancer drugs\nmay provide a promising approach for clinical application.\nReferences\n1.Weiss SW, Goldblum JR. Enzinger andWeiss’s soft tis-\n5. Turc-Carel C, Dal Cin P, Limon J, et al. Involvement of\nchromosome X in primary cytogenetic change in hu-\nman neoplasia: nonrandom translocation in synovial\nsarcoma. Proc Natl Acad Sci U S A 1987;84:1981 ^ 5.\n6. Clark J, Rocques PJ, Crew AJ, et al. Identifi-\ninvolved in\ncation of novel genes, SYT and SSX,\nthe t(X;18)(p11.2;q11.2) translocation found in human\nsynovial sarcoma. Nat Genet 1994;7:502 ^ 8.\n15. Tanaka A, Miyamoto K, Matsuo H, Matsumoto K,\nYoshida H. Human androgen-induced growth factor in\nprostate and breast cancer cells: its molecular cloning\nand growth properties. FEBS Lett 1995;363:226 ^ 30.\n16. Dorkin TJ, Robinson MC, March C, Bjartell A,\nNeal DE, Leung HY. FGF8 over-expression in pros-\ntate cancer is associated with decreased patient\nsurvival and persists in androgen independent dis-\nease. Oncogene 1999;18:2755 ^ 61.\n27. TakahashiJ, Mori H, Fukumoto M, et al. Gene expres-\nsion of fibroblast growth factors in human gliomas and\nmeningiomas: demonstration of cellular source of basic\nfibroblast growth factor mRNA and peptide in tumor\ntissues. Proc Natl Acad Sci US A1990;87:5710 ^ 4.\n28. Tanaka A, Miyamoto K, Minamino N, et al. Cloning\nand characterization of an androgen-induced growth\nfactor essential for the androgen-dependent growth\nof mouse mammary carcinoma cells. Proc Natl Acad\nSci U S A 1992;89:8928 ^ 32.\n32. Miyamoto M, Naruo K, Seko C, Matsumoto S,\nKondo T, Kurokawa T. Molecular cloning of a novel\ncytokine cDNA encoding the ninth member of the\nfibroblast growth factor family, which has a unique\nsecretion property. Mol Cell Biol 1993;13:4251 ^ 9.\n33. Smallwood PM, Munoz-Sanjuan I,Tong P, et al. Fi-\nbroblast growth factor (FGF) homologous factors:\nnew members of the FGF family implicated in nervous",
         "T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. Nishijo,K. Shibata,Y. Shima,S. Nagayama,T. Katagiri,Yusuke Nakamura,Takashi Nakamura,J. Toguchida",
         null
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "Replica shufﬂed versions of interactive decoders of turbo codes, low-density parity-check codes\nand turbo product codes are presented. The proposed schemes converge faster than standard and\npreviously proposed shufﬂed approaches. Simulations show that the new schedules offer good\nperformance versus complexity/latency trade-offs.",
         "This work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in whole or in part\nwithout payment of fee is granted for nonproﬁt educational and research purposes provided that all such whole or partial copies include\nthe following: a notice that such copying is by permission of Mitsubishi Electric Research Laboratories, Inc.; an acknowledgment of\nthe authors and individual contributions to the work; and all applicable portions of the copyright notice. Copying, reproduction, or\nrepublishing for any other purpose shall require a license with payment of fee to Mitsubishi Electric Research Laboratories, Inc. All\nrights reserved.\nMITSUBISHI ELECTRIC RESEARCH LABORATORIES\nhttp://www.merl.com\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang, Marc Fossorier, and Jonathan S. Yedidia\nAbstract\nReplica shufﬂed versions of iterative decoders of turbo codes, low-density parity-check\ncodes and turbo product codes are presented. The proposed schemes converge faster\nthan standard and previously proposed “shufﬂed” approaches. Simulations show that\nthe new schedules offer good performance versus complexity/latency trade-offs.\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in\nwhole or in part without payment of fee is granted for nonproﬁt educational and research purposes provided that all such\nwhole or partial copies include the following: a notice that such copying is by permission of Mitsubishi Electric Research\nLaboratories, Inc.; an acknowledgment of the authors and individual contributions to the work; and all applicable portions\nof the copyright notice. Copying, reproduction, or republishing for any other purpose shall require a license with payment\nof fee to Mitsubishi Electric Research Laboratories, Inc. All rights reserved.\nTo appear in the Proceedings of the 2005 International Symposium on Information Theory.\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang and Marc Fossorier\nDepartment of Electrical Engineering\nUniversity of Hawaii at Manoa\nHonolulu, HI 96822\nEmail: juntan, yige, marc@spectra.eng.hawaii.edu\nJonathan S. Yedidia\nMitsubishi Electric Research Laboratories\n201 Broadway, Cambridge, MA 02139\nEmail: yedidia@merl.com\nI. INTRODUCTION\nIterative decoding based on belief propagation (BP) [1] has\nreceived signiﬁcant attention recently, mostly due to its near-\nShannon-limit error performance for the decoding of low-\ndensity parity-check (LDPC) codes [2] and turbo codes [3].\nLike maximum a posterior probability (MAP) decoding [4],\nit is a symbol-by-symbol soft-in/soft-out decoding algorithm.\nIt processes the received symbols recursively to improve the\nreliability of each symbol based on constraints that specify\nthe code. In the ﬁrst iteration, the decoder only uses the\nchannel output, and generates soft output for each symbol.\nSubsequently, the output reliability measures of the decoded\nsymbols at the end of each decoding iteration are used as\ninputs for the next iteration. The decoding iteration process\ncontinues until a certain stopping condition is satisﬁed. Then\nhard decisions are made based on the output reliability mea-\nsures of the decoded symbols at the last decoding iteration.\nThe standard BP decoder of LDPC codes often needs several\ntens or hundreds of iterations for the decoding process to\nconverge, which is not desirable because of high decoding\ndelay. Furthermore, LDPC codes of interest can have large\nthe\ncodeword length and it can be difﬁcult\ndecoding in hardware in a fully parallel way. Because of\nthe serial property of the BCJR algorithm, decoders of turbo\ncodes cause high delay as well. In [5], [6] and [7], “shufﬂed”\nmethods were presented to reduce the required number of\niterations for decoding LDPC and turbo codes, respectively.\nA speedup factor of 2 for LDPC codes and the saving of\none iteration for turbo codes were reported. The aim of this\npaper is to introduce a “replica shufﬂed” scheme which further\naccelerate the decoding process for turbo codes, LDPC codes,\nturbo product codes and other iterative decodable codes.\nto implement\nII. ITERATIVE DECODING OF TURBO CODE\nA turbo code [3] encoder is constructed using a concatena-\ntion of two (or more) convolutional encoders, and its decoder\nconsists of two (or more) soft-in/soft-out convolutional de-\ncoders which feed reliability information to each other. For\nis the output code block at\nsimplicity, we consider a turbo code that consists of two rate-\n1/n systematic convolutional codes with encoders in feedback\nform. Let u = (u1, u2, . . . , uK) be an information block of\nlength K and c = (c1, c2, . . . , cK) be the corresponding\ncoded sequence, where ck = (ck,1, ck,2, . . . , ck,n), for k =\n1, 2, . . . , K,\ntime k. Suppose\nBPSK transmission over an AWGN channel, with uk and\nck,j all taking values in {+1, -1} for k = 1, 2, . . . , K and\nj = 1, 2, . . . , n. Let y = (y1, y2, . . . , yK) be the received\nsequence, where yk = (yk,1, yk,2, . . . , yk,n) is the received\nblock at time k. Let ˆu = {ˆu1, ˆu2, . . . , ˆuK} denote the estimate\nof u. Let sk denote the encoder state at time k. Following\n1 ), γk(s, s) = p(sk =\n[4], deﬁne: αk(s) = p(sk = s, yk\ns, yk|sk−1 = s), βk(s) = p(yK\na =\n(ya, ya+1, . . . , yb), and let α(m)\n(s) be\nk\nthe corresponding values computed in component decoder m,\nwith m = 1, 2. Let L(i)\nem(ˆuk) denote the extrinsic value of the\nestimated information bit ˆuk delivered by component decoder\nm at the ith iteration [8].\nk+1|sk = s), where yb\n(s), γ(m)\nA. Standard serial and parallel turbo decoding\nThe decoding approach proposed in [3] operates in serial\nmode, i.e., the component decoders take turns generating the\nextrinsic values of the estimated information symbols, and\neach component decoder uses the extrinsic messages delivered\nby the last component decoder as the a priori values of the\ninformation symbols. The disadvantage of this scheme is high\ndecoding delay. In the parallel turbo decoding algorithm [9],\nall component decoders operate in parallel at any given time.\nAfter each iteration, each component decoder delivers extrinsic\nmessages to other decoder(s) which use these messages as a\npriori values at the next iteration.\nB. Plain shufﬂed turbo decoding\nAlthough the parallel turbo decoding reduces the decoding\ndelay of serial decoding by half, the extrinsic messages are not\ntaken advantage of as soon as they become available, because\nthe extrinsic messages are delivered to component decoders\nonly after each iteration is completed. The aim of the shufﬂed\nturbo decoding is to use the more reliable extrinsic messages\nat each time. Let ˜u = (˜u1, ˜u2, . . . , ˜uK) be the sequence\npermuted by the interleaver corresponding to the original\ninformation sequence u = (u1, u2, . . . , uK), according to the\nmapping ˜uk = uπ(k), for k = 1, 2, . . . , K. We assume that k =\nπ(k), ∀k. There is a unique corresponding reverse mapping\nuk = ˜uπ−(k), for k = 1, 2, . . . , K and k = π−(k), ∀k. In\nshufﬂed turbo decoding, the two component decoders operate\nsimultaneously as in the parallel turbo decoding scheme, but\nthe messages are updated during each iteration based on π(k)\nand π−(k) [7]. Correspondingly it provides a faster decoding\nconvergence.\nC. Replica shufﬂed turbo decoding\n←−\nD 1 and\n−→\nD 1 and\n−→\nD 1, which are denoted by\nIn the plain shufﬂed turbo decoding summarized in Section\nthe component decoders process the\nII-B, we assume all\nbackward recursion followed by the forward recursion. Let\n−→\nus refer to the two component decoders as\nD 2.\nNaturally another possible scheme is to operate in the reverse\norder, i.e, all the component decoders process the forward\nrecursion followed by the backward recursion and we refer\n←−\nto them as\nD 2. In terms of error performance, there\nis no difference between these two approaches. However, the\nreliabilities of the extrinsic messages concerning a certain\ninformation bit delivered by these two shufﬂed turbo decoders\nare not the same. In general, the more independent information\nis used, the more reliable the delivered messages become.\nTherefore for the extrinsic messages delivered by component\n(i)\ndecoder\ne1 (ˆuk), the larger k\nis, the more reliable this message is. Following a similar\n←−\nanalysis, for the extrinsic message\nD 1,\nthe smaller k is, the more reliable this message is. It is natural\nto expect a faster decoding convergence if these two shufﬂed\nturbo decoders operate cooperatively instead of independently.\nBecause in this approach two sets of shufﬂed component\ndecoders are used to decode the same sequence of information\nbits, we refer to it as replica shufﬂed turbo decoding. In replica\nshufﬂed turbo decoding, two plain shufﬂed turbo decoders\n−→\n(processing recursions in opposite directions)\nD 2 and\n←−\n←−\nD 1 ,\nD 2 operate simultaneously and exchange more reliable\nextrinsic messages. We assume that the component decoders\n−→\n\n\ndeliver extrinsic messages synchronously, i.e.,\nk =\nk =\nT\n←−\n←−\n−→\n←−\n←−\n\n\n\n\n\nk)and\nk (\nT\nk, where the\nk) denote the times\nT\nT\nT\nk =\nT\n←−\n−→\n←−\n−→\nat which\nD 2) deliver the extrinsic values\nD 2 (\nD 1) and\nD 1 (\nof the k-th ((K + 1 − k)-th) estimated symbol of the original\ninformation sequence u and of the interleaved sequence ˜u,\nrespectively.\n(i)\ne1 (ˆuk) delivered by\n−→\nD 1 ,\n−→\nD 1. After time\nLet us ﬁrst consider the forward recursion stage at the i-\n−→\n\nk−1,\nT\nth iteration of component decoder\nthe values of −→α (1)\nk (s) should be updated and the values\nof −→γ (1)\nk (s) are needed. There are two possible cases. The\nﬁrst case is k > π−(k), which means the extrinsic value\n−→\n(i)\ne2 (ˆuk) of the information bit ˆuk has already been delivered\nL\n−→\nD 2. As in plain shufﬂed turbo decoding, this newly\nby decoder\n−→\n(i)\ne2 (ˆuk) is used to compute the values −→γ (1)\nL\navailable\nk (s),\n−→\n(i)\n−→α (1)\ne1 (ˆuk). The second case is k < π−(k), which\nL\nk (s), and\n(i)\ne2 (ˆuk) of the information bit ˆuk\nmeans the extrinsic value\n−→\nD 2. Then in plain shufﬂed\nhas not been delivered yet by\nturbo decoding, the values α(1)\ne1 (ˆuk) are updated\nk (s) and L(i)\n−→\nL\nk (s), and\nbased on the extrinsic messages delivered at last iteration.\nIn replica shufﬂed turbo decoding, however, there are two\nfurther subcases. The ﬁrst subcase is K + 1 − k < π−(k),\n(i)\ne2 (ˆuk) of the information\nwhich means the extrinsic value\n←−\nbit ˆuk has already been delivered by decoder\nD 2. Then this\n(i)\n(i−1)\ne2 (ˆuk), instead of\n(ˆuk) is used to\nnewly available\ne2\n−→\n(i)\ncompute the values −→γ (1)\nk (s), −→α (1)\ne1 (ˆuk). The\nL\nsecond subcase is K + 1 − k < π−(k), which means both\n(i)\nextrinsic messages of the information bit ˆuk, i.e,\ne2 (ˆuk)\n−→\n(i)\ne2 (ˆuk) are not available yet. In this subcase, the values\nL\nand\n(i)\nof −→α (1)\ne1 (ˆuk) are updated based on the extrinsic\nk (s) and\nmessages delivered at (i − 1)-th iteration. The recursions of\n←−\n←−\ncomponent decoders\nD 2 are realized based on\nD 1 and\nthe same principle. After Imax iterations, the shufﬂed turbo\ndecoding algorithm outputs ˆu = (ˆu1, ˆu2, . . . , ˆuK) as the de-\n←−\n(i)\ncoded codeword, where ˆuk =sgn[(\ne1 (ˆuk))/2 +\nL\n−→\n(i)\ne2 (ˆuk))/2 + 4\nyk,1], which is different from\nL\n(\nN0\nthat in the standard turbo decoding [3] and plain shufﬂed\nturbo decoding. It is straightforward to generalize the replica\nshufﬂed turbo decoding to multiple turbo codes which consist\nof more than two component codes. Also group of bits can\nbe updated periodically only to reduce information exchanges\nbetween replicas. Based on the above descriptions with two\nreplicas, the total computational complexity of the replica\nshufﬂed turbo decoding for multiple turbo codes at each\ndecoding iteration is about twice that of the parallel turbo\ndecoding.\n(i)\ne2 (ˆuk) +\n(i)\ne1 (ˆuk) +\n−→\nL",
         "D. Simulation results\nIII. ITERATIVE DECODING OF LDPC CODES\nLet H = [Hmn] be the parity check matrix which deﬁnes\nan LDPC code. We denote the set of bits that participate in\ncheck m by N (m) = {n : Hmn = 1} and the set of checks\nin which bit n participates as M(n) = {m : Hmn = 1}.\nAssume a codeword w = (w1, w2, . . . , wN ) is transmitted\nover an AWGN channel with zero mean and variance N0/2\nusing BPSK signaling and let y = (y1, y2, . . . , yN ) be the\ncorresponding received sequence.\nA. Standard BP for iterative decoding of LDPC codes\nBased on [1], let Fn be the log-likelihood ratio (LLR) of\nbit n and initially set Fn = 4\nmn be the\nN0\nLLR of bit n which is sent from check node m to bit node n\nand sent from the bit node n to check node m, respectively.\nLet z(i)\nn denote the a posteriori LLR of bit n. The standard\nBP algorithm [1] is carried out as follows:\nyn. Let ε(i)\nInitialization Set i = 1, maximum number of iterations to\nIM ax. For each m,n, set z(0)\nmn = Fn.\n(i) Horizontal Step, for 1 ≤ n ≤ N and each m ∈\nM(n), process:\nτ (i)\nmn =\nn∈N (m)\\n\n1 + τ (i)\nmn\n1 − τ (i)\nmn\n(ii) Vertical Step, for 1 ≤ n ≤ N and each m ∈\nε(i)\nmn = log\n(2)\nz(i)\nmn = Fn +\nz(i)\nn = Fn +\n(i)\nStep 2 Hard decision and stopping criterion test:\nCreate ˆw(i) = [ ˆw(i)\nn ] such that ˆw(i)\nz(i)\nn > 0, and ˆw(i)\nn = 1 if z(i)\nn < 0.\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 3. Otherwise set\ni := i + 1 and go to Step 1.\nn = 0 if\n(ii)\nStep 3 Output ˆw(i) as the decoded codeword.\nB. Plain shufﬂed BP for iterative decoding of LDPC codes\nAt the i-th iteration of the standard BP algorithm, ﬁrst all\nvalues of the check-to-bit messages are updated by using the\nvalues of the bit-to-check messages obtained at the (i − 1)-\nth iteration, i.e., each ε(i)\n:\nn ∈ N (m)\\n}. Then, all values of the bit-to-check messages\nare updated by using the values of the check-to-bit messages\nnewly obtained at the i-th iteration, i.e., each z(i)\nmn is updated\nfrom {ε(i)\nmn is updated by using {z(i−1)\nmn\nmn : m ∈ M(n)\\m}.\nWe observe that certain values z(i)\nin (1) to compute the remaining values ε(i)\nmn could already be\ncomputed in (3) based on a partial computation of the values\nε(i)\nmn obtained from (2), and then could be used instead of\nz(i−1)\nmn. Hence Step\nmn\n1 of the shufﬂed BP algorithm is performed as: for 1 ≤ n ≤ N\nand each m ∈ M(n), process the horizontal step and vertical\nstep jointly, with (1) modiﬁed as:\n\nτ (i)\nmn =\nn∈N (m)\\n\nn>n\nSince this algorithm becomes totally serial, bits can be grouped\nto maintain a sufﬁcient level of parallelism with the same error\nperformance [7].\nC. Replica shufﬂed BP decoding for LDPC codes\nPlain shufﬂed BP decoding is a bit-based sequential ap-\nproach and the method described in Section III-B is based\non a natural increasing order, i.e, belief messages concerning\nbit nodes are updated according to order i = 1, 2, . . . , N .\nThe larger the value of i, the more independent informations\nare used to update the beliefs of bit note i and the more\nreliable these belief messages are. Therefore the reliability of\nbit nodes increases and the error rate decreases as i increases.\nFollowing a similar analysis, in plain shufﬂed BP decoding\nbased on a natural decreasing order, after each iteration, the\nreliability of bit nodes decreases as i increases. As in replica\nshufﬂed turbo decoding, in replica shufﬂed BP decoding, two\nreplica shufﬂed subdecoders based on different updating orders\noperate simultaneously and cooperatively. After each iteration,\neach subdecoder receives more reliable messages from and\nsends more reliable messages to another subdecoder. Based on\nthese more reliable messages, both replica subdecoders begin\n←−\nthe next iteration of decoding. Let\nD denote the replica\nsubdecoder with natural increasing and decreasing updating\norder, respectively. Let −→ε (i)\nmn be the variables\n←−\nassociated with\nD\nare deﬁned in a similar way. The replica shufﬂed BP decoding\nwith two replica subdecoders is carried out as followings:\n−→\nD at iteration i. Variables associated with\nmn and −→z i\n−→\nD and\nInitialization Set i = 1, maximum number of iteration to\nIM ax. For each m,n, set −→z (0)\nmn = ←−z (0)\nmn = Fn.\nStep 1\nEach replica subdecoder process as the following\ntwo steps simultaneously. For 0 ≤ n ≤ N − 1 and\neach m ∈ M(n), process\nHorizontal Step\nn∈N (m)\\n\nn>n\n1 + −→τ (i)\nmn\n1 − −→τ (i)\nmn\nn∈N (m)\\n\nn<n\n1 + ←−τ (i)\nmn\n1 − ←−τ (i)\nmn\n(ii)\nVertical Step\n−→z (i)\nmn = Fn +\nmn = Fn +\n←−ε (i)\nmn\nStep 2 Exchange of\nthe more reliable messages. Set\nmn = −→z (i)\nmn for 0 ≤ n < N/2 and ←−z (i)\nmn = ←−z (i)\n−→z (i)\nfor N/2 ≤ n ≤ N − 1.\nStep 3\nHard decision and stopping criterion test:\nA. Conventional decoding methods of TPC\nn ] such that for 0 ≤ n <\nn = 1 if Fn +\nmn <\nn = 0 otherwise; for N/2 < n ≤\nn = 1 if Fn +\nmn <\nn = 0 otherwise.\nCreate ˆw(i) = [ ˆw(i)\nN/2, ˆw(i)\n0, and ˆw(i)\nN −1, ˆw(i)\n0, and ˆw(i)\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 4. Otherwise set\ni := i + 1 and go to Step 1.\n−→ε (i)\nStep 4 Output ˆw(i) as the decoded codeword.\nAn alternative approach that can be used is to make the\ntwo subdecoders exchange the more reliable messages after\nupdating the reliability message of each bit (or a group of\nbits). We observe that this “simultaneous replica” approach\nprovides a faster convergence, especially for Gallager-type\nLDPC codes. Thus all simulation results of LDPC codes are\nbased on simultaneous updating of each bit (or a group of\nbits). It is also straightforward to extend the replica shufﬂed\nBP decoding to the cases in which more than two replica\nsubdecoders are used. In order to decrease decoding delay\nof plain shufﬂed BP decoding, a parallel version of shufﬂed\nBP named group shufﬂed BP was developed in [7]. In a\nsimilar way, group replica shufﬂed BP can also preserve the\nparallelism advantage of the standard BP algorithm. It is also\nclear this idea can be extended to other other grouping scheme\n(e.g, [11]) and other iterative decoding algorithms, such as bit\nﬂipping and weighted bit ﬂipping decoding.\nD. Simulation results\nIV. ITERATIVE DECODING OF TURBO PRODUCT CODES\nA two-dimension turbo product code (TPC) can be denoted\nas C1\nC2, where C1 and C2 are two linear block codes.\nPlace k1 × k2 information symbols in an array of k1 rows\nand k2 columns, and then encode the k1 rows using code C2.\nAfterwards, the resulting n2 columns are encoded using code\nC1. Usually, we choose C1 the same as C2.\nThe conventional TPC decoder performs row and column\ndecoding in a serial fashion. A soft input soft output (SISO)\ndecoder, such as MAP, is used to decode each row or column.\nA low complexity decoding approach is provided in [13]. It\napplies the Chase algorithm iteratively on the row and column\ndecoding, but still in a serial fashion. In order to halve the\ndecoding latency, a parallel TPC decoder has been proposed\nin [10]. As opposed to the conventional serial TPC decoder,\nthe row and column decoders in this method operate in\nparallel and send each other the updated extrinsic information\nimmediately after a row or column has been decoded. The\nsimulation results reveal that this parallel decoder can reduce\ndecoding latency by half of that of the original decoder.\nB. Replica decoding of TPC\nBased on the parallel decoder, we propose a replica parallel\ndecoder as shown in Fig. 4, which can further reduce decoding\nlatency. Both row and column decoders are duplicated, but\nwork from opposite extremes, which means the two row\ndecoders process rows from the top and bottom, respectively\nand the two column decoders process columns from the\nleft and right, respectively. Row (Column) decoders send to\nboth column (row) decoders immediately the latest extrinsic\nmatrices, [Wcol\nB ] ([Wrow\nR ]), after a row\n(column) has been decoded.\nL ] and [Wrow\nT ] and [Wcol\nThe procedure of the replica parallel decoding is shown\nin Fig. 5. The circles denote bit positions that were already\nupdated by other decoders. Their number is much larger than\nthat in the parallel decoder, which greatly beneﬁts the decoding\nbecause most bits have more accurate priori information. The\narrows with letters aT , bT , etc., represent the processing order\nof different decoders. After both row (column) decoders ﬁnish\ndecoding all the rows (columns), the most reliable parts of\nthem are combined and the resulting extrinsic matrix for the\nnext iteration is transmitted to both column (row) decoders.\nthe performance of TPC (64, 57, 4)2.\nare\nThe weighting factors with the\n[0.1, 0.2, 0.3, 0.5, 0.7, 0.9, 1.0, 1.0]. The MAP algorithm is\nused to decode the two linear block codes. As shown in Fig.\n6, the performance at the third iteration in the replica decoder\nis better than that at the fourth iteration in the parallel decoder.",
         "",
         "",
         "Replica shufﬂed versions of interactive decoders of turbo codes, low-density parity-check codes\nand turbo product codes are presented. The proposed schemes converge faster than standard and\npreviously proposed shufﬂed approaches. Simulations show that the new schedules offer good\nperformance versus complexity/latency trade-offs.\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in whole or in part\nwithout payment of fee is granted for nonproﬁt educational and research purposes provided that all such whole or partial copies include\nthe following: a notice that such copying is by permission of Mitsubishi Electric Research Laboratories, Inc.; an acknowledgment of\nthe authors and individual contributions to the work; and all applicable portions of the copyright notice. Copying, reproduction, or\nrepublishing for any other purpose shall require a license with payment of fee to Mitsubishi Electric Research Laboratories, Inc. All\nrights reserved.\nMITSUBISHI ELECTRIC RESEARCH LABORATORIES\nhttp://www.merl.com\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang, Marc Fossorier, and Jonathan S. Yedidia\nAbstract\nReplica shufﬂed versions of iterative decoders of turbo codes, low-density parity-check\ncodes and turbo product codes are presented. The proposed schemes converge faster\nthan standard and previously proposed “shufﬂed” approaches. Simulations show that\nthe new schedules offer good performance versus complexity/latency trade-offs.\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in\nwhole or in part without payment of fee is granted for nonproﬁt educational and research purposes provided that all such\nwhole or partial copies include the following: a notice that such copying is by permission of Mitsubishi Electric Research\nLaboratories, Inc.; an acknowledgment of the authors and individual contributions to the work; and all applicable portions\nof the copyright notice. Copying, reproduction, or republishing for any other purpose shall require a license with payment\nof fee to Mitsubishi Electric Research Laboratories, Inc. All rights reserved.\nTo appear in the Proceedings of the 2005 International Symposium on Information Theory.\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang and Marc Fossorier\nDepartment of Electrical Engineering\nUniversity of Hawaii at Manoa\nHonolulu, HI 96822\nEmail: juntan, yige, marc@spectra.eng.hawaii.edu\nJonathan S. Yedidia\nMitsubishi Electric Research Laboratories\n201 Broadway, Cambridge, MA 02139\nEmail: yedidia@merl.com\nI. INTRODUCTION\nIterative decoding based on belief propagation (BP) [1] has\nreceived signiﬁcant attention recently, mostly due to its near-\nShannon-limit error performance for the decoding of low-\ndensity parity-check (LDPC) codes [2] and turbo codes [3].\nLike maximum a posterior probability (MAP) decoding [4],\nit is a symbol-by-symbol soft-in/soft-out decoding algorithm.\nIt processes the received symbols recursively to improve the\nreliability of each symbol based on constraints that specify\nthe code. In the ﬁrst iteration, the decoder only uses the\nchannel output, and generates soft output for each symbol.\nSubsequently, the output reliability measures of the decoded\nsymbols at the end of each decoding iteration are used as\ninputs for the next iteration. The decoding iteration process\ncontinues until a certain stopping condition is satisﬁed. Then\nhard decisions are made based on the output reliability mea-\nsures of the decoded symbols at the last decoding iteration.\nThe standard BP decoder of LDPC codes often needs several\ntens or hundreds of iterations for the decoding process to\nconverge, which is not desirable because of high decoding\ndelay. Furthermore, LDPC codes of interest can have large\nthe\ncodeword length and it can be difﬁcult\ndecoding in hardware in a fully parallel way. Because of\nthe serial property of the BCJR algorithm, decoders of turbo\ncodes cause high delay as well. In [5], [6] and [7], “shufﬂed”\nmethods were presented to reduce the required number of\niterations for decoding LDPC and turbo codes, respectively.\nA speedup factor of 2 for LDPC codes and the saving of\none iteration for turbo codes were reported. The aim of this\npaper is to introduce a “replica shufﬂed” scheme which further\naccelerate the decoding process for turbo codes, LDPC codes,\nturbo product codes and other iterative decodable codes.\nto implement\nII. ITERATIVE DECODING OF TURBO CODE\nA turbo code [3] encoder is constructed using a concatena-\ntion of two (or more) convolutional encoders, and its decoder\nconsists of two (or more) soft-in/soft-out convolutional de-\ncoders which feed reliability information to each other. For\nis the output code block at\nsimplicity, we consider a turbo code that consists of two rate-\n1/n systematic convolutional codes with encoders in feedback\nform. Let u = (u1, u2, . . . , uK) be an information block of\nlength K and c = (c1, c2, . . . , cK) be the corresponding\ncoded sequence, where ck = (ck,1, ck,2, . . . , ck,n), for k =\n1, 2, . . . , K,\ntime k. Suppose\nBPSK transmission over an AWGN channel, with uk and\nck,j all taking values in {+1, -1} for k = 1, 2, . . . , K and\nj = 1, 2, . . . , n. Let y = (y1, y2, . . . , yK) be the received\nsequence, where yk = (yk,1, yk,2, . . . , yk,n) is the received\nblock at time k. Let ˆu = {ˆu1, ˆu2, . . . , ˆuK} denote the estimate\nof u. Let sk denote the encoder state at time k. Following\n1 ), γk(s, s) = p(sk =\n[4], deﬁne: αk(s) = p(sk = s, yk\ns, yk|sk−1 = s), βk(s) = p(yK\na =\n(ya, ya+1, . . . , yb), and let α(m)\n(s) be\nk\nthe corresponding values computed in component decoder m,\nwith m = 1, 2. Let L(i)\nem(ˆuk) denote the extrinsic value of the\nestimated information bit ˆuk delivered by component decoder\nm at the ith iteration [8].\nk+1|sk = s), where yb\n(s), γ(m)\nA. Standard serial and parallel turbo decoding\nThe decoding approach proposed in [3] operates in serial\nmode, i.e., the component decoders take turns generating the\nextrinsic values of the estimated information symbols, and\neach component decoder uses the extrinsic messages delivered\nby the last component decoder as the a priori values of the\ninformation symbols. The disadvantage of this scheme is high\ndecoding delay. In the parallel turbo decoding algorithm [9],\nall component decoders operate in parallel at any given time.\nAfter each iteration, each component decoder delivers extrinsic\nmessages to other decoder(s) which use these messages as a\npriori values at the next iteration.\nB. Plain shufﬂed turbo decoding\nAlthough the parallel turbo decoding reduces the decoding\ndelay of serial decoding by half, the extrinsic messages are not\ntaken advantage of as soon as they become available, because\nthe extrinsic messages are delivered to component decoders\nonly after each iteration is completed. The aim of the shufﬂed\nturbo decoding is to use the more reliable extrinsic messages\nat each time. Let ˜u = (˜u1, ˜u2, . . . , ˜uK) be the sequence\npermuted by the interleaver corresponding to the original\ninformation sequence u = (u1, u2, . . . , uK), according to the\nmapping ˜uk = uπ(k), for k = 1, 2, . . . , K. We assume that k =\nπ(k), ∀k. There is a unique corresponding reverse mapping\nuk = ˜uπ−(k), for k = 1, 2, . . . , K and k = π−(k), ∀k. In\nshufﬂed turbo decoding, the two component decoders operate\nsimultaneously as in the parallel turbo decoding scheme, but\nthe messages are updated during each iteration based on π(k)\nand π−(k) [7]. Correspondingly it provides a faster decoding\nconvergence.\nC. Replica shufﬂed turbo decoding\n←−\nD 1 and\n−→\nD 1 and\n−→\nD 1, which are denoted by\nIn the plain shufﬂed turbo decoding summarized in Section\nthe component decoders process the\nII-B, we assume all\nbackward recursion followed by the forward recursion. Let\n−→\nus refer to the two component decoders as\nD 2.\nNaturally another possible scheme is to operate in the reverse\norder, i.e, all the component decoders process the forward\nrecursion followed by the backward recursion and we refer\n←−\nto them as\nD 2. In terms of error performance, there\nis no difference between these two approaches. However, the\nreliabilities of the extrinsic messages concerning a certain\ninformation bit delivered by these two shufﬂed turbo decoders\nare not the same. In general, the more independent information\nis used, the more reliable the delivered messages become.\nTherefore for the extrinsic messages delivered by component\n(i)\ndecoder\ne1 (ˆuk), the larger k\nis, the more reliable this message is. Following a similar\n←−\nanalysis, for the extrinsic message\nD 1,\nthe smaller k is, the more reliable this message is. It is natural\nto expect a faster decoding convergence if these two shufﬂed\nturbo decoders operate cooperatively instead of independently.\nBecause in this approach two sets of shufﬂed component\ndecoders are used to decode the same sequence of information\nbits, we refer to it as replica shufﬂed turbo decoding. In replica\nshufﬂed turbo decoding, two plain shufﬂed turbo decoders\n−→\n(processing recursions in opposite directions)\nD 2 and\n←−\n←−\nD 1 ,\nD 2 operate simultaneously and exchange more reliable\nextrinsic messages. We assume that the component decoders\n−→\n\n\ndeliver extrinsic messages synchronously, i.e.,\nk =\nk =\nT\n←−\n←−\n−→\n←−\n←−\n\n\n\n\n\nk)and\nk (\nT\nk, where the\nk) denote the times\nT\nT\nT\nk =\nT\n←−\n−→\n←−\n−→\nat which\nD 2) deliver the extrinsic values\nD 2 (\nD 1) and\nD 1 (\nof the k-th ((K + 1 − k)-th) estimated symbol of the original\ninformation sequence u and of the interleaved sequence ˜u,\nrespectively.\n(i)\ne1 (ˆuk) delivered by\n−→\nD 1 ,\n−→\nD 1. After time\nLet us ﬁrst consider the forward recursion stage at the i-\n−→\n\nk−1,\nT\nth iteration of component decoder\nthe values of −→α (1)\nk (s) should be updated and the values\nof −→γ (1)\nk (s) are needed. There are two possible cases. The\nﬁrst case is k > π−(k), which means the extrinsic value\n−→\n(i)\ne2 (ˆuk) of the information bit ˆuk has already been delivered\nL\n−→\nD 2. As in plain shufﬂed turbo decoding, this newly\nby decoder\n−→\n(i)\ne2 (ˆuk) is used to compute the values −→γ (1)\nL\navailable\nk (s),\n−→\n(i)\n−→α (1)\ne1 (ˆuk). The second case is k < π−(k), which\nL\nk (s), and\n(i)\ne2 (ˆuk) of the information bit ˆuk\nmeans the extrinsic value\n−→\nD 2. Then in plain shufﬂed\nhas not been delivered yet by\nturbo decoding, the values α(1)\ne1 (ˆuk) are updated\nk (s) and L(i)\n−→\nL\nk (s), and\nbased on the extrinsic messages delivered at last iteration.\nIn replica shufﬂed turbo decoding, however, there are two\nfurther subcases. The ﬁrst subcase is K + 1 − k < π−(k),\n(i)\ne2 (ˆuk) of the information\nwhich means the extrinsic value\n←−\nbit ˆuk has already been delivered by decoder\nD 2. Then this\n(i)\n(i−1)\ne2 (ˆuk), instead of\n(ˆuk) is used to\nnewly available\ne2\n−→\n(i)\ncompute the values −→γ (1)\nk (s), −→α (1)\ne1 (ˆuk). The\nL\nsecond subcase is K + 1 − k < π−(k), which means both\n(i)\nextrinsic messages of the information bit ˆuk, i.e,\ne2 (ˆuk)\n−→\n(i)\ne2 (ˆuk) are not available yet. In this subcase, the values\nL\nand\n(i)\nof −→α (1)\ne1 (ˆuk) are updated based on the extrinsic\nk (s) and\nmessages delivered at (i − 1)-th iteration. The recursions of\n←−\n←−\ncomponent decoders\nD 2 are realized based on\nD 1 and\nthe same principle. After Imax iterations, the shufﬂed turbo\ndecoding algorithm outputs ˆu = (ˆu1, ˆu2, . . . , ˆuK) as the de-\n←−\n(i)\ncoded codeword, where ˆuk =sgn[(\ne1 (ˆuk))/2 +\nL\n−→\n(i)\ne2 (ˆuk))/2 + 4\nyk,1], which is different from\nL\n(\nN0\nthat in the standard turbo decoding [3] and plain shufﬂed\nturbo decoding. It is straightforward to generalize the replica\nshufﬂed turbo decoding to multiple turbo codes which consist\nof more than two component codes. Also group of bits can\nbe updated periodically only to reduce information exchanges\nbetween replicas. Based on the above descriptions with two\nreplicas, the total computational complexity of the replica\nshufﬂed turbo decoding for multiple turbo codes at each\ndecoding iteration is about twice that of the parallel turbo\ndecoding.\n(i)\ne2 (ˆuk) +\n(i)\ne1 (ˆuk) +\n−→\nL\nD. Simulation results\nIII. ITERATIVE DECODING OF LDPC CODES\nLet H = [Hmn] be the parity check matrix which deﬁnes\nan LDPC code. We denote the set of bits that participate in\ncheck m by N (m) = {n : Hmn = 1} and the set of checks\nin which bit n participates as M(n) = {m : Hmn = 1}.\nAssume a codeword w = (w1, w2, . . . , wN ) is transmitted\nover an AWGN channel with zero mean and variance N0/2\nusing BPSK signaling and let y = (y1, y2, . . . , yN ) be the\ncorresponding received sequence.\nA. Standard BP for iterative decoding of LDPC codes\nBased on [1], let Fn be the log-likelihood ratio (LLR) of\nbit n and initially set Fn = 4\nmn be the\nN0\nLLR of bit n which is sent from check node m to bit node n\nand sent from the bit node n to check node m, respectively.\nLet z(i)\nn denote the a posteriori LLR of bit n. The standard\nBP algorithm [1] is carried out as follows:\nyn. Let ε(i)\nInitialization Set i = 1, maximum number of iterations to\nIM ax. For each m,n, set z(0)\nmn = Fn.\n(i) Horizontal Step, for 1 ≤ n ≤ N and each m ∈\nM(n), process:\nτ (i)\nmn =\nn∈N (m)\\n\n1 + τ (i)\nmn\n1 − τ (i)\nmn\n(ii) Vertical Step, for 1 ≤ n ≤ N and each m ∈\nε(i)\nmn = log\n(2)\nz(i)\nmn = Fn +\nz(i)\nn = Fn +\n(i)\nStep 2 Hard decision and stopping criterion test:\nCreate ˆw(i) = [ ˆw(i)\nn ] such that ˆw(i)\nz(i)\nn > 0, and ˆw(i)\nn = 1 if z(i)\nn < 0.\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 3. Otherwise set\ni := i + 1 and go to Step 1.\nn = 0 if\n(ii)\nStep 3 Output ˆw(i) as the decoded codeword.\nB. Plain shufﬂed BP for iterative decoding of LDPC codes\nAt the i-th iteration of the standard BP algorithm, ﬁrst all\nvalues of the check-to-bit messages are updated by using the\nvalues of the bit-to-check messages obtained at the (i − 1)-\nth iteration, i.e., each ε(i)\n:\nn ∈ N (m)\\n}. Then, all values of the bit-to-check messages\nare updated by using the values of the check-to-bit messages\nnewly obtained at the i-th iteration, i.e., each z(i)\nmn is updated\nfrom {ε(i)\nmn is updated by using {z(i−1)\nmn\nmn : m ∈ M(n)\\m}.\nWe observe that certain values z(i)\nin (1) to compute the remaining values ε(i)\nmn could already be\ncomputed in (3) based on a partial computation of the values\nε(i)\nmn obtained from (2), and then could be used instead of\nz(i−1)\nmn. Hence Step\nmn\n1 of the shufﬂed BP algorithm is performed as: for 1 ≤ n ≤ N\nand each m ∈ M(n), process the horizontal step and vertical\nstep jointly, with (1) modiﬁed as:\n\nτ (i)\nmn =\nn∈N (m)\\n\nn>n\nSince this algorithm becomes totally serial, bits can be grouped\nto maintain a sufﬁcient level of parallelism with the same error\nperformance [7].\nC. Replica shufﬂed BP decoding for LDPC codes\nPlain shufﬂed BP decoding is a bit-based sequential ap-\nproach and the method described in Section III-B is based\non a natural increasing order, i.e, belief messages concerning\nbit nodes are updated according to order i = 1, 2, . . . , N .\nThe larger the value of i, the more independent informations\nare used to update the beliefs of bit note i and the more\nreliable these belief messages are. Therefore the reliability of\nbit nodes increases and the error rate decreases as i increases.\nFollowing a similar analysis, in plain shufﬂed BP decoding\nbased on a natural decreasing order, after each iteration, the\nreliability of bit nodes decreases as i increases. As in replica\nshufﬂed turbo decoding, in replica shufﬂed BP decoding, two\nreplica shufﬂed subdecoders based on different updating orders\noperate simultaneously and cooperatively. After each iteration,\neach subdecoder receives more reliable messages from and\nsends more reliable messages to another subdecoder. Based on\nthese more reliable messages, both replica subdecoders begin\n←−\nthe next iteration of decoding. Let\nD denote the replica\nsubdecoder with natural increasing and decreasing updating\norder, respectively. Let −→ε (i)\nmn be the variables\n←−\nassociated with\nD\nare deﬁned in a similar way. The replica shufﬂed BP decoding\nwith two replica subdecoders is carried out as followings:\n−→\nD at iteration i. Variables associated with\nmn and −→z i\n−→\nD and\nInitialization Set i = 1, maximum number of iteration to\nIM ax. For each m,n, set −→z (0)\nmn = ←−z (0)\nmn = Fn.\nStep 1\nEach replica subdecoder process as the following\ntwo steps simultaneously. For 0 ≤ n ≤ N − 1 and\neach m ∈ M(n), process\nHorizontal Step\nn∈N (m)\\n\nn>n\n1 + −→τ (i)\nmn\n1 − −→τ (i)\nmn\nn∈N (m)\\n\nn<n\n1 + ←−τ (i)\nmn\n1 − ←−τ (i)\nmn\n(ii)\nVertical Step\n−→z (i)\nmn = Fn +\nmn = Fn +\n←−ε (i)\nmn\nStep 2 Exchange of\nthe more reliable messages. Set\nmn = −→z (i)\nmn for 0 ≤ n < N/2 and ←−z (i)\nmn = ←−z (i)\n−→z (i)\nfor N/2 ≤ n ≤ N − 1.\nStep 3\nHard decision and stopping criterion test:\nA. Conventional decoding methods of TPC\nn ] such that for 0 ≤ n <\nn = 1 if Fn +\nmn <\nn = 0 otherwise; for N/2 < n ≤\nn = 1 if Fn +\nmn <\nn = 0 otherwise.\nCreate ˆw(i) = [ ˆw(i)\nN/2, ˆw(i)\n0, and ˆw(i)\nN −1, ˆw(i)\n0, and ˆw(i)\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 4. Otherwise set\ni := i + 1 and go to Step 1.\n−→ε (i)\nStep 4 Output ˆw(i) as the decoded codeword.\nAn alternative approach that can be used is to make the\ntwo subdecoders exchange the more reliable messages after\nupdating the reliability message of each bit (or a group of\nbits). We observe that this “simultaneous replica” approach\nprovides a faster convergence, especially for Gallager-type\nLDPC codes. Thus all simulation results of LDPC codes are\nbased on simultaneous updating of each bit (or a group of\nbits). It is also straightforward to extend the replica shufﬂed\nBP decoding to the cases in which more than two replica\nsubdecoders are used. In order to decrease decoding delay\nof plain shufﬂed BP decoding, a parallel version of shufﬂed\nBP named group shufﬂed BP was developed in [7]. In a\nsimilar way, group replica shufﬂed BP can also preserve the\nparallelism advantage of the standard BP algorithm. It is also\nclear this idea can be extended to other other grouping scheme\n(e.g, [11]) and other iterative decoding algorithms, such as bit\nﬂipping and weighted bit ﬂipping decoding.\nD. Simulation results\nIV. ITERATIVE DECODING OF TURBO PRODUCT CODES\nA two-dimension turbo product code (TPC) can be denoted\nas C1\nC2, where C1 and C2 are two linear block codes.\nPlace k1 × k2 information symbols in an array of k1 rows\nand k2 columns, and then encode the k1 rows using code C2.\nAfterwards, the resulting n2 columns are encoded using code\nC1. Usually, we choose C1 the same as C2.\nThe conventional TPC decoder performs row and column\ndecoding in a serial fashion. A soft input soft output (SISO)\ndecoder, such as MAP, is used to decode each row or column.\nA low complexity decoding approach is provided in [13]. It\napplies the Chase algorithm iteratively on the row and column\ndecoding, but still in a serial fashion. In order to halve the\ndecoding latency, a parallel TPC decoder has been proposed\nin [10]. As opposed to the conventional serial TPC decoder,\nthe row and column decoders in this method operate in\nparallel and send each other the updated extrinsic information\nimmediately after a row or column has been decoded. The\nsimulation results reveal that this parallel decoder can reduce\ndecoding latency by half of that of the original decoder.\nB. Replica decoding of TPC\nBased on the parallel decoder, we propose a replica parallel\ndecoder as shown in Fig. 4, which can further reduce decoding\nlatency. Both row and column decoders are duplicated, but\nwork from opposite extremes, which means the two row\ndecoders process rows from the top and bottom, respectively\nand the two column decoders process columns from the\nleft and right, respectively. Row (Column) decoders send to\nboth column (row) decoders immediately the latest extrinsic\nmatrices, [Wcol\nB ] ([Wrow\nR ]), after a row\n(column) has been decoded.\nL ] and [Wrow\nT ] and [Wcol\nThe procedure of the replica parallel decoding is shown\nin Fig. 5. The circles denote bit positions that were already\nupdated by other decoders. Their number is much larger than\nthat in the parallel decoder, which greatly beneﬁts the decoding\nbecause most bits have more accurate priori information. The\narrows with letters aT , bT , etc., represent the processing order\nof different decoders. After both row (column) decoders ﬁnish\ndecoding all the rows (columns), the most reliable parts of\nthem are combined and the resulting extrinsic matrix for the\nnext iteration is transmitted to both column (row) decoders.\nthe performance of TPC (64, 57, 4)2.\nare\nThe weighting factors with the\n[0.1, 0.2, 0.3, 0.5, 0.7, 0.9, 1.0, 1.0]. The MAP algorithm is\nused to decode the two linear block codes. As shown in Fig.\n6, the performance at the third iteration in the replica decoder\nis better than that at the fourth iteration in the parallel decoder.",
         "Juntan Zhang,M. Fossorier",
         null
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus\nBUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath.\nIn endothelium-intact aortic rings, A. membranaceus extract induced a signiﬁcant dose-dependent relaxation of\nthe rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(ww)-nitro-arginine\nmethyl  ester  or  methylthioninium  chloride.  In  endothelium-denuded  ones,  the  extract  could  dose-dependently\nrelax  the  rings  contracted  by  phenylephrine,  not  by  KCl;  and  it  could  also  attenuate  contractile  response  to\nphenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca2-free medium; but it failed to affect the CaCl2-\ninduced enhancement of contractile response to phenylephrine in Ca2-free medium. These results indicate that\nnitric oxide signaling and Ca2-handling pathway are involved in the A. membranaceus extract-induced vasodi-\nlatation.",
         "The  roots  of  Astragalus  membranaceus BUNGE (Legumi-\nnosae) are amongst the most popular and important “Qi (pro-\nnounce  chee)  tonifying”  adaptogenic  herbs  in  China,  their\nuse  dates  back  more  than  2000  years,  and  are  recorded  in\nShen  Nong’s Materia  Medica.  According  to  ethnobotancical\ndata  collected  in  China,  A.  membranaceus has  been  pre-\nscribed  for  centuries  for  diarrhea,  frequent  colds,  sponta-\nneous  sweats,  fatigue,  and  edema.  Furthermore,  A.  mem-\nbranaceus is  alleged  to  possess  the  effects  of  antioxidative\ndamage,1) immune-stimulation,2) and antiviral infection.3)\nRecently,  it  has  received  more  attention  that  the  fact\nrecorded in ancient Chinese texts that A. membranaceus pos-\nsessed an antihypertensive effect by raising the yang qi of the\nspleen and kidney, since deﬁciency qi formed by the disorder\nof spleen and kidney is considered to be the basic pathogene-\nses  of  hypertension.4) A  great  deal  of  studies  demonstrated\nthat  A.  membranaceus was  beneﬁcial  on  the  process  of  hy-\npertension  in  animals  and  humans,5—7) and  a  cooperative\nmechanism  of\nrenin  angiotensin  aldosterone  system,\nkallikrein bradykinin system, and central neuropeptide8) was\nused  to  explain  this  effect  of  A.  membranaceus.  We  also\nfound  in  the  past  experiments  that  the  administration  of  A.\nmembranaceus could attenuate the progression of blood pres-\nsure  in  spontaneously  hypertensive  rats  concomitant  with\nrestoration  of  baroreﬂex  sensitivity  and  reduction  of  Ca2\nconcentration  in  lymphocyte  and  vascular  smooth  muscle\ncells  (VSMC).9—11) Additionally,  in  vivo  experiment  had\ndemonstrated that A. membranaceus could induce the vasodi-\nlatation.12) However,  there  is  no  experiment  evidence  avail-\nable  to  show  the  exact  mechanism  of  vasorelaxation  pro-\nduced by A. membranaceus. The present study is conducted\nto  elucidate  various  possible  mechanisms  in  vasomotor  re-\nsponse of A. membranaceus.",
         "M)  induced  a  similar  sustained  contraction\nof  aortic  rings  in  each  group  with  a  peak  tension  of  about\n3.110.27 g  in  intact  aortic  rings  and  3.020.19 g  in  de-\nnuded ones.\nEffects  of  A.  membranaceus on  the  Vasotension All\ndoses (2.0105—2.0101 g/l) of A. membranaceus exhib-\nited  negligible  vasomotor  actions  on  aortic  rings  with  or\nwithout  endothelium  at  resting  tension  (Fig.  1).  A.  mem-\nbranaceus (2.0105—2.0101 g/l)  produced  dose-depen-\ndent  relaxation  in  endothelium-denuded  rings  (Fig.  2),  after\nthe  plateau  tension  induced  by  PE.  However,  in  PE-precon-\ntracted  endothelium-intact  rings,  low  doses  (2.0104—\n6.0103 g/l) of A. membranaceus showed a more signiﬁcant\nrelaxation than that in denuded ones (Fig. 2), while high dose\n(2.0102—2.0101 g/l) of the extract induced a transient\ncontraction (Figs. 2, 3).\nEndothelium-Independent  Vasodilatation  by  A.  mem-\nIn  contrast  to  the  effect  on  PE-contracted  en-\nbranaceus\ndothelium-denuded rings in K–H solution, A. membranaceus\n(2.0104—2.0101 g/l)  had  no  dilatation-effect  on  rings\nstimulated by KCl (3.0102\nM) (Fig. 4).\nAnd in the experiment with Ca2-free medium, aortic con-\ntraction  elicited  by  PE,  caffeine  or  PD  was  decreased  sub-\nstantially, as these vascular contractions in Ca2-free medium\nare mediated only via the release of Ca2 from the intracellu-\nlar stores or phosphorylation of myosin light chain. A. mem-\nbranaceus (6.0103 g/l)  could  inhibit  the  contraction  in-\nduced  by  PE  (3.0107\nM),  but  not  block  the  extracellular\nCa2 inﬂow,  because  2.5 mM CaCl2 induced  a  similar  en-\nhancement of contraction by PE, compared with the control\n(95.842.63%  vs. 96.961.87%,  p0.05).  Furthermore,  A.\nmembranaceus (6.0103 g/l) failed to induce an attenuation\nof  contractile  response  by  PD  (1.0106\nM)  and  caffeine\n(2.5102\nM) (Fig. 5).\nEndothelium-Dependent  Vasomotor  Effects  by  A.\nmembranaceus Pre-incubation  of  the  intact  rings  with  L-\nM) was added, the effect of A. membranaceus (2.0102,\n6.0102 and 2.0101 g/l, showed by a, b, c point) was obtained. For it was similar with Fig. 2, the effect of A. membranaceus (2.0105—6.0103 g/l) was cut. Values of both\ngroups are expressed as meanS.E.M. of 8 experiments. Contractile responses are expressed as the percentage from maximal contraction elicited by phenylephrine (3.0107\nM).\n∗ p0.05, ∗∗ p0.01, compared with corresponding values before administration.\nNAME  (1.0104\nM)  markedly,  incompletely  inhibited  A.\nmembranaceus (6.0103 g/l)-induced  relaxation  from  36.0\nto 7.3% (Fig. 6). Similarly, MC (1.0105\nM) also showed an\ninhibition  of  endothelium-dependent  vasodilatation  from\n36.0  to  8.0%  (Fig.  6).  Thus,  NO  signaling  might  play  a\nmajor, but not full role in the vasodilatation effect of A. mem-\nbranaceus.\nWhen the dose of A. membranaceus in the organ bath was\nstepwise raised to 2.0102—2.0101 g/l, a transient con-\ntraction  could  be  observed  in  endothelium-intact  rings.  Pre-\ntreatment  of  endothelium-intact  rings  with  phosphoramidon\n(5.0106\nM)  signiﬁcantly  attenuated  the  transient  contrac-\ntion response (Fig. 3).",
         "DISCUSSION\nThe present study ﬁrst and detailedly explored the in vitro\nvascular effects of A. membranaceus in aortic rings isolated\nfrom rats. In the course of the study, a number of novel ob-\nservations  had  been  made,  which  contribute  to  better  scien-\ntiﬁc  understanding  of  their  therapeutic  use  for  antihyperten-\nsion associated with Qi deﬁciency caused by the disorder of\nspleen and kidney in China.\nFirst, although neurohumoral factors may contribute to an-\ntihypertension for the crude extract of A. membranaceus ob-\nserved previously,8) it is apparent that direct endothelium-de-\npendent  and  independent  vasorelaxation  should  be  promi-\nnently taken into account according to this study.\nSecond,  A.  membranaceus dose-dependently  dilated  en-\ndothelium-denuded  rings  contracted  with  PE.  However,  the\nagent had no effect on the vessel contracted by KCl. The cel-\nlular  mechanism  of  contraction  involved  in  the  response  to\nKCl and PE is different. KCl induce Ca2 inﬂux via voltage-\ndependent  Ca2 channel,  which  further  activate  Ca2-in-\nduced Ca2 release through ryanodine-receptor. PE increase\nintracellular  Ca2 concentration  with  two  mechanisms,  1)\nthrough  receptor-gated  Ca2 channels;  2)  mobilizes  Ca2\nfrom  intracellular  stores  via the  inositol-1,4,5-trisphosphate\n(IP3)  receptor  or  induces  secondly  myosin  light  chain  phos-\nphorylation via activating PKC.17,18) That is, the vasorelaxant\naction of A. membranaceus seems to occur in a receptor-de-\npendent manner in SMVC of rat aortas. However, the result\nthat A. membranaceus failed to inhibit the contraction caused\nby addition of CaCl2 in Ca2 free solution was probably only\ndue to its effects on the intracellular pathways. This hypothe-\nsis was conﬁrmed in the further experiments that the contrac-\ntions induced by PD and caffeine in Ca2-free medium were\nnot  affected  by  A.  membranaceus equally.  Thus,  it  appears\nthat the IP3-induced Ca2 release channels might be the site\nof action for A. membranaceus on endothelium-removed aor-\ntas.\nThird,  NO  is  a  potent  vasodilator  synthesized  in  the  en-\ndothelium19) by  NO  synthase,  and  causes  VSMC  relaxation\nthrough  the  activation  of  soluble  guanylate  cyclase.20) The\npresent  study  demonstrated  that  the  extract  of  A.  mem-\nbranaceus dose-dependently  inhibited  the  contraction  in-\nduced  by  PE  in  intact  aorta  isolated  from  rats.  This  vasore-\nlaxant  action  was  mostly  inhibited  by  treatment  with  L-\nNAME  or  MC.  Endothelium-dependent  relaxation  of  A.\nmembranaceus seemed  to  be  associated  with  NO  signaling\nvia guanylate cyclase activation since both L-NAME and MC\ncould  attenuate  this  response.  This  result  was  supported  by\nother studies which had shown that A. membranaceus could\nincrease the plasma level of NO21) and induce the release of\nNO  from  VSMC  in  vitro  by  NO-soluble  guanylate  cyclase-\ncyclic guanosine monophosphate pathway.22)\nFinally,  it  was  also  observed  that  high  dose  (2.0102—\n2.0101 g/l) A. membranaceus induced a transient contrac-\ntion  in  arterial  rings  pre-constricted  by  PE.  This  result  was\nidentical with that obtained in another report where treatment\nwith  high  dose  of  A.  membranaceus reversibly  increased\nblood pressure of patients without hypertension previously.23)\nThe  mechanism  underlying  is  under  investigation.  Blockage\nof  the  contraction  by  endothelin-converting  enzyme  phos-\nphoramidon  may  suggest  the  involvement  of  an  endothelin\nreceptor mediated signaling.\nIn summary, the ﬁndings indicated that A. membranaceus\nhave  diphasic  effects  of  relaxation  and  contraction  on  the\naorta rings. The mechanisms of relaxation may include inhi-\nbition  of  intracellular  Ca2 release  in  vascular  smooth  mus-\ncle cells, and endothelium dependent relaxation mediated by\nIn the untreatment group, A. membranaceus or vehicle group was not pretreated with\nL-NAME or MC. Each column with a bar represents meanS.E.M. of 8 experiments.\nContractile  responses  are  expressed  as  the  percentage  from  maximal  contraction\nelicited by phenylephrine (3.0107\nM). ∗ p0.05, ∗∗ p0.01, compared with value of\nthe  corresponding  vehicle  group.   p0.01,  compared  with  value  of  the  experiment\nsubgroup in the untreatment group.\nthe NO-GC pathway, whilst the effect of transient contraction\nof A. membranaceus at a high dose was related with endothe-\nlin release.\nAcknowledgements This work was supported by grants\nfrom  Province  Administration  of  Traditional  Chinese  Medi-\ncine,  Zhejiang,  P.R.  China  (Project  Number:  2000C54  and\n2003C078);  Foundation  for  Returnee  from  Personal  Depart-\nment  of  Zhejiang  Province,  P.R.  China  (Project  Number:\nZRZ-2001275);  and  by  E-Institutes  of  Shanghai  Municipal\nEducation  Commission,  P.R.  China  (Project  Number:  E-\n04010). The authors are grateful to Prof. Wenquan Liang of\nthe  Department  of  Pharmacology,  Zhejiang  University,  for\nhis technical help.\n21) He L., Wang R. G., You Z. L., Yang Z. W., Chen X. D., Li J., Zhan X.\nL., Wang J., Acad. J. Hunan Tradit. Chin. Med., 23, 4—6 (2003).\n22) Wang K. H., Foreign Med. Sci.: TCM Section, 18, 38—39 (1996).\n23) Wang  C.  H.,  Zhang  J.  J.,  Shandong  J.  Tradit.  Chin.  Med.,  15,  351",
         "MATERIALS AND METHODS\nPlant  Material  and  Preparation  of  the  Extracts A.\nmembranaceus roots were collected in the northern region of\nGansu Province, China, in the autumn (2001) and identiﬁed\nby the Quality Assessment of Di’ao Group, Sichuang, China.\nAnd a voucher specimen was deposited in the Academic De-\npartment of Di’ao Group (STQ-C-00203).\nA. membranaceus roots were air dried. A crude extract was\nprepared  by  decoction  of  2.0102 g  in  500 ml  water  for  3\ntimes (90 min per time). The obtained extract was combined,\nﬁltered  and  concentrated.  Then  all  the  aqueous  extract  was\nextracted  with  400 ml  ethanol  (75%  and  85%)  twice.  After\nthe ethanol was retrieved, the active fractions were recovered,\nﬁltered  and  lyophilized  (yield  0.28%).  Consequently,  the\nﬁnal  extract  was  used  in  the  study  after  dissolution  in  dis-\ntilled water at 37 °C and stirred for 60 min.\nChemicals Phenylephrine  (PE),  acetylcholine  (ACh),  L-\nN(w)-nitro-arginine  methyl  ester  (L-NAME),  methylthion-\ninium  chloride  (MC),  phosphoramidon,  ethylene  glycol-bis\n[b-aminoethyl  ether]-N,N,N,N-tetraacetic  acid  (EGTA),\ncaffeine  and  phorbol-12,13-diacetate  (PD)  were  purchased\nfrom  Sigma  Chemical  Co.  (St.  Louis,  MO,  U.S.A.).  They\nwere  dissolved\nin  distilled  water  and  diluted  with\nKrebs–Henseleit  (K–H)  solution  before  use.  All  chemicals\nwere of the highest purity available.\nExperiment  Animals The  investigation  conformed  to\nthe  Chinese  Regulation  for  Administration  of  Laboratory\nAnimal  (SSTC  1988/2).  Health  male  Sprague–Dawley  rats\nweighing  260—280 g  (Laboratory  Animal  Center  of  Zhe-\njiang  College  of  Traditional  Chinese  Medicine,  Hangzhou,\nChina) were housed in conventional cages with free access to\nwater and rodent chow at the controlled temperature and hu-\nmidity with a 12-h light/dark cycle.\nPreparation of Thoracic Arterial Rings from Sprague–\nDawley  Rats Rats  were  euthanized  by  decapitation  using\nsmall animal guillotine. The thoracic aorta was immediately\nisolated  and  immersed  in  oxygenated  K–H  solution  4 °C  of\nthe  following  composition  (in  mM):  NaCl  118.3;  KCl  4.7;\nMgSO4 1.2; KH2PO4 1.2; NaHCO3 25.0; CaCl2 2.5; Glucose\n11.0.  Then  the  adherent  connective  tissue  was  cleaned  and\nblood  vessel  was  cut  into  3—4 mm  rings,  with  the  special\ncare  of  avoiding  damage  of  endothelium.  In  some  prepara-\ntions, endothelium was mechanically removed by gently rub-\nbing  the  lumen.  The  rings  were  mounted  horizontally  be-\ntween two stirrups in organ bath ﬁlled with 10 ml K–H solu-\ntion  at  37 °C,  ventilated  continuously  with  95%  O2 and  5%\nCO2. The isometric tension was recorded with a force trans-\nducer  (JZ101,  metrical  range:  0—5.0 g)  and  MedLab  5.0v\nrecording system (Nanjing Medease Science and Technology\nCo. Ltd).\nExperimental  Procedure  of  Isometric  Tension Rings\nwere equilibrated for 60 min at 2.0 g resting tension, and then\nchallenged with KCl (6.0102\nM) at least 3 times until a re-\nproducible maximal contractile response was obtained. After\na  further  equilibration  period  of  30 min,  the  integrity  of  the\nendothelium was assessed in all preparations by determining\nthe ability of ACh (1.0105\nM) to induce more than 80% re-\nlaxation  of  rings  pre-contracted  with  PE  (1.0106\nM).  The\nendothelium  was  considered  to  be  removed  when  there  was\nless than 10% relaxation response to ACh.\nProtocol 1: Effects of A. membranaceus on Vascular Tone:\nThe ﬁrst series of the experiments were conducted to assess\nendothelium-dependent  or  independent  effects  of  A.  mem-\nbranaceus on isolated aortic rings. When the tension was at\nresting  state  or  reached  a  plateau  induced  by  PE  (3.0\n107\nM),  A.  membranaceus (2.0105—2.0101 g/l)  was\ncumulatively added into the organ bath with 10 min interval.\nThe  rings  with  intact  or  denuded  endothelium  were  always\ntested in parallel.\nProtocol  2:  Endothelium-Independent  Vasomotor  Re-\nsponse by A. membranaceus: The second series of the exper-\niments  were  designed  to  determine  the  underlying  mecha-\nnisms  of  A.  membranaceus-induced  vasomotor  response  in\naortic rings without endothelium. In contrast to the response\nof  A.  membranaceus (2.0105—2.0101 g/l)  to  PE,  KCl\n(3.0102\nM)  was  also  employed  to  constrict  the  denuded\naortic rings. A plateau contractile response to KCl or PE was\nalso obtained as a time control.\nM), PD (1.0106\nAdditionally,  after  the  preparations  were  serially  washed\nand equilibrated for 60 min by Ca2-free medium (mM: NaCl\n118.3, KCl 4.7, KH2PO4 1.2, MgSO4 1.2, NaHCO3 25, Glu-\ncose 11.0, EGTA 0.05), PE (3.0107\nM)\n(an  activator  of  protein  kinase  C,  PKC),13) or  caffeine\n(2.5102\nM) (an activator of ryanodine receptor)14) was used\nto contract the denuded rings. In order to explore the proba-\nble  effect  of  extracellular  Ca2 inﬂow,  2.5 mM CaCl2 was\nadded  after  PE  stimulated  the  rings.  Then  the  effect  of  A.\nmembranaceus (6.0103 g/l)  was  tested.  A  time  control\n(same protocol as above except for the omission of A. mem-\nbranaceus to  the  second  application  of  PE,  PD  or  caffeine)\nwas  always  run  in  parallel.  Following  the  initial  control  re-\nsponse in Ca2-free medium and before the second test con-\ntraction (to PE or caffeine), a 6.0102\nM KCl-induced con-\ntraction  in  K–H  solution  was  obtained  to  reﬁll  the  Ca2\nstores in the vascular tissues.\nProtocol 3: Endothelium-Dependent Vasomotor Response\nby A. membranaceus: In the third series of experiments, the\npotential inﬂuence of A. membranaceus on PE-induced vaso-\nmotor function in the intact rings was examined. For pretreat-\nment,  endothelium-intact  rings  were  incubated  with  L-\nNAME (1.0104\nM) (a speciﬁc inhibitor of nitric oxide syn-\nthase), MC (1.0105\nM) (an inhibitor of guanylate cyclase)\nfor 15 min15) and phosphoramidon (5.0106\nM) (an inhibitor\nof  endothelin  converting  enzyme)  for  20 min16) before  ad-\nministration  of  PE.  Then  the  response  curves  of  A.  mem-\nbranaceus were recorded. The rings with or without the treat-\nment of inhibitors were always tested in parallel.\nStatistical  Analysis Relaxant  responses  were  expressed\nas the percentage decreases of the magnitude of the contrac-\ntion  induced  by  PE  or  KCl  (100%)  before  the  application\nof  vasodilators,  and  the  contractile  responses  as  the  ratio  of\nthe  responses  following  and  before  the  application  of  A.\nmembranaceus (second  response/ﬁrst  response).  All  results\nare  expressed  as  meanS.E.M.  Statistical  analysis  was  per-\nformed  with  t-test  or  ANOVA  followed  by  Newman–Keuls\ntest. Differences were accepted as statistically signiﬁcant at p\nvalues 0.05 (GraphPad Prism).\nRESULTS\nPE  (1.0106",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus\nBUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath.\nIn endothelium-intact aortic rings, A. membranaceus extract induced a signiﬁcant dose-dependent relaxation of\nthe rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(ww)-nitro-arginine\nmethyl  ester  or  methylthioninium  chloride.  In  endothelium-denuded  ones,  the  extract  could  dose-dependently\nrelax  the  rings  contracted  by  phenylephrine,  not  by  KCl;  and  it  could  also  attenuate  contractile  response  to\nphenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca2-free medium; but it failed to affect the CaCl2-\ninduced enhancement of contractile response to phenylephrine in Ca2-free medium. These results indicate that\nnitric oxide signaling and Ca2-handling pathway are involved in the A. membranaceus extract-induced vasodi-\nlatation.\nThe  roots  of  Astragalus  membranaceus BUNGE (Legumi-\nnosae) are amongst the most popular and important “Qi (pro-\nnounce  chee)  tonifying”  adaptogenic  herbs  in  China,  their\nuse  dates  back  more  than  2000  years,  and  are  recorded  in\nShen  Nong’s Materia  Medica.  According  to  ethnobotancical\ndata  collected  in  China,  A.  membranaceus has  been  pre-\nscribed  for  centuries  for  diarrhea,  frequent  colds,  sponta-\nneous  sweats,  fatigue,  and  edema.  Furthermore,  A.  mem-\nbranaceus is  alleged  to  possess  the  effects  of  antioxidative\ndamage,1) immune-stimulation,2) and antiviral infection.3)\nRecently,  it  has  received  more  attention  that  the  fact\nrecorded in ancient Chinese texts that A. membranaceus pos-\nsessed an antihypertensive effect by raising the yang qi of the\nspleen and kidney, since deﬁciency qi formed by the disorder\nof spleen and kidney is considered to be the basic pathogene-\nses  of  hypertension.4) A  great  deal  of  studies  demonstrated\nthat  A.  membranaceus was  beneﬁcial  on  the  process  of  hy-\npertension  in  animals  and  humans,5—7) and  a  cooperative\nmechanism  of\nrenin  angiotensin  aldosterone  system,\nkallikrein bradykinin system, and central neuropeptide8) was\nused  to  explain  this  effect  of  A.  membranaceus.  We  also\nfound  in  the  past  experiments  that  the  administration  of  A.\nmembranaceus could attenuate the progression of blood pres-\nsure  in  spontaneously  hypertensive  rats  concomitant  with\nrestoration  of  baroreﬂex  sensitivity  and  reduction  of  Ca2\nconcentration  in  lymphocyte  and  vascular  smooth  muscle\ncells  (VSMC).9—11) Additionally,  in  vivo  experiment  had\ndemonstrated that A. membranaceus could induce the vasodi-\nlatation.12) However,  there  is  no  experiment  evidence  avail-\nable  to  show  the  exact  mechanism  of  vasorelaxation  pro-\nduced by A. membranaceus. The present study is conducted\nto  elucidate  various  possible  mechanisms  in  vasomotor  re-\nsponse of A. membranaceus.\nMATERIALS AND METHODS\nPlant  Material  and  Preparation  of  the  Extracts A.\nmembranaceus roots were collected in the northern region of\nGansu Province, China, in the autumn (2001) and identiﬁed\nby the Quality Assessment of Di’ao Group, Sichuang, China.\nAnd a voucher specimen was deposited in the Academic De-\npartment of Di’ao Group (STQ-C-00203).\nA. membranaceus roots were air dried. A crude extract was\nprepared  by  decoction  of  2.0102 g  in  500 ml  water  for  3\ntimes (90 min per time). The obtained extract was combined,\nﬁltered  and  concentrated.  Then  all  the  aqueous  extract  was\nextracted  with  400 ml  ethanol  (75%  and  85%)  twice.  After\nthe ethanol was retrieved, the active fractions were recovered,\nﬁltered  and  lyophilized  (yield  0.28%).  Consequently,  the\nﬁnal  extract  was  used  in  the  study  after  dissolution  in  dis-\ntilled water at 37 °C and stirred for 60 min.\nChemicals Phenylephrine  (PE),  acetylcholine  (ACh),  L-\nN(w)-nitro-arginine  methyl  ester  (L-NAME),  methylthion-\ninium  chloride  (MC),  phosphoramidon,  ethylene  glycol-bis\n[b-aminoethyl  ether]-N,N,N,N-tetraacetic  acid  (EGTA),\ncaffeine  and  phorbol-12,13-diacetate  (PD)  were  purchased\nfrom  Sigma  Chemical  Co.  (St.  Louis,  MO,  U.S.A.).  They\nwere  dissolved\nin  distilled  water  and  diluted  with\nKrebs–Henseleit  (K–H)  solution  before  use.  All  chemicals\nwere of the highest purity available.\nExperiment  Animals The  investigation  conformed  to\nthe  Chinese  Regulation  for  Administration  of  Laboratory\nAnimal  (SSTC  1988/2).  Health  male  Sprague–Dawley  rats\nweighing  260—280 g  (Laboratory  Animal  Center  of  Zhe-\njiang  College  of  Traditional  Chinese  Medicine,  Hangzhou,\nChina) were housed in conventional cages with free access to\nwater and rodent chow at the controlled temperature and hu-\nmidity with a 12-h light/dark cycle.\nPreparation of Thoracic Arterial Rings from Sprague–\nDawley  Rats Rats  were  euthanized  by  decapitation  using\nsmall animal guillotine. The thoracic aorta was immediately\nisolated  and  immersed  in  oxygenated  K–H  solution  4 °C  of\nthe  following  composition  (in  mM):  NaCl  118.3;  KCl  4.7;\nMgSO4 1.2; KH2PO4 1.2; NaHCO3 25.0; CaCl2 2.5; Glucose\n11.0.  Then  the  adherent  connective  tissue  was  cleaned  and\nblood  vessel  was  cut  into  3—4 mm  rings,  with  the  special\ncare  of  avoiding  damage  of  endothelium.  In  some  prepara-\ntions, endothelium was mechanically removed by gently rub-\nbing  the  lumen.  The  rings  were  mounted  horizontally  be-\ntween two stirrups in organ bath ﬁlled with 10 ml K–H solu-\ntion  at  37 °C,  ventilated  continuously  with  95%  O2 and  5%\nCO2. The isometric tension was recorded with a force trans-\nducer  (JZ101,  metrical  range:  0—5.0 g)  and  MedLab  5.0v\nrecording system (Nanjing Medease Science and Technology\nCo. Ltd).\nExperimental  Procedure  of  Isometric  Tension Rings\nwere equilibrated for 60 min at 2.0 g resting tension, and then\nchallenged with KCl (6.0102\nM) at least 3 times until a re-\nproducible maximal contractile response was obtained. After\na  further  equilibration  period  of  30 min,  the  integrity  of  the\nendothelium was assessed in all preparations by determining\nthe ability of ACh (1.0105\nM) to induce more than 80% re-\nlaxation  of  rings  pre-contracted  with  PE  (1.0106\nM).  The\nendothelium  was  considered  to  be  removed  when  there  was\nless than 10% relaxation response to ACh.\nProtocol 1: Effects of A. membranaceus on Vascular Tone:\nThe ﬁrst series of the experiments were conducted to assess\nendothelium-dependent  or  independent  effects  of  A.  mem-\nbranaceus on isolated aortic rings. When the tension was at\nresting  state  or  reached  a  plateau  induced  by  PE  (3.0\n107\nM),  A.  membranaceus (2.0105—2.0101 g/l)  was\ncumulatively added into the organ bath with 10 min interval.\nThe  rings  with  intact  or  denuded  endothelium  were  always\ntested in parallel.\nProtocol  2:  Endothelium-Independent  Vasomotor  Re-\nsponse by A. membranaceus: The second series of the exper-\niments  were  designed  to  determine  the  underlying  mecha-\nnisms  of  A.  membranaceus-induced  vasomotor  response  in\naortic rings without endothelium. In contrast to the response\nof  A.  membranaceus (2.0105—2.0101 g/l)  to  PE,  KCl\n(3.0102\nM)  was  also  employed  to  constrict  the  denuded\naortic rings. A plateau contractile response to KCl or PE was\nalso obtained as a time control.\nM), PD (1.0106\nAdditionally,  after  the  preparations  were  serially  washed\nand equilibrated for 60 min by Ca2-free medium (mM: NaCl\n118.3, KCl 4.7, KH2PO4 1.2, MgSO4 1.2, NaHCO3 25, Glu-\ncose 11.0, EGTA 0.05), PE (3.0107\nM)\n(an  activator  of  protein  kinase  C,  PKC),13) or  caffeine\n(2.5102\nM) (an activator of ryanodine receptor)14) was used\nto contract the denuded rings. In order to explore the proba-\nble  effect  of  extracellular  Ca2 inﬂow,  2.5 mM CaCl2 was\nadded  after  PE  stimulated  the  rings.  Then  the  effect  of  A.\nmembranaceus (6.0103 g/l)  was  tested.  A  time  control\n(same protocol as above except for the omission of A. mem-\nbranaceus to  the  second  application  of  PE,  PD  or  caffeine)\nwas  always  run  in  parallel.  Following  the  initial  control  re-\nsponse in Ca2-free medium and before the second test con-\ntraction (to PE or caffeine), a 6.0102\nM KCl-induced con-\ntraction  in  K–H  solution  was  obtained  to  reﬁll  the  Ca2\nstores in the vascular tissues.\nProtocol 3: Endothelium-Dependent Vasomotor Response\nby A. membranaceus: In the third series of experiments, the\npotential inﬂuence of A. membranaceus on PE-induced vaso-\nmotor function in the intact rings was examined. For pretreat-\nment,  endothelium-intact  rings  were  incubated  with  L-\nNAME (1.0104\nM) (a speciﬁc inhibitor of nitric oxide syn-\nthase), MC (1.0105\nM) (an inhibitor of guanylate cyclase)\nfor 15 min15) and phosphoramidon (5.0106\nM) (an inhibitor\nof  endothelin  converting  enzyme)  for  20 min16) before  ad-\nministration  of  PE.  Then  the  response  curves  of  A.  mem-\nbranaceus were recorded. The rings with or without the treat-\nment of inhibitors were always tested in parallel.\nStatistical  Analysis Relaxant  responses  were  expressed\nas the percentage decreases of the magnitude of the contrac-\ntion  induced  by  PE  or  KCl  (100%)  before  the  application\nof  vasodilators,  and  the  contractile  responses  as  the  ratio  of\nthe  responses  following  and  before  the  application  of  A.\nmembranaceus (second  response/ﬁrst  response).  All  results\nare  expressed  as  meanS.E.M.  Statistical  analysis  was  per-\nformed  with  t-test  or  ANOVA  followed  by  Newman–Keuls\ntest. Differences were accepted as statistically signiﬁcant at p\nvalues 0.05 (GraphPad Prism).\nRESULTS\nPE  (1.0106\nM)  induced  a  similar  sustained  contraction\nof  aortic  rings  in  each  group  with  a  peak  tension  of  about\n3.110.27 g  in  intact  aortic  rings  and  3.020.19 g  in  de-\nnuded ones.\nEffects  of  A.  membranaceus on  the  Vasotension All\ndoses (2.0105—2.0101 g/l) of A. membranaceus exhib-\nited  negligible  vasomotor  actions  on  aortic  rings  with  or\nwithout  endothelium  at  resting  tension  (Fig.  1).  A.  mem-\nbranaceus (2.0105—2.0101 g/l)  produced  dose-depen-\ndent  relaxation  in  endothelium-denuded  rings  (Fig.  2),  after\nthe  plateau  tension  induced  by  PE.  However,  in  PE-precon-\ntracted  endothelium-intact  rings,  low  doses  (2.0104—\n6.0103 g/l) of A. membranaceus showed a more signiﬁcant\nrelaxation than that in denuded ones (Fig. 2), while high dose\n(2.0102—2.0101 g/l) of the extract induced a transient\ncontraction (Figs. 2, 3).\nEndothelium-Independent  Vasodilatation  by  A.  mem-\nIn  contrast  to  the  effect  on  PE-contracted  en-\nbranaceus\ndothelium-denuded rings in K–H solution, A. membranaceus\n(2.0104—2.0101 g/l)  had  no  dilatation-effect  on  rings\nstimulated by KCl (3.0102\nM) (Fig. 4).\nAnd in the experiment with Ca2-free medium, aortic con-\ntraction  elicited  by  PE,  caffeine  or  PD  was  decreased  sub-\nstantially, as these vascular contractions in Ca2-free medium\nare mediated only via the release of Ca2 from the intracellu-\nlar stores or phosphorylation of myosin light chain. A. mem-\nbranaceus (6.0103 g/l)  could  inhibit  the  contraction  in-\nduced  by  PE  (3.0107\nM),  but  not  block  the  extracellular\nCa2 inﬂow,  because  2.5 mM CaCl2 induced  a  similar  en-\nhancement of contraction by PE, compared with the control\n(95.842.63%  vs. 96.961.87%,  p0.05).  Furthermore,  A.\nmembranaceus (6.0103 g/l) failed to induce an attenuation\nof  contractile  response  by  PD  (1.0106\nM)  and  caffeine\n(2.5102\nM) (Fig. 5).\nEndothelium-Dependent  Vasomotor  Effects  by  A.\nmembranaceus Pre-incubation  of  the  intact  rings  with  L-\nM) was added, the effect of A. membranaceus (2.0102,\n6.0102 and 2.0101 g/l, showed by a, b, c point) was obtained. For it was similar with Fig. 2, the effect of A. membranaceus (2.0105—6.0103 g/l) was cut. Values of both\ngroups are expressed as meanS.E.M. of 8 experiments. Contractile responses are expressed as the percentage from maximal contraction elicited by phenylephrine (3.0107\nM).\n∗ p0.05, ∗∗ p0.01, compared with corresponding values before administration.\nNAME  (1.0104\nM)  markedly,  incompletely  inhibited  A.\nmembranaceus (6.0103 g/l)-induced  relaxation  from  36.0\nto 7.3% (Fig. 6). Similarly, MC (1.0105\nM) also showed an\ninhibition  of  endothelium-dependent  vasodilatation  from\n36.0  to  8.0%  (Fig.  6).  Thus,  NO  signaling  might  play  a\nmajor, but not full role in the vasodilatation effect of A. mem-\nbranaceus.\nWhen the dose of A. membranaceus in the organ bath was\nstepwise raised to 2.0102—2.0101 g/l, a transient con-\ntraction  could  be  observed  in  endothelium-intact  rings.  Pre-\ntreatment  of  endothelium-intact  rings  with  phosphoramidon\n(5.0106\nM)  signiﬁcantly  attenuated  the  transient  contrac-\ntion response (Fig. 3).\nDISCUSSION\nThe present study ﬁrst and detailedly explored the in vitro\nvascular effects of A. membranaceus in aortic rings isolated\nfrom rats. In the course of the study, a number of novel ob-\nservations  had  been  made,  which  contribute  to  better  scien-\ntiﬁc  understanding  of  their  therapeutic  use  for  antihyperten-\nsion associated with Qi deﬁciency caused by the disorder of\nspleen and kidney in China.\nFirst, although neurohumoral factors may contribute to an-\ntihypertension for the crude extract of A. membranaceus ob-\nserved previously,8) it is apparent that direct endothelium-de-\npendent  and  independent  vasorelaxation  should  be  promi-\nnently taken into account according to this study.\nSecond,  A.  membranaceus dose-dependently  dilated  en-\ndothelium-denuded  rings  contracted  with  PE.  However,  the\nagent had no effect on the vessel contracted by KCl. The cel-\nlular  mechanism  of  contraction  involved  in  the  response  to\nKCl and PE is different. KCl induce Ca2 inﬂux via voltage-\ndependent  Ca2 channel,  which  further  activate  Ca2-in-\nduced Ca2 release through ryanodine-receptor. PE increase\nintracellular  Ca2 concentration  with  two  mechanisms,  1)\nthrough  receptor-gated  Ca2 channels;  2)  mobilizes  Ca2\nfrom  intracellular  stores  via the  inositol-1,4,5-trisphosphate\n(IP3)  receptor  or  induces  secondly  myosin  light  chain  phos-\nphorylation via activating PKC.17,18) That is, the vasorelaxant\naction of A. membranaceus seems to occur in a receptor-de-\npendent manner in SMVC of rat aortas. However, the result\nthat A. membranaceus failed to inhibit the contraction caused\nby addition of CaCl2 in Ca2 free solution was probably only\ndue to its effects on the intracellular pathways. This hypothe-\nsis was conﬁrmed in the further experiments that the contrac-\ntions induced by PD and caffeine in Ca2-free medium were\nnot  affected  by  A.  membranaceus equally.  Thus,  it  appears\nthat the IP3-induced Ca2 release channels might be the site\nof action for A. membranaceus on endothelium-removed aor-\ntas.\nThird,  NO  is  a  potent  vasodilator  synthesized  in  the  en-\ndothelium19) by  NO  synthase,  and  causes  VSMC  relaxation\nthrough  the  activation  of  soluble  guanylate  cyclase.20) The\npresent  study  demonstrated  that  the  extract  of  A.  mem-\nbranaceus dose-dependently  inhibited  the  contraction  in-\nduced  by  PE  in  intact  aorta  isolated  from  rats.  This  vasore-\nlaxant  action  was  mostly  inhibited  by  treatment  with  L-\nNAME  or  MC.  Endothelium-dependent  relaxation  of  A.\nmembranaceus seemed  to  be  associated  with  NO  signaling\nvia guanylate cyclase activation since both L-NAME and MC\ncould  attenuate  this  response.  This  result  was  supported  by\nother studies which had shown that A. membranaceus could\nincrease the plasma level of NO21) and induce the release of\nNO  from  VSMC  in  vitro  by  NO-soluble  guanylate  cyclase-\ncyclic guanosine monophosphate pathway.22)\nFinally,  it  was  also  observed  that  high  dose  (2.0102—\n2.0101 g/l) A. membranaceus induced a transient contrac-\ntion  in  arterial  rings  pre-constricted  by  PE.  This  result  was\nidentical with that obtained in another report where treatment\nwith  high  dose  of  A.  membranaceus reversibly  increased\nblood pressure of patients without hypertension previously.23)\nThe  mechanism  underlying  is  under  investigation.  Blockage\nof  the  contraction  by  endothelin-converting  enzyme  phos-\nphoramidon  may  suggest  the  involvement  of  an  endothelin\nreceptor mediated signaling.\nIn summary, the ﬁndings indicated that A. membranaceus\nhave  diphasic  effects  of  relaxation  and  contraction  on  the\naorta rings. The mechanisms of relaxation may include inhi-\nbition  of  intracellular  Ca2 release  in  vascular  smooth  mus-\ncle cells, and endothelium dependent relaxation mediated by\nIn the untreatment group, A. membranaceus or vehicle group was not pretreated with\nL-NAME or MC. Each column with a bar represents meanS.E.M. of 8 experiments.\nContractile  responses  are  expressed  as  the  percentage  from  maximal  contraction\nelicited by phenylephrine (3.0107\nM). ∗ p0.05, ∗∗ p0.01, compared with value of\nthe  corresponding  vehicle  group.   p0.01,  compared  with  value  of  the  experiment\nsubgroup in the untreatment group.\nthe NO-GC pathway, whilst the effect of transient contraction\nof A. membranaceus at a high dose was related with endothe-\nlin release.\nAcknowledgements This work was supported by grants\nfrom  Province  Administration  of  Traditional  Chinese  Medi-\ncine,  Zhejiang,  P.R.  China  (Project  Number:  2000C54  and\n2003C078);  Foundation  for  Returnee  from  Personal  Depart-\nment  of  Zhejiang  Province,  P.R.  China  (Project  Number:\nZRZ-2001275);  and  by  E-Institutes  of  Shanghai  Municipal\nEducation  Commission,  P.R.  China  (Project  Number:  E-\n04010). The authors are grateful to Prof. Wenquan Liang of\nthe  Department  of  Pharmacology,  Zhejiang  University,  for\nhis technical help.\n21) He L., Wang R. G., You Z. L., Yang Z. W., Chen X. D., Li J., Zhan X.\nL., Wang J., Acad. J. Hunan Tradit. Chin. Med., 23, 4—6 (2003).\n22) Wang K. H., Foreign Med. Sci.: TCM Section, 18, 38—39 (1996).\n23) Wang  C.  H.,  Zhang  J.  J.,  Shandong  J.  Tradit.  Chin.  Med.,  15,  351",
         "Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,Jian Sun,Q. Xia,K. Bian",
         null
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "In this report, we use zebraﬁsh as a model system to understand the importance of CD146 in vascular\ndevelopment. Endothelial-speciﬁc expression of CD146 was veriﬁed by whole-mount in situ hybridization.\nSuppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly\ndeveloped intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood ﬂow through the\nISV region, despite that ﬂuorescence microangiography showed that the ISVs were present. This ﬁnding\nsuggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally,\nremodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of\nCD146 protein expression resulted in a circulation shunt that caused the circulation to by-pass part of the\ncaudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino\noligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N\nterminus deleted in an in-frame manner. This region, therefore, is likely to contain elements critical for\nCD146 function. This study provides the ﬁrst in vivo functional assessment of CD146 in embryonic\ndevelopment by showing that knockdown of CD146 protein expression severely hinders vascular\ndevelopment in zebraﬁsh. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley-Liss, Inc.",
         "INTRODUCTION\nCD146, also known as MUC18, A32,\nMel-CAM, and S-Endo1 (Lehmann et\nal., 1989; Shih, 1999), is a transmem-\nbrane glycoprotein that belongs to the\nimmunoglobulin superfamily with a\ncharacteristic V-V-C2-C2-C2 domain\n(Williams and Barclay,\nstructure\n1988; Sers et al., 1993; Shih, 1999).\nAlthough it was ﬁrst identiﬁed as a\nhuman melanoma-associated antigen\n(Holzmann et al., 1987; Lehmann et\nal., 1987), this protein is also highly\nexpressed in the endothelium (Bardin\net al., 1996a,b). In fact, it was deter-\nmined that CD146 was the antigen\nrecognized by a highly endothelial-\nspeciﬁc monoclonal antibody called\nP1H12 (Solovey et al., 2001).\nAlthough its cellular function is rel-\natively poorly understood, CD146 is\nbelieved to play a role in cell– cell ad-\nhesion. It has been shown by immuno-\nhistochemical studies that CD146 is\nlocalized at the intercellular junctions\nof conﬂuent endothelial cells in vitro\n(Bardin et al., 1996a, 2001), although\nCD146 does not appear to form part of\nthe tight junctions (Lampugnani and\nDejana, 1997) nor the adherens junc-\ntions (Bardin et al., 2001). In addition,\nthe cell surface expression of CD146\nwas shown to increase by twofold as\nhuman umbilical vein endothelial\ncells (HUVECs) grew from subconﬂu-\nency to complete conﬂuency (Bardin et\nal., 2001). Furthermore, overexpres-\nsion of CD146 in ﬁbroblasts decreased\nparacellular permeability (Bardin et\nal., 2001), whereas treatment of con-\nﬂuent human microvascular endothe-\nlial cells (HMVECs) with monoclonal\nantibody P1H12 increased permeabil-\nity to albumin in vitro (Solovey et al.,\n2001).\nThe nature of CD146 adhesive prop-\nerty has been addressed in vitro\nmainly through cell-aggregation and\nsolid-phase binding assays. Results\nfrom these experiments suggest that\nCD146 confers cell– cell adhesion by\nmeans of interactions with a hetero-\nphilic ligand that has yet to be identi-\nﬁed (Johnson et al., 1997; Shih et al.,\n1997). This interaction is divalent-cat-\nion independent and can be inhibited\nby a recombinant full-length CD146\nprotein and a polyclonal antiserum\nraised against CD146 (Johnson et al.,\n1997; Shih et al., 1997).\nIn addition to its potential role in\ncell– cell adhesion, CD146 has also\nbeen shown to be a signaling molecule\nthat induces several intracellular mo-\nlecular events upon cross-linking by a\nmonoclonal antibody. For example,\ntreatment of HMVEC with the mono-\nclonal antibody P1H12 caused caveo-\nlin to redistribute from the cell periph-\nery to the cytoplasm and nuclear entry\nof the transcription factor NF-B (So-\nlovey et al., 2001). In addition, anti-\nbody-mediated cross-linking of CD146\nin vitro recruited Fyn to the cytoplas-\nmic domain of CD146, triggered a\nphospholipase C – dependent store-\noperated calcium mobilization and in-\nduced phosphorylation of cellular pro-\nteins such as Pyk2, FAK, CAS, and\npaxillin (Anfosso et al., 1998, 2001;\nSolovey et al., 2001). However, CD146\nhas not been shown to interact with\nother cell-surface adhesive molecules\nsuch as VE-cadherin, PECAM-1, or 1\nintegrin (Bardin et al., 2001).\nRecently, a fully humanized CD146\nmonoclonal antibody ABX-MA1 (Mills\net al., 2002) and a mouse monoclonal\nantibody AA98 (Yan et al., 2003) were\ndeveloped. When tested in mouse tu-\nmor models, both antibodies inhibited\ntumor growth and metastasis and re-\nduced microvascular density in tu-\nmors (Mills et al., 2002; Yan et al.,\n2003). Of interest, the antibody AA98\nwas shown to bind only to stimulated\nbut not serum-starved HUVECs in\nvitro (Yan et al., 2003). Most impor-\ntantly, this antibody was shown to\nrecognize an epitope in CD146 that\nwas only present in tumor endothe-\nin endothelial cells\nlium but not\npresent in normal tissues (Yan et al.,\n2003). AA98 monoclonal antibody was\nshown to inhibit proliferation and mi-\ngration of HUVECs in vitro and sig-\nniﬁcantly\nin\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003). Collectively,\nthese studies suggest\nthat CD146\nplays a signiﬁcant role in blood vessel\ndevelopment as well as in solid tumor\ngrowth.\nThe ease of manipulation makes the\nzebraﬁsh system ideal to identify en-\ndothelial speciﬁc genes. Speciﬁcally,\nthe antisense morpholino-oligonucleo-\ntide technology (Heasman et al., 2000;\nNasevicius and Ekker, 2000) provides\na rapid way to suppress protein ex-\npression to address the functionality\nof a speciﬁc gene in a developmental\ncontext. One of our goals is to use this\nmodel to rapidly characterize endo-\nthelial-speciﬁc genes with unknown in\nvivo functions. We chose CD146 as our\nﬁrst target gene, because a CD146 ge-\nnetic knockout mouse is not available\nto date. Here, we report the use of the\nzebraﬁsh system to understand the\npotential role of CD146 in vascular\ndevelopment.\nRESULTS\nCloning of the Zebraﬁsh\nCD146 Gene",
         "A cDNA clone (clone number, 3512;\nGenBank accession no. BG985668) re-\nported in the zebraﬁsh cDNA data-\nbase\n(http://zf.nichd.nih.gov/pubzf/)\nshows a high level of sequence iden-\ntity to mouse and chicken CD146 at\nthe RNA and protein levels, respec-\ntively (Kudoh et al., 2001). Therefore,\nwe performed 5 and 3 rapid ampliﬁ-\ncation of cDNA ends (RACEs) based\non this sequence information to clone\nthe full-length zebraﬁsh CD146 (Fig.\n1).\nThe transcript is approximately 2.3\nkb long, harboring an open reading\nframe that encodes a 617-amino-acid\nprotein. The 5 untranslated region is\nrelatively short (approximately 73 nu-\ncleotides), similar to that of the tran-\nscript of human origin (26 nucleotides;\nSers et al., 1993). The 3 untranslated\nregion is approximately 350 nucleo-\ntides long and contains a canonical\npolyadenylation\nsequence\nAAUAAA just 5 to the poly-A tail\n(Fig. 1). When this mRNA sequence\nwas compared with the zebraﬁsh\ngenomic sequence, a 200-kb genomic\nfragment (ctg24502.1) was identiﬁed\nto contain the ﬁrst 13 predicted exons\nof CD146 (not shown). This genomic\nfragment covers nucleotides 1 to 1685\nof the mRNA. A similar search was\nperformed at the ENSEMBL Web site\n(http://www.ensembl.org/Danio_rerio/\nblastview). The search result indi-\ncates that CD146 is located on chro-\nmosome 19 of the zebraﬁsh genome.\nThe zebraﬁsh CD146 protein shows\napproximately 30% overall sequence\nidentity to that of the human, avian,\nmouse, and rat orthologs (Fig. 2A). Of\ninterest, the transmembrane and the\ncytoplasmic domains are the most con-\nserved regions. When the protein se-\nquence was scanned for protein motifs\n(at http://scansite.mit.edu/; Yaffe et\nal., 2001), ﬁve immunoglobulin do-\nmains (Ig1, K35 to V123; Ig2, C168 to\nV232; Ig3, G273 to S324; Ig4, G359 to\nV408; Ig5, N445 to A507) were identi-\nﬁed in the extracellular region of the\nprotein, consistent with the character-\nistic Ig domain structure present in\nthe subgroup of immunoglobulin su-\nperfamily that CD146 belongs to\n(Johnson et al., 1996). The two puta-\ntive PKC phosphorylation sequences\n(S593GK and S614DK) in human (Alais\net al., 2001) are also conserved in the\nzebraﬁsh\n(S575GK and\nS596GK). The result of a phylogenetic\nanalysis is shown in Figure 2B.\nprotein\nTransient expression of zebraﬁsh\nCD146 tagged with a ﬂuorescent pro-\ntein at the C terminus in 293 cells\nshowed cell surface expression pat-\ntern similar to that of transiently ex-\npressed human CD146/YFP (data not\nshown). In addition, the ﬁrst 11 amino\nacids of zebraﬁsh CD146 directed the\nsecretion of YFP into the culture me-\ndium when fused to the N terminus of\nthe protein and transiently expressed\nin 293 cells, suggesting that this re-\ngion contains a leader-peptide se-\nquence and that Met1 is the authentic\ninitiation methionine\nnot\nshown). We also tested two commer-\ncially available antibodies against hu-\nman CD146 (a polyclonal antibody\nfrom ZYMED and a monoclonal anti-\nbody from CHEMICON) in Western\nblot analysis and determined that the\nantibodies did not cross-react with the\nzebraﬁsh protein overexpressed in\n293 cells (data not shown).\n(data\nExpression of CD146 in\nZebraﬁsh Embryos Is Highly\nEndothelial Speciﬁc\nNext, CD146 expression pattern in\nzebraﬁsh embryo was examined by\nwhole-mount in situ hybridization\nby using a probe that was derived\nfrom the ﬁrst 0.6 kb of the mRNA\n(Fig. 3A). In a 24-hr-old embryo,\nmRNA of CD146 was detected in the\ndorsal aorta (Fig. 3B, red arrow),\ncaudal artery (Fig. 3B, red arrow-\nhead), caudal vein (Fig. 3B, blue ar-\nrowhead), and the sprouting inter-\nsomitic vessels\n(Fig. 3B, green\narrow). Expression in neuronal cells\nwas also detected (Fig. 3B, black ar-\nrow). This expression pattern is es-\nsentially identical to that reported in\nthe zebraﬁsh cDNA database (http://\nzf.nichd.nih.gov/pubzf/), where the\nwhole-mount in situ experiment was\nperformed using a 0.5-kb probe de-\nrived from the 3 end of the mRNA.\nOf interest, only a very low level of\nCD146 mRNA expression was de-\ntected in the posterior cardinal vein\nat this stage (Fig. 3B, blue arrow).\nThe expression speciﬁcity was best\nillustrated by a cross section of the\nembryo (Fig. 3D). Intense staining of\nthe message was detected in the\naorta (Fig. 3D, red arrow); only a\nvery low level of staining was de-\ntected in the posterior cardinal vein\n(Fig. 3D, blue arrow). In contrast,\nCD146 expression in the posterior car-\ndinal vein was detected at day 2 (Fig.\n3E, blue arrow) and day 3 (data not\nshown). This ﬁnding highlights the\ndynamics of the expression pattern of\nthis endothelial marker during devel-\nopment. mRNA staining of CD146 in\nthe head showed clear endothelial ex-\npression pattern (Fig. 3C). Certain\ncranial vessels (e.g., the middle cere-\nbral vein) were stained, while others\n(e.g., the dorsal ciliary and median\npalatocerebral veins) did not appear\nto express CD146. Some other vascu-\nlar structures stained included the\naortic arches and the heart.\nProtein Expression\nKnockdown of CD146\nInhibits Proper Development\nof Intersomitic Vessels\nTo address the functional\nimpor-\ntance of CD146 in embryonic vascu-\nlar development, we performed pro-\ntein expression knockdown exper-\niments by using antisense MOs. The\nﬁrst MO (splice blocker 1) was de-\nsigned to target the seven nucleo-\ntides at the 3 end of exon 10 and the\nﬁrst 18 nucleotides of intron 10 (Fig.\n4A). Microinjection of splice blocker\n1 into the embryos resulted in aber-\nrant splicing. When total RNA har-\nvested from control MO-injected em-\nbryos was used as the template for\nreverse\ntranscriptase-polymerase\nchain reaction (RT-PCR), a speciﬁc\nband of 300 bp was ampliﬁed (Fig.\n5A, lane 1). The identity of this PCR\nproduct was conﬁrmed to be wild-\ntype CD146 by DNA sequencing\n(Fig. 6A). When the same analysis\nwas performed using total RNA har-\nvested from splice blocker 1–injected\nembryos, a PCR product with the\nsize 300 bp was obtained (Fig. 5A,\nlanes 3 and 5). DNA sequencing re-\nvealed that this product was derived\nfrom an aberrantly spliced CD146\nmRNA with nucleotides 1260 to 1282\ndeleted (Fig. 6B). Apparently, with\nA subpopulation of embryos injected\nwith the MO showed different de-\ngrees of curvature at the trunk/tail\nregion. An example is shown in\nFigure 7D. However, there did not\nappear to be a correlation between\nvascular defects and the body curva-\nture.\nReduced Protein Expression\nof CD146 Causes Disruption\nof Caudal Plexus\nReorganization\nAdditional vascular defects were also\ndetected in CD146 morphants. For ex-\nample, the angiogenic remodeling pro-\ncess of the caudal plexus into func-\ntional vascular tubes was complete at\nday 3 in the control MO-injected ﬁsh\n(Movie 2A). Blood could be seen trav-\neling through the entire length of the\nembryo by means of the dorsal aorta,\nthe caudal artery, the caudal vein, and\nthe posterior cardinal vein. However,\nthis vascular reorganization of the\nplexus appeared to be defective in the\nsplice blocker 1–injected embryos\n(Supplementary Movie 2B). The cau-\ndal plexus failed to completely re-\nmodel and organize into functional\ntubes. There was a circulation shunt\nthat caused the circulation to by-pass\nmuch of the caudal artery/vein sys-\ntem. A stagnant pool of blood cells ac-\ncumulated at the tail. Although no cir-\nculation was detected, these trapped\ncells appeared to exhibit slight pulsa-\ntile motion, suggesting that the cau-\ndal plexus might not be developed and\norganized properly for circulation but\nconnected enough to allow hemody-\nnamic forces to reach the tail region.\nCD146 Appears To Be\nImportant in Formation of\nVessels With Proper Lumen\nSize\nTo investigate the integrity of the vas-\nculature, we performed microangiog-\nraphy to detect lumenized vessels.\nEmbryos at 3 dpf were injected with\nred ﬂuorescent microbeads with an\naverage diameter of 0.02 m. The re-\nsults are shown in Figure 8. In control\nMO-injected ﬁsh, vascular develop-\nment as seen by microangiography ap-\npeared to be normal. In contrast, the\noverall vessel integrity of the splice\nthe predicted splice donor site of in-\ntron 10 blocked by the morpholino\noligonucleotide (see Fig. 4A), G1260\nof the mRNA was used as an alter-\nnate, cryptic splice donor site (Fig.\n6B),\naberrantly\nspliced mRNA harboring a prema-\nture stop codon 186 amino acids N\nterminal to the predicted transmem-\nbrane domain (Fig. 1, deleted amino\nacids underlined and the premature\nstop codon in bold and italic).\nan\nTo conﬁrm the effect of splice\nblocker 1 on CD146 mRNA splicing,\nanother set of PCRs was performed in\ntriplicate by using a pair of isoform-\nspeciﬁc primers. This primer set was\ndesigned to detect\nthe full-length,\nwild-type CD146 cDNA speciﬁcally.\nAs shown in Figure 5B, the full-length\nCD146 was only detected in the con-\ntrol MO injected but not the splice\nblocker 1–injected embryos. As con-\ntrol, detection of another endothelial\nspeciﬁc gene ﬂi-1 was also shown.\nTo ascertain the role of CD146 in\nvessel development, blood circulation\nof embryos at 3 days postfertilization\n(dpf) was monitored (Fig. 7, boxed re-\ngions). Control injected ﬁsh showed\nnormal circulation. Blood cells were\nseen traveling through the length of\nthe embryo by means of the artery/\nvein system (Supplementary Movie\n1A, which is available online at www.\ninterscience.wiley.com/jpages/1058-\n8388/suppmat). Importantly, circula-\ntion was also detected throughout the\nintersomitic vessels, indicating that\nthese vessels were functional at this\nstage of development. In contrast, cir-\nculation was abnormal in embryos in-\njected with splice blocker 1. Although\nblood cells were seen traveling in the\nartery and vein, little or no circulation\nwas detected in the intersomitic ves-\nsels (Supplementary Movie 1B). This\nobservation suggests that suppression\nof CD146 protein expression blocked\nthe development of the intersomitic\nvessels. This effect was tabulated, and\nthe results are shown in Table 1. We\ndecided to score the vessel develop-\nment of the embryos by the percent-\nage of functional intersomitic vessels\npresent at 3 dpf. This titration exper-\niment was done multiple times in\nsmall-scale pilot experiments, and a\nsimilar inhibition pattern was consis-\ntently observed. Next, we chose to in-\nject the embryos with either 1.2 pmol\ncontrol MO or 0.8 pmol splice blocker\n1 in three additional independent ex-\nperiments. The results are summa-\nrized in Figure 7C. Splice blocker 1\nclearly had a deleterious effect on\nthe development of intersomitic ves-\nsels. The deleted amino acids, there-\nfore, are critical to CD146 functions.\nBecause the transmembrane and cy-\ntoplasmic domains of CD146 were\ndeleted in the morphants, intracellu-\nlar signaling events triggered by\nCD146 were presumably abolished.\nblocker 1–injected ﬁsh appeared to be\ncompromised. For example, distinct\ncranial vessels were seen in the con-\ntrol-injected ﬁsh (Fig. 8A). In contrast,\nthe cranial vessels of\nthe splice\nblocker 1-injected embryo appeared to\nform less-distinct tubular structures\n(Fig. 8B). In particular, a group of ves-\nsels called the cranial division of the\ninternal carotid was less organized\ncompared with the control\n(arrow-\nheads in Fig. 8A,B). Second, the inter-\nsomitic vessels were narrower com-\npared with the control, even though\nthese vessels were detected by mi-\ncroangiography. A movie clip of the\ncirculation again revealed that the de-\nvelopment of intersomitic vessels was\n(Supplementary\nseverely\nMovie 3). However, these ISVs were\nlumenized as seen by ﬂuorescence mi-\ncroangiography (Fig. 8D). A similar\ndefect was observed in the caudal\nplexus region. When comparing the\nmicroangiogram and the blood circu-\nlation of the splice blocker–injected\nembryos, we observed that blood did\nnot ﬂow through part of the lumenized\ncaudal artery and vein vessels (corre-\nsponded to the region marked be-\ntween the arrowhead and the arrow in\nFig. 8D), resulting in a circulation\nshunt. This ﬁnding suggests that the\nlumen at the site where the circula-\ntion shunt occurred was too narrow\nfor blood cells to pass through. Our\nobservations suggest that CD146 may\nbe important in formation of vessels\nwith proper lumen size.\nElements Critical to CD146\nFunction Are Present at the\nN-Terminus of the Protein\nTo validate our data with splice\nblocker 1, we designed a second an-\ntisense morpholino oligonucleotide\nto target the 3 end of exon 2 and the\npredicted GT splice donor site of in-\ntron 2 (Fig. 4). When this MO was\ndelivered into the embryos by micro-\ninjection,\nit also caused aberrant\nmRNA splicing. RT-PCR analyses\nwere performed with total RNA har-\nvested from control MO- and splice\nblocker 2–injected embryos. The con-\ntrol MO did not affect splicing at the\nexon2/intron2 junction of CD146\npre-mRNA (Fig. 6C). However, se-\nquencing data revealed that splice\nblocker 2 inhibited normal splicing\nbetween exons 2 and 3 (Fig. 6D).\nWith the predicted splice donor site\nof\nthe\nintron 2 blocked, G94 of\nmRNA was used as an alternate,\ncryptic splice donor site. This aber-\nrant splicing event yielded a mutant\nCD146 with an in-frame deletion of\n26 amino acids (V32 to T57) at the N\nterminus (Fig. 1, deleted region in\nbold).\nDeletion of amino acids V32 to T57\nﬁsh, no circulation was seen in the\nintersomitic vessels (Supplementary\nMovie 4B and Table 2), a phenotype\nvery similar to that generated by\nsplice blocker 1–induced CD146 pro-\ntein expression knockdown. There-\nfore, these two MOs appeared to have\nphenocopied each other. Furthermore,\nthis result strongly suggests that the\nregion between V32 and T57 is likely\nto contain elements critical for CD146\nfunction.\nsimilarity\nIn this report, we cloned and charac-\nterized the CD146 ortholog in ze-\nbraﬁsh. The overall amino acid se-\nquence\nthe\nzebraﬁsh and human proteins was ap-\nproximately 30%. Whole-mount\nin\nsitu hybridization results indicate\nthat the CD146 has a dynamic expres-\nsion pattern in the vascular tree dur-\ning development. At 1 dpf, CD146\ntranscript is detected in the dorsal\naorta, caudal artery, and the inter-\nsomitic vessels. In the venous system,\nhowever, CD146 is detected only in\nthe caudal vein but not in the poste-\nrior cardinal vein at this stage (Fig.\n3B). However, CD146 began to ex-\npress in the posterior cardinal vein at\n2 dpf (Fig. 3E). The functional impor-\ntance of the dynamic expression of\nCD146 in the venous system is yet to\nbe understood. CD146 is also ex-\npressed in the heart of developing ze-\nbraﬁsh embryos and may play a role\nin cardiac development. We occasion-\nally observed pericardial edema in the\nmorphants, but there did not seem to\nbe a correlation between this pheno-\ntype and the vascular defects de-\nscribed here. Future studies will in-\nclude the analysis of the relationship\nand\nbetween heart\nCD146 functions. In this report, we\nfocused on the vascular functions of\nCD146 primarily because of the ap-\nparent roles CD146 plays in tumor an-\ngiogenesis (Mills et al., 2002; Yan et\nal., 2003).\ndevelopment\nWhen the protein expression of\nCD146 was knocked down, vascular\ndevelopment in the zebraﬁsh embryo\nwas severely affected. One most prom-\ninent defect was the lack of blood cir-\nculating in the intersomitic vessels by\n3 dpf (Supplementary Movies 1A and\nB). This lack of circulation was not\nof CD146 also affected the develop-\nment of intersomitic vessels in ze-\nbraﬁsh embryos. At day three, circu-\nlation was normal in embryos injected\nwith the control MO (Supplementary\nMovie 4A). In splice blocker 2–injected\ndue to the absence of intersomitic ves-\nsels per se, because these vessel struc-\ntures were clearly detected by ﬂuores-\ncence microangiography,\nalthough\nmany of them appeared to have un-\neven thickness and stenosis was evi-\ndent even when observed under a ﬂu-\n(Fig. 8B,D).\norescence microscope\nAlthough the lumen size of the inter-\nsomitic vessels in zebraﬁsh has not\nbeen reported in literature, the diam-\neters of artery and vein in zebraﬁsh\nembryos have been measured to be\n9.0 m,\n12.8\nrespectively\n(Fritsche et al., 2000). The use of ﬂu-\norescent microbeads with an average\ndiameter of 0.02 m allowed us to de-\ntect vessels with a diameter 400\ntimes smaller than that of the vein.\nBased on our results, we speculate\nthat, in the absence of CD146, the\nintersomitic vessels cannot develop\ninto functional tubes with a lumen\nopening large enough for blood cells to\npass through.\nand\nThe development of the intersomitic\nvessels is a highly organized and cho-\nreographed process. It was reported\nrecently that formation of the inter-\nsomitic vessels involves three differ-\nent endothelial cell types sprouting\nfrom the dorsal aorta (Childs et al.,\n2002). The ﬁrst two form a T-like\nstructure at the dorsal side of the dor-\nsal aorta and the ventral side of the\ndorsal longitudinal anastomotic ves-\nsel. The third one connects these two\nT structures. Because the intersomitic\nvessels were detected in the mor-\nphants with CD146 protein expres-\nsion suppressed, this ﬁnding suggests\nthat CD146 may play a role in the\nstep(s) after the basic architecture of\nthe intersomitic vessels has been laid\nout by these three types of endothelial\ncells.\nData from our morpholino experi-\nments indicate that CD146 may play\nan important role in the remodeling of\nthe caudal vein plexus into functional\nvascular tubes as well. In the CD146\nmorphants, circulation ﬂowed from\nthe dorsal aorta into the caudal artery\nand returned prematurely into cardi-\nnal vein, creating a circulation shunt\nin the caudal artery/vein system (Sup-\nplementary Movie 2B). This circula-\ntion shunt appears to be distinct from\nthat observed in the zebraﬁsh mutant\nmindbomb (mib) defective in Notch\nsignaling (Lawson et al., 2001). In the\nmib mutant, the circulation shunt oc-\ncurred in the trunk region where cir-\nculation ﬂowed directly from the dor-\nsal aorta into the posterior cardinal\nvein without entering the caudal ar-\ntery/vein system. This circulation by-\npass was speculated to arise from dis-\norganization of the boundary between\nthe dorsal aorta and the posterior car-\ndinal vein based on histological data\n(Lawson et al., 2001). In the CD146\nmorphants described here, blood cir-\nculation entered the caudal artery but\nreturned into the venous system pre-\nmaturely into the caudal vein. How-\never, ﬂuorescence microangiography\nrevealed that the caudal plexus was\nlumenized (Fig. 8D, see region be-\ntween the arrow and the arrowhead).\nTherefore, the caudal shunt present in\nthe CD146 morphants may be a result\nof restricted lumen diameter, a struc-\ntural defect similar to that observed in\nthe intersomitic vessels.\nOur observation that CD146 is im-\nportant in vascular development is in\nagreement with several studies re-\nported in the literature. For example,\nit was shown the CD146 expression\nwas up-regulated in human microvas-\ncular endothelial cells that had under-\ngone tubular morphological changes\non matrigel (Glienke et al., 2000). In\nadditional, the ability of endothelial\ncells to form tubes on matrigel can be\nblocked by a CD146 neutralizing anti-\nbody (Mills et al., 2002). Functional\nsuppression of CD146 by an antibody\ncan also inhibit endothelial cell migra-\ntion in vitro and neovascularization in\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003).\nWe have also identiﬁed V32 to T57\nof CD146 to be an important sequence\nelement in its protein function. We\nused an antisense morpholino oligo-\nnucleotide targeting the predicted\nexon 2/intron 2 junction of zebraﬁsh\nCD146 pre-mRNA to create a mutant\nprotein with those 26 amino acids de-\nleted in an in-frame manner. This\ntruncated protein appeared to be non-\nfunctional. This region may contain a\nrecognition epitope for CD146 func-\ntions. Alternatively, the deletion may\nhave affected proper protein folding,\nas the region deleted contains a con-\nserved cysteine residue in the ﬁrst im-\nmunoglobulin domain.\nVascular development requires ex-\ntensive dynamic changes of endothe-\nlial morphology. These dramatic mor-\nphological\nchanges were recorded\nrecently by time-lapse microscopy in a\ndeveloping zebraﬁsh embryo in real-\ntime (Lawson and Weinstein, 2002).\nEndothelial cells were seen to extend\nand retract long ﬁlopodial processes\nduring path ﬁnding. Communication\nappears to occur between these acti-\nvated endothelial cells through ﬁlopo-\ndial contacts, resulting in proper vas-\ncular remodeling and regression to\nensure the development of functional\nlumens\n(Lawson and Weinstein,\n2002). The exact molecular basis of\nthese cellular behaviors is certainly\ncomplex, but highly orchestrated and\ndynamic changes of the cytoskeleton\norganization are probably involved\n(Davis et al., 2002). For example,\nCdc42, Rac1, and integrins are some\nof the signaling molecules intimately\nlinked to cytoskeleton functions that\nhave been shown to be critical in en-\ndothelial tube and lumen formation in\nvitro (Davis and Camarillo, 1996; Bay-\neral integrins that are known to be\nexpressed in endothelial cells (Alais et\nal., 2001; Hodivala-Dilke et al., 2003).\nAll these cellular events appear to be\nrelated to the cytoskeleton. Perhaps,\nCD146 mediates the formation of\nproper vessels through regulating the\ndynamic changes in the organization\nof the cytoskeleton during develop-\nment. Further studies that are de-\nsigned to dissect the intracellular sig-\nnaling events important in CD146\nfunctions are currently under way.\nEXPERIMENTAL\nPROCEDURES\nZebraﬁsh Care\nZebraﬁsh (Danio rerio) was raised and\nmaintained as described (Westerﬁeld,\n2000).\n5 and 3 RACES\nTotal RNA was collected from ze-\nbraﬁsh embryos at 48 hours postfertil-\nization (hpf) by using either the\nRNAaqueous-4PCR kit from Ambion\n(Austin, TX) or the RNeasy Mini kit\nfrom Qiagen (Valencia, CA). The 5\nand 3 RACEs were carried out by us-\ning the GeneRacer Kit from Invitro-\ngen (Carlsbad, CA) according to the\nmanufacturer’s protocol. The kit was\nchosen because the transcriptional\nstart site of the mRNA can be ob-\ntained directly. Brieﬂy, 5 RACE was\nperformed by using 5-GGCACAGG-\nGAGTGTCTTCTTGCCTTATA-3\nas\nthe gene-speciﬁc primer for ﬁrst-\nstrand cDNA synthesis. The cDNA\nwas ampliﬁed by nested PCR using\nthe GC-Advantage DNA polymerase\nfrom Clontech (Palo Alto, CA). The\nprimers for the ﬁrst round of PCR\nwere the GeneRacer 5 Primer (pro-\nvided in the GeneRacer Kit) and the\ngene-speciﬁc primer. For the second\nround of ampliﬁcation, the GeneRacer\n5 Nested Primer (provided in the\nGeneRacer Kit) and a primer with the\nsequence 5-GCAGCTTTCCCTTCTT-\nGTAGAGGTAAT-3 were used. The\nPCR products were cloned into the\nTOPO cloning kit from Invitrogen.\nProtein sequence alignment was per-\nhttp://prodes.toulouse.\nformed\ninra.fr/multalin/multalin.html\n(Cor-\npet, 1988). The mRNA sequence has\nbeen deposited in GenBank (accession\nno. AY497061).\nless et al., 2000; Bayless and Davis,\n2002; Davis and Bayless, 2003).\nAlthough the molecular mechanism\nby which CD146 affects proper lumen\nformation observed in this report is at\nthis point unclear, it is tempting to\nspeculate that CD146 may achieve\nsuch function by regulating the cy-\ntoskeleton. It is known that antibody-\nmediated cross-linking of CD146 in\nvitro initiates a cascade of intracellu-\nlar signaling events that\nincludes\nphosphorylation of focal adhesion ki-\nnase, paxillin, and p130CAS (Anfosso\net al., 1998, 2001). Additionally,\nCD146 has been shown to interact\nwith the actin cytoskeleton in HUVEC\n(Bardin et al., 2001). Moreover, the\nCD146 ortholog in chicken has been\nshown to modulate expression of sev-\nIn Situ Hybridization\nNucleotides spanning the 5 untrans-\nlated region and the ﬁrst 592 bases of\nthe open reading frame of zebraﬁsh\nCD146 were ampliﬁed and cloned into\na vector derived from pCR-BluntII-\nTOPO (Invitrogen). To\ngenerate\ndigoxigenin (DIG)-labeled antisense\nprobes, the DNA plasmid was linear-\nized with EcoRI, followed by tran-\nscription using SP6 polymerase and\nthe 10 DIG RNA labeling mix from\nRoche (Indianapolis, IN). In situ hy-\nbridization was then performed ac-\ncording to a published procedure\n(Broadbent and Read, 1999; Read,\n2000).\nMicroinjection and\nFluorescence\nMicroangiography\nTwo antisense (splice blocker 1. 5-\nCAGTTGTAGTGACCCACCTCGCA-\nCG-3; splice blocker 2. 5-GTCTTTT-\nTCTCACCGTAAACCAGAT-3) and a\ncontrol (5-GACTCTATATTGCCTAC-\nCTTTACAG-3) morpholino oligo-\nnucleotides were purchased from\nGene-Tools (Philomath, OR). The\nmorpholino oligonucleotides were in-\njected into the yolk sac of zebraﬁsh\nembryos between one- and four-cell\nstages. Embryos were raised in egg\nwater in the presence of 0.003% (w/v)\n1-phenyl-2-thiourea (Sigma, St. Louis,\nMO) to prevent pigmentation (Wester-\nﬁeld, 2000). To verify successful block-\nage of CD146 pre-mRNA splicing by\nthe morpholino oligonucleotides, total\nRNA was collected from the zebraﬁsh\nembryos at 2 dpf by using the RNeasy\nMini kit from Qiagen. Reverse tran-\nscription was carried out by using ei-\nther Thermoscript or Superscript III\nfrom Invitrogen with the gene-speciﬁc\nprimer (5-GGCACAGGGAGTGTCT-\nTCTTGCCTTATA-3). PCR was then\nperformed by using GC-Advantage\nDNA polymerase (Clontech) and the\nfollowing ampliﬁcation primers: 5-\nGATATCACCTTGACTTGCAACG-\nCATTGTCT-3 and 5-GATTGGG-\nTAGCCTTGTGCATAGCAGGTCAG-3\nfor\nsplice blocker 1 injection;\n5-ATGACCTACACCGCACTGCTGC-\nTCGCGGGA-3 and 5-AGGATC-\nGAGAGGGAAGCTGCCCTTGTAC-\nCA-3 for splice blocker 2 injection. A\npair of isoform speciﬁc primers that am-\npliﬁed only the wild-type CD146 cDNA\n(nucleotides 126 to 1287) has the follow-\ning sequences: 5-TACTTACACTCT-\nGACGGAAGAAGCCAAGAT-3 and\n5-CTTTCCTCGCACGTTTACTTGCAC-\nTGA-3. Primers used to ampliﬁed Fli-1\nwere 5-GAAAATGGACGGAACTAT-\nTAAGGAGGCGCT-3 and 5-TCTTC-\nGAGTGCAGTTCAAGTTTTGGCATT-\nTAAGGGT-3.\nBlood circulation in anesthetized\nﬁsh embryos was recorded as time-\nlapse images using an inverted phase\ncontrast microscope (Nikon Corpora-\ntion, Tokyo, Japan), a Spot RT cam-\nera, and the Advanced SPOT software\n(Diagnostic Instruments, Inc., Ster-\nling Heights, MI). The images were\nconverted into QuickTime movies us-\ning a program called ImageJ.\nmicroangiography\nwas performed by using the red ﬂuo-\ncarboxylate-modiﬁed Fluo-\nrescent\nSpheres (0.02 m)\nfrom Molecular\nProbes (Eugene, OR) as described\n(Weinstein et al., 1995). Fluorescence\nmicroangiograms were recorded with\nthe Nikon ﬂuorescence microscope us-\ning a ﬁlter set (41035:ex. 546; em. 605)\nfrom Chroma (Rockingham, VT).\nACKNOWLEGMENTS\nThis study was supported by seed\nfunds from Beth Israel Deaconess\nMedical Center (Boston, MA).\nREFERENCES\nAnfosso F, Bardin N, Vivier E, Sabatier F,\nSampol J, Dignat-George F. 2001. Out-\nside-in signaling pathway linked to\nCD146 engagement in human endothe-\nlial cells. J Biol Chem 276:1564 –1569.\nBardin N, Frances V, Lesaule G,\nHorschowski N, George F, Sampol J.\n1996a. Identiﬁcation of the S-Endo 1 en-\ndothelial-associated antigen. Biochem\nBiophys Res Commun 218:210 –216.\nBardin N, George F, Mutin M, Brisson C,\nHorschowski N, Frances V, Lesaule G,\nSampol J. 1996b. S-Endo 1, a pan-endo-\nthelial monoclonal antibody recognizing\na novel human endothelial antigen. Tis-\nsue Antigens 48:531–539.",
         "",
         "Total RNA was collected from ze-\nbraﬁsh embryos at 48 hours postfertil-\nization (hpf) by using either the\nRNAaqueous-4PCR kit from Ambion\n(Austin, TX) or the RNeasy Mini kit\nfrom Qiagen (Valencia, CA). The 5\nand 3 RACEs were carried out by us-\ning the GeneRacer Kit from Invitro-\ngen (Carlsbad, CA) according to the\nmanufacturer’s protocol. The kit was\nchosen because the transcriptional\nstart site of the mRNA can be ob-\ntained directly. Brieﬂy, 5 RACE was\nperformed by using 5-GGCACAGG-\nGAGTGTCTTCTTGCCTTATA-3\nas\nthe gene-speciﬁc primer for ﬁrst-\nstrand cDNA synthesis. The cDNA\nwas ampliﬁed by nested PCR using\nthe GC-Advantage DNA polymerase\nfrom Clontech (Palo Alto, CA). The\nprimers for the ﬁrst round of PCR\nwere the GeneRacer 5 Primer (pro-\nvided in the GeneRacer Kit) and the\ngene-speciﬁc primer. For the second\nround of ampliﬁcation, the GeneRacer\n5 Nested Primer (provided in the\nGeneRacer Kit) and a primer with the\nsequence 5-GCAGCTTTCCCTTCTT-\nGTAGAGGTAAT-3 were used. The\nPCR products were cloned into the\nTOPO cloning kit from Invitrogen.\nProtein sequence alignment was per-\nhttp://prodes.toulouse.\nformed\ninra.fr/multalin/multalin.html\n(Cor-\npet, 1988). The mRNA sequence has\nbeen deposited in GenBank (accession\nno. AY497061).\nless et al., 2000; Bayless and Davis,\n2002; Davis and Bayless, 2003).\nAlthough the molecular mechanism\nby which CD146 affects proper lumen\nformation observed in this report is at\nthis point unclear, it is tempting to\nspeculate that CD146 may achieve\nsuch function by regulating the cy-\ntoskeleton. It is known that antibody-\nmediated cross-linking of CD146 in\nvitro initiates a cascade of intracellu-\nlar signaling events that\nincludes\nphosphorylation of focal adhesion ki-\nnase, paxillin, and p130CAS (Anfosso\net al., 1998, 2001). Additionally,\nCD146 has been shown to interact\nwith the actin cytoskeleton in HUVEC\n(Bardin et al., 2001). Moreover, the\nCD146 ortholog in chicken has been\nshown to modulate expression of sev-\nIn Situ Hybridization\nNucleotides spanning the 5 untrans-\nlated region and the ﬁrst 592 bases of\nthe open reading frame of zebraﬁsh\nCD146 were ampliﬁed and cloned into\na vector derived from pCR-BluntII-\nTOPO (Invitrogen). To\ngenerate\ndigoxigenin (DIG)-labeled antisense\nprobes, the DNA plasmid was linear-\nized with EcoRI, followed by tran-\nscription using SP6 polymerase and\nthe 10 DIG RNA labeling mix from\nRoche (Indianapolis, IN). In situ hy-\nbridization was then performed ac-\ncording to a published procedure\n(Broadbent and Read, 1999; Read,\n2000).\nMicroinjection and\nFluorescence\nMicroangiography\nTwo antisense (splice blocker 1. 5-\nCAGTTGTAGTGACCCACCTCGCA-\nCG-3; splice blocker 2. 5-GTCTTTT-\nTCTCACCGTAAACCAGAT-3) and a\ncontrol (5-GACTCTATATTGCCTAC-\nCTTTACAG-3) morpholino oligo-\nnucleotides were purchased from\nGene-Tools (Philomath, OR). The\nmorpholino oligonucleotides were in-\njected into the yolk sac of zebraﬁsh\nembryos between one- and four-cell\nstages. Embryos were raised in egg\nwater in the presence of 0.003% (w/v)\n1-phenyl-2-thiourea (Sigma, St. Louis,\nMO) to prevent pigmentation (Wester-\nﬁeld, 2000). To verify successful block-\nage of CD146 pre-mRNA splicing by\nthe morpholino oligonucleotides, total\nRNA was collected from the zebraﬁsh\nembryos at 2 dpf by using the RNeasy\nMini kit from Qiagen. Reverse tran-\nscription was carried out by using ei-\nther Thermoscript or Superscript III\nfrom Invitrogen with the gene-speciﬁc\nprimer (5-GGCACAGGGAGTGTCT-\nTCTTGCCTTATA-3). PCR was then\nperformed by using GC-Advantage\nDNA polymerase (Clontech) and the\nfollowing ampliﬁcation primers: 5-\nGATATCACCTTGACTTGCAACG-\nCATTGTCT-3 and 5-GATTGGG-\nTAGCCTTGTGCATAGCAGGTCAG-3\nfor\nsplice blocker 1 injection;\n5-ATGACCTACACCGCACTGCTGC-\nTCGCGGGA-3 and 5-AGGATC-\nGAGAGGGAAGCTGCCCTTGTAC-\nCA-3 for splice blocker 2 injection. A\npair of isoform speciﬁc primers that am-\npliﬁed only the wild-type CD146 cDNA\n(nucleotides 126 to 1287) has the follow-\ning sequences: 5-TACTTACACTCT-\nGACGGAAGAAGCCAAGAT-3 and\n5-CTTTCCTCGCACGTTTACTTGCAC-\nTGA-3. Primers used to ampliﬁed Fli-1\nwere 5-GAAAATGGACGGAACTAT-\nTAAGGAGGCGCT-3 and 5-TCTTC-\nGAGTGCAGTTCAAGTTTTGGCATT-\nTAAGGGT-3.\nBlood circulation in anesthetized\nﬁsh embryos was recorded as time-\nlapse images using an inverted phase\ncontrast microscope (Nikon Corpora-\ntion, Tokyo, Japan), a Spot RT cam-\nera, and the Advanced SPOT software\n(Diagnostic Instruments, Inc., Ster-\nling Heights, MI). The images were\nconverted into QuickTime movies us-\ning a program called ImageJ.\nmicroangiography\nwas performed by using the red ﬂuo-\ncarboxylate-modiﬁed Fluo-\nrescent\nSpheres (0.02 m)\nfrom Molecular\nProbes (Eugene, OR) as described\n(Weinstein et al., 1995). Fluorescence\nmicroangiograms were recorded with\nthe Nikon ﬂuorescence microscope us-\ning a ﬁlter set (41035:ex. 546; em. 605)\nfrom Chroma (Rockingham, VT).\nACKNOWLEGMENTS\nThis study was supported by seed\nfunds from Beth Israel Deaconess\nMedical Center (Boston, MA).\nREFERENCES\nAnfosso F, Bardin N, Vivier E, Sabatier F,\nSampol J, Dignat-George F. 2001. Out-\nside-in signaling pathway linked to\nCD146 engagement in human endothe-\nlial cells. J Biol Chem 276:1564 –1569.\nBardin N, Frances V, Lesaule G,\nHorschowski N, George F, Sampol J.\n1996a. Identiﬁcation of the S-Endo 1 en-\ndothelial-associated antigen. Biochem\nBiophys Res Commun 218:210 –216.\nBardin N, George F, Mutin M, Brisson C,\nHorschowski N, Frances V, Lesaule G,\nSampol J. 1996b. S-Endo 1, a pan-endo-\nthelial monoclonal antibody recognizing\na novel human endothelial antigen. Tis-\nsue Antigens 48:531–539.",
         "In this report, we use zebraﬁsh as a model system to understand the importance of CD146 in vascular\ndevelopment. Endothelial-speciﬁc expression of CD146 was veriﬁed by whole-mount in situ hybridization.\nSuppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly\ndeveloped intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood ﬂow through the\nISV region, despite that ﬂuorescence microangiography showed that the ISVs were present. This ﬁnding\nsuggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally,\nremodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of\nCD146 protein expression resulted in a circulation shunt that caused the circulation to by-pass part of the\ncaudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino\noligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N\nterminus deleted in an in-frame manner. This region, therefore, is likely to contain elements critical for\nCD146 function. This study provides the ﬁrst in vivo functional assessment of CD146 in embryonic\ndevelopment by showing that knockdown of CD146 protein expression severely hinders vascular\ndevelopment in zebraﬁsh. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley-Liss, Inc.\nINTRODUCTION\nCD146, also known as MUC18, A32,\nMel-CAM, and S-Endo1 (Lehmann et\nal., 1989; Shih, 1999), is a transmem-\nbrane glycoprotein that belongs to the\nimmunoglobulin superfamily with a\ncharacteristic V-V-C2-C2-C2 domain\n(Williams and Barclay,\nstructure\n1988; Sers et al., 1993; Shih, 1999).\nAlthough it was ﬁrst identiﬁed as a\nhuman melanoma-associated antigen\n(Holzmann et al., 1987; Lehmann et\nal., 1987), this protein is also highly\nexpressed in the endothelium (Bardin\net al., 1996a,b). In fact, it was deter-\nmined that CD146 was the antigen\nrecognized by a highly endothelial-\nspeciﬁc monoclonal antibody called\nP1H12 (Solovey et al., 2001).\nAlthough its cellular function is rel-\natively poorly understood, CD146 is\nbelieved to play a role in cell– cell ad-\nhesion. It has been shown by immuno-\nhistochemical studies that CD146 is\nlocalized at the intercellular junctions\nof conﬂuent endothelial cells in vitro\n(Bardin et al., 1996a, 2001), although\nCD146 does not appear to form part of\nthe tight junctions (Lampugnani and\nDejana, 1997) nor the adherens junc-\ntions (Bardin et al., 2001). In addition,\nthe cell surface expression of CD146\nwas shown to increase by twofold as\nhuman umbilical vein endothelial\ncells (HUVECs) grew from subconﬂu-\nency to complete conﬂuency (Bardin et\nal., 2001). Furthermore, overexpres-\nsion of CD146 in ﬁbroblasts decreased\nparacellular permeability (Bardin et\nal., 2001), whereas treatment of con-\nﬂuent human microvascular endothe-\nlial cells (HMVECs) with monoclonal\nantibody P1H12 increased permeabil-\nity to albumin in vitro (Solovey et al.,\n2001).\nThe nature of CD146 adhesive prop-\nerty has been addressed in vitro\nmainly through cell-aggregation and\nsolid-phase binding assays. Results\nfrom these experiments suggest that\nCD146 confers cell– cell adhesion by\nmeans of interactions with a hetero-\nphilic ligand that has yet to be identi-\nﬁed (Johnson et al., 1997; Shih et al.,\n1997). This interaction is divalent-cat-\nion independent and can be inhibited\nby a recombinant full-length CD146\nprotein and a polyclonal antiserum\nraised against CD146 (Johnson et al.,\n1997; Shih et al., 1997).\nIn addition to its potential role in\ncell– cell adhesion, CD146 has also\nbeen shown to be a signaling molecule\nthat induces several intracellular mo-\nlecular events upon cross-linking by a\nmonoclonal antibody. For example,\ntreatment of HMVEC with the mono-\nclonal antibody P1H12 caused caveo-\nlin to redistribute from the cell periph-\nery to the cytoplasm and nuclear entry\nof the transcription factor NF-B (So-\nlovey et al., 2001). In addition, anti-\nbody-mediated cross-linking of CD146\nin vitro recruited Fyn to the cytoplas-\nmic domain of CD146, triggered a\nphospholipase C – dependent store-\noperated calcium mobilization and in-\nduced phosphorylation of cellular pro-\nteins such as Pyk2, FAK, CAS, and\npaxillin (Anfosso et al., 1998, 2001;\nSolovey et al., 2001). However, CD146\nhas not been shown to interact with\nother cell-surface adhesive molecules\nsuch as VE-cadherin, PECAM-1, or 1\nintegrin (Bardin et al., 2001).\nRecently, a fully humanized CD146\nmonoclonal antibody ABX-MA1 (Mills\net al., 2002) and a mouse monoclonal\nantibody AA98 (Yan et al., 2003) were\ndeveloped. When tested in mouse tu-\nmor models, both antibodies inhibited\ntumor growth and metastasis and re-\nduced microvascular density in tu-\nmors (Mills et al., 2002; Yan et al.,\n2003). Of interest, the antibody AA98\nwas shown to bind only to stimulated\nbut not serum-starved HUVECs in\nvitro (Yan et al., 2003). Most impor-\ntantly, this antibody was shown to\nrecognize an epitope in CD146 that\nwas only present in tumor endothe-\nin endothelial cells\nlium but not\npresent in normal tissues (Yan et al.,\n2003). AA98 monoclonal antibody was\nshown to inhibit proliferation and mi-\ngration of HUVECs in vitro and sig-\nniﬁcantly\nin\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003). Collectively,\nthese studies suggest\nthat CD146\nplays a signiﬁcant role in blood vessel\ndevelopment as well as in solid tumor\ngrowth.\nThe ease of manipulation makes the\nzebraﬁsh system ideal to identify en-\ndothelial speciﬁc genes. Speciﬁcally,\nthe antisense morpholino-oligonucleo-\ntide technology (Heasman et al., 2000;\nNasevicius and Ekker, 2000) provides\na rapid way to suppress protein ex-\npression to address the functionality\nof a speciﬁc gene in a developmental\ncontext. One of our goals is to use this\nmodel to rapidly characterize endo-\nthelial-speciﬁc genes with unknown in\nvivo functions. We chose CD146 as our\nﬁrst target gene, because a CD146 ge-\nnetic knockout mouse is not available\nto date. Here, we report the use of the\nzebraﬁsh system to understand the\npotential role of CD146 in vascular\ndevelopment.\nRESULTS\nCloning of the Zebraﬁsh\nCD146 Gene\nA cDNA clone (clone number, 3512;\nGenBank accession no. BG985668) re-\nported in the zebraﬁsh cDNA data-\nbase\n(http://zf.nichd.nih.gov/pubzf/)\nshows a high level of sequence iden-\ntity to mouse and chicken CD146 at\nthe RNA and protein levels, respec-\ntively (Kudoh et al., 2001). Therefore,\nwe performed 5 and 3 rapid ampliﬁ-\ncation of cDNA ends (RACEs) based\non this sequence information to clone\nthe full-length zebraﬁsh CD146 (Fig.\n1).\nThe transcript is approximately 2.3\nkb long, harboring an open reading\nframe that encodes a 617-amino-acid\nprotein. The 5 untranslated region is\nrelatively short (approximately 73 nu-\ncleotides), similar to that of the tran-\nscript of human origin (26 nucleotides;\nSers et al., 1993). The 3 untranslated\nregion is approximately 350 nucleo-\ntides long and contains a canonical\npolyadenylation\nsequence\nAAUAAA just 5 to the poly-A tail\n(Fig. 1). When this mRNA sequence\nwas compared with the zebraﬁsh\ngenomic sequence, a 200-kb genomic\nfragment (ctg24502.1) was identiﬁed\nto contain the ﬁrst 13 predicted exons\nof CD146 (not shown). This genomic\nfragment covers nucleotides 1 to 1685\nof the mRNA. A similar search was\nperformed at the ENSEMBL Web site\n(http://www.ensembl.org/Danio_rerio/\nblastview). The search result indi-\ncates that CD146 is located on chro-\nmosome 19 of the zebraﬁsh genome.\nThe zebraﬁsh CD146 protein shows\napproximately 30% overall sequence\nidentity to that of the human, avian,\nmouse, and rat orthologs (Fig. 2A). Of\ninterest, the transmembrane and the\ncytoplasmic domains are the most con-\nserved regions. When the protein se-\nquence was scanned for protein motifs\n(at http://scansite.mit.edu/; Yaffe et\nal., 2001), ﬁve immunoglobulin do-\nmains (Ig1, K35 to V123; Ig2, C168 to\nV232; Ig3, G273 to S324; Ig4, G359 to\nV408; Ig5, N445 to A507) were identi-\nﬁed in the extracellular region of the\nprotein, consistent with the character-\nistic Ig domain structure present in\nthe subgroup of immunoglobulin su-\nperfamily that CD146 belongs to\n(Johnson et al., 1996). The two puta-\ntive PKC phosphorylation sequences\n(S593GK and S614DK) in human (Alais\net al., 2001) are also conserved in the\nzebraﬁsh\n(S575GK and\nS596GK). The result of a phylogenetic\nanalysis is shown in Figure 2B.\nprotein\nTransient expression of zebraﬁsh\nCD146 tagged with a ﬂuorescent pro-\ntein at the C terminus in 293 cells\nshowed cell surface expression pat-\ntern similar to that of transiently ex-\npressed human CD146/YFP (data not\nshown). In addition, the ﬁrst 11 amino\nacids of zebraﬁsh CD146 directed the\nsecretion of YFP into the culture me-\ndium when fused to the N terminus of\nthe protein and transiently expressed\nin 293 cells, suggesting that this re-\ngion contains a leader-peptide se-\nquence and that Met1 is the authentic\ninitiation methionine\nnot\nshown). We also tested two commer-\ncially available antibodies against hu-\nman CD146 (a polyclonal antibody\nfrom ZYMED and a monoclonal anti-\nbody from CHEMICON) in Western\nblot analysis and determined that the\nantibodies did not cross-react with the\nzebraﬁsh protein overexpressed in\n293 cells (data not shown).\n(data\nExpression of CD146 in\nZebraﬁsh Embryos Is Highly\nEndothelial Speciﬁc\nNext, CD146 expression pattern in\nzebraﬁsh embryo was examined by\nwhole-mount in situ hybridization\nby using a probe that was derived\nfrom the ﬁrst 0.6 kb of the mRNA\n(Fig. 3A). In a 24-hr-old embryo,\nmRNA of CD146 was detected in the\ndorsal aorta (Fig. 3B, red arrow),\ncaudal artery (Fig. 3B, red arrow-\nhead), caudal vein (Fig. 3B, blue ar-\nrowhead), and the sprouting inter-\nsomitic vessels\n(Fig. 3B, green\narrow). Expression in neuronal cells\nwas also detected (Fig. 3B, black ar-\nrow). This expression pattern is es-\nsentially identical to that reported in\nthe zebraﬁsh cDNA database (http://\nzf.nichd.nih.gov/pubzf/), where the\nwhole-mount in situ experiment was\nperformed using a 0.5-kb probe de-\nrived from the 3 end of the mRNA.\nOf interest, only a very low level of\nCD146 mRNA expression was de-\ntected in the posterior cardinal vein\nat this stage (Fig. 3B, blue arrow).\nThe expression speciﬁcity was best\nillustrated by a cross section of the\nembryo (Fig. 3D). Intense staining of\nthe message was detected in the\naorta (Fig. 3D, red arrow); only a\nvery low level of staining was de-\ntected in the posterior cardinal vein\n(Fig. 3D, blue arrow). In contrast,\nCD146 expression in the posterior car-\ndinal vein was detected at day 2 (Fig.\n3E, blue arrow) and day 3 (data not\nshown). This ﬁnding highlights the\ndynamics of the expression pattern of\nthis endothelial marker during devel-\nopment. mRNA staining of CD146 in\nthe head showed clear endothelial ex-\npression pattern (Fig. 3C). Certain\ncranial vessels (e.g., the middle cere-\nbral vein) were stained, while others\n(e.g., the dorsal ciliary and median\npalatocerebral veins) did not appear\nto express CD146. Some other vascu-\nlar structures stained included the\naortic arches and the heart.\nProtein Expression\nKnockdown of CD146\nInhibits Proper Development\nof Intersomitic Vessels\nTo address the functional\nimpor-\ntance of CD146 in embryonic vascu-\nlar development, we performed pro-\ntein expression knockdown exper-\niments by using antisense MOs. The\nﬁrst MO (splice blocker 1) was de-\nsigned to target the seven nucleo-\ntides at the 3 end of exon 10 and the\nﬁrst 18 nucleotides of intron 10 (Fig.\n4A). Microinjection of splice blocker\n1 into the embryos resulted in aber-\nrant splicing. When total RNA har-\nvested from control MO-injected em-\nbryos was used as the template for\nreverse\ntranscriptase-polymerase\nchain reaction (RT-PCR), a speciﬁc\nband of 300 bp was ampliﬁed (Fig.\n5A, lane 1). The identity of this PCR\nproduct was conﬁrmed to be wild-\ntype CD146 by DNA sequencing\n(Fig. 6A). When the same analysis\nwas performed using total RNA har-\nvested from splice blocker 1–injected\nembryos, a PCR product with the\nsize 300 bp was obtained (Fig. 5A,\nlanes 3 and 5). DNA sequencing re-\nvealed that this product was derived\nfrom an aberrantly spliced CD146\nmRNA with nucleotides 1260 to 1282\ndeleted (Fig. 6B). Apparently, with\nA subpopulation of embryos injected\nwith the MO showed different de-\ngrees of curvature at the trunk/tail\nregion. An example is shown in\nFigure 7D. However, there did not\nappear to be a correlation between\nvascular defects and the body curva-\nture.\nReduced Protein Expression\nof CD146 Causes Disruption\nof Caudal Plexus\nReorganization\nAdditional vascular defects were also\ndetected in CD146 morphants. For ex-\nample, the angiogenic remodeling pro-\ncess of the caudal plexus into func-\ntional vascular tubes was complete at\nday 3 in the control MO-injected ﬁsh\n(Movie 2A). Blood could be seen trav-\neling through the entire length of the\nembryo by means of the dorsal aorta,\nthe caudal artery, the caudal vein, and\nthe posterior cardinal vein. However,\nthis vascular reorganization of the\nplexus appeared to be defective in the\nsplice blocker 1–injected embryos\n(Supplementary Movie 2B). The cau-\ndal plexus failed to completely re-\nmodel and organize into functional\ntubes. There was a circulation shunt\nthat caused the circulation to by-pass\nmuch of the caudal artery/vein sys-\ntem. A stagnant pool of blood cells ac-\ncumulated at the tail. Although no cir-\nculation was detected, these trapped\ncells appeared to exhibit slight pulsa-\ntile motion, suggesting that the cau-\ndal plexus might not be developed and\norganized properly for circulation but\nconnected enough to allow hemody-\nnamic forces to reach the tail region.\nCD146 Appears To Be\nImportant in Formation of\nVessels With Proper Lumen\nSize\nTo investigate the integrity of the vas-\nculature, we performed microangiog-\nraphy to detect lumenized vessels.\nEmbryos at 3 dpf were injected with\nred ﬂuorescent microbeads with an\naverage diameter of 0.02 m. The re-\nsults are shown in Figure 8. In control\nMO-injected ﬁsh, vascular develop-\nment as seen by microangiography ap-\npeared to be normal. In contrast, the\noverall vessel integrity of the splice\nthe predicted splice donor site of in-\ntron 10 blocked by the morpholino\noligonucleotide (see Fig. 4A), G1260\nof the mRNA was used as an alter-\nnate, cryptic splice donor site (Fig.\n6B),\naberrantly\nspliced mRNA harboring a prema-\nture stop codon 186 amino acids N\nterminal to the predicted transmem-\nbrane domain (Fig. 1, deleted amino\nacids underlined and the premature\nstop codon in bold and italic).\nan\nTo conﬁrm the effect of splice\nblocker 1 on CD146 mRNA splicing,\nanother set of PCRs was performed in\ntriplicate by using a pair of isoform-\nspeciﬁc primers. This primer set was\ndesigned to detect\nthe full-length,\nwild-type CD146 cDNA speciﬁcally.\nAs shown in Figure 5B, the full-length\nCD146 was only detected in the con-\ntrol MO injected but not the splice\nblocker 1–injected embryos. As con-\ntrol, detection of another endothelial\nspeciﬁc gene ﬂi-1 was also shown.\nTo ascertain the role of CD146 in\nvessel development, blood circulation\nof embryos at 3 days postfertilization\n(dpf) was monitored (Fig. 7, boxed re-\ngions). Control injected ﬁsh showed\nnormal circulation. Blood cells were\nseen traveling through the length of\nthe embryo by means of the artery/\nvein system (Supplementary Movie\n1A, which is available online at www.\ninterscience.wiley.com/jpages/1058-\n8388/suppmat). Importantly, circula-\ntion was also detected throughout the\nintersomitic vessels, indicating that\nthese vessels were functional at this\nstage of development. In contrast, cir-\nculation was abnormal in embryos in-\njected with splice blocker 1. Although\nblood cells were seen traveling in the\nartery and vein, little or no circulation\nwas detected in the intersomitic ves-\nsels (Supplementary Movie 1B). This\nobservation suggests that suppression\nof CD146 protein expression blocked\nthe development of the intersomitic\nvessels. This effect was tabulated, and\nthe results are shown in Table 1. We\ndecided to score the vessel develop-\nment of the embryos by the percent-\nage of functional intersomitic vessels\npresent at 3 dpf. This titration exper-\niment was done multiple times in\nsmall-scale pilot experiments, and a\nsimilar inhibition pattern was consis-\ntently observed. Next, we chose to in-\nject the embryos with either 1.2 pmol\ncontrol MO or 0.8 pmol splice blocker\n1 in three additional independent ex-\nperiments. The results are summa-\nrized in Figure 7C. Splice blocker 1\nclearly had a deleterious effect on\nthe development of intersomitic ves-\nsels. The deleted amino acids, there-\nfore, are critical to CD146 functions.\nBecause the transmembrane and cy-\ntoplasmic domains of CD146 were\ndeleted in the morphants, intracellu-\nlar signaling events triggered by\nCD146 were presumably abolished.\nblocker 1–injected ﬁsh appeared to be\ncompromised. For example, distinct\ncranial vessels were seen in the con-\ntrol-injected ﬁsh (Fig. 8A). In contrast,\nthe cranial vessels of\nthe splice\nblocker 1-injected embryo appeared to\nform less-distinct tubular structures\n(Fig. 8B). In particular, a group of ves-\nsels called the cranial division of the\ninternal carotid was less organized\ncompared with the control\n(arrow-\nheads in Fig. 8A,B). Second, the inter-\nsomitic vessels were narrower com-\npared with the control, even though\nthese vessels were detected by mi-\ncroangiography. A movie clip of the\ncirculation again revealed that the de-\nvelopment of intersomitic vessels was\n(Supplementary\nseverely\nMovie 3). However, these ISVs were\nlumenized as seen by ﬂuorescence mi-\ncroangiography (Fig. 8D). A similar\ndefect was observed in the caudal\nplexus region. When comparing the\nmicroangiogram and the blood circu-\nlation of the splice blocker–injected\nembryos, we observed that blood did\nnot ﬂow through part of the lumenized\ncaudal artery and vein vessels (corre-\nsponded to the region marked be-\ntween the arrowhead and the arrow in\nFig. 8D), resulting in a circulation\nshunt. This ﬁnding suggests that the\nlumen at the site where the circula-\ntion shunt occurred was too narrow\nfor blood cells to pass through. Our\nobservations suggest that CD146 may\nbe important in formation of vessels\nwith proper lumen size.\nElements Critical to CD146\nFunction Are Present at the\nN-Terminus of the Protein\nTo validate our data with splice\nblocker 1, we designed a second an-\ntisense morpholino oligonucleotide\nto target the 3 end of exon 2 and the\npredicted GT splice donor site of in-\ntron 2 (Fig. 4). When this MO was\ndelivered into the embryos by micro-\ninjection,\nit also caused aberrant\nmRNA splicing. RT-PCR analyses\nwere performed with total RNA har-\nvested from control MO- and splice\nblocker 2–injected embryos. The con-\ntrol MO did not affect splicing at the\nexon2/intron2 junction of CD146\npre-mRNA (Fig. 6C). However, se-\nquencing data revealed that splice\nblocker 2 inhibited normal splicing\nbetween exons 2 and 3 (Fig. 6D).\nWith the predicted splice donor site\nof\nthe\nintron 2 blocked, G94 of\nmRNA was used as an alternate,\ncryptic splice donor site. This aber-\nrant splicing event yielded a mutant\nCD146 with an in-frame deletion of\n26 amino acids (V32 to T57) at the N\nterminus (Fig. 1, deleted region in\nbold).\nDeletion of amino acids V32 to T57\nﬁsh, no circulation was seen in the\nintersomitic vessels (Supplementary\nMovie 4B and Table 2), a phenotype\nvery similar to that generated by\nsplice blocker 1–induced CD146 pro-\ntein expression knockdown. There-\nfore, these two MOs appeared to have\nphenocopied each other. Furthermore,\nthis result strongly suggests that the\nregion between V32 and T57 is likely\nto contain elements critical for CD146\nfunction.\nsimilarity\nIn this report, we cloned and charac-\nterized the CD146 ortholog in ze-\nbraﬁsh. The overall amino acid se-\nquence\nthe\nzebraﬁsh and human proteins was ap-\nproximately 30%. Whole-mount\nin\nsitu hybridization results indicate\nthat the CD146 has a dynamic expres-\nsion pattern in the vascular tree dur-\ning development. At 1 dpf, CD146\ntranscript is detected in the dorsal\naorta, caudal artery, and the inter-\nsomitic vessels. In the venous system,\nhowever, CD146 is detected only in\nthe caudal vein but not in the poste-\nrior cardinal vein at this stage (Fig.\n3B). However, CD146 began to ex-\npress in the posterior cardinal vein at\n2 dpf (Fig. 3E). The functional impor-\ntance of the dynamic expression of\nCD146 in the venous system is yet to\nbe understood. CD146 is also ex-\npressed in the heart of developing ze-\nbraﬁsh embryos and may play a role\nin cardiac development. We occasion-\nally observed pericardial edema in the\nmorphants, but there did not seem to\nbe a correlation between this pheno-\ntype and the vascular defects de-\nscribed here. Future studies will in-\nclude the analysis of the relationship\nand\nbetween heart\nCD146 functions. In this report, we\nfocused on the vascular functions of\nCD146 primarily because of the ap-\nparent roles CD146 plays in tumor an-\ngiogenesis (Mills et al., 2002; Yan et\nal., 2003).\ndevelopment\nWhen the protein expression of\nCD146 was knocked down, vascular\ndevelopment in the zebraﬁsh embryo\nwas severely affected. One most prom-\ninent defect was the lack of blood cir-\nculating in the intersomitic vessels by\n3 dpf (Supplementary Movies 1A and\nB). This lack of circulation was not\nof CD146 also affected the develop-\nment of intersomitic vessels in ze-\nbraﬁsh embryos. At day three, circu-\nlation was normal in embryos injected\nwith the control MO (Supplementary\nMovie 4A). In splice blocker 2–injected\ndue to the absence of intersomitic ves-\nsels per se, because these vessel struc-\ntures were clearly detected by ﬂuores-\ncence microangiography,\nalthough\nmany of them appeared to have un-\neven thickness and stenosis was evi-\ndent even when observed under a ﬂu-\n(Fig. 8B,D).\norescence microscope\nAlthough the lumen size of the inter-\nsomitic vessels in zebraﬁsh has not\nbeen reported in literature, the diam-\neters of artery and vein in zebraﬁsh\nembryos have been measured to be\n9.0 m,\n12.8\nrespectively\n(Fritsche et al., 2000). The use of ﬂu-\norescent microbeads with an average\ndiameter of 0.02 m allowed us to de-\ntect vessels with a diameter 400\ntimes smaller than that of the vein.\nBased on our results, we speculate\nthat, in the absence of CD146, the\nintersomitic vessels cannot develop\ninto functional tubes with a lumen\nopening large enough for blood cells to\npass through.\nand\nThe development of the intersomitic\nvessels is a highly organized and cho-\nreographed process. It was reported\nrecently that formation of the inter-\nsomitic vessels involves three differ-\nent endothelial cell types sprouting\nfrom the dorsal aorta (Childs et al.,\n2002). The ﬁrst two form a T-like\nstructure at the dorsal side of the dor-\nsal aorta and the ventral side of the\ndorsal longitudinal anastomotic ves-\nsel. The third one connects these two\nT structures. Because the intersomitic\nvessels were detected in the mor-\nphants with CD146 protein expres-\nsion suppressed, this ﬁnding suggests\nthat CD146 may play a role in the\nstep(s) after the basic architecture of\nthe intersomitic vessels has been laid\nout by these three types of endothelial\ncells.\nData from our morpholino experi-\nments indicate that CD146 may play\nan important role in the remodeling of\nthe caudal vein plexus into functional\nvascular tubes as well. In the CD146\nmorphants, circulation ﬂowed from\nthe dorsal aorta into the caudal artery\nand returned prematurely into cardi-\nnal vein, creating a circulation shunt\nin the caudal artery/vein system (Sup-\nplementary Movie 2B). This circula-\ntion shunt appears to be distinct from\nthat observed in the zebraﬁsh mutant\nmindbomb (mib) defective in Notch\nsignaling (Lawson et al., 2001). In the\nmib mutant, the circulation shunt oc-\ncurred in the trunk region where cir-\nculation ﬂowed directly from the dor-\nsal aorta into the posterior cardinal\nvein without entering the caudal ar-\ntery/vein system. This circulation by-\npass was speculated to arise from dis-\norganization of the boundary between\nthe dorsal aorta and the posterior car-\ndinal vein based on histological data\n(Lawson et al., 2001). In the CD146\nmorphants described here, blood cir-\nculation entered the caudal artery but\nreturned into the venous system pre-\nmaturely into the caudal vein. How-\never, ﬂuorescence microangiography\nrevealed that the caudal plexus was\nlumenized (Fig. 8D, see region be-\ntween the arrow and the arrowhead).\nTherefore, the caudal shunt present in\nthe CD146 morphants may be a result\nof restricted lumen diameter, a struc-\ntural defect similar to that observed in\nthe intersomitic vessels.\nOur observation that CD146 is im-\nportant in vascular development is in\nagreement with several studies re-\nported in the literature. For example,\nit was shown the CD146 expression\nwas up-regulated in human microvas-\ncular endothelial cells that had under-\ngone tubular morphological changes\non matrigel (Glienke et al., 2000). In\nadditional, the ability of endothelial\ncells to form tubes on matrigel can be\nblocked by a CD146 neutralizing anti-\nbody (Mills et al., 2002). Functional\nsuppression of CD146 by an antibody\ncan also inhibit endothelial cell migra-\ntion in vitro and neovascularization in\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003).\nWe have also identiﬁed V32 to T57\nof CD146 to be an important sequence\nelement in its protein function. We\nused an antisense morpholino oligo-\nnucleotide targeting the predicted\nexon 2/intron 2 junction of zebraﬁsh\nCD146 pre-mRNA to create a mutant\nprotein with those 26 amino acids de-\nleted in an in-frame manner. This\ntruncated protein appeared to be non-\nfunctional. This region may contain a\nrecognition epitope for CD146 func-\ntions. Alternatively, the deletion may\nhave affected proper protein folding,\nas the region deleted contains a con-\nserved cysteine residue in the ﬁrst im-\nmunoglobulin domain.\nVascular development requires ex-\ntensive dynamic changes of endothe-\nlial morphology. These dramatic mor-\nphological\nchanges were recorded\nrecently by time-lapse microscopy in a\ndeveloping zebraﬁsh embryo in real-\ntime (Lawson and Weinstein, 2002).\nEndothelial cells were seen to extend\nand retract long ﬁlopodial processes\nduring path ﬁnding. Communication\nappears to occur between these acti-\nvated endothelial cells through ﬁlopo-\ndial contacts, resulting in proper vas-\ncular remodeling and regression to\nensure the development of functional\nlumens\n(Lawson and Weinstein,\n2002). The exact molecular basis of\nthese cellular behaviors is certainly\ncomplex, but highly orchestrated and\ndynamic changes of the cytoskeleton\norganization are probably involved\n(Davis et al., 2002). For example,\nCdc42, Rac1, and integrins are some\nof the signaling molecules intimately\nlinked to cytoskeleton functions that\nhave been shown to be critical in en-\ndothelial tube and lumen formation in\nvitro (Davis and Camarillo, 1996; Bay-\neral integrins that are known to be\nexpressed in endothelial cells (Alais et\nal., 2001; Hodivala-Dilke et al., 2003).\nAll these cellular events appear to be\nrelated to the cytoskeleton. Perhaps,\nCD146 mediates the formation of\nproper vessels through regulating the\ndynamic changes in the organization\nof the cytoskeleton during develop-\nment. Further studies that are de-\nsigned to dissect the intracellular sig-\nnaling events important in CD146\nfunctions are currently under way.\nEXPERIMENTAL\nPROCEDURES\nZebraﬁsh Care\nZebraﬁsh (Danio rerio) was raised and\nmaintained as described (Westerﬁeld,\n2000).\n5 and 3 RACES\nTotal RNA was collected from ze-\nbraﬁsh embryos at 48 hours postfertil-\nization (hpf) by using either the\nRNAaqueous-4PCR kit from Ambion\n(Austin, TX) or the RNeasy Mini kit\nfrom Qiagen (Valencia, CA). The 5\nand 3 RACEs were carried out by us-\ning the GeneRacer Kit from Invitro-\ngen (Carlsbad, CA) according to the\nmanufacturer’s protocol. The kit was\nchosen because the transcriptional\nstart site of the mRNA can be ob-\ntained directly. Brieﬂy, 5 RACE was\nperformed by using 5-GGCACAGG-\nGAGTGTCTTCTTGCCTTATA-3\nas\nthe gene-speciﬁc primer for ﬁrst-\nstrand cDNA synthesis. The cDNA\nwas ampliﬁed by nested PCR using\nthe GC-Advantage DNA polymerase\nfrom Clontech (Palo Alto, CA). The\nprimers for the ﬁrst round of PCR\nwere the GeneRacer 5 Primer (pro-\nvided in the GeneRacer Kit) and the\ngene-speciﬁc primer. For the second\nround of ampliﬁcation, the GeneRacer\n5 Nested Primer (provided in the\nGeneRacer Kit) and a primer with the\nsequence 5-GCAGCTTTCCCTTCTT-\nGTAGAGGTAAT-3 were used. The\nPCR products were cloned into the\nTOPO cloning kit from Invitrogen.\nProtein sequence alignment was per-\nhttp://prodes.toulouse.\nformed\ninra.fr/multalin/multalin.html\n(Cor-\npet, 1988). The mRNA sequence has\nbeen deposited in GenBank (accession\nno. AY497061).\nless et al., 2000; Bayless and Davis,\n2002; Davis and Bayless, 2003).\nAlthough the molecular mechanism\nby which CD146 affects proper lumen\nformation observed in this report is at\nthis point unclear, it is tempting to\nspeculate that CD146 may achieve\nsuch function by regulating the cy-\ntoskeleton. It is known that antibody-\nmediated cross-linking of CD146 in\nvitro initiates a cascade of intracellu-\nlar signaling events that\nincludes\nphosphorylation of focal adhesion ki-\nnase, paxillin, and p130CAS (Anfosso\net al., 1998, 2001). Additionally,\nCD146 has been shown to interact\nwith the actin cytoskeleton in HUVEC\n(Bardin et al., 2001). Moreover, the\nCD146 ortholog in chicken has been\nshown to modulate expression of sev-\nIn Situ Hybridization\nNucleotides spanning the 5 untrans-\nlated region and the ﬁrst 592 bases of\nthe open reading frame of zebraﬁsh\nCD146 were ampliﬁed and cloned into\na vector derived from pCR-BluntII-\nTOPO (Invitrogen). To\ngenerate\ndigoxigenin (DIG)-labeled antisense\nprobes, the DNA plasmid was linear-\nized with EcoRI, followed by tran-\nscription using SP6 polymerase and\nthe 10 DIG RNA labeling mix from\nRoche (Indianapolis, IN). In situ hy-\nbridization was then performed ac-\ncording to a published procedure\n(Broadbent and Read, 1999; Read,\n2000).\nMicroinjection and\nFluorescence\nMicroangiography\nTwo antisense (splice blocker 1. 5-\nCAGTTGTAGTGACCCACCTCGCA-\nCG-3; splice blocker 2. 5-GTCTTTT-\nTCTCACCGTAAACCAGAT-3) and a\ncontrol (5-GACTCTATATTGCCTAC-\nCTTTACAG-3) morpholino oligo-\nnucleotides were purchased from\nGene-Tools (Philomath, OR). The\nmorpholino oligonucleotides were in-\njected into the yolk sac of zebraﬁsh\nembryos between one- and four-cell\nstages. Embryos were raised in egg\nwater in the presence of 0.003% (w/v)\n1-phenyl-2-thiourea (Sigma, St. Louis,\nMO) to prevent pigmentation (Wester-\nﬁeld, 2000). To verify successful block-\nage of CD146 pre-mRNA splicing by\nthe morpholino oligonucleotides, total\nRNA was collected from the zebraﬁsh\nembryos at 2 dpf by using the RNeasy\nMini kit from Qiagen. Reverse tran-\nscription was carried out by using ei-\nther Thermoscript or Superscript III\nfrom Invitrogen with the gene-speciﬁc\nprimer (5-GGCACAGGGAGTGTCT-\nTCTTGCCTTATA-3). PCR was then\nperformed by using GC-Advantage\nDNA polymerase (Clontech) and the\nfollowing ampliﬁcation primers: 5-\nGATATCACCTTGACTTGCAACG-\nCATTGTCT-3 and 5-GATTGGG-\nTAGCCTTGTGCATAGCAGGTCAG-3\nfor\nsplice blocker 1 injection;\n5-ATGACCTACACCGCACTGCTGC-\nTCGCGGGA-3 and 5-AGGATC-\nGAGAGGGAAGCTGCCCTTGTAC-\nCA-3 for splice blocker 2 injection. A\npair of isoform speciﬁc primers that am-\npliﬁed only the wild-type CD146 cDNA\n(nucleotides 126 to 1287) has the follow-\ning sequences: 5-TACTTACACTCT-\nGACGGAAGAAGCCAAGAT-3 and\n5-CTTTCCTCGCACGTTTACTTGCAC-\nTGA-3. Primers used to ampliﬁed Fli-1\nwere 5-GAAAATGGACGGAACTAT-\nTAAGGAGGCGCT-3 and 5-TCTTC-\nGAGTGCAGTTCAAGTTTTGGCATT-\nTAAGGGT-3.\nBlood circulation in anesthetized\nﬁsh embryos was recorded as time-\nlapse images using an inverted phase\ncontrast microscope (Nikon Corpora-\ntion, Tokyo, Japan), a Spot RT cam-\nera, and the Advanced SPOT software\n(Diagnostic Instruments, Inc., Ster-\nling Heights, MI). The images were\nconverted into QuickTime movies us-\ning a program called ImageJ.\nmicroangiography\nwas performed by using the red ﬂuo-\ncarboxylate-modiﬁed Fluo-\nrescent\nSpheres (0.02 m)\nfrom Molecular\nProbes (Eugene, OR) as described\n(Weinstein et al., 1995). Fluorescence\nmicroangiograms were recorded with\nthe Nikon ﬂuorescence microscope us-\ning a ﬁlter set (41035:ex. 546; em. 605)\nfrom Chroma (Rockingham, VT).\nACKNOWLEGMENTS\nThis study was supported by seed\nfunds from Beth Israel Deaconess\nMedical Center (Boston, MA).\nREFERENCES\nAnfosso F, Bardin N, Vivier E, Sabatier F,\nSampol J, Dignat-George F. 2001. Out-\nside-in signaling pathway linked to\nCD146 engagement in human endothe-\nlial cells. J Biol Chem 276:1564 –1569.\nBardin N, Frances V, Lesaule G,\nHorschowski N, George F, Sampol J.\n1996a. Identiﬁcation of the S-Endo 1 en-\ndothelial-associated antigen. Biochem\nBiophys Res Commun 218:210 –216.\nBardin N, George F, Mutin M, Brisson C,\nHorschowski N, Frances V, Lesaule G,\nSampol J. 1996b. S-Endo 1, a pan-endo-\nthelial monoclonal antibody recognizing\na novel human endothelial antigen. Tis-\nsue Antigens 48:531–539.",
         "Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V. Sukhatme",
         null
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "Cardiac troponin I is a phosphorylation target for endothelin-\nactivated protein kinase C. Earlier work in cardiac myocytes\nexpressing nonphosphorylatable slow skeletal troponin I provided\nevidence that protein kinase C-mediated cardiac troponin I phos-\nphorylation accelerates relaxation. However, replacement with the\nslow skeletal isoform also alters the myofilament pH response and\nthe Ca2 transient, which could influence endothelin-mediated\nrelaxation. Here, differences in the Ca2 transient could not explain\nthe divergent relaxation response to endothelin in myocytes\nexpressing cardiac versus slow skeletal troponin I nor could activa-\ntion of Na/H exchange. Three separate clusters within cardiac\ntroponin I are phosphorylated by protein kinase C, and we set out to\ndetermine the contribution of the Thr144 and Ser23/Ser24 clusters to\nthe endothelin-mediated contractile response. Myocyte replace-\nment with a cardiac troponin I containing a Thr144 substituted with\nthe Pro residue found in slow skeletal troponin I resulted in pro-\nlonged relaxation in response to acute endothelin compared with\ncontrol myocytes. Ser23/Ser24 also is a target for protein kinase C\nphosphorylation of purified cardiac troponin I, and although this\ncluster was not acutely phosphorylated in intact myocytes, signifi-\ncant phosphorylation developed within 1 h after adding endothelin.\nReplacement of Ser23/Ser24 with Ala indicated that this cluster con-\ntributes significantly to relaxation during more prolonged endothe-\nlin stimulation. Overall, results with these mutants provide evi-\ndence that Thr144 plays an important role in the acute acceleration\nof relaxation, whereas Ser23/Ser24 contributes to relaxation during\nmore prolonged activation of protein kinase C by endothelin.",
         "Protein kinase C (PKC)2 activation is an important pathway involved\nin modulating cardiac contractile function (1–5). A number of end tar-\nget proteins involved in excitation-contraction coupling are phospho-\nrylated in response to PKC activation, including the L-type Ca2 chan-\nnel (6, 7) and myofilament proteins such as cardiac troponin I (cTnI)\n(8 –11), cardiac troponin T (10, 12), and myosin light chain 2 (MLC2) (8,\n13, 14). The contribution of each end target to the contractile response\nto PKC is not well understood. Our laboratory is interested in under-\nstanding the role of cTnI in the PKC-mediated contractile function\nresponse. Increased PKC expression is observed in failing human hearts\n(15, 16), and the relationship between PKC and cTnI phosphorylation\nmay exert an important influence on cardiac function under physiolog-\nical and pathophysiological conditions (17).\n* The costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked “advertisement” in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\nEarlier work demonstrated that cTnI phosphorylation is correlated\nwith the positive inotropic response to ET (4, 8, 18). Accelerated relax-\nation also is observed in response to PKC activation by ET in myocytes\nexpressing cTnI (4). PKC phosphorylation of purified cTnI decreases\nmyofilament Ca2 sensitivity in reconstituted myofilaments (11), and\nthis desensitization is expected to contribute to the accelerated relax-\nation. In contrast, peak shortening is diminished and relaxation is\ndelayed rather than accelerated in response to ET when endogenous\ncTnI is replaced with nonphosphorylatable slow skeletal troponin I\n(ssTnI) in adult myocytes. These results are consistent with the idea that\ncTnI phosphorylation enhances peak myocyte shortening and acceler-\nates relaxation rate in response to PKC activation by ET. However,\nssTnI expression in adult myocytes also increases myofilament Ca2\nsensitivity and slows the Ca2 transient (19, 20). Thus, the differential\nresponse to ET observed in myocytes expressing cTnI versus ssTnI may\nresult from the direct actions of cTnI phosphorylation on shortening\nand relaxation or from alterations in the Ca2 transient induced by\nssTnI expression.\nOne goal of the present study is to establish whether PKC-mediated\ncTnI phosphorylation acts directly to accelerate relaxation or is\nexplained by parallel alterations in cellular Ca2. Calcium and shorten-\ning responses are measured simultaneously during the ET response for\nthese experiments. Stimulation frequency also alters the Ca2 transient,\nwith accelerated sarcoplasmic reticulum Ca2 uptake observed at\nhigher pacing frequencies (21). The relative contribution of phospho-\nrylated cTnI to the ET-induced relaxation response may decrease as\npacing frequency increases in the event that alterations in cellular Ca2\nare primarily responsible for this relaxation response. Thus, the contri-\nbution of cTnI phosphorylation to the ET-induced relaxation response\nis examined over a range of frequencies.\nA second goal of our study was to differentiate between cTnI phos-\nphorylation-induced and alkalosis-induced changes in the ET-mediated\ncontractile performance response. Previous investigators showed that\nET increases Na/H exchange via PKC (22, 23). The ensuing alkalosis\nincreases myofilament Ca2 sensitivity (24), and ET-induced alkalosis\nhas been shown to contribute to the enhanced peak shortening and\nincreased rate of contraction (22, 23). The diminished peak-shortening\nresponse to ET observed in ssTnI-expressing myocytes (4) also could\nresult from the enhanced basal myofilament Ca2 sensitivity and dimin-\nished pH sensitivity relative to cTnI-expressing myocytes (19). In the\npresent study, the influence of ET-induced alkalosis on peak contrac-\ntion was tested by inhibiting Na/H exchange in myocytes expressing\ncTnI and ssTnI.\nThe final aim of the present work was to determine the contribution\nof individual phosphorylation sites within cTnI to the PKC-mediated\nchanges in intact adult myocyte relaxation rate in response to ET.\nDetailed work on purified cTnI indicates that PKC phosphorylates three\nseparate clusters of residues, including Ser23/Ser24, Ser43/Ser45, and\nThr144 within cTnI (11, 25). However, the myocardial response to phos-\nphorylation of individual sites within the intact myocyte has yet to be\nPKC Phosphorylation of Troponin I and Contractile Function\nthoroughly investigated. Studies here are focused on investigating the\ninfluence of the rat cTnI Ser23/Ser24 and Thr144 sites on contractile\nfunction in response to ET. For studies of cTnIThr144, the goal is to\ndetermine whether Thr144 influences the peak contraction and relax-\nation response to PKC activated by ET, by substitution with the non-\nphosphorylatable Pro residue. The Ser23/Ser24 cluster also is a target for\nPKC in purified cTnI and is a major myofilament target for protein\nkinase A (PKA) (26). Phosphorylation of this cluster by PKA signifi-\ncantly decreases myofilament Ca2 sensitivity (27) and accelerates\nrelaxation during -adrenergic activation (28). A similar increase in\nrelaxation is expected if this Ser cluster is phosphorylated by PKC in\nintact adult myocytes. For these studies, the Ser23/Ser24 residues were\nsubstituted with Ala to investigate whether this cluster influences the\nability of cTnI to maintain relaxation in response to ET activation of\nPKC.",
         "EXPERIMENTAL PROCEDURES\nMutagenesis Strategy—Wild-type rat cTnI cDNA (kind gift of A.\nMurphy) was used to construct the cTnIT144P and cTnIS23A/S24A\nmutants. Primers used for mutagenesis of cTnIT144P were 5-gcaagtt-\ntaagcggccgcctctccgaagagtg-3 (sense) and 5-cactcttcggagaggcggccgct-\ntaaacttgc-3 (antisense), and for cTnIS23A/S24A the primers were\n5-ccgacgtcgcgccgcggccaactaccg-3 (sense) and 5-cggtagttggccgcg-\ngcgcgacgtcgg-3 (antisense) using a pGEM-3Z vector and the Strat-\nagene QuikChange site-directed mutagenesis kit (29). The presence of\neach mutant was verified by DNA sequencing.\nGeneration of Adenoviral Vectors—Recombinant adenovirus vectors\nwere constructed by cotransfection of pCA4cTnI cDNA with pJM17 by\ncalcium phosphate into a HEK 293 cell\nline. Replication-deficient,\nrecombinant adenovirus containing wild-type rat cTnI, wild-type\nssTnI, or mutant ssTnI cDNAs, a cytomegalovirus promoter, and an\nSV40 polyadenylation signal were obtained following homologous\nrecombination (19).\nMyocyte Isolation—Adult rat ventricular myocytes were isolated as\ndescribed earlier (4, 30). Myocytes were plated on laminin-coated cov-\nerslips in Dulbecco’s modified Eagle’s medium supplemented with 50\nunits/ml penicillin, 50 g/ml streptomycin (pen/strep) and 5% serum\nfor 2 h prior to replacing the medium with serum-free Dulbecco’s mod-\nified Eagle’s medium containing recombinant adenovirus vector (mul-\ntiplicity of infection  500). Gene transfer of exogenous TnI at a mul-\ntiplicity of infection of 500 results in virtually all cells expressing and\nincorporating exogenous TnI into the myofilament (29). After 1 h, Dul-\nbecco’s modified Eagle’s medium with pen/strep (2 ml) was added, with\nregular replacement of media over 4 days in culture.\nAnalysis of Protein Expression—Control and virus-treated adult rat\ncardiac myocytes were scraped from coverslips into sample buffer 4\ndays after gene transfer (31). In some experiments, myocytes were per-\nmeabilized in 0.1% Triton X-100 for 1 min to evaluate incorporation of\nTnI protein into the myofilament after gene transfer. In earlier work,\nexpression in intact and permeabilized myocytes was comparable when\nmyofilament proteins were incorporated into the myofilament (29, 32).\nProteins were then separated by gel electrophoresis and blotted onto\npolyvinylidene difluoride membrane for 2000 V-h with immunodetec-\ntion carried out as described earlier (30) using a 1:1000 dilution of MAB\n1691 (Chemicon Inc., Temecula, CA), a monoclonal antibody that rec-\nognizes all TnI isoforms. A phospho-Ser23/Ser24-specific polyclonal\nantibody (1:1000; Cell Signaling Technology, Beverly, MA) and a paired\nhorseradish peroxidase-conjugated secondary antibody were used to\ndetect Ser23/Ser24 phosphorylation. Myocytes collected for detection of\nphosphorylation were incubated at 37 °C with calyculin A (10 nM) alone\nor with the selected PKA and PKC agonist and/or inhibitor for time\nintervals ranging from 2.5 min up to 1 h.\nMeasurement of Sarcomere Shortening in Single Cardiac Myocytes—\nMyocytes to be used for shortening assays were transferred to a Plexiglas\nstimulation chamber consisting of 8 wells containing platinum elec-\ntrodes. Myocytes were stimulated electrically (2.5-ms pulse, 0.2 Hz) in\nmedium 199 supplemented with pen/strep (M199), 10 mM HEPES, 0.2\nmg/ml bovine serum albumin, and 10 mM glutathione (M199), and\nthe media was replaced every 12 h for days 2– 4 post-gene transfer (33).\nThe voltage was set so that 20% of the myocytes were stimulated on\neach coverslip. Four days after gene transfer, individual coverslips were\ntransferred to a temperature-controlled (37 °C) chamber mounted on a\nNikon microscope stage. A video-based detection system (Ionoptix,\nMilton, MA) was used to detect sarcomere shortening in intact myo-\ncytes. For frequency-response studies, myocytes were stimulated at\neach frequency (0.2, 0.5, 0.75, 1, and 2 Hz) until a steady state contrac-\ntion was reached. Sarcomere shortening also was recorded at 0.2 Hz\nbefore and after the increases in stimulation frequency to ensure there\nwere no changes in the rate of contraction, peak contraction, and rate of\nrelaxation after testing at all frequencies. Experiments were recorded\nusing SarcLen software, and an average of 10 twitches/myocyte was\ncollected under basal conditions. Signal-averaged data were analyzed to\ndetermine resting sarcomere length (SLR), percent change in peak sar-\ncomere shortening above base line (Sb), time to peak shortening (TTP),\ntime to 25, 50, and 75% relaxation (TTR25, TTR50, and TTR75, respec-\ntively), and maximum normalized shortening and relaxation velocities\n(dl/dtmax, dl/dtmax, respectively). Media containing ET (10 nM),\n5-N(ethyl-N-iospropyl)amiloride (EIPA, 500 nM) or both reagents were\nperfused via separate individual perfusion tubing to eliminate introduc-\ntion of residual agonists/inhibitors.\nExperiments for Ca2 transient and sarcomere shortening measure-\nments were carried out on myocytes loaded with Fura-2 AM, as\ndescribed previously (34). Briefly, myocytes were loaded for 5 min with\n5 M Fura-2 AM (final Me2SO  0.6%) at 37 °C followed by a 5-min\nwash-out for de-esterification. Myocytes were then stimulated at 0.2 Hz\nat 37 °C, and the base-line sarcomere length and Ca2 transient were\nrecorded from a single cell. Recordings were then made 5, 10, and 15\nmin after the addition of ET (10 nM), with the cell maintained in the dark\nbetween sampling times. Optimal cTnI phosphorylation was observed\nin response to 10 –100 nM ET in an earlier study (4). Ten twitches/\nmyocyte were collected for each sample. In addition to sarcomere short-\nening measurements, the resting and peak ratios, maximum Ca2 tran-\nsient rate (dCa/dtmax), maximum Ca2 decay rate (dCa/dtmax), and\ntime to 50 and 75% decay (TTD50, TTD75, respectively) in the Ca2\ntransient were measured. The Ca2 transient and shortening measure-\nments were maintained over this same time interval in myocytes main-\ntained under basal conditions.\nStatistical Analysis—Values are expressed as mean  S.E. Grouped\ncomparisons are analyzed using a Student’s t test or analysis of variance\ntesting (ANOVA). Statistical differences detected by ANOVA are com-\npared with a Newman-Keuls multiple comparison test with p  0.05\nconsidered significantly different.\nRESULTS\nRelationship between Ca2 Transient and Contractile Response to\nEndothelin—Sarcomere shortening and Ca2 transients were measured\nbefore and after ET in Fura-2-loaded myocytes expressing cTnI or ssTnI\n4 days after gene transfer (Fig. 1, TABLE ONE). The increased peak\nshortening and enhanced relaxation rate observed in response to 10 nM\nET in earlier work on myocytes expressing cTnI (4), was similarly\nPKC Phosphorylation of Troponin I and Contractile Function\nobserved in myocytes loaded with Fura-2 (Fig. 1A). In myocytes\nexpressing nonphosphorylatable ssTnI, the delayed relaxation and loss\nof the enhanced peak shortening observed in response to ET (4) simi-\nlarly developed in Fura-2-loaded cells (Fig. 1B, TABLE ONE).\nCa2 transient measurements also were examined in these myocytes.\nThe time to 50% decay for basal Ca2 transients reportedly slows in\ntransgenic mouse myocytes expressing ssTnI (20), and similar results\ndeveloped in rat myocytes expressing ssTnI (TABLE TWO). Peak Ca2\nand the TTP remained unchanged in the ssTnI-expressing myocytes\ncompared with values observed in controls at 0.2 Hz. The Ca2 tran-\nsient change observed in response to ET was then analyzed in myocytes\nexpressing cTnI and ssTnI (TABLE ONE). The percent change in peak\nCa2 amplitude and maximum rate of Ca2 decay (dCa/dtmax) in\nresponse to ET was not significantly changed in myocytes expressing\neither cTnI or ssTnI (Fig. 1, A and B; TABLE ONE). However, signifi-\ncant delays were observed in the time to 75% Ca2 decay in both groups\nof myocytes. The similar Ca2 transient responses to ET coupled with\nthe divergent shortening response of cTnI- versus ssTnI-expressing\nmyocytes indicates that cTnI contributes significantly to the enhanced\npeak shortening and accelerated relaxation rate observed in response to\nPKC activation by ET compared with myocytes expressing ssTnI.\nAdditional studies were performed to investigate whether the contri-\nbution of phosphorylated TnI to the ET-mediated change in contractile\nfunction is diminished with higher frequencies. The shortening  fre-\nquency relationship described previously in rodents (33, 35) was similar\nunder basal conditions and in response to ET over a range of frequencies\n(0.2–2 Hz), (TABLE THREE) in control myocytes. The percent decrease\nin peak shortening, dl/dtmax and dl/dtmax in response to increased\nfrequency was similar before and after ET. The consistency of this\nresponse provides evidence that the contribution of cTnI phosphoryla-\ntion to contractile function is not altered by frequency-induced changes\nin the Ca2 cycle.\nContribution of Cellular Alkalosis to the Myofilament Response to\nEndothelin—An important component of the contractile response to\nET depends on activation of Na/H exchange and the generation of a\ncellular alkalosis (22, 23). Experiments were carried out in the presence\nof the amiloride analog EIPA, an inhibitor of Na/H exchange (23, 36),\nto identify responses resulting from alkalosis. The myocyte response to\nEIPA is comparable in nontreated myocytes and myocytes expressing\ncTnI after gene transfer (results not shown); the pooled results are\nshown in Fig. 2. Myocyte shortening amplitude in myocytes expressing\nPKC Phosphorylation of Troponin I and Contractile Function\ncTnI decreased slightly after the addition of EIPA but did not signifi-\ncantly influence peak amplitude in myocytes expressing ssTnI (Fig. 2, A\nand B). Resting sarcomere length (SLR), dl/dtmax, and dl/dtmax\n(dl/dt and dl/dt in Fig. 2, respectively) were not significantly altered\nby EIPA in myocytes expressing cTnI or ssTnI (Fig. 2B). However, EIPA\nblocked a major portion (70%) of the ET-induced increase in peak\nshortening amplitude observed in cTnI-expressing myocytes. In con-\ntrast, EIPA did not significantly modify the accelerated relaxation\nresponse to ET in myocytes expressing cTnI, and the delayed relaxation\nobserved in response to ET was comparable with and without EIPA in\nmyocytes expressing ssTnI (Figs. 1 and 2). These results support the idea\nthat Na/H exchange and the ensuing alkalosis is important for the\nenhanced peak shortening response, whereas cTnI plays a more critical\nrole in the accelerated relaxation rate response to ET.\nRole of cTnI Phosphorylation Residues in the Endothelin Response—\nTo determine the role of the cTnIThr144 phosphorylation site in the\nrelaxation response, the ssTnI Pro residue was substituted for Thr144 to\nform cTnIT144P, and replacement was measured 4 days after gene\ntransfer. The stoichiometric replacement of cTnI with cTnIT144P-\nFLAG was 46  2% (n  13) and comparable with the 53  2% (n  14)\nreplacement with cTnI-FLAG 4 days after gene transfer (Fig. 3).\nReplacement of nontagged cTnI and cTnIT144P is expected to be com-\nparable in myocytes over the same time interval (29, 37). Total TnI\nexpression remained constant in myocytes expressing cTnI and\ncTnIT144P based on the ratio of TnI intensity to the intensity of a\nsilver-stained band on the gel (cTnI set at 1.0; cTnIT144P  1.13  0.10,\nn  11), and isoform expression of tropomyosin and troponin T was\ncomparable in myocytes expressing cTnI and cTnIT144P (results not\nshown). Expression of total TnI also was comparable in intact (I) and\npermeabilized (P) myocytes (Fig. 3, I/P ratio for cTnI  1.10  0.08, n \n5; cTnIT144P  1.05  0.06, n  5), which is consistent with myofila-\nment incorporation of the delivered TnI.\nMeasures of basal contractile function in myocytes expressing\ncTnIT144P are not significantly different from values obtained from\nmyocytes expressing cTnI (TABLE TWO, Fig. 4A). These results are in\nagreement with the earlier reverse substitution of Pro110 with Thr in fast\nskeletal TnI (38). Base-line Ca2 transients in myocytes expressing\ncTnIT144P (Fig. 4B) also showed no significant differences from control\nvalues. Peak shortening increased significantly in response to ET in\ncTnIT144P-expressing myocytes (Fig. 4) and was comparable with the\nET-induced enhanced peak shortening in control myocytes (Fig. 1B).\nHowever, several indices of relaxation, including dl/dtmax and TTR75,\nwere significantly delayed in response to ET in myocytes expressing\ncTnIT144P compared with controls (TABLE ONE). The amplitude of\nthe Ca2 transient response to ET was not significantly different com-\npared with control values for myocytes expressing cTnIT144P (TABLE\nONE, Fig. 4). As with control myocytes, a frequency change from 0.2 to\n2 Hz produced similar changes before and after ET in myocytes express-\ning cTnIT144P (results not shown). The major difference between myo-\ncytes expressing cTnIT144P versus cTnI was the decrease in relaxation\nrate in response to ET; this finding indicates a specific role for Thr144 in\naccelerating relaxation in response to ET.\nThe contribution of Ser23/Ser24 phosphorylation to the ET-induced\nchange in contractile function also was determined. A phospho-specific\nSer23/Ser24 antibody was used for these studies. Myocytes were incu-\nbated with the PKA agonist dobutamine (DOB) for 2 min to verify\ndetection of the well documented PKA-mediated cTnISer23/Ser24 phos-\nPKC Phosphorylation of Troponin I and Contractile Function\nphorylation site (39, 40) (Fig. 5A). Detection of cTnI phosphorylation by\nthis antibody was specific, as it was blocked when, in addition to DOB,\nthe -adrenergic inhibitor propranolol was added to myocytes (results\nnot shown). There was no significant phosphorylation detected in myo-\ncytes incubated with ET for 10 min with (Fig. 5A) or without the PKC\ninhibitor bis-indolylmaleimide-1 (bis-1, results not shown). However,\nsignificant phosphorylation of Ser23/Ser24 developed in control myo-\ncytes over a 1-h time span, and this ET-mediated phosphorylation was\nsignificantly reduced in the presence of the PKC inhibitor bis-1 but not\nby the -adrenergic antagonist propranolol (Fig. 5A). Treatment of\nmyocytes with other PKC agonists, including phenylephrine and phor-\nbol 12-myristate 13-acetate for 1 h also resulted in significant cTnI\nphosphorylation of Ser23/Ser24. Incubation with fibroblast growth fac-\ntor-2 was previously shown to activate PKC (41), yet this agonist did not\nsignificantly influence cTnI phosphorylation over a 1-h time span\n(results not shown). These results provide evidence for temporal phos-\nphorylation of cTnI Ser23/Ser24 in response to PKC activation by several\nPKC agonists.\nTo directly determine the contribution of Ser23/Ser24 to the contract-\nile function response to ET, cTnIS23A/S24A was expressed in adult\nmyocytes. By 4 days after gene transfer, 44  5% of cTnI was replaced\nwith the cTnIS23A/S24A (n  15, Fig. 5B). DOB phosphorylation of\ncTnI was significantly reduced in myocytes expressing cTnIS23A/S24A\nor cTnIS23A/S24A-FLAG (results not shown). In addition, Ser23/Ser24\nphosphorylation was significantly attenuated after 1 h of ET in myocytes\nexpressing tagged and nontagged cTnIS23A/S24A (Fig. 5C). This S23A/\nS24A substitution has previously been shown to have little influence on\nbasal myofilament function (42), and no significant differences were\nobserved in SLR, peak amplitude, dl/dtmax, ordl/dt max in myocytes\nexpressing cTnIS23A/S24A compared with controls (results not\nshown). Control myocytes expressing cTnI or\nthe substituted\ncTnIS23A/S24A responded similarly to ET over the first 15 min (Fig. 6).\nHowever, a significant difference in the response was observed 1 h after\nthe addition of ET in myocytes expressing cTnIS23A/S24A compared\nPKC Phosphorylation of Troponin I and Contractile Function\nwith control values (Fig. 6). Specifically, peak shortening amplitude was\nnot increased and both dl/dtmax and TTR75 indicated relaxation was\nsignificantly prolonged in myocytes expressing cTnIS23A/S24A com-\npared with control myocytes and were comparable with the results\nobtained in myocytes expressing ssTnI (Fig. 6). These results demon-\nstrate Ser23/Ser24 phosphorylation contributes to the accelerated relax-\nation during more prolonged myocyte PKC activation by ET.",
         "DISCUSSION\nResults from the present study demonstrate the important direct role\nfor cTnI phosphorylation in accelerating relaxation in response to PKC\nactivated by ET in adult myocytes (Figs. 1–3). Our results now demon-\nstrate a clear role for cTnI Thr144 and Ser23/Ser24 in contributing to the\naccelerated relaxation in response to ET (Figs. 4 and 6). Substitution\nwith a Pro residue at Thr144 results in the delayed relaxation previously\nobserved in myocytes expressing nonphosphorylatable ssTnI and pro-\nvides evidence that the Thr144 site contributes to accelerated relaxation\nduring the ET response. The Ser23/Ser24 cluster does not appear to\ncontribute to the initial response to ET but, instead, is important for the\naccelerated relaxation during more prolonged stimulation by ET. This\nfinding indicates that the relaxation rate response to ET-induced PKC\npathway activation depends on temporal phosphorylation of individual\nclusters within cTnI.\nTnI Phosphorylation Plays a Direct Role in the Relaxation Response to\nEndothelin—Results from the present study demonstrate the important\nrole of cTnI phosphorylation in the relaxation response during the pos-\nitive inotropic response to ET, independent of cellular Ca2 handling.\nThe only aspect of Ca2 transient that changed significantly in response\nto 10 nM ET was the percent change in TTD75, which decreased in both\ncontrol and ssTnI-expressing myocytes (Fig. 1, TABLE ONE). All other\nmeasurements of the Ca2 transient remained unchanged during the\nET response, and work by other laboratories supports this finding (43,\n44). There are reports that ET activation of PKC enhances the Ca2\ntransient at higher doses of ET (45), which may be due to increased Ca2\nentry via the L-type Ca2 channel (46). However, results from Ebihara et\nal. (47) indicate the increased Ca2 transient is not necessary for the\nenhanced contractile response to ET.\nExperiments with the NHE1 inhibitor EIPA also demonstrate that\ninhibition of NHE1 prevents the ET-mediated enhancement in peak\nshortening amplitude (Fig. 2). ET in the concentration range used here\nactivates PKC-induced NHE1 (48) and causes a cellular alkalosis (22,\n23). The alkalosis-mediated increase in myofilament Ca2 sensitivity\n(24) is responsible for the increased peak contractile shortening (22, 23).\nThis enhancement in peak shortening is absent from myocytes express-\ning ssTnI (Fig. 1), which is likely due to the combination of increased\nbasal myofilament Ca2 sensitivity (19, 20) and a decreased pH response\nto alkalosis in these myocytes compared with cTnI-expressing myocytes\n(19).\nIn contrast to peak shortening, the divergent influence of cTnI versus\nssTnI on the relaxation response to ET was not influenced significantly\nby EIPA (Fig. 2). Although the slowing of relaxation in myocytes\nexpressing ssTnI was attenuated by the presence of EIPA during the ET\nresponse, relaxation was significantly accelerated in myocytes express-\ning cTnI versus ssTnI during this response. Our results now provide\nevidence that phosphorylated cTnI maintains relaxation regardless of\nthe alkalosis. The delayed relaxation response to ET observed in myo-\ncytes expressing cTnIT144P also supports this finding (Fig. 4).\nWhat Role Does cTnI Phosphorylation Play in the Contractile\nResponse to ET?—The role of cTnI phosphorylation in the contractile\nfunction response to PKC agonists remains controversial. Earlier in\nvitro work showed that PKC phosphorylation of cTnI decreases myofil-\nament Ca2 sensitivity as well as maximum actomyosin ATPase activity\n(11, 25). PKA phosphorylation of cTnI also decreases myofilament Ca2\nsensitivity and results in accelerated myocyte relaxation (27, 28), and\nsubstitution of Ser23/Ser24 with Asp to mimic phosphorylation acceler-\nates relaxation in transgenic mice (39). However, PKC phosphorylates\nSer43/Ser45 and Thr144 in addition to the Ser23/Ser24 cluster (11). More\nrecent studies in which the Ser23/Ser24, Ser43/Ser45, and Thr144 clusters\nare all replaced with Ala in knock-in mice (49, 50) or substituted with\nAsp to mimic phosphorylation in transgenic mice (37) suggest that cTnI\nworks primarily to delay cardiac relaxation during PKC activation.\nThese results are in contrast to results presented here demonstrating an\naccelerated relaxation response attributable to cTnI phosphorylation\n(TABLE ONE), and the in vitro work demonstrating decreased myofil-\nament Ca2 sensitivity (11). Differences in experimental conditions,\nsuch as load and/or temperature in these various models, do not appear\nto account for the divergent functional response. We propose that these\nseemingly divergent responses may be due to a hierarchy of cTnI phos-\nphorylation in response to a given agonist, with different cTnI clusters\nphosphorylated in response to PKC activation. Ser43/Ser45 phosphoryl-\nPKC Phosphorylation of Troponin I and Contractile Function\nation may have a dominant influence on contractile relaxation via its\ninfluence on maximum ATPase activity when all five phosphorylation\nsites are replaced by Ala (49) or replaced with a negatively charged Asp\n(37). However, PKC activation by a specific agonist may differentially\nand/or temporally phosphorylate one or more of the cTnI residues\nrather than phosphorylate all three clusters. In support of this idea, in\nvitro motility assays carried out with cTnISer43/Ser45 containing\ncharged Glu substitutions demonstrated a decrease in myofilament\nCa2 sensitivity and decreased sliding speed, whereas cTnIT144G only\ndecreased myofilament Ca2 sensitivity (51). NMR analysis of Ser43/\nSer45 substituted with Asp provides evidence that phosphorylation of\nthis site may be dominant in that it influences the Ca2 binding affinity\nto TnC as well as protein-protein interactions between TnI and TnC\n(52). Thus, the divergent results are predicted to stem from differential\nphosphorylation of the three cTnI phosphorylation clusters and/or the\nrelative dominance of one cluster in response to activation and translo-\ncation of specific PKC isoforms by a given agonist. This idea will be\nimportant for rigorous testing, with a variety of agonists under physio-\nlogical and pathophysiological conditions, to better understand the\nPKC signaling pathway and its ability to modulate both cTnI phospho-\nrylation and contractile performance.\nRole of Individual cTnI-phosphorylated Residues in the Contractile\nResponse to ET—Detailed studies on purified cTnI support the idea that\nthe residue phosphorylated by PKC is important for the resulting func-\ntional relaxation response. Phosphorylation of Thr144 in cTnI produces\na rightward shift in myofilament Ca2 sensitivity for both actomyosin\nATPase activity (11) and force generation in permeabilized trabeculae\n(51, 53). The anticipated cellular response is an increase in myocyte\nrelaxation rate. Our results now show that substitution of Thr144 with\nPro significantly delays relaxation during the ET response (Fig. 4,\nTABLE THREE). The Thr144 site is particularly important because the\ninhibitory peptide region encompassing this phosphorylation target\n(amino acids 129 –149) is important for determining TnI binding to\nactin in the absence of Ca2 and diminished binding to actin in the\npresence of Ca2 (54). Thus, the inhibitory peptide region is important\nfor the molecular switch activity of TnI (55, 56). The delayed relaxation\nresponse to ET in myocytes expressing cTnIT144P compared with con-\ntrol myocytes is consistent with the idea that phosphorylation of this\nresidue in response to ET plays a significant role in the accompanying\nrelaxation rate response.\nExperiments with the phospho-cTnISer23/Ser24 antibody and with\ncTnIS23A/S24A also demonstrate a key role for this cluster during\nmore extended activation of PKC by ET. Phosphorylation of Ser23/Ser24\ndecreases myofilament Ca2 sensitivity and accelerates relaxation in\nresponse to PKA activation (27, 28), and similar shifts in myofilament\nCa2 sensitivity are reported in response to PKC phosphorylation of\nSer23/Ser24 (11). Our results now demonstrate a temporal pattern of\nSer23/Ser24 phosphorylation that is important for enhanced relaxation\nin response to longer intervals of ET in intact myocytes (Fig. 6). In\nbiochemical experiments with purified cTnI, longer treatment with\nPKC also was required for Ser23/Ser24 phosphorylation (11), yet it\nremained unclear whether this result indicated that there was temporal\nphosphorylation in the intact cell or instead indicated that Ser23/Ser24\nwas not a typical in vivo target for activated PKC. Results with myocytes\nexpressing cTnIS23A/S24A (Fig. 6) clearly demonstrate that there is\ntemporal phosphorylation of this site by PKC activated in response to\nET. Several PKC agonists, including ET, are released in a paracrine/\nautocrine pattern (57, 58), and phosphorylation of Ser23/Ser24 during\nmore prolonged intervals of PKC stimulation would then be important\nfor preventing delays in cellular relaxation. Taken together, the results\nobtained in experiments to investigate Thr144 and Ser23/Ser24 provide\nevidence that both clusters are important in maintaining relaxation dur-\ning PKC activation. The relative importance, as well as the interactive\neffects, of these sites and the Ser43/Ser45 site in response to PKC ago-\nnists, which appears primarily to decrease peak contraction and slow\nrelaxation (51), will be important to investigate in the future for a com-\nplete understanding of the role played by cTnI phosphorylation in mod-\nulating cardiac function.\nAcknowledgments—We thank Andrea Borton (Cell Signaling Inc.) for the gift\nof the phospho-Ser23/Ser24 antibody. We also thank Ayako Ohata for technical\nassistance, and Sharlene Day, Joseph Metzger, and Daniel Michele for helpful\ncomments on earlier versions of this manuscript.\n1. Gwathmey, J., and Hajjar, R. (1990) Circ. Res. 67, 744 –752\n2. Buenaventura, P., Cao-Danh, H., Takeuchi, K., Takahashi, S., Simpleaceanu, E.,\nMcGowan, F. J., and del Nido, P. (1995) Ann. Thorac. Surg. 60, Suppl. 6, S505–S508\n3. Puceat, M., Clement, O., Lechene, P., Pelosin, J., Ventura-Clapier, R., and Vassort, G.\n4. Westfall, M. V., and Borton, A. R. (2003) J. Biol. Chem. 278, 33694 –33700\n5. Metzger, J. M., and Westfall, M. V. (2004) Circ. Res. 94, 146 –158\n6. He, J. Q., Pi, Y., Walker, J. W., and Kamp, T. J. (2000) J. Physiol. 524, 807– 820\n7. Alden, K. J., Goldspink, P. H., Ruch, S. W., Buttrick, P. M., and Garcia, J. (2002) Am. J.\n9. Noland, T. A., Jr., and Kuo, J. F. (1991) J. Biol. Chem. 266, 4974 – 4978\n10. Mazzei, G. J., and Kuo, J. F. (1984) Biochem. J. 218, 361–369\n11. Noland, T. A., Jr., Raynor, R. L., and Kuo, J. F. (1989) J. Biol. Chem. 264, 20778 –20785\n12. Noland, T. A., Jr., and Kuo, J. F. (1992) Biochem. J. 288, 123–129\n13. Noland, T. A., Jr., and Kuo, J. F. (1993) Biochem. Biophys. Res. Commun. 193, 254 –260\n14. Clement, O., Puceat, M., Walsh, M., and Vassort, G. (1992) Biochem. J. 285, 311–317\n15. Bodor, G. S., Oakeley, A. E., Allen, P. D., Crimmins, D. L., Ladenson, J. H., and\n21. DeSantiago, J., Maier, L. S., and Bers, D. M. (2002) J. Mol. Cell Cardiol. 34, 975–984\n22. Kramer, B. K., Smith, T. W., and Kelly, R. A. (1991) Circ. Res. 68, 269 –279\n23. Goldberg, A. T., Bond, B. R., Mukherjee, R., New, R. B., Zellner, J. L., Crawford, F. A.,\nPKC Phosphorylation of Troponin I and Contractile Function",
         "EXPERIMENTAL PROCEDURES\nMutagenesis Strategy—Wild-type rat cTnI cDNA (kind gift of A.\nMurphy) was used to construct the cTnIT144P and cTnIS23A/S24A\nmutants. Primers used for mutagenesis of cTnIT144P were 5-gcaagtt-\ntaagcggccgcctctccgaagagtg-3 (sense) and 5-cactcttcggagaggcggccgct-\ntaaacttgc-3 (antisense), and for cTnIS23A/S24A the primers were\n5-ccgacgtcgcgccgcggccaactaccg-3 (sense) and 5-cggtagttggccgcg-\ngcgcgacgtcgg-3 (antisense) using a pGEM-3Z vector and the Strat-\nagene QuikChange site-directed mutagenesis kit (29). The presence of\neach mutant was verified by DNA sequencing.\nGeneration of Adenoviral Vectors—Recombinant adenovirus vectors\nwere constructed by cotransfection of pCA4cTnI cDNA with pJM17 by\ncalcium phosphate into a HEK 293 cell\nline. Replication-deficient,\nrecombinant adenovirus containing wild-type rat cTnI, wild-type\nssTnI, or mutant ssTnI cDNAs, a cytomegalovirus promoter, and an\nSV40 polyadenylation signal were obtained following homologous\nrecombination (19).\nMyocyte Isolation—Adult rat ventricular myocytes were isolated as\ndescribed earlier (4, 30). Myocytes were plated on laminin-coated cov-\nerslips in Dulbecco’s modified Eagle’s medium supplemented with 50\nunits/ml penicillin, 50 g/ml streptomycin (pen/strep) and 5% serum\nfor 2 h prior to replacing the medium with serum-free Dulbecco’s mod-\nified Eagle’s medium containing recombinant adenovirus vector (mul-\ntiplicity of infection  500). Gene transfer of exogenous TnI at a mul-\ntiplicity of infection of 500 results in virtually all cells expressing and\nincorporating exogenous TnI into the myofilament (29). After 1 h, Dul-\nbecco’s modified Eagle’s medium with pen/strep (2 ml) was added, with\nregular replacement of media over 4 days in culture.\nAnalysis of Protein Expression—Control and virus-treated adult rat\ncardiac myocytes were scraped from coverslips into sample buffer 4\ndays after gene transfer (31). In some experiments, myocytes were per-\nmeabilized in 0.1% Triton X-100 for 1 min to evaluate incorporation of\nTnI protein into the myofilament after gene transfer. In earlier work,\nexpression in intact and permeabilized myocytes was comparable when\nmyofilament proteins were incorporated into the myofilament (29, 32).\nProteins were then separated by gel electrophoresis and blotted onto\npolyvinylidene difluoride membrane for 2000 V-h with immunodetec-\ntion carried out as described earlier (30) using a 1:1000 dilution of MAB\n1691 (Chemicon Inc., Temecula, CA), a monoclonal antibody that rec-\nognizes all TnI isoforms. A phospho-Ser23/Ser24-specific polyclonal\nantibody (1:1000; Cell Signaling Technology, Beverly, MA) and a paired\nhorseradish peroxidase-conjugated secondary antibody were used to\ndetect Ser23/Ser24 phosphorylation. Myocytes collected for detection of\nphosphorylation were incubated at 37 °C with calyculin A (10 nM) alone\nor with the selected PKA and PKC agonist and/or inhibitor for time\nintervals ranging from 2.5 min up to 1 h.\nMeasurement of Sarcomere Shortening in Single Cardiac Myocytes—\nMyocytes to be used for shortening assays were transferred to a Plexiglas\nstimulation chamber consisting of 8 wells containing platinum elec-\ntrodes. Myocytes were stimulated electrically (2.5-ms pulse, 0.2 Hz) in\nmedium 199 supplemented with pen/strep (M199), 10 mM HEPES, 0.2\nmg/ml bovine serum albumin, and 10 mM glutathione (M199), and\nthe media was replaced every 12 h for days 2– 4 post-gene transfer (33).\nThe voltage was set so that 20% of the myocytes were stimulated on\neach coverslip. Four days after gene transfer, individual coverslips were\ntransferred to a temperature-controlled (37 °C) chamber mounted on a\nNikon microscope stage. A video-based detection system (Ionoptix,\nMilton, MA) was used to detect sarcomere shortening in intact myo-\ncytes. For frequency-response studies, myocytes were stimulated at\neach frequency (0.2, 0.5, 0.75, 1, and 2 Hz) until a steady state contrac-\ntion was reached. Sarcomere shortening also was recorded at 0.2 Hz\nbefore and after the increases in stimulation frequency to ensure there\nwere no changes in the rate of contraction, peak contraction, and rate of\nrelaxation after testing at all frequencies. Experiments were recorded\nusing SarcLen software, and an average of 10 twitches/myocyte was\ncollected under basal conditions. Signal-averaged data were analyzed to\ndetermine resting sarcomere length (SLR), percent change in peak sar-\ncomere shortening above base line (Sb), time to peak shortening (TTP),\ntime to 25, 50, and 75% relaxation (TTR25, TTR50, and TTR75, respec-\ntively), and maximum normalized shortening and relaxation velocities\n(dl/dtmax, dl/dtmax, respectively). Media containing ET (10 nM),\n5-N(ethyl-N-iospropyl)amiloride (EIPA, 500 nM) or both reagents were\nperfused via separate individual perfusion tubing to eliminate introduc-\ntion of residual agonists/inhibitors.\nExperiments for Ca2 transient and sarcomere shortening measure-\nments were carried out on myocytes loaded with Fura-2 AM, as\ndescribed previously (34). Briefly, myocytes were loaded for 5 min with\n5 M Fura-2 AM (final Me2SO  0.6%) at 37 °C followed by a 5-min\nwash-out for de-esterification. Myocytes were then stimulated at 0.2 Hz\nat 37 °C, and the base-line sarcomere length and Ca2 transient were\nrecorded from a single cell. Recordings were then made 5, 10, and 15\nmin after the addition of ET (10 nM), with the cell maintained in the dark\nbetween sampling times. Optimal cTnI phosphorylation was observed\nin response to 10 –100 nM ET in an earlier study (4). Ten twitches/\nmyocyte were collected for each sample. In addition to sarcomere short-\nening measurements, the resting and peak ratios, maximum Ca2 tran-\nsient rate (dCa/dtmax), maximum Ca2 decay rate (dCa/dtmax), and\ntime to 50 and 75% decay (TTD50, TTD75, respectively) in the Ca2\ntransient were measured. The Ca2 transient and shortening measure-\nments were maintained over this same time interval in myocytes main-\ntained under basal conditions.\nStatistical Analysis—Values are expressed as mean  S.E. Grouped\ncomparisons are analyzed using a Student’s t test or analysis of variance\ntesting (ANOVA). Statistical differences detected by ANOVA are com-\npared with a Newman-Keuls multiple comparison test with p  0.05\nconsidered significantly different.",
         "Cardiac troponin I is a phosphorylation target for endothelin-\nactivated protein kinase C. Earlier work in cardiac myocytes\nexpressing nonphosphorylatable slow skeletal troponin I provided\nevidence that protein kinase C-mediated cardiac troponin I phos-\nphorylation accelerates relaxation. However, replacement with the\nslow skeletal isoform also alters the myofilament pH response and\nthe Ca2 transient, which could influence endothelin-mediated\nrelaxation. Here, differences in the Ca2 transient could not explain\nthe divergent relaxation response to endothelin in myocytes\nexpressing cardiac versus slow skeletal troponin I nor could activa-\ntion of Na/H exchange. Three separate clusters within cardiac\ntroponin I are phosphorylated by protein kinase C, and we set out to\ndetermine the contribution of the Thr144 and Ser23/Ser24 clusters to\nthe endothelin-mediated contractile response. Myocyte replace-\nment with a cardiac troponin I containing a Thr144 substituted with\nthe Pro residue found in slow skeletal troponin I resulted in pro-\nlonged relaxation in response to acute endothelin compared with\ncontrol myocytes. Ser23/Ser24 also is a target for protein kinase C\nphosphorylation of purified cardiac troponin I, and although this\ncluster was not acutely phosphorylated in intact myocytes, signifi-\ncant phosphorylation developed within 1 h after adding endothelin.\nReplacement of Ser23/Ser24 with Ala indicated that this cluster con-\ntributes significantly to relaxation during more prolonged endothe-\nlin stimulation. Overall, results with these mutants provide evi-\ndence that Thr144 plays an important role in the acute acceleration\nof relaxation, whereas Ser23/Ser24 contributes to relaxation during\nmore prolonged activation of protein kinase C by endothelin.\nProtein kinase C (PKC)2 activation is an important pathway involved\nin modulating cardiac contractile function (1–5). A number of end tar-\nget proteins involved in excitation-contraction coupling are phospho-\nrylated in response to PKC activation, including the L-type Ca2 chan-\nnel (6, 7) and myofilament proteins such as cardiac troponin I (cTnI)\n(8 –11), cardiac troponin T (10, 12), and myosin light chain 2 (MLC2) (8,\n13, 14). The contribution of each end target to the contractile response\nto PKC is not well understood. Our laboratory is interested in under-\nstanding the role of cTnI in the PKC-mediated contractile function\nresponse. Increased PKC expression is observed in failing human hearts\n(15, 16), and the relationship between PKC and cTnI phosphorylation\nmay exert an important influence on cardiac function under physiolog-\nical and pathophysiological conditions (17).\n* The costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked “advertisement” in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\nEarlier work demonstrated that cTnI phosphorylation is correlated\nwith the positive inotropic response to ET (4, 8, 18). Accelerated relax-\nation also is observed in response to PKC activation by ET in myocytes\nexpressing cTnI (4). PKC phosphorylation of purified cTnI decreases\nmyofilament Ca2 sensitivity in reconstituted myofilaments (11), and\nthis desensitization is expected to contribute to the accelerated relax-\nation. In contrast, peak shortening is diminished and relaxation is\ndelayed rather than accelerated in response to ET when endogenous\ncTnI is replaced with nonphosphorylatable slow skeletal troponin I\n(ssTnI) in adult myocytes. These results are consistent with the idea that\ncTnI phosphorylation enhances peak myocyte shortening and acceler-\nates relaxation rate in response to PKC activation by ET. However,\nssTnI expression in adult myocytes also increases myofilament Ca2\nsensitivity and slows the Ca2 transient (19, 20). Thus, the differential\nresponse to ET observed in myocytes expressing cTnI versus ssTnI may\nresult from the direct actions of cTnI phosphorylation on shortening\nand relaxation or from alterations in the Ca2 transient induced by\nssTnI expression.\nOne goal of the present study is to establish whether PKC-mediated\ncTnI phosphorylation acts directly to accelerate relaxation or is\nexplained by parallel alterations in cellular Ca2. Calcium and shorten-\ning responses are measured simultaneously during the ET response for\nthese experiments. Stimulation frequency also alters the Ca2 transient,\nwith accelerated sarcoplasmic reticulum Ca2 uptake observed at\nhigher pacing frequencies (21). The relative contribution of phospho-\nrylated cTnI to the ET-induced relaxation response may decrease as\npacing frequency increases in the event that alterations in cellular Ca2\nare primarily responsible for this relaxation response. Thus, the contri-\nbution of cTnI phosphorylation to the ET-induced relaxation response\nis examined over a range of frequencies.\nA second goal of our study was to differentiate between cTnI phos-\nphorylation-induced and alkalosis-induced changes in the ET-mediated\ncontractile performance response. Previous investigators showed that\nET increases Na/H exchange via PKC (22, 23). The ensuing alkalosis\nincreases myofilament Ca2 sensitivity (24), and ET-induced alkalosis\nhas been shown to contribute to the enhanced peak shortening and\nincreased rate of contraction (22, 23). The diminished peak-shortening\nresponse to ET observed in ssTnI-expressing myocytes (4) also could\nresult from the enhanced basal myofilament Ca2 sensitivity and dimin-\nished pH sensitivity relative to cTnI-expressing myocytes (19). In the\npresent study, the influence of ET-induced alkalosis on peak contrac-\ntion was tested by inhibiting Na/H exchange in myocytes expressing\ncTnI and ssTnI.\nThe final aim of the present work was to determine the contribution\nof individual phosphorylation sites within cTnI to the PKC-mediated\nchanges in intact adult myocyte relaxation rate in response to ET.\nDetailed work on purified cTnI indicates that PKC phosphorylates three\nseparate clusters of residues, including Ser23/Ser24, Ser43/Ser45, and\nThr144 within cTnI (11, 25). However, the myocardial response to phos-\nphorylation of individual sites within the intact myocyte has yet to be\nPKC Phosphorylation of Troponin I and Contractile Function\nthoroughly investigated. Studies here are focused on investigating the\ninfluence of the rat cTnI Ser23/Ser24 and Thr144 sites on contractile\nfunction in response to ET. For studies of cTnIThr144, the goal is to\ndetermine whether Thr144 influences the peak contraction and relax-\nation response to PKC activated by ET, by substitution with the non-\nphosphorylatable Pro residue. The Ser23/Ser24 cluster also is a target for\nPKC in purified cTnI and is a major myofilament target for protein\nkinase A (PKA) (26). Phosphorylation of this cluster by PKA signifi-\ncantly decreases myofilament Ca2 sensitivity (27) and accelerates\nrelaxation during -adrenergic activation (28). A similar increase in\nrelaxation is expected if this Ser cluster is phosphorylated by PKC in\nintact adult myocytes. For these studies, the Ser23/Ser24 residues were\nsubstituted with Ala to investigate whether this cluster influences the\nability of cTnI to maintain relaxation in response to ET activation of\nPKC.\nEXPERIMENTAL PROCEDURES\nMutagenesis Strategy—Wild-type rat cTnI cDNA (kind gift of A.\nMurphy) was used to construct the cTnIT144P and cTnIS23A/S24A\nmutants. Primers used for mutagenesis of cTnIT144P were 5-gcaagtt-\ntaagcggccgcctctccgaagagtg-3 (sense) and 5-cactcttcggagaggcggccgct-\ntaaacttgc-3 (antisense), and for cTnIS23A/S24A the primers were\n5-ccgacgtcgcgccgcggccaactaccg-3 (sense) and 5-cggtagttggccgcg-\ngcgcgacgtcgg-3 (antisense) using a pGEM-3Z vector and the Strat-\nagene QuikChange site-directed mutagenesis kit (29). The presence of\neach mutant was verified by DNA sequencing.\nGeneration of Adenoviral Vectors—Recombinant adenovirus vectors\nwere constructed by cotransfection of pCA4cTnI cDNA with pJM17 by\ncalcium phosphate into a HEK 293 cell\nline. Replication-deficient,\nrecombinant adenovirus containing wild-type rat cTnI, wild-type\nssTnI, or mutant ssTnI cDNAs, a cytomegalovirus promoter, and an\nSV40 polyadenylation signal were obtained following homologous\nrecombination (19).\nMyocyte Isolation—Adult rat ventricular myocytes were isolated as\ndescribed earlier (4, 30). Myocytes were plated on laminin-coated cov-\nerslips in Dulbecco’s modified Eagle’s medium supplemented with 50\nunits/ml penicillin, 50 g/ml streptomycin (pen/strep) and 5% serum\nfor 2 h prior to replacing the medium with serum-free Dulbecco’s mod-\nified Eagle’s medium containing recombinant adenovirus vector (mul-\ntiplicity of infection  500). Gene transfer of exogenous TnI at a mul-\ntiplicity of infection of 500 results in virtually all cells expressing and\nincorporating exogenous TnI into the myofilament (29). After 1 h, Dul-\nbecco’s modified Eagle’s medium with pen/strep (2 ml) was added, with\nregular replacement of media over 4 days in culture.\nAnalysis of Protein Expression—Control and virus-treated adult rat\ncardiac myocytes were scraped from coverslips into sample buffer 4\ndays after gene transfer (31). In some experiments, myocytes were per-\nmeabilized in 0.1% Triton X-100 for 1 min to evaluate incorporation of\nTnI protein into the myofilament after gene transfer. In earlier work,\nexpression in intact and permeabilized myocytes was comparable when\nmyofilament proteins were incorporated into the myofilament (29, 32).\nProteins were then separated by gel electrophoresis and blotted onto\npolyvinylidene difluoride membrane for 2000 V-h with immunodetec-\ntion carried out as described earlier (30) using a 1:1000 dilution of MAB\n1691 (Chemicon Inc., Temecula, CA), a monoclonal antibody that rec-\nognizes all TnI isoforms. A phospho-Ser23/Ser24-specific polyclonal\nantibody (1:1000; Cell Signaling Technology, Beverly, MA) and a paired\nhorseradish peroxidase-conjugated secondary antibody were used to\ndetect Ser23/Ser24 phosphorylation. Myocytes collected for detection of\nphosphorylation were incubated at 37 °C with calyculin A (10 nM) alone\nor with the selected PKA and PKC agonist and/or inhibitor for time\nintervals ranging from 2.5 min up to 1 h.\nMeasurement of Sarcomere Shortening in Single Cardiac Myocytes—\nMyocytes to be used for shortening assays were transferred to a Plexiglas\nstimulation chamber consisting of 8 wells containing platinum elec-\ntrodes. Myocytes were stimulated electrically (2.5-ms pulse, 0.2 Hz) in\nmedium 199 supplemented with pen/strep (M199), 10 mM HEPES, 0.2\nmg/ml bovine serum albumin, and 10 mM glutathione (M199), and\nthe media was replaced every 12 h for days 2– 4 post-gene transfer (33).\nThe voltage was set so that 20% of the myocytes were stimulated on\neach coverslip. Four days after gene transfer, individual coverslips were\ntransferred to a temperature-controlled (37 °C) chamber mounted on a\nNikon microscope stage. A video-based detection system (Ionoptix,\nMilton, MA) was used to detect sarcomere shortening in intact myo-\ncytes. For frequency-response studies, myocytes were stimulated at\neach frequency (0.2, 0.5, 0.75, 1, and 2 Hz) until a steady state contrac-\ntion was reached. Sarcomere shortening also was recorded at 0.2 Hz\nbefore and after the increases in stimulation frequency to ensure there\nwere no changes in the rate of contraction, peak contraction, and rate of\nrelaxation after testing at all frequencies. Experiments were recorded\nusing SarcLen software, and an average of 10 twitches/myocyte was\ncollected under basal conditions. Signal-averaged data were analyzed to\ndetermine resting sarcomere length (SLR), percent change in peak sar-\ncomere shortening above base line (Sb), time to peak shortening (TTP),\ntime to 25, 50, and 75% relaxation (TTR25, TTR50, and TTR75, respec-\ntively), and maximum normalized shortening and relaxation velocities\n(dl/dtmax, dl/dtmax, respectively). Media containing ET (10 nM),\n5-N(ethyl-N-iospropyl)amiloride (EIPA, 500 nM) or both reagents were\nperfused via separate individual perfusion tubing to eliminate introduc-\ntion of residual agonists/inhibitors.\nExperiments for Ca2 transient and sarcomere shortening measure-\nments were carried out on myocytes loaded with Fura-2 AM, as\ndescribed previously (34). Briefly, myocytes were loaded for 5 min with\n5 M Fura-2 AM (final Me2SO  0.6%) at 37 °C followed by a 5-min\nwash-out for de-esterification. Myocytes were then stimulated at 0.2 Hz\nat 37 °C, and the base-line sarcomere length and Ca2 transient were\nrecorded from a single cell. Recordings were then made 5, 10, and 15\nmin after the addition of ET (10 nM), with the cell maintained in the dark\nbetween sampling times. Optimal cTnI phosphorylation was observed\nin response to 10 –100 nM ET in an earlier study (4). Ten twitches/\nmyocyte were collected for each sample. In addition to sarcomere short-\nening measurements, the resting and peak ratios, maximum Ca2 tran-\nsient rate (dCa/dtmax), maximum Ca2 decay rate (dCa/dtmax), and\ntime to 50 and 75% decay (TTD50, TTD75, respectively) in the Ca2\ntransient were measured. The Ca2 transient and shortening measure-\nments were maintained over this same time interval in myocytes main-\ntained under basal conditions.\nStatistical Analysis—Values are expressed as mean  S.E. Grouped\ncomparisons are analyzed using a Student’s t test or analysis of variance\ntesting (ANOVA). Statistical differences detected by ANOVA are com-\npared with a Newman-Keuls multiple comparison test with p  0.05\nconsidered significantly different.\nRESULTS\nRelationship between Ca2 Transient and Contractile Response to\nEndothelin—Sarcomere shortening and Ca2 transients were measured\nbefore and after ET in Fura-2-loaded myocytes expressing cTnI or ssTnI\n4 days after gene transfer (Fig. 1, TABLE ONE). The increased peak\nshortening and enhanced relaxation rate observed in response to 10 nM\nET in earlier work on myocytes expressing cTnI (4), was similarly\nPKC Phosphorylation of Troponin I and Contractile Function\nobserved in myocytes loaded with Fura-2 (Fig. 1A). In myocytes\nexpressing nonphosphorylatable ssTnI, the delayed relaxation and loss\nof the enhanced peak shortening observed in response to ET (4) simi-\nlarly developed in Fura-2-loaded cells (Fig. 1B, TABLE ONE).\nCa2 transient measurements also were examined in these myocytes.\nThe time to 50% decay for basal Ca2 transients reportedly slows in\ntransgenic mouse myocytes expressing ssTnI (20), and similar results\ndeveloped in rat myocytes expressing ssTnI (TABLE TWO). Peak Ca2\nand the TTP remained unchanged in the ssTnI-expressing myocytes\ncompared with values observed in controls at 0.2 Hz. The Ca2 tran-\nsient change observed in response to ET was then analyzed in myocytes\nexpressing cTnI and ssTnI (TABLE ONE). The percent change in peak\nCa2 amplitude and maximum rate of Ca2 decay (dCa/dtmax) in\nresponse to ET was not significantly changed in myocytes expressing\neither cTnI or ssTnI (Fig. 1, A and B; TABLE ONE). However, signifi-\ncant delays were observed in the time to 75% Ca2 decay in both groups\nof myocytes. The similar Ca2 transient responses to ET coupled with\nthe divergent shortening response of cTnI- versus ssTnI-expressing\nmyocytes indicates that cTnI contributes significantly to the enhanced\npeak shortening and accelerated relaxation rate observed in response to\nPKC activation by ET compared with myocytes expressing ssTnI.\nAdditional studies were performed to investigate whether the contri-\nbution of phosphorylated TnI to the ET-mediated change in contractile\nfunction is diminished with higher frequencies. The shortening  fre-\nquency relationship described previously in rodents (33, 35) was similar\nunder basal conditions and in response to ET over a range of frequencies\n(0.2–2 Hz), (TABLE THREE) in control myocytes. The percent decrease\nin peak shortening, dl/dtmax and dl/dtmax in response to increased\nfrequency was similar before and after ET. The consistency of this\nresponse provides evidence that the contribution of cTnI phosphoryla-\ntion to contractile function is not altered by frequency-induced changes\nin the Ca2 cycle.\nContribution of Cellular Alkalosis to the Myofilament Response to\nEndothelin—An important component of the contractile response to\nET depends on activation of Na/H exchange and the generation of a\ncellular alkalosis (22, 23). Experiments were carried out in the presence\nof the amiloride analog EIPA, an inhibitor of Na/H exchange (23, 36),\nto identify responses resulting from alkalosis. The myocyte response to\nEIPA is comparable in nontreated myocytes and myocytes expressing\ncTnI after gene transfer (results not shown); the pooled results are\nshown in Fig. 2. Myocyte shortening amplitude in myocytes expressing\nPKC Phosphorylation of Troponin I and Contractile Function\ncTnI decreased slightly after the addition of EIPA but did not signifi-\ncantly influence peak amplitude in myocytes expressing ssTnI (Fig. 2, A\nand B). Resting sarcomere length (SLR), dl/dtmax, and dl/dtmax\n(dl/dt and dl/dt in Fig. 2, respectively) were not significantly altered\nby EIPA in myocytes expressing cTnI or ssTnI (Fig. 2B). However, EIPA\nblocked a major portion (70%) of the ET-induced increase in peak\nshortening amplitude observed in cTnI-expressing myocytes. In con-\ntrast, EIPA did not significantly modify the accelerated relaxation\nresponse to ET in myocytes expressing cTnI, and the delayed relaxation\nobserved in response to ET was comparable with and without EIPA in\nmyocytes expressing ssTnI (Figs. 1 and 2). These results support the idea\nthat Na/H exchange and the ensuing alkalosis is important for the\nenhanced peak shortening response, whereas cTnI plays a more critical\nrole in the accelerated relaxation rate response to ET.\nRole of cTnI Phosphorylation Residues in the Endothelin Response—\nTo determine the role of the cTnIThr144 phosphorylation site in the\nrelaxation response, the ssTnI Pro residue was substituted for Thr144 to\nform cTnIT144P, and replacement was measured 4 days after gene\ntransfer. The stoichiometric replacement of cTnI with cTnIT144P-\nFLAG was 46  2% (n  13) and comparable with the 53  2% (n  14)\nreplacement with cTnI-FLAG 4 days after gene transfer (Fig. 3).\nReplacement of nontagged cTnI and cTnIT144P is expected to be com-\nparable in myocytes over the same time interval (29, 37). Total TnI\nexpression remained constant in myocytes expressing cTnI and\ncTnIT144P based on the ratio of TnI intensity to the intensity of a\nsilver-stained band on the gel (cTnI set at 1.0; cTnIT144P  1.13  0.10,\nn  11), and isoform expression of tropomyosin and troponin T was\ncomparable in myocytes expressing cTnI and cTnIT144P (results not\nshown). Expression of total TnI also was comparable in intact (I) and\npermeabilized (P) myocytes (Fig. 3, I/P ratio for cTnI  1.10  0.08, n \n5; cTnIT144P  1.05  0.06, n  5), which is consistent with myofila-\nment incorporation of the delivered TnI.\nMeasures of basal contractile function in myocytes expressing\ncTnIT144P are not significantly different from values obtained from\nmyocytes expressing cTnI (TABLE TWO, Fig. 4A). These results are in\nagreement with the earlier reverse substitution of Pro110 with Thr in fast\nskeletal TnI (38). Base-line Ca2 transients in myocytes expressing\ncTnIT144P (Fig. 4B) also showed no significant differences from control\nvalues. Peak shortening increased significantly in response to ET in\ncTnIT144P-expressing myocytes (Fig. 4) and was comparable with the\nET-induced enhanced peak shortening in control myocytes (Fig. 1B).\nHowever, several indices of relaxation, including dl/dtmax and TTR75,\nwere significantly delayed in response to ET in myocytes expressing\ncTnIT144P compared with controls (TABLE ONE). The amplitude of\nthe Ca2 transient response to ET was not significantly different com-\npared with control values for myocytes expressing cTnIT144P (TABLE\nONE, Fig. 4). As with control myocytes, a frequency change from 0.2 to\n2 Hz produced similar changes before and after ET in myocytes express-\ning cTnIT144P (results not shown). The major difference between myo-\ncytes expressing cTnIT144P versus cTnI was the decrease in relaxation\nrate in response to ET; this finding indicates a specific role for Thr144 in\naccelerating relaxation in response to ET.\nThe contribution of Ser23/Ser24 phosphorylation to the ET-induced\nchange in contractile function also was determined. A phospho-specific\nSer23/Ser24 antibody was used for these studies. Myocytes were incu-\nbated with the PKA agonist dobutamine (DOB) for 2 min to verify\ndetection of the well documented PKA-mediated cTnISer23/Ser24 phos-\nPKC Phosphorylation of Troponin I and Contractile Function\nphorylation site (39, 40) (Fig. 5A). Detection of cTnI phosphorylation by\nthis antibody was specific, as it was blocked when, in addition to DOB,\nthe -adrenergic inhibitor propranolol was added to myocytes (results\nnot shown). There was no significant phosphorylation detected in myo-\ncytes incubated with ET for 10 min with (Fig. 5A) or without the PKC\ninhibitor bis-indolylmaleimide-1 (bis-1, results not shown). However,\nsignificant phosphorylation of Ser23/Ser24 developed in control myo-\ncytes over a 1-h time span, and this ET-mediated phosphorylation was\nsignificantly reduced in the presence of the PKC inhibitor bis-1 but not\nby the -adrenergic antagonist propranolol (Fig. 5A). Treatment of\nmyocytes with other PKC agonists, including phenylephrine and phor-\nbol 12-myristate 13-acetate for 1 h also resulted in significant cTnI\nphosphorylation of Ser23/Ser24. Incubation with fibroblast growth fac-\ntor-2 was previously shown to activate PKC (41), yet this agonist did not\nsignificantly influence cTnI phosphorylation over a 1-h time span\n(results not shown). These results provide evidence for temporal phos-\nphorylation of cTnI Ser23/Ser24 in response to PKC activation by several\nPKC agonists.\nTo directly determine the contribution of Ser23/Ser24 to the contract-\nile function response to ET, cTnIS23A/S24A was expressed in adult\nmyocytes. By 4 days after gene transfer, 44  5% of cTnI was replaced\nwith the cTnIS23A/S24A (n  15, Fig. 5B). DOB phosphorylation of\ncTnI was significantly reduced in myocytes expressing cTnIS23A/S24A\nor cTnIS23A/S24A-FLAG (results not shown). In addition, Ser23/Ser24\nphosphorylation was significantly attenuated after 1 h of ET in myocytes\nexpressing tagged and nontagged cTnIS23A/S24A (Fig. 5C). This S23A/\nS24A substitution has previously been shown to have little influence on\nbasal myofilament function (42), and no significant differences were\nobserved in SLR, peak amplitude, dl/dtmax, ordl/dt max in myocytes\nexpressing cTnIS23A/S24A compared with controls (results not\nshown). Control myocytes expressing cTnI or\nthe substituted\ncTnIS23A/S24A responded similarly to ET over the first 15 min (Fig. 6).\nHowever, a significant difference in the response was observed 1 h after\nthe addition of ET in myocytes expressing cTnIS23A/S24A compared\nPKC Phosphorylation of Troponin I and Contractile Function\nwith control values (Fig. 6). Specifically, peak shortening amplitude was\nnot increased and both dl/dtmax and TTR75 indicated relaxation was\nsignificantly prolonged in myocytes expressing cTnIS23A/S24A com-\npared with control myocytes and were comparable with the results\nobtained in myocytes expressing ssTnI (Fig. 6). These results demon-\nstrate Ser23/Ser24 phosphorylation contributes to the accelerated relax-\nation during more prolonged myocyte PKC activation by ET.\nDISCUSSION\nResults from the present study demonstrate the important direct role\nfor cTnI phosphorylation in accelerating relaxation in response to PKC\nactivated by ET in adult myocytes (Figs. 1–3). Our results now demon-\nstrate a clear role for cTnI Thr144 and Ser23/Ser24 in contributing to the\naccelerated relaxation in response to ET (Figs. 4 and 6). Substitution\nwith a Pro residue at Thr144 results in the delayed relaxation previously\nobserved in myocytes expressing nonphosphorylatable ssTnI and pro-\nvides evidence that the Thr144 site contributes to accelerated relaxation\nduring the ET response. The Ser23/Ser24 cluster does not appear to\ncontribute to the initial response to ET but, instead, is important for the\naccelerated relaxation during more prolonged stimulation by ET. This\nfinding indicates that the relaxation rate response to ET-induced PKC\npathway activation depends on temporal phosphorylation of individual\nclusters within cTnI.\nTnI Phosphorylation Plays a Direct Role in the Relaxation Response to\nEndothelin—Results from the present study demonstrate the important\nrole of cTnI phosphorylation in the relaxation response during the pos-\nitive inotropic response to ET, independent of cellular Ca2 handling.\nThe only aspect of Ca2 transient that changed significantly in response\nto 10 nM ET was the percent change in TTD75, which decreased in both\ncontrol and ssTnI-expressing myocytes (Fig. 1, TABLE ONE). All other\nmeasurements of the Ca2 transient remained unchanged during the\nET response, and work by other laboratories supports this finding (43,\n44). There are reports that ET activation of PKC enhances the Ca2\ntransient at higher doses of ET (45), which may be due to increased Ca2\nentry via the L-type Ca2 channel (46). However, results from Ebihara et\nal. (47) indicate the increased Ca2 transient is not necessary for the\nenhanced contractile response to ET.\nExperiments with the NHE1 inhibitor EIPA also demonstrate that\ninhibition of NHE1 prevents the ET-mediated enhancement in peak\nshortening amplitude (Fig. 2). ET in the concentration range used here\nactivates PKC-induced NHE1 (48) and causes a cellular alkalosis (22,\n23). The alkalosis-mediated increase in myofilament Ca2 sensitivity\n(24) is responsible for the increased peak contractile shortening (22, 23).\nThis enhancement in peak shortening is absent from myocytes express-\ning ssTnI (Fig. 1), which is likely due to the combination of increased\nbasal myofilament Ca2 sensitivity (19, 20) and a decreased pH response\nto alkalosis in these myocytes compared with cTnI-expressing myocytes\n(19).\nIn contrast to peak shortening, the divergent influence of cTnI versus\nssTnI on the relaxation response to ET was not influenced significantly\nby EIPA (Fig. 2). Although the slowing of relaxation in myocytes\nexpressing ssTnI was attenuated by the presence of EIPA during the ET\nresponse, relaxation was significantly accelerated in myocytes express-\ning cTnI versus ssTnI during this response. Our results now provide\nevidence that phosphorylated cTnI maintains relaxation regardless of\nthe alkalosis. The delayed relaxation response to ET observed in myo-\ncytes expressing cTnIT144P also supports this finding (Fig. 4).\nWhat Role Does cTnI Phosphorylation Play in the Contractile\nResponse to ET?—The role of cTnI phosphorylation in the contractile\nfunction response to PKC agonists remains controversial. Earlier in\nvitro work showed that PKC phosphorylation of cTnI decreases myofil-\nament Ca2 sensitivity as well as maximum actomyosin ATPase activity\n(11, 25). PKA phosphorylation of cTnI also decreases myofilament Ca2\nsensitivity and results in accelerated myocyte relaxation (27, 28), and\nsubstitution of Ser23/Ser24 with Asp to mimic phosphorylation acceler-\nates relaxation in transgenic mice (39). However, PKC phosphorylates\nSer43/Ser45 and Thr144 in addition to the Ser23/Ser24 cluster (11). More\nrecent studies in which the Ser23/Ser24, Ser43/Ser45, and Thr144 clusters\nare all replaced with Ala in knock-in mice (49, 50) or substituted with\nAsp to mimic phosphorylation in transgenic mice (37) suggest that cTnI\nworks primarily to delay cardiac relaxation during PKC activation.\nThese results are in contrast to results presented here demonstrating an\naccelerated relaxation response attributable to cTnI phosphorylation\n(TABLE ONE), and the in vitro work demonstrating decreased myofil-\nament Ca2 sensitivity (11). Differences in experimental conditions,\nsuch as load and/or temperature in these various models, do not appear\nto account for the divergent functional response. We propose that these\nseemingly divergent responses may be due to a hierarchy of cTnI phos-\nphorylation in response to a given agonist, with different cTnI clusters\nphosphorylated in response to PKC activation. Ser43/Ser45 phosphoryl-\nPKC Phosphorylation of Troponin I and Contractile Function\nation may have a dominant influence on contractile relaxation via its\ninfluence on maximum ATPase activity when all five phosphorylation\nsites are replaced by Ala (49) or replaced with a negatively charged Asp\n(37). However, PKC activation by a specific agonist may differentially\nand/or temporally phosphorylate one or more of the cTnI residues\nrather than phosphorylate all three clusters. In support of this idea, in\nvitro motility assays carried out with cTnISer43/Ser45 containing\ncharged Glu substitutions demonstrated a decrease in myofilament\nCa2 sensitivity and decreased sliding speed, whereas cTnIT144G only\ndecreased myofilament Ca2 sensitivity (51). NMR analysis of Ser43/\nSer45 substituted with Asp provides evidence that phosphorylation of\nthis site may be dominant in that it influences the Ca2 binding affinity\nto TnC as well as protein-protein interactions between TnI and TnC\n(52). Thus, the divergent results are predicted to stem from differential\nphosphorylation of the three cTnI phosphorylation clusters and/or the\nrelative dominance of one cluster in response to activation and translo-\ncation of specific PKC isoforms by a given agonist. This idea will be\nimportant for rigorous testing, with a variety of agonists under physio-\nlogical and pathophysiological conditions, to better understand the\nPKC signaling pathway and its ability to modulate both cTnI phospho-\nrylation and contractile performance.\nRole of Individual cTnI-phosphorylated Residues in the Contractile\nResponse to ET—Detailed studies on purified cTnI support the idea that\nthe residue phosphorylated by PKC is important for the resulting func-\ntional relaxation response. Phosphorylation of Thr144 in cTnI produces\na rightward shift in myofilament Ca2 sensitivity for both actomyosin\nATPase activity (11) and force generation in permeabilized trabeculae\n(51, 53). The anticipated cellular response is an increase in myocyte\nrelaxation rate. Our results now show that substitution of Thr144 with\nPro significantly delays relaxation during the ET response (Fig. 4,\nTABLE THREE). The Thr144 site is particularly important because the\ninhibitory peptide region encompassing this phosphorylation target\n(amino acids 129 –149) is important for determining TnI binding to\nactin in the absence of Ca2 and diminished binding to actin in the\npresence of Ca2 (54). Thus, the inhibitory peptide region is important\nfor the molecular switch activity of TnI (55, 56). The delayed relaxation\nresponse to ET in myocytes expressing cTnIT144P compared with con-\ntrol myocytes is consistent with the idea that phosphorylation of this\nresidue in response to ET plays a significant role in the accompanying\nrelaxation rate response.\nExperiments with the phospho-cTnISer23/Ser24 antibody and with\ncTnIS23A/S24A also demonstrate a key role for this cluster during\nmore extended activation of PKC by ET. Phosphorylation of Ser23/Ser24\ndecreases myofilament Ca2 sensitivity and accelerates relaxation in\nresponse to PKA activation (27, 28), and similar shifts in myofilament\nCa2 sensitivity are reported in response to PKC phosphorylation of\nSer23/Ser24 (11). Our results now demonstrate a temporal pattern of\nSer23/Ser24 phosphorylation that is important for enhanced relaxation\nin response to longer intervals of ET in intact myocytes (Fig. 6). In\nbiochemical experiments with purified cTnI, longer treatment with\nPKC also was required for Ser23/Ser24 phosphorylation (11), yet it\nremained unclear whether this result indicated that there was temporal\nphosphorylation in the intact cell or instead indicated that Ser23/Ser24\nwas not a typical in vivo target for activated PKC. Results with myocytes\nexpressing cTnIS23A/S24A (Fig. 6) clearly demonstrate that there is\ntemporal phosphorylation of this site by PKC activated in response to\nET. Several PKC agonists, including ET, are released in a paracrine/\nautocrine pattern (57, 58), and phosphorylation of Ser23/Ser24 during\nmore prolonged intervals of PKC stimulation would then be important\nfor preventing delays in cellular relaxation. Taken together, the results\nobtained in experiments to investigate Thr144 and Ser23/Ser24 provide\nevidence that both clusters are important in maintaining relaxation dur-\ning PKC activation. The relative importance, as well as the interactive\neffects, of these sites and the Ser43/Ser45 site in response to PKC ago-\nnists, which appears primarily to decrease peak contraction and slow\nrelaxation (51), will be important to investigate in the future for a com-\nplete understanding of the role played by cTnI phosphorylation in mod-\nulating cardiac function.\nAcknowledgments—We thank Andrea Borton (Cell Signaling Inc.) for the gift\nof the phospho-Ser23/Ser24 antibody. We also thank Ayako Ohata for technical\nassistance, and Sharlene Day, Joseph Metzger, and Daniel Michele for helpful\ncomments on earlier versions of this manuscript.\n1. Gwathmey, J., and Hajjar, R. (1990) Circ. Res. 67, 744 –752\n2. Buenaventura, P., Cao-Danh, H., Takeuchi, K., Takahashi, S., Simpleaceanu, E.,\nMcGowan, F. J., and del Nido, P. (1995) Ann. Thorac. Surg. 60, Suppl. 6, S505–S508\n3. Puceat, M., Clement, O., Lechene, P., Pelosin, J., Ventura-Clapier, R., and Vassort, G.\n4. Westfall, M. V., and Borton, A. R. (2003) J. Biol. Chem. 278, 33694 –33700\n5. Metzger, J. M., and Westfall, M. V. (2004) Circ. Res. 94, 146 –158\n6. He, J. Q., Pi, Y., Walker, J. W., and Kamp, T. J. (2000) J. Physiol. 524, 807– 820\n7. Alden, K. J., Goldspink, P. H., Ruch, S. W., Buttrick, P. M., and Garcia, J. (2002) Am. J.\n9. Noland, T. A., Jr., and Kuo, J. F. (1991) J. Biol. Chem. 266, 4974 – 4978\n10. Mazzei, G. J., and Kuo, J. F. (1984) Biochem. J. 218, 361–369\n11. Noland, T. A., Jr., Raynor, R. L., and Kuo, J. F. (1989) J. Biol. Chem. 264, 20778 –20785\n12. Noland, T. A., Jr., and Kuo, J. F. (1992) Biochem. J. 288, 123–129\n13. Noland, T. A., Jr., and Kuo, J. F. (1993) Biochem. Biophys. Res. Commun. 193, 254 –260\n14. Clement, O., Puceat, M., Walsh, M., and Vassort, G. (1992) Biochem. J. 285, 311–317\n15. Bodor, G. S., Oakeley, A. E., Allen, P. D., Crimmins, D. L., Ladenson, J. H., and\n21. DeSantiago, J., Maier, L. S., and Bers, D. M. (2002) J. Mol. Cell Cardiol. 34, 975–984\n22. Kramer, B. K., Smith, T. W., and Kelly, R. A. (1991) Circ. Res. 68, 269 –279\n23. Goldberg, A. T., Bond, B. R., Mukherjee, R., New, R. B., Zellner, J. L., Crawford, F. A.,\nPKC Phosphorylation of Troponin I and Contractile Function",
         "M. Westfall,A. M. Lee,D. Robinson",
         null
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "ABSTRACT\nThis paper documents a study, carried out in the academic\nyear 2003-2004, on ﬁfteen factors that may inﬂuence per-\nformance on a ﬁrst year object-oriented programming mod-\nule. The factors included prior academic experience, prior\ncomputer experience, self-perception of programming per-\nformance and comfort level on the module and speciﬁc cog-\nnitive skills. The study found that a student’s perception of\ntheir understanding of the module had the strongest corre-\nlation with programming performance, r = 0.76, p < 0.01.\nIn addition, Leaving Certiﬁcate (LC) mathematics and sci-\nence scores were shown to have a strong correlation with\nperformance. A regression module, based upon a student’s\nperception of their understanding of the module, gender, LC\nmathematics score and comfort level was able to account for\n79% of the variance in programming performance results.",
         "INTRODUCTION\nStudent retention on third-level (post high school or equiv-\nalent) Computer Science (CS) and Information Technology\n(IT) courses is a signiﬁcant problem. Students ﬁnd com-\nputer programming diﬃcult and struggle to master the core\nconcepts. A multi-national, multi-institutional study on the\nprogramming skills of ﬁrst year CS students found that stu-\ndents struggled to achieve an average above 30% on assess-\nments administered as part of their study [14]. Further-\nmore, introductory programming modules tend to have a\nPermission to make digital or hard copies of all or part of this work for\npersonal or classroom use is granted without fee provided that copies are\nnot made or distributed for proﬁt or commercial advantage and that copies\nbear this notice and the full citation on the ﬁrst page. To copy otherwise, to\nrepublish, to post on servers or to redistribute to lists, requires prior speciﬁc\npermission and/or a fee.\nSIGCSE’05 February 23–27, 2005 St. Louis, Missouri, USA.\nCopyright2005ACM1-58113-997-7/05/0002...$5.00.\nvery high student to lecturer ratio (100:1 or greater) and\noften lecturers do not know how well students are doing un-\ntil after the ﬁrst assessment. In general the ﬁrst assessment\ndoes not take place until six or eight weeks after the mod-\nule has commenced and given the typically high number of\nstudents, marking the assessments can take a considerable\nlength of time. At this stage, it may be too late to intervene\nto prevent struggling students from failing. Even if inter-\nvention is possible, a lecturer is unlikely to know a student\nwell enough or be able to identify individual student prob-\nlems and therefore recognize the most suitable interventions\nto make.\nThe research documented in this paper is part of a lon-\ngitudinal study on early identiﬁable factors that inﬂuence\nperformance on an introductory programming module.\nIf\nsuch factors can be identiﬁed then it may be possible to de-\nvelop a tool to provide an early diagnosis of a student’s likely\nperformance on a programming module. Interested parties\ncould use the tool to make more informed decisions on ap-\npropriate courses of action and to decide upon personalized\ninterventions that foster a student’s intellectual strengths.\n2. RELATED RESEARCH\nWhile a considerable amount of research has been carried\nout on factors that aﬀect programming performance our in-\nterest is on factors that aﬀect programming performance\non an introductory third-level object-oriented programming\nmodule where such factors can be determined early in the\nacademic year. These factors can be categorized as: (1)\nprevious academic and computer experience, (2) cognitive\nskills, (3) personal information and (4) experience on the\nmodule. A brief review of some research studies in each of\nthese categories is presented next.\nPrevious academic experience and programming experi-\nence have often been cited as predictors of programming suc-\ncess. Several studies have found that mathematical ability\nand exposure to maths courses are important predictors of\nperformance on introductory computer science modules [2,\n3, 4, 6, 13, 19]. Similarly, although less studied, performance\nin and experience of science subjects has also been shown\nto be important [2, 19]. Studies have also found prior pro-\ngramming experience and non-programming computer ex-\nperience (for example, experience of computer applications,\nemailing, game playing and surﬁng the web) to be related\nto programming performance [9, 10, 6, 4, 18].\nThe role of cognitive factors in programming has also re-\nceived research attention. Certain cognitive factors, includ-\ning problem-solving, abstract reasoning, problem translation\nskills, logical ability and cognitive style have been identiﬁed\nas possible predictors [7, 12, 11, 15].\nNumerous studies have been carried out on demographic\nand self-reported personal information. Some studies have\nexamined speciﬁc attributes related to study, for example\npreference for working alone or in a group to solve a pro-\ngramming problem and encouragement or support from oth-\ners to study computers [4, 2, 8].\nIn recent times researchers have examined the relation-\nship between students’ expectations of and experiences on\nan introductory computing module. A positive relationship\nbetween a student’s mental models of programming and self-\neﬃcacy for programming and performance has recently been\nidentiﬁed [18]. The grade a student expected to achieve in\nan introductory module was found to be the most impor-\ntant indicator of performance in another recent study[17].\nA recent longitudinal study found that the most important\npredictor of students’ performance on an introductory com-\nputer science course was comfort level, determined by the\ndegree of anxiety a student felt about the course [4].\nAlthough a considerable number of research studies have\nexamined factors that inﬂuence programming performance,\ncomparisons between the various studies and application of\nthe results are diﬃcult because the studies are carried out\nusing diﬀerent parameters. These parameters include:\n• the type of students (novice to experienced program-\nmers, academic students to employees),\n• the content of the course (some courses are solely pro-\ngramming courses while others are introductory com-\nputing courses),\n• the programming language being taught (older studies\ntend to be based on procedural languages),\n• the educational setting (many of the research studies\nare based on the US educational system) and",
         "",
         "",
         "",
         "ABSTRACT\nThis paper documents a study, carried out in the academic\nyear 2003-2004, on ﬁfteen factors that may inﬂuence per-\nformance on a ﬁrst year object-oriented programming mod-\nule. The factors included prior academic experience, prior\ncomputer experience, self-perception of programming per-\nformance and comfort level on the module and speciﬁc cog-\nnitive skills. The study found that a student’s perception of\ntheir understanding of the module had the strongest corre-\nlation with programming performance, r = 0.76, p < 0.01.\nIn addition, Leaving Certiﬁcate (LC) mathematics and sci-\nence scores were shown to have a strong correlation with\nperformance. A regression module, based upon a student’s\nperception of their understanding of the module, gender, LC\nmathematics score and comfort level was able to account for\n79% of the variance in programming performance results.\nINTRODUCTION\nStudent retention on third-level (post high school or equiv-\nalent) Computer Science (CS) and Information Technology\n(IT) courses is a signiﬁcant problem. Students ﬁnd com-\nputer programming diﬃcult and struggle to master the core\nconcepts. A multi-national, multi-institutional study on the\nprogramming skills of ﬁrst year CS students found that stu-\ndents struggled to achieve an average above 30% on assess-\nments administered as part of their study [14]. Further-\nmore, introductory programming modules tend to have a\nPermission to make digital or hard copies of all or part of this work for\npersonal or classroom use is granted without fee provided that copies are\nnot made or distributed for proﬁt or commercial advantage and that copies\nbear this notice and the full citation on the ﬁrst page. To copy otherwise, to\nrepublish, to post on servers or to redistribute to lists, requires prior speciﬁc\npermission and/or a fee.\nSIGCSE’05 February 23–27, 2005 St. Louis, Missouri, USA.\nCopyright2005ACM1-58113-997-7/05/0002...$5.00.\nvery high student to lecturer ratio (100:1 or greater) and\noften lecturers do not know how well students are doing un-\ntil after the ﬁrst assessment. In general the ﬁrst assessment\ndoes not take place until six or eight weeks after the mod-\nule has commenced and given the typically high number of\nstudents, marking the assessments can take a considerable\nlength of time. At this stage, it may be too late to intervene\nto prevent struggling students from failing. Even if inter-\nvention is possible, a lecturer is unlikely to know a student\nwell enough or be able to identify individual student prob-\nlems and therefore recognize the most suitable interventions\nto make.\nThe research documented in this paper is part of a lon-\ngitudinal study on early identiﬁable factors that inﬂuence\nperformance on an introductory programming module.\nIf\nsuch factors can be identiﬁed then it may be possible to de-\nvelop a tool to provide an early diagnosis of a student’s likely\nperformance on a programming module. Interested parties\ncould use the tool to make more informed decisions on ap-\npropriate courses of action and to decide upon personalized\ninterventions that foster a student’s intellectual strengths.\n2. RELATED RESEARCH\nWhile a considerable amount of research has been carried\nout on factors that aﬀect programming performance our in-\nterest is on factors that aﬀect programming performance\non an introductory third-level object-oriented programming\nmodule where such factors can be determined early in the\nacademic year. These factors can be categorized as: (1)\nprevious academic and computer experience, (2) cognitive\nskills, (3) personal information and (4) experience on the\nmodule. A brief review of some research studies in each of\nthese categories is presented next.\nPrevious academic experience and programming experi-\nence have often been cited as predictors of programming suc-\ncess. Several studies have found that mathematical ability\nand exposure to maths courses are important predictors of\nperformance on introductory computer science modules [2,\n3, 4, 6, 13, 19]. Similarly, although less studied, performance\nin and experience of science subjects has also been shown\nto be important [2, 19]. Studies have also found prior pro-\ngramming experience and non-programming computer ex-\nperience (for example, experience of computer applications,\nemailing, game playing and surﬁng the web) to be related\nto programming performance [9, 10, 6, 4, 18].\nThe role of cognitive factors in programming has also re-\nceived research attention. Certain cognitive factors, includ-\ning problem-solving, abstract reasoning, problem translation\nskills, logical ability and cognitive style have been identiﬁed\nas possible predictors [7, 12, 11, 15].\nNumerous studies have been carried out on demographic\nand self-reported personal information. Some studies have\nexamined speciﬁc attributes related to study, for example\npreference for working alone or in a group to solve a pro-\ngramming problem and encouragement or support from oth-\ners to study computers [4, 2, 8].\nIn recent times researchers have examined the relation-\nship between students’ expectations of and experiences on\nan introductory computing module. A positive relationship\nbetween a student’s mental models of programming and self-\neﬃcacy for programming and performance has recently been\nidentiﬁed [18]. The grade a student expected to achieve in\nan introductory module was found to be the most impor-\ntant indicator of performance in another recent study[17].\nA recent longitudinal study found that the most important\npredictor of students’ performance on an introductory com-\nputer science course was comfort level, determined by the\ndegree of anxiety a student felt about the course [4].\nAlthough a considerable number of research studies have\nexamined factors that inﬂuence programming performance,\ncomparisons between the various studies and application of\nthe results are diﬃcult because the studies are carried out\nusing diﬀerent parameters. These parameters include:\n• the type of students (novice to experienced program-\nmers, academic students to employees),\n• the content of the course (some courses are solely pro-\ngramming courses while others are introductory com-\nputing courses),\n• the programming language being taught (older studies\ntend to be based on procedural languages),\n• the educational setting (many of the research studies\nare based on the US educational system) and",
         "Susan Bergin,R. Reilly",
         null
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced\nmonocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and\nshedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs).\nMethods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of\nTF-positive monocytes in HRs (32.03.5%) versus LRs (11.21.2%; P0.05), along with a stronger mean fluorescence intensity\nof TF signal in HRs (7.10.5 arbitrary units [AU]) compared with LRs (5.40.4 AU; P0.05). The LPS-treated blood of the HR\ngroup contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular\naccumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs\nversus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of\nsurface TF, whereas MNCs from LRs only had a minor increase in surface TF levels.",
         "ponents of the plasma coagulation cascade is important\nfor maintenance of normal rheological properties of blood.\nFailure to manipulate TF levels available for the initiation of\nblood clotting leads to thrombotic or bleeding disorders in\nhumans. Circulating monocytes are presumably the major cell\ntype that respond to variable stimuli by developing coagulant\nactivity1 through the expression of TF.2\nOriginally, intersubject variability in developing of mono-\ncyte TF activity was described by Østerud et al.3 By compar-\ning lipopolysaccharide (LPS)-induced monocyte TF activity\nand tumor necrosis factor- (TNF-) production in a whole\nblood system, an up to a 50-fold difference between individ-\nuals was observed.4 This finding was defined as the “high–\nlow responder phenomenon.”4 Also noteworthy, the individ-\nual usually remains a high responder (HR) or low responder\n(LR) for several years.5,6 Later, high intersubject variability in\ncytokine production by LPS-stimulated monocytes was dem-\nonstrated.7 It was also shown that patients with high levels of\nTNF- production were more susceptible to heart transplant\nrejection.8 Monocytes isolated from septic shock patient\nsurvivors revealed higher TNF- production than monocytes\nfrom nonsurvivors.9\nMany studies have been undertaken to describe the signifi-\ncance of this phenomenon, but so far, no general explanation has\nbeen found. Diverse plasma factors and direct cell interactions\nplay an important role in the development of monocyte TF\nactivity.10 –15 High expression of monocyte TF activity is asso-\nciated with higher risk of acute coronary syndrome.16\nPlatelets have been suggested to be responsible for\ninducing monocyte TF activity.17 Platelet-rich plasma\ninduced significantly higher TF activity in LPS-stimulated\nmonocytes than platelet-poor plasma.18 Moreover, when\nblood cells without platelets from HRs were mixed with\nplatelet-rich plasma of an LR, LPS-induced TF activity\nwas reduced up to 76% compared with an autologous\nsystem.18 It was shown that granulocytes enhance LPS-\ninduced monocyte TF activity in a platelet-dependent\nreaction involving P-selectin, platelet factor 4, platelet-\nactivating factor, hydroxyl-eicosatetraenoic acid, and\nplatelet-derived growth factor.18 –21\nHere we report several observations concerning the rela-\ntionships between intracellular- and membrane-located TF\nantigen in resting and LPS-stimulated monocytes in groups of\nHRs and LRs, using fluorescence-activated cell sorter\n(FACS) analysis, fluorescence confocal microscopy, in-cell\nOriginal received February 24, 2005; final version accepted April 14, 2005.\nFrom the Department of Biochemistry (E.M.E., M.A.S., G.B., J.O.O., B.O.), Institute of Medical Biology, University of Tromsø, Norway; the\nDepartment of Immunology and Transfusion Medicine (F.X.E.G.), University Hospital of Northern Norway, Tromsø; and the Department of Biochemistry\n(B.P.-S., K.G.M.), University of Vermont, Burlington.\nE.M.E. and M.A.S. contributed equally to this work.\nCorrespondence to Elena M. Egorina, MD, Department of Biochemistry, Institute of Medical Biology, University of Tromsø, N-9037, Tromsø,\nArterioscler Thromb Vasc Biol.\nWestern assay, and immunoblotting. In response to LPS\nstimulation, we clearly found different patterns of intracellu-\nlar accumulation and externalization of TF onto the surface of\nMNCs isolated from whole blood of HRs and LRs.",
         "Results\nReactivity of MNCs in HRs and LRs\nThe LPS-induced responses of MNCs from individuals enrolled\nin our study have been repeatedly assessed to determine their\nindividual TF activity and concentration levels of TNF-. These\nhealthy individuals were allocated to HR or LR groups accord-\ning to criteria developed in our laboratory.16 Briefly, healthy\nvolunteers were defined as HRs if their MNC-TF activity\nexceeded 20 mU/106 cells and their plasma TNF- concentra-\ntion levels were 3 ng/mL taken from whole blood stimulated\nwith LPS for 2 hours. We assigned individuals with these\nparameters lower than indicated above to the LR group.\nMeasurements of TF procoagulant activity in lysates of\nMNCs isolated from ex vivo LPS-stimulated whole blood\nrevealed 2 distinct responses. In HRs, LPS stimulation\ninduced significantly higher values of MNC-TF activity than\nin LRs after 2 hours (50.114.3 versus 20.94.2 mU/106\ncells; P0.05) and 4 hours (56.016.3 mU/106 versus\n21.24.9 mU/106 cells; P0.05; Figure 1). We found no\nincrease in TF activity in MNC lysates from time-matched\nnonstimulated samples (Figure 1).\nEgorina et al\nTF in High and Low Responders\nThe differences of MNC-TF activity and the production of\nTNF- in our experimental model could not be caused by\nvariations in degree of cell death because the number of dead\ncells measured by trypan blue exclusion assay did not differ\nbetween groups and was within 3% of the total cell count.\nThe variable degrees of responsiveness to LPS treatment,\ncharacterized by significantly elevated TF activity in MNC\nlysates in HRs,\nthe comparisons of\nintracellular and surface TF antigen levels under resting and\nstimulated conditions in these individuals.\nled us to carry out\nBecause we planned to use immunostaining-based methods for\nnative TF antigen detection (such as fluorescence-activated cell\nsorting, fluorescence confocal microscopy and in-cell Western\nassay), we applied RNA interference against TF expression in\nplated HeLa cells. Similarly, the same antibodies detected a 3-fold\nreduction of the in-cell Western assay signal intensity in the wells\nthat contained HeLa cells transfected with siRNA against TF when\ncompared with cells transfected with control siRNA. The signal\nfrom the in-cell Western assay against GAPDH showed no differ-\nences in the same wells (Figure IB).\nTo characterize levels of TF antigen on the surface and\ninterior of resting and LPS-stimulated monocytes from HRs\nand LRs, we chose FACS analyses of whole blood, fluores-\ncence confocal microscopy, and in-cell Western assay of\npermeabilized and nonpermeabilized MNCs.\nArterioscler Thromb Vasc Biol.\nWe found no baseline difference in the surface expression\nof TF on nonstimulated CD14 cells from LRs (1.40.1%)\nand HRs (1.50.2%; Figure 3A and 3B). MFI did not differ\nbetween groups and was 4.50.3 AU and 4.20.1 AU in\nLRs and HRs, respectively (Figure 3C).\nBy use of fluorescence confocal microscopy of isolated\nMNCs in resting and LPS-stimulated conditions, we visual-\nized the distribution of TF antigen between cytoplasmic and\nmembrane compartments. Only a few CD14 cells expressed\nTF in permeabilized and nonpermeabilized conditions (Fig-\nure 4A and 4B). Furthermore, we noticed 2 phenotypes when\nimaging nonpermeabilized resting CD14 cells: 1 did not\nexpress TF on the membrane surface, whereas another did\n(Figure 4C and 4D). The pattern of TF surface distribution could\nbe characterized by the presence of 5 to 7 TF-enriched dots or\n“patches.” When permeabilized, resting CD14 cells have TF-\ncontaining speckles in the proximity of the plasma membrane,\nready to be exocytosed to the cell exterior (Figure 4E).\nAfter 2 hours of LPS stimulation, permeabilized mono-\ncytes had their membranes enriched with TF antigen. The\ncytoplasm was depleted from TF-containing granules except\nfor a strong TF-positive staining unilaterally from the nucleus\n(Figure 4F). This could be newly synthesized TF protein.\nTo verify TF antigen distribution data from flow cytometry\nand fluorescence confocal microscopy, we used a quantitative\nimmunofluorescence staining in-cell Western assay.\nsignificantly higher LPS-induced accumulation of intracellu-\nlar TF antigen in LRs compared with HRs (Figure 5B). At the\nsame time, LPS treatment caused a 4-fold increase of surface\nTF signal intensity in HRs. However, no such increase was\nfound in the surface TF signal of LRs (Figure 5B).\nData from the immunoblotting of MNC lysates against TF\nantigen confirmed that after 2 hours of LPS stimulation,\nMNCs of LRs indeed accumulated TF antigen, whereas in\nHRs,\nthis accumulation was minor (Figure 5C and 5D).\nMNCs from time-matched samples of LRs experienced vir-\ntually no accumulation of the TF antigen in contrast to the\nsignificant accumulation of TF in MNCs of HRs (Figure 5D).\nComparison of real-time PCR data on de novo TF synthesis in\nLPS-stimulated conditions revealed similar mean cycle threshold\nvalues in MNCs of HRs and LRs (Table I, available online at\nhttp://atvb.ahajournals.org). Therefore, LPS stimulation is not only\nnecessary for the induction of TF synthesis, but also for its transport\nvia exocytotic pathway and shedding from the cell membrane.\nDiscussion\nIn the present ex vivo study, using a whole blood system, we\nreport several novel observations characterizing the distribu-\ntion of TF antigen between the interior and surface of MNCs\nfrom HRs and LRs. MNCs from representatives of these\ngroups exhibit distinct patterns of intracellular TF accumula-\ntion, externalization onto cell membrane, and shedding of\nTF-containing microparticles in LPS-treated conditions.\nEgorina et al\nTF in High and Low Responders\nCD14 cells expose TF on the cell surface after LPS-\nstimulation when compared with LRs.\nData of FACS analyses were in accordance with our\nfindings from fluorescence confocal microscopy, in which we\ndemonstrated redistribution of TF antigen on stimulation with\nLPS onto the cell surface. In stimulated conditions, TF was\ntransported and localized mostly on the membrane because\nthere was a clear depletion of TF staining from the cytoplasm.\nIntense dots present unilaterally from the nucleus could\nrepresent de novo synthesized TF.\ntime-matched nonstimulated cells. Together, it is possible that\nmonocytes from HRs could synthesize equal or even higher\namounts of TF antigen than those from LRs.\nArterioscler Thromb Vasc Biol.\nImmunostaining against TF and fluorescent confocal mi-\ncroscopy of permeabilized and nonpermeabilized resting\nmonocytes support our data from flow cytometry because\nonly few monocytes expressed TF antigen. The localization\npattern of TF on the monocyte surface in the present study is\nconfined to dotted structures or “patches.” Intracellular local-\nized TF in CD14 cells was restricted to round vesicle-like\nstructures in the proximity of the plasma membrane. Mulder\net al demonstrated the presence of an intracellular TF pool\nassociated with caveolae and multivesicular bodies.24\nOur data obtained from FACS and immunoblotting are\nconsistent with the previous accepted opinion that resting human\nmonocytes contain nearly undetectable levels of TF antigen.25–30\nHowever, we provide new information about the relationship\nbetween the intracellular and surface pool of TF in MNCs by\nusing in-cell Western assay. This enzyme-independent method\nof antigen detection via antibodies conjugated with infrared dyes\ngives highly reproducible data characterizing antigen levels.\nComparisons of the TF signal from resting MNCs between\npermeabilized and nonpermeabilized conditions revealed that in\nresting conditions, MNCs contain equally distributed surface\nand intracellular TF antigen in HRs and LRs.\nIn conclusion, MNCs of HRs and LRs contain similar, nearly\nundetectable intracellular levels of TF antigen at baseline condi-\ntions. On LPS stimulation, TF is apparently transported faster to the\ncell surface and in a greater amount in the group of HRs. This group\nalso shed more of TF-containing microparticles. The cells of LRs\nhad a lower ability to be stimulated by LPS and retained more TF\nantigen, probably because of a slower exocytosis rate. Furthermore,\nLR monocytes express less TF antigen on their surface and shed\nfewer TF-containing microparticles. The findings of our present\nstudy indicate that elevated levels of the blood coagulation cascade\ninitiator on the surface of circulating monocytes and microparticles\nin HRs might make them more susceptible to hypercoagulation.\nAcknowledgments\nThis work was supported by program project grant HL46703 from\nthe National Institutes of Health (K.G.M.). We thank Dr Terje\nJohansen for sharing his knowledge and necessary reagents, Aud\nØvervatn for invaluable technical expertise, Drs Peter McCourt and\nPaul Johe for their help in preparation of this manuscript, and Serhiy\nPankiv for fruitful scientific discussions.\nReferences\n1. Conkling PR, Greenberg CS, Weinberg JB. Tumor necrosis factor\ninduces tissue factor-like activity in human leukemia cell line U937 and\nperipheral blood monocytes. Blood. 1988;72:128 –133.\n2. Edwards RL, Rickles FR, Bobrove AM. Mononuclear cell tissue factor:\ncell of origin and requirements for activation. Blood. 1979;54:359 –370.\n3. Østerud B, Bjorklid E. The production and availability of tissue throm-\nboplastin in cellular populations of whole blood exposed to various\nconcentrations of endotoxin. An assay for detection of endotoxin. Scand\nJ Haematol. 1982;29:175–184.\n12. Evangelista V, Piccardoni P, White JG, de Gaetano G, Cerletti C.\nCathepsin G-dependent platelet stimulation by activated poly-\nmorphonuclear leukocytes and its inhibition by antiproteinases: role of\nP-selectin-mediated cell-cell adhesion. Blood. 1993;81:2947–2957.\n13. Del Prete G, De Carli M, Lammel RM, D’Elios MM, Daniel KC, Giusti\nB, Abbate R, Romagnani S. Th1 and Th2 T-helper cells exert opposite\nregulatory effects on procoagulant activity and tissue factor production by\nhuman monocytes. Blood. 1995;86:250 –257.\n25. Niemetz J, Herbert V. The role of protein synthesis on the generation of tissue\nfactor activity by leukocytes. Proc Soc Exp Biol Med. 1972;139:1276–1279.\n26. Drake TA, Ruf W, Morrissey JH, Edgington TS. Functional tissue factor\nis entirely cell surface expressed on lipopolysaccharide-stimulated human\nblood monocytes and a constitutively tissue factor-producing neoplastic\ncell line. J Cell Biol. 1989;109:389 –395.",
         "",
         "Materials and Methods\nBlood Sampling and Experimental Design\nBlood samples from 16 healthy volunteers (HRs n8; LRs n8)\nwere obtained under protocols approved by the regional committee\nfor medical research ethics. In time-course ex vivo experiments,\nLPS-stimulated and -nonstimulated blood aliquots were incubated\nfor different times with agitation at 37°C. For TF activity assay and\nWestern blotting MNCs were isolated using Lymphoprep (Axis-\nShield) according to manufacturer instructions. For TNF- quanti-\nfication, platelet-poor plasma was obtained from blood aliquots by\ncentrifugation for 10 minutes. For a detailed description, please see\nthe online supplement, available at http://atvb.ahajournals.org.\nQuantification of Monocyte TF Activity\nTF was measured in the in frozen/thawed preparations of MNCs\nusing a 2-stage clotting assay based on the ability of TF to accelerate\nthe activation of factor X by factor VIIa as described previously.19\nQuantification of TNF-\nThe concentration of TNF- in platelet-poor plasma was determined by\nusing PeliKine compact ELISA kit according to manufacturer instructions.\nAntibody Quality Control and Small\nInterfering RNA\nTo ascertain the suitability of the monoclonal anti-human TF antibody used\nin the present study, we used TF gene silencing by RNA interference. For\na detailed description, please see the online supplement.\nFlow Cytometry\nPopulation of monocytes expressing TF (CD14TF cells) was deter-\nmined by staining of whole blood aliquots treated as described in\nexperimental design, using phycoerythrin (PE)-conjugated anti-human\nCD14 monoclonal antibodies (BD Biosciences PharMingen) and\nAlexa488-conjugated mouse anti-human TF monoclonal antibodies.\nMouse monoclonal antibodies recognizing extracellular domain of\nhuman TF were prepared by the monoclonal antibody facility, Depart-\nment of Biochemistry, University of Vermont, Burlington. For a\ndetailed description of the procedure, please see the online supplement.\nFluorescence Confocal Microscopy\nAfter Lymphoprep isolation, MNCs were plated for 30 minutes onto\nflat-bottomed 8-well chambers and fixed. Part of the samples was\npermeabilized with 70% methanol. TF and monocytes were visual-\nized by immunostaining with Alexa488-conjugated mouse anti-\nhuman TF monoclonal antibodies and PE-conjugated mouse anti-\nhuman CD14 monoclonal antibodies. Nuclei of the cells were stained\nusing DRAQ5 dye (Biostatus, Ltd). A fluorescent laser scanning\nmicroscope (LSM510 META; Carl Zeiss AG) with C-Apochromat\n40/1.2 W lens was used for image acquisition.\nWestern Blotting\nTF protein levels in lysates of resting and LPS-stimulated 106 MNCs\nwere detected using 10% SDS-PAGE, and immunoblotting of nitrocel-\nlulose membranes with mouse anti-human TF monoclonal antibodies\nand horseradish peroxidase– conjugated secondary antibodies. We used\ndensitometrical reading of 43-kDa immunopositive band for statistical\ncomparisons. For details, please the online supplement.\nRT-PCR and Real-Time Polymerase\nChain Reaction\nTotal RNA was extracted from MNC pellets with an RNeasy kit\n(Qiagen). Thereafter, single-stranded cDNA was synthesized using\nTaqMan Gold RT-PCR Kit, and real-time PCR analyses of TF\nmRNA expression was performed in duplicates with assay-on-\ndemand real-time PCR kit using ABI PRISM 7700 Sequence\nDetection System (Applied Biosystems). For data normalization, we\nanalyzed the expression of 18S rRNA by using an assay-on-demand\nkit. For details, please see the online supplement.\nCell Count and Viability\nCell counts were performed on a Sysmex K1000 (TOA Medical\nElectronics Co Ltd) in a whole blood system and after isolation of\nMNCs. Cell viability was assessed by using a trypan blue dye exclusion\nassay according to manufacturer instructions (Invitrogen Corp).\nStatistics\nStatistical comparisons were performed using SigmaPlot 8.0 (SPPS,\nLtd.) and MS Excel. The presented data shown are meanSEM.\nComparisons between mean values were performed using the Stu-\ndent paired t test. P value 0.05 was considered significant.",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced\nmonocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and\nshedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs).\nMethods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of\nTF-positive monocytes in HRs (32.03.5%) versus LRs (11.21.2%; P0.05), along with a stronger mean fluorescence intensity\nof TF signal in HRs (7.10.5 arbitrary units [AU]) compared with LRs (5.40.4 AU; P0.05). The LPS-treated blood of the HR\ngroup contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular\naccumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs\nversus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of\nsurface TF, whereas MNCs from LRs only had a minor increase in surface TF levels.\nponents of the plasma coagulation cascade is important\nfor maintenance of normal rheological properties of blood.\nFailure to manipulate TF levels available for the initiation of\nblood clotting leads to thrombotic or bleeding disorders in\nhumans. Circulating monocytes are presumably the major cell\ntype that respond to variable stimuli by developing coagulant\nactivity1 through the expression of TF.2\nOriginally, intersubject variability in developing of mono-\ncyte TF activity was described by Østerud et al.3 By compar-\ning lipopolysaccharide (LPS)-induced monocyte TF activity\nand tumor necrosis factor- (TNF-) production in a whole\nblood system, an up to a 50-fold difference between individ-\nuals was observed.4 This finding was defined as the “high–\nlow responder phenomenon.”4 Also noteworthy, the individ-\nual usually remains a high responder (HR) or low responder\n(LR) for several years.5,6 Later, high intersubject variability in\ncytokine production by LPS-stimulated monocytes was dem-\nonstrated.7 It was also shown that patients with high levels of\nTNF- production were more susceptible to heart transplant\nrejection.8 Monocytes isolated from septic shock patient\nsurvivors revealed higher TNF- production than monocytes\nfrom nonsurvivors.9\nMany studies have been undertaken to describe the signifi-\ncance of this phenomenon, but so far, no general explanation has\nbeen found. Diverse plasma factors and direct cell interactions\nplay an important role in the development of monocyte TF\nactivity.10 –15 High expression of monocyte TF activity is asso-\nciated with higher risk of acute coronary syndrome.16\nPlatelets have been suggested to be responsible for\ninducing monocyte TF activity.17 Platelet-rich plasma\ninduced significantly higher TF activity in LPS-stimulated\nmonocytes than platelet-poor plasma.18 Moreover, when\nblood cells without platelets from HRs were mixed with\nplatelet-rich plasma of an LR, LPS-induced TF activity\nwas reduced up to 76% compared with an autologous\nsystem.18 It was shown that granulocytes enhance LPS-\ninduced monocyte TF activity in a platelet-dependent\nreaction involving P-selectin, platelet factor 4, platelet-\nactivating factor, hydroxyl-eicosatetraenoic acid, and\nplatelet-derived growth factor.18 –21\nHere we report several observations concerning the rela-\ntionships between intracellular- and membrane-located TF\nantigen in resting and LPS-stimulated monocytes in groups of\nHRs and LRs, using fluorescence-activated cell sorter\n(FACS) analysis, fluorescence confocal microscopy, in-cell\nOriginal received February 24, 2005; final version accepted April 14, 2005.\nFrom the Department of Biochemistry (E.M.E., M.A.S., G.B., J.O.O., B.O.), Institute of Medical Biology, University of Tromsø, Norway; the\nDepartment of Immunology and Transfusion Medicine (F.X.E.G.), University Hospital of Northern Norway, Tromsø; and the Department of Biochemistry\n(B.P.-S., K.G.M.), University of Vermont, Burlington.\nE.M.E. and M.A.S. contributed equally to this work.\nCorrespondence to Elena M. Egorina, MD, Department of Biochemistry, Institute of Medical Biology, University of Tromsø, N-9037, Tromsø,\nArterioscler Thromb Vasc Biol.\nWestern assay, and immunoblotting. In response to LPS\nstimulation, we clearly found different patterns of intracellu-\nlar accumulation and externalization of TF onto the surface of\nMNCs isolated from whole blood of HRs and LRs.\nMaterials and Methods\nBlood Sampling and Experimental Design\nBlood samples from 16 healthy volunteers (HRs n8; LRs n8)\nwere obtained under protocols approved by the regional committee\nfor medical research ethics. In time-course ex vivo experiments,\nLPS-stimulated and -nonstimulated blood aliquots were incubated\nfor different times with agitation at 37°C. For TF activity assay and\nWestern blotting MNCs were isolated using Lymphoprep (Axis-\nShield) according to manufacturer instructions. For TNF- quanti-\nfication, platelet-poor plasma was obtained from blood aliquots by\ncentrifugation for 10 minutes. For a detailed description, please see\nthe online supplement, available at http://atvb.ahajournals.org.\nQuantification of Monocyte TF Activity\nTF was measured in the in frozen/thawed preparations of MNCs\nusing a 2-stage clotting assay based on the ability of TF to accelerate\nthe activation of factor X by factor VIIa as described previously.19\nQuantification of TNF-\nThe concentration of TNF- in platelet-poor plasma was determined by\nusing PeliKine compact ELISA kit according to manufacturer instructions.\nAntibody Quality Control and Small\nInterfering RNA\nTo ascertain the suitability of the monoclonal anti-human TF antibody used\nin the present study, we used TF gene silencing by RNA interference. For\na detailed description, please see the online supplement.\nFlow Cytometry\nPopulation of monocytes expressing TF (CD14TF cells) was deter-\nmined by staining of whole blood aliquots treated as described in\nexperimental design, using phycoerythrin (PE)-conjugated anti-human\nCD14 monoclonal antibodies (BD Biosciences PharMingen) and\nAlexa488-conjugated mouse anti-human TF monoclonal antibodies.\nMouse monoclonal antibodies recognizing extracellular domain of\nhuman TF were prepared by the monoclonal antibody facility, Depart-\nment of Biochemistry, University of Vermont, Burlington. For a\ndetailed description of the procedure, please see the online supplement.\nFluorescence Confocal Microscopy\nAfter Lymphoprep isolation, MNCs were plated for 30 minutes onto\nflat-bottomed 8-well chambers and fixed. Part of the samples was\npermeabilized with 70% methanol. TF and monocytes were visual-\nized by immunostaining with Alexa488-conjugated mouse anti-\nhuman TF monoclonal antibodies and PE-conjugated mouse anti-\nhuman CD14 monoclonal antibodies. Nuclei of the cells were stained\nusing DRAQ5 dye (Biostatus, Ltd). A fluorescent laser scanning\nmicroscope (LSM510 META; Carl Zeiss AG) with C-Apochromat\n40/1.2 W lens was used for image acquisition.\nWestern Blotting\nTF protein levels in lysates of resting and LPS-stimulated 106 MNCs\nwere detected using 10% SDS-PAGE, and immunoblotting of nitrocel-\nlulose membranes with mouse anti-human TF monoclonal antibodies\nand horseradish peroxidase– conjugated secondary antibodies. We used\ndensitometrical reading of 43-kDa immunopositive band for statistical\ncomparisons. For details, please the online supplement.\nRT-PCR and Real-Time Polymerase\nChain Reaction\nTotal RNA was extracted from MNC pellets with an RNeasy kit\n(Qiagen). Thereafter, single-stranded cDNA was synthesized using\nTaqMan Gold RT-PCR Kit, and real-time PCR analyses of TF\nmRNA expression was performed in duplicates with assay-on-\ndemand real-time PCR kit using ABI PRISM 7700 Sequence\nDetection System (Applied Biosystems). For data normalization, we\nanalyzed the expression of 18S rRNA by using an assay-on-demand\nkit. For details, please see the online supplement.\nCell Count and Viability\nCell counts were performed on a Sysmex K1000 (TOA Medical\nElectronics Co Ltd) in a whole blood system and after isolation of\nMNCs. Cell viability was assessed by using a trypan blue dye exclusion\nassay according to manufacturer instructions (Invitrogen Corp).\nStatistics\nStatistical comparisons were performed using SigmaPlot 8.0 (SPPS,\nLtd.) and MS Excel. The presented data shown are meanSEM.\nComparisons between mean values were performed using the Stu-\ndent paired t test. P value 0.05 was considered significant.\nResults\nReactivity of MNCs in HRs and LRs\nThe LPS-induced responses of MNCs from individuals enrolled\nin our study have been repeatedly assessed to determine their\nindividual TF activity and concentration levels of TNF-. These\nhealthy individuals were allocated to HR or LR groups accord-\ning to criteria developed in our laboratory.16 Briefly, healthy\nvolunteers were defined as HRs if their MNC-TF activity\nexceeded 20 mU/106 cells and their plasma TNF- concentra-\ntion levels were 3 ng/mL taken from whole blood stimulated\nwith LPS for 2 hours. We assigned individuals with these\nparameters lower than indicated above to the LR group.\nMeasurements of TF procoagulant activity in lysates of\nMNCs isolated from ex vivo LPS-stimulated whole blood\nrevealed 2 distinct responses. In HRs, LPS stimulation\ninduced significantly higher values of MNC-TF activity than\nin LRs after 2 hours (50.114.3 versus 20.94.2 mU/106\ncells; P0.05) and 4 hours (56.016.3 mU/106 versus\n21.24.9 mU/106 cells; P0.05; Figure 1). We found no\nincrease in TF activity in MNC lysates from time-matched\nnonstimulated samples (Figure 1).\nEgorina et al\nTF in High and Low Responders\nThe differences of MNC-TF activity and the production of\nTNF- in our experimental model could not be caused by\nvariations in degree of cell death because the number of dead\ncells measured by trypan blue exclusion assay did not differ\nbetween groups and was within 3% of the total cell count.\nThe variable degrees of responsiveness to LPS treatment,\ncharacterized by significantly elevated TF activity in MNC\nlysates in HRs,\nthe comparisons of\nintracellular and surface TF antigen levels under resting and\nstimulated conditions in these individuals.\nled us to carry out\nBecause we planned to use immunostaining-based methods for\nnative TF antigen detection (such as fluorescence-activated cell\nsorting, fluorescence confocal microscopy and in-cell Western\nassay), we applied RNA interference against TF expression in\nplated HeLa cells. Similarly, the same antibodies detected a 3-fold\nreduction of the in-cell Western assay signal intensity in the wells\nthat contained HeLa cells transfected with siRNA against TF when\ncompared with cells transfected with control siRNA. The signal\nfrom the in-cell Western assay against GAPDH showed no differ-\nences in the same wells (Figure IB).\nTo characterize levels of TF antigen on the surface and\ninterior of resting and LPS-stimulated monocytes from HRs\nand LRs, we chose FACS analyses of whole blood, fluores-\ncence confocal microscopy, and in-cell Western assay of\npermeabilized and nonpermeabilized MNCs.\nArterioscler Thromb Vasc Biol.\nWe found no baseline difference in the surface expression\nof TF on nonstimulated CD14 cells from LRs (1.40.1%)\nand HRs (1.50.2%; Figure 3A and 3B). MFI did not differ\nbetween groups and was 4.50.3 AU and 4.20.1 AU in\nLRs and HRs, respectively (Figure 3C).\nBy use of fluorescence confocal microscopy of isolated\nMNCs in resting and LPS-stimulated conditions, we visual-\nized the distribution of TF antigen between cytoplasmic and\nmembrane compartments. Only a few CD14 cells expressed\nTF in permeabilized and nonpermeabilized conditions (Fig-\nure 4A and 4B). Furthermore, we noticed 2 phenotypes when\nimaging nonpermeabilized resting CD14 cells: 1 did not\nexpress TF on the membrane surface, whereas another did\n(Figure 4C and 4D). The pattern of TF surface distribution could\nbe characterized by the presence of 5 to 7 TF-enriched dots or\n“patches.” When permeabilized, resting CD14 cells have TF-\ncontaining speckles in the proximity of the plasma membrane,\nready to be exocytosed to the cell exterior (Figure 4E).\nAfter 2 hours of LPS stimulation, permeabilized mono-\ncytes had their membranes enriched with TF antigen. The\ncytoplasm was depleted from TF-containing granules except\nfor a strong TF-positive staining unilaterally from the nucleus\n(Figure 4F). This could be newly synthesized TF protein.\nTo verify TF antigen distribution data from flow cytometry\nand fluorescence confocal microscopy, we used a quantitative\nimmunofluorescence staining in-cell Western assay.\nsignificantly higher LPS-induced accumulation of intracellu-\nlar TF antigen in LRs compared with HRs (Figure 5B). At the\nsame time, LPS treatment caused a 4-fold increase of surface\nTF signal intensity in HRs. However, no such increase was\nfound in the surface TF signal of LRs (Figure 5B).\nData from the immunoblotting of MNC lysates against TF\nantigen confirmed that after 2 hours of LPS stimulation,\nMNCs of LRs indeed accumulated TF antigen, whereas in\nHRs,\nthis accumulation was minor (Figure 5C and 5D).\nMNCs from time-matched samples of LRs experienced vir-\ntually no accumulation of the TF antigen in contrast to the\nsignificant accumulation of TF in MNCs of HRs (Figure 5D).\nComparison of real-time PCR data on de novo TF synthesis in\nLPS-stimulated conditions revealed similar mean cycle threshold\nvalues in MNCs of HRs and LRs (Table I, available online at\nhttp://atvb.ahajournals.org). Therefore, LPS stimulation is not only\nnecessary for the induction of TF synthesis, but also for its transport\nvia exocytotic pathway and shedding from the cell membrane.\nDiscussion\nIn the present ex vivo study, using a whole blood system, we\nreport several novel observations characterizing the distribu-\ntion of TF antigen between the interior and surface of MNCs\nfrom HRs and LRs. MNCs from representatives of these\ngroups exhibit distinct patterns of intracellular TF accumula-\ntion, externalization onto cell membrane, and shedding of\nTF-containing microparticles in LPS-treated conditions.\nEgorina et al\nTF in High and Low Responders\nCD14 cells expose TF on the cell surface after LPS-\nstimulation when compared with LRs.\nData of FACS analyses were in accordance with our\nfindings from fluorescence confocal microscopy, in which we\ndemonstrated redistribution of TF antigen on stimulation with\nLPS onto the cell surface. In stimulated conditions, TF was\ntransported and localized mostly on the membrane because\nthere was a clear depletion of TF staining from the cytoplasm.\nIntense dots present unilaterally from the nucleus could\nrepresent de novo synthesized TF.\ntime-matched nonstimulated cells. Together, it is possible that\nmonocytes from HRs could synthesize equal or even higher\namounts of TF antigen than those from LRs.\nArterioscler Thromb Vasc Biol.\nImmunostaining against TF and fluorescent confocal mi-\ncroscopy of permeabilized and nonpermeabilized resting\nmonocytes support our data from flow cytometry because\nonly few monocytes expressed TF antigen. The localization\npattern of TF on the monocyte surface in the present study is\nconfined to dotted structures or “patches.” Intracellular local-\nized TF in CD14 cells was restricted to round vesicle-like\nstructures in the proximity of the plasma membrane. Mulder\net al demonstrated the presence of an intracellular TF pool\nassociated with caveolae and multivesicular bodies.24\nOur data obtained from FACS and immunoblotting are\nconsistent with the previous accepted opinion that resting human\nmonocytes contain nearly undetectable levels of TF antigen.25–30\nHowever, we provide new information about the relationship\nbetween the intracellular and surface pool of TF in MNCs by\nusing in-cell Western assay. This enzyme-independent method\nof antigen detection via antibodies conjugated with infrared dyes\ngives highly reproducible data characterizing antigen levels.\nComparisons of the TF signal from resting MNCs between\npermeabilized and nonpermeabilized conditions revealed that in\nresting conditions, MNCs contain equally distributed surface\nand intracellular TF antigen in HRs and LRs.\nIn conclusion, MNCs of HRs and LRs contain similar, nearly\nundetectable intracellular levels of TF antigen at baseline condi-\ntions. On LPS stimulation, TF is apparently transported faster to the\ncell surface and in a greater amount in the group of HRs. This group\nalso shed more of TF-containing microparticles. The cells of LRs\nhad a lower ability to be stimulated by LPS and retained more TF\nantigen, probably because of a slower exocytosis rate. Furthermore,\nLR monocytes express less TF antigen on their surface and shed\nfewer TF-containing microparticles. The findings of our present\nstudy indicate that elevated levels of the blood coagulation cascade\ninitiator on the surface of circulating monocytes and microparticles\nin HRs might make them more susceptible to hypercoagulation.\nAcknowledgments\nThis work was supported by program project grant HL46703 from\nthe National Institutes of Health (K.G.M.). We thank Dr Terje\nJohansen for sharing his knowledge and necessary reagents, Aud\nØvervatn for invaluable technical expertise, Drs Peter McCourt and\nPaul Johe for their help in preparation of this manuscript, and Serhiy\nPankiv for fruitful scientific discussions.\nReferences\n1. Conkling PR, Greenberg CS, Weinberg JB. Tumor necrosis factor\ninduces tissue factor-like activity in human leukemia cell line U937 and\nperipheral blood monocytes. Blood. 1988;72:128 –133.\n2. Edwards RL, Rickles FR, Bobrove AM. Mononuclear cell tissue factor:\ncell of origin and requirements for activation. Blood. 1979;54:359 –370.\n3. Østerud B, Bjorklid E. The production and availability of tissue throm-\nboplastin in cellular populations of whole blood exposed to various\nconcentrations of endotoxin. An assay for detection of endotoxin. Scand\nJ Haematol. 1982;29:175–184.\n12. Evangelista V, Piccardoni P, White JG, de Gaetano G, Cerletti C.\nCathepsin G-dependent platelet stimulation by activated poly-\nmorphonuclear leukocytes and its inhibition by antiproteinases: role of\nP-selectin-mediated cell-cell adhesion. Blood. 1993;81:2947–2957.\n13. Del Prete G, De Carli M, Lammel RM, D’Elios MM, Daniel KC, Giusti\nB, Abbate R, Romagnani S. Th1 and Th2 T-helper cells exert opposite\nregulatory effects on procoagulant activity and tissue factor production by\nhuman monocytes. Blood. 1995;86:250 –257.\n25. Niemetz J, Herbert V. The role of protein synthesis on the generation of tissue\nfactor activity by leukocytes. Proc Soc Exp Biol Med. 1972;139:1276–1279.\n26. Drake TA, Ruf W, Morrissey JH, Edgington TS. Functional tissue factor\nis entirely cell surface expressed on lipopolysaccharide-stimulated human\nblood monocytes and a constitutively tissue factor-producing neoplastic\ncell line. J Cell Biol. 1989;109:389 –395.",
         "E. Egorina,M. Sovershaev,G. Bjørkøy,F. Gruber,J. Olsen,B. Parhami-Seren,K. Mann,B. Østerud",
         null
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin\nclinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-\nsusceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of\nmethicillin-resistant S. aureus strains were D-test positive.",
         "Clindamycin-susceptible, erythromycin-resistant Staphylo-\ncoccus aureus (clindamycin-erythromycin discordant) may de-\nvelop clindamycin resistance (7, 8). The erm gene product is a\nribosome methylase whose expression is normally minimal.\nErythromycin induces the production of this methylase, which\nis why these strains are erythromycin resistant, but mutations\nin the promoter region of erm allow production of methylase\nwithout an inducer (18, 19). These mutants are stably erythro-\nmycin and clindamycin resistant. Since erythromycin resistance\ncan occur with other mechanisms (e.g., efﬂux pumps and en-\nzymatic modiﬁcation) (15), the D-test identiﬁes inducible re-\nsistance that might presage mutational clindamycin constitu-\ntive resistance. The D-test is performed by placing clindamycin\nand erythromycin disks at an edge-to-edge distance of 15 to 20\nmm and looking for ﬂattening of the clindamycin zone nearest\nthe erythromycin disk (2). A positive D-test suggests the pres-\nence of an erm gene that could result in constitutive clindamy-\ncin resistance and clinical failure.\nThere are few published clinical failures of clindamycin with\nemergence of resistance (1, 3, 9, 12, 16). However, there are\nalso reports of successful use of clindamycin in treating pa-\ntients with D-test-positive isolates (3, 9). We report a clinical\nfailure with documented emergence of resistance. In order to\navoid poor clinical outcomes but retain the usefulness of clin-\ndamycin, it would be helpful to know the prevalence of induc-\nible resistance in clindamycin-erythromycin discordant bacte-\nria. This prevalence varies by geographic location, patient age,\nbacterial species, and bacterial susceptibility proﬁle (4, 5, 10,\n11, 13, 14, 16). For example, a pediatric population in Houston,\nTex., with methicillin-resistant S. aureus (MRSA) had a D-test\npositivity rate of 2.2% (9) compared to a similar population of\nchildren with MRSA in Chicago, Ill., whose D-test positivity\nrate was 94% (3). We determined the prevalence of D-test\npositivity in clindamycin-erythromycin discordant S. aureus iso-\nlates in our institution in early 2004. We tested isolates from\ninfected body sites where clindamycin might be considered for\ntherapy.\nThe patient was a 44-year-old man with fever, chills, and\nincreased low back pain 10 days after a lumbar discectomy. At\nadmission he had a temperature of 102.4°F and complained of\n3 days of erythema, bloody drainage, and pain along his inci-\nsion. He had a wound exploration, and the tissue Gram stain\nshowed white blood cells and gram-positive cocci. Fluid and\nblood cultures grew clindamycin-erythromycin discordant me-\nthicillin-susceptible S. aureus (MSSA), which was treated with\n2 weeks of oxacillin followed by 2 weeks of oral clindamycin.\nTen days after completing clindamycin therapy, he had severe\nback pain, chills, a temperature of 101.6°F, and purulent drain-\nage from his wound. A magnetic resonance image showed new\nevidence of lumbar osteomyelitis, discitis, and enhancement of\nparavertebral musculature with a small ﬂuid collection. On\nvancomycin, he had surgical debridement. Bone, blood, and\nwound cultures all grew MSSA with the same antibiogram as\nthat of his prior isolate except for new clindamycin resistance.\nHe successfully completed therapy with intravenous (oxacillin)\nand oral (combinations of minocycline, trimethoprim-sulfame-\nthoxazole, and rifampin) antibiotics for a total of 5 months.\nThe D-test on the original ﬂuid isolate was positive (Fig. 1);\nhowever, this result was not available at the time of his original\ntherapy. The original S. aureus wound isolate and the blood\nand bone isolates collected after completion of primary ther-\napy were indistinguishable by ribotyping with EcoRI restriction\nenzyme (Christiana Care Health Services, Wilmington, Del.).\nWe collected all 168 S. aureus isolates from wounds, ab-\nscesses, bone, cerebrospinal ﬂuid, pleural ﬂuid, ascitic ﬂuid,\nand joint ﬂuid from 1 January to 28 February 2004 at Temple\nUniversity Hospital (Philadelphia, Pa.). The strains (adult, pe-\ndiatric, inpatient, and outpatient) were stored in glycerol at\n74°C until batch testing in April 2004. Antibiotic susceptibil-\nities were determined by the Vitek-1 test (bioMerieux, Hazel-\nwood, Mo.); erythromycin-intermediate strains were considered\nresistant. The D-test was performed on clindamycin-erythromycin\ndiscordant strains by preparing an 0.5 McFarland suspension of\nbacteria in tryptic soy broth (2) inoculated on Mueller-Hinton\nplates incubated at 35°C for 18 to 24 h with erythromycin (15 g)\nand clindamycin (2 g) disks placed 17 mm apart (edge to edge).\nControls with known D-test positivity and negativity were also\ntested. Results were photographed and visually inspected by two\ninvestigators.\nNinety-one (54.2%) of 168 isolates were clindamycin-eryth-\nromycin discordant. There was 100% concordance of the D-\ntest results between the investigators. The majority (72.2%) of\nthe discordant strains were MRSA; however, only 8 (12.3%) of\n65 strains were D-test positive (Fig. 2). Twenty-ﬁve (27.5%) of\nthe 91 discordant strains were MSSA, but 17 (68%) of the 25\nstrains were D-test positive. Overall, 25 (27.5%) of the 91\ndiscordant strains were D-test positive.\nThe mechanisms of clindamycin resistance have been stud-\nied for years, and the D-test was developed to identify poten-\ntial clindamycin resistance so that possible ineffective therapy\nis not started when conventional tests show clindamycin MICs\nwithin the susceptible range (0.5 g/ml). Molecular markers\nfor the erm gene are available, but they are costly and incon-\nvenient for everyday use (8, 15, 17). The D-test is easy to\nperform and interpret, reproducible, and inexpensive but still\nnot universally used. There is a good correlation of standard\nD-tests with a modiﬁcation applied to purity plates done at the\ntime of specimen preparation for the Vitek-2 test (6).\nClindamycin is frequently used to treat skin and bone infec-\ntions because of its tolerability, cost, oral form, and good tissue\npenetration. With this high prevalence of D-test positivity, why\nare clindamycin clinical failures seldom reported? It may take\ntime for a mutant strain to develop, and the immune system\nmay have already controlled the infection. New agents active\nagainst gram-positive bacteria have been developed, so clinda-\nmycin may now be used less often. Finally, although a D-test-\npositive isolate may mutate on therapy, the mutation rate in\nclinical infections is unknown and may be rare.\nWe studied the prevalence of D-test positivity in isolates that\nmight be treated with clindamycin. We documented a clinda-\nmycin failure when resistance developed in a D-test-positive\nMSSA isolate. S. aureus clindamycin susceptibility reporting\nshould be suppressed when erythromycin resistance is found\n(unless a D-test is negative). If D-testing is delayed until re-\nsistance testing is complete, the results may not be available for\nmaximal clinical utility. Alternatively, the clinician may have to\nrequest it speciﬁcally.\nIt may be risky to use clindamycin when erythromycin testing\nshows a resistant or intermediate phenotype. The risk might\ncurrently be lower with MRSA, but this disparity may not\ncontinue or not be true elsewhere. Routine D-testing might\nallow clinicians to retain conﬁdence in clindamycin when eryth-\nromycin resistance is present.\n13. Sanchez, M. L., K. K. Flint, and R. N. Jones. 1993. Occurrence of macrolide-\nlincosamide-streptogramin resistance among staphylococcal clinical isolates\nat a university medical center. Diagn. Microbiol. Infect. Dis. 16:205–213.\n14. Sattler, C. A., E. O. Mason, Jr., and S. L. Kaplan. 2002. Prospective com-\nparison of risk factors and demographic and clinical characteristics of com-\nmunity-acquired, methicillin-resistant versus methicillin-susceptible Staphy-\nlococcus aureus infection in children. Pediatr. Infect. Dis. J. 21:910–916.\n15. Schmitz, F.-J., J. Petridou, A. C. Fluit, U. Hadding, G. Peters, and C. von\nEiff. 2000. Distribution of macrolide-resistant genes in Staphylococcus aureus\nblood-culture isolates from ﬁfteen German university hospitals. Eur. J. Clin.\nMicrobiol. Infect. Dis. 19:385–387.",
         "",
         "",
         "",
         "The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin\nclinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-\nsusceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of\nmethicillin-resistant S. aureus strains were D-test positive.\nClindamycin-susceptible, erythromycin-resistant Staphylo-\ncoccus aureus (clindamycin-erythromycin discordant) may de-\nvelop clindamycin resistance (7, 8). The erm gene product is a\nribosome methylase whose expression is normally minimal.\nErythromycin induces the production of this methylase, which\nis why these strains are erythromycin resistant, but mutations\nin the promoter region of erm allow production of methylase\nwithout an inducer (18, 19). These mutants are stably erythro-\nmycin and clindamycin resistant. Since erythromycin resistance\ncan occur with other mechanisms (e.g., efﬂux pumps and en-\nzymatic modiﬁcation) (15), the D-test identiﬁes inducible re-\nsistance that might presage mutational clindamycin constitu-\ntive resistance. The D-test is performed by placing clindamycin\nand erythromycin disks at an edge-to-edge distance of 15 to 20\nmm and looking for ﬂattening of the clindamycin zone nearest\nthe erythromycin disk (2). A positive D-test suggests the pres-\nence of an erm gene that could result in constitutive clindamy-\ncin resistance and clinical failure.\nThere are few published clinical failures of clindamycin with\nemergence of resistance (1, 3, 9, 12, 16). However, there are\nalso reports of successful use of clindamycin in treating pa-\ntients with D-test-positive isolates (3, 9). We report a clinical\nfailure with documented emergence of resistance. In order to\navoid poor clinical outcomes but retain the usefulness of clin-\ndamycin, it would be helpful to know the prevalence of induc-\nible resistance in clindamycin-erythromycin discordant bacte-\nria. This prevalence varies by geographic location, patient age,\nbacterial species, and bacterial susceptibility proﬁle (4, 5, 10,\n11, 13, 14, 16). For example, a pediatric population in Houston,\nTex., with methicillin-resistant S. aureus (MRSA) had a D-test\npositivity rate of 2.2% (9) compared to a similar population of\nchildren with MRSA in Chicago, Ill., whose D-test positivity\nrate was 94% (3). We determined the prevalence of D-test\npositivity in clindamycin-erythromycin discordant S. aureus iso-\nlates in our institution in early 2004. We tested isolates from\ninfected body sites where clindamycin might be considered for\ntherapy.\nThe patient was a 44-year-old man with fever, chills, and\nincreased low back pain 10 days after a lumbar discectomy. At\nadmission he had a temperature of 102.4°F and complained of\n3 days of erythema, bloody drainage, and pain along his inci-\nsion. He had a wound exploration, and the tissue Gram stain\nshowed white blood cells and gram-positive cocci. Fluid and\nblood cultures grew clindamycin-erythromycin discordant me-\nthicillin-susceptible S. aureus (MSSA), which was treated with\n2 weeks of oxacillin followed by 2 weeks of oral clindamycin.\nTen days after completing clindamycin therapy, he had severe\nback pain, chills, a temperature of 101.6°F, and purulent drain-\nage from his wound. A magnetic resonance image showed new\nevidence of lumbar osteomyelitis, discitis, and enhancement of\nparavertebral musculature with a small ﬂuid collection. On\nvancomycin, he had surgical debridement. Bone, blood, and\nwound cultures all grew MSSA with the same antibiogram as\nthat of his prior isolate except for new clindamycin resistance.\nHe successfully completed therapy with intravenous (oxacillin)\nand oral (combinations of minocycline, trimethoprim-sulfame-\nthoxazole, and rifampin) antibiotics for a total of 5 months.\nThe D-test on the original ﬂuid isolate was positive (Fig. 1);\nhowever, this result was not available at the time of his original\ntherapy. The original S. aureus wound isolate and the blood\nand bone isolates collected after completion of primary ther-\napy were indistinguishable by ribotyping with EcoRI restriction\nenzyme (Christiana Care Health Services, Wilmington, Del.).\nWe collected all 168 S. aureus isolates from wounds, ab-\nscesses, bone, cerebrospinal ﬂuid, pleural ﬂuid, ascitic ﬂuid,\nand joint ﬂuid from 1 January to 28 February 2004 at Temple\nUniversity Hospital (Philadelphia, Pa.). The strains (adult, pe-\ndiatric, inpatient, and outpatient) were stored in glycerol at\n74°C until batch testing in April 2004. Antibiotic susceptibil-\nities were determined by the Vitek-1 test (bioMerieux, Hazel-\nwood, Mo.); erythromycin-intermediate strains were considered\nresistant. The D-test was performed on clindamycin-erythromycin\ndiscordant strains by preparing an 0.5 McFarland suspension of\nbacteria in tryptic soy broth (2) inoculated on Mueller-Hinton\nplates incubated at 35°C for 18 to 24 h with erythromycin (15 g)\nand clindamycin (2 g) disks placed 17 mm apart (edge to edge).\nControls with known D-test positivity and negativity were also\ntested. Results were photographed and visually inspected by two\ninvestigators.\nNinety-one (54.2%) of 168 isolates were clindamycin-eryth-\nromycin discordant. There was 100% concordance of the D-\ntest results between the investigators. The majority (72.2%) of\nthe discordant strains were MRSA; however, only 8 (12.3%) of\n65 strains were D-test positive (Fig. 2). Twenty-ﬁve (27.5%) of\nthe 91 discordant strains were MSSA, but 17 (68%) of the 25\nstrains were D-test positive. Overall, 25 (27.5%) of the 91\ndiscordant strains were D-test positive.\nThe mechanisms of clindamycin resistance have been stud-\nied for years, and the D-test was developed to identify poten-\ntial clindamycin resistance so that possible ineffective therapy\nis not started when conventional tests show clindamycin MICs\nwithin the susceptible range (0.5 g/ml). Molecular markers\nfor the erm gene are available, but they are costly and incon-\nvenient for everyday use (8, 15, 17). The D-test is easy to\nperform and interpret, reproducible, and inexpensive but still\nnot universally used. There is a good correlation of standard\nD-tests with a modiﬁcation applied to purity plates done at the\ntime of specimen preparation for the Vitek-2 test (6).\nClindamycin is frequently used to treat skin and bone infec-\ntions because of its tolerability, cost, oral form, and good tissue\npenetration. With this high prevalence of D-test positivity, why\nare clindamycin clinical failures seldom reported? It may take\ntime for a mutant strain to develop, and the immune system\nmay have already controlled the infection. New agents active\nagainst gram-positive bacteria have been developed, so clinda-\nmycin may now be used less often. Finally, although a D-test-\npositive isolate may mutate on therapy, the mutation rate in\nclinical infections is unknown and may be rare.\nWe studied the prevalence of D-test positivity in isolates that\nmight be treated with clindamycin. We documented a clinda-\nmycin failure when resistance developed in a D-test-positive\nMSSA isolate. S. aureus clindamycin susceptibility reporting\nshould be suppressed when erythromycin resistance is found\n(unless a D-test is negative). If D-testing is delayed until re-\nsistance testing is complete, the results may not be available for\nmaximal clinical utility. Alternatively, the clinician may have to\nrequest it speciﬁcally.\nIt may be risky to use clindamycin when erythromycin testing\nshows a resistant or intermediate phenotype. The risk might\ncurrently be lower with MRSA, but this disparity may not\ncontinue or not be true elsewhere. Routine D-testing might\nallow clinicians to retain conﬁdence in clindamycin when eryth-\nromycin resistance is present.\n13. Sanchez, M. L., K. K. Flint, and R. N. Jones. 1993. Occurrence of macrolide-\nlincosamide-streptogramin resistance among staphylococcal clinical isolates\nat a university medical center. Diagn. Microbiol. Infect. Dis. 16:205–213.\n14. Sattler, C. A., E. O. Mason, Jr., and S. L. Kaplan. 2002. Prospective com-\nparison of risk factors and demographic and clinical characteristics of com-\nmunity-acquired, methicillin-resistant versus methicillin-susceptible Staphy-\nlococcus aureus infection in children. Pediatr. Infect. Dis. J. 21:910–916.\n15. Schmitz, F.-J., J. Petridou, A. C. Fluit, U. Hadding, G. Peters, and C. von\nEiff. 2000. Distribution of macrolide-resistant genes in Staphylococcus aureus\nblood-culture isolates from ﬁfteen German university hospitals. Eur. J. Clin.\nMicrobiol. Infect. Dis. 19:385–387.",
         "T. Levin,B. Suh,P. Axelrod,A. Truant,T. Fekete",
         null
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         null,
         null,
         null,
         null,
         null,
         null,
         "N. Gehrig,P. Dragotti",
         "captcha"
        ],
        [
         "9",
         "000a2234245d642c786a7355a15623444c33c1dc",
         "Cbl proteins have been implicated in ligand-induced TCR/CD3 down-modulation, but underlying mechanisms are unclear. We\nanalyzed the effect of mutation of a cbl-binding site on ZAP-70 (ZAP-Y292F) on dynamics, internalization, and degradation of the\nTCR/CD3 complex in response to distinct stimuli. Naive CD8 T cells expressing the P14 transgenic TCR from ZAP-Y292F mice\nwere selectively affected in TCR/CD3 down-modulation in response to antigenic stimulation, whereas neither anti-CD3 Ab-, and\nPMA-induced TCR down-modulation, nor constitutive receptor endocytosis/cycling were impaired. We further established that\nthe defect in TCR/CD3 down-modulation in response to Ag was paralleled by an impaired TCR/CD3 internalization and CD3\ndegradation. Analysis of T/APC conjugates revealed that delayed redistribution of TCR at the T/APC contact zone was paralleled\nby a delay in TCR internalization in the synaptic zone in ZAP-Y292F compared with ZAP-wild-type T cells. Cbl recruitment to\nthe synapse was also retarded in ZAP-Y292F T cells, although F-actin and LFA-1 redistribution was similar for both cell types.\nThis study identiﬁes a step involving ZAP-70/cbl interaction that is critical for rapid internalization of the TCR/CD3 complex at\nthe CD8 T cell/APC synapse. The Journal of Immunology, 2005, 175: 3140 –3149.",
         "face when associated with the CD3 complex composed of\nthe CD3/CD3, CD3/CD3, and CD3- dimers. Its\nlevel of surface expression is ﬁnely tuned by mechanisms control-\nling its assembly, membrane dynamics, and degradation, which\nvary along T cell development and as a consequence of T cell\nactivation. In particular, T cell stimulation by APC presenting the\ncognate peptide/MHC complex leads to a rapid decrease in the\nlevel of surface-expressed TCR. This phenomenon, known as\nTCR/CD3 down-modulation, occurs mainly by internalization of\nthe whole receptor complex through endosomal pathways (1, 2),\nalthough mechanisms invoking exocytosis of vesicles containing\nthe receptor have also been described (3).\nAg-induced TCR down-modulation is intimately linked to TCR\nsignaling and, in particular, to protein tyrosine kinase (PTK)4 ac-\ntivation, including Src family PTKs p56lck and p59fyn. These ki-\n4 Abbreviations used in this paper: PTK, protein tyrosine kinase; PKC, protein kinase\nC; IS, immune synapse; SMAC, supramolecular activation cluster; c-SMAC, central\nSMAC; p-SMAC, peripheral SMAC; CIN85, Cbl-interacting protein of 85 kDa; RTK,\nreceptor tyrosine kinase; CD2AP, CD2-associated protein; LN, lymph node; CTX,\ncholera toxin; MFI, mean ﬂuorescence intensity; PAF, paraformaldehyde; RT, room\ntemperature; BFA, brefeldin A; WT, wild type; AT,\nacid treated; NT,\nnot treated.\nnases phosphorylate tyrosines in the ITAM motifs of CD3 cyto-\nplasmic domains, allowing for binding of the ZAP-70 PTK via its\nSH2 domains. Subsequent phosphorylation of adaptor molecules\nand protein-protein interactions further transduce positive and neg-\native signals for functional activation of the T cells (4). In this\ncontext, TCR/CD3 down-modulation has been suggested to be one\nof the mechanisms that contribute to the arrest of signaling initi-\nated by Ag. Alternatively, it could be reasoned that the failure to\ndisengage peptide/MHC complexes by receptor clearance would\nprevent the serial engagement of new receptors that is required for\noptimal T cell stimulation by the limiting amount of peptide/MHC\ncomplexes available on the APCs (4, 5).\nApproaches at unraveling the complexities of TCR dynamics\nand signaling have included the analysis of T cells expressing\nTCR/CD3 complexes modiﬁed at potential internalization motifs,\nor in components of the receptor endocytosis/degradation pathway.\nIn particular, each CD3 cytoplasmic domain contains tyrosine-\nbased motifs in their ITAMs that are able to bind to the \n2 chain\nof the clathrin adaptor AP2 (6). Furthermore, the CD3 and CD3\nchains each present a di-leucine motif (7) capable of binding the \n\nsubunit of the AP2 complex (8). The CD3 chain di-leucine motif\nappears necessary for TCR/CD3 constitutive cycling (9), whereas\nthe proximal serine-126 on CD3is additionally required for phor-\nbol ester-induced TCR/CD3 internalization (10), a protein kinase\nC (PKC)-dependent event\ninvolves receptor sequestration\nwithout subsequent degradation (11). For mAb-induced TCR/CD3\ndown-regulation, a process dependent upon tyrosine kinase activ-\nity (12–14), but not PKC, absence of both CD3 and CD3 cyto-\nplasmic domains led to deﬁcient receptor down-modulation, sug-\ngesting that upon T cell activation, unveiling of the di-leucine and\ntyrosine-based motifs present in each of these chains may be in-\nvolved (1). However, Ag-induced TCR down-modulation was not\naffected by the combined truncations of CD3 and CD3 cyto-\nplasmic domains (15), suggesting that a distinct mechanism in-\nvolving ZAP-70 activity (14), was rate limiting in this process.\nthat\nThe Journal of Immunology\nThe particularity of Ag-induced TCR down-modulation is that it\noccurs upon engagement of the TCR by peptide/MHC complexes\nwithin the immune synapse (IS) formed at the T cell/APC contact\nzone. This specialized structure, associated with a redistribution of\nmembrane receptors, develops over a 5–30 min period. It subdi-\nvides the T cell/APC interface into supramolecular activation clus-\nters (SMACs) that are spatially organized into a central (c-SMAC)\nregion where TCR-MHC/peptide and CD28-CD80 interactions\ntake place, and a peripheral (p-SMAC) area enriched in cytoskel-\netal components and adhesion molecules such as LFA-1 (16, 17),\nand CD2 (18). However, the rapid induction of PTK activation\nupon Ag-induced T cell stimulation precedes the redistribution of\nthe TCR/CD3 complex in the c-SMAC (19 –21). The observation\nthat surface TCR disappeared from the c-SMAC may signify its\nlocalized internalization (21).\nTyrosine-292 in ZAP-70, when phosphorylated upon TCR ac-\ntivation, is a binding site for c-cbl and cbl-b via the cbl tyrosine\nkinase binding domain (22–24). Contrasting results were reported\nfor thymocytes of mice expressing an altered c-cbl mutated in its\ntyrosine kinase binding domain, because this led to constitutive\nactivation of rac without affecting ZAP-70 phosphorylation (25).\nHowever, the effect of this mutation could not be studied in pe-\nripheral T cells, where mostly cbl-b is expressed.\nAdaptor proteins of the Cbl-interacting protein of 85 kDa\n(CIN85) family have recently been shown to bind both cbl and\nendophilins, and thereby to regulate the endocytosis of activated\nreceptor tyrosine kinases (RTK) (26 –29). Cbl is also involved in\nRTK monoubiquitination allowing for receptor binding to the\nclathrin adaptor Eps15 (28, 30, 31). In T cells, the CIN85 family\nmember CD2-associated protein (CD2AP) has recently been im-\nplicated in the pathway of TCR/CD3 down-modulation (32, 33).\nFurthermore, we have recently described that T cells from mice\nexpressing ZAP-Y292F presented hyper-tyrosine-phosphorylation\nin response to TCR/CD3 engagement but were partially inhibited\nin Ag-induced TCR down-modulation (34).\nIn the present work, we analyzed the mechanisms by which\nTCR down-modulation is impaired in CD8 T cells expressing the\nZAP-Y292F mutation. Our results showed that the defect in TCR\ndown-modulation was speciﬁc for Ag-induced TCR internalization\nand was associated with a delayed recruitment of TCR and cbl at\nthe T/APC synapse zone.\nMaterials and Methods\nMice\nAg-presenting cells\nTap-2-deﬁcient RMA-S H-2b cells (35) were generally used as APCs, and\nsimilar results were obtained when LPS blasts from Tap-1°/° mice were\nused (results not shown). Before loading with peptide (LCMV gp33 (36) or\ncontrol peptide HY (37)), Tap-negative cells were incubated overnight at\n31°C in RPMI 1640 with 5% dialyzed FCS.\nT cell preparation\nAbs and reagents for immunoﬂuorescence\nvisualized with the cholera toxin -chain (CTX) coupled to FITC (Sigma-\nAldrich), or Alexa488 (Molecular Probes). F-actin was visualized using\nphalloidin Alexa488 (Molecular Probes). Rabbit anti-cbl Ring domain was\nprepared as described (26), and revealed with anti-rabbit Ig Zenon A546\n(Molecular Probes). The speciﬁcity of the Ab for cbl was further estab-\nlished in T cell lysates, because only the cbl band (at 120 kDa) was de-\ntected by immunoblot (results not shown).\nFluorescence analysis\nMeasurement of surface ﬂuorescence and FACS analysis were as described\nusing mean ﬂuorescence intensity (MFI) values (15). TCR down-modula-\ntion was calculated as (1  MFI experimental/MFI medium)  100. For\nintracellular CD3 staining, cells were washed (4 min; 4°C) in PBS before\n5-min ﬁxation with 2% paraformaldehyde (PAF) at room temperature\n(RT), washed again with PBS, and permeabilized for 10 min at RT with\n0.5% saponin in PBS, 2% FCS, and 0.02% NaN3 containing 1 mM van-\nadate (saponin buffer). Biotinylated mAb H146.968 was diluted in the same\nsaponin buffer and added for 30 min at 4°C. After permeabilization, cells\nwere washed and centrifuged at 2000 rpm for 5 min at 4°C. For calibration,\nwe set up optimal mAb dilutions to obtain a linear decrease in labeling in\nthe presence of increasing amounts of the CD3 peptide recognized by the\nmAb. CD3 degradation was calculated as follows: (1  MFI experimen-\ntal/MFI medium)  100.\nMeasurement of internalized TCR\nOur approach involved ﬁrst biotinylation of cell surface proteins with SS-\nbiotin to discriminate between surface and internalized TCR (45, 46).\nthis treatment prevented subsequent Ag-induced P14-TCR\nHowever,\ndown-modulation and T cell activation as measured by CD69 expression\n(results not shown). We then chose to prelabel T cells with anti-TCR mAb\nat 4°C, before Ag stimulation at 37°C, followed by acid treatment to re-\nlease mAb remaining at the cell surface (as described in Ref. 2). The\nanti-TCRV2 B20.1 mAb was unsuitable for this approach because it rap-\nidly dissociates from the TCR. Therefore anti-TCR (H57), which stably\nbinds the TCR, was used for this study, as follows. T cells were incubated\nat 4°C with anti-TCR(H57) mAb A488 (Molecular Probes kit) before Ag\nstimulation at 37°C. At different times, samples were divided, and one-half\nwas directly used to measure total ﬂuorescence, the other half was treated\nwith an acid solution (0.15 M acetic acid, 0.15 M NaCl) for 5 min on ice\nbefore two washes in medium and measurement of remaining ﬂuorescence\ncorresponding to internalized mAb (2). Prelabeling of T cells with anti-\nTCR (H57) did not per se activate the P14 T cells and did not prevent\nsubsequent Ag-induced TCR down-modulation and signaling for CD69\nexpression (see Fig. 2).\nConfocal analysis\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nResults\nAg-induced TCR down-modulation is defective in ZAP-Y292F\ncompared with ZAP-wild-type (WT) CD8 T cells\nOur previous results indicated that TCR down-modulation in re-\nsponse to antigenic stimulation was partially inhibited in CD8 T\ncells from mice expressing ZAP-Y292F molecules (34). We ﬁrst\nconﬁrmed this ﬁnding in a kinetic experiment shown in Fig. 1. As\nseen in Fig. 1A, exposure of ZAP-WT CD8 T cells to APC loaded\nwith the gp33 LCMV peptide at 107 M decreased the level of\nsurface TCR from 830 to 400 MFI units after 60 min, and to MFI\n113 after 270 min. A similar treatment of the ZAP-Y292F CD8 T\ncells decreased the level of surface TCR from 870 to 590 after 60\nmin, and to 330 after 270 min. The rate of TCR down-modulation\nin response to gp33 107 M/APC by the ZAP-Y292F CD8 T cells\nwas about half (20%/h) that observed for the ZAP-WT CD8 T cells\n(38%/h) (Fig. 1B). The large SDs of the 60-min time points cor-\nrespond to the variability of TCR down-modulation observed from\none experiment to the other at that early time, but the respective\nrelative values for mutant to WT T cells were very reproducible in\nthe different experiments (decreased rate of TCR down-modulation\n52  11%). The loading of the RMA-S used as APC was per-\nformed with gp33 concentrations between 107 and 109 M, and\nwe observed a linear dose effect on TCR down-modulation and\nactivation, with a differential rate of TCR down-modulation be-\ntween ZAP-WT and ZAP-Y292F T cells maintained at all con-\ncentrations tested (results not shown).\nWe next analyzed the effect of this mutation on other means of\ninducing TCR down-modulation, such as anti-CD3 mAb or PMA\nstimulations (15, 47). The rate of TCR down-modulation induced\nby anti-CD3 mAb was the same for the ZAP-WT and ZAP-Y292F\nT cells, i.e., 55%/h (Fig. 1B). Receptor down-modulation was\nalso similar for the two types of T cells at any dose of anti-CD3\nmAb used, whether soluble (Fig. 1C) or plastic-coated (results not\nshown). When cells were stimulated by PMA, we also observed a\nsimilar rate of TCR down-modulation for both strains (results not\nshown) (see Fig. 2B).\nAll experiments were controlled for T cell activation by mea-\nsuring the induction of CD69, and we conﬁrmed that there was no\ndifference between the two strains in that assay (34).\nIn conclusion, the ZAP-Y292F mutation caused a reduced rate\n(40 –50% decrease) of TCR down-modulation in response to an-\ntigenic stimulation compared with that observed in ZAP-WT T\ncells within a range of antigenic peptide from 107 to 109 M.\nFurthermore, this study showed that the ZAP-Y292F mutation se-\nlectively affects Ag-induced, and not anti-CD3 mAb- or PMA-\ninduced receptor down-modulation.\nUse of brefeldin A (BFA) revealed that the ZAP-Y292F mutation\ndoes not affect spontaneous TCR cycling and shows no recycling\nof anti-CD3 mAb- or Ag-induced TCR down-modulation in both\ntypes of CD8 T cells\nConstitutive TCR endocytosis and cycling can be demonstrated by\nthe effect on receptor surface expression of BFA, which inhibits\nexport from the endoplasmic reticulum/Golgi and endosome recy-\ncling (45). We have examined the effect of BFA on constitutive\nTCR down-modulation, as well as on Ag- or anti-CD3-induced\nTCR down-modulation of ZAP-WT and ZAP-Y292F CD8 T cells.\nResults shown in Table I are expressed as the ratio of TCR down-\na BFA was added or not to T cells for 1 h before and during the down-modulation\nexperiment, in medium (constitutive cycling), with APC (Ag-induced down-modu-\nlation), or with anti-CD3 mAb (CD3-induced down-modulation). Results are pre-\nsented for the 120-min time point, and are expressed as the ratio between TCR down-\nmodulation in the absence and presence of BFA.\nThe Journal of Immunology\nmodulation between untreated and BFA-treated T cells. As previ-\nously described for naive T cells (48), P14 CD8 T cells do not\nthrive when cultured unstimulated at 37°C. They showed a gradual\ndecrease of TCR expression (5%/h), which was slower than the\nrate previously described for naive CD4 T cells (45). The presence\nof BFA increased receptor down-modulation (15–20%/h), so that,\nafter 2 h, the ratio of TCR down-modulation between untreated\nand BFA-treated T cells was of 0.34 for ZAP-WT and 0.38 for\nZAP-Y292F T cells, respectively (Table I), indicating that both\ntypes of T cells have\nconstitutive\nTCR/CD3 cycling.\na\nAg- or anti-CD3-induced TCR down-modulation was the same,\nwhether BFA was present or not, for both ZAP-WT and ZAP-\nY292F T cells (Table I). As a control for the activity of the BFA\ntreatment, we observed that no CD69 expression was induced\nwhen T cells were stimulated with Ag (results not shown), as pre-\ndicted by the inhibitory effect of BFA on protein secretion (49).\nFurthermore, when cycloheximide was added to prevent de novo\nreceptor synthesis during the incubation with or without BFA, no\ndifference appeared for the two types of T cells either.\nTogether, these results suggest that Ag- and anti-CD3-induced\nTCR down-modulation are not affected by prevention of TCR re-\ncycling by BFA.\nAg-induced TCR internalization is affected in ZAP-Y292F\ncompared with ZAP-WT CD8 T cells\nThe level of surface TCR expression is the result of an equilibrium\nbetween de novo synthesis, recycling, internalization, and degra-\ndation of its constituents. To establish which step was affected in\nthe Ag-induced TCR/CD3 down-modulation observed for ZAP-\nY292F T cells, we next determined the rate of receptor internal-\nization in mutant and WT T cells. For this, surface receptor has to\nbe labeled before induction of internalization. As described in Ma-\nterials and Methods, we prelabeled the surface TCR with anti-\nTCR (H57) mAb (A488) and visualized TCR internalization ei-\nther after acid treatment by FACS analysis or directly by confocal\nmicroscopy (see Fig. 4B). One can note that anti-TCR (H57)\nmAb binding was stable at 37°C (Fig. 2A). Furthermore, this treat-\nment did not per se induce TCR down-modulation or T cell acti-\nvation (measured by CD69 surface expression), and it did not af-\nfect Ag-induced TCR down-modulation (Fig. 2).\nThe ratio between the MFI on acid-treated (AT; measuring in-\nternalized Ab) and untreated cells (not treated (NT); measuring\ntotal mAb at the cell surface and inside) from the same sample\ncorresponds to the fraction of internalized TCR. After acidic treat-\nment, the total ﬂuorescence dropped to 10 and 9 when, respec-\ntively, ZAP-WT and ZAP-Y292F CD8 T cells were maintained at\n0°C for 90 min, and to 22 and 16 when they were incubated for the\nsame time at 37°C. This corresponds for both mouse strains to\n30% internalized TCR in medium at 37°C, compared with\n14% at 0°C (Fig. 2B). When cells were incubated with 107 M\ngp33 loaded APCs, acid-resistant ﬂuorescence increased to MFI of\n47 for the ZAP-WT and to only 33 for the ZAP-Y292F cells (Fig.\n2A), corresponding, respectively, to 70 and 45% internalized TCR\nafter 60-min stimulation (Fig. 2B). By contrast, when cells were\nstimulated with PMA, we observed a similar rate of TCR inter-\nnalization (70% internalized TCR after 60-min stimulation; Fig.\n2B) for both strains.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nAg (46), compared with spontaneous receptor cycling, which con-\ntrast with other data (45).\nAnalysis of the fate of CD3 during TCR internalization\nUsing anti-CD3 mAb H146.968, we developed a sensitive intra-\ncellular FACS analysis to measure CD3levels (see Materials and\nMethods). We previously compared this intracellular ﬂuorescence\nmeasurement with immunoblots on cell lysates using the same\nmAb to CD3 and obtained comparable results (M. Buferne and\nC. Boyer, unpublished results), indicating that a decrease in anti-\nCD3 MFI corresponds to the degradation of intracellular CD3.\nAll cultures were performed at 37°C in a medium containing cy-\ncloheximide to prevent protein synthesis.\nIn Fig. 3A, we observe similar levels of CD3 for P14 ZAP-WT\nor ZAP-Y292F CD8 T cells when cultured in medium (MFI, 433\nand 384, respectively). In the ZAP-WT T cells, a pronounced de-\ncrease in CD3 was observed in response to APC/gp33 (105 M)\n(MFI, 217), whereas its level remained high in control cultures\nwith APC/HY (MFI 362). This decrease corresponds to 50% CD3\ndegradation in 60% of the cells (Fig. 3, A and B). For P14 ZAP-\nY292F CD8 T cells, by contrast, we observed a moderate decrease\nin CD3 in response to APC/gp33 (105 M) (MFI, 260) compared\nwith its level in control cultures with APC/HY (MFI, 332). This\ncorresponds to 38% CD3 degradation in only 25% of the cells\n(Fig. 3, A and B). Note that CD3 degradation and TCR down-\nmodulation both increase in proportion to Ag dose (Fig. 3, com-\npare B and C), in agreement with recent work (46).\nContrasting results were obtained for T cells responding to anti-\nCD3 mAb or PMA. Stimulation with anti-CD3 led to very efﬁcient\nTCR down-modulation (80% after 4 h) and extensive CD3deg-\nradation (60 –70%) in CD8 T cells from both strains, whereas the\nPMA-induced TCR down-modulation (50%), led to only minor\nCD3 degradation (15%). The latter pattern was also similar for\nZAP-WT and ZAP-Y292F CD8 T cells (Fig. 3, B and C). This\nobservation is in agreement with published work (11), and with the\nnotion that PMA induces TCR sequestration, without driving the\nreceptor to the degradation pathway.\nThus, ligand-induced receptor degradation is dependent upon its\ninternalization, but all internalization, such as that induced by\nPMA, does not necessarily lead to degradation.\nTogether, these results suggest that ZAP-Y292F CD8 T cells are\nselectively affected at a step of Ag-induced receptor internalization\nand that, once internalized, the CD3 component is similarly de-\ngraded. However, these results do not exclude the possibility that\nthe ZAP-Y292F mutation may additionally affect the degradation\nof the CD3 complex.\nTo further address the stage at which the fate of the TCR/CD3\ncomplex is affected in the ZAP-Y292F CD8 T, we examined the\ndynamics of the receptor with respect to T cell/APC interaction\nand formation of the IS.\nAg-induced surface TCR redistribution at the T/APC contact is\ndefective and delayed for ZAP-Y292F CD8 T cells\nThe Journal of Immunology\nthe conjugated ZAP-Y292F T cells (85% of the analyzed con-\njugates; Table II), which is also shown in three-dimensional de-\nconvolution (Fig. 4A, third row, right). By contrast, F-actin was\nconcentrated at the T/APC contact zone of ZAP-Y292F T cells as\nfor ZAP-WT T cells. The poor CD3 redistribution in ZAP-Y292F\nT cell conjugates was partly compensated after 45 min, where we\nobserved that 40% of the T cells had a patched CD3 distribution\n(Table II). For both types of CD8 T cells, the TCR/CD3 was not\nredistributed if the irrelevant peptide HY was presented by the\nAPC (Fig. 4A, ﬁrst row). One can notice that, in these conjugates,\nTCR/CD3 and F-actin are homogenously distributed around the\ncell, and that F-actin is poorly labeled in unactivated T cells.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\ngliosides and raft labeling (50) and analyzed by confocal micros-\ncopy. As a control, T cells were also incubated alone, and we\nobserved a homogenous cell surface labeling for both TCR (red)\nand CTX (green), yielding a yellow-to-orange colocalization la-\nbeling (Fig. 4B, ﬁrst row, no APC, 45 min). For the P14 ZAP-WT\nCD8 T cells, after 15-min T/gp33 APC conjugate formation, we\nobserved a redistribution of the anti-TCR, indicating a localization\nof the TCR either at the plasma membrane in the T/APC interac-\ntion zone, or inside the cell where it is no longer colocalized with\nthe green CTX membrane staining (Fig. 4B, left panel, second\nrow). Furthermore, some (but not all) of the internalized TCR ap-\npeared in organelles budding from the T/APC synapse zone, as\nalready suggested by others (21). After 45 min of conjugate for-\nmation, most of the TCR was in organelles inside the cell (Fig. 4B,\nthird row) and did not colocalize with the surface-labeled CTX.\nThis pattern was observed for 90% of T/APC conjugates analyzed,\nboth after 15 and 45 min of T/APC contact. One can notice that\nCTX is rather poorly relocalized in the T/APC contact zone, a\npattern previously described for human CD8 T cells (51), in con-\ntrast to the CTX concentration in the T/APC zone described for\nmouse CD4 T cell lines (52). Additionally, the RMA-S cells used\nas APC were very poorly labeled with CTX, consequently CTX\ncould be used to mark CD8 T cell membranes.\nTCR redistribution was different for P14 ZAP-Y292F CD8 T\ncells, as shown in Fig. 4B (right panels). After 15-min T/gp33\nAPC conjugate formation, we observed very few TCR patches and\nno TCR redistribution to the T/APC contact zone. Both TCR and\nsurface CTX were still homogeneously distributed at the T cell\nsurface. However, after 45-min conjugation, we observed some\nTCR patches inside the T cell (see also Table II), as well as re-\nmaining TCR cell surface expression, suggesting that for the ZAP-\nY292F T cells, TCR redistribution and internalization was delayed\nand at a lower rate, in agreement with the previous results (Figs. 2\nand 4A). Together, these results indicate that TCR redistribution\nand internalization, in the T/APC zone are delayed and partially\ninhibited by the ZAP-Y292F mutation. By contrast, anti-CD3\nmAb-induced TCR patching and internalization was not affected\nby the ZAP-Y292F mutation when analyzed by confocal micros-\ncopy (results not shown).\nRecruitment of cbl at the T/APC contact zone is delayed and\ndecreased in ZAP-Y292F compared with ZAP-WT T cells\nBecause the TCR clustering at the T/APC conjugate contact zone\nwas delayed in ZAP-Y292F T cells, we were interested in analyz-\ning the localization of cbl, to establish whether the loss of the\ncbl-binding site on ZAP-Y292F would affect its redistribution. In-\ndeed, cbl was previously observed to be recruited to the T/APC\ncontact zone (53), and to be colocalized with the signaling phos-\nphotyrosine zone (54). Therefore, we compared the localization of\ncbl and F-actin in T/APC conjugates formed for different periods\nof time (15– 45 min). This study was performed by confocal mi-\ncroscopy on ﬁxed and permeabilized T/APC conjugates further\nlabeled for cbl (rabbit anti-Ring cbl/anti-rabbit-A546, red), and for\nF-actin (phalloidin-A488, green), as described in Materials and\nMethods (Fig. 5).\nrow, ZAP-WT and ZAP-Y292F). However, most of the ZAP-WT\nT/APC conjugates (65%; Table II) presented an additional local-\nization in the middle of the contact zone (Fig. 5A, second row,\nZAP-WT; see bold arrow). Such a localization was observed in\nonly 30% of the ZAP-Y292F T/APC conjugates (Table II). A his-\ntogram analysis of the ﬂuorescence distribution (Fig. 5C, Cbl 1)\nshowed that the cbl labeling was concentrated at the T/APC con-\ntact zone, bulging out of the F-actin zone within the T cell, before\nthe blue (Cy5) boundary of the APC membrane. The frequency of\nthis pattern increased for both strains at 45 min, reaching 80% for\nthe ZAP-WT and 50% for the ZAP-Y292F T/APC conjugates (Ta-\nble II). Despite variability between experiments, the difference be-\ntween the two strains was always observed (three independent\nexperiments).\nIn Fig. 5B, we analyzed the distribution of LFA-1 in T/APC\nconjugates. A high redistribution of LFA-1 at the T/gp33 APC\ncontact zone was observed for both ZAP-WT and ZAP-Y292F\nCD8 T cells (Fig. 5B, second row), which was not observed in the\nT/HY APC conjugates ( ﬁrst row). On the histograms representing\nthe ﬂuorescence distribution within T/APC conjugates, one can\nobserve that LFA-1 labeling is concentrated at the T/APC contact\nzone on the T cell side, proximal to the blue boundary of the APC\nmembrane. The LFA-1 redistribution remained rather homogenous\nat the T/APC contact zone even at 45 min, as described for other\nmarkers (PKC and Talin) in naive CD8 T cell/APC conjugates\n(55). This contrasts with the description of a p-SMAC distribution\nafter 30-min conjugate formation in other T cell lines including\nnaive CD4 T cells (21, 56).\nThese results showed that the redistribution of LFA-1 and\nF-actin at the T/APC conjugate contact zone was not affected by\nthe ZAP-Y292F mutation. In contrast, the recruitment of cbl to\nthe central region, which was readily observed in the ZAP-WT\nT/APC conjugates at an early time, was defective in the ZAP-\nY292F T cells. Whether this recruitment corresponds to an en-\ndosomal localization of cbl associated with the activated TCR\ncomplex needs further investigation. It is consistent with the\nfact that cbl and associated CMS/CIN85/CD2AP can be in-\nvolved in endosome organization via rab4 (57), and with the\ndescribed effect of the ZAP-Y292F mutation as preventing the\nZAP-cbl association (58). However, this effect was partially com-\npensated with time, which suggests that other cbl associations may\nsecondarily develop in the IS. These may involve vav linked to\nZAP Y315 (59), CD28 via PI3K (60), the src PTK (61), CD2AP\nbinding to CD2 (62), or connection of CD2AP/CIN85 proteins to\nthe actin cytoskeleton (63, 64).\nDiscussion\nIn the present work, we report that naive CD8 P14 T lymphocytes\nexpressing the mutated ZAP-Y292F were affected in Ag-induced\nTCR down-modulation, and presented a defect and delay in TCR\nand cbl recruitment at the T/APC conjugate interaction zone com-\npared with ZAP-WT T cells. However, LFA-1 and F-actin were\nclearly redistributed in the ZAP-Y292F T/APC contact zone as in\nZAP-WT T/APC conjugates.\nThe Journal of Immunology\nthat TCR internalization is affected by the ZAP-Y292F mutation,\nusing an assay independent of a mAb ligand. However, it has not\nbeen ruled out that the ZAP-Y292F mutation may additionally\naffect the degradation of some of the TCR/CD3 components. Con-\nstitutive endocytosis, which depends mainly on a CD3 di-leucine\nmotif (9), or on several motifs from all CD3 components (45), was\nnot altered by the ZAP-Y292F mutation. Anti-CD3- and PMA-\ninduced TCR internalization, which depend on CD3 or CD3\ncytoplasmic domains (1, 65), were not affected either. Therefore,\nthe observed defects, although partial, would appear to implicate\nthe ZAP-Y292/cbl interaction (58) at a step that selectively affects\nAPC-induced TCR internalization.\nBinding of the cbl adaptor to phosphorylated ZAP-292Y (22)\nhas a negative effect on the activity of ZAP-70, which was mainly\nattributed to the E3-Ubiquitin ligase activity of cbl and its Ring\ndomain (66). Consistent with this, cbl was found to be involved in\nubiquitination of ZAP-70 (67) and of CD3 (68). Cbl has also\nrecently been implicated at various steps of RTK surface receptor\ninternalization and degradation. First, cbl may participate in the\nformation of clathrin-coated vesicles by adapting endophilin via a\nprotein of the CIN85 family; additionally, the ubiquitin-ligase ac-\ntivity of cbl allows the ubiquitinated receptor to be recognized by\nubiquitin interacting motif domain bearing proteins like Eps15,\nwhich is a clathrin adaptor (26 –29, 31).\nIn T cells, the double genetic deletion of c-cbl/cbl-b was asso-\nciated with a partial deﬁciency in anti-CD3-induced TCR internal-\nization, whereas the effect on Ag stimulation was not reported in\nthat work (23). CD2AP, a member of the CIN85 family (62) ca-\npable of binding cbl (69), was recently reported to play a major\nrole in T cell signaling and TCR down-modulation (32). Indeed, T\ncells from CD2AP knockout mice failed to down-modulate their\nTCR in response to Ag stimulation, and showed defects in the ﬁne\nstructure of the c-SMAC and p-SMAC (32, 33). However, in con-\ntrast to the c-cbl/cbl-b double knockout, CD2AP knockout T cells\nwere not affected in mAb-induced TCR down-modulation, at least\nat early steps (32).\nThe fact that the defect in ZAP/cbl association selectively affects\nAPC-induced, and not constitutive or anti-CD3-induced TCR in-\nternalization, together with the observation of the delay in TCR\nand cbl recruitment to the T-APC interaction zone in ZAP-Y292F\nT cells strongly suggests that the defect in TCR internalization\nmay be the consequence of its delayed recruitment to the IS. The\nalternative, that the delay in TCR recruitment and c-SMAC for-\nmation would be the consequence of a defect in TCR internaliza-\ntion, appears difﬁcult to entertain. Indeed, in contrast to receptor\nengagement by mAb, stimulation of TCRs by peptide/MHC com-\nplexes on the APC is most efﬁcient when receptors are concen-\ntrated within the IS. Consistent with this, internalized TCR is\nmainly detected at the site of the IS (Ref. 21; Fig. 4B), whereas\nmAb-induced internalization is observed all around the plasma\nmembrane (results not shown). The molecular basis for the defect\nin c-SMAC formation in the absence of CD2AP is not fully elu-\ncidated (32). Similarly, it is not clear in the ZAP-Y292F T cells 1)\nwhether the delay in recruiting the TCR/CD3 at the IS is due to the\nabsence of CD2AP/cbl/ZAP binding, and 2) which molecular in-\nteraction of a putative CD2AP/cbl/ZAP complex is required to\nform the IS. Although such CD2AP/cbl/epidermal growth factor\nreceptor complexes have been described to connect to the actin\ncytoskeleton via the actin-related protein 2/3 complex (64), it is not\nclear whether such connections function in the TCR pathway.\nThey have been suggested for CD2 signaling (63).\nThrough 1) the ubiquitination pathway, cbl may dampen strong\nPTK signals, independently from the IS, and through 2) its asso-\nciation with CD2AP, it may contribute to the concentration of\nTCR/CD3/ZAP complexes within a c-SMAC, a step required for\nsustained signaling in the absence of apoptosis (32). The latter\ncontribution of the ZAP-70/cbl-b interaction to promote T cell ac-\ntivation may be balanced by the increased rate of TCR/CD3 com-\nplex internalization within the c-SMAC.\nThe unique properties of TCR/CD3 redistribution within the\nT/APC interface may explain why distinct motifs within the re-\nceptor complex appear to control its constitutive or mAb-induced\ninternalization (45) that have implicated di-leucine motifs (9), and\nAPC-induced internalization for which phospho-CD3-associated\nZAP-Y292 appears to be a rate-limiting motif. However, the fact\nthat the defect in the ZAP-Y292F T was partial, suggests that other\nmechanisms may lead to APC-induced TCR/CD3 internalization.\nThese may involve other cbl/CD2AP binding sites within the re-\nceptor complex, cbl being a scaffold protein that may bind other\nproximal ligands (25). It is also possible that another ubiquitin\nligase is able to bind to the activated TCR complex, akin to the\nNedd4 ubiquitinin ligase that was also localized in the raft fraction\nof IgE in activated RBL cells (71).\nConcerning the relative control of TCR/CD3 complex internal-\nization and degradation, we found that for Ag-, as for anti-CD3-\ninduced receptor down-modulation, CD3 degradation was pro-\nportional to receptor internalization and down-modulation (Fig. 3).\nThis result is in agreement with other work showing that during\nconstitutive, and mAb-induced TCR internalization, all TCR/CD3\ncomponents are degraded with a rate that is proportional to the rate\nof TCR down-modulation (46) through a mechanism that is de-\npendent on lysosomal activity (72). However, an exception exists\nfor PMA-induced TCR internalization that leads to poor CD3\ndegradation (Fig. 3).\nTogether with data from the literature, the present results are\ncompatible with a role for cbl in the Ag-induced TCR/CD3 inter-\nnalization/degradation following schemes also proposed for the\nepidermal growth factor receptor (29, 30, 33). Clearly, questions\nremain about the mechanisms by which the TCR/CD3 complex is\ndriven into the endocytic pathway at the IS, and in particular\nwhether, as for the BCR, internalization via clathrin-coated vesi-\ncles is dependent on integral raft components (73, 74).\nAcknowledgments\nWe thank the members of the Centre d’Immunologie de Marseille-Luminy\n(CIML) Imaging Service, Marc Barad, Nicole Brun, and Mathieu Fallet for\ntheir help, Corinne Be´ziers-Guigue for artwork, Ste´phane Me´resse and\nNicolas Lapaque for reagents, and the CIML personnel involved in animal\ncare. We thank L. Leserman, N. Auphan, A. Guimezanes, H.-T. He, and\nG. Verdeil for critical reading of the manuscript.\nDisclosures\nThe authors have no ﬁnancial conﬂict of interest.\nReferences\n1. Luton, F., M. Buferne, V. Legendre, C. Boyer, and A.-M. Schmitt-Verhulst.\n1997. Role of CD3 and CD3 cytoplasmic domains in cytolytic T cell clones\nfunctions and TCR/CD3 down-modulation. J. Immunol. 158: 4162– 4170.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\n6. Ohno, H., J. Stewart, M.-C. Fournier, H. Bosshart, I. Rhee, S. Miyatake, T. Saito,\nA. Gallusser, T. Kirchausen, and J. S. Bonifacino. 1995. Interaction of tyrosine-\nbased sorting signals with clathrin-associated proteins. Science 269: 1872–1875.\n7. Letourneur, F., and R. D. Klausner. 1992. A novel di-leucine motif and a ty-\nrosine-based motif independently mediate lysosomal targeting and endocytosis of\nCD3 chains. Cell 69: 1143–1157.\n25. Thien, C. B., R. M. Scaife, J. M. Papadimitriou, M. A. Murphy, D. D. Bowtell,\nand W. Y. Langdon. 2003. A mouse with a loss-of-function mutation in the c-Cbl\nTKB domain shows perturbed thymocyte signaling without enhancing the activ-\nity of the ZAP-70 tyrosine kinase. J. Exp. Med. 197: 503–513.\n28. Polo, S., S. Sigismund, M. Faretta, M. Guidi, M. R. Capua, G. Bossi, H. Chen,\nP. De Camilli, and P. P. Di Fiore. 2002. A single motif responsible for ubiquitin\nrecognition and monoubiquitination in endocytic proteins. Nature 416: 451– 455.\n29. Oved, S., and Y. Yarden. 2002. Signal transduction: molecular ticket to enter\n30. Riezman, H. 2002. Cell biology: the ubiquitin connection. Nature 416: 381–383.\n31. de Melker, A. A., G. van der Horst, and J. Borst. 2004. Ubiquitin ligase activity\nof c-Cbl guides the epidermal growth factor receptor into clathrin-coated pits by\ntwo distinct modes of Eps15 recruitment. J. Biol. Chem. 279: 55465–55473.\n32. Lee, K. H., A. R. Dinner, C. Tu, G. Campi, S. Raychaudhuri, R. Varma,\nT. N. Sims, W. R. Burack, H. Wu, J. Wang, et al. 2003. The immunological\nsynapse balances T cell receptor signaling and degradation. Science 302:\n1218 –1222.\n54. Bunnell, S. C., D. I. Hong, J. R. Kardon, T. Yamazaki, C. J. McGlade, V. A. Barr,\nand L. E. Samelson. 2002. T cell receptor ligation induces the formation of\ndynamically regulated signaling assemblies. J. Cell Biol. 158: 1263–1275.\n55. O’Keefe, J. P., K. Blaine, M. L. Alegre, and T. F. Gajewski. 2004. Formation of\na central supramolecular activation cluster is not required for activation of naive\nCD8 T cells. Proc. Natl. Acad. Sci. USA 101: 9351–9356.\nThe Journal of Immunology\n64. Lynch, D. K., S. C. Winata, R. J. Lyons, W. E. Hughes, G. M. Lehrbach,\nV. Wasinger, G. Corthals, S. Cordwell, and R. J. Daly. 2003. A Cortactin-CD2-\nassociated protein (CD2AP) complex provides a novel link between epidermal\ngrowth factor receptor endocytosis and the actin cytoskeleton. J. Biol. Chem. 278:\n21805–21813.",
         "",
         "",
         "",
         "Cbl proteins have been implicated in ligand-induced TCR/CD3 down-modulation, but underlying mechanisms are unclear. We\nanalyzed the effect of mutation of a cbl-binding site on ZAP-70 (ZAP-Y292F) on dynamics, internalization, and degradation of the\nTCR/CD3 complex in response to distinct stimuli. Naive CD8 T cells expressing the P14 transgenic TCR from ZAP-Y292F mice\nwere selectively affected in TCR/CD3 down-modulation in response to antigenic stimulation, whereas neither anti-CD3 Ab-, and\nPMA-induced TCR down-modulation, nor constitutive receptor endocytosis/cycling were impaired. We further established that\nthe defect in TCR/CD3 down-modulation in response to Ag was paralleled by an impaired TCR/CD3 internalization and CD3\ndegradation. Analysis of T/APC conjugates revealed that delayed redistribution of TCR at the T/APC contact zone was paralleled\nby a delay in TCR internalization in the synaptic zone in ZAP-Y292F compared with ZAP-wild-type T cells. Cbl recruitment to\nthe synapse was also retarded in ZAP-Y292F T cells, although F-actin and LFA-1 redistribution was similar for both cell types.\nThis study identiﬁes a step involving ZAP-70/cbl interaction that is critical for rapid internalization of the TCR/CD3 complex at\nthe CD8 T cell/APC synapse. The Journal of Immunology, 2005, 175: 3140 –3149.\nface when associated with the CD3 complex composed of\nthe CD3/CD3, CD3/CD3, and CD3- dimers. Its\nlevel of surface expression is ﬁnely tuned by mechanisms control-\nling its assembly, membrane dynamics, and degradation, which\nvary along T cell development and as a consequence of T cell\nactivation. In particular, T cell stimulation by APC presenting the\ncognate peptide/MHC complex leads to a rapid decrease in the\nlevel of surface-expressed TCR. This phenomenon, known as\nTCR/CD3 down-modulation, occurs mainly by internalization of\nthe whole receptor complex through endosomal pathways (1, 2),\nalthough mechanisms invoking exocytosis of vesicles containing\nthe receptor have also been described (3).\nAg-induced TCR down-modulation is intimately linked to TCR\nsignaling and, in particular, to protein tyrosine kinase (PTK)4 ac-\ntivation, including Src family PTKs p56lck and p59fyn. These ki-\n4 Abbreviations used in this paper: PTK, protein tyrosine kinase; PKC, protein kinase\nC; IS, immune synapse; SMAC, supramolecular activation cluster; c-SMAC, central\nSMAC; p-SMAC, peripheral SMAC; CIN85, Cbl-interacting protein of 85 kDa; RTK,\nreceptor tyrosine kinase; CD2AP, CD2-associated protein; LN, lymph node; CTX,\ncholera toxin; MFI, mean ﬂuorescence intensity; PAF, paraformaldehyde; RT, room\ntemperature; BFA, brefeldin A; WT, wild type; AT,\nacid treated; NT,\nnot treated.\nnases phosphorylate tyrosines in the ITAM motifs of CD3 cyto-\nplasmic domains, allowing for binding of the ZAP-70 PTK via its\nSH2 domains. Subsequent phosphorylation of adaptor molecules\nand protein-protein interactions further transduce positive and neg-\native signals for functional activation of the T cells (4). In this\ncontext, TCR/CD3 down-modulation has been suggested to be one\nof the mechanisms that contribute to the arrest of signaling initi-\nated by Ag. Alternatively, it could be reasoned that the failure to\ndisengage peptide/MHC complexes by receptor clearance would\nprevent the serial engagement of new receptors that is required for\noptimal T cell stimulation by the limiting amount of peptide/MHC\ncomplexes available on the APCs (4, 5).\nApproaches at unraveling the complexities of TCR dynamics\nand signaling have included the analysis of T cells expressing\nTCR/CD3 complexes modiﬁed at potential internalization motifs,\nor in components of the receptor endocytosis/degradation pathway.\nIn particular, each CD3 cytoplasmic domain contains tyrosine-\nbased motifs in their ITAMs that are able to bind to the \n2 chain\nof the clathrin adaptor AP2 (6). Furthermore, the CD3 and CD3\nchains each present a di-leucine motif (7) capable of binding the \n\nsubunit of the AP2 complex (8). The CD3 chain di-leucine motif\nappears necessary for TCR/CD3 constitutive cycling (9), whereas\nthe proximal serine-126 on CD3is additionally required for phor-\nbol ester-induced TCR/CD3 internalization (10), a protein kinase\nC (PKC)-dependent event\ninvolves receptor sequestration\nwithout subsequent degradation (11). For mAb-induced TCR/CD3\ndown-regulation, a process dependent upon tyrosine kinase activ-\nity (12–14), but not PKC, absence of both CD3 and CD3 cyto-\nplasmic domains led to deﬁcient receptor down-modulation, sug-\ngesting that upon T cell activation, unveiling of the di-leucine and\ntyrosine-based motifs present in each of these chains may be in-\nvolved (1). However, Ag-induced TCR down-modulation was not\naffected by the combined truncations of CD3 and CD3 cyto-\nplasmic domains (15), suggesting that a distinct mechanism in-\nvolving ZAP-70 activity (14), was rate limiting in this process.\nthat\nThe Journal of Immunology\nThe particularity of Ag-induced TCR down-modulation is that it\noccurs upon engagement of the TCR by peptide/MHC complexes\nwithin the immune synapse (IS) formed at the T cell/APC contact\nzone. This specialized structure, associated with a redistribution of\nmembrane receptors, develops over a 5–30 min period. It subdi-\nvides the T cell/APC interface into supramolecular activation clus-\nters (SMACs) that are spatially organized into a central (c-SMAC)\nregion where TCR-MHC/peptide and CD28-CD80 interactions\ntake place, and a peripheral (p-SMAC) area enriched in cytoskel-\netal components and adhesion molecules such as LFA-1 (16, 17),\nand CD2 (18). However, the rapid induction of PTK activation\nupon Ag-induced T cell stimulation precedes the redistribution of\nthe TCR/CD3 complex in the c-SMAC (19 –21). The observation\nthat surface TCR disappeared from the c-SMAC may signify its\nlocalized internalization (21).\nTyrosine-292 in ZAP-70, when phosphorylated upon TCR ac-\ntivation, is a binding site for c-cbl and cbl-b via the cbl tyrosine\nkinase binding domain (22–24). Contrasting results were reported\nfor thymocytes of mice expressing an altered c-cbl mutated in its\ntyrosine kinase binding domain, because this led to constitutive\nactivation of rac without affecting ZAP-70 phosphorylation (25).\nHowever, the effect of this mutation could not be studied in pe-\nripheral T cells, where mostly cbl-b is expressed.\nAdaptor proteins of the Cbl-interacting protein of 85 kDa\n(CIN85) family have recently been shown to bind both cbl and\nendophilins, and thereby to regulate the endocytosis of activated\nreceptor tyrosine kinases (RTK) (26 –29). Cbl is also involved in\nRTK monoubiquitination allowing for receptor binding to the\nclathrin adaptor Eps15 (28, 30, 31). In T cells, the CIN85 family\nmember CD2-associated protein (CD2AP) has recently been im-\nplicated in the pathway of TCR/CD3 down-modulation (32, 33).\nFurthermore, we have recently described that T cells from mice\nexpressing ZAP-Y292F presented hyper-tyrosine-phosphorylation\nin response to TCR/CD3 engagement but were partially inhibited\nin Ag-induced TCR down-modulation (34).\nIn the present work, we analyzed the mechanisms by which\nTCR down-modulation is impaired in CD8 T cells expressing the\nZAP-Y292F mutation. Our results showed that the defect in TCR\ndown-modulation was speciﬁc for Ag-induced TCR internalization\nand was associated with a delayed recruitment of TCR and cbl at\nthe T/APC synapse zone.\nMaterials and Methods\nMice\nAg-presenting cells\nTap-2-deﬁcient RMA-S H-2b cells (35) were generally used as APCs, and\nsimilar results were obtained when LPS blasts from Tap-1°/° mice were\nused (results not shown). Before loading with peptide (LCMV gp33 (36) or\ncontrol peptide HY (37)), Tap-negative cells were incubated overnight at\n31°C in RPMI 1640 with 5% dialyzed FCS.\nT cell preparation\nAbs and reagents for immunoﬂuorescence\nvisualized with the cholera toxin -chain (CTX) coupled to FITC (Sigma-\nAldrich), or Alexa488 (Molecular Probes). F-actin was visualized using\nphalloidin Alexa488 (Molecular Probes). Rabbit anti-cbl Ring domain was\nprepared as described (26), and revealed with anti-rabbit Ig Zenon A546\n(Molecular Probes). The speciﬁcity of the Ab for cbl was further estab-\nlished in T cell lysates, because only the cbl band (at 120 kDa) was de-\ntected by immunoblot (results not shown).\nFluorescence analysis\nMeasurement of surface ﬂuorescence and FACS analysis were as described\nusing mean ﬂuorescence intensity (MFI) values (15). TCR down-modula-\ntion was calculated as (1  MFI experimental/MFI medium)  100. For\nintracellular CD3 staining, cells were washed (4 min; 4°C) in PBS before\n5-min ﬁxation with 2% paraformaldehyde (PAF) at room temperature\n(RT), washed again with PBS, and permeabilized for 10 min at RT with\n0.5% saponin in PBS, 2% FCS, and 0.02% NaN3 containing 1 mM van-\nadate (saponin buffer). Biotinylated mAb H146.968 was diluted in the same\nsaponin buffer and added for 30 min at 4°C. After permeabilization, cells\nwere washed and centrifuged at 2000 rpm for 5 min at 4°C. For calibration,\nwe set up optimal mAb dilutions to obtain a linear decrease in labeling in\nthe presence of increasing amounts of the CD3 peptide recognized by the\nmAb. CD3 degradation was calculated as follows: (1  MFI experimen-\ntal/MFI medium)  100.\nMeasurement of internalized TCR\nOur approach involved ﬁrst biotinylation of cell surface proteins with SS-\nbiotin to discriminate between surface and internalized TCR (45, 46).\nthis treatment prevented subsequent Ag-induced P14-TCR\nHowever,\ndown-modulation and T cell activation as measured by CD69 expression\n(results not shown). We then chose to prelabel T cells with anti-TCR mAb\nat 4°C, before Ag stimulation at 37°C, followed by acid treatment to re-\nlease mAb remaining at the cell surface (as described in Ref. 2). The\nanti-TCRV2 B20.1 mAb was unsuitable for this approach because it rap-\nidly dissociates from the TCR. Therefore anti-TCR (H57), which stably\nbinds the TCR, was used for this study, as follows. T cells were incubated\nat 4°C with anti-TCR(H57) mAb A488 (Molecular Probes kit) before Ag\nstimulation at 37°C. At different times, samples were divided, and one-half\nwas directly used to measure total ﬂuorescence, the other half was treated\nwith an acid solution (0.15 M acetic acid, 0.15 M NaCl) for 5 min on ice\nbefore two washes in medium and measurement of remaining ﬂuorescence\ncorresponding to internalized mAb (2). Prelabeling of T cells with anti-\nTCR (H57) did not per se activate the P14 T cells and did not prevent\nsubsequent Ag-induced TCR down-modulation and signaling for CD69\nexpression (see Fig. 2).\nConfocal analysis\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nResults\nAg-induced TCR down-modulation is defective in ZAP-Y292F\ncompared with ZAP-wild-type (WT) CD8 T cells\nOur previous results indicated that TCR down-modulation in re-\nsponse to antigenic stimulation was partially inhibited in CD8 T\ncells from mice expressing ZAP-Y292F molecules (34). We ﬁrst\nconﬁrmed this ﬁnding in a kinetic experiment shown in Fig. 1. As\nseen in Fig. 1A, exposure of ZAP-WT CD8 T cells to APC loaded\nwith the gp33 LCMV peptide at 107 M decreased the level of\nsurface TCR from 830 to 400 MFI units after 60 min, and to MFI\n113 after 270 min. A similar treatment of the ZAP-Y292F CD8 T\ncells decreased the level of surface TCR from 870 to 590 after 60\nmin, and to 330 after 270 min. The rate of TCR down-modulation\nin response to gp33 107 M/APC by the ZAP-Y292F CD8 T cells\nwas about half (20%/h) that observed for the ZAP-WT CD8 T cells\n(38%/h) (Fig. 1B). The large SDs of the 60-min time points cor-\nrespond to the variability of TCR down-modulation observed from\none experiment to the other at that early time, but the respective\nrelative values for mutant to WT T cells were very reproducible in\nthe different experiments (decreased rate of TCR down-modulation\n52  11%). The loading of the RMA-S used as APC was per-\nformed with gp33 concentrations between 107 and 109 M, and\nwe observed a linear dose effect on TCR down-modulation and\nactivation, with a differential rate of TCR down-modulation be-\ntween ZAP-WT and ZAP-Y292F T cells maintained at all con-\ncentrations tested (results not shown).\nWe next analyzed the effect of this mutation on other means of\ninducing TCR down-modulation, such as anti-CD3 mAb or PMA\nstimulations (15, 47). The rate of TCR down-modulation induced\nby anti-CD3 mAb was the same for the ZAP-WT and ZAP-Y292F\nT cells, i.e., 55%/h (Fig. 1B). Receptor down-modulation was\nalso similar for the two types of T cells at any dose of anti-CD3\nmAb used, whether soluble (Fig. 1C) or plastic-coated (results not\nshown). When cells were stimulated by PMA, we also observed a\nsimilar rate of TCR down-modulation for both strains (results not\nshown) (see Fig. 2B).\nAll experiments were controlled for T cell activation by mea-\nsuring the induction of CD69, and we conﬁrmed that there was no\ndifference between the two strains in that assay (34).\nIn conclusion, the ZAP-Y292F mutation caused a reduced rate\n(40 –50% decrease) of TCR down-modulation in response to an-\ntigenic stimulation compared with that observed in ZAP-WT T\ncells within a range of antigenic peptide from 107 to 109 M.\nFurthermore, this study showed that the ZAP-Y292F mutation se-\nlectively affects Ag-induced, and not anti-CD3 mAb- or PMA-\ninduced receptor down-modulation.\nUse of brefeldin A (BFA) revealed that the ZAP-Y292F mutation\ndoes not affect spontaneous TCR cycling and shows no recycling\nof anti-CD3 mAb- or Ag-induced TCR down-modulation in both\ntypes of CD8 T cells\nConstitutive TCR endocytosis and cycling can be demonstrated by\nthe effect on receptor surface expression of BFA, which inhibits\nexport from the endoplasmic reticulum/Golgi and endosome recy-\ncling (45). We have examined the effect of BFA on constitutive\nTCR down-modulation, as well as on Ag- or anti-CD3-induced\nTCR down-modulation of ZAP-WT and ZAP-Y292F CD8 T cells.\nResults shown in Table I are expressed as the ratio of TCR down-\na BFA was added or not to T cells for 1 h before and during the down-modulation\nexperiment, in medium (constitutive cycling), with APC (Ag-induced down-modu-\nlation), or with anti-CD3 mAb (CD3-induced down-modulation). Results are pre-\nsented for the 120-min time point, and are expressed as the ratio between TCR down-\nmodulation in the absence and presence of BFA.\nThe Journal of Immunology\nmodulation between untreated and BFA-treated T cells. As previ-\nously described for naive T cells (48), P14 CD8 T cells do not\nthrive when cultured unstimulated at 37°C. They showed a gradual\ndecrease of TCR expression (5%/h), which was slower than the\nrate previously described for naive CD4 T cells (45). The presence\nof BFA increased receptor down-modulation (15–20%/h), so that,\nafter 2 h, the ratio of TCR down-modulation between untreated\nand BFA-treated T cells was of 0.34 for ZAP-WT and 0.38 for\nZAP-Y292F T cells, respectively (Table I), indicating that both\ntypes of T cells have\nconstitutive\nTCR/CD3 cycling.\na\nAg- or anti-CD3-induced TCR down-modulation was the same,\nwhether BFA was present or not, for both ZAP-WT and ZAP-\nY292F T cells (Table I). As a control for the activity of the BFA\ntreatment, we observed that no CD69 expression was induced\nwhen T cells were stimulated with Ag (results not shown), as pre-\ndicted by the inhibitory effect of BFA on protein secretion (49).\nFurthermore, when cycloheximide was added to prevent de novo\nreceptor synthesis during the incubation with or without BFA, no\ndifference appeared for the two types of T cells either.\nTogether, these results suggest that Ag- and anti-CD3-induced\nTCR down-modulation are not affected by prevention of TCR re-\ncycling by BFA.\nAg-induced TCR internalization is affected in ZAP-Y292F\ncompared with ZAP-WT CD8 T cells\nThe level of surface TCR expression is the result of an equilibrium\nbetween de novo synthesis, recycling, internalization, and degra-\ndation of its constituents. To establish which step was affected in\nthe Ag-induced TCR/CD3 down-modulation observed for ZAP-\nY292F T cells, we next determined the rate of receptor internal-\nization in mutant and WT T cells. For this, surface receptor has to\nbe labeled before induction of internalization. As described in Ma-\nterials and Methods, we prelabeled the surface TCR with anti-\nTCR (H57) mAb (A488) and visualized TCR internalization ei-\nther after acid treatment by FACS analysis or directly by confocal\nmicroscopy (see Fig. 4B). One can note that anti-TCR (H57)\nmAb binding was stable at 37°C (Fig. 2A). Furthermore, this treat-\nment did not per se induce TCR down-modulation or T cell acti-\nvation (measured by CD69 surface expression), and it did not af-\nfect Ag-induced TCR down-modulation (Fig. 2).\nThe ratio between the MFI on acid-treated (AT; measuring in-\nternalized Ab) and untreated cells (not treated (NT); measuring\ntotal mAb at the cell surface and inside) from the same sample\ncorresponds to the fraction of internalized TCR. After acidic treat-\nment, the total ﬂuorescence dropped to 10 and 9 when, respec-\ntively, ZAP-WT and ZAP-Y292F CD8 T cells were maintained at\n0°C for 90 min, and to 22 and 16 when they were incubated for the\nsame time at 37°C. This corresponds for both mouse strains to\n30% internalized TCR in medium at 37°C, compared with\n14% at 0°C (Fig. 2B). When cells were incubated with 107 M\ngp33 loaded APCs, acid-resistant ﬂuorescence increased to MFI of\n47 for the ZAP-WT and to only 33 for the ZAP-Y292F cells (Fig.\n2A), corresponding, respectively, to 70 and 45% internalized TCR\nafter 60-min stimulation (Fig. 2B). By contrast, when cells were\nstimulated with PMA, we observed a similar rate of TCR inter-\nnalization (70% internalized TCR after 60-min stimulation; Fig.\n2B) for both strains.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nAg (46), compared with spontaneous receptor cycling, which con-\ntrast with other data (45).\nAnalysis of the fate of CD3 during TCR internalization\nUsing anti-CD3 mAb H146.968, we developed a sensitive intra-\ncellular FACS analysis to measure CD3levels (see Materials and\nMethods). We previously compared this intracellular ﬂuorescence\nmeasurement with immunoblots on cell lysates using the same\nmAb to CD3 and obtained comparable results (M. Buferne and\nC. Boyer, unpublished results), indicating that a decrease in anti-\nCD3 MFI corresponds to the degradation of intracellular CD3.\nAll cultures were performed at 37°C in a medium containing cy-\ncloheximide to prevent protein synthesis.\nIn Fig. 3A, we observe similar levels of CD3 for P14 ZAP-WT\nor ZAP-Y292F CD8 T cells when cultured in medium (MFI, 433\nand 384, respectively). In the ZAP-WT T cells, a pronounced de-\ncrease in CD3 was observed in response to APC/gp33 (105 M)\n(MFI, 217), whereas its level remained high in control cultures\nwith APC/HY (MFI 362). This decrease corresponds to 50% CD3\ndegradation in 60% of the cells (Fig. 3, A and B). For P14 ZAP-\nY292F CD8 T cells, by contrast, we observed a moderate decrease\nin CD3 in response to APC/gp33 (105 M) (MFI, 260) compared\nwith its level in control cultures with APC/HY (MFI, 332). This\ncorresponds to 38% CD3 degradation in only 25% of the cells\n(Fig. 3, A and B). Note that CD3 degradation and TCR down-\nmodulation both increase in proportion to Ag dose (Fig. 3, com-\npare B and C), in agreement with recent work (46).\nContrasting results were obtained for T cells responding to anti-\nCD3 mAb or PMA. Stimulation with anti-CD3 led to very efﬁcient\nTCR down-modulation (80% after 4 h) and extensive CD3deg-\nradation (60 –70%) in CD8 T cells from both strains, whereas the\nPMA-induced TCR down-modulation (50%), led to only minor\nCD3 degradation (15%). The latter pattern was also similar for\nZAP-WT and ZAP-Y292F CD8 T cells (Fig. 3, B and C). This\nobservation is in agreement with published work (11), and with the\nnotion that PMA induces TCR sequestration, without driving the\nreceptor to the degradation pathway.\nThus, ligand-induced receptor degradation is dependent upon its\ninternalization, but all internalization, such as that induced by\nPMA, does not necessarily lead to degradation.\nTogether, these results suggest that ZAP-Y292F CD8 T cells are\nselectively affected at a step of Ag-induced receptor internalization\nand that, once internalized, the CD3 component is similarly de-\ngraded. However, these results do not exclude the possibility that\nthe ZAP-Y292F mutation may additionally affect the degradation\nof the CD3 complex.\nTo further address the stage at which the fate of the TCR/CD3\ncomplex is affected in the ZAP-Y292F CD8 T, we examined the\ndynamics of the receptor with respect to T cell/APC interaction\nand formation of the IS.\nAg-induced surface TCR redistribution at the T/APC contact is\ndefective and delayed for ZAP-Y292F CD8 T cells\nThe Journal of Immunology\nthe conjugated ZAP-Y292F T cells (85% of the analyzed con-\njugates; Table II), which is also shown in three-dimensional de-\nconvolution (Fig. 4A, third row, right). By contrast, F-actin was\nconcentrated at the T/APC contact zone of ZAP-Y292F T cells as\nfor ZAP-WT T cells. The poor CD3 redistribution in ZAP-Y292F\nT cell conjugates was partly compensated after 45 min, where we\nobserved that 40% of the T cells had a patched CD3 distribution\n(Table II). For both types of CD8 T cells, the TCR/CD3 was not\nredistributed if the irrelevant peptide HY was presented by the\nAPC (Fig. 4A, ﬁrst row). One can notice that, in these conjugates,\nTCR/CD3 and F-actin are homogenously distributed around the\ncell, and that F-actin is poorly labeled in unactivated T cells.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\ngliosides and raft labeling (50) and analyzed by confocal micros-\ncopy. As a control, T cells were also incubated alone, and we\nobserved a homogenous cell surface labeling for both TCR (red)\nand CTX (green), yielding a yellow-to-orange colocalization la-\nbeling (Fig. 4B, ﬁrst row, no APC, 45 min). For the P14 ZAP-WT\nCD8 T cells, after 15-min T/gp33 APC conjugate formation, we\nobserved a redistribution of the anti-TCR, indicating a localization\nof the TCR either at the plasma membrane in the T/APC interac-\ntion zone, or inside the cell where it is no longer colocalized with\nthe green CTX membrane staining (Fig. 4B, left panel, second\nrow). Furthermore, some (but not all) of the internalized TCR ap-\npeared in organelles budding from the T/APC synapse zone, as\nalready suggested by others (21). After 45 min of conjugate for-\nmation, most of the TCR was in organelles inside the cell (Fig. 4B,\nthird row) and did not colocalize with the surface-labeled CTX.\nThis pattern was observed for 90% of T/APC conjugates analyzed,\nboth after 15 and 45 min of T/APC contact. One can notice that\nCTX is rather poorly relocalized in the T/APC contact zone, a\npattern previously described for human CD8 T cells (51), in con-\ntrast to the CTX concentration in the T/APC zone described for\nmouse CD4 T cell lines (52). Additionally, the RMA-S cells used\nas APC were very poorly labeled with CTX, consequently CTX\ncould be used to mark CD8 T cell membranes.\nTCR redistribution was different for P14 ZAP-Y292F CD8 T\ncells, as shown in Fig. 4B (right panels). After 15-min T/gp33\nAPC conjugate formation, we observed very few TCR patches and\nno TCR redistribution to the T/APC contact zone. Both TCR and\nsurface CTX were still homogeneously distributed at the T cell\nsurface. However, after 45-min conjugation, we observed some\nTCR patches inside the T cell (see also Table II), as well as re-\nmaining TCR cell surface expression, suggesting that for the ZAP-\nY292F T cells, TCR redistribution and internalization was delayed\nand at a lower rate, in agreement with the previous results (Figs. 2\nand 4A). Together, these results indicate that TCR redistribution\nand internalization, in the T/APC zone are delayed and partially\ninhibited by the ZAP-Y292F mutation. By contrast, anti-CD3\nmAb-induced TCR patching and internalization was not affected\nby the ZAP-Y292F mutation when analyzed by confocal micros-\ncopy (results not shown).\nRecruitment of cbl at the T/APC contact zone is delayed and\ndecreased in ZAP-Y292F compared with ZAP-WT T cells\nBecause the TCR clustering at the T/APC conjugate contact zone\nwas delayed in ZAP-Y292F T cells, we were interested in analyz-\ning the localization of cbl, to establish whether the loss of the\ncbl-binding site on ZAP-Y292F would affect its redistribution. In-\ndeed, cbl was previously observed to be recruited to the T/APC\ncontact zone (53), and to be colocalized with the signaling phos-\nphotyrosine zone (54). Therefore, we compared the localization of\ncbl and F-actin in T/APC conjugates formed for different periods\nof time (15– 45 min). This study was performed by confocal mi-\ncroscopy on ﬁxed and permeabilized T/APC conjugates further\nlabeled for cbl (rabbit anti-Ring cbl/anti-rabbit-A546, red), and for\nF-actin (phalloidin-A488, green), as described in Materials and\nMethods (Fig. 5).\nrow, ZAP-WT and ZAP-Y292F). However, most of the ZAP-WT\nT/APC conjugates (65%; Table II) presented an additional local-\nization in the middle of the contact zone (Fig. 5A, second row,\nZAP-WT; see bold arrow). Such a localization was observed in\nonly 30% of the ZAP-Y292F T/APC conjugates (Table II). A his-\ntogram analysis of the ﬂuorescence distribution (Fig. 5C, Cbl 1)\nshowed that the cbl labeling was concentrated at the T/APC con-\ntact zone, bulging out of the F-actin zone within the T cell, before\nthe blue (Cy5) boundary of the APC membrane. The frequency of\nthis pattern increased for both strains at 45 min, reaching 80% for\nthe ZAP-WT and 50% for the ZAP-Y292F T/APC conjugates (Ta-\nble II). Despite variability between experiments, the difference be-\ntween the two strains was always observed (three independent\nexperiments).\nIn Fig. 5B, we analyzed the distribution of LFA-1 in T/APC\nconjugates. A high redistribution of LFA-1 at the T/gp33 APC\ncontact zone was observed for both ZAP-WT and ZAP-Y292F\nCD8 T cells (Fig. 5B, second row), which was not observed in the\nT/HY APC conjugates ( ﬁrst row). On the histograms representing\nthe ﬂuorescence distribution within T/APC conjugates, one can\nobserve that LFA-1 labeling is concentrated at the T/APC contact\nzone on the T cell side, proximal to the blue boundary of the APC\nmembrane. The LFA-1 redistribution remained rather homogenous\nat the T/APC contact zone even at 45 min, as described for other\nmarkers (PKC and Talin) in naive CD8 T cell/APC conjugates\n(55). This contrasts with the description of a p-SMAC distribution\nafter 30-min conjugate formation in other T cell lines including\nnaive CD4 T cells (21, 56).\nThese results showed that the redistribution of LFA-1 and\nF-actin at the T/APC conjugate contact zone was not affected by\nthe ZAP-Y292F mutation. In contrast, the recruitment of cbl to\nthe central region, which was readily observed in the ZAP-WT\nT/APC conjugates at an early time, was defective in the ZAP-\nY292F T cells. Whether this recruitment corresponds to an en-\ndosomal localization of cbl associated with the activated TCR\ncomplex needs further investigation. It is consistent with the\nfact that cbl and associated CMS/CIN85/CD2AP can be in-\nvolved in endosome organization via rab4 (57), and with the\ndescribed effect of the ZAP-Y292F mutation as preventing the\nZAP-cbl association (58). However, this effect was partially com-\npensated with time, which suggests that other cbl associations may\nsecondarily develop in the IS. These may involve vav linked to\nZAP Y315 (59), CD28 via PI3K (60), the src PTK (61), CD2AP\nbinding to CD2 (62), or connection of CD2AP/CIN85 proteins to\nthe actin cytoskeleton (63, 64).\nDiscussion\nIn the present work, we report that naive CD8 P14 T lymphocytes\nexpressing the mutated ZAP-Y292F were affected in Ag-induced\nTCR down-modulation, and presented a defect and delay in TCR\nand cbl recruitment at the T/APC conjugate interaction zone com-\npared with ZAP-WT T cells. However, LFA-1 and F-actin were\nclearly redistributed in the ZAP-Y292F T/APC contact zone as in\nZAP-WT T/APC conjugates.\nThe Journal of Immunology\nthat TCR internalization is affected by the ZAP-Y292F mutation,\nusing an assay independent of a mAb ligand. However, it has not\nbeen ruled out that the ZAP-Y292F mutation may additionally\naffect the degradation of some of the TCR/CD3 components. Con-\nstitutive endocytosis, which depends mainly on a CD3 di-leucine\nmotif (9), or on several motifs from all CD3 components (45), was\nnot altered by the ZAP-Y292F mutation. Anti-CD3- and PMA-\ninduced TCR internalization, which depend on CD3 or CD3\ncytoplasmic domains (1, 65), were not affected either. Therefore,\nthe observed defects, although partial, would appear to implicate\nthe ZAP-Y292/cbl interaction (58) at a step that selectively affects\nAPC-induced TCR internalization.\nBinding of the cbl adaptor to phosphorylated ZAP-292Y (22)\nhas a negative effect on the activity of ZAP-70, which was mainly\nattributed to the E3-Ubiquitin ligase activity of cbl and its Ring\ndomain (66). Consistent with this, cbl was found to be involved in\nubiquitination of ZAP-70 (67) and of CD3 (68). Cbl has also\nrecently been implicated at various steps of RTK surface receptor\ninternalization and degradation. First, cbl may participate in the\nformation of clathrin-coated vesicles by adapting endophilin via a\nprotein of the CIN85 family; additionally, the ubiquitin-ligase ac-\ntivity of cbl allows the ubiquitinated receptor to be recognized by\nubiquitin interacting motif domain bearing proteins like Eps15,\nwhich is a clathrin adaptor (26 –29, 31).\nIn T cells, the double genetic deletion of c-cbl/cbl-b was asso-\nciated with a partial deﬁciency in anti-CD3-induced TCR internal-\nization, whereas the effect on Ag stimulation was not reported in\nthat work (23). CD2AP, a member of the CIN85 family (62) ca-\npable of binding cbl (69), was recently reported to play a major\nrole in T cell signaling and TCR down-modulation (32). Indeed, T\ncells from CD2AP knockout mice failed to down-modulate their\nTCR in response to Ag stimulation, and showed defects in the ﬁne\nstructure of the c-SMAC and p-SMAC (32, 33). However, in con-\ntrast to the c-cbl/cbl-b double knockout, CD2AP knockout T cells\nwere not affected in mAb-induced TCR down-modulation, at least\nat early steps (32).\nThe fact that the defect in ZAP/cbl association selectively affects\nAPC-induced, and not constitutive or anti-CD3-induced TCR in-\nternalization, together with the observation of the delay in TCR\nand cbl recruitment to the T-APC interaction zone in ZAP-Y292F\nT cells strongly suggests that the defect in TCR internalization\nmay be the consequence of its delayed recruitment to the IS. The\nalternative, that the delay in TCR recruitment and c-SMAC for-\nmation would be the consequence of a defect in TCR internaliza-\ntion, appears difﬁcult to entertain. Indeed, in contrast to receptor\nengagement by mAb, stimulation of TCRs by peptide/MHC com-\nplexes on the APC is most efﬁcient when receptors are concen-\ntrated within the IS. Consistent with this, internalized TCR is\nmainly detected at the site of the IS (Ref. 21; Fig. 4B), whereas\nmAb-induced internalization is observed all around the plasma\nmembrane (results not shown). The molecular basis for the defect\nin c-SMAC formation in the absence of CD2AP is not fully elu-\ncidated (32). Similarly, it is not clear in the ZAP-Y292F T cells 1)\nwhether the delay in recruiting the TCR/CD3 at the IS is due to the\nabsence of CD2AP/cbl/ZAP binding, and 2) which molecular in-\nteraction of a putative CD2AP/cbl/ZAP complex is required to\nform the IS. Although such CD2AP/cbl/epidermal growth factor\nreceptor complexes have been described to connect to the actin\ncytoskeleton via the actin-related protein 2/3 complex (64), it is not\nclear whether such connections function in the TCR pathway.\nThey have been suggested for CD2 signaling (63).\nThrough 1) the ubiquitination pathway, cbl may dampen strong\nPTK signals, independently from the IS, and through 2) its asso-\nciation with CD2AP, it may contribute to the concentration of\nTCR/CD3/ZAP complexes within a c-SMAC, a step required for\nsustained signaling in the absence of apoptosis (32). The latter\ncontribution of the ZAP-70/cbl-b interaction to promote T cell ac-\ntivation may be balanced by the increased rate of TCR/CD3 com-\nplex internalization within the c-SMAC.\nThe unique properties of TCR/CD3 redistribution within the\nT/APC interface may explain why distinct motifs within the re-\nceptor complex appear to control its constitutive or mAb-induced\ninternalization (45) that have implicated di-leucine motifs (9), and\nAPC-induced internalization for which phospho-CD3-associated\nZAP-Y292 appears to be a rate-limiting motif. However, the fact\nthat the defect in the ZAP-Y292F T was partial, suggests that other\nmechanisms may lead to APC-induced TCR/CD3 internalization.\nThese may involve other cbl/CD2AP binding sites within the re-\nceptor complex, cbl being a scaffold protein that may bind other\nproximal ligands (25). It is also possible that another ubiquitin\nligase is able to bind to the activated TCR complex, akin to the\nNedd4 ubiquitinin ligase that was also localized in the raft fraction\nof IgE in activated RBL cells (71).\nConcerning the relative control of TCR/CD3 complex internal-\nization and degradation, we found that for Ag-, as for anti-CD3-\ninduced receptor down-modulation, CD3 degradation was pro-\nportional to receptor internalization and down-modulation (Fig. 3).\nThis result is in agreement with other work showing that during\nconstitutive, and mAb-induced TCR internalization, all TCR/CD3\ncomponents are degraded with a rate that is proportional to the rate\nof TCR down-modulation (46) through a mechanism that is de-\npendent on lysosomal activity (72). However, an exception exists\nfor PMA-induced TCR internalization that leads to poor CD3\ndegradation (Fig. 3).\nTogether with data from the literature, the present results are\ncompatible with a role for cbl in the Ag-induced TCR/CD3 inter-\nnalization/degradation following schemes also proposed for the\nepidermal growth factor receptor (29, 30, 33). Clearly, questions\nremain about the mechanisms by which the TCR/CD3 complex is\ndriven into the endocytic pathway at the IS, and in particular\nwhether, as for the BCR, internalization via clathrin-coated vesi-\ncles is dependent on integral raft components (73, 74).\nAcknowledgments\nWe thank the members of the Centre d’Immunologie de Marseille-Luminy\n(CIML) Imaging Service, Marc Barad, Nicole Brun, and Mathieu Fallet for\ntheir help, Corinne Be´ziers-Guigue for artwork, Ste´phane Me´resse and\nNicolas Lapaque for reagents, and the CIML personnel involved in animal\ncare. We thank L. Leserman, N. Auphan, A. Guimezanes, H.-T. He, and\nG. Verdeil for critical reading of the manuscript.\nDisclosures\nThe authors have no ﬁnancial conﬂict of interest.\nReferences\n1. Luton, F., M. Buferne, V. Legendre, C. Boyer, and A.-M. Schmitt-Verhulst.\n1997. Role of CD3 and CD3 cytoplasmic domains in cytolytic T cell clones\nfunctions and TCR/CD3 down-modulation. J. Immunol. 158: 4162– 4170.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\n6. Ohno, H., J. Stewart, M.-C. Fournier, H. Bosshart, I. Rhee, S. Miyatake, T. Saito,\nA. Gallusser, T. Kirchausen, and J. S. Bonifacino. 1995. Interaction of tyrosine-\nbased sorting signals with clathrin-associated proteins. Science 269: 1872–1875.\n7. Letourneur, F., and R. D. Klausner. 1992. A novel di-leucine motif and a ty-\nrosine-based motif independently mediate lysosomal targeting and endocytosis of\nCD3 chains. Cell 69: 1143–1157.\n25. Thien, C. B., R. M. Scaife, J. M. Papadimitriou, M. A. Murphy, D. D. Bowtell,\nand W. Y. Langdon. 2003. A mouse with a loss-of-function mutation in the c-Cbl\nTKB domain shows perturbed thymocyte signaling without enhancing the activ-\nity of the ZAP-70 tyrosine kinase. J. Exp. Med. 197: 503–513.\n28. Polo, S., S. Sigismund, M. Faretta, M. Guidi, M. R. Capua, G. Bossi, H. Chen,\nP. De Camilli, and P. P. Di Fiore. 2002. A single motif responsible for ubiquitin\nrecognition and monoubiquitination in endocytic proteins. Nature 416: 451– 455.\n29. Oved, S., and Y. Yarden. 2002. Signal transduction: molecular ticket to enter\n30. Riezman, H. 2002. Cell biology: the ubiquitin connection. Nature 416: 381–383.\n31. de Melker, A. A., G. van der Horst, and J. Borst. 2004. Ubiquitin ligase activity\nof c-Cbl guides the epidermal growth factor receptor into clathrin-coated pits by\ntwo distinct modes of Eps15 recruitment. J. Biol. Chem. 279: 55465–55473.\n32. Lee, K. H., A. R. Dinner, C. Tu, G. Campi, S. Raychaudhuri, R. Varma,\nT. N. Sims, W. R. Burack, H. Wu, J. Wang, et al. 2003. The immunological\nsynapse balances T cell receptor signaling and degradation. Science 302:\n1218 –1222.\n54. Bunnell, S. C., D. I. Hong, J. R. Kardon, T. Yamazaki, C. J. McGlade, V. A. Barr,\nand L. E. Samelson. 2002. T cell receptor ligation induces the formation of\ndynamically regulated signaling assemblies. J. Cell Biol. 158: 1263–1275.\n55. O’Keefe, J. P., K. Blaine, M. L. Alegre, and T. F. Gajewski. 2004. Formation of\na central supramolecular activation cluster is not required for activation of naive\nCD8 T cells. Proc. Natl. Acad. Sci. USA 101: 9351–9356.\nThe Journal of Immunology\n64. Lynch, D. K., S. C. Winata, R. J. Lyons, W. E. Hughes, G. M. Lehrbach,\nV. Wasinger, G. Corthals, S. Cordwell, and R. J. Daly. 2003. A Cortactin-CD2-\nassociated protein (CD2AP) complex provides a novel link between epidermal\ngrowth factor receptor endocytosis and the actin cytoskeleton. J. Biol. Chem. 278:\n21805–21813.",
         "Suzel Davanture,J. Leignadier,Pascale Milani,P. Soubeyran,B. Malissen,M. Malissen,A. Schmitt-Verhulst,C. Boyer",
         null
        ],
        [
         "10",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "Abstract In this work we use Machine Learning (ML) tech-\nniques to learn the CPU time-slice utilization behavior of known\nprograms in a Linux system. Learning is done by an analysis of\ncertain static and dynamic attributes of the processes while they\nare being run. Our objective was to discover the most important\nstatic and dynamic attributes of the processes that can help best\nin prediction of CPU burst times which minimize the process\nTaT (Turn-around-Time). In our experimentation we modify the\nLinux Kernel scheduler (version 2.4.20-8) to allow scheduling\nwith customized time slices. The Waikato Environment for\nKnowledge Analysis (Weka), an open source machine-learning\ntool is used to nd the most suitable ML method to characterize\nour programs. We experimentally nd that the C4.5 Decision\nTree algorithm most effectively solved the problem. We nd that\npredictive scheduling could reduce TaT in the range of 1:4%\nto 5:8%. This was due to a reduction in the number of context\nswitches needed to complete the process execution. We nd our\nresult interesting in the context that generally operating systems\npresently never make use of a program’s previous execution\nhistory in their scheduling behavior.",
         "slices to a process according to a scheduling algorithm\nthat does not use any previous execution history of the\nprocess. It might be possible to better utilize resources if\nwe could recognize a program and predict\nits resource\nrequirements. For example, consider a process which was\nallocated a xed CPU time is pre-empted while having a\nvery small CPU time needed for completion. Such preemption,\nincreases number of context switches (also called as a process\nswitch or task switch), it causes invalidation of caches and\npipelines, swapping of buffers and so on [6]. Thus ultimately\nthis increases TaT of the program. (TaT is the delay between\nprocess submission and process completion). Thus we observe\nthat by characterizing or recognizing programs it may be\npossible to understand their previous execution history and\npredict their resource requirements. In this paper we address\nthe problem of, how to minimize the TaT of programs by\nusing ML techniques.We discover certain static and dynamic\ncharacteristics of a program are taken as features which the\nmachine learning techniques use to predict CPU burst times.\nWe call Special Time Slice or STS as the CPU burst time that\nminimizes turnaround time.\nThe rest of this paper rst discusses related work in Section II,\nan overview of machine learning techniques in Section III, we\nrerview Linux process scheduling in Section IV, and then de-\nscribes our implementation in Section V. Section VI describes\nour experimentation. Section VII presents the conclusions and\ndescribes possible future work.\nII. RELATED WORK\nThe work [10] by Suranauwarat and Taniguchi, presents\nan approach to remember the previous execution behavior of\ncertain well-known programs. They study the process times\nof these programs in variousimilaritys states. The knowledge\nof the program ow sequence (PFS) is used to extend the\nCP U time slice of a process. P F S of a process is computed\nfrom its past execution history. P F S characterizes the process\nexecution behavior and is used to decide whether the program\nexecuting currently needs additional time. They set a threshold\nTm called as maximum dispatch delay time, which determines\nthe time limit for context switching; and is a multiple of the\ndelay for minimum process switching time. They control the\nCP U time of a process Tp, by either reducing or increasing\nsome scaling feature. They conclude from experimental results\nthat overall processing time is reduced for known programs.\nTo schedule a process they search for its name in the PFS\nknowledge base and thus improve its behavior.\nIn the paper by Smith et al. [7], the authors predicted\nthe application run times using historical information. They\npresent a technique for deriving predictions for the run\ntimes of parallel applications from the run times of similar\napplications that have executed in the past. They use the\nfollowing characteristics to dene similarity: user, queue, load\nleveler script, arguments, network adapter, number of nodes,\nmaximum run time, submission time, start time, run time.\nThese characteristics are used to make a template which can\nnd the similarity by matching. They use genetic algorithms\nfor identifying good templates for a particular workload.\nA template denes the similarity between two applications.\nThey use Genetic Algorithm (GA) techniques to determine\nthose application characteristics that yield the best denition\nof similarity for making predictions. GAs are probabilistic\ntechniques and are well known for exploring large search\nspaces. However from a machine learning perspective, GAs\nare expensive in terms of computation [15] and also their\nresults are considered fragile.\nIn the paper by Gibbons [5], the authors used Statistical\nRegression methods for prediction. Wh ile regression methods\nwork well on numeric data but cannot readily be used for\nnominal data. In the paper by Fredrik et al. [4], the authors\ndescribed an application signature model used for predicting\napplication performance on a given set of grid resources.\nIn this model,\nthey introduced the notion of application\nintrinsic behaviour to separate the performance effects of the\nruntime system from the behaviour inherent in the application\nitself.The signature model is used as the basis for performance\npredictions. So their approach combines the knowledge of\napplication intrinsic behaviour with run-time predictions of\nresources. They also dene application intrinsic metrics as\nmetrics that are solely dependent on the application code and\nproblem parameters.\nFrom this brief review of related literature, we draw the\nfollowing conclusions:\n It is possible to protably predict scheduling behavior.\n However the success of the approach depends upon\nthe ML technique used to train on previous programs\nexecution behavior.\n More importantly for the ML techniques to succeed, it\nneeds a suitable characterization of the program attributes\n(features) that are most signicant for prediction.\nIII. AN INTRODUCTION TO MACHINE LEARNING\nTECHNIQUES USED\nIn this section we bring in some background information\nabout the machine learning techniques which were used in\nthe experimentation. The eld of machine learning (ML) [15]\nis concerned with the question of how to construct computer\nprograms that automatically improve with experience. ML\nmethods are also similar to Pattern Recognition methods\nis that while training\nas [16]. The most\nor learning stage several attributes are used to describe the\nobjects being studied. In our case the objects being studied\nare programs and we have come up with the best attributes\nthat can desribe programs for prediction. [8]. However the\nsuccess of a ML method also depends upon the classiers\nused. Certain classiers are more successful for different\ndata or problem domains. Here we just give a cursosry\nintroduction to the two most effective which were used [8]\nimportant aspect\nThe Learning Algorithms (Classiers): For our experi-\nments, we selected a representative set of standard machine\nlearning algorithms with different model classes. All of these\nare available in the Waikato Environment for Knowledge\nAnalysis (Weka) [9] , an open source machine learning tool.\nThe selected learner types (classier types) were: Trees and\nLazy which are listed below:\n C4.5 (or J48 in Weka [9] ) is a Reduced-Pruned Decision\nTree Learner. Decision Tree learning is one of the most\nwidely used and practical methods for inductive infer-\nence. It is a method for approximating discrete-valued\nfunctions, and is robust to noisy data and capable of\nlearning disjunctive expressions.\n IB k Learner (k-NN): This is an instance based learner\nantd assumes all instances are points in an n-dimensional\nspace. The nearest neighbors (nn) of an instance are\ndened in terms of the standard Euclidean Distance. The\nclass label of a new instance is found from the k instances\nnearest to it by assigning to it the majority label of the\nk-NN [15] [9].\nIV. A REVIEW OF LINUX PROCESS SCHEDULING\nCONCEPTS\nIn this section, we review the Linux process scheduling\nconcepts which we use to develop the modied scheduler as\ndescribed in Section V.\nScheduler: The idea behind a scheduler is simple. Its\nobjective is to best utilize processor time. Assuming there are\nrunnable processes, a process should always be running. If\nthere are more processes than processors in a system, some\nprocesses will not always be running. These processes are\nwaiting to run. Deciding which process runs next, given a\nset of runnable processes, is the fundamnetal decision the\nscheduler must take.\nTime slice: Linux uses a tick to allocate processor time\nto individual processes. There are several different hardware\ntimers that the kernel can access. One is loaded at boot time\nwith values to interrupt the system every tick. The kernel\nuses the CPU clock speed given as HZ in the kernel to set\nthe length of a single tick to approximately 2.5 nanoseconds\n[2]. Consequently, every 2.5 nanoseconds, the system takes\nover, decrements the amount of ticks that the current process\nhas, and returns the CPU to the process for another tick.\nIf the process running is out of CPU time, the scheduler\nis invoked. If there are more processes in the run-queue\nthat have been allocated CPU time, the scheduler will nd\nthe best one and switch to it. The best process is selected\nusing the task timeslice() macro dened in sched:c. This\ntask timeslice() macro calculates a weight for the process\nusing bonus which is calculated from nice values etc.. The\nhighest weight is the best process to schedule [14] [2].\nProcess Descriptor: The process descriptor [13] is dened\nin sched:h and holds all information about a process. The\namount of ticks of CPU time that a process receives before\nanother process is scheduled is stored in the process descriptor.\nWhen all processes in the run-queue have exhausted their\nCPU time,\nthe scheduler recalculates the amount of CPU\ntime for each process using the macro task timeslice(). The\ntask timeslice() uses the process’ nice value in the process\ndescriptor to determine the number of ticks the process gets.\nHere a process is referred to as p. Variables in the process\ndescriptor are referred to as p!variable. For example p!nice\nrefers to the nice value of process p.\nContext switching: The context of a process [3] consists\nof the contents of its (user) address space and the contents of\nhardware registers and kernel data structures that relate to the\nprocess. In multi-processing environments, a context switch\n(or process switch or task switch) is when one process is\nsuspended from execution , its context is recorded and another\nprocess starts its execution in the CPU.",
         "E. Experimental environment\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\nVI. EXPERIMENTS AND RESULTS\nWe explain the experimentation process by taking the matrix\nmultiplication program as an example.\nA. Effectiveness of STS\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\nbest technique2 to predict the STS [8].\nC. Finding the best attributes which can characterize STS\nclass\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.",
         "",
         "E. Experimental environment\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\nVI. EXPERIMENTS AND RESULTS\nWe explain the experimentation process by taking the matrix\nmultiplication program as an example.\nA. Effectiveness of STS\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\nbest technique2 to predict the STS [8].\nC. Finding the best attributes which can characterize STS\nclass\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.",
         "Abstract In this work we use Machine Learning (ML) tech-\nniques to learn the CPU time-slice utilization behavior of known\nprograms in a Linux system. Learning is done by an analysis of\ncertain static and dynamic attributes of the processes while they\nare being run. Our objective was to discover the most important\nstatic and dynamic attributes of the processes that can help best\nin prediction of CPU burst times which minimize the process\nTaT (Turn-around-Time). In our experimentation we modify the\nLinux Kernel scheduler (version 2.4.20-8) to allow scheduling\nwith customized time slices. The Waikato Environment for\nKnowledge Analysis (Weka), an open source machine-learning\ntool is used to nd the most suitable ML method to characterize\nour programs. We experimentally nd that the C4.5 Decision\nTree algorithm most effectively solved the problem. We nd that\npredictive scheduling could reduce TaT in the range of 1:4%\nto 5:8%. This was due to a reduction in the number of context\nswitches needed to complete the process execution. We nd our\nresult interesting in the context that generally operating systems\npresently never make use of a program’s previous execution\nhistory in their scheduling behavior.\nslices to a process according to a scheduling algorithm\nthat does not use any previous execution history of the\nprocess. It might be possible to better utilize resources if\nwe could recognize a program and predict\nits resource\nrequirements. For example, consider a process which was\nallocated a xed CPU time is pre-empted while having a\nvery small CPU time needed for completion. Such preemption,\nincreases number of context switches (also called as a process\nswitch or task switch), it causes invalidation of caches and\npipelines, swapping of buffers and so on [6]. Thus ultimately\nthis increases TaT of the program. (TaT is the delay between\nprocess submission and process completion). Thus we observe\nthat by characterizing or recognizing programs it may be\npossible to understand their previous execution history and\npredict their resource requirements. In this paper we address\nthe problem of, how to minimize the TaT of programs by\nusing ML techniques.We discover certain static and dynamic\ncharacteristics of a program are taken as features which the\nmachine learning techniques use to predict CPU burst times.\nWe call Special Time Slice or STS as the CPU burst time that\nminimizes turnaround time.\nThe rest of this paper rst discusses related work in Section II,\nan overview of machine learning techniques in Section III, we\nrerview Linux process scheduling in Section IV, and then de-\nscribes our implementation in Section V. Section VI describes\nour experimentation. Section VII presents the conclusions and\ndescribes possible future work.\nII. RELATED WORK\nThe work [10] by Suranauwarat and Taniguchi, presents\nan approach to remember the previous execution behavior of\ncertain well-known programs. They study the process times\nof these programs in variousimilaritys states. The knowledge\nof the program ow sequence (PFS) is used to extend the\nCP U time slice of a process. P F S of a process is computed\nfrom its past execution history. P F S characterizes the process\nexecution behavior and is used to decide whether the program\nexecuting currently needs additional time. They set a threshold\nTm called as maximum dispatch delay time, which determines\nthe time limit for context switching; and is a multiple of the\ndelay for minimum process switching time. They control the\nCP U time of a process Tp, by either reducing or increasing\nsome scaling feature. They conclude from experimental results\nthat overall processing time is reduced for known programs.\nTo schedule a process they search for its name in the PFS\nknowledge base and thus improve its behavior.\nIn the paper by Smith et al. [7], the authors predicted\nthe application run times using historical information. They\npresent a technique for deriving predictions for the run\ntimes of parallel applications from the run times of similar\napplications that have executed in the past. They use the\nfollowing characteristics to dene similarity: user, queue, load\nleveler script, arguments, network adapter, number of nodes,\nmaximum run time, submission time, start time, run time.\nThese characteristics are used to make a template which can\nnd the similarity by matching. They use genetic algorithms\nfor identifying good templates for a particular workload.\nA template denes the similarity between two applications.\nThey use Genetic Algorithm (GA) techniques to determine\nthose application characteristics that yield the best denition\nof similarity for making predictions. GAs are probabilistic\ntechniques and are well known for exploring large search\nspaces. However from a machine learning perspective, GAs\nare expensive in terms of computation [15] and also their\nresults are considered fragile.\nIn the paper by Gibbons [5], the authors used Statistical\nRegression methods for prediction. Wh ile regression methods\nwork well on numeric data but cannot readily be used for\nnominal data. In the paper by Fredrik et al. [4], the authors\ndescribed an application signature model used for predicting\napplication performance on a given set of grid resources.\nIn this model,\nthey introduced the notion of application\nintrinsic behaviour to separate the performance effects of the\nruntime system from the behaviour inherent in the application\nitself.The signature model is used as the basis for performance\npredictions. So their approach combines the knowledge of\napplication intrinsic behaviour with run-time predictions of\nresources. They also dene application intrinsic metrics as\nmetrics that are solely dependent on the application code and\nproblem parameters.\nFrom this brief review of related literature, we draw the\nfollowing conclusions:\n It is possible to protably predict scheduling behavior.\n However the success of the approach depends upon\nthe ML technique used to train on previous programs\nexecution behavior.\n More importantly for the ML techniques to succeed, it\nneeds a suitable characterization of the program attributes\n(features) that are most signicant for prediction.\nIII. AN INTRODUCTION TO MACHINE LEARNING\nTECHNIQUES USED\nIn this section we bring in some background information\nabout the machine learning techniques which were used in\nthe experimentation. The eld of machine learning (ML) [15]\nis concerned with the question of how to construct computer\nprograms that automatically improve with experience. ML\nmethods are also similar to Pattern Recognition methods\nis that while training\nas [16]. The most\nor learning stage several attributes are used to describe the\nobjects being studied. In our case the objects being studied\nare programs and we have come up with the best attributes\nthat can desribe programs for prediction. [8]. However the\nsuccess of a ML method also depends upon the classiers\nused. Certain classiers are more successful for different\ndata or problem domains. Here we just give a cursosry\nintroduction to the two most effective which were used [8]\nimportant aspect\nThe Learning Algorithms (Classiers): For our experi-\nments, we selected a representative set of standard machine\nlearning algorithms with different model classes. All of these\nare available in the Waikato Environment for Knowledge\nAnalysis (Weka) [9] , an open source machine learning tool.\nThe selected learner types (classier types) were: Trees and\nLazy which are listed below:\n C4.5 (or J48 in Weka [9] ) is a Reduced-Pruned Decision\nTree Learner. Decision Tree learning is one of the most\nwidely used and practical methods for inductive infer-\nence. It is a method for approximating discrete-valued\nfunctions, and is robust to noisy data and capable of\nlearning disjunctive expressions.\n IB k Learner (k-NN): This is an instance based learner\nantd assumes all instances are points in an n-dimensional\nspace. The nearest neighbors (nn) of an instance are\ndened in terms of the standard Euclidean Distance. The\nclass label of a new instance is found from the k instances\nnearest to it by assigning to it the majority label of the\nk-NN [15] [9].\nIV. A REVIEW OF LINUX PROCESS SCHEDULING\nCONCEPTS\nIn this section, we review the Linux process scheduling\nconcepts which we use to develop the modied scheduler as\ndescribed in Section V.\nScheduler: The idea behind a scheduler is simple. Its\nobjective is to best utilize processor time. Assuming there are\nrunnable processes, a process should always be running. If\nthere are more processes than processors in a system, some\nprocesses will not always be running. These processes are\nwaiting to run. Deciding which process runs next, given a\nset of runnable processes, is the fundamnetal decision the\nscheduler must take.\nTime slice: Linux uses a tick to allocate processor time\nto individual processes. There are several different hardware\ntimers that the kernel can access. One is loaded at boot time\nwith values to interrupt the system every tick. The kernel\nuses the CPU clock speed given as HZ in the kernel to set\nthe length of a single tick to approximately 2.5 nanoseconds\n[2]. Consequently, every 2.5 nanoseconds, the system takes\nover, decrements the amount of ticks that the current process\nhas, and returns the CPU to the process for another tick.\nIf the process running is out of CPU time, the scheduler\nis invoked. If there are more processes in the run-queue\nthat have been allocated CPU time, the scheduler will nd\nthe best one and switch to it. The best process is selected\nusing the task timeslice() macro dened in sched:c. This\ntask timeslice() macro calculates a weight for the process\nusing bonus which is calculated from nice values etc.. The\nhighest weight is the best process to schedule [14] [2].\nProcess Descriptor: The process descriptor [13] is dened\nin sched:h and holds all information about a process. The\namount of ticks of CPU time that a process receives before\nanother process is scheduled is stored in the process descriptor.\nWhen all processes in the run-queue have exhausted their\nCPU time,\nthe scheduler recalculates the amount of CPU\ntime for each process using the macro task timeslice(). The\ntask timeslice() uses the process’ nice value in the process\ndescriptor to determine the number of ticks the process gets.\nHere a process is referred to as p. Variables in the process\ndescriptor are referred to as p!variable. For example p!nice\nrefers to the nice value of process p.\nContext switching: The context of a process [3] consists\nof the contents of its (user) address space and the contents of\nhardware registers and kernel data structures that relate to the\nprocess. In multi-processing environments, a context switch\n(or process switch or task switch) is when one process is\nsuspended from execution , its context is recorded and another\nprocess starts its execution in the CPU.\nV. IMPLEMENTATION\nIn this section, we briey desribe about kernel modications\nand process charcterization. The kernel modications subsec-\ntion shows the kernel data structures modied for nding STS.\nThe process characterization section shows, how to nd the\nstatic and dynamic characteristics of a process.\nA. Kernel modications\nOur idea of kernel modications is similar to that of [12],\nbut the difference is that we describe the modications to\nO(1) scheduler of Linux Kernel 2.4.20-8.\n1) Process descriptor: The Linux process descriptor is\ndened in sched:h as a structure called task struct. More than\none process can request a larger amount of processor time and\nthe amount of extra time to be given is specic to the process.\nTherefore the extra time a process gets is stored in the\nprocess descriptor. A single integer eld, special time slice is\nadded to the process descriptor and is sufcient to store the\nnumber of ticks that a CPU-bound process should receive.\nThis single variable can also be used to distinguish between\nCPU-bound process from the other processes in the system.\nIf p!special time slice is less than 0 then this process should\nbe allocated the standard amount of ticks dened by the\nmacro task timeslice() in sched:c. If p!special time slice\nis greater than 0 then the process is given special time slice\nticks of processing time instead of using the standard macro,\ntask timeslice(). This requires that when processes are created,\nthe special time slice variable must be initialized to 1 in\nfork.c. The next sections describe how the modied scheduler\nuses the special time slice variable in the process descriptor.\n2) System calls: Two system calls have been added to\nthe kernel for controlling CPU-bound processes. One system\ncall is necessary to allocate more processor time for a single\nprocess. This system call performs two simple tasks. First,\nthe special time slice variable is set to the number of ticks\nrequested by the process via an argument to the system call.\nSecond, the time slice variable of the process descriptor is set\nto the value passed to the system call. The process then returns\nto run on the processor with an increased amount of processor\ntime and special handling from the scheduler. Therefore the\nprocess gets special time slice ticks of processor time, each\ntime the scheduler recalculates the process time slices.\nThe second system call\nis used to return a process to\nthe normal state in which p!time slice is calculated using\ntask timeslice() and p!special time slice is less than 0.\nto\nThe next\ntask timeslice(p), which is the standard method of assigning\np!time slice.\ntime the scheduler runs the time slice is set\n3) Modications to O(1) scheduler data structures: The\nscheduler tick() function of sched:c has been modied to\nhandle any process with special time slice > 0 differently\nfrom other processes.If this value is exhausted (decremented to\n0) then the scheduler gives the processor to the next process.\nThe run-queue is a circular doubly linked list of process\ndescriptors in the TASK RUNNING state.The rst thing that the\nnew scheduler does is check whether process currently running\nhas been granted more processor time (p! special time slice\n> 0). If this is true, and the process still has some CPU time\nleft (p!time slice > 0), then the scheduler immediately gives\nthe processor back to the current process. This guarantees that\neach processor-bound process uses all of its processor time\nimmediately, even if the scheduler is called in the middle\nof its running. If the CPU-bound process has used all of its\nCPU time (p!time slice > 0), then the scheduler continues\nits normal operation and switch to the next process.\nCalculating the goodness [2] for each process remains\nthe same. Since CPU-bound processes have large time slice\nvalues, they are not favored over other processes. This is\nacceptable since the CPU-bound processes are typically the\nonly things using the processor on a system. The CPU-\nbound process may be last to run, but there should not be\nmuch waiting to run in front of them, and from the previous\nmodication these processes is guaranteed to stay on the\nprocessor until they have used all of there CPU time.\nA slight modication to the recalculation of the time slice\nvariables checks each process to see if it can be allot-\nted more than task timeslice(p) ticks of processing time. If\nthe p!special time slice > 0, then p!time slice is set to\np!special time slice. There is a small amount of overhead\nadded to the scheduling algorithm, because each process has\nto be checked to see if it is a CPU-bound process. All other\nprocesses receive the same time slice value, as they would\nunder normal operation of the scheduler. The rest of the\nscheduling algorithm remains the same.\nB. Process characterization\nThis section describes , how we characterized the programs.\nWe studied the execution behaviour of several programs to nd\nout the characteristics that can be used to predict the STS. We\nhave taken representative programs : matrix multiplication,\nsorting programs, recursive bnocci number generating pro-\ngrams and random number generator programs. Any Machine\nlearning technique [15] requires fully labeled (categorized)\ndata for training. We categorized our data into classes and\ntrained the classiers (machine-learning techniques) [15].\nThe experimental procedure is divided into two phases. In\nthe rst phase, we create the data set from the program’s run\ntraces and make the data base with the static and dynamic\ncharactersitics of the programs and train it with machine\nlearning techniques. In the second phase, we classify this data\naccording to the tness function, STS by using machine-\nlearning techniques. The trained classiers are then used on\na different data set called as a test data set. Often due to\nlimitations in size of the data, we train classiers using a leave-\none-out technique [16] which is a standard technique.\n1) Creating the dataset: To characterize the program ex-\necution behaviour , we need to nd the static and dynamic\ncharacteristics. We used readelf [1]and size commands to\nget the attributes as shown in Table I. To nd STS, a script\nran the matrix multiplication program of size 700 x 700\nmultiple times with different values of STS on P4 Linux\nDenition\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for\ndynamic linking.\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for dy-\nnamic linking, most commonly the strings that\nrepresent\nthe names associated with symbol\ntable entries .\nThis is size ( bytes) of the dynamic linking\nsymbol table .\nThis is the global offset table size in bytes.\nThis is the procedure linkage table size in\nbytes.\nThis is read-only data size (bytes) that typically\ncontribute to a non-writable segment\nin the\nprocess image.\nThis is the size (bytes) of the rellocation infor-\nmation size.\nThis is the text or executable instructions,\nsize (bytes )of a program.\nThis is the size(bytes) of the initialized data\nthat contribute to the size of the program’s\nmemory image size.\nThis section holds uninitialized data size\nThis is total size (bytes) of the program.\nThis is name of the program and a nominal\nattribute.\nValue of InputSize which depends on the pro-\ngrams type.\nType of the Input which is a nominal value.\nValue of the best time slice.\nClass of the STSs\nSystem and selected the STS, which gives minimum TaT. The\nrst 18 (characteristics) attributes are used to predict the target\nattribute STSclass.\nWe build the data set of 84 execution instances of ve\nprograms: matrix multiplication, quick sort, merge sort, heap\nsort and a recursive Fibonacci number generator and the\nTable II shows an example of a training example or an\ninstance. We collected the data like the above for 5 programs\nwith different input sizes and different best STSs. Data of 84\ninstances of the ve programs was collected and made into\n11 categories based on the attribute STSclass with each class\nhaving an interval of 50 ticks.\n2) Training and Testing methodology: We performed two\ntypes of tests on the training examples with all the learners\ndescribed in the section, Overview of machine learning\nalgorithms, on the data sets collected in the rst phase. The\ntests are:\n Use Training Set [9]: The classier is evaluated on how\nwell it predicts the class of the instance it was trained on.\n Cross-Validation [9]: The classier is evaluated by cross-\nvalidation, using the number of folds that are entered\nin the Folds text eld (Weka). Recognition accuracy\nwas tested via cross-validation. In this test, the training\nexamples are divided into 10 parts and the classier\nclassies by taking one part as a test set and other 9\nparts as training set. Likewise, we continue for all parts.\nThe results of these tests explained in the next section Eval-\nuation and Results.\n3) Extracting the best attributes: Attributes selection1 [11]\ninvolves searching through all possible combinations of at-\ntributes in the data to nd which subset of attributes works best\nfor predicting the program execution behavior. Here our goal\nis to predict the target attribute. To do this, two objects must\nbe set up ; an attribute evaluator and a search method. The\nevaluator determines what method is used to assign a value\n(weight or worth) to each subset of attributes. Here, we used\nExhaustive and Genetic search methods, and Co-rrelation\nbased feature selection (CfsSubsetval) [9] evaluation method.\nA detailed description of feature selection is presented in [11].\nC. The design of modied scheduling process\nThe Figure 1 shows the design of modied scheduling\nprocess and the steps to minimize TaT of a program. The steps\nto minimize TaT of a program are as shown in the Figure 1\nwith numbers from 1 to 5.\n1) The program ’X’ is given to C4.5 decision tree as an\n2) The decision tree will classify ’X’ and output the STS.\n3) We send this STS information to modied scheduler\nthrough a system call.\n4) The scheduler instructs the CPU such that CPU allocates\nper second.le:///usr/share/doc/HTML/index.html Therefore\non 1.6GHz Intel P4 machine , we can save 1600MHz = 16 x\n108 clocks/second = [16 x 108 x no.of pipelines x (60320)]\nlow-level operations , with special time slice (STS) = 200.\nSTS ticks to ’X’.\n5) The CPU allocates STS ticks to ’X’ and it will run with\nB. Selecting the best machine Learning technique\nminimum TaT.\nD. Overall method\nOur method is explained as follows :\n1) Run the programs with different special time slices\nwith modied O(1) scheduler and nd STS (best spe-\ncial time slice) which gives minimum turn-around-time\n(TaT).\n2) Build the knowledge base of static and dynamic char-\nacteristics of the programs from the run traces obtained\nin step 1 and train them with the C4.5 decision tree\nalgorithm.\n3) If a new program comes, classify it and run the program\nwith this predicted STS.\n4) If the new program instance is not in the knowledge-\nbase, go to step 1.\nE. Experimental environment\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\nVI. EXPERIMENTS AND RESULTS\nWe explain the experimentation process by taking the matrix\nmultiplication program as an example.\nA. Effectiveness of STS\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\nbest technique2 to predict the STS [8].\nC. Finding the best attributes which can characterize STS\nclass\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.",
         "A. Negi,K. P. Kishore",
         null
        ],
        [
         "11",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "Neuronal leucine-rich repeat proteins (NLRRs) are type I transmembrane proteins and expressed in\nneuronal tissues, but their function remains unknown. Here, we describe the identiﬁcation and characteriza-\ntion of a new member of the NLRR family, NLRR4, and its potential role in long-lasting memory. We generated\nNLRR4-deﬁcient (NLRR4/) mice and found that they showed impaired memory retention. In hippocampus-\ndependent learning tasks, NLRR4/ mice were able to learn and maintain the memories for one day but\nunable to retain the memories for four days after learning. In contrast, in a hippocampus-independent task,\nNLRR4/ mice were able to retain the memory normally for at least seven days. These results suggest that\nNLRR4 plays a key role in hippocampus-dependent long-lasting memory.",
         "The hippocampus plays a role in declarative memory such as\nepisodic and spatial memories, but unimodal memories such as\ncued fear memories occur independently of the hippocampus.\nHippocampal lesions result in severe amnesia of contextual\nand spatial learning and memory but do not affect normal\nlearning, as seen by the cued fear-conditioning test (3, 19, 24).\nMemory formation can be divided into three phases by differ-\nences in the length of the time from acquisition; short-term\nmemory, long-term memory, and long-lasting memory (22).\nNew protein synthesis is not required for short-term memory\nbut is required for long-term memory (18, 21).\nThe hippocampus plays a role in the acquisition of short-\nterm memory and also for the conversion of short-term mem-\nory to long-term memory within several hours after acquisition,\na process known as cellular memory consolidation (22). Long-\nterm memory is converted to long-lasting memory that is per-\nmanently stored in the cortex (7, 8). This process is known as\nsystem consolidation, and the memory acquired in the past is\nkept intact even if the hippocampus is damaged. Some proteins\nimplicated in cellular memory consolidation have been re-\nported, such as N-methyl-D-aspartate receptor, calcium-calm-\nodulin-dependent protein kinase IV (CaMKIV), and cyclic\nAMP (cAMP)-responsive element-binding protein (CREB) in\nmice (5, 16, 18, 26). Cellular consolidation requires new pro-\ntein synthesis directed by CREB-mediated transcription.\n-Calcium-calmodulin-dependent protein kinase II (CaMKII)\nis the only molecule known to be involved in system consoli-\ndation, and CaMKII heterozygous mice are defective in long-\nlasting memory and cortical long-term potentiation (7). How-\never, how CaMKII is involved in this process is not fully\nunderstood.\nIn this study, we describe a novel type I transmembrane\nprotein termed NLRR4 which exhibits similarity to neuronal\nleucine-rich repeat proteins NLRR1 to NLRR3. NLRR1 and\nNLRR2 are expressed in the developing nervous system (4, 14,\n28, 29). NLRR3 is induced by brain injury and regulated by\nRas-mitogen-activated protein kinase signaling (9, 10, 15).\nHowever, their functions remain to be studied. To uncover the\nfunction of NLRR4 in vivo, we generated NLRR4-deﬁcient\n(NLRR4/) mice by replacing its exons with the -galactosi-\ndase gene by homologous\nrecombination. Homozygous\nNLRR4/ mice were viable and fertile. We found that their\nlong-term memory in hippocampus-dependent tasks was se-\nlong-term potentiation\nverely impaired, while hippocampal\nwas intact. Interestingly, hippocampus-independent memory\nassessed by the cued fear conditioning in NLRR4/ mice was\nnormal. These data indicate that NLRR4 is important for\nmaintenance of hippocampus-dependent memories.\nMice. C57/BL6J and ICR mice were purchased from Nippon CLEA (Tokyo,\nJapan) and mutant mice were maintained and mated in our animal facility. All\nmice were housed in a speciﬁc-pathogen-free barrier animal facility. NLRR4/\nmice were backcrossed to the C57BL/6J strain at least 6 times before being used\nfor behavioral and electrophysiological experiments. All experiments were per-\nformed according to the guidelines of the Animal Care and Experimentation\nCommittee of University of Tokyo and Wakayama Medical University.\nExpression cloning of NLRR4 cDNA. Expression cloning of a cDNA encoding\nthe B61 antigen was carried out using COS7 cells and a cDNA library prepared\nfrom LO cells as previously described (13). We used anti-rat immunoglobulin\nG-conjugated magnetic beads and a magnetic cell sorter to enrich COS7 cells\nexpressing the B61 antigen. Plasmids recovered from enriched COS7 cells were\ntransfected in COS7 cells. By repeating this selection 3 times, we obtained a\ncDNA encoding the B61 antigen NLRR4.\nthe exon with initiation codon was replaced with the -galactosidase gene. The\nneomycin resistance gene was inserted between the homologous regions, and the\nherpes simplex virus thymidine kinase cassette was ligated to the 3 end of the\nvector. TT2 embryonic stem (ES) cells were electroporated and selected with\nG418 and 2-deoxy-2-ﬂuoro--D-arabinofuranosyl-5-iodouracil. Drug-resistant\nclones were analyzed by Southern blotting. BamHI- or EcoRI-digested genomic\nDNA was hybridized with the 400-bp 3 genomic fragment that corresponded to\ngenomic sequences outside of the targeting vector or 500-bp 5 genomic frag-\nment. Chimeric mice were generated by the aggregation method as previously\n60 s were guided to the platform and allowed to stay there for 30 s. Each mouse\nwas trained four trials per day for 5 consecutive days. A different starting point\nwas used for each of the trials. The time to reach the platform was recorded\n(escape latency). Probe tests were performed at three time points (retention day)\nafter the last training. In the probe test, the platform was removed and the mice\nwere allowed to swim for 60 s. The swimming speed was calculated based on\nmovement data. For all experiments, the movement of each mouse was moni-\ntored by a charge-coupled device camera and the data were processed with NIH\nImage WM 2.12r (O’Hara & Co.).\nElectrophysiology. All experiments were performed to compare NLRR4/\nmice with wild-type mice in a blind fashion using male littermates. Nine- to\n11-week-old NLRR4/ and wild-type mice were decapitated under deep halo-\nthane anesthesia, and both hippocampi were removed. Hippocampal slices (400\nm thick) were cut with a Vibratome tissue slicer and placed in a humidiﬁed\nholding chamber for at least 2 h. A single slice was then transferred to a\nrecording chamber maintained at 25°C, and submerged beneath a continuously\nperfusing medium that had been saturated with 95% O2/5% CO2. The medium\ncomprised 119 mM NaCl, 2.5 mM KCl, 1.3 mM MgSO4, 2.5 mM CaCl2, 1.0 mM\nNaH2PO4, 26.2 mM NaHCO3 and 11 mM glucose. All perfusing solutions con-\ntained picrotoxin (100 M) to block gamma-aminobutyric acid A receptor-\nmediated inhibitory synaptic responses. Field potential recordings were made\nusing a glass electrode ﬁlled with 3 M NaCl and placed in the stratum radiatum\nin the hippocampal CA1 region. An Axopatch-1D ampliﬁer was used, and the\nsignal was ﬁltered at 1 kHz and digitized at 10 kHz.\nTo evoke synaptic responses, a bipolar tungsten electrode was placed in the\nstratum radiatum, and Schaffer collateral and commissural ﬁbers were stimulated\nat 0.1 Hz (test pulses). A single high-frequency stimulus train (100 Hz, 1 s) was\napplied with the test pulse intensity to induce long-term potentiation. The input-\noutput relationship of basal synaptic responses was examined in the presence of\nD-2-amino-5-phosphonovaleric acid (25 M) to block N-methyl-D-aspartate re-\nceptor-mediated synaptic responses. A low concentration of 6-cyano-7-nitroqui-\nnoxaline-2,3-dione (1 M) was also present to partially block alpha-amino-3-\nhydroxy-5-methyl-4-isoxazole propionate (AMPA) receptors. This enables more\naccurate measurements of the input-output relationship, since the presence of\nlow concentrations of 6-cyano-7-nitroquinoxaline-2,3-dione reduces the nonlin-\near summation of ﬁeld excitatory postsynaptic potentials when strong stimulus\nstrengths are used. For measurements of paired-pulse facilitation, afferent ﬁbers\nwere stimulated twice at intervals of 50, 100, and 200 ms in the presence of 25\nM D-2-amino-5-phosphonovaleric acid. All values were expressed as the mean\n standard errors of the mean. Student’s t test (two-tailed, unpaired) was used\nto determine whether there was a signiﬁcant difference in the means between two\nsets of data.",
         "RESULTS\nMolecular cloning of the NLRR4 gene. In our studies on the\ndevelopment of hematopoiesis, we generated a number of\nmonoclonal antibodies against LO cells (12), a hemangioblast-\nlike cell line. B61 is one such antibody and recognizes a cell\nsurface molecule on LO cells. We employed an expression\ncloning strategy to identify the B61 antigen by using COS7 cells\nand an expression library of LO cells, and we obtained a 2.7-kb\ncDNA encoding the B61 antigen. While the B61 antigen was\nhighly expressed in the hemangioblast-like cell line LO, B61\nmRNA was not found in hematopoietic tissues (data not\nshown). Instead, we found its expression in neuronal tissues as\ndescribed below. Sequence analysis revealed that the B61\ncDNA has an open reading frame encoding 735 amino acid\nresidues with two hydrophobic segments, one at the amino\nterminus (amino acids 1 to 21) and the second (amino acids\n679 to 704) in the internal region. There are two distinct motifs\nin the extracellular region, leucine-rich repeats and ﬁbronectin\ntype III-like repeats and a small intracellular region consisting\nof only 30 amino acid residues without any known motifs. Due\nto the similarity to the previously identiﬁed neuronal leucine\nrich-repeat proteins NLRR1 to NLRR3 (4, 14, 28, 29), we\nnamed B61 NLRR4. These members are type I transmem-\n-Galactosidase staining of whole brains. For -galactosidase staining, brains\nwere ﬁxed in phosphate-buffered saline containing 2% paraformaldehyde, 0.2%\nglutaraldehyde, and 0.02% NP-40 and sectioned at 300-m thickness by vi-\nbratome. After washing, the sections were placed in staining buffer (44 mM\nHEPES [pH 7.4], 3 mM potassium ferricyanide, 15 mM NaCl, 1.3 mM MgCl2, 0.5\nmg/ml 5-bromo-4-chloro-3-indolyl-- D-galactopyranoside [X-Gal] in phosphate-\nbuffered saline) at room temperature.\nContextual fear-conditioning tests. For each experiment, a mouse was placed\nin the conditioning chamber and given a series of foot shocks alone (2-s duration,\n0.3 mA, 1 min apart). To test its memory, the mouse was placed in the same\nconditioning chamber and freezing responses were analyzed using Image FZC\n(O’Hara & Co.), with modiﬁed software available in the public-domain National\nInstitutes of Health Image program. In the experiment, a mouse was trained by\nthree foot shocks and then its behavior was tested 1, 4, or 7 days after training.\nCued fear-conditioning tests. For training, a mouse was placed in the condi-\ntioning chamber for 2 min before the onset of conditioned stimuli, a tone which\nlasted for 30 s at 10 kHz and 70 dB. The last 2 s of the conditioned stimulus was\npaired with am unconditioned stimulus, 0.3 mA of a continuous electric foot\nshock. After an additional 30 s in the chamber, the mouse was returned to its\nhome cage. Four and seven days after training, the mouse was placed in a\ndifferent chamber in which the same tone as the conditioned stimuli was applied\nfor 120 s after a 1-min habituation period (preconditioned stimulus).\nMorris water maze test. The water pool used in these experiments was 1 m in\ndiameter and made up of white polyvinyl chloride and the temperature of the\nwater was 27  1°C. For the hidden-platform task, an acrylic transparent plat-\nform (10 cm in diameter) was submerged 5 mm below the surface of water that\nwas made opaque by adding nontoxic odorless white paint. The location of the\nplatform was ﬁxed during the trials for each mouse. If it located the platform\nwithin 60 s, it was kept on it for 30 s. Mice that failed to ﬁnd the platform within\nbrane proteins with multiple leucine-rich repeats and a ﬁ-\nbronectin type III-like domain.\nAn NLRR4-homologous gene was found in the human ge-\nnome but not in Caenorhabditis elegans or Drosophila melano-\ngaster. A human cDNA homologous to NLRR4 encoded a type\nI transmembrane protein with 640 amino acids (Fig. 1a) with\nan overall homology of 64% at the amino acid level. NLRR4\nshowed 21% and 20% homology at the amino acid level to\nNLRR1 and NLRR3 (28, 29), respectively. Although there are\ntwo immunoglobulin C2 loops in NLRR1 and NLRR3, no such\nstructure was found in NLRR4 (Fig. 1b). Northern blot anal-\nysis revealed that NLRR4 mRNA was expressed in lung, heart,\nand ovary (Fig. 2b). In the adult brain, NLRR4 was strongly\nexpressed in the hippocampus and weakly in the cerebellum\n(Fig. 2a).\nTargeted disruption of the NLRR4 gene. To uncover the\nfunction of NLRR4 in vivo, we generated an NLRR4-deﬁcient\nmouse line. As shown in Fig. 3a, we replaced the exons of the\nNLRR4 gene with the -galactosidase gene and the neomycin\nresistance gene (neo). The deleted exon encoded the initiation\ncodon, the signal peptide, and the leucine-rich repeat domains,\nand the targeting vector was designed to express -galactosi-\ndase from NLRR4 regulatory elements. We isolated six inde-\npendent ES clones with homologous recombination (Fig. 3b),\nand chimeric male mice were generated from three of the\nclones. Heterozygous embryos from three independent chi-\nmeric mice exhibited the same -galactosidase expression pat-\ntern. Genotyping by Southern blotting showed successful tar-\ngeting of the NLRR4 gene (Fig. 3c).\nNLRR4 protein expression was not detected in NLRR4/\nembryos by ﬂow cytometric analysis of primary culture cells\nderived from wild-type and NLRR4/ embryos with an anti-\nNLRR4 monoclonal antibody (Fig. 3d). We genotyped 372\npups born from heterozygous matings to assess whether the\nexpected number of knockout and heterozygote pups was af-\nfected. The number of NLRR4/ to NLRR4/ to\nNLRR4/ was 83:196:93, as expected for unbiased Mende-\nlian frequencies (NLRR4/), and the mice grew with normal\nbody weights. Both males and females are fertile, and no in-\ncreased mortality was observed up to 1 year of age.\nExpression of NLRR4 in the adult brain and anatomy of the\nNLRR4/ hippocampus. Since NLRR4 mRNA was found in\nthe brain by Northern blotting analysis (Fig. 2), we precisely\nanalyzed the expression of NLRR4 in the brain. As the -ga-\nlactosidase gene was inserted in the exon of NLRR4, we per-\nformed -galactosidase staining of the adult brain of two-\nmonth-old heterozygous mice (NLRR4/) (Fig. 4a, b, c, and\ne). NLRR4 was expressed in the CA1 and CA3 regions and\ndentate gyrus of the hippocampus. In addition, NLRR4 was\nexpressed in layers V and VI in the neocortex and piriform\ncortex. It was also expressed in the inner granular layer and\nPurkinje cells of the cerebellum. The NLRR4/ brain was\nalso stained for -galactosidase activity, and there was no dif-\nference in the staining pattern between NLRR4/ and\nNLRR4/ mice (Fig. 4c, d, e, and f).\nWe performed histochemical staining to examine hippocam-\npal anatomy in wild-type and NLRR4/ mice. Hematoxylin-\neosin staining and immunohistochemical analysis of NeuN, a\nneuron-speciﬁc DNA binding protein, showed no difference\nbetween wild-type and NLRR4/ mice (Fig. 4g, h, i, and j).\nThese results indicated that the gross anatomy of hippocampus\nwas not affected by the NLRR4 mutation.\nDefective contextual fear memory in NLRR4/ mice. Since\nNLRR4 is expressed in the hippocampus, which plays a role in\nlearning and memory (3, 24), we considered the possibility that\nNLRR4 plays a role in memory formation. To address this, we\nexamined the associative emotional memory of the mutant\nmice using the contextual fear-conditioning test. If mice learn\nfear such as that resulting from an electric foot shock in a\nspatial context, they show some defensive responses, including\nfreezing behavior in the conditioning chamber (3).\nfear-conditioning test,\nWithout experiencing the electric shock, wild-type and\nNLRR4/ mice showed little of this behavior. Mice with\neither genotype spent the same time freezing in the context at\n24 h after the training, indicating that NLRR4/ mice are\nable to learn fear by contextual fear-conditioning. However,\nNLRR4/ mice froze signiﬁcantly less than wild-type mice 4\nand 7 days after the training (Fig. 5a). There were no signiﬁ-\ncant differences between the two genotypes in terms of noci-\nceptive sensitivity to an electrical foot shock (Fig. 5b). For the\ncontextual\nfear-eliciting experiences\nsuch as an electric foot shock are input into the amygdala and\nspatial information such as the conditioning chamber is sent to\nthe hippocampus. Both sets of information are consolidated\nand stored in a manner dependent on the hippocampus and\namygdala. As NLRR4/ mice showed normal freezing behav-\nior during the 1-day retention period, cellular consolidation in\nNLRR4/ mice was normal. However, as the freezing time\nwas signiﬁcantly reduced in NLRR4/ mice 4 days after the\ntraining, their memory was impaired after long retention delay.\nNormal tone-dependent cued fear memory in NLRR4/\nmice. As contextual fear-conditioning tasks are dependent on\nthe hippocampus and amygdala, we examined hippocampus-\nindependent fear memory to assess the function of amygdala in\nNLRR4/ mice. The auditory cued fear-conditioning test is\nan amygdala-dependent but hippocampus-independent task\n(19). Wild-type and NLRR4/ mice were exposed simulta-\nneously to a tone (conditioned stimulus) and an electric foot\nshock (unconditioned stimulus). After this learning process, we\nmeasured the freezing time when the tone was replayed in a\ndifferent chamber. Mice with either genotype did not show any\nfreezing behavior before presenting conditioned stimuli (pre-\nconditioned stimuli). In the presence of a conditioned stimu-\nlus, mice of either genotype showed the same level of freezing\ntime at both time points after the conditioning (Fig. 5c). No\nsigniﬁcant differences were observed in freezing time between\nthe two genotypes 4 and 7 days after the training, indicating\nthat NLRR4 is dispensable for tone information processing\nand fear memory is fully integrated into the amygdala without\nNLRR4 after long retention.\nDefective spatial memory in NLRR4/ mice. For the fear\nmemory tests, NLRR4/ mice showed defects in the hippocam-\npus-dependent memory (contextual fear-conditioning test), but\nnot the hippocampus-independent memory (tone fear-condition-\ning test). These results indicated that NLRR4/ mice are defec-\ntive in hippocampus-dependent fear memories. We then exam-\nined another type of hippocampus-dependent memory without\nthe experience of fear in NLRR4/ mice. We employed the\nhidden-platform Morris water maze test, which is a hippocampus-\ndependent learning task. Wild-type and NLRR4/ mice im-\nproved their performance during 5 days of training at 4 trials/day\nNormal hippocampal long-term potentiation in NLRR4/\nmutants. The ﬁnding that NLRR4/ mice showed normal\nshort-term memory but\nimpaired long-lasting memory\nprompted us to examine whether the physiological properties\nof the hippocampus were altered by the lack of NLRR4. We\nﬁrst examined the input-output relationship of synaptic trans-\nmission to assess possible changes in basal synaptic properties,\nusing hippocampal slices. AMPA receptor-mediated excitatory\npostsynaptic potentials in the CA1 region in response to vari-\nintensities were indistinguishable between\nous\nstimulus\nNLRR4/ and wild-type mice (Fig. 7a), indicating that the\nbasal synaptic efﬁcacy was not altered in NLRR4/ mice.\nrelease probability were intact\nWe then examined paired-pulse facilitation, which is a pre-\nsynaptic form of short-term plasticity and is sensitive to pre-\nsynaptic release probability (20). The paired-pulse facilitation\nobserved in slices from NLRR4/ mice did not signiﬁcantly\ndiffer from that of wild-type mice at any interstimulus interval\nexamined (Fig. 7b), indicating that presynaptic properties in-\ncluding\nin\nNLRR4/ mice. Furthermore, we analyzed long-term synap-\ntic plasticity and found that a single high-frequency stimulus\ntrain (100 Hz/s) produced long-term potentiation in mutant\nslices that was indistinguishable from that of wild-type slices\n(Fig. 7c and d), which is consistent with the results that\nNLRR4/ mice performed normally hippocampus-depen-\ndent memory tasks 24 h after the training. Since there was a\ntendency of decreased long-term potentiation 60 min after the\ninduction in NLRR4/ mice, we also followed the time\ncourse of long-term potentiation for another hour, but found\nthat long-term potentiation 120 min after the induction was\nalso normal (Fig. 7e and f). Taken together, these results\nindicate that NLRR4 is not required for basal synaptic trans-\nmission or short-term and the early phase of long-term plas-\nticity at the Schaffer collateral-CA1 synapse in the hippocampus.",
         "DISCUSSION\nWe isolated a cDNA encoding a new member of the NLRR\nfamily, a type I transmembrane protein with leucine-rich re-\npeats and a ﬁbronectin type III repeat. Leucine-rich repeats\nconsist of multiple copies of the conserved sequence. Proteins\nwith leucine-rich repeats mediate homophilic or heterophilic\nprotein-protein interactions. Slits function as a ligand for the\nRobo receptors through its leucine-rich repeat (2, 11), and\nNgR and Trks are receptors for Nogo and neurotrophic factors\n(6, 17), respectively. CAPS and connectin function as cell rec-\nognition molecules in neuromuscular development via their\nhomophilic interaction (23, 27). We therefore tested a ho-\nmophilic interaction of NLRR4 protein. However, cell aggre-\ngation assays using NLRR4-expressing cells failed to show any\ninteraction (data not shown), suggesting that NLRR4 may be\ninvolved in cell-cell interactions as a component or it may be a\nreceptor or a ligand for an unknown molecule.\nNLRR4-deﬁcient mice generated by inserting the -galac-\ntosidase cDNA into the NLRR4 gene locus developed nor-\nmally and exhibited no anatomical abnormalities. The NLRR4\nexpression revealed by -galactosidase activity indicates that\nNLRR4 is expressed in the hippocampus, layers V and VI in\nthe cortex and piriform cortex, the internal granule layer, and\nPurkinje cell in the cerebellum. The hippocampus plays a role\nin the initial storage and the cellular consolidation of spatial\ninformation (22, 26). Since NLRR4 is highly expressed in the\nhippocampus, it is possible that NLRR4 is involved in learning\nand/or memory. In fact, we found that\nthe memory of\nNLRR4/ mice was severely impaired for two learning tests\nthat are dependent on the hippocampus; the contextual fear-\nconditioning and hidden-platform Morris water maze tests.\nIn the contextual fear-conditioning test, NLRR4/ mice\nwere able to learn normally as shown by the normal freezing\nbehavior 1 day after training. However, NLRR4/ mice failed\nto retain the memory longer as their freezing behavior was\nsigniﬁcantly reduced 4 and 7 days after training. We obtained\nsimilar results in the hidden-platform Morris water maze\ntest. Thus,\nin the hippocampus-dependent behavioral tests\nNLRR4/ mice are able to learn and retain memories nor-\nmally memory 1 day after training. Consistently, NLRR4/\nmice showed normal long-term potentiation in the hippocam-\npus, indicating that initial memory and cellular consolidation\nare normal in NLRR4/ mice. Contextual and spatial mem-\nories are known to be retained in the hippocampus for several\ndays and long-lasting memory is permanently stored in the\ncortex (7, 8). As memory retention 4 days after training was\nseverely impaired in NLRR4/ mutant mice, it is likely that\nNLRR4 is required for long-lasting memory.\nIn contrast to hippocampus-dependent memories, NLRR4/\nmice showed normal cued fear memory in which a tone was\npaired with an electric foot shock. This task depends on the\namygdala but is independent of the hippocampus. It is known\nthat hippocampus lesion results in impaired spatial memory\nbut normal memory in the cued fear-conditioning tests (19).\nLikewise, the hippocampal CA1 region-speciﬁc knockout of\nthe NR1 gene encoding an essential subunit of the N-methyl-\nD-aspartate receptor results in amnesia of contextual fear\nmemory without affecting cued fear memory (26). Axon of\nCA1 pyramidal cells mainly extend to the layer V in entorhinal\ncortex and NLRR4 is expressed in CA1, CA3, and dentate\ngyrus in the hippocampus and in layers V and VI in the cortex.\nThese results collectively suggest that the impaired retention of\nmemories in NLRR4/ mice depends on the hippocampal\nfunction but is independent of the amygdala.\nAmnesia in NLRR4/ mice is similar to that in CaMKII\nheterozygous mice, which exhibit severely impaired hippocam-\npus-dependent\nlong-lasting memory, while the molecular\nmechanism how CaMKII is involved in long-lasting memory\nstill remains to be elucidated, there may be a link between\nNLRR and CaMKII. CREB is required for the cellular con-\nsolidation of long-term memory, indicating that a protein(s)\nregulated by CREB plays a role in long-term memory (1, 5,\n18). Therefore, the impaired memory in NLRR4/ mice\nmight be due to an alteration in this pathway. As NLRR4 is a\ntransmembrane protein with protein-protein interaction mo-\ntifs, it is tempting to speculate that NLRR4 may generate an\nintracellular signal that affects the CREB pathway in the hip-\npocampus. It will be of interest to ﬁnd a molecule that interacts\nwith NLRR4.\nWe thank S. Kida for helpful discussion and critical reading of the\nmanuscript and T. Fukuda, T. Watanabe, H. Kawano, and T. Sato for\ntechnical advice. We are also grateful to T. Akiyama and his staff for\ntheir help for behavioral tests and to Y. Iwakura for providing us with\na mouse genomic DNA library and the LacZ plasmid.\nThis work was supported in part by Grants-in-Aid for Scientiﬁc\nResearch, the 21st Century COE program, the Center for Biosignaling\nand Integrated Brain Medical Science, and Special Coordination\nFunds for Promoting Science and Technology from the Ministry of\nEducation, Culture, Sports and Technology, by RISTEX and CREST\nof the Japan Science and Technology Agency, and by the Inoue Foun-\ndation for Science.\n2. Battye, R., A. Stevens, R. L. Perry, and J. R. Jacobs. 2001. Repellent signal-\ning by Slit requires the leucine-rich repeats. J. Neurosci. 21:4290–4298.\n3. Best, P. J., A. M. White, and A. Minai. 2001. Spatial processing in the brain:\nthe activity of hippocampal place cells. Annu. Rev. Neurosci. 24:459–486.\n4. Bormann, P., L. W. Roth, D. Andel, M. Ackermann, and E. Reinhard. 1999.\nzfNLRR, a novel leucine-rich repeat protein is preferentially expressed dur-\ning regeneration in zebraﬁsh. Mol. Cell. Neurosci. 13:167–179.",
         "",
         "Neuronal leucine-rich repeat proteins (NLRRs) are type I transmembrane proteins and expressed in\nneuronal tissues, but their function remains unknown. Here, we describe the identiﬁcation and characteriza-\ntion of a new member of the NLRR family, NLRR4, and its potential role in long-lasting memory. We generated\nNLRR4-deﬁcient (NLRR4/) mice and found that they showed impaired memory retention. In hippocampus-\ndependent learning tasks, NLRR4/ mice were able to learn and maintain the memories for one day but\nunable to retain the memories for four days after learning. In contrast, in a hippocampus-independent task,\nNLRR4/ mice were able to retain the memory normally for at least seven days. These results suggest that\nNLRR4 plays a key role in hippocampus-dependent long-lasting memory.\nThe hippocampus plays a role in declarative memory such as\nepisodic and spatial memories, but unimodal memories such as\ncued fear memories occur independently of the hippocampus.\nHippocampal lesions result in severe amnesia of contextual\nand spatial learning and memory but do not affect normal\nlearning, as seen by the cued fear-conditioning test (3, 19, 24).\nMemory formation can be divided into three phases by differ-\nences in the length of the time from acquisition; short-term\nmemory, long-term memory, and long-lasting memory (22).\nNew protein synthesis is not required for short-term memory\nbut is required for long-term memory (18, 21).\nThe hippocampus plays a role in the acquisition of short-\nterm memory and also for the conversion of short-term mem-\nory to long-term memory within several hours after acquisition,\na process known as cellular memory consolidation (22). Long-\nterm memory is converted to long-lasting memory that is per-\nmanently stored in the cortex (7, 8). This process is known as\nsystem consolidation, and the memory acquired in the past is\nkept intact even if the hippocampus is damaged. Some proteins\nimplicated in cellular memory consolidation have been re-\nported, such as N-methyl-D-aspartate receptor, calcium-calm-\nodulin-dependent protein kinase IV (CaMKIV), and cyclic\nAMP (cAMP)-responsive element-binding protein (CREB) in\nmice (5, 16, 18, 26). Cellular consolidation requires new pro-\ntein synthesis directed by CREB-mediated transcription.\n-Calcium-calmodulin-dependent protein kinase II (CaMKII)\nis the only molecule known to be involved in system consoli-\ndation, and CaMKII heterozygous mice are defective in long-\nlasting memory and cortical long-term potentiation (7). How-\never, how CaMKII is involved in this process is not fully\nunderstood.\nIn this study, we describe a novel type I transmembrane\nprotein termed NLRR4 which exhibits similarity to neuronal\nleucine-rich repeat proteins NLRR1 to NLRR3. NLRR1 and\nNLRR2 are expressed in the developing nervous system (4, 14,\n28, 29). NLRR3 is induced by brain injury and regulated by\nRas-mitogen-activated protein kinase signaling (9, 10, 15).\nHowever, their functions remain to be studied. To uncover the\nfunction of NLRR4 in vivo, we generated NLRR4-deﬁcient\n(NLRR4/) mice by replacing its exons with the -galactosi-\ndase gene by homologous\nrecombination. Homozygous\nNLRR4/ mice were viable and fertile. We found that their\nlong-term memory in hippocampus-dependent tasks was se-\nlong-term potentiation\nverely impaired, while hippocampal\nwas intact. Interestingly, hippocampus-independent memory\nassessed by the cued fear conditioning in NLRR4/ mice was\nnormal. These data indicate that NLRR4 is important for\nmaintenance of hippocampus-dependent memories.\nMice. C57/BL6J and ICR mice were purchased from Nippon CLEA (Tokyo,\nJapan) and mutant mice were maintained and mated in our animal facility. All\nmice were housed in a speciﬁc-pathogen-free barrier animal facility. NLRR4/\nmice were backcrossed to the C57BL/6J strain at least 6 times before being used\nfor behavioral and electrophysiological experiments. All experiments were per-\nformed according to the guidelines of the Animal Care and Experimentation\nCommittee of University of Tokyo and Wakayama Medical University.\nExpression cloning of NLRR4 cDNA. Expression cloning of a cDNA encoding\nthe B61 antigen was carried out using COS7 cells and a cDNA library prepared\nfrom LO cells as previously described (13). We used anti-rat immunoglobulin\nG-conjugated magnetic beads and a magnetic cell sorter to enrich COS7 cells\nexpressing the B61 antigen. Plasmids recovered from enriched COS7 cells were\ntransfected in COS7 cells. By repeating this selection 3 times, we obtained a\ncDNA encoding the B61 antigen NLRR4.\nthe exon with initiation codon was replaced with the -galactosidase gene. The\nneomycin resistance gene was inserted between the homologous regions, and the\nherpes simplex virus thymidine kinase cassette was ligated to the 3 end of the\nvector. TT2 embryonic stem (ES) cells were electroporated and selected with\nG418 and 2-deoxy-2-ﬂuoro--D-arabinofuranosyl-5-iodouracil. Drug-resistant\nclones were analyzed by Southern blotting. BamHI- or EcoRI-digested genomic\nDNA was hybridized with the 400-bp 3 genomic fragment that corresponded to\ngenomic sequences outside of the targeting vector or 500-bp 5 genomic frag-\nment. Chimeric mice were generated by the aggregation method as previously\n60 s were guided to the platform and allowed to stay there for 30 s. Each mouse\nwas trained four trials per day for 5 consecutive days. A different starting point\nwas used for each of the trials. The time to reach the platform was recorded\n(escape latency). Probe tests were performed at three time points (retention day)\nafter the last training. In the probe test, the platform was removed and the mice\nwere allowed to swim for 60 s. The swimming speed was calculated based on\nmovement data. For all experiments, the movement of each mouse was moni-\ntored by a charge-coupled device camera and the data were processed with NIH\nImage WM 2.12r (O’Hara & Co.).\nElectrophysiology. All experiments were performed to compare NLRR4/\nmice with wild-type mice in a blind fashion using male littermates. Nine- to\n11-week-old NLRR4/ and wild-type mice were decapitated under deep halo-\nthane anesthesia, and both hippocampi were removed. Hippocampal slices (400\nm thick) were cut with a Vibratome tissue slicer and placed in a humidiﬁed\nholding chamber for at least 2 h. A single slice was then transferred to a\nrecording chamber maintained at 25°C, and submerged beneath a continuously\nperfusing medium that had been saturated with 95% O2/5% CO2. The medium\ncomprised 119 mM NaCl, 2.5 mM KCl, 1.3 mM MgSO4, 2.5 mM CaCl2, 1.0 mM\nNaH2PO4, 26.2 mM NaHCO3 and 11 mM glucose. All perfusing solutions con-\ntained picrotoxin (100 M) to block gamma-aminobutyric acid A receptor-\nmediated inhibitory synaptic responses. Field potential recordings were made\nusing a glass electrode ﬁlled with 3 M NaCl and placed in the stratum radiatum\nin the hippocampal CA1 region. An Axopatch-1D ampliﬁer was used, and the\nsignal was ﬁltered at 1 kHz and digitized at 10 kHz.\nTo evoke synaptic responses, a bipolar tungsten electrode was placed in the\nstratum radiatum, and Schaffer collateral and commissural ﬁbers were stimulated\nat 0.1 Hz (test pulses). A single high-frequency stimulus train (100 Hz, 1 s) was\napplied with the test pulse intensity to induce long-term potentiation. The input-\noutput relationship of basal synaptic responses was examined in the presence of\nD-2-amino-5-phosphonovaleric acid (25 M) to block N-methyl-D-aspartate re-\nceptor-mediated synaptic responses. A low concentration of 6-cyano-7-nitroqui-\nnoxaline-2,3-dione (1 M) was also present to partially block alpha-amino-3-\nhydroxy-5-methyl-4-isoxazole propionate (AMPA) receptors. This enables more\naccurate measurements of the input-output relationship, since the presence of\nlow concentrations of 6-cyano-7-nitroquinoxaline-2,3-dione reduces the nonlin-\near summation of ﬁeld excitatory postsynaptic potentials when strong stimulus\nstrengths are used. For measurements of paired-pulse facilitation, afferent ﬁbers\nwere stimulated twice at intervals of 50, 100, and 200 ms in the presence of 25\nM D-2-amino-5-phosphonovaleric acid. All values were expressed as the mean\n standard errors of the mean. Student’s t test (two-tailed, unpaired) was used\nto determine whether there was a signiﬁcant difference in the means between two\nsets of data.\nRESULTS\nMolecular cloning of the NLRR4 gene. In our studies on the\ndevelopment of hematopoiesis, we generated a number of\nmonoclonal antibodies against LO cells (12), a hemangioblast-\nlike cell line. B61 is one such antibody and recognizes a cell\nsurface molecule on LO cells. We employed an expression\ncloning strategy to identify the B61 antigen by using COS7 cells\nand an expression library of LO cells, and we obtained a 2.7-kb\ncDNA encoding the B61 antigen. While the B61 antigen was\nhighly expressed in the hemangioblast-like cell line LO, B61\nmRNA was not found in hematopoietic tissues (data not\nshown). Instead, we found its expression in neuronal tissues as\ndescribed below. Sequence analysis revealed that the B61\ncDNA has an open reading frame encoding 735 amino acid\nresidues with two hydrophobic segments, one at the amino\nterminus (amino acids 1 to 21) and the second (amino acids\n679 to 704) in the internal region. There are two distinct motifs\nin the extracellular region, leucine-rich repeats and ﬁbronectin\ntype III-like repeats and a small intracellular region consisting\nof only 30 amino acid residues without any known motifs. Due\nto the similarity to the previously identiﬁed neuronal leucine\nrich-repeat proteins NLRR1 to NLRR3 (4, 14, 28, 29), we\nnamed B61 NLRR4. These members are type I transmem-\n-Galactosidase staining of whole brains. For -galactosidase staining, brains\nwere ﬁxed in phosphate-buffered saline containing 2% paraformaldehyde, 0.2%\nglutaraldehyde, and 0.02% NP-40 and sectioned at 300-m thickness by vi-\nbratome. After washing, the sections were placed in staining buffer (44 mM\nHEPES [pH 7.4], 3 mM potassium ferricyanide, 15 mM NaCl, 1.3 mM MgCl2, 0.5\nmg/ml 5-bromo-4-chloro-3-indolyl-- D-galactopyranoside [X-Gal] in phosphate-\nbuffered saline) at room temperature.\nContextual fear-conditioning tests. For each experiment, a mouse was placed\nin the conditioning chamber and given a series of foot shocks alone (2-s duration,\n0.3 mA, 1 min apart). To test its memory, the mouse was placed in the same\nconditioning chamber and freezing responses were analyzed using Image FZC\n(O’Hara & Co.), with modiﬁed software available in the public-domain National\nInstitutes of Health Image program. In the experiment, a mouse was trained by\nthree foot shocks and then its behavior was tested 1, 4, or 7 days after training.\nCued fear-conditioning tests. For training, a mouse was placed in the condi-\ntioning chamber for 2 min before the onset of conditioned stimuli, a tone which\nlasted for 30 s at 10 kHz and 70 dB. The last 2 s of the conditioned stimulus was\npaired with am unconditioned stimulus, 0.3 mA of a continuous electric foot\nshock. After an additional 30 s in the chamber, the mouse was returned to its\nhome cage. Four and seven days after training, the mouse was placed in a\ndifferent chamber in which the same tone as the conditioned stimuli was applied\nfor 120 s after a 1-min habituation period (preconditioned stimulus).\nMorris water maze test. The water pool used in these experiments was 1 m in\ndiameter and made up of white polyvinyl chloride and the temperature of the\nwater was 27  1°C. For the hidden-platform task, an acrylic transparent plat-\nform (10 cm in diameter) was submerged 5 mm below the surface of water that\nwas made opaque by adding nontoxic odorless white paint. The location of the\nplatform was ﬁxed during the trials for each mouse. If it located the platform\nwithin 60 s, it was kept on it for 30 s. Mice that failed to ﬁnd the platform within\nbrane proteins with multiple leucine-rich repeats and a ﬁ-\nbronectin type III-like domain.\nAn NLRR4-homologous gene was found in the human ge-\nnome but not in Caenorhabditis elegans or Drosophila melano-\ngaster. A human cDNA homologous to NLRR4 encoded a type\nI transmembrane protein with 640 amino acids (Fig. 1a) with\nan overall homology of 64% at the amino acid level. NLRR4\nshowed 21% and 20% homology at the amino acid level to\nNLRR1 and NLRR3 (28, 29), respectively. Although there are\ntwo immunoglobulin C2 loops in NLRR1 and NLRR3, no such\nstructure was found in NLRR4 (Fig. 1b). Northern blot anal-\nysis revealed that NLRR4 mRNA was expressed in lung, heart,\nand ovary (Fig. 2b). In the adult brain, NLRR4 was strongly\nexpressed in the hippocampus and weakly in the cerebellum\n(Fig. 2a).\nTargeted disruption of the NLRR4 gene. To uncover the\nfunction of NLRR4 in vivo, we generated an NLRR4-deﬁcient\nmouse line. As shown in Fig. 3a, we replaced the exons of the\nNLRR4 gene with the -galactosidase gene and the neomycin\nresistance gene (neo). The deleted exon encoded the initiation\ncodon, the signal peptide, and the leucine-rich repeat domains,\nand the targeting vector was designed to express -galactosi-\ndase from NLRR4 regulatory elements. We isolated six inde-\npendent ES clones with homologous recombination (Fig. 3b),\nand chimeric male mice were generated from three of the\nclones. Heterozygous embryos from three independent chi-\nmeric mice exhibited the same -galactosidase expression pat-\ntern. Genotyping by Southern blotting showed successful tar-\ngeting of the NLRR4 gene (Fig. 3c).\nNLRR4 protein expression was not detected in NLRR4/\nembryos by ﬂow cytometric analysis of primary culture cells\nderived from wild-type and NLRR4/ embryos with an anti-\nNLRR4 monoclonal antibody (Fig. 3d). We genotyped 372\npups born from heterozygous matings to assess whether the\nexpected number of knockout and heterozygote pups was af-\nfected. The number of NLRR4/ to NLRR4/ to\nNLRR4/ was 83:196:93, as expected for unbiased Mende-\nlian frequencies (NLRR4/), and the mice grew with normal\nbody weights. Both males and females are fertile, and no in-\ncreased mortality was observed up to 1 year of age.\nExpression of NLRR4 in the adult brain and anatomy of the\nNLRR4/ hippocampus. Since NLRR4 mRNA was found in\nthe brain by Northern blotting analysis (Fig. 2), we precisely\nanalyzed the expression of NLRR4 in the brain. As the -ga-\nlactosidase gene was inserted in the exon of NLRR4, we per-\nformed -galactosidase staining of the adult brain of two-\nmonth-old heterozygous mice (NLRR4/) (Fig. 4a, b, c, and\ne). NLRR4 was expressed in the CA1 and CA3 regions and\ndentate gyrus of the hippocampus. In addition, NLRR4 was\nexpressed in layers V and VI in the neocortex and piriform\ncortex. It was also expressed in the inner granular layer and\nPurkinje cells of the cerebellum. The NLRR4/ brain was\nalso stained for -galactosidase activity, and there was no dif-\nference in the staining pattern between NLRR4/ and\nNLRR4/ mice (Fig. 4c, d, e, and f).\nWe performed histochemical staining to examine hippocam-\npal anatomy in wild-type and NLRR4/ mice. Hematoxylin-\neosin staining and immunohistochemical analysis of NeuN, a\nneuron-speciﬁc DNA binding protein, showed no difference\nbetween wild-type and NLRR4/ mice (Fig. 4g, h, i, and j).\nThese results indicated that the gross anatomy of hippocampus\nwas not affected by the NLRR4 mutation.\nDefective contextual fear memory in NLRR4/ mice. Since\nNLRR4 is expressed in the hippocampus, which plays a role in\nlearning and memory (3, 24), we considered the possibility that\nNLRR4 plays a role in memory formation. To address this, we\nexamined the associative emotional memory of the mutant\nmice using the contextual fear-conditioning test. If mice learn\nfear such as that resulting from an electric foot shock in a\nspatial context, they show some defensive responses, including\nfreezing behavior in the conditioning chamber (3).\nfear-conditioning test,\nWithout experiencing the electric shock, wild-type and\nNLRR4/ mice showed little of this behavior. Mice with\neither genotype spent the same time freezing in the context at\n24 h after the training, indicating that NLRR4/ mice are\nable to learn fear by contextual fear-conditioning. However,\nNLRR4/ mice froze signiﬁcantly less than wild-type mice 4\nand 7 days after the training (Fig. 5a). There were no signiﬁ-\ncant differences between the two genotypes in terms of noci-\nceptive sensitivity to an electrical foot shock (Fig. 5b). For the\ncontextual\nfear-eliciting experiences\nsuch as an electric foot shock are input into the amygdala and\nspatial information such as the conditioning chamber is sent to\nthe hippocampus. Both sets of information are consolidated\nand stored in a manner dependent on the hippocampus and\namygdala. As NLRR4/ mice showed normal freezing behav-\nior during the 1-day retention period, cellular consolidation in\nNLRR4/ mice was normal. However, as the freezing time\nwas signiﬁcantly reduced in NLRR4/ mice 4 days after the\ntraining, their memory was impaired after long retention delay.\nNormal tone-dependent cued fear memory in NLRR4/\nmice. As contextual fear-conditioning tasks are dependent on\nthe hippocampus and amygdala, we examined hippocampus-\nindependent fear memory to assess the function of amygdala in\nNLRR4/ mice. The auditory cued fear-conditioning test is\nan amygdala-dependent but hippocampus-independent task\n(19). Wild-type and NLRR4/ mice were exposed simulta-\nneously to a tone (conditioned stimulus) and an electric foot\nshock (unconditioned stimulus). After this learning process, we\nmeasured the freezing time when the tone was replayed in a\ndifferent chamber. Mice with either genotype did not show any\nfreezing behavior before presenting conditioned stimuli (pre-\nconditioned stimuli). In the presence of a conditioned stimu-\nlus, mice of either genotype showed the same level of freezing\ntime at both time points after the conditioning (Fig. 5c). No\nsigniﬁcant differences were observed in freezing time between\nthe two genotypes 4 and 7 days after the training, indicating\nthat NLRR4 is dispensable for tone information processing\nand fear memory is fully integrated into the amygdala without\nNLRR4 after long retention.\nDefective spatial memory in NLRR4/ mice. For the fear\nmemory tests, NLRR4/ mice showed defects in the hippocam-\npus-dependent memory (contextual fear-conditioning test), but\nnot the hippocampus-independent memory (tone fear-condition-\ning test). These results indicated that NLRR4/ mice are defec-\ntive in hippocampus-dependent fear memories. We then exam-\nined another type of hippocampus-dependent memory without\nthe experience of fear in NLRR4/ mice. We employed the\nhidden-platform Morris water maze test, which is a hippocampus-\ndependent learning task. Wild-type and NLRR4/ mice im-\nproved their performance during 5 days of training at 4 trials/day\nNormal hippocampal long-term potentiation in NLRR4/\nmutants. The ﬁnding that NLRR4/ mice showed normal\nshort-term memory but\nimpaired long-lasting memory\nprompted us to examine whether the physiological properties\nof the hippocampus were altered by the lack of NLRR4. We\nﬁrst examined the input-output relationship of synaptic trans-\nmission to assess possible changes in basal synaptic properties,\nusing hippocampal slices. AMPA receptor-mediated excitatory\npostsynaptic potentials in the CA1 region in response to vari-\nintensities were indistinguishable between\nous\nstimulus\nNLRR4/ and wild-type mice (Fig. 7a), indicating that the\nbasal synaptic efﬁcacy was not altered in NLRR4/ mice.\nrelease probability were intact\nWe then examined paired-pulse facilitation, which is a pre-\nsynaptic form of short-term plasticity and is sensitive to pre-\nsynaptic release probability (20). The paired-pulse facilitation\nobserved in slices from NLRR4/ mice did not signiﬁcantly\ndiffer from that of wild-type mice at any interstimulus interval\nexamined (Fig. 7b), indicating that presynaptic properties in-\ncluding\nin\nNLRR4/ mice. Furthermore, we analyzed long-term synap-\ntic plasticity and found that a single high-frequency stimulus\ntrain (100 Hz/s) produced long-term potentiation in mutant\nslices that was indistinguishable from that of wild-type slices\n(Fig. 7c and d), which is consistent with the results that\nNLRR4/ mice performed normally hippocampus-depen-\ndent memory tasks 24 h after the training. Since there was a\ntendency of decreased long-term potentiation 60 min after the\ninduction in NLRR4/ mice, we also followed the time\ncourse of long-term potentiation for another hour, but found\nthat long-term potentiation 120 min after the induction was\nalso normal (Fig. 7e and f). Taken together, these results\nindicate that NLRR4 is not required for basal synaptic trans-\nmission or short-term and the early phase of long-term plas-\nticity at the Schaffer collateral-CA1 synapse in the hippocampus.\nDISCUSSION\nWe isolated a cDNA encoding a new member of the NLRR\nfamily, a type I transmembrane protein with leucine-rich re-\npeats and a ﬁbronectin type III repeat. Leucine-rich repeats\nconsist of multiple copies of the conserved sequence. Proteins\nwith leucine-rich repeats mediate homophilic or heterophilic\nprotein-protein interactions. Slits function as a ligand for the\nRobo receptors through its leucine-rich repeat (2, 11), and\nNgR and Trks are receptors for Nogo and neurotrophic factors\n(6, 17), respectively. CAPS and connectin function as cell rec-\nognition molecules in neuromuscular development via their\nhomophilic interaction (23, 27). We therefore tested a ho-\nmophilic interaction of NLRR4 protein. However, cell aggre-\ngation assays using NLRR4-expressing cells failed to show any\ninteraction (data not shown), suggesting that NLRR4 may be\ninvolved in cell-cell interactions as a component or it may be a\nreceptor or a ligand for an unknown molecule.\nNLRR4-deﬁcient mice generated by inserting the -galac-\ntosidase cDNA into the NLRR4 gene locus developed nor-\nmally and exhibited no anatomical abnormalities. The NLRR4\nexpression revealed by -galactosidase activity indicates that\nNLRR4 is expressed in the hippocampus, layers V and VI in\nthe cortex and piriform cortex, the internal granule layer, and\nPurkinje cell in the cerebellum. The hippocampus plays a role\nin the initial storage and the cellular consolidation of spatial\ninformation (22, 26). Since NLRR4 is highly expressed in the\nhippocampus, it is possible that NLRR4 is involved in learning\nand/or memory. In fact, we found that\nthe memory of\nNLRR4/ mice was severely impaired for two learning tests\nthat are dependent on the hippocampus; the contextual fear-\nconditioning and hidden-platform Morris water maze tests.\nIn the contextual fear-conditioning test, NLRR4/ mice\nwere able to learn normally as shown by the normal freezing\nbehavior 1 day after training. However, NLRR4/ mice failed\nto retain the memory longer as their freezing behavior was\nsigniﬁcantly reduced 4 and 7 days after training. We obtained\nsimilar results in the hidden-platform Morris water maze\ntest. Thus,\nin the hippocampus-dependent behavioral tests\nNLRR4/ mice are able to learn and retain memories nor-\nmally memory 1 day after training. Consistently, NLRR4/\nmice showed normal long-term potentiation in the hippocam-\npus, indicating that initial memory and cellular consolidation\nare normal in NLRR4/ mice. Contextual and spatial mem-\nories are known to be retained in the hippocampus for several\ndays and long-lasting memory is permanently stored in the\ncortex (7, 8). As memory retention 4 days after training was\nseverely impaired in NLRR4/ mutant mice, it is likely that\nNLRR4 is required for long-lasting memory.\nIn contrast to hippocampus-dependent memories, NLRR4/\nmice showed normal cued fear memory in which a tone was\npaired with an electric foot shock. This task depends on the\namygdala but is independent of the hippocampus. It is known\nthat hippocampus lesion results in impaired spatial memory\nbut normal memory in the cued fear-conditioning tests (19).\nLikewise, the hippocampal CA1 region-speciﬁc knockout of\nthe NR1 gene encoding an essential subunit of the N-methyl-\nD-aspartate receptor results in amnesia of contextual fear\nmemory without affecting cued fear memory (26). Axon of\nCA1 pyramidal cells mainly extend to the layer V in entorhinal\ncortex and NLRR4 is expressed in CA1, CA3, and dentate\ngyrus in the hippocampus and in layers V and VI in the cortex.\nThese results collectively suggest that the impaired retention of\nmemories in NLRR4/ mice depends on the hippocampal\nfunction but is independent of the amygdala.\nAmnesia in NLRR4/ mice is similar to that in CaMKII\nheterozygous mice, which exhibit severely impaired hippocam-\npus-dependent\nlong-lasting memory, while the molecular\nmechanism how CaMKII is involved in long-lasting memory\nstill remains to be elucidated, there may be a link between\nNLRR and CaMKII. CREB is required for the cellular con-\nsolidation of long-term memory, indicating that a protein(s)\nregulated by CREB plays a role in long-term memory (1, 5,\n18). Therefore, the impaired memory in NLRR4/ mice\nmight be due to an alteration in this pathway. As NLRR4 is a\ntransmembrane protein with protein-protein interaction mo-\ntifs, it is tempting to speculate that NLRR4 may generate an\nintracellular signal that affects the CREB pathway in the hip-\npocampus. It will be of interest to ﬁnd a molecule that interacts\nwith NLRR4.\nWe thank S. Kida for helpful discussion and critical reading of the\nmanuscript and T. Fukuda, T. Watanabe, H. Kawano, and T. Sato for\ntechnical advice. We are also grateful to T. Akiyama and his staff for\ntheir help for behavioral tests and to Y. Iwakura for providing us with\na mouse genomic DNA library and the LacZ plasmid.\nThis work was supported in part by Grants-in-Aid for Scientiﬁc\nResearch, the 21st Century COE program, the Center for Biosignaling\nand Integrated Brain Medical Science, and Special Coordination\nFunds for Promoting Science and Technology from the Ministry of\nEducation, Culture, Sports and Technology, by RISTEX and CREST\nof the Japan Science and Technology Agency, and by the Inoue Foun-\ndation for Science.\n2. Battye, R., A. Stevens, R. L. Perry, and J. R. Jacobs. 2001. Repellent signal-\ning by Slit requires the leucine-rich repeats. J. Neurosci. 21:4290–4298.\n3. Best, P. J., A. M. White, and A. Minai. 2001. Spatial processing in the brain:\nthe activity of hippocampal place cells. Annu. Rev. Neurosci. 24:459–486.\n4. Bormann, P., L. W. Roth, D. Andel, M. Ackermann, and E. Reinhard. 1999.\nzfNLRR, a novel leucine-rich repeat protein is preferentially expressed dur-\ning regeneration in zebraﬁsh. Mol. Cell. Neurosci. 13:167–179.",
         "Takayoshi Bando,K. Sekine,Shizuka Kobayashi,A. M. Watabe,Armin Rump,Minoru Tanaka,Yoshikuni Suda,S. Kato,Y. Morikawa,T. Manabe,A. Miyajima",
         null
        ],
        [
         "12",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "Abstract— We present an encoding framework which exploits\nsemantics for video content delivery. The video content is orga-\nnized based on the idea of main content message. In the work\nreported in this paper, the main content message is extracted from\nthe video data through semantic video analysis, an application-\ndependent process that separates relevant information from non\nrelevant information. We use here semantic analysis and the\ncorresponding content annotation under a new perspective: the\nresults of the analysis are exploited for object-based encoders,\nsuch as MPEG-4, as well as for frame-based encoders, such as\nMPEG-1. Moreover, the use of MPEG-7 content descriptors in\nconjunction with the video is used for improving content visual-\nization for narrow channels and devices with limited capabilities.\nFinally, we analyze and evaluate the impact of semantic video\nanalysis in video encoding and show that the use of semantic\nvideo analysis prior to encoding sensibly reduces the bandwidth\nrequirements compared to traditional encoders not only for an\nobject-based encoder but also for a frame-based encoder.",
         "I. INTRODUCTION\nThe diffusion of network appliances such as cellular phones,\npersonal digital assistants, and hand-held computers creates a\nnew challenge for content delivery: how to adapt the media\ntransmission to various device capabilities, network charac-\nteristics, and user preferences [1], [2], [3]. Each device is\ncharacterized by certain display capabilities and processing\npower. Moreover, such appliances are connected through dif-\nferent kinds of networks with diverse bandwidths. Finally,\nusers with different preferences access the same multimedia\ncontent. Therefore there exists a need to personalize the way\nmedia content is delivered to the end user. In addition to the\nabove, recent devices, such as digital radio receivers, and new\napplications, such as intelligent visual surveillance, require\nnovel forms of video analysis for content adaptation and sum-\nmarization. Digital radios allow for the display of additional\ninformation alongside the traditional audio stream to enrich the\naudio content. For instance, digital audio broadcasting (DAB)\nallocates 128 Kb/s to streaming audio, whereas 8Kb/s can\nbe used to send additional information, such as visual data\n[4]. Moreover, the growth of video surveillance systems poses\nchallenging problems for the automatic analysis, interpretation\nand indexing of video data as well as for selective content\nﬁltering for privacy preservation. Finally, the instantaneous\nindexing of video content is also an desirable feature for sports\nbroadcasting [5].\nTo cope with these challenges, video content needs to be\nautomatically analyzed and adapted to the needs of the speciﬁc\napplication, to the capabilities of the connected terminal and\nnetwork, and to the preferences of the user. Three main\nstrategies for adaptive content delivery have been proposed\nthroughout the literature, namely Info Pyramid, scalable cod-\ning and transcoding. The work presented in this paper aims\nto go beyond traditional adaptation techniques. We focus on\nsemantic encoding by looking to exploit video analysis prior\nto encoding (Figure 1). Speciﬁcally, we use semantic video\nanalysis to extract relevant areas of a video. These areas are\nencoded at a higher level of quality or summarized in textual\nform. The idea behind this approach is to organize the content\nso that a particular network or device does not inhibit the main\ncontent message. The main content message is dependent on\nthe speciﬁc application. In particular, for applications such as\nvideo surveillance and sport video the main content message\nis deﬁned based on motion information.\nThe contribution of this paper is twofold. On the one\nhand, a framework for adaptive video delivery is deﬁned and\nimplemented based on video objects and on their associated\nmetadata. On the other hand, two new modalities of video\ndelivery are proposed in such a framework. The ﬁrst modality\ncombines semantic analysis with a traditional frame-based\nvideo encoder. The second modality uses metadata to efﬁ-\nciently encode the main content message. In particular, the\nuse of metadata enables not only to make the content more\nsearchable, but also to improve visualization and to preserve\nprivacy in video-based applications.\nThe paper is organized as follows. Section II is an overview\nof existing adaptation techniques. Section III presents the\nalgorithm for extracting the main content message, the frame-\nwork for adaptive video delivery and automatic description\nusing semantic video analysis. Section IV discusses quality\nassessment issues, whereas experimental results are presented\nin Section V. Finally, Section VI concludes the paper.",
         "of semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\nlocators of the foreground objects,\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.",
         "",
         "of semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\nlocators of the foreground objects,\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.",
         "Abstract— We present an encoding framework which exploits\nsemantics for video content delivery. The video content is orga-\nnized based on the idea of main content message. In the work\nreported in this paper, the main content message is extracted from\nthe video data through semantic video analysis, an application-\ndependent process that separates relevant information from non\nrelevant information. We use here semantic analysis and the\ncorresponding content annotation under a new perspective: the\nresults of the analysis are exploited for object-based encoders,\nsuch as MPEG-4, as well as for frame-based encoders, such as\nMPEG-1. Moreover, the use of MPEG-7 content descriptors in\nconjunction with the video is used for improving content visual-\nization for narrow channels and devices with limited capabilities.\nFinally, we analyze and evaluate the impact of semantic video\nanalysis in video encoding and show that the use of semantic\nvideo analysis prior to encoding sensibly reduces the bandwidth\nrequirements compared to traditional encoders not only for an\nobject-based encoder but also for a frame-based encoder.\nI. INTRODUCTION\nThe diffusion of network appliances such as cellular phones,\npersonal digital assistants, and hand-held computers creates a\nnew challenge for content delivery: how to adapt the media\ntransmission to various device capabilities, network charac-\nteristics, and user preferences [1], [2], [3]. Each device is\ncharacterized by certain display capabilities and processing\npower. Moreover, such appliances are connected through dif-\nferent kinds of networks with diverse bandwidths. Finally,\nusers with different preferences access the same multimedia\ncontent. Therefore there exists a need to personalize the way\nmedia content is delivered to the end user. In addition to the\nabove, recent devices, such as digital radio receivers, and new\napplications, such as intelligent visual surveillance, require\nnovel forms of video analysis for content adaptation and sum-\nmarization. Digital radios allow for the display of additional\ninformation alongside the traditional audio stream to enrich the\naudio content. For instance, digital audio broadcasting (DAB)\nallocates 128 Kb/s to streaming audio, whereas 8Kb/s can\nbe used to send additional information, such as visual data\n[4]. Moreover, the growth of video surveillance systems poses\nchallenging problems for the automatic analysis, interpretation\nand indexing of video data as well as for selective content\nﬁltering for privacy preservation. Finally, the instantaneous\nindexing of video content is also an desirable feature for sports\nbroadcasting [5].\nTo cope with these challenges, video content needs to be\nautomatically analyzed and adapted to the needs of the speciﬁc\napplication, to the capabilities of the connected terminal and\nnetwork, and to the preferences of the user. Three main\nstrategies for adaptive content delivery have been proposed\nthroughout the literature, namely Info Pyramid, scalable cod-\ning and transcoding. The work presented in this paper aims\nto go beyond traditional adaptation techniques. We focus on\nsemantic encoding by looking to exploit video analysis prior\nto encoding (Figure 1). Speciﬁcally, we use semantic video\nanalysis to extract relevant areas of a video. These areas are\nencoded at a higher level of quality or summarized in textual\nform. The idea behind this approach is to organize the content\nso that a particular network or device does not inhibit the main\ncontent message. The main content message is dependent on\nthe speciﬁc application. In particular, for applications such as\nvideo surveillance and sport video the main content message\nis deﬁned based on motion information.\nThe contribution of this paper is twofold. On the one\nhand, a framework for adaptive video delivery is deﬁned and\nimplemented based on video objects and on their associated\nmetadata. On the other hand, two new modalities of video\ndelivery are proposed in such a framework. The ﬁrst modality\ncombines semantic analysis with a traditional frame-based\nvideo encoder. The second modality uses metadata to efﬁ-\nciently encode the main content message. In particular, the\nuse of metadata enables not only to make the content more\nsearchable, but also to improve visualization and to preserve\nprivacy in video-based applications.\nThe paper is organized as follows. Section II is an overview\nof existing adaptation techniques. Section III presents the\nalgorithm for extracting the main content message, the frame-\nwork for adaptive video delivery and automatic description\nusing semantic video analysis. Section IV discusses quality\nassessment issues, whereas experimental results are presented\nin Section V. Finally, Section VI concludes the paper.\nII. BACKGROUND\nThree main approaches have been presented in the literature\nto provide adaptive content delivery, namely Info Pyramid,\nscalable coding and transcoding. Info Pyramid provides a\ngeneral framework for managing and manipulating media\nobjects [6], [7]. Info Pyramid manages different versions, or\nvariations, of media objects with different modalities (e.g.,\nvideo, image, text, and audio) and ﬁdelities (summarized, com-\npressed, and scaled variations). Moreover, it deﬁnes methods\nfor manipulating, translating, transcoding, and generating the\ncontent. When a client device requests a multimedia document,\nthe server selects and delivers the most appropriate variation.\nThe selection is made based on network characteristics and\nterminal capabilities, such as display size, frame rate, color\ndepth and storage capacity.\nAs opposed to Info Pyramid, scalable coding processes\nmultimedia content only once. Lower qualities, lower spatial\nand temporal resolutions of the same content are then obtained\nby truncating certain layers or bits from the original stream [8].\nBasic modes of video scalability include quality scalability,\nspatial scalability, temporal scalability, and frequency scala-\nbility. Combinations of these basic modes are also possible.\nQuality scalability is deﬁned as the representation of a video\nsequence with varying accuracies in the color patterns. This\nis typically obtained by quantizing the color values with\nincreasingly ﬁner quantization step sizes. Spatial scalability\nis the representation of the same video in varying spatial\nresolutions. Temporal scalability is the representation of the\nsame video at varying temporal resolutions or frame rates.\nFrequency scalability includes different frequency components\nin each layer, with the base layer containing low-frequency\ncomponents and the other layers containing increasingly high-\nfrequency components. Such decomposition can be achieved\nvia frequency transforms like the DCT or wavelet transforms.\nFinally, the basic scalability schemes can be combined to reach\nﬁne-granularity scalability, such as in MPEG–4 FGS [9].\nThe various scalable coding methods introduced previously\nperform the same operation over the entire video frame. In\nobject-based temporal scalability (OTS), the frame rate of\nforeground objects is enhanced so that it has a smoother\nmotion than the background.\nVideo transcoding is the process of converting a compressed\nvideo signal into another compressed signal with different\nproperties. Early solutions to video transcoding determine the\noutput format based on network and appliance constraints,\nindependently of the semantics in the content (content-blind\ntranscoding). Content-blind transcoding strategies include spa-\ntial resolution reduction, temporal resolution reduction, and\nbit-rate reduction [10]. Recent transcoding techniques make\nuse of semantics to minimize the degradation of important\nimage regions [11], [12]. In [13], optimal quantization pa-\nrameters and frame skip are determined for each video object\nindividually. The bit-rate budget for each object is allocated by\na difﬁculty hint, a weight indicating the relative encoding com-\nplexity of each object. Frame skip is controlled by a shape hint,\nwhich measures the difference between two consecutive shapes\nto determine whether an object can be temporally downsam-\npled without visible composition problems. Key objects are\nselected based on motion activity and on bit complexity.\nThe transcoding strategies described thus far are referred\nto as intramedia transcoding strategies and do not change the\nmedia nature of the input signal. On the other hand, intermedia\ntranscoding, or transmoding,\nis the process of converting\nthe media input\ninto another media format. Examples of\nintermedia transcoding include speech-to-text [14] and video-\nto-text [15] translation. Both the intramedia and the intermedia\nadaptation concepts are used in this paper for video encoding,\nas described in the following section.\nIII. ADAPTIVE CONTENT DELIVERY AND DESCRIPTION\nUSING SEMANTICS\nThe proposed framework for adaptive video delivery and\nautomatic description uses video content analysis and semantic\npre-ﬁltering prior to encoding (Figure 1) in order to improve\nthe perceived content quality and to provide additional func-\ntionalities, such as privacy preservation and automatic video\nindexing. Semantic video analysis and semantic encoding are\ndescribed next.\nA. Semantic video analysis\nSemantic video analysis is used to extract the main content\nmessage from the video. The semantics to be included in\nthe analysis is dependent on the speciﬁc application. In the\nfollowing, we discuss possible semantics and, in particular,\nwe describe the use of motion as semantics.\nSemantic video analysis refers to a human abstraction and\nuses a priori information to translate the semantics into rules.\nThe rules are then applied through an algorithm. Examples\nof semantic video analysis based on a priori information are\ntemplate matching, extraction of captions and text, face de-\ntection, and moving object segmentation. Template matching\nis used to implement the semantics when the shape objects\nwe want to segment is known a priori. In this case, which\nincludes in particular the detection of captions and text, the\nextraction method searches for speciﬁc object features in terms\nof geometry. For segmenting faces of people, color-based\nsegmentation can be used [16]. The face detection task consists\nin ﬁnding the pixels whose spectral characteristics lie in a\nspeciﬁc region in the chromaticity diagram. For extracting\nmoving objects, motion information can be used as semantics.\nSeveral applications, such as sport broadcasting and video\nsurveillance, deal with segmenting moving objects.\nA typical tool used to tackle the problem of object segmen-\ntation based on motion is change detection. Different change\ndetection techniques can be employed for moving camera\nand static camera conditions. If the camera moves, change\ndetection aims at recognizing coherent and incoherent moving\nareas. The former correspond to background areas, the latter\nto video objects. If the camera is static, the goal of change\ndetection is to recognize moving objects (foreground) and the\nstatic background. The semantic analysis we use addresses\nthe static camera problem and is applicable in the case of a\nmoving camera after global motion compensation. The change\ndetector decides whether in each pixel position the foreground\nsignal corresponding to an object is present. This decision\nis taken by thresholding the frame difference between the\ncurrent frame and a frame representing the background. The\nframe representing the background is dynamically generated\nbased on temporal information [17]. The thresholding aims\nat discarding the effect of the camera noise after frame\ndifferencing. A locally adaptive threshold, τ (i, j), is used that\nmodels the noise statistics and applies a signiﬁcance test. To\nthis end, we want to determine the probability that frame\ndifference at a given position (i, j) is due to noise, and not to\nother causes. Let us suppose that there is no moving object in\nthe frame difference. We refer to this hypothesis as the null\nhypothesis, H0. Let g(i, j) be the sum of the absolute values\nof the frame difference in an observation window of q pixels\naround (i, j). Moreover, let us assume that the camera noise is\nadditive and follows a Gaussian distribution with variance σ.\nGiven H0, the conditional pdf of the frame difference follows\na χ2\nq distribution with q degrees of freedom deﬁned by\nf\nH0\ng(i, j)\n|\n(1)\n\nwhere Γ(\n) is the Gamma function, that can be evaluated as\n·\nΓ(x+ 1) = xΓ(x), and Γ(1/2) = √π. To obtain a good trade-\noff between robustness to noise and accuracy in the detection\n5 window centered in (i, j)). It is now\nwe choose q = 25 (5\npossible to derive the signiﬁcance test as\ng(i, j)\n{\nH0}\nτ (i, j)\n|\n≥\nΓ(q/2, g(i, j)2/2σ2)\nΓ(q/2)\n(2)\nWhen this probability is smaller than a certain signiﬁcance\nlevel, α, we consider that H0 is not satisﬁed at the pixel\nposition (i, j). Therefore we label that pixel as belonging to a\nmoving object. The signiﬁcance level α is a stable parameter\nthat does not need manual tuning along a sequence or for\ndifferent sequences. Experimental results indicate that valid\nvalues fall in the range from 10−2 to 10−6.\nThe change detection process produces the segmentation of\nthe moving objects from the background (Figure 2) and, cou-\npled with video object tracking [18], enables the subsequent\nextraction of object metadata, as described in the following\nsection.\nB. Semantic encoding\nThe decomposition computed with semantic video analysis\nis used with an object-based encoder as well as with a\ntraditional frame-based encoder. We will refer to the former\ncase as object-based encoding and to the latter as frame-\nbased encoding. Furthermore, metadata are used to efﬁciently\nencode relevant information and to enhance relevant part of a\nlow-quality coded video. These approaches are referred to as\nmetadata-based encoding and metadata-enhanced encoding,\nrespectively. Relevant examples of the modalities presented\nin this section are illustrated in Figure 3. The analysis and\nevaluation of the different approaches in terms of results and\nbandwidth requirements are presented in Section V.\n1) Object-based encoding: With object-based encoding, the\nencoder needs to support the coding of individual video objects\n(e.g., MPEG–4 object-based). Each video object is assigned to\na distinct object class, according to its importance in the scene.\nThe encoding quality can be set depending on the object class:\nthe higher the relevance, the higher the encoding quality. One\nadvantage of this approach is the possibility of controlling\nthe sequencing of objects: video objects may be encoded\nwith different degree of compression, thus allowing better\ngranularity for the areas in the video that are of more interest to\nthe viewer. Moreover, objects may be decoded in their order\nof priority, and the relevant content can be viewed without\nhaving to reconstruct the entire image (network limitations).\nAnother advantage is the possibility of using a simpliﬁed\nbackground (appliance limitation), so as to enhance the rel-\nevant objects. Using a simpliﬁed background aims at taking\nadvantage of the task-oriented behavior of the human visual\nsystem for improving compression ratios. Recent work on\nfoveation [19] demonstrated that using nonlinear integration\nof low-level visual cues mimicking the processing in primate\noccipital and posterior parietal cortex allows one to sensibly\nincrease compression ratios. Moreover, the work reported in\n[20] demonstrated that an overall increase in image quality\ncan be obtained when the increase in quality of the relevant\nareas of an image more than compensates for the decrease in\nquality of the image background.\n2) Semantic frame-based encoding: The semantic frame-\nbased encoding mode exploits semantics in a traditional frame-\nbased encoding framework (e.g., MPEG–1). The use of the\ndecomposition of the scene into meaningful objects prior\nto encoding, referred here as semantic pre-ﬁltering, helps\nsupport low bandwidth transmission. The areas belonging to\nthe foreground class, or semantic objects, are used as region\nof interest. The areas not included in the region of interest\nmay either be eliminated, that is set to a constant value, or\nlowered in importance by using a low-pass ﬁlter. The latter\nsolution simpliﬁes the information in the background, while\nstill retaining essential contextual information. An example\nof this solution is reported in Fig. 4 (a). On the other hand,\nﬁltering the entire image inhibits the main content message\nFig. 4 (b). Another way to take into account less relevant\nportions of an image before coding is to take advantage of the\nspeciﬁcs of the coding algorithm. In the case of block-based\ncoding, each background macroblock can be replaced by its\nDC value. Semantic frame-based encoding mimics the way\nhumans perceive visual information and allows for a reduction\nof information to be coded.\n3) Metadata-based and metadata-enhanced encoding: A\nfurther processing of the video content is performed to cope\nwith limited device or network capabilities as well as to\nautomatically generate metadata. Such processing transforms\nthe foreground objects extracted through semantic analysis\ninto quantitative descriptors and enables video annotation.\nVideo annotation is desirable for applications such as video\nsurveillance, where terabytes of data are produced and need\nto be searched quickly. Moreover, the descriptors can be\ntransmitted instead of the video content itself and superim-\nposed by the terminal on a still background. For example, an\nobject identiﬁer and a shape descriptor are used in [21]. The\nobject identiﬁer is a unique numerical identiﬁer describing\nthe spatial location of each object in the scene. The shape\ndescriptor is used to represent the shape of an object, ranging\nfrom a bounding box to a polygonal representation with a\ndifferent number of vertices (Figure 3(c)). This approach is\nuseful to preserve privacy in video surveillance applications\nas well as to reduce bandwidth requirements under critical\nnetwork conditions. A progressive representation is used where\nthe number of vertices corresponding to the best resolution\nis computed, and any number of vertices smaller that this\nmaximum can be used according to the requirements of the\napplication. In addition to the above, other features such as\ncolor and texture descriptors may be added in the description\nprocess. The choice of these additional features depends on\nthe application at hand.\nIn addition to the above, the descriptors can be transmitted\nalong with the video itself and used for rendering the video\ncontent. This solution, consisting in a mixture of video-based\nand text-based modalities, is here referred toe as metadata-\nenhanced encoding. Using metadata-enhanced encoding, con-\ntent descriptors help enhance parts of the video that are hidden\nor difﬁcult to perceive due to heavy compression. In this case,\nthe video itself is the background and the descriptors highlight\nrelevant portions of the data. One example is the ball in a\nfootball match for transmission to a PDA or a mobile phone,\nas shown in Figure 3(e).\nis a difﬁcult\nPerceptual video quality assessment\ntask\nalready when dealing with traditional coders [22]. When\ndealing with object-based coders,\nthe task becomes even\nmore challenging. For this reason, we use a combination of\nsubjective and objective evaluation techniques to compare the\nperformance of the different encoding modalities. Subjective\nevaluation includes the visual comparison of frames and frame\ndetails. This analysis is performed at different bitrates and\nat different frame resolutions. Objective evaluation includes\ntemporal signal-to-noise ratio analysis and the analysis of rate-\ndistortion curves.\nA. Semantic peak signal-to-noise ratio\nTraditional peak signal-to-noise ratio (PSNR) analysis uni-\nformly weights the contribution of each pixel in an image\nwhen computing the mean squared error (MSE). This analysis\ngives the same importance to relevant as well as less relevant\nareas of an image. To account for the way humans perceive\nvisual\ninformation, different areas of an image, or object\nclasses, should be considered [11]. We take into account\nobject classes through a distortion measure, the semantic mean\nsquared error, SMSE, deﬁned as:\nwk ·\n(3)\nP\nXk=1\nwhere N is the number of object classes and wk the weight of\nclass k. Class weights are chosen depending on the semantics,\nN\ni=1 wk = 1. The mean\nwith wk ≥\nsquared error of each class, MSEk, can be written as\nk = 1, . . . , N and\n∀\n|\nCk|\n|\nCk| X(i,j)∈Ck\nwhere Ck is the set of pixels belonging to the object class k\nand\nis its cardinality. The class membership of each pixel\n(i, j) is deﬁned by semantic video analysis. The error d(i, j)\nbetween the original image IO and the distorted image ID in\nEq.(4) is the pixel-wise color distance. The color distance is\ncomputed in the 1976 CIE Lab color space in order to consider\nperceptually uniform color distances with the Euclidean norm\nand is expressed as:\nevaluation metric,\nSPSNR, is the following:\nthe semantic peak signal-to-noise ratio,\nSPSNR = 10 log10\nV 2\nM\nSMSE !\nwhere VM is the maximum peak-to-peak value of the color\nrange. When the object classes are foreground and back-\nground, then N = 2 in Eq (3). If we denote with wf the\nforeground weight, then SPSNR\nPSNR when wf = 0.5.\nThe larger wf , the more important the contribution of the\nforeground. When wf = 1,\nthen the foreground only is\nconsidered in the evaluation of the peak signal-to-noise ratio.\nAn illustration of the impact of wf in the distortion measure\nis shown in in Fig. 5. The ﬁgure presents a comparison of the\nIllustration of the impact of wf in the distortion measure: average\nFig. 5.\nSPSNR vs. foreground weight for Hall monitor sequence. The ﬁve labels\ncorrespond to the following sequence types: (1) coded original; (2) tempo-\nrally down-sampled; (3) spatially down-sampled; (4) static background; (5)\nsimpliﬁed background. Content-blind coding methods (1)-(3) decrease their\nperformance when the foreground is given more importance. Methods based\non semantic, (4) and (5), increase their performance when the foreground is\ngiven more importance\naverage SPSNR of the sequence Hall Monitor for the different\nencoding modalities described in Section III-B as function of\nwf . The value of wf to be used is estimated as described in\nthe next section.\nB. Determination of the foreground weight\nSubjective performance evaluation experiments have been\nperformed to estimate the foreground weight leading to the\nclosest match between SPSNR prediction and human judg-\nment. Twenty non-expert observers of different ages and\nbackgrounds have been presented a series of video sequences\naccording to ITU-T Recommendation P.910, Absolute Cate-\ngory Rating [23]. The evaluation has been carried out using\nthe MPEG–4 test sequences Akiyo, Hall Monitor, Children,\nand Coastguard. Video sequences have been generated using\nthe encoding strategies described in Section III-B, at different\nbitrates, and rated by the observers on a scale ranging from 0\n(bad) to 100 (excellent). This range of values was presented\nto the observers in a training phase.\nThe foreground weight, wf , is determined for each test\nsequence by maximizing the Pearson correlation [24] between\nSPSNR and subjective results. The results are summarized in\nTable I. For the sequence Akiyo, where the foreground covers\na large area of each frame and the background is simple, the\nthe observers focused mostly on foreground, thus leading to a\nvalue of wf = 0.97. For Hall Monitor, whose background is\nmore complex and objects are smaller, the foreground attracted\nslightly more the attention than the background (wf = 0.55).\nThe sequence Children has a very complex and colored back-\nground that attracted the observers’ attention, thus resulting\nin foreground and background being equally weighted (wf =\n0.5). The sequence Coastguard contains camera motion that\nprevented the observer from focusing on background steadily,\neven though the background is quite complex. In this case, the\nresulting foreground weight is wf = 0.7. In general, results\nconﬁrm that large moving objects and complex background\ntend to attract user’s attention. Based on the data collected with\nsubjective experiments, it is possible to predict the foreground\nweight based on the following formula:\nand\nCb|\n|\nCb|\n|\nCf |\n/(\n|\nwhere r represents the portion of the image occupied by\nforeground pixels, expressed as r =\n), with\nCf |\n|\nrepresenting the number of foreground and\nCf |\n|\nbackground pixels, respectively. The background complexity\nis taken into account with σb, the standard deviation of the\nluminance of background pixels. The presence of camera\nmotion is considered with the term v: v = 1 for moving\ncamera, and v = 0 otherwise. α, β, γ, and δ are constants\nwhose values are determined based on the results of the\nsubjective experiments and are the following: α = 5.7, β =\n0.108, γ = 0.2 and δ = 0.01. The ﬁnal value of wf is the\naverage of the foreground weights over the sequence.\nIn addition to semantic weight, Table I provides information\nabout accuracy, monotonicity and consistency of the SPSNR\nmetric. Accuracy is given by Pearson linear correlation coefﬁ-\ncient rp, monotonicity by Spearman rank-order correlation co-\nefﬁcient rs, and consistency by outliers ratio ro [24]. Pearson\ncorrelation of PSNR, rp(0.5), is given for comparison. Pearson\ncorrelation rp and Spearman correlation rs are close to 1 for\nall sequences. Thus, accuracy and monotonicity of SPSNR are\nhigh. Outliers ratio ro is around 10%, thus consistency of the\nmetric is good as well. Note that using semantics improves\naccuracy by up to 8% (Akiyo), as compared to PSNR.\nV. EXPERIMENTAL RESULTS\nIn this section, experimental results of the proposed seman-\ntic video encoding and annotation framework with standard\ntest sequences are presented. The results illustrate the impact\nof semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\nlocators of the foreground objects,\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.",
         "A. Cavallaro,Olivier Steiger,T. Ebrahimi",
         null
        ],
        [
         "13",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "Of the nasopharyngeal cultures recovered from 942 day care center (DCC) attendees in Lisbon, Portugal, 591\n(62%) yielded Streptococcus pneumoniae during a surveillance performed in February and March of 1999. Forty\npercent of the isolates were resistant to one or more antimicrobial agents. In particular, 2% were penicillin\nresistant and 20% had intermediate penicillin resistance. Multidrug resistance to macrolides, lincosamides,\nand tetracycline was the most frequent antibiotype (17% of all isolates). Serotyping and molecular typing by\npulsed-ﬁeld gel electrophoresis were performed for 202 out of 237 drug-resistant pneumococci (DRPn). The\nmost frequent serotypes were 6B (26%), 14 (22%), 19F (16%), 23F (10%), and nontypeable (12%). The majority\n(67%) of the DRPn strains were representatives of nine international clones included in the Pneumococcal\nMolecular Epidemiology Network; eight of them had been detected in previous studies. Fourteen novel clones\nwere identiﬁed, corresponding to 26% of the DRPn strains. The remaining 7% of the strains were local clones\ndetected in our previous studies. Comparison with studies conducted since 1996 in Portuguese DCCs identiﬁed\nseveral trends: (i) the rate of DRPn frequency has ﬂuctuated between 40 and 50%; (ii) the serotypes most\nfrequently recovered have remained the same; (iii) nontypeable strains appear to be increasing in frequency;\nand (iv) a clone of serotype 33F emerged in 1999. Together, our observations highlight that the nasopharynxes\nof children in DCCs are a melting pot of successful DRPn clones that are important to study and monitor if\nwe aim to gain a better understanding on the epidemiology of this pathogen.",
         "The ecological niche of Streptococcus pneumoniae is the na-\nsopharynxes of humans. Although pneumococci are widely\nspread in the community, carriage rates are particularly high in\npreschool children attending day care centers (DCCs) (24, 45).\nS. pneumoniae can cause a spectrum of diseases including\ninvasive disease, such as meningitis, bloodstream infections,\nand pneumonia, as well as infections of the upper respiratory\ntract. S. pneumoniae is a major cause of morbidity and mor-\ntality in young children (6). In the last decade an increasing\nproportion of pneumococci that are not susceptible to penicil-\nlin has been observed, reaching up to 50% in some areas of the\nworld (1, 10, 16, 37, 39, 43).\nIn 1996 our group began yearly surveillance of the nasopha-\nryngeal carriage of pneumococci and other bacterial respira-\ntory tract pathogens in DCCs located in the Lisbon area of\nPortugal. Since 1996, a total of 23 DCCs and over 1,800 chil-\ndren participated in this study. Serotypes, antibiotypes, and\nmolecular types of the majority of drug-resistant S. pneu-\nmoniae isolates were determined during these annual surveil-\nlances (8, 9, 34, 35), allowing one to obtain a view of temporal\nchanges of drug-resistant pneumococci (DRPn) inhabiting the\nnasopharyngeal ﬂora, an important ecological reservoir which\nmay be a major source of drug-resistant strains causing pedi-\natric and adult disease. The importance of such surveillances\nfor both the national and international communities has been\nhighlighted in recent reports (15, 32, 38).\nIn the study described here we provide new data on the\nnasopharyngeal carriage of drug-resistant pneumococci during\nthe 1999 surveillance. We also summarize temporal changes\nnoted in the same surveillance system between 1996 and 1999.\nStudy population. Between February and March of 1999, nasopharyngeal\nsamples were recovered from children attending DCCs in the Lisbon area.\nLisbon is the capital of Portugal and, according to the last census (of 2001),\napproximately 2 million people live in the city and its suburbs. The 14 DCCs were\nselected to be of diverse geographic locations and sizes and to include children\nof different social strata. The target population was 1,454 children, of which a\ntotal of 942 children (480 females and 462 males), with ages ranging from 6\nmonths to 6 years (average, 3.3 years old), participated in the study.\nQuestionnaire. Questionnaires on antibiotic consumption habits of the chil-\ndren were addressed to their parents and/or guardians at the time of sampling.\nThe questions were as follows. (i) Is the child taking any antibiotics now? (ii) Has\nthe child taken antibiotics within the last month? (iii) Has the child received\nrepeated courses (more than three) of antibiotics in the last 6 months? (iv) If the\nNasopharyngeal swab and isolation of S. pneumoniae. Nasopharyngeal secre-\ntions were taken by a pediatric nurse using swabs and modiﬁed Stuart’s medium\n(Mini-Tip Culturette; Becton Dickinson Microbiology Systems, Cockeysville,\nMd.). Swabs were transferred to the central research laboratory at the Instituto\nde Tecnologia Quı´mica e Biolo´gica and were inoculated into appropriate culture\nmedia within 4 h after the sample collection, to select and identify S. pneumoniae.\nBlood agar with gentamicin (5 mg/ml) and anaerobic incubation were used for\nselective culture of pneumococci. The pneumococci were identiﬁed by optochin\nsusceptibility and colony morphology. One colony was picked and a pure culture\nwas frozen. All isolates were frozen in Mueller-Hinton broth (Difco, Detroit,\nMich.) and 15% (vol/vol) glycerol (Merck, Darmstadt, Germany) and kept at\n70°C (8).\nAntimicrobial susceptibility testing. Susceptibility testing was performed using\nthe Kirby-Bauer technique, according to the NCCLS recommendations and\ndeﬁnitions (28). The antimicrobial agents tested were chloramphenicol, eryth-\nromycin, clindamycin, tetracycline, and sulfamethoxazole-trimethoprim (SXT).\nAntibiotic disks were purchased from Oxoid (Hampshire, England). Isolates\nwere also screened for MICs of penicillin and ceftriaxone with the E-test (AB\nBiodisk, Solna, Sweden) according to the manufacturer’s recommendations. In\nthe interpretation of decreased penicillin susceptibility, isolates were divided into\nintermediately resistant (0.1 g/ml  MIC  1.5 g/ml) and resistant (MIC \n1.5 g/ml) strains, according to NCCLS guidelines (28).\nPFGE. The strains that were serotyped were also typed by pulsed-ﬁeld gel\nelectrophoresis (PFGE). Preparation of chromosomal DNA, restriction with\nSmaI endonuclease, and PFGE were done as previously described (35). PFGE\npatterns were assigned by visual inspection of the macrorestriction proﬁles, by\nusing currently accepted criteria (42). The particular PFGE patterns, which were\ndetected in previous studies (8, 35), received the same uppercase letter; the new\nPFGE patterns were assigned two distinct uppercase letters.",
         "RESULTS\nStudy population. Between February and March of 1999,\n942 children were screened, corresponding to 65% of the chil-\ndren attending the 14 DCCs studied. The number of children\nenrolled at each day care center varied from 44 to 178, with an\naverage of 104 children. Participation rates in this study in each\nDCC ranged from 40 to 75%, and the number of children\nsampled in each center ranged from 20 to 132.\nPneumococcal carriage and resistance to antimicrobial\nagents. A total of 591 children (63%) of the 942 sampled were\npneumococcal carriers. Of the 591 pneumococcal isolates, 60%\nwere susceptible to all antimicrobial agents tested, 18% were\nsusceptible to penicillin but resistant to at least one of the\nother antimicrobials, and 22% had decreased susceptibility to\npenicillin: the penicillin MIC for 2% of the isolates was equal\nto or higher than 1.5 g/ml and 20% had intermediate resis-\ntance. Rates of resistance to antimicrobial agents other than\npenicillin were 26% for tetracycline, 25% for SXT, 8.5% for\nchloramphenicol, and 2% for ceftriaxone. Concerning macro-\nlide-lincosamide resistance, 22% of the isolates were resistant\nto erythromycin and 20.5% were resistant to clindamycin. Mul-\ntidrug resistance (deﬁned as resistance to three or more classes\nof antimicrobial agents) was common (26% of all pneumococ-\ncal isolates) and frequently (17%) included resistance to eryth-\nromycin, clindamycin, and tetracycline (Table 1).\nCarriage of antibiotic-resistant S. pneumoniae varied widely\nPG R, SXT\nPG R, CHL, TET, SXT\nPG I, CHL, ERY, CLI, TET, SXT\nPG I, CHL, ERY, CLI, TET\nPG I, CHL, ERY, CLI, SXT\nPG I, ERY, CLI, TET, SXT\nPG I, CHL, TET, SXT\nPG I, ERY, CLI, TET\nPG I, ERY, CLI, SXT\nPG I, ERY, TET, SXT\nPG I, CHL, SXT\nPG I, CHL, TET\nPG I, ERY, CHL\nPG I, ERY, TET\nPG I, TET, SXT\nPG I, ERY\nPG I, TET\nPG I, SXT\nPG I\nCHL, ERY, CLI, TET, SXT\nCHL, ERY, CLI, TET\nERY, CLI, TET, SXT\nERY, CLI, TET\nERY, CLI, SXT\nERY, TET, SXT\nERY, CLI\nERY, SXT\nTET, SXT\nERY\nSXT\nfrom one DCC to another (Fig. 1), reinforcing the idea that\nDCCs are autonomous epidemiological units (35). In DCC17,\nfor example, all strains recovered (n  13) were susceptible to\npenicillin, but half of them were resistant to at least another\nantimicrobial agent. By contrast, in DCC19, all strains (n  17)\nbut one were susceptible to all antimicrobial agents tested; in\nDCC22, similar proportions of isolates susceptible to all drugs\n(n  13), susceptible to penicillin but resistant to some other\ndrugs (n  17), and intermediately resistant to penicillin (n \n13) were found. We used chi-square and Fisher exact tests to\ndetermine whether this diversity was associated with antibiotic\nintake in the month prior to the sampling and found no sig-\nniﬁcant associations at a 95% conﬁdence level (data not\nshown).\nQuestionnaire. Seven percent of the 942 children were tak-\ning antimicrobials at the time of sampling; close to 23% of the\nchildren had taken antimicrobial agents in the previous month,\nand 21% had taken three or more courses of antimicrobial\nagents in the previous 6 months. The antibiotics most fre-\nquently reported by the parents were amoxicillin (39%) and\namoxicillin-clavulanic acid (41%); antibiotic use was mainly\ndue to otitis media (50%) and tonsillitis (24%).\nSerotyping and molecular typing of DRPn isolates. A total\nof 202 DRPn out of 237 were characterized by serotyping and\nPFGE. The 35 out of 237 DRPn that were resistant to SXT\nonly were not further characterized, because previous studies\ndemonstrated a high degree of diversity among SXT-resistant\nstrains in terms of serotypes and molecular types, suggesting\nfrequent independent emergence of resistance to this drug (8).\nAmong the 12 serotypes found the most frequent were 6B\n(26%), 14 (22%), 19F (16%), and 23F (10%). Other capsular\ntypes detected were 3 (1%), 6A (1%), 9V (4%), 15A (4%),\n15B (1%), 19A (1%), 23A (1%), and 33F (2%). Twelve\npercent of the DRPn were nontypeable.\nPFGE analysis grouped the isolates in 29 patterns, some of\nwhich are shown in Fig. 2. Table 2 summarizes relevant prop-\nerties of the 202 DRPn strains, grouping them by level of\nresistance to penicillin and PFGE pattern. Serotypes, antibi-\notic resistance proﬁles, and distribution of isolates by DCC are\nalso shown. PFGE patterns were associated with particular\nserotypes; the association with antibiotypes was less obvious. In\nall DCCs but one (DCC19, in which only one drug-resistant\nstrain was detected), four or more PFGE patterns were iden-\ntiﬁed among the DRPn isolates.\nBy comparing the 29 PFGE patterns (clonal types) with\npatterns from previous studies (8, 34), the 29 clones were\ngrouped as follows: representatives of Pneumococcal Molecu-\nlar Epidemiology Network (PMEN) clones; Lisbon DCC\nclones, which included strains that do not belong to PMEN\nclones and have PFGE patterns identiﬁed in our previous\nstudies (8, 34); and novel clones, which included strains with\nPFGE proﬁles that we could not identify in our previous stud-\nies (1996 to 1998). The majority (67%) of drug-resistant pneu-\nmococci were PMEN clones, 26% were novel clones, and 7%\nwere Lisbon DCC clones. PMEN clones were present in all day\ncare centers and, together, they accounted from 40 to 100% of\nthe DRPn strains in various DCCs, thus representing the ma-\njority of the isolates. Strains belonging to the Lisbon DCC\nclones were detected in 5 DCCs, and the novel clones were\nidentiﬁed in 11 of the 14 DCCs (Table 3). Some relevant\nproperties of the clones of each group are described below.\nPMEN clones. Nine PMEN clones were detected: clone\nSpain23F-1 present in ﬁve DCCs; Spain6B-2 in one DCC;\nSpain9V-3 in 12 DCCs; England14-9 in three DCCs;\nPoland6B-20 in six DCCs; clone Portugal19F-21 in ﬁve DCCs;\nclone Greece6B-22 in ﬁve DCCs; clone Sweden15A-25 in three\nDCCs, and clone Colombia23F-26 in two DCCs.\nClone B (Spain9V-3) was the most prevalent (20% of the\nDRPn strains) and also the most widely disseminated—de-\ntected in 12 out of 14 DCCs. Strains were either of serotype 9V\nor 14. Although the majority of the strains had intermediate\nresistance to penicillin and were also resistant to erythromycin\nand clindamycin, strains with other biotypes were also de-\ntected. Clone SI (Spain6B-2) was detected in DCC16. In our\nprevious studies (8, 34) no strains of this clone were identiﬁed.\nMLST was performed for two strains and the allelic proﬁle\nST90 characteristic of the Spain6B-2 clone reference strain was\nidentiﬁed (Table 4) (27). The remaining PMEN clones\nPortugal19F-21,\n(Spain23F-1, England6B-9,\nGreece6B-22, Sweden15A-25, and Colombia23F-26) had all been\nrecovered in 1996 to 1998 and had the same properties de-\nscribed previously (8, 34).\nPoland6B-20,\nLisbon DCC clones. Six clonal types were identiﬁed: NNN,\nD, X, XXX, EEE, and WWW. The six strains of clone NNN\nwere all nontypeable, were resistant to erythromycin, clinda-\nmycin, tetracycline, and SXT, and were either susceptible to\npenicillin or intermediately resistant to penicillin. This clone\nwas detected in four DCCs. In our earlier studies clone NNN\nwas recovered in a single DCC only (DCC2) in 1997 and 1998\n(35). By MLST this clone had ST344, i.e., had the same allelic\nproﬁle of two nontypeable strains recovered from patients with\ninvasive disease in Norway in 1996 and Australia in 1998 (Ta-\nble 4). Four strains of clone D and capsular type 19F were\nrecovered in DCC15. The strains had intermediate resistance\nto penicillin and were also resistant to erythromycin, clinda-\nmycin, tetracycline, and SXT. This clone was also detected in\nour earlier studies since 1996 but only in DCC7, and the pre-\nvious isolates were resistant to penicillin and SXT only (8).\nNovel clones. Three of the 14 novel clonal types (NF, NK,\nand BE) were nontypeable. The other clones were of serotypes\n6A, 15A, 19A, 19F, and 33F. Three clones (BE, AI, and AG)\nhad a high number of isolates accounting for 52% of these\nisolates. Representatives of these three clones were selected\nfor MLST.\nClone BE was found in two DCCs and all the strains were\nnontypeable and resistant to erythromycin, clindamycin, tetra-\ncycline, and SXT; six out of eight strains also had intermediate\nresistance to penicillin. This clone, although distinct by PFGE,\naccording to MLST is a double-locus variant of the clone of\nPFGE type NNN (also nontypeable), described above (Table\n4).\nAll strains of clone AI had capsular type 19F and were\nresistant to chloramphenicol, tetracycline, and SXT. These\nstrains were recovered in three DCCs (DCC16, -20, and -22).\nStrains recovered from children in DCC20 were also resistant\nto erythromycin and clindamycin. MLST of two representative\nisolates yielded two sequence types (ST87 and ST1198), dif-\nfering in the gki and ddl alleles. Comparison with data from the\nMLST database identiﬁed one strain also with ST87, which was\nfound to cause invasive disease in Denmark in 1997, and three\nisolates with double-locus variants of ST87 that were detected\nin Spain in 1989 and 1997 and in 1999 in Italy (Table 4).\nAll strains of clone AG had capsular type 33F and were\nresistant to erythromycin and clindamycin and were found in\nthree DCCs (DCC7, -18, and -23). By MLST, two ST types\nwere identiﬁed, ST1199 and ST717 (differing only in the ddl\nPG, CHL, ERY (4), CLI (4), TET (8), SXT\nPG, ERY, CLI, SXT\nPG\nPG, CHL (2), ERY, CLI, TET, SXT (3)\nPG, ERY, CLI, TET\nPG, ERY, CLI, TET, SXT\nPG, ERY, TET, SXT\nPG, ERY, CLI, TET\nPG, ERY, CLI, TET, SXT\nPG, CHL, ERY (3), CLI (3), TET, SXT\nPG, TET, SXT\nPG, ERY, CLI, TET, SXT (1)\nPG, ERY, CLI, TET, SXT\nERY, CLI, TET (8), SXT (6)\nERY, CLI, TET (4)\nERY, CLI (5), SXT (2)\nCHL (2), ERY, CLI, TET, SXT (5)\nERY (12), CLI (12), TET, SXT (1)\nERY, CLI\nERY, CLI, TET\nERY, CLI, TET\nERY, CLI, TET, SXT\nERY, CLI (2), TET, SXT\nERY (1), CLI (1), TET, SXT\n16 (10)\n22 (2)\n2 (2), 16 (1)\n2 (3), 5 (2), 7 (1), 16 (2), 22 (6)\n1 (1), 20 (3), 21 (2)\n1 (1), 2 (1), 9 (1)\n2 (2)\n9 (1), 22 (1)\n15 (4)\n16 (9), 20 (3), 22 (2)\n2 (3)\n21 (1), 22 (1)\n21 (5), 5 (1)\n1 (4), 17 (2), 21 (3)\n1 (1), 2 (1), 17 (2), 18 (3), 20 (2)\n9 (6), 21 (2)\n5 (1), 7 (6), 16 (1), 21 (1), 17 (2)\n2 (3), 5 (2), 22 (7), 23 (1)\n7 (3), 18 (1), 23 (1)\n16 (2)\n1 (2)\n21 (2)\n2 (1), 15 (2)\n22 (1), 23 (2)\nallele). ST717 was previously identiﬁed in one invasive strain\nfrom the United Kingdom in 2003, which was also of type 33F\n(Table 4).",
         "DISCUSSION\nThis study was a follow-up of the Lisbon Day Care Center\nInitiative which started in 1996 and was conducted on a yearly\nbasis (8, 9, 34, 35). The main purpose of the initiative was to\nmonitor the prevalence of drug-resistant pneumococcal strains\namong children attending DCCs and to characterize the strains\nby serotyping and molecular typing techniques. No such stud-\nies had ever been conducted in Portugal and, to our best\nknowledge, this remains the only surveillance program of this\ntype in the country. In addition, although other studies on the\npneumococci carried by DCC attendees have been conducted\nin other countries (3, 4, 19, 29), we are not aware of efforts that\nhave lasted for several years and are of such large magnitude,\nwhere all DRPn (including strains susceptible to penicillin)\nhave been extensively characterized by serotyping and geno-\ntyping (and antibiotyping), with the possible exception of a\nstudy described by Nilsson and Laurell (29) in Malmo, Sweden,\nwhich included close to 3,000 children attending 63 DCCs\nduring a 3-year period (1995 to 1997). In that study, molecular\ntyping was done by AP-PCR and BOX-A PCR. Other DCC\nstudies carried out in Greece (4), The Netherlands (3), and\nIsrael (19) have been described. These studies had smaller\nnumbers of children and of pneumococcal isolates. In addition,\nthese studies were point-prevalence ones and did not include\nyearly surveillances like ours, and they also had different de-\nsigns. In particular, the Israeli study was a 5-month longitudi-\nnal study of 252 children; the authors used ribotyping to char-\na PMEN clones found in our study: A, clone Spain23F-1; SI, clone Spain6B-2;\nB, clone Spain9V-3; R, clone England14-9; E, clone Poland6B-20; H, clone\nPortugal19F-21; M, clone Greece6B-22; DDD, clone Sweden15A-25; FF, clone\nColombia23F-26. Lisbon DCC clones, clones detected in our previous day care\ncenter studies; Novel clones, clones found in this study.\nacterize the 594 pneumococci (19). Two Greek studies were\nperformed since December of 1995. The ﬁrst one was per-\nformed with 338 children attending seven day care centers. The\nstudy was conducted from December of 1995 to February of\n1996. The second study, was conducted from February of 1997\nto February of 1999, but only 95 of the 2,448 children sampled\nattended day care centers; the authors used RFEL and MLST\nto characterize 128 isolates susceptible to penicillin but resis-\ntant to other antimicrobials (4). A study from The Netherlands\ncompared 259 children attending 16 DCCs with 276 children\nwho did not attend DCCs. Children were sampled twice from\nJanuary to March of 1999, and a total of 305 pneumococci were\nrecovered (3).\nIn fact, in the 4 years of the study, 23 DCCs and 1,800\nchildren were enrolled, generating 3,053 nasopharyngeal sam-\nples of which 1,687 were positive for pneumococci. All pneu-\nmococcal isolates were characterized by antibiotype. Of the\n740 DRPn recovered, 615 pneumococci strains were com-\npletely characterized by serotyping and PFGE typing (8, 35).\nThese data have been a rich source of information on the\npneumococcal population and have enabled us to observe\ntrends of the DRPn in its ecological niche. In addition, these\nstudies have provided a good baseline for two intervention\nstudies that began in 2001 aimed to decrease carriage of\nDRPn: one based on education on antibiotic use and hygiene\ncontrol measures (the EURIS project, European Resistance\nIntervention Study) and the other based on the licensed 7-va-\nlent conjugate vaccine Prevenar (in preparation). The EURIS\nproject is now being continued in a sequel named PREVIS\n(Pneumococcal Resistance, Epidemicity, and Virulence—an\nInternational Study) and, by 2006, we will have completed a\ndecade of annual-biannual surveillance of pneumococci car-\nried by DCC attendees.\nThe data from the 1999 surveillance described here indicate\nthat the carriage rate of S. pneumoniae in Portuguese DCCs\nhas increased continuously since 1996, from 47% to 48, 60, and\n63% in subsequent years (35). The reasons for this increase are\nnot clear. We can only speculate that the nurses who collected\nthe samples have been mastering their sampling technique,\nyielding higher pneumococcal recovery rates. Similar high\nrates of pneumococcal carriage in DCC children have been\nreported in other countries such as Iceland (50%) (23), Swe-\nden (52%) (22), Lithuania (51%) (23), France (54.7%) (12),\nIsrael (63%) (45), and Greece (48%) (44).\nComparison of the 1999 results with results of the previous\nyears of surveillance indicated that the proportion of DRPn\nranged between 40 to 50% of all isolates. The carriage rate of\npenicillin-resistant and penicillin-intermediate pneumococci\nalso ﬂuctuated—ranging from 2 to 8% and 12 to 27%, respec-\ntively—over time. Interestingly, 1999 was the year with the\nlowest rate of penicillin resistance (2%) (Fig. 3).\nAnalysis of Portuguese invasive strains, recovered between\n1999 and 2001 from pediatric patients of the same age group,\nidentiﬁed higher values of DRPn (63.8%) and of resistance to\npenicillin (4.7%) (38). However, drug resistance to antimicro-\nbials other than penicillin was 15.9% among invasive pediatric\nisolates, a value comparable to the 18% carriage rate of such\nstrains identiﬁed in our study. In 1999, 64% of the penicillin-\nsusceptible strains were coresistant to macrolides, clindamycin,\nand tetracycline, a trend previously observed in 1997 and 1998\nstudies (17, 33–35). DCC studies in Greece (44) found anti-\nbiotypes and levels of erythromycin and tetracycline resistance\ncomparable to those of our study. The Portuguese and Greek\ndata contrast with data from pneumococcal isolates from DCC\nattendees in Sweden which had similar levels of penicillin non-\nsusceptibility (20%) (36) but only rarely were resistant to\nerythromycin, tetracycline, and/or SXT.\nThe antimicrobial consumption was similar in all years and\nthe antibiotics most frequently used were always amoxicillin-\nclavulanic acid and amoxicillin. In our studies otitis has always\nbeen the major cause for antibiotic consumption reported by\nthe parents. Otitis is highly recognized as the most frequent\nafﬂiction responsible for visits to pediatricians and for antibi-\notic prescription, and thus is a major cause of morbidity and\nhas high costs associated with it (6). In addition, a study on the\nsimilarity of strains recovered from otitis and carriage (20)\nsuggests that intervention aimed to decrease carriage will also\ntarget otitis media.\nThe most frequent serotypes recovered in 1999 (6B, 14, 19F,\n23F, and nontypeable) were all (except for nontypeable iso-\nlates) included in the 7-valent conjugate vaccine. The 7-valent\nconjugate vaccine covered 78% of the DRPn isolates. These\ncapsular types had already been included among the most\nfrequent serotypes recovered in the past years (8, 34). These\nwere also the serotypes most commonly found in healthy chil-\ndren in Sweden (36), Israel (31), the United Kingdom (5), and\nGreece (41), although in a different rank order. Serotypes 14,\n23F, and 6B were also among the ﬁve most common capsular\ntypes in invasive strains recovered in 1999 to 2001 in Portugal\nfrom children younger than 6 years old (38). In our study,\nnonvaccine types (3, 6A, 15A, 15B, 19A, and 33F) together\nwith nontypeable strains accounted for 22% of the DRPn. All\nthese nonvaccine types had been detected in previous years\napart from serotype 33F (8, 35).\nMolecular typing identiﬁed several PMEN clones among the\nisolates. The international clone Spain9V-3, ﬁrst detected in Spain\n(7) and France (18, 25), was the clone most frequently isolated in\nour study. Interestingly, strains belonging to this clonal type also\nincluded isolates that had intermediate levels of resistance or\nwere fully susceptible to penicillin. Strains of Spain9V-3 have most\noften been described as fully resistant to penicillin (27) and iso-\nlates of invasive origin have been recovered in several countries\n(11, 13, 26). In our study, the majority of strains belonging to this\nclonal type expressed serotype 14, which represents one of the\nmost frequent serotypes among invasive isolates in Europe, the\nUnited States, and Canada (21).\nInterestingly, the relative proportion of strains of serotype\n23F (10%) decreased when compared to those from 1998 (P \n0.0038), reﬂecting the lower prevalence of international clone\nSpain23F-1 in the present study (Table 5). On the other hand, the\nmultiresistant Spain6B-2 clone was now detected, for the ﬁrst\ntime, in Portuguese DCCs. The clone was detected in a single\nDCC. Another six PMEN clones were identiﬁed (England14-9,\nPoland6B-20, Portugal19F-21, Greece6B-22, Sweden15A-25, and\nColombia23F-26); all of them had also been recovered in earlier\nPortuguese DCC studies (8, 34). Clones D and NNN, found since\n1996 in a single DCC (35), continued to be detected in other\nDCCs and thus are persisting and appear to be spreading through\ntime (Table 5).\nNovel clones were detected and accounted for 26% of the\nDRPn strains isolated. Three of them, BE, AI, and AG, had\nMLST sequence types that were identical or differed in only\none or two loci from strains of invasive origin recovered in\nother countries. Interestingly, clone AG was associated with\nserotype 33F, an unusual capsular type in carriage (2); strains\nof clone BE were nontypeable. Both clones have “capsules”\nnot included in 7-valent conjugate vaccine and are multiresis-\ntant to antimicrobial agents. An increased prevalence of these\nstrains among carried isolates could be, as postulated by others\n(36), a preview of what will soon appear in the pneumococcal\ndisease population.\nIn summary, the majority of drug-resistant strains recovered\nfrom DCC children in Portugal are members of multidrug-\nresistant international clones that can cause illness (invasive\nand noninvasive). The emergence of a serotype 33F clone and\nthe increasing number of nontypeable clones, both with capac-\nity to cause acute otitis media (5, 30), highlight that the naso-\npharynxes of DCC children are a melting pot that is important\nto monitor (2) if we aim to gain a better understanding of the\nepidemiology of this pathogen.\nPartial support for this work was provided by contracts EURIS\n(QLK2-CT-2000-01020) from the European Community, PRAXIS/P/\nSAU/14051/1998 from Fundac¸a˜o para a Cieˆncia e Tecnologia, Portu-\ngal, and a grant from Fundac¸a˜o Calouste Gulbenkian, Portugal,\nawarded to H. de Lencastre. R. Sa´-Lea˜o and J. Carric¸o were supported\nby grants SFRH/BPD/14596/2003 and SFRH/BD/3123/2000, respec-\ntively, both from Fundac¸a˜o para a Cieˆncia e Tecnologia, Portugal. S.\nNunes was supported by grant 011/BIC/01 from contract QLK2-CT-\n2000-01020 and C. R. Alves was supported by grant 001/99/BIC/P from\ncontract PRAXIS/P/SAU/14051/1998.\n1. Baquero, F., J. A. Garcia-Rodriguez, J. Garcia de Lomas, and L. Aguilar,\nand The Spanish Surveillance Group for Respiratory Pathogens. 1999. An-\ntimicrobial resistance of 1,113 Streptococcus pneumoniae isolates from pa-\ntients with respiratory tract infections in Spain: results of a 1-year (1996-\n1997) multicenter surveillance study. Antimicrob. Agents Chemother 43:\n357–359.\n9. de Lencastre, H., I. Santos Sanches, A. Brito-Avoˆ, R. Sa´-Lea˜o, J. Saldanha,\nK. G. Kristinsson, and A. Tomasz. 1999. Carriage and antibiotic resistance of\nrespiratory pathogens and molecular epidemiology of antibiotic-resistant\nStreptococcus pneumoniae colonizing children in day-care centers in Lisbon:\nthe Portuguese day-care center initiative. Clin. Microbiol. Infect. 5:S55–S63.\n10. Di Fabio, J. L., E. Castaneda, C. I. Agudelo, F. De La Hoz, M. Hortal, T.\nCamou, G. Echaniz-Aviles, M. Noemi, C. Barajas, I. Heitmann, J. C.\nHormazabal, M. C. Brandileone, V. S. Dias Vieira, M. Regueira, R. Ruvin-\nski, A. Corso, M. Lovgren, J. A. Talbot, C. De Quadros, et al. 2001. Evolution\nof Streptococcus pneumoniae serotypes and penicillin susceptibility in Latin\nAmerica, Sireva-Vigia Group, 1993 to 1999. Pediatr. Infect. Dis. J. 20:959–\n967.\n16. Felmingham, D., R. R. Reinert, Y. Hirakata, and A. Rodloff. 2002. Increasing\nprevalence of antimicrobial resistance among isolates of Streptococcus pneu-\nmoniae from the PROTEKT surveillance study, and comparative in vitro\nactivity of the ketolide, telithromycin. J. Antimicrob. Chemother. 50:25–37.\n17. Garcia-Rey, C., E. Bouza, L. Aguilar, J. Garcia-de-Lomas, and F. Baquero.\n2003. Evolution of penicillin and erythromycin co-resistance in Streptococcus\npneumoniae in Spain. Int. J. Antimicrob. Agents 22:541–544.\n22. Henriques Normark, B., B. Christensson, A. Sandgren, B. Noreen, S. Sylvan,\nL. G. Burman, and B. Olsson-Liljequist. 2003. Clonal analysis of Streptococ-\ncus pneumoniae nonsusceptible to penicillin at day-care centers with index\ncases, in a region with low incidence of resistance: emergence of an invasive\ntype 35B clone among carriers. Microb. Drug Resist. 9:337–344.\n32. Sahm, D. F., J. A. Karlowsky, L. J. Kelly, I. A. Critchley, M. E. Jones, C.\nThornsberry, Y. Mauriz, and J. Kahn. 2001. Need for annual surveillance of\nantimicrobial resistance in Streptococcus pneumoniae in the United States:\n2-year longitudinal analysis. Antimicrob. Agents Chemother. 45:1037–1042.\n33. Sa´-Lea˜o, R., A. Tomasz, and H. de Lencastre. 2001. Multilocus sequence\ntyping of Streptococcus pneumoniae clones with unusual drug resistance\npatterns: genetic backgrounds and relatedness to other epidemic clones.\nJ Infect. Dis. 184:1206–1210.\n35. Sa´-Lea˜o, R., A. Tomasz, I. S. Sanches, S. Nunes, C. R. Alves, A. B. Avoˆ, J.\nSaldanha, K. G. Kristinsson, and H. de Lencastre. 2000. Genetic diversity\nand clonal patterns among antibiotic-susceptible and -resistant Streptococcus\npneumoniae colonizing children: day care centers as autonomous epidemio-\nlogical units. J. Clin. Microbiol. 38:4137–4144.\n39. Song, J. H., N. Y. Lee, S. Ichiyama, R. Yoshida, Y. Hirakata, W. Fu, A.\nChongthaleong, N. Aswapokee, C. H. Chiu, M. K. Lalitha, K. Thomas, J.\nPerera, T. T. Yee, F. Jamal, U. C. Warsa, B. X. Vinh, M. R. Jacobs, P. C.\nAppelbaum, and C. H. Pai. 1999. Spread of drug-resistant Streptococcus\npneumoniae in Asian countries: Asian Network for Surveillance of Resistant\nPathogens (ANSORP) Study. Clin. Infect. Dis. 28:1206–1211.\n43. Thornsberry, C., D. F. Sahm, L. J. Kelly, I. A. Critchley, M. E. Jones, A. T.\nEvangelista, and J. A. Karlowsky. 2002. Regional trends in antimicrobial\nresistance among clinical isolates of Streptococcus pneumoniae, Haemophilus\ninﬂuenzae, and Moraxella catarrhalis in the United States: results from the\nTRUST Surveillance Program, 1999-2000. Clin. Infect. Dis. 34:S4–S16.\n44. Tsolia, M., G. Kouppari, A. Zaphiropoulou, S. Gavrili, M. Tsirepa, D.\nKafetzis, and T. Karpathios. 1999. Prevalence and patterns of resistance of\nStreptococcus pneumoniae strains isolated from carriers attending day care\ncenters in the area of Athens. Microb. Drug Resist. 5:271–278.",
         "",
         "Of the nasopharyngeal cultures recovered from 942 day care center (DCC) attendees in Lisbon, Portugal, 591\n(62%) yielded Streptococcus pneumoniae during a surveillance performed in February and March of 1999. Forty\npercent of the isolates were resistant to one or more antimicrobial agents. In particular, 2% were penicillin\nresistant and 20% had intermediate penicillin resistance. Multidrug resistance to macrolides, lincosamides,\nand tetracycline was the most frequent antibiotype (17% of all isolates). Serotyping and molecular typing by\npulsed-ﬁeld gel electrophoresis were performed for 202 out of 237 drug-resistant pneumococci (DRPn). The\nmost frequent serotypes were 6B (26%), 14 (22%), 19F (16%), 23F (10%), and nontypeable (12%). The majority\n(67%) of the DRPn strains were representatives of nine international clones included in the Pneumococcal\nMolecular Epidemiology Network; eight of them had been detected in previous studies. Fourteen novel clones\nwere identiﬁed, corresponding to 26% of the DRPn strains. The remaining 7% of the strains were local clones\ndetected in our previous studies. Comparison with studies conducted since 1996 in Portuguese DCCs identiﬁed\nseveral trends: (i) the rate of DRPn frequency has ﬂuctuated between 40 and 50%; (ii) the serotypes most\nfrequently recovered have remained the same; (iii) nontypeable strains appear to be increasing in frequency;\nand (iv) a clone of serotype 33F emerged in 1999. Together, our observations highlight that the nasopharynxes\nof children in DCCs are a melting pot of successful DRPn clones that are important to study and monitor if\nwe aim to gain a better understanding on the epidemiology of this pathogen.\nThe ecological niche of Streptococcus pneumoniae is the na-\nsopharynxes of humans. Although pneumococci are widely\nspread in the community, carriage rates are particularly high in\npreschool children attending day care centers (DCCs) (24, 45).\nS. pneumoniae can cause a spectrum of diseases including\ninvasive disease, such as meningitis, bloodstream infections,\nand pneumonia, as well as infections of the upper respiratory\ntract. S. pneumoniae is a major cause of morbidity and mor-\ntality in young children (6). In the last decade an increasing\nproportion of pneumococci that are not susceptible to penicil-\nlin has been observed, reaching up to 50% in some areas of the\nworld (1, 10, 16, 37, 39, 43).\nIn 1996 our group began yearly surveillance of the nasopha-\nryngeal carriage of pneumococci and other bacterial respira-\ntory tract pathogens in DCCs located in the Lisbon area of\nPortugal. Since 1996, a total of 23 DCCs and over 1,800 chil-\ndren participated in this study. Serotypes, antibiotypes, and\nmolecular types of the majority of drug-resistant S. pneu-\nmoniae isolates were determined during these annual surveil-\nlances (8, 9, 34, 35), allowing one to obtain a view of temporal\nchanges of drug-resistant pneumococci (DRPn) inhabiting the\nnasopharyngeal ﬂora, an important ecological reservoir which\nmay be a major source of drug-resistant strains causing pedi-\natric and adult disease. The importance of such surveillances\nfor both the national and international communities has been\nhighlighted in recent reports (15, 32, 38).\nIn the study described here we provide new data on the\nnasopharyngeal carriage of drug-resistant pneumococci during\nthe 1999 surveillance. We also summarize temporal changes\nnoted in the same surveillance system between 1996 and 1999.\nStudy population. Between February and March of 1999, nasopharyngeal\nsamples were recovered from children attending DCCs in the Lisbon area.\nLisbon is the capital of Portugal and, according to the last census (of 2001),\napproximately 2 million people live in the city and its suburbs. The 14 DCCs were\nselected to be of diverse geographic locations and sizes and to include children\nof different social strata. The target population was 1,454 children, of which a\ntotal of 942 children (480 females and 462 males), with ages ranging from 6\nmonths to 6 years (average, 3.3 years old), participated in the study.\nQuestionnaire. Questionnaires on antibiotic consumption habits of the chil-\ndren were addressed to their parents and/or guardians at the time of sampling.\nThe questions were as follows. (i) Is the child taking any antibiotics now? (ii) Has\nthe child taken antibiotics within the last month? (iii) Has the child received\nrepeated courses (more than three) of antibiotics in the last 6 months? (iv) If the\nNasopharyngeal swab and isolation of S. pneumoniae. Nasopharyngeal secre-\ntions were taken by a pediatric nurse using swabs and modiﬁed Stuart’s medium\n(Mini-Tip Culturette; Becton Dickinson Microbiology Systems, Cockeysville,\nMd.). Swabs were transferred to the central research laboratory at the Instituto\nde Tecnologia Quı´mica e Biolo´gica and were inoculated into appropriate culture\nmedia within 4 h after the sample collection, to select and identify S. pneumoniae.\nBlood agar with gentamicin (5 mg/ml) and anaerobic incubation were used for\nselective culture of pneumococci. The pneumococci were identiﬁed by optochin\nsusceptibility and colony morphology. One colony was picked and a pure culture\nwas frozen. All isolates were frozen in Mueller-Hinton broth (Difco, Detroit,\nMich.) and 15% (vol/vol) glycerol (Merck, Darmstadt, Germany) and kept at\n70°C (8).\nAntimicrobial susceptibility testing. Susceptibility testing was performed using\nthe Kirby-Bauer technique, according to the NCCLS recommendations and\ndeﬁnitions (28). The antimicrobial agents tested were chloramphenicol, eryth-\nromycin, clindamycin, tetracycline, and sulfamethoxazole-trimethoprim (SXT).\nAntibiotic disks were purchased from Oxoid (Hampshire, England). Isolates\nwere also screened for MICs of penicillin and ceftriaxone with the E-test (AB\nBiodisk, Solna, Sweden) according to the manufacturer’s recommendations. In\nthe interpretation of decreased penicillin susceptibility, isolates were divided into\nintermediately resistant (0.1 g/ml  MIC  1.5 g/ml) and resistant (MIC \n1.5 g/ml) strains, according to NCCLS guidelines (28).\nPFGE. The strains that were serotyped were also typed by pulsed-ﬁeld gel\nelectrophoresis (PFGE). Preparation of chromosomal DNA, restriction with\nSmaI endonuclease, and PFGE were done as previously described (35). PFGE\npatterns were assigned by visual inspection of the macrorestriction proﬁles, by\nusing currently accepted criteria (42). The particular PFGE patterns, which were\ndetected in previous studies (8, 35), received the same uppercase letter; the new\nPFGE patterns were assigned two distinct uppercase letters.\nRESULTS\nStudy population. Between February and March of 1999,\n942 children were screened, corresponding to 65% of the chil-\ndren attending the 14 DCCs studied. The number of children\nenrolled at each day care center varied from 44 to 178, with an\naverage of 104 children. Participation rates in this study in each\nDCC ranged from 40 to 75%, and the number of children\nsampled in each center ranged from 20 to 132.\nPneumococcal carriage and resistance to antimicrobial\nagents. A total of 591 children (63%) of the 942 sampled were\npneumococcal carriers. Of the 591 pneumococcal isolates, 60%\nwere susceptible to all antimicrobial agents tested, 18% were\nsusceptible to penicillin but resistant to at least one of the\nother antimicrobials, and 22% had decreased susceptibility to\npenicillin: the penicillin MIC for 2% of the isolates was equal\nto or higher than 1.5 g/ml and 20% had intermediate resis-\ntance. Rates of resistance to antimicrobial agents other than\npenicillin were 26% for tetracycline, 25% for SXT, 8.5% for\nchloramphenicol, and 2% for ceftriaxone. Concerning macro-\nlide-lincosamide resistance, 22% of the isolates were resistant\nto erythromycin and 20.5% were resistant to clindamycin. Mul-\ntidrug resistance (deﬁned as resistance to three or more classes\nof antimicrobial agents) was common (26% of all pneumococ-\ncal isolates) and frequently (17%) included resistance to eryth-\nromycin, clindamycin, and tetracycline (Table 1).\nCarriage of antibiotic-resistant S. pneumoniae varied widely\nPG R, SXT\nPG R, CHL, TET, SXT\nPG I, CHL, ERY, CLI, TET, SXT\nPG I, CHL, ERY, CLI, TET\nPG I, CHL, ERY, CLI, SXT\nPG I, ERY, CLI, TET, SXT\nPG I, CHL, TET, SXT\nPG I, ERY, CLI, TET\nPG I, ERY, CLI, SXT\nPG I, ERY, TET, SXT\nPG I, CHL, SXT\nPG I, CHL, TET\nPG I, ERY, CHL\nPG I, ERY, TET\nPG I, TET, SXT\nPG I, ERY\nPG I, TET\nPG I, SXT\nPG I\nCHL, ERY, CLI, TET, SXT\nCHL, ERY, CLI, TET\nERY, CLI, TET, SXT\nERY, CLI, TET\nERY, CLI, SXT\nERY, TET, SXT\nERY, CLI\nERY, SXT\nTET, SXT\nERY\nSXT\nfrom one DCC to another (Fig. 1), reinforcing the idea that\nDCCs are autonomous epidemiological units (35). In DCC17,\nfor example, all strains recovered (n  13) were susceptible to\npenicillin, but half of them were resistant to at least another\nantimicrobial agent. By contrast, in DCC19, all strains (n  17)\nbut one were susceptible to all antimicrobial agents tested; in\nDCC22, similar proportions of isolates susceptible to all drugs\n(n  13), susceptible to penicillin but resistant to some other\ndrugs (n  17), and intermediately resistant to penicillin (n \n13) were found. We used chi-square and Fisher exact tests to\ndetermine whether this diversity was associated with antibiotic\nintake in the month prior to the sampling and found no sig-\nniﬁcant associations at a 95% conﬁdence level (data not\nshown).\nQuestionnaire. Seven percent of the 942 children were tak-\ning antimicrobials at the time of sampling; close to 23% of the\nchildren had taken antimicrobial agents in the previous month,\nand 21% had taken three or more courses of antimicrobial\nagents in the previous 6 months. The antibiotics most fre-\nquently reported by the parents were amoxicillin (39%) and\namoxicillin-clavulanic acid (41%); antibiotic use was mainly\ndue to otitis media (50%) and tonsillitis (24%).\nSerotyping and molecular typing of DRPn isolates. A total\nof 202 DRPn out of 237 were characterized by serotyping and\nPFGE. The 35 out of 237 DRPn that were resistant to SXT\nonly were not further characterized, because previous studies\ndemonstrated a high degree of diversity among SXT-resistant\nstrains in terms of serotypes and molecular types, suggesting\nfrequent independent emergence of resistance to this drug (8).\nAmong the 12 serotypes found the most frequent were 6B\n(26%), 14 (22%), 19F (16%), and 23F (10%). Other capsular\ntypes detected were 3 (1%), 6A (1%), 9V (4%), 15A (4%),\n15B (1%), 19A (1%), 23A (1%), and 33F (2%). Twelve\npercent of the DRPn were nontypeable.\nPFGE analysis grouped the isolates in 29 patterns, some of\nwhich are shown in Fig. 2. Table 2 summarizes relevant prop-\nerties of the 202 DRPn strains, grouping them by level of\nresistance to penicillin and PFGE pattern. Serotypes, antibi-\notic resistance proﬁles, and distribution of isolates by DCC are\nalso shown. PFGE patterns were associated with particular\nserotypes; the association with antibiotypes was less obvious. In\nall DCCs but one (DCC19, in which only one drug-resistant\nstrain was detected), four or more PFGE patterns were iden-\ntiﬁed among the DRPn isolates.\nBy comparing the 29 PFGE patterns (clonal types) with\npatterns from previous studies (8, 34), the 29 clones were\ngrouped as follows: representatives of Pneumococcal Molecu-\nlar Epidemiology Network (PMEN) clones; Lisbon DCC\nclones, which included strains that do not belong to PMEN\nclones and have PFGE patterns identiﬁed in our previous\nstudies (8, 34); and novel clones, which included strains with\nPFGE proﬁles that we could not identify in our previous stud-\nies (1996 to 1998). The majority (67%) of drug-resistant pneu-\nmococci were PMEN clones, 26% were novel clones, and 7%\nwere Lisbon DCC clones. PMEN clones were present in all day\ncare centers and, together, they accounted from 40 to 100% of\nthe DRPn strains in various DCCs, thus representing the ma-\njority of the isolates. Strains belonging to the Lisbon DCC\nclones were detected in 5 DCCs, and the novel clones were\nidentiﬁed in 11 of the 14 DCCs (Table 3). Some relevant\nproperties of the clones of each group are described below.\nPMEN clones. Nine PMEN clones were detected: clone\nSpain23F-1 present in ﬁve DCCs; Spain6B-2 in one DCC;\nSpain9V-3 in 12 DCCs; England14-9 in three DCCs;\nPoland6B-20 in six DCCs; clone Portugal19F-21 in ﬁve DCCs;\nclone Greece6B-22 in ﬁve DCCs; clone Sweden15A-25 in three\nDCCs, and clone Colombia23F-26 in two DCCs.\nClone B (Spain9V-3) was the most prevalent (20% of the\nDRPn strains) and also the most widely disseminated—de-\ntected in 12 out of 14 DCCs. Strains were either of serotype 9V\nor 14. Although the majority of the strains had intermediate\nresistance to penicillin and were also resistant to erythromycin\nand clindamycin, strains with other biotypes were also de-\ntected. Clone SI (Spain6B-2) was detected in DCC16. In our\nprevious studies (8, 34) no strains of this clone were identiﬁed.\nMLST was performed for two strains and the allelic proﬁle\nST90 characteristic of the Spain6B-2 clone reference strain was\nidentiﬁed (Table 4) (27). The remaining PMEN clones\nPortugal19F-21,\n(Spain23F-1, England6B-9,\nGreece6B-22, Sweden15A-25, and Colombia23F-26) had all been\nrecovered in 1996 to 1998 and had the same properties de-\nscribed previously (8, 34).\nPoland6B-20,\nLisbon DCC clones. Six clonal types were identiﬁed: NNN,\nD, X, XXX, EEE, and WWW. The six strains of clone NNN\nwere all nontypeable, were resistant to erythromycin, clinda-\nmycin, tetracycline, and SXT, and were either susceptible to\npenicillin or intermediately resistant to penicillin. This clone\nwas detected in four DCCs. In our earlier studies clone NNN\nwas recovered in a single DCC only (DCC2) in 1997 and 1998\n(35). By MLST this clone had ST344, i.e., had the same allelic\nproﬁle of two nontypeable strains recovered from patients with\ninvasive disease in Norway in 1996 and Australia in 1998 (Ta-\nble 4). Four strains of clone D and capsular type 19F were\nrecovered in DCC15. The strains had intermediate resistance\nto penicillin and were also resistant to erythromycin, clinda-\nmycin, tetracycline, and SXT. This clone was also detected in\nour earlier studies since 1996 but only in DCC7, and the pre-\nvious isolates were resistant to penicillin and SXT only (8).\nNovel clones. Three of the 14 novel clonal types (NF, NK,\nand BE) were nontypeable. The other clones were of serotypes\n6A, 15A, 19A, 19F, and 33F. Three clones (BE, AI, and AG)\nhad a high number of isolates accounting for 52% of these\nisolates. Representatives of these three clones were selected\nfor MLST.\nClone BE was found in two DCCs and all the strains were\nnontypeable and resistant to erythromycin, clindamycin, tetra-\ncycline, and SXT; six out of eight strains also had intermediate\nresistance to penicillin. This clone, although distinct by PFGE,\naccording to MLST is a double-locus variant of the clone of\nPFGE type NNN (also nontypeable), described above (Table\n4).\nAll strains of clone AI had capsular type 19F and were\nresistant to chloramphenicol, tetracycline, and SXT. These\nstrains were recovered in three DCCs (DCC16, -20, and -22).\nStrains recovered from children in DCC20 were also resistant\nto erythromycin and clindamycin. MLST of two representative\nisolates yielded two sequence types (ST87 and ST1198), dif-\nfering in the gki and ddl alleles. Comparison with data from the\nMLST database identiﬁed one strain also with ST87, which was\nfound to cause invasive disease in Denmark in 1997, and three\nisolates with double-locus variants of ST87 that were detected\nin Spain in 1989 and 1997 and in 1999 in Italy (Table 4).\nAll strains of clone AG had capsular type 33F and were\nresistant to erythromycin and clindamycin and were found in\nthree DCCs (DCC7, -18, and -23). By MLST, two ST types\nwere identiﬁed, ST1199 and ST717 (differing only in the ddl\nPG, CHL, ERY (4), CLI (4), TET (8), SXT\nPG, ERY, CLI, SXT\nPG\nPG, CHL (2), ERY, CLI, TET, SXT (3)\nPG, ERY, CLI, TET\nPG, ERY, CLI, TET, SXT\nPG, ERY, TET, SXT\nPG, ERY, CLI, TET\nPG, ERY, CLI, TET, SXT\nPG, CHL, ERY (3), CLI (3), TET, SXT\nPG, TET, SXT\nPG, ERY, CLI, TET, SXT (1)\nPG, ERY, CLI, TET, SXT\nERY, CLI, TET (8), SXT (6)\nERY, CLI, TET (4)\nERY, CLI (5), SXT (2)\nCHL (2), ERY, CLI, TET, SXT (5)\nERY (12), CLI (12), TET, SXT (1)\nERY, CLI\nERY, CLI, TET\nERY, CLI, TET\nERY, CLI, TET, SXT\nERY, CLI (2), TET, SXT\nERY (1), CLI (1), TET, SXT\n16 (10)\n22 (2)\n2 (2), 16 (1)\n2 (3), 5 (2), 7 (1), 16 (2), 22 (6)\n1 (1), 20 (3), 21 (2)\n1 (1), 2 (1), 9 (1)\n2 (2)\n9 (1), 22 (1)\n15 (4)\n16 (9), 20 (3), 22 (2)\n2 (3)\n21 (1), 22 (1)\n21 (5), 5 (1)\n1 (4), 17 (2), 21 (3)\n1 (1), 2 (1), 17 (2), 18 (3), 20 (2)\n9 (6), 21 (2)\n5 (1), 7 (6), 16 (1), 21 (1), 17 (2)\n2 (3), 5 (2), 22 (7), 23 (1)\n7 (3), 18 (1), 23 (1)\n16 (2)\n1 (2)\n21 (2)\n2 (1), 15 (2)\n22 (1), 23 (2)\nallele). ST717 was previously identiﬁed in one invasive strain\nfrom the United Kingdom in 2003, which was also of type 33F\n(Table 4).\nDISCUSSION\nThis study was a follow-up of the Lisbon Day Care Center\nInitiative which started in 1996 and was conducted on a yearly\nbasis (8, 9, 34, 35). The main purpose of the initiative was to\nmonitor the prevalence of drug-resistant pneumococcal strains\namong children attending DCCs and to characterize the strains\nby serotyping and molecular typing techniques. No such stud-\nies had ever been conducted in Portugal and, to our best\nknowledge, this remains the only surveillance program of this\ntype in the country. In addition, although other studies on the\npneumococci carried by DCC attendees have been conducted\nin other countries (3, 4, 19, 29), we are not aware of efforts that\nhave lasted for several years and are of such large magnitude,\nwhere all DRPn (including strains susceptible to penicillin)\nhave been extensively characterized by serotyping and geno-\ntyping (and antibiotyping), with the possible exception of a\nstudy described by Nilsson and Laurell (29) in Malmo, Sweden,\nwhich included close to 3,000 children attending 63 DCCs\nduring a 3-year period (1995 to 1997). In that study, molecular\ntyping was done by AP-PCR and BOX-A PCR. Other DCC\nstudies carried out in Greece (4), The Netherlands (3), and\nIsrael (19) have been described. These studies had smaller\nnumbers of children and of pneumococcal isolates. In addition,\nthese studies were point-prevalence ones and did not include\nyearly surveillances like ours, and they also had different de-\nsigns. In particular, the Israeli study was a 5-month longitudi-\nnal study of 252 children; the authors used ribotyping to char-\na PMEN clones found in our study: A, clone Spain23F-1; SI, clone Spain6B-2;\nB, clone Spain9V-3; R, clone England14-9; E, clone Poland6B-20; H, clone\nPortugal19F-21; M, clone Greece6B-22; DDD, clone Sweden15A-25; FF, clone\nColombia23F-26. Lisbon DCC clones, clones detected in our previous day care\ncenter studies; Novel clones, clones found in this study.\nacterize the 594 pneumococci (19). Two Greek studies were\nperformed since December of 1995. The ﬁrst one was per-\nformed with 338 children attending seven day care centers. The\nstudy was conducted from December of 1995 to February of\n1996. The second study, was conducted from February of 1997\nto February of 1999, but only 95 of the 2,448 children sampled\nattended day care centers; the authors used RFEL and MLST\nto characterize 128 isolates susceptible to penicillin but resis-\ntant to other antimicrobials (4). A study from The Netherlands\ncompared 259 children attending 16 DCCs with 276 children\nwho did not attend DCCs. Children were sampled twice from\nJanuary to March of 1999, and a total of 305 pneumococci were\nrecovered (3).\nIn fact, in the 4 years of the study, 23 DCCs and 1,800\nchildren were enrolled, generating 3,053 nasopharyngeal sam-\nples of which 1,687 were positive for pneumococci. All pneu-\nmococcal isolates were characterized by antibiotype. Of the\n740 DRPn recovered, 615 pneumococci strains were com-\npletely characterized by serotyping and PFGE typing (8, 35).\nThese data have been a rich source of information on the\npneumococcal population and have enabled us to observe\ntrends of the DRPn in its ecological niche. In addition, these\nstudies have provided a good baseline for two intervention\nstudies that began in 2001 aimed to decrease carriage of\nDRPn: one based on education on antibiotic use and hygiene\ncontrol measures (the EURIS project, European Resistance\nIntervention Study) and the other based on the licensed 7-va-\nlent conjugate vaccine Prevenar (in preparation). The EURIS\nproject is now being continued in a sequel named PREVIS\n(Pneumococcal Resistance, Epidemicity, and Virulence—an\nInternational Study) and, by 2006, we will have completed a\ndecade of annual-biannual surveillance of pneumococci car-\nried by DCC attendees.\nThe data from the 1999 surveillance described here indicate\nthat the carriage rate of S. pneumoniae in Portuguese DCCs\nhas increased continuously since 1996, from 47% to 48, 60, and\n63% in subsequent years (35). The reasons for this increase are\nnot clear. We can only speculate that the nurses who collected\nthe samples have been mastering their sampling technique,\nyielding higher pneumococcal recovery rates. Similar high\nrates of pneumococcal carriage in DCC children have been\nreported in other countries such as Iceland (50%) (23), Swe-\nden (52%) (22), Lithuania (51%) (23), France (54.7%) (12),\nIsrael (63%) (45), and Greece (48%) (44).\nComparison of the 1999 results with results of the previous\nyears of surveillance indicated that the proportion of DRPn\nranged between 40 to 50% of all isolates. The carriage rate of\npenicillin-resistant and penicillin-intermediate pneumococci\nalso ﬂuctuated—ranging from 2 to 8% and 12 to 27%, respec-\ntively—over time. Interestingly, 1999 was the year with the\nlowest rate of penicillin resistance (2%) (Fig. 3).\nAnalysis of Portuguese invasive strains, recovered between\n1999 and 2001 from pediatric patients of the same age group,\nidentiﬁed higher values of DRPn (63.8%) and of resistance to\npenicillin (4.7%) (38). However, drug resistance to antimicro-\nbials other than penicillin was 15.9% among invasive pediatric\nisolates, a value comparable to the 18% carriage rate of such\nstrains identiﬁed in our study. In 1999, 64% of the penicillin-\nsusceptible strains were coresistant to macrolides, clindamycin,\nand tetracycline, a trend previously observed in 1997 and 1998\nstudies (17, 33–35). DCC studies in Greece (44) found anti-\nbiotypes and levels of erythromycin and tetracycline resistance\ncomparable to those of our study. The Portuguese and Greek\ndata contrast with data from pneumococcal isolates from DCC\nattendees in Sweden which had similar levels of penicillin non-\nsusceptibility (20%) (36) but only rarely were resistant to\nerythromycin, tetracycline, and/or SXT.\nThe antimicrobial consumption was similar in all years and\nthe antibiotics most frequently used were always amoxicillin-\nclavulanic acid and amoxicillin. In our studies otitis has always\nbeen the major cause for antibiotic consumption reported by\nthe parents. Otitis is highly recognized as the most frequent\nafﬂiction responsible for visits to pediatricians and for antibi-\notic prescription, and thus is a major cause of morbidity and\nhas high costs associated with it (6). In addition, a study on the\nsimilarity of strains recovered from otitis and carriage (20)\nsuggests that intervention aimed to decrease carriage will also\ntarget otitis media.\nThe most frequent serotypes recovered in 1999 (6B, 14, 19F,\n23F, and nontypeable) were all (except for nontypeable iso-\nlates) included in the 7-valent conjugate vaccine. The 7-valent\nconjugate vaccine covered 78% of the DRPn isolates. These\ncapsular types had already been included among the most\nfrequent serotypes recovered in the past years (8, 34). These\nwere also the serotypes most commonly found in healthy chil-\ndren in Sweden (36), Israel (31), the United Kingdom (5), and\nGreece (41), although in a different rank order. Serotypes 14,\n23F, and 6B were also among the ﬁve most common capsular\ntypes in invasive strains recovered in 1999 to 2001 in Portugal\nfrom children younger than 6 years old (38). In our study,\nnonvaccine types (3, 6A, 15A, 15B, 19A, and 33F) together\nwith nontypeable strains accounted for 22% of the DRPn. All\nthese nonvaccine types had been detected in previous years\napart from serotype 33F (8, 35).\nMolecular typing identiﬁed several PMEN clones among the\nisolates. The international clone Spain9V-3, ﬁrst detected in Spain\n(7) and France (18, 25), was the clone most frequently isolated in\nour study. Interestingly, strains belonging to this clonal type also\nincluded isolates that had intermediate levels of resistance or\nwere fully susceptible to penicillin. Strains of Spain9V-3 have most\noften been described as fully resistant to penicillin (27) and iso-\nlates of invasive origin have been recovered in several countries\n(11, 13, 26). In our study, the majority of strains belonging to this\nclonal type expressed serotype 14, which represents one of the\nmost frequent serotypes among invasive isolates in Europe, the\nUnited States, and Canada (21).\nInterestingly, the relative proportion of strains of serotype\n23F (10%) decreased when compared to those from 1998 (P \n0.0038), reﬂecting the lower prevalence of international clone\nSpain23F-1 in the present study (Table 5). On the other hand, the\nmultiresistant Spain6B-2 clone was now detected, for the ﬁrst\ntime, in Portuguese DCCs. The clone was detected in a single\nDCC. Another six PMEN clones were identiﬁed (England14-9,\nPoland6B-20, Portugal19F-21, Greece6B-22, Sweden15A-25, and\nColombia23F-26); all of them had also been recovered in earlier\nPortuguese DCC studies (8, 34). Clones D and NNN, found since\n1996 in a single DCC (35), continued to be detected in other\nDCCs and thus are persisting and appear to be spreading through\ntime (Table 5).\nNovel clones were detected and accounted for 26% of the\nDRPn strains isolated. Three of them, BE, AI, and AG, had\nMLST sequence types that were identical or differed in only\none or two loci from strains of invasive origin recovered in\nother countries. Interestingly, clone AG was associated with\nserotype 33F, an unusual capsular type in carriage (2); strains\nof clone BE were nontypeable. Both clones have “capsules”\nnot included in 7-valent conjugate vaccine and are multiresis-\ntant to antimicrobial agents. An increased prevalence of these\nstrains among carried isolates could be, as postulated by others\n(36), a preview of what will soon appear in the pneumococcal\ndisease population.\nIn summary, the majority of drug-resistant strains recovered\nfrom DCC children in Portugal are members of multidrug-\nresistant international clones that can cause illness (invasive\nand noninvasive). The emergence of a serotype 33F clone and\nthe increasing number of nontypeable clones, both with capac-\nity to cause acute otitis media (5, 30), highlight that the naso-\npharynxes of DCC children are a melting pot that is important\nto monitor (2) if we aim to gain a better understanding of the\nepidemiology of this pathogen.\nPartial support for this work was provided by contracts EURIS\n(QLK2-CT-2000-01020) from the European Community, PRAXIS/P/\nSAU/14051/1998 from Fundac¸a˜o para a Cieˆncia e Tecnologia, Portu-\ngal, and a grant from Fundac¸a˜o Calouste Gulbenkian, Portugal,\nawarded to H. de Lencastre. R. Sa´-Lea˜o and J. Carric¸o were supported\nby grants SFRH/BPD/14596/2003 and SFRH/BD/3123/2000, respec-\ntively, both from Fundac¸a˜o para a Cieˆncia e Tecnologia, Portugal. S.\nNunes was supported by grant 011/BIC/01 from contract QLK2-CT-\n2000-01020 and C. R. Alves was supported by grant 001/99/BIC/P from\ncontract PRAXIS/P/SAU/14051/1998.\n1. Baquero, F., J. A. Garcia-Rodriguez, J. Garcia de Lomas, and L. Aguilar,\nand The Spanish Surveillance Group for Respiratory Pathogens. 1999. An-\ntimicrobial resistance of 1,113 Streptococcus pneumoniae isolates from pa-\ntients with respiratory tract infections in Spain: results of a 1-year (1996-\n1997) multicenter surveillance study. Antimicrob. Agents Chemother 43:\n357–359.\n9. de Lencastre, H., I. Santos Sanches, A. Brito-Avoˆ, R. Sa´-Lea˜o, J. Saldanha,\nK. G. Kristinsson, and A. Tomasz. 1999. Carriage and antibiotic resistance of\nrespiratory pathogens and molecular epidemiology of antibiotic-resistant\nStreptococcus pneumoniae colonizing children in day-care centers in Lisbon:\nthe Portuguese day-care center initiative. Clin. Microbiol. Infect. 5:S55–S63.\n10. Di Fabio, J. L., E. Castaneda, C. I. Agudelo, F. De La Hoz, M. Hortal, T.\nCamou, G. Echaniz-Aviles, M. Noemi, C. Barajas, I. Heitmann, J. C.\nHormazabal, M. C. Brandileone, V. S. Dias Vieira, M. Regueira, R. Ruvin-\nski, A. Corso, M. Lovgren, J. A. Talbot, C. De Quadros, et al. 2001. Evolution\nof Streptococcus pneumoniae serotypes and penicillin susceptibility in Latin\nAmerica, Sireva-Vigia Group, 1993 to 1999. Pediatr. Infect. Dis. J. 20:959–\n967.\n16. Felmingham, D., R. R. Reinert, Y. Hirakata, and A. Rodloff. 2002. Increasing\nprevalence of antimicrobial resistance among isolates of Streptococcus pneu-\nmoniae from the PROTEKT surveillance study, and comparative in vitro\nactivity of the ketolide, telithromycin. J. Antimicrob. Chemother. 50:25–37.\n17. Garcia-Rey, C., E. Bouza, L. Aguilar, J. Garcia-de-Lomas, and F. Baquero.\n2003. Evolution of penicillin and erythromycin co-resistance in Streptococcus\npneumoniae in Spain. Int. J. Antimicrob. Agents 22:541–544.\n22. Henriques Normark, B., B. Christensson, A. Sandgren, B. Noreen, S. Sylvan,\nL. G. Burman, and B. Olsson-Liljequist. 2003. Clonal analysis of Streptococ-\ncus pneumoniae nonsusceptible to penicillin at day-care centers with index\ncases, in a region with low incidence of resistance: emergence of an invasive\ntype 35B clone among carriers. Microb. Drug Resist. 9:337–344.\n32. Sahm, D. F., J. A. Karlowsky, L. J. Kelly, I. A. Critchley, M. E. Jones, C.\nThornsberry, Y. Mauriz, and J. Kahn. 2001. Need for annual surveillance of\nantimicrobial resistance in Streptococcus pneumoniae in the United States:\n2-year longitudinal analysis. Antimicrob. Agents Chemother. 45:1037–1042.\n33. Sa´-Lea˜o, R., A. Tomasz, and H. de Lencastre. 2001. Multilocus sequence\ntyping of Streptococcus pneumoniae clones with unusual drug resistance\npatterns: genetic backgrounds and relatedness to other epidemic clones.\nJ Infect. Dis. 184:1206–1210.\n35. Sa´-Lea˜o, R., A. Tomasz, I. S. Sanches, S. Nunes, C. R. Alves, A. B. Avoˆ, J.\nSaldanha, K. G. Kristinsson, and H. de Lencastre. 2000. Genetic diversity\nand clonal patterns among antibiotic-susceptible and -resistant Streptococcus\npneumoniae colonizing children: day care centers as autonomous epidemio-\nlogical units. J. Clin. Microbiol. 38:4137–4144.\n39. Song, J. H., N. Y. Lee, S. Ichiyama, R. Yoshida, Y. Hirakata, W. Fu, A.\nChongthaleong, N. Aswapokee, C. H. Chiu, M. K. Lalitha, K. Thomas, J.\nPerera, T. T. Yee, F. Jamal, U. C. Warsa, B. X. Vinh, M. R. Jacobs, P. C.\nAppelbaum, and C. H. Pai. 1999. Spread of drug-resistant Streptococcus\npneumoniae in Asian countries: Asian Network for Surveillance of Resistant\nPathogens (ANSORP) Study. Clin. Infect. Dis. 28:1206–1211.\n43. Thornsberry, C., D. F. Sahm, L. J. Kelly, I. A. Critchley, M. E. Jones, A. T.\nEvangelista, and J. A. Karlowsky. 2002. Regional trends in antimicrobial\nresistance among clinical isolates of Streptococcus pneumoniae, Haemophilus\ninﬂuenzae, and Moraxella catarrhalis in the United States: results from the\nTRUST Surveillance Program, 1999-2000. Clin. Infect. Dis. 34:S4–S16.\n44. Tsolia, M., G. Kouppari, A. Zaphiropoulou, S. Gavrili, M. Tsirepa, D.\nKafetzis, and T. Karpathios. 1999. Prevalence and patterns of resistance of\nStreptococcus pneumoniae strains isolated from carriers attending day care\ncenters in the area of Athens. Microb. Drug Resist. 5:271–278.",
         "S. Nunes,R. Sá-Leão,João André Carriço,C. Alves,R. Mato,A. B. Avô,J. Saldanha,J. Almeida,I. Sanches,H. de Lencastre",
         null
        ],
        [
         "14",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "Brucellae are class 3 organisms and potential agents\nof  bioterrorism.  Because  of  effective  public  health  meas-\nures, brucellosis has become a rare disease in industrial-\nized  countries,  and  clinical  microbiology  laboratories  are\nfrequently unfamiliar with the genus. A low index of suspi-\ncion by physicians or failure to notify the laboratory, equiv-\nocal  Gram-stain  results,  misidentification  of  the  organism\nby  commercial  systems,  unsafe  laboratory  practices,  and\nlaboratory accidents have been responsible for numerous\ncases of exposure to the organism and laboratory-acquired\ndisease in recent years. Discovery of a laboratory exposure\nto  brucellae  should  prompt  an  exhaustive  investigation  of\nthe event and its circumstances, definition of the population\nat  risk,  enforcement  of  safe  laboratory  practices,  and\nantimicrobial  drug  prophylaxis  for  exposed  persons.\nInadvertent exposures to brucellae in the clinical laboratory\nindicate  a  widespread  lack  of  preparedness  to  cope  with\neventual biologic threats involving use of the organism.",
         "by contact with infected animals and consumption of\ncontaminated animal products (1,2). Because of compulso-\nry pasteurization of milk products and strict control of the\ndisease  in  dairy  cattle,  the  incidence  of  brucellosis  has\nsteadily  declined  in  most  industrialized  countries  during\nthe last 50 years. However, the disease remains among the\nmost commonly recognized causes of laboratory-transmit-\nted infections; 2% of all brucellosis cases are laboratory-\nacquired (1,3–9).\nThe Organism\nSeveral  biologic  characteristics  make  brucellae  easily\ntransmissible within the close confinement of the clinical\nmicrobiology laboratory, including the facts that the infect-\ning dose for humans is low, and the organism may enter the\nbody in many ways relevant to laboratory practices (e.g.,\nthrough  the  respiratory  mucosa,  conjunctivae,  gastroin-\ntestinal tract, or abraded skin) (1,2). Rare cases of acquisi-\ntion of the organism through organ transplantation, sexual\ncontact,  breastfeeding,  or  the  transplacental  route  have\nalso been reported (1). Because person-to-person transmis-\nsion does not occur, infected persons do not pose a threat\nto their surroundings.\nreproducing\nSoon  after  entry  into  the  body  through  the  skin  or\nmucous  membranes,  brucellae  are  ingested  by  polymor-\nphonuclear and mononuclear phagocytes. The organism is\nable to escape phagocytic killing by inhibiting the phago-\nsome-lysosome\ninside\nmacrophages  (1,10).  After  a  variable  incubation  period\nranging  from  <1  week  to  several  months  (usually  2–4\nweeks),  nonspecific  systemic  symptoms  such  as  fever,\nheadache,  malaise,  night  sweats,  and  arthralgia  follow,\nresembling a flulike disease (1,2). During the early stages\nof  the  disease,  patients  are  frequently  bacteremic.  This\nbacteremia  has  a  continuous  pattern,  making  circulating\nbrucellae  easily  detectable  by  blood  culture.  Once  in  the\nbloodstream, the organism is seeded to multiple organ sys-\ntems and especially to those rich in reticuloendothelial tis-\nsue,  such  as  the  liver,  spleen,  and  the  skeletal  and\nhematopoietic systems, where it may cause localized dis-\nease such as hepatitis or arthritis (1,2).\nBecause  of  the  variable  manifestations  of  human  bru-\ncellosis, a wide array of different clinical specimens may\ncontain viable brucellae, including pus, blood, bone mar-\nrow,  synovial  fluid  and  tissues,  and  more  rarely,  cere-\nbrospinal  fluid,  urine,  and  genital  exudates.  The\nconcentration  of Brucella organisms  in  the  blood  (11,12)\nand synovial fluid (13) of patients with brucellosis is usu-\nally low, and therefore, these clinical specimens probably\npose  a  low  risk  for  contagion  for  laboratory  personnel.\nHowever,  the  danger  of  clinically  relevant  exposure\nincreases exponentially after incubation of both solid and\nliquid media. Seeded media harbor considerable amounts\nof  viable Brucella organisms,  and  routine  bacteriologic\nprocedures  such  as  preparing,  centrifuging,  and  vigorous\nagitation (vortexing) of bacterial suspensions, performing\nsubcultures  and  biochemical  testing,  and  particularly  the\ncatalase test, may create dangerous aerosols and the poten-\ntial for accidental spillage (14).\nMost  cases  of  laboratory-acquired  brucellosis  have\nbeen  caused  by  the  more  virulent  Brucella  melitensis\nspecies  (3,4,6,8,14–29).  B.  suis (4,6),  B.  abortus (3,6,8),\nand B. canis (30) have also been implicated, and transmis-\nsion of the attenuated B. abortus 19 and B. melitensis Rev-\n1 vaccine strains has also been reported (31,32).\nClinical Manifestations of\nLaboratory-acquired Infections\nPatients involved in laboratory outbreaks of brucellosis\nhave shown almost the entire range of clinical manifesta-\ntions of the disease, ranging from the common prolonged\nfebrile  syndrome  (undulant  fever)  (14)  and  a  flulike  dis-\nease (25,27), to focal signs and symptoms, such as hepati-\ntis (17,33), lymphadenopathy (17,25), uveitis (14), breast\nabscess  (28)  epididymitis  (29),  arthritis  (17),  discitis\n(29,34), pneumonitis (17), deep vein thrombosis (29), and\nmeningitis  (31).  Deaths  are  rare  (<5%)  even  among\nuntreated  patients  with  brucellosis,  but  5  fatal  cases\noccurred among the 426 persons with laboratory-acquired\ndisease summarized by Pike in 1978 (5).\nLaboratory Exposure to Brucella Organisms\nBecause  of  effective  control  measures  in  animals  and\nanimal  products,  brucellosis  has  been  almost  eradicated\nfrom  most  industrialized  countries,  where  the  disease  is\nusually limited to persons who have traveled to developing\ncountries  or  ingested  imported  contaminated  food  (14).\nHuman brucellosis has become so rare in the United States\nthat <300 cases have been reported annually in the last 4\ndecades (9,15) and ≈100 per year in the last 10 years (2).\nBecause of the rare occurrence of brucellosis, technicians\nworking in industrialized countries have become unfamil-\niar with the staining and other phenotypic characteristics of\nthe  organism.  In  addition,  physicians  frequently  do  not\nconsider  the  diagnosis  of  brucellosis  or  fail  to  communi-\ncate their suspicion to the clinical microbiology laborato-\nry, which results in inadvertent handling of cultures on an\nopen bench.\nAlthough  the  identification  of  the  genus  Brucella is\nstraightforward  (small  gram-negative  coccobacilli;  posi-\ntive oxidase, catalase, and urease test results; no sugar fer-\nmentation;  and  requirement  of  aerobic  conditions  and\nadded CO2 for its growth), laboratory-acquired disease has\nfrequently resulted from misidentification of the organism\n(35). In 2 published outbreaks, the Brucella isolate resisted\ndecoloration  and  appeared  as  a  gram-positive  or  gram-\nvariable  coccobacillus  and  thus  was  misidentified  as\nmicrococcus  or  a  coryneform  bacillus  (14,29).  In  other\ncases,  B.  melitensis organisms  tested  with  the  API20NE\n(bioMérieux,  Marcy  l’Etoile,  France)  identification  kit\nproduced  the  1200004  or  1000004  biochemical  profile\nleading  to  misidentification  of  the  isolate  as  Moraxella\nphenylpyruvica at  a  good  identification  level  (90.5%)\n(8,18–21). The  frequent  failure  of  clinical  laboratories  to\ncorrectly identify isolates as brucellae is particularly wor-\nrisome because these organisms are regarded as potential\nagents for bioterrorism. Brucellae are inexpensive to pro-\nduce and disperse, and transmission to humans may result\nin prolonged illness and long-term sequelae. These organ-\nisms  are  considered  category  B  select  agents  (2). The\noccurrence in the last few years of inadvertent exposures to\nbrucellae  in  laboratories  in  industrialized  countries\n(7,8,14,17–26)  indicates  a  lack  of  preparedness  to  deal\nwith a real biologic threat. Lack of recognition of an iso-\nlate as a Brucella sp. by a laboratory may enable a bioter-\nrorism-related  attack  to  go  undetected,  whereas  a\nfalse-positive identification may cause unnecessary alarm.\nIn laboratories serving countries in which human bru-\ncellosis is still endemic, the degree of potential exposure to\nthe  organism  may  be  extremely  high.  In  a  microbiology\nlaboratory in Ankara, Turkey, the average annual number\nof cultures positive for Brucella spp. reaches 400 (34). A\nlaboratory serving a brucellosis-endemic area in southern\nIsrael processes ≈150 per year; at the peak of the Brucella\nseason  (April–June),  10%  of  all  positive  blood  cultures\ngrow B. melitensis (27). The risk for exposure in develop-\ning  countries  is  frequently  aggravated  by  lack  of  safety\nequipment and inadequate laboratory facilities (34). In the\naforementioned Turkish laboratory, the disease affected 10\n(18%) of 55 laboratory workers, and the calculated hazard\nwas 8% per employee-year (34).\nthe\nThe probable source of the infection is usually apparent\nwhen disease occurs in laboratories where isolation of bru-\ncellae  is  rare  (21).  In  laboratories  serving  brucellosis-\ntime  and  circumstances  of\nendemic  areas,\ncontamination  are  more  difficult  to  trace  because  of  the\nexistence  of  multiple  potential  sources  (22,27). Although\nthe exact form of transmission often remains speculative,\naerosols have been implicated in most cases (24,27,29,32).\nThis  assumption  has  been  strengthened  by  spread  of  the\ndisease  to  distant  areas  through  common  ventilation\n(27,32).\nLaboratory  accidents  such  as  breakage  of  centrifuge\ntubes (7) or blood-culture vials (16) play a minor role in\nlaboratory-acquired  disease  and  are  responsible  for  only\n20% of cases (6). More commonly, exposures are the result\nof  unsafe  laboratory  practices,  such  as  sniffing  plates\n(8,24,26,29,34);  working  on  an  open  bench  with  viable\norganisms  (14,16,17,20,22);  not  using  protective  equip-\nment such as gloves, masks, and goggles (34); or ingesting\nsuspensions of living brucellae during mouth pipetting (3).\nPERSPECTIVE\nHowever, even when no apparent breach in safety proce-\ndures  is  apparent,  transmission  may  occur  (17).  Rare\nevents—such as self-inoculation of brucellae by syringes\nloaded with a suspension of the organism or with synovial\nfluid from an infected patient (3,16), injury to the conjunc-\ntiva with a broken tube that had contained a living culture\n(3), or participation in a laboratory exercise in which still-\nviable  organisms  were  inadvertently  used  (15)—are  also\nresponsible  for  a  small  number  of  laboratory  exposures\nand outbreaks of disease.\nMost  cases  of  laboratory-acquired  brucellosis  have\noccurred  in  clinical  laboratories,  but  transmission  of  the\ndisease  in  research  facilities  (4,25)  and  laboratories  that\nmanufactured Brucella vaccines has also been document-\ned  (21,22).  A laboratory  technician  at  the  Centers  for\nDisease  Control  and  Prevention  (CDC)  became  infected\nwhile working in a safety cabinet with a B. melitensis iso-\nlate that had originally caused an outbreak of laboratory-\nacquired  disease  in  a  community  hospital  (17).  In  a\nseparate episode, the organism isolated from a microbiolo-\ngy technologist with laboratory-acquired disease infected a\nlaboratory worker at the hospital to which the technologist\nhad been admitted (14).\nThe attack rate of laboratory-associated infections has\nranged  from  30%  to  100%  depending,  among  other  fac-\ntors, on the location of workers, whether aerosol-generat-\ning  procedures  have  been  performed,  and\nthe\nconcentration  of  microorganisms  in  the  contaminated\nmedia  (7,8).  However,  the  hazard  of  transmission  is  not\nlimited to persons who worked with the isolate. Among the\n74 reported cases of laboratory-acquired brucellosis from\n1897  to  1939  reviewed  by  Meyer  and  Eddie,  the  disease\nalso affected janitors and occasional visitors (3). In a large\noutbreak of laboratory-acquired brucellosis, 3 of 7 cases of\ndisease  occurred  among  persons  who  briefly  visited  the\nfacility but did not enter the room where Brucella cultures\nwere processed (27).\nInvestigating Outbreaks of Brucellosis\nRecognizing sporadic cases and even outbreaks of lab-\noratory-acquired brucellosis is not always easy because the\ndisease lacks distinctive clinical features (14,17) and has a\nwidely variable incubation period (7,17,19,32). In a cluster\nof disease that originated in a single exposure, the onset of\nsymptoms of the 7 affected laboratory workers spanned 5\nmonths (17). Although a temporal clustering of cases usu-\nally  suggests  a  large  exposure  from  a  pinpoint  source\n(7,32),  this  is  not  always  the  case.  When  biotyping  was\nperformed in the isolates recovered within a 5-week peri-\nod  from  7  infected  hospital  workers  in  southern  Israel,  3\ndistinct  strains  were  recognized,  demonstrating  that  the\noutbreak was caused by at least 3 separate exposures (27).\nGuidelines for Investigating Outbreaks\nIn-depth  investigation  of  laboratory-acquired  cases  of\nbrucellosis may lead to the identification of unsafe bacte-\nriologic  practices,  suitable  for  correction  by  educational\nand technical measures. To guide the investigation, the fol-\nlowing  recommendations  are  made.  1)  Send  isolates  to  a\nreference  laboratory  for  confirmation.  Organisms  should\nbe shipped as “dangerous goods,” according to the guide-\nlines established by the World Health Organization and the\nOffice  of  Biosafety  at  CDC  (36).  In  the  United  States,\norganisms presumptively identified as brucellae should be\nsent to CDC or another public health laboratory following\nthe specific guidelines for transferring “select agents.” 2)\nInform infection control services and public health author-\nities,  who  may  choose  to  involve  CDC.  3)  Conduct  a\nmeticulous epidemiologic investigation. 4) Determine the\ndate  and  circumstances  of  the  exposure  (17).  5)  Exclude\nother  potential  sources  of  transmission,  such  as  previous\nlaboratory exposures, travels to brucellosis-endemic areas,\nconsumption  of  unpasteurized  dairy  products,  and  han-\ndling  of  farm  or  laboratory  animals  (14,15,17,27,29).  6)\nKeep all relevant data and records. 7) Define the exposed\npopulation. 8) Determine the level of risk on the basis of\ntype of laboratory procedures performed, proximity to the\nsource,  duration  of  the  exposure,  and  the  like,  to  define\npersons  to  whom  postexposure  prophylaxis  should  be\noffered (8,32). 9) Check biologic safety cabinets (27,29).\n10) Check the ventilation system (27). 11) Collect baseline\nserum  samples  from  all  known  potentially  exposed  per-\nsons  (17,27).  12)  Freeze  isolates  for  future  typing,  espe-\ncially in brucellosis-endemic areas (23,27).\nPrevention of Laboratory-acquired Cases\nBrucellae  are  considered  class  3  organisms.  CDC  has\nstrongly recommended that live Brucella cultures and sus-\npicious  organisms  be  manipulated  in  a  class  II  biologic\nsafety  cabinet  (37).  This  recommendation,  however,  is\nclearly insufficient for preventing laboratory-acquired dis-\nease because by the time the organism is suspected or con-\nfirmed as Brucella, exposure of laboratory personnel may\nhave occurred (27). On the other hand, converting a large\nand busy clinical microbiology laboratory into a biosafety\nlevel III facility, where all specimens are handled in biolog-\nic safety cabinets (32), is both impractical and unnecessary,\nespecially in laboratories in areas where the disease is not\nendemic. However, in regions where brucellosis is highly\nprevalent, an enhanced safety policy should be adopted. All\nblood  culture  vials  detected  as  positive  by  the  automated\nblood culture system, as well as all bone marrow and syn-\novial  fluid  specimens,  should  be  manipulated  in  biologic\nsafety hoods until the isolated microorganisms are defini-\ntively determined to be other than Brucella sp. (27).\nFor  laboratories  serving  areas  where  brucellosis  is\nuncommon, the following recommendations are made. 1)\nBecause of the low incidence of brucellosis, physicians in\nareas not endemic for the disease are unfamiliar with the\nclinical and epidemiologic features of human brucellosis,\nand the possibility of brucellosis is rarely considered in the\ndifferential diagnosis. Periodic education of physicians on\nthis  subject  is  indicated.  2)  Communication  between\nattending  physicians  and  the  laboratory  should  be\nimproved. The clinical microbiology laboratory should be\ninformed  in  advance  when  clinical  specimens  had  been\nobtained  from  patients  with  risk  factors  for  brucellosis,\nsuch  as  recent  history  of  travel  to  brucellosis-endemic\nareas,  consumption  of  local  or  imported  unpasteurized\ndairy products, or professional exposures in veterinarians,\nshepherds,  slaughterhouse  employees,  and  laboratory\nworkers. 3) The use of automated, continuous monitoring\nblood culture systems for patients with suspected brucel-\nlosis  should  be  preferred  over  the  lysis-centrifugation\nmethod  because  the  latter  involves  centrifuging  clinical\nspecimens  and  visually  inspecting  plates  to  detect  the\norganism and probably increases the risk for transmission\n(11,27). Although the use of blood lysis-based methods has\nbeen  advocated  in  the  past  for  improving  detection  of\nBrucella bacteremia, modern automated blood culture sys-\ntems are faster and more sensitive (11,12). 4) The familiar-\nity of laboratory technicians with the characteristics of the\norganism,  as  well  as  with  the  safe  handling  of  cultures,\nshould be improved and maintained through periodic edu-\ncation.  The  reader  is  referred  to  the  excellent  review  by\nGilligan  and York  for  handling  and  identifying  presump-\ntive Brucella organisms (35). 5) Standard precautions and\nstrict adherence to good laboratory practices must be com-\npletely  adopted,  reinforced,  and  regularly  monitored.  6)\nAll work with gram-negative or gram-variable small rods\nor coccobacilli isolated from tissues, blood, bone marrow,\nbone, or synovial fluid exudates should be carried out in a\nbiosafety  cabinet  until  Brucella has  been  ruled  out.  7)\nPlates  should  be  sealed  for  safety  when  not  in  use  and\nappropriately disposed and sterilized as soon as they are no\nlonger being actively used (17). 8) Because brucellae are\nrelatively slow-growing bacteria, cultures for the organism\nhave been traditionally kept for several weeks. However,\nmodern  blood  culture  systems  enable  brucellae  to  be\ndetected within the routine 5-day incubation period insti-\ntuted  in  most  clinical  laboratories  (12). Therefore,  safety\nprecautions should not be limited to organisms that tend to\ngrow slowly (8,18,20,26). 9) Antimicrobial drug–suscepti-\nbility  testing  of  Brucella  organisms  is  not  indicated\nbecause the therapeutic regimen for brucellosis is standard,\nand  the  organism  does  not  usually  acquire  antimicrobial\nresistance.  Performance  of  this  and  other  unnecessary\ntests,  and  especially  of  laboratory  procedures  known  to\nproduce aerosols, should be strongly discouraged (27). 10)\nIf a suspension of living brucellae is spilled and the organ-\nism is recognized, the entire laboratory should be immedi-\nately  evacuated,  doors  should  be  shut,  and  an  effective\ngermicide  such  as  3%  phenol  or  10%  bleach  should  be\napplied  by  a  trained  person  wearing  a  safety  mask,  gog-\ngles, an impermeable laboratory gown, and gloves (7).\nPostexposure Prophylaxis\nBecause  of  ethical  considerations,  the  heterogeneous\nnature of the events leading to Brucella exposure, difficul-\nties in determining the actual risk for individual workers,\nlate recognition of outbreaks, and the small number of per-\nsons involved in each outbreak, no controlled studies have\nbeen performed to assess the value of administering post-\nexposure  prophylaxis  to  persons  at  risk.  However,  anec-\ndotal  evidence  suggests  that  administering  prophylactic\nantimicrobial drug therapy may reduce the risk of develop-\ning clinical disease (7,8,26).\nIn a recently reported event, an isolate from a chest wall\nexudate  culture  from  an  Indian  patient,  which  was  origi-\nnally identified as M. phenylpyruvica, was correctly recog-\nnized  as  B.  melitensis 22  days  after  the  specimen  was\nobtained (8). By that time, 26 laboratory workers had been\npotentially  exposed,  and  6  had  actually  manipulated  the\norganism. These 6 workers were considered to be at high\nrisk and offered a 3-week prophylactic course of combined\ndoxycycline-rifampin  or  trimethoprim-sulfamethoxazole\ntherapy.  None  of  the  5  laboratory  technologists  who\nreceived  postexposure  antimicrobial  drugs  became  ill  or\ndeveloped an antibody response. However, Brucella bac-\nteremia  developed  in  the  only  laboratory  worker  who\nrefused therapy, and she seroconverted 10 weeks after the\nspecimen  was  received  (8).  Neither  clinical  disease  nor\nseroconversion developed in the remaining 19 laboratory\nworkers who were only present in the laboratory but were\nnot in direct contact with the organism (8).\nIn  another  event,  3  laboratory  technologists  who  had\ninadvertently  worked  with  a  B.  melitensis isolate  on  an\nopen bench and sniffed the plates were given 1 week of pro-\nphylactic  doxycycline  within  24  hours  of  the  exposure;\nnone of them became ill or seroconverted (26). In a large\nexposure caused by breakage of a tube containing B. abor-\ntus, patients who started therapy immediately after serocon-\nversion was detected and before symptoms developed (10\nweeks after the laboratory accident) had a benign clinical\ncourse, suggesting that even if the disease was not prevent-\ned, a certain degree of attenuation probably occurred (7).\nBecause  of  the  high  attack  rate  of  brucellosis  among\nexposed  workers,  the  unpredictable  and  often  chronic\ncourse of the disease, and the difficulties in eradicating the\norganism  once  a  symptomatic  infection  has  been  estab-\nlished, postexposure prophylaxis is probably indicated for\nall  persons  after  an  obvious  exposure  to  living  brucellae\n(26). Antimicrobial  drug  prophylaxis  with  a  combination\nof oral doxycycline 100 mg twice a day plus rifampin 600\nmg 4 × per day for 3 weeks should be started as soon as the\nexposure  to  confirmed  Brucella organisms  is  recognized\n(8). For pregnant women, administration of trimethoprim-\nsulfamethoxazole 160/800 mg 2 × per day for 3 weeks has\nbeen advocated (8).\nPostexposure Follow-up\nWhether  exposed  persons  have  received  prophylactic\ntherapy or not, increased surveillance for clinical signs of\ndisease should be conducted for at least 6 months. In addi-\ntion,  exposed  laboratory  workers  should  be  followed  for\npossible subclinical infections or early signs of disease by\nperiodic serologic testing. Weekly or semiweekly serolog-\nic surveillance is recommended for the first 3 months and\nonce  a  month  thereafter  for  3  to  9  additional  months  to\ndetect late infections resulting from prolonged incubation\n(7,8).\nHuman brucellosis has variable clinical manifestations\nand may mimic other infections, particularly influenza, as\nwell as noninfectious conditions. Therefore, the diagnosis\nof  the  disease  requires  a  high  index  of  suspicion.\nEducation  of  exposed  personnel  on  the  symptoms  of  the\ndisease and the need for periodic and timely serologic fol-\nlow-up are particularly important. Increased vigilance dur-\ning  the  flu  season  employing  a  broad  case  definition  is\nclearly  needed,  and  administration  of  influenza  vaccine\nshould be strongly recommended to all Brucella-exposed\npersons. If symptoms develop, blood cultures and cultures\nof other normally sterile body fluids, as clinically indicat-\ned, should be obtained, and a new antibody titer should be\ndetermined.  Although  use  of  nucleic  acid  amplification\nmethods  has  been  shown  to  enable  early  detection  of\ninfected persons (38), these tests are not yet commercially\navailable.\nTreating Infected Persons and\nPosttreatment Follow-up\nPersons  in  whom  culture-confirmed  or  serologically\nproven disease develops should receive therapy with oral\ndoxycycline and rifampin for 6 weeks, or a combination of\noral doxycycline for 6 weeks and an intramuscular amino-\nglycoside  (gentamicin  or  streptomycin)  for  the  first  2\nweeks (2,7). Use of 3 drugs is usually reserved for compli-\ncated cases or life-threatening clinical manifestations, such\nas endocarditis or meningitis (1). Pregnant women should\nreceive  trimethoprim-sulfamethoxazole  160/800  mg  2  ×\nper day for 6 weeks (1).\nEven  when  patients  are  appropriately  treated,  the  risk\nfor  relapse  remains  high  (≈20%)  (39,40).  Therefore,\npatients who have completed a full therapeutic antimicro-\nbial course should be followed clinically and serologically\nfor  1  year.  Failure  to  show  declining  antibody  titers  may\nindicate incomplete cure. If symptoms consistent with bru-\ncellosis develop, blood cultures and serologic tests should\nbe performed to detect relapses of the disease (39). Use of\nnucleic acid amplification methods in the future may fur-\nther improve the detection of patients in whom the organ-\nism was not eliminated (40).\nAlthough human brucellosis has been eradicated from\nmost  industrialized  countries,  isolated  cases  of  disease,\nusually  related  to  travel  or  import  of  contaminated  food\nfrom disease-endemic areas, continue to occur. Because of\nthe  low  incidence  of  disease  in  industrialized  countries,\nclinical  laboratory  technologists  have  become  unfamiliar\nwith  identifying  and  handling  Brucella species.  Unsafe\nlaboratory  practices  while  manipulating  Brucella isolates\nhave resulted in inadvertent exposure to the organism and\nmany cases of laboratory-acquired disease, indicating lack\nof preparedness to cope with bioterrorism threats involv-\ning  brucellae.  Education  of  laboratory  personnel  on  the\nidentification  of  Brucella  species,  adherence  to  and\nenforcement  of  standard  precautions,  thorough  investiga-\ntion of laboratory exposures, administration of prophylac-\ntic  antimicrobial  drug  therapy,  and  close  follow-up  of\ndirectly exposed persons are strongly recommended.\nDr.  Yagupsky  is  the  director  of  the  Clinical  Microbiology\nLaboratory  of  the  Soroka  University  Medical  Center,  Beer-\nSheva, Israel. His main research areas are Kingella kingae infec-\ntions and diagnosis of human brucellosis.\nDr.  Baron  is  director  of  the  Clinical  Microbiology\nLaboratory at Stanford University Medical Center and professor\nof pathology at Stanford University Medical College. Her inter-\nests include the development of rapid infectious disease diagnos-\ntic  tests  and  educating  both  scientists  and  the  public  about\nrelevant aspects of infectious diseases.\n22. Martin-Mazuelos  E,  Nogales  MC,  Florez  C,  Gomez-Mateos  JM,\nLozano F, Sanchez A. Outbreak of Brucella melitensis among micro-\nbiology laboratory workers. J Clin Microbiol. 1994;32:2035–6.\n23. Gruner E, Bernasconi E, Galeazzi RL, Buhl D, Heinzle R, Nadal D.\nBrucellosis; an occupational hazard for medical laboratory personnel.\nReport of five cases. Infection. 1994;22:33–6.",
         "",
         "",
         "",
         "Brucellae are class 3 organisms and potential agents\nof  bioterrorism.  Because  of  effective  public  health  meas-\nures, brucellosis has become a rare disease in industrial-\nized  countries,  and  clinical  microbiology  laboratories  are\nfrequently unfamiliar with the genus. A low index of suspi-\ncion by physicians or failure to notify the laboratory, equiv-\nocal  Gram-stain  results,  misidentification  of  the  organism\nby  commercial  systems,  unsafe  laboratory  practices,  and\nlaboratory accidents have been responsible for numerous\ncases of exposure to the organism and laboratory-acquired\ndisease in recent years. Discovery of a laboratory exposure\nto  brucellae  should  prompt  an  exhaustive  investigation  of\nthe event and its circumstances, definition of the population\nat  risk,  enforcement  of  safe  laboratory  practices,  and\nantimicrobial  drug  prophylaxis  for  exposed  persons.\nInadvertent exposures to brucellae in the clinical laboratory\nindicate  a  widespread  lack  of  preparedness  to  cope  with\neventual biologic threats involving use of the organism.\nby contact with infected animals and consumption of\ncontaminated animal products (1,2). Because of compulso-\nry pasteurization of milk products and strict control of the\ndisease  in  dairy  cattle,  the  incidence  of  brucellosis  has\nsteadily  declined  in  most  industrialized  countries  during\nthe last 50 years. However, the disease remains among the\nmost commonly recognized causes of laboratory-transmit-\nted infections; 2% of all brucellosis cases are laboratory-\nacquired (1,3–9).\nThe Organism\nSeveral  biologic  characteristics  make  brucellae  easily\ntransmissible within the close confinement of the clinical\nmicrobiology laboratory, including the facts that the infect-\ning dose for humans is low, and the organism may enter the\nbody in many ways relevant to laboratory practices (e.g.,\nthrough  the  respiratory  mucosa,  conjunctivae,  gastroin-\ntestinal tract, or abraded skin) (1,2). Rare cases of acquisi-\ntion of the organism through organ transplantation, sexual\ncontact,  breastfeeding,  or  the  transplacental  route  have\nalso been reported (1). Because person-to-person transmis-\nsion does not occur, infected persons do not pose a threat\nto their surroundings.\nreproducing\nSoon  after  entry  into  the  body  through  the  skin  or\nmucous  membranes,  brucellae  are  ingested  by  polymor-\nphonuclear and mononuclear phagocytes. The organism is\nable to escape phagocytic killing by inhibiting the phago-\nsome-lysosome\ninside\nmacrophages  (1,10).  After  a  variable  incubation  period\nranging  from  <1  week  to  several  months  (usually  2–4\nweeks),  nonspecific  systemic  symptoms  such  as  fever,\nheadache,  malaise,  night  sweats,  and  arthralgia  follow,\nresembling a flulike disease (1,2). During the early stages\nof  the  disease,  patients  are  frequently  bacteremic.  This\nbacteremia  has  a  continuous  pattern,  making  circulating\nbrucellae  easily  detectable  by  blood  culture.  Once  in  the\nbloodstream, the organism is seeded to multiple organ sys-\ntems and especially to those rich in reticuloendothelial tis-\nsue,  such  as  the  liver,  spleen,  and  the  skeletal  and\nhematopoietic systems, where it may cause localized dis-\nease such as hepatitis or arthritis (1,2).\nBecause  of  the  variable  manifestations  of  human  bru-\ncellosis, a wide array of different clinical specimens may\ncontain viable brucellae, including pus, blood, bone mar-\nrow,  synovial  fluid  and  tissues,  and  more  rarely,  cere-\nbrospinal  fluid,  urine,  and  genital  exudates.  The\nconcentration  of Brucella organisms  in  the  blood  (11,12)\nand synovial fluid (13) of patients with brucellosis is usu-\nally low, and therefore, these clinical specimens probably\npose  a  low  risk  for  contagion  for  laboratory  personnel.\nHowever,  the  danger  of  clinically  relevant  exposure\nincreases exponentially after incubation of both solid and\nliquid media. Seeded media harbor considerable amounts\nof  viable Brucella organisms,  and  routine  bacteriologic\nprocedures  such  as  preparing,  centrifuging,  and  vigorous\nagitation (vortexing) of bacterial suspensions, performing\nsubcultures  and  biochemical  testing,  and  particularly  the\ncatalase test, may create dangerous aerosols and the poten-\ntial for accidental spillage (14).\nMost  cases  of  laboratory-acquired  brucellosis  have\nbeen  caused  by  the  more  virulent  Brucella  melitensis\nspecies  (3,4,6,8,14–29).  B.  suis (4,6),  B.  abortus (3,6,8),\nand B. canis (30) have also been implicated, and transmis-\nsion of the attenuated B. abortus 19 and B. melitensis Rev-\n1 vaccine strains has also been reported (31,32).\nClinical Manifestations of\nLaboratory-acquired Infections\nPatients involved in laboratory outbreaks of brucellosis\nhave shown almost the entire range of clinical manifesta-\ntions of the disease, ranging from the common prolonged\nfebrile  syndrome  (undulant  fever)  (14)  and  a  flulike  dis-\nease (25,27), to focal signs and symptoms, such as hepati-\ntis (17,33), lymphadenopathy (17,25), uveitis (14), breast\nabscess  (28)  epididymitis  (29),  arthritis  (17),  discitis\n(29,34), pneumonitis (17), deep vein thrombosis (29), and\nmeningitis  (31).  Deaths  are  rare  (<5%)  even  among\nuntreated  patients  with  brucellosis,  but  5  fatal  cases\noccurred among the 426 persons with laboratory-acquired\ndisease summarized by Pike in 1978 (5).\nLaboratory Exposure to Brucella Organisms\nBecause  of  effective  control  measures  in  animals  and\nanimal  products,  brucellosis  has  been  almost  eradicated\nfrom  most  industrialized  countries,  where  the  disease  is\nusually limited to persons who have traveled to developing\ncountries  or  ingested  imported  contaminated  food  (14).\nHuman brucellosis has become so rare in the United States\nthat <300 cases have been reported annually in the last 4\ndecades (9,15) and ≈100 per year in the last 10 years (2).\nBecause of the rare occurrence of brucellosis, technicians\nworking in industrialized countries have become unfamil-\niar with the staining and other phenotypic characteristics of\nthe  organism.  In  addition,  physicians  frequently  do  not\nconsider  the  diagnosis  of  brucellosis  or  fail  to  communi-\ncate their suspicion to the clinical microbiology laborato-\nry, which results in inadvertent handling of cultures on an\nopen bench.\nAlthough  the  identification  of  the  genus  Brucella is\nstraightforward  (small  gram-negative  coccobacilli;  posi-\ntive oxidase, catalase, and urease test results; no sugar fer-\nmentation;  and  requirement  of  aerobic  conditions  and\nadded CO2 for its growth), laboratory-acquired disease has\nfrequently resulted from misidentification of the organism\n(35). In 2 published outbreaks, the Brucella isolate resisted\ndecoloration  and  appeared  as  a  gram-positive  or  gram-\nvariable  coccobacillus  and  thus  was  misidentified  as\nmicrococcus  or  a  coryneform  bacillus  (14,29).  In  other\ncases,  B.  melitensis organisms  tested  with  the  API20NE\n(bioMérieux,  Marcy  l’Etoile,  France)  identification  kit\nproduced  the  1200004  or  1000004  biochemical  profile\nleading  to  misidentification  of  the  isolate  as  Moraxella\nphenylpyruvica at  a  good  identification  level  (90.5%)\n(8,18–21). The  frequent  failure  of  clinical  laboratories  to\ncorrectly identify isolates as brucellae is particularly wor-\nrisome because these organisms are regarded as potential\nagents for bioterrorism. Brucellae are inexpensive to pro-\nduce and disperse, and transmission to humans may result\nin prolonged illness and long-term sequelae. These organ-\nisms  are  considered  category  B  select  agents  (2). The\noccurrence in the last few years of inadvertent exposures to\nbrucellae  in  laboratories  in  industrialized  countries\n(7,8,14,17–26)  indicates  a  lack  of  preparedness  to  deal\nwith a real biologic threat. Lack of recognition of an iso-\nlate as a Brucella sp. by a laboratory may enable a bioter-\nrorism-related  attack  to  go  undetected,  whereas  a\nfalse-positive identification may cause unnecessary alarm.\nIn laboratories serving countries in which human bru-\ncellosis is still endemic, the degree of potential exposure to\nthe  organism  may  be  extremely  high.  In  a  microbiology\nlaboratory in Ankara, Turkey, the average annual number\nof cultures positive for Brucella spp. reaches 400 (34). A\nlaboratory serving a brucellosis-endemic area in southern\nIsrael processes ≈150 per year; at the peak of the Brucella\nseason  (April–June),  10%  of  all  positive  blood  cultures\ngrow B. melitensis (27). The risk for exposure in develop-\ning  countries  is  frequently  aggravated  by  lack  of  safety\nequipment and inadequate laboratory facilities (34). In the\naforementioned Turkish laboratory, the disease affected 10\n(18%) of 55 laboratory workers, and the calculated hazard\nwas 8% per employee-year (34).\nthe\nThe probable source of the infection is usually apparent\nwhen disease occurs in laboratories where isolation of bru-\ncellae  is  rare  (21).  In  laboratories  serving  brucellosis-\ntime  and  circumstances  of\nendemic  areas,\ncontamination  are  more  difficult  to  trace  because  of  the\nexistence  of  multiple  potential  sources  (22,27). Although\nthe exact form of transmission often remains speculative,\naerosols have been implicated in most cases (24,27,29,32).\nThis  assumption  has  been  strengthened  by  spread  of  the\ndisease  to  distant  areas  through  common  ventilation\n(27,32).\nLaboratory  accidents  such  as  breakage  of  centrifuge\ntubes (7) or blood-culture vials (16) play a minor role in\nlaboratory-acquired  disease  and  are  responsible  for  only\n20% of cases (6). More commonly, exposures are the result\nof  unsafe  laboratory  practices,  such  as  sniffing  plates\n(8,24,26,29,34);  working  on  an  open  bench  with  viable\norganisms  (14,16,17,20,22);  not  using  protective  equip-\nment such as gloves, masks, and goggles (34); or ingesting\nsuspensions of living brucellae during mouth pipetting (3).\nPERSPECTIVE\nHowever, even when no apparent breach in safety proce-\ndures  is  apparent,  transmission  may  occur  (17).  Rare\nevents—such as self-inoculation of brucellae by syringes\nloaded with a suspension of the organism or with synovial\nfluid from an infected patient (3,16), injury to the conjunc-\ntiva with a broken tube that had contained a living culture\n(3), or participation in a laboratory exercise in which still-\nviable  organisms  were  inadvertently  used  (15)—are  also\nresponsible  for  a  small  number  of  laboratory  exposures\nand outbreaks of disease.\nMost  cases  of  laboratory-acquired  brucellosis  have\noccurred  in  clinical  laboratories,  but  transmission  of  the\ndisease  in  research  facilities  (4,25)  and  laboratories  that\nmanufactured Brucella vaccines has also been document-\ned  (21,22).  A laboratory  technician  at  the  Centers  for\nDisease  Control  and  Prevention  (CDC)  became  infected\nwhile working in a safety cabinet with a B. melitensis iso-\nlate that had originally caused an outbreak of laboratory-\nacquired  disease  in  a  community  hospital  (17).  In  a\nseparate episode, the organism isolated from a microbiolo-\ngy technologist with laboratory-acquired disease infected a\nlaboratory worker at the hospital to which the technologist\nhad been admitted (14).\nThe attack rate of laboratory-associated infections has\nranged  from  30%  to  100%  depending,  among  other  fac-\ntors, on the location of workers, whether aerosol-generat-\ning  procedures  have  been  performed,  and\nthe\nconcentration  of  microorganisms  in  the  contaminated\nmedia  (7,8).  However,  the  hazard  of  transmission  is  not\nlimited to persons who worked with the isolate. Among the\n74 reported cases of laboratory-acquired brucellosis from\n1897  to  1939  reviewed  by  Meyer  and  Eddie,  the  disease\nalso affected janitors and occasional visitors (3). In a large\noutbreak of laboratory-acquired brucellosis, 3 of 7 cases of\ndisease  occurred  among  persons  who  briefly  visited  the\nfacility but did not enter the room where Brucella cultures\nwere processed (27).\nInvestigating Outbreaks of Brucellosis\nRecognizing sporadic cases and even outbreaks of lab-\noratory-acquired brucellosis is not always easy because the\ndisease lacks distinctive clinical features (14,17) and has a\nwidely variable incubation period (7,17,19,32). In a cluster\nof disease that originated in a single exposure, the onset of\nsymptoms of the 7 affected laboratory workers spanned 5\nmonths (17). Although a temporal clustering of cases usu-\nally  suggests  a  large  exposure  from  a  pinpoint  source\n(7,32),  this  is  not  always  the  case.  When  biotyping  was\nperformed in the isolates recovered within a 5-week peri-\nod  from  7  infected  hospital  workers  in  southern  Israel,  3\ndistinct  strains  were  recognized,  demonstrating  that  the\noutbreak was caused by at least 3 separate exposures (27).\nGuidelines for Investigating Outbreaks\nIn-depth  investigation  of  laboratory-acquired  cases  of\nbrucellosis may lead to the identification of unsafe bacte-\nriologic  practices,  suitable  for  correction  by  educational\nand technical measures. To guide the investigation, the fol-\nlowing  recommendations  are  made.  1)  Send  isolates  to  a\nreference  laboratory  for  confirmation.  Organisms  should\nbe shipped as “dangerous goods,” according to the guide-\nlines established by the World Health Organization and the\nOffice  of  Biosafety  at  CDC  (36).  In  the  United  States,\norganisms presumptively identified as brucellae should be\nsent to CDC or another public health laboratory following\nthe specific guidelines for transferring “select agents.” 2)\nInform infection control services and public health author-\nities,  who  may  choose  to  involve  CDC.  3)  Conduct  a\nmeticulous epidemiologic investigation. 4) Determine the\ndate  and  circumstances  of  the  exposure  (17).  5)  Exclude\nother  potential  sources  of  transmission,  such  as  previous\nlaboratory exposures, travels to brucellosis-endemic areas,\nconsumption  of  unpasteurized  dairy  products,  and  han-\ndling  of  farm  or  laboratory  animals  (14,15,17,27,29).  6)\nKeep all relevant data and records. 7) Define the exposed\npopulation. 8) Determine the level of risk on the basis of\ntype of laboratory procedures performed, proximity to the\nsource,  duration  of  the  exposure,  and  the  like,  to  define\npersons  to  whom  postexposure  prophylaxis  should  be\noffered (8,32). 9) Check biologic safety cabinets (27,29).\n10) Check the ventilation system (27). 11) Collect baseline\nserum  samples  from  all  known  potentially  exposed  per-\nsons  (17,27).  12)  Freeze  isolates  for  future  typing,  espe-\ncially in brucellosis-endemic areas (23,27).\nPrevention of Laboratory-acquired Cases\nBrucellae  are  considered  class  3  organisms.  CDC  has\nstrongly recommended that live Brucella cultures and sus-\npicious  organisms  be  manipulated  in  a  class  II  biologic\nsafety  cabinet  (37).  This  recommendation,  however,  is\nclearly insufficient for preventing laboratory-acquired dis-\nease because by the time the organism is suspected or con-\nfirmed as Brucella, exposure of laboratory personnel may\nhave occurred (27). On the other hand, converting a large\nand busy clinical microbiology laboratory into a biosafety\nlevel III facility, where all specimens are handled in biolog-\nic safety cabinets (32), is both impractical and unnecessary,\nespecially in laboratories in areas where the disease is not\nendemic. However, in regions where brucellosis is highly\nprevalent, an enhanced safety policy should be adopted. All\nblood  culture  vials  detected  as  positive  by  the  automated\nblood culture system, as well as all bone marrow and syn-\novial  fluid  specimens,  should  be  manipulated  in  biologic\nsafety hoods until the isolated microorganisms are defini-\ntively determined to be other than Brucella sp. (27).\nFor  laboratories  serving  areas  where  brucellosis  is\nuncommon, the following recommendations are made. 1)\nBecause of the low incidence of brucellosis, physicians in\nareas not endemic for the disease are unfamiliar with the\nclinical and epidemiologic features of human brucellosis,\nand the possibility of brucellosis is rarely considered in the\ndifferential diagnosis. Periodic education of physicians on\nthis  subject  is  indicated.  2)  Communication  between\nattending  physicians  and  the  laboratory  should  be\nimproved. The clinical microbiology laboratory should be\ninformed  in  advance  when  clinical  specimens  had  been\nobtained  from  patients  with  risk  factors  for  brucellosis,\nsuch  as  recent  history  of  travel  to  brucellosis-endemic\nareas,  consumption  of  local  or  imported  unpasteurized\ndairy products, or professional exposures in veterinarians,\nshepherds,  slaughterhouse  employees,  and  laboratory\nworkers. 3) The use of automated, continuous monitoring\nblood culture systems for patients with suspected brucel-\nlosis  should  be  preferred  over  the  lysis-centrifugation\nmethod  because  the  latter  involves  centrifuging  clinical\nspecimens  and  visually  inspecting  plates  to  detect  the\norganism and probably increases the risk for transmission\n(11,27). Although the use of blood lysis-based methods has\nbeen  advocated  in  the  past  for  improving  detection  of\nBrucella bacteremia, modern automated blood culture sys-\ntems are faster and more sensitive (11,12). 4) The familiar-\nity of laboratory technicians with the characteristics of the\norganism,  as  well  as  with  the  safe  handling  of  cultures,\nshould be improved and maintained through periodic edu-\ncation.  The  reader  is  referred  to  the  excellent  review  by\nGilligan  and York  for  handling  and  identifying  presump-\ntive Brucella organisms (35). 5) Standard precautions and\nstrict adherence to good laboratory practices must be com-\npletely  adopted,  reinforced,  and  regularly  monitored.  6)\nAll work with gram-negative or gram-variable small rods\nor coccobacilli isolated from tissues, blood, bone marrow,\nbone, or synovial fluid exudates should be carried out in a\nbiosafety  cabinet  until  Brucella has  been  ruled  out.  7)\nPlates  should  be  sealed  for  safety  when  not  in  use  and\nappropriately disposed and sterilized as soon as they are no\nlonger being actively used (17). 8) Because brucellae are\nrelatively slow-growing bacteria, cultures for the organism\nhave been traditionally kept for several weeks. However,\nmodern  blood  culture  systems  enable  brucellae  to  be\ndetected within the routine 5-day incubation period insti-\ntuted  in  most  clinical  laboratories  (12). Therefore,  safety\nprecautions should not be limited to organisms that tend to\ngrow slowly (8,18,20,26). 9) Antimicrobial drug–suscepti-\nbility  testing  of  Brucella  organisms  is  not  indicated\nbecause the therapeutic regimen for brucellosis is standard,\nand  the  organism  does  not  usually  acquire  antimicrobial\nresistance.  Performance  of  this  and  other  unnecessary\ntests,  and  especially  of  laboratory  procedures  known  to\nproduce aerosols, should be strongly discouraged (27). 10)\nIf a suspension of living brucellae is spilled and the organ-\nism is recognized, the entire laboratory should be immedi-\nately  evacuated,  doors  should  be  shut,  and  an  effective\ngermicide  such  as  3%  phenol  or  10%  bleach  should  be\napplied  by  a  trained  person  wearing  a  safety  mask,  gog-\ngles, an impermeable laboratory gown, and gloves (7).\nPostexposure Prophylaxis\nBecause  of  ethical  considerations,  the  heterogeneous\nnature of the events leading to Brucella exposure, difficul-\nties in determining the actual risk for individual workers,\nlate recognition of outbreaks, and the small number of per-\nsons involved in each outbreak, no controlled studies have\nbeen performed to assess the value of administering post-\nexposure  prophylaxis  to  persons  at  risk.  However,  anec-\ndotal  evidence  suggests  that  administering  prophylactic\nantimicrobial drug therapy may reduce the risk of develop-\ning clinical disease (7,8,26).\nIn a recently reported event, an isolate from a chest wall\nexudate  culture  from  an  Indian  patient,  which  was  origi-\nnally identified as M. phenylpyruvica, was correctly recog-\nnized  as  B.  melitensis 22  days  after  the  specimen  was\nobtained (8). By that time, 26 laboratory workers had been\npotentially  exposed,  and  6  had  actually  manipulated  the\norganism. These 6 workers were considered to be at high\nrisk and offered a 3-week prophylactic course of combined\ndoxycycline-rifampin  or  trimethoprim-sulfamethoxazole\ntherapy.  None  of  the  5  laboratory  technologists  who\nreceived  postexposure  antimicrobial  drugs  became  ill  or\ndeveloped an antibody response. However, Brucella bac-\nteremia  developed  in  the  only  laboratory  worker  who\nrefused therapy, and she seroconverted 10 weeks after the\nspecimen  was  received  (8).  Neither  clinical  disease  nor\nseroconversion developed in the remaining 19 laboratory\nworkers who were only present in the laboratory but were\nnot in direct contact with the organism (8).\nIn  another  event,  3  laboratory  technologists  who  had\ninadvertently  worked  with  a  B.  melitensis isolate  on  an\nopen bench and sniffed the plates were given 1 week of pro-\nphylactic  doxycycline  within  24  hours  of  the  exposure;\nnone of them became ill or seroconverted (26). In a large\nexposure caused by breakage of a tube containing B. abor-\ntus, patients who started therapy immediately after serocon-\nversion was detected and before symptoms developed (10\nweeks after the laboratory accident) had a benign clinical\ncourse, suggesting that even if the disease was not prevent-\ned, a certain degree of attenuation probably occurred (7).\nBecause  of  the  high  attack  rate  of  brucellosis  among\nexposed  workers,  the  unpredictable  and  often  chronic\ncourse of the disease, and the difficulties in eradicating the\norganism  once  a  symptomatic  infection  has  been  estab-\nlished, postexposure prophylaxis is probably indicated for\nall  persons  after  an  obvious  exposure  to  living  brucellae\n(26). Antimicrobial  drug  prophylaxis  with  a  combination\nof oral doxycycline 100 mg twice a day plus rifampin 600\nmg 4 × per day for 3 weeks should be started as soon as the\nexposure  to  confirmed  Brucella organisms  is  recognized\n(8). For pregnant women, administration of trimethoprim-\nsulfamethoxazole 160/800 mg 2 × per day for 3 weeks has\nbeen advocated (8).\nPostexposure Follow-up\nWhether  exposed  persons  have  received  prophylactic\ntherapy or not, increased surveillance for clinical signs of\ndisease should be conducted for at least 6 months. In addi-\ntion,  exposed  laboratory  workers  should  be  followed  for\npossible subclinical infections or early signs of disease by\nperiodic serologic testing. Weekly or semiweekly serolog-\nic surveillance is recommended for the first 3 months and\nonce  a  month  thereafter  for  3  to  9  additional  months  to\ndetect late infections resulting from prolonged incubation\n(7,8).\nHuman brucellosis has variable clinical manifestations\nand may mimic other infections, particularly influenza, as\nwell as noninfectious conditions. Therefore, the diagnosis\nof  the  disease  requires  a  high  index  of  suspicion.\nEducation  of  exposed  personnel  on  the  symptoms  of  the\ndisease and the need for periodic and timely serologic fol-\nlow-up are particularly important. Increased vigilance dur-\ning  the  flu  season  employing  a  broad  case  definition  is\nclearly  needed,  and  administration  of  influenza  vaccine\nshould be strongly recommended to all Brucella-exposed\npersons. If symptoms develop, blood cultures and cultures\nof other normally sterile body fluids, as clinically indicat-\ned, should be obtained, and a new antibody titer should be\ndetermined.  Although  use  of  nucleic  acid  amplification\nmethods  has  been  shown  to  enable  early  detection  of\ninfected persons (38), these tests are not yet commercially\navailable.\nTreating Infected Persons and\nPosttreatment Follow-up\nPersons  in  whom  culture-confirmed  or  serologically\nproven disease develops should receive therapy with oral\ndoxycycline and rifampin for 6 weeks, or a combination of\noral doxycycline for 6 weeks and an intramuscular amino-\nglycoside  (gentamicin  or  streptomycin)  for  the  first  2\nweeks (2,7). Use of 3 drugs is usually reserved for compli-\ncated cases or life-threatening clinical manifestations, such\nas endocarditis or meningitis (1). Pregnant women should\nreceive  trimethoprim-sulfamethoxazole  160/800  mg  2  ×\nper day for 6 weeks (1).\nEven  when  patients  are  appropriately  treated,  the  risk\nfor  relapse  remains  high  (≈20%)  (39,40).  Therefore,\npatients who have completed a full therapeutic antimicro-\nbial course should be followed clinically and serologically\nfor  1  year.  Failure  to  show  declining  antibody  titers  may\nindicate incomplete cure. If symptoms consistent with bru-\ncellosis develop, blood cultures and serologic tests should\nbe performed to detect relapses of the disease (39). Use of\nnucleic acid amplification methods in the future may fur-\nther improve the detection of patients in whom the organ-\nism was not eliminated (40).\nAlthough human brucellosis has been eradicated from\nmost  industrialized  countries,  isolated  cases  of  disease,\nusually  related  to  travel  or  import  of  contaminated  food\nfrom disease-endemic areas, continue to occur. Because of\nthe  low  incidence  of  disease  in  industrialized  countries,\nclinical  laboratory  technologists  have  become  unfamiliar\nwith  identifying  and  handling  Brucella species.  Unsafe\nlaboratory  practices  while  manipulating  Brucella isolates\nhave resulted in inadvertent exposure to the organism and\nmany cases of laboratory-acquired disease, indicating lack\nof preparedness to cope with bioterrorism threats involv-\ning  brucellae.  Education  of  laboratory  personnel  on  the\nidentification  of  Brucella  species,  adherence  to  and\nenforcement  of  standard  precautions,  thorough  investiga-\ntion of laboratory exposures, administration of prophylac-\ntic  antimicrobial  drug  therapy,  and  close  follow-up  of\ndirectly exposed persons are strongly recommended.\nDr.  Yagupsky  is  the  director  of  the  Clinical  Microbiology\nLaboratory  of  the  Soroka  University  Medical  Center,  Beer-\nSheva, Israel. His main research areas are Kingella kingae infec-\ntions and diagnosis of human brucellosis.\nDr.  Baron  is  director  of  the  Clinical  Microbiology\nLaboratory at Stanford University Medical Center and professor\nof pathology at Stanford University Medical College. Her inter-\nests include the development of rapid infectious disease diagnos-\ntic  tests  and  educating  both  scientists  and  the  public  about\nrelevant aspects of infectious diseases.\n22. Martin-Mazuelos  E,  Nogales  MC,  Florez  C,  Gomez-Mateos  JM,\nLozano F, Sanchez A. Outbreak of Brucella melitensis among micro-\nbiology laboratory workers. J Clin Microbiol. 1994;32:2035–6.\n23. Gruner E, Bernasconi E, Galeazzi RL, Buhl D, Heinzle R, Nadal D.\nBrucellosis; an occupational hazard for medical laboratory personnel.\nReport of five cases. Infection. 1994;22:33–6.",
         "P. Yagupsky,E. Baron",
         null
        ],
        [
         "15",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "Abstract— This paper addresses the problem of motion plan-\nning (MP) in dynamic environments. It is ﬁrst argued that\ndynamic environments impose a real-time constraint upon MP:\nit has a limited time only to compute a motion, the time available\nbeing a function of the dynamicity of the environment. Now,\ngiven the intrinsic complexity of MP, computing a complete\nmotion to the goal within the time available is impossible to\nachieve in most real situations. Partial Motion Planning (PMP)\nis the answer to this problem proposed in this paper. PMP is a\nmotion planning scheme with an anytime ﬂavor: when the time\navailable is over, PMP returns the best partial motion to the goal\ncomputed so far. Like reactive navigation scheme, PMP faces a\nsafety issue: what guarantee is there that the system will never\nend up in a critical situation yielding an inevitable collision?\nThe answer proposed in this paper to this safety issue relies\nupon the concept of Inevitable Collision States (ICS). ICS takes\ninto account the dynamics of both the system and the moving\nobstacles. By computing ICS-free partial motion, the system\nsafety can be guaranteed. Application of PMP to the case of a\ncar-like system in a dynamic environment is presented.",
         "The problem of autonomous navigation has attracted a lot\nof interest for various robotic systems during last decades.\nTwo main paradigms have arisen to tackle this problem, the\ndeliberative approaches resulting in global motion planning\nschemes, i.e. the determination of a complete plan based on a\npriori known information and the reactive approaches taking\ndecision from real-time data by means of exteroceptive sen-\nsors, while moving. Though ﬁrst problems involved mostly\nsimple systems evolving among a stationary environment,\nit became clear that for real applications, dynamic systems\nas well as a dynamic environment had to be considered.\nthe system has\nWithin a dynamic environment however,\nthe obligation to make a decision, within a bounded time,\notherwise it might be in danger by the sole fact of being\npassive. This limited available time for the system to make\na decision, i.e. plan a motion, depends on the nature and\ndynamicity of the environment and is a hard real\ntime\nconstraint. Unfortunately, global motion planning schemes\nbased on techniques from computational geometry that were\nused for the ﬁrst problems [1], did not give much hope to\nfulﬁl such a timing constraint due to their inherent NPHard\ncomplexity [2]. Hence, reactive methods have been preferred\nand several schemes have been presented [3], [4], [5]. Nev-\nertheless, reactive approaches exhibit strong limitations. In\norder to take fast decisions, these methods explore locally\nthe velocity space of the system from which one admissible\ncontrol is selected at a time. As a consequence these methods\nexhibit a lack of lookahead, conducting the robot\nto be\ntrapped in local minima during its trip, and a weak goal\ndirectedness keeping the robot from reaching the objective.\nDespite recent modiﬁcations on a few schemes in order to\nimprove these techniques, once systems with kinematic or\ndynamic constraints are considered, speciﬁc schemes are pro-\nposed [3], [6], [7] but cannot handle such complex systems\nin a general form. Fortunately, in the mid 90’s, probabilistic\nplanners appeared [8] and brought a new powerful\ntool\nfor rapid exploration of high dimensional state-time space,\nframework presented in [9] used for complex systems and\ndynamic environment description. Some recent work could\ndemonstrate fast planning within a dynamic environment\nusing probabilistic techniques [10] and [11].\nHowever, this work does not take into consideration the\nfact that a complete trajectory to the goal might not be\nfound before the available allotted time has elapsed. In fact,\nas [12], we believe that for complex systems or environment,\na complete trajectory to the goal cannot be found over a\nlimited time, in general. In such a case, it becomes of the\nutmost importance to consider the behaviour of the system\nat the end of the trajectory. What if a car ends its trajectory\nin front of a wall at high speed? It becomes clear that strong\nguarantees should be given to this trajectory in order to\nhandle the safety issues raised by such a partial planning.\nA few papers only have addressed the issue of safety\nfor partial or incomplete trajectory by providing, in another\ncontext braking policies [13] or more recently by calculating\nevasive plan [14] or by insuring the system to be collision\nfree during an empirical period of time τ [12]. We believe\nthat these methods propose partial answers to the problem of\nsafety. In Fig. 1 we consider a selected milestone of a point\nmass robot with non zero velocity moving to the right (a state\nof P is therefore characterised by its position (x, y) and its\nspeed v). Depending upon its speed there is a region of states\n(in grey) for which P, even though it is not in collision and\nis τ -safe, will not have the time to brake and avoid collision.\nThese states for which no matter what the future trajectory of\nthe system is, a collision with the obstacle occurs, have been\ncharacterized by [15] as Inevitable Collision States (ICS).\nThe framework of ICS provides a suitable framework to\nestablish the relation of the collision states and the dynamic\nconstraints in order to have strong safety guarantees.\nB. Contribution of the Paper\nIn this paper, we ﬁrst detail the mechanism of a Partial Mo-\ntion Planner (PMP) designed to handle the partial plans that\nwould result from planning under hard real time constraint\nwithin a dynamic environment. Then, we address the safety\nissue related to such a scheme by calculating trajectories\nof safe states using the framework of ICS. We provide a\nsufﬁcient condition for hard safety guarantee for the PMP,\nallowing discrete calculation of the ICS, thus reducing the\ncomputational complexity of the scheme while insuring the\nsafety of the whole trajectory.\nThe paper is organised as follows: §2 brieﬂy reviews\nthe work related to trajectory planning within a dynamic\nenvironment and detail the existing safety guarantees that\nhave been provided in recent approaches. After introducing\nin §3 useful notations and deﬁnitions, we detail in §4 the\nmechanism of the PMP algorithm for which we provide\nand demonstrate in §5 a sufﬁcient hard safety condition. We\npresent ﬁnally in §6 our preliminary results for a car-like\nrobot and draw our conclusion in §7.",
         "D. Preliminary Results\nIn this implementation, the duration of a cycle δc is 1s,\nand the integration step of the differential equation is 0.5s.\nIn Fig. 7 we can see an example of a navigation from a\nstill starting state to a still goal state. The environment is\ncluttered by moving and static obstacles. In 7(a) one can\nobserve in front of the car how the safe partial trajectory is\ncalculated and planned within the time-state space in order\nto avoid the obstacle moving upward. The states behind\nthe car, deﬁne the trajectory, built from partial trajectories\nfrom the previous PMP cycles and (ideally) executed by\nthe robot. In 7(b) we can observe that the car was obliged\nto slow down at the intersection of several obstacles, since\nno other safe trajectories could be found, before to re-\naccelerate. In 7(d) the system has planned a partial trajectory\nthat avoids the last static obstacle. Videos of this simula-\ntion can be found at http://emotion.inrialpes.fr\n/film-gallery.php.\nThis work is currently being integrated on a real platform\n(ﬁg. 8(a)) moving within a parking lot. External cameras\nabove the parking (ﬁg. 8(b)) have been installed in order to\nfully observe the environment and thus localize our system\nand detect static (parked cars) as well as moving obstacles\n(pedestrians and cars) whereas the trajectories of the moving\nobstacles will be predicted using the work of [23].\nThe exploration method used is the well known Rapidly-\nExploring Random Tree method (RRT) [24]. RRT incre-\nmentally builds a tree in the state space of A. The basic\nprinciple of RRT is depicted in Fig. 6. A state sr is randomly\nselected ﬁrst. Then, the closest node in the tree, say sc,\nis determined. Constant controls selected from U={ (α, ˙ξ)\nVII. DISCUSSION AND CONCLUSION\nIn this paper we tackled the problem of motion planning\nwithin a dynamic environment and proposed a Partial Motion\nPlanning scheme (PMP) which handles the real-time con-\nstraint inherent to such environment. The PMP algorithm\nconsists in iteratively exploring the state-time space during\na ﬁxed limited time, by building a tree using probabilistic\ntechniques. During a cycle, a complete trajectory calculation\nto the goal can not be guaranteed in general, which raises\nthe issue of the safety of our system. We use the formalism\nof the Inevitable collision States (ICS) as the theoretical\nanswer to this safety problem. A trajectory of ICS-free states\nguarantees that our system always have a possibility to escape\ncritical situations. ICS computation remains however a very\ncomplicated task in its general form and we have to rely\non conservative approximation for practical ICS calculation.\nIn this paper, we also demonstrate a property that simpliﬁes\nthe safety checking of a trajectory. Finally, we present an\nimplementation of the PMP algorithm for a car-like robot\nand provide simulation results of the safe trajectory found\nby PMP for our system.\nFuture work includes the coupling of the PMP algorithm\nwith a closed loop control and its integration on an exper-\nimental vehicle. Our goal is to perform experimentations\nwithin a real environment, for which a model of the future\nobstacles’ behaviour will be determined thanks to a predic-\ntion technique.",
         "",
         "",
         "Abstract— This paper addresses the problem of motion plan-\nning (MP) in dynamic environments. It is ﬁrst argued that\ndynamic environments impose a real-time constraint upon MP:\nit has a limited time only to compute a motion, the time available\nbeing a function of the dynamicity of the environment. Now,\ngiven the intrinsic complexity of MP, computing a complete\nmotion to the goal within the time available is impossible to\nachieve in most real situations. Partial Motion Planning (PMP)\nis the answer to this problem proposed in this paper. PMP is a\nmotion planning scheme with an anytime ﬂavor: when the time\navailable is over, PMP returns the best partial motion to the goal\ncomputed so far. Like reactive navigation scheme, PMP faces a\nsafety issue: what guarantee is there that the system will never\nend up in a critical situation yielding an inevitable collision?\nThe answer proposed in this paper to this safety issue relies\nupon the concept of Inevitable Collision States (ICS). ICS takes\ninto account the dynamics of both the system and the moving\nobstacles. By computing ICS-free partial motion, the system\nsafety can be guaranteed. Application of PMP to the case of a\ncar-like system in a dynamic environment is presented.\nThe problem of autonomous navigation has attracted a lot\nof interest for various robotic systems during last decades.\nTwo main paradigms have arisen to tackle this problem, the\ndeliberative approaches resulting in global motion planning\nschemes, i.e. the determination of a complete plan based on a\npriori known information and the reactive approaches taking\ndecision from real-time data by means of exteroceptive sen-\nsors, while moving. Though ﬁrst problems involved mostly\nsimple systems evolving among a stationary environment,\nit became clear that for real applications, dynamic systems\nas well as a dynamic environment had to be considered.\nthe system has\nWithin a dynamic environment however,\nthe obligation to make a decision, within a bounded time,\notherwise it might be in danger by the sole fact of being\npassive. This limited available time for the system to make\na decision, i.e. plan a motion, depends on the nature and\ndynamicity of the environment and is a hard real\ntime\nconstraint. Unfortunately, global motion planning schemes\nbased on techniques from computational geometry that were\nused for the ﬁrst problems [1], did not give much hope to\nfulﬁl such a timing constraint due to their inherent NPHard\ncomplexity [2]. Hence, reactive methods have been preferred\nand several schemes have been presented [3], [4], [5]. Nev-\nertheless, reactive approaches exhibit strong limitations. In\norder to take fast decisions, these methods explore locally\nthe velocity space of the system from which one admissible\ncontrol is selected at a time. As a consequence these methods\nexhibit a lack of lookahead, conducting the robot\nto be\ntrapped in local minima during its trip, and a weak goal\ndirectedness keeping the robot from reaching the objective.\nDespite recent modiﬁcations on a few schemes in order to\nimprove these techniques, once systems with kinematic or\ndynamic constraints are considered, speciﬁc schemes are pro-\nposed [3], [6], [7] but cannot handle such complex systems\nin a general form. Fortunately, in the mid 90’s, probabilistic\nplanners appeared [8] and brought a new powerful\ntool\nfor rapid exploration of high dimensional state-time space,\nframework presented in [9] used for complex systems and\ndynamic environment description. Some recent work could\ndemonstrate fast planning within a dynamic environment\nusing probabilistic techniques [10] and [11].\nHowever, this work does not take into consideration the\nfact that a complete trajectory to the goal might not be\nfound before the available allotted time has elapsed. In fact,\nas [12], we believe that for complex systems or environment,\na complete trajectory to the goal cannot be found over a\nlimited time, in general. In such a case, it becomes of the\nutmost importance to consider the behaviour of the system\nat the end of the trajectory. What if a car ends its trajectory\nin front of a wall at high speed? It becomes clear that strong\nguarantees should be given to this trajectory in order to\nhandle the safety issues raised by such a partial planning.\nA few papers only have addressed the issue of safety\nfor partial or incomplete trajectory by providing, in another\ncontext braking policies [13] or more recently by calculating\nevasive plan [14] or by insuring the system to be collision\nfree during an empirical period of time τ [12]. We believe\nthat these methods propose partial answers to the problem of\nsafety. In Fig. 1 we consider a selected milestone of a point\nmass robot with non zero velocity moving to the right (a state\nof P is therefore characterised by its position (x, y) and its\nspeed v). Depending upon its speed there is a region of states\n(in grey) for which P, even though it is not in collision and\nis τ -safe, will not have the time to brake and avoid collision.\nThese states for which no matter what the future trajectory of\nthe system is, a collision with the obstacle occurs, have been\ncharacterized by [15] as Inevitable Collision States (ICS).\nThe framework of ICS provides a suitable framework to\nestablish the relation of the collision states and the dynamic\nconstraints in order to have strong safety guarantees.\nB. Contribution of the Paper\nIn this paper, we ﬁrst detail the mechanism of a Partial Mo-\ntion Planner (PMP) designed to handle the partial plans that\nwould result from planning under hard real time constraint\nwithin a dynamic environment. Then, we address the safety\nissue related to such a scheme by calculating trajectories\nof safe states using the framework of ICS. We provide a\nsufﬁcient condition for hard safety guarantee for the PMP,\nallowing discrete calculation of the ICS, thus reducing the\ncomputational complexity of the scheme while insuring the\nsafety of the whole trajectory.\nThe paper is organised as follows: §2 brieﬂy reviews\nthe work related to trajectory planning within a dynamic\nenvironment and detail the existing safety guarantees that\nhave been provided in recent approaches. After introducing\nin §3 useful notations and deﬁnitions, we detail in §4 the\nmechanism of the PMP algorithm for which we provide\nand demonstrate in §5 a sufﬁcient hard safety condition. We\npresent ﬁnally in §6 our preliminary results for a car-like\nrobot and draw our conclusion in §7.\nII. PREVIOUS WORKS\nRecently, a few reactive approaches have been modiﬁed\nin order to increase their lookahead and goal directedness.\nBasically, these modiﬁed methods rely on a map, built at\nexecution-time, like an occupancy grid, or known a priori.\nThese modiﬁed reactive methods use techniques stemming\nin order to ex-\nfrom global motion planning techniques,\nplore the velocity space of the system and select a discrete\nsequence of velocities, introducing in a sense the concept\nof partial planning. The trajectory to be followed, ie this\ndiscrete sequence of velocities, is determined by mean of\na navigation function [16] or an incremental algorithm that\nbuilds a tree within the velocity space [17], [18], [19].\nHowever, complex systems or environments remain difﬁcult\nto handle in a general form.\nthe environment, resulting thus in safer motion. Beside early\nwork based on dynamic programming, like the dynamic A*\nalgorithm (D*) [20], [21],\nthe approach of motion plan-\nning has been mainly reconsidered with the new tools of\nprobabilistic techniques based on graph or tree construction,\nshowing impressive results [10], [11]. Only latest work on\nthis issue, recognise the possibility of complete trajectory\nplanning failure [12], [14] and attempts to provide safety\nguarantee. However the guarantees apply over speciﬁc, em-\npirical time period without obvious physical relation with the\nconsidered dynamic system. In [14] the authors complement\nthe work presented in [10] and deﬁne safe planning as\nthe capacity of the planner to calculate an escape plan in\ncase it has failed ﬁnding a complete trajectory to the goal\nduring the allotted time. But no details with respect to the\nvalid time length of the escape plan is given. In the work\nof [12] the authors do not attempt to plan at each step a\ncomplete trajectory, the computational constraints being to\nhigh, hence considering the eventuality of partial planning.\nThey present the concept of τ -safety as a guarantee of no\ncollision during τ seconds for each milestone of the tree.\nSince the planner builds a tree between primary milestones\nat the equilibrium state, such a criteria might sufﬁce in the\npresented scheme, nevertheless it is too weak for a general\nkinodynamic framework.\nIII. NOTATIONS AND DEFINITIONS\nLet A denote a robotic system placed in a workspace\nW. The motion model of A is described by a differential\nequation of the form ˙s = f (s, u) where s ∈ S is the state\nof the system, ˙s its time derivative and u ∈ U a control. S\nis the state space and U the control space of A. Let φ ∈ Φ:\n[t0, tf ] −→ U denote a control input, ie. a time-sequence\nof controls. Starting from an initial state s0, at time t0, and\nunder the action of a control input φ, the state of the system\nA at time t is denoted by s(t) = φ(s0, t). An initial state and\na control input deﬁne a trajectory for A, ie a time sequence of\nstates. The environment is cluttered with a set of obstacles.\nAn obstacle B is a closed subset of W. The deﬁnition of\na moving obstacle is time-dependent and denoted by B(t).\nLet B(t0, ∞) denotes the obstacle B from time t0 to ∞, it\nmodels the future behaviour of B. A state s is a collision\nstate at time t if and only if ∃B such that A(s) ∩ B(t) = ∅.\nAs per [15], let us recall the formal deﬁnition of an Inevitable\nCollision State.\nDEFINITION 1 (INEVITABLE COLLISION STATE)\nA state s is an Inevitable Collision State (ICS) if and only if\n∀φ, ∃tc (for a given φ) such that φ(s, tc) is a collision state.\nA safe trajectory is deﬁned as:\nGlobal motion planning schemes have been modiﬁed as\nwell in order to gain some reactivity toward changes within\nDEFINITION 2 (SAFE TRAJECTORY)\nA trajectory, deﬁned by the initial state s0 at time t0 and the\ncontrol input φ over [t0, tf ], is safe if and only if ∀t ∈ [t0, tf ],\ns(t) = φ(s0, t) is not an ICS.\nUsing these notations and deﬁnitions,\ndetails the Partial Motion Planning (PMP) algorithm.\nIV. PARTIAL MOTION PLANNER\nIn general, a robotic system cannot safely stand still in\na dynamic environment (it might be collided by a moving\nobstacle). It has to plan a motion within a bounded time and\nthen execute it in order to remain safe. The time δc available\nto calculate a new motion is function of the nature and\ndynamicity of the environment. Today, the only motion plan-\nning techniques able to deal with reasonably complex robotic\nsystems are randomized. Unfortunately, these techniques do\nnot have any running-time upper-bound: there is no guarantee\nwhatsoever that a complete motion can be computed within\nthe time δc available. Like motion planning, partial motion\nplanning requires a model of the environment, the ﬁrst step\nis aimed at getting this model. This paper does not address\nhowever the problem of model construction. This model can\nbe given a priori or built using sensor observations. In space\napplications for instance, most of the moving obstacles follow\nKepler’s law, which provides a means to compute a full a\npriori model of the future [12]. In many other applications\nhowever, the moving obstacles have their own free will and\ntheir future behaviour is only partially predictable (if at all).\nIn this kind of situations,\nthe model of the future must\nbe predicted using prediction techniques such as the ones\npresented in [22] or [23].\nNote that when predictions are used, it is likely that the\nmodel of the future that is obtained will have a limited\nvalidity duration δv. This is an additional argument for\npartial motion planning: what is the point of a complete\nexploration of the future if the model of the future used is\nlikely to be grossly inaccurate? It is better to iterate a partial\nmotion planning process taking as input a regularly updated\npredicted model of the future. The periodic iterative PMP\nscheme proposed in this paper therefore accounts for both\nthe planning time constraints and the validity duration of the\nmodel of the environment.\nThus, the PMP algorithm iterates over a cycle of duration\nδc ≤ δv as depicted in Fig. 2. It is assumed that the initial\nstate of A is ICS-free. Let us focus on the planning iteration\nstarting at time ti:\n1) An updated model of the future is acquired, ie B(ti, ∞)\nfor each moving obstacles.\n2) The state-time space of A is searched using an ran-\ndomised tree rooted at the state s(ti+1) with ti+1 =\nti + δc.\n3) At time ti+1, the current iteration is over, the best safe\npartial trajectory φi in the tree is selected according to a\ngiven criterion and is fed to the robot that will execute\n]\nit from now on. φi is deﬁned over [ti+1, ti+1 + δhi\nwith δhi the trajectory duration.\nThe algorithm operates until the last state of the planned\ntrajectory reaches a neighbourhood of the goal state.\nV. SAFETY ISSUES\nLike every method that computes partial motion only, PMP\nhas to face a safety issue: since PMP has no control over\nthe duration of the partial trajectory that is computed, what\nguarantee do we have that A will never end up in a critical\nsituations yielding an inevitable collision? The answer to that\nproblem lies in the very fact that the partial trajectory that is\ncomputed is ICS-free. Meaning that, even in the worst case\nscenario where the duration δhi of the partial trajectory is\nshorter than the cycle time δc, A can always execute one of\nthe existing safe trajectory. The overall safety is guaranteed\nas long as the initial state is ICS-free (which is something\nthat has been assumed).\nNow, determining whether a given state of A is an ICS\nor not is a complex task since it requires to consider all\npossible future trajectories for A. However, it i possible to\ntake advantage of the approximation property demonstrated\nin [15] in order to compute a conservative approximation of\nthe set of ICS. This is done by considering only a subset of\nthe full set of possible future trajectories (see §VI.B).\nBesides, in order to further reduce the complexity of the\nPMP algorithm, we present a property that simpliﬁes the\nsafety checking of a trajectory. To begin with, we demon-\nstrate a property that states that for a given control input, all\nthe states between an ICS and the corresponding collision\nstate, are ICS.\nPROPERTY 1\nLet s be an ICS at time t0. For a given control input φ, let\ntc denote the time at which a collision occurs. Then ∀t ∈\n[t0, tc], s(t) = φ(s, t) is also an ICS.\nProof: suppose that a state si = φ(s, ti), with ti ∈ [0, tc],\nis not an ICS. By deﬁnition, ∃φj that yields no collision when\napplied to si. Let φi denote the part of φ deﬁned over [t0, ti].\nClearly, the combination of φi and φj also yields no collision\nwhen applied to s  contradiction. \nWe can now provide a sufﬁcient safety condition for a\npartial\ntrajectory that states that provided a trajectory is\ncollision free, if the last state of the trajectory is not an\ninevitable collision state then none of the states of the\ntrajectory are inevitable collision states.\nPROPERTY 2 (PMP SUFFICIENT SAFETY CONDITION)\nGiven a trajectory deﬁned over [t0, tf ], if\n(H1) the trajectory is collision-free and\n(H2) s(tf ) is not an ICS\nthen ∀t ∈ [t0, tf ], s(t) is not an ICS.\nProof: Suppose that ∃ti ∈ [t0, tf ] such that si = s(ti) is\nan ICS. Then, by deﬁnition, ∀φ, ∃tc such that φ(si, tc) is a\ncollision state. If t0 ≤ tc ≤ tf then collision occurs before\ntf  contradiction with H1. Now, if tc > tf then by previous\nproperty P1, we must have s(tf ) is an ICS  contradiction\nwith H2. \ny\nx\nIn this section we present the application of PMP to the\ncase of a car-like vehicle A moving on a planar surface\nW and within a fully observable environment cluttered with\nstationary and dynamic obstacles.\nA. Vehicle Model\nA moves like a car-like vehicle and its dynamics follows\nthe bicycle model. A state of A is deﬁned by the 5-tuple\ns = (x, y, θ, v, ξ) where (x, y) are the coordinates of the\nrear wheel, θ is the main orientation of A, v is the linear\nvelocity of the rear wheel, and ξ is the orientation of the\nfront wheels (Fig. 3). A control of A is deﬁned by the couple\n(α, γ) where α is the rear wheel linear acceleration. and γ\nthe steering velocity. The motion of A is governed by the\nfollowing differential equations:\n\n\n\n\n\n\n˙x\n˙y\n˙θ\n˙vr\n˙ξ\n\n\n\n\n\n\n\n\n\n\nvr cos θ\nvr sin θ\ntan ξvr\nL\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\nwith α ∈ [αmin, αmax]\n(acceleration bounds), γ ∈\n[γmin, γmax] (steering velocity bounds), and |ξ| ≤ ξmax\n(steering angle bounds). L is the wheelbase of A.\nB. ICS Calculation\nIn general, computing the ICS for a given system is an\nintricate problem since it would require to consider the set\nof all the possible future trajectories. To compute in practice\nthe ICS for a system such as A, it is taken advantage of\nthe approximation property established in [15] showing that\na conservative approximation of the ICS can be obtained\nby considering only a ﬁnite subset I of the whole set of\npossible future trajectories. For practical reasons, the duration\nof the trajectories of I has to be limited to a given time\nhorizon that determines the overall level of safety of A.\nthis horizon can be\nIn partially predictable environment,\nset up to δv, thus providing the maximum possible safety\nguarantee. In our case, the subset I considered includes\nthe braking trajectories with a constant control selected\nfrom [(αmin, ˙ξmax), (αmin, 0), (αmin, ˙ξmin)], and applied\nover the time necessary for A to stop.\nFor instance, the shaded regions in subﬁgure (a) corre-\nspond to states with (θ, v) = (0.9, 1.0) for which the braking\ntrajectory of control (αmin, ˙ξmin) yields a collision with one\nof the obstacles, ﬁxed or moving. In the bottom subﬁgures,\nthe three braking trajectories are considered.\nIn PMP, checking whether a state is an ICS or not is carried\nout by testing if all the braking trajectories yield a collision\nwith one of the moving obstacles. In case all the trajectories\nappear to be in collision in the future, this state is an ICS\nand is not selected. Fig. 5 illustrates how a state of the partial\ntrajectory is checked to be an ICS or not. The collision states\nrepresent the collision that will occur in the future from this\nstate for all trajectories of I. In this case, since all trajectories\ncollide in the future, this state is an ICS.\n| α ∈(αmin, 0, αmax) and ˙ξ ∈( ˙ξmax, 0, ˙ξmin) } are then\napplied to sc for a duration , they yield a set of candi-\ndate trajectories ending in given states sni. These candidate\ntrajectories are pruned out: only are kept the trajectories\nthat are collision-free and whose ﬁnal state is ICS-free (as\nper property 2, such trajectories are ICS-free). Finally, the\ntrajectory whose ﬁnal state is closer to sr is selected and\nadded up to the tree. This process is repeated until the end of\nthe time available where the best partial trajectory extracted\nfrom the tree is returned.\nD. Preliminary Results\nIn this implementation, the duration of a cycle δc is 1s,\nand the integration step of the differential equation is 0.5s.\nIn Fig. 7 we can see an example of a navigation from a\nstill starting state to a still goal state. The environment is\ncluttered by moving and static obstacles. In 7(a) one can\nobserve in front of the car how the safe partial trajectory is\ncalculated and planned within the time-state space in order\nto avoid the obstacle moving upward. The states behind\nthe car, deﬁne the trajectory, built from partial trajectories\nfrom the previous PMP cycles and (ideally) executed by\nthe robot. In 7(b) we can observe that the car was obliged\nto slow down at the intersection of several obstacles, since\nno other safe trajectories could be found, before to re-\naccelerate. In 7(d) the system has planned a partial trajectory\nthat avoids the last static obstacle. Videos of this simula-\ntion can be found at http://emotion.inrialpes.fr\n/film-gallery.php.\nThis work is currently being integrated on a real platform\n(ﬁg. 8(a)) moving within a parking lot. External cameras\nabove the parking (ﬁg. 8(b)) have been installed in order to\nfully observe the environment and thus localize our system\nand detect static (parked cars) as well as moving obstacles\n(pedestrians and cars) whereas the trajectories of the moving\nobstacles will be predicted using the work of [23].\nThe exploration method used is the well known Rapidly-\nExploring Random Tree method (RRT) [24]. RRT incre-\nmentally builds a tree in the state space of A. The basic\nprinciple of RRT is depicted in Fig. 6. A state sr is randomly\nselected ﬁrst. Then, the closest node in the tree, say sc,\nis determined. Constant controls selected from U={ (α, ˙ξ)\nVII. DISCUSSION AND CONCLUSION\nIn this paper we tackled the problem of motion planning\nwithin a dynamic environment and proposed a Partial Motion\nPlanning scheme (PMP) which handles the real-time con-\nstraint inherent to such environment. The PMP algorithm\nconsists in iteratively exploring the state-time space during\na ﬁxed limited time, by building a tree using probabilistic\ntechniques. During a cycle, a complete trajectory calculation\nto the goal can not be guaranteed in general, which raises\nthe issue of the safety of our system. We use the formalism\nof the Inevitable collision States (ICS) as the theoretical\nanswer to this safety problem. A trajectory of ICS-free states\nguarantees that our system always have a possibility to escape\ncritical situations. ICS computation remains however a very\ncomplicated task in its general form and we have to rely\non conservative approximation for practical ICS calculation.\nIn this paper, we also demonstrate a property that simpliﬁes\nthe safety checking of a trajectory. Finally, we present an\nimplementation of the PMP algorithm for a car-like robot\nand provide simulation results of the safe trajectory found\nby PMP for our system.\nFuture work includes the coupling of the PMP algorithm\nwith a closed loop control and its integration on an exper-\nimental vehicle. Our goal is to perform experimentations\nwithin a real environment, for which a model of the future\nobstacles’ behaviour will be determined thanks to a predic-\ntion technique.",
         "S. Petti,Thierry Fraichard",
         null
        ],
        [
         "16",
         "0012aecca19e4d3e41741599b1dbd00fb46d0c22",
         null,
         null,
         null,
         null,
         null,
         null,
         "R. Rajagopal,M. Deakin,A. Fawole,J. Elder,J. Elder,V. Smith,R. Strange,A. Fryer",
         "captcha"
        ],
        [
         "17",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "In eukaryotic cellular proteins, protein N-myristoylation has been recog-\nnized as a protein modiﬁcation that occurs mainly on cytoplasmic or\nnucleoplasmic proteins. In this study, to search for a eukaryotic N-myris-\ntoylated transmembrane protein, the susceptibility of the N-terminus of\nseveral G-protein-coupled receptors (GPCRs) to protein N-myristoylation\nwas evaluated by in vitro and in vivo metabolic labeling. It was found that\nthe N-terminal 10 residues of B96Bom, a Bombyx mori GPCR, efﬁciently\ndirected the protein N-myristoylation. Analysis of a tumor necrosis factor\n(TNF) fusion protein with the N-terminal 90 residues of B96Bom at its\nN-terminus revealed that (a) transmembrane domain 1 of B96Bom func-\ntioned as a type I signal anchor sequence, (b) the N-myristoylated N-ter-\nminal domain (58 residues) was translocated across the membrane, and (c)\ntwo N-glycosylation motifs located in this domain were efﬁciently N-gly-\ncosylated. In addition, when Ala4 in the N-myristoylation motif of\nB96Bom90-TNF, Met-Gly-Gln-Ala-Ala-Thr(1–6), was replaced with Asn\nto generate a new N-glycosylation motif, Asn-Ala-Thr(4–6), efﬁcient N-gly-\ncosylation was observed on this newly introduced N-glycosylation site in\nthe expressed protein. These results indicate that\nthe N-myristoylated\nN-terminus of B96Bom is translocated across the membrane and exposed\nto the extracellular surface. To our knowledge, this is the ﬁrst report show-\ning that a eukaryotic transmembrane protein can be N-myristoylated and\nthat the N-myristoylated N-terminus of the protein can be translocated\nacross the membrane.",
         "Covalent attachment of fatty acids to proteins is a\nwell-recognized form of protein modiﬁcation. Two\nforms of protein fatty acylation, protein N-myristoy-\nlation and protein palmitoylation, occur on a wide\nvariety of\neukaryotic and viral proteins. Protein\nN-myristoylation is the attachment of myristic acid, a\n14-carbon saturated fatty acid, to the N-terminal Gly\nof proteins [1–5]. Many of the N-myristoylated pro-\nteins play key roles in regulating cellular structure and\nfunction. They include proteins involved in a wide\nvariety of cellular signal-transduction pathways such\nas protein kinases, phosphatases, guanine nucleotide-\nbinding proteins, and Ca2+-binding proteins. In many\ncases, the functions of these N-myristoylated proteins\nare regulated by reversible membrane binding medi-\nated by protein N-myristoylation.\nProtein N-myristoylation has been recognized as a\nprotein modiﬁcation that occurs mainly on cytoplasmic\nand nucleoplasmic proteins, and no eukaryotic trans-\nmembrane protein has been demonstrated to be\nN-myristoylated so far. In viral proteins, however,\nprotein N-myristoylation has been found to occur on\ntransmembrane proteins such as the large L envelope\nprotein of hepatitis B virus (HBV), a polytopic mem-\nbrane protein [6,7]. Therefore, it seemed possible that\nN-myristoylated eukaryotic transmembrane proteins\nexist. In this study, we tried to search for eukaryotic\ntransmembrane proteins\nthat are N-myristoylated.\nAs protein palmitoylation, another type of protein\nacylation,\nis often observed on G-protein-coupled\nreceptors (GPCRs) [8], which are seven transmembrane\nreceptors, we focused our attention on these recep-\ntors to search for N-myristoylated transmembrane\nproteins.\nGPCRs represent a superfamily of proteins\nthe\nfunction of which is dependent on heterotrimeric\nGTP binding and hydrolyzing proteins (G-proteins).\nThey link the cells to their environment by receiving\nstimuli, relaying the message to the cells, and regula-\nting the cellular response [9,10]. GPCRs represent one\nof the largest protein superfamilies in the eukaryotic\ngenome [11,12]. In fact,  9000 entries are listed in the\nGPCR database which is available as a public WWW-\nserver (http://www.gpcr.org/7tm/) [13]. To search for\nN-myristoylated GPCRs, possible candidates with an\nN-myristoylation consensus motif at their N-termini\nwere ﬁrst extracted from the GPCR database and then\nthe susceptibility of the proteins to protein N-myristoy-\nlation was examined experimentally. It was found that\nthe N-terminus of B96Bom [14,15], a Bombyx mori\nGPCR (tyramine receptor), is efﬁciently N-myristoylat-\ned and that\nthe N-myristoylated N-terminus of\nB96Bom is translocated across the membrane and\nexposed to the extracellular surface. To our knowledge,\nthis is the ﬁrst report showing that a eukaryotic trans-\nmembrane protein can be N-myristoylated and that the\nN-myristoylated N-terminus of\nthe protein can be\ntranslocated across the membrane.\nResults\nThe N-terminus of B96Bom, an insect GPCR,\nis N-myristoylated",
         "To search for N-myristoylated transmembrane pro-\nteins, the susceptibility of the N-terminal sequence of\nseveral GPCRs to protein N-myristoylation was evalu-\nated by metabolic labeling using an in vitro translation\nsystem. Three GPCRs\ntyramine receptor\n(B96Bom), human dopamine D4 receptor (DRD4),\nand human b-adrenoceptor type 1 (ADRb1)] with an\nN-myristoylation consensus motif at their N-termini\nwere selected as possible candidates (Fig. 1).\n[B. mori\nThe N-terminal 10 residues, including the initiating\nMet of the mature domain of tumor necrosis factor\n(TNF), were replaced with those of these three recep-\ntors to generate B96Bom10-TNF, DRD4-TNF, and\nADRb1-TNF, respectively. Then, the susceptibility of\nthese proteins to protein N-myristoylation was evalu-\nated by in vitro metabolic labeling using rabbit reticu-\nlocyte lysate. In this case, Dpro-TNF,\nthe mature\ndomain of TNF in which the initiating Met was intro-\nduced at the N-terminus, was used as a nonmyristoyl-\nated control TNF. As shown in Fig. 2B, translation of\nmRNAs\ncoding for Dpro-TNF, B96Bom10-TNF,\nDRD4-TNF, and ADRb1-TNF in the presence of\n[3H]leucine gave rise to protein bands with the expec-\nted molecular mass (17 kDa). Translation of mRNA\ncoding for Gi1a-TNF, in which the N-terminal 10 resi-\ndues of the naturally N-myristoylated Gi1a protein\nwere linked to the N-terminus of the mature domain\nof TNF, gave rise to a protein band with an expected\nmolecular mass of 19 kDa. The fainter protein bands\nwith a molecular mass  2 kDa larger than expected\nwere also observed in the translation products of Gi1a-\n[3H]Myristic acid labeling\nTNF and Dpro-TNF.\nrevealed that Dpro-TNF and ADRb1-TNF were not\nN-myristoylated,\nand\nDRD4-TNF were N-myristoylated (Fig. 2C). The efﬁ-\nciency of [3H]myristic acid incorporation ([3H]myristic\nacid incorporation ⁄ [3H]leucine\ninto\nB96Bom10-TNF was comparable to that into Gi1a-\nindicating that B96Bom10-TNF is efﬁciently\nTNF,\nwhereas\nA\nB\nC\nA\nB\nC\nN-myristoylated. In contrast, the efﬁciency of [3H]my-\nristic acid incorporation into DRD4-TNF was relat-\nively low.\nTo conﬁrm the N-myristoylation of B96Bom, the\nsusceptibility of B96Bom to protein N-myristoylation\nwas evaluated by in vitro metabolic labeling using\nintact B96Bom cDNA. As the efﬁciency of protein\nexpression directed by full-length B96Bom cDNA was\nvery low in both in vitro and in vivo expression systems\n(data not shown), two fusion proteins (B96Bom58-\nTNF, B96Bom90-TNF) in which the N-terminal 58 or\n90 residues of B96Bom were linked to the mature\ndomain of TNF were generated. Then their susceptibil-\nity to protein N-myristoylation was evaluated. The\nN-terminal 58 residues are the putative N-terminal ex-\noplasmic domain of B96Bom, and N-terminal 90 resi-\ndues include transmembrane domain 1 (TM1) and its\nN-terminal and C-terminal ﬂanking regions. As shown\nin Fig. 3B,\nfor\nB96Bom58-TNF gave rise to two translation products:\none was the major product with an apparent molecular\nmass of 28 kDa, and the other was a fainter band with\nan  4 kDa larger molecular mass. [3H]Myristic acid\nlabeling revealed that the 28 kDa major protein band\nwas efﬁciently N-myristoylated (Fig. 3C). Translation\nof mRNA coding for B96Bom90-TNF gave rise to two\ntranslation products: one was the major product with\nan apparent molecular mass of 27 kDa, and the other\nwas a fainter band with a molecular mass of 20 kDa.\n[3H]Myristic acid labeling revealed that the 27 kDa\nmajor protein band was also efﬁciently N-myristoylat-\ned (Fig. 3C). These results indicate that the N-terminus\nof intact B96Bom is cotranslationally N-myristoylated.\nThe N-terminal domain of B96Bom is\ntranslocated across the membrane\nIn many of the GPCRs, the N-terminal domain is\ntranslocated across the endoplasmic reticulum during\nprotein synthesis and then targeted to the plasma\nmembrane and exposed to the extracellular space. To\ndetermine whether\nthe N-myristoylated N-terminal\ndomain of B96Bom is translocated across the mem-\nbrane, the susceptibility of two N-glycosylation motifs\n[Asn-Tyr-Thr(11–13), Asn-Tyr-Thr(16–18)]\nlocated at\nthe N-terminal domain of B96Bom to protein N-glyco-\nsylation was evaluated by in vivo metabolic labeling in\ntransfected COS-1 cells. When pro-GLC-TNF, a\nmodel N-glycosylated type II transmembrane protein\nwith an N-glycosylation site in the mature domain of\nTNF, was expressed in COS-1 cells, a protein band\nwith a molecular mass of 30 kDa was generated, as\nshown in Fig. 4B (lane 1). Glycopeptidase F treatment\nof this protein gave rise to a protein band with an\n 4 kDa smaller molecular mass (26 kDa), indicating\nthe pro-GLC-TNF is N-glycosylated when\nthat\nexpressed in COS-1 cells (lane 2). When B96Bom58-\nTNF cDNA was\ntransfected into COS-1 cells, a\n28 kDa protein was expressed (lane 3); however, the\nA\nB\nmolecular size of this protein was not affected by treat-\nment with glycopeptidase F (lane 4). In contrast,\nexpression of B96Bom90-TNF cDNA gave rise to two\nproteins: one was the major product with a molecular\nmass of 33 kDa, and the other was a fainter band of\n27 kDa. Glycopeptidase F treatment of these two pro-\nteins gave rise to three protein bands: a major 27 kDa\nprotein, a minor 30 kDa protein and a trace amout of\n33 kDa protein. These results indicate that both of the\ntwo N-glycosylation motifs located at the N-terminal\ndomain of B96Bom90-TNF directed protein N-glyco-\nsylation, and diglycosylated protein with a molecular\nmass of 33 kDa was generated. [3H]Myristic acid labe-\nling revealed that both the diglycosylated 33 kDa pro-\nprotein were\ntein\n27 kDa\nefﬁciently N-myristoylated (Fig. 4C,\nlane 3). These\nresults clearly indicate that the TM1 of B96Bom func-\ntioned as a type I signal anchor sequence, and the\nN-terminal domain (58 residues) was\ntranslocated\nacross the membrane.\nand\nThe N-myristoylated N-terminus of B96Bom\nis translocated across the membrane\nIn the membrane proteins with extracellular N-termini\nthat lack a cleaved signal sequence, the TM1 functions\nas a type I signal anchor sequence, and the portion\nbefore TM1 translocates the membrane post-transla-\ntionally [16,17]. It is clear from the results obtained in\nsection that Asn11 and Asn16 of\nthe previous\nB96Bom is translocated across the membrane, as these\ntwo Asn residues are N-glycosylated. However, whe-\nther or not\nthe N-myristoylated N-terminus of\nB96Bom is translocated across the membrane is not\nclear from these experimental results. To clarify this,\nwe introduced an additional N-glycosylation site at a\nposition close to the N-terminus and evaluated its\nthis\nsusceptibility to protein N-glycosylation. For\npurpose, Ala4 in B96Bom90-TNF was replaced with\nAsn to generate a new N-glycosylation site, Asn-\nAla-Thr(4–6), and the N-glycosylation of the mutant\nobtained B96Bom90A4N-TNF expressed in COS-1\ncells was evaluated by glycopeptidase F treatment.\nWhen the B96Bom90A4N-TNF was expressed in\nCOS-1 cells, two protein bands were obtained: one\nwas a major protein band with an apparent molecular\nmass of 35 kDa, and the other was a fainter protein\nband with 27 kDa molecular mass, as\nshown in\nFig. 5B (lane 3). The molecular size of the major pro-\ntein band was slightly larger than that of diglycosylat-\nindicating that\ned B96Bom90-TNF (lanes 1 and 3),\nthe expressed B96Bom90A4N-TNF was triglycosylat-\ned. As shown in Fig. 5B lane 2, mild treatment of the\nA\nB\nC\nthat the N-myristoylated N-terminus of B96Bom90-\nTNF is\nthe membrane and\nexposed to the extracellular space.\ntranslocated across\nN-Myristoylation of the N-terminus of B96Bom is\nnot involved in the membrane translocation of\nthe N-terminal domain of B96Bom\nthe\nTo determine whether the N-myristoylation of\nN-terminus of B96Bom plays a role in the membrane\ntranslocation of the N-terminal domain of B96Bom,\nthe Gly residue at position 2 of B96Bom90-TNF was\nreplaced with Ala to generate a nonmyristoylated\nmutant, B96Bom90G2A-TNF, and its membrane\ntranslocation was evaluated by assessing the suscepti-\nbility of this protein to protein N-glycosylation. As\nin vitro translation of\nshown in Fig. 6B lane 1,\nB96Bom90-TNF cDNA gave rise to two protein\nbands: one was the major product with an apparent\nmolecular mass of 27 kDa, and the other was a fainter\nband with a molecular mass of 20 kDa. In contrast,\nin vitro translation of B96Bom90G2A-TNF cDNA\ngave rise to three protein bands: a major protein band\nA\nB\nimmunoprecipitated B96Bom90-TNF with glycopepti-\ndase F gave rise to one protein band with an interme-\ndiate size between those of the two original proteins.\nIn contrast, the same treatment of the immunoprecipi-\ntated B96Bom90A4N-TNF gave rise to two protein\nbands between the two original protein bands (lane 4),\nindicating that the major product of B96Bom90A4N-\nTNF expressed in COS-1 cells\ntriglycosylated.\n[3H]Myristic acid labeling revealed that the triglycosyl-\nated B96Bom90-TNF is N-myristoylated, and the\nN-myristoylated triglycosylated protein band was shif-\nted to a nonmyristoylated 27 kDa protein band by\ntreatment with glycopeptidase F (Fig. 5D, lanes 3 and\n4). These results indicate that Asn4, which is close to\nin B96Bom90A4N-TNF\nthe N-myristoylated Gly2,\nis quite probable\nwas N-glycosylated. Therefore,\nthe N-terminus of B96Bom is not\nFig. 6. N-Myristoylation of\ninvolved in the membrane translocation of the N-terminal domain\nof B96Bom.\nand\nB96Bom90G2A-TNF were translated in vitro in the presence of\n[3H]leucine or [3H]myristic acid using rabbit reticulocyte lysate. After\nimmunoprecipitation with antibody to TNF, the labeled translation\nproducts were analyzed by SDS ⁄ PAGE and ﬂuorography (B).\ncDNAs encoding B96Bom90-TNF and B96Bom90G2A-TNF were\ntransfected into COS-1 cells, and their expression was evaluated\nby Western blotting analysis. N-Glycosylation of the expressed pro-\ntein was determined by the change in its molecular mass by treat-\nment of\nthe immunoprecipitated protein with glycopeptidase F\n(GPF) (C).\nwith an apparent molecular mass of 29 kDa and two\nfainter bands with molecular masses of 31 kDa and\n[3H]Myristic acid labeling revealed\n20 kDa (lane 2).\nthat none of these three protein bands was N-myristo-\nlane 4). When B96Bom90G2A-TNF\nylated (Fig. 6B,\nwas expressed in COS-1 cells, three protein bands with\nmolecular masses of 35, 32, and 29 kDa were detected\n(Fig. 6C, lane 3). Glycopeptidase F treatment of these\nproteins gave rise to a single protein band with a\nmolecular mass of 29 kDa (lane 4),\nindicating that\nthe two protein bands with higher molecular mass\nwere N-glycosylated. These results clearly indicated\nthe N-terminal domain of nonmyristoylated\nthat\nB96Bom90G2A-TNF was\nthe\nmembrane as efﬁciently as that of N-myristoylated\nB96Bom90-TNF. These results reveal that N-myristoy-\nlation of the N-terminus of B96Bom does not play a\nrole in the membrane translocation of the N-terminal\ndomain of B96Bom.\ntranslocated across\ntranslocation of\nIn eukaryotic cellular proteins, protein N-myristoyla-\ntion occurs mainly on cytoplasmic or nucleoplasmic\nproteins. In viral proteins, however, protein N-myris-\ntoylation has been found to occur on transmembrane\nproteins. It has been found that the large L envelope\nprotein of HBV, a polytopic membrane protein,\nis\nN-myristoylated, and this modiﬁcation is involved in\nthe intracellular retention of this protein and thus is\nessential for viral\ninfectivity [6,7,18,19]. This protein\nhas the peculiar capacity to form two transmembrane\ntopologies via an as yet uncharacterized process of\nits pre-S\npartial post-translational\ndomain across membranes [20–22]. The HBV L protein\nwith its N-myristoylated N-terminal pre-S domain\nplays vital roles in the viral\nlife cycle by mediating\nreceptor binding during host cell attachment, by per-\nforming a matrix-like function in nucleocapsid envel-\nopment and by exerting various regulatory effects. The\nmultifunctional nature of HBV L protein depends on\nits dual transmembrane topology, which is established\nafter the completion of polypeptide synthesis [23,24].\nThus, protein N-myristoylation occurs on this viral\ntransmembrane protein and seems to be involved in\nthe speciﬁc functions of the protein. However, so far,\nprotein N-myristoylation has not been observed on\neukaryotic transmembrane proteins. In this study, to\nsearch for a eukaryotic N-myristoylated transmem-\nbrane protein, the susceptibility of the N-terminus of\nGPCR, a polytopic transmembrane protein with vital\nroles in intracellular signal-transduction cascades, to\nprotein N-myristoylation was evaluated.\n[1,26,27]. These\nTo search for N-myristoylated GPCRs, possible can-\ndidates with an N-myristoylation consensus motif at\ntheir N-terminus were ﬁrst extracted from the GPCR\ndatabase. Protein N-myristoylation is catalyzed by\nN-myristoyltransferase, a member of the GCN5 acetyl-\ntransferase superfamily of proteins [25]. The precise\nsubstrate speciﬁcity of this enzyme has been character-\nized using puriﬁed enzyme and synthetic peptides\nsequences of known\nderived from the N-terminal\nN-myristoylated proteins\nstudies\nrevealed that,\nin addition to Gly at position 2, the\namino acids at positions 3, 6, and 7 play an important\nrole in substrate recognition by N-myristoyltransferase.\nRecently, we studied the amino-acid requirements at\npositions 3, 6 and 7 in the N-myristoylation motif by\nvertical-scanning mutagenesis of amino acids in a\nthe\nmodel N-myristoylation motif. We found that\ncombination of amino acids at positions 3, 6 and 7 is\na major determinant of protein N-myristoylation\n[28,29]. For example, when Ser was located at position\n6, as many as 11 amino acids (Gly, Ala, Ser, Cys, Thr,\nVal, Asn, Leu, Ile, Gln, His) were permitted at posi-\ntion 3 to allow efﬁcient protein N-myristoylation. In\ncontrast, when other amino acids such as Thr, Ala or\nPhe were located at position 6, only three amino acids\n(Ala, Asn, Gln) were permitted at position 3 to allow\nefﬁcient protein N-myristoylation. To search for\nN-myristloylated GPCRs, these rules were applied to\nthe N-terminal sequence of GPCRs listed in the GPCR\ndatabase [13]. As a result, several biogenic amine re-\nceptors such as octopamine receptor, tyramine recep-\ntor, dopamine receptor D4, and serotonin receptor\nwere found to have N-terminal sequences that are con-\nsistent with these rules.\nIn previous studies, we have shown that the suscepti-\nbility of a protein to cotranslational N-myristoylation\ncan easily be detected by metabolic labeling in an in vitro\nor in vivo expression system using a fusion protein in\nwhich the N-terminal sequence ( 10 amino-acid resi-\ndues) of the query protein was fused to a model protein\n[28–32]. In the present study, we used this strategy to\ndetect protein N-myristoylation of GPCRs. We found\nthat the N-terminus of B96Bom, a B. mori GPCR, is\nefﬁciently N-myristoylated. We have recently cloned the\ncDNA of this protein and expressed it in HEK-293 cells.\nWe showed that B96Bom is a B. mori tyramine receptor\nwhich is negatively coupled to adenylate cyclase [14,15].\nUsing this B96Bom cDNA, we further analyzed the\nN-myristoylation of this protein.\nThe seven transmembrane domains characteristic of\nGPCRs give rise to several structural features. Each\nGPCR has an N-terminal extracellular domain, the\nseven transmembrane helices, three extracellular and\nthree intracellular loops, and an intracellular C-ter-\nminal domain. In the membrane proteins with extracel-\nlular N-termini that lack a cleaved signal sequence,\nincluding GPCRs, TM1 functions as a type I signal\nanchor sequence, and the portion before TM1 translo-\ncates the endoplasmic reticulum post-translationally\n[16,17]. Protein N-myristoylation is a very early\ncotranslational protein modiﬁcation, which may occur\nas early as when the ﬁrst 50 amino acids of the nascent\npolypeptide chain are polymerized [33]. Therefore,\nit\nseems likely that the N-myristoylation of B96Bom pre-\ncedes the translocation of the N-terminus of this pro-\nIn many\ntein across\nN-myristoylated proteins, the N-terminal myristic acid\nmoiety functions as a membrane anchor and immobil-\nizes the protein on the membrane. Therefore, it was\npossible that the membrane binding mediated by the\nN-myristoylated N-terminus of B96Bom inihibits the\nmembrane translocation of the N-terminal domain of\nthis protein. The experimental results obtained with\nB96Bom90A4N-TNF clearly show that this is not the\ncase. The oligosaccharyl transferase is required for the\ntransfer of oligosaccharides from a dolichol carrier to\nAsn-X-Ser ⁄ Thr acceptor sites for N-glycosylation in\nnascent polypeptides. N-linked glycosylation occurs on\nthe lumenal side of the endoplasmic reticulum during\nor soon after translocation of nascent polypetides [34].\nAs the active site of the oligosaccharyl transferase is\nlocated  15 residues away from the endoplasmic reti-\nculum membrane [35], the fact that Asn4 close to the\nN-myristoylated Gly2 in this mutant was N-glycosylat-\ned strongly indicated that the N-myristoylated N-ter-\nminus of B96Bom90-TNF is translocated across the\nendoplasmic reticulum membrane through transloca-\ntion machinery and exposed to the lumenal side of the\nendoplasmic reticulum. It\nis not clear whether the\nN-myristoylated N-terminus of B96Bom anchors to\nthe membrane or not. Therefore, there are two poss-\nible models for the transmembrane topology of the\nN-terminal domain of B96Bom, as shown in Fig. 7.\nThe physiological roles of protein N-myristoylation\nof B96Bom are not clear at present. The fact that the\nN-terminus of nonmyristoylated B96BomG2A-TNF\nwas translocated across the membrane as efﬁciently as\nthat of N-myristoylated B96Bom90-TNF indicated\nthat N-myristoylation of the N-terminus of B96Bom\ndoes not play a role in the membrane translocation of\nthe N-terminal domain of B96Bom. It has been found\nthat post-translational modiﬁcations such as N-glyco-\nsylation, phosphorylation, and palmitoylation play\nimportant roles in the functional regulation of GPCRs\n[8,36,37]. Of these three modiﬁcations, only N-glycosy-\nlation occurs on the N-terminal extracellular domain,\nas does protein N-myristoylation. N-Glycosylation of\nGPCRs on one or more Asn residues in the N-terminal\ndomain seems to be involved in ligand binding and cell\nsurface expression of the receptor [37]. Therefore, it is\npossible that N-myristoylation of B96Bom is also\nIn addition to the\ninvolved in these phenomena.\nB96Bom receptor, several other biogenic amine recep-\ntors\nsuch as octopamine receptors, dopamine D4\nreceptors, and a serotonin receptor were found to have\nthe N-myristoylation consensus motif at their N-ter-\nmini (Fig. 1). However, none of these receptors has\nbeen shown to be N-myristoylated so far. To our\nknowledge, no obvious similarity has been reported in\nthe function of these receptors except that all these are\nstudies will be\nbiogenic amine receptors. Further\nrequired to elucidate the roles of protein N-myristoyla-\ntion in the functions of the B96Bom receptor or other\nbiogenic amine receptors.\nExperimental procedures",
         "",
         "Materials\nRestriction\nenzymes,\nendonucleases, DNA-modifying\nRNase inhibitor, and Taq DNA polymerase were purchased\nfrom Takara Shuzo, Kyoto Japan. The mCAP RNA cap-\nping kit and proteinase K were from Stratagene (La Jolla,\nCA, USA). RNase was purchased from Boehringer-Mann-\nheim (Mannheim, Germany). Rabbit reticulocyte lysate was\nfrom Promega (Madison, WI, USA). [3H]Leucine, [3H]my-\nristic acid and Amplify were from Amersham, Piscataway,\nNJ, USA. The Dye Terminator Cycle Sequencing kit was\nfrom Applied Biosystems (Foster City, CA, USA). Polyclonal\nantibody to human TNF was purchased from R & D Sys-\ntems (Minneapolis, MN, USA). Protein G–Sepharose was\nfrom Pharmacia Biotech (Piscataway, NJ, USA). Other rea-\ngents purchased from Wako Pure Chemical (Osaka, Japan),\nDaiichi Pure Chemicals (Tokyo, Japan), and Seikagaku\nKogyo (Tokyo, Japan) were of analytical or DNA grade.\nPlasmid construction\nPlasmid pBluescript II SK(+) lacking ApaI and HindIII\nsites was constructed as previously described [38], and des-\nignated pB. Plasmid pBDpro-TNF, which contains a cDNA\ncoding for the mature domain of TNF, was constructed as\ndescribed [32]. Plasmid pBB96Bom10-TNF was constructed\nusing PCR. pBDpro-TNF served as the template and two\noligonucleotides (Bom10, B1) served as primers (Table 1).\nAfter digestion with BamHI and PstI, the ampliﬁed product\nwas subcloned into pB at the BamHI and PstI sites. Plas-\nmids pBDRD4-TNF and ADRb1-TNF were constructed\nby a method similar to that used for pBB96Bom10-TNF\nusing two oligonucleotide pairs\n(DRD4 plus B1 and\nADRB1 plus B1, respectively) as primers (Table 1). Plasmid\npBGi1a-TNF was constructed as previously described [32].\nThe cDNAs coding for B96Bom58-TNF and B96Bom90-\nin which the N-terminal 58 and 90 residues of\nTNF,\nB96Bom were linked to the N-terminus of\nthe mature\ndomain of TNF, were constructed by using PCR. For this\nprocedure, previously cloned B96Bom cDNA [14] sub-\ncloned in pcDNA3 served as the template, and two oligo-\nnucleotide pairs (BomN plus Bom58 and BomN plus\nBom90, respectively) served as primers (Table 1). After\ndigestion with BamHI and XhoI, the ampliﬁed products\nwere subcloned into pBGi1a-TNF at the BamHI and XhoI\nsites. Plasmid pBB96Bom90A4N-TNF was constructed by\na method similar to that used for pBB96Bom90-TNF using\ntwo oligonucleotides\nas primers\n(Table 1). Plasmid pBB96Bom90G2A-TNF was constructed\nby a method similar to that used for pBB96Bom90-TNF\nusing two oligonucleotides (BomG2A, Bom90) as primers\n(Table 1). Plasmid pBpro-GLC-TNF was constructed by\nusing PCR. In this mutant, an N-glycosylation site (Asn-X-\nSer) was introduced at amino acid positions 45–47 in the\nmature domain of TNF. For this procedure, pBpro-TNF\nserved as the template and two sets of oligonucleotides (T3\nplus GLC-6 and MUTB1 plus B1) as primers. After diges-\ntion with SacI and KpnI, the ampliﬁed product was sub-\ncloned into pB at the SacI and KpnI sites. The DNA\nsequences of these recombinant cDNAs were conﬁrmed by\nthe dideoxynucleotide chain termination method [39]. The\ncDNAs coding for TNF mutants were subcloned into\npcDNA3, and the pcDNA3 constructs obtained were used\nfor transfection assays.\nIn vitro transcription and translation\nMethods essentially the same as those described previously\nwere used [29]. T3 polymerase was used to obtain transcripts\nof these cDNAs subcloned into pB vector. These transcripts\nwere puriﬁed by phenol ⁄ chloroform extraction and ethanol\nprecipitation before use. Subsequently,\ntranslation was\ncarried out using rabbit reticulocyte lysate (Promega) in the\npresence of [3H]leucine or [3H]myristic acid under conditions\nrecommended by the manufacturer. The mixture [17.5 lL\nrabbit reticulocyte lysate; 0.5 lL 1 mm leucine-free amino-\nacid mixture or 1 mm complete amino acid mixture; 4.0 lL\n[3H]leucine (5 lCi) or [3H]myristic acid (25 lCi); 3.0 lL\nmRNA] was incubated at 30 C for 90 min.\nTransfection of COS-1 cells and determination\nof N-myristoylated and N-glycosylated proteins\nThe simian virus 40-transformed African Green monkey\nkidney cell\nline, COS-1, was maintained in Dulbecco’s\nmodiﬁed Eagle’s medium (Invitrogen, Carlsbad, CA, USA)\nsupplemented with 10% fetal bovine serum (Invitrogen).\nCells (2 · 105) were plated on to 35-mm diameter dishes\n1 day before transfection. pcDNA3 construct (2 lg; Invitro-\ngen) containing mutant TNF cDNA was used to transfect\neach plate of COS-1 cells along with 3 lL LipofectAmine\n(2 mgÆmL)1; Invitrogen) in 1 mL serum-free medium. After\nincubation for 5 h at 37 C,\nthe cells were refed with\nserum-containing medium and incubated again at 37 C for\n24 h. The cells were then washed twice with 1 mL serum-\nfree Dulbecco’s modiﬁed Eagle’s medium and incubated for\n5 h in 1 mL Dulbecco’s modiﬁed Eagle’s medium with 2%\nacid\nfetal\n(100 lCiÆmL)1). Subsequently, the cells were washed three\ntimes with Dulbecco’s phosphate buffered saline, collected\nwith cell scrapers, and lysed with 200 lL RIPA buffer\n[50 mm Tris ⁄ HCl (pH 7.5), 150 mm NaCl, 1% Nonidet\nP40, 0.5% sodium deoxycholate, 0.1% SDS, proteinase\ninhibitors] on ice for 20 min. The cell lysates were centri-\nfuged at 21 000 g at 4 C for 15 min in a microcentrifuge\n(Hitachi-CF15D2), and the supernatants were collected.\nAfter immunoprecipitation with antibody to TNF, the sam-\nples were analyzed by SDS ⁄ PAGE and ﬂuorography. To\nexamine the N-glycosylation of TNF mutants, 20 lL total\ncell lysate of each group of transfected cells or TNF sample\nimmunoprecipitated from total cell lysates were treated with\n10 or 50 mUÆmL)1 glycopeptidase F at 37 C for 1 h and\nthen analyzed by Western blotting.\nWestern blotting\nTNF samples immunoprecipitated from in vitro translation\nproducts or total cell lysates of each group of transfected\ncells were resolved by SDS ⁄ PAGE (12.5% gel) and then\ntransferred to an Immobilon-P transfer membrane (Milli-\npore, Billerica, MA, USA). After being blocked with nonfat\nmilk, the membrane was probed with a speciﬁc goat anti-\nbody to human TNF as described previously [40]. Immuno-\nreactive proteins were speciﬁcally detected by incubation\nwith horseradish peroxidase-conjugated anti-goat\nIgG\n(Santa Cruz, Biotech, Santa Cruz, CA, USA). The mem-\nbrane was developed with ECL Western blotting reagent\n(Amersham) and exposed to an X-ray ﬁlm (Kodak). Quanti-\ntative analysis of immunoreactive proteins on the membrane\nwas carried out using a storage phosphor imaging screen and\nGS-250 Molecular Imager (Bio-Rad, Hercules, CA, USA).\nImmunoprecipitation\nSamples containing TNF mutants were immunoprecipitated\nwith a speciﬁc goat polyclonal antibody to human TNF (R\n& D Systems) as described [38].\nSDS ⁄ PAGE and ﬂuorography\nSamples were denatured by boiling for 3 min in SDS sam-\nple buffer followed by analysis by SDS ⁄ PAGE on a 12.5%\ngel. Thereafter, the gel was ﬁxed and soaked in AmplifyTM\n(Amersham) for 30 min. The gel was dried under vacuum\nand exposed to an X-ray ﬁlm (Kodak) for an appropriate\nperiod. Quantitative analysis of the labeled proteins was\ncarried out by scanning the ﬂuorogram using an imaging\ndensitometer (Bio-Rad GS-700).\nAcknowledgements\nPart of this work was supported by a Grant-in-Aid for\nScientiﬁc Research (No. 12660080, No. 15580080)\nfrom the Ministry of Education, Science, and Culture\nof Japan.\n1 Towler DA, Gordon JI, Adams SP & Glaser L (1988)\nThe biology and enzymology of eukaryotic protein acy-\nlation. Annu Rev Biochem 57, 69–99.\n2 Spiegel AM, Backlund PS, Butrynski JE, Jones TLZ &\nSimonds WF (1991) The G protein connection: molecu-\nlar basis of membrane association. Trends Biochem\nSci 16, 338–341.\n3 Boutin JA (1997) Myristoylation. Cell Signal 9, 15–35.\n4 Resh MD (1999) Fatty acylation of proteins: new\ninsights into membrane targeting of myristoylated and\npalmitoylated proteins. Biochim Biophys Acta 1451,\n1–16.\n5 Farazi TA, Waksman G & Gordon JI (2001) The biol-\nogy and enzymology of protein N-myristoylation. J Biol\nChem 276, 39501–39504.\n6 Persing D, Varmus H & Ganem D (1987) The preS1\nprotein of hepatitis B virus is acylated at its amino ter-\nminus with myristic acid. J Virol 61, 1672–1677.\n7 Prange R, Clemen A & Streeck RE (1991) Myristylation\nis involved in intracellular retention of hepatitis B virus\nenvelope proteins. J Virol 65, 3919–3923.\n8 Qanbar R & Bouvier M (2003) Role of\npalmitoylation ⁄ depalmitoylation reactions in\nG-protein-coupled receptor function. Pharmacol Ther\n97, 1–33.\n9 Gether U (2000) Uncovering molecular mechanisms\ninvolved in activation of G protein-coupled receptors.\nEndocr Rev 21, 90–113.\n10 Pierce L, Premont RT & Lefkowitz RJ (2002) Seven-\ntransmembrane receptors. Nat Rev Mol Cell Biol 3,\n639–650.\n11 Bockaert J & Pin JP (1999) Molecular tinkering of G\nprotein-coupled receptors: an evolutionary success.\nEMBO J 18, 1723–1729.\n12 Bockaert J, Claeysen S, Becamel C, Pinloche S &\nDumuis A (2002) G protein-coupled receptors: domi-\nnant players in cell-cell communication. Int Rev Cytol\n212, 63–132.\n13 Horn F, Vriend G & Cohen FE (2001) Collecting\nand harvesting biological data: the GPCRDB and\nNucleaRDB information systems. Nucleic Acids Res 29,\n346–349.\n14 Ohta H, Utsumi T & Ozoe Y (2003) B96Bom encodes a\nBombyx mori tyramine receptor negatively coupled to\nadenylate cyclase. Insect Mol Biol 12, 217–223.\n15 Ohta H, Utsumi T & Ozoe Y (2004) Amino acid\n28 Utsumi T, Nakano K, Funakoshi T, Kayano Y, Nakao\nS, Sakurai N, Iwata H & Ishisaka R (2004) Vertical-\nscanning mutagenesis of amino acids in a model N-myr-\nistoylation motif reveals the major amino-terminal\nsequence requirements for protein N-myristoylation. Eur\nJ Biochem 271, 863–874.\nresidues involved in interaction with tyramine in the\nBombyx mori tyramine receptor. Insect Mol Biol 13,\n531–538.\n16 Friedlander M & Blobel G (1985) Bovine opsin has\nmore than one signal sequence. Nature 318, 338–343.\n17 Kanner EM, Klein IK, Friedlander M & Simon SM\n(2002) The amino terminus of opsin translocates ‘post-\ntranslationally’ as efﬁciently as cotranslationally.\nBiochemistry 41, 7707–7715.\n18 Gripon P, Le Seyec J, Rumin S & Guguen-Guillouzo C\n(1995) Myristylation of the hepatitis B virus large sur-\nface protein is essential for viral infectivity. Virology\n213, 292–299.\n19 Bruss V, Hagelstein J, Gerhardt E & Galle PR (1996)\nMyristylation of the large surface protein is required for\nhepatitis B virus in vitro infectivity. Virology 218,\n396–399.\n20 Bruss V, Lu X, Thomssen R & Gerlich WH (1994)\nPost-translational alterations in transmembrane topol-\nogy of the hepatitis B virus large envelope protein.\nEMBO J 13, 2273–2279.\n21 Ostapchuk P, Hearing P & Ganem D (1994) A dramatic\nshift in the transmembrane topology of a viral envelope\nglycoprotein accompanies hepatitis B viral morphogen-\nesis. EMBO J 13, 1048–1057.\n22 Lambert C & Prange R (2001) Dual topology of the\nhepatitis B virus large envelope protein: determinants\ninﬂuencing post-translational pre-S translocation. J Biol\nChem 276, 22265–22272.\n23 Bruss V & Vieluf K (1995) Infection process of the\nhepatitis B virus depends on the presence of a deﬁned\nsequence in the pre-S1 domain. J Virol 69, 6652–6657.\n24 Le Seyec J, Chouteau P, Cannie I, Guguen-Guillouzo C\n& Gripon P (1999) Infection process of the hepatitis B\nvirus depends on the presence of a deﬁned sequence in\nthe pre-S1 domain. J Virol 73, 2052–2057.\n25 Dyda F, Klein DC & Hickman AB (2000) GCN5-\nrelated N-acetyltransferases: a structural overview. Annu\nRev Biophys Biomol Struct 29, 81–103.\n26 Towler DA, Adams SP, Eubanks SR, Towery DS, Jack-\nson-Machelski E, Glaser L & Gordon JI (1988) Myris-\ntoyl CoA: protein N-myristoyltransferase activities from\nrat liver and yeast possess overlapping yet distinct pep-\ntide substrate speciﬁcities. J Biol Chem 263, 1784–1790.\n27 Rocque WJ, McWherter CA, Wood DC & Gordon JI\n(1993) A comparative analysis of the kinetic mechanism\nand peptide substrate speciﬁcity of human and Saccharo-\nmyces cerevisiae myristoyl-CoA:protein N-myristoyl-\ntransferase. J Biol Chem 268, 9964–9971.\n29 Utsumi T, Sato M, Nakano K, Takemura D, Iwata H\n& Ishisaka R (2001) Amino acid residue penultimate to\nthe amino-terminal Gly residue strongly affects two\ncotranslational protein modiﬁcations, N-myristoylation\nand N-acetylation. J Biol Chem 276, 10505–10513.\n30 Utsumi T, Tou E, Takemura D, Ishisaka R, Yabuki M\n& Iwata H (1998) Met-Gly-Cys motif from G-protein\nalpha subunit cannot direct palmitoylation when fused\nto heterologous protein. Arch Biochem Biophys 349,\n216–224.\n31 Utsumi T, Sakurai N, Nakano K & Ishisaka R (2003)\nC-terminal 15 kDa fragment of cytoskeletal actin is\nposttranslationally N-myristoylated upon caspase-\nmediated cleavage and targeted to mitochondria. FEBS\nLett 539, 37–44.\n32 Utsumi T, Kuranami J, Tou E, Ide A, Akimaru K,\nHung MC & Klostergaard J (1996) In vitro synthesis of\nan N-myristoylated fusion protein that binds to the lipo-\nsomal surface. Arch Biochem Biophys 326, 179–184.\n33 Deichaite I, Casson LP, Ling HP & Resh MD (1988) In\nvitro synthesis of pp60v-src: myristoylation in a cell-free\nsystem. Mol Cell Biol 8, 4295–4301.\n34 Kaplan HA, Welply JK & Lennarz WJ (1987) Oligosac-\ncharyl transferase: the central enzyme in the pathway\nof glycoprotein assembly. Biochim Biophys Acta 906,\n161–173.\n35 Nilsson I & von Heijne G (1993) Determination of the\ndistance between the oligosaccharyltransferase active site\nand the endoplasmic reticulum membrane. J Biol Chem\n268, 5798–5801.\n36 Bohm SK, Grady EF & Bunnett NW (1997) Regulatory\nmechanisms that modulate signalling by G-protein-\ncoupled receptors. Biochem J 322, 1–18.\n37 Kaushal S, Ridge KD & Khorana HG (1994) Structure\nand function in rhodopsin: the role of asparagine-linked\nglycosylation. Proc Natl Acad Sci USA 91, 4024–4028.\n38 Utsumi T, Akimaru K, Kawabata Z, Levitan A, Toku-\nnaga T, Tang P, Ide A, Hung MC & Klostergaard J\n(1995) Human pro-tumor necrosis factor: Molecular\ndeterminants of membrane translocation, sorting and\nmaturation. Mol Cell Biol 15, 6389–6405.\n39 Sanger F, Nicklen S & Coulson AR (1977) DNA\nsequencing with chain-terminating inhibitors. Proc Natl\nAcad Sci USA 74, 5463–5467.\n40 Utsumi T, Levitan A, Hung MC & Klostergaad J\n(1993) Effects of truncation of human pro-tumor necro-\nsis factor transmembrane domain on cellular targeting.\nJ Biol Chem 268, 9511–9516.",
         "In eukaryotic cellular proteins, protein N-myristoylation has been recog-\nnized as a protein modiﬁcation that occurs mainly on cytoplasmic or\nnucleoplasmic proteins. In this study, to search for a eukaryotic N-myris-\ntoylated transmembrane protein, the susceptibility of the N-terminus of\nseveral G-protein-coupled receptors (GPCRs) to protein N-myristoylation\nwas evaluated by in vitro and in vivo metabolic labeling. It was found that\nthe N-terminal 10 residues of B96Bom, a Bombyx mori GPCR, efﬁciently\ndirected the protein N-myristoylation. Analysis of a tumor necrosis factor\n(TNF) fusion protein with the N-terminal 90 residues of B96Bom at its\nN-terminus revealed that (a) transmembrane domain 1 of B96Bom func-\ntioned as a type I signal anchor sequence, (b) the N-myristoylated N-ter-\nminal domain (58 residues) was translocated across the membrane, and (c)\ntwo N-glycosylation motifs located in this domain were efﬁciently N-gly-\ncosylated. In addition, when Ala4 in the N-myristoylation motif of\nB96Bom90-TNF, Met-Gly-Gln-Ala-Ala-Thr(1–6), was replaced with Asn\nto generate a new N-glycosylation motif, Asn-Ala-Thr(4–6), efﬁcient N-gly-\ncosylation was observed on this newly introduced N-glycosylation site in\nthe expressed protein. These results indicate that\nthe N-myristoylated\nN-terminus of B96Bom is translocated across the membrane and exposed\nto the extracellular surface. To our knowledge, this is the ﬁrst report show-\ning that a eukaryotic transmembrane protein can be N-myristoylated and\nthat the N-myristoylated N-terminus of the protein can be translocated\nacross the membrane.\nCovalent attachment of fatty acids to proteins is a\nwell-recognized form of protein modiﬁcation. Two\nforms of protein fatty acylation, protein N-myristoy-\nlation and protein palmitoylation, occur on a wide\nvariety of\neukaryotic and viral proteins. Protein\nN-myristoylation is the attachment of myristic acid, a\n14-carbon saturated fatty acid, to the N-terminal Gly\nof proteins [1–5]. Many of the N-myristoylated pro-\nteins play key roles in regulating cellular structure and\nfunction. They include proteins involved in a wide\nvariety of cellular signal-transduction pathways such\nas protein kinases, phosphatases, guanine nucleotide-\nbinding proteins, and Ca2+-binding proteins. In many\ncases, the functions of these N-myristoylated proteins\nare regulated by reversible membrane binding medi-\nated by protein N-myristoylation.\nProtein N-myristoylation has been recognized as a\nprotein modiﬁcation that occurs mainly on cytoplasmic\nand nucleoplasmic proteins, and no eukaryotic trans-\nmembrane protein has been demonstrated to be\nN-myristoylated so far. In viral proteins, however,\nprotein N-myristoylation has been found to occur on\ntransmembrane proteins such as the large L envelope\nprotein of hepatitis B virus (HBV), a polytopic mem-\nbrane protein [6,7]. Therefore, it seemed possible that\nN-myristoylated eukaryotic transmembrane proteins\nexist. In this study, we tried to search for eukaryotic\ntransmembrane proteins\nthat are N-myristoylated.\nAs protein palmitoylation, another type of protein\nacylation,\nis often observed on G-protein-coupled\nreceptors (GPCRs) [8], which are seven transmembrane\nreceptors, we focused our attention on these recep-\ntors to search for N-myristoylated transmembrane\nproteins.\nGPCRs represent a superfamily of proteins\nthe\nfunction of which is dependent on heterotrimeric\nGTP binding and hydrolyzing proteins (G-proteins).\nThey link the cells to their environment by receiving\nstimuli, relaying the message to the cells, and regula-\nting the cellular response [9,10]. GPCRs represent one\nof the largest protein superfamilies in the eukaryotic\ngenome [11,12]. In fact,  9000 entries are listed in the\nGPCR database which is available as a public WWW-\nserver (http://www.gpcr.org/7tm/) [13]. To search for\nN-myristoylated GPCRs, possible candidates with an\nN-myristoylation consensus motif at their N-termini\nwere ﬁrst extracted from the GPCR database and then\nthe susceptibility of the proteins to protein N-myristoy-\nlation was examined experimentally. It was found that\nthe N-terminus of B96Bom [14,15], a Bombyx mori\nGPCR (tyramine receptor), is efﬁciently N-myristoylat-\ned and that\nthe N-myristoylated N-terminus of\nB96Bom is translocated across the membrane and\nexposed to the extracellular surface. To our knowledge,\nthis is the ﬁrst report showing that a eukaryotic trans-\nmembrane protein can be N-myristoylated and that the\nN-myristoylated N-terminus of\nthe protein can be\ntranslocated across the membrane.\nResults\nThe N-terminus of B96Bom, an insect GPCR,\nis N-myristoylated\nTo search for N-myristoylated transmembrane pro-\nteins, the susceptibility of the N-terminal sequence of\nseveral GPCRs to protein N-myristoylation was evalu-\nated by metabolic labeling using an in vitro translation\nsystem. Three GPCRs\ntyramine receptor\n(B96Bom), human dopamine D4 receptor (DRD4),\nand human b-adrenoceptor type 1 (ADRb1)] with an\nN-myristoylation consensus motif at their N-termini\nwere selected as possible candidates (Fig. 1).\n[B. mori\nThe N-terminal 10 residues, including the initiating\nMet of the mature domain of tumor necrosis factor\n(TNF), were replaced with those of these three recep-\ntors to generate B96Bom10-TNF, DRD4-TNF, and\nADRb1-TNF, respectively. Then, the susceptibility of\nthese proteins to protein N-myristoylation was evalu-\nated by in vitro metabolic labeling using rabbit reticu-\nlocyte lysate. In this case, Dpro-TNF,\nthe mature\ndomain of TNF in which the initiating Met was intro-\nduced at the N-terminus, was used as a nonmyristoyl-\nated control TNF. As shown in Fig. 2B, translation of\nmRNAs\ncoding for Dpro-TNF, B96Bom10-TNF,\nDRD4-TNF, and ADRb1-TNF in the presence of\n[3H]leucine gave rise to protein bands with the expec-\nted molecular mass (17 kDa). Translation of mRNA\ncoding for Gi1a-TNF, in which the N-terminal 10 resi-\ndues of the naturally N-myristoylated Gi1a protein\nwere linked to the N-terminus of the mature domain\nof TNF, gave rise to a protein band with an expected\nmolecular mass of 19 kDa. The fainter protein bands\nwith a molecular mass  2 kDa larger than expected\nwere also observed in the translation products of Gi1a-\n[3H]Myristic acid labeling\nTNF and Dpro-TNF.\nrevealed that Dpro-TNF and ADRb1-TNF were not\nN-myristoylated,\nand\nDRD4-TNF were N-myristoylated (Fig. 2C). The efﬁ-\nciency of [3H]myristic acid incorporation ([3H]myristic\nacid incorporation ⁄ [3H]leucine\ninto\nB96Bom10-TNF was comparable to that into Gi1a-\nindicating that B96Bom10-TNF is efﬁciently\nTNF,\nwhereas\nA\nB\nC\nA\nB\nC\nN-myristoylated. In contrast, the efﬁciency of [3H]my-\nristic acid incorporation into DRD4-TNF was relat-\nively low.\nTo conﬁrm the N-myristoylation of B96Bom, the\nsusceptibility of B96Bom to protein N-myristoylation\nwas evaluated by in vitro metabolic labeling using\nintact B96Bom cDNA. As the efﬁciency of protein\nexpression directed by full-length B96Bom cDNA was\nvery low in both in vitro and in vivo expression systems\n(data not shown), two fusion proteins (B96Bom58-\nTNF, B96Bom90-TNF) in which the N-terminal 58 or\n90 residues of B96Bom were linked to the mature\ndomain of TNF were generated. Then their susceptibil-\nity to protein N-myristoylation was evaluated. The\nN-terminal 58 residues are the putative N-terminal ex-\noplasmic domain of B96Bom, and N-terminal 90 resi-\ndues include transmembrane domain 1 (TM1) and its\nN-terminal and C-terminal ﬂanking regions. As shown\nin Fig. 3B,\nfor\nB96Bom58-TNF gave rise to two translation products:\none was the major product with an apparent molecular\nmass of 28 kDa, and the other was a fainter band with\nan  4 kDa larger molecular mass. [3H]Myristic acid\nlabeling revealed that the 28 kDa major protein band\nwas efﬁciently N-myristoylated (Fig. 3C). Translation\nof mRNA coding for B96Bom90-TNF gave rise to two\ntranslation products: one was the major product with\nan apparent molecular mass of 27 kDa, and the other\nwas a fainter band with a molecular mass of 20 kDa.\n[3H]Myristic acid labeling revealed that the 27 kDa\nmajor protein band was also efﬁciently N-myristoylat-\ned (Fig. 3C). These results indicate that the N-terminus\nof intact B96Bom is cotranslationally N-myristoylated.\nThe N-terminal domain of B96Bom is\ntranslocated across the membrane\nIn many of the GPCRs, the N-terminal domain is\ntranslocated across the endoplasmic reticulum during\nprotein synthesis and then targeted to the plasma\nmembrane and exposed to the extracellular space. To\ndetermine whether\nthe N-myristoylated N-terminal\ndomain of B96Bom is translocated across the mem-\nbrane, the susceptibility of two N-glycosylation motifs\n[Asn-Tyr-Thr(11–13), Asn-Tyr-Thr(16–18)]\nlocated at\nthe N-terminal domain of B96Bom to protein N-glyco-\nsylation was evaluated by in vivo metabolic labeling in\ntransfected COS-1 cells. When pro-GLC-TNF, a\nmodel N-glycosylated type II transmembrane protein\nwith an N-glycosylation site in the mature domain of\nTNF, was expressed in COS-1 cells, a protein band\nwith a molecular mass of 30 kDa was generated, as\nshown in Fig. 4B (lane 1). Glycopeptidase F treatment\nof this protein gave rise to a protein band with an\n 4 kDa smaller molecular mass (26 kDa), indicating\nthe pro-GLC-TNF is N-glycosylated when\nthat\nexpressed in COS-1 cells (lane 2). When B96Bom58-\nTNF cDNA was\ntransfected into COS-1 cells, a\n28 kDa protein was expressed (lane 3); however, the\nA\nB\nmolecular size of this protein was not affected by treat-\nment with glycopeptidase F (lane 4). In contrast,\nexpression of B96Bom90-TNF cDNA gave rise to two\nproteins: one was the major product with a molecular\nmass of 33 kDa, and the other was a fainter band of\n27 kDa. Glycopeptidase F treatment of these two pro-\nteins gave rise to three protein bands: a major 27 kDa\nprotein, a minor 30 kDa protein and a trace amout of\n33 kDa protein. These results indicate that both of the\ntwo N-glycosylation motifs located at the N-terminal\ndomain of B96Bom90-TNF directed protein N-glyco-\nsylation, and diglycosylated protein with a molecular\nmass of 33 kDa was generated. [3H]Myristic acid labe-\nling revealed that both the diglycosylated 33 kDa pro-\nprotein were\ntein\n27 kDa\nefﬁciently N-myristoylated (Fig. 4C,\nlane 3). These\nresults clearly indicate that the TM1 of B96Bom func-\ntioned as a type I signal anchor sequence, and the\nN-terminal domain (58 residues) was\ntranslocated\nacross the membrane.\nand\nThe N-myristoylated N-terminus of B96Bom\nis translocated across the membrane\nIn the membrane proteins with extracellular N-termini\nthat lack a cleaved signal sequence, the TM1 functions\nas a type I signal anchor sequence, and the portion\nbefore TM1 translocates the membrane post-transla-\ntionally [16,17]. It is clear from the results obtained in\nsection that Asn11 and Asn16 of\nthe previous\nB96Bom is translocated across the membrane, as these\ntwo Asn residues are N-glycosylated. However, whe-\nther or not\nthe N-myristoylated N-terminus of\nB96Bom is translocated across the membrane is not\nclear from these experimental results. To clarify this,\nwe introduced an additional N-glycosylation site at a\nposition close to the N-terminus and evaluated its\nthis\nsusceptibility to protein N-glycosylation. For\npurpose, Ala4 in B96Bom90-TNF was replaced with\nAsn to generate a new N-glycosylation site, Asn-\nAla-Thr(4–6), and the N-glycosylation of the mutant\nobtained B96Bom90A4N-TNF expressed in COS-1\ncells was evaluated by glycopeptidase F treatment.\nWhen the B96Bom90A4N-TNF was expressed in\nCOS-1 cells, two protein bands were obtained: one\nwas a major protein band with an apparent molecular\nmass of 35 kDa, and the other was a fainter protein\nband with 27 kDa molecular mass, as\nshown in\nFig. 5B (lane 3). The molecular size of the major pro-\ntein band was slightly larger than that of diglycosylat-\nindicating that\ned B96Bom90-TNF (lanes 1 and 3),\nthe expressed B96Bom90A4N-TNF was triglycosylat-\ned. As shown in Fig. 5B lane 2, mild treatment of the\nA\nB\nC\nthat the N-myristoylated N-terminus of B96Bom90-\nTNF is\nthe membrane and\nexposed to the extracellular space.\ntranslocated across\nN-Myristoylation of the N-terminus of B96Bom is\nnot involved in the membrane translocation of\nthe N-terminal domain of B96Bom\nthe\nTo determine whether the N-myristoylation of\nN-terminus of B96Bom plays a role in the membrane\ntranslocation of the N-terminal domain of B96Bom,\nthe Gly residue at position 2 of B96Bom90-TNF was\nreplaced with Ala to generate a nonmyristoylated\nmutant, B96Bom90G2A-TNF, and its membrane\ntranslocation was evaluated by assessing the suscepti-\nbility of this protein to protein N-glycosylation. As\nin vitro translation of\nshown in Fig. 6B lane 1,\nB96Bom90-TNF cDNA gave rise to two protein\nbands: one was the major product with an apparent\nmolecular mass of 27 kDa, and the other was a fainter\nband with a molecular mass of 20 kDa. In contrast,\nin vitro translation of B96Bom90G2A-TNF cDNA\ngave rise to three protein bands: a major protein band\nA\nB\nimmunoprecipitated B96Bom90-TNF with glycopepti-\ndase F gave rise to one protein band with an interme-\ndiate size between those of the two original proteins.\nIn contrast, the same treatment of the immunoprecipi-\ntated B96Bom90A4N-TNF gave rise to two protein\nbands between the two original protein bands (lane 4),\nindicating that the major product of B96Bom90A4N-\nTNF expressed in COS-1 cells\ntriglycosylated.\n[3H]Myristic acid labeling revealed that the triglycosyl-\nated B96Bom90-TNF is N-myristoylated, and the\nN-myristoylated triglycosylated protein band was shif-\nted to a nonmyristoylated 27 kDa protein band by\ntreatment with glycopeptidase F (Fig. 5D, lanes 3 and\n4). These results indicate that Asn4, which is close to\nin B96Bom90A4N-TNF\nthe N-myristoylated Gly2,\nis quite probable\nwas N-glycosylated. Therefore,\nthe N-terminus of B96Bom is not\nFig. 6. N-Myristoylation of\ninvolved in the membrane translocation of the N-terminal domain\nof B96Bom.\nand\nB96Bom90G2A-TNF were translated in vitro in the presence of\n[3H]leucine or [3H]myristic acid using rabbit reticulocyte lysate. After\nimmunoprecipitation with antibody to TNF, the labeled translation\nproducts were analyzed by SDS ⁄ PAGE and ﬂuorography (B).\ncDNAs encoding B96Bom90-TNF and B96Bom90G2A-TNF were\ntransfected into COS-1 cells, and their expression was evaluated\nby Western blotting analysis. N-Glycosylation of the expressed pro-\ntein was determined by the change in its molecular mass by treat-\nment of\nthe immunoprecipitated protein with glycopeptidase F\n(GPF) (C).\nwith an apparent molecular mass of 29 kDa and two\nfainter bands with molecular masses of 31 kDa and\n[3H]Myristic acid labeling revealed\n20 kDa (lane 2).\nthat none of these three protein bands was N-myristo-\nlane 4). When B96Bom90G2A-TNF\nylated (Fig. 6B,\nwas expressed in COS-1 cells, three protein bands with\nmolecular masses of 35, 32, and 29 kDa were detected\n(Fig. 6C, lane 3). Glycopeptidase F treatment of these\nproteins gave rise to a single protein band with a\nmolecular mass of 29 kDa (lane 4),\nindicating that\nthe two protein bands with higher molecular mass\nwere N-glycosylated. These results clearly indicated\nthe N-terminal domain of nonmyristoylated\nthat\nB96Bom90G2A-TNF was\nthe\nmembrane as efﬁciently as that of N-myristoylated\nB96Bom90-TNF. These results reveal that N-myristoy-\nlation of the N-terminus of B96Bom does not play a\nrole in the membrane translocation of the N-terminal\ndomain of B96Bom.\ntranslocated across\ntranslocation of\nIn eukaryotic cellular proteins, protein N-myristoyla-\ntion occurs mainly on cytoplasmic or nucleoplasmic\nproteins. In viral proteins, however, protein N-myris-\ntoylation has been found to occur on transmembrane\nproteins. It has been found that the large L envelope\nprotein of HBV, a polytopic membrane protein,\nis\nN-myristoylated, and this modiﬁcation is involved in\nthe intracellular retention of this protein and thus is\nessential for viral\ninfectivity [6,7,18,19]. This protein\nhas the peculiar capacity to form two transmembrane\ntopologies via an as yet uncharacterized process of\nits pre-S\npartial post-translational\ndomain across membranes [20–22]. The HBV L protein\nwith its N-myristoylated N-terminal pre-S domain\nplays vital roles in the viral\nlife cycle by mediating\nreceptor binding during host cell attachment, by per-\nforming a matrix-like function in nucleocapsid envel-\nopment and by exerting various regulatory effects. The\nmultifunctional nature of HBV L protein depends on\nits dual transmembrane topology, which is established\nafter the completion of polypeptide synthesis [23,24].\nThus, protein N-myristoylation occurs on this viral\ntransmembrane protein and seems to be involved in\nthe speciﬁc functions of the protein. However, so far,\nprotein N-myristoylation has not been observed on\neukaryotic transmembrane proteins. In this study, to\nsearch for a eukaryotic N-myristoylated transmem-\nbrane protein, the susceptibility of the N-terminus of\nGPCR, a polytopic transmembrane protein with vital\nroles in intracellular signal-transduction cascades, to\nprotein N-myristoylation was evaluated.\n[1,26,27]. These\nTo search for N-myristoylated GPCRs, possible can-\ndidates with an N-myristoylation consensus motif at\ntheir N-terminus were ﬁrst extracted from the GPCR\ndatabase. Protein N-myristoylation is catalyzed by\nN-myristoyltransferase, a member of the GCN5 acetyl-\ntransferase superfamily of proteins [25]. The precise\nsubstrate speciﬁcity of this enzyme has been character-\nized using puriﬁed enzyme and synthetic peptides\nsequences of known\nderived from the N-terminal\nN-myristoylated proteins\nstudies\nrevealed that,\nin addition to Gly at position 2, the\namino acids at positions 3, 6, and 7 play an important\nrole in substrate recognition by N-myristoyltransferase.\nRecently, we studied the amino-acid requirements at\npositions 3, 6 and 7 in the N-myristoylation motif by\nvertical-scanning mutagenesis of amino acids in a\nthe\nmodel N-myristoylation motif. We found that\ncombination of amino acids at positions 3, 6 and 7 is\na major determinant of protein N-myristoylation\n[28,29]. For example, when Ser was located at position\n6, as many as 11 amino acids (Gly, Ala, Ser, Cys, Thr,\nVal, Asn, Leu, Ile, Gln, His) were permitted at posi-\ntion 3 to allow efﬁcient protein N-myristoylation. In\ncontrast, when other amino acids such as Thr, Ala or\nPhe were located at position 6, only three amino acids\n(Ala, Asn, Gln) were permitted at position 3 to allow\nefﬁcient protein N-myristoylation. To search for\nN-myristloylated GPCRs, these rules were applied to\nthe N-terminal sequence of GPCRs listed in the GPCR\ndatabase [13]. As a result, several biogenic amine re-\nceptors such as octopamine receptor, tyramine recep-\ntor, dopamine receptor D4, and serotonin receptor\nwere found to have N-terminal sequences that are con-\nsistent with these rules.\nIn previous studies, we have shown that the suscepti-\nbility of a protein to cotranslational N-myristoylation\ncan easily be detected by metabolic labeling in an in vitro\nor in vivo expression system using a fusion protein in\nwhich the N-terminal sequence ( 10 amino-acid resi-\ndues) of the query protein was fused to a model protein\n[28–32]. In the present study, we used this strategy to\ndetect protein N-myristoylation of GPCRs. We found\nthat the N-terminus of B96Bom, a B. mori GPCR, is\nefﬁciently N-myristoylated. We have recently cloned the\ncDNA of this protein and expressed it in HEK-293 cells.\nWe showed that B96Bom is a B. mori tyramine receptor\nwhich is negatively coupled to adenylate cyclase [14,15].\nUsing this B96Bom cDNA, we further analyzed the\nN-myristoylation of this protein.\nThe seven transmembrane domains characteristic of\nGPCRs give rise to several structural features. Each\nGPCR has an N-terminal extracellular domain, the\nseven transmembrane helices, three extracellular and\nthree intracellular loops, and an intracellular C-ter-\nminal domain. In the membrane proteins with extracel-\nlular N-termini that lack a cleaved signal sequence,\nincluding GPCRs, TM1 functions as a type I signal\nanchor sequence, and the portion before TM1 translo-\ncates the endoplasmic reticulum post-translationally\n[16,17]. Protein N-myristoylation is a very early\ncotranslational protein modiﬁcation, which may occur\nas early as when the ﬁrst 50 amino acids of the nascent\npolypeptide chain are polymerized [33]. Therefore,\nit\nseems likely that the N-myristoylation of B96Bom pre-\ncedes the translocation of the N-terminus of this pro-\nIn many\ntein across\nN-myristoylated proteins, the N-terminal myristic acid\nmoiety functions as a membrane anchor and immobil-\nizes the protein on the membrane. Therefore, it was\npossible that the membrane binding mediated by the\nN-myristoylated N-terminus of B96Bom inihibits the\nmembrane translocation of the N-terminal domain of\nthis protein. The experimental results obtained with\nB96Bom90A4N-TNF clearly show that this is not the\ncase. The oligosaccharyl transferase is required for the\ntransfer of oligosaccharides from a dolichol carrier to\nAsn-X-Ser ⁄ Thr acceptor sites for N-glycosylation in\nnascent polypeptides. N-linked glycosylation occurs on\nthe lumenal side of the endoplasmic reticulum during\nor soon after translocation of nascent polypetides [34].\nAs the active site of the oligosaccharyl transferase is\nlocated  15 residues away from the endoplasmic reti-\nculum membrane [35], the fact that Asn4 close to the\nN-myristoylated Gly2 in this mutant was N-glycosylat-\ned strongly indicated that the N-myristoylated N-ter-\nminus of B96Bom90-TNF is translocated across the\nendoplasmic reticulum membrane through transloca-\ntion machinery and exposed to the lumenal side of the\nendoplasmic reticulum. It\nis not clear whether the\nN-myristoylated N-terminus of B96Bom anchors to\nthe membrane or not. Therefore, there are two poss-\nible models for the transmembrane topology of the\nN-terminal domain of B96Bom, as shown in Fig. 7.\nThe physiological roles of protein N-myristoylation\nof B96Bom are not clear at present. The fact that the\nN-terminus of nonmyristoylated B96BomG2A-TNF\nwas translocated across the membrane as efﬁciently as\nthat of N-myristoylated B96Bom90-TNF indicated\nthat N-myristoylation of the N-terminus of B96Bom\ndoes not play a role in the membrane translocation of\nthe N-terminal domain of B96Bom. It has been found\nthat post-translational modiﬁcations such as N-glyco-\nsylation, phosphorylation, and palmitoylation play\nimportant roles in the functional regulation of GPCRs\n[8,36,37]. Of these three modiﬁcations, only N-glycosy-\nlation occurs on the N-terminal extracellular domain,\nas does protein N-myristoylation. N-Glycosylation of\nGPCRs on one or more Asn residues in the N-terminal\ndomain seems to be involved in ligand binding and cell\nsurface expression of the receptor [37]. Therefore, it is\npossible that N-myristoylation of B96Bom is also\nIn addition to the\ninvolved in these phenomena.\nB96Bom receptor, several other biogenic amine recep-\ntors\nsuch as octopamine receptors, dopamine D4\nreceptors, and a serotonin receptor were found to have\nthe N-myristoylation consensus motif at their N-ter-\nmini (Fig. 1). However, none of these receptors has\nbeen shown to be N-myristoylated so far. To our\nknowledge, no obvious similarity has been reported in\nthe function of these receptors except that all these are\nstudies will be\nbiogenic amine receptors. Further\nrequired to elucidate the roles of protein N-myristoyla-\ntion in the functions of the B96Bom receptor or other\nbiogenic amine receptors.\nExperimental procedures\nMaterials\nRestriction\nenzymes,\nendonucleases, DNA-modifying\nRNase inhibitor, and Taq DNA polymerase were purchased\nfrom Takara Shuzo, Kyoto Japan. The mCAP RNA cap-\nping kit and proteinase K were from Stratagene (La Jolla,\nCA, USA). RNase was purchased from Boehringer-Mann-\nheim (Mannheim, Germany). Rabbit reticulocyte lysate was\nfrom Promega (Madison, WI, USA). [3H]Leucine, [3H]my-\nristic acid and Amplify were from Amersham, Piscataway,\nNJ, USA. The Dye Terminator Cycle Sequencing kit was\nfrom Applied Biosystems (Foster City, CA, USA). Polyclonal\nantibody to human TNF was purchased from R & D Sys-\ntems (Minneapolis, MN, USA). Protein G–Sepharose was\nfrom Pharmacia Biotech (Piscataway, NJ, USA). Other rea-\ngents purchased from Wako Pure Chemical (Osaka, Japan),\nDaiichi Pure Chemicals (Tokyo, Japan), and Seikagaku\nKogyo (Tokyo, Japan) were of analytical or DNA grade.\nPlasmid construction\nPlasmid pBluescript II SK(+) lacking ApaI and HindIII\nsites was constructed as previously described [38], and des-\nignated pB. Plasmid pBDpro-TNF, which contains a cDNA\ncoding for the mature domain of TNF, was constructed as\ndescribed [32]. Plasmid pBB96Bom10-TNF was constructed\nusing PCR. pBDpro-TNF served as the template and two\noligonucleotides (Bom10, B1) served as primers (Table 1).\nAfter digestion with BamHI and PstI, the ampliﬁed product\nwas subcloned into pB at the BamHI and PstI sites. Plas-\nmids pBDRD4-TNF and ADRb1-TNF were constructed\nby a method similar to that used for pBB96Bom10-TNF\nusing two oligonucleotide pairs\n(DRD4 plus B1 and\nADRB1 plus B1, respectively) as primers (Table 1). Plasmid\npBGi1a-TNF was constructed as previously described [32].\nThe cDNAs coding for B96Bom58-TNF and B96Bom90-\nin which the N-terminal 58 and 90 residues of\nTNF,\nB96Bom were linked to the N-terminus of\nthe mature\ndomain of TNF, were constructed by using PCR. For this\nprocedure, previously cloned B96Bom cDNA [14] sub-\ncloned in pcDNA3 served as the template, and two oligo-\nnucleotide pairs (BomN plus Bom58 and BomN plus\nBom90, respectively) served as primers (Table 1). After\ndigestion with BamHI and XhoI, the ampliﬁed products\nwere subcloned into pBGi1a-TNF at the BamHI and XhoI\nsites. Plasmid pBB96Bom90A4N-TNF was constructed by\na method similar to that used for pBB96Bom90-TNF using\ntwo oligonucleotides\nas primers\n(Table 1). Plasmid pBB96Bom90G2A-TNF was constructed\nby a method similar to that used for pBB96Bom90-TNF\nusing two oligonucleotides (BomG2A, Bom90) as primers\n(Table 1). Plasmid pBpro-GLC-TNF was constructed by\nusing PCR. In this mutant, an N-glycosylation site (Asn-X-\nSer) was introduced at amino acid positions 45–47 in the\nmature domain of TNF. For this procedure, pBpro-TNF\nserved as the template and two sets of oligonucleotides (T3\nplus GLC-6 and MUTB1 plus B1) as primers. After diges-\ntion with SacI and KpnI, the ampliﬁed product was sub-\ncloned into pB at the SacI and KpnI sites. The DNA\nsequences of these recombinant cDNAs were conﬁrmed by\nthe dideoxynucleotide chain termination method [39]. The\ncDNAs coding for TNF mutants were subcloned into\npcDNA3, and the pcDNA3 constructs obtained were used\nfor transfection assays.\nIn vitro transcription and translation\nMethods essentially the same as those described previously\nwere used [29]. T3 polymerase was used to obtain transcripts\nof these cDNAs subcloned into pB vector. These transcripts\nwere puriﬁed by phenol ⁄ chloroform extraction and ethanol\nprecipitation before use. Subsequently,\ntranslation was\ncarried out using rabbit reticulocyte lysate (Promega) in the\npresence of [3H]leucine or [3H]myristic acid under conditions\nrecommended by the manufacturer. The mixture [17.5 lL\nrabbit reticulocyte lysate; 0.5 lL 1 mm leucine-free amino-\nacid mixture or 1 mm complete amino acid mixture; 4.0 lL\n[3H]leucine (5 lCi) or [3H]myristic acid (25 lCi); 3.0 lL\nmRNA] was incubated at 30 C for 90 min.\nTransfection of COS-1 cells and determination\nof N-myristoylated and N-glycosylated proteins\nThe simian virus 40-transformed African Green monkey\nkidney cell\nline, COS-1, was maintained in Dulbecco’s\nmodiﬁed Eagle’s medium (Invitrogen, Carlsbad, CA, USA)\nsupplemented with 10% fetal bovine serum (Invitrogen).\nCells (2 · 105) were plated on to 35-mm diameter dishes\n1 day before transfection. pcDNA3 construct (2 lg; Invitro-\ngen) containing mutant TNF cDNA was used to transfect\neach plate of COS-1 cells along with 3 lL LipofectAmine\n(2 mgÆmL)1; Invitrogen) in 1 mL serum-free medium. After\nincubation for 5 h at 37 C,\nthe cells were refed with\nserum-containing medium and incubated again at 37 C for\n24 h. The cells were then washed twice with 1 mL serum-\nfree Dulbecco’s modiﬁed Eagle’s medium and incubated for\n5 h in 1 mL Dulbecco’s modiﬁed Eagle’s medium with 2%\nacid\nfetal\n(100 lCiÆmL)1). Subsequently, the cells were washed three\ntimes with Dulbecco’s phosphate buffered saline, collected\nwith cell scrapers, and lysed with 200 lL RIPA buffer\n[50 mm Tris ⁄ HCl (pH 7.5), 150 mm NaCl, 1% Nonidet\nP40, 0.5% sodium deoxycholate, 0.1% SDS, proteinase\ninhibitors] on ice for 20 min. The cell lysates were centri-\nfuged at 21 000 g at 4 C for 15 min in a microcentrifuge\n(Hitachi-CF15D2), and the supernatants were collected.\nAfter immunoprecipitation with antibody to TNF, the sam-\nples were analyzed by SDS ⁄ PAGE and ﬂuorography. To\nexamine the N-glycosylation of TNF mutants, 20 lL total\ncell lysate of each group of transfected cells or TNF sample\nimmunoprecipitated from total cell lysates were treated with\n10 or 50 mUÆmL)1 glycopeptidase F at 37 C for 1 h and\nthen analyzed by Western blotting.\nWestern blotting\nTNF samples immunoprecipitated from in vitro translation\nproducts or total cell lysates of each group of transfected\ncells were resolved by SDS ⁄ PAGE (12.5% gel) and then\ntransferred to an Immobilon-P transfer membrane (Milli-\npore, Billerica, MA, USA). After being blocked with nonfat\nmilk, the membrane was probed with a speciﬁc goat anti-\nbody to human TNF as described previously [40]. Immuno-\nreactive proteins were speciﬁcally detected by incubation\nwith horseradish peroxidase-conjugated anti-goat\nIgG\n(Santa Cruz, Biotech, Santa Cruz, CA, USA). The mem-\nbrane was developed with ECL Western blotting reagent\n(Amersham) and exposed to an X-ray ﬁlm (Kodak). Quanti-\ntative analysis of immunoreactive proteins on the membrane\nwas carried out using a storage phosphor imaging screen and\nGS-250 Molecular Imager (Bio-Rad, Hercules, CA, USA).\nImmunoprecipitation\nSamples containing TNF mutants were immunoprecipitated\nwith a speciﬁc goat polyclonal antibody to human TNF (R\n& D Systems) as described [38].\nSDS ⁄ PAGE and ﬂuorography\nSamples were denatured by boiling for 3 min in SDS sam-\nple buffer followed by analysis by SDS ⁄ PAGE on a 12.5%\ngel. Thereafter, the gel was ﬁxed and soaked in AmplifyTM\n(Amersham) for 30 min. The gel was dried under vacuum\nand exposed to an X-ray ﬁlm (Kodak) for an appropriate\nperiod. Quantitative analysis of the labeled proteins was\ncarried out by scanning the ﬂuorogram using an imaging\ndensitometer (Bio-Rad GS-700).\nAcknowledgements\nPart of this work was supported by a Grant-in-Aid for\nScientiﬁc Research (No. 12660080, No. 15580080)\nfrom the Ministry of Education, Science, and Culture\nof Japan.\n1 Towler DA, Gordon JI, Adams SP & Glaser L (1988)\nThe biology and enzymology of eukaryotic protein acy-\nlation. Annu Rev Biochem 57, 69–99.\n2 Spiegel AM, Backlund PS, Butrynski JE, Jones TLZ &\nSimonds WF (1991) The G protein connection: molecu-\nlar basis of membrane association. Trends Biochem\nSci 16, 338–341.\n3 Boutin JA (1997) Myristoylation. Cell Signal 9, 15–35.\n4 Resh MD (1999) Fatty acylation of proteins: new\ninsights into membrane targeting of myristoylated and\npalmitoylated proteins. Biochim Biophys Acta 1451,\n1–16.\n5 Farazi TA, Waksman G & Gordon JI (2001) The biol-\nogy and enzymology of protein N-myristoylation. J Biol\nChem 276, 39501–39504.\n6 Persing D, Varmus H & Ganem D (1987) The preS1\nprotein of hepatitis B virus is acylated at its amino ter-\nminus with myristic acid. J Virol 61, 1672–1677.\n7 Prange R, Clemen A & Streeck RE (1991) Myristylation\nis involved in intracellular retention of hepatitis B virus\nenvelope proteins. J Virol 65, 3919–3923.\n8 Qanbar R & Bouvier M (2003) Role of\npalmitoylation ⁄ depalmitoylation reactions in\nG-protein-coupled receptor function. Pharmacol Ther\n97, 1–33.\n9 Gether U (2000) Uncovering molecular mechanisms\ninvolved in activation of G protein-coupled receptors.\nEndocr Rev 21, 90–113.\n10 Pierce L, Premont RT & Lefkowitz RJ (2002) Seven-\ntransmembrane receptors. Nat Rev Mol Cell Biol 3,\n639–650.\n11 Bockaert J & Pin JP (1999) Molecular tinkering of G\nprotein-coupled receptors: an evolutionary success.\nEMBO J 18, 1723–1729.\n12 Bockaert J, Claeysen S, Becamel C, Pinloche S &\nDumuis A (2002) G protein-coupled receptors: domi-\nnant players in cell-cell communication. Int Rev Cytol\n212, 63–132.\n13 Horn F, Vriend G & Cohen FE (2001) Collecting\nand harvesting biological data: the GPCRDB and\nNucleaRDB information systems. Nucleic Acids Res 29,\n346–349.\n14 Ohta H, Utsumi T & Ozoe Y (2003) B96Bom encodes a\nBombyx mori tyramine receptor negatively coupled to\nadenylate cyclase. Insect Mol Biol 12, 217–223.\n15 Ohta H, Utsumi T & Ozoe Y (2004) Amino acid\n28 Utsumi T, Nakano K, Funakoshi T, Kayano Y, Nakao\nS, Sakurai N, Iwata H & Ishisaka R (2004) Vertical-\nscanning mutagenesis of amino acids in a model N-myr-\nistoylation motif reveals the major amino-terminal\nsequence requirements for protein N-myristoylation. Eur\nJ Biochem 271, 863–874.\nresidues involved in interaction with tyramine in the\nBombyx mori tyramine receptor. Insect Mol Biol 13,\n531–538.\n16 Friedlander M & Blobel G (1985) Bovine opsin has\nmore than one signal sequence. Nature 318, 338–343.\n17 Kanner EM, Klein IK, Friedlander M & Simon SM\n(2002) The amino terminus of opsin translocates ‘post-\ntranslationally’ as efﬁciently as cotranslationally.\nBiochemistry 41, 7707–7715.\n18 Gripon P, Le Seyec J, Rumin S & Guguen-Guillouzo C\n(1995) Myristylation of the hepatitis B virus large sur-\nface protein is essential for viral infectivity. Virology\n213, 292–299.\n19 Bruss V, Hagelstein J, Gerhardt E & Galle PR (1996)\nMyristylation of the large surface protein is required for\nhepatitis B virus in vitro infectivity. Virology 218,\n396–399.\n20 Bruss V, Lu X, Thomssen R & Gerlich WH (1994)\nPost-translational alterations in transmembrane topol-\nogy of the hepatitis B virus large envelope protein.\nEMBO J 13, 2273–2279.\n21 Ostapchuk P, Hearing P & Ganem D (1994) A dramatic\nshift in the transmembrane topology of a viral envelope\nglycoprotein accompanies hepatitis B viral morphogen-\nesis. EMBO J 13, 1048–1057.\n22 Lambert C & Prange R (2001) Dual topology of the\nhepatitis B virus large envelope protein: determinants\ninﬂuencing post-translational pre-S translocation. J Biol\nChem 276, 22265–22272.\n23 Bruss V & Vieluf K (1995) Infection process of the\nhepatitis B virus depends on the presence of a deﬁned\nsequence in the pre-S1 domain. J Virol 69, 6652–6657.\n24 Le Seyec J, Chouteau P, Cannie I, Guguen-Guillouzo C\n& Gripon P (1999) Infection process of the hepatitis B\nvirus depends on the presence of a deﬁned sequence in\nthe pre-S1 domain. J Virol 73, 2052–2057.\n25 Dyda F, Klein DC & Hickman AB (2000) GCN5-\nrelated N-acetyltransferases: a structural overview. Annu\nRev Biophys Biomol Struct 29, 81–103.\n26 Towler DA, Adams SP, Eubanks SR, Towery DS, Jack-\nson-Machelski E, Glaser L & Gordon JI (1988) Myris-\ntoyl CoA: protein N-myristoyltransferase activities from\nrat liver and yeast possess overlapping yet distinct pep-\ntide substrate speciﬁcities. J Biol Chem 263, 1784–1790.\n27 Rocque WJ, McWherter CA, Wood DC & Gordon JI\n(1993) A comparative analysis of the kinetic mechanism\nand peptide substrate speciﬁcity of human and Saccharo-\nmyces cerevisiae myristoyl-CoA:protein N-myristoyl-\ntransferase. J Biol Chem 268, 9964–9971.\n29 Utsumi T, Sato M, Nakano K, Takemura D, Iwata H\n& Ishisaka R (2001) Amino acid residue penultimate to\nthe amino-terminal Gly residue strongly affects two\ncotranslational protein modiﬁcations, N-myristoylation\nand N-acetylation. J Biol Chem 276, 10505–10513.\n30 Utsumi T, Tou E, Takemura D, Ishisaka R, Yabuki M\n& Iwata H (1998) Met-Gly-Cys motif from G-protein\nalpha subunit cannot direct palmitoylation when fused\nto heterologous protein. Arch Biochem Biophys 349,\n216–224.\n31 Utsumi T, Sakurai N, Nakano K & Ishisaka R (2003)\nC-terminal 15 kDa fragment of cytoskeletal actin is\nposttranslationally N-myristoylated upon caspase-\nmediated cleavage and targeted to mitochondria. FEBS\nLett 539, 37–44.\n32 Utsumi T, Kuranami J, Tou E, Ide A, Akimaru K,\nHung MC & Klostergaard J (1996) In vitro synthesis of\nan N-myristoylated fusion protein that binds to the lipo-\nsomal surface. Arch Biochem Biophys 326, 179–184.\n33 Deichaite I, Casson LP, Ling HP & Resh MD (1988) In\nvitro synthesis of pp60v-src: myristoylation in a cell-free\nsystem. Mol Cell Biol 8, 4295–4301.\n34 Kaplan HA, Welply JK & Lennarz WJ (1987) Oligosac-\ncharyl transferase: the central enzyme in the pathway\nof glycoprotein assembly. Biochim Biophys Acta 906,\n161–173.\n35 Nilsson I & von Heijne G (1993) Determination of the\ndistance between the oligosaccharyltransferase active site\nand the endoplasmic reticulum membrane. J Biol Chem\n268, 5798–5801.\n36 Bohm SK, Grady EF & Bunnett NW (1997) Regulatory\nmechanisms that modulate signalling by G-protein-\ncoupled receptors. Biochem J 322, 1–18.\n37 Kaushal S, Ridge KD & Khorana HG (1994) Structure\nand function in rhodopsin: the role of asparagine-linked\nglycosylation. Proc Natl Acad Sci USA 91, 4024–4028.\n38 Utsumi T, Akimaru K, Kawabata Z, Levitan A, Toku-\nnaga T, Tang P, Ide A, Hung MC & Klostergaard J\n(1995) Human pro-tumor necrosis factor: Molecular\ndeterminants of membrane translocation, sorting and\nmaturation. Mol Cell Biol 15, 6389–6405.\n39 Sanger F, Nicklen S & Coulson AR (1977) DNA\nsequencing with chain-terminating inhibitors. Proc Natl\nAcad Sci USA 74, 5463–5467.\n40 Utsumi T, Levitan A, Hung MC & Klostergaad J\n(1993) Effects of truncation of human pro-tumor necro-\nsis factor transmembrane domain on cellular targeting.\nJ Biol Chem 268, 9511–9516.",
         "T. Utsumi,H. Ohta,Y. Kayano,N. Sakurai,Y. Ozoe",
         null
        ],
        [
         "18",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "Tissue remodelling is dependent on the integration of signals\nthat control turnover of ECM (extracellular matrix). Breakdown\nand endocytosis of collagen, a major component of the ECM, is\ncentral to this process. Whereas controlled secretion of matrix-\ndegrading enzymes (such as matrix metalloproteinases) has long\nbeen known to mediate ECM breakdown, it is becoming clear\nthat uPARAP/Endo180 (where uPARAP stands for urokinase\nplasminogen activator receptor-associated protein) serves as a\nreceptor that mediates endocytosis of collagen by several types of\ncells. In the liver, the stellate cells play a major role in turnover\nof ECM including collagens. These cells synthesize various colla-\ngens and also produce matrix metalloproteinases. In the present\nstudy, we investigated the capacity of rat hepatic stellate cells\nto endocytose and degrade 125I-labelled heat-denatured collagen\nI. It was found that the collagen is efﬁciently taken up and\ndegraded by these cells. Degradation was inhibited by inhibitors of\nlysosomal proteases (leupeptin and E-64d) and the vacuolar pro-\nton pump (concanamycin A), indicating that it takes place in lyso-\nsomes. Furthermore, endocytosed FITC-labelled collagen was\nshown to reach late endocytic compartments in which it co-\nlocalized with LysoTracker (a marker of late endocytic compart-\nments). Competition experiments showed that uPA and unlabelled\ncollagen are capable of inhibiting binding and uptake of [125I]col-\nlagen in a dose-dependent manner. Moreover, Western-blot\nanalysis of cell lysate (using a polyclonal rabbit human-Endo180\nantiserum) revealed a single band at 180 kDa. In addition,\nthe antiserum was capable of reducing [125I]collagen binding\nto the cell surface. Finally, using two primers designed from the\nhuman uPARAP/Endo180 mRNA sequence, the expression of\nuPARAP/Endo180 mRNA was detected by reverse transcriptase–\nPCR. These results together suggest\nthat uPARAP/Endo180\nmediates endocytosis of collagen in rat liver stellate cells.",
         "INTRODUCTION\nHepatic stellate cells are located within the perisinusoidal space\nbetween the basolateral surface of hepatocytes and the endothelial\ncells [1]. They are relatively scarce, < 10 % of the total resident\nliver cells are stellate cells [2]. A number of important functions\nhave been attributed to stellate cells. They store vitamin A,\nreceived from the neighbouring hepatocytes [3,4], and, since they\nencircle the hepatic sinusoids and have contractile properties,\nthey may regulate blood ﬂow through the sinusoids [5–7]. They se-\ncrete cytokines and express a number of receptors that play a\ncentral role in cell–matrix and cell–cell interactions, both in\nnormal and in injured liver [2,8]. The stellate cells exhibit two\nmain phenotypes, the ‘quiescent state’ and the ‘activated state’.\nThe quiescent cells have low proliferation rate and low ﬁbrogenic\nactivity, but after any injury to the liver the cells shift towards\na myoﬁbroblastic-like phenotype [2,5,8]. A similar activation\nprocess is observed when primary stellate cells are kept in culture\non plastic support [9,10]: the cells elongate, lose their vitamin A\ndroplets and develop myoﬁlaments in their cytoplasm [5,8]. The\nactivated cells gain functions such as proliferation and produce\nvarious growth factors and most of the ECM (extracellular\nmatrix) components [9–12]. Activated stellate cells also produce\nenzymes that can degrade the normal pericellular ECM before\nsynthesis of ﬁbrous tissue. Such enzymes include MMPs (matrix\nmetalloproteinases), a family of zinc-dependent enzymes [13,14].\nAlthough activated stellate cells produce several factors needed\nfor ECM degradation, it is conceivable that they in addition may\nremove components of the ECM by means of endocytosis. Very\nfew studies have focused on this process in stellate cells. It was\nshown by using an in vitro system that retinol-binding protein,\nreleased from hepatocytes may be internalized in the stellate cells\nby receptor-mediated endocytosis [4]. A similar mechanism may\noperate when stellate cells take up retinol-binding protein from\nblood [15].\nThe aim of the present study was to determine whether stellate\ncells are able to endocytose collagen, a main component of the\nECM. To this end we used cultured rat stellate cells, and heat-\ndenatured collagen I, labelled with 125I, was chosen as a repres-\nentative ligand. Initial experiments demonstrated that the stellate\ncells internalize and degrade [125I]collagen, and that the degrad-\nation is reduced by inhibitors of lysosomal function, indicating\nthat the cells are able to endocytose [125I]collagen. Moreover, the\nbinding of [125I]collagen was shown to be saturable, indicating\nthat it is receptor-mediated. Further investigations indicated that\nthe receptor in question could be the surface receptor uPARAP/\nEndo180 [where uPARAP stands for urokinase plasminogen\nactivator receptor-associated protein]. uPARAP/Endo180 was\nS. A. Mousavi and others\noriginally identiﬁed as a constitutively recycling surface receptor\n[16] and has recently been shown to function as a collagen recep-\ntor [17–20]. A large pool (70–90 %) of the total uPARAP/Endo180\nis maintained in endosomal compartments and the small plasma\nmembrane pool (10–30 %) is associated with clathrin-coated\npits [16,21]. uPARAP/Endo180 has also been called uPARAP\nbecause it can form a ternary complex with uPAR-bound pro-uPA.\nCollagen can block the formation of this trimolecular complex\n[22]. uPARAP/Endo180 is a member of the mannose receptor\nfamily, which also comprises the mannose receptor, the phospho-\nlipase A2 receptor and the DEC-205/gp200-MR6 receptor [23].\nThese proteins are large in size (175–200 kDa) and contain\nseveral distinct domains including an NH2-terminal cysteine-rich\ndomain, a ﬁbronectin-like type II domain, followed by either eight\nor ten tandemly arranged CTLDs (C-type lectin-like domains,\nnamed CTLD1–CTLD10), a single transmembrane domain and a\nshort cytoplasmic domain containing one or two endocytic motifs\nto direct their internalization through clathrin-coated pits [23,24].\nThe ﬁbronectin-like type II domains of all family members are\nconsidered to have collagen-binding capacity [23], but only the\nmannose receptor (through its CTLD4) and uPARAP/Endo180\n(through its CTLD2) have been shown to bind carbohydrates\n[25,26]. uPARAP/Endo180 has been identiﬁed in many cell\ntypes including monocyte-like U937 cells, vascular smooth\nmuscle cells [22], ﬁbroblasts, vessel endothelial cells, macro-\nphages [16,24,26], osteoblasts [27] and chondrocytes of young\nmice [28]. However, as far as we know, uPARAP/Endo180 has\nnot been reported to be present in stellate cells.\nThe present study presents data indicating that uPARAP/\nEndo180 may be the main receptor responsible for endocytosis of\ndenatured collagen in activated rat hepatic stellate cells.\nEXPERIMENTAL",
         "In initial experiments, we examined the ability of stellate cells\nto take up and degrade the [125I]collagen. Cells were incubated\nwith [125I]collagen at 37 ◦C for different time periods and uptake\nand degradation of ligand were determined as described in the\nExperimental section. As shown in Figure 1(A), stellate cells take\nup [125I]collagen efﬁciently. After 1 h at 37 ◦C in the presence\nof [125I]collagen, the cells had removed approx. 5 % of the avail-\nable ligand from the medium. To determine whether the ligand\nwas transported to lysosomes for degradation, acid soluble and\nacid precipitable radioactivities were determined in cells and me-\ndium. The results show that degraded ligand accumulated both\ninside the cells and in the medium. Release of acid soluble radio-\nactivity to the medium started after a lag phase lasting approx.\n15 min. Acid precipitable radioactivity in the cells increased for\napprox. 30 min and then levelled off, indicating that an equi-\nlibrium between transport to and degradation in lysosomes had\nbeen reached. Similar results were obtained when denatured 125I-\nlabelled collagen IV was used (results not shown). That de-\ngradation of collagen takes place intracellularly was veriﬁed\nby the ﬁnding that degradation products formed from [125I]TC-\ncollagen were not released to the medium during the incubation\ntime (120 min) (results not shown). Labelled degradation pro-\nducts formed from this probe are trapped in the degradative\ncompartment [33].\nWe also analysed the transport of ﬂuorescently labelled de-\nnatured collagen to late endocytic compartments in pulse–\nchase experiments. Fluorescence microscopy using FITC-labelled\ncollagen (Figure 2) showed lysosomal/late endosomal staining, as\nindicated by co-localization (yellow) due to overlap of the inter-\nnalized ligand (green) and LysoTracker (red), a marker of acidic\nendocytic compartments.\nBinding and internalization of [125I]collagen to hepatic stellate\ncells is saturable and pH-dependent, but not calcium-dependent\nTo determine whether endocytosis of [125I]collagen is receptor-\nmediated, cells were incubated with [125I]collagen in the presence\nof increasing concentrations of unlabelled collagen. The results\nobtained (Figure 1B) show that both binding (at 4 ◦C) and uptake\n(at 37 ◦C) of [125I]collagen were reduced by adding unlabelled\ncollagen. At 6 µg/ml of unlabelled collagen, binding and uptake of\n[125I]collagen were decreased by 70 and 90 % respectively. Taken\ntogether, these experiments indicate that the uptake of collagen\nrepresents a receptor-mediated process and not merely ﬂuid-phase\nendocytosis. Binding of [125I]collagen was pH-dependent, and was\nreduced by approx. 50 % when pH was reduced to pH 5.5 (results\nnot shown). Binding was, however, retained in the presence of the\ncalcium chelator EGTA (2.2 mM).\nDegradation of [125I]collagen is inhibited by protease\nand proton pump inhibitors\nLeupeptin, an inhibitor of cysteine proteases, inhibited the degra-\ndation of [125I]collagen partially (Figure 3, upper panel). After\nS. A. Mousavi and others\n(A) Time course of receptor-mediated uptake and degradation of collagen: cells (∼ 106 cells/\nwell) were incubated with [125I]collagen (0.2 µg/ml) at 37 ◦C for the indicated times, washed and\nthen lysed. Cell lysates and medium were treated with TCA to determine acid-soluble and acid-\nprecipitable radioactivities as described in the Experimental section. Values are expressed as\npercentage of total initial acid precipitable radioactivity in the medium and are means +\n− S.D.\nresults for triplicate samples. Equivalent results were obtained in at least six separate experiments.\n(B) Concentration-dependent binding and uptake of collagen in stellate cells: cells were pre-\nincubated at 4 ◦C for 30 min with the indicated concentrations of unlabelled collagen. After\npreincubation, [125I]collagen (20 ng/ml) was added and the cells were incubated for 120 min\nat 4 ◦C or 30 min at 37 ◦C. The cells were then washed three times and the cell-associated\nradioactivity was determined. Values are the means +\n− S.D. results for triplicate samples.\nEquivalent results were obtained in at least three separate experiments.\n90 min of incubation at 37 ◦C, formation of acid soluble radio-\nactivity was reduced by approx. 50 % in the presence of leupeptin\n(2 mM). Release of acid-soluble degradation products from the\ncells was, however, reduced by nearly 80 % by the inhibitor. These\nresults suggest that the processing of internalized [125I]collagen\nﬁrst produces acid-soluble peptides, which are then further\ndegraded to amino acids (released from the cells). Leupeptin\nprimarily inhibits degradation of the smaller peptides. Preventing\ncollagen degradation caused, as expected, the accumulation of\nboth acid-precipitable and acid-soluble radioactivity in the cells.\nWe also assessed the effect of E-64d, a selective and irreversible\ninhibitor of lysosomal cysteine proteases, on the degradation\nof internalized [125I]collagen. As indicated in Figure 3 (upper\npanel), treatment of cells with E-64d gave results similar to\nthose seen with leupeptin treatment, although at a much lower\nconcentration (50 µM) of the inhibitor. Lysosomal degradation\nof endocytosed ligands depends on a functional vacuolar H+-\nATPase (V-ATPase) in the endosomal/lysosomal compartments\n[34]. To determine whether the inactivation of V-ATPase affects\ncollagen degradation, the cells were incubated in the presence\nof concanamycin A, a selective inhibitor of V-ATPase [35]. The\nresults obtained (Figure 3, lower panel) show that concanamycin\nA inhibited both uptake and degradation of [125I]collagen. In cells\nincubated for 90 min at 37 ◦C, the degradation of labelled colla-\ngen was reduced approx. 85 % in the presence of 0.1 µM con-\ncanamycin A. Uptake was reduced approx. 60 % at this drug\nconcentration.\nEffects of possible competitive ligands and anti-integrin antibodies\non [125I]collagen binding\nThe rapid removal of collagen from the culture medium sug-\ngested the existence of a speciﬁc receptor(s) in hepatic stellate\ncells. Several integrins have been reported to act as binding recep-\ntors for denatured collagen, including α2β1 [36,37] and αvβ3\n[38,39]. We therefore used anti-integrin antibodies and ligands of\nthese receptors in competition experiments to see whether they\nwere capable of competing with [125I]collagen for binding. At high\nconcentrations (> 20 µg/ml), the anti-integrin α2 chain inhibited\n[125I]collagen binding by approx. 10 %, whereas anti-integrin β1\nchain did not signiﬁcantly inhibit the binding of [125I]collagen. No\nadditional inhibition was observed using both antibodies together.\nPeptides containing the RGD (Arg-Gly-Asp) sequence have been\nshown to inhibit speciﬁcally the binding of denatured type I\ncollagen to the αvβ3 integrin, possibly because this sequence\nbecomes exposed on collagen denaturation [38], whereas the\nsequence DGEA (Asp-Gly-Glu-Ala) binds to α2β1 integrin [40].\n[125I]Collagen binding was therefore examined in the presence\nof the tripeptide Arg-Gly-Asp and the tetrapeptide Asp-Gly-\nGlu-Ala. Neither of these peptides (> 0.5 mg/ml) signiﬁcantly\ninhibited the binding of [125I]collagen (results not shown).\nScavenger receptors have also been shown to serve as receptors\nfor denatured collagen [41]. No inhibitory effects on the binding\nand uptake of [125I]collagen were, however, observed when cells\nwere incubated with a 50 M excess of formaldehyde-treated BSA,\na ligand of scavenger receptors [42] (results not shown).\nuPA inhibits binding and uptake of [125I]collagen\nCollagen has been shown to block the association of Endo180 with\nuPAR-bound pro-uPA [22]. Conceivably, uPA might also bind to\nthe collagen-binding receptor in the stellate cells if this receptor\nis related to the uPARAP/Endo180 receptor. It was found that uPA\ninhibits both binding and uptake of [125I]collagen to the stellate\ncells in a concentration-dependent manner (Figure 4). At 3 mg/ml,\nuPA reduced both binding and uptake of labelled collagen by\napprox. 80 %. It may, however, be argued that the reduced binding\nof collagen in the presence of uPA is due to the activation of\ncollagen-degrading enzymes (such as MMP-13) secreted by the\ncells into the medium during the 3.5 h incubation (30 min at 37 ◦C\nand 3 h at 4 ◦C). To rule out this possibility, similar experiments\nwere performed in the presence of tPA, which binds receptors\nother than uPAR [43]. However, incubation with 2.2 mg/ml tPA\ndid not inhibit signiﬁcantly the binding of collagen to the cells\n(binding as percentage of total: 1.04 +− 0.19 and 0.96 +− 0.09;\nn = 3, for control and tPA respectively), indicating that the reduced\nbinding of collagen is due to binding of uPA to its receptor (or\nCollagen endocytosis in stellate cells\nCells were pulse-labelled at 37 ◦C for 15 min with FITC-collagen (green) in the presence or absence of LysoTracker (red), then washed and either ﬁxed (images on the left) or chased for 90 min, after\nwhich the incubation continued for an additional 30 min in the presence of LysoTracker (images on the right). Co-localization of FITC-collagen with LysoTracker (yellow) is shown in the merged\nimages (lowest panels). Results represent two or three experiments.\nto uPARAP/Endo180, see below) and not to an indirect effect of\nuPA on the activation status of collagenases in the medium.\nuPARAP/Endo180 is expressed in rat hepatic stellate cells\nResult from experiments with uPA suggested that uPARAP/\nEndo180 is probably involved in the uptake of collagen. Therefore\nwe performed Western-blot analysis, using a polyclonal antibody\nagainst puriﬁed human uPARAP/Endo180 [16]. U937 cells, the\ncell line originally used to deﬁne uPARAP activity [22], were\nused as positive controls, and HepG2 cells were used as negative\ncontrols since they do not express uPARAP/Endo180 [24]. As\nshown in Figure 5, no band was detected in the preparation from\nHepG2 cells, whereas a speciﬁc band at 180 kDa was revealed\nin preparations from both hepatic stellate cells and U937 cells.\nWe also tested the ability of the Endo180 antiserum to interfere\nwith collagen binding to the cells and found that the binding\nof [125I]collagen to the stellate cells was signiﬁcantly reduced\nby 50 % when 20 µl/ml of the antiserum was used (results not\nshown). Analysis by RT–PCR showed that uPARAP/Endo180 is\nabundantly expressed in stellate cells, but is not detectable in\nKupffer cells, liver endothelial cells, hepatocytes and J774 cells\nUptake of OVA\nCTLD2 in Endo180 has been shown to contain a Ca2+-dependent\ncarbohydrate-binding determinant, which speciﬁcally binds to\nN-acetylglucosamine, mannose and fucose [26]. To determine\nwhether hepatic stellate cells express such binding activity, we\nmeasured the uptake of OVA (which contains sugars terminating\nin N-acetylglucosamine and mannose [44]). It was found that the\nstellate cells are also capable of taking up OVA. As shown in\nFigure 7, uptake and degradation of [125I]TC-OVA increases as\na function of time. Moreover, the uptake (measured after 30 min\nincubation at 37 ◦C) was reduced by 90 +− 4 and 87 +− 5 % by the\nS. A. Mousavi and others\nThe whole lysates (40 µg) of hepatic stellate cells (lane 1), U937 cells (lane 3) and HepG2 cells\n(lane 4) were analysed by SDS/PAGE on a 6 % gel and then transferred on to membrane. The\nblot was incubated ﬁrst with a polyclonal antiserum directed against Endo180 and subsequently\nwith mouse anti-rabbit IgG conjugated to horseradish peroxidase followed by detection with\nECL®. Lane 2 corresponds to the cell membrane fraction (20 µg) prepared from stellate cells.\nThe membrane was then stripped, blocked and incubated with anti-β-actin. Relative molecular\nmass standards are shown on the left (masses in kDa). The results shown were reproduced in\nat least three independent experiments.\nCells were preincubated with either leupeptin (2 mM), E-64d (2 µM) (upper panel) or increasing\nconcentrations of concanamycin (lower panel) for 15 min at 37 ◦C. After preincubation, radio-\nlabelled collagen (40 ng/ml) was added, followed by 90 min incubation at 37 ◦C and three\nwashes. Cell\nlysates and medium were treated with TCA to determine acid-soluble and\nacid-precipitable radioactivities as described in the Experimental section. Values are expressed\nas percentage of controls and are the means +\n− S.D. results for triplicate samples. Equivalent\nresults were obtained in at least three separate experiments.\nFour different\nliver cells [liver endothelial cells (rLEC), Kupffer cells (rKC),\nparenchymal cells (rPC) and stellate cells (rStel)] and J774 mouse macrophages were tested\nfor mRNA expression of Endo180 (lanes 2–6 respectively) with β-actin mRNA as an internal\ncontrol (lanes 7–11). Three experiments gave equivalent results. Sizes are indicated in bp.\nCells were preincubated with increasing concentrations of uPA for 30 min at 4 ◦C. After\npreincubation, radiolabelled collagen (20 ng/ml) was added, followed by 120 min incubation\nat 4 ◦C and three washes. For uptake, uPA was added just before the addition of [125I]collagen\nand cells were incubated at 37 ◦C for 30 min. Results are means +\n− S.D. of a representative\nexperiment performed in triplicate.\nCells (∼ 106 cells/well) were incubated at 37 ◦C with 2 nM [125I]TC-OVA in the presence and\nabsence of a 50-fold excess of unlabelled OVA or 2.2 mM EGTA. At timed intervals the cells were\nwashed and cell-associated radioactivity (acid-soluble and acid-precipitable) was measured as\ndescribed in the Experimental section. Inset: the effect of unlabelled OVA and EGTA on the\nuptake.\naddition of 50-fold excess of unlabelled OVA and 2.2 mM EGTA\nrespectively (Figure 7, inset).",
         "DISCUSSION\nThe present results demonstrate that the uptake of heat-denatured\n125I-labelled collagen in rat hepatic stellate cells is mediated by\na saturable binding site. To identify the receptor(s) involved, we\nmeasured the uptake of [125I]collagen in the presence and absence\nof antibodies and ligands that bind to potential receptor candi-\ndates. The results obtained seemed to exclude integrins (α2β1 and\nαvβ3) and scavenger receptors. The discoidin domain receptors\n(DDR1 and DDR2) are, probably, not involved in binding, as these\nreceptors require the native, triple-helical conﬁguration of the\ncollagen [45,46]. On the other hand, several observations pointed\nto the uPARAP/Endo180 as the receptor involved in binding colla-\ngen. Behrendt et al. [22] found that collagen prevents the form-\nation of a complex between pro-uPA-uPAR and uPARAP/\nEndo180. We found that the reciprocal interaction, the inter-\naction of uPA-uPAR with uPARAP/Endo180, prevents uPARAP/\nEndo180 from binding to collagen. Alternatively or in addition,\nuPA can directly bind to Endo180. This result suggested that\n[125I]collagen binding in stellate cells could be mediated by\nthe uPARAP/Endo180. That the receptor in question really is\nuPARAP/Endo180 was veriﬁed using Western-blot analysis and\nRT–PCR. Moreover, collagen binding was signiﬁcantly reduced\nby Endo180 antiserum, strongly supporting the involvement of\nuPARAP/Endo180 in mediating the uptake of collagen. Taken\ntogether, these observations are compatible with the newly deﬁned\nrole of uPARAP/Endo180 as an endocytic collagen receptor\n[17–20]. The stellate cells also bind [125I]TC-OVA, compatible\nwith the observation that uPARAP/Endo180, which belongs to\nthe mannose-receptor family, can also function as a lectin, binding\nligands with terminal mannose, fucose and N-acetylglucosamine\n[26]. Moreover, the binding of the OVA is calcium-dependent,\nwhereas the binding of collagen is not. This is in agreement with\nearlier studies of the uPARAP/Endo180 receptor [18].\nInhibition of [125I]collagen degradation by leupeptin and E-64d,\ntwo inhibitors of lysosomal thiol proteinases and concanamycin\nA, an inhibitor of the vacuolar proton pump [35,47], shows that the\nligand is effectively transported to late endocytic compartments\nfor degradation. These observations are in agreement with recent\nreports demonstrating that uPARAP/Endo180 mediates endo-\ncytosis and lysosomal degradation of collagens in ﬁbroblasts [20]\nand in a breast cancer cell line (MCF-7) [18]. To the best of our\nknowledge, the present study is the ﬁrst to demonstrate receptor-\nmediated endocytosis of collagen in hepatic stellate cells.\nThe afﬁnity of the uPARAP/Endo180 for denatured collagen\nis reduced at pH  6.5 and the ﬁnding that the proton pump in-\nhibitor also reduces cellular uptake of [125I]collagen is probably\na result of increases in pH in early or late endosomes. This will\nprevent dissociation of [125I]collagen from the collagen receptor\nin endosomes, and the ligand will recycle to the plasma membrane\nbound to the receptor. uPARAP/Endo180 has been regarded as a\nconstitutively recycling surface receptor.\nA collagen receptor in activated stellate cells may play a crucial\nrole in the turnover of ECM in liver. The hepatic stellate cells are\nthe major producers of ECM both in normal liver and in the in-\njured liver [48,49]. They synthesize a large variety of ECM\ncomponents [2,5,8] and also participate very actively in their de-\ngradation. To this end different calcium- and zinc-dependent\nenzymes are produced, each being speciﬁc for a group of matrix\ncomponents. MMPs comprise collagenases, gelatinases, strombe-\nlysins and metalloelastases. The stellate cells are the main source\nof MMP synthesis during liver ﬁbrosis [2,9,50]. The main func-\nCollagen endocytosis in stellate cells\ntion of the uPARAP/Endo180 may be to mediate endocytosis of\npartially degraded collagens (and glycoproteins) produced in the\nECM by MMPs. MMP-mediated cleavage of ﬁbrillar collagen\nresults in altered structure accompanied by rapid denaturation\n[51]. These denatured fragments may be treated by uPARAP/\nEndo180 in vivo as heat-denatured collagen is in vitro. The\nuPARAP/Endo180 in stellate cells would thereby co-operate with\nthe secreted MMPs in the remodelling of the matrix, especially\nduring the recovery phase after liver injury.\nWe thank Dr C. M. Isacke and D. Wienke (The Breakthrough Breast Cancer Research\nCentre, Institute of Cancer Research, London, U.K.) for kindly providing the antibody\nagainst Endo180, and Dr L. Malerod for valuable help in the preparation of this paper.\nThis work was supported by the Norwegian Research Council and the Norwegian Cancer\nSociety.\nS. A. Mousavi and others\n42 Matsumoto, K., Sano, H., Nagai, R., Suzuki, H., Kodama, T., Yoshida, M., Ueda, S.,\nSmedsrod, B. and Horiuchi, S. (2000) Endocytic uptake of advanced glycation end\nproducts by mouse liver sinusoidal endothelial cells is mediated by a scavenger\nreceptor distinct from the macrophage scavenger receptor class A. Biochem. J. 352,\n233–240",
         "Materials\nType I calf skin collagen and type IV collagen were from\nSigma and they were heat denatured by incubation at 60 ◦C for\n30 min. High molecular mass urokinase was from Diagnostica &\nAnalys Service AB (G¨oteborg, Sweden). tPA (tissue plasminogen\nactivator) and E-64d were from Calbiochem (Oslo, Norway).\nAnti-rat CD49b (integrin α2 chain) and anti-rat CD29 (integrin β1\nchain) were from PharMingen International (Laborel AS, Oslo,\nNorway). The tetrapeptide Asp-Gly-Glu-Ala and the tripeptide\nArg-Gly-Asp were from Bachem (Bubendorf, Switzerland). 125I\nwas from Amersham Biosciences. Polyclonal rabbit antibody to\nEndo180 was a gift from Dr C. M. Isacke (The Breakthrough\nBreast Cancer Research Centre, Institute of Cancer Research,\nLondon, U.K.). All other chemicals were purchased from Sigma\nunless otherwise mentioned.\nat 5–10 passages. These stellate cells were in ‘activated state’, as\njudged by the presence of desmin and loss of lipid droplets [9,10].\nCells were suspended by trypsinization when conﬂuent and main-\ntained on ﬂasks as described above. Endocytosis experiments were\nperformed with cells grown on 6- or 24-well plates. After the cells\nhad reached approx. 60–80 % conﬂuence, the wells were washed\nonce with serum-free medium and then incubated in 1 ml of\nserum-free medium containing 2 mM L-glutamine, and 1 % BSA\n(to block non-speciﬁc-binding sites) at 37 ◦C for 30 min before\nexperiments were started. Studies of ligand binding were per-\nformed as described above with the exception that the medium\ncontained 20 mM Hepes and the experiments were started after an\nadditional incubation at 4 ◦C for 30 min. Binding of radiolabelled\nligands to wells without cells in the presence of 1 % BSA was\nnegligible (< 0.05 %).\nRadiolabelling of ligands\nDenatured collagen (100 µg in 100 µl of PBS) was incubated with\n10 µl of 125I (40 MBq) in an Iodogen-coated tube (IODO-GEN;\nBio-Rad Laboratories). After 45–60 min, the labelled protein\nwas separated from free Na125I using a Sephadex G-25 column\n(Amersham Biosciences). The speciﬁc activity of [125I]collagen\nwas 8 × 105 c.p.m./µg. OVA (ovalbumin) or denatured colla-\ngen was labelled with [125I]TC ([125I]tyramine cellobiose) as des-\ncribed previously [32]. The speciﬁc activities of [125I]TC-collagen\nand [125I]TC-OVA were 3 × 105 and 5 × 105 c.p.m./µg res-\npectively.\nEndocytosis experiments\nFor studies of endocytic uptake of [125I]collagen, the cells were\nincubated with the labelled ligand at 37 ◦C. In experiments using\ninhibitors, cells were incubated for 15 min with inhibitors before\nthe addition of [125I]collagen. At indicated time-points duplicate\n0.1 ml aliquots were taken from the medium of each well for the\ndetermination of the amount of acid-soluble radioactivity released\nto the medium, and the cells were gently washed with ice-cold\nPBS and scraped in lysis buffer (0.1 % SDS, 0.1 M NaOH). The\nsamples were precipitated with TCA (trichloracetic acid) and BSA\nat ﬁnal concentrations of 10 and 0.5 % respectively, and cen-\ntrifuged at 1000 g for 10 min at 4 ◦C. Acid-soluble and acid-\nprecipitable radioactivities were counted in a gamma counter.\nCell-associated ligand was calculated by subtracting radioactivity\nmeasured at time zero from cell-associated radioactivity measured\nat later time points. To determine the actual amount of degraded\nligand released to the medium from the cells, acid-soluble radio-\nactivity in wells without cells was subtracted from values in cor-\nresponding wells with cells. Total cellular uptake is expressed as\nthe sum of cell-associated [125I]collagen and acid soluble [125I]-\ncollagen released to the medium.\nIsolation and culture of stellate cells\nHepatic parenchymal and non-parenchymal cells were isolated\nfrom male Wistar rats, weighing 250–300 g, by the collagenase\nperfusion method [29,30]. After the separation of parenchymal\ncells by low-speed centrifugation at 50 g for 2 min, the supernatant\ncontaining stellate cells, other non-parenchymal cells and con-\ntaminating parenchymal cells were further centrifuged in a Percoll\ndensity gradient to isolate stellate cells [31]. After primary cul-\nture in plastic ﬂasks using Dulbecco’s modiﬁed Eagle’s medium\n(BioWittaker, MedProbe, Oslo, Norway) containing 10 % (v/v)\nfetal bovine serum and 2 mM L-glutamine and antibiotics\n(BioWhittaker) at 37 ◦C in a humidiﬁed atmosphere of 95 %\nair and 5 % CO2, the stellate cells were used for experiments\nConfocal ﬂuorescence microscopy\nSerum-free Dulbecco’s modiﬁed Eagle’s medium (120 µl), con-\ntaining 50 000 cells, was seeded on poly(D-lysine)-coated cover\nglasses in 24-well plates for 60 min at 37 ◦C. After washing away\nnon-adherent cells, the cells were incubated with the uptake me-\ndium (serum-free medium containing 5 µg/ml heat-denatured\ntype IV collagen labelled with FITC and 1 % BSA) at 37 ◦C.\nUptake was allowed to proceed for 15 min in the presence and\nabsence of LysoTracker Red (Molecular Probes; 0.5 µM), after\nwhich the cells were washed several times with PBS. The cells\nincubated in the presence of LysoTraker were then ﬁxed in 3 %\n(w/v) paraformaldehyde, whereas cells incubated with FITC-\ncollagen alone were ﬁrst chased for 90 min in ligand-free medium\nand thereafter for 30 min in the presence of LysoTracker. After\nﬁxation, cover glasses were washed in PBS and then mounted\non to Mowiol. Images were collected using an IX81 confocal\nmicroscope (OLYMPUS, FLUOVIEW, FV1000). FITC was ex-\ncited using an argon laser at 488 nm, whereas LysoTraker was\nexcited using a helium–neon laser at 543 nm.\nLigand-binding experiments\nBinding experiments were performed with cells incubated at 4 ◦C\nwith [125I]collagen in the presence of increasing concentrations of\nunlabelled denatured collagen. For competition experiments, the\ncells were preincubated in the absence or presence of competitive\nmolecules for 60 min at 4 ◦C, then [125I]collagen was added, and\nafter incubation for 2 h at 4 ◦C, the wells were washed gently with\ncold PBS to remove unbound ligand, and the cells were scraped\noff in lysis buffer before the determination of surface-bound [125I]-\ncollagen. Non-speciﬁc binding was calculated by subtracting\nradioactivity measured in the presence of a 50-fold excess of un-\nlabelled denatured collagen.\nPreparation of cell membrane proteins\nHepatic stellate cell membrane fractions containing the integral\nmembrane proteins were prepared using a membrane extraction\nreagent kit (Mem-PER; Pierce) according to the manufacturer’s\ninstructions.\nLysate preparation, electrophoresis and immunoblot analysis\nFor SDS/PAGE experiments, cells were allowed to detach from\nthe culture ﬂasks by incubating with PBS at 37 ◦C with occasional\ngentle agitation. The cells were pelleted by centrifugation and then\nlysed in lysis buffer (0.3 M NaCl, 20 mM Hepes, 0.2 mM EDTA,\n1.5 mM MgCl2, 0.1 % Triton X-100 and protease inhibitors) at\n4 ◦C. The whole lysates were centrifuged at 10 000 g for 2 min and\nthe clariﬁed lysates were diluted 1:1 in sample buffer [Tris buffer,\npH 6.8, 2 % SDS and 10 % (w/v) sucrose], heated for 5 min and\nthen separated by SDS/PAGE under non-reducing conditions.\nAfter transfer on to PVDF membranes (Millipore), the membranes\nwere blocked in 5 % (w/v) skimmed milk, incubated with the\nprimary antibody (polyclonal rabbit anti-human-Endo180, diluted\n1:5000) for 1 h, and thereafter washed three times in PBS con-\ntaining 0.1 % Tween 20, and incubated with the secondary anti-\nbody (mouse anti-rabbit IgG conjugated to horseradish peroxi-\ndase) for 1 h. Finally, the membranes were washed three times and\nprotein bands were detected using enhanced chemiluminescence\nECL® reagent (Amersham Biosciences).\nAnalysis of gene expression by RT (reverse transcriptase)–PCR\nTotal RNA was isolated using TRIzol® (Invitrogen) from rat hepa-\ntic stellate cells, rat Kupffer cells, rat liver endothelial cells, rat\nparenchymal cells and J774 mouse macrophage cell-line accord-\ning to the manufacturer’s instructions. First-strand cDNA was\nsynthesized from 5 µg of total RNA using SuperScript RT\n(Invitrogen). After cDNA synthesis, 20 µl of mixture was diluted\nto a ﬁnal volume of 50 µl and from this cDNA mixture 1 µl was\nused as template in PCR. Primers were designed using the Pri-\nmer 3 Output program (http://frodo.wi.mit.edu/cgi-bin/primer3/\nprimer3 www.cgi) to amplify Endo180 in rat liver stellate cells\nfrom the human Endo180 mRNA sequence. The primers used\nwere: for Endo180: 5-CCTGGGCATGTATGAGTGTG-3 and\n5-TGTCAGTCAGCTGGTCCTTG-3, and for β-actin: 5-AG-\nCCATGTACGTAGCCATCC-3 and 5-TCTCAGCTGTGGTG-\nGTGAAG-3. Routine PCR was performed using DyNAzyme\nCollagen endocytosis in stellate cells\n(Finnzymes, Espoo, Finland) according to the manufacturer’s\ninstructions and performed at the following conditions: 94 ◦C for\n5 min, 35 cycles each 30 s at 94 ◦C, 60 ◦C for 30 s and 72 ◦C\nfor 30 s. The PCR products were cloned into pGEM-T Easy Vector\n(Promega) and sequenced (GATC-Biotech, Konstanz, Germany).\nPCR marker was provided by New England Biolabs (Beverly,\nMA, U.S.A.).\nRESULTS\nUptake and degradation of [125I]collagen by stellate cells",
         "Tissue remodelling is dependent on the integration of signals\nthat control turnover of ECM (extracellular matrix). Breakdown\nand endocytosis of collagen, a major component of the ECM, is\ncentral to this process. Whereas controlled secretion of matrix-\ndegrading enzymes (such as matrix metalloproteinases) has long\nbeen known to mediate ECM breakdown, it is becoming clear\nthat uPARAP/Endo180 (where uPARAP stands for urokinase\nplasminogen activator receptor-associated protein) serves as a\nreceptor that mediates endocytosis of collagen by several types of\ncells. In the liver, the stellate cells play a major role in turnover\nof ECM including collagens. These cells synthesize various colla-\ngens and also produce matrix metalloproteinases. In the present\nstudy, we investigated the capacity of rat hepatic stellate cells\nto endocytose and degrade 125I-labelled heat-denatured collagen\nI. It was found that the collagen is efﬁciently taken up and\ndegraded by these cells. Degradation was inhibited by inhibitors of\nlysosomal proteases (leupeptin and E-64d) and the vacuolar pro-\nton pump (concanamycin A), indicating that it takes place in lyso-\nsomes. Furthermore, endocytosed FITC-labelled collagen was\nshown to reach late endocytic compartments in which it co-\nlocalized with LysoTracker (a marker of late endocytic compart-\nments). Competition experiments showed that uPA and unlabelled\ncollagen are capable of inhibiting binding and uptake of [125I]col-\nlagen in a dose-dependent manner. Moreover, Western-blot\nanalysis of cell lysate (using a polyclonal rabbit human-Endo180\nantiserum) revealed a single band at 180 kDa. In addition,\nthe antiserum was capable of reducing [125I]collagen binding\nto the cell surface. Finally, using two primers designed from the\nhuman uPARAP/Endo180 mRNA sequence, the expression of\nuPARAP/Endo180 mRNA was detected by reverse transcriptase–\nPCR. These results together suggest\nthat uPARAP/Endo180\nmediates endocytosis of collagen in rat liver stellate cells.\nINTRODUCTION\nHepatic stellate cells are located within the perisinusoidal space\nbetween the basolateral surface of hepatocytes and the endothelial\ncells [1]. They are relatively scarce, < 10 % of the total resident\nliver cells are stellate cells [2]. A number of important functions\nhave been attributed to stellate cells. They store vitamin A,\nreceived from the neighbouring hepatocytes [3,4], and, since they\nencircle the hepatic sinusoids and have contractile properties,\nthey may regulate blood ﬂow through the sinusoids [5–7]. They se-\ncrete cytokines and express a number of receptors that play a\ncentral role in cell–matrix and cell–cell interactions, both in\nnormal and in injured liver [2,8]. The stellate cells exhibit two\nmain phenotypes, the ‘quiescent state’ and the ‘activated state’.\nThe quiescent cells have low proliferation rate and low ﬁbrogenic\nactivity, but after any injury to the liver the cells shift towards\na myoﬁbroblastic-like phenotype [2,5,8]. A similar activation\nprocess is observed when primary stellate cells are kept in culture\non plastic support [9,10]: the cells elongate, lose their vitamin A\ndroplets and develop myoﬁlaments in their cytoplasm [5,8]. The\nactivated cells gain functions such as proliferation and produce\nvarious growth factors and most of the ECM (extracellular\nmatrix) components [9–12]. Activated stellate cells also produce\nenzymes that can degrade the normal pericellular ECM before\nsynthesis of ﬁbrous tissue. Such enzymes include MMPs (matrix\nmetalloproteinases), a family of zinc-dependent enzymes [13,14].\nAlthough activated stellate cells produce several factors needed\nfor ECM degradation, it is conceivable that they in addition may\nremove components of the ECM by means of endocytosis. Very\nfew studies have focused on this process in stellate cells. It was\nshown by using an in vitro system that retinol-binding protein,\nreleased from hepatocytes may be internalized in the stellate cells\nby receptor-mediated endocytosis [4]. A similar mechanism may\noperate when stellate cells take up retinol-binding protein from\nblood [15].\nThe aim of the present study was to determine whether stellate\ncells are able to endocytose collagen, a main component of the\nECM. To this end we used cultured rat stellate cells, and heat-\ndenatured collagen I, labelled with 125I, was chosen as a repres-\nentative ligand. Initial experiments demonstrated that the stellate\ncells internalize and degrade [125I]collagen, and that the degrad-\nation is reduced by inhibitors of lysosomal function, indicating\nthat the cells are able to endocytose [125I]collagen. Moreover, the\nbinding of [125I]collagen was shown to be saturable, indicating\nthat it is receptor-mediated. Further investigations indicated that\nthe receptor in question could be the surface receptor uPARAP/\nEndo180 [where uPARAP stands for urokinase plasminogen\nactivator receptor-associated protein]. uPARAP/Endo180 was\nS. A. Mousavi and others\noriginally identiﬁed as a constitutively recycling surface receptor\n[16] and has recently been shown to function as a collagen recep-\ntor [17–20]. A large pool (70–90 %) of the total uPARAP/Endo180\nis maintained in endosomal compartments and the small plasma\nmembrane pool (10–30 %) is associated with clathrin-coated\npits [16,21]. uPARAP/Endo180 has also been called uPARAP\nbecause it can form a ternary complex with uPAR-bound pro-uPA.\nCollagen can block the formation of this trimolecular complex\n[22]. uPARAP/Endo180 is a member of the mannose receptor\nfamily, which also comprises the mannose receptor, the phospho-\nlipase A2 receptor and the DEC-205/gp200-MR6 receptor [23].\nThese proteins are large in size (175–200 kDa) and contain\nseveral distinct domains including an NH2-terminal cysteine-rich\ndomain, a ﬁbronectin-like type II domain, followed by either eight\nor ten tandemly arranged CTLDs (C-type lectin-like domains,\nnamed CTLD1–CTLD10), a single transmembrane domain and a\nshort cytoplasmic domain containing one or two endocytic motifs\nto direct their internalization through clathrin-coated pits [23,24].\nThe ﬁbronectin-like type II domains of all family members are\nconsidered to have collagen-binding capacity [23], but only the\nmannose receptor (through its CTLD4) and uPARAP/Endo180\n(through its CTLD2) have been shown to bind carbohydrates\n[25,26]. uPARAP/Endo180 has been identiﬁed in many cell\ntypes including monocyte-like U937 cells, vascular smooth\nmuscle cells [22], ﬁbroblasts, vessel endothelial cells, macro-\nphages [16,24,26], osteoblasts [27] and chondrocytes of young\nmice [28]. However, as far as we know, uPARAP/Endo180 has\nnot been reported to be present in stellate cells.\nThe present study presents data indicating that uPARAP/\nEndo180 may be the main receptor responsible for endocytosis of\ndenatured collagen in activated rat hepatic stellate cells.\nEXPERIMENTAL\nMaterials\nType I calf skin collagen and type IV collagen were from\nSigma and they were heat denatured by incubation at 60 ◦C for\n30 min. High molecular mass urokinase was from Diagnostica &\nAnalys Service AB (G¨oteborg, Sweden). tPA (tissue plasminogen\nactivator) and E-64d were from Calbiochem (Oslo, Norway).\nAnti-rat CD49b (integrin α2 chain) and anti-rat CD29 (integrin β1\nchain) were from PharMingen International (Laborel AS, Oslo,\nNorway). The tetrapeptide Asp-Gly-Glu-Ala and the tripeptide\nArg-Gly-Asp were from Bachem (Bubendorf, Switzerland). 125I\nwas from Amersham Biosciences. Polyclonal rabbit antibody to\nEndo180 was a gift from Dr C. M. Isacke (The Breakthrough\nBreast Cancer Research Centre, Institute of Cancer Research,\nLondon, U.K.). All other chemicals were purchased from Sigma\nunless otherwise mentioned.\nat 5–10 passages. These stellate cells were in ‘activated state’, as\njudged by the presence of desmin and loss of lipid droplets [9,10].\nCells were suspended by trypsinization when conﬂuent and main-\ntained on ﬂasks as described above. Endocytosis experiments were\nperformed with cells grown on 6- or 24-well plates. After the cells\nhad reached approx. 60–80 % conﬂuence, the wells were washed\nonce with serum-free medium and then incubated in 1 ml of\nserum-free medium containing 2 mM L-glutamine, and 1 % BSA\n(to block non-speciﬁc-binding sites) at 37 ◦C for 30 min before\nexperiments were started. Studies of ligand binding were per-\nformed as described above with the exception that the medium\ncontained 20 mM Hepes and the experiments were started after an\nadditional incubation at 4 ◦C for 30 min. Binding of radiolabelled\nligands to wells without cells in the presence of 1 % BSA was\nnegligible (< 0.05 %).\nRadiolabelling of ligands\nDenatured collagen (100 µg in 100 µl of PBS) was incubated with\n10 µl of 125I (40 MBq) in an Iodogen-coated tube (IODO-GEN;\nBio-Rad Laboratories). After 45–60 min, the labelled protein\nwas separated from free Na125I using a Sephadex G-25 column\n(Amersham Biosciences). The speciﬁc activity of [125I]collagen\nwas 8 × 105 c.p.m./µg. OVA (ovalbumin) or denatured colla-\ngen was labelled with [125I]TC ([125I]tyramine cellobiose) as des-\ncribed previously [32]. The speciﬁc activities of [125I]TC-collagen\nand [125I]TC-OVA were 3 × 105 and 5 × 105 c.p.m./µg res-\npectively.\nEndocytosis experiments\nFor studies of endocytic uptake of [125I]collagen, the cells were\nincubated with the labelled ligand at 37 ◦C. In experiments using\ninhibitors, cells were incubated for 15 min with inhibitors before\nthe addition of [125I]collagen. At indicated time-points duplicate\n0.1 ml aliquots were taken from the medium of each well for the\ndetermination of the amount of acid-soluble radioactivity released\nto the medium, and the cells were gently washed with ice-cold\nPBS and scraped in lysis buffer (0.1 % SDS, 0.1 M NaOH). The\nsamples were precipitated with TCA (trichloracetic acid) and BSA\nat ﬁnal concentrations of 10 and 0.5 % respectively, and cen-\ntrifuged at 1000 g for 10 min at 4 ◦C. Acid-soluble and acid-\nprecipitable radioactivities were counted in a gamma counter.\nCell-associated ligand was calculated by subtracting radioactivity\nmeasured at time zero from cell-associated radioactivity measured\nat later time points. To determine the actual amount of degraded\nligand released to the medium from the cells, acid-soluble radio-\nactivity in wells without cells was subtracted from values in cor-\nresponding wells with cells. Total cellular uptake is expressed as\nthe sum of cell-associated [125I]collagen and acid soluble [125I]-\ncollagen released to the medium.\nIsolation and culture of stellate cells\nHepatic parenchymal and non-parenchymal cells were isolated\nfrom male Wistar rats, weighing 250–300 g, by the collagenase\nperfusion method [29,30]. After the separation of parenchymal\ncells by low-speed centrifugation at 50 g for 2 min, the supernatant\ncontaining stellate cells, other non-parenchymal cells and con-\ntaminating parenchymal cells were further centrifuged in a Percoll\ndensity gradient to isolate stellate cells [31]. After primary cul-\nture in plastic ﬂasks using Dulbecco’s modiﬁed Eagle’s medium\n(BioWittaker, MedProbe, Oslo, Norway) containing 10 % (v/v)\nfetal bovine serum and 2 mM L-glutamine and antibiotics\n(BioWhittaker) at 37 ◦C in a humidiﬁed atmosphere of 95 %\nair and 5 % CO2, the stellate cells were used for experiments\nConfocal ﬂuorescence microscopy\nSerum-free Dulbecco’s modiﬁed Eagle’s medium (120 µl), con-\ntaining 50 000 cells, was seeded on poly(D-lysine)-coated cover\nglasses in 24-well plates for 60 min at 37 ◦C. After washing away\nnon-adherent cells, the cells were incubated with the uptake me-\ndium (serum-free medium containing 5 µg/ml heat-denatured\ntype IV collagen labelled with FITC and 1 % BSA) at 37 ◦C.\nUptake was allowed to proceed for 15 min in the presence and\nabsence of LysoTracker Red (Molecular Probes; 0.5 µM), after\nwhich the cells were washed several times with PBS. The cells\nincubated in the presence of LysoTraker were then ﬁxed in 3 %\n(w/v) paraformaldehyde, whereas cells incubated with FITC-\ncollagen alone were ﬁrst chased for 90 min in ligand-free medium\nand thereafter for 30 min in the presence of LysoTracker. After\nﬁxation, cover glasses were washed in PBS and then mounted\non to Mowiol. Images were collected using an IX81 confocal\nmicroscope (OLYMPUS, FLUOVIEW, FV1000). FITC was ex-\ncited using an argon laser at 488 nm, whereas LysoTraker was\nexcited using a helium–neon laser at 543 nm.\nLigand-binding experiments\nBinding experiments were performed with cells incubated at 4 ◦C\nwith [125I]collagen in the presence of increasing concentrations of\nunlabelled denatured collagen. For competition experiments, the\ncells were preincubated in the absence or presence of competitive\nmolecules for 60 min at 4 ◦C, then [125I]collagen was added, and\nafter incubation for 2 h at 4 ◦C, the wells were washed gently with\ncold PBS to remove unbound ligand, and the cells were scraped\noff in lysis buffer before the determination of surface-bound [125I]-\ncollagen. Non-speciﬁc binding was calculated by subtracting\nradioactivity measured in the presence of a 50-fold excess of un-\nlabelled denatured collagen.\nPreparation of cell membrane proteins\nHepatic stellate cell membrane fractions containing the integral\nmembrane proteins were prepared using a membrane extraction\nreagent kit (Mem-PER; Pierce) according to the manufacturer’s\ninstructions.\nLysate preparation, electrophoresis and immunoblot analysis\nFor SDS/PAGE experiments, cells were allowed to detach from\nthe culture ﬂasks by incubating with PBS at 37 ◦C with occasional\ngentle agitation. The cells were pelleted by centrifugation and then\nlysed in lysis buffer (0.3 M NaCl, 20 mM Hepes, 0.2 mM EDTA,\n1.5 mM MgCl2, 0.1 % Triton X-100 and protease inhibitors) at\n4 ◦C. The whole lysates were centrifuged at 10 000 g for 2 min and\nthe clariﬁed lysates were diluted 1:1 in sample buffer [Tris buffer,\npH 6.8, 2 % SDS and 10 % (w/v) sucrose], heated for 5 min and\nthen separated by SDS/PAGE under non-reducing conditions.\nAfter transfer on to PVDF membranes (Millipore), the membranes\nwere blocked in 5 % (w/v) skimmed milk, incubated with the\nprimary antibody (polyclonal rabbit anti-human-Endo180, diluted\n1:5000) for 1 h, and thereafter washed three times in PBS con-\ntaining 0.1 % Tween 20, and incubated with the secondary anti-\nbody (mouse anti-rabbit IgG conjugated to horseradish peroxi-\ndase) for 1 h. Finally, the membranes were washed three times and\nprotein bands were detected using enhanced chemiluminescence\nECL® reagent (Amersham Biosciences).\nAnalysis of gene expression by RT (reverse transcriptase)–PCR\nTotal RNA was isolated using TRIzol® (Invitrogen) from rat hepa-\ntic stellate cells, rat Kupffer cells, rat liver endothelial cells, rat\nparenchymal cells and J774 mouse macrophage cell-line accord-\ning to the manufacturer’s instructions. First-strand cDNA was\nsynthesized from 5 µg of total RNA using SuperScript RT\n(Invitrogen). After cDNA synthesis, 20 µl of mixture was diluted\nto a ﬁnal volume of 50 µl and from this cDNA mixture 1 µl was\nused as template in PCR. Primers were designed using the Pri-\nmer 3 Output program (http://frodo.wi.mit.edu/cgi-bin/primer3/\nprimer3 www.cgi) to amplify Endo180 in rat liver stellate cells\nfrom the human Endo180 mRNA sequence. The primers used\nwere: for Endo180: 5-CCTGGGCATGTATGAGTGTG-3 and\n5-TGTCAGTCAGCTGGTCCTTG-3, and for β-actin: 5-AG-\nCCATGTACGTAGCCATCC-3 and 5-TCTCAGCTGTGGTG-\nGTGAAG-3. Routine PCR was performed using DyNAzyme\nCollagen endocytosis in stellate cells\n(Finnzymes, Espoo, Finland) according to the manufacturer’s\ninstructions and performed at the following conditions: 94 ◦C for\n5 min, 35 cycles each 30 s at 94 ◦C, 60 ◦C for 30 s and 72 ◦C\nfor 30 s. The PCR products were cloned into pGEM-T Easy Vector\n(Promega) and sequenced (GATC-Biotech, Konstanz, Germany).\nPCR marker was provided by New England Biolabs (Beverly,\nMA, U.S.A.).\nRESULTS\nUptake and degradation of [125I]collagen by stellate cells\nIn initial experiments, we examined the ability of stellate cells\nto take up and degrade the [125I]collagen. Cells were incubated\nwith [125I]collagen at 37 ◦C for different time periods and uptake\nand degradation of ligand were determined as described in the\nExperimental section. As shown in Figure 1(A), stellate cells take\nup [125I]collagen efﬁciently. After 1 h at 37 ◦C in the presence\nof [125I]collagen, the cells had removed approx. 5 % of the avail-\nable ligand from the medium. To determine whether the ligand\nwas transported to lysosomes for degradation, acid soluble and\nacid precipitable radioactivities were determined in cells and me-\ndium. The results show that degraded ligand accumulated both\ninside the cells and in the medium. Release of acid soluble radio-\nactivity to the medium started after a lag phase lasting approx.\n15 min. Acid precipitable radioactivity in the cells increased for\napprox. 30 min and then levelled off, indicating that an equi-\nlibrium between transport to and degradation in lysosomes had\nbeen reached. Similar results were obtained when denatured 125I-\nlabelled collagen IV was used (results not shown). That de-\ngradation of collagen takes place intracellularly was veriﬁed\nby the ﬁnding that degradation products formed from [125I]TC-\ncollagen were not released to the medium during the incubation\ntime (120 min) (results not shown). Labelled degradation pro-\nducts formed from this probe are trapped in the degradative\ncompartment [33].\nWe also analysed the transport of ﬂuorescently labelled de-\nnatured collagen to late endocytic compartments in pulse–\nchase experiments. Fluorescence microscopy using FITC-labelled\ncollagen (Figure 2) showed lysosomal/late endosomal staining, as\nindicated by co-localization (yellow) due to overlap of the inter-\nnalized ligand (green) and LysoTracker (red), a marker of acidic\nendocytic compartments.\nBinding and internalization of [125I]collagen to hepatic stellate\ncells is saturable and pH-dependent, but not calcium-dependent\nTo determine whether endocytosis of [125I]collagen is receptor-\nmediated, cells were incubated with [125I]collagen in the presence\nof increasing concentrations of unlabelled collagen. The results\nobtained (Figure 1B) show that both binding (at 4 ◦C) and uptake\n(at 37 ◦C) of [125I]collagen were reduced by adding unlabelled\ncollagen. At 6 µg/ml of unlabelled collagen, binding and uptake of\n[125I]collagen were decreased by 70 and 90 % respectively. Taken\ntogether, these experiments indicate that the uptake of collagen\nrepresents a receptor-mediated process and not merely ﬂuid-phase\nendocytosis. Binding of [125I]collagen was pH-dependent, and was\nreduced by approx. 50 % when pH was reduced to pH 5.5 (results\nnot shown). Binding was, however, retained in the presence of the\ncalcium chelator EGTA (2.2 mM).\nDegradation of [125I]collagen is inhibited by protease\nand proton pump inhibitors\nLeupeptin, an inhibitor of cysteine proteases, inhibited the degra-\ndation of [125I]collagen partially (Figure 3, upper panel). After\nS. A. Mousavi and others\n(A) Time course of receptor-mediated uptake and degradation of collagen: cells (∼ 106 cells/\nwell) were incubated with [125I]collagen (0.2 µg/ml) at 37 ◦C for the indicated times, washed and\nthen lysed. Cell lysates and medium were treated with TCA to determine acid-soluble and acid-\nprecipitable radioactivities as described in the Experimental section. Values are expressed as\npercentage of total initial acid precipitable radioactivity in the medium and are means +\n− S.D.\nresults for triplicate samples. Equivalent results were obtained in at least six separate experiments.\n(B) Concentration-dependent binding and uptake of collagen in stellate cells: cells were pre-\nincubated at 4 ◦C for 30 min with the indicated concentrations of unlabelled collagen. After\npreincubation, [125I]collagen (20 ng/ml) was added and the cells were incubated for 120 min\nat 4 ◦C or 30 min at 37 ◦C. The cells were then washed three times and the cell-associated\nradioactivity was determined. Values are the means +\n− S.D. results for triplicate samples.\nEquivalent results were obtained in at least three separate experiments.\n90 min of incubation at 37 ◦C, formation of acid soluble radio-\nactivity was reduced by approx. 50 % in the presence of leupeptin\n(2 mM). Release of acid-soluble degradation products from the\ncells was, however, reduced by nearly 80 % by the inhibitor. These\nresults suggest that the processing of internalized [125I]collagen\nﬁrst produces acid-soluble peptides, which are then further\ndegraded to amino acids (released from the cells). Leupeptin\nprimarily inhibits degradation of the smaller peptides. Preventing\ncollagen degradation caused, as expected, the accumulation of\nboth acid-precipitable and acid-soluble radioactivity in the cells.\nWe also assessed the effect of E-64d, a selective and irreversible\ninhibitor of lysosomal cysteine proteases, on the degradation\nof internalized [125I]collagen. As indicated in Figure 3 (upper\npanel), treatment of cells with E-64d gave results similar to\nthose seen with leupeptin treatment, although at a much lower\nconcentration (50 µM) of the inhibitor. Lysosomal degradation\nof endocytosed ligands depends on a functional vacuolar H+-\nATPase (V-ATPase) in the endosomal/lysosomal compartments\n[34]. To determine whether the inactivation of V-ATPase affects\ncollagen degradation, the cells were incubated in the presence\nof concanamycin A, a selective inhibitor of V-ATPase [35]. The\nresults obtained (Figure 3, lower panel) show that concanamycin\nA inhibited both uptake and degradation of [125I]collagen. In cells\nincubated for 90 min at 37 ◦C, the degradation of labelled colla-\ngen was reduced approx. 85 % in the presence of 0.1 µM con-\ncanamycin A. Uptake was reduced approx. 60 % at this drug\nconcentration.\nEffects of possible competitive ligands and anti-integrin antibodies\non [125I]collagen binding\nThe rapid removal of collagen from the culture medium sug-\ngested the existence of a speciﬁc receptor(s) in hepatic stellate\ncells. Several integrins have been reported to act as binding recep-\ntors for denatured collagen, including α2β1 [36,37] and αvβ3\n[38,39]. We therefore used anti-integrin antibodies and ligands of\nthese receptors in competition experiments to see whether they\nwere capable of competing with [125I]collagen for binding. At high\nconcentrations (> 20 µg/ml), the anti-integrin α2 chain inhibited\n[125I]collagen binding by approx. 10 %, whereas anti-integrin β1\nchain did not signiﬁcantly inhibit the binding of [125I]collagen. No\nadditional inhibition was observed using both antibodies together.\nPeptides containing the RGD (Arg-Gly-Asp) sequence have been\nshown to inhibit speciﬁcally the binding of denatured type I\ncollagen to the αvβ3 integrin, possibly because this sequence\nbecomes exposed on collagen denaturation [38], whereas the\nsequence DGEA (Asp-Gly-Glu-Ala) binds to α2β1 integrin [40].\n[125I]Collagen binding was therefore examined in the presence\nof the tripeptide Arg-Gly-Asp and the tetrapeptide Asp-Gly-\nGlu-Ala. Neither of these peptides (> 0.5 mg/ml) signiﬁcantly\ninhibited the binding of [125I]collagen (results not shown).\nScavenger receptors have also been shown to serve as receptors\nfor denatured collagen [41]. No inhibitory effects on the binding\nand uptake of [125I]collagen were, however, observed when cells\nwere incubated with a 50 M excess of formaldehyde-treated BSA,\na ligand of scavenger receptors [42] (results not shown).\nuPA inhibits binding and uptake of [125I]collagen\nCollagen has been shown to block the association of Endo180 with\nuPAR-bound pro-uPA [22]. Conceivably, uPA might also bind to\nthe collagen-binding receptor in the stellate cells if this receptor\nis related to the uPARAP/Endo180 receptor. It was found that uPA\ninhibits both binding and uptake of [125I]collagen to the stellate\ncells in a concentration-dependent manner (Figure 4). At 3 mg/ml,\nuPA reduced both binding and uptake of labelled collagen by\napprox. 80 %. It may, however, be argued that the reduced binding\nof collagen in the presence of uPA is due to the activation of\ncollagen-degrading enzymes (such as MMP-13) secreted by the\ncells into the medium during the 3.5 h incubation (30 min at 37 ◦C\nand 3 h at 4 ◦C). To rule out this possibility, similar experiments\nwere performed in the presence of tPA, which binds receptors\nother than uPAR [43]. However, incubation with 2.2 mg/ml tPA\ndid not inhibit signiﬁcantly the binding of collagen to the cells\n(binding as percentage of total: 1.04 +− 0.19 and 0.96 +− 0.09;\nn = 3, for control and tPA respectively), indicating that the reduced\nbinding of collagen is due to binding of uPA to its receptor (or\nCollagen endocytosis in stellate cells\nCells were pulse-labelled at 37 ◦C for 15 min with FITC-collagen (green) in the presence or absence of LysoTracker (red), then washed and either ﬁxed (images on the left) or chased for 90 min, after\nwhich the incubation continued for an additional 30 min in the presence of LysoTracker (images on the right). Co-localization of FITC-collagen with LysoTracker (yellow) is shown in the merged\nimages (lowest panels). Results represent two or three experiments.\nto uPARAP/Endo180, see below) and not to an indirect effect of\nuPA on the activation status of collagenases in the medium.\nuPARAP/Endo180 is expressed in rat hepatic stellate cells\nResult from experiments with uPA suggested that uPARAP/\nEndo180 is probably involved in the uptake of collagen. Therefore\nwe performed Western-blot analysis, using a polyclonal antibody\nagainst puriﬁed human uPARAP/Endo180 [16]. U937 cells, the\ncell line originally used to deﬁne uPARAP activity [22], were\nused as positive controls, and HepG2 cells were used as negative\ncontrols since they do not express uPARAP/Endo180 [24]. As\nshown in Figure 5, no band was detected in the preparation from\nHepG2 cells, whereas a speciﬁc band at 180 kDa was revealed\nin preparations from both hepatic stellate cells and U937 cells.\nWe also tested the ability of the Endo180 antiserum to interfere\nwith collagen binding to the cells and found that the binding\nof [125I]collagen to the stellate cells was signiﬁcantly reduced\nby 50 % when 20 µl/ml of the antiserum was used (results not\nshown). Analysis by RT–PCR showed that uPARAP/Endo180 is\nabundantly expressed in stellate cells, but is not detectable in\nKupffer cells, liver endothelial cells, hepatocytes and J774 cells\nUptake of OVA\nCTLD2 in Endo180 has been shown to contain a Ca2+-dependent\ncarbohydrate-binding determinant, which speciﬁcally binds to\nN-acetylglucosamine, mannose and fucose [26]. To determine\nwhether hepatic stellate cells express such binding activity, we\nmeasured the uptake of OVA (which contains sugars terminating\nin N-acetylglucosamine and mannose [44]). It was found that the\nstellate cells are also capable of taking up OVA. As shown in\nFigure 7, uptake and degradation of [125I]TC-OVA increases as\na function of time. Moreover, the uptake (measured after 30 min\nincubation at 37 ◦C) was reduced by 90 +− 4 and 87 +− 5 % by the\nS. A. Mousavi and others\nThe whole lysates (40 µg) of hepatic stellate cells (lane 1), U937 cells (lane 3) and HepG2 cells\n(lane 4) were analysed by SDS/PAGE on a 6 % gel and then transferred on to membrane. The\nblot was incubated ﬁrst with a polyclonal antiserum directed against Endo180 and subsequently\nwith mouse anti-rabbit IgG conjugated to horseradish peroxidase followed by detection with\nECL®. Lane 2 corresponds to the cell membrane fraction (20 µg) prepared from stellate cells.\nThe membrane was then stripped, blocked and incubated with anti-β-actin. Relative molecular\nmass standards are shown on the left (masses in kDa). The results shown were reproduced in\nat least three independent experiments.\nCells were preincubated with either leupeptin (2 mM), E-64d (2 µM) (upper panel) or increasing\nconcentrations of concanamycin (lower panel) for 15 min at 37 ◦C. After preincubation, radio-\nlabelled collagen (40 ng/ml) was added, followed by 90 min incubation at 37 ◦C and three\nwashes. Cell\nlysates and medium were treated with TCA to determine acid-soluble and\nacid-precipitable radioactivities as described in the Experimental section. Values are expressed\nas percentage of controls and are the means +\n− S.D. results for triplicate samples. Equivalent\nresults were obtained in at least three separate experiments.\nFour different\nliver cells [liver endothelial cells (rLEC), Kupffer cells (rKC),\nparenchymal cells (rPC) and stellate cells (rStel)] and J774 mouse macrophages were tested\nfor mRNA expression of Endo180 (lanes 2–6 respectively) with β-actin mRNA as an internal\ncontrol (lanes 7–11). Three experiments gave equivalent results. Sizes are indicated in bp.\nCells were preincubated with increasing concentrations of uPA for 30 min at 4 ◦C. After\npreincubation, radiolabelled collagen (20 ng/ml) was added, followed by 120 min incubation\nat 4 ◦C and three washes. For uptake, uPA was added just before the addition of [125I]collagen\nand cells were incubated at 37 ◦C for 30 min. Results are means +\n− S.D. of a representative\nexperiment performed in triplicate.\nCells (∼ 106 cells/well) were incubated at 37 ◦C with 2 nM [125I]TC-OVA in the presence and\nabsence of a 50-fold excess of unlabelled OVA or 2.2 mM EGTA. At timed intervals the cells were\nwashed and cell-associated radioactivity (acid-soluble and acid-precipitable) was measured as\ndescribed in the Experimental section. Inset: the effect of unlabelled OVA and EGTA on the\nuptake.\naddition of 50-fold excess of unlabelled OVA and 2.2 mM EGTA\nrespectively (Figure 7, inset).\nDISCUSSION\nThe present results demonstrate that the uptake of heat-denatured\n125I-labelled collagen in rat hepatic stellate cells is mediated by\na saturable binding site. To identify the receptor(s) involved, we\nmeasured the uptake of [125I]collagen in the presence and absence\nof antibodies and ligands that bind to potential receptor candi-\ndates. The results obtained seemed to exclude integrins (α2β1 and\nαvβ3) and scavenger receptors. The discoidin domain receptors\n(DDR1 and DDR2) are, probably, not involved in binding, as these\nreceptors require the native, triple-helical conﬁguration of the\ncollagen [45,46]. On the other hand, several observations pointed\nto the uPARAP/Endo180 as the receptor involved in binding colla-\ngen. Behrendt et al. [22] found that collagen prevents the form-\nation of a complex between pro-uPA-uPAR and uPARAP/\nEndo180. We found that the reciprocal interaction, the inter-\naction of uPA-uPAR with uPARAP/Endo180, prevents uPARAP/\nEndo180 from binding to collagen. Alternatively or in addition,\nuPA can directly bind to Endo180. This result suggested that\n[125I]collagen binding in stellate cells could be mediated by\nthe uPARAP/Endo180. That the receptor in question really is\nuPARAP/Endo180 was veriﬁed using Western-blot analysis and\nRT–PCR. Moreover, collagen binding was signiﬁcantly reduced\nby Endo180 antiserum, strongly supporting the involvement of\nuPARAP/Endo180 in mediating the uptake of collagen. Taken\ntogether, these observations are compatible with the newly deﬁned\nrole of uPARAP/Endo180 as an endocytic collagen receptor\n[17–20]. The stellate cells also bind [125I]TC-OVA, compatible\nwith the observation that uPARAP/Endo180, which belongs to\nthe mannose-receptor family, can also function as a lectin, binding\nligands with terminal mannose, fucose and N-acetylglucosamine\n[26]. Moreover, the binding of the OVA is calcium-dependent,\nwhereas the binding of collagen is not. This is in agreement with\nearlier studies of the uPARAP/Endo180 receptor [18].\nInhibition of [125I]collagen degradation by leupeptin and E-64d,\ntwo inhibitors of lysosomal thiol proteinases and concanamycin\nA, an inhibitor of the vacuolar proton pump [35,47], shows that the\nligand is effectively transported to late endocytic compartments\nfor degradation. These observations are in agreement with recent\nreports demonstrating that uPARAP/Endo180 mediates endo-\ncytosis and lysosomal degradation of collagens in ﬁbroblasts [20]\nand in a breast cancer cell line (MCF-7) [18]. To the best of our\nknowledge, the present study is the ﬁrst to demonstrate receptor-\nmediated endocytosis of collagen in hepatic stellate cells.\nThe afﬁnity of the uPARAP/Endo180 for denatured collagen\nis reduced at pH  6.5 and the ﬁnding that the proton pump in-\nhibitor also reduces cellular uptake of [125I]collagen is probably\na result of increases in pH in early or late endosomes. This will\nprevent dissociation of [125I]collagen from the collagen receptor\nin endosomes, and the ligand will recycle to the plasma membrane\nbound to the receptor. uPARAP/Endo180 has been regarded as a\nconstitutively recycling surface receptor.\nA collagen receptor in activated stellate cells may play a crucial\nrole in the turnover of ECM in liver. The hepatic stellate cells are\nthe major producers of ECM both in normal liver and in the in-\njured liver [48,49]. They synthesize a large variety of ECM\ncomponents [2,5,8] and also participate very actively in their de-\ngradation. To this end different calcium- and zinc-dependent\nenzymes are produced, each being speciﬁc for a group of matrix\ncomponents. MMPs comprise collagenases, gelatinases, strombe-\nlysins and metalloelastases. The stellate cells are the main source\nof MMP synthesis during liver ﬁbrosis [2,9,50]. The main func-\nCollagen endocytosis in stellate cells\ntion of the uPARAP/Endo180 may be to mediate endocytosis of\npartially degraded collagens (and glycoproteins) produced in the\nECM by MMPs. MMP-mediated cleavage of ﬁbrillar collagen\nresults in altered structure accompanied by rapid denaturation\n[51]. These denatured fragments may be treated by uPARAP/\nEndo180 in vivo as heat-denatured collagen is in vitro. The\nuPARAP/Endo180 in stellate cells would thereby co-operate with\nthe secreted MMPs in the remodelling of the matrix, especially\nduring the recovery phase after liver injury.\nWe thank Dr C. M. Isacke and D. Wienke (The Breakthrough Breast Cancer Research\nCentre, Institute of Cancer Research, London, U.K.) for kindly providing the antibody\nagainst Endo180, and Dr L. Malerod for valuable help in the preparation of this paper.\nThis work was supported by the Norwegian Research Council and the Norwegian Cancer\nSociety.\nS. A. Mousavi and others\n42 Matsumoto, K., Sano, H., Nagai, R., Suzuki, H., Kodama, T., Yoshida, M., Ueda, S.,\nSmedsrod, B. and Horiuchi, S. (2000) Endocytic uptake of advanced glycation end\nproducts by mouse liver sinusoidal endothelial cells is mediated by a scavenger\nreceptor distinct from the macrophage scavenger receptor class A. Biochem. J. 352,\n233–240",
         "S. Mousavi,Mitsuru Sato,Marita Sporstøl,B. Smedsrød,T. Berg,N. Kojima,H. Senoo",
         null
        ],
        [
         "19",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "Several expanded-spectrum -lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enter-\nobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional\nmechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.",
         "An increasing number of reports of expanded-spectrum\n-lactamases of the CTX-M type and plasmid-mediated cepha-\nlosporinases of the AmpC type in Enterobacteriaceae raise the\nquestion of their acquisition (5, 24, 26). The blaCTX-M genes\nbelonging to three out of the ﬁve known clusters (the CTX-\nM-1, -M-2, and -M-9 clusters) are associated with ISEcp1-like\ninsertion sequences (IS) (2, 5–7, 11, 12, 18, 21, 23). Neverthe-\nless, the distance that separates the -lactamase gene from\nISEcp1 varies within a given cluster of CTX-M-type genes,\nindicating that different insertion events have occurred (13,\n19). Analysis of GenBank databases reveals that some plasmid-\nencoded cephalosporinase genes, such as the blaCMY-type\ngenes, may also be associated with ISEcp1 (3, 10, 17).\nISEcp1 is an IS weakly related to other IS elements and\nbelonging to the IS1380 family (IS Database HomePage\n[http://www-is.biotoul.fr/page-is.html]) (8). Nothing is known\nabout the mobilization properties of ISEcp1. Recently, we\ncharacterized ISEcp1B associated with a blaCTX-M-19 gene in a\nKlebsiella pneumoniae ILT-3 clinical\nisolate from Vietnam\n(21). ISEcp1B differs from ISEcp1 (GenBank accession no.\nAJ242809) by three nucleotide substitutions. Their inverted\nrepeat (IR) sequences are identical, and their transposases\ndiffer by a single amino acid change (18). Since ISEcp1-like\nelements are located upstream of several -lactamase genes,\nanalysis of the variable sequences separating these IS elements\nfrom the initiation codons of these genes allowed us to deter-\nmine its boundaries.\nRecently, it has been shown that ISEcp1B brought promoter\nsequences for high-level expression of the blaCTX-M-14/-18,\n-lactamase genes that belong to\nblaCTX-M-17, and blaCTX-M-19\nthe same blaCTX-M-9 subgroup (5, 6, 18) and might be also\nresponsible for their mobilization. The aim of this study was to\nanalyze the mobilization process mediated by ISEcp1B.\nISEcp1B possesses two imperfect IRs likely made of 14 bp,\npreviously estimated to be 18 bp (18), with 12 of these 14 bp\nbeing complementary (Table 1). A detailed analysis of a ca.\n9.5-kb DNA fragment from natural plasmid pILT-3 from K.\npneumoniae ILT-3 revealed that the gene encoding ORF1 in\ntransposon Tn1721 was interrupted by a 4.8-kb potential trans-\nposon bracketed on the left side by the left IR (IRL) of\nISEcp1B and on the right side by an imperfect right IR (IRR)\nnamed IRR1 made of a 14-bp sequence belonging to a gene\nencoding a putative iron transport protein (Fig. 1) (18). This\npotential transposon comprises ISEcp1B, blaCTX-M-19, IS903D,\nand part of the gene encoding an iron transport protein. We\nhave hypothesized that ISEcp1B could mobilize adjacent se-\nquences by transposition.\nSeveral transposition experiments were performed to deter-\nmine (i) the mobility of this potential transposon, (ii) the role\nof ISEcp1B in mobilizing the adjacent -lactamase gene, and\n(iii) the role of IS903D. Different recombinant plasmids were\nconstructed by inserting PCR products that encompassed ei-\nther one or both ISs that bracketed the blaCTX-M-19 gene in\nplasmid pILT-3 into the EcoRV site of plasmid pACYC184\n(Table 2) and then transformed into Escherichia coli reference\nstrain DH10B. Recombinant plasmids were constructed with\nthe pACYC184 backbone to prevent overexpression of the\nISEcp1B transposase. Plasmid pC-1 contained the sequences\nof the potential transposon identiﬁed in plasmid pILT-3,\nwhereas the following constructs lacked the right part of it\n(Fig. 1). Plasmid pC-2 possessed the ISEcp1B sequence but\nlacked part of IS903D, and plasmid pC-3 possessed the entire\nIS903D sequence and a truncated ISEcp1B transposase gene\n(Fig. 1; Table 2). E. coli RR1023 (pOX38) was transformed by\nelectroporation with these recombinant plasmids (Table 2) (9).\nPlasmid pOX38 is a self-conjugative and IS-free plasmid.\nTransposition events were searched between inserts of several\nrecombinant plasmids and recipient plasmid pOX38 after 24 h\nof growth in Trypticase soy broth. Transfer of the recombinant\nplasmids with the pOX38 backbone into streptomycin-resistant\nE. coli reference strain DH10B (Table 2) was then performed by\nconjugation as described previously (20), and transconjugants\nwere selected on agar plates containing 100 g of amoxicillin per\nml (plasmid resistance marker) and 20 g of streptomycin per ml\n(chromosomal marker). Several amoxicillin-resistant transconju-\ngants were extracted and sequenced in part.\nSusceptibility to chloramphenicol was always observed in\ntransconjugant strains, thus ruling out full integration (chlor-\namphenicol resistance) of the recombinant plasmids derived\nfrom pACYC184 into plasmid pOX38. Detection of ISEcp1B\nand IS903D was performed using two sets of primers (primer\nISEcp1A [Table 2] in combination with primer ISEcp1B [5-T\nTTCCGCAGCACCGTTTGC-3] and primer IS903A [5-CA\nIRL of ISEcp1B\nDeduced perfect IRR of ISEcp1B\nIRR of ISEcp1B\nIRR1 at the potential transposon end of pILT-3\nIRR2 in plasmid pACYC184\nIRR3 in plasmid pBK-CMV\nIRR4 in the tnpA gene of IS903D\nIRR5 in the iron gene of pILT-3\nIRR6 in the iron gene of pILT-3\nIRR7 in the iron gene of pILT-3\nIRR8 in the iron gene of pILT-3\nCATGAAATCATCTGCGC-3] in combination with primer\nIS903B [Table 2], respectively).\ntransconjugants were not obtained\nwhen E. coli RR1023 (pC-3) lacking ISEcp1B (Fig. 1) was used\nas a donor, suggesting that the IS903D element located down-\nstream of the -lactamase gene did not mobilize this latter\ngene. On the other hand, amoxicillin-resistant transconjugants\nwere obtained when E. coli RR1023 (pC-1) and E. coli RR1023\n(pC-2) were used as donors with a transposition efﬁciency of\nca. 107 per donor. These results indicated that ISEcp1B was\nessential for movement of the blaCTX-M-19 gene.\nUsing E. coli RR1023 (pC-1) as a donor, all amoxicillin-\nresistant transconjugants except one contained ISEcp1B and\nIS903D elements, with the blaCTX-M-19 gene and DNA se-\nquences being located on the right part of IS903D in plasmid\npILT-3 (Fig. 1). Surprisingly, none of the 11 transconjugants\nthat were sequenced had the entire potential transposon iden-\ntiﬁed in natural plasmid pILT-3. Nevertheless, the transposi-\ntion mechanism mediated by ISEcp1B was evident, since 5-bp\nduplications were observed at the insertion sites in plasmid\npOX38 (see below). These data suggested that the IRR se-\nquences recognized by ISEcp1B were located inside the poten-\ntial transposon. Analysis of the boundaries of the transposed\nfragments identiﬁed sequences, similar to that of the IRR of\nISEcp1B and deﬁned as IRR-like sequences, that had been\nrecognized as such by the transposase of ISEcp1B during a\nmobilization process.\nFive different IRRs (IRR4 to IRR8) (Table 1) were identi-\nﬁed in ﬁve amoxicillin-resistant transconjugants that were an-\nalyzed, four of them being located in the gene encoding the\niron transport protein and one of them being located in the\ntnpA gene of IS903D (Table 2; Fig. 1). The number of identical\nbase pairs among the sequence deﬁned as IRR varied from\nthree to six, corresponding to less than 50%.\nAnalysis of E. coli RR1023 (pOX38) transconjugants resis-\ntant to amoxicillin, obtained with plasmid pC-2 as a donor, and\nthus lacking the sequence used as IRR in pILT-3, also showed\nthat transposition had occurred using nearby sequences of\npACYC184. The IRR-like sequences bracketing the trans-\nposed fragment in these transconjugants remained unchanged.",
         "",
         "",
         "",
         "Several expanded-spectrum -lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enter-\nobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional\nmechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.\nAn increasing number of reports of expanded-spectrum\n-lactamases of the CTX-M type and plasmid-mediated cepha-\nlosporinases of the AmpC type in Enterobacteriaceae raise the\nquestion of their acquisition (5, 24, 26). The blaCTX-M genes\nbelonging to three out of the ﬁve known clusters (the CTX-\nM-1, -M-2, and -M-9 clusters) are associated with ISEcp1-like\ninsertion sequences (IS) (2, 5–7, 11, 12, 18, 21, 23). Neverthe-\nless, the distance that separates the -lactamase gene from\nISEcp1 varies within a given cluster of CTX-M-type genes,\nindicating that different insertion events have occurred (13,\n19). Analysis of GenBank databases reveals that some plasmid-\nencoded cephalosporinase genes, such as the blaCMY-type\ngenes, may also be associated with ISEcp1 (3, 10, 17).\nISEcp1 is an IS weakly related to other IS elements and\nbelonging to the IS1380 family (IS Database HomePage\n[http://www-is.biotoul.fr/page-is.html]) (8). Nothing is known\nabout the mobilization properties of ISEcp1. Recently, we\ncharacterized ISEcp1B associated with a blaCTX-M-19 gene in a\nKlebsiella pneumoniae ILT-3 clinical\nisolate from Vietnam\n(21). ISEcp1B differs from ISEcp1 (GenBank accession no.\nAJ242809) by three nucleotide substitutions. Their inverted\nrepeat (IR) sequences are identical, and their transposases\ndiffer by a single amino acid change (18). Since ISEcp1-like\nelements are located upstream of several -lactamase genes,\nanalysis of the variable sequences separating these IS elements\nfrom the initiation codons of these genes allowed us to deter-\nmine its boundaries.\nRecently, it has been shown that ISEcp1B brought promoter\nsequences for high-level expression of the blaCTX-M-14/-18,\n-lactamase genes that belong to\nblaCTX-M-17, and blaCTX-M-19\nthe same blaCTX-M-9 subgroup (5, 6, 18) and might be also\nresponsible for their mobilization. The aim of this study was to\nanalyze the mobilization process mediated by ISEcp1B.\nISEcp1B possesses two imperfect IRs likely made of 14 bp,\npreviously estimated to be 18 bp (18), with 12 of these 14 bp\nbeing complementary (Table 1). A detailed analysis of a ca.\n9.5-kb DNA fragment from natural plasmid pILT-3 from K.\npneumoniae ILT-3 revealed that the gene encoding ORF1 in\ntransposon Tn1721 was interrupted by a 4.8-kb potential trans-\nposon bracketed on the left side by the left IR (IRL) of\nISEcp1B and on the right side by an imperfect right IR (IRR)\nnamed IRR1 made of a 14-bp sequence belonging to a gene\nencoding a putative iron transport protein (Fig. 1) (18). This\npotential transposon comprises ISEcp1B, blaCTX-M-19, IS903D,\nand part of the gene encoding an iron transport protein. We\nhave hypothesized that ISEcp1B could mobilize adjacent se-\nquences by transposition.\nSeveral transposition experiments were performed to deter-\nmine (i) the mobility of this potential transposon, (ii) the role\nof ISEcp1B in mobilizing the adjacent -lactamase gene, and\n(iii) the role of IS903D. Different recombinant plasmids were\nconstructed by inserting PCR products that encompassed ei-\nther one or both ISs that bracketed the blaCTX-M-19 gene in\nplasmid pILT-3 into the EcoRV site of plasmid pACYC184\n(Table 2) and then transformed into Escherichia coli reference\nstrain DH10B. Recombinant plasmids were constructed with\nthe pACYC184 backbone to prevent overexpression of the\nISEcp1B transposase. Plasmid pC-1 contained the sequences\nof the potential transposon identiﬁed in plasmid pILT-3,\nwhereas the following constructs lacked the right part of it\n(Fig. 1). Plasmid pC-2 possessed the ISEcp1B sequence but\nlacked part of IS903D, and plasmid pC-3 possessed the entire\nIS903D sequence and a truncated ISEcp1B transposase gene\n(Fig. 1; Table 2). E. coli RR1023 (pOX38) was transformed by\nelectroporation with these recombinant plasmids (Table 2) (9).\nPlasmid pOX38 is a self-conjugative and IS-free plasmid.\nTransposition events were searched between inserts of several\nrecombinant plasmids and recipient plasmid pOX38 after 24 h\nof growth in Trypticase soy broth. Transfer of the recombinant\nplasmids with the pOX38 backbone into streptomycin-resistant\nE. coli reference strain DH10B (Table 2) was then performed by\nconjugation as described previously (20), and transconjugants\nwere selected on agar plates containing 100 g of amoxicillin per\nml (plasmid resistance marker) and 20 g of streptomycin per ml\n(chromosomal marker). Several amoxicillin-resistant transconju-\ngants were extracted and sequenced in part.\nSusceptibility to chloramphenicol was always observed in\ntransconjugant strains, thus ruling out full integration (chlor-\namphenicol resistance) of the recombinant plasmids derived\nfrom pACYC184 into plasmid pOX38. Detection of ISEcp1B\nand IS903D was performed using two sets of primers (primer\nISEcp1A [Table 2] in combination with primer ISEcp1B [5-T\nTTCCGCAGCACCGTTTGC-3] and primer IS903A [5-CA\nIRL of ISEcp1B\nDeduced perfect IRR of ISEcp1B\nIRR of ISEcp1B\nIRR1 at the potential transposon end of pILT-3\nIRR2 in plasmid pACYC184\nIRR3 in plasmid pBK-CMV\nIRR4 in the tnpA gene of IS903D\nIRR5 in the iron gene of pILT-3\nIRR6 in the iron gene of pILT-3\nIRR7 in the iron gene of pILT-3\nIRR8 in the iron gene of pILT-3\nCATGAAATCATCTGCGC-3] in combination with primer\nIS903B [Table 2], respectively).\ntransconjugants were not obtained\nwhen E. coli RR1023 (pC-3) lacking ISEcp1B (Fig. 1) was used\nas a donor, suggesting that the IS903D element located down-\nstream of the -lactamase gene did not mobilize this latter\ngene. On the other hand, amoxicillin-resistant transconjugants\nwere obtained when E. coli RR1023 (pC-1) and E. coli RR1023\n(pC-2) were used as donors with a transposition efﬁciency of\nca. 107 per donor. These results indicated that ISEcp1B was\nessential for movement of the blaCTX-M-19 gene.\nUsing E. coli RR1023 (pC-1) as a donor, all amoxicillin-\nresistant transconjugants except one contained ISEcp1B and\nIS903D elements, with the blaCTX-M-19 gene and DNA se-\nquences being located on the right part of IS903D in plasmid\npILT-3 (Fig. 1). Surprisingly, none of the 11 transconjugants\nthat were sequenced had the entire potential transposon iden-\ntiﬁed in natural plasmid pILT-3. Nevertheless, the transposi-\ntion mechanism mediated by ISEcp1B was evident, since 5-bp\nduplications were observed at the insertion sites in plasmid\npOX38 (see below). These data suggested that the IRR se-\nquences recognized by ISEcp1B were located inside the poten-\ntial transposon. Analysis of the boundaries of the transposed\nfragments identiﬁed sequences, similar to that of the IRR of\nISEcp1B and deﬁned as IRR-like sequences, that had been\nrecognized as such by the transposase of ISEcp1B during a\nmobilization process.\nFive different IRRs (IRR4 to IRR8) (Table 1) were identi-\nﬁed in ﬁve amoxicillin-resistant transconjugants that were an-\nalyzed, four of them being located in the gene encoding the\niron transport protein and one of them being located in the\ntnpA gene of IS903D (Table 2; Fig. 1). The number of identical\nbase pairs among the sequence deﬁned as IRR varied from\nthree to six, corresponding to less than 50%.\nAnalysis of E. coli RR1023 (pOX38) transconjugants resis-\ntant to amoxicillin, obtained with plasmid pC-2 as a donor, and\nthus lacking the sequence used as IRR in pILT-3, also showed\nthat transposition had occurred using nearby sequences of\npACYC184. The IRR-like sequences bracketing the trans-\nposed fragment in these transconjugants remained unchanged.",
         "L. Poirel,M. Lartigue,J. Decousser,P. Nordmann",
         null
        ],
        [
         "20",
         "0018ece872bb78b06eb26ea35d5ed27f53702ce6",
         null,
         null,
         null,
         null,
         null,
         null,
         "Shailesh V. Date,E. Marcotte",
         "captcha"
        ],
        [
         "21",
         "001a3ef673ee3e7703c7db597386a2162a2dc402",
         null,
         null,
         null,
         null,
         null,
         null,
         "R. Fuller,C. Baer,J. Travis",
         "captcha"
        ],
        [
         "22",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "",
         "1. Introduction",
         "",
         "",
         "",
         "1. Introduction",
         "I. Kuzmin,G. Hughes,A. Botvinkin,L. Orciari,C. Rupprecht",
         null
        ],
        [
         "23",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         null,
         null,
         null,
         null,
         null,
         null,
         "Y. Sugase-Miyamoto,B. Richmond",
         "broken link"
        ],
        [
         "24",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "Apart from alterations in the RETPTC-RAS-BRAF pathway, com-\nparatively little is known about the genetics of papillary thyroid\ncarcinoma (PTC). We show that numerous microRNAs (miRNAs) are\ntranscriptionally up-regulated in PTC tumors compared with unaf-\nfected thyroid tissue. A set of ﬁve miRNAs, including the three most\nup-regulated ones (miR-221, -222, and -146), distinguished un-\nequivocally between PTC and normal thyroid. Additionally, miR-\n221 was up-regulated in unaffected thyroid tissue in several PTC\npatients, presumably an early event in carcinogenesis. Tumors in\nwhich the up-regulation (11- to 19-fold) of miR-221, -222, and -146\nwas strongest showed dramatic loss of KIT transcript and Kit\nprotein. In 5 of 10 such cases, this down expression was associated\nwith germline single-nucleotide changes in the two recognition\nsequences in KIT for these miRNAs. We conclude that up-regulation\nof several miRs and regulation of KIT are involved in PTC patho-\ngenesis, and that sequence changes in genes targeted by miRNAs\ncan contribute to their regulation.",
         "malignancy in thyroid tissue, accounting for 80% of all\nthyroid cancers. The incidence of PTC in the United States has\nincreased in recent years (1). Genetically, PTC is characterized\nby alterations in the RETPTC-RAS-BRAF signaling pathway\n(2, 3). Activating mutations in BRAF and RETPTC gene\nrearrangements are frequent genetic changes in PTC tumors\n(4–6). A strong inherited genetic predisposition is suggested by\ncase-control studies showing a 3- to 8-fold risk in first-degree\nrelatives, one of the highest of all cancers (7, 8). Despite\nunequivocal evidence of an inherited predisposition, large fam-\nilies displaying Mendelian inheritance of PTC are rare, and no\npredisposing gene mutations have been found, even though\nseveral putative loci have been identified by linkage analysis\n(9–11).\nMicroRNAs (miRNAs) represent a previously uncharacter-\nized class of gene products that are believed to function as\nnegative regulators of gene expression (12–16). Recently,\nmiRNA genes have been implicated in several cancers (14,\n17–25). The expression of miRNAs varies between cancer and\nnormal cells and varies among different types of cancer. In most\ncancers studied so far, the expression of miRNAs seems to be\nlower than in the corresponding normal tissue (17, 18, 21, 22, 24).\nWe reasoned that the previous failure to identify genes\npredisposing or contributing to PTC might be because these\ngenes show low penetrance. The mechanisms may require the\ninteraction of two or more genes; thus, regulatory, rather than\nprotein-encoding, genes might be involved. We undertook this\nstudy to elucidate the role of regulatory genes such as the\nmiRNAs in the predisposition and development of PTC.\nMethods\nPatient and Control Samples and Cell Lines: Nucleic Acid Extraction.\nAfter approval of the Institutional Review Board and patient\nconsent, fresh samples from the tumor tissue PTC (T-PTC) and\nnormal thyroid tissue adjacent to PTC tumors (N-PTC) were\nobtained from 20 patients with sporadic PTC undergoing sur-\ngical resection. The samples were snap-frozen in liquid nitrogen\nand stored at 80°C. Clinical data and information on the\nspecimens are shown in Table 2, which is published as supporting\ninformation on the PNAS web site. Normal thyroid tissue\n(N-Thy, n  6) was collected from consenting individuals who\nhad surgery because of laryngeal malignancy but no thyroid\ndisease. Other samples included: paraffin blocks of thyroid tissue\nfrom Finnish sporadic PTC patients (n  135); blood DNA\nindividuals (n  100);\nsamples of random Finnish control\nlymphoblastoid cell DNA samples (n  24) from Centre d’Etude\ndu Polymorphisme Humain control individuals who are not\ngenetically related. Thyroid cancer cell lines K1, K2, and NPA87\nwere cultured in DMEM mixed with F12 and MCDB (ratio\n2:1:1) medium supplemented with 10% FBS and glutamine in a\n5% CO2 incubator. Genomic DNA and RNA were extracted\nfrom the samples by standard techniques. Total RNA was\nextracted with TRIzol solution (Invitrogen), and the integrity of\nRNA was assessed by using an Agilent BioAnalyzer 2100 (Agi-\nlent, Palo Alto, CA).\nmiRNA Microarrays. Biotin-labeled miRNA was hybridized on\nmiRNA chips as described in ref. 26. Briefly, 5 g of total RNA\nfrom each sample was reverse transcribed by using biotin end-\nlabeled random octamers. Hybridization was carried out on our\nnew custom miRNA microarray chip (OSUCCC version 2.0),\nwhich contained 460 mature miRNA probes spotted in quadru-\nplicate (235 Homo sapiens, 222 Mus musculus, and 3 Arabidopsis\nthaliana) with annotated active sites. The hybridized chips were\nwashed and processed to detect biotin-containing transcripts by\nstreptavidin-Alexa647 conjugate and scanned by using an Axon\n4000B (Axon Instruments). Scanned images were quantified by\nGENEPIX PRO 6.0 (Axon Instruments).\nGene Expression Arrays. Affymetrix HG-U133 plus two chips\n(Affymetrix, Santa Clara, CA) were used to evaluate genome-\nwide gene expression levels. The experimental procedure was\naccording to the Affymetrix Gene-Chip Expression Analysis\nmanual. Double-stranded cDNA was synthesized by using the\nSuperscript Choice System, followed by an in vitro transcription\nreaction with a T-7 (dT24) primer to produce biotinylated\ncRNA. The full-length cRNAs were fragmented to 20 to 200 bp\nand hybridized to Affymetrix GeneChip Human Genome U133\nPlus 2.0 Array.\nStatistical and Bioinformatics Analysis of Microarray Data. Raw data\nfrom miRNA chips were assembled by using BRB ArrayTools\n(Richard Simon and Amy Peng Lam, National Cancer Institute,\nBethesda). BRB ArrayTools generated an average value of the\nfour spot replicates for each miRNA. Background subtracted\nintensities were thresholded to 10 and log transformed. Flagged\nspots corresponding to absent or low-quality signals were re-\nmoved from the analysis before global median normalization.\nSignificant analysis of microarrays and prediction analysis of\nmicroarrays were performed to identify differentially expressed\nmiRNAs in PTC tumors and miRNAs for tumor classification,\nrespectively. Data from Affymetrix chips were analyzed by using\nthe robust multichip average to calculate normalized expression\nvalues. Significance analysis of microarrays and permutation\ntests were used to find differentially expressed genes.\nNorthern Blot Analysis of miRNA. Northern blotting was performed\nas described in ref. 17. Ten micrograms of total RNA from\nsnap-frozen tissues and cell lines was loaded onto a precast 15%\ndenaturing polyacrylamide gel (Bio-Rad). The RNA was then\nelectrophoretically transferred to Bright-Star blotting mem-\nbranes (Ambian). The oligonucleotides used as probes were the\ncomplementary sequences of the mature miRNA (miRNA\nRegistry): miR-221, 5-GAAACCCAGCAGACAATGTA-\nGCT-3; miR-146b, 5-AGCCTATGGAATTCAGTTCTCA-3.\nProbes were end-labeled with [-32P]ATP by T4 polynucleotide\nkinase (USB, Cleveland). Prehybridization and hybridization\nwere carried out in Ultrahyb Oligo solution (Ambian) containing\n106 cpmml probes overnight at 42°C. The most stringent wash\nwas with 2 SSC and 1% SDS at 42°C. For reuse, blots were\nstripped by boiling and reprobed. As a loading control, 5S rRNA\nwas visualized by ethidium bromide staining of the gel before\ntransfer. The image of Northern hybridization signals was pro-\nduced by using STORMSCANNER and IMAGEQUANT TL software\n(Molecular Dynamics).\nSemiquantitative and Quantitative RT-PCR. Total RNA was first\ntreated with DNase-1 (Ambian) and then reverse transcribed\nto cDNA with the SuperScript First Strand Synthesis system\n(Invitrogen). Target genes and an endogenous control gene\nglyceraldehyde-3-phosphate dehydrogenase (GAPDH) were in-\ncluded in the same PCR reaction for semiquantitative RT-PCR.\nAll PCR reactions were verified to be in the linear range by\ntesting with different cycle numbers. Quantitative real-time PCR\nwas performed by using an ABI PRISM 7700 DNA Sequence\nDetection System (Applied Biosystems) and a SYBR Green\nPCR kit (Applied Biosystems). The comparative threshold cycle\nmethod was used to calculate the relative gene expression.\nPrimers for amplification of GAPDH, miR-221, miR-222, miR-\n146a, miR-146b, and CITED1 are listed in Table 3, which is\npublished as supporting information on the PNAS web site.\nWestern Blot. Total protein extract was prepared by using lysis\nbuffer containing 50 mM TrisHCl (pH 7.4), 150 mM NaCl, 1%\nTriton X-100, 0.1% SDS, 1 mM EDTA, plus proteinase inhib-\nitors. The protein concentration was determined by the Bio-Rad\nprotein assay. Thirty micrograms of each protein lysate was\nloaded onto SDSPAGE under reducing conditions and trans-\nferred to poly(vinylidene difluoride) membrane (Millipore). The\nmembrane was incubated in 5% nonfat dry milk in T-TBS (18\nmM TrisHCl, pH 7.6122 mM NaCl0.1% Tween 20) at room\ntemperature for 2 h followed by incubation with a monoclonal\nanti-Kit antibody (BD Biosciences) and a secondary antibody of\nhorseradish peroxidase-conjugated goat anti-mouse IgG. Im-\nmune complexes were detected with the Supersignal Western\nPico Chemiluminescent Substrate (Pierce). Loading control was\ndone by reincubating the same membrane with an anti-GAPDH\nantibody (Novus Biological).\nSequencing. DNA specimens were amplified by using a standard\nPCR protocol. The PCR products were purified with Ex-\noSAP-IT purification kit (USB) and sequenced in both direc-\ntions with the ABI sequencing system (PerkinElmer Applied\nBiosystems). The sequencing results were analyzed by using\nDNAStar SEQMAN software. The PCR primers used for KIT\nexons and miRNA sequencing are listed in Table 3.\nHybridization Prediction. The minimum free energy hybridization\nof miRNAs and target mRNA with all polymorphic variants were\npredicted by RNAHYBRID software (27).\nResults\nDistinct Overexpression of miRNAs in PTC Tumor Tissues. Initially\nmiRNA expression was analyzed in paired PTC tumor and\nunaffected thyroid tissue from nine patients. Pairwise signifi-\ncance analysis of microarray analysis indicated that six miRNA\ngenes were significantly overexpressed in T-PTC with fold\nchanges 1.5 in at least seven patients. These miRNAs were as\nfollows: miR-146, miR-221, miR-222, miR-21, miR-155, and\nmiR-181a (Table 1). The top three miRNAs (miR-146, miR-221,\nand miR-222) showed dramatic overexpression, with 11- to\n19-fold higher level in PTC tumors compared to the adjacent\nunaffected thyroid tissue. We repeated the miRNA chip exper-\niments with samples from another six PTC patients. The over-\nexpression of the six miRNAs identified previously was fully\nconfirmed in these new samples. Data analysis of the combined\n30 samples from 15 PTC patients indicated that the up-\nregulation of miRNAs in PTC tumors represented a distinct\nmiRNA signature comprising 23 miRNAs. Among the signifi-\ncant differentially expressed miRNA genes, overexpression oc-\ncurred in 17. Underexpression occurred in only six miRNAs, and\nin all of them, the change was 2-fold (Table 1).\nTo identify the smallest set of predictive miRNAs distinguish-\ning between normal and cancer tissue, we performed class\nprediction analysis by using the first set (18 samples) to predict\nthe outcome of the second set (12 samples). Results from the\nprediction analysis of microarrays analysis indicated that only\nfive overexpressed miRNAs (miR-221, miR-222, miR-146, miR-\n21, and miR-181a) were sufficient to successfully predict cancer\nstatus. No classification errors were made in this analysis (Fig. 5,\nwhich is published as supporting information on the PNAS web\nsite).\nConfirmation of Overexpression of miR-221, miR-222, and miR-146b in\nPTC Tumors. We performed RT-PCR and Northern blot analysis\nof the top three overexpressed miRNAs in PTC tissue samples\nand in thyroid tumor cell lines and normal thyroid tissue samples.\nOverexpression of miR-221 and miR-222 in PTC tumors was\nconfirmed by both semiquantitative RT-PCR and Northern blot\n(Fig. 1 A and B). The expression patterns of miR-221 and\nmiR-222 in PTC tumors were concordant in most of the cases.\nmiR-221 and miR-222 are located close to each other on the X\nchromosome. The concordant expression patterns in different\nPTC samples suggest shared regulatory mechanisms for the\nexpression of these two clustered miRNAs.\nThe probe sequence of miR-146 on the chip was designed\ncorresponding to miR-146a, located on chromosome 5. Recently\nmiR-146b on chromosome 10 was identified. The mature forms\nof these two miRNAs have high sequence homology, only two\nnucleotides being different. Using primers specific for miR-146a\nand miR-146b in their premature forms, no visible RT-PCR\nband of miR-146a was detectable in thyroid tissue. In contrast,\nmiR-146b was significantly overexpressed in PTC tumor samples\n(Fig. 1 A). This result indicated that miR-146a either was not\nexpressed in thyroid tissue or the expression level was very low.\nThus the overexpressed miR-146 on the microchips apparently\nreflects miR-146b. Indeed, the mature form of miR-146b was\nstrongly overexpressed in PTC tumors as confirmed by Northern\nblot (Fig. 1B).\nOverexpression of miR-221 in N-PTC. The expression of miR-221 was\ndetectable in normal thyroid tissue adjacent to PTC tumors\n(N-PTC) from all patients (Fig. 1 A). Interestingly, the miR-221\nexpression level in at least two N-PTC samples was much higher\nthan in the rest of them. For example, there was a strong band\nof miR-221 in samples N-PTC1 and N-PTC3 shown by semi-\nquantitative RT-PCR (Fig. 1 A). To examine whether this ob-\nservation might be an artifact due to tumor tissue contaminating\nthe ‘‘normal’’ tissue, we used the same samples to demonstrate\nthe expression of CITED1, a molecule that is known to be highly\nexpressed in PTC tumors but not in normal thyroid (28).\nExpression of the CITED1 gene was not detectable in the same\nset of RNA samples of these normal thyroid tissues, but over-\nexpressed in the corresponding paired tumors (Fig. 1 A) exclud-\ning significant contamination. The variation of miR-221 in\nnormal thyroid tissues was further tested with other N-PTC\nsamples and compared to normal thyroid tissue from individuals\nwithout any clinical thyroid disease. Quantitative real-time PCR\nrevealed the same observation in five sample pairs (Fig. 2A). The\noverexpressed miR-221 in two N-PTC samples was not due to\ncontamination of the RNA preparation with genomic DNA (Fig.\n2B). Considered together, our data suggest that increased ex-\npression of miR-221 in normal thyroid tissue might be an early\ngenetic event in PTC carcinogenesis.\nSmall Percentage of Putative miRNA Target Genes Showing Down-\nExpression at the mRNA Level in PTC Tumors. miRNAs interact with\ntheir target genes and, thereby, play regulatory roles in many\nphysiological functions and pathophysiological processes. We\nanalyzed the predicted targets of the three most significantly\noverexpressed miRNAs (221, 222, and 146). The analysis was\nperformed by using three publicly available algorithms to predict\nhuman miRNA gene targets, i.e., MIRANDA (29), TARGETSCAN\n(30), and PICTAR (31). To reduce the number of false positives,\nwe listed only putative target genes predicted by at least two of\nthe programs (Table 4, which is published as supporting infor-\nmation on the PNAS web site). The above approach produced\n130 putative targets of miR-221. Although KIT was only pre-\ndicted by MIRANDA, we included KIT in this list, because KIT is\na target of miR-221 as demonstrated by biological methods (32).\nIt is believed that miRNAs interact with target genes at\nspecific sites by inducing cleavage of the targeted message or by\ninhibiting translation (12). Therefore, one might predict that an\noverexpressed miRNA would be associated with down-\nexpression of its targets at the transcript or protein level. We\nexamined the gene expression level of the putative targets of\nmiR-221, -222, and -146 in the same set of RNA samples. This\nanalysis was done first by evaluating genomewide gene expres-\nsion levels in PTC tumors and matched normal thyroid tissues by\nusing the Affymetrix chip. Significance analysis of microarray\nanalysis revealed a list of genes differentially expressed (unpub-\nlished data). Most of these genes showed expression behaviors\nconsistent with our previously published data (28). For example,\noverexpression of the CITED1 gene was found in all PTC tumors\ntested and was confirmed by semiquantitative RT-PCR (Fig.\n1 A). Comparing the lists of putative miRNA targets (as defined\nabove) and differentially expressed genes, we found that the\ndifferentially expressed mRNAs accounted only for 15% of the\ntotal target genes (data not shown). Further analysis of miR-221\ntargets was performed by k-means clustering of the relative\nexpression changes between tumornormal pairs of target genes\n(using gene expression ratios) and miRNAs (using inverse\nmiRNA expression ratios) by using the EPCLUST program\n(http:ep.ebi.ac.ukEPEPCLUST). It appeared that a group\nof target genes (84 probe sets) that showed down-expression in\nPTC tumors was clustered together with miR-221 and miR-222.\nWithin this cluster, 19 genes were identified whose expression\nlevel was significantly down in PTC tumors compared to their\npaired unaffected tissue (random variance t test P value 0.005;\npermutation test P value 0.05; fold changes 1.5). The ex-\npression patterns of miR-221, miR-222, and these 19 targets are\nillustrated in Fig. 6, which is published as supporting information\non the PNAS web site. KIT was one of the genes showing\nprofound down-expression of mRNA in PTC tumors.\nDown-Expression of KIT Transcript and Protein in PTC Tumors. West-\nern blotting of Kit protein was performed with protein extracts\nobtained from five patients and two cell lines (Fig. 3). Hybrid-\nization with a monoclonal anti-Kit antibody revealed two bands\nof 140 kDa and 120 kDa, respectively, corresponding to the\nmature fully glycosylated and the partially glycosylated Kit (33)\n(Fig. 3). Both forms of Kit were readily detected in normal\nthyroid tissue but were strongly reduced or absent in the four\nsamples showing strong reduction of KIT transcript. The down-\nexpression of KIT transcript and Kit protein level correlated well\nwith the strong overexpression of miRs-221, -222, and -146b in\nthese four PTC patient samples (Fig. 1). However, in two further\ncases with KIT transcript down-regulation, Kit protein was\nunchanged (data not shown).\nMutations in the KIT Gene. We sequenced the miR-221, -222, and\n-146b genes and their flanking regions in samples from 48 PTC\npatients and three PTC cell lines. No previously unidentified\nsequence changes or DNA polymorphisms were found (data not\nshown).\nmiRNA-binding domains in the KIT sequence were predicted\nwith several software programs (27, 29, 31). We sequenced the\nregions harboring two binding domains, one for miR-221 and\nmiR-222, and another one for miR-146a and miR-146b,\nin\ngenomic DNA samples from the PTC patients. We found a\npolymorphism in each recognition site. The 3169G3A SNP\n(rs17084733) is located within the crucial region of the miR-221\nand miR-222 domain in the KIT 3 UTR region. We found GA\nheterozygosity in five PTC patients. Heterozygosity for\n3169G3A leads to a conformation change with an increase of\nfree energy (Fig. 4A). The synonymous 2607G3 C SNP\n(rs3733542) in exon 18 is located within the crucial region of the\nmiR-146a and miR-146b domain. This genotype leads to changes\nin miRNA:target gene mRNA duplex conformation and results\nin hybridization with a different region (Fig. 4B). We found GC\nheterozygosity in the same five PTC patients, suggesting linkage\ndisequilibrium. Notably, the tumors of all these five doubly\nheterozygous patients (PTC3, PTC6, PTC8, PTC10, and\nPTC11) had up-regulation of the three top miRs and profound\ndown-regulation of the KIT transcript. Sequencing of all of the\nremaining KIT exons in nine PTC cases and three cell lines\ndisclosed one additional, previously unreported change in exon\n17 (ex17  33, CT) in one PTC patient. To exclude that the\nrare alleles at these SNPs themselves predispose to PTC, we\ncompared the allele frequencies in cohorts of sporadic PTC\npatients and healthy controls (both cohorts from Finland). No\nassociation was found (Table 5, which is published as supporting\ninformation on the PNAS web site).\nDiscussion\nA relatively unexpected finding of ours was the highly prefer-\nential\nincrease rather than decrease in transcript of many\nmiRNA genes in the tumors compared with paired unaffected\nthyroid tissue. Such up-regulation has previously been noted in\na few cancers, but decreases in the tumor seem to be more\ncommon (21, 22, 24). For example, miR-15 and miR-16 are\nfrequently down-regulated in B cell chronic lymphocytic leuke-\nmia (17); miR-143 and miR-145 showed decreased expression in\ncolorectal tumors (34); miRNA let-7 was down-expressed in\nhuman lung cancer, which is associated with poor prognosis (24).\nDown-expression of miRNAs was also found in cancers of the\nbreast, kidney, prostate, and uterus (21, 22). Over-expression of\nmiRNAs was reported in some tumors: miR-155 was overex-\npressed in Burkitt’s lymphomas and human B cell lymphoma\n(35), and a cluster of miRNAs, the miR-17–92 polycistron, were\noverexpressed in B cell lymphoma patients (23). Although we\nhave no clear explanation of the preferential up- rather than\ndown-regulation of miRNAs in PTC, we note that this cancer has\nunusually little loss of heterozygosity (LOH) (36, 37). If LOH is\ngenerally associated with loss of mRNA transcript, then the\npaucity of LOH might, in part, explain our findings. It follows\nthat genomic amplification might explain abundant miRNA\ntranscript.\nComparatively little is known about genetic changes in PTC,\nand no predisposing mutations have been described. Our find-\nings are compatible with the hypothesis that RNA-based regu-\nlatory mechanisms involving miRNAs, and possibly other RNA\ngenes, characterize this tumor. The existence of regulatory\nnetworks, including inherited changes therein, is now established\nin principle, although almost nothing is known about the genes\nthemselves (38).\nWe observed that miR-221 expression was not only up-\nregulated in PTC tumors, but also showed clear-cut variation in\nthe unaffected thyroid tissue among patients with PTC. In at\nleast 3 of 15 patients, the up-regulation was strong. This obser-\nvation suggests that unaffected normal thyroid tissue adjacent to\ntumors may harbor genetic changes before the appearance of\nmorphological malignancy. Overexpression of miR-221 could be\na premalignant change in PTC. Taken together, miRNA dereg-\nulation in the thyroid could be a critical component of PTC\ninitiation and development. We hypothesize that miR-221 may\nfunction as an oncogene in the thyroid. Future study is warranted\nto explore the significance of the overexpression of miRNAs in\nthyroid tissues from PTC patients.\nKit is an important tyrosine kinase receptor in cell differen-\ntiation and growth; it functions as an oncogene in many cancers\n(39, 40). The transcript level of KIT in PTC tumor cells is known\nto be extremely low (41, 42), which is consistent with our data.\nIn contrast, reports about the protein level in PTC are contra-\ndictory (43, 44). Although in several PTC tumors we found Kit\nprotein to be down-regulated, this observation did not apply to\nall tumors, emphasizing the complexity of these regulatory\npathways, which may well be organ- or cell-specific.\nComputational predictions and experimental approaches sup-\nport the idea that different miRNAs target the same mRNA\n(29). Multiple miRNAs have been predicted to target KIT,\nincluding those overexpressed in PTC (27, 29, 31). We show here\nthat in most PTCs at least three miRs (221, 222, and 146)\ntargeting KIT were up-regulated. It is entirely possible that the\nmultiple interaction opportunities provided by networks or\n‘‘signatures’’ of miRNA dysregulation create different responses\nin target genes under different circumstances and combinations.\nPerhaps most interestingly, our results highlight the role of the\ntarget genes themselves in their responses to miRNA interaction.\nWe found a SNP in KIT precisely in the crucial region of the 3\nUTR-binding domain for miR-221 and -222, and another SNP\nprecisely in the crucial region of the exonic binding domain for\nmiR-146a and -146b. The crucial region is a stretch of seven\nnucleotides in the miRNA:mRNA duplex, usually located in the\n5 end of the miRNA, starting at the first or second position. It\nis likely that variants within the binding regions can acquire new\nmiRNAs as regulating factors due to sequence changes such as\nSNPs. Remarkably, of 10 tumors in which the key miRNA genes\nwere up-regulated, all 10 showed low or very low KIT transcript,\nand 5 of these patients were the only ones (of 15 tested) showing\nheterozygosity for the germline SNPs. These findings suggest\nthat not only changes in miRNAs but also in their target genes\n(inherited or perhaps acquired) profoundly influence PTC car-\ncinogenesis. Our association study suggests that these events are\ndownstream of the putative genetic initiating or predisposing\nevents. More detailed functional studies must address questions\nsuch as how miR-221, -222, and -146b interact with their binding\ndomains in the KIT gene, the effects of the two SNPs on the\ninteractions, and the combinational effects of miR-221, -222, and\n-146b on KIT and other possible targets.\nWe thank Jan Lockman and Heejei Yoon for help and advice. This work\nwas supported by National Institutes of Health (NIH) Grants CA16058\n(to A.d.l.C.) and P01 CA78890 (to C.M.C.). BRB ArrayTools was\ndeveloped by Dr. Richard Simon and Amy Peng Lam. Thyroid tissue\nsamples were provided by the Cooperative Human Tissue Network,\nwhich is funded by the National Cancer Institute.\n4. Cohen, Y., Xing, M., Mambo, E., Guo, Z., Wu, G., Trink, B., Beller, U., Westra,\nW. H., Ladenson, P. W. & Sidransky, D. (2003) J. Natl. Cancer Inst. 95, 625–627.\n5. Nikiforova, M. N., Kimura, E. T., Gandhi, M., Biddinger, P. W., Knauf, J. A.,\nBasolo, F., Zhu, Z., Giannini, R., Salvatore, G., Fusco, A., et al. (2003) J. Clin.\nEndocrinol. Metab. 88, 5399–5404.",
         "",
         "",
         "",
         "Apart from alterations in the RETPTC-RAS-BRAF pathway, com-\nparatively little is known about the genetics of papillary thyroid\ncarcinoma (PTC). We show that numerous microRNAs (miRNAs) are\ntranscriptionally up-regulated in PTC tumors compared with unaf-\nfected thyroid tissue. A set of ﬁve miRNAs, including the three most\nup-regulated ones (miR-221, -222, and -146), distinguished un-\nequivocally between PTC and normal thyroid. Additionally, miR-\n221 was up-regulated in unaffected thyroid tissue in several PTC\npatients, presumably an early event in carcinogenesis. Tumors in\nwhich the up-regulation (11- to 19-fold) of miR-221, -222, and -146\nwas strongest showed dramatic loss of KIT transcript and Kit\nprotein. In 5 of 10 such cases, this down expression was associated\nwith germline single-nucleotide changes in the two recognition\nsequences in KIT for these miRNAs. We conclude that up-regulation\nof several miRs and regulation of KIT are involved in PTC patho-\ngenesis, and that sequence changes in genes targeted by miRNAs\ncan contribute to their regulation.\nmalignancy in thyroid tissue, accounting for 80% of all\nthyroid cancers. The incidence of PTC in the United States has\nincreased in recent years (1). Genetically, PTC is characterized\nby alterations in the RETPTC-RAS-BRAF signaling pathway\n(2, 3). Activating mutations in BRAF and RETPTC gene\nrearrangements are frequent genetic changes in PTC tumors\n(4–6). A strong inherited genetic predisposition is suggested by\ncase-control studies showing a 3- to 8-fold risk in first-degree\nrelatives, one of the highest of all cancers (7, 8). Despite\nunequivocal evidence of an inherited predisposition, large fam-\nilies displaying Mendelian inheritance of PTC are rare, and no\npredisposing gene mutations have been found, even though\nseveral putative loci have been identified by linkage analysis\n(9–11).\nMicroRNAs (miRNAs) represent a previously uncharacter-\nized class of gene products that are believed to function as\nnegative regulators of gene expression (12–16). Recently,\nmiRNA genes have been implicated in several cancers (14,\n17–25). The expression of miRNAs varies between cancer and\nnormal cells and varies among different types of cancer. In most\ncancers studied so far, the expression of miRNAs seems to be\nlower than in the corresponding normal tissue (17, 18, 21, 22, 24).\nWe reasoned that the previous failure to identify genes\npredisposing or contributing to PTC might be because these\ngenes show low penetrance. The mechanisms may require the\ninteraction of two or more genes; thus, regulatory, rather than\nprotein-encoding, genes might be involved. We undertook this\nstudy to elucidate the role of regulatory genes such as the\nmiRNAs in the predisposition and development of PTC.\nMethods\nPatient and Control Samples and Cell Lines: Nucleic Acid Extraction.\nAfter approval of the Institutional Review Board and patient\nconsent, fresh samples from the tumor tissue PTC (T-PTC) and\nnormal thyroid tissue adjacent to PTC tumors (N-PTC) were\nobtained from 20 patients with sporadic PTC undergoing sur-\ngical resection. The samples were snap-frozen in liquid nitrogen\nand stored at 80°C. Clinical data and information on the\nspecimens are shown in Table 2, which is published as supporting\ninformation on the PNAS web site. Normal thyroid tissue\n(N-Thy, n  6) was collected from consenting individuals who\nhad surgery because of laryngeal malignancy but no thyroid\ndisease. Other samples included: paraffin blocks of thyroid tissue\nfrom Finnish sporadic PTC patients (n  135); blood DNA\nindividuals (n  100);\nsamples of random Finnish control\nlymphoblastoid cell DNA samples (n  24) from Centre d’Etude\ndu Polymorphisme Humain control individuals who are not\ngenetically related. Thyroid cancer cell lines K1, K2, and NPA87\nwere cultured in DMEM mixed with F12 and MCDB (ratio\n2:1:1) medium supplemented with 10% FBS and glutamine in a\n5% CO2 incubator. Genomic DNA and RNA were extracted\nfrom the samples by standard techniques. Total RNA was\nextracted with TRIzol solution (Invitrogen), and the integrity of\nRNA was assessed by using an Agilent BioAnalyzer 2100 (Agi-\nlent, Palo Alto, CA).\nmiRNA Microarrays. Biotin-labeled miRNA was hybridized on\nmiRNA chips as described in ref. 26. Briefly, 5 g of total RNA\nfrom each sample was reverse transcribed by using biotin end-\nlabeled random octamers. Hybridization was carried out on our\nnew custom miRNA microarray chip (OSUCCC version 2.0),\nwhich contained 460 mature miRNA probes spotted in quadru-\nplicate (235 Homo sapiens, 222 Mus musculus, and 3 Arabidopsis\nthaliana) with annotated active sites. The hybridized chips were\nwashed and processed to detect biotin-containing transcripts by\nstreptavidin-Alexa647 conjugate and scanned by using an Axon\n4000B (Axon Instruments). Scanned images were quantified by\nGENEPIX PRO 6.0 (Axon Instruments).\nGene Expression Arrays. Affymetrix HG-U133 plus two chips\n(Affymetrix, Santa Clara, CA) were used to evaluate genome-\nwide gene expression levels. The experimental procedure was\naccording to the Affymetrix Gene-Chip Expression Analysis\nmanual. Double-stranded cDNA was synthesized by using the\nSuperscript Choice System, followed by an in vitro transcription\nreaction with a T-7 (dT24) primer to produce biotinylated\ncRNA. The full-length cRNAs were fragmented to 20 to 200 bp\nand hybridized to Affymetrix GeneChip Human Genome U133\nPlus 2.0 Array.\nStatistical and Bioinformatics Analysis of Microarray Data. Raw data\nfrom miRNA chips were assembled by using BRB ArrayTools\n(Richard Simon and Amy Peng Lam, National Cancer Institute,\nBethesda). BRB ArrayTools generated an average value of the\nfour spot replicates for each miRNA. Background subtracted\nintensities were thresholded to 10 and log transformed. Flagged\nspots corresponding to absent or low-quality signals were re-\nmoved from the analysis before global median normalization.\nSignificant analysis of microarrays and prediction analysis of\nmicroarrays were performed to identify differentially expressed\nmiRNAs in PTC tumors and miRNAs for tumor classification,\nrespectively. Data from Affymetrix chips were analyzed by using\nthe robust multichip average to calculate normalized expression\nvalues. Significance analysis of microarrays and permutation\ntests were used to find differentially expressed genes.\nNorthern Blot Analysis of miRNA. Northern blotting was performed\nas described in ref. 17. Ten micrograms of total RNA from\nsnap-frozen tissues and cell lines was loaded onto a precast 15%\ndenaturing polyacrylamide gel (Bio-Rad). The RNA was then\nelectrophoretically transferred to Bright-Star blotting mem-\nbranes (Ambian). The oligonucleotides used as probes were the\ncomplementary sequences of the mature miRNA (miRNA\nRegistry): miR-221, 5-GAAACCCAGCAGACAATGTA-\nGCT-3; miR-146b, 5-AGCCTATGGAATTCAGTTCTCA-3.\nProbes were end-labeled with [-32P]ATP by T4 polynucleotide\nkinase (USB, Cleveland). Prehybridization and hybridization\nwere carried out in Ultrahyb Oligo solution (Ambian) containing\n106 cpmml probes overnight at 42°C. The most stringent wash\nwas with 2 SSC and 1% SDS at 42°C. For reuse, blots were\nstripped by boiling and reprobed. As a loading control, 5S rRNA\nwas visualized by ethidium bromide staining of the gel before\ntransfer. The image of Northern hybridization signals was pro-\nduced by using STORMSCANNER and IMAGEQUANT TL software\n(Molecular Dynamics).\nSemiquantitative and Quantitative RT-PCR. Total RNA was first\ntreated with DNase-1 (Ambian) and then reverse transcribed\nto cDNA with the SuperScript First Strand Synthesis system\n(Invitrogen). Target genes and an endogenous control gene\nglyceraldehyde-3-phosphate dehydrogenase (GAPDH) were in-\ncluded in the same PCR reaction for semiquantitative RT-PCR.\nAll PCR reactions were verified to be in the linear range by\ntesting with different cycle numbers. Quantitative real-time PCR\nwas performed by using an ABI PRISM 7700 DNA Sequence\nDetection System (Applied Biosystems) and a SYBR Green\nPCR kit (Applied Biosystems). The comparative threshold cycle\nmethod was used to calculate the relative gene expression.\nPrimers for amplification of GAPDH, miR-221, miR-222, miR-\n146a, miR-146b, and CITED1 are listed in Table 3, which is\npublished as supporting information on the PNAS web site.\nWestern Blot. Total protein extract was prepared by using lysis\nbuffer containing 50 mM TrisHCl (pH 7.4), 150 mM NaCl, 1%\nTriton X-100, 0.1% SDS, 1 mM EDTA, plus proteinase inhib-\nitors. The protein concentration was determined by the Bio-Rad\nprotein assay. Thirty micrograms of each protein lysate was\nloaded onto SDSPAGE under reducing conditions and trans-\nferred to poly(vinylidene difluoride) membrane (Millipore). The\nmembrane was incubated in 5% nonfat dry milk in T-TBS (18\nmM TrisHCl, pH 7.6122 mM NaCl0.1% Tween 20) at room\ntemperature for 2 h followed by incubation with a monoclonal\nanti-Kit antibody (BD Biosciences) and a secondary antibody of\nhorseradish peroxidase-conjugated goat anti-mouse IgG. Im-\nmune complexes were detected with the Supersignal Western\nPico Chemiluminescent Substrate (Pierce). Loading control was\ndone by reincubating the same membrane with an anti-GAPDH\nantibody (Novus Biological).\nSequencing. DNA specimens were amplified by using a standard\nPCR protocol. The PCR products were purified with Ex-\noSAP-IT purification kit (USB) and sequenced in both direc-\ntions with the ABI sequencing system (PerkinElmer Applied\nBiosystems). The sequencing results were analyzed by using\nDNAStar SEQMAN software. The PCR primers used for KIT\nexons and miRNA sequencing are listed in Table 3.\nHybridization Prediction. The minimum free energy hybridization\nof miRNAs and target mRNA with all polymorphic variants were\npredicted by RNAHYBRID software (27).\nResults\nDistinct Overexpression of miRNAs in PTC Tumor Tissues. Initially\nmiRNA expression was analyzed in paired PTC tumor and\nunaffected thyroid tissue from nine patients. Pairwise signifi-\ncance analysis of microarray analysis indicated that six miRNA\ngenes were significantly overexpressed in T-PTC with fold\nchanges 1.5 in at least seven patients. These miRNAs were as\nfollows: miR-146, miR-221, miR-222, miR-21, miR-155, and\nmiR-181a (Table 1). The top three miRNAs (miR-146, miR-221,\nand miR-222) showed dramatic overexpression, with 11- to\n19-fold higher level in PTC tumors compared to the adjacent\nunaffected thyroid tissue. We repeated the miRNA chip exper-\niments with samples from another six PTC patients. The over-\nexpression of the six miRNAs identified previously was fully\nconfirmed in these new samples. Data analysis of the combined\n30 samples from 15 PTC patients indicated that the up-\nregulation of miRNAs in PTC tumors represented a distinct\nmiRNA signature comprising 23 miRNAs. Among the signifi-\ncant differentially expressed miRNA genes, overexpression oc-\ncurred in 17. Underexpression occurred in only six miRNAs, and\nin all of them, the change was 2-fold (Table 1).\nTo identify the smallest set of predictive miRNAs distinguish-\ning between normal and cancer tissue, we performed class\nprediction analysis by using the first set (18 samples) to predict\nthe outcome of the second set (12 samples). Results from the\nprediction analysis of microarrays analysis indicated that only\nfive overexpressed miRNAs (miR-221, miR-222, miR-146, miR-\n21, and miR-181a) were sufficient to successfully predict cancer\nstatus. No classification errors were made in this analysis (Fig. 5,\nwhich is published as supporting information on the PNAS web\nsite).\nConfirmation of Overexpression of miR-221, miR-222, and miR-146b in\nPTC Tumors. We performed RT-PCR and Northern blot analysis\nof the top three overexpressed miRNAs in PTC tissue samples\nand in thyroid tumor cell lines and normal thyroid tissue samples.\nOverexpression of miR-221 and miR-222 in PTC tumors was\nconfirmed by both semiquantitative RT-PCR and Northern blot\n(Fig. 1 A and B). The expression patterns of miR-221 and\nmiR-222 in PTC tumors were concordant in most of the cases.\nmiR-221 and miR-222 are located close to each other on the X\nchromosome. The concordant expression patterns in different\nPTC samples suggest shared regulatory mechanisms for the\nexpression of these two clustered miRNAs.\nThe probe sequence of miR-146 on the chip was designed\ncorresponding to miR-146a, located on chromosome 5. Recently\nmiR-146b on chromosome 10 was identified. The mature forms\nof these two miRNAs have high sequence homology, only two\nnucleotides being different. Using primers specific for miR-146a\nand miR-146b in their premature forms, no visible RT-PCR\nband of miR-146a was detectable in thyroid tissue. In contrast,\nmiR-146b was significantly overexpressed in PTC tumor samples\n(Fig. 1 A). This result indicated that miR-146a either was not\nexpressed in thyroid tissue or the expression level was very low.\nThus the overexpressed miR-146 on the microchips apparently\nreflects miR-146b. Indeed, the mature form of miR-146b was\nstrongly overexpressed in PTC tumors as confirmed by Northern\nblot (Fig. 1B).\nOverexpression of miR-221 in N-PTC. The expression of miR-221 was\ndetectable in normal thyroid tissue adjacent to PTC tumors\n(N-PTC) from all patients (Fig. 1 A). Interestingly, the miR-221\nexpression level in at least two N-PTC samples was much higher\nthan in the rest of them. For example, there was a strong band\nof miR-221 in samples N-PTC1 and N-PTC3 shown by semi-\nquantitative RT-PCR (Fig. 1 A). To examine whether this ob-\nservation might be an artifact due to tumor tissue contaminating\nthe ‘‘normal’’ tissue, we used the same samples to demonstrate\nthe expression of CITED1, a molecule that is known to be highly\nexpressed in PTC tumors but not in normal thyroid (28).\nExpression of the CITED1 gene was not detectable in the same\nset of RNA samples of these normal thyroid tissues, but over-\nexpressed in the corresponding paired tumors (Fig. 1 A) exclud-\ning significant contamination. The variation of miR-221 in\nnormal thyroid tissues was further tested with other N-PTC\nsamples and compared to normal thyroid tissue from individuals\nwithout any clinical thyroid disease. Quantitative real-time PCR\nrevealed the same observation in five sample pairs (Fig. 2A). The\noverexpressed miR-221 in two N-PTC samples was not due to\ncontamination of the RNA preparation with genomic DNA (Fig.\n2B). Considered together, our data suggest that increased ex-\npression of miR-221 in normal thyroid tissue might be an early\ngenetic event in PTC carcinogenesis.\nSmall Percentage of Putative miRNA Target Genes Showing Down-\nExpression at the mRNA Level in PTC Tumors. miRNAs interact with\ntheir target genes and, thereby, play regulatory roles in many\nphysiological functions and pathophysiological processes. We\nanalyzed the predicted targets of the three most significantly\noverexpressed miRNAs (221, 222, and 146). The analysis was\nperformed by using three publicly available algorithms to predict\nhuman miRNA gene targets, i.e., MIRANDA (29), TARGETSCAN\n(30), and PICTAR (31). To reduce the number of false positives,\nwe listed only putative target genes predicted by at least two of\nthe programs (Table 4, which is published as supporting infor-\nmation on the PNAS web site). The above approach produced\n130 putative targets of miR-221. Although KIT was only pre-\ndicted by MIRANDA, we included KIT in this list, because KIT is\na target of miR-221 as demonstrated by biological methods (32).\nIt is believed that miRNAs interact with target genes at\nspecific sites by inducing cleavage of the targeted message or by\ninhibiting translation (12). Therefore, one might predict that an\noverexpressed miRNA would be associated with down-\nexpression of its targets at the transcript or protein level. We\nexamined the gene expression level of the putative targets of\nmiR-221, -222, and -146 in the same set of RNA samples. This\nanalysis was done first by evaluating genomewide gene expres-\nsion levels in PTC tumors and matched normal thyroid tissues by\nusing the Affymetrix chip. Significance analysis of microarray\nanalysis revealed a list of genes differentially expressed (unpub-\nlished data). Most of these genes showed expression behaviors\nconsistent with our previously published data (28). For example,\noverexpression of the CITED1 gene was found in all PTC tumors\ntested and was confirmed by semiquantitative RT-PCR (Fig.\n1 A). Comparing the lists of putative miRNA targets (as defined\nabove) and differentially expressed genes, we found that the\ndifferentially expressed mRNAs accounted only for 15% of the\ntotal target genes (data not shown). Further analysis of miR-221\ntargets was performed by k-means clustering of the relative\nexpression changes between tumornormal pairs of target genes\n(using gene expression ratios) and miRNAs (using inverse\nmiRNA expression ratios) by using the EPCLUST program\n(http:ep.ebi.ac.ukEPEPCLUST). It appeared that a group\nof target genes (84 probe sets) that showed down-expression in\nPTC tumors was clustered together with miR-221 and miR-222.\nWithin this cluster, 19 genes were identified whose expression\nlevel was significantly down in PTC tumors compared to their\npaired unaffected tissue (random variance t test P value 0.005;\npermutation test P value 0.05; fold changes 1.5). The ex-\npression patterns of miR-221, miR-222, and these 19 targets are\nillustrated in Fig. 6, which is published as supporting information\non the PNAS web site. KIT was one of the genes showing\nprofound down-expression of mRNA in PTC tumors.\nDown-Expression of KIT Transcript and Protein in PTC Tumors. West-\nern blotting of Kit protein was performed with protein extracts\nobtained from five patients and two cell lines (Fig. 3). Hybrid-\nization with a monoclonal anti-Kit antibody revealed two bands\nof 140 kDa and 120 kDa, respectively, corresponding to the\nmature fully glycosylated and the partially glycosylated Kit (33)\n(Fig. 3). Both forms of Kit were readily detected in normal\nthyroid tissue but were strongly reduced or absent in the four\nsamples showing strong reduction of KIT transcript. The down-\nexpression of KIT transcript and Kit protein level correlated well\nwith the strong overexpression of miRs-221, -222, and -146b in\nthese four PTC patient samples (Fig. 1). However, in two further\ncases with KIT transcript down-regulation, Kit protein was\nunchanged (data not shown).\nMutations in the KIT Gene. We sequenced the miR-221, -222, and\n-146b genes and their flanking regions in samples from 48 PTC\npatients and three PTC cell lines. No previously unidentified\nsequence changes or DNA polymorphisms were found (data not\nshown).\nmiRNA-binding domains in the KIT sequence were predicted\nwith several software programs (27, 29, 31). We sequenced the\nregions harboring two binding domains, one for miR-221 and\nmiR-222, and another one for miR-146a and miR-146b,\nin\ngenomic DNA samples from the PTC patients. We found a\npolymorphism in each recognition site. The 3169G3A SNP\n(rs17084733) is located within the crucial region of the miR-221\nand miR-222 domain in the KIT 3 UTR region. We found GA\nheterozygosity in five PTC patients. Heterozygosity for\n3169G3A leads to a conformation change with an increase of\nfree energy (Fig. 4A). The synonymous 2607G3 C SNP\n(rs3733542) in exon 18 is located within the crucial region of the\nmiR-146a and miR-146b domain. This genotype leads to changes\nin miRNA:target gene mRNA duplex conformation and results\nin hybridization with a different region (Fig. 4B). We found GC\nheterozygosity in the same five PTC patients, suggesting linkage\ndisequilibrium. Notably, the tumors of all these five doubly\nheterozygous patients (PTC3, PTC6, PTC8, PTC10, and\nPTC11) had up-regulation of the three top miRs and profound\ndown-regulation of the KIT transcript. Sequencing of all of the\nremaining KIT exons in nine PTC cases and three cell lines\ndisclosed one additional, previously unreported change in exon\n17 (ex17  33, CT) in one PTC patient. To exclude that the\nrare alleles at these SNPs themselves predispose to PTC, we\ncompared the allele frequencies in cohorts of sporadic PTC\npatients and healthy controls (both cohorts from Finland). No\nassociation was found (Table 5, which is published as supporting\ninformation on the PNAS web site).\nDiscussion\nA relatively unexpected finding of ours was the highly prefer-\nential\nincrease rather than decrease in transcript of many\nmiRNA genes in the tumors compared with paired unaffected\nthyroid tissue. Such up-regulation has previously been noted in\na few cancers, but decreases in the tumor seem to be more\ncommon (21, 22, 24). For example, miR-15 and miR-16 are\nfrequently down-regulated in B cell chronic lymphocytic leuke-\nmia (17); miR-143 and miR-145 showed decreased expression in\ncolorectal tumors (34); miRNA let-7 was down-expressed in\nhuman lung cancer, which is associated with poor prognosis (24).\nDown-expression of miRNAs was also found in cancers of the\nbreast, kidney, prostate, and uterus (21, 22). Over-expression of\nmiRNAs was reported in some tumors: miR-155 was overex-\npressed in Burkitt’s lymphomas and human B cell lymphoma\n(35), and a cluster of miRNAs, the miR-17–92 polycistron, were\noverexpressed in B cell lymphoma patients (23). Although we\nhave no clear explanation of the preferential up- rather than\ndown-regulation of miRNAs in PTC, we note that this cancer has\nunusually little loss of heterozygosity (LOH) (36, 37). If LOH is\ngenerally associated with loss of mRNA transcript, then the\npaucity of LOH might, in part, explain our findings. It follows\nthat genomic amplification might explain abundant miRNA\ntranscript.\nComparatively little is known about genetic changes in PTC,\nand no predisposing mutations have been described. Our find-\nings are compatible with the hypothesis that RNA-based regu-\nlatory mechanisms involving miRNAs, and possibly other RNA\ngenes, characterize this tumor. The existence of regulatory\nnetworks, including inherited changes therein, is now established\nin principle, although almost nothing is known about the genes\nthemselves (38).\nWe observed that miR-221 expression was not only up-\nregulated in PTC tumors, but also showed clear-cut variation in\nthe unaffected thyroid tissue among patients with PTC. In at\nleast 3 of 15 patients, the up-regulation was strong. This obser-\nvation suggests that unaffected normal thyroid tissue adjacent to\ntumors may harbor genetic changes before the appearance of\nmorphological malignancy. Overexpression of miR-221 could be\na premalignant change in PTC. Taken together, miRNA dereg-\nulation in the thyroid could be a critical component of PTC\ninitiation and development. We hypothesize that miR-221 may\nfunction as an oncogene in the thyroid. Future study is warranted\nto explore the significance of the overexpression of miRNAs in\nthyroid tissues from PTC patients.\nKit is an important tyrosine kinase receptor in cell differen-\ntiation and growth; it functions as an oncogene in many cancers\n(39, 40). The transcript level of KIT in PTC tumor cells is known\nto be extremely low (41, 42), which is consistent with our data.\nIn contrast, reports about the protein level in PTC are contra-\ndictory (43, 44). Although in several PTC tumors we found Kit\nprotein to be down-regulated, this observation did not apply to\nall tumors, emphasizing the complexity of these regulatory\npathways, which may well be organ- or cell-specific.\nComputational predictions and experimental approaches sup-\nport the idea that different miRNAs target the same mRNA\n(29). Multiple miRNAs have been predicted to target KIT,\nincluding those overexpressed in PTC (27, 29, 31). We show here\nthat in most PTCs at least three miRs (221, 222, and 146)\ntargeting KIT were up-regulated. It is entirely possible that the\nmultiple interaction opportunities provided by networks or\n‘‘signatures’’ of miRNA dysregulation create different responses\nin target genes under different circumstances and combinations.\nPerhaps most interestingly, our results highlight the role of the\ntarget genes themselves in their responses to miRNA interaction.\nWe found a SNP in KIT precisely in the crucial region of the 3\nUTR-binding domain for miR-221 and -222, and another SNP\nprecisely in the crucial region of the exonic binding domain for\nmiR-146a and -146b. The crucial region is a stretch of seven\nnucleotides in the miRNA:mRNA duplex, usually located in the\n5 end of the miRNA, starting at the first or second position. It\nis likely that variants within the binding regions can acquire new\nmiRNAs as regulating factors due to sequence changes such as\nSNPs. Remarkably, of 10 tumors in which the key miRNA genes\nwere up-regulated, all 10 showed low or very low KIT transcript,\nand 5 of these patients were the only ones (of 15 tested) showing\nheterozygosity for the germline SNPs. These findings suggest\nthat not only changes in miRNAs but also in their target genes\n(inherited or perhaps acquired) profoundly influence PTC car-\ncinogenesis. Our association study suggests that these events are\ndownstream of the putative genetic initiating or predisposing\nevents. More detailed functional studies must address questions\nsuch as how miR-221, -222, and -146b interact with their binding\ndomains in the KIT gene, the effects of the two SNPs on the\ninteractions, and the combinational effects of miR-221, -222, and\n-146b on KIT and other possible targets.\nWe thank Jan Lockman and Heejei Yoon for help and advice. This work\nwas supported by National Institutes of Health (NIH) Grants CA16058\n(to A.d.l.C.) and P01 CA78890 (to C.M.C.). BRB ArrayTools was\ndeveloped by Dr. Richard Simon and Amy Peng Lam. Thyroid tissue\nsamples were provided by the Cooperative Human Tissue Network,\nwhich is funded by the National Cancer Institute.\n4. Cohen, Y., Xing, M., Mambo, E., Guo, Z., Wu, G., Trink, B., Beller, U., Westra,\nW. H., Ladenson, P. W. & Sidransky, D. (2003) J. Natl. Cancer Inst. 95, 625–627.\n5. Nikiforova, M. N., Kimura, E. T., Gandhi, M., Biddinger, P. W., Knauf, J. A.,\nBasolo, F., Zhu, Z., Giannini, R., Salvatore, G., Fusco, A., et al. (2003) J. Clin.\nEndocrinol. Metab. 88, 5399–5404.",
         "Huiling He,K. Jażdżewski,Wei Li,S. Liyanarachchi,R. Nagy,S. Volinia,G. Calin,Chang-gong Liu,K. Franssila,S. Suster,R. Kloos,C. Croce,A. de la Chapelle",
         null
        ],
        [
         "25",
         "001ce484190e733e0d1dee181f9ee37fd0b15581",
         null,
         null,
         null,
         null,
         null,
         null,
         "L. Beuchat,C. Pettigrew,M. Tremblay,B. J. Roselle,A. Scouten",
         "captcha"
        ],
        [
         "26",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "plays an important role in cellular homeostasis.1 Three types of\ncell death are known: apoptosis (type I), autophagy (type II), and\nnecrosis (type III).2 Apoptosis is induced by a number of stimuli,\nincluding anticancer drugs, growth factor deprivation, and irradia-\ntion. The molecular mechanism by which apoptosis occurs has been\nextensively examined. Apoptosis in response to chemotherapy is\nknown to correlate with a reduction in tumor volume, and resistance\nto apoptosis causes drug resistance.3–5 Introduction of a proapoptotic\ngene such as Bax,6,7 and inhibition of an antiapoptotic gene such as\nBcl-2,8,9 enhances the effects of chemotherapy. Thus, apoptosis\nclearly inﬂuences the antitumor effect(s) of chemotherapeutic agents.\nIn solid tumors, hypoxia and glucose deprivation enable tumor\nprogression,10 whereas depletion of adenosine 5-triphosphate (ATP)\nresults in necrosis. The production of lactic acid under anaerobic\nconditions does not necessarily induce necrosis, rather, autophagic\ncell death may depend on susceptibility to apoptosis, which can be\ninhibited by activation of Akt by environmental stimuli.10,11 Drug\nresistance might involve resistance to autophagic cell death, as well as\napoptosis. In fact, treatment with tamoxifen causes autophagy of\nMCF-7 breast carcinoma cells, as well as apoptotic cell death via a\ncaspase 3-independent pathway.12 Autophagy is also induced by au-\nrintricarboxylic acid via activation of the ERK1/2 signaling pathway in\nRas-mutated HT-29 colon carcinoma cells.13 In addition, a human\ntumor suppressor gene, known as Beclin 1, induces autophagy.14,15\nCANCER April 15, 2005 / Volume 103 / Number 8\nReduced expression of this gene occurs in breast and\novarian carcinoma due to a 40 –75% deletion of the\ngene.16 Although resistance to autophagy and apopto-\nsis is likely involved in tumor progression and acqui-\nsition of drug resistance by solid tumors, the exact\nmolecular mechanism(s) by which this occurs, and the\nrelation between autophagy and apoptosis, remain\nunknown. Here, we review evidence regarding the\nmolecular mechanism(s) of anticancer drug-induced\ncell death and discuss potential therapeutic targets by\nwhich signal transduction pathways leading to cell\ndeath might be modulated by cancer therapy.",
         "APOPTOTIC PATHWAYS OF CELL DEATH\nCaspase-Dependent Pathway\nGiven that a number of anticancer agents induce ap-\noptosis, a lot of research has focused on the various\npathways that mediate apoptosis. Two are known to\nmediate anticancer drug-induced apoptosis, a death\nreceptor-dependent (extrinsic) and a mitochondria-\ndependent (intrinsic) pathway.17 The death receptor-\ndependent pathway involves activation of death re-\nceptors, such as Fas and TRAIL receptors (DR4, DR5).\nDeath receptor activation is mediated by a death-\ninducing signaling complex, which is formed by re-\ncruitment of a Fas-associated death domain and pro-\ncaspase 8 to the death receptor, thereby activating\ncaspase 8.3,18 Caspase 8 directly activates caspase 3,\nleading to apoptosis (type I cell death).19 The mito-\nchondria-dependent pathway involves cleavage of a\nproapoptotic protein, Bid, resulting in the production\nof truncated Bid (tBid), by caspase 8, within the mito-\nchondria. Heterodimerization of tBid and Bak results\nin release of cytochrome c from mitochondria and\nactivation of caspase 3 and caspase 9 (type II cell\ndeath).19 The predominance of one pathway over an-\nother depends on cell type.\nUnlike the death receptor-dependent pathway,\nthe mitochondria-dependent pathway is mediated by\nBax/Bak and involves the release of cytochrome c.3,18\nIn general, Bax is associated with 14-3-3 protein,\nwhich anchors Bax in the cytoplasm. However, in the\npresence of DNA damage, c-Jun NH(2)-terminal ki-\nnase phosphorylates 14-3-3, resulting in dissociation\nof Bax from this protein.20 Homodimerization of Bax,\nor heterodimerization of Bax ank Bak, results in trans-\nlocation of Bax from the cytoplasm to mitochondria.21\nBax homodimers and heterodimers interact with a\nvoltage-dependent anion channel (VDAC) in the mi-\ntochondrial outer membrane to release cytochrome c\nby increasing mitochondrial membrane permeability\n(MMP) via opening of the mitochondrial permeability\ntransition pore.22 The mitochondrial permeability\ntransition pore complex (PTPC) mediates permeability\nof the mitochondrial membrane during apoptosis, ne-\ncrosis, and autophagy. PTPC is composed of an ade-\nnine nucleotide translocator and VDAC, both of which\nare targeted by a variety of proapoptotic inducers. The\nrelease of cytochrome c results in the formation of an\napoptosome, consisting of apoptosis activating factor\n1 and procaspase 9, after which dATP activates\ncaspase 9, leading to apoptosis.23 The activated initi-\nator caspase, caspase 9, leads to activation of one of\nthe effector caspases, caspase 3 or caspase 7, which\ncleaves an inhibitor of caspase-activated DNAase and\nproduces poly (ADP-ribose) polymerase (PARP), re-\nsulting in DNA fragmentation. The release of cyto-\nchrome c through the VDAC-mediated permeability\ntransition pore is inhibited by Bcl-2 and Bcl-xL. In\naddition to cytochrome c, Smac/Diablo is released\nfrom mitochondria when there is a loss of membrane\npotential. Smac/Diablo binds to X chromosome-en-\ncoded IAP, cellular IAP-1 (cIAP-1), and cellular IAP-2,\nas well as survivin, to inhibit apoptosis,24 and it di-\nrectly activates caspase 3. Inhibitors of apoptosis pro-\nteins (IAPs) block apoptosis by binding to and inhib-\niting caspases, as well as by activating caspase-\nindependent mechanisms.25\ninhibit\nactivation of executioner caspases activated by extrin-\nsic or intrinsic pathways. The Smac/Diablo complex\ninduces apoptosis via the apoptosome-dependent\npathway,26 and release of Smac/Diablo is inhibited by\nBcl-2 and Bcl-xL.27 There is also a report that Smac\ninduces cytochrome c release and apoptosis in the\nabsence of Bax/Bcl-xL via activation of caspase 3 in\nhuman HCT116 and DU145 colon carcinoma cells.28\nSmac is capable of circumventing defects in mito-\nchondrial apoptosis signaling, such as loss of Bax or\noverexpression of Bcl-xL, which frequently occurs in\ntumor cells resistant to anticancer therapy. A recent\nstudy suggests that an IAP family protein, Apollon,\nbinds to, ubiquitinates, and facilitates the proteasomal\ndegradation of Smac and caspase 9, thereby prevent-\ning Smac-induced apoptosis.29 The role of Apollon in\ndrug resistance remains to be clariﬁed. Translocation\nof endogenous Smac into the cytosol, and release of\nSmac/Diablo during anticancer drug-induced apopto-\nsis, does not appear to play a major role in cell death\nafter treatment of human lung carcinoma with etopo-\nside,30 because cytochrome c and mitochondrial pro-\ntease Omi/HtrA2 are still detectable in the cytosol in\nthe absence of Smac. Thus, the signiﬁcance of Smac/\nDiablo in anticancer drug-induced mitochondrial\npathway-mediated apoptosis remains unclear.\ncan\nCaspase-Independent Pathway\nAlthough caspase cysteine proteases execute apopto-\nsis, a mitochondrial serine protease released into the\nCell Death with Chemotherapy/Kim\ncytosol during apoptosis, Omi/HtrA2, is required to\nantagonize inhibitors of apoptosis (IAPs), thereby en-\nabling apoptosis to occur via an indirect mechanism.31\nCleavage of c-IAP-1 by Smac/Diablo and Omi/HtrA2 is\nirreversible, thus, signiﬁcant inactivation of IAPs oc-\ncurs, enabling increased caspase activity after tumor\nnecrosis factor apoptosis related ligand (TRAIL)-in-\nduced apoptosis.32 IAP cleavage by Omi is indepen-\ndent of caspase activity. In addition, Omi/HtrA2 might\nactivate more than one pathway of caspase activation\nbecause extramitochondrial expression of Omi-HtrA2\nresults in increased mitochondrial permeability and\ncytochrome c-induced caspase activation in HeLa\ncells.33\nApoptosis-inducing factor (AIF) is a phylogeneti-\ncally conserved ﬂavoprotein within the mitochondrial\nmembrane, which has the ability to induce apoptosis\nvia a caspase-independent pathway.34 AIF induces nu-\nclear chromatin condensation and large-scale DNA\nfragmentation (producing approximately 50-kilobase\n[kb] fragments), and is essential for programmed cell\ndeath. On lethal signaling, AIF translocates through\nthe cytosol to the nucleus where it binds to DNA and\nprovokes caspase-independent chromatin condensa-\ntion. Responses to injection of anti-AIF antibodies, or\nknockout of the AIF gene, suggest that AIF may be\nrequired for cell death in response to certain stimuli.35\nStaurosporine (STS) induces mitochondrial dysfunc-\ntion and translocation of AIF into the nucleus after\nactivation of nuclear apoptosis in nonsmall cell lung\ncarcinoma cells resistant to radiotherapy or chemo-\ntherapy.36 This suggests that, although caspase-de-\npendent and independent pathways coexist,\nthe\ncaspase-dependent pathway might be less efﬁcient.\nFurthermore, resistant cells might be made more sus-\nceptible to treatment by activation of the AIF-medi-\nated caspase-independent pathway. Moreover, recent\nreports indicate that AIF is necessary for PARP-1– de-\npendent cell death.37,38 PARP-1 generates a number of\nlong, branched PARPS after DNA damage. Activation\nof PARP-1 initiates a nuclear signal that triggers the\nrelease of AIF from mitochondria. AIF then shuttles\nfrom mitochondria to the nucleus and induces periph-\neral chromatin condensation and large-scale DNA\nfragmentation.38 Endonuclease G (Endo G) is a nucle-\nar-encoded mitochondrial protein believed to be im-\nportant for nuclear DNA fragmentation during apo-\nptosis, as well as mitochondrial DNA replication. Cells\nfrom Endo G heterozygous mutant mice are resistant\nto tumor necrosis factor alpha and STS-induced cell\ndeath.39\nThus, increased MMP in response to DNA damage\nresults in the release of a number of small molecules,\nsuch as cytochrome c, Smac/Diablo, and AIF from\nmitochondria. This results in activation of caspase-\ndependent and independent apoptotic pathways, de-\npending on which triggers and cell types are involved.\nCANCER April 15, 2005 / Volume 103 / Number 8\nOur current understanding regarding the role of mi-\ntochondria in cell death is summarized in Figure 1.\nAUTOPHAGIC PATHWAY OF CELL DEATH\nAutophagic cell death, otherwise known as type II cell\ndeath, is characterized by the appearance of large auto-\nphagic vacuoles in the cytoplasm. It occurs during em-\nbryogenesis and differs from type I programmed cell\ndeath.40 Autophagy is a bulk protein degradation system\nthat is essential for normal cell activity and survival when\nnutrients are scarce.41 The ﬁrst step that occurs in auto-\nphagic cell death is the formation of a double-mem-\nbrane vacuole, the autophagosome, which is derived\nfrom part of the endoplasmic reticulum42 or from the\ncytoplasmic lipid pool.43 The autophagosome fuses with\na lysosome, after which its contents are degraded by\nlysosomal hydrolytic proteases. Despite recent advances\nregarding the molecular mechanism(s) underlying auto-\nphagy, many details remain poorly understood. Of note,\nclass I phosphoinositide 3-kinase (PI 3-kinase) inhibits,\nwhereas Class III PI 3-kinase enhances, autophagy in\nHT-29 human colon carcinoma cells.44 Class I PI 3-ki-\nnase mediates various cellular functions, including insu-\nlin and glycogen synthesis, and suppresses apoptosis\nthrough the Akt-signaling pathway, whereas Class III PI\n3-kinase inﬂuences membrane trafﬁcking. Despite the\nﬁnding that insulin-like growth factor I (IGF-I) normally\ninhibits apoptosis, IGF-I accelerates cell death when glu-\ncose and amino acids are limited, leading to accumula-\ntion of autophagic vacuoles within the cytoplasm and\nevidence of chromosome condensation.40 Autophagic\ncell death is inhibited by 3-methyladenine (3-MA) and\nLY294002, which inhibit PI 3-kinase, but not by the pan-\ncaspase inhibitor, z-VAD-fmk. Of importance, LY294002\nhas no effect on cell death in PC12 and HepG2 carci-\nnoma cells during glucose deprivation,45 suggesting that\ninterference with the PI 3-kinase/Akt signaling pathway\ncauses autophagy of cancer cells but not normal cells.\nA mammalian gene capable of inducing autoph-\nagy, Beclin 1, is a component of the PI 3-kinase com-\nplex responsible for autophagic vesicle nucleation.16\nThis gene is deleted in 75% of ovarian, 50% of breast,\nand 40% of prostate carcinoma cases.16 Frequent al-\nlelic deletions are observed in breast carcinoma cell\nlines in association with signiﬁcant decreases in Beclin\n1 protein levels. Beclin 1 functions as a tumor sup-\npressor, because there is an attenuation of MCF-7–\nmediated tumorigenicity in nude mice overexpressing\nBeclin 1.46 Although Beclin 1 is known to interact with\nBcl-2 in yeast, the relation between this association\nand tumor suppression and autophagy is not yet clear.\nDownregulation of Bcl-2 by antisense Bcl-2 in HL-60\ncells induces autophagy independent of caspase-de-\npendent and mitochondria-dependent pathways,47\nsuggesting a potential role of Bcl-2 in autophagic cell\ndeath, as well as apoptosis. Other autophagy-inducing\ngenes, including bridging integrator 1 (Bin 1) and\ndeath-associated protein kinase (DAPK), exist. Bin 1\ninteracts with c-Myc and inhibits its transformation\nactivity.48 Bin 1 is regulated by alternate splicing, and\nthe splice forms that interact with c-Myc have tumor\nsuppressor properties. This gene is frequently missing\nor functionally inactivated in malignant melanoma,\nbreast, and prostate carcinoma cells, and loss of Bin 1\nexpression may promote tumor progression, while\nlimiting susceptibility to farnesyltransferase inhibitors\n(FTIs). Bin 1 adapter proteins act downstream or par-\nallel to RhoB to exert their effect on cell signaling.49\nDAPK is a calcium-regulated serine/threonine kinase,\nand an important mediator of autophagy induced by\nnutrient deprivation, antiestrogens, and tumor necro-\nsis factor.50 Cancer and cancer cell lines frequently\ndemonstrate decreased DAPK expression due to DNA\nmethylation.51 Transfection of Beclin 1, Bin 1, or an\nactive form of DAPK, into cancer cells induces auto-\nphagic cell death. A human homolog of the Drosophila\nspin gene product (HSpin 1) exists, which binds to\nBcl-2 and Bcl-xL. HSpin 1-induced cell death results in\nformation of autophagic vacuoles and release of a\nmature form of cathepsin D, suggesting a novel\ncaspase-independent cell death, pathway resulting in\nautophagy.52\nThe tumor suppressor phosphatase and tensin\nhomolog deleted on chromosome10 (PTEN), which is\nan inhibitor of the PI 3-kinase/Akt signaling pathway,\npromotes autophagy in HT-29 colon carcinoma\ncells.53 PTEN PI phosphatase activity inhibits Akt ac-\ntivity, resulting in induction of autophagy. Loss of\nfunction of PTEN increases the expression of Class I PI\n3-kinase and Akt in human cancer cells. Activation of\nAkt and constitutive expression of active Akt block\nautophagic cell death, leading to inhibition of apopto-\nsis and activation of a mammalian target of rapamycin\n(mTor) protein. It is likely that activation of Akt and\nthe mTor signaling pathway, due to loss of PTEN,\ncontributes to malignant transformation by simulta-\nneous inhibition of autophagic and apoptotic cell\ndeath. Another oncoprotein, Ha-Ras, induces autoph-\nagic cell death in glioma and gastric carcinoma cells54\nvia a p53-independent pathway that is not blocked by\nBcl-2. Another Ras family, including K-Ras and N-Ras\nproteins, also induces autophagic cell death.\nIn-\ncreased Ras expression is observed in spontaneous\nregression of neuroblastoma, which is characterized\nby autophagic cell death and a lack of apoptosis.55 In\naddition, Ha-Ras overexpression induces autophagic\ncell death in neuroblastoma cell\nlines. Ha-Ras–in-\nduced autophagy in HT-29 colon carcinoma cells has\nCell Death with Chemotherapy/Kim\nsome overlap with the Raf 1/ERK signaling pathway of\nautophagic cell death.56 The ability of Ras to cause\nautophagy might involve activation of ErbB2 and\nErbB3. It is interesting to note that one of the signaling\npathways regulated by Ras, the Raf1/ERK pathway,\ntriggers autophagy, whereas the PI 3-kinase/Akt path-\nway, which is downstream of Ras signaling, inhibits it.\nRas-induced increases in the expression and trafﬁck-\ning of lysosomal cathepsins might contribute to Ras-\ninduced autophagy. A proposed molecular model of\nautophagy is illustrated in Figure 2.\nA death-promoting “BH3-only” family protein, Bcl-2/\nadenovirus E1B 19-kilodalton interacting protein 3\n(BNIP3), differs from other proteins of this class.\nBNIP3 can directly integrate into the mitochondrial\nouter membrane via its BH3 domain in the absence of\nBax/Bak, thereby disrupting the MMP, generating re-\nactive oxygen species (ROS) and promoting autoph-\nagic degeneration.57 Overexpression of BNIP3, a hy-\npoxia-inducible protein with a BH3 domain, results in\nautophagic cell death in cancer cell lines, after cyto-\nchrome c or AIF nuclear translocation. BNIP3-induced\nautophagy can be attenuated by overexpression of Bcl-2,\nwhich interacts with BNIP3 protein. Autophagic cell\ndeath can be activated in cancer cells in response to\nvarious anticancer drugs. Treatment of MCF-7 breast\ncarcinoma cells with tamoxifen induces autophagy be-\nfore apoptotic cell death.12 Treatment with estradiol and\n3-MA inhibits tamoxifen-induced cell death. Similarly,\nother anticancer drugs, such as vinblastin, paclitaxel,\ninduce autophagic cell death.58 – 60\nand rapamycin,\nTreatment of malignant glioma cells with arsenic triox-\nide induces autophagic cell death in association with\nG2/M arrest.61 In addition, radiotherapy induces auto-\nphagic cell death in several cancer cell lines, including\nbreast, colon, and prostate carcinoma, as well as glio-\nblastoma multiforme.62 Despite several reports of induc-\ntion of autophagy by treatment with anticancer drugs\nand/or irradiation, the clinical signiﬁcance of autophagy\nas a result of chemotherapy remains unclear.\nINTERACTION BETWEEN APOPTOTIC AND\nAUTOPHAGIC PATHWAYS OF CELL DEATH\nDifferent types of cell death differ in their morphologic\ncharacteristics (Fig. 3 and Table 1). In fact, ceramide,\nwhich is believed to activate apoptosis, also induces\nCANCER April 15, 2005 / Volume 103 / Number 8\nChromatin condensation\nDNA laddering and fragmentation\nPyknosis\nCytoplasmic condensation\nFragmentation to apoptotic bodies\nIncrease in MMP\nActivation of caspase cascade\nPotential release of lysosomal enzyme\nOrganelles do not swell\nBlebbing\nCaspases such as caspase 3\nYes\nz-VAD-fmk\nXIAP\nBcl-2/Bcl-xL\nSometimes with actinomycin D\nSometimes with cyclophosphamide\nDNA laddering test\nCaspase activation and the substrate\nTUNEL and annexin V staining\nFACS analysis\nElectron microscopy\nautophagy.63 Similarly, proteins like DAPK are capable\nof inducing both apoptosis and autophagy depending\non the cell type involved, suggesting that they may\nfunction as molecular switches or integrators of these\ntwo types of programmed cell death. Commitment to\nautophagy or apoptosis may depend on the cell type\ninvolved, as well as the microenvironment. Depending\non the setting, initiation of autophagy may induce,\ndelay, or antagonize apoptosis. Three scenarios must\nbe considered when contemplating the interaction be-\ntween apoptosis and autophagic cell death.\nFirst, autophagy may precede apoptosis. Treat-\nment of sympathetic neurons with cytosine arabino-\nside, thereby depleting neural growth factor, results in\na 30-fold increase in autophagy, followed by DNA\ndegradation and apoptosis.64 Delayed induction of ap-\noptosis occurs with 3-MA, which inhibits cytochrome\nc release and caspase activation. These ﬁndings may\nmodel cell death after treatment with anticancer drugs\nor growth factor deprivation.\nSecond, autophagy may antagonize apoptosis.\nConversely, a reduction in autophagy might increase\nthe susceptibility of cells to apoptotic stimuli. Accel-\nerated sulindac sulﬁde-induced apoptosis occurs in\nmutant clone HT-29 colon carcinoma cells, which lack\nautophagic capability due to overexpression of a mu-\ntant G (alpha i3) protein.65 In contrast, treatment of\nparent HT-29 cells with 3-MA increases sensitivity to\nsulindac sulﬁde-induced apoptosis. Even though\nthere are no differences in the expression patterns of\nCOX-2, Bcl-2, Bcl-xL, Bax, and Akt/PKB among mutant\nand parent HT-29 cells, greater cytochrome c release\noccurs in mutant G (alpha i3) protein overexpressing\ncells, compared with parent cells. It is possible that\nautophagy retards apoptosis in colon carcinoma cells\nby sequestering mitochondrial death-promoting fac-\ntors, such as cytochrome c.\nThird, apoptosis and autophagy may be mutually\nexclusive. Inhibition of autophagy may lead to apo-\nptotic cell death. A recent report reveals that treat-\nment of malignant glioma cells with arsenic trioxide\ninduces autophagic cell death. However, inhibition of\nautophagy by treatment with baﬁlomycin, a H-AT-\nleads to apoptosis.61 However, al-\nPase inhibitor,\nthough treatment of neurons and HeLa cells with a\npancaspase inhibitor prevents apoptosis, selective mi-\nimpairment results in autophagy.66 In\ntochondrial\nlight of several reports indicating that treatment with\ncaspase inhibitors causes a shift from apoptotic cell\ndeath to autophagy, it is possible that this also occurs\nunder other circumstances. For example, inhibition of\napoptosis in Bax/Bak double-deﬁcient DKO cells in-\nduces autophagic cell death.67 The ability of cells to\nrevert between apoptosis and autophagy might be\nclinically signiﬁcant because drug resistance to apo-\nptosis might be overcome by activation of autophagic\ncell death. Along this line, multidrug-resistant MCF-7\ncells can be made susceptible to doxorubicin by treat-\nment with verapamil, after which autophagic cell\ndeath occurs, suggesting that the autophagic cell\ndeath pathway remains viable after inhibition of the\napoptotic pathway (unpublished data). The molecular\nCell Death with Chemotherapy/Kim\nmechanism(s) by which reversion from apoptosis to\nautophagy occurs remains unclear. Although overex-\npression of Bcl-2 or Bcl-xL may inhibit autophagic cell\ndeath, the role of these antiapoptotic proteins in au-\ntophagic cell death requires further clariﬁcation.\nThe role of Bcl-2 in apoptosis in response to mi-\ntochondrial damage is well established. However, sig-\nnaling as a result of mitochondrial damage might also\nactivate the autophagic pathway of cell death. It ap-\npears that mitochondria are involved in integrating\nthe apoptotic and autophagic pathways of cell death.\nLysosomal enzymes might activate apoptosis because\nlysosome rupture activates apoptosis. It is interesting\nto note that apoptosis after lysosome rupture is pre-\nceded by autophagy. Sequestration of mitochondria\nwithin autophagic vacuoles might protect cells from\napoptosis by delaying the release of cytochrome c.\nThus, autophagy might regulate apoptosis through ly-\nsosome formation and enzyme activity. With regard to\nthe possibility of signaling between lysosomes and\nmitochondria, a recent study reveals that production\nof H2O2 from ROS in response to DNA damage acti-\nvates lysosomal enzymes, such as cathepsin D and B.68\nCathepsin B directly or indirectly results in transloca-\ntion of proapoptotic proteins, such as Bid and Bax,\nfrom the cytoplasm to mitochondria. This alters MMP\nand results in caspase activation subsequent to the\nrelease of cytochrome c, Smac/Diablo, and AIF from\nthe mitochondria. Conversely, under different circum-\nstances, production of H2O2 from ROS in response to\nDNA damage may inhibit the release of small mole-\ncules, such as cytochrome c, Smac/Diablo, and AIF,\nfrom the mitochondrial membrane and activate lyso-\nsomal activity. Thus, crosstalk might occur between\nthe pathways mediating autophagy and apoptosis by\nway of lysosomes and mitochondria. Further research\nis required to clarify the signaling that occurs between\nthese pathways.\nFUTURE PERSPECTIVES REGARDING CELL DEATH\nAND CANCER THERAPY\nIncreasing information is known regarding the molec-\nular mechanism(s) by which cell death is mediated by\nanticancer drugs. Proapoptotic and antiapoptotic pro-\nteins are now being targeted to enhance the effect(s)\nof chemotherapy. In general, the antitumor effect of\nan anticancer agent is measured in terms of its ability\nto induce apoptotic cell death within tumors, which is\nwell correlated with outcome in most preclinical and\nclinical studies. Nevertheless, despite signiﬁcant re-\nductions in tumor volume, high rates of apoptosis do\nnot always occur within solid tumors after chemother-\napy. This suggests that other types of cell death, such\nas autophagy and necrosis, might also occur as a result\nCANCER April 15, 2005 / Volume 103 / Number 8\nof chemotherapy. Research into other types of cell\ndeath, particularly autophagic cell death, is important\nto better understand the effects of anticancer drugs. In\naddition, the relation between autophagic cell death\nand apoptosis needs to be examined, because several\nautophagy-related proteins, including Beclin 1, BNIP3,\nDAPK, and HSpin 1, may inﬂuence apoptosis. Because\napoptosis and autophagy are mutually exclusive in\nsome cell lines, molecular switches between the two\nprocesses must exist.\nResistance to autophagy, as well as apoptosis,\nmight lend resistance to anticancer drugs. Likewise,\ninduction of autophagic cell death in cancer cells may\nbe of therapeutic beneﬁt. Cells transformed with the\ntumor suppressor gene Beclin 1 are less likely to un-\ndergo autophagy and are more susceptible to onco-\ngenesis. Alterations in autophagic genes reduce the\nincidence of autophagy in cancer cells. However, even\nin cells resistant to apoptosis or cells in which apo-\nptosis is inhibited, nonapoptotic cell death pathways,\nsuch as autophagy, may be activated. Targeting alter-\nnative pathways of cell death might overcome drug\nresistance or explain why the effects of chemotherapy\nand irradiation complement each other in the clinical\nsetting. Alterations or defects in apoptotic signal\ntransduction pathways might result in drug resistance,\nhowever, irradiation induces autophagic cell death.\nAlterations of one or both pathways (autophagic or\napoptotic) might determine susceptibility to cell death\nafter chemotherapy because anticancer drugs might\ntrigger both apoptotic caspase-dependent and auto-\nphagic caspase-independent pathways at the same\ntime.\nAutophagic cell death might also inhibit angio-\ngenesis.69 The angiogenesis inhibitor, endostatin, in-\nduces endothelial cell death via the caspase-indepen-\ndent autophagic pathway. In addition, the tumor\nsuppressor gene, PTEN, which induces autophagy, in-\nhibits tumor angiogenesis in nude mice with ortho-\ntopic brain tumors.70 Reconstitution of wild-type\nPTEN has no effect on in vitro proliferation, but dra-\nmatically decreases tumor growth in vivo and pro-\nlongs survival in mice, suggesting that PTEN regulates\ntumor-induced angiogenesis via regulation of PI-de-\npendent signals. The role of autophagy in tumor an-\ngiogenesis might be an important means of inhibiting\ntumor progression and increasing the effectiveness of\nanticancer therapy.\nIt is now clear that a number of different types of\ncell death, including apoptosis and autophagy, play a\nrole in the antitumor effect(s) of treatment with anti-\ncancer drugs. Targeting therapy at genes related to\napoptosis might increase the therapeutic efﬁcacy of\nanticancer agents by modulating signal transduction\npathways, and other cell death pathways might be\ninﬂuenced by changes in the microenvironment and\nby crosstalk between the various cell death pathways.\nFurther research into the molecular mechanism(s) be-\nhind autophagic cell death as a result of chemother-\napy might provide a new strategy for overcoming drug\nresistance by exploring this alternative pathway of cell\ndeath.\n28. Hasenjager A, Gillissen B, Muller A, et al. Smac induces\ncytochrome c release and apoptosis independently from\nBax/Bcl-x(L) in a strictly caspase-3-dependent manner in\nhuman carcinoma cells. Oncogene. 2004;23:4523– 4535.\n29. Hao Y, Sekine K, Kawabata A, et al. Apollon ubiquitinates\nSMAC and caspase-9, and has an essential cytoprotection\nfunction. Nat Cell Biol. 2004;6:849 – 860.\nCell Death with Chemotherapy/Kim\n40. Schweichel JU, Merker HJ. The morphology of various types\nof cell death in prenatal tissues. Teratology. 1973;7:253–266.\n41. Mizushima N, Yamamoto A, Matsui M, Yoshimori T, Oh-\nsumi Y. In vivo analysis of autophagy in response to nutrient\nstarvation using transgenic mice expressing a ﬂuorescent\nautophagosome marker. Mol Biol Cell. 2004;15:1101–1111.\n42. Dunn WA Jr. Studies on the mechanisms of autophagy:\nformation of the autophagic vacuole. J Cell Biol. 1990;110:\n1923–1933.\nCANCER April 15, 2005 / Volume 103 / Number 8",
         "",
         "",
         "",
         "plays an important role in cellular homeostasis.1 Three types of\ncell death are known: apoptosis (type I), autophagy (type II), and\nnecrosis (type III).2 Apoptosis is induced by a number of stimuli,\nincluding anticancer drugs, growth factor deprivation, and irradia-\ntion. The molecular mechanism by which apoptosis occurs has been\nextensively examined. Apoptosis in response to chemotherapy is\nknown to correlate with a reduction in tumor volume, and resistance\nto apoptosis causes drug resistance.3–5 Introduction of a proapoptotic\ngene such as Bax,6,7 and inhibition of an antiapoptotic gene such as\nBcl-2,8,9 enhances the effects of chemotherapy. Thus, apoptosis\nclearly inﬂuences the antitumor effect(s) of chemotherapeutic agents.\nIn solid tumors, hypoxia and glucose deprivation enable tumor\nprogression,10 whereas depletion of adenosine 5-triphosphate (ATP)\nresults in necrosis. The production of lactic acid under anaerobic\nconditions does not necessarily induce necrosis, rather, autophagic\ncell death may depend on susceptibility to apoptosis, which can be\ninhibited by activation of Akt by environmental stimuli.10,11 Drug\nresistance might involve resistance to autophagic cell death, as well as\napoptosis. In fact, treatment with tamoxifen causes autophagy of\nMCF-7 breast carcinoma cells, as well as apoptotic cell death via a\ncaspase 3-independent pathway.12 Autophagy is also induced by au-\nrintricarboxylic acid via activation of the ERK1/2 signaling pathway in\nRas-mutated HT-29 colon carcinoma cells.13 In addition, a human\ntumor suppressor gene, known as Beclin 1, induces autophagy.14,15\nCANCER April 15, 2005 / Volume 103 / Number 8\nReduced expression of this gene occurs in breast and\novarian carcinoma due to a 40 –75% deletion of the\ngene.16 Although resistance to autophagy and apopto-\nsis is likely involved in tumor progression and acqui-\nsition of drug resistance by solid tumors, the exact\nmolecular mechanism(s) by which this occurs, and the\nrelation between autophagy and apoptosis, remain\nunknown. Here, we review evidence regarding the\nmolecular mechanism(s) of anticancer drug-induced\ncell death and discuss potential therapeutic targets by\nwhich signal transduction pathways leading to cell\ndeath might be modulated by cancer therapy.\nAPOPTOTIC PATHWAYS OF CELL DEATH\nCaspase-Dependent Pathway\nGiven that a number of anticancer agents induce ap-\noptosis, a lot of research has focused on the various\npathways that mediate apoptosis. Two are known to\nmediate anticancer drug-induced apoptosis, a death\nreceptor-dependent (extrinsic) and a mitochondria-\ndependent (intrinsic) pathway.17 The death receptor-\ndependent pathway involves activation of death re-\nceptors, such as Fas and TRAIL receptors (DR4, DR5).\nDeath receptor activation is mediated by a death-\ninducing signaling complex, which is formed by re-\ncruitment of a Fas-associated death domain and pro-\ncaspase 8 to the death receptor, thereby activating\ncaspase 8.3,18 Caspase 8 directly activates caspase 3,\nleading to apoptosis (type I cell death).19 The mito-\nchondria-dependent pathway involves cleavage of a\nproapoptotic protein, Bid, resulting in the production\nof truncated Bid (tBid), by caspase 8, within the mito-\nchondria. Heterodimerization of tBid and Bak results\nin release of cytochrome c from mitochondria and\nactivation of caspase 3 and caspase 9 (type II cell\ndeath).19 The predominance of one pathway over an-\nother depends on cell type.\nUnlike the death receptor-dependent pathway,\nthe mitochondria-dependent pathway is mediated by\nBax/Bak and involves the release of cytochrome c.3,18\nIn general, Bax is associated with 14-3-3 protein,\nwhich anchors Bax in the cytoplasm. However, in the\npresence of DNA damage, c-Jun NH(2)-terminal ki-\nnase phosphorylates 14-3-3, resulting in dissociation\nof Bax from this protein.20 Homodimerization of Bax,\nor heterodimerization of Bax ank Bak, results in trans-\nlocation of Bax from the cytoplasm to mitochondria.21\nBax homodimers and heterodimers interact with a\nvoltage-dependent anion channel (VDAC) in the mi-\ntochondrial outer membrane to release cytochrome c\nby increasing mitochondrial membrane permeability\n(MMP) via opening of the mitochondrial permeability\ntransition pore.22 The mitochondrial permeability\ntransition pore complex (PTPC) mediates permeability\nof the mitochondrial membrane during apoptosis, ne-\ncrosis, and autophagy. PTPC is composed of an ade-\nnine nucleotide translocator and VDAC, both of which\nare targeted by a variety of proapoptotic inducers. The\nrelease of cytochrome c results in the formation of an\napoptosome, consisting of apoptosis activating factor\n1 and procaspase 9, after which dATP activates\ncaspase 9, leading to apoptosis.23 The activated initi-\nator caspase, caspase 9, leads to activation of one of\nthe effector caspases, caspase 3 or caspase 7, which\ncleaves an inhibitor of caspase-activated DNAase and\nproduces poly (ADP-ribose) polymerase (PARP), re-\nsulting in DNA fragmentation. The release of cyto-\nchrome c through the VDAC-mediated permeability\ntransition pore is inhibited by Bcl-2 and Bcl-xL. In\naddition to cytochrome c, Smac/Diablo is released\nfrom mitochondria when there is a loss of membrane\npotential. Smac/Diablo binds to X chromosome-en-\ncoded IAP, cellular IAP-1 (cIAP-1), and cellular IAP-2,\nas well as survivin, to inhibit apoptosis,24 and it di-\nrectly activates caspase 3. Inhibitors of apoptosis pro-\nteins (IAPs) block apoptosis by binding to and inhib-\niting caspases, as well as by activating caspase-\nindependent mechanisms.25\ninhibit\nactivation of executioner caspases activated by extrin-\nsic or intrinsic pathways. The Smac/Diablo complex\ninduces apoptosis via the apoptosome-dependent\npathway,26 and release of Smac/Diablo is inhibited by\nBcl-2 and Bcl-xL.27 There is also a report that Smac\ninduces cytochrome c release and apoptosis in the\nabsence of Bax/Bcl-xL via activation of caspase 3 in\nhuman HCT116 and DU145 colon carcinoma cells.28\nSmac is capable of circumventing defects in mito-\nchondrial apoptosis signaling, such as loss of Bax or\noverexpression of Bcl-xL, which frequently occurs in\ntumor cells resistant to anticancer therapy. A recent\nstudy suggests that an IAP family protein, Apollon,\nbinds to, ubiquitinates, and facilitates the proteasomal\ndegradation of Smac and caspase 9, thereby prevent-\ning Smac-induced apoptosis.29 The role of Apollon in\ndrug resistance remains to be clariﬁed. Translocation\nof endogenous Smac into the cytosol, and release of\nSmac/Diablo during anticancer drug-induced apopto-\nsis, does not appear to play a major role in cell death\nafter treatment of human lung carcinoma with etopo-\nside,30 because cytochrome c and mitochondrial pro-\ntease Omi/HtrA2 are still detectable in the cytosol in\nthe absence of Smac. Thus, the signiﬁcance of Smac/\nDiablo in anticancer drug-induced mitochondrial\npathway-mediated apoptosis remains unclear.\ncan\nCaspase-Independent Pathway\nAlthough caspase cysteine proteases execute apopto-\nsis, a mitochondrial serine protease released into the\nCell Death with Chemotherapy/Kim\ncytosol during apoptosis, Omi/HtrA2, is required to\nantagonize inhibitors of apoptosis (IAPs), thereby en-\nabling apoptosis to occur via an indirect mechanism.31\nCleavage of c-IAP-1 by Smac/Diablo and Omi/HtrA2 is\nirreversible, thus, signiﬁcant inactivation of IAPs oc-\ncurs, enabling increased caspase activity after tumor\nnecrosis factor apoptosis related ligand (TRAIL)-in-\nduced apoptosis.32 IAP cleavage by Omi is indepen-\ndent of caspase activity. In addition, Omi/HtrA2 might\nactivate more than one pathway of caspase activation\nbecause extramitochondrial expression of Omi-HtrA2\nresults in increased mitochondrial permeability and\ncytochrome c-induced caspase activation in HeLa\ncells.33\nApoptosis-inducing factor (AIF) is a phylogeneti-\ncally conserved ﬂavoprotein within the mitochondrial\nmembrane, which has the ability to induce apoptosis\nvia a caspase-independent pathway.34 AIF induces nu-\nclear chromatin condensation and large-scale DNA\nfragmentation (producing approximately 50-kilobase\n[kb] fragments), and is essential for programmed cell\ndeath. On lethal signaling, AIF translocates through\nthe cytosol to the nucleus where it binds to DNA and\nprovokes caspase-independent chromatin condensa-\ntion. Responses to injection of anti-AIF antibodies, or\nknockout of the AIF gene, suggest that AIF may be\nrequired for cell death in response to certain stimuli.35\nStaurosporine (STS) induces mitochondrial dysfunc-\ntion and translocation of AIF into the nucleus after\nactivation of nuclear apoptosis in nonsmall cell lung\ncarcinoma cells resistant to radiotherapy or chemo-\ntherapy.36 This suggests that, although caspase-de-\npendent and independent pathways coexist,\nthe\ncaspase-dependent pathway might be less efﬁcient.\nFurthermore, resistant cells might be made more sus-\nceptible to treatment by activation of the AIF-medi-\nated caspase-independent pathway. Moreover, recent\nreports indicate that AIF is necessary for PARP-1– de-\npendent cell death.37,38 PARP-1 generates a number of\nlong, branched PARPS after DNA damage. Activation\nof PARP-1 initiates a nuclear signal that triggers the\nrelease of AIF from mitochondria. AIF then shuttles\nfrom mitochondria to the nucleus and induces periph-\neral chromatin condensation and large-scale DNA\nfragmentation.38 Endonuclease G (Endo G) is a nucle-\nar-encoded mitochondrial protein believed to be im-\nportant for nuclear DNA fragmentation during apo-\nptosis, as well as mitochondrial DNA replication. Cells\nfrom Endo G heterozygous mutant mice are resistant\nto tumor necrosis factor alpha and STS-induced cell\ndeath.39\nThus, increased MMP in response to DNA damage\nresults in the release of a number of small molecules,\nsuch as cytochrome c, Smac/Diablo, and AIF from\nmitochondria. This results in activation of caspase-\ndependent and independent apoptotic pathways, de-\npending on which triggers and cell types are involved.\nCANCER April 15, 2005 / Volume 103 / Number 8\nOur current understanding regarding the role of mi-\ntochondria in cell death is summarized in Figure 1.\nAUTOPHAGIC PATHWAY OF CELL DEATH\nAutophagic cell death, otherwise known as type II cell\ndeath, is characterized by the appearance of large auto-\nphagic vacuoles in the cytoplasm. It occurs during em-\nbryogenesis and differs from type I programmed cell\ndeath.40 Autophagy is a bulk protein degradation system\nthat is essential for normal cell activity and survival when\nnutrients are scarce.41 The ﬁrst step that occurs in auto-\nphagic cell death is the formation of a double-mem-\nbrane vacuole, the autophagosome, which is derived\nfrom part of the endoplasmic reticulum42 or from the\ncytoplasmic lipid pool.43 The autophagosome fuses with\na lysosome, after which its contents are degraded by\nlysosomal hydrolytic proteases. Despite recent advances\nregarding the molecular mechanism(s) underlying auto-\nphagy, many details remain poorly understood. Of note,\nclass I phosphoinositide 3-kinase (PI 3-kinase) inhibits,\nwhereas Class III PI 3-kinase enhances, autophagy in\nHT-29 human colon carcinoma cells.44 Class I PI 3-ki-\nnase mediates various cellular functions, including insu-\nlin and glycogen synthesis, and suppresses apoptosis\nthrough the Akt-signaling pathway, whereas Class III PI\n3-kinase inﬂuences membrane trafﬁcking. Despite the\nﬁnding that insulin-like growth factor I (IGF-I) normally\ninhibits apoptosis, IGF-I accelerates cell death when glu-\ncose and amino acids are limited, leading to accumula-\ntion of autophagic vacuoles within the cytoplasm and\nevidence of chromosome condensation.40 Autophagic\ncell death is inhibited by 3-methyladenine (3-MA) and\nLY294002, which inhibit PI 3-kinase, but not by the pan-\ncaspase inhibitor, z-VAD-fmk. Of importance, LY294002\nhas no effect on cell death in PC12 and HepG2 carci-\nnoma cells during glucose deprivation,45 suggesting that\ninterference with the PI 3-kinase/Akt signaling pathway\ncauses autophagy of cancer cells but not normal cells.\nA mammalian gene capable of inducing autoph-\nagy, Beclin 1, is a component of the PI 3-kinase com-\nplex responsible for autophagic vesicle nucleation.16\nThis gene is deleted in 75% of ovarian, 50% of breast,\nand 40% of prostate carcinoma cases.16 Frequent al-\nlelic deletions are observed in breast carcinoma cell\nlines in association with signiﬁcant decreases in Beclin\n1 protein levels. Beclin 1 functions as a tumor sup-\npressor, because there is an attenuation of MCF-7–\nmediated tumorigenicity in nude mice overexpressing\nBeclin 1.46 Although Beclin 1 is known to interact with\nBcl-2 in yeast, the relation between this association\nand tumor suppression and autophagy is not yet clear.\nDownregulation of Bcl-2 by antisense Bcl-2 in HL-60\ncells induces autophagy independent of caspase-de-\npendent and mitochondria-dependent pathways,47\nsuggesting a potential role of Bcl-2 in autophagic cell\ndeath, as well as apoptosis. Other autophagy-inducing\ngenes, including bridging integrator 1 (Bin 1) and\ndeath-associated protein kinase (DAPK), exist. Bin 1\ninteracts with c-Myc and inhibits its transformation\nactivity.48 Bin 1 is regulated by alternate splicing, and\nthe splice forms that interact with c-Myc have tumor\nsuppressor properties. This gene is frequently missing\nor functionally inactivated in malignant melanoma,\nbreast, and prostate carcinoma cells, and loss of Bin 1\nexpression may promote tumor progression, while\nlimiting susceptibility to farnesyltransferase inhibitors\n(FTIs). Bin 1 adapter proteins act downstream or par-\nallel to RhoB to exert their effect on cell signaling.49\nDAPK is a calcium-regulated serine/threonine kinase,\nand an important mediator of autophagy induced by\nnutrient deprivation, antiestrogens, and tumor necro-\nsis factor.50 Cancer and cancer cell lines frequently\ndemonstrate decreased DAPK expression due to DNA\nmethylation.51 Transfection of Beclin 1, Bin 1, or an\nactive form of DAPK, into cancer cells induces auto-\nphagic cell death. A human homolog of the Drosophila\nspin gene product (HSpin 1) exists, which binds to\nBcl-2 and Bcl-xL. HSpin 1-induced cell death results in\nformation of autophagic vacuoles and release of a\nmature form of cathepsin D, suggesting a novel\ncaspase-independent cell death, pathway resulting in\nautophagy.52\nThe tumor suppressor phosphatase and tensin\nhomolog deleted on chromosome10 (PTEN), which is\nan inhibitor of the PI 3-kinase/Akt signaling pathway,\npromotes autophagy in HT-29 colon carcinoma\ncells.53 PTEN PI phosphatase activity inhibits Akt ac-\ntivity, resulting in induction of autophagy. Loss of\nfunction of PTEN increases the expression of Class I PI\n3-kinase and Akt in human cancer cells. Activation of\nAkt and constitutive expression of active Akt block\nautophagic cell death, leading to inhibition of apopto-\nsis and activation of a mammalian target of rapamycin\n(mTor) protein. It is likely that activation of Akt and\nthe mTor signaling pathway, due to loss of PTEN,\ncontributes to malignant transformation by simulta-\nneous inhibition of autophagic and apoptotic cell\ndeath. Another oncoprotein, Ha-Ras, induces autoph-\nagic cell death in glioma and gastric carcinoma cells54\nvia a p53-independent pathway that is not blocked by\nBcl-2. Another Ras family, including K-Ras and N-Ras\nproteins, also induces autophagic cell death.\nIn-\ncreased Ras expression is observed in spontaneous\nregression of neuroblastoma, which is characterized\nby autophagic cell death and a lack of apoptosis.55 In\naddition, Ha-Ras overexpression induces autophagic\ncell death in neuroblastoma cell\nlines. Ha-Ras–in-\nduced autophagy in HT-29 colon carcinoma cells has\nCell Death with Chemotherapy/Kim\nsome overlap with the Raf 1/ERK signaling pathway of\nautophagic cell death.56 The ability of Ras to cause\nautophagy might involve activation of ErbB2 and\nErbB3. It is interesting to note that one of the signaling\npathways regulated by Ras, the Raf1/ERK pathway,\ntriggers autophagy, whereas the PI 3-kinase/Akt path-\nway, which is downstream of Ras signaling, inhibits it.\nRas-induced increases in the expression and trafﬁck-\ning of lysosomal cathepsins might contribute to Ras-\ninduced autophagy. A proposed molecular model of\nautophagy is illustrated in Figure 2.\nA death-promoting “BH3-only” family protein, Bcl-2/\nadenovirus E1B 19-kilodalton interacting protein 3\n(BNIP3), differs from other proteins of this class.\nBNIP3 can directly integrate into the mitochondrial\nouter membrane via its BH3 domain in the absence of\nBax/Bak, thereby disrupting the MMP, generating re-\nactive oxygen species (ROS) and promoting autoph-\nagic degeneration.57 Overexpression of BNIP3, a hy-\npoxia-inducible protein with a BH3 domain, results in\nautophagic cell death in cancer cell lines, after cyto-\nchrome c or AIF nuclear translocation. BNIP3-induced\nautophagy can be attenuated by overexpression of Bcl-2,\nwhich interacts with BNIP3 protein. Autophagic cell\ndeath can be activated in cancer cells in response to\nvarious anticancer drugs. Treatment of MCF-7 breast\ncarcinoma cells with tamoxifen induces autophagy be-\nfore apoptotic cell death.12 Treatment with estradiol and\n3-MA inhibits tamoxifen-induced cell death. Similarly,\nother anticancer drugs, such as vinblastin, paclitaxel,\ninduce autophagic cell death.58 – 60\nand rapamycin,\nTreatment of malignant glioma cells with arsenic triox-\nide induces autophagic cell death in association with\nG2/M arrest.61 In addition, radiotherapy induces auto-\nphagic cell death in several cancer cell lines, including\nbreast, colon, and prostate carcinoma, as well as glio-\nblastoma multiforme.62 Despite several reports of induc-\ntion of autophagy by treatment with anticancer drugs\nand/or irradiation, the clinical signiﬁcance of autophagy\nas a result of chemotherapy remains unclear.\nINTERACTION BETWEEN APOPTOTIC AND\nAUTOPHAGIC PATHWAYS OF CELL DEATH\nDifferent types of cell death differ in their morphologic\ncharacteristics (Fig. 3 and Table 1). In fact, ceramide,\nwhich is believed to activate apoptosis, also induces\nCANCER April 15, 2005 / Volume 103 / Number 8\nChromatin condensation\nDNA laddering and fragmentation\nPyknosis\nCytoplasmic condensation\nFragmentation to apoptotic bodies\nIncrease in MMP\nActivation of caspase cascade\nPotential release of lysosomal enzyme\nOrganelles do not swell\nBlebbing\nCaspases such as caspase 3\nYes\nz-VAD-fmk\nXIAP\nBcl-2/Bcl-xL\nSometimes with actinomycin D\nSometimes with cyclophosphamide\nDNA laddering test\nCaspase activation and the substrate\nTUNEL and annexin V staining\nFACS analysis\nElectron microscopy\nautophagy.63 Similarly, proteins like DAPK are capable\nof inducing both apoptosis and autophagy depending\non the cell type involved, suggesting that they may\nfunction as molecular switches or integrators of these\ntwo types of programmed cell death. Commitment to\nautophagy or apoptosis may depend on the cell type\ninvolved, as well as the microenvironment. Depending\non the setting, initiation of autophagy may induce,\ndelay, or antagonize apoptosis. Three scenarios must\nbe considered when contemplating the interaction be-\ntween apoptosis and autophagic cell death.\nFirst, autophagy may precede apoptosis. Treat-\nment of sympathetic neurons with cytosine arabino-\nside, thereby depleting neural growth factor, results in\na 30-fold increase in autophagy, followed by DNA\ndegradation and apoptosis.64 Delayed induction of ap-\noptosis occurs with 3-MA, which inhibits cytochrome\nc release and caspase activation. These ﬁndings may\nmodel cell death after treatment with anticancer drugs\nor growth factor deprivation.\nSecond, autophagy may antagonize apoptosis.\nConversely, a reduction in autophagy might increase\nthe susceptibility of cells to apoptotic stimuli. Accel-\nerated sulindac sulﬁde-induced apoptosis occurs in\nmutant clone HT-29 colon carcinoma cells, which lack\nautophagic capability due to overexpression of a mu-\ntant G (alpha i3) protein.65 In contrast, treatment of\nparent HT-29 cells with 3-MA increases sensitivity to\nsulindac sulﬁde-induced apoptosis. Even though\nthere are no differences in the expression patterns of\nCOX-2, Bcl-2, Bcl-xL, Bax, and Akt/PKB among mutant\nand parent HT-29 cells, greater cytochrome c release\noccurs in mutant G (alpha i3) protein overexpressing\ncells, compared with parent cells. It is possible that\nautophagy retards apoptosis in colon carcinoma cells\nby sequestering mitochondrial death-promoting fac-\ntors, such as cytochrome c.\nThird, apoptosis and autophagy may be mutually\nexclusive. Inhibition of autophagy may lead to apo-\nptotic cell death. A recent report reveals that treat-\nment of malignant glioma cells with arsenic trioxide\ninduces autophagic cell death. However, inhibition of\nautophagy by treatment with baﬁlomycin, a H-AT-\nleads to apoptosis.61 However, al-\nPase inhibitor,\nthough treatment of neurons and HeLa cells with a\npancaspase inhibitor prevents apoptosis, selective mi-\nimpairment results in autophagy.66 In\ntochondrial\nlight of several reports indicating that treatment with\ncaspase inhibitors causes a shift from apoptotic cell\ndeath to autophagy, it is possible that this also occurs\nunder other circumstances. For example, inhibition of\napoptosis in Bax/Bak double-deﬁcient DKO cells in-\nduces autophagic cell death.67 The ability of cells to\nrevert between apoptosis and autophagy might be\nclinically signiﬁcant because drug resistance to apo-\nptosis might be overcome by activation of autophagic\ncell death. Along this line, multidrug-resistant MCF-7\ncells can be made susceptible to doxorubicin by treat-\nment with verapamil, after which autophagic cell\ndeath occurs, suggesting that the autophagic cell\ndeath pathway remains viable after inhibition of the\napoptotic pathway (unpublished data). The molecular\nCell Death with Chemotherapy/Kim\nmechanism(s) by which reversion from apoptosis to\nautophagy occurs remains unclear. Although overex-\npression of Bcl-2 or Bcl-xL may inhibit autophagic cell\ndeath, the role of these antiapoptotic proteins in au-\ntophagic cell death requires further clariﬁcation.\nThe role of Bcl-2 in apoptosis in response to mi-\ntochondrial damage is well established. However, sig-\nnaling as a result of mitochondrial damage might also\nactivate the autophagic pathway of cell death. It ap-\npears that mitochondria are involved in integrating\nthe apoptotic and autophagic pathways of cell death.\nLysosomal enzymes might activate apoptosis because\nlysosome rupture activates apoptosis. It is interesting\nto note that apoptosis after lysosome rupture is pre-\nceded by autophagy. Sequestration of mitochondria\nwithin autophagic vacuoles might protect cells from\napoptosis by delaying the release of cytochrome c.\nThus, autophagy might regulate apoptosis through ly-\nsosome formation and enzyme activity. With regard to\nthe possibility of signaling between lysosomes and\nmitochondria, a recent study reveals that production\nof H2O2 from ROS in response to DNA damage acti-\nvates lysosomal enzymes, such as cathepsin D and B.68\nCathepsin B directly or indirectly results in transloca-\ntion of proapoptotic proteins, such as Bid and Bax,\nfrom the cytoplasm to mitochondria. This alters MMP\nand results in caspase activation subsequent to the\nrelease of cytochrome c, Smac/Diablo, and AIF from\nthe mitochondria. Conversely, under different circum-\nstances, production of H2O2 from ROS in response to\nDNA damage may inhibit the release of small mole-\ncules, such as cytochrome c, Smac/Diablo, and AIF,\nfrom the mitochondrial membrane and activate lyso-\nsomal activity. Thus, crosstalk might occur between\nthe pathways mediating autophagy and apoptosis by\nway of lysosomes and mitochondria. Further research\nis required to clarify the signaling that occurs between\nthese pathways.\nFUTURE PERSPECTIVES REGARDING CELL DEATH\nAND CANCER THERAPY\nIncreasing information is known regarding the molec-\nular mechanism(s) by which cell death is mediated by\nanticancer drugs. Proapoptotic and antiapoptotic pro-\nteins are now being targeted to enhance the effect(s)\nof chemotherapy. In general, the antitumor effect of\nan anticancer agent is measured in terms of its ability\nto induce apoptotic cell death within tumors, which is\nwell correlated with outcome in most preclinical and\nclinical studies. Nevertheless, despite signiﬁcant re-\nductions in tumor volume, high rates of apoptosis do\nnot always occur within solid tumors after chemother-\napy. This suggests that other types of cell death, such\nas autophagy and necrosis, might also occur as a result\nCANCER April 15, 2005 / Volume 103 / Number 8\nof chemotherapy. Research into other types of cell\ndeath, particularly autophagic cell death, is important\nto better understand the effects of anticancer drugs. In\naddition, the relation between autophagic cell death\nand apoptosis needs to be examined, because several\nautophagy-related proteins, including Beclin 1, BNIP3,\nDAPK, and HSpin 1, may inﬂuence apoptosis. Because\napoptosis and autophagy are mutually exclusive in\nsome cell lines, molecular switches between the two\nprocesses must exist.\nResistance to autophagy, as well as apoptosis,\nmight lend resistance to anticancer drugs. Likewise,\ninduction of autophagic cell death in cancer cells may\nbe of therapeutic beneﬁt. Cells transformed with the\ntumor suppressor gene Beclin 1 are less likely to un-\ndergo autophagy and are more susceptible to onco-\ngenesis. Alterations in autophagic genes reduce the\nincidence of autophagy in cancer cells. However, even\nin cells resistant to apoptosis or cells in which apo-\nptosis is inhibited, nonapoptotic cell death pathways,\nsuch as autophagy, may be activated. Targeting alter-\nnative pathways of cell death might overcome drug\nresistance or explain why the effects of chemotherapy\nand irradiation complement each other in the clinical\nsetting. Alterations or defects in apoptotic signal\ntransduction pathways might result in drug resistance,\nhowever, irradiation induces autophagic cell death.\nAlterations of one or both pathways (autophagic or\napoptotic) might determine susceptibility to cell death\nafter chemotherapy because anticancer drugs might\ntrigger both apoptotic caspase-dependent and auto-\nphagic caspase-independent pathways at the same\ntime.\nAutophagic cell death might also inhibit angio-\ngenesis.69 The angiogenesis inhibitor, endostatin, in-\nduces endothelial cell death via the caspase-indepen-\ndent autophagic pathway. In addition, the tumor\nsuppressor gene, PTEN, which induces autophagy, in-\nhibits tumor angiogenesis in nude mice with ortho-\ntopic brain tumors.70 Reconstitution of wild-type\nPTEN has no effect on in vitro proliferation, but dra-\nmatically decreases tumor growth in vivo and pro-\nlongs survival in mice, suggesting that PTEN regulates\ntumor-induced angiogenesis via regulation of PI-de-\npendent signals. The role of autophagy in tumor an-\ngiogenesis might be an important means of inhibiting\ntumor progression and increasing the effectiveness of\nanticancer therapy.\nIt is now clear that a number of different types of\ncell death, including apoptosis and autophagy, play a\nrole in the antitumor effect(s) of treatment with anti-\ncancer drugs. Targeting therapy at genes related to\napoptosis might increase the therapeutic efﬁcacy of\nanticancer agents by modulating signal transduction\npathways, and other cell death pathways might be\ninﬂuenced by changes in the microenvironment and\nby crosstalk between the various cell death pathways.\nFurther research into the molecular mechanism(s) be-\nhind autophagic cell death as a result of chemother-\napy might provide a new strategy for overcoming drug\nresistance by exploring this alternative pathway of cell\ndeath.\n28. Hasenjager A, Gillissen B, Muller A, et al. Smac induces\ncytochrome c release and apoptosis independently from\nBax/Bcl-x(L) in a strictly caspase-3-dependent manner in\nhuman carcinoma cells. Oncogene. 2004;23:4523– 4535.\n29. Hao Y, Sekine K, Kawabata A, et al. Apollon ubiquitinates\nSMAC and caspase-9, and has an essential cytoprotection\nfunction. Nat Cell Biol. 2004;6:849 – 860.\nCell Death with Chemotherapy/Kim\n40. Schweichel JU, Merker HJ. The morphology of various types\nof cell death in prenatal tissues. Teratology. 1973;7:253–266.\n41. Mizushima N, Yamamoto A, Matsui M, Yoshimori T, Oh-\nsumi Y. In vivo analysis of autophagy in response to nutrient\nstarvation using transgenic mice expressing a ﬂuorescent\nautophagosome marker. Mol Biol Cell. 2004;15:1101–1111.\n42. Dunn WA Jr. Studies on the mechanisms of autophagy:\nformation of the autophagic vacuole. J Cell Biol. 1990;110:\n1923–1933.\nCANCER April 15, 2005 / Volume 103 / Number 8",
         "R. Kim",
         null
        ],
        [
         "27",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "Phytochromes are photochromic biliproteins found in plants as well as in\nsome cyanotrophic, photoautotrophic and heterotrophic bacteria. In many\nbacteria, their function is largely unknown. Here we describe the biochemi-\ncal and spectroscopic characterization of recombinant bacterial phyto-\naeruginosa\nchrome\n(PaBphP). The recombinant protein displays all the characteristic features\nof a bonaﬁde phytochrome. In contrast with cyanobacteria and plants, the\nchromophore of this bacterial phytochrome is biliverdin IXa, which is pro-\nduced by the heme oxygenase BphO in P. aeruginosa. This chromophore\nwas shown to be covalently attached via its A-ring endo-vinyl group to a\ncysteine residue outside the deﬁned bilin lyase domain of plant and cyano-\nbacterial phytochromes. Site-directed mutagenesis identiﬁed Cys12 and\nHis247 as being important for chromophore binding and photoreversibility,\nrespectively. PaBphP is synthesized in the dark in the red-light-absorbing\nPr form and immediately converted into a far-red-light-absorbing Pfr-\nenriched form. It shows the characteristic red ⁄ far-red-light-induced photo-\nreversibility of phytochromes. A chromophore analog that\nlacks the\nC15 ⁄ 16 double bond was used to show that this photoreversibility is due to\na 15Z ⁄ 15E isomerization of the biliverdin chromophore. Autophosphoryla-\ntion of PaBphP was demonstrated, conﬁrming its role as a sensor kinase of\na bacterial two-component signaling system.",
         "Phytochromes are biliprotein photoreceptors in plants\nbut have recently also been discovered in bacteria [1].\nIn plants, the family of phytochromes sense red and\nfar-red light and therefore play a key role in mediating\nresponses to light quality, quantity, direction and dur-\nation throughout plant development [2]. Plant phyto-\nchromes are homodimers composed of  125-kDa\nsubunits each with a thioether-linked phytochromobi-\nlin prosthetic group [3]. Unlike the light-harvesting\ncyanobacterial phycobiliproteins which require a lyase\nfor the covalent attachment of the linear tetrapyrrole\n(bilin) chromophore, bilin attachment to apo-phyto-\nchromes is autocatalytic [4]. The action of phyto-\nchrome depends on its ability to photointerconvert\nbetween the red-light-absorbing Pr form and the far-\nred-light-absorbing Pfr form, a property conferred by\nthe covalently bound phytochromobilin in the plant\nholophytochrome. The ﬁrst phytochrome from a bac-\nterial source to be discovered was Cph1 (cyanobac-\nterial phytochrome 1) from Synechocystis sp. PCC6803\nwhich was followed by the discovery of bacterial phy-\ntochromes (BphPs) from nonphotosynthetic bacteria\n[1,5,6]. BphPs are typical sensor kinases of a two-com-\nponent signaling system. Most BphPs including that of\nPseudomonas aeruginosa (PaBphP) carry a C-terminal\nhistidine kinase module, and it has been shown that\nmany of them, such as Synechocystis Cph1 [6], Agro-\nbacterium Agp1 and Agp2 [7,8] and also Pseudomonas\nsyringae BphP [9], are light-regulated histidine kinases.\nUnlike plant and cyanobacterial phytochromes,\nwhich carry a phytochromobilin or phycocyanobilin\nchromophore, BphPs have been shown to utilize a bili-\nverdin chromophore [9]. Apart from Cph1, most mem-\nbers of the BphP family lack the conserved cysteine\nresidue in the conserved bilin lyase domain (BLD).\nThis domain has been deﬁned as the minimal GAF\ndomain, capable of autocatalytic assembly with bilin\nchromophores [10]. GAF domains are small\nligand-\nbinding domains found in vertebrate cGMP-speciﬁc\nphosphodiesterases, cyanobacterial adenylate cyclases\nand the formate hydrogen lyase transcription activator\nFhlA [11]. In most phytochromes, the BLD is preceded\nby the P2 domain, which is often recognized as a PAS\ndomain in the Pfam database (protein families data-\nbase; http://www.sanger.ac.uk/Software/Pfam/)\n[12].\nPAS domains are tandem repeats ﬁrst described in the\ntranscriptional regulatory proteins period clock (PER)\nfrom Drosophila melanogaster,\nthe murine aromatic\nhydrocarbon receptor nuclear translocator (ARNT)\nand single minded (SIM) from D. melanogaster [13].\nInterestingly, a cysteine residue in this P2 domain has\nbeen shown to be the site of chromophore attachment\nin Agp1\n[7,14].\nAnother characteristic domain in phytochromes is the\nPHY domain which corresponds to a GAF-related\ndomain located C-terminally to the BLD (Scheme 1).\nfrom Agrobacterium tumefaciens\nRecently we have shown that BphP from P. aerugi-\nnosa is able to bind biliverdin IXa and biliverdin IXd,\nwhich are produced by the two heme oxygenases BphO\nand PigA [15]. As bphO is chromosomally located\nupstream of bphP and the afﬁnity for biliverdin IXa\nwas about ﬁvefold higher than for biliverdin IXd, we\nconcluded that biliverdin IXa is the natural chromo-\nphore of PaBphP. Furthermore, we presented data\nindicating an involvement of BphP in biliverdin release\nfrom BphO, as this is the rate-limiting step of the\nBphO reaction.\nHere we describe the further biochemical and spect-\nroscopic characterization of BphP.\nC\nP2 BLD PH Y HKD\nResults\nExpression, puriﬁcation and initial characterization\nof recombinant P. aeruginosa phytochrome",
         "The P. aeruginosa bphP was expressed using a tet pro-\nmoter-driven C-terminal Strep tag expression system.\nRecombinant BphP was always puriﬁed in the apo\nform, and the homogeneity after puriﬁcation was\n 98% (Fig. 1,\ninset). A single band migrating at\n 80 kDa was obtained on SDS ⁄ PAGE, which corre-\nlates with the predicted molecular mass calculated\nfrom the amino-acid composition (80.1 kDa). The\nyield of puriﬁed BphP was typically 5 mg per litre of\nbacterial culture. Analytical gel permeation chromato-\ngraphy revealed that the apo form, as well as the\nassembled holo form, of BphP is eluted as a dimer\nfrom a Superdex 200 column (data not shown; [15]).\nAssembly and chromophore binding\nPaBphP is able to autocatalytically form a photocon-\nvertible holo-phytochrome with the proposed natural\nchromophore biliverdin IXa. Illumination of recombin-\nant holo-BphP with saturating red light (630 nm) resul-\nted in the formation of the Pfr form (Pfr-enriched)\nwhich could be converted back into the Pr form through\nillumination with far-red light (750 nm) (Fig. 1A). The\nresultant calculated difference spectrum shows the char-\nacteristic phytochrome signature (Fig. 1B) with maxima\nof 700 and 754 nm for the Pr and Pfr form, respectively.\nThese far-red absorbance maxima seem to be typical of\nbiliverdin-binding phytochromes and represent the most\nred-shifted phytochrome forms described so far [7,16].\nThe covalent binding of biliverdin IXa was conﬁrmed\nby zinc-induced red ﬂuorescence (Fig. 3C).\nThe form initially synthesized after the addition of\nbiliverdin IXa to apo-BphP in the dark is the Pr form,\nwhich is immediately converted nonphotochemically\ninto a Pfr-enriched form. This nonphotochemical con-\nversion reaches an equilibrium between Pr and Pfr\nforms after 90 min (Fig. 2A). Irradiation with far-red\nlight leads to the formation of the Pr form with one\npeak at 700 nm, which can be converted back into the\nPfr form by irradiation with red light. Both the Pr and\nthe Pfr form are unstable in the dark and convert back\ninto a dark form, a Pfr-enriched mixture of Pr and Pfr\n(Fig. 2B,C).\nChromophore–protein interaction\nTo determine which part of the bilin chromophore is\ninvolved in covalent attachment to the protein, various\nA\nB\nA\nB\nbiliverdin derivatives were used (see Fig. 3B for chem-\nical structures). The resultant chromoproteins were\ncharacterized by red ⁄ far-red-light-induced difference\nspectroscopy (Fig. 3A). The spectral properties are\nsummarized in Table 1. BphP is able to covalently bind\nbiliverdin IXd and biliverdin XIIIa, which was con-\nﬁrmed by zinc-induced red ﬂuorescence (Fig. 3C). Fur-\nthermore, these biliverdin adducts were able to form\na photoconvertible holophytochrome. No characteristic\ndifference spectrum nor covalent binding was observed\nwith biliverdin IXb, biliverdin IXc, mesobiliverdin,\n31,32-dihydrobiliverdin and biliverdin IIIa (Fig. 3B,C\nand [15]). The common feature of all covalently bound\nbiliverdin derivatives is an A-ring endo-vinyl group,\nindicating that this side chain is absolutely required for\ncovalent attachment. Furthermore, these results imply\nthat the ring substituents of the other pyrrole rings do\nnot seem to be critical for photoconversion.\nA\nB\nC\nPhotoisomerization of PaBphP\nThe primary photoreaction of plant phytochromes is\nknown to be the 15Z ⁄ 15E isomerization of the phyto-\nchromobilin chromophore [17]. If the C15 double bond\nis missing (i.e. in phycoerythrobilin), the corresponding\nphytochrome adduct is unable to undergo photoiso-\nmerization but instead is highly ﬂuorescent [18]. This\nﬂuorescent adduct of phytochromes is also known as a\nphytoﬂuor [19]. To elucidate whether the photoisomeri-\nzation in PaBphP is also due to a 15Z ⁄ 15E isomeriza-\ntion of\nthe bilin prosthetic group (in this case\nbiliverdin), we incubated apo-BphP with 15,16-dihydro-\nbiliverdin (see Fig. 4 for structure). 15,16-Dihydrobiliv-\nerdin can be synthesized in vitro from biliverdin by the\nferredoxin-dependent bilin reductase PebA [20]. Apo-\nBphP is able to bind 15,16-dihydrobiliverdin and is\norange ﬂuorescent under UV light\n(312 nm). This\nphenomenon was investigated ﬂuorospectrometrically.\nExcitation at 570 nm resulted in a ﬂuorescent phyto-\nﬂuor with an emission maximum of 630 nm (Fig. 4).\nChromophore attachment site\nBphP lacks the conserved cysteine residue involved in\ncovalent bilin attachment in plant and most cyanobac-\nterial phytochromes, and therefore the site and kind of\nattachment of the bilin chromophore in the bacterial\nphytochromes is controversial [1,21,22]. To investigate\nwhether the chromophore is attached via a thioether\nlinkage to a cysteine residue, the protein was treated\nwith iodoacetamide. This reagent speciﬁcally modiﬁes\ncysteine residues. If a chromophore-binding cysteine is\naccessible to iodoacetamide, a subsequent covalent\nchromophore attachment should be inhibited. Addition\nof increasing amounts of iodoacetamide leads to a\nreduction in photoisomerzation, as visualized by differ-\nence spectroscopy and covalent chromophore binding\n(i.e. decreased zinc-induced red ﬂuorecence). Full inhi-\nbition was observed with 1 mm iodoacetamide (data\nnot shown). These results imply that the site of chro-\nmophore attachment in P. aeruginosa BphP is most\nlikely a cysteine residue. BphP contains twelve cysteine\nresidues, two of which, at position 12 and 248, could\npossibly serve as the chromophore-binding site. A cys-\nteine corresponding to position 12 has already been\nreported to be the site of chromophore attachment in\nAgp1 from A. tumefaciens\nlocated\nwithin the BLD and is adjacent to the chromophore-\nbinding site in cyanobacterial and plant phytochromes.\nTo further investigate the potential site of chromo-\nphore attachment, site-directed mutants (C12A, C12S\nand C248A) were generated and analyzed using the\nabove methods. Neither BphP C12A nor C12S showed\ncharacteristic difference spectra. The difference spectra\nof these variants (Fig. 5A) were very similar to the\niodoacetamide-blocked wild-type spectra (data not\nshown). The variant BphP C248A was able to form a\nphotoconvertible holoform with maximum and mini-\nmum identical with those of the wild-type (Table 2).\nOnly the C248A variant showed covalent biliverdin\nbinding, as demonstrated by zinc-induced red ﬂuores-\ncence (Fig. 5B). The covalent attachment of biliverdin\nto BphP was further conﬁrmed using a biliverdin\nreductase (BVR) assay. In this assay, only free biliver-\ndin can be converted by BVR into bilirubin. The addi-\ntion of BVR and NADPH to C12A:biliverdin and\nC12S:biliverdin resulted in the conversion of the bound\nA\nbiliverdin into bilirubin, which was accompanied by a\ncolor change from green to yellow, indicating the forma-\ntion of bilirubin. No biliverdin conversion was detected\nafter addition of BVR to wild-type BphP and the other\nvariants investigated (data not shown). Overall, these\nresults are in agreement with the data from Agp1 and\nindicate the importance of Cys12 in covalent chromo-\nphore binding.\nBesides this N-terminally located cysteine residue, a\nhistidine residue in the BLD has been discussed as the\nchromophore-binding site in Deinococcus radiodurans\nBphP and Calothrix sp. PCC7601 CphB [1,23]. This his-\ntidine residue is located adjacent to the conserved cys-\nteine residue in cyanobacterial and plant phytochromes.\nTo investigate the role of this histidine residue, a H247Q\nmutant was generated. H247Q was able to form a pho-\ntoconvertible holoform with blue-shifted extrema (694\nand 746 nm) (Table 2). For this variant, covalent bili-\nverdin binding was demonstrated using zinc-induced\nred ﬂuorescence and the BVR assay (data not shown).\nAutophosphorylation of BphP\nLight-regulated His phosphorylation has been demon-\nstrated for several bacterial phytochromes. Amino-acid\nsequence analysis revealed that BphP also contains a\nhistidine kinase module (Scheme 1). Autophosphoryla-\ntion of BphP was determined after incubation of puri-\nﬁed apo-BphP and holo-BphP (Pr and Pfr form) with\n[32P]ATP[cP]. Both forms of BphP displayed auto-\nphosphorylation activity (Fig. 6A). Although the Pfr-\nenriched form shows slightly higher kinase actvity, no\nstrong light-dependence could be detected. BphP was\nconﬁrmed to be a histidine kinase, as the phosphoryla-\ntion was stable in alkaline solution and labile in acid\n(Fig. 6B). This was further conﬁrmed by replacing the\npotential phosphorylation site (H513) by alanine. No\nautophosphorylation was detected in this H513A\nmutant (data not shown).\nDiscussion\nPaBphP is a bacterial phytochrome using a\nbiliverdin chromophore\nAll chemicals were purchased from Sigma (Munich, Ger-\nmany) and were American Chemical Society grade or bet-\nter. Restriction enzymes were from Invitrogen (Cleveland,\nOH, USA). MasterTaqTM was purchased from Eppendorf\nScientiﬁc (Westbury, NY, USA). The expression vector\npASK-IBA3, Strep Tactin Sepharose, and anhydrotetra-\ncycline were obtained from IBA GmbH (Go¨ ttingen, Ger-\nmany). Centricon-10 concentrator devices were purchased\nfrom Amicon (Beverly, MA, USA). Biliverdin IXa was\nobtained from Frontier Scientiﬁc (Logan, UT, USA).\n15,16-Dihydrobiliverdin, biliverdin IXb, biliverdin IXc, bili-\nverdin IXd and phycocyanobilin were prepared as described\npreviously [15,20]. Biliverdin XIIIa, biliverdin IIIa, 31,32-\ndihydrobiliverdin and mesobiliverdin were gifts\nfrom\nJ.C. Lagarias (UC Davis, CA, USA) and K. Inomata\n(Kanazawa University, Japan) [21,26,27].\nConstruction of expression vectors\nThe P. aeruginosa bphP (PA 4117) gene was ampliﬁed by\nPCR from chromosomal DNA using a hot start proto-\ncol with the following primers, which contained the\nindicated and underlined restriction sites: bphPXbaRBSfwd:\n5¢-CGTCTAGATAACGAGGGCAAAAAATGACGAG\nCATCACCCGGTTACC-3¢;\n5¢-CC\nCTCGAGGGACGAGGAGCCGGTCTCCG-3¢. The PCR\nproduct was digested with the indicated enzymes and\ncloned into XbaI ⁄ XhoI-digested expression vector pASK-\nIBA3 (IBA). The integrity of the plasmid construct was veri-\nﬁed by DNA sequence determination of the insert (SeqLab,\nGo¨ ttingen, Germany). The resulting ORF encodes BphP\nwith a C-terminal Strep Tag with a total addition of 20\namino-acid residues under the control of a tet promoter.\nSite-directed mutagenesis of bphP was performed using the\nQuikChange site-directed mutagenesis kit (Stratagene, La\nJolla, CA, USA), according to the instructions of\nthe\nmanufacturer with the following primers (only one primer\nshown, second primer is complement, the underlined codon\nrepresents the introduced mutation). bphP C12A, 5¢-GGT\nTACCCTGGCGAACGCCGAGGACGAACCCATCC-3¢;\nbphP C12S 5¢-GGTTACCCTGGCGAACTCCGAGGAC\nGAACCCATCC-3¢; bphP H247Q, 5¢-GCAGCGTTTCG\nCCGATCCAGTGCGAATACCTGACC-3¢; bphP C248A,\n5¢-CGTTTCGCCGATCCACGCCGAATACCTGACCA\nAC-3¢ and bphP H513A, 5¢-GCGGTGCTCGGCGCCG\nACCTGCGCAAC-3¢. Mutants were also conﬁrmed by\nDNA sequencing (SeqLab).\nProtein production and puriﬁcation\nRecombinant P. aeruginosa BphP was produced using a\ntet promoter-driven Strep tag system ([28]; IBA) in the\nEscherichia coli strain DH5a and was grown at 37 C\nampicillin (100\nin Luria–Bertani medium containing\nlgÆmL)1) to an A578 of 0.5. Cultures were induced by the\naddition of 0.2 lgÆmL)1 anhydrotetracycline and incuba-\nted at 25 C overnight. The bacterial pellet from 3 L of\nculture was resuspended in lysis buffer (50 mm Tris ⁄ HCl,\npH 8.0, 100 mm NaCl, 0.05% Triton X-100) (3 mL buffer\nper g of cells) and disrupted by sonication. Cell debris\nwas removed by ultracentrifugation (30 min, 100 000 g),\nand the supernatant was subjected to a 40% (NH4)2SO4\ncut. The resultant pellet was dissolved in buffer W\n(20 mm\n1 mm\ndithiothreitol), and after 20 min centrifugation (23 000 g),\nthe supernatant was incubated with 40 lgÆmL)1 avidin\nleast 10 min on ice. The\n(ﬁnal concentration)\nresulting supernatant was loaded on to a Strep-Tactin\nSepharose column (5 mL), which had previously been\nequilibrated with buffer W. The puriﬁcation was per-\nformed according to the instructions\nsupplied by the\nmanufacturer (IBA). Fractions containing BphP were fur-\nther puriﬁed using anion-exchange chromatography on\nQ Sepharose (Amersham Biosciences) using a linear gra-\nin 50 mm Hepes ⁄ KOH, pH 8.0.\ndient of KCl\nBphP was eluted with 500 mm KCl from the Q Sepha-\nrose column.\n20 mm NaCl,\nfor at\nProtein determination\nProtein concentration was determined by the Bradford\nmethod with BSA as standard [29] or by measuring A280\nusing the calculated e280nm ¼ 78 457 m)1Æcm)1 for BphP\n[30].\nAnalytical gel permeation chromatography\nGel permeation chromatography experiments were carried\nout using a Superdex 200 HR10 ⁄ 30 column as described\npreviously [15].\nAssembly of PaBphP\nIn vitro chromophore assembly of PaBphP was tested using\n20 lm recombinant apo-BphP, which was incubated with\n40 lm chromophore for 30 min at room temperature in\nthe dark (ﬁnal volume 50 lL). Absorbance spectra were\nobtained after 3 min of incubation with red light at 630 nm\n(Pfr spectrum) and after 3 min of incubation with far red\nin a volume of 500 lL\nlight at 750 nm (Pr spectrum)\n(50 mm Hepes ⁄ HCl, pH 8.0, 20 mm KCl), and the differ-\nence was calculated.\nTo characterize the different forms of holo-BphP spec-\ntroscopically, absorbance spectra between 500 and 800 nm\nwere obtained. Biliverdin IXa (20 lm) was added to 10 lm\nBphP in a ﬁnal volume of 500 lL, and spectra were meas-\nured during incubation in the dark or during irradiation\nwith red and far red light, respectively.\nTo test covalent chromophore attachment to BphP, cova-\nlently bound bilins were visualized by zinc-induced red\nﬂuorescence as described previously [31]. For iodoacetamide\ntreatment, BphP apoprotein was mixed with different con-\ncentrations of the blocking reagent from a 5 mm stock solu-\ntion and incubated for 20 min at room temperature [32].\nSpectra and zinc-induced red ﬂuorescence were measured as\ndescribed above.\nFluorescence spectroscopy\nRoom temperature ﬂuorescence emission and excitation\nspectra were recorded using a Perkin–Elmer LS50B spectro-\nﬂuorimeter. Fluorescence spectra were measured at 570 nm\nexcitation (the absorption maximum of dihydrobiliverdin)\nor at 630 nm emission.\nBVR assay\nA BVR assay was used to characterize the complex of\nBphP–biliverdin IXa. BVR catalyzes the conversion of bili-\nverdin IXa into bilirubin IXa, which absorbs at 450 nm.\nBVR can only convert noncovalenty bound biliverdin IXa\ninto bilirubin. Apo-BphP was incubated with an excess of\nbiliverdin IXa for 30 min in the dark. The BphP–biliverdin\ncomplex was separated from free biliverdin IXa using\nNAP-5 desalting columns (Amersham Biosciences), which\nwere equilibrated with buffer (100 mm Tris ⁄ HCl, pH 8.7).\nThe concentration of protein-bound biliverdin IXa was\nmeasured spectroscopically and estimated using e680 ¼\n12 400 m)1Æcm)1 for free biliverdin IXa. An aliquot of\n 5 lg crude soluble protein extract of recombinant rat\nBVR was added to 20 lm biliverdin in a complex of BphP–\nbiliverdin in 100 mm Tris ⁄ HCl, pH 8.7. The reaction was\nstarted by the addition of an NADPH-regenerating system\ncontaining 6.5 mm glucose 6-phosphate, 0.82 mm NADP+\nand 1.1 UÆmL)1 glucose-6-phosphate dehydrogenase. Spec-\ntral changes between 300 and 800 nm were monitored.\nProtein kinase assays\nAutophosphorylation was performed as described for Cph1\n[6]. Holo-BphP was irradiated with saturating red (630 nm)\nor far-red (750 nm) light before the addition of [32P]ATP[cP]\nand subsequently incubated for 30 min at room temperature\nwith the corresponding light. Radioisotope imaging was\nmonitored using a Bio-Rad Molecular Imager FX.\nAcknowledgements\nWe are grateful to Drs Inomata (Kanazawa University,\nKanazawa, Japan) and Lagarias (UC Davis, Davis,\nCA, USA) for the gift of chromophores. We thank\nMaria Sowa and Thorben Dammeyer for technical\nassistance. This work was supported by the Emmy-\nNoether-Program of\nthe Deutsche Forschungsge-\nmeinschaft and funds from the Fonds der Chemischen\nIndustrie to N.F.-D.\n1 Davis SJ, Vener AV & Vierstra RD (1999) Bacteriophy-\ntochromes: phytochrome-like photoreceptors from non-\nphotosynthetic eubacteria. Science 286, 2517–2520.\n2 Quail PH, Boylan MT, Parks BM, Short TW, Xu Y &\nWagner D (1995) Phytochromes: photosensory percep-\ntion and signal transduction. Science 268, 675–680.\n3 Lagarias JC & Rapoport H (1980) Chromopeptides\nfrom phytochrome. The structure and linkage of the Pr\nform of the phytochrome chromophore. J Am Chem\nSoc 102, 4821–4828.\n4 Terry MJ, Wahleithner JA & Lagarias JC (1993) Bio-\nsynthesis of the plant photoreceptor phytochrome. Arch\nBiochem Biophys 306, 1–15.\n5 Hughes J, Lamparter T, Mittmann F, Hartmann E,\nGa¨ rtner W, Wilde A & Bo¨ rner T (1997) A prokaryotic\nphytochrome. Nature 386, 663.\n6 Yeh KC, Wu SH, Murphy JT & Lagarias JC (1997) A\ncyanobacterial phytochrome two-component light sen-\nsory system. Science 277, 1505–1508.\n7 Lamparter T, Michael N, Mittmann F & Esteban B\n(2002) Phytochrome from Agrobacterium tumefaciens\nhas unusual spectral properties and reveals an N-term-\ninal chromophore attachment site. Proc Natl Acad Sci\nUSA 99, 11628–11633.\n8 Karniol B & Vierstra RD (2003) The pair of bacterio-\nphytochromes from Agrobacterium tumefaciens are histi-\ndine kinases with opposing photobiological properties.\nProc Natl Acad Sci USA 100, 2807–2812.\n9 Bhoo SH, Davis SJ, Walker J, Karniol B & Vierstra\nRD (2001) Bacteriophytochromes are photochromic his-\ntidine kinases using a biliverdin chromophore. Nature\n414, 776–779.\n10 Wu SH & Lagarias JC (2000) Deﬁning the bilin lyase\ndomain: lessons from the extended phytochrome super-\nfamily. Biochemistry 39, 13487–13495.\n11 Aravind L & Ponting CP (1997) The GAF domain: an\nevolutionary link between diverse phototransducing pro-\nteins. Trends Biochem Sci 22, 458–459.\n20 Frankenberg N, Mukougawa K, Kohchi T & Lagarias\nJC (2001) Functional genomic analysis of the HY2\nfamily of ferredoxin-dependent bilin reductases from\noxygenic photosynthetic organisms. Plant Cell 13,\n965–978.\n21 Lamparter T, Michael N, Caspani O, Miyata T, Shirai\nK & Inomata K (2003) Biliverdin binds covalently to\nagrobacterium phytochrome Agp1 via its ring A vinyl\nside chain. J Biol Chem 278, 33786–33792.\n22 Jorissen HJ, Quest B, Remberg A, Coursin T, Braslav-\nsky SE, Schaffner K, de Marsac NT & Ga¨ rtner W\n(2002) Two independent, light-sensing two-component\nsystems in a ﬁlamentous cyanobacterium. Eur J Biochem\n269, 2662–2671.\n23 Quest B & Ga¨ rtner W (2004) Chromophore selectivity\nin bacterial phytochromes: dissecting the process of\nchromophore attachment. Eur J Biochem 271, 1117–\n1126.\n24 Smith H (2000) Phytochromes and light signal percep-\ntion by plants: an emerging synthesis. Nature 407,\n585–591.\n25 Lamparter T (2004) Evolution of cyanobacterial and\n12 Montgomery BL & Lagarias JC (2002) Phytochrome\nplant phytochromes. FEBS Lett 573, 1–5.\nancestry: sensors of bilins and light. Trends Plant Sci 7,\n357–366.\n13 Huang ZJ, Edery I & Rosbash M (1993) PAS is a\ndimerization domain common to Drosophila period and\nseveral transcription factors. Nature 364, 259–262.\n14 Lamparter T, Carrascal M, Michael N, Martinez E,\nRottwinkel G & Abian J (2004) The biliverdin chromo-\nphore binds covalently to a conserved cysteine residue\nin the N-terminus of Agrobacterium phytochrome Agp1.\nBiochemistry 43, 3659–3669.\n15 Wegele R, Tasler R, Zeng Y, Rivera M & Frankenberg-\nDinkel N (2004) The heme oxgygenase(s)-phytochrome\nsystem of Pseudomonas aeruginosa. J Biol Chem 279,\n45791–45802.\n16 Giraud E, Fardoux J, Fourrier N, Hannibal L, Genty\nB, Bouyer P, Dreyfus B & Vermeglio A (2002) Bacterio-\nphytochrome controls photosystem synthesis in anoxy-\ngenic bacteria. Nature 417, 202–205.\n17 Li L & Lagarias JC (1992) Phytochrome assembly.\nDeﬁning chromophore structural requirements for cova-\nlent attachment and photoreversibility. J Biol Chem 267,\n19204–19210.\n18 Li L, Murphy JT & Lagarias JC (1995) Continuous\nﬂuorescence assay of phytochrome assembly in vitro.\nBiochemistry 34, 7923–7930.\n19 Murphy JT & Lagarias JC (1997) The phytoﬂuors: a new\nclass of ﬂuorescent protein probes. Curr Biol 7, 870–876.\n26 Elich TD, McDonagh AF, Palma LA & Lagarias JC\n(1989) Phytochrome chromophore biosynthesis. Treat-\nment of tetrapyrrole-deﬁcient Avena explants with\nnatural and non-natural bilatrienes leads to formation\nof spectrally active holoproteins. J Biol Chem 264,\n183–189.\n27 Cornejo J, Beale SI, Terry MJ & Lagarias JC (1992)\nPhytochrome assembly: the structure and biological\nactivity of 2(R),3(E)-phytochromobilin derived from\nphycobiliproteins. J Biol Chem 267, 14790–14798.\n28 Skerra A (1994) Use of the tetracycline promoter for\nthe tightly regulated production of a murine antibody\nfragment in Escherichia coli. Gene 151, 131–135.\n29 Bradford MM (1976) A rapid and sensitive method for\nthe quantitation of microgram quantities of protein util-\nizing the principle of protein-dye binding. Anal Biochem\n72, 248–254.\n30 Gill SC & von Hippel PH (1989) Calculation of protein\nextinction coefﬁcients from amino acid sequence. Anal\nBiochem 182, 319–326.\n31 Berkelman TR & Lagarias JC (1986) Visualization of\nbilin-linked peptides and proteins in polyacrylamide\ngels. Anal Biochem 156, 194–201.\n32 Lamparter T, Esteban B & Hughes J (2001) Phyto-\nchrome Cph1 from the cyanobacterium Synechocystis\nPCC6803. Puriﬁcation, assembly, and quaternary struc-\nture. Eur J Biochem 268, 4720–4730.",
         "PaBphP was among the ﬁrst bacterial phytochromes\nto be discovered, and it has already been shown that\nthis BphP together with other members of this phyto-\nchrome class utilizes a biliverdin chromophore [1,9,15].\nthe\nOur laboratory has recently demonstrated that\nPaBphP chromophore is produced by one of the two\nP. aeruginosa heme oxygenases. The BphO heme oxy-\ngenase is encoded in the same operon as bphP, and we\nwere able to show that BphP is involved in the release\nof the biliverdin produced from BphO. The major\nfunction of BphP remains unknown, but these results\nprovide biochemical evidence that recombinant BphP\nhas all the characteristics of a red ⁄ far-red-light-respon-\nsive photoreceptor.\nTo date only a few bacterial phytochromes have\nbeen biochemically characterized in detail. Among\nthem are Agp1 and Agp2 (also known as AtBphP1\nand AtBphP2) from A. tumefaciens. The most interest-\ning spectral observation for several BphPs including\nPaBphP is the Pfr-like ground state [8,16]. Assembly\nof PaBphP with biliverdin ﬁrst generates a transient\nPr-like intermediate, which is then nonphotochemically\ntransformed into a stable Pfr-enriched form (Fig. 2A).\nInterestingly, illumination with red light does not fully\nconvert this form into a solely Pfr form (Fig. 2C). The\nPfr-enriched form found after dark assembly is differ-\nent from that obtained through dark conversion of\neither the Pr or Pfr forms (Fig. 2B,C). Incubation of\npre-illuminated BphP always resulted in the formation\nof a Pr ⁄ Pfr equilibrium in the dark.\nAlthough autophosphorylation activity was demon-\nstrated for BphP, only a weak light-dependence has\nbeen observed with the Pfr-enriched form diplaying\nhighest kinase activity. However, this may also be due\nto the amount of Pr present in the Pfr-enriched form.\nThe observed Pfr ground state is the opposite of that\nused by almost all other known members of the classic\nphytochrome family [12,24]. More recent reports also\nrevealed the presence of the Pfr ground state in Agp2\nfrom A. tumefaciens and the BphPs of Rhodopseudo-\nmonas palustris and Bradyrhizobium ORS278 [8,16].\nFor the latter organisms, the Pfr ground state has been\nimplicated to be necessary for maximal photoregula-\ntion of photosynthesis by not overlapping with chloro-\nphyll absorption [16]. The reason for the Pfr ground\nstate in Agp2 and PaBphP is still not known, but if\nPaBphP indeed functions as a photoreceptor\nin\nP. aeruginosa, it would be expected to serve as a sensor\nof the ratio between far-red and red light.\nAn A-ring endo-vinyl group is required for\ncovalent attachment\nOur data obtained using biliverdin derivatives and site-\ndirected mutagenesis of PaBphP are in agreement with\ndata obtained for Agp1 from A. tumefaciens [7,21].\nBoth BphPs seem to covalently bind the biliverdin\nchromophore at a conserved cysteine residue in the P2\ndomain close to the N-terminus of the protein. An\nA-ring endo-vinyl group of the chromophore is abso-\nlutely required for this covalent attachment [15,21].\nOur data support the proposal that the lack of the\nconserved cysteine residue in the BLD correlates with\nthe use of biliverdin as the chromophore and the bind-\ning to a conserved cysteine residue in the P2 domain\n[25]. Nevertheless, the BLD still seems to be quite\nimportant for the photochemical reaction, as a H247Q\nmutation resulted in a spectral shift of the Pr and Pfr\nforms. Therefore, the BLD may play a role in stabil-\nthe chromophore and\nization and co-ordination of\npossibly its covalent attachment to Cys12 (i.e. the bilin\nlyase function).\nInterestingly, PaBphP Cys12 mutants assembled bili-\nverdin, but the afﬁnity of biliverdin was about ﬁvefold\nlower than wild-type BphP. The assembled Cys12 vari-\nants displayed a Pr-like aborption spectrum, which did\nnot alter upon red-light illumination (data not shown).\nThis observation is in contrast with data obtained for\nAgp1. An Agp1 C20A mutant was fully photoreversi-\nble, but had a reduced absorption coefﬁcient, a blue-\nshifted Pfr maximum, and a reduced ratio of Pfr to Pr\nabsorption [7]. In the case of PaBphP, the mutation of\nthis conserved cysteine residue is much more dramatic\nthan in Agp1. It seems that, in PaBphP, mutation of\nthis residue not only abolishes covalent binding, but\nstabilization of the Pfr form is also lost. This may be\ndue to a loosely or wrongly oriented biliverdin in the\nchromophore pocket. At this point it is worth men-\ntioning that, although many of our data point towards\nCys12 as the site of covalent chromophore attachment,\nit still cannot be ruled out that this cysteine residue\nonly plays a structural role (i.e. disulﬁde bond forma-\ntion). Consequently, its mutation would lead to a loss\nof the structural environment necessary for covalent\nbinding.\nPhotoconversion of BphPs involves 15Z ⁄ 15E\nisomerization\nit has always been\nSince the discovery of BphPs,\nassumed that the photochemical reaction is similar to\nthat found in plant phytochromes, which involves a\n15Z ⁄ 15E isomerization of the phytochromobilin chro-\nmophore [17]. This assumption has not yet been\nexperimentally conﬁrmed. We used a chromophore\nanalog that lacks the C15 ⁄ C16 double bond to investi-\ngate the photoisomerization of BphP. The phytoﬂuor\nthe\nadduct obtained conﬁrmed the involvement of\nC15 ⁄ C16 double bond in photoisomerization, as the\ndihydrobiliverdin adduct is highly ﬂuorescent. Further-\nmore, these data imply that, although the chromo-\nphore is attached at a different position, the geometry\nin PaBphP is\nof\nprobably similar to that of plant phytochromes.\nthe chromophore-binding pocket\nConclusion and outlook\nWe have shown that recombinant PaBphP has all\nfeatures of a ‘true’ phytochrome photoreceptor. It cova-\nlently binds a biliverdin chromophore, which, upon illu-\nmination with red and far-red light, photoisomerizes at\nthe C15 ⁄ C16 double bond. Furthermore, we conﬁrmed\nthat PaBphP is a histidine kinase. The results of this\nwork support the proposal of a separate bacterial phyto-\nchrome class with a new chromophore-binding site in\nthe P2 domain, although a solely structural role for this\nresidue cannot be completely ruled out. The function of\nBphPs in nonphotosynthetic micro-organisms remains a\nmystery. To elucidate this further, we have constructed\nchromosomal knock-out mutations in the P. aeruginosa\nbphOP operon, which are currently being investigated\nusing proteomic and transcriptomics analysis.\nExperimental procedures\nReagents",
         "All chemicals were purchased from Sigma (Munich, Ger-\nmany) and were American Chemical Society grade or bet-\nter. Restriction enzymes were from Invitrogen (Cleveland,\nOH, USA). MasterTaqTM was purchased from Eppendorf\nScientiﬁc (Westbury, NY, USA). The expression vector\npASK-IBA3, Strep Tactin Sepharose, and anhydrotetra-\ncycline were obtained from IBA GmbH (Go¨ ttingen, Ger-\nmany). Centricon-10 concentrator devices were purchased\nfrom Amicon (Beverly, MA, USA). Biliverdin IXa was\nobtained from Frontier Scientiﬁc (Logan, UT, USA).\n15,16-Dihydrobiliverdin, biliverdin IXb, biliverdin IXc, bili-\nverdin IXd and phycocyanobilin were prepared as described\npreviously [15,20]. Biliverdin XIIIa, biliverdin IIIa, 31,32-\ndihydrobiliverdin and mesobiliverdin were gifts\nfrom\nJ.C. Lagarias (UC Davis, CA, USA) and K. Inomata\n(Kanazawa University, Japan) [21,26,27].\nConstruction of expression vectors\nThe P. aeruginosa bphP (PA 4117) gene was ampliﬁed by\nPCR from chromosomal DNA using a hot start proto-\ncol with the following primers, which contained the\nindicated and underlined restriction sites: bphPXbaRBSfwd:\n5¢-CGTCTAGATAACGAGGGCAAAAAATGACGAG\nCATCACCCGGTTACC-3¢;\n5¢-CC\nCTCGAGGGACGAGGAGCCGGTCTCCG-3¢. The PCR\nproduct was digested with the indicated enzymes and\ncloned into XbaI ⁄ XhoI-digested expression vector pASK-\nIBA3 (IBA). The integrity of the plasmid construct was veri-\nﬁed by DNA sequence determination of the insert (SeqLab,\nGo¨ ttingen, Germany). The resulting ORF encodes BphP\nwith a C-terminal Strep Tag with a total addition of 20\namino-acid residues under the control of a tet promoter.\nSite-directed mutagenesis of bphP was performed using the\nQuikChange site-directed mutagenesis kit (Stratagene, La\nJolla, CA, USA), according to the instructions of\nthe\nmanufacturer with the following primers (only one primer\nshown, second primer is complement, the underlined codon\nrepresents the introduced mutation). bphP C12A, 5¢-GGT\nTACCCTGGCGAACGCCGAGGACGAACCCATCC-3¢;\nbphP C12S 5¢-GGTTACCCTGGCGAACTCCGAGGAC\nGAACCCATCC-3¢; bphP H247Q, 5¢-GCAGCGTTTCG\nCCGATCCAGTGCGAATACCTGACC-3¢; bphP C248A,\n5¢-CGTTTCGCCGATCCACGCCGAATACCTGACCA\nAC-3¢ and bphP H513A, 5¢-GCGGTGCTCGGCGCCG\nACCTGCGCAAC-3¢. Mutants were also conﬁrmed by\nDNA sequencing (SeqLab).\nProtein production and puriﬁcation\nRecombinant P. aeruginosa BphP was produced using a\ntet promoter-driven Strep tag system ([28]; IBA) in the\nEscherichia coli strain DH5a and was grown at 37 C\nampicillin (100\nin Luria–Bertani medium containing\nlgÆmL)1) to an A578 of 0.5. Cultures were induced by the\naddition of 0.2 lgÆmL)1 anhydrotetracycline and incuba-\nted at 25 C overnight. The bacterial pellet from 3 L of\nculture was resuspended in lysis buffer (50 mm Tris ⁄ HCl,\npH 8.0, 100 mm NaCl, 0.05% Triton X-100) (3 mL buffer\nper g of cells) and disrupted by sonication. Cell debris\nwas removed by ultracentrifugation (30 min, 100 000 g),\nand the supernatant was subjected to a 40% (NH4)2SO4\ncut. The resultant pellet was dissolved in buffer W\n(20 mm\n1 mm\ndithiothreitol), and after 20 min centrifugation (23 000 g),\nthe supernatant was incubated with 40 lgÆmL)1 avidin\nleast 10 min on ice. The\n(ﬁnal concentration)\nresulting supernatant was loaded on to a Strep-Tactin\nSepharose column (5 mL), which had previously been\nequilibrated with buffer W. The puriﬁcation was per-\nformed according to the instructions\nsupplied by the\nmanufacturer (IBA). Fractions containing BphP were fur-\nther puriﬁed using anion-exchange chromatography on\nQ Sepharose (Amersham Biosciences) using a linear gra-\nin 50 mm Hepes ⁄ KOH, pH 8.0.\ndient of KCl\nBphP was eluted with 500 mm KCl from the Q Sepha-\nrose column.\n20 mm NaCl,\nfor at\nProtein determination\nProtein concentration was determined by the Bradford\nmethod with BSA as standard [29] or by measuring A280\nusing the calculated e280nm ¼ 78 457 m)1Æcm)1 for BphP\n[30].\nAnalytical gel permeation chromatography\nGel permeation chromatography experiments were carried\nout using a Superdex 200 HR10 ⁄ 30 column as described\npreviously [15].\nAssembly of PaBphP\nIn vitro chromophore assembly of PaBphP was tested using\n20 lm recombinant apo-BphP, which was incubated with\n40 lm chromophore for 30 min at room temperature in\nthe dark (ﬁnal volume 50 lL). Absorbance spectra were\nobtained after 3 min of incubation with red light at 630 nm\n(Pfr spectrum) and after 3 min of incubation with far red\nin a volume of 500 lL\nlight at 750 nm (Pr spectrum)\n(50 mm Hepes ⁄ HCl, pH 8.0, 20 mm KCl), and the differ-\nence was calculated.\nTo characterize the different forms of holo-BphP spec-\ntroscopically, absorbance spectra between 500 and 800 nm\nwere obtained. Biliverdin IXa (20 lm) was added to 10 lm\nBphP in a ﬁnal volume of 500 lL, and spectra were meas-\nured during incubation in the dark or during irradiation\nwith red and far red light, respectively.\nTo test covalent chromophore attachment to BphP, cova-\nlently bound bilins were visualized by zinc-induced red\nﬂuorescence as described previously [31]. For iodoacetamide\ntreatment, BphP apoprotein was mixed with different con-\ncentrations of the blocking reagent from a 5 mm stock solu-\ntion and incubated for 20 min at room temperature [32].\nSpectra and zinc-induced red ﬂuorescence were measured as\ndescribed above.\nFluorescence spectroscopy\nRoom temperature ﬂuorescence emission and excitation\nspectra were recorded using a Perkin–Elmer LS50B spectro-\nﬂuorimeter. Fluorescence spectra were measured at 570 nm\nexcitation (the absorption maximum of dihydrobiliverdin)\nor at 630 nm emission.\nBVR assay\nA BVR assay was used to characterize the complex of\nBphP–biliverdin IXa. BVR catalyzes the conversion of bili-\nverdin IXa into bilirubin IXa, which absorbs at 450 nm.\nBVR can only convert noncovalenty bound biliverdin IXa\ninto bilirubin. Apo-BphP was incubated with an excess of\nbiliverdin IXa for 30 min in the dark. The BphP–biliverdin\ncomplex was separated from free biliverdin IXa using\nNAP-5 desalting columns (Amersham Biosciences), which\nwere equilibrated with buffer (100 mm Tris ⁄ HCl, pH 8.7).\nThe concentration of protein-bound biliverdin IXa was\nmeasured spectroscopically and estimated using e680 ¼\n12 400 m)1Æcm)1 for free biliverdin IXa. An aliquot of\n 5 lg crude soluble protein extract of recombinant rat\nBVR was added to 20 lm biliverdin in a complex of BphP–\nbiliverdin in 100 mm Tris ⁄ HCl, pH 8.7. The reaction was\nstarted by the addition of an NADPH-regenerating system\ncontaining 6.5 mm glucose 6-phosphate, 0.82 mm NADP+\nand 1.1 UÆmL)1 glucose-6-phosphate dehydrogenase. Spec-\ntral changes between 300 and 800 nm were monitored.\nProtein kinase assays\nAutophosphorylation was performed as described for Cph1\n[6]. Holo-BphP was irradiated with saturating red (630 nm)\nor far-red (750 nm) light before the addition of [32P]ATP[cP]\nand subsequently incubated for 30 min at room temperature\nwith the corresponding light. Radioisotope imaging was\nmonitored using a Bio-Rad Molecular Imager FX.\nAcknowledgements\nWe are grateful to Drs Inomata (Kanazawa University,\nKanazawa, Japan) and Lagarias (UC Davis, Davis,\nCA, USA) for the gift of chromophores. We thank\nMaria Sowa and Thorben Dammeyer for technical\nassistance. This work was supported by the Emmy-\nNoether-Program of\nthe Deutsche Forschungsge-\nmeinschaft and funds from the Fonds der Chemischen\nIndustrie to N.F.-D.\n1 Davis SJ, Vener AV & Vierstra RD (1999) Bacteriophy-\ntochromes: phytochrome-like photoreceptors from non-\nphotosynthetic eubacteria. Science 286, 2517–2520.\n2 Quail PH, Boylan MT, Parks BM, Short TW, Xu Y &\nWagner D (1995) Phytochromes: photosensory percep-\ntion and signal transduction. Science 268, 675–680.\n3 Lagarias JC & Rapoport H (1980) Chromopeptides\nfrom phytochrome. The structure and linkage of the Pr\nform of the phytochrome chromophore. J Am Chem\nSoc 102, 4821–4828.\n4 Terry MJ, Wahleithner JA & Lagarias JC (1993) Bio-\nsynthesis of the plant photoreceptor phytochrome. Arch\nBiochem Biophys 306, 1–15.\n5 Hughes J, Lamparter T, Mittmann F, Hartmann E,\nGa¨ rtner W, Wilde A & Bo¨ rner T (1997) A prokaryotic\nphytochrome. Nature 386, 663.\n6 Yeh KC, Wu SH, Murphy JT & Lagarias JC (1997) A\ncyanobacterial phytochrome two-component light sen-\nsory system. Science 277, 1505–1508.\n7 Lamparter T, Michael N, Mittmann F & Esteban B\n(2002) Phytochrome from Agrobacterium tumefaciens\nhas unusual spectral properties and reveals an N-term-\ninal chromophore attachment site. Proc Natl Acad Sci\nUSA 99, 11628–11633.\n8 Karniol B & Vierstra RD (2003) The pair of bacterio-\nphytochromes from Agrobacterium tumefaciens are histi-\ndine kinases with opposing photobiological properties.\nProc Natl Acad Sci USA 100, 2807–2812.\n9 Bhoo SH, Davis SJ, Walker J, Karniol B & Vierstra\nRD (2001) Bacteriophytochromes are photochromic his-\ntidine kinases using a biliverdin chromophore. Nature\n414, 776–779.\n10 Wu SH & Lagarias JC (2000) Deﬁning the bilin lyase\ndomain: lessons from the extended phytochrome super-\nfamily. Biochemistry 39, 13487–13495.\n11 Aravind L & Ponting CP (1997) The GAF domain: an\nevolutionary link between diverse phototransducing pro-\nteins. Trends Biochem Sci 22, 458–459.\n20 Frankenberg N, Mukougawa K, Kohchi T & Lagarias\nJC (2001) Functional genomic analysis of the HY2\nfamily of ferredoxin-dependent bilin reductases from\noxygenic photosynthetic organisms. Plant Cell 13,\n965–978.\n21 Lamparter T, Michael N, Caspani O, Miyata T, Shirai\nK & Inomata K (2003) Biliverdin binds covalently to\nagrobacterium phytochrome Agp1 via its ring A vinyl\nside chain. J Biol Chem 278, 33786–33792.\n22 Jorissen HJ, Quest B, Remberg A, Coursin T, Braslav-\nsky SE, Schaffner K, de Marsac NT & Ga¨ rtner W\n(2002) Two independent, light-sensing two-component\nsystems in a ﬁlamentous cyanobacterium. Eur J Biochem\n269, 2662–2671.\n23 Quest B & Ga¨ rtner W (2004) Chromophore selectivity\nin bacterial phytochromes: dissecting the process of\nchromophore attachment. Eur J Biochem 271, 1117–\n1126.\n24 Smith H (2000) Phytochromes and light signal percep-\ntion by plants: an emerging synthesis. Nature 407,\n585–591.\n25 Lamparter T (2004) Evolution of cyanobacterial and\n12 Montgomery BL & Lagarias JC (2002) Phytochrome\nplant phytochromes. FEBS Lett 573, 1–5.\nancestry: sensors of bilins and light. Trends Plant Sci 7,\n357–366.\n13 Huang ZJ, Edery I & Rosbash M (1993) PAS is a\ndimerization domain common to Drosophila period and\nseveral transcription factors. Nature 364, 259–262.\n14 Lamparter T, Carrascal M, Michael N, Martinez E,\nRottwinkel G & Abian J (2004) The biliverdin chromo-\nphore binds covalently to a conserved cysteine residue\nin the N-terminus of Agrobacterium phytochrome Agp1.\nBiochemistry 43, 3659–3669.\n15 Wegele R, Tasler R, Zeng Y, Rivera M & Frankenberg-\nDinkel N (2004) The heme oxgygenase(s)-phytochrome\nsystem of Pseudomonas aeruginosa. J Biol Chem 279,\n45791–45802.\n16 Giraud E, Fardoux J, Fourrier N, Hannibal L, Genty\nB, Bouyer P, Dreyfus B & Vermeglio A (2002) Bacterio-\nphytochrome controls photosystem synthesis in anoxy-\ngenic bacteria. Nature 417, 202–205.\n17 Li L & Lagarias JC (1992) Phytochrome assembly.\nDeﬁning chromophore structural requirements for cova-\nlent attachment and photoreversibility. J Biol Chem 267,\n19204–19210.\n18 Li L, Murphy JT & Lagarias JC (1995) Continuous\nﬂuorescence assay of phytochrome assembly in vitro.\nBiochemistry 34, 7923–7930.\n19 Murphy JT & Lagarias JC (1997) The phytoﬂuors: a new\nclass of ﬂuorescent protein probes. Curr Biol 7, 870–876.\n26 Elich TD, McDonagh AF, Palma LA & Lagarias JC\n(1989) Phytochrome chromophore biosynthesis. Treat-\nment of tetrapyrrole-deﬁcient Avena explants with\nnatural and non-natural bilatrienes leads to formation\nof spectrally active holoproteins. J Biol Chem 264,\n183–189.\n27 Cornejo J, Beale SI, Terry MJ & Lagarias JC (1992)\nPhytochrome assembly: the structure and biological\nactivity of 2(R),3(E)-phytochromobilin derived from\nphycobiliproteins. J Biol Chem 267, 14790–14798.\n28 Skerra A (1994) Use of the tetracycline promoter for\nthe tightly regulated production of a murine antibody\nfragment in Escherichia coli. Gene 151, 131–135.\n29 Bradford MM (1976) A rapid and sensitive method for\nthe quantitation of microgram quantities of protein util-\nizing the principle of protein-dye binding. Anal Biochem\n72, 248–254.\n30 Gill SC & von Hippel PH (1989) Calculation of protein\nextinction coefﬁcients from amino acid sequence. Anal\nBiochem 182, 319–326.\n31 Berkelman TR & Lagarias JC (1986) Visualization of\nbilin-linked peptides and proteins in polyacrylamide\ngels. Anal Biochem 156, 194–201.\n32 Lamparter T, Esteban B & Hughes J (2001) Phyto-\nchrome Cph1 from the cyanobacterium Synechocystis\nPCC6803. Puriﬁcation, assembly, and quaternary struc-\nture. Eur J Biochem 268, 4720–4730.",
         "Phytochromes are photochromic biliproteins found in plants as well as in\nsome cyanotrophic, photoautotrophic and heterotrophic bacteria. In many\nbacteria, their function is largely unknown. Here we describe the biochemi-\ncal and spectroscopic characterization of recombinant bacterial phyto-\naeruginosa\nchrome\n(PaBphP). The recombinant protein displays all the characteristic features\nof a bonaﬁde phytochrome. In contrast with cyanobacteria and plants, the\nchromophore of this bacterial phytochrome is biliverdin IXa, which is pro-\nduced by the heme oxygenase BphO in P. aeruginosa. This chromophore\nwas shown to be covalently attached via its A-ring endo-vinyl group to a\ncysteine residue outside the deﬁned bilin lyase domain of plant and cyano-\nbacterial phytochromes. Site-directed mutagenesis identiﬁed Cys12 and\nHis247 as being important for chromophore binding and photoreversibility,\nrespectively. PaBphP is synthesized in the dark in the red-light-absorbing\nPr form and immediately converted into a far-red-light-absorbing Pfr-\nenriched form. It shows the characteristic red ⁄ far-red-light-induced photo-\nreversibility of phytochromes. A chromophore analog that\nlacks the\nC15 ⁄ 16 double bond was used to show that this photoreversibility is due to\na 15Z ⁄ 15E isomerization of the biliverdin chromophore. Autophosphoryla-\ntion of PaBphP was demonstrated, conﬁrming its role as a sensor kinase of\na bacterial two-component signaling system.\nPhytochromes are biliprotein photoreceptors in plants\nbut have recently also been discovered in bacteria [1].\nIn plants, the family of phytochromes sense red and\nfar-red light and therefore play a key role in mediating\nresponses to light quality, quantity, direction and dur-\nation throughout plant development [2]. Plant phyto-\nchromes are homodimers composed of  125-kDa\nsubunits each with a thioether-linked phytochromobi-\nlin prosthetic group [3]. Unlike the light-harvesting\ncyanobacterial phycobiliproteins which require a lyase\nfor the covalent attachment of the linear tetrapyrrole\n(bilin) chromophore, bilin attachment to apo-phyto-\nchromes is autocatalytic [4]. The action of phyto-\nchrome depends on its ability to photointerconvert\nbetween the red-light-absorbing Pr form and the far-\nred-light-absorbing Pfr form, a property conferred by\nthe covalently bound phytochromobilin in the plant\nholophytochrome. The ﬁrst phytochrome from a bac-\nterial source to be discovered was Cph1 (cyanobac-\nterial phytochrome 1) from Synechocystis sp. PCC6803\nwhich was followed by the discovery of bacterial phy-\ntochromes (BphPs) from nonphotosynthetic bacteria\n[1,5,6]. BphPs are typical sensor kinases of a two-com-\nponent signaling system. Most BphPs including that of\nPseudomonas aeruginosa (PaBphP) carry a C-terminal\nhistidine kinase module, and it has been shown that\nmany of them, such as Synechocystis Cph1 [6], Agro-\nbacterium Agp1 and Agp2 [7,8] and also Pseudomonas\nsyringae BphP [9], are light-regulated histidine kinases.\nUnlike plant and cyanobacterial phytochromes,\nwhich carry a phytochromobilin or phycocyanobilin\nchromophore, BphPs have been shown to utilize a bili-\nverdin chromophore [9]. Apart from Cph1, most mem-\nbers of the BphP family lack the conserved cysteine\nresidue in the conserved bilin lyase domain (BLD).\nThis domain has been deﬁned as the minimal GAF\ndomain, capable of autocatalytic assembly with bilin\nchromophores [10]. GAF domains are small\nligand-\nbinding domains found in vertebrate cGMP-speciﬁc\nphosphodiesterases, cyanobacterial adenylate cyclases\nand the formate hydrogen lyase transcription activator\nFhlA [11]. In most phytochromes, the BLD is preceded\nby the P2 domain, which is often recognized as a PAS\ndomain in the Pfam database (protein families data-\nbase; http://www.sanger.ac.uk/Software/Pfam/)\n[12].\nPAS domains are tandem repeats ﬁrst described in the\ntranscriptional regulatory proteins period clock (PER)\nfrom Drosophila melanogaster,\nthe murine aromatic\nhydrocarbon receptor nuclear translocator (ARNT)\nand single minded (SIM) from D. melanogaster [13].\nInterestingly, a cysteine residue in this P2 domain has\nbeen shown to be the site of chromophore attachment\nin Agp1\n[7,14].\nAnother characteristic domain in phytochromes is the\nPHY domain which corresponds to a GAF-related\ndomain located C-terminally to the BLD (Scheme 1).\nfrom Agrobacterium tumefaciens\nRecently we have shown that BphP from P. aerugi-\nnosa is able to bind biliverdin IXa and biliverdin IXd,\nwhich are produced by the two heme oxygenases BphO\nand PigA [15]. As bphO is chromosomally located\nupstream of bphP and the afﬁnity for biliverdin IXa\nwas about ﬁvefold higher than for biliverdin IXd, we\nconcluded that biliverdin IXa is the natural chromo-\nphore of PaBphP. Furthermore, we presented data\nindicating an involvement of BphP in biliverdin release\nfrom BphO, as this is the rate-limiting step of the\nBphO reaction.\nHere we describe the further biochemical and spect-\nroscopic characterization of BphP.\nC\nP2 BLD PH Y HKD\nResults\nExpression, puriﬁcation and initial characterization\nof recombinant P. aeruginosa phytochrome\nThe P. aeruginosa bphP was expressed using a tet pro-\nmoter-driven C-terminal Strep tag expression system.\nRecombinant BphP was always puriﬁed in the apo\nform, and the homogeneity after puriﬁcation was\n 98% (Fig. 1,\ninset). A single band migrating at\n 80 kDa was obtained on SDS ⁄ PAGE, which corre-\nlates with the predicted molecular mass calculated\nfrom the amino-acid composition (80.1 kDa). The\nyield of puriﬁed BphP was typically 5 mg per litre of\nbacterial culture. Analytical gel permeation chromato-\ngraphy revealed that the apo form, as well as the\nassembled holo form, of BphP is eluted as a dimer\nfrom a Superdex 200 column (data not shown; [15]).\nAssembly and chromophore binding\nPaBphP is able to autocatalytically form a photocon-\nvertible holo-phytochrome with the proposed natural\nchromophore biliverdin IXa. Illumination of recombin-\nant holo-BphP with saturating red light (630 nm) resul-\nted in the formation of the Pfr form (Pfr-enriched)\nwhich could be converted back into the Pr form through\nillumination with far-red light (750 nm) (Fig. 1A). The\nresultant calculated difference spectrum shows the char-\nacteristic phytochrome signature (Fig. 1B) with maxima\nof 700 and 754 nm for the Pr and Pfr form, respectively.\nThese far-red absorbance maxima seem to be typical of\nbiliverdin-binding phytochromes and represent the most\nred-shifted phytochrome forms described so far [7,16].\nThe covalent binding of biliverdin IXa was conﬁrmed\nby zinc-induced red ﬂuorescence (Fig. 3C).\nThe form initially synthesized after the addition of\nbiliverdin IXa to apo-BphP in the dark is the Pr form,\nwhich is immediately converted nonphotochemically\ninto a Pfr-enriched form. This nonphotochemical con-\nversion reaches an equilibrium between Pr and Pfr\nforms after 90 min (Fig. 2A). Irradiation with far-red\nlight leads to the formation of the Pr form with one\npeak at 700 nm, which can be converted back into the\nPfr form by irradiation with red light. Both the Pr and\nthe Pfr form are unstable in the dark and convert back\ninto a dark form, a Pfr-enriched mixture of Pr and Pfr\n(Fig. 2B,C).\nChromophore–protein interaction\nTo determine which part of the bilin chromophore is\ninvolved in covalent attachment to the protein, various\nA\nB\nA\nB\nbiliverdin derivatives were used (see Fig. 3B for chem-\nical structures). The resultant chromoproteins were\ncharacterized by red ⁄ far-red-light-induced difference\nspectroscopy (Fig. 3A). The spectral properties are\nsummarized in Table 1. BphP is able to covalently bind\nbiliverdin IXd and biliverdin XIIIa, which was con-\nﬁrmed by zinc-induced red ﬂuorescence (Fig. 3C). Fur-\nthermore, these biliverdin adducts were able to form\na photoconvertible holophytochrome. No characteristic\ndifference spectrum nor covalent binding was observed\nwith biliverdin IXb, biliverdin IXc, mesobiliverdin,\n31,32-dihydrobiliverdin and biliverdin IIIa (Fig. 3B,C\nand [15]). The common feature of all covalently bound\nbiliverdin derivatives is an A-ring endo-vinyl group,\nindicating that this side chain is absolutely required for\ncovalent attachment. Furthermore, these results imply\nthat the ring substituents of the other pyrrole rings do\nnot seem to be critical for photoconversion.\nA\nB\nC\nPhotoisomerization of PaBphP\nThe primary photoreaction of plant phytochromes is\nknown to be the 15Z ⁄ 15E isomerization of the phyto-\nchromobilin chromophore [17]. If the C15 double bond\nis missing (i.e. in phycoerythrobilin), the corresponding\nphytochrome adduct is unable to undergo photoiso-\nmerization but instead is highly ﬂuorescent [18]. This\nﬂuorescent adduct of phytochromes is also known as a\nphytoﬂuor [19]. To elucidate whether the photoisomeri-\nzation in PaBphP is also due to a 15Z ⁄ 15E isomeriza-\ntion of\nthe bilin prosthetic group (in this case\nbiliverdin), we incubated apo-BphP with 15,16-dihydro-\nbiliverdin (see Fig. 4 for structure). 15,16-Dihydrobiliv-\nerdin can be synthesized in vitro from biliverdin by the\nferredoxin-dependent bilin reductase PebA [20]. Apo-\nBphP is able to bind 15,16-dihydrobiliverdin and is\norange ﬂuorescent under UV light\n(312 nm). This\nphenomenon was investigated ﬂuorospectrometrically.\nExcitation at 570 nm resulted in a ﬂuorescent phyto-\nﬂuor with an emission maximum of 630 nm (Fig. 4).\nChromophore attachment site\nBphP lacks the conserved cysteine residue involved in\ncovalent bilin attachment in plant and most cyanobac-\nterial phytochromes, and therefore the site and kind of\nattachment of the bilin chromophore in the bacterial\nphytochromes is controversial [1,21,22]. To investigate\nwhether the chromophore is attached via a thioether\nlinkage to a cysteine residue, the protein was treated\nwith iodoacetamide. This reagent speciﬁcally modiﬁes\ncysteine residues. If a chromophore-binding cysteine is\naccessible to iodoacetamide, a subsequent covalent\nchromophore attachment should be inhibited. Addition\nof increasing amounts of iodoacetamide leads to a\nreduction in photoisomerzation, as visualized by differ-\nence spectroscopy and covalent chromophore binding\n(i.e. decreased zinc-induced red ﬂuorecence). Full inhi-\nbition was observed with 1 mm iodoacetamide (data\nnot shown). These results imply that the site of chro-\nmophore attachment in P. aeruginosa BphP is most\nlikely a cysteine residue. BphP contains twelve cysteine\nresidues, two of which, at position 12 and 248, could\npossibly serve as the chromophore-binding site. A cys-\nteine corresponding to position 12 has already been\nreported to be the site of chromophore attachment in\nAgp1 from A. tumefaciens\nlocated\nwithin the BLD and is adjacent to the chromophore-\nbinding site in cyanobacterial and plant phytochromes.\nTo further investigate the potential site of chromo-\nphore attachment, site-directed mutants (C12A, C12S\nand C248A) were generated and analyzed using the\nabove methods. Neither BphP C12A nor C12S showed\ncharacteristic difference spectra. The difference spectra\nof these variants (Fig. 5A) were very similar to the\niodoacetamide-blocked wild-type spectra (data not\nshown). The variant BphP C248A was able to form a\nphotoconvertible holoform with maximum and mini-\nmum identical with those of the wild-type (Table 2).\nOnly the C248A variant showed covalent biliverdin\nbinding, as demonstrated by zinc-induced red ﬂuores-\ncence (Fig. 5B). The covalent attachment of biliverdin\nto BphP was further conﬁrmed using a biliverdin\nreductase (BVR) assay. In this assay, only free biliver-\ndin can be converted by BVR into bilirubin. The addi-\ntion of BVR and NADPH to C12A:biliverdin and\nC12S:biliverdin resulted in the conversion of the bound\nA\nbiliverdin into bilirubin, which was accompanied by a\ncolor change from green to yellow, indicating the forma-\ntion of bilirubin. No biliverdin conversion was detected\nafter addition of BVR to wild-type BphP and the other\nvariants investigated (data not shown). Overall, these\nresults are in agreement with the data from Agp1 and\nindicate the importance of Cys12 in covalent chromo-\nphore binding.\nBesides this N-terminally located cysteine residue, a\nhistidine residue in the BLD has been discussed as the\nchromophore-binding site in Deinococcus radiodurans\nBphP and Calothrix sp. PCC7601 CphB [1,23]. This his-\ntidine residue is located adjacent to the conserved cys-\nteine residue in cyanobacterial and plant phytochromes.\nTo investigate the role of this histidine residue, a H247Q\nmutant was generated. H247Q was able to form a pho-\ntoconvertible holoform with blue-shifted extrema (694\nand 746 nm) (Table 2). For this variant, covalent bili-\nverdin binding was demonstrated using zinc-induced\nred ﬂuorescence and the BVR assay (data not shown).\nAutophosphorylation of BphP\nLight-regulated His phosphorylation has been demon-\nstrated for several bacterial phytochromes. Amino-acid\nsequence analysis revealed that BphP also contains a\nhistidine kinase module (Scheme 1). Autophosphoryla-\ntion of BphP was determined after incubation of puri-\nﬁed apo-BphP and holo-BphP (Pr and Pfr form) with\n[32P]ATP[cP]. Both forms of BphP displayed auto-\nphosphorylation activity (Fig. 6A). Although the Pfr-\nenriched form shows slightly higher kinase actvity, no\nstrong light-dependence could be detected. BphP was\nconﬁrmed to be a histidine kinase, as the phosphoryla-\ntion was stable in alkaline solution and labile in acid\n(Fig. 6B). This was further conﬁrmed by replacing the\npotential phosphorylation site (H513) by alanine. No\nautophosphorylation was detected in this H513A\nmutant (data not shown).\nDiscussion\nPaBphP is a bacterial phytochrome using a\nbiliverdin chromophore\nPaBphP was among the ﬁrst bacterial phytochromes\nto be discovered, and it has already been shown that\nthis BphP together with other members of this phyto-\nchrome class utilizes a biliverdin chromophore [1,9,15].\nthe\nOur laboratory has recently demonstrated that\nPaBphP chromophore is produced by one of the two\nP. aeruginosa heme oxygenases. The BphO heme oxy-\ngenase is encoded in the same operon as bphP, and we\nwere able to show that BphP is involved in the release\nof the biliverdin produced from BphO. The major\nfunction of BphP remains unknown, but these results\nprovide biochemical evidence that recombinant BphP\nhas all the characteristics of a red ⁄ far-red-light-respon-\nsive photoreceptor.\nTo date only a few bacterial phytochromes have\nbeen biochemically characterized in detail. Among\nthem are Agp1 and Agp2 (also known as AtBphP1\nand AtBphP2) from A. tumefaciens. The most interest-\ning spectral observation for several BphPs including\nPaBphP is the Pfr-like ground state [8,16]. Assembly\nof PaBphP with biliverdin ﬁrst generates a transient\nPr-like intermediate, which is then nonphotochemically\ntransformed into a stable Pfr-enriched form (Fig. 2A).\nInterestingly, illumination with red light does not fully\nconvert this form into a solely Pfr form (Fig. 2C). The\nPfr-enriched form found after dark assembly is differ-\nent from that obtained through dark conversion of\neither the Pr or Pfr forms (Fig. 2B,C). Incubation of\npre-illuminated BphP always resulted in the formation\nof a Pr ⁄ Pfr equilibrium in the dark.\nAlthough autophosphorylation activity was demon-\nstrated for BphP, only a weak light-dependence has\nbeen observed with the Pfr-enriched form diplaying\nhighest kinase activity. However, this may also be due\nto the amount of Pr present in the Pfr-enriched form.\nThe observed Pfr ground state is the opposite of that\nused by almost all other known members of the classic\nphytochrome family [12,24]. More recent reports also\nrevealed the presence of the Pfr ground state in Agp2\nfrom A. tumefaciens and the BphPs of Rhodopseudo-\nmonas palustris and Bradyrhizobium ORS278 [8,16].\nFor the latter organisms, the Pfr ground state has been\nimplicated to be necessary for maximal photoregula-\ntion of photosynthesis by not overlapping with chloro-\nphyll absorption [16]. The reason for the Pfr ground\nstate in Agp2 and PaBphP is still not known, but if\nPaBphP indeed functions as a photoreceptor\nin\nP. aeruginosa, it would be expected to serve as a sensor\nof the ratio between far-red and red light.\nAn A-ring endo-vinyl group is required for\ncovalent attachment\nOur data obtained using biliverdin derivatives and site-\ndirected mutagenesis of PaBphP are in agreement with\ndata obtained for Agp1 from A. tumefaciens [7,21].\nBoth BphPs seem to covalently bind the biliverdin\nchromophore at a conserved cysteine residue in the P2\ndomain close to the N-terminus of the protein. An\nA-ring endo-vinyl group of the chromophore is abso-\nlutely required for this covalent attachment [15,21].\nOur data support the proposal that the lack of the\nconserved cysteine residue in the BLD correlates with\nthe use of biliverdin as the chromophore and the bind-\ning to a conserved cysteine residue in the P2 domain\n[25]. Nevertheless, the BLD still seems to be quite\nimportant for the photochemical reaction, as a H247Q\nmutation resulted in a spectral shift of the Pr and Pfr\nforms. Therefore, the BLD may play a role in stabil-\nthe chromophore and\nization and co-ordination of\npossibly its covalent attachment to Cys12 (i.e. the bilin\nlyase function).\nInterestingly, PaBphP Cys12 mutants assembled bili-\nverdin, but the afﬁnity of biliverdin was about ﬁvefold\nlower than wild-type BphP. The assembled Cys12 vari-\nants displayed a Pr-like aborption spectrum, which did\nnot alter upon red-light illumination (data not shown).\nThis observation is in contrast with data obtained for\nAgp1. An Agp1 C20A mutant was fully photoreversi-\nble, but had a reduced absorption coefﬁcient, a blue-\nshifted Pfr maximum, and a reduced ratio of Pfr to Pr\nabsorption [7]. In the case of PaBphP, the mutation of\nthis conserved cysteine residue is much more dramatic\nthan in Agp1. It seems that, in PaBphP, mutation of\nthis residue not only abolishes covalent binding, but\nstabilization of the Pfr form is also lost. This may be\ndue to a loosely or wrongly oriented biliverdin in the\nchromophore pocket. At this point it is worth men-\ntioning that, although many of our data point towards\nCys12 as the site of covalent chromophore attachment,\nit still cannot be ruled out that this cysteine residue\nonly plays a structural role (i.e. disulﬁde bond forma-\ntion). Consequently, its mutation would lead to a loss\nof the structural environment necessary for covalent\nbinding.\nPhotoconversion of BphPs involves 15Z ⁄ 15E\nisomerization\nit has always been\nSince the discovery of BphPs,\nassumed that the photochemical reaction is similar to\nthat found in plant phytochromes, which involves a\n15Z ⁄ 15E isomerization of the phytochromobilin chro-\nmophore [17]. This assumption has not yet been\nexperimentally conﬁrmed. We used a chromophore\nanalog that lacks the C15 ⁄ C16 double bond to investi-\ngate the photoisomerization of BphP. The phytoﬂuor\nthe\nadduct obtained conﬁrmed the involvement of\nC15 ⁄ C16 double bond in photoisomerization, as the\ndihydrobiliverdin adduct is highly ﬂuorescent. Further-\nmore, these data imply that, although the chromo-\nphore is attached at a different position, the geometry\nin PaBphP is\nof\nprobably similar to that of plant phytochromes.\nthe chromophore-binding pocket\nConclusion and outlook\nWe have shown that recombinant PaBphP has all\nfeatures of a ‘true’ phytochrome photoreceptor. It cova-\nlently binds a biliverdin chromophore, which, upon illu-\nmination with red and far-red light, photoisomerizes at\nthe C15 ⁄ C16 double bond. Furthermore, we conﬁrmed\nthat PaBphP is a histidine kinase. The results of this\nwork support the proposal of a separate bacterial phyto-\nchrome class with a new chromophore-binding site in\nthe P2 domain, although a solely structural role for this\nresidue cannot be completely ruled out. The function of\nBphPs in nonphotosynthetic micro-organisms remains a\nmystery. To elucidate this further, we have constructed\nchromosomal knock-out mutations in the P. aeruginosa\nbphOP operon, which are currently being investigated\nusing proteomic and transcriptomics analysis.\nExperimental procedures\nReagents\nAll chemicals were purchased from Sigma (Munich, Ger-\nmany) and were American Chemical Society grade or bet-\nter. Restriction enzymes were from Invitrogen (Cleveland,\nOH, USA). MasterTaqTM was purchased from Eppendorf\nScientiﬁc (Westbury, NY, USA). The expression vector\npASK-IBA3, Strep Tactin Sepharose, and anhydrotetra-\ncycline were obtained from IBA GmbH (Go¨ ttingen, Ger-\nmany). Centricon-10 concentrator devices were purchased\nfrom Amicon (Beverly, MA, USA). Biliverdin IXa was\nobtained from Frontier Scientiﬁc (Logan, UT, USA).\n15,16-Dihydrobiliverdin, biliverdin IXb, biliverdin IXc, bili-\nverdin IXd and phycocyanobilin were prepared as described\npreviously [15,20]. Biliverdin XIIIa, biliverdin IIIa, 31,32-\ndihydrobiliverdin and mesobiliverdin were gifts\nfrom\nJ.C. Lagarias (UC Davis, CA, USA) and K. Inomata\n(Kanazawa University, Japan) [21,26,27].\nConstruction of expression vectors\nThe P. aeruginosa bphP (PA 4117) gene was ampliﬁed by\nPCR from chromosomal DNA using a hot start proto-\ncol with the following primers, which contained the\nindicated and underlined restriction sites: bphPXbaRBSfwd:\n5¢-CGTCTAGATAACGAGGGCAAAAAATGACGAG\nCATCACCCGGTTACC-3¢;\n5¢-CC\nCTCGAGGGACGAGGAGCCGGTCTCCG-3¢. The PCR\nproduct was digested with the indicated enzymes and\ncloned into XbaI ⁄ XhoI-digested expression vector pASK-\nIBA3 (IBA). The integrity of the plasmid construct was veri-\nﬁed by DNA sequence determination of the insert (SeqLab,\nGo¨ ttingen, Germany). The resulting ORF encodes BphP\nwith a C-terminal Strep Tag with a total addition of 20\namino-acid residues under the control of a tet promoter.\nSite-directed mutagenesis of bphP was performed using the\nQuikChange site-directed mutagenesis kit (Stratagene, La\nJolla, CA, USA), according to the instructions of\nthe\nmanufacturer with the following primers (only one primer\nshown, second primer is complement, the underlined codon\nrepresents the introduced mutation). bphP C12A, 5¢-GGT\nTACCCTGGCGAACGCCGAGGACGAACCCATCC-3¢;\nbphP C12S 5¢-GGTTACCCTGGCGAACTCCGAGGAC\nGAACCCATCC-3¢; bphP H247Q, 5¢-GCAGCGTTTCG\nCCGATCCAGTGCGAATACCTGACC-3¢; bphP C248A,\n5¢-CGTTTCGCCGATCCACGCCGAATACCTGACCA\nAC-3¢ and bphP H513A, 5¢-GCGGTGCTCGGCGCCG\nACCTGCGCAAC-3¢. Mutants were also conﬁrmed by\nDNA sequencing (SeqLab).\nProtein production and puriﬁcation\nRecombinant P. aeruginosa BphP was produced using a\ntet promoter-driven Strep tag system ([28]; IBA) in the\nEscherichia coli strain DH5a and was grown at 37 C\nampicillin (100\nin Luria–Bertani medium containing\nlgÆmL)1) to an A578 of 0.5. Cultures were induced by the\naddition of 0.2 lgÆmL)1 anhydrotetracycline and incuba-\nted at 25 C overnight. The bacterial pellet from 3 L of\nculture was resuspended in lysis buffer (50 mm Tris ⁄ HCl,\npH 8.0, 100 mm NaCl, 0.05% Triton X-100) (3 mL buffer\nper g of cells) and disrupted by sonication. Cell debris\nwas removed by ultracentrifugation (30 min, 100 000 g),\nand the supernatant was subjected to a 40% (NH4)2SO4\ncut. The resultant pellet was dissolved in buffer W\n(20 mm\n1 mm\ndithiothreitol), and after 20 min centrifugation (23 000 g),\nthe supernatant was incubated with 40 lgÆmL)1 avidin\nleast 10 min on ice. The\n(ﬁnal concentration)\nresulting supernatant was loaded on to a Strep-Tactin\nSepharose column (5 mL), which had previously been\nequilibrated with buffer W. The puriﬁcation was per-\nformed according to the instructions\nsupplied by the\nmanufacturer (IBA). Fractions containing BphP were fur-\nther puriﬁed using anion-exchange chromatography on\nQ Sepharose (Amersham Biosciences) using a linear gra-\nin 50 mm Hepes ⁄ KOH, pH 8.0.\ndient of KCl\nBphP was eluted with 500 mm KCl from the Q Sepha-\nrose column.\n20 mm NaCl,\nfor at\nProtein determination\nProtein concentration was determined by the Bradford\nmethod with BSA as standard [29] or by measuring A280\nusing the calculated e280nm ¼ 78 457 m)1Æcm)1 for BphP\n[30].\nAnalytical gel permeation chromatography\nGel permeation chromatography experiments were carried\nout using a Superdex 200 HR10 ⁄ 30 column as described\npreviously [15].\nAssembly of PaBphP\nIn vitro chromophore assembly of PaBphP was tested using\n20 lm recombinant apo-BphP, which was incubated with\n40 lm chromophore for 30 min at room temperature in\nthe dark (ﬁnal volume 50 lL). Absorbance spectra were\nobtained after 3 min of incubation with red light at 630 nm\n(Pfr spectrum) and after 3 min of incubation with far red\nin a volume of 500 lL\nlight at 750 nm (Pr spectrum)\n(50 mm Hepes ⁄ HCl, pH 8.0, 20 mm KCl), and the differ-\nence was calculated.\nTo characterize the different forms of holo-BphP spec-\ntroscopically, absorbance spectra between 500 and 800 nm\nwere obtained. Biliverdin IXa (20 lm) was added to 10 lm\nBphP in a ﬁnal volume of 500 lL, and spectra were meas-\nured during incubation in the dark or during irradiation\nwith red and far red light, respectively.\nTo test covalent chromophore attachment to BphP, cova-\nlently bound bilins were visualized by zinc-induced red\nﬂuorescence as described previously [31]. For iodoacetamide\ntreatment, BphP apoprotein was mixed with different con-\ncentrations of the blocking reagent from a 5 mm stock solu-\ntion and incubated for 20 min at room temperature [32].\nSpectra and zinc-induced red ﬂuorescence were measured as\ndescribed above.\nFluorescence spectroscopy\nRoom temperature ﬂuorescence emission and excitation\nspectra were recorded using a Perkin–Elmer LS50B spectro-\nﬂuorimeter. Fluorescence spectra were measured at 570 nm\nexcitation (the absorption maximum of dihydrobiliverdin)\nor at 630 nm emission.\nBVR assay\nA BVR assay was used to characterize the complex of\nBphP–biliverdin IXa. BVR catalyzes the conversion of bili-\nverdin IXa into bilirubin IXa, which absorbs at 450 nm.\nBVR can only convert noncovalenty bound biliverdin IXa\ninto bilirubin. Apo-BphP was incubated with an excess of\nbiliverdin IXa for 30 min in the dark. The BphP–biliverdin\ncomplex was separated from free biliverdin IXa using\nNAP-5 desalting columns (Amersham Biosciences), which\nwere equilibrated with buffer (100 mm Tris ⁄ HCl, pH 8.7).\nThe concentration of protein-bound biliverdin IXa was\nmeasured spectroscopically and estimated using e680 ¼\n12 400 m)1Æcm)1 for free biliverdin IXa. An aliquot of\n 5 lg crude soluble protein extract of recombinant rat\nBVR was added to 20 lm biliverdin in a complex of BphP–\nbiliverdin in 100 mm Tris ⁄ HCl, pH 8.7. The reaction was\nstarted by the addition of an NADPH-regenerating system\ncontaining 6.5 mm glucose 6-phosphate, 0.82 mm NADP+\nand 1.1 UÆmL)1 glucose-6-phosphate dehydrogenase. Spec-\ntral changes between 300 and 800 nm were monitored.\nProtein kinase assays\nAutophosphorylation was performed as described for Cph1\n[6]. Holo-BphP was irradiated with saturating red (630 nm)\nor far-red (750 nm) light before the addition of [32P]ATP[cP]\nand subsequently incubated for 30 min at room temperature\nwith the corresponding light. Radioisotope imaging was\nmonitored using a Bio-Rad Molecular Imager FX.\nAcknowledgements\nWe are grateful to Drs Inomata (Kanazawa University,\nKanazawa, Japan) and Lagarias (UC Davis, Davis,\nCA, USA) for the gift of chromophores. We thank\nMaria Sowa and Thorben Dammeyer for technical\nassistance. This work was supported by the Emmy-\nNoether-Program of\nthe Deutsche Forschungsge-\nmeinschaft and funds from the Fonds der Chemischen\nIndustrie to N.F.-D.\n1 Davis SJ, Vener AV & Vierstra RD (1999) Bacteriophy-\ntochromes: phytochrome-like photoreceptors from non-\nphotosynthetic eubacteria. Science 286, 2517–2520.\n2 Quail PH, Boylan MT, Parks BM, Short TW, Xu Y &\nWagner D (1995) Phytochromes: photosensory percep-\ntion and signal transduction. Science 268, 675–680.\n3 Lagarias JC & Rapoport H (1980) Chromopeptides\nfrom phytochrome. The structure and linkage of the Pr\nform of the phytochrome chromophore. J Am Chem\nSoc 102, 4821–4828.\n4 Terry MJ, Wahleithner JA & Lagarias JC (1993) Bio-\nsynthesis of the plant photoreceptor phytochrome. Arch\nBiochem Biophys 306, 1–15.\n5 Hughes J, Lamparter T, Mittmann F, Hartmann E,\nGa¨ rtner W, Wilde A & Bo¨ rner T (1997) A prokaryotic\nphytochrome. Nature 386, 663.\n6 Yeh KC, Wu SH, Murphy JT & Lagarias JC (1997) A\ncyanobacterial phytochrome two-component light sen-\nsory system. Science 277, 1505–1508.\n7 Lamparter T, Michael N, Mittmann F & Esteban B\n(2002) Phytochrome from Agrobacterium tumefaciens\nhas unusual spectral properties and reveals an N-term-\ninal chromophore attachment site. Proc Natl Acad Sci\nUSA 99, 11628–11633.\n8 Karniol B & Vierstra RD (2003) The pair of bacterio-\nphytochromes from Agrobacterium tumefaciens are histi-\ndine kinases with opposing photobiological properties.\nProc Natl Acad Sci USA 100, 2807–2812.\n9 Bhoo SH, Davis SJ, Walker J, Karniol B & Vierstra\nRD (2001) Bacteriophytochromes are photochromic his-\ntidine kinases using a biliverdin chromophore. Nature\n414, 776–779.\n10 Wu SH & Lagarias JC (2000) Deﬁning the bilin lyase\ndomain: lessons from the extended phytochrome super-\nfamily. Biochemistry 39, 13487–13495.\n11 Aravind L & Ponting CP (1997) The GAF domain: an\nevolutionary link between diverse phototransducing pro-\nteins. Trends Biochem Sci 22, 458–459.\n20 Frankenberg N, Mukougawa K, Kohchi T & Lagarias\nJC (2001) Functional genomic analysis of the HY2\nfamily of ferredoxin-dependent bilin reductases from\noxygenic photosynthetic organisms. Plant Cell 13,\n965–978.\n21 Lamparter T, Michael N, Caspani O, Miyata T, Shirai\nK & Inomata K (2003) Biliverdin binds covalently to\nagrobacterium phytochrome Agp1 via its ring A vinyl\nside chain. J Biol Chem 278, 33786–33792.\n22 Jorissen HJ, Quest B, Remberg A, Coursin T, Braslav-\nsky SE, Schaffner K, de Marsac NT & Ga¨ rtner W\n(2002) Two independent, light-sensing two-component\nsystems in a ﬁlamentous cyanobacterium. Eur J Biochem\n269, 2662–2671.\n23 Quest B & Ga¨ rtner W (2004) Chromophore selectivity\nin bacterial phytochromes: dissecting the process of\nchromophore attachment. Eur J Biochem 271, 1117–\n1126.\n24 Smith H (2000) Phytochromes and light signal percep-\ntion by plants: an emerging synthesis. Nature 407,\n585–591.\n25 Lamparter T (2004) Evolution of cyanobacterial and\n12 Montgomery BL & Lagarias JC (2002) Phytochrome\nplant phytochromes. FEBS Lett 573, 1–5.\nancestry: sensors of bilins and light. Trends Plant Sci 7,\n357–366.\n13 Huang ZJ, Edery I & Rosbash M (1993) PAS is a\ndimerization domain common to Drosophila period and\nseveral transcription factors. Nature 364, 259–262.\n14 Lamparter T, Carrascal M, Michael N, Martinez E,\nRottwinkel G & Abian J (2004) The biliverdin chromo-\nphore binds covalently to a conserved cysteine residue\nin the N-terminus of Agrobacterium phytochrome Agp1.\nBiochemistry 43, 3659–3669.\n15 Wegele R, Tasler R, Zeng Y, Rivera M & Frankenberg-\nDinkel N (2004) The heme oxgygenase(s)-phytochrome\nsystem of Pseudomonas aeruginosa. J Biol Chem 279,\n45791–45802.\n16 Giraud E, Fardoux J, Fourrier N, Hannibal L, Genty\nB, Bouyer P, Dreyfus B & Vermeglio A (2002) Bacterio-\nphytochrome controls photosystem synthesis in anoxy-\ngenic bacteria. Nature 417, 202–205.\n17 Li L & Lagarias JC (1992) Phytochrome assembly.\nDeﬁning chromophore structural requirements for cova-\nlent attachment and photoreversibility. J Biol Chem 267,\n19204–19210.\n18 Li L, Murphy JT & Lagarias JC (1995) Continuous\nﬂuorescence assay of phytochrome assembly in vitro.\nBiochemistry 34, 7923–7930.\n19 Murphy JT & Lagarias JC (1997) The phytoﬂuors: a new\nclass of ﬂuorescent protein probes. Curr Biol 7, 870–876.\n26 Elich TD, McDonagh AF, Palma LA & Lagarias JC\n(1989) Phytochrome chromophore biosynthesis. Treat-\nment of tetrapyrrole-deﬁcient Avena explants with\nnatural and non-natural bilatrienes leads to formation\nof spectrally active holoproteins. J Biol Chem 264,\n183–189.\n27 Cornejo J, Beale SI, Terry MJ & Lagarias JC (1992)\nPhytochrome assembly: the structure and biological\nactivity of 2(R),3(E)-phytochromobilin derived from\nphycobiliproteins. J Biol Chem 267, 14790–14798.\n28 Skerra A (1994) Use of the tetracycline promoter for\nthe tightly regulated production of a murine antibody\nfragment in Escherichia coli. Gene 151, 131–135.\n29 Bradford MM (1976) A rapid and sensitive method for\nthe quantitation of microgram quantities of protein util-\nizing the principle of protein-dye binding. Anal Biochem\n72, 248–254.\n30 Gill SC & von Hippel PH (1989) Calculation of protein\nextinction coefﬁcients from amino acid sequence. Anal\nBiochem 182, 319–326.\n31 Berkelman TR & Lagarias JC (1986) Visualization of\nbilin-linked peptides and proteins in polyacrylamide\ngels. Anal Biochem 156, 194–201.\n32 Lamparter T, Esteban B & Hughes J (2001) Phyto-\nchrome Cph1 from the cyanobacterium Synechocystis\nPCC6803. Puriﬁcation, assembly, and quaternary struc-\nture. Eur J Biochem 268, 4720–4730.",
         "Ronja Tasler,T. Moises,N. Frankenberg‐Dinkel",
         null
        ],
        [
         "28",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "Abstract Two  subfamilies  of  the  polar  glycopeptidolipids\n(GPLs) located on the surface of\n,\nMycobacterium smegmatis\nalong with unknown phospholipids, were recently shown to\nparticipate in the nonopsonic phagocytosis of mycobacteria\nby human macrophages (Villeneuve, C., G. Etienne, V. Abadie,\nH. Montrozier, C. Bordier, F. Laval, M. Daffe, I. Maridonneau-\nParini, and C. Astarie-Dequeker. 2003. Surface-exposed gly-\ncopeptidolipids  of\nspeciﬁcally\ninhibit the phagocytosis of mycobacteria by human macro-\nphages. Identiﬁcation of a novel family of glycopeptidolipids.\n278: 51291–51300). As demonstrated herein, a\nJ. Biol. Chem.\nphospholipid mixture that derived from the methanol-insol-\nuble fraction inhibited the phagocytosis of\n. In-\nhibition was essentially attributable to phosphatidylinositol\nmannosides  (PIMs),  namely  PIM\n,  because  the\nand  PIM\n\n\npuriﬁed phosphatidylethanolamine, phosphatidylglycerol, and\nphosphatidylinositol  were  inactive.  This  was  further  con-\nﬁrmed  using  puriﬁed  PIM\nBCG\nM.  bovis\nfrom\nand  PIM\n\n\nthat  decreased  by  half  the  internalization  of\nM.  smegmatis\n.\nBoth compounds also inhibited the uptake of\nM. tuberculosis\nand\nbut had no effect on the internalization of zy-\nmosan used as a control particle of the phagocytic process.\nWhen coated on latex beads, PIM\nand polar GPL (GPL III)\n\nfavored the particle entry through complement receptor 3.\nGPL III, but not PIM\nalso directed particle entry through\n2,\nthe mannose receptor.\nTherefore, surface-exposed myco-\nbacterial  PIM  and  polar  GPL  participate  in  the  receptor-\ndependent  internalization  of  mycobacteria  in  human  mac-\nrophages.\n—Villeneuve, C., M. Gilleron, I. Maridonneau-Parini,\nMycobacte-\nM. Daffé, C. Astarie-Dequeker, and G. Etienne.\nria  use  their  surface-exposed  glycolipids  to  infect  human\nmacrophages through a receptor-dependent process.\nJ. Lipid\nRes.",
         "The outcome of mycobacterial infections is thought to\nbe  critically  dependent  on  the  nonopsonic  invasion  and\ncolonization of macrophages during primary infection of\ninhaled bacteria. This is why the initial step of mycobacte-\nria phagocytosis by macrophages received considerable at-\ntention, including efforts to delineate the molecular mech-\nanisms that underlie this process.\nNonopsonic  internalization  of  mycobacteria  in  macro-\nphages  can  be  mediated  by  different  kinds  of  receptors\nthat speciﬁcally recognize ligands expressed at the surface\nof bacilli (1, 2). Glycophosphoinositol-anchored receptors\nsuch as CD14 (3, 4), scavenger receptor (5), and transfer-\nrin receptor (6, 7) have been suggested as putative routes\nof entry for mycobacteria into macrophages. Additionally,\ncomplement  receptor  3  (CR3)  (8–10)  and  mannose  re-\nceptor (MR) (8, 11) are both well-known receptors for the\nbinding  and  internalization  of  mycobacteria  in  macro-\nphages.  Both  receptors  exhibited  lectin-like  properties.\nIndeed, MR is a monomeric transmembrane protein with\nan  extracellular  domain  containing  eight  carbohydrate\nrecognition  domains  with  C-type  lectin  properties  that\nrecognize mannose, fucose, and\n-acetylglucosamine resi-\ndues present on the cell surface of pathogens, such as my-\ncobacteria  (12–14).  CR3  is  a  heterodimeric  type  1  trans-\nmembrane  glycoprotein  consisting  of  a  CD18\nsubunit\nnoncovalently associated with the CD11b\nchain. The ex-\ntracellular portion of CD11b contained distinct functional\nbinding domains. One of these, called the I domain, is es-\nsential for the recognition and internalization of C3bi op-\nsonized  particles  (15,  16).  Another  one,  located  on  the\ncarboxyl-terminal  end  of  the  extracellular  portion  of\nCD11b, is a nonopsonic binding site with lectin properties\n(17). This latter domain is mainly involved in phagocytosis\nof  particles  containing\n-glucan,  such  as  zymosan  (18),\nbut also interacts with mannose, glucose, and\n-glu-\ncosamine (17). More recently, a second nonopsonic bind-\ning  domain  distinct  from  the  lectin  one  has  been  pro-\nposed to bind and internalize mycobacteria (19).\n-acetyl-\nmegmatis\nWe  recently  demonstrated  that  mycobacteria  express\nmolecules at their surface that play a role in their internal-\nization  by  human  macrophages  (20).  Using  a  biological\ntest  of  phagocytosis  inhibition,  we  found  that  lipids,  but\nnot proteins, isolated from the outermost layer of\nMycobac-\nterium\nby  a  mechanical  treatment  decrease  the\nnonopsonic phagocytosis of mycobacteria by human mac-\nrophages without affecting the internalization of particles\nsuch as zymosan or latex beads (20). A detailed analysis of\nthe lipid fractions has indicated that the major inhibitory\ncompounds  belong  to  the  glycopeptidolipid  (GPL)  and\nphospholipid  families  (20).  When  the  former  family  was\nchemically  and  structurally  analyzed,  several  inhibitory\nC-type GPLs were identiﬁed, among which a novel class of\nsuccinylated GPLs (\n) passed unnoticed until now (20).\nWe  have  also  proposed  that  their  capacity  to  inhibit  the\nphagocytosis  of  mycobacteria  is  dependent  on  common\nstructural features. Identiﬁcation of these molecules and\ntheir  putative  role  as  surface-exposed  ligands  of  phago-\ncytic receptors would help us understand the host-patho-\ngen interactions at the molecular level. The present study\nwas  designed  ﬁrst  to  identify  the  mycobacteria  surface-\nexposed  phospholipids  that  play  a  role  in  their  internal-\nization  by  human  macrophages  and  second  to  examine\nthe roles of GPLs and phospholipids as potential ligands\nof phagocytic receptors.\nMETHODS\nReagents and antibodies",
         "In the present study, we ﬁrst attempted to identify the\nsurface-exposed phospholipids implicated in the internal-\nization  of  mycobacteria.  For  this  purpose,  lipid  extract\nfrom\nwas ﬁrst enriched in phospholipids by\nmethanol precipitation. As depicted in\n, the metha-\nnol-insoluble fraction, analyzed by TLC, contained several\ncompounds: PIMs, in particular PIM\n(Fig. 1) as\nthe  main  subspecies,  PI  (Fig.  2A),  PG  (Fig.  2B),  and  PE\n(Fig. 2B). Examination of the blocking effect of the meth-\nanol-insoluble fraction on phagocytosis was the approach\nchosen  to  determine  the  role  of  phospholipids  in  the\nphagocytic process of mycobacteria. MDM were pretreated\nfor 15 min with the phospholipid-enriched extract at con-\ncentrations  ranging  from  10  pg  to  100\ng  lipid/ml  and\nthen put into contact with\nfor 45 min. At the\nlowest tested concentration (i.e., 10 pg lipid/ml), treated\nand untreated macrophages equally internalized\nM. smeg-\n\nmatis\n2). In con-\ntrast, increasing lipid concentration to 10 ng/ml resulted\n\nin a 30% decrease in the uptake of\nM. smegmatis\n6% to 45\n2). This inhibitory effect remained\nstable up to 100\n\ng/ml.\n(62\nPuriﬁcation and identiﬁcation of the active phospholipid\nfractions of\nM. smegmatis\nTo establish the nature of the active phospholipids, we\n\ng/ml\n. Prein-\ng/ml commercial-\nindividually tested the effect of the addition of 10\nPI, PE, and PG on the phagocytosis of\nM. smegmatis\n\ncubation of MDM for 15 min with 10\n3). Similarly, 10\nM. smegmatis\n\n4% and 49\ncompared with untreated cells (52\nized PI did not inﬂuence the internalization of\nM. smegma-\n\ntis\n5%,\nrespectively; n\ng/ml PE or PG, puri-\nﬁed  from  mycobacteria  cell  extracts  by  preparative  TLC,\ndid not affect the phagocytosis of\ncompared\n\nwith untreated macrophages (43\n3%, re-\n\nspectively,  vs.  50\n3).  These  data  strongly  sug-\ngested  that  the  observed  inhibitory  activity  of  the  phos-\npholipid-rich  methanol-insoluble  fraction  was  essentially\nattributable  to  PIM.  This  hypothesis  was  further  con-\nﬁrmed using PIM-enriched fractions obtained from a\nM.\nsmegmatis\nlipid  extract  as  follows:  methanol-insoluble  lip-\nids  were  fractionated  by  ion-exchange  chromatography\ninto 14 fractions eluted with increasing concentrations of\nammonium  acetate  in  chloroform-methanol.  TLC  and\nMALDI-TOF analysis showed that the ﬁrst 8 fractions con-\ntained  traces,  if  any,  of  phospholipids  (data  not  shown),\nwhereas  fractions  9–14  contained  essentially  PIM  and  PI\nFig. 3A\n(\n). Therefore, these last six fractions were tested on\nphagocytosis of\nat 10 ng lipid/ml, the low-\nest active concentration for the phospholipid mixture. As\nshown in Fig. 3B, the tested fractions exhibited an inhibi-\ntory  effect  of\n50%.  This  set  of  experiments  indicated\nthat PIMs were the compounds responsible for the activity\nof  the  phospholipid  mixture  and  therefore  were  the  es-\nsential  phospholipids  involved  in  the  phagocytic  process\nof mycobacteria.\nM. smegmatis\n(Ac\nsubfamilies\n(27).  As  depicted  in\nsubfamilies isolated from\nInhibition of mycobacterial internalization by PIM\nand PIM\nPIM\nsubfamilies  reduced  phagocytosis  of\nPIM\n\nand  triacylated  PIM\npathogenic species (Fig. 4B) comparable to what was ob-\nserved  with  nanomolar  concentrations  of\nM.  smegmatis\n(Fig. 4A). Nevertheless, these observations clearly demon-\nstrated that subfamilies of PIM are involved in the nonop-\nsonic phagocytic process of mycobacteria.\nRecognition of PIM and GPL by nonopsonic\nphagocytic receptors\nRecently,  we  reported  that  subfamilies  of  another  sur-\n,  GPL  (Fig.1),  in-\nface-exposed  glycolipid  of\nhibit the phagocytosis of mycobacteria (20). We have also\ndemonstrated  that  the  nonopsonic  phagocytosis  of  both\npathogenic  and  nonpathogenic  mycobacteria  by  human\nphagocytes is mediated at least in part by two major recep-\ntors, CR3 and MR (11, 29). Therefore, the ability of PIM\nand GPL to inhibit mycobacteria internalization by block-\ning their entry through these receptors was suspected and\ninvestigated. To this aim, latex beads were coated with pu-\nriﬁed PIM or GPL using the method previously described\nby  Schlesinger,  Hull,  and  Kaufman  (26).  As  PIM\nand\ntween the internalization of PIM2- or GPL- and BSA-coated\nbeads (data not shown). As expected, pretreatment of MDM\nwith 2LPM (5 g/ml) for 30 min at 37C signiﬁcantly de-\ncreased  the  internalization  of  serum-opsonized  and  non-\nopsonized zymosan but did not affect the phagocytosis of\nBSA-coated beads (Fig. 5A). Such pretreatment also reduced\nby approximately one-third the entry of both GPL III- and\nPIM2-coated beads but not that of GPL I (Fig. 5A), indicat-\ning  that  entry  of  PIM2-  and  GPL  III-coated  beads,  but  not\nthat of beads coated with GPL I, involves CR3.\nPIM6 displayed comparable activities on the phagocytosis\nof  mycobacteria,  PIM2,  the  quantitatively  major  constitu-\nent of PIM, was isolated from M. bovis BCG and used for\nthe  subsequent  experiment  as  representative  of  the  PIM\nfamily. Beads were also coated with either GPL III or GPL I\nfrom M. smegmatis, two classes of GPL that have been shown\nto exhibit different inhibitory activities on MDM (20).\nThe capacity of the coated beads to enter macrophages\nthrough MR or CR3 was then investigated using a compe-\ntition approach with blocking antibodies directed against\nreceptors or soluble ligands. Monoclonal antibody 2LPM\nwas selected for its capacity to prevent both opsonic and\nnonopsonic internalization through CR3 (19, 30, 31). Se-\nrum-opsonized  and  nonopsonized  zymosan  were  used  as\npositive CR3 binding particles (19) and BSA-latex beads as\na negative control. We did not observe any difference be-\nThe entry of beads through the MR was also evaluated\nusing a competitive ligand, Man-BSA (11), or a goat anti-\nhuman  MR  antiserum  selected  for  its  capacity  to  block\nMR-dependent  functions  (32,  33).  Man-BSA-  and  BSA-\ncoated latex beads were used as positive and negative con-\ntrols of MR-dependent phagocytosis, respectively (11). In\nthis  set  of  experiments,  GPL  III-coated  beads  were  inter-\nnalized as efﬁciently as beads coated with Man-BSA (45 \n5% vs. 33  7%; n  3), whereas PIM2-coated beads were\nslightly better internalized (58  1% vs. 33  7%; n  3).\nAs  expected,  pretreatment  of  MDM  with  anti-MR  antise-\nrum (1:50) or 300 g/ml Man-BSA for 30 min at 37C dra-\nmatically  reduced  the  internalization  of  Man-BSA-coated\nbeads  (Fig.  5B).  Such  pretreatments  did  not  affect  the\nphagocytosis of either GPL I- or PIM2-coated beads. In con-\ntrast,  both  treatments  slightly  but  signiﬁcantly  decreased\nthe  phagocytosis of GPL III-coated beads  (Fig.  5B), indi-\ncating that only GPL III-coated beads enter MDM through\nthe MR. Taken together, these data demonstrate that both\nPIM2 and GPL III participate to the internalization of my-\ncobacteria and thereby are potential ligands of phagocytic\nreceptors expressed at the surface of MDM.",
         "DISCUSSION\nThe  outermost  cell  envelope  layer,  also  called  a  cap-\nsule in the case of pathogenic mycobacteria, represents a\nprivileged interface between mycobacteria and host cells.\nSome  of  its  components,  such  as  glucan  (34)  and  GPL\n(20),  have  been  implicated  in  the  interaction  with  mac-\nrophages. Other constituents of the mycobacterial cell en-\nvelope  have  also  been  shown  to  induce  various  host  cell\nresponses (35), but their localization at the cell surface is\nstill a matter of debate. We show here that the outermost\nlayer  of  M.  smegmatis  contained  PIMs  that  also  partici-\npate in the nonopsonic phagocytosis of mycobacteria. The\nPIMs, described by Ballou and colleagues (36) in the 1960s,\nare  based  on  PI  and  are  formed  of  one  to  six  mannosyl\nresidues  attached  to  the  inositol  moiety  of  the  phospho-\nlipid. The PIMs usually occur as a mixture of compounds\ndiffering  from  one  another  by  the  number  of  mannosyl\nresidues and fatty acyl groups. The nonpathogenic strain\nM.  smegmatis  ATCC  607  has  been  described  to  contain\nmainly  two  PIM  subfamilies,  PIM2  and  PIM6  (24,  27),  in\nagreement  with  the  composition  of  the  crude  PIM  frac-\ntions reported herein.\nAlthough other forms of PIM have been reported in the\nliterature (24), they were undetectable under our experi-\nmental  conditions,  probably  because  they  occur  in  too\nsmall quantities. Therefore, the crude PIM-dependent in-\nhibition  of  mycobacterial  phagocytosis  was  essentially  at-\ntributable to PIM2 and PIM6. This was further conﬁrmed\nusing enriched PIM2 and PIM6 fractions isolated from M.\nbovis  BCG.  Nanomolar  concentrations  of  PIM  decreased\nthe internalization of M. smegmatis by MDM, whereas much\nhigher concentrations of lipids were required to affect M.\navium  or  M.  tuberculosis  phagocytosis  to  the  same  extent.\nPIMs  are  found  on  the  bacterial  surface  of  both  patho-\ngenic  and  nonpathogenic  mycobacteria  (27,  37).  How-\never, there are strain differences in the surface exposure\nof  PIMs  that  are  inversely  correlated  with  the  abundant\nexpression of capsular polysaccharides composed of -glu-\ncan that can partially mask capsular PIM and directly act\non  phagocytic  receptor,  such  as  CR3  (30).  If  we  assume\nthat PIMs, exposed at the surface of M. smegmatis, have a\nlower afﬁnity for the phagocytic receptors than the outer-\nmost polysaccharides of pathogenic species (38), this may\nexplain why different PIM concentrations are required to\ninhibit  to  a  comparable  extent  the  phagocytosis  of  M.\nsmegmatis and M. avium or M. tuberculosis.\nIt has to be noted that the occurrence of a carbohydrate\nportion on phospholipid is critical for the inhibitory activ-\nity, because PI displayed no effect but PIM2 did. However,\nsurprisingly,  the  length  of  the  oligomannosyl  chain  does\nnot appear to be important for the activity, because PIM2\nand PIM6 equally inhibited phagocytosis. Previous studies\nhave  reported  that  soya  PI  and,  to  a  lesser  extent,  myco-\nbacterial PI inhibited the nonopsonic binding of M. tuber-\nculosis to murine macrophages (39, 40). This is in apparent\nconﬂict with the data presented here. Such a discrepancy\ncould  be  attributed  to  interspecies  differences  in  the\nPI  mode  of  action  between  murine  and  human  macro-\nphages. Our data, together with our previous report (20),\ncollectively  indicate  that  surface-exposed  PIM  and  GPL\nparticipate in the internalization of mycobacteria by mac-\nrophages and, as such, are potential ligands of phagocytic\nreceptors.  It  is  possible,  however,  that  other  constituents\nof  the  mycobacterial  envelope,  such  as  glucan  and  un-\nidentiﬁed molecules, could also be involved in the interac-\ntions of bacilli with macrophages, as illustrated by the vari-\nety of routes of entry of mycobacteria into host cells.\nThe  molecular  mechanisms  by  which  surface-exposed\nGPLs  and  PIMs  participate  in  the  phagocytosis  of  myco-\nbacteria deserve consideration. PIMs have been proposed\nto  participate  in  the  receptor-mediated  uptake  of  myco-\nbacteria (26, 41). For instance, they have been  shown to\nact  as  an  adhesin,  allowing  the  binding  of  M.  tuberculosis\nand  M.  smegmatis  to  hamster  ovary  ﬁbroblasts,  either  di-\nrectly or after opsonization with mannose binding protein\n(41). Here, we demonstrate that coating latex beads with\nPIM2  favors  their  internalization  through  a  CR3-depen-\ndent process, probably through the lectin binding site of\nCR3, which might recognize the mannosyl moiety of PIM,\nas mentioned above. Like other mannosylated conjugates,\nPIMs have been suspected to be ligands of the MR. We ad-\ndressed  this  question  and  show,  for  the  ﬁrst  time,  that\nPIMs, at least PIM2, do not in fact interact with this recep-\ntor in our cell model. As mentioned above, the length of\nthe  oligomannosyl  chain  was  not  critical  for  PIM  inhibi-\ntion of phagocytosis, suggesting that PIM6 was not a part-\nner of MR. The mode of action of GPL as a putative ligand\nof  macrophages  has  been  poorly  investigated.  We  found\nthat  GPL  III  ensures  the  entry  of  particles  through  both\nCR3 and MR. Although all types of GPLs share the same\npeptidolipid core, GPL I, which is inactive, lacks the succi-\nnyl  substituent  that  occurs  in  GPL  III  and  is  3,4-O-acety-\nlated on the 6-deoxytalosyl moiety (20). At present, these\nmoieties have not been proposed among those described\nto be recognized by MR and CR3, and their putative bind-\ning  to  these  receptors  need  to  be  further  investigated.\nNonetheless,  we  propose  that  surface-exposed  PIMs  and\nGPLs coated on latex beads are potential ligands of CR3\nand/or MR, thereby mimicking part of the entry process\nof  mycobacteria  in  macrophages.  When  used  as  soluble\nmolecules,  PIMs  (present  study)  as  well  as  GPLs  (20)  af-\nfect the internalization of mycobacteria but not that of zy-\nmosan, whereas the two processes are well known to involve\nboth MR and CR3. Concerning CR3, the fact that zymosan\nand mycobacteria use distinct nonopsonic binding sites of\nthe receptor (19) suggests that the site involved in zymo-\nsan internalization does not recognize PIMs and GPLs.\nIn conclusion, we propose that surface-exposed GPLs and\nPIMs interact with the nonopsonic receptors CR3 and/or\nMR involved in the phagocytosis of mycobacteria. Charac-\nterization of their binding properties is beyond the scope\nof this report, but it would clearly help to further investi-\ngate  their  signaling  pathways.  Understanding  the  myco-\nbacterial  strategy  to  infect  macrophages  without  trigger-\ning  bactericidal  activity  is  still  a  challenge.  Finally,  PIMs\nand GPLs, as inhibitors of mycobacteria phagocytosis, may\nalso  help  to  design  new  pharmacological  drugs  for  the\ncontrol of macrophage infection.\nThe authors are grateful to Drs. G. Puzo and C. Guilhot [Insti-\ntut de Pharmacologie et Biologie Structurale (IPBS), Toulouse,\nFrance)]  for  critical  reading  of  the  manuscript  and  to  Drs.  F.\nLaval  and  H.  Montrozier  (IPBS)  for  technical  assistance  and\nhelpful discussions. This work was supported in part by Euro-\npean Community Grant QLK2CT 19990193.\n25. Laval,  F.,  M.  A.  Laneelle,  C.  Deon,  B.  Monsarrat,  and  M.  Daffe.\n2001. Accurate molecular mass determination of mycolic acids by\nMALDI-TOF mass spectrometry. Anal. Chem. 73: 4537–4544.\n26. Schlesinger, L. S., S. R. Hull, and T. M. Kaufman. 1994. Binding of\nthe  terminal  mannosyl  units  of  lipoarabinomannan  from  a  viru-\nlent strain of Mycobacterium tuberculosis to human macrophages. J.\nImmunol. 152: 4070–4079.\n37. Ortalo-Magne, A., A. B. Andersen, and M. Daffe. 1996. The outer-\nmost capsular arabinomannans and other mannoconjugates of vir-\nulent and avirulent tubercle bacilli. Microbiology. 142: 927–935.\n38. Lemassu, A., A. Ortalo-Magne, F. Bardou, G. Silve, M. A. Laneelle,\nand M. Daffe. 1996. Extracellular and surface-exposed polysaccha-\nrides  of  non-tuberculous  mycobacteria.  Microbiology.  142:  1513–\n1520.",
         "Chemical Co. (St Louis, MO). Polystyrene microspheres were from\nPolysciences, Inc. RPMI 1640, Ficoll-Hypaque,\n-glutamine, and\nantibiotics were purchased from Eurobio (Les Ulysses, France).\nMouse  antibodies  against  the  CD11b  extracellular  domain  was\nused [2LPM (1:20); IgG1\n; Dako, Glostrup, Denmark]. Goat anti-\n\n(\n);\nhuman MR was a generous gift of P. Stahl (1:50). Anti-mycobac-\nterium rabbit antibody Camelia (1:50) was obtained as previously\ndescribed (19). Secondary Abs were purchased from Sigma.\nH37Rv (ATCC\n27294)  were  grown  as  surface  pellicles  on  liquid  Sauton’s  me-\n\ndium at 37\n(serovar 4), known to\nform more clumps when grown as surface pellicles than the two\nformer species, was cultured under shaking at 250 rpm in Mid-\ndlebroock  7H9  medium  supplemented  with  10%  albumin-dex-\ntrose-catalase  enrichment  (Difco).  It  is  important  to  point  out\nthat the qualitative composition of the surface-exposed materials\nis independent of growth conditions (i.e., surface pellicles or\nshaken cultures) (21). Single cell suspensions were prepared with\nlate log-phase cultures (6 days for\nM. tuber-\nculosis\n) as previously described (20). Brieﬂy, pelli-\nM. avium\ncles (\n) were harvested by pouring\noff the medium, dispersed by gentle shaking for 30 s with 5 g of\nglass beads (4 mm diameter), and resuspended in PBS, pH 7.4.\nIn the case of\nM. avium\n, the culture was recovered by centrifuga-\ntion at 10,000\nfor 10 min. To remove the remaining clumps, the\ng\nbacterial suspensions were centrifuged for 10 min at 200\n. Up\nto  90%  of  mycobacteria  were  individualized,  and  their  viability\naveraged  85%.  When  required,  mycobacteria  were  labeled  with\nFITC, as previously described (22). We checked that their rate of\ninternalization and their route of entry within macrophages de-\nrived from monocytes (MDM) were identical to those obtained\nby  green  ﬂuorescent  protein-expressing\n(data  not\nshown), ruling out a side effect of FITC labeling.\nExtraction and puriﬁcation of glycolipids\nGlycolipids and phospholipids are present in both the outer-\nmost layer of the cell envelope and whole mycobacteria (21, 23).\nTherefore,  the  present  study  was  performed  on  lipid  extract\nfrom\nwhole cells to get enough material for further\nfractionation experiments (20). Mycobacterial wet cells were ex-\ntracted  with  CHCl\nOH  (1:2,  v/v)  and  then  at  last  three\n\ntimes  with  CHCl\nOH  (1:1,  v/v).  The  organic  phases  were\n/CH\n\npooled,  extensively  washed  with  water,  and  evaporated  to  dry-\nness. Lipids were then resuspended in a minimal volume of chlo-\nroform and precipitated with trickling methanol. After standing\n\nfor 2 h at 4\nC, methanol-insoluble lipids were recovered by cen-\n\ntrifugation at 4\n). Lipids were further sepa-\nrated using an anion-exchange QMA-silica gel (Chromabond SB;\nMacherey-Nagel). Phospholipids were eluted using a stepwise gra-\ndient: 0.1 M ammonium acetate in 33% methanol in chloroform\n(fractions 9 and 10), 0.2 M ammonium acetate in 33% methanol\nin chloroform (fractions 11–13), and 0.2 M ammonium acetate\nin  methanol  (fraction  14).  Puriﬁcation  of  phosphatidylethanol-\namine (PE) and phosphatidylglycerol (PG) was achieved by pre-\nparative  TLC.  Plates  were  developed  in  CHCl\nO\n\n(56:38:10, v/v/v); silica bands were scraped and extracted three\ntimes in the same solvent, adding trace amounts of acetic acid.\nThe  puriﬁcation  of  GPL  has  been  described  in  detail  (20).\nBrieﬂy, the methanol-soluble lipids were chromatographed on a\nFlorisil column irrigated with chloroform and then with a step-\nwise gradient of increasing concentrations of methanol and wa-\nter in chloroform. GPL I was eluted with CHCl\nOH (90:10,\n\nv/v), whereas GPL III was eluted in more polar fractions with\na  mixture  of  CHCl\nO  (65:25:4,  v/v/v);  the  latter\nAll of the puriﬁcation steps were monitored by TLC on silica\ngel  60-precoated  plates  (0.25  mm  thickness;  Merck)  developed\nwith  CHCl\nO  (60:35:8)  for  the  phospholipid-con-\n\ntaining  fractions.  Sugar-containing  compounds  were  visualized\nby  spraying  plates  with  1.5%  orcinol  in  30%  aqueous  sulfuric\nacid,  followed  by  heating,  whereas  the  Dittmer-Lester  reagent\nand  ninhydrin  were  used  to  detect  phosphorus-  and  amino\ngroup-containing  substances.  Phosphatidylinositol  mannosides\ntype 2 and 6 (PIM\n) (Fig. 1) were puriﬁed on a QMA-\n\nSpherosil M (BioSepra) column as previously described (24).\nMass spectrometry\nMatrix-assisted laser-desorption/ionization time of ﬂight (MALDI-\nTOF) mass spectrometry analysis of lipids was performed as previ-\nously described (25). MALDI-TOF spectra were acquired on a Voy-\nager-DE STR mass spectrometer (PerSeptive Biosystems) equipped\nwith  a  pulsed  nitrogen  laser  emitting  at  337  nm  and  were  ana-\nlyzed in the Reﬂectron mode using an extraction delay time set\nat 100 ns and an accelerating voltage operating in negative ion\nmode of 20 kV.\nHuman MDM\nHuman  peripheral  blood  monocytes  were  isolated  as  previ-\nously described (11) and cultured for 7 days on sterile glass cov-\nerslips on 24-well tissue culture plates (5\ncells/well) con-\ntaining RPMI medium with 10% heat-inactivated fetal calf serum\nin 5% CO\n. The culture medium was renewed on the third day.\n\nBefore use, MDM were washed twice with fresh RPMI medium.\nCoating of latex beads\nLatex  beads  were  coated  as  described  by  Schlesinger,  Hull,\n\nand Kaufman (26). Brieﬂy, 2\npolystyrene microspheres of\n\n\n, 10 min) in 0.05 M\ncarbonate-bicarbonate  buffer,  pH  9.6.  Beads  were  then  incu-\n\nbated for 1 h at 37\ng of\nvarious glycolipids, Man-BSA, or BSA in a reaction volume of 1\nml. The microspheres were washed twice, incubated in 5% BSA\n\nin  PBS  for  2  h  at  37\nC,  resuspended  in  5%  BSA  in  PBS,  and\n\n.\nstored at 4\nC until the experiments\nInfection of adherent macrophages and\nphagocytosis assay\n\nWhen speciﬁed, MDM were ﬁrst pretreated for 15 min at 37\nC\nwith the puriﬁed lipid fractions, which were previously resuspended\nin  sterile  apyrogenic  water  and  sonicated  for  10  min,  and  then\nadded with particles at the appropriate multiplicity of infection.\nAt  the  end  of  infection,  cells  were  washed  twice  with  fresh  me-\ndium to remove unbound particles. Phagocytosis of mycobacteria,\nzymosan,  and  latex  beads  was  assessed  by  immunoﬂuorescence\nas  previously  described  (11).  For  each  set  of  conditions,  dupli-\ncate experiments were performed, and at least 100 cells per slide\nwere  counted  by  ﬂuorescence  microscopy.  The  percentage  of\nphagocytic cells having ingested at least one particle or bacterium\nwas calculated as follows: number of phagocytic cells/number of to-\ntal cells\n100. Data are expressed as the percentage of phagocyto-\nsis inhibition determined as follows: number of phagocytic cells\nin  the  control  minus  number  of  phagocytic  cells  under  treated\nconditions/number of phagocytic cells in the control\nRESULTS\nInhibitory activity of surface-exposed phospholipids on\nthe phagocytosis of\nand PIM\nM. smegmatis",
         "Abstract Two  subfamilies  of  the  polar  glycopeptidolipids\n(GPLs) located on the surface of\n,\nMycobacterium smegmatis\nalong with unknown phospholipids, were recently shown to\nparticipate in the nonopsonic phagocytosis of mycobacteria\nby human macrophages (Villeneuve, C., G. Etienne, V. Abadie,\nH. Montrozier, C. Bordier, F. Laval, M. Daffe, I. Maridonneau-\nParini, and C. Astarie-Dequeker. 2003. Surface-exposed gly-\ncopeptidolipids  of\nspeciﬁcally\ninhibit the phagocytosis of mycobacteria by human macro-\nphages. Identiﬁcation of a novel family of glycopeptidolipids.\n278: 51291–51300). As demonstrated herein, a\nJ. Biol. Chem.\nphospholipid mixture that derived from the methanol-insol-\nuble fraction inhibited the phagocytosis of\n. In-\nhibition was essentially attributable to phosphatidylinositol\nmannosides  (PIMs),  namely  PIM\n,  because  the\nand  PIM\n\n\npuriﬁed phosphatidylethanolamine, phosphatidylglycerol, and\nphosphatidylinositol  were  inactive.  This  was  further  con-\nﬁrmed  using  puriﬁed  PIM\nBCG\nM.  bovis\nfrom\nand  PIM\n\n\nthat  decreased  by  half  the  internalization  of\nM.  smegmatis\n.\nBoth compounds also inhibited the uptake of\nM. tuberculosis\nand\nbut had no effect on the internalization of zy-\nmosan used as a control particle of the phagocytic process.\nWhen coated on latex beads, PIM\nand polar GPL (GPL III)\n\nfavored the particle entry through complement receptor 3.\nGPL III, but not PIM\nalso directed particle entry through\n2,\nthe mannose receptor.\nTherefore, surface-exposed myco-\nbacterial  PIM  and  polar  GPL  participate  in  the  receptor-\ndependent  internalization  of  mycobacteria  in  human  mac-\nrophages.\n—Villeneuve, C., M. Gilleron, I. Maridonneau-Parini,\nMycobacte-\nM. Daffé, C. Astarie-Dequeker, and G. Etienne.\nria  use  their  surface-exposed  glycolipids  to  infect  human\nmacrophages through a receptor-dependent process.\nJ. Lipid\nRes.\nThe outcome of mycobacterial infections is thought to\nbe  critically  dependent  on  the  nonopsonic  invasion  and\ncolonization of macrophages during primary infection of\ninhaled bacteria. This is why the initial step of mycobacte-\nria phagocytosis by macrophages received considerable at-\ntention, including efforts to delineate the molecular mech-\nanisms that underlie this process.\nNonopsonic  internalization  of  mycobacteria  in  macro-\nphages  can  be  mediated  by  different  kinds  of  receptors\nthat speciﬁcally recognize ligands expressed at the surface\nof bacilli (1, 2). Glycophosphoinositol-anchored receptors\nsuch as CD14 (3, 4), scavenger receptor (5), and transfer-\nrin receptor (6, 7) have been suggested as putative routes\nof entry for mycobacteria into macrophages. Additionally,\ncomplement  receptor  3  (CR3)  (8–10)  and  mannose  re-\nceptor (MR) (8, 11) are both well-known receptors for the\nbinding  and  internalization  of  mycobacteria  in  macro-\nphages.  Both  receptors  exhibited  lectin-like  properties.\nIndeed, MR is a monomeric transmembrane protein with\nan  extracellular  domain  containing  eight  carbohydrate\nrecognition  domains  with  C-type  lectin  properties  that\nrecognize mannose, fucose, and\n-acetylglucosamine resi-\ndues present on the cell surface of pathogens, such as my-\ncobacteria  (12–14).  CR3  is  a  heterodimeric  type  1  trans-\nmembrane  glycoprotein  consisting  of  a  CD18\nsubunit\nnoncovalently associated with the CD11b\nchain. The ex-\ntracellular portion of CD11b contained distinct functional\nbinding domains. One of these, called the I domain, is es-\nsential for the recognition and internalization of C3bi op-\nsonized  particles  (15,  16).  Another  one,  located  on  the\ncarboxyl-terminal  end  of  the  extracellular  portion  of\nCD11b, is a nonopsonic binding site with lectin properties\n(17). This latter domain is mainly involved in phagocytosis\nof  particles  containing\n-glucan,  such  as  zymosan  (18),\nbut also interacts with mannose, glucose, and\n-glu-\ncosamine (17). More recently, a second nonopsonic bind-\ning  domain  distinct  from  the  lectin  one  has  been  pro-\nposed to bind and internalize mycobacteria (19).\n-acetyl-\nmegmatis\nWe  recently  demonstrated  that  mycobacteria  express\nmolecules at their surface that play a role in their internal-\nization  by  human  macrophages  (20).  Using  a  biological\ntest  of  phagocytosis  inhibition,  we  found  that  lipids,  but\nnot proteins, isolated from the outermost layer of\nMycobac-\nterium\nby  a  mechanical  treatment  decrease  the\nnonopsonic phagocytosis of mycobacteria by human mac-\nrophages without affecting the internalization of particles\nsuch as zymosan or latex beads (20). A detailed analysis of\nthe lipid fractions has indicated that the major inhibitory\ncompounds  belong  to  the  glycopeptidolipid  (GPL)  and\nphospholipid  families  (20).  When  the  former  family  was\nchemically  and  structurally  analyzed,  several  inhibitory\nC-type GPLs were identiﬁed, among which a novel class of\nsuccinylated GPLs (\n) passed unnoticed until now (20).\nWe  have  also  proposed  that  their  capacity  to  inhibit  the\nphagocytosis  of  mycobacteria  is  dependent  on  common\nstructural features. Identiﬁcation of these molecules and\ntheir  putative  role  as  surface-exposed  ligands  of  phago-\ncytic receptors would help us understand the host-patho-\ngen interactions at the molecular level. The present study\nwas  designed  ﬁrst  to  identify  the  mycobacteria  surface-\nexposed  phospholipids  that  play  a  role  in  their  internal-\nization  by  human  macrophages  and  second  to  examine\nthe roles of GPLs and phospholipids as potential ligands\nof phagocytic receptors.\nMETHODS\nReagents and antibodies\nChemical Co. (St Louis, MO). Polystyrene microspheres were from\nPolysciences, Inc. RPMI 1640, Ficoll-Hypaque,\n-glutamine, and\nantibiotics were purchased from Eurobio (Les Ulysses, France).\nMouse  antibodies  against  the  CD11b  extracellular  domain  was\nused [2LPM (1:20); IgG1\n; Dako, Glostrup, Denmark]. Goat anti-\n\n(\n);\nhuman MR was a generous gift of P. Stahl (1:50). Anti-mycobac-\nterium rabbit antibody Camelia (1:50) was obtained as previously\ndescribed (19). Secondary Abs were purchased from Sigma.\nH37Rv (ATCC\n27294)  were  grown  as  surface  pellicles  on  liquid  Sauton’s  me-\n\ndium at 37\n(serovar 4), known to\nform more clumps when grown as surface pellicles than the two\nformer species, was cultured under shaking at 250 rpm in Mid-\ndlebroock  7H9  medium  supplemented  with  10%  albumin-dex-\ntrose-catalase  enrichment  (Difco).  It  is  important  to  point  out\nthat the qualitative composition of the surface-exposed materials\nis independent of growth conditions (i.e., surface pellicles or\nshaken cultures) (21). Single cell suspensions were prepared with\nlate log-phase cultures (6 days for\nM. tuber-\nculosis\n) as previously described (20). Brieﬂy, pelli-\nM. avium\ncles (\n) were harvested by pouring\noff the medium, dispersed by gentle shaking for 30 s with 5 g of\nglass beads (4 mm diameter), and resuspended in PBS, pH 7.4.\nIn the case of\nM. avium\n, the culture was recovered by centrifuga-\ntion at 10,000\nfor 10 min. To remove the remaining clumps, the\ng\nbacterial suspensions were centrifuged for 10 min at 200\n. Up\nto  90%  of  mycobacteria  were  individualized,  and  their  viability\naveraged  85%.  When  required,  mycobacteria  were  labeled  with\nFITC, as previously described (22). We checked that their rate of\ninternalization and their route of entry within macrophages de-\nrived from monocytes (MDM) were identical to those obtained\nby  green  ﬂuorescent  protein-expressing\n(data  not\nshown), ruling out a side effect of FITC labeling.\nExtraction and puriﬁcation of glycolipids\nGlycolipids and phospholipids are present in both the outer-\nmost layer of the cell envelope and whole mycobacteria (21, 23).\nTherefore,  the  present  study  was  performed  on  lipid  extract\nfrom\nwhole cells to get enough material for further\nfractionation experiments (20). Mycobacterial wet cells were ex-\ntracted  with  CHCl\nOH  (1:2,  v/v)  and  then  at  last  three\n\ntimes  with  CHCl\nOH  (1:1,  v/v).  The  organic  phases  were\n/CH\n\npooled,  extensively  washed  with  water,  and  evaporated  to  dry-\nness. Lipids were then resuspended in a minimal volume of chlo-\nroform and precipitated with trickling methanol. After standing\n\nfor 2 h at 4\nC, methanol-insoluble lipids were recovered by cen-\n\ntrifugation at 4\n). Lipids were further sepa-\nrated using an anion-exchange QMA-silica gel (Chromabond SB;\nMacherey-Nagel). Phospholipids were eluted using a stepwise gra-\ndient: 0.1 M ammonium acetate in 33% methanol in chloroform\n(fractions 9 and 10), 0.2 M ammonium acetate in 33% methanol\nin chloroform (fractions 11–13), and 0.2 M ammonium acetate\nin  methanol  (fraction  14).  Puriﬁcation  of  phosphatidylethanol-\namine (PE) and phosphatidylglycerol (PG) was achieved by pre-\nparative  TLC.  Plates  were  developed  in  CHCl\nO\n\n(56:38:10, v/v/v); silica bands were scraped and extracted three\ntimes in the same solvent, adding trace amounts of acetic acid.\nThe  puriﬁcation  of  GPL  has  been  described  in  detail  (20).\nBrieﬂy, the methanol-soluble lipids were chromatographed on a\nFlorisil column irrigated with chloroform and then with a step-\nwise gradient of increasing concentrations of methanol and wa-\nter in chloroform. GPL I was eluted with CHCl\nOH (90:10,\n\nv/v), whereas GPL III was eluted in more polar fractions with\na  mixture  of  CHCl\nO  (65:25:4,  v/v/v);  the  latter\nAll of the puriﬁcation steps were monitored by TLC on silica\ngel  60-precoated  plates  (0.25  mm  thickness;  Merck)  developed\nwith  CHCl\nO  (60:35:8)  for  the  phospholipid-con-\n\ntaining  fractions.  Sugar-containing  compounds  were  visualized\nby  spraying  plates  with  1.5%  orcinol  in  30%  aqueous  sulfuric\nacid,  followed  by  heating,  whereas  the  Dittmer-Lester  reagent\nand  ninhydrin  were  used  to  detect  phosphorus-  and  amino\ngroup-containing  substances.  Phosphatidylinositol  mannosides\ntype 2 and 6 (PIM\n) (Fig. 1) were puriﬁed on a QMA-\n\nSpherosil M (BioSepra) column as previously described (24).\nMass spectrometry\nMatrix-assisted laser-desorption/ionization time of ﬂight (MALDI-\nTOF) mass spectrometry analysis of lipids was performed as previ-\nously described (25). MALDI-TOF spectra were acquired on a Voy-\nager-DE STR mass spectrometer (PerSeptive Biosystems) equipped\nwith  a  pulsed  nitrogen  laser  emitting  at  337  nm  and  were  ana-\nlyzed in the Reﬂectron mode using an extraction delay time set\nat 100 ns and an accelerating voltage operating in negative ion\nmode of 20 kV.\nHuman MDM\nHuman  peripheral  blood  monocytes  were  isolated  as  previ-\nously described (11) and cultured for 7 days on sterile glass cov-\nerslips on 24-well tissue culture plates (5\ncells/well) con-\ntaining RPMI medium with 10% heat-inactivated fetal calf serum\nin 5% CO\n. The culture medium was renewed on the third day.\n\nBefore use, MDM were washed twice with fresh RPMI medium.\nCoating of latex beads\nLatex  beads  were  coated  as  described  by  Schlesinger,  Hull,\n\nand Kaufman (26). Brieﬂy, 2\npolystyrene microspheres of\n\n\n, 10 min) in 0.05 M\ncarbonate-bicarbonate  buffer,  pH  9.6.  Beads  were  then  incu-\n\nbated for 1 h at 37\ng of\nvarious glycolipids, Man-BSA, or BSA in a reaction volume of 1\nml. The microspheres were washed twice, incubated in 5% BSA\n\nin  PBS  for  2  h  at  37\nC,  resuspended  in  5%  BSA  in  PBS,  and\n\n.\nstored at 4\nC until the experiments\nInfection of adherent macrophages and\nphagocytosis assay\n\nWhen speciﬁed, MDM were ﬁrst pretreated for 15 min at 37\nC\nwith the puriﬁed lipid fractions, which were previously resuspended\nin  sterile  apyrogenic  water  and  sonicated  for  10  min,  and  then\nadded with particles at the appropriate multiplicity of infection.\nAt  the  end  of  infection,  cells  were  washed  twice  with  fresh  me-\ndium to remove unbound particles. Phagocytosis of mycobacteria,\nzymosan,  and  latex  beads  was  assessed  by  immunoﬂuorescence\nas  previously  described  (11).  For  each  set  of  conditions,  dupli-\ncate experiments were performed, and at least 100 cells per slide\nwere  counted  by  ﬂuorescence  microscopy.  The  percentage  of\nphagocytic cells having ingested at least one particle or bacterium\nwas calculated as follows: number of phagocytic cells/number of to-\ntal cells\n100. Data are expressed as the percentage of phagocyto-\nsis inhibition determined as follows: number of phagocytic cells\nin  the  control  minus  number  of  phagocytic  cells  under  treated\nconditions/number of phagocytic cells in the control\nRESULTS\nInhibitory activity of surface-exposed phospholipids on\nthe phagocytosis of\nand PIM\nM. smegmatis\nIn the present study, we ﬁrst attempted to identify the\nsurface-exposed phospholipids implicated in the internal-\nization  of  mycobacteria.  For  this  purpose,  lipid  extract\nfrom\nwas ﬁrst enriched in phospholipids by\nmethanol precipitation. As depicted in\n, the metha-\nnol-insoluble fraction, analyzed by TLC, contained several\ncompounds: PIMs, in particular PIM\n(Fig. 1) as\nthe  main  subspecies,  PI  (Fig.  2A),  PG  (Fig.  2B),  and  PE\n(Fig. 2B). Examination of the blocking effect of the meth-\nanol-insoluble fraction on phagocytosis was the approach\nchosen  to  determine  the  role  of  phospholipids  in  the\nphagocytic process of mycobacteria. MDM were pretreated\nfor 15 min with the phospholipid-enriched extract at con-\ncentrations  ranging  from  10  pg  to  100\ng  lipid/ml  and\nthen put into contact with\nfor 45 min. At the\nlowest tested concentration (i.e., 10 pg lipid/ml), treated\nand untreated macrophages equally internalized\nM. smeg-\n\nmatis\n2). In con-\ntrast, increasing lipid concentration to 10 ng/ml resulted\n\nin a 30% decrease in the uptake of\nM. smegmatis\n6% to 45\n2). This inhibitory effect remained\nstable up to 100\n\ng/ml.\n(62\nPuriﬁcation and identiﬁcation of the active phospholipid\nfractions of\nM. smegmatis\nTo establish the nature of the active phospholipids, we\n\ng/ml\n. Prein-\ng/ml commercial-\nindividually tested the effect of the addition of 10\nPI, PE, and PG on the phagocytosis of\nM. smegmatis\n\ncubation of MDM for 15 min with 10\n3). Similarly, 10\nM. smegmatis\n\n4% and 49\ncompared with untreated cells (52\nized PI did not inﬂuence the internalization of\nM. smegma-\n\ntis\n5%,\nrespectively; n\ng/ml PE or PG, puri-\nﬁed  from  mycobacteria  cell  extracts  by  preparative  TLC,\ndid not affect the phagocytosis of\ncompared\n\nwith untreated macrophages (43\n3%, re-\n\nspectively,  vs.  50\n3).  These  data  strongly  sug-\ngested  that  the  observed  inhibitory  activity  of  the  phos-\npholipid-rich  methanol-insoluble  fraction  was  essentially\nattributable  to  PIM.  This  hypothesis  was  further  con-\nﬁrmed using PIM-enriched fractions obtained from a\nM.\nsmegmatis\nlipid  extract  as  follows:  methanol-insoluble  lip-\nids  were  fractionated  by  ion-exchange  chromatography\ninto 14 fractions eluted with increasing concentrations of\nammonium  acetate  in  chloroform-methanol.  TLC  and\nMALDI-TOF analysis showed that the ﬁrst 8 fractions con-\ntained  traces,  if  any,  of  phospholipids  (data  not  shown),\nwhereas  fractions  9–14  contained  essentially  PIM  and  PI\nFig. 3A\n(\n). Therefore, these last six fractions were tested on\nphagocytosis of\nat 10 ng lipid/ml, the low-\nest active concentration for the phospholipid mixture. As\nshown in Fig. 3B, the tested fractions exhibited an inhibi-\ntory  effect  of\n50%.  This  set  of  experiments  indicated\nthat PIMs were the compounds responsible for the activity\nof  the  phospholipid  mixture  and  therefore  were  the  es-\nsential  phospholipids  involved  in  the  phagocytic  process\nof mycobacteria.\nM. smegmatis\n(Ac\nsubfamilies\n(27).  As  depicted  in\nsubfamilies isolated from\nInhibition of mycobacterial internalization by PIM\nand PIM\nPIM\nsubfamilies  reduced  phagocytosis  of\nPIM\n\nand  triacylated  PIM\npathogenic species (Fig. 4B) comparable to what was ob-\nserved  with  nanomolar  concentrations  of\nM.  smegmatis\n(Fig. 4A). Nevertheless, these observations clearly demon-\nstrated that subfamilies of PIM are involved in the nonop-\nsonic phagocytic process of mycobacteria.\nRecognition of PIM and GPL by nonopsonic\nphagocytic receptors\nRecently,  we  reported  that  subfamilies  of  another  sur-\n,  GPL  (Fig.1),  in-\nface-exposed  glycolipid  of\nhibit the phagocytosis of mycobacteria (20). We have also\ndemonstrated  that  the  nonopsonic  phagocytosis  of  both\npathogenic  and  nonpathogenic  mycobacteria  by  human\nphagocytes is mediated at least in part by two major recep-\ntors, CR3 and MR (11, 29). Therefore, the ability of PIM\nand GPL to inhibit mycobacteria internalization by block-\ning their entry through these receptors was suspected and\ninvestigated. To this aim, latex beads were coated with pu-\nriﬁed PIM or GPL using the method previously described\nby  Schlesinger,  Hull,  and  Kaufman  (26).  As  PIM\nand\ntween the internalization of PIM2- or GPL- and BSA-coated\nbeads (data not shown). As expected, pretreatment of MDM\nwith 2LPM (5 g/ml) for 30 min at 37C signiﬁcantly de-\ncreased  the  internalization  of  serum-opsonized  and  non-\nopsonized zymosan but did not affect the phagocytosis of\nBSA-coated beads (Fig. 5A). Such pretreatment also reduced\nby approximately one-third the entry of both GPL III- and\nPIM2-coated beads but not that of GPL I (Fig. 5A), indicat-\ning  that  entry  of  PIM2-  and  GPL  III-coated  beads,  but  not\nthat of beads coated with GPL I, involves CR3.\nPIM6 displayed comparable activities on the phagocytosis\nof  mycobacteria,  PIM2,  the  quantitatively  major  constitu-\nent of PIM, was isolated from M. bovis BCG and used for\nthe  subsequent  experiment  as  representative  of  the  PIM\nfamily. Beads were also coated with either GPL III or GPL I\nfrom M. smegmatis, two classes of GPL that have been shown\nto exhibit different inhibitory activities on MDM (20).\nThe capacity of the coated beads to enter macrophages\nthrough MR or CR3 was then investigated using a compe-\ntition approach with blocking antibodies directed against\nreceptors or soluble ligands. Monoclonal antibody 2LPM\nwas selected for its capacity to prevent both opsonic and\nnonopsonic internalization through CR3 (19, 30, 31). Se-\nrum-opsonized  and  nonopsonized  zymosan  were  used  as\npositive CR3 binding particles (19) and BSA-latex beads as\na negative control. We did not observe any difference be-\nThe entry of beads through the MR was also evaluated\nusing a competitive ligand, Man-BSA (11), or a goat anti-\nhuman  MR  antiserum  selected  for  its  capacity  to  block\nMR-dependent  functions  (32,  33).  Man-BSA-  and  BSA-\ncoated latex beads were used as positive and negative con-\ntrols of MR-dependent phagocytosis, respectively (11). In\nthis  set  of  experiments,  GPL  III-coated  beads  were  inter-\nnalized as efﬁciently as beads coated with Man-BSA (45 \n5% vs. 33  7%; n  3), whereas PIM2-coated beads were\nslightly better internalized (58  1% vs. 33  7%; n  3).\nAs  expected,  pretreatment  of  MDM  with  anti-MR  antise-\nrum (1:50) or 300 g/ml Man-BSA for 30 min at 37C dra-\nmatically  reduced  the  internalization  of  Man-BSA-coated\nbeads  (Fig.  5B).  Such  pretreatments  did  not  affect  the\nphagocytosis of either GPL I- or PIM2-coated beads. In con-\ntrast,  both  treatments  slightly  but  signiﬁcantly  decreased\nthe  phagocytosis of GPL III-coated beads  (Fig.  5B), indi-\ncating that only GPL III-coated beads enter MDM through\nthe MR. Taken together, these data demonstrate that both\nPIM2 and GPL III participate to the internalization of my-\ncobacteria and thereby are potential ligands of phagocytic\nreceptors expressed at the surface of MDM.\nDISCUSSION\nThe  outermost  cell  envelope  layer,  also  called  a  cap-\nsule in the case of pathogenic mycobacteria, represents a\nprivileged interface between mycobacteria and host cells.\nSome  of  its  components,  such  as  glucan  (34)  and  GPL\n(20),  have  been  implicated  in  the  interaction  with  mac-\nrophages. Other constituents of the mycobacterial cell en-\nvelope  have  also  been  shown  to  induce  various  host  cell\nresponses (35), but their localization at the cell surface is\nstill a matter of debate. We show here that the outermost\nlayer  of  M.  smegmatis  contained  PIMs  that  also  partici-\npate in the nonopsonic phagocytosis of mycobacteria. The\nPIMs, described by Ballou and colleagues (36) in the 1960s,\nare  based  on  PI  and  are  formed  of  one  to  six  mannosyl\nresidues  attached  to  the  inositol  moiety  of  the  phospho-\nlipid. The PIMs usually occur as a mixture of compounds\ndiffering  from  one  another  by  the  number  of  mannosyl\nresidues and fatty acyl groups. The nonpathogenic strain\nM.  smegmatis  ATCC  607  has  been  described  to  contain\nmainly  two  PIM  subfamilies,  PIM2  and  PIM6  (24,  27),  in\nagreement  with  the  composition  of  the  crude  PIM  frac-\ntions reported herein.\nAlthough other forms of PIM have been reported in the\nliterature (24), they were undetectable under our experi-\nmental  conditions,  probably  because  they  occur  in  too\nsmall quantities. Therefore, the crude PIM-dependent in-\nhibition  of  mycobacterial  phagocytosis  was  essentially  at-\ntributable to PIM2 and PIM6. This was further conﬁrmed\nusing enriched PIM2 and PIM6 fractions isolated from M.\nbovis  BCG.  Nanomolar  concentrations  of  PIM  decreased\nthe internalization of M. smegmatis by MDM, whereas much\nhigher concentrations of lipids were required to affect M.\navium  or  M.  tuberculosis  phagocytosis  to  the  same  extent.\nPIMs  are  found  on  the  bacterial  surface  of  both  patho-\ngenic  and  nonpathogenic  mycobacteria  (27,  37).  How-\never, there are strain differences in the surface exposure\nof  PIMs  that  are  inversely  correlated  with  the  abundant\nexpression of capsular polysaccharides composed of -glu-\ncan that can partially mask capsular PIM and directly act\non  phagocytic  receptor,  such  as  CR3  (30).  If  we  assume\nthat PIMs, exposed at the surface of M. smegmatis, have a\nlower afﬁnity for the phagocytic receptors than the outer-\nmost polysaccharides of pathogenic species (38), this may\nexplain why different PIM concentrations are required to\ninhibit  to  a  comparable  extent  the  phagocytosis  of  M.\nsmegmatis and M. avium or M. tuberculosis.\nIt has to be noted that the occurrence of a carbohydrate\nportion on phospholipid is critical for the inhibitory activ-\nity, because PI displayed no effect but PIM2 did. However,\nsurprisingly,  the  length  of  the  oligomannosyl  chain  does\nnot appear to be important for the activity, because PIM2\nand PIM6 equally inhibited phagocytosis. Previous studies\nhave  reported  that  soya  PI  and,  to  a  lesser  extent,  myco-\nbacterial PI inhibited the nonopsonic binding of M. tuber-\nculosis to murine macrophages (39, 40). This is in apparent\nconﬂict with the data presented here. Such a discrepancy\ncould  be  attributed  to  interspecies  differences  in  the\nPI  mode  of  action  between  murine  and  human  macro-\nphages. Our data, together with our previous report (20),\ncollectively  indicate  that  surface-exposed  PIM  and  GPL\nparticipate in the internalization of mycobacteria by mac-\nrophages and, as such, are potential ligands of phagocytic\nreceptors.  It  is  possible,  however,  that  other  constituents\nof  the  mycobacterial  envelope,  such  as  glucan  and  un-\nidentiﬁed molecules, could also be involved in the interac-\ntions of bacilli with macrophages, as illustrated by the vari-\nety of routes of entry of mycobacteria into host cells.\nThe  molecular  mechanisms  by  which  surface-exposed\nGPLs  and  PIMs  participate  in  the  phagocytosis  of  myco-\nbacteria deserve consideration. PIMs have been proposed\nto  participate  in  the  receptor-mediated  uptake  of  myco-\nbacteria (26, 41). For instance, they have been  shown to\nact  as  an  adhesin,  allowing  the  binding  of  M.  tuberculosis\nand  M.  smegmatis  to  hamster  ovary  ﬁbroblasts,  either  di-\nrectly or after opsonization with mannose binding protein\n(41). Here, we demonstrate that coating latex beads with\nPIM2  favors  their  internalization  through  a  CR3-depen-\ndent process, probably through the lectin binding site of\nCR3, which might recognize the mannosyl moiety of PIM,\nas mentioned above. Like other mannosylated conjugates,\nPIMs have been suspected to be ligands of the MR. We ad-\ndressed  this  question  and  show,  for  the  ﬁrst  time,  that\nPIMs, at least PIM2, do not in fact interact with this recep-\ntor in our cell model. As mentioned above, the length of\nthe  oligomannosyl  chain  was  not  critical  for  PIM  inhibi-\ntion of phagocytosis, suggesting that PIM6 was not a part-\nner of MR. The mode of action of GPL as a putative ligand\nof  macrophages  has  been  poorly  investigated.  We  found\nthat  GPL  III  ensures  the  entry  of  particles  through  both\nCR3 and MR. Although all types of GPLs share the same\npeptidolipid core, GPL I, which is inactive, lacks the succi-\nnyl  substituent  that  occurs  in  GPL  III  and  is  3,4-O-acety-\nlated on the 6-deoxytalosyl moiety (20). At present, these\nmoieties have not been proposed among those described\nto be recognized by MR and CR3, and their putative bind-\ning  to  these  receptors  need  to  be  further  investigated.\nNonetheless,  we  propose  that  surface-exposed  PIMs  and\nGPLs coated on latex beads are potential ligands of CR3\nand/or MR, thereby mimicking part of the entry process\nof  mycobacteria  in  macrophages.  When  used  as  soluble\nmolecules,  PIMs  (present  study)  as  well  as  GPLs  (20)  af-\nfect the internalization of mycobacteria but not that of zy-\nmosan, whereas the two processes are well known to involve\nboth MR and CR3. Concerning CR3, the fact that zymosan\nand mycobacteria use distinct nonopsonic binding sites of\nthe receptor (19) suggests that the site involved in zymo-\nsan internalization does not recognize PIMs and GPLs.\nIn conclusion, we propose that surface-exposed GPLs and\nPIMs interact with the nonopsonic receptors CR3 and/or\nMR involved in the phagocytosis of mycobacteria. Charac-\nterization of their binding properties is beyond the scope\nof this report, but it would clearly help to further investi-\ngate  their  signaling  pathways.  Understanding  the  myco-\nbacterial  strategy  to  infect  macrophages  without  trigger-\ning  bactericidal  activity  is  still  a  challenge.  Finally,  PIMs\nand GPLs, as inhibitors of mycobacteria phagocytosis, may\nalso  help  to  design  new  pharmacological  drugs  for  the\ncontrol of macrophage infection.\nThe authors are grateful to Drs. G. Puzo and C. Guilhot [Insti-\ntut de Pharmacologie et Biologie Structurale (IPBS), Toulouse,\nFrance)]  for  critical  reading  of  the  manuscript  and  to  Drs.  F.\nLaval  and  H.  Montrozier  (IPBS)  for  technical  assistance  and\nhelpful discussions. This work was supported in part by Euro-\npean Community Grant QLK2CT 19990193.\n25. Laval,  F.,  M.  A.  Laneelle,  C.  Deon,  B.  Monsarrat,  and  M.  Daffe.\n2001. Accurate molecular mass determination of mycolic acids by\nMALDI-TOF mass spectrometry. Anal. Chem. 73: 4537–4544.\n26. Schlesinger, L. S., S. R. Hull, and T. M. Kaufman. 1994. Binding of\nthe  terminal  mannosyl  units  of  lipoarabinomannan  from  a  viru-\nlent strain of Mycobacterium tuberculosis to human macrophages. J.\nImmunol. 152: 4070–4079.\n37. Ortalo-Magne, A., A. B. Andersen, and M. Daffe. 1996. The outer-\nmost capsular arabinomannans and other mannoconjugates of vir-\nulent and avirulent tubercle bacilli. Microbiology. 142: 927–935.\n38. Lemassu, A., A. Ortalo-Magne, F. Bardou, G. Silve, M. A. Laneelle,\nand M. Daffe. 1996. Extracellular and surface-exposed polysaccha-\nrides  of  non-tuberculous  mycobacteria.  Microbiology.  142:  1513–\n1520.",
         "C. Villeneuve,M. Gilleron,I. Maridonneau-Parini,M. Daffé,C. Astarie-Dequeker,G. Etienne",
         null
        ],
        [
         "29",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "Fetal growth restriction in developing countries has\nmultifactorial origins, but attempts to quantify the con-\ntribution attributable to each cause indicate that under-\nnutrition is likely to be the most important single factor\n(Kramer, 1987, 2003). Both acute and chronic exposures\ncan be critical, although it seems likely that acute energy\nor nutrient insufﬁciency has to be quite harsh before it has\nreproducibly demonstrable effects on the baby (Susser,\n1991). Chronic food shortages have their effect by creating\na small mother, which imposes ‘maternal uterine con-\nstraint’ on the fetus probably mediated through a complex\ninterplay between insulin-like growth factors and their\nbinding proteins (Kelsey et al. 1999). These constraints\nthe fetus does not outgrow the\nattempt\nmother’s capacity to carry it safely to term and to deliver it\nnaturally. Such growth-restricted babies may be ‘small but\nperfectly formed’, and it\nto assume that\nthey necessarily carry the same perinatal risks as a low-\nbirth-weight baby of a much larger mother (Wilcox, 2001;\nMelve & Skjaerven, 2003). They may be thought of as\nhaving a harmonious growth well adapted for their uterine\nenvironment. Since the uterine environment\nis itself a\nreﬂection of the historic dietary conditions of the family,\nthe baby can be viewed as well adapted to its recent\nhistory, and hence to its best available prediction of likely\nfuture circumstances.",
         "Problems arise when this harmony of growth is\ndisrupted. The best documented evidence of this dishar-\nmony comes from collaborative studies between Ranjan\nYajnik and colleagues in Pune, India and Caroline Fall and\ncolleagues in Southampton, UK (Bavdekar et al. 1999;\nYajnik, 2000; Yajnik et al. 2002a,b). They have demon-\nstrated a phenomenon that Yajnik (Yajnik et al. 2002a,b)\ndescribes as the ‘thin–fat’ baby syndrome (Prentice, 2003).\nThe Indian mothers are small but carry a disproportion-\nately high level of abdominal fat. Comparison between\nthe Indian and Southampton babies reﬂects a similar\npicture, with the Indian babies being much smaller than\ntheir UK counterparts in all dimensions except triceps and\nthe brunt of\nThrifty genotypes\nGeneralisability of the thrifty genotype concept\nprevalence of obesity and diabetes within some isolated\npopulations, particularly on Polynesian islands (West,\n1974; Joffe & Zimmet, 1998).\nthe\nhighly-susceptible islanders are the descendants of a few\nfounders who had been selected as the survivors of a long\nsea crossing in which their less-thrifty shipmates died of\nhunger. Such a view has been widely held by Western\nscientists, but refuted as ethnographically inaccurate by\nPolynesian historians who claim that their forebears were\nmasters of the sea and able to travel\nindeﬁnitely by\ncatching rainwater and eating ﬁsh and turtles (Prentice,\n2001).\nThe issue can be taken a step further and it can be\nargued that, irrespective of the validity or otherwise of the\nfounder-effect\ntheory in populations with very high\nsusceptibility to diabetes, the metabolic legacy of man-\nkind’s struggles against starvation has imparted a much\nwider inﬂuence on the human genome.\nit was the dawn of agriculture that\nheralded episodes of mass starvation (Diamond, 1993;\nPrentice, 2001). It is believed that before this time the\nhunter–gatherer forbears of man avoided starvation by\nexploiting a highly-varied diet, living in small sustainable\ngroups and being prepared to move to new hunting\ngrounds when the need arose. They undoubtedly would\nhave experienced hunger between successful kills, but\nwould have very rarely been exposed to prolonged\nstarvation. Agriculture, on the other hand, fuelled popula-\ntion growth and diminished dietary diversity (McCance,\n1975; Fagan, 2000). This situation is sustainable as long as\nclimatic conditions remain stable (and in the absence of\nwarfare) but is highly vulnerable to even a single year’s\npoor rain, a single night of unseasonable frost or a crop\npest or infection such as potato blight.\nA selection of the mass of historical evidence that\nrecords the inﬂuence of widespread catastrophic famine on\nthe human race has been documented in greater detail\nelsewhere (Prentice, 2001). These records document\nextreme privation, with frequent references to cannibalism\n(especially of children) as a means of survival. Some of the\nearliest and best-known references are the following:\n‘I am mourning on my high throne for the vast\nmisfortune, because the Nile ﬂood in my time has not\ncome for seven years! Light is the grain; there is a lack\nof crops and of all kinds of food. Each man has become\na thief\nto his neighbour. They desire to hasten and\ncannot walk. . . The counsel of the great ones in the\ncourt\nis but emptiness. Torn open are the chests of\nprovisions, but instead of contents there is air. Every-\nthing is exhausted’ The Stella of Famine; chiselled into\nEgyptian rock in the time of Tcheser about 2000BC\n‘All of Upper Egypt was dying of hunger, to such a\ndegree that everyone had come to eating his children\n. . . the entire country had become like a starved grass-\nhopper. . .’ The Sepulchers of Ankhtiﬁ 2180–60BC\n‘If there be a cutting down of the food offerings of the\ngods,\nthen a million men perish among mortals,\ncovetousness is practiced, the entire land is in a fury,\ngenes’ that at ﬁrst sight seems surprising,\nto\nbelieve in thrifty genes is to accept that the natural state of\naffairs is unthrifty; why might this situation exist?\ni.e.\nBelief in ‘thrifty genes’ implies that the normal state\nis unthrifty",
         "",
         "Conclusions\nThe twin concepts of ‘thrifty phenotype’ and ‘thrifty\ngenotype’ each have merit insofar as they are based on\nintuitively-attractive ideas that, in the broadest of terms,\nhave plenty of support within biology. Each theory,\nhowever, currently lacks the level of precise evidential\nsupport\nthat will be needed to transform them from\nhypothesis to accepted fact. In the continued search for\nsuch evidence, which may ultimately inform the under-\nstanding of\nindividual variability in susceptibility to\nobesity and hence to reﬁned therapeutic strategies, it is\nlikely that research in the developing world will be most\nfruitful. In the present review an attempt has been made\nto suggest some guidelines to inform a more efﬁcient\nsearch.\nMitchell SM, Hattersley AT, Knight B, Turner T, Metcalf BS,\nVoss LD, Davies D, McCarthy A, Wilkin TJ, Smith GD,\nBen-Shlomo Y & Frayling TM (2004) Lack of support for a\nrole of the insulin gene variable number of tandem repeats\nminisatellite (INS-VNTR) locus in fetal growth or type 2\ndiabetes-related intermediate traits in United Kingdom popula-\ntions. Journal of Clinical Endocrinology and Metabolism 89,\n310–317.\nOng KK, Petry CJ, Barratt BJ, Ring S, Cordell HJ, Wingate DL,\nPembrey ME, Todd JA & Dunger DB (2004a) Maternal-\nfetal\ninteractions and birth order inﬂuence insulin variable\nnumber of tandem repeats allele class associations with head size\nat birth and childhood weight gain. Diabetes 53, 1128–1133.\nOng KK, Phillips DI, Fall C, Poulton J, Bennett ST,\nGolding J, Todd JA & Dunger DB (1999) The insulin gene\nVNTR, type 2 diabetes and birth weight. Nature Genetics 21,\n262–263.",
         "",
         "Fetal growth restriction in developing countries has\nmultifactorial origins, but attempts to quantify the con-\ntribution attributable to each cause indicate that under-\nnutrition is likely to be the most important single factor\n(Kramer, 1987, 2003). Both acute and chronic exposures\ncan be critical, although it seems likely that acute energy\nor nutrient insufﬁciency has to be quite harsh before it has\nreproducibly demonstrable effects on the baby (Susser,\n1991). Chronic food shortages have their effect by creating\na small mother, which imposes ‘maternal uterine con-\nstraint’ on the fetus probably mediated through a complex\ninterplay between insulin-like growth factors and their\nbinding proteins (Kelsey et al. 1999). These constraints\nthe fetus does not outgrow the\nattempt\nmother’s capacity to carry it safely to term and to deliver it\nnaturally. Such growth-restricted babies may be ‘small but\nperfectly formed’, and it\nto assume that\nthey necessarily carry the same perinatal risks as a low-\nbirth-weight baby of a much larger mother (Wilcox, 2001;\nMelve & Skjaerven, 2003). They may be thought of as\nhaving a harmonious growth well adapted for their uterine\nenvironment. Since the uterine environment\nis itself a\nreﬂection of the historic dietary conditions of the family,\nthe baby can be viewed as well adapted to its recent\nhistory, and hence to its best available prediction of likely\nfuture circumstances.\nProblems arise when this harmony of growth is\ndisrupted. The best documented evidence of this dishar-\nmony comes from collaborative studies between Ranjan\nYajnik and colleagues in Pune, India and Caroline Fall and\ncolleagues in Southampton, UK (Bavdekar et al. 1999;\nYajnik, 2000; Yajnik et al. 2002a,b). They have demon-\nstrated a phenomenon that Yajnik (Yajnik et al. 2002a,b)\ndescribes as the ‘thin–fat’ baby syndrome (Prentice, 2003).\nThe Indian mothers are small but carry a disproportion-\nately high level of abdominal fat. Comparison between\nthe Indian and Southampton babies reﬂects a similar\npicture, with the Indian babies being much smaller than\ntheir UK counterparts in all dimensions except triceps and\nthe brunt of\nThrifty genotypes\nGeneralisability of the thrifty genotype concept\nprevalence of obesity and diabetes within some isolated\npopulations, particularly on Polynesian islands (West,\n1974; Joffe & Zimmet, 1998).\nthe\nhighly-susceptible islanders are the descendants of a few\nfounders who had been selected as the survivors of a long\nsea crossing in which their less-thrifty shipmates died of\nhunger. Such a view has been widely held by Western\nscientists, but refuted as ethnographically inaccurate by\nPolynesian historians who claim that their forebears were\nmasters of the sea and able to travel\nindeﬁnitely by\ncatching rainwater and eating ﬁsh and turtles (Prentice,\n2001).\nThe issue can be taken a step further and it can be\nargued that, irrespective of the validity or otherwise of the\nfounder-effect\ntheory in populations with very high\nsusceptibility to diabetes, the metabolic legacy of man-\nkind’s struggles against starvation has imparted a much\nwider inﬂuence on the human genome.\nit was the dawn of agriculture that\nheralded episodes of mass starvation (Diamond, 1993;\nPrentice, 2001). It is believed that before this time the\nhunter–gatherer forbears of man avoided starvation by\nexploiting a highly-varied diet, living in small sustainable\ngroups and being prepared to move to new hunting\ngrounds when the need arose. They undoubtedly would\nhave experienced hunger between successful kills, but\nwould have very rarely been exposed to prolonged\nstarvation. Agriculture, on the other hand, fuelled popula-\ntion growth and diminished dietary diversity (McCance,\n1975; Fagan, 2000). This situation is sustainable as long as\nclimatic conditions remain stable (and in the absence of\nwarfare) but is highly vulnerable to even a single year’s\npoor rain, a single night of unseasonable frost or a crop\npest or infection such as potato blight.\nA selection of the mass of historical evidence that\nrecords the inﬂuence of widespread catastrophic famine on\nthe human race has been documented in greater detail\nelsewhere (Prentice, 2001). These records document\nextreme privation, with frequent references to cannibalism\n(especially of children) as a means of survival. Some of the\nearliest and best-known references are the following:\n‘I am mourning on my high throne for the vast\nmisfortune, because the Nile ﬂood in my time has not\ncome for seven years! Light is the grain; there is a lack\nof crops and of all kinds of food. Each man has become\na thief\nto his neighbour. They desire to hasten and\ncannot walk. . . The counsel of the great ones in the\ncourt\nis but emptiness. Torn open are the chests of\nprovisions, but instead of contents there is air. Every-\nthing is exhausted’ The Stella of Famine; chiselled into\nEgyptian rock in the time of Tcheser about 2000BC\n‘All of Upper Egypt was dying of hunger, to such a\ndegree that everyone had come to eating his children\n. . . the entire country had become like a starved grass-\nhopper. . .’ The Sepulchers of Ankhtiﬁ 2180–60BC\n‘If there be a cutting down of the food offerings of the\ngods,\nthen a million men perish among mortals,\ncovetousness is practiced, the entire land is in a fury,\ngenes’ that at ﬁrst sight seems surprising,\nto\nbelieve in thrifty genes is to accept that the natural state of\naffairs is unthrifty; why might this situation exist?\ni.e.\nBelief in ‘thrifty genes’ implies that the normal state\nis unthrifty\nIt is unlikely that evolution has left man in a physiologi-\ncally-unthrifty state through simple carelessness. Instead, it\nis much more likely that there is a series of reasons for\nenergetic proﬂigacy, the advantages of which outweigh\ntheir energy costs. Some possible candidates are listed in\nTable 1; there are no doubt many more candidates.\nThe fructose-1,6-biphosphatase–phosphofructokinase\nshuttle is the most famous example of a so-called ‘futile\ncycle’ in which substrate is utilised without a net gain in\nATP. The favoured teleological explanation for\nthis\nit enhances metabolic\napparently ‘futile’ cycle is that\ncontrol enabling substrate to be rapidly provided, since one\nside of a rapidly-shuttling cycle can be shut down (hence\ndiverting substrate) much more rapidly than the other side\ncould be started up. Similar lines of argument are used to\nexplain why the extrahepatic extra-adipocyte fatty acid\ncycle circulates much more substrate than is required\n(Wolfe et al. 1990), and to explain the high rates of muscle\nglutamine cycling (Parry-Billings & Newsholme, 1991;\nNewsholme, 1994).\nSubstrate usage for thermoregulatory thermogenesis is\nan obvious reason to maintain an unthrifty state in small\nand cold-adapted animals, but\nlimited\nimportance in man.\nis probably of\nThe need to quench oxidant stress, especially in the\nmitochondria, may also require a proﬂigate use of reduced\nsubstrates. Brand and colleagues (Brand, 2000) have shown\nthat all mammalian organisms so far studied appear to\nhave a high rate of proton leakage across the mitochondrial\ninner membrane. In his ‘uncoupling to survive’ hypothesis\nBrand (2000) argues that this ‘inefﬁciency’ is necessary to\ncounteract\nthe high levels of reactive oxygen species\ngenerated during normal mitochondrial metabolism.\nAnother well-known reason that an organism might need\nto intentionally burn off energy is in order to concentrate\ndiets with inadequate levels of protein or micronutrients.\nThe multiple possible forms of thrift\nencapsulate\nCharacteristics predicted for a thrifty gene\nPossible candidates for metabolic thrifty genes\nWithin the areas listed in Table 3 several authors (for\nexample, see Auwerx, 1999; Corbo & Scacchi, 1999; Ong\n& Dunger, 2000; Kimm et al. 2002) have made claims for\nspeciﬁc ‘thrifty genes’. Some of these claims are mislead-\ning, and hence extremely unhelpful, since they apply the\nconcept in an imprecise manner. For instance, it has been\nstated that ‘. . .PPARg was shown to have a key role in\nadipogenesis and be a master controller of the ‘thrifty gene\nresponse’ leading to efﬁcient energy storage.’ (Auwerx,\n1999), but no evidence is provided to suggest that PPARg\nhas been selected for thrift or exhibits any of the deﬁning\ncharacteristics set out earlier. Hence, emphasis will be\ngiven to several candidates with more speciﬁc, albeit very\nearly, evidence that they may make good candidates for\nthrifty genes.\nGenes regulating fuel channelling\nNeel’s (1962) original ideas focused on the possibility that\nalterations in the insulin signalling pathway could mod-\nulate fuel channelling between utilization and storage, and\nhence provide a likely mechanism for rapid fuel deposition\nin times of plenty. (This general idea also underlies the\nHales & Barker (1992, 2001) thrifty phenotype concept.)\nThe entirety of\nthe insulin signalling pathways are\ncomplex and contain many possible candidate genes,\nalthough to date none has been reliably pin-pointed as\nsuch.\nAn interesting candidate that is currently being investi-\ngating in our Gambian studies is the insulin variable\nnumber of\ntandem repeats (INS-VNTR) microsatellite\n(Bell et al. 1984). This gene lies on the promoter region\nof the insulin gene and is upstream of the gene encoding\ninsulin-like growth factor 2. In non-African populations\nthere are only three main lineages (classes I, II and III),\nbased on the number of\nrepeat motifs, and class I\npredominates (Stead et al. 2003). Associations have been\nreported with both type 1 and type 2 diabetes (Bell et al.\n1984; Bennett et al. 1995; Ong et al. 1999). In contrast,\nAfrican populations possess at least twenty-one different\nvariants, with class III predominating (Stead et al. 2003).\ninterpretation is that\nThe uncoupling proteins 2 and 3\nbehaviour, for instance that UCP3 is up regulated not down\nthese UCP are not a\nregulated by fasting,\nimply that\nthe energy regulatory\nstraightforward component of\napparatus as had ﬁrst been anticipated (Dulloo & Samec,\n2001). Instead, they may have more esoteric roles such as\nthe quenching of oxidant stress. These latest insights may\nargue against their candidacy as thrifty genes.\nNotwithstanding this argument, there has been interest\nin the possibility of identifying thrifty traits within the\nUCP, but little has emerged. Suggestive evidence has been\nobtained that a UCP3 exon 5 variant is associated with\nresting energy expenditure in African-American\nlower\nwomen (but not white women) and hence may be a thrifty\nallele. This evidence was, however, based on a small\nsample size and, judging by the history of such genetic\nobservations,\nis unlikely to replicate in future studies\n(Kimm et al. 2002). Currently, there appear to be no other\nsimilar claims.\nApoE\nThe global distribution of apoE4 (the variant associated\nwith raised cardiovascular\nrisk) shows a signature of\nselection of the type predicted for a good functional thrifty\ngene candidate (Table 5). ApoE2 is most common in the\nMediterranean basin where agriculture is thought to have\nbegun. ApoE4 (the ancestral lineage) is more common\nin pigmies,\nthe Khoi San, Aboriginals, some Native\nAmericans and Lapps, leading Corbo and colleagues (Corbo\n& Scacchi, 1999) to speculate that it could represent a\nmore lipid-thrifty variant\nin\nsequestering cholesterol and has been better maintained\nin populations with a tenuous dietary supply.\n(This\nargument runs counter to the thesis that it is agricultural\nsocieties that are more susceptible to episodic famine; see\nalso the ‘famine’ quotes concerning the Mediterranean\nbasin listed earlier (see p. 155).)\nthat\nCorbo (Corbo et al. 2004a,b) argues that the selection\neffect may be mediated through the fact that the apoE4\nIn\nvariant\nfavours steroidogenesis and hence fertility.\nThe double puzzle of diabetes\n1623 (Appelby, 1978). The Tudor famine caused a 5-fold\nincrease in mortality and a coincident 3-fold drop in\nconceptions (as judged by the number of christenings). An\noutbreak of plague that occurred just 23 years later caused\na very similar increase in deaths, but no change in birth\nrate until a few years later.\nAny genetic trait that allows certain women to continue\nto reproduce in energy-depleted circumstances when others\naround them have stopped could be viewed as thrifty. The\nexample of apoE4 given earlier could be one such gene.\nFig. 1 illustrates how such genes could have been under\ncontinuous selection in addition to the episodic selection of\ncatastrophic famines. It shows the very strong seasonal\nvariation in birth rates in two populations exposed to\nin Bangladesh and The\nseasonal agricultural patterns,\nGambia. The precise reason for this striking variation is\nnot known with certainty. However, in The Gambia it is\nentirely consistent with the Frisch (1982) hypothesis of\nsuppressed fertility during times of maternal energy\ndepletion, since the lowest birth rate falls exactly 9 months\nafter the time of greatest maternal weight loss during each\nyear’s hungry season. The implication is that there are\nsome women who manage to conceive all year round,\nwhilst others are constrained to the more energy-replete\nseason of the year. Do these women have a thrifty trait?\nDoes this trait result in greater overall fertility? These\nissues are currently under analysis.\nselection by famine. This aspect\nThere is an additional reason to believe that genes\ninﬂuencing reproduction and fetal well-being are likely\ncandidates\nis\nsome available\nillustrated in Fig. 2, which shows\napproximations for oocyte, embryo and fetal\nlosses in\nnormal human pregnancies. It shows the little-appreciated\nfact that the majority of natural selection of human genes\nprobably occurs before birth. If this concept is valid, then\ngenes favouring fetal survival will be under very strong\nselective pressure. This explanation would be consistent\nwith some of the previous observations that metabolic\nplasticity is probably maximised during pregnancy (Pre-\nntice & Goldberg, 2000).\nConclusions\nThe twin concepts of ‘thrifty phenotype’ and ‘thrifty\ngenotype’ each have merit insofar as they are based on\nintuitively-attractive ideas that, in the broadest of terms,\nhave plenty of support within biology. Each theory,\nhowever, currently lacks the level of precise evidential\nsupport\nthat will be needed to transform them from\nhypothesis to accepted fact. In the continued search for\nsuch evidence, which may ultimately inform the under-\nstanding of\nindividual variability in susceptibility to\nobesity and hence to reﬁned therapeutic strategies, it is\nlikely that research in the developing world will be most\nfruitful. In the present review an attempt has been made\nto suggest some guidelines to inform a more efﬁcient\nsearch.\nMitchell SM, Hattersley AT, Knight B, Turner T, Metcalf BS,\nVoss LD, Davies D, McCarthy A, Wilkin TJ, Smith GD,\nBen-Shlomo Y & Frayling TM (2004) Lack of support for a\nrole of the insulin gene variable number of tandem repeats\nminisatellite (INS-VNTR) locus in fetal growth or type 2\ndiabetes-related intermediate traits in United Kingdom popula-\ntions. Journal of Clinical Endocrinology and Metabolism 89,\n310–317.\nOng KK, Petry CJ, Barratt BJ, Ring S, Cordell HJ, Wingate DL,\nPembrey ME, Todd JA & Dunger DB (2004a) Maternal-\nfetal\ninteractions and birth order inﬂuence insulin variable\nnumber of tandem repeats allele class associations with head size\nat birth and childhood weight gain. Diabetes 53, 1128–1133.\nOng KK, Phillips DI, Fall C, Poulton J, Bennett ST,\nGolding J, Todd JA & Dunger DB (1999) The insulin gene\nVNTR, type 2 diabetes and birth weight. Nature Genetics 21,\n262–263.",
         "A. Prentice,P. Rayco-Solon,S. Moore",
         null
        ],
        [
         "30",
         "00206ca6a32791adcca5effdfed6eecd035075f4",
         null,
         null,
         null,
         null,
         null,
         null,
         "A. Atiya,Mohamed A. Aly,A. Parlos",
         "captcha"
        ],
        [
         "31",
         "0020d5bdfc5f13026e7a1aca78e94724e8342b62",
         null,
         null,
         null,
         null,
         null,
         null,
         "K. Turcotte,S. Gauthier,A. Tuite,A. Mullick,D. Malo,P. Gros",
         "captcha"
        ],
        [
         "32",
         "0022e70878c2f2a7c3c679a3ef3160a5cb803e20",
         null,
         null,
         null,
         null,
         null,
         null,
         "Jing Huang,Xiaodong Li,C. Maguire,R. Hilf,R. Bambara,M. Muyan",
         "captcha"
        ],
        [
         "33",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "",
         "",
         "",
         "",
         "",
         "",
         "Manish Kushwaha,K. Molnár,J. Sallai,P. Völgyesi,M. Maróti,Á. Lédeczi",
         null
        ],
        [
         "34",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "and  the  Melissa  virus  (in  1999)  occurred  when  hackers  exploited  flaws  in  software.  The\ndamage  due  to  Code  Red  was  estimated  at  $2.1  bn  and  due  to  Melissa  at  $1.1bn3.  The\nGartner  Group  estimates  that  the  system  downtime  caused  by  security  vulnerabilities\nwould triple from 5% of the total downtime in 2004 to 15% of the total downtime in 2008.4\nIn  2003,  Computer  Emergency  Response  Team  (CERT)  reported  around  250,000  self\nreported incidences of system breaches, most of which exploit vulnerabilities in software\ncode  (Applewhite  2004).  Microsoft’s  $200m  campaign  for  .NET  was  marred  by  the\ndiscovery of a security flaw in Visual C++ .NET barely a month after Microsoft Chairman\nBill  Gates  directed  employees  to  focus  on  building  more  secure  software  (dubbed  the\n‘Trustworthy Computing’ initiative). Moreover, vulnerability disclosure is finding its way\ninto  firms’  strategy  toolkits  as  it  evident  from  a  WSJ  report  in  Feb  2004  that  software\nvendors are spending time and effort in discovering flaws in their rivals’ software products\nin  order  to  influence  the  rivals’  stock  prices.  For  example,  security  software  vendor  IDS\nreleased a vulnerability alert on rival Checkpoint’s firewall software on the day Checkpoint\nusually  holds  its  annual  US  investor  conference.  Some  examples  of  vulnerability\nannouncements reported in popular press are:\n•  News.com(04/25/2000)  “A  computer  security\nfirm  has  discovered  a  serious\nvulnerability  in  Red  Hat’s  newest  version  of  Linux  that  could  let  attackers  destroy  or\ndeface a Web site  - or possibly even take over the machine itself……..”\n•  WSJ(02/11/2004) “Microsoft Corp. warned customers about serious security problems\nwith  its  Windows  software  that  let  hackers  quietly  break  into  their  computers  to  steal\nfiles, delete data or eavesdrop on sensitive information……..”\nInspite  of  all  these  concerns  about  software  vulnerabilities,  not  much  has  been\nmentioned  in  literature  about  the  incentive  of  software  vendors  to  invest  in  defect-free\nsoftware. Literature on software risks fails to include any measure for security related risks\n(Wallace, Keil and Rai, 2004; Barki, Rivard and Talbot, 1993). The closest literature to the\ntopic of security vulnerabilities is that on software quality (Basili and Musa, 1991; Harter,\nKrishnan  and  Slaughter,  2000).  However,  in  traditional  literature,  quality  is  measured  in\nterms  of  reliability  and  integrity  of  the  source  code  –  which  essentially  tests  software\nagainst specified streams of input from users. However, in today’s internet age, software\ndesigners  must  not  only  think  of  users,  but  also  malicious  adversaries  (Devanbu  and\nStubblebine,  2000).  Some  quality  models  such  as  ISO9126  fail  to  include  computer\nsecurity  (Pfleegar  1997).  Therefore  software  which  has  been  certified  as  high  quality,\nbased  on  existing  definitions  of  software  quality,  can  have  many  security  flaws.\nResearchers in computer science are working on better integration of the two disciplines of\nsoftware  quality  and  software  security  while  designing  software  (Carr,  Tynan  and  Davis\n1991, Wang and Wang, 2003; McGraw 2004).\nSo far, there has only been anecdotal evidence that software vulnerabilities are causing\nvendors to lose market value. For example, the Wall Street Journal (11/09/2004) reported\nthat Microsoft’s Internet Explorer (IE) is losing market share in the web browser market to\ncompetitors like Mozilla’s Firefox, due to numerous flaws discovered in IE. For example,\nnearly  8  million  people  have  downloaded  the  Firefox  browser  between  September-\nNovember 2004. While the damage that firms suffer as a result of security breaches can be\nmeasured  in  terms  of  downtime  and  maintenance  activity,  the  cost  implication  of\nvulnerability disclosures on the software vendors is neither clear nor well studied.  This is\nthe major goal of this paper. Using an event study methodology, we estimate how the stock\nprice of the software vendors is affected when vulnerability information in their products is\npublished in popular press.\nPrior  literature  on  product  defects  (Jarrell  and  Peltzman  1985)  predicts  that  product\nrecall announcements in drug and auto industries are associated with loss in market value\nof a firm. Davidson and Worrell (1992) confirm the negative impact of product defects on\nstock prices in non-auto industries and also analyze the impact of different types of recall\n(e.g.  total  product  recall  versus  a  recall  for  repair;  government  ordered  recall  versus\nvoluntary recall). However the results of these prior studies on product defects cannot be\ndirectly  applied  to  the  software  industry  because  of  the  following  characteristics  of\nsoftware products: One, software products generally come with a click-wrap agreement (or\nEULA  –  End  User  License  Agreement)  which  limits  the  vendors’  liability.  Two,  the\ngeneral philosophy held by software vendors, software customers and the US courts is that\nsoftware is a uniquely complex product that will probably have some defects (Cusumano\n2004).  Therefore  it  is  not  clear  whether  markets  will  react  adversely  to  the  news  of  a\nsoftware  vulnerability;  because  over  the  long  run,  markets  will  anticipate  the  effect  of  a\nvulnerability announcements on cash flows of software vendors, so the impact of a specific\nannouncement  might  not  be  significant.  Three, software vulnerability announcements are\ngenerally  accompanied  with  a  remedial  patch  which  potentially  protects  customers  from\nmalicious  exploits.  Hence,  the  market  may  not  behave  adversely  towards  a  vendor  if  it\nreleases  a  patch  along-with  the  vulnerability  announcement.    Finally,  vulnerability\nannouncements  are  directly  related  to  the  installed  base  of  a  software  product.  Popular\nsoftware like those from Microsoft are constantly subject to malicious and non-malicious\nattacks  and  as  such  have  a  greater  proportion  of  flaws  reported  in  them  as  compared  to\nsoftware by Apple, where the user base is comparatively smaller. Therefore, the presence\nof  vulnerabilities  may  not  always  signal  a  lower  quality  product;  it  may  in-fact  signal\nsuperiority  over  competition.  E.g.  John  Thomson,  CEO  of  Symantec,  predicts  that  the\nflaws  in  Linux  will  likely  increase  as  the  installed  base  increases.  In  view  of  these\narguments, it would be interesting to understand whether and how the market responds to\nvulnerability disclosures in software products.\nMotivated  by  these  observations,  in  this  research  we  try  to  quantify  the  losses  that\nsoftware  vendors  bear  when  a  vulnerability  is  disclosed  in  their  product.  The  main\nquestions that we seek to answer are:\n1.  What  is  the  impact  of  vulnerability  disclosures  on  the  market  value  of  a  software\n2.  How do the characteristics of the vulnerability impact this change in market value?\nOur  research  has  important  policy  implications  in  terms  of  understanding  vendors’\nincentive to improve pre- and post-launch quality of their software products.  While there\nis lot of hype surrounding the poor quality dished out by software vendors, their incentives\nto provide more secure software is still unclear. If we indeed find that the market is willing\nto  punish  vulnerability  announcements  because  it  perceives  these  announcements  as  a\nsignal of poor quality software which either increase vendor costs to fix them or erode their\nmarket share in the long run, then our research provides a direct evidence of incentives to\nprovide  more  secure  software.  Vulnerabilities  are  disclosed  by  vendors  or  by  third\nparty/competitors  with  or  without  a  patch.  Since  we  measure  if  there  are  significant\ndifferences in market reaction to such disclosures, our paper provides policy guidelines to\nvendors about whether and how they may disclose the information themselves. Moreover,\nthis  also  provides  evidence  on  what  is  the  value  of  patches  when  vulnerabilities  are\ndisclosed.  Finally,  we  also  examine  how  vulnerability  characteristics,  effect  of  9/11  etc\naffect the market reaction.\nUsing  an  event-study  approach,  we  collect  data  on  about  146  vulnerability  disclosure\nannouncements over the period of over 5 years for 18 publicly traded vendors. Our results\nconfirm  that  vulnerability  disclosure  adversely  and  significantly  affects  the  stock\nperformance  of  a  software  vendor.  We  show  that,  on  average,  a  software  vendor  loses\naround  0.63%  of  market  value  on  the  day  of  the  vulnerability  announcement.  This\ntranslates  to  a  dollar  amount  of  $0.86  bn  loss  in  market  value.  We  also  find  that\nvulnerabilities  where  the  vendor  fails  to  provide  a  patch  at  the  time  of  disclosure  yield\nmore negative returns than when the vendor discloses a patch. On average, a vendor loses\n0.8% more in market value when a patch is not available. This provides evidence as to why\nvendors are trying to push for legalizing the limited disclosure norms. We also show that\nvulnerabilities  which  create  confidentiality  related  breaches  result  in  a  greater  loss  than\nvulnerabilities  which  create  non-confidentiality  related  breaches.  For  example,  vendors\nlose  0.75%  less  in  market  value  when  the  vulnerability  can  lead  to  a  non-confidentiality\nrelated breach than when it can lead to a confidentiality related breach. We also find that\nmore severe software vulnerabilities result in a greater loss in market value than less severe\nvulnerabilities.\nThe rest of the paper is organized as follows. In the next section, section 2, we provide\na literature review. We develop our hypotheses in Section 3. In Section 4, we discuss the\nmethodology of data collection and also describe the event study methodology. In Section\n5, we present our results using multivariate regression analysis and test various hypothesis\nrelated  to  how  the  vulnerability  characteristics  effect  the  change  in  stock  prices.  Finally,\nwe present the concluding remarks in Section 6.\n2. Literature Review\nMost  prior  research  on  information  security  discusses  the  economics  of  such\ninvestments from a customer perspective, rather than from a software vendor perspective\n(Anderson  2001).  Gordon  and  Loeb  (2002)  show  that  firms  should  make  investments  in\ninformation  security  far  less  than  the  expected  loss  from  a  security  breach.  Gordon  et  al\n(2002),  Gal-Or  and  Ghose  (2003)  discuss  the  economics  of  information  sharing  among\nfirms  on  security  related  issues.  Prior  event  study  analyses  on  information  security  have\nfocused on the change in market value of firms whose systems are breached (Cavusoglu et\nal (2004) and Kannan, Rees and Sridhar (2004)). These studies show that announcements\nof  a  security  breach  negatively  impact  the CAR  (Cumulative  Abnormal  Return)  of  firms\nwhose information systems have been breached. Campbell et al (2003) conduct a similar\nevent  study  and  find  that  only  the  impact  of  confidentiality  related  security  breaches  is\nnegative and significant; the impact of non-confidentiality related security breaches is not\nsignificantly  different  from  zero.    Hovav  and  D’Arcy  (2003)  show  similar  results  by\nfinding  that  Denial  of  Service  (DoS)  type  attacks  are  not  associated  with  any  significant\nloss in value for firms5.\nDisclosure of vulnerabilities has been one contentious area. Typically, a major portion\nof the vulnerabilities is reported by benign independent security analysts (ISA). Since no\nlegal  guidelines  exists  which  dictate  how  vulnerabilities  should  be  handled  by  the\ndiscoverer,  some ISAs report the vulnerability to the vendor and give it sufficient time to\ncome  up  with  a  patch  (The  OIS  recommends  a  time  period  of  30  days  to  be  given  to\nvendors  to  come  up  with  a  patch).  However,  some  other  ISAs  follow  the  policy  of  full\ndisclosure, i.e. they immediately post the vulnerability to a public listing like Bugtraq. One\nmajor goal of full disclosure is to eventually force vendors to come up with more secure\nsoftware.  The  debate  on  disclosure  policies  can  be  summarized  by  the  following  quotes\n(Applewhite  2000):  Jennifer  Granick  of  Stanford  University  defends  full  disclosure  as\n“You  want  a  free  flow  of  information,  just  like  in  other  scientific  fields,  ideally  without\nbetraying important interests.” Scott Blake of OIS opposes full disclosure as, “As soon as\nyou  tell  one  person,  you  have  tipped  the  balance  and  people  will  begin  producing\nexploits.”  Some  recent  academic  work  is  examining  this  issue  more  formally.  Arora,\nTelang and Xu (2004) study the optimal timing of vulnerability disclosure and show how\ndisclosure can force vendors to release patches quickly. Kannan and Telang (2004) explore\nthe welfare implications of a market mechanism for software vulnerabilities and report that\na market based mechanism for software vulnerabilities always underperforms a CERT-type\nmechanism.\nHowever  none  of  these  studies  measures  the  impact  of  disclosure  on  vendor’s  market\nvalue or profitability. While one major goal of full disclosure is to eventually force vendors\nto develop secure software, empirically, there is no evidence that suggests that disclosure\nindeed  creates  such  incentives.  Our  paper  provides  an  understanding  of  whether  such\ndisclosures create incentives for the vendors to produce secure software in the first place.\nOur  methodology  follows  closely  from  prior  event  study  analysis.  Campbell  et  al\n(1997)  present  a  useful  summary  of  the  event  study  analysis  highlighting  the  history  as\nwell  as  the  commonly  followed  methodologies.  Event  study  methodologies  are  well\naccepted for studying the implications of public announcements on stock prices. Hendricks\nand  Singhal  (1996)  study  the  impact  of  quality  award  winning  announcements  on  the\nmarket  value  of  firms  and  observe  positive  abnormal  returns  generated  by  winning  a\nquality  award.  They  further  note  that  awards  given  by  independent  organizations  and\nannouncements by small firms are more likely to have a significant impact on the firms’\nmarket  value.  In  the  field  of  Information  Systems,  Subramani  and  Walden  (2001)  used\nevent study analyses to show that e-commerce announcements lead to significant increases\nin the stock price of firms. Chatterjee, Richardson and Zmud (2001) examine the influence\nof  newly  announced  CIO  positions  on  shareholder  wealth.  Im,  Dow  and  Grover  (2001)\nexamine  the  changes  in  market  value  of  a  firm  in  response  to  IT  investment\nannouncements. Bharadwaj and Keil (2003) investigate the impact of IT failures on firms’\nmarket value.\nThe  main  contribution  of  this  research  is  that  this  is  one  of  the  first  comprehensive\nstudies,  to  our  knowledge,  that  has  tried  to  measure  the  impact  of  vulnerabilities  on\nsoftware  vendors.  Thus,  we  extend  prior  literature  on  product  defects  and  confirm  that\nsoftware  vendors  too  suffer  a  loss  in  market  value  when  a  flaw  is  discovered  in  their\nproduct.  This  is  inspite  of  the  fact  that  software  vulnerabilities  are  prevalent  among\nsoftware of almost all major vendors and that vendors face no legal liability if clients suffer\nlosses  due  to  the  software  flaw.  As  we  noted  in  the  introduction  that  this  has  important\nimplications in terms of vendors making investment in software quality as well as policy\nand legal issues which govern vulnerability disclosures.\n3. Hypotheses\nMuch of the prior literature on product defects (Jarrell and Peltzman (1985), Davidson\nand Worrell (1992)) shows that generally defective product and recall announcements are\nassociated  with  loss  in  market  value  of  a  firm.  Banker  and  Slaughter  (1998)  find  that\nunplanned and critical maintenance activities increase software maintenance costs.  Thus,\nfixing  bugs  entails  cost  to  firms.  For  example,  the  security  fixes  may  cost  about  $2000-\n$9000 when done during testing phase. However, they may cost more than 4-8 times when\nfixed after the application has been shipped.6 Slaughter et al (1998) and Westland (2003)\nsuggest  that  software  defects  are  harder  and  costlier  to  fix  if  discovered  later  in  the\nsoftware  development  cycle  (e.g.  when  the  product  has  been  shipped  to  the  customer).\nMoreover, security breach announcements by the user organizations have been known to\nhave a negative impact on the share value for firms. Cavusoglu et al (2004) show that the\nmarket  capitalization  values  of  firms  decreases,  on  average,  by  $2.1  billion  within  two\ndays  of  a  security  breach.  Cyber  insurance  firm  J.S.  Wurzler  charges  an  additional\npremium  to  firms  for  using  Windows  NT  due  to  the  number  of  security  breaches  in  the\nsoftware  (Gordon,  Loeb  and  Sohail,  2003).  Clearly,  poor  security  costs  the  users  of  the\nsoftware.  Thus, the cost to the vendor can be written as\nCost of vulnerability disclosure to vendor = Cost of patching the vulnerability + λ * (Cost\nto the users of the software due to exploitation, and/or cost to patch the system)\nHere,  λ  is  the  internalization  factor.  That  is  the  user  loss  that  is  internalized  by  the\nvendor  due  to  lost  sale  or  reputation  loss  (or  liability  if  imposed  in  future).  Clearly,  λ\ndepends on how willing users are to “punish” the vendor, how competitive the market is\nand the characteristics of vulnerability (Arora, Telang and Xu 2004). For example, this is a\n\"We  are  extremely  concerned  by  the  high  amount  of  vulnerabilities  and  patches  from\nMicrosoft. This goes against the credibility of what they have been saying,\"\nMichael Kamens, global security director at Thermo Electron Corp.7\nGiven the fact the competitors seem to be using vulnerability disclosure as a strategic way\nto  undermine  the  rivals’  reputations,  it  seems  that  vulnerability  disclosure  signals  a\npotential loss in future cash flows for a software vendor owing to customer dissatisfaction\nbecause customers suffer a loss if their systems get breached. It also signals an increase in\nproduct  related  costs  due  to  the  time  and  effort  that  the  vendor  spends  in  developing  a\npatch or a fix for the flaw. Therefore, we hypothesize that\nH1: A software vendor suffers a loss in market value when a security related vulnerability\nis announced in its products.\nOur second hypothesis pertains to whether the software vendor releases a patch for the\nproduct  at  the  time  of  the  vulnerability  announcement.    As  per  the  popular  convention\nfollowed in the vulnerabilities market, vendors are given some time to work on a patch for\nthe vulnerability before it is made public. Vendors may also provide a workaround (such as\ndisabling features of the software) when a vulnerability is disclosed and choose to address\nthe  vulnerability  in  a  future  upgrade.  Presence  of  the  patch  is  also  likely  to  reduce\ncustomers’  loss  if  they  apply  the  patch.  Since  presence  of  patch  also  reflects  vendor’s\ncommitment  to  its  customers  we  expect  that  vulnerabilities  disclosed  with  a  patch  will\ncompensate, to an extent, the negative signal due to vulnerability disclosure. Vendors are\nalso  pushing  for  limited  disclosures  so  that  they  can  release  the  remedial  patch  in  time.\nThis also suggests that patches play a critical role. Therefore our second hypothesis is:\nH2: CAR (Cumulative Abnormal Return) [negative] of a stock is greater for vulnerabilities\nwhere  the  software  vendor  does  not  release  a  patch  at  the  time  of  the  vulnerability\ndisclosure.\nThe  damages  suffered  by  a  software  vendor’s  customers  due  to  a  vulnerability  in  the\nvendor’s software  depend on the  type of security breach that the vulnerability  facilitates.\nCampbell  et  at  (2003)  classify  the  security  breaches  as  confidentiality  related  and  non-\nconfidentiality  related.  Confidentiality  related  breaches  involve  attacks  where  an  intruder\ncan  gain  access  into  a  system  and  can  steal  sensitive  information.  Non-confidentiality\nrelated breaches include attacks like denial of service (DoS) attacks where the most likely\nscenario  is  a  disruption  and/or  a  downtime.  Campbell  et  al  (2003)  further  show  that  the\nloss  in  market  value  is  more  for  confidentiality  related  breaches  than  for  non-\nconfidentiality related breaches. Hovav and D’Arcy (2003) show that DoS attacks are not\nassociated with any significant loss in market value for a firm. Therefore we would expect\nthat  the  vendor  loses  more  market  value  if  the  vulnerability  in  its  software  causes  a\nconfidentiality  related  breach.  The  intuition  is  that  the  negative  reaction  due  to  the\nvulnerability  is  more  if  the  customers  can  potentially  suffer  greater  losses  due  to  the\nvulnerability. Therefore, our next hypothesis is:\nH3:  CAR  is  greater  for  a  vulnerability  which  can  potentially  cause  a  breach  in\nconfidentiality as compared to non-confidentiality related breaches.\nThe  impact  of  a  software  flaw  on  a  vendor  also  depends  on  how  severe  the\nvulnerability is. Davidson and Worrell (1992) conduct an event study with product defect\nannouncements  in  the  tire  industry  and  showed  that  the  impact  of  severe  flaws  (which\ninvolve  a  recall)  is  more  than  that  of  less  severe  flaws  (which  involve  repairs  but  nor\nrecall). Sometimes an ‘exploit’ exists for the vulnerability at the time of announcement. An\n‘exploit’  is  a  piece  of  code  which  anyone  can  use  to  compromise  the  security  of  the\nsoftware product and subsequently of other information assets. Therefore, we propose our\nnext set of hypotheses as:\nH4a:  The  loss  in  market  value  of  a  software  vendor  is  greater  if  the  announced\nvulnerability has a higher severity.\nH4b: The loss in market value of a software vendor is greater if an exploit exists publicly\nat the time of the vulnerability announcement.\nA  recent  article  in  the  Wall  Street  Journal  hinted  that  firms  are  using  vulnerability\ndisclosure as a strategic weapon against competitors. E.g. ISS disclosed a vulnerability in\nrival  Checkpoint’s  flagship  firewall  product  just  ahead  of  Checkpoint’s  investor  summit.\nVendors themselves disclose vulnerability information in their products routinely. In fact,\nmany believe that vendors would prefer not to disclose information at all but they fear that\nsomeone  else  would  do  it.  Generally  vendors  are  likely  to  be  more  careful  about  the\ndisclosure  as  opposed  to  third  party.  Moreover,  disclosure  by  vendors  would  signal  their\ncommitment to providing secure software, we hypothesize that\nH5:  The  loss  in  market  value  for  a  software  vendor  is  lower  in  case  the  security\nvulnerability is discovered by the vendor itself rather than by rivals or third party security\nfirms.\nOur  final  hypothesis  relates  to  the  publication  where  vulnerability  is  reported.  Some\nvulnerabilities are reported in popular press. Others find mention only in industry sources\nlike  CERT  or  technology  news  portals  like  news.com.  The  question  which  arises  is\nwhether  both  sources  are  equally  influential  in  affecting  a  stock’s  CAR?  One  would\nsuspect the popular press of being more influential than industry sources because it reaches\na broader audience. So our next hypothesis is:\nH6:  The  magnitude  of  CAR  is  more  when  the  vulnerability  is  reported  in  popular  press\nthan in industry sources.\n4. Data Description & Methodology\n4.1 Vulnerability Disclosure Process\nThe typical process of vulnerability disclosure takes place as shown in Figure 1.\nVulnerability\nDiscovery\nThe  process  starts  with  the  vulnerability  discovery.  There  are  various  sources  for\nvulnerability discovery. Some vulnerabilities are discovered by amateur researchers or by\nindependent  security  analysts  (ISA)  like  eEye  Security.    If  the  ISA  chooses  to  follow\n‘limited  disclosure’,  it  reports  the  vulnerability  to  the  concerned  vendor  or  to  an\nindependent body like CERT who disclose the vulnerability to public later (generally after\nthe  fix  has  been  made  available)  in  a  limited  way8.    Sometimes,  the  vendor  itself  might\ndiscover  a  vulnerability  in  its  products.  On  the  other  hand,  if  the  ISA  chooses  to  follow\n‘full  disclosure’  or  if  a  hacker  discovers  the  vulnerability,  the  vulnerability  may  get\nannounced  directly  to  public.  Forums  such  as  Bugtraq  allow  for  full  disclosure  of\nvulnerability  information.  In  any  case,  after  the  vulnerability  has  been  announced,\ninvestors  might  re-evaluate  their  projections  on  the  software  vendors’  profitability  based\non  the  vulnerability  characteristics.  Consequentially,  the  stock  price  of  the  vendor  might\nshow an abnormal return due to the vulnerability announcement.\n4.2 Data\nOur data comes from the vulnerability disclosures in popular press as well as from the\nadvisory  reports  in  CERT.  We  include  articles  published  by  news  networks  like\nBusinesswire,  Newswire  and  daily  articles  in  popular  press  like  WSJ,  NY  Times,\nWashington Post and LA Times. We search for these news articles in Proquest and Lexis-\nNexis  Academic  databases  which,  between  them,  maintain  news  articles  from  major\nnewspapers  and  news  networks  all  over  the  country.  We  also  include  articles  from\nnews.com  which  is  a  CNET  owned  site  and  is  a  premier  source  for  round-the-clock,\nbreaking  technology  news  coverage.  We  used  the  following  terms  in  our  search:\n‘vulnerability  AND  disclosure’,  ‘software  AND  vulnerability’,  ‘software  AND  flaw’,\n‘virus AND vulnerability’& ‘vulnerability AND patch’.\nWe  also  searched  the  vulnerability  announcements  for  information  on  the  type  of\nvulnerabilities. Based on this, we classify vulnerabilities into various categories.\n•  If  the  announcements  contained  words  such  as  ‘serious’  or  ‘severe’  or  ‘dangerous’  to\ndescribe  the  vulnerability,  we  characterized  the  vulnerability  as  ‘Severe’.  If  the\nannouncement  characterized  the  vulnerability  as  ‘moderately  severe’  or  ‘with  low\nseverity’, we characterized it as ‘Non-severe’.\n•  The  vulnerability  announcements  also  had  references  to  what  kind  to  security  breach\ncould be facilitated if attackers exploited the vulnerability. If the vulnerability contained\nterms  such  as  ‘cause  denial  of  service’,    or  ‘disrupt  operations’,  we  classified  the\nvulnerability as type ‘DoS’; otherwise if the vulnerability contained terms such as ‘gain\naccess’  or  ‘steal  information’  or  ‘take  control’,  we  classified  the  vulnerability  as\n•  Further,  the  announcements  also  described  whether  the  vendor  released  a  patch  at  the\ntime of the vulnerability announcement. If the vendor announced a patch at the time of\nvulnerability disclosure, we classified the vulnerability as ‘Fix Available’.\n•  Finally,  we  also  classify  vulnerabilities  on  whether  an  ‘exploit’  exists  for  the\nvulnerability  in  the  public  domain.  If  the  vulnerability  announcement  contained  terms\nsuch as ‘an exploit for the vulnerability is circulating’, we classify the vulnerability as\ntype ‘Exploit’.\nAs per convention in prior event study literature (Hendricks & Singhal 1996), we excluded\nthe following type of announcements from our sample:\n•  Vulnerability  announcements  in  non-daily  periodicals  like  magazines  because  of  the\ndifficulty in determining the exact date of the announcement.\n•  Repeat  announcements  of  the  same  event  in  a  different  publication  at  a  later  date.  In\ncase of such repeat announcements, the earliest announcement date was chosen as the\n•  Announcements which were associated with other confounding events like stock splits\nand mergers on the event date.\n•  Announcements related to firms not traded on any public exchange in the US.\n•  Announcements  that  point  to  a  fundamental  protocol  flaw  rather  than  a  particular\nsoftware. E.g. a flaw in the FTP protocol affects multiple vendors. The reason behind\ndropping this category is that the flaw exists in the software only because it follows a\nflawed protocol, and not due to the vendor.\n•  Software flaws which are not security related.\nOur dataset contains 146 vulnerability announcements pertaining to around 18 firms in\nthe time period January 1999 – May 2004. We capture information on the following details\nabout the vulnerability announcement: date, firm name, product, who discovered the flaw,\nnews source, whether patch is available and severity. The descriptive statistics of the data\nare given in Table 1.\nNumber of firms\n%age of vulnerabilities announcements in popular press\n%age of vulnerabilities for which vendor has patch available at the\ntime of the announcement.\n%age of vulnerabilities discovered by the vendor itself\n% of vulnerabilities that could potentially result in a security breach\nrelated to confidentiality\n%age of vulnerabilities for which the announcement contained\ninformation of a publicly available ‘exploit’\nThe year-wise distribution of announcements is given in Table 2 below.\nYear\n4.3 Methodology\nWe use the standard event study methodology for this analysis. An event study assumes\nthat returns on a stock are significantly impacted by an event of interest (in our case, the\nevent of interest is the vulnerability disclosure announcement).  The period of interest for\nwhich we observe the event is known as the event window. The smallest event window is\none  day  (day  of  the  announcement  or  day  ‘0’9).  In  practice,  the  event  window  is  often\nexpanded  to  include  two  days  (day  0  and  day  110)  to  capture  the  effect  of  price\nannouncements  made  after  the  close  of  the  markets  on  a  particular  day.  Sometimes\nresearchers include a day before the announcements to incorporate any information leaks\nabout the event. In our study we define a one day event window (day 0)11. Hendricks and\nSinghal (1996) cite two reasons to use a one day event period. One, a shorter event period\npermits a better estimation of the effects of information of stock prices since it reduces the\npossibility  of  other  confounding  factors  not  related  to  the  announcement.  Two,  it  also\nincreases the power of the statistical tests.\nAbnormal  returns  are  defined  as  the  difference  between  the  actual  return  of  the  stock\nover the event window minus the expected return of the stock over the event window. The\nexpected return on the stock is calculated in several ways, but in our analysis, we use the\nmarket  model  which  assumes  a  stable  linear  relation  between  the  market  return  and  the\nreturn  on  the  stock.  We  also  verify  our  results  using  other  methods  such  as  the  market-\nadjusted  method  and  the  mean-adjusted  method.  The  coefficients  of  the  linear  model  are\ncalculated  by  choosing  a  portion  of  the  data  as  the  estimation  window.  The  estimation\nwindow,  generally  between  120  days  and  200  days  used  in  most  studies,  is  the  period\nimmediately before the event window. In our case, we use an estimation window of size\n160 days, from day -175 to day -16.\nThere  are  three  main  methods  followed  in  the  event  study  methodology  (Campbell,\nAndrew  and  MacKinlay,  1997;  Hendricks  and  Singhal,  1996)  to  estimate  the  abnormal\n1. The Market Model\nIn the market model, the abnormal returns are estimated as follows:\nwhere i denotes the event (i=1,2……N), m denotes the market and t denotes the  day of\nthe event (e.g t = 0 denotes the day of the vulnerability announcement.). ARit denotes the\nabnormal return of event i at time t, Rit denotes the actual return and Rmt denotes the market\nreturn  at  time  period  t13.\ndenotes  the  normal  return  of  the  firm  due  to  the\nmarket-wide  movement.  The  abnormal  return  is  defined  as  the  difference  between  the\nactual  return  and  the  normal  return.  This  is  the  part  of  the  actual  return  that  cannot  be\nexplained  by  market  movements  and  captures  the  effect  of  the  event.  Since  most  of  the\ntech stocks are listed on NASDAQ, we use this as our indicator for market returns. We use\nordinary  least  squares  regression  to  estimate  the  coefficients  α  and  β  for  the  above\n2. The Market Adjusted Model\nIn this case, the abnormal returns are given as\n=\nwhere the terms have the usual meaning as in the Market Model.\n3. The Mean Adjusted Model\nis  the  mean  return  on  the  stock  which  made  a  vulnerability\nannouncement during event i,  over the duration of the estimation period and T is number\nof days in the estimation period (in our study, T=160).\nThe  mean  abnormal  return  across  all  observations  on  day  t  of  the  event  is  given  as\n. The cumulative abnormal return\nfor the event is defined as the\nsum  of  the  abnormal  returns  over  the  event  window.  Prior  research  on  event  studies\n(Brown  &  Warner  1980,  Brown  &  Warner  1985)  presents  a  comprehensive  analysis  of\nby  more  than  one  vendor  on  a  given  day,  our  statistic  should  allow  for  event  day\nclustering.  The  following  t-statistic  proposed  by  Brown  &  Warner  (1985)  takes  in  to\naccount event day clustering as well as cross-sectional dependence in the security specific\nt =\nA\nt\nS\nWhere\n⎛\n⎜\n⎝\n(\n−\nand T is the number of days in the estimation period\nand\n⎛\n= ∑\n⎜\nT\n⎝\n⎞\n⎟\n⎠\nThe  null  hypothesis  is  that  the  abnormal  returns  are  not  significantly  different  from\nzero.  Under  the  null  hypothesis,  the  abnormal  returns  are  independent  and  identically\ndistributed  and  normal  with  a  mean  of  zero  and  the  variance  given  by  the  variance  of\nabnormal returns over the estimation period.\nWe  also  do  multivariate  regression  and  test  our  hypothesis  of  how  CAR  varies  with\nvarious vendors, or vulnerability characteristics. But before we present the details on our\nregression analysis, we present the results of the event study.\n4.2 Event Study Results\nvulnerability disclosures on the stock prices of software vendors for our entire sample of\n146 announcements (p-values are in parenthesis)\nDay 0     CAR\nMarket Model  Market\nMean Model\n-0.5    (0.09)\n-0.55  (0.01)\nPercent Less than Zero\n58.7%  (0.03)\nAdjusted Model\nWe  calculate  CARs  under  there  different  models  (Market  Model,  Market  Adjusted\nModel and Mean Adjusted Model).  For each of the three models, we use three different\ntest  statistics  (Mean  Abnormal  Return,  Median  Abnormal  Return  and  Percent  Less  than\nZero). The Mean Abnormal Return Test (equations (1) to (4)) is parametric in nature and\nmakes  assumptions  about  the  distribution  of  abnormal  returns.  We  also  use  two  non-\nparametric tests to strengthen our results. One, we use the Wilcoxon Signed Rank Test to\ncalculate the p-value for the median abnormal return; two we use the Sign Test to calculate\nthe p-value for the percent negative returns. The Wilcoxon Signed Rank Test tests whether\nthe  median  of  the  abnormal  returns  is  different  than  zero.  The  Sign  Test  is  based  on  the\nsign  rather  than  the  magnitude  of  the  abnormal  returns  and  requires  that  under  the  null\nhypothesis, the proportion of abnormal returns greater than (or less than) zero is 50%.\nFrom Table 3, we note that the CAR for day 0 is negative across all the three different\nmodels. E.g. the Mean Abnormal Return varies between 0.5% - 0.67% depending on the\nmodel  used.  Further  the  Market  Model  and  the  Market  Adjusted  Model  are  statistically\nsignificant at p < 0.01, while the Mean Adjusted Model is statistically significant at p < 0.1\nlevel.  The Median Abnormal Returns range between 0.44% - 0.55% and are significant at\nthe 0.01 level. Finally, the percent observations less than zero range between 57.8% - 64%\nand  are  significant  at  the  0.05  level.  It  is  clear  that  CAR  is  negative  and  statistically\nsignificant for all three models and all three tests.\nThus,  our  results  suggest  that  software  vendors  do  tend  to  lose  market  value  when  a\nvulnerability is announced in their product. To check for the robustness of our results, we\nnot  only  test  the  mean  abnormal  returns,  but  also  the  median  abnormal  returns  and  the\npercent  less  than  zero.  This  provides  support  for  hypothesis  H1  that  vulnerability\nannouncements are associated with a loss in market value of software vendors.\nWe also calculate the abnormal returns using different event windows (beyond 0 days)\nusing  the  market  model  (the  results  do  not  change  substantially  for  other  models).  The\nresults are given in Table 4.\n0 to 10\n(0.36)\nFrom the table, it is clear that the CAR on day 0 is negative and significant at the 0.01\nevel. However, CAR for day 0 and day 1 combined is significant only at the 0.1 level. This\nsuggests  that  the  stock  market  is  efficient  in  the  sense  that  the  effect  of  a  software\nvulnerability  announcement  is  quickly  incorporated  into  a  vendors’  stock  price.  The  p-\nvalues for day -1 is not statistically significant. A possible explanation for this is that the\neffect of news leakage through forums like SecurityFocus is not significant. The CARs in\ncolumns  3,  4,  5  and  6  are  negative  but  not  statistically  significant.  However,  it  is\ninteresting to note that the CARs are negative for even a 10 day window.  Figure 2 shows\nthe CAR over a 16 day event window (from day -5 to day 10).\nOur result corroborates prior work on defective products (Jarrell et al 1985, Davidson et\nal 1992) by showing that product defects lead to a loss in market value of a firm. Our study\nanalyzes returns on stocks of software vendors and we find that a defective software which\ncompromises the security of customers’ information systems leads to a negative impact on\nthe  market  value  of  the  software  vendor.    The  extent  of  losses  suffered  by  a  vendor,  on\naverage,  is  moderate  at  about  0.63%  of  its  market  capitalization  value  on  the  day  the\nvulnerability gets announced.\nMarket Capitalization\nWe  also  calculated  the  abnormal  change  in  market  capitalization  values  of  the  software\nvendor due to the vulnerability announcement14. For each firm, the day 0 change in market\ncapitalization  value  was  calculated  by  multiplying  the  day  -1  market  capitalization  value\nwith the abnormal returns on day 0. On average, we calculate that the software vendors in\nour sample lost $0.86 billion in market capitalization value on the day of the vulnerability\ndisclosure.  Since Microsoft accounts for more than 40% of our sample, we sub-divide our\nsample into Microsoft and non-Microsoft samples. For the Microsoft sample, the average\nchange in market capitalization is around $0.92 billion and for the non-Microsoft sample,\nthe average change in market value is $0.81billion.\nRobustness Checks: We also perform the following robustness checks on our results, as\nspecified in the event study by Cooper, Dimitrov and Rau (2001).\n1.  Robustness to Outliers:\nTo  check  the  robustness  of  our  results  to  exclude  the  effect  of  outliers,  we  compute  the\nCAR for our sample after excluding the top 10 percentile and the bottom 10 percentile of\nobservations  (ranked  according  to  the  day  0  mean  abnormal  returns).  We  find  that  our\nresults remain qualitatively the same. For example, mean abnormal returns for this sample\nare 0.53% (against 0.63% for the entire sample) and these are significant at the 5% level.\nThis suggests that our results are robust to outliers in the data.\n2.  Momentum Effect:\nIt can be argued that the day 0 abnormal returns are caused simply by market momentum\nthan by the underlying event. As a simple check, we compute the correlation between the\nabnormal returns before the event and those after the event. In specific, we check the pair-\nwise correlation (along with level of significance) for three pairs of values : one, day -10 to\nday -1 CAR and day 0 to day 10 CAR; two, day -10 to day -1 CAR and day 0 CAR; and\nthree, day -1 CAR and day 0 CAR. The pair-wise correlations are as follows:\n•  day -10 to day -1 CAR and day 0 to day 10 CAR (correlation: 0.13, p-value 0.12)\n•  day -10 to day -1 CAR and day 0 CAR (correlation: -0.05, p-value 0.5)\n•  day -1 CAR and day 0 CAR (correlation: 0.03, p-value 0.67)\nThus  we  find  that  none  of  the  correlations  is  strong  or  significant  at  the  10%  level  and\nhence the momentum in the stock prices does not seem to be driving our results.\n5. Regression Analysis\nTo test the other hypothesis, we develop both a regression model and simple means test\nto explain the effect of various vulnerability characteristics on abnormal returns. However,\nordinary  least  squares  model  might  not  be  appropriate  because  it  does  not  account  for\nheterogeneity  among  firms.  The  issue  of  heterogeneity  is  an  important  consideration  in\nanalyzing panel data. For example, the level of abnormal returns could differ across firms\nif investors use different valuation models  across  firms  (Malatesta  and  Thompson  1985).\nEstimating  aggregate  parameters  while  ignoring  heterogeneity  could  lead  to  biased  and\ninconsistent estimates (Hsiao 1986).\nTo incorporate the impact of firm specific heterogeneity in our data, we propose a fixed\neffects  model.  The  fixed  effects  model  controls  for  unobservable  firm  specific  variables\nthat are constant over time. This is equivalent to generating dummy variables for each firm\nand including them in an ordinary linear regression to control for firm specific effects. The\nfixed effects model can be specified as:\n=\n+⋅\nεµβ\nit\ni\nwhere i = 1…..N (N is the total number of firms) and t = 1……..T (T is the total number\nof events). yit is the Abnormal Return (ARit) for firm i at event t as calculated according to\nthe  market  model  in  equation  (1)15.  Xit  are  the  independent  variables  which  capture  the\nvarious  vulnerability  characteristics,  ui  is  the  firm  specific  dummy  variable.  The\ndescription of the independent variables is as follows:\nSEVR:  whether  the  vulnerability  has  been  classified  as  severe;  SEVR  =  1  for  a  severe\nvulnerability and 0 otherwise.\nPATCH: Whether a patch is available at the time of the vulnerability disclosure. PATCH\n=1 if a patch is available and 0 otherwise.\nDISC:  Whether  the  vulnerability  was  disclosed  by  the  third  party.  DISC  =  1  if  the\nvulnerability was disclosed by the third party and 0 if disclosed by the vendor.\nEXPLOIT:  If  an  exploit  is  publicly  available  at  the  time  of  the  vulnerability\nannouncement, then EXPLOIT = 1; otherwise it is zero.\nCERT: If the vulnerability was first reported in CERT, then CERT = 1, it is 0 otherwise.\nPRESS: If the vulnerability was first reported in popular press, the variable PRESS = 1,\notherwise it is zero.16\nDOS:  If  the  vulnerability  can  potentially  lead  to  a  denial  of  service  type  attack  or  a\ndisruption in services, then the variable DOS =1, otherwise it is zero.\nEffect of Investor Sentiments:\nTo control for abnormal returns due to overall market sentiments, we introduce a set of\ndummy  variables  based  on  the  time  when  the  vulnerability  was  announced.  We  use  the\nevents surrounding 9/11 as the basis for segmenting our sample into various time periods.\nThe stock market crash in late 2000 could also play a role in the negative abnormal returns.\nWe introduce the following dummy variables in our model:\nPost_911: If the vulnerability was announced within a year of 9/11, i.e. if the date of\nthe  announcement  was  between  Sept  11,  2001  and  Sept,  11  2002.  This  constitutes  19\nobservations from our sample.\nPre_911: If the vulnerability was announced within a year before 9/11, i.e. if the date of\nthe announcement was between Sept 11, 2000 and Sept, 11 2001. This was also the time\nafter the stock market crashed in mid 2000 and lasted till the first three quarters of 2001\n(Wall  Street  Journal,  200017,  Kannan  et  al  2003,  Wall  Street  Journal,  200318).  This\nconstitutes 28 observations from our sample.\nYear_99-00: if the announcement was made prior to September 11, 2000, i.e. if the date\nof the announcement is between Jan 1, 1999 and September 11, 2000.  This constitutes 23\nobservations from our sample.\nYear_02-03:  If  the  date  of  the  announcement  is  between  September  11,  2002  and\nSeptember 11, 2003. This constitutes 44 observations from our sample.\nYear_03-04: If the date of the announcement is between September 11, 2003 and June\n1, 2004. This constitutes 32 observations from our sample. This is the baseline category for\n5.1 Results\nWe  first  compare  the  mean  CARs  of  different  sub  samples  based  on  vulnerability\ncharacteristics  to  understand  how  various  factors  affect  market  value  of  the  firms.  The\ncharacteristics that we consider are explained above. This is similar to the method followed\nby Subramani and Walden (2001), Chatterjee, Richardson and Zmud (2001) Im, Dow and\nGrover (2001). The results are summarized in Table 5.\nVulnerability\nCharacteristic\nVariable\np-value\nHigh Severity (81%)\nSeverity\nLow Severity (19%)\nSource of Discovery\nDiscovered by Vendor\n(36%)\nDiscovered by Third\nParty (64%)\nPatch Available (24%)\nPatch Availability\nPatch Not Available (76%)\nCERT (34%)\nSource of Disclosure\nPress (35%)\nNews.com (30%)\nDenial of Service (24%)\nType of Attack Possible  Confidentiality (76%)\nMicrosoft vs Non\nMicrosoft\nMicrosoft (46%)\n0.13\nFrom the means analysis it is clear that more severe vulnerabilities affect the stock price\nmore.  Surprisingly,  when  vendors  disclose  the  information  then  the  investors  seem  to\nperceives  it  more  negatively  than  when  some  third  party  releases  it.  As  expected,\navailability  of  patch  mitigates  the  negative  impact  of  disclosure  somewhat.  Since  the\nvendors  typically  disclose  the  information  with  a  patch,  we  may  find  during  the\nmultivariate regression that vendor release effect may disappear. The source of disclosure\nis  also  relevant.  Investors  seem  to  pay  more  attention  to  vulnerabilities  published  in\nmainstream newspapers than to CERT. Confidentially breaches are considered more severe\nand hence have higher significant impact than denial of service vulnerabilities. Finally, we\ndo not find an evidence of Microsoft effect though, non-Microsoft firms seem to lose more\nvalue.\nIn  the  second  method,  we  test  our  hypothesis  related  to  the  impact  of  various\nvulnerability  characteristics  on  the  market  value  of  a  software  vendor  using  the\nmultivariate regression model outlined in equation (5). This is similar to the methodology\nfollowed  by  prior  event  studies  such  as  Hendricks  and  Singhal  (1997),  Chatterjee,\nRichardson and Zmud (2001), Lane and Robertson (1995). The regression method has the\nadvantage over the sub-sample method used in Table 5 that the regression method captures\nthe  effect  of  all  the  independent  variables simultaneously.  The  sub-sample  method  could\ngive spurious results because the effects of a sub-sample, such as patch vs no-patch, can be\nexplained by a relationship between patch and other independent variables. The parameter\nestimates as well as the p-values for the various parameters of our fixed effects regression\nare given in the Table 6 below.\nVariable\nSEVR\nEXPLOIT\nPATCH\nDISC\nCERT\nPRESS\nDOS\nY_99-00\nPre_911\nPost_911\nY_02-03\nConstant\n-0.007\n-0.011*\n-0.02*\n-0.01*\n0.01*\n0.05\nThe  R2  for  this  regression  is  16.6%.  The  F-test  (p-value  0.001)  for  the  overall  model\nsuggests that our model is highly significant. The constant in the regression is simply the\naverage  of  all  the  fixed  effects  in  the  model.  Our  regression  provides  several  interesting\nobservations regarding the effect of the vulnerability characteristics on the stock price of\n1.  Severity: The coefficient on the SEVR variable in Table 6 is negative and significant\n(though  at  p  =  0.1).  This  is  consistent  with  results  in  Table  5.  More  severe\nvulnerabilities have a higher potential to cause damage and hence have a larger adverse\nimpact  on  CAR.  On  average,  a  severe  vulnerability  can  cost  a  software  vendor  0.6%\nmore than a non-severe vulnerability, providing support to hypothesis H4.\n2.  Patch:  We  also  find  that  the  non-availability  of  a  patch  is  positive  and  significantly\ncorrelated with the market value. This provides support to hypothesis H2 because the\ncoefficient  on  the  PATCH  variable  suggests  that  on  average,  firms  which  do  not\nprovide a patch at the time of the vulnerability disclosure suffer a loss of 0.83% more\nthan firms which provide a patch.\n3.  Confidentiality:  The  coefficient  of  the  DOS  variable  is  0.0076.  This  implies  that\nvulnerabilities which result in a non-confidential breach (denial of service, disruptions\nin  service)  have  an  abnormal  return  of  0.76%  higher  than  the  vulnerabilities  which\nresult in a confidentiality-related breach. This confirms our hypothesis H3.\nNone of the other coefficients are significant (except time coefficients). Thus our results\nfail to provide support for hypothesis H5 and H6. From table 5 also, we can observe that\nnone  of  the  sub-samples  (other  than  the  Press  sub-sample)  has  a  significant  coefficient.\nThe rejection of hypothesis H5 is especially interesting because it suggests that the markets\ndo not penalize a vendor any more if the vulnerability is discovered by a third party than\nby the vendor itself.\nWe also control for the effect of the time period during which the vulnerabilities were\nannounced  to  control  for  time-specific  investor  sentiments  which  can  affect  abnormal\nreturns.  We  find  that  software  vendors  suffered  the  greatest  abnormal  returns  in  the  one\nyear period post 9/11; for example, on average, vendors lost 2% more in market value for\neach vulnerability announcement in the year following 9/11 than they did in the baseline\nperiod  (2003-04).  This  suggests  that  security  concerns  among  investors  were  highest\nduring this period, as an aftermath of September 11. The next highest period of abnormal\nreturns was the one-year time frame after the dot-com crash, i.e. from Sept 2000 to Sept\n2001.  In  this  period,  software  vendors  lost,  on  average,  1.0%  more  on  vulnerability\nannouncements than in the baseline period. The results also suggest that year 2003-04 is\nwhen the vendors suffered the least. It is possible  that  vulnerability  announcements  have\nbecome  commonplace  and  are  not  affecting  the  vendors  as  much.  But,  more  research  is\nneeded to provide a definitive answer.\n6. Conclusions and Discussion\nTo  the  best  of  our  knowledge,  this  is  the  first  study  to  analyze  the  impact  of  product\ndefects on software vendors. We also analyze the information content of the vulnerability\ndisclosure  announcement  and  classify  vulnerabilities  into  various  sub-types  based  on  the\nthe  source  of  vulnerability  disclosure,  severity  of\nvulnerability, availability of a fix, whether an exploit was publicly available at the time of\ndiscovery,  the  type  of  security  breach  caused  by  the  vulnerability  and  the  source  of\nvulnerability discovery. Our results show that vulnerability disclosure leads to a significant\nloss  of  market  value  for  software  vendors.  This  indicates  that  the  stock  markets  react\nnegatively  to  the  news  of  a  vulnerability  disclosure,  because  the  discovery  of  a\nvulnerability  could  suggest  a  loss  in  future  cash  flow  of  the  software  vendors.  Software\nvulnerabilities affect the cash flows of a vendor in two main ways: One, the vendor has to\nspend time and effort in providing a patch for the vulnerability, which increases the overall\ncost  of  the  software  product  and  hence  reduces  profits.  Vulnerabilities  may  also  lead  to\ncustomer  dissatisfaction  because  the  customer  has  to  spend  time  and  effort  in  installing\npatches  and  is  exposed  to  a  risk  of  security  related  attacks.  This  could  further  lead  to\ncustomers  shifting  to  competitor’s  products  and  hence  reduces  the  cash  flow  for  the\nvendors’  products.  This  has  implications  for  software  vendors  to  invest  in  improving  the\nquality of their software. While vendors would like to launch software products as soon as\npossible, our study shows that vendors need to focus testing in  areas that can potentially\ncontain  greater  number  of  security  vulnerabilities.  We  also  show  that  the  effect  of  a\nvulnerability  announcement  is  quickly  incorporated  into  the  stock  price  and  after  the\nsecond day, there is no significant impact on the stock prices. We check for the robustness\nof our results in terms of checking for outliers and checking for momentum effects in the\nstock returns. We find that our results are also robust to the effect of investor sentiments\nsince we control for the effect of various time periods in our regression.\nOur study also provides preliminary evidence that firms should integrate security into\nsoftware  quality  practices.  In  a  firm  with  limited  resources,  this  would  mean  focusing\ntesting  efforts  in  areas  that  have  a  greater  number  of  security  vulnerabilities.  Although\nresearchers  in  computer  science  have  stressed  on  this  fact  (McGraw  2004),  there  hardly\nexists  any  literature  in  software  engineering  economics  which  measures  the  return  on\ninvestment  of  incorporating  security  based  metrics  in  software  quality  or  software  risk\nassessment.  While  software  quality  traditionally  deals  with  functional  testing,  complete\nsecurity  testing  would  incorporate  non-functional  testing  we  well,  i.e.  subjecting  the\nsoftware to misspecified input streams (Potter and McGraw 2004).\nOur  results  also  show  that  vulnerabilities  which  are  severe  or  which  can  cause  a\nconfidentiality  related  attack  are  likely  to  result  in  a  greater  loss  in  market  value  for  a\nvendor. This has implications for the software vendor in terms of allocating resources for\nsoftware  testing  and  quality  improvements.  For  example,  vendors  should  focus  on\nspending more resources in testing those software modules where a flaw is more likely to\ncause  a  confidentiality  related  breach.  Campbell  et  al  (2003)  show  that  confidentiality\nrelated breaches cause a greater loss to firms; we complement that result by showing that\nsoftware vulnerabilities which cause a confidentiality related breach are likely to result in a\ngreater loss in market value for the vendor.\nComparison with prior event studies:\nIt is interesting to compare how the abnormal returns in our event study compare with\nresults in prior event studies. Specifically, we compare our results with event studies in the\nfollowing  categories:  security  breach  related  announcements,  IT  investment  related\nannouncements and product defect related announcements.\nproduct  recall  announcements.  It  is  especially  interesting  to  note  that  the  loss  in  market\nvalue that vendors suffer due to a security vulnerability is much less than that suffered by\nfirms  during  a  security  breach.  A  possible  reason  could  be  that  software  vendors  are\nprotected  by  click-wrap  agreements  and  have  only  limited  liability  for  any  flaw  in  their\nproducts.  Another  reason  is  that  firms  usually  supply  a  patch  with  the  vulnerability\ndisclosure (almost 76% of observations in our sample have a patch available at the time of\ndisclosure).  Therefore  security  breaches  are  not  so  much  caused  due  to  unprotected\nvulnerabilities as due to lack of adequate patching done by firms. E.g. the SQL Slammer\nvirus, which affected millions of servers worldwide, was created when hackers exploited a\nsix month old vulnerability in SQL. Microsoft had already released a patch for the same,\nbut,  as  demonstrated  by  the  Slammer,  many  firms  had  not  adequately  protected  their\nservers by applying the patch.\nClassification of Event\nStudy\nImpact of Vulnerability\nDisclosures on\nSoftware Vendors\nImpact of Security\nBreaches on Firms\nAuthors\nTelang R and S Wattal (2004)\nTime\nPeriod\n1999-2004\n-0.63%\nCampbell K, Gordon LA, Loeb MP\nand L Zhou (2003)\nCavusoglu H, Mishra B and S\nRaghunathan (2004)\n-2.1%\nImpact of Product\nRecall Announcements\nImpact of IT\nInvestment\nAnnouncements\nHovav A and J D’Arcy (2003)\n1998-2002\nKannan K, Rees J and S Sridhar\n(2004)\nJarrell G and S Peltzman (1985)\n1987-1998\nDavidson WL III and DL Worrell\n(1992)\nChatterjee D, Richardson VJ and\nRW Zmud (2001)\nIm KS, Dow KE and V Grover\n(2001)\nSubramani M and E Walden (2001)  Oct 1998-\nDec 1998\n1981-1988\nDos Santos BL, Peffers K and DC\nMauer (1993)\n1981-1996\n-0.81%\n(for auto)\n-0.36%\n(day -1)\n1.16%\n1%\nImpact of Winning a\nQuality Award\nHendricks KB and Singhal VR\n(1996)\nImplications for Software Quality and Disclosure Policy\n0.59%\nAs we noted in introduction, one major argument given by the full disclosure group\nis that it will eventually force the vendors to improve the quality of their product. From our\nanalysis, there seems to be some support for this argument. Disclosure, in general (with or\nwithout a patch) adversely affects the market valuation of the vendors. It is more severe in\ncase  of  without  patch  (which  is  what  generally  happens  during  full  disclosure).  Thus,\ndisclosure clearly creates some incentives for vendors to produce better quality software.\nHowever,  market  value  is  only  one  metric  to  capture  the  impact  of  disclosure.  A  more\ninteresting  and  comprehensive  work  would  be  to  measure  the  impact  of  disclosure  on\nprofit or market share of these firms. But our paper does provide a starting point for why\nwe  should  analyze  this  issue  in  more  detail.    Another  potential  area  of  future  research\nwould  be  to  capture  and  test  the  link  between  security  based  risks  and  the  quality  of\nsoftware systems.\nFrom  our  discussion,  it  is  also  clear  why  vendors  are  pushing  for  a  limited\ndisclosure policy. Recently, Organization for Internet Safety (OIS), which is consortia of\n11  large  software  vendors,  announced  a  limited-disclosure  policy  which  requires  the\ndiscoverer to notify the vendors and give them some time before making the information\npublic. From our analysis, such a policy benefits vendors because limited disclosure gives\nthem time to release a patch for the vulnerability and availability of patch mitigates some\nadverse  effects  of  disclosure.    We  also  find  that  whether  the  vulnerability  is  reported  by\nvendors themselves, or other parties, it has essentially similar impact on market value. To\nan  extent,  our  study  points  that  vendors  are  not  necessarily  better  off  disclosing\ninformation themselves. Generally, an argument could be made that vendors should release\nthe  information  themselves,  for  if  not,  someone  else  will  and  it  will  lead  to  worse\nconsequences. However, we do not find any evidence of this. Vendors may be better off\nkeeping  quiet  and  integrate  their  fixes  as  either  service  packs  (which  do  not  give  micro-\ndetails on what it fixes) or newer versions and announce the patch only if someone else has\ndisclosed it. However, more research is required; especially more data on announcements\nwhen hackers exploit the vulnerabilities.\nAnother issue raised in discussing software flaws is whether software vendors should be\nheld responsible by law for vulnerabilities discovered in their products. Currently, the use\nof  End  User  License  Agreements  (EULA  or  Click-Wrap  Agreements)  that  come  with\nsoftware  products  limit  the  liability  that  vendors  face  if  customers  suffer  a  loss  due  to  a\nflaw in software. Our results show that liability laws (which anyway are not likely to pass\nsometimes soon) are not the only way to ‘punish’ software vendors for flaws discovered in\ntheir products. Market seems to act on this information and punishes a vendor, who on an\naverage, loses around 0.63% of its market value on the day a vulnerability is reported in its\nproducts.  Software  liability  could  certainly  cause  the  market  value  of  the  vendors  to\ndecline further if a vulnerability is reported in their product.\nSome  vulnerabilities  are  posted  on  a  public  listing  such  as  Bugtraq  before  these  are\nannounced in popular press or CERT. In that case, the actual vulnerability announcement\nmay  have  little  surprise  value.  Therefore  our  results  are  a  lower  bound  for  the  actual\ndecrease  in  stock  prices  experienced  by  the  software  vendor  if  a  flaw  is  reported  in  its\nproduct. We do not include the vulnerabilities reported on Bugtraq since most of these are\nnot confirmed vulnerabilities at the time they are posted online.\nA  limitation  of  our  study  is  that  most  of  the  data  points  in  our  sample  are\nannouncements  regarding  off-the-shelf  software  products.  Our  analysis  does  not  cover\nsoftware development projects where a security flaw can cause millions of dollars worth of\ndamage.  The  main  reason  for  excluding  them  was  the  lack  of  availability  of  data  on\nsoftware failures in such cases.  We also reiterate that further analysis in terms of software\nquality,  market  share  or  profitability  is  needed  to  fully  understand  how  vulnerability\ndisclosure  signals  poorer  quality  and  how  it  affects  the  vendors’  incentives  to  provide\nbetter quality software. Our paper takes the first step in this direction.\n•  Adams  EN  III  (1980)  ‘Minimizing  Cost  Impacts  of  Software  Defects’  IBM  Research\nReport, RC 8228 April\n•  Applewhite A (2004) ‘Whose Bug Is It Anyway? The Battle over Handling Software\nFlaws’ IEEE Software, March/April 2004, 94-97\n•  Anderson  R  (2001)  ‘Why  Information  Security  is  Hard  –  an  Economic  Perspective’\nProceeding of 17th Annual Computer Security Applications Conference, New Orleans,\nLouisiana\n•  Arora,  A.,  Caulkins,  J.P.  and  R  Telang,  (2004).  ‘Sell  First,  Fix  Later:  Impact  of\nPatching on Software Quality’, Carnegie Mellon University, working paper.\n•  Arora, A., Telang, R. and  H Xu, (2004). ‘Optimal  Policy  for  Software  Vulnerability\nDisclosure’, Carnegie Mellon University, working paper.\n•  Banker RD and SA Slaughter (1997) ‘A Field Study of Scale Economies in Software\nMaintenance’, Management Science, 43(12), 1709-1725\n•  Barki  H,  Rivard  S  and  J  Talbot  ‘Toward  an  Assessment  of  Software  Development\nRisk’, Journal of Management Information Systems, 10(2),  203-225\n•  Basili, VR and JD Musa (1991) ‘The Future Engineering of Software: A Management\nPerspective’ IEEE Computing, 20(4), 90-96\n•  Bharadwaj, A and M Keil (2003) ‘The Effects of Information Technology Failures on\nthe Market Value of Firms’, Working paper.\n•  Brown SJ and JB Warner (1980) ‘Measuring Security Price Performance’, Journal of\nFinancial Economics, 14(1985), 3-31\n•  Brown  SJ  and  JB  Warner  (1985)  ‘Using  Daily  Stock  Returns:  The  Case  of  Event\nStudies’, Journal of Financial Economics, 14(1985), 3-31\n•  Campbell JY, Andrew WL and AC MacKinlay (1997) ‘The Econometrics of Financial\nMarkets’ Princeton University Press\n•  Campbell  K,  LA  Gordon  LA,  Loeb  MP  and  L  Zhou  (2003)  ‘The  Economic  Cost  of\nPublicly  Announced  Information  Security  Breaches:  Empirical  Evidence  from  the\nStock Market’, Journal of Computer Security, 11(3), 431-448\n•  Carr  R,  Tynan  M  and  R  Davis  (1991)  ‘Quality  and  Security:  They  Work  Together’\nIEEE AES Systems Magazine, 6(9), 15-19\n•  Cavusoglu  H,  Mishra  B  and  S  Raghunathan  (2004)  ‘The  Effect  of  Internet  Security\nBreach  Announcements  on  Market  Value:  Capital  Market  Reactions  for  Breached\nFirms  and  Internet  Security  Developers’  International  Journal  of  Electronic\nCommerce, 9(1), 69\n•  Chaney  PK,  Devinney  TM  and  RS  Winer  (1991)  ‘The  Impact  of  New  Product\nIntroductions on the Market Value of Firms’, The Journal of Business, 64(4), 573-610\n•  Chatterjee  D,  Richardson  VJ  and  RW  Zmud  (2001)  ‘Examining  the  Shareholder\nWealth  Effects  of  Announcements  of  Newly  Created  CIO  Positions’,  MIS  Quarterly,\n25(1), 43-70\n•  Cooper MJ, O Dimitrov and PR Rau (2001) ‘A Rose.com by Any Other Name’, The\nJournal of Finance, (6), 2371-2387\n•  Cusumano,  MA  (2004)  ‘Who  is  Liable  for  Bugs  and  Security  Flaws  in  Software?’\nCommunications of the ACM, 47(3), 25-27\n•  Davidson WL III, DL Worrell(1992) ‘The Effect of Product Recall Announcements on\nShareholder Wealth’ Strategic Management Journal, 13(6), 467-473\n•  Devanbu P and S Stubblebine (2000) ‘Software Engineering for Security : a Roadmap’\nFuture of Software Engineering, ACM 225-239\n•  Dolley J (1933) ‘Characteristics and Procedure  of Common Stock Split-Ups,’ Harvard\n•  Dos  Santos  BL,  Peffers  K  and  D  Mauer  (1993)  ‘The  Impact  of  Information\nTechnology  on  the  Market  Value  of  the  Firm’,  Information  Systems  Research,  4\n(March), 1-23\n•  Gal-Or,  E.  &  A  Ghose  (2003).  ‘The  Economic  Consequences  of  Sharing  Security\nInformation’, In 2nd Workshop on Economics and Information Security, May 29-30.\n•  Gordon LA & MP Loeb(2002) ‘The Economics of Information Security Investments’\nACM Transactions on Information and Systems Security,5(4),438-457\n•  Gordon,  L.A.,  Loeb,  M.P.  &  Lucyshyn,  W. (2002) ‘An Economic Perspective on the\nSharing  of  Information  Related  to  Security  Breaches:  Concepts  and  Empirical\nEvidence’, In The 1stWorkshop on Economics and Information Security, May16-17.\n•  Gordon LA and MP Loeb and T Sohail (2003) ‘A Framework for Using Insurance for\nCyber Risk Management’ Communications of the ACM, 46(3), 81-85\n•  Harter  DE,  Krishnan  MS  and  Slaughter  SA  (2000)  ‘Effects  of  Process  Maturity  on\nQuality,  Cycle  Time,  and  Effort  in  Software  Product  Development’  Management\nScience, 46(4), 451-466\n•  Hendricks  KB  and  Singhal  VR  (1996)  ‘Quality  Awards  and  the  Market  Value  of  the\nFirm: An Empirical Investigation’ Management Science, 42(2), 415-436\n•  Hendricks KB and Singhal VR (1997) ‘Delays in New Product Introductions and the\nMarket  Value  of  the  Firm:  The  Consequences  of  Being  Late  to  the  Market’,\nManagement Science, 43(4), 422-436\n•  Hovav  A  and  J  D’Arcy  (2003)  ‘The  Impact  of  Denial-of-Service  Attack\nAnnouncements  of  the  Market  Value  of  Firms’,  Risk  Management  and  Insurance\nReview, 6(2), 97-121\n•  Hsiao, C (2002) ‘Analysis of Panel Data’ Cambridge University Press\n•\nIm  KS,  Dow  KE  and  V  Grover  (2001)  ‘Research  Report:  A  Reexamination  of  IT\nInvestment  and  the  Market  Value  of  the  Firm  –  An  Event  Study  Methodology’,\nInformation Systems Research, 12(1), 103-117\n•  Jarrell  G  and  S  Peltzman  (1985)  ‘The  Impact  of  Product  Recalls  on  the  Wealth  of\nSellers’ The Journal of Political Economy, 93(1), 512-536\n•  Kannan  K  and  R  Telang  (2004)  ‘Market  for  Software  Vulnerabilities?  Think  Again’\n•  Kannan  K,  Rees  J  and  S  Sridhar  (2004)  ‘Reexamining  the  Impact  of  Information\nSecurity Breach Announcements on Firm Performance’ Working Paper\n•  Lane  V  and  R  Jacobson  (1995)  ‘Stock  Market  Reactions  to  Brand  Extension\nAnnouncements: The Effects of Brand Attitude and Familiarity’, Journal of Marketing,\n59 (January), 63-77\n•  McGraw G (2004) ‘Software Security’ IEEE Security and Privacy, 2(2), 80-83\n•  Malatesta, P and R Thompson (1985) ‘Partially Anticipated Events: A Model of Stock\nPrice Reactions with an Application to Corporate Acquisitions’, Journal of Financial\nEconomics, 14(2), 237-250\n•  Paulk M Weber C, Curtis W and Chrissis M (1994) ‘The Capability Maturity Model:\nGuidelines for Improving the Software Process’ Carnegie Mellon University : Software\nEngineering Institute\n•  Pfleegar CP (1997) ‘The Fundamentals of Information Security’ IEEE Software, 14(1),\n•  Potter  B  and  G  McGraw  (2004)  ‘Software  Security  Testing’  IEEE  Security  and\n•  Slaughter SA, DE Harter and MS Krishnan (1998) ‘Evaluating the Cost of Software\nQuality’ Communications of the ACM, 41(8), 67-73\n•  Wallace L, Keil M and A Rai (2004) ‘How Software Project Risk Affects Project\nPerformance: An Investigation of the Dimensions of Risk and An Exploratory Model’,\nDecision Scineces, 35(2), 289-321\n•  Wang H and C Wang (2003) ‘Taxonomy of Security Considerations and Software\nQuality’ Communications of the ACM, 46(6), 75-78\n•  Westland, JC (2003) ‘The Cost Behavior of Software Defects’, Decision Sciences, 37,\n229-238",
         "",
         "",
         "",
         "",
         "and  the  Melissa  virus  (in  1999)  occurred  when  hackers  exploited  flaws  in  software.  The\ndamage  due  to  Code  Red  was  estimated  at  $2.1  bn  and  due  to  Melissa  at  $1.1bn3.  The\nGartner  Group  estimates  that  the  system  downtime  caused  by  security  vulnerabilities\nwould triple from 5% of the total downtime in 2004 to 15% of the total downtime in 2008.4\nIn  2003,  Computer  Emergency  Response  Team  (CERT)  reported  around  250,000  self\nreported incidences of system breaches, most of which exploit vulnerabilities in software\ncode  (Applewhite  2004).  Microsoft’s  $200m  campaign  for  .NET  was  marred  by  the\ndiscovery of a security flaw in Visual C++ .NET barely a month after Microsoft Chairman\nBill  Gates  directed  employees  to  focus  on  building  more  secure  software  (dubbed  the\n‘Trustworthy Computing’ initiative). Moreover, vulnerability disclosure is finding its way\ninto  firms’  strategy  toolkits  as  it  evident  from  a  WSJ  report  in  Feb  2004  that  software\nvendors are spending time and effort in discovering flaws in their rivals’ software products\nin  order  to  influence  the  rivals’  stock  prices.  For  example,  security  software  vendor  IDS\nreleased a vulnerability alert on rival Checkpoint’s firewall software on the day Checkpoint\nusually  holds  its  annual  US  investor  conference.  Some  examples  of  vulnerability\nannouncements reported in popular press are:\n•  News.com(04/25/2000)  “A  computer  security\nfirm  has  discovered  a  serious\nvulnerability  in  Red  Hat’s  newest  version  of  Linux  that  could  let  attackers  destroy  or\ndeface a Web site  - or possibly even take over the machine itself……..”\n•  WSJ(02/11/2004) “Microsoft Corp. warned customers about serious security problems\nwith  its  Windows  software  that  let  hackers  quietly  break  into  their  computers  to  steal\nfiles, delete data or eavesdrop on sensitive information……..”\nInspite  of  all  these  concerns  about  software  vulnerabilities,  not  much  has  been\nmentioned  in  literature  about  the  incentive  of  software  vendors  to  invest  in  defect-free\nsoftware. Literature on software risks fails to include any measure for security related risks\n(Wallace, Keil and Rai, 2004; Barki, Rivard and Talbot, 1993). The closest literature to the\ntopic of security vulnerabilities is that on software quality (Basili and Musa, 1991; Harter,\nKrishnan  and  Slaughter,  2000).  However,  in  traditional  literature,  quality  is  measured  in\nterms  of  reliability  and  integrity  of  the  source  code  –  which  essentially  tests  software\nagainst specified streams of input from users. However, in today’s internet age, software\ndesigners  must  not  only  think  of  users,  but  also  malicious  adversaries  (Devanbu  and\nStubblebine,  2000).  Some  quality  models  such  as  ISO9126  fail  to  include  computer\nsecurity  (Pfleegar  1997).  Therefore  software  which  has  been  certified  as  high  quality,\nbased  on  existing  definitions  of  software  quality,  can  have  many  security  flaws.\nResearchers in computer science are working on better integration of the two disciplines of\nsoftware  quality  and  software  security  while  designing  software  (Carr,  Tynan  and  Davis\n1991, Wang and Wang, 2003; McGraw 2004).\nSo far, there has only been anecdotal evidence that software vulnerabilities are causing\nvendors to lose market value. For example, the Wall Street Journal (11/09/2004) reported\nthat Microsoft’s Internet Explorer (IE) is losing market share in the web browser market to\ncompetitors like Mozilla’s Firefox, due to numerous flaws discovered in IE. For example,\nnearly  8  million  people  have  downloaded  the  Firefox  browser  between  September-\nNovember 2004. While the damage that firms suffer as a result of security breaches can be\nmeasured  in  terms  of  downtime  and  maintenance  activity,  the  cost  implication  of\nvulnerability disclosures on the software vendors is neither clear nor well studied.  This is\nthe major goal of this paper. Using an event study methodology, we estimate how the stock\nprice of the software vendors is affected when vulnerability information in their products is\npublished in popular press.\nPrior  literature  on  product  defects  (Jarrell  and  Peltzman  1985)  predicts  that  product\nrecall announcements in drug and auto industries are associated with loss in market value\nof a firm. Davidson and Worrell (1992) confirm the negative impact of product defects on\nstock prices in non-auto industries and also analyze the impact of different types of recall\n(e.g.  total  product  recall  versus  a  recall  for  repair;  government  ordered  recall  versus\nvoluntary recall). However the results of these prior studies on product defects cannot be\ndirectly  applied  to  the  software  industry  because  of  the  following  characteristics  of\nsoftware products: One, software products generally come with a click-wrap agreement (or\nEULA  –  End  User  License  Agreement)  which  limits  the  vendors’  liability.  Two,  the\ngeneral philosophy held by software vendors, software customers and the US courts is that\nsoftware is a uniquely complex product that will probably have some defects (Cusumano\n2004).  Therefore  it  is  not  clear  whether  markets  will  react  adversely  to  the  news  of  a\nsoftware  vulnerability;  because  over  the  long  run,  markets  will  anticipate  the  effect  of  a\nvulnerability announcements on cash flows of software vendors, so the impact of a specific\nannouncement  might  not  be  significant.  Three, software vulnerability announcements are\ngenerally  accompanied  with  a  remedial  patch  which  potentially  protects  customers  from\nmalicious  exploits.  Hence,  the  market  may  not  behave  adversely  towards  a  vendor  if  it\nreleases  a  patch  along-with  the  vulnerability  announcement.    Finally,  vulnerability\nannouncements  are  directly  related  to  the  installed  base  of  a  software  product.  Popular\nsoftware like those from Microsoft are constantly subject to malicious and non-malicious\nattacks  and  as  such  have  a  greater  proportion  of  flaws  reported  in  them  as  compared  to\nsoftware by Apple, where the user base is comparatively smaller. Therefore, the presence\nof  vulnerabilities  may  not  always  signal  a  lower  quality  product;  it  may  in-fact  signal\nsuperiority  over  competition.  E.g.  John  Thomson,  CEO  of  Symantec,  predicts  that  the\nflaws  in  Linux  will  likely  increase  as  the  installed  base  increases.  In  view  of  these\narguments, it would be interesting to understand whether and how the market responds to\nvulnerability disclosures in software products.\nMotivated  by  these  observations,  in  this  research  we  try  to  quantify  the  losses  that\nsoftware  vendors  bear  when  a  vulnerability  is  disclosed  in  their  product.  The  main\nquestions that we seek to answer are:\n1.  What  is  the  impact  of  vulnerability  disclosures  on  the  market  value  of  a  software\n2.  How do the characteristics of the vulnerability impact this change in market value?\nOur  research  has  important  policy  implications  in  terms  of  understanding  vendors’\nincentive to improve pre- and post-launch quality of their software products.  While there\nis lot of hype surrounding the poor quality dished out by software vendors, their incentives\nto provide more secure software is still unclear. If we indeed find that the market is willing\nto  punish  vulnerability  announcements  because  it  perceives  these  announcements  as  a\nsignal of poor quality software which either increase vendor costs to fix them or erode their\nmarket share in the long run, then our research provides a direct evidence of incentives to\nprovide  more  secure  software.  Vulnerabilities  are  disclosed  by  vendors  or  by  third\nparty/competitors  with  or  without  a  patch.  Since  we  measure  if  there  are  significant\ndifferences in market reaction to such disclosures, our paper provides policy guidelines to\nvendors about whether and how they may disclose the information themselves. Moreover,\nthis  also  provides  evidence  on  what  is  the  value  of  patches  when  vulnerabilities  are\ndisclosed.  Finally,  we  also  examine  how  vulnerability  characteristics,  effect  of  9/11  etc\naffect the market reaction.\nUsing  an  event-study  approach,  we  collect  data  on  about  146  vulnerability  disclosure\nannouncements over the period of over 5 years for 18 publicly traded vendors. Our results\nconfirm  that  vulnerability  disclosure  adversely  and  significantly  affects  the  stock\nperformance  of  a  software  vendor.  We  show  that,  on  average,  a  software  vendor  loses\naround  0.63%  of  market  value  on  the  day  of  the  vulnerability  announcement.  This\ntranslates  to  a  dollar  amount  of  $0.86  bn  loss  in  market  value.  We  also  find  that\nvulnerabilities  where  the  vendor  fails  to  provide  a  patch  at  the  time  of  disclosure  yield\nmore negative returns than when the vendor discloses a patch. On average, a vendor loses\n0.8% more in market value when a patch is not available. This provides evidence as to why\nvendors are trying to push for legalizing the limited disclosure norms. We also show that\nvulnerabilities  which  create  confidentiality  related  breaches  result  in  a  greater  loss  than\nvulnerabilities  which  create  non-confidentiality  related  breaches.  For  example,  vendors\nlose  0.75%  less  in  market  value  when  the  vulnerability  can  lead  to  a  non-confidentiality\nrelated breach than when it can lead to a confidentiality related breach. We also find that\nmore severe software vulnerabilities result in a greater loss in market value than less severe\nvulnerabilities.\nThe rest of the paper is organized as follows. In the next section, section 2, we provide\na literature review. We develop our hypotheses in Section 3. In Section 4, we discuss the\nmethodology of data collection and also describe the event study methodology. In Section\n5, we present our results using multivariate regression analysis and test various hypothesis\nrelated  to  how  the  vulnerability  characteristics  effect  the  change  in  stock  prices.  Finally,\nwe present the concluding remarks in Section 6.\n2. Literature Review\nMost  prior  research  on  information  security  discusses  the  economics  of  such\ninvestments from a customer perspective, rather than from a software vendor perspective\n(Anderson  2001).  Gordon  and  Loeb  (2002)  show  that  firms  should  make  investments  in\ninformation  security  far  less  than  the  expected  loss  from  a  security  breach.  Gordon  et  al\n(2002),  Gal-Or  and  Ghose  (2003)  discuss  the  economics  of  information  sharing  among\nfirms  on  security  related  issues.  Prior  event  study  analyses  on  information  security  have\nfocused on the change in market value of firms whose systems are breached (Cavusoglu et\nal (2004) and Kannan, Rees and Sridhar (2004)). These studies show that announcements\nof  a  security  breach  negatively  impact  the CAR  (Cumulative  Abnormal  Return)  of  firms\nwhose information systems have been breached. Campbell et al (2003) conduct a similar\nevent  study  and  find  that  only  the  impact  of  confidentiality  related  security  breaches  is\nnegative and significant; the impact of non-confidentiality related security breaches is not\nsignificantly  different  from  zero.    Hovav  and  D’Arcy  (2003)  show  similar  results  by\nfinding  that  Denial  of  Service  (DoS)  type  attacks  are  not  associated  with  any  significant\nloss in value for firms5.\nDisclosure of vulnerabilities has been one contentious area. Typically, a major portion\nof the vulnerabilities is reported by benign independent security analysts (ISA). Since no\nlegal  guidelines  exists  which  dictate  how  vulnerabilities  should  be  handled  by  the\ndiscoverer,  some ISAs report the vulnerability to the vendor and give it sufficient time to\ncome  up  with  a  patch  (The  OIS  recommends  a  time  period  of  30  days  to  be  given  to\nvendors  to  come  up  with  a  patch).  However,  some  other  ISAs  follow  the  policy  of  full\ndisclosure, i.e. they immediately post the vulnerability to a public listing like Bugtraq. One\nmajor goal of full disclosure is to eventually force vendors to come up with more secure\nsoftware.  The  debate  on  disclosure  policies  can  be  summarized  by  the  following  quotes\n(Applewhite  2000):  Jennifer  Granick  of  Stanford  University  defends  full  disclosure  as\n“You  want  a  free  flow  of  information,  just  like  in  other  scientific  fields,  ideally  without\nbetraying important interests.” Scott Blake of OIS opposes full disclosure as, “As soon as\nyou  tell  one  person,  you  have  tipped  the  balance  and  people  will  begin  producing\nexploits.”  Some  recent  academic  work  is  examining  this  issue  more  formally.  Arora,\nTelang and Xu (2004) study the optimal timing of vulnerability disclosure and show how\ndisclosure can force vendors to release patches quickly. Kannan and Telang (2004) explore\nthe welfare implications of a market mechanism for software vulnerabilities and report that\na market based mechanism for software vulnerabilities always underperforms a CERT-type\nmechanism.\nHowever  none  of  these  studies  measures  the  impact  of  disclosure  on  vendor’s  market\nvalue or profitability. While one major goal of full disclosure is to eventually force vendors\nto develop secure software, empirically, there is no evidence that suggests that disclosure\nindeed  creates  such  incentives.  Our  paper  provides  an  understanding  of  whether  such\ndisclosures create incentives for the vendors to produce secure software in the first place.\nOur  methodology  follows  closely  from  prior  event  study  analysis.  Campbell  et  al\n(1997)  present  a  useful  summary  of  the  event  study  analysis  highlighting  the  history  as\nwell  as  the  commonly  followed  methodologies.  Event  study  methodologies  are  well\naccepted for studying the implications of public announcements on stock prices. Hendricks\nand  Singhal  (1996)  study  the  impact  of  quality  award  winning  announcements  on  the\nmarket  value  of  firms  and  observe  positive  abnormal  returns  generated  by  winning  a\nquality  award.  They  further  note  that  awards  given  by  independent  organizations  and\nannouncements by small firms are more likely to have a significant impact on the firms’\nmarket  value.  In  the  field  of  Information  Systems,  Subramani  and  Walden  (2001)  used\nevent study analyses to show that e-commerce announcements lead to significant increases\nin the stock price of firms. Chatterjee, Richardson and Zmud (2001) examine the influence\nof  newly  announced  CIO  positions  on  shareholder  wealth.  Im,  Dow  and  Grover  (2001)\nexamine  the  changes  in  market  value  of  a  firm  in  response  to  IT  investment\nannouncements. Bharadwaj and Keil (2003) investigate the impact of IT failures on firms’\nmarket value.\nThe  main  contribution  of  this  research  is  that  this  is  one  of  the  first  comprehensive\nstudies,  to  our  knowledge,  that  has  tried  to  measure  the  impact  of  vulnerabilities  on\nsoftware  vendors.  Thus,  we  extend  prior  literature  on  product  defects  and  confirm  that\nsoftware  vendors  too  suffer  a  loss  in  market  value  when  a  flaw  is  discovered  in  their\nproduct.  This  is  inspite  of  the  fact  that  software  vulnerabilities  are  prevalent  among\nsoftware of almost all major vendors and that vendors face no legal liability if clients suffer\nlosses  due  to  the  software  flaw.  As  we  noted  in  the  introduction  that  this  has  important\nimplications in terms of vendors making investment in software quality as well as policy\nand legal issues which govern vulnerability disclosures.\n3. Hypotheses\nMuch of the prior literature on product defects (Jarrell and Peltzman (1985), Davidson\nand Worrell (1992)) shows that generally defective product and recall announcements are\nassociated  with  loss  in  market  value  of  a  firm.  Banker  and  Slaughter  (1998)  find  that\nunplanned and critical maintenance activities increase software maintenance costs.  Thus,\nfixing  bugs  entails  cost  to  firms.  For  example,  the  security  fixes  may  cost  about  $2000-\n$9000 when done during testing phase. However, they may cost more than 4-8 times when\nfixed after the application has been shipped.6 Slaughter et al (1998) and Westland (2003)\nsuggest  that  software  defects  are  harder  and  costlier  to  fix  if  discovered  later  in  the\nsoftware  development  cycle  (e.g.  when  the  product  has  been  shipped  to  the  customer).\nMoreover, security breach announcements by the user organizations have been known to\nhave a negative impact on the share value for firms. Cavusoglu et al (2004) show that the\nmarket  capitalization  values  of  firms  decreases,  on  average,  by  $2.1  billion  within  two\ndays  of  a  security  breach.  Cyber  insurance  firm  J.S.  Wurzler  charges  an  additional\npremium  to  firms  for  using  Windows  NT  due  to  the  number  of  security  breaches  in  the\nsoftware  (Gordon,  Loeb  and  Sohail,  2003).  Clearly,  poor  security  costs  the  users  of  the\nsoftware.  Thus, the cost to the vendor can be written as\nCost of vulnerability disclosure to vendor = Cost of patching the vulnerability + λ * (Cost\nto the users of the software due to exploitation, and/or cost to patch the system)\nHere,  λ  is  the  internalization  factor.  That  is  the  user  loss  that  is  internalized  by  the\nvendor  due  to  lost  sale  or  reputation  loss  (or  liability  if  imposed  in  future).  Clearly,  λ\ndepends on how willing users are to “punish” the vendor, how competitive the market is\nand the characteristics of vulnerability (Arora, Telang and Xu 2004). For example, this is a\n\"We  are  extremely  concerned  by  the  high  amount  of  vulnerabilities  and  patches  from\nMicrosoft. This goes against the credibility of what they have been saying,\"\nMichael Kamens, global security director at Thermo Electron Corp.7\nGiven the fact the competitors seem to be using vulnerability disclosure as a strategic way\nto  undermine  the  rivals’  reputations,  it  seems  that  vulnerability  disclosure  signals  a\npotential loss in future cash flows for a software vendor owing to customer dissatisfaction\nbecause customers suffer a loss if their systems get breached. It also signals an increase in\nproduct  related  costs  due  to  the  time  and  effort  that  the  vendor  spends  in  developing  a\npatch or a fix for the flaw. Therefore, we hypothesize that\nH1: A software vendor suffers a loss in market value when a security related vulnerability\nis announced in its products.\nOur second hypothesis pertains to whether the software vendor releases a patch for the\nproduct  at  the  time  of  the  vulnerability  announcement.    As  per  the  popular  convention\nfollowed in the vulnerabilities market, vendors are given some time to work on a patch for\nthe vulnerability before it is made public. Vendors may also provide a workaround (such as\ndisabling features of the software) when a vulnerability is disclosed and choose to address\nthe  vulnerability  in  a  future  upgrade.  Presence  of  the  patch  is  also  likely  to  reduce\ncustomers’  loss  if  they  apply  the  patch.  Since  presence  of  patch  also  reflects  vendor’s\ncommitment  to  its  customers  we  expect  that  vulnerabilities  disclosed  with  a  patch  will\ncompensate, to an extent, the negative signal due to vulnerability disclosure. Vendors are\nalso  pushing  for  limited  disclosures  so  that  they  can  release  the  remedial  patch  in  time.\nThis also suggests that patches play a critical role. Therefore our second hypothesis is:\nH2: CAR (Cumulative Abnormal Return) [negative] of a stock is greater for vulnerabilities\nwhere  the  software  vendor  does  not  release  a  patch  at  the  time  of  the  vulnerability\ndisclosure.\nThe  damages  suffered  by  a  software  vendor’s  customers  due  to  a  vulnerability  in  the\nvendor’s software  depend on the  type of security breach that the vulnerability  facilitates.\nCampbell  et  at  (2003)  classify  the  security  breaches  as  confidentiality  related  and  non-\nconfidentiality  related.  Confidentiality  related  breaches  involve  attacks  where  an  intruder\ncan  gain  access  into  a  system  and  can  steal  sensitive  information.  Non-confidentiality\nrelated breaches include attacks like denial of service (DoS) attacks where the most likely\nscenario  is  a  disruption  and/or  a  downtime.  Campbell  et  al  (2003)  further  show  that  the\nloss  in  market  value  is  more  for  confidentiality  related  breaches  than  for  non-\nconfidentiality related breaches. Hovav and D’Arcy (2003) show that DoS attacks are not\nassociated with any significant loss in market value for a firm. Therefore we would expect\nthat  the  vendor  loses  more  market  value  if  the  vulnerability  in  its  software  causes  a\nconfidentiality  related  breach.  The  intuition  is  that  the  negative  reaction  due  to  the\nvulnerability  is  more  if  the  customers  can  potentially  suffer  greater  losses  due  to  the\nvulnerability. Therefore, our next hypothesis is:\nH3:  CAR  is  greater  for  a  vulnerability  which  can  potentially  cause  a  breach  in\nconfidentiality as compared to non-confidentiality related breaches.\nThe  impact  of  a  software  flaw  on  a  vendor  also  depends  on  how  severe  the\nvulnerability is. Davidson and Worrell (1992) conduct an event study with product defect\nannouncements  in  the  tire  industry  and  showed  that  the  impact  of  severe  flaws  (which\ninvolve  a  recall)  is  more  than  that  of  less  severe  flaws  (which  involve  repairs  but  nor\nrecall). Sometimes an ‘exploit’ exists for the vulnerability at the time of announcement. An\n‘exploit’  is  a  piece  of  code  which  anyone  can  use  to  compromise  the  security  of  the\nsoftware product and subsequently of other information assets. Therefore, we propose our\nnext set of hypotheses as:\nH4a:  The  loss  in  market  value  of  a  software  vendor  is  greater  if  the  announced\nvulnerability has a higher severity.\nH4b: The loss in market value of a software vendor is greater if an exploit exists publicly\nat the time of the vulnerability announcement.\nA  recent  article  in  the  Wall  Street  Journal  hinted  that  firms  are  using  vulnerability\ndisclosure as a strategic weapon against competitors. E.g. ISS disclosed a vulnerability in\nrival  Checkpoint’s  flagship  firewall  product  just  ahead  of  Checkpoint’s  investor  summit.\nVendors themselves disclose vulnerability information in their products routinely. In fact,\nmany believe that vendors would prefer not to disclose information at all but they fear that\nsomeone  else  would  do  it.  Generally  vendors  are  likely  to  be  more  careful  about  the\ndisclosure  as  opposed  to  third  party.  Moreover,  disclosure  by  vendors  would  signal  their\ncommitment to providing secure software, we hypothesize that\nH5:  The  loss  in  market  value  for  a  software  vendor  is  lower  in  case  the  security\nvulnerability is discovered by the vendor itself rather than by rivals or third party security\nfirms.\nOur  final  hypothesis  relates  to  the  publication  where  vulnerability  is  reported.  Some\nvulnerabilities are reported in popular press. Others find mention only in industry sources\nlike  CERT  or  technology  news  portals  like  news.com.  The  question  which  arises  is\nwhether  both  sources  are  equally  influential  in  affecting  a  stock’s  CAR?  One  would\nsuspect the popular press of being more influential than industry sources because it reaches\na broader audience. So our next hypothesis is:\nH6:  The  magnitude  of  CAR  is  more  when  the  vulnerability  is  reported  in  popular  press\nthan in industry sources.\n4. Data Description & Methodology\n4.1 Vulnerability Disclosure Process\nThe typical process of vulnerability disclosure takes place as shown in Figure 1.\nVulnerability\nDiscovery\nThe  process  starts  with  the  vulnerability  discovery.  There  are  various  sources  for\nvulnerability discovery. Some vulnerabilities are discovered by amateur researchers or by\nindependent  security  analysts  (ISA)  like  eEye  Security.    If  the  ISA  chooses  to  follow\n‘limited  disclosure’,  it  reports  the  vulnerability  to  the  concerned  vendor  or  to  an\nindependent body like CERT who disclose the vulnerability to public later (generally after\nthe  fix  has  been  made  available)  in  a  limited  way8.    Sometimes,  the  vendor  itself  might\ndiscover  a  vulnerability  in  its  products.  On  the  other  hand,  if  the  ISA  chooses  to  follow\n‘full  disclosure’  or  if  a  hacker  discovers  the  vulnerability,  the  vulnerability  may  get\nannounced  directly  to  public.  Forums  such  as  Bugtraq  allow  for  full  disclosure  of\nvulnerability  information.  In  any  case,  after  the  vulnerability  has  been  announced,\ninvestors  might  re-evaluate  their  projections  on  the  software  vendors’  profitability  based\non  the  vulnerability  characteristics.  Consequentially,  the  stock  price  of  the  vendor  might\nshow an abnormal return due to the vulnerability announcement.\n4.2 Data\nOur data comes from the vulnerability disclosures in popular press as well as from the\nadvisory  reports  in  CERT.  We  include  articles  published  by  news  networks  like\nBusinesswire,  Newswire  and  daily  articles  in  popular  press  like  WSJ,  NY  Times,\nWashington Post and LA Times. We search for these news articles in Proquest and Lexis-\nNexis  Academic  databases  which,  between  them,  maintain  news  articles  from  major\nnewspapers  and  news  networks  all  over  the  country.  We  also  include  articles  from\nnews.com  which  is  a  CNET  owned  site  and  is  a  premier  source  for  round-the-clock,\nbreaking  technology  news  coverage.  We  used  the  following  terms  in  our  search:\n‘vulnerability  AND  disclosure’,  ‘software  AND  vulnerability’,  ‘software  AND  flaw’,\n‘virus AND vulnerability’& ‘vulnerability AND patch’.\nWe  also  searched  the  vulnerability  announcements  for  information  on  the  type  of\nvulnerabilities. Based on this, we classify vulnerabilities into various categories.\n•  If  the  announcements  contained  words  such  as  ‘serious’  or  ‘severe’  or  ‘dangerous’  to\ndescribe  the  vulnerability,  we  characterized  the  vulnerability  as  ‘Severe’.  If  the\nannouncement  characterized  the  vulnerability  as  ‘moderately  severe’  or  ‘with  low\nseverity’, we characterized it as ‘Non-severe’.\n•  The  vulnerability  announcements  also  had  references  to  what  kind  to  security  breach\ncould be facilitated if attackers exploited the vulnerability. If the vulnerability contained\nterms  such  as  ‘cause  denial  of  service’,    or  ‘disrupt  operations’,  we  classified  the\nvulnerability as type ‘DoS’; otherwise if the vulnerability contained terms such as ‘gain\naccess’  or  ‘steal  information’  or  ‘take  control’,  we  classified  the  vulnerability  as\n•  Further,  the  announcements  also  described  whether  the  vendor  released  a  patch  at  the\ntime of the vulnerability announcement. If the vendor announced a patch at the time of\nvulnerability disclosure, we classified the vulnerability as ‘Fix Available’.\n•  Finally,  we  also  classify  vulnerabilities  on  whether  an  ‘exploit’  exists  for  the\nvulnerability  in  the  public  domain.  If  the  vulnerability  announcement  contained  terms\nsuch as ‘an exploit for the vulnerability is circulating’, we classify the vulnerability as\ntype ‘Exploit’.\nAs per convention in prior event study literature (Hendricks & Singhal 1996), we excluded\nthe following type of announcements from our sample:\n•  Vulnerability  announcements  in  non-daily  periodicals  like  magazines  because  of  the\ndifficulty in determining the exact date of the announcement.\n•  Repeat  announcements  of  the  same  event  in  a  different  publication  at  a  later  date.  In\ncase of such repeat announcements, the earliest announcement date was chosen as the\n•  Announcements which were associated with other confounding events like stock splits\nand mergers on the event date.\n•  Announcements related to firms not traded on any public exchange in the US.\n•  Announcements  that  point  to  a  fundamental  protocol  flaw  rather  than  a  particular\nsoftware. E.g. a flaw in the FTP protocol affects multiple vendors. The reason behind\ndropping this category is that the flaw exists in the software only because it follows a\nflawed protocol, and not due to the vendor.\n•  Software flaws which are not security related.\nOur dataset contains 146 vulnerability announcements pertaining to around 18 firms in\nthe time period January 1999 – May 2004. We capture information on the following details\nabout the vulnerability announcement: date, firm name, product, who discovered the flaw,\nnews source, whether patch is available and severity. The descriptive statistics of the data\nare given in Table 1.\nNumber of firms\n%age of vulnerabilities announcements in popular press\n%age of vulnerabilities for which vendor has patch available at the\ntime of the announcement.\n%age of vulnerabilities discovered by the vendor itself\n% of vulnerabilities that could potentially result in a security breach\nrelated to confidentiality\n%age of vulnerabilities for which the announcement contained\ninformation of a publicly available ‘exploit’\nThe year-wise distribution of announcements is given in Table 2 below.\nYear\n4.3 Methodology\nWe use the standard event study methodology for this analysis. An event study assumes\nthat returns on a stock are significantly impacted by an event of interest (in our case, the\nevent of interest is the vulnerability disclosure announcement).  The period of interest for\nwhich we observe the event is known as the event window. The smallest event window is\none  day  (day  of  the  announcement  or  day  ‘0’9).  In  practice,  the  event  window  is  often\nexpanded  to  include  two  days  (day  0  and  day  110)  to  capture  the  effect  of  price\nannouncements  made  after  the  close  of  the  markets  on  a  particular  day.  Sometimes\nresearchers include a day before the announcements to incorporate any information leaks\nabout the event. In our study we define a one day event window (day 0)11. Hendricks and\nSinghal (1996) cite two reasons to use a one day event period. One, a shorter event period\npermits a better estimation of the effects of information of stock prices since it reduces the\npossibility  of  other  confounding  factors  not  related  to  the  announcement.  Two,  it  also\nincreases the power of the statistical tests.\nAbnormal  returns  are  defined  as  the  difference  between  the  actual  return  of  the  stock\nover the event window minus the expected return of the stock over the event window. The\nexpected return on the stock is calculated in several ways, but in our analysis, we use the\nmarket  model  which  assumes  a  stable  linear  relation  between  the  market  return  and  the\nreturn  on  the  stock.  We  also  verify  our  results  using  other  methods  such  as  the  market-\nadjusted  method  and  the  mean-adjusted  method.  The  coefficients  of  the  linear  model  are\ncalculated  by  choosing  a  portion  of  the  data  as  the  estimation  window.  The  estimation\nwindow,  generally  between  120  days  and  200  days  used  in  most  studies,  is  the  period\nimmediately before the event window. In our case, we use an estimation window of size\n160 days, from day -175 to day -16.\nThere  are  three  main  methods  followed  in  the  event  study  methodology  (Campbell,\nAndrew  and  MacKinlay,  1997;  Hendricks  and  Singhal,  1996)  to  estimate  the  abnormal\n1. The Market Model\nIn the market model, the abnormal returns are estimated as follows:\nwhere i denotes the event (i=1,2……N), m denotes the market and t denotes the  day of\nthe event (e.g t = 0 denotes the day of the vulnerability announcement.). ARit denotes the\nabnormal return of event i at time t, Rit denotes the actual return and Rmt denotes the market\nreturn  at  time  period  t13.\ndenotes  the  normal  return  of  the  firm  due  to  the\nmarket-wide  movement.  The  abnormal  return  is  defined  as  the  difference  between  the\nactual  return  and  the  normal  return.  This  is  the  part  of  the  actual  return  that  cannot  be\nexplained  by  market  movements  and  captures  the  effect  of  the  event.  Since  most  of  the\ntech stocks are listed on NASDAQ, we use this as our indicator for market returns. We use\nordinary  least  squares  regression  to  estimate  the  coefficients  α  and  β  for  the  above\n2. The Market Adjusted Model\nIn this case, the abnormal returns are given as\n=\nwhere the terms have the usual meaning as in the Market Model.\n3. The Mean Adjusted Model\nis  the  mean  return  on  the  stock  which  made  a  vulnerability\nannouncement during event i,  over the duration of the estimation period and T is number\nof days in the estimation period (in our study, T=160).\nThe  mean  abnormal  return  across  all  observations  on  day  t  of  the  event  is  given  as\n. The cumulative abnormal return\nfor the event is defined as the\nsum  of  the  abnormal  returns  over  the  event  window.  Prior  research  on  event  studies\n(Brown  &  Warner  1980,  Brown  &  Warner  1985)  presents  a  comprehensive  analysis  of\nby  more  than  one  vendor  on  a  given  day,  our  statistic  should  allow  for  event  day\nclustering.  The  following  t-statistic  proposed  by  Brown  &  Warner  (1985)  takes  in  to\naccount event day clustering as well as cross-sectional dependence in the security specific\nt =\nA\nt\nS\nWhere\n⎛\n⎜\n⎝\n(\n−\nand T is the number of days in the estimation period\nand\n⎛\n= ∑\n⎜\nT\n⎝\n⎞\n⎟\n⎠\nThe  null  hypothesis  is  that  the  abnormal  returns  are  not  significantly  different  from\nzero.  Under  the  null  hypothesis,  the  abnormal  returns  are  independent  and  identically\ndistributed  and  normal  with  a  mean  of  zero  and  the  variance  given  by  the  variance  of\nabnormal returns over the estimation period.\nWe  also  do  multivariate  regression  and  test  our  hypothesis  of  how  CAR  varies  with\nvarious vendors, or vulnerability characteristics. But before we present the details on our\nregression analysis, we present the results of the event study.\n4.2 Event Study Results\nvulnerability disclosures on the stock prices of software vendors for our entire sample of\n146 announcements (p-values are in parenthesis)\nDay 0     CAR\nMarket Model  Market\nMean Model\n-0.5    (0.09)\n-0.55  (0.01)\nPercent Less than Zero\n58.7%  (0.03)\nAdjusted Model\nWe  calculate  CARs  under  there  different  models  (Market  Model,  Market  Adjusted\nModel and Mean Adjusted Model).  For each of the three models, we use three different\ntest  statistics  (Mean  Abnormal  Return,  Median  Abnormal  Return  and  Percent  Less  than\nZero). The Mean Abnormal Return Test (equations (1) to (4)) is parametric in nature and\nmakes  assumptions  about  the  distribution  of  abnormal  returns.  We  also  use  two  non-\nparametric tests to strengthen our results. One, we use the Wilcoxon Signed Rank Test to\ncalculate the p-value for the median abnormal return; two we use the Sign Test to calculate\nthe p-value for the percent negative returns. The Wilcoxon Signed Rank Test tests whether\nthe  median  of  the  abnormal  returns  is  different  than  zero.  The  Sign  Test  is  based  on  the\nsign  rather  than  the  magnitude  of  the  abnormal  returns  and  requires  that  under  the  null\nhypothesis, the proportion of abnormal returns greater than (or less than) zero is 50%.\nFrom Table 3, we note that the CAR for day 0 is negative across all the three different\nmodels. E.g. the Mean Abnormal Return varies between 0.5% - 0.67% depending on the\nmodel  used.  Further  the  Market  Model  and  the  Market  Adjusted  Model  are  statistically\nsignificant at p < 0.01, while the Mean Adjusted Model is statistically significant at p < 0.1\nlevel.  The Median Abnormal Returns range between 0.44% - 0.55% and are significant at\nthe 0.01 level. Finally, the percent observations less than zero range between 57.8% - 64%\nand  are  significant  at  the  0.05  level.  It  is  clear  that  CAR  is  negative  and  statistically\nsignificant for all three models and all three tests.\nThus,  our  results  suggest  that  software  vendors  do  tend  to  lose  market  value  when  a\nvulnerability is announced in their product. To check for the robustness of our results, we\nnot  only  test  the  mean  abnormal  returns,  but  also  the  median  abnormal  returns  and  the\npercent  less  than  zero.  This  provides  support  for  hypothesis  H1  that  vulnerability\nannouncements are associated with a loss in market value of software vendors.\nWe also calculate the abnormal returns using different event windows (beyond 0 days)\nusing  the  market  model  (the  results  do  not  change  substantially  for  other  models).  The\nresults are given in Table 4.\n0 to 10\n(0.36)\nFrom the table, it is clear that the CAR on day 0 is negative and significant at the 0.01\nevel. However, CAR for day 0 and day 1 combined is significant only at the 0.1 level. This\nsuggests  that  the  stock  market  is  efficient  in  the  sense  that  the  effect  of  a  software\nvulnerability  announcement  is  quickly  incorporated  into  a  vendors’  stock  price.  The  p-\nvalues for day -1 is not statistically significant. A possible explanation for this is that the\neffect of news leakage through forums like SecurityFocus is not significant. The CARs in\ncolumns  3,  4,  5  and  6  are  negative  but  not  statistically  significant.  However,  it  is\ninteresting to note that the CARs are negative for even a 10 day window.  Figure 2 shows\nthe CAR over a 16 day event window (from day -5 to day 10).\nOur result corroborates prior work on defective products (Jarrell et al 1985, Davidson et\nal 1992) by showing that product defects lead to a loss in market value of a firm. Our study\nanalyzes returns on stocks of software vendors and we find that a defective software which\ncompromises the security of customers’ information systems leads to a negative impact on\nthe  market  value  of  the  software  vendor.    The  extent  of  losses  suffered  by  a  vendor,  on\naverage,  is  moderate  at  about  0.63%  of  its  market  capitalization  value  on  the  day  the\nvulnerability gets announced.\nMarket Capitalization\nWe  also  calculated  the  abnormal  change  in  market  capitalization  values  of  the  software\nvendor due to the vulnerability announcement14. For each firm, the day 0 change in market\ncapitalization  value  was  calculated  by  multiplying  the  day  -1  market  capitalization  value\nwith the abnormal returns on day 0. On average, we calculate that the software vendors in\nour sample lost $0.86 billion in market capitalization value on the day of the vulnerability\ndisclosure.  Since Microsoft accounts for more than 40% of our sample, we sub-divide our\nsample into Microsoft and non-Microsoft samples. For the Microsoft sample, the average\nchange in market capitalization is around $0.92 billion and for the non-Microsoft sample,\nthe average change in market value is $0.81billion.\nRobustness Checks: We also perform the following robustness checks on our results, as\nspecified in the event study by Cooper, Dimitrov and Rau (2001).\n1.  Robustness to Outliers:\nTo  check  the  robustness  of  our  results  to  exclude  the  effect  of  outliers,  we  compute  the\nCAR for our sample after excluding the top 10 percentile and the bottom 10 percentile of\nobservations  (ranked  according  to  the  day  0  mean  abnormal  returns).  We  find  that  our\nresults remain qualitatively the same. For example, mean abnormal returns for this sample\nare 0.53% (against 0.63% for the entire sample) and these are significant at the 5% level.\nThis suggests that our results are robust to outliers in the data.\n2.  Momentum Effect:\nIt can be argued that the day 0 abnormal returns are caused simply by market momentum\nthan by the underlying event. As a simple check, we compute the correlation between the\nabnormal returns before the event and those after the event. In specific, we check the pair-\nwise correlation (along with level of significance) for three pairs of values : one, day -10 to\nday -1 CAR and day 0 to day 10 CAR; two, day -10 to day -1 CAR and day 0 CAR; and\nthree, day -1 CAR and day 0 CAR. The pair-wise correlations are as follows:\n•  day -10 to day -1 CAR and day 0 to day 10 CAR (correlation: 0.13, p-value 0.12)\n•  day -10 to day -1 CAR and day 0 CAR (correlation: -0.05, p-value 0.5)\n•  day -1 CAR and day 0 CAR (correlation: 0.03, p-value 0.67)\nThus  we  find  that  none  of  the  correlations  is  strong  or  significant  at  the  10%  level  and\nhence the momentum in the stock prices does not seem to be driving our results.\n5. Regression Analysis\nTo test the other hypothesis, we develop both a regression model and simple means test\nto explain the effect of various vulnerability characteristics on abnormal returns. However,\nordinary  least  squares  model  might  not  be  appropriate  because  it  does  not  account  for\nheterogeneity  among  firms.  The  issue  of  heterogeneity  is  an  important  consideration  in\nanalyzing panel data. For example, the level of abnormal returns could differ across firms\nif investors use different valuation models  across  firms  (Malatesta  and  Thompson  1985).\nEstimating  aggregate  parameters  while  ignoring  heterogeneity  could  lead  to  biased  and\ninconsistent estimates (Hsiao 1986).\nTo incorporate the impact of firm specific heterogeneity in our data, we propose a fixed\neffects  model.  The  fixed  effects  model  controls  for  unobservable  firm  specific  variables\nthat are constant over time. This is equivalent to generating dummy variables for each firm\nand including them in an ordinary linear regression to control for firm specific effects. The\nfixed effects model can be specified as:\n=\n+⋅\nεµβ\nit\ni\nwhere i = 1…..N (N is the total number of firms) and t = 1……..T (T is the total number\nof events). yit is the Abnormal Return (ARit) for firm i at event t as calculated according to\nthe  market  model  in  equation  (1)15.  Xit  are  the  independent  variables  which  capture  the\nvarious  vulnerability  characteristics,  ui  is  the  firm  specific  dummy  variable.  The\ndescription of the independent variables is as follows:\nSEVR:  whether  the  vulnerability  has  been  classified  as  severe;  SEVR  =  1  for  a  severe\nvulnerability and 0 otherwise.\nPATCH: Whether a patch is available at the time of the vulnerability disclosure. PATCH\n=1 if a patch is available and 0 otherwise.\nDISC:  Whether  the  vulnerability  was  disclosed  by  the  third  party.  DISC  =  1  if  the\nvulnerability was disclosed by the third party and 0 if disclosed by the vendor.\nEXPLOIT:  If  an  exploit  is  publicly  available  at  the  time  of  the  vulnerability\nannouncement, then EXPLOIT = 1; otherwise it is zero.\nCERT: If the vulnerability was first reported in CERT, then CERT = 1, it is 0 otherwise.\nPRESS: If the vulnerability was first reported in popular press, the variable PRESS = 1,\notherwise it is zero.16\nDOS:  If  the  vulnerability  can  potentially  lead  to  a  denial  of  service  type  attack  or  a\ndisruption in services, then the variable DOS =1, otherwise it is zero.\nEffect of Investor Sentiments:\nTo control for abnormal returns due to overall market sentiments, we introduce a set of\ndummy  variables  based  on  the  time  when  the  vulnerability  was  announced.  We  use  the\nevents surrounding 9/11 as the basis for segmenting our sample into various time periods.\nThe stock market crash in late 2000 could also play a role in the negative abnormal returns.\nWe introduce the following dummy variables in our model:\nPost_911: If the vulnerability was announced within a year of 9/11, i.e. if the date of\nthe  announcement  was  between  Sept  11,  2001  and  Sept,  11  2002.  This  constitutes  19\nobservations from our sample.\nPre_911: If the vulnerability was announced within a year before 9/11, i.e. if the date of\nthe announcement was between Sept 11, 2000 and Sept, 11 2001. This was also the time\nafter the stock market crashed in mid 2000 and lasted till the first three quarters of 2001\n(Wall  Street  Journal,  200017,  Kannan  et  al  2003,  Wall  Street  Journal,  200318).  This\nconstitutes 28 observations from our sample.\nYear_99-00: if the announcement was made prior to September 11, 2000, i.e. if the date\nof the announcement is between Jan 1, 1999 and September 11, 2000.  This constitutes 23\nobservations from our sample.\nYear_02-03:  If  the  date  of  the  announcement  is  between  September  11,  2002  and\nSeptember 11, 2003. This constitutes 44 observations from our sample.\nYear_03-04: If the date of the announcement is between September 11, 2003 and June\n1, 2004. This constitutes 32 observations from our sample. This is the baseline category for\n5.1 Results\nWe  first  compare  the  mean  CARs  of  different  sub  samples  based  on  vulnerability\ncharacteristics  to  understand  how  various  factors  affect  market  value  of  the  firms.  The\ncharacteristics that we consider are explained above. This is similar to the method followed\nby Subramani and Walden (2001), Chatterjee, Richardson and Zmud (2001) Im, Dow and\nGrover (2001). The results are summarized in Table 5.\nVulnerability\nCharacteristic\nVariable\np-value\nHigh Severity (81%)\nSeverity\nLow Severity (19%)\nSource of Discovery\nDiscovered by Vendor\n(36%)\nDiscovered by Third\nParty (64%)\nPatch Available (24%)\nPatch Availability\nPatch Not Available (76%)\nCERT (34%)\nSource of Disclosure\nPress (35%)\nNews.com (30%)\nDenial of Service (24%)\nType of Attack Possible  Confidentiality (76%)\nMicrosoft vs Non\nMicrosoft\nMicrosoft (46%)\n0.13\nFrom the means analysis it is clear that more severe vulnerabilities affect the stock price\nmore.  Surprisingly,  when  vendors  disclose  the  information  then  the  investors  seem  to\nperceives  it  more  negatively  than  when  some  third  party  releases  it.  As  expected,\navailability  of  patch  mitigates  the  negative  impact  of  disclosure  somewhat.  Since  the\nvendors  typically  disclose  the  information  with  a  patch,  we  may  find  during  the\nmultivariate regression that vendor release effect may disappear. The source of disclosure\nis  also  relevant.  Investors  seem  to  pay  more  attention  to  vulnerabilities  published  in\nmainstream newspapers than to CERT. Confidentially breaches are considered more severe\nand hence have higher significant impact than denial of service vulnerabilities. Finally, we\ndo not find an evidence of Microsoft effect though, non-Microsoft firms seem to lose more\nvalue.\nIn  the  second  method,  we  test  our  hypothesis  related  to  the  impact  of  various\nvulnerability  characteristics  on  the  market  value  of  a  software  vendor  using  the\nmultivariate regression model outlined in equation (5). This is similar to the methodology\nfollowed  by  prior  event  studies  such  as  Hendricks  and  Singhal  (1997),  Chatterjee,\nRichardson and Zmud (2001), Lane and Robertson (1995). The regression method has the\nadvantage over the sub-sample method used in Table 5 that the regression method captures\nthe  effect  of  all  the  independent  variables simultaneously.  The  sub-sample  method  could\ngive spurious results because the effects of a sub-sample, such as patch vs no-patch, can be\nexplained by a relationship between patch and other independent variables. The parameter\nestimates as well as the p-values for the various parameters of our fixed effects regression\nare given in the Table 6 below.\nVariable\nSEVR\nEXPLOIT\nPATCH\nDISC\nCERT\nPRESS\nDOS\nY_99-00\nPre_911\nPost_911\nY_02-03\nConstant\n-0.007\n-0.011*\n-0.02*\n-0.01*\n0.01*\n0.05\nThe  R2  for  this  regression  is  16.6%.  The  F-test  (p-value  0.001)  for  the  overall  model\nsuggests that our model is highly significant. The constant in the regression is simply the\naverage  of  all  the  fixed  effects  in  the  model.  Our  regression  provides  several  interesting\nobservations regarding the effect of the vulnerability characteristics on the stock price of\n1.  Severity: The coefficient on the SEVR variable in Table 6 is negative and significant\n(though  at  p  =  0.1).  This  is  consistent  with  results  in  Table  5.  More  severe\nvulnerabilities have a higher potential to cause damage and hence have a larger adverse\nimpact  on  CAR.  On  average,  a  severe  vulnerability  can  cost  a  software  vendor  0.6%\nmore than a non-severe vulnerability, providing support to hypothesis H4.\n2.  Patch:  We  also  find  that  the  non-availability  of  a  patch  is  positive  and  significantly\ncorrelated with the market value. This provides support to hypothesis H2 because the\ncoefficient  on  the  PATCH  variable  suggests  that  on  average,  firms  which  do  not\nprovide a patch at the time of the vulnerability disclosure suffer a loss of 0.83% more\nthan firms which provide a patch.\n3.  Confidentiality:  The  coefficient  of  the  DOS  variable  is  0.0076.  This  implies  that\nvulnerabilities which result in a non-confidential breach (denial of service, disruptions\nin  service)  have  an  abnormal  return  of  0.76%  higher  than  the  vulnerabilities  which\nresult in a confidentiality-related breach. This confirms our hypothesis H3.\nNone of the other coefficients are significant (except time coefficients). Thus our results\nfail to provide support for hypothesis H5 and H6. From table 5 also, we can observe that\nnone  of  the  sub-samples  (other  than  the  Press  sub-sample)  has  a  significant  coefficient.\nThe rejection of hypothesis H5 is especially interesting because it suggests that the markets\ndo not penalize a vendor any more if the vulnerability is discovered by a third party than\nby the vendor itself.\nWe also control for the effect of the time period during which the vulnerabilities were\nannounced  to  control  for  time-specific  investor  sentiments  which  can  affect  abnormal\nreturns.  We  find  that  software  vendors  suffered  the  greatest  abnormal  returns  in  the  one\nyear period post 9/11; for example, on average, vendors lost 2% more in market value for\neach vulnerability announcement in the year following 9/11 than they did in the baseline\nperiod  (2003-04).  This  suggests  that  security  concerns  among  investors  were  highest\nduring this period, as an aftermath of September 11. The next highest period of abnormal\nreturns was the one-year time frame after the dot-com crash, i.e. from Sept 2000 to Sept\n2001.  In  this  period,  software  vendors  lost,  on  average,  1.0%  more  on  vulnerability\nannouncements than in the baseline period. The results also suggest that year 2003-04 is\nwhen the vendors suffered the least. It is possible  that  vulnerability  announcements  have\nbecome  commonplace  and  are  not  affecting  the  vendors  as  much.  But,  more  research  is\nneeded to provide a definitive answer.\n6. Conclusions and Discussion\nTo  the  best  of  our  knowledge,  this  is  the  first  study  to  analyze  the  impact  of  product\ndefects on software vendors. We also analyze the information content of the vulnerability\ndisclosure  announcement  and  classify  vulnerabilities  into  various  sub-types  based  on  the\nthe  source  of  vulnerability  disclosure,  severity  of\nvulnerability, availability of a fix, whether an exploit was publicly available at the time of\ndiscovery,  the  type  of  security  breach  caused  by  the  vulnerability  and  the  source  of\nvulnerability discovery. Our results show that vulnerability disclosure leads to a significant\nloss  of  market  value  for  software  vendors.  This  indicates  that  the  stock  markets  react\nnegatively  to  the  news  of  a  vulnerability  disclosure,  because  the  discovery  of  a\nvulnerability  could  suggest  a  loss  in  future  cash  flow  of  the  software  vendors.  Software\nvulnerabilities affect the cash flows of a vendor in two main ways: One, the vendor has to\nspend time and effort in providing a patch for the vulnerability, which increases the overall\ncost  of  the  software  product  and  hence  reduces  profits.  Vulnerabilities  may  also  lead  to\ncustomer  dissatisfaction  because  the  customer  has  to  spend  time  and  effort  in  installing\npatches  and  is  exposed  to  a  risk  of  security  related  attacks.  This  could  further  lead  to\ncustomers  shifting  to  competitor’s  products  and  hence  reduces  the  cash  flow  for  the\nvendors’  products.  This  has  implications  for  software  vendors  to  invest  in  improving  the\nquality of their software. While vendors would like to launch software products as soon as\npossible, our study shows that vendors need to focus testing in  areas that can potentially\ncontain  greater  number  of  security  vulnerabilities.  We  also  show  that  the  effect  of  a\nvulnerability  announcement  is  quickly  incorporated  into  the  stock  price  and  after  the\nsecond day, there is no significant impact on the stock prices. We check for the robustness\nof our results in terms of checking for outliers and checking for momentum effects in the\nstock returns. We find that our results are also robust to the effect of investor sentiments\nsince we control for the effect of various time periods in our regression.\nOur study also provides preliminary evidence that firms should integrate security into\nsoftware  quality  practices.  In  a  firm  with  limited  resources,  this  would  mean  focusing\ntesting  efforts  in  areas  that  have  a  greater  number  of  security  vulnerabilities.  Although\nresearchers  in  computer  science  have  stressed  on  this  fact  (McGraw  2004),  there  hardly\nexists  any  literature  in  software  engineering  economics  which  measures  the  return  on\ninvestment  of  incorporating  security  based  metrics  in  software  quality  or  software  risk\nassessment.  While  software  quality  traditionally  deals  with  functional  testing,  complete\nsecurity  testing  would  incorporate  non-functional  testing  we  well,  i.e.  subjecting  the\nsoftware to misspecified input streams (Potter and McGraw 2004).\nOur  results  also  show  that  vulnerabilities  which  are  severe  or  which  can  cause  a\nconfidentiality  related  attack  are  likely  to  result  in  a  greater  loss  in  market  value  for  a\nvendor. This has implications for the software vendor in terms of allocating resources for\nsoftware  testing  and  quality  improvements.  For  example,  vendors  should  focus  on\nspending more resources in testing those software modules where a flaw is more likely to\ncause  a  confidentiality  related  breach.  Campbell  et  al  (2003)  show  that  confidentiality\nrelated breaches cause a greater loss to firms; we complement that result by showing that\nsoftware vulnerabilities which cause a confidentiality related breach are likely to result in a\ngreater loss in market value for the vendor.\nComparison with prior event studies:\nIt is interesting to compare how the abnormal returns in our event study compare with\nresults in prior event studies. Specifically, we compare our results with event studies in the\nfollowing  categories:  security  breach  related  announcements,  IT  investment  related\nannouncements and product defect related announcements.\nproduct  recall  announcements.  It  is  especially  interesting  to  note  that  the  loss  in  market\nvalue that vendors suffer due to a security vulnerability is much less than that suffered by\nfirms  during  a  security  breach.  A  possible  reason  could  be  that  software  vendors  are\nprotected  by  click-wrap  agreements  and  have  only  limited  liability  for  any  flaw  in  their\nproducts.  Another  reason  is  that  firms  usually  supply  a  patch  with  the  vulnerability\ndisclosure (almost 76% of observations in our sample have a patch available at the time of\ndisclosure).  Therefore  security  breaches  are  not  so  much  caused  due  to  unprotected\nvulnerabilities as due to lack of adequate patching done by firms. E.g. the SQL Slammer\nvirus, which affected millions of servers worldwide, was created when hackers exploited a\nsix month old vulnerability in SQL. Microsoft had already released a patch for the same,\nbut,  as  demonstrated  by  the  Slammer,  many  firms  had  not  adequately  protected  their\nservers by applying the patch.\nClassification of Event\nStudy\nImpact of Vulnerability\nDisclosures on\nSoftware Vendors\nImpact of Security\nBreaches on Firms\nAuthors\nTelang R and S Wattal (2004)\nTime\nPeriod\n1999-2004\n-0.63%\nCampbell K, Gordon LA, Loeb MP\nand L Zhou (2003)\nCavusoglu H, Mishra B and S\nRaghunathan (2004)\n-2.1%\nImpact of Product\nRecall Announcements\nImpact of IT\nInvestment\nAnnouncements\nHovav A and J D’Arcy (2003)\n1998-2002\nKannan K, Rees J and S Sridhar\n(2004)\nJarrell G and S Peltzman (1985)\n1987-1998\nDavidson WL III and DL Worrell\n(1992)\nChatterjee D, Richardson VJ and\nRW Zmud (2001)\nIm KS, Dow KE and V Grover\n(2001)\nSubramani M and E Walden (2001)  Oct 1998-\nDec 1998\n1981-1988\nDos Santos BL, Peffers K and DC\nMauer (1993)\n1981-1996\n-0.81%\n(for auto)\n-0.36%\n(day -1)\n1.16%\n1%\nImpact of Winning a\nQuality Award\nHendricks KB and Singhal VR\n(1996)\nImplications for Software Quality and Disclosure Policy\n0.59%\nAs we noted in introduction, one major argument given by the full disclosure group\nis that it will eventually force the vendors to improve the quality of their product. From our\nanalysis, there seems to be some support for this argument. Disclosure, in general (with or\nwithout a patch) adversely affects the market valuation of the vendors. It is more severe in\ncase  of  without  patch  (which  is  what  generally  happens  during  full  disclosure).  Thus,\ndisclosure clearly creates some incentives for vendors to produce better quality software.\nHowever,  market  value  is  only  one  metric  to  capture  the  impact  of  disclosure.  A  more\ninteresting  and  comprehensive  work  would  be  to  measure  the  impact  of  disclosure  on\nprofit or market share of these firms. But our paper does provide a starting point for why\nwe  should  analyze  this  issue  in  more  detail.    Another  potential  area  of  future  research\nwould  be  to  capture  and  test  the  link  between  security  based  risks  and  the  quality  of\nsoftware systems.\nFrom  our  discussion,  it  is  also  clear  why  vendors  are  pushing  for  a  limited\ndisclosure policy. Recently, Organization for Internet Safety (OIS), which is consortia of\n11  large  software  vendors,  announced  a  limited-disclosure  policy  which  requires  the\ndiscoverer to notify the vendors and give them some time before making the information\npublic. From our analysis, such a policy benefits vendors because limited disclosure gives\nthem time to release a patch for the vulnerability and availability of patch mitigates some\nadverse  effects  of  disclosure.    We  also  find  that  whether  the  vulnerability  is  reported  by\nvendors themselves, or other parties, it has essentially similar impact on market value. To\nan  extent,  our  study  points  that  vendors  are  not  necessarily  better  off  disclosing\ninformation themselves. Generally, an argument could be made that vendors should release\nthe  information  themselves,  for  if  not,  someone  else  will  and  it  will  lead  to  worse\nconsequences. However, we do not find any evidence of this. Vendors may be better off\nkeeping  quiet  and  integrate  their  fixes  as  either  service  packs  (which  do  not  give  micro-\ndetails on what it fixes) or newer versions and announce the patch only if someone else has\ndisclosed it. However, more research is required; especially more data on announcements\nwhen hackers exploit the vulnerabilities.\nAnother issue raised in discussing software flaws is whether software vendors should be\nheld responsible by law for vulnerabilities discovered in their products. Currently, the use\nof  End  User  License  Agreements  (EULA  or  Click-Wrap  Agreements)  that  come  with\nsoftware  products  limit  the  liability  that  vendors  face  if  customers  suffer  a  loss  due  to  a\nflaw in software. Our results show that liability laws (which anyway are not likely to pass\nsometimes soon) are not the only way to ‘punish’ software vendors for flaws discovered in\ntheir products. Market seems to act on this information and punishes a vendor, who on an\naverage, loses around 0.63% of its market value on the day a vulnerability is reported in its\nproducts.  Software  liability  could  certainly  cause  the  market  value  of  the  vendors  to\ndecline further if a vulnerability is reported in their product.\nSome  vulnerabilities  are  posted  on  a  public  listing  such  as  Bugtraq  before  these  are\nannounced in popular press or CERT. In that case, the actual vulnerability announcement\nmay  have  little  surprise  value.  Therefore  our  results  are  a  lower  bound  for  the  actual\ndecrease  in  stock  prices  experienced  by  the  software  vendor  if  a  flaw  is  reported  in  its\nproduct. We do not include the vulnerabilities reported on Bugtraq since most of these are\nnot confirmed vulnerabilities at the time they are posted online.\nA  limitation  of  our  study  is  that  most  of  the  data  points  in  our  sample  are\nannouncements  regarding  off-the-shelf  software  products.  Our  analysis  does  not  cover\nsoftware development projects where a security flaw can cause millions of dollars worth of\ndamage.  The  main  reason  for  excluding  them  was  the  lack  of  availability  of  data  on\nsoftware failures in such cases.  We also reiterate that further analysis in terms of software\nquality,  market  share  or  profitability  is  needed  to  fully  understand  how  vulnerability\ndisclosure  signals  poorer  quality  and  how  it  affects  the  vendors’  incentives  to  provide\nbetter quality software. Our paper takes the first step in this direction.\n•  Adams  EN  III  (1980)  ‘Minimizing  Cost  Impacts  of  Software  Defects’  IBM  Research\nReport, RC 8228 April\n•  Applewhite A (2004) ‘Whose Bug Is It Anyway? The Battle over Handling Software\nFlaws’ IEEE Software, March/April 2004, 94-97\n•  Anderson  R  (2001)  ‘Why  Information  Security  is  Hard  –  an  Economic  Perspective’\nProceeding of 17th Annual Computer Security Applications Conference, New Orleans,\nLouisiana\n•  Arora,  A.,  Caulkins,  J.P.  and  R  Telang,  (2004).  ‘Sell  First,  Fix  Later:  Impact  of\nPatching on Software Quality’, Carnegie Mellon University, working paper.\n•  Arora, A., Telang, R. and  H Xu, (2004). ‘Optimal  Policy  for  Software  Vulnerability\nDisclosure’, Carnegie Mellon University, working paper.\n•  Banker RD and SA Slaughter (1997) ‘A Field Study of Scale Economies in Software\nMaintenance’, Management Science, 43(12), 1709-1725\n•  Barki  H,  Rivard  S  and  J  Talbot  ‘Toward  an  Assessment  of  Software  Development\nRisk’, Journal of Management Information Systems, 10(2),  203-225\n•  Basili, VR and JD Musa (1991) ‘The Future Engineering of Software: A Management\nPerspective’ IEEE Computing, 20(4), 90-96\n•  Bharadwaj, A and M Keil (2003) ‘The Effects of Information Technology Failures on\nthe Market Value of Firms’, Working paper.\n•  Brown SJ and JB Warner (1980) ‘Measuring Security Price Performance’, Journal of\nFinancial Economics, 14(1985), 3-31\n•  Brown  SJ  and  JB  Warner  (1985)  ‘Using  Daily  Stock  Returns:  The  Case  of  Event\nStudies’, Journal of Financial Economics, 14(1985), 3-31\n•  Campbell JY, Andrew WL and AC MacKinlay (1997) ‘The Econometrics of Financial\nMarkets’ Princeton University Press\n•  Campbell  K,  LA  Gordon  LA,  Loeb  MP  and  L  Zhou  (2003)  ‘The  Economic  Cost  of\nPublicly  Announced  Information  Security  Breaches:  Empirical  Evidence  from  the\nStock Market’, Journal of Computer Security, 11(3), 431-448\n•  Carr  R,  Tynan  M  and  R  Davis  (1991)  ‘Quality  and  Security:  They  Work  Together’\nIEEE AES Systems Magazine, 6(9), 15-19\n•  Cavusoglu  H,  Mishra  B  and  S  Raghunathan  (2004)  ‘The  Effect  of  Internet  Security\nBreach  Announcements  on  Market  Value:  Capital  Market  Reactions  for  Breached\nFirms  and  Internet  Security  Developers’  International  Journal  of  Electronic\nCommerce, 9(1), 69\n•  Chaney  PK,  Devinney  TM  and  RS  Winer  (1991)  ‘The  Impact  of  New  Product\nIntroductions on the Market Value of Firms’, The Journal of Business, 64(4), 573-610\n•  Chatterjee  D,  Richardson  VJ  and  RW  Zmud  (2001)  ‘Examining  the  Shareholder\nWealth  Effects  of  Announcements  of  Newly  Created  CIO  Positions’,  MIS  Quarterly,\n25(1), 43-70\n•  Cooper MJ, O Dimitrov and PR Rau (2001) ‘A Rose.com by Any Other Name’, The\nJournal of Finance, (6), 2371-2387\n•  Cusumano,  MA  (2004)  ‘Who  is  Liable  for  Bugs  and  Security  Flaws  in  Software?’\nCommunications of the ACM, 47(3), 25-27\n•  Davidson WL III, DL Worrell(1992) ‘The Effect of Product Recall Announcements on\nShareholder Wealth’ Strategic Management Journal, 13(6), 467-473\n•  Devanbu P and S Stubblebine (2000) ‘Software Engineering for Security : a Roadmap’\nFuture of Software Engineering, ACM 225-239\n•  Dolley J (1933) ‘Characteristics and Procedure  of Common Stock Split-Ups,’ Harvard\n•  Dos  Santos  BL,  Peffers  K  and  D  Mauer  (1993)  ‘The  Impact  of  Information\nTechnology  on  the  Market  Value  of  the  Firm’,  Information  Systems  Research,  4\n(March), 1-23\n•  Gal-Or,  E.  &  A  Ghose  (2003).  ‘The  Economic  Consequences  of  Sharing  Security\nInformation’, In 2nd Workshop on Economics and Information Security, May 29-30.\n•  Gordon LA & MP Loeb(2002) ‘The Economics of Information Security Investments’\nACM Transactions on Information and Systems Security,5(4),438-457\n•  Gordon,  L.A.,  Loeb,  M.P.  &  Lucyshyn,  W. (2002) ‘An Economic Perspective on the\nSharing  of  Information  Related  to  Security  Breaches:  Concepts  and  Empirical\nEvidence’, In The 1stWorkshop on Economics and Information Security, May16-17.\n•  Gordon LA and MP Loeb and T Sohail (2003) ‘A Framework for Using Insurance for\nCyber Risk Management’ Communications of the ACM, 46(3), 81-85\n•  Harter  DE,  Krishnan  MS  and  Slaughter  SA  (2000)  ‘Effects  of  Process  Maturity  on\nQuality,  Cycle  Time,  and  Effort  in  Software  Product  Development’  Management\nScience, 46(4), 451-466\n•  Hendricks  KB  and  Singhal  VR  (1996)  ‘Quality  Awards  and  the  Market  Value  of  the\nFirm: An Empirical Investigation’ Management Science, 42(2), 415-436\n•  Hendricks KB and Singhal VR (1997) ‘Delays in New Product Introductions and the\nMarket  Value  of  the  Firm:  The  Consequences  of  Being  Late  to  the  Market’,\nManagement Science, 43(4), 422-436\n•  Hovav  A  and  J  D’Arcy  (2003)  ‘The  Impact  of  Denial-of-Service  Attack\nAnnouncements  of  the  Market  Value  of  Firms’,  Risk  Management  and  Insurance\nReview, 6(2), 97-121\n•  Hsiao, C (2002) ‘Analysis of Panel Data’ Cambridge University Press\n•\nIm  KS,  Dow  KE  and  V  Grover  (2001)  ‘Research  Report:  A  Reexamination  of  IT\nInvestment  and  the  Market  Value  of  the  Firm  –  An  Event  Study  Methodology’,\nInformation Systems Research, 12(1), 103-117\n•  Jarrell  G  and  S  Peltzman  (1985)  ‘The  Impact  of  Product  Recalls  on  the  Wealth  of\nSellers’ The Journal of Political Economy, 93(1), 512-536\n•  Kannan  K  and  R  Telang  (2004)  ‘Market  for  Software  Vulnerabilities?  Think  Again’\n•  Kannan  K,  Rees  J  and  S  Sridhar  (2004)  ‘Reexamining  the  Impact  of  Information\nSecurity Breach Announcements on Firm Performance’ Working Paper\n•  Lane  V  and  R  Jacobson  (1995)  ‘Stock  Market  Reactions  to  Brand  Extension\nAnnouncements: The Effects of Brand Attitude and Familiarity’, Journal of Marketing,\n59 (January), 63-77\n•  McGraw G (2004) ‘Software Security’ IEEE Security and Privacy, 2(2), 80-83\n•  Malatesta, P and R Thompson (1985) ‘Partially Anticipated Events: A Model of Stock\nPrice Reactions with an Application to Corporate Acquisitions’, Journal of Financial\nEconomics, 14(2), 237-250\n•  Paulk M Weber C, Curtis W and Chrissis M (1994) ‘The Capability Maturity Model:\nGuidelines for Improving the Software Process’ Carnegie Mellon University : Software\nEngineering Institute\n•  Pfleegar CP (1997) ‘The Fundamentals of Information Security’ IEEE Software, 14(1),\n•  Potter  B  and  G  McGraw  (2004)  ‘Software  Security  Testing’  IEEE  Security  and\n•  Slaughter SA, DE Harter and MS Krishnan (1998) ‘Evaluating the Cost of Software\nQuality’ Communications of the ACM, 41(8), 67-73\n•  Wallace L, Keil M and A Rai (2004) ‘How Software Project Risk Affects Project\nPerformance: An Investigation of the Dimensions of Risk and An Exploratory Model’,\nDecision Scineces, 35(2), 289-321\n•  Wang H and C Wang (2003) ‘Taxonomy of Security Considerations and Software\nQuality’ Communications of the ACM, 46(6), 75-78\n•  Westland, JC (2003) ‘The Cost Behavior of Software Defects’, Decision Sciences, 37,\n229-238",
         "Rahul Telang,S. Wattal",
         null
        ],
        [
         "35",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "Plant material. Three different cultivars of pepper plants were\nused:  susceptible  cv. Yolo  Wonder,  and  SCM331  and  PI201234,\neach  with  different  levels  of  resistance  to  P.  capsici,  SCM331\nbeing the most resistant (4). Moreover, a susceptible local cultivar\n(Padrón)  was  included  because  of  its  economic  value  in  north-\nwestern Spain. Plants were transferred into pots and grown sepa-\nrately in a chamber at 25°C and a photoperiod of 16 h of light and\n8 h of darkness until they were 3 weeks old.",
         "Pathogen  inoculation,  quantification  bioassay,  and  disease\nassessment.  A  suspension  of  zoospores  from  UDC196Pc  and\nUDC248Pc were obtained according to Larkin et al. (30). Pepper\nplants were inoculated with a concentration of 104 zoospores ml–1\nby  adding  5  ml  of  the  inoculum  into  the  soil.  For  the  quantifi-\ncation of pathogen DNA, samples of leaves, stems, and roots were\ncollected  at  8,  24,  48,  and  72  h  postinoculation  in  the  different\ncultivars. Three independent experiments were carried out.\nIn parallel, assays were carried out with the aim of studying the\ndisease  progression  in  different  cultivars  during  a  time  period.\nWith this purpose, 10 plants of each cultivar were inoculated with\nthe  two  different  isolates  (UDC196Pc  and  UDC248Pc).  Disease\nseverity was rated for 20 days after inoculation on a scale of 0 to\n5, where 0 = no visible symptoms; 1 = ≤50% of stem with water-\nsoaked lesions, ≤50% of leaves wilted, or both; 2 = >50% of stem\nwith  water-soaked  lesions  or  >50%  of  leaves  wilted;  3  =  both\n≤50%  of  stem  with  water-soaked  lesions  and  >50%  of  leaves\nwilted or both >50% of stem with water-soaked lesions and ≤50%\nof leaves wilted; 4 = >50% of stem with water-soaked lesions and\n>50% of leaves wilted; and 5 = dead plant. Percentage of diseased\nplants (incidence) also was recorded during the time course of in-\nfection. In this case, two independent experiments were performed.\nIsolation of P. capsici from artificially infected plants. Colo-\nnization of pepper plants by P. capsici was determined in each of\nthe  pepper  cultivars  studied  by  isolating  the  pathogen  on  PDA.\nDuring  the  host–plant  infection  assay,  samples  of  plants  were\ncollected at 8, 24, 48, and 72 h postinoculation. Entire plants were\nsurface  disinfected  with  NaClO  (10%)  for  5  min,  washed  with\ndistilled water,  dried,  and plated onto PDA  medium. Plates were\nincubated at 24°C in the dark for 7 days.\non the fourth day after inoculation, although the disease progress\nover time was higher in plants inoculated with UDC196Pc than in\nthose infected with UDC248Pc. In the first case, a maximal infec-\ntion (severity index = 4.8) was reached at 17 days after infection\n(Fig. 1A) and the percentage  of infected plants at the end of the\nbioassay  was  95%  (data  not  shown).  In  the  Yolo  Wonder–\nUDC248Pc interaction, however, disease progression was slower\nand the severity rating achieved at the end of the ongoing period\nwas only 1.5 (Fig. 1B), with  only 30%  diseased  plants (data not\nshown). After  inoculation  with  UDC196Pc,  the  infection  pattern\ndisplayed by local cv. Padrón was similar to that in Yolo Wonder,\nbut showed the highest degree of susceptibility (severity rate = 5\nand  percentage  of  infected  plants  =  100%)  (Fig.  1A).  However,\nafter  inoculation  with  UDC248Pc,  the  severity  achieved  by Yolo\nWonder was higher than in Padrón (Fig. 1B).\nIn  contrast,  resistant  genotypes  showed  the  lowest  disease\nseverity.  No  disease  symptoms  were  observed  when  the  plants\nwere infected with UDC248Pc, and slower progress curves were\nobtained in the PI201234– or SCM331–UDC196Pc system com-\npared  with  those  in  susceptible  cultivars.  In  the  last  case,  more\nsymptoms developed in PI201234 than in SCM331, with 40 and\n20%  disease  percentage  (data  not  shown)  and  2  and  1  severity\nindex, respectively, at the  end  of the trials.  Noninoculated plants\ndid not show any symptoms (data not shown).\nP. capsici was isolated successfully from leaves, stems, and roots\nin  all  pepper  genotypes  inoculated  with  both  isolates  at  24,  48,\nand  72  h  after  inoculation.  However,  no  pathogen  isolation  was\npossible at 8 h in any pepper–P. capsici system (data not shown).\nReal-time PCR adjustment and DNA standard curves. Pre-\nliminary runs of real-time PCR without the step at 84°C for 15 s\nwere  carried  out  with  the  aim  of  detecting  some  possible  non-\nand  increased  sharply  over  time  whereas,  in  the  resistant  plants,\nthe variations in pathogen development were slighter. Significant\n(P ≤ 0.05) differences were found among resistant and susceptible\ncultivars  as  early  as  24  h  postinoculation  in  the  stem  for  the\nUDC196Pc  isolate  (Fig.  4A)  although,  at  48  h,  there  was  no\nsignificant  differences.  The  differences  were  significant  at  72  h\npostinoculation in all the organs and isolates, except for the com-\nbination of the roots and the isolate UDC196Pc (Fig. 5A). Com-\nparing  the  cultivars  separately,  susceptible  cvs.  Padrón  and Yolo\nWonder  displayed  a  similar  pattern  of  pathogen  invasion,  but\nsignificant differences between them were found in the roots at 48\nand 72 h postinoculation (Fig. 5A and B). For the resistant culti-\nvars,  significant  differences  were  found  in  SCM331  compared\nwith the partially resistant cv. PI201234 only in the case of roots\nat 48 h postinoculation (Fig. 5A and B). In any case, the accumu-\nlation  of  pathogen  DNA  in  these  resistant  genotypes  was  slower\nthan  that  in  susceptible  cultivars  and  the  amount  of  pathogen\nDNA generally remained lower during the study.\nWith regard to the two P. capsici isolates, a different pattern of\ninfection  was  observed.  Up  to  5.9  ng  g–1  FW  of  pathogen  DNA\nwas  observed  as  soon  as  8  h  postinoculation  in  the  Padrón–  or\nYolo  Wonder–UDC196Pc  interactions  (Fig.  4A)  whereas,  in\nsusceptible cultivar–UDC248Pc interactions, the maximal amount\nof DNA detectable was 0.2 ng g–1 FW (Fig. 4B). In the same way,\nthe  average  pathogen  DNA  in  the  resistant  cultivar–UDC196Pc\nsystem  was  1.4  ng  g–1  FW,  whereas  it  could  not  be  detected  in\nPI201234 or in SCM331 until 24 h after infection, when the plants\nwere  inoculated  with  the  least  virulent  pathogen  (UDC248Pc).\nDuring the remaining period, the amount of UDC196Pc (the iso-\nlate previously characterized as more virulent) DNA was up to 10\nfound  in  other  pathogen–plant  interactions  (15–17)  as  well  as  in\nother  Phytophthora  spp.,  although  using  different  real-time\nchemistries  (26).  The  Ct  values  depend  on  both  the  amount  of\nstarting  DNA  and  the  genomic  organization  of  the  target  se-\nquences  recognized  by  the  sequence-specific  primer  pair;  there-\nfore,  the  efficiency  of  the  technique  will  improve  if  the  target\nDNA is present in many copies in the genome, as is the case with\nrDNA.  This  sensitivity  was  similar  to  that  achieved  in  previous\nworks using conventional PCR but less than that obtained with a\nnested PCR assay (50). These results agree with those published\nrecently  by  Bonants  et  al.  (6)  working  on  P.  fragariae.  They\nobtained  equivalent  detection  limits  by  using  either  conventional\nPCR or a TaqMan assay but higher sensitivity with a nested PCR.\nIn any case, with this sensitivity, pathogen abundance can be re-\nliably  quantified  from  the  very  beginning  of  the  infection  time\ncourse and even in tissues exhibiting a low degree of infection.\nIn  a  parallel  assay,  the  influence  of  plant  samples  on  quanti-\nfication was tested (data not shown). Serial dilutions (10-fold) of\npathogen DNA were mixed with an equal amount of plant DNA.\nThis  mixture  possibly  could  reflect  an  infection  from  a  range  of\n1:1 (a very heavy infection) to a range of 1:106 (an early stage of\ninfection). The presence of a host sample of DNA did not produce\nany interference in the quantification. Furthermore, with this tech-\nnique, the sensitivity of the method was not affected by the pres-\nence  of  a  plant  DNA  sample,  as  occurs  with  conventional  and\nnested PCR (50).\nIn a real-time PCR assay, the accumulation of the amplicon can\nbe  detected  using  either  specific  detection  methods  such  as\nTaqMan  probes  (25,54,57)  or  nonspecific  methods  like  SYBR\nGreen (36). The advantage of SYBR Green is that, in contrast to\nfluorogenic probes, it does not require the design of specific com-\nplementary fragments to the target DNA and, therefore, the reac-\ntion cost is limited compared with real-time PCRs that depend on\nhybridization of labeled  sequence-specific probes. The  main dis-\nadvantage  of  this  method  is  that  it  binds  double-stranded  DNA\nand, therefore, it also can associate with primer-dimers and non-\nspecific products, which can seriously disturb the interpretation of\nthe results. However, this major drawback can be overcome with\nthe  use  of  melting  curve  temperature  analysis  at  the  end  of  the\nPCR assay (46). At the end of the cycling reactions, the products\nare melted by slowly increasing the temperature while continually\nmonitoring  the  fluorescence. After  melting  curve  analysis  in  our\nassay, the use of unspecific SYBR Green dye for quantifying the\ntarget amplicon (≈452 bp, Tm = 86.5°C) required a 84°C fluores-\ncence emission reading step in the amplification program to elimi-\nnate  any  interference  caused  by  the  accumulation  of  unspecific\nPCR products. At that temperature, the nonspecific products will\nbe  denatured  and  only  the  fluorogenic  emission  from  the  target\namplicon would result.\nof  plant  symptoms,  which  is  time  consuming  and  sometimes  in-\nconclusive.\nIn terms of pathogen ability to infect the plant, it seemed clear\nthat  the  more  virulent  isolate  had  a  better  capacity  not  only  to\ninfect the plant sooner but also to spread quicker during the time\ncourse  of  infection.  In  general,  the  average  amount  of  pathogen\nDNA  was  higher  for  UDC196Pc-infected  plants  than  for  the\nUDC248Pc-infected  ones.  Moreover,  the  accumulation  of  patho-\ngen DNA was rapid in susceptible genotype–most virulent isolate\ninteractions whereas, in less compatible interactions, it was more\nprogressive  and  notable  increases  in  pathogen  amount  did  not\ntake  place,  the  quantity  of  pathogen  DNA  being  more  constant\nover  time.  Considering  all  these  results,  the  difference  in  viru-\nlence  of  the  two  P.  capsici  isolates  could  be  attributed  to  differ-\nences in pathogen amount in infected plants, among other factors.\nHowever, a clear correlation between the virulence of a pathogen\nand the amount of pathogen present in the plant does not always\nexist.  Sometimes,  more  virulent  isolates  do  not  involve  an  in-\ncrease in pathogen colonization (34,53).\nAnother interesting aspect was the different amounts of patho-\ngen  DNA  in  the  different  tissues  collected.  Comparatively,  the\namount  of  pathogen  DNA  in  leaves  was  much  lower  than  that\nquantified in roots which, in turn, was less than that occurring in\nstems. This tissue difference suggests either a temporal sequence\nof  systemic  colonization  by  the  pathogen  or  a  differential  re-\nsponse of the plant defense mechanisms. Differences in the level\nof  colonization  of  different  tissues  by  the  pathogen  have  been\nreported in other plant–pathogen interactions (34,59). It has been\nreported  that  some  plant  defense-related  genes,  which  encode\npathogenesis-related proteins, display a differential expression in\nvarious  pepper  organs  in  response  to  pathogen  infection  (27,28,\n49).  Such  responses  could  be  responsible  for  that  differential\ndistribution, preventing pathogen advance, especially in the culti-\nvars that are more resistant.\nA final conclusion of the present work was that the local pepper\ncv.  Padrón  displayed  an  infection  and  quantification  pattern\nsimilar  to  that  in  Yolo  Wonder,  suggesting  that  it  may  have  a\nsimilar  degree  of  susceptibility.  Padrón  is  a  pepper  cultivar  of\nhigh  economic  importance  in  Galicia  (northwestern  Spain)  (14).\nThe results in this  study  have  demonstrated its level of suscepti-\nbility  to  Phytophthora  root  rot,  the  most  important  disease  of\npepper  crops  in  this  region  (44).  The  real-time  PCR  developed\nhere, together with disease severity studies, will enable the selec-\ntion  of  the  most  resistant  plants  within  the  Padrón  cultivar  for\ndisease resistance breeding in the future.\n2.  Bates, J. A., Taylor, E. J. A., Kenyon, D. M., and Thomas, J. E. 2001. The\napplication of real-time PCR to the identification, detection and quantifi-\ncation of Pyrenophora species in barley seed. Mol. Plant Pathol. 2:49-57.\n3.  Bermingham, S., Maltby, L., and Cooke, R. C. 1995. A critical assessment\nof  the  validity  of  ergosterol  as  an  indicator  of  fungal  biomass.  Mycol.\nRes. 99:479-484.\n33.  McCartney, H. A., Foster, S. J., Fraaije, B. A., and Ward, E. 2003. Molecular\ndiagnostics for fungal plant pathogens. Pest. Manage. Sci. 59:129-142.\n34.  Mercado-Blanco, J., Collado-Romero, M., Parrilla-Araujo, S., Rodríguez-\nJurado,  D.,  and  Jiménez-Díaz,  R.  M.  2003.  Quantitative  monitoring  of\ncolonization  of  olive  genotypes  by  Verticillium  dahliae  pathotypes  with\nreal-time  polymerase  chain  reaction.  Physiol.  Mol.  Plant  Pathol.  63:91-\n105.",
         "",
         "",
         "",
         "Plant material. Three different cultivars of pepper plants were\nused:  susceptible  cv. Yolo  Wonder,  and  SCM331  and  PI201234,\neach  with  different  levels  of  resistance  to  P.  capsici,  SCM331\nbeing the most resistant (4). Moreover, a susceptible local cultivar\n(Padrón)  was  included  because  of  its  economic  value  in  north-\nwestern Spain. Plants were transferred into pots and grown sepa-\nrately in a chamber at 25°C and a photoperiod of 16 h of light and\n8 h of darkness until they were 3 weeks old.\nPathogen  inoculation,  quantification  bioassay,  and  disease\nassessment.  A  suspension  of  zoospores  from  UDC196Pc  and\nUDC248Pc were obtained according to Larkin et al. (30). Pepper\nplants were inoculated with a concentration of 104 zoospores ml–1\nby  adding  5  ml  of  the  inoculum  into  the  soil.  For  the  quantifi-\ncation of pathogen DNA, samples of leaves, stems, and roots were\ncollected  at  8,  24,  48,  and  72  h  postinoculation  in  the  different\ncultivars. Three independent experiments were carried out.\nIn parallel, assays were carried out with the aim of studying the\ndisease  progression  in  different  cultivars  during  a  time  period.\nWith this purpose, 10 plants of each cultivar were inoculated with\nthe  two  different  isolates  (UDC196Pc  and  UDC248Pc).  Disease\nseverity was rated for 20 days after inoculation on a scale of 0 to\n5, where 0 = no visible symptoms; 1 = ≤50% of stem with water-\nsoaked lesions, ≤50% of leaves wilted, or both; 2 = >50% of stem\nwith  water-soaked  lesions  or  >50%  of  leaves  wilted;  3  =  both\n≤50%  of  stem  with  water-soaked  lesions  and  >50%  of  leaves\nwilted or both >50% of stem with water-soaked lesions and ≤50%\nof leaves wilted; 4 = >50% of stem with water-soaked lesions and\n>50% of leaves wilted; and 5 = dead plant. Percentage of diseased\nplants (incidence) also was recorded during the time course of in-\nfection. In this case, two independent experiments were performed.\nIsolation of P. capsici from artificially infected plants. Colo-\nnization of pepper plants by P. capsici was determined in each of\nthe  pepper  cultivars  studied  by  isolating  the  pathogen  on  PDA.\nDuring  the  host–plant  infection  assay,  samples  of  plants  were\ncollected at 8, 24, 48, and 72 h postinoculation. Entire plants were\nsurface  disinfected  with  NaClO  (10%)  for  5  min,  washed  with\ndistilled water,  dried,  and plated onto PDA  medium. Plates were\nincubated at 24°C in the dark for 7 days.\non the fourth day after inoculation, although the disease progress\nover time was higher in plants inoculated with UDC196Pc than in\nthose infected with UDC248Pc. In the first case, a maximal infec-\ntion (severity index = 4.8) was reached at 17 days after infection\n(Fig. 1A) and the percentage  of infected plants at the end of the\nbioassay  was  95%  (data  not  shown).  In  the  Yolo  Wonder–\nUDC248Pc interaction, however, disease progression was slower\nand the severity rating achieved at the end of the ongoing period\nwas only 1.5 (Fig. 1B), with  only 30%  diseased  plants (data not\nshown). After  inoculation  with  UDC196Pc,  the  infection  pattern\ndisplayed by local cv. Padrón was similar to that in Yolo Wonder,\nbut showed the highest degree of susceptibility (severity rate = 5\nand  percentage  of  infected  plants  =  100%)  (Fig.  1A).  However,\nafter  inoculation  with  UDC248Pc,  the  severity  achieved  by Yolo\nWonder was higher than in Padrón (Fig. 1B).\nIn  contrast,  resistant  genotypes  showed  the  lowest  disease\nseverity.  No  disease  symptoms  were  observed  when  the  plants\nwere infected with UDC248Pc, and slower progress curves were\nobtained in the PI201234– or SCM331–UDC196Pc system com-\npared  with  those  in  susceptible  cultivars.  In  the  last  case,  more\nsymptoms developed in PI201234 than in SCM331, with 40 and\n20%  disease  percentage  (data  not  shown)  and  2  and  1  severity\nindex, respectively, at the  end  of the trials.  Noninoculated plants\ndid not show any symptoms (data not shown).\nP. capsici was isolated successfully from leaves, stems, and roots\nin  all  pepper  genotypes  inoculated  with  both  isolates  at  24,  48,\nand  72  h  after  inoculation.  However,  no  pathogen  isolation  was\npossible at 8 h in any pepper–P. capsici system (data not shown).\nReal-time PCR adjustment and DNA standard curves. Pre-\nliminary runs of real-time PCR without the step at 84°C for 15 s\nwere  carried  out  with  the  aim  of  detecting  some  possible  non-\nand  increased  sharply  over  time  whereas,  in  the  resistant  plants,\nthe variations in pathogen development were slighter. Significant\n(P ≤ 0.05) differences were found among resistant and susceptible\ncultivars  as  early  as  24  h  postinoculation  in  the  stem  for  the\nUDC196Pc  isolate  (Fig.  4A)  although,  at  48  h,  there  was  no\nsignificant  differences.  The  differences  were  significant  at  72  h\npostinoculation in all the organs and isolates, except for the com-\nbination of the roots and the isolate UDC196Pc (Fig. 5A). Com-\nparing  the  cultivars  separately,  susceptible  cvs.  Padrón  and Yolo\nWonder  displayed  a  similar  pattern  of  pathogen  invasion,  but\nsignificant differences between them were found in the roots at 48\nand 72 h postinoculation (Fig. 5A and B). For the resistant culti-\nvars,  significant  differences  were  found  in  SCM331  compared\nwith the partially resistant cv. PI201234 only in the case of roots\nat 48 h postinoculation (Fig. 5A and B). In any case, the accumu-\nlation  of  pathogen  DNA  in  these  resistant  genotypes  was  slower\nthan  that  in  susceptible  cultivars  and  the  amount  of  pathogen\nDNA generally remained lower during the study.\nWith regard to the two P. capsici isolates, a different pattern of\ninfection  was  observed.  Up  to  5.9  ng  g–1  FW  of  pathogen  DNA\nwas  observed  as  soon  as  8  h  postinoculation  in  the  Padrón–  or\nYolo  Wonder–UDC196Pc  interactions  (Fig.  4A)  whereas,  in\nsusceptible cultivar–UDC248Pc interactions, the maximal amount\nof DNA detectable was 0.2 ng g–1 FW (Fig. 4B). In the same way,\nthe  average  pathogen  DNA  in  the  resistant  cultivar–UDC196Pc\nsystem  was  1.4  ng  g–1  FW,  whereas  it  could  not  be  detected  in\nPI201234 or in SCM331 until 24 h after infection, when the plants\nwere  inoculated  with  the  least  virulent  pathogen  (UDC248Pc).\nDuring the remaining period, the amount of UDC196Pc (the iso-\nlate previously characterized as more virulent) DNA was up to 10\nfound  in  other  pathogen–plant  interactions  (15–17)  as  well  as  in\nother  Phytophthora  spp.,  although  using  different  real-time\nchemistries  (26).  The  Ct  values  depend  on  both  the  amount  of\nstarting  DNA  and  the  genomic  organization  of  the  target  se-\nquences  recognized  by  the  sequence-specific  primer  pair;  there-\nfore,  the  efficiency  of  the  technique  will  improve  if  the  target\nDNA is present in many copies in the genome, as is the case with\nrDNA.  This  sensitivity  was  similar  to  that  achieved  in  previous\nworks using conventional PCR but less than that obtained with a\nnested PCR assay (50). These results agree with those published\nrecently  by  Bonants  et  al.  (6)  working  on  P.  fragariae.  They\nobtained  equivalent  detection  limits  by  using  either  conventional\nPCR or a TaqMan assay but higher sensitivity with a nested PCR.\nIn any case, with this sensitivity, pathogen abundance can be re-\nliably  quantified  from  the  very  beginning  of  the  infection  time\ncourse and even in tissues exhibiting a low degree of infection.\nIn  a  parallel  assay,  the  influence  of  plant  samples  on  quanti-\nfication was tested (data not shown). Serial dilutions (10-fold) of\npathogen DNA were mixed with an equal amount of plant DNA.\nThis  mixture  possibly  could  reflect  an  infection  from  a  range  of\n1:1 (a very heavy infection) to a range of 1:106 (an early stage of\ninfection). The presence of a host sample of DNA did not produce\nany interference in the quantification. Furthermore, with this tech-\nnique, the sensitivity of the method was not affected by the pres-\nence  of  a  plant  DNA  sample,  as  occurs  with  conventional  and\nnested PCR (50).\nIn a real-time PCR assay, the accumulation of the amplicon can\nbe  detected  using  either  specific  detection  methods  such  as\nTaqMan  probes  (25,54,57)  or  nonspecific  methods  like  SYBR\nGreen (36). The advantage of SYBR Green is that, in contrast to\nfluorogenic probes, it does not require the design of specific com-\nplementary fragments to the target DNA and, therefore, the reac-\ntion cost is limited compared with real-time PCRs that depend on\nhybridization of labeled  sequence-specific probes. The  main dis-\nadvantage  of  this  method  is  that  it  binds  double-stranded  DNA\nand, therefore, it also can associate with primer-dimers and non-\nspecific products, which can seriously disturb the interpretation of\nthe results. However, this major drawback can be overcome with\nthe  use  of  melting  curve  temperature  analysis  at  the  end  of  the\nPCR assay (46). At the end of the cycling reactions, the products\nare melted by slowly increasing the temperature while continually\nmonitoring  the  fluorescence. After  melting  curve  analysis  in  our\nassay, the use of unspecific SYBR Green dye for quantifying the\ntarget amplicon (≈452 bp, Tm = 86.5°C) required a 84°C fluores-\ncence emission reading step in the amplification program to elimi-\nnate  any  interference  caused  by  the  accumulation  of  unspecific\nPCR products. At that temperature, the nonspecific products will\nbe  denatured  and  only  the  fluorogenic  emission  from  the  target\namplicon would result.\nof  plant  symptoms,  which  is  time  consuming  and  sometimes  in-\nconclusive.\nIn terms of pathogen ability to infect the plant, it seemed clear\nthat  the  more  virulent  isolate  had  a  better  capacity  not  only  to\ninfect the plant sooner but also to spread quicker during the time\ncourse  of  infection.  In  general,  the  average  amount  of  pathogen\nDNA  was  higher  for  UDC196Pc-infected  plants  than  for  the\nUDC248Pc-infected  ones.  Moreover,  the  accumulation  of  patho-\ngen DNA was rapid in susceptible genotype–most virulent isolate\ninteractions whereas, in less compatible interactions, it was more\nprogressive  and  notable  increases  in  pathogen  amount  did  not\ntake  place,  the  quantity  of  pathogen  DNA  being  more  constant\nover  time.  Considering  all  these  results,  the  difference  in  viru-\nlence  of  the  two  P.  capsici  isolates  could  be  attributed  to  differ-\nences in pathogen amount in infected plants, among other factors.\nHowever, a clear correlation between the virulence of a pathogen\nand the amount of pathogen present in the plant does not always\nexist.  Sometimes,  more  virulent  isolates  do  not  involve  an  in-\ncrease in pathogen colonization (34,53).\nAnother interesting aspect was the different amounts of patho-\ngen  DNA  in  the  different  tissues  collected.  Comparatively,  the\namount  of  pathogen  DNA  in  leaves  was  much  lower  than  that\nquantified in roots which, in turn, was less than that occurring in\nstems. This tissue difference suggests either a temporal sequence\nof  systemic  colonization  by  the  pathogen  or  a  differential  re-\nsponse of the plant defense mechanisms. Differences in the level\nof  colonization  of  different  tissues  by  the  pathogen  have  been\nreported in other plant–pathogen interactions (34,59). It has been\nreported  that  some  plant  defense-related  genes,  which  encode\npathogenesis-related proteins, display a differential expression in\nvarious  pepper  organs  in  response  to  pathogen  infection  (27,28,\n49).  Such  responses  could  be  responsible  for  that  differential\ndistribution, preventing pathogen advance, especially in the culti-\nvars that are more resistant.\nA final conclusion of the present work was that the local pepper\ncv.  Padrón  displayed  an  infection  and  quantification  pattern\nsimilar  to  that  in  Yolo  Wonder,  suggesting  that  it  may  have  a\nsimilar  degree  of  susceptibility.  Padrón  is  a  pepper  cultivar  of\nhigh  economic  importance  in  Galicia  (northwestern  Spain)  (14).\nThe results in this  study  have  demonstrated its level of suscepti-\nbility  to  Phytophthora  root  rot,  the  most  important  disease  of\npepper  crops  in  this  region  (44).  The  real-time  PCR  developed\nhere, together with disease severity studies, will enable the selec-\ntion  of  the  most  resistant  plants  within  the  Padrón  cultivar  for\ndisease resistance breeding in the future.\n2.  Bates, J. A., Taylor, E. J. A., Kenyon, D. M., and Thomas, J. E. 2001. The\napplication of real-time PCR to the identification, detection and quantifi-\ncation of Pyrenophora species in barley seed. Mol. Plant Pathol. 2:49-57.\n3.  Bermingham, S., Maltby, L., and Cooke, R. C. 1995. A critical assessment\nof  the  validity  of  ergosterol  as  an  indicator  of  fungal  biomass.  Mycol.\nRes. 99:479-484.\n33.  McCartney, H. A., Foster, S. J., Fraaije, B. A., and Ward, E. 2003. Molecular\ndiagnostics for fungal plant pathogens. Pest. Manage. Sci. 59:129-142.\n34.  Mercado-Blanco, J., Collado-Romero, M., Parrilla-Araujo, S., Rodríguez-\nJurado,  D.,  and  Jiménez-Díaz,  R.  M.  2003.  Quantitative  monitoring  of\ncolonization  of  olive  genotypes  by  Verticillium  dahliae  pathotypes  with\nreal-time  polymerase  chain  reaction.  Physiol.  Mol.  Plant  Pathol.  63:91-\n105.",
         "C. Silvar,J. Diaz,F. Merino",
         null
        ],
        [
         "36",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "Growth, patterning, and apoptosis are mutually interactive during development. For example, cells that\nselect an abnormal fate in a developing ﬁeld are frequently removed by apoptosis. An important issue in this\nprocess that needs to be resolved is the mechanism used by cells to discern their correct fate from an abnormal\nfate. In order to examine this issue, we developed an animal model that expresses the dioxin receptor homolog\nSpineless (Ss) ectopically in the Drosophila wing. The presence of mosaic clones ectopically expressing ss results\nin a local transformation of organ identity, homeosis, from wing into a leg or antenna. The cells with\nmisspeciﬁed fates subsequently activate c-Jun N-terminal kinase to undergo apoptosis in an autonomous or\nnonautonomous manner depending on their position within the wing, suggesting that a cell-cell interaction is,\nat least in some cases, involved in the detection of misspeciﬁed cells. Similar position dependence is commonly\nobserved when various homeotic genes controlling the body segments are ectopically expressed. The autono-\nmous and nonautonomous apoptosis caused by ss is regulated by a novel leucine-rich repeat family trans-\nmembrane protein, Fish-lips (Fili) that interacts with surrounding normal cells. These data support a mech-\nanism in which the lack of some membrane proteins helps to recognize the presence of different cell types and\ndirect these cells to an apoptotic fate in order to exclude them from the normal developing ﬁeld.",
         "Signals that regulate cell growth, patterning, and apoptosis\nare interdependent during development (1, 19, 29, 37). In\nparticular, there are likely to be multiple ways to elicit an\napoptotic cell fate since apoptosis serves a variety of functions\nin multicellular organisms (29). In regard to the cell autonomy\nof apoptosis, a cell-autonomous apoptosis occurs during cell\ncompetition, a phenomenon whereby cells that grow slowly due\nto the mutation of essential genes such as Minute or ras are\nremoved later in development (39, 40, 49, 56). In contrast, it\nhas been reported that nonautonomous cell death is often\nassociated with cell fate changes by altering the morphogen\nactivities (2, 3) of Decapentaplegic (Dpp) (32, 41) or Wingless\n(Wg) (42, 60). In these cases, apoptotic cell death occurs both\nwithin and outside of the abnormal cell population and is\nreferred to as morphogenetic apoptosis. Similarly, when spalt\n(sal), a target gene of the Dpp signal, is ectopically expressed\napoptosis is induced through interaction with the surrounding\nnormal cells (38). Thus, the removal of physiologically abnor-\nmal cells by apoptosis can occur by more than one mechanism,\nalthough the details of the various processes remain to be\nelucidated. In order to search for and analyze the ways to\nremove other types of abnormal cell, we have ectopically ex-\npressed various master genes, including homeotic genes, to\ncreate cell populations with developmentally abnormal fates.\nIn most cases, cells with abnormal fate induced severe apopto-\nsis in a cell autonomous or nonautonomous manner. Among\nthese, we focus on the master gene ss, of which overexpression\nin the wing changes the organ identity into a ventral appendage\nsuch as a leg or antenna.\nBased on sequence identity and splice site conservation, Ss is\nthe closest Drosophila homolog of the mammalian dioxin re-\nceptor (arylhydrocarbon receptor [Ahr]) (17, 27). Both mam-\nmalian and Drosophila proteins can also bind to the xenotoxin\nresponsive element (XRE) and stimulate transcription from\ngenes containing this cis-acting element (18). Although Ss has\nnot been shown to bind to arylhydrocarbons, it regulates nor-\nmal morphogenesis of the leg or antenna and bristles, all of\nwhich are major Drosophila sensor organs or tissues that re-\nspond to environmental chemicals.\nOur results reveal that ectopic ss provokes a wing-to-leg\nand/or antenna homeosis that subsequently elicits apoptosis in\nan autonomous or nonautonomous manner. This apoptotic\nresponse is regulated by a novel transmembrane leucine-rich\nrepeat (LRR) protein, Fili, and may be a common process\ninduced by ectopic expression of various homeotic genes.\nThese results indicate that homeosis elicits a complex set of\nsignals that inﬂuence the survival of the transformed cells and\ntheir surrounding cells. In addition, these results suggest a\nhypothesis that different LRR family transmembrane proteins\nfunction in different subdomains of a developing ﬁeld to rec-\nognize developmentally misspeciﬁed cells and to regulate cell\nsurvival.\nFly strains. We used two different GAL4 enhancer trap lines of scalloped (sd)\nin the present study. The (s) and (w) lines induce a strong and weak expression\nof UAS-driven genes, respectively. The plasmid construct for tgo-RNAi was made\nby inserting an inverted tgo cDNA into the 5 region of the standard UAS-tgo\nconstruct. When tgo-RNAi was induced singly in the wing, the ﬂy wing showed no\nphenotype (data not shown).\nThe name ﬁsh-lips was derived from the observation that the expression pat-\ntern of ﬁli in the wing disc made a ﬁsh lips-like shape. ﬁli is identical to the\ntemporarily assigned hypothetical gene CG4054 in the Berkeley Drosophila Ge-\nnome Project. The ﬁli-lacZ ﬂy was identiﬁed during a screen of lacZ enhancer\ntrap strains with P-lacW, a derivative of the transposon P-element (6). The ﬁli\nmutant allele 102 was isolated by an imprecise excision of P-lacW in ﬁli-lacZ\naccording to the standard procedure. The UAS-ﬁli ﬂy was made by inserting the\nﬁli cDNA (2,297 bp, including full-length open reading frame, 18 bp of the 5\nuntranslated region and 62 bp of the 3 untranslated region) into the pUAST\nvector (7) with a white marker and transforming the white ﬂy according to\nstandard procedures.\nThe other lacZ enhancer trap lines we used are sdETX4, Dll01092, hth05745,\nhid05014, ombP1, fz3SW076-A7.3XW, trnS064117, and pucE69. hid05014 was also used as\na mutant (see Fig. 4E and G). At the late-third-instar larval stage, this hid-lacZ\nis expressed irrespective of apoptosis around the dorsal-ventral boundary and at\nthe two spots in the dorsal and ventral hinge region (data not shown). However,\nat the mid-third-instar larval stage, these expressions do not begin, and the\nexpression is associated with apoptosis. Therefore, we observed the hepCA- or\nss-induced hid-lacZ expression by using the mid-third-instar larvae.\nDetection of apoptosis. Caspase-3 plays a central role in many types of apo-\nptosis, whereas c-Jun N-terminal kinase (JNK) activation elicits a limited group\nof apoptotic events (14). In the Drosophila larval wing disc, activation of JNK\nalways leads to the activation of caspase-3 (2). Puckered (Puc) is a protein\nphosphatase speciﬁcally inactivating JNK, and its transcription is induced by JNK\nsignal, thereby making a negative-regulatory circuit (36). Most of the endoge-\nnous and ectopic puc-lacZ expression observed in the present study was elimi-\nnated in a mutant background of the hep gene (20), which encodes the homolog\nof mitogen-activated protein kinase kinase-7 that activates JNK. Immunoﬂuo-\nrescence was carried out according to the standard procedure. The puc-lacZ\nexpression is detected by measuring mouse anti--galactosidase antibody (1:200\ndilution; Promega Z378) and Cy3-labeled anti-mouse immunoglobulin (1:200\ndilution; Jackson Immunoresearch). The rabbit anti-active caspase-3 antibody\n(CM1) was a gift from Idun Pharmaceuticals. It speciﬁcally recognizes the\ncleaved and activated form of mammalian caspase-3 and has been shown to also\nrecognize the cleaved form of the Drosophila caspase-3 homolog, Drice (59).\nThus, active caspase-3/Drice was detected by CM1 (1:4,000 dilution) and Cy5-\nlabeled anti-rabbit immunoglobulin (1:200 dilution; Jackson Immunoresearch).\nAntibodies. The other antibodies we used were rat anti-Al (from G. Campbell\n[1:1,000 dilution]), rat anti-Sal (from R. Barrio [1:400 dilution]), mouse anti-Dac\n(MABDAC2-3 from the Developmental Studies Hybridoma Bank [1:10 dilution\nof supernatant]), rabbit anti-Vg (from S. Carroll [1:200 dilution]), rabbit anti-p-\nMad (PS1 from P. ten Dijke [1:200 dilution]), mouse anti-Arm (N2 7A1 from the\nDevelopmental Studies Hybridoma Bank [1:10 dilution of supernatant]), mouse\nanti-Dll (MAb DMDll.1 from D. Duncan [1:500 dilution]), and rabbit anti-Caps\n(from A. Nose [1:200 dilution]).\nMosaic overexpression analysis. Discs were prepared from larvae carrying the\nAyGAL4 (actin promoter-FRT-yellow,\nterminator-FRT-GAL4 [28]), hs-FLP\n(yeast FLP recombinase gene driven by heat shock promoter), UAS-ss, and\nUAS-GFP transgenes. The GAL4-expressing clones were induced by heat treat-\nment at 37°C for 20 min at 48 to 72 h after egg-laying and observed 48 to 72 h\nafter heat treatment. In each experiment, we observed more than 10 wing discs\nto conﬁrm the results.",
         "RESULTS\nOverexpression of ss leads to a disappearance of the wing,\nwhich is dependent on tgo. Using the GAL4/UAS system (7),\nwe overexpressed ss in the wing blade region by scalloped (sd)\n(s)-GAL4 (Fig. 1A). As a result, no adults developed wings\n(Fig. 1C). When we expressed ss in a more restricted region\nusing the weaker allele, sd(w)-GAL4 (Fig. 1B), the wing mar-\ngin structure disappeared (Fig. 1D and E). These results sug-\ngest that ectopic expression of ss in the wing leads to massive\napoptosis.\nMammalian Ahr is known to function together with a struc-\nturally similar protein, Arylhydrocarbon receptor nuclear\ntranslocator (Arnt), by forming a heterodimeric transcription\nfactor (27). To test whether the wing apoptosis phenotype by ss\noverexpression is mediated by Tango (Tgo) (43), the Drosoph-\nila homolog of Arnt, we examined ﬂies in which we simulta-\nneously induced both ss expression and tgo-RNAi. Prior to this\nexperiment, we examined whether tgo-RNAi can remove en-\ndogenous tgo function. When tgo-RNAi was induced in the\nantenna by using Distal-less (Dll)-GAL4, the antenna trans-\nformed to a leg-like organ as seen in the mutant of tgo (18)\nOverexpression of ss leads to a homeotic transformation\nfrom wing to ventral appendage before disappearance of the\ncells. ss is known to specify the identities of tarsus (distal leg)\nand arista (distal antenna) during normal development (17,\n18). Thus, we thought that the ss overexpression in the wing\nmay have removed the organ identity as a wing and underwent\na homeotic change to a leg, antenna, or other ventral append-\nage that does not actually exist in the normal ﬂy prior to\ndisappearance of the wing. To investigate this possibility, we\nnext examined the expression of marker genes speciﬁc to the\nwing or haltere or to leg or antenna. The sd gene, which is\nnormally expressed exclusively in the wing and haltere (10)\n(Fig. 2A), was found to be less strongly expressed in ss-over-\nexpressing clones (Fig. 2C and C). This result suggests that the\nss-overexpressing clones had lost, at least partially, their wing\nidentity. In contrast, Dll, a master gene that directs the identity\nof the distal leg or whole antenna (13, 45) (Fig. 2A), was\nectopically and cell autonomously induced in the ss-overex-\npressing clones (Fig. 2D and D). During normal wing devel-\nopment, Dll expression is induced in response to Wg signaling.\nHowever, Dll induction in the ss-overexpressing clones ap-\npeared to be independent of the Wg signal since Dfrizzled-3\n(fz3) induction and Armadillo (Arm) accumulation did not\ncoincide with the ss-overexpressing region, as demonstrated\nbelow. Therefore, we hypothesized that this Dll\ninduction\nmight provoke a different Dll function to redirect the identity\nof the wing to that of the distal leg or whole antenna. In fact,\nhigh-level induction of Dll in the wing has been reported to\nlead to its transformation into a distal leg (21). Although nor-\nmal expression of ss in the leg is known to be downstream of\nDll (17), Dll expression is regulated by ectopic ss in the wing,\nindicating the presence of a positive feedback loop between Dll\nand ectopic ss.\nNext, we examined the expression of various genes that are\nmarkers for different subdomains of the leg or antenna (Fig.\n2E to H) (8, 9, 15, 16, 35, 45; Flybase [http://ﬂybase.bio.indiana\n.edu/]). In summary, ss-overexpressing clones in the blade re-\ngion of the wing have most likely transformed their identity to\nthat of a distal leg (t1 area) rather than that of an antenna (Fig.\n2B). In the hinge region of the wing, Dll expression was not\ninduced by ss (Fig. 2D). Instead, the expression of hth, a master\ngene that directs the identity of the proximal leg or whole\nantenna (12, 44) was enhanced in the ss-overexpressing clones\n(Fig. 2H and H). However, in the normal leg or antenna there\nwas no segment that expresses both ss and hth but not Dll (Fig.\n2B). Thus, the fate of the ss-overexpressing clones in the hinge\nregion of the wing should have transformed to that of a ventral\nappendage, although we have not identiﬁed which normal ap-\npendage segment that the ss-overexpressing clones resemble.\nIn comparison, a leg-antenna intermediate and a hypothetical\nancestor appendage have been observed in Dll and ss mutants,\nrespectively (17, 18).\nWe also obtained other evidence demonstrating that the\ncells with ss overexpression display characters of the ventral\nappendages. First, an ectopic appendage-like structure was\noften generated (Fig. 2I) when ss-induced apoptosis was par-\ntially inhibited by coexpression of Drosophila inhibitor of ap-\noptosis protein 1 (DIAP1 [26]). The structure most similar to\nthis appendage that is seen in external morphology of normal\nﬂy is the tarsus of leg (Fig. 2I). Second, during normal devel-\nopment expression of patched (ptc) was dependent on Hedge-\nhog signaling, which induces a band-like expression of dpp in\nthe wing and the dorsal leg region and that of wg in the ventral\nleg region (5) (Fig. 2J and J). When ss expression was induced\nby ptc-GAL4, Wg was ectopically induced only in the ventral\nhalf of the ptc-expressing region (Fig. 2K and K), as found in\nthe normal leg or antenna. However, the level of wg induction\nin the ptc-expressing region was weaker than that found in the\nnormal leg. Therefore, the ss-expressing wing appears to have\npartially transformed its identity to that of a leg or antenna.\nThis mixed or conﬂicted identity may also be responsible for\ninduction of apoptosis. Although it has been shown previously\nthat the ectopic expression of ss by ptc-GAL4 leads to a dele-\ntion of the central wing (17), the molecular mechanism was not\nidentiﬁed. Our results suggest that the deletion is triggered\nprimarily by homeotic transformation.\nOverexpression of ss leads to apoptosis in which cell auton-\nomy is position dependent. To study the detailed mechanism\nleading to apoptosis by ectopic expression of ss, we next exam-\nined the cell autonomy of apoptosis in ss-overexpressing\nclones. We monitored caspase-3 and JNK activation by using\nan antibody that recognizes active caspase-3 and the expression\nof a reporter gene, puckered (puc)-lacZ, respectively. After\n48 h of mosaic induction, we observed two types of apoptosis\n(Fig. 3A). The ss-overexpressing clones inside the wing blade\nprimordium showed strong activation of caspase-3 and a weak\nactivation of JNK (Fig. 3A and A). Activation of caspase-3 in\nthe vicinity of the dorsoventral compartment boundary (red\nline in Fig. 3A) was relatively weak and delayed. We can\nrecognize the delay of the cell removal around the dorsoven-\ntral boundary more readily at later stage after the clone induc-\ntion (data not shown). All of these responses were always cell\nautonomous. In contrast, the ss-overexpressing clones outside\nof the wing blade region (that is the hinge region) showed weak\nactivation of caspase-3 and strong activation of JNK in cells on\neither side of the clone boundary (Fig. 3A). In this region,\nthe outline of the clones which were probably induced at an\nearlier stage, are severely disrupted with strong activation of\nboth caspase-3 and JNK (Fig. 3A).\nThe presence of the two types of apoptosis can clearly be\nrecognized in ss overexpression by ptc-GAL4, which is ex-\npressed in a band intersecting the dorsoventral boundary (Fig.\n3B to B). Strong, autonomous caspase-3 activation was ob-\nserved in the blade region, whereas strong, nonautonomous\nJNK activation was observed around the hinge region. The\nregion around the dorsoventral boundary did not show both\nJNK and caspase-3 activation. Thus, the wing disc can be\ndivided into three subdomains based on the nature of the\napoptotic response to ss overexpression in these regions (Fig.\n3C). Interestingly, these subdomains correlate with the expres-\nsion domains of several key transcriptional regulators. The\nDll-expressing subdomain showed an autonomous and delayed\nresponse. The vestigial (vg)-expressing subdomain showed an\nautonomous and acute response. The homothorax (hth)-ex-\npressing subdomain showed a nonautonomous response.\nThese subdomains can be recognized by the foldings of the disc\ncell layer or by distance between the nuclei, through counter\nstaining with DAPI (4,6-diamidino-2-phenylindole). The\nvariance of apoptotic responses are summarized in Table 1.\nSimilar traits in position-dependent cell autonomy of apoptosis\nhave also been found in the clones mutant for the Dpp recep-\ntor Thick veins (Tkv) (3).\nss-induced apoptosis is dependent on Dll that is responsible\nfor homeotic transformation from wing to leg or antenna. To\nelucidate the role of Dll in ss-induced apoptosis, we placed the\nss-overexpressing clones in a strong Dll mutant Dll5/Dll9 back-\nground. As shown in Fig. 3D, the activation of both JNK and\ncaspase-3 was strongly suppressed by this Dll mutation in the\nwing blade region. Activation was also suppressed in the hinge\nregion, although ss did not induce Dll expression in this region\n(Fig. 2D). A possible explanation is that broader expression of\nDll in the hinge region at an earlier stage of development (data\nnot shown) is required for transformation by ss in later stages.\nThe Dll mutation also suppressed the ss-induced adult wing\nphenotype (Fig. 3E). To test whether Dll overexpression is also\nable to lead to apoptosis, we next generated Dll-overexpressing\nclones in the wing. As shown in Fig. 3F, JNK and caspase-3\nactivation are observed cell autonomously in the Dll-overex-\npressing clones in the wing blade region. Other areas show\nnonautonomy. The wing notching phenotype induced by Dll\noverexpression is found to be similar to that induced by ss\noverexpression (Fig. 3G). Thus, ss-induced apoptosis is likely\nmediated by a pathway that is dependent upon Dll. A discrep-\nancy existed in that the normal expression of Dll in the wing did\nnot induce leg identity or apoptosis. However, the elevated\nexpression of Dll in the wing can induce some leg structure\n(21). Thus, the level of Dll expression, rather than its timing or\nplace, may be crucial for determining the fate as wing or leg.\nCoincidence of round shape and apoptotic response. Apo-\nptosis in each of these regions was dependent on Hemipterous\n(Hep, an activator of JNK [20]), DIAP1, Head involution de-\nfective (Hid; a proapoptotic protein [23]) and Tgo (Fig. 4B to\nF). DIAP1 is known to degrade caspase through its ubiquitin-\nprotein ligase activity. Hid can bind to DIAP1 to stimulate\nautoubiquitination of DIAP1, which prevents the degradation\nof caspase (58). Interestingly, the ss-overexpressing clones in\nhep or hid mutant or in DIAP1 overproducer backgrounds\nretained a round shape, as seen with ss-overexpressing clones\nin the wild-type background (Fig. 4A to E). In contrast, the\nss-overexpressing clones in the tgo-RNAi background lost their\nround shape and invaded the surrounding normal cells (Fig.\n4F). These results indicate that hep and hid only affect the\napoptosis pathway, whereas tgo mediated events more up-\nstream, such as the cell fate change induced by ss. Ectopic\nexpression of ss may make a difference in cell afﬁnity between\nthe clones and surrounding normal cells, which leads to a\nround shape of the clones and subsequent apoptotic response.\nConsistently, activation of JNK pathway by constitutively\nactive Hep leads to hid expression (Fig. 4G). Besides, the\nss-overexpressing clone induces hid expression autonomously\nin the wing blade region but nonautonomously in the wing\nhinge region (Fig. 4H).\nEffect of ss overexpression on the intensity of Dpp and Wg\nsignaling. The JNK-dependent cell death on either side of the\nclone boundary is quite similar to what has been referred to as\n“morphogenetic apoptosis,” through which discontinuities in\nthe Dpp and Wg morphogen activity gradients are corrected\n(2). We next examined the activities of certain morphogen\nsignals in the vicinity of the ss-overexpressing clones. With\nregard to Dpp signaling, we can ﬁnd a slight reduction in the\nexpression of the lacZ enhancer trap of optomotor-blind (omb),\na target gene of the Dpp signal (24) (see Fig. S1A and B in\nthe supplemental material). However, the reduction is always\nfound around the center of the clones and does not coincide\nwith the outline of the clones. We also see a similarly moderate\nalteration of the level of phospho-Mad, an active form of the\nDpp signal transducer “Mothers against dpp” (Mad) (48, 52,\n57) (see Fig. S1A and B in the supplemental material). In the\ncase of Wg signaling, we examined expression of the fz3-lacZ\nenhancer trap (51, 55), a target gene of the Wg signal, and the\nlevel of Arm (46, 54), a Wg signal transducer that is known to\naccumulate in response to the Wg signal (25, 47). We observed\na reduction of fz3-lacZ expression interior to the clone bound-\nary of the ss-overexpressing clones (see Fig. S1C, C, and D to\nD in the supplemental material). However, in the most central\ncells, cytoplasmic accumulation of Arm was conversely ob-\nserved (see Fig. S1C, C, D, and D in the supplemental ma-\nterial). These results suggested that the level of Wg signal is\nthe ss-overexpressing\nnot uniformly received throughout\nclones and is not directly regulated by Ss. Similar responses\nwere observed in clones both in the wing blade and in the hinge\nregions, as seen in Fig. S1C in the supplemental material.\nTaken together, these results indicate that discontinuities in\nthe strength of Dpp and Wg signal reception does not precisely\ncoincide with the boundary of the ss-overexpressing clones,\nsuggesting that other factors contribute to the nonautonomous\nJNK activation in this case.\nPosition-dependent variance in the autonomy of apoptosis is\nfound also in clones ectopically expressing homeotic genes. We\nnext examined whether the position-dependent variance in cell\nautonomy of apoptosis is speciﬁc for ss-induced homeosis or a\ngeneral trait exhibited by ectopic expression of diverse ho-\nmeotic genes. The identity of each body segment is deﬁned by\na series of homeotic genes belonging to ANTP-C and BX-C.\nHowever, the Drosophila wing requires no input from ANTP-C\nand BX-C during its development (11). Instead, these ho-\nmeotic genes repress wing development in all segments, except\nfor the mid-thorax, where the wing normally develops. In order\nto produce a misspeciﬁed fate in wing cells, we expressed\nvarious homeotic genes in the wing discs. When labial (lab),\nproboscipedia (pb), Antennapedia (Antp), Ultrabithorax (Ubx),\nabdominal-A (abd-A), or Abdominal-B (Abd-B) were expressed\nby using sd-GAL4 driver, the adult wings showed abnormal\nvein patterns and incision of the wing margin (Fig. 5A to F). In\nthe cases of Ubx or abd-A expression, wing incision was less\nfrequently observed. In these cases, the wing has been shown\nto transform to an organ similar to a haltere (22). When mo-\nsaic clones expressing these homeotic genes are generated,\nJNK activation was found autonomously in the blade region\nbut nonautonomously in the hinge region. All of these re-\nsponses are quite similar to the results of ss overexpression, as\nhas been described above. Thus, the position-dependent vari-\nance in cell autonomy of apoptosis may be a general phenom-\nenon accompanying homeotic transformation.\nCharacterization of the novel gene ﬁli that encodes a LRR\nfamily transmembrane protein. Recently, clones with sal-in-\nduced misspeciﬁed fate were reported to be recognized by the\nsurrounding normal cells through the LRR family transmem-\nbrane proteins, Capricious (Caps) and Tartan (Trn), to prevent\napoptosis (38). The expression pattern of both caps and trn are\ncomplementary to the pattern of sal expression in the late third\ninstar wing (38) (Fig. 6C). Thus, we investigated the role of a\nnewly identiﬁed LRR family protein, Fili, which was expressed\nin the wing in a nearly complementary pattern as that of Dll\n(Fig. 6A and B), in regulating apoptosis in ss-overexpressing\nclones.\nThe ﬂy strain ﬁli-lacZ was isolated by a screen of enhancer\ntrap lines that displayed a previously unknown expression pat-\ntern. The transposon P-element carrying the lacZ gene was\ninserted at 56 relative to the presumptive transcription start\nsite, TTAGTT (Fig. 7A). The deduced protein, Fili, was com-\nposed of 738 amino acid residues and had a single transmem-\nbrane domain and 14 LRR domains (Fig. 7B). These features\nare common to Caps and Trn, which share 55 and 56% ho-\nmologies with Fili in the LRR domains, respectively (Fig. 7C\nand D). The homologies also extend to the amino- and car-\nboxyl-terminus domains (Fig. 7E and F), suggesting that Fili\nplays a role that is especially related to Caps and Trn among\nnumerous members of LRR family with a variety of functions.\nA mutant, 102, was isolated by imprecise excision of the\nP-element in ﬁli-lacZ. It lacks 534 bp, including a part of the\nﬁrst exon, and is lethal at late embryonic stages. The main\ncause of this lethality is unknown. In the wing disc, ﬁli exhibited\na pattern of gene expression that was nearly complementary to\nthat of Dll (Fig. 6A and B). In addition, ﬁli expression was\ncomplementary to Dll expression in antenna discs (Fig. 6A\nand B). In contrast, in the leg disc, ﬁli seemed to partially\noverlap with Dll (Fig. 6A and B). However, we found that\nmost of the cells expressing the respective genes were in dif-\nferent layers of the leg disc folding.\nss-induced apoptosis can be controlled by Fili. If ﬁli is in-\nvolved in the control of apoptosis in ss-overexpressing clone, its\nexpression should be downregulated by ss. As expected, ﬁli\nexpression is repressed in the ss-overexpressing clones (Fig.\n8A). Interestingly, in contrast, expression of caps (Fig. 8A)\nand trn (data not shown) is enhanced by ss. Similar responses\nin ﬁli and caps can also be observed in Dll-overexpressing\nclones (Fig. 8B to B).\nWhen ss was induced together with ﬁli, JNK and caspase-3\nactivation was greatly reduced (Fig. 3A and 9A to A). Resid-\nual JNK and caspase-3 activation was observed only in the\nmost peripheral hinge region, where ﬁli is not expressed in the\nnormal wing. Similarly, in the central wing region, where ﬁli is\nnot expressed in the normal wing, caspase-3 activation was not\nsuppressed by ﬁli coexpression. As a control experiment, we\nobserved that coexpression of both caps and ss did not suppress\nthe ss-induced apoptotic response (Fig. 9B and B). Consis-\ntently, ss overexpression does not repress but rather elevates\nthe caps expression (Fig. 8A). Thus, we conclude that ﬁli\nexpression is sufﬁcient for survival of ss-overexpressing cells in\nthe wing region where endogenous ﬁli is expressed, probably\nthrough recognition by surrounding normal cells transmitting a\nsurvival signal, as shown for Caps and Trn (38).\nWe further carried out a control experiment to test the effect\nof ﬁli on sal-induced apoptosis because sal-induced apoptosis\nwas reported to be suppressed by coexpression of caps or trn.\nAt ﬁrst, we observed that the sal-overexpressing clones dis-\nplayed a nonautonomous activation of JNK in the wing region\nwhere sal is not normally expressed (Fig. 9C to C). This\nnonautonomy was quite similar to the morphogenetic apopto-\nsis and ss-induced apoptosis, suggesting that the regulation of\nall of these apoptotic responses may share similar molecular\nmechanism(s). When sal was induced together with ﬁli, JNK,\nand caspase-3 activation was not affected signiﬁcantly (Fig. 9D\nto D). Thus, these results indicate that ﬁli does not suppress\nall cases of nonautonomously induced apoptosis.\nShape and apoptosis of clones mutant for or overexpressing\nﬁli. Next, we examined the relationship of the clone shape and\napoptosis in a simple ﬁli mutant or in its overexpression. The\nﬁli mutant clones showed an irregular shape irrespective of\ntheir position in the wing disc (Fig. 10A) and did not display\nany apoptotic signs. These results are not always discrepant\nfrom the other results and are similar to observations in mu-\ntants for caps or trn alone (38). The other LRR family proteins\nmight function with Fili in a redundant manner in the gener-\nation of afﬁnity with surrounding normal cells. In contrast to\nthe ﬁli mutant, the ﬁli-overexpressing clones showed a smooth\noutline (Fig. 10B). The smooth outlines were found even\nwhere endogenous ﬁli was expressed, which may be due to the\nimmense amount of its expression by using UAS/GAL4 system.\nHowever, the clone in the central region where endogenous ﬁli\nwas not expressed exhibited the most rounded shape (Fig.\n10B). Interestingly, unlike the case of caps and trn, the over-\nexpression of ﬁli alone induced apoptosis in a nonautonomous\nmanner. When UAS-ﬁli was induced by ptc-GAL4 (Fig. 10D),\nnonautonomous activation of JNK was apparent in the cells\naround the posterior edge of ptc expression but not in the cells\naround the anterior edge (Fig. 10D to D). The posterior edge\nis known to create a sharp discontinuity of ptc expression\nlevels, whereas the anterior edge does not (5). More interest-\ningly, the area where JNK was activated did not overlap the\narea where endogenous ﬁli was expressed. As a result, the\nareas with JNK activation were divided into three regions\n(arrowheads in Fig. 10D). Also, forced expression of ﬁli in the\ncentral wing blade in the adult wing generated a severely\nnotched phenotype by apoptosis (Fig. 10C). These data\nstrongly suggest that the occurrence of a large discontinuity in\nthe ﬁli expression level activates JNK nonautonomously.",
         "DISCUSSION\nEctopic ss induces a homeosis-dependent apoptosis that\nmay cause a rare frequency of wing-to-leg transdetermination.\nHomeosis is a naturally occurring rare phenomenon with for-\nmation of a local body part having characteristics that are\nnormally found in a related part at another location in the\nbody. This phenomenon had been studied exclusively in Dro-\nsophila melanogaster but can also be found in a variety of\ninsects and other organisms (53). Although homeosis is rarely\nfound in wild-type ﬂies, transdetermination, a similar phenom-\nenon, can be easily induced by in vivo culture and subsequent\ntransplantation of imaginal discs between different individuals\nor by directly manipulating functions of various master and/or\nhomeotic genes. Although such developmentally abnormal\ncells occurring in homeosis might a priori be expected to be\nremoved during development to guarantee normal morpho-\ngenesis, apoptosis has not been previously observed to accom-\npany homeosis. Our results suggest that the cells undergoing\nhomeosis do in fact induce apoptotic responses to help prevent\nthem from developing abnormal structures. The rare fre-\nquency in wing-to-leg transdetermination, which was observed\nabout 30 years ago (30), may be derived partly from the high\nfrequency of apoptosis associated with this particular transfor-\nmation.\nss-induced autonomous apoptosis is related to cell compe-\ntition. Cell competition is a phenomenon whereby cell clones\nwith a slow growth rate are eliminated during development.\nThe autonomous cell death in cell competition has been dem-\nonstrated to be caused by an impaired reception of extracel-\nlular survival factors. As representative examples, clones with\ncells heterozygous for the M(2)60E mutation show an auton-\nomous death as a result of reduced Dpp reception (40). Thus,\nthe autonomous death in ss-overexpressing cells in the wing\nblade might be a type of cell competition response. In this case,\nhowever, the extracellular factor that the ss-overexpressing\ncells fail to receive is not likely to be Dpp, since we observed\nthat the Dpp signaling level is not strongly affected in the\nss-overexpressing cells (see Fig. S1A and B in the supplemental\nmaterial). Also, ss-overexpressing cells can be removed from\nthe lateral region of the wing blade where the level of Dpp\nsignaling is normally low. In contrast, cells heterozygous for\nM(2)60E can survive in this area due to the loss of competition\nfor Dpp (40). Therefore, Wg or other extracellular survival\nfactors might be affecting the survival of ss-overexpressing\ncells, as discussed below.\nPosition-dependent variance in cell autonomy is found in\nthe apoptotic response to ss overexpression. We have not yet\naddressed the question of why the variance in cell autonomy of\nss-induced apoptosis is related to dorsoventral patterning or\nblade/hinge subdomains. However, the variance suggests an\ninteresting model in which signals that normally regulate dor-\nsoventral appendage patterning or blade/hinge subdomains\nalso directly affect the apoptotic response or speciﬁcation of\norgan identity as cryptic mechanisms. Interestingly, ectopic\nexpression of most of the segment-specifying homeotic genes\ncommonly shows a similar position-dependent variance in cell\nautonomy (Fig. 5). In this view, Wg, the morphogen controlling\nthe dorsoventral patterning (42, 60), or Vg, the selector of wing\nsubdomain (31, 34), may be involved in the regulation of po-\nsition-dependent variance in the cell autonomy of apoptosis.\nHowever, deﬁnitively demonstrating the involvement of Wg or\nVg is difﬁcult because changing Wg signal or Vg activity alone\nalso induces a severe unrelated apoptosis (2). Thus, the simul-\ntaneous manipulation of ss and Wg/Vg activities may lead to a\ncomplicated result that is difﬁcult to interpret.\nImplication of various LRR family transmembrane proteins\nin the regulation of apoptosis to remove misspeciﬁed cells. A\nrecent study concerning the relation of apoptosis in sal-over-\nexpressing cells and LRR family proteins Caps/Trn has pro-\nvided a new insights into how cell survival of misspeciﬁed cells\nis controlled (38). Cells that overexpress sal lack caps/trn ex-\npression and are removed from the area where sal is not\nnormally expressed. However, cells expressing both sal and\ncaps can survive. In order to control cell survival in the wing,\nCaps and Trn presumably provide position-dependent recog-\nnition cues along the anteroposterior axis, whereas Fili appears\nvation of mammalian Ahr is known to affect apoptosis, cell\nproliferation, differentiation, and morphogenesis in various tis-\nsues (50). Polycyclic aromatic hydrocarbon, a ligand for Ahr,\ninduced apoptosis accompanied by the activation of JNK (33).\nIt should also be noted that dioxin-induced responses such as\nP450 expression in the liver are not observed in all cells ex-\nposed to dioxin but are observed in a subset of centrilobular\ncells with the highest sensitivity to dioxin (4). Thus, the mosaic\nactivation of Ss and subsequent JNK activation in surrounding\ncells in Drosophila induce circumstances similar to that of liver\ncells exposed to dioxin. With regard to morphogenesis, the\ntype of morphogenetic signals that are modiﬁed or distorted\nduring dioxin action is unknown, despite many examples of\ndioxin-induced mismorphogenesis that have been reported.\nTherefore, our studies on Ss in Drosophila may provide at least\nin part, a common molecular and cellular basis for understand-\ning Ahr-induced apoptosis and mismorphogenesis. It could\ninvolve homeotic transformation of organ identity, recognition\nby surrounding cells through LRR protein, and apoptosis-me-\ndiated large deletion of tissues that can cause a malformation\nduring organogenesis. Interestingly, all of the genes involved in\nthis sequential process are evolutionarily conserved in mam-\nmals. Similar and detailed analysis in mammalian system will\nresolve these issues.\nWe thank R. Barrio, S. Campbell, S. T. Crews, A. Garcı´a-Bellido,\nB. A. Hay, Y. Hiromi, Y. H. Inoue, K. Irvine, W. Janning, E. Martı´n-\nBlanco, A. Martinez-Arias, A. Nose, S. Noselli, C. Rauskolb, M.\nShinza-Kameda, G. Struhl, K. Takahashi, and D. Yamamoto, Bloom-\nington Stock Center, and GETDB (GAL4-Enhancer Trap Data Base)\nin the National Institute of Genetics in Japan, for the ﬂy strains; R.\nBarrio, G. Campbell, S. B. Carroll, D. Duncan, A. Nose, M. Shinza-\nKameda, and P. ten Dijke, Developmental Studies Hybridoma Bank,\nIowa University, and Idun Pharmaceutical, Inc., for antibodies; J. Kim\nand Y. Lee for technical advice; and Y. Aoki and O. Habara for\ntechnical assistance.\nto provide such cues to neighboring cells along the dorsoven-\ntral axis. These results raise the possibility that cells can rec-\nognize the type of neighboring cells using distinct LRR family\nmembers, depending on the position of the cells in the primor-\ndial tissues. The Drosophila genome contains at least 10 mem-\nbers of LRR family, most of which have not been examined for\nfunction (Flybase).\nFunctional relationship between Drosophila Ss and mamma-\nlian Ahr. In the present study, several cellular responses to ss\noverexpression have been described. Clonal overexpression of\nss induces a sequential response composed of an induction of\nleg- or antenna-speciﬁc genes, autonomous homeosis, round-\ning, and autonomous or nonautonomous apoptosis. The acti-\n17. Duncan, D. M., E. A. Burgess, and I. Duncan. 1998. Control of distal\nantennal identity and tarsal development in Drosophila by spineless-aristape-\ndia, a homolog of the mammalian dioxin receptor. Genes Dev. 12:1290–1303.\n18. Emmons, R. B., D. Duncan, P. A. Estes, P. Kiefel, J. T. Mosher, M. Son-\nnenfeld, M. P. Ward, I. Duncan, and S. T. Crews. 1999. The Spineless-\nAristapedia and Tango bHLH-PAS proteins interact to control antennal and\ntarsal development in Drosophila. Development 126:3937–3945.\n31. Kim, J., A. Sebring, J. J. Esch, M. E. Kraus, K. Vorwerk, J. Magee, and S. B.\nCarroll. 1996. Integration of positional signals and regulation of wing for-\nmation and identity by Drosophila vestigial gene. Nature 382:133–138.\n32. Lecuit, T., W. J. Brook, M. Ng, M. Calleja, H. Sun, and S. M. Cohen. 1996.\nTwo distinct mechanisms for long-range patterning by Decapentaplegic in\nthe Drosophila wing. Nature 381:387–393.",
         "",
         "Growth, patterning, and apoptosis are mutually interactive during development. For example, cells that\nselect an abnormal fate in a developing ﬁeld are frequently removed by apoptosis. An important issue in this\nprocess that needs to be resolved is the mechanism used by cells to discern their correct fate from an abnormal\nfate. In order to examine this issue, we developed an animal model that expresses the dioxin receptor homolog\nSpineless (Ss) ectopically in the Drosophila wing. The presence of mosaic clones ectopically expressing ss results\nin a local transformation of organ identity, homeosis, from wing into a leg or antenna. The cells with\nmisspeciﬁed fates subsequently activate c-Jun N-terminal kinase to undergo apoptosis in an autonomous or\nnonautonomous manner depending on their position within the wing, suggesting that a cell-cell interaction is,\nat least in some cases, involved in the detection of misspeciﬁed cells. Similar position dependence is commonly\nobserved when various homeotic genes controlling the body segments are ectopically expressed. The autono-\nmous and nonautonomous apoptosis caused by ss is regulated by a novel leucine-rich repeat family trans-\nmembrane protein, Fish-lips (Fili) that interacts with surrounding normal cells. These data support a mech-\nanism in which the lack of some membrane proteins helps to recognize the presence of different cell types and\ndirect these cells to an apoptotic fate in order to exclude them from the normal developing ﬁeld.\nSignals that regulate cell growth, patterning, and apoptosis\nare interdependent during development (1, 19, 29, 37). In\nparticular, there are likely to be multiple ways to elicit an\napoptotic cell fate since apoptosis serves a variety of functions\nin multicellular organisms (29). In regard to the cell autonomy\nof apoptosis, a cell-autonomous apoptosis occurs during cell\ncompetition, a phenomenon whereby cells that grow slowly due\nto the mutation of essential genes such as Minute or ras are\nremoved later in development (39, 40, 49, 56). In contrast, it\nhas been reported that nonautonomous cell death is often\nassociated with cell fate changes by altering the morphogen\nactivities (2, 3) of Decapentaplegic (Dpp) (32, 41) or Wingless\n(Wg) (42, 60). In these cases, apoptotic cell death occurs both\nwithin and outside of the abnormal cell population and is\nreferred to as morphogenetic apoptosis. Similarly, when spalt\n(sal), a target gene of the Dpp signal, is ectopically expressed\napoptosis is induced through interaction with the surrounding\nnormal cells (38). Thus, the removal of physiologically abnor-\nmal cells by apoptosis can occur by more than one mechanism,\nalthough the details of the various processes remain to be\nelucidated. In order to search for and analyze the ways to\nremove other types of abnormal cell, we have ectopically ex-\npressed various master genes, including homeotic genes, to\ncreate cell populations with developmentally abnormal fates.\nIn most cases, cells with abnormal fate induced severe apopto-\nsis in a cell autonomous or nonautonomous manner. Among\nthese, we focus on the master gene ss, of which overexpression\nin the wing changes the organ identity into a ventral appendage\nsuch as a leg or antenna.\nBased on sequence identity and splice site conservation, Ss is\nthe closest Drosophila homolog of the mammalian dioxin re-\nceptor (arylhydrocarbon receptor [Ahr]) (17, 27). Both mam-\nmalian and Drosophila proteins can also bind to the xenotoxin\nresponsive element (XRE) and stimulate transcription from\ngenes containing this cis-acting element (18). Although Ss has\nnot been shown to bind to arylhydrocarbons, it regulates nor-\nmal morphogenesis of the leg or antenna and bristles, all of\nwhich are major Drosophila sensor organs or tissues that re-\nspond to environmental chemicals.\nOur results reveal that ectopic ss provokes a wing-to-leg\nand/or antenna homeosis that subsequently elicits apoptosis in\nan autonomous or nonautonomous manner. This apoptotic\nresponse is regulated by a novel transmembrane leucine-rich\nrepeat (LRR) protein, Fili, and may be a common process\ninduced by ectopic expression of various homeotic genes.\nThese results indicate that homeosis elicits a complex set of\nsignals that inﬂuence the survival of the transformed cells and\ntheir surrounding cells. In addition, these results suggest a\nhypothesis that different LRR family transmembrane proteins\nfunction in different subdomains of a developing ﬁeld to rec-\nognize developmentally misspeciﬁed cells and to regulate cell\nsurvival.\nFly strains. We used two different GAL4 enhancer trap lines of scalloped (sd)\nin the present study. The (s) and (w) lines induce a strong and weak expression\nof UAS-driven genes, respectively. The plasmid construct for tgo-RNAi was made\nby inserting an inverted tgo cDNA into the 5 region of the standard UAS-tgo\nconstruct. When tgo-RNAi was induced singly in the wing, the ﬂy wing showed no\nphenotype (data not shown).\nThe name ﬁsh-lips was derived from the observation that the expression pat-\ntern of ﬁli in the wing disc made a ﬁsh lips-like shape. ﬁli is identical to the\ntemporarily assigned hypothetical gene CG4054 in the Berkeley Drosophila Ge-\nnome Project. The ﬁli-lacZ ﬂy was identiﬁed during a screen of lacZ enhancer\ntrap strains with P-lacW, a derivative of the transposon P-element (6). The ﬁli\nmutant allele 102 was isolated by an imprecise excision of P-lacW in ﬁli-lacZ\naccording to the standard procedure. The UAS-ﬁli ﬂy was made by inserting the\nﬁli cDNA (2,297 bp, including full-length open reading frame, 18 bp of the 5\nuntranslated region and 62 bp of the 3 untranslated region) into the pUAST\nvector (7) with a white marker and transforming the white ﬂy according to\nstandard procedures.\nThe other lacZ enhancer trap lines we used are sdETX4, Dll01092, hth05745,\nhid05014, ombP1, fz3SW076-A7.3XW, trnS064117, and pucE69. hid05014 was also used as\na mutant (see Fig. 4E and G). At the late-third-instar larval stage, this hid-lacZ\nis expressed irrespective of apoptosis around the dorsal-ventral boundary and at\nthe two spots in the dorsal and ventral hinge region (data not shown). However,\nat the mid-third-instar larval stage, these expressions do not begin, and the\nexpression is associated with apoptosis. Therefore, we observed the hepCA- or\nss-induced hid-lacZ expression by using the mid-third-instar larvae.\nDetection of apoptosis. Caspase-3 plays a central role in many types of apo-\nptosis, whereas c-Jun N-terminal kinase (JNK) activation elicits a limited group\nof apoptotic events (14). In the Drosophila larval wing disc, activation of JNK\nalways leads to the activation of caspase-3 (2). Puckered (Puc) is a protein\nphosphatase speciﬁcally inactivating JNK, and its transcription is induced by JNK\nsignal, thereby making a negative-regulatory circuit (36). Most of the endoge-\nnous and ectopic puc-lacZ expression observed in the present study was elimi-\nnated in a mutant background of the hep gene (20), which encodes the homolog\nof mitogen-activated protein kinase kinase-7 that activates JNK. Immunoﬂuo-\nrescence was carried out according to the standard procedure. The puc-lacZ\nexpression is detected by measuring mouse anti--galactosidase antibody (1:200\ndilution; Promega Z378) and Cy3-labeled anti-mouse immunoglobulin (1:200\ndilution; Jackson Immunoresearch). The rabbit anti-active caspase-3 antibody\n(CM1) was a gift from Idun Pharmaceuticals. It speciﬁcally recognizes the\ncleaved and activated form of mammalian caspase-3 and has been shown to also\nrecognize the cleaved form of the Drosophila caspase-3 homolog, Drice (59).\nThus, active caspase-3/Drice was detected by CM1 (1:4,000 dilution) and Cy5-\nlabeled anti-rabbit immunoglobulin (1:200 dilution; Jackson Immunoresearch).\nAntibodies. The other antibodies we used were rat anti-Al (from G. Campbell\n[1:1,000 dilution]), rat anti-Sal (from R. Barrio [1:400 dilution]), mouse anti-Dac\n(MABDAC2-3 from the Developmental Studies Hybridoma Bank [1:10 dilution\nof supernatant]), rabbit anti-Vg (from S. Carroll [1:200 dilution]), rabbit anti-p-\nMad (PS1 from P. ten Dijke [1:200 dilution]), mouse anti-Arm (N2 7A1 from the\nDevelopmental Studies Hybridoma Bank [1:10 dilution of supernatant]), mouse\nanti-Dll (MAb DMDll.1 from D. Duncan [1:500 dilution]), and rabbit anti-Caps\n(from A. Nose [1:200 dilution]).\nMosaic overexpression analysis. Discs were prepared from larvae carrying the\nAyGAL4 (actin promoter-FRT-yellow,\nterminator-FRT-GAL4 [28]), hs-FLP\n(yeast FLP recombinase gene driven by heat shock promoter), UAS-ss, and\nUAS-GFP transgenes. The GAL4-expressing clones were induced by heat treat-\nment at 37°C for 20 min at 48 to 72 h after egg-laying and observed 48 to 72 h\nafter heat treatment. In each experiment, we observed more than 10 wing discs\nto conﬁrm the results.\nRESULTS\nOverexpression of ss leads to a disappearance of the wing,\nwhich is dependent on tgo. Using the GAL4/UAS system (7),\nwe overexpressed ss in the wing blade region by scalloped (sd)\n(s)-GAL4 (Fig. 1A). As a result, no adults developed wings\n(Fig. 1C). When we expressed ss in a more restricted region\nusing the weaker allele, sd(w)-GAL4 (Fig. 1B), the wing mar-\ngin structure disappeared (Fig. 1D and E). These results sug-\ngest that ectopic expression of ss in the wing leads to massive\napoptosis.\nMammalian Ahr is known to function together with a struc-\nturally similar protein, Arylhydrocarbon receptor nuclear\ntranslocator (Arnt), by forming a heterodimeric transcription\nfactor (27). To test whether the wing apoptosis phenotype by ss\noverexpression is mediated by Tango (Tgo) (43), the Drosoph-\nila homolog of Arnt, we examined ﬂies in which we simulta-\nneously induced both ss expression and tgo-RNAi. Prior to this\nexperiment, we examined whether tgo-RNAi can remove en-\ndogenous tgo function. When tgo-RNAi was induced in the\nantenna by using Distal-less (Dll)-GAL4, the antenna trans-\nformed to a leg-like organ as seen in the mutant of tgo (18)\nOverexpression of ss leads to a homeotic transformation\nfrom wing to ventral appendage before disappearance of the\ncells. ss is known to specify the identities of tarsus (distal leg)\nand arista (distal antenna) during normal development (17,\n18). Thus, we thought that the ss overexpression in the wing\nmay have removed the organ identity as a wing and underwent\na homeotic change to a leg, antenna, or other ventral append-\nage that does not actually exist in the normal ﬂy prior to\ndisappearance of the wing. To investigate this possibility, we\nnext examined the expression of marker genes speciﬁc to the\nwing or haltere or to leg or antenna. The sd gene, which is\nnormally expressed exclusively in the wing and haltere (10)\n(Fig. 2A), was found to be less strongly expressed in ss-over-\nexpressing clones (Fig. 2C and C). This result suggests that the\nss-overexpressing clones had lost, at least partially, their wing\nidentity. In contrast, Dll, a master gene that directs the identity\nof the distal leg or whole antenna (13, 45) (Fig. 2A), was\nectopically and cell autonomously induced in the ss-overex-\npressing clones (Fig. 2D and D). During normal wing devel-\nopment, Dll expression is induced in response to Wg signaling.\nHowever, Dll induction in the ss-overexpressing clones ap-\npeared to be independent of the Wg signal since Dfrizzled-3\n(fz3) induction and Armadillo (Arm) accumulation did not\ncoincide with the ss-overexpressing region, as demonstrated\nbelow. Therefore, we hypothesized that this Dll\ninduction\nmight provoke a different Dll function to redirect the identity\nof the wing to that of the distal leg or whole antenna. In fact,\nhigh-level induction of Dll in the wing has been reported to\nlead to its transformation into a distal leg (21). Although nor-\nmal expression of ss in the leg is known to be downstream of\nDll (17), Dll expression is regulated by ectopic ss in the wing,\nindicating the presence of a positive feedback loop between Dll\nand ectopic ss.\nNext, we examined the expression of various genes that are\nmarkers for different subdomains of the leg or antenna (Fig.\n2E to H) (8, 9, 15, 16, 35, 45; Flybase [http://ﬂybase.bio.indiana\n.edu/]). In summary, ss-overexpressing clones in the blade re-\ngion of the wing have most likely transformed their identity to\nthat of a distal leg (t1 area) rather than that of an antenna (Fig.\n2B). In the hinge region of the wing, Dll expression was not\ninduced by ss (Fig. 2D). Instead, the expression of hth, a master\ngene that directs the identity of the proximal leg or whole\nantenna (12, 44) was enhanced in the ss-overexpressing clones\n(Fig. 2H and H). However, in the normal leg or antenna there\nwas no segment that expresses both ss and hth but not Dll (Fig.\n2B). Thus, the fate of the ss-overexpressing clones in the hinge\nregion of the wing should have transformed to that of a ventral\nappendage, although we have not identiﬁed which normal ap-\npendage segment that the ss-overexpressing clones resemble.\nIn comparison, a leg-antenna intermediate and a hypothetical\nancestor appendage have been observed in Dll and ss mutants,\nrespectively (17, 18).\nWe also obtained other evidence demonstrating that the\ncells with ss overexpression display characters of the ventral\nappendages. First, an ectopic appendage-like structure was\noften generated (Fig. 2I) when ss-induced apoptosis was par-\ntially inhibited by coexpression of Drosophila inhibitor of ap-\noptosis protein 1 (DIAP1 [26]). The structure most similar to\nthis appendage that is seen in external morphology of normal\nﬂy is the tarsus of leg (Fig. 2I). Second, during normal devel-\nopment expression of patched (ptc) was dependent on Hedge-\nhog signaling, which induces a band-like expression of dpp in\nthe wing and the dorsal leg region and that of wg in the ventral\nleg region (5) (Fig. 2J and J). When ss expression was induced\nby ptc-GAL4, Wg was ectopically induced only in the ventral\nhalf of the ptc-expressing region (Fig. 2K and K), as found in\nthe normal leg or antenna. However, the level of wg induction\nin the ptc-expressing region was weaker than that found in the\nnormal leg. Therefore, the ss-expressing wing appears to have\npartially transformed its identity to that of a leg or antenna.\nThis mixed or conﬂicted identity may also be responsible for\ninduction of apoptosis. Although it has been shown previously\nthat the ectopic expression of ss by ptc-GAL4 leads to a dele-\ntion of the central wing (17), the molecular mechanism was not\nidentiﬁed. Our results suggest that the deletion is triggered\nprimarily by homeotic transformation.\nOverexpression of ss leads to apoptosis in which cell auton-\nomy is position dependent. To study the detailed mechanism\nleading to apoptosis by ectopic expression of ss, we next exam-\nined the cell autonomy of apoptosis in ss-overexpressing\nclones. We monitored caspase-3 and JNK activation by using\nan antibody that recognizes active caspase-3 and the expression\nof a reporter gene, puckered (puc)-lacZ, respectively. After\n48 h of mosaic induction, we observed two types of apoptosis\n(Fig. 3A). The ss-overexpressing clones inside the wing blade\nprimordium showed strong activation of caspase-3 and a weak\nactivation of JNK (Fig. 3A and A). Activation of caspase-3 in\nthe vicinity of the dorsoventral compartment boundary (red\nline in Fig. 3A) was relatively weak and delayed. We can\nrecognize the delay of the cell removal around the dorsoven-\ntral boundary more readily at later stage after the clone induc-\ntion (data not shown). All of these responses were always cell\nautonomous. In contrast, the ss-overexpressing clones outside\nof the wing blade region (that is the hinge region) showed weak\nactivation of caspase-3 and strong activation of JNK in cells on\neither side of the clone boundary (Fig. 3A). In this region,\nthe outline of the clones which were probably induced at an\nearlier stage, are severely disrupted with strong activation of\nboth caspase-3 and JNK (Fig. 3A).\nThe presence of the two types of apoptosis can clearly be\nrecognized in ss overexpression by ptc-GAL4, which is ex-\npressed in a band intersecting the dorsoventral boundary (Fig.\n3B to B). Strong, autonomous caspase-3 activation was ob-\nserved in the blade region, whereas strong, nonautonomous\nJNK activation was observed around the hinge region. The\nregion around the dorsoventral boundary did not show both\nJNK and caspase-3 activation. Thus, the wing disc can be\ndivided into three subdomains based on the nature of the\napoptotic response to ss overexpression in these regions (Fig.\n3C). Interestingly, these subdomains correlate with the expres-\nsion domains of several key transcriptional regulators. The\nDll-expressing subdomain showed an autonomous and delayed\nresponse. The vestigial (vg)-expressing subdomain showed an\nautonomous and acute response. The homothorax (hth)-ex-\npressing subdomain showed a nonautonomous response.\nThese subdomains can be recognized by the foldings of the disc\ncell layer or by distance between the nuclei, through counter\nstaining with DAPI (4,6-diamidino-2-phenylindole). The\nvariance of apoptotic responses are summarized in Table 1.\nSimilar traits in position-dependent cell autonomy of apoptosis\nhave also been found in the clones mutant for the Dpp recep-\ntor Thick veins (Tkv) (3).\nss-induced apoptosis is dependent on Dll that is responsible\nfor homeotic transformation from wing to leg or antenna. To\nelucidate the role of Dll in ss-induced apoptosis, we placed the\nss-overexpressing clones in a strong Dll mutant Dll5/Dll9 back-\nground. As shown in Fig. 3D, the activation of both JNK and\ncaspase-3 was strongly suppressed by this Dll mutation in the\nwing blade region. Activation was also suppressed in the hinge\nregion, although ss did not induce Dll expression in this region\n(Fig. 2D). A possible explanation is that broader expression of\nDll in the hinge region at an earlier stage of development (data\nnot shown) is required for transformation by ss in later stages.\nThe Dll mutation also suppressed the ss-induced adult wing\nphenotype (Fig. 3E). To test whether Dll overexpression is also\nable to lead to apoptosis, we next generated Dll-overexpressing\nclones in the wing. As shown in Fig. 3F, JNK and caspase-3\nactivation are observed cell autonomously in the Dll-overex-\npressing clones in the wing blade region. Other areas show\nnonautonomy. The wing notching phenotype induced by Dll\noverexpression is found to be similar to that induced by ss\noverexpression (Fig. 3G). Thus, ss-induced apoptosis is likely\nmediated by a pathway that is dependent upon Dll. A discrep-\nancy existed in that the normal expression of Dll in the wing did\nnot induce leg identity or apoptosis. However, the elevated\nexpression of Dll in the wing can induce some leg structure\n(21). Thus, the level of Dll expression, rather than its timing or\nplace, may be crucial for determining the fate as wing or leg.\nCoincidence of round shape and apoptotic response. Apo-\nptosis in each of these regions was dependent on Hemipterous\n(Hep, an activator of JNK [20]), DIAP1, Head involution de-\nfective (Hid; a proapoptotic protein [23]) and Tgo (Fig. 4B to\nF). DIAP1 is known to degrade caspase through its ubiquitin-\nprotein ligase activity. Hid can bind to DIAP1 to stimulate\nautoubiquitination of DIAP1, which prevents the degradation\nof caspase (58). Interestingly, the ss-overexpressing clones in\nhep or hid mutant or in DIAP1 overproducer backgrounds\nretained a round shape, as seen with ss-overexpressing clones\nin the wild-type background (Fig. 4A to E). In contrast, the\nss-overexpressing clones in the tgo-RNAi background lost their\nround shape and invaded the surrounding normal cells (Fig.\n4F). These results indicate that hep and hid only affect the\napoptosis pathway, whereas tgo mediated events more up-\nstream, such as the cell fate change induced by ss. Ectopic\nexpression of ss may make a difference in cell afﬁnity between\nthe clones and surrounding normal cells, which leads to a\nround shape of the clones and subsequent apoptotic response.\nConsistently, activation of JNK pathway by constitutively\nactive Hep leads to hid expression (Fig. 4G). Besides, the\nss-overexpressing clone induces hid expression autonomously\nin the wing blade region but nonautonomously in the wing\nhinge region (Fig. 4H).\nEffect of ss overexpression on the intensity of Dpp and Wg\nsignaling. The JNK-dependent cell death on either side of the\nclone boundary is quite similar to what has been referred to as\n“morphogenetic apoptosis,” through which discontinuities in\nthe Dpp and Wg morphogen activity gradients are corrected\n(2). We next examined the activities of certain morphogen\nsignals in the vicinity of the ss-overexpressing clones. With\nregard to Dpp signaling, we can ﬁnd a slight reduction in the\nexpression of the lacZ enhancer trap of optomotor-blind (omb),\na target gene of the Dpp signal (24) (see Fig. S1A and B in\nthe supplemental material). However, the reduction is always\nfound around the center of the clones and does not coincide\nwith the outline of the clones. We also see a similarly moderate\nalteration of the level of phospho-Mad, an active form of the\nDpp signal transducer “Mothers against dpp” (Mad) (48, 52,\n57) (see Fig. S1A and B in the supplemental material). In the\ncase of Wg signaling, we examined expression of the fz3-lacZ\nenhancer trap (51, 55), a target gene of the Wg signal, and the\nlevel of Arm (46, 54), a Wg signal transducer that is known to\naccumulate in response to the Wg signal (25, 47). We observed\na reduction of fz3-lacZ expression interior to the clone bound-\nary of the ss-overexpressing clones (see Fig. S1C, C, and D to\nD in the supplemental material). However, in the most central\ncells, cytoplasmic accumulation of Arm was conversely ob-\nserved (see Fig. S1C, C, D, and D in the supplemental ma-\nterial). These results suggested that the level of Wg signal is\nthe ss-overexpressing\nnot uniformly received throughout\nclones and is not directly regulated by Ss. Similar responses\nwere observed in clones both in the wing blade and in the hinge\nregions, as seen in Fig. S1C in the supplemental material.\nTaken together, these results indicate that discontinuities in\nthe strength of Dpp and Wg signal reception does not precisely\ncoincide with the boundary of the ss-overexpressing clones,\nsuggesting that other factors contribute to the nonautonomous\nJNK activation in this case.\nPosition-dependent variance in the autonomy of apoptosis is\nfound also in clones ectopically expressing homeotic genes. We\nnext examined whether the position-dependent variance in cell\nautonomy of apoptosis is speciﬁc for ss-induced homeosis or a\ngeneral trait exhibited by ectopic expression of diverse ho-\nmeotic genes. The identity of each body segment is deﬁned by\na series of homeotic genes belonging to ANTP-C and BX-C.\nHowever, the Drosophila wing requires no input from ANTP-C\nand BX-C during its development (11). Instead, these ho-\nmeotic genes repress wing development in all segments, except\nfor the mid-thorax, where the wing normally develops. In order\nto produce a misspeciﬁed fate in wing cells, we expressed\nvarious homeotic genes in the wing discs. When labial (lab),\nproboscipedia (pb), Antennapedia (Antp), Ultrabithorax (Ubx),\nabdominal-A (abd-A), or Abdominal-B (Abd-B) were expressed\nby using sd-GAL4 driver, the adult wings showed abnormal\nvein patterns and incision of the wing margin (Fig. 5A to F). In\nthe cases of Ubx or abd-A expression, wing incision was less\nfrequently observed. In these cases, the wing has been shown\nto transform to an organ similar to a haltere (22). When mo-\nsaic clones expressing these homeotic genes are generated,\nJNK activation was found autonomously in the blade region\nbut nonautonomously in the hinge region. All of these re-\nsponses are quite similar to the results of ss overexpression, as\nhas been described above. Thus, the position-dependent vari-\nance in cell autonomy of apoptosis may be a general phenom-\nenon accompanying homeotic transformation.\nCharacterization of the novel gene ﬁli that encodes a LRR\nfamily transmembrane protein. Recently, clones with sal-in-\nduced misspeciﬁed fate were reported to be recognized by the\nsurrounding normal cells through the LRR family transmem-\nbrane proteins, Capricious (Caps) and Tartan (Trn), to prevent\napoptosis (38). The expression pattern of both caps and trn are\ncomplementary to the pattern of sal expression in the late third\ninstar wing (38) (Fig. 6C). Thus, we investigated the role of a\nnewly identiﬁed LRR family protein, Fili, which was expressed\nin the wing in a nearly complementary pattern as that of Dll\n(Fig. 6A and B), in regulating apoptosis in ss-overexpressing\nclones.\nThe ﬂy strain ﬁli-lacZ was isolated by a screen of enhancer\ntrap lines that displayed a previously unknown expression pat-\ntern. The transposon P-element carrying the lacZ gene was\ninserted at 56 relative to the presumptive transcription start\nsite, TTAGTT (Fig. 7A). The deduced protein, Fili, was com-\nposed of 738 amino acid residues and had a single transmem-\nbrane domain and 14 LRR domains (Fig. 7B). These features\nare common to Caps and Trn, which share 55 and 56% ho-\nmologies with Fili in the LRR domains, respectively (Fig. 7C\nand D). The homologies also extend to the amino- and car-\nboxyl-terminus domains (Fig. 7E and F), suggesting that Fili\nplays a role that is especially related to Caps and Trn among\nnumerous members of LRR family with a variety of functions.\nA mutant, 102, was isolated by imprecise excision of the\nP-element in ﬁli-lacZ. It lacks 534 bp, including a part of the\nﬁrst exon, and is lethal at late embryonic stages. The main\ncause of this lethality is unknown. In the wing disc, ﬁli exhibited\na pattern of gene expression that was nearly complementary to\nthat of Dll (Fig. 6A and B). In addition, ﬁli expression was\ncomplementary to Dll expression in antenna discs (Fig. 6A\nand B). In contrast, in the leg disc, ﬁli seemed to partially\noverlap with Dll (Fig. 6A and B). However, we found that\nmost of the cells expressing the respective genes were in dif-\nferent layers of the leg disc folding.\nss-induced apoptosis can be controlled by Fili. If ﬁli is in-\nvolved in the control of apoptosis in ss-overexpressing clone, its\nexpression should be downregulated by ss. As expected, ﬁli\nexpression is repressed in the ss-overexpressing clones (Fig.\n8A). Interestingly, in contrast, expression of caps (Fig. 8A)\nand trn (data not shown) is enhanced by ss. Similar responses\nin ﬁli and caps can also be observed in Dll-overexpressing\nclones (Fig. 8B to B).\nWhen ss was induced together with ﬁli, JNK and caspase-3\nactivation was greatly reduced (Fig. 3A and 9A to A). Resid-\nual JNK and caspase-3 activation was observed only in the\nmost peripheral hinge region, where ﬁli is not expressed in the\nnormal wing. Similarly, in the central wing region, where ﬁli is\nnot expressed in the normal wing, caspase-3 activation was not\nsuppressed by ﬁli coexpression. As a control experiment, we\nobserved that coexpression of both caps and ss did not suppress\nthe ss-induced apoptotic response (Fig. 9B and B). Consis-\ntently, ss overexpression does not repress but rather elevates\nthe caps expression (Fig. 8A). Thus, we conclude that ﬁli\nexpression is sufﬁcient for survival of ss-overexpressing cells in\nthe wing region where endogenous ﬁli is expressed, probably\nthrough recognition by surrounding normal cells transmitting a\nsurvival signal, as shown for Caps and Trn (38).\nWe further carried out a control experiment to test the effect\nof ﬁli on sal-induced apoptosis because sal-induced apoptosis\nwas reported to be suppressed by coexpression of caps or trn.\nAt ﬁrst, we observed that the sal-overexpressing clones dis-\nplayed a nonautonomous activation of JNK in the wing region\nwhere sal is not normally expressed (Fig. 9C to C). This\nnonautonomy was quite similar to the morphogenetic apopto-\nsis and ss-induced apoptosis, suggesting that the regulation of\nall of these apoptotic responses may share similar molecular\nmechanism(s). When sal was induced together with ﬁli, JNK,\nand caspase-3 activation was not affected signiﬁcantly (Fig. 9D\nto D). Thus, these results indicate that ﬁli does not suppress\nall cases of nonautonomously induced apoptosis.\nShape and apoptosis of clones mutant for or overexpressing\nﬁli. Next, we examined the relationship of the clone shape and\napoptosis in a simple ﬁli mutant or in its overexpression. The\nﬁli mutant clones showed an irregular shape irrespective of\ntheir position in the wing disc (Fig. 10A) and did not display\nany apoptotic signs. These results are not always discrepant\nfrom the other results and are similar to observations in mu-\ntants for caps or trn alone (38). The other LRR family proteins\nmight function with Fili in a redundant manner in the gener-\nation of afﬁnity with surrounding normal cells. In contrast to\nthe ﬁli mutant, the ﬁli-overexpressing clones showed a smooth\noutline (Fig. 10B). The smooth outlines were found even\nwhere endogenous ﬁli was expressed, which may be due to the\nimmense amount of its expression by using UAS/GAL4 system.\nHowever, the clone in the central region where endogenous ﬁli\nwas not expressed exhibited the most rounded shape (Fig.\n10B). Interestingly, unlike the case of caps and trn, the over-\nexpression of ﬁli alone induced apoptosis in a nonautonomous\nmanner. When UAS-ﬁli was induced by ptc-GAL4 (Fig. 10D),\nnonautonomous activation of JNK was apparent in the cells\naround the posterior edge of ptc expression but not in the cells\naround the anterior edge (Fig. 10D to D). The posterior edge\nis known to create a sharp discontinuity of ptc expression\nlevels, whereas the anterior edge does not (5). More interest-\ningly, the area where JNK was activated did not overlap the\narea where endogenous ﬁli was expressed. As a result, the\nareas with JNK activation were divided into three regions\n(arrowheads in Fig. 10D). Also, forced expression of ﬁli in the\ncentral wing blade in the adult wing generated a severely\nnotched phenotype by apoptosis (Fig. 10C). These data\nstrongly suggest that the occurrence of a large discontinuity in\nthe ﬁli expression level activates JNK nonautonomously.\nDISCUSSION\nEctopic ss induces a homeosis-dependent apoptosis that\nmay cause a rare frequency of wing-to-leg transdetermination.\nHomeosis is a naturally occurring rare phenomenon with for-\nmation of a local body part having characteristics that are\nnormally found in a related part at another location in the\nbody. This phenomenon had been studied exclusively in Dro-\nsophila melanogaster but can also be found in a variety of\ninsects and other organisms (53). Although homeosis is rarely\nfound in wild-type ﬂies, transdetermination, a similar phenom-\nenon, can be easily induced by in vivo culture and subsequent\ntransplantation of imaginal discs between different individuals\nor by directly manipulating functions of various master and/or\nhomeotic genes. Although such developmentally abnormal\ncells occurring in homeosis might a priori be expected to be\nremoved during development to guarantee normal morpho-\ngenesis, apoptosis has not been previously observed to accom-\npany homeosis. Our results suggest that the cells undergoing\nhomeosis do in fact induce apoptotic responses to help prevent\nthem from developing abnormal structures. The rare fre-\nquency in wing-to-leg transdetermination, which was observed\nabout 30 years ago (30), may be derived partly from the high\nfrequency of apoptosis associated with this particular transfor-\nmation.\nss-induced autonomous apoptosis is related to cell compe-\ntition. Cell competition is a phenomenon whereby cell clones\nwith a slow growth rate are eliminated during development.\nThe autonomous cell death in cell competition has been dem-\nonstrated to be caused by an impaired reception of extracel-\nlular survival factors. As representative examples, clones with\ncells heterozygous for the M(2)60E mutation show an auton-\nomous death as a result of reduced Dpp reception (40). Thus,\nthe autonomous death in ss-overexpressing cells in the wing\nblade might be a type of cell competition response. In this case,\nhowever, the extracellular factor that the ss-overexpressing\ncells fail to receive is not likely to be Dpp, since we observed\nthat the Dpp signaling level is not strongly affected in the\nss-overexpressing cells (see Fig. S1A and B in the supplemental\nmaterial). Also, ss-overexpressing cells can be removed from\nthe lateral region of the wing blade where the level of Dpp\nsignaling is normally low. In contrast, cells heterozygous for\nM(2)60E can survive in this area due to the loss of competition\nfor Dpp (40). Therefore, Wg or other extracellular survival\nfactors might be affecting the survival of ss-overexpressing\ncells, as discussed below.\nPosition-dependent variance in cell autonomy is found in\nthe apoptotic response to ss overexpression. We have not yet\naddressed the question of why the variance in cell autonomy of\nss-induced apoptosis is related to dorsoventral patterning or\nblade/hinge subdomains. However, the variance suggests an\ninteresting model in which signals that normally regulate dor-\nsoventral appendage patterning or blade/hinge subdomains\nalso directly affect the apoptotic response or speciﬁcation of\norgan identity as cryptic mechanisms. Interestingly, ectopic\nexpression of most of the segment-specifying homeotic genes\ncommonly shows a similar position-dependent variance in cell\nautonomy (Fig. 5). In this view, Wg, the morphogen controlling\nthe dorsoventral patterning (42, 60), or Vg, the selector of wing\nsubdomain (31, 34), may be involved in the regulation of po-\nsition-dependent variance in the cell autonomy of apoptosis.\nHowever, deﬁnitively demonstrating the involvement of Wg or\nVg is difﬁcult because changing Wg signal or Vg activity alone\nalso induces a severe unrelated apoptosis (2). Thus, the simul-\ntaneous manipulation of ss and Wg/Vg activities may lead to a\ncomplicated result that is difﬁcult to interpret.\nImplication of various LRR family transmembrane proteins\nin the regulation of apoptosis to remove misspeciﬁed cells. A\nrecent study concerning the relation of apoptosis in sal-over-\nexpressing cells and LRR family proteins Caps/Trn has pro-\nvided a new insights into how cell survival of misspeciﬁed cells\nis controlled (38). Cells that overexpress sal lack caps/trn ex-\npression and are removed from the area where sal is not\nnormally expressed. However, cells expressing both sal and\ncaps can survive. In order to control cell survival in the wing,\nCaps and Trn presumably provide position-dependent recog-\nnition cues along the anteroposterior axis, whereas Fili appears\nvation of mammalian Ahr is known to affect apoptosis, cell\nproliferation, differentiation, and morphogenesis in various tis-\nsues (50). Polycyclic aromatic hydrocarbon, a ligand for Ahr,\ninduced apoptosis accompanied by the activation of JNK (33).\nIt should also be noted that dioxin-induced responses such as\nP450 expression in the liver are not observed in all cells ex-\nposed to dioxin but are observed in a subset of centrilobular\ncells with the highest sensitivity to dioxin (4). Thus, the mosaic\nactivation of Ss and subsequent JNK activation in surrounding\ncells in Drosophila induce circumstances similar to that of liver\ncells exposed to dioxin. With regard to morphogenesis, the\ntype of morphogenetic signals that are modiﬁed or distorted\nduring dioxin action is unknown, despite many examples of\ndioxin-induced mismorphogenesis that have been reported.\nTherefore, our studies on Ss in Drosophila may provide at least\nin part, a common molecular and cellular basis for understand-\ning Ahr-induced apoptosis and mismorphogenesis. It could\ninvolve homeotic transformation of organ identity, recognition\nby surrounding cells through LRR protein, and apoptosis-me-\ndiated large deletion of tissues that can cause a malformation\nduring organogenesis. Interestingly, all of the genes involved in\nthis sequential process are evolutionarily conserved in mam-\nmals. Similar and detailed analysis in mammalian system will\nresolve these issues.\nWe thank R. Barrio, S. Campbell, S. T. Crews, A. Garcı´a-Bellido,\nB. A. Hay, Y. Hiromi, Y. H. Inoue, K. Irvine, W. Janning, E. Martı´n-\nBlanco, A. Martinez-Arias, A. Nose, S. Noselli, C. Rauskolb, M.\nShinza-Kameda, G. Struhl, K. Takahashi, and D. Yamamoto, Bloom-\nington Stock Center, and GETDB (GAL4-Enhancer Trap Data Base)\nin the National Institute of Genetics in Japan, for the ﬂy strains; R.\nBarrio, G. Campbell, S. B. Carroll, D. Duncan, A. Nose, M. Shinza-\nKameda, and P. ten Dijke, Developmental Studies Hybridoma Bank,\nIowa University, and Idun Pharmaceutical, Inc., for antibodies; J. Kim\nand Y. Lee for technical advice; and Y. Aoki and O. Habara for\ntechnical assistance.\nto provide such cues to neighboring cells along the dorsoven-\ntral axis. These results raise the possibility that cells can rec-\nognize the type of neighboring cells using distinct LRR family\nmembers, depending on the position of the cells in the primor-\ndial tissues. The Drosophila genome contains at least 10 mem-\nbers of LRR family, most of which have not been examined for\nfunction (Flybase).\nFunctional relationship between Drosophila Ss and mamma-\nlian Ahr. In the present study, several cellular responses to ss\noverexpression have been described. Clonal overexpression of\nss induces a sequential response composed of an induction of\nleg- or antenna-speciﬁc genes, autonomous homeosis, round-\ning, and autonomous or nonautonomous apoptosis. The acti-\n17. Duncan, D. M., E. A. Burgess, and I. Duncan. 1998. Control of distal\nantennal identity and tarsal development in Drosophila by spineless-aristape-\ndia, a homolog of the mammalian dioxin receptor. Genes Dev. 12:1290–1303.\n18. Emmons, R. B., D. Duncan, P. A. Estes, P. Kiefel, J. T. Mosher, M. Son-\nnenfeld, M. P. Ward, I. Duncan, and S. T. Crews. 1999. The Spineless-\nAristapedia and Tango bHLH-PAS proteins interact to control antennal and\ntarsal development in Drosophila. Development 126:3937–3945.\n31. Kim, J., A. Sebring, J. J. Esch, M. E. Kraus, K. Vorwerk, J. Magee, and S. B.\nCarroll. 1996. Integration of positional signals and regulation of wing for-\nmation and identity by Drosophila vestigial gene. Nature 382:133–138.\n32. Lecuit, T., W. J. Brook, M. Ng, M. Calleja, H. Sun, and S. M. Cohen. 1996.\nTwo distinct mechanisms for long-range patterning by Decapentaplegic in\nthe Drosophila wing. Nature 381:387–393.",
         "T. Adachi-Yamada,Toshiyuki Harumoto,Kayoko Sakurai,R. Ueda,K. Saigo,M. O’Connor,H. Nakato",
         null
        ],
        [
         "37",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "macromolecules in routine circulation. The system is not foolproof, however, and in certain\ndiseases, the immune system responds to self-derived antigens, perhaps because their location,\nabundance, modified form or other features appear unfamiliar. Cancer patients often produce\nresponses to self-proteins that are expressed by their tumors, called tumor antigens, most of\nwhich are altered in some form that renders them immunogenic. These proteins may be unique\nto cancer and germ cells (the “cancer-testis” antigens), found only in specific tumors (prostate-\nspecific antigen)1 or in most tumors (telomerase)2.",
         "They may be mutated (p53)3, misfolded4, overexpressed (NY-ESO-1) 5, aberrantly\ndegraded6 or aberrantly glycosylated (MUC-1)7. The magnitude of the immune response to\ncancer, in general, is lower than the immune response to infectious agents and the potential\nnumber of tumor antigens encompasses the entire tumor proteome in all its variations. At\npresent, we have a limited understanding of the breadth, extent, impact, and dynamic variation\nof the immune response to cancer (the “cancer immunome”). Identifying the specific targets\nof B- and T- lymphocyte immunity to cancer may 1) identify potential biomarkers for cancer\ndiagnosis, classification, and monitoring of response, 2) determine the impact of immune\nregulation on cancer progression, and 3) identify potential antigens and mechanisms for\nimmunotherapy development.\nThe natural immune response is achieved through a tightly regulated, yet flexible network\nincluding antibodies, antigen presenting cells, T lymphocytes, cytokines, chemokines,\nregulatory systems, as well as microenvironmental signals (Figure 1). Of these responses, the\ntargeted responses to protein (and carbohydrate) antigens relies on the development of\nantibodies and/or T lymphocytes to target epitopes. T lymphocytes can respond to antigens\nderived from within cells and without. They primarily recognize short peptides (8-22mer)\nderived from intracellular proteins (i.e., viral antigens) bound to self-MHC molecules for\npresentation to CD8+ T lymphocytes. Exogenous antigens are endocytosed, degraded, and\npresented to CD4+ lymphocytes (Figure 2). Antibody responses increase antigen presentation\nby enhancing uptake through the Fcγ receptors on antigen presenting cells. As a result, antibody\ntargets may contain epitopes that are also recognized by T lymphocytes. This has formed the\nbasis for using antibody responses to identify T cell antigens for immunotherapy.\nLessons from Autoimmune Diseases\nSpontaneous autoantibodies were first identified in a series of clinical disorders in which the\npatients' immune systems mount a vigorous response to self antigens, in some cases leading\nto debilitating symptoms. Systemic lupus, myasthenia gravis, rheumatoid arthritis and others\nall involve this process, called autoimmunity. In these illnesses, the titers of autoantibodies\noften track with the severity of the illness and thus have long been used as serum biomarkers\n(reviewed in8, 9).\nAlthough post hoc arguments can usually be constructed explaining why proven autoantigens\nmay have appeared unfamiliar to the immune system; in general, we do not understand enough\nabout the characteristics that determine whether a protein will act as an autoantigen to predict\nthem a priori. To date, only one or two percent of proteins have been identified as autoantibody\ntargets10. In autoimmune diseases, the autoantigens identified, such as Ro/SSA, La/SSB, Sm,\nRNP, Scl-70 and Jo-1 are primarily intracellular antigens that function in large complexes with\nnucleic acids involved in protein synthesis8, and are thought to arise as a result of\noverexpression, apoptosis, reduced degradation, or similarity to cross-reacting foreign antigens\n(mimicry). It is often not clear whether these autoantibodies have any direct role in disease\npathophysiology. In contrast, antibodies directed at membrane proteins, such as the target\nantigens of hemolytic anemia or the acetocholine receptor in myasthenia gravis may contribute\nto the disease process through complement activation, antibody-dependent cytotoxicity, or\ninterference with receptor/ligand interactions. Certain structural motifs enhance antigenicity,\nsuch as carbohydrates, multivalency, epitope repetition, and coiled-coils11. Some of these\nantigens are linked to degradation during apoptosis12, or are targets of granzyme B cleavage\nduring T lymphocyte-induced cytolysis13, 14. Further elucidation of the mechanisms\nunderlying autoantibody generation will assist in predicting and identifying target antigens.\np53 as a Model Tumor Autoantigen\nPerhaps the most well-studied autoantigen in cancer is the p53 protein (reviewed in 3). In 1979,\nDeLeo et al15 demonstrated that autoantibodies to some tumor cells in mice were directed\nagainst the p53 protein. Subsequent studies have demonstrated that the half-life of mutated p53\n(several hours) was markedly increased compared to wild-type (several minutes) resulting in\naccumulation in the cell nucleus. p53 autoantibodies are dependent on the type of p53\nmutation16-18. Notably, the immunogenic epitopes have been mapped primarily to both the\nN- and C-terminal portions of the molecule, which are heavily glycosylated, but not to the\ncentral portion of the molecule, which harbors the mutations, suggesting that the accumulation\nof protein, rather than the mutations per se, results in autoantibody generation. Multiple assays\nusing recombinant antigens and p53-derived peptides have shown that some antigenic\ndeterminants are dependent on full-length antigen, conformation, or phosphorylation3, 19.\nThe detection of p53 autoantibodies in serum correlates strongly with cancer (p<10-4). p53\nautoantibodies have been studied in over 9489 patients with a wide variety of tumors3. Despite\nthe strong specificity of the response, only 20-40% of patients with cancers harboring p53\nmissense mutations will have p53 antibodies in their sera3. Therefore, there are additional\nunknown factors beyond antigen accumulation that impact the development of autoantibodies.\nThis illustrates one of the major challenges facing the use of single autoantibody tests for cancer\ndetection. Although the appearance of such antibodies is highly suggestive of cancer, many\npatients will be missed because they do not respond to p53. p53 autoantibodies have been\ndetected in body fluids, such as ascites, pleural effusions, and saliva, and these correlate with\nserum autoantibodies19-21. Interestingly, gliomas, which are associated with p53 mutations,\nare not associated with the development of p53 antibodies22, 23, suggesting either that\ndevelopment of tumors in the CNS leads to immunologic privilege, or the use of steroids in\nthis patient population could dampen the serologic immune response.\nAnti-tumor Effects of Autoantibodies in Cancer\nIt is not known if autoantibodies reflect underlying immunosurveillance of cancer or have an\nimpact on the clinical outcome of the disease. For example, multiple studies have attempted\nto correlate p53 autoantibodies with prognosis showing mixed results3. As the development\nof B cell immunity often depends on concordant T cell immunity, autoantibody identification\ncan lead to the identification of relevant T cell antigens24-26. In patients who have undergone\nautologous tumor vaccination, certain autoantibodies correlate with tumor response to\ntreatment25, 27. In bone marrow transplantation, the development of autoantibodies may\nidentify minor histocompatibility antigens associated with graft versus host disease28 or tumor\nantigens associated with the graft-vs-leukemia (GVL) effects of donor lymphocyte\ninfusion29, 30. In murine tumor models, it is the coordinated activities of CD4+ and CD8+ T\ncells, CD1d-restricted invariant NKT cells, and antibodies that accomplish protective\nimmunity 31, 32 33 34.\nThe serologic identification of tumor antigens has directly led to the development of a number\nof cancer vaccines currently in clinical trials. The proteinase-3 antigen, first identified as the\ntarget antigen of the ANCA antibody assay for Wegener's granulomatosis, is being tested as a\ntarget for vaccination in CML35. NY-ESO-1 has been used as an immunogen in multiple\nclinical trials36. Antibody response to tumor vaccines have been shown to correlate with\nimproved disease-free and overall survival in stage II melanoma patients vaccinated with a\npolyvalent vaccine37. Melanoma patients vaccinated with autologous tumor cells may develop\nhighly-individualized antibody responses38. The identification of these autoantibodies has\ndirectly led to mapping of CD4+ and CD8+ T lymphocyte epitopes. The clinical benefit of\nantigen-specific tumor vaccination strategies is just beginning to emerge, with the recent\ndemonstration of a survival benefit of a dendritic cell-based vaccine in hormone-refractory\nprostate cancer39.\nAutoantibodies as Biomarkers of Cancer\nIn 2003, it is estimated that 1,334,100 people in the US were newly diagnosed with cancer,\nwith an estimated 556,500 patients dying of the disease40. In this setting, there is intense effort\nin the search for biomarkers that can predict disease, identify biologic subtypes of disease,\ntrack response to therapeutic interventions, predict side effect profiles, and monitor for disease\nprogression and recurrence41.\nThere are a limited number of serum protein biomarkers that are widely used in clinical\noncology for prognosis and treatment monitoring, and their use in early cancer diagnosis has\nbeen hampered by false-positive rates in the normal population. The identification and\ndevelopment of these biomarkers took decades of research and required large prospective trials.\nCarcinoembryonic antigen (CEA), widely used for the monitoring of adenocarcinomas, was\noriginally identified from tumor lysates because of its immunogenicity in rabbits42. The\nsubsequent identification of the prostate specific antigen (PSA)43 and the ovarian cancer\nbiomarker CA-12544, among others, have demonstrated both the validity and limitations of\nserum biomarkers in the diagnosis of cancer. Although PSA is now routinely used for screening\nhealthy populations for prostate cancer diagnosis, most serum biomarkers are used for\nmonitoring treatment or for screening highly selected high-risk populations. More recent\ndevelopments in serum proteomics have been reviewed in this issue and elsewhere45.\nCompared with other polypeptides, autoantibodies have many appealing features as\nbiomarkers. First, although tumor antigens may circulate only briefly or in low concentration,\nperhaps due to transient shedding by tumors, rapid degradation in the serum, or rapid clearance,\nthe corresponding antibody response is likely to be persistent. Second, antibodies are highly\nstable in serum samples and are not subject to the types of proteolysis that are commonly\nobserved for other polypeptides (discussed elsewhere in this issue), making sample handling\nmuch easier. Third, the t1/2 of antibodies in circulation is >7 days, so hourly or daily fluctuations\nare expected to be minimal, simplifying sample collection. Finally, the biochemical properties\nof antibodies are well understood and there are many available reagents for their detection,\nsimplifying assay development. Given these advantages, the challenge that remains is\ndemonstrating that antibodies can be sufficiently informative to reliably detect cancer.\nEarly Disease Diagnosis—Very limited data are available on the use of autoantibodies to\nreveal early disease. Antibodies have been detected as early as several years before the clinical\nappearance of cancer46, 47 and in patients with preneoplastic disease48. Anecdotal studies\nhave detected p53 autoantibodies in heavy smokers prior to the diagnosis of lung cancer47.\nAnother study showed p53 autoantibodies in individuals exposed to vinyl chloride, a risk factor\nfor the development of angiosarcoma of the liver46. The p53 autoantibodies predated the\ndiagnosis of angiosarcoma by several years. Although encouraging, these few studies are not\nyet sufficient to endorse the value of antibodies in the early detection setting.\nMonitoring treatment response and predicting recurrence—Antibodies to tumor\nantigens have been detected in early stages of disease and fluctuate with tumor response. In\npatients with both stage III and stage IV neuroblastoma, 10% have evidence of antibodies to\nthe NY-ESO-1 antigen, but the antibody is not seen in the sera of patients in clinical remission\nor in earlier stages of disease49. In melanoma, antibodies to TA90 antigen have been detected\nin only 12% of patients with 1-2mm primary, node negative melanoma who subsequently\nrelapsed, but 62% of case-controlled patients who did not50.\nAntibodies to the her2/neu tumor antigen have been detected in the sera of 20% of patients\nwith her2+ early-stage breast cancer51 but only 7% of late-stage breast and ovarian cancer\npatients52. Some of the early-stage patients had titers>1:5000, suggesting a strong, possibly\nprotective immune response. In patients with both stage III and stage IV neuroblastoma, 10%\nhave evidence of antibodies to the NY-ESO-1 antigen, but the antibody is not seen in the sera\nof patients in clinical remission or in earlier stages of disease49.\nThe titer of antibodies to NY-ESO-1 have been shown to correlate with disease progression,\nas well as with disease response/resection53. The titer of anti-p53 antibodies have been shown\nto increase in a subset of patients with early-stage breast cancer prior to disease relapse54, and\ndisappearance of anti-p53 antibodies has been observed in 27 patients after resection of\ncolorectal cancer55. Tumor antigen/antibody immune complexes have been detected up to 19\nmonths prior to the development of clinical relapse and correlate with survival in patients with\nearly-stage melanoma56.\nMethods for assessing autoantibodies\nThe development process for autoantibodies as biomarkers is outlined in Figure 3. Historically,\nthis cycle begins with identifying antigens that are detected by the sera of cancer patients.\nAntigens must then be selected and tested by comparing sera from both patients and healthy\ndonors to determine if the antigens are “informative”, i.e., responses are limited to patients. As\npart of this process, the sensitivity and specificity of the antigens should be determined.\nPromising antigens are then tested on a set of samples separate from the training set to validate\ntheir usefulness. Ultimately prospective trials will establish their value as biomarkers. The\ncycle is completed by using the sera from patients not detected by the test to screen for new\nantigens. In some of the newer proteomic approaches, two of these steps may combine into a\nsingle step.\nIdentification of novel tumor antigens\nSEREX: A variety of techniques have been developed that use patient sera as probes against\ncandidate antigens derived from tumor cells and tumor cell lysates to find novel autoantigens.\nProminent among these, serologic expression cloning (SEREX) was developed ten years ago\n(Figure 4). This method uses patient sera to probe blotted phage expression libraries derived\nfrom tumor cells and has resulted in the identification of over 2000 autoantigens recognized\nby patient sera5, 57-79. Through a database established by the SEREX collaborative group,\nsequences from over 1390 genes have been deposited 80 and\nhttp://www2.licr.org/CancerImmunomeDB/. As SEREX relies on immunoblotting, these\nantigens are limited to linear epitopes and those gene products that can be expressed in bacteria.\nAs with the autoimmune targets, most of the antigens identified with SEREX are nuclear or\nintracellular antigens, many of which are upregulated, mutated, or specifically expressed in\ntumors. However, altered degradative pathways, such as sensitivity to granzyme B cleavage,\nhas been shown to have a marked impact on the immunogenicity of a hepatocellular tumor\nantigen6. In addition, frameshifts and alternative reading frames, can result in autoantibodies.\nUsing colon cancer sera, the antigen ADO34 was identified with a frameshift insertion81, and\nCDX2 had a frameshift mutation82. Antibodies to the OGFr protein can recognize an\nalternative-reading frame of the molecule79.\nThe SEREX method identifies autoantigens whether or not they are informative. At present\nonly a handful of antigens have undergone comparison testing, in part because of the technical\nchallenges associated with historical assays for validating autoantigens.\nPhage display: Combinatorial phage display is a new and attractive approach for the\nidentification of tumor antigens. By expressing antigens as fusions with phage proteins,\nantigens can be detected more rapidly and with less serum. Rather than immunoblotting as\nwith SEREX, phage display relies on successive rounds of immunoprecipitation of phage\nlibraries using patient serum. This powerful approach has been used in the field of\nrheumatology to identify autoantigens in multiple sclerosis, lupus, rheumatoid arthritis and\nothers83-85. For tumor antigens, one study showed that only 4 of 13 antigens identified with\nprostate cancer sera had already been identified by SEREX technologies, demonstrating that\nnovel tumor antigens may be identified with different screening approaches 86. Phage display\nhas been successfully used to identify tumor autoantigens in cancer86-89. However, antigens\nexpressed by phage display may not be in native conformation, do not have mammalian post-\ntranslational modifications, and each positive phage clone must be individually sequenced.\nYeast display systems are also being investigated90.\nValidating autoantibodies—To be useful as biomarkers, the tumor antigens must\ndistinguish between individuals with and without cancer. Thus once a candidate autoantigen\nis identified, comparisons must be made regarding the responses of individuals from both\ngroups. The measurement of an autoantibody response is most often accomplished using the\nenzyme linked immunosorbent assay (ELISA), in which purified protein (the autoantigen) is\nimmobilized in the wells of a microtiter dish and exposed to serum samples (Fig. 4). After\nadequate washing, any bound antibody is revealed using standard anti-human antibody\nreagents linked to enzymatic markers. If the response is very strong, it may be useful to perform\nserial dilutions of the serum to determine the titer of the antibody. By comparing multiple\npatient and normal donor samples it is possible to determine the frequency of response. For\nexample, antibodies to dsDNA are present in only 40-60% of SLE patients, but antibodies to\nhistones are present in >95% of patients with drug-induced SLE. To compute the sensitivity\nand specificity of the test, a threshold value must be set, such that the test is considered positive\nif the antibody response achieves that value or above. Typically, this threshold is either 2 or 3\nstandard deviations above the average response in the normal population.\nThe development of clinical ELISAs require the use of recombinant proteins or peptides, which\nmay be complicated by batch-to-batch variations, loss of conformational epitopes or lack of\nmammalian post-translational processing91, 92. Antigenic epitopes from proteins such as Ro,\nLa, SmB, SmD, Scl-70, can be linear epitopes of 10-22 amino acids conformational, or cryptic.\nFurthermore, epitopes may be naturally post-translationally modified by multiple means,\nincluding phosphorylation, glycosylation, acetylation, and methylation (SmD1, SmD3,\nfibrillarin, nucleolin)93, 94, all of which must be considered prior to the development of a\nclinical diagnostic assay. Of the many candidate antigens identified so far, only a handful have\ngone through the validation process to determine their sensitivity and specificity as cancer\npredictors, in part because of the challenges associated with setting up ELISAs.\nProteomic Approaches for the Identification of Tumor Antigens\nThe Power of Multiplexing—With all of their potential advantages, the Achilles heel of\nautoantibodies as biomarkers is their sensitivity, i.e., the fraction of true positives that have a\npositive test. In large part, this may reflect the nature of cancer. Unlike infections in which the\nvast majority of patients respond to the same immunodominant antigens, even cancers of the\nsame type represent a mix of different biological subtypes. Thus, patients are likely to mount\nimmune responses to different tumor antigens, and no single antigen is likely to detect all\ncancers. Typically, only 15-20% of patients demonstrate a response to any given antigen.\nHowever, proteomics may hold the key to success because it provides the means to multiplex.\nBy linking the responses to several antigens together, the sensitivity and specificity of the test\nincreases considerably, presumably because the chance that a patient will respond to at least\none of the antigens is increased76, 95-102. Koziol et al demonstrated that a panel of just seven\ntumor-associated antigens (myc, cyclin B1, p62, IMP-1, Koc, p53, and survivin) could be used\nto segregate sera from patients with cancers from healthy donor sera95. In that study, sera from\nbreast, colorectal, gastric, hepatocellular, lung, and prostate cancers were distinguished with\nsensitivities from 0.77-0.92 and specificities from 0.85-0.91 using no more than 3 tumor\nantigens for any cancer cohort. Scanlan et. al. studied the serologic responses to 13 defined\ntumor antigens in sera from colon cancer patients. Of these, 46% of patients, but not healthy\ndonors, had antibodies to at least 1 of these antigens103.\nThese results demonstrate the potential power of simultaneously analyzing multiple\nautoantigens. It provides a greater likelihood of detecting and diagnosing the appropriate cancer\npresumably because the sensitivity will improve (increased chance that the patient will have a\nresponse to at least one of the tested antigens) and the specificity improves (positive responses\non multiple antigens increases certainty). By coupling this with the appropriate statistical\nmodeling, it is likely that patterns or weighted schemes of antibody responses, rather than\nindividual responses, will have the greatest utility in clinical assays. The early experiments\nabove were performed by using recursive partitioning. Given the technical challenges inherent\nin that method for large numbers of antigens, newer technologies are needed for high\nthroughput analysis. To this end, recent development of several proteomic technologies have\nbeen adapted for tumor antigen identification and biomarker development. These include\nprobing fractionated tumor cell lysates, phage display, and protein microarrays (See Table 1).\nProbing fractionated tumor cell lysate blots with serum\nIn this approach, tumor cell lysates are fractionated to separate the various protein species and\nblotted onto a membrane or microarray and then probed with patient or control sera. Response\npatterns are then analyzed to differentiate between the two (Fig. 4). This has been used to\ndemonstrate antibodies in the sera of lung cancer patients96, 104. A similar approach was used\nto distinguish sera from prostate cancer patients from healthy donor sera with 98%\naccuracy105. The development of automated separation systems using liquid-based\nchromatography with subsequent microarray spotting of lysate fractions enhances the\nreproducibility and speed of immunogenic fraction identification. Advantages of this approach\ninclude the ability to query a large fraction of the tumor cell proteome and the preservation of\nthe post translational modifications of proteins. Moreover, when this is executed using a\nmicroarray format, only small amounts of serum are required. However, the reproducibility\ndemanded by a clinical assay requires an identified and validated antigen. Thus, as with other\nchromatographic separation systems, the identity of the proteins in the fraction must eventually\nbe determined, presumably using sensitive mass spectrometric analysis. As each lysate fraction\nmay contain many proteins at very different concentrations, of which the minor component\nmay be the immunogen, antigen identification can be difficult.\nProbing Candidate Antigen Arrays\nKnown or predicted tumor antigens may be directly spotted on microarrays and probed with\nhuman sera, with the advantage of reproducibility and more rapid screening of small amounts\nof sera (Figure 4). In this case, the identity of a single protein at each feature on the array is\nknown a priori. This approach has been successfully used to screen autoimmune patient\nsera100, 101. A powerful advantage of this approach is that it offers the ability to screen for\ninformative autoantigens by comparing responses of patients to controls. Because each feature\nof the array represents a single protein, whose identity is known, it is possible to calculate\nsensitivity and specificity values for the response to each candidate antigen during the screening\nphase. This allows the rapid determination of which antigens are informative for cancer\ndetection. Ultimately, focused arrays of tumor antigens previously identified by other means\nmay have the greatest utility for immunodiagnostics.\nTheir theoretical advantages notwithstanding, protein microarrays have still not found\nwidespread use, in part because producing them is challenging. Historically it has required the\nhigh-throughput production and purification of protein, which then must be spotted on the\narrays. Once printed, concerns remain about the shelf life of proteins on the arrays. Recently,\nprogrammable protein microarrays that consist of anchored cDNA's and in situ transcription\nand translation of tagged proteins have been shown to result in highly reproducible protein\narrays without the requirement of heterologous protein expression and purification106. As the\nproteins are synthesized at the time of the assay, shelf life is not an issue and the proteins are\ntranslated with a mammalian reticulocyte lysate, which reproduces the folding and some of the\npost-translational processing of antigenic epitopes. Unlike tumor lysate fractionation, antigen\nidentification is automatic, although epitopes that depend on abnormal processing by tumor\ncells would not be identified, and transmembrane proteins have not yet been tested in this\nsystem. Like most antigen presentation methods (SEREX, ELISA, phage display, etc.), these\nprogrammable arrays require access to cDNAs to express the antigens. Fortunately, nearly all\nknown tumor antigen genes are available because they were identified with methods that\ninvolved cDNA identification. Moreover, libraries of cDNA clones representing most of the\nhuman proteome are increasingly available.\nProteomics and the Identification of T cell Antigens\nIn contrast to the rapid identification of B cell antigens, the identification of T cell antigens\nremains much more difficult. Since T lymphocytes specifically recognize peptides derived\nfrom protein antigens, the isolation and confirmation of these antigens have traditionally relied\non laborious T cell isolation and cloning. Alternatively, direct isolation and sequencing of\nMHC-associated peptides from cells has been limited by the overall low concentration of\nspecific peptide bound to MHC molecules, and the highly polymorphic nature of the MHC\nmolecules themselves. These challenges are being overcome by advances in high-throughput\nMHC-peptide binding studies, the bioinformatics of epitope prediction, and in particular, mass\nspectrometry.\nWith high-throughput peptide synthesis, direct MHC-peptide binding assays may now be\nperformed on peptides that span target antigens, or using combinatorial peptide libraries107.\nDetailed analysis of peptide binding to common HLA alleles has resulted in the development\nof bioinformatics tools for epitope prediction108, 109. Further limitations imposed by\nproteasomal cleavage patterns can enhance prediction of peptide epitopes. Multiple prediction\nmodels are available, primarily for MHC Class I-binding peptides, due to the more restrictive\nnature of the peptide binding groove. These systems include BIMAS110, SYFPEITHI111,\nMHCPEP112, and RANKPEP113, among others.\nAdvances in mass spectrometry have also greatly aided antigen identification. Original studies\nof direct peptide sequencing from purified MHC molecules from cell lines114-116\ndemonstrated that the majority of peptides within the MHC grooves are derived from self-\nproteins, but required billions of cells for the source of antigen. More recent approaches to\nidentify tumor-specific T cell epitopes have used capillary-scale chromatography and tandem\nmass spectrometry (Figure 5) (nanoLC-MS/MS)117-120. To enhance sensitivity, nanospray\nquadrupole-TOF combined with Poisson algorithms121 to specifically evaluate the likelihood\nof predicted peptides in unfractionated mixtures has markedly increased the sensitivity of the\nassay to the range of 1 copy number of peptide/cell in a murine influenza viral model, but is\nlimited to predicted MHC-binding peptide epitopes.\nSummary\nThe development of proteomic-based methods of cell lysate fractionation, phage display,\nprotein microarrays, bioinformatics, and mass spectrometry is resulting in the rapid\nidentification of both B and T cell tumor antigens. These have the potential for clinical\ndiagnosis, identifying targets for immunotherapies, monitoring disease response, and to\nunderstand the breadth, scope, and impact of the immune response to cancer. As each new\ntechnology leads to the discovery of novel antigenic targets, systematic approaches of target\nvalidation and assessment of clinical applications, especially in the area of biomarkers and\ndiagnostic testing, will need to be developed.\nAcknowledgements\n1. Pavlenko M, Roos AK, Lundqvist A, Palmborg A, Miller AM, Ozenci V, Bergman B, Egevad L,\nHellstrom M, Kiessling R, Masucci G, Wersall P, Nilsson S, Pisa P. A phase I trial of DNA vaccination\nwith a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate\ncancer. Br J Cancer 2004;91:688–694. [PubMed: 15280930]\n2. Vonderheide RH, Hahn WC, Schultze JL, Nadler LM. The telomerase catalytic subunit is a widely\nexpressed tumor-associated antigen recognized by cytotoxic T lymphocytes. Immunity 1999;10:673–\n679. [PubMed: 10403642]\n3. Soussi T. p53 Antibodies in the sera of patients with various types of cancer: a review. Cancer Res\n4. Schubert U, Anton LC, Gibbs J, Norbury CC, Yewdell JW, Bennink JR. Rapid degradation of a large\nfraction of newly synthesized proteins by proteasomes. Nature 2000;404:770–774. [PubMed:\n10783891]\n5. Chen YT, Scanlan MJ, Sahin U, Tureci O, Gure AO, Tsang S, Williamson B, Stockert E, Pfreundschuh\nM, Old LJ. A testicular antigen aberrantly expressed in human cancers detected by autologous antibody\nscreening. Proc Natl Acad Sci U S A 1997;94:1914–1918. [PubMed: 9050879]\n6. Ulanet DB, Torbenson M, Dang CV, Casciola-Rosen L, Rosen A. Unique conformation of cancer\nautoantigen B23 in hepatoma: a mechanism for specificity in the autoimmune response. Proc Natl\nAcad Sci U S A 2003;100:12361–12366. [PubMed: 14519847]\n7. von Mensdorff-Pouilly S, Petrakou E, Kenemans P, van Uffelen K, Verstraeten AA, Snijdewint FG,\nvan Kamp GJ, Schol DJ, Reis CA, Price MR, Livingston PO, Hilgers J. Reactivity of natural and\ninduced human antibodies to MUC1 mucin with MUC1 peptides and n-acetylgalactosamine (GalNAc)\npeptides. Int J Cancer 2000;86:702–712. [PubMed: 10797294]\n8. Routsias JG, Tzioufas AG, Moutsopoulos HM. The clinical value of intracellular autoantigens B-cell\nepitopes in systemic rheumatic diseases. Clin Chim Acta 2004;340:1–25. [PubMed: 14734193]\n9. Lernmark A. Autoimmune diseases: are markers ready for prediction? J Clin Invest 2001;108:1091–\n10. Plotz PH. The autoantibody repertoire: searching for order. Nat Rev Immunol 2003;3:73–78.\n11. Brendel V, Dohlman J, Blaisdell BE, Karlin S. Very long charge runs in systemic lupus erythematosus-\nassociated autoantigens. Proc Natl Acad Sci U S A 1991;88:1536–1540. [PubMed: 1996354]\n12. Utz PJ, Hottelet M, Le TM, Kim SJ, Geiger ME, van Venrooij WJ, Anderson P. The 72-kDa\ncomponent of signal recognition particle is cleaved during apoptosis. J Biol Chem 1998;273:35362–\n35370. [PubMed: 9857079]\n13. Casciola-Rosen L, Andrade F, Ulanet D, Wong WB, Rosen A. Cleavage by granzyme B is strongly\npredictive of autoantigen status: implications for initiation of autoimmunity. J Exp Med\n1999;190:815–826. [PubMed: 10499920]\n14. Raben N, Nichols R, Dohlman J, McPhie P, Sridhar V, Hyde C, Leff R, Plotz P. A motif in human\nhistidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-\ncoil that is essential for enzymatic activity and contains the major autoantigenic epitope. J Biol Chem\n1994;269:24277–24283. [PubMed: 7523371]\n15. DeLeo AB, Jay G, Appella E, Dubois GC, Law LW, Old LJ. Detection of a transformation-related\nantigen in chemically induced sarcomas and other transformed cells of the mouse. Proc Natl Acad\nSci U S A 1979;76:2420–2424. [PubMed: 221923]\n16. Casey G, Lopez ME, Ramos JC, Plummer SJ, Arboleda MJ, Shaughnessy M, Karlan B, Slamon DJ.\nDNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to\ndetect all known p53 alterations in human malignancies. Oncogene 1996;13:1971–1981. [PubMed:\n8934544]\n17. Winter SF, Minna JD, Johnson BE, Takahashi T, Gazdar AF, Carbone DP. Development of antibodies\nagainst p53 in lung cancer patients appears to be dependent on the type of p53 mutation. Cancer Res\n1992;52:4168–4174. [PubMed: 1322237]\n18. Dowell SP, Wilson PO, Derias NW, Lane DP, Hall PA. Clinical utility of the immunocytochemical\ndetection of p53 protein in cytological specimens. Cancer Res 1994;54:2914–2918. [PubMed:\n8187077]\n19. Vennegoor CJ, Nijman HW, Drijfhout JW, Vernie L, Verstraeten RA, von Mensdorff-Pouilly S,\nHilgers J, Verheijen RH, Kast WM, Melief CJ, Kenemans P. Autoantibodies to p53 in ovarian cancer\npatients and healthy women: a comparison between whole p53 protein and 18-mer peptides for\nscreening purposes. Cancer Lett 1997;116:93–101. [PubMed: 9177463]\n20. Angelopoulou K, Yu H, Bharaj B, Giai M, Diamandis EP. p53 gene mutation, tumor p53 protein\noverexpression, and serum p53 autoantibody generation in patients with breast cancer. Clin Biochem\n2000;33:53–62. [PubMed: 10693987]\n21. Angelopoulou K, Stratis M, Diamandis EP. Humoral immune response against p53 protein in patients\nwith colorectal carcinoma. Int J Cancer 1997;70:46–51. [PubMed: 8985089]\n22. Rainov NG, Dobberstein KU, Fittkau M, Bahn H, Holzhausen HJ, Gantchev L, Burkert W. Absence\nof p53 autoantibodies in sera from glioma patients. Clin Cancer Res 1995;1:775–781. [PubMed:\n9816045]\n23. Weller M, Bornemann A, Stander M, Schabet M, Dichgans J, Meyermann R. Humoral immune\nresponse to p53 in malignant glioma. J Neurol 1998;245:169–172. [PubMed: 9553848]\n24. Zorn E, Miklos DB, Floyd BH, Mattes-Ritz A, Guo L, Soiffer RJ, Antin JH, Ritz J. Minor\nHistocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem\nCell Transplantation. J Exp Med 2004;199:1133–1142. [PubMed: 15096539]\n25. Schmollinger JC, Vonderheide RH, Hoar KM, Maecker B, Schultze JL, Hodi FS, Soiffer RJ, Jung\nK, Kuroda MJ, Letvin NL, Greenfield EA, Mihm M, Kutok JL, Dranoff G. Melanoma inhibitor of\napoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction. Proc Natl Acad Sci\nU S A 2003;100:3398–3403. [PubMed: 12626761]\n26. Molldrem JJ, Lee PP, Wang C, Felio K, Kantarjian HM, Champlin RE, Davis MM. Evidence that\nspecific T lymphocytes may participate in the elimination of chronic myelogenous leukemia. Nat\nMed 2000;6:1018–1023. [PubMed: 10973322]\n27. Hodi FS, Schmollinger JC, Soiffer RJ, Salgia R, Lynch T, Ritz J, Alyea EP, Yang J, Neuberg D,\nMihm M, Dranoff G. ATP6S1 elicits potent humoral responses associated with immune-mediated\ntumor destruction. Proc Natl Acad Sci U S A 2002;99:6919–6924. [PubMed: 11983866]\n28. Miklos DB, Kim HT, Zorn E, Hochberg EP, Guo L, Mattes-Ritz A, Viatte S, Soiffer RJ, Antin JH,\nRitz J. Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem\ncell transplantation and in healthy female donors. Blood 2004;103:353–359. [PubMed: 14512314]\n29. Bellucci R, Wu CJ, Chiaretti S, Weller E, Davies FE, Alyea EP, Dranoff G, Anderson KC, Munshi\nNC, Ritz J. Complete response to donor lymphocyte infusion in multiple myeloma is associated with\nantibody responses to highly expressed antigens. Blood 2004;103:656–663. [PubMed: 14563636]\n30. Yang XF, Wu CJ, Chen L, Alyea EP, Canning C, Kantoff P, Soiffer RJ, Dranoff G, Ritz J. CML28\nis a broadly immunogenic antigen, which is overexpressed in tumor cells. Cancer Res 2002;62:5517–\n5522. [PubMed: 12359762]\n31. Dranoff G, Jaffee E, Lazenby A, Golumbek P, Levitsky H, Brose K, Jackson V, Hamada H, Pardoll\nD, Mulligan RC. Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-\nmacrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor\nimmunity. Proc Natl Acad Sci U S A 1993;90:3539–3543. [PubMed: 8097319]\n32. Huang AY, Golumbek P, Ahmadzadeh M, Jaffee E, Pardoll D, Levitsky H. Role of bone marrow-\nderived cells in presenting MHC class I-restricted tumor antigens. Science 1994;264:961–965.\n[PubMed: 7513904]\n33. Hung K, Hayashi R, Lafond-Walker A, Lowenstein C, Pardoll D, Levitsky H. The central role of\nCD4(+) T cells in the antitumor immune response. J Exp Med 1998;188:2357–2368. [PubMed:\n9858522]\n34. Gillessen S, Naumov YN, Nieuwenhuis EE, Exley MA, Lee FS, Mach N, Luster AD, Blumberg RS,\nTaniguchi M, Balk SP, Strominger JL, Dranoff G, Wilson SB. CD1d-restricted T cells regulate\ndendritic cell function and antitumor immunity in a granulocyte-macrophage colony-stimulating\nfactor-dependent fashion. Proc Natl Acad Sci U S A 2003;100:8874–8879. [PubMed: 12847287]\n35. Heslop HE, Stevenson FK, Molldrem JJ. Immunotherapy of hematologic malignancy. Hematology\n(Am Soc Hematol Educ Program) 2003:331–349. [PubMed: 14633789]\n36. Davis ID, Chen W, Jackson H, Parente P, Shackleton M, Hopkins W, Chen Q, Dimopoulos N, Luke\nT, Murphy R, Scott AM, Maraskovsky E, McArthur G, MacGregor D, Sturrock S, Tai TY, Green S,\nCuthbertson A, Maher D, Miloradovic L, Mitchell SV, Ritter G, Jungbluth AA, Chen YT, Gnjatic\nS, Hoffman EW, Old LJ, Cebon JS. Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant\ninduces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans. Proc Natl\nAcad Sci U S A 2004;101:10697–10702. [PubMed: 15252201]\n37. DiFronzo LA, Gupta RK, Essner R, Foshag LJ, O'Day SJ, Wanek LA, Stern SL, Morton DL. Enhanced\nhumoral immune response correlates with improved disease-free and overall survival in American\nJoint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine. J Clin\nOncol 2002;20:3242–3248. [PubMed: 12149297]\n38. Ehlken H, Schadendorf D, Eichmuller S. Humoral immune response against melanoma antigens\ninduced by vaccination with cytokine gene-modified autologous tumor cells. Int J Cancer\n2004;108:307–313. [PubMed: 14639620]\n39. Small, EJ.; S, PF.; Higano, C.; Neumanaitis, J.; Valone, F.; Herschberg, RM. Immunotherapy\n(APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3\nrandomized placebo-controlled trial; 2005 ASCO Prostate Cancer Symposium; 2005.\n40. Cancer Facts and Figures. 2004.\n41. Srinivas PR, Kramer BS, Srivastava S. Trends in biomarker research for cancer detection. Lancet\nOncol 2001;2:698–704. [PubMed: 11902541]\n42. Gold P, Freedman SO. Specific carcinoembryonic antigens of the human digestive system. J Exp\nMed 1965;122:467–481. [PubMed: 4953873]\n43. Wang MC, Valenzuela LA, Murphy GP, Chu TM. Purification of a human prostate specific antigen.\nInvest Urol 1979;17:159–163. [PubMed: 89106]\n44. Niloff JM, Knapp RC, Schaetzl E, Reynolds C, Bast RC Jr. CA125 antigen levels in obstetric and\ngynecologic patients. Obstet Gynecol 1984;64:703–707. [PubMed: 6208522]\n45. Petricoin E, Wulfkuhle J, Espina V, Liotta LA. Clinical proteomics: revolutionizing disease detection\nand patient tailoring therapy. J Proteome Res 2004;3:209–217. [PubMed: 15113096]\n46. Trivers GE, Cawley HL, DeBenedetti VM, Hollstein M, Marion MJ, Bennett WP, Hoover ML, Prives\nCC, Tamburro CC, Harris CC. Anti-p53 antibodies in sera of workers occupationally exposed to\nvinyl chloride. J Natl Cancer Inst 1995;87:1400–1407. [PubMed: 7658501]\n47. Trivers GE, De Benedetti VM, Cawley HL, Caron G, Harrington AM, Bennett WP, Jett JR, Colby\nTV, Tazelaar H, Pairolero P, Miller RD, Harris CC. Anti-p53 antibodies in sera from patients with\nchronic obstructive pulmonary disease can predate a diagnosis of cancer. Clin Cancer Res\n1996;2:1767–1775. [PubMed: 9816128]\n48. Suzuki H, Graziano DF, McKolanis J, Finn OJ. T cell-dependent antibody responses against\naberrantly expressed cyclin B1 protein in patients with cancer and premalignant disease. Clin Cancer\nRes 2005;11:1521–1526. [PubMed: 15746055]\n49. Rodolfo M, Luksch R, Stockert E, Chen YT, Collini P, Ranzani T, Lombardo C, Dalerba P, Rivoltini\nL, Arienti F, Fossati-Bellani F, Old LJ, Parmiani G, Castelli C. Antigen-specific immunity in\nneuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen. Cancer Res\n2003;63:6948–6955. [PubMed: 14583496]\n50. Litvak DA, Gupta RK, Yee R, Wanek LA, Ye W, Morton DL. Endogenous immune response to early-\nand intermediate-stage melanoma is correlated with outcomes and is independent of locoregional\nrelapse and standard prognostic factors. J Am Coll Surg 2004;198:27–35. [PubMed: 14698308]\n51. Disis ML, Pupa SM, Gralow JR, Dittadi R, Menard S, Cheever MA. High-titer HER-2/neu protein-\nspecific antibody can be detected in patients with early-stage breast cancer. J Clin Oncol\n1997;15:3363–3367. [PubMed: 9363867]\n52. Disis ML, Knutson KL, Schiffman K, Rinn K, McNeel DG. Pre-existent immunity to the HER-2/neu\noncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer. Breast\nCancer Res Treat 2000;62:245–252. [PubMed: 11072789]\n53. Jager E, Stockert E, Zidianakis Z, Chen YT, Karbach J, Jager D, Arand M, Ritter G, Old LJ, Knuth\nA. Humoral immune responses of cancer patients against “Cancer-Testis” antigen NY-ESO-1:\ncorrelation with clinical events. Int J Cancer 1999;84:506–510. [PubMed: 10502728]\n54. Regele S, Vogl FD, Kohler T, Kreienberg R, Runnebaum IB. p53 autoantibodies can be indicative\nof the development of breast cancer relapse. Anticancer Res 2003;23:761–764. [PubMed: 12680180]\n55. Takeda A, Shimada H, Nakajima K, Imaseki H, Suzuki T, Asano T, Ochiai T, Isono K. Monitoring\nof p53 autoantibodies after resection of colorectal cancer: relationship to operative curability. Eur J\nSurg 2001;167:50–53. [PubMed: 11213822]\n56. Kelley MC, Gupta RK, Hsueh EC, Yee R, Stern S, Morton DL. Tumor-associated antigen TA90\nimmune complex assay predicts recurrence and survival after surgical treatment of stage I-III\nmelanoma. J Clin Oncol 2001;19:1176–1182. [PubMed: 11181684]\n57. Tajima K, Obata Y, Tamaki H, Yoshida M, Chen YT, Scanlan MJ, Old LJ, Kuwano H, Takahashi T,\nMitsudomi T. Expression of cancer/testis (CT) antigens in lung cancer. Lung Cancer 2003;42:23–\n33. [PubMed: 14512184]\n58. Lee SY, Obata Y, Yoshida M, Stockert E, Williamson B, Jungbluth AA, Chen YT, Old LJ, Scanlan\nMJ. Immunomic analysis of human sarcoma. Proc Natl Acad Sci U S A 2003;100:2651–2656.\n[PubMed: 12601173]\n59. Ayyoub M, Hesdorffer CS, Montes M, Merlo A, Speiser D, Rimoldi D, Cerottini JC, Ritter G, Scanlan\nM, Old LJ, Valmori D. An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in\nassociation with HLA-DR. J Clin Invest 2004;113:1225–1233. [PubMed: 15085202]\n60. Sugita Y, Wada H, Fujita S, Nakata T, Sato S, Noguchi Y, Jungbluth AA, Yamaguchi M, Chen YT,\nStockert E, Gnjatic S, Williamson B, Scanlan MJ, Ono T, Sakita I, Yasui M, Miyoshi Y, Tamaki Y,\nMatsuura N, Noguchi S, Old LJ, Nakayama E, Monden M. NY-ESO-1 expression and\nimmunogenicity in malignant and benign breast tumors. Cancer Res 2004;64:2199–2204. [PubMed:\n15026363]\n61. Odunsi K, Jungbluth AA, Stockert E, Qian F, Gnjatic S, Tammela J, Intengan M, Beck A, Keitz B,\nSantiago D, Williamson B, Scanlan MJ, Ritter G, Chen YT, Driscoll D, Sood A, Lele S, Old LJ. NY-\nESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial\novarian cancer. Cancer Res 2003;63:6076–6083. [PubMed: 14522938]\n62. Scanlan MJ, Gout I, Gordon CM, Williamson B, Stockert E, Gure AO, Jager D, Chen YT, Mackay\nA, O'Hare MJ, Old LJ. Humoral immunity to human breast cancer: antigen definition and quantitative\nanalysis of mRNA expression. Cancer Immun 2001;1:4. [PubMed: 12747765]\n63. Jager D, Unkelbach M, Frei C, Bert F, Scanlan MJ, Jager E, Old LJ, Chen YT, Knuth A. Identification\nof tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by\nserological screening of a testicular library with breast cancer serum. Cancer Immun 2002;2:5.\n[PubMed: 12747750]\n64. Koroleva EP, Lagarkova MA, Mesheryakov AA, Scanlan MJ, Old LJ, Nedospasov SA, Kuprash DV.\nSerological identification of antigens associated with renal cell carcinoma. Russ J Immunol\n2002;7:229–238. [PubMed: 12674932]\n65. Scanlan MJ, Gure AO, Jungbluth AA, Old LJ, Chen YT. Cancer/testis antigens: an expanding family\nof targets for cancer immunotherapy. Immunol Rev 2002;188:22–32. [PubMed: 12445278]\n66. Forti S, Scanlan MJ, Invernizzi A, Castiglioni F, Pupa S, Agresti R, Fontanelli R, Morelli D, Old LJ,\nPupa SM, Menard S. Identification of breast cancer-restricted antigens by antibody screening of\nSKBR3 cDNA library using a preselected patient's serum. Breast Cancer Res Treat 2002;73:245–\n256. [PubMed: 12160330]\n67. Scanlan MJ, Gordon CM, Williamson B, Lee SY, Chen YT, Stockert E, Jungbluth A, Ritter G, Jager\nD, Jager E, Knuth A, Old LJ. Identification of cancer/testis genes by database mining and mRNA\nexpression analysis. Int J Cancer 2002;98:485–492. [PubMed: 11920606]\n68. Jager D, Stockert E, Gure AO, Scanlan MJ, Karbach J, Jager E, Knuth A, Old LJ, Chen YT.\nIdentification of a tissue-specific putative transcription factor in breast tissue by serological screening\nof a breast cancer library. Cancer Res 2001;61:2055–2061. [PubMed: 11280766]\n69. Gure AO, Stockert E, Scanlan MJ, Keresztes RS, Jager D, Altorki NK, Old LJ, Chen YT. Serological\nidentification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung\ncancer. Proc Natl Acad Sci U S A 2000;97:4198–4203. [PubMed: 10760287]\n70. Scanlan MJ, Altorki NK, Gure AO, Williamson B, Jungbluth A, Chen YT, Old LJ. Expression of\ncancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a\nnew CT gene, CT9. Cancer Lett 2000;150:155–164. [PubMed: 10704737]\n71. Gure AO, Stockert E, Arden KC, Boyer AD, Viars CS, Scanlan MJ, Old LJ, Chen YT. CT10: a new\ncancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-\ndifference analysis. Int J Cancer 2000;85:726–732. [PubMed: 10699956]\n72. Jager D, Stockert E, Scanlan MJ, Gure AO, Jager E, Knuth A, Old LJ, Chen YT. Cancer-testis antigens\nand ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-\nspecific ING1 transcripts and a homologue gene. Cancer Res 1999;59:6197–6204. [PubMed:\n10626813]\n73. Scanlan MJ, Gordan JD, Williamson B, Stockert E, Bander NH, Jongeneel V, Gure AO, Jager D,\nJager E, Knuth A, Chen YT, Old LJ. Antigens recognized by autologous antibody in patients with\nrenal-cell carcinoma. Int J Cancer 1999;83:456–464. [PubMed: 10508479]\n74. Scanlan MJ, Williamson B, Jungbluth A, Stockert E, Arden KC, Viars CS, Gure AO, Gordan JD,\nChen YT, Old LJ. Isoforms of the human PDZ-73 protein exhibit differential tissue expression.\nBiochim Biophys Acta 1999;1445:39–52. [PubMed: 10209257]\n75. Scanlan MJ, Chen YT, Williamson B, Gure AO, Stockert E, Gordan JD, Tureci O, Sahin U,\nPfreundschuh M, Old LJ. Characterization of human colon cancer antigens recognized by autologous\nantibodies. Int J Cancer 1998;76:652–658. [PubMed: 9610721]\n76. Stockert E, Jager E, Chen YT, Scanlan MJ, Gout I, Karbach J, Arand M, Knuth A, Old LJ. A survey\nof the humoral immune response of cancer patients to a panel of human tumor antigens. J Exp Med\n1998;187:1349–1354. [PubMed: 9547346]\n77. Gure AO, Altorki NK, Stockert E, Scanlan MJ, Old LJ, Chen YT. Human lung cancer antigens\nrecognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor\ngene locus on chromosome 3p21.3. Cancer Res 1998;58:1034–1041. [PubMed: 9500467]\n78. Gure AO, Tureci O, Sahin U, Tsang S, Scanlan MJ, Jager E, Knuth A, Pfreundschuh M, Old LJ, Chen\nYT. SSX: a multigene family with several members transcribed in normal testis and human cancer.\nInt J Cancer 1997;72:965–971. [PubMed: 9378559]\n79. Mollick JA, Hodi FS, Soiffer RJ, Nadler LM, Dranoff G. MUC1-like tandem repeat proteins are\nbroadly immunogenic in cancer patients. Cancer Immun 2003;3:3. [PubMed: 12747745]\n80. Chen, YT.; S, M.; Obata, Y.; Old, LJ. Identification of human tumor antigens by serological expression\ncloning. In: SA, R., editor. Principles and practice of the biologic therapy of cancer. 3rd. Lippincott\nWilliams & Wilkins; Philadelphia: 2000. p. 557-570.\n81. Line A, Slucka Z, Stengrevics A, Silina K, Li G, Rees RC. Characterisation of tumour-associated\nantigens in colon cancer. Cancer Immunol Immunother 2002;51:574–582. [PubMed: 12384809]\n82. Ishikawa T, Fujita T, Suzuki Y, Okabe S, Yuasa Y, Iwai T, Kawakami Y. Tumor-specific\nimmunological recognition of frameshift-mutated peptides in colon cancer with microsatellite\ninstability. Cancer Res 2003;63:5564–5572. [PubMed: 14500396]\n83. Kemp EH, Herd LM, Waterman EA, Wilson AG, Weetman AP, Watson PP. Immunoscreening of\nphage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease.\nBiochem Biophys Res Commun 2002;298:169–177. [PubMed: 12379236]\n84. Jensen LB, Riise E, Nielsen LK, Dziegiel M, Fugger L, Engberg J. Efficient purification of unique\nantibodies using peptide affinity-matrix columns. J Immunol Methods 2004;284:45–54. [PubMed:\n14736416]\n85. Sioud M, Hansen M, Dybwad A. Profiling the immune responses in patient sera with peptide and\ncDNA display libraries. Int J Mol Med 2000;6:123–128. [PubMed: 10891554]\n86. Fossa A, Alsoe L, Crameri R, Funderud S, Gaudernack G, Smeland EB. Serological cloning of cancer/\ntestis antigens expressed in prostate cancer using cDNA phage surface display. Cancer Immunol\nImmunother 2004;53:431–438. [PubMed: 14747957]\n87. Sioud M, Hansen MH. Profiling the immune response in patients with breast cancer by phage-\ndisplayed cDNA libraries. Eur J Immunol 2001;31:716–725. [PubMed: 11241275]\n88. Minenkova O, Pucci A, Pavoni E, De Tomassi A, Fortugno P, Gargano N, Cianfriglia M, Barca S,\nDe Placido S, Martignetti A, Felici F, Cortese R, Monaci P. Identification of tumor-associated\nantigens by screening phage-displayed human cDNA libraries with sera from tumor patients. Int J\nCancer 2003;106:534–544. [PubMed: 12845649]\n89. Fernandez-Madrid F, Tang N, Alansari H, Granda JL, Tait L, Amirikia KC, Moroianu M, Wang X,\nKarvonen RL. Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast\nCancer. Cancer Res 2004;64:5089–5096. [PubMed: 15289310]\n90. Mischo A, Wadle A, Watzig K, Jager D, Stockert E, Santiago D, Ritter G, Regitz E, Jager E, Knuth\nA, Old L, Pfreundschuh M, Renner C. Recombinant antigen expression on yeast surface (RAYS) for\nthe detection of serological immune responses in cancer patients. Cancer Immun 2003;3:5. [PubMed:\n12828452]\n91. Yan SC, Grinnell BW, Wold F. Post-translational modifications of proteins: some problems left to\nsolve. Trends Biochem Sci 1989;14:264–268. [PubMed: 2672445]\n92. St Clair EW, Kenan D, Burch JA Jr, Keene JD, Pisetsky DS. The fine specificity of anti-La antibodies\ninduced in mice by immunization with recombinant human La autoantigen. J Immunol\n1990;144:3868–3876. [PubMed: 1692063]\n93. Doyle HA, Mamula MJ. Post-translational protein modifications in antigen recognition and\nautoimmunity. Trends Immunol 2001;22:443–449. [PubMed: 11473834]\n94. Doyle HA, Mamula MJ. Posttranslational protein modifications: new flavors in the menu of\nautoantigens. Curr Opin Rheumatol 2002;14:244–249. [PubMed: 11981321]\n95. Koziol JA, Zhang JY, Casiano CA, Peng XX, Shi FD, Feng AC, Chan EK, Tan EM. Recursive\npartitioning as an approach to selection of immune markers for tumor diagnosis. Clin Cancer Res\n2003;9:5120–5126. [PubMed: 14613989]\n96. Zhang JY, Casiano CA, Peng XX, Koziol JA, Chan EK, Tan EM. Enhancement of antibody detection\nin cancer using panel of recombinant tumor-associated antigens. Cancer Epidemiol Biomarkers Prev\n2003;12:136–143. [PubMed: 12582023]\n97. Joos TO, Stoll D, Templin MF. Miniaturised multiplexed immunoassays. Curr Opin Chem Biol\n98. Stoll D, Templin MF, Schrenk M, Traub PC, Vohringer CF, Joos TO. Protein microarray technology.\nFront Biosci 2002;7:c13–32. [PubMed: 11779717]\n99. Templin MF, Stoll D, Schwenk JM, Potz O, Kramer S, Joos TO. Protein microarrays: promising tools\nfor proteomic research. Proteomics 2003;3:2155–2166. [PubMed: 14595815]\n100. Robinson WH, DiGennaro C, Hueber W, Haab BB, Kamachi M, Dean EJ, Fournel S, Fong D,\nGenovese MC, de Vegvar HE, Skriner K, Hirschberg DL, Morris RI, Muller S, Pruijn GJ, van\nVenrooij WJ, Smolen JS, Brown PO, Steinman L, Utz PJ. Autoantigen microarrays for multiplex\ncharacterization of autoantibody responses. Nat Med 2002;8:295–301. [PubMed: 11875502]\n101. Robinson WH, Steinman L, Utz PJ. Protein arrays for autoantibody profiling and fine-specificity\nmapping. Proteomics 2003;3:2077–2084. [PubMed: 14595805]\n102. Shin BK, Wang H, Hanash S. Proteomics approaches to uncover the repertoire of circulating\nbiomarkers for breast cancer. J Mammary Gland Biol Neoplasia 2002;7:407–413. [PubMed:\n12882525]\n103. Scanlan MJ, Welt S, Gordon CM, Chen YT, Gure AO, Stockert E, Jungbluth AA, Ritter G, Jager\nD, Jager E, Knuth A, Old LJ. Cancer-related serological recognition of human colon cancer:\nidentification of potential diagnostic and immunotherapeutic targets. Cancer Res 2002;62:4041–\n4047. [PubMed: 12124339]\n104. Qiu J, Madoz-Gurpide J, Misek DE, Kuick R, Brenner DE, Michailidis G, Haab BB, Omenn GS,\nHanash S. Development of natural protein microarrays for diagnosing cancer based on an antibody\nresponse to tumor antigens. J Proteome Res 2004;3:261–267. [PubMed: 15113102]\n105. Bouwman K, Qiu J, Zhou H, Schotanus M, Mangold LA, Vogt R, Erlandson E, Trenkle J, Partin\nAW, Misek D, Omenn GS, Haab BB, Hanash S. Microarrays of tumor cell derived proteins uncover\na distinct pattern of prostate cancer serum immunoreactivity. Proteomics 2003;3:2200–2207.\n[PubMed: 14595819]\n106. Ramachandran N, Hainsworth E, Bhullar B, Eisenstein S, Rosen B, Lau AY, Walter JC, LaBaer J.\nSelf-assembling protein microarrays. Science 2004;305:86–90. [PubMed: 15232106]\n107. Lawendowski CA, Giurleo GM, Huang YY, Franklin GJ, Kaplan JM, Roberts BL, Nicolette CA.\nSolid-phase epitope recovery: a high throughput method for antigen identification and epitope\noptimization. J Immunol 2002;169:2414–2421. [PubMed: 12193709]\n108. Rammensee HG, Weinschenk T, Gouttefangeas C, Stevanovic S. Towards patient-specific tumor\nantigen selection for vaccination. Immunol Rev 2002;188:164–176. [PubMed: 12445290]\n109. Sette A, Keogh E, Ishioka G, Sidney J, Tangri S, Livingston B, McKinney D, Newman M, Chesnut\nR, Fikes J. Epitope identification and vaccine design for cancer immunotherapy. Curr Opin Investig\nDrugs 2002;3:132–139.\n110. Parker KC, Bednarek MA, Coligan JE. Scheme for ranking potential HLA-A2 binding peptides\nbased on independent binding of individual peptide side-chains. J Immunol 1994;152:163–175.\n[PubMed: 8254189]\n111. Rammensee H, Bachmann J, Emmerich NP, Bachor OA, Stevanovic S. SYFPEITHI: database for\nMHC ligands and peptide motifs. Immunogenetics 1999;50:213–219. [PubMed: 10602881]\n112. Brusic V, Rudy G, Harrison LC. MHCPEP, a database of MHC-binding peptides: update 1997.\nNucleic Acids Res 1998;26:368–371. [PubMed: 9399876]\n113. Reche PA, Glutting JP, Zhang H, Reinherz EL. Enhancement to the RANKPEP resource for the\nprediction of peptide binding to MHC molecules using profiles. Immunogenetics 2004;56:405–419.\n[PubMed: 15349703]\n114. Hunt DF, Henderson RA, Shabanowitz J, Sakaguchi K, Michel H, Sevilir N, Cox AL, Appella E,\nEngelhard VH. Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass\nspectrometry. Science 1992;255:1261–1263. [PubMed: 1546328]\n115. Wei ML, Cresswell P. HLA-A2 molecules in an antigen-processing mutant cell contain signal\nsequence-derived peptides. Nature 1992;356:443–446. [PubMed: 1557127]\n116. Rotzschke O, Falk K, Wallny HJ, Faath S, Rammensee HG. Characterization of naturally occurring\nminor histocompatibility peptides including H-4 and H-Y. Science 1990;249:283–287. [PubMed:\n1695760]\n117. Kao H, Marto JA, Hoffmann TK, Shabanowitz J, Finkelstein SD, Whiteside TL, Hunt DF, Finn OJ.\nIdentification of cyclin B1 as a shared human epithelial tumor-associated antigen recognized by T\ncells. J Exp Med 2001;194:1313–1323. [PubMed: 11696596]\n118. Hogan KT, Coppola MA, Gatlin CL, Thompson LW, Shabanowitz J, Hunt DF, Engelhard VH,\nSlingluff CL Jr, Ross MM. Identification of a shared epitope recognized by melanoma-specific,\nHLA-A3-restricted cytotoxic T lymphocytes. Immunol Lett 2003;90:131–135. [PubMed:\n14687714]\n119. Lemmel C, Stevanovic S. The use of HPLC-MS in T-cell epitope identification. Methods\n120. Weinschenk T, Gouttefangeas C, Schirle M, Obermayr F, Walter S, Schoor O, Kurek R, Loeser W,\nBichler KH, Wernet D, Stevanovic S, Rammensee HG. Integrated functional genomics approach\nfor the design of patient-individual antitumor vaccines. Cancer Res 2002;62:5818–5827. [PubMed:\n12384544]\n121. Zhong W, Reche PA, Lai CC, Reinhold B, Reinherz EL. Genome-wide characterization of a viral\ncytotoxic T lymphocyte epitope repertoire. J Biol Chem 2003;278:45135–45144. [PubMed:\n12960169]\nCurrent Methodologies for the Detection of Autoantigens",
         "",
         "",
         "",
         "macromolecules in routine circulation. The system is not foolproof, however, and in certain\ndiseases, the immune system responds to self-derived antigens, perhaps because their location,\nabundance, modified form or other features appear unfamiliar. Cancer patients often produce\nresponses to self-proteins that are expressed by their tumors, called tumor antigens, most of\nwhich are altered in some form that renders them immunogenic. These proteins may be unique\nto cancer and germ cells (the “cancer-testis” antigens), found only in specific tumors (prostate-\nspecific antigen)1 or in most tumors (telomerase)2.\nThey may be mutated (p53)3, misfolded4, overexpressed (NY-ESO-1) 5, aberrantly\ndegraded6 or aberrantly glycosylated (MUC-1)7. The magnitude of the immune response to\ncancer, in general, is lower than the immune response to infectious agents and the potential\nnumber of tumor antigens encompasses the entire tumor proteome in all its variations. At\npresent, we have a limited understanding of the breadth, extent, impact, and dynamic variation\nof the immune response to cancer (the “cancer immunome”). Identifying the specific targets\nof B- and T- lymphocyte immunity to cancer may 1) identify potential biomarkers for cancer\ndiagnosis, classification, and monitoring of response, 2) determine the impact of immune\nregulation on cancer progression, and 3) identify potential antigens and mechanisms for\nimmunotherapy development.\nThe natural immune response is achieved through a tightly regulated, yet flexible network\nincluding antibodies, antigen presenting cells, T lymphocytes, cytokines, chemokines,\nregulatory systems, as well as microenvironmental signals (Figure 1). Of these responses, the\ntargeted responses to protein (and carbohydrate) antigens relies on the development of\nantibodies and/or T lymphocytes to target epitopes. T lymphocytes can respond to antigens\nderived from within cells and without. They primarily recognize short peptides (8-22mer)\nderived from intracellular proteins (i.e., viral antigens) bound to self-MHC molecules for\npresentation to CD8+ T lymphocytes. Exogenous antigens are endocytosed, degraded, and\npresented to CD4+ lymphocytes (Figure 2). Antibody responses increase antigen presentation\nby enhancing uptake through the Fcγ receptors on antigen presenting cells. As a result, antibody\ntargets may contain epitopes that are also recognized by T lymphocytes. This has formed the\nbasis for using antibody responses to identify T cell antigens for immunotherapy.\nLessons from Autoimmune Diseases\nSpontaneous autoantibodies were first identified in a series of clinical disorders in which the\npatients' immune systems mount a vigorous response to self antigens, in some cases leading\nto debilitating symptoms. Systemic lupus, myasthenia gravis, rheumatoid arthritis and others\nall involve this process, called autoimmunity. In these illnesses, the titers of autoantibodies\noften track with the severity of the illness and thus have long been used as serum biomarkers\n(reviewed in8, 9).\nAlthough post hoc arguments can usually be constructed explaining why proven autoantigens\nmay have appeared unfamiliar to the immune system; in general, we do not understand enough\nabout the characteristics that determine whether a protein will act as an autoantigen to predict\nthem a priori. To date, only one or two percent of proteins have been identified as autoantibody\ntargets10. In autoimmune diseases, the autoantigens identified, such as Ro/SSA, La/SSB, Sm,\nRNP, Scl-70 and Jo-1 are primarily intracellular antigens that function in large complexes with\nnucleic acids involved in protein synthesis8, and are thought to arise as a result of\noverexpression, apoptosis, reduced degradation, or similarity to cross-reacting foreign antigens\n(mimicry). It is often not clear whether these autoantibodies have any direct role in disease\npathophysiology. In contrast, antibodies directed at membrane proteins, such as the target\nantigens of hemolytic anemia or the acetocholine receptor in myasthenia gravis may contribute\nto the disease process through complement activation, antibody-dependent cytotoxicity, or\ninterference with receptor/ligand interactions. Certain structural motifs enhance antigenicity,\nsuch as carbohydrates, multivalency, epitope repetition, and coiled-coils11. Some of these\nantigens are linked to degradation during apoptosis12, or are targets of granzyme B cleavage\nduring T lymphocyte-induced cytolysis13, 14. Further elucidation of the mechanisms\nunderlying autoantibody generation will assist in predicting and identifying target antigens.\np53 as a Model Tumor Autoantigen\nPerhaps the most well-studied autoantigen in cancer is the p53 protein (reviewed in 3). In 1979,\nDeLeo et al15 demonstrated that autoantibodies to some tumor cells in mice were directed\nagainst the p53 protein. Subsequent studies have demonstrated that the half-life of mutated p53\n(several hours) was markedly increased compared to wild-type (several minutes) resulting in\naccumulation in the cell nucleus. p53 autoantibodies are dependent on the type of p53\nmutation16-18. Notably, the immunogenic epitopes have been mapped primarily to both the\nN- and C-terminal portions of the molecule, which are heavily glycosylated, but not to the\ncentral portion of the molecule, which harbors the mutations, suggesting that the accumulation\nof protein, rather than the mutations per se, results in autoantibody generation. Multiple assays\nusing recombinant antigens and p53-derived peptides have shown that some antigenic\ndeterminants are dependent on full-length antigen, conformation, or phosphorylation3, 19.\nThe detection of p53 autoantibodies in serum correlates strongly with cancer (p<10-4). p53\nautoantibodies have been studied in over 9489 patients with a wide variety of tumors3. Despite\nthe strong specificity of the response, only 20-40% of patients with cancers harboring p53\nmissense mutations will have p53 antibodies in their sera3. Therefore, there are additional\nunknown factors beyond antigen accumulation that impact the development of autoantibodies.\nThis illustrates one of the major challenges facing the use of single autoantibody tests for cancer\ndetection. Although the appearance of such antibodies is highly suggestive of cancer, many\npatients will be missed because they do not respond to p53. p53 autoantibodies have been\ndetected in body fluids, such as ascites, pleural effusions, and saliva, and these correlate with\nserum autoantibodies19-21. Interestingly, gliomas, which are associated with p53 mutations,\nare not associated with the development of p53 antibodies22, 23, suggesting either that\ndevelopment of tumors in the CNS leads to immunologic privilege, or the use of steroids in\nthis patient population could dampen the serologic immune response.\nAnti-tumor Effects of Autoantibodies in Cancer\nIt is not known if autoantibodies reflect underlying immunosurveillance of cancer or have an\nimpact on the clinical outcome of the disease. For example, multiple studies have attempted\nto correlate p53 autoantibodies with prognosis showing mixed results3. As the development\nof B cell immunity often depends on concordant T cell immunity, autoantibody identification\ncan lead to the identification of relevant T cell antigens24-26. In patients who have undergone\nautologous tumor vaccination, certain autoantibodies correlate with tumor response to\ntreatment25, 27. In bone marrow transplantation, the development of autoantibodies may\nidentify minor histocompatibility antigens associated with graft versus host disease28 or tumor\nantigens associated with the graft-vs-leukemia (GVL) effects of donor lymphocyte\ninfusion29, 30. In murine tumor models, it is the coordinated activities of CD4+ and CD8+ T\ncells, CD1d-restricted invariant NKT cells, and antibodies that accomplish protective\nimmunity 31, 32 33 34.\nThe serologic identification of tumor antigens has directly led to the development of a number\nof cancer vaccines currently in clinical trials. The proteinase-3 antigen, first identified as the\ntarget antigen of the ANCA antibody assay for Wegener's granulomatosis, is being tested as a\ntarget for vaccination in CML35. NY-ESO-1 has been used as an immunogen in multiple\nclinical trials36. Antibody response to tumor vaccines have been shown to correlate with\nimproved disease-free and overall survival in stage II melanoma patients vaccinated with a\npolyvalent vaccine37. Melanoma patients vaccinated with autologous tumor cells may develop\nhighly-individualized antibody responses38. The identification of these autoantibodies has\ndirectly led to mapping of CD4+ and CD8+ T lymphocyte epitopes. The clinical benefit of\nantigen-specific tumor vaccination strategies is just beginning to emerge, with the recent\ndemonstration of a survival benefit of a dendritic cell-based vaccine in hormone-refractory\nprostate cancer39.\nAutoantibodies as Biomarkers of Cancer\nIn 2003, it is estimated that 1,334,100 people in the US were newly diagnosed with cancer,\nwith an estimated 556,500 patients dying of the disease40. In this setting, there is intense effort\nin the search for biomarkers that can predict disease, identify biologic subtypes of disease,\ntrack response to therapeutic interventions, predict side effect profiles, and monitor for disease\nprogression and recurrence41.\nThere are a limited number of serum protein biomarkers that are widely used in clinical\noncology for prognosis and treatment monitoring, and their use in early cancer diagnosis has\nbeen hampered by false-positive rates in the normal population. The identification and\ndevelopment of these biomarkers took decades of research and required large prospective trials.\nCarcinoembryonic antigen (CEA), widely used for the monitoring of adenocarcinomas, was\noriginally identified from tumor lysates because of its immunogenicity in rabbits42. The\nsubsequent identification of the prostate specific antigen (PSA)43 and the ovarian cancer\nbiomarker CA-12544, among others, have demonstrated both the validity and limitations of\nserum biomarkers in the diagnosis of cancer. Although PSA is now routinely used for screening\nhealthy populations for prostate cancer diagnosis, most serum biomarkers are used for\nmonitoring treatment or for screening highly selected high-risk populations. More recent\ndevelopments in serum proteomics have been reviewed in this issue and elsewhere45.\nCompared with other polypeptides, autoantibodies have many appealing features as\nbiomarkers. First, although tumor antigens may circulate only briefly or in low concentration,\nperhaps due to transient shedding by tumors, rapid degradation in the serum, or rapid clearance,\nthe corresponding antibody response is likely to be persistent. Second, antibodies are highly\nstable in serum samples and are not subject to the types of proteolysis that are commonly\nobserved for other polypeptides (discussed elsewhere in this issue), making sample handling\nmuch easier. Third, the t1/2 of antibodies in circulation is >7 days, so hourly or daily fluctuations\nare expected to be minimal, simplifying sample collection. Finally, the biochemical properties\nof antibodies are well understood and there are many available reagents for their detection,\nsimplifying assay development. Given these advantages, the challenge that remains is\ndemonstrating that antibodies can be sufficiently informative to reliably detect cancer.\nEarly Disease Diagnosis—Very limited data are available on the use of autoantibodies to\nreveal early disease. Antibodies have been detected as early as several years before the clinical\nappearance of cancer46, 47 and in patients with preneoplastic disease48. Anecdotal studies\nhave detected p53 autoantibodies in heavy smokers prior to the diagnosis of lung cancer47.\nAnother study showed p53 autoantibodies in individuals exposed to vinyl chloride, a risk factor\nfor the development of angiosarcoma of the liver46. The p53 autoantibodies predated the\ndiagnosis of angiosarcoma by several years. Although encouraging, these few studies are not\nyet sufficient to endorse the value of antibodies in the early detection setting.\nMonitoring treatment response and predicting recurrence—Antibodies to tumor\nantigens have been detected in early stages of disease and fluctuate with tumor response. In\npatients with both stage III and stage IV neuroblastoma, 10% have evidence of antibodies to\nthe NY-ESO-1 antigen, but the antibody is not seen in the sera of patients in clinical remission\nor in earlier stages of disease49. In melanoma, antibodies to TA90 antigen have been detected\nin only 12% of patients with 1-2mm primary, node negative melanoma who subsequently\nrelapsed, but 62% of case-controlled patients who did not50.\nAntibodies to the her2/neu tumor antigen have been detected in the sera of 20% of patients\nwith her2+ early-stage breast cancer51 but only 7% of late-stage breast and ovarian cancer\npatients52. Some of the early-stage patients had titers>1:5000, suggesting a strong, possibly\nprotective immune response. In patients with both stage III and stage IV neuroblastoma, 10%\nhave evidence of antibodies to the NY-ESO-1 antigen, but the antibody is not seen in the sera\nof patients in clinical remission or in earlier stages of disease49.\nThe titer of antibodies to NY-ESO-1 have been shown to correlate with disease progression,\nas well as with disease response/resection53. The titer of anti-p53 antibodies have been shown\nto increase in a subset of patients with early-stage breast cancer prior to disease relapse54, and\ndisappearance of anti-p53 antibodies has been observed in 27 patients after resection of\ncolorectal cancer55. Tumor antigen/antibody immune complexes have been detected up to 19\nmonths prior to the development of clinical relapse and correlate with survival in patients with\nearly-stage melanoma56.\nMethods for assessing autoantibodies\nThe development process for autoantibodies as biomarkers is outlined in Figure 3. Historically,\nthis cycle begins with identifying antigens that are detected by the sera of cancer patients.\nAntigens must then be selected and tested by comparing sera from both patients and healthy\ndonors to determine if the antigens are “informative”, i.e., responses are limited to patients. As\npart of this process, the sensitivity and specificity of the antigens should be determined.\nPromising antigens are then tested on a set of samples separate from the training set to validate\ntheir usefulness. Ultimately prospective trials will establish their value as biomarkers. The\ncycle is completed by using the sera from patients not detected by the test to screen for new\nantigens. In some of the newer proteomic approaches, two of these steps may combine into a\nsingle step.\nIdentification of novel tumor antigens\nSEREX: A variety of techniques have been developed that use patient sera as probes against\ncandidate antigens derived from tumor cells and tumor cell lysates to find novel autoantigens.\nProminent among these, serologic expression cloning (SEREX) was developed ten years ago\n(Figure 4). This method uses patient sera to probe blotted phage expression libraries derived\nfrom tumor cells and has resulted in the identification of over 2000 autoantigens recognized\nby patient sera5, 57-79. Through a database established by the SEREX collaborative group,\nsequences from over 1390 genes have been deposited 80 and\nhttp://www2.licr.org/CancerImmunomeDB/. As SEREX relies on immunoblotting, these\nantigens are limited to linear epitopes and those gene products that can be expressed in bacteria.\nAs with the autoimmune targets, most of the antigens identified with SEREX are nuclear or\nintracellular antigens, many of which are upregulated, mutated, or specifically expressed in\ntumors. However, altered degradative pathways, such as sensitivity to granzyme B cleavage,\nhas been shown to have a marked impact on the immunogenicity of a hepatocellular tumor\nantigen6. In addition, frameshifts and alternative reading frames, can result in autoantibodies.\nUsing colon cancer sera, the antigen ADO34 was identified with a frameshift insertion81, and\nCDX2 had a frameshift mutation82. Antibodies to the OGFr protein can recognize an\nalternative-reading frame of the molecule79.\nThe SEREX method identifies autoantigens whether or not they are informative. At present\nonly a handful of antigens have undergone comparison testing, in part because of the technical\nchallenges associated with historical assays for validating autoantigens.\nPhage display: Combinatorial phage display is a new and attractive approach for the\nidentification of tumor antigens. By expressing antigens as fusions with phage proteins,\nantigens can be detected more rapidly and with less serum. Rather than immunoblotting as\nwith SEREX, phage display relies on successive rounds of immunoprecipitation of phage\nlibraries using patient serum. This powerful approach has been used in the field of\nrheumatology to identify autoantigens in multiple sclerosis, lupus, rheumatoid arthritis and\nothers83-85. For tumor antigens, one study showed that only 4 of 13 antigens identified with\nprostate cancer sera had already been identified by SEREX technologies, demonstrating that\nnovel tumor antigens may be identified with different screening approaches 86. Phage display\nhas been successfully used to identify tumor autoantigens in cancer86-89. However, antigens\nexpressed by phage display may not be in native conformation, do not have mammalian post-\ntranslational modifications, and each positive phage clone must be individually sequenced.\nYeast display systems are also being investigated90.\nValidating autoantibodies—To be useful as biomarkers, the tumor antigens must\ndistinguish between individuals with and without cancer. Thus once a candidate autoantigen\nis identified, comparisons must be made regarding the responses of individuals from both\ngroups. The measurement of an autoantibody response is most often accomplished using the\nenzyme linked immunosorbent assay (ELISA), in which purified protein (the autoantigen) is\nimmobilized in the wells of a microtiter dish and exposed to serum samples (Fig. 4). After\nadequate washing, any bound antibody is revealed using standard anti-human antibody\nreagents linked to enzymatic markers. If the response is very strong, it may be useful to perform\nserial dilutions of the serum to determine the titer of the antibody. By comparing multiple\npatient and normal donor samples it is possible to determine the frequency of response. For\nexample, antibodies to dsDNA are present in only 40-60% of SLE patients, but antibodies to\nhistones are present in >95% of patients with drug-induced SLE. To compute the sensitivity\nand specificity of the test, a threshold value must be set, such that the test is considered positive\nif the antibody response achieves that value or above. Typically, this threshold is either 2 or 3\nstandard deviations above the average response in the normal population.\nThe development of clinical ELISAs require the use of recombinant proteins or peptides, which\nmay be complicated by batch-to-batch variations, loss of conformational epitopes or lack of\nmammalian post-translational processing91, 92. Antigenic epitopes from proteins such as Ro,\nLa, SmB, SmD, Scl-70, can be linear epitopes of 10-22 amino acids conformational, or cryptic.\nFurthermore, epitopes may be naturally post-translationally modified by multiple means,\nincluding phosphorylation, glycosylation, acetylation, and methylation (SmD1, SmD3,\nfibrillarin, nucleolin)93, 94, all of which must be considered prior to the development of a\nclinical diagnostic assay. Of the many candidate antigens identified so far, only a handful have\ngone through the validation process to determine their sensitivity and specificity as cancer\npredictors, in part because of the challenges associated with setting up ELISAs.\nProteomic Approaches for the Identification of Tumor Antigens\nThe Power of Multiplexing—With all of their potential advantages, the Achilles heel of\nautoantibodies as biomarkers is their sensitivity, i.e., the fraction of true positives that have a\npositive test. In large part, this may reflect the nature of cancer. Unlike infections in which the\nvast majority of patients respond to the same immunodominant antigens, even cancers of the\nsame type represent a mix of different biological subtypes. Thus, patients are likely to mount\nimmune responses to different tumor antigens, and no single antigen is likely to detect all\ncancers. Typically, only 15-20% of patients demonstrate a response to any given antigen.\nHowever, proteomics may hold the key to success because it provides the means to multiplex.\nBy linking the responses to several antigens together, the sensitivity and specificity of the test\nincreases considerably, presumably because the chance that a patient will respond to at least\none of the antigens is increased76, 95-102. Koziol et al demonstrated that a panel of just seven\ntumor-associated antigens (myc, cyclin B1, p62, IMP-1, Koc, p53, and survivin) could be used\nto segregate sera from patients with cancers from healthy donor sera95. In that study, sera from\nbreast, colorectal, gastric, hepatocellular, lung, and prostate cancers were distinguished with\nsensitivities from 0.77-0.92 and specificities from 0.85-0.91 using no more than 3 tumor\nantigens for any cancer cohort. Scanlan et. al. studied the serologic responses to 13 defined\ntumor antigens in sera from colon cancer patients. Of these, 46% of patients, but not healthy\ndonors, had antibodies to at least 1 of these antigens103.\nThese results demonstrate the potential power of simultaneously analyzing multiple\nautoantigens. It provides a greater likelihood of detecting and diagnosing the appropriate cancer\npresumably because the sensitivity will improve (increased chance that the patient will have a\nresponse to at least one of the tested antigens) and the specificity improves (positive responses\non multiple antigens increases certainty). By coupling this with the appropriate statistical\nmodeling, it is likely that patterns or weighted schemes of antibody responses, rather than\nindividual responses, will have the greatest utility in clinical assays. The early experiments\nabove were performed by using recursive partitioning. Given the technical challenges inherent\nin that method for large numbers of antigens, newer technologies are needed for high\nthroughput analysis. To this end, recent development of several proteomic technologies have\nbeen adapted for tumor antigen identification and biomarker development. These include\nprobing fractionated tumor cell lysates, phage display, and protein microarrays (See Table 1).\nProbing fractionated tumor cell lysate blots with serum\nIn this approach, tumor cell lysates are fractionated to separate the various protein species and\nblotted onto a membrane or microarray and then probed with patient or control sera. Response\npatterns are then analyzed to differentiate between the two (Fig. 4). This has been used to\ndemonstrate antibodies in the sera of lung cancer patients96, 104. A similar approach was used\nto distinguish sera from prostate cancer patients from healthy donor sera with 98%\naccuracy105. The development of automated separation systems using liquid-based\nchromatography with subsequent microarray spotting of lysate fractions enhances the\nreproducibility and speed of immunogenic fraction identification. Advantages of this approach\ninclude the ability to query a large fraction of the tumor cell proteome and the preservation of\nthe post translational modifications of proteins. Moreover, when this is executed using a\nmicroarray format, only small amounts of serum are required. However, the reproducibility\ndemanded by a clinical assay requires an identified and validated antigen. Thus, as with other\nchromatographic separation systems, the identity of the proteins in the fraction must eventually\nbe determined, presumably using sensitive mass spectrometric analysis. As each lysate fraction\nmay contain many proteins at very different concentrations, of which the minor component\nmay be the immunogen, antigen identification can be difficult.\nProbing Candidate Antigen Arrays\nKnown or predicted tumor antigens may be directly spotted on microarrays and probed with\nhuman sera, with the advantage of reproducibility and more rapid screening of small amounts\nof sera (Figure 4). In this case, the identity of a single protein at each feature on the array is\nknown a priori. This approach has been successfully used to screen autoimmune patient\nsera100, 101. A powerful advantage of this approach is that it offers the ability to screen for\ninformative autoantigens by comparing responses of patients to controls. Because each feature\nof the array represents a single protein, whose identity is known, it is possible to calculate\nsensitivity and specificity values for the response to each candidate antigen during the screening\nphase. This allows the rapid determination of which antigens are informative for cancer\ndetection. Ultimately, focused arrays of tumor antigens previously identified by other means\nmay have the greatest utility for immunodiagnostics.\nTheir theoretical advantages notwithstanding, protein microarrays have still not found\nwidespread use, in part because producing them is challenging. Historically it has required the\nhigh-throughput production and purification of protein, which then must be spotted on the\narrays. Once printed, concerns remain about the shelf life of proteins on the arrays. Recently,\nprogrammable protein microarrays that consist of anchored cDNA's and in situ transcription\nand translation of tagged proteins have been shown to result in highly reproducible protein\narrays without the requirement of heterologous protein expression and purification106. As the\nproteins are synthesized at the time of the assay, shelf life is not an issue and the proteins are\ntranslated with a mammalian reticulocyte lysate, which reproduces the folding and some of the\npost-translational processing of antigenic epitopes. Unlike tumor lysate fractionation, antigen\nidentification is automatic, although epitopes that depend on abnormal processing by tumor\ncells would not be identified, and transmembrane proteins have not yet been tested in this\nsystem. Like most antigen presentation methods (SEREX, ELISA, phage display, etc.), these\nprogrammable arrays require access to cDNAs to express the antigens. Fortunately, nearly all\nknown tumor antigen genes are available because they were identified with methods that\ninvolved cDNA identification. Moreover, libraries of cDNA clones representing most of the\nhuman proteome are increasingly available.\nProteomics and the Identification of T cell Antigens\nIn contrast to the rapid identification of B cell antigens, the identification of T cell antigens\nremains much more difficult. Since T lymphocytes specifically recognize peptides derived\nfrom protein antigens, the isolation and confirmation of these antigens have traditionally relied\non laborious T cell isolation and cloning. Alternatively, direct isolation and sequencing of\nMHC-associated peptides from cells has been limited by the overall low concentration of\nspecific peptide bound to MHC molecules, and the highly polymorphic nature of the MHC\nmolecules themselves. These challenges are being overcome by advances in high-throughput\nMHC-peptide binding studies, the bioinformatics of epitope prediction, and in particular, mass\nspectrometry.\nWith high-throughput peptide synthesis, direct MHC-peptide binding assays may now be\nperformed on peptides that span target antigens, or using combinatorial peptide libraries107.\nDetailed analysis of peptide binding to common HLA alleles has resulted in the development\nof bioinformatics tools for epitope prediction108, 109. Further limitations imposed by\nproteasomal cleavage patterns can enhance prediction of peptide epitopes. Multiple prediction\nmodels are available, primarily for MHC Class I-binding peptides, due to the more restrictive\nnature of the peptide binding groove. These systems include BIMAS110, SYFPEITHI111,\nMHCPEP112, and RANKPEP113, among others.\nAdvances in mass spectrometry have also greatly aided antigen identification. Original studies\nof direct peptide sequencing from purified MHC molecules from cell lines114-116\ndemonstrated that the majority of peptides within the MHC grooves are derived from self-\nproteins, but required billions of cells for the source of antigen. More recent approaches to\nidentify tumor-specific T cell epitopes have used capillary-scale chromatography and tandem\nmass spectrometry (Figure 5) (nanoLC-MS/MS)117-120. To enhance sensitivity, nanospray\nquadrupole-TOF combined with Poisson algorithms121 to specifically evaluate the likelihood\nof predicted peptides in unfractionated mixtures has markedly increased the sensitivity of the\nassay to the range of 1 copy number of peptide/cell in a murine influenza viral model, but is\nlimited to predicted MHC-binding peptide epitopes.\nSummary\nThe development of proteomic-based methods of cell lysate fractionation, phage display,\nprotein microarrays, bioinformatics, and mass spectrometry is resulting in the rapid\nidentification of both B and T cell tumor antigens. These have the potential for clinical\ndiagnosis, identifying targets for immunotherapies, monitoring disease response, and to\nunderstand the breadth, scope, and impact of the immune response to cancer. As each new\ntechnology leads to the discovery of novel antigenic targets, systematic approaches of target\nvalidation and assessment of clinical applications, especially in the area of biomarkers and\ndiagnostic testing, will need to be developed.\nAcknowledgements\n1. Pavlenko M, Roos AK, Lundqvist A, Palmborg A, Miller AM, Ozenci V, Bergman B, Egevad L,\nHellstrom M, Kiessling R, Masucci G, Wersall P, Nilsson S, Pisa P. A phase I trial of DNA vaccination\nwith a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate\ncancer. Br J Cancer 2004;91:688–694. [PubMed: 15280930]\n2. Vonderheide RH, Hahn WC, Schultze JL, Nadler LM. The telomerase catalytic subunit is a widely\nexpressed tumor-associated antigen recognized by cytotoxic T lymphocytes. Immunity 1999;10:673–\n679. [PubMed: 10403642]\n3. Soussi T. p53 Antibodies in the sera of patients with various types of cancer: a review. Cancer Res\n4. Schubert U, Anton LC, Gibbs J, Norbury CC, Yewdell JW, Bennink JR. Rapid degradation of a large\nfraction of newly synthesized proteins by proteasomes. Nature 2000;404:770–774. [PubMed:\n10783891]\n5. Chen YT, Scanlan MJ, Sahin U, Tureci O, Gure AO, Tsang S, Williamson B, Stockert E, Pfreundschuh\nM, Old LJ. A testicular antigen aberrantly expressed in human cancers detected by autologous antibody\nscreening. Proc Natl Acad Sci U S A 1997;94:1914–1918. [PubMed: 9050879]\n6. Ulanet DB, Torbenson M, Dang CV, Casciola-Rosen L, Rosen A. Unique conformation of cancer\nautoantigen B23 in hepatoma: a mechanism for specificity in the autoimmune response. Proc Natl\nAcad Sci U S A 2003;100:12361–12366. [PubMed: 14519847]\n7. von Mensdorff-Pouilly S, Petrakou E, Kenemans P, van Uffelen K, Verstraeten AA, Snijdewint FG,\nvan Kamp GJ, Schol DJ, Reis CA, Price MR, Livingston PO, Hilgers J. Reactivity of natural and\ninduced human antibodies to MUC1 mucin with MUC1 peptides and n-acetylgalactosamine (GalNAc)\npeptides. Int J Cancer 2000;86:702–712. [PubMed: 10797294]\n8. Routsias JG, Tzioufas AG, Moutsopoulos HM. The clinical value of intracellular autoantigens B-cell\nepitopes in systemic rheumatic diseases. Clin Chim Acta 2004;340:1–25. [PubMed: 14734193]\n9. Lernmark A. Autoimmune diseases: are markers ready for prediction? J Clin Invest 2001;108:1091–\n10. Plotz PH. The autoantibody repertoire: searching for order. Nat Rev Immunol 2003;3:73–78.\n11. Brendel V, Dohlman J, Blaisdell BE, Karlin S. Very long charge runs in systemic lupus erythematosus-\nassociated autoantigens. Proc Natl Acad Sci U S A 1991;88:1536–1540. [PubMed: 1996354]\n12. Utz PJ, Hottelet M, Le TM, Kim SJ, Geiger ME, van Venrooij WJ, Anderson P. The 72-kDa\ncomponent of signal recognition particle is cleaved during apoptosis. J Biol Chem 1998;273:35362–\n35370. [PubMed: 9857079]\n13. Casciola-Rosen L, Andrade F, Ulanet D, Wong WB, Rosen A. Cleavage by granzyme B is strongly\npredictive of autoantigen status: implications for initiation of autoimmunity. J Exp Med\n1999;190:815–826. [PubMed: 10499920]\n14. Raben N, Nichols R, Dohlman J, McPhie P, Sridhar V, Hyde C, Leff R, Plotz P. A motif in human\nhistidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-\ncoil that is essential for enzymatic activity and contains the major autoantigenic epitope. J Biol Chem\n1994;269:24277–24283. [PubMed: 7523371]\n15. DeLeo AB, Jay G, Appella E, Dubois GC, Law LW, Old LJ. Detection of a transformation-related\nantigen in chemically induced sarcomas and other transformed cells of the mouse. Proc Natl Acad\nSci U S A 1979;76:2420–2424. [PubMed: 221923]\n16. Casey G, Lopez ME, Ramos JC, Plummer SJ, Arboleda MJ, Shaughnessy M, Karlan B, Slamon DJ.\nDNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to\ndetect all known p53 alterations in human malignancies. Oncogene 1996;13:1971–1981. [PubMed:\n8934544]\n17. Winter SF, Minna JD, Johnson BE, Takahashi T, Gazdar AF, Carbone DP. Development of antibodies\nagainst p53 in lung cancer patients appears to be dependent on the type of p53 mutation. Cancer Res\n1992;52:4168–4174. [PubMed: 1322237]\n18. Dowell SP, Wilson PO, Derias NW, Lane DP, Hall PA. Clinical utility of the immunocytochemical\ndetection of p53 protein in cytological specimens. Cancer Res 1994;54:2914–2918. [PubMed:\n8187077]\n19. Vennegoor CJ, Nijman HW, Drijfhout JW, Vernie L, Verstraeten RA, von Mensdorff-Pouilly S,\nHilgers J, Verheijen RH, Kast WM, Melief CJ, Kenemans P. Autoantibodies to p53 in ovarian cancer\npatients and healthy women: a comparison between whole p53 protein and 18-mer peptides for\nscreening purposes. Cancer Lett 1997;116:93–101. [PubMed: 9177463]\n20. Angelopoulou K, Yu H, Bharaj B, Giai M, Diamandis EP. p53 gene mutation, tumor p53 protein\noverexpression, and serum p53 autoantibody generation in patients with breast cancer. Clin Biochem\n2000;33:53–62. [PubMed: 10693987]\n21. Angelopoulou K, Stratis M, Diamandis EP. Humoral immune response against p53 protein in patients\nwith colorectal carcinoma. Int J Cancer 1997;70:46–51. [PubMed: 8985089]\n22. Rainov NG, Dobberstein KU, Fittkau M, Bahn H, Holzhausen HJ, Gantchev L, Burkert W. Absence\nof p53 autoantibodies in sera from glioma patients. Clin Cancer Res 1995;1:775–781. [PubMed:\n9816045]\n23. Weller M, Bornemann A, Stander M, Schabet M, Dichgans J, Meyermann R. Humoral immune\nresponse to p53 in malignant glioma. J Neurol 1998;245:169–172. [PubMed: 9553848]\n24. Zorn E, Miklos DB, Floyd BH, Mattes-Ritz A, Guo L, Soiffer RJ, Antin JH, Ritz J. Minor\nHistocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem\nCell Transplantation. J Exp Med 2004;199:1133–1142. [PubMed: 15096539]\n25. Schmollinger JC, Vonderheide RH, Hoar KM, Maecker B, Schultze JL, Hodi FS, Soiffer RJ, Jung\nK, Kuroda MJ, Letvin NL, Greenfield EA, Mihm M, Kutok JL, Dranoff G. Melanoma inhibitor of\napoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction. Proc Natl Acad Sci\nU S A 2003;100:3398–3403. [PubMed: 12626761]\n26. Molldrem JJ, Lee PP, Wang C, Felio K, Kantarjian HM, Champlin RE, Davis MM. Evidence that\nspecific T lymphocytes may participate in the elimination of chronic myelogenous leukemia. Nat\nMed 2000;6:1018–1023. [PubMed: 10973322]\n27. Hodi FS, Schmollinger JC, Soiffer RJ, Salgia R, Lynch T, Ritz J, Alyea EP, Yang J, Neuberg D,\nMihm M, Dranoff G. ATP6S1 elicits potent humoral responses associated with immune-mediated\ntumor destruction. Proc Natl Acad Sci U S A 2002;99:6919–6924. [PubMed: 11983866]\n28. Miklos DB, Kim HT, Zorn E, Hochberg EP, Guo L, Mattes-Ritz A, Viatte S, Soiffer RJ, Antin JH,\nRitz J. Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem\ncell transplantation and in healthy female donors. Blood 2004;103:353–359. [PubMed: 14512314]\n29. Bellucci R, Wu CJ, Chiaretti S, Weller E, Davies FE, Alyea EP, Dranoff G, Anderson KC, Munshi\nNC, Ritz J. Complete response to donor lymphocyte infusion in multiple myeloma is associated with\nantibody responses to highly expressed antigens. Blood 2004;103:656–663. [PubMed: 14563636]\n30. Yang XF, Wu CJ, Chen L, Alyea EP, Canning C, Kantoff P, Soiffer RJ, Dranoff G, Ritz J. CML28\nis a broadly immunogenic antigen, which is overexpressed in tumor cells. Cancer Res 2002;62:5517–\n5522. [PubMed: 12359762]\n31. Dranoff G, Jaffee E, Lazenby A, Golumbek P, Levitsky H, Brose K, Jackson V, Hamada H, Pardoll\nD, Mulligan RC. Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-\nmacrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor\nimmunity. Proc Natl Acad Sci U S A 1993;90:3539–3543. [PubMed: 8097319]\n32. Huang AY, Golumbek P, Ahmadzadeh M, Jaffee E, Pardoll D, Levitsky H. Role of bone marrow-\nderived cells in presenting MHC class I-restricted tumor antigens. Science 1994;264:961–965.\n[PubMed: 7513904]\n33. Hung K, Hayashi R, Lafond-Walker A, Lowenstein C, Pardoll D, Levitsky H. The central role of\nCD4(+) T cells in the antitumor immune response. J Exp Med 1998;188:2357–2368. [PubMed:\n9858522]\n34. Gillessen S, Naumov YN, Nieuwenhuis EE, Exley MA, Lee FS, Mach N, Luster AD, Blumberg RS,\nTaniguchi M, Balk SP, Strominger JL, Dranoff G, Wilson SB. CD1d-restricted T cells regulate\ndendritic cell function and antitumor immunity in a granulocyte-macrophage colony-stimulating\nfactor-dependent fashion. Proc Natl Acad Sci U S A 2003;100:8874–8879. [PubMed: 12847287]\n35. Heslop HE, Stevenson FK, Molldrem JJ. Immunotherapy of hematologic malignancy. Hematology\n(Am Soc Hematol Educ Program) 2003:331–349. [PubMed: 14633789]\n36. Davis ID, Chen W, Jackson H, Parente P, Shackleton M, Hopkins W, Chen Q, Dimopoulos N, Luke\nT, Murphy R, Scott AM, Maraskovsky E, McArthur G, MacGregor D, Sturrock S, Tai TY, Green S,\nCuthbertson A, Maher D, Miloradovic L, Mitchell SV, Ritter G, Jungbluth AA, Chen YT, Gnjatic\nS, Hoffman EW, Old LJ, Cebon JS. Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant\ninduces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans. Proc Natl\nAcad Sci U S A 2004;101:10697–10702. [PubMed: 15252201]\n37. DiFronzo LA, Gupta RK, Essner R, Foshag LJ, O'Day SJ, Wanek LA, Stern SL, Morton DL. Enhanced\nhumoral immune response correlates with improved disease-free and overall survival in American\nJoint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine. J Clin\nOncol 2002;20:3242–3248. [PubMed: 12149297]\n38. Ehlken H, Schadendorf D, Eichmuller S. Humoral immune response against melanoma antigens\ninduced by vaccination with cytokine gene-modified autologous tumor cells. Int J Cancer\n2004;108:307–313. [PubMed: 14639620]\n39. Small, EJ.; S, PF.; Higano, C.; Neumanaitis, J.; Valone, F.; Herschberg, RM. Immunotherapy\n(APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3\nrandomized placebo-controlled trial; 2005 ASCO Prostate Cancer Symposium; 2005.\n40. Cancer Facts and Figures. 2004.\n41. Srinivas PR, Kramer BS, Srivastava S. Trends in biomarker research for cancer detection. Lancet\nOncol 2001;2:698–704. [PubMed: 11902541]\n42. Gold P, Freedman SO. Specific carcinoembryonic antigens of the human digestive system. J Exp\nMed 1965;122:467–481. [PubMed: 4953873]\n43. Wang MC, Valenzuela LA, Murphy GP, Chu TM. Purification of a human prostate specific antigen.\nInvest Urol 1979;17:159–163. [PubMed: 89106]\n44. Niloff JM, Knapp RC, Schaetzl E, Reynolds C, Bast RC Jr. CA125 antigen levels in obstetric and\ngynecologic patients. Obstet Gynecol 1984;64:703–707. [PubMed: 6208522]\n45. Petricoin E, Wulfkuhle J, Espina V, Liotta LA. Clinical proteomics: revolutionizing disease detection\nand patient tailoring therapy. J Proteome Res 2004;3:209–217. [PubMed: 15113096]\n46. Trivers GE, Cawley HL, DeBenedetti VM, Hollstein M, Marion MJ, Bennett WP, Hoover ML, Prives\nCC, Tamburro CC, Harris CC. Anti-p53 antibodies in sera of workers occupationally exposed to\nvinyl chloride. J Natl Cancer Inst 1995;87:1400–1407. [PubMed: 7658501]\n47. Trivers GE, De Benedetti VM, Cawley HL, Caron G, Harrington AM, Bennett WP, Jett JR, Colby\nTV, Tazelaar H, Pairolero P, Miller RD, Harris CC. Anti-p53 antibodies in sera from patients with\nchronic obstructive pulmonary disease can predate a diagnosis of cancer. Clin Cancer Res\n1996;2:1767–1775. [PubMed: 9816128]\n48. Suzuki H, Graziano DF, McKolanis J, Finn OJ. T cell-dependent antibody responses against\naberrantly expressed cyclin B1 protein in patients with cancer and premalignant disease. Clin Cancer\nRes 2005;11:1521–1526. [PubMed: 15746055]\n49. Rodolfo M, Luksch R, Stockert E, Chen YT, Collini P, Ranzani T, Lombardo C, Dalerba P, Rivoltini\nL, Arienti F, Fossati-Bellani F, Old LJ, Parmiani G, Castelli C. Antigen-specific immunity in\nneuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen. Cancer Res\n2003;63:6948–6955. [PubMed: 14583496]\n50. Litvak DA, Gupta RK, Yee R, Wanek LA, Ye W, Morton DL. Endogenous immune response to early-\nand intermediate-stage melanoma is correlated with outcomes and is independent of locoregional\nrelapse and standard prognostic factors. J Am Coll Surg 2004;198:27–35. [PubMed: 14698308]\n51. Disis ML, Pupa SM, Gralow JR, Dittadi R, Menard S, Cheever MA. High-titer HER-2/neu protein-\nspecific antibody can be detected in patients with early-stage breast cancer. J Clin Oncol\n1997;15:3363–3367. [PubMed: 9363867]\n52. Disis ML, Knutson KL, Schiffman K, Rinn K, McNeel DG. Pre-existent immunity to the HER-2/neu\noncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer. Breast\nCancer Res Treat 2000;62:245–252. [PubMed: 11072789]\n53. Jager E, Stockert E, Zidianakis Z, Chen YT, Karbach J, Jager D, Arand M, Ritter G, Old LJ, Knuth\nA. Humoral immune responses of cancer patients against “Cancer-Testis” antigen NY-ESO-1:\ncorrelation with clinical events. Int J Cancer 1999;84:506–510. [PubMed: 10502728]\n54. Regele S, Vogl FD, Kohler T, Kreienberg R, Runnebaum IB. p53 autoantibodies can be indicative\nof the development of breast cancer relapse. Anticancer Res 2003;23:761–764. [PubMed: 12680180]\n55. Takeda A, Shimada H, Nakajima K, Imaseki H, Suzuki T, Asano T, Ochiai T, Isono K. Monitoring\nof p53 autoantibodies after resection of colorectal cancer: relationship to operative curability. Eur J\nSurg 2001;167:50–53. [PubMed: 11213822]\n56. Kelley MC, Gupta RK, Hsueh EC, Yee R, Stern S, Morton DL. Tumor-associated antigen TA90\nimmune complex assay predicts recurrence and survival after surgical treatment of stage I-III\nmelanoma. J Clin Oncol 2001;19:1176–1182. [PubMed: 11181684]\n57. Tajima K, Obata Y, Tamaki H, Yoshida M, Chen YT, Scanlan MJ, Old LJ, Kuwano H, Takahashi T,\nMitsudomi T. Expression of cancer/testis (CT) antigens in lung cancer. Lung Cancer 2003;42:23–\n33. [PubMed: 14512184]\n58. Lee SY, Obata Y, Yoshida M, Stockert E, Williamson B, Jungbluth AA, Chen YT, Old LJ, Scanlan\nMJ. Immunomic analysis of human sarcoma. Proc Natl Acad Sci U S A 2003;100:2651–2656.\n[PubMed: 12601173]\n59. Ayyoub M, Hesdorffer CS, Montes M, Merlo A, Speiser D, Rimoldi D, Cerottini JC, Ritter G, Scanlan\nM, Old LJ, Valmori D. An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in\nassociation with HLA-DR. J Clin Invest 2004;113:1225–1233. [PubMed: 15085202]\n60. Sugita Y, Wada H, Fujita S, Nakata T, Sato S, Noguchi Y, Jungbluth AA, Yamaguchi M, Chen YT,\nStockert E, Gnjatic S, Williamson B, Scanlan MJ, Ono T, Sakita I, Yasui M, Miyoshi Y, Tamaki Y,\nMatsuura N, Noguchi S, Old LJ, Nakayama E, Monden M. NY-ESO-1 expression and\nimmunogenicity in malignant and benign breast tumors. Cancer Res 2004;64:2199–2204. [PubMed:\n15026363]\n61. Odunsi K, Jungbluth AA, Stockert E, Qian F, Gnjatic S, Tammela J, Intengan M, Beck A, Keitz B,\nSantiago D, Williamson B, Scanlan MJ, Ritter G, Chen YT, Driscoll D, Sood A, Lele S, Old LJ. NY-\nESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial\novarian cancer. Cancer Res 2003;63:6076–6083. [PubMed: 14522938]\n62. Scanlan MJ, Gout I, Gordon CM, Williamson B, Stockert E, Gure AO, Jager D, Chen YT, Mackay\nA, O'Hare MJ, Old LJ. Humoral immunity to human breast cancer: antigen definition and quantitative\nanalysis of mRNA expression. Cancer Immun 2001;1:4. [PubMed: 12747765]\n63. Jager D, Unkelbach M, Frei C, Bert F, Scanlan MJ, Jager E, Old LJ, Chen YT, Knuth A. Identification\nof tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by\nserological screening of a testicular library with breast cancer serum. Cancer Immun 2002;2:5.\n[PubMed: 12747750]\n64. Koroleva EP, Lagarkova MA, Mesheryakov AA, Scanlan MJ, Old LJ, Nedospasov SA, Kuprash DV.\nSerological identification of antigens associated with renal cell carcinoma. Russ J Immunol\n2002;7:229–238. [PubMed: 12674932]\n65. Scanlan MJ, Gure AO, Jungbluth AA, Old LJ, Chen YT. Cancer/testis antigens: an expanding family\nof targets for cancer immunotherapy. Immunol Rev 2002;188:22–32. [PubMed: 12445278]\n66. Forti S, Scanlan MJ, Invernizzi A, Castiglioni F, Pupa S, Agresti R, Fontanelli R, Morelli D, Old LJ,\nPupa SM, Menard S. Identification of breast cancer-restricted antigens by antibody screening of\nSKBR3 cDNA library using a preselected patient's serum. Breast Cancer Res Treat 2002;73:245–\n256. [PubMed: 12160330]\n67. Scanlan MJ, Gordon CM, Williamson B, Lee SY, Chen YT, Stockert E, Jungbluth A, Ritter G, Jager\nD, Jager E, Knuth A, Old LJ. Identification of cancer/testis genes by database mining and mRNA\nexpression analysis. Int J Cancer 2002;98:485–492. [PubMed: 11920606]\n68. Jager D, Stockert E, Gure AO, Scanlan MJ, Karbach J, Jager E, Knuth A, Old LJ, Chen YT.\nIdentification of a tissue-specific putative transcription factor in breast tissue by serological screening\nof a breast cancer library. Cancer Res 2001;61:2055–2061. [PubMed: 11280766]\n69. Gure AO, Stockert E, Scanlan MJ, Keresztes RS, Jager D, Altorki NK, Old LJ, Chen YT. Serological\nidentification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung\ncancer. Proc Natl Acad Sci U S A 2000;97:4198–4203. [PubMed: 10760287]\n70. Scanlan MJ, Altorki NK, Gure AO, Williamson B, Jungbluth A, Chen YT, Old LJ. Expression of\ncancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a\nnew CT gene, CT9. Cancer Lett 2000;150:155–164. [PubMed: 10704737]\n71. Gure AO, Stockert E, Arden KC, Boyer AD, Viars CS, Scanlan MJ, Old LJ, Chen YT. CT10: a new\ncancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-\ndifference analysis. Int J Cancer 2000;85:726–732. [PubMed: 10699956]\n72. Jager D, Stockert E, Scanlan MJ, Gure AO, Jager E, Knuth A, Old LJ, Chen YT. Cancer-testis antigens\nand ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-\nspecific ING1 transcripts and a homologue gene. Cancer Res 1999;59:6197–6204. [PubMed:\n10626813]\n73. Scanlan MJ, Gordan JD, Williamson B, Stockert E, Bander NH, Jongeneel V, Gure AO, Jager D,\nJager E, Knuth A, Chen YT, Old LJ. Antigens recognized by autologous antibody in patients with\nrenal-cell carcinoma. Int J Cancer 1999;83:456–464. [PubMed: 10508479]\n74. Scanlan MJ, Williamson B, Jungbluth A, Stockert E, Arden KC, Viars CS, Gure AO, Gordan JD,\nChen YT, Old LJ. Isoforms of the human PDZ-73 protein exhibit differential tissue expression.\nBiochim Biophys Acta 1999;1445:39–52. [PubMed: 10209257]\n75. Scanlan MJ, Chen YT, Williamson B, Gure AO, Stockert E, Gordan JD, Tureci O, Sahin U,\nPfreundschuh M, Old LJ. Characterization of human colon cancer antigens recognized by autologous\nantibodies. Int J Cancer 1998;76:652–658. [PubMed: 9610721]\n76. Stockert E, Jager E, Chen YT, Scanlan MJ, Gout I, Karbach J, Arand M, Knuth A, Old LJ. A survey\nof the humoral immune response of cancer patients to a panel of human tumor antigens. J Exp Med\n1998;187:1349–1354. [PubMed: 9547346]\n77. Gure AO, Altorki NK, Stockert E, Scanlan MJ, Old LJ, Chen YT. Human lung cancer antigens\nrecognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor\ngene locus on chromosome 3p21.3. Cancer Res 1998;58:1034–1041. [PubMed: 9500467]\n78. Gure AO, Tureci O, Sahin U, Tsang S, Scanlan MJ, Jager E, Knuth A, Pfreundschuh M, Old LJ, Chen\nYT. SSX: a multigene family with several members transcribed in normal testis and human cancer.\nInt J Cancer 1997;72:965–971. [PubMed: 9378559]\n79. Mollick JA, Hodi FS, Soiffer RJ, Nadler LM, Dranoff G. MUC1-like tandem repeat proteins are\nbroadly immunogenic in cancer patients. Cancer Immun 2003;3:3. [PubMed: 12747745]\n80. Chen, YT.; S, M.; Obata, Y.; Old, LJ. Identification of human tumor antigens by serological expression\ncloning. In: SA, R., editor. Principles and practice of the biologic therapy of cancer. 3rd. Lippincott\nWilliams & Wilkins; Philadelphia: 2000. p. 557-570.\n81. Line A, Slucka Z, Stengrevics A, Silina K, Li G, Rees RC. Characterisation of tumour-associated\nantigens in colon cancer. Cancer Immunol Immunother 2002;51:574–582. [PubMed: 12384809]\n82. Ishikawa T, Fujita T, Suzuki Y, Okabe S, Yuasa Y, Iwai T, Kawakami Y. Tumor-specific\nimmunological recognition of frameshift-mutated peptides in colon cancer with microsatellite\ninstability. Cancer Res 2003;63:5564–5572. [PubMed: 14500396]\n83. Kemp EH, Herd LM, Waterman EA, Wilson AG, Weetman AP, Watson PP. Immunoscreening of\nphage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease.\nBiochem Biophys Res Commun 2002;298:169–177. [PubMed: 12379236]\n84. Jensen LB, Riise E, Nielsen LK, Dziegiel M, Fugger L, Engberg J. Efficient purification of unique\nantibodies using peptide affinity-matrix columns. J Immunol Methods 2004;284:45–54. [PubMed:\n14736416]\n85. Sioud M, Hansen M, Dybwad A. Profiling the immune responses in patient sera with peptide and\ncDNA display libraries. Int J Mol Med 2000;6:123–128. [PubMed: 10891554]\n86. Fossa A, Alsoe L, Crameri R, Funderud S, Gaudernack G, Smeland EB. Serological cloning of cancer/\ntestis antigens expressed in prostate cancer using cDNA phage surface display. Cancer Immunol\nImmunother 2004;53:431–438. [PubMed: 14747957]\n87. Sioud M, Hansen MH. Profiling the immune response in patients with breast cancer by phage-\ndisplayed cDNA libraries. Eur J Immunol 2001;31:716–725. [PubMed: 11241275]\n88. Minenkova O, Pucci A, Pavoni E, De Tomassi A, Fortugno P, Gargano N, Cianfriglia M, Barca S,\nDe Placido S, Martignetti A, Felici F, Cortese R, Monaci P. Identification of tumor-associated\nantigens by screening phage-displayed human cDNA libraries with sera from tumor patients. Int J\nCancer 2003;106:534–544. [PubMed: 12845649]\n89. Fernandez-Madrid F, Tang N, Alansari H, Granda JL, Tait L, Amirikia KC, Moroianu M, Wang X,\nKarvonen RL. Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast\nCancer. Cancer Res 2004;64:5089–5096. [PubMed: 15289310]\n90. Mischo A, Wadle A, Watzig K, Jager D, Stockert E, Santiago D, Ritter G, Regitz E, Jager E, Knuth\nA, Old L, Pfreundschuh M, Renner C. Recombinant antigen expression on yeast surface (RAYS) for\nthe detection of serological immune responses in cancer patients. Cancer Immun 2003;3:5. [PubMed:\n12828452]\n91. Yan SC, Grinnell BW, Wold F. Post-translational modifications of proteins: some problems left to\nsolve. Trends Biochem Sci 1989;14:264–268. [PubMed: 2672445]\n92. St Clair EW, Kenan D, Burch JA Jr, Keene JD, Pisetsky DS. The fine specificity of anti-La antibodies\ninduced in mice by immunization with recombinant human La autoantigen. J Immunol\n1990;144:3868–3876. [PubMed: 1692063]\n93. Doyle HA, Mamula MJ. Post-translational protein modifications in antigen recognition and\nautoimmunity. Trends Immunol 2001;22:443–449. [PubMed: 11473834]\n94. Doyle HA, Mamula MJ. Posttranslational protein modifications: new flavors in the menu of\nautoantigens. Curr Opin Rheumatol 2002;14:244–249. [PubMed: 11981321]\n95. Koziol JA, Zhang JY, Casiano CA, Peng XX, Shi FD, Feng AC, Chan EK, Tan EM. Recursive\npartitioning as an approach to selection of immune markers for tumor diagnosis. Clin Cancer Res\n2003;9:5120–5126. [PubMed: 14613989]\n96. Zhang JY, Casiano CA, Peng XX, Koziol JA, Chan EK, Tan EM. Enhancement of antibody detection\nin cancer using panel of recombinant tumor-associated antigens. Cancer Epidemiol Biomarkers Prev\n2003;12:136–143. [PubMed: 12582023]\n97. Joos TO, Stoll D, Templin MF. Miniaturised multiplexed immunoassays. Curr Opin Chem Biol\n98. Stoll D, Templin MF, Schrenk M, Traub PC, Vohringer CF, Joos TO. Protein microarray technology.\nFront Biosci 2002;7:c13–32. [PubMed: 11779717]\n99. Templin MF, Stoll D, Schwenk JM, Potz O, Kramer S, Joos TO. Protein microarrays: promising tools\nfor proteomic research. Proteomics 2003;3:2155–2166. [PubMed: 14595815]\n100. Robinson WH, DiGennaro C, Hueber W, Haab BB, Kamachi M, Dean EJ, Fournel S, Fong D,\nGenovese MC, de Vegvar HE, Skriner K, Hirschberg DL, Morris RI, Muller S, Pruijn GJ, van\nVenrooij WJ, Smolen JS, Brown PO, Steinman L, Utz PJ. Autoantigen microarrays for multiplex\ncharacterization of autoantibody responses. Nat Med 2002;8:295–301. [PubMed: 11875502]\n101. Robinson WH, Steinman L, Utz PJ. Protein arrays for autoantibody profiling and fine-specificity\nmapping. Proteomics 2003;3:2077–2084. [PubMed: 14595805]\n102. Shin BK, Wang H, Hanash S. Proteomics approaches to uncover the repertoire of circulating\nbiomarkers for breast cancer. J Mammary Gland Biol Neoplasia 2002;7:407–413. [PubMed:\n12882525]\n103. Scanlan MJ, Welt S, Gordon CM, Chen YT, Gure AO, Stockert E, Jungbluth AA, Ritter G, Jager\nD, Jager E, Knuth A, Old LJ. Cancer-related serological recognition of human colon cancer:\nidentification of potential diagnostic and immunotherapeutic targets. Cancer Res 2002;62:4041–\n4047. [PubMed: 12124339]\n104. Qiu J, Madoz-Gurpide J, Misek DE, Kuick R, Brenner DE, Michailidis G, Haab BB, Omenn GS,\nHanash S. Development of natural protein microarrays for diagnosing cancer based on an antibody\nresponse to tumor antigens. J Proteome Res 2004;3:261–267. [PubMed: 15113102]\n105. Bouwman K, Qiu J, Zhou H, Schotanus M, Mangold LA, Vogt R, Erlandson E, Trenkle J, Partin\nAW, Misek D, Omenn GS, Haab BB, Hanash S. Microarrays of tumor cell derived proteins uncover\na distinct pattern of prostate cancer serum immunoreactivity. Proteomics 2003;3:2200–2207.\n[PubMed: 14595819]\n106. Ramachandran N, Hainsworth E, Bhullar B, Eisenstein S, Rosen B, Lau AY, Walter JC, LaBaer J.\nSelf-assembling protein microarrays. Science 2004;305:86–90. [PubMed: 15232106]\n107. Lawendowski CA, Giurleo GM, Huang YY, Franklin GJ, Kaplan JM, Roberts BL, Nicolette CA.\nSolid-phase epitope recovery: a high throughput method for antigen identification and epitope\noptimization. J Immunol 2002;169:2414–2421. [PubMed: 12193709]\n108. Rammensee HG, Weinschenk T, Gouttefangeas C, Stevanovic S. Towards patient-specific tumor\nantigen selection for vaccination. Immunol Rev 2002;188:164–176. [PubMed: 12445290]\n109. Sette A, Keogh E, Ishioka G, Sidney J, Tangri S, Livingston B, McKinney D, Newman M, Chesnut\nR, Fikes J. Epitope identification and vaccine design for cancer immunotherapy. Curr Opin Investig\nDrugs 2002;3:132–139.\n110. Parker KC, Bednarek MA, Coligan JE. Scheme for ranking potential HLA-A2 binding peptides\nbased on independent binding of individual peptide side-chains. J Immunol 1994;152:163–175.\n[PubMed: 8254189]\n111. Rammensee H, Bachmann J, Emmerich NP, Bachor OA, Stevanovic S. SYFPEITHI: database for\nMHC ligands and peptide motifs. Immunogenetics 1999;50:213–219. [PubMed: 10602881]\n112. Brusic V, Rudy G, Harrison LC. MHCPEP, a database of MHC-binding peptides: update 1997.\nNucleic Acids Res 1998;26:368–371. [PubMed: 9399876]\n113. Reche PA, Glutting JP, Zhang H, Reinherz EL. Enhancement to the RANKPEP resource for the\nprediction of peptide binding to MHC molecules using profiles. Immunogenetics 2004;56:405–419.\n[PubMed: 15349703]\n114. Hunt DF, Henderson RA, Shabanowitz J, Sakaguchi K, Michel H, Sevilir N, Cox AL, Appella E,\nEngelhard VH. Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass\nspectrometry. Science 1992;255:1261–1263. [PubMed: 1546328]\n115. Wei ML, Cresswell P. HLA-A2 molecules in an antigen-processing mutant cell contain signal\nsequence-derived peptides. Nature 1992;356:443–446. [PubMed: 1557127]\n116. Rotzschke O, Falk K, Wallny HJ, Faath S, Rammensee HG. Characterization of naturally occurring\nminor histocompatibility peptides including H-4 and H-Y. Science 1990;249:283–287. [PubMed:\n1695760]\n117. Kao H, Marto JA, Hoffmann TK, Shabanowitz J, Finkelstein SD, Whiteside TL, Hunt DF, Finn OJ.\nIdentification of cyclin B1 as a shared human epithelial tumor-associated antigen recognized by T\ncells. J Exp Med 2001;194:1313–1323. [PubMed: 11696596]\n118. Hogan KT, Coppola MA, Gatlin CL, Thompson LW, Shabanowitz J, Hunt DF, Engelhard VH,\nSlingluff CL Jr, Ross MM. Identification of a shared epitope recognized by melanoma-specific,\nHLA-A3-restricted cytotoxic T lymphocytes. Immunol Lett 2003;90:131–135. [PubMed:\n14687714]\n119. Lemmel C, Stevanovic S. The use of HPLC-MS in T-cell epitope identification. Methods\n120. Weinschenk T, Gouttefangeas C, Schirle M, Obermayr F, Walter S, Schoor O, Kurek R, Loeser W,\nBichler KH, Wernet D, Stevanovic S, Rammensee HG. Integrated functional genomics approach\nfor the design of patient-individual antitumor vaccines. Cancer Res 2002;62:5818–5827. [PubMed:\n12384544]\n121. Zhong W, Reche PA, Lai CC, Reinhold B, Reinherz EL. Genome-wide characterization of a viral\ncytotoxic T lymphocyte epitope repertoire. J Biol Chem 2003;278:45135–45144. [PubMed:\n12960169]\nCurrent Methodologies for the Detection of Autoantigens",
         "K. Anderson,J. LaBaer",
         null
        ],
        [
         "38",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "Two variants of the renal Na-K-Cl cotransporter (NKCC2),\ncalled NKCC2A and NKCC2F, display marked differences in Na,\nRb, and Cl affinities, yet are identical to one another except for a\n23-residue membrane-associated domain that is derived from alter-\nnatively spliced exons. The proximal portion of these exons is pre-\ndicted to encode the second transmembrane domain (tm2) in the\nform of an -helix, and the distal portion, part of the following\nconnecting segment (cs1a). In recent studies, we have taken advan-\ntage of the A–F differences in kinetic behavior to determine which\nregions in tm2-cs1a are involved in ion transport. Functional char-\nacterizations of chimeras in which tm2 or cs1a were interchanged\nbetween the variants showed that both regions are important in\nspecifying ion affinities, but did not allow delineating the contribu-\ntion of individual residues. Here, we have extended these structure-\nfunction analyses by studying additional mutants in which variant\nresidues between A and F were interchanged individually in the\ntm2-cs1a region (amino acid number 216, 220, 223, 229, or 233 in\nNKCC2). None of the substitutions were found to affect Km (C1),\nsuggesting that the affinity difference for anion transport is con-\nveyed by a combination of variant residues in this domain. However,\n2 substitutions in the tm2 of F were found to affect cation constants\nspecifically; interestingly, one of these mutations (residue 216) only\naffected Km (Rb) while the other (residue 220) only affected\nKm (Na). We have thus identified two novel residues in NKCC2 that\nplay a key role in cation transport. Because such residues should be\nadjacent to one another on the vertical axis of the tm2 -helix, our\nresults imply, furthermore, that the ion transport sites in NKCC2\ncould be physically linked.",
         "* This work was supported by Grants MOP-68949 and MOP-15405 from the Canadian\nInstitute of Health and Research and the Kidney Foundation of Canada. The costs of\npublication of this article were defrayed in part by the payment of page charges. This\narticle must therefore be hereby marked “advertisement” in accordance with 18 U.S.C.\nSection 1734 solely to indicate this fact.\n3 The abbreviations used are: NKCC, Na-K-Cl cotransporter; cs, connecting segment;\nFR, flux rate; NMG, N-methyl glucamine; o, extracellular; R, regular; R, hypertonic\nregular; R, modified regular; sa, Squalus acanthias; saA, S. acanthias NKCC2A; saA/F,\nchimera in which tm2 is from saNKCC2A and cs1a from saNKCC2F; saF, S. acanthias\nNKCC2F; saF/A, chimera in which tm2 is from saNKCC2F and cs1a from saNKCC2A; tm,\ntransmembrane domain.\nsplicing of the primary transcript leads to the formation of three variants\n(called NKCC2B, -A, and -F) that are identical to one another except for\nthe exon 4-encoding region, which corresponds to the second trans-\nmembrane domain (tm2) and part of the following connecting segment\n(cs1a). Interestingly, these variants are distributed differentially along\nthe thick ascending limb as follows: B is in macula densa cells, A in the\nouter medullary stripe, and F in the inner medullary stripe (6, 7). In\ncontrast to NKCC2, the first isoform of NKCC (NKCC1) is widely dis-\ntributed, arising in non-polarized cells as well as in the basolateral pole\nof many secretory epithelia (8 –12).\nThe main function of both NKCC1 and NKCC2 is to couple the\nmovement of Na-K-Cl across the cell surface. Based on various\nstudies, the stoichiometry of ion transport has been shown to be 1Na:\n1K:2Cl, implying that there is no net movement of charge during the\ntranslocation cycle (13–15). In some studies, however, alternate trans-\nport modes have been proposed, e.g. 2Na:1K:3Cl for squid NKCC1\n(16), 1Na:1K:1Cl for NKCC2A (17, 18), and K/K exchange for\nboth NKCC1 and NKCC2A (17–19). We and others have hypothesized\nthat large differences in Cl affinities between the 2 Cl binding sites\ncould account for the apparent 1Na:1K:1Cl stoichiometry (17, 18),\nand incomplete reactions during the transport cycle, for the Na-inde-\npendent exchange mode (18 –19).\nThe kinetic characteristics of NKCC1 vary widely among species and\nthose of NKCC2, among the splice variants; for example, ion constants\nare lower for human NKCC1 compared with shark (sa) NKCC1 (9,\n22–26), and lower in NKCC2A compared with F (17, 18, 27, 28). In\nprevious work, we have exploited such differences to identify residues\nthat are involved in ion movement by the transporters. Indeed, struc-\nture-function studies of wild-type and chimeric NKCCs have shown\nthat the kinetic characteristics of these proteins are specified, at least in\npart, by variant residues that are localized in the tm2 and/or cs1a regions\n(18, 22–25). Interestingly, such studies have also led to the identification\nof a high affinity-Cl binding site in cs1a.\nChimera approaches can be used quite effectively to identify variant\nresidues that convey differences in the operations of homologous carri-\ners. The main disadvantage of such studies is that key residues will be\nmissed if they are fully conserved between the carriers. This limitation is\nillustrated by recent analyses in which the behavior of various chimeras\nled to slightly different interpretations of where affinity-modifying\ndomains are localized depending on which NKCC was used to gen-\nerate the mutants. Based on these analyses, for instance, it was not\npossible to determine whether the cs1a region of NKCC1 is involved\nin ion transport (18, 22–25).\nThe NKCC ion binding sites are known to be functionally interde-\npendent; that is, a change in ion affinity at one site induces a change in\nion affinity at another site (18 –21). One implication of this character-\nThe Role of tm2 in Cation Transport by NKCC2\nistic is that it could be difficult to determine which ion constants are\nmodified directly as a result of a residue substitution. Fortunately, we\nhave recently determined the manner in which these constants are\ninterrelated based on kinetic studies of wild-type NKCC1 expressed in\nHEK-293 cells (24, 25). These data, which were analyzed in the context\nof previous kinetic models (19 –21), indicate that at high extracellular\n(o) [Na] or [K], changes in either Na or K affinity will have small\neffects on Km (Cl), whereas at high [Cl]o, a change in Cl affinity will\nhave a larger effect on Km (Na) and Km (Rb).\nOne of the structure-function studies that was discussed above led to\nthe identification of five candidate residues in tm2-cs1a that play a def-\ninite role in Cl transport by NKCC2, but the same degree of resolution\nconcerning the structural determinants of cation transport was not\nobtained (18). The approach that was used here was valuable in this\nregard, allowing us to uncover two residues that are important in deter-\nmining Na and Rb affinities. Quite interestingly, our kinetic studies\nalso suggest that the residues identified belong to distinct ion binding\nsites in the tm2-cs1 region.",
         "RESULTS\nPreamble—Throughout the following sections, the kinetic behaviors\nof our 10 mutants will be compared with those of four other carriers\n(saA, saF, saA/F, and saF/A) that were tested in a previous study using\nthe same expression system (18). We elected not to repeat previous\ncharacterizations for the following reasons. 1) We have observed that\nthe kinetic features of saNKCC2s in X. laevis oocytes were very repro-\nducible over time. 2) The functional assays were carried out by the same\ninvestigators. 3) The main conclusions in this article are based on the\nkinetic characteristics of the 10 novel mutants. 4) As will be reported\nlater, mutant carriers in which a substitution was predicted to produce\nno effect on kinetic parameters behaved as such, exhibiting ion con-\nstants that were identical to previously published values (18); for saA,\nthese values, expressed as Km (Na), Km (Rb), and Km (Cl), were 15.3, 2.2,\nand 6.9 mM, and for saF, 93.0, 4.6, and 69.1 mM. 5) Recent experiments\n(n  1) revealed that the kinetic behavior of saA/F and saF/A was con-\nsistent with expectation based on our previous publication (18), which\nshowed Km (Na), Km (Rb), and Km (Cl) for saA/F of 18.0, 2.2, and 70.0\nmM, and Km for saF/A of 22.6, 1.7, and 8.7 mM.\nExpression of saNKCC2s in Oocytes—Immunofluorescence studies of\noocytes expressing wild-type or mutant saNKCC2s revealed that the\nanti-P-NKCC Ig, which was used as primary antibody, labels the heter-\nologous proteins specifically, and that most of the signal occurs at the\ncell surface (results not shown). These studies also revealed that protein\nexpression levels were relatively similar among all of the saNKCC-ex-\npressing oocytes tested. Functional analyses, during which cotrans-\nporter activity was determined under subsaturating concentrations of\nNa, Rb, and Cl, confirmed the localization data in showing that all\nof the expressed carriers generate bumetanide-sensitive 86Rb influx\nrates that are clearly above those of the X. laevis NKCC1. As seen in Fig.\n2, e.g. these rates are between 0.8 and 7.3 nmol/oocyte/h for the mutant\nNKCC2s, that is, at least 4-fold above background for all of the carriers\nor 8-fold if saA(S229C) is excluded.\nKinetics of Na Transport: Role of tm2—As mentioned earlier, we and\nothers (17, 18, 27, 28) have observed that saA and saF exhibit substantial\ndifferences in regard to apparent Na affinities, that is, Km (Na) for saA\nis 5-fold lower compared with saF. We have also observed that\nKm (Na) remains low even when the tm2 of saA is replaced by that of\nsaF, as in the chimera saF/A, suggesting that variant residues in this\ndomain convey no differences in Na affinity as long as cs1a is from saA\n(18). Hence, replacing residue 216 or 220 in saA by those of saF should\nnot lead to changes in Km (Na) values.\nTo confirm this prediction, we measured the dependence of 86Rb\ninflux on [Na] for saA(I216V) and saA(I220L), and analyzed results in the\nlight of previously observed behaviors for saA, saF, and saF/A (18). Data\nare shown in Fig. 3A (upper row) fit by a model of Na binding at a single\nsite, and Km (Na) derived from the activity versus [ion] relationships are\nshown on Fig. 3B (first 2 bars). In TABLE THREE, similar data are\npresented except that the Km values were calculated with best values of\nHill coefficients. As expected, these mutants exhibit high Na affinities\nthat are in the range of those reported for saA and saF/A. It can be noted,\nadditionally, that NKCC activity at 0 mM Na\no is quite substantial, as\nhad also been reported for saA and saF/A, suggesting that residues 216\nor 220 do not account for the ability of certain NKCCs to induce Rb/\nRb(K) exchange at the cell surface.\nIn our recent structure-function study of saNKCC2 mutants, the kinetic\nbehavior of saF/A in regard to Na transport was also seen to be very similar\nto that of saA/F (18). These findings, which were somewhat unexpected to\nus, imply that Km (Na) for saF decreases following a Phe 3 Ala tm2 substi-\ntution, and that variant residues in this domain do convey differences in\nNa affinity but only when cs1a is from saF. They also imply that differ-\nences between saF and saA/F can be exploited to determine which residues\nin the tm2 of saA are important in determining Km (Na).\nThe Role of tm2 in Cation Transport by NKCC2\nmM\n16.8  2.4\n22.7  5.2\n93.3  15.1\n38.2  5.9\n11.7  4.1\n15.5  5.8\n15.7  2.9\n85.5  14.1\n84.6  8.5\n82.0  10.4\n1.6  0.5\n1.7  0.4\n2.5  0.3\n4.5  0.9\n2.5  0.2\n2.2  0.4\n1.0  0.6\n4.7  1.1\n6.6  1.3\n6.7  1.5\n12.8  3.4\n8.3  5.6\n102.2  25.1\n78.0  12.4\n22.0  3.7\n20.3  3.4\n13.5  4.6\n85.9  10.2\n65.8  13.3\n66.5  11.9\n0.82  0.21\n0.71  0.24\n1.04  0.12\n1.13  0.17\n1.03  0.03\n0.81  0.21\n0.71  0.15\n1.03  0.06\n1.04  0.16\n1.11  0.09\n1.18  0.07\n1.06  0.14\n1.12  0.03\n0.97  0.06\n0.96  0.19\n1.10  0.15\n1.02  0.08\n1.04  0.17\n0.82  0.22\n0.91  0.23\n1.13  0.17\n0.92  0.21\n1.74  0.12\n1.73  0.13\n0.88  0.19\n0.87  0.11\n1.21  0.07\n1.71  0.09\n1.76  0.05\n1.80  0.21\n3B (last 2 bars) and TABLE THREE; data are illustrated as above using\nthe same models of ion binding. It is seen here, that Km (Na) for\nsaF(V216I) is much higher than those for saA(I216V) or saA(I220L), but that\nKm (Na) for saF(L220I) is intermediate between those measured for the\nsaA mutants and saF(V216I). These new observations suggest that residue\n220 in the tm2 of saF accounts, at least in part, for the functional char-\nacteristic that was displayed by saA/F in regard to Na transport. It can\nalso be noted here that carrier activity at 0 mM is, as would have been\nexpected, lower for saF(V216I) and saF(L220I) compared with saA(I216V)\nand saA(I220L), but similar to that reported for saF and saA/F previously.\nKinetics of Rb Transport: Role of tm2—For either of the recently\ncharacterized saA, saF/A, saA/F, and saF, the kinetics of Rb transport\nwere shown to be very similar to those of Na transport with the excep-\ntion that NKCC activity at 0 mM Rb and 0 mM K is equally low for all\nthe carriers tested (18). Otherwise, saA, saF/A, and saA/F display a\nsimilar affinity for Rb that is 2-fold higher compared with saF and\nthe activity versus [ion] relationships is best fit with a model of substrate\nbinding at one site. Accordingly, Km (Rb) values for saA(I216V) and\nsaA(I220L) should be similar to those for saA and saF/A, whereas Km (Rb)\nvalues for saF(V216I) or saF(L220I) could potentially differ when compared\nwith those for saF.\nThe kinetics of Rb transport by saA(I216V), saA(I220L), saF(V216I), and\nsaF(L220I) are summarized in Fig. 4 and TABLE THREE with the repre-\nsentation schemes and binding models that were used to describe Na\ntransport. Here, the behavior of saA(I216V) and saA(I220L) confirms our\npredictions; Km (Rb) values for these mutants are in the low range, sim-\nilar to those reported for saA and saA/F. The behaviors of saF(V216I) and\nsaF(L220I), on the other hand, differ from one another; here, Km (Rb) for\nsaF(L220I) is similar to that reported for saF, but for saF(V216I), it is inter-\nmediate between that for saA(I216V) or saA(I220L) and saF(L220I). These\nfindings suggest that residue 216 accounts to some extent for the\ndecrease in Rb affinity that occurs following a Phe 3 Ala tm2 substi-\ntution. They also indicate that residues 216 and 220 play different roles\nin cation transport.\nKinetics of Cl Transport: Role of tm2—Kinetic parameters for the\nrecently characterized saF/A were found to be similar to those for saA in\nregard to Cl transport, suggesting that substitution of residue 216 or\n220 in saA by the corresponding residue in saF will produce no effects\non Km (Cl) (18). As seen in Fig. 5A (upper row), Fig. 5B (first 2 bars) and\nThe Role of tm2 in Cation Transport by NKCC2\nIn our structure-function studies of chimeric saNKCC2s, we\nreported that Cl transport kinetics for saF/A are identical to those for\nsaA, but that the behavior of saA/F is unlike that of either saA or saF\n(18). Indeed, Km (Cl) for saA/F is similar to that for saF but calculated\nHill(Cl\n) similar to that for saA. Hence, substituting residue 216 or 220\nin the tm2 of saF by the corresponding residue in saA should lead to no\nchanges in Km (Cl) values but could still alter the calculated Hill(Cl\n)s.\nBased on the kinetic characteristics of saF(V216I) and saF(L220I) (lower row\nof Fig. 5A, last 2 bars of Fig. 5B, and TABLE THREE), it is in fact seen\nthat both Km (Cl) and Hill(Cl\n) values are similar to those reported for\nsaF. These results suggest that other residues in the tm2 of saF or,\nconceivably, the presence of both residues in this domain account for\ndifferences in apparent number of transport sites for Cl.\nKinetics of Cation Transport: Role of cs1a—The kinetic behavior of\nsaA/F and saF/A showed that cs1a could also play an important, perhaps\nindirect role in cation transport (18). Accordingly, we measured the\ndependence of saA(L223I), saA(S229C), saA(C233V), saF(I223L), saF(C229S),\nsaF(V233C) on [Na] or [Rb], expecting that residue substitutions in the\ncs1a of saA would generate no kinetic changes but that residues substi-\ntutions in the cs1a of saF could alter cation affinities substantially.\nResults of these studies are illustrated in Figs. 6 and 7 using a single\nsite ion binding model and in TABLE THREE. As anticipated, Km values\nfor saA(L223I), saA(S229C), and saA(C233V) (higher rows) are all in the low\nrange, but surprisingly, those for saF(I223L), saF(C229S), or saF(V233C)\n(lower rows) are all in the high range, similar to those reported for saF.\nThese observations suggest that other residues in the cs1a of saF or\nperhaps a combination of specific residues among those that were sub-\nstituted account for the previously observed differences in Km (Na) and\nKm (Rb) between the saF and saF/A.\nKinetics Cl Transport: Role of cs1a—Although tm2 has been shown\nto play an important role in Cl transport, we have been unable to\nidentify the individual residues involved (18). Interestingly, differences\nbetween saF and saA/F had revealed the existence of a low affinity Cl\nbinding site in tm2 (Km (Cl) values for both carriers are in the high range\nbut calculated Hill(Cl\n) values are lower for saF/A) implying that the\nThe Role of tm2 in Cation Transport by NKCC2\nhigh affinity Cl binding site is in cs1a. In fact, the behavior of saA and\nsaF/A in our recent study suggested that this is precisely the case and\ncould have explained why cation affinities for saF/A are surprisingly\nhigh. However, because neither of residues 223, 229, or 233 have been\nfound to modulate cation transport in the current study, it is unlikely\nthat Km (Cl) values for saA(L223I), saA(S229C), saA(C233V), saF(I223L),\nsaF(C229S), and saF(V233C) will differ from those of the parent carriers.\nThe data in Fig. 8 and TABLE THREE, which show the dependence of\nthese six mutants on [Cl] and kinetic parameters derived from the\nactivity versus [ion] relationships, are once again consistent with expec-\ntations. Indeed, all of the saAx mutants are seen to have higher affinities\nthan the saFx mutants, and the range of Km values observed are similar\nto those reported for their parent carriers. Hence, the residues or com-\nbination of residues that account for differences in Cl transport kinet-\nics between saF and saF/A or between saA and saA/F have not been\nrevealed through these mutagenic analyses.",
         "DISCUSSION\nIn this study, differences in kinetic properties between saA and saF\nwere exploited to identify novel ion transport sites in the tm2-cs1a\nregion of NKCC. Our main approach was to interchange variant resi-\ndues between saNKCC2s, creating a single point substitution for each\nmutant, and determine which of these residues account for the distinc-\ntive behaviors. The role of variant residues was analyzed further by\ncomparing the effects of single point substitutions to those of combined\nsubstitutions, exploiting our recent observation that both saA/F and\nsaF/A also differ in kinetic behavior when compared with either saA or\nsaF (18). Based on such analyses, we were able to identify two residues\nthat account for differences in cation affinity between saA and saF.\nQuite remarkably, we found that one residue (position 220) is important\nin determining Na affinity, whereas the other residue (position 216 in\nthe same domain) is important in determining Rb affinity.\nHydropathy plot models of NKCC2 have led to the prediction that the\n5-end of exon 4 encodes tm2 and the 3-end of this exon, cs1a. They\nhave also led to the prediction that the central domain of NKCC2 is\nconstituted of 12 tms that are organized as -helicoidal protein seg-\nments (8 –11, 32, 33). For NKCC1, a related carrier that share high levels\nof homology with NKCC2, such models have been confirmed indirectly\nby the identification of regions that behave as tm when expressed in the\nform of fusion proteins by rabbit reticulocyte lysates (34). Based on\nthese structural considerations, the identification of two residues that\nmodulate Na and Rb (K) affinity differentially, but that are very\nclose to one another in the tm2 region, bears a number of important\nimplications.\nOne such implication is that the ion transport sites of NKCC,\nalthough distinct kinetically, may be capable of interacting with one and\nanother through certain key residues. Indeed, the structural models of\nthe tm2-cs1a region also predicts that residues 216 and 220 (Ile and Ile in\nsaA and Val and Leu in saF) are adjacent to one another on the same\nvertical axis of the -helix, suggesting that their peptide backbones\nThe Role of tm2 in Cation Transport by NKCC2\nand/or aliphatic R groups are connected through H atoms (35, 36). In\nthe case of NKCC, for which ions are proposed to move single file down\na translocation pocket (19 –21, 24, 25), such physical interactions could\nrepresent a key aspect of the carrier’s modus operandi, accounting, in\nparticular, for the kinetic interdependence of the ion binding sites.\nAnother implication of the findings that are discussed here is that we\nhave identified a coherent structural correlate to the widely accepted\nmodel of ordered ion binding proposed by Lytle et al. (19). Indeed, this\nmodel predicts that extracellular ions bind to the translocation pocket\nof NKCCs in the order Na-Cl-K-Cl and are released from this\npocket on the inside of the cell in the exact same order. One would thus\nassume based on such a binding scheme that the Na transport site is\nseated more deeply than the K transport site relative to the extracel-\nlular space. Our findings are in fact consistent with this interpretation in\nthat residue 216, which is localized in the middle of tm2, confers differ-\nences in Km (Rb) between saA and saF, and residue 220, which is local-\nized further into the bilayer, confers differences in Km (Na).\nA third implication related to the identification of affinity-modifying\nresidues that are “ion-specific” is that the observed changes in Km values\nwere probably not due to substitution-induced conformational arti-\nfacts. Because Michaelis constants for ion movement through a sym-\nporter are related to several kinetic parameters that are not limited to\nthe “true binding affinities” for transported substrates (14, 24, 25), such\nartifacts could have accounted for a change in kinetic behavior by affect-\ning, e.g. a rate constant during the translocation cycle. Under such cir-\ncumstances, however, one would have expected parallel changes in Km\nvalues for all of the cotransported ions. Our results are thus reconcilable\nwith a simple model in which Km values were altered individually\nthrough substitutions of affinity-modifying residues.\nResidues 216 and 220 are nonpolar and, accordingly, predicted to lie\non the hydrophobic pole of the -helix. This localization does not nec-\nessarily imply that these two residues play no role in ion transport.\nIndeed, their peptide backbone or their R groups could still be close to\nthe hydrophilic pole of the -helix and interact directly with ions. Alter-\nnatively, residues 216 and 220 could play an important role in ion coor-\ndination from the hydrophobic pole of the -helix by influencing ion\nbinding sites through local steric interactions. To this effect, it should be\nmentioned that the current mutagenic analysis was design to identify\nresidues that confer differences in kinetic parameters and that such\nresidues, even if they are included in the Na-K-Cl transport sites, may\nnot be the same as those that are considered essential for substrate\nbinding.\nWe have stated earlier that ion constants for NKCC1 are interrelated\nin a predictable manner but that the effect of a change in one constant\nwas not necessarily the same on all of the other constants (19 –21, 24,\n25). For example, it has been shown that the apparent affinity of an ion\nis most influenced by changes in the true affinities of the ion that binds\nnext in the translocation pocket, implying that the effect of change in\nKm (Cl) on Km (Na) and Km (Rb) will be more important than the effect\nof a change in Km (Na) and Km (Rb) on Km (Cl). Based on such inter-\nrelationships, it is reasonable to propose that the isolated decrease in\nKm (Na) resulting from a residue substitution at amino acid number 220\nin saF was related to an increase in true Na affinity, and that the\nisolated decrease in Km (Rb) resulting from a residue substitution at\namino acid number 216 was related to an increase in true Rb affinity.\nThe validity of these interpretations is substantiated further by the find-\ning that none of the substitutions in cs1a led to changes Km (Na) or\nKm (Rb).\nCertain residues in the transport sites of NKCC1 could influence ion\naffinity only when they operate in concert with specific sets of contigu-\nous residues. Hence, the importance of variant residues in conferring\nkinetic differences between saA and saF could have gone undetected in\nthis work because our main approach was to determine the effect of\nsingle point mutations on Michaelis parameters. An example that illus-\ntrates this possibility comes from the observation that Km (Cl) values for\nsaF/A, which corresponds to a saNKCC2 harboring five substitutions in\ncs1a, were significantly different from those for saA (18), whereas\nKm (Cl) values for saA(L223I), saA(S229C), or saA(C233V), which correspond\nto saNKCC2s harboring single point substitutions in cs1a, were identi-\ncal to those for saA. It may be thus useful in future studies to examine\nthe importance of cs1a in conferring kinetic singularities by exchanging\nvariant residues 2 by 2 or 3 by 3.\nThe concept of affinity-modifying residue groups might also explain\nwhy a number of the complementary mutants did not exhibit reciprocal\nkinetic behaviors. As described under “Results,” for example, substitu-\ntions in the tm2 of saA (I216V or I220L) led to no changes in kinetic\nconstants, whereas substitutions in the tm2 of saF (V216I or L220I) did\nproduce very noticeable changes. These findings imply that certain\ncombinations of residues (e.g. Gly222-Leu223-Thr225-Ser229-Cys233) may\nplay such a preponderant role in determining ion constants, either\ndirectly or through conformational interactions between the transport\nsites, that they could mask the importance of contiguous affinity-mod-\nifying residues (Ile216-Ile220 or Val216-Leu220), whereas other combina-\ntions of residues at the same positions (e.g. Cys222-Ile223-Met225-Cys229-\nVal233) may play a less influential role.\nIn this work, functional expression levels tended to differ among\nmutants (Fig. 2), whereas protein expression levels were relatively sim-\nilar based on the IF studies. Although, in principle, these observations\ncould suggest that certain substitutions altered Vmax, it is not possible at\nthis time to draw any firm conclusions in this regard without the avail-\nability of more precise quantitative estimates of protein expression at\nthe cell surface. On the other hand, it appears unlikely that changes in\ncell surface delivery or in Vmax, if they occurred, accounted for the\nobserved changes in Km values. Indeed, previous studies in oocytes have\nshown that the dependence of saA on [Na], [Rb], or [Cl] remains\nstable whether the carrier is expressed at low or high levels by varying\nthe quantities of cRNA injected.4 Here, in addition, we have observed\nthat all of the cs1a mutants that were expected to behave as saA or as saF\ndid so despite the noted differences in Vmax.\nIn conclusion, we have identified two residues in NKCC that appear to\nplay an important role in cation transport. These residues, which are in the\ntm2-cs1a region of saNKCC2 (position 216 and 220) and which differ\nbetween saA and saF, convey variant behaviors to each of the wild-type\ncarrier and they also modulate Km (Na) and Km (Rb) differentially. Interest-\ningly, the proximity in which they occur implies that the interdependence\nof kinetic constants for this type of carrier could be mediated through com-\nplex physical interactions between the ion binding sites. Based on the effect\nof various substitutions on Km (Cl), in addition, we can postulate that\nkinetic constants for the NKCC transport system are probably each speci-\nfied by different sets of residue combinations.\nThe Role of tm2 in Cation Transport by NKCC2\nAcknowledgments—We thank Luc Caron and Micheline Noe¨l for superb tech-\nnical assistance.\n32. Hebert, S. C., Mount, D. B., and Gamba, G. (2004) Pflugers Arch. 447, 580 –593\n33. Park, J. H., and Saier, M. H. Jr. (1996) J. Membr. Biol. 149, 161–168\n34. Gerelsaikhan, T., and Turner, R. J. (2000) J. Biol. Chem. 275, 40471– 40477\n35. Chou, P. Y., and Fasman, G. D. (1978) Adv. Enzymol. 47, 45–148\n36. Doyle, D. A. (2004) Eur. Biophys. J. 33, 175–179",
         "MATERIALS AND METHODS\nOverview—Structure-function studies were carried out to identify\naffinity-modifying residues in the tm2-cs1a region of Squalus acanthias\n(sa) NKCC2A and saNKCC2F (called saA and saF in this work). Mutant\nsaNKCC2A and saNKCCF were expressed in Xenopus laevis oocytes\nand characterized through flux assays. Unless mentioned otherwise,\nchemicals, reagents, and kits were from different suppliers or collabo-\nrators.\ncDNAs—A total of 10 different cDNA constructs were used in these\nassays. They were subcloned in Pol1, a X. laevis oocyte expression vec-\ntor that is derived from pGEM and that contains the T7 promoter, a\nmultiple cloning site flanked by the X. laevis -globin untranslated\nregions and a poly(A) tract (17, 18, 29, 30). All of the constructs were\npropagated in XL1-blue cells (Stratagene) and purified on silica-based\ncolumns (Qiagen).\nsaA, saF, saA/F, and saF/A—These constructs were generated in pre-\nvious studies but were not recharacterized for this analysis (18). They\ncorrespond to wild-type saA and saF, and to a pair of complementary\nchimeras in which tm2 or cs1a was interchanged between saA and saF.\nThese chimeras are called saA/F (tm2 from A, cs1a from F) and saF/A\n(tm2 from F, cs1a from A). The wild-type saA and the tm2-cs1a regions\nof saA, saF, saA/F, and saF/A are represented in Fig. 1 through hydrop-\nathy plot models to illustrate how they differ between each other and to\nshow where the junction point of the chimeras occur, thereby providing\na rationale for the choice of substitutions that were created in the cur-\nrent work.\nsaF(I223L),\nsaA(S229C), saF(C229S), saA(C233V), and saF(V233C)—These constructs con-\nsist of saA or saF inserts in which one of the tm2-cs1a codons was\ninterchanged between the variants. All of the mutations were generated\nthrough pairs of mutagenic oligonucleotides (Sigma) using the\nQuikChange Mutagenesis kit (Stratagene). The resulting constructs\nwere called saA(x) or saF(x), where A and F indicate the NKCC2 variant\nin which the residue was substituted, and x, the substitution that was\ngenerated based on the amino acid number in saNKCC2. The muta-\ngenic oligonucleotides that were used to create these mutants are shown\nin TABLE ONE, and the position of the substituted residues are also\nillustrated in Fig. 1.\nIt should be mentioned here that the choice of substitutions was not\nbased only on differences in residues between saA and saF but also on\nthe results of recent structure-function analyses that were concerned\nwith the identification of ion transport sites in both NKCC1 and -2 (18,\n22–25). In saA/F or saF/A, for example, 9 residue substitutions were\ncreated in tm2 and 5 residue substitutions in cs1a, suggesting that a\nnumber of these residues were responsible for differences in kinetic\nbehaviors between saA and saF. However, based on the behavior of\nsaNKCC1 and human (h) NKCC1 chimeras, both these carriers also\nexhibit dramatic differences in ion affinities, we had predicted that non-\nconserved residues between the two NKCC1s as well as between high\nand low affinity wild-type NKCCs (from shark, rat, rabbit, human)\nwould be important in determining cation affinities, whereas noncon-\nserved residues between high affinity and low affinity NKCC2s (from\nshark, rat, rabbit) would be important in determining anion affinity (17,\n18, 22–28). This interpretation led to the suggestion that the main affin-\nity-modifying residues in saNKCC2 would be found at position 216,\n220, 223, 229, or 233.\nExpression in X. laevis Oocytes—To obtain synthesis of saNKCC2s in\nthis system, defolliculated stage V-VI oocytes were injected with\nsaNKCC2/Pol1-derived cRNA (5–25 ng/oocyte diluted in H2O) and\nThe Role of tm2 in Cation Transport by NKCC2\nca att act TGC ATC tct ATG tcc gct ata\nTGT acc aat gga GTT gtt cgt gga ggt gg\ngtc tta aca GGG TTG tct ACC tca gcc att\nTCG acc aac gga TGT gtt cga g\nctg gct act GTG gta aca tc\nctg agt acg ATA gta act gtc\nct ata gta aca tca CTT acg ggg ttg tc\ncg gtg gta act gtc ATA aca tgt atc tc\nca att act ggg ATC tct acc tct gc\nc tta aca tgt TTG tct atg tca gcc\ncc tct gct ata TGC aca aat gg\ng tca gcc att TCA acc aat g\nca aat gga GTT gtt cgt gga gg\ncc aat gga TGT gtt cga gga gg\nmaintained for 3 days at 18 °C in Barth’s (B) medium (see TABLE TWO)\n 125 M furosemide. Groups of oocytes were also injected with H2O\nalone as controls.\nsaNKCC2-expressing X. laevis\nof\nOocytes—They were performed as previously described (29, 30). Here,\nthe anti-P-NKCC immunoglobulin (Ig), which is specific to the phos-\nphorylated form of various NKCCs (29, 31), and which was a gift from\nDr. B. Forbush (Yale University), was used as primary antibody, whereas\nthe goat Alexa Fluor® 488-conjugated goat anti-rabbit anti-IgG (Molec-\nular Probes, A11008) was used as secondary antibody.\nKinetic Studies—Ion transport rates were determined by 86Rb influx\nmeasurements at 22 °C using different types of media (shown in\nTABLE TWO). Each of these media is derived from a regular (R)\nmedium that is iso-osmolar relative to plasma for the X. laevis frog. One\nof the modified media (called R) was obtained by adding 84 mOsM\nsucrose, and the other types of media (called R) were obtained by\nreplacing the ions of medium R with K or N-methyl gluconate (as\n2 (as anion substituents).\ncation substituents) or with gluconate  SO4\nBefore, the flux assay per se, furosemide was first removed through\nseveral rinses in medium R. Eggs were then subjected to two consecutive\nincubation steps: one of 60 min in medium R to activate the cotrans-\nporter and another of 45 min in various medium R  1–2 Ci/ml\n86Rb  10 M ouabain  250 M bumetanide to measure NKCC-\nspecific accumulation of the isotope. During this second 45-min time\ninterval, transport activity versus time is in a linear range, regardless of\nthe ion concentration used. Here, the various medium R were pro-\nduced by varying [Na] from 0 to 87 mM, [Rb] from 0.1 to 20 mM or\n[Cl] from 0 to 86 mM (see TABLE TWO and figure legends). Fluxes\nwere terminated with washes in a high K medium R  10 M oua-\nbain  250 M bumetanide, and oocytes were transferred to 96-well\nplates (1 oocyte/well) prefilled with 2% SDS and scintillation fluid. After\na short stabilization period, accumulated 86Rb was detected with the\nTopCountNXT microplate counter (Packard).\nTo minimize cell damage during the flux assays, all of the incubation\nsteps were carried out in 12-well plates by transferring oocytes with a\nPasteur pipette from one prefilled well to another, rather than by chang-\ning the media through aspiration and refilling of the same well. To avoid\nsolution contamination, however, each of these steps was performed\nusing three different wells prefilled with the same solution, the first two\nwells to incubate oocytes for a very short period of time, and the last well\nto complete the incubation step per se (except for the final wash during\nwhich all of the incubation times were brief). With this method, we have\nfound that ion concentrations in the final incubating solutions change\nby less than 0.5%.4\nIn each experiment, counts among several oocytes (generally 4 – 6)\nwere averaged and transposed into flux rates (FRs). The equation used\nfor this calculation was FRs  (counts/oocyte  [85Rb] in the flux\nsolution)  ([counts] in the flux solution  time of incubation in the\nflux solution), assuming that mean membrane surface areas between\ngroups of stage V-VI oocytes are very similar; for kinetic determinations\n(see below), mean FRs were also normalized to those obtained at the\nThe Role of tm2 in Cation Transport by NKCC2\nhighest ion concentration. All of these mean normalized or non-nor-\nmalized values were subsequently reaveraged from several replicate\nexperiments (n  2–5), and the data obtained were expressed as\nmean  S.E.\nIon constants (expressed as Km) and closeness of fits were calculated\nwith the programs PLOT (Biff Forbush) or SigmaPlot 4.00 for Windows\nusing the Michaelis-Menten equation (for a one binding site model), the\nHill equation with a coefficient (n) of 2 (for a two binding site model), or\nthe Hill equation with an undetermined value of n. In each case, least\nsquare analyses were carried out with a 3 or 4parameter equation (vari-\n Hill) to allow a 0 offset, assuming that 86Rb\nables: Vmax, Km, and Vmin\ninflux is never completely absent even if one of the cotransported ions is\nremoved from the flux solution (as occurs when the transporter is in the\nK/K exchange mode for example). In the current study, these kinetic\nparameters are presented as 1) averages of mean values from replicate\nexperiments (in which case differences between variables were analyzed\nthrough Student’s two tailed t-tests, assuming a null hypothesis for p\nvalues  0.05), and as 2) the results of single fits using activity versus\nconcentration data points that were also obtained by averaging mean\nvalues from replicate experiments. The latter method was employed to\nadjust for intra-experimental variability, which could introduce system-\natic errors, and because it is not possible to obtain time-matched data\nfor all of the conditions tested. The types of calculations carried out, and\nvariables that were used are specified further in the legends of tables or\nfigures.\nLast of all, it should be noted that kinetic parameters were calculated\nwithout subtracting bumetanide-sensitive 86Rb influxes measured in\nH20-injected eggs. In fact, these eggs exhibit very low levels of 86Rb\ninfluxes under the experimental conditions used for the flux assays (17,\n18, 29, 30). Hence, the dependence of saNKCC2 activity on [Na],\n[Rb], or [Cl] should remain unaltered whether this background activ-\nity is subtracted or not from the total counts. As will be discussed later,\nprevious studies have confirmed this prediction on multiple occasions\neven for heterologously expressed carriers with transport activities that\nwere as low as 3-fold above background.",
         "Two variants of the renal Na-K-Cl cotransporter (NKCC2),\ncalled NKCC2A and NKCC2F, display marked differences in Na,\nRb, and Cl affinities, yet are identical to one another except for a\n23-residue membrane-associated domain that is derived from alter-\nnatively spliced exons. The proximal portion of these exons is pre-\ndicted to encode the second transmembrane domain (tm2) in the\nform of an -helix, and the distal portion, part of the following\nconnecting segment (cs1a). In recent studies, we have taken advan-\ntage of the A–F differences in kinetic behavior to determine which\nregions in tm2-cs1a are involved in ion transport. Functional char-\nacterizations of chimeras in which tm2 or cs1a were interchanged\nbetween the variants showed that both regions are important in\nspecifying ion affinities, but did not allow delineating the contribu-\ntion of individual residues. Here, we have extended these structure-\nfunction analyses by studying additional mutants in which variant\nresidues between A and F were interchanged individually in the\ntm2-cs1a region (amino acid number 216, 220, 223, 229, or 233 in\nNKCC2). None of the substitutions were found to affect Km (C1),\nsuggesting that the affinity difference for anion transport is con-\nveyed by a combination of variant residues in this domain. However,\n2 substitutions in the tm2 of F were found to affect cation constants\nspecifically; interestingly, one of these mutations (residue 216) only\naffected Km (Rb) while the other (residue 220) only affected\nKm (Na). We have thus identified two novel residues in NKCC2 that\nplay a key role in cation transport. Because such residues should be\nadjacent to one another on the vertical axis of the tm2 -helix, our\nresults imply, furthermore, that the ion transport sites in NKCC2\ncould be physically linked.\n* This work was supported by Grants MOP-68949 and MOP-15405 from the Canadian\nInstitute of Health and Research and the Kidney Foundation of Canada. The costs of\npublication of this article were defrayed in part by the payment of page charges. This\narticle must therefore be hereby marked “advertisement” in accordance with 18 U.S.C.\nSection 1734 solely to indicate this fact.\n3 The abbreviations used are: NKCC, Na-K-Cl cotransporter; cs, connecting segment;\nFR, flux rate; NMG, N-methyl glucamine; o, extracellular; R, regular; R, hypertonic\nregular; R, modified regular; sa, Squalus acanthias; saA, S. acanthias NKCC2A; saA/F,\nchimera in which tm2 is from saNKCC2A and cs1a from saNKCC2F; saF, S. acanthias\nNKCC2F; saF/A, chimera in which tm2 is from saNKCC2F and cs1a from saNKCC2A; tm,\ntransmembrane domain.\nsplicing of the primary transcript leads to the formation of three variants\n(called NKCC2B, -A, and -F) that are identical to one another except for\nthe exon 4-encoding region, which corresponds to the second trans-\nmembrane domain (tm2) and part of the following connecting segment\n(cs1a). Interestingly, these variants are distributed differentially along\nthe thick ascending limb as follows: B is in macula densa cells, A in the\nouter medullary stripe, and F in the inner medullary stripe (6, 7). In\ncontrast to NKCC2, the first isoform of NKCC (NKCC1) is widely dis-\ntributed, arising in non-polarized cells as well as in the basolateral pole\nof many secretory epithelia (8 –12).\nThe main function of both NKCC1 and NKCC2 is to couple the\nmovement of Na-K-Cl across the cell surface. Based on various\nstudies, the stoichiometry of ion transport has been shown to be 1Na:\n1K:2Cl, implying that there is no net movement of charge during the\ntranslocation cycle (13–15). In some studies, however, alternate trans-\nport modes have been proposed, e.g. 2Na:1K:3Cl for squid NKCC1\n(16), 1Na:1K:1Cl for NKCC2A (17, 18), and K/K exchange for\nboth NKCC1 and NKCC2A (17–19). We and others have hypothesized\nthat large differences in Cl affinities between the 2 Cl binding sites\ncould account for the apparent 1Na:1K:1Cl stoichiometry (17, 18),\nand incomplete reactions during the transport cycle, for the Na-inde-\npendent exchange mode (18 –19).\nThe kinetic characteristics of NKCC1 vary widely among species and\nthose of NKCC2, among the splice variants; for example, ion constants\nare lower for human NKCC1 compared with shark (sa) NKCC1 (9,\n22–26), and lower in NKCC2A compared with F (17, 18, 27, 28). In\nprevious work, we have exploited such differences to identify residues\nthat are involved in ion movement by the transporters. Indeed, struc-\nture-function studies of wild-type and chimeric NKCCs have shown\nthat the kinetic characteristics of these proteins are specified, at least in\npart, by variant residues that are localized in the tm2 and/or cs1a regions\n(18, 22–25). Interestingly, such studies have also led to the identification\nof a high affinity-Cl binding site in cs1a.\nChimera approaches can be used quite effectively to identify variant\nresidues that convey differences in the operations of homologous carri-\ners. The main disadvantage of such studies is that key residues will be\nmissed if they are fully conserved between the carriers. This limitation is\nillustrated by recent analyses in which the behavior of various chimeras\nled to slightly different interpretations of where affinity-modifying\ndomains are localized depending on which NKCC was used to gen-\nerate the mutants. Based on these analyses, for instance, it was not\npossible to determine whether the cs1a region of NKCC1 is involved\nin ion transport (18, 22–25).\nThe NKCC ion binding sites are known to be functionally interde-\npendent; that is, a change in ion affinity at one site induces a change in\nion affinity at another site (18 –21). One implication of this character-\nThe Role of tm2 in Cation Transport by NKCC2\nistic is that it could be difficult to determine which ion constants are\nmodified directly as a result of a residue substitution. Fortunately, we\nhave recently determined the manner in which these constants are\ninterrelated based on kinetic studies of wild-type NKCC1 expressed in\nHEK-293 cells (24, 25). These data, which were analyzed in the context\nof previous kinetic models (19 –21), indicate that at high extracellular\n(o) [Na] or [K], changes in either Na or K affinity will have small\neffects on Km (Cl), whereas at high [Cl]o, a change in Cl affinity will\nhave a larger effect on Km (Na) and Km (Rb).\nOne of the structure-function studies that was discussed above led to\nthe identification of five candidate residues in tm2-cs1a that play a def-\ninite role in Cl transport by NKCC2, but the same degree of resolution\nconcerning the structural determinants of cation transport was not\nobtained (18). The approach that was used here was valuable in this\nregard, allowing us to uncover two residues that are important in deter-\nmining Na and Rb affinities. Quite interestingly, our kinetic studies\nalso suggest that the residues identified belong to distinct ion binding\nsites in the tm2-cs1 region.\nMATERIALS AND METHODS\nOverview—Structure-function studies were carried out to identify\naffinity-modifying residues in the tm2-cs1a region of Squalus acanthias\n(sa) NKCC2A and saNKCC2F (called saA and saF in this work). Mutant\nsaNKCC2A and saNKCCF were expressed in Xenopus laevis oocytes\nand characterized through flux assays. Unless mentioned otherwise,\nchemicals, reagents, and kits were from different suppliers or collabo-\nrators.\ncDNAs—A total of 10 different cDNA constructs were used in these\nassays. They were subcloned in Pol1, a X. laevis oocyte expression vec-\ntor that is derived from pGEM and that contains the T7 promoter, a\nmultiple cloning site flanked by the X. laevis -globin untranslated\nregions and a poly(A) tract (17, 18, 29, 30). All of the constructs were\npropagated in XL1-blue cells (Stratagene) and purified on silica-based\ncolumns (Qiagen).\nsaA, saF, saA/F, and saF/A—These constructs were generated in pre-\nvious studies but were not recharacterized for this analysis (18). They\ncorrespond to wild-type saA and saF, and to a pair of complementary\nchimeras in which tm2 or cs1a was interchanged between saA and saF.\nThese chimeras are called saA/F (tm2 from A, cs1a from F) and saF/A\n(tm2 from F, cs1a from A). The wild-type saA and the tm2-cs1a regions\nof saA, saF, saA/F, and saF/A are represented in Fig. 1 through hydrop-\nathy plot models to illustrate how they differ between each other and to\nshow where the junction point of the chimeras occur, thereby providing\na rationale for the choice of substitutions that were created in the cur-\nrent work.\nsaF(I223L),\nsaA(S229C), saF(C229S), saA(C233V), and saF(V233C)—These constructs con-\nsist of saA or saF inserts in which one of the tm2-cs1a codons was\ninterchanged between the variants. All of the mutations were generated\nthrough pairs of mutagenic oligonucleotides (Sigma) using the\nQuikChange Mutagenesis kit (Stratagene). The resulting constructs\nwere called saA(x) or saF(x), where A and F indicate the NKCC2 variant\nin which the residue was substituted, and x, the substitution that was\ngenerated based on the amino acid number in saNKCC2. The muta-\ngenic oligonucleotides that were used to create these mutants are shown\nin TABLE ONE, and the position of the substituted residues are also\nillustrated in Fig. 1.\nIt should be mentioned here that the choice of substitutions was not\nbased only on differences in residues between saA and saF but also on\nthe results of recent structure-function analyses that were concerned\nwith the identification of ion transport sites in both NKCC1 and -2 (18,\n22–25). In saA/F or saF/A, for example, 9 residue substitutions were\ncreated in tm2 and 5 residue substitutions in cs1a, suggesting that a\nnumber of these residues were responsible for differences in kinetic\nbehaviors between saA and saF. However, based on the behavior of\nsaNKCC1 and human (h) NKCC1 chimeras, both these carriers also\nexhibit dramatic differences in ion affinities, we had predicted that non-\nconserved residues between the two NKCC1s as well as between high\nand low affinity wild-type NKCCs (from shark, rat, rabbit, human)\nwould be important in determining cation affinities, whereas noncon-\nserved residues between high affinity and low affinity NKCC2s (from\nshark, rat, rabbit) would be important in determining anion affinity (17,\n18, 22–28). This interpretation led to the suggestion that the main affin-\nity-modifying residues in saNKCC2 would be found at position 216,\n220, 223, 229, or 233.\nExpression in X. laevis Oocytes—To obtain synthesis of saNKCC2s in\nthis system, defolliculated stage V-VI oocytes were injected with\nsaNKCC2/Pol1-derived cRNA (5–25 ng/oocyte diluted in H2O) and\nThe Role of tm2 in Cation Transport by NKCC2\nca att act TGC ATC tct ATG tcc gct ata\nTGT acc aat gga GTT gtt cgt gga ggt gg\ngtc tta aca GGG TTG tct ACC tca gcc att\nTCG acc aac gga TGT gtt cga g\nctg gct act GTG gta aca tc\nctg agt acg ATA gta act gtc\nct ata gta aca tca CTT acg ggg ttg tc\ncg gtg gta act gtc ATA aca tgt atc tc\nca att act ggg ATC tct acc tct gc\nc tta aca tgt TTG tct atg tca gcc\ncc tct gct ata TGC aca aat gg\ng tca gcc att TCA acc aat g\nca aat gga GTT gtt cgt gga gg\ncc aat gga TGT gtt cga gga gg\nmaintained for 3 days at 18 °C in Barth’s (B) medium (see TABLE TWO)\n 125 M furosemide. Groups of oocytes were also injected with H2O\nalone as controls.\nsaNKCC2-expressing X. laevis\nof\nOocytes—They were performed as previously described (29, 30). Here,\nthe anti-P-NKCC immunoglobulin (Ig), which is specific to the phos-\nphorylated form of various NKCCs (29, 31), and which was a gift from\nDr. B. Forbush (Yale University), was used as primary antibody, whereas\nthe goat Alexa Fluor® 488-conjugated goat anti-rabbit anti-IgG (Molec-\nular Probes, A11008) was used as secondary antibody.\nKinetic Studies—Ion transport rates were determined by 86Rb influx\nmeasurements at 22 °C using different types of media (shown in\nTABLE TWO). Each of these media is derived from a regular (R)\nmedium that is iso-osmolar relative to plasma for the X. laevis frog. One\nof the modified media (called R) was obtained by adding 84 mOsM\nsucrose, and the other types of media (called R) were obtained by\nreplacing the ions of medium R with K or N-methyl gluconate (as\n2 (as anion substituents).\ncation substituents) or with gluconate  SO4\nBefore, the flux assay per se, furosemide was first removed through\nseveral rinses in medium R. Eggs were then subjected to two consecutive\nincubation steps: one of 60 min in medium R to activate the cotrans-\nporter and another of 45 min in various medium R  1–2 Ci/ml\n86Rb  10 M ouabain  250 M bumetanide to measure NKCC-\nspecific accumulation of the isotope. During this second 45-min time\ninterval, transport activity versus time is in a linear range, regardless of\nthe ion concentration used. Here, the various medium R were pro-\nduced by varying [Na] from 0 to 87 mM, [Rb] from 0.1 to 20 mM or\n[Cl] from 0 to 86 mM (see TABLE TWO and figure legends). Fluxes\nwere terminated with washes in a high K medium R  10 M oua-\nbain  250 M bumetanide, and oocytes were transferred to 96-well\nplates (1 oocyte/well) prefilled with 2% SDS and scintillation fluid. After\na short stabilization period, accumulated 86Rb was detected with the\nTopCountNXT microplate counter (Packard).\nTo minimize cell damage during the flux assays, all of the incubation\nsteps were carried out in 12-well plates by transferring oocytes with a\nPasteur pipette from one prefilled well to another, rather than by chang-\ning the media through aspiration and refilling of the same well. To avoid\nsolution contamination, however, each of these steps was performed\nusing three different wells prefilled with the same solution, the first two\nwells to incubate oocytes for a very short period of time, and the last well\nto complete the incubation step per se (except for the final wash during\nwhich all of the incubation times were brief). With this method, we have\nfound that ion concentrations in the final incubating solutions change\nby less than 0.5%.4\nIn each experiment, counts among several oocytes (generally 4 – 6)\nwere averaged and transposed into flux rates (FRs). The equation used\nfor this calculation was FRs  (counts/oocyte  [85Rb] in the flux\nsolution)  ([counts] in the flux solution  time of incubation in the\nflux solution), assuming that mean membrane surface areas between\ngroups of stage V-VI oocytes are very similar; for kinetic determinations\n(see below), mean FRs were also normalized to those obtained at the\nThe Role of tm2 in Cation Transport by NKCC2\nhighest ion concentration. All of these mean normalized or non-nor-\nmalized values were subsequently reaveraged from several replicate\nexperiments (n  2–5), and the data obtained were expressed as\nmean  S.E.\nIon constants (expressed as Km) and closeness of fits were calculated\nwith the programs PLOT (Biff Forbush) or SigmaPlot 4.00 for Windows\nusing the Michaelis-Menten equation (for a one binding site model), the\nHill equation with a coefficient (n) of 2 (for a two binding site model), or\nthe Hill equation with an undetermined value of n. In each case, least\nsquare analyses were carried out with a 3 or 4parameter equation (vari-\n Hill) to allow a 0 offset, assuming that 86Rb\nables: Vmax, Km, and Vmin\ninflux is never completely absent even if one of the cotransported ions is\nremoved from the flux solution (as occurs when the transporter is in the\nK/K exchange mode for example). In the current study, these kinetic\nparameters are presented as 1) averages of mean values from replicate\nexperiments (in which case differences between variables were analyzed\nthrough Student’s two tailed t-tests, assuming a null hypothesis for p\nvalues  0.05), and as 2) the results of single fits using activity versus\nconcentration data points that were also obtained by averaging mean\nvalues from replicate experiments. The latter method was employed to\nadjust for intra-experimental variability, which could introduce system-\natic errors, and because it is not possible to obtain time-matched data\nfor all of the conditions tested. The types of calculations carried out, and\nvariables that were used are specified further in the legends of tables or\nfigures.\nLast of all, it should be noted that kinetic parameters were calculated\nwithout subtracting bumetanide-sensitive 86Rb influxes measured in\nH20-injected eggs. In fact, these eggs exhibit very low levels of 86Rb\ninfluxes under the experimental conditions used for the flux assays (17,\n18, 29, 30). Hence, the dependence of saNKCC2 activity on [Na],\n[Rb], or [Cl] should remain unaltered whether this background activ-\nity is subtracted or not from the total counts. As will be discussed later,\nprevious studies have confirmed this prediction on multiple occasions\neven for heterologously expressed carriers with transport activities that\nwere as low as 3-fold above background.\nRESULTS\nPreamble—Throughout the following sections, the kinetic behaviors\nof our 10 mutants will be compared with those of four other carriers\n(saA, saF, saA/F, and saF/A) that were tested in a previous study using\nthe same expression system (18). We elected not to repeat previous\ncharacterizations for the following reasons. 1) We have observed that\nthe kinetic features of saNKCC2s in X. laevis oocytes were very repro-\nducible over time. 2) The functional assays were carried out by the same\ninvestigators. 3) The main conclusions in this article are based on the\nkinetic characteristics of the 10 novel mutants. 4) As will be reported\nlater, mutant carriers in which a substitution was predicted to produce\nno effect on kinetic parameters behaved as such, exhibiting ion con-\nstants that were identical to previously published values (18); for saA,\nthese values, expressed as Km (Na), Km (Rb), and Km (Cl), were 15.3, 2.2,\nand 6.9 mM, and for saF, 93.0, 4.6, and 69.1 mM. 5) Recent experiments\n(n  1) revealed that the kinetic behavior of saA/F and saF/A was con-\nsistent with expectation based on our previous publication (18), which\nshowed Km (Na), Km (Rb), and Km (Cl) for saA/F of 18.0, 2.2, and 70.0\nmM, and Km for saF/A of 22.6, 1.7, and 8.7 mM.\nExpression of saNKCC2s in Oocytes—Immunofluorescence studies of\noocytes expressing wild-type or mutant saNKCC2s revealed that the\nanti-P-NKCC Ig, which was used as primary antibody, labels the heter-\nologous proteins specifically, and that most of the signal occurs at the\ncell surface (results not shown). These studies also revealed that protein\nexpression levels were relatively similar among all of the saNKCC-ex-\npressing oocytes tested. Functional analyses, during which cotrans-\nporter activity was determined under subsaturating concentrations of\nNa, Rb, and Cl, confirmed the localization data in showing that all\nof the expressed carriers generate bumetanide-sensitive 86Rb influx\nrates that are clearly above those of the X. laevis NKCC1. As seen in Fig.\n2, e.g. these rates are between 0.8 and 7.3 nmol/oocyte/h for the mutant\nNKCC2s, that is, at least 4-fold above background for all of the carriers\nor 8-fold if saA(S229C) is excluded.\nKinetics of Na Transport: Role of tm2—As mentioned earlier, we and\nothers (17, 18, 27, 28) have observed that saA and saF exhibit substantial\ndifferences in regard to apparent Na affinities, that is, Km (Na) for saA\nis 5-fold lower compared with saF. We have also observed that\nKm (Na) remains low even when the tm2 of saA is replaced by that of\nsaF, as in the chimera saF/A, suggesting that variant residues in this\ndomain convey no differences in Na affinity as long as cs1a is from saA\n(18). Hence, replacing residue 216 or 220 in saA by those of saF should\nnot lead to changes in Km (Na) values.\nTo confirm this prediction, we measured the dependence of 86Rb\ninflux on [Na] for saA(I216V) and saA(I220L), and analyzed results in the\nlight of previously observed behaviors for saA, saF, and saF/A (18). Data\nare shown in Fig. 3A (upper row) fit by a model of Na binding at a single\nsite, and Km (Na) derived from the activity versus [ion] relationships are\nshown on Fig. 3B (first 2 bars). In TABLE THREE, similar data are\npresented except that the Km values were calculated with best values of\nHill coefficients. As expected, these mutants exhibit high Na affinities\nthat are in the range of those reported for saA and saF/A. It can be noted,\nadditionally, that NKCC activity at 0 mM Na\no is quite substantial, as\nhad also been reported for saA and saF/A, suggesting that residues 216\nor 220 do not account for the ability of certain NKCCs to induce Rb/\nRb(K) exchange at the cell surface.\nIn our recent structure-function study of saNKCC2 mutants, the kinetic\nbehavior of saF/A in regard to Na transport was also seen to be very similar\nto that of saA/F (18). These findings, which were somewhat unexpected to\nus, imply that Km (Na) for saF decreases following a Phe 3 Ala tm2 substi-\ntution, and that variant residues in this domain do convey differences in\nNa affinity but only when cs1a is from saF. They also imply that differ-\nences between saF and saA/F can be exploited to determine which residues\nin the tm2 of saA are important in determining Km (Na).\nThe Role of tm2 in Cation Transport by NKCC2\nmM\n16.8  2.4\n22.7  5.2\n93.3  15.1\n38.2  5.9\n11.7  4.1\n15.5  5.8\n15.7  2.9\n85.5  14.1\n84.6  8.5\n82.0  10.4\n1.6  0.5\n1.7  0.4\n2.5  0.3\n4.5  0.9\n2.5  0.2\n2.2  0.4\n1.0  0.6\n4.7  1.1\n6.6  1.3\n6.7  1.5\n12.8  3.4\n8.3  5.6\n102.2  25.1\n78.0  12.4\n22.0  3.7\n20.3  3.4\n13.5  4.6\n85.9  10.2\n65.8  13.3\n66.5  11.9\n0.82  0.21\n0.71  0.24\n1.04  0.12\n1.13  0.17\n1.03  0.03\n0.81  0.21\n0.71  0.15\n1.03  0.06\n1.04  0.16\n1.11  0.09\n1.18  0.07\n1.06  0.14\n1.12  0.03\n0.97  0.06\n0.96  0.19\n1.10  0.15\n1.02  0.08\n1.04  0.17\n0.82  0.22\n0.91  0.23\n1.13  0.17\n0.92  0.21\n1.74  0.12\n1.73  0.13\n0.88  0.19\n0.87  0.11\n1.21  0.07\n1.71  0.09\n1.76  0.05\n1.80  0.21\n3B (last 2 bars) and TABLE THREE; data are illustrated as above using\nthe same models of ion binding. It is seen here, that Km (Na) for\nsaF(V216I) is much higher than those for saA(I216V) or saA(I220L), but that\nKm (Na) for saF(L220I) is intermediate between those measured for the\nsaA mutants and saF(V216I). These new observations suggest that residue\n220 in the tm2 of saF accounts, at least in part, for the functional char-\nacteristic that was displayed by saA/F in regard to Na transport. It can\nalso be noted here that carrier activity at 0 mM is, as would have been\nexpected, lower for saF(V216I) and saF(L220I) compared with saA(I216V)\nand saA(I220L), but similar to that reported for saF and saA/F previously.\nKinetics of Rb Transport: Role of tm2—For either of the recently\ncharacterized saA, saF/A, saA/F, and saF, the kinetics of Rb transport\nwere shown to be very similar to those of Na transport with the excep-\ntion that NKCC activity at 0 mM Rb and 0 mM K is equally low for all\nthe carriers tested (18). Otherwise, saA, saF/A, and saA/F display a\nsimilar affinity for Rb that is 2-fold higher compared with saF and\nthe activity versus [ion] relationships is best fit with a model of substrate\nbinding at one site. Accordingly, Km (Rb) values for saA(I216V) and\nsaA(I220L) should be similar to those for saA and saF/A, whereas Km (Rb)\nvalues for saF(V216I) or saF(L220I) could potentially differ when compared\nwith those for saF.\nThe kinetics of Rb transport by saA(I216V), saA(I220L), saF(V216I), and\nsaF(L220I) are summarized in Fig. 4 and TABLE THREE with the repre-\nsentation schemes and binding models that were used to describe Na\ntransport. Here, the behavior of saA(I216V) and saA(I220L) confirms our\npredictions; Km (Rb) values for these mutants are in the low range, sim-\nilar to those reported for saA and saA/F. The behaviors of saF(V216I) and\nsaF(L220I), on the other hand, differ from one another; here, Km (Rb) for\nsaF(L220I) is similar to that reported for saF, but for saF(V216I), it is inter-\nmediate between that for saA(I216V) or saA(I220L) and saF(L220I). These\nfindings suggest that residue 216 accounts to some extent for the\ndecrease in Rb affinity that occurs following a Phe 3 Ala tm2 substi-\ntution. They also indicate that residues 216 and 220 play different roles\nin cation transport.\nKinetics of Cl Transport: Role of tm2—Kinetic parameters for the\nrecently characterized saF/A were found to be similar to those for saA in\nregard to Cl transport, suggesting that substitution of residue 216 or\n220 in saA by the corresponding residue in saF will produce no effects\non Km (Cl) (18). As seen in Fig. 5A (upper row), Fig. 5B (first 2 bars) and\nThe Role of tm2 in Cation Transport by NKCC2\nIn our structure-function studies of chimeric saNKCC2s, we\nreported that Cl transport kinetics for saF/A are identical to those for\nsaA, but that the behavior of saA/F is unlike that of either saA or saF\n(18). Indeed, Km (Cl) for saA/F is similar to that for saF but calculated\nHill(Cl\n) similar to that for saA. Hence, substituting residue 216 or 220\nin the tm2 of saF by the corresponding residue in saA should lead to no\nchanges in Km (Cl) values but could still alter the calculated Hill(Cl\n)s.\nBased on the kinetic characteristics of saF(V216I) and saF(L220I) (lower row\nof Fig. 5A, last 2 bars of Fig. 5B, and TABLE THREE), it is in fact seen\nthat both Km (Cl) and Hill(Cl\n) values are similar to those reported for\nsaF. These results suggest that other residues in the tm2 of saF or,\nconceivably, the presence of both residues in this domain account for\ndifferences in apparent number of transport sites for Cl.\nKinetics of Cation Transport: Role of cs1a—The kinetic behavior of\nsaA/F and saF/A showed that cs1a could also play an important, perhaps\nindirect role in cation transport (18). Accordingly, we measured the\ndependence of saA(L223I), saA(S229C), saA(C233V), saF(I223L), saF(C229S),\nsaF(V233C) on [Na] or [Rb], expecting that residue substitutions in the\ncs1a of saA would generate no kinetic changes but that residues substi-\ntutions in the cs1a of saF could alter cation affinities substantially.\nResults of these studies are illustrated in Figs. 6 and 7 using a single\nsite ion binding model and in TABLE THREE. As anticipated, Km values\nfor saA(L223I), saA(S229C), and saA(C233V) (higher rows) are all in the low\nrange, but surprisingly, those for saF(I223L), saF(C229S), or saF(V233C)\n(lower rows) are all in the high range, similar to those reported for saF.\nThese observations suggest that other residues in the cs1a of saF or\nperhaps a combination of specific residues among those that were sub-\nstituted account for the previously observed differences in Km (Na) and\nKm (Rb) between the saF and saF/A.\nKinetics Cl Transport: Role of cs1a—Although tm2 has been shown\nto play an important role in Cl transport, we have been unable to\nidentify the individual residues involved (18). Interestingly, differences\nbetween saF and saA/F had revealed the existence of a low affinity Cl\nbinding site in tm2 (Km (Cl) values for both carriers are in the high range\nbut calculated Hill(Cl\n) values are lower for saF/A) implying that the\nThe Role of tm2 in Cation Transport by NKCC2\nhigh affinity Cl binding site is in cs1a. In fact, the behavior of saA and\nsaF/A in our recent study suggested that this is precisely the case and\ncould have explained why cation affinities for saF/A are surprisingly\nhigh. However, because neither of residues 223, 229, or 233 have been\nfound to modulate cation transport in the current study, it is unlikely\nthat Km (Cl) values for saA(L223I), saA(S229C), saA(C233V), saF(I223L),\nsaF(C229S), and saF(V233C) will differ from those of the parent carriers.\nThe data in Fig. 8 and TABLE THREE, which show the dependence of\nthese six mutants on [Cl] and kinetic parameters derived from the\nactivity versus [ion] relationships, are once again consistent with expec-\ntations. Indeed, all of the saAx mutants are seen to have higher affinities\nthan the saFx mutants, and the range of Km values observed are similar\nto those reported for their parent carriers. Hence, the residues or com-\nbination of residues that account for differences in Cl transport kinet-\nics between saF and saF/A or between saA and saA/F have not been\nrevealed through these mutagenic analyses.\nDISCUSSION\nIn this study, differences in kinetic properties between saA and saF\nwere exploited to identify novel ion transport sites in the tm2-cs1a\nregion of NKCC. Our main approach was to interchange variant resi-\ndues between saNKCC2s, creating a single point substitution for each\nmutant, and determine which of these residues account for the distinc-\ntive behaviors. The role of variant residues was analyzed further by\ncomparing the effects of single point substitutions to those of combined\nsubstitutions, exploiting our recent observation that both saA/F and\nsaF/A also differ in kinetic behavior when compared with either saA or\nsaF (18). Based on such analyses, we were able to identify two residues\nthat account for differences in cation affinity between saA and saF.\nQuite remarkably, we found that one residue (position 220) is important\nin determining Na affinity, whereas the other residue (position 216 in\nthe same domain) is important in determining Rb affinity.\nHydropathy plot models of NKCC2 have led to the prediction that the\n5-end of exon 4 encodes tm2 and the 3-end of this exon, cs1a. They\nhave also led to the prediction that the central domain of NKCC2 is\nconstituted of 12 tms that are organized as -helicoidal protein seg-\nments (8 –11, 32, 33). For NKCC1, a related carrier that share high levels\nof homology with NKCC2, such models have been confirmed indirectly\nby the identification of regions that behave as tm when expressed in the\nform of fusion proteins by rabbit reticulocyte lysates (34). Based on\nthese structural considerations, the identification of two residues that\nmodulate Na and Rb (K) affinity differentially, but that are very\nclose to one another in the tm2 region, bears a number of important\nimplications.\nOne such implication is that the ion transport sites of NKCC,\nalthough distinct kinetically, may be capable of interacting with one and\nanother through certain key residues. Indeed, the structural models of\nthe tm2-cs1a region also predicts that residues 216 and 220 (Ile and Ile in\nsaA and Val and Leu in saF) are adjacent to one another on the same\nvertical axis of the -helix, suggesting that their peptide backbones\nThe Role of tm2 in Cation Transport by NKCC2\nand/or aliphatic R groups are connected through H atoms (35, 36). In\nthe case of NKCC, for which ions are proposed to move single file down\na translocation pocket (19 –21, 24, 25), such physical interactions could\nrepresent a key aspect of the carrier’s modus operandi, accounting, in\nparticular, for the kinetic interdependence of the ion binding sites.\nAnother implication of the findings that are discussed here is that we\nhave identified a coherent structural correlate to the widely accepted\nmodel of ordered ion binding proposed by Lytle et al. (19). Indeed, this\nmodel predicts that extracellular ions bind to the translocation pocket\nof NKCCs in the order Na-Cl-K-Cl and are released from this\npocket on the inside of the cell in the exact same order. One would thus\nassume based on such a binding scheme that the Na transport site is\nseated more deeply than the K transport site relative to the extracel-\nlular space. Our findings are in fact consistent with this interpretation in\nthat residue 216, which is localized in the middle of tm2, confers differ-\nences in Km (Rb) between saA and saF, and residue 220, which is local-\nized further into the bilayer, confers differences in Km (Na).\nA third implication related to the identification of affinity-modifying\nresidues that are “ion-specific” is that the observed changes in Km values\nwere probably not due to substitution-induced conformational arti-\nfacts. Because Michaelis constants for ion movement through a sym-\nporter are related to several kinetic parameters that are not limited to\nthe “true binding affinities” for transported substrates (14, 24, 25), such\nartifacts could have accounted for a change in kinetic behavior by affect-\ning, e.g. a rate constant during the translocation cycle. Under such cir-\ncumstances, however, one would have expected parallel changes in Km\nvalues for all of the cotransported ions. Our results are thus reconcilable\nwith a simple model in which Km values were altered individually\nthrough substitutions of affinity-modifying residues.\nResidues 216 and 220 are nonpolar and, accordingly, predicted to lie\non the hydrophobic pole of the -helix. This localization does not nec-\nessarily imply that these two residues play no role in ion transport.\nIndeed, their peptide backbone or their R groups could still be close to\nthe hydrophilic pole of the -helix and interact directly with ions. Alter-\nnatively, residues 216 and 220 could play an important role in ion coor-\ndination from the hydrophobic pole of the -helix by influencing ion\nbinding sites through local steric interactions. To this effect, it should be\nmentioned that the current mutagenic analysis was design to identify\nresidues that confer differences in kinetic parameters and that such\nresidues, even if they are included in the Na-K-Cl transport sites, may\nnot be the same as those that are considered essential for substrate\nbinding.\nWe have stated earlier that ion constants for NKCC1 are interrelated\nin a predictable manner but that the effect of a change in one constant\nwas not necessarily the same on all of the other constants (19 –21, 24,\n25). For example, it has been shown that the apparent affinity of an ion\nis most influenced by changes in the true affinities of the ion that binds\nnext in the translocation pocket, implying that the effect of change in\nKm (Cl) on Km (Na) and Km (Rb) will be more important than the effect\nof a change in Km (Na) and Km (Rb) on Km (Cl). Based on such inter-\nrelationships, it is reasonable to propose that the isolated decrease in\nKm (Na) resulting from a residue substitution at amino acid number 220\nin saF was related to an increase in true Na affinity, and that the\nisolated decrease in Km (Rb) resulting from a residue substitution at\namino acid number 216 was related to an increase in true Rb affinity.\nThe validity of these interpretations is substantiated further by the find-\ning that none of the substitutions in cs1a led to changes Km (Na) or\nKm (Rb).\nCertain residues in the transport sites of NKCC1 could influence ion\naffinity only when they operate in concert with specific sets of contigu-\nous residues. Hence, the importance of variant residues in conferring\nkinetic differences between saA and saF could have gone undetected in\nthis work because our main approach was to determine the effect of\nsingle point mutations on Michaelis parameters. An example that illus-\ntrates this possibility comes from the observation that Km (Cl) values for\nsaF/A, which corresponds to a saNKCC2 harboring five substitutions in\ncs1a, were significantly different from those for saA (18), whereas\nKm (Cl) values for saA(L223I), saA(S229C), or saA(C233V), which correspond\nto saNKCC2s harboring single point substitutions in cs1a, were identi-\ncal to those for saA. It may be thus useful in future studies to examine\nthe importance of cs1a in conferring kinetic singularities by exchanging\nvariant residues 2 by 2 or 3 by 3.\nThe concept of affinity-modifying residue groups might also explain\nwhy a number of the complementary mutants did not exhibit reciprocal\nkinetic behaviors. As described under “Results,” for example, substitu-\ntions in the tm2 of saA (I216V or I220L) led to no changes in kinetic\nconstants, whereas substitutions in the tm2 of saF (V216I or L220I) did\nproduce very noticeable changes. These findings imply that certain\ncombinations of residues (e.g. Gly222-Leu223-Thr225-Ser229-Cys233) may\nplay such a preponderant role in determining ion constants, either\ndirectly or through conformational interactions between the transport\nsites, that they could mask the importance of contiguous affinity-mod-\nifying residues (Ile216-Ile220 or Val216-Leu220), whereas other combina-\ntions of residues at the same positions (e.g. Cys222-Ile223-Met225-Cys229-\nVal233) may play a less influential role.\nIn this work, functional expression levels tended to differ among\nmutants (Fig. 2), whereas protein expression levels were relatively sim-\nilar based on the IF studies. Although, in principle, these observations\ncould suggest that certain substitutions altered Vmax, it is not possible at\nthis time to draw any firm conclusions in this regard without the avail-\nability of more precise quantitative estimates of protein expression at\nthe cell surface. On the other hand, it appears unlikely that changes in\ncell surface delivery or in Vmax, if they occurred, accounted for the\nobserved changes in Km values. Indeed, previous studies in oocytes have\nshown that the dependence of saA on [Na], [Rb], or [Cl] remains\nstable whether the carrier is expressed at low or high levels by varying\nthe quantities of cRNA injected.4 Here, in addition, we have observed\nthat all of the cs1a mutants that were expected to behave as saA or as saF\ndid so despite the noted differences in Vmax.\nIn conclusion, we have identified two residues in NKCC that appear to\nplay an important role in cation transport. These residues, which are in the\ntm2-cs1a region of saNKCC2 (position 216 and 220) and which differ\nbetween saA and saF, convey variant behaviors to each of the wild-type\ncarrier and they also modulate Km (Na) and Km (Rb) differentially. Interest-\ningly, the proximity in which they occur implies that the interdependence\nof kinetic constants for this type of carrier could be mediated through com-\nplex physical interactions between the ion binding sites. Based on the effect\nof various substitutions on Km (Cl), in addition, we can postulate that\nkinetic constants for the NKCC transport system are probably each speci-\nfied by different sets of residue combinations.\nThe Role of tm2 in Cation Transport by NKCC2\nAcknowledgments—We thank Luc Caron and Micheline Noe¨l for superb tech-\nnical assistance.\n32. Hebert, S. C., Mount, D. B., and Gamba, G. (2004) Pflugers Arch. 447, 580 –593\n33. Park, J. H., and Saier, M. H. Jr. (1996) J. Membr. Biol. 149, 161–168\n34. Gerelsaikhan, T., and Turner, R. J. (2000) J. Biol. Chem. 275, 40471– 40477\n35. Chou, P. Y., and Fasman, G. D. (1978) Adv. Enzymol. 47, 45–148\n36. Doyle, D. A. (2004) Eur. Biophys. J. 33, 175–179",
         "E. Gagnon,M. Bergeron,Nikolas D. Daigle,Marie-Hélène Lefoll,P. Isenring",
         null
        ],
        [
         "39",
         "00295936451c8a6b8a982d6ba9ab64cb647370ae",
         null,
         null,
         null,
         null,
         null,
         null,
         "Kerry‐Ann da Costa,Christopher E. Gaffney,L. Fischer,S. Zeisel",
         "captcha"
        ],
        [
         "40",
         "002a108f32eb157c0b88aea3c6f29d4b614c3b78",
         null,
         null,
         null,
         null,
         null,
         null,
         "C. Bowman,K. Turner,M. Sar,N. Barlow,K. Gaido,P. Foster",
         "captcha"
        ],
        [
         "41",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         null,
         null,
         null,
         null,
         null,
         null,
         "A. Wax,J. Pyhtila,R. Graf,R. Nines,C. Boone,R. Dasari,M. Feld,V. Steele,G. Stoner",
         "captcha"
        ],
        [
         "42",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "",
         "",
         "",
         "",
         "",
         "",
         "T. Brosch,D. Sharma",
         null
        ],
        [
         "43",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "1)Shionogi Discovery Research Laboratories, Shionogi Co., Ltd., 3–1–1 Futaba-cho, Toyonaka 561-0825,\n2)Division of Immunology and Embryology, Department of Cell Biology, Tohoku University School of\nMedicine, Sendai 980-8575, 3)Division of Dermatology, Department of Clinical Molecular\nMedicine, Kobe University Graduate School of Medicine, Kobe 650-0017, Japan\nAbstract:  WBN/Kob-Ht rats (Ht-rats) raised under conventional conditions spontaneously\ndeveloped dermatitis.  In this study, we carried out histopathological analysis to elucidate\nthe  pathological  features  of  the  dermatitis  in  Ht-rats.    We  then  tried  to  detect\nStaphylococcus species recovered from the skin lesions of Ht-rats.  We also measured the\nserum levels of total IgE, IL-4 and IFN-γ in these rats.  The histopathological data indicated\nthat  inflammatory  cells  had  infiltrated  the  skin  lesions.    Staphylococcus  aureus  was\nrecovered from the skin lesions, and the serum levels of total IgE and IL-4 were elevated in\nHt-rats with dermatitis.  These results suggest that dermatitis in Ht-rats is similar to that in\nthe DS-Nh mice, which has recently been proposed as an animal model for human atopic\ndermatitis.\nKey words:  atopic dermatitis, DS-Nh mice, WBN/Kob-Ht rats\nThe WBN/Kob-Ht rat (Ht-rat) is a spontaneous hair-\nless mutant of the Wistar strain that was established at\nIshikawa  Laboratory  Animal  Company,  Saitama,  Ja-\npan.    The  inheritance  mode  of  the  Ht  mutation  is\nautosomal  dominant,  and  the  Ht  locus  is  mapped  on\nchromosome 10 [1].  Since the inheritance mode of the\nHt gene is dominant and Ht homozygote rats have high\nmorbidity, the Ht-rat used for this study was of the Ht\nheterozygote type.  In a previous study, these rats spon-\ntaneously  developed  dermatitis  under  conventional\nconditions, with an incidence of approximately 4% at\n20 weeks of age [17].  However, the pathogenic mecha-\nnism  and  characteristics  of  the  dermatitis  were  not\nexamined from the viewpoint of studying allergy.",
         "In human atopic dermatitis (AD), an increase in the\nnumber of infiltrating CD4- bearing T cells, mast cells,\nand eosinophils has been detected in acute skin lesions\nas compared with the normal skin [8, 9, 14].  The le-\nsions show a significant increase in the number of cells\nexpressing IL-4, IL-5 and IL-13 mRNA, this suggest-\ning preferential accumulation of Th2-type cells.  In ad-\ndition, an increase in the expression of IL-4 and of IL-5\nhas been detected in CD4- and CD8-bearing T cells in\npatients  with  AD  [12,  18].    The  Th2-type  cytokines\nhave a critical role in the initiation of the allergic re-\nsponse in Staphylococcus aureus-associated dermatitis\n[10].  IL-4 plays an important role in the induction of\nIgE  production  [13]  and  as  an  inhibitor  of  bacterial\nclearance  [5,  16].    Kasamatsu  et  al.  demonstrated  an\nincreased amount of serum IL-4 in acute AD compared\nwith  the  normal  control  [7].    Furthermore,  up  to  107\ncolony-forming units of S. aureus can be isolated from\nskin  lesions  of  more  than  90%  of  AD  patients  [11],\nwhile only 5% of normal subjects carry S. aureus on\ntheir skin.\nIn this study, we analyzed dermatitis in the Ht-rat to\npursue its potential as an animal model for investigat-\ning the pathogenesis of atopic dermatitis (AD) and to\ndevelop new therapeutic approaches or drugs to treat\nAD.  We discuss the characteristics of Ht-rats in com-\nparison  with  Nh-mice,  which  have  recently  been\nproposed as an animal model for human AD [3, 4, 19].\nHt-rats kept under a conventional condition until 25\nweeks of age displayed prominent symptoms of derma-\ntitis  including  edema  and  erythema  at  the  neck  (Fig.\n1A).  Dermatitis was observed in all Ht-rats that were\nmaintained under conventional conditions for 20 weeks,\nbut not in those maintained under SPF conditions (Fig.\n1B).  The rats could develop dermatitis only under con-\nventional conditions.  To evaluate the involvement of\nbacteria in the formation of the lesions, bacterial cul-\ntures were obtained from the facial skin surface of each\nHt-rat  kept  under  conventional  or  SPF  conditions.\nSamples  were  inoculated  onto  a  salt,  egg  yolk,  agar\nplate  (Nissui,  Tokyo,  Japan),  and  incubated  at  37°C.\nTen colonies per rat were randomly picked up, and the\nsub- species of Staphylococci were identified with an\nAN-ID Test-SP18 kit (Nissui, Tokyo, Japan).\nThe animals were humanely euthanized in accordance\nwith  the  guidelines  for  animal  experimentation  at\nShionogi  Research  Laboratories.    Skin  samples  were\nfixed in formalin or frozen in a freezer.  Four-microme-\nter thick paraffin sections of skin lesions of the Ht-rat\nwith or without dermatitis were stained with hematoxy-\nlin-eosin (HE) for histopathological observation, acidic\ntoluidine  blue  to  detect  mast  cells,  and  by  the  Luna\nmethod for eosinophils.  Frozen sections of the Ht-rat\nskin  with  or  without  dermatitis  were  immunostained\nusing  mouse  anti-rat  CD4  (OX-35,  PharMingen,  San\nDiego,  CA,  USA)  and  mouse  anti-rat  CD8a  (OX-8,\nPharMingen) antibodies.  Positive reactions on the sec-\ntions were visualized with peroxidase-diaminobenzidine,\nand the number of positive cells was counted.\nStaphylococcus  aureus  was  not  isolated  from  25-\nweek-old Ht-rats maintained under the SPF condition\n(Table 1), and approximately 70% of the isolated bac-\nteria  from  the  animals  housed  conventionally  were\ncommon  Staphylococcus  species  (S.  cohniiB  and  S.\nxylosus); the rest could not be identified.  These bacte-\nrial  strains  were  completely  replaced  by  S.  aureus  in\nconventionally  housed  Ht-rats  with  dermatitis  (Table\n1).  In view of these results, S. aureus appears to play\nan important role in the development of dermatitis in\nHt-rats.  Significant histopathological changes observed\nin the skin lesion of all Ht-rats included hyperkeratosis,\nacanthosis, slight intracellular edema of the epidermis,\nslight swelling of the epidermal cells, and the presence\nof inflammatory cells (Fig. 2A).  Higher levels of eosi-\nnophils  and  whole  and  degranulated  mast  cells  were\nobserved in the skin lesions of Ht-rats with dermatitis\nas compared with skin samples from the rats without\ndermatitis (Table 1, Figs. 2Ac and 2Ad).  In rats main-\ntained under the conventional condition, the number of\nCD4-bearing  T  cells  increased,  while  CD8-bearing  T\ncells  appeared  to  show  a  slight  increase  in  number.\nThe  ratio  of  CD4  to  CD8  cells  was  constantly  high.\nBoth CD4- and CD8-bearing T cells were rarely found\nin Ht-rats maintained under the SPF condition (Table\n1, Figs. 2Ae and 2Af).\nWe measured the serum IgE and IL-4 levels, since\nIgE and IL-4 play important roles in the pathogenesis\nof human AD [2, 12].  Serum samples from rats main-\ntained under the SPF or conventional conditions were\ncollected.  Total IgE and IL-4 levels in the serum were\nmeasured using sandwich ELISA kits (Bethyl, Mont-\ngomery,  TX,  USA;  and  Biosource,  Camarillo,  CA,\nUSA).  The level of serum IgE increased only in the\nHt-rats  that  were  maintained  under  the  conventional\ncondition, but not in those maintained under the SPF\nconditions.  Serum IL-4 was detected only in Ht-rats\nwith dermatitis (Fig. 2B).  Serum IFN-γ was not detect-\nable in any of the rats (data not shown).  Considering\nthe results, we propose that the Ht-rat can serve as a\nnew animal model for human AD.\nThe present rat model was compared with the DS-Nh\nmouse, which was recently introduced by Hikita et al.\n[4] and Yoshioka et al. [19].  The rat and mouse mod-\nels  may  be  similar  to  human  AD  with  respect  to  the\nfollowing features [4, 8, 11, 14, 15]: (1) S. aureus can\nbe isolated from skin lesions, (2) serum levels of IgE\nand IL-4 increase significantly, and (3) the number of\nwhole mast cells and CD4-bearing T cells significantly\nincrease.  Although both rodent models have much in\ncommon, there are some differences.  We found that all\nthe  S.  aureus  from  the  DS-Nh  mice  with  dermatitis\nsecrete  staphylococcal  enterotoxin  (SE)  C,  while  S.\naureus from the Ht-rats did not secrete superantigens\n(SEA, SEB, SEC, SED, SEE, and toxic shock syndrome\ntoxin)  (data  not  shown).    In  addition,  the  number  of\neosinophils significantly increased only in rats that were\nmaintained  under  the  conventional  condition,  but  no\nincrease was observed in the mice.\nAs described above, there are some genetic similari-\nties  between  Ht  and  Nh  mutations.    The  inheritance\nmode of both mutations is autosomal dominant [1, 3].\nThe Nh locus is mapped on mouse chromosome 11 and\nthe Ht locus on rat chromosome 10 [1, 3], and both the\ngenes  are  located  in  contiguity  with  the  linkage  site\n(unpublished data).  Furthermore, there are some phe-\nnotypical  similarities  between  both  mutated  rodents.\nHigher levels of mast cells were observed in the skin\nlesions  of  Ht-  or  Nh-mutated  rodents  compared  with\nskin samples of age-matched rodents without these mu-\ntations (data not shown).  In Ht-rats and Nh-mice, we\nconsidered  that  the  increasing  number  of  mast  cells\ncaused by these mutations and S. aureus colonization\nmay play an important role in developing AD-like der-\nmatitis.    We  are  currently  confirming  whether  these\ngenes are the same by using the positional cloning tech-\nnique.\nIn conclusion, Ht-rats develop dermatitis similar to\nthat in DS-Nh mice and should be useful as an addi-\ntional model for human atopic dermatitis.",
         "",
         "",
         "",
         "1)Shionogi Discovery Research Laboratories, Shionogi Co., Ltd., 3–1–1 Futaba-cho, Toyonaka 561-0825,\n2)Division of Immunology and Embryology, Department of Cell Biology, Tohoku University School of\nMedicine, Sendai 980-8575, 3)Division of Dermatology, Department of Clinical Molecular\nMedicine, Kobe University Graduate School of Medicine, Kobe 650-0017, Japan\nAbstract:  WBN/Kob-Ht rats (Ht-rats) raised under conventional conditions spontaneously\ndeveloped dermatitis.  In this study, we carried out histopathological analysis to elucidate\nthe  pathological  features  of  the  dermatitis  in  Ht-rats.    We  then  tried  to  detect\nStaphylococcus species recovered from the skin lesions of Ht-rats.  We also measured the\nserum levels of total IgE, IL-4 and IFN-γ in these rats.  The histopathological data indicated\nthat  inflammatory  cells  had  infiltrated  the  skin  lesions.    Staphylococcus  aureus  was\nrecovered from the skin lesions, and the serum levels of total IgE and IL-4 were elevated in\nHt-rats with dermatitis.  These results suggest that dermatitis in Ht-rats is similar to that in\nthe DS-Nh mice, which has recently been proposed as an animal model for human atopic\ndermatitis.\nKey words:  atopic dermatitis, DS-Nh mice, WBN/Kob-Ht rats\nThe WBN/Kob-Ht rat (Ht-rat) is a spontaneous hair-\nless mutant of the Wistar strain that was established at\nIshikawa  Laboratory  Animal  Company,  Saitama,  Ja-\npan.    The  inheritance  mode  of  the  Ht  mutation  is\nautosomal  dominant,  and  the  Ht  locus  is  mapped  on\nchromosome 10 [1].  Since the inheritance mode of the\nHt gene is dominant and Ht homozygote rats have high\nmorbidity, the Ht-rat used for this study was of the Ht\nheterozygote type.  In a previous study, these rats spon-\ntaneously  developed  dermatitis  under  conventional\nconditions, with an incidence of approximately 4% at\n20 weeks of age [17].  However, the pathogenic mecha-\nnism  and  characteristics  of  the  dermatitis  were  not\nexamined from the viewpoint of studying allergy.\nIn human atopic dermatitis (AD), an increase in the\nnumber of infiltrating CD4- bearing T cells, mast cells,\nand eosinophils has been detected in acute skin lesions\nas compared with the normal skin [8, 9, 14].  The le-\nsions show a significant increase in the number of cells\nexpressing IL-4, IL-5 and IL-13 mRNA, this suggest-\ning preferential accumulation of Th2-type cells.  In ad-\ndition, an increase in the expression of IL-4 and of IL-5\nhas been detected in CD4- and CD8-bearing T cells in\npatients  with  AD  [12,  18].    The  Th2-type  cytokines\nhave a critical role in the initiation of the allergic re-\nsponse in Staphylococcus aureus-associated dermatitis\n[10].  IL-4 plays an important role in the induction of\nIgE  production  [13]  and  as  an  inhibitor  of  bacterial\nclearance  [5,  16].    Kasamatsu  et  al.  demonstrated  an\nincreased amount of serum IL-4 in acute AD compared\nwith  the  normal  control  [7].    Furthermore,  up  to  107\ncolony-forming units of S. aureus can be isolated from\nskin  lesions  of  more  than  90%  of  AD  patients  [11],\nwhile only 5% of normal subjects carry S. aureus on\ntheir skin.\nIn this study, we analyzed dermatitis in the Ht-rat to\npursue its potential as an animal model for investigat-\ning the pathogenesis of atopic dermatitis (AD) and to\ndevelop new therapeutic approaches or drugs to treat\nAD.  We discuss the characteristics of Ht-rats in com-\nparison  with  Nh-mice,  which  have  recently  been\nproposed as an animal model for human AD [3, 4, 19].\nHt-rats kept under a conventional condition until 25\nweeks of age displayed prominent symptoms of derma-\ntitis  including  edema  and  erythema  at  the  neck  (Fig.\n1A).  Dermatitis was observed in all Ht-rats that were\nmaintained under conventional conditions for 20 weeks,\nbut not in those maintained under SPF conditions (Fig.\n1B).  The rats could develop dermatitis only under con-\nventional conditions.  To evaluate the involvement of\nbacteria in the formation of the lesions, bacterial cul-\ntures were obtained from the facial skin surface of each\nHt-rat  kept  under  conventional  or  SPF  conditions.\nSamples  were  inoculated  onto  a  salt,  egg  yolk,  agar\nplate  (Nissui,  Tokyo,  Japan),  and  incubated  at  37°C.\nTen colonies per rat were randomly picked up, and the\nsub- species of Staphylococci were identified with an\nAN-ID Test-SP18 kit (Nissui, Tokyo, Japan).\nThe animals were humanely euthanized in accordance\nwith  the  guidelines  for  animal  experimentation  at\nShionogi  Research  Laboratories.    Skin  samples  were\nfixed in formalin or frozen in a freezer.  Four-microme-\nter thick paraffin sections of skin lesions of the Ht-rat\nwith or without dermatitis were stained with hematoxy-\nlin-eosin (HE) for histopathological observation, acidic\ntoluidine  blue  to  detect  mast  cells,  and  by  the  Luna\nmethod for eosinophils.  Frozen sections of the Ht-rat\nskin  with  or  without  dermatitis  were  immunostained\nusing  mouse  anti-rat  CD4  (OX-35,  PharMingen,  San\nDiego,  CA,  USA)  and  mouse  anti-rat  CD8a  (OX-8,\nPharMingen) antibodies.  Positive reactions on the sec-\ntions were visualized with peroxidase-diaminobenzidine,\nand the number of positive cells was counted.\nStaphylococcus  aureus  was  not  isolated  from  25-\nweek-old Ht-rats maintained under the SPF condition\n(Table 1), and approximately 70% of the isolated bac-\nteria  from  the  animals  housed  conventionally  were\ncommon  Staphylococcus  species  (S.  cohniiB  and  S.\nxylosus); the rest could not be identified.  These bacte-\nrial  strains  were  completely  replaced  by  S.  aureus  in\nconventionally  housed  Ht-rats  with  dermatitis  (Table\n1).  In view of these results, S. aureus appears to play\nan important role in the development of dermatitis in\nHt-rats.  Significant histopathological changes observed\nin the skin lesion of all Ht-rats included hyperkeratosis,\nacanthosis, slight intracellular edema of the epidermis,\nslight swelling of the epidermal cells, and the presence\nof inflammatory cells (Fig. 2A).  Higher levels of eosi-\nnophils  and  whole  and  degranulated  mast  cells  were\nobserved in the skin lesions of Ht-rats with dermatitis\nas compared with skin samples from the rats without\ndermatitis (Table 1, Figs. 2Ac and 2Ad).  In rats main-\ntained under the conventional condition, the number of\nCD4-bearing  T  cells  increased,  while  CD8-bearing  T\ncells  appeared  to  show  a  slight  increase  in  number.\nThe  ratio  of  CD4  to  CD8  cells  was  constantly  high.\nBoth CD4- and CD8-bearing T cells were rarely found\nin Ht-rats maintained under the SPF condition (Table\n1, Figs. 2Ae and 2Af).\nWe measured the serum IgE and IL-4 levels, since\nIgE and IL-4 play important roles in the pathogenesis\nof human AD [2, 12].  Serum samples from rats main-\ntained under the SPF or conventional conditions were\ncollected.  Total IgE and IL-4 levels in the serum were\nmeasured using sandwich ELISA kits (Bethyl, Mont-\ngomery,  TX,  USA;  and  Biosource,  Camarillo,  CA,\nUSA).  The level of serum IgE increased only in the\nHt-rats  that  were  maintained  under  the  conventional\ncondition, but not in those maintained under the SPF\nconditions.  Serum IL-4 was detected only in Ht-rats\nwith dermatitis (Fig. 2B).  Serum IFN-γ was not detect-\nable in any of the rats (data not shown).  Considering\nthe results, we propose that the Ht-rat can serve as a\nnew animal model for human AD.\nThe present rat model was compared with the DS-Nh\nmouse, which was recently introduced by Hikita et al.\n[4] and Yoshioka et al. [19].  The rat and mouse mod-\nels  may  be  similar  to  human  AD  with  respect  to  the\nfollowing features [4, 8, 11, 14, 15]: (1) S. aureus can\nbe isolated from skin lesions, (2) serum levels of IgE\nand IL-4 increase significantly, and (3) the number of\nwhole mast cells and CD4-bearing T cells significantly\nincrease.  Although both rodent models have much in\ncommon, there are some differences.  We found that all\nthe  S.  aureus  from  the  DS-Nh  mice  with  dermatitis\nsecrete  staphylococcal  enterotoxin  (SE)  C,  while  S.\naureus from the Ht-rats did not secrete superantigens\n(SEA, SEB, SEC, SED, SEE, and toxic shock syndrome\ntoxin)  (data  not  shown).    In  addition,  the  number  of\neosinophils significantly increased only in rats that were\nmaintained  under  the  conventional  condition,  but  no\nincrease was observed in the mice.\nAs described above, there are some genetic similari-\nties  between  Ht  and  Nh  mutations.    The  inheritance\nmode of both mutations is autosomal dominant [1, 3].\nThe Nh locus is mapped on mouse chromosome 11 and\nthe Ht locus on rat chromosome 10 [1, 3], and both the\ngenes  are  located  in  contiguity  with  the  linkage  site\n(unpublished data).  Furthermore, there are some phe-\nnotypical  similarities  between  both  mutated  rodents.\nHigher levels of mast cells were observed in the skin\nlesions  of  Ht-  or  Nh-mutated  rodents  compared  with\nskin samples of age-matched rodents without these mu-\ntations (data not shown).  In Ht-rats and Nh-mice, we\nconsidered  that  the  increasing  number  of  mast  cells\ncaused by these mutations and S. aureus colonization\nmay play an important role in developing AD-like der-\nmatitis.    We  are  currently  confirming  whether  these\ngenes are the same by using the positional cloning tech-\nnique.\nIn conclusion, Ht-rats develop dermatitis similar to\nthat in DS-Nh mice and should be useful as an addi-\ntional model for human atopic dermatitis.",
         "M. Asakawa,T. Yoshioka,I. Hikita,T. Matsutani,T. Hirasawa,A. Arimura,T. Sakata,T. Horikawa",
         null
        ],
        [
         "44",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "Vertebrate members of the nuclear receptor NR5A sub-\nfamily, which includes steroidogenic factor 1 (SF-1) and\nliver receptor homolog 1 (LRH-1), regulate crucial as-\npects of development, endocrine homeostasis, and\nmetabolism. Mouse LRH-1 is believed to be a ligand-\nindependent transcription factor with a large and\nempty hydrophobic pocket. Here we present struc-\ntural and biochemical data for three other NR5A mem-\nbers—mouse and human SF-1 and human LRH-1—\nwhich reveal that these receptors bind phosphatidyl\ninositol second messengers and that ligand binding\nis required for maximal activity. Evolutionary analysis\nof structure-function relationships across the SF-1/\nLRH-1 subfamily indicates that ligand binding is the\nancestral state of NR5A receptors and was uniquely\ndiminished or altered in the rodent LRH-1 lineage. We\npropose that phospholipids regulate gene expression\nby directly binding to NR5A nuclear receptors.",
         "Introduction\nUnlike most nuclear receptors, which are regulated by\nsteroid, retinoid, and other nonpolar ligands, NR sub-\nfamily 5A are constitutively active in many cell types,\nand obvious ligand candidates have not been forth-\ncoming. SF-1 (NR5A1) orchestrates endocrine tissue\ndevelopment and male sexual differentiation and con-\ntrols steroid biosynthesis (Ingraham et al., 1994; Luo et\nal., 1994). LRH-1 (NR5A2) is essential in early develop-\nment and regulates bile acid synthesis, cholesterol\ntransport, and ovarian function (Fayard et al., 2004; Lu\net al., 2000). This subfamily is distinct in that members\nare able to bind as monomers to DNA, and they pos-\nsess an unusually large hinge domain that is subject to\nposttranslational modifications, including phosphoryla-\ntion and sumoylation (Desclozeaux et al., 2002; Lee et\nal., 2005). Although physiological roles for these recep-\ntors have emerged, ligand candidates have remained\ncontroversial. The proposed SF-1 ligand 25-hydro-\nxycholesterol, while appealing because of the recep-\ntor's regulation of steroid synthesis, appears to be in-\ncorrect because oxysterols fail to activate SF-1 in most\ncell lines (Christenson et al., 1998; Lala et al., 1997; Mel-\nlon and Bair, 1998). Further, the notion that ligands\nmight be dispensable for this subfamily of receptors\nwas bolstered by the crystal structure of mouse LRH-1\nligand binding domain (LBD), showing an active confor-\nmation in the absence of either ligand or coregulator\npeptide. Unlike other orphan nuclear receptors, which\neither lack a ligand binding pocket (e.g., Nurr1, DHR38,\nor ERRα) or have a constitutively bound lipid (e.g.,\nHNF4α, HNF4γ, or USP), mLRH-1 was found to possess\na large unoccupied hydrophobic pocket of w830 Å (re-\nviewed in Li et al. [2003]). Remarkably, mutations that\nfilled the mLRH-1 pocket failed to reduce transcrip-\ntional activity (Sablin et al., 2003).\nWhereas an elegant structural paradigm has devel-\noped to explain the activation of ligand-regulated re-\nceptors, no uniform mechanism has been proposed to\naccount for the modulation of orphan nuclear receptor\nactivity (Steinmetz et al., 2001; Weatherman et al.,\n1999). The presence of fortuitous lipid molecules re-\nvealed in crystallization experiments can be associated\nwith an inactive LBD conformation as noted for the\ndipterian USP LBD (Billas et al., 2001). On the other\nhand, lipid components have been proposed to stabi-\nlize an active LBD conformation in constitutively active\nnuclear receptors, with HNF4 binding fatty acids (Dhe-\nPaganon et al., 2002; Wisely et al., 2002) and RORα\nbinding cholesterol (Kallen et al., 2002). In this context,\nthe structure of the murine LRH-1 with its empty ligand\nbinding pocket remains a puzzle. To further explore\nhow NR5A orphan receptors are regulated, we obtained\nthe LBD crystal structures of other members of this\nsubfamily, including mSF-1, hSF-1, and hLRH-1. Here\nwe provide structural and biochemical data supporting\nligand regulation for NR5A orphan nuclear receptors.\nResults and Discussion\nCell\nhLRH-1/TIF-2\na (Å)\nb (Å)\nc (Å)\nSpace group\nMolecules per asymmetric unit\nResolution (Å)\nNumber of unique reflections\nData redundancy\nCompleteness (%)\nRsymm (%)\n< I/s(I) >\nRefinement\nR\nRfree\nRms deviation from ideality\nBond length (Å)\nBond angle (0)\nAverage B factor (A2)\nAll atoms\nProtein atoms\nWater molecules\n73.9\n73.9\n117.0\nP41212\n\n25–1.2\n100,431\n13.1\n99.9\n6.1\n39.2\n128.4\n66.0\n141.0\nP21\n\n20–2.5\n71,698\n\n97.9\n4.5\n28.0\n73.1\n73.1\n139.4\nP41212\n\n20–2.9\n15,614\n\n99.8\n5.8\n30.0\n59.9\n67.2\n79.6\nP212121\n\n40–2.5\n10,787\n\n91.9\n3.7\n44.0",
         "were obtained, with and without TIF-2 peptide, and\nwere determined by MAD and SAD methods using sele-\nnomethionine substituted proteins, respectively (Mad-\nauss et al., 2004). The structure of hLRH-1 was det-\nermined by molecular replacement using the atomic\ncoordinates for hSF-1. The current best structures are\nrefined to 1.2 Å, 2.5 Å, and 2.5 Å for mSF-1, hLRH-1, and\nhSF-1, respectively. Relevant features of data collection\nand refinement statistics for each protein crystal are\nprovided in Table 1. All three LBDs exhibit a character-\nistic protein fold noted for other LBDs, with 12 α heli-\nces, two β strands, and helix H12 positioned in an\nactive conformation (Figure 1). A fourth layer is pro-\nvided by a structured helix H2, as previously shown for\nmLRH-1, although the position and length of this helix\nare the most varied features as judged by comparison\nof mSF-1 with hSF-1 LBD (Figure 1C) and mLRH-1 with\nhLRH-1 (Figure 1D). Upon refinement of these three\nLBD structures, additional electron density was present\nin the ligand binding pockets consistent with the pres-\nence of a phospholipid; the presence of ligand con-\ntrasts the reported structure of mLRH-1 showing no\ndensity inside or at the opening of the pocket.\nPhospholipids as NR5A Receptor Ligands\n(A) Nondenaturing mass spectrometry was carried out on all four LBD proteins used for crystallization (mSF-1, hSF-1 and mLRH-1, hLRH-1).\nSpectra are shown for mLRH-1 (left panel) and mSF-1 (right panel), with the apo [A], and the bound [B] species indicated. The molecular\nweight difference between the two peaks is also shown (arrow).\n(B) The concentrations of all bound LBD species, including the mSF-1 A270W pocket mutant, were determined as described and are listed\nfor equal amounts of LBD protein (w30 µM), with the exception of hSF-1, where more protein was used due to the poorer quality of protein\n(50 µM).\n(C) Product ions following collision-induced decomposition using denaturing electrospray tandem mass spectrometry (ESI MS/MS) on m/z\n745 (blue diamond) are shown for hSF-1 and reveal a phosphatidyl glycerol (chemical structure) ligand with two acyl chains of C16 and C18.\n(D) Electron density for mSF-1 is from a 2Fo−Fc map contoured at 1.2 sigma that shows the phospholipid ligand flanked by surrounding\nresidues (sticks). Three residues are labeled: Gly342, Tyr437, and Ala270, and the phosphate in the PG ligand is indicated (P).\n(E) Electron density found for both mSF-1 (blue) and hSF-1 ± TIF peptide (white or gray) ligands are superposed showing an identical position\nin the hydrophobic ligand binding pocket.\nPhospholipids as NR5A Receptor Ligands\nCell\nIn the upper left corner, a flow diagram is shown for ligand analyses of mSF-1 LBD protein following incubation with different liposomes, as\ndescribed in Experimental Procedures. Mass spectra profiles (ESI-LC/MS) are shown for the basal state (No Liposomes, panel A), and after\nincubations with the neutral liposomes of phosphatidyl choline (PC, panel B), with phosphatidic acid liposomes (PC:PA [9:1], panel C), and\nwith phosphatidyl inositol liposomes PC:PI(3,4)P2 [9:1] (PIP2, panel D) or PC:PI(3,4,5)P3 [9:1] (PIP3, panel E). In (B), (D), and (E), the small\npeaks at mass 28730 Da are PC liganded receptor (PC), and in (C), the peak at mass 28646 is PA liganded receptor (PA). Direct mass\nspectrometry analysis (without gel filtration) on the PC:PA-receptor sample revealed a strong peak of free phosphatidic acid (data not shown),\nshowing that ample PA was present in the PC:PA liposomes.\nThe binding of phospholipids was further explored\nby mass spectrometry analysis of purified LBD proteins\nafter incubation with liposomes containing phosphati-\ndyl choline (PC), mixed with either phosphatidic acid\n(PA), PI(3,4)P2, or PI(3,4,5)P3 in a 9:1 ratio. Remarkably,\nincubation of the mSF-1 LBD with the neutral PC lipo-\nsome showed an almost complete loss of the bacterial\nPG phospholipid with very little exchange or binding by\nPC; similar results were found with the PC:PA liposome\n(Figures 4B and 4C) with both profiles differing greatly\nfrom spectra observed with the PG bound state of the\nmSF-1 LBD (Figure 4A, No Liposomes). In contrast, effi-\ncient binding was observed for PI(3,4)P2 or PI(3,4,5)P3\nas evidenced by the increase mass of 970 and 1050 Da,\nrespectively, with only a small uptake of PC observed\n(Figures 4D and 4E). A similar experiment carried out\nwith mLRH-1 LBD revealed less selectivity compared\nto that of mSF-1, with equal exchange of PC, PI(3,4)P2,\nor PI(3,4,5)P3, as well as some liganded PA detected\n(see Supplemental Figure S2). These data taken to-\ngether with the PIP strip binding data suggest that SF-\n1s and hLRH-1 bind much more robustly and selec-\ntively to phosphorylated PIs than does mLRH-1. The\nstructural basis for this species difference in selective\nphospholipid binding is discussed below.\nure 5A). These orphan nuclear receptors contain a pu-\ntative LBD but no DBD; their capacity to repress SF-1,\nLRH-1, and steroid hormone receptors is mediated by\nregions within the very N- and C-terminal LBD (Holter\net al., 2002; Ito et al., 1997; Nachtigal et al., 1998). Un-\nlike NR4 orphan receptors (Nurr1, DHR38) that have\nlost their coactivator cleft (Baker et al., 2003; Wang et\nal., 2003), all four LBD structures of SF-1 and LRH-1\nexhibit a well-formed docking site for coregulators.\nPeptide binding assays were carried out\nfor both\nmLRH-1 and mSF-1 LBD proteins using C-terminal flu-\norescently tagged SHP and Dax-1 peptides and fluo-\nrescence anisotropy, as described (Moore et al., 2004).\nSelective high-affinity binding was found for both\nmLRH-1 and mSF-1, with SF-1 binding more tightly to\npeptides from Dax-1; only the N-terminal LXXLL motif\nin the Dax-1 LBD region is able to bind both mSF-1 and\nmLRH-1 (Figures 5A and 5B).\nPhospholipids as NR5A Receptor Ligands\n(A) Schematics of SHP and Dax-1 are shown\nwith colored boxes corresponding to LXXLL\npeptides used in the fluorescence anisot-\nropy assay, as depicted and listed in the\ntable.\n(B) Relative binding affinities for labeled pep-\ntides are indicated by color-coded affinities\n(Kd values) ranging from 0.5–10.0, 10.1–30.0,\nand >30.0 µM and were determined as de-\nscribed in Experimental Procedures. The\nmSHP-1 peptide used in crystallization of\nmSF-1 is RPTILYALLSPSPR (mSHP-1).\n(C) Ribbon depiction of the mSHP-1 peptide\nin the coactivator cleft of mSF-1 (orange rib-\nbon) or superposed with the mLRH-1 struc-\nture (beige ribbon). Residues are numbered\naccording to the database entry Q62227 for\nmSHP.\nLRH-1 regulate Dax-1 and SHP in a negative feedback\nregulatory mechanism (Lu et al., 2000).\nCell\nIn the top panel, ribbon diagrams of partial structures of mLRH-1\nand mSF-1 are shown. Helices H2 (red) and H3 (pink) of mSF-1 are\nsuperposed with the helices H2 and H3 of mLRH-1 (light yellow),\nand the phosphatidyl glycerol ligand is shown (blue stick). The H2\nof mSF-1 is approximately half the length of helix H2 in mLRH-1\nH2. The bottom panel shows reporter activity in HepG2 cells with\nincreasing concentrations of wild-type and helix H5 mutants for\nboth mLRH-1 and mSF-1. Relative luciferase activities are shown.\nSF-1. Further analysis shows residues within this clus-\nter to be crucial for phospholipid binding. Indeed, bind-\ning of this ligand is optimized by coordination of the\nphosphate head group at the base of the pocket, with\nhydrogen bonding provided by Gly342 at the beginning\nof H6, and Tyr437 and Lys441 at the end of H11, which\nare 2.8 Å, 2.6 Å, and 2.8 Å from the ligand, respectively\n(Figure 7B). These interactions effectively seal the bot-\ntom of the ligand binding pocket with the phosphate\ngroup from the ligand contributing to this closure (Fig-\nure 7C).\nA different scenario occurs in mLRH-1, where Glu440\nforms a strong ionic interaction with the opposing\nLys539 in helix H11 (analogous to Lys441 in mSF-1),\nsealing the pocket and stabilizing the mLRH-1 LBD in\nthe absence of the ligand (Figure 7C). The other unique\nresidues spanning helices H6-H7 in mLRH-1 (Thr439\nand Phe443, Figure 7A) are also predicted to minimize\nligand binding by clashing with both the phosphate\ngroup and aliphatic chains, respectively. To test the hy-\npothesis that key residues interacting with the lipid’s\nphosphate group are critical to ligand binding, mLRH-1\nwas “humanized” by an E440G mutation. Wild-type and\nmutant receptors were assayed before and after reduc-\ning cellular pools of phosphatidic acid and subsequent\nphosphatidyl\ninositols with a competitive inhibitor of\nphospholipase D (Fang et al., 2001). N-butanol treat-\nment had a marginal effect on the activity of mLRH-1\nbut substantially reduced activation by the humanized\nE440G mLRH-1, and to a lesser extent wild-type mSF-1\n(Figure 7D). Collectively, these data are consistent with\nthe lowered selectively observed for mLRH-1 phospho-\nlipid binding and are predicted by structures of the\nLRH-1 and SF-1 orthologs.\nRecent Divergence in the Rodent LRH-1\nEvolutionary analysis of structure-function relations in\nthe LRH-1/SF-1 subfamily indicates that ligand binding\nis the ancestral state for these proteins where ligand\nbinding was subsequently reduced or altered in the lin-\neage leading to the rodent LRH-1. We coded the pres-\nence or absence of ligand binding as a phylogenetic\ncharacter and reconstructed its evolution on the phy-\nlogeny of the NR5A gene family (Figure 8). As described\nin Experimental Procedures, both maximum parsimony\nand maximum likelihoods were used to evaluate two\nhypotheses concerning the function of the common an-\ncestor of the entire NR5A subfamily—that the ancestor\nwas liganded (H1) and that it was unliganded (H0). Un-\nder H1, a single loss of ligand binding is required on the\nbranch leading to the rodent LRH-1, whereas H0 re-\nquires ligand binding to have been gained indepen-\ndently in the SF-1 and human LRH-1 lineages. H1 is\ntherefore the most parsimonious hypothesis. A maxi-\nmum likelihood analysis assuming that ligand binding\ncan be gained or lost under a simple reversible Markov\nprocess also supports H1, albeit nonsignificantly, indi-\ncating a loss of ligand binding in rodent LRH-1s (likeli-\nhood ratio = 3.05).\nPhospholipids as NR5A Receptor Ligands\n(A) Sequence alignment of the region span-\nning helices H6 and H7 is shown for a limited\nnumber of species. Rodent LRH-1-specific\nchanges are highlighted in red.\n(B) Electron density showing coordination of\nthe phosphate group by three residues in\nmSF-1. These include Gly342 (H7) and Tyr437\nand Lys441 (H11) with hydrogen bonds indi-\ncated (dashed line). The phosphate is la-\nbeled (P).\n(C) Interactions between helices H7 and H11\nare depicted for mSF-1, with phospholipid li-\ngand (white stick) and for mLRH-1 without\nligand. The salt bridge between Glu440 and\nLys539 in mLRH-1 is depicted.\n(D) Reporter activity is shown after depletion\nof phosphatidic acid by butanol with wild-type\nmLRH-1, mSF-1, and “humanized” mLRH-1\nE440G mutant transfected into HepG2 cells.\nActivity is expressed as fold activation using\nthe aromatase luciferase reporter.\nare known for only four family members, sequences are\navailable for dozens. We used maximum likelihood\n(Yang et al., 1995) to determine ancestral amino acid\nstates and evolutionary changes in the H6-H7 residues\ncorresponding to Thr439, Glu440, and Phe443 in rodent\nLRH-1 (TEF, Figure 8). These sites were chosen be-\ncause, as discussed above, structural and experimen-\ntal evidence indicates that they exclude or diminish\nphospholipid binding in mLRH-1, but the homologous\nresidues, Ala, Gly, and Leu, confer ligand binding on\nhLRH-1 and the SF-1s (AGL, Figure 8). The NR5A an-\ncestral sequence has all three ligand-associated states\n(AGL), and this reconstruction has extremely high confi-\ndence (posterior probability 100% for each site). The\nderived TEF residues that close the pocket in the ab-\nsence of ligand all evolved much later on than the\nbranch leading to the rodent LRH-1 (posterior prob-\nability 100% for each site). Examination of the genetic\ncode indicates that each of these replacements can be\nconferred by a single base-pair mutation.\nacid replacements. The lost or altered ligand binding\nstate in rodent LRH-1 is the second example of a taxon-\nspecific nuclear receptor losing the capacity to bind li-\ngand from a liganded ancestor; the other is the estro-\ngen receptor ortholog of the mollusc Aplysia californica\n(Thornton et al., 2003). These findings cast doubt on\nthe hypothesis that ligand binding in the nuclear recep-\ntor superfamily was independently gained numerous\ntimes from an ancestral orphan receptor (Escriva et al.,\n2000). If the loss of ligand dependence is common and\nrequires only a few sequence changes, the opposite\nscenario—repeated loss of ligand binding from a li-\nganded ancestor—may better characterize NR evo-\nlution.\nCell\nThe maximum posterior probability phylogeny of the NR5 gene\nfamily is shown, including LRH-1s (red), SF-1s (blue), and outgroup\nsequences (black, with number of sequences analyzed shown in\nparentheses). Residues homologous to mLRH-1 positions 439, 440,\nand 443 are shown for each LRH-1/SF-1 sequence (aa). Maximum\nlikelihood reconstructions of ancestral amino acid states are shown\nfor SF-1 and LRH-1 proteins in the last common ancestor of verte-\nbrates (squares), of tetrapods (triangles), and of mammals (circles),\nas well as for the common ancestral gene preceding the duplica-\ntion that split the SF-1 and LRH-1 lineages (star). All ancestral re-\nconstructions have 100% posterior probability. Green arrowhead\nshows acquisition of amino acids that exclude ligand in the rodent\nLRH-1 lineage and loss of ligand binding on the same branch. Spe-\ncies names and accession numbers are provided in Supplemental\nTable S2.\nimplies that a distinct pool of cellular lipids signal via\nmodulation of the NR5A nuclear receptors. Indeed, the\nliterature suggests that nuclear phosphatidyl\ninositol\nlipids exist as well as enzymes affecting their metabo-\nlism, such as phosphatidyl inositol 3-kinase (PI-3K) (Ir-\nvine, 2002; Ledeen and Wu, 2004; Martelli et al., 2003).\nAdditionally, both PI-3 and PI-5 phosphatases exhibit\nprominent nuclear staining (Bacqueville et al., 2001; De-\nleris et al., 2003), thus providing a mechanism to down-\nregulate PI nuclear signaling. Although we found that\nWortmannin (PI-3 kinase inhibitor) and the PI-3 phos-\nphatase PTEN affected the activities of wild-type and\npocket mutant SF-1 receptors differently (I.N.K., unpub-\nlished data), further studies and additional pharmaco-\nlogical tools are needed to selectively disrupt nuclear\nphospholipid signaling pathways. Nonetheless, the link\nbetween PI-3 kinase signaling and NR5A receptor ac-\ntivity is intriguing given the roles that both SF-1 and\nLRH-1 play in cell proliferation. Indeed, haploinsuffi-\nciency of both sf-1 and lrh-1 leads to decreased cell\nproliferation (Bland et al., 2000; Botrugno et al., 2004),\nwhile increased gene dosage of SF-1 is reported to be\nassociated with human adrenal cortical tumors (de Fi-\ngueiredo et al., 2004). Because PI-3 kinase pathways\nare known to regulate cell cycle progression and cell\ngrowth, it is tempting to speculate that extracellular\nevents that trigger this pathway might also elevate\nnuclear PIs, and thus modulate SF-1/LRH-1 activities.\nFinally, it is worth considering how phospholipids are\ndelivered to nuclear receptors. Cytoplasmic shuttling of\nPIs to the plasma membrane is carried out by phospha-\ntidyl inositol transport proteins (PITP) in an ATP-inde-\npendent manner; these proteins are also present in the\nnucleus. Crystal structures of phospholipid binding\nproteins show a large hydrophobic pocket or tunnel\nable to bind lipids (Yoder et al., 2001; Hsuan and Cock-\ncroft, 2001; Soccio and Breslow, 2003). Whether PIs are\ndirectly taken up by SF-1/LRH-1 from membranes (due\nto transient receptor shuttling between the cytoplasm\nand nucleus) or are delivered by nuclear phospholipid\nbinding proteins remains unclear. While many ques-\ntions are still to be answered, our current study leads\nus to hypothesize phosphatidyl inositols as bona fide\nligands for NR5A receptors and, as such, provide a di-\nrect link between transcription and phospholipid sig-\nnaling.\nExperimental Procedures\nPhospholipids as NR5A Receptor Ligands\ndiffracted to 1.1–1.2 Å. Data were integrated using DENZO and\nscaled with SCALEPACK. For hLRH-1, the protein and TIF-2 pep-\ntide were mixed in equal volumes with 1.2 M sodium potassium\nphosphate (pH 4.9), 4% (v/v) glycerol, 4% (v/v) ethylene glycol.\nCrystals were grown at 22°C and appeared in one week and\nreached 100–200 m in each dimension over a period of 3 weeks.\nPrior to data collection, hLRH-1 crystals were dipped in cryopro-\ntectant (reservoir solution containing 11.5% glycerol and 11.5%\nethylene glycol) and flash-frozen in liquid nitrogen. The TIF-2 pep-\ntide sequence is KENALLRYLLDKDD and was used for both hLRH-1\nand hSF-1 crystallization. X-ray data were collected at IMCA 17-ID\nline (Argonne, Illinois), and data were processed and scaled with\nHKL2000. The hLRH-1 model was built by molecular replacement\nwith Amore program using hSF-1 as a search model and refined\nwith CNS. In this model, a second TIF-2 peptide is found in the\nasymmetric unit and forms loose contacts with the tail of the first\nTIF-2 peptide positioned in the coactivator cleft formed by H3, H4,\nand H12 of the hLRH-1 receptor. Rebuilding between rounds of\nrefinement was performed with the graphics program QUANTA.\nCell Culture and Transfections\nHuman HepG2 liver carcinoma cells were plated the day of trans-\nfection in 24 well plates and transfected with FUGENE 6 with\namounts of DNA indicated in figures and legends. For all experi-\nments, 200 ng/well of luciferase reporter were used and pCI-Neo-\nβ-gal (50 ng/well) were transfected along with wild-type or mutant\nreceptors. Luciferase activity was measured after 48 hr, as pre-\nviously described (Sablin et al., 2003). Expression of all mutants\nwas tested by Western blot analyses using anti-HA antibody (Co-\nvance) directed against an N-terminal tag. In all cases, mutant re-\nceptors were expressed at equivalent or higher levels than wild-\ntype protein. For N-butanol experiments, 0.1% 1-butanol was\nadded for 2 hr. All transfection experiments were done in triplicates\nand repeated at least three times; representative experiments are\nshown with standard errors indicated by bars.\nLigand Analyses and Phospholipid Binding Assays\nProtein samples were desalted and injected onto a column packed\nwith P6 gel (BioRad) to exchange nonvolatile buffers with 20 mM\nammonium acetate (pH 6.8). Buffer-exchanged protein was col-\nlected and back flushed at a flow rate of 0.75–1 µl/min into a na-\nnoflow electrospray ionization source fitted on a Waters/Micromass\nQTOF-1 (Beverly, Massachusetts) mass spectrometer. Data were col-\nlected in positive ion mode over a mass range of 1000–4000 m/z. For\nstructural determination of ligands, a similar in-gel filtration appara-\ntus was connected to an orthogonal acceleration time-of-flight\nmass spectrometer (LCMSD-TOF, Agilent) and a quadrupole ion\ntrap mass spectrometer (LCMSD Trap SL, Agilent). Samples were\nexamined utilizing negative ion electrospray using conditions that\ndenatured the protein-ligand complex.\nthese conditions, PI and PA lipids should mix completely into sta-\nble bilayers and consist primarily of PC. LBD proteins (2 mg/ml)\nwere exchanged into liposome buffer and combined with the lipo-\nsomes to produce a 25:1 molar ratio of PC to protein and a 2.5:1\nmolar ratio of PA, PI(3,4)P2, or PI(3,4,5)P3 to protein. Mixtures were\nincubated for 20 hr at room temperature, and protein was sepa-\nrated from liposomes by gel-filtration on Superdex 75 (Pharmacia),\nconcentrated, subjected to another round of gel filtration, and ana-\nlyzed by mass spectrometry as described above. The amount of\nSF-1 LBD protein associated with free liposomes (void volume) fol-\nliposome\nlowing gel filtration was determined to be <1% for all\nspecies.\nPeptide Binding Assays\nLBD proteins were serially diluted from 150 µM to 0.002 µM in bind-\ning buffer (50 mM NaPO4, 150 mM NaCl, pH 7.2, 1 mM DTT, 1 mM\nEDTA, 0.01% NP40, 10% glycerol). Diluted protein was added to\nfluorescent coregulator peptide (20 nM) in 384 well plates yielding\nfinal protein concentrations of 75–0.001 µM and 10 nM of fluores-\ncent peptide. Samples were analyzed as previously described\n(Moore et al., 2004). Construction of 20 aa coregulator peptides\nwith the general motif of CXXXXXXXLXXL/AL/AXXXXXXX were as\nfollows: peptides were synthesized in parallel using standard fluor-\nenylmethoxycarbonyl (Fmoc) chemistry in 48 well synthesis blocks\n(FlexChem System, Robbins) (Wellings and Atherton, 1997). Cou-\npling efficiency was monitored and the completed peptides were\ncleaved from the resin, followed by reversed-phase chromatogra-\nphy and mass spectrometry (MALDI-TOF/ESI) to purify the pep-\ntides, and are described in Moore et al. (2004).\nEvolutionary Analysis\nProtein sequences of 48 SF-1/LRH-1/FtzF1 family members with\nfull-length ligand binding domains were obtained from GenBank by\nBLAST search using Homo sapiens SF-1 as a query. Twenty addi-\ntional sequences from the closely related ERR, GCNF/GRF, and\nRXR groups (Thornton and DeSalle, 2000) were also obtained to\nserve as outgroup sequences. (See Supplemental Table S2 for ac-\ncession numbers and species names). These sequences were\naligned using ClustalX 1.83 software (Thompson et al., 1994) as-\nsuming a gap:change cost ratio of 10. The phylogeny of the gene\nfamily was then inferred from the aligned sequences using the\nBayesian Markov Chain Monte Carlo method (Ronquist and\nHuelsenbeck, 2003) implemented in MrBayes v. 3.0 software. Tree\nand parameter space were explored with four simultaneous chains\n(three heated) for 300,000 generations each, using the Jones model\nof protein evolution with a four-category discrete γ model of\namong-site rate variation. Priors were equal for tree topologies,\nuniform (0,10) for branch lengths, and uniform (0.1, 5) for the α\nshape parameter of the γ distribution. The initial 100,000 trees, a\npoint well past stationarity, were discarded as burn-in.\nPhylogenetic reconstruction of the ancestral states at internal\nnodes for ligand binding or ligand independence, coded as a binary\ncharacter, were conducted by parsimony and maximum likelihood.\nFor the former, we used Fitch’s parsimony algorithm (Fitch, 1971)\non a simple tree consisting of the four taxa for which ligand binding\nstates are known (Sablin et al., 2003; this paper). For maximum\nlikelihood reconstruction, we calculated the likelihood of each an-\ncestral state 0 (no ligand binding) and 1 (ligand binding) as the\nprobability of evolving the terminal states in mLRH-1, hLRH-1,\nmSF-1, and hSF-1, given the phylogeny that relates these genes,\nand a simple reversible Markov process for transitions between\nstates 0 and 1 (Pagel, 1999). The probability Pij of observing a state\nchange on a tree branch of length k is Pij =\ne−k.\nBranch lengths were assumed equal a priori and scaled to their\nmaximum likelihood values by hill climbing; unknown states at in-\nternal nodes other than the NR5A ancestor were treated as nui-\nsance parameters and optimized by maximum likelihood.\nassuming a Jones model of amino acid evolution and a four-cate-\ngory γ model of rate variation among sites (α optimized by ML). The\ntree for ancestral state reconstruction was the maximum posterior\nprobability tree from the Bayesian analysis, adjusted at three nodes\nthat had posterior probability <0.95 to make them consistent with\nknown taxonomic relationships among species (see Supplemental\nFigure S3). Branch lengths for the ancestral reconstruction were\noptimized by maximum likelihood using PAML software.\nSupplemental Data\nSupplemental Data include three figures and two tables and can\nbe found with this article online at http://www.cell.com/cgi/\ncontent/full/120/3/343/DC1/.\nAcknowledgments\nWe would like to thank Drs. D. Julius, K. Shokat, D. Stokoe, S.\nWadekar, N. Jouravel, D. Moore, and F. Szoka for critical com-\nments, experimental suggestions, and reagents. The IMCA-CAT\nbeamline 17-ID (or 17-BM) at the Advanced Photon Source was\nsupported by the companies of the Industrial Macromolecular\nCrystallography Association through a contract with Illinois Insti-\ntute of Technology. We would like to thank Drs. Cheng Yang and\nJoseph Ferrara from Rigaku/MSC Inc. for an invitation to this facil-\nity. This work was funded by NIH 1R21-GM70792-1 and the Oregon\nMedical Research Foundation to J.W.T., a HHMI predoctoral fellow-\nship to J.A.M., and a NIDDK-PO1 support to R.J.F., R.K.G., and\nH.A.I.\nReceived: November 3, 2004\nRevised: January 13, 2005\nAccepted: January 27, 2005\nPublished: February 10, 2005\nReferences\nBacqueville, D., Deleris, P., Mendre, C., Pieraggi, M.T., Chap, H.,\nGuillon, G., Perret, B., and Breton-Douillon, M. (2001). Characteriza-\ntion of a G protein-activated phosphoinositide 3-kinase in vascular\nsmooth muscle cell nuclei. J. Biol. Chem. 276, 22170–22176.\nBaker, K.D., Shewchuk, L.M., Kozlova, T., Makishima, M., Hassell,\nA., Wisely, B., Caravella, J.A., Lambert, M.H., Reinking, J.L., Krause,\nH., et al. (2003). The Drosophila orphan nuclear receptor DHR38\nmediates an atypical ecdysteroid signaling pathway. Cell 113,\n731–742.\nBillas, I.M., Moulinier, L., Rochel, N., and Moras, D. (2001). Crystal\nstructure of the ligand-binding domain of the ultraspiracle protein\nUSP, the ortholog of retinoid X receptors in insects. J. Biol. Chem.\n276, 7465–7474.\nBland, M.L., Jamieson, C.A., Akana, S.F., Bornstein, S.R., Eisen-\nhofer, G., Dallman, M.F., and Ingraham, H.A. (2000). Haploinsuffi-\nciency of steroidogenic factor-1 in mice disrupts adrenal develop-\nment leading to an impaired stress response. Proc. Natl. Acad. Sci.\nUSA 97, 14488–14493.\nBotrugno, O.A., Fayard, E., Annicotte, J.S., Haby, C., Brennan, T.,\nWendling, O., Tanaka, T., Kodama, T., Thomas, W., Auwerx, J., and\nSchoonjans, K. (2004). Synergy between LRH-1 and beta-catenin\ninduces G1 cyclin-mediated cell proliferation. Mol. Cell 15, 499–\n509.\nChristenson, L.K., McAllister, J.M., Martin, K.O., Javitt, N.B., Os-\nborne, T.F., and Strauss, J.F., 3rd. (1998). Oxysterol regulation of\nsteroidogenic acute regulatory protein gene expression. Structural\nspecificity and transcriptional and posttranscriptional actions. J.\nBiol. Chem. 273, 30729–30735.\nDarimont, B.D., Wagner, R.L., Apriletti, J.W., Stallcup, M.R., Kush-\nner, P.J., Baxter, J.D., Fletterick, R.J., and Yamamoto, K.R. (1998).\nStructure and specificity of nuclear receptor-coactivator interac-\ntions. Genes Dev. 273, 30729–30735.\nR., Ribeiro, R.C., Zambetti, G., Delacerda, L., Rodrigues, G.A., and\nHaddad, B.R. (2004). Amplification of the steroidogenic factor 1\n(SF-1) gene in childhood adrenocortical tumors. J. Clin. Endocrinol.\nMetab. Published online November 2004. 10.1210/jc.2004-0942\nDeleris, P., Bacqueville, D., Gayral, S., Carrez, L., Salles, J.P., Perret,\nB., and Breton-Douillon, M. (2003). SHIP-2 and PTEN are expressed\nand active in vascular smooth muscle cell nuclei, but only SHIP-2 is\nassociated with nuclear speckles. J. Biol. Chem. 278, 38884–38891.\nDelerive, P., Galardi, C.M., Bisi, J.E., Nicodeme, E., and Goodwin,\nB. (2004). Identification of liver receptor homolog-1 as a novel regu-\nlator of apolipoprotein AI gene transcription. Mol. Endocrinol. 18,\n2378–2387.\nDesclozeaux, M., Krylova, I.N., Horn, F., Fletterick, R.J., and Ingra-\nham, H.A. (2002). Phosphorylation and intramolecular stabilization\nof the ligand binding domain in the nuclear receptor steroidogenic\nfactor 1. Mol. Cell. Biol. 22, 7193–7203.\nDhe-Paganon, S., Duda, K., Iwamoto, M., Chi, Y.I., and Shoelson,\nS.E. (2002). Crystal structure of the HNF4 alpha ligand binding do-\nmain in complex with endogenous fatty acid ligand. J. Biol. Chem.\n277, 37973–37976.\nDietschy, J.M., and Turley, S.D. (2002). Control of cholesterol turn-\nover in the mouse. J. Biol. Chem. 277, 3801–3804.\nEscriva, H., Delaunay, F., and Laudet, V. (2000). Ligand binding and\nnuclear receptor evolution. Bioessays 22, 717–727.\nFang, Y., Vilella-Bach, M., Bachmann, R., Flanigan, A., and Chen, J.\n(2001). Phosphatidic acid-mediated mitogenic activation of mTOR\nsignaling. Science 294, 1942–1945.\nFayard, E., Auwerx, J., and Schoonjans, K. (2004). LRH-1: an or-\nphan nuclear receptor involved in development, metabolism and\nsteroidogenesis. Trends Cell Biol. 14, 250–260.\nFitch, W.M. (1971). Toward defining the course of evolution: minimal\nchange for a specific tree topology. Syst. Zool. 20, 406–416.\nHolter, E., Kotaja, N., Makela, S., Strauss, L., Kietz, S., Janne, O.A.,\nGustafsson, J.A., Palvimo, J.J., and Treuter, E. (2002). Inhibition of\nandrogen receptor\nfunction by the reproductive orphan\nnuclear receptor DAX-1. Mol. Endocrinol. 16, 515–528.\n(AR)\nHsuan, J., and Cockcroft, S. (2001). The PITP family of phosphati-\ndylinositol transfer proteins. Genome Biol. 2, REVIEWS3011.\nIngraham, H.A., Lala, D.S., Ikeda, Y., Luo, X., Shen, W.H., Nachtigal,\nM.W., Abbud, R., Nilson, J.H., and Parker, K.L. (1994). The nuclear\nreceptor steroidogenic factor 1 acts at multiple levels of the repro-\nductive axis. Genes Dev. 8, 2302–2312.\nIrvine, R.F. (2002). Nuclear lipid signaling. Sci. STKE, http://stke.\nsciencemag.org/cgi/content/full/OC_sigtrans;2002/150/re13.\nIto, M., Yu, R., and Jameson, J.L. (1997). DAX-1 inhibits SF-1-medi-\nated transactivation via a carboxy-terminal domain that is deleted\nin adrenal hypoplasia congenita. Mol. Cell. Biol. 17, 1476–1483.\nKallen, J.A., Schlaeppi, J.M., Bitsch, F., Geisse, S., Geiser, M., Del-\nhon, I., and Fournier, B. (2002). X-ray structure of the hRORalpha\nLBD at 1.63 A. Structural and functional data that cholesterol or a\ncholesterol derivative is the natural ligand of RORalpha. Structure\n(Camb.) 10, 1697–1707.\nLala, D.S., Syka, P.M., Lazarchik, S.B., Mangelsdorf, D.J., Parker,\nK.L., and Heyman, R.A. (1997). Activation of the orphan nuclear\nreceptor steroidogenic factor 1 by oxysterols. Proc. Natl. Acad. Sci.\nUSA 94, 4895–4900.\nLedeen, R.W., and Wu, G. (2004). Nuclear lipids: key signaling ef-\nfectors in the nervous system and other tissues. J. Lipid Res. 45, –.\nLee, M.K., Lebedeva, L., Suzawa, M., Wadekar, S., Desclozeaux,\nM., and Ingraham, H.A. (2005). Repression of orphan nuclear recep-\ntor activity via SUMO-modification and the Dead-Box Protein\nDP103 (Ddx20, Gemin-3). Mol. Biol. Cell 25, 1879–1890.\nLi, Y., Lambert, M.H., and Xu, H.E. (2003). Activation of nuclear re-\nceptors: a perspective from structural genomics. Structure (Camb.)\n11, 741–746.\nL.R., and Helmkamp, G.M., Jr. (2001). Structure of a multifunctional\nprotein. Mammalian phosphatidylinositol\ntransfer protein com-\nplexed with phosphatidylcholine. J. Biol. Chem. 276, 9246–9252.\nAccession Numbers\nCoordinates have been deposited in the Protein Data Bank as\nfollows: mSF-1 PDB ID 1YMT, RCSB code RCSB031696, hLRH-1\nPDB ID 1YOK and RCSB031753, and hSF-1 PDB ID 1YOW and\nRCSB031764.\nPhospholipids as NR5A Receptor Ligands\nLu, T.T., Makishima, M., Repa, J.J., Schoonjans, K., Kerr, T.A., Au-\nwerx, J., and Mangelsdorf, D.J. (2000). Molecular basis for feed-\nback regulation of bile acid synthesis by nuclear receptors. Mol.\nCell 6, 507–515.\nLudbrook, L.M., and Harley, V.R. (2004). Sex determination: a ‘win-\ndow’ of DAX1 activity. Trends Endocrinol. Metab. 15, 116–121.\nLuo, X., Ikeda, Y., and Parker, K.L. (1994). A cell-specific nuclear\nreceptor is essential for adrenal and gonadal development and sex-\nual differentiation. Cell 77, 481–490.\nMadauss, K., Juzumiene, D., Waitt, G., Williams, J.P., and Williams,\nS.P. (2004). Generation and characterization of human steroido-\ngenic factor 1 LBD crystals with and without bound co-factor pep-\ntide. Endocr. Res. 30, 775–785.\nMartelli, A.M., Tabellini, G., Borgatti, P., Bortul, R., Capitani, S., and\nNeri, L.M. (2003). Nuclear lipids: new functions for old molecules?\nJ. Cell. Biochem. 88, 455–461.\nMellon, S.H., and Bair, S.R. (1998). 25-Hydroxycholesterol is not a\nligand for the orphan nuclear receptor steroidogenic factor-1 (SF-\n1). Endocrinology 139, 3026–3029.\nMoore, J.M., Galicia, S.J., McReynolds, A.C., Nguyen, N.H., Scan-\nlan, T.S., and Guy, R.K. (2004). Quantitative proteomics of the thy-\nroid hormone receptor-coregulator interactions. J. Biol. Chem. 279,\n27584–27590.\nNachtigal, M.W., Hirokawa, Y., Enyeart-VanHouten, D.L., Flanagan,\nJ.N., Hammer, G.D., and Ingraham, H.A. (1998). Wilms’ tumor 1 and\nDax-1 modulate the orphan nuclear receptor SF-1 in sex-specific\ngene expression. Cell 93, 445–454.\nPagel, M. (1999). Inferring the historical patterns of biological evo-\nlution. Nature 401, 877–884.\nRonquist, F., and Huelsenbeck, J.P. (2003). MrBayes 3: Bayesian\nphylogenetic inference under mixed models. Bioinformatics 19,\n1572–1574.\nSablin, E.P., Krylova,\nI.N., Fletterick, R.J., and Ingraham, H.A.\n(2003). Structural basis for ligand-independent activation of the or-\nphan nuclear receptor LRH-1. Mol. Cell 11, 1575–1585.\nSoccio, R.E., and Breslow, J.L. (2003). StAR-related lipid transfer\n(START) proteins: mediators of intracellular lipid metabolism. J.\nBiol. Chem. 278, 22183–22186.\nSteinmetz, A.C., Renaud, J.P., and Moras, D. (2001). Binding of li-\ngands and activation of transcription by nuclear receptors. Annu.\nRev. Biophys. Biomol. Struct. 30, 329–359.\nThompson, J.D., Higgins, D.G., and Gibson, T.J. (1994). CLUSTALW:\nImproving the sensitivity of progressive multiple sequence align-\nment through sequence weighting, position-specific gap penalties\nand weight matrix choice. Nucleic Acids Res. 22, 4673–4680.\nThornton, J.W., and DeSalle, R. (2000). A new method to localize\nand test the significance of incongruence: detecting domain shuf-\nfling in the nuclear receptor superfamily. Syst. Biol. 49, 183–201.\nThornton, J.W., Need, E., and Crews, D. (2003). Resurrecting the\nancestral steroid receptor: ancient origin of estrogen signaling. Sci-\nence 301, 1714–1717.\nWang, Z., Benoit, G., Liu, J., Prasad, S., Aarnisalo, P., Liu, X., Xu,\nH., Walker, N.P., and Perlmann, T. (2003). Structure and function of\nNurr1 identifies a class of ligand-independent nuclear receptors.\nNature 423, 555–560.\nWeatherman, R.V., Fletterick, R.J., and Scanlan, T.S.\n(1999).\nNuclear-receptor ligands and ligand-binding domains. Annu. Rev.\nBiochem. 68, 559–581.\nWellings, D.A., and Atherton, E. (1997). Standard Fmoc protocols.\nMethods Enzymol. 289, 44–67.\nWisely, G.B., Miller, A.B., Davis, R.G., Thornquest, A.D., Jr., John-\nson, R., Spitzer, T., Sefler, A., Shearer, B., Moore, J.T., Willson, T.M.,\nand Williams, S.P. (2002). Hepatocyte nuclear factor 4 is a transcrip-\ntion factor that constitutively binds fatty acids. Structure (Camb.)\n10, 1225–1234.\nYang, Z., Kumar, S., and Nei, M. (1995). A new method of inference\nof ancestral nucleotide and amino acid sequences. Genetics 141,\n1641–1650.",
         "were obtained, with and without TIF-2 peptide, and\nwere determined by MAD and SAD methods using sele-\nnomethionine substituted proteins, respectively (Mad-\nauss et al., 2004). The structure of hLRH-1 was det-\nermined by molecular replacement using the atomic\ncoordinates for hSF-1. The current best structures are\nrefined to 1.2 Å, 2.5 Å, and 2.5 Å for mSF-1, hLRH-1, and\nhSF-1, respectively. Relevant features of data collection\nand refinement statistics for each protein crystal are\nprovided in Table 1. All three LBDs exhibit a character-\nistic protein fold noted for other LBDs, with 12 α heli-\nces, two β strands, and helix H12 positioned in an\nactive conformation (Figure 1). A fourth layer is pro-\nvided by a structured helix H2, as previously shown for\nmLRH-1, although the position and length of this helix\nare the most varied features as judged by comparison\nof mSF-1 with hSF-1 LBD (Figure 1C) and mLRH-1 with\nhLRH-1 (Figure 1D). Upon refinement of these three\nLBD structures, additional electron density was present\nin the ligand binding pockets consistent with the pres-\nence of a phospholipid; the presence of ligand con-\ntrasts the reported structure of mLRH-1 showing no\ndensity inside or at the opening of the pocket.\nPhospholipids as NR5A Receptor Ligands\n(A) Nondenaturing mass spectrometry was carried out on all four LBD proteins used for crystallization (mSF-1, hSF-1 and mLRH-1, hLRH-1).\nSpectra are shown for mLRH-1 (left panel) and mSF-1 (right panel), with the apo [A], and the bound [B] species indicated. The molecular\nweight difference between the two peaks is also shown (arrow).\n(B) The concentrations of all bound LBD species, including the mSF-1 A270W pocket mutant, were determined as described and are listed\nfor equal amounts of LBD protein (w30 µM), with the exception of hSF-1, where more protein was used due to the poorer quality of protein\n(50 µM).\n(C) Product ions following collision-induced decomposition using denaturing electrospray tandem mass spectrometry (ESI MS/MS) on m/z\n745 (blue diamond) are shown for hSF-1 and reveal a phosphatidyl glycerol (chemical structure) ligand with two acyl chains of C16 and C18.\n(D) Electron density for mSF-1 is from a 2Fo−Fc map contoured at 1.2 sigma that shows the phospholipid ligand flanked by surrounding\nresidues (sticks). Three residues are labeled: Gly342, Tyr437, and Ala270, and the phosphate in the PG ligand is indicated (P).\n(E) Electron density found for both mSF-1 (blue) and hSF-1 ± TIF peptide (white or gray) ligands are superposed showing an identical position\nin the hydrophobic ligand binding pocket.\nPhospholipids as NR5A Receptor Ligands\nCell\nIn the upper left corner, a flow diagram is shown for ligand analyses of mSF-1 LBD protein following incubation with different liposomes, as\ndescribed in Experimental Procedures. Mass spectra profiles (ESI-LC/MS) are shown for the basal state (No Liposomes, panel A), and after\nincubations with the neutral liposomes of phosphatidyl choline (PC, panel B), with phosphatidic acid liposomes (PC:PA [9:1], panel C), and\nwith phosphatidyl inositol liposomes PC:PI(3,4)P2 [9:1] (PIP2, panel D) or PC:PI(3,4,5)P3 [9:1] (PIP3, panel E). In (B), (D), and (E), the small\npeaks at mass 28730 Da are PC liganded receptor (PC), and in (C), the peak at mass 28646 is PA liganded receptor (PA). Direct mass\nspectrometry analysis (without gel filtration) on the PC:PA-receptor sample revealed a strong peak of free phosphatidic acid (data not shown),\nshowing that ample PA was present in the PC:PA liposomes.\nThe binding of phospholipids was further explored\nby mass spectrometry analysis of purified LBD proteins\nafter incubation with liposomes containing phosphati-\ndyl choline (PC), mixed with either phosphatidic acid\n(PA), PI(3,4)P2, or PI(3,4,5)P3 in a 9:1 ratio. Remarkably,\nincubation of the mSF-1 LBD with the neutral PC lipo-\nsome showed an almost complete loss of the bacterial\nPG phospholipid with very little exchange or binding by\nPC; similar results were found with the PC:PA liposome\n(Figures 4B and 4C) with both profiles differing greatly\nfrom spectra observed with the PG bound state of the\nmSF-1 LBD (Figure 4A, No Liposomes). In contrast, effi-\ncient binding was observed for PI(3,4)P2 or PI(3,4,5)P3\nas evidenced by the increase mass of 970 and 1050 Da,\nrespectively, with only a small uptake of PC observed\n(Figures 4D and 4E). A similar experiment carried out\nwith mLRH-1 LBD revealed less selectivity compared\nto that of mSF-1, with equal exchange of PC, PI(3,4)P2,\nor PI(3,4,5)P3, as well as some liganded PA detected\n(see Supplemental Figure S2). These data taken to-\ngether with the PIP strip binding data suggest that SF-\n1s and hLRH-1 bind much more robustly and selec-\ntively to phosphorylated PIs than does mLRH-1. The\nstructural basis for this species difference in selective\nphospholipid binding is discussed below.\nure 5A). These orphan nuclear receptors contain a pu-\ntative LBD but no DBD; their capacity to repress SF-1,\nLRH-1, and steroid hormone receptors is mediated by\nregions within the very N- and C-terminal LBD (Holter\net al., 2002; Ito et al., 1997; Nachtigal et al., 1998). Un-\nlike NR4 orphan receptors (Nurr1, DHR38) that have\nlost their coactivator cleft (Baker et al., 2003; Wang et\nal., 2003), all four LBD structures of SF-1 and LRH-1\nexhibit a well-formed docking site for coregulators.\nPeptide binding assays were carried out\nfor both\nmLRH-1 and mSF-1 LBD proteins using C-terminal flu-\norescently tagged SHP and Dax-1 peptides and fluo-\nrescence anisotropy, as described (Moore et al., 2004).\nSelective high-affinity binding was found for both\nmLRH-1 and mSF-1, with SF-1 binding more tightly to\npeptides from Dax-1; only the N-terminal LXXLL motif\nin the Dax-1 LBD region is able to bind both mSF-1 and\nmLRH-1 (Figures 5A and 5B).\nPhospholipids as NR5A Receptor Ligands\n(A) Schematics of SHP and Dax-1 are shown\nwith colored boxes corresponding to LXXLL\npeptides used in the fluorescence anisot-\nropy assay, as depicted and listed in the\ntable.\n(B) Relative binding affinities for labeled pep-\ntides are indicated by color-coded affinities\n(Kd values) ranging from 0.5–10.0, 10.1–30.0,\nand >30.0 µM and were determined as de-\nscribed in Experimental Procedures. The\nmSHP-1 peptide used in crystallization of\nmSF-1 is RPTILYALLSPSPR (mSHP-1).\n(C) Ribbon depiction of the mSHP-1 peptide\nin the coactivator cleft of mSF-1 (orange rib-\nbon) or superposed with the mLRH-1 struc-\nture (beige ribbon). Residues are numbered\naccording to the database entry Q62227 for\nmSHP.\nLRH-1 regulate Dax-1 and SHP in a negative feedback\nregulatory mechanism (Lu et al., 2000).\nCell\nIn the top panel, ribbon diagrams of partial structures of mLRH-1\nand mSF-1 are shown. Helices H2 (red) and H3 (pink) of mSF-1 are\nsuperposed with the helices H2 and H3 of mLRH-1 (light yellow),\nand the phosphatidyl glycerol ligand is shown (blue stick). The H2\nof mSF-1 is approximately half the length of helix H2 in mLRH-1\nH2. The bottom panel shows reporter activity in HepG2 cells with\nincreasing concentrations of wild-type and helix H5 mutants for\nboth mLRH-1 and mSF-1. Relative luciferase activities are shown.\nSF-1. Further analysis shows residues within this clus-\nter to be crucial for phospholipid binding. Indeed, bind-\ning of this ligand is optimized by coordination of the\nphosphate head group at the base of the pocket, with\nhydrogen bonding provided by Gly342 at the beginning\nof H6, and Tyr437 and Lys441 at the end of H11, which\nare 2.8 Å, 2.6 Å, and 2.8 Å from the ligand, respectively\n(Figure 7B). These interactions effectively seal the bot-\ntom of the ligand binding pocket with the phosphate\ngroup from the ligand contributing to this closure (Fig-\nure 7C).\nA different scenario occurs in mLRH-1, where Glu440\nforms a strong ionic interaction with the opposing\nLys539 in helix H11 (analogous to Lys441 in mSF-1),\nsealing the pocket and stabilizing the mLRH-1 LBD in\nthe absence of the ligand (Figure 7C). The other unique\nresidues spanning helices H6-H7 in mLRH-1 (Thr439\nand Phe443, Figure 7A) are also predicted to minimize\nligand binding by clashing with both the phosphate\ngroup and aliphatic chains, respectively. To test the hy-\npothesis that key residues interacting with the lipid’s\nphosphate group are critical to ligand binding, mLRH-1\nwas “humanized” by an E440G mutation. Wild-type and\nmutant receptors were assayed before and after reduc-\ning cellular pools of phosphatidic acid and subsequent\nphosphatidyl\ninositols with a competitive inhibitor of\nphospholipase D (Fang et al., 2001). N-butanol treat-\nment had a marginal effect on the activity of mLRH-1\nbut substantially reduced activation by the humanized\nE440G mLRH-1, and to a lesser extent wild-type mSF-1\n(Figure 7D). Collectively, these data are consistent with\nthe lowered selectively observed for mLRH-1 phospho-\nlipid binding and are predicted by structures of the\nLRH-1 and SF-1 orthologs.\nRecent Divergence in the Rodent LRH-1\nEvolutionary analysis of structure-function relations in\nthe LRH-1/SF-1 subfamily indicates that ligand binding\nis the ancestral state for these proteins where ligand\nbinding was subsequently reduced or altered in the lin-\neage leading to the rodent LRH-1. We coded the pres-\nence or absence of ligand binding as a phylogenetic\ncharacter and reconstructed its evolution on the phy-\nlogeny of the NR5A gene family (Figure 8). As described\nin Experimental Procedures, both maximum parsimony\nand maximum likelihoods were used to evaluate two\nhypotheses concerning the function of the common an-\ncestor of the entire NR5A subfamily—that the ancestor\nwas liganded (H1) and that it was unliganded (H0). Un-\nder H1, a single loss of ligand binding is required on the\nbranch leading to the rodent LRH-1, whereas H0 re-\nquires ligand binding to have been gained indepen-\ndently in the SF-1 and human LRH-1 lineages. H1 is\ntherefore the most parsimonious hypothesis. A maxi-\nmum likelihood analysis assuming that ligand binding\ncan be gained or lost under a simple reversible Markov\nprocess also supports H1, albeit nonsignificantly, indi-\ncating a loss of ligand binding in rodent LRH-1s (likeli-\nhood ratio = 3.05).\nPhospholipids as NR5A Receptor Ligands\n(A) Sequence alignment of the region span-\nning helices H6 and H7 is shown for a limited\nnumber of species. Rodent LRH-1-specific\nchanges are highlighted in red.\n(B) Electron density showing coordination of\nthe phosphate group by three residues in\nmSF-1. These include Gly342 (H7) and Tyr437\nand Lys441 (H11) with hydrogen bonds indi-\ncated (dashed line). The phosphate is la-\nbeled (P).\n(C) Interactions between helices H7 and H11\nare depicted for mSF-1, with phospholipid li-\ngand (white stick) and for mLRH-1 without\nligand. The salt bridge between Glu440 and\nLys539 in mLRH-1 is depicted.\n(D) Reporter activity is shown after depletion\nof phosphatidic acid by butanol with wild-type\nmLRH-1, mSF-1, and “humanized” mLRH-1\nE440G mutant transfected into HepG2 cells.\nActivity is expressed as fold activation using\nthe aromatase luciferase reporter.\nare known for only four family members, sequences are\navailable for dozens. We used maximum likelihood\n(Yang et al., 1995) to determine ancestral amino acid\nstates and evolutionary changes in the H6-H7 residues\ncorresponding to Thr439, Glu440, and Phe443 in rodent\nLRH-1 (TEF, Figure 8). These sites were chosen be-\ncause, as discussed above, structural and experimen-\ntal evidence indicates that they exclude or diminish\nphospholipid binding in mLRH-1, but the homologous\nresidues, Ala, Gly, and Leu, confer ligand binding on\nhLRH-1 and the SF-1s (AGL, Figure 8). The NR5A an-\ncestral sequence has all three ligand-associated states\n(AGL), and this reconstruction has extremely high confi-\ndence (posterior probability 100% for each site). The\nderived TEF residues that close the pocket in the ab-\nsence of ligand all evolved much later on than the\nbranch leading to the rodent LRH-1 (posterior prob-\nability 100% for each site). Examination of the genetic\ncode indicates that each of these replacements can be\nconferred by a single base-pair mutation.\nacid replacements. The lost or altered ligand binding\nstate in rodent LRH-1 is the second example of a taxon-\nspecific nuclear receptor losing the capacity to bind li-\ngand from a liganded ancestor; the other is the estro-\ngen receptor ortholog of the mollusc Aplysia californica\n(Thornton et al., 2003). These findings cast doubt on\nthe hypothesis that ligand binding in the nuclear recep-\ntor superfamily was independently gained numerous\ntimes from an ancestral orphan receptor (Escriva et al.,\n2000). If the loss of ligand dependence is common and\nrequires only a few sequence changes, the opposite\nscenario—repeated loss of ligand binding from a li-\nganded ancestor—may better characterize NR evo-\nlution.\nCell\nThe maximum posterior probability phylogeny of the NR5 gene\nfamily is shown, including LRH-1s (red), SF-1s (blue), and outgroup\nsequences (black, with number of sequences analyzed shown in\nparentheses). Residues homologous to mLRH-1 positions 439, 440,\nand 443 are shown for each LRH-1/SF-1 sequence (aa). Maximum\nlikelihood reconstructions of ancestral amino acid states are shown\nfor SF-1 and LRH-1 proteins in the last common ancestor of verte-\nbrates (squares), of tetrapods (triangles), and of mammals (circles),\nas well as for the common ancestral gene preceding the duplica-\ntion that split the SF-1 and LRH-1 lineages (star). All ancestral re-\nconstructions have 100% posterior probability. Green arrowhead\nshows acquisition of amino acids that exclude ligand in the rodent\nLRH-1 lineage and loss of ligand binding on the same branch. Spe-\ncies names and accession numbers are provided in Supplemental\nTable S2.\nimplies that a distinct pool of cellular lipids signal via\nmodulation of the NR5A nuclear receptors. Indeed, the\nliterature suggests that nuclear phosphatidyl\ninositol\nlipids exist as well as enzymes affecting their metabo-\nlism, such as phosphatidyl inositol 3-kinase (PI-3K) (Ir-\nvine, 2002; Ledeen and Wu, 2004; Martelli et al., 2003).\nAdditionally, both PI-3 and PI-5 phosphatases exhibit\nprominent nuclear staining (Bacqueville et al., 2001; De-\nleris et al., 2003), thus providing a mechanism to down-\nregulate PI nuclear signaling. Although we found that\nWortmannin (PI-3 kinase inhibitor) and the PI-3 phos-\nphatase PTEN affected the activities of wild-type and\npocket mutant SF-1 receptors differently (I.N.K., unpub-\nlished data), further studies and additional pharmaco-\nlogical tools are needed to selectively disrupt nuclear\nphospholipid signaling pathways. Nonetheless, the link\nbetween PI-3 kinase signaling and NR5A receptor ac-\ntivity is intriguing given the roles that both SF-1 and\nLRH-1 play in cell proliferation. Indeed, haploinsuffi-\nciency of both sf-1 and lrh-1 leads to decreased cell\nproliferation (Bland et al., 2000; Botrugno et al., 2004),\nwhile increased gene dosage of SF-1 is reported to be\nassociated with human adrenal cortical tumors (de Fi-\ngueiredo et al., 2004). Because PI-3 kinase pathways\nare known to regulate cell cycle progression and cell\ngrowth, it is tempting to speculate that extracellular\nevents that trigger this pathway might also elevate\nnuclear PIs, and thus modulate SF-1/LRH-1 activities.\nFinally, it is worth considering how phospholipids are\ndelivered to nuclear receptors. Cytoplasmic shuttling of\nPIs to the plasma membrane is carried out by phospha-\ntidyl inositol transport proteins (PITP) in an ATP-inde-\npendent manner; these proteins are also present in the\nnucleus. Crystal structures of phospholipid binding\nproteins show a large hydrophobic pocket or tunnel\nable to bind lipids (Yoder et al., 2001; Hsuan and Cock-\ncroft, 2001; Soccio and Breslow, 2003). Whether PIs are\ndirectly taken up by SF-1/LRH-1 from membranes (due\nto transient receptor shuttling between the cytoplasm\nand nucleus) or are delivered by nuclear phospholipid\nbinding proteins remains unclear. While many ques-\ntions are still to be answered, our current study leads\nus to hypothesize phosphatidyl inositols as bona fide\nligands for NR5A receptors and, as such, provide a di-\nrect link between transcription and phospholipid sig-\nnaling.\nExperimental Procedures\nPhospholipids as NR5A Receptor Ligands",
         "diffracted to 1.1–1.2 Å. Data were integrated using DENZO and\nscaled with SCALEPACK. For hLRH-1, the protein and TIF-2 pep-\ntide were mixed in equal volumes with 1.2 M sodium potassium\nphosphate (pH 4.9), 4% (v/v) glycerol, 4% (v/v) ethylene glycol.\nCrystals were grown at 22°C and appeared in one week and\nreached 100–200 m in each dimension over a period of 3 weeks.\nPrior to data collection, hLRH-1 crystals were dipped in cryopro-\ntectant (reservoir solution containing 11.5% glycerol and 11.5%\nethylene glycol) and flash-frozen in liquid nitrogen. The TIF-2 pep-\ntide sequence is KENALLRYLLDKDD and was used for both hLRH-1\nand hSF-1 crystallization. X-ray data were collected at IMCA 17-ID\nline (Argonne, Illinois), and data were processed and scaled with\nHKL2000. The hLRH-1 model was built by molecular replacement\nwith Amore program using hSF-1 as a search model and refined\nwith CNS. In this model, a second TIF-2 peptide is found in the\nasymmetric unit and forms loose contacts with the tail of the first\nTIF-2 peptide positioned in the coactivator cleft formed by H3, H4,\nand H12 of the hLRH-1 receptor. Rebuilding between rounds of\nrefinement was performed with the graphics program QUANTA.\nCell Culture and Transfections\nHuman HepG2 liver carcinoma cells were plated the day of trans-\nfection in 24 well plates and transfected with FUGENE 6 with\namounts of DNA indicated in figures and legends. For all experi-\nments, 200 ng/well of luciferase reporter were used and pCI-Neo-\nβ-gal (50 ng/well) were transfected along with wild-type or mutant\nreceptors. Luciferase activity was measured after 48 hr, as pre-\nviously described (Sablin et al., 2003). Expression of all mutants\nwas tested by Western blot analyses using anti-HA antibody (Co-\nvance) directed against an N-terminal tag. In all cases, mutant re-\nceptors were expressed at equivalent or higher levels than wild-\ntype protein. For N-butanol experiments, 0.1% 1-butanol was\nadded for 2 hr. All transfection experiments were done in triplicates\nand repeated at least three times; representative experiments are\nshown with standard errors indicated by bars.\nLigand Analyses and Phospholipid Binding Assays\nProtein samples were desalted and injected onto a column packed\nwith P6 gel (BioRad) to exchange nonvolatile buffers with 20 mM\nammonium acetate (pH 6.8). Buffer-exchanged protein was col-\nlected and back flushed at a flow rate of 0.75–1 µl/min into a na-\nnoflow electrospray ionization source fitted on a Waters/Micromass\nQTOF-1 (Beverly, Massachusetts) mass spectrometer. Data were col-\nlected in positive ion mode over a mass range of 1000–4000 m/z. For\nstructural determination of ligands, a similar in-gel filtration appara-\ntus was connected to an orthogonal acceleration time-of-flight\nmass spectrometer (LCMSD-TOF, Agilent) and a quadrupole ion\ntrap mass spectrometer (LCMSD Trap SL, Agilent). Samples were\nexamined utilizing negative ion electrospray using conditions that\ndenatured the protein-ligand complex.\nthese conditions, PI and PA lipids should mix completely into sta-\nble bilayers and consist primarily of PC. LBD proteins (2 mg/ml)\nwere exchanged into liposome buffer and combined with the lipo-\nsomes to produce a 25:1 molar ratio of PC to protein and a 2.5:1\nmolar ratio of PA, PI(3,4)P2, or PI(3,4,5)P3 to protein. Mixtures were\nincubated for 20 hr at room temperature, and protein was sepa-\nrated from liposomes by gel-filtration on Superdex 75 (Pharmacia),\nconcentrated, subjected to another round of gel filtration, and ana-\nlyzed by mass spectrometry as described above. The amount of\nSF-1 LBD protein associated with free liposomes (void volume) fol-\nliposome\nlowing gel filtration was determined to be <1% for all\nspecies.\nPeptide Binding Assays\nLBD proteins were serially diluted from 150 µM to 0.002 µM in bind-\ning buffer (50 mM NaPO4, 150 mM NaCl, pH 7.2, 1 mM DTT, 1 mM\nEDTA, 0.01% NP40, 10% glycerol). Diluted protein was added to\nfluorescent coregulator peptide (20 nM) in 384 well plates yielding\nfinal protein concentrations of 75–0.001 µM and 10 nM of fluores-\ncent peptide. Samples were analyzed as previously described\n(Moore et al., 2004). Construction of 20 aa coregulator peptides\nwith the general motif of CXXXXXXXLXXL/AL/AXXXXXXX were as\nfollows: peptides were synthesized in parallel using standard fluor-\nenylmethoxycarbonyl (Fmoc) chemistry in 48 well synthesis blocks\n(FlexChem System, Robbins) (Wellings and Atherton, 1997). Cou-\npling efficiency was monitored and the completed peptides were\ncleaved from the resin, followed by reversed-phase chromatogra-\nphy and mass spectrometry (MALDI-TOF/ESI) to purify the pep-\ntides, and are described in Moore et al. (2004).\nEvolutionary Analysis\nProtein sequences of 48 SF-1/LRH-1/FtzF1 family members with\nfull-length ligand binding domains were obtained from GenBank by\nBLAST search using Homo sapiens SF-1 as a query. Twenty addi-\ntional sequences from the closely related ERR, GCNF/GRF, and\nRXR groups (Thornton and DeSalle, 2000) were also obtained to\nserve as outgroup sequences. (See Supplemental Table S2 for ac-\ncession numbers and species names). These sequences were\naligned using ClustalX 1.83 software (Thompson et al., 1994) as-\nsuming a gap:change cost ratio of 10. The phylogeny of the gene\nfamily was then inferred from the aligned sequences using the\nBayesian Markov Chain Monte Carlo method (Ronquist and\nHuelsenbeck, 2003) implemented in MrBayes v. 3.0 software. Tree\nand parameter space were explored with four simultaneous chains\n(three heated) for 300,000 generations each, using the Jones model\nof protein evolution with a four-category discrete γ model of\namong-site rate variation. Priors were equal for tree topologies,\nuniform (0,10) for branch lengths, and uniform (0.1, 5) for the α\nshape parameter of the γ distribution. The initial 100,000 trees, a\npoint well past stationarity, were discarded as burn-in.\nPhylogenetic reconstruction of the ancestral states at internal\nnodes for ligand binding or ligand independence, coded as a binary\ncharacter, were conducted by parsimony and maximum likelihood.\nFor the former, we used Fitch’s parsimony algorithm (Fitch, 1971)\non a simple tree consisting of the four taxa for which ligand binding\nstates are known (Sablin et al., 2003; this paper). For maximum\nlikelihood reconstruction, we calculated the likelihood of each an-\ncestral state 0 (no ligand binding) and 1 (ligand binding) as the\nprobability of evolving the terminal states in mLRH-1, hLRH-1,\nmSF-1, and hSF-1, given the phylogeny that relates these genes,\nand a simple reversible Markov process for transitions between\nstates 0 and 1 (Pagel, 1999). The probability Pij of observing a state\nchange on a tree branch of length k is Pij =\ne−k.\nBranch lengths were assumed equal a priori and scaled to their\nmaximum likelihood values by hill climbing; unknown states at in-\nternal nodes other than the NR5A ancestor were treated as nui-\nsance parameters and optimized by maximum likelihood.\nassuming a Jones model of amino acid evolution and a four-cate-\ngory γ model of rate variation among sites (α optimized by ML). The\ntree for ancestral state reconstruction was the maximum posterior\nprobability tree from the Bayesian analysis, adjusted at three nodes\nthat had posterior probability <0.95 to make them consistent with\nknown taxonomic relationships among species (see Supplemental\nFigure S3). Branch lengths for the ancestral reconstruction were\noptimized by maximum likelihood using PAML software.\nSupplemental Data\nSupplemental Data include three figures and two tables and can\nbe found with this article online at http://www.cell.com/cgi/\ncontent/full/120/3/343/DC1/.\nAcknowledgments\nWe would like to thank Drs. D. Julius, K. Shokat, D. Stokoe, S.\nWadekar, N. Jouravel, D. Moore, and F. Szoka for critical com-\nments, experimental suggestions, and reagents. The IMCA-CAT\nbeamline 17-ID (or 17-BM) at the Advanced Photon Source was\nsupported by the companies of the Industrial Macromolecular\nCrystallography Association through a contract with Illinois Insti-\ntute of Technology. We would like to thank Drs. Cheng Yang and\nJoseph Ferrara from Rigaku/MSC Inc. for an invitation to this facil-\nity. This work was funded by NIH 1R21-GM70792-1 and the Oregon\nMedical Research Foundation to J.W.T., a HHMI predoctoral fellow-\nship to J.A.M., and a NIDDK-PO1 support to R.J.F., R.K.G., and\nH.A.I.\nReceived: November 3, 2004\nRevised: January 13, 2005\nAccepted: January 27, 2005\nPublished: February 10, 2005\nReferences\nBacqueville, D., Deleris, P., Mendre, C., Pieraggi, M.T., Chap, H.,\nGuillon, G., Perret, B., and Breton-Douillon, M. (2001). Characteriza-\ntion of a G protein-activated phosphoinositide 3-kinase in vascular\nsmooth muscle cell nuclei. J. Biol. Chem. 276, 22170–22176.\nBaker, K.D., Shewchuk, L.M., Kozlova, T., Makishima, M., Hassell,\nA., Wisely, B., Caravella, J.A., Lambert, M.H., Reinking, J.L., Krause,\nH., et al. (2003). The Drosophila orphan nuclear receptor DHR38\nmediates an atypical ecdysteroid signaling pathway. Cell 113,\n731–742.\nBillas, I.M., Moulinier, L., Rochel, N., and Moras, D. (2001). Crystal\nstructure of the ligand-binding domain of the ultraspiracle protein\nUSP, the ortholog of retinoid X receptors in insects. J. Biol. Chem.\n276, 7465–7474.\nBland, M.L., Jamieson, C.A., Akana, S.F., Bornstein, S.R., Eisen-\nhofer, G., Dallman, M.F., and Ingraham, H.A. (2000). Haploinsuffi-\nciency of steroidogenic factor-1 in mice disrupts adrenal develop-\nment leading to an impaired stress response. Proc. Natl. Acad. Sci.\nUSA 97, 14488–14493.\nBotrugno, O.A., Fayard, E., Annicotte, J.S., Haby, C., Brennan, T.,\nWendling, O., Tanaka, T., Kodama, T., Thomas, W., Auwerx, J., and\nSchoonjans, K. (2004). Synergy between LRH-1 and beta-catenin\ninduces G1 cyclin-mediated cell proliferation. Mol. Cell 15, 499–\n509.\nChristenson, L.K., McAllister, J.M., Martin, K.O., Javitt, N.B., Os-\nborne, T.F., and Strauss, J.F., 3rd. (1998). Oxysterol regulation of\nsteroidogenic acute regulatory protein gene expression. Structural\nspecificity and transcriptional and posttranscriptional actions. J.\nBiol. Chem. 273, 30729–30735.\nDarimont, B.D., Wagner, R.L., Apriletti, J.W., Stallcup, M.R., Kush-\nner, P.J., Baxter, J.D., Fletterick, R.J., and Yamamoto, K.R. (1998).\nStructure and specificity of nuclear receptor-coactivator interac-\ntions. Genes Dev. 273, 30729–30735.\nR., Ribeiro, R.C., Zambetti, G., Delacerda, L., Rodrigues, G.A., and\nHaddad, B.R. (2004). Amplification of the steroidogenic factor 1\n(SF-1) gene in childhood adrenocortical tumors. J. Clin. Endocrinol.\nMetab. Published online November 2004. 10.1210/jc.2004-0942\nDeleris, P., Bacqueville, D., Gayral, S., Carrez, L., Salles, J.P., Perret,\nB., and Breton-Douillon, M. (2003). SHIP-2 and PTEN are expressed\nand active in vascular smooth muscle cell nuclei, but only SHIP-2 is\nassociated with nuclear speckles. J. Biol. Chem. 278, 38884–38891.\nDelerive, P., Galardi, C.M., Bisi, J.E., Nicodeme, E., and Goodwin,\nB. (2004). Identification of liver receptor homolog-1 as a novel regu-\nlator of apolipoprotein AI gene transcription. Mol. Endocrinol. 18,\n2378–2387.\nDesclozeaux, M., Krylova, I.N., Horn, F., Fletterick, R.J., and Ingra-\nham, H.A. (2002). Phosphorylation and intramolecular stabilization\nof the ligand binding domain in the nuclear receptor steroidogenic\nfactor 1. Mol. Cell. Biol. 22, 7193–7203.\nDhe-Paganon, S., Duda, K., Iwamoto, M., Chi, Y.I., and Shoelson,\nS.E. (2002). Crystal structure of the HNF4 alpha ligand binding do-\nmain in complex with endogenous fatty acid ligand. J. Biol. Chem.\n277, 37973–37976.\nDietschy, J.M., and Turley, S.D. (2002). Control of cholesterol turn-\nover in the mouse. J. Biol. Chem. 277, 3801–3804.\nEscriva, H., Delaunay, F., and Laudet, V. (2000). Ligand binding and\nnuclear receptor evolution. Bioessays 22, 717–727.\nFang, Y., Vilella-Bach, M., Bachmann, R., Flanigan, A., and Chen, J.\n(2001). Phosphatidic acid-mediated mitogenic activation of mTOR\nsignaling. Science 294, 1942–1945.\nFayard, E., Auwerx, J., and Schoonjans, K. (2004). LRH-1: an or-\nphan nuclear receptor involved in development, metabolism and\nsteroidogenesis. Trends Cell Biol. 14, 250–260.\nFitch, W.M. (1971). Toward defining the course of evolution: minimal\nchange for a specific tree topology. Syst. Zool. 20, 406–416.\nHolter, E., Kotaja, N., Makela, S., Strauss, L., Kietz, S., Janne, O.A.,\nGustafsson, J.A., Palvimo, J.J., and Treuter, E. (2002). Inhibition of\nandrogen receptor\nfunction by the reproductive orphan\nnuclear receptor DAX-1. Mol. Endocrinol. 16, 515–528.\n(AR)\nHsuan, J., and Cockcroft, S. (2001). The PITP family of phosphati-\ndylinositol transfer proteins. Genome Biol. 2, REVIEWS3011.\nIngraham, H.A., Lala, D.S., Ikeda, Y., Luo, X., Shen, W.H., Nachtigal,\nM.W., Abbud, R., Nilson, J.H., and Parker, K.L. (1994). The nuclear\nreceptor steroidogenic factor 1 acts at multiple levels of the repro-\nductive axis. Genes Dev. 8, 2302–2312.\nIrvine, R.F. (2002). Nuclear lipid signaling. Sci. STKE, http://stke.\nsciencemag.org/cgi/content/full/OC_sigtrans;2002/150/re13.\nIto, M., Yu, R., and Jameson, J.L. (1997). DAX-1 inhibits SF-1-medi-\nated transactivation via a carboxy-terminal domain that is deleted\nin adrenal hypoplasia congenita. Mol. Cell. Biol. 17, 1476–1483.\nKallen, J.A., Schlaeppi, J.M., Bitsch, F., Geisse, S., Geiser, M., Del-\nhon, I., and Fournier, B. (2002). X-ray structure of the hRORalpha\nLBD at 1.63 A. Structural and functional data that cholesterol or a\ncholesterol derivative is the natural ligand of RORalpha. Structure\n(Camb.) 10, 1697–1707.\nLala, D.S., Syka, P.M., Lazarchik, S.B., Mangelsdorf, D.J., Parker,\nK.L., and Heyman, R.A. (1997). Activation of the orphan nuclear\nreceptor steroidogenic factor 1 by oxysterols. Proc. Natl. Acad. Sci.\nUSA 94, 4895–4900.\nLedeen, R.W., and Wu, G. (2004). Nuclear lipids: key signaling ef-\nfectors in the nervous system and other tissues. J. Lipid Res. 45, –.\nLee, M.K., Lebedeva, L., Suzawa, M., Wadekar, S., Desclozeaux,\nM., and Ingraham, H.A. (2005). Repression of orphan nuclear recep-\ntor activity via SUMO-modification and the Dead-Box Protein\nDP103 (Ddx20, Gemin-3). Mol. Biol. Cell 25, 1879–1890.\nLi, Y., Lambert, M.H., and Xu, H.E. (2003). Activation of nuclear re-\nceptors: a perspective from structural genomics. Structure (Camb.)\n11, 741–746.\nL.R., and Helmkamp, G.M., Jr. (2001). Structure of a multifunctional\nprotein. Mammalian phosphatidylinositol\ntransfer protein com-\nplexed with phosphatidylcholine. J. Biol. Chem. 276, 9246–9252.\nAccession Numbers\nCoordinates have been deposited in the Protein Data Bank as\nfollows: mSF-1 PDB ID 1YMT, RCSB code RCSB031696, hLRH-1\nPDB ID 1YOK and RCSB031753, and hSF-1 PDB ID 1YOW and\nRCSB031764.\nPhospholipids as NR5A Receptor Ligands\nLu, T.T., Makishima, M., Repa, J.J., Schoonjans, K., Kerr, T.A., Au-\nwerx, J., and Mangelsdorf, D.J. (2000). Molecular basis for feed-\nback regulation of bile acid synthesis by nuclear receptors. Mol.\nCell 6, 507–515.\nLudbrook, L.M., and Harley, V.R. (2004). Sex determination: a ‘win-\ndow’ of DAX1 activity. Trends Endocrinol. Metab. 15, 116–121.\nLuo, X., Ikeda, Y., and Parker, K.L. (1994). A cell-specific nuclear\nreceptor is essential for adrenal and gonadal development and sex-\nual differentiation. Cell 77, 481–490.\nMadauss, K., Juzumiene, D., Waitt, G., Williams, J.P., and Williams,\nS.P. (2004). Generation and characterization of human steroido-\ngenic factor 1 LBD crystals with and without bound co-factor pep-\ntide. Endocr. Res. 30, 775–785.\nMartelli, A.M., Tabellini, G., Borgatti, P., Bortul, R., Capitani, S., and\nNeri, L.M. (2003). Nuclear lipids: new functions for old molecules?\nJ. Cell. Biochem. 88, 455–461.\nMellon, S.H., and Bair, S.R. (1998). 25-Hydroxycholesterol is not a\nligand for the orphan nuclear receptor steroidogenic factor-1 (SF-\n1). Endocrinology 139, 3026–3029.\nMoore, J.M., Galicia, S.J., McReynolds, A.C., Nguyen, N.H., Scan-\nlan, T.S., and Guy, R.K. (2004). Quantitative proteomics of the thy-\nroid hormone receptor-coregulator interactions. J. Biol. Chem. 279,\n27584–27590.\nNachtigal, M.W., Hirokawa, Y., Enyeart-VanHouten, D.L., Flanagan,\nJ.N., Hammer, G.D., and Ingraham, H.A. (1998). Wilms’ tumor 1 and\nDax-1 modulate the orphan nuclear receptor SF-1 in sex-specific\ngene expression. Cell 93, 445–454.\nPagel, M. (1999). Inferring the historical patterns of biological evo-\nlution. Nature 401, 877–884.\nRonquist, F., and Huelsenbeck, J.P. (2003). MrBayes 3: Bayesian\nphylogenetic inference under mixed models. Bioinformatics 19,\n1572–1574.\nSablin, E.P., Krylova,\nI.N., Fletterick, R.J., and Ingraham, H.A.\n(2003). Structural basis for ligand-independent activation of the or-\nphan nuclear receptor LRH-1. Mol. Cell 11, 1575–1585.\nSoccio, R.E., and Breslow, J.L. (2003). StAR-related lipid transfer\n(START) proteins: mediators of intracellular lipid metabolism. J.\nBiol. Chem. 278, 22183–22186.\nSteinmetz, A.C., Renaud, J.P., and Moras, D. (2001). Binding of li-\ngands and activation of transcription by nuclear receptors. Annu.\nRev. Biophys. Biomol. Struct. 30, 329–359.\nThompson, J.D., Higgins, D.G., and Gibson, T.J. (1994). CLUSTALW:\nImproving the sensitivity of progressive multiple sequence align-\nment through sequence weighting, position-specific gap penalties\nand weight matrix choice. Nucleic Acids Res. 22, 4673–4680.\nThornton, J.W., and DeSalle, R. (2000). A new method to localize\nand test the significance of incongruence: detecting domain shuf-\nfling in the nuclear receptor superfamily. Syst. Biol. 49, 183–201.\nThornton, J.W., Need, E., and Crews, D. (2003). Resurrecting the\nancestral steroid receptor: ancient origin of estrogen signaling. Sci-\nence 301, 1714–1717.\nWang, Z., Benoit, G., Liu, J., Prasad, S., Aarnisalo, P., Liu, X., Xu,\nH., Walker, N.P., and Perlmann, T. (2003). Structure and function of\nNurr1 identifies a class of ligand-independent nuclear receptors.\nNature 423, 555–560.\nWeatherman, R.V., Fletterick, R.J., and Scanlan, T.S.\n(1999).\nNuclear-receptor ligands and ligand-binding domains. Annu. Rev.\nBiochem. 68, 559–581.\nWellings, D.A., and Atherton, E. (1997). Standard Fmoc protocols.\nMethods Enzymol. 289, 44–67.\nWisely, G.B., Miller, A.B., Davis, R.G., Thornquest, A.D., Jr., John-\nson, R., Spitzer, T., Sefler, A., Shearer, B., Moore, J.T., Willson, T.M.,\nand Williams, S.P. (2002). Hepatocyte nuclear factor 4 is a transcrip-\ntion factor that constitutively binds fatty acids. Structure (Camb.)\n10, 1225–1234.\nYang, Z., Kumar, S., and Nei, M. (1995). A new method of inference\nof ancestral nucleotide and amino acid sequences. Genetics 141,\n1641–1650.",
         "Vertebrate members of the nuclear receptor NR5A sub-\nfamily, which includes steroidogenic factor 1 (SF-1) and\nliver receptor homolog 1 (LRH-1), regulate crucial as-\npects of development, endocrine homeostasis, and\nmetabolism. Mouse LRH-1 is believed to be a ligand-\nindependent transcription factor with a large and\nempty hydrophobic pocket. Here we present struc-\ntural and biochemical data for three other NR5A mem-\nbers—mouse and human SF-1 and human LRH-1—\nwhich reveal that these receptors bind phosphatidyl\ninositol second messengers and that ligand binding\nis required for maximal activity. Evolutionary analysis\nof structure-function relationships across the SF-1/\nLRH-1 subfamily indicates that ligand binding is the\nancestral state of NR5A receptors and was uniquely\ndiminished or altered in the rodent LRH-1 lineage. We\npropose that phospholipids regulate gene expression\nby directly binding to NR5A nuclear receptors.\nIntroduction\nUnlike most nuclear receptors, which are regulated by\nsteroid, retinoid, and other nonpolar ligands, NR sub-\nfamily 5A are constitutively active in many cell types,\nand obvious ligand candidates have not been forth-\ncoming. SF-1 (NR5A1) orchestrates endocrine tissue\ndevelopment and male sexual differentiation and con-\ntrols steroid biosynthesis (Ingraham et al., 1994; Luo et\nal., 1994). LRH-1 (NR5A2) is essential in early develop-\nment and regulates bile acid synthesis, cholesterol\ntransport, and ovarian function (Fayard et al., 2004; Lu\net al., 2000). This subfamily is distinct in that members\nare able to bind as monomers to DNA, and they pos-\nsess an unusually large hinge domain that is subject to\nposttranslational modifications, including phosphoryla-\ntion and sumoylation (Desclozeaux et al., 2002; Lee et\nal., 2005). Although physiological roles for these recep-\ntors have emerged, ligand candidates have remained\ncontroversial. The proposed SF-1 ligand 25-hydro-\nxycholesterol, while appealing because of the recep-\ntor's regulation of steroid synthesis, appears to be in-\ncorrect because oxysterols fail to activate SF-1 in most\ncell lines (Christenson et al., 1998; Lala et al., 1997; Mel-\nlon and Bair, 1998). Further, the notion that ligands\nmight be dispensable for this subfamily of receptors\nwas bolstered by the crystal structure of mouse LRH-1\nligand binding domain (LBD), showing an active confor-\nmation in the absence of either ligand or coregulator\npeptide. Unlike other orphan nuclear receptors, which\neither lack a ligand binding pocket (e.g., Nurr1, DHR38,\nor ERRα) or have a constitutively bound lipid (e.g.,\nHNF4α, HNF4γ, or USP), mLRH-1 was found to possess\na large unoccupied hydrophobic pocket of w830 Å (re-\nviewed in Li et al. [2003]). Remarkably, mutations that\nfilled the mLRH-1 pocket failed to reduce transcrip-\ntional activity (Sablin et al., 2003).\nWhereas an elegant structural paradigm has devel-\noped to explain the activation of ligand-regulated re-\nceptors, no uniform mechanism has been proposed to\naccount for the modulation of orphan nuclear receptor\nactivity (Steinmetz et al., 2001; Weatherman et al.,\n1999). The presence of fortuitous lipid molecules re-\nvealed in crystallization experiments can be associated\nwith an inactive LBD conformation as noted for the\ndipterian USP LBD (Billas et al., 2001). On the other\nhand, lipid components have been proposed to stabi-\nlize an active LBD conformation in constitutively active\nnuclear receptors, with HNF4 binding fatty acids (Dhe-\nPaganon et al., 2002; Wisely et al., 2002) and RORα\nbinding cholesterol (Kallen et al., 2002). In this context,\nthe structure of the murine LRH-1 with its empty ligand\nbinding pocket remains a puzzle. To further explore\nhow NR5A orphan receptors are regulated, we obtained\nthe LBD crystal structures of other members of this\nsubfamily, including mSF-1, hSF-1, and hLRH-1. Here\nwe provide structural and biochemical data supporting\nligand regulation for NR5A orphan nuclear receptors.\nResults and Discussion\nCell\nhLRH-1/TIF-2\na (Å)\nb (Å)\nc (Å)\nSpace group\nMolecules per asymmetric unit\nResolution (Å)\nNumber of unique reflections\nData redundancy\nCompleteness (%)\nRsymm (%)\n< I/s(I) >\nRefinement\nR\nRfree\nRms deviation from ideality\nBond length (Å)\nBond angle (0)\nAverage B factor (A2)\nAll atoms\nProtein atoms\nWater molecules\n73.9\n73.9\n117.0\nP41212\n\n25–1.2\n100,431\n13.1\n99.9\n6.1\n39.2\n128.4\n66.0\n141.0\nP21\n\n20–2.5\n71,698\n\n97.9\n4.5\n28.0\n73.1\n73.1\n139.4\nP41212\n\n20–2.9\n15,614\n\n99.8\n5.8\n30.0\n59.9\n67.2\n79.6\nP212121\n\n40–2.5\n10,787\n\n91.9\n3.7\n44.0\nwere obtained, with and without TIF-2 peptide, and\nwere determined by MAD and SAD methods using sele-\nnomethionine substituted proteins, respectively (Mad-\nauss et al., 2004). The structure of hLRH-1 was det-\nermined by molecular replacement using the atomic\ncoordinates for hSF-1. The current best structures are\nrefined to 1.2 Å, 2.5 Å, and 2.5 Å for mSF-1, hLRH-1, and\nhSF-1, respectively. Relevant features of data collection\nand refinement statistics for each protein crystal are\nprovided in Table 1. All three LBDs exhibit a character-\nistic protein fold noted for other LBDs, with 12 α heli-\nces, two β strands, and helix H12 positioned in an\nactive conformation (Figure 1). A fourth layer is pro-\nvided by a structured helix H2, as previously shown for\nmLRH-1, although the position and length of this helix\nare the most varied features as judged by comparison\nof mSF-1 with hSF-1 LBD (Figure 1C) and mLRH-1 with\nhLRH-1 (Figure 1D). Upon refinement of these three\nLBD structures, additional electron density was present\nin the ligand binding pockets consistent with the pres-\nence of a phospholipid; the presence of ligand con-\ntrasts the reported structure of mLRH-1 showing no\ndensity inside or at the opening of the pocket.\nPhospholipids as NR5A Receptor Ligands\n(A) Nondenaturing mass spectrometry was carried out on all four LBD proteins used for crystallization (mSF-1, hSF-1 and mLRH-1, hLRH-1).\nSpectra are shown for mLRH-1 (left panel) and mSF-1 (right panel), with the apo [A], and the bound [B] species indicated. The molecular\nweight difference between the two peaks is also shown (arrow).\n(B) The concentrations of all bound LBD species, including the mSF-1 A270W pocket mutant, were determined as described and are listed\nfor equal amounts of LBD protein (w30 µM), with the exception of hSF-1, where more protein was used due to the poorer quality of protein\n(50 µM).\n(C) Product ions following collision-induced decomposition using denaturing electrospray tandem mass spectrometry (ESI MS/MS) on m/z\n745 (blue diamond) are shown for hSF-1 and reveal a phosphatidyl glycerol (chemical structure) ligand with two acyl chains of C16 and C18.\n(D) Electron density for mSF-1 is from a 2Fo−Fc map contoured at 1.2 sigma that shows the phospholipid ligand flanked by surrounding\nresidues (sticks). Three residues are labeled: Gly342, Tyr437, and Ala270, and the phosphate in the PG ligand is indicated (P).\n(E) Electron density found for both mSF-1 (blue) and hSF-1 ± TIF peptide (white or gray) ligands are superposed showing an identical position\nin the hydrophobic ligand binding pocket.\nPhospholipids as NR5A Receptor Ligands\nCell\nIn the upper left corner, a flow diagram is shown for ligand analyses of mSF-1 LBD protein following incubation with different liposomes, as\ndescribed in Experimental Procedures. Mass spectra profiles (ESI-LC/MS) are shown for the basal state (No Liposomes, panel A), and after\nincubations with the neutral liposomes of phosphatidyl choline (PC, panel B), with phosphatidic acid liposomes (PC:PA [9:1], panel C), and\nwith phosphatidyl inositol liposomes PC:PI(3,4)P2 [9:1] (PIP2, panel D) or PC:PI(3,4,5)P3 [9:1] (PIP3, panel E). In (B), (D), and (E), the small\npeaks at mass 28730 Da are PC liganded receptor (PC), and in (C), the peak at mass 28646 is PA liganded receptor (PA). Direct mass\nspectrometry analysis (without gel filtration) on the PC:PA-receptor sample revealed a strong peak of free phosphatidic acid (data not shown),\nshowing that ample PA was present in the PC:PA liposomes.\nThe binding of phospholipids was further explored\nby mass spectrometry analysis of purified LBD proteins\nafter incubation with liposomes containing phosphati-\ndyl choline (PC), mixed with either phosphatidic acid\n(PA), PI(3,4)P2, or PI(3,4,5)P3 in a 9:1 ratio. Remarkably,\nincubation of the mSF-1 LBD with the neutral PC lipo-\nsome showed an almost complete loss of the bacterial\nPG phospholipid with very little exchange or binding by\nPC; similar results were found with the PC:PA liposome\n(Figures 4B and 4C) with both profiles differing greatly\nfrom spectra observed with the PG bound state of the\nmSF-1 LBD (Figure 4A, No Liposomes). In contrast, effi-\ncient binding was observed for PI(3,4)P2 or PI(3,4,5)P3\nas evidenced by the increase mass of 970 and 1050 Da,\nrespectively, with only a small uptake of PC observed\n(Figures 4D and 4E). A similar experiment carried out\nwith mLRH-1 LBD revealed less selectivity compared\nto that of mSF-1, with equal exchange of PC, PI(3,4)P2,\nor PI(3,4,5)P3, as well as some liganded PA detected\n(see Supplemental Figure S2). These data taken to-\ngether with the PIP strip binding data suggest that SF-\n1s and hLRH-1 bind much more robustly and selec-\ntively to phosphorylated PIs than does mLRH-1. The\nstructural basis for this species difference in selective\nphospholipid binding is discussed below.\nure 5A). These orphan nuclear receptors contain a pu-\ntative LBD but no DBD; their capacity to repress SF-1,\nLRH-1, and steroid hormone receptors is mediated by\nregions within the very N- and C-terminal LBD (Holter\net al., 2002; Ito et al., 1997; Nachtigal et al., 1998). Un-\nlike NR4 orphan receptors (Nurr1, DHR38) that have\nlost their coactivator cleft (Baker et al., 2003; Wang et\nal., 2003), all four LBD structures of SF-1 and LRH-1\nexhibit a well-formed docking site for coregulators.\nPeptide binding assays were carried out\nfor both\nmLRH-1 and mSF-1 LBD proteins using C-terminal flu-\norescently tagged SHP and Dax-1 peptides and fluo-\nrescence anisotropy, as described (Moore et al., 2004).\nSelective high-affinity binding was found for both\nmLRH-1 and mSF-1, with SF-1 binding more tightly to\npeptides from Dax-1; only the N-terminal LXXLL motif\nin the Dax-1 LBD region is able to bind both mSF-1 and\nmLRH-1 (Figures 5A and 5B).\nPhospholipids as NR5A Receptor Ligands\n(A) Schematics of SHP and Dax-1 are shown\nwith colored boxes corresponding to LXXLL\npeptides used in the fluorescence anisot-\nropy assay, as depicted and listed in the\ntable.\n(B) Relative binding affinities for labeled pep-\ntides are indicated by color-coded affinities\n(Kd values) ranging from 0.5–10.0, 10.1–30.0,\nand >30.0 µM and were determined as de-\nscribed in Experimental Procedures. The\nmSHP-1 peptide used in crystallization of\nmSF-1 is RPTILYALLSPSPR (mSHP-1).\n(C) Ribbon depiction of the mSHP-1 peptide\nin the coactivator cleft of mSF-1 (orange rib-\nbon) or superposed with the mLRH-1 struc-\nture (beige ribbon). Residues are numbered\naccording to the database entry Q62227 for\nmSHP.\nLRH-1 regulate Dax-1 and SHP in a negative feedback\nregulatory mechanism (Lu et al., 2000).\nCell\nIn the top panel, ribbon diagrams of partial structures of mLRH-1\nand mSF-1 are shown. Helices H2 (red) and H3 (pink) of mSF-1 are\nsuperposed with the helices H2 and H3 of mLRH-1 (light yellow),\nand the phosphatidyl glycerol ligand is shown (blue stick). The H2\nof mSF-1 is approximately half the length of helix H2 in mLRH-1\nH2. The bottom panel shows reporter activity in HepG2 cells with\nincreasing concentrations of wild-type and helix H5 mutants for\nboth mLRH-1 and mSF-1. Relative luciferase activities are shown.\nSF-1. Further analysis shows residues within this clus-\nter to be crucial for phospholipid binding. Indeed, bind-\ning of this ligand is optimized by coordination of the\nphosphate head group at the base of the pocket, with\nhydrogen bonding provided by Gly342 at the beginning\nof H6, and Tyr437 and Lys441 at the end of H11, which\nare 2.8 Å, 2.6 Å, and 2.8 Å from the ligand, respectively\n(Figure 7B). These interactions effectively seal the bot-\ntom of the ligand binding pocket with the phosphate\ngroup from the ligand contributing to this closure (Fig-\nure 7C).\nA different scenario occurs in mLRH-1, where Glu440\nforms a strong ionic interaction with the opposing\nLys539 in helix H11 (analogous to Lys441 in mSF-1),\nsealing the pocket and stabilizing the mLRH-1 LBD in\nthe absence of the ligand (Figure 7C). The other unique\nresidues spanning helices H6-H7 in mLRH-1 (Thr439\nand Phe443, Figure 7A) are also predicted to minimize\nligand binding by clashing with both the phosphate\ngroup and aliphatic chains, respectively. To test the hy-\npothesis that key residues interacting with the lipid’s\nphosphate group are critical to ligand binding, mLRH-1\nwas “humanized” by an E440G mutation. Wild-type and\nmutant receptors were assayed before and after reduc-\ning cellular pools of phosphatidic acid and subsequent\nphosphatidyl\ninositols with a competitive inhibitor of\nphospholipase D (Fang et al., 2001). N-butanol treat-\nment had a marginal effect on the activity of mLRH-1\nbut substantially reduced activation by the humanized\nE440G mLRH-1, and to a lesser extent wild-type mSF-1\n(Figure 7D). Collectively, these data are consistent with\nthe lowered selectively observed for mLRH-1 phospho-\nlipid binding and are predicted by structures of the\nLRH-1 and SF-1 orthologs.\nRecent Divergence in the Rodent LRH-1\nEvolutionary analysis of structure-function relations in\nthe LRH-1/SF-1 subfamily indicates that ligand binding\nis the ancestral state for these proteins where ligand\nbinding was subsequently reduced or altered in the lin-\neage leading to the rodent LRH-1. We coded the pres-\nence or absence of ligand binding as a phylogenetic\ncharacter and reconstructed its evolution on the phy-\nlogeny of the NR5A gene family (Figure 8). As described\nin Experimental Procedures, both maximum parsimony\nand maximum likelihoods were used to evaluate two\nhypotheses concerning the function of the common an-\ncestor of the entire NR5A subfamily—that the ancestor\nwas liganded (H1) and that it was unliganded (H0). Un-\nder H1, a single loss of ligand binding is required on the\nbranch leading to the rodent LRH-1, whereas H0 re-\nquires ligand binding to have been gained indepen-\ndently in the SF-1 and human LRH-1 lineages. H1 is\ntherefore the most parsimonious hypothesis. A maxi-\nmum likelihood analysis assuming that ligand binding\ncan be gained or lost under a simple reversible Markov\nprocess also supports H1, albeit nonsignificantly, indi-\ncating a loss of ligand binding in rodent LRH-1s (likeli-\nhood ratio = 3.05).\nPhospholipids as NR5A Receptor Ligands\n(A) Sequence alignment of the region span-\nning helices H6 and H7 is shown for a limited\nnumber of species. Rodent LRH-1-specific\nchanges are highlighted in red.\n(B) Electron density showing coordination of\nthe phosphate group by three residues in\nmSF-1. These include Gly342 (H7) and Tyr437\nand Lys441 (H11) with hydrogen bonds indi-\ncated (dashed line). The phosphate is la-\nbeled (P).\n(C) Interactions between helices H7 and H11\nare depicted for mSF-1, with phospholipid li-\ngand (white stick) and for mLRH-1 without\nligand. The salt bridge between Glu440 and\nLys539 in mLRH-1 is depicted.\n(D) Reporter activity is shown after depletion\nof phosphatidic acid by butanol with wild-type\nmLRH-1, mSF-1, and “humanized” mLRH-1\nE440G mutant transfected into HepG2 cells.\nActivity is expressed as fold activation using\nthe aromatase luciferase reporter.\nare known for only four family members, sequences are\navailable for dozens. We used maximum likelihood\n(Yang et al., 1995) to determine ancestral amino acid\nstates and evolutionary changes in the H6-H7 residues\ncorresponding to Thr439, Glu440, and Phe443 in rodent\nLRH-1 (TEF, Figure 8). These sites were chosen be-\ncause, as discussed above, structural and experimen-\ntal evidence indicates that they exclude or diminish\nphospholipid binding in mLRH-1, but the homologous\nresidues, Ala, Gly, and Leu, confer ligand binding on\nhLRH-1 and the SF-1s (AGL, Figure 8). The NR5A an-\ncestral sequence has all three ligand-associated states\n(AGL), and this reconstruction has extremely high confi-\ndence (posterior probability 100% for each site). The\nderived TEF residues that close the pocket in the ab-\nsence of ligand all evolved much later on than the\nbranch leading to the rodent LRH-1 (posterior prob-\nability 100% for each site). Examination of the genetic\ncode indicates that each of these replacements can be\nconferred by a single base-pair mutation.\nacid replacements. The lost or altered ligand binding\nstate in rodent LRH-1 is the second example of a taxon-\nspecific nuclear receptor losing the capacity to bind li-\ngand from a liganded ancestor; the other is the estro-\ngen receptor ortholog of the mollusc Aplysia californica\n(Thornton et al., 2003). These findings cast doubt on\nthe hypothesis that ligand binding in the nuclear recep-\ntor superfamily was independently gained numerous\ntimes from an ancestral orphan receptor (Escriva et al.,\n2000). If the loss of ligand dependence is common and\nrequires only a few sequence changes, the opposite\nscenario—repeated loss of ligand binding from a li-\nganded ancestor—may better characterize NR evo-\nlution.\nCell\nThe maximum posterior probability phylogeny of the NR5 gene\nfamily is shown, including LRH-1s (red), SF-1s (blue), and outgroup\nsequences (black, with number of sequences analyzed shown in\nparentheses). Residues homologous to mLRH-1 positions 439, 440,\nand 443 are shown for each LRH-1/SF-1 sequence (aa). Maximum\nlikelihood reconstructions of ancestral amino acid states are shown\nfor SF-1 and LRH-1 proteins in the last common ancestor of verte-\nbrates (squares), of tetrapods (triangles), and of mammals (circles),\nas well as for the common ancestral gene preceding the duplica-\ntion that split the SF-1 and LRH-1 lineages (star). All ancestral re-\nconstructions have 100% posterior probability. Green arrowhead\nshows acquisition of amino acids that exclude ligand in the rodent\nLRH-1 lineage and loss of ligand binding on the same branch. Spe-\ncies names and accession numbers are provided in Supplemental\nTable S2.\nimplies that a distinct pool of cellular lipids signal via\nmodulation of the NR5A nuclear receptors. Indeed, the\nliterature suggests that nuclear phosphatidyl\ninositol\nlipids exist as well as enzymes affecting their metabo-\nlism, such as phosphatidyl inositol 3-kinase (PI-3K) (Ir-\nvine, 2002; Ledeen and Wu, 2004; Martelli et al., 2003).\nAdditionally, both PI-3 and PI-5 phosphatases exhibit\nprominent nuclear staining (Bacqueville et al., 2001; De-\nleris et al., 2003), thus providing a mechanism to down-\nregulate PI nuclear signaling. Although we found that\nWortmannin (PI-3 kinase inhibitor) and the PI-3 phos-\nphatase PTEN affected the activities of wild-type and\npocket mutant SF-1 receptors differently (I.N.K., unpub-\nlished data), further studies and additional pharmaco-\nlogical tools are needed to selectively disrupt nuclear\nphospholipid signaling pathways. Nonetheless, the link\nbetween PI-3 kinase signaling and NR5A receptor ac-\ntivity is intriguing given the roles that both SF-1 and\nLRH-1 play in cell proliferation. Indeed, haploinsuffi-\nciency of both sf-1 and lrh-1 leads to decreased cell\nproliferation (Bland et al., 2000; Botrugno et al., 2004),\nwhile increased gene dosage of SF-1 is reported to be\nassociated with human adrenal cortical tumors (de Fi-\ngueiredo et al., 2004). Because PI-3 kinase pathways\nare known to regulate cell cycle progression and cell\ngrowth, it is tempting to speculate that extracellular\nevents that trigger this pathway might also elevate\nnuclear PIs, and thus modulate SF-1/LRH-1 activities.\nFinally, it is worth considering how phospholipids are\ndelivered to nuclear receptors. Cytoplasmic shuttling of\nPIs to the plasma membrane is carried out by phospha-\ntidyl inositol transport proteins (PITP) in an ATP-inde-\npendent manner; these proteins are also present in the\nnucleus. Crystal structures of phospholipid binding\nproteins show a large hydrophobic pocket or tunnel\nable to bind lipids (Yoder et al., 2001; Hsuan and Cock-\ncroft, 2001; Soccio and Breslow, 2003). Whether PIs are\ndirectly taken up by SF-1/LRH-1 from membranes (due\nto transient receptor shuttling between the cytoplasm\nand nucleus) or are delivered by nuclear phospholipid\nbinding proteins remains unclear. While many ques-\ntions are still to be answered, our current study leads\nus to hypothesize phosphatidyl inositols as bona fide\nligands for NR5A receptors and, as such, provide a di-\nrect link between transcription and phospholipid sig-\nnaling.\nExperimental Procedures\nPhospholipids as NR5A Receptor Ligands\ndiffracted to 1.1–1.2 Å. Data were integrated using DENZO and\nscaled with SCALEPACK. For hLRH-1, the protein and TIF-2 pep-\ntide were mixed in equal volumes with 1.2 M sodium potassium\nphosphate (pH 4.9), 4% (v/v) glycerol, 4% (v/v) ethylene glycol.\nCrystals were grown at 22°C and appeared in one week and\nreached 100–200 m in each dimension over a period of 3 weeks.\nPrior to data collection, hLRH-1 crystals were dipped in cryopro-\ntectant (reservoir solution containing 11.5% glycerol and 11.5%\nethylene glycol) and flash-frozen in liquid nitrogen. The TIF-2 pep-\ntide sequence is KENALLRYLLDKDD and was used for both hLRH-1\nand hSF-1 crystallization. X-ray data were collected at IMCA 17-ID\nline (Argonne, Illinois), and data were processed and scaled with\nHKL2000. The hLRH-1 model was built by molecular replacement\nwith Amore program using hSF-1 as a search model and refined\nwith CNS. In this model, a second TIF-2 peptide is found in the\nasymmetric unit and forms loose contacts with the tail of the first\nTIF-2 peptide positioned in the coactivator cleft formed by H3, H4,\nand H12 of the hLRH-1 receptor. Rebuilding between rounds of\nrefinement was performed with the graphics program QUANTA.\nCell Culture and Transfections\nHuman HepG2 liver carcinoma cells were plated the day of trans-\nfection in 24 well plates and transfected with FUGENE 6 with\namounts of DNA indicated in figures and legends. For all experi-\nments, 200 ng/well of luciferase reporter were used and pCI-Neo-\nβ-gal (50 ng/well) were transfected along with wild-type or mutant\nreceptors. Luciferase activity was measured after 48 hr, as pre-\nviously described (Sablin et al., 2003). Expression of all mutants\nwas tested by Western blot analyses using anti-HA antibody (Co-\nvance) directed against an N-terminal tag. In all cases, mutant re-\nceptors were expressed at equivalent or higher levels than wild-\ntype protein. For N-butanol experiments, 0.1% 1-butanol was\nadded for 2 hr. All transfection experiments were done in triplicates\nand repeated at least three times; representative experiments are\nshown with standard errors indicated by bars.\nLigand Analyses and Phospholipid Binding Assays\nProtein samples were desalted and injected onto a column packed\nwith P6 gel (BioRad) to exchange nonvolatile buffers with 20 mM\nammonium acetate (pH 6.8). Buffer-exchanged protein was col-\nlected and back flushed at a flow rate of 0.75–1 µl/min into a na-\nnoflow electrospray ionization source fitted on a Waters/Micromass\nQTOF-1 (Beverly, Massachusetts) mass spectrometer. Data were col-\nlected in positive ion mode over a mass range of 1000–4000 m/z. For\nstructural determination of ligands, a similar in-gel filtration appara-\ntus was connected to an orthogonal acceleration time-of-flight\nmass spectrometer (LCMSD-TOF, Agilent) and a quadrupole ion\ntrap mass spectrometer (LCMSD Trap SL, Agilent). Samples were\nexamined utilizing negative ion electrospray using conditions that\ndenatured the protein-ligand complex.\nthese conditions, PI and PA lipids should mix completely into sta-\nble bilayers and consist primarily of PC. LBD proteins (2 mg/ml)\nwere exchanged into liposome buffer and combined with the lipo-\nsomes to produce a 25:1 molar ratio of PC to protein and a 2.5:1\nmolar ratio of PA, PI(3,4)P2, or PI(3,4,5)P3 to protein. Mixtures were\nincubated for 20 hr at room temperature, and protein was sepa-\nrated from liposomes by gel-filtration on Superdex 75 (Pharmacia),\nconcentrated, subjected to another round of gel filtration, and ana-\nlyzed by mass spectrometry as described above. The amount of\nSF-1 LBD protein associated with free liposomes (void volume) fol-\nliposome\nlowing gel filtration was determined to be <1% for all\nspecies.\nPeptide Binding Assays\nLBD proteins were serially diluted from 150 µM to 0.002 µM in bind-\ning buffer (50 mM NaPO4, 150 mM NaCl, pH 7.2, 1 mM DTT, 1 mM\nEDTA, 0.01% NP40, 10% glycerol). Diluted protein was added to\nfluorescent coregulator peptide (20 nM) in 384 well plates yielding\nfinal protein concentrations of 75–0.001 µM and 10 nM of fluores-\ncent peptide. Samples were analyzed as previously described\n(Moore et al., 2004). Construction of 20 aa coregulator peptides\nwith the general motif of CXXXXXXXLXXL/AL/AXXXXXXX were as\nfollows: peptides were synthesized in parallel using standard fluor-\nenylmethoxycarbonyl (Fmoc) chemistry in 48 well synthesis blocks\n(FlexChem System, Robbins) (Wellings and Atherton, 1997). Cou-\npling efficiency was monitored and the completed peptides were\ncleaved from the resin, followed by reversed-phase chromatogra-\nphy and mass spectrometry (MALDI-TOF/ESI) to purify the pep-\ntides, and are described in Moore et al. (2004).\nEvolutionary Analysis\nProtein sequences of 48 SF-1/LRH-1/FtzF1 family members with\nfull-length ligand binding domains were obtained from GenBank by\nBLAST search using Homo sapiens SF-1 as a query. Twenty addi-\ntional sequences from the closely related ERR, GCNF/GRF, and\nRXR groups (Thornton and DeSalle, 2000) were also obtained to\nserve as outgroup sequences. (See Supplemental Table S2 for ac-\ncession numbers and species names). These sequences were\naligned using ClustalX 1.83 software (Thompson et al., 1994) as-\nsuming a gap:change cost ratio of 10. The phylogeny of the gene\nfamily was then inferred from the aligned sequences using the\nBayesian Markov Chain Monte Carlo method (Ronquist and\nHuelsenbeck, 2003) implemented in MrBayes v. 3.0 software. Tree\nand parameter space were explored with four simultaneous chains\n(three heated) for 300,000 generations each, using the Jones model\nof protein evolution with a four-category discrete γ model of\namong-site rate variation. Priors were equal for tree topologies,\nuniform (0,10) for branch lengths, and uniform (0.1, 5) for the α\nshape parameter of the γ distribution. The initial 100,000 trees, a\npoint well past stationarity, were discarded as burn-in.\nPhylogenetic reconstruction of the ancestral states at internal\nnodes for ligand binding or ligand independence, coded as a binary\ncharacter, were conducted by parsimony and maximum likelihood.\nFor the former, we used Fitch’s parsimony algorithm (Fitch, 1971)\non a simple tree consisting of the four taxa for which ligand binding\nstates are known (Sablin et al., 2003; this paper). For maximum\nlikelihood reconstruction, we calculated the likelihood of each an-\ncestral state 0 (no ligand binding) and 1 (ligand binding) as the\nprobability of evolving the terminal states in mLRH-1, hLRH-1,\nmSF-1, and hSF-1, given the phylogeny that relates these genes,\nand a simple reversible Markov process for transitions between\nstates 0 and 1 (Pagel, 1999). The probability Pij of observing a state\nchange on a tree branch of length k is Pij =\ne−k.\nBranch lengths were assumed equal a priori and scaled to their\nmaximum likelihood values by hill climbing; unknown states at in-\nternal nodes other than the NR5A ancestor were treated as nui-\nsance parameters and optimized by maximum likelihood.\nassuming a Jones model of amino acid evolution and a four-cate-\ngory γ model of rate variation among sites (α optimized by ML). The\ntree for ancestral state reconstruction was the maximum posterior\nprobability tree from the Bayesian analysis, adjusted at three nodes\nthat had posterior probability <0.95 to make them consistent with\nknown taxonomic relationships among species (see Supplemental\nFigure S3). Branch lengths for the ancestral reconstruction were\noptimized by maximum likelihood using PAML software.\nSupplemental Data\nSupplemental Data include three figures and two tables and can\nbe found with this article online at http://www.cell.com/cgi/\ncontent/full/120/3/343/DC1/.\nAcknowledgments\nWe would like to thank Drs. D. Julius, K. Shokat, D. Stokoe, S.\nWadekar, N. Jouravel, D. Moore, and F. Szoka for critical com-\nments, experimental suggestions, and reagents. The IMCA-CAT\nbeamline 17-ID (or 17-BM) at the Advanced Photon Source was\nsupported by the companies of the Industrial Macromolecular\nCrystallography Association through a contract with Illinois Insti-\ntute of Technology. We would like to thank Drs. Cheng Yang and\nJoseph Ferrara from Rigaku/MSC Inc. for an invitation to this facil-\nity. This work was funded by NIH 1R21-GM70792-1 and the Oregon\nMedical Research Foundation to J.W.T., a HHMI predoctoral fellow-\nship to J.A.M., and a NIDDK-PO1 support to R.J.F., R.K.G., and\nH.A.I.\nReceived: November 3, 2004\nRevised: January 13, 2005\nAccepted: January 27, 2005\nPublished: February 10, 2005\nReferences\nBacqueville, D., Deleris, P., Mendre, C., Pieraggi, M.T., Chap, H.,\nGuillon, G., Perret, B., and Breton-Douillon, M. (2001). Characteriza-\ntion of a G protein-activated phosphoinositide 3-kinase in vascular\nsmooth muscle cell nuclei. J. Biol. Chem. 276, 22170–22176.\nBaker, K.D., Shewchuk, L.M., Kozlova, T., Makishima, M., Hassell,\nA., Wisely, B., Caravella, J.A., Lambert, M.H., Reinking, J.L., Krause,\nH., et al. (2003). The Drosophila orphan nuclear receptor DHR38\nmediates an atypical ecdysteroid signaling pathway. Cell 113,\n731–742.\nBillas, I.M., Moulinier, L., Rochel, N., and Moras, D. (2001). Crystal\nstructure of the ligand-binding domain of the ultraspiracle protein\nUSP, the ortholog of retinoid X receptors in insects. J. Biol. Chem.\n276, 7465–7474.\nBland, M.L., Jamieson, C.A., Akana, S.F., Bornstein, S.R., Eisen-\nhofer, G., Dallman, M.F., and Ingraham, H.A. (2000). Haploinsuffi-\nciency of steroidogenic factor-1 in mice disrupts adrenal develop-\nment leading to an impaired stress response. Proc. Natl. Acad. Sci.\nUSA 97, 14488–14493.\nBotrugno, O.A., Fayard, E., Annicotte, J.S., Haby, C., Brennan, T.,\nWendling, O., Tanaka, T., Kodama, T., Thomas, W., Auwerx, J., and\nSchoonjans, K. (2004). Synergy between LRH-1 and beta-catenin\ninduces G1 cyclin-mediated cell proliferation. Mol. Cell 15, 499–\n509.\nChristenson, L.K., McAllister, J.M., Martin, K.O., Javitt, N.B., Os-\nborne, T.F., and Strauss, J.F., 3rd. (1998). Oxysterol regulation of\nsteroidogenic acute regulatory protein gene expression. Structural\nspecificity and transcriptional and posttranscriptional actions. J.\nBiol. Chem. 273, 30729–30735.\nDarimont, B.D., Wagner, R.L., Apriletti, J.W., Stallcup, M.R., Kush-\nner, P.J., Baxter, J.D., Fletterick, R.J., and Yamamoto, K.R. (1998).\nStructure and specificity of nuclear receptor-coactivator interac-\ntions. Genes Dev. 273, 30729–30735.\nR., Ribeiro, R.C., Zambetti, G., Delacerda, L., Rodrigues, G.A., and\nHaddad, B.R. (2004). Amplification of the steroidogenic factor 1\n(SF-1) gene in childhood adrenocortical tumors. J. Clin. Endocrinol.\nMetab. Published online November 2004. 10.1210/jc.2004-0942\nDeleris, P., Bacqueville, D., Gayral, S., Carrez, L., Salles, J.P., Perret,\nB., and Breton-Douillon, M. (2003). SHIP-2 and PTEN are expressed\nand active in vascular smooth muscle cell nuclei, but only SHIP-2 is\nassociated with nuclear speckles. J. Biol. Chem. 278, 38884–38891.\nDelerive, P., Galardi, C.M., Bisi, J.E., Nicodeme, E., and Goodwin,\nB. (2004). Identification of liver receptor homolog-1 as a novel regu-\nlator of apolipoprotein AI gene transcription. Mol. Endocrinol. 18,\n2378–2387.\nDesclozeaux, M., Krylova, I.N., Horn, F., Fletterick, R.J., and Ingra-\nham, H.A. (2002). Phosphorylation and intramolecular stabilization\nof the ligand binding domain in the nuclear receptor steroidogenic\nfactor 1. Mol. Cell. Biol. 22, 7193–7203.\nDhe-Paganon, S., Duda, K., Iwamoto, M., Chi, Y.I., and Shoelson,\nS.E. (2002). Crystal structure of the HNF4 alpha ligand binding do-\nmain in complex with endogenous fatty acid ligand. J. Biol. Chem.\n277, 37973–37976.\nDietschy, J.M., and Turley, S.D. (2002). Control of cholesterol turn-\nover in the mouse. J. Biol. Chem. 277, 3801–3804.\nEscriva, H., Delaunay, F., and Laudet, V. (2000). Ligand binding and\nnuclear receptor evolution. Bioessays 22, 717–727.\nFang, Y., Vilella-Bach, M., Bachmann, R., Flanigan, A., and Chen, J.\n(2001). Phosphatidic acid-mediated mitogenic activation of mTOR\nsignaling. Science 294, 1942–1945.\nFayard, E., Auwerx, J., and Schoonjans, K. (2004). LRH-1: an or-\nphan nuclear receptor involved in development, metabolism and\nsteroidogenesis. Trends Cell Biol. 14, 250–260.\nFitch, W.M. (1971). Toward defining the course of evolution: minimal\nchange for a specific tree topology. Syst. Zool. 20, 406–416.\nHolter, E., Kotaja, N., Makela, S., Strauss, L., Kietz, S., Janne, O.A.,\nGustafsson, J.A., Palvimo, J.J., and Treuter, E. (2002). Inhibition of\nandrogen receptor\nfunction by the reproductive orphan\nnuclear receptor DAX-1. Mol. Endocrinol. 16, 515–528.\n(AR)\nHsuan, J., and Cockcroft, S. (2001). The PITP family of phosphati-\ndylinositol transfer proteins. Genome Biol. 2, REVIEWS3011.\nIngraham, H.A., Lala, D.S., Ikeda, Y., Luo, X., Shen, W.H., Nachtigal,\nM.W., Abbud, R., Nilson, J.H., and Parker, K.L. (1994). The nuclear\nreceptor steroidogenic factor 1 acts at multiple levels of the repro-\nductive axis. Genes Dev. 8, 2302–2312.\nIrvine, R.F. (2002). Nuclear lipid signaling. Sci. STKE, http://stke.\nsciencemag.org/cgi/content/full/OC_sigtrans;2002/150/re13.\nIto, M., Yu, R., and Jameson, J.L. (1997). DAX-1 inhibits SF-1-medi-\nated transactivation via a carboxy-terminal domain that is deleted\nin adrenal hypoplasia congenita. Mol. Cell. Biol. 17, 1476–1483.\nKallen, J.A., Schlaeppi, J.M., Bitsch, F., Geisse, S., Geiser, M., Del-\nhon, I., and Fournier, B. (2002). X-ray structure of the hRORalpha\nLBD at 1.63 A. Structural and functional data that cholesterol or a\ncholesterol derivative is the natural ligand of RORalpha. Structure\n(Camb.) 10, 1697–1707.\nLala, D.S., Syka, P.M., Lazarchik, S.B., Mangelsdorf, D.J., Parker,\nK.L., and Heyman, R.A. (1997). Activation of the orphan nuclear\nreceptor steroidogenic factor 1 by oxysterols. Proc. Natl. Acad. Sci.\nUSA 94, 4895–4900.\nLedeen, R.W., and Wu, G. (2004). Nuclear lipids: key signaling ef-\nfectors in the nervous system and other tissues. J. Lipid Res. 45, –.\nLee, M.K., Lebedeva, L., Suzawa, M., Wadekar, S., Desclozeaux,\nM., and Ingraham, H.A. (2005). Repression of orphan nuclear recep-\ntor activity via SUMO-modification and the Dead-Box Protein\nDP103 (Ddx20, Gemin-3). Mol. Biol. Cell 25, 1879–1890.\nLi, Y., Lambert, M.H., and Xu, H.E. (2003). Activation of nuclear re-\nceptors: a perspective from structural genomics. Structure (Camb.)\n11, 741–746.\nL.R., and Helmkamp, G.M., Jr. (2001). Structure of a multifunctional\nprotein. Mammalian phosphatidylinositol\ntransfer protein com-\nplexed with phosphatidylcholine. J. Biol. Chem. 276, 9246–9252.\nAccession Numbers\nCoordinates have been deposited in the Protein Data Bank as\nfollows: mSF-1 PDB ID 1YMT, RCSB code RCSB031696, hLRH-1\nPDB ID 1YOK and RCSB031753, and hSF-1 PDB ID 1YOW and\nRCSB031764.\nPhospholipids as NR5A Receptor Ligands\nLu, T.T., Makishima, M., Repa, J.J., Schoonjans, K., Kerr, T.A., Au-\nwerx, J., and Mangelsdorf, D.J. (2000). Molecular basis for feed-\nback regulation of bile acid synthesis by nuclear receptors. Mol.\nCell 6, 507–515.\nLudbrook, L.M., and Harley, V.R. (2004). Sex determination: a ‘win-\ndow’ of DAX1 activity. Trends Endocrinol. Metab. 15, 116–121.\nLuo, X., Ikeda, Y., and Parker, K.L. (1994). A cell-specific nuclear\nreceptor is essential for adrenal and gonadal development and sex-\nual differentiation. Cell 77, 481–490.\nMadauss, K., Juzumiene, D., Waitt, G., Williams, J.P., and Williams,\nS.P. (2004). Generation and characterization of human steroido-\ngenic factor 1 LBD crystals with and without bound co-factor pep-\ntide. Endocr. Res. 30, 775–785.\nMartelli, A.M., Tabellini, G., Borgatti, P., Bortul, R., Capitani, S., and\nNeri, L.M. (2003). Nuclear lipids: new functions for old molecules?\nJ. Cell. Biochem. 88, 455–461.\nMellon, S.H., and Bair, S.R. (1998). 25-Hydroxycholesterol is not a\nligand for the orphan nuclear receptor steroidogenic factor-1 (SF-\n1). Endocrinology 139, 3026–3029.\nMoore, J.M., Galicia, S.J., McReynolds, A.C., Nguyen, N.H., Scan-\nlan, T.S., and Guy, R.K. (2004). Quantitative proteomics of the thy-\nroid hormone receptor-coregulator interactions. J. Biol. Chem. 279,\n27584–27590.\nNachtigal, M.W., Hirokawa, Y., Enyeart-VanHouten, D.L., Flanagan,\nJ.N., Hammer, G.D., and Ingraham, H.A. (1998). Wilms’ tumor 1 and\nDax-1 modulate the orphan nuclear receptor SF-1 in sex-specific\ngene expression. Cell 93, 445–454.\nPagel, M. (1999). Inferring the historical patterns of biological evo-\nlution. Nature 401, 877–884.\nRonquist, F., and Huelsenbeck, J.P. (2003). MrBayes 3: Bayesian\nphylogenetic inference under mixed models. Bioinformatics 19,\n1572–1574.\nSablin, E.P., Krylova,\nI.N., Fletterick, R.J., and Ingraham, H.A.\n(2003). Structural basis for ligand-independent activation of the or-\nphan nuclear receptor LRH-1. Mol. Cell 11, 1575–1585.\nSoccio, R.E., and Breslow, J.L. (2003). StAR-related lipid transfer\n(START) proteins: mediators of intracellular lipid metabolism. J.\nBiol. Chem. 278, 22183–22186.\nSteinmetz, A.C., Renaud, J.P., and Moras, D. (2001). Binding of li-\ngands and activation of transcription by nuclear receptors. Annu.\nRev. Biophys. Biomol. Struct. 30, 329–359.\nThompson, J.D., Higgins, D.G., and Gibson, T.J. (1994). CLUSTALW:\nImproving the sensitivity of progressive multiple sequence align-\nment through sequence weighting, position-specific gap penalties\nand weight matrix choice. Nucleic Acids Res. 22, 4673–4680.\nThornton, J.W., and DeSalle, R. (2000). A new method to localize\nand test the significance of incongruence: detecting domain shuf-\nfling in the nuclear receptor superfamily. Syst. Biol. 49, 183–201.\nThornton, J.W., Need, E., and Crews, D. (2003). Resurrecting the\nancestral steroid receptor: ancient origin of estrogen signaling. Sci-\nence 301, 1714–1717.\nWang, Z., Benoit, G., Liu, J., Prasad, S., Aarnisalo, P., Liu, X., Xu,\nH., Walker, N.P., and Perlmann, T. (2003). Structure and function of\nNurr1 identifies a class of ligand-independent nuclear receptors.\nNature 423, 555–560.\nWeatherman, R.V., Fletterick, R.J., and Scanlan, T.S.\n(1999).\nNuclear-receptor ligands and ligand-binding domains. Annu. Rev.\nBiochem. 68, 559–581.\nWellings, D.A., and Atherton, E. (1997). Standard Fmoc protocols.\nMethods Enzymol. 289, 44–67.\nWisely, G.B., Miller, A.B., Davis, R.G., Thornquest, A.D., Jr., John-\nson, R., Spitzer, T., Sefler, A., Shearer, B., Moore, J.T., Willson, T.M.,\nand Williams, S.P. (2002). Hepatocyte nuclear factor 4 is a transcrip-\ntion factor that constitutively binds fatty acids. Structure (Camb.)\n10, 1225–1234.\nYang, Z., Kumar, S., and Nei, M. (1995). A new method of inference\nof ancestral nucleotide and amino acid sequences. Genetics 141,\n1641–1650.",
         "I. Krylova,E. Sablin,Jamie M. R. Moore,R. Xu,G. Waitt,J. A. MacKay,D. Juzumiene,J. Bynum,K. Madauss,V. Montana,L. Lebedeva,M. Suzawa,Jon D. Williams,Shawn P. Williams,R. Guy,J. W. Thornton,R. Fletterick,T. Willson,H. Ingraham",
         null
        ],
        [
         "45",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "Abstract—The dopaminergic and renin-angiotensin systems regulate blood pressure, in part, by affecting sodium transport\nin renal proximal tubules (RPTs). We have reported that activation of a D1-like receptor decreases AT1 receptor\nexpression in the mouse kidney and in immortalized RPT cells from Wistar-Kyoto (WKY) rats. The current studies were\ndesigned to test the hypothesis that activation of the AT1 receptor can also regulate the D1 receptor in RPT cells, and\nthis regulation is aberrant in spontaneously hypertensive rats (SHRs). Long-term (24 hours) stimulation of RPT cells\nwith angiotensin II, via AT1 receptors increased total cellular D1 receptor protein in a time- and concentration-dependent\nmanner in WKY but not in SHR cells. Short-term stimulation (15 minutes) with angiotensin II did not affect total\ncellular D1 receptor protein in either rat strain. However, in the short-term experiments, angiotensin II decreased cell\nsurface membrane D1 receptor protein in WKY but not in SHR cells. D1 and AT1 receptors colocalized (confocal\nmicroscopy) and their coimmunoprecipitation was greater in WKY than in SHRs. However, AT1/D1 receptor\ncoimmunoprecipitation was decreased by angiotensin II (108M/24 hours) to a similar extent in WKY (228%) and\nSHRs (2212%). In summary, these studies show that AT1 and D1 receptors interact differently in RPT cells from\nWKY and SHRs. It is possible that an angiotensin II-mediated increase in D1 receptors and dissociation of AT1 from\nD1 receptors serve to counter regulate the long-term action of angiotensin II in WKY rats; different effects are seen in\nSHRs. (Hypertension. 2005;46:799-805.)\nKey Words: dopamine  kidney  receptors, angiotensin II  rats, spontaneously hypertensive\nA ngiotensin II and dopamine are important regulators of\nsodium and water transport across the renal proximal\ntubule (RPT).1– 8 Angiotensin II receptor subtypes (AT1, AT2,\nand AT4) are expressed in brush border and basolateral\nmembranes of RPTs.1–5 The activation of AT1 receptors by\nlow concentrations of angiotensin II (picomolar) causes an\nincrease in sodium reabsorption in RPTs.1,3,6 All the dopa-\nmine receptor subtypes, D1-like (D1 and D5) and D2-like (D2,\nD3, and D4) are also expressed in brush border and basolateral\nmembranes of RPTs.7–10 In contrast to the stimulatory effect\nof AT1 receptors on renal sodium transport, activation of D1\nand D3 receptors decreases renal sodium reabsorption.7,8,11–13\nSeveral reports have shown an interaction between the\ndopamine and renin-angiotensin system. Intrarenally pro-\nduced angiotensin II opposes the natriuretic action of the D1-\nlike dopamine receptor agonist, fenoldopam, in rats.14 D2-like\nreceptor agonists also antagonize the stimulatory effect of\nangiotensin II, acting via AT1 receptors, on renal proximal\ntubular luminal sodium transport.15,16 When angiotensin II\ngeneration is inhibited or AT1 receptors are blocked, the\nnatriuretic effect of dopaminergic drugs is enhanced.17\nThe AT1 receptor mediates the anti-natriuretic effect of\nangiotensin II, whereas the D1 dopamine receptor is respon-\nsible for 80% of D1-like receptor activity in RPTs.18 We\nhave reported that in RPT cells of Wistar-Kyoto (WKY) rats,\nthe D1-like agonist, fenoldopam, increased D1 receptor but\ndecreased AT1 receptor protein expression. In contrast, in\nRPT cells of spontaneously hypertensive rats (SHRs),\nfenoldopam also decreased AT1 receptor expression but no\nlonger stimulated D1 receptor expression. We also reported\nthat AT1 and D1 receptors coimmunoprecipitate in RPT cells;\nfenoldopam increased this coimmunoprecipitation in WKY\nRPT cells but decreased it in SHR RPT cells.19 We hypoth-\nesize that the AT1 receptor may also regulate the D1 receptor,\nincluding its expression. Therefore, we studied the effect of\nangiotensin II on D1 receptor expression and AT1 and D1\nReceived June 23, 2005; first decision July 11, 2005; revision accepted August 18, 2005.\nFrom the Department of Cardiology (C.Z.), Daping Hospital, Third Military Medical University, Chongqing, P.R. China; Departments of Pediatrics\n(C.Z., Z.W., L.D.A., G.M.E., P.A.J.) and Physiology and Biophysics (P.A.J.), and Internal Medicine (G.M.E.), Georgetown University Medical Center,\nWashington, DC; Department of Physiology and Biophysics (U.H.), Case Western Reserve School of Medicine, Cleveland, Ohio; Department of\nPathology (R.A.F.), University of Virginia Health Sciences Center, Charlottesville, Va.",
         "receptor interaction in immortalized rat RPT cells. We have\npreviously shown that\nthe D1 receptor effects on signal\ntransduction and sodium transporters in immortalized RPT\ncells are similar to those obtained in freshly obtained renal\nproximal tubules.20 –22 Parenti et al have also reported that the\nAT1 receptor behaves similarly in freshly obtained renal\ntubules and immortalized RPT cells.23\nWe now report that AT1 and D1 receptors colocalize in RPT\ncells. D1 and AT1\nreceptor coimmunoprecipitation was\nin agreement with our\ngreater in WKY than in SHRs,\nprevious report.19 In this earlier study, long-term fenoldopam\nstimulation (24 hours) increased the coimmunoprecipitation\nof D1 and AT1 receptors in WKY but decreased it in SHRs.19\nWe now show that angiotensin II decreases D1 and AT1\ncoimmunoprecipitation similarly in WKY and SHRs. How-\never, D1 receptor expression is increased by angiotensin II in\nWKY but not in SHR. In contrast, short-term angiotensin II,\nvia AT1 receptors (15 minutes), decreases cell surface mem-\nbrane expression of D1 receptors in WKY but not in SHRs.\nThese studies show that angiotensin II differentially affects\ntotal cellular and surface membrane D1 receptor expression in\nRPT cells from WKY and SHRs.",
         "",
         "",
         "Methods\nCell Culture\nImmortalized RPT cells from 4- to 8-week-old WKY and SHRs were\ncultured at 37°C in 95% air/5% CO2 atmosphere in DMEM/F-12\nculture media, as previously described.21–26 The cells (80% conflu-\nence) were extracted in ice-cold lysis buffer (phosphate-buffered\nsaline with 1% NP40, 0.5% sodium deoxycholate, 0.1% SDS,\n1 mmol/L EDTA, 1 mmol/L EGTA, 1 mmol/L PMSF, 10 g/mL\naprotinin, and 10 g/mL leupeptin), sonicated, kept on ice for 1 hour,\nand centrifuged at 16 000g for 30 minutes. The supernatants were\nstored at 70°C until use\nimmunoblotting and/or\nimmunoprecipitation.\nImmunoblotting\nThe antibodies are polyclonal purified antipeptides. The amino acid\nsequence for the immunogenic peptide (rabbit anti-human AT1\nreceptor antibody) is QDDCPKAGRHC, amino acids 15 to 24 of the\nAT1 receptor. The specificity of this antibody to the AT1 receptor is\nindicated as previously reported.27,28 The amino acid sequence for rat\nD1\nreceptor antibody corresponds to positions 299 to 307\n(GSEETQPFC) on D1 receptor (Research Genetics). The specificity\nof this D1 receptor antibody has also been reported.19,29 Rat RPT\ncells were treated with vehicle (dH2O), angiotensin II or an AT1\nreceptor antagonist (losartan) at the indicated concentrations and\ntimes. Immunoblotting was performed as previously reported,22,26\nexcept that the transblots were probed with the D1 (1:800) or the AT1\nreceptor antibody (1:400). The amount of protein transferred onto the\nmembranes was determined by Ponceau-S staining and immunoblot-\nting for -actin.19\nantibody followed by the secondary goat anti-mouse antibody\nlabeled with Alexa 568 (Molecular Probes).22 Cells on coverslips\nwere mounted with the ProLong Antifade Kit (Molecular Probes).\nThe immunofluorescence densities and images were acquired\n(Olympus AX70) at an excitation wavelength of 488 nm and 568 nm;\nemission was detected at 535 and 645 nm.22,30\nImmunoprecipitation\nRPT cells were incubated with vehicle or angiotensin II (108 M) for\n24 hours, as described. The cells were lysed with ice-cold lysis buffer\nfor 1 hour and centrifuged at 16 000g for 30 minutes. Equal amounts\nof lysates (500 g protein/mL supernatant for RPT cells from both\nSHR and WKY rats) were incubated with affinity-purified anti-D1\nreceptor antibody (2 L/mL) for 1 hour and protein-G agarose at 4°C\nfor 12 hours. The immunoprecipitates were pelleted and washed 4\ntimes with lysis buffer. The pellets were suspended in sample buffer,\nboiled for 10 minutes, and subjected to immunoblotting with the AT1\nreceptor antibody. To determine the specificity of the bands, pre-\nimmune serum of D1 receptor antibody (negative control) and AT1\nreceptor antibody (positive control) were used as immunoprecipi-\ntants, instead of the D1 receptor antibody. The densities of the bands\nwere quantified by densitometry using Quantiscan (Ferguson, Mo) as\npreviously reported.19,22,26\nCell Surface D1 Receptor Expression\nCultured RPT cells were starved in serum-free medium for 2 hours,\nand then treated with angiotensin II (108 M) for 15 minutes. Surface\nmembrane proteins were biotinylated by adding sulfo-NHS-LC-\nbiotin (final concentration 250 g/mL) into the medium 5 minutes\nbefore adding angiotensin II. The cells were washed 3 times with\nwash buffer. Then, the cells were lysed with lysis buffer, sonicated\nand placed on ice for 1 hour. The supernatant from the cell lysate was\nimmunoprecipitated with the anti-rat D1 receptor antibody, followed\nby immunoblotting. The membrane sheets were blocked with 5%\nmilk overnight and after washing (3 times), the sheets were incu-\nbated with peroxidase-conjugated streptavidin (Jackson ImmunoRe-\nsearch Laboratory, Inc, in wash buffer at 1/5000 dilution for 30\nminutes. The biotinylated protein bands were visualized by enhanced\nchemiluminescence (Western Blotting Detection Kit; Amersham,\nArlington Heights, Ill).31\nMaterials\nRabbit anti-human AT1 receptor antibodies were purchased from\nSanta Cruz Biotechnology, Inc (Santa Cruz, Calif). Rabbit anti-rat D1\nreceptor antibody was produced against a synthetic oligopeptide\nfrom the amino acid sequence of rat D1 receptor (amino acids 299 to\n307) (Research Genetics). Monoclonal mouse anti-human AT1 re-\nceptor was purchased from Abcam Limited (Cambridge, UK).\nAngiotensin II was purchased from Peninsula Laboratory, Inc (St\nLouis, Mo). Losartan was a gift from Merck & Co (Philadelphia, Pa).\nPeroxidase-conjugated streptavidin was purchased from Jackson\nImmunoResearch Laboratory, Inc (West Grove, Pa). Sulfo-NHS-LC-\nbiotin was purchased from PIERCE (Rockford, Ill). Other chemicals\nfor various buffers were of the highest purity available and purchased\neither from Sigma or Gibco.\nStatistical Analysis\nThe data are expressed as meanSEM. Comparison within groups\nwas made by ANOVA for repeated measures, and comparison\namong groups was made by factorial ANOVA with Duncan’s test.\nP0.05 was considered significant.\nZeng et al\nAT1 Receptor Regulation of D1 Receptors in Kidney\nM to 105 M) for 24 hours. Consistent with the results in\nFigure 1C, angiotensin II had no effect on D1 receptor\nexpression in SHRs (Figure 1E).\nmanner. The stimulatory effect was evident at 109 M (Figure\n1A) as early as 16 hours and maintained for at least 30 hours\n(Figure 1B). Consistent with the stimulatory effect of angio-\ntensin II on D1 receptor expression (Figure 1A and 1B), a 24\nhour-incubation with angiotensin II (108 M) increased D1\nreceptors in RPT cells from WKY rats. In contrast, this effect\nwas not observed in RPT cells from SHRs (WKY: con-\ntrol0.770.15, angiotensin II1.330.19; SHR: con-\ntrol0.870.13, angiotensin II1.040.17; n7) (Figure 1C).\nTo determine the specificity of angiotensin II action on\nAT1 receptors, we studied the effect of the AT1 receptor\nantagonist, losartan. Consistent with the results in WKY rats\nin Figure 1A, 1B, and 1C, angiotensin II (108 M/24 hours)\nincreased D1 receptor expression (control0.830.15, angio-\ntensin II1.480.15; n8). The AT1 receptor antagonist, losar-\ntan (108 M), by itself, had no effect on D1 receptor expression\n(0.80.13) but reversed the stimulatory effect of angiotensin II\non D1 receptor expression (0.70.08) (Figure 1D).\nthere is a right-shift of\nHypertension\nOctober 2005\nActivation of AT1 Receptors Decreases the D1\nReceptor Expression in Surface Membranes of\nRPT Cells From WKY Rats But Not From SHRs\nThe effect of short-term stimulation with angiotensin II on\ncell surface D1 receptors is shown in Figure 4. Angiotensin II\n(108 M) decreased the quantity of D1 receptors on the cell\nsurface membranes in WKY RPT cells, but not in SHR RPT\ncells (WKY: control459 DU, angiotensin II175 DU;\nSHR: control193 DU, angiotensin II195 DU, n6).\nin renal proximal\nDiscussion\nPolygenic essential hypertension is associated with increased\ntubules.32,33 As men-\nsodium transport\ntioned, dopamine and angiotensin II are important regulators\nof sodium and water reabsorption in the kidney, serving\ncontrasting functions in the proximal tubule.1,5,17,34,35 Both\ndopamine and renin-angiotensin systems exist in the RPT,\nand the components of the renin-angiotensin system, includ-\ning angiotensinogen mRNA,\nrenin, and angiotensin-\nconverting enzyme, have been localized to the proximal\ntubule.1,6,36,37 The proximal tubule is also the site of local\ndopamine production.38 – 40 Urinary dopamine and angiotensin\nII concentrations exceed circulating levels, suggesting both\nsystems may have autocrine and/or paracrine effects in\nRPTs.1,6,38,41 Angiotensin II stimulates sodium reabsorption\nvia activation of sodium transporting proteins that are inhib-\nited by dopamine (NHE3, Cl/HCO3 exchanger, Na/HCO3\ncotransporter, and Na/K ATPase).34,35 Dopamine and D1-\nlike and D2-like receptor agonists have been reported to\nantagonize the actions of angiotensin II on NHE3 and\nNaK-ATPase activities in brush border and basolateral\nmembranes of renal proximal tubules.15,16 Dopamine, via\nD1-like receptors, has also been reported to decrease the AT1\nreceptor mRNA and protein, as well as angiotensin II binding\nsites in renal proximal tubules.19,42 The effect of angiotensin\nII on D1-like receptors has not been reported.\nIn WKY RPT cells, stimulation of the AT1 receptor with\nangiotensin II for 2 different periods (short-term or long-\nterm) yields opposite results; short-term stimulation de-\ncreases cell surface D1 receptor expression, whereas long-\nterm stimulation increases whole cell D1 receptor expression.\nIt is unlikely that the change in the short-term could account\nfor the long-term result. A decrease of cell surface D1 receptor\nexpression would remove a restraint allowing AT1 receptor\nactivation to increase sodium reabsorption to a greater extent.\nHowever, the increase in D1 receptor expression would tend\nto counterbalance the prolonged angiotensin II effect.\nCell surface membrane localization of G protein-coupled\nreceptors is decreased by phosphorylation of the receptor.8,43\nWe have reported that G protein-coupled receptor kinases\n(GRKs), GRK2 and GRK4, are involved in D1 receptor\nphosphorylation.44 Among the GRKs, GRK2 is most impor-\nZeng et al\nAT1 Receptor Regulation of D1 Receptors in Kidney\nis\nin promoting AT1 receptor phosphorylation.45 It\ntant\npossible that the GRK2 activation subsequent to AT1 receptor\nstimulation also results in the alteration of the phosphoryla-\ntion and membrane localization of the D1 receptor. Such a\nscenario can explain the ability of AT1 receptors to decrease\nthe membrane localization of the D1 receptor in WKY. This\ndoes not occur in SHRs because the D1 receptor is already\nhyperphosphorylated and fewer D1 receptors are found on\nrenal cell surface membranes in SHR.46 Although GRK4 is\nthe major GRK regulating D1 receptor function,43,44 GRK4 is\nnot involved in the desensitization of the AT1 receptor.47 The\npossibility exists that the AT1 receptor can cause heterologous\ndesensitization of the D1 receptor via stimulation of effector\nenzymes, but it is not clear which one.48\nThe dephosphorylation of D1 receptors in renal proximal\ntubules is also different in WKY and SHRs. We have reported\nthat short-term D1-like receptor stimulation increases protein\nphosphatase 2A activity in renal proximal tubules in WKY\nbut not in SHRs49. It is possible that in the WKY, the AT1\nreceptor inhibits protein phosphatase 2A activity and thus\nprevents dephosphorylation and recycling of the D1 receptor\nto the cell surface membrane. Although there is no report to\nsupport this mechanism, AT2 receptor has been reported to\nstimulate protein phosphatase 2A activity.50,51\nLong-term stimulation of AT1 receptors increases total\ncellular D1 receptor expression in WKY. This finding in vitro\nwas also found in vivo; chronic angiotensin II infusion in\nmice also caused an increase in D1 receptor expression\n(unpublished data). Presumably, cell surface D1 receptor\nexpression is also increased. Although the mechanism by\nwhich AT1 receptor stimulation increases D1 receptor expres-\nsion remains to be determined, such an effect could serve as\na physiological negative feedback under normal circum-\nstances (in WKY). However, in SHR cells, this upregulation\nof D1 receptors by stimulation of AT1 receptors is lost because\nof decreased responsiveness of the hyperphosphorylated D1\nreceptor.46,52,53 We have reported that the renal D1 receptor is\nhyperphosphorylated in SHRs as a consequence of increased\nactivity of GRK454 and decreased protein phosphatase 2A\nactivity.49\nLong-term incubation with angiotensin II decreases D1/AT1\nreceptor coimmunoprecipitation to the same degree in cells\nfrom SHR and WKY rats. Our previous study showed that\nlong-term exposure to angiotensin II decreases AT1 receptor\nprotein expression in RPT cells from WKY rats but increases\nit in SHRs.26 The decreased interaction between D1 and AT1\nreceptors in renal proximal tubules in WKY rats is, therefore,\nnot the result of decreased AT1 receptor expression, per se,\nbecause a similar response is seen in SHRs where AT1\nexpression is increased by angiotensin II. Thus, angiotensin\nII, via the AT1 receptor, probably causes a decrease in the\nphysical interaction of AT1 and D1 receptors in both WKY\nand SHR cells. Further studies are needed to determine\nwhether the decreased interaction between these 2 receptors\nis direct or indirect, possibly by the alteration of an adaptor\ngene or adaptor proteins.\nfunction in hypertension is not caused by abnormalities in G\nproteins, effectors, or ion transporting proteins, such as\nadenylyl cyclase, NHE3 or Na-KATPase. Rather, the renal\nD1-like receptor is uncoupled from G protein subunits,\nleading to decreased D1-like receptor interaction with G\nprotein subunits, and resulting in decreased production of\nsecond messengers and decreased interaction between G\nprotein subunits, effector enzymes, and ion transporters.8,43,53\nTo determine whether the impaired D1 receptor function\noccurs at the receptor level, we investigated the expression of\nD1 receptor expression in cell surface membranes and whole\ncells. We found that cell surface membrane expression of D1\nreceptors is lower and D1 receptor phosphorylation is higher\nin SHR cells than in WKY cells, although total and D1\nreceptor expression are not different in the 2 cell lines.46,54\nRenal D1 receptor phosphorylation and GRK4 activity impact\non the regulation of blood pressure. Inhibition of renal GRK4\nactivity by antisense oligonucleotides attenuates the increase\nin blood pressure with age in SHRs without affecting blood\npressure in WKY rats.55\nIn summary, we have demonstrated that long-term stimu-\nlation of AT1 receptors positively regulates the expression of\nD1 receptors in RPT cells from WKY but not from SHRs. AT1\nand D1 receptors colocalize in RPT cells from both rat strains,\nbut the basal level of cell surface membrane D1 receptor\nexpression is higher in WKY than in SHR RPT cells. It\nappears that a greater proportion of D1 receptors are located\non the cell surface membrane in WKY cells than in SHR\ncells. Short-term AT1 receptor stimulation decreases cell\nsurface membrane D1 receptor expression in WKY rats but\nnot in SHRs. It is possible that differences in AT1 receptor\nregulation of D1 receptor expression and cell surface expres-\nsion may participate in the abnormal regulation of renal\nproximal sodium transport in genetic hypertension.\nPerspectives\nThe dopaminergic and renin-angiotensin system are 2 impor-\ntant systems that regulate blood pressure.1– 8 Dopamine pro-\nmotes natriuresis, whereas angiotensin II decreases sodium\nexcretion.1– 8 The major D1-like receptor subtype mediating\nthe increase in sodium excretion is probably the D1 receptor,\nwhereas the major angiotensin II receptor mediating the\ndecrease in renal sodium excretion is the AT1 receptor.1– 8 In\nSHRs, renal proximal tubular D1 receptor function is im-\npaired.7,8 Regardless of the mechanism involved in the\ninteraction between D1 and AT1 receptors, the dissociation of\nD1 from AT1 receptors after angiotensin II stimulation may\nallow the AT1 and D1 receptor to exert\ntheir functions\nseparately. Renal proximal tubular D1 receptors are functional\nin WKY but not in SHRs. In contrast, AT1 receptor function\nis enhanced in SHRs.32 The inability of AT1 receptors to\nincrease D1 receptor expression in renal proximal tubules in\nSHRs and a decreased D1 and AT1 receptor interaction could\nlead to enhanced AT1 receptor function.\nHypertension\nOctober 2005\nReferences\n1. Navar LG, Lewis L, Hymel A, Braam B, Mitchell KD. Tubular fluid\nconcentrations and kidney contents of angiotensins I and II in anesthe-\ntized rats. J Am Soc Nephrol. 1994;5:1153–1158.\n5. Handa RK, Krebs LT, Harding JW, Handa SE. Angiotensin IV AT4-\nreceptor system in the rat kidney. Am J Physiol. 1998;274:F290 –F299.\n6. Navar LG, Harrison-Bernard LM, Imig JD, Wang CT, Cervenka L,\nMitchell KD. Intrarenal angiotensin II generation and renal effects of AT1\nreceptor blockade. J Am Soc Nephrol. 1999;10(Suppl 12):S266 –S272.\n7. Hussain T, Lokhandwala MF. Renal dopamine receptor function in\n10. Ricci A, Marchal-Victorion S, Bronzetti E, Parini A, Amenta F, Tayebati\nSK. Dopamine D4 receptor expression in rat kidney: evidence for pre- and\npostjunctional localization. J Histochem Cytochem. 2002;50:1091–1096.\n11. Luippold G, Zimmermann C, Mai M, Kloor D, Starck D, Gross G,\nMuhlbauer B. Dopamine D3 receptors and salt-dependent hypertension.\nJ Am Soc Nephrol. 2001;12:2272–2279.\n14. Chen CJ, Apparsundaram S, Lokhandwala MF. Intrarenally produced\nangiotensin II opposes the natriuretic action of the dopamine-1 receptor\nagonist fenoldopam in rats. J Pharmacol Exp Ther. 1991;256:486 – 491.\n15. Sheikh-Hamad D, Wang YP, Jo OD, Yanagawa N. Dopamine antag-\nonizes the actions of angiotensin II in renal brush-border membrane. Am J\nPhysiol. 1993;264:F737–F743.\nZeng et al\nAT1 Receptor Regulation of D1 Receptors in Kidney\n50. Huang XC, Sumners C, Richards EM. Angiotensin II stimulates protein\nphosphatase 2A activity in cultured neuronal cells via type 2 receptors in\na pertussis toxin sensitive fashion. Adv Exp Med Biol. 1996;396:209 –215.\n51. Gelband CH, Sumners C, Lu D, Raizada MK. Angiotensin receptors and\nnorepinephrine neuromodulation: implications of functional coupling.\nRegul Pept. 1998;73:141–147.\n53. Felder RA, Sanada H, Xu J, Yu PY, Wang Z, Watanabe H, Asico LD,\nWang W, Zheng S, Yamaguchi I, Williams SM, Gainer J, Brown NJ,\nHazen-Martin D, Wong LJ, Robillard JE, Carey RM, Eisner GM, Jose\nPA. G protein-coupled receptor kinase 4 gene variants in human essential\nhypertension. Proc Natl Acad Sci U S A. 2002;99:3872–3877.",
         "Abstract—The dopaminergic and renin-angiotensin systems regulate blood pressure, in part, by affecting sodium transport\nin renal proximal tubules (RPTs). We have reported that activation of a D1-like receptor decreases AT1 receptor\nexpression in the mouse kidney and in immortalized RPT cells from Wistar-Kyoto (WKY) rats. The current studies were\ndesigned to test the hypothesis that activation of the AT1 receptor can also regulate the D1 receptor in RPT cells, and\nthis regulation is aberrant in spontaneously hypertensive rats (SHRs). Long-term (24 hours) stimulation of RPT cells\nwith angiotensin II, via AT1 receptors increased total cellular D1 receptor protein in a time- and concentration-dependent\nmanner in WKY but not in SHR cells. Short-term stimulation (15 minutes) with angiotensin II did not affect total\ncellular D1 receptor protein in either rat strain. However, in the short-term experiments, angiotensin II decreased cell\nsurface membrane D1 receptor protein in WKY but not in SHR cells. D1 and AT1 receptors colocalized (confocal\nmicroscopy) and their coimmunoprecipitation was greater in WKY than in SHRs. However, AT1/D1 receptor\ncoimmunoprecipitation was decreased by angiotensin II (108M/24 hours) to a similar extent in WKY (228%) and\nSHRs (2212%). In summary, these studies show that AT1 and D1 receptors interact differently in RPT cells from\nWKY and SHRs. It is possible that an angiotensin II-mediated increase in D1 receptors and dissociation of AT1 from\nD1 receptors serve to counter regulate the long-term action of angiotensin II in WKY rats; different effects are seen in\nSHRs. (Hypertension. 2005;46:799-805.)\nKey Words: dopamine  kidney  receptors, angiotensin II  rats, spontaneously hypertensive\nA ngiotensin II and dopamine are important regulators of\nsodium and water transport across the renal proximal\ntubule (RPT).1– 8 Angiotensin II receptor subtypes (AT1, AT2,\nand AT4) are expressed in brush border and basolateral\nmembranes of RPTs.1–5 The activation of AT1 receptors by\nlow concentrations of angiotensin II (picomolar) causes an\nincrease in sodium reabsorption in RPTs.1,3,6 All the dopa-\nmine receptor subtypes, D1-like (D1 and D5) and D2-like (D2,\nD3, and D4) are also expressed in brush border and basolateral\nmembranes of RPTs.7–10 In contrast to the stimulatory effect\nof AT1 receptors on renal sodium transport, activation of D1\nand D3 receptors decreases renal sodium reabsorption.7,8,11–13\nSeveral reports have shown an interaction between the\ndopamine and renin-angiotensin system. Intrarenally pro-\nduced angiotensin II opposes the natriuretic action of the D1-\nlike dopamine receptor agonist, fenoldopam, in rats.14 D2-like\nreceptor agonists also antagonize the stimulatory effect of\nangiotensin II, acting via AT1 receptors, on renal proximal\ntubular luminal sodium transport.15,16 When angiotensin II\ngeneration is inhibited or AT1 receptors are blocked, the\nnatriuretic effect of dopaminergic drugs is enhanced.17\nThe AT1 receptor mediates the anti-natriuretic effect of\nangiotensin II, whereas the D1 dopamine receptor is respon-\nsible for 80% of D1-like receptor activity in RPTs.18 We\nhave reported that in RPT cells of Wistar-Kyoto (WKY) rats,\nthe D1-like agonist, fenoldopam, increased D1 receptor but\ndecreased AT1 receptor protein expression. In contrast, in\nRPT cells of spontaneously hypertensive rats (SHRs),\nfenoldopam also decreased AT1 receptor expression but no\nlonger stimulated D1 receptor expression. We also reported\nthat AT1 and D1 receptors coimmunoprecipitate in RPT cells;\nfenoldopam increased this coimmunoprecipitation in WKY\nRPT cells but decreased it in SHR RPT cells.19 We hypoth-\nesize that the AT1 receptor may also regulate the D1 receptor,\nincluding its expression. Therefore, we studied the effect of\nangiotensin II on D1 receptor expression and AT1 and D1\nReceived June 23, 2005; first decision July 11, 2005; revision accepted August 18, 2005.\nFrom the Department of Cardiology (C.Z.), Daping Hospital, Third Military Medical University, Chongqing, P.R. China; Departments of Pediatrics\n(C.Z., Z.W., L.D.A., G.M.E., P.A.J.) and Physiology and Biophysics (P.A.J.), and Internal Medicine (G.M.E.), Georgetown University Medical Center,\nWashington, DC; Department of Physiology and Biophysics (U.H.), Case Western Reserve School of Medicine, Cleveland, Ohio; Department of\nPathology (R.A.F.), University of Virginia Health Sciences Center, Charlottesville, Va.\nreceptor interaction in immortalized rat RPT cells. We have\npreviously shown that\nthe D1 receptor effects on signal\ntransduction and sodium transporters in immortalized RPT\ncells are similar to those obtained in freshly obtained renal\nproximal tubules.20 –22 Parenti et al have also reported that the\nAT1 receptor behaves similarly in freshly obtained renal\ntubules and immortalized RPT cells.23\nWe now report that AT1 and D1 receptors colocalize in RPT\ncells. D1 and AT1\nreceptor coimmunoprecipitation was\nin agreement with our\ngreater in WKY than in SHRs,\nprevious report.19 In this earlier study, long-term fenoldopam\nstimulation (24 hours) increased the coimmunoprecipitation\nof D1 and AT1 receptors in WKY but decreased it in SHRs.19\nWe now show that angiotensin II decreases D1 and AT1\ncoimmunoprecipitation similarly in WKY and SHRs. How-\never, D1 receptor expression is increased by angiotensin II in\nWKY but not in SHR. In contrast, short-term angiotensin II,\nvia AT1 receptors (15 minutes), decreases cell surface mem-\nbrane expression of D1 receptors in WKY but not in SHRs.\nThese studies show that angiotensin II differentially affects\ntotal cellular and surface membrane D1 receptor expression in\nRPT cells from WKY and SHRs.\nMethods\nCell Culture\nImmortalized RPT cells from 4- to 8-week-old WKY and SHRs were\ncultured at 37°C in 95% air/5% CO2 atmosphere in DMEM/F-12\nculture media, as previously described.21–26 The cells (80% conflu-\nence) were extracted in ice-cold lysis buffer (phosphate-buffered\nsaline with 1% NP40, 0.5% sodium deoxycholate, 0.1% SDS,\n1 mmol/L EDTA, 1 mmol/L EGTA, 1 mmol/L PMSF, 10 g/mL\naprotinin, and 10 g/mL leupeptin), sonicated, kept on ice for 1 hour,\nand centrifuged at 16 000g for 30 minutes. The supernatants were\nstored at 70°C until use\nimmunoblotting and/or\nimmunoprecipitation.\nImmunoblotting\nThe antibodies are polyclonal purified antipeptides. The amino acid\nsequence for the immunogenic peptide (rabbit anti-human AT1\nreceptor antibody) is QDDCPKAGRHC, amino acids 15 to 24 of the\nAT1 receptor. The specificity of this antibody to the AT1 receptor is\nindicated as previously reported.27,28 The amino acid sequence for rat\nD1\nreceptor antibody corresponds to positions 299 to 307\n(GSEETQPFC) on D1 receptor (Research Genetics). The specificity\nof this D1 receptor antibody has also been reported.19,29 Rat RPT\ncells were treated with vehicle (dH2O), angiotensin II or an AT1\nreceptor antagonist (losartan) at the indicated concentrations and\ntimes. Immunoblotting was performed as previously reported,22,26\nexcept that the transblots were probed with the D1 (1:800) or the AT1\nreceptor antibody (1:400). The amount of protein transferred onto the\nmembranes was determined by Ponceau-S staining and immunoblot-\nting for -actin.19\nantibody followed by the secondary goat anti-mouse antibody\nlabeled with Alexa 568 (Molecular Probes).22 Cells on coverslips\nwere mounted with the ProLong Antifade Kit (Molecular Probes).\nThe immunofluorescence densities and images were acquired\n(Olympus AX70) at an excitation wavelength of 488 nm and 568 nm;\nemission was detected at 535 and 645 nm.22,30\nImmunoprecipitation\nRPT cells were incubated with vehicle or angiotensin II (108 M) for\n24 hours, as described. The cells were lysed with ice-cold lysis buffer\nfor 1 hour and centrifuged at 16 000g for 30 minutes. Equal amounts\nof lysates (500 g protein/mL supernatant for RPT cells from both\nSHR and WKY rats) were incubated with affinity-purified anti-D1\nreceptor antibody (2 L/mL) for 1 hour and protein-G agarose at 4°C\nfor 12 hours. The immunoprecipitates were pelleted and washed 4\ntimes with lysis buffer. The pellets were suspended in sample buffer,\nboiled for 10 minutes, and subjected to immunoblotting with the AT1\nreceptor antibody. To determine the specificity of the bands, pre-\nimmune serum of D1 receptor antibody (negative control) and AT1\nreceptor antibody (positive control) were used as immunoprecipi-\ntants, instead of the D1 receptor antibody. The densities of the bands\nwere quantified by densitometry using Quantiscan (Ferguson, Mo) as\npreviously reported.19,22,26\nCell Surface D1 Receptor Expression\nCultured RPT cells were starved in serum-free medium for 2 hours,\nand then treated with angiotensin II (108 M) for 15 minutes. Surface\nmembrane proteins were biotinylated by adding sulfo-NHS-LC-\nbiotin (final concentration 250 g/mL) into the medium 5 minutes\nbefore adding angiotensin II. The cells were washed 3 times with\nwash buffer. Then, the cells were lysed with lysis buffer, sonicated\nand placed on ice for 1 hour. The supernatant from the cell lysate was\nimmunoprecipitated with the anti-rat D1 receptor antibody, followed\nby immunoblotting. The membrane sheets were blocked with 5%\nmilk overnight and after washing (3 times), the sheets were incu-\nbated with peroxidase-conjugated streptavidin (Jackson ImmunoRe-\nsearch Laboratory, Inc, in wash buffer at 1/5000 dilution for 30\nminutes. The biotinylated protein bands were visualized by enhanced\nchemiluminescence (Western Blotting Detection Kit; Amersham,\nArlington Heights, Ill).31\nMaterials\nRabbit anti-human AT1 receptor antibodies were purchased from\nSanta Cruz Biotechnology, Inc (Santa Cruz, Calif). Rabbit anti-rat D1\nreceptor antibody was produced against a synthetic oligopeptide\nfrom the amino acid sequence of rat D1 receptor (amino acids 299 to\n307) (Research Genetics). Monoclonal mouse anti-human AT1 re-\nceptor was purchased from Abcam Limited (Cambridge, UK).\nAngiotensin II was purchased from Peninsula Laboratory, Inc (St\nLouis, Mo). Losartan was a gift from Merck & Co (Philadelphia, Pa).\nPeroxidase-conjugated streptavidin was purchased from Jackson\nImmunoResearch Laboratory, Inc (West Grove, Pa). Sulfo-NHS-LC-\nbiotin was purchased from PIERCE (Rockford, Ill). Other chemicals\nfor various buffers were of the highest purity available and purchased\neither from Sigma or Gibco.\nStatistical Analysis\nThe data are expressed as meanSEM. Comparison within groups\nwas made by ANOVA for repeated measures, and comparison\namong groups was made by factorial ANOVA with Duncan’s test.\nP0.05 was considered significant.\nZeng et al\nAT1 Receptor Regulation of D1 Receptors in Kidney\nM to 105 M) for 24 hours. Consistent with the results in\nFigure 1C, angiotensin II had no effect on D1 receptor\nexpression in SHRs (Figure 1E).\nmanner. The stimulatory effect was evident at 109 M (Figure\n1A) as early as 16 hours and maintained for at least 30 hours\n(Figure 1B). Consistent with the stimulatory effect of angio-\ntensin II on D1 receptor expression (Figure 1A and 1B), a 24\nhour-incubation with angiotensin II (108 M) increased D1\nreceptors in RPT cells from WKY rats. In contrast, this effect\nwas not observed in RPT cells from SHRs (WKY: con-\ntrol0.770.15, angiotensin II1.330.19; SHR: con-\ntrol0.870.13, angiotensin II1.040.17; n7) (Figure 1C).\nTo determine the specificity of angiotensin II action on\nAT1 receptors, we studied the effect of the AT1 receptor\nantagonist, losartan. Consistent with the results in WKY rats\nin Figure 1A, 1B, and 1C, angiotensin II (108 M/24 hours)\nincreased D1 receptor expression (control0.830.15, angio-\ntensin II1.480.15; n8). The AT1 receptor antagonist, losar-\ntan (108 M), by itself, had no effect on D1 receptor expression\n(0.80.13) but reversed the stimulatory effect of angiotensin II\non D1 receptor expression (0.70.08) (Figure 1D).\nthere is a right-shift of\nHypertension\nOctober 2005\nActivation of AT1 Receptors Decreases the D1\nReceptor Expression in Surface Membranes of\nRPT Cells From WKY Rats But Not From SHRs\nThe effect of short-term stimulation with angiotensin II on\ncell surface D1 receptors is shown in Figure 4. Angiotensin II\n(108 M) decreased the quantity of D1 receptors on the cell\nsurface membranes in WKY RPT cells, but not in SHR RPT\ncells (WKY: control459 DU, angiotensin II175 DU;\nSHR: control193 DU, angiotensin II195 DU, n6).\nin renal proximal\nDiscussion\nPolygenic essential hypertension is associated with increased\ntubules.32,33 As men-\nsodium transport\ntioned, dopamine and angiotensin II are important regulators\nof sodium and water reabsorption in the kidney, serving\ncontrasting functions in the proximal tubule.1,5,17,34,35 Both\ndopamine and renin-angiotensin systems exist in the RPT,\nand the components of the renin-angiotensin system, includ-\ning angiotensinogen mRNA,\nrenin, and angiotensin-\nconverting enzyme, have been localized to the proximal\ntubule.1,6,36,37 The proximal tubule is also the site of local\ndopamine production.38 – 40 Urinary dopamine and angiotensin\nII concentrations exceed circulating levels, suggesting both\nsystems may have autocrine and/or paracrine effects in\nRPTs.1,6,38,41 Angiotensin II stimulates sodium reabsorption\nvia activation of sodium transporting proteins that are inhib-\nited by dopamine (NHE3, Cl/HCO3 exchanger, Na/HCO3\ncotransporter, and Na/K ATPase).34,35 Dopamine and D1-\nlike and D2-like receptor agonists have been reported to\nantagonize the actions of angiotensin II on NHE3 and\nNaK-ATPase activities in brush border and basolateral\nmembranes of renal proximal tubules.15,16 Dopamine, via\nD1-like receptors, has also been reported to decrease the AT1\nreceptor mRNA and protein, as well as angiotensin II binding\nsites in renal proximal tubules.19,42 The effect of angiotensin\nII on D1-like receptors has not been reported.\nIn WKY RPT cells, stimulation of the AT1 receptor with\nangiotensin II for 2 different periods (short-term or long-\nterm) yields opposite results; short-term stimulation de-\ncreases cell surface D1 receptor expression, whereas long-\nterm stimulation increases whole cell D1 receptor expression.\nIt is unlikely that the change in the short-term could account\nfor the long-term result. A decrease of cell surface D1 receptor\nexpression would remove a restraint allowing AT1 receptor\nactivation to increase sodium reabsorption to a greater extent.\nHowever, the increase in D1 receptor expression would tend\nto counterbalance the prolonged angiotensin II effect.\nCell surface membrane localization of G protein-coupled\nreceptors is decreased by phosphorylation of the receptor.8,43\nWe have reported that G protein-coupled receptor kinases\n(GRKs), GRK2 and GRK4, are involved in D1 receptor\nphosphorylation.44 Among the GRKs, GRK2 is most impor-\nZeng et al\nAT1 Receptor Regulation of D1 Receptors in Kidney\nis\nin promoting AT1 receptor phosphorylation.45 It\ntant\npossible that the GRK2 activation subsequent to AT1 receptor\nstimulation also results in the alteration of the phosphoryla-\ntion and membrane localization of the D1 receptor. Such a\nscenario can explain the ability of AT1 receptors to decrease\nthe membrane localization of the D1 receptor in WKY. This\ndoes not occur in SHRs because the D1 receptor is already\nhyperphosphorylated and fewer D1 receptors are found on\nrenal cell surface membranes in SHR.46 Although GRK4 is\nthe major GRK regulating D1 receptor function,43,44 GRK4 is\nnot involved in the desensitization of the AT1 receptor.47 The\npossibility exists that the AT1 receptor can cause heterologous\ndesensitization of the D1 receptor via stimulation of effector\nenzymes, but it is not clear which one.48\nThe dephosphorylation of D1 receptors in renal proximal\ntubules is also different in WKY and SHRs. We have reported\nthat short-term D1-like receptor stimulation increases protein\nphosphatase 2A activity in renal proximal tubules in WKY\nbut not in SHRs49. It is possible that in the WKY, the AT1\nreceptor inhibits protein phosphatase 2A activity and thus\nprevents dephosphorylation and recycling of the D1 receptor\nto the cell surface membrane. Although there is no report to\nsupport this mechanism, AT2 receptor has been reported to\nstimulate protein phosphatase 2A activity.50,51\nLong-term stimulation of AT1 receptors increases total\ncellular D1 receptor expression in WKY. This finding in vitro\nwas also found in vivo; chronic angiotensin II infusion in\nmice also caused an increase in D1 receptor expression\n(unpublished data). Presumably, cell surface D1 receptor\nexpression is also increased. Although the mechanism by\nwhich AT1 receptor stimulation increases D1 receptor expres-\nsion remains to be determined, such an effect could serve as\na physiological negative feedback under normal circum-\nstances (in WKY). However, in SHR cells, this upregulation\nof D1 receptors by stimulation of AT1 receptors is lost because\nof decreased responsiveness of the hyperphosphorylated D1\nreceptor.46,52,53 We have reported that the renal D1 receptor is\nhyperphosphorylated in SHRs as a consequence of increased\nactivity of GRK454 and decreased protein phosphatase 2A\nactivity.49\nLong-term incubation with angiotensin II decreases D1/AT1\nreceptor coimmunoprecipitation to the same degree in cells\nfrom SHR and WKY rats. Our previous study showed that\nlong-term exposure to angiotensin II decreases AT1 receptor\nprotein expression in RPT cells from WKY rats but increases\nit in SHRs.26 The decreased interaction between D1 and AT1\nreceptors in renal proximal tubules in WKY rats is, therefore,\nnot the result of decreased AT1 receptor expression, per se,\nbecause a similar response is seen in SHRs where AT1\nexpression is increased by angiotensin II. Thus, angiotensin\nII, via the AT1 receptor, probably causes a decrease in the\nphysical interaction of AT1 and D1 receptors in both WKY\nand SHR cells. Further studies are needed to determine\nwhether the decreased interaction between these 2 receptors\nis direct or indirect, possibly by the alteration of an adaptor\ngene or adaptor proteins.\nfunction in hypertension is not caused by abnormalities in G\nproteins, effectors, or ion transporting proteins, such as\nadenylyl cyclase, NHE3 or Na-KATPase. Rather, the renal\nD1-like receptor is uncoupled from G protein subunits,\nleading to decreased D1-like receptor interaction with G\nprotein subunits, and resulting in decreased production of\nsecond messengers and decreased interaction between G\nprotein subunits, effector enzymes, and ion transporters.8,43,53\nTo determine whether the impaired D1 receptor function\noccurs at the receptor level, we investigated the expression of\nD1 receptor expression in cell surface membranes and whole\ncells. We found that cell surface membrane expression of D1\nreceptors is lower and D1 receptor phosphorylation is higher\nin SHR cells than in WKY cells, although total and D1\nreceptor expression are not different in the 2 cell lines.46,54\nRenal D1 receptor phosphorylation and GRK4 activity impact\non the regulation of blood pressure. Inhibition of renal GRK4\nactivity by antisense oligonucleotides attenuates the increase\nin blood pressure with age in SHRs without affecting blood\npressure in WKY rats.55\nIn summary, we have demonstrated that long-term stimu-\nlation of AT1 receptors positively regulates the expression of\nD1 receptors in RPT cells from WKY but not from SHRs. AT1\nand D1 receptors colocalize in RPT cells from both rat strains,\nbut the basal level of cell surface membrane D1 receptor\nexpression is higher in WKY than in SHR RPT cells. It\nappears that a greater proportion of D1 receptors are located\non the cell surface membrane in WKY cells than in SHR\ncells. Short-term AT1 receptor stimulation decreases cell\nsurface membrane D1 receptor expression in WKY rats but\nnot in SHRs. It is possible that differences in AT1 receptor\nregulation of D1 receptor expression and cell surface expres-\nsion may participate in the abnormal regulation of renal\nproximal sodium transport in genetic hypertension.\nPerspectives\nThe dopaminergic and renin-angiotensin system are 2 impor-\ntant systems that regulate blood pressure.1– 8 Dopamine pro-\nmotes natriuresis, whereas angiotensin II decreases sodium\nexcretion.1– 8 The major D1-like receptor subtype mediating\nthe increase in sodium excretion is probably the D1 receptor,\nwhereas the major angiotensin II receptor mediating the\ndecrease in renal sodium excretion is the AT1 receptor.1– 8 In\nSHRs, renal proximal tubular D1 receptor function is im-\npaired.7,8 Regardless of the mechanism involved in the\ninteraction between D1 and AT1 receptors, the dissociation of\nD1 from AT1 receptors after angiotensin II stimulation may\nallow the AT1 and D1 receptor to exert\ntheir functions\nseparately. Renal proximal tubular D1 receptors are functional\nin WKY but not in SHRs. In contrast, AT1 receptor function\nis enhanced in SHRs.32 The inability of AT1 receptors to\nincrease D1 receptor expression in renal proximal tubules in\nSHRs and a decreased D1 and AT1 receptor interaction could\nlead to enhanced AT1 receptor function.\nHypertension\nOctober 2005\nReferences\n1. Navar LG, Lewis L, Hymel A, Braam B, Mitchell KD. Tubular fluid\nconcentrations and kidney contents of angiotensins I and II in anesthe-\ntized rats. J Am Soc Nephrol. 1994;5:1153–1158.\n5. Handa RK, Krebs LT, Harding JW, Handa SE. Angiotensin IV AT4-\nreceptor system in the rat kidney. Am J Physiol. 1998;274:F290 –F299.\n6. Navar LG, Harrison-Bernard LM, Imig JD, Wang CT, Cervenka L,\nMitchell KD. Intrarenal angiotensin II generation and renal effects of AT1\nreceptor blockade. J Am Soc Nephrol. 1999;10(Suppl 12):S266 –S272.\n7. Hussain T, Lokhandwala MF. Renal dopamine receptor function in\n10. Ricci A, Marchal-Victorion S, Bronzetti E, Parini A, Amenta F, Tayebati\nSK. Dopamine D4 receptor expression in rat kidney: evidence for pre- and\npostjunctional localization. J Histochem Cytochem. 2002;50:1091–1096.\n11. Luippold G, Zimmermann C, Mai M, Kloor D, Starck D, Gross G,\nMuhlbauer B. Dopamine D3 receptors and salt-dependent hypertension.\nJ Am Soc Nephrol. 2001;12:2272–2279.\n14. Chen CJ, Apparsundaram S, Lokhandwala MF. Intrarenally produced\nangiotensin II opposes the natriuretic action of the dopamine-1 receptor\nagonist fenoldopam in rats. J Pharmacol Exp Ther. 1991;256:486 – 491.\n15. Sheikh-Hamad D, Wang YP, Jo OD, Yanagawa N. Dopamine antag-\nonizes the actions of angiotensin II in renal brush-border membrane. Am J\nPhysiol. 1993;264:F737–F743.\nZeng et al\nAT1 Receptor Regulation of D1 Receptors in Kidney\n50. Huang XC, Sumners C, Richards EM. Angiotensin II stimulates protein\nphosphatase 2A activity in cultured neuronal cells via type 2 receptors in\na pertussis toxin sensitive fashion. Adv Exp Med Biol. 1996;396:209 –215.\n51. Gelband CH, Sumners C, Lu D, Raizada MK. Angiotensin receptors and\nnorepinephrine neuromodulation: implications of functional coupling.\nRegul Pept. 1998;73:141–147.\n53. Felder RA, Sanada H, Xu J, Yu PY, Wang Z, Watanabe H, Asico LD,\nWang W, Zheng S, Yamaguchi I, Williams SM, Gainer J, Brown NJ,\nHazen-Martin D, Wong LJ, Robillard JE, Carey RM, Eisner GM, Jose\nPA. G protein-coupled receptor kinase 4 gene variants in human essential\nhypertension. Proc Natl Acad Sci U S A. 2002;99:3872–3877.",
         "C. Zeng,Z. Wang,U. Hopfer,L. Asico,G. Eisner,R. Felder,P. Jose",
         null
        ],
        [
         "46",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         null,
         null,
         null,
         null,
         null,
         null,
         "Giulio Rossi,G. Sartori,Riccardo Valli,F. Bertolini,Nazzarena Bigiani,L. Schirosi,A. Cavazza,G. Luppi",
         "broken link"
        ],
        [
         "47",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "1National Institute for Physiological Sciences, Okazaki 444-8787, Japan, 2The Graduate University of\nAdvanced Studies, Okazaki 444-8787, Japan, 3CREST of Japan Science and Technology Agency, Kawaguchi\n332-0012, Japan, 4Faculty of Textile Science and Technology, Shinshu University, Ueda 386-8567, Japan and\n5Graduate School of Biosphere Science, Hiroshima University, Higashi-Hiroshima 739-8528, Japan\nAbstract\nThe present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated\nrat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes\nwere placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-\ndependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15– 45 min\nrather than 46– 120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity tran-\nsiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-\nphase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed\noocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat\noocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent\nlevel of aged (120 min) or enucleated (1 60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2\nkinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were\ntreated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated\noocytes. A signiﬁcantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control\noocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus\ncell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage.\nThese results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for\ntheir inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemi-\ncals such as MG132 is in part effective for rat oocytes to promote PCC and further development.\nReproduction (2005) 129 171–180\nSuccessful animal cloning by somatic cell nuclear transfer\n(NT) has been reported in large domestic species (Wilmut\net al. 1997, Kato et al. 1998, Baguisi et al. 1999, Onishi\net al. 2000, Galli et al. 2003). As for small rodents,\nWakayama et al. (1998) were the ﬁrst to produce cloned\nmice using G0/G1-phase cumulus cells, and very recently\nZhou et al. (2003) reported the production of fertile cloned\nrats using M-phase fetal ﬁbroblast cells. However, the suc-\ncess rates of producing cloned rodents by NT are still low.",
         "Introduction\nresulting in premature chromosome condensation (PCC),\nwith subsequent multiple pseudo-pronuclei formation fol-\nlowing activation (Wakayama et al. 1998, 1999, Ogura\net al. 2000). It is considered in rodents, at least, that PCC\nis a critical aspect contributing to successful cloning\n(Wakayama et al. 1999, Ogura et al. 2000). However,\nPCC is not essential for somatic cell cloning in all species,\nsince telophase-II\n(TII) oocytes have promoted embryo\ndevelopment in goats (Baguisi et al. 1999) and cattle (Kur-\nosaka et al. 2002).\nThere is a major difference between mouse and rat, in\nthe behavior of somatic nuclei upon exposure to oocyte\ncytoplasm. We have reported that the PCC of both mouse\nand rat cumulus cell nuclei occurred in enucleated mouse\noocytes but not at all\nin enucleated rat oocytes\n(Hirabayashi et al. 2003a). Even if cumulus nuclei were\ninjected into the rat oocytes without enucleation, the rate\nof PCC formation was less than half (Hirabayashi et al.\n2003b).\nWe also reported that\nthe ability of rat oocytes to\npromote PCC of the injected nuclei was dependent on\nthe characteristics of the oocytes, such as age (4- to 5-\nweek-old, superior to .10-week-old rats) or strain of the\ndonor rats (Wistar and LEW, superior to the Donryu and\nF344 strains) and the timing of oocyte recovery (14 h post-\nhuman chorionic gonadotropin (hCG), superior to 17 h\npost hCG) (Hirabayashi et al. 2003b). In the same study,\nthe time management of NT completion (less than 45 min\nafter donor animal killing) and addition to the oocyte-\nhandling media\nof N-acetylleucylleucylnorleucinal\n(ALLN), a neutral cysteine protease inhibitor, were also\nfound important for intact rat oocytes to promote PCC.\nThe successful production of cloned rats (Zhou et al.\n2003) may be attributed to injecting the metaphase-\narrested fetal ﬁbroblasts into oocytes treated with MG132,\na proteasome inhibitor, followed by quick removal of the\nrecipient metaphase-plate. Therefore, it is a possibility that\nrecipient oocytes arrested at the metaphase-II (MII) stage\nby the inhibition of protein degradation have essential\nroles in nuclear reprogramming including PCC formation.\nOvulated rat oocytes are known to activate spon-\ntaneously but abortively during in vitro culture (Zernicka-\nGoetz 1991). We also conﬁrmed that more than half of\nthe oocytes from Wistar rats progressed to anaphase-II\n(AII)/TII stages at 70 min after killing of\nthe donor rats\n(Kato et al. 2001). In vertebrate oocytes, it has been con-\nsidered that high activity of maturation promoting factor\n(MPF) affects the chromosome condensation and the\nmaintenance of the meiotic spindle, both of which are\ninvolved in the meiotic arrest at\nthe MII stage (Jones\n2004). MPF is a heterodimer composed of p34cdc2 kinase\nand cyclin B1, controlling the cell cycle. When the\nactivity was inactivated by the degradation of cyclin B\n(Nurse 1990), oocytes were released from the arrest at the\nMII stage. The p34cdc2 kinase in oocytes was maintained\nat a high level until ovulation in many species (Kubiak\net al. 1993, Naito et al. 1995, Abrieu et al. 2001, Ito &\nShimada 2005), suggesting that p34cdc2 kinase activity in\nrat ovulated oocytes was rapidly decreased in vitro. More-\nover, destruction of\nthe nuclear envelope in mitotic\nsomatic cells occurred during the G2/M-phase transition\nby p34cdc2 kinase-related phosphorylation of lamin, which\nis one of\nthe nuclear membrane proteins (Peter et al.\n1990). It has been reported that ALLN, which was used in\nour previous study (Hirabayashi et al. 2003b), has an\neffect on the inhibition of cyclin B destruction (Sherwood\net al. 1993). Therefore, it is possible that the low rate of\nPCC of injected somatic cell nuclei resulted from rapidly\ndecreasing p34cdc2 kinase activity in rat ovulated oocytes,\nwhile direct evidence that p34cdc2 kinase activity is\ninvolved in the PCC incidence has not yet been shown.\nClarifying the relationship between p34cdc2 kinase activity\nand PCC incidence would contribute to understanding the\nreprogramming mechanism in mammalian\nnuclear\noocytes.\nIn the present study, two experiments were conducted\nto examine the effects of in vitro aging and enucleation of\nrat oocytes on the kinetics of p34cdc2 kinase and the inci-\ndence of PCC of microinjected cumulus cell nuclei, with\nsome comparisons with mouse oocytes. In addition, two\nmore experiments were conducted to explain a possible\nrole of p34cdc2 kinase for promoting PCC, by treatment of\nthe rat oocytes with roscovitine, an inhibitor of p34cdc2\nkinase (Lazar et al. 2002), and MG132, a proteasome\ninhibitor which has an effect on the inhibition of cyclin\nB1 degradation (Josefsberg et al. 2000).\nMaterials and Methods\nOocyte preparation",
         "The effect of in vitro aging on the p34cdc2 kinase activity\nand the PCC incidence was investigated using rat oocytes.\nTen oocytes for each test were sampled for\nin vitro\np34cdc2 kinase assay immediately after recovery (15 min\ngroup), and after culture for 30 min (45 min group), 60 min\n(75 min group) or 105 min (120 min group) in mKRB at\n37 8C in 5% CO2 in air. Nuclear conﬁguration of\nthe\noocytes was also examined immediately after recovery\nand every subsequent 30 min up to 120 min. The other\noocytes were placed in Hepes-mKRB, and the cumulus\ncell nuclei were then injected into the oocytes up to\n120 min. The injected oocytes were divided into two\ngroups based on the time from the animal killing to NT\ncompletion (15 –45 min group and 46 – 120 min group).\nThe incidence of PCC was evaluated 1 h after the last\ninjection among each group.\nExperiment 2: effect of enucleation\nThe effect of oocyte enucleation on the p34cdc2 kinase\nactivity and the PCC incidence was investigated using\nboth rat and mouse oocytes. For the in vitro p34cdc2\nkinase assay, oocytes derived from the same donor rats\nand mice were allocated to the non-enucleation group\nand the enucleation group, and the non-enucleated and\nenucleated oocytes were sampled 45 min after the animal\nkilling. Enucleation was performed by aspiration of the\nmetaphase-plate with a part of the volume (,10–20%) of\ncytoplasm in Hepes-R1ECM containing\nsurrounding\n5 mg/ml cytochalasin B (Sigma-Aldrich). In addition to the\nnon-enucleation and enucleation groups, denuded rat\noocytes were allocated to a cytoplasm-removal group\n(a part of the cytoplasm was mechanically aspirated as a\nsham control.). All the treatments were completed within\n45 min. The oocyte samples were harvested at 45, 75, 90\nand 105 min (0, 30, 45 and 60 min after enucleation\nrespectively). For PCC evaluation, nuclear injection into\nnon-enucleated and enucleated mouse and rat oocytes,\nand into cytoplasm-removed rat oocytes, was completed\nwithin 45 min after animal killing.\nExperiment 3: effect of roscovitine treatment\nWe investigated whether the decreased level of p34cdc2\nkinase activity in non-enucleated rat oocytes negatively\naffected the PCC incidence of injected cell nuclei. Ros-\ncovitine (Sigma-Aldrich) dissolved at 10 mM in DMSO\nand stored at 2 80 8C, a speciﬁc inhibitor of p34cdc2\nkinase, was added to all media throughout the process of\noocyte recovery and the in vitro culture at the ﬁnal con-\ncentration of 150 mM, as reported by Lazar et al. (2002).\nOocytes were sampled for in vitro p34cdc2 kinase assay at\n15 (immediately after recovery), 45, 75 and 120 min as in\nExperiment 1. Because the 45 min treatment of rat oocytes\nwith 150 mM roscovitine decreased their p34cdc2 kinase\nactivity to an equivalent level of oocytes 120 min after in\nvitro aging or 60 min after enucleation, nuclear injection\ninto the roscovitine-treated oocytes was performed within\n45 min, and the incidence of PCC was then evaluated\nafter a further 1 h.\nExperiment 4: effect of MG132 treatment\nWe investigated whether the treatment of rat oocytes with a\nproteasome inhibitor, by which the p34cdc2 kinase activity\nwould be maintained, could support the PCC incidence of\nthe injected donor cell nuclei. MG132 (Sigma-Aldrich), an\ninhibitor which can block cyclin B1 degradation, was used\nat the ﬁnal concentration of 5 mM, as reported by Josefsberg\net al. (2000). Non-enucleated oocytes were exposed to\nMG132,\nthe process of oocyte recovery to\nin vitro culture or nuclear injection. Oocyte sampling for\nin vitro p34cdc2 kinase assay was performed at 15 min\n(immediately after recovery), 45, 75 and 120 min. Nuclear\ninjection into the MG132-treated oocytes and PCC evalu-\nation were performed as in Experiment 3.\nthroughout\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nIn the ﬁrst group, oocytes exposed to MG132 from oocyte\nrecovery to just before enucleation were used as recipi-\nents. From the result of Experiment 4, MG132 treatment is\nexpected to support PCC even in enucleated oocytes,\nbecause the p34cdc2 kinase activity was maintained at a\nhigh level during the treatment. The procedure of enuclea-\ntion was as described above. The cytoplasts were then\ninjected with cumulus cell nuclei. One hour later, a high\nincidence of PCC was expected.\nIn the second group,\noocytes with the extrusion of a second polar body were\nselected from 2 h cultures of non-treated oocytes, and\nthen enucleated and nuclear injected as well. The reduced\nincidence of PCC under a lowered MPF was expected. For\nboth groups, the oocytes 1 h after nuclear injection were\nactivated by two direct current pulses (100 V/mm, 99 ms)\nand 4 h treatment with 2 mM 6-dimethylaminopurine\n(6-DMAP)\nin mR1ECM medium. The\nreconstructed oocytes were cultured in 100 ml microdrops\nof mR1ECM at 37 8C in 5% CO2 in air, and number of\noocytes forming one or two pseudo-pronuclei and cleav-\ning to the two-cell stage was recorded at 6 and 24 h of\nculture respectively.\n(Sigma-Aldrich)\nStatistical analysis\nEach experiment had at least three replicates. Data on\np34cdc2 kinase activity were compared by ANOVA and\nFisher’s protected least signiﬁcant difference test using the\nStatView program (Abacus Concepts, Inc., Berkeley, CA,\nUSA). Data on PCC incidence and NT embryo develop-\nment were compared by Fisher’s exact probability test. A\nvalue of P , 0.05 was chosen as an indication of statisti-\ncal signiﬁcance.\nResults\nExperiment 1: effect of in vitro aging\nThe time-dependent change in p34cdc2 kinase activity in\nrat oocytes is shown in Fig. 1. The p34cdc2 kinase activity\nof\nrat oocytes decreased signiﬁcantly at 45 min (after\n30 min cultivation) and more signiﬁcantly at 75 min (after\n60 min cultivation). The decreased activity was maintained\nuntil 120 min (after 105 min cultivation). Probably due to\nthe decrease of p34cdc2 kinase activity, the release from\nmeiotic arrest of ovulated oocytes occurred in a time-\ndependent manner, as shown in Fig. 2. Within 75 min\nafter animal killing, more than half of the oocytes pro-\ngressed beyond the MII stage. At 135 min, most of the\noocytes were at either the AII or the TII stage. The pro-\nportion of in vitro-aged rat oocytes promoting PCC of the\ninjected cumulus cell nuclei is shown in Table 1. When\nnuclear injection was completed within 45 min from ani-\nmal killing, a positive sign of PCC was observed in 32.5%\nof the NT oocytes. On the other hand, PCC occurred in\nonly 2.6% of the oocytes into which cumulus cell nuclei\nwere injected between 46 and 120 min.\nExperiment 2: effect of oocyte enucleation\nThe activities of p34cdc2 kinase in non-enucleated and\nenucleated oocytes within 45 min were compared in rats\nand mice, as shown in Fig. 3. In both species, there was\na slight decrease in the p34cdc2 kinase activity after enu-\ncleation in both rat and mouse oocytes. A considerable\nRole of MPF in PCC induction of rat oocytes 175\ntreatment (enucleation or cytoplasm removal) is shown in\nTable 2. In rats, despite little reduction of p34cdc2 kinase\nactivity, few of the oocytes that either had a small portion\nof cytoplasm-removed (17.1%) or the non-treated controls\n(28.2%) could support PCC at 105 min after animal\nin the\nkilling. However, PCC was not observed at all\nfound in the p34cdc2 kinase activity\ndifference was\nbetween rat and mouse oocytes, because the mouse\noocytes carried a p34cdc2 kinase activity 2.5 times that of\nrat oocytes either immediately or up to 60 min after enu-\ncleation or cytoplasm removal and compared with non-\ntreated, age-matched controls, as shown in Fig. 4. The\nactivities of p34cdc2 kinase were measured in rat oocytes\nimmediately after or 60 min after one of three treatments\n(enucleated, cytoplasm-removed and non-treated), as\nshown in Fig. 4. Surprisingly, a higher p34cdc2 kinase\nactivity was detected in the oocytes immediately after\nenucleation, although the p34cdc2 kinase activity of\noocytes immediately after cytoplasm removal was not\nfrom that of non-treated oocytes. The p34cdc2\ndifferent\nin enucleated oocytes dramatically\nactivity\nkinase\ndecreased within 30 min after enucleation. At 45 min\nafter enucleation, the activity was further decreased and\nthereafter maintained until 60 min. However, the activity\nin non-treated or cytoplasm-removed oocytes did not sig-\nniﬁcantly differ by 60 min. The proportion of rat and\nmouse oocytes promoting PCC in relation to mechanical\nenucleated oocytes where p34cdc2 kinase activity was sig-\nniﬁcantly decreased at 60 min after completion of enu-\ncleation. In contrast, PCC of mouse cumulus cell nuclei\noccurred in most of the mouse oocytes before enucleation\n(96.7%) and even after enucleation (93.1%).\nExperiment 3: effect of roscovitine treatment\nThe time-dependent decrease of p34cdc2 kinase activity in\nroscovitine-treated rat oocytes is shown in Fig. 5. The\nactivity\nlinearly\nin roscovitine-treated oocytes was\ndecreased until 75 min and maintained until 120 min.\nAt 45, 75 and 120 min, each activity in the roscovitine-\ntreated oocytes was signiﬁcantly lower than that of non-\ntreated control oocytes. The result of PCC incidence in the\noocytes treated with or without roscovitine for 45 min is\nshown in Table 3. None of the roscovitine-treated oocytes\npromoted PCC, although 41.7% of non-treated oocytes\npromoted it.\nExperiment 4: effect of MG132 treatment\nThe activities of p34cdc2 kinase in MG132-treated rat\noocytes are shown in Fig. 6. The level of p34cdc2 kinase\nactivity still decreased in the MG132-treated oocytes\nafter 30 min of culture, but were signiﬁcantly greater\nthan in control oocytes. The p34cdc2 kinase activity was\nmaintained during the further incubation up to 105 min.\nThe kinase activities of MG132-treated oocytes at 60\nand 105 min cultivation were signiﬁcantly greater than\nthose of control oocytes. As shown in Table 4, the pro-\nportion of MG132-treated oocytes promoting PCC\n(66.0%) was higher than that of the non-treated oocytes\n(46.5%) when the nuclear\ninjection was completed\nwithin 45 min.\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nthe reconstructed embryos\nAfter oocyte activation with the electrical stimulus plus 6-\nDMAP,\nfrom both groups\n(MG132-treated MII oocytes and non-treated TII oocytes\nat the enucleation) formed one or two pseudo-pronuclei\n(91.7 and 87.5% in total respectively). After activation, a\ngreater proportion of oocytes enucleated at the MII stage\ntended to form two pseudo-pronuclei compared with\nthose enucleated at the TII stage. A signiﬁcantly higher\nproportion of the NT embryos in the MG132-treated group\nvs the non-treated group (59.1 vs 7.1%) developed to the\ntwo-cell stage.",
         "Discussion\nRat oocytes have poor potential\nfor promoting PCC of\ninjected somatic cell nuclei, and the rate was markedly\ndecreased in aged or enucleated oocytes (Hirabayashi\nIn the present study, rat oocytes into\net al. 2003a,b).\nwhich cumulus cell nuclei were injected within 45 min\nRole of MPF in PCC induction of rat oocytes 177\nthe non-treated oocytes during the same period, may\ninﬂuence the outcome of PCC. The high concentration of\nroscovitine affects the other cyclin-dependent protein\nkinases such as cdk2 (De Azevado et al. 1997). The\nunknown pathway and/or chemical toxicity of roscovitine\ncannot be ruled out\nfor complete suppression of PCC\ninduction. Moreover, the level of activity in rat oocytes\nwas less than half compared with that in mouse oocytes,\nwhich had a high potential for promoting PCC. Judging\nfrom these results, we estimated that the high activity of\np34cdc2 kinase is associated with promoting PCC of\ninjected somatic cell nuclei, and the low level of p34cdc2\nkinase activity is one of the reasons for the poor potential\nto promote PCC in rat oocytes compared with that of\nmouse oocytes. The reason why the potential\nlevel of\np34cdc2 kinase activity is different between mouse and rat\noocytes remains unclear. Our previous study indicated, at\nleast, that the PCC-inducing potential is determined by\nthe origin of the recipient cytoplasm, not by the origin of\nin the interspecies NT experiment\nthe donor nucleus,\nbetween mouse and rat\nIn\n(Hirabayashi et al. 2003a).\nsomatic cell division, dissolution of the nuclear envelope\noccurs during G2/M transition in mitosis (Nurse 1990) and\nis induced by phosphorylation of lamin, one of the major\nnuclear membrane proteins (Peter et al. 1990). In oocytes,\np34cdc2 kinase had the potential to phosphorylate lamins,\ninducing meiotic breakdown of the nuclear envelope and\nthe meiotic resumption (Dessev & Goldman 1988). There-\nfore, it is possible that p34cdc2 kinase plays an important\nrole in the destruction of the nuclear membrane of the\ninjected somatic cell through phosphorylation of lamin.\nOne possible pathway known for inactivation of p34cdc2\nkinase is the calmodulin-dependent pathway via Ca2þ.\nWe have reported that the Ca2þ-dependent pathway trig-\ngers the degradation of cyclin B and results in inactivation\nof p34cdc2 kinase in porcine oocytes (Ito et al. 2003,\n2004a). In Xenopus oocytes (Lorca et al. 1994, Morin et al.\n1994), activation of calmodulin-dependent protein kinase\nII induces the degradation of cyclin B via the ubiquitin\nproteasome pathway. Based on these literature sources\nand preliminary data, we have investigated the effect of a\nproteasome inhibitor MG132 on p34cdc2 kinase activity\nand PCC incidence. The PCC incidence in rat oocytes was\nsigniﬁcantly improved by MG132 treatment (Table 4). This\nresult matched well with the report by Josefsberg et al.\n(2000), who demonstrated that MG132-treated rat oocytes\ncontained a high amount of cyclin B1.\nIn the present\nstudy, MG132 also provided a suppressive effect on inacti-\nvation of p34cdc2 kinase (Fig. 6). Use of ALLN, a neutral\ncysteine protease inhibitor, was also slightly effective for\nimproving the PCC incidence in rat oocytes (Hirabayashi\net al. 2003b). Thus, the Ca2þ-dependent degradation path-\nway, which may induce a decrease in the cyclin B1 level,\nis activated in aged rat oocytes, which results in a low\nlevel of p34cdc2 kinase activity, and a low potential\nto promote PCC. Our unpublished data (M Hirabayashi\nand J Ito, unpublished data) showing that the decreased\nIn mouse oocytes, PCC was initiated at\npromoted PCC (frequency 28–47% in Tables 1 –4), but\nthe incidence of PCC decreased with oocyte aging with a\nsigniﬁcant decrease in p34cdc2 kinase activity (Table 1,\nFig. 1).\nleast\n30 min after nuclear injection and lasted until 2 h (Chen\net al. 2004), therefore the timing of PCC observation in\nthe present study (60 min after nuclear injection) might be\noptimal. When oocytes are treated with roscovitine,\nwhich decreases the level of p34cdc2 kinase activity, PCC\nformation was never observed even though nuclear injec-\ntion is completed within 45 min. Roscovitine is widely\nused as cyclin-dependent kinase inhibitor\nto suppress\np34cdc2 kinase in rodent oocytes (Phillips et al. 2002).\nTherefore, the rapid decrease of p34cdc2 kinase activity in\nthe roscovitine-treated oocytes found between 45 and\n75 min (Fig. 5), when compared with the slow decrease in\na Oocytes were treated with 5 mM MG132 until just before enucleation after oocyte recovery.\nb Oocytes were cultured for about 2 h and only oocytes with a completely extruded polar body (the TII stage) were used.\nc Percentage of reconstructed embryos with 1 or 2 pronuclei.\nd – e Different superscripts denote signiﬁcant difference (P , 0.05).\nconcentration of extracellular free calcium in the handling\nmedium partially inhibits spontaneous activation of rat\noocytes, support this hypothesis.\nThe other pathway for inactivation by phosphorylation\nof 14Thr and 15Tyr of p34cdc2 kinase has been reported in\naged porcine oocytes (Kikuchi et al. 2000). In the same\nstudy, it has been reported that caffeine suppressed the\nfrom MPF to pre-MPF (14Thr/15Tyr-phosphorylated\nshift\np34cdc2 kinase þ cyclin B1) in the aged porcine oocytes.\nOkadaic acid is also known as a protein phosphatase\ninhibitor which prevents phosphorylation of 14Thr and\n15Tyr of p34cdc2 kinase (Rime et al. 1995). In our prelimi-\nnary study, PCC incidence of injected cumulus cell nuclei\nwas slightly improved in rat oocytes treated with okadaic\nacid (M Hirabayashi and J Ito, unpublished data). There-\nfore, the phosphorylation of p34cdc2 kinase as well as the\ncyclin degradation may be involved in spontaneous acti-\nvation of rat oocytes during in vitro culture.\nThe level of p34cdc2 kinase activity in non-treated rat\noocytes was similar to that of cytoplasm-removed or enu-\ncleated oocytes in the case where the oocytes were\nsampled immediately after the treatment\n(Fig. 4). The\nmechanical puncture of the oocyte membrane and the\naspiration of the cytoplasm or karyoplast did not inﬂuence\nthe p34cdc2 kinase activity immediately after the enuclea-\ntion procedure beyond the level expected from the rela-\ntive volume of aspirated cytoplasm (, 10–20%). These\nresults strongly suggest that active p34cdc2 kinase is not\naccumulated near the metaphase spindles in rat oocytes.\nThere was signiﬁcant difference in the p34cdc2 kinase\nactivity between 45 and 75 min in both non-treated and\nenucleated groups (Fig. 4). A dramatic decrease in the\nactivity of p34cdc2 kinase was observed in the rat oocytes\nenucleated and cultured for 30 min compared with the\nnon-treated group (Fig. 4), suggesting that\nthe oocyte\nmetaphases per se are involved in the maintenance of\np34cdc2 kinase activity. We have previously reported in\nporcine oocytes that activation of p34cdc2 kinase was\ndependent on the synthesis of cyclin B and that the syn-\nthesis was induced by the activation of mitogen-activated\nprotein (MAP) kinase (Ito et al. 2004b). The MAP kinase\nsuppressed Myt1 activity that phosphorylated p34cdc2\nkinase at 14Thr and 15Tyr (Mueller et al. 1995, Palmer et al.\n1998). Because activated MAP kinase is localized on the\nmeiotic spindle in porcine oocytes (Lee et al. 2000), it is\npossible that enucleation results in the loss of MAP kinase\nfrom the recipient oocytes.\ninjection into the MG132-treated, MII-enu-\ncleated rat oocytes resulted in a higher development to\nthe two-cell stage when compared with non-treated, TII-\nenucleated rat oocytes after 24 h cultivation (Table 5).\nThe favorable circumstance of recipient cytoplasm with\na high MPF activity must induce PCC and promote sub-\nsequent development of NT embryos. Wakayama &\nYanagimachi\nthe injected\n(2001) reported in mice that\ncumulus cell nuclei undergoing PCC transformation to\nthe two pseudo-pronuclei and such NT embryos are able\nto develop to full-term. In contrast, a different situation\nhas been reported in the cloning of large domestic ani-\nmals. The somatic cell NT in the livestock species is suc-\ncessfully reproducible when enucleated and activated\noocytes with a low MPF activity are used as recipient\ncytoplasm (Baguisi et al. 1999, Kurosaka et al. 2002,\nShin et al. 2002). Such a difference between rodents\nand large domestic animals remains unclear, but our\nthat\npresent data suggest\ninduction of PCC in recon-\nstructed embryos\nrelated to the high activity of\nis\np34cdc2 kinase, which seems to play a key role in the\nsuccessful rat cloning.\nthe decreased level of p34cdc2 kinase\nactivity in aged or enucleated rat oocytes is responsible\nfor their inability to promote PCC of microinjected donor\ncell nuclei. Inhibition of p34cdc2 kinase inactivation by\nchemicals such as MG132 is in part effective for rat\noocytes to promote PCC and cleavage. These data may\nexpand our existing knowledge on characteristics of rat\noocytes and contribute to the development of a rat NT\nprotocol using G0/G1-stage donor cells.\nAcknowledgements\nThe authors thank Mrs A Ishikawa and Miss A Takata for their\ntechnical assistance. This work was supported in part by\nGrant-in-Aids for the Scientiﬁc Research (No. 15082211, No.\n16300139; to M H and S H), and the 21st Century COE Pro-\ngram (to S H) from the Ministry of Education, Culture, Sports,\nScience and Technology of Japan, and a Research Fellowship\nfor Young Scientists (No. 8254; to J I.) from the Japan Society\nfor the Promotion of Science.\nReferences\nDessev G & Goldman R 1988 Meiotic breakdown of nuclear envel-\nope in oocytes of Spisula solidissima involves phosphorylation and\nrelease of nuclear lamin. Developmental Biology 130 543–550.\nGalli C, Lagutina I, Crotti G, Colleoni S, Turini P, Ponderato N et al.\n2003 Pregnancy: a cloned horse born to its dam twin. Nature 424\n635.\nHirabayashi M, Ito K, Sekimoto A, Hochi S & Ueda M 2001 Pro-\nduction of transgenic rats using young Sprague–Dawley females\ntreated with PMSG and hCG. Experimental Animals 50 365–369.\nHirabayashi M, Kato M, Ishikawa A & Hochi S 2003a Factors inﬂuen-\ncing chromosome condensation and development of cloned rat\nembryos. Cloning and Stem Cells 5 35–42.\nIto J & Shimada M 2005 Timing of MAP kinase inactivation effects\non emission of polar body in porcine oocytes activated by Ca2þ\nionophore. Molecular Reproduction and Development 70 64– 69.\nIto J, Shimada M & Terada T 2001 Progression of nuclear maturation\nand p34cdc2 kinase activity in porcine oocytes during in vitro cul-\nture in different media. Journal of Mammalian Ova Research 18\n39– 43.\nRole of MPF in PCC induction of rat oocytes 179\nKikuchi K, Naito K, Noguchi J, Shimada A, Kaneko H, Yamashita M\net al. 2000 Maturation/M-phase promoting factor: a regulator of\naging in porcine oocytes. Biology of Reproduction 63 715– 722.\nKimura Y & Yanagimachi R 1995 Mouse oocytes injected with testi-\ncular spermatozoa or round spermatids can develop into normal\noffspring. Development 121 2397–2405.\nRime H, Jessus C & Ozon R 1995 Tyrosine phosphorylation of\np34cdc2 is regulated by protein phosphatase 2A in growing imma-\nture Xenopus oocytes. Experimental Cell Research 219 29– 38.\nSherwood SW, Kung AL, Roitelman J, Simoni RD & Schimke RT\n1993 In vivo inhibition of cyclin B degradation and induction\nof cell-cycle arrest in mammalian cells by the neutral cysteine\nprotease\ninhibitor N-acetylleucylleucylnorleucinal. PNAS 90\n3353–3357.",
         "Speciﬁc-pathogen-free Crj:Wistar female rats (3 –5 weeks\nold) or Crj:BDF1 female mice (7 weeks old) were pur-\nchased from Charles River Japan, Inc. (Kanagawa, Japan).\nThey were housed under controlled lighting (lights on at\ntemperature (23 ^ 2 8C) and humidity\n0600 –1800 h),\n(55 ^ 5%), with free access\n(MF;\nOriental Yeast Co., Tokyo, Japan) and water. Rats were\nsuperovulated by i.p. injections of 30 IU equine chorio-\nnic gonadotropin (eCG) (Nippon Zenyaku, Co., Fukush-\nima, Japan) and 30 IU hCG (Sankyo Yell Yakuhin, Co.,\nTokyo, Japan) at 48 h intervals (Hirabayashi et al. 2001).\nMice were superovulated as well with 5 IU eCG and 5\nIU hCG.\nto laboratory diet\nFourteen hours after the hCG injection, cumulus –oocyte\ncomplexes (COCs) were collected from the oviductal\nampullae of donor rats or mice with modiﬁed Krebs–\nRinger bicarbonate solution (mKRB) in Experiments 1 and\n3 (Toyoda & Chang 1974) and modiﬁed rat one-cell\nembryo culture medium (mR1ECM) in Experiments 2, 4\nand 5 (Oh et al. 1998) supplemented with 0.1% hyaluroni-\ndase (Sigma-Aldrich Corp., St Louis, MO, USA) respect-\nively. Two to ﬁve minutes later, the denuded oocytes were\nwashed three times with fresh mKRB or R1ECM and kept in\nthe same medium at 37 8C until being subjected to the\ntreatments. The time of killing of the donor animals was\ndeﬁned as 0 min, and it usually took 10 –15 min from the\nanimal killing to the preparation of denuded oocytes.\nMeasurement of p34cdc2 kinase activity\nThe p34cdc2 kinase assay was performed according to the\nmethod as described by Ito et al. (2001) with some modiﬁ-\ncations. In brief, the oocytes were washed several times\nwith PBS and placed into cell lysis buffer composed of\n20 mM Tris, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1%\nTriton X-100, 2.5 mM sodium pyrophosphate, 1 mM b-gly-\ncerophosphate, 1 mM Na3VO4, 1 mg/ml\nleupeptin (all\nchemicals from Cell Signaling Technology, Beverly, MA,\nUSA) and 1 mM phenylmethylsulfonyl ﬂuoride (Sigma-\nAldrich). The oocyte lysates were stored at 2 80 8C until\nthe ELISA assay.\n(MBL, Nagoya,\nThe lysate (ten oocytes/5 ml cell lysis buffer) was mixed\nwith 45 ml of kinase assay buffer-A composed of 25 mM\nHepes buffer\nJapan), 10 mM\nMgCl2 (MBL), 10% (v/v) mouse vimentin peptide solution\n(SLYSSPGGAYC) (MBL) and 0.1 mM ATP (Sigma-Aldrich),\nand the mixture was incubated for 30 min at 30 8C. The\nreaction was terminated by the addition of 200 ml PBS\ncontaining 50 mM EGTA (MBL). The phosphorylation of\nmouse vimentin peptides was detected using an ELISA\nanalysis (MESACUP cdc2 kinase assay kit\n(code no.\n5234); MBL). Data were expressed in terms of the strength\nof p34cdc2 kinase activity in denuded rat oocytes 15 min\n(Experiments 1, 3 and 4) or 45 min (Experiment 2) after\nanimal killing.\nAssessment of nuclear conﬁguration\nRat oocytes immediately after recovery and 30, 60, 90\nand 120 min after culture in mKRB were stained with\n5 mg/ml Hoechst 33342 (Sigma-Aldrich). Their nuclear\nconﬁguration was then assessed under UV light at 330–\n380 nm, according to the classiﬁcations reported pre-\nviously (Kato et al. 2001): MII, characterized by a meta-\nphase spindle with the ﬁrst polar body; AII, characterized\nby elongated spindle without the second polar body; and\nTII, characterized by elongated spindle with extrusion of\nthe second polar body.\nNuclear injection and PCC evaluation\nCumulus cells dispersed from COCs by the hyaluronidase\ntreatment were placed in mKRB (Experiments 1 and 3) or\nR1ECM (Experiments 2, 4 and 5) containing 22 mM Hepes\nand 5 mM NaHCO3 (abbreviated hereafter as Hepes-\nmKRB and Hepes-R1ECM respectively). An aliquot (2 ml)\nof the cumulus cell suspension was transferred to 10 ml of\nHepes-mKRB or Hepes-R1ECM supplemented with 12%\nw/v 360 kDa polyvinylpyrrolidone (ICN Pharmaceuticals,\nInc., Costa Mesa, CA, USA). Nuclei were removed from\ncumulus cells and gently aspirated in and out of the injec-\ntion pipette (7 –10 mm) until\ntheir nuclei were largely\ndevoid of visible cytoplasmic materials. Each nucleus was\ninjected into a separate enucleated oocyte within 5 min of\nits isolation, as described by Kimura & Yanagimachi\n(1995). One hour after the nuclear injection, the oocytes\nwere stained with 5 mg/ml Hoechst 33342 and evaluated\nfor incidence of PCC under UV light at 330–380 nm, as\ndescribed previously (Hirabayashi et al. 2003a,b). The\ninjected somatic cell nucleus was distinguished from the\noocyte nucleus by the position of the opening in the zona\npellucida at the nuclear microinjection site.\nRole of MPF in PCC induction of rat oocytes 173\nExperimental design\nExperiment 1: effect of in vitro aging\nThe effect of in vitro aging on the p34cdc2 kinase activity\nand the PCC incidence was investigated using rat oocytes.\nTen oocytes for each test were sampled for\nin vitro\np34cdc2 kinase assay immediately after recovery (15 min\ngroup), and after culture for 30 min (45 min group), 60 min\n(75 min group) or 105 min (120 min group) in mKRB at\n37 8C in 5% CO2 in air. Nuclear conﬁguration of\nthe\noocytes was also examined immediately after recovery\nand every subsequent 30 min up to 120 min. The other\noocytes were placed in Hepes-mKRB, and the cumulus\ncell nuclei were then injected into the oocytes up to\n120 min. The injected oocytes were divided into two\ngroups based on the time from the animal killing to NT\ncompletion (15 –45 min group and 46 – 120 min group).\nThe incidence of PCC was evaluated 1 h after the last\ninjection among each group.\nExperiment 2: effect of enucleation\nThe effect of oocyte enucleation on the p34cdc2 kinase\nactivity and the PCC incidence was investigated using\nboth rat and mouse oocytes. For the in vitro p34cdc2\nkinase assay, oocytes derived from the same donor rats\nand mice were allocated to the non-enucleation group\nand the enucleation group, and the non-enucleated and\nenucleated oocytes were sampled 45 min after the animal\nkilling. Enucleation was performed by aspiration of the\nmetaphase-plate with a part of the volume (,10–20%) of\ncytoplasm in Hepes-R1ECM containing\nsurrounding\n5 mg/ml cytochalasin B (Sigma-Aldrich). In addition to the\nnon-enucleation and enucleation groups, denuded rat\noocytes were allocated to a cytoplasm-removal group\n(a part of the cytoplasm was mechanically aspirated as a\nsham control.). All the treatments were completed within\n45 min. The oocyte samples were harvested at 45, 75, 90\nand 105 min (0, 30, 45 and 60 min after enucleation\nrespectively). For PCC evaluation, nuclear injection into\nnon-enucleated and enucleated mouse and rat oocytes,\nand into cytoplasm-removed rat oocytes, was completed\nwithin 45 min after animal killing.\nExperiment 3: effect of roscovitine treatment\nWe investigated whether the decreased level of p34cdc2\nkinase activity in non-enucleated rat oocytes negatively\naffected the PCC incidence of injected cell nuclei. Ros-\ncovitine (Sigma-Aldrich) dissolved at 10 mM in DMSO\nand stored at 2 80 8C, a speciﬁc inhibitor of p34cdc2\nkinase, was added to all media throughout the process of\noocyte recovery and the in vitro culture at the ﬁnal con-\ncentration of 150 mM, as reported by Lazar et al. (2002).\nOocytes were sampled for in vitro p34cdc2 kinase assay at\n15 (immediately after recovery), 45, 75 and 120 min as in\nExperiment 1. Because the 45 min treatment of rat oocytes\nwith 150 mM roscovitine decreased their p34cdc2 kinase\nactivity to an equivalent level of oocytes 120 min after in\nvitro aging or 60 min after enucleation, nuclear injection\ninto the roscovitine-treated oocytes was performed within\n45 min, and the incidence of PCC was then evaluated\nafter a further 1 h.\nExperiment 4: effect of MG132 treatment\nWe investigated whether the treatment of rat oocytes with a\nproteasome inhibitor, by which the p34cdc2 kinase activity\nwould be maintained, could support the PCC incidence of\nthe injected donor cell nuclei. MG132 (Sigma-Aldrich), an\ninhibitor which can block cyclin B1 degradation, was used\nat the ﬁnal concentration of 5 mM, as reported by Josefsberg\net al. (2000). Non-enucleated oocytes were exposed to\nMG132,\nthe process of oocyte recovery to\nin vitro culture or nuclear injection. Oocyte sampling for\nin vitro p34cdc2 kinase assay was performed at 15 min\n(immediately after recovery), 45, 75 and 120 min. Nuclear\ninjection into the MG132-treated oocytes and PCC evalu-\nation were performed as in Experiment 3.\nthroughout\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nIn the ﬁrst group, oocytes exposed to MG132 from oocyte\nrecovery to just before enucleation were used as recipi-\nents. From the result of Experiment 4, MG132 treatment is\nexpected to support PCC even in enucleated oocytes,\nbecause the p34cdc2 kinase activity was maintained at a\nhigh level during the treatment. The procedure of enuclea-\ntion was as described above. The cytoplasts were then\ninjected with cumulus cell nuclei. One hour later, a high\nincidence of PCC was expected.\nIn the second group,\noocytes with the extrusion of a second polar body were\nselected from 2 h cultures of non-treated oocytes, and\nthen enucleated and nuclear injected as well. The reduced\nincidence of PCC under a lowered MPF was expected. For\nboth groups, the oocytes 1 h after nuclear injection were\nactivated by two direct current pulses (100 V/mm, 99 ms)\nand 4 h treatment with 2 mM 6-dimethylaminopurine\n(6-DMAP)\nin mR1ECM medium. The\nreconstructed oocytes were cultured in 100 ml microdrops\nof mR1ECM at 37 8C in 5% CO2 in air, and number of\noocytes forming one or two pseudo-pronuclei and cleav-\ning to the two-cell stage was recorded at 6 and 24 h of\nculture respectively.\n(Sigma-Aldrich)\nStatistical analysis\nEach experiment had at least three replicates. Data on\np34cdc2 kinase activity were compared by ANOVA and\nFisher’s protected least signiﬁcant difference test using the\nStatView program (Abacus Concepts, Inc., Berkeley, CA,\nUSA). Data on PCC incidence and NT embryo develop-\nment were compared by Fisher’s exact probability test. A\nvalue of P , 0.05 was chosen as an indication of statisti-\ncal signiﬁcance.\nResults\nExperiment 1: effect of in vitro aging",
         "1National Institute for Physiological Sciences, Okazaki 444-8787, Japan, 2The Graduate University of\nAdvanced Studies, Okazaki 444-8787, Japan, 3CREST of Japan Science and Technology Agency, Kawaguchi\n332-0012, Japan, 4Faculty of Textile Science and Technology, Shinshu University, Ueda 386-8567, Japan and\n5Graduate School of Biosphere Science, Hiroshima University, Higashi-Hiroshima 739-8528, Japan\nAbstract\nThe present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated\nrat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes\nwere placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-\ndependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15– 45 min\nrather than 46– 120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity tran-\nsiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-\nphase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed\noocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat\noocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent\nlevel of aged (120 min) or enucleated (1 60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2\nkinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were\ntreated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated\noocytes. A signiﬁcantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control\noocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus\ncell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage.\nThese results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for\ntheir inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemi-\ncals such as MG132 is in part effective for rat oocytes to promote PCC and further development.\nReproduction (2005) 129 171–180\nIntroduction\nSuccessful animal cloning by somatic cell nuclear transfer\n(NT) has been reported in large domestic species (Wilmut\net al. 1997, Kato et al. 1998, Baguisi et al. 1999, Onishi\net al. 2000, Galli et al. 2003). As for small rodents,\nWakayama et al. (1998) were the ﬁrst to produce cloned\nmice using G0/G1-phase cumulus cells, and very recently\nZhou et al. (2003) reported the production of fertile cloned\nrats using M-phase fetal ﬁbroblast cells. However, the suc-\ncess rates of producing cloned rodents by NT are still low.\nresulting in premature chromosome condensation (PCC),\nwith subsequent multiple pseudo-pronuclei formation fol-\nlowing activation (Wakayama et al. 1998, 1999, Ogura\net al. 2000). It is considered in rodents, at least, that PCC\nis a critical aspect contributing to successful cloning\n(Wakayama et al. 1999, Ogura et al. 2000). However,\nPCC is not essential for somatic cell cloning in all species,\nsince telophase-II\n(TII) oocytes have promoted embryo\ndevelopment in goats (Baguisi et al. 1999) and cattle (Kur-\nosaka et al. 2002).\nThere is a major difference between mouse and rat, in\nthe behavior of somatic nuclei upon exposure to oocyte\ncytoplasm. We have reported that the PCC of both mouse\nand rat cumulus cell nuclei occurred in enucleated mouse\noocytes but not at all\nin enucleated rat oocytes\n(Hirabayashi et al. 2003a). Even if cumulus nuclei were\ninjected into the rat oocytes without enucleation, the rate\nof PCC formation was less than half (Hirabayashi et al.\n2003b).\nWe also reported that\nthe ability of rat oocytes to\npromote PCC of the injected nuclei was dependent on\nthe characteristics of the oocytes, such as age (4- to 5-\nweek-old, superior to .10-week-old rats) or strain of the\ndonor rats (Wistar and LEW, superior to the Donryu and\nF344 strains) and the timing of oocyte recovery (14 h post-\nhuman chorionic gonadotropin (hCG), superior to 17 h\npost hCG) (Hirabayashi et al. 2003b). In the same study,\nthe time management of NT completion (less than 45 min\nafter donor animal killing) and addition to the oocyte-\nhandling media\nof N-acetylleucylleucylnorleucinal\n(ALLN), a neutral cysteine protease inhibitor, were also\nfound important for intact rat oocytes to promote PCC.\nThe successful production of cloned rats (Zhou et al.\n2003) may be attributed to injecting the metaphase-\narrested fetal ﬁbroblasts into oocytes treated with MG132,\na proteasome inhibitor, followed by quick removal of the\nrecipient metaphase-plate. Therefore, it is a possibility that\nrecipient oocytes arrested at the metaphase-II (MII) stage\nby the inhibition of protein degradation have essential\nroles in nuclear reprogramming including PCC formation.\nOvulated rat oocytes are known to activate spon-\ntaneously but abortively during in vitro culture (Zernicka-\nGoetz 1991). We also conﬁrmed that more than half of\nthe oocytes from Wistar rats progressed to anaphase-II\n(AII)/TII stages at 70 min after killing of\nthe donor rats\n(Kato et al. 2001). In vertebrate oocytes, it has been con-\nsidered that high activity of maturation promoting factor\n(MPF) affects the chromosome condensation and the\nmaintenance of the meiotic spindle, both of which are\ninvolved in the meiotic arrest at\nthe MII stage (Jones\n2004). MPF is a heterodimer composed of p34cdc2 kinase\nand cyclin B1, controlling the cell cycle. When the\nactivity was inactivated by the degradation of cyclin B\n(Nurse 1990), oocytes were released from the arrest at the\nMII stage. The p34cdc2 kinase in oocytes was maintained\nat a high level until ovulation in many species (Kubiak\net al. 1993, Naito et al. 1995, Abrieu et al. 2001, Ito &\nShimada 2005), suggesting that p34cdc2 kinase activity in\nrat ovulated oocytes was rapidly decreased in vitro. More-\nover, destruction of\nthe nuclear envelope in mitotic\nsomatic cells occurred during the G2/M-phase transition\nby p34cdc2 kinase-related phosphorylation of lamin, which\nis one of\nthe nuclear membrane proteins (Peter et al.\n1990). It has been reported that ALLN, which was used in\nour previous study (Hirabayashi et al. 2003b), has an\neffect on the inhibition of cyclin B destruction (Sherwood\net al. 1993). Therefore, it is possible that the low rate of\nPCC of injected somatic cell nuclei resulted from rapidly\ndecreasing p34cdc2 kinase activity in rat ovulated oocytes,\nwhile direct evidence that p34cdc2 kinase activity is\ninvolved in the PCC incidence has not yet been shown.\nClarifying the relationship between p34cdc2 kinase activity\nand PCC incidence would contribute to understanding the\nreprogramming mechanism in mammalian\nnuclear\noocytes.\nIn the present study, two experiments were conducted\nto examine the effects of in vitro aging and enucleation of\nrat oocytes on the kinetics of p34cdc2 kinase and the inci-\ndence of PCC of microinjected cumulus cell nuclei, with\nsome comparisons with mouse oocytes. In addition, two\nmore experiments were conducted to explain a possible\nrole of p34cdc2 kinase for promoting PCC, by treatment of\nthe rat oocytes with roscovitine, an inhibitor of p34cdc2\nkinase (Lazar et al. 2002), and MG132, a proteasome\ninhibitor which has an effect on the inhibition of cyclin\nB1 degradation (Josefsberg et al. 2000).\nMaterials and Methods\nOocyte preparation\nSpeciﬁc-pathogen-free Crj:Wistar female rats (3 –5 weeks\nold) or Crj:BDF1 female mice (7 weeks old) were pur-\nchased from Charles River Japan, Inc. (Kanagawa, Japan).\nThey were housed under controlled lighting (lights on at\ntemperature (23 ^ 2 8C) and humidity\n0600 –1800 h),\n(55 ^ 5%), with free access\n(MF;\nOriental Yeast Co., Tokyo, Japan) and water. Rats were\nsuperovulated by i.p. injections of 30 IU equine chorio-\nnic gonadotropin (eCG) (Nippon Zenyaku, Co., Fukush-\nima, Japan) and 30 IU hCG (Sankyo Yell Yakuhin, Co.,\nTokyo, Japan) at 48 h intervals (Hirabayashi et al. 2001).\nMice were superovulated as well with 5 IU eCG and 5\nIU hCG.\nto laboratory diet\nFourteen hours after the hCG injection, cumulus –oocyte\ncomplexes (COCs) were collected from the oviductal\nampullae of donor rats or mice with modiﬁed Krebs–\nRinger bicarbonate solution (mKRB) in Experiments 1 and\n3 (Toyoda & Chang 1974) and modiﬁed rat one-cell\nembryo culture medium (mR1ECM) in Experiments 2, 4\nand 5 (Oh et al. 1998) supplemented with 0.1% hyaluroni-\ndase (Sigma-Aldrich Corp., St Louis, MO, USA) respect-\nively. Two to ﬁve minutes later, the denuded oocytes were\nwashed three times with fresh mKRB or R1ECM and kept in\nthe same medium at 37 8C until being subjected to the\ntreatments. The time of killing of the donor animals was\ndeﬁned as 0 min, and it usually took 10 –15 min from the\nanimal killing to the preparation of denuded oocytes.\nMeasurement of p34cdc2 kinase activity\nThe p34cdc2 kinase assay was performed according to the\nmethod as described by Ito et al. (2001) with some modiﬁ-\ncations. In brief, the oocytes were washed several times\nwith PBS and placed into cell lysis buffer composed of\n20 mM Tris, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1%\nTriton X-100, 2.5 mM sodium pyrophosphate, 1 mM b-gly-\ncerophosphate, 1 mM Na3VO4, 1 mg/ml\nleupeptin (all\nchemicals from Cell Signaling Technology, Beverly, MA,\nUSA) and 1 mM phenylmethylsulfonyl ﬂuoride (Sigma-\nAldrich). The oocyte lysates were stored at 2 80 8C until\nthe ELISA assay.\n(MBL, Nagoya,\nThe lysate (ten oocytes/5 ml cell lysis buffer) was mixed\nwith 45 ml of kinase assay buffer-A composed of 25 mM\nHepes buffer\nJapan), 10 mM\nMgCl2 (MBL), 10% (v/v) mouse vimentin peptide solution\n(SLYSSPGGAYC) (MBL) and 0.1 mM ATP (Sigma-Aldrich),\nand the mixture was incubated for 30 min at 30 8C. The\nreaction was terminated by the addition of 200 ml PBS\ncontaining 50 mM EGTA (MBL). The phosphorylation of\nmouse vimentin peptides was detected using an ELISA\nanalysis (MESACUP cdc2 kinase assay kit\n(code no.\n5234); MBL). Data were expressed in terms of the strength\nof p34cdc2 kinase activity in denuded rat oocytes 15 min\n(Experiments 1, 3 and 4) or 45 min (Experiment 2) after\nanimal killing.\nAssessment of nuclear conﬁguration\nRat oocytes immediately after recovery and 30, 60, 90\nand 120 min after culture in mKRB were stained with\n5 mg/ml Hoechst 33342 (Sigma-Aldrich). Their nuclear\nconﬁguration was then assessed under UV light at 330–\n380 nm, according to the classiﬁcations reported pre-\nviously (Kato et al. 2001): MII, characterized by a meta-\nphase spindle with the ﬁrst polar body; AII, characterized\nby elongated spindle without the second polar body; and\nTII, characterized by elongated spindle with extrusion of\nthe second polar body.\nNuclear injection and PCC evaluation\nCumulus cells dispersed from COCs by the hyaluronidase\ntreatment were placed in mKRB (Experiments 1 and 3) or\nR1ECM (Experiments 2, 4 and 5) containing 22 mM Hepes\nand 5 mM NaHCO3 (abbreviated hereafter as Hepes-\nmKRB and Hepes-R1ECM respectively). An aliquot (2 ml)\nof the cumulus cell suspension was transferred to 10 ml of\nHepes-mKRB or Hepes-R1ECM supplemented with 12%\nw/v 360 kDa polyvinylpyrrolidone (ICN Pharmaceuticals,\nInc., Costa Mesa, CA, USA). Nuclei were removed from\ncumulus cells and gently aspirated in and out of the injec-\ntion pipette (7 –10 mm) until\ntheir nuclei were largely\ndevoid of visible cytoplasmic materials. Each nucleus was\ninjected into a separate enucleated oocyte within 5 min of\nits isolation, as described by Kimura & Yanagimachi\n(1995). One hour after the nuclear injection, the oocytes\nwere stained with 5 mg/ml Hoechst 33342 and evaluated\nfor incidence of PCC under UV light at 330–380 nm, as\ndescribed previously (Hirabayashi et al. 2003a,b). The\ninjected somatic cell nucleus was distinguished from the\noocyte nucleus by the position of the opening in the zona\npellucida at the nuclear microinjection site.\nRole of MPF in PCC induction of rat oocytes 173\nExperimental design\nExperiment 1: effect of in vitro aging\nThe effect of in vitro aging on the p34cdc2 kinase activity\nand the PCC incidence was investigated using rat oocytes.\nTen oocytes for each test were sampled for\nin vitro\np34cdc2 kinase assay immediately after recovery (15 min\ngroup), and after culture for 30 min (45 min group), 60 min\n(75 min group) or 105 min (120 min group) in mKRB at\n37 8C in 5% CO2 in air. Nuclear conﬁguration of\nthe\noocytes was also examined immediately after recovery\nand every subsequent 30 min up to 120 min. The other\noocytes were placed in Hepes-mKRB, and the cumulus\ncell nuclei were then injected into the oocytes up to\n120 min. The injected oocytes were divided into two\ngroups based on the time from the animal killing to NT\ncompletion (15 –45 min group and 46 – 120 min group).\nThe incidence of PCC was evaluated 1 h after the last\ninjection among each group.\nExperiment 2: effect of enucleation\nThe effect of oocyte enucleation on the p34cdc2 kinase\nactivity and the PCC incidence was investigated using\nboth rat and mouse oocytes. For the in vitro p34cdc2\nkinase assay, oocytes derived from the same donor rats\nand mice were allocated to the non-enucleation group\nand the enucleation group, and the non-enucleated and\nenucleated oocytes were sampled 45 min after the animal\nkilling. Enucleation was performed by aspiration of the\nmetaphase-plate with a part of the volume (,10–20%) of\ncytoplasm in Hepes-R1ECM containing\nsurrounding\n5 mg/ml cytochalasin B (Sigma-Aldrich). In addition to the\nnon-enucleation and enucleation groups, denuded rat\noocytes were allocated to a cytoplasm-removal group\n(a part of the cytoplasm was mechanically aspirated as a\nsham control.). All the treatments were completed within\n45 min. The oocyte samples were harvested at 45, 75, 90\nand 105 min (0, 30, 45 and 60 min after enucleation\nrespectively). For PCC evaluation, nuclear injection into\nnon-enucleated and enucleated mouse and rat oocytes,\nand into cytoplasm-removed rat oocytes, was completed\nwithin 45 min after animal killing.\nExperiment 3: effect of roscovitine treatment\nWe investigated whether the decreased level of p34cdc2\nkinase activity in non-enucleated rat oocytes negatively\naffected the PCC incidence of injected cell nuclei. Ros-\ncovitine (Sigma-Aldrich) dissolved at 10 mM in DMSO\nand stored at 2 80 8C, a speciﬁc inhibitor of p34cdc2\nkinase, was added to all media throughout the process of\noocyte recovery and the in vitro culture at the ﬁnal con-\ncentration of 150 mM, as reported by Lazar et al. (2002).\nOocytes were sampled for in vitro p34cdc2 kinase assay at\n15 (immediately after recovery), 45, 75 and 120 min as in\nExperiment 1. Because the 45 min treatment of rat oocytes\nwith 150 mM roscovitine decreased their p34cdc2 kinase\nactivity to an equivalent level of oocytes 120 min after in\nvitro aging or 60 min after enucleation, nuclear injection\ninto the roscovitine-treated oocytes was performed within\n45 min, and the incidence of PCC was then evaluated\nafter a further 1 h.\nExperiment 4: effect of MG132 treatment\nWe investigated whether the treatment of rat oocytes with a\nproteasome inhibitor, by which the p34cdc2 kinase activity\nwould be maintained, could support the PCC incidence of\nthe injected donor cell nuclei. MG132 (Sigma-Aldrich), an\ninhibitor which can block cyclin B1 degradation, was used\nat the ﬁnal concentration of 5 mM, as reported by Josefsberg\net al. (2000). Non-enucleated oocytes were exposed to\nMG132,\nthe process of oocyte recovery to\nin vitro culture or nuclear injection. Oocyte sampling for\nin vitro p34cdc2 kinase assay was performed at 15 min\n(immediately after recovery), 45, 75 and 120 min. Nuclear\ninjection into the MG132-treated oocytes and PCC evalu-\nation were performed as in Experiment 3.\nthroughout\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nIn the ﬁrst group, oocytes exposed to MG132 from oocyte\nrecovery to just before enucleation were used as recipi-\nents. From the result of Experiment 4, MG132 treatment is\nexpected to support PCC even in enucleated oocytes,\nbecause the p34cdc2 kinase activity was maintained at a\nhigh level during the treatment. The procedure of enuclea-\ntion was as described above. The cytoplasts were then\ninjected with cumulus cell nuclei. One hour later, a high\nincidence of PCC was expected.\nIn the second group,\noocytes with the extrusion of a second polar body were\nselected from 2 h cultures of non-treated oocytes, and\nthen enucleated and nuclear injected as well. The reduced\nincidence of PCC under a lowered MPF was expected. For\nboth groups, the oocytes 1 h after nuclear injection were\nactivated by two direct current pulses (100 V/mm, 99 ms)\nand 4 h treatment with 2 mM 6-dimethylaminopurine\n(6-DMAP)\nin mR1ECM medium. The\nreconstructed oocytes were cultured in 100 ml microdrops\nof mR1ECM at 37 8C in 5% CO2 in air, and number of\noocytes forming one or two pseudo-pronuclei and cleav-\ning to the two-cell stage was recorded at 6 and 24 h of\nculture respectively.\n(Sigma-Aldrich)\nStatistical analysis\nEach experiment had at least three replicates. Data on\np34cdc2 kinase activity were compared by ANOVA and\nFisher’s protected least signiﬁcant difference test using the\nStatView program (Abacus Concepts, Inc., Berkeley, CA,\nUSA). Data on PCC incidence and NT embryo develop-\nment were compared by Fisher’s exact probability test. A\nvalue of P , 0.05 was chosen as an indication of statisti-\ncal signiﬁcance.\nResults\nExperiment 1: effect of in vitro aging\nThe time-dependent change in p34cdc2 kinase activity in\nrat oocytes is shown in Fig. 1. The p34cdc2 kinase activity\nof\nrat oocytes decreased signiﬁcantly at 45 min (after\n30 min cultivation) and more signiﬁcantly at 75 min (after\n60 min cultivation). The decreased activity was maintained\nuntil 120 min (after 105 min cultivation). Probably due to\nthe decrease of p34cdc2 kinase activity, the release from\nmeiotic arrest of ovulated oocytes occurred in a time-\ndependent manner, as shown in Fig. 2. Within 75 min\nafter animal killing, more than half of the oocytes pro-\ngressed beyond the MII stage. At 135 min, most of the\noocytes were at either the AII or the TII stage. The pro-\nportion of in vitro-aged rat oocytes promoting PCC of the\ninjected cumulus cell nuclei is shown in Table 1. When\nnuclear injection was completed within 45 min from ani-\nmal killing, a positive sign of PCC was observed in 32.5%\nof the NT oocytes. On the other hand, PCC occurred in\nonly 2.6% of the oocytes into which cumulus cell nuclei\nwere injected between 46 and 120 min.\nExperiment 2: effect of oocyte enucleation\nThe activities of p34cdc2 kinase in non-enucleated and\nenucleated oocytes within 45 min were compared in rats\nand mice, as shown in Fig. 3. In both species, there was\na slight decrease in the p34cdc2 kinase activity after enu-\ncleation in both rat and mouse oocytes. A considerable\nRole of MPF in PCC induction of rat oocytes 175\ntreatment (enucleation or cytoplasm removal) is shown in\nTable 2. In rats, despite little reduction of p34cdc2 kinase\nactivity, few of the oocytes that either had a small portion\nof cytoplasm-removed (17.1%) or the non-treated controls\n(28.2%) could support PCC at 105 min after animal\nin the\nkilling. However, PCC was not observed at all\nfound in the p34cdc2 kinase activity\ndifference was\nbetween rat and mouse oocytes, because the mouse\noocytes carried a p34cdc2 kinase activity 2.5 times that of\nrat oocytes either immediately or up to 60 min after enu-\ncleation or cytoplasm removal and compared with non-\ntreated, age-matched controls, as shown in Fig. 4. The\nactivities of p34cdc2 kinase were measured in rat oocytes\nimmediately after or 60 min after one of three treatments\n(enucleated, cytoplasm-removed and non-treated), as\nshown in Fig. 4. Surprisingly, a higher p34cdc2 kinase\nactivity was detected in the oocytes immediately after\nenucleation, although the p34cdc2 kinase activity of\noocytes immediately after cytoplasm removal was not\nfrom that of non-treated oocytes. The p34cdc2\ndifferent\nin enucleated oocytes dramatically\nactivity\nkinase\ndecreased within 30 min after enucleation. At 45 min\nafter enucleation, the activity was further decreased and\nthereafter maintained until 60 min. However, the activity\nin non-treated or cytoplasm-removed oocytes did not sig-\nniﬁcantly differ by 60 min. The proportion of rat and\nmouse oocytes promoting PCC in relation to mechanical\nenucleated oocytes where p34cdc2 kinase activity was sig-\nniﬁcantly decreased at 60 min after completion of enu-\ncleation. In contrast, PCC of mouse cumulus cell nuclei\noccurred in most of the mouse oocytes before enucleation\n(96.7%) and even after enucleation (93.1%).\nExperiment 3: effect of roscovitine treatment\nThe time-dependent decrease of p34cdc2 kinase activity in\nroscovitine-treated rat oocytes is shown in Fig. 5. The\nactivity\nlinearly\nin roscovitine-treated oocytes was\ndecreased until 75 min and maintained until 120 min.\nAt 45, 75 and 120 min, each activity in the roscovitine-\ntreated oocytes was signiﬁcantly lower than that of non-\ntreated control oocytes. The result of PCC incidence in the\noocytes treated with or without roscovitine for 45 min is\nshown in Table 3. None of the roscovitine-treated oocytes\npromoted PCC, although 41.7% of non-treated oocytes\npromoted it.\nExperiment 4: effect of MG132 treatment\nThe activities of p34cdc2 kinase in MG132-treated rat\noocytes are shown in Fig. 6. The level of p34cdc2 kinase\nactivity still decreased in the MG132-treated oocytes\nafter 30 min of culture, but were signiﬁcantly greater\nthan in control oocytes. The p34cdc2 kinase activity was\nmaintained during the further incubation up to 105 min.\nThe kinase activities of MG132-treated oocytes at 60\nand 105 min cultivation were signiﬁcantly greater than\nthose of control oocytes. As shown in Table 4, the pro-\nportion of MG132-treated oocytes promoting PCC\n(66.0%) was higher than that of the non-treated oocytes\n(46.5%) when the nuclear\ninjection was completed\nwithin 45 min.\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nthe reconstructed embryos\nAfter oocyte activation with the electrical stimulus plus 6-\nDMAP,\nfrom both groups\n(MG132-treated MII oocytes and non-treated TII oocytes\nat the enucleation) formed one or two pseudo-pronuclei\n(91.7 and 87.5% in total respectively). After activation, a\ngreater proportion of oocytes enucleated at the MII stage\ntended to form two pseudo-pronuclei compared with\nthose enucleated at the TII stage. A signiﬁcantly higher\nproportion of the NT embryos in the MG132-treated group\nvs the non-treated group (59.1 vs 7.1%) developed to the\ntwo-cell stage.\nDiscussion\nRat oocytes have poor potential\nfor promoting PCC of\ninjected somatic cell nuclei, and the rate was markedly\ndecreased in aged or enucleated oocytes (Hirabayashi\nIn the present study, rat oocytes into\net al. 2003a,b).\nwhich cumulus cell nuclei were injected within 45 min\nRole of MPF in PCC induction of rat oocytes 177\nthe non-treated oocytes during the same period, may\ninﬂuence the outcome of PCC. The high concentration of\nroscovitine affects the other cyclin-dependent protein\nkinases such as cdk2 (De Azevado et al. 1997). The\nunknown pathway and/or chemical toxicity of roscovitine\ncannot be ruled out\nfor complete suppression of PCC\ninduction. Moreover, the level of activity in rat oocytes\nwas less than half compared with that in mouse oocytes,\nwhich had a high potential for promoting PCC. Judging\nfrom these results, we estimated that the high activity of\np34cdc2 kinase is associated with promoting PCC of\ninjected somatic cell nuclei, and the low level of p34cdc2\nkinase activity is one of the reasons for the poor potential\nto promote PCC in rat oocytes compared with that of\nmouse oocytes. The reason why the potential\nlevel of\np34cdc2 kinase activity is different between mouse and rat\noocytes remains unclear. Our previous study indicated, at\nleast, that the PCC-inducing potential is determined by\nthe origin of the recipient cytoplasm, not by the origin of\nin the interspecies NT experiment\nthe donor nucleus,\nbetween mouse and rat\nIn\n(Hirabayashi et al. 2003a).\nsomatic cell division, dissolution of the nuclear envelope\noccurs during G2/M transition in mitosis (Nurse 1990) and\nis induced by phosphorylation of lamin, one of the major\nnuclear membrane proteins (Peter et al. 1990). In oocytes,\np34cdc2 kinase had the potential to phosphorylate lamins,\ninducing meiotic breakdown of the nuclear envelope and\nthe meiotic resumption (Dessev & Goldman 1988). There-\nfore, it is possible that p34cdc2 kinase plays an important\nrole in the destruction of the nuclear membrane of the\ninjected somatic cell through phosphorylation of lamin.\nOne possible pathway known for inactivation of p34cdc2\nkinase is the calmodulin-dependent pathway via Ca2þ.\nWe have reported that the Ca2þ-dependent pathway trig-\ngers the degradation of cyclin B and results in inactivation\nof p34cdc2 kinase in porcine oocytes (Ito et al. 2003,\n2004a). In Xenopus oocytes (Lorca et al. 1994, Morin et al.\n1994), activation of calmodulin-dependent protein kinase\nII induces the degradation of cyclin B via the ubiquitin\nproteasome pathway. Based on these literature sources\nand preliminary data, we have investigated the effect of a\nproteasome inhibitor MG132 on p34cdc2 kinase activity\nand PCC incidence. The PCC incidence in rat oocytes was\nsigniﬁcantly improved by MG132 treatment (Table 4). This\nresult matched well with the report by Josefsberg et al.\n(2000), who demonstrated that MG132-treated rat oocytes\ncontained a high amount of cyclin B1.\nIn the present\nstudy, MG132 also provided a suppressive effect on inacti-\nvation of p34cdc2 kinase (Fig. 6). Use of ALLN, a neutral\ncysteine protease inhibitor, was also slightly effective for\nimproving the PCC incidence in rat oocytes (Hirabayashi\net al. 2003b). Thus, the Ca2þ-dependent degradation path-\nway, which may induce a decrease in the cyclin B1 level,\nis activated in aged rat oocytes, which results in a low\nlevel of p34cdc2 kinase activity, and a low potential\nto promote PCC. Our unpublished data (M Hirabayashi\nand J Ito, unpublished data) showing that the decreased\nIn mouse oocytes, PCC was initiated at\npromoted PCC (frequency 28–47% in Tables 1 –4), but\nthe incidence of PCC decreased with oocyte aging with a\nsigniﬁcant decrease in p34cdc2 kinase activity (Table 1,\nFig. 1).\nleast\n30 min after nuclear injection and lasted until 2 h (Chen\net al. 2004), therefore the timing of PCC observation in\nthe present study (60 min after nuclear injection) might be\noptimal. When oocytes are treated with roscovitine,\nwhich decreases the level of p34cdc2 kinase activity, PCC\nformation was never observed even though nuclear injec-\ntion is completed within 45 min. Roscovitine is widely\nused as cyclin-dependent kinase inhibitor\nto suppress\np34cdc2 kinase in rodent oocytes (Phillips et al. 2002).\nTherefore, the rapid decrease of p34cdc2 kinase activity in\nthe roscovitine-treated oocytes found between 45 and\n75 min (Fig. 5), when compared with the slow decrease in\na Oocytes were treated with 5 mM MG132 until just before enucleation after oocyte recovery.\nb Oocytes were cultured for about 2 h and only oocytes with a completely extruded polar body (the TII stage) were used.\nc Percentage of reconstructed embryos with 1 or 2 pronuclei.\nd – e Different superscripts denote signiﬁcant difference (P , 0.05).\nconcentration of extracellular free calcium in the handling\nmedium partially inhibits spontaneous activation of rat\noocytes, support this hypothesis.\nThe other pathway for inactivation by phosphorylation\nof 14Thr and 15Tyr of p34cdc2 kinase has been reported in\naged porcine oocytes (Kikuchi et al. 2000). In the same\nstudy, it has been reported that caffeine suppressed the\nfrom MPF to pre-MPF (14Thr/15Tyr-phosphorylated\nshift\np34cdc2 kinase þ cyclin B1) in the aged porcine oocytes.\nOkadaic acid is also known as a protein phosphatase\ninhibitor which prevents phosphorylation of 14Thr and\n15Tyr of p34cdc2 kinase (Rime et al. 1995). In our prelimi-\nnary study, PCC incidence of injected cumulus cell nuclei\nwas slightly improved in rat oocytes treated with okadaic\nacid (M Hirabayashi and J Ito, unpublished data). There-\nfore, the phosphorylation of p34cdc2 kinase as well as the\ncyclin degradation may be involved in spontaneous acti-\nvation of rat oocytes during in vitro culture.\nThe level of p34cdc2 kinase activity in non-treated rat\noocytes was similar to that of cytoplasm-removed or enu-\ncleated oocytes in the case where the oocytes were\nsampled immediately after the treatment\n(Fig. 4). The\nmechanical puncture of the oocyte membrane and the\naspiration of the cytoplasm or karyoplast did not inﬂuence\nthe p34cdc2 kinase activity immediately after the enuclea-\ntion procedure beyond the level expected from the rela-\ntive volume of aspirated cytoplasm (, 10–20%). These\nresults strongly suggest that active p34cdc2 kinase is not\naccumulated near the metaphase spindles in rat oocytes.\nThere was signiﬁcant difference in the p34cdc2 kinase\nactivity between 45 and 75 min in both non-treated and\nenucleated groups (Fig. 4). A dramatic decrease in the\nactivity of p34cdc2 kinase was observed in the rat oocytes\nenucleated and cultured for 30 min compared with the\nnon-treated group (Fig. 4), suggesting that\nthe oocyte\nmetaphases per se are involved in the maintenance of\np34cdc2 kinase activity. We have previously reported in\nporcine oocytes that activation of p34cdc2 kinase was\ndependent on the synthesis of cyclin B and that the syn-\nthesis was induced by the activation of mitogen-activated\nprotein (MAP) kinase (Ito et al. 2004b). The MAP kinase\nsuppressed Myt1 activity that phosphorylated p34cdc2\nkinase at 14Thr and 15Tyr (Mueller et al. 1995, Palmer et al.\n1998). Because activated MAP kinase is localized on the\nmeiotic spindle in porcine oocytes (Lee et al. 2000), it is\npossible that enucleation results in the loss of MAP kinase\nfrom the recipient oocytes.\ninjection into the MG132-treated, MII-enu-\ncleated rat oocytes resulted in a higher development to\nthe two-cell stage when compared with non-treated, TII-\nenucleated rat oocytes after 24 h cultivation (Table 5).\nThe favorable circumstance of recipient cytoplasm with\na high MPF activity must induce PCC and promote sub-\nsequent development of NT embryos. Wakayama &\nYanagimachi\nthe injected\n(2001) reported in mice that\ncumulus cell nuclei undergoing PCC transformation to\nthe two pseudo-pronuclei and such NT embryos are able\nto develop to full-term. In contrast, a different situation\nhas been reported in the cloning of large domestic ani-\nmals. The somatic cell NT in the livestock species is suc-\ncessfully reproducible when enucleated and activated\noocytes with a low MPF activity are used as recipient\ncytoplasm (Baguisi et al. 1999, Kurosaka et al. 2002,\nShin et al. 2002). Such a difference between rodents\nand large domestic animals remains unclear, but our\nthat\npresent data suggest\ninduction of PCC in recon-\nstructed embryos\nrelated to the high activity of\nis\np34cdc2 kinase, which seems to play a key role in the\nsuccessful rat cloning.\nthe decreased level of p34cdc2 kinase\nactivity in aged or enucleated rat oocytes is responsible\nfor their inability to promote PCC of microinjected donor\ncell nuclei. Inhibition of p34cdc2 kinase inactivation by\nchemicals such as MG132 is in part effective for rat\noocytes to promote PCC and cleavage. These data may\nexpand our existing knowledge on characteristics of rat\noocytes and contribute to the development of a rat NT\nprotocol using G0/G1-stage donor cells.\nAcknowledgements\nThe authors thank Mrs A Ishikawa and Miss A Takata for their\ntechnical assistance. This work was supported in part by\nGrant-in-Aids for the Scientiﬁc Research (No. 15082211, No.\n16300139; to M H and S H), and the 21st Century COE Pro-\ngram (to S H) from the Ministry of Education, Culture, Sports,\nScience and Technology of Japan, and a Research Fellowship\nfor Young Scientists (No. 8254; to J I.) from the Japan Society\nfor the Promotion of Science.\nReferences\nDessev G & Goldman R 1988 Meiotic breakdown of nuclear envel-\nope in oocytes of Spisula solidissima involves phosphorylation and\nrelease of nuclear lamin. Developmental Biology 130 543–550.\nGalli C, Lagutina I, Crotti G, Colleoni S, Turini P, Ponderato N et al.\n2003 Pregnancy: a cloned horse born to its dam twin. Nature 424\n635.\nHirabayashi M, Ito K, Sekimoto A, Hochi S & Ueda M 2001 Pro-\nduction of transgenic rats using young Sprague–Dawley females\ntreated with PMSG and hCG. Experimental Animals 50 365–369.\nHirabayashi M, Kato M, Ishikawa A & Hochi S 2003a Factors inﬂuen-\ncing chromosome condensation and development of cloned rat\nembryos. Cloning and Stem Cells 5 35–42.\nIto J & Shimada M 2005 Timing of MAP kinase inactivation effects\non emission of polar body in porcine oocytes activated by Ca2þ\nionophore. Molecular Reproduction and Development 70 64– 69.\nIto J, Shimada M & Terada T 2001 Progression of nuclear maturation\nand p34cdc2 kinase activity in porcine oocytes during in vitro cul-\nture in different media. Journal of Mammalian Ova Research 18\n39– 43.\nRole of MPF in PCC induction of rat oocytes 179\nKikuchi K, Naito K, Noguchi J, Shimada A, Kaneko H, Yamashita M\net al. 2000 Maturation/M-phase promoting factor: a regulator of\naging in porcine oocytes. Biology of Reproduction 63 715– 722.\nKimura Y & Yanagimachi R 1995 Mouse oocytes injected with testi-\ncular spermatozoa or round spermatids can develop into normal\noffspring. Development 121 2397–2405.\nRime H, Jessus C & Ozon R 1995 Tyrosine phosphorylation of\np34cdc2 is regulated by protein phosphatase 2A in growing imma-\nture Xenopus oocytes. Experimental Cell Research 219 29– 38.\nSherwood SW, Kung AL, Roitelman J, Simoni RD & Schimke RT\n1993 In vivo inhibition of cyclin B degradation and induction\nof cell-cycle arrest in mammalian cells by the neutral cysteine\nprotease\ninhibitor N-acetylleucylleucylnorleucinal. PNAS 90\n3353–3357.",
         "J. Ito,M. Hirabayashi,Megumi Kato,A. Takeuchi,Mayumi Ito,M. Shimada,S. Hochi",
         null
        ],
        [
         "48",
         "002cc4c2e457c3cc0fb0f704d0483a003e8a0575",
         null,
         null,
         null,
         null,
         null,
         null,
         "L. Halbert,J. Kaneene,L. Mansfield,P. Ruegg,L. Warnick,S. Wells,C. Fossler,A. Campbell,Angela M Geiger-Zwald",
         "captcha"
        ],
        [
         "49",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "",
         "",
         "",
         "",
         "",
         "",
         "Y. Kuznetsov,W. Govaerts,E. Doedel,Annick Dhooge",
         null
        ]
       ],
       "shape": {
        "columns": 9,
        "rows": 100
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>pdf_abstract</th>\n",
       "      <th>pdf_introduction</th>\n",
       "      <th>pdf_results</th>\n",
       "      <th>pdf_discussion</th>\n",
       "      <th>pdf_methods</th>\n",
       "      <th>pdf_text</th>\n",
       "      <th>author_list</th>\n",
       "      <th>problem</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>Abstract Purpose: Synovial sarcoma is a soft t...</td>\n",
       "      <td>Synovial sarcoma is the most frequent soft-tis...</td>\n",
       "      <td>Results\\nSynovial sarcoma cell lines expressed...</td>\n",
       "      <td>Discussion\\nFGFs may activate genetic programs...</td>\n",
       "      <td>Materials and Methods\\nTumor tissues were obta...</td>\n",
       "      <td>Abstract Purpose: Synovial sarcoma is a soft t...</td>\n",
       "      <td>T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>Replica shufﬂed versions of interactive decode...</td>\n",
       "      <td>This work may not be copied or reproduced in w...</td>\n",
       "      <td>D. Simulation results\\nIII. ITERATIVE DECODING...</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>Replica shufﬂed versions of interactive decode...</td>\n",
       "      <td>Juntan Zhang,M. Fossorier</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td>The  roots  of  Astragalus  membranaceus BUNGE...</td>\n",
       "      <td>M)  induced  a  similar  sustained  contractio...</td>\n",
       "      <td>DISCUSSION\\nThe present study ﬁrst and detaile...</td>\n",
       "      <td>MATERIALS AND METHODS\\nPlant  Material  and  P...</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td>Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>In this report, we use zebraﬁsh as a model sys...</td>\n",
       "      <td>INTRODUCTION\\nCD146, also known as MUC18, A32,...</td>\n",
       "      <td>A cDNA clone (clone number, 3512;\\nGenBank acc...</td>\n",
       "      <td></td>\n",
       "      <td>Total RNA was collected from ze-\\nbraﬁsh embry...</td>\n",
       "      <td>In this report, we use zebraﬁsh as a model sys...</td>\n",
       "      <td>Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td>Protein kinase C (PKC)2 activation is an impor...</td>\n",
       "      <td>EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...</td>\n",
       "      <td>DISCUSSION\\nResults from the present study dem...</td>\n",
       "      <td>EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td>M. Westfall,A. M. Lee,D. Robinson</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>004d7fddf20a6932e6391fc9dcf9dca8c52abcaa</td>\n",
       "      <td>There are several factors that inﬂuence the ra...</td>\n",
       "      <td>INTRODUCTION\\nIn the recent years, a lot of ef...</td>\n",
       "      <td>RESULTS\\nThe estimated relative uncertainty in...</td>\n",
       "      <td>DISCUSSION\\nMany interesting results can be fo...</td>\n",
       "      <td>Introductory remarks\\nIn the paper by Ha˚kanss...</td>\n",
       "      <td>There are several factors that inﬂuence the ra...</td>\n",
       "      <td>M. Båth,M. Håkansson,S. Börjesson,S. Kheddache...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>004dbb1be00568e82684981115029880f24f1967</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Y. Dewoody,J. A. Dewoody</td>\n",
       "      <td>captcha</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>004e863417f890203b7a5464569bfe5ea0b7b35c</td>\n",
       "      <td>The innate immune system serves\\nas an initial...</td>\n",
       "      <td>At first glance, NKT cells may\\nbe easily mist...</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>The innate immune system serves\\nas an initial...</td>\n",
       "      <td>L. Kaer,S. Joyce</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>005032c8f2b4ae916505f4fb43c2d638b6b72b78</td>\n",
       "      <td>Abstract Purpose: t(12;21)(p13; q22), present ...</td>\n",
       "      <td>The t(12;21)(p13;q22) occurs in f25% of childh...</td>\n",
       "      <td>Results\\nThe mRNA expression levels of TEL, AM...</td>\n",
       "      <td>Discussion\\nIn the present study, we examined ...</td>\n",
       "      <td>Materials and Methods\\nPatient samples. Bone m...</td>\n",
       "      <td>Abstract Purpose: t(12;21)(p13; q22), present ...</td>\n",
       "      <td>W. Stams,M. D. den Boer,H. Beverloo,J. Meijeri...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>0050af201c04b89ad09acc60e24071d60aee1232</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>M. Infanger,M. Shakibaei,P. Kossmehl,Siri M. H...</td>\n",
       "      <td>captcha</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>100 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     paperId  \\\n",
       "0   0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1   000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2   000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3   0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4   00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "..                                       ...   \n",
       "95  004d7fddf20a6932e6391fc9dcf9dca8c52abcaa   \n",
       "96  004dbb1be00568e82684981115029880f24f1967   \n",
       "97  004e863417f890203b7a5464569bfe5ea0b7b35c   \n",
       "98  005032c8f2b4ae916505f4fb43c2d638b6b72b78   \n",
       "99  0050af201c04b89ad09acc60e24071d60aee1232   \n",
       "\n",
       "                                         pdf_abstract  \\\n",
       "0   Abstract Purpose: Synovial sarcoma is a soft t...   \n",
       "1   Replica shufﬂed versions of interactive decode...   \n",
       "2   This study was designed to investigate the eff...   \n",
       "3   In this report, we use zebraﬁsh as a model sys...   \n",
       "4   Cardiac troponin I is a phosphorylation target...   \n",
       "..                                                ...   \n",
       "95  There are several factors that inﬂuence the ra...   \n",
       "96                                               None   \n",
       "97  The innate immune system serves\\nas an initial...   \n",
       "98  Abstract Purpose: t(12;21)(p13; q22), present ...   \n",
       "99                                               None   \n",
       "\n",
       "                                     pdf_introduction  \\\n",
       "0   Synovial sarcoma is the most frequent soft-tis...   \n",
       "1   This work may not be copied or reproduced in w...   \n",
       "2   The  roots  of  Astragalus  membranaceus BUNGE...   \n",
       "3   INTRODUCTION\\nCD146, also known as MUC18, A32,...   \n",
       "4   Protein kinase C (PKC)2 activation is an impor...   \n",
       "..                                                ...   \n",
       "95  INTRODUCTION\\nIn the recent years, a lot of ef...   \n",
       "96                                               None   \n",
       "97  At first glance, NKT cells may\\nbe easily mist...   \n",
       "98  The t(12;21)(p13;q22) occurs in f25% of childh...   \n",
       "99                                               None   \n",
       "\n",
       "                                          pdf_results  \\\n",
       "0   Results\\nSynovial sarcoma cell lines expressed...   \n",
       "1   D. Simulation results\\nIII. ITERATIVE DECODING...   \n",
       "2   M)  induced  a  similar  sustained  contractio...   \n",
       "3   A cDNA clone (clone number, 3512;\\nGenBank acc...   \n",
       "4   EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...   \n",
       "..                                                ...   \n",
       "95  RESULTS\\nThe estimated relative uncertainty in...   \n",
       "96                                               None   \n",
       "97                                                      \n",
       "98  Results\\nThe mRNA expression levels of TEL, AM...   \n",
       "99                                               None   \n",
       "\n",
       "                                       pdf_discussion  \\\n",
       "0   Discussion\\nFGFs may activate genetic programs...   \n",
       "1                                                       \n",
       "2   DISCUSSION\\nThe present study ﬁrst and detaile...   \n",
       "3                                                       \n",
       "4   DISCUSSION\\nResults from the present study dem...   \n",
       "..                                                ...   \n",
       "95  DISCUSSION\\nMany interesting results can be fo...   \n",
       "96                                               None   \n",
       "97                                                      \n",
       "98  Discussion\\nIn the present study, we examined ...   \n",
       "99                                               None   \n",
       "\n",
       "                                          pdf_methods  \\\n",
       "0   Materials and Methods\\nTumor tissues were obta...   \n",
       "1                                                       \n",
       "2   MATERIALS AND METHODS\\nPlant  Material  and  P...   \n",
       "3   Total RNA was collected from ze-\\nbraﬁsh embry...   \n",
       "4   EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...   \n",
       "..                                                ...   \n",
       "95  Introductory remarks\\nIn the paper by Ha˚kanss...   \n",
       "96                                               None   \n",
       "97                                                      \n",
       "98  Materials and Methods\\nPatient samples. Bone m...   \n",
       "99                                               None   \n",
       "\n",
       "                                             pdf_text  \\\n",
       "0   Abstract Purpose: Synovial sarcoma is a soft t...   \n",
       "1   Replica shufﬂed versions of interactive decode...   \n",
       "2   This study was designed to investigate the eff...   \n",
       "3   In this report, we use zebraﬁsh as a model sys...   \n",
       "4   Cardiac troponin I is a phosphorylation target...   \n",
       "..                                                ...   \n",
       "95  There are several factors that inﬂuence the ra...   \n",
       "96                                               None   \n",
       "97  The innate immune system serves\\nas an initial...   \n",
       "98  Abstract Purpose: t(12;21)(p13; q22), present ...   \n",
       "99                                               None   \n",
       "\n",
       "                                          author_list  problem  \n",
       "0   T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...     None  \n",
       "1                           Juntan Zhang,M. Fossorier     None  \n",
       "2   Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...     None  \n",
       "3   Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....     None  \n",
       "4                   M. Westfall,A. M. Lee,D. Robinson     None  \n",
       "..                                                ...      ...  \n",
       "95  M. Båth,M. Håkansson,S. Börjesson,S. Kheddache...     None  \n",
       "96                           Y. Dewoody,J. A. Dewoody  captcha  \n",
       "97                                   L. Kaer,S. Joyce     None  \n",
       "98  W. Stams,M. D. den Boer,H. Beverloo,J. Meijeri...     None  \n",
       "99  M. Infanger,M. Shakibaei,P. Kossmehl,Siri M. H...  captcha  \n",
       "\n",
       "[100 rows x 9 columns]"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "display(textdata)\n",
    "pd.DataFrame([pdf for result in results for pdf in result])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "fieldsOfStudy",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "venue",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorName",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "citationCount",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "influentialCitationCount",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "publicationDate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "publicationYear",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "referenceIds",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "6408f879-8c4d-4518-a900-108e0e072359",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "https://aacrjournals.org/clincancerres/article-pdf/11/7/2702/1962059/2702-2712.pdf",
         "Biology,Medicine",
         "Clinical Cancer Research",
         "T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. Nishijo,K. Shibata,Y. Shima,S. Nagayama,T. Katagiri,Yusuke Nakamura,Takashi Nakamura,J. Toguchida",
         "14839966,10239302,2000448727,3201672,7147569,2066435363,31907929,4360342,7367122,144863658,145103616,3630533",
         "88",
         "6.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "http://www.merl.com/publications/docs/TR2005-063.pdf",
         "Computer Science,Mathematics",
         "IEEE Transactions on Communications",
         "Juntan Zhang,M. Fossorier",
         "3193350,48891339",
         "319",
         "26.0",
         "2005-03-07",
         "2005",
         "17d0afe4f814e5abc6252e572e3b039082bf6f07,02464a444975fdfbef09c40e9e6d140b2f10d213,5adb4ba3b3717d86d507e57ec7b04ca91b0daa32,3870f98d6575420d9e3d02a5f299fa483123caaa,610cd4607f39edb48ce8d8dda982b04cb9cb1d75,4f773884c351152c49b1111d98c07446b1543dc2,e28fc01c3f8ca4151abf9b61296b56dc318d3b26,fbf85b75d9d3d8c224de034005d2f2cbc7c4fac2,64e7badd52b7841ebc25d09e6b38e6d7dbea5f38,293620136b122dbfb258fe95fd45ca85c814236c,0bf5d709b14dd074522d96d2ffb4474b3184b984,444e24fd9dff66dfafeb70914b6b3c83e9003853",
         "Low-Density Parity-Check Codes Turbo decoder architectures for low-density parity-check codes Shuffled belief propagation decoding A parallel decoder for low latency decoding of turbo product codes Parallel versus sequential updating for belief propagation decoding High throughput low-density parity-check decoder architectures Good error-correcting codes based on very sparse matrices Near optimum error correcting coding and decoding: turbo-codes Multiple turbo codes for deep-space communications Optimal decoding of linear codes for minimizing symbol error rate (Corresp.) Generic Architecture for LDPC Codes Decoding Optimal decoding of linear codes for minimizing symbol error rate"
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "https://www.jstage.jst.go.jp/article/bpb/28/8/28_8_1450/_pdf",
         "Medicine",
         "Biological and Pharmaceutical Bulletin",
         "Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,Jian Sun,Q. Xia,K. Bian",
         "11809650,6617669,123635715,11305673,2152146096,144130337,2144753879",
         "24",
         "2.0",
         "2005-08-01",
         "2005",
         "",
         ""
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/dvdy.20220",
         "Biology,Medicine",
         "Developmental Dynamics",
         "Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V. Sukhatme",
         "35690923,2114125286,2113252574,2134773,3658233",
         "53",
         "2.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "http://www.jbc.org/article/S0021925820589394/pdf",
         "Chemistry,Medicine",
         "Journal of Biological Chemistry",
         "M. Westfall,A. M. Lee,D. Robinson",
         "5025983,49249911,48706510",
         "70",
         "3.0",
         "2005-12-16",
         "2005",
         "",
         ""
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "http://dl.acm.org/ft_gateway.cfm?id=1047480&type=pdf",
         "Computer Science",
         "Technical Symposium on Computer Science Education",
         "Susan Bergin,R. Reilly",
         "145985158,2688685",
         "287",
         "26.0",
         "2005-02-23",
         "2005",
         "b8123b4807efdee5d3a408e6f862ff438c642127,afae3da8f9b6e9b644a5dc695b50796e020d4194,cbaddfc496cebf6dc683ed8aad2306a359aa58ce,839b94477a293d0bd10acd2411589fd0cad2512c,259ec7be9d7bbb5b633978c72cf2a93bd86b25f9,78033d2947f98bf7189c5522c78251d4f9bf2eec,499da8c546288d9b12dd5acddc6d1cba3c9a5f56,5453190a0090efe599f136df5a8a2a529ad6ac9a,978a0c742ebfcbb41b9b9b472519512d9759accd,4ea72d8b5f946427398474c32f4bcff74b5f871b,ef3cf9f8bf4aa787afbdb0eaec91b1cc808a0898,3a6651c691c57e1c5badd03bb9bcb2b565832159,500a21438777d026ea95489c5a77d1cb1ac5d5a8,dbae45bc40eeeac237f360c5006f40611360b856,0ae8f4656278b6c683984cd60f3c3a0606da5bc1,f60e8ea65110900e1f4c3816bc720591fcc41f79,fdc5a2d599b5f9c4990f3ab4dc3ce86ae8251627",
         "The impact of prior experience in an information technology programming course sequence Predictors of success and failure in a CS1 course A multi-national, multi-institutional study of assessment of programming skills of first-year CS students The effect of student attributes on success in programming Contributing to success in an introductory computer science course: a study of twelve factors Does it help to have some programming experience before beginning a computing degree program? Factors affecting performance in first-year computing The effect of a constructivist learning environment for field-dependent/independent students on achievement in introductory computer programming What best predicts computer proficiency? Predicting student performance in a beginning computer science class Predicting the success of freshmen in a computer science major Predicting student success in an introductory programming course Predicting success in a first programming course Investigating the relationship between the development of abstract reasoning and performance in an introductory programming class Factors affecting course outcomes in introductory programming An Overview of the Integration of Problem Based Learning into an existing Computer Science Programming Module Discovering the likely performance of novice programmers on an introductory programming module Differential Performance by gender and achievement in the Certificate Examination Results 2000/2001: Final Report, Dublin: National Council for Curriculum and Assessment Examining Gender Cognitive skills needed in computer programming speciﬁc cognitive skills: as measured by an in-house cognitive test previous academic and computer experience: as measured by performance in the Irish Leaving Certiﬁcate (LC) examinations in"
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.ATV.0000168413.29874.d7",
         "Medicine,Biology",
         "Arteriosclerosis, Thrombosis and Vascular Biology",
         "E. Egorina,M. Sovershaev,G. Bjørkøy,F. Gruber,J. Olsen,B. Parhami-Seren,K. Mann,B. Østerud",
         "8887769,4962786,11414707,50148632,33502527,1398546666,3162218,5466077",
         "148",
         "2.0",
         "2005-07-01",
         "2005",
         "65eeaee9e552393e75a743c0344ebe4581a6e6a7,8bf514513122d0af4d0aa2b9789994b4211bd7cd,05d6048fa796174028c92893168a0773e927e454,8be482509b14152ffdb18fe3ae5dc7d1afb009a5,c847dd5549f0c843199c1a50718d0f139aaefcd7,ac3cc136968e89551fc3206b23c3bfb1cc150bbd,4498654bf9d23ded98cee2eb667944f9c27da787,832d3e34e1d44cc5c92702ff3e7e97fa6d43ee44,154f2466119853b7ba397eee95c14f010572275d,41f73ad3c17b7c021670dda4e2e1e0e948d5f90c,e6a10b6e88a706227967bae80080553b1e2b51f5,8527e6d03a0f79a038fc130a4344fa4853c8335a,49e92bb9ad78b8b307e99944834e6d1bc631297f,be3f99f27c1ad7a8cf0a8eca7e8b549ef1b6a6dd,4511c51ef9c78ef929a5c9766af97db701da82af,4ca60e84496caca48756bb5b4f6f24236ca17c8f,d24472339b51de01a39c49b3b201c8e43b16f242,01bdc51fe1630bc406cf0661f7ac67200b449966,0be3116d5897256dd59fef2dcf2ba5401359a10d,0db9a7f606d2539f5e1bd2587ba2c0cb7166d66d,2b4563c2e99ab1208d889c22ad7df414c183dd3e,886b5036def2f1c34f63a621738e9ebfe40332ce,18f594d1e008e6bc4e0eaaf7e771acd7f9b6c6ff,ed3087c11d9f60b01ce54b720df2a2420749e77c,516f18c8244c665e450172bbe183c4286bbaa1d4,25405d6b24240aa2adb1f7b7049b80efeb40ce3e,878ce5ee726c6651d087291f37cc046a9f980d54,4e7a537a06f4b75c72bfb551c9a18f99d9679cef,551e55506bd0f8cfc13f22816c9b20315a51970c,48cc95821fe443d86a1b63d0dbd1486a0b012ce1,e3acf2654d08a2006fd32dce370e4fa4e5653b22",
         "The production and availability of tissue thromboplastin in cellular populations of whole blood exposed to various concentrations of endotoxin. An assay for detection of endotoxin. Tissue factor activity in whole blood. Nicotinamide inhibits endotoxin‐induced monocyte tissue factor expression Cellular activation responses in blood in relation to lipid pattern: healthy men and women in families with myocardial infarction or cancer Tissue factor activity in human monocytes is regulated by plasma: implications for the high and low responder phenomenon Role of Platelet P-Selectin and CD40 Ligand in the Induction of Monocytic Tissue Factor Expression Eicosanoids and Iso-Eicosanoids: Constitutive, Inducible and Transcellular Biosynthesis in Vascular Disease Cytokine gene polymorphism and heart transplant rejection. Platelet-derived growth factor-BB and monocyte chemotactic protein-1 induce human peripheral blood monocytes to express tissue factor. Association of smooth muscle cell tissue factor with caveolae. Manifestation of cryptic fibroblast tissue factor occurs at detergent concentrations which dissolve the plasma membrane A novel biological effect of platelet factor 4 (PF4): enhancement of LPS‐induced tissue factor activity in monocytes Th1 and Th2 T-helper cells exert opposite regulatory effects on procoagulant activity and tissue factor production by human monocytes. Granulocytes enhance LPS‐induced tissue factor activity in monocytes via an interaction with platelets Cathepsin G-dependent platelet stimulation by activated polymorphonuclear leukocytes and its inhibition by antiproteinases: role of P-selectin-mediated cell-cell adhesion. Relations between factor VIIa binding and expression of factor VIIa/tissue factor catalytic activity on cell surfaces. Platelet activating factor enhancement of lipopolysaccharide‐induced tissue factor activity in monocytes: requirement of platelets and granulocytes Dysregulation of in vitro cytokine production by monocytes during sepsis. The role of arachidonic acid release and lipoxygenase pathway in lipopolysaccharide-induced thromboplastin activity in monocytes. In vitro production of IL 1β, IL 1α, TNF and IL 2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation Functional tissue factor is entirely cell surface expressed on lipopolysaccharide-stimulated human blood monocytes and a constitutively tissue factor-producing neoplastic cell line Localization of tissue factor in the normal vessel wall and in the atherosclerotic plaque. Cytokine-induced procoagulant activity in monocytes and endothelial cells. Further enhancement by cyclosporine. Tumor necrosis factor induces tissue factor-like activity in human leukemia cell line U937 and peripheral blood monocytes. Enhancement of mononuclear procoagulant activity by platelet 12-hydroxyeicosatetraenoic acid. The stimulatory effect of platelets and platelet membranes on the procoagulant activity of leukocytes. The Role of Protein Synthesis on the Generation of Tissue Factor Activity by Leukocytes Monocytes. The high responder phenomenon: enhancement of LPS induced tissue factor activity in monocytes by platelets and granulocytes. Selective cellular expression of tissue factor in human tissues. Implications for disorders of hemostasis and thrombosis. In vitro production of IL 1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation. Mononuclear cell tissue factor: cell of origin and requirements for activation. After blocking with 5% non-fat skimmed milk, membranes were incubated with mouse anti-human TF monoclonal antibodies After Lymphoprep isolation, total RNA was extracted from MNC pellets with an OR) anti-human TF monoclonal antibodies (10 µg/ml) for 30 min on ice. Mouse monoclonal antibodies recognizing extracellular domain of human TF were prepared by the Monoclonal Antibody Facility"
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "https://europepmc.org/articles/pmc549252?pdf=render",
         "Biology,Medicine",
         "Antimicrobial Agents and Chemotherapy",
         "T. Levin,B. Suh,P. Axelrod,A. Truant,T. Fekete",
         "24218070,1941629,144104608,4961128,49307125",
         "179",
         "10.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         "http://www.commsp.ee.ic.ac.uk/~ng1/pdf/gehrigD_CL_Jan05.pdf",
         "Computer Science,Mathematics",
         "IEEE Communications Letters",
         "N. Gehrig,P. Dragotti",
         "3017159,2752909",
         "59",
         "5.0",
         "2005-01-10",
         "2005",
         "a0de588554dc558fc9a7c968f02626444c917679,05c21f03eefa2d756229000ebe133b6bc6d08e1c,5e0eff55c4e3e3158b17e5452e213226fd8f21ac,33e627cb93a476a501308fb500015d282297ac03,d7cfbf051867bf3b35c3a2738a4731a148e06cc5,6e3da810748d174ccf2107591019ed318eaf6e40,f2b32222ec8955c4378fd2804757f4f8449d9241,29c907591b8be92bb535cc4e7453a4423d770238,c41ecfe2e42872cf03ced942f1d5daf918a00b5f,71c969f0a7204259cdd394283ab02d0fc34e5fc2",
         "Design of Slepian-Wolf codes by channel code partitioning On some new approaches to practical Slepian-Wolf compression inspired by channel coding Distributed code constructions for the entire Slepian-Wolf rate region for arbitrarily correlated sources Compression of binary sources with side information at the decoder using LDPC codes Compression with side information using turbo codes Compression of correlated binary sources using turbo codes The capacity of low-density parity-check codes under message-passing decoding Distributed source coding: symmetric rates and applications to sensor networks Distributed source coding using syndromes (DISCUS): design and construction Noiseless coding of correlated information sources"
        ],
        [
         "9",
         "000a2234245d642c786a7355a15623444c33c1dc",
         "https://www.jimmunol.org/content/jimmunol/175/5/3140.full.pdf",
         "Biology,Medicine",
         "Journal of Immunology",
         "Suzel Davanture,J. Leignadier,Pascale Milani,P. Soubeyran,B. Malissen,M. Malissen,A. Schmitt-Verhulst,C. Boyer",
         "6084539,5629146,153940998,3941723,5534674,5153661,1400945749,1786102",
         "29",
         "0.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "10",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "http://www.cs.ucr.edu/%7Ekishore/papers/tencon.pdf",
         "Computer Science",
         "IEEE Region 10 Conference",
         "A. Negi,K. P. Kishore",
         "1728262,2341041934",
         "41",
         "3.0",
         "2005-11-01",
         "2005",
         "a6ce59fb59ba15a8833a12bfe6692dfcd436824a,aa03424e6c7e7c9415b554a68643c16085df0187,100ab9c177c6e0abdda1130eada7a3fe8015606c,25be4335206df53c769244cfce81ebc5914a46cf,1feb6fb70407c123de7af972a4271c07472fe3a6,1f0a71167544d4018089c51861b10365993ff151,956615751a21c1071f18ab90193f218d6319013e,6bc43977fb11cceed0b9aa55b23c6dd29dd9a132,9308cfdabf5303534b97d9ce5bfbb2c919a3f9cb,da7a3280ec5e77c007045eee8f6f657f567ab97d,3c520bf9d75f785f4d4935ef8bd9e4b38c7c5db2",
         "Performance Contracts: Predicting and Monitoring Grid Application Behavior The design, implementation and initial evaluation of an advanced knowledge-based process scheduler Understanding the Linux Kernel Predicting Application Run Times Using Historical Information A Historical Application Profiler for Use by Parallel Schedulers Context Switching and IPC Performance Comparison between uClinux and Linux on the ARM9 based Processor Characterizing Process Execution Behaviour Using Machine Learning Techniques The Pearson Education Linux Kernel Development , 1st ed Correlation-based Feature Selection for Machine Learning A Dynamically Adaptive CPU Scheduler Linux Kernel 2.4 Internals Co-rrelation based feature selection for Machine Learning, Master Thesis, Department of Computer Science, University of Waikato Machine Learning , 1st ed, pp: 52-75, 154-183, 230-244, The WEKA: The Waikato Environment for Knowledge Analysis ELF: From The Programmer's Perspective The Design of the UNIX Operating System Build the knowledge base of static and dynamic characteristics of the programs from the run traces obtained in step 1 and train them with the C4.5 decision tree algorithm is a Masters student in the Department of Computer and Informations Sciences We send this STS information to modi(cid:2)ed scheduler through a system call The scheduler instructs the CPU such that CPU allocates STS ticks to ’X’ decision tree will classify ’X’ and output the STS. per second.(cid:2)le Aivazian , Tigran , Linux Kernel 2 . 4 Internals , The Linux Documentation Project , August , 2002"
        ],
        [
         "11",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "https://europepmc.org/articles/pmc1087730?pdf=render",
         "Biology,Medicine",
         "Molecular and Cellular Biology",
         "Takayoshi Bando,K. Sekine,Shizuka Kobayashi,A. M. Watabe,Armin Rump,Minoru Tanaka,Yoshikuni Suda,S. Kato,Y. Morikawa,T. Manabe,A. Miyajima",
         "24043805,2063038091,2110065840,40447710,32435082,2110038433,36341155,2568535,32275049,5658239,4789048",
         "38",
         "3.0",
         "2005-05-01",
         "2005",
         "",
         ""
        ],
        [
         "12",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "https://infoscience.epfl.ch/record/87259/files/Cavallaro2005_1275.pdf",
         "Computer Science",
         "IEEE transactions on circuits and systems for video technology (Print)",
         "A. Cavallaro,Olivier Steiger,T. Ebrahimi",
         "145440152,2359024,1681498",
         "120",
         "7.0",
         "2005-10-01",
         "2005",
         "bbb456acfec69d279b3f653b53dc1182d1b44eb6,cedf72be1fe814ef2ee9d65633dc3226f80f0785,dcdef9d1abf4be052e309058d695f19015868209,168033139bddbd1c02f761dd905375203cc2f73a,c69e28503a607c30f042fbd781de47c2a34b57fb,37eec7c527780a71578ecb32b5a48d563e651c7d,51a5d7d650ea5fdc12ae6e359ed052f67c3482c3,972fc844400ad601ca5c9274912ecb8007457790,d91e1b1ee1f83e2b3dc54ba7811a3c38f77ea701,48c7afd55f620cc665cabb52755968c5c57d9eda,ea3bcd105081726ab403513af553f5d0c32a2c6c,f494794fd30f451365aa69a96227ba8384ca492b,8b2f244b587207e8ee5415d507e432d5220d6163,a741ea5e483061f2ad1767290770ef35e74fcab4,3b9f60928a95709d90cf6b2f0705b16c07fca05b,d497613415c71d3ab544f3531871173749b31226,ce8883dbe8c695d8ef770eefe34e48efe7c0cf64,c1347daf84d6750a4fcb16cd04a1124ea3e7b775,100c788dea25ed53de43b1d5505afd19f4d85e30,8406c229176f56903f7de8e913081326341dcda9,89a05047a487ba4e9c32687ceb7fc00f59a834f2,5e52ecd0f0ab96bcb0b362e4ea4f0207ec996e66,292889c24c0fa339af5927bb4845b25b89a5e57a,86bbc82e2891d5baf900e40668f897e919d9ed87,332da8685270e4d39bdcededc7a00d1631c645d4,efa69b21767d15897e00f11bdf4625a8dcf1c1c4,febae7da65fbd5ec7d1227cf82e82cfbe48532dd,319b9413a46218c2832d6f3571880ac0d55adfbe,ba01ca8a463e294fc6a36473f6391095cbb164f2,e32c87769fef343508bd9bb2f7ee083417354245,dbc8987f77aaecdb058aac10c175124b06b1dbaf,cffb520ae667ed8737bbe3f5a05d86e8f85b9dba,0c646a8923d70ae9a93ee8945c05e47246707d80,cc032af2d72406bff21b34a473fdc3588c2fe920,770429367eb26731adc1abc31ba3693f7126e3a5",
         "Automatic foveation for video compression using a neurobiological model of visual attention Text information extraction in images and video: a survey DAVE: a system for quality driven adaptive video delivery Video transmission adaptation on mobile devices Visual attention for region of interest coding in JPEG 2000 Semantic segmentation and description for video transcoding Resource adaptation based on MPEG-21 usage environment descriptions Video transcoding architectures and techniques: an overview Metadata-driven multimedia access Semantic transcoding for live video server Multiple video object tracking in complex scenes Complexity-quality analysis of transcoding architectures for reduced spatial resolution Instantly indexed multimedia databases of real world events Face detection in color images MPEG-7 description of generic video objects for scene reconstruction A new content-based hybrid video transcoding method Video Processing and Communications Media conversions to support mobile users Object-based transcoding for adaptable video content delivery Overview of fine granularity scalability in MPEG-4 video standard Video object extraction based on adaptive background and statistical change detection Video transcoding for universal multimedia access Heterogeneous Video Transcoding to Lower Spatio-Temporal Resolutions and Different Encoding Formats Scalable multimedia delivery for pervasive computing Adapting Multimedia Internet Content for Universal Access A frequency-domain video transcoder for dynamic bit-rate reduction of MPEG-2 bit streams Multimedia content description in the InfoPyramid Objective methods for assessment of video quality : state of the art On the methods and performances of rational downsizing video transcoding Shadow detection in image sequences Video Transcoding Architectures and Techniques : An Overview Visual radio: should we paint pictures with workds, or pictures? Dave: A system for qual ity driven adaptive video delivery,”Proceedings of the 5th ACM SIGMM international workshop on Multimedia information retriev Processing and Information technology – coding of audio-visual objects – part 2 visual–amendment 2: Streaming video profiles Statistics, 3 ed Continuous speech recognit  io Submitted by Information technology — Generic coding of moving pictures and associated audio information — Part 2: Video SUBJECTIVE VIDEO QUALITY ASSESSMENT METHODS FOR MULTIMEDIA APPLICATIONS"
        ],
        [
         "13",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "https://europepmc.org/articles/pmc1081291?pdf=render",
         "Medicine,Biology",
         "Journal of Clinical Microbiology",
         "S. Nunes,R. Sá-Leão,João André Carriço,C. Alves,R. Mato,A. B. Avô,J. Saldanha,J. Almeida,I. Sanches,H. de Lencastre",
         "3688641,1398826424,4908451,2053407517,12533308,10268213,40282052,116697293,2452668,2290923",
         "67",
         "2.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "14",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "https://wwwnc.cdc.gov/eid/article/11/8/pdfs/04-1197.pdf",
         "Medicine",
         "Emerging Infectious Diseases",
         "P. Yagupsky,E. Baron",
         "6618694,39055987",
         "185",
         "2.0",
         "2005-08-01",
         "2005",
         "25f20e694999a6840a06702d74cfd37d95d667f3,1ff74908befd25fee4b2ab003de4f83a41a41e45,66cbab4b36bb6b4952ea76981c60f27ab3275898,bb816e36ad186f5dfde74d71885070d766e2d747,e8d3010e9528076bca3e5fba37538edaa1635eb1,2ae2e7ee84a9137b010e909f76fc936ff775ad6e,af5b6fcc6f7f7a1b4c60f0d7b74ca15e67a14519,7cd9837410cc2ef6f967852c5ffa0cbe336dbdea,256f57d10a7a759d7e388a7cbbaa2fd54613a54b,e59cf90691ee9a5bd3aa34c70030e27b623ffd14,c39da3e000e116a2ebf7e6b115799a90c4d1923f,3fdffab87e6d38e294b3d76c2df8568fcdf8a0c6,0d18a3b83e52b92ee2a0fb372eaa4346feb551ca,a162170411238cb7a93e9c93040982aa25454c19,c41cac3f6e8765bef847ec9b406aaacffe0e4fa1,39eb0675b38ca75201203941d13ba0f4a186e283,594b085d66a958b96aba9d1933972d3b34bb7678,e40f0229ec9bbe688f6356438951dfbb36fc59cb,0ca7f50704a2e0ab7da2896839f913a571b2e777,df8e94c95ce7f4d08c04cb2d562200b1eb0ddc59,4e4469c2a2132167fae10e5133dbd033b50d7471,f8b32e9384596d0545c70b5623e957a3aa7f60f9,1e0d986827359e89c42fdc145007c65b89a15ab5,db1a7e39ea496679bee7a5c6073402b6fd17e8d2,647d1a66e8764bdca4767f352d54f97fcd5252ac,cf4b52becaed752bb036d66d6cba6c0d1785af1d,24fab9ab18c998c12f7e99c14f31ae41c822402f,dd97f7c782c7ff9d1a98d2454f07dba3fc9e2906,29debcd880647e9d5ec1e1033228f5ca91753a8c,a03a55d8331f010683a46d3f2a808c56459961ad,1f7571e5b811f06b77bd16c7f277ed9849cd69e6,31a5d17ec14a53090757e7042ca707f4b8f3a35a,736a26b7dbf151a54974d500bad5c08b0b723ebb,f64fee05bc23ffc94ffec0e1061461c398ddf905,60fedd708bd15ec43d3e4c891b4f389db9c5fecf,e397ef4308fcf8f9cdc71dd5a4e43bb4b5d20887,ee5e98347d31991dd2b088ae627039ae24f0b10f,362c818a28759e5fca9a7f0d4e92ed695951a5fb,a01715bd830311a59ceda880f94f11a860224873,2b233f3fe9887477b9333af2751d2476954bc901,558e59a9972530acb403739cc7ef0239143f5a8f,ed002af5819415ec0a8b31a69ce941f6e7d53f64",
         "Laboratory-acquired Brucellosis Acute hepatitis due to brucellosis in a laboratory technician. Prevention of Laboratory-Acquired Brucellosis Analysis of risk factors for laboratory-acquired brucella infections. Use of the Isolator 1.5 Microbial Tube for Detection of Brucella melitensis in Synovial Fluid Bacterial Pathogens as Biological Weapons and Agents of Bioterrorism Unusual Complication of Breast Implants: Brucella Infection Brucellosis in laboratory workers at a Saudi Arabian hospital. Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases Brucella abortus Infection Acquired in Microbiology Laboratories Posttreatment Follow-Up of Brucellosis by PCR Assay Detection of Brucellae in Blood Cultures Rapid diagnosis of human brucellosis by peripheral-blood PCR assay Failure of short antimicrobial treatments for human brucellosis Comparison of BACTEC 9240 Peds Plus medium and isolator 1.5 microbial tube for detection of Brucella melitensis from blood cultures Exposure to Brucella in the laboratory Brucellosis from sniffing bacteriological cultures Grand Rounds—Hammersmith Hospital: A case of laboratory acquired brucellosis Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases Live Brucella spp. fail to induce tumor necrosis factor alpha excretion upon infection of U937-derived phagocytes Outbreak of Brucella melitensis among microbiology laboratory workers Unsuspected brucellosis diagnosed in a child as a result of an outbreak of laboratory-acquired brucellosis. Brucellosis: imported and laboratory-acquired cases, and an overview of treatment trials. Biochemical mis-identification of Brucella melitensis and subsequent laboratory-acquired infections. Laboratory diagnosis of brucella infection: some pitfalls Outbreak of Brucella melitensis among microbiology laboratory workers in a community hospital. An outbreak of Brucella melitensis infection by airborne transmission among laboratory workers. Laboratory-acquired meningitis caused by Brucella abortus strain 19. Past and present hazards of working with infectious agents. From the Center for Disease Control Brucella canis: A cause of undulant fever An unusual outbreak of brucellosis. Brucellosis: I. Laboratory-Acquired Acute Infection Brucellosis in the United States. Brucella species Brucellosis: An occupational hazard for medical laboratory personnel. Report of five cases Beer-Sheva 84101, Israel; fax: 972-8640-3541 Prevention of laboratory-acquired brucellosis. (revised). Geneva: World Health Organization Basic protocols for level A laboratories for the presumptive identification of Brucella species. Washington: American Society for Microbiology Exposure of hospital personnel to Brucella melitensis and occurrence of laboratory-acquired disease in an endemic area. Biosafety in microbiological and biomedical laboratories Laboratory-associated infections: incidence, fatalities, causes, and prevention. Principles and practice of infectious diseases. Vols 1 and 2. Brucellosis in the United States, 1965-1974. Laboratory Infections Due to Brucella"
        ],
        [
         "15",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "https://hal.inria.fr/inria-00182046/file/2005-petti-fraichard-iros.pdf",
         "Engineering,Computer Science",
         "2005 IEEE/RSJ International Conference on Intelligent Robots and Systems",
         "S. Petti,Thierry Fraichard",
         "40180963,1694234",
         "287",
         "17.0",
         "2005-12-05",
         "2005",
         "3d89fe08f16bf8ecabb7bb26115b8baccbdd8712,64be486150521eddcf83ef356a0719d1ef65ceb8,0cfc462c907a790ecc56fba3ffeade9f1fbae757,3c0ced2de89291e1e94a68217a2d27f4b8ba1b64,9cec1297bfce7ddd63390670467f1af6ccae4165,e4ed63ae63e6a8634e115925284f8c644b943623,9dddc932f553f31c14fdf941abe4f8961eef34f8,1281372e4b79bdb0d58a3d1dd2a813d23b2490c0,b0501adbca9917c19ae7b343e74b6e163753a227,62113b0fcb182853446679430aaefa9c367fe438,dc6e901b8c62d90c2d77877488b51c46f8bdf8a8,60f389d8def6c93aab17faf369f0727edbd51f47,afcd60559f9bc54bc2e0fd063fc0eb2bef87482e,9ba11955f7d8c93163288c33f9c3ed4cffc89a31,3f77f8b0e1a3e3dcedb0953a42018b6948b60617,ccfb06935ea9c8832ec6086000f71cf584a79679,7e81dac6260c4768af3a28ac21c78c5a38a5f7d0,2078d9a14df37d0992283c03cf42f591d71aeb3b,4d95e5c7c3b7d8a2a8a273e69a5b4fb98557b912,90c8e1ff97ffede807ce0e4bd20b984dd96725d9,ba40bb0b5d17afdab9adfcad9648a184b0a6c7f9,e8d60f2e1d51566953256020c71fee569865c948,09ddcaaf43aa3115ffa86aeb106604c9233717d5,6260d36c5fcb7fefb997ede20c2ba983adfa2470",
         "Motion prediction for moving objects: a statistical approach Inevitable collision states. A step towards safer robots? Towards real-time global motion planning in a dynamic environment using the NLVO concept An integrated approach to goal-directed obstacle avoidance under dynamic constraints for dynamic environments Real-time randomized path planning for robot navigation Improved fast replanning for robot navigation in unknown terrain Learning motion patterns of persons for mobile service robots Reactive navigation for non-holonomic robots using the ego-kinematic space Randomized Kinodynamic Motion Planning with Moving Obstacles Real-time motion planning for agile autonomous vehicles Kinodynamic motion planning amidst moving obstacles VFH/sup */: local obstacle avoidance with look-ahead verification Randomized Kinodynamic Planning High-speed navigation using the global dynamic window approach Motion Planning in Dynamic Environments Using Velocity Obstacles The dynamic window approach to collision avoidance Probabilistic roadmaps for path planning in high-dimensional configuration spaces The curvature-velocity method for local obstacle avoidance The Focussed D* Algorithm for Real-Time Replanning Reflexive collision avoidance: a generalized approach Motion planning in the presence of moving obstacles Robot Motion Planning Sensor-Based Motion Control for Mobile Robots Kinodynamic planning in a structured and time-varying 2-D workspace"
        ],
        [
         "16",
         "0012aecca19e4d3e41741599b1dbd00fb46d0c22",
         "https://academic.oup.com/carcin/article-pdf/26/12/2157/7088641/bgi195.pdf",
         "Medicine",
         "Carcinogenesis",
         "R. Rajagopal,M. Deakin,A. Fawole,J. Elder,J. Elder,V. Smith,R. Strange,A. Fryer",
         "2573963,2004776,15074846,145503858,145503858,2056036524,1931855,34067155",
         "19",
         "1.0",
         "2005-12-01",
         "2005",
         "",
         ""
        ],
        [
         "17",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2004.04487.x",
         "Biology,Medicine",
         "The FEBS Journal",
         "T. Utsumi,H. Ohta,Y. Kayano,N. Sakurai,Y. Ozoe",
         "6319788,122141924,49256899,48638495,6146172",
         "22",
         "0.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "18",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "https://europepmc.org/articles/pmc1134930?pdf=render",
         "Medicine,Biology",
         "Biochemical Journal",
         "S. Mousavi,Mitsuru Sato,Marita Sporstøl,B. Smedsrød,T. Berg,N. Kojima,H. Senoo",
         "145278181,47702160,1402638735,1432387861,145476071,50662160,4710114",
         "37",
         "1.0",
         "2005-04-01",
         "2005",
         "a6d3bbeb84717deccff079bd319413fc54b50e16,100f45a6b0045379892054aa70a738df681c6e97,85f7f66dc1be37706f712d6616387d33bb209a44,f86dc51f49b3f3d30e8e008c327f8b5402cddaf5,00be5eb58ac778cc624178ac010099a0601b7296,7fa779abd761c9306c433a9e87c1d11f8df19c1b,eb97c3f3cf1881a3908a6908f5e5a37e6d9c2145,cd648ef152b1f5a5d2b0f4645192b1c9b0ef17ec,2120667e80c26b7240ccac884467e869f6ce4c5f,3ea0c614b0b3edc4cb61b22054bb4f35e2bbeb5c,1c54f9a2d4aed39b8ec37e71cc63424a47526376,1ac7d0b397db7e0260c7bcf5ae4b4736f44f09a2,252991b9bbe9a0eb815051a797ae3a154600b74a,2dbfc38f53fc5d1c5441892b50d6cb0e5dfd170d,961e7f9ed82dbb973b0db57f7103e3c867d158ef,50c37eea83465e0ac7ebca071c3b75dc559f81d1,ae03a6754c9d89bdeca4474383cb078429c3830f,2303a910b94222ed2388c2206fa36060bdfeaf28,116139463e233dfbe4e8b71aa1a858b0b1145c22,c65bae3717eda5fa4a9cb63a6bf7c09040773c00,e0400ec8e27dd9036ad838755534df4bdfad87d9,2170efb311a61e99b0c7a960aa715a39d237aed1,c76fd3c2bb59cd99649849e51b66fbb8ee75cd36,c7b72147f381caa4b35b44ca665b144474030d9c,0fa5ea6e6e3b8020866ffbd9b3a3235e295a1432,eca866fb91011bad32af30aab5638562bea0275b,c9d968f1085827fd0af82e323ec482e0f1caa8a3,da432fa0f66c6a0fe29a8032ed229112c6679365,075cf9dd18564794e48c3d4736a84a9c4433562d,b0d137da67e4e7ecd9887ee57a7260f18d4710d9,bdad5d7454e41e03771edff7c7c86a83d512094c,8a7b2a7500846faa86f207bf2c90aff7c07691b0,8aa62d7c8426b02ddafb2df88390688a92690c1f,14253d459b1d6329ff175d332ece39d6ef03b9ed,7269b727ca35ac19da971ed902ed32dd7b8785f4,f999493daa293f2d24782a9120dcf3d1d880b0fa,6a4b503f0840c26dab7ce2f0012715b47d054152,9c9ba006ad1b6be24f473a4a1ad36681f7fbc744,fb76c9e322005d0334146cfb3b248d32f9837f93,1c610c79254816af5845d65e4e4ff90213f00267,2d4cea38f64c8dcc06d2d1cee84f90f55821c32f,c3aacfc7fc9e06c6960d46653260edbee87858fc,b8a05a8ab5e142cc0348cf5a31a16d0aed1568f8,c99120417ac2adc583085b340766dbaf52d071c2,bc0ebe8a73c8efbdb7d0016eec8c513dcdd0a748,634668d47753c48db112e9377445fc37f9eeca22,26270ce70c5c8e8d40201fa3dd9fa717c2029329,f0d468bd28cbaa9d08f27278bcf6b372a6d846f9,e2b117fe55064f148332db4e1b36f52a55f2b3b0,38017c1032349f35c0b1efeafaf5861a9aa4c718,3cf858a3295d7b6f4b5fc7b234945ab25b65a51e,c568ad1bc32539c8bb0b8809873e7ff9fe73396e,1db7bf5ff49493dcfc46c66ed29a4b663cd78295",
         "uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV. Identification and characterization of the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor. A targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen uptake Liver extracellular matrix in health and disease Hepatic stellate cells: unique characteristics in cell biology and phenotype. uPARAP/Endo180 is essential for cellular uptake of collagen and promotes fibroblast collagen adhesion Characterization of Sugar Binding by the Mannose Receptor Family Member, Endo180* The mannose receptor family. The C-type Lectin Receptor Endo180 Displays Internalization and Recycling Properties Distinct from Other Members of the Mannose Receptor Family* Collagen-receptor signaling in health and disease. The Urokinase Plasminogen Activator Receptor–Associated Protein/Endo180 Is Coexpressed with Its Interaction Partners Urokinase Plasminogen Activator Receptor and Matrix Metalloprotease-13 during Osteogenesis Hepatic Blood Flow Regulation by Stellate Cells in Normal and Injured Liver Extracellular Matrix Degradation and the Role of Hepatic Stellate Cells Wortmannin-sensitive trafficking steps in the endocytic pathway in rat liver endothelial cells. Effects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy. Endo180, an endocytic recycling glycoprotein related to the macrophage mannose receptor is expressed on fibroblasts, endothelial cells and macrophages and functions as a lectin receptor. Selective adhesion of macrophages to denatured forms of type I collagen is mediated by scavenger receptors. A Urokinase Receptor-associated Protein with Specific Collagen Binding Properties* The C‐type lectin superfamily in the immune system An orphan receptor tyrosine kinase family whose members serve as nonintegrin collagen receptors. The discoidin domain receptor tyrosine kinases are activated by collagen. Vascular Smooth Muscle αvβ3 Integrin Mediates Arteriolar Vasodilation in Response to RGD Peptides Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis. Cell adhesion receptors for native and denatured type I collagens and fibronectin in rabbit arterial smooth muscle cells in culture. An endothelial cell receptor for plasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of t-PA-dependent plasminogen activation. Differential expression of matrix-metalloproteinase-1 and -2 genes in normal and fibrotic human liver. Transfer of retinol-binding protein from HepG2 human hepatoma cells to cocultured rat stellate cells. Fat-storing cells as liver-specific pericytes. Spatial dynamics of agonist-stimulated intracellular calcium transients. Affinity of integrins for damaged extracellular matrix: αvβ3 binds to denatured collagen type I through RGD sites Identification of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin in collagen. Internalization of retinol-binding protein in parenchymal and stellate cells of rat liver. p180, a novel recycling transmembrane glycoprotein with restricted cell type expression The alpha 2 beta 1 integrin cell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen. Activation of cultured rat hepatic lipocytes by Kupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation of cell proliferation via induction of platelet-derived growth factor receptors. Characterization of desmin‐positive rat liver sinusoidal cells Hepatic lipocytes: the principal collagen-producing cells of normal rat liver. A radioiodinated, intracellularly trapped ligand for determining the sites of plasma protein degradation in vivo. The effect of leupeptin on intracellular digestion of asialofetuin in rat hepatocytes. The structures of the galactose-containing sugar chains of ovalbumin. HIGH-YIELD PREPARATION OF ISOLATED RAT LIVER PARENCHYMAL CELLS Distribution of Endo180 receptor and ligand in developing articular cartilage. Endocytic uptake of advanced glycation end products by mouse liver sinusoidal endothelial cells is mediated by a scavenger receptor distinct from the macrophage scavenger receptor class A. Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-α and TGF-β1 Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1. Vascular smooth muscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD peptides. Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis. Affinity of integrins for damaged extracellular matrix: alpha v beta 3 binds to denatured collagen type I through RGD sites. Expression of the gene of the α-smooth muscle-actin isoform in rat liver and in rat fat-storing (ITO) cells Acidification of the endocytic and exocytic pathways. Collagen production by rat liver fat-storing cells in primary culture. Newly administered [3H]retinol is transferred from hepatocytes to stellate cells in liver for storage. Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes), their related structure in and around the liver sinusoids, and vitamin A-storing cells in extrahepatic organs. Preparation of isolated rat liver cells. Hepatic stellate cells: morphology, function, and regulation Isolation of perisinusoidal stellate cells (vitamin A-storting cells, fat-storting cells) of the rat liver"
        ],
        [
         "19",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "https://europepmc.org/articles/pmc538921?pdf=render",
         "Medicine,Biology",
         "Antimicrobial Agents and Chemotherapy",
         "L. Poirel,M. Lartigue,J. Decousser,P. Nordmann",
         "6593096,40361075,46491845,5228247",
         "233",
         "8.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "20",
         "0018ece872bb78b06eb26ea35d5ed27f53702ce6",
         "https://academic.oup.com/bioinformatics/article-pdf/21/10/2558/48972005/bioinformatics_21_10_2558.pdf",
         "Computer Science,Medicine,Biology",
         "Bioinform.",
         "Shailesh V. Date,E. Marcotte",
         "2124202,1853998",
         "18",
         "0.0",
         "2005-05-15",
         "2005",
         "",
         ""
        ],
        [
         "21",
         "001a3ef673ee3e7703c7db597386a2162a2dc402",
         "https://academic.oup.com/icb/article-pdf/45/3/391/1674343/i1540-7063-045-03-0391.pdf",
         "Medicine,Biology",
         "Integrative and Comparative Biology",
         "R. Fuller,C. Baer,J. Travis",
         "7240006,2435258,145389296",
         "137",
         "8.0",
         "2005-06-01",
         "2005",
         "",
         ""
        ],
        [
         "22",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "https://zenodo.org/record/1259441/files/article.pdf",
         "Biology,Medicine",
         "Virus Research",
         "I. Kuzmin,G. Hughes,A. Botvinkin,L. Orciari,C. Rupprecht",
         "31991394,145378585,4167987,4216439,1727205",
         "187",
         "15.0",
         "2005-07-01",
         "2005",
         "f9833b6bdeb64d7b59c9f7d9b344ce25d5c90dba,2b97169a29a168ab41c3ec1a78164b96a6969a41,7a98ad002f81a2f7aab8d8491cadbc88aa8c3155,d1b3e4ff02d07be96c06fbfaf555406f17196741,33be5abf0ecb7dfdc6402e68dc86a1126c13adc8,1a9bc3e0a07a9d95d4f35323ab0f1c8c1413ef76,4001a31540fceaa58f5f393584a71522e1e31e4c,6b4c6ff3535d650d61c57ab38337d12b62314d01,7bcecb53c6e78442750e3e9b72ba3a80e769f70f,511a070c6dfd315dc7c4ae0e59c07957737618d8,dd06144588032875303e5ad4531de249890411a2,51a951e0338efd7eac40cb05a8d8bb1f9c0b2dc9,3e25fda5c88fe7969682f3203a9707ef42f0e551,6bb3b70653cf04a5b2c68240e740df7412f30ea2,725a3cbac5e12ee0dcd1dbd036929dc6d7783b58,ebb01f73c93b2d963c3dd09e607d85ac2921bd08,283f8d4a6b328ecf87aa6896307fa92a4159e282,a4bd32fbe148185f7f96bb6a8f057a8bee258e55,7696a27bd12a890797d339f730585d240ed59c1f,f63ea68c269895152c1aff73e5b1927b6fb32949,ad56575dae7f33f0a62df28b1bdc1c8344e4352a,8d54df6996acbb4c6758544b046278166afa4a9a,b0caf52d27797f3f413444bb15f5d6692b6375c9,cd2628e16a0c5978ccd8ab019308853ec712cc4b,aedc61581cafb2a48d3a0095d8694570f6d1e5d0,93260ef642602445acfac1a3130c86e85873cc38,2fa65c170ab7e679cbc0eecc5b03d3e25dac6556,5d8c958b183dad733d0e1003b74569b79a349478,1dfb94db76de9db1b86b2bd896a1991cdaf33c66,37815c1e6cce8b572a44892b360e619b10bfd28f,321bbf45869d401205db0123dd09aa5ddb78324b,f3b9fe7f78f8dd0673a811fec3b3741d652dcd8e,36f2bc416239289425d64110703dd037196571da,f1f369b6f62a2374dc6cba39c6ca58aa2544b016,1de7a444fac66d2c1f3a318d415714d524940c9d,e63bdf8efd8a01e5b53deee54a5837c8538b7cfe,970725ff0887cbdbf57e820588e4d7bb047383bf,c0f8a73ab829481eb8f97763d38a4d8eedaaf1d5,47c403f4295ab506971e1581475501908ac30e4e,385852642f3ada1e3dbf8ed90c005263a0be280a,dd20fc8f997d821dee710fe01a27049c12b5dfaa,54dfa2d112efd63b772c8c70795cd9cd7d3fc8dd,5685328fbbbf401d8b5b77924c616a825a56ea91,ba000c39195024b6f16e4eda48468db5be1506c0,f3ac4157691f496505db5c7b808b079f0e9d2363,682a5700489eec2f9c1f4f6246b73e5bcdb5fa97,f8d1d71b86d771cebd9061fba6e8459db171bbfc,096a93c9e0841bd3f99bfb486b170ed2c1b5d434,762cb1a4d0b79d9e9287cea102930294c87bf3ab,19b2eaa20f170259f94c5a84e1ff1952237bf92e,53bd1d94388d92e50aee35e9f5eb43a501b2a5e9,029463508fe0009869be5a3736075ab528d63664,adfd3c0cf96e20f5714a2deb0bb69772182b80da,2507c1996314911d9e998281f95f2a597358f74b,3842589a2100702965d4123501ff19858dc62cee,ec6f2977cb6c80aa845eafc14d46417a92ec3a00,17eab3eab209ab9b87425de3b51540b5efd9dd9e,b1d9cae15b280af1d4bc9001f066b757df14f5c4,e9038f1b41d390f5e6eeb9d8da000a94eec21e86,d3f4ba36f4de1d317078f90aa82b7e1526b5fb94,6993234eb3aba662d7d19b2b2c9b2f6f7304f830,a9c8b15829024098a9825321727fcd9e0d12c57d,0ae262d9cf78536754bc064e07113ab5e978f208,73679f1ba00de9e73492f3f5d53bdcef50af3fb4,85e77924e184aa659599cbc1675ac66142a3683c,924c6ec5ea4524ab6aa2e8f6e66bcdbcc4a7ba2b,94f848c5d0621d840bb13305ae945100c3551e77,cb592f5cbc4a9b871a79d9256404617ef6e5892f,2d10e6a2e1ffcb5608fa78ea47bab7927e53994a,0b8eb3ae0d8c670378d55f04ba69fa2f26ae2d31",
         "Novel Lyssaviruses Isolated from Bats in Russia Bat lyssaviruses (Aravan and Khujand) from Central Asia: phylogenetic relationships according to N, P and G gene sequences. A comparison of DNA vaccines for the rabies-related virus, Mokola. New Lyssavirus Genotype from the Lesser Mouse-eared Bat (Myotis blythi), Kyrghyzstan A molecular epidemiological study of Australian bat lyssavirus. Competition Commission inquiry into POMs. Characterisation of an Australian bat lyssavirus variant isolated from an insectivorous bat. Phylogenetic comparison of the genus Lyssavirus using distal coding sequences of the glycoprotein and nucleoprotein genes Lyssavirus P gene characterisation provides insights into the phylogeny of the genus and identifies structural similarities and diversity within the encoded phosphoprotein. Increased positive selection pressure in persistent (SSPE) versus acute measles virus infections. European Bat Lyssavirus Infection in Spanish Bat Populations Serologic Evidence of Lyssavirus Infections among Bats, the Philippines Genetic constraints and the adaptive evolution of rabies virus in nature. Expression of the B-cell and T-cell epitopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG and induction of an humoral response in mice. MEGA2: molecular evolutionary genetics analysis software Extensive Attenuation of Rabies Virus by Simultaneously Modifying the Dynein Light Chain Binding Site in the P Protein and Replacing Arg333 in the G Protein Molecular basis for the interaction between rabies virus phosphoprotein P and the dynein light chain LC8: dissociation of dynein-binding properties and transcriptional functionality of P. Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains Host Switching in Lyssavirus History from the Chiroptera to the Carnivora Orders Evidence of Two Lyssavirus Phylogroups with Distinct Pathogenicity and Immunogenicity Antigenic and genetic divergence of rabies viruses from bat species indigenous to Canada. Codon-substitution models for heterogeneous selection pressure at amino acid sites. Sequencing and position analysis of the glycoprotein gene of four Chinese rabies viruses Ecology and evolution of rabies virus in Europe. Phylogeographic patterns exhibited by Ontario rabies virus variants Chimeric Lyssavirus Glycoproteins with Increased Immunological Potential Multiple sequence alignment with Clustal X. Rabies virus quasispecies: implications for pathogenesis. DNA-based immunization for exploring the enlargement of immunological cross-reactivity against the lyssaviruses. PAML: a program package for phylogenetic analysis by maximum likelihood Evolution of European bat lyssaviruses. The complete Mokola virus genome sequence: structure of the RNA-dependent RNA polymerase. A Biographical History of Veterinary Pathology Ten-year survey of British bats for the existence of rabies Encephalitis caused by a Lyssavirus in fruit bats in Australia. Characterization of a unique variant of bat rabies virus responsible for newly emerging human cases in North America. The design of strain-specific polymerase chain reactions for discrimination of the racoon rabies virus strain from indigenous rabies viruses of Ontario. Genetic polymorphism in the rabies virus nucleoprotein gene. Molecular epidemiology of lyssaviruses: focus on the glycoprotein and pseudogenes. Molecular diversity of the Lyssavirus genus. Isolation and characterization of 115 street rabies virus isolates from Ethiopia by using monoclonal antibodies: identification of 2 isolates as Mokola and Lagos bat viruses. [A lyssavirus with an unusual antigenic structure isolated from a bat in southern Kyrgyzstan]. Rapid sequence evolution of street rabies glycoprotein is related to the highly heterogeneous nature of the viral population. PCR technique as an alternative method for diagnosis and molecular epidemiology of rabies virus. T helper cell epitope of rabies virus nucleoprotein defined by tri‐ and tetrapeptides Molecular cloning and complete nucleotide sequence of the attenuated rabies virus SAD B19. Rabies-related Yuli virus; identification with a panel of monoclonal antibodies. Efficacy of rabies vaccines against Duvenhage virus isolated from European house bats (Eptesicus serotinus), classic rabies and rabies-related viruses. Immunity against the European bat rabies (Duvenhage) virus induced by rabies vaccines: an experimental study in mice. Localization and immunological characterization of antigenic domains of the rabies virus internal N and NS proteins. Walking along the rabies genome: is the large G-L intergenic region a remnant gene? HUMAN RABIES OF BAT ORIGIN IN EUROPE Rabies in the Tropics Rabies-related viruses. Rabies virus infection of a flying fox bat, Pteropus policephalus in Chandigarh, Northern India. Giulio Alfredo Maccacaro. Experimental infection of dogs and monkeys with two rabies serogroup viruses, Lagos bat and Mokola (IbAn 27377): clinical, serologic, virologic, and fluorescent-antibody studies. An unusual case of human rabies thought to be of chiropteran origin. Two African Viruses Serologically and Morphologically Related to Rabies Virus Isolation of Rabies Virus from Fruit Bats in Thailand Isolation of a virus from Nigerian fruit bats. New Cases of Mokola Virus Infection in South Africa: A Genotypic Comparison of Southern African Virus Isolates BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT MODELTEST: testing the model of DNA substitution Immunoprotection by rabies virus nucleoprotein. Nucleotide and deduced amino acid sequences of the nominal nonstructural phosphoprotein of the ERA, PM and CVS-11 strains of rabies virus. Rabies and Rabies-related viruses in Zimbabwe historical, virological and ecological aspects. Rabies-Related Viruses Characterization of an antigenic determinant of the glycoprotein that correlates with pathogenicity of rabies virus. Antigenic characterization of the bangui strain (ANCB-672d) of lagos bat virus"
        ],
        [
         "23",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         "https://www.jneurosci.org/content/jneuro/25/48/11071.full.pdf",
         "Psychology,Medicine",
         "Journal of Neuroscience",
         "Y. Sugase-Miyamoto,B. Richmond",
         "1400827629,3007646",
         "110",
         "5.0",
         "2005-11-30",
         "2005",
         "",
         ""
        ],
        [
         "24",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "https://europepmc.org/articles/pmc1323209?pdf=render",
         "Biology,Medicine",
         "Proceedings of the National Academy of Sciences of the United States of America",
         "Huiling He,K. Jażdżewski,Wei Li,S. Liyanarachchi,R. Nagy,S. Volinia,G. Calin,Chang-gong Liu,K. Franssila,S. Suster,R. Kloos,C. Croce,A. de la Chapelle",
         "2328535,3744462,2157336158,2934528,32344884,1906416,2124909,16073087,4761141,3162753,3481071,144749275,3597724",
         "1251",
         "59.0",
         "2005-12-27",
         "2005",
         "5631e50fd39d9c7a54dec2bb5e5a016f8402c8b5,10bae8b93245299d0e4c4079e76f4d8c10458709,8de8119e007331a5b9e11c3ef99e15b2c7c8edce,e181f1874c85c6d42bf794f4e8aef5be2f77306a,c7497d6d32d3dd6f893609a17848c807134e1cc4,d8138beef46811bcb2ec68a4ae029b274ee4e1d1,77ef9aac7e0199184cb3169ac5e4de1d5fd7152b,94e90f0d2dba39b2357283e78f1a2dcb3f3ec358,240aa3fbe87b1fb085815baa87cca8bd04f52f70,631cf40b915e9f9b176d7a9be8a22c66a27a1acc,aab9e4d3502a20eb48d22ae8bb6ce29aabfe6a0a,01ae7d930255dd36a3308bd912d29d79859934db,62278466704924429db9b60bebc4ce8b4b7f2521,f60f9040c85eb47659079adf779a31bdb58014eb,c87c7646aa4273e73cdb1f2a7a8804281b478b4e,c8bb4cfcb04928f5312c66cc0239cca93839ff8b,72136fce3ef7aa30bf7ce640f44715daafe93c03,eab6ab732d7951e804769646a42a261665ed7297,8c1f06689340bbbd5f56d90511a338ebc958c517,2459b8e56f02c9d9c3b19ae8fcd75fedf05391d5,d577f656be9a1aa6d951dfd8cb02867d7194f4a8,3ed944a9b0fcd2a77298049f092a3b97b4633bc7,cde7b10619a54868cfbdacf99d051bc631bb86d7,97c4ead52f67e84d6ab55bb4f73a192533a95e1a,0442c89a09bfd5ba7f09340d0b23143f6c3f32b7,c81a5f7469419748daff6272230ed330001ee8f2,06cc16cdaf46fa657bb001faec28021b8ab5a3a1,5be20043c72560471655eba6f2a23329f5ea5f5e,b8f81ba06d314b8b112298c5b600557108c45872,92dcca877b55b85ceaa139bc5a6d4a563d598971,8259057fd4d7fb899894ff37c3fa9c2d1cca8798,726fa5af9a99f441cdddb2f86ffec6d6c968a6d5,80ce85acd05007231bbb58f41a4ee746ff5a6bd6,a96a1fee948a06d083e9d7bf57480666bd1653f3,4dac942817175d54e581803c90d20209cb0752b2,df292306c38ea75a87b649b3bd1833cdde89b128,f73bf5bf32c20821b00c854e60b21cff6f6a8e56,f5779281d7d71ae5a9d2f6a7ddc57699293d28aa,7a50462b065cdfddb9f96730fa68706174005e33,ab539ed05d3fc88082d19e3c5829da1d167ca066,234fd0f30e731e7d96e713d041c84fdc3bea8299,09b60a85059b98b1f3df8646742cf086edc512ea,b0d11c4e598c4c683cecac6640fae6e5330ce1ed,1bab97c3a3e9dc5c9f6545753bbd6fc7b58f6e68,bbefe294e85cb153d4bbf417b24b4073e0ec6999,cb9627792a808bf3921d00b62173fba1a2d2dfd3",
         "MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation. MicroRNA gene expression deregulation in human breast cancer. MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells. miRNAs, Cancer, and Stem Cell Division MicroRNA expression profiles classify human cancers Cancer genomics: Small RNAs with big impacts A microRNA polycistron as a potential human oncogene Small regulatory RNAs in mammals. Combinatorial microRNA target predictions The RET/PTC-RAS-BRAF linear signaling cascade mediates the motile and mitogenic phenotype of thyroid cancer cells. MicroRNAs: a developing story. RAS Is Regulated by the let-7 MicroRNA Family Accumulation of miR-155 and BIC RNA in human B cell lymphomas. A new mechanism of BRAF activation in human thyroid papillary carcinomas. Detection of Overexpressed and Phosphorylated Wild-Type Kit Receptor in Surgical Specimens of Small Cell Lung Cancer Kit as a human oncogenic tyrosine kinase. Oncogenic protein tyrosine kinases Human MicroRNA Targets Fast and effective prediction of microRNA/target duplexes. Genetic analysis of genome-wide variation in human gene expression MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias. Annual report to the nation on the status of cancer, 1975–2001, with a special feature regarding survival An oligonucleotide microchip for genome-wide microRNA profiling in human and mouse tissues. Reduced Expression of the let-7 MicroRNAs in Human Lung Cancers in Association with Shortened Postoperative Survival MicroRNAs Genomics, Biogenesis, Mechanism, and Function Prediction of Mammalian MicroRNA Targets BRAF mutations in thyroid tumors are restricted to papillary carcinomas and anaplastic or poorly differentiated carcinomas arising from papillary carcinomas. Reduced accumulation of specific microRNAs in colorectal neoplasia. Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma BRAF mutation in papillary thyroid carcinoma. High prevalence of BRAF mutations in thyroid cancer: genetic evidence for constitutive activation of the RET/PTC-RAS-BRAF signaling pathway in papillary thyroid carcinoma. Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia Environmental and heritable causes of cancer among 9.6 million individuals in the Swedish family‐cancer database Gene expression in papillary thyroid carcinoma reveals highly consistent profiles Localization of a susceptibility gene for familial nonmedullary thyroid carcinoma to chromosome 2q21. Papillary thyroid carcinoma associated with papillary renal neoplasia: genetic linkage analysis of a distinct heritable tumor syndrome. The biology of stem cell factor and its receptor C-kit. A gene predisposing to familial thyroid tumors with cell oxyphilia maps to chromosome 19p13.2. Paraffin section detection of the c-kit gene product (CD117) in human tissues: value in the diagnosis of mast cell disorders. c-Kit proto-oncogene is more likely to lose expression in differentiated thyroid carcinoma than three thyroid-specific genes: thyroid peroxidase, thyroglobulin, and thyroid stimulating hormone receptor. Transformation of thyroid epithelium is associated with loss of c-kit receptor. Systematic population-based assessment of cancer risk in first-degree relatives of cancer probands. Expression of c-kit receptor in normal and transformed human nonlymphoid tissues. RNA silencing. M  ED  IC  A  L  SC  IE  N  CE Loss of heterozygosity in papillary and follicular thyroid carcinoma: a mini review. de la Chapelle, A. Age at diagnosis Gender Race Diagnosis"
        ],
        [
         "25",
         "001ce484190e733e0d1dee181f9ee37fd0b15581",
         "https://academic.oup.com/jimb/article-pdf/32/7/301/34739283/jimb0301.pdf",
         "Biology,Medicine",
         "Journal of Industrial Microbiology & Biotechnology",
         "L. Beuchat,C. Pettigrew,M. Tremblay,B. J. Roselle,A. Scouten",
         "5992117,5613293,2054407528,2080517093,144590251",
         "47",
         "6.0",
         "2005-04-22",
         "2005",
         "ebdc8132e63badc8486ea08b463ca45bbac920f2,bb9a03d7fc726a5caff85716756521682fa33599,542676cd70a4a13b6520be34840b3928627ac1af,26b975dc9e3dd5605b024be8d0b29612cf80fb95,c587419aa127e80e61e17ffcf11ad30862a792a3,e9d641ae53747c724f9e3de913bec18162003673,715ce6488cfee5aa05a74d05da1bed03c00190a7,a87d00cbd3d138f74205878feea606bfc1653091,f8129a6f2927c0cf33b078fef7268d1ee5f5a66b,6b65eceb0ec96f7d472afa7491ebbd0637a456da,65e28bd82cb982bbce54f201121d6f5c47165db3,aedb3b17c9021bd5fdfc8e2446d030bfb58180ad,24279652e866eff900e765a39514b4679dbfa608,d3ce20ca3d60bfb72e91c641e34546c85e7bb4fa,2564ab2faeab37139d6f3b40dcfac765a5278ca8,f1dddee8f7a3f47eaeed12e25f132a4798252989,27a2ef7c94d7f65bfe7ee99741741e0c81e2a618,e06fe7341ef360c9e8482ad305f282823feb41e4,7a77c028f8a5cb55c53132c85630ff110701a9ff,4ce9ddba67cc1dc2117dce76d8df8a9b3b8d691c,025f4f1eb58e140f52a303fbf4446c714b496db3,06fad055a38bb1e8446bff362de92e2a5612c866,f341765d3914f817359a4ebdeeb831b17ade67a5,576523cc2193662b0cac3a498e490b0bcc1dc7a7,0296aa37d73332d74117909893b81293d11e7ce0",
         "Sanitizers: Halogens, Surface-Active Agents, and Peroxides Mechanisms of killing of Bacillus subtilis spores by hypochlorite and chlorine dioxide Bacillus anthracis: current knowledge in relation to contamination of food. Peracetic acid as an alternative wastewater disinfectant to chlorine dioxide Mechanisms of killing spores of Bacillus subtilis by acid, alkali and ethanol Differential efficacy of a chlorine dioxide‐containing sanitizer against single species and binary biofilms of a dairy‐associated Bacillus cereus and a Pseudomonas fluorescens isolate Reduction of listeria monocytogenes on green peppers (Capsicum annuum L.) by gaseous and aqueous chlorine dioxide and water washing and its growth at 7 degrees C. A study on the effects of high pressure and heat on Bacillus subtilis spores at low pH. Inactivation of Escherichia coli O157:H7 on surface-uninjured and -injured green pepper (Capsicum annuum L.) by chlorine dioxide gas as demonstrated by confocal laser scanning microscopy Resistance of Bacillus Endospores to Extreme Terrestrial and Extraterrestrial Environments Antiseptics and Disinfectants: Activity, Action, and Resistance Sensitivity of Bacillus coagulans spores to combinations of high hydrostatic pressure, heat, acidity and nisin The effects of various disinfectants againstListeria monocytogeneson fresh-cut vegetables Comparative sporicidal effects of liquid chemical agents Microbiological Control of Cucumber Hydrocooling Water with Chlorine Dioxide †. Interaction of Bacillus subtilis spores with sodium hypochlorite, sodium dichloroisocyanurate and chloramine-T. Bacterial spores and chemical sporicidal agents Bacillus Species Effects of Organic Acids on Thermal Inactivation of Bacillus stearothermophilus and Bacillus coagulans Spores in Frankfurter Emulsion Slurry. Sporicidal activity of mixtures of alcohol and hypochlorite. Ultrastructural and chemical changes in spores of Bacillus cereus after action of disinfectants. A sporulation medium for Bacillus anthracis. Sporicidal properties of some halogens. Kinetics and mechanism of bacterial disinfection by chlorine dioxide. Resistance of bacterial spores. In: Storz G, Hengge-Aronis R (eds) Bacterial stress Principles and practices of disinfection, reservation and sterilization Surface decontamination of fruits and vegetables eaten raw: a review. Food Safety Issues, Food Safety Unit, World Health Organization These include: Evicting an unwelcome tenant: anthrax"
        ],
        [
         "26",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/cncr.20947",
         "Biology,Medicine",
         "Cancer",
         "R. Kim",
         "2592641",
         "315",
         "15.0",
         "2005-04-15",
         "2005",
         "376109aaa3cbc518162072d892982ee7d7c61716,ce3c0b44cd2216676a66ed0c658d2bc298883f47,ef47428ff7b435d697f530965245854043545ab3,aeeddeb465b14955c0c207ae5d8facd8d4e01ce8,f172956a2b5badf538115971e667d34bebd87d22,9e4b1bab7acb0526340bd10134c672d3db8d445d,cdf7719d569d5b5ee447d329a25bbf19fdcc37c3,1c624d6499bd865abc6f8e400b24c8b79f6b620f,529a77664ace0111bcc3e9220770f8897ef023e5,c548c49f6baee6772d6e9fe3cbb9dde108a707b3,bf5f409029a90feff7df4cacdbd467d8d1eb00d8,6d6a4eef1ac55af590484c713f2f576c85c78d98,f357979269b1757c92d0c092be99730778c09d24,b3a2c655c5e01b71f590b2151cff8f00a02e1226,c45854aadb1d8e2e411e87a467f9f065f4866966,92f639e758c61f702a9c00cbea9be0976b20d894,53a50ae0268994aed5cf952d6c99043de7e35ebd,d37d31faa3a00280d21f5e1a648f98b66c8e3432,11b3fec8df3f80b2bec1488a6f0df6c4d8441c71,799ab82ec856050d460526143b7d0718b96e14cb,b3fae1ae6bc52cfe558a594ba83f5dc8a700f43c,51525a9c8be7e63f959ac2bfa3158f9db5b10d7c,b5672e0714e39e7bda99ea73b0bcebfdc6b05571,6da399d8f4d3460dde33aa226c27b631d47734ba,e83e6c40a82928c05944bc56ab2ae4a7d8ccf302,05d42e39ca41624d718474dffb3bcdfd01359b4e,a675c38d04171783d9a60886cc54ece593194dfe,bd437570db7af2cab4d48a6ce8301493ced7d301,045bc283c609699990dff04aa9b4514027dfceca,ad8cba15117d0a7073e3638b8a22daaac3588483,0bb77a1432b8d5752838df435480e44e3c50fff0,2ba0fec7e8cf702393a1382ca856855a7837556f,0291f1c9ff0da520e91419ae2db08999ee399725,e54cab9b65df38aa1f0a9519923c159b51fb68b6,5df427a0bafeb5240b6f577879c0de8e63ee7417,bd0da4411086bad78115eaf6dd0b45004209ff44,a2b0a9954056c4b757c828e6b16144251a158da1,7b6d94906866ab52104a903207582f2436ea697d,4fe6cc56b37705ac433ce4cd16853d5fe9fceeac,3c49e9197ceb2728b27e913342fc321637cf6238,3c8a2cf8e6c03282842828cd36f189d835c21f6e,fd0960e7ef910dc32ca134e477f05dc9818f6351,1b7d30f14e6b49cc7f21c2392bd968bf0cfe625f,382eec2dc4340b1331798554b7a49218ccc478e3,e806d85d49e988ee33f5e36bffd2a984449cda39,1b4b00fb7e6f96fe6e94b2d2e274ee431d4796f3,27930771ff138d36d235401e67bc181c075ab02f,43325ff8d344de359f0f6ccc23ba171a7cc26748,65e6570ac8da2eab3e549c8d0fa921e6b56a442c,803b1672ed76dc13f50b020abd30e76a010be24b,75f6318d79eb3ef37dc5e6ef5069864829e4e1d0,8b3b03b3843bc2489aea05e5bd575719abdc31c3,367f3536f0b382b32d38034fcf9b2bc30304d6c6,6a0effba2107618e090d29a519bc8cc0d95574bc,2368dde26aa4ed09380222cfc8d8081de22ab899,4feb355910a6276d6bb1a790d1b76315bae25072,55617d0d12a80e300ee19353cb9234189d563685,6b5e978b6ee2b44ed9651d1a391eff867f9582b4,4af3826564c80d0e09c2c58b4919a6839dd544bc,c8c7d9d1cdefc34132dd7af4cf41042bc8142c77,9bb678d996f92800823bbb881c51c7ae5d1d6fe7,f50232e9fd7cb6b62e70966fc65676393a2ff40e,6a064731895d6c8b46146a46dd99be1f8fba54dd,607f07d507527a3765bceb8483c69486e1ff86a6,d211b8513a89e4274e69a40062724877c389d783,3a40fdacf4cd8ece6443f92033e8f6539ce19720,83c29f099cb279c1bee03261845205562f74fa29,be92bf9b710be989ba332bef5013ae49be45fb60,c2f7cf5511d33da829564a022e41742fc1255477",
         "Neutralization of Smac/Diablo by Inhibitors of Apoptosis (IAPs) Aging as a catabolic malfunction. Apoptosis, autophagy, and more. Dynamic evolution of the adenine nucleotide translocase interactome during chemotherapy-induced apoptosis Bax is an important determinant for radiation sensitivity in esophageal carcinoma cells. Apollon ubiquitinates SMAC and caspase-9, and has an essential cytoprotection function A Phase I Pharmacokinetic and Biological Correlative Study of Oblimersen Sodium (Genasense, G3139), an Antisense Oligonucleotide to the Bcl-2 mRNA, and of Docetaxel in Patients with Hormone-Refractory Prostate Cancer DAP-kinase as a target for drug design in cancer and diseases associated with accelerated cell death. Endogenously released Smac is insufficient to mediate cell death of human lung carcinoma in response to etoposide. Smac induces cytochrome c release and apoptosis independently from Bax/Bcl-xL in a strictly caspase-3-dependent manner in human carcinoma cells Nuclear and mitochondrial conversations in cell death: PARP-1 and AIF signaling. Ceramide-mediated Macroautophagy Involves Inhibition of Protein Kinase B and Up-regulation of Beclin 1* JNK promotes Bax translocation to mitochondria through phosphorylation of 14‐3‐3 proteins Multiple cell death pathways as regulators of tumour initiation and progression Apoptosis defects and chemotherapy resistance: molecular interaction maps and networks Changes in Therapy for Solid Tumors: Potential for Overcoming Drug Resistance In Vivo with Molecular Targeting Agents Molecular therapy for peritoneal dissemination of xenotransplanted human MKN-45 gastric cancer cells with adenovirus mediated Bax gene transfer Mitochondrial protease Omi/HtrA2 enhances caspase activation through multiple pathways Apoptosis pathways in cancer and cancer therapy In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker. Endonuclease G is required for early embryogenesis and normal apoptosis in mice Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor Phosphoinositide 3-kinase accelerates autophagic cell death during glucose deprivation in the rat cardiomyocyte-derived cell line H9c2 Endostatin induces autophagic cell death in EAhy926 human endothelial cells. HSpin1, a transmembrane protein interacting with Bcl-2/Bcl-xL, induces a caspase-independent autophagic cell death Direct Interaction between Survivin and Smac/DIABLO Is Essential for the Anti-apoptotic Activity of Survivin during Taxol-induced Apoptosis* Omi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP) irreversibly inactivates IAPs and facilitates caspase activity in apoptosis. Transformation-selective apoptotic program triggered by farnesyltransferase inhibitors requires Bin1 The CD95 type I/type II model. Amino Acids Interfere with the ERK1/2-dependent Control of Macroautophagy by Controlling the Activation of Raf-1 in Human Colon Cancer HT-29 Cells* Induction of autophagic cell death in malignant glioma cells by arsenic trioxide. Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs. Chemotherapy enhances TNF-related apoptosis-inducing ligand DISC assembly in HT29 human colon cancer cells Molecular response of human glioblastoma multiforme cells to ionizing radiation: cell cycle arrest, modulation of the expression of cyclin-dependent kinase inhibitors, and autophagy. Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death? Current status of the molecular mechanisms of anticancer drug-induced apoptosis Hypoxia and Nitric Oxide Treatment Confer Tolerance to Glucose Starvation in a 5′-AMP-activated Protein Kinase-dependent Manner* Mediation of Poly(ADP-Ribose) Polymerase-1-Dependent Cell Death by Apoptosis-Inducing Factor DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death Bcl-2 and Bcl-xL Inhibit CD95-mediated Apoptosis by Preventing Mitochondrial Release of Smac/DIABLO and Subsequent Inactivation of X-linked Inhibitor-of-Apoptosis Protein* Increased Ras expression and caspase-independent neuroblastoma cell death: possible mechanism of spontaneous neuroblastoma regression. Paclitaxel triggers cell death primarily via caspase-independent routes in the non-small cell lung cancer cell line NCI-H460. The Apaf-1 apoptosome: a large caspase-activating complex. Apoptosis-inducing factor (AIF): a novel caspase-independent death effector released from mitochondria. Identification of Omi/HtrA2 as a Mitochondrial Apoptotic Serine Protease That Disrupts Inhibitor of Apoptosis Protein-Caspase Interaction* The Tumor Suppressor PTEN Positively Regulates Macroautophagy by Inhibiting the Phosphatidylinositol 3-Kinase/Protein Kinase B Pathway* Autophagy delays sulindac sulfide-induced apoptosis in the human intestinal colon cancer cell line HT-29. Bax translocation is crucial for the sensitivity of leukaemic cells to etoposide-induced apoptosis The target of rapamycin (TOR) proteins Proapoptotic BAX and BAK: A Requisite Gateway to Mitochondrial Dysfunction and Death Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function. PTEN controls tumor-induced angiogenesis Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis Bcl-2 down-regulation causes autophagy in a caspase-independent manner in human leukemic HL60 cells Molecular Determinants of the Caspase-promoting Activity of Smac/DIABLO and Its Role in the Death Receptor Pathway* BNIP3 and Genetic Control of Necrosis-Like Cell Death through the Mitochondrial Permeability Transition Pore Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments. Distinct Classes of Phosphatidylinositol 3′-Kinases Are Involved in Signaling Pathways That Control Macroautophagy in HT-29 Cells* Induction of autophagy and inhibition of tumorigenesis by beclin 1 Autophagy Is Activated by Apoptotic Signalling in Sympathetic Neurons: An Alternative Mechanism of Death Execution Cloning and genomic organization of beclin 1, a candidate tumor suppressor gene on chromosome 17q21. Oncogenic Ras triggers cell suicide through the activation of a caspase-independent cell death program in human cancer cells BIN1 is a novel MYC–interacting protein with features of a tumour suppressor Ultrastructural and immunocytochemical characterization of autophagic vacuoles in isolated hepatocytes: effects of vinblastine and asparagine on vacuole distributions. Studies on the mechanisms of autophagy: formation of the autophagic vacuole Vinblastine-induced autophagic vacuoles in mouse liver and Ehrlich ascites tumor cells as assessed by freeze-fracture electron microscopy. The morphology of various types of cell death in prenatal tissues. Inducing cancer cell death by targeting transcription factors. Mitochondrial dysfunction is an essential step for killing of non-small cell lung carcinomas resistant to conventional treatment"
        ],
        [
         "27",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2005.04623.x",
         "Biology,Medicine",
         "The FEBS Journal",
         "Ronja Tasler,T. Moises,N. Frankenberg‐Dinkel",
         "5801680,14847601,1399196272",
         "104",
         "6.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "28",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "http://www.jlr.org/article/S0022227520340323/pdf",
         "Biology,Medicine",
         "Journal of Lipid Research",
         "C. Villeneuve,M. Gilleron,I. Maridonneau-Parini,M. Daffé,C. Astarie-Dequeker,G. Etienne",
         "48980754,144505377,1397966565,5407662,1401604315,47231979",
         "107",
         "6.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "29",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D4334A1A0F9096B08FAE923BAB0CA36D/S0029665105000194a.pdf/div-class-title-insights-from-the-developing-world-thrifty-genotypes-and-thrifty-phenotypes-div.pdf",
         "Medicine,Biology",
         "Proceedings of the Nutrition Society",
         "A. Prentice,P. Rayco-Solon,S. Moore",
         "3674816,1401327164,48773910",
         "205",
         "7.0",
         "2005-05-01",
         "2005",
         "bc9ce03c2c4c174bd6c3b1d0be8900145edff1db,06acdfded5f54064fede109f07d9f36d8873834d,78d45a925322ae5987536246dd9a512b3d791ed7,c28997f8a141c0176ae7f5c06db2818ad40e220d,c57f0bf1bf34631a10d5f1d27191b88ed2f9b687,b3211005e2be14d595907435efc89659f76fa5c8,b02f1a164c91647b502be658d3c0e9e4c1f57f2c,27396abd2923770a9663d7ea65aa7a04ca6efc83,b23eef326baea71254d7c63656e97f638b9192ae,c137226bbdf92efc3da973dfd9e87a19dd042d68,eefefe28334b759b8b53e25caefca368a161dc98,2ae29359abae064e93f7d910e0c25aed49a88f4e,89c1180a10cd2d4f6b69b05faabcc6bb87929593,9ebf1a1f44e3658b2dd389b0526829653d05f0fd,de71e00c157f2a6252d6332c5a1e6063e6c4158f,f624515d506367c34202b1ccfb3a62566caf54b4,2f5d63fc5dd3fd93d6c8632642e9799cdb2dc667,8b025f19fc6c3a2565f2753b3d22219f0967e756,3b0189be73140b24df1fd28a3633c5debad6a958,4e4e8e7c72f63610dc20f83f07c02d220c438373,4a7f9ea4dd88c1d1793b79539dc887b93ee88c1c,a587a2b28dd5e8577fbd4d0a98dbf07506df40d6,d475e72c9d720f611b3728771f23082782877ebb,57784681819cecc1028626455fb79c57fe1030fd,95acbce1e13dea72bf4ecceef8070cd632c1905c,a1c3dfb03a261423e7f757718e61af931c1ac36d,0f4caae8c99e0d72292d71addb310524aec78c0d,a00002c79570f71907233f59273571ea44e67a27,7cfb934057f219e58da9c815e49c687cd5f3e305,c61d6adf94da35fa7b1ebd4332607ece26b12cff,0b69c9ba93b0e2d5d10c6b81cb65b179b0ce0e0e,00462a440fcfd0497a8769f2e3e94e40c50929c1,545f324e5db2615ddae40d7c685fd96ccb624f5d,e50748d4ac19be31f2f1ffe6591ef8b61a3bc136,864414f69552e3955b428c875a040785e2351d83,b7da62272a15b2eab222feeb806974255ffc6eb6,5c0dde78840d8c1c914f59179afc4c0993ce60fa,a37ef5d0f1eb4d9f2ee2c5e1ea49b3bceb1ca8eb,e946add8a70509c349ae15054c3c1052a981165d,5363b5010f319dc63abcdbfd1c868daebd725306,1ec845128a00e13b08eea726e8e8d784f844dfbd,795430c208a98348adb6fbbe7a962453d149c787,7454fa4f1eb0acfc56c67a288ac61bdebeb35970,8b6c6100fc489b4e951b9ee4f33f743d14b321e6,8246e8f6dbd713319cb843bf7e4990ad29df914e,f70b5135140b82250b8ccbcacde4973320f57b05,6ce812c3d9eb7a671e1a987517564a2b739870be,7d9c2c4768b5de3a533aa002b359449fabcdc151,b212cd8394f5fec259916490c9cf7668ed72160b,bc2e45abdfc013d33d5f1ba5c0934a8d4cc3574b,51c82b8dfc9ac7e33c7ecfaf023ddfe5c507de26,5522565260f1bd56280a2f5c50c5496b4ccad736,fb2b7baf210fb38b4a0501b2f5cd200abce4bb54,6a03fe8f413a6d83198f817eeaf8e1f27d9912fd,f83f5704a37c5776d21c4e950a45c27aaaecaac3,1036fd9c08f2f55be8974e20253875fd60ac16e0,323b16f39deeceae6fb5b285a9bd48c3e069a783,fc1819f465702c13883870784f9508e4054d00dc,935ec7c48c94bf948f1fbee7eb961255aa9eef24,c530eb353abee06e3b3cc5eea087d28aa0a800bb,6f38ee886f39daa656b1ac9ceb8a3c0785e100eb,d72154ce1a31048035a9ed91860acd6c895f735c,b5b8c35ff6c516a6106cfa930490e0779e8c43a4,d7d39f115e1b9b5482867486fcf2184576e8b72d,2a308549d0abed485e230c108a8bbe9338e2c926,90644eb817116ef816566ac87e3cc4b2804aaf26,94bdea56b64a2a3269c927af53a5f70a0abf389e,c49d0548da17d0e28c7d48fbfe7154cf5d523d7a,85d0d6037f14ad6bc592310d1f3350d87d424004,b79916ed1461af81a0559432924f559a13032c6f,9b47fdd07197bf012ba0b0bd9153b9dd70981bc1,8861cd3593c4334c59f75a859df8652cc4bb688d",
         "Fetal origins of insulin resistance and obesity Variation at the insulin gene VNTR (variable number tandem repeat) polymorphism and early growth: studies in a large Finnish birth cohort. Apolipoprotein E polymorphism and fertility: a study in pre-industrial populations. Opposing influences of prenatal and postnatal weight gain on adrenarche in normal boys and girls. Maternal-fetal interactions and birth order influence insulin variable number of tandem repeats allele class associations with head size at birth and childhood weight gain. The human obesity gene map: the 2003 update. Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA Intrauterine factors, adiposity, and hyperinsulinaemia Global haplotype diversity in the human insulin gene region. Birthweight and perinatal mortality: paradoxes, social class, and sibling dependencies. The double puzzle of diabetes The epidemiology of adverse pregnancy outcomes: an overview. Neonatal anthropometry: the thin–fat Indian baby. The Pune Maternal Nutrition Study Adiposity and hyperinsulinemia in Indians are present at birth. The role of uncoupling proteins in pathophysiological states. Racial differences in the relation between uncoupling protein genes and resting energy expenditure. An overview on the nutrition transition and its health implications: the Bellagio meeting An adipose-specific control of thermogenesis in body weight regulation Insulin gene variable number of tandem repeat genotype and the low birth weight, precocious pubarche, and hyperinsulinism sequence. On the importance--and the unimportance--of birthweight. The thrifty phenotype hypothesis. Obesity and undernutrition and cardiovascular risk factors in rural and urban Gambian communities. Glucose, insulin and lipid metabolism in rural Gambians exposed to early malnutrition Uncoupling proteins: functional characteristics and role in the pathogenesis of obesity and Type II diabetes Uncoupling proteins: their roles in adaptive thermogenesis and substrate metabolism reconsidered Fires of life: the struggles of an ancient metabolism in a modern world The nutrition transition and obesity in the developing world. Thrifty Genotypes and Phenotypes in the Pathogenesis of Type 2 Diabetes Mellitus Uncoupling to survive? The role of mitochondrial inefficiency in ageing The Fetal and Childhood Growth of Persons Who Develop Type 2 Diabetes Energy adaptations in human pregnancy: limits and long-term consequences. Interactions of perturbations in intrauterine growth and growth during childhood on the risk of adult-onset disease Insulin resistance syndrome in 8-year-old Indian children: small at birth, big at 8 years, or both? Transmission ratio distortion at the INS-IGF2 VNTR Apolipoprotein E (APOE) allele distribution in the world. Is APOE*4 a ‘thrifty’ allele? The insulin gene VNTR, type 2 diabetes and birth weight The thrifty genotype in type 2 diabetes The INS 5′ Variable Number of Tandem Repeats Is Associated with IGF2 Expression in Humans* Association of the INS VNTR with size at birth The Great Famine: Northern Europe in the Early Fourteenth Century Bergmann's rule and the thrifty genotype. Imprinted and genotype-specific expression of genes at the IDDM2 locus in pancreas and leucocytes. Susceptibility to human type 1 diabetes at IDDM2 is determined by tandem repeat variation at the insulin gene minisatellite locus Biochemical control logic and the metabolism of glutamine. Type 2 (non-insulin-dependent) diabetes mellitus: the thrifty phenotype hypothesis The possible role of glutamine substrate cycles in skeletal muscle. Maternal weight gain, infant birth weight, and diet: causal sequences. Role of triglyceride-fatty acid cycle in controlling fat metabolism in humans during and after exercise. A Polymorphic Locus Near the Human Insulin Gene Is Associated with Insulin-dependent Diabetes Melliitus Poverty and famine: An essay on entitlement and deprivation : , Oxford, New York: Oxford University Press, 1981. xi + 257 pp., index. $17.95 Malnutrition and fertility. Poverty and famines. An essay on entitlement and deprivation [incl. case studies of famines in Bengal, Ethiopia, Sahel, and Bangladesh]. Famine in Tudor and Stuart England. The high prevalence of diabetes mellitus on a Central Pacific island Famine and Human Development: The Dutch Hunger Winter 1944-1945 Faminee of history and of today Diabetes in American Indians and Other Native Populations of the New World The determinants of growth and form. Review lecture - The determinants of growth and form Diabetes mellitus: a \"thrifty\" genotype rendered detrimental by \"progress\"? Biology of Human Starvation A critical evaluation of the fetal origins hypothesis and its implications for developing countries. Lack of support for a role of the insulin gene variable number of tandem repeats minisatellite (INS-VNTR) locus in fetal growth or type 2 diabetes-related intermediate traits in United Kingdom populations. Differential reproductive efficiency associated with common apolipoprotein e alleles in postreproductive-aged subjects. Eating, exercise, and \"thrifty\" genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases. The insulin gene variable number tandem repeat class I/III polymorphism is in linkage disequilibrium with birth weight but not Type 2 diabetes in the Pima population. Floods, Famines and Emperors. London: Pimlico PPARgamma, the ultimate thrifty gene. Fetal origins of adult disease-the hypothesis revisited. Genomic imprinting of fetal growth The Demography of Famine: An Indian Historical Perspective Biochemical control logic and the metabolism of glutamine. Agriculture’s mixed blessings The Third Chimpanzee Determinants of low birth weight: methodological assessment and meta-analysis. Origin of Species"
        ],
        [
         "30",
         "00206ca6a32791adcca5effdfed6eecd035075f4",
         "http://www.vision.caltech.edu/malaa/publications/atiya05sparse.pdf",
         "Computer Science,Medicine",
         "IEEE Transactions on Neural Networks",
         "A. Atiya,Mohamed A. Aly,A. Parlos",
         "1686506,144933112,1838904",
         "35",
         "1.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "31",
         "0020d5bdfc5f13026e7a1aca78e94724e8342b62",
         "https://rupress.org/jem/article-pdf/201/6/881/1153938/jem2016881.pdf",
         "Biology,Medicine",
         "Journal of Experimental Medicine",
         "K. Turcotte,S. Gauthier,A. Tuite,A. Mullick,D. Malo,P. Gros",
         "2106108767,46813877,6563462,145705716,144605952,144829585",
         "109",
         "6.0",
         "2005-03-21",
         "2005",
         "",
         ""
        ],
        [
         "32",
         "0022e70878c2f2a7c3c679a3ef3160a5cb803e20",
         "https://academic.oup.com/mend/article-pdf/19/11/2696/10717141/mend2696.pdf",
         "Biology,Medicine",
         "Molecular Endocrinology",
         "Jing Huang,Xiaodong Li,C. Maguire,R. Hilf,R. Bambara,M. Muyan",
         "2145740413,2118899174,12605892,4551110,5522510,6431895",
         "45",
         "0.0",
         "2005-11-01",
         "2005",
         "",
         ""
        ],
        [
         "33",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "https://ir.vanderbilt.edu/bitstream/1803/13704/1/ms-thesis-manish-kushwaha.pdf",
         "Computer Science",
         "IEEE International Conference on Mobile Adhoc and Sensor Systems Conference, 2005.",
         "Manish Kushwaha,K. Molnár,J. Sallai,P. Völgyesi,M. Maróti,Á. Lédeczi",
         "2181068,2065280567,1717457,1740862,1692190,1721042",
         "70",
         "5.0",
         "2005-12-12",
         "2005",
         "813ffc490393534e9244924dcd0e4337b4bf5fa0,571eab8ea81d7c54bf5454c67c2d28f6aac3f086,35b225cf1cb2ff030eff1ffd9c554a87418b16ee,5e1c7fcad70403f2da76849eaf4551143c008116,a4ac5b5e63942b95c5ebbf51019869950282212d,cd608034cca0eb48676f3fc4f6082e4a504df42f,25a5f84ee5164162ad04e0a1d69c35525ce556de,26d7477c2f05594eb98a191826dcbdb836c9fb45,8b2815e893b0d4a476f27b15b69cec7f2a43d8e8,2cdd6e0dd343a94480adfeba247d3f33c429113a,d7d1ba245d7397c0cead0191c08aeb0da6fb2316,24741a5d686df667874f43d79e6359346444cd3e,6d5239e7d50c83dad3621b65a447ba8191b7694c,be8ac44dd2bb5f5d23cdb902dabd332969222c9a,869b5434530fd46dc84ca30661ebf8b3a697600b,2d4ad6826ed95a6b17709278b9f2b6c729b1163f,fba126954f354e24cc57d1159dfe7d5ae16d11ef,234efad9a8efb5d417743c70fc9f2d0ca6eac97f,6d6e4c92401f60682f00777238d4afeb729340bc,4f264407361a47c81695a03a8d96c078e1f588b0,0e4ade87ce468690926f153bc413d53a5034a9e4,22188b246b50e273b2f8de308c091d0c082aca03,15f705d41cfb39860c33d15435defaa4c6469d29,93172a2fb5c410498079361e806e9c43549c8d2b,fb5232fe382f4c801a0a0ecb4a6e92dab0f72889,fa9d7cdee95b6e780b5006606a8a96cdd62417d8,ba062f1deca36fdc287609771e47a92d26973ed9,0115f7ae387b4116acaafe84176f38bb2e7bab6f,96aa4d55a3870193b23c9c5d3b838e2be6f17c33,b2fe77a389e749359f12ba483664429e11182f62,46a1dbf20cd13dc3204eb0cc35fc34aaa3e2a70c,f5ed53760000b1851c10734f271a97b7017ecfb8",
         "Resilient Localization for Sensor Networks in Outdoor Environments Mobile-assisted localization in wireless sensor networks Sensor network-based countersniper system Robust distributed network localization with noisy range measurements The flooding time synchronization protocol Localization of wireless sensor networks with a mobile beacon Smart sensors to network the world. Range-free localization schemes for large scale sensor networks The n-Hop Multilateration Primitive for Node Localization Problems Ad hoc positioning system (APS) using AOA Mica: A Wireless Platform for Deeply Embedded Networks Middleware challenges for wireless sensor networks The bits and flops of the n-hop multilateration primitive for node localization problems Wireless sensor networks for habitat monitoring A survey on position-based routing in mobile ad hoc networks Robust range estimation using acoustic and multimodal sensing Location Systems for Ubiquitous Computing Smart kindergarten: sensor-based wireless networks for smart developmental problem-solving environments Exposure in wireless Ad-Hoc sensor networks Dynamic fine-grained localization in Ad-Hoc networks of sensors The cricket compass for context-aware mobile applications GPS-free Positioning in Mobile Ad Hoc Networks GPS-less low-cost outdoor localization for very small devices Embedding the Internet: introduction Acoustic Ranging in Resource-Constrained Sensor Networks IEEE Computer Graphics & Applications: Projects In VR Pervasive Pose-Aware Applications and Infrastructure VR Pervasive Pose-Aware Applications and Infrastructure. IEEE Distributed Systems Online Range-free localization schemes in large scale sensor network This copyright notice must be included in the reproduced paper. USENIX acknowledges all trademarks herein. Robust Positioning Algorithms for Distributed Ad-Hoc Wireless Sensor Networks The Design of Calamari : an Ad-hoc Localization System for Sensor Networks Complex Behavior at Scale: An Experimental Study of Low-Power Wireless Sensor Networks Convex Optimization Methods for Sensor Node Position Estimation SpotON: An Indoor 3D Location Sensing Technology Based on RF Signal Strength The Cricket location-support system Global Positioning System Overview (last modified 5.1.2000) Numerical Optimization, pages 250–275 Theory and application of digital signal processing If both | µ i | and σ i are large then again location might be affected by echo and we follow the same procedure as in case 2 above If µ i is close to zero but σ i is large then either the range deviations are spread around zero or few large deviations caused σ i to be large If | µ i | is large but σ i is small then all elements of ∆ i are large i.e. the node location is deﬁnitely incorrect"
        ],
        [
         "34",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "http://archive.nyu.edu/bitstream/2451/14996/2/Infosec_Book%2bWattal%2bTelang.pdf",
         "Business,Computer Science",
         "Workshop on the Economics of Information Security",
         "Rahul Telang,S. Wattal",
         "145961354,2628487",
         "80",
         "3.0",
         "2005-02-01",
         "2005",
         "",
         ""
        ],
        [
         "35",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-95-1423",
         "Biology,Medicine",
         "Phytopathology",
         "C. Silvar,J. Diaz,F. Merino",
         "6248808,152435507,145620908",
         "63",
         "1.0",
         "2005-12-01",
         "2005",
         "",
         ""
        ],
        [
         "36",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "https://europepmc.org/articles/pmc1069588?pdf=render",
         "Biology,Medicine",
         "Molecular and Cellular Biology",
         "T. Adachi-Yamada,Toshiyuki Harumoto,Kayoko Sakurai,R. Ueda,K. Saigo,M. O’Connor,H. Nakato",
         "1399592175,6637499,3515490,34604795,33751317,7615853,3971771",
         "23",
         "1.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "37",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "https://europepmc.org/articles/pmc2522321?pdf=render",
         "Medicine,Biology",
         "Journal of Proteome Research",
         "K. Anderson,J. LaBaer",
         "2700838,2112063",
         "311",
         "12.0",
         "2005-06-18",
         "2005",
         "a5fec970c69680e7d7ec9fe90b057291b4e7773d,8ef30a4dcb0ef1af8522d2a4ac3922457ba62a8c,733b342cac9c3cf6fbb70715bc3ecca5b730b349,c3c2808e82d98a4159cabcad3802dd7dc1705f7f,58ee959e3fe9a781bf59ab64549810149f593474,0069e9285f277056500448265d34b03a44372a08,a9d867d185ae257493226d2a53a94b7b682328a2,13bb6ea8a95a04aff56f95a013c3219d3cb57ff3,c4130add7c4c78affcff33f330dec1019373ec22,a982595680c8bcce7fcea5f16ac98e00e1113d2a,13d7452b1f93ae19f9db16cbe60f681616d02198,6e2c87806883391644aaec82861a2d4e64d3a503,7cb9f8e54e2786196b7d0f7178af3e32a66a294f,405a39a2fbd43b5bb6a73f12e6d880a38c0ec469,00ce84e1bc0fb55d3e4ac1eafef11137b842b674,94cbbf1805238625054ddaee315edbe98db15b99,e37bd6ce0c355fbe970a72b5d5ab2c61e76f1e31,7933e84c6162015607118f650818a04029a123a9,1a9168875ee9115b0130a978779867d6c86c164d,d14f3e86ee66258fe9e194939b2eff10a26deac8,251328149f10c2264a5b81e1e2163518d83e7af2,8458139de08cf9af5a8a3db11f9f56b641e0f9ae,3dea83a9f695382d933eb3cd5e89df303b16383c,0c70af8c234cad6a3716ec219fde55a52bca5d8c,96cb8f7f09a8fde293b5675da621836cd4a25442,17c551aca1c09efe7308f3a370e14b6f43eed293,2ce4c74262c38b472473cd089b3f6e1546ba989b,617018364b7ad94d15b34c751700371258488a24,363c94c7ab5aa4baebc089c8a6697c2d6188f054,b9cd2802cfb0e4b3d9c01132f470a2daced6a6b0,9aecd5b116595b1355f5dad17c919dcf448a4d6c,49de046c674e6b1be12427b81126f921c66e06e3,09e506bad5617ee5c1c9fcb2fc1baff71d33c898,1de2e10678cd99ffc6c12a7c80b45300c1f74b9e,9411bbe9e8ba032141dbc5c1904c65b8dce131bd,2b7da19d167b41357fa323a83fec7f69db36a6f6,6285b7e9f8e7cdf299920247ca0e0e14b37b1e8b,640a97b4aeb462b135971ed4acc2a243b55f6b4c,13cadee9e67644d3729d471bc3757c70e9fe173f,ad7be2b4e10846bcd85bdbf69c3e6652d50af488,c0adf2095391d71f8e92eb33df92c66b45ff51f1,48d6fc3223842c715e80bf6096c43cb3a8730e76,02074b7a8b2742ce30eecff1fead09b735be7b58,576afd62b7e1da025747950bfc0031ac070e14d9,901bde6f666a8933f2a1631a28b3c2e85f552cd4,d18a6692c38fa5c91bbe586aaca0d79f7d73ab1f,7d56a74dd196c2e81415b5d95a55b232a60d52f6,c5e573e73615df34b8e755dcd37c71f51a2aef8b,d36b4e618097ff1f97fb54e4fe8338e6539b9419,906df3ac3c0efc4ae49abb03eccb8a1a56eb6d8d,cacf432fbf7361cfc1b5a21e7653f4abeeb608bc,89a91f216ed67b56dbd2e897888fb03d3b7689b4,b8d5884927255b8a1ee8666d42dce3c7d8edc8bc,adc0cef49de7640091c30c64b5b0971fa15f8cec,77c7be9fba69091cdd99b89ae78bc1224eb6427f,296a0473a4537088e01704a8af41c749916998de,dc2aba9d4f2ca0d1a87e3fc0c3d5dbd89347657f,45e8b0e8aba7bce41416d9dc50f1175b8e546537,faf3728220c32d50d6f6311096ec3d70b0efa2af,417636fc2fa5a79d0a9e151ca2fadf4d896c3a6a,06f479de8c40c51ccbe591c004ee358beea7e3cc,d41eb0fce92491576a05850c935f894bbf09bde8,75521ab6ea29e022f51c4ed91382365836db205e,1690493dc16a64b7103da134383d970e407576af,24b56291be5bfeec9b69ecda07e5bc8167b4b326,a566936accc6ba41396d83ab68232ad0083689ee,d028d2e7b78ceb3103cb6e51ac8866e32ca28952,a8df84e42991b3fe1eac759a9491d542673b7c12,ca6ac75d2408d9fbc3ff49e8d62341821574337b,f26551154538a2f1d5799b9540b24cb74fb89763,61dc4870984aa539428bd0fea54fda55149f0eeb,e92fe6ff345e23d95f5d4c263ef2b66f1cbaa947,cf9c0737484df260b4407aceca30c6dbd7e0c08e,24aacae008a93fc5f912537d90b3808da25245bd,1ca341b8f40b748249717f6b26db00517a65645b,9649c703b781c322fad5f5d2e44af14460049978,e29c64ec56f634049c8931616bbc34807e53ccbe,b76586c209648e3f66a48b075a5a3b85b780c8e2,6783882e887d1fd25d84d4ef22b7c4fb6d5982e6,2bc35ccc858ecff282da44ce7c971b3efe3f6edb,1557de0758d8e90f89806d5aac1e02a788941a0e,eb61c808a6b210f8134d3c3ecbb7eb4cec0df4a6,49790e49ee605e518c6c164d6dcd3709b3b3d366,4bd9e587ccb7e702f9b83ce79e9a14d5889a365b,0a7e447ffa5921c3cefbd0a4110d0b73fa24fae4,cc2f46e6bdf0681fd05b7b5d41a74f52aa131813,e4a27eac6551682770c0af50db31723ad34edd55,6ea42f29588a3699a8daaf03e613cdc385a07a98,6984707cd20cab11a337aae74d31aa28f28506f1,bfd9b207d5df59cb51ee9ed7cd70749a9abcb154,16f54f42f2b97dc6548eac350b67227bf4f24715,f09ce709ee7be9223c98fe2729ac39e3654eb432,6046d06aa295bfb20be8e952b5124375ac25d5bd,eec606cec01c5c86a6195abd33dcf9178e4c9dd6,ce54d815b5ea33a108a04557a2a5855c259bd5de,9d4028799d9b9b282a190b3e96498d824aa0fe31,2a0b10e367b0786d4600b336aefc5a15072e5d3e,31cc5c4ab5f488e2a50b692de8264c4e5886e3bc,477e6297459532d8a651d8947d471b69a10c0bee,a5f1d509eb95129272afe5db38d1fa08afc1cf82,0b3b00a3c3dcd8577c6594781da13e90d11a25da,1e672c6d143b318e918a8424e1026a9433fc5952,54e66e26f22f9573b1ab00ef154e4b9ec97662fb,df9f47703c6299a257cd25ef20c82a66d39fd355,dcb0df34754e316e80826416031bc497c2eec162,341d10aacb81f6688bd0eb8566f6b76b1a8ad7b3,fc612a59bce380077f4cb8080bf4f7dcfb3ee494,f7c6c19558fbfec06849749ffa2f18fc4de63455,8c140318c3811d98a91afc7e3bbec42f62dee579,432d239dac42a5263ed2010313d582cfc6c32018,478504ff6ae608efcfcc5d125196cd8e6df85d3e,28a9f8b464aeda844f2233a2da2a00ac86597c56,7e520b5bdc68f2520e402593aa7c5b56ef1a3ad4,33c9970d75ac2c8b0561cfb2c3a9db7f319ada71,c0e41bffc9acee371c0365d1a79d37475c27aff4,238954a402bc9a038f88f0cbde3d801628f64e59,df36002142e55b86221f4ec5ac2722edfb86f73c,c28b6762b25ff1f624cea7a775d7c6a2a4214832,61e58171fd3059916c2e021dee9683e5bcb901b3",
         "T Cell–Dependent Antibody Responses against Aberrantly Expressed Cyclin B1 Protein in Patients with Cancer and Premalignant Disease Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast Cancer A phase I trial of DNA vaccination with a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate cancer Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant induces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans. Self-Assembling Protein Microarrays Protein microarray technology Minor Histocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem Cell Transplantation An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in association with HLA-DR. Development of natural protein microarrays for diagnosing cancer based on an antibody response to tumor antigens. NY-ESO-1 Expression and Immunogenicity in Malignant and Benign Breast Tumors Clinical proteomics: revolutionizing disease detection and patient tailoring therapy. The clinical value of intracellular autoantigens B-cell epitopes in systemic rheumatic diseases. Serological cloning of cancer/testis antigens expressed in prostate cancer using cDNA phage surface display Complete response to donor lymphocyte infusion in multiple myeloma is associated with antibody responses to highly expressed antigens. Humoral immune response against melanoma antigens induced by vaccination with cytokine gene‐modified autologous tumor cells Identification of a shared epitope recognized by melanoma-specific, HLA-A3-restricted cytotoxic T lymphocytes. Genome-wide Characterization of a Viral Cytotoxic T Lymphocyte Epitope Repertoire* Microarrays of tumor cell derived proteins uncover a distinct pattern of prostate cancer serum immunoreactivity Protein arrays for autoantibody profiling and fine‐specificity mapping Protein microarrays: Promising tools for proteomic research Recursive partitioning as an approach to selection of immune markers for tumor diagnosis. Antigen-specific immunity in neuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen. Expression of cancer/testis (CT) antigens in lung cancer. Unique conformation of cancer autoantigen B23 in hepatoma: A mechanism for specificity in the autoimmune response NY-ESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial ovarian cancer. Identification of tumor‐associated antigens by screening phage‐displayed human cDNA libraries with sera from tumor patients Tumor-specific immunological recognition of frameshift-mutated peptides in colon cancer with microsatellite instability. CD1d-restricted T cells regulate dendritic cell function and antitumor immunity in a granulocyte–macrophage colony-stimulating factor-dependent fashion Melanoma inhibitor of apoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction The use of HPLC-MS in T-cell epitope identification. Immunomic analysis of human sarcoma Enhancement of antibody detection in cancer using panel of recombinant tumor-associated antigens. Immunoscreening of phage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease. Integrated functional genomics approach for the design of patient-individual antitumor vaccines. Proteomics Approaches to Uncover the Repertoire of Circulating Biomarkers for Breast Cancer CML28 is a broadly immunogenic antigen, which is overexpressed in tumor cells. Towards patient‐specific tumor antigen selection for vaccination Cancer/testis antigens: an expanding family of targets for cancer immunotherapy Serological identification of antigens associated with renal cell carcinoma. Characterisation of tumour-associated antigens in colon cancer Solid-Phase Epitope Recovery: A High Throughput Method for Antigen Identification and Epitope Optimization Enhanced humoral immune response correlates with improved disease-free and overall survival in American Joint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine. Identification of Breast Cancer-Restricted Antigens by Antibody Screening of SKBR3 cDNA Library Using a Preselected Patient's Serum Posttranslational protein modifications: new flavors in the menu of autoantigens ATP6S1 elicits potent humoral responses associated with immune-mediated tumor destruction Identification of cancer/testis genes by database mining and mRNA expression analysis Autoantigen microarrays for multiplex characterization of autoantibody responses Miniaturised multiplexed immunoassays. Identification of Cyclin B1 as a Shared Human Epithelial Tumor-Associated Antigen Recognized by T Cells Trends in biomarker research for cancer detection. Autoimmune diseases: are markers ready for prediction? Post-translational protein modifications in antigen recognition and autoimmunity. Identification of a tissue-specific putative transcription factor in breast tissue by serological screening of a breast cancer library. Profiling the immune response in patients with breast cancer by phage‐displayed cDNA libraries Tumor-associated antigen TA90 immune complex assay predicts recurrence and survival after surgical treatment of stage I-III melanoma. Evidence that specific T lymphocytes may participate in the elimination of chronic myelogenous leukemia Pre-existent immunity to the HER-2/neu oncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer Profiling the immune responses in patient sera with peptide and cDNA display libraries. Reactivity of natural and induced human antibodies to MUC1 mucin with MUC1 peptides and n‐acetylgalactosamine (GalNAc) peptides Rapid degradation of a large fraction of newly synthesized proteins by proteasomes Serological identification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung cancer. p53 Antibodies in the sera of patients with various types of cancer: a review. Expression of cancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a new CT gene, CT9. CT10: A new cancer‐testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational‐difference analysis p53 gene mutation, tumor p53 protein overexpression, and serum p53 autoantibody generation in patients with breast cancer. Cancer-testis antigens and ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-specific ING1 transcripts and a homologue gene. Antigens recognized by autologous antibody in patients with renal‐cell carcinoma SYFPEITHI: database for MHC ligands and peptide motifs Cleavage by Granzyme B Is Strongly Predictive of Autoantigen Status Humoral immune responses of cancer patients against \"Cancer-Testis\" antigen NY-ESO-1: correlation with clinical events. The telomerase catalytic subunit is a widely expressed tumor-associated antigen recognized by cytotoxic T lymphocytes. Isoforms of the human PDZ-73 protein exhibit differential tissue expression. The 72-kDa Component of Signal Recognition Particle Is Cleaved during Apoptosis* The Central Role of CD4+ T Cells in the Antitumor Immune Response Characterization of human colon cancer antigens recognized by autologous antibodies A Survey of the Humoral Immune Response of Cancer Patients to a Panel of Human Tumor Antigens Human lung cancer antigens recognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor gene locus on chromosome 3p21.3. High-titer HER-2/neu protein-specific antibody can be detected in patients with early-stage breast cancer. SSX: A multigene family with several members transcribed in normal testis and human cancer Autoantibodies to p53 in ovarian cancer patients and healthy women: a comparison between whole p53 protein and 18-mer peptides for screening purposes. A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening. Humoral immune response against p53 protein in patients with colorectal carcinoma DNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to detect all known p53 alterations in human malignancies. Anti-p53 antibodies in sera from patients with chronic obstructive pulmonary disease can predate a diagnosis of cancer. Anti-p53 antibodies in sera of workers occupationally exposed to vinyl chloride. Absence of p53 autoantibodies in sera from glioma patients. A motif in human histidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-coil that is essential for enzymatic activity and contains the major autoantigenic epitope. Clinical utility of the immunocytochemical detection of p53 protein in cytological specimens. Role of bone marrow-derived cells in presenting MHC class I-restricted tumor antigens. Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains. Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-macrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor immunity. Development of antibodies against p53 in lung cancer patients appears to be dependent on the type of p53 mutation. HLA-A2 molecules in an antigen-processing mutant cell contain signal sequence-derived peptides Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry. Very long charge runs in systemic lupus erythematosus-associated autoantigens. Characterization of naturally occurring minor histocompatibility peptides including H-4 and H-Y. The fine specificity of anti-La antibodies induced in mice by immunization with recombinant human La autoantigen. Post-translational modifications of proteins: some problems left to solve. CA125 Antigen Levels in Obstetric and Gynecologic Patients Purification of a human prostate specific antigen. Detection of a transformation-related antigen in chemically induced sarcomas and other transformed cells of the mouse. SPECIFIC CARCINOEMBRYONIC ANTIGENS OF THE HUMAN DIGESTIVE SYSTEM Immunotherapy (APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3 randomized placebo-controlled trial Efficient purification of unique antibodies using peptide affinity-matrix columns. Endogenous immune response to early- and intermediate-stage melanoma is correlated with outcomes and is independent of locoregional relapse and standard prognostic factors. Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem cell transplantation and in healthy female donors. Recombinant antigen expression on yeast surface (RAYS) for the detection of serological immune responses in cancer patients. Immunotherapy of hematologic malignancy. p53 autoantibodies can be indicative of the development of breast cancer relapse. The autoantibody repertoire: searching for order MUC1-like tandem repeat proteins are broadly immunogenic in cancer patients. Epitope identification and vaccine design for cancer immunotherapy. Immunotherapeutic Targets Cancer : Identification of Potential Diagnostic and Cancer-related Serological Recognition of Human Colon Updated Identification of tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by serological screening of a testicular library with breast cancer serum. Humoral immunity to human breast cancer: antigen definition and quantitative analysis of mRNA expression. Monitoring of p53 autoantibodies after resection of colorectal cancer: relationship to operative curability. Gure, A. O. et al. CT10: a new cancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-difference analysis. Int. J. Cancer 85, 726-732 MHCPEP, a database of MHC-binding peptides: update 1997 Cancer,Facts and Figures about."
        ],
        [
         "38",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "http://www.jbc.org/article/S0021925820792379/pdf",
         "Chemistry,Medicine",
         "Journal of Biological Chemistry",
         "E. Gagnon,M. Bergeron,Nikolas D. Daigle,Marie-Hélène Lefoll,P. Isenring",
         "117842903,5297299,40563519,4943153,144981400",
         "30",
         "1.0",
         "2005-09-16",
         "2005",
         "",
         ""
        ],
        [
         "39",
         "00295936451c8a6b8a982d6ba9ab64cb647370ae",
         "https://academic.oup.com/ajcn/article-pdf/81/2/440/23875105/440.pdf",
         "Medicine,Biology",
         "American Journal of Clinical Nutrition",
         "Kerry‐Ann da Costa,Christopher E. Gaffney,L. Fischer,S. Zeisel",
         "4038445,2054146022,38060603,2804284",
         "156",
         "1.0",
         "2005-02-01",
         "2005",
         "4d4669217589611bb901b17698fcb18a144740a7,0263d8843e92dc3e34506ad3fde379fad828f8a1,e2bd4ffcddd7af7c8b5f94cc49601aff57a7a21c,b72287e075cef4d3f7c8856498688358f61397da,9a4483b65e2c4c7b930f90b1d4a922a5dfee0ba6,d1395d688cec8093ddcccf1cc9d6ae04b0d3c23a,4efa51fbe900205a49bc7f6a68154ad4296eea0b,f204856fc5373437f814c0c2402778c27594ef2b,d2b625d574ca8fa283873237b1349fb49c785059,603601d2c4ef1e4412f1d89dea14c835213b83a0,b04ea587a39c3720482f3692ed743f98bd4bbee8,e9d74e7b48df8378510d1d6bd25a27f3fba5f150,a2e6970a571def1f90636db927b6ef9c59f69482,0873ef5d9de496dcdaa33415423647019752d873,eb9f6bcc3f5bdb4716c37c1e35c6db8997190097,ed6edc00cc4726200771bf11829207f2683462d9,0cad9d3316e0a9010cb0dacbb79e5f58e457c642,6a3444760b5ebe174e24c8d29fd97b5afa743ee6,b4c067e2e1b52ef152581aa4c2e9f40c072f6208,a957011ab44cca7976f56e23536c3d89fc14f6bd,49f5e5d64de07d62c6b2b0c8e03b20eaf1161d3a,9431c778f18d1f51cfb8e95c5035b4f3216c6998,e1ec7fbfe68d62be64eb8a7666d8ac6ec891ff13,008a3ea9d8e594a215ca16720893f456f7787cc5,a32c15814b6175e935ee3477514c4342b2242ccb,20cb3ad6e81475861e92b6d477d3d8c3386818c1,2e29ce6a2b65aa08a1bfe0a80b45b74d72edb6e7,569d20baf31b7d9cf72e621c30b0105bcb1abe6f,222b5b859bf2972cf7b74b1af5c1301a2c83a701,e426f902f9e694c64b430208317433c7f930d7f8,e24f25dba6659b9c7410676bd5a08b6b5caef2bf,9a395ca341ae350e2c374b3251865ccb1e4dd5c0",
         "Choline- and betaine-defined diets for use in clinical research and for the management of trimethylaminuria. Elevated serum creatine phosphokinase in choline-deficient humans: mechanistic studies in C2C12 mouse myoblasts. Betaine as a Determinant of Postmethionine Load Total Plasma Homocysteine Before and After B-Vitamin Supplementation Folate status response to controlled folate intake is affected by the methylenetetrahydrofolate reductase 677C-->T polymorphism in young women. Effect of folic acid on fenofibrate-induced elevation of homocysteine and cysteine. Betaine supplementation lowers plasma homocysteine in healthy men and women. Concentrations of choline-containing compounds and betaine in common foods. Phosphatidylethanolamine N-methyltransferase (PEMT) knockout mice have hepatic steatosis and abnormal hepatic choline metabolite concentrations despite ingesting a recommended dietary intake of choline. Effects of cytidine 5'-diphosphocholine on plasma homocysteine levels in rat. Quantitation of choline and its metabolites in tissues and foods by liquid chromatography/electrospray ionization-isotope dilution mass spectrometry. Hyperhomocysteinemia due to methionine synthase deficiency, cblG: structure of the MTR gene, genotype diversity, and recognition of a common mutation, P1173L. Choline deficiency causes reversible hepatic abnormalities in patients receiving parenteral nutrition: proof of a human choline requirement: a placebo-controlled trial. The 1298A-->C polymorphism in methylenetetrahydrofolate reductase (MTHFR): in vitro expression and association with homocysteine. Determinants of plasma total homocysteine concentration in the Framingham Offspring cohort. Betaine-homocysteine methyltransferase expression in porcine and human tissues and chromosomal localization of the human gene. The use of homocysteine and other metabolites in the specific diagnosis of vitamin B-12 deficiency. Choline deficiency: A cause of hepatic steatosis during parenteral nutrition that can be reversed with intravenous choline supplementation Evidence that homocysteine is an independent risk factor for atherosclerosis in hyperlipidemic patients. Vitamin B-6 deficiency vs folate deficiency: comparison of responses to methionine loading in rats. AIN-93 purified diets for laboratory rodents: final report of the American Institute of Nutrition ad hoc writing committee on the reformulation of the AIN-76A rodent diet. Low plasma free choline is prevalent in patients receiving long term parenteral nutrition and is associated with hepatic aminotransferase abnormalities. Accumulation of 1,2-sn-diradylglycerol with increased membrane-associated protein kinase C may be the mechanism for spontaneous hepatocarcinogenesis in choline-deficient rats. Rapid high-performance liquid chromatographic assay for total homocysteine levels in human serum. Choline, an essential nutrient for humans Lactobacillus casei microbiological assay of folic acid derivatives in 96-well microtiter plates. The active synthesis of phosphatidylcholine is required for very low density lipoprotein secretion from rat hepatocytes. Plasma choline concentration in humans fed parenterally. Betaine in the treatment of homocystinuria due to 5,10-methylenetetrahydrofolate reductase deficiency A rapid method of total lipid extraction and purification. Effect of methotrexate on homocysteine and other sulfur compounds in tissues of rats fed a normal or a defined, choline-deficient diet Pathways and Regulation of Homocysteine Metabolism in Mammals Institute of Medicine, National Academy of Sciences USA. Dietary reference intakes for folate, thiamin, riboflavin, niacin, vitamin B-12, panthothenic acid, biotin, and choline The biosynthesis of choline and its relation to phospholipid metabolism."
        ],
        [
         "40",
         "002a108f32eb157c0b88aea3c6f29d4b614c3b78",
         "https://academic.oup.com/toxsci/article-pdf/86/1/161/4657319/kfi172.pdf",
         "Biology,Medicine",
         "Toxicological Sciences",
         "C. Bowman,K. Turner,M. Sar,N. Barlow,K. Gaido,P. Foster",
         "5835636,121132152,4754930,39737755,4832338,41131499",
         "34",
         "0.0",
         "2005-07-01",
         "2005",
         "",
         ""
        ],
        [
         "41",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         "https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-10/issue-5/051604/Prospective-grading-of-neoplastic-change-in-rat-esophagus-epithelium-using/10.1117/1.2102767.pdf",
         "Medicine",
         "Journal of Biomedical Optics",
         "A. Wax,J. Pyhtila,R. Graf,R. Nines,C. Boone,R. Dasari,M. Feld,V. Steele,G. Stoner",
         "6601645,2596802,39855573,5362380,48807863,6398265,145506659,5086274,3306303",
         "65",
         "2.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "42",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "https://access.archive-ouverte.unige.ch/access/metadata/414fbfe1-4481-4240-92c1-e62a3593f0f0/download",
         "Psychology,Medicine",
         "Emotion",
         "T. Brosch,D. Sharma",
         "2256291,34546725",
         "153",
         "10.0",
         "2005-09-01",
         "2005",
         "804eeefaf41247d1927a7e9d4ab27f06c3917fab,24cea7f3a3d8e4d490725d55933a724421b8c086,e35f63ac711e7253049a4811a354b93478d4959e,240247d67cc26cd7474e02ed75de6682e181839d,494771f2b2bcff4d795dc4e974524c5a7ec5572f,2da6cce61dd4defa4c44e82d1e0bb9920ee193bb,3542ff0be48d6196677c85fb6b65dbf7180d0e46,bdfeee608d25196733e80800f43a57b768a039ab,fbe379683d148cdf82a03010f1e79f22d16657e6,4f3ef78375b4c147ab794ca9a8e9bca1fe4daf4e,76361a44e145732a39dbc68d9418871038c83be2,6871c3babd4f47895d616d15b17b7ba471afa89a,bc44f92d97e409449ce3af72f7d2cb4ac544245a,c26f4aef22b2682db192d60b932455df09702e92,d506a8fdc86aeb83c28d4b5a87cf35e3c0e14910,46c99f0a5935572dd8b19936afef8faa8de25fbf,05c2b29d567263cb034f28eefe078545ad357d73,bdeee63dc6398e2d9d1fe9571d23dcf141903ee1,2445bf8b2442a04e9a4db6819aaa267f8e45f4a7,9508efad2d667fa38e4c9a50efd91c117a7a7943,74257ff0d6645ab73650fdc9faaf8fe1a6a430b5,7f57c5462b5728661608c308e76be0e2f4a0408a",
         "Attentional control of the processing of neural and emotional stimuli. Searching for threat Emotion drives attention: detecting the snake in the grass. The face in the crowd revisited: a threat advantage with schematic stimuli. Fear and anxiety: animal models and human cognitive psychophysiology. Facial Expressions of Emotion: Are Angry Faces Detected More Efficiently? A subcortical pathway to the right amygdala mediating \"unseen\" fear. It Takes a Confounded Face to Pop Out of a Crowd On the automatic nature of phobic fear: conditioned electrodermal responses to masked fear-relevant stimuli. Orienting and schizophrenia: stimulus significance, attention, and distraction in a signaled reaction time task. A feature-integration theory of attention The premise of equipotentiality in human classical conditioning: conditioned electrodermal responses to potentially phobic stimuli. One-trial learning and superior resistance to extinction of autonomic responses conditioned to potentially phobic stimuli. Phobias and preparedness The cognitive neuroscience of emotion Fears, phobias, and preparedness: toward an evolved module of fear and fear learning. Unconscious emotion : Evolutionary perspectives, psychophysiological data, and neuropsychological mechanisms A threat imminence approach to human fear responding : direction of threath, aversive contexts, and electrodermal responses Preparedness and electrodermal fear-conditioning: ontogenetic vs phylogenetic explanations. Finding the face in the crowd: an anger superiority effect. Feature analysis in early vision: evidence from search asymmetries. The emotional brain. E-Prime 32306-1270), as editor of the Clinician's Research Digest newsletter for"
        ],
        [
         "43",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "https://www.jstage.jst.go.jp/article/expanim/54/5/54_5_461/_pdf",
         "Medicine",
         "Experimental animals",
         "M. Asakawa,T. Yoshioka,I. Hikita,T. Matsutani,T. Hirasawa,A. Arimura,T. Sakata,T. Horikawa",
         "50156427,2001345,5223435,5352485,34764605,47810324,48565673,143893766",
         "23",
         "1.0",
         "2005-10-01",
         "2005",
         "abd6726370ffc4c6e7ed4b64164bf5d2b42e1fcb,d1076a4457fff22f01c397ef080a731c23e5e807,a884242808a1ace5f72ba4d59c97a6360e3644e7,2571e34834098d5d1ec06e89c81458ec84a41b8e,6c644bd483d2639bcfddeaf8bcb696cd2df1ef77,e056d6b051a9582e3f3f1d81d84a99b329946b64,b3327aae68ba3bd0c99e3dc1504d0a118d9fa133",
         "Role of the Nh (Non-hair) mutation in the development of dermatitis and hyperproduction of IgE in DS-Nh mice. DS–Nh as an experimental model of atopic dermatitis induced by Staphylococcus aureus producing staphylococcal enterotoxin C Characterization of dermatitis arising spontaneously in DS-Nh mice maintained under conventional conditions: another possible model for atopic dermatitis. Atopic dermatitis: new insights and opportunities for therapeutic intervention. Locus of dominant hairless gene (Ht) causing abnormal hair and keratinization maps to rat chromosome 10. Characteristics of spontaneous erythema appeared in hairless rats. IgE production by normal human lymphocytes is induced by interleukin 4 and suppressed by interferons gamma and alpha and prostaglandin E2."
        ],
        [
         "44",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "http://www.cell.com/article/S009286740500098X/pdf",
         "Biology,Medicine",
         "Cell",
         "I. Krylova,E. Sablin,Jamie M. R. Moore,R. Xu,G. Waitt,J. A. MacKay,D. Juzumiene,J. Bynum,K. Madauss,V. Montana,L. Lebedeva,M. Suzawa,Jon D. Williams,Shawn P. Williams,R. Guy,J. W. Thornton,R. Fletterick,T. Willson,H. Ingraham",
         "34437421,5695081,2108562213,4605417,7978640,144066376,47866582,48218038,3502869,4051165,2106094788,40606710,51017082,23919632,144399402,3722546,4009116,5064590,3514504",
         "405",
         "18.0",
         "2005-02-11",
         "2005",
         "1d6bd7b0c8db990d8d67f496ede67ae2827cae6f,9603421bed7605ac327a47ae9982d0ab8c6eeb58,c84e4fce3b6dadc50ce3c174b3bc4759442e38e9,d91a2295136ab9944fcd756bddab143c84804b59,ce6902fec41ee6fd52c9e3eefa059bc6bb415eef,44ef03895ff4e14bf5d226ef3d05f0e097d80486,87f895a0ecdc49df01a9cbef037f898e90d2c91d,89544b7b1dfd705c9d5199e342e8cd0fa7221a51,926e9a38d4e6ee4ff925995e6625106a2513183a,cb4f09dcf20c7700c3a5c1c5e2ca4c981182ce68,f256f33b7a0cf35735fba4d5defa2c3da988e490,fe6adae44ba4ded648fe28fe3837a9d22eab5d08,63523500744c4a42596cef3a31049061b7999a0c,4bd5ee4101ea924889a1ea398fb3cdd39310ba74,7599925bf30ff65b9c4e63464a1d7d7d014c09a7,bf6d61b49f81eeea09540d0a5924f026230c8d14,69faa8f5d27ed9f6cb09a4ba5a251a52aa698fea,83011a2d9fafdebe71b685684d22e7d5e37af1f7,badcfc9340eee368b92961ba52fb94012a7f031f,3902088e094524712a09886b265e77cb3d5d4bb3,701fab8b80fffcd0765465a8ed7a523fa3429208,c740aba3971f2e536e4e70ae166ba46debd24086,52ca6d637dd35ebbe5659f31342260a24b354988,2a9dce8ca5350d4137a5debd70e144e145e0468e,bada3a161bb601bd7b0ef07a901379d8ff22298c,2f1bdce0bca9ff5556e0c237c232b92fc64fe1e5,ee5445492daa6341cab66a5ecf24aa041bb6129f,b3ac08c338b82741e97e0671aea2a3381cd0a57b,bac05ffabbbe20bbfd3f9ac7bf2bb0d627b364e0,6160c4d09e8745db0d669351d138547cfab8909c,885007d7cdff2b2f9330a93eb9d568d0bc76403a,2bb11f7aae32348cc314171336d2e27bebf32513,d615c6e4c2161a309b9c63972867db1adf646a7e,2a8fa95224b6a8a54368262f621e3bb36e6a62c6,eb4c2bdb02b2fbca400d8a645440fcf16d25b0e5,2749817f25ac423a826977acad0284405c163d0e,67c9984328066bf9019152a7c4c325941694bfa7,193b40d2589ce17655a376a82f32747b6edaafc0,91b1bd09d655da66f30a0b2b416be982383e4f93,363c04c59e1ccf958552fe68eed66e274977c70d,f9845ded6f23c4d2e620a88f7a765ec38dc22bef,8df18f32bb193c4c9cfae684ea4a80d616601331,20cc588436334fd28ca613a68cffe81c9ec29844,ecbf24f75bd3d9dd4b3b303994f77850f57e2ca7,847eab3db971b86aa434f2a60ffeef7317be5d42,0aef55484c9bcde366848592dd43b45aa85d55a5,4ab6abd506eb7b4b10836e83a03ff274278bb0c5,cbb0f3690661c288f1a383293a67d42218081e2c,1117b106700a5d022a3af1103c9c370f95ea0fcc,e08c7acadbbeada1f429d43231aa02d92c3b3ba1,6332dfdced172ca5ffa1971c1b285c5ab3119281",
         "The DEAD-Box Protein DP103 (Ddx20 or Gemin-3) Represses Orphan Nuclear Receptor Activity via SUMO Modification Amplification of the steroidogenic factor 1 gene in childhood adrenocortical tumors. Identification of liver receptor homolog-1 as a novel regulator of apolipoprotein AI gene transcription. Synergy between LRH-1 and beta-catenin induces G1 cyclin-mediated cell proliferation. Quantitative Proteomics of the Thyroid Hormone Receptor-Coregulator Interactions*[boxs] LRH-1: an orphan nuclear receptor involved in development, metabolism and steroidogenesis. Sex determination: a ‘window’ of DAX1 activity Generation and Characterization of Human Steroidogenic Factor 1 LBD Crystals with and without Bound Cofactor Peptide SHIP-2 and PTEN Are Expressed and Active in Vascular Smooth Muscle Cell Nuclei, but Only SHIP-2 Is Associated with Nuclear Speckles* Resurrecting the Ancestral Steroid Receptor: Ancient Origin of Estrogen Signaling MrBayes 3: Bayesian phylogenetic inference under mixed models Activation of nuclear receptors: a perspective from structural genomics. StAR-related Lipid Transfer (START) Proteins: Mediators of Intracellular Lipid Metabolism* The Drosophila Orphan Nuclear Receptor DHR38 Mediates an Atypical Ecdysteroid Signaling Pathway Structural basis for ligand-independent activation of the orphan nuclear receptor LRH-1. Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors Nuclear lipid signalling Nuclear lipids: New functions for old molecules? X-Ray Structure of the hRORα LBD at 1.63 Å Elevation of plasma phospholipid transfer protein in transgenic mice increases VLDL secretion Published, JLR Papers in Press, August 16, 2002. DOI 10.1194/jlr.M200166-JLR200 Crystal Structure of the HNF4α Ligand Binding Domain in Complex with Endogenous Fatty Acid Ligand* Phosphorylation and Intramolecular Stabilization of the Ligand Binding Domain in the Nuclear Receptor Steroidogenic Factor 1 Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids. Inhibition of androgen receptor (AR) function by the reproductive orphan nuclear receptor DAX-1. Control of Cholesterol Turnover in the Mouse* Phosphatidic Acid-Mediated Mitogenic Activation of mTOR Signaling The PITP family of phosphatidylinositol transfer proteins Characterization of a G Protein-activated Phosphoinositide 3-Kinase in Vascular Smooth Muscle Cell Nuclei* Binding of ligands and activation of transcription by nuclear receptors. Structure of a Multifunctional Protein Crystal Structure of the Ligand-binding Domain of the Ultraspiracle Protein USP, the Ortholog of Retinoid X Receptors in Insects* Haploinsufficiency of steroidogenic factor-1 in mice disrupts adrenal development leading to an impaired stress response. Nuclear Lipid Signaling Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors. Ligand binding and nuclear receptor evolution A new method to localize and test the significance of incongruence: detecting domain shuffling in the nuclear receptor superfamily. Inferring the historical patterns of biological evolution Oxysterol Regulation of Steroidogenic Acute Regulatory Protein Gene Expression Structure and specificity of nuclear receptor-coactivator interactions. 25-Hydroxycholesterol is not a ligand for the orphan nuclear receptor steroidogenic factor-1 (SF-1). Wilms' Tumor 1 and Dax-1 Modulate the Orphan Nuclear Receptor SF-1 in Sex-Specific Gene Expression Activation of the orphan nuclear receptor steroidogenic factor 1 by oxysterols. DAX-1 inhibits SF-1-mediated transactivation via a carboxy-terminal domain that is deleted in adrenal hypoplasia congenita A new method of inference of ancestral nucleotide and amino acid sequences. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. The nuclear receptor steroidogenic factor 1 acts at multiple levels of the reproductive axis. A cell-specific nuclear receptor is essential for adrenal and gonadal development and sexual differentiation Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology Repression of orphan nuclear recep - or activity via SUMO - modification and the Dead - Box Protein P 103 ( Ddx 20 , Gemin - 3 ) Nuclear lipids: key signaling effectors in the nervous system and other tissues X-ray structure of the hRORalpha LBD at 1.63 A: structural and functional data that cholesterol or a cholesterol derivative is the natural ligand of RORalpha. Nuclear-receptor ligands and ligand-binding domains. Standard Fmoc protocols. Methods Enzymol"
        ],
        [
         "45",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.HYP.0000184251.01159.72",
         "Chemistry,Medicine",
         "HYPERTENSION",
         "C. Zeng,Z. Wang,U. Hopfer,L. Asico,G. Eisner,R. Felder,P. Jose",
         "49386778,48708059,4337525,6316449,38877762,1955072,3561892",
         "39",
         "1.0",
         "2005-10-01",
         "2005",
         "",
         ""
        ],
        [
         "46",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         "https://jcp.bmj.com/content/jclinpath/58/9/991.full.pdf",
         "Biology,Medicine",
         "Journal of Clinical Pathology",
         "Giulio Rossi,G. Sartori,Riccardo Valli,F. Bertolini,Nazzarena Bigiani,L. Schirosi,A. Cavazza,G. Luppi",
         "2248858530,143679786,2251605562,2251555357,12435222,48791235,3835552,2251360364",
         "23",
         "3.0",
         "2005-08-26",
         "2005",
         "",
         ""
        ],
        [
         "47",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/129/2/1290171.pdf",
         "Medicine,Biology",
         "Reproduction",
         "J. Ito,M. Hirabayashi,Megumi Kato,A. Takeuchi,Mayumi Ito,M. Shimada,S. Hochi",
         "38681521,34621985,32885513,152918649,50230871,10276380,5629712",
         "41",
         "3.0",
         "2005-02-01",
         "2005",
         "",
         ""
        ],
        [
         "48",
         "002cc4c2e457c3cc0fb0f704d0483a003e8a0575",
         "https://academic.oup.com/jac/article-pdf/56/4/686/2308205/dki309.pdf",
         "Medicine,Biology",
         "Journal of Antimicrobial Chemotherapy",
         "L. Halbert,J. Kaneene,L. Mansfield,P. Ruegg,L. Warnick,S. Wells,C. Fossler,A. Campbell,Angela M Geiger-Zwald",
         "12508766,5363293,3666808,6381510,5534287,32612292,12064236,144131845,1422352824",
         "18",
         "1.0",
         "2005-10-01",
         "2005",
         "",
         ""
        ],
        [
         "49",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "http://arxiv.org/pdf/1111.4445.pdf",
         "Mathematics,Computer Science",
         "SIAM Journal on Numerical Analysis",
         "Y. Kuznetsov,W. Govaerts,E. Doedel,Annick Dhooge",
         "1775504,1797969,2966430,2713848",
         "35",
         "2.0",
         "2005-04-01",
         "2005",
         "56ff3caaf8a0dd6ed415f8b84f6c8f9041d2a0fb,6ed02cffaaf644f63912f41fbfa00f6f4406c464,7838818b776dd024c885b857c56815ddf215cf60,435ab7306b96edfc3fad6fe165e22d6d6abe531b,526c44f9a4fdac6824d7b1813869649797defd08,3239959373f015e193b6493d7fd64b603547c6a2,8b6f7d0b305d6cba163ef568cdc361ada56e6b7a,cd060346f7d865fff39eb01922207776811cdfa4,b32aba832ca2be4a84c687b040bc39fc8a8413bb,09f52abb09d783865c4ef2cf5af6454802bd2b93,42763fcda07ec7c1a6a93aba8c8f83352c6299c3,cf2c8d80c7bcba7a507801c6e3da8cac70b85bd8,0cbe5c98f1b9c0273629c41745fbec5750b6cf9c,23a8fe3d34a78f2a2fe3f8b0ff32919586dbe96b,340d7ac3b6e13a63e0c47b46543e4377a55f6963,2b6adbad7d8f4c5f1ade291264df221e1b7e22eb,fc93cfbfe36648f105fd4034bbc2471c3367dff8,a594a834dd17b28e2b5ddd42cbd1d9e62efc1b7a,90c0d56498401779d928cd4d8d24c66705fad561,93b1a0baf18ec6b3278805931585392000877ff8,6df4bbd7d59ccce8b034ad9eca8ee1c5a4355592,54cce58d16bd817c766c9495c57eac4f77993b71,1cdcc6db6b8139eb1c1110ba44933727760a17dd,1c46bd1bfbfe4248aa642f5210841379d5923763,d84c4a0dc989082a28956c95ba8ff3a220182091,d7cbbcb3b90deef39fc3b1f4e2eecb80e78a3c08,17be3ad189a2e01f96648ab4b890ddc1c2cf641f,c2e294e7b709dcdae66b600a3c8d3fa459745d0f,2ce5b9b35b5941749cb614e72a69e7ee0de9184c,d35064f6159576e2d9a49a443f8abcdb681cc49a,70e3da6c426ca384f78f77474cbbf00a436038a2,52c0a963506ad2df44e4d5e4358b1e12f500b983,747fa5fcf394aeedc790488ce725a35e3351f310,0b41448a2d92cccfcf4a9ce1e0d68371ce200cfb,39f578060187f86bc80f645b39d735479e664d86,03894e8313ad4c1b36af338151a01baa88cc479e,eea110aa0ea13cc090cebe08f30c3178cc0ad359,7921f44af253340514c9c98af417b0cb07c75afd,4ae84da06e7dbb22fcf248ce4bfa31569dfbad6b,2f59d80b6cccd2b0a28a437d81f360c9fc701287,36ba81e158b864ed1576224f83da232b6031465c,6c1638a43f9bab7409de246138866814cffde0f2,b862a20ee79a56c79e03701fefd4c5b14d53a542",
         "Numerical Methods for Two-Parameter Local Bifurcation Analysis of Maps Newton's descent method for the determination of invariant tori. A Newton Method for Locating Invariant Tori of Maps Sensitivity Analysis of ODES/DAES Using the Taylor Series Method Numerical Normal Forms for Codim 2 Bifurcations of Fixed Points with at Most Two Critical Eigenvalues Numerical Methods for the Continuation of Invariant Tori MATCONT: A MATLAB package for numerical bifurcation analysis of ODEs Cl_matcont: a continuation toolbox in Matlab Computation of Periodic Solution Bifurcations in ODEs Using Bordered Systems Chapter 4 – Numerical Continuation, and Computation of Normal Forms Computing Periodic Orbits and their Bifurcations with Automatic Differentiation Numerical Normalization Techniques for All Codim 2 Bifurcations of Equilibria in ODE's Topics in bifurcation theory and applications Elements of applied bifurcation theory (2nd ed.) Harmonic balance analysis of period-doubling bifurcations with implications for control of nonlinear dynamics Algorithm 755: ADOL-C: a package for the automatic differentiation of algorithms written in C/C++ Bifurcation and predictability analysis of a low-order atmospheric circulation model Models of complex dynamics in nonlinear systems Bifurcation sequences at 1:4 resonance: an inventory Harmonic balance methods for the analysis of chaotic dynamics in nonlinear systems BIFURCATIONS AND CHAOS IN A PERIODIC PREDATOR-PREY MODEL NUMERICAL ANALYSIS AND CONTROL OF BIFURCATION PROBLEMS (II): BIFURCATION IN INFINITE DIMENSIONS Numerical Calculation of Invariant Tori THE QUASIPERIODIC ROUTE TO CHAOS IN A MODEL OF THE PEROXIDASE-OXIDASE REACTION Global Characterization of the Normal Form for a Vector Field Near a Closed Orbit A simple global characterization for normal forms of singular vector fields Normal form reduction for time-periodically driven differential equations Numerical computation of invariant circles of maps NONLINEAR OSCILLATIONS, DYNAMICAL SYSTEMS, AND BIFURCATIONS OF VECTOR FIELDS (Applied Mathematical Sciences, 42) Irregularity: a fundamental property of the atmosphere* Nonlinear Oscillations, Dynamical Systems, and Bifurcations of Vector Fields Collocation at Gaussian Points Continuation of Quasi-periodic Invariant Tori The fold-flip bifurcation On the computation of characteristic multipliers for predicting limit cycle bifurcations Elements of applied bifurcation theory AUTO 2000 : CONTINUATION AND BIFURCATION SOFTWARE FOR ORDINARY DIFFERENTIAL EQUATIONS (with HomCont) CONTENT: A multiplatform environment for analyzing dynamical systems Normal forms of bifurcating periodic orbits Numerical computation of periodic solution branches and oscillatory dynamics of the stirred tank reactor with A → B → C reactions Geometrical Methods in the Theory of Ordinary Differential Equations Elementary stability and bifurcation theory Bifurcation of maps and applications Topics in bifurcation theory Numerical solution of boundary value problems in ordinary differential equations Numerical Continuation, And Computation Of Normal Forms CAPD: Computer assisted proofs in dynamics"
        ]
       ],
       "shape": {
        "columns": 12,
        "rows": 69988
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>fieldsOfStudy</th>\n",
       "      <th>venue</th>\n",
       "      <th>authorName</th>\n",
       "      <th>authorId</th>\n",
       "      <th>citationCount</th>\n",
       "      <th>influentialCitationCount</th>\n",
       "      <th>publicationDate</th>\n",
       "      <th>publicationYear</th>\n",
       "      <th>referenceIds</th>\n",
       "      <th>referenceTitles</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>https://aacrjournals.org/clincancerres/article...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Clinical Cancer Research</td>\n",
       "      <td>T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...</td>\n",
       "      <td>14839966,10239302,2000448727,3201672,7147569,2...</td>\n",
       "      <td>88</td>\n",
       "      <td>6.0</td>\n",
       "      <td>2005-04-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>http://www.merl.com/publications/docs/TR2005-0...</td>\n",
       "      <td>Computer Science,Mathematics</td>\n",
       "      <td>IEEE Transactions on Communications</td>\n",
       "      <td>Juntan Zhang,M. Fossorier</td>\n",
       "      <td>3193350,48891339</td>\n",
       "      <td>319</td>\n",
       "      <td>26.0</td>\n",
       "      <td>2005-03-07</td>\n",
       "      <td>2005</td>\n",
       "      <td>17d0afe4f814e5abc6252e572e3b039082bf6f07,02464...</td>\n",
       "      <td>Low-Density Parity-Check Codes Turbo decoder a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>https://www.jstage.jst.go.jp/article/bpb/28/8/...</td>\n",
       "      <td>Medicine</td>\n",
       "      <td>Biological and Pharmaceutical Bulletin</td>\n",
       "      <td>Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...</td>\n",
       "      <td>11809650,6617669,123635715,11305673,2152146096...</td>\n",
       "      <td>24</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2005-08-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/pdfdirect/...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Developmental Dynamics</td>\n",
       "      <td>Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....</td>\n",
       "      <td>35690923,2114125286,2113252574,2134773,3658233</td>\n",
       "      <td>53</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2005-01-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820589394/pdf</td>\n",
       "      <td>Chemistry,Medicine</td>\n",
       "      <td>Journal of Biological Chemistry</td>\n",
       "      <td>M. Westfall,A. M. Lee,D. Robinson</td>\n",
       "      <td>5025983,49249911,48706510</td>\n",
       "      <td>70</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2005-12-16</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69983</th>\n",
       "      <td>fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6</td>\n",
       "      <td>https://academic.oup.com/humrep/article-pdf/20...</td>\n",
       "      <td>Medicine,Biology</td>\n",
       "      <td>Human Reproduction</td>\n",
       "      <td>S. Doi,M. Al-Zaid,P. Towers,C. Scott,K. A. Al-...</td>\n",
       "      <td>40151220,1404195567,37556460,78745210,1403335938</td>\n",
       "      <td>46</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2005-09-01</td>\n",
       "      <td>2005</td>\n",
       "      <td>3d2aaa390e58ff4d10cb2b6bfb4813e48565998c,8abd1...</td>\n",
       "      <td>Predictors for treatment failure after laparos...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69984</th>\n",
       "      <td>fffbc9b37f20515de2ff4fe60353d52dc37e65f5</td>\n",
       "      <td>http://www.cs.virginia.edu/~jl3aq/courses/CS85...</td>\n",
       "      <td>Computer Science</td>\n",
       "      <td>CCGrid 2005. IEEE International Symposium on C...</td>\n",
       "      <td>A. Barak,A. Shiloh,Lior Amar</td>\n",
       "      <td>144214996,40399772,34848329</td>\n",
       "      <td>42</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2005-05-09</td>\n",
       "      <td>2005</td>\n",
       "      <td>1713c96f258a9925b3ca0d814fdd0641c07bb3f1,3342b...</td>\n",
       "      <td>From sandbox to playground: dynamic virtual en...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69985</th>\n",
       "      <td>fffd2d6d54b4023ece28bfdebffdfb00502a09ec</td>\n",
       "      <td>https://europepmc.org/articles/pmc543423?pdf=r...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Molecular and Cellular Biology</td>\n",
       "      <td>Ying Huang,R. Intine,Amy M Mozlin,Samuel A. Ha...</td>\n",
       "      <td>48355811,6534085,15003627,5662843,3935489</td>\n",
       "      <td>64</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2005-01-01</td>\n",
       "      <td>2005</td>\n",
       "      <td>49bb10549894f0596e527139cf4a3bfb28fd6f2c,ced74...</td>\n",
       "      <td>Nuclear surveillance and degradation of hypomo...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69986</th>\n",
       "      <td>ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820789519/pdf</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Journal of Biological Chemistry</td>\n",
       "      <td>Faye Gordon,K. Luger,J. Hansen</td>\n",
       "      <td>48200695,3864710,34597814</td>\n",
       "      <td>143</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2005-10-07</td>\n",
       "      <td>2005</td>\n",
       "      <td>83a02dee50f2f83987d5490c8c9501609fd8f2df,c2150...</td>\n",
       "      <td>Histone modifications: combinatorial complexit...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69987</th>\n",
       "      <td>ffffeaedbe7931d161ab532015de3be15f99b180</td>\n",
       "      <td>https://pubs.asahq.org/anesthesiology/article-...</td>\n",
       "      <td>Medicine</td>\n",
       "      <td>Anesthesiology</td>\n",
       "      <td>S. Soriano,K. Anand,C. Rovnaghi,P. Hickey</td>\n",
       "      <td>3744726,145876135,4577001,4185720</td>\n",
       "      <td>85</td>\n",
       "      <td>12.0</td>\n",
       "      <td>2005-04-01</td>\n",
       "      <td>2005</td>\n",
       "      <td>27205842511218fa3dd0576405f70622232d03ee,1caa8...</td>\n",
       "      <td>Developmental neurotoxicity of ketamine: morph...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>69988 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paperId  \\\n",
       "0      0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1      000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2      000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3      0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4      00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "...                                         ...   \n",
       "69983  fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6   \n",
       "69984  fffbc9b37f20515de2ff4fe60353d52dc37e65f5   \n",
       "69985  fffd2d6d54b4023ece28bfdebffdfb00502a09ec   \n",
       "69986  ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3   \n",
       "69987  ffffeaedbe7931d161ab532015de3be15f99b180   \n",
       "\n",
       "                                           openAccessPdf  \\\n",
       "0      https://aacrjournals.org/clincancerres/article...   \n",
       "1      http://www.merl.com/publications/docs/TR2005-0...   \n",
       "2      https://www.jstage.jst.go.jp/article/bpb/28/8/...   \n",
       "3      https://onlinelibrary.wiley.com/doi/pdfdirect/...   \n",
       "4       http://www.jbc.org/article/S0021925820589394/pdf   \n",
       "...                                                  ...   \n",
       "69983  https://academic.oup.com/humrep/article-pdf/20...   \n",
       "69984  http://www.cs.virginia.edu/~jl3aq/courses/CS85...   \n",
       "69985  https://europepmc.org/articles/pmc543423?pdf=r...   \n",
       "69986   http://www.jbc.org/article/S0021925820789519/pdf   \n",
       "69987  https://pubs.asahq.org/anesthesiology/article-...   \n",
       "\n",
       "                      fieldsOfStudy  \\\n",
       "0                  Biology,Medicine   \n",
       "1      Computer Science,Mathematics   \n",
       "2                          Medicine   \n",
       "3                  Biology,Medicine   \n",
       "4                Chemistry,Medicine   \n",
       "...                             ...   \n",
       "69983              Medicine,Biology   \n",
       "69984              Computer Science   \n",
       "69985              Biology,Medicine   \n",
       "69986              Biology,Medicine   \n",
       "69987                      Medicine   \n",
       "\n",
       "                                                   venue  \\\n",
       "0                               Clinical Cancer Research   \n",
       "1                    IEEE Transactions on Communications   \n",
       "2                 Biological and Pharmaceutical Bulletin   \n",
       "3                                 Developmental Dynamics   \n",
       "4                        Journal of Biological Chemistry   \n",
       "...                                                  ...   \n",
       "69983                                 Human Reproduction   \n",
       "69984  CCGrid 2005. IEEE International Symposium on C...   \n",
       "69985                     Molecular and Cellular Biology   \n",
       "69986                    Journal of Biological Chemistry   \n",
       "69987                                     Anesthesiology   \n",
       "\n",
       "                                              authorName  \\\n",
       "0      T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...   \n",
       "1                              Juntan Zhang,M. Fossorier   \n",
       "2      Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...   \n",
       "3      Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....   \n",
       "4                      M. Westfall,A. M. Lee,D. Robinson   \n",
       "...                                                  ...   \n",
       "69983  S. Doi,M. Al-Zaid,P. Towers,C. Scott,K. A. Al-...   \n",
       "69984                       A. Barak,A. Shiloh,Lior Amar   \n",
       "69985  Ying Huang,R. Intine,Amy M Mozlin,Samuel A. Ha...   \n",
       "69986                     Faye Gordon,K. Luger,J. Hansen   \n",
       "69987          S. Soriano,K. Anand,C. Rovnaghi,P. Hickey   \n",
       "\n",
       "                                                authorId  citationCount  \\\n",
       "0      14839966,10239302,2000448727,3201672,7147569,2...             88   \n",
       "1                                       3193350,48891339            319   \n",
       "2      11809650,6617669,123635715,11305673,2152146096...             24   \n",
       "3         35690923,2114125286,2113252574,2134773,3658233             53   \n",
       "4                              5025983,49249911,48706510             70   \n",
       "...                                                  ...            ...   \n",
       "69983   40151220,1404195567,37556460,78745210,1403335938             46   \n",
       "69984                        144214996,40399772,34848329             42   \n",
       "69985          48355811,6534085,15003627,5662843,3935489             64   \n",
       "69986                          48200695,3864710,34597814            143   \n",
       "69987                  3744726,145876135,4577001,4185720             85   \n",
       "\n",
       "       influentialCitationCount publicationDate  publicationYear  \\\n",
       "0                           6.0      2005-04-01             2005   \n",
       "1                          26.0      2005-03-07             2005   \n",
       "2                           2.0      2005-08-01             2005   \n",
       "3                           2.0      2005-01-01             2005   \n",
       "4                           3.0      2005-12-16             2005   \n",
       "...                         ...             ...              ...   \n",
       "69983                       0.0      2005-09-01             2005   \n",
       "69984                       1.0      2005-05-09             2005   \n",
       "69985                       1.0      2005-01-01             2005   \n",
       "69986                       3.0      2005-10-07             2005   \n",
       "69987                      12.0      2005-04-01             2005   \n",
       "\n",
       "                                            referenceIds  \\\n",
       "0                                                          \n",
       "1      17d0afe4f814e5abc6252e572e3b039082bf6f07,02464...   \n",
       "2                                                          \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "69983  3d2aaa390e58ff4d10cb2b6bfb4813e48565998c,8abd1...   \n",
       "69984  1713c96f258a9925b3ca0d814fdd0641c07bb3f1,3342b...   \n",
       "69985  49bb10549894f0596e527139cf4a3bfb28fd6f2c,ced74...   \n",
       "69986  83a02dee50f2f83987d5490c8c9501609fd8f2df,c2150...   \n",
       "69987  27205842511218fa3dd0576405f70622232d03ee,1caa8...   \n",
       "\n",
       "                                         referenceTitles  \n",
       "0                                                         \n",
       "1      Low-Density Parity-Check Codes Turbo decoder a...  \n",
       "2                                                         \n",
       "3                                                         \n",
       "4                                                         \n",
       "...                                                  ...  \n",
       "69983  Predictors for treatment failure after laparos...  \n",
       "69984  From sandbox to playground: dynamic virtual en...  \n",
       "69985  Nuclear surveillance and degradation of hypomo...  \n",
       "69986  Histone modifications: combinatorial complexit...  \n",
       "69987  Developmental neurotoxicity of ketamine: morph...  \n",
       "\n",
       "[69988 rows x 12 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_abstract",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_introduction",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_results",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_discussion",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_methods",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_text",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "problem",
         "rawType": "object",
         "type": "unknown"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "e17ad750-c31a-4bba-9c18-e45495af00fd",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "Disruption of Fibroblast Growth Factor Signal Pathway Inhibits the Growth of Synovial Sarcomas: Potential Application of Signal Inhibitors to Molecular Target Therapy",
         "Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which are unknown. We investigated the involvement of fibroblast growth factor (FGF) signals in synovial sarcoma and evaluated the therapeutic effect of inhibiting the FGF signal. Experimental Design: The expression of 22 FGF and 4 FGF receptor (FGFR) genes in 18 primary tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR. Effects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein kinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory effects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo. Results: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed in neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma cell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal–regulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the FGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle arrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the FGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not p38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sarcoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2. Conclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibitory molecules will be of potential use for molecular target therapy in synovial sarcoma.",
         "",
         "https://aacrjournals.org/clincancerres/article-pdf/11/7/2702/1962059/2702-2712.pdf",
         "Abstract Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which\nare unknown.We investigated the involvement of fibroblast growth factor (FGF) signals in synovial\nsarcoma and evaluated the therapeutic effect of inhibiting the FGF signal.\nExperimental Design:The expression of 22 FGF and 4 FGF receptor (FGFR) genes in18 prima-\nry tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR.\nEffects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein\nkinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory\neffects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo.\nResults: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed\nin neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma\ncell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal ^\nregulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the\nFGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle ar-\nrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the\nFGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not\np38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sar-\ncoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2.\nConclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibi-\ntory molecules will be of potential use for molecular target therapy in synovial sarcoma.",
         "Synovial sarcoma is the most frequent soft-tissue sarcoma\n(STS) among patients in the third to fourth decade of life (1)\nand accounts for about 7% to 10% of all human STSs (2). It\npredominantly affects the lower extremities but can occur in\nany part of the body. Surgical resection with an adequate\nfor\nsurgical margin is the definitive choice of\nprimary tumors and has been shown to control\nlocal\nrecurrence (3, 4). Disseminated distant metastasis is the major\ncause of poor outcome, and several reports describing the\nresults of current therapy showed a 5-year survival rate of\naround 50% to 60% (3, 4). The efficacy of adjuvant\nchemotherapy is still a matter of debate, and the development\nof a new therapeutic approach is required to improve the\nprognosis.\nthe reciprocal\nDespite little progress in clinical treatment during the last\n20 years, cytogenetic and molecular genetic analyses have greatly\nimproved the understanding of this type of tumor, especially\nwith the discovery of\ntranslocation\nt(18;X)(q11;p11) creating an SYT-SSX fusion gene as a synovial\nsarcoma – specific genetic alteration (5, 6). Thus far, three SSX\ngenes (SSX1, SSX2, and SSX4) have been identified as a partner\nof the SYT gene, and >95% of synovial sarcoma tumors carried\none of these fusion genes (7). Although the precise function and\nthe mechanism of oncogenesis are not yet clearly shown, the\nhigh sensitivity and specificity of the SYT-SSX fusion gene in\nsynovial sarcoma have proven to be useful for molecular\ndiagnosis (8). In particular types of tumors with a specific\nreciprocal translocation such as PML-RARa in acute promyelo-\ncytic leukemia (9) and BCR-ABL in chronic myelogenous\nleukemia (10), fusion gene products themselves serve as targets\nfor the therapy. Although immunotherapy using a peptide\nderived from SYT-SSX protein as a specific vaccine has been\ninvestigated (11), no therapeutic approach has been discovered\nfor directly targeting the fusion protein or its function.\nGene expression profiling of tumors has been shown a\npowerful tool with which to isolate a molecular target for\ntherapy (12). We have done a gene expression analysis of\nsynovial sarcoma using a genome-wide cDNA microarray and\nfound that synovial sarcoma shared its molecular signature with\nmalignant peripheral nerve sheath tumor, of which the\nprecursors were neural crest – derived cells, and also identified\na set of genes commonly up-regulated in synovial sarcoma\nincluding the fibroblast growth factor 18 (FGF18) gene (13). The\nFGF signaling pathway seems to play significant roles in tumor\ndevelopment and progression (14 – 16), and recently FGF18 was\nidentified as an autocrine growth factor involved in colon\ncancers (17). Thus far, 22 genes have been identified as members\nof the FGF family, and our cDNA microarray contained 10 FGF\ngenes (FGF2, FGF3, FGF4, FGF7, FGF9, FGF11, FGF12, FGF13,\nFGF18, and FGF19). We found that some FGF genes other\nthan FGF18 were also expressed in synovial sarcoma (data\nshown). Based on these results, we have done an\nnot\nintensive analysis of FGF and its receptor (FGFR) genes in\nsynovial sarcoma and also investigated whether inhibition of\nthe FGF signal is a new therapeutic modality for synovial\nsarcoma.",
         "Results\nSynovial sarcoma cell lines expressed multiple fibroblast growth\nfactor and fibroblast growth factor receptor genes. The mRNA\nexpression of 22 subtypes of FGF genes was analyzed by RT-\nPCR in five synovial sarcoma and five other cell lines (Fig. 1A).\nfive synovial sarcoma cell\nNone of the five synovial sarcoma cell lines expressed the\nFGF5, FGF6, FGF12, FGF14, FGF16, FGF17, FGF22, or FGF23\ngene, whereas the FGF2, FGF8, FGF9, FGF11, and FGF18 genes\nwere expressed in all\nlines.\nExpression of the remaining nine FGF genes varied among\ncell\nlines. The expression profile in NMS-2 (malignant\nperipheral nerve sheath tumor) was similar to that in synovial\nsarcoma, which may relate to the overall similarity of the gene\nexpression profile between synovial sarcoma and malignant\nperipheral nerve sheath tumor (13). HT1080 (fibrosarcoma)\nexpressed a similar set of FGF genes to malignant peripheral\nnerve sheath tumor but at much lower levels, and the\nexpression profile of Saos2 (osteosarcoma) was quite different\nbeing positive only for FGF1 and FGF13. Interestingly, synovial\nsarcoma shared with colon carcinoma cell lines the expression\nof some FGF genes such as FGF3 and FGF18 genes, which were\nnot expressed in mesenchymal cell lines. Secretion of these FGF\nproteins by synovial sarcoma cell lines was confirmed by the\nWestern blotting of culture supernatant. FGF18 proteins were\nclearly detected in culture supernatant of synovial sarcoma cell\nlines, although the level of protein was not completely agreed\nwith the level of RNA expression (Fig. 1C).\nIn contrast with the heterogeneous expression pattern of\nthe expression profile of receptors (FGFR) was\nligands,\nrelatively homogeneous among synovial sarcoma cell lines,\nbeing positive for all subtypes analyzed except FGFR2b, which\nis an epithelia-specific FGFR expressed in colon carcinomas\n(Fig. 1B). In addition to FGFR2b, NMS-2 was negative for\nFGFR3, and HT1080 was negative for FGFR2c. Expression of\nFGFR3 in synovial sarcoma cell lines was confirmed at the\nprotein level by the Western blotting using cell lysates of each\ncell line (Fig. 1D).\nExpression of fibroblast growth factor and fibroblast growth\nfactor receptor genes in tumor tissues.\nThe FGF2, FGF8,\nFGF9, FGF11, and FGF18 genes, which were commonly ex-\npressed in the five synovial sarcoma cell lines, were inves-\ntigated in tumor tissues of 18 synovial sarcomas and 11 STSs\nof other types (Fig. 2A). In accordance with the results in the\ncell lines, almost all 18 synovial sarcoma tumors expressed\nall of these FGF genes (Fig. 2A). Among five FGF genes, the\nFGF2, FGF9, and FGF11 genes were also expressed in some\ntumors other than synovial sarcoma. In contrast, expression of\nFGF8 and FGF18 genes was only weakly detected in other\ntumors. Expression profiles of these five FGF genes were further\nanalyzed by the quantitative RT-PCR (Fig. 2B). The level of\nexpression of FGF8 and FGF18 genes in synovial sarcoma\ntumors were significantly higher than in tumor tissues of other\ntype of tumors, whereas those of FGF2, FGF9, and FGF11 genes\nshowed no difference.\nAs for receptor genes, synovial sarcoma tumors expressed all\ntypes of FGFR genes analyzed including the FGFR2b gene\nGrowth stimulatory effects of\nfibroblast growth factors in\nsynovial sarcoma cell lines. Expression of FGF and FGFR genes\nand proteins suggested the involvement of FGFs as autocrine/\nparacrine growth factors in synovial sarcoma. Among FGF genes\nexpressed in synovial sarcoma both in vitro and in vivo, we\nfocused on the FGF2, FGF8, and FGF18 genes in subsequent\nexperiments. The FGF2 gene was selected as a gene commonly\nup-regulated in STS (Fig. 2A). FGF8 and FGF18 genes were\nselected as synovial sarcoma – specific up-regulated FGF genes\n(Fig. 2A).\nGrowth stimulatory effects of each rhFGF protein were\nserum-starved conditions. Recombinant\nanalyzed under\nhFGF8 showed a dose-dependent growth-promoting effect\nin all synovial sarcoma cell lines (Fig. 3A). On the treatment\nwith rhFGF18, only HS-SY-II\nshowed a dose-dependent\nincrease (Fig. 3B). A heterogeneous response to FGF was\nmore clearly observed in the case of rhFGF2, in which the\ngrowth of HS-SY-II\nincrease,\nwhereas a growth inhibitory effect was observed in the case\nof YaFuSS (Fig. 3C).\nshowed a dose-dependent\nActivation of mitogen-activated protein kinases by fibroblast\ngrowth factor signals. One of major signal pathways located\ndownstream of FGFR is the MAPKs, for which three subtypes are\nknown: ERK, p38 MAPKs, and c-Jun NH2-terminal kinases\n(JNK; ref. 25). To investigate the events downstream of activated\nFGFRs, the phosphorylation of these MAPKs was investigated. In\nall synovial sarcoma cell lines analyzed, ERK1/2 was phosphor-\nylated under serum-free conditions, suggesting the presence of\nendogenous stimulatory signals (Fig. 4A). The amount of\nphosphorylated ERK1/2 (pERK1/2) increased on treatment\nwith FGF8 in a dose-dependent manner in all four cell lines\nanalyzed, with the response of HS-SY-II being the highest\nlevels. A dose-dependent\nFibroblast growth factor receptor – specific inhibitors reduce the\nlines in accordance with\ngrowth of synovial sarcoma cell\ninactivation of fibroblast growth factor receptor. MAPKs in\nsynovial sarcoma are activated by intrinsic pathways, and\nexogenous FGFs can stimulate the growth of synovial sarcoma,\nsuggesting that inhibition of the intrinsic FGF signal pathway\nmay inhibit the growth of synovial sarcoma. SU5402 is known\nto inhibit FGFR autophosphorylation at an IC50 of 10 to\n20 Amol/L (22). To confirm the effect of SU5402 on synovial\nsarcoma cell lines, the phosphorylation status of FGFR3 was\nanalyzed by the immunoprecipitation and Western blotting in\nthree synovial sarcoma cell lines (YaFuSS, SYO-1, and 1273/\n99; Fig. 5A). Phosphorylated FGFR3 was detected in these cell\nlines with serum-starved condition, and the treatment with\nrhFGF18 protein increased the amount of phosphorylated\nFGFR3 (Fig. 5A), which was compatible with the autocrine/\nparacrine model of FGFs in synovial sarcoma cells. Treatment\nwith SU5402 effectively decreased the phosphorylation of\nFGFR3 into almost undetectable level (Fig. 5A), indicating that\nSU5402 can be used as an inhibitor of FGFR in synovial\nsarcoma cells. By the treatment with SU5402, the growth of all\nfive synovial sarcoma cells was inhibited in a dose-dependent\nmanner (Fig. 5B), and the IC50 value ranged from 8.5 to 17.2\nAmol/L (Table 1). The growth inhibitory effect of SU5402 for\ncell lines other than synovial sarcoma was not remarkable. The\nsame experiments were done under low-serum conditions (1%\nFBS, Fig. 5C), resulting in similar growth inhibition in all\nsynovial sarcoma cell lines. The IC50 was almost identical in\nthe three cell lines (YaFuSS, HS-SY-II, and SYO-1), whereas the\nlines (Fuji and 1273/99) required a higher\nother two cell\nconcentration of SU5402 (Table 1). Growth inhibitory effects\nof another FGFR-specific inhibitor, PD166866 (23), was also\ninvestigated under the low-serum conditions. All five synovial\nsarcoma cell lines had an IC50 value (0.08-4.7 Amol/L) much\nlower than those of other cell lines (>10 Amol/L, Table 1).\nSimilar to the data obtained using SU5402, IC50 values in\nYaFuSS, HS-SY-II, and SYO-1 were much lower than those in\nFuji and 1273/99.\n11.5 F 1.8\n15.1 F3.4\n8.5 F 2.5\n17.2 F 1.8\n12.8 F 6.1\n46.4 F 6.9\n65.4 F 2.6\n>80\n>80\n54.4 F 21.5\n11.4 F 3.6\n4.9 F 0.5\n8.6 F 1.6\n25.3 F 3.6\n29.9 F 16.6\n60.2 F 20.6\n63.7 F 27.8\n55.8 F 19.1\n>80\n78.1 F20.3\n0.08 F 0.04\n0.77 F 0.52\n0.18 F 0.11\n4.0 F 1.6\n2.7 F 0.4\n>10\n>10\n>10\n>10\n>10\nGrowth inhibition of fibroblast growth factor receptor inhibitor\nassociated with the reduction of phosphorylation of extracellular\nsignal – regulated kinase 1/2 but not of p38.\nTo investigate\nwhich signals may contribute to the growth of synovial\nsarcoma, the phosphorylation of ERK1/2 and p38 before and\nafter the treatment with SU5402 was investigated. In three\nsynovial sarcoma cell lines (YaFuSS, HS-SY-II, and SYO-1),\nwhich were relatively sensitive to the growth inhibitory effect of\nSU5402, phosphorylation of ERK1/2 under low-serum con-\nditions was completely inhibited by SU5402, whereas the\namount of phosphorylated p38 showed no change (Fig. 6A). A\nsmall amount of pERK1/2 was still observed after SU5402\ntreatment in 1273/99, whereas no significant decrease was\nobserved in Fuji, both of which showed no significant change in\nthe pp38 (Fig. 6A). In cell lines other than synovial sarcoma,\nSU5402 showed no significant changes in the amount of\npERK1/2 or pp38 (data not shown). These results suggested that\nthe signal to induce the phosphorylation of ERK1/2 in synovial\nsarcoma was sent mainly through the FGF and FGFR pathway\nand that although the FGF signal induced the phosphorylation\nof both ERK1/2 and p38, the growth stimulatory effects were\ntransmitted through ERK1/2. To further confirm that the growth\ninhibition of SU5402 was due to the reduction in pERK1/2, the\nfunction of MAPK/ERK kinase 1/2 in synovial sarcoma, which is\na kinase of ERK1/2, was inhibited by U0216. Phosphorylation\nof ERK1/2 was effectively inhibited in all five cell lines, in which\n1273/99 still had a small amount of pERK1/2 at the lower\nconcentration (Fig. 6B). BrdUrd uptake was inhibited in a dose-\ndependent manner in all five cell lines, in which 1273/99 was\nrelatively resistant at the lower concentration, consistent with\nthe status of pERK1/2 (Fig. 6C). These results further indicated\nthat the growth inhibition of SU5402 in synovial sarcoma was\nachieved through the inhibition of signals through ERK1/2.\nGrowth inhibition by fibroblast growth factor receptor inhibitor\nin synovial sarcoma is associated with cell cycle arrest. The\nfraction of each cell cycle phase was analyzed before and after\nthe treatment with SU5402 in YaFuSS cells. SU5402 signifi-\ncantly increased the G1 and subG1 fractions, and decreased the\nS fraction (Fig. 6D). These data suggested that the growth\ninhibition of synovial sarcoma by SU5402 was due to its\ninduction of G1 arrest as previously reported for other tyrosine\nkinase inhibitors (26).\nFibroblast growth factor receptor inhibitors reduced the\nFor in vivo study,\ngrowth of synovial sarcoma cells in vivo.\nwe used SYO-1 because of\nits consistency in developing\ntumor, and PD166866 was used because of its lower IC50\nvalue. To confirm its efficacy in vivo, tumor tissue was taken\n30 minutes after the injection of PD166866. We found that\n0.1 mg of PD166866 was enough to reduce the phosphory-\nlated ERK1/2 in the tumor (Fig. 7A). Based on this result,\nthe growth inhibitory effect of PD166866 (0.1 and 0.5 mg/\nbody) was analyzed in vivo. Although PD166866 was unable\nto stop the growth of the tumor completely, a significant\ninhibition of growth was observed during the period of\nadministration (P < 0.01), and the effect was retained even at\nday 21 at the higher dose (P = 0.02, Fig. 7B). No definite\nreduction of body weight or pathologic changes in vital\norgans such as liver and kidney was observed (data not\nshown). No growth inhibitory effect was observed when\nHT1080 was used instead of SYO-1 even at the higher dose\n(data not shown).\nHistologic examination of tumor tissues showed no signifi-\ncant morphologic changes or focal necrosis. The number of\nPCNA-positive cells, however, was clearly smaller in PD166866-\ntreated tumors (19.3 F 6.6%, Fig. 7D) than control tumors\n(53.0 F 2.0%, Fig. 7C), indicating that PD166866 reduced the\nnumber of cells in S phase. No such difference was observed\nwhen HT1080 was used instead of SYO-1 (data not shown).",
         "Discussion\nFGFs may activate genetic programs which promote cell\ngrowth by at least one of three general mechanisms: first, as\nmitogens\nsecond, by\npromoting angiogenesis to supply a growing tumor; and\nthird, by inhibiting apoptosis and allowing tumor cells to\ncontinue to grow beyond normal constraints. We found that\nsynovial sarcoma cells expressed a number of FGF genes, of\nwhich FGF2 , FGF8 , FGF9 , FGF11 , and FGF18 were\ncommonly expressed. The involvement of FGF2 in malignant\ntumors as an autocrine growth factor and/or a paracrine\nangiogenic factor was first shown in brain tumors (27), and\nsubsequently in non – small cell carcinoma (14). FGF8 was\noriginally identified as an androgen-induced growth factor\nfound in conditioned medium of androgen-dependent mouse\nmammary carcinoma SC-3 cells and reported to have a\ngrowth stimulatory effect (28). The human FGF8 gene was\nexpressed in breast and prostate cancers (15, 16, 29) and\nassociated with anchorage-independent proliferation and\ninvasion (30, 31). FGF9 was originally purified from the\nconditioned medium of\nline NMC-G1 and\ndesignated as glia-activating factor due to its mitogenic\nactivity toward glial cells (32). FGF11 was one of four FGF\nhomology factors, which shared structural homology with\nfailed to activate FGFRs (33). Recently,\nclassic FGFs but\nFGF11 was identified as one of the gene products enriched in\nneuronal precursors during the development of the nervous\nsystem (34). FGF18 is cloned by its homology to FGF8 and\nFGF9 (35, 36). FGF18, in association with FGF8, is expressed\nat various times and places during embryogenesis, especially\nneurogenesis (33, 37). Mitotic activity was shown in colon\ncarcinomas,\nin which the expression of FGF18 was up-\nregulated by WNT signals (17). Overall, we found that\nsynovial sarcoma expressed several FGF genes, particularly\nthose expressed in neural tissues, which may further suggest\nthe neural origin of precursor cells of synovial sarcoma as we\npreviously proposed (13). Mitotic activity of exogeneous FGFs\nwas not remarkable in some cell lines (Fig. 3). The serum-\nfree medium used in this study contained insulin, which is\nknown to potentiate the effect of FGF, and the further\naddition of insulin showed no increase of the BrdUrd uptake\n(data not shown). It is possible that the extent of autocrine\nstimulation may already be saturating so that further FGF\nstimulation does not lead to additional growth.\nThe expression of FGFR1, FGFR2c, FGFR3, and FGFR4, to\nwhich FGF2 (1c, 3c and 4), FGF8 (2c, 3c, 4), and FGF18 (2c,\n3c, 4) preferentially bind (38, 39), was confirmed in both tumor\ntissues and cell lines. The expression of FGFR2b gene, which was\ndetected in tumor tissues but not in cell lines, is intriguing.\nFGFR2b is a receptor for keratinocyte growth factor (FGF7), and\nexpressed in epithelial cells (40). As expected, all biphasic\nsynovial sarcomas expressed the FGFR2b gene, whereas the\nexpression was low in most of the monophasic synovial\nsarcomas and other STSs. It is, however, intriguing that some\nmonophasic synovial sarcomas with no apparent epithelial\nstructures expressed the FGFR2b gene at a level comparable with\nthat in biphasic synovial sarcoma (Fig. 2D). All five synovial\nsarcoma cell lines used in this study including SYO-1, which was\nestablished from a biphasic synovial sarcoma, were negative for\nthe FGFR2b gene. These results suggested that synovial sarcoma\ntumor cells may have an intrinsic mechanism to express FGFR2b\nin vivo, which was somehow inhibited or disappeared in vitro.\nAlternatively, synovial sarcoma cells positive for FGFR2b may\nhave growth disadvantages, and therefore only negative cells\nwere established as cell\nlines. Recently, single nucleotide\npolymorphism in FGFR4 is reported to associate with poor\nprognosis in high-grade soft tissue sarcomas including synovial\nsarcomas (41) and both FGF8 and FGF18 bind to FGFR4.\nAlthough the status of polymorphism was not investigated in our\nsamples, FGF signals through FGFR4 may contribute to the\naggressiveness of synovial sarcoma.\nThe binding of FGF to FGFR leads to receptor dimerization\nand tyrosine autophosphorylation, followed by downstream\nactivation of PLCg, Crk, and SNT-1/FRS2 signaling pathways\namong which the SNT-1/FRS2 is the major pathway for\nmitogenic signals (25). Phosphorylated SNT-1/FRS2 directly\nbinds to the GRB2-SOS complex, and then membrane-\nassociated RAS recruits RAF-1, which in turn activates MAPKs\n(42). MAPKs are composed of three well-characterized sub-\nfamilies, ERK, p38 MAPK, and JNK (25), and we found that the\ngrowth-promoting signal of FGF was transmitted mainly\nthrough ERK1/2. Although ERK1/2 are activated by signals\nthrough other receptor tyrosine kinases (43), phosphorylation\nof ERK1/2 was completely inhibited by FGFR inhibitors, which\nindicated that the major signal to activate ERK1/2 in synovial\nsarcoma is FGF signaling, suggesting the important role of FGF\nin synovial sarcoma.\nWe have no clear explanation for the heterogeneous response\nin synovial sarcoma cell lines to FGF2, which stimulated the\ngrowth of HS-SY-II, but inhibited the growth of YaFuSS. In\ncontrast with many studies describing the role of FGF2 as a\nmitogen, several reports showed that FGF2 acted as a growth\ninhibitor of tumors of Ewing’s sarcoma family, which are\nsupposed to be of neuroectodermal origin (44, 45). Multiple\nFGF signals are involved in the biological process in synovial\nsarcoma, and the response may depend on the cell specificity.\nNevertheless, the disruption of FGF signals was found to cause\ngrowth inhibition in all synovial sarcoma cell lines, suggesting\nthat an inhibitory molecule for FGFR will be a promising tool\nfor molecular target therapy in synovial sarcoma.\nMolecular therapy targeted to FGFR has been investigated\nwith hematologic malignancies carrying a translocation involv-\ning the FGFR gene such as chronic myeloid leukemia with the\nBCR-FGFR1 gene (46) and multiple myeloma with the MMSET-\nFGFR3 fusion gene (47). In both instances, SU5402 was found\nto effectively inhibit the growth of cell lines with an IC50\nequivalent to that in synovial sarcoma. For experiments in vivo,\nwe used PD166866 instead of SU5402 because of its lower IC50\nvalue and found that i.p. given PD166866 efficiently inhibited\ntumor causing an\nthe phosphorylation of ERK1/2 in s.c.\ninhibition of growth without significant side effects. As in the\ncase of other tyrosine kinase inhibitors, PD166866 did not\nstrongly induce apoptosis or necrosis, and therefore no\ncomplete tumor remission was observed. However, cell cycle\narrest caused by the treatment with PD166866 may enhance\nthe cytotoxic effects of anticancer drugs as previously shown in\nsome instances (48, 49), and therefore further in vivo studies\nusing the treatment combined with suitable anticancer drugs\nmay provide a promising approach for clinical application.",
         "Materials and Methods\nTumor tissues were obtained at either\nbiopsy or resection surgery and kept at 80jC. Informed consent was\nobtained from each patient, and tumor samples were approved for\nanalysis by the Ethics Committee of the Faculty of Medicine, Kyoto\nUniversity. Five human synovial sarcoma cell lines (YaFuSS, HS-SY-II,\nSYO-1, Fuji, and 1273/99) were used in this study. YaFuSS and HS-SY-II\ncells have the SYT-SSX1 fusion gene and the others have the SYT-SSX2\nfusion gene (data not shown). YaFuSS was established in our laboratory\nfrom a monophasic synovial sarcoma in a 28-year-old male. HS-SY-II\nwas a gift from H. Sonobe (Kochi University, Japan; ref. 18), SYO-1\nfrom A. Kawai (Okayama University, Japan; ref. 19), Fuji from S.\nTanaka (Hokkaido University, Japan; ref. 20), and 1273/99 from\nO. Larsson (Karolinska Institute, Sweden). Among control cell lines,\nNMS-2 (malignant peripheral nerve sheath tumor; ref. 21) was\nprovided by A. Ogose (Niigata University,\nJapan), and Saos2\n(osteosarcoma), HT1080 (fibrosarcoma), COLO205 (colon cancer),\nand SW480 (colon cancer) cells were purchased from American Type\nCulture Collection (Manassas, VA). Cells were maintained in RPMI\n1640 (Sigma-Aldrich, St. Louis, MI) for Fuji, SW480, and COLO205\nwith 10% fetal bovine serum (FBS, HyClone, Road Logan, UT) or\nDMEM (Sigma, St. Louis, MO) for other cells with 10% FBS. OPTI-\nMEM I, which contains insulin and transferrin as protein supplements\n(Invitrogen, Carlsberg, CA), was used in serum-free cultures.\nThe FGFR-specific tyrosine kinase\ninhibitor SU5402 was purchased from Calbiochem (La Jolla, CA;\nref. 22). PD166866 was provided by Pfizer Global Research and\nDevelopment (Groton, CT; ref. 23). The MAPK/extracellular signal –\nregulated kinase (ERK) kinase (MEK) inhibitor U0126 was purchased\nfrom Promega (Madison, WI; ref. 24). Recombinant human FGF18\n(rhFGF18) was purchased from Wako Pure Chemical Industry (Osaka,\nJapan),\nrhFGF8 from PeproTech (London, United Kingdom),\nand rhFGF2 from Oncogene Research Products (San Diego, CA).\nAnti-ERK1 (M12320), anti-phosphorylated-p38 (P39520), anti-pan-\np38 (P19820), anti-phosphorylated-JNK1/SAPK1 (S37220), and anti-\npan-JNK1/SAPK1 (M54920) antibodies were purchased from\nBD Biosciences PharMingen (San Diego, CA); anti-phosphorylated-\nERK1/2 (sc-7383), anti-FGFR3 (sc-123), and peroxidase-conjugated\nanti-mouse IgG (sc-2005) from Santa Cruz Biotechnology (Santa\nCruz, CA); anti-phosphotyrosine (PY20) from Zymed Laboratories\n(South San Francisco, CA); anti-FGF18 (MAB667) from R&D Systems\n(Minneapolis, MN); peroxidase-conjugated anti-rabbitimmunoglobu-\nlin (P0399) from DakoCytomation (Glostrup, Denmark).\nTotal RNA was extracted using TRIzol\nreagent (Invitrogen) following the manufacturer’s instructions. After\ntreatment with DNase I (Nippon Gene, Osaka, Japan), 1 Ag of total\nRNA was reverse transcribed for single-stranded cDNAs using oligo(dT)\nprimer and Superscript II reverse transcriptase (Invitrogen). PCR was\ndone using 1 AL of RT product in a final volume of 25 AL containing\n20 pmol each of the sense and antisense primers, 2.5 mmol/L MgCl2,\n0.2 mmol/L of each deoxynucleotide triphosphate, and 1 unit of rTaq\npolymerase (TOYOBO, Osaka, Japan). All PCRs were done using\nGeneAmp 9700 (PE Applied Biosystems, Foster City, CA). Information\nof the primers are available upon request.\nQuantitative reverse transcription-PCR (RT-PCR) analyses were done\nin selected FGF and FGFR genes with ABI PRISM 7700 Sequence\nDetection System (PE Applied Biosystems). One microliter of RT\nproduct in a final volume of 25 AL containing 12.5 AL of 2 SYBR\nGreen Mastermix (PE Applied Biosystems). Information for primers are\navailable upon requests. The mean of triplicated data was used to\ncalculate the ratio of target gene/b-actin expression. Statistical analysis\nwas done by t test after logarithmic transformation.\nTo detect FGF18 protein in the supernatant,\ncells were cultured up to 80% confluency in 100-mm dish. After\nwashing the dish, cells were further incubated in 5 mL OPTI-MEM I for\n4 days. The supernatant was harvested, mixed with 8 mL ice-cold\nacetone, and kept at 80jC for 1 hour. The mixture was centrifuged at\n10,000  g for 15 minutes, and the precipitate was suspended by lysis\nbuffer (100 AL) containing aprotinin (1 Ag/mL), leupeptin (1 Ag/mL),\npepstatin A (1 Ag/mL), and phenylmethylsulfonyl fluoride (1 mmol/L)\nfollowed by sonication and centrifugation. Twenty microliters of the\nsupernatant were electrophoresed on a 10% SDS-polyacrylamide gel\nand transferred onto a polyvinylidene difluoride membrane (Millipore,\nBedford, MA). After blocking with 3% skim milk, membranes were\nprobed with anti-FGF18 antibody at 1:1,000 dilutions for 1 hour and\nwith peroxidase-conjugated anti-mouse IgG 1:2,000 for 1 hour.\nImmunoreactive bands were detected with Enhanced Chemilumines-\ncence Plus (Amersham Biosciences, Little Chalfont Buckinghamshire,\nUnited Kingdom).\nTo detect FGFR3 protein in cell lysate, cells were harvested by the\nsame lysis buffer mentioned above. The lysate (40 Ag) was electro-\nphoresed using 8% SDS-polyacrylamide gel, blotted, and blocked by\n5% skim milk. The membrane was probed with anti-FGFR3 antibody at\n1:500 dilutions and with peroxidase-conjugated anti-rabbit immuno-\nglobulin at 1:2,000.\nTo evaluate the phosphorylation status\nof FGFR3, serum-starved cells were treated by 20 Amol/L SU5402 for\n30 minutes followed by the treatment with rhFGF18 or vehicle. The\ncells were harvested by 1 mL modified radioimmunoprecipitation assay\nbuffer containing protease inhibitors and sodium orthovanadate,\nsonicated, and centrifuged at 15,000  g for 10 minutes. After\npreclearing by 100 AL protein A agarose (Upstate, Lake Placid, NY),\n1 mg of the protein was added with 4 Ag anti-FGFR3 antibody and\nrotated for 2 hours at 4jC, and the immunocomplex was mixed with\n100 AL protein A agarose and rotated for another 2 hours. The mixture\nwas washed twice by PBS, and the precipitate was boiled for 5 minutes\nwith 60 AL of 2 sample buffer, centrifuged, and the supernatant was\nused for Western blotting using anti-FGFR3 antibody (1:500) or anti-\nphosphotyrosine (1:500).\nTo evaluate the phosphorylation of MAPKs, cells (4  105) were\nseeded on 60-mm dishes and incubated overnight with medium\ncontaining 10% FBS followed by serum starvation for 24 hours. After\ntreatment with each reagent for 15 minutes in the serum-free medium,\ncells were lysed in a buffer (100 AL) containing sodium orthovanadate\n(5 mmol/L) and protease inhibitors, sonicated, and centrifuged.\nProteins (15 Ag) were used for Western blotting.\nCell cycle analysis. Cells (1  105) were incubated overnight in\n60-mm dishes\nfollowed by serum\nstarvation for 24 hours. After treatment with SU5402 (20 Amol/L)\nfor 30 minutes, FBS (1%) was added to the medium and cells were\nfurther incubated for 48 hours. Cells were then harvested, washed\nonce with PBS, and fixed in 70% ethanol overnight. After an\nincubation in PBS containing propidium iodide (10 Ag/mL) and\nRNaseA (10 Ag/mL) for 30 minutes at 37jC, the DNA content of\nthese cells was analyzed using FACScan and CELL Quest Software\nIn vivo growth assay. All animal studies were approved by the\nAnimal Research Committee, Graduate School of Medicine, Kyoto\nUniversity and done according to the Guideline for Animal Experi-\nments of Kyoto University. First,\nthe effect of PD166866 on\nphosphorylated ERK1/2 in vivo was investigated. SYO-1 (5  106)\ncells suspended in 100 AL of PBS were s.c. injected into the hind\nflank region of male BALB/c nu/nu athymic mice at 5 weeks of age\n(Japan SLC, Hamamatsu,\nJapan). PD166866 (0.1 or 0.5 mg)\nsuspended in 50 AL of DMSO was given i.p. when the tumor\nvolume was f1,500 mm3. Tumors were dissected 30 minutes after\nthe injection, and tissue blocks (5  5  5 mm) were prepared from\nthe central and peripheral portion of tumors. Whole cell lysates were\nprepared from each tissue block and used for Western blot. Growth\ninhibition experiments were done using SYO-1 and HT1080. Cells\n(5  106) were inoculated as described above. When the tumor\nvolume reached about 50 mm3 (usually 11-12 days after inocula-\ntion), PD166866 was given i.p., a treatment which was repeated\nthereafter once a day, 6 days a week, for 2 weeks. Tumor size was\nmeasured with vernier calipers, and the volume was calculated as k/\n6  length  width  height.\nImmunohistochemical\nstaining of proliferating cell nuclear antigen (PCNA) was done using\nEPOS anti-PCNA/horseradish peroxidase system (DakoCytomation).\nOn day 13 (the day of the last administration), tumor tissues were\ndissected, fixed in 10% formalin, and embedded in paraffin. Sections\n(5-Am-thick) were probed with anti-PCNA antibody for 1 hour and\ncounterstained with hematoxylin. To quantify the PCNA-positive area,",
         "Abstract Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which\nare unknown.We investigated the involvement of fibroblast growth factor (FGF) signals in synovial\nsarcoma and evaluated the therapeutic effect of inhibiting the FGF signal.\nExperimental Design:The expression of 22 FGF and 4 FGF receptor (FGFR) genes in18 prima-\nry tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR.\nEffects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein\nkinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory\neffects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo.\nResults: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed\nin neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma\ncell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal ^\nregulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the\nFGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle ar-\nrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the\nFGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not\np38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sar-\ncoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2.\nConclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibi-\ntory molecules will be of potential use for molecular target therapy in synovial sarcoma.\nSynovial sarcoma is the most frequent soft-tissue sarcoma\n(STS) among patients in the third to fourth decade of life (1)\nand accounts for about 7% to 10% of all human STSs (2). It\npredominantly affects the lower extremities but can occur in\nany part of the body. Surgical resection with an adequate\nfor\nsurgical margin is the definitive choice of\nprimary tumors and has been shown to control\nlocal\nrecurrence (3, 4). Disseminated distant metastasis is the major\ncause of poor outcome, and several reports describing the\nresults of current therapy showed a 5-year survival rate of\naround 50% to 60% (3, 4). The efficacy of adjuvant\nchemotherapy is still a matter of debate, and the development\nof a new therapeutic approach is required to improve the\nprognosis.\nthe reciprocal\nDespite little progress in clinical treatment during the last\n20 years, cytogenetic and molecular genetic analyses have greatly\nimproved the understanding of this type of tumor, especially\nwith the discovery of\ntranslocation\nt(18;X)(q11;p11) creating an SYT-SSX fusion gene as a synovial\nsarcoma – specific genetic alteration (5, 6). Thus far, three SSX\ngenes (SSX1, SSX2, and SSX4) have been identified as a partner\nof the SYT gene, and >95% of synovial sarcoma tumors carried\none of these fusion genes (7). Although the precise function and\nthe mechanism of oncogenesis are not yet clearly shown, the\nhigh sensitivity and specificity of the SYT-SSX fusion gene in\nsynovial sarcoma have proven to be useful for molecular\ndiagnosis (8). In particular types of tumors with a specific\nreciprocal translocation such as PML-RARa in acute promyelo-\ncytic leukemia (9) and BCR-ABL in chronic myelogenous\nleukemia (10), fusion gene products themselves serve as targets\nfor the therapy. Although immunotherapy using a peptide\nderived from SYT-SSX protein as a specific vaccine has been\ninvestigated (11), no therapeutic approach has been discovered\nfor directly targeting the fusion protein or its function.\nGene expression profiling of tumors has been shown a\npowerful tool with which to isolate a molecular target for\ntherapy (12). We have done a gene expression analysis of\nsynovial sarcoma using a genome-wide cDNA microarray and\nfound that synovial sarcoma shared its molecular signature with\nmalignant peripheral nerve sheath tumor, of which the\nprecursors were neural crest – derived cells, and also identified\na set of genes commonly up-regulated in synovial sarcoma\nincluding the fibroblast growth factor 18 (FGF18) gene (13). The\nFGF signaling pathway seems to play significant roles in tumor\ndevelopment and progression (14 – 16), and recently FGF18 was\nidentified as an autocrine growth factor involved in colon\ncancers (17). Thus far, 22 genes have been identified as members\nof the FGF family, and our cDNA microarray contained 10 FGF\ngenes (FGF2, FGF3, FGF4, FGF7, FGF9, FGF11, FGF12, FGF13,\nFGF18, and FGF19). We found that some FGF genes other\nthan FGF18 were also expressed in synovial sarcoma (data\nshown). Based on these results, we have done an\nnot\nintensive analysis of FGF and its receptor (FGFR) genes in\nsynovial sarcoma and also investigated whether inhibition of\nthe FGF signal is a new therapeutic modality for synovial\nsarcoma.\nMaterials and Methods\nTumor tissues were obtained at either\nbiopsy or resection surgery and kept at 80jC. Informed consent was\nobtained from each patient, and tumor samples were approved for\nanalysis by the Ethics Committee of the Faculty of Medicine, Kyoto\nUniversity. Five human synovial sarcoma cell lines (YaFuSS, HS-SY-II,\nSYO-1, Fuji, and 1273/99) were used in this study. YaFuSS and HS-SY-II\ncells have the SYT-SSX1 fusion gene and the others have the SYT-SSX2\nfusion gene (data not shown). YaFuSS was established in our laboratory\nfrom a monophasic synovial sarcoma in a 28-year-old male. HS-SY-II\nwas a gift from H. Sonobe (Kochi University, Japan; ref. 18), SYO-1\nfrom A. Kawai (Okayama University, Japan; ref. 19), Fuji from S.\nTanaka (Hokkaido University, Japan; ref. 20), and 1273/99 from\nO. Larsson (Karolinska Institute, Sweden). Among control cell lines,\nNMS-2 (malignant peripheral nerve sheath tumor; ref. 21) was\nprovided by A. Ogose (Niigata University,\nJapan), and Saos2\n(osteosarcoma), HT1080 (fibrosarcoma), COLO205 (colon cancer),\nand SW480 (colon cancer) cells were purchased from American Type\nCulture Collection (Manassas, VA). Cells were maintained in RPMI\n1640 (Sigma-Aldrich, St. Louis, MI) for Fuji, SW480, and COLO205\nwith 10% fetal bovine serum (FBS, HyClone, Road Logan, UT) or\nDMEM (Sigma, St. Louis, MO) for other cells with 10% FBS. OPTI-\nMEM I, which contains insulin and transferrin as protein supplements\n(Invitrogen, Carlsberg, CA), was used in serum-free cultures.\nThe FGFR-specific tyrosine kinase\ninhibitor SU5402 was purchased from Calbiochem (La Jolla, CA;\nref. 22). PD166866 was provided by Pfizer Global Research and\nDevelopment (Groton, CT; ref. 23). The MAPK/extracellular signal –\nregulated kinase (ERK) kinase (MEK) inhibitor U0126 was purchased\nfrom Promega (Madison, WI; ref. 24). Recombinant human FGF18\n(rhFGF18) was purchased from Wako Pure Chemical Industry (Osaka,\nJapan),\nrhFGF8 from PeproTech (London, United Kingdom),\nand rhFGF2 from Oncogene Research Products (San Diego, CA).\nAnti-ERK1 (M12320), anti-phosphorylated-p38 (P39520), anti-pan-\np38 (P19820), anti-phosphorylated-JNK1/SAPK1 (S37220), and anti-\npan-JNK1/SAPK1 (M54920) antibodies were purchased from\nBD Biosciences PharMingen (San Diego, CA); anti-phosphorylated-\nERK1/2 (sc-7383), anti-FGFR3 (sc-123), and peroxidase-conjugated\nanti-mouse IgG (sc-2005) from Santa Cruz Biotechnology (Santa\nCruz, CA); anti-phosphotyrosine (PY20) from Zymed Laboratories\n(South San Francisco, CA); anti-FGF18 (MAB667) from R&D Systems\n(Minneapolis, MN); peroxidase-conjugated anti-rabbitimmunoglobu-\nlin (P0399) from DakoCytomation (Glostrup, Denmark).\nTotal RNA was extracted using TRIzol\nreagent (Invitrogen) following the manufacturer’s instructions. After\ntreatment with DNase I (Nippon Gene, Osaka, Japan), 1 Ag of total\nRNA was reverse transcribed for single-stranded cDNAs using oligo(dT)\nprimer and Superscript II reverse transcriptase (Invitrogen). PCR was\ndone using 1 AL of RT product in a final volume of 25 AL containing\n20 pmol each of the sense and antisense primers, 2.5 mmol/L MgCl2,\n0.2 mmol/L of each deoxynucleotide triphosphate, and 1 unit of rTaq\npolymerase (TOYOBO, Osaka, Japan). All PCRs were done using\nGeneAmp 9700 (PE Applied Biosystems, Foster City, CA). Information\nof the primers are available upon request.\nQuantitative reverse transcription-PCR (RT-PCR) analyses were done\nin selected FGF and FGFR genes with ABI PRISM 7700 Sequence\nDetection System (PE Applied Biosystems). One microliter of RT\nproduct in a final volume of 25 AL containing 12.5 AL of 2 SYBR\nGreen Mastermix (PE Applied Biosystems). Information for primers are\navailable upon requests. The mean of triplicated data was used to\ncalculate the ratio of target gene/b-actin expression. Statistical analysis\nwas done by t test after logarithmic transformation.\nTo detect FGF18 protein in the supernatant,\ncells were cultured up to 80% confluency in 100-mm dish. After\nwashing the dish, cells were further incubated in 5 mL OPTI-MEM I for\n4 days. The supernatant was harvested, mixed with 8 mL ice-cold\nacetone, and kept at 80jC for 1 hour. The mixture was centrifuged at\n10,000  g for 15 minutes, and the precipitate was suspended by lysis\nbuffer (100 AL) containing aprotinin (1 Ag/mL), leupeptin (1 Ag/mL),\npepstatin A (1 Ag/mL), and phenylmethylsulfonyl fluoride (1 mmol/L)\nfollowed by sonication and centrifugation. Twenty microliters of the\nsupernatant were electrophoresed on a 10% SDS-polyacrylamide gel\nand transferred onto a polyvinylidene difluoride membrane (Millipore,\nBedford, MA). After blocking with 3% skim milk, membranes were\nprobed with anti-FGF18 antibody at 1:1,000 dilutions for 1 hour and\nwith peroxidase-conjugated anti-mouse IgG 1:2,000 for 1 hour.\nImmunoreactive bands were detected with Enhanced Chemilumines-\ncence Plus (Amersham Biosciences, Little Chalfont Buckinghamshire,\nUnited Kingdom).\nTo detect FGFR3 protein in cell lysate, cells were harvested by the\nsame lysis buffer mentioned above. The lysate (40 Ag) was electro-\nphoresed using 8% SDS-polyacrylamide gel, blotted, and blocked by\n5% skim milk. The membrane was probed with anti-FGFR3 antibody at\n1:500 dilutions and with peroxidase-conjugated anti-rabbit immuno-\nglobulin at 1:2,000.\nTo evaluate the phosphorylation status\nof FGFR3, serum-starved cells were treated by 20 Amol/L SU5402 for\n30 minutes followed by the treatment with rhFGF18 or vehicle. The\ncells were harvested by 1 mL modified radioimmunoprecipitation assay\nbuffer containing protease inhibitors and sodium orthovanadate,\nsonicated, and centrifuged at 15,000  g for 10 minutes. After\npreclearing by 100 AL protein A agarose (Upstate, Lake Placid, NY),\n1 mg of the protein was added with 4 Ag anti-FGFR3 antibody and\nrotated for 2 hours at 4jC, and the immunocomplex was mixed with\n100 AL protein A agarose and rotated for another 2 hours. The mixture\nwas washed twice by PBS, and the precipitate was boiled for 5 minutes\nwith 60 AL of 2 sample buffer, centrifuged, and the supernatant was\nused for Western blotting using anti-FGFR3 antibody (1:500) or anti-\nphosphotyrosine (1:500).\nTo evaluate the phosphorylation of MAPKs, cells (4  105) were\nseeded on 60-mm dishes and incubated overnight with medium\ncontaining 10% FBS followed by serum starvation for 24 hours. After\ntreatment with each reagent for 15 minutes in the serum-free medium,\ncells were lysed in a buffer (100 AL) containing sodium orthovanadate\n(5 mmol/L) and protease inhibitors, sonicated, and centrifuged.\nProteins (15 Ag) were used for Western blotting.\nCell cycle analysis. Cells (1  105) were incubated overnight in\n60-mm dishes\nfollowed by serum\nstarvation for 24 hours. After treatment with SU5402 (20 Amol/L)\nfor 30 minutes, FBS (1%) was added to the medium and cells were\nfurther incubated for 48 hours. Cells were then harvested, washed\nonce with PBS, and fixed in 70% ethanol overnight. After an\nincubation in PBS containing propidium iodide (10 Ag/mL) and\nRNaseA (10 Ag/mL) for 30 minutes at 37jC, the DNA content of\nthese cells was analyzed using FACScan and CELL Quest Software\nIn vivo growth assay. All animal studies were approved by the\nAnimal Research Committee, Graduate School of Medicine, Kyoto\nUniversity and done according to the Guideline for Animal Experi-\nments of Kyoto University. First,\nthe effect of PD166866 on\nphosphorylated ERK1/2 in vivo was investigated. SYO-1 (5  106)\ncells suspended in 100 AL of PBS were s.c. injected into the hind\nflank region of male BALB/c nu/nu athymic mice at 5 weeks of age\n(Japan SLC, Hamamatsu,\nJapan). PD166866 (0.1 or 0.5 mg)\nsuspended in 50 AL of DMSO was given i.p. when the tumor\nvolume was f1,500 mm3. Tumors were dissected 30 minutes after\nthe injection, and tissue blocks (5  5  5 mm) were prepared from\nthe central and peripheral portion of tumors. Whole cell lysates were\nprepared from each tissue block and used for Western blot. Growth\ninhibition experiments were done using SYO-1 and HT1080. Cells\n(5  106) were inoculated as described above. When the tumor\nvolume reached about 50 mm3 (usually 11-12 days after inocula-\ntion), PD166866 was given i.p., a treatment which was repeated\nthereafter once a day, 6 days a week, for 2 weeks. Tumor size was\nmeasured with vernier calipers, and the volume was calculated as k/\n6  length  width  height.\nImmunohistochemical\nstaining of proliferating cell nuclear antigen (PCNA) was done using\nEPOS anti-PCNA/horseradish peroxidase system (DakoCytomation).\nOn day 13 (the day of the last administration), tumor tissues were\ndissected, fixed in 10% formalin, and embedded in paraffin. Sections\n(5-Am-thick) were probed with anti-PCNA antibody for 1 hour and\ncounterstained with hematoxylin. To quantify the PCNA-positive area,\nResults\nSynovial sarcoma cell lines expressed multiple fibroblast growth\nfactor and fibroblast growth factor receptor genes. The mRNA\nexpression of 22 subtypes of FGF genes was analyzed by RT-\nPCR in five synovial sarcoma and five other cell lines (Fig. 1A).\nfive synovial sarcoma cell\nNone of the five synovial sarcoma cell lines expressed the\nFGF5, FGF6, FGF12, FGF14, FGF16, FGF17, FGF22, or FGF23\ngene, whereas the FGF2, FGF8, FGF9, FGF11, and FGF18 genes\nwere expressed in all\nlines.\nExpression of the remaining nine FGF genes varied among\ncell\nlines. The expression profile in NMS-2 (malignant\nperipheral nerve sheath tumor) was similar to that in synovial\nsarcoma, which may relate to the overall similarity of the gene\nexpression profile between synovial sarcoma and malignant\nperipheral nerve sheath tumor (13). HT1080 (fibrosarcoma)\nexpressed a similar set of FGF genes to malignant peripheral\nnerve sheath tumor but at much lower levels, and the\nexpression profile of Saos2 (osteosarcoma) was quite different\nbeing positive only for FGF1 and FGF13. Interestingly, synovial\nsarcoma shared with colon carcinoma cell lines the expression\nof some FGF genes such as FGF3 and FGF18 genes, which were\nnot expressed in mesenchymal cell lines. Secretion of these FGF\nproteins by synovial sarcoma cell lines was confirmed by the\nWestern blotting of culture supernatant. FGF18 proteins were\nclearly detected in culture supernatant of synovial sarcoma cell\nlines, although the level of protein was not completely agreed\nwith the level of RNA expression (Fig. 1C).\nIn contrast with the heterogeneous expression pattern of\nthe expression profile of receptors (FGFR) was\nligands,\nrelatively homogeneous among synovial sarcoma cell lines,\nbeing positive for all subtypes analyzed except FGFR2b, which\nis an epithelia-specific FGFR expressed in colon carcinomas\n(Fig. 1B). In addition to FGFR2b, NMS-2 was negative for\nFGFR3, and HT1080 was negative for FGFR2c. Expression of\nFGFR3 in synovial sarcoma cell lines was confirmed at the\nprotein level by the Western blotting using cell lysates of each\ncell line (Fig. 1D).\nExpression of fibroblast growth factor and fibroblast growth\nfactor receptor genes in tumor tissues.\nThe FGF2, FGF8,\nFGF9, FGF11, and FGF18 genes, which were commonly ex-\npressed in the five synovial sarcoma cell lines, were inves-\ntigated in tumor tissues of 18 synovial sarcomas and 11 STSs\nof other types (Fig. 2A). In accordance with the results in the\ncell lines, almost all 18 synovial sarcoma tumors expressed\nall of these FGF genes (Fig. 2A). Among five FGF genes, the\nFGF2, FGF9, and FGF11 genes were also expressed in some\ntumors other than synovial sarcoma. In contrast, expression of\nFGF8 and FGF18 genes was only weakly detected in other\ntumors. Expression profiles of these five FGF genes were further\nanalyzed by the quantitative RT-PCR (Fig. 2B). The level of\nexpression of FGF8 and FGF18 genes in synovial sarcoma\ntumors were significantly higher than in tumor tissues of other\ntype of tumors, whereas those of FGF2, FGF9, and FGF11 genes\nshowed no difference.\nAs for receptor genes, synovial sarcoma tumors expressed all\ntypes of FGFR genes analyzed including the FGFR2b gene\nGrowth stimulatory effects of\nfibroblast growth factors in\nsynovial sarcoma cell lines. Expression of FGF and FGFR genes\nand proteins suggested the involvement of FGFs as autocrine/\nparacrine growth factors in synovial sarcoma. Among FGF genes\nexpressed in synovial sarcoma both in vitro and in vivo, we\nfocused on the FGF2, FGF8, and FGF18 genes in subsequent\nexperiments. The FGF2 gene was selected as a gene commonly\nup-regulated in STS (Fig. 2A). FGF8 and FGF18 genes were\nselected as synovial sarcoma – specific up-regulated FGF genes\n(Fig. 2A).\nGrowth stimulatory effects of each rhFGF protein were\nserum-starved conditions. Recombinant\nanalyzed under\nhFGF8 showed a dose-dependent growth-promoting effect\nin all synovial sarcoma cell lines (Fig. 3A). On the treatment\nwith rhFGF18, only HS-SY-II\nshowed a dose-dependent\nincrease (Fig. 3B). A heterogeneous response to FGF was\nmore clearly observed in the case of rhFGF2, in which the\ngrowth of HS-SY-II\nincrease,\nwhereas a growth inhibitory effect was observed in the case\nof YaFuSS (Fig. 3C).\nshowed a dose-dependent\nActivation of mitogen-activated protein kinases by fibroblast\ngrowth factor signals. One of major signal pathways located\ndownstream of FGFR is the MAPKs, for which three subtypes are\nknown: ERK, p38 MAPKs, and c-Jun NH2-terminal kinases\n(JNK; ref. 25). To investigate the events downstream of activated\nFGFRs, the phosphorylation of these MAPKs was investigated. In\nall synovial sarcoma cell lines analyzed, ERK1/2 was phosphor-\nylated under serum-free conditions, suggesting the presence of\nendogenous stimulatory signals (Fig. 4A). The amount of\nphosphorylated ERK1/2 (pERK1/2) increased on treatment\nwith FGF8 in a dose-dependent manner in all four cell lines\nanalyzed, with the response of HS-SY-II being the highest\nlevels. A dose-dependent\nFibroblast growth factor receptor – specific inhibitors reduce the\nlines in accordance with\ngrowth of synovial sarcoma cell\ninactivation of fibroblast growth factor receptor. MAPKs in\nsynovial sarcoma are activated by intrinsic pathways, and\nexogenous FGFs can stimulate the growth of synovial sarcoma,\nsuggesting that inhibition of the intrinsic FGF signal pathway\nmay inhibit the growth of synovial sarcoma. SU5402 is known\nto inhibit FGFR autophosphorylation at an IC50 of 10 to\n20 Amol/L (22). To confirm the effect of SU5402 on synovial\nsarcoma cell lines, the phosphorylation status of FGFR3 was\nanalyzed by the immunoprecipitation and Western blotting in\nthree synovial sarcoma cell lines (YaFuSS, SYO-1, and 1273/\n99; Fig. 5A). Phosphorylated FGFR3 was detected in these cell\nlines with serum-starved condition, and the treatment with\nrhFGF18 protein increased the amount of phosphorylated\nFGFR3 (Fig. 5A), which was compatible with the autocrine/\nparacrine model of FGFs in synovial sarcoma cells. Treatment\nwith SU5402 effectively decreased the phosphorylation of\nFGFR3 into almost undetectable level (Fig. 5A), indicating that\nSU5402 can be used as an inhibitor of FGFR in synovial\nsarcoma cells. By the treatment with SU5402, the growth of all\nfive synovial sarcoma cells was inhibited in a dose-dependent\nmanner (Fig. 5B), and the IC50 value ranged from 8.5 to 17.2\nAmol/L (Table 1). The growth inhibitory effect of SU5402 for\ncell lines other than synovial sarcoma was not remarkable. The\nsame experiments were done under low-serum conditions (1%\nFBS, Fig. 5C), resulting in similar growth inhibition in all\nsynovial sarcoma cell lines. The IC50 was almost identical in\nthe three cell lines (YaFuSS, HS-SY-II, and SYO-1), whereas the\nlines (Fuji and 1273/99) required a higher\nother two cell\nconcentration of SU5402 (Table 1). Growth inhibitory effects\nof another FGFR-specific inhibitor, PD166866 (23), was also\ninvestigated under the low-serum conditions. All five synovial\nsarcoma cell lines had an IC50 value (0.08-4.7 Amol/L) much\nlower than those of other cell lines (>10 Amol/L, Table 1).\nSimilar to the data obtained using SU5402, IC50 values in\nYaFuSS, HS-SY-II, and SYO-1 were much lower than those in\nFuji and 1273/99.\n11.5 F 1.8\n15.1 F3.4\n8.5 F 2.5\n17.2 F 1.8\n12.8 F 6.1\n46.4 F 6.9\n65.4 F 2.6\n>80\n>80\n54.4 F 21.5\n11.4 F 3.6\n4.9 F 0.5\n8.6 F 1.6\n25.3 F 3.6\n29.9 F 16.6\n60.2 F 20.6\n63.7 F 27.8\n55.8 F 19.1\n>80\n78.1 F20.3\n0.08 F 0.04\n0.77 F 0.52\n0.18 F 0.11\n4.0 F 1.6\n2.7 F 0.4\n>10\n>10\n>10\n>10\n>10\nGrowth inhibition of fibroblast growth factor receptor inhibitor\nassociated with the reduction of phosphorylation of extracellular\nsignal – regulated kinase 1/2 but not of p38.\nTo investigate\nwhich signals may contribute to the growth of synovial\nsarcoma, the phosphorylation of ERK1/2 and p38 before and\nafter the treatment with SU5402 was investigated. In three\nsynovial sarcoma cell lines (YaFuSS, HS-SY-II, and SYO-1),\nwhich were relatively sensitive to the growth inhibitory effect of\nSU5402, phosphorylation of ERK1/2 under low-serum con-\nditions was completely inhibited by SU5402, whereas the\namount of phosphorylated p38 showed no change (Fig. 6A). A\nsmall amount of pERK1/2 was still observed after SU5402\ntreatment in 1273/99, whereas no significant decrease was\nobserved in Fuji, both of which showed no significant change in\nthe pp38 (Fig. 6A). In cell lines other than synovial sarcoma,\nSU5402 showed no significant changes in the amount of\npERK1/2 or pp38 (data not shown). These results suggested that\nthe signal to induce the phosphorylation of ERK1/2 in synovial\nsarcoma was sent mainly through the FGF and FGFR pathway\nand that although the FGF signal induced the phosphorylation\nof both ERK1/2 and p38, the growth stimulatory effects were\ntransmitted through ERK1/2. To further confirm that the growth\ninhibition of SU5402 was due to the reduction in pERK1/2, the\nfunction of MAPK/ERK kinase 1/2 in synovial sarcoma, which is\na kinase of ERK1/2, was inhibited by U0216. Phosphorylation\nof ERK1/2 was effectively inhibited in all five cell lines, in which\n1273/99 still had a small amount of pERK1/2 at the lower\nconcentration (Fig. 6B). BrdUrd uptake was inhibited in a dose-\ndependent manner in all five cell lines, in which 1273/99 was\nrelatively resistant at the lower concentration, consistent with\nthe status of pERK1/2 (Fig. 6C). These results further indicated\nthat the growth inhibition of SU5402 in synovial sarcoma was\nachieved through the inhibition of signals through ERK1/2.\nGrowth inhibition by fibroblast growth factor receptor inhibitor\nin synovial sarcoma is associated with cell cycle arrest. The\nfraction of each cell cycle phase was analyzed before and after\nthe treatment with SU5402 in YaFuSS cells. SU5402 signifi-\ncantly increased the G1 and subG1 fractions, and decreased the\nS fraction (Fig. 6D). These data suggested that the growth\ninhibition of synovial sarcoma by SU5402 was due to its\ninduction of G1 arrest as previously reported for other tyrosine\nkinase inhibitors (26).\nFibroblast growth factor receptor inhibitors reduced the\nFor in vivo study,\ngrowth of synovial sarcoma cells in vivo.\nwe used SYO-1 because of\nits consistency in developing\ntumor, and PD166866 was used because of its lower IC50\nvalue. To confirm its efficacy in vivo, tumor tissue was taken\n30 minutes after the injection of PD166866. We found that\n0.1 mg of PD166866 was enough to reduce the phosphory-\nlated ERK1/2 in the tumor (Fig. 7A). Based on this result,\nthe growth inhibitory effect of PD166866 (0.1 and 0.5 mg/\nbody) was analyzed in vivo. Although PD166866 was unable\nto stop the growth of the tumor completely, a significant\ninhibition of growth was observed during the period of\nadministration (P < 0.01), and the effect was retained even at\nday 21 at the higher dose (P = 0.02, Fig. 7B). No definite\nreduction of body weight or pathologic changes in vital\norgans such as liver and kidney was observed (data not\nshown). No growth inhibitory effect was observed when\nHT1080 was used instead of SYO-1 even at the higher dose\n(data not shown).\nHistologic examination of tumor tissues showed no signifi-\ncant morphologic changes or focal necrosis. The number of\nPCNA-positive cells, however, was clearly smaller in PD166866-\ntreated tumors (19.3 F 6.6%, Fig. 7D) than control tumors\n(53.0 F 2.0%, Fig. 7C), indicating that PD166866 reduced the\nnumber of cells in S phase. No such difference was observed\nwhen HT1080 was used instead of SYO-1 (data not shown).\nDiscussion\nFGFs may activate genetic programs which promote cell\ngrowth by at least one of three general mechanisms: first, as\nmitogens\nsecond, by\npromoting angiogenesis to supply a growing tumor; and\nthird, by inhibiting apoptosis and allowing tumor cells to\ncontinue to grow beyond normal constraints. We found that\nsynovial sarcoma cells expressed a number of FGF genes, of\nwhich FGF2 , FGF8 , FGF9 , FGF11 , and FGF18 were\ncommonly expressed. The involvement of FGF2 in malignant\ntumors as an autocrine growth factor and/or a paracrine\nangiogenic factor was first shown in brain tumors (27), and\nsubsequently in non – small cell carcinoma (14). FGF8 was\noriginally identified as an androgen-induced growth factor\nfound in conditioned medium of androgen-dependent mouse\nmammary carcinoma SC-3 cells and reported to have a\ngrowth stimulatory effect (28). The human FGF8 gene was\nexpressed in breast and prostate cancers (15, 16, 29) and\nassociated with anchorage-independent proliferation and\ninvasion (30, 31). FGF9 was originally purified from the\nconditioned medium of\nline NMC-G1 and\ndesignated as glia-activating factor due to its mitogenic\nactivity toward glial cells (32). FGF11 was one of four FGF\nhomology factors, which shared structural homology with\nfailed to activate FGFRs (33). Recently,\nclassic FGFs but\nFGF11 was identified as one of the gene products enriched in\nneuronal precursors during the development of the nervous\nsystem (34). FGF18 is cloned by its homology to FGF8 and\nFGF9 (35, 36). FGF18, in association with FGF8, is expressed\nat various times and places during embryogenesis, especially\nneurogenesis (33, 37). Mitotic activity was shown in colon\ncarcinomas,\nin which the expression of FGF18 was up-\nregulated by WNT signals (17). Overall, we found that\nsynovial sarcoma expressed several FGF genes, particularly\nthose expressed in neural tissues, which may further suggest\nthe neural origin of precursor cells of synovial sarcoma as we\npreviously proposed (13). Mitotic activity of exogeneous FGFs\nwas not remarkable in some cell lines (Fig. 3). The serum-\nfree medium used in this study contained insulin, which is\nknown to potentiate the effect of FGF, and the further\naddition of insulin showed no increase of the BrdUrd uptake\n(data not shown). It is possible that the extent of autocrine\nstimulation may already be saturating so that further FGF\nstimulation does not lead to additional growth.\nThe expression of FGFR1, FGFR2c, FGFR3, and FGFR4, to\nwhich FGF2 (1c, 3c and 4), FGF8 (2c, 3c, 4), and FGF18 (2c,\n3c, 4) preferentially bind (38, 39), was confirmed in both tumor\ntissues and cell lines. The expression of FGFR2b gene, which was\ndetected in tumor tissues but not in cell lines, is intriguing.\nFGFR2b is a receptor for keratinocyte growth factor (FGF7), and\nexpressed in epithelial cells (40). As expected, all biphasic\nsynovial sarcomas expressed the FGFR2b gene, whereas the\nexpression was low in most of the monophasic synovial\nsarcomas and other STSs. It is, however, intriguing that some\nmonophasic synovial sarcomas with no apparent epithelial\nstructures expressed the FGFR2b gene at a level comparable with\nthat in biphasic synovial sarcoma (Fig. 2D). All five synovial\nsarcoma cell lines used in this study including SYO-1, which was\nestablished from a biphasic synovial sarcoma, were negative for\nthe FGFR2b gene. These results suggested that synovial sarcoma\ntumor cells may have an intrinsic mechanism to express FGFR2b\nin vivo, which was somehow inhibited or disappeared in vitro.\nAlternatively, synovial sarcoma cells positive for FGFR2b may\nhave growth disadvantages, and therefore only negative cells\nwere established as cell\nlines. Recently, single nucleotide\npolymorphism in FGFR4 is reported to associate with poor\nprognosis in high-grade soft tissue sarcomas including synovial\nsarcomas (41) and both FGF8 and FGF18 bind to FGFR4.\nAlthough the status of polymorphism was not investigated in our\nsamples, FGF signals through FGFR4 may contribute to the\naggressiveness of synovial sarcoma.\nThe binding of FGF to FGFR leads to receptor dimerization\nand tyrosine autophosphorylation, followed by downstream\nactivation of PLCg, Crk, and SNT-1/FRS2 signaling pathways\namong which the SNT-1/FRS2 is the major pathway for\nmitogenic signals (25). Phosphorylated SNT-1/FRS2 directly\nbinds to the GRB2-SOS complex, and then membrane-\nassociated RAS recruits RAF-1, which in turn activates MAPKs\n(42). MAPKs are composed of three well-characterized sub-\nfamilies, ERK, p38 MAPK, and JNK (25), and we found that the\ngrowth-promoting signal of FGF was transmitted mainly\nthrough ERK1/2. Although ERK1/2 are activated by signals\nthrough other receptor tyrosine kinases (43), phosphorylation\nof ERK1/2 was completely inhibited by FGFR inhibitors, which\nindicated that the major signal to activate ERK1/2 in synovial\nsarcoma is FGF signaling, suggesting the important role of FGF\nin synovial sarcoma.\nWe have no clear explanation for the heterogeneous response\nin synovial sarcoma cell lines to FGF2, which stimulated the\ngrowth of HS-SY-II, but inhibited the growth of YaFuSS. In\ncontrast with many studies describing the role of FGF2 as a\nmitogen, several reports showed that FGF2 acted as a growth\ninhibitor of tumors of Ewing’s sarcoma family, which are\nsupposed to be of neuroectodermal origin (44, 45). Multiple\nFGF signals are involved in the biological process in synovial\nsarcoma, and the response may depend on the cell specificity.\nNevertheless, the disruption of FGF signals was found to cause\ngrowth inhibition in all synovial sarcoma cell lines, suggesting\nthat an inhibitory molecule for FGFR will be a promising tool\nfor molecular target therapy in synovial sarcoma.\nMolecular therapy targeted to FGFR has been investigated\nwith hematologic malignancies carrying a translocation involv-\ning the FGFR gene such as chronic myeloid leukemia with the\nBCR-FGFR1 gene (46) and multiple myeloma with the MMSET-\nFGFR3 fusion gene (47). In both instances, SU5402 was found\nto effectively inhibit the growth of cell lines with an IC50\nequivalent to that in synovial sarcoma. For experiments in vivo,\nwe used PD166866 instead of SU5402 because of its lower IC50\nvalue and found that i.p. given PD166866 efficiently inhibited\ntumor causing an\nthe phosphorylation of ERK1/2 in s.c.\ninhibition of growth without significant side effects. As in the\ncase of other tyrosine kinase inhibitors, PD166866 did not\nstrongly induce apoptosis or necrosis, and therefore no\ncomplete tumor remission was observed. However, cell cycle\narrest caused by the treatment with PD166866 may enhance\nthe cytotoxic effects of anticancer drugs as previously shown in\nsome instances (48, 49), and therefore further in vivo studies\nusing the treatment combined with suitable anticancer drugs\nmay provide a promising approach for clinical application.",
         null
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "Shuffled iterative decoding",
         "Shuffled versions of iterative decoding of low-density parity-check codes and turbo codes are presented. The proposed schemes have about the same computational complexity as the standard versions, and converge faster. Simulations show that the new schedules offer better performance/complexity tradeoffs, especially when the maximum number of iterations has to remain small.",
         "Low-Density Parity-Check Codes Turbo decoder architectures for low-density parity-check codes Shuffled belief propagation decoding A parallel decoder for low latency decoding of turbo product codes Parallel versus sequential updating for belief propagation decoding High throughput low-density parity-check decoder architectures Good error-correcting codes based on very sparse matrices Near optimum error correcting coding and decoding: turbo-codes Multiple turbo codes for deep-space communications Optimal decoding of linear codes for minimizing symbol error rate (Corresp.) Generic Architecture for LDPC Codes Decoding Optimal decoding of linear codes for minimizing symbol error rate",
         "http://www.merl.com/publications/docs/TR2005-063.pdf",
         "Replica shufﬂed versions of interactive decoders of turbo codes, low-density parity-check codes\nand turbo product codes are presented. The proposed schemes converge faster than standard and\npreviously proposed shufﬂed approaches. Simulations show that the new schedules offer good\nperformance versus complexity/latency trade-offs.",
         "This work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in whole or in part\nwithout payment of fee is granted for nonproﬁt educational and research purposes provided that all such whole or partial copies include\nthe following: a notice that such copying is by permission of Mitsubishi Electric Research Laboratories, Inc.; an acknowledgment of\nthe authors and individual contributions to the work; and all applicable portions of the copyright notice. Copying, reproduction, or\nrepublishing for any other purpose shall require a license with payment of fee to Mitsubishi Electric Research Laboratories, Inc. All\nrights reserved.\nMITSUBISHI ELECTRIC RESEARCH LABORATORIES\nhttp://www.merl.com\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang, Marc Fossorier, and Jonathan S. Yedidia\nAbstract\nReplica shufﬂed versions of iterative decoders of turbo codes, low-density parity-check\ncodes and turbo product codes are presented. The proposed schemes converge faster\nthan standard and previously proposed “shufﬂed” approaches. Simulations show that\nthe new schedules offer good performance versus complexity/latency trade-offs.\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in\nwhole or in part without payment of fee is granted for nonproﬁt educational and research purposes provided that all such\nwhole or partial copies include the following: a notice that such copying is by permission of Mitsubishi Electric Research\nLaboratories, Inc.; an acknowledgment of the authors and individual contributions to the work; and all applicable portions\nof the copyright notice. Copying, reproduction, or republishing for any other purpose shall require a license with payment\nof fee to Mitsubishi Electric Research Laboratories, Inc. All rights reserved.\nTo appear in the Proceedings of the 2005 International Symposium on Information Theory.\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang and Marc Fossorier\nDepartment of Electrical Engineering\nUniversity of Hawaii at Manoa\nHonolulu, HI 96822\nEmail: juntan, yige, marc@spectra.eng.hawaii.edu\nJonathan S. Yedidia\nMitsubishi Electric Research Laboratories\n201 Broadway, Cambridge, MA 02139\nEmail: yedidia@merl.com\nI. INTRODUCTION\nIterative decoding based on belief propagation (BP) [1] has\nreceived signiﬁcant attention recently, mostly due to its near-\nShannon-limit error performance for the decoding of low-\ndensity parity-check (LDPC) codes [2] and turbo codes [3].\nLike maximum a posterior probability (MAP) decoding [4],\nit is a symbol-by-symbol soft-in/soft-out decoding algorithm.\nIt processes the received symbols recursively to improve the\nreliability of each symbol based on constraints that specify\nthe code. In the ﬁrst iteration, the decoder only uses the\nchannel output, and generates soft output for each symbol.\nSubsequently, the output reliability measures of the decoded\nsymbols at the end of each decoding iteration are used as\ninputs for the next iteration. The decoding iteration process\ncontinues until a certain stopping condition is satisﬁed. Then\nhard decisions are made based on the output reliability mea-\nsures of the decoded symbols at the last decoding iteration.\nThe standard BP decoder of LDPC codes often needs several\ntens or hundreds of iterations for the decoding process to\nconverge, which is not desirable because of high decoding\ndelay. Furthermore, LDPC codes of interest can have large\nthe\ncodeword length and it can be difﬁcult\ndecoding in hardware in a fully parallel way. Because of\nthe serial property of the BCJR algorithm, decoders of turbo\ncodes cause high delay as well. In [5], [6] and [7], “shufﬂed”\nmethods were presented to reduce the required number of\niterations for decoding LDPC and turbo codes, respectively.\nA speedup factor of 2 for LDPC codes and the saving of\none iteration for turbo codes were reported. The aim of this\npaper is to introduce a “replica shufﬂed” scheme which further\naccelerate the decoding process for turbo codes, LDPC codes,\nturbo product codes and other iterative decodable codes.\nto implement\nII. ITERATIVE DECODING OF TURBO CODE\nA turbo code [3] encoder is constructed using a concatena-\ntion of two (or more) convolutional encoders, and its decoder\nconsists of two (or more) soft-in/soft-out convolutional de-\ncoders which feed reliability information to each other. For\nis the output code block at\nsimplicity, we consider a turbo code that consists of two rate-\n1/n systematic convolutional codes with encoders in feedback\nform. Let u = (u1, u2, . . . , uK) be an information block of\nlength K and c = (c1, c2, . . . , cK) be the corresponding\ncoded sequence, where ck = (ck,1, ck,2, . . . , ck,n), for k =\n1, 2, . . . , K,\ntime k. Suppose\nBPSK transmission over an AWGN channel, with uk and\nck,j all taking values in {+1, -1} for k = 1, 2, . . . , K and\nj = 1, 2, . . . , n. Let y = (y1, y2, . . . , yK) be the received\nsequence, where yk = (yk,1, yk,2, . . . , yk,n) is the received\nblock at time k. Let ˆu = {ˆu1, ˆu2, . . . , ˆuK} denote the estimate\nof u. Let sk denote the encoder state at time k. Following\n1 ), γk(s, s) = p(sk =\n[4], deﬁne: αk(s) = p(sk = s, yk\ns, yk|sk−1 = s), βk(s) = p(yK\na =\n(ya, ya+1, . . . , yb), and let α(m)\n(s) be\nk\nthe corresponding values computed in component decoder m,\nwith m = 1, 2. Let L(i)\nem(ˆuk) denote the extrinsic value of the\nestimated information bit ˆuk delivered by component decoder\nm at the ith iteration [8].\nk+1|sk = s), where yb\n(s), γ(m)\nA. Standard serial and parallel turbo decoding\nThe decoding approach proposed in [3] operates in serial\nmode, i.e., the component decoders take turns generating the\nextrinsic values of the estimated information symbols, and\neach component decoder uses the extrinsic messages delivered\nby the last component decoder as the a priori values of the\ninformation symbols. The disadvantage of this scheme is high\ndecoding delay. In the parallel turbo decoding algorithm [9],\nall component decoders operate in parallel at any given time.\nAfter each iteration, each component decoder delivers extrinsic\nmessages to other decoder(s) which use these messages as a\npriori values at the next iteration.\nB. Plain shufﬂed turbo decoding\nAlthough the parallel turbo decoding reduces the decoding\ndelay of serial decoding by half, the extrinsic messages are not\ntaken advantage of as soon as they become available, because\nthe extrinsic messages are delivered to component decoders\nonly after each iteration is completed. The aim of the shufﬂed\nturbo decoding is to use the more reliable extrinsic messages\nat each time. Let ˜u = (˜u1, ˜u2, . . . , ˜uK) be the sequence\npermuted by the interleaver corresponding to the original\ninformation sequence u = (u1, u2, . . . , uK), according to the\nmapping ˜uk = uπ(k), for k = 1, 2, . . . , K. We assume that k =\nπ(k), ∀k. There is a unique corresponding reverse mapping\nuk = ˜uπ−(k), for k = 1, 2, . . . , K and k = π−(k), ∀k. In\nshufﬂed turbo decoding, the two component decoders operate\nsimultaneously as in the parallel turbo decoding scheme, but\nthe messages are updated during each iteration based on π(k)\nand π−(k) [7]. Correspondingly it provides a faster decoding\nconvergence.\nC. Replica shufﬂed turbo decoding\n←−\nD 1 and\n−→\nD 1 and\n−→\nD 1, which are denoted by\nIn the plain shufﬂed turbo decoding summarized in Section\nthe component decoders process the\nII-B, we assume all\nbackward recursion followed by the forward recursion. Let\n−→\nus refer to the two component decoders as\nD 2.\nNaturally another possible scheme is to operate in the reverse\norder, i.e, all the component decoders process the forward\nrecursion followed by the backward recursion and we refer\n←−\nto them as\nD 2. In terms of error performance, there\nis no difference between these two approaches. However, the\nreliabilities of the extrinsic messages concerning a certain\ninformation bit delivered by these two shufﬂed turbo decoders\nare not the same. In general, the more independent information\nis used, the more reliable the delivered messages become.\nTherefore for the extrinsic messages delivered by component\n(i)\ndecoder\ne1 (ˆuk), the larger k\nis, the more reliable this message is. Following a similar\n←−\nanalysis, for the extrinsic message\nD 1,\nthe smaller k is, the more reliable this message is. It is natural\nto expect a faster decoding convergence if these two shufﬂed\nturbo decoders operate cooperatively instead of independently.\nBecause in this approach two sets of shufﬂed component\ndecoders are used to decode the same sequence of information\nbits, we refer to it as replica shufﬂed turbo decoding. In replica\nshufﬂed turbo decoding, two plain shufﬂed turbo decoders\n−→\n(processing recursions in opposite directions)\nD 2 and\n←−\n←−\nD 1 ,\nD 2 operate simultaneously and exchange more reliable\nextrinsic messages. We assume that the component decoders\n−→\n\n\ndeliver extrinsic messages synchronously, i.e.,\nk =\nk =\nT\n←−\n←−\n−→\n←−\n←−\n\n\n\n\n\nk)and\nk (\nT\nk, where the\nk) denote the times\nT\nT\nT\nk =\nT\n←−\n−→\n←−\n−→\nat which\nD 2) deliver the extrinsic values\nD 2 (\nD 1) and\nD 1 (\nof the k-th ((K + 1 − k)-th) estimated symbol of the original\ninformation sequence u and of the interleaved sequence ˜u,\nrespectively.\n(i)\ne1 (ˆuk) delivered by\n−→\nD 1 ,\n−→\nD 1. After time\nLet us ﬁrst consider the forward recursion stage at the i-\n−→\n\nk−1,\nT\nth iteration of component decoder\nthe values of −→α (1)\nk (s) should be updated and the values\nof −→γ (1)\nk (s) are needed. There are two possible cases. The\nﬁrst case is k > π−(k), which means the extrinsic value\n−→\n(i)\ne2 (ˆuk) of the information bit ˆuk has already been delivered\nL\n−→\nD 2. As in plain shufﬂed turbo decoding, this newly\nby decoder\n−→\n(i)\ne2 (ˆuk) is used to compute the values −→γ (1)\nL\navailable\nk (s),\n−→\n(i)\n−→α (1)\ne1 (ˆuk). The second case is k < π−(k), which\nL\nk (s), and\n(i)\ne2 (ˆuk) of the information bit ˆuk\nmeans the extrinsic value\n−→\nD 2. Then in plain shufﬂed\nhas not been delivered yet by\nturbo decoding, the values α(1)\ne1 (ˆuk) are updated\nk (s) and L(i)\n−→\nL\nk (s), and\nbased on the extrinsic messages delivered at last iteration.\nIn replica shufﬂed turbo decoding, however, there are two\nfurther subcases. The ﬁrst subcase is K + 1 − k < π−(k),\n(i)\ne2 (ˆuk) of the information\nwhich means the extrinsic value\n←−\nbit ˆuk has already been delivered by decoder\nD 2. Then this\n(i)\n(i−1)\ne2 (ˆuk), instead of\n(ˆuk) is used to\nnewly available\ne2\n−→\n(i)\ncompute the values −→γ (1)\nk (s), −→α (1)\ne1 (ˆuk). The\nL\nsecond subcase is K + 1 − k < π−(k), which means both\n(i)\nextrinsic messages of the information bit ˆuk, i.e,\ne2 (ˆuk)\n−→\n(i)\ne2 (ˆuk) are not available yet. In this subcase, the values\nL\nand\n(i)\nof −→α (1)\ne1 (ˆuk) are updated based on the extrinsic\nk (s) and\nmessages delivered at (i − 1)-th iteration. The recursions of\n←−\n←−\ncomponent decoders\nD 2 are realized based on\nD 1 and\nthe same principle. After Imax iterations, the shufﬂed turbo\ndecoding algorithm outputs ˆu = (ˆu1, ˆu2, . . . , ˆuK) as the de-\n←−\n(i)\ncoded codeword, where ˆuk =sgn[(\ne1 (ˆuk))/2 +\nL\n−→\n(i)\ne2 (ˆuk))/2 + 4\nyk,1], which is different from\nL\n(\nN0\nthat in the standard turbo decoding [3] and plain shufﬂed\nturbo decoding. It is straightforward to generalize the replica\nshufﬂed turbo decoding to multiple turbo codes which consist\nof more than two component codes. Also group of bits can\nbe updated periodically only to reduce information exchanges\nbetween replicas. Based on the above descriptions with two\nreplicas, the total computational complexity of the replica\nshufﬂed turbo decoding for multiple turbo codes at each\ndecoding iteration is about twice that of the parallel turbo\ndecoding.\n(i)\ne2 (ˆuk) +\n(i)\ne1 (ˆuk) +\n−→\nL",
         "D. Simulation results\nIII. ITERATIVE DECODING OF LDPC CODES\nLet H = [Hmn] be the parity check matrix which deﬁnes\nan LDPC code. We denote the set of bits that participate in\ncheck m by N (m) = {n : Hmn = 1} and the set of checks\nin which bit n participates as M(n) = {m : Hmn = 1}.\nAssume a codeword w = (w1, w2, . . . , wN ) is transmitted\nover an AWGN channel with zero mean and variance N0/2\nusing BPSK signaling and let y = (y1, y2, . . . , yN ) be the\ncorresponding received sequence.\nA. Standard BP for iterative decoding of LDPC codes\nBased on [1], let Fn be the log-likelihood ratio (LLR) of\nbit n and initially set Fn = 4\nmn be the\nN0\nLLR of bit n which is sent from check node m to bit node n\nand sent from the bit node n to check node m, respectively.\nLet z(i)\nn denote the a posteriori LLR of bit n. The standard\nBP algorithm [1] is carried out as follows:\nyn. Let ε(i)\nInitialization Set i = 1, maximum number of iterations to\nIM ax. For each m,n, set z(0)\nmn = Fn.\n(i) Horizontal Step, for 1 ≤ n ≤ N and each m ∈\nM(n), process:\nτ (i)\nmn =\nn∈N (m)\\n\n1 + τ (i)\nmn\n1 − τ (i)\nmn\n(ii) Vertical Step, for 1 ≤ n ≤ N and each m ∈\nε(i)\nmn = log\n(2)\nz(i)\nmn = Fn +\nz(i)\nn = Fn +\n(i)\nStep 2 Hard decision and stopping criterion test:\nCreate ˆw(i) = [ ˆw(i)\nn ] such that ˆw(i)\nz(i)\nn > 0, and ˆw(i)\nn = 1 if z(i)\nn < 0.\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 3. Otherwise set\ni := i + 1 and go to Step 1.\nn = 0 if\n(ii)\nStep 3 Output ˆw(i) as the decoded codeword.\nB. Plain shufﬂed BP for iterative decoding of LDPC codes\nAt the i-th iteration of the standard BP algorithm, ﬁrst all\nvalues of the check-to-bit messages are updated by using the\nvalues of the bit-to-check messages obtained at the (i − 1)-\nth iteration, i.e., each ε(i)\n:\nn ∈ N (m)\\n}. Then, all values of the bit-to-check messages\nare updated by using the values of the check-to-bit messages\nnewly obtained at the i-th iteration, i.e., each z(i)\nmn is updated\nfrom {ε(i)\nmn is updated by using {z(i−1)\nmn\nmn : m ∈ M(n)\\m}.\nWe observe that certain values z(i)\nin (1) to compute the remaining values ε(i)\nmn could already be\ncomputed in (3) based on a partial computation of the values\nε(i)\nmn obtained from (2), and then could be used instead of\nz(i−1)\nmn. Hence Step\nmn\n1 of the shufﬂed BP algorithm is performed as: for 1 ≤ n ≤ N\nand each m ∈ M(n), process the horizontal step and vertical\nstep jointly, with (1) modiﬁed as:\n\nτ (i)\nmn =\nn∈N (m)\\n\nn>n\nSince this algorithm becomes totally serial, bits can be grouped\nto maintain a sufﬁcient level of parallelism with the same error\nperformance [7].\nC. Replica shufﬂed BP decoding for LDPC codes\nPlain shufﬂed BP decoding is a bit-based sequential ap-\nproach and the method described in Section III-B is based\non a natural increasing order, i.e, belief messages concerning\nbit nodes are updated according to order i = 1, 2, . . . , N .\nThe larger the value of i, the more independent informations\nare used to update the beliefs of bit note i and the more\nreliable these belief messages are. Therefore the reliability of\nbit nodes increases and the error rate decreases as i increases.\nFollowing a similar analysis, in plain shufﬂed BP decoding\nbased on a natural decreasing order, after each iteration, the\nreliability of bit nodes decreases as i increases. As in replica\nshufﬂed turbo decoding, in replica shufﬂed BP decoding, two\nreplica shufﬂed subdecoders based on different updating orders\noperate simultaneously and cooperatively. After each iteration,\neach subdecoder receives more reliable messages from and\nsends more reliable messages to another subdecoder. Based on\nthese more reliable messages, both replica subdecoders begin\n←−\nthe next iteration of decoding. Let\nD denote the replica\nsubdecoder with natural increasing and decreasing updating\norder, respectively. Let −→ε (i)\nmn be the variables\n←−\nassociated with\nD\nare deﬁned in a similar way. The replica shufﬂed BP decoding\nwith two replica subdecoders is carried out as followings:\n−→\nD at iteration i. Variables associated with\nmn and −→z i\n−→\nD and\nInitialization Set i = 1, maximum number of iteration to\nIM ax. For each m,n, set −→z (0)\nmn = ←−z (0)\nmn = Fn.\nStep 1\nEach replica subdecoder process as the following\ntwo steps simultaneously. For 0 ≤ n ≤ N − 1 and\neach m ∈ M(n), process\nHorizontal Step\nn∈N (m)\\n\nn>n\n1 + −→τ (i)\nmn\n1 − −→τ (i)\nmn\nn∈N (m)\\n\nn<n\n1 + ←−τ (i)\nmn\n1 − ←−τ (i)\nmn\n(ii)\nVertical Step\n−→z (i)\nmn = Fn +\nmn = Fn +\n←−ε (i)\nmn\nStep 2 Exchange of\nthe more reliable messages. Set\nmn = −→z (i)\nmn for 0 ≤ n < N/2 and ←−z (i)\nmn = ←−z (i)\n−→z (i)\nfor N/2 ≤ n ≤ N − 1.\nStep 3\nHard decision and stopping criterion test:\nA. Conventional decoding methods of TPC\nn ] such that for 0 ≤ n <\nn = 1 if Fn +\nmn <\nn = 0 otherwise; for N/2 < n ≤\nn = 1 if Fn +\nmn <\nn = 0 otherwise.\nCreate ˆw(i) = [ ˆw(i)\nN/2, ˆw(i)\n0, and ˆw(i)\nN −1, ˆw(i)\n0, and ˆw(i)\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 4. Otherwise set\ni := i + 1 and go to Step 1.\n−→ε (i)\nStep 4 Output ˆw(i) as the decoded codeword.\nAn alternative approach that can be used is to make the\ntwo subdecoders exchange the more reliable messages after\nupdating the reliability message of each bit (or a group of\nbits). We observe that this “simultaneous replica” approach\nprovides a faster convergence, especially for Gallager-type\nLDPC codes. Thus all simulation results of LDPC codes are\nbased on simultaneous updating of each bit (or a group of\nbits). It is also straightforward to extend the replica shufﬂed\nBP decoding to the cases in which more than two replica\nsubdecoders are used. In order to decrease decoding delay\nof plain shufﬂed BP decoding, a parallel version of shufﬂed\nBP named group shufﬂed BP was developed in [7]. In a\nsimilar way, group replica shufﬂed BP can also preserve the\nparallelism advantage of the standard BP algorithm. It is also\nclear this idea can be extended to other other grouping scheme\n(e.g, [11]) and other iterative decoding algorithms, such as bit\nﬂipping and weighted bit ﬂipping decoding.\nD. Simulation results\nIV. ITERATIVE DECODING OF TURBO PRODUCT CODES\nA two-dimension turbo product code (TPC) can be denoted\nas C1\nC2, where C1 and C2 are two linear block codes.\nPlace k1 × k2 information symbols in an array of k1 rows\nand k2 columns, and then encode the k1 rows using code C2.\nAfterwards, the resulting n2 columns are encoded using code\nC1. Usually, we choose C1 the same as C2.\nThe conventional TPC decoder performs row and column\ndecoding in a serial fashion. A soft input soft output (SISO)\ndecoder, such as MAP, is used to decode each row or column.\nA low complexity decoding approach is provided in [13]. It\napplies the Chase algorithm iteratively on the row and column\ndecoding, but still in a serial fashion. In order to halve the\ndecoding latency, a parallel TPC decoder has been proposed\nin [10]. As opposed to the conventional serial TPC decoder,\nthe row and column decoders in this method operate in\nparallel and send each other the updated extrinsic information\nimmediately after a row or column has been decoded. The\nsimulation results reveal that this parallel decoder can reduce\ndecoding latency by half of that of the original decoder.\nB. Replica decoding of TPC\nBased on the parallel decoder, we propose a replica parallel\ndecoder as shown in Fig. 4, which can further reduce decoding\nlatency. Both row and column decoders are duplicated, but\nwork from opposite extremes, which means the two row\ndecoders process rows from the top and bottom, respectively\nand the two column decoders process columns from the\nleft and right, respectively. Row (Column) decoders send to\nboth column (row) decoders immediately the latest extrinsic\nmatrices, [Wcol\nB ] ([Wrow\nR ]), after a row\n(column) has been decoded.\nL ] and [Wrow\nT ] and [Wcol\nThe procedure of the replica parallel decoding is shown\nin Fig. 5. The circles denote bit positions that were already\nupdated by other decoders. Their number is much larger than\nthat in the parallel decoder, which greatly beneﬁts the decoding\nbecause most bits have more accurate priori information. The\narrows with letters aT , bT , etc., represent the processing order\nof different decoders. After both row (column) decoders ﬁnish\ndecoding all the rows (columns), the most reliable parts of\nthem are combined and the resulting extrinsic matrix for the\nnext iteration is transmitted to both column (row) decoders.\nthe performance of TPC (64, 57, 4)2.\nare\nThe weighting factors with the\n[0.1, 0.2, 0.3, 0.5, 0.7, 0.9, 1.0, 1.0]. The MAP algorithm is\nused to decode the two linear block codes. As shown in Fig.\n6, the performance at the third iteration in the replica decoder\nis better than that at the fourth iteration in the parallel decoder.",
         "",
         "",
         "Replica shufﬂed versions of interactive decoders of turbo codes, low-density parity-check codes\nand turbo product codes are presented. The proposed schemes converge faster than standard and\npreviously proposed shufﬂed approaches. Simulations show that the new schedules offer good\nperformance versus complexity/latency trade-offs.\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in whole or in part\nwithout payment of fee is granted for nonproﬁt educational and research purposes provided that all such whole or partial copies include\nthe following: a notice that such copying is by permission of Mitsubishi Electric Research Laboratories, Inc.; an acknowledgment of\nthe authors and individual contributions to the work; and all applicable portions of the copyright notice. Copying, reproduction, or\nrepublishing for any other purpose shall require a license with payment of fee to Mitsubishi Electric Research Laboratories, Inc. All\nrights reserved.\nMITSUBISHI ELECTRIC RESEARCH LABORATORIES\nhttp://www.merl.com\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang, Marc Fossorier, and Jonathan S. Yedidia\nAbstract\nReplica shufﬂed versions of iterative decoders of turbo codes, low-density parity-check\ncodes and turbo product codes are presented. The proposed schemes converge faster\nthan standard and previously proposed “shufﬂed” approaches. Simulations show that\nthe new schedules offer good performance versus complexity/latency trade-offs.\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in\nwhole or in part without payment of fee is granted for nonproﬁt educational and research purposes provided that all such\nwhole or partial copies include the following: a notice that such copying is by permission of Mitsubishi Electric Research\nLaboratories, Inc.; an acknowledgment of the authors and individual contributions to the work; and all applicable portions\nof the copyright notice. Copying, reproduction, or republishing for any other purpose shall require a license with payment\nof fee to Mitsubishi Electric Research Laboratories, Inc. All rights reserved.\nTo appear in the Proceedings of the 2005 International Symposium on Information Theory.\nReplica Shufﬂed Iterative Decoding\nJuntan Zhang, Yige Wang and Marc Fossorier\nDepartment of Electrical Engineering\nUniversity of Hawaii at Manoa\nHonolulu, HI 96822\nEmail: juntan, yige, marc@spectra.eng.hawaii.edu\nJonathan S. Yedidia\nMitsubishi Electric Research Laboratories\n201 Broadway, Cambridge, MA 02139\nEmail: yedidia@merl.com\nI. INTRODUCTION\nIterative decoding based on belief propagation (BP) [1] has\nreceived signiﬁcant attention recently, mostly due to its near-\nShannon-limit error performance for the decoding of low-\ndensity parity-check (LDPC) codes [2] and turbo codes [3].\nLike maximum a posterior probability (MAP) decoding [4],\nit is a symbol-by-symbol soft-in/soft-out decoding algorithm.\nIt processes the received symbols recursively to improve the\nreliability of each symbol based on constraints that specify\nthe code. In the ﬁrst iteration, the decoder only uses the\nchannel output, and generates soft output for each symbol.\nSubsequently, the output reliability measures of the decoded\nsymbols at the end of each decoding iteration are used as\ninputs for the next iteration. The decoding iteration process\ncontinues until a certain stopping condition is satisﬁed. Then\nhard decisions are made based on the output reliability mea-\nsures of the decoded symbols at the last decoding iteration.\nThe standard BP decoder of LDPC codes often needs several\ntens or hundreds of iterations for the decoding process to\nconverge, which is not desirable because of high decoding\ndelay. Furthermore, LDPC codes of interest can have large\nthe\ncodeword length and it can be difﬁcult\ndecoding in hardware in a fully parallel way. Because of\nthe serial property of the BCJR algorithm, decoders of turbo\ncodes cause high delay as well. In [5], [6] and [7], “shufﬂed”\nmethods were presented to reduce the required number of\niterations for decoding LDPC and turbo codes, respectively.\nA speedup factor of 2 for LDPC codes and the saving of\none iteration for turbo codes were reported. The aim of this\npaper is to introduce a “replica shufﬂed” scheme which further\naccelerate the decoding process for turbo codes, LDPC codes,\nturbo product codes and other iterative decodable codes.\nto implement\nII. ITERATIVE DECODING OF TURBO CODE\nA turbo code [3] encoder is constructed using a concatena-\ntion of two (or more) convolutional encoders, and its decoder\nconsists of two (or more) soft-in/soft-out convolutional de-\ncoders which feed reliability information to each other. For\nis the output code block at\nsimplicity, we consider a turbo code that consists of two rate-\n1/n systematic convolutional codes with encoders in feedback\nform. Let u = (u1, u2, . . . , uK) be an information block of\nlength K and c = (c1, c2, . . . , cK) be the corresponding\ncoded sequence, where ck = (ck,1, ck,2, . . . , ck,n), for k =\n1, 2, . . . , K,\ntime k. Suppose\nBPSK transmission over an AWGN channel, with uk and\nck,j all taking values in {+1, -1} for k = 1, 2, . . . , K and\nj = 1, 2, . . . , n. Let y = (y1, y2, . . . , yK) be the received\nsequence, where yk = (yk,1, yk,2, . . . , yk,n) is the received\nblock at time k. Let ˆu = {ˆu1, ˆu2, . . . , ˆuK} denote the estimate\nof u. Let sk denote the encoder state at time k. Following\n1 ), γk(s, s) = p(sk =\n[4], deﬁne: αk(s) = p(sk = s, yk\ns, yk|sk−1 = s), βk(s) = p(yK\na =\n(ya, ya+1, . . . , yb), and let α(m)\n(s) be\nk\nthe corresponding values computed in component decoder m,\nwith m = 1, 2. Let L(i)\nem(ˆuk) denote the extrinsic value of the\nestimated information bit ˆuk delivered by component decoder\nm at the ith iteration [8].\nk+1|sk = s), where yb\n(s), γ(m)\nA. Standard serial and parallel turbo decoding\nThe decoding approach proposed in [3] operates in serial\nmode, i.e., the component decoders take turns generating the\nextrinsic values of the estimated information symbols, and\neach component decoder uses the extrinsic messages delivered\nby the last component decoder as the a priori values of the\ninformation symbols. The disadvantage of this scheme is high\ndecoding delay. In the parallel turbo decoding algorithm [9],\nall component decoders operate in parallel at any given time.\nAfter each iteration, each component decoder delivers extrinsic\nmessages to other decoder(s) which use these messages as a\npriori values at the next iteration.\nB. Plain shufﬂed turbo decoding\nAlthough the parallel turbo decoding reduces the decoding\ndelay of serial decoding by half, the extrinsic messages are not\ntaken advantage of as soon as they become available, because\nthe extrinsic messages are delivered to component decoders\nonly after each iteration is completed. The aim of the shufﬂed\nturbo decoding is to use the more reliable extrinsic messages\nat each time. Let ˜u = (˜u1, ˜u2, . . . , ˜uK) be the sequence\npermuted by the interleaver corresponding to the original\ninformation sequence u = (u1, u2, . . . , uK), according to the\nmapping ˜uk = uπ(k), for k = 1, 2, . . . , K. We assume that k =\nπ(k), ∀k. There is a unique corresponding reverse mapping\nuk = ˜uπ−(k), for k = 1, 2, . . . , K and k = π−(k), ∀k. In\nshufﬂed turbo decoding, the two component decoders operate\nsimultaneously as in the parallel turbo decoding scheme, but\nthe messages are updated during each iteration based on π(k)\nand π−(k) [7]. Correspondingly it provides a faster decoding\nconvergence.\nC. Replica shufﬂed turbo decoding\n←−\nD 1 and\n−→\nD 1 and\n−→\nD 1, which are denoted by\nIn the plain shufﬂed turbo decoding summarized in Section\nthe component decoders process the\nII-B, we assume all\nbackward recursion followed by the forward recursion. Let\n−→\nus refer to the two component decoders as\nD 2.\nNaturally another possible scheme is to operate in the reverse\norder, i.e, all the component decoders process the forward\nrecursion followed by the backward recursion and we refer\n←−\nto them as\nD 2. In terms of error performance, there\nis no difference between these two approaches. However, the\nreliabilities of the extrinsic messages concerning a certain\ninformation bit delivered by these two shufﬂed turbo decoders\nare not the same. In general, the more independent information\nis used, the more reliable the delivered messages become.\nTherefore for the extrinsic messages delivered by component\n(i)\ndecoder\ne1 (ˆuk), the larger k\nis, the more reliable this message is. Following a similar\n←−\nanalysis, for the extrinsic message\nD 1,\nthe smaller k is, the more reliable this message is. It is natural\nto expect a faster decoding convergence if these two shufﬂed\nturbo decoders operate cooperatively instead of independently.\nBecause in this approach two sets of shufﬂed component\ndecoders are used to decode the same sequence of information\nbits, we refer to it as replica shufﬂed turbo decoding. In replica\nshufﬂed turbo decoding, two plain shufﬂed turbo decoders\n−→\n(processing recursions in opposite directions)\nD 2 and\n←−\n←−\nD 1 ,\nD 2 operate simultaneously and exchange more reliable\nextrinsic messages. We assume that the component decoders\n−→\n\n\ndeliver extrinsic messages synchronously, i.e.,\nk =\nk =\nT\n←−\n←−\n−→\n←−\n←−\n\n\n\n\n\nk)and\nk (\nT\nk, where the\nk) denote the times\nT\nT\nT\nk =\nT\n←−\n−→\n←−\n−→\nat which\nD 2) deliver the extrinsic values\nD 2 (\nD 1) and\nD 1 (\nof the k-th ((K + 1 − k)-th) estimated symbol of the original\ninformation sequence u and of the interleaved sequence ˜u,\nrespectively.\n(i)\ne1 (ˆuk) delivered by\n−→\nD 1 ,\n−→\nD 1. After time\nLet us ﬁrst consider the forward recursion stage at the i-\n−→\n\nk−1,\nT\nth iteration of component decoder\nthe values of −→α (1)\nk (s) should be updated and the values\nof −→γ (1)\nk (s) are needed. There are two possible cases. The\nﬁrst case is k > π−(k), which means the extrinsic value\n−→\n(i)\ne2 (ˆuk) of the information bit ˆuk has already been delivered\nL\n−→\nD 2. As in plain shufﬂed turbo decoding, this newly\nby decoder\n−→\n(i)\ne2 (ˆuk) is used to compute the values −→γ (1)\nL\navailable\nk (s),\n−→\n(i)\n−→α (1)\ne1 (ˆuk). The second case is k < π−(k), which\nL\nk (s), and\n(i)\ne2 (ˆuk) of the information bit ˆuk\nmeans the extrinsic value\n−→\nD 2. Then in plain shufﬂed\nhas not been delivered yet by\nturbo decoding, the values α(1)\ne1 (ˆuk) are updated\nk (s) and L(i)\n−→\nL\nk (s), and\nbased on the extrinsic messages delivered at last iteration.\nIn replica shufﬂed turbo decoding, however, there are two\nfurther subcases. The ﬁrst subcase is K + 1 − k < π−(k),\n(i)\ne2 (ˆuk) of the information\nwhich means the extrinsic value\n←−\nbit ˆuk has already been delivered by decoder\nD 2. Then this\n(i)\n(i−1)\ne2 (ˆuk), instead of\n(ˆuk) is used to\nnewly available\ne2\n−→\n(i)\ncompute the values −→γ (1)\nk (s), −→α (1)\ne1 (ˆuk). The\nL\nsecond subcase is K + 1 − k < π−(k), which means both\n(i)\nextrinsic messages of the information bit ˆuk, i.e,\ne2 (ˆuk)\n−→\n(i)\ne2 (ˆuk) are not available yet. In this subcase, the values\nL\nand\n(i)\nof −→α (1)\ne1 (ˆuk) are updated based on the extrinsic\nk (s) and\nmessages delivered at (i − 1)-th iteration. The recursions of\n←−\n←−\ncomponent decoders\nD 2 are realized based on\nD 1 and\nthe same principle. After Imax iterations, the shufﬂed turbo\ndecoding algorithm outputs ˆu = (ˆu1, ˆu2, . . . , ˆuK) as the de-\n←−\n(i)\ncoded codeword, where ˆuk =sgn[(\ne1 (ˆuk))/2 +\nL\n−→\n(i)\ne2 (ˆuk))/2 + 4\nyk,1], which is different from\nL\n(\nN0\nthat in the standard turbo decoding [3] and plain shufﬂed\nturbo decoding. It is straightforward to generalize the replica\nshufﬂed turbo decoding to multiple turbo codes which consist\nof more than two component codes. Also group of bits can\nbe updated periodically only to reduce information exchanges\nbetween replicas. Based on the above descriptions with two\nreplicas, the total computational complexity of the replica\nshufﬂed turbo decoding for multiple turbo codes at each\ndecoding iteration is about twice that of the parallel turbo\ndecoding.\n(i)\ne2 (ˆuk) +\n(i)\ne1 (ˆuk) +\n−→\nL\nD. Simulation results\nIII. ITERATIVE DECODING OF LDPC CODES\nLet H = [Hmn] be the parity check matrix which deﬁnes\nan LDPC code. We denote the set of bits that participate in\ncheck m by N (m) = {n : Hmn = 1} and the set of checks\nin which bit n participates as M(n) = {m : Hmn = 1}.\nAssume a codeword w = (w1, w2, . . . , wN ) is transmitted\nover an AWGN channel with zero mean and variance N0/2\nusing BPSK signaling and let y = (y1, y2, . . . , yN ) be the\ncorresponding received sequence.\nA. Standard BP for iterative decoding of LDPC codes\nBased on [1], let Fn be the log-likelihood ratio (LLR) of\nbit n and initially set Fn = 4\nmn be the\nN0\nLLR of bit n which is sent from check node m to bit node n\nand sent from the bit node n to check node m, respectively.\nLet z(i)\nn denote the a posteriori LLR of bit n. The standard\nBP algorithm [1] is carried out as follows:\nyn. Let ε(i)\nInitialization Set i = 1, maximum number of iterations to\nIM ax. For each m,n, set z(0)\nmn = Fn.\n(i) Horizontal Step, for 1 ≤ n ≤ N and each m ∈\nM(n), process:\nτ (i)\nmn =\nn∈N (m)\\n\n1 + τ (i)\nmn\n1 − τ (i)\nmn\n(ii) Vertical Step, for 1 ≤ n ≤ N and each m ∈\nε(i)\nmn = log\n(2)\nz(i)\nmn = Fn +\nz(i)\nn = Fn +\n(i)\nStep 2 Hard decision and stopping criterion test:\nCreate ˆw(i) = [ ˆw(i)\nn ] such that ˆw(i)\nz(i)\nn > 0, and ˆw(i)\nn = 1 if z(i)\nn < 0.\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 3. Otherwise set\ni := i + 1 and go to Step 1.\nn = 0 if\n(ii)\nStep 3 Output ˆw(i) as the decoded codeword.\nB. Plain shufﬂed BP for iterative decoding of LDPC codes\nAt the i-th iteration of the standard BP algorithm, ﬁrst all\nvalues of the check-to-bit messages are updated by using the\nvalues of the bit-to-check messages obtained at the (i − 1)-\nth iteration, i.e., each ε(i)\n:\nn ∈ N (m)\\n}. Then, all values of the bit-to-check messages\nare updated by using the values of the check-to-bit messages\nnewly obtained at the i-th iteration, i.e., each z(i)\nmn is updated\nfrom {ε(i)\nmn is updated by using {z(i−1)\nmn\nmn : m ∈ M(n)\\m}.\nWe observe that certain values z(i)\nin (1) to compute the remaining values ε(i)\nmn could already be\ncomputed in (3) based on a partial computation of the values\nε(i)\nmn obtained from (2), and then could be used instead of\nz(i−1)\nmn. Hence Step\nmn\n1 of the shufﬂed BP algorithm is performed as: for 1 ≤ n ≤ N\nand each m ∈ M(n), process the horizontal step and vertical\nstep jointly, with (1) modiﬁed as:\n\nτ (i)\nmn =\nn∈N (m)\\n\nn>n\nSince this algorithm becomes totally serial, bits can be grouped\nto maintain a sufﬁcient level of parallelism with the same error\nperformance [7].\nC. Replica shufﬂed BP decoding for LDPC codes\nPlain shufﬂed BP decoding is a bit-based sequential ap-\nproach and the method described in Section III-B is based\non a natural increasing order, i.e, belief messages concerning\nbit nodes are updated according to order i = 1, 2, . . . , N .\nThe larger the value of i, the more independent informations\nare used to update the beliefs of bit note i and the more\nreliable these belief messages are. Therefore the reliability of\nbit nodes increases and the error rate decreases as i increases.\nFollowing a similar analysis, in plain shufﬂed BP decoding\nbased on a natural decreasing order, after each iteration, the\nreliability of bit nodes decreases as i increases. As in replica\nshufﬂed turbo decoding, in replica shufﬂed BP decoding, two\nreplica shufﬂed subdecoders based on different updating orders\noperate simultaneously and cooperatively. After each iteration,\neach subdecoder receives more reliable messages from and\nsends more reliable messages to another subdecoder. Based on\nthese more reliable messages, both replica subdecoders begin\n←−\nthe next iteration of decoding. Let\nD denote the replica\nsubdecoder with natural increasing and decreasing updating\norder, respectively. Let −→ε (i)\nmn be the variables\n←−\nassociated with\nD\nare deﬁned in a similar way. The replica shufﬂed BP decoding\nwith two replica subdecoders is carried out as followings:\n−→\nD at iteration i. Variables associated with\nmn and −→z i\n−→\nD and\nInitialization Set i = 1, maximum number of iteration to\nIM ax. For each m,n, set −→z (0)\nmn = ←−z (0)\nmn = Fn.\nStep 1\nEach replica subdecoder process as the following\ntwo steps simultaneously. For 0 ≤ n ≤ N − 1 and\neach m ∈ M(n), process\nHorizontal Step\nn∈N (m)\\n\nn>n\n1 + −→τ (i)\nmn\n1 − −→τ (i)\nmn\nn∈N (m)\\n\nn<n\n1 + ←−τ (i)\nmn\n1 − ←−τ (i)\nmn\n(ii)\nVertical Step\n−→z (i)\nmn = Fn +\nmn = Fn +\n←−ε (i)\nmn\nStep 2 Exchange of\nthe more reliable messages. Set\nmn = −→z (i)\nmn for 0 ≤ n < N/2 and ←−z (i)\nmn = ←−z (i)\n−→z (i)\nfor N/2 ≤ n ≤ N − 1.\nStep 3\nHard decision and stopping criterion test:\nA. Conventional decoding methods of TPC\nn ] such that for 0 ≤ n <\nn = 1 if Fn +\nmn <\nn = 0 otherwise; for N/2 < n ≤\nn = 1 if Fn +\nmn <\nn = 0 otherwise.\nCreate ˆw(i) = [ ˆw(i)\nN/2, ˆw(i)\n0, and ˆw(i)\nN −1, ˆw(i)\n0, and ˆw(i)\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 4. Otherwise set\ni := i + 1 and go to Step 1.\n−→ε (i)\nStep 4 Output ˆw(i) as the decoded codeword.\nAn alternative approach that can be used is to make the\ntwo subdecoders exchange the more reliable messages after\nupdating the reliability message of each bit (or a group of\nbits). We observe that this “simultaneous replica” approach\nprovides a faster convergence, especially for Gallager-type\nLDPC codes. Thus all simulation results of LDPC codes are\nbased on simultaneous updating of each bit (or a group of\nbits). It is also straightforward to extend the replica shufﬂed\nBP decoding to the cases in which more than two replica\nsubdecoders are used. In order to decrease decoding delay\nof plain shufﬂed BP decoding, a parallel version of shufﬂed\nBP named group shufﬂed BP was developed in [7]. In a\nsimilar way, group replica shufﬂed BP can also preserve the\nparallelism advantage of the standard BP algorithm. It is also\nclear this idea can be extended to other other grouping scheme\n(e.g, [11]) and other iterative decoding algorithms, such as bit\nﬂipping and weighted bit ﬂipping decoding.\nD. Simulation results\nIV. ITERATIVE DECODING OF TURBO PRODUCT CODES\nA two-dimension turbo product code (TPC) can be denoted\nas C1\nC2, where C1 and C2 are two linear block codes.\nPlace k1 × k2 information symbols in an array of k1 rows\nand k2 columns, and then encode the k1 rows using code C2.\nAfterwards, the resulting n2 columns are encoded using code\nC1. Usually, we choose C1 the same as C2.\nThe conventional TPC decoder performs row and column\ndecoding in a serial fashion. A soft input soft output (SISO)\ndecoder, such as MAP, is used to decode each row or column.\nA low complexity decoding approach is provided in [13]. It\napplies the Chase algorithm iteratively on the row and column\ndecoding, but still in a serial fashion. In order to halve the\ndecoding latency, a parallel TPC decoder has been proposed\nin [10]. As opposed to the conventional serial TPC decoder,\nthe row and column decoders in this method operate in\nparallel and send each other the updated extrinsic information\nimmediately after a row or column has been decoded. The\nsimulation results reveal that this parallel decoder can reduce\ndecoding latency by half of that of the original decoder.\nB. Replica decoding of TPC\nBased on the parallel decoder, we propose a replica parallel\ndecoder as shown in Fig. 4, which can further reduce decoding\nlatency. Both row and column decoders are duplicated, but\nwork from opposite extremes, which means the two row\ndecoders process rows from the top and bottom, respectively\nand the two column decoders process columns from the\nleft and right, respectively. Row (Column) decoders send to\nboth column (row) decoders immediately the latest extrinsic\nmatrices, [Wcol\nB ] ([Wrow\nR ]), after a row\n(column) has been decoded.\nL ] and [Wrow\nT ] and [Wcol\nThe procedure of the replica parallel decoding is shown\nin Fig. 5. The circles denote bit positions that were already\nupdated by other decoders. Their number is much larger than\nthat in the parallel decoder, which greatly beneﬁts the decoding\nbecause most bits have more accurate priori information. The\narrows with letters aT , bT , etc., represent the processing order\nof different decoders. After both row (column) decoders ﬁnish\ndecoding all the rows (columns), the most reliable parts of\nthem are combined and the resulting extrinsic matrix for the\nnext iteration is transmitted to both column (row) decoders.\nthe performance of TPC (64, 57, 4)2.\nare\nThe weighting factors with the\n[0.1, 0.2, 0.3, 0.5, 0.7, 0.9, 1.0, 1.0]. The MAP algorithm is\nused to decode the two linear block codes. As shown in Fig.\n6, the performance at the third iteration in the replica decoder\nis better than that at the fourth iteration in the parallel decoder.",
         null
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "Diphasic effects of Astragalus membranaceus BUNGE (Leguminosae) on vascular tone in rat thoracic aorta.",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus BUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath. In endothelium-intact aortic rings, A. membranaceus extract induced a significant dose-dependent relaxation of the rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(omega)-nitro-arginine methyl ester or methylthioninium chloride. In endothelium-denuded ones, the extract could dose-dependently relax the rings contracted by phenylephrine, not by KCl; and it could also attenuate contractile response to phenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca(2+)-free medium; but it failed to affect the CaCl(2)-induced enhancement of contractile response to phenylephrine in Ca(2+)-free medium. These results indicate that nitric oxide signaling and Ca(2+)-handling pathway are involved in the A. membranaceus extract-induced vasodilatation.",
         "",
         "https://www.jstage.jst.go.jp/article/bpb/28/8/28_8_1450/_pdf",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus\nBUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath.\nIn endothelium-intact aortic rings, A. membranaceus extract induced a signiﬁcant dose-dependent relaxation of\nthe rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(ww)-nitro-arginine\nmethyl  ester  or  methylthioninium  chloride.  In  endothelium-denuded  ones,  the  extract  could  dose-dependently\nrelax  the  rings  contracted  by  phenylephrine,  not  by  KCl;  and  it  could  also  attenuate  contractile  response  to\nphenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca2-free medium; but it failed to affect the CaCl2-\ninduced enhancement of contractile response to phenylephrine in Ca2-free medium. These results indicate that\nnitric oxide signaling and Ca2-handling pathway are involved in the A. membranaceus extract-induced vasodi-\nlatation.",
         "The  roots  of  Astragalus  membranaceus BUNGE (Legumi-\nnosae) are amongst the most popular and important “Qi (pro-\nnounce  chee)  tonifying”  adaptogenic  herbs  in  China,  their\nuse  dates  back  more  than  2000  years,  and  are  recorded  in\nShen  Nong’s Materia  Medica.  According  to  ethnobotancical\ndata  collected  in  China,  A.  membranaceus has  been  pre-\nscribed  for  centuries  for  diarrhea,  frequent  colds,  sponta-\nneous  sweats,  fatigue,  and  edema.  Furthermore,  A.  mem-\nbranaceus is  alleged  to  possess  the  effects  of  antioxidative\ndamage,1) immune-stimulation,2) and antiviral infection.3)\nRecently,  it  has  received  more  attention  that  the  fact\nrecorded in ancient Chinese texts that A. membranaceus pos-\nsessed an antihypertensive effect by raising the yang qi of the\nspleen and kidney, since deﬁciency qi formed by the disorder\nof spleen and kidney is considered to be the basic pathogene-\nses  of  hypertension.4) A  great  deal  of  studies  demonstrated\nthat  A.  membranaceus was  beneﬁcial  on  the  process  of  hy-\npertension  in  animals  and  humans,5—7) and  a  cooperative\nmechanism  of\nrenin  angiotensin  aldosterone  system,\nkallikrein bradykinin system, and central neuropeptide8) was\nused  to  explain  this  effect  of  A.  membranaceus.  We  also\nfound  in  the  past  experiments  that  the  administration  of  A.\nmembranaceus could attenuate the progression of blood pres-\nsure  in  spontaneously  hypertensive  rats  concomitant  with\nrestoration  of  baroreﬂex  sensitivity  and  reduction  of  Ca2\nconcentration  in  lymphocyte  and  vascular  smooth  muscle\ncells  (VSMC).9—11) Additionally,  in  vivo  experiment  had\ndemonstrated that A. membranaceus could induce the vasodi-\nlatation.12) However,  there  is  no  experiment  evidence  avail-\nable  to  show  the  exact  mechanism  of  vasorelaxation  pro-\nduced by A. membranaceus. The present study is conducted\nto  elucidate  various  possible  mechanisms  in  vasomotor  re-\nsponse of A. membranaceus.",
         "M)  induced  a  similar  sustained  contraction\nof  aortic  rings  in  each  group  with  a  peak  tension  of  about\n3.110.27 g  in  intact  aortic  rings  and  3.020.19 g  in  de-\nnuded ones.\nEffects  of  A.  membranaceus on  the  Vasotension All\ndoses (2.0105—2.0101 g/l) of A. membranaceus exhib-\nited  negligible  vasomotor  actions  on  aortic  rings  with  or\nwithout  endothelium  at  resting  tension  (Fig.  1).  A.  mem-\nbranaceus (2.0105—2.0101 g/l)  produced  dose-depen-\ndent  relaxation  in  endothelium-denuded  rings  (Fig.  2),  after\nthe  plateau  tension  induced  by  PE.  However,  in  PE-precon-\ntracted  endothelium-intact  rings,  low  doses  (2.0104—\n6.0103 g/l) of A. membranaceus showed a more signiﬁcant\nrelaxation than that in denuded ones (Fig. 2), while high dose\n(2.0102—2.0101 g/l) of the extract induced a transient\ncontraction (Figs. 2, 3).\nEndothelium-Independent  Vasodilatation  by  A.  mem-\nIn  contrast  to  the  effect  on  PE-contracted  en-\nbranaceus\ndothelium-denuded rings in K–H solution, A. membranaceus\n(2.0104—2.0101 g/l)  had  no  dilatation-effect  on  rings\nstimulated by KCl (3.0102\nM) (Fig. 4).\nAnd in the experiment with Ca2-free medium, aortic con-\ntraction  elicited  by  PE,  caffeine  or  PD  was  decreased  sub-\nstantially, as these vascular contractions in Ca2-free medium\nare mediated only via the release of Ca2 from the intracellu-\nlar stores or phosphorylation of myosin light chain. A. mem-\nbranaceus (6.0103 g/l)  could  inhibit  the  contraction  in-\nduced  by  PE  (3.0107\nM),  but  not  block  the  extracellular\nCa2 inﬂow,  because  2.5 mM CaCl2 induced  a  similar  en-\nhancement of contraction by PE, compared with the control\n(95.842.63%  vs. 96.961.87%,  p0.05).  Furthermore,  A.\nmembranaceus (6.0103 g/l) failed to induce an attenuation\nof  contractile  response  by  PD  (1.0106\nM)  and  caffeine\n(2.5102\nM) (Fig. 5).\nEndothelium-Dependent  Vasomotor  Effects  by  A.\nmembranaceus Pre-incubation  of  the  intact  rings  with  L-\nM) was added, the effect of A. membranaceus (2.0102,\n6.0102 and 2.0101 g/l, showed by a, b, c point) was obtained. For it was similar with Fig. 2, the effect of A. membranaceus (2.0105—6.0103 g/l) was cut. Values of both\ngroups are expressed as meanS.E.M. of 8 experiments. Contractile responses are expressed as the percentage from maximal contraction elicited by phenylephrine (3.0107\nM).\n∗ p0.05, ∗∗ p0.01, compared with corresponding values before administration.\nNAME  (1.0104\nM)  markedly,  incompletely  inhibited  A.\nmembranaceus (6.0103 g/l)-induced  relaxation  from  36.0\nto 7.3% (Fig. 6). Similarly, MC (1.0105\nM) also showed an\ninhibition  of  endothelium-dependent  vasodilatation  from\n36.0  to  8.0%  (Fig.  6).  Thus,  NO  signaling  might  play  a\nmajor, but not full role in the vasodilatation effect of A. mem-\nbranaceus.\nWhen the dose of A. membranaceus in the organ bath was\nstepwise raised to 2.0102—2.0101 g/l, a transient con-\ntraction  could  be  observed  in  endothelium-intact  rings.  Pre-\ntreatment  of  endothelium-intact  rings  with  phosphoramidon\n(5.0106\nM)  signiﬁcantly  attenuated  the  transient  contrac-\ntion response (Fig. 3).",
         "DISCUSSION\nThe present study ﬁrst and detailedly explored the in vitro\nvascular effects of A. membranaceus in aortic rings isolated\nfrom rats. In the course of the study, a number of novel ob-\nservations  had  been  made,  which  contribute  to  better  scien-\ntiﬁc  understanding  of  their  therapeutic  use  for  antihyperten-\nsion associated with Qi deﬁciency caused by the disorder of\nspleen and kidney in China.\nFirst, although neurohumoral factors may contribute to an-\ntihypertension for the crude extract of A. membranaceus ob-\nserved previously,8) it is apparent that direct endothelium-de-\npendent  and  independent  vasorelaxation  should  be  promi-\nnently taken into account according to this study.\nSecond,  A.  membranaceus dose-dependently  dilated  en-\ndothelium-denuded  rings  contracted  with  PE.  However,  the\nagent had no effect on the vessel contracted by KCl. The cel-\nlular  mechanism  of  contraction  involved  in  the  response  to\nKCl and PE is different. KCl induce Ca2 inﬂux via voltage-\ndependent  Ca2 channel,  which  further  activate  Ca2-in-\nduced Ca2 release through ryanodine-receptor. PE increase\nintracellular  Ca2 concentration  with  two  mechanisms,  1)\nthrough  receptor-gated  Ca2 channels;  2)  mobilizes  Ca2\nfrom  intracellular  stores  via the  inositol-1,4,5-trisphosphate\n(IP3)  receptor  or  induces  secondly  myosin  light  chain  phos-\nphorylation via activating PKC.17,18) That is, the vasorelaxant\naction of A. membranaceus seems to occur in a receptor-de-\npendent manner in SMVC of rat aortas. However, the result\nthat A. membranaceus failed to inhibit the contraction caused\nby addition of CaCl2 in Ca2 free solution was probably only\ndue to its effects on the intracellular pathways. This hypothe-\nsis was conﬁrmed in the further experiments that the contrac-\ntions induced by PD and caffeine in Ca2-free medium were\nnot  affected  by  A.  membranaceus equally.  Thus,  it  appears\nthat the IP3-induced Ca2 release channels might be the site\nof action for A. membranaceus on endothelium-removed aor-\ntas.\nThird,  NO  is  a  potent  vasodilator  synthesized  in  the  en-\ndothelium19) by  NO  synthase,  and  causes  VSMC  relaxation\nthrough  the  activation  of  soluble  guanylate  cyclase.20) The\npresent  study  demonstrated  that  the  extract  of  A.  mem-\nbranaceus dose-dependently  inhibited  the  contraction  in-\nduced  by  PE  in  intact  aorta  isolated  from  rats.  This  vasore-\nlaxant  action  was  mostly  inhibited  by  treatment  with  L-\nNAME  or  MC.  Endothelium-dependent  relaxation  of  A.\nmembranaceus seemed  to  be  associated  with  NO  signaling\nvia guanylate cyclase activation since both L-NAME and MC\ncould  attenuate  this  response.  This  result  was  supported  by\nother studies which had shown that A. membranaceus could\nincrease the plasma level of NO21) and induce the release of\nNO  from  VSMC  in  vitro  by  NO-soluble  guanylate  cyclase-\ncyclic guanosine monophosphate pathway.22)\nFinally,  it  was  also  observed  that  high  dose  (2.0102—\n2.0101 g/l) A. membranaceus induced a transient contrac-\ntion  in  arterial  rings  pre-constricted  by  PE.  This  result  was\nidentical with that obtained in another report where treatment\nwith  high  dose  of  A.  membranaceus reversibly  increased\nblood pressure of patients without hypertension previously.23)\nThe  mechanism  underlying  is  under  investigation.  Blockage\nof  the  contraction  by  endothelin-converting  enzyme  phos-\nphoramidon  may  suggest  the  involvement  of  an  endothelin\nreceptor mediated signaling.\nIn summary, the ﬁndings indicated that A. membranaceus\nhave  diphasic  effects  of  relaxation  and  contraction  on  the\naorta rings. The mechanisms of relaxation may include inhi-\nbition  of  intracellular  Ca2 release  in  vascular  smooth  mus-\ncle cells, and endothelium dependent relaxation mediated by\nIn the untreatment group, A. membranaceus or vehicle group was not pretreated with\nL-NAME or MC. Each column with a bar represents meanS.E.M. of 8 experiments.\nContractile  responses  are  expressed  as  the  percentage  from  maximal  contraction\nelicited by phenylephrine (3.0107\nM). ∗ p0.05, ∗∗ p0.01, compared with value of\nthe  corresponding  vehicle  group.   p0.01,  compared  with  value  of  the  experiment\nsubgroup in the untreatment group.\nthe NO-GC pathway, whilst the effect of transient contraction\nof A. membranaceus at a high dose was related with endothe-\nlin release.",
         "MATERIALS AND METHODS\nPlant  Material  and  Preparation  of  the  Extracts A.\nmembranaceus roots were collected in the northern region of\nGansu Province, China, in the autumn (2001) and identiﬁed\nby the Quality Assessment of Di’ao Group, Sichuang, China.\nAnd a voucher specimen was deposited in the Academic De-\npartment of Di’ao Group (STQ-C-00203).\nA. membranaceus roots were air dried. A crude extract was\nprepared  by  decoction  of  2.0102 g  in  500 ml  water  for  3\ntimes (90 min per time). The obtained extract was combined,\nﬁltered  and  concentrated.  Then  all  the  aqueous  extract  was\nextracted  with  400 ml  ethanol  (75%  and  85%)  twice.  After\nthe ethanol was retrieved, the active fractions were recovered,\nﬁltered  and  lyophilized  (yield  0.28%).  Consequently,  the\nﬁnal  extract  was  used  in  the  study  after  dissolution  in  dis-\ntilled water at 37 °C and stirred for 60 min.\nChemicals Phenylephrine  (PE),  acetylcholine  (ACh),  L-\nN(w)-nitro-arginine  methyl  ester  (L-NAME),  methylthion-\ninium  chloride  (MC),  phosphoramidon,  ethylene  glycol-bis\n[b-aminoethyl  ether]-N,N,N,N-tetraacetic  acid  (EGTA),\ncaffeine  and  phorbol-12,13-diacetate  (PD)  were  purchased\nfrom  Sigma  Chemical  Co.  (St.  Louis,  MO,  U.S.A.).  They\nwere  dissolved\nin  distilled  water  and  diluted  with\nKrebs–Henseleit  (K–H)  solution  before  use.  All  chemicals\nwere of the highest purity available.\nExperiment  Animals The  investigation  conformed  to\nthe  Chinese  Regulation  for  Administration  of  Laboratory\nAnimal  (SSTC  1988/2).  Health  male  Sprague–Dawley  rats\nweighing  260—280 g  (Laboratory  Animal  Center  of  Zhe-\njiang  College  of  Traditional  Chinese  Medicine,  Hangzhou,\nChina) were housed in conventional cages with free access to\nwater and rodent chow at the controlled temperature and hu-\nmidity with a 12-h light/dark cycle.\nPreparation of Thoracic Arterial Rings from Sprague–\nDawley  Rats Rats  were  euthanized  by  decapitation  using\nsmall animal guillotine. The thoracic aorta was immediately\nisolated  and  immersed  in  oxygenated  K–H  solution  4 °C  of\nthe  following  composition  (in  mM):  NaCl  118.3;  KCl  4.7;\nMgSO4 1.2; KH2PO4 1.2; NaHCO3 25.0; CaCl2 2.5; Glucose\n11.0.  Then  the  adherent  connective  tissue  was  cleaned  and\nblood  vessel  was  cut  into  3—4 mm  rings,  with  the  special\ncare  of  avoiding  damage  of  endothelium.  In  some  prepara-\ntions, endothelium was mechanically removed by gently rub-\nbing  the  lumen.  The  rings  were  mounted  horizontally  be-\ntween two stirrups in organ bath ﬁlled with 10 ml K–H solu-\ntion  at  37 °C,  ventilated  continuously  with  95%  O2 and  5%\nCO2. The isometric tension was recorded with a force trans-\nducer  (JZ101,  metrical  range:  0—5.0 g)  and  MedLab  5.0v\nrecording system (Nanjing Medease Science and Technology\nCo. Ltd).\nExperimental  Procedure  of  Isometric  Tension Rings\nwere equilibrated for 60 min at 2.0 g resting tension, and then\nchallenged with KCl (6.0102\nM) at least 3 times until a re-\nproducible maximal contractile response was obtained. After\na  further  equilibration  period  of  30 min,  the  integrity  of  the\nendothelium was assessed in all preparations by determining\nthe ability of ACh (1.0105\nM) to induce more than 80% re-\nlaxation  of  rings  pre-contracted  with  PE  (1.0106\nM).  The\nendothelium  was  considered  to  be  removed  when  there  was\nless than 10% relaxation response to ACh.\nProtocol 1: Effects of A. membranaceus on Vascular Tone:\nThe ﬁrst series of the experiments were conducted to assess\nendothelium-dependent  or  independent  effects  of  A.  mem-\nbranaceus on isolated aortic rings. When the tension was at\nresting  state  or  reached  a  plateau  induced  by  PE  (3.0\n107\nM),  A.  membranaceus (2.0105—2.0101 g/l)  was\ncumulatively added into the organ bath with 10 min interval.\nThe  rings  with  intact  or  denuded  endothelium  were  always\ntested in parallel.\nProtocol  2:  Endothelium-Independent  Vasomotor  Re-\nsponse by A. membranaceus: The second series of the exper-\niments  were  designed  to  determine  the  underlying  mecha-\nnisms  of  A.  membranaceus-induced  vasomotor  response  in\naortic rings without endothelium. In contrast to the response\nof  A.  membranaceus (2.0105—2.0101 g/l)  to  PE,  KCl\n(3.0102\nM)  was  also  employed  to  constrict  the  denuded\naortic rings. A plateau contractile response to KCl or PE was\nalso obtained as a time control.\nM), PD (1.0106\nAdditionally,  after  the  preparations  were  serially  washed\nand equilibrated for 60 min by Ca2-free medium (mM: NaCl\n118.3, KCl 4.7, KH2PO4 1.2, MgSO4 1.2, NaHCO3 25, Glu-\ncose 11.0, EGTA 0.05), PE (3.0107\nM)\n(an  activator  of  protein  kinase  C,  PKC),13) or  caffeine\n(2.5102\nM) (an activator of ryanodine receptor)14) was used\nto contract the denuded rings. In order to explore the proba-\nble  effect  of  extracellular  Ca2 inﬂow,  2.5 mM CaCl2 was\nadded  after  PE  stimulated  the  rings.  Then  the  effect  of  A.\nmembranaceus (6.0103 g/l)  was  tested.  A  time  control\n(same protocol as above except for the omission of A. mem-\nbranaceus to  the  second  application  of  PE,  PD  or  caffeine)\nwas  always  run  in  parallel.  Following  the  initial  control  re-\nsponse in Ca2-free medium and before the second test con-\ntraction (to PE or caffeine), a 6.0102\nM KCl-induced con-\ntraction  in  K–H  solution  was  obtained  to  reﬁll  the  Ca2\nstores in the vascular tissues.\nProtocol 3: Endothelium-Dependent Vasomotor Response\nby A. membranaceus: In the third series of experiments, the\npotential inﬂuence of A. membranaceus on PE-induced vaso-\nmotor function in the intact rings was examined. For pretreat-\nment,  endothelium-intact  rings  were  incubated  with  L-\nNAME (1.0104\nM) (a speciﬁc inhibitor of nitric oxide syn-\nthase), MC (1.0105\nM) (an inhibitor of guanylate cyclase)\nfor 15 min15) and phosphoramidon (5.0106\nM) (an inhibitor\nof  endothelin  converting  enzyme)  for  20 min16) before  ad-\nministration  of  PE.  Then  the  response  curves  of  A.  mem-\nbranaceus were recorded. The rings with or without the treat-\nment of inhibitors were always tested in parallel.\nStatistical  Analysis Relaxant  responses  were  expressed\nas the percentage decreases of the magnitude of the contrac-\ntion  induced  by  PE  or  KCl  (100%)  before  the  application\nof  vasodilators,  and  the  contractile  responses  as  the  ratio  of\nthe  responses  following  and  before  the  application  of  A.\nmembranaceus (second  response/ﬁrst  response).  All  results\nare  expressed  as  meanS.E.M.  Statistical  analysis  was  per-\nformed  with  t-test  or  ANOVA  followed  by  Newman–Keuls\ntest. Differences were accepted as statistically signiﬁcant at p\nvalues 0.05 (GraphPad Prism).\nRESULTS\nPE  (1.0106",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus\nBUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath.\nIn endothelium-intact aortic rings, A. membranaceus extract induced a signiﬁcant dose-dependent relaxation of\nthe rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(ww)-nitro-arginine\nmethyl  ester  or  methylthioninium  chloride.  In  endothelium-denuded  ones,  the  extract  could  dose-dependently\nrelax  the  rings  contracted  by  phenylephrine,  not  by  KCl;  and  it  could  also  attenuate  contractile  response  to\nphenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca2-free medium; but it failed to affect the CaCl2-\ninduced enhancement of contractile response to phenylephrine in Ca2-free medium. These results indicate that\nnitric oxide signaling and Ca2-handling pathway are involved in the A. membranaceus extract-induced vasodi-\nlatation.\nThe  roots  of  Astragalus  membranaceus BUNGE (Legumi-\nnosae) are amongst the most popular and important “Qi (pro-\nnounce  chee)  tonifying”  adaptogenic  herbs  in  China,  their\nuse  dates  back  more  than  2000  years,  and  are  recorded  in\nShen  Nong’s Materia  Medica.  According  to  ethnobotancical\ndata  collected  in  China,  A.  membranaceus has  been  pre-\nscribed  for  centuries  for  diarrhea,  frequent  colds,  sponta-\nneous  sweats,  fatigue,  and  edema.  Furthermore,  A.  mem-\nbranaceus is  alleged  to  possess  the  effects  of  antioxidative\ndamage,1) immune-stimulation,2) and antiviral infection.3)\nRecently,  it  has  received  more  attention  that  the  fact\nrecorded in ancient Chinese texts that A. membranaceus pos-\nsessed an antihypertensive effect by raising the yang qi of the\nspleen and kidney, since deﬁciency qi formed by the disorder\nof spleen and kidney is considered to be the basic pathogene-\nses  of  hypertension.4) A  great  deal  of  studies  demonstrated\nthat  A.  membranaceus was  beneﬁcial  on  the  process  of  hy-\npertension  in  animals  and  humans,5—7) and  a  cooperative\nmechanism  of\nrenin  angiotensin  aldosterone  system,\nkallikrein bradykinin system, and central neuropeptide8) was\nused  to  explain  this  effect  of  A.  membranaceus.  We  also\nfound  in  the  past  experiments  that  the  administration  of  A.\nmembranaceus could attenuate the progression of blood pres-\nsure  in  spontaneously  hypertensive  rats  concomitant  with\nrestoration  of  baroreﬂex  sensitivity  and  reduction  of  Ca2\nconcentration  in  lymphocyte  and  vascular  smooth  muscle\ncells  (VSMC).9—11) Additionally,  in  vivo  experiment  had\ndemonstrated that A. membranaceus could induce the vasodi-\nlatation.12) However,  there  is  no  experiment  evidence  avail-\nable  to  show  the  exact  mechanism  of  vasorelaxation  pro-\nduced by A. membranaceus. The present study is conducted\nto  elucidate  various  possible  mechanisms  in  vasomotor  re-\nsponse of A. membranaceus.\nMATERIALS AND METHODS\nPlant  Material  and  Preparation  of  the  Extracts A.\nmembranaceus roots were collected in the northern region of\nGansu Province, China, in the autumn (2001) and identiﬁed\nby the Quality Assessment of Di’ao Group, Sichuang, China.\nAnd a voucher specimen was deposited in the Academic De-\npartment of Di’ao Group (STQ-C-00203).\nA. membranaceus roots were air dried. A crude extract was\nprepared  by  decoction  of  2.0102 g  in  500 ml  water  for  3\ntimes (90 min per time). The obtained extract was combined,\nﬁltered  and  concentrated.  Then  all  the  aqueous  extract  was\nextracted  with  400 ml  ethanol  (75%  and  85%)  twice.  After\nthe ethanol was retrieved, the active fractions were recovered,\nﬁltered  and  lyophilized  (yield  0.28%).  Consequently,  the\nﬁnal  extract  was  used  in  the  study  after  dissolution  in  dis-\ntilled water at 37 °C and stirred for 60 min.\nChemicals Phenylephrine  (PE),  acetylcholine  (ACh),  L-\nN(w)-nitro-arginine  methyl  ester  (L-NAME),  methylthion-\ninium  chloride  (MC),  phosphoramidon,  ethylene  glycol-bis\n[b-aminoethyl  ether]-N,N,N,N-tetraacetic  acid  (EGTA),\ncaffeine  and  phorbol-12,13-diacetate  (PD)  were  purchased\nfrom  Sigma  Chemical  Co.  (St.  Louis,  MO,  U.S.A.).  They\nwere  dissolved\nin  distilled  water  and  diluted  with\nKrebs–Henseleit  (K–H)  solution  before  use.  All  chemicals\nwere of the highest purity available.\nExperiment  Animals The  investigation  conformed  to\nthe  Chinese  Regulation  for  Administration  of  Laboratory\nAnimal  (SSTC  1988/2).  Health  male  Sprague–Dawley  rats\nweighing  260—280 g  (Laboratory  Animal  Center  of  Zhe-\njiang  College  of  Traditional  Chinese  Medicine,  Hangzhou,\nChina) were housed in conventional cages with free access to\nwater and rodent chow at the controlled temperature and hu-\nmidity with a 12-h light/dark cycle.\nPreparation of Thoracic Arterial Rings from Sprague–\nDawley  Rats Rats  were  euthanized  by  decapitation  using\nsmall animal guillotine. The thoracic aorta was immediately\nisolated  and  immersed  in  oxygenated  K–H  solution  4 °C  of\nthe  following  composition  (in  mM):  NaCl  118.3;  KCl  4.7;\nMgSO4 1.2; KH2PO4 1.2; NaHCO3 25.0; CaCl2 2.5; Glucose\n11.0.  Then  the  adherent  connective  tissue  was  cleaned  and\nblood  vessel  was  cut  into  3—4 mm  rings,  with  the  special\ncare  of  avoiding  damage  of  endothelium.  In  some  prepara-\ntions, endothelium was mechanically removed by gently rub-\nbing  the  lumen.  The  rings  were  mounted  horizontally  be-\ntween two stirrups in organ bath ﬁlled with 10 ml K–H solu-\ntion  at  37 °C,  ventilated  continuously  with  95%  O2 and  5%\nCO2. The isometric tension was recorded with a force trans-\nducer  (JZ101,  metrical  range:  0—5.0 g)  and  MedLab  5.0v\nrecording system (Nanjing Medease Science and Technology\nCo. Ltd).\nExperimental  Procedure  of  Isometric  Tension Rings\nwere equilibrated for 60 min at 2.0 g resting tension, and then\nchallenged with KCl (6.0102\nM) at least 3 times until a re-\nproducible maximal contractile response was obtained. After\na  further  equilibration  period  of  30 min,  the  integrity  of  the\nendothelium was assessed in all preparations by determining\nthe ability of ACh (1.0105\nM) to induce more than 80% re-\nlaxation  of  rings  pre-contracted  with  PE  (1.0106\nM).  The\nendothelium  was  considered  to  be  removed  when  there  was\nless than 10% relaxation response to ACh.\nProtocol 1: Effects of A. membranaceus on Vascular Tone:\nThe ﬁrst series of the experiments were conducted to assess\nendothelium-dependent  or  independent  effects  of  A.  mem-\nbranaceus on isolated aortic rings. When the tension was at\nresting  state  or  reached  a  plateau  induced  by  PE  (3.0\n107\nM),  A.  membranaceus (2.0105—2.0101 g/l)  was\ncumulatively added into the organ bath with 10 min interval.\nThe  rings  with  intact  or  denuded  endothelium  were  always\ntested in parallel.\nProtocol  2:  Endothelium-Independent  Vasomotor  Re-\nsponse by A. membranaceus: The second series of the exper-\niments  were  designed  to  determine  the  underlying  mecha-\nnisms  of  A.  membranaceus-induced  vasomotor  response  in\naortic rings without endothelium. In contrast to the response\nof  A.  membranaceus (2.0105—2.0101 g/l)  to  PE,  KCl\n(3.0102\nM)  was  also  employed  to  constrict  the  denuded\naortic rings. A plateau contractile response to KCl or PE was\nalso obtained as a time control.\nM), PD (1.0106\nAdditionally,  after  the  preparations  were  serially  washed\nand equilibrated for 60 min by Ca2-free medium (mM: NaCl\n118.3, KCl 4.7, KH2PO4 1.2, MgSO4 1.2, NaHCO3 25, Glu-\ncose 11.0, EGTA 0.05), PE (3.0107\nM)\n(an  activator  of  protein  kinase  C,  PKC),13) or  caffeine\n(2.5102\nM) (an activator of ryanodine receptor)14) was used\nto contract the denuded rings. In order to explore the proba-\nble  effect  of  extracellular  Ca2 inﬂow,  2.5 mM CaCl2 was\nadded  after  PE  stimulated  the  rings.  Then  the  effect  of  A.\nmembranaceus (6.0103 g/l)  was  tested.  A  time  control\n(same protocol as above except for the omission of A. mem-\nbranaceus to  the  second  application  of  PE,  PD  or  caffeine)\nwas  always  run  in  parallel.  Following  the  initial  control  re-\nsponse in Ca2-free medium and before the second test con-\ntraction (to PE or caffeine), a 6.0102\nM KCl-induced con-\ntraction  in  K–H  solution  was  obtained  to  reﬁll  the  Ca2\nstores in the vascular tissues.\nProtocol 3: Endothelium-Dependent Vasomotor Response\nby A. membranaceus: In the third series of experiments, the\npotential inﬂuence of A. membranaceus on PE-induced vaso-\nmotor function in the intact rings was examined. For pretreat-\nment,  endothelium-intact  rings  were  incubated  with  L-\nNAME (1.0104\nM) (a speciﬁc inhibitor of nitric oxide syn-\nthase), MC (1.0105\nM) (an inhibitor of guanylate cyclase)\nfor 15 min15) and phosphoramidon (5.0106\nM) (an inhibitor\nof  endothelin  converting  enzyme)  for  20 min16) before  ad-\nministration  of  PE.  Then  the  response  curves  of  A.  mem-\nbranaceus were recorded. The rings with or without the treat-\nment of inhibitors were always tested in parallel.\nStatistical  Analysis Relaxant  responses  were  expressed\nas the percentage decreases of the magnitude of the contrac-\ntion  induced  by  PE  or  KCl  (100%)  before  the  application\nof  vasodilators,  and  the  contractile  responses  as  the  ratio  of\nthe  responses  following  and  before  the  application  of  A.\nmembranaceus (second  response/ﬁrst  response).  All  results\nare  expressed  as  meanS.E.M.  Statistical  analysis  was  per-\nformed  with  t-test  or  ANOVA  followed  by  Newman–Keuls\ntest. Differences were accepted as statistically signiﬁcant at p\nvalues 0.05 (GraphPad Prism).\nRESULTS\nPE  (1.0106\nM)  induced  a  similar  sustained  contraction\nof  aortic  rings  in  each  group  with  a  peak  tension  of  about\n3.110.27 g  in  intact  aortic  rings  and  3.020.19 g  in  de-\nnuded ones.\nEffects  of  A.  membranaceus on  the  Vasotension All\ndoses (2.0105—2.0101 g/l) of A. membranaceus exhib-\nited  negligible  vasomotor  actions  on  aortic  rings  with  or\nwithout  endothelium  at  resting  tension  (Fig.  1).  A.  mem-\nbranaceus (2.0105—2.0101 g/l)  produced  dose-depen-\ndent  relaxation  in  endothelium-denuded  rings  (Fig.  2),  after\nthe  plateau  tension  induced  by  PE.  However,  in  PE-precon-\ntracted  endothelium-intact  rings,  low  doses  (2.0104—\n6.0103 g/l) of A. membranaceus showed a more signiﬁcant\nrelaxation than that in denuded ones (Fig. 2), while high dose\n(2.0102—2.0101 g/l) of the extract induced a transient\ncontraction (Figs. 2, 3).\nEndothelium-Independent  Vasodilatation  by  A.  mem-\nIn  contrast  to  the  effect  on  PE-contracted  en-\nbranaceus\ndothelium-denuded rings in K–H solution, A. membranaceus\n(2.0104—2.0101 g/l)  had  no  dilatation-effect  on  rings\nstimulated by KCl (3.0102\nM) (Fig. 4).\nAnd in the experiment with Ca2-free medium, aortic con-\ntraction  elicited  by  PE,  caffeine  or  PD  was  decreased  sub-\nstantially, as these vascular contractions in Ca2-free medium\nare mediated only via the release of Ca2 from the intracellu-\nlar stores or phosphorylation of myosin light chain. A. mem-\nbranaceus (6.0103 g/l)  could  inhibit  the  contraction  in-\nduced  by  PE  (3.0107\nM),  but  not  block  the  extracellular\nCa2 inﬂow,  because  2.5 mM CaCl2 induced  a  similar  en-\nhancement of contraction by PE, compared with the control\n(95.842.63%  vs. 96.961.87%,  p0.05).  Furthermore,  A.\nmembranaceus (6.0103 g/l) failed to induce an attenuation\nof  contractile  response  by  PD  (1.0106\nM)  and  caffeine\n(2.5102\nM) (Fig. 5).\nEndothelium-Dependent  Vasomotor  Effects  by  A.\nmembranaceus Pre-incubation  of  the  intact  rings  with  L-\nM) was added, the effect of A. membranaceus (2.0102,\n6.0102 and 2.0101 g/l, showed by a, b, c point) was obtained. For it was similar with Fig. 2, the effect of A. membranaceus (2.0105—6.0103 g/l) was cut. Values of both\ngroups are expressed as meanS.E.M. of 8 experiments. Contractile responses are expressed as the percentage from maximal contraction elicited by phenylephrine (3.0107\nM).\n∗ p0.05, ∗∗ p0.01, compared with corresponding values before administration.\nNAME  (1.0104\nM)  markedly,  incompletely  inhibited  A.\nmembranaceus (6.0103 g/l)-induced  relaxation  from  36.0\nto 7.3% (Fig. 6). Similarly, MC (1.0105\nM) also showed an\ninhibition  of  endothelium-dependent  vasodilatation  from\n36.0  to  8.0%  (Fig.  6).  Thus,  NO  signaling  might  play  a\nmajor, but not full role in the vasodilatation effect of A. mem-\nbranaceus.\nWhen the dose of A. membranaceus in the organ bath was\nstepwise raised to 2.0102—2.0101 g/l, a transient con-\ntraction  could  be  observed  in  endothelium-intact  rings.  Pre-\ntreatment  of  endothelium-intact  rings  with  phosphoramidon\n(5.0106\nM)  signiﬁcantly  attenuated  the  transient  contrac-\ntion response (Fig. 3).\nDISCUSSION\nThe present study ﬁrst and detailedly explored the in vitro\nvascular effects of A. membranaceus in aortic rings isolated\nfrom rats. In the course of the study, a number of novel ob-\nservations  had  been  made,  which  contribute  to  better  scien-\ntiﬁc  understanding  of  their  therapeutic  use  for  antihyperten-\nsion associated with Qi deﬁciency caused by the disorder of\nspleen and kidney in China.\nFirst, although neurohumoral factors may contribute to an-\ntihypertension for the crude extract of A. membranaceus ob-\nserved previously,8) it is apparent that direct endothelium-de-\npendent  and  independent  vasorelaxation  should  be  promi-\nnently taken into account according to this study.\nSecond,  A.  membranaceus dose-dependently  dilated  en-\ndothelium-denuded  rings  contracted  with  PE.  However,  the\nagent had no effect on the vessel contracted by KCl. The cel-\nlular  mechanism  of  contraction  involved  in  the  response  to\nKCl and PE is different. KCl induce Ca2 inﬂux via voltage-\ndependent  Ca2 channel,  which  further  activate  Ca2-in-\nduced Ca2 release through ryanodine-receptor. PE increase\nintracellular  Ca2 concentration  with  two  mechanisms,  1)\nthrough  receptor-gated  Ca2 channels;  2)  mobilizes  Ca2\nfrom  intracellular  stores  via the  inositol-1,4,5-trisphosphate\n(IP3)  receptor  or  induces  secondly  myosin  light  chain  phos-\nphorylation via activating PKC.17,18) That is, the vasorelaxant\naction of A. membranaceus seems to occur in a receptor-de-\npendent manner in SMVC of rat aortas. However, the result\nthat A. membranaceus failed to inhibit the contraction caused\nby addition of CaCl2 in Ca2 free solution was probably only\ndue to its effects on the intracellular pathways. This hypothe-\nsis was conﬁrmed in the further experiments that the contrac-\ntions induced by PD and caffeine in Ca2-free medium were\nnot  affected  by  A.  membranaceus equally.  Thus,  it  appears\nthat the IP3-induced Ca2 release channels might be the site\nof action for A. membranaceus on endothelium-removed aor-\ntas.\nThird,  NO  is  a  potent  vasodilator  synthesized  in  the  en-\ndothelium19) by  NO  synthase,  and  causes  VSMC  relaxation\nthrough  the  activation  of  soluble  guanylate  cyclase.20) The\npresent  study  demonstrated  that  the  extract  of  A.  mem-\nbranaceus dose-dependently  inhibited  the  contraction  in-\nduced  by  PE  in  intact  aorta  isolated  from  rats.  This  vasore-\nlaxant  action  was  mostly  inhibited  by  treatment  with  L-\nNAME  or  MC.  Endothelium-dependent  relaxation  of  A.\nmembranaceus seemed  to  be  associated  with  NO  signaling\nvia guanylate cyclase activation since both L-NAME and MC\ncould  attenuate  this  response.  This  result  was  supported  by\nother studies which had shown that A. membranaceus could\nincrease the plasma level of NO21) and induce the release of\nNO  from  VSMC  in  vitro  by  NO-soluble  guanylate  cyclase-\ncyclic guanosine monophosphate pathway.22)\nFinally,  it  was  also  observed  that  high  dose  (2.0102—\n2.0101 g/l) A. membranaceus induced a transient contrac-\ntion  in  arterial  rings  pre-constricted  by  PE.  This  result  was\nidentical with that obtained in another report where treatment\nwith  high  dose  of  A.  membranaceus reversibly  increased\nblood pressure of patients without hypertension previously.23)\nThe  mechanism  underlying  is  under  investigation.  Blockage\nof  the  contraction  by  endothelin-converting  enzyme  phos-\nphoramidon  may  suggest  the  involvement  of  an  endothelin\nreceptor mediated signaling.\nIn summary, the ﬁndings indicated that A. membranaceus\nhave  diphasic  effects  of  relaxation  and  contraction  on  the\naorta rings. The mechanisms of relaxation may include inhi-\nbition  of  intracellular  Ca2 release  in  vascular  smooth  mus-\ncle cells, and endothelium dependent relaxation mediated by\nIn the untreatment group, A. membranaceus or vehicle group was not pretreated with\nL-NAME or MC. Each column with a bar represents meanS.E.M. of 8 experiments.\nContractile  responses  are  expressed  as  the  percentage  from  maximal  contraction\nelicited by phenylephrine (3.0107\nM). ∗ p0.05, ∗∗ p0.01, compared with value of\nthe  corresponding  vehicle  group.   p0.01,  compared  with  value  of  the  experiment\nsubgroup in the untreatment group.\nthe NO-GC pathway, whilst the effect of transient contraction\nof A. membranaceus at a high dose was related with endothe-\nlin release.",
         null
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "Critical roles of CD146 in zebrafish vascular development",
         "In this report, we use zebrafish as a model system to understand the importance of CD146 in vascular development. Endothelial‐specific expression of CD146 was verified by whole‐mount in situ hybridization. Suppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly developed intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood flow through the ISV region, despite that fluorescence microangiography showed that the ISVs were present. This finding suggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally, remodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of CD146 protein expression resulted in a circulation shunt that caused the circulation to by‐pass part of the caudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino oligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N terminus deleted in an in‐frame manner. This region, therefore, is likely to contain elements critical for CD146 function. This study provides the first in vivo functional assessment of CD146 in embryonic development by showing that knockdown of CD146 protein expression severely hinders vascular development in zebrafish. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley‐Liss, Inc.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/dvdy.20220",
         "In this report, we use zebraﬁsh as a model system to understand the importance of CD146 in vascular\ndevelopment. Endothelial-speciﬁc expression of CD146 was veriﬁed by whole-mount in situ hybridization.\nSuppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly\ndeveloped intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood ﬂow through the\nISV region, despite that ﬂuorescence microangiography showed that the ISVs were present. This ﬁnding\nsuggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally,\nremodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of\nCD146 protein expression resulted in a circulation shunt that caused the circulation to by-pass part of the\ncaudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino\noligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N\nterminus deleted in an in-frame manner. This region, therefore, is likely to contain elements critical for\nCD146 function. This study provides the ﬁrst in vivo functional assessment of CD146 in embryonic\ndevelopment by showing that knockdown of CD146 protein expression severely hinders vascular\ndevelopment in zebraﬁsh. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley-Liss, Inc.",
         "INTRODUCTION\nCD146, also known as MUC18, A32,\nMel-CAM, and S-Endo1 (Lehmann et\nal., 1989; Shih, 1999), is a transmem-\nbrane glycoprotein that belongs to the\nimmunoglobulin superfamily with a\ncharacteristic V-V-C2-C2-C2 domain\n(Williams and Barclay,\nstructure\n1988; Sers et al., 1993; Shih, 1999).\nAlthough it was ﬁrst identiﬁed as a\nhuman melanoma-associated antigen\n(Holzmann et al., 1987; Lehmann et\nal., 1987), this protein is also highly\nexpressed in the endothelium (Bardin\net al., 1996a,b). In fact, it was deter-\nmined that CD146 was the antigen\nrecognized by a highly endothelial-\nspeciﬁc monoclonal antibody called\nP1H12 (Solovey et al., 2001).\nAlthough its cellular function is rel-\natively poorly understood, CD146 is\nbelieved to play a role in cell– cell ad-\nhesion. It has been shown by immuno-\nhistochemical studies that CD146 is\nlocalized at the intercellular junctions\nof conﬂuent endothelial cells in vitro\n(Bardin et al., 1996a, 2001), although\nCD146 does not appear to form part of\nthe tight junctions (Lampugnani and\nDejana, 1997) nor the adherens junc-\ntions (Bardin et al., 2001). In addition,\nthe cell surface expression of CD146\nwas shown to increase by twofold as\nhuman umbilical vein endothelial\ncells (HUVECs) grew from subconﬂu-\nency to complete conﬂuency (Bardin et\nal., 2001). Furthermore, overexpres-\nsion of CD146 in ﬁbroblasts decreased\nparacellular permeability (Bardin et\nal., 2001), whereas treatment of con-\nﬂuent human microvascular endothe-\nlial cells (HMVECs) with monoclonal\nantibody P1H12 increased permeabil-\nity to albumin in vitro (Solovey et al.,\n2001).\nThe nature of CD146 adhesive prop-\nerty has been addressed in vitro\nmainly through cell-aggregation and\nsolid-phase binding assays. Results\nfrom these experiments suggest that\nCD146 confers cell– cell adhesion by\nmeans of interactions with a hetero-\nphilic ligand that has yet to be identi-\nﬁed (Johnson et al., 1997; Shih et al.,\n1997). This interaction is divalent-cat-\nion independent and can be inhibited\nby a recombinant full-length CD146\nprotein and a polyclonal antiserum\nraised against CD146 (Johnson et al.,\n1997; Shih et al., 1997).\nIn addition to its potential role in\ncell– cell adhesion, CD146 has also\nbeen shown to be a signaling molecule\nthat induces several intracellular mo-\nlecular events upon cross-linking by a\nmonoclonal antibody. For example,\ntreatment of HMVEC with the mono-\nclonal antibody P1H12 caused caveo-\nlin to redistribute from the cell periph-\nery to the cytoplasm and nuclear entry\nof the transcription factor NF-B (So-\nlovey et al., 2001). In addition, anti-\nbody-mediated cross-linking of CD146\nin vitro recruited Fyn to the cytoplas-\nmic domain of CD146, triggered a\nphospholipase C – dependent store-\noperated calcium mobilization and in-\nduced phosphorylation of cellular pro-\nteins such as Pyk2, FAK, CAS, and\npaxillin (Anfosso et al., 1998, 2001;\nSolovey et al., 2001). However, CD146\nhas not been shown to interact with\nother cell-surface adhesive molecules\nsuch as VE-cadherin, PECAM-1, or 1\nintegrin (Bardin et al., 2001).\nRecently, a fully humanized CD146\nmonoclonal antibody ABX-MA1 (Mills\net al., 2002) and a mouse monoclonal\nantibody AA98 (Yan et al., 2003) were\ndeveloped. When tested in mouse tu-\nmor models, both antibodies inhibited\ntumor growth and metastasis and re-\nduced microvascular density in tu-\nmors (Mills et al., 2002; Yan et al.,\n2003). Of interest, the antibody AA98\nwas shown to bind only to stimulated\nbut not serum-starved HUVECs in\nvitro (Yan et al., 2003). Most impor-\ntantly, this antibody was shown to\nrecognize an epitope in CD146 that\nwas only present in tumor endothe-\nin endothelial cells\nlium but not\npresent in normal tissues (Yan et al.,\n2003). AA98 monoclonal antibody was\nshown to inhibit proliferation and mi-\ngration of HUVECs in vitro and sig-\nniﬁcantly\nin\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003). Collectively,\nthese studies suggest\nthat CD146\nplays a signiﬁcant role in blood vessel\ndevelopment as well as in solid tumor\ngrowth.\nThe ease of manipulation makes the\nzebraﬁsh system ideal to identify en-\ndothelial speciﬁc genes. Speciﬁcally,\nthe antisense morpholino-oligonucleo-\ntide technology (Heasman et al., 2000;\nNasevicius and Ekker, 2000) provides\na rapid way to suppress protein ex-\npression to address the functionality\nof a speciﬁc gene in a developmental\ncontext. One of our goals is to use this\nmodel to rapidly characterize endo-\nthelial-speciﬁc genes with unknown in\nvivo functions. We chose CD146 as our\nﬁrst target gene, because a CD146 ge-\nnetic knockout mouse is not available\nto date. Here, we report the use of the\nzebraﬁsh system to understand the\npotential role of CD146 in vascular\ndevelopment.\nRESULTS\nCloning of the Zebraﬁsh\nCD146 Gene",
         "A cDNA clone (clone number, 3512;\nGenBank accession no. BG985668) re-\nported in the zebraﬁsh cDNA data-\nbase\n(http://zf.nichd.nih.gov/pubzf/)\nshows a high level of sequence iden-\ntity to mouse and chicken CD146 at\nthe RNA and protein levels, respec-\ntively (Kudoh et al., 2001). Therefore,\nwe performed 5 and 3 rapid ampliﬁ-\ncation of cDNA ends (RACEs) based\non this sequence information to clone\nthe full-length zebraﬁsh CD146 (Fig.\n1).\nThe transcript is approximately 2.3\nkb long, harboring an open reading\nframe that encodes a 617-amino-acid\nprotein. The 5 untranslated region is\nrelatively short (approximately 73 nu-\ncleotides), similar to that of the tran-\nscript of human origin (26 nucleotides;\nSers et al., 1993). The 3 untranslated\nregion is approximately 350 nucleo-\ntides long and contains a canonical\npolyadenylation\nsequence\nAAUAAA just 5 to the poly-A tail\n(Fig. 1). When this mRNA sequence\nwas compared with the zebraﬁsh\ngenomic sequence, a 200-kb genomic\nfragment (ctg24502.1) was identiﬁed\nto contain the ﬁrst 13 predicted exons\nof CD146 (not shown). This genomic\nfragment covers nucleotides 1 to 1685\nof the mRNA. A similar search was\nperformed at the ENSEMBL Web site\n(http://www.ensembl.org/Danio_rerio/\nblastview). The search result indi-\ncates that CD146 is located on chro-\nmosome 19 of the zebraﬁsh genome.\nThe zebraﬁsh CD146 protein shows\napproximately 30% overall sequence\nidentity to that of the human, avian,\nmouse, and rat orthologs (Fig. 2A). Of\ninterest, the transmembrane and the\ncytoplasmic domains are the most con-\nserved regions. When the protein se-\nquence was scanned for protein motifs\n(at http://scansite.mit.edu/; Yaffe et\nal., 2001), ﬁve immunoglobulin do-\nmains (Ig1, K35 to V123; Ig2, C168 to\nV232; Ig3, G273 to S324; Ig4, G359 to\nV408; Ig5, N445 to A507) were identi-\nﬁed in the extracellular region of the\nprotein, consistent with the character-\nistic Ig domain structure present in\nthe subgroup of immunoglobulin su-\nperfamily that CD146 belongs to\n(Johnson et al., 1996). The two puta-\ntive PKC phosphorylation sequences\n(S593GK and S614DK) in human (Alais\net al., 2001) are also conserved in the\nzebraﬁsh\n(S575GK and\nS596GK). The result of a phylogenetic\nanalysis is shown in Figure 2B.\nprotein\nTransient expression of zebraﬁsh\nCD146 tagged with a ﬂuorescent pro-\ntein at the C terminus in 293 cells\nshowed cell surface expression pat-\ntern similar to that of transiently ex-\npressed human CD146/YFP (data not\nshown). In addition, the ﬁrst 11 amino\nacids of zebraﬁsh CD146 directed the\nsecretion of YFP into the culture me-\ndium when fused to the N terminus of\nthe protein and transiently expressed\nin 293 cells, suggesting that this re-\ngion contains a leader-peptide se-\nquence and that Met1 is the authentic\ninitiation methionine\nnot\nshown). We also tested two commer-\ncially available antibodies against hu-\nman CD146 (a polyclonal antibody\nfrom ZYMED and a monoclonal anti-\nbody from CHEMICON) in Western\nblot analysis and determined that the\nantibodies did not cross-react with the\nzebraﬁsh protein overexpressed in\n293 cells (data not shown).\n(data\nExpression of CD146 in\nZebraﬁsh Embryos Is Highly\nEndothelial Speciﬁc\nNext, CD146 expression pattern in\nzebraﬁsh embryo was examined by\nwhole-mount in situ hybridization\nby using a probe that was derived\nfrom the ﬁrst 0.6 kb of the mRNA\n(Fig. 3A). In a 24-hr-old embryo,\nmRNA of CD146 was detected in the\ndorsal aorta (Fig. 3B, red arrow),\ncaudal artery (Fig. 3B, red arrow-\nhead), caudal vein (Fig. 3B, blue ar-\nrowhead), and the sprouting inter-\nsomitic vessels\n(Fig. 3B, green\narrow). Expression in neuronal cells\nwas also detected (Fig. 3B, black ar-\nrow). This expression pattern is es-\nsentially identical to that reported in\nthe zebraﬁsh cDNA database (http://\nzf.nichd.nih.gov/pubzf/), where the\nwhole-mount in situ experiment was\nperformed using a 0.5-kb probe de-\nrived from the 3 end of the mRNA.\nOf interest, only a very low level of\nCD146 mRNA expression was de-\ntected in the posterior cardinal vein\nat this stage (Fig. 3B, blue arrow).\nThe expression speciﬁcity was best\nillustrated by a cross section of the\nembryo (Fig. 3D). Intense staining of\nthe message was detected in the\naorta (Fig. 3D, red arrow); only a\nvery low level of staining was de-\ntected in the posterior cardinal vein\n(Fig. 3D, blue arrow). In contrast,\nCD146 expression in the posterior car-\ndinal vein was detected at day 2 (Fig.\n3E, blue arrow) and day 3 (data not\nshown). This ﬁnding highlights the\ndynamics of the expression pattern of\nthis endothelial marker during devel-\nopment. mRNA staining of CD146 in\nthe head showed clear endothelial ex-\npression pattern (Fig. 3C). Certain\ncranial vessels (e.g., the middle cere-\nbral vein) were stained, while others\n(e.g., the dorsal ciliary and median\npalatocerebral veins) did not appear\nto express CD146. Some other vascu-\nlar structures stained included the\naortic arches and the heart.\nProtein Expression\nKnockdown of CD146\nInhibits Proper Development\nof Intersomitic Vessels\nTo address the functional\nimpor-\ntance of CD146 in embryonic vascu-\nlar development, we performed pro-\ntein expression knockdown exper-\niments by using antisense MOs. The\nﬁrst MO (splice blocker 1) was de-\nsigned to target the seven nucleo-\ntides at the 3 end of exon 10 and the\nﬁrst 18 nucleotides of intron 10 (Fig.\n4A). Microinjection of splice blocker\n1 into the embryos resulted in aber-\nrant splicing. When total RNA har-\nvested from control MO-injected em-\nbryos was used as the template for\nreverse\ntranscriptase-polymerase\nchain reaction (RT-PCR), a speciﬁc\nband of 300 bp was ampliﬁed (Fig.\n5A, lane 1). The identity of this PCR\nproduct was conﬁrmed to be wild-\ntype CD146 by DNA sequencing\n(Fig. 6A). When the same analysis\nwas performed using total RNA har-\nvested from splice blocker 1–injected\nembryos, a PCR product with the\nsize 300 bp was obtained (Fig. 5A,\nlanes 3 and 5). DNA sequencing re-\nvealed that this product was derived\nfrom an aberrantly spliced CD146\nmRNA with nucleotides 1260 to 1282\ndeleted (Fig. 6B). Apparently, with\nA subpopulation of embryos injected\nwith the MO showed different de-\ngrees of curvature at the trunk/tail\nregion. An example is shown in\nFigure 7D. However, there did not\nappear to be a correlation between\nvascular defects and the body curva-\nture.\nReduced Protein Expression\nof CD146 Causes Disruption\nof Caudal Plexus\nReorganization\nAdditional vascular defects were also\ndetected in CD146 morphants. For ex-\nample, the angiogenic remodeling pro-\ncess of the caudal plexus into func-\ntional vascular tubes was complete at\nday 3 in the control MO-injected ﬁsh\n(Movie 2A). Blood could be seen trav-\neling through the entire length of the\nembryo by means of the dorsal aorta,\nthe caudal artery, the caudal vein, and\nthe posterior cardinal vein. However,\nthis vascular reorganization of the\nplexus appeared to be defective in the\nsplice blocker 1–injected embryos\n(Supplementary Movie 2B). The cau-\ndal plexus failed to completely re-\nmodel and organize into functional\ntubes. There was a circulation shunt\nthat caused the circulation to by-pass\nmuch of the caudal artery/vein sys-\ntem. A stagnant pool of blood cells ac-\ncumulated at the tail. Although no cir-\nculation was detected, these trapped\ncells appeared to exhibit slight pulsa-\ntile motion, suggesting that the cau-\ndal plexus might not be developed and\norganized properly for circulation but\nconnected enough to allow hemody-\nnamic forces to reach the tail region.\nCD146 Appears To Be\nImportant in Formation of\nVessels With Proper Lumen\nSize\nTo investigate the integrity of the vas-\nculature, we performed microangiog-\nraphy to detect lumenized vessels.\nEmbryos at 3 dpf were injected with\nred ﬂuorescent microbeads with an\naverage diameter of 0.02 m. The re-\nsults are shown in Figure 8. In control\nMO-injected ﬁsh, vascular develop-\nment as seen by microangiography ap-\npeared to be normal. In contrast, the\noverall vessel integrity of the splice\nthe predicted splice donor site of in-\ntron 10 blocked by the morpholino\noligonucleotide (see Fig. 4A), G1260\nof the mRNA was used as an alter-\nnate, cryptic splice donor site (Fig.\n6B),\naberrantly\nspliced mRNA harboring a prema-\nture stop codon 186 amino acids N\nterminal to the predicted transmem-\nbrane domain (Fig. 1, deleted amino\nacids underlined and the premature\nstop codon in bold and italic).\nan\nTo conﬁrm the effect of splice\nblocker 1 on CD146 mRNA splicing,\nanother set of PCRs was performed in\ntriplicate by using a pair of isoform-\nspeciﬁc primers. This primer set was\ndesigned to detect\nthe full-length,\nwild-type CD146 cDNA speciﬁcally.\nAs shown in Figure 5B, the full-length\nCD146 was only detected in the con-\ntrol MO injected but not the splice\nblocker 1–injected embryos. As con-\ntrol, detection of another endothelial\nspeciﬁc gene ﬂi-1 was also shown.\nTo ascertain the role of CD146 in\nvessel development, blood circulation\nof embryos at 3 days postfertilization\n(dpf) was monitored (Fig. 7, boxed re-\ngions). Control injected ﬁsh showed\nnormal circulation. Blood cells were\nseen traveling through the length of\nthe embryo by means of the artery/\nvein system (Supplementary Movie\n1A, which is available online at www.\ninterscience.wiley.com/jpages/1058-\n8388/suppmat). Importantly, circula-\ntion was also detected throughout the\nintersomitic vessels, indicating that\nthese vessels were functional at this\nstage of development. In contrast, cir-\nculation was abnormal in embryos in-\njected with splice blocker 1. Although\nblood cells were seen traveling in the\nartery and vein, little or no circulation\nwas detected in the intersomitic ves-\nsels (Supplementary Movie 1B). This\nobservation suggests that suppression\nof CD146 protein expression blocked\nthe development of the intersomitic\nvessels. This effect was tabulated, and\nthe results are shown in Table 1. We\ndecided to score the vessel develop-\nment of the embryos by the percent-\nage of functional intersomitic vessels\npresent at 3 dpf. This titration exper-\niment was done multiple times in\nsmall-scale pilot experiments, and a\nsimilar inhibition pattern was consis-\ntently observed. Next, we chose to in-\nject the embryos with either 1.2 pmol\ncontrol MO or 0.8 pmol splice blocker\n1 in three additional independent ex-\nperiments. The results are summa-\nrized in Figure 7C. Splice blocker 1\nclearly had a deleterious effect on\nthe development of intersomitic ves-\nsels. The deleted amino acids, there-\nfore, are critical to CD146 functions.\nBecause the transmembrane and cy-\ntoplasmic domains of CD146 were\ndeleted in the morphants, intracellu-\nlar signaling events triggered by\nCD146 were presumably abolished.\nblocker 1–injected ﬁsh appeared to be\ncompromised. For example, distinct\ncranial vessels were seen in the con-\ntrol-injected ﬁsh (Fig. 8A). In contrast,\nthe cranial vessels of\nthe splice\nblocker 1-injected embryo appeared to\nform less-distinct tubular structures\n(Fig. 8B). In particular, a group of ves-\nsels called the cranial division of the\ninternal carotid was less organized\ncompared with the control\n(arrow-\nheads in Fig. 8A,B). Second, the inter-\nsomitic vessels were narrower com-\npared with the control, even though\nthese vessels were detected by mi-\ncroangiography. A movie clip of the\ncirculation again revealed that the de-\nvelopment of intersomitic vessels was\n(Supplementary\nseverely\nMovie 3). However, these ISVs were\nlumenized as seen by ﬂuorescence mi-\ncroangiography (Fig. 8D). A similar\ndefect was observed in the caudal\nplexus region. When comparing the\nmicroangiogram and the blood circu-\nlation of the splice blocker–injected\nembryos, we observed that blood did\nnot ﬂow through part of the lumenized\ncaudal artery and vein vessels (corre-\nsponded to the region marked be-\ntween the arrowhead and the arrow in\nFig. 8D), resulting in a circulation\nshunt. This ﬁnding suggests that the\nlumen at the site where the circula-\ntion shunt occurred was too narrow\nfor blood cells to pass through. Our\nobservations suggest that CD146 may\nbe important in formation of vessels\nwith proper lumen size.\nElements Critical to CD146\nFunction Are Present at the\nN-Terminus of the Protein\nTo validate our data with splice\nblocker 1, we designed a second an-\ntisense morpholino oligonucleotide\nto target the 3 end of exon 2 and the\npredicted GT splice donor site of in-\ntron 2 (Fig. 4). When this MO was\ndelivered into the embryos by micro-\ninjection,\nit also caused aberrant\nmRNA splicing. RT-PCR analyses\nwere performed with total RNA har-\nvested from control MO- and splice\nblocker 2–injected embryos. The con-\ntrol MO did not affect splicing at the\nexon2/intron2 junction of CD146\npre-mRNA (Fig. 6C). However, se-\nquencing data revealed that splice\nblocker 2 inhibited normal splicing\nbetween exons 2 and 3 (Fig. 6D).\nWith the predicted splice donor site\nof\nthe\nintron 2 blocked, G94 of\nmRNA was used as an alternate,\ncryptic splice donor site. This aber-\nrant splicing event yielded a mutant\nCD146 with an in-frame deletion of\n26 amino acids (V32 to T57) at the N\nterminus (Fig. 1, deleted region in\nbold).\nDeletion of amino acids V32 to T57\nﬁsh, no circulation was seen in the\nintersomitic vessels (Supplementary\nMovie 4B and Table 2), a phenotype\nvery similar to that generated by\nsplice blocker 1–induced CD146 pro-\ntein expression knockdown. There-\nfore, these two MOs appeared to have\nphenocopied each other. Furthermore,\nthis result strongly suggests that the\nregion between V32 and T57 is likely\nto contain elements critical for CD146\nfunction.\nsimilarity\nIn this report, we cloned and charac-\nterized the CD146 ortholog in ze-\nbraﬁsh. The overall amino acid se-\nquence\nthe\nzebraﬁsh and human proteins was ap-\nproximately 30%. Whole-mount\nin\nsitu hybridization results indicate\nthat the CD146 has a dynamic expres-\nsion pattern in the vascular tree dur-\ning development. At 1 dpf, CD146\ntranscript is detected in the dorsal\naorta, caudal artery, and the inter-\nsomitic vessels. In the venous system,\nhowever, CD146 is detected only in\nthe caudal vein but not in the poste-\nrior cardinal vein at this stage (Fig.\n3B). However, CD146 began to ex-\npress in the posterior cardinal vein at\n2 dpf (Fig. 3E). The functional impor-\ntance of the dynamic expression of\nCD146 in the venous system is yet to\nbe understood. CD146 is also ex-\npressed in the heart of developing ze-\nbraﬁsh embryos and may play a role\nin cardiac development. We occasion-\nally observed pericardial edema in the\nmorphants, but there did not seem to\nbe a correlation between this pheno-\ntype and the vascular defects de-\nscribed here. Future studies will in-\nclude the analysis of the relationship\nand\nbetween heart\nCD146 functions. In this report, we\nfocused on the vascular functions of\nCD146 primarily because of the ap-\nparent roles CD146 plays in tumor an-\ngiogenesis (Mills et al., 2002; Yan et\nal., 2003).\ndevelopment\nWhen the protein expression of\nCD146 was knocked down, vascular\ndevelopment in the zebraﬁsh embryo\nwas severely affected. One most prom-\ninent defect was the lack of blood cir-\nculating in the intersomitic vessels by\n3 dpf (Supplementary Movies 1A and\nB). This lack of circulation was not\nof CD146 also affected the develop-\nment of intersomitic vessels in ze-\nbraﬁsh embryos. At day three, circu-\nlation was normal in embryos injected\nwith the control MO (Supplementary\nMovie 4A). In splice blocker 2–injected\ndue to the absence of intersomitic ves-\nsels per se, because these vessel struc-\ntures were clearly detected by ﬂuores-\ncence microangiography,\nalthough\nmany of them appeared to have un-\neven thickness and stenosis was evi-\ndent even when observed under a ﬂu-\n(Fig. 8B,D).\norescence microscope\nAlthough the lumen size of the inter-\nsomitic vessels in zebraﬁsh has not\nbeen reported in literature, the diam-\neters of artery and vein in zebraﬁsh\nembryos have been measured to be\n9.0 m,\n12.8\nrespectively\n(Fritsche et al., 2000). The use of ﬂu-\norescent microbeads with an average\ndiameter of 0.02 m allowed us to de-\ntect vessels with a diameter 400\ntimes smaller than that of the vein.\nBased on our results, we speculate\nthat, in the absence of CD146, the\nintersomitic vessels cannot develop\ninto functional tubes with a lumen\nopening large enough for blood cells to\npass through.\nand\nThe development of the intersomitic\nvessels is a highly organized and cho-\nreographed process. It was reported\nrecently that formation of the inter-\nsomitic vessels involves three differ-\nent endothelial cell types sprouting\nfrom the dorsal aorta (Childs et al.,\n2002). The ﬁrst two form a T-like\nstructure at the dorsal side of the dor-\nsal aorta and the ventral side of the\ndorsal longitudinal anastomotic ves-\nsel. The third one connects these two\nT structures. Because the intersomitic\nvessels were detected in the mor-\nphants with CD146 protein expres-\nsion suppressed, this ﬁnding suggests\nthat CD146 may play a role in the\nstep(s) after the basic architecture of\nthe intersomitic vessels has been laid\nout by these three types of endothelial\ncells.\nData from our morpholino experi-\nments indicate that CD146 may play\nan important role in the remodeling of\nthe caudal vein plexus into functional\nvascular tubes as well. In the CD146\nmorphants, circulation ﬂowed from\nthe dorsal aorta into the caudal artery\nand returned prematurely into cardi-\nnal vein, creating a circulation shunt\nin the caudal artery/vein system (Sup-\nplementary Movie 2B). This circula-\ntion shunt appears to be distinct from\nthat observed in the zebraﬁsh mutant\nmindbomb (mib) defective in Notch\nsignaling (Lawson et al., 2001). In the\nmib mutant, the circulation shunt oc-\ncurred in the trunk region where cir-\nculation ﬂowed directly from the dor-\nsal aorta into the posterior cardinal\nvein without entering the caudal ar-\ntery/vein system. This circulation by-\npass was speculated to arise from dis-\norganization of the boundary between\nthe dorsal aorta and the posterior car-\ndinal vein based on histological data\n(Lawson et al., 2001). In the CD146\nmorphants described here, blood cir-\nculation entered the caudal artery but\nreturned into the venous system pre-\nmaturely into the caudal vein. How-\never, ﬂuorescence microangiography\nrevealed that the caudal plexus was\nlumenized (Fig. 8D, see region be-\ntween the arrow and the arrowhead).\nTherefore, the caudal shunt present in\nthe CD146 morphants may be a result\nof restricted lumen diameter, a struc-\ntural defect similar to that observed in\nthe intersomitic vessels.\nOur observation that CD146 is im-\nportant in vascular development is in\nagreement with several studies re-\nported in the literature. For example,\nit was shown the CD146 expression\nwas up-regulated in human microvas-\ncular endothelial cells that had under-\ngone tubular morphological changes\non matrigel (Glienke et al., 2000). In\nadditional, the ability of endothelial\ncells to form tubes on matrigel can be\nblocked by a CD146 neutralizing anti-\nbody (Mills et al., 2002). Functional\nsuppression of CD146 by an antibody\ncan also inhibit endothelial cell migra-\ntion in vitro and neovascularization in\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003).\nWe have also identiﬁed V32 to T57\nof CD146 to be an important sequence\nelement in its protein function. We\nused an antisense morpholino oligo-\nnucleotide targeting the predicted\nexon 2/intron 2 junction of zebraﬁsh\nCD146 pre-mRNA to create a mutant\nprotein with those 26 amino acids de-\nleted in an in-frame manner. This\ntruncated protein appeared to be non-\nfunctional. This region may contain a\nrecognition epitope for CD146 func-\ntions. Alternatively, the deletion may\nhave affected proper protein folding,\nas the region deleted contains a con-\nserved cysteine residue in the ﬁrst im-\nmunoglobulin domain.\nVascular development requires ex-\ntensive dynamic changes of endothe-\nlial morphology. These dramatic mor-\nphological\nchanges were recorded\nrecently by time-lapse microscopy in a\ndeveloping zebraﬁsh embryo in real-\ntime (Lawson and Weinstein, 2002).\nEndothelial cells were seen to extend\nand retract long ﬁlopodial processes\nduring path ﬁnding. Communication\nappears to occur between these acti-\nvated endothelial cells through ﬁlopo-\ndial contacts, resulting in proper vas-\ncular remodeling and regression to\nensure the development of functional\nlumens\n(Lawson and Weinstein,\n2002). The exact molecular basis of\nthese cellular behaviors is certainly\ncomplex, but highly orchestrated and\ndynamic changes of the cytoskeleton\norganization are probably involved\n(Davis et al., 2002). For example,\nCdc42, Rac1, and integrins are some\nof the signaling molecules intimately\nlinked to cytoskeleton functions that\nhave been shown to be critical in en-\ndothelial tube and lumen formation in\nvitro (Davis and Camarillo, 1996; Bay-\neral integrins that are known to be\nexpressed in endothelial cells (Alais et\nal., 2001; Hodivala-Dilke et al., 2003).\nAll these cellular events appear to be\nrelated to the cytoskeleton. Perhaps,\nCD146 mediates the formation of\nproper vessels through regulating the\ndynamic changes in the organization\nof the cytoskeleton during develop-\nment. Further studies that are de-\nsigned to dissect the intracellular sig-\nnaling events important in CD146\nfunctions are currently under way.\nEXPERIMENTAL\nPROCEDURES\nZebraﬁsh Care\nZebraﬁsh (Danio rerio) was raised and\nmaintained as described (Westerﬁeld,\n2000).\n5 and 3 RACES\nTotal RNA was collected from ze-\nbraﬁsh embryos at 48 hours postfertil-\nization (hpf) by using either the\nRNAaqueous-4PCR kit from Ambion\n(Austin, TX) or the RNeasy Mini kit\nfrom Qiagen (Valencia, CA). The 5\nand 3 RACEs were carried out by us-\ning the GeneRacer Kit from Invitro-\ngen (Carlsbad, CA) according to the\nmanufacturer’s protocol. The kit was\nchosen because the transcriptional\nstart site of the mRNA can be ob-\ntained directly. Brieﬂy, 5 RACE was\nperformed by using 5-GGCACAGG-\nGAGTGTCTTCTTGCCTTATA-3\nas\nthe gene-speciﬁc primer for ﬁrst-\nstrand cDNA synthesis. The cDNA\nwas ampliﬁed by nested PCR using\nthe GC-Advantage DNA polymerase\nfrom Clontech (Palo Alto, CA). The\nprimers for the ﬁrst round of PCR\nwere the GeneRacer 5 Primer (pro-\nvided in the GeneRacer Kit) and the\ngene-speciﬁc primer. For the second\nround of ampliﬁcation, the GeneRacer\n5 Nested Primer (provided in the\nGeneRacer Kit) and a primer with the\nsequence 5-GCAGCTTTCCCTTCTT-\nGTAGAGGTAAT-3 were used. The\nPCR products were cloned into the\nTOPO cloning kit from Invitrogen.\nProtein sequence alignment was per-\nhttp://prodes.toulouse.\nformed\ninra.fr/multalin/multalin.html\n(Cor-\npet, 1988). The mRNA sequence has\nbeen deposited in GenBank (accession\nno. AY497061).\nless et al., 2000; Bayless and Davis,\n2002; Davis and Bayless, 2003).\nAlthough the molecular mechanism\nby which CD146 affects proper lumen\nformation observed in this report is at\nthis point unclear, it is tempting to\nspeculate that CD146 may achieve\nsuch function by regulating the cy-\ntoskeleton. It is known that antibody-\nmediated cross-linking of CD146 in\nvitro initiates a cascade of intracellu-\nlar signaling events that\nincludes\nphosphorylation of focal adhesion ki-\nnase, paxillin, and p130CAS (Anfosso\net al., 1998, 2001). Additionally,\nCD146 has been shown to interact\nwith the actin cytoskeleton in HUVEC\n(Bardin et al., 2001). Moreover, the\nCD146 ortholog in chicken has been\nshown to modulate expression of sev-\nIn Situ Hybridization\nNucleotides spanning the 5 untrans-\nlated region and the ﬁrst 592 bases of\nthe open reading frame of zebraﬁsh\nCD146 were ampliﬁed and cloned into\na vector derived from pCR-BluntII-\nTOPO (Invitrogen). To\ngenerate\ndigoxigenin (DIG)-labeled antisense\nprobes, the DNA plasmid was linear-\nized with EcoRI, followed by tran-\nscription using SP6 polymerase and\nthe 10 DIG RNA labeling mix from\nRoche (Indianapolis, IN). In situ hy-\nbridization was then performed ac-\ncording to a published procedure\n(Broadbent and Read, 1999; Read,\n2000).\nMicroinjection and\nFluorescence\nMicroangiography\nTwo antisense (splice blocker 1. 5-\nCAGTTGTAGTGACCCACCTCGCA-\nCG-3; splice blocker 2. 5-GTCTTTT-\nTCTCACCGTAAACCAGAT-3) and a\ncontrol (5-GACTCTATATTGCCTAC-\nCTTTACAG-3) morpholino oligo-\nnucleotides were purchased from\nGene-Tools (Philomath, OR). The\nmorpholino oligonucleotides were in-\njected into the yolk sac of zebraﬁsh\nembryos between one- and four-cell\nstages. Embryos were raised in egg\nwater in the presence of 0.003% (w/v)\n1-phenyl-2-thiourea (Sigma, St. Louis,\nMO) to prevent pigmentation (Wester-\nﬁeld, 2000). To verify successful block-\nage of CD146 pre-mRNA splicing by\nthe morpholino oligonucleotides, total\nRNA was collected from the zebraﬁsh\nembryos at 2 dpf by using the RNeasy\nMini kit from Qiagen. Reverse tran-\nscription was carried out by using ei-\nther Thermoscript or Superscript III\nfrom Invitrogen with the gene-speciﬁc\nprimer (5-GGCACAGGGAGTGTCT-\nTCTTGCCTTATA-3). PCR was then\nperformed by using GC-Advantage\nDNA polymerase (Clontech) and the\nfollowing ampliﬁcation primers: 5-\nGATATCACCTTGACTTGCAACG-\nCATTGTCT-3 and 5-GATTGGG-\nTAGCCTTGTGCATAGCAGGTCAG-3\nfor\nsplice blocker 1 injection;\n5-ATGACCTACACCGCACTGCTGC-\nTCGCGGGA-3 and 5-AGGATC-\nGAGAGGGAAGCTGCCCTTGTAC-\nCA-3 for splice blocker 2 injection. A\npair of isoform speciﬁc primers that am-\npliﬁed only the wild-type CD146 cDNA\n(nucleotides 126 to 1287) has the follow-\ning sequences: 5-TACTTACACTCT-\nGACGGAAGAAGCCAAGAT-3 and\n5-CTTTCCTCGCACGTTTACTTGCAC-\nTGA-3. Primers used to ampliﬁed Fli-1\nwere 5-GAAAATGGACGGAACTAT-\nTAAGGAGGCGCT-3 and 5-TCTTC-\nGAGTGCAGTTCAAGTTTTGGCATT-\nTAAGGGT-3.\nBlood circulation in anesthetized\nﬁsh embryos was recorded as time-\nlapse images using an inverted phase\ncontrast microscope (Nikon Corpora-\ntion, Tokyo, Japan), a Spot RT cam-\nera, and the Advanced SPOT software\n(Diagnostic Instruments, Inc., Ster-\nling Heights, MI). The images were\nconverted into QuickTime movies us-\ning a program called ImageJ.\nmicroangiography\nwas performed by using the red ﬂuo-\ncarboxylate-modiﬁed Fluo-\nrescent\nSpheres (0.02 m)\nfrom Molecular\nProbes (Eugene, OR) as described\n(Weinstein et al., 1995). Fluorescence\nmicroangiograms were recorded with\nthe Nikon ﬂuorescence microscope us-\ning a ﬁlter set (41035:ex. 546; em. 605)\nfrom Chroma (Rockingham, VT).",
         "",
         "Total RNA was collected from ze-\nbraﬁsh embryos at 48 hours postfertil-\nization (hpf) by using either the\nRNAaqueous-4PCR kit from Ambion\n(Austin, TX) or the RNeasy Mini kit\nfrom Qiagen (Valencia, CA). The 5\nand 3 RACEs were carried out by us-\ning the GeneRacer Kit from Invitro-\ngen (Carlsbad, CA) according to the\nmanufacturer’s protocol. The kit was\nchosen because the transcriptional\nstart site of the mRNA can be ob-\ntained directly. Brieﬂy, 5 RACE was\nperformed by using 5-GGCACAGG-\nGAGTGTCTTCTTGCCTTATA-3\nas\nthe gene-speciﬁc primer for ﬁrst-\nstrand cDNA synthesis. The cDNA\nwas ampliﬁed by nested PCR using\nthe GC-Advantage DNA polymerase\nfrom Clontech (Palo Alto, CA). The\nprimers for the ﬁrst round of PCR\nwere the GeneRacer 5 Primer (pro-\nvided in the GeneRacer Kit) and the\ngene-speciﬁc primer. For the second\nround of ampliﬁcation, the GeneRacer\n5 Nested Primer (provided in the\nGeneRacer Kit) and a primer with the\nsequence 5-GCAGCTTTCCCTTCTT-\nGTAGAGGTAAT-3 were used. The\nPCR products were cloned into the\nTOPO cloning kit from Invitrogen.\nProtein sequence alignment was per-\nhttp://prodes.toulouse.\nformed\ninra.fr/multalin/multalin.html\n(Cor-\npet, 1988). The mRNA sequence has\nbeen deposited in GenBank (accession\nno. AY497061).\nless et al., 2000; Bayless and Davis,\n2002; Davis and Bayless, 2003).\nAlthough the molecular mechanism\nby which CD146 affects proper lumen\nformation observed in this report is at\nthis point unclear, it is tempting to\nspeculate that CD146 may achieve\nsuch function by regulating the cy-\ntoskeleton. It is known that antibody-\nmediated cross-linking of CD146 in\nvitro initiates a cascade of intracellu-\nlar signaling events that\nincludes\nphosphorylation of focal adhesion ki-\nnase, paxillin, and p130CAS (Anfosso\net al., 1998, 2001). Additionally,\nCD146 has been shown to interact\nwith the actin cytoskeleton in HUVEC\n(Bardin et al., 2001). Moreover, the\nCD146 ortholog in chicken has been\nshown to modulate expression of sev-\nIn Situ Hybridization\nNucleotides spanning the 5 untrans-\nlated region and the ﬁrst 592 bases of\nthe open reading frame of zebraﬁsh\nCD146 were ampliﬁed and cloned into\na vector derived from pCR-BluntII-\nTOPO (Invitrogen). To\ngenerate\ndigoxigenin (DIG)-labeled antisense\nprobes, the DNA plasmid was linear-\nized with EcoRI, followed by tran-\nscription using SP6 polymerase and\nthe 10 DIG RNA labeling mix from\nRoche (Indianapolis, IN). In situ hy-\nbridization was then performed ac-\ncording to a published procedure\n(Broadbent and Read, 1999; Read,\n2000).\nMicroinjection and\nFluorescence\nMicroangiography\nTwo antisense (splice blocker 1. 5-\nCAGTTGTAGTGACCCACCTCGCA-\nCG-3; splice blocker 2. 5-GTCTTTT-\nTCTCACCGTAAACCAGAT-3) and a\ncontrol (5-GACTCTATATTGCCTAC-\nCTTTACAG-3) morpholino oligo-\nnucleotides were purchased from\nGene-Tools (Philomath, OR). The\nmorpholino oligonucleotides were in-\njected into the yolk sac of zebraﬁsh\nembryos between one- and four-cell\nstages. Embryos were raised in egg\nwater in the presence of 0.003% (w/v)\n1-phenyl-2-thiourea (Sigma, St. Louis,\nMO) to prevent pigmentation (Wester-\nﬁeld, 2000). To verify successful block-\nage of CD146 pre-mRNA splicing by\nthe morpholino oligonucleotides, total\nRNA was collected from the zebraﬁsh\nembryos at 2 dpf by using the RNeasy\nMini kit from Qiagen. Reverse tran-\nscription was carried out by using ei-\nther Thermoscript or Superscript III\nfrom Invitrogen with the gene-speciﬁc\nprimer (5-GGCACAGGGAGTGTCT-\nTCTTGCCTTATA-3). PCR was then\nperformed by using GC-Advantage\nDNA polymerase (Clontech) and the\nfollowing ampliﬁcation primers: 5-\nGATATCACCTTGACTTGCAACG-\nCATTGTCT-3 and 5-GATTGGG-\nTAGCCTTGTGCATAGCAGGTCAG-3\nfor\nsplice blocker 1 injection;\n5-ATGACCTACACCGCACTGCTGC-\nTCGCGGGA-3 and 5-AGGATC-\nGAGAGGGAAGCTGCCCTTGTAC-\nCA-3 for splice blocker 2 injection. A\npair of isoform speciﬁc primers that am-\npliﬁed only the wild-type CD146 cDNA\n(nucleotides 126 to 1287) has the follow-\ning sequences: 5-TACTTACACTCT-\nGACGGAAGAAGCCAAGAT-3 and\n5-CTTTCCTCGCACGTTTACTTGCAC-\nTGA-3. Primers used to ampliﬁed Fli-1\nwere 5-GAAAATGGACGGAACTAT-\nTAAGGAGGCGCT-3 and 5-TCTTC-\nGAGTGCAGTTCAAGTTTTGGCATT-\nTAAGGGT-3.\nBlood circulation in anesthetized\nﬁsh embryos was recorded as time-\nlapse images using an inverted phase\ncontrast microscope (Nikon Corpora-\ntion, Tokyo, Japan), a Spot RT cam-\nera, and the Advanced SPOT software\n(Diagnostic Instruments, Inc., Ster-\nling Heights, MI). The images were\nconverted into QuickTime movies us-\ning a program called ImageJ.\nmicroangiography\nwas performed by using the red ﬂuo-\ncarboxylate-modiﬁed Fluo-\nrescent\nSpheres (0.02 m)\nfrom Molecular\nProbes (Eugene, OR) as described\n(Weinstein et al., 1995). Fluorescence\nmicroangiograms were recorded with\nthe Nikon ﬂuorescence microscope us-\ning a ﬁlter set (41035:ex. 546; em. 605)\nfrom Chroma (Rockingham, VT).",
         "In this report, we use zebraﬁsh as a model system to understand the importance of CD146 in vascular\ndevelopment. Endothelial-speciﬁc expression of CD146 was veriﬁed by whole-mount in situ hybridization.\nSuppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly\ndeveloped intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood ﬂow through the\nISV region, despite that ﬂuorescence microangiography showed that the ISVs were present. This ﬁnding\nsuggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally,\nremodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of\nCD146 protein expression resulted in a circulation shunt that caused the circulation to by-pass part of the\ncaudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino\noligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N\nterminus deleted in an in-frame manner. This region, therefore, is likely to contain elements critical for\nCD146 function. This study provides the ﬁrst in vivo functional assessment of CD146 in embryonic\ndevelopment by showing that knockdown of CD146 protein expression severely hinders vascular\ndevelopment in zebraﬁsh. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley-Liss, Inc.\nINTRODUCTION\nCD146, also known as MUC18, A32,\nMel-CAM, and S-Endo1 (Lehmann et\nal., 1989; Shih, 1999), is a transmem-\nbrane glycoprotein that belongs to the\nimmunoglobulin superfamily with a\ncharacteristic V-V-C2-C2-C2 domain\n(Williams and Barclay,\nstructure\n1988; Sers et al., 1993; Shih, 1999).\nAlthough it was ﬁrst identiﬁed as a\nhuman melanoma-associated antigen\n(Holzmann et al., 1987; Lehmann et\nal., 1987), this protein is also highly\nexpressed in the endothelium (Bardin\net al., 1996a,b). In fact, it was deter-\nmined that CD146 was the antigen\nrecognized by a highly endothelial-\nspeciﬁc monoclonal antibody called\nP1H12 (Solovey et al., 2001).\nAlthough its cellular function is rel-\natively poorly understood, CD146 is\nbelieved to play a role in cell– cell ad-\nhesion. It has been shown by immuno-\nhistochemical studies that CD146 is\nlocalized at the intercellular junctions\nof conﬂuent endothelial cells in vitro\n(Bardin et al., 1996a, 2001), although\nCD146 does not appear to form part of\nthe tight junctions (Lampugnani and\nDejana, 1997) nor the adherens junc-\ntions (Bardin et al., 2001). In addition,\nthe cell surface expression of CD146\nwas shown to increase by twofold as\nhuman umbilical vein endothelial\ncells (HUVECs) grew from subconﬂu-\nency to complete conﬂuency (Bardin et\nal., 2001). Furthermore, overexpres-\nsion of CD146 in ﬁbroblasts decreased\nparacellular permeability (Bardin et\nal., 2001), whereas treatment of con-\nﬂuent human microvascular endothe-\nlial cells (HMVECs) with monoclonal\nantibody P1H12 increased permeabil-\nity to albumin in vitro (Solovey et al.,\n2001).\nThe nature of CD146 adhesive prop-\nerty has been addressed in vitro\nmainly through cell-aggregation and\nsolid-phase binding assays. Results\nfrom these experiments suggest that\nCD146 confers cell– cell adhesion by\nmeans of interactions with a hetero-\nphilic ligand that has yet to be identi-\nﬁed (Johnson et al., 1997; Shih et al.,\n1997). This interaction is divalent-cat-\nion independent and can be inhibited\nby a recombinant full-length CD146\nprotein and a polyclonal antiserum\nraised against CD146 (Johnson et al.,\n1997; Shih et al., 1997).\nIn addition to its potential role in\ncell– cell adhesion, CD146 has also\nbeen shown to be a signaling molecule\nthat induces several intracellular mo-\nlecular events upon cross-linking by a\nmonoclonal antibody. For example,\ntreatment of HMVEC with the mono-\nclonal antibody P1H12 caused caveo-\nlin to redistribute from the cell periph-\nery to the cytoplasm and nuclear entry\nof the transcription factor NF-B (So-\nlovey et al., 2001). In addition, anti-\nbody-mediated cross-linking of CD146\nin vitro recruited Fyn to the cytoplas-\nmic domain of CD146, triggered a\nphospholipase C – dependent store-\noperated calcium mobilization and in-\nduced phosphorylation of cellular pro-\nteins such as Pyk2, FAK, CAS, and\npaxillin (Anfosso et al., 1998, 2001;\nSolovey et al., 2001). However, CD146\nhas not been shown to interact with\nother cell-surface adhesive molecules\nsuch as VE-cadherin, PECAM-1, or 1\nintegrin (Bardin et al., 2001).\nRecently, a fully humanized CD146\nmonoclonal antibody ABX-MA1 (Mills\net al., 2002) and a mouse monoclonal\nantibody AA98 (Yan et al., 2003) were\ndeveloped. When tested in mouse tu-\nmor models, both antibodies inhibited\ntumor growth and metastasis and re-\nduced microvascular density in tu-\nmors (Mills et al., 2002; Yan et al.,\n2003). Of interest, the antibody AA98\nwas shown to bind only to stimulated\nbut not serum-starved HUVECs in\nvitro (Yan et al., 2003). Most impor-\ntantly, this antibody was shown to\nrecognize an epitope in CD146 that\nwas only present in tumor endothe-\nin endothelial cells\nlium but not\npresent in normal tissues (Yan et al.,\n2003). AA98 monoclonal antibody was\nshown to inhibit proliferation and mi-\ngration of HUVECs in vitro and sig-\nniﬁcantly\nin\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003). Collectively,\nthese studies suggest\nthat CD146\nplays a signiﬁcant role in blood vessel\ndevelopment as well as in solid tumor\ngrowth.\nThe ease of manipulation makes the\nzebraﬁsh system ideal to identify en-\ndothelial speciﬁc genes. Speciﬁcally,\nthe antisense morpholino-oligonucleo-\ntide technology (Heasman et al., 2000;\nNasevicius and Ekker, 2000) provides\na rapid way to suppress protein ex-\npression to address the functionality\nof a speciﬁc gene in a developmental\ncontext. One of our goals is to use this\nmodel to rapidly characterize endo-\nthelial-speciﬁc genes with unknown in\nvivo functions. We chose CD146 as our\nﬁrst target gene, because a CD146 ge-\nnetic knockout mouse is not available\nto date. Here, we report the use of the\nzebraﬁsh system to understand the\npotential role of CD146 in vascular\ndevelopment.\nRESULTS\nCloning of the Zebraﬁsh\nCD146 Gene\nA cDNA clone (clone number, 3512;\nGenBank accession no. BG985668) re-\nported in the zebraﬁsh cDNA data-\nbase\n(http://zf.nichd.nih.gov/pubzf/)\nshows a high level of sequence iden-\ntity to mouse and chicken CD146 at\nthe RNA and protein levels, respec-\ntively (Kudoh et al., 2001). Therefore,\nwe performed 5 and 3 rapid ampliﬁ-\ncation of cDNA ends (RACEs) based\non this sequence information to clone\nthe full-length zebraﬁsh CD146 (Fig.\n1).\nThe transcript is approximately 2.3\nkb long, harboring an open reading\nframe that encodes a 617-amino-acid\nprotein. The 5 untranslated region is\nrelatively short (approximately 73 nu-\ncleotides), similar to that of the tran-\nscript of human origin (26 nucleotides;\nSers et al., 1993). The 3 untranslated\nregion is approximately 350 nucleo-\ntides long and contains a canonical\npolyadenylation\nsequence\nAAUAAA just 5 to the poly-A tail\n(Fig. 1). When this mRNA sequence\nwas compared with the zebraﬁsh\ngenomic sequence, a 200-kb genomic\nfragment (ctg24502.1) was identiﬁed\nto contain the ﬁrst 13 predicted exons\nof CD146 (not shown). This genomic\nfragment covers nucleotides 1 to 1685\nof the mRNA. A similar search was\nperformed at the ENSEMBL Web site\n(http://www.ensembl.org/Danio_rerio/\nblastview). The search result indi-\ncates that CD146 is located on chro-\nmosome 19 of the zebraﬁsh genome.\nThe zebraﬁsh CD146 protein shows\napproximately 30% overall sequence\nidentity to that of the human, avian,\nmouse, and rat orthologs (Fig. 2A). Of\ninterest, the transmembrane and the\ncytoplasmic domains are the most con-\nserved regions. When the protein se-\nquence was scanned for protein motifs\n(at http://scansite.mit.edu/; Yaffe et\nal., 2001), ﬁve immunoglobulin do-\nmains (Ig1, K35 to V123; Ig2, C168 to\nV232; Ig3, G273 to S324; Ig4, G359 to\nV408; Ig5, N445 to A507) were identi-\nﬁed in the extracellular region of the\nprotein, consistent with the character-\nistic Ig domain structure present in\nthe subgroup of immunoglobulin su-\nperfamily that CD146 belongs to\n(Johnson et al., 1996). The two puta-\ntive PKC phosphorylation sequences\n(S593GK and S614DK) in human (Alais\net al., 2001) are also conserved in the\nzebraﬁsh\n(S575GK and\nS596GK). The result of a phylogenetic\nanalysis is shown in Figure 2B.\nprotein\nTransient expression of zebraﬁsh\nCD146 tagged with a ﬂuorescent pro-\ntein at the C terminus in 293 cells\nshowed cell surface expression pat-\ntern similar to that of transiently ex-\npressed human CD146/YFP (data not\nshown). In addition, the ﬁrst 11 amino\nacids of zebraﬁsh CD146 directed the\nsecretion of YFP into the culture me-\ndium when fused to the N terminus of\nthe protein and transiently expressed\nin 293 cells, suggesting that this re-\ngion contains a leader-peptide se-\nquence and that Met1 is the authentic\ninitiation methionine\nnot\nshown). We also tested two commer-\ncially available antibodies against hu-\nman CD146 (a polyclonal antibody\nfrom ZYMED and a monoclonal anti-\nbody from CHEMICON) in Western\nblot analysis and determined that the\nantibodies did not cross-react with the\nzebraﬁsh protein overexpressed in\n293 cells (data not shown).\n(data\nExpression of CD146 in\nZebraﬁsh Embryos Is Highly\nEndothelial Speciﬁc\nNext, CD146 expression pattern in\nzebraﬁsh embryo was examined by\nwhole-mount in situ hybridization\nby using a probe that was derived\nfrom the ﬁrst 0.6 kb of the mRNA\n(Fig. 3A). In a 24-hr-old embryo,\nmRNA of CD146 was detected in the\ndorsal aorta (Fig. 3B, red arrow),\ncaudal artery (Fig. 3B, red arrow-\nhead), caudal vein (Fig. 3B, blue ar-\nrowhead), and the sprouting inter-\nsomitic vessels\n(Fig. 3B, green\narrow). Expression in neuronal cells\nwas also detected (Fig. 3B, black ar-\nrow). This expression pattern is es-\nsentially identical to that reported in\nthe zebraﬁsh cDNA database (http://\nzf.nichd.nih.gov/pubzf/), where the\nwhole-mount in situ experiment was\nperformed using a 0.5-kb probe de-\nrived from the 3 end of the mRNA.\nOf interest, only a very low level of\nCD146 mRNA expression was de-\ntected in the posterior cardinal vein\nat this stage (Fig. 3B, blue arrow).\nThe expression speciﬁcity was best\nillustrated by a cross section of the\nembryo (Fig. 3D). Intense staining of\nthe message was detected in the\naorta (Fig. 3D, red arrow); only a\nvery low level of staining was de-\ntected in the posterior cardinal vein\n(Fig. 3D, blue arrow). In contrast,\nCD146 expression in the posterior car-\ndinal vein was detected at day 2 (Fig.\n3E, blue arrow) and day 3 (data not\nshown). This ﬁnding highlights the\ndynamics of the expression pattern of\nthis endothelial marker during devel-\nopment. mRNA staining of CD146 in\nthe head showed clear endothelial ex-\npression pattern (Fig. 3C). Certain\ncranial vessels (e.g., the middle cere-\nbral vein) were stained, while others\n(e.g., the dorsal ciliary and median\npalatocerebral veins) did not appear\nto express CD146. Some other vascu-\nlar structures stained included the\naortic arches and the heart.\nProtein Expression\nKnockdown of CD146\nInhibits Proper Development\nof Intersomitic Vessels\nTo address the functional\nimpor-\ntance of CD146 in embryonic vascu-\nlar development, we performed pro-\ntein expression knockdown exper-\niments by using antisense MOs. The\nﬁrst MO (splice blocker 1) was de-\nsigned to target the seven nucleo-\ntides at the 3 end of exon 10 and the\nﬁrst 18 nucleotides of intron 10 (Fig.\n4A). Microinjection of splice blocker\n1 into the embryos resulted in aber-\nrant splicing. When total RNA har-\nvested from control MO-injected em-\nbryos was used as the template for\nreverse\ntranscriptase-polymerase\nchain reaction (RT-PCR), a speciﬁc\nband of 300 bp was ampliﬁed (Fig.\n5A, lane 1). The identity of this PCR\nproduct was conﬁrmed to be wild-\ntype CD146 by DNA sequencing\n(Fig. 6A). When the same analysis\nwas performed using total RNA har-\nvested from splice blocker 1–injected\nembryos, a PCR product with the\nsize 300 bp was obtained (Fig. 5A,\nlanes 3 and 5). DNA sequencing re-\nvealed that this product was derived\nfrom an aberrantly spliced CD146\nmRNA with nucleotides 1260 to 1282\ndeleted (Fig. 6B). Apparently, with\nA subpopulation of embryos injected\nwith the MO showed different de-\ngrees of curvature at the trunk/tail\nregion. An example is shown in\nFigure 7D. However, there did not\nappear to be a correlation between\nvascular defects and the body curva-\nture.\nReduced Protein Expression\nof CD146 Causes Disruption\nof Caudal Plexus\nReorganization\nAdditional vascular defects were also\ndetected in CD146 morphants. For ex-\nample, the angiogenic remodeling pro-\ncess of the caudal plexus into func-\ntional vascular tubes was complete at\nday 3 in the control MO-injected ﬁsh\n(Movie 2A). Blood could be seen trav-\neling through the entire length of the\nembryo by means of the dorsal aorta,\nthe caudal artery, the caudal vein, and\nthe posterior cardinal vein. However,\nthis vascular reorganization of the\nplexus appeared to be defective in the\nsplice blocker 1–injected embryos\n(Supplementary Movie 2B). The cau-\ndal plexus failed to completely re-\nmodel and organize into functional\ntubes. There was a circulation shunt\nthat caused the circulation to by-pass\nmuch of the caudal artery/vein sys-\ntem. A stagnant pool of blood cells ac-\ncumulated at the tail. Although no cir-\nculation was detected, these trapped\ncells appeared to exhibit slight pulsa-\ntile motion, suggesting that the cau-\ndal plexus might not be developed and\norganized properly for circulation but\nconnected enough to allow hemody-\nnamic forces to reach the tail region.\nCD146 Appears To Be\nImportant in Formation of\nVessels With Proper Lumen\nSize\nTo investigate the integrity of the vas-\nculature, we performed microangiog-\nraphy to detect lumenized vessels.\nEmbryos at 3 dpf were injected with\nred ﬂuorescent microbeads with an\naverage diameter of 0.02 m. The re-\nsults are shown in Figure 8. In control\nMO-injected ﬁsh, vascular develop-\nment as seen by microangiography ap-\npeared to be normal. In contrast, the\noverall vessel integrity of the splice\nthe predicted splice donor site of in-\ntron 10 blocked by the morpholino\noligonucleotide (see Fig. 4A), G1260\nof the mRNA was used as an alter-\nnate, cryptic splice donor site (Fig.\n6B),\naberrantly\nspliced mRNA harboring a prema-\nture stop codon 186 amino acids N\nterminal to the predicted transmem-\nbrane domain (Fig. 1, deleted amino\nacids underlined and the premature\nstop codon in bold and italic).\nan\nTo conﬁrm the effect of splice\nblocker 1 on CD146 mRNA splicing,\nanother set of PCRs was performed in\ntriplicate by using a pair of isoform-\nspeciﬁc primers. This primer set was\ndesigned to detect\nthe full-length,\nwild-type CD146 cDNA speciﬁcally.\nAs shown in Figure 5B, the full-length\nCD146 was only detected in the con-\ntrol MO injected but not the splice\nblocker 1–injected embryos. As con-\ntrol, detection of another endothelial\nspeciﬁc gene ﬂi-1 was also shown.\nTo ascertain the role of CD146 in\nvessel development, blood circulation\nof embryos at 3 days postfertilization\n(dpf) was monitored (Fig. 7, boxed re-\ngions). Control injected ﬁsh showed\nnormal circulation. Blood cells were\nseen traveling through the length of\nthe embryo by means of the artery/\nvein system (Supplementary Movie\n1A, which is available online at www.\ninterscience.wiley.com/jpages/1058-\n8388/suppmat). Importantly, circula-\ntion was also detected throughout the\nintersomitic vessels, indicating that\nthese vessels were functional at this\nstage of development. In contrast, cir-\nculation was abnormal in embryos in-\njected with splice blocker 1. Although\nblood cells were seen traveling in the\nartery and vein, little or no circulation\nwas detected in the intersomitic ves-\nsels (Supplementary Movie 1B). This\nobservation suggests that suppression\nof CD146 protein expression blocked\nthe development of the intersomitic\nvessels. This effect was tabulated, and\nthe results are shown in Table 1. We\ndecided to score the vessel develop-\nment of the embryos by the percent-\nage of functional intersomitic vessels\npresent at 3 dpf. This titration exper-\niment was done multiple times in\nsmall-scale pilot experiments, and a\nsimilar inhibition pattern was consis-\ntently observed. Next, we chose to in-\nject the embryos with either 1.2 pmol\ncontrol MO or 0.8 pmol splice blocker\n1 in three additional independent ex-\nperiments. The results are summa-\nrized in Figure 7C. Splice blocker 1\nclearly had a deleterious effect on\nthe development of intersomitic ves-\nsels. The deleted amino acids, there-\nfore, are critical to CD146 functions.\nBecause the transmembrane and cy-\ntoplasmic domains of CD146 were\ndeleted in the morphants, intracellu-\nlar signaling events triggered by\nCD146 were presumably abolished.\nblocker 1–injected ﬁsh appeared to be\ncompromised. For example, distinct\ncranial vessels were seen in the con-\ntrol-injected ﬁsh (Fig. 8A). In contrast,\nthe cranial vessels of\nthe splice\nblocker 1-injected embryo appeared to\nform less-distinct tubular structures\n(Fig. 8B). In particular, a group of ves-\nsels called the cranial division of the\ninternal carotid was less organized\ncompared with the control\n(arrow-\nheads in Fig. 8A,B). Second, the inter-\nsomitic vessels were narrower com-\npared with the control, even though\nthese vessels were detected by mi-\ncroangiography. A movie clip of the\ncirculation again revealed that the de-\nvelopment of intersomitic vessels was\n(Supplementary\nseverely\nMovie 3). However, these ISVs were\nlumenized as seen by ﬂuorescence mi-\ncroangiography (Fig. 8D). A similar\ndefect was observed in the caudal\nplexus region. When comparing the\nmicroangiogram and the blood circu-\nlation of the splice blocker–injected\nembryos, we observed that blood did\nnot ﬂow through part of the lumenized\ncaudal artery and vein vessels (corre-\nsponded to the region marked be-\ntween the arrowhead and the arrow in\nFig. 8D), resulting in a circulation\nshunt. This ﬁnding suggests that the\nlumen at the site where the circula-\ntion shunt occurred was too narrow\nfor blood cells to pass through. Our\nobservations suggest that CD146 may\nbe important in formation of vessels\nwith proper lumen size.\nElements Critical to CD146\nFunction Are Present at the\nN-Terminus of the Protein\nTo validate our data with splice\nblocker 1, we designed a second an-\ntisense morpholino oligonucleotide\nto target the 3 end of exon 2 and the\npredicted GT splice donor site of in-\ntron 2 (Fig. 4). When this MO was\ndelivered into the embryos by micro-\ninjection,\nit also caused aberrant\nmRNA splicing. RT-PCR analyses\nwere performed with total RNA har-\nvested from control MO- and splice\nblocker 2–injected embryos. The con-\ntrol MO did not affect splicing at the\nexon2/intron2 junction of CD146\npre-mRNA (Fig. 6C). However, se-\nquencing data revealed that splice\nblocker 2 inhibited normal splicing\nbetween exons 2 and 3 (Fig. 6D).\nWith the predicted splice donor site\nof\nthe\nintron 2 blocked, G94 of\nmRNA was used as an alternate,\ncryptic splice donor site. This aber-\nrant splicing event yielded a mutant\nCD146 with an in-frame deletion of\n26 amino acids (V32 to T57) at the N\nterminus (Fig. 1, deleted region in\nbold).\nDeletion of amino acids V32 to T57\nﬁsh, no circulation was seen in the\nintersomitic vessels (Supplementary\nMovie 4B and Table 2), a phenotype\nvery similar to that generated by\nsplice blocker 1–induced CD146 pro-\ntein expression knockdown. There-\nfore, these two MOs appeared to have\nphenocopied each other. Furthermore,\nthis result strongly suggests that the\nregion between V32 and T57 is likely\nto contain elements critical for CD146\nfunction.\nsimilarity\nIn this report, we cloned and charac-\nterized the CD146 ortholog in ze-\nbraﬁsh. The overall amino acid se-\nquence\nthe\nzebraﬁsh and human proteins was ap-\nproximately 30%. Whole-mount\nin\nsitu hybridization results indicate\nthat the CD146 has a dynamic expres-\nsion pattern in the vascular tree dur-\ning development. At 1 dpf, CD146\ntranscript is detected in the dorsal\naorta, caudal artery, and the inter-\nsomitic vessels. In the venous system,\nhowever, CD146 is detected only in\nthe caudal vein but not in the poste-\nrior cardinal vein at this stage (Fig.\n3B). However, CD146 began to ex-\npress in the posterior cardinal vein at\n2 dpf (Fig. 3E). The functional impor-\ntance of the dynamic expression of\nCD146 in the venous system is yet to\nbe understood. CD146 is also ex-\npressed in the heart of developing ze-\nbraﬁsh embryos and may play a role\nin cardiac development. We occasion-\nally observed pericardial edema in the\nmorphants, but there did not seem to\nbe a correlation between this pheno-\ntype and the vascular defects de-\nscribed here. Future studies will in-\nclude the analysis of the relationship\nand\nbetween heart\nCD146 functions. In this report, we\nfocused on the vascular functions of\nCD146 primarily because of the ap-\nparent roles CD146 plays in tumor an-\ngiogenesis (Mills et al., 2002; Yan et\nal., 2003).\ndevelopment\nWhen the protein expression of\nCD146 was knocked down, vascular\ndevelopment in the zebraﬁsh embryo\nwas severely affected. One most prom-\ninent defect was the lack of blood cir-\nculating in the intersomitic vessels by\n3 dpf (Supplementary Movies 1A and\nB). This lack of circulation was not\nof CD146 also affected the develop-\nment of intersomitic vessels in ze-\nbraﬁsh embryos. At day three, circu-\nlation was normal in embryos injected\nwith the control MO (Supplementary\nMovie 4A). In splice blocker 2–injected\ndue to the absence of intersomitic ves-\nsels per se, because these vessel struc-\ntures were clearly detected by ﬂuores-\ncence microangiography,\nalthough\nmany of them appeared to have un-\neven thickness and stenosis was evi-\ndent even when observed under a ﬂu-\n(Fig. 8B,D).\norescence microscope\nAlthough the lumen size of the inter-\nsomitic vessels in zebraﬁsh has not\nbeen reported in literature, the diam-\neters of artery and vein in zebraﬁsh\nembryos have been measured to be\n9.0 m,\n12.8\nrespectively\n(Fritsche et al., 2000). The use of ﬂu-\norescent microbeads with an average\ndiameter of 0.02 m allowed us to de-\ntect vessels with a diameter 400\ntimes smaller than that of the vein.\nBased on our results, we speculate\nthat, in the absence of CD146, the\nintersomitic vessels cannot develop\ninto functional tubes with a lumen\nopening large enough for blood cells to\npass through.\nand\nThe development of the intersomitic\nvessels is a highly organized and cho-\nreographed process. It was reported\nrecently that formation of the inter-\nsomitic vessels involves three differ-\nent endothelial cell types sprouting\nfrom the dorsal aorta (Childs et al.,\n2002). The ﬁrst two form a T-like\nstructure at the dorsal side of the dor-\nsal aorta and the ventral side of the\ndorsal longitudinal anastomotic ves-\nsel. The third one connects these two\nT structures. Because the intersomitic\nvessels were detected in the mor-\nphants with CD146 protein expres-\nsion suppressed, this ﬁnding suggests\nthat CD146 may play a role in the\nstep(s) after the basic architecture of\nthe intersomitic vessels has been laid\nout by these three types of endothelial\ncells.\nData from our morpholino experi-\nments indicate that CD146 may play\nan important role in the remodeling of\nthe caudal vein plexus into functional\nvascular tubes as well. In the CD146\nmorphants, circulation ﬂowed from\nthe dorsal aorta into the caudal artery\nand returned prematurely into cardi-\nnal vein, creating a circulation shunt\nin the caudal artery/vein system (Sup-\nplementary Movie 2B). This circula-\ntion shunt appears to be distinct from\nthat observed in the zebraﬁsh mutant\nmindbomb (mib) defective in Notch\nsignaling (Lawson et al., 2001). In the\nmib mutant, the circulation shunt oc-\ncurred in the trunk region where cir-\nculation ﬂowed directly from the dor-\nsal aorta into the posterior cardinal\nvein without entering the caudal ar-\ntery/vein system. This circulation by-\npass was speculated to arise from dis-\norganization of the boundary between\nthe dorsal aorta and the posterior car-\ndinal vein based on histological data\n(Lawson et al., 2001). In the CD146\nmorphants described here, blood cir-\nculation entered the caudal artery but\nreturned into the venous system pre-\nmaturely into the caudal vein. How-\never, ﬂuorescence microangiography\nrevealed that the caudal plexus was\nlumenized (Fig. 8D, see region be-\ntween the arrow and the arrowhead).\nTherefore, the caudal shunt present in\nthe CD146 morphants may be a result\nof restricted lumen diameter, a struc-\ntural defect similar to that observed in\nthe intersomitic vessels.\nOur observation that CD146 is im-\nportant in vascular development is in\nagreement with several studies re-\nported in the literature. For example,\nit was shown the CD146 expression\nwas up-regulated in human microvas-\ncular endothelial cells that had under-\ngone tubular morphological changes\non matrigel (Glienke et al., 2000). In\nadditional, the ability of endothelial\ncells to form tubes on matrigel can be\nblocked by a CD146 neutralizing anti-\nbody (Mills et al., 2002). Functional\nsuppression of CD146 by an antibody\ncan also inhibit endothelial cell migra-\ntion in vitro and neovascularization in\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003).\nWe have also identiﬁed V32 to T57\nof CD146 to be an important sequence\nelement in its protein function. We\nused an antisense morpholino oligo-\nnucleotide targeting the predicted\nexon 2/intron 2 junction of zebraﬁsh\nCD146 pre-mRNA to create a mutant\nprotein with those 26 amino acids de-\nleted in an in-frame manner. This\ntruncated protein appeared to be non-\nfunctional. This region may contain a\nrecognition epitope for CD146 func-\ntions. Alternatively, the deletion may\nhave affected proper protein folding,\nas the region deleted contains a con-\nserved cysteine residue in the ﬁrst im-\nmunoglobulin domain.\nVascular development requires ex-\ntensive dynamic changes of endothe-\nlial morphology. These dramatic mor-\nphological\nchanges were recorded\nrecently by time-lapse microscopy in a\ndeveloping zebraﬁsh embryo in real-\ntime (Lawson and Weinstein, 2002).\nEndothelial cells were seen to extend\nand retract long ﬁlopodial processes\nduring path ﬁnding. Communication\nappears to occur between these acti-\nvated endothelial cells through ﬁlopo-\ndial contacts, resulting in proper vas-\ncular remodeling and regression to\nensure the development of functional\nlumens\n(Lawson and Weinstein,\n2002). The exact molecular basis of\nthese cellular behaviors is certainly\ncomplex, but highly orchestrated and\ndynamic changes of the cytoskeleton\norganization are probably involved\n(Davis et al., 2002). For example,\nCdc42, Rac1, and integrins are some\nof the signaling molecules intimately\nlinked to cytoskeleton functions that\nhave been shown to be critical in en-\ndothelial tube and lumen formation in\nvitro (Davis and Camarillo, 1996; Bay-\neral integrins that are known to be\nexpressed in endothelial cells (Alais et\nal., 2001; Hodivala-Dilke et al., 2003).\nAll these cellular events appear to be\nrelated to the cytoskeleton. Perhaps,\nCD146 mediates the formation of\nproper vessels through regulating the\ndynamic changes in the organization\nof the cytoskeleton during develop-\nment. Further studies that are de-\nsigned to dissect the intracellular sig-\nnaling events important in CD146\nfunctions are currently under way.\nEXPERIMENTAL\nPROCEDURES\nZebraﬁsh Care\nZebraﬁsh (Danio rerio) was raised and\nmaintained as described (Westerﬁeld,\n2000).\n5 and 3 RACES\nTotal RNA was collected from ze-\nbraﬁsh embryos at 48 hours postfertil-\nization (hpf) by using either the\nRNAaqueous-4PCR kit from Ambion\n(Austin, TX) or the RNeasy Mini kit\nfrom Qiagen (Valencia, CA). The 5\nand 3 RACEs were carried out by us-\ning the GeneRacer Kit from Invitro-\ngen (Carlsbad, CA) according to the\nmanufacturer’s protocol. The kit was\nchosen because the transcriptional\nstart site of the mRNA can be ob-\ntained directly. Brieﬂy, 5 RACE was\nperformed by using 5-GGCACAGG-\nGAGTGTCTTCTTGCCTTATA-3\nas\nthe gene-speciﬁc primer for ﬁrst-\nstrand cDNA synthesis. The cDNA\nwas ampliﬁed by nested PCR using\nthe GC-Advantage DNA polymerase\nfrom Clontech (Palo Alto, CA). The\nprimers for the ﬁrst round of PCR\nwere the GeneRacer 5 Primer (pro-\nvided in the GeneRacer Kit) and the\ngene-speciﬁc primer. For the second\nround of ampliﬁcation, the GeneRacer\n5 Nested Primer (provided in the\nGeneRacer Kit) and a primer with the\nsequence 5-GCAGCTTTCCCTTCTT-\nGTAGAGGTAAT-3 were used. The\nPCR products were cloned into the\nTOPO cloning kit from Invitrogen.\nProtein sequence alignment was per-\nhttp://prodes.toulouse.\nformed\ninra.fr/multalin/multalin.html\n(Cor-\npet, 1988). The mRNA sequence has\nbeen deposited in GenBank (accession\nno. AY497061).\nless et al., 2000; Bayless and Davis,\n2002; Davis and Bayless, 2003).\nAlthough the molecular mechanism\nby which CD146 affects proper lumen\nformation observed in this report is at\nthis point unclear, it is tempting to\nspeculate that CD146 may achieve\nsuch function by regulating the cy-\ntoskeleton. It is known that antibody-\nmediated cross-linking of CD146 in\nvitro initiates a cascade of intracellu-\nlar signaling events that\nincludes\nphosphorylation of focal adhesion ki-\nnase, paxillin, and p130CAS (Anfosso\net al., 1998, 2001). Additionally,\nCD146 has been shown to interact\nwith the actin cytoskeleton in HUVEC\n(Bardin et al., 2001). Moreover, the\nCD146 ortholog in chicken has been\nshown to modulate expression of sev-\nIn Situ Hybridization\nNucleotides spanning the 5 untrans-\nlated region and the ﬁrst 592 bases of\nthe open reading frame of zebraﬁsh\nCD146 were ampliﬁed and cloned into\na vector derived from pCR-BluntII-\nTOPO (Invitrogen). To\ngenerate\ndigoxigenin (DIG)-labeled antisense\nprobes, the DNA plasmid was linear-\nized with EcoRI, followed by tran-\nscription using SP6 polymerase and\nthe 10 DIG RNA labeling mix from\nRoche (Indianapolis, IN). In situ hy-\nbridization was then performed ac-\ncording to a published procedure\n(Broadbent and Read, 1999; Read,\n2000).\nMicroinjection and\nFluorescence\nMicroangiography\nTwo antisense (splice blocker 1. 5-\nCAGTTGTAGTGACCCACCTCGCA-\nCG-3; splice blocker 2. 5-GTCTTTT-\nTCTCACCGTAAACCAGAT-3) and a\ncontrol (5-GACTCTATATTGCCTAC-\nCTTTACAG-3) morpholino oligo-\nnucleotides were purchased from\nGene-Tools (Philomath, OR). The\nmorpholino oligonucleotides were in-\njected into the yolk sac of zebraﬁsh\nembryos between one- and four-cell\nstages. Embryos were raised in egg\nwater in the presence of 0.003% (w/v)\n1-phenyl-2-thiourea (Sigma, St. Louis,\nMO) to prevent pigmentation (Wester-\nﬁeld, 2000). To verify successful block-\nage of CD146 pre-mRNA splicing by\nthe morpholino oligonucleotides, total\nRNA was collected from the zebraﬁsh\nembryos at 2 dpf by using the RNeasy\nMini kit from Qiagen. Reverse tran-\nscription was carried out by using ei-\nther Thermoscript or Superscript III\nfrom Invitrogen with the gene-speciﬁc\nprimer (5-GGCACAGGGAGTGTCT-\nTCTTGCCTTATA-3). PCR was then\nperformed by using GC-Advantage\nDNA polymerase (Clontech) and the\nfollowing ampliﬁcation primers: 5-\nGATATCACCTTGACTTGCAACG-\nCATTGTCT-3 and 5-GATTGGG-\nTAGCCTTGTGCATAGCAGGTCAG-3\nfor\nsplice blocker 1 injection;\n5-ATGACCTACACCGCACTGCTGC-\nTCGCGGGA-3 and 5-AGGATC-\nGAGAGGGAAGCTGCCCTTGTAC-\nCA-3 for splice blocker 2 injection. A\npair of isoform speciﬁc primers that am-\npliﬁed only the wild-type CD146 cDNA\n(nucleotides 126 to 1287) has the follow-\ning sequences: 5-TACTTACACTCT-\nGACGGAAGAAGCCAAGAT-3 and\n5-CTTTCCTCGCACGTTTACTTGCAC-\nTGA-3. Primers used to ampliﬁed Fli-1\nwere 5-GAAAATGGACGGAACTAT-\nTAAGGAGGCGCT-3 and 5-TCTTC-\nGAGTGCAGTTCAAGTTTTGGCATT-\nTAAGGGT-3.\nBlood circulation in anesthetized\nﬁsh embryos was recorded as time-\nlapse images using an inverted phase\ncontrast microscope (Nikon Corpora-\ntion, Tokyo, Japan), a Spot RT cam-\nera, and the Advanced SPOT software\n(Diagnostic Instruments, Inc., Ster-\nling Heights, MI). The images were\nconverted into QuickTime movies us-\ning a program called ImageJ.\nmicroangiography\nwas performed by using the red ﬂuo-\ncarboxylate-modiﬁed Fluo-\nrescent\nSpheres (0.02 m)\nfrom Molecular\nProbes (Eugene, OR) as described\n(Weinstein et al., 1995). Fluorescence\nmicroangiograms were recorded with\nthe Nikon ﬂuorescence microscope us-\ning a ﬁlter set (41035:ex. 546; em. 605)\nfrom Chroma (Rockingham, VT).",
         null
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "Differential Contribution of Troponin I Phosphorylation Sites to the Endothelin-modulated Contractile Response*",
         "Cardiac troponin I is a phosphorylation target for endothelin-activated protein kinase C. Earlier work in cardiac myocytes expressing nonphosphorylatable slow skeletal troponin I provided evidence that protein kinase C-mediated cardiac troponin I phosphorylation accelerates relaxation. However, replacement with the slow skeletal isoform also alters the myofilament pH response and the Ca2+ transient, which could influence endothelin-mediated relaxation. Here, differences in the Ca2+ transient could not explain the divergent relaxation response to endothelin in myocytes expressing cardiac versus slow skeletal troponin I nor could activation of Na+/H+ exchange. Three separate clusters within cardiac troponin I are phosphorylated by protein kinase C, and we set out to determine the contribution of the Thr144 and Ser23/Ser24 clusters to the endothelin-mediated contractile response. Myocyte replacement with a cardiac troponin I containing a Thr144 substituted with the Pro residue found in slow skeletal troponin I resulted in prolonged relaxation in response to acute endothelin compared with control myocytes. Ser23/Ser24 also is a target for protein kinase C phosphorylation of purified cardiac troponin I, and although this cluster was not acutely phosphorylated in intact myocytes, significant phosphorylation developed within 1 h after adding endothelin. Replacement of Ser23/Ser24 with Ala indicated that this cluster contributes significantly to relaxation during more prolonged endothelin stimulation. Overall, results with these mutants provide evidence that Thr144 plays an important role in the acute acceleration of relaxation, whereas Ser23/Ser24 contributes to relaxation during more prolonged activation of protein kinase C by endothelin.",
         "",
         "http://www.jbc.org/article/S0021925820589394/pdf",
         "Cardiac troponin I is a phosphorylation target for endothelin-\nactivated protein kinase C. Earlier work in cardiac myocytes\nexpressing nonphosphorylatable slow skeletal troponin I provided\nevidence that protein kinase C-mediated cardiac troponin I phos-\nphorylation accelerates relaxation. However, replacement with the\nslow skeletal isoform also alters the myofilament pH response and\nthe Ca2 transient, which could influence endothelin-mediated\nrelaxation. Here, differences in the Ca2 transient could not explain\nthe divergent relaxation response to endothelin in myocytes\nexpressing cardiac versus slow skeletal troponin I nor could activa-\ntion of Na/H exchange. Three separate clusters within cardiac\ntroponin I are phosphorylated by protein kinase C, and we set out to\ndetermine the contribution of the Thr144 and Ser23/Ser24 clusters to\nthe endothelin-mediated contractile response. Myocyte replace-\nment with a cardiac troponin I containing a Thr144 substituted with\nthe Pro residue found in slow skeletal troponin I resulted in pro-\nlonged relaxation in response to acute endothelin compared with\ncontrol myocytes. Ser23/Ser24 also is a target for protein kinase C\nphosphorylation of purified cardiac troponin I, and although this\ncluster was not acutely phosphorylated in intact myocytes, signifi-\ncant phosphorylation developed within 1 h after adding endothelin.\nReplacement of Ser23/Ser24 with Ala indicated that this cluster con-\ntributes significantly to relaxation during more prolonged endothe-\nlin stimulation. Overall, results with these mutants provide evi-\ndence that Thr144 plays an important role in the acute acceleration\nof relaxation, whereas Ser23/Ser24 contributes to relaxation during\nmore prolonged activation of protein kinase C by endothelin.",
         "Protein kinase C (PKC)2 activation is an important pathway involved\nin modulating cardiac contractile function (1–5). A number of end tar-\nget proteins involved in excitation-contraction coupling are phospho-\nrylated in response to PKC activation, including the L-type Ca2 chan-\nnel (6, 7) and myofilament proteins such as cardiac troponin I (cTnI)\n(8 –11), cardiac troponin T (10, 12), and myosin light chain 2 (MLC2) (8,\n13, 14). The contribution of each end target to the contractile response\nto PKC is not well understood. Our laboratory is interested in under-\nstanding the role of cTnI in the PKC-mediated contractile function\nresponse. Increased PKC expression is observed in failing human hearts\n(15, 16), and the relationship between PKC and cTnI phosphorylation\nmay exert an important influence on cardiac function under physiolog-\nical and pathophysiological conditions (17).\n* The costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked “advertisement” in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\nEarlier work demonstrated that cTnI phosphorylation is correlated\nwith the positive inotropic response to ET (4, 8, 18). Accelerated relax-\nation also is observed in response to PKC activation by ET in myocytes\nexpressing cTnI (4). PKC phosphorylation of purified cTnI decreases\nmyofilament Ca2 sensitivity in reconstituted myofilaments (11), and\nthis desensitization is expected to contribute to the accelerated relax-\nation. In contrast, peak shortening is diminished and relaxation is\ndelayed rather than accelerated in response to ET when endogenous\ncTnI is replaced with nonphosphorylatable slow skeletal troponin I\n(ssTnI) in adult myocytes. These results are consistent with the idea that\ncTnI phosphorylation enhances peak myocyte shortening and acceler-\nates relaxation rate in response to PKC activation by ET. However,\nssTnI expression in adult myocytes also increases myofilament Ca2\nsensitivity and slows the Ca2 transient (19, 20). Thus, the differential\nresponse to ET observed in myocytes expressing cTnI versus ssTnI may\nresult from the direct actions of cTnI phosphorylation on shortening\nand relaxation or from alterations in the Ca2 transient induced by\nssTnI expression.\nOne goal of the present study is to establish whether PKC-mediated\ncTnI phosphorylation acts directly to accelerate relaxation or is\nexplained by parallel alterations in cellular Ca2. Calcium and shorten-\ning responses are measured simultaneously during the ET response for\nthese experiments. Stimulation frequency also alters the Ca2 transient,\nwith accelerated sarcoplasmic reticulum Ca2 uptake observed at\nhigher pacing frequencies (21). The relative contribution of phospho-\nrylated cTnI to the ET-induced relaxation response may decrease as\npacing frequency increases in the event that alterations in cellular Ca2\nare primarily responsible for this relaxation response. Thus, the contri-\nbution of cTnI phosphorylation to the ET-induced relaxation response\nis examined over a range of frequencies.\nA second goal of our study was to differentiate between cTnI phos-\nphorylation-induced and alkalosis-induced changes in the ET-mediated\ncontractile performance response. Previous investigators showed that\nET increases Na/H exchange via PKC (22, 23). The ensuing alkalosis\nincreases myofilament Ca2 sensitivity (24), and ET-induced alkalosis\nhas been shown to contribute to the enhanced peak shortening and\nincreased rate of contraction (22, 23). The diminished peak-shortening\nresponse to ET observed in ssTnI-expressing myocytes (4) also could\nresult from the enhanced basal myofilament Ca2 sensitivity and dimin-\nished pH sensitivity relative to cTnI-expressing myocytes (19). In the\npresent study, the influence of ET-induced alkalosis on peak contrac-\ntion was tested by inhibiting Na/H exchange in myocytes expressing\ncTnI and ssTnI.\nThe final aim of the present work was to determine the contribution\nof individual phosphorylation sites within cTnI to the PKC-mediated\nchanges in intact adult myocyte relaxation rate in response to ET.\nDetailed work on purified cTnI indicates that PKC phosphorylates three\nseparate clusters of residues, including Ser23/Ser24, Ser43/Ser45, and\nThr144 within cTnI (11, 25). However, the myocardial response to phos-\nphorylation of individual sites within the intact myocyte has yet to be\nPKC Phosphorylation of Troponin I and Contractile Function\nthoroughly investigated. Studies here are focused on investigating the\ninfluence of the rat cTnI Ser23/Ser24 and Thr144 sites on contractile\nfunction in response to ET. For studies of cTnIThr144, the goal is to\ndetermine whether Thr144 influences the peak contraction and relax-\nation response to PKC activated by ET, by substitution with the non-\nphosphorylatable Pro residue. The Ser23/Ser24 cluster also is a target for\nPKC in purified cTnI and is a major myofilament target for protein\nkinase A (PKA) (26). Phosphorylation of this cluster by PKA signifi-\ncantly decreases myofilament Ca2 sensitivity (27) and accelerates\nrelaxation during -adrenergic activation (28). A similar increase in\nrelaxation is expected if this Ser cluster is phosphorylated by PKC in\nintact adult myocytes. For these studies, the Ser23/Ser24 residues were\nsubstituted with Ala to investigate whether this cluster influences the\nability of cTnI to maintain relaxation in response to ET activation of\nPKC.",
         "EXPERIMENTAL PROCEDURES\nMutagenesis Strategy—Wild-type rat cTnI cDNA (kind gift of A.\nMurphy) was used to construct the cTnIT144P and cTnIS23A/S24A\nmutants. Primers used for mutagenesis of cTnIT144P were 5-gcaagtt-\ntaagcggccgcctctccgaagagtg-3 (sense) and 5-cactcttcggagaggcggccgct-\ntaaacttgc-3 (antisense), and for cTnIS23A/S24A the primers were\n5-ccgacgtcgcgccgcggccaactaccg-3 (sense) and 5-cggtagttggccgcg-\ngcgcgacgtcgg-3 (antisense) using a pGEM-3Z vector and the Strat-\nagene QuikChange site-directed mutagenesis kit (29). The presence of\neach mutant was verified by DNA sequencing.\nGeneration of Adenoviral Vectors—Recombinant adenovirus vectors\nwere constructed by cotransfection of pCA4cTnI cDNA with pJM17 by\ncalcium phosphate into a HEK 293 cell\nline. Replication-deficient,\nrecombinant adenovirus containing wild-type rat cTnI, wild-type\nssTnI, or mutant ssTnI cDNAs, a cytomegalovirus promoter, and an\nSV40 polyadenylation signal were obtained following homologous\nrecombination (19).\nMyocyte Isolation—Adult rat ventricular myocytes were isolated as\ndescribed earlier (4, 30). Myocytes were plated on laminin-coated cov-\nerslips in Dulbecco’s modified Eagle’s medium supplemented with 50\nunits/ml penicillin, 50 g/ml streptomycin (pen/strep) and 5% serum\nfor 2 h prior to replacing the medium with serum-free Dulbecco’s mod-\nified Eagle’s medium containing recombinant adenovirus vector (mul-\ntiplicity of infection  500). Gene transfer of exogenous TnI at a mul-\ntiplicity of infection of 500 results in virtually all cells expressing and\nincorporating exogenous TnI into the myofilament (29). After 1 h, Dul-\nbecco’s modified Eagle’s medium with pen/strep (2 ml) was added, with\nregular replacement of media over 4 days in culture.\nAnalysis of Protein Expression—Control and virus-treated adult rat\ncardiac myocytes were scraped from coverslips into sample buffer 4\ndays after gene transfer (31). In some experiments, myocytes were per-\nmeabilized in 0.1% Triton X-100 for 1 min to evaluate incorporation of\nTnI protein into the myofilament after gene transfer. In earlier work,\nexpression in intact and permeabilized myocytes was comparable when\nmyofilament proteins were incorporated into the myofilament (29, 32).\nProteins were then separated by gel electrophoresis and blotted onto\npolyvinylidene difluoride membrane for 2000 V-h with immunodetec-\ntion carried out as described earlier (30) using a 1:1000 dilution of MAB\n1691 (Chemicon Inc., Temecula, CA), a monoclonal antibody that rec-\nognizes all TnI isoforms. A phospho-Ser23/Ser24-specific polyclonal\nantibody (1:1000; Cell Signaling Technology, Beverly, MA) and a paired\nhorseradish peroxidase-conjugated secondary antibody were used to\ndetect Ser23/Ser24 phosphorylation. Myocytes collected for detection of\nphosphorylation were incubated at 37 °C with calyculin A (10 nM) alone\nor with the selected PKA and PKC agonist and/or inhibitor for time\nintervals ranging from 2.5 min up to 1 h.\nMeasurement of Sarcomere Shortening in Single Cardiac Myocytes—\nMyocytes to be used for shortening assays were transferred to a Plexiglas\nstimulation chamber consisting of 8 wells containing platinum elec-\ntrodes. Myocytes were stimulated electrically (2.5-ms pulse, 0.2 Hz) in\nmedium 199 supplemented with pen/strep (M199), 10 mM HEPES, 0.2\nmg/ml bovine serum albumin, and 10 mM glutathione (M199), and\nthe media was replaced every 12 h for days 2– 4 post-gene transfer (33).\nThe voltage was set so that 20% of the myocytes were stimulated on\neach coverslip. Four days after gene transfer, individual coverslips were\ntransferred to a temperature-controlled (37 °C) chamber mounted on a\nNikon microscope stage. A video-based detection system (Ionoptix,\nMilton, MA) was used to detect sarcomere shortening in intact myo-\ncytes. For frequency-response studies, myocytes were stimulated at\neach frequency (0.2, 0.5, 0.75, 1, and 2 Hz) until a steady state contrac-\ntion was reached. Sarcomere shortening also was recorded at 0.2 Hz\nbefore and after the increases in stimulation frequency to ensure there\nwere no changes in the rate of contraction, peak contraction, and rate of\nrelaxation after testing at all frequencies. Experiments were recorded\nusing SarcLen software, and an average of 10 twitches/myocyte was\ncollected under basal conditions. Signal-averaged data were analyzed to\ndetermine resting sarcomere length (SLR), percent change in peak sar-\ncomere shortening above base line (Sb), time to peak shortening (TTP),\ntime to 25, 50, and 75% relaxation (TTR25, TTR50, and TTR75, respec-\ntively), and maximum normalized shortening and relaxation velocities\n(dl/dtmax, dl/dtmax, respectively). Media containing ET (10 nM),\n5-N(ethyl-N-iospropyl)amiloride (EIPA, 500 nM) or both reagents were\nperfused via separate individual perfusion tubing to eliminate introduc-\ntion of residual agonists/inhibitors.\nExperiments for Ca2 transient and sarcomere shortening measure-\nments were carried out on myocytes loaded with Fura-2 AM, as\ndescribed previously (34). Briefly, myocytes were loaded for 5 min with\n5 M Fura-2 AM (final Me2SO  0.6%) at 37 °C followed by a 5-min\nwash-out for de-esterification. Myocytes were then stimulated at 0.2 Hz\nat 37 °C, and the base-line sarcomere length and Ca2 transient were\nrecorded from a single cell. Recordings were then made 5, 10, and 15\nmin after the addition of ET (10 nM), with the cell maintained in the dark\nbetween sampling times. Optimal cTnI phosphorylation was observed\nin response to 10 –100 nM ET in an earlier study (4). Ten twitches/\nmyocyte were collected for each sample. In addition to sarcomere short-\nening measurements, the resting and peak ratios, maximum Ca2 tran-\nsient rate (dCa/dtmax), maximum Ca2 decay rate (dCa/dtmax), and\ntime to 50 and 75% decay (TTD50, TTD75, respectively) in the Ca2\ntransient were measured. The Ca2 transient and shortening measure-\nments were maintained over this same time interval in myocytes main-\ntained under basal conditions.\nStatistical Analysis—Values are expressed as mean  S.E. Grouped\ncomparisons are analyzed using a Student’s t test or analysis of variance\ntesting (ANOVA). Statistical differences detected by ANOVA are com-\npared with a Newman-Keuls multiple comparison test with p  0.05\nconsidered significantly different.\nRESULTS\nRelationship between Ca2 Transient and Contractile Response to\nEndothelin—Sarcomere shortening and Ca2 transients were measured\nbefore and after ET in Fura-2-loaded myocytes expressing cTnI or ssTnI\n4 days after gene transfer (Fig. 1, TABLE ONE). The increased peak\nshortening and enhanced relaxation rate observed in response to 10 nM\nET in earlier work on myocytes expressing cTnI (4), was similarly\nPKC Phosphorylation of Troponin I and Contractile Function\nobserved in myocytes loaded with Fura-2 (Fig. 1A). In myocytes\nexpressing nonphosphorylatable ssTnI, the delayed relaxation and loss\nof the enhanced peak shortening observed in response to ET (4) simi-\nlarly developed in Fura-2-loaded cells (Fig. 1B, TABLE ONE).\nCa2 transient measurements also were examined in these myocytes.\nThe time to 50% decay for basal Ca2 transients reportedly slows in\ntransgenic mouse myocytes expressing ssTnI (20), and similar results\ndeveloped in rat myocytes expressing ssTnI (TABLE TWO). Peak Ca2\nand the TTP remained unchanged in the ssTnI-expressing myocytes\ncompared with values observed in controls at 0.2 Hz. The Ca2 tran-\nsient change observed in response to ET was then analyzed in myocytes\nexpressing cTnI and ssTnI (TABLE ONE). The percent change in peak\nCa2 amplitude and maximum rate of Ca2 decay (dCa/dtmax) in\nresponse to ET was not significantly changed in myocytes expressing\neither cTnI or ssTnI (Fig. 1, A and B; TABLE ONE). However, signifi-\ncant delays were observed in the time to 75% Ca2 decay in both groups\nof myocytes. The similar Ca2 transient responses to ET coupled with\nthe divergent shortening response of cTnI- versus ssTnI-expressing\nmyocytes indicates that cTnI contributes significantly to the enhanced\npeak shortening and accelerated relaxation rate observed in response to\nPKC activation by ET compared with myocytes expressing ssTnI.\nAdditional studies were performed to investigate whether the contri-\nbution of phosphorylated TnI to the ET-mediated change in contractile\nfunction is diminished with higher frequencies. The shortening  fre-\nquency relationship described previously in rodents (33, 35) was similar\nunder basal conditions and in response to ET over a range of frequencies\n(0.2–2 Hz), (TABLE THREE) in control myocytes. The percent decrease\nin peak shortening, dl/dtmax and dl/dtmax in response to increased\nfrequency was similar before and after ET. The consistency of this\nresponse provides evidence that the contribution of cTnI phosphoryla-\ntion to contractile function is not altered by frequency-induced changes\nin the Ca2 cycle.\nContribution of Cellular Alkalosis to the Myofilament Response to\nEndothelin—An important component of the contractile response to\nET depends on activation of Na/H exchange and the generation of a\ncellular alkalosis (22, 23). Experiments were carried out in the presence\nof the amiloride analog EIPA, an inhibitor of Na/H exchange (23, 36),\nto identify responses resulting from alkalosis. The myocyte response to\nEIPA is comparable in nontreated myocytes and myocytes expressing\ncTnI after gene transfer (results not shown); the pooled results are\nshown in Fig. 2. Myocyte shortening amplitude in myocytes expressing\nPKC Phosphorylation of Troponin I and Contractile Function\ncTnI decreased slightly after the addition of EIPA but did not signifi-\ncantly influence peak amplitude in myocytes expressing ssTnI (Fig. 2, A\nand B). Resting sarcomere length (SLR), dl/dtmax, and dl/dtmax\n(dl/dt and dl/dt in Fig. 2, respectively) were not significantly altered\nby EIPA in myocytes expressing cTnI or ssTnI (Fig. 2B). However, EIPA\nblocked a major portion (70%) of the ET-induced increase in peak\nshortening amplitude observed in cTnI-expressing myocytes. In con-\ntrast, EIPA did not significantly modify the accelerated relaxation\nresponse to ET in myocytes expressing cTnI, and the delayed relaxation\nobserved in response to ET was comparable with and without EIPA in\nmyocytes expressing ssTnI (Figs. 1 and 2). These results support the idea\nthat Na/H exchange and the ensuing alkalosis is important for the\nenhanced peak shortening response, whereas cTnI plays a more critical\nrole in the accelerated relaxation rate response to ET.\nRole of cTnI Phosphorylation Residues in the Endothelin Response—\nTo determine the role of the cTnIThr144 phosphorylation site in the\nrelaxation response, the ssTnI Pro residue was substituted for Thr144 to\nform cTnIT144P, and replacement was measured 4 days after gene\ntransfer. The stoichiometric replacement of cTnI with cTnIT144P-\nFLAG was 46  2% (n  13) and comparable with the 53  2% (n  14)\nreplacement with cTnI-FLAG 4 days after gene transfer (Fig. 3).\nReplacement of nontagged cTnI and cTnIT144P is expected to be com-\nparable in myocytes over the same time interval (29, 37). Total TnI\nexpression remained constant in myocytes expressing cTnI and\ncTnIT144P based on the ratio of TnI intensity to the intensity of a\nsilver-stained band on the gel (cTnI set at 1.0; cTnIT144P  1.13  0.10,\nn  11), and isoform expression of tropomyosin and troponin T was\ncomparable in myocytes expressing cTnI and cTnIT144P (results not\nshown). Expression of total TnI also was comparable in intact (I) and\npermeabilized (P) myocytes (Fig. 3, I/P ratio for cTnI  1.10  0.08, n \n5; cTnIT144P  1.05  0.06, n  5), which is consistent with myofila-\nment incorporation of the delivered TnI.\nMeasures of basal contractile function in myocytes expressing\ncTnIT144P are not significantly different from values obtained from\nmyocytes expressing cTnI (TABLE TWO, Fig. 4A). These results are in\nagreement with the earlier reverse substitution of Pro110 with Thr in fast\nskeletal TnI (38). Base-line Ca2 transients in myocytes expressing\ncTnIT144P (Fig. 4B) also showed no significant differences from control\nvalues. Peak shortening increased significantly in response to ET in\ncTnIT144P-expressing myocytes (Fig. 4) and was comparable with the\nET-induced enhanced peak shortening in control myocytes (Fig. 1B).\nHowever, several indices of relaxation, including dl/dtmax and TTR75,\nwere significantly delayed in response to ET in myocytes expressing\ncTnIT144P compared with controls (TABLE ONE). The amplitude of\nthe Ca2 transient response to ET was not significantly different com-\npared with control values for myocytes expressing cTnIT144P (TABLE\nONE, Fig. 4). As with control myocytes, a frequency change from 0.2 to\n2 Hz produced similar changes before and after ET in myocytes express-\ning cTnIT144P (results not shown). The major difference between myo-\ncytes expressing cTnIT144P versus cTnI was the decrease in relaxation\nrate in response to ET; this finding indicates a specific role for Thr144 in\naccelerating relaxation in response to ET.\nThe contribution of Ser23/Ser24 phosphorylation to the ET-induced\nchange in contractile function also was determined. A phospho-specific\nSer23/Ser24 antibody was used for these studies. Myocytes were incu-\nbated with the PKA agonist dobutamine (DOB) for 2 min to verify\ndetection of the well documented PKA-mediated cTnISer23/Ser24 phos-\nPKC Phosphorylation of Troponin I and Contractile Function\nphorylation site (39, 40) (Fig. 5A). Detection of cTnI phosphorylation by\nthis antibody was specific, as it was blocked when, in addition to DOB,\nthe -adrenergic inhibitor propranolol was added to myocytes (results\nnot shown). There was no significant phosphorylation detected in myo-\ncytes incubated with ET for 10 min with (Fig. 5A) or without the PKC\ninhibitor bis-indolylmaleimide-1 (bis-1, results not shown). However,\nsignificant phosphorylation of Ser23/Ser24 developed in control myo-\ncytes over a 1-h time span, and this ET-mediated phosphorylation was\nsignificantly reduced in the presence of the PKC inhibitor bis-1 but not\nby the -adrenergic antagonist propranolol (Fig. 5A). Treatment of\nmyocytes with other PKC agonists, including phenylephrine and phor-\nbol 12-myristate 13-acetate for 1 h also resulted in significant cTnI\nphosphorylation of Ser23/Ser24. Incubation with fibroblast growth fac-\ntor-2 was previously shown to activate PKC (41), yet this agonist did not\nsignificantly influence cTnI phosphorylation over a 1-h time span\n(results not shown). These results provide evidence for temporal phos-\nphorylation of cTnI Ser23/Ser24 in response to PKC activation by several\nPKC agonists.\nTo directly determine the contribution of Ser23/Ser24 to the contract-\nile function response to ET, cTnIS23A/S24A was expressed in adult\nmyocytes. By 4 days after gene transfer, 44  5% of cTnI was replaced\nwith the cTnIS23A/S24A (n  15, Fig. 5B). DOB phosphorylation of\ncTnI was significantly reduced in myocytes expressing cTnIS23A/S24A\nor cTnIS23A/S24A-FLAG (results not shown). In addition, Ser23/Ser24\nphosphorylation was significantly attenuated after 1 h of ET in myocytes\nexpressing tagged and nontagged cTnIS23A/S24A (Fig. 5C). This S23A/\nS24A substitution has previously been shown to have little influence on\nbasal myofilament function (42), and no significant differences were\nobserved in SLR, peak amplitude, dl/dtmax, ordl/dt max in myocytes\nexpressing cTnIS23A/S24A compared with controls (results not\nshown). Control myocytes expressing cTnI or\nthe substituted\ncTnIS23A/S24A responded similarly to ET over the first 15 min (Fig. 6).\nHowever, a significant difference in the response was observed 1 h after\nthe addition of ET in myocytes expressing cTnIS23A/S24A compared\nPKC Phosphorylation of Troponin I and Contractile Function\nwith control values (Fig. 6). Specifically, peak shortening amplitude was\nnot increased and both dl/dtmax and TTR75 indicated relaxation was\nsignificantly prolonged in myocytes expressing cTnIS23A/S24A com-\npared with control myocytes and were comparable with the results\nobtained in myocytes expressing ssTnI (Fig. 6). These results demon-\nstrate Ser23/Ser24 phosphorylation contributes to the accelerated relax-\nation during more prolonged myocyte PKC activation by ET.",
         "DISCUSSION\nResults from the present study demonstrate the important direct role\nfor cTnI phosphorylation in accelerating relaxation in response to PKC\nactivated by ET in adult myocytes (Figs. 1–3). Our results now demon-\nstrate a clear role for cTnI Thr144 and Ser23/Ser24 in contributing to the\naccelerated relaxation in response to ET (Figs. 4 and 6). Substitution\nwith a Pro residue at Thr144 results in the delayed relaxation previously\nobserved in myocytes expressing nonphosphorylatable ssTnI and pro-\nvides evidence that the Thr144 site contributes to accelerated relaxation\nduring the ET response. The Ser23/Ser24 cluster does not appear to\ncontribute to the initial response to ET but, instead, is important for the\naccelerated relaxation during more prolonged stimulation by ET. This\nfinding indicates that the relaxation rate response to ET-induced PKC\npathway activation depends on temporal phosphorylation of individual\nclusters within cTnI.\nTnI Phosphorylation Plays a Direct Role in the Relaxation Response to\nEndothelin—Results from the present study demonstrate the important\nrole of cTnI phosphorylation in the relaxation response during the pos-\nitive inotropic response to ET, independent of cellular Ca2 handling.\nThe only aspect of Ca2 transient that changed significantly in response\nto 10 nM ET was the percent change in TTD75, which decreased in both\ncontrol and ssTnI-expressing myocytes (Fig. 1, TABLE ONE). All other\nmeasurements of the Ca2 transient remained unchanged during the\nET response, and work by other laboratories supports this finding (43,\n44). There are reports that ET activation of PKC enhances the Ca2\ntransient at higher doses of ET (45), which may be due to increased Ca2\nentry via the L-type Ca2 channel (46). However, results from Ebihara et\nal. (47) indicate the increased Ca2 transient is not necessary for the\nenhanced contractile response to ET.\nExperiments with the NHE1 inhibitor EIPA also demonstrate that\ninhibition of NHE1 prevents the ET-mediated enhancement in peak\nshortening amplitude (Fig. 2). ET in the concentration range used here\nactivates PKC-induced NHE1 (48) and causes a cellular alkalosis (22,\n23). The alkalosis-mediated increase in myofilament Ca2 sensitivity\n(24) is responsible for the increased peak contractile shortening (22, 23).\nThis enhancement in peak shortening is absent from myocytes express-\ning ssTnI (Fig. 1), which is likely due to the combination of increased\nbasal myofilament Ca2 sensitivity (19, 20) and a decreased pH response\nto alkalosis in these myocytes compared with cTnI-expressing myocytes\n(19).\nIn contrast to peak shortening, the divergent influence of cTnI versus\nssTnI on the relaxation response to ET was not influenced significantly\nby EIPA (Fig. 2). Although the slowing of relaxation in myocytes\nexpressing ssTnI was attenuated by the presence of EIPA during the ET\nresponse, relaxation was significantly accelerated in myocytes express-\ning cTnI versus ssTnI during this response. Our results now provide\nevidence that phosphorylated cTnI maintains relaxation regardless of\nthe alkalosis. The delayed relaxation response to ET observed in myo-\ncytes expressing cTnIT144P also supports this finding (Fig. 4).\nWhat Role Does cTnI Phosphorylation Play in the Contractile\nResponse to ET?—The role of cTnI phosphorylation in the contractile\nfunction response to PKC agonists remains controversial. Earlier in\nvitro work showed that PKC phosphorylation of cTnI decreases myofil-\nament Ca2 sensitivity as well as maximum actomyosin ATPase activity\n(11, 25). PKA phosphorylation of cTnI also decreases myofilament Ca2\nsensitivity and results in accelerated myocyte relaxation (27, 28), and\nsubstitution of Ser23/Ser24 with Asp to mimic phosphorylation acceler-\nates relaxation in transgenic mice (39). However, PKC phosphorylates\nSer43/Ser45 and Thr144 in addition to the Ser23/Ser24 cluster (11). More\nrecent studies in which the Ser23/Ser24, Ser43/Ser45, and Thr144 clusters\nare all replaced with Ala in knock-in mice (49, 50) or substituted with\nAsp to mimic phosphorylation in transgenic mice (37) suggest that cTnI\nworks primarily to delay cardiac relaxation during PKC activation.\nThese results are in contrast to results presented here demonstrating an\naccelerated relaxation response attributable to cTnI phosphorylation\n(TABLE ONE), and the in vitro work demonstrating decreased myofil-\nament Ca2 sensitivity (11). Differences in experimental conditions,\nsuch as load and/or temperature in these various models, do not appear\nto account for the divergent functional response. We propose that these\nseemingly divergent responses may be due to a hierarchy of cTnI phos-\nphorylation in response to a given agonist, with different cTnI clusters\nphosphorylated in response to PKC activation. Ser43/Ser45 phosphoryl-\nPKC Phosphorylation of Troponin I and Contractile Function\nation may have a dominant influence on contractile relaxation via its\ninfluence on maximum ATPase activity when all five phosphorylation\nsites are replaced by Ala (49) or replaced with a negatively charged Asp\n(37). However, PKC activation by a specific agonist may differentially\nand/or temporally phosphorylate one or more of the cTnI residues\nrather than phosphorylate all three clusters. In support of this idea, in\nvitro motility assays carried out with cTnISer43/Ser45 containing\ncharged Glu substitutions demonstrated a decrease in myofilament\nCa2 sensitivity and decreased sliding speed, whereas cTnIT144G only\ndecreased myofilament Ca2 sensitivity (51). NMR analysis of Ser43/\nSer45 substituted with Asp provides evidence that phosphorylation of\nthis site may be dominant in that it influences the Ca2 binding affinity\nto TnC as well as protein-protein interactions between TnI and TnC\n(52). Thus, the divergent results are predicted to stem from differential\nphosphorylation of the three cTnI phosphorylation clusters and/or the\nrelative dominance of one cluster in response to activation and translo-\ncation of specific PKC isoforms by a given agonist. This idea will be\nimportant for rigorous testing, with a variety of agonists under physio-\nlogical and pathophysiological conditions, to better understand the\nPKC signaling pathway and its ability to modulate both cTnI phospho-\nrylation and contractile performance.\nRole of Individual cTnI-phosphorylated Residues in the Contractile\nResponse to ET—Detailed studies on purified cTnI support the idea that\nthe residue phosphorylated by PKC is important for the resulting func-\ntional relaxation response. Phosphorylation of Thr144 in cTnI produces\na rightward shift in myofilament Ca2 sensitivity for both actomyosin\nATPase activity (11) and force generation in permeabilized trabeculae\n(51, 53). The anticipated cellular response is an increase in myocyte\nrelaxation rate. Our results now show that substitution of Thr144 with\nPro significantly delays relaxation during the ET response (Fig. 4,\nTABLE THREE). The Thr144 site is particularly important because the\ninhibitory peptide region encompassing this phosphorylation target\n(amino acids 129 –149) is important for determining TnI binding to\nactin in the absence of Ca2 and diminished binding to actin in the\npresence of Ca2 (54). Thus, the inhibitory peptide region is important\nfor the molecular switch activity of TnI (55, 56). The delayed relaxation\nresponse to ET in myocytes expressing cTnIT144P compared with con-\ntrol myocytes is consistent with the idea that phosphorylation of this\nresidue in response to ET plays a significant role in the accompanying\nrelaxation rate response.\nExperiments with the phospho-cTnISer23/Ser24 antibody and with\ncTnIS23A/S24A also demonstrate a key role for this cluster during\nmore extended activation of PKC by ET. Phosphorylation of Ser23/Ser24\ndecreases myofilament Ca2 sensitivity and accelerates relaxation in\nresponse to PKA activation (27, 28), and similar shifts in myofilament\nCa2 sensitivity are reported in response to PKC phosphorylation of\nSer23/Ser24 (11). Our results now demonstrate a temporal pattern of\nSer23/Ser24 phosphorylation that is important for enhanced relaxation\nin response to longer intervals of ET in intact myocytes (Fig. 6). In\nbiochemical experiments with purified cTnI, longer treatment with\nPKC also was required for Ser23/Ser24 phosphorylation (11), yet it\nremained unclear whether this result indicated that there was temporal\nphosphorylation in the intact cell or instead indicated that Ser23/Ser24\nwas not a typical in vivo target for activated PKC. Results with myocytes\nexpressing cTnIS23A/S24A (Fig. 6) clearly demonstrate that there is\ntemporal phosphorylation of this site by PKC activated in response to\nET. Several PKC agonists, including ET, are released in a paracrine/\nautocrine pattern (57, 58), and phosphorylation of Ser23/Ser24 during\nmore prolonged intervals of PKC stimulation would then be important\nfor preventing delays in cellular relaxation. Taken together, the results\nobtained in experiments to investigate Thr144 and Ser23/Ser24 provide\nevidence that both clusters are important in maintaining relaxation dur-\ning PKC activation. The relative importance, as well as the interactive\neffects, of these sites and the Ser43/Ser45 site in response to PKC ago-\nnists, which appears primarily to decrease peak contraction and slow\nrelaxation (51), will be important to investigate in the future for a com-\nplete understanding of the role played by cTnI phosphorylation in mod-\nulating cardiac function.\nAcknowledgments—We thank Andrea Borton (Cell Signaling Inc.) for the gift\nof the phospho-Ser23/Ser24 antibody. We also thank Ayako Ohata for technical\nassistance, and Sharlene Day, Joseph Metzger, and Daniel Michele for helpful\ncomments on earlier versions of this manuscript.",
         "EXPERIMENTAL PROCEDURES\nMutagenesis Strategy—Wild-type rat cTnI cDNA (kind gift of A.\nMurphy) was used to construct the cTnIT144P and cTnIS23A/S24A\nmutants. Primers used for mutagenesis of cTnIT144P were 5-gcaagtt-\ntaagcggccgcctctccgaagagtg-3 (sense) and 5-cactcttcggagaggcggccgct-\ntaaacttgc-3 (antisense), and for cTnIS23A/S24A the primers were\n5-ccgacgtcgcgccgcggccaactaccg-3 (sense) and 5-cggtagttggccgcg-\ngcgcgacgtcgg-3 (antisense) using a pGEM-3Z vector and the Strat-\nagene QuikChange site-directed mutagenesis kit (29). The presence of\neach mutant was verified by DNA sequencing.\nGeneration of Adenoviral Vectors—Recombinant adenovirus vectors\nwere constructed by cotransfection of pCA4cTnI cDNA with pJM17 by\ncalcium phosphate into a HEK 293 cell\nline. Replication-deficient,\nrecombinant adenovirus containing wild-type rat cTnI, wild-type\nssTnI, or mutant ssTnI cDNAs, a cytomegalovirus promoter, and an\nSV40 polyadenylation signal were obtained following homologous\nrecombination (19).\nMyocyte Isolation—Adult rat ventricular myocytes were isolated as\ndescribed earlier (4, 30). Myocytes were plated on laminin-coated cov-\nerslips in Dulbecco’s modified Eagle’s medium supplemented with 50\nunits/ml penicillin, 50 g/ml streptomycin (pen/strep) and 5% serum\nfor 2 h prior to replacing the medium with serum-free Dulbecco’s mod-\nified Eagle’s medium containing recombinant adenovirus vector (mul-\ntiplicity of infection  500). Gene transfer of exogenous TnI at a mul-\ntiplicity of infection of 500 results in virtually all cells expressing and\nincorporating exogenous TnI into the myofilament (29). After 1 h, Dul-\nbecco’s modified Eagle’s medium with pen/strep (2 ml) was added, with\nregular replacement of media over 4 days in culture.\nAnalysis of Protein Expression—Control and virus-treated adult rat\ncardiac myocytes were scraped from coverslips into sample buffer 4\ndays after gene transfer (31). In some experiments, myocytes were per-\nmeabilized in 0.1% Triton X-100 for 1 min to evaluate incorporation of\nTnI protein into the myofilament after gene transfer. In earlier work,\nexpression in intact and permeabilized myocytes was comparable when\nmyofilament proteins were incorporated into the myofilament (29, 32).\nProteins were then separated by gel electrophoresis and blotted onto\npolyvinylidene difluoride membrane for 2000 V-h with immunodetec-\ntion carried out as described earlier (30) using a 1:1000 dilution of MAB\n1691 (Chemicon Inc., Temecula, CA), a monoclonal antibody that rec-\nognizes all TnI isoforms. A phospho-Ser23/Ser24-specific polyclonal\nantibody (1:1000; Cell Signaling Technology, Beverly, MA) and a paired\nhorseradish peroxidase-conjugated secondary antibody were used to\ndetect Ser23/Ser24 phosphorylation. Myocytes collected for detection of\nphosphorylation were incubated at 37 °C with calyculin A (10 nM) alone\nor with the selected PKA and PKC agonist and/or inhibitor for time\nintervals ranging from 2.5 min up to 1 h.\nMeasurement of Sarcomere Shortening in Single Cardiac Myocytes—\nMyocytes to be used for shortening assays were transferred to a Plexiglas\nstimulation chamber consisting of 8 wells containing platinum elec-\ntrodes. Myocytes were stimulated electrically (2.5-ms pulse, 0.2 Hz) in\nmedium 199 supplemented with pen/strep (M199), 10 mM HEPES, 0.2\nmg/ml bovine serum albumin, and 10 mM glutathione (M199), and\nthe media was replaced every 12 h for days 2– 4 post-gene transfer (33).\nThe voltage was set so that 20% of the myocytes were stimulated on\neach coverslip. Four days after gene transfer, individual coverslips were\ntransferred to a temperature-controlled (37 °C) chamber mounted on a\nNikon microscope stage. A video-based detection system (Ionoptix,\nMilton, MA) was used to detect sarcomere shortening in intact myo-\ncytes. For frequency-response studies, myocytes were stimulated at\neach frequency (0.2, 0.5, 0.75, 1, and 2 Hz) until a steady state contrac-\ntion was reached. Sarcomere shortening also was recorded at 0.2 Hz\nbefore and after the increases in stimulation frequency to ensure there\nwere no changes in the rate of contraction, peak contraction, and rate of\nrelaxation after testing at all frequencies. Experiments were recorded\nusing SarcLen software, and an average of 10 twitches/myocyte was\ncollected under basal conditions. Signal-averaged data were analyzed to\ndetermine resting sarcomere length (SLR), percent change in peak sar-\ncomere shortening above base line (Sb), time to peak shortening (TTP),\ntime to 25, 50, and 75% relaxation (TTR25, TTR50, and TTR75, respec-\ntively), and maximum normalized shortening and relaxation velocities\n(dl/dtmax, dl/dtmax, respectively). Media containing ET (10 nM),\n5-N(ethyl-N-iospropyl)amiloride (EIPA, 500 nM) or both reagents were\nperfused via separate individual perfusion tubing to eliminate introduc-\ntion of residual agonists/inhibitors.\nExperiments for Ca2 transient and sarcomere shortening measure-\nments were carried out on myocytes loaded with Fura-2 AM, as\ndescribed previously (34). Briefly, myocytes were loaded for 5 min with\n5 M Fura-2 AM (final Me2SO  0.6%) at 37 °C followed by a 5-min\nwash-out for de-esterification. Myocytes were then stimulated at 0.2 Hz\nat 37 °C, and the base-line sarcomere length and Ca2 transient were\nrecorded from a single cell. Recordings were then made 5, 10, and 15\nmin after the addition of ET (10 nM), with the cell maintained in the dark\nbetween sampling times. Optimal cTnI phosphorylation was observed\nin response to 10 –100 nM ET in an earlier study (4). Ten twitches/\nmyocyte were collected for each sample. In addition to sarcomere short-\nening measurements, the resting and peak ratios, maximum Ca2 tran-\nsient rate (dCa/dtmax), maximum Ca2 decay rate (dCa/dtmax), and\ntime to 50 and 75% decay (TTD50, TTD75, respectively) in the Ca2\ntransient were measured. The Ca2 transient and shortening measure-\nments were maintained over this same time interval in myocytes main-\ntained under basal conditions.\nStatistical Analysis—Values are expressed as mean  S.E. Grouped\ncomparisons are analyzed using a Student’s t test or analysis of variance\ntesting (ANOVA). Statistical differences detected by ANOVA are com-\npared with a Newman-Keuls multiple comparison test with p  0.05\nconsidered significantly different.",
         "Cardiac troponin I is a phosphorylation target for endothelin-\nactivated protein kinase C. Earlier work in cardiac myocytes\nexpressing nonphosphorylatable slow skeletal troponin I provided\nevidence that protein kinase C-mediated cardiac troponin I phos-\nphorylation accelerates relaxation. However, replacement with the\nslow skeletal isoform also alters the myofilament pH response and\nthe Ca2 transient, which could influence endothelin-mediated\nrelaxation. Here, differences in the Ca2 transient could not explain\nthe divergent relaxation response to endothelin in myocytes\nexpressing cardiac versus slow skeletal troponin I nor could activa-\ntion of Na/H exchange. Three separate clusters within cardiac\ntroponin I are phosphorylated by protein kinase C, and we set out to\ndetermine the contribution of the Thr144 and Ser23/Ser24 clusters to\nthe endothelin-mediated contractile response. Myocyte replace-\nment with a cardiac troponin I containing a Thr144 substituted with\nthe Pro residue found in slow skeletal troponin I resulted in pro-\nlonged relaxation in response to acute endothelin compared with\ncontrol myocytes. Ser23/Ser24 also is a target for protein kinase C\nphosphorylation of purified cardiac troponin I, and although this\ncluster was not acutely phosphorylated in intact myocytes, signifi-\ncant phosphorylation developed within 1 h after adding endothelin.\nReplacement of Ser23/Ser24 with Ala indicated that this cluster con-\ntributes significantly to relaxation during more prolonged endothe-\nlin stimulation. Overall, results with these mutants provide evi-\ndence that Thr144 plays an important role in the acute acceleration\nof relaxation, whereas Ser23/Ser24 contributes to relaxation during\nmore prolonged activation of protein kinase C by endothelin.\nProtein kinase C (PKC)2 activation is an important pathway involved\nin modulating cardiac contractile function (1–5). A number of end tar-\nget proteins involved in excitation-contraction coupling are phospho-\nrylated in response to PKC activation, including the L-type Ca2 chan-\nnel (6, 7) and myofilament proteins such as cardiac troponin I (cTnI)\n(8 –11), cardiac troponin T (10, 12), and myosin light chain 2 (MLC2) (8,\n13, 14). The contribution of each end target to the contractile response\nto PKC is not well understood. Our laboratory is interested in under-\nstanding the role of cTnI in the PKC-mediated contractile function\nresponse. Increased PKC expression is observed in failing human hearts\n(15, 16), and the relationship between PKC and cTnI phosphorylation\nmay exert an important influence on cardiac function under physiolog-\nical and pathophysiological conditions (17).\n* The costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked “advertisement” in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\nEarlier work demonstrated that cTnI phosphorylation is correlated\nwith the positive inotropic response to ET (4, 8, 18). Accelerated relax-\nation also is observed in response to PKC activation by ET in myocytes\nexpressing cTnI (4). PKC phosphorylation of purified cTnI decreases\nmyofilament Ca2 sensitivity in reconstituted myofilaments (11), and\nthis desensitization is expected to contribute to the accelerated relax-\nation. In contrast, peak shortening is diminished and relaxation is\ndelayed rather than accelerated in response to ET when endogenous\ncTnI is replaced with nonphosphorylatable slow skeletal troponin I\n(ssTnI) in adult myocytes. These results are consistent with the idea that\ncTnI phosphorylation enhances peak myocyte shortening and acceler-\nates relaxation rate in response to PKC activation by ET. However,\nssTnI expression in adult myocytes also increases myofilament Ca2\nsensitivity and slows the Ca2 transient (19, 20). Thus, the differential\nresponse to ET observed in myocytes expressing cTnI versus ssTnI may\nresult from the direct actions of cTnI phosphorylation on shortening\nand relaxation or from alterations in the Ca2 transient induced by\nssTnI expression.\nOne goal of the present study is to establish whether PKC-mediated\ncTnI phosphorylation acts directly to accelerate relaxation or is\nexplained by parallel alterations in cellular Ca2. Calcium and shorten-\ning responses are measured simultaneously during the ET response for\nthese experiments. Stimulation frequency also alters the Ca2 transient,\nwith accelerated sarcoplasmic reticulum Ca2 uptake observed at\nhigher pacing frequencies (21). The relative contribution of phospho-\nrylated cTnI to the ET-induced relaxation response may decrease as\npacing frequency increases in the event that alterations in cellular Ca2\nare primarily responsible for this relaxation response. Thus, the contri-\nbution of cTnI phosphorylation to the ET-induced relaxation response\nis examined over a range of frequencies.\nA second goal of our study was to differentiate between cTnI phos-\nphorylation-induced and alkalosis-induced changes in the ET-mediated\ncontractile performance response. Previous investigators showed that\nET increases Na/H exchange via PKC (22, 23). The ensuing alkalosis\nincreases myofilament Ca2 sensitivity (24), and ET-induced alkalosis\nhas been shown to contribute to the enhanced peak shortening and\nincreased rate of contraction (22, 23). The diminished peak-shortening\nresponse to ET observed in ssTnI-expressing myocytes (4) also could\nresult from the enhanced basal myofilament Ca2 sensitivity and dimin-\nished pH sensitivity relative to cTnI-expressing myocytes (19). In the\npresent study, the influence of ET-induced alkalosis on peak contrac-\ntion was tested by inhibiting Na/H exchange in myocytes expressing\ncTnI and ssTnI.\nThe final aim of the present work was to determine the contribution\nof individual phosphorylation sites within cTnI to the PKC-mediated\nchanges in intact adult myocyte relaxation rate in response to ET.\nDetailed work on purified cTnI indicates that PKC phosphorylates three\nseparate clusters of residues, including Ser23/Ser24, Ser43/Ser45, and\nThr144 within cTnI (11, 25). However, the myocardial response to phos-\nphorylation of individual sites within the intact myocyte has yet to be\nPKC Phosphorylation of Troponin I and Contractile Function\nthoroughly investigated. Studies here are focused on investigating the\ninfluence of the rat cTnI Ser23/Ser24 and Thr144 sites on contractile\nfunction in response to ET. For studies of cTnIThr144, the goal is to\ndetermine whether Thr144 influences the peak contraction and relax-\nation response to PKC activated by ET, by substitution with the non-\nphosphorylatable Pro residue. The Ser23/Ser24 cluster also is a target for\nPKC in purified cTnI and is a major myofilament target for protein\nkinase A (PKA) (26). Phosphorylation of this cluster by PKA signifi-\ncantly decreases myofilament Ca2 sensitivity (27) and accelerates\nrelaxation during -adrenergic activation (28). A similar increase in\nrelaxation is expected if this Ser cluster is phosphorylated by PKC in\nintact adult myocytes. For these studies, the Ser23/Ser24 residues were\nsubstituted with Ala to investigate whether this cluster influences the\nability of cTnI to maintain relaxation in response to ET activation of\nPKC.\nEXPERIMENTAL PROCEDURES\nMutagenesis Strategy—Wild-type rat cTnI cDNA (kind gift of A.\nMurphy) was used to construct the cTnIT144P and cTnIS23A/S24A\nmutants. Primers used for mutagenesis of cTnIT144P were 5-gcaagtt-\ntaagcggccgcctctccgaagagtg-3 (sense) and 5-cactcttcggagaggcggccgct-\ntaaacttgc-3 (antisense), and for cTnIS23A/S24A the primers were\n5-ccgacgtcgcgccgcggccaactaccg-3 (sense) and 5-cggtagttggccgcg-\ngcgcgacgtcgg-3 (antisense) using a pGEM-3Z vector and the Strat-\nagene QuikChange site-directed mutagenesis kit (29). The presence of\neach mutant was verified by DNA sequencing.\nGeneration of Adenoviral Vectors—Recombinant adenovirus vectors\nwere constructed by cotransfection of pCA4cTnI cDNA with pJM17 by\ncalcium phosphate into a HEK 293 cell\nline. Replication-deficient,\nrecombinant adenovirus containing wild-type rat cTnI, wild-type\nssTnI, or mutant ssTnI cDNAs, a cytomegalovirus promoter, and an\nSV40 polyadenylation signal were obtained following homologous\nrecombination (19).\nMyocyte Isolation—Adult rat ventricular myocytes were isolated as\ndescribed earlier (4, 30). Myocytes were plated on laminin-coated cov-\nerslips in Dulbecco’s modified Eagle’s medium supplemented with 50\nunits/ml penicillin, 50 g/ml streptomycin (pen/strep) and 5% serum\nfor 2 h prior to replacing the medium with serum-free Dulbecco’s mod-\nified Eagle’s medium containing recombinant adenovirus vector (mul-\ntiplicity of infection  500). Gene transfer of exogenous TnI at a mul-\ntiplicity of infection of 500 results in virtually all cells expressing and\nincorporating exogenous TnI into the myofilament (29). After 1 h, Dul-\nbecco’s modified Eagle’s medium with pen/strep (2 ml) was added, with\nregular replacement of media over 4 days in culture.\nAnalysis of Protein Expression—Control and virus-treated adult rat\ncardiac myocytes were scraped from coverslips into sample buffer 4\ndays after gene transfer (31). In some experiments, myocytes were per-\nmeabilized in 0.1% Triton X-100 for 1 min to evaluate incorporation of\nTnI protein into the myofilament after gene transfer. In earlier work,\nexpression in intact and permeabilized myocytes was comparable when\nmyofilament proteins were incorporated into the myofilament (29, 32).\nProteins were then separated by gel electrophoresis and blotted onto\npolyvinylidene difluoride membrane for 2000 V-h with immunodetec-\ntion carried out as described earlier (30) using a 1:1000 dilution of MAB\n1691 (Chemicon Inc., Temecula, CA), a monoclonal antibody that rec-\nognizes all TnI isoforms. A phospho-Ser23/Ser24-specific polyclonal\nantibody (1:1000; Cell Signaling Technology, Beverly, MA) and a paired\nhorseradish peroxidase-conjugated secondary antibody were used to\ndetect Ser23/Ser24 phosphorylation. Myocytes collected for detection of\nphosphorylation were incubated at 37 °C with calyculin A (10 nM) alone\nor with the selected PKA and PKC agonist and/or inhibitor for time\nintervals ranging from 2.5 min up to 1 h.\nMeasurement of Sarcomere Shortening in Single Cardiac Myocytes—\nMyocytes to be used for shortening assays were transferred to a Plexiglas\nstimulation chamber consisting of 8 wells containing platinum elec-\ntrodes. Myocytes were stimulated electrically (2.5-ms pulse, 0.2 Hz) in\nmedium 199 supplemented with pen/strep (M199), 10 mM HEPES, 0.2\nmg/ml bovine serum albumin, and 10 mM glutathione (M199), and\nthe media was replaced every 12 h for days 2– 4 post-gene transfer (33).\nThe voltage was set so that 20% of the myocytes were stimulated on\neach coverslip. Four days after gene transfer, individual coverslips were\ntransferred to a temperature-controlled (37 °C) chamber mounted on a\nNikon microscope stage. A video-based detection system (Ionoptix,\nMilton, MA) was used to detect sarcomere shortening in intact myo-\ncytes. For frequency-response studies, myocytes were stimulated at\neach frequency (0.2, 0.5, 0.75, 1, and 2 Hz) until a steady state contrac-\ntion was reached. Sarcomere shortening also was recorded at 0.2 Hz\nbefore and after the increases in stimulation frequency to ensure there\nwere no changes in the rate of contraction, peak contraction, and rate of\nrelaxation after testing at all frequencies. Experiments were recorded\nusing SarcLen software, and an average of 10 twitches/myocyte was\ncollected under basal conditions. Signal-averaged data were analyzed to\ndetermine resting sarcomere length (SLR), percent change in peak sar-\ncomere shortening above base line (Sb), time to peak shortening (TTP),\ntime to 25, 50, and 75% relaxation (TTR25, TTR50, and TTR75, respec-\ntively), and maximum normalized shortening and relaxation velocities\n(dl/dtmax, dl/dtmax, respectively). Media containing ET (10 nM),\n5-N(ethyl-N-iospropyl)amiloride (EIPA, 500 nM) or both reagents were\nperfused via separate individual perfusion tubing to eliminate introduc-\ntion of residual agonists/inhibitors.\nExperiments for Ca2 transient and sarcomere shortening measure-\nments were carried out on myocytes loaded with Fura-2 AM, as\ndescribed previously (34). Briefly, myocytes were loaded for 5 min with\n5 M Fura-2 AM (final Me2SO  0.6%) at 37 °C followed by a 5-min\nwash-out for de-esterification. Myocytes were then stimulated at 0.2 Hz\nat 37 °C, and the base-line sarcomere length and Ca2 transient were\nrecorded from a single cell. Recordings were then made 5, 10, and 15\nmin after the addition of ET (10 nM), with the cell maintained in the dark\nbetween sampling times. Optimal cTnI phosphorylation was observed\nin response to 10 –100 nM ET in an earlier study (4). Ten twitches/\nmyocyte were collected for each sample. In addition to sarcomere short-\nening measurements, the resting and peak ratios, maximum Ca2 tran-\nsient rate (dCa/dtmax), maximum Ca2 decay rate (dCa/dtmax), and\ntime to 50 and 75% decay (TTD50, TTD75, respectively) in the Ca2\ntransient were measured. The Ca2 transient and shortening measure-\nments were maintained over this same time interval in myocytes main-\ntained under basal conditions.\nStatistical Analysis—Values are expressed as mean  S.E. Grouped\ncomparisons are analyzed using a Student’s t test or analysis of variance\ntesting (ANOVA). Statistical differences detected by ANOVA are com-\npared with a Newman-Keuls multiple comparison test with p  0.05\nconsidered significantly different.\nRESULTS\nRelationship between Ca2 Transient and Contractile Response to\nEndothelin—Sarcomere shortening and Ca2 transients were measured\nbefore and after ET in Fura-2-loaded myocytes expressing cTnI or ssTnI\n4 days after gene transfer (Fig. 1, TABLE ONE). The increased peak\nshortening and enhanced relaxation rate observed in response to 10 nM\nET in earlier work on myocytes expressing cTnI (4), was similarly\nPKC Phosphorylation of Troponin I and Contractile Function\nobserved in myocytes loaded with Fura-2 (Fig. 1A). In myocytes\nexpressing nonphosphorylatable ssTnI, the delayed relaxation and loss\nof the enhanced peak shortening observed in response to ET (4) simi-\nlarly developed in Fura-2-loaded cells (Fig. 1B, TABLE ONE).\nCa2 transient measurements also were examined in these myocytes.\nThe time to 50% decay for basal Ca2 transients reportedly slows in\ntransgenic mouse myocytes expressing ssTnI (20), and similar results\ndeveloped in rat myocytes expressing ssTnI (TABLE TWO). Peak Ca2\nand the TTP remained unchanged in the ssTnI-expressing myocytes\ncompared with values observed in controls at 0.2 Hz. The Ca2 tran-\nsient change observed in response to ET was then analyzed in myocytes\nexpressing cTnI and ssTnI (TABLE ONE). The percent change in peak\nCa2 amplitude and maximum rate of Ca2 decay (dCa/dtmax) in\nresponse to ET was not significantly changed in myocytes expressing\neither cTnI or ssTnI (Fig. 1, A and B; TABLE ONE). However, signifi-\ncant delays were observed in the time to 75% Ca2 decay in both groups\nof myocytes. The similar Ca2 transient responses to ET coupled with\nthe divergent shortening response of cTnI- versus ssTnI-expressing\nmyocytes indicates that cTnI contributes significantly to the enhanced\npeak shortening and accelerated relaxation rate observed in response to\nPKC activation by ET compared with myocytes expressing ssTnI.\nAdditional studies were performed to investigate whether the contri-\nbution of phosphorylated TnI to the ET-mediated change in contractile\nfunction is diminished with higher frequencies. The shortening  fre-\nquency relationship described previously in rodents (33, 35) was similar\nunder basal conditions and in response to ET over a range of frequencies\n(0.2–2 Hz), (TABLE THREE) in control myocytes. The percent decrease\nin peak shortening, dl/dtmax and dl/dtmax in response to increased\nfrequency was similar before and after ET. The consistency of this\nresponse provides evidence that the contribution of cTnI phosphoryla-\ntion to contractile function is not altered by frequency-induced changes\nin the Ca2 cycle.\nContribution of Cellular Alkalosis to the Myofilament Response to\nEndothelin—An important component of the contractile response to\nET depends on activation of Na/H exchange and the generation of a\ncellular alkalosis (22, 23). Experiments were carried out in the presence\nof the amiloride analog EIPA, an inhibitor of Na/H exchange (23, 36),\nto identify responses resulting from alkalosis. The myocyte response to\nEIPA is comparable in nontreated myocytes and myocytes expressing\ncTnI after gene transfer (results not shown); the pooled results are\nshown in Fig. 2. Myocyte shortening amplitude in myocytes expressing\nPKC Phosphorylation of Troponin I and Contractile Function\ncTnI decreased slightly after the addition of EIPA but did not signifi-\ncantly influence peak amplitude in myocytes expressing ssTnI (Fig. 2, A\nand B). Resting sarcomere length (SLR), dl/dtmax, and dl/dtmax\n(dl/dt and dl/dt in Fig. 2, respectively) were not significantly altered\nby EIPA in myocytes expressing cTnI or ssTnI (Fig. 2B). However, EIPA\nblocked a major portion (70%) of the ET-induced increase in peak\nshortening amplitude observed in cTnI-expressing myocytes. In con-\ntrast, EIPA did not significantly modify the accelerated relaxation\nresponse to ET in myocytes expressing cTnI, and the delayed relaxation\nobserved in response to ET was comparable with and without EIPA in\nmyocytes expressing ssTnI (Figs. 1 and 2). These results support the idea\nthat Na/H exchange and the ensuing alkalosis is important for the\nenhanced peak shortening response, whereas cTnI plays a more critical\nrole in the accelerated relaxation rate response to ET.\nRole of cTnI Phosphorylation Residues in the Endothelin Response—\nTo determine the role of the cTnIThr144 phosphorylation site in the\nrelaxation response, the ssTnI Pro residue was substituted for Thr144 to\nform cTnIT144P, and replacement was measured 4 days after gene\ntransfer. The stoichiometric replacement of cTnI with cTnIT144P-\nFLAG was 46  2% (n  13) and comparable with the 53  2% (n  14)\nreplacement with cTnI-FLAG 4 days after gene transfer (Fig. 3).\nReplacement of nontagged cTnI and cTnIT144P is expected to be com-\nparable in myocytes over the same time interval (29, 37). Total TnI\nexpression remained constant in myocytes expressing cTnI and\ncTnIT144P based on the ratio of TnI intensity to the intensity of a\nsilver-stained band on the gel (cTnI set at 1.0; cTnIT144P  1.13  0.10,\nn  11), and isoform expression of tropomyosin and troponin T was\ncomparable in myocytes expressing cTnI and cTnIT144P (results not\nshown). Expression of total TnI also was comparable in intact (I) and\npermeabilized (P) myocytes (Fig. 3, I/P ratio for cTnI  1.10  0.08, n \n5; cTnIT144P  1.05  0.06, n  5), which is consistent with myofila-\nment incorporation of the delivered TnI.\nMeasures of basal contractile function in myocytes expressing\ncTnIT144P are not significantly different from values obtained from\nmyocytes expressing cTnI (TABLE TWO, Fig. 4A). These results are in\nagreement with the earlier reverse substitution of Pro110 with Thr in fast\nskeletal TnI (38). Base-line Ca2 transients in myocytes expressing\ncTnIT144P (Fig. 4B) also showed no significant differences from control\nvalues. Peak shortening increased significantly in response to ET in\ncTnIT144P-expressing myocytes (Fig. 4) and was comparable with the\nET-induced enhanced peak shortening in control myocytes (Fig. 1B).\nHowever, several indices of relaxation, including dl/dtmax and TTR75,\nwere significantly delayed in response to ET in myocytes expressing\ncTnIT144P compared with controls (TABLE ONE). The amplitude of\nthe Ca2 transient response to ET was not significantly different com-\npared with control values for myocytes expressing cTnIT144P (TABLE\nONE, Fig. 4). As with control myocytes, a frequency change from 0.2 to\n2 Hz produced similar changes before and after ET in myocytes express-\ning cTnIT144P (results not shown). The major difference between myo-\ncytes expressing cTnIT144P versus cTnI was the decrease in relaxation\nrate in response to ET; this finding indicates a specific role for Thr144 in\naccelerating relaxation in response to ET.\nThe contribution of Ser23/Ser24 phosphorylation to the ET-induced\nchange in contractile function also was determined. A phospho-specific\nSer23/Ser24 antibody was used for these studies. Myocytes were incu-\nbated with the PKA agonist dobutamine (DOB) for 2 min to verify\ndetection of the well documented PKA-mediated cTnISer23/Ser24 phos-\nPKC Phosphorylation of Troponin I and Contractile Function\nphorylation site (39, 40) (Fig. 5A). Detection of cTnI phosphorylation by\nthis antibody was specific, as it was blocked when, in addition to DOB,\nthe -adrenergic inhibitor propranolol was added to myocytes (results\nnot shown). There was no significant phosphorylation detected in myo-\ncytes incubated with ET for 10 min with (Fig. 5A) or without the PKC\ninhibitor bis-indolylmaleimide-1 (bis-1, results not shown). However,\nsignificant phosphorylation of Ser23/Ser24 developed in control myo-\ncytes over a 1-h time span, and this ET-mediated phosphorylation was\nsignificantly reduced in the presence of the PKC inhibitor bis-1 but not\nby the -adrenergic antagonist propranolol (Fig. 5A). Treatment of\nmyocytes with other PKC agonists, including phenylephrine and phor-\nbol 12-myristate 13-acetate for 1 h also resulted in significant cTnI\nphosphorylation of Ser23/Ser24. Incubation with fibroblast growth fac-\ntor-2 was previously shown to activate PKC (41), yet this agonist did not\nsignificantly influence cTnI phosphorylation over a 1-h time span\n(results not shown). These results provide evidence for temporal phos-\nphorylation of cTnI Ser23/Ser24 in response to PKC activation by several\nPKC agonists.\nTo directly determine the contribution of Ser23/Ser24 to the contract-\nile function response to ET, cTnIS23A/S24A was expressed in adult\nmyocytes. By 4 days after gene transfer, 44  5% of cTnI was replaced\nwith the cTnIS23A/S24A (n  15, Fig. 5B). DOB phosphorylation of\ncTnI was significantly reduced in myocytes expressing cTnIS23A/S24A\nor cTnIS23A/S24A-FLAG (results not shown). In addition, Ser23/Ser24\nphosphorylation was significantly attenuated after 1 h of ET in myocytes\nexpressing tagged and nontagged cTnIS23A/S24A (Fig. 5C). This S23A/\nS24A substitution has previously been shown to have little influence on\nbasal myofilament function (42), and no significant differences were\nobserved in SLR, peak amplitude, dl/dtmax, ordl/dt max in myocytes\nexpressing cTnIS23A/S24A compared with controls (results not\nshown). Control myocytes expressing cTnI or\nthe substituted\ncTnIS23A/S24A responded similarly to ET over the first 15 min (Fig. 6).\nHowever, a significant difference in the response was observed 1 h after\nthe addition of ET in myocytes expressing cTnIS23A/S24A compared\nPKC Phosphorylation of Troponin I and Contractile Function\nwith control values (Fig. 6). Specifically, peak shortening amplitude was\nnot increased and both dl/dtmax and TTR75 indicated relaxation was\nsignificantly prolonged in myocytes expressing cTnIS23A/S24A com-\npared with control myocytes and were comparable with the results\nobtained in myocytes expressing ssTnI (Fig. 6). These results demon-\nstrate Ser23/Ser24 phosphorylation contributes to the accelerated relax-\nation during more prolonged myocyte PKC activation by ET.\nDISCUSSION\nResults from the present study demonstrate the important direct role\nfor cTnI phosphorylation in accelerating relaxation in response to PKC\nactivated by ET in adult myocytes (Figs. 1–3). Our results now demon-\nstrate a clear role for cTnI Thr144 and Ser23/Ser24 in contributing to the\naccelerated relaxation in response to ET (Figs. 4 and 6). Substitution\nwith a Pro residue at Thr144 results in the delayed relaxation previously\nobserved in myocytes expressing nonphosphorylatable ssTnI and pro-\nvides evidence that the Thr144 site contributes to accelerated relaxation\nduring the ET response. The Ser23/Ser24 cluster does not appear to\ncontribute to the initial response to ET but, instead, is important for the\naccelerated relaxation during more prolonged stimulation by ET. This\nfinding indicates that the relaxation rate response to ET-induced PKC\npathway activation depends on temporal phosphorylation of individual\nclusters within cTnI.\nTnI Phosphorylation Plays a Direct Role in the Relaxation Response to\nEndothelin—Results from the present study demonstrate the important\nrole of cTnI phosphorylation in the relaxation response during the pos-\nitive inotropic response to ET, independent of cellular Ca2 handling.\nThe only aspect of Ca2 transient that changed significantly in response\nto 10 nM ET was the percent change in TTD75, which decreased in both\ncontrol and ssTnI-expressing myocytes (Fig. 1, TABLE ONE). All other\nmeasurements of the Ca2 transient remained unchanged during the\nET response, and work by other laboratories supports this finding (43,\n44). There are reports that ET activation of PKC enhances the Ca2\ntransient at higher doses of ET (45), which may be due to increased Ca2\nentry via the L-type Ca2 channel (46). However, results from Ebihara et\nal. (47) indicate the increased Ca2 transient is not necessary for the\nenhanced contractile response to ET.\nExperiments with the NHE1 inhibitor EIPA also demonstrate that\ninhibition of NHE1 prevents the ET-mediated enhancement in peak\nshortening amplitude (Fig. 2). ET in the concentration range used here\nactivates PKC-induced NHE1 (48) and causes a cellular alkalosis (22,\n23). The alkalosis-mediated increase in myofilament Ca2 sensitivity\n(24) is responsible for the increased peak contractile shortening (22, 23).\nThis enhancement in peak shortening is absent from myocytes express-\ning ssTnI (Fig. 1), which is likely due to the combination of increased\nbasal myofilament Ca2 sensitivity (19, 20) and a decreased pH response\nto alkalosis in these myocytes compared with cTnI-expressing myocytes\n(19).\nIn contrast to peak shortening, the divergent influence of cTnI versus\nssTnI on the relaxation response to ET was not influenced significantly\nby EIPA (Fig. 2). Although the slowing of relaxation in myocytes\nexpressing ssTnI was attenuated by the presence of EIPA during the ET\nresponse, relaxation was significantly accelerated in myocytes express-\ning cTnI versus ssTnI during this response. Our results now provide\nevidence that phosphorylated cTnI maintains relaxation regardless of\nthe alkalosis. The delayed relaxation response to ET observed in myo-\ncytes expressing cTnIT144P also supports this finding (Fig. 4).\nWhat Role Does cTnI Phosphorylation Play in the Contractile\nResponse to ET?—The role of cTnI phosphorylation in the contractile\nfunction response to PKC agonists remains controversial. Earlier in\nvitro work showed that PKC phosphorylation of cTnI decreases myofil-\nament Ca2 sensitivity as well as maximum actomyosin ATPase activity\n(11, 25). PKA phosphorylation of cTnI also decreases myofilament Ca2\nsensitivity and results in accelerated myocyte relaxation (27, 28), and\nsubstitution of Ser23/Ser24 with Asp to mimic phosphorylation acceler-\nates relaxation in transgenic mice (39). However, PKC phosphorylates\nSer43/Ser45 and Thr144 in addition to the Ser23/Ser24 cluster (11). More\nrecent studies in which the Ser23/Ser24, Ser43/Ser45, and Thr144 clusters\nare all replaced with Ala in knock-in mice (49, 50) or substituted with\nAsp to mimic phosphorylation in transgenic mice (37) suggest that cTnI\nworks primarily to delay cardiac relaxation during PKC activation.\nThese results are in contrast to results presented here demonstrating an\naccelerated relaxation response attributable to cTnI phosphorylation\n(TABLE ONE), and the in vitro work demonstrating decreased myofil-\nament Ca2 sensitivity (11). Differences in experimental conditions,\nsuch as load and/or temperature in these various models, do not appear\nto account for the divergent functional response. We propose that these\nseemingly divergent responses may be due to a hierarchy of cTnI phos-\nphorylation in response to a given agonist, with different cTnI clusters\nphosphorylated in response to PKC activation. Ser43/Ser45 phosphoryl-\nPKC Phosphorylation of Troponin I and Contractile Function\nation may have a dominant influence on contractile relaxation via its\ninfluence on maximum ATPase activity when all five phosphorylation\nsites are replaced by Ala (49) or replaced with a negatively charged Asp\n(37). However, PKC activation by a specific agonist may differentially\nand/or temporally phosphorylate one or more of the cTnI residues\nrather than phosphorylate all three clusters. In support of this idea, in\nvitro motility assays carried out with cTnISer43/Ser45 containing\ncharged Glu substitutions demonstrated a decrease in myofilament\nCa2 sensitivity and decreased sliding speed, whereas cTnIT144G only\ndecreased myofilament Ca2 sensitivity (51). NMR analysis of Ser43/\nSer45 substituted with Asp provides evidence that phosphorylation of\nthis site may be dominant in that it influences the Ca2 binding affinity\nto TnC as well as protein-protein interactions between TnI and TnC\n(52). Thus, the divergent results are predicted to stem from differential\nphosphorylation of the three cTnI phosphorylation clusters and/or the\nrelative dominance of one cluster in response to activation and translo-\ncation of specific PKC isoforms by a given agonist. This idea will be\nimportant for rigorous testing, with a variety of agonists under physio-\nlogical and pathophysiological conditions, to better understand the\nPKC signaling pathway and its ability to modulate both cTnI phospho-\nrylation and contractile performance.\nRole of Individual cTnI-phosphorylated Residues in the Contractile\nResponse to ET—Detailed studies on purified cTnI support the idea that\nthe residue phosphorylated by PKC is important for the resulting func-\ntional relaxation response. Phosphorylation of Thr144 in cTnI produces\na rightward shift in myofilament Ca2 sensitivity for both actomyosin\nATPase activity (11) and force generation in permeabilized trabeculae\n(51, 53). The anticipated cellular response is an increase in myocyte\nrelaxation rate. Our results now show that substitution of Thr144 with\nPro significantly delays relaxation during the ET response (Fig. 4,\nTABLE THREE). The Thr144 site is particularly important because the\ninhibitory peptide region encompassing this phosphorylation target\n(amino acids 129 –149) is important for determining TnI binding to\nactin in the absence of Ca2 and diminished binding to actin in the\npresence of Ca2 (54). Thus, the inhibitory peptide region is important\nfor the molecular switch activity of TnI (55, 56). The delayed relaxation\nresponse to ET in myocytes expressing cTnIT144P compared with con-\ntrol myocytes is consistent with the idea that phosphorylation of this\nresidue in response to ET plays a significant role in the accompanying\nrelaxation rate response.\nExperiments with the phospho-cTnISer23/Ser24 antibody and with\ncTnIS23A/S24A also demonstrate a key role for this cluster during\nmore extended activation of PKC by ET. Phosphorylation of Ser23/Ser24\ndecreases myofilament Ca2 sensitivity and accelerates relaxation in\nresponse to PKA activation (27, 28), and similar shifts in myofilament\nCa2 sensitivity are reported in response to PKC phosphorylation of\nSer23/Ser24 (11). Our results now demonstrate a temporal pattern of\nSer23/Ser24 phosphorylation that is important for enhanced relaxation\nin response to longer intervals of ET in intact myocytes (Fig. 6). In\nbiochemical experiments with purified cTnI, longer treatment with\nPKC also was required for Ser23/Ser24 phosphorylation (11), yet it\nremained unclear whether this result indicated that there was temporal\nphosphorylation in the intact cell or instead indicated that Ser23/Ser24\nwas not a typical in vivo target for activated PKC. Results with myocytes\nexpressing cTnIS23A/S24A (Fig. 6) clearly demonstrate that there is\ntemporal phosphorylation of this site by PKC activated in response to\nET. Several PKC agonists, including ET, are released in a paracrine/\nautocrine pattern (57, 58), and phosphorylation of Ser23/Ser24 during\nmore prolonged intervals of PKC stimulation would then be important\nfor preventing delays in cellular relaxation. Taken together, the results\nobtained in experiments to investigate Thr144 and Ser23/Ser24 provide\nevidence that both clusters are important in maintaining relaxation dur-\ning PKC activation. The relative importance, as well as the interactive\neffects, of these sites and the Ser43/Ser45 site in response to PKC ago-\nnists, which appears primarily to decrease peak contraction and slow\nrelaxation (51), will be important to investigate in the future for a com-\nplete understanding of the role played by cTnI phosphorylation in mod-\nulating cardiac function.\nAcknowledgments—We thank Andrea Borton (Cell Signaling Inc.) for the gift\nof the phospho-Ser23/Ser24 antibody. We also thank Ayako Ohata for technical\nassistance, and Sharlene Day, Joseph Metzger, and Daniel Michele for helpful\ncomments on earlier versions of this manuscript.",
         null
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "Programming: factors that influence success",
         "This paper documents a study, carried out in the academic year 2003-2004, on fifteen factors that may influence performance on a first year object-oriented programming module. The factors included prior academic experience, prior computer experience, self-perception of programming performance and comfort level on the module and specific cognitive skills. The study found that a student's perception of their understanding of the module had the strongest correlation with programming performance, r=0.76, p‹0.01. In addition, Leaving Certificate (LC) mathematics and science scores were shown to have a strong correlation with performance. A regression module, based upon a student's perception of their understanding of the module, gender, LC mathematics score and comfort level was able to account for 79% of the variance in programming performance results.",
         "The impact of prior experience in an information technology programming course sequence Predictors of success and failure in a CS1 course A multi-national, multi-institutional study of assessment of programming skills of first-year CS students The effect of student attributes on success in programming Contributing to success in an introductory computer science course: a study of twelve factors Does it help to have some programming experience before beginning a computing degree program? Factors affecting performance in first-year computing The effect of a constructivist learning environment for field-dependent/independent students on achievement in introductory computer programming What best predicts computer proficiency? Predicting student performance in a beginning computer science class Predicting the success of freshmen in a computer science major Predicting student success in an introductory programming course Predicting success in a first programming course Investigating the relationship between the development of abstract reasoning and performance in an introductory programming class Factors affecting course outcomes in introductory programming An Overview of the Integration of Problem Based Learning into an existing Computer Science Programming Module Discovering the likely performance of novice programmers on an introductory programming module Differential Performance by gender and achievement in the Certificate Examination Results 2000/2001: Final Report, Dublin: National Council for Curriculum and Assessment Examining Gender Cognitive skills needed in computer programming speciﬁc cognitive skills: as measured by an in-house cognitive test previous academic and computer experience: as measured by performance in the Irish Leaving Certiﬁcate (LC) examinations in",
         "http://dl.acm.org/ft_gateway.cfm?id=1047480&type=pdf",
         "ABSTRACT\nThis paper documents a study, carried out in the academic\nyear 2003-2004, on ﬁfteen factors that may inﬂuence per-\nformance on a ﬁrst year object-oriented programming mod-\nule. The factors included prior academic experience, prior\ncomputer experience, self-perception of programming per-\nformance and comfort level on the module and speciﬁc cog-\nnitive skills. The study found that a student’s perception of\ntheir understanding of the module had the strongest corre-\nlation with programming performance, r = 0.76, p < 0.01.\nIn addition, Leaving Certiﬁcate (LC) mathematics and sci-\nence scores were shown to have a strong correlation with\nperformance. A regression module, based upon a student’s\nperception of their understanding of the module, gender, LC\nmathematics score and comfort level was able to account for\n79% of the variance in programming performance results.",
         "INTRODUCTION\nStudent retention on third-level (post high school or equiv-\nalent) Computer Science (CS) and Information Technology\n(IT) courses is a signiﬁcant problem. Students ﬁnd com-\nputer programming diﬃcult and struggle to master the core\nconcepts. A multi-national, multi-institutional study on the\nprogramming skills of ﬁrst year CS students found that stu-\ndents struggled to achieve an average above 30% on assess-\nments administered as part of their study [14]. Further-\nmore, introductory programming modules tend to have a\nPermission to make digital or hard copies of all or part of this work for\npersonal or classroom use is granted without fee provided that copies are\nnot made or distributed for proﬁt or commercial advantage and that copies\nbear this notice and the full citation on the ﬁrst page. To copy otherwise, to\nrepublish, to post on servers or to redistribute to lists, requires prior speciﬁc\npermission and/or a fee.\nSIGCSE’05 February 23–27, 2005 St. Louis, Missouri, USA.\nCopyright2005ACM1-58113-997-7/05/0002...$5.00.\nvery high student to lecturer ratio (100:1 or greater) and\noften lecturers do not know how well students are doing un-\ntil after the ﬁrst assessment. In general the ﬁrst assessment\ndoes not take place until six or eight weeks after the mod-\nule has commenced and given the typically high number of\nstudents, marking the assessments can take a considerable\nlength of time. At this stage, it may be too late to intervene\nto prevent struggling students from failing. Even if inter-\nvention is possible, a lecturer is unlikely to know a student\nwell enough or be able to identify individual student prob-\nlems and therefore recognize the most suitable interventions\nto make.\nThe research documented in this paper is part of a lon-\ngitudinal study on early identiﬁable factors that inﬂuence\nperformance on an introductory programming module.\nIf\nsuch factors can be identiﬁed then it may be possible to de-\nvelop a tool to provide an early diagnosis of a student’s likely\nperformance on a programming module. Interested parties\ncould use the tool to make more informed decisions on ap-\npropriate courses of action and to decide upon personalized\ninterventions that foster a student’s intellectual strengths.\n2. RELATED RESEARCH\nWhile a considerable amount of research has been carried\nout on factors that aﬀect programming performance our in-\nterest is on factors that aﬀect programming performance\non an introductory third-level object-oriented programming\nmodule where such factors can be determined early in the\nacademic year. These factors can be categorized as: (1)\nprevious academic and computer experience, (2) cognitive\nskills, (3) personal information and (4) experience on the\nmodule. A brief review of some research studies in each of\nthese categories is presented next.\nPrevious academic experience and programming experi-\nence have often been cited as predictors of programming suc-\ncess. Several studies have found that mathematical ability\nand exposure to maths courses are important predictors of\nperformance on introductory computer science modules [2,\n3, 4, 6, 13, 19]. Similarly, although less studied, performance\nin and experience of science subjects has also been shown\nto be important [2, 19]. Studies have also found prior pro-\ngramming experience and non-programming computer ex-\nperience (for example, experience of computer applications,\nemailing, game playing and surﬁng the web) to be related\nto programming performance [9, 10, 6, 4, 18].\nThe role of cognitive factors in programming has also re-\nceived research attention. Certain cognitive factors, includ-\ning problem-solving, abstract reasoning, problem translation\nskills, logical ability and cognitive style have been identiﬁed\nas possible predictors [7, 12, 11, 15].\nNumerous studies have been carried out on demographic\nand self-reported personal information. Some studies have\nexamined speciﬁc attributes related to study, for example\npreference for working alone or in a group to solve a pro-\ngramming problem and encouragement or support from oth-\ners to study computers [4, 2, 8].\nIn recent times researchers have examined the relation-\nship between students’ expectations of and experiences on\nan introductory computing module. A positive relationship\nbetween a student’s mental models of programming and self-\neﬃcacy for programming and performance has recently been\nidentiﬁed [18]. The grade a student expected to achieve in\nan introductory module was found to be the most impor-\ntant indicator of performance in another recent study[17].\nA recent longitudinal study found that the most important\npredictor of students’ performance on an introductory com-\nputer science course was comfort level, determined by the\ndegree of anxiety a student felt about the course [4].\nAlthough a considerable number of research studies have\nexamined factors that inﬂuence programming performance,\ncomparisons between the various studies and application of\nthe results are diﬃcult because the studies are carried out\nusing diﬀerent parameters. These parameters include:\n• the type of students (novice to experienced program-\nmers, academic students to employees),\n• the content of the course (some courses are solely pro-\ngramming courses while others are introductory com-\nputing courses),\n• the programming language being taught (older studies\ntend to be based on procedural languages),\n• the educational setting (many of the research studies\nare based on the US educational system) and\n• the reference criterion (for example continuous assess-\nment, end-of-year exam, job performance).\nIn the rest of this paper we describe our study, which\nbuilds upon existing research, in particular the work of [2,\n4], to identify early factors that inﬂuence performance of\nﬁrst year students, on an introductory object-oriented pro-\ngramming module, using the Java programming language.\nThe course is based on the Irish educational system and\nthe reference criterion is the overall result achieved on the\nmodule.\n3. RESEARCH DESIGN\nThe introductory programming module at our university\nis composed of a one and a half hour Problem-Based Learn-\ning (PBL) workshop, a one and a half hour laboratory ses-\nsion and three one-hour lectures per week over two semesters.\nStudents in Ireland do not study programming in secondary\nschool and the majority of students taking this module have\nrecently completed second level education.\nSelection of factors for this study was restricted for a num-\nber of reasons, including availability of participants, length\nof completion time needed for each instrument and stage in\nyear. With this in mind we attempted to examine the re-\nlationship between and the predictability of ﬁfteen factors\nand performance on our introductory module. The factors\nfall into four broad categories:\n1. previous academic and computer experience: as mea-\nsured by performance in the Irish Leaving Certiﬁcate\n(LC) examinations in mathematics and science sub-\njects and self-reported computer experience,\n2. speciﬁc cognitive skills: as measured by an in-house\n3. personal information: gender, age, work-style prefer-\nence, encouragement from others and the number of\nhours per week working at a part-time job and\n4. experience on the module: students own perception of\nhow well they are doing and how comfortable they feel\nwith the module material.\nPerformance on this module is based on continuous assess-\nment (30% of the overall mark) and a ﬁnal examination (70%\nof the overall mark). The measure of performance reported\nupon in this paper is the overall module mark. Continu-\nous assessment and ﬁnal examination marks render similar\nresults and are reported in [1].\n3.1 Participants\nThe study was carried out in the academic year 2003-2004.\nStudents enrolled in the ﬁrst year ‘Introduction to Program-\nming’ module in our department voluntarily participated in\nthis study. Ninety-six students completed the module in the\nacademic year 2003-2004.\n3.2\nInstruments\nTwo instruments were used to collect data: a question-\nnaire and a custom-made cognitive test. The questionnaire\ncollected data on the following items: (1) LC mathematics\ngrade, (2) LC physics grade, (3) LC biology grade, (4) LC\nchemistry grade, (5) highest LC science grade, (6) comfort\nlevel on the module, (7) perceived understanding of the mod-\nule material, (8) prior programming experience, (9) prior\nnon-programming computer experience, (10) work-style pref-\nerence (preference to work-alone or as part of a group), (11)\nencouragement from others to study computer science, (12)\nnumber of hours per week working at a (part-time) job. The\ncognitive test was developed in-house1 and comprised items\ninvolving numerical and letter sequencing, arithmetic rea-\nsoning, problem translation skills and logical ability. In ad-\ndition, information on gender, age and overall module results\nwas available for all students taking the module.\nIn total 80 students (49 male, 31 female) completed the\ncognitive test and 30 (19 male, 11 female) students com-\npleted the survey. Both instruments were completed in the\nsecond semester of the module and data collection for both\nwas paper-based.",
         "4. RESULTS\nAn a priori analysis was carried out to verify no signif-\nicant diﬀerence existed between the mean overall module\nscores of the class and the sample. Test assumptions on\nnormality (Kolmogorov-Smirnov test) and the equality of\nvariance (Levine test) were performed and a t-test on the\noverall results, (t(124) = 0.795, p = 0.428), found no signif-\nicant diﬀerences between the mean scores of the class and\n1Developed by Jacqueline McQuillan, Department of Com-\nputer Science, NUI Maynooth.\nLC\nMaths\nLC\nPhys\nLC\nChem\nLC\nBio\n0.48**\nValues\nMale, Female\nGender\nUnder 23, 23+\nAge\nIndividual, Group\nWork style preference\nEncouragement\nYes, No\nPart-time employment No, Yes\n0.89** 0.88\nFemale only\nr\nn\nMale only\n0.27\nr\nn\n\n** Correlation is signiﬁcant at the 0.01 level (2-tailed).\n* Correlation is signiﬁcant at the 0.05 level (2-tailed).\nthe sample. In the remainder of this section the ﬁndings on\nthe relationship between each of the factors studied and pro-\ngramming performance is presented, followed by an analysis\nof the combination of factors that best predicts performance.\n4.1 Previous academic and computer experi-\nence\nTo establish the relationship between previous academic\nexperience in mathematics and science, the achievable grades\nfor each subject were ranked, with the highest rank given\nto the highest possible grade and the lowest rank given to\nthe lowest possible grade. Table 1. provides the Pearson\ncorrelations for each of these measures and notable relation-\nships are identiﬁed. LC mathematics was found to have a\nstatistically signiﬁcant relationship with performance, r =\n0.46, p < 0.01. LC physics was found to be moderately\nstrong and signiﬁcant, r = 0.59, p < 0.05 as was LC biol-\nogy, r = 0.75, p < 0.05 for the ﬁnal examination. Highest\nscience result, which includes other less commonly studied\nscience subjects, was also found to be statistically signiﬁ-\ncant, r = 0.48, p < 0.01. No relationship was found be-\ntween LC chemistry and performance. Secondary analysis,\nbased on gender revealed that none of the measures were\nsigniﬁcant for male students and resulted in notably higher\ncorrelations for the female students, as shown in Table 1. A\nrecent study on gender diﬀerences in LC examinations found\nthat (1) more female students are taking higher level LC ex-\naminations than male students and (2) female students are\noutperforming male students on LC maths and physics ex-\naminations (no other science subjects were reviewed in the\nstudy) [5]. This may relate to our ﬁndings and further re-\nsearch is necessary.\nThe ﬁndings on the relationship between experience in\nmathematics and science subjects, and programming per-\nformance is in line with previous research ﬁndings. The\nstrength of the correlations between LC physics scores and\nparticularly LC biology scores and programming performance\nis interesting and would suggest that science in general has\na signiﬁcant inﬂuence on performance. However, the lack\nof correlation with LC chemistry appears contradictory and\nfurther research is required.\nr\nn\nComfort level\n0.55**\nUnderstanding\n0.76**\nFemale only\nr\nn\nMale only\nr\nn\n** Correlation is signiﬁcant at the 0.01 level (2-tailed).\n* Correlation is signiﬁcant at the 0.05 level (2-tailed).\nPrevious computer experience was measured by prior pro-\ngramming experience and previous non-programming com-\nputer experience. In both cases student responses were sepa-\nrated into those with previous experience and those without\nprevious experience. Descriptive statistics for each group\nare given in Table 4. T-tests for independent samples were\nused to examine the diﬀerences between the overall mod-\nule results of each group. Before each t-test was carried\nout assumptions of normality and equality of variance were\nconﬁrmed. No signiﬁcant diﬀerences were found between\nstudents with or without previous programming experience\nor between students with or without non-programming com-\nputer experience and performance module. Although pre-\nvious research has found previous programming experience\nand non-programming computing experience to be indica-\ntors of success our results may be partially accounted for by\nthe fact that students cannot study programming or appli-\ncation software at examination level in secondary schools in\nIreland.\n4.2 Speciﬁc cognitive skills\nA correlation of r = 0.31, p < 0.01 was found between\nperformance on the cognitive test and performance on the\nmodule. Although this result is weak, subsequent analysis\nfound that a number of items in the test were highly corre-\nlated with programming performance. We anticipate that a\nredesign of the test could result in more signiﬁcant ﬁndings\nin the future.\n4.3 Personal information\nGender, age, work-style preference, encouragement by oth-\ners and part-time employment were treated as dichotomous\nGender\nAge\nWork-style preference\nEncouragement\nPart-time job\nProg. exp.\nn Mean\nFemale\nMale\nUnder 23\n23+\nIndividual 12\n\nGroup\n\nNo\n\nYes\n\nNo\n\nYes\n\nNone\n\nSome\n\nNone\n\nSome\n(%)\nvariables for analysis purposes. The possible values of each\nfactor are given in Table 2. Students were grouped accord-\ning to the responses they provided for each of the factors.\nDescriptive statistics for each group are given in Table 4.\nT-tests for independent samples were used to examine the\ndiﬀerences between the overall module results for each of\nthe factor values, for example the mean overall module re-\nsult for male students was compared to the mean overall\nmodule result for female students. Before each t-test was\ncarried out assumptions of normality and equality of vari-\nance were conﬁrmed. In each instance, the t-tests revealed\nno signiﬁcant diﬀerences between any of the factors and the\noverall results on the module. We intend to further examine\nthe relationship between work-style preference and perfor-\nmance, as since the introduction of PBL workshops into the\nmodule mean scores have increased, at the top, middle and\nbottom levels of the class. We feel this is a result of the\nPBL workshops and students repeating the module appear\nto concur with us [16].\n4.4 Experience on the module\nComfort level was measured as the cumulative response\nto questions on a student’s understanding of programming\nconcepts, diﬃculty designing programs without help and dif-\nﬁculty for completing lab assignments. Each question had\na number of ranked answers and the cumulative rank was\nused to analyze comfort level. The Pearson correlations are\ngiven in table 3. Comfort level was found to be a statistically\nsigniﬁcant indicator of performance with r = 0.516, p < .01.\nUnderstanding was measured by ranked responses to a\nsingle question ‘How do you rate your level of understand-\ning of the programming module?’ A strong signiﬁcant rela-\ntionship between understanding and performance was found,\nr = 0.76, p < 0.01.\nGiven the earlier ﬁndings on gender diﬀerences between\nprevious academic experience and programming performance,\ngender based analysis was carried out on comfort level and\nunderstanding. Comfort level was found to have a higher\ncorrelation with performance for male students.\nLike the Cantwell Wilson and Shrock [4] study comfort\nlevel was found to be highly correlated with programming\nperformance. The most signiﬁcant ﬁnding, however, is the\nvery strong correlation between a students’ perception of\ntheir understanding of the programming module. As this\nstudy was carried out in the second semester we intend to\nconduct a further study to identify the point in time per-\nception of module understanding becomes such a reliable\nindicator. If a similarly high correlation can be found early\non in the module then it would be very powerful in diagnos-\ning and subsequently mediating struggling students.\n4.5 Regression Analysis\nTo investigate whether the various factors studied were\npredictive of performance on the module a number of re-\ngression analyzes were conducted. Each analysis was moti-\nvated by the literature review, the authors’ experience work-\ning with ﬁrst year students and the strength of the corre-\nlation coeﬃcients generated in this study. Although, both\nLC biology and LC physics rendered high correlation coeﬃ-\ncients for programming performance, neither variables were\ndirectly included in the regression models as the sample size\nfor each was deemed too small (n=10, n=18 respectively).\nInstead the highest LC science result was included (n=28).\nThe ﬁrst model was designed to determine the earliest\nindicators of programming performance. Consideration was\ngiven to gender, previous academic experience, cognitive test\nscore, previous programming and non-programming com-\nputer performance, encouragement from others, work-style\npreference and hours working at a part-time job. Using a\nstepwise regression method a signiﬁcant model emerged with\nF (2, 27) = 7.113, p < 0.01 with an adjusted R square =30%.\nSigniﬁcant values were found for: LC maths (β = 0.390, p =\n0.021) and gender (β = −0.368, p = 0.028).\nThe second model considered all of the predictors used in\nthe ﬁrst model but also included the results of the ﬁrst class\ntest. Class tests are typically the ﬁrst test given to ﬁrst year\nstudents and although they do not test a student’s ability\nto design and code up a solution to a programming problem\nthey do test a students’ understanding of basic programming\nconcepts. A stepwise regression method found a signiﬁcant\nmodel of F (2, 27) = 14.882, p < 0.001 and adjusted R square\n=49%. Signiﬁcant values were found for the class test (β =\n0.563, p = 0.000) and LC maths (β = 0.375, p = 0.01).\nThe third model included the predictors from the sec-\nond model but also considers the results of the ﬁrst lab\ntest. The lab test is similar to the ﬁnal examination in\nthat students are required to design and code up a solu-\ntion to a programming problem. Although the lab test may\nbe a better predictor of the overall result, a trade oﬀ takes\nplace in that this information is not available until near the\nend of the ﬁrst Semester and at this stage struggling stu-\ndents may have dropped out or given-up hope of succeed-\ning. A stepwise regression method resulted in a signiﬁcant\nmodel of F (2, 25) = 26.38, p < 0.001 and an adjusted R\nsquare = 65%. Signiﬁcant factors were found for the ﬁrst\nlab test (β = 0.700, p = 0.000) and highest science result\n(β = 0.700, p = 0.000).\nThe fourth model includes the predictors from the sec-\nond model but takes into account a students’ comfort level\nwith the module and perceived understanding of how they\nare doing. Using a stepwise regression method a signiﬁcant\nmodel emerged with F (4, 23) = 26.03, p < 0.001, adjusted\nR square = 79%. Signiﬁcant values were found for: under-\nstanding (β = 0.505, p = 0.000), gender (β = −0.494, p =\n0.000), comfort level (β = 0.301, p = 0.022), and LC maths\n(β = 0.197, p = 0.047). If the results of the ﬁrst lab test is\nalso considered 84% of the variance in performance can be\naccounted for with F (4, 23) = 36.92, p < 0.001.\nThe factors known at the start of the academic year re-\nsult in a poor prediction of programming performance. The\nresults of the ﬁrst class test (model 2) and subsequently\nthe ﬁrst lab exam (model 3) results in an improved predic-\ntion ability. However, when a students’ perception of their\nunderstanding of the module is considered a very strong\nprediction model occurs. As with the strength of the Pear-\nson correlation coeﬃcient for this variable a further study\nto determine the stage at which a student’s self-perception\nbecomes so accurate would be valuable.",
         "5. CONCLUSIONS\nThis study examined the relationship and predictive abil-\nity between ﬁfteen factors and performance on a program-\nming module. Comfort level on the module, LC maths and\nLC science scores were shown to have a strong correlation\nwith performance, with notable gender diﬀerences identi-\nﬁed. A predictive combination of factors was found to be a\nstudent’s perception of their understanding of the module,\ncomfort level on the module, LC maths score and gender,\naccounting for 79% of the variance in programming perfor-\nmance.\nThe study found that the strongest relationship existed\nbetween a student’s perception of their understanding of\nthe module and programming performance. The need to\nunderstand the role of self-perception in the process and to\ninvestigate how early it becomes a reliable predictor war-\nrants further research.",
         "",
         "ABSTRACT\nThis paper documents a study, carried out in the academic\nyear 2003-2004, on ﬁfteen factors that may inﬂuence per-\nformance on a ﬁrst year object-oriented programming mod-\nule. The factors included prior academic experience, prior\ncomputer experience, self-perception of programming per-\nformance and comfort level on the module and speciﬁc cog-\nnitive skills. The study found that a student’s perception of\ntheir understanding of the module had the strongest corre-\nlation with programming performance, r = 0.76, p < 0.01.\nIn addition, Leaving Certiﬁcate (LC) mathematics and sci-\nence scores were shown to have a strong correlation with\nperformance. A regression module, based upon a student’s\nperception of their understanding of the module, gender, LC\nmathematics score and comfort level was able to account for\n79% of the variance in programming performance results.\nINTRODUCTION\nStudent retention on third-level (post high school or equiv-\nalent) Computer Science (CS) and Information Technology\n(IT) courses is a signiﬁcant problem. Students ﬁnd com-\nputer programming diﬃcult and struggle to master the core\nconcepts. A multi-national, multi-institutional study on the\nprogramming skills of ﬁrst year CS students found that stu-\ndents struggled to achieve an average above 30% on assess-\nments administered as part of their study [14]. Further-\nmore, introductory programming modules tend to have a\nPermission to make digital or hard copies of all or part of this work for\npersonal or classroom use is granted without fee provided that copies are\nnot made or distributed for proﬁt or commercial advantage and that copies\nbear this notice and the full citation on the ﬁrst page. To copy otherwise, to\nrepublish, to post on servers or to redistribute to lists, requires prior speciﬁc\npermission and/or a fee.\nSIGCSE’05 February 23–27, 2005 St. Louis, Missouri, USA.\nCopyright2005ACM1-58113-997-7/05/0002...$5.00.\nvery high student to lecturer ratio (100:1 or greater) and\noften lecturers do not know how well students are doing un-\ntil after the ﬁrst assessment. In general the ﬁrst assessment\ndoes not take place until six or eight weeks after the mod-\nule has commenced and given the typically high number of\nstudents, marking the assessments can take a considerable\nlength of time. At this stage, it may be too late to intervene\nto prevent struggling students from failing. Even if inter-\nvention is possible, a lecturer is unlikely to know a student\nwell enough or be able to identify individual student prob-\nlems and therefore recognize the most suitable interventions\nto make.\nThe research documented in this paper is part of a lon-\ngitudinal study on early identiﬁable factors that inﬂuence\nperformance on an introductory programming module.\nIf\nsuch factors can be identiﬁed then it may be possible to de-\nvelop a tool to provide an early diagnosis of a student’s likely\nperformance on a programming module. Interested parties\ncould use the tool to make more informed decisions on ap-\npropriate courses of action and to decide upon personalized\ninterventions that foster a student’s intellectual strengths.\n2. RELATED RESEARCH\nWhile a considerable amount of research has been carried\nout on factors that aﬀect programming performance our in-\nterest is on factors that aﬀect programming performance\non an introductory third-level object-oriented programming\nmodule where such factors can be determined early in the\nacademic year. These factors can be categorized as: (1)\nprevious academic and computer experience, (2) cognitive\nskills, (3) personal information and (4) experience on the\nmodule. A brief review of some research studies in each of\nthese categories is presented next.\nPrevious academic experience and programming experi-\nence have often been cited as predictors of programming suc-\ncess. Several studies have found that mathematical ability\nand exposure to maths courses are important predictors of\nperformance on introductory computer science modules [2,\n3, 4, 6, 13, 19]. Similarly, although less studied, performance\nin and experience of science subjects has also been shown\nto be important [2, 19]. Studies have also found prior pro-\ngramming experience and non-programming computer ex-\nperience (for example, experience of computer applications,\nemailing, game playing and surﬁng the web) to be related\nto programming performance [9, 10, 6, 4, 18].\nThe role of cognitive factors in programming has also re-\nceived research attention. Certain cognitive factors, includ-\ning problem-solving, abstract reasoning, problem translation\nskills, logical ability and cognitive style have been identiﬁed\nas possible predictors [7, 12, 11, 15].\nNumerous studies have been carried out on demographic\nand self-reported personal information. Some studies have\nexamined speciﬁc attributes related to study, for example\npreference for working alone or in a group to solve a pro-\ngramming problem and encouragement or support from oth-\ners to study computers [4, 2, 8].\nIn recent times researchers have examined the relation-\nship between students’ expectations of and experiences on\nan introductory computing module. A positive relationship\nbetween a student’s mental models of programming and self-\neﬃcacy for programming and performance has recently been\nidentiﬁed [18]. The grade a student expected to achieve in\nan introductory module was found to be the most impor-\ntant indicator of performance in another recent study[17].\nA recent longitudinal study found that the most important\npredictor of students’ performance on an introductory com-\nputer science course was comfort level, determined by the\ndegree of anxiety a student felt about the course [4].\nAlthough a considerable number of research studies have\nexamined factors that inﬂuence programming performance,\ncomparisons between the various studies and application of\nthe results are diﬃcult because the studies are carried out\nusing diﬀerent parameters. These parameters include:\n• the type of students (novice to experienced program-\nmers, academic students to employees),\n• the content of the course (some courses are solely pro-\ngramming courses while others are introductory com-\nputing courses),\n• the programming language being taught (older studies\ntend to be based on procedural languages),\n• the educational setting (many of the research studies\nare based on the US educational system) and\n• the reference criterion (for example continuous assess-\nment, end-of-year exam, job performance).\nIn the rest of this paper we describe our study, which\nbuilds upon existing research, in particular the work of [2,\n4], to identify early factors that inﬂuence performance of\nﬁrst year students, on an introductory object-oriented pro-\ngramming module, using the Java programming language.\nThe course is based on the Irish educational system and\nthe reference criterion is the overall result achieved on the\nmodule.\n3. RESEARCH DESIGN\nThe introductory programming module at our university\nis composed of a one and a half hour Problem-Based Learn-\ning (PBL) workshop, a one and a half hour laboratory ses-\nsion and three one-hour lectures per week over two semesters.\nStudents in Ireland do not study programming in secondary\nschool and the majority of students taking this module have\nrecently completed second level education.\nSelection of factors for this study was restricted for a num-\nber of reasons, including availability of participants, length\nof completion time needed for each instrument and stage in\nyear. With this in mind we attempted to examine the re-\nlationship between and the predictability of ﬁfteen factors\nand performance on our introductory module. The factors\nfall into four broad categories:\n1. previous academic and computer experience: as mea-\nsured by performance in the Irish Leaving Certiﬁcate\n(LC) examinations in mathematics and science sub-\njects and self-reported computer experience,\n2. speciﬁc cognitive skills: as measured by an in-house\n3. personal information: gender, age, work-style prefer-\nence, encouragement from others and the number of\nhours per week working at a part-time job and\n4. experience on the module: students own perception of\nhow well they are doing and how comfortable they feel\nwith the module material.\nPerformance on this module is based on continuous assess-\nment (30% of the overall mark) and a ﬁnal examination (70%\nof the overall mark). The measure of performance reported\nupon in this paper is the overall module mark. Continu-\nous assessment and ﬁnal examination marks render similar\nresults and are reported in [1].\n3.1 Participants\nThe study was carried out in the academic year 2003-2004.\nStudents enrolled in the ﬁrst year ‘Introduction to Program-\nming’ module in our department voluntarily participated in\nthis study. Ninety-six students completed the module in the\nacademic year 2003-2004.\n3.2\nInstruments\nTwo instruments were used to collect data: a question-\nnaire and a custom-made cognitive test. The questionnaire\ncollected data on the following items: (1) LC mathematics\ngrade, (2) LC physics grade, (3) LC biology grade, (4) LC\nchemistry grade, (5) highest LC science grade, (6) comfort\nlevel on the module, (7) perceived understanding of the mod-\nule material, (8) prior programming experience, (9) prior\nnon-programming computer experience, (10) work-style pref-\nerence (preference to work-alone or as part of a group), (11)\nencouragement from others to study computer science, (12)\nnumber of hours per week working at a (part-time) job. The\ncognitive test was developed in-house1 and comprised items\ninvolving numerical and letter sequencing, arithmetic rea-\nsoning, problem translation skills and logical ability. In ad-\ndition, information on gender, age and overall module results\nwas available for all students taking the module.\nIn total 80 students (49 male, 31 female) completed the\ncognitive test and 30 (19 male, 11 female) students com-\npleted the survey. Both instruments were completed in the\nsecond semester of the module and data collection for both\nwas paper-based.\n4. RESULTS\nAn a priori analysis was carried out to verify no signif-\nicant diﬀerence existed between the mean overall module\nscores of the class and the sample. Test assumptions on\nnormality (Kolmogorov-Smirnov test) and the equality of\nvariance (Levine test) were performed and a t-test on the\noverall results, (t(124) = 0.795, p = 0.428), found no signif-\nicant diﬀerences between the mean scores of the class and\n1Developed by Jacqueline McQuillan, Department of Com-\nputer Science, NUI Maynooth.\nLC\nMaths\nLC\nPhys\nLC\nChem\nLC\nBio\n0.48**\nValues\nMale, Female\nGender\nUnder 23, 23+\nAge\nIndividual, Group\nWork style preference\nEncouragement\nYes, No\nPart-time employment No, Yes\n0.89** 0.88\nFemale only\nr\nn\nMale only\n0.27\nr\nn\n\n** Correlation is signiﬁcant at the 0.01 level (2-tailed).\n* Correlation is signiﬁcant at the 0.05 level (2-tailed).\nthe sample. In the remainder of this section the ﬁndings on\nthe relationship between each of the factors studied and pro-\ngramming performance is presented, followed by an analysis\nof the combination of factors that best predicts performance.\n4.1 Previous academic and computer experi-\nence\nTo establish the relationship between previous academic\nexperience in mathematics and science, the achievable grades\nfor each subject were ranked, with the highest rank given\nto the highest possible grade and the lowest rank given to\nthe lowest possible grade. Table 1. provides the Pearson\ncorrelations for each of these measures and notable relation-\nships are identiﬁed. LC mathematics was found to have a\nstatistically signiﬁcant relationship with performance, r =\n0.46, p < 0.01. LC physics was found to be moderately\nstrong and signiﬁcant, r = 0.59, p < 0.05 as was LC biol-\nogy, r = 0.75, p < 0.05 for the ﬁnal examination. Highest\nscience result, which includes other less commonly studied\nscience subjects, was also found to be statistically signiﬁ-\ncant, r = 0.48, p < 0.01. No relationship was found be-\ntween LC chemistry and performance. Secondary analysis,\nbased on gender revealed that none of the measures were\nsigniﬁcant for male students and resulted in notably higher\ncorrelations for the female students, as shown in Table 1. A\nrecent study on gender diﬀerences in LC examinations found\nthat (1) more female students are taking higher level LC ex-\naminations than male students and (2) female students are\noutperforming male students on LC maths and physics ex-\naminations (no other science subjects were reviewed in the\nstudy) [5]. This may relate to our ﬁndings and further re-\nsearch is necessary.\nThe ﬁndings on the relationship between experience in\nmathematics and science subjects, and programming per-\nformance is in line with previous research ﬁndings. The\nstrength of the correlations between LC physics scores and\nparticularly LC biology scores and programming performance\nis interesting and would suggest that science in general has\na signiﬁcant inﬂuence on performance. However, the lack\nof correlation with LC chemistry appears contradictory and\nfurther research is required.\nr\nn\nComfort level\n0.55**\nUnderstanding\n0.76**\nFemale only\nr\nn\nMale only\nr\nn\n** Correlation is signiﬁcant at the 0.01 level (2-tailed).\n* Correlation is signiﬁcant at the 0.05 level (2-tailed).\nPrevious computer experience was measured by prior pro-\ngramming experience and previous non-programming com-\nputer experience. In both cases student responses were sepa-\nrated into those with previous experience and those without\nprevious experience. Descriptive statistics for each group\nare given in Table 4. T-tests for independent samples were\nused to examine the diﬀerences between the overall mod-\nule results of each group. Before each t-test was carried\nout assumptions of normality and equality of variance were\nconﬁrmed. No signiﬁcant diﬀerences were found between\nstudents with or without previous programming experience\nor between students with or without non-programming com-\nputer experience and performance module. Although pre-\nvious research has found previous programming experience\nand non-programming computing experience to be indica-\ntors of success our results may be partially accounted for by\nthe fact that students cannot study programming or appli-\ncation software at examination level in secondary schools in\nIreland.\n4.2 Speciﬁc cognitive skills\nA correlation of r = 0.31, p < 0.01 was found between\nperformance on the cognitive test and performance on the\nmodule. Although this result is weak, subsequent analysis\nfound that a number of items in the test were highly corre-\nlated with programming performance. We anticipate that a\nredesign of the test could result in more signiﬁcant ﬁndings\nin the future.\n4.3 Personal information\nGender, age, work-style preference, encouragement by oth-\ners and part-time employment were treated as dichotomous\nGender\nAge\nWork-style preference\nEncouragement\nPart-time job\nProg. exp.\nn Mean\nFemale\nMale\nUnder 23\n23+\nIndividual 12\n\nGroup\n\nNo\n\nYes\n\nNo\n\nYes\n\nNone\n\nSome\n\nNone\n\nSome\n(%)\nvariables for analysis purposes. The possible values of each\nfactor are given in Table 2. Students were grouped accord-\ning to the responses they provided for each of the factors.\nDescriptive statistics for each group are given in Table 4.\nT-tests for independent samples were used to examine the\ndiﬀerences between the overall module results for each of\nthe factor values, for example the mean overall module re-\nsult for male students was compared to the mean overall\nmodule result for female students. Before each t-test was\ncarried out assumptions of normality and equality of vari-\nance were conﬁrmed. In each instance, the t-tests revealed\nno signiﬁcant diﬀerences between any of the factors and the\noverall results on the module. We intend to further examine\nthe relationship between work-style preference and perfor-\nmance, as since the introduction of PBL workshops into the\nmodule mean scores have increased, at the top, middle and\nbottom levels of the class. We feel this is a result of the\nPBL workshops and students repeating the module appear\nto concur with us [16].\n4.4 Experience on the module\nComfort level was measured as the cumulative response\nto questions on a student’s understanding of programming\nconcepts, diﬃculty designing programs without help and dif-\nﬁculty for completing lab assignments. Each question had\na number of ranked answers and the cumulative rank was\nused to analyze comfort level. The Pearson correlations are\ngiven in table 3. Comfort level was found to be a statistically\nsigniﬁcant indicator of performance with r = 0.516, p < .01.\nUnderstanding was measured by ranked responses to a\nsingle question ‘How do you rate your level of understand-\ning of the programming module?’ A strong signiﬁcant rela-\ntionship between understanding and performance was found,\nr = 0.76, p < 0.01.\nGiven the earlier ﬁndings on gender diﬀerences between\nprevious academic experience and programming performance,\ngender based analysis was carried out on comfort level and\nunderstanding. Comfort level was found to have a higher\ncorrelation with performance for male students.\nLike the Cantwell Wilson and Shrock [4] study comfort\nlevel was found to be highly correlated with programming\nperformance. The most signiﬁcant ﬁnding, however, is the\nvery strong correlation between a students’ perception of\ntheir understanding of the programming module. As this\nstudy was carried out in the second semester we intend to\nconduct a further study to identify the point in time per-\nception of module understanding becomes such a reliable\nindicator. If a similarly high correlation can be found early\non in the module then it would be very powerful in diagnos-\ning and subsequently mediating struggling students.\n4.5 Regression Analysis\nTo investigate whether the various factors studied were\npredictive of performance on the module a number of re-\ngression analyzes were conducted. Each analysis was moti-\nvated by the literature review, the authors’ experience work-\ning with ﬁrst year students and the strength of the corre-\nlation coeﬃcients generated in this study. Although, both\nLC biology and LC physics rendered high correlation coeﬃ-\ncients for programming performance, neither variables were\ndirectly included in the regression models as the sample size\nfor each was deemed too small (n=10, n=18 respectively).\nInstead the highest LC science result was included (n=28).\nThe ﬁrst model was designed to determine the earliest\nindicators of programming performance. Consideration was\ngiven to gender, previous academic experience, cognitive test\nscore, previous programming and non-programming com-\nputer performance, encouragement from others, work-style\npreference and hours working at a part-time job. Using a\nstepwise regression method a signiﬁcant model emerged with\nF (2, 27) = 7.113, p < 0.01 with an adjusted R square =30%.\nSigniﬁcant values were found for: LC maths (β = 0.390, p =\n0.021) and gender (β = −0.368, p = 0.028).\nThe second model considered all of the predictors used in\nthe ﬁrst model but also included the results of the ﬁrst class\ntest. Class tests are typically the ﬁrst test given to ﬁrst year\nstudents and although they do not test a student’s ability\nto design and code up a solution to a programming problem\nthey do test a students’ understanding of basic programming\nconcepts. A stepwise regression method found a signiﬁcant\nmodel of F (2, 27) = 14.882, p < 0.001 and adjusted R square\n=49%. Signiﬁcant values were found for the class test (β =\n0.563, p = 0.000) and LC maths (β = 0.375, p = 0.01).\nThe third model included the predictors from the sec-\nond model but also considers the results of the ﬁrst lab\ntest. The lab test is similar to the ﬁnal examination in\nthat students are required to design and code up a solu-\ntion to a programming problem. Although the lab test may\nbe a better predictor of the overall result, a trade oﬀ takes\nplace in that this information is not available until near the\nend of the ﬁrst Semester and at this stage struggling stu-\ndents may have dropped out or given-up hope of succeed-\ning. A stepwise regression method resulted in a signiﬁcant\nmodel of F (2, 25) = 26.38, p < 0.001 and an adjusted R\nsquare = 65%. Signiﬁcant factors were found for the ﬁrst\nlab test (β = 0.700, p = 0.000) and highest science result\n(β = 0.700, p = 0.000).\nThe fourth model includes the predictors from the sec-\nond model but takes into account a students’ comfort level\nwith the module and perceived understanding of how they\nare doing. Using a stepwise regression method a signiﬁcant\nmodel emerged with F (4, 23) = 26.03, p < 0.001, adjusted\nR square = 79%. Signiﬁcant values were found for: under-\nstanding (β = 0.505, p = 0.000), gender (β = −0.494, p =\n0.000), comfort level (β = 0.301, p = 0.022), and LC maths\n(β = 0.197, p = 0.047). If the results of the ﬁrst lab test is\nalso considered 84% of the variance in performance can be\naccounted for with F (4, 23) = 36.92, p < 0.001.\nThe factors known at the start of the academic year re-\nsult in a poor prediction of programming performance. The\nresults of the ﬁrst class test (model 2) and subsequently\nthe ﬁrst lab exam (model 3) results in an improved predic-\ntion ability. However, when a students’ perception of their\nunderstanding of the module is considered a very strong\nprediction model occurs. As with the strength of the Pear-\nson correlation coeﬃcient for this variable a further study\nto determine the stage at which a student’s self-perception\nbecomes so accurate would be valuable.\n5. CONCLUSIONS\nThis study examined the relationship and predictive abil-\nity between ﬁfteen factors and performance on a program-\nming module. Comfort level on the module, LC maths and\nLC science scores were shown to have a strong correlation\nwith performance, with notable gender diﬀerences identi-\nﬁed. A predictive combination of factors was found to be a\nstudent’s perception of their understanding of the module,\ncomfort level on the module, LC maths score and gender,\naccounting for 79% of the variance in programming perfor-\nmance.\nThe study found that the strongest relationship existed\nbetween a student’s perception of their understanding of\nthe module and programming performance. The need to\nunderstand the role of self-perception in the process and to\ninvestigate how early it becomes a reliable predictor war-\nrants further research.",
         null
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "Intracellular and Surface Distribution of Monocyte Tissue Factor: Application to Intersubject Variability",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced monocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and shedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs). Methods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of TF-positive monocytes in HRs (32.0±3.5%) versus LRs (11.2±1.2%; P≤0.05), along with a stronger mean fluorescence intensity of TF signal in HRs (7.1±0.5 arbitrary units [AU]) compared with LRs (5.4±0.4 AU; P≤0.05). The LPS-treated blood of the HR group contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular accumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs versus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of surface TF, whereas MNCs from LRs only had a minor increase in surface TF levels. Conclusions—The higher availability of surface TF antigen on MNCs from HRs and TF-containing microparticles might make these individuals more susceptible to hypercoagulation.",
         "The production and availability of tissue thromboplastin in cellular populations of whole blood exposed to various concentrations of endotoxin. An assay for detection of endotoxin. Tissue factor activity in whole blood. Nicotinamide inhibits endotoxin‐induced monocyte tissue factor expression Cellular activation responses in blood in relation to lipid pattern: healthy men and women in families with myocardial infarction or cancer Tissue factor activity in human monocytes is regulated by plasma: implications for the high and low responder phenomenon Role of Platelet P-Selectin and CD40 Ligand in the Induction of Monocytic Tissue Factor Expression Eicosanoids and Iso-Eicosanoids: Constitutive, Inducible and Transcellular Biosynthesis in Vascular Disease Cytokine gene polymorphism and heart transplant rejection. Platelet-derived growth factor-BB and monocyte chemotactic protein-1 induce human peripheral blood monocytes to express tissue factor. Association of smooth muscle cell tissue factor with caveolae. Manifestation of cryptic fibroblast tissue factor occurs at detergent concentrations which dissolve the plasma membrane A novel biological effect of platelet factor 4 (PF4): enhancement of LPS‐induced tissue factor activity in monocytes Th1 and Th2 T-helper cells exert opposite regulatory effects on procoagulant activity and tissue factor production by human monocytes. Granulocytes enhance LPS‐induced tissue factor activity in monocytes via an interaction with platelets Cathepsin G-dependent platelet stimulation by activated polymorphonuclear leukocytes and its inhibition by antiproteinases: role of P-selectin-mediated cell-cell adhesion. Relations between factor VIIa binding and expression of factor VIIa/tissue factor catalytic activity on cell surfaces. Platelet activating factor enhancement of lipopolysaccharide‐induced tissue factor activity in monocytes: requirement of platelets and granulocytes Dysregulation of in vitro cytokine production by monocytes during sepsis. The role of arachidonic acid release and lipoxygenase pathway in lipopolysaccharide-induced thromboplastin activity in monocytes. In vitro production of IL 1β, IL 1α, TNF and IL 2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation Functional tissue factor is entirely cell surface expressed on lipopolysaccharide-stimulated human blood monocytes and a constitutively tissue factor-producing neoplastic cell line Localization of tissue factor in the normal vessel wall and in the atherosclerotic plaque. Cytokine-induced procoagulant activity in monocytes and endothelial cells. Further enhancement by cyclosporine. Tumor necrosis factor induces tissue factor-like activity in human leukemia cell line U937 and peripheral blood monocytes. Enhancement of mononuclear procoagulant activity by platelet 12-hydroxyeicosatetraenoic acid. The stimulatory effect of platelets and platelet membranes on the procoagulant activity of leukocytes. The Role of Protein Synthesis on the Generation of Tissue Factor Activity by Leukocytes Monocytes. The high responder phenomenon: enhancement of LPS induced tissue factor activity in monocytes by platelets and granulocytes. Selective cellular expression of tissue factor in human tissues. Implications for disorders of hemostasis and thrombosis. In vitro production of IL 1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation. Mononuclear cell tissue factor: cell of origin and requirements for activation. After blocking with 5% non-fat skimmed milk, membranes were incubated with mouse anti-human TF monoclonal antibodies After Lymphoprep isolation, total RNA was extracted from MNC pellets with an OR) anti-human TF monoclonal antibodies (10 µg/ml) for 30 min on ice. Mouse monoclonal antibodies recognizing extracellular domain of human TF were prepared by the Monoclonal Antibody Facility",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.ATV.0000168413.29874.d7",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced\nmonocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and\nshedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs).\nMethods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of\nTF-positive monocytes in HRs (32.03.5%) versus LRs (11.21.2%; P0.05), along with a stronger mean fluorescence intensity\nof TF signal in HRs (7.10.5 arbitrary units [AU]) compared with LRs (5.40.4 AU; P0.05). The LPS-treated blood of the HR\ngroup contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular\naccumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs\nversus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of\nsurface TF, whereas MNCs from LRs only had a minor increase in surface TF levels.",
         "ponents of the plasma coagulation cascade is important\nfor maintenance of normal rheological properties of blood.\nFailure to manipulate TF levels available for the initiation of\nblood clotting leads to thrombotic or bleeding disorders in\nhumans. Circulating monocytes are presumably the major cell\ntype that respond to variable stimuli by developing coagulant\nactivity1 through the expression of TF.2\nOriginally, intersubject variability in developing of mono-\ncyte TF activity was described by Østerud et al.3 By compar-\ning lipopolysaccharide (LPS)-induced monocyte TF activity\nand tumor necrosis factor- (TNF-) production in a whole\nblood system, an up to a 50-fold difference between individ-\nuals was observed.4 This finding was defined as the “high–\nlow responder phenomenon.”4 Also noteworthy, the individ-\nual usually remains a high responder (HR) or low responder\n(LR) for several years.5,6 Later, high intersubject variability in\ncytokine production by LPS-stimulated monocytes was dem-\nonstrated.7 It was also shown that patients with high levels of\nTNF- production were more susceptible to heart transplant\nrejection.8 Monocytes isolated from septic shock patient\nsurvivors revealed higher TNF- production than monocytes\nfrom nonsurvivors.9\nMany studies have been undertaken to describe the signifi-\ncance of this phenomenon, but so far, no general explanation has\nbeen found. Diverse plasma factors and direct cell interactions\nplay an important role in the development of monocyte TF\nactivity.10 –15 High expression of monocyte TF activity is asso-\nciated with higher risk of acute coronary syndrome.16\nPlatelets have been suggested to be responsible for\ninducing monocyte TF activity.17 Platelet-rich plasma\ninduced significantly higher TF activity in LPS-stimulated\nmonocytes than platelet-poor plasma.18 Moreover, when\nblood cells without platelets from HRs were mixed with\nplatelet-rich plasma of an LR, LPS-induced TF activity\nwas reduced up to 76% compared with an autologous\nsystem.18 It was shown that granulocytes enhance LPS-\ninduced monocyte TF activity in a platelet-dependent\nreaction involving P-selectin, platelet factor 4, platelet-\nactivating factor, hydroxyl-eicosatetraenoic acid, and\nplatelet-derived growth factor.18 –21\nHere we report several observations concerning the rela-\ntionships between intracellular- and membrane-located TF\nantigen in resting and LPS-stimulated monocytes in groups of\nHRs and LRs, using fluorescence-activated cell sorter\n(FACS) analysis, fluorescence confocal microscopy, in-cell\nOriginal received February 24, 2005; final version accepted April 14, 2005.\nFrom the Department of Biochemistry (E.M.E., M.A.S., G.B., J.O.O., B.O.), Institute of Medical Biology, University of Tromsø, Norway; the\nDepartment of Immunology and Transfusion Medicine (F.X.E.G.), University Hospital of Northern Norway, Tromsø; and the Department of Biochemistry\n(B.P.-S., K.G.M.), University of Vermont, Burlington.\nE.M.E. and M.A.S. contributed equally to this work.\nCorrespondence to Elena M. Egorina, MD, Department of Biochemistry, Institute of Medical Biology, University of Tromsø, N-9037, Tromsø,\nArterioscler Thromb Vasc Biol.\nWestern assay, and immunoblotting. In response to LPS\nstimulation, we clearly found different patterns of intracellu-\nlar accumulation and externalization of TF onto the surface of\nMNCs isolated from whole blood of HRs and LRs.",
         "Results\nReactivity of MNCs in HRs and LRs\nThe LPS-induced responses of MNCs from individuals enrolled\nin our study have been repeatedly assessed to determine their\nindividual TF activity and concentration levels of TNF-. These\nhealthy individuals were allocated to HR or LR groups accord-\ning to criteria developed in our laboratory.16 Briefly, healthy\nvolunteers were defined as HRs if their MNC-TF activity\nexceeded 20 mU/106 cells and their plasma TNF- concentra-\ntion levels were 3 ng/mL taken from whole blood stimulated\nwith LPS for 2 hours. We assigned individuals with these\nparameters lower than indicated above to the LR group.\nMeasurements of TF procoagulant activity in lysates of\nMNCs isolated from ex vivo LPS-stimulated whole blood\nrevealed 2 distinct responses. In HRs, LPS stimulation\ninduced significantly higher values of MNC-TF activity than\nin LRs after 2 hours (50.114.3 versus 20.94.2 mU/106\ncells; P0.05) and 4 hours (56.016.3 mU/106 versus\n21.24.9 mU/106 cells; P0.05; Figure 1). We found no\nincrease in TF activity in MNC lysates from time-matched\nnonstimulated samples (Figure 1).\nEgorina et al\nTF in High and Low Responders\nThe differences of MNC-TF activity and the production of\nTNF- in our experimental model could not be caused by\nvariations in degree of cell death because the number of dead\ncells measured by trypan blue exclusion assay did not differ\nbetween groups and was within 3% of the total cell count.\nThe variable degrees of responsiveness to LPS treatment,\ncharacterized by significantly elevated TF activity in MNC\nlysates in HRs,\nthe comparisons of\nintracellular and surface TF antigen levels under resting and\nstimulated conditions in these individuals.\nled us to carry out\nBecause we planned to use immunostaining-based methods for\nnative TF antigen detection (such as fluorescence-activated cell\nsorting, fluorescence confocal microscopy and in-cell Western\nassay), we applied RNA interference against TF expression in\nplated HeLa cells. Similarly, the same antibodies detected a 3-fold\nreduction of the in-cell Western assay signal intensity in the wells\nthat contained HeLa cells transfected with siRNA against TF when\ncompared with cells transfected with control siRNA. The signal\nfrom the in-cell Western assay against GAPDH showed no differ-\nences in the same wells (Figure IB).\nTo characterize levels of TF antigen on the surface and\ninterior of resting and LPS-stimulated monocytes from HRs\nand LRs, we chose FACS analyses of whole blood, fluores-\ncence confocal microscopy, and in-cell Western assay of\npermeabilized and nonpermeabilized MNCs.\nArterioscler Thromb Vasc Biol.\nWe found no baseline difference in the surface expression\nof TF on nonstimulated CD14 cells from LRs (1.40.1%)\nand HRs (1.50.2%; Figure 3A and 3B). MFI did not differ\nbetween groups and was 4.50.3 AU and 4.20.1 AU in\nLRs and HRs, respectively (Figure 3C).\nBy use of fluorescence confocal microscopy of isolated\nMNCs in resting and LPS-stimulated conditions, we visual-\nized the distribution of TF antigen between cytoplasmic and\nmembrane compartments. Only a few CD14 cells expressed\nTF in permeabilized and nonpermeabilized conditions (Fig-\nure 4A and 4B). Furthermore, we noticed 2 phenotypes when\nimaging nonpermeabilized resting CD14 cells: 1 did not\nexpress TF on the membrane surface, whereas another did\n(Figure 4C and 4D). The pattern of TF surface distribution could\nbe characterized by the presence of 5 to 7 TF-enriched dots or\n“patches.” When permeabilized, resting CD14 cells have TF-\ncontaining speckles in the proximity of the plasma membrane,\nready to be exocytosed to the cell exterior (Figure 4E).\nAfter 2 hours of LPS stimulation, permeabilized mono-\ncytes had their membranes enriched with TF antigen. The\ncytoplasm was depleted from TF-containing granules except\nfor a strong TF-positive staining unilaterally from the nucleus\n(Figure 4F). This could be newly synthesized TF protein.\nTo verify TF antigen distribution data from flow cytometry\nand fluorescence confocal microscopy, we used a quantitative\nimmunofluorescence staining in-cell Western assay.\nsignificantly higher LPS-induced accumulation of intracellu-\nlar TF antigen in LRs compared with HRs (Figure 5B). At the\nsame time, LPS treatment caused a 4-fold increase of surface\nTF signal intensity in HRs. However, no such increase was\nfound in the surface TF signal of LRs (Figure 5B).\nData from the immunoblotting of MNC lysates against TF\nantigen confirmed that after 2 hours of LPS stimulation,\nMNCs of LRs indeed accumulated TF antigen, whereas in\nHRs,\nthis accumulation was minor (Figure 5C and 5D).\nMNCs from time-matched samples of LRs experienced vir-\ntually no accumulation of the TF antigen in contrast to the\nsignificant accumulation of TF in MNCs of HRs (Figure 5D).\nComparison of real-time PCR data on de novo TF synthesis in\nLPS-stimulated conditions revealed similar mean cycle threshold\nvalues in MNCs of HRs and LRs (Table I, available online at\nhttp://atvb.ahajournals.org). Therefore, LPS stimulation is not only\nnecessary for the induction of TF synthesis, but also for its transport\nvia exocytotic pathway and shedding from the cell membrane.\nDiscussion\nIn the present ex vivo study, using a whole blood system, we\nreport several novel observations characterizing the distribu-\ntion of TF antigen between the interior and surface of MNCs\nfrom HRs and LRs. MNCs from representatives of these\ngroups exhibit distinct patterns of intracellular TF accumula-\ntion, externalization onto cell membrane, and shedding of\nTF-containing microparticles in LPS-treated conditions.\nEgorina et al\nTF in High and Low Responders\nCD14 cells expose TF on the cell surface after LPS-\nstimulation when compared with LRs.\nData of FACS analyses were in accordance with our\nfindings from fluorescence confocal microscopy, in which we\ndemonstrated redistribution of TF antigen on stimulation with\nLPS onto the cell surface. In stimulated conditions, TF was\ntransported and localized mostly on the membrane because\nthere was a clear depletion of TF staining from the cytoplasm.\nIntense dots present unilaterally from the nucleus could\nrepresent de novo synthesized TF.\ntime-matched nonstimulated cells. Together, it is possible that\nmonocytes from HRs could synthesize equal or even higher\namounts of TF antigen than those from LRs.\nArterioscler Thromb Vasc Biol.\nImmunostaining against TF and fluorescent confocal mi-\ncroscopy of permeabilized and nonpermeabilized resting\nmonocytes support our data from flow cytometry because\nonly few monocytes expressed TF antigen. The localization\npattern of TF on the monocyte surface in the present study is\nconfined to dotted structures or “patches.” Intracellular local-\nized TF in CD14 cells was restricted to round vesicle-like\nstructures in the proximity of the plasma membrane. Mulder\net al demonstrated the presence of an intracellular TF pool\nassociated with caveolae and multivesicular bodies.24\nOur data obtained from FACS and immunoblotting are\nconsistent with the previous accepted opinion that resting human\nmonocytes contain nearly undetectable levels of TF antigen.25–30\nHowever, we provide new information about the relationship\nbetween the intracellular and surface pool of TF in MNCs by\nusing in-cell Western assay. This enzyme-independent method\nof antigen detection via antibodies conjugated with infrared dyes\ngives highly reproducible data characterizing antigen levels.\nComparisons of the TF signal from resting MNCs between\npermeabilized and nonpermeabilized conditions revealed that in\nresting conditions, MNCs contain equally distributed surface\nand intracellular TF antigen in HRs and LRs.\nIn conclusion, MNCs of HRs and LRs contain similar, nearly\nundetectable intracellular levels of TF antigen at baseline condi-\ntions. On LPS stimulation, TF is apparently transported faster to the\ncell surface and in a greater amount in the group of HRs. This group\nalso shed more of TF-containing microparticles. The cells of LRs\nhad a lower ability to be stimulated by LPS and retained more TF\nantigen, probably because of a slower exocytosis rate. Furthermore,\nLR monocytes express less TF antigen on their surface and shed\nfewer TF-containing microparticles. The findings of our present\nstudy indicate that elevated levels of the blood coagulation cascade\ninitiator on the surface of circulating monocytes and microparticles\nin HRs might make them more susceptible to hypercoagulation.",
         "",
         "Materials and Methods\nBlood Sampling and Experimental Design\nBlood samples from 16 healthy volunteers (HRs n8; LRs n8)\nwere obtained under protocols approved by the regional committee\nfor medical research ethics. In time-course ex vivo experiments,\nLPS-stimulated and -nonstimulated blood aliquots were incubated\nfor different times with agitation at 37°C. For TF activity assay and\nWestern blotting MNCs were isolated using Lymphoprep (Axis-\nShield) according to manufacturer instructions. For TNF- quanti-\nfication, platelet-poor plasma was obtained from blood aliquots by\ncentrifugation for 10 minutes. For a detailed description, please see\nthe online supplement, available at http://atvb.ahajournals.org.\nQuantification of Monocyte TF Activity\nTF was measured in the in frozen/thawed preparations of MNCs\nusing a 2-stage clotting assay based on the ability of TF to accelerate\nthe activation of factor X by factor VIIa as described previously.19\nQuantification of TNF-\nThe concentration of TNF- in platelet-poor plasma was determined by\nusing PeliKine compact ELISA kit according to manufacturer instructions.\nAntibody Quality Control and Small\nInterfering RNA\nTo ascertain the suitability of the monoclonal anti-human TF antibody used\nin the present study, we used TF gene silencing by RNA interference. For\na detailed description, please see the online supplement.\nFlow Cytometry\nPopulation of monocytes expressing TF (CD14TF cells) was deter-\nmined by staining of whole blood aliquots treated as described in\nexperimental design, using phycoerythrin (PE)-conjugated anti-human\nCD14 monoclonal antibodies (BD Biosciences PharMingen) and\nAlexa488-conjugated mouse anti-human TF monoclonal antibodies.\nMouse monoclonal antibodies recognizing extracellular domain of\nhuman TF were prepared by the monoclonal antibody facility, Depart-\nment of Biochemistry, University of Vermont, Burlington. For a\ndetailed description of the procedure, please see the online supplement.\nFluorescence Confocal Microscopy\nAfter Lymphoprep isolation, MNCs were plated for 30 minutes onto\nflat-bottomed 8-well chambers and fixed. Part of the samples was\npermeabilized with 70% methanol. TF and monocytes were visual-\nized by immunostaining with Alexa488-conjugated mouse anti-\nhuman TF monoclonal antibodies and PE-conjugated mouse anti-\nhuman CD14 monoclonal antibodies. Nuclei of the cells were stained\nusing DRAQ5 dye (Biostatus, Ltd). A fluorescent laser scanning\nmicroscope (LSM510 META; Carl Zeiss AG) with C-Apochromat\n40/1.2 W lens was used for image acquisition.\nWestern Blotting\nTF protein levels in lysates of resting and LPS-stimulated 106 MNCs\nwere detected using 10% SDS-PAGE, and immunoblotting of nitrocel-\nlulose membranes with mouse anti-human TF monoclonal antibodies\nand horseradish peroxidase– conjugated secondary antibodies. We used\ndensitometrical reading of 43-kDa immunopositive band for statistical\ncomparisons. For details, please the online supplement.\nRT-PCR and Real-Time Polymerase\nChain Reaction\nTotal RNA was extracted from MNC pellets with an RNeasy kit\n(Qiagen). Thereafter, single-stranded cDNA was synthesized using\nTaqMan Gold RT-PCR Kit, and real-time PCR analyses of TF\nmRNA expression was performed in duplicates with assay-on-\ndemand real-time PCR kit using ABI PRISM 7700 Sequence\nDetection System (Applied Biosystems). For data normalization, we\nanalyzed the expression of 18S rRNA by using an assay-on-demand\nkit. For details, please see the online supplement.\nCell Count and Viability\nCell counts were performed on a Sysmex K1000 (TOA Medical\nElectronics Co Ltd) in a whole blood system and after isolation of\nMNCs. Cell viability was assessed by using a trypan blue dye exclusion\nassay according to manufacturer instructions (Invitrogen Corp).\nStatistics\nStatistical comparisons were performed using SigmaPlot 8.0 (SPPS,\nLtd.) and MS Excel. The presented data shown are meanSEM.\nComparisons between mean values were performed using the Stu-\ndent paired t test. P value 0.05 was considered significant.",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced\nmonocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and\nshedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs).\nMethods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of\nTF-positive monocytes in HRs (32.03.5%) versus LRs (11.21.2%; P0.05), along with a stronger mean fluorescence intensity\nof TF signal in HRs (7.10.5 arbitrary units [AU]) compared with LRs (5.40.4 AU; P0.05). The LPS-treated blood of the HR\ngroup contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular\naccumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs\nversus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of\nsurface TF, whereas MNCs from LRs only had a minor increase in surface TF levels.\nponents of the plasma coagulation cascade is important\nfor maintenance of normal rheological properties of blood.\nFailure to manipulate TF levels available for the initiation of\nblood clotting leads to thrombotic or bleeding disorders in\nhumans. Circulating monocytes are presumably the major cell\ntype that respond to variable stimuli by developing coagulant\nactivity1 through the expression of TF.2\nOriginally, intersubject variability in developing of mono-\ncyte TF activity was described by Østerud et al.3 By compar-\ning lipopolysaccharide (LPS)-induced monocyte TF activity\nand tumor necrosis factor- (TNF-) production in a whole\nblood system, an up to a 50-fold difference between individ-\nuals was observed.4 This finding was defined as the “high–\nlow responder phenomenon.”4 Also noteworthy, the individ-\nual usually remains a high responder (HR) or low responder\n(LR) for several years.5,6 Later, high intersubject variability in\ncytokine production by LPS-stimulated monocytes was dem-\nonstrated.7 It was also shown that patients with high levels of\nTNF- production were more susceptible to heart transplant\nrejection.8 Monocytes isolated from septic shock patient\nsurvivors revealed higher TNF- production than monocytes\nfrom nonsurvivors.9\nMany studies have been undertaken to describe the signifi-\ncance of this phenomenon, but so far, no general explanation has\nbeen found. Diverse plasma factors and direct cell interactions\nplay an important role in the development of monocyte TF\nactivity.10 –15 High expression of monocyte TF activity is asso-\nciated with higher risk of acute coronary syndrome.16\nPlatelets have been suggested to be responsible for\ninducing monocyte TF activity.17 Platelet-rich plasma\ninduced significantly higher TF activity in LPS-stimulated\nmonocytes than platelet-poor plasma.18 Moreover, when\nblood cells without platelets from HRs were mixed with\nplatelet-rich plasma of an LR, LPS-induced TF activity\nwas reduced up to 76% compared with an autologous\nsystem.18 It was shown that granulocytes enhance LPS-\ninduced monocyte TF activity in a platelet-dependent\nreaction involving P-selectin, platelet factor 4, platelet-\nactivating factor, hydroxyl-eicosatetraenoic acid, and\nplatelet-derived growth factor.18 –21\nHere we report several observations concerning the rela-\ntionships between intracellular- and membrane-located TF\nantigen in resting and LPS-stimulated monocytes in groups of\nHRs and LRs, using fluorescence-activated cell sorter\n(FACS) analysis, fluorescence confocal microscopy, in-cell\nOriginal received February 24, 2005; final version accepted April 14, 2005.\nFrom the Department of Biochemistry (E.M.E., M.A.S., G.B., J.O.O., B.O.), Institute of Medical Biology, University of Tromsø, Norway; the\nDepartment of Immunology and Transfusion Medicine (F.X.E.G.), University Hospital of Northern Norway, Tromsø; and the Department of Biochemistry\n(B.P.-S., K.G.M.), University of Vermont, Burlington.\nE.M.E. and M.A.S. contributed equally to this work.\nCorrespondence to Elena M. Egorina, MD, Department of Biochemistry, Institute of Medical Biology, University of Tromsø, N-9037, Tromsø,\nArterioscler Thromb Vasc Biol.\nWestern assay, and immunoblotting. In response to LPS\nstimulation, we clearly found different patterns of intracellu-\nlar accumulation and externalization of TF onto the surface of\nMNCs isolated from whole blood of HRs and LRs.\nMaterials and Methods\nBlood Sampling and Experimental Design\nBlood samples from 16 healthy volunteers (HRs n8; LRs n8)\nwere obtained under protocols approved by the regional committee\nfor medical research ethics. In time-course ex vivo experiments,\nLPS-stimulated and -nonstimulated blood aliquots were incubated\nfor different times with agitation at 37°C. For TF activity assay and\nWestern blotting MNCs were isolated using Lymphoprep (Axis-\nShield) according to manufacturer instructions. For TNF- quanti-\nfication, platelet-poor plasma was obtained from blood aliquots by\ncentrifugation for 10 minutes. For a detailed description, please see\nthe online supplement, available at http://atvb.ahajournals.org.\nQuantification of Monocyte TF Activity\nTF was measured in the in frozen/thawed preparations of MNCs\nusing a 2-stage clotting assay based on the ability of TF to accelerate\nthe activation of factor X by factor VIIa as described previously.19\nQuantification of TNF-\nThe concentration of TNF- in platelet-poor plasma was determined by\nusing PeliKine compact ELISA kit according to manufacturer instructions.\nAntibody Quality Control and Small\nInterfering RNA\nTo ascertain the suitability of the monoclonal anti-human TF antibody used\nin the present study, we used TF gene silencing by RNA interference. For\na detailed description, please see the online supplement.\nFlow Cytometry\nPopulation of monocytes expressing TF (CD14TF cells) was deter-\nmined by staining of whole blood aliquots treated as described in\nexperimental design, using phycoerythrin (PE)-conjugated anti-human\nCD14 monoclonal antibodies (BD Biosciences PharMingen) and\nAlexa488-conjugated mouse anti-human TF monoclonal antibodies.\nMouse monoclonal antibodies recognizing extracellular domain of\nhuman TF were prepared by the monoclonal antibody facility, Depart-\nment of Biochemistry, University of Vermont, Burlington. For a\ndetailed description of the procedure, please see the online supplement.\nFluorescence Confocal Microscopy\nAfter Lymphoprep isolation, MNCs were plated for 30 minutes onto\nflat-bottomed 8-well chambers and fixed. Part of the samples was\npermeabilized with 70% methanol. TF and monocytes were visual-\nized by immunostaining with Alexa488-conjugated mouse anti-\nhuman TF monoclonal antibodies and PE-conjugated mouse anti-\nhuman CD14 monoclonal antibodies. Nuclei of the cells were stained\nusing DRAQ5 dye (Biostatus, Ltd). A fluorescent laser scanning\nmicroscope (LSM510 META; Carl Zeiss AG) with C-Apochromat\n40/1.2 W lens was used for image acquisition.\nWestern Blotting\nTF protein levels in lysates of resting and LPS-stimulated 106 MNCs\nwere detected using 10% SDS-PAGE, and immunoblotting of nitrocel-\nlulose membranes with mouse anti-human TF monoclonal antibodies\nand horseradish peroxidase– conjugated secondary antibodies. We used\ndensitometrical reading of 43-kDa immunopositive band for statistical\ncomparisons. For details, please the online supplement.\nRT-PCR and Real-Time Polymerase\nChain Reaction\nTotal RNA was extracted from MNC pellets with an RNeasy kit\n(Qiagen). Thereafter, single-stranded cDNA was synthesized using\nTaqMan Gold RT-PCR Kit, and real-time PCR analyses of TF\nmRNA expression was performed in duplicates with assay-on-\ndemand real-time PCR kit using ABI PRISM 7700 Sequence\nDetection System (Applied Biosystems). For data normalization, we\nanalyzed the expression of 18S rRNA by using an assay-on-demand\nkit. For details, please see the online supplement.\nCell Count and Viability\nCell counts were performed on a Sysmex K1000 (TOA Medical\nElectronics Co Ltd) in a whole blood system and after isolation of\nMNCs. Cell viability was assessed by using a trypan blue dye exclusion\nassay according to manufacturer instructions (Invitrogen Corp).\nStatistics\nStatistical comparisons were performed using SigmaPlot 8.0 (SPPS,\nLtd.) and MS Excel. The presented data shown are meanSEM.\nComparisons between mean values were performed using the Stu-\ndent paired t test. P value 0.05 was considered significant.\nResults\nReactivity of MNCs in HRs and LRs\nThe LPS-induced responses of MNCs from individuals enrolled\nin our study have been repeatedly assessed to determine their\nindividual TF activity and concentration levels of TNF-. These\nhealthy individuals were allocated to HR or LR groups accord-\ning to criteria developed in our laboratory.16 Briefly, healthy\nvolunteers were defined as HRs if their MNC-TF activity\nexceeded 20 mU/106 cells and their plasma TNF- concentra-\ntion levels were 3 ng/mL taken from whole blood stimulated\nwith LPS for 2 hours. We assigned individuals with these\nparameters lower than indicated above to the LR group.\nMeasurements of TF procoagulant activity in lysates of\nMNCs isolated from ex vivo LPS-stimulated whole blood\nrevealed 2 distinct responses. In HRs, LPS stimulation\ninduced significantly higher values of MNC-TF activity than\nin LRs after 2 hours (50.114.3 versus 20.94.2 mU/106\ncells; P0.05) and 4 hours (56.016.3 mU/106 versus\n21.24.9 mU/106 cells; P0.05; Figure 1). We found no\nincrease in TF activity in MNC lysates from time-matched\nnonstimulated samples (Figure 1).\nEgorina et al\nTF in High and Low Responders\nThe differences of MNC-TF activity and the production of\nTNF- in our experimental model could not be caused by\nvariations in degree of cell death because the number of dead\ncells measured by trypan blue exclusion assay did not differ\nbetween groups and was within 3% of the total cell count.\nThe variable degrees of responsiveness to LPS treatment,\ncharacterized by significantly elevated TF activity in MNC\nlysates in HRs,\nthe comparisons of\nintracellular and surface TF antigen levels under resting and\nstimulated conditions in these individuals.\nled us to carry out\nBecause we planned to use immunostaining-based methods for\nnative TF antigen detection (such as fluorescence-activated cell\nsorting, fluorescence confocal microscopy and in-cell Western\nassay), we applied RNA interference against TF expression in\nplated HeLa cells. Similarly, the same antibodies detected a 3-fold\nreduction of the in-cell Western assay signal intensity in the wells\nthat contained HeLa cells transfected with siRNA against TF when\ncompared with cells transfected with control siRNA. The signal\nfrom the in-cell Western assay against GAPDH showed no differ-\nences in the same wells (Figure IB).\nTo characterize levels of TF antigen on the surface and\ninterior of resting and LPS-stimulated monocytes from HRs\nand LRs, we chose FACS analyses of whole blood, fluores-\ncence confocal microscopy, and in-cell Western assay of\npermeabilized and nonpermeabilized MNCs.\nArterioscler Thromb Vasc Biol.\nWe found no baseline difference in the surface expression\nof TF on nonstimulated CD14 cells from LRs (1.40.1%)\nand HRs (1.50.2%; Figure 3A and 3B). MFI did not differ\nbetween groups and was 4.50.3 AU and 4.20.1 AU in\nLRs and HRs, respectively (Figure 3C).\nBy use of fluorescence confocal microscopy of isolated\nMNCs in resting and LPS-stimulated conditions, we visual-\nized the distribution of TF antigen between cytoplasmic and\nmembrane compartments. Only a few CD14 cells expressed\nTF in permeabilized and nonpermeabilized conditions (Fig-\nure 4A and 4B). Furthermore, we noticed 2 phenotypes when\nimaging nonpermeabilized resting CD14 cells: 1 did not\nexpress TF on the membrane surface, whereas another did\n(Figure 4C and 4D). The pattern of TF surface distribution could\nbe characterized by the presence of 5 to 7 TF-enriched dots or\n“patches.” When permeabilized, resting CD14 cells have TF-\ncontaining speckles in the proximity of the plasma membrane,\nready to be exocytosed to the cell exterior (Figure 4E).\nAfter 2 hours of LPS stimulation, permeabilized mono-\ncytes had their membranes enriched with TF antigen. The\ncytoplasm was depleted from TF-containing granules except\nfor a strong TF-positive staining unilaterally from the nucleus\n(Figure 4F). This could be newly synthesized TF protein.\nTo verify TF antigen distribution data from flow cytometry\nand fluorescence confocal microscopy, we used a quantitative\nimmunofluorescence staining in-cell Western assay.\nsignificantly higher LPS-induced accumulation of intracellu-\nlar TF antigen in LRs compared with HRs (Figure 5B). At the\nsame time, LPS treatment caused a 4-fold increase of surface\nTF signal intensity in HRs. However, no such increase was\nfound in the surface TF signal of LRs (Figure 5B).\nData from the immunoblotting of MNC lysates against TF\nantigen confirmed that after 2 hours of LPS stimulation,\nMNCs of LRs indeed accumulated TF antigen, whereas in\nHRs,\nthis accumulation was minor (Figure 5C and 5D).\nMNCs from time-matched samples of LRs experienced vir-\ntually no accumulation of the TF antigen in contrast to the\nsignificant accumulation of TF in MNCs of HRs (Figure 5D).\nComparison of real-time PCR data on de novo TF synthesis in\nLPS-stimulated conditions revealed similar mean cycle threshold\nvalues in MNCs of HRs and LRs (Table I, available online at\nhttp://atvb.ahajournals.org). Therefore, LPS stimulation is not only\nnecessary for the induction of TF synthesis, but also for its transport\nvia exocytotic pathway and shedding from the cell membrane.\nDiscussion\nIn the present ex vivo study, using a whole blood system, we\nreport several novel observations characterizing the distribu-\ntion of TF antigen between the interior and surface of MNCs\nfrom HRs and LRs. MNCs from representatives of these\ngroups exhibit distinct patterns of intracellular TF accumula-\ntion, externalization onto cell membrane, and shedding of\nTF-containing microparticles in LPS-treated conditions.\nEgorina et al\nTF in High and Low Responders\nCD14 cells expose TF on the cell surface after LPS-\nstimulation when compared with LRs.\nData of FACS analyses were in accordance with our\nfindings from fluorescence confocal microscopy, in which we\ndemonstrated redistribution of TF antigen on stimulation with\nLPS onto the cell surface. In stimulated conditions, TF was\ntransported and localized mostly on the membrane because\nthere was a clear depletion of TF staining from the cytoplasm.\nIntense dots present unilaterally from the nucleus could\nrepresent de novo synthesized TF.\ntime-matched nonstimulated cells. Together, it is possible that\nmonocytes from HRs could synthesize equal or even higher\namounts of TF antigen than those from LRs.\nArterioscler Thromb Vasc Biol.\nImmunostaining against TF and fluorescent confocal mi-\ncroscopy of permeabilized and nonpermeabilized resting\nmonocytes support our data from flow cytometry because\nonly few monocytes expressed TF antigen. The localization\npattern of TF on the monocyte surface in the present study is\nconfined to dotted structures or “patches.” Intracellular local-\nized TF in CD14 cells was restricted to round vesicle-like\nstructures in the proximity of the plasma membrane. Mulder\net al demonstrated the presence of an intracellular TF pool\nassociated with caveolae and multivesicular bodies.24\nOur data obtained from FACS and immunoblotting are\nconsistent with the previous accepted opinion that resting human\nmonocytes contain nearly undetectable levels of TF antigen.25–30\nHowever, we provide new information about the relationship\nbetween the intracellular and surface pool of TF in MNCs by\nusing in-cell Western assay. This enzyme-independent method\nof antigen detection via antibodies conjugated with infrared dyes\ngives highly reproducible data characterizing antigen levels.\nComparisons of the TF signal from resting MNCs between\npermeabilized and nonpermeabilized conditions revealed that in\nresting conditions, MNCs contain equally distributed surface\nand intracellular TF antigen in HRs and LRs.\nIn conclusion, MNCs of HRs and LRs contain similar, nearly\nundetectable intracellular levels of TF antigen at baseline condi-\ntions. On LPS stimulation, TF is apparently transported faster to the\ncell surface and in a greater amount in the group of HRs. This group\nalso shed more of TF-containing microparticles. The cells of LRs\nhad a lower ability to be stimulated by LPS and retained more TF\nantigen, probably because of a slower exocytosis rate. Furthermore,\nLR monocytes express less TF antigen on their surface and shed\nfewer TF-containing microparticles. The findings of our present\nstudy indicate that elevated levels of the blood coagulation cascade\ninitiator on the surface of circulating monocytes and microparticles\nin HRs might make them more susceptible to hypercoagulation.",
         null
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "Potential Clindamycin Resistance in Clindamycin-Susceptible, Erythromycin-Resistant Staphylococcus aureus: Report of a Clinical Failure",
         "ABSTRACT The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin clinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-susceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of methicillin-resistant S. aureus strains were D-test positive.",
         "",
         "https://europepmc.org/articles/pmc549252?pdf=render",
         "The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin\nclinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-\nsusceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of\nmethicillin-resistant S. aureus strains were D-test positive.",
         "Clindamycin-susceptible, erythromycin-resistant Staphylo-\ncoccus aureus (clindamycin-erythromycin discordant) may de-\nvelop clindamycin resistance (7, 8). The erm gene product is a\nribosome methylase whose expression is normally minimal.\nErythromycin induces the production of this methylase, which\nis why these strains are erythromycin resistant, but mutations\nin the promoter region of erm allow production of methylase\nwithout an inducer (18, 19). These mutants are stably erythro-\nmycin and clindamycin resistant. Since erythromycin resistance\ncan occur with other mechanisms (e.g., efﬂux pumps and en-\nzymatic modiﬁcation) (15), the D-test identiﬁes inducible re-\nsistance that might presage mutational clindamycin constitu-\ntive resistance. The D-test is performed by placing clindamycin\nand erythromycin disks at an edge-to-edge distance of 15 to 20\nmm and looking for ﬂattening of the clindamycin zone nearest\nthe erythromycin disk (2). A positive D-test suggests the pres-\nence of an erm gene that could result in constitutive clindamy-\ncin resistance and clinical failure.\nThere are few published clinical failures of clindamycin with\nemergence of resistance (1, 3, 9, 12, 16). However, there are\nalso reports of successful use of clindamycin in treating pa-\ntients with D-test-positive isolates (3, 9). We report a clinical\nfailure with documented emergence of resistance. In order to\navoid poor clinical outcomes but retain the usefulness of clin-\ndamycin, it would be helpful to know the prevalence of induc-\nible resistance in clindamycin-erythromycin discordant bacte-\nria. This prevalence varies by geographic location, patient age,\nbacterial species, and bacterial susceptibility proﬁle (4, 5, 10,\n11, 13, 14, 16). For example, a pediatric population in Houston,\nTex., with methicillin-resistant S. aureus (MRSA) had a D-test\npositivity rate of 2.2% (9) compared to a similar population of\nchildren with MRSA in Chicago, Ill., whose D-test positivity\nrate was 94% (3). We determined the prevalence of D-test\npositivity in clindamycin-erythromycin discordant S. aureus iso-\nlates in our institution in early 2004. We tested isolates from\ninfected body sites where clindamycin might be considered for\ntherapy.\nThe patient was a 44-year-old man with fever, chills, and\nincreased low back pain 10 days after a lumbar discectomy. At\nadmission he had a temperature of 102.4°F and complained of\n3 days of erythema, bloody drainage, and pain along his inci-\nsion. He had a wound exploration, and the tissue Gram stain\nshowed white blood cells and gram-positive cocci. Fluid and\nblood cultures grew clindamycin-erythromycin discordant me-\nthicillin-susceptible S. aureus (MSSA), which was treated with\n2 weeks of oxacillin followed by 2 weeks of oral clindamycin.\nTen days after completing clindamycin therapy, he had severe\nback pain, chills, a temperature of 101.6°F, and purulent drain-\nage from his wound. A magnetic resonance image showed new\nevidence of lumbar osteomyelitis, discitis, and enhancement of\nparavertebral musculature with a small ﬂuid collection. On\nvancomycin, he had surgical debridement. Bone, blood, and\nwound cultures all grew MSSA with the same antibiogram as\nthat of his prior isolate except for new clindamycin resistance.\nHe successfully completed therapy with intravenous (oxacillin)\nand oral (combinations of minocycline, trimethoprim-sulfame-\nthoxazole, and rifampin) antibiotics for a total of 5 months.\nThe D-test on the original ﬂuid isolate was positive (Fig. 1);\nhowever, this result was not available at the time of his original\ntherapy. The original S. aureus wound isolate and the blood\nand bone isolates collected after completion of primary ther-\napy were indistinguishable by ribotyping with EcoRI restriction\nenzyme (Christiana Care Health Services, Wilmington, Del.).\nWe collected all 168 S. aureus isolates from wounds, ab-\nscesses, bone, cerebrospinal ﬂuid, pleural ﬂuid, ascitic ﬂuid,\nand joint ﬂuid from 1 January to 28 February 2004 at Temple\nUniversity Hospital (Philadelphia, Pa.). The strains (adult, pe-\ndiatric, inpatient, and outpatient) were stored in glycerol at\n74°C until batch testing in April 2004. Antibiotic susceptibil-\nities were determined by the Vitek-1 test (bioMerieux, Hazel-\nwood, Mo.); erythromycin-intermediate strains were considered\nresistant. The D-test was performed on clindamycin-erythromycin\ndiscordant strains by preparing an 0.5 McFarland suspension of\nbacteria in tryptic soy broth (2) inoculated on Mueller-Hinton\nplates incubated at 35°C for 18 to 24 h with erythromycin (15 g)\nand clindamycin (2 g) disks placed 17 mm apart (edge to edge).\nControls with known D-test positivity and negativity were also\ntested. Results were photographed and visually inspected by two\ninvestigators.\nNinety-one (54.2%) of 168 isolates were clindamycin-eryth-\nromycin discordant. There was 100% concordance of the D-\ntest results between the investigators. The majority (72.2%) of\nthe discordant strains were MRSA; however, only 8 (12.3%) of\n65 strains were D-test positive (Fig. 2). Twenty-ﬁve (27.5%) of\nthe 91 discordant strains were MSSA, but 17 (68%) of the 25\nstrains were D-test positive. Overall, 25 (27.5%) of the 91\ndiscordant strains were D-test positive.\nThe mechanisms of clindamycin resistance have been stud-\nied for years, and the D-test was developed to identify poten-\ntial clindamycin resistance so that possible ineffective therapy\nis not started when conventional tests show clindamycin MICs\nwithin the susceptible range (0.5 g/ml). Molecular markers\nfor the erm gene are available, but they are costly and incon-\nvenient for everyday use (8, 15, 17). The D-test is easy to\nperform and interpret, reproducible, and inexpensive but still\nnot universally used. There is a good correlation of standard\nD-tests with a modiﬁcation applied to purity plates done at the\ntime of specimen preparation for the Vitek-2 test (6).\nClindamycin is frequently used to treat skin and bone infec-\ntions because of its tolerability, cost, oral form, and good tissue\npenetration. With this high prevalence of D-test positivity, why\nare clindamycin clinical failures seldom reported? It may take\ntime for a mutant strain to develop, and the immune system\nmay have already controlled the infection. New agents active\nagainst gram-positive bacteria have been developed, so clinda-\nmycin may now be used less often. Finally, although a D-test-\npositive isolate may mutate on therapy, the mutation rate in\nclinical infections is unknown and may be rare.\nWe studied the prevalence of D-test positivity in isolates that\nmight be treated with clindamycin. We documented a clinda-\nmycin failure when resistance developed in a D-test-positive\nMSSA isolate. S. aureus clindamycin susceptibility reporting\nshould be suppressed when erythromycin resistance is found\n(unless a D-test is negative). If D-testing is delayed until re-\nsistance testing is complete, the results may not be available for\nmaximal clinical utility. Alternatively, the clinician may have to\nrequest it speciﬁcally.\nIt may be risky to use clindamycin when erythromycin testing\nshows a resistant or intermediate phenotype. The risk might\ncurrently be lower with MRSA, but this disparity may not\ncontinue or not be true elsewhere. Routine D-testing might\nallow clinicians to retain conﬁdence in clindamycin when eryth-\nromycin resistance is present.",
         "",
         "",
         "",
         "The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin\nclinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-\nsusceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of\nmethicillin-resistant S. aureus strains were D-test positive.\nClindamycin-susceptible, erythromycin-resistant Staphylo-\ncoccus aureus (clindamycin-erythromycin discordant) may de-\nvelop clindamycin resistance (7, 8). The erm gene product is a\nribosome methylase whose expression is normally minimal.\nErythromycin induces the production of this methylase, which\nis why these strains are erythromycin resistant, but mutations\nin the promoter region of erm allow production of methylase\nwithout an inducer (18, 19). These mutants are stably erythro-\nmycin and clindamycin resistant. Since erythromycin resistance\ncan occur with other mechanisms (e.g., efﬂux pumps and en-\nzymatic modiﬁcation) (15), the D-test identiﬁes inducible re-\nsistance that might presage mutational clindamycin constitu-\ntive resistance. The D-test is performed by placing clindamycin\nand erythromycin disks at an edge-to-edge distance of 15 to 20\nmm and looking for ﬂattening of the clindamycin zone nearest\nthe erythromycin disk (2). A positive D-test suggests the pres-\nence of an erm gene that could result in constitutive clindamy-\ncin resistance and clinical failure.\nThere are few published clinical failures of clindamycin with\nemergence of resistance (1, 3, 9, 12, 16). However, there are\nalso reports of successful use of clindamycin in treating pa-\ntients with D-test-positive isolates (3, 9). We report a clinical\nfailure with documented emergence of resistance. In order to\navoid poor clinical outcomes but retain the usefulness of clin-\ndamycin, it would be helpful to know the prevalence of induc-\nible resistance in clindamycin-erythromycin discordant bacte-\nria. This prevalence varies by geographic location, patient age,\nbacterial species, and bacterial susceptibility proﬁle (4, 5, 10,\n11, 13, 14, 16). For example, a pediatric population in Houston,\nTex., with methicillin-resistant S. aureus (MRSA) had a D-test\npositivity rate of 2.2% (9) compared to a similar population of\nchildren with MRSA in Chicago, Ill., whose D-test positivity\nrate was 94% (3). We determined the prevalence of D-test\npositivity in clindamycin-erythromycin discordant S. aureus iso-\nlates in our institution in early 2004. We tested isolates from\ninfected body sites where clindamycin might be considered for\ntherapy.\nThe patient was a 44-year-old man with fever, chills, and\nincreased low back pain 10 days after a lumbar discectomy. At\nadmission he had a temperature of 102.4°F and complained of\n3 days of erythema, bloody drainage, and pain along his inci-\nsion. He had a wound exploration, and the tissue Gram stain\nshowed white blood cells and gram-positive cocci. Fluid and\nblood cultures grew clindamycin-erythromycin discordant me-\nthicillin-susceptible S. aureus (MSSA), which was treated with\n2 weeks of oxacillin followed by 2 weeks of oral clindamycin.\nTen days after completing clindamycin therapy, he had severe\nback pain, chills, a temperature of 101.6°F, and purulent drain-\nage from his wound. A magnetic resonance image showed new\nevidence of lumbar osteomyelitis, discitis, and enhancement of\nparavertebral musculature with a small ﬂuid collection. On\nvancomycin, he had surgical debridement. Bone, blood, and\nwound cultures all grew MSSA with the same antibiogram as\nthat of his prior isolate except for new clindamycin resistance.\nHe successfully completed therapy with intravenous (oxacillin)\nand oral (combinations of minocycline, trimethoprim-sulfame-\nthoxazole, and rifampin) antibiotics for a total of 5 months.\nThe D-test on the original ﬂuid isolate was positive (Fig. 1);\nhowever, this result was not available at the time of his original\ntherapy. The original S. aureus wound isolate and the blood\nand bone isolates collected after completion of primary ther-\napy were indistinguishable by ribotyping with EcoRI restriction\nenzyme (Christiana Care Health Services, Wilmington, Del.).\nWe collected all 168 S. aureus isolates from wounds, ab-\nscesses, bone, cerebrospinal ﬂuid, pleural ﬂuid, ascitic ﬂuid,\nand joint ﬂuid from 1 January to 28 February 2004 at Temple\nUniversity Hospital (Philadelphia, Pa.). The strains (adult, pe-\ndiatric, inpatient, and outpatient) were stored in glycerol at\n74°C until batch testing in April 2004. Antibiotic susceptibil-\nities were determined by the Vitek-1 test (bioMerieux, Hazel-\nwood, Mo.); erythromycin-intermediate strains were considered\nresistant. The D-test was performed on clindamycin-erythromycin\ndiscordant strains by preparing an 0.5 McFarland suspension of\nbacteria in tryptic soy broth (2) inoculated on Mueller-Hinton\nplates incubated at 35°C for 18 to 24 h with erythromycin (15 g)\nand clindamycin (2 g) disks placed 17 mm apart (edge to edge).\nControls with known D-test positivity and negativity were also\ntested. Results were photographed and visually inspected by two\ninvestigators.\nNinety-one (54.2%) of 168 isolates were clindamycin-eryth-\nromycin discordant. There was 100% concordance of the D-\ntest results between the investigators. The majority (72.2%) of\nthe discordant strains were MRSA; however, only 8 (12.3%) of\n65 strains were D-test positive (Fig. 2). Twenty-ﬁve (27.5%) of\nthe 91 discordant strains were MSSA, but 17 (68%) of the 25\nstrains were D-test positive. Overall, 25 (27.5%) of the 91\ndiscordant strains were D-test positive.\nThe mechanisms of clindamycin resistance have been stud-\nied for years, and the D-test was developed to identify poten-\ntial clindamycin resistance so that possible ineffective therapy\nis not started when conventional tests show clindamycin MICs\nwithin the susceptible range (0.5 g/ml). Molecular markers\nfor the erm gene are available, but they are costly and incon-\nvenient for everyday use (8, 15, 17). The D-test is easy to\nperform and interpret, reproducible, and inexpensive but still\nnot universally used. There is a good correlation of standard\nD-tests with a modiﬁcation applied to purity plates done at the\ntime of specimen preparation for the Vitek-2 test (6).\nClindamycin is frequently used to treat skin and bone infec-\ntions because of its tolerability, cost, oral form, and good tissue\npenetration. With this high prevalence of D-test positivity, why\nare clindamycin clinical failures seldom reported? It may take\ntime for a mutant strain to develop, and the immune system\nmay have already controlled the infection. New agents active\nagainst gram-positive bacteria have been developed, so clinda-\nmycin may now be used less often. Finally, although a D-test-\npositive isolate may mutate on therapy, the mutation rate in\nclinical infections is unknown and may be rare.\nWe studied the prevalence of D-test positivity in isolates that\nmight be treated with clindamycin. We documented a clinda-\nmycin failure when resistance developed in a D-test-positive\nMSSA isolate. S. aureus clindamycin susceptibility reporting\nshould be suppressed when erythromycin resistance is found\n(unless a D-test is negative). If D-testing is delayed until re-\nsistance testing is complete, the results may not be available for\nmaximal clinical utility. Alternatively, the clinician may have to\nrequest it speciﬁcally.\nIt may be risky to use clindamycin when erythromycin testing\nshows a resistant or intermediate phenotype. The risk might\ncurrently be lower with MRSA, but this disparity may not\ncontinue or not be true elsewhere. Routine D-testing might\nallow clinicians to retain conﬁdence in clindamycin when eryth-\nromycin resistance is present.",
         null
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         "Symmetric and asymmetric Slepian-Wolf codes with systematic and nonsystematic linear codes",
         "We propose a constructive approach for distributed source coding of correlated binary sources using linear channel codes that can achieve any point of the Slepian-Wolf achievable rate region. Our approach is very intuitive and can be used with systematic and nonsystematic linear codes. Moreover, the proposed coding strategy can easily be extended to the case of more than two sources.",
         "Design of Slepian-Wolf codes by channel code partitioning On some new approaches to practical Slepian-Wolf compression inspired by channel coding Distributed code constructions for the entire Slepian-Wolf rate region for arbitrarily correlated sources Compression of binary sources with side information at the decoder using LDPC codes Compression with side information using turbo codes Compression of correlated binary sources using turbo codes The capacity of low-density parity-check codes under message-passing decoding Distributed source coding: symmetric rates and applications to sensor networks Distributed source coding using syndromes (DISCUS): design and construction Noiseless coding of correlated information sources",
         "http://www.commsp.ee.ic.ac.uk/~ng1/pdf/gehrigD_CL_Jan05.pdf",
         "Notice that similar approaches have recently been proposed\nin [7], [8] and [9]. Although our own approach is relatively\nsimilar in spirit to the one in [7], our scheme can also be\nused with non-systematic codes, whereas their technique can\nonly be used with systematic codes. Note that good capacity\nachieving LDPC codes are usually non-systematic. In [8]\nand [9], iterative decoding procedures are proposed in order\nto decode the two correlated blocks simultaneously. Their\nmain strategy is to apply the standard sum-product algorithm\n(message-passing decoding [10]) on an extended factor graph,\ncorresponding to two standard LDPC decoders connected\nthrough correlation nodes modeling the joint distribution be-\ntween the sources. In Section III, we show that our approach\ndoes not require the use of such extended graphs since our\nmethods only needs to decode one single block (the difference\npattern). A standard iterative decoding scheme similar to the\none proposed in [5] can therefore be used in our case. Our\napproach can thus be seen as an intuitive extension of the\nasymmetric approach proposed in [5] in order to achieve the\nentire Slepian-Wolf achievable rate region.",
         "We know that a codeword x belongs to the Hamming code C\nif and only if its syndrome is equal to zero:\nsx = HxT = 0 ⇐⇒ x ∈ C.\n(2)\nThe minimum distance between any two of the 16 codewords\nof the Hamming code is three. This code is therefore able to\ncorrect up to one bit error per codeword. Assume ei is the\nerror pattern corresponding to an error at bit position i (ei has\nsix 0s and one 1 at position i). We deﬁne y = x ⊕ ei where\n⊕ corresponds to the binary addition. This codeword y does\nclearly not belong to C since its distance from x is equal to\none. Its syndrome is given by:\nsy = HyT = H(x ⊕ ei)T = HxT ⊕ HeT\ni = HeT\ni .\nj respectively. We can see that:\nk = H(x ⊕ y)T = HxT ⊕ HyT = sx ⊕ sy.\nx = [xa xb]\nyb]\nH = [Ha Hb]\nsT\nx ] and encoder 2: [y3\nxa\nEncoder 1\nEncoder 2\nsy\nb (sx ⊕ HaxT\nIII. CONSTRUCTIVE APPROACH\nUSING ANY LINEAR CHANNEL CODE\nAssume we have an (n, k) binary linear code C with parity\ncheck matrix H in its reduced form such that: H = [H1 H2],\nwhere H1 and H2 are of size (n − k × k) and (n − k × n − k)\nrespectively. Assume without loss of generality that H2 is non\nsingular. Notice that if the code is systematic, we know that\nthe generator matrix G is of the form: G = [Ik P], where P\nis of size (k×n−k). The parity check matrix H has to satisfy:\nGHT = 0 and can therefore be given by: H = [−PT\nIn−k].\nH2 is thus simply the identity matrix in that case.\nAssume C is able to correct up to M errors per n-bit code\nblock. We know that the following relation must hold:\n(6)\nThis code C generates 2n−k cosets each containing 2k code-\nwords of length n. We know that x belongs to the coset i,\nsuch that sx = HxT = HeT\n(ei is the coset leader of coset\ni\nnumber i, i.e., the codeword with minimum weight).\nbi\nLet xi be a binary block of length n represented as: xi =\nqi], where ai, bi and qi are of length k1, k2 and n−k\n[ai\nrespectively (k1 and k2 are chosen such that their sum is equal\nbi]T ⊕\nto k). The syndrome of xi is deﬁned as: si = H1[ai\nH2qT\ni .\nConsider now two n-bit blocks x1 and x2, correlated such\nthat their Hamming distance dH (x1, x2) is at most m (Assume\nthat M ≥ m). Our distributed coding strategy consists in\nsT\n2 ] from the encoders 1 and\nsending only [a1\n2 respectively. The transmission bit-rates are therefore given\nby: R1 = n − k2 bits and R2 = n − k1 bits, corresponding to\na total of 2n − k bits.\nsT\n1 ] and [b2\nd = H(xT\nAt the receiver, we let ed correspond to the “difference\npattern” between x1 and x2 as: ed = x1 ⊕ x2. We know that\nthe syndrome of ed is given by sd = HeT\n2 ) =\ns1⊕s2. We can now retrieve the error pattern ed corresponding\nto this syndrome sd using one of the following techniques:\nIf the code is not too large, a simple lookup table storing\nthe corresponding pattern error for each possible syndrome\ncan be used. For larger code, an iterative method has to be\nused. Using an iterative decoding scheme such as the one\nproposed in [5], we can recover ed as the closest codeword\nto the all zero sequence satisfying the syndrome sd. Notice\nthat this iterative decoding approach is particularly suited for\nLDPC codes which are amongst the best block codes known\nfor memoryless channels [10].\nb1]T ⊕ H2qT\nb1]T . We can now retrieve q1 by computing: qT\nk ﬁrst bits of x1 and x2 are easily obtained as: [a2\n[a1\n2 z1. Notice that H−1\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\nProposition 2: Assume x1, . . . , xL are L binary sequences\nlength n correlated such that\nthe Hamming distance\nof\nbetween two consecutive sequences is at most m (i.e.,\ndH (xi, xi+1) ≤ m for i = 1, . . . , L − 1). Consider an (n, k)\nlinear channel code C that can correct up to M ≥ m errors per\nn-bit block. The following distributed coding strategy uses a\ntotal of n + (L − 1)(n − k) bits to encode the L sequences\nand is sufﬁcient to allow for a perfect reconstruction of all of\nthem at the decoder:\n From each encoder, send the syndrome si of the corre-\n Send only complementary subsets of their ﬁrst k bits such\nthat each bit position is sent from only one encoder.\nthe L − 1 difference patterns can be\nrecovered from the L syndromes, allowing then to complete\nthe ﬁrst k bits of each sequence. The decoding method used\nhere is similar to the one presented in the previous section.\nThe difference pattern of each pair of consecutive blocks is\nretrieved by running the standard iterative decoding method\nwith the sum of the two syndromes. Knowing all the difference\npatterns and having received complementary subsets of the\nﬁrst k bits, the ﬁrst k bits of each block can then be recovered.\nFinally, each original block xi := [ai\nqi] (i = 1, . . . , L) is\ncompleted by recovering its last n − k bits as:\ni = H−1\nqT\n2 (si ⊕ H1aT\n Send the syndromes of X and Y from their respective\nIt is possible to show that this coding strategy is optimal\nfor some particular cases.",
         "",
         "",
         "",
         "Notice that similar approaches have recently been proposed\nin [7], [8] and [9]. Although our own approach is relatively\nsimilar in spirit to the one in [7], our scheme can also be\nused with non-systematic codes, whereas their technique can\nonly be used with systematic codes. Note that good capacity\nachieving LDPC codes are usually non-systematic. In [8]\nand [9], iterative decoding procedures are proposed in order\nto decode the two correlated blocks simultaneously. Their\nmain strategy is to apply the standard sum-product algorithm\n(message-passing decoding [10]) on an extended factor graph,\ncorresponding to two standard LDPC decoders connected\nthrough correlation nodes modeling the joint distribution be-\ntween the sources. In Section III, we show that our approach\ndoes not require the use of such extended graphs since our\nmethods only needs to decode one single block (the difference\npattern). A standard iterative decoding scheme similar to the\none proposed in [5] can therefore be used in our case. Our\napproach can thus be seen as an intuitive extension of the\nasymmetric approach proposed in [5] in order to achieve the\nentire Slepian-Wolf achievable rate region.\nWe know that a codeword x belongs to the Hamming code C\nif and only if its syndrome is equal to zero:\nsx = HxT = 0 ⇐⇒ x ∈ C.\n(2)\nThe minimum distance between any two of the 16 codewords\nof the Hamming code is three. This code is therefore able to\ncorrect up to one bit error per codeword. Assume ei is the\nerror pattern corresponding to an error at bit position i (ei has\nsix 0s and one 1 at position i). We deﬁne y = x ⊕ ei where\n⊕ corresponds to the binary addition. This codeword y does\nclearly not belong to C since its distance from x is equal to\none. Its syndrome is given by:\nsy = HyT = H(x ⊕ ei)T = HxT ⊕ HeT\ni = HeT\ni .\nj respectively. We can see that:\nk = H(x ⊕ y)T = HxT ⊕ HyT = sx ⊕ sy.\nx = [xa xb]\nyb]\nH = [Ha Hb]\nsT\nx ] and encoder 2: [y3\nxa\nEncoder 1\nEncoder 2\nsy\nb (sx ⊕ HaxT\nIII. CONSTRUCTIVE APPROACH\nUSING ANY LINEAR CHANNEL CODE\nAssume we have an (n, k) binary linear code C with parity\ncheck matrix H in its reduced form such that: H = [H1 H2],\nwhere H1 and H2 are of size (n − k × k) and (n − k × n − k)\nrespectively. Assume without loss of generality that H2 is non\nsingular. Notice that if the code is systematic, we know that\nthe generator matrix G is of the form: G = [Ik P], where P\nis of size (k×n−k). The parity check matrix H has to satisfy:\nGHT = 0 and can therefore be given by: H = [−PT\nIn−k].\nH2 is thus simply the identity matrix in that case.\nAssume C is able to correct up to M errors per n-bit code\nblock. We know that the following relation must hold:\n(6)\nThis code C generates 2n−k cosets each containing 2k code-\nwords of length n. We know that x belongs to the coset i,\nsuch that sx = HxT = HeT\n(ei is the coset leader of coset\ni\nnumber i, i.e., the codeword with minimum weight).\nbi\nLet xi be a binary block of length n represented as: xi =\nqi], where ai, bi and qi are of length k1, k2 and n−k\n[ai\nrespectively (k1 and k2 are chosen such that their sum is equal\nbi]T ⊕\nto k). The syndrome of xi is deﬁned as: si = H1[ai\nH2qT\ni .\nConsider now two n-bit blocks x1 and x2, correlated such\nthat their Hamming distance dH (x1, x2) is at most m (Assume\nthat M ≥ m). Our distributed coding strategy consists in\nsT\n2 ] from the encoders 1 and\nsending only [a1\n2 respectively. The transmission bit-rates are therefore given\nby: R1 = n − k2 bits and R2 = n − k1 bits, corresponding to\na total of 2n − k bits.\nsT\n1 ] and [b2\nd = H(xT\nAt the receiver, we let ed correspond to the “difference\npattern” between x1 and x2 as: ed = x1 ⊕ x2. We know that\nthe syndrome of ed is given by sd = HeT\n2 ) =\ns1⊕s2. We can now retrieve the error pattern ed corresponding\nto this syndrome sd using one of the following techniques:\nIf the code is not too large, a simple lookup table storing\nthe corresponding pattern error for each possible syndrome\ncan be used. For larger code, an iterative method has to be\nused. Using an iterative decoding scheme such as the one\nproposed in [5], we can recover ed as the closest codeword\nto the all zero sequence satisfying the syndrome sd. Notice\nthat this iterative decoding approach is particularly suited for\nLDPC codes which are amongst the best block codes known\nfor memoryless channels [10].\nb1]T ⊕ H2qT\nb1]T . We can now retrieve q1 by computing: qT\nk ﬁrst bits of x1 and x2 are easily obtained as: [a2\n[a1\n2 z1. Notice that H−1\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\nProposition 2: Assume x1, . . . , xL are L binary sequences\nlength n correlated such that\nthe Hamming distance\nof\nbetween two consecutive sequences is at most m (i.e.,\ndH (xi, xi+1) ≤ m for i = 1, . . . , L − 1). Consider an (n, k)\nlinear channel code C that can correct up to M ≥ m errors per\nn-bit block. The following distributed coding strategy uses a\ntotal of n + (L − 1)(n − k) bits to encode the L sequences\nand is sufﬁcient to allow for a perfect reconstruction of all of\nthem at the decoder:\n From each encoder, send the syndrome si of the corre-\n Send only complementary subsets of their ﬁrst k bits such\nthat each bit position is sent from only one encoder.\nthe L − 1 difference patterns can be\nrecovered from the L syndromes, allowing then to complete\nthe ﬁrst k bits of each sequence. The decoding method used\nhere is similar to the one presented in the previous section.\nThe difference pattern of each pair of consecutive blocks is\nretrieved by running the standard iterative decoding method\nwith the sum of the two syndromes. Knowing all the difference\npatterns and having received complementary subsets of the\nﬁrst k bits, the ﬁrst k bits of each block can then be recovered.\nFinally, each original block xi := [ai\nqi] (i = 1, . . . , L) is\ncompleted by recovering its last n − k bits as:\ni = H−1\nqT\n2 (si ⊕ H1aT\n Send the syndromes of X and Y from their respective\nIt is possible to show that this coding strategy is optimal\nfor some particular cases.",
         null
        ],
        [
         "9",
         "000a2234245d642c786a7355a15623444c33c1dc",
         "Selective Defect in Antigen-Induced TCR Internalization at the Immune Synapse of CD8 T Cells Bearing the ZAP-70(Y292F) Mutation1",
         "Cbl proteins have been implicated in ligand-induced TCR/CD3 down-modulation, but underlying mechanisms are unclear. We analyzed the effect of mutation of a cbl-binding site on ZAP-70 (ZAP-Y292F) on dynamics, internalization, and degradation of the TCR/CD3 complex in response to distinct stimuli. Naive CD8 T cells expressing the P14 transgenic TCR from ZAP-Y292F mice were selectively affected in TCR/CD3 down-modulation in response to antigenic stimulation, whereas neither anti-CD3 Ab-, and PMA-induced TCR down-modulation, nor constitutive receptor endocytosis/cycling were impaired. We further established that the defect in TCR/CD3 down-modulation in response to Ag was paralleled by an impaired TCR/CD3 internalization and CD3ζ degradation. Analysis of T/APC conjugates revealed that delayed redistribution of TCR at the T/APC contact zone was paralleled by a delay in TCR internalization in the synaptic zone in ZAP-Y292F compared with ZAP-wild-type T cells. Cbl recruitment to the synapse was also retarded in ZAP-Y292F T cells, although F-actin and LFA-1 redistribution was similar for both cell types. This study identifies a step involving ZAP-70/cbl interaction that is critical for rapid internalization of the TCR/CD3 complex at the CD8 T cell/APC synapse.",
         "",
         "https://www.jimmunol.org/content/jimmunol/175/5/3140.full.pdf",
         "Cbl proteins have been implicated in ligand-induced TCR/CD3 down-modulation, but underlying mechanisms are unclear. We\nanalyzed the effect of mutation of a cbl-binding site on ZAP-70 (ZAP-Y292F) on dynamics, internalization, and degradation of the\nTCR/CD3 complex in response to distinct stimuli. Naive CD8 T cells expressing the P14 transgenic TCR from ZAP-Y292F mice\nwere selectively affected in TCR/CD3 down-modulation in response to antigenic stimulation, whereas neither anti-CD3 Ab-, and\nPMA-induced TCR down-modulation, nor constitutive receptor endocytosis/cycling were impaired. We further established that\nthe defect in TCR/CD3 down-modulation in response to Ag was paralleled by an impaired TCR/CD3 internalization and CD3\ndegradation. Analysis of T/APC conjugates revealed that delayed redistribution of TCR at the T/APC contact zone was paralleled\nby a delay in TCR internalization in the synaptic zone in ZAP-Y292F compared with ZAP-wild-type T cells. Cbl recruitment to\nthe synapse was also retarded in ZAP-Y292F T cells, although F-actin and LFA-1 redistribution was similar for both cell types.\nThis study identiﬁes a step involving ZAP-70/cbl interaction that is critical for rapid internalization of the TCR/CD3 complex at\nthe CD8 T cell/APC synapse. The Journal of Immunology, 2005, 175: 3140 –3149.",
         "face when associated with the CD3 complex composed of\nthe CD3/CD3, CD3/CD3, and CD3- dimers. Its\nlevel of surface expression is ﬁnely tuned by mechanisms control-\nling its assembly, membrane dynamics, and degradation, which\nvary along T cell development and as a consequence of T cell\nactivation. In particular, T cell stimulation by APC presenting the\ncognate peptide/MHC complex leads to a rapid decrease in the\nlevel of surface-expressed TCR. This phenomenon, known as\nTCR/CD3 down-modulation, occurs mainly by internalization of\nthe whole receptor complex through endosomal pathways (1, 2),\nalthough mechanisms invoking exocytosis of vesicles containing\nthe receptor have also been described (3).\nAg-induced TCR down-modulation is intimately linked to TCR\nsignaling and, in particular, to protein tyrosine kinase (PTK)4 ac-\ntivation, including Src family PTKs p56lck and p59fyn. These ki-\n4 Abbreviations used in this paper: PTK, protein tyrosine kinase; PKC, protein kinase\nC; IS, immune synapse; SMAC, supramolecular activation cluster; c-SMAC, central\nSMAC; p-SMAC, peripheral SMAC; CIN85, Cbl-interacting protein of 85 kDa; RTK,\nreceptor tyrosine kinase; CD2AP, CD2-associated protein; LN, lymph node; CTX,\ncholera toxin; MFI, mean ﬂuorescence intensity; PAF, paraformaldehyde; RT, room\ntemperature; BFA, brefeldin A; WT, wild type; AT,\nacid treated; NT,\nnot treated.\nnases phosphorylate tyrosines in the ITAM motifs of CD3 cyto-\nplasmic domains, allowing for binding of the ZAP-70 PTK via its\nSH2 domains. Subsequent phosphorylation of adaptor molecules\nand protein-protein interactions further transduce positive and neg-\native signals for functional activation of the T cells (4). In this\ncontext, TCR/CD3 down-modulation has been suggested to be one\nof the mechanisms that contribute to the arrest of signaling initi-\nated by Ag. Alternatively, it could be reasoned that the failure to\ndisengage peptide/MHC complexes by receptor clearance would\nprevent the serial engagement of new receptors that is required for\noptimal T cell stimulation by the limiting amount of peptide/MHC\ncomplexes available on the APCs (4, 5).\nApproaches at unraveling the complexities of TCR dynamics\nand signaling have included the analysis of T cells expressing\nTCR/CD3 complexes modiﬁed at potential internalization motifs,\nor in components of the receptor endocytosis/degradation pathway.\nIn particular, each CD3 cytoplasmic domain contains tyrosine-\nbased motifs in their ITAMs that are able to bind to the \n2 chain\nof the clathrin adaptor AP2 (6). Furthermore, the CD3 and CD3\nchains each present a di-leucine motif (7) capable of binding the \n\nsubunit of the AP2 complex (8). The CD3 chain di-leucine motif\nappears necessary for TCR/CD3 constitutive cycling (9), whereas\nthe proximal serine-126 on CD3is additionally required for phor-\nbol ester-induced TCR/CD3 internalization (10), a protein kinase\nC (PKC)-dependent event\ninvolves receptor sequestration\nwithout subsequent degradation (11). For mAb-induced TCR/CD3\ndown-regulation, a process dependent upon tyrosine kinase activ-\nity (12–14), but not PKC, absence of both CD3 and CD3 cyto-\nplasmic domains led to deﬁcient receptor down-modulation, sug-\ngesting that upon T cell activation, unveiling of the di-leucine and\ntyrosine-based motifs present in each of these chains may be in-\nvolved (1). However, Ag-induced TCR down-modulation was not\naffected by the combined truncations of CD3 and CD3 cyto-\nplasmic domains (15), suggesting that a distinct mechanism in-\nvolving ZAP-70 activity (14), was rate limiting in this process.\nthat\nThe Journal of Immunology\nThe particularity of Ag-induced TCR down-modulation is that it\noccurs upon engagement of the TCR by peptide/MHC complexes\nwithin the immune synapse (IS) formed at the T cell/APC contact\nzone. This specialized structure, associated with a redistribution of\nmembrane receptors, develops over a 5–30 min period. It subdi-\nvides the T cell/APC interface into supramolecular activation clus-\nters (SMACs) that are spatially organized into a central (c-SMAC)\nregion where TCR-MHC/peptide and CD28-CD80 interactions\ntake place, and a peripheral (p-SMAC) area enriched in cytoskel-\netal components and adhesion molecules such as LFA-1 (16, 17),\nand CD2 (18). However, the rapid induction of PTK activation\nupon Ag-induced T cell stimulation precedes the redistribution of\nthe TCR/CD3 complex in the c-SMAC (19 –21). The observation\nthat surface TCR disappeared from the c-SMAC may signify its\nlocalized internalization (21).\nTyrosine-292 in ZAP-70, when phosphorylated upon TCR ac-\ntivation, is a binding site for c-cbl and cbl-b via the cbl tyrosine\nkinase binding domain (22–24). Contrasting results were reported\nfor thymocytes of mice expressing an altered c-cbl mutated in its\ntyrosine kinase binding domain, because this led to constitutive\nactivation of rac without affecting ZAP-70 phosphorylation (25).\nHowever, the effect of this mutation could not be studied in pe-\nripheral T cells, where mostly cbl-b is expressed.\nAdaptor proteins of the Cbl-interacting protein of 85 kDa\n(CIN85) family have recently been shown to bind both cbl and\nendophilins, and thereby to regulate the endocytosis of activated\nreceptor tyrosine kinases (RTK) (26 –29). Cbl is also involved in\nRTK monoubiquitination allowing for receptor binding to the\nclathrin adaptor Eps15 (28, 30, 31). In T cells, the CIN85 family\nmember CD2-associated protein (CD2AP) has recently been im-\nplicated in the pathway of TCR/CD3 down-modulation (32, 33).\nFurthermore, we have recently described that T cells from mice\nexpressing ZAP-Y292F presented hyper-tyrosine-phosphorylation\nin response to TCR/CD3 engagement but were partially inhibited\nin Ag-induced TCR down-modulation (34).\nIn the present work, we analyzed the mechanisms by which\nTCR down-modulation is impaired in CD8 T cells expressing the\nZAP-Y292F mutation. Our results showed that the defect in TCR\ndown-modulation was speciﬁc for Ag-induced TCR internalization\nand was associated with a delayed recruitment of TCR and cbl at\nthe T/APC synapse zone.\nMaterials and Methods\nMice\nAg-presenting cells\nTap-2-deﬁcient RMA-S H-2b cells (35) were generally used as APCs, and\nsimilar results were obtained when LPS blasts from Tap-1°/° mice were\nused (results not shown). Before loading with peptide (LCMV gp33 (36) or\ncontrol peptide HY (37)), Tap-negative cells were incubated overnight at\n31°C in RPMI 1640 with 5% dialyzed FCS.\nT cell preparation\nAbs and reagents for immunoﬂuorescence\nvisualized with the cholera toxin -chain (CTX) coupled to FITC (Sigma-\nAldrich), or Alexa488 (Molecular Probes). F-actin was visualized using\nphalloidin Alexa488 (Molecular Probes). Rabbit anti-cbl Ring domain was\nprepared as described (26), and revealed with anti-rabbit Ig Zenon A546\n(Molecular Probes). The speciﬁcity of the Ab for cbl was further estab-\nlished in T cell lysates, because only the cbl band (at 120 kDa) was de-\ntected by immunoblot (results not shown).\nFluorescence analysis\nMeasurement of surface ﬂuorescence and FACS analysis were as described\nusing mean ﬂuorescence intensity (MFI) values (15). TCR down-modula-\ntion was calculated as (1  MFI experimental/MFI medium)  100. For\nintracellular CD3 staining, cells were washed (4 min; 4°C) in PBS before\n5-min ﬁxation with 2% paraformaldehyde (PAF) at room temperature\n(RT), washed again with PBS, and permeabilized for 10 min at RT with\n0.5% saponin in PBS, 2% FCS, and 0.02% NaN3 containing 1 mM van-\nadate (saponin buffer). Biotinylated mAb H146.968 was diluted in the same\nsaponin buffer and added for 30 min at 4°C. After permeabilization, cells\nwere washed and centrifuged at 2000 rpm for 5 min at 4°C. For calibration,\nwe set up optimal mAb dilutions to obtain a linear decrease in labeling in\nthe presence of increasing amounts of the CD3 peptide recognized by the\nmAb. CD3 degradation was calculated as follows: (1  MFI experimen-\ntal/MFI medium)  100.\nMeasurement of internalized TCR\nOur approach involved ﬁrst biotinylation of cell surface proteins with SS-\nbiotin to discriminate between surface and internalized TCR (45, 46).\nthis treatment prevented subsequent Ag-induced P14-TCR\nHowever,\ndown-modulation and T cell activation as measured by CD69 expression\n(results not shown). We then chose to prelabel T cells with anti-TCR mAb\nat 4°C, before Ag stimulation at 37°C, followed by acid treatment to re-\nlease mAb remaining at the cell surface (as described in Ref. 2). The\nanti-TCRV2 B20.1 mAb was unsuitable for this approach because it rap-\nidly dissociates from the TCR. Therefore anti-TCR (H57), which stably\nbinds the TCR, was used for this study, as follows. T cells were incubated\nat 4°C with anti-TCR(H57) mAb A488 (Molecular Probes kit) before Ag\nstimulation at 37°C. At different times, samples were divided, and one-half\nwas directly used to measure total ﬂuorescence, the other half was treated\nwith an acid solution (0.15 M acetic acid, 0.15 M NaCl) for 5 min on ice\nbefore two washes in medium and measurement of remaining ﬂuorescence\ncorresponding to internalized mAb (2). Prelabeling of T cells with anti-\nTCR (H57) did not per se activate the P14 T cells and did not prevent\nsubsequent Ag-induced TCR down-modulation and signaling for CD69\nexpression (see Fig. 2).\nConfocal analysis\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nResults\nAg-induced TCR down-modulation is defective in ZAP-Y292F\ncompared with ZAP-wild-type (WT) CD8 T cells\nOur previous results indicated that TCR down-modulation in re-\nsponse to antigenic stimulation was partially inhibited in CD8 T\ncells from mice expressing ZAP-Y292F molecules (34). We ﬁrst\nconﬁrmed this ﬁnding in a kinetic experiment shown in Fig. 1. As\nseen in Fig. 1A, exposure of ZAP-WT CD8 T cells to APC loaded\nwith the gp33 LCMV peptide at 107 M decreased the level of\nsurface TCR from 830 to 400 MFI units after 60 min, and to MFI\n113 after 270 min. A similar treatment of the ZAP-Y292F CD8 T\ncells decreased the level of surface TCR from 870 to 590 after 60\nmin, and to 330 after 270 min. The rate of TCR down-modulation\nin response to gp33 107 M/APC by the ZAP-Y292F CD8 T cells\nwas about half (20%/h) that observed for the ZAP-WT CD8 T cells\n(38%/h) (Fig. 1B). The large SDs of the 60-min time points cor-\nrespond to the variability of TCR down-modulation observed from\none experiment to the other at that early time, but the respective\nrelative values for mutant to WT T cells were very reproducible in\nthe different experiments (decreased rate of TCR down-modulation\n52  11%). The loading of the RMA-S used as APC was per-\nformed with gp33 concentrations between 107 and 109 M, and\nwe observed a linear dose effect on TCR down-modulation and\nactivation, with a differential rate of TCR down-modulation be-\ntween ZAP-WT and ZAP-Y292F T cells maintained at all con-\ncentrations tested (results not shown).\nWe next analyzed the effect of this mutation on other means of\ninducing TCR down-modulation, such as anti-CD3 mAb or PMA\nstimulations (15, 47). The rate of TCR down-modulation induced\nby anti-CD3 mAb was the same for the ZAP-WT and ZAP-Y292F\nT cells, i.e., 55%/h (Fig. 1B). Receptor down-modulation was\nalso similar for the two types of T cells at any dose of anti-CD3\nmAb used, whether soluble (Fig. 1C) or plastic-coated (results not\nshown). When cells were stimulated by PMA, we also observed a\nsimilar rate of TCR down-modulation for both strains (results not\nshown) (see Fig. 2B).\nAll experiments were controlled for T cell activation by mea-\nsuring the induction of CD69, and we conﬁrmed that there was no\ndifference between the two strains in that assay (34).\nIn conclusion, the ZAP-Y292F mutation caused a reduced rate\n(40 –50% decrease) of TCR down-modulation in response to an-\ntigenic stimulation compared with that observed in ZAP-WT T\ncells within a range of antigenic peptide from 107 to 109 M.\nFurthermore, this study showed that the ZAP-Y292F mutation se-\nlectively affects Ag-induced, and not anti-CD3 mAb- or PMA-\ninduced receptor down-modulation.\nUse of brefeldin A (BFA) revealed that the ZAP-Y292F mutation\ndoes not affect spontaneous TCR cycling and shows no recycling\nof anti-CD3 mAb- or Ag-induced TCR down-modulation in both\ntypes of CD8 T cells\nConstitutive TCR endocytosis and cycling can be demonstrated by\nthe effect on receptor surface expression of BFA, which inhibits\nexport from the endoplasmic reticulum/Golgi and endosome recy-\ncling (45). We have examined the effect of BFA on constitutive\nTCR down-modulation, as well as on Ag- or anti-CD3-induced\nTCR down-modulation of ZAP-WT and ZAP-Y292F CD8 T cells.\nResults shown in Table I are expressed as the ratio of TCR down-\na BFA was added or not to T cells for 1 h before and during the down-modulation\nexperiment, in medium (constitutive cycling), with APC (Ag-induced down-modu-\nlation), or with anti-CD3 mAb (CD3-induced down-modulation). Results are pre-\nsented for the 120-min time point, and are expressed as the ratio between TCR down-\nmodulation in the absence and presence of BFA.\nThe Journal of Immunology\nmodulation between untreated and BFA-treated T cells. As previ-\nously described for naive T cells (48), P14 CD8 T cells do not\nthrive when cultured unstimulated at 37°C. They showed a gradual\ndecrease of TCR expression (5%/h), which was slower than the\nrate previously described for naive CD4 T cells (45). The presence\nof BFA increased receptor down-modulation (15–20%/h), so that,\nafter 2 h, the ratio of TCR down-modulation between untreated\nand BFA-treated T cells was of 0.34 for ZAP-WT and 0.38 for\nZAP-Y292F T cells, respectively (Table I), indicating that both\ntypes of T cells have\nconstitutive\nTCR/CD3 cycling.\na\nAg- or anti-CD3-induced TCR down-modulation was the same,\nwhether BFA was present or not, for both ZAP-WT and ZAP-\nY292F T cells (Table I). As a control for the activity of the BFA\ntreatment, we observed that no CD69 expression was induced\nwhen T cells were stimulated with Ag (results not shown), as pre-\ndicted by the inhibitory effect of BFA on protein secretion (49).\nFurthermore, when cycloheximide was added to prevent de novo\nreceptor synthesis during the incubation with or without BFA, no\ndifference appeared for the two types of T cells either.\nTogether, these results suggest that Ag- and anti-CD3-induced\nTCR down-modulation are not affected by prevention of TCR re-\ncycling by BFA.\nAg-induced TCR internalization is affected in ZAP-Y292F\ncompared with ZAP-WT CD8 T cells\nThe level of surface TCR expression is the result of an equilibrium\nbetween de novo synthesis, recycling, internalization, and degra-\ndation of its constituents. To establish which step was affected in\nthe Ag-induced TCR/CD3 down-modulation observed for ZAP-\nY292F T cells, we next determined the rate of receptor internal-\nization in mutant and WT T cells. For this, surface receptor has to\nbe labeled before induction of internalization. As described in Ma-\nterials and Methods, we prelabeled the surface TCR with anti-\nTCR (H57) mAb (A488) and visualized TCR internalization ei-\nther after acid treatment by FACS analysis or directly by confocal\nmicroscopy (see Fig. 4B). One can note that anti-TCR (H57)\nmAb binding was stable at 37°C (Fig. 2A). Furthermore, this treat-\nment did not per se induce TCR down-modulation or T cell acti-\nvation (measured by CD69 surface expression), and it did not af-\nfect Ag-induced TCR down-modulation (Fig. 2).\nThe ratio between the MFI on acid-treated (AT; measuring in-\nternalized Ab) and untreated cells (not treated (NT); measuring\ntotal mAb at the cell surface and inside) from the same sample\ncorresponds to the fraction of internalized TCR. After acidic treat-\nment, the total ﬂuorescence dropped to 10 and 9 when, respec-\ntively, ZAP-WT and ZAP-Y292F CD8 T cells were maintained at\n0°C for 90 min, and to 22 and 16 when they were incubated for the\nsame time at 37°C. This corresponds for both mouse strains to\n30% internalized TCR in medium at 37°C, compared with\n14% at 0°C (Fig. 2B). When cells were incubated with 107 M\ngp33 loaded APCs, acid-resistant ﬂuorescence increased to MFI of\n47 for the ZAP-WT and to only 33 for the ZAP-Y292F cells (Fig.\n2A), corresponding, respectively, to 70 and 45% internalized TCR\nafter 60-min stimulation (Fig. 2B). By contrast, when cells were\nstimulated with PMA, we observed a similar rate of TCR inter-\nnalization (70% internalized TCR after 60-min stimulation; Fig.\n2B) for both strains.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nAg (46), compared with spontaneous receptor cycling, which con-\ntrast with other data (45).\nAnalysis of the fate of CD3 during TCR internalization\nUsing anti-CD3 mAb H146.968, we developed a sensitive intra-\ncellular FACS analysis to measure CD3levels (see Materials and\nMethods). We previously compared this intracellular ﬂuorescence\nmeasurement with immunoblots on cell lysates using the same\nmAb to CD3 and obtained comparable results (M. Buferne and\nC. Boyer, unpublished results), indicating that a decrease in anti-\nCD3 MFI corresponds to the degradation of intracellular CD3.\nAll cultures were performed at 37°C in a medium containing cy-\ncloheximide to prevent protein synthesis.\nIn Fig. 3A, we observe similar levels of CD3 for P14 ZAP-WT\nor ZAP-Y292F CD8 T cells when cultured in medium (MFI, 433\nand 384, respectively). In the ZAP-WT T cells, a pronounced de-\ncrease in CD3 was observed in response to APC/gp33 (105 M)\n(MFI, 217), whereas its level remained high in control cultures\nwith APC/HY (MFI 362). This decrease corresponds to 50% CD3\ndegradation in 60% of the cells (Fig. 3, A and B). For P14 ZAP-\nY292F CD8 T cells, by contrast, we observed a moderate decrease\nin CD3 in response to APC/gp33 (105 M) (MFI, 260) compared\nwith its level in control cultures with APC/HY (MFI, 332). This\ncorresponds to 38% CD3 degradation in only 25% of the cells\n(Fig. 3, A and B). Note that CD3 degradation and TCR down-\nmodulation both increase in proportion to Ag dose (Fig. 3, com-\npare B and C), in agreement with recent work (46).\nContrasting results were obtained for T cells responding to anti-\nCD3 mAb or PMA. Stimulation with anti-CD3 led to very efﬁcient\nTCR down-modulation (80% after 4 h) and extensive CD3deg-\nradation (60 –70%) in CD8 T cells from both strains, whereas the\nPMA-induced TCR down-modulation (50%), led to only minor\nCD3 degradation (15%). The latter pattern was also similar for\nZAP-WT and ZAP-Y292F CD8 T cells (Fig. 3, B and C). This\nobservation is in agreement with published work (11), and with the\nnotion that PMA induces TCR sequestration, without driving the\nreceptor to the degradation pathway.\nThus, ligand-induced receptor degradation is dependent upon its\ninternalization, but all internalization, such as that induced by\nPMA, does not necessarily lead to degradation.\nTogether, these results suggest that ZAP-Y292F CD8 T cells are\nselectively affected at a step of Ag-induced receptor internalization\nand that, once internalized, the CD3 component is similarly de-\ngraded. However, these results do not exclude the possibility that\nthe ZAP-Y292F mutation may additionally affect the degradation\nof the CD3 complex.\nTo further address the stage at which the fate of the TCR/CD3\ncomplex is affected in the ZAP-Y292F CD8 T, we examined the\ndynamics of the receptor with respect to T cell/APC interaction\nand formation of the IS.\nAg-induced surface TCR redistribution at the T/APC contact is\ndefective and delayed for ZAP-Y292F CD8 T cells\nThe Journal of Immunology\nthe conjugated ZAP-Y292F T cells (85% of the analyzed con-\njugates; Table II), which is also shown in three-dimensional de-\nconvolution (Fig. 4A, third row, right). By contrast, F-actin was\nconcentrated at the T/APC contact zone of ZAP-Y292F T cells as\nfor ZAP-WT T cells. The poor CD3 redistribution in ZAP-Y292F\nT cell conjugates was partly compensated after 45 min, where we\nobserved that 40% of the T cells had a patched CD3 distribution\n(Table II). For both types of CD8 T cells, the TCR/CD3 was not\nredistributed if the irrelevant peptide HY was presented by the\nAPC (Fig. 4A, ﬁrst row). One can notice that, in these conjugates,\nTCR/CD3 and F-actin are homogenously distributed around the\ncell, and that F-actin is poorly labeled in unactivated T cells.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\ngliosides and raft labeling (50) and analyzed by confocal micros-\ncopy. As a control, T cells were also incubated alone, and we\nobserved a homogenous cell surface labeling for both TCR (red)\nand CTX (green), yielding a yellow-to-orange colocalization la-\nbeling (Fig. 4B, ﬁrst row, no APC, 45 min). For the P14 ZAP-WT\nCD8 T cells, after 15-min T/gp33 APC conjugate formation, we\nobserved a redistribution of the anti-TCR, indicating a localization\nof the TCR either at the plasma membrane in the T/APC interac-\ntion zone, or inside the cell where it is no longer colocalized with\nthe green CTX membrane staining (Fig. 4B, left panel, second\nrow). Furthermore, some (but not all) of the internalized TCR ap-\npeared in organelles budding from the T/APC synapse zone, as\nalready suggested by others (21). After 45 min of conjugate for-\nmation, most of the TCR was in organelles inside the cell (Fig. 4B,\nthird row) and did not colocalize with the surface-labeled CTX.\nThis pattern was observed for 90% of T/APC conjugates analyzed,\nboth after 15 and 45 min of T/APC contact. One can notice that\nCTX is rather poorly relocalized in the T/APC contact zone, a\npattern previously described for human CD8 T cells (51), in con-\ntrast to the CTX concentration in the T/APC zone described for\nmouse CD4 T cell lines (52). Additionally, the RMA-S cells used\nas APC were very poorly labeled with CTX, consequently CTX\ncould be used to mark CD8 T cell membranes.\nTCR redistribution was different for P14 ZAP-Y292F CD8 T\ncells, as shown in Fig. 4B (right panels). After 15-min T/gp33\nAPC conjugate formation, we observed very few TCR patches and\nno TCR redistribution to the T/APC contact zone. Both TCR and\nsurface CTX were still homogeneously distributed at the T cell\nsurface. However, after 45-min conjugation, we observed some\nTCR patches inside the T cell (see also Table II), as well as re-\nmaining TCR cell surface expression, suggesting that for the ZAP-\nY292F T cells, TCR redistribution and internalization was delayed\nand at a lower rate, in agreement with the previous results (Figs. 2\nand 4A). Together, these results indicate that TCR redistribution\nand internalization, in the T/APC zone are delayed and partially\ninhibited by the ZAP-Y292F mutation. By contrast, anti-CD3\nmAb-induced TCR patching and internalization was not affected\nby the ZAP-Y292F mutation when analyzed by confocal micros-\ncopy (results not shown).\nRecruitment of cbl at the T/APC contact zone is delayed and\ndecreased in ZAP-Y292F compared with ZAP-WT T cells\nBecause the TCR clustering at the T/APC conjugate contact zone\nwas delayed in ZAP-Y292F T cells, we were interested in analyz-\ning the localization of cbl, to establish whether the loss of the\ncbl-binding site on ZAP-Y292F would affect its redistribution. In-\ndeed, cbl was previously observed to be recruited to the T/APC\ncontact zone (53), and to be colocalized with the signaling phos-\nphotyrosine zone (54). Therefore, we compared the localization of\ncbl and F-actin in T/APC conjugates formed for different periods\nof time (15– 45 min). This study was performed by confocal mi-\ncroscopy on ﬁxed and permeabilized T/APC conjugates further\nlabeled for cbl (rabbit anti-Ring cbl/anti-rabbit-A546, red), and for\nF-actin (phalloidin-A488, green), as described in Materials and\nMethods (Fig. 5).\nrow, ZAP-WT and ZAP-Y292F). However, most of the ZAP-WT\nT/APC conjugates (65%; Table II) presented an additional local-\nization in the middle of the contact zone (Fig. 5A, second row,\nZAP-WT; see bold arrow). Such a localization was observed in\nonly 30% of the ZAP-Y292F T/APC conjugates (Table II). A his-\ntogram analysis of the ﬂuorescence distribution (Fig. 5C, Cbl 1)\nshowed that the cbl labeling was concentrated at the T/APC con-\ntact zone, bulging out of the F-actin zone within the T cell, before\nthe blue (Cy5) boundary of the APC membrane. The frequency of\nthis pattern increased for both strains at 45 min, reaching 80% for\nthe ZAP-WT and 50% for the ZAP-Y292F T/APC conjugates (Ta-\nble II). Despite variability between experiments, the difference be-\ntween the two strains was always observed (three independent\nexperiments).\nIn Fig. 5B, we analyzed the distribution of LFA-1 in T/APC\nconjugates. A high redistribution of LFA-1 at the T/gp33 APC\ncontact zone was observed for both ZAP-WT and ZAP-Y292F\nCD8 T cells (Fig. 5B, second row), which was not observed in the\nT/HY APC conjugates ( ﬁrst row). On the histograms representing\nthe ﬂuorescence distribution within T/APC conjugates, one can\nobserve that LFA-1 labeling is concentrated at the T/APC contact\nzone on the T cell side, proximal to the blue boundary of the APC\nmembrane. The LFA-1 redistribution remained rather homogenous\nat the T/APC contact zone even at 45 min, as described for other\nmarkers (PKC and Talin) in naive CD8 T cell/APC conjugates\n(55). This contrasts with the description of a p-SMAC distribution\nafter 30-min conjugate formation in other T cell lines including\nnaive CD4 T cells (21, 56).\nThese results showed that the redistribution of LFA-1 and\nF-actin at the T/APC conjugate contact zone was not affected by\nthe ZAP-Y292F mutation. In contrast, the recruitment of cbl to\nthe central region, which was readily observed in the ZAP-WT\nT/APC conjugates at an early time, was defective in the ZAP-\nY292F T cells. Whether this recruitment corresponds to an en-\ndosomal localization of cbl associated with the activated TCR\ncomplex needs further investigation. It is consistent with the\nfact that cbl and associated CMS/CIN85/CD2AP can be in-\nvolved in endosome organization via rab4 (57), and with the\ndescribed effect of the ZAP-Y292F mutation as preventing the\nZAP-cbl association (58). However, this effect was partially com-\npensated with time, which suggests that other cbl associations may\nsecondarily develop in the IS. These may involve vav linked to\nZAP Y315 (59), CD28 via PI3K (60), the src PTK (61), CD2AP\nbinding to CD2 (62), or connection of CD2AP/CIN85 proteins to\nthe actin cytoskeleton (63, 64).\nDiscussion\nIn the present work, we report that naive CD8 P14 T lymphocytes\nexpressing the mutated ZAP-Y292F were affected in Ag-induced\nTCR down-modulation, and presented a defect and delay in TCR\nand cbl recruitment at the T/APC conjugate interaction zone com-\npared with ZAP-WT T cells. However, LFA-1 and F-actin were\nclearly redistributed in the ZAP-Y292F T/APC contact zone as in\nZAP-WT T/APC conjugates.\nThe Journal of Immunology\nthat TCR internalization is affected by the ZAP-Y292F mutation,\nusing an assay independent of a mAb ligand. However, it has not\nbeen ruled out that the ZAP-Y292F mutation may additionally\naffect the degradation of some of the TCR/CD3 components. Con-\nstitutive endocytosis, which depends mainly on a CD3 di-leucine\nmotif (9), or on several motifs from all CD3 components (45), was\nnot altered by the ZAP-Y292F mutation. Anti-CD3- and PMA-\ninduced TCR internalization, which depend on CD3 or CD3\ncytoplasmic domains (1, 65), were not affected either. Therefore,\nthe observed defects, although partial, would appear to implicate\nthe ZAP-Y292/cbl interaction (58) at a step that selectively affects\nAPC-induced TCR internalization.\nBinding of the cbl adaptor to phosphorylated ZAP-292Y (22)\nhas a negative effect on the activity of ZAP-70, which was mainly\nattributed to the E3-Ubiquitin ligase activity of cbl and its Ring\ndomain (66). Consistent with this, cbl was found to be involved in\nubiquitination of ZAP-70 (67) and of CD3 (68). Cbl has also\nrecently been implicated at various steps of RTK surface receptor\ninternalization and degradation. First, cbl may participate in the\nformation of clathrin-coated vesicles by adapting endophilin via a\nprotein of the CIN85 family; additionally, the ubiquitin-ligase ac-\ntivity of cbl allows the ubiquitinated receptor to be recognized by\nubiquitin interacting motif domain bearing proteins like Eps15,\nwhich is a clathrin adaptor (26 –29, 31).\nIn T cells, the double genetic deletion of c-cbl/cbl-b was asso-\nciated with a partial deﬁciency in anti-CD3-induced TCR internal-\nization, whereas the effect on Ag stimulation was not reported in\nthat work (23). CD2AP, a member of the CIN85 family (62) ca-\npable of binding cbl (69), was recently reported to play a major\nrole in T cell signaling and TCR down-modulation (32). Indeed, T\ncells from CD2AP knockout mice failed to down-modulate their\nTCR in response to Ag stimulation, and showed defects in the ﬁne\nstructure of the c-SMAC and p-SMAC (32, 33). However, in con-\ntrast to the c-cbl/cbl-b double knockout, CD2AP knockout T cells\nwere not affected in mAb-induced TCR down-modulation, at least\nat early steps (32).\nThe fact that the defect in ZAP/cbl association selectively affects\nAPC-induced, and not constitutive or anti-CD3-induced TCR in-\nternalization, together with the observation of the delay in TCR\nand cbl recruitment to the T-APC interaction zone in ZAP-Y292F\nT cells strongly suggests that the defect in TCR internalization\nmay be the consequence of its delayed recruitment to the IS. The\nalternative, that the delay in TCR recruitment and c-SMAC for-\nmation would be the consequence of a defect in TCR internaliza-\ntion, appears difﬁcult to entertain. Indeed, in contrast to receptor\nengagement by mAb, stimulation of TCRs by peptide/MHC com-\nplexes on the APC is most efﬁcient when receptors are concen-\ntrated within the IS. Consistent with this, internalized TCR is\nmainly detected at the site of the IS (Ref. 21; Fig. 4B), whereas\nmAb-induced internalization is observed all around the plasma\nmembrane (results not shown). The molecular basis for the defect\nin c-SMAC formation in the absence of CD2AP is not fully elu-\ncidated (32). Similarly, it is not clear in the ZAP-Y292F T cells 1)\nwhether the delay in recruiting the TCR/CD3 at the IS is due to the\nabsence of CD2AP/cbl/ZAP binding, and 2) which molecular in-\nteraction of a putative CD2AP/cbl/ZAP complex is required to\nform the IS. Although such CD2AP/cbl/epidermal growth factor\nreceptor complexes have been described to connect to the actin\ncytoskeleton via the actin-related protein 2/3 complex (64), it is not\nclear whether such connections function in the TCR pathway.\nThey have been suggested for CD2 signaling (63).\nThrough 1) the ubiquitination pathway, cbl may dampen strong\nPTK signals, independently from the IS, and through 2) its asso-\nciation with CD2AP, it may contribute to the concentration of\nTCR/CD3/ZAP complexes within a c-SMAC, a step required for\nsustained signaling in the absence of apoptosis (32). The latter\ncontribution of the ZAP-70/cbl-b interaction to promote T cell ac-\ntivation may be balanced by the increased rate of TCR/CD3 com-\nplex internalization within the c-SMAC.\nThe unique properties of TCR/CD3 redistribution within the\nT/APC interface may explain why distinct motifs within the re-\nceptor complex appear to control its constitutive or mAb-induced\ninternalization (45) that have implicated di-leucine motifs (9), and\nAPC-induced internalization for which phospho-CD3-associated\nZAP-Y292 appears to be a rate-limiting motif. However, the fact\nthat the defect in the ZAP-Y292F T was partial, suggests that other\nmechanisms may lead to APC-induced TCR/CD3 internalization.\nThese may involve other cbl/CD2AP binding sites within the re-\nceptor complex, cbl being a scaffold protein that may bind other\nproximal ligands (25). It is also possible that another ubiquitin\nligase is able to bind to the activated TCR complex, akin to the\nNedd4 ubiquitinin ligase that was also localized in the raft fraction\nof IgE in activated RBL cells (71).\nConcerning the relative control of TCR/CD3 complex internal-\nization and degradation, we found that for Ag-, as for anti-CD3-\ninduced receptor down-modulation, CD3 degradation was pro-\nportional to receptor internalization and down-modulation (Fig. 3).\nThis result is in agreement with other work showing that during\nconstitutive, and mAb-induced TCR internalization, all TCR/CD3\ncomponents are degraded with a rate that is proportional to the rate\nof TCR down-modulation (46) through a mechanism that is de-\npendent on lysosomal activity (72). However, an exception exists\nfor PMA-induced TCR internalization that leads to poor CD3\ndegradation (Fig. 3).\nTogether with data from the literature, the present results are\ncompatible with a role for cbl in the Ag-induced TCR/CD3 inter-\nnalization/degradation following schemes also proposed for the\nepidermal growth factor receptor (29, 30, 33). Clearly, questions\nremain about the mechanisms by which the TCR/CD3 complex is\ndriven into the endocytic pathway at the IS, and in particular\nwhether, as for the BCR, internalization via clathrin-coated vesi-\ncles is dependent on integral raft components (73, 74).",
         "",
         "",
         "",
         "Cbl proteins have been implicated in ligand-induced TCR/CD3 down-modulation, but underlying mechanisms are unclear. We\nanalyzed the effect of mutation of a cbl-binding site on ZAP-70 (ZAP-Y292F) on dynamics, internalization, and degradation of the\nTCR/CD3 complex in response to distinct stimuli. Naive CD8 T cells expressing the P14 transgenic TCR from ZAP-Y292F mice\nwere selectively affected in TCR/CD3 down-modulation in response to antigenic stimulation, whereas neither anti-CD3 Ab-, and\nPMA-induced TCR down-modulation, nor constitutive receptor endocytosis/cycling were impaired. We further established that\nthe defect in TCR/CD3 down-modulation in response to Ag was paralleled by an impaired TCR/CD3 internalization and CD3\ndegradation. Analysis of T/APC conjugates revealed that delayed redistribution of TCR at the T/APC contact zone was paralleled\nby a delay in TCR internalization in the synaptic zone in ZAP-Y292F compared with ZAP-wild-type T cells. Cbl recruitment to\nthe synapse was also retarded in ZAP-Y292F T cells, although F-actin and LFA-1 redistribution was similar for both cell types.\nThis study identiﬁes a step involving ZAP-70/cbl interaction that is critical for rapid internalization of the TCR/CD3 complex at\nthe CD8 T cell/APC synapse. The Journal of Immunology, 2005, 175: 3140 –3149.\nface when associated with the CD3 complex composed of\nthe CD3/CD3, CD3/CD3, and CD3- dimers. Its\nlevel of surface expression is ﬁnely tuned by mechanisms control-\nling its assembly, membrane dynamics, and degradation, which\nvary along T cell development and as a consequence of T cell\nactivation. In particular, T cell stimulation by APC presenting the\ncognate peptide/MHC complex leads to a rapid decrease in the\nlevel of surface-expressed TCR. This phenomenon, known as\nTCR/CD3 down-modulation, occurs mainly by internalization of\nthe whole receptor complex through endosomal pathways (1, 2),\nalthough mechanisms invoking exocytosis of vesicles containing\nthe receptor have also been described (3).\nAg-induced TCR down-modulation is intimately linked to TCR\nsignaling and, in particular, to protein tyrosine kinase (PTK)4 ac-\ntivation, including Src family PTKs p56lck and p59fyn. These ki-\n4 Abbreviations used in this paper: PTK, protein tyrosine kinase; PKC, protein kinase\nC; IS, immune synapse; SMAC, supramolecular activation cluster; c-SMAC, central\nSMAC; p-SMAC, peripheral SMAC; CIN85, Cbl-interacting protein of 85 kDa; RTK,\nreceptor tyrosine kinase; CD2AP, CD2-associated protein; LN, lymph node; CTX,\ncholera toxin; MFI, mean ﬂuorescence intensity; PAF, paraformaldehyde; RT, room\ntemperature; BFA, brefeldin A; WT, wild type; AT,\nacid treated; NT,\nnot treated.\nnases phosphorylate tyrosines in the ITAM motifs of CD3 cyto-\nplasmic domains, allowing for binding of the ZAP-70 PTK via its\nSH2 domains. Subsequent phosphorylation of adaptor molecules\nand protein-protein interactions further transduce positive and neg-\native signals for functional activation of the T cells (4). In this\ncontext, TCR/CD3 down-modulation has been suggested to be one\nof the mechanisms that contribute to the arrest of signaling initi-\nated by Ag. Alternatively, it could be reasoned that the failure to\ndisengage peptide/MHC complexes by receptor clearance would\nprevent the serial engagement of new receptors that is required for\noptimal T cell stimulation by the limiting amount of peptide/MHC\ncomplexes available on the APCs (4, 5).\nApproaches at unraveling the complexities of TCR dynamics\nand signaling have included the analysis of T cells expressing\nTCR/CD3 complexes modiﬁed at potential internalization motifs,\nor in components of the receptor endocytosis/degradation pathway.\nIn particular, each CD3 cytoplasmic domain contains tyrosine-\nbased motifs in their ITAMs that are able to bind to the \n2 chain\nof the clathrin adaptor AP2 (6). Furthermore, the CD3 and CD3\nchains each present a di-leucine motif (7) capable of binding the \n\nsubunit of the AP2 complex (8). The CD3 chain di-leucine motif\nappears necessary for TCR/CD3 constitutive cycling (9), whereas\nthe proximal serine-126 on CD3is additionally required for phor-\nbol ester-induced TCR/CD3 internalization (10), a protein kinase\nC (PKC)-dependent event\ninvolves receptor sequestration\nwithout subsequent degradation (11). For mAb-induced TCR/CD3\ndown-regulation, a process dependent upon tyrosine kinase activ-\nity (12–14), but not PKC, absence of both CD3 and CD3 cyto-\nplasmic domains led to deﬁcient receptor down-modulation, sug-\ngesting that upon T cell activation, unveiling of the di-leucine and\ntyrosine-based motifs present in each of these chains may be in-\nvolved (1). However, Ag-induced TCR down-modulation was not\naffected by the combined truncations of CD3 and CD3 cyto-\nplasmic domains (15), suggesting that a distinct mechanism in-\nvolving ZAP-70 activity (14), was rate limiting in this process.\nthat\nThe Journal of Immunology\nThe particularity of Ag-induced TCR down-modulation is that it\noccurs upon engagement of the TCR by peptide/MHC complexes\nwithin the immune synapse (IS) formed at the T cell/APC contact\nzone. This specialized structure, associated with a redistribution of\nmembrane receptors, develops over a 5–30 min period. It subdi-\nvides the T cell/APC interface into supramolecular activation clus-\nters (SMACs) that are spatially organized into a central (c-SMAC)\nregion where TCR-MHC/peptide and CD28-CD80 interactions\ntake place, and a peripheral (p-SMAC) area enriched in cytoskel-\netal components and adhesion molecules such as LFA-1 (16, 17),\nand CD2 (18). However, the rapid induction of PTK activation\nupon Ag-induced T cell stimulation precedes the redistribution of\nthe TCR/CD3 complex in the c-SMAC (19 –21). The observation\nthat surface TCR disappeared from the c-SMAC may signify its\nlocalized internalization (21).\nTyrosine-292 in ZAP-70, when phosphorylated upon TCR ac-\ntivation, is a binding site for c-cbl and cbl-b via the cbl tyrosine\nkinase binding domain (22–24). Contrasting results were reported\nfor thymocytes of mice expressing an altered c-cbl mutated in its\ntyrosine kinase binding domain, because this led to constitutive\nactivation of rac without affecting ZAP-70 phosphorylation (25).\nHowever, the effect of this mutation could not be studied in pe-\nripheral T cells, where mostly cbl-b is expressed.\nAdaptor proteins of the Cbl-interacting protein of 85 kDa\n(CIN85) family have recently been shown to bind both cbl and\nendophilins, and thereby to regulate the endocytosis of activated\nreceptor tyrosine kinases (RTK) (26 –29). Cbl is also involved in\nRTK monoubiquitination allowing for receptor binding to the\nclathrin adaptor Eps15 (28, 30, 31). In T cells, the CIN85 family\nmember CD2-associated protein (CD2AP) has recently been im-\nplicated in the pathway of TCR/CD3 down-modulation (32, 33).\nFurthermore, we have recently described that T cells from mice\nexpressing ZAP-Y292F presented hyper-tyrosine-phosphorylation\nin response to TCR/CD3 engagement but were partially inhibited\nin Ag-induced TCR down-modulation (34).\nIn the present work, we analyzed the mechanisms by which\nTCR down-modulation is impaired in CD8 T cells expressing the\nZAP-Y292F mutation. Our results showed that the defect in TCR\ndown-modulation was speciﬁc for Ag-induced TCR internalization\nand was associated with a delayed recruitment of TCR and cbl at\nthe T/APC synapse zone.\nMaterials and Methods\nMice\nAg-presenting cells\nTap-2-deﬁcient RMA-S H-2b cells (35) were generally used as APCs, and\nsimilar results were obtained when LPS blasts from Tap-1°/° mice were\nused (results not shown). Before loading with peptide (LCMV gp33 (36) or\ncontrol peptide HY (37)), Tap-negative cells were incubated overnight at\n31°C in RPMI 1640 with 5% dialyzed FCS.\nT cell preparation\nAbs and reagents for immunoﬂuorescence\nvisualized with the cholera toxin -chain (CTX) coupled to FITC (Sigma-\nAldrich), or Alexa488 (Molecular Probes). F-actin was visualized using\nphalloidin Alexa488 (Molecular Probes). Rabbit anti-cbl Ring domain was\nprepared as described (26), and revealed with anti-rabbit Ig Zenon A546\n(Molecular Probes). The speciﬁcity of the Ab for cbl was further estab-\nlished in T cell lysates, because only the cbl band (at 120 kDa) was de-\ntected by immunoblot (results not shown).\nFluorescence analysis\nMeasurement of surface ﬂuorescence and FACS analysis were as described\nusing mean ﬂuorescence intensity (MFI) values (15). TCR down-modula-\ntion was calculated as (1  MFI experimental/MFI medium)  100. For\nintracellular CD3 staining, cells were washed (4 min; 4°C) in PBS before\n5-min ﬁxation with 2% paraformaldehyde (PAF) at room temperature\n(RT), washed again with PBS, and permeabilized for 10 min at RT with\n0.5% saponin in PBS, 2% FCS, and 0.02% NaN3 containing 1 mM van-\nadate (saponin buffer). Biotinylated mAb H146.968 was diluted in the same\nsaponin buffer and added for 30 min at 4°C. After permeabilization, cells\nwere washed and centrifuged at 2000 rpm for 5 min at 4°C. For calibration,\nwe set up optimal mAb dilutions to obtain a linear decrease in labeling in\nthe presence of increasing amounts of the CD3 peptide recognized by the\nmAb. CD3 degradation was calculated as follows: (1  MFI experimen-\ntal/MFI medium)  100.\nMeasurement of internalized TCR\nOur approach involved ﬁrst biotinylation of cell surface proteins with SS-\nbiotin to discriminate between surface and internalized TCR (45, 46).\nthis treatment prevented subsequent Ag-induced P14-TCR\nHowever,\ndown-modulation and T cell activation as measured by CD69 expression\n(results not shown). We then chose to prelabel T cells with anti-TCR mAb\nat 4°C, before Ag stimulation at 37°C, followed by acid treatment to re-\nlease mAb remaining at the cell surface (as described in Ref. 2). The\nanti-TCRV2 B20.1 mAb was unsuitable for this approach because it rap-\nidly dissociates from the TCR. Therefore anti-TCR (H57), which stably\nbinds the TCR, was used for this study, as follows. T cells were incubated\nat 4°C with anti-TCR(H57) mAb A488 (Molecular Probes kit) before Ag\nstimulation at 37°C. At different times, samples were divided, and one-half\nwas directly used to measure total ﬂuorescence, the other half was treated\nwith an acid solution (0.15 M acetic acid, 0.15 M NaCl) for 5 min on ice\nbefore two washes in medium and measurement of remaining ﬂuorescence\ncorresponding to internalized mAb (2). Prelabeling of T cells with anti-\nTCR (H57) did not per se activate the P14 T cells and did not prevent\nsubsequent Ag-induced TCR down-modulation and signaling for CD69\nexpression (see Fig. 2).\nConfocal analysis\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nResults\nAg-induced TCR down-modulation is defective in ZAP-Y292F\ncompared with ZAP-wild-type (WT) CD8 T cells\nOur previous results indicated that TCR down-modulation in re-\nsponse to antigenic stimulation was partially inhibited in CD8 T\ncells from mice expressing ZAP-Y292F molecules (34). We ﬁrst\nconﬁrmed this ﬁnding in a kinetic experiment shown in Fig. 1. As\nseen in Fig. 1A, exposure of ZAP-WT CD8 T cells to APC loaded\nwith the gp33 LCMV peptide at 107 M decreased the level of\nsurface TCR from 830 to 400 MFI units after 60 min, and to MFI\n113 after 270 min. A similar treatment of the ZAP-Y292F CD8 T\ncells decreased the level of surface TCR from 870 to 590 after 60\nmin, and to 330 after 270 min. The rate of TCR down-modulation\nin response to gp33 107 M/APC by the ZAP-Y292F CD8 T cells\nwas about half (20%/h) that observed for the ZAP-WT CD8 T cells\n(38%/h) (Fig. 1B). The large SDs of the 60-min time points cor-\nrespond to the variability of TCR down-modulation observed from\none experiment to the other at that early time, but the respective\nrelative values for mutant to WT T cells were very reproducible in\nthe different experiments (decreased rate of TCR down-modulation\n52  11%). The loading of the RMA-S used as APC was per-\nformed with gp33 concentrations between 107 and 109 M, and\nwe observed a linear dose effect on TCR down-modulation and\nactivation, with a differential rate of TCR down-modulation be-\ntween ZAP-WT and ZAP-Y292F T cells maintained at all con-\ncentrations tested (results not shown).\nWe next analyzed the effect of this mutation on other means of\ninducing TCR down-modulation, such as anti-CD3 mAb or PMA\nstimulations (15, 47). The rate of TCR down-modulation induced\nby anti-CD3 mAb was the same for the ZAP-WT and ZAP-Y292F\nT cells, i.e., 55%/h (Fig. 1B). Receptor down-modulation was\nalso similar for the two types of T cells at any dose of anti-CD3\nmAb used, whether soluble (Fig. 1C) or plastic-coated (results not\nshown). When cells were stimulated by PMA, we also observed a\nsimilar rate of TCR down-modulation for both strains (results not\nshown) (see Fig. 2B).\nAll experiments were controlled for T cell activation by mea-\nsuring the induction of CD69, and we conﬁrmed that there was no\ndifference between the two strains in that assay (34).\nIn conclusion, the ZAP-Y292F mutation caused a reduced rate\n(40 –50% decrease) of TCR down-modulation in response to an-\ntigenic stimulation compared with that observed in ZAP-WT T\ncells within a range of antigenic peptide from 107 to 109 M.\nFurthermore, this study showed that the ZAP-Y292F mutation se-\nlectively affects Ag-induced, and not anti-CD3 mAb- or PMA-\ninduced receptor down-modulation.\nUse of brefeldin A (BFA) revealed that the ZAP-Y292F mutation\ndoes not affect spontaneous TCR cycling and shows no recycling\nof anti-CD3 mAb- or Ag-induced TCR down-modulation in both\ntypes of CD8 T cells\nConstitutive TCR endocytosis and cycling can be demonstrated by\nthe effect on receptor surface expression of BFA, which inhibits\nexport from the endoplasmic reticulum/Golgi and endosome recy-\ncling (45). We have examined the effect of BFA on constitutive\nTCR down-modulation, as well as on Ag- or anti-CD3-induced\nTCR down-modulation of ZAP-WT and ZAP-Y292F CD8 T cells.\nResults shown in Table I are expressed as the ratio of TCR down-\na BFA was added or not to T cells for 1 h before and during the down-modulation\nexperiment, in medium (constitutive cycling), with APC (Ag-induced down-modu-\nlation), or with anti-CD3 mAb (CD3-induced down-modulation). Results are pre-\nsented for the 120-min time point, and are expressed as the ratio between TCR down-\nmodulation in the absence and presence of BFA.\nThe Journal of Immunology\nmodulation between untreated and BFA-treated T cells. As previ-\nously described for naive T cells (48), P14 CD8 T cells do not\nthrive when cultured unstimulated at 37°C. They showed a gradual\ndecrease of TCR expression (5%/h), which was slower than the\nrate previously described for naive CD4 T cells (45). The presence\nof BFA increased receptor down-modulation (15–20%/h), so that,\nafter 2 h, the ratio of TCR down-modulation between untreated\nand BFA-treated T cells was of 0.34 for ZAP-WT and 0.38 for\nZAP-Y292F T cells, respectively (Table I), indicating that both\ntypes of T cells have\nconstitutive\nTCR/CD3 cycling.\na\nAg- or anti-CD3-induced TCR down-modulation was the same,\nwhether BFA was present or not, for both ZAP-WT and ZAP-\nY292F T cells (Table I). As a control for the activity of the BFA\ntreatment, we observed that no CD69 expression was induced\nwhen T cells were stimulated with Ag (results not shown), as pre-\ndicted by the inhibitory effect of BFA on protein secretion (49).\nFurthermore, when cycloheximide was added to prevent de novo\nreceptor synthesis during the incubation with or without BFA, no\ndifference appeared for the two types of T cells either.\nTogether, these results suggest that Ag- and anti-CD3-induced\nTCR down-modulation are not affected by prevention of TCR re-\ncycling by BFA.\nAg-induced TCR internalization is affected in ZAP-Y292F\ncompared with ZAP-WT CD8 T cells\nThe level of surface TCR expression is the result of an equilibrium\nbetween de novo synthesis, recycling, internalization, and degra-\ndation of its constituents. To establish which step was affected in\nthe Ag-induced TCR/CD3 down-modulation observed for ZAP-\nY292F T cells, we next determined the rate of receptor internal-\nization in mutant and WT T cells. For this, surface receptor has to\nbe labeled before induction of internalization. As described in Ma-\nterials and Methods, we prelabeled the surface TCR with anti-\nTCR (H57) mAb (A488) and visualized TCR internalization ei-\nther after acid treatment by FACS analysis or directly by confocal\nmicroscopy (see Fig. 4B). One can note that anti-TCR (H57)\nmAb binding was stable at 37°C (Fig. 2A). Furthermore, this treat-\nment did not per se induce TCR down-modulation or T cell acti-\nvation (measured by CD69 surface expression), and it did not af-\nfect Ag-induced TCR down-modulation (Fig. 2).\nThe ratio between the MFI on acid-treated (AT; measuring in-\nternalized Ab) and untreated cells (not treated (NT); measuring\ntotal mAb at the cell surface and inside) from the same sample\ncorresponds to the fraction of internalized TCR. After acidic treat-\nment, the total ﬂuorescence dropped to 10 and 9 when, respec-\ntively, ZAP-WT and ZAP-Y292F CD8 T cells were maintained at\n0°C for 90 min, and to 22 and 16 when they were incubated for the\nsame time at 37°C. This corresponds for both mouse strains to\n30% internalized TCR in medium at 37°C, compared with\n14% at 0°C (Fig. 2B). When cells were incubated with 107 M\ngp33 loaded APCs, acid-resistant ﬂuorescence increased to MFI of\n47 for the ZAP-WT and to only 33 for the ZAP-Y292F cells (Fig.\n2A), corresponding, respectively, to 70 and 45% internalized TCR\nafter 60-min stimulation (Fig. 2B). By contrast, when cells were\nstimulated with PMA, we observed a similar rate of TCR inter-\nnalization (70% internalized TCR after 60-min stimulation; Fig.\n2B) for both strains.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\nAg (46), compared with spontaneous receptor cycling, which con-\ntrast with other data (45).\nAnalysis of the fate of CD3 during TCR internalization\nUsing anti-CD3 mAb H146.968, we developed a sensitive intra-\ncellular FACS analysis to measure CD3levels (see Materials and\nMethods). We previously compared this intracellular ﬂuorescence\nmeasurement with immunoblots on cell lysates using the same\nmAb to CD3 and obtained comparable results (M. Buferne and\nC. Boyer, unpublished results), indicating that a decrease in anti-\nCD3 MFI corresponds to the degradation of intracellular CD3.\nAll cultures were performed at 37°C in a medium containing cy-\ncloheximide to prevent protein synthesis.\nIn Fig. 3A, we observe similar levels of CD3 for P14 ZAP-WT\nor ZAP-Y292F CD8 T cells when cultured in medium (MFI, 433\nand 384, respectively). In the ZAP-WT T cells, a pronounced de-\ncrease in CD3 was observed in response to APC/gp33 (105 M)\n(MFI, 217), whereas its level remained high in control cultures\nwith APC/HY (MFI 362). This decrease corresponds to 50% CD3\ndegradation in 60% of the cells (Fig. 3, A and B). For P14 ZAP-\nY292F CD8 T cells, by contrast, we observed a moderate decrease\nin CD3 in response to APC/gp33 (105 M) (MFI, 260) compared\nwith its level in control cultures with APC/HY (MFI, 332). This\ncorresponds to 38% CD3 degradation in only 25% of the cells\n(Fig. 3, A and B). Note that CD3 degradation and TCR down-\nmodulation both increase in proportion to Ag dose (Fig. 3, com-\npare B and C), in agreement with recent work (46).\nContrasting results were obtained for T cells responding to anti-\nCD3 mAb or PMA. Stimulation with anti-CD3 led to very efﬁcient\nTCR down-modulation (80% after 4 h) and extensive CD3deg-\nradation (60 –70%) in CD8 T cells from both strains, whereas the\nPMA-induced TCR down-modulation (50%), led to only minor\nCD3 degradation (15%). The latter pattern was also similar for\nZAP-WT and ZAP-Y292F CD8 T cells (Fig. 3, B and C). This\nobservation is in agreement with published work (11), and with the\nnotion that PMA induces TCR sequestration, without driving the\nreceptor to the degradation pathway.\nThus, ligand-induced receptor degradation is dependent upon its\ninternalization, but all internalization, such as that induced by\nPMA, does not necessarily lead to degradation.\nTogether, these results suggest that ZAP-Y292F CD8 T cells are\nselectively affected at a step of Ag-induced receptor internalization\nand that, once internalized, the CD3 component is similarly de-\ngraded. However, these results do not exclude the possibility that\nthe ZAP-Y292F mutation may additionally affect the degradation\nof the CD3 complex.\nTo further address the stage at which the fate of the TCR/CD3\ncomplex is affected in the ZAP-Y292F CD8 T, we examined the\ndynamics of the receptor with respect to T cell/APC interaction\nand formation of the IS.\nAg-induced surface TCR redistribution at the T/APC contact is\ndefective and delayed for ZAP-Y292F CD8 T cells\nThe Journal of Immunology\nthe conjugated ZAP-Y292F T cells (85% of the analyzed con-\njugates; Table II), which is also shown in three-dimensional de-\nconvolution (Fig. 4A, third row, right). By contrast, F-actin was\nconcentrated at the T/APC contact zone of ZAP-Y292F T cells as\nfor ZAP-WT T cells. The poor CD3 redistribution in ZAP-Y292F\nT cell conjugates was partly compensated after 45 min, where we\nobserved that 40% of the T cells had a patched CD3 distribution\n(Table II). For both types of CD8 T cells, the TCR/CD3 was not\nredistributed if the irrelevant peptide HY was presented by the\nAPC (Fig. 4A, ﬁrst row). One can notice that, in these conjugates,\nTCR/CD3 and F-actin are homogenously distributed around the\ncell, and that F-actin is poorly labeled in unactivated T cells.\nROLE FOR ZAP/cbl INTERACTION IN Ag-INDUCED TCR INTERNALIZATION\ngliosides and raft labeling (50) and analyzed by confocal micros-\ncopy. As a control, T cells were also incubated alone, and we\nobserved a homogenous cell surface labeling for both TCR (red)\nand CTX (green), yielding a yellow-to-orange colocalization la-\nbeling (Fig. 4B, ﬁrst row, no APC, 45 min). For the P14 ZAP-WT\nCD8 T cells, after 15-min T/gp33 APC conjugate formation, we\nobserved a redistribution of the anti-TCR, indicating a localization\nof the TCR either at the plasma membrane in the T/APC interac-\ntion zone, or inside the cell where it is no longer colocalized with\nthe green CTX membrane staining (Fig. 4B, left panel, second\nrow). Furthermore, some (but not all) of the internalized TCR ap-\npeared in organelles budding from the T/APC synapse zone, as\nalready suggested by others (21). After 45 min of conjugate for-\nmation, most of the TCR was in organelles inside the cell (Fig. 4B,\nthird row) and did not colocalize with the surface-labeled CTX.\nThis pattern was observed for 90% of T/APC conjugates analyzed,\nboth after 15 and 45 min of T/APC contact. One can notice that\nCTX is rather poorly relocalized in the T/APC contact zone, a\npattern previously described for human CD8 T cells (51), in con-\ntrast to the CTX concentration in the T/APC zone described for\nmouse CD4 T cell lines (52). Additionally, the RMA-S cells used\nas APC were very poorly labeled with CTX, consequently CTX\ncould be used to mark CD8 T cell membranes.\nTCR redistribution was different for P14 ZAP-Y292F CD8 T\ncells, as shown in Fig. 4B (right panels). After 15-min T/gp33\nAPC conjugate formation, we observed very few TCR patches and\nno TCR redistribution to the T/APC contact zone. Both TCR and\nsurface CTX were still homogeneously distributed at the T cell\nsurface. However, after 45-min conjugation, we observed some\nTCR patches inside the T cell (see also Table II), as well as re-\nmaining TCR cell surface expression, suggesting that for the ZAP-\nY292F T cells, TCR redistribution and internalization was delayed\nand at a lower rate, in agreement with the previous results (Figs. 2\nand 4A). Together, these results indicate that TCR redistribution\nand internalization, in the T/APC zone are delayed and partially\ninhibited by the ZAP-Y292F mutation. By contrast, anti-CD3\nmAb-induced TCR patching and internalization was not affected\nby the ZAP-Y292F mutation when analyzed by confocal micros-\ncopy (results not shown).\nRecruitment of cbl at the T/APC contact zone is delayed and\ndecreased in ZAP-Y292F compared with ZAP-WT T cells\nBecause the TCR clustering at the T/APC conjugate contact zone\nwas delayed in ZAP-Y292F T cells, we were interested in analyz-\ning the localization of cbl, to establish whether the loss of the\ncbl-binding site on ZAP-Y292F would affect its redistribution. In-\ndeed, cbl was previously observed to be recruited to the T/APC\ncontact zone (53), and to be colocalized with the signaling phos-\nphotyrosine zone (54). Therefore, we compared the localization of\ncbl and F-actin in T/APC conjugates formed for different periods\nof time (15– 45 min). This study was performed by confocal mi-\ncroscopy on ﬁxed and permeabilized T/APC conjugates further\nlabeled for cbl (rabbit anti-Ring cbl/anti-rabbit-A546, red), and for\nF-actin (phalloidin-A488, green), as described in Materials and\nMethods (Fig. 5).\nrow, ZAP-WT and ZAP-Y292F). However, most of the ZAP-WT\nT/APC conjugates (65%; Table II) presented an additional local-\nization in the middle of the contact zone (Fig. 5A, second row,\nZAP-WT; see bold arrow). Such a localization was observed in\nonly 30% of the ZAP-Y292F T/APC conjugates (Table II). A his-\ntogram analysis of the ﬂuorescence distribution (Fig. 5C, Cbl 1)\nshowed that the cbl labeling was concentrated at the T/APC con-\ntact zone, bulging out of the F-actin zone within the T cell, before\nthe blue (Cy5) boundary of the APC membrane. The frequency of\nthis pattern increased for both strains at 45 min, reaching 80% for\nthe ZAP-WT and 50% for the ZAP-Y292F T/APC conjugates (Ta-\nble II). Despite variability between experiments, the difference be-\ntween the two strains was always observed (three independent\nexperiments).\nIn Fig. 5B, we analyzed the distribution of LFA-1 in T/APC\nconjugates. A high redistribution of LFA-1 at the T/gp33 APC\ncontact zone was observed for both ZAP-WT and ZAP-Y292F\nCD8 T cells (Fig. 5B, second row), which was not observed in the\nT/HY APC conjugates ( ﬁrst row). On the histograms representing\nthe ﬂuorescence distribution within T/APC conjugates, one can\nobserve that LFA-1 labeling is concentrated at the T/APC contact\nzone on the T cell side, proximal to the blue boundary of the APC\nmembrane. The LFA-1 redistribution remained rather homogenous\nat the T/APC contact zone even at 45 min, as described for other\nmarkers (PKC and Talin) in naive CD8 T cell/APC conjugates\n(55). This contrasts with the description of a p-SMAC distribution\nafter 30-min conjugate formation in other T cell lines including\nnaive CD4 T cells (21, 56).\nThese results showed that the redistribution of LFA-1 and\nF-actin at the T/APC conjugate contact zone was not affected by\nthe ZAP-Y292F mutation. In contrast, the recruitment of cbl to\nthe central region, which was readily observed in the ZAP-WT\nT/APC conjugates at an early time, was defective in the ZAP-\nY292F T cells. Whether this recruitment corresponds to an en-\ndosomal localization of cbl associated with the activated TCR\ncomplex needs further investigation. It is consistent with the\nfact that cbl and associated CMS/CIN85/CD2AP can be in-\nvolved in endosome organization via rab4 (57), and with the\ndescribed effect of the ZAP-Y292F mutation as preventing the\nZAP-cbl association (58). However, this effect was partially com-\npensated with time, which suggests that other cbl associations may\nsecondarily develop in the IS. These may involve vav linked to\nZAP Y315 (59), CD28 via PI3K (60), the src PTK (61), CD2AP\nbinding to CD2 (62), or connection of CD2AP/CIN85 proteins to\nthe actin cytoskeleton (63, 64).\nDiscussion\nIn the present work, we report that naive CD8 P14 T lymphocytes\nexpressing the mutated ZAP-Y292F were affected in Ag-induced\nTCR down-modulation, and presented a defect and delay in TCR\nand cbl recruitment at the T/APC conjugate interaction zone com-\npared with ZAP-WT T cells. However, LFA-1 and F-actin were\nclearly redistributed in the ZAP-Y292F T/APC contact zone as in\nZAP-WT T/APC conjugates.\nThe Journal of Immunology\nthat TCR internalization is affected by the ZAP-Y292F mutation,\nusing an assay independent of a mAb ligand. However, it has not\nbeen ruled out that the ZAP-Y292F mutation may additionally\naffect the degradation of some of the TCR/CD3 components. Con-\nstitutive endocytosis, which depends mainly on a CD3 di-leucine\nmotif (9), or on several motifs from all CD3 components (45), was\nnot altered by the ZAP-Y292F mutation. Anti-CD3- and PMA-\ninduced TCR internalization, which depend on CD3 or CD3\ncytoplasmic domains (1, 65), were not affected either. Therefore,\nthe observed defects, although partial, would appear to implicate\nthe ZAP-Y292/cbl interaction (58) at a step that selectively affects\nAPC-induced TCR internalization.\nBinding of the cbl adaptor to phosphorylated ZAP-292Y (22)\nhas a negative effect on the activity of ZAP-70, which was mainly\nattributed to the E3-Ubiquitin ligase activity of cbl and its Ring\ndomain (66). Consistent with this, cbl was found to be involved in\nubiquitination of ZAP-70 (67) and of CD3 (68). Cbl has also\nrecently been implicated at various steps of RTK surface receptor\ninternalization and degradation. First, cbl may participate in the\nformation of clathrin-coated vesicles by adapting endophilin via a\nprotein of the CIN85 family; additionally, the ubiquitin-ligase ac-\ntivity of cbl allows the ubiquitinated receptor to be recognized by\nubiquitin interacting motif domain bearing proteins like Eps15,\nwhich is a clathrin adaptor (26 –29, 31).\nIn T cells, the double genetic deletion of c-cbl/cbl-b was asso-\nciated with a partial deﬁciency in anti-CD3-induced TCR internal-\nization, whereas the effect on Ag stimulation was not reported in\nthat work (23). CD2AP, a member of the CIN85 family (62) ca-\npable of binding cbl (69), was recently reported to play a major\nrole in T cell signaling and TCR down-modulation (32). Indeed, T\ncells from CD2AP knockout mice failed to down-modulate their\nTCR in response to Ag stimulation, and showed defects in the ﬁne\nstructure of the c-SMAC and p-SMAC (32, 33). However, in con-\ntrast to the c-cbl/cbl-b double knockout, CD2AP knockout T cells\nwere not affected in mAb-induced TCR down-modulation, at least\nat early steps (32).\nThe fact that the defect in ZAP/cbl association selectively affects\nAPC-induced, and not constitutive or anti-CD3-induced TCR in-\nternalization, together with the observation of the delay in TCR\nand cbl recruitment to the T-APC interaction zone in ZAP-Y292F\nT cells strongly suggests that the defect in TCR internalization\nmay be the consequence of its delayed recruitment to the IS. The\nalternative, that the delay in TCR recruitment and c-SMAC for-\nmation would be the consequence of a defect in TCR internaliza-\ntion, appears difﬁcult to entertain. Indeed, in contrast to receptor\nengagement by mAb, stimulation of TCRs by peptide/MHC com-\nplexes on the APC is most efﬁcient when receptors are concen-\ntrated within the IS. Consistent with this, internalized TCR is\nmainly detected at the site of the IS (Ref. 21; Fig. 4B), whereas\nmAb-induced internalization is observed all around the plasma\nmembrane (results not shown). The molecular basis for the defect\nin c-SMAC formation in the absence of CD2AP is not fully elu-\ncidated (32). Similarly, it is not clear in the ZAP-Y292F T cells 1)\nwhether the delay in recruiting the TCR/CD3 at the IS is due to the\nabsence of CD2AP/cbl/ZAP binding, and 2) which molecular in-\nteraction of a putative CD2AP/cbl/ZAP complex is required to\nform the IS. Although such CD2AP/cbl/epidermal growth factor\nreceptor complexes have been described to connect to the actin\ncytoskeleton via the actin-related protein 2/3 complex (64), it is not\nclear whether such connections function in the TCR pathway.\nThey have been suggested for CD2 signaling (63).\nThrough 1) the ubiquitination pathway, cbl may dampen strong\nPTK signals, independently from the IS, and through 2) its asso-\nciation with CD2AP, it may contribute to the concentration of\nTCR/CD3/ZAP complexes within a c-SMAC, a step required for\nsustained signaling in the absence of apoptosis (32). The latter\ncontribution of the ZAP-70/cbl-b interaction to promote T cell ac-\ntivation may be balanced by the increased rate of TCR/CD3 com-\nplex internalization within the c-SMAC.\nThe unique properties of TCR/CD3 redistribution within the\nT/APC interface may explain why distinct motifs within the re-\nceptor complex appear to control its constitutive or mAb-induced\ninternalization (45) that have implicated di-leucine motifs (9), and\nAPC-induced internalization for which phospho-CD3-associated\nZAP-Y292 appears to be a rate-limiting motif. However, the fact\nthat the defect in the ZAP-Y292F T was partial, suggests that other\nmechanisms may lead to APC-induced TCR/CD3 internalization.\nThese may involve other cbl/CD2AP binding sites within the re-\nceptor complex, cbl being a scaffold protein that may bind other\nproximal ligands (25). It is also possible that another ubiquitin\nligase is able to bind to the activated TCR complex, akin to the\nNedd4 ubiquitinin ligase that was also localized in the raft fraction\nof IgE in activated RBL cells (71).\nConcerning the relative control of TCR/CD3 complex internal-\nization and degradation, we found that for Ag-, as for anti-CD3-\ninduced receptor down-modulation, CD3 degradation was pro-\nportional to receptor internalization and down-modulation (Fig. 3).\nThis result is in agreement with other work showing that during\nconstitutive, and mAb-induced TCR internalization, all TCR/CD3\ncomponents are degraded with a rate that is proportional to the rate\nof TCR down-modulation (46) through a mechanism that is de-\npendent on lysosomal activity (72). However, an exception exists\nfor PMA-induced TCR internalization that leads to poor CD3\ndegradation (Fig. 3).\nTogether with data from the literature, the present results are\ncompatible with a role for cbl in the Ag-induced TCR/CD3 inter-\nnalization/degradation following schemes also proposed for the\nepidermal growth factor receptor (29, 30, 33). Clearly, questions\nremain about the mechanisms by which the TCR/CD3 complex is\ndriven into the endocytic pathway at the IS, and in particular\nwhether, as for the BCR, internalization via clathrin-coated vesi-\ncles is dependent on integral raft components (73, 74).",
         null
        ],
        [
         "10",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "Applying Machine Learning Techniques to Improve Linux Process Scheduling",
         "In this work we use Machine Learning (ML) techniques to learn the CPU time-slice utilization behavior of known programs in a Linux system. Learning is done by an analysis of certain static and dynamic attributes of the processes while they are being run. Our objective was to discover the most important static and dynamic attributes of the processes that can help best in prediction of CPU burst times which minimize the process TaT (Turn-around-Time). In our experimentation we modify the Linux Kernel scheduler (version 2.4.20-8) to allow scheduling with customized time slices. The \"Waikato Environment for Knowledge Analysis\" (Weka), an open source machine-learning tool is used to find the most suitable ML method to characterize our programs. We experimentally find that the C'4.5 Decision Tree algorithm most effectively solved the problem. We find that predictive scheduling could reduce TaT in the range of 1.4% to 5.8%. This was due to a reduction in the number of context switches needed to complete the process execution. We find our result interesting in the context that generally operating systems presently never make use of a program's previous execution history in their scheduling behavior.",
         "Performance Contracts: Predicting and Monitoring Grid Application Behavior The design, implementation and initial evaluation of an advanced knowledge-based process scheduler Understanding the Linux Kernel Predicting Application Run Times Using Historical Information A Historical Application Profiler for Use by Parallel Schedulers Context Switching and IPC Performance Comparison between uClinux and Linux on the ARM9 based Processor Characterizing Process Execution Behaviour Using Machine Learning Techniques The Pearson Education Linux Kernel Development , 1st ed Correlation-based Feature Selection for Machine Learning A Dynamically Adaptive CPU Scheduler Linux Kernel 2.4 Internals Co-rrelation based feature selection for Machine Learning, Master Thesis, Department of Computer Science, University of Waikato Machine Learning , 1st ed, pp: 52-75, 154-183, 230-244, The WEKA: The Waikato Environment for Knowledge Analysis ELF: From The Programmer's Perspective The Design of the UNIX Operating System Build the knowledge base of static and dynamic characteristics of the programs from the run traces obtained in step 1 and train them with the C4.5 decision tree algorithm is a Masters student in the Department of Computer and Informations Sciences We send this STS information to modi(cid:2)ed scheduler through a system call The scheduler instructs the CPU such that CPU allocates STS ticks to ’X’ decision tree will classify ’X’ and output the STS. per second.(cid:2)le Aivazian , Tigran , Linux Kernel 2 . 4 Internals , The Linux Documentation Project , August , 2002",
         "http://www.cs.ucr.edu/%7Ekishore/papers/tencon.pdf",
         "Abstract In this work we use Machine Learning (ML) tech-\nniques to learn the CPU time-slice utilization behavior of known\nprograms in a Linux system. Learning is done by an analysis of\ncertain static and dynamic attributes of the processes while they\nare being run. Our objective was to discover the most important\nstatic and dynamic attributes of the processes that can help best\nin prediction of CPU burst times which minimize the process\nTaT (Turn-around-Time). In our experimentation we modify the\nLinux Kernel scheduler (version 2.4.20-8) to allow scheduling\nwith customized time slices. The Waikato Environment for\nKnowledge Analysis (Weka), an open source machine-learning\ntool is used to nd the most suitable ML method to characterize\nour programs. We experimentally nd that the C4.5 Decision\nTree algorithm most effectively solved the problem. We nd that\npredictive scheduling could reduce TaT in the range of 1:4%\nto 5:8%. This was due to a reduction in the number of context\nswitches needed to complete the process execution. We nd our\nresult interesting in the context that generally operating systems\npresently never make use of a program’s previous execution\nhistory in their scheduling behavior.",
         "slices to a process according to a scheduling algorithm\nthat does not use any previous execution history of the\nprocess. It might be possible to better utilize resources if\nwe could recognize a program and predict\nits resource\nrequirements. For example, consider a process which was\nallocated a xed CPU time is pre-empted while having a\nvery small CPU time needed for completion. Such preemption,\nincreases number of context switches (also called as a process\nswitch or task switch), it causes invalidation of caches and\npipelines, swapping of buffers and so on [6]. Thus ultimately\nthis increases TaT of the program. (TaT is the delay between\nprocess submission and process completion). Thus we observe\nthat by characterizing or recognizing programs it may be\npossible to understand their previous execution history and\npredict their resource requirements. In this paper we address\nthe problem of, how to minimize the TaT of programs by\nusing ML techniques.We discover certain static and dynamic\ncharacteristics of a program are taken as features which the\nmachine learning techniques use to predict CPU burst times.\nWe call Special Time Slice or STS as the CPU burst time that\nminimizes turnaround time.\nThe rest of this paper rst discusses related work in Section II,\nan overview of machine learning techniques in Section III, we\nrerview Linux process scheduling in Section IV, and then de-\nscribes our implementation in Section V. Section VI describes\nour experimentation. Section VII presents the conclusions and\ndescribes possible future work.\nII. RELATED WORK\nThe work [10] by Suranauwarat and Taniguchi, presents\nan approach to remember the previous execution behavior of\ncertain well-known programs. They study the process times\nof these programs in variousimilaritys states. The knowledge\nof the program ow sequence (PFS) is used to extend the\nCP U time slice of a process. P F S of a process is computed\nfrom its past execution history. P F S characterizes the process\nexecution behavior and is used to decide whether the program\nexecuting currently needs additional time. They set a threshold\nTm called as maximum dispatch delay time, which determines\nthe time limit for context switching; and is a multiple of the\ndelay for minimum process switching time. They control the\nCP U time of a process Tp, by either reducing or increasing\nsome scaling feature. They conclude from experimental results\nthat overall processing time is reduced for known programs.\nTo schedule a process they search for its name in the PFS\nknowledge base and thus improve its behavior.\nIn the paper by Smith et al. [7], the authors predicted\nthe application run times using historical information. They\npresent a technique for deriving predictions for the run\ntimes of parallel applications from the run times of similar\napplications that have executed in the past. They use the\nfollowing characteristics to dene similarity: user, queue, load\nleveler script, arguments, network adapter, number of nodes,\nmaximum run time, submission time, start time, run time.\nThese characteristics are used to make a template which can\nnd the similarity by matching. They use genetic algorithms\nfor identifying good templates for a particular workload.\nA template denes the similarity between two applications.\nThey use Genetic Algorithm (GA) techniques to determine\nthose application characteristics that yield the best denition\nof similarity for making predictions. GAs are probabilistic\ntechniques and are well known for exploring large search\nspaces. However from a machine learning perspective, GAs\nare expensive in terms of computation [15] and also their\nresults are considered fragile.\nIn the paper by Gibbons [5], the authors used Statistical\nRegression methods for prediction. Wh ile regression methods\nwork well on numeric data but cannot readily be used for\nnominal data. In the paper by Fredrik et al. [4], the authors\ndescribed an application signature model used for predicting\napplication performance on a given set of grid resources.\nIn this model,\nthey introduced the notion of application\nintrinsic behaviour to separate the performance effects of the\nruntime system from the behaviour inherent in the application\nitself.The signature model is used as the basis for performance\npredictions. So their approach combines the knowledge of\napplication intrinsic behaviour with run-time predictions of\nresources. They also dene application intrinsic metrics as\nmetrics that are solely dependent on the application code and\nproblem parameters.\nFrom this brief review of related literature, we draw the\nfollowing conclusions:\n It is possible to protably predict scheduling behavior.\n However the success of the approach depends upon\nthe ML technique used to train on previous programs\nexecution behavior.\n More importantly for the ML techniques to succeed, it\nneeds a suitable characterization of the program attributes\n(features) that are most signicant for prediction.\nIII. AN INTRODUCTION TO MACHINE LEARNING\nTECHNIQUES USED\nIn this section we bring in some background information\nabout the machine learning techniques which were used in\nthe experimentation. The eld of machine learning (ML) [15]\nis concerned with the question of how to construct computer\nprograms that automatically improve with experience. ML\nmethods are also similar to Pattern Recognition methods\nis that while training\nas [16]. The most\nor learning stage several attributes are used to describe the\nobjects being studied. In our case the objects being studied\nare programs and we have come up with the best attributes\nthat can desribe programs for prediction. [8]. However the\nsuccess of a ML method also depends upon the classiers\nused. Certain classiers are more successful for different\ndata or problem domains. Here we just give a cursosry\nintroduction to the two most effective which were used [8]\nimportant aspect\nThe Learning Algorithms (Classiers): For our experi-\nments, we selected a representative set of standard machine\nlearning algorithms with different model classes. All of these\nare available in the Waikato Environment for Knowledge\nAnalysis (Weka) [9] , an open source machine learning tool.\nThe selected learner types (classier types) were: Trees and\nLazy which are listed below:\n C4.5 (or J48 in Weka [9] ) is a Reduced-Pruned Decision\nTree Learner. Decision Tree learning is one of the most\nwidely used and practical methods for inductive infer-\nence. It is a method for approximating discrete-valued\nfunctions, and is robust to noisy data and capable of\nlearning disjunctive expressions.\n IB k Learner (k-NN): This is an instance based learner\nantd assumes all instances are points in an n-dimensional\nspace. The nearest neighbors (nn) of an instance are\ndened in terms of the standard Euclidean Distance. The\nclass label of a new instance is found from the k instances\nnearest to it by assigning to it the majority label of the\nk-NN [15] [9].\nIV. A REVIEW OF LINUX PROCESS SCHEDULING\nCONCEPTS\nIn this section, we review the Linux process scheduling\nconcepts which we use to develop the modied scheduler as\ndescribed in Section V.\nScheduler: The idea behind a scheduler is simple. Its\nobjective is to best utilize processor time. Assuming there are\nrunnable processes, a process should always be running. If\nthere are more processes than processors in a system, some\nprocesses will not always be running. These processes are\nwaiting to run. Deciding which process runs next, given a\nset of runnable processes, is the fundamnetal decision the\nscheduler must take.\nTime slice: Linux uses a tick to allocate processor time\nto individual processes. There are several different hardware\ntimers that the kernel can access. One is loaded at boot time\nwith values to interrupt the system every tick. The kernel\nuses the CPU clock speed given as HZ in the kernel to set\nthe length of a single tick to approximately 2.5 nanoseconds\n[2]. Consequently, every 2.5 nanoseconds, the system takes\nover, decrements the amount of ticks that the current process\nhas, and returns the CPU to the process for another tick.\nIf the process running is out of CPU time, the scheduler\nis invoked. If there are more processes in the run-queue\nthat have been allocated CPU time, the scheduler will nd\nthe best one and switch to it. The best process is selected\nusing the task timeslice() macro dened in sched:c. This\ntask timeslice() macro calculates a weight for the process\nusing bonus which is calculated from nice values etc.. The\nhighest weight is the best process to schedule [14] [2].\nProcess Descriptor: The process descriptor [13] is dened\nin sched:h and holds all information about a process. The\namount of ticks of CPU time that a process receives before\nanother process is scheduled is stored in the process descriptor.\nWhen all processes in the run-queue have exhausted their\nCPU time,\nthe scheduler recalculates the amount of CPU\ntime for each process using the macro task timeslice(). The\ntask timeslice() uses the process’ nice value in the process\ndescriptor to determine the number of ticks the process gets.\nHere a process is referred to as p. Variables in the process\ndescriptor are referred to as p!variable. For example p!nice\nrefers to the nice value of process p.\nContext switching: The context of a process [3] consists\nof the contents of its (user) address space and the contents of\nhardware registers and kernel data structures that relate to the\nprocess. In multi-processing environments, a context switch\n(or process switch or task switch) is when one process is\nsuspended from execution , its context is recorded and another\nprocess starts its execution in the CPU.",
         "E. Experimental environment\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\nVI. EXPERIMENTS AND RESULTS\nWe explain the experimentation process by taking the matrix\nmultiplication program as an example.\nA. Effectiveness of STS\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\nbest technique2 to predict the STS [8].\nC. Finding the best attributes which can characterize STS\nclass\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.\nACKNOWLEDGMENT\nWe would like to thank Sasi Kanth Ala for giving valuable\nadvice through out of our work.",
         "V. IMPLEMENTATION\nIn this section, we briey desribe about kernel modications\nand process charcterization. The kernel modications subsec-\ntion shows the kernel data structures modied for nding STS.\nThe process characterization section shows, how to nd the\nstatic and dynamic characteristics of a process.\nA. Kernel modications\nOur idea of kernel modications is similar to that of [12],\nbut the difference is that we describe the modications to\nO(1) scheduler of Linux Kernel 2.4.20-8.\n1) Process descriptor: The Linux process descriptor is\ndened in sched:h as a structure called task struct. More than\none process can request a larger amount of processor time and\nthe amount of extra time to be given is specic to the process.\nTherefore the extra time a process gets is stored in the\nprocess descriptor. A single integer eld, special time slice is\nadded to the process descriptor and is sufcient to store the\nnumber of ticks that a CPU-bound process should receive.\nThis single variable can also be used to distinguish between\nCPU-bound process from the other processes in the system.\nIf p!special time slice is less than 0 then this process should\nbe allocated the standard amount of ticks dened by the\nmacro task timeslice() in sched:c. If p!special time slice\nis greater than 0 then the process is given special time slice\nticks of processing time instead of using the standard macro,\ntask timeslice(). This requires that when processes are created,\nthe special time slice variable must be initialized to 1 in\nfork.c. The next sections describe how the modied scheduler\nuses the special time slice variable in the process descriptor.\n2) System calls: Two system calls have been added to\nthe kernel for controlling CPU-bound processes. One system\ncall is necessary to allocate more processor time for a single\nprocess. This system call performs two simple tasks. First,\nthe special time slice variable is set to the number of ticks\nrequested by the process via an argument to the system call.\nSecond, the time slice variable of the process descriptor is set\nto the value passed to the system call. The process then returns\nto run on the processor with an increased amount of processor\ntime and special handling from the scheduler. Therefore the\nprocess gets special time slice ticks of processor time, each\ntime the scheduler recalculates the process time slices.\nThe second system call\nis used to return a process to\nthe normal state in which p!time slice is calculated using\ntask timeslice() and p!special time slice is less than 0.\nto\nThe next\ntask timeslice(p), which is the standard method of assigning\np!time slice.\ntime the scheduler runs the time slice is set\n3) Modications to O(1) scheduler data structures: The\nscheduler tick() function of sched:c has been modied to\nhandle any process with special time slice > 0 differently\nfrom other processes.If this value is exhausted (decremented to\n0) then the scheduler gives the processor to the next process.\nThe run-queue is a circular doubly linked list of process\ndescriptors in the TASK RUNNING state.The rst thing that the\nnew scheduler does is check whether process currently running\nhas been granted more processor time (p! special time slice\n> 0). If this is true, and the process still has some CPU time\nleft (p!time slice > 0), then the scheduler immediately gives\nthe processor back to the current process. This guarantees that\neach processor-bound process uses all of its processor time\nimmediately, even if the scheduler is called in the middle\nof its running. If the CPU-bound process has used all of its\nCPU time (p!time slice > 0), then the scheduler continues\nits normal operation and switch to the next process.\nCalculating the goodness [2] for each process remains\nthe same. Since CPU-bound processes have large time slice\nvalues, they are not favored over other processes. This is\nacceptable since the CPU-bound processes are typically the\nonly things using the processor on a system. The CPU-\nbound process may be last to run, but there should not be\nmuch waiting to run in front of them, and from the previous\nmodication these processes is guaranteed to stay on the\nprocessor until they have used all of there CPU time.\nA slight modication to the recalculation of the time slice\nvariables checks each process to see if it can be allot-\nted more than task timeslice(p) ticks of processing time. If\nthe p!special time slice > 0, then p!time slice is set to\np!special time slice. There is a small amount of overhead\nadded to the scheduling algorithm, because each process has\nto be checked to see if it is a CPU-bound process. All other\nprocesses receive the same time slice value, as they would\nunder normal operation of the scheduler. The rest of the\nscheduling algorithm remains the same.\nB. Process characterization\nThis section describes , how we characterized the programs.\nWe studied the execution behaviour of several programs to nd\nout the characteristics that can be used to predict the STS. We\nhave taken representative programs : matrix multiplication,\nsorting programs, recursive bnocci number generating pro-\ngrams and random number generator programs. Any Machine\nlearning technique [15] requires fully labeled (categorized)\ndata for training. We categorized our data into classes and\ntrained the classiers (machine-learning techniques) [15].\nThe experimental procedure is divided into two phases. In\nthe rst phase, we create the data set from the program’s run\ntraces and make the data base with the static and dynamic\ncharactersitics of the programs and train it with machine\nlearning techniques. In the second phase, we classify this data\naccording to the tness function, STS by using machine-\nlearning techniques. The trained classiers are then used on\na different data set called as a test data set. Often due to\nlimitations in size of the data, we train classiers using a leave-\none-out technique [16] which is a standard technique.\n1) Creating the dataset: To characterize the program ex-\necution behaviour , we need to nd the static and dynamic\ncharacteristics. We used readelf [1]and size commands to\nget the attributes as shown in Table I. To nd STS, a script\nran the matrix multiplication program of size 700 x 700\nmultiple times with different values of STS on P4 Linux\nDenition\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for\ndynamic linking.\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for dy-\nnamic linking, most commonly the strings that\nrepresent\nthe names associated with symbol\ntable entries .\nThis is size ( bytes) of the dynamic linking\nsymbol table .\nThis is the global offset table size in bytes.\nThis is the procedure linkage table size in\nbytes.\nThis is read-only data size (bytes) that typically\ncontribute to a non-writable segment\nin the\nprocess image.\nThis is the size (bytes) of the rellocation infor-\nmation size.\nThis is the text or executable instructions,\nsize (bytes )of a program.\nThis is the size(bytes) of the initialized data\nthat contribute to the size of the program’s\nmemory image size.\nThis section holds uninitialized data size\nThis is total size (bytes) of the program.\nThis is name of the program and a nominal\nattribute.\nValue of InputSize which depends on the pro-\ngrams type.\nType of the Input which is a nominal value.\nValue of the best time slice.\nClass of the STSs\nSystem and selected the STS, which gives minimum TaT. The\nrst 18 (characteristics) attributes are used to predict the target\nattribute STSclass.\nWe build the data set of 84 execution instances of ve\nprograms: matrix multiplication, quick sort, merge sort, heap\nsort and a recursive Fibonacci number generator and the\nTable II shows an example of a training example or an\ninstance. We collected the data like the above for 5 programs\nwith different input sizes and different best STSs. Data of 84\ninstances of the ve programs was collected and made into\n11 categories based on the attribute STSclass with each class\nhaving an interval of 50 ticks.\n2) Training and Testing methodology: We performed two\ntypes of tests on the training examples with all the learners\ndescribed in the section, Overview of machine learning\nalgorithms, on the data sets collected in the rst phase. The\ntests are:\n Use Training Set [9]: The classier is evaluated on how\nwell it predicts the class of the instance it was trained on.\n Cross-Validation [9]: The classier is evaluated by cross-\nvalidation, using the number of folds that are entered\nin the Folds text eld (Weka). Recognition accuracy\nwas tested via cross-validation. In this test, the training\nexamples are divided into 10 parts and the classier\nclassies by taking one part as a test set and other 9\nparts as training set. Likewise, we continue for all parts.\nThe results of these tests explained in the next section Eval-\nuation and Results.\n3) Extracting the best attributes: Attributes selection1 [11]\ninvolves searching through all possible combinations of at-\ntributes in the data to nd which subset of attributes works best\nfor predicting the program execution behavior. Here our goal\nis to predict the target attribute. To do this, two objects must\nbe set up ; an attribute evaluator and a search method. The\nevaluator determines what method is used to assign a value\n(weight or worth) to each subset of attributes. Here, we used\nExhaustive and Genetic search methods, and Co-rrelation\nbased feature selection (CfsSubsetval) [9] evaluation method.\nA detailed description of feature selection is presented in [11].\nC. The design of modied scheduling process\nThe Figure 1 shows the design of modied scheduling\nprocess and the steps to minimize TaT of a program. The steps\nto minimize TaT of a program are as shown in the Figure 1\nwith numbers from 1 to 5.\n1) The program ’X’ is given to C4.5 decision tree as an\n2) The decision tree will classify ’X’ and output the STS.\n3) We send this STS information to modied scheduler\nthrough a system call.\n4) The scheduler instructs the CPU such that CPU allocates\nper second.le:///usr/share/doc/HTML/index.html Therefore\non 1.6GHz Intel P4 machine , we can save 1600MHz = 16 x\n108 clocks/second = [16 x 108 x no.of pipelines x (60320)]\nlow-level operations , with special time slice (STS) = 200.\nSTS ticks to ’X’.\n5) The CPU allocates STS ticks to ’X’ and it will run with\nB. Selecting the best machine Learning technique\nminimum TaT.\nD. Overall method\nOur method is explained as follows :\n1) Run the programs with different special time slices\nwith modied O(1) scheduler and nd STS (best spe-\ncial time slice) which gives minimum turn-around-time\n(TaT).\n2) Build the knowledge base of static and dynamic char-\nacteristics of the programs from the run traces obtained\nin step 1 and train them with the C4.5 decision tree\nalgorithm.\n3) If a new program comes, classify it and run the program\nwith this predicted STS.\n4) If the new program instance is not in the knowledge-\nbase, go to step 1.\nE. Experimental environment\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\nVI. EXPERIMENTS AND RESULTS\nWe explain the experimentation process by taking the matrix\nmultiplication program as an example.\nA. Effectiveness of STS\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\nbest technique2 to predict the STS [8].\nC. Finding the best attributes which can characterize STS\nclass\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.\nACKNOWLEDGMENT\nWe would like to thank Sasi Kanth Ala for giving valuable\nadvice through out of our work.",
         "E. Experimental environment\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\nVI. EXPERIMENTS AND RESULTS\nWe explain the experimentation process by taking the matrix\nmultiplication program as an example.\nA. Effectiveness of STS\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\nbest technique2 to predict the STS [8].\nC. Finding the best attributes which can characterize STS\nclass\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.\nACKNOWLEDGMENT\nWe would like to thank Sasi Kanth Ala for giving valuable\nadvice through out of our work.",
         "Abstract In this work we use Machine Learning (ML) tech-\nniques to learn the CPU time-slice utilization behavior of known\nprograms in a Linux system. Learning is done by an analysis of\ncertain static and dynamic attributes of the processes while they\nare being run. Our objective was to discover the most important\nstatic and dynamic attributes of the processes that can help best\nin prediction of CPU burst times which minimize the process\nTaT (Turn-around-Time). In our experimentation we modify the\nLinux Kernel scheduler (version 2.4.20-8) to allow scheduling\nwith customized time slices. The Waikato Environment for\nKnowledge Analysis (Weka), an open source machine-learning\ntool is used to nd the most suitable ML method to characterize\nour programs. We experimentally nd that the C4.5 Decision\nTree algorithm most effectively solved the problem. We nd that\npredictive scheduling could reduce TaT in the range of 1:4%\nto 5:8%. This was due to a reduction in the number of context\nswitches needed to complete the process execution. We nd our\nresult interesting in the context that generally operating systems\npresently never make use of a program’s previous execution\nhistory in their scheduling behavior.\nslices to a process according to a scheduling algorithm\nthat does not use any previous execution history of the\nprocess. It might be possible to better utilize resources if\nwe could recognize a program and predict\nits resource\nrequirements. For example, consider a process which was\nallocated a xed CPU time is pre-empted while having a\nvery small CPU time needed for completion. Such preemption,\nincreases number of context switches (also called as a process\nswitch or task switch), it causes invalidation of caches and\npipelines, swapping of buffers and so on [6]. Thus ultimately\nthis increases TaT of the program. (TaT is the delay between\nprocess submission and process completion). Thus we observe\nthat by characterizing or recognizing programs it may be\npossible to understand their previous execution history and\npredict their resource requirements. In this paper we address\nthe problem of, how to minimize the TaT of programs by\nusing ML techniques.We discover certain static and dynamic\ncharacteristics of a program are taken as features which the\nmachine learning techniques use to predict CPU burst times.\nWe call Special Time Slice or STS as the CPU burst time that\nminimizes turnaround time.\nThe rest of this paper rst discusses related work in Section II,\nan overview of machine learning techniques in Section III, we\nrerview Linux process scheduling in Section IV, and then de-\nscribes our implementation in Section V. Section VI describes\nour experimentation. Section VII presents the conclusions and\ndescribes possible future work.\nII. RELATED WORK\nThe work [10] by Suranauwarat and Taniguchi, presents\nan approach to remember the previous execution behavior of\ncertain well-known programs. They study the process times\nof these programs in variousimilaritys states. The knowledge\nof the program ow sequence (PFS) is used to extend the\nCP U time slice of a process. P F S of a process is computed\nfrom its past execution history. P F S characterizes the process\nexecution behavior and is used to decide whether the program\nexecuting currently needs additional time. They set a threshold\nTm called as maximum dispatch delay time, which determines\nthe time limit for context switching; and is a multiple of the\ndelay for minimum process switching time. They control the\nCP U time of a process Tp, by either reducing or increasing\nsome scaling feature. They conclude from experimental results\nthat overall processing time is reduced for known programs.\nTo schedule a process they search for its name in the PFS\nknowledge base and thus improve its behavior.\nIn the paper by Smith et al. [7], the authors predicted\nthe application run times using historical information. They\npresent a technique for deriving predictions for the run\ntimes of parallel applications from the run times of similar\napplications that have executed in the past. They use the\nfollowing characteristics to dene similarity: user, queue, load\nleveler script, arguments, network adapter, number of nodes,\nmaximum run time, submission time, start time, run time.\nThese characteristics are used to make a template which can\nnd the similarity by matching. They use genetic algorithms\nfor identifying good templates for a particular workload.\nA template denes the similarity between two applications.\nThey use Genetic Algorithm (GA) techniques to determine\nthose application characteristics that yield the best denition\nof similarity for making predictions. GAs are probabilistic\ntechniques and are well known for exploring large search\nspaces. However from a machine learning perspective, GAs\nare expensive in terms of computation [15] and also their\nresults are considered fragile.\nIn the paper by Gibbons [5], the authors used Statistical\nRegression methods for prediction. Wh ile regression methods\nwork well on numeric data but cannot readily be used for\nnominal data. In the paper by Fredrik et al. [4], the authors\ndescribed an application signature model used for predicting\napplication performance on a given set of grid resources.\nIn this model,\nthey introduced the notion of application\nintrinsic behaviour to separate the performance effects of the\nruntime system from the behaviour inherent in the application\nitself.The signature model is used as the basis for performance\npredictions. So their approach combines the knowledge of\napplication intrinsic behaviour with run-time predictions of\nresources. They also dene application intrinsic metrics as\nmetrics that are solely dependent on the application code and\nproblem parameters.\nFrom this brief review of related literature, we draw the\nfollowing conclusions:\n It is possible to protably predict scheduling behavior.\n However the success of the approach depends upon\nthe ML technique used to train on previous programs\nexecution behavior.\n More importantly for the ML techniques to succeed, it\nneeds a suitable characterization of the program attributes\n(features) that are most signicant for prediction.\nIII. AN INTRODUCTION TO MACHINE LEARNING\nTECHNIQUES USED\nIn this section we bring in some background information\nabout the machine learning techniques which were used in\nthe experimentation. The eld of machine learning (ML) [15]\nis concerned with the question of how to construct computer\nprograms that automatically improve with experience. ML\nmethods are also similar to Pattern Recognition methods\nis that while training\nas [16]. The most\nor learning stage several attributes are used to describe the\nobjects being studied. In our case the objects being studied\nare programs and we have come up with the best attributes\nthat can desribe programs for prediction. [8]. However the\nsuccess of a ML method also depends upon the classiers\nused. Certain classiers are more successful for different\ndata or problem domains. Here we just give a cursosry\nintroduction to the two most effective which were used [8]\nimportant aspect\nThe Learning Algorithms (Classiers): For our experi-\nments, we selected a representative set of standard machine\nlearning algorithms with different model classes. All of these\nare available in the Waikato Environment for Knowledge\nAnalysis (Weka) [9] , an open source machine learning tool.\nThe selected learner types (classier types) were: Trees and\nLazy which are listed below:\n C4.5 (or J48 in Weka [9] ) is a Reduced-Pruned Decision\nTree Learner. Decision Tree learning is one of the most\nwidely used and practical methods for inductive infer-\nence. It is a method for approximating discrete-valued\nfunctions, and is robust to noisy data and capable of\nlearning disjunctive expressions.\n IB k Learner (k-NN): This is an instance based learner\nantd assumes all instances are points in an n-dimensional\nspace. The nearest neighbors (nn) of an instance are\ndened in terms of the standard Euclidean Distance. The\nclass label of a new instance is found from the k instances\nnearest to it by assigning to it the majority label of the\nk-NN [15] [9].\nIV. A REVIEW OF LINUX PROCESS SCHEDULING\nCONCEPTS\nIn this section, we review the Linux process scheduling\nconcepts which we use to develop the modied scheduler as\ndescribed in Section V.\nScheduler: The idea behind a scheduler is simple. Its\nobjective is to best utilize processor time. Assuming there are\nrunnable processes, a process should always be running. If\nthere are more processes than processors in a system, some\nprocesses will not always be running. These processes are\nwaiting to run. Deciding which process runs next, given a\nset of runnable processes, is the fundamnetal decision the\nscheduler must take.\nTime slice: Linux uses a tick to allocate processor time\nto individual processes. There are several different hardware\ntimers that the kernel can access. One is loaded at boot time\nwith values to interrupt the system every tick. The kernel\nuses the CPU clock speed given as HZ in the kernel to set\nthe length of a single tick to approximately 2.5 nanoseconds\n[2]. Consequently, every 2.5 nanoseconds, the system takes\nover, decrements the amount of ticks that the current process\nhas, and returns the CPU to the process for another tick.\nIf the process running is out of CPU time, the scheduler\nis invoked. If there are more processes in the run-queue\nthat have been allocated CPU time, the scheduler will nd\nthe best one and switch to it. The best process is selected\nusing the task timeslice() macro dened in sched:c. This\ntask timeslice() macro calculates a weight for the process\nusing bonus which is calculated from nice values etc.. The\nhighest weight is the best process to schedule [14] [2].\nProcess Descriptor: The process descriptor [13] is dened\nin sched:h and holds all information about a process. The\namount of ticks of CPU time that a process receives before\nanother process is scheduled is stored in the process descriptor.\nWhen all processes in the run-queue have exhausted their\nCPU time,\nthe scheduler recalculates the amount of CPU\ntime for each process using the macro task timeslice(). The\ntask timeslice() uses the process’ nice value in the process\ndescriptor to determine the number of ticks the process gets.\nHere a process is referred to as p. Variables in the process\ndescriptor are referred to as p!variable. For example p!nice\nrefers to the nice value of process p.\nContext switching: The context of a process [3] consists\nof the contents of its (user) address space and the contents of\nhardware registers and kernel data structures that relate to the\nprocess. In multi-processing environments, a context switch\n(or process switch or task switch) is when one process is\nsuspended from execution , its context is recorded and another\nprocess starts its execution in the CPU.\nV. IMPLEMENTATION\nIn this section, we briey desribe about kernel modications\nand process charcterization. The kernel modications subsec-\ntion shows the kernel data structures modied for nding STS.\nThe process characterization section shows, how to nd the\nstatic and dynamic characteristics of a process.\nA. Kernel modications\nOur idea of kernel modications is similar to that of [12],\nbut the difference is that we describe the modications to\nO(1) scheduler of Linux Kernel 2.4.20-8.\n1) Process descriptor: The Linux process descriptor is\ndened in sched:h as a structure called task struct. More than\none process can request a larger amount of processor time and\nthe amount of extra time to be given is specic to the process.\nTherefore the extra time a process gets is stored in the\nprocess descriptor. A single integer eld, special time slice is\nadded to the process descriptor and is sufcient to store the\nnumber of ticks that a CPU-bound process should receive.\nThis single variable can also be used to distinguish between\nCPU-bound process from the other processes in the system.\nIf p!special time slice is less than 0 then this process should\nbe allocated the standard amount of ticks dened by the\nmacro task timeslice() in sched:c. If p!special time slice\nis greater than 0 then the process is given special time slice\nticks of processing time instead of using the standard macro,\ntask timeslice(). This requires that when processes are created,\nthe special time slice variable must be initialized to 1 in\nfork.c. The next sections describe how the modied scheduler\nuses the special time slice variable in the process descriptor.\n2) System calls: Two system calls have been added to\nthe kernel for controlling CPU-bound processes. One system\ncall is necessary to allocate more processor time for a single\nprocess. This system call performs two simple tasks. First,\nthe special time slice variable is set to the number of ticks\nrequested by the process via an argument to the system call.\nSecond, the time slice variable of the process descriptor is set\nto the value passed to the system call. The process then returns\nto run on the processor with an increased amount of processor\ntime and special handling from the scheduler. Therefore the\nprocess gets special time slice ticks of processor time, each\ntime the scheduler recalculates the process time slices.\nThe second system call\nis used to return a process to\nthe normal state in which p!time slice is calculated using\ntask timeslice() and p!special time slice is less than 0.\nto\nThe next\ntask timeslice(p), which is the standard method of assigning\np!time slice.\ntime the scheduler runs the time slice is set\n3) Modications to O(1) scheduler data structures: The\nscheduler tick() function of sched:c has been modied to\nhandle any process with special time slice > 0 differently\nfrom other processes.If this value is exhausted (decremented to\n0) then the scheduler gives the processor to the next process.\nThe run-queue is a circular doubly linked list of process\ndescriptors in the TASK RUNNING state.The rst thing that the\nnew scheduler does is check whether process currently running\nhas been granted more processor time (p! special time slice\n> 0). If this is true, and the process still has some CPU time\nleft (p!time slice > 0), then the scheduler immediately gives\nthe processor back to the current process. This guarantees that\neach processor-bound process uses all of its processor time\nimmediately, even if the scheduler is called in the middle\nof its running. If the CPU-bound process has used all of its\nCPU time (p!time slice > 0), then the scheduler continues\nits normal operation and switch to the next process.\nCalculating the goodness [2] for each process remains\nthe same. Since CPU-bound processes have large time slice\nvalues, they are not favored over other processes. This is\nacceptable since the CPU-bound processes are typically the\nonly things using the processor on a system. The CPU-\nbound process may be last to run, but there should not be\nmuch waiting to run in front of them, and from the previous\nmodication these processes is guaranteed to stay on the\nprocessor until they have used all of there CPU time.\nA slight modication to the recalculation of the time slice\nvariables checks each process to see if it can be allot-\nted more than task timeslice(p) ticks of processing time. If\nthe p!special time slice > 0, then p!time slice is set to\np!special time slice. There is a small amount of overhead\nadded to the scheduling algorithm, because each process has\nto be checked to see if it is a CPU-bound process. All other\nprocesses receive the same time slice value, as they would\nunder normal operation of the scheduler. The rest of the\nscheduling algorithm remains the same.\nB. Process characterization\nThis section describes , how we characterized the programs.\nWe studied the execution behaviour of several programs to nd\nout the characteristics that can be used to predict the STS. We\nhave taken representative programs : matrix multiplication,\nsorting programs, recursive bnocci number generating pro-\ngrams and random number generator programs. Any Machine\nlearning technique [15] requires fully labeled (categorized)\ndata for training. We categorized our data into classes and\ntrained the classiers (machine-learning techniques) [15].\nThe experimental procedure is divided into two phases. In\nthe rst phase, we create the data set from the program’s run\ntraces and make the data base with the static and dynamic\ncharactersitics of the programs and train it with machine\nlearning techniques. In the second phase, we classify this data\naccording to the tness function, STS by using machine-\nlearning techniques. The trained classiers are then used on\na different data set called as a test data set. Often due to\nlimitations in size of the data, we train classiers using a leave-\none-out technique [16] which is a standard technique.\n1) Creating the dataset: To characterize the program ex-\necution behaviour , we need to nd the static and dynamic\ncharacteristics. We used readelf [1]and size commands to\nget the attributes as shown in Table I. To nd STS, a script\nran the matrix multiplication program of size 700 x 700\nmultiple times with different values of STS on P4 Linux\nDenition\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for\ndynamic linking.\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for dy-\nnamic linking, most commonly the strings that\nrepresent\nthe names associated with symbol\ntable entries .\nThis is size ( bytes) of the dynamic linking\nsymbol table .\nThis is the global offset table size in bytes.\nThis is the procedure linkage table size in\nbytes.\nThis is read-only data size (bytes) that typically\ncontribute to a non-writable segment\nin the\nprocess image.\nThis is the size (bytes) of the rellocation infor-\nmation size.\nThis is the text or executable instructions,\nsize (bytes )of a program.\nThis is the size(bytes) of the initialized data\nthat contribute to the size of the program’s\nmemory image size.\nThis section holds uninitialized data size\nThis is total size (bytes) of the program.\nThis is name of the program and a nominal\nattribute.\nValue of InputSize which depends on the pro-\ngrams type.\nType of the Input which is a nominal value.\nValue of the best time slice.\nClass of the STSs\nSystem and selected the STS, which gives minimum TaT. The\nrst 18 (characteristics) attributes are used to predict the target\nattribute STSclass.\nWe build the data set of 84 execution instances of ve\nprograms: matrix multiplication, quick sort, merge sort, heap\nsort and a recursive Fibonacci number generator and the\nTable II shows an example of a training example or an\ninstance. We collected the data like the above for 5 programs\nwith different input sizes and different best STSs. Data of 84\ninstances of the ve programs was collected and made into\n11 categories based on the attribute STSclass with each class\nhaving an interval of 50 ticks.\n2) Training and Testing methodology: We performed two\ntypes of tests on the training examples with all the learners\ndescribed in the section, Overview of machine learning\nalgorithms, on the data sets collected in the rst phase. The\ntests are:\n Use Training Set [9]: The classier is evaluated on how\nwell it predicts the class of the instance it was trained on.\n Cross-Validation [9]: The classier is evaluated by cross-\nvalidation, using the number of folds that are entered\nin the Folds text eld (Weka). Recognition accuracy\nwas tested via cross-validation. In this test, the training\nexamples are divided into 10 parts and the classier\nclassies by taking one part as a test set and other 9\nparts as training set. Likewise, we continue for all parts.\nThe results of these tests explained in the next section Eval-\nuation and Results.\n3) Extracting the best attributes: Attributes selection1 [11]\ninvolves searching through all possible combinations of at-\ntributes in the data to nd which subset of attributes works best\nfor predicting the program execution behavior. Here our goal\nis to predict the target attribute. To do this, two objects must\nbe set up ; an attribute evaluator and a search method. The\nevaluator determines what method is used to assign a value\n(weight or worth) to each subset of attributes. Here, we used\nExhaustive and Genetic search methods, and Co-rrelation\nbased feature selection (CfsSubsetval) [9] evaluation method.\nA detailed description of feature selection is presented in [11].\nC. The design of modied scheduling process\nThe Figure 1 shows the design of modied scheduling\nprocess and the steps to minimize TaT of a program. The steps\nto minimize TaT of a program are as shown in the Figure 1\nwith numbers from 1 to 5.\n1) The program ’X’ is given to C4.5 decision tree as an\n2) The decision tree will classify ’X’ and output the STS.\n3) We send this STS information to modied scheduler\nthrough a system call.\n4) The scheduler instructs the CPU such that CPU allocates\nper second.le:///usr/share/doc/HTML/index.html Therefore\non 1.6GHz Intel P4 machine , we can save 1600MHz = 16 x\n108 clocks/second = [16 x 108 x no.of pipelines x (60320)]\nlow-level operations , with special time slice (STS) = 200.\nSTS ticks to ’X’.\n5) The CPU allocates STS ticks to ’X’ and it will run with\nB. Selecting the best machine Learning technique\nminimum TaT.\nD. Overall method\nOur method is explained as follows :\n1) Run the programs with different special time slices\nwith modied O(1) scheduler and nd STS (best spe-\ncial time slice) which gives minimum turn-around-time\n(TaT).\n2) Build the knowledge base of static and dynamic char-\nacteristics of the programs from the run traces obtained\nin step 1 and train them with the C4.5 decision tree\nalgorithm.\n3) If a new program comes, classify it and run the program\nwith this predicted STS.\n4) If the new program instance is not in the knowledge-\nbase, go to step 1.\nE. Experimental environment\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\nVI. EXPERIMENTS AND RESULTS\nWe explain the experimentation process by taking the matrix\nmultiplication program as an example.\nA. Effectiveness of STS\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\nbest technique2 to predict the STS [8].\nC. Finding the best attributes which can characterize STS\nclass\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.\nACKNOWLEDGMENT\nWe would like to thank Sasi Kanth Ala for giving valuable\nadvice through out of our work.",
         null
        ],
        [
         "11",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "Neuronal Leucine-Rich Repeat Protein 4 Functions in Hippocampus-Dependent Long-Lasting Memory",
         "ABSTRACT Neuronal leucine-rich repeat proteins (NLRRs) are type I transmembrane proteins and expressed in neuronal tissues, but their function remains unknown. Here, we describe the identification and characterization of a new member of the NLRR family, NLRR4, and its potential role in long-lasting memory. We generated NLRR4-deficient (NLRR4−/−) mice and found that they showed impaired memory retention. In hippocampus-dependent learning tasks, NLRR4−/− mice were able to learn and maintain the memories for one day but unable to retain the memories for four days after learning. In contrast, in a hippocampus-independent task, NLRR4−/− mice were able to retain the memory normally for at least seven days. These results suggest that NLRR4 plays a key role in hippocampus-dependent long-lasting memory.",
         "",
         "https://europepmc.org/articles/pmc1087730?pdf=render",
         null,
         null,
         null,
         null,
         null,
         null,
         null
        ],
        [
         "12",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "Semantic video analysis for adaptive content delivery and automatic description",
         "We present an encoding framework which exploits semantics for video content delivery. The video content is organized based on the idea of main content message. In the work reported in this paper, the main content message is extracted from the video data through semantic video analysis, an application-dependent process that separates relevant information from non relevant information. We use here semantic analysis and the corresponding content annotation under a new perspective: the results of the analysis are exploited for object-based encoders, such as MPEG-4, as well as for frame-based encoders, such as MPEG-1. Moreover, the use of MPEG-7 content descriptors in conjunction with the video is used for improving content visualization for narrow channels and devices with limited capabilities. Finally, we analyze and evaluate the impact of semantic video analysis in video encoding and show that the use of semantic video analysis prior to encoding sensibly reduces the bandwidth requirements compared to traditional encoders not only for an object-based encoder but also for a frame-based encoder.",
         "Automatic foveation for video compression using a neurobiological model of visual attention Text information extraction in images and video: a survey DAVE: a system for quality driven adaptive video delivery Video transmission adaptation on mobile devices Visual attention for region of interest coding in JPEG 2000 Semantic segmentation and description for video transcoding Resource adaptation based on MPEG-21 usage environment descriptions Video transcoding architectures and techniques: an overview Metadata-driven multimedia access Semantic transcoding for live video server Multiple video object tracking in complex scenes Complexity-quality analysis of transcoding architectures for reduced spatial resolution Instantly indexed multimedia databases of real world events Face detection in color images MPEG-7 description of generic video objects for scene reconstruction A new content-based hybrid video transcoding method Video Processing and Communications Media conversions to support mobile users Object-based transcoding for adaptable video content delivery Overview of fine granularity scalability in MPEG-4 video standard Video object extraction based on adaptive background and statistical change detection Video transcoding for universal multimedia access Heterogeneous Video Transcoding to Lower Spatio-Temporal Resolutions and Different Encoding Formats Scalable multimedia delivery for pervasive computing Adapting Multimedia Internet Content for Universal Access A frequency-domain video transcoder for dynamic bit-rate reduction of MPEG-2 bit streams Multimedia content description in the InfoPyramid Objective methods for assessment of video quality : state of the art On the methods and performances of rational downsizing video transcoding Shadow detection in image sequences Video Transcoding Architectures and Techniques : An Overview Visual radio: should we paint pictures with workds, or pictures? Dave: A system for qual ity driven adaptive video delivery,”Proceedings of the 5th ACM SIGMM international workshop on Multimedia information retriev Processing and Information technology – coding of audio-visual objects – part 2 visual–amendment 2: Streaming video profiles Statistics, 3 ed Continuous speech recognit  io Submitted by Information technology — Generic coding of moving pictures and associated audio information — Part 2: Video SUBJECTIVE VIDEO QUALITY ASSESSMENT METHODS FOR MULTIMEDIA APPLICATIONS",
         "https://infoscience.epfl.ch/record/87259/files/Cavallaro2005_1275.pdf",
         "Abstract— We present an encoding framework which exploits\nsemantics for video content delivery. The video content is orga-\nnized based on the idea of main content message. In the work\nreported in this paper, the main content message is extracted from\nthe video data through semantic video analysis, an application-\ndependent process that separates relevant information from non\nrelevant information. We use here semantic analysis and the\ncorresponding content annotation under a new perspective: the\nresults of the analysis are exploited for object-based encoders,\nsuch as MPEG-4, as well as for frame-based encoders, such as\nMPEG-1. Moreover, the use of MPEG-7 content descriptors in\nconjunction with the video is used for improving content visual-\nization for narrow channels and devices with limited capabilities.\nFinally, we analyze and evaluate the impact of semantic video\nanalysis in video encoding and show that the use of semantic\nvideo analysis prior to encoding sensibly reduces the bandwidth\nrequirements compared to traditional encoders not only for an\nobject-based encoder but also for a frame-based encoder.",
         "I. INTRODUCTION\nThe diffusion of network appliances such as cellular phones,\npersonal digital assistants, and hand-held computers creates a\nnew challenge for content delivery: how to adapt the media\ntransmission to various device capabilities, network charac-\nteristics, and user preferences [1], [2], [3]. Each device is\ncharacterized by certain display capabilities and processing\npower. Moreover, such appliances are connected through dif-\nferent kinds of networks with diverse bandwidths. Finally,\nusers with different preferences access the same multimedia\ncontent. Therefore there exists a need to personalize the way\nmedia content is delivered to the end user. In addition to the\nabove, recent devices, such as digital radio receivers, and new\napplications, such as intelligent visual surveillance, require\nnovel forms of video analysis for content adaptation and sum-\nmarization. Digital radios allow for the display of additional\ninformation alongside the traditional audio stream to enrich the\naudio content. For instance, digital audio broadcasting (DAB)\nallocates 128 Kb/s to streaming audio, whereas 8Kb/s can\nbe used to send additional information, such as visual data\n[4]. Moreover, the growth of video surveillance systems poses\nchallenging problems for the automatic analysis, interpretation\nand indexing of video data as well as for selective content\nﬁltering for privacy preservation. Finally, the instantaneous\nindexing of video content is also an desirable feature for sports\nbroadcasting [5].\nTo cope with these challenges, video content needs to be\nautomatically analyzed and adapted to the needs of the speciﬁc\napplication, to the capabilities of the connected terminal and\nnetwork, and to the preferences of the user. Three main\nstrategies for adaptive content delivery have been proposed\nthroughout the literature, namely Info Pyramid, scalable cod-\ning and transcoding. The work presented in this paper aims\nto go beyond traditional adaptation techniques. We focus on\nsemantic encoding by looking to exploit video analysis prior\nto encoding (Figure 1). Speciﬁcally, we use semantic video\nanalysis to extract relevant areas of a video. These areas are\nencoded at a higher level of quality or summarized in textual\nform. The idea behind this approach is to organize the content\nso that a particular network or device does not inhibit the main\ncontent message. The main content message is dependent on\nthe speciﬁc application. In particular, for applications such as\nvideo surveillance and sport video the main content message\nis deﬁned based on motion information.\nThe contribution of this paper is twofold. On the one\nhand, a framework for adaptive video delivery is deﬁned and\nimplemented based on video objects and on their associated\nmetadata. On the other hand, two new modalities of video\ndelivery are proposed in such a framework. The ﬁrst modality\ncombines semantic analysis with a traditional frame-based\nvideo encoder. The second modality uses metadata to efﬁ-\nciently encode the main content message. In particular, the\nuse of metadata enables not only to make the content more\nsearchable, but also to improve visualization and to preserve\nprivacy in video-based applications.\nThe paper is organized as follows. Section II is an overview\nof existing adaptation techniques. Section III presents the\nalgorithm for extracting the main content message, the frame-\nwork for adaptive video delivery and automatic description\nusing semantic video analysis. Section IV discusses quality\nassessment issues, whereas experimental results are presented\nin Section V. Finally, Section VI concludes the paper.",
         "of semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\nlocators of the foreground objects,\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.",
         "of semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\nlocators of the foreground objects,\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.",
         "of semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\nlocators of the foreground objects,\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.",
         "Abstract— We present an encoding framework which exploits\nsemantics for video content delivery. The video content is orga-\nnized based on the idea of main content message. In the work\nreported in this paper, the main content message is extracted from\nthe video data through semantic video analysis, an application-\ndependent process that separates relevant information from non\nrelevant information. We use here semantic analysis and the\ncorresponding content annotation under a new perspective: the\nresults of the analysis are exploited for object-based encoders,\nsuch as MPEG-4, as well as for frame-based encoders, such as\nMPEG-1. Moreover, the use of MPEG-7 content descriptors in\nconjunction with the video is used for improving content visual-\nization for narrow channels and devices with limited capabilities.\nFinally, we analyze and evaluate the impact of semantic video\nanalysis in video encoding and show that the use of semantic\nvideo analysis prior to encoding sensibly reduces the bandwidth\nrequirements compared to traditional encoders not only for an\nobject-based encoder but also for a frame-based encoder.\nI. INTRODUCTION\nThe diffusion of network appliances such as cellular phones,\npersonal digital assistants, and hand-held computers creates a\nnew challenge for content delivery: how to adapt the media\ntransmission to various device capabilities, network charac-\nteristics, and user preferences [1], [2], [3]. Each device is\ncharacterized by certain display capabilities and processing\npower. Moreover, such appliances are connected through dif-\nferent kinds of networks with diverse bandwidths. Finally,\nusers with different preferences access the same multimedia\ncontent. Therefore there exists a need to personalize the way\nmedia content is delivered to the end user. In addition to the\nabove, recent devices, such as digital radio receivers, and new\napplications, such as intelligent visual surveillance, require\nnovel forms of video analysis for content adaptation and sum-\nmarization. Digital radios allow for the display of additional\ninformation alongside the traditional audio stream to enrich the\naudio content. For instance, digital audio broadcasting (DAB)\nallocates 128 Kb/s to streaming audio, whereas 8Kb/s can\nbe used to send additional information, such as visual data\n[4]. Moreover, the growth of video surveillance systems poses\nchallenging problems for the automatic analysis, interpretation\nand indexing of video data as well as for selective content\nﬁltering for privacy preservation. Finally, the instantaneous\nindexing of video content is also an desirable feature for sports\nbroadcasting [5].\nTo cope with these challenges, video content needs to be\nautomatically analyzed and adapted to the needs of the speciﬁc\napplication, to the capabilities of the connected terminal and\nnetwork, and to the preferences of the user. Three main\nstrategies for adaptive content delivery have been proposed\nthroughout the literature, namely Info Pyramid, scalable cod-\ning and transcoding. The work presented in this paper aims\nto go beyond traditional adaptation techniques. We focus on\nsemantic encoding by looking to exploit video analysis prior\nto encoding (Figure 1). Speciﬁcally, we use semantic video\nanalysis to extract relevant areas of a video. These areas are\nencoded at a higher level of quality or summarized in textual\nform. The idea behind this approach is to organize the content\nso that a particular network or device does not inhibit the main\ncontent message. The main content message is dependent on\nthe speciﬁc application. In particular, for applications such as\nvideo surveillance and sport video the main content message\nis deﬁned based on motion information.\nThe contribution of this paper is twofold. On the one\nhand, a framework for adaptive video delivery is deﬁned and\nimplemented based on video objects and on their associated\nmetadata. On the other hand, two new modalities of video\ndelivery are proposed in such a framework. The ﬁrst modality\ncombines semantic analysis with a traditional frame-based\nvideo encoder. The second modality uses metadata to efﬁ-\nciently encode the main content message. In particular, the\nuse of metadata enables not only to make the content more\nsearchable, but also to improve visualization and to preserve\nprivacy in video-based applications.\nThe paper is organized as follows. Section II is an overview\nof existing adaptation techniques. Section III presents the\nalgorithm for extracting the main content message, the frame-\nwork for adaptive video delivery and automatic description\nusing semantic video analysis. Section IV discusses quality\nassessment issues, whereas experimental results are presented\nin Section V. Finally, Section VI concludes the paper.\nII. BACKGROUND\nThree main approaches have been presented in the literature\nto provide adaptive content delivery, namely Info Pyramid,\nscalable coding and transcoding. Info Pyramid provides a\ngeneral framework for managing and manipulating media\nobjects [6], [7]. Info Pyramid manages different versions, or\nvariations, of media objects with different modalities (e.g.,\nvideo, image, text, and audio) and ﬁdelities (summarized, com-\npressed, and scaled variations). Moreover, it deﬁnes methods\nfor manipulating, translating, transcoding, and generating the\ncontent. When a client device requests a multimedia document,\nthe server selects and delivers the most appropriate variation.\nThe selection is made based on network characteristics and\nterminal capabilities, such as display size, frame rate, color\ndepth and storage capacity.\nAs opposed to Info Pyramid, scalable coding processes\nmultimedia content only once. Lower qualities, lower spatial\nand temporal resolutions of the same content are then obtained\nby truncating certain layers or bits from the original stream [8].\nBasic modes of video scalability include quality scalability,\nspatial scalability, temporal scalability, and frequency scala-\nbility. Combinations of these basic modes are also possible.\nQuality scalability is deﬁned as the representation of a video\nsequence with varying accuracies in the color patterns. This\nis typically obtained by quantizing the color values with\nincreasingly ﬁner quantization step sizes. Spatial scalability\nis the representation of the same video in varying spatial\nresolutions. Temporal scalability is the representation of the\nsame video at varying temporal resolutions or frame rates.\nFrequency scalability includes different frequency components\nin each layer, with the base layer containing low-frequency\ncomponents and the other layers containing increasingly high-\nfrequency components. Such decomposition can be achieved\nvia frequency transforms like the DCT or wavelet transforms.\nFinally, the basic scalability schemes can be combined to reach\nﬁne-granularity scalability, such as in MPEG–4 FGS [9].\nThe various scalable coding methods introduced previously\nperform the same operation over the entire video frame. In\nobject-based temporal scalability (OTS), the frame rate of\nforeground objects is enhanced so that it has a smoother\nmotion than the background.\nVideo transcoding is the process of converting a compressed\nvideo signal into another compressed signal with different\nproperties. Early solutions to video transcoding determine the\noutput format based on network and appliance constraints,\nindependently of the semantics in the content (content-blind\ntranscoding). Content-blind transcoding strategies include spa-\ntial resolution reduction, temporal resolution reduction, and\nbit-rate reduction [10]. Recent transcoding techniques make\nuse of semantics to minimize the degradation of important\nimage regions [11], [12]. In [13], optimal quantization pa-\nrameters and frame skip are determined for each video object\nindividually. The bit-rate budget for each object is allocated by\na difﬁculty hint, a weight indicating the relative encoding com-\nplexity of each object. Frame skip is controlled by a shape hint,\nwhich measures the difference between two consecutive shapes\nto determine whether an object can be temporally downsam-\npled without visible composition problems. Key objects are\nselected based on motion activity and on bit complexity.\nThe transcoding strategies described thus far are referred\nto as intramedia transcoding strategies and do not change the\nmedia nature of the input signal. On the other hand, intermedia\ntranscoding, or transmoding,\nis the process of converting\nthe media input\ninto another media format. Examples of\nintermedia transcoding include speech-to-text [14] and video-\nto-text [15] translation. Both the intramedia and the intermedia\nadaptation concepts are used in this paper for video encoding,\nas described in the following section.\nIII. ADAPTIVE CONTENT DELIVERY AND DESCRIPTION\nUSING SEMANTICS\nThe proposed framework for adaptive video delivery and\nautomatic description uses video content analysis and semantic\npre-ﬁltering prior to encoding (Figure 1) in order to improve\nthe perceived content quality and to provide additional func-\ntionalities, such as privacy preservation and automatic video\nindexing. Semantic video analysis and semantic encoding are\ndescribed next.\nA. Semantic video analysis\nSemantic video analysis is used to extract the main content\nmessage from the video. The semantics to be included in\nthe analysis is dependent on the speciﬁc application. In the\nfollowing, we discuss possible semantics and, in particular,\nwe describe the use of motion as semantics.\nSemantic video analysis refers to a human abstraction and\nuses a priori information to translate the semantics into rules.\nThe rules are then applied through an algorithm. Examples\nof semantic video analysis based on a priori information are\ntemplate matching, extraction of captions and text, face de-\ntection, and moving object segmentation. Template matching\nis used to implement the semantics when the shape objects\nwe want to segment is known a priori. In this case, which\nincludes in particular the detection of captions and text, the\nextraction method searches for speciﬁc object features in terms\nof geometry. For segmenting faces of people, color-based\nsegmentation can be used [16]. The face detection task consists\nin ﬁnding the pixels whose spectral characteristics lie in a\nspeciﬁc region in the chromaticity diagram. For extracting\nmoving objects, motion information can be used as semantics.\nSeveral applications, such as sport broadcasting and video\nsurveillance, deal with segmenting moving objects.\nA typical tool used to tackle the problem of object segmen-\ntation based on motion is change detection. Different change\ndetection techniques can be employed for moving camera\nand static camera conditions. If the camera moves, change\ndetection aims at recognizing coherent and incoherent moving\nareas. The former correspond to background areas, the latter\nto video objects. If the camera is static, the goal of change\ndetection is to recognize moving objects (foreground) and the\nstatic background. The semantic analysis we use addresses\nthe static camera problem and is applicable in the case of a\nmoving camera after global motion compensation. The change\ndetector decides whether in each pixel position the foreground\nsignal corresponding to an object is present. This decision\nis taken by thresholding the frame difference between the\ncurrent frame and a frame representing the background. The\nframe representing the background is dynamically generated\nbased on temporal information [17]. The thresholding aims\nat discarding the effect of the camera noise after frame\ndifferencing. A locally adaptive threshold, τ (i, j), is used that\nmodels the noise statistics and applies a signiﬁcance test. To\nthis end, we want to determine the probability that frame\ndifference at a given position (i, j) is due to noise, and not to\nother causes. Let us suppose that there is no moving object in\nthe frame difference. We refer to this hypothesis as the null\nhypothesis, H0. Let g(i, j) be the sum of the absolute values\nof the frame difference in an observation window of q pixels\naround (i, j). Moreover, let us assume that the camera noise is\nadditive and follows a Gaussian distribution with variance σ.\nGiven H0, the conditional pdf of the frame difference follows\na χ2\nq distribution with q degrees of freedom deﬁned by\nf\nH0\ng(i, j)\n|\n(1)\n\nwhere Γ(\n) is the Gamma function, that can be evaluated as\n·\nΓ(x+ 1) = xΓ(x), and Γ(1/2) = √π. To obtain a good trade-\noff between robustness to noise and accuracy in the detection\n5 window centered in (i, j)). It is now\nwe choose q = 25 (5\npossible to derive the signiﬁcance test as\ng(i, j)\n{\nH0}\nτ (i, j)\n|\n≥\nΓ(q/2, g(i, j)2/2σ2)\nΓ(q/2)\n(2)\nWhen this probability is smaller than a certain signiﬁcance\nlevel, α, we consider that H0 is not satisﬁed at the pixel\nposition (i, j). Therefore we label that pixel as belonging to a\nmoving object. The signiﬁcance level α is a stable parameter\nthat does not need manual tuning along a sequence or for\ndifferent sequences. Experimental results indicate that valid\nvalues fall in the range from 10−2 to 10−6.\nThe change detection process produces the segmentation of\nthe moving objects from the background (Figure 2) and, cou-\npled with video object tracking [18], enables the subsequent\nextraction of object metadata, as described in the following\nsection.\nB. Semantic encoding\nThe decomposition computed with semantic video analysis\nis used with an object-based encoder as well as with a\ntraditional frame-based encoder. We will refer to the former\ncase as object-based encoding and to the latter as frame-\nbased encoding. Furthermore, metadata are used to efﬁciently\nencode relevant information and to enhance relevant part of a\nlow-quality coded video. These approaches are referred to as\nmetadata-based encoding and metadata-enhanced encoding,\nrespectively. Relevant examples of the modalities presented\nin this section are illustrated in Figure 3. The analysis and\nevaluation of the different approaches in terms of results and\nbandwidth requirements are presented in Section V.\n1) Object-based encoding: With object-based encoding, the\nencoder needs to support the coding of individual video objects\n(e.g., MPEG–4 object-based). Each video object is assigned to\na distinct object class, according to its importance in the scene.\nThe encoding quality can be set depending on the object class:\nthe higher the relevance, the higher the encoding quality. One\nadvantage of this approach is the possibility of controlling\nthe sequencing of objects: video objects may be encoded\nwith different degree of compression, thus allowing better\ngranularity for the areas in the video that are of more interest to\nthe viewer. Moreover, objects may be decoded in their order\nof priority, and the relevant content can be viewed without\nhaving to reconstruct the entire image (network limitations).\nAnother advantage is the possibility of using a simpliﬁed\nbackground (appliance limitation), so as to enhance the rel-\nevant objects. Using a simpliﬁed background aims at taking\nadvantage of the task-oriented behavior of the human visual\nsystem for improving compression ratios. Recent work on\nfoveation [19] demonstrated that using nonlinear integration\nof low-level visual cues mimicking the processing in primate\noccipital and posterior parietal cortex allows one to sensibly\nincrease compression ratios. Moreover, the work reported in\n[20] demonstrated that an overall increase in image quality\ncan be obtained when the increase in quality of the relevant\nareas of an image more than compensates for the decrease in\nquality of the image background.\n2) Semantic frame-based encoding: The semantic frame-\nbased encoding mode exploits semantics in a traditional frame-\nbased encoding framework (e.g., MPEG–1). The use of the\ndecomposition of the scene into meaningful objects prior\nto encoding, referred here as semantic pre-ﬁltering, helps\nsupport low bandwidth transmission. The areas belonging to\nthe foreground class, or semantic objects, are used as region\nof interest. The areas not included in the region of interest\nmay either be eliminated, that is set to a constant value, or\nlowered in importance by using a low-pass ﬁlter. The latter\nsolution simpliﬁes the information in the background, while\nstill retaining essential contextual information. An example\nof this solution is reported in Fig. 4 (a). On the other hand,\nﬁltering the entire image inhibits the main content message\nFig. 4 (b). Another way to take into account less relevant\nportions of an image before coding is to take advantage of the\nspeciﬁcs of the coding algorithm. In the case of block-based\ncoding, each background macroblock can be replaced by its\nDC value. Semantic frame-based encoding mimics the way\nhumans perceive visual information and allows for a reduction\nof information to be coded.\n3) Metadata-based and metadata-enhanced encoding: A\nfurther processing of the video content is performed to cope\nwith limited device or network capabilities as well as to\nautomatically generate metadata. Such processing transforms\nthe foreground objects extracted through semantic analysis\ninto quantitative descriptors and enables video annotation.\nVideo annotation is desirable for applications such as video\nsurveillance, where terabytes of data are produced and need\nto be searched quickly. Moreover, the descriptors can be\ntransmitted instead of the video content itself and superim-\nposed by the terminal on a still background. For example, an\nobject identiﬁer and a shape descriptor are used in [21]. The\nobject identiﬁer is a unique numerical identiﬁer describing\nthe spatial location of each object in the scene. The shape\ndescriptor is used to represent the shape of an object, ranging\nfrom a bounding box to a polygonal representation with a\ndifferent number of vertices (Figure 3(c)). This approach is\nuseful to preserve privacy in video surveillance applications\nas well as to reduce bandwidth requirements under critical\nnetwork conditions. A progressive representation is used where\nthe number of vertices corresponding to the best resolution\nis computed, and any number of vertices smaller that this\nmaximum can be used according to the requirements of the\napplication. In addition to the above, other features such as\ncolor and texture descriptors may be added in the description\nprocess. The choice of these additional features depends on\nthe application at hand.\nIn addition to the above, the descriptors can be transmitted\nalong with the video itself and used for rendering the video\ncontent. This solution, consisting in a mixture of video-based\nand text-based modalities, is here referred toe as metadata-\nenhanced encoding. Using metadata-enhanced encoding, con-\ntent descriptors help enhance parts of the video that are hidden\nor difﬁcult to perceive due to heavy compression. In this case,\nthe video itself is the background and the descriptors highlight\nrelevant portions of the data. One example is the ball in a\nfootball match for transmission to a PDA or a mobile phone,\nas shown in Figure 3(e).\nis a difﬁcult\nPerceptual video quality assessment\ntask\nalready when dealing with traditional coders [22]. When\ndealing with object-based coders,\nthe task becomes even\nmore challenging. For this reason, we use a combination of\nsubjective and objective evaluation techniques to compare the\nperformance of the different encoding modalities. Subjective\nevaluation includes the visual comparison of frames and frame\ndetails. This analysis is performed at different bitrates and\nat different frame resolutions. Objective evaluation includes\ntemporal signal-to-noise ratio analysis and the analysis of rate-\ndistortion curves.\nA. Semantic peak signal-to-noise ratio\nTraditional peak signal-to-noise ratio (PSNR) analysis uni-\nformly weights the contribution of each pixel in an image\nwhen computing the mean squared error (MSE). This analysis\ngives the same importance to relevant as well as less relevant\nareas of an image. To account for the way humans perceive\nvisual\ninformation, different areas of an image, or object\nclasses, should be considered [11]. We take into account\nobject classes through a distortion measure, the semantic mean\nsquared error, SMSE, deﬁned as:\nwk ·\n(3)\nP\nXk=1\nwhere N is the number of object classes and wk the weight of\nclass k. Class weights are chosen depending on the semantics,\nN\ni=1 wk = 1. The mean\nwith wk ≥\nsquared error of each class, MSEk, can be written as\nk = 1, . . . , N and\n∀\n|\nCk|\n|\nCk| X(i,j)∈Ck\nwhere Ck is the set of pixels belonging to the object class k\nand\nis its cardinality. The class membership of each pixel\n(i, j) is deﬁned by semantic video analysis. The error d(i, j)\nbetween the original image IO and the distorted image ID in\nEq.(4) is the pixel-wise color distance. The color distance is\ncomputed in the 1976 CIE Lab color space in order to consider\nperceptually uniform color distances with the Euclidean norm\nand is expressed as:\nevaluation metric,\nSPSNR, is the following:\nthe semantic peak signal-to-noise ratio,\nSPSNR = 10 log10\nV 2\nM\nSMSE !\nwhere VM is the maximum peak-to-peak value of the color\nrange. When the object classes are foreground and back-\nground, then N = 2 in Eq (3). If we denote with wf the\nforeground weight, then SPSNR\nPSNR when wf = 0.5.\nThe larger wf , the more important the contribution of the\nforeground. When wf = 1,\nthen the foreground only is\nconsidered in the evaluation of the peak signal-to-noise ratio.\nAn illustration of the impact of wf in the distortion measure\nis shown in in Fig. 5. The ﬁgure presents a comparison of the\nIllustration of the impact of wf in the distortion measure: average\nFig. 5.\nSPSNR vs. foreground weight for Hall monitor sequence. The ﬁve labels\ncorrespond to the following sequence types: (1) coded original; (2) tempo-\nrally down-sampled; (3) spatially down-sampled; (4) static background; (5)\nsimpliﬁed background. Content-blind coding methods (1)-(3) decrease their\nperformance when the foreground is given more importance. Methods based\non semantic, (4) and (5), increase their performance when the foreground is\ngiven more importance\naverage SPSNR of the sequence Hall Monitor for the different\nencoding modalities described in Section III-B as function of\nwf . The value of wf to be used is estimated as described in\nthe next section.\nB. Determination of the foreground weight\nSubjective performance evaluation experiments have been\nperformed to estimate the foreground weight leading to the\nclosest match between SPSNR prediction and human judg-\nment. Twenty non-expert observers of different ages and\nbackgrounds have been presented a series of video sequences\naccording to ITU-T Recommendation P.910, Absolute Cate-\ngory Rating [23]. The evaluation has been carried out using\nthe MPEG–4 test sequences Akiyo, Hall Monitor, Children,\nand Coastguard. Video sequences have been generated using\nthe encoding strategies described in Section III-B, at different\nbitrates, and rated by the observers on a scale ranging from 0\n(bad) to 100 (excellent). This range of values was presented\nto the observers in a training phase.\nThe foreground weight, wf , is determined for each test\nsequence by maximizing the Pearson correlation [24] between\nSPSNR and subjective results. The results are summarized in\nTable I. For the sequence Akiyo, where the foreground covers\na large area of each frame and the background is simple, the\nthe observers focused mostly on foreground, thus leading to a\nvalue of wf = 0.97. For Hall Monitor, whose background is\nmore complex and objects are smaller, the foreground attracted\nslightly more the attention than the background (wf = 0.55).\nThe sequence Children has a very complex and colored back-\nground that attracted the observers’ attention, thus resulting\nin foreground and background being equally weighted (wf =\n0.5). The sequence Coastguard contains camera motion that\nprevented the observer from focusing on background steadily,\neven though the background is quite complex. In this case, the\nresulting foreground weight is wf = 0.7. In general, results\nconﬁrm that large moving objects and complex background\ntend to attract user’s attention. Based on the data collected with\nsubjective experiments, it is possible to predict the foreground\nweight based on the following formula:\nand\nCb|\n|\nCb|\n|\nCf |\n/(\n|\nwhere r represents the portion of the image occupied by\nforeground pixels, expressed as r =\n), with\nCf |\n|\nrepresenting the number of foreground and\nCf |\n|\nbackground pixels, respectively. The background complexity\nis taken into account with σb, the standard deviation of the\nluminance of background pixels. The presence of camera\nmotion is considered with the term v: v = 1 for moving\ncamera, and v = 0 otherwise. α, β, γ, and δ are constants\nwhose values are determined based on the results of the\nsubjective experiments and are the following: α = 5.7, β =\n0.108, γ = 0.2 and δ = 0.01. The ﬁnal value of wf is the\naverage of the foreground weights over the sequence.\nIn addition to semantic weight, Table I provides information\nabout accuracy, monotonicity and consistency of the SPSNR\nmetric. Accuracy is given by Pearson linear correlation coefﬁ-\ncient rp, monotonicity by Spearman rank-order correlation co-\nefﬁcient rs, and consistency by outliers ratio ro [24]. Pearson\ncorrelation of PSNR, rp(0.5), is given for comparison. Pearson\ncorrelation rp and Spearman correlation rs are close to 1 for\nall sequences. Thus, accuracy and monotonicity of SPSNR are\nhigh. Outliers ratio ro is around 10%, thus consistency of the\nmetric is good as well. Note that using semantics improves\naccuracy by up to 8% (Akiyo), as compared to PSNR.\nV. EXPERIMENTAL RESULTS\nIn this section, experimental results of the proposed seman-\ntic video encoding and annotation framework with standard\ntest sequences are presented. The results illustrate the impact\nof semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\nlocators of the foreground objects,\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.",
         null
        ],
        [
         "13",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "Trends in Drug Resistance, Serotypes, and Molecular Types of Streptococcus pneumoniae Colonizing Preschool-Age Children Attending Day Care Centers in Lisbon, Portugal: a Summary of 4 Years of Annual Surveillance",
         "ABSTRACT Of the nasopharyngeal cultures recovered from 942 day care center (DCC) attendees in Lisbon, Portugal, 591 (62%) yielded Streptococcus pneumoniae during a surveillance performed in February and March of 1999. Forty percent of the isolates were resistant to one or more antimicrobial agents. In particular, 2% were penicillin resistant and 20% had intermediate penicillin resistance. Multidrug resistance to macrolides, lincosamides, and tetracycline was the most frequent antibiotype (17% of all isolates). Serotyping and molecular typing by pulsed-field gel electrophoresis were performed for 202 out of 237 drug-resistant pneumococci (DRPn). The most frequent serotypes were 6B (26%), 14 (22%), 19F (16%), 23F (10%), and nontypeable (12%). The majority (67%) of the DRPn strains were representatives of nine international clones included in the Pneumococcal Molecular Epidemiology Network; eight of them had been detected in previous studies. Fourteen novel clones were identified, corresponding to 26% of the DRPn strains. The remaining 7% of the strains were local clones detected in our previous studies. Comparison with studies conducted since 1996 in Portuguese DCCs identified several trends: (i) the rate of DRPn frequency has fluctuated between 40 and 50%; (ii) the serotypes most frequently recovered have remained the same; (iii) nontypeable strains appear to be increasing in frequency; and (iv) a clone of serotype 33F emerged in 1999. Together, our observations highlight that the nasopharynxes of children in DCCs are a melting pot of successful DRPn clones that are important to study and monitor if we aim to gain a better understanding on the epidemiology of this pathogen.",
         "",
         "https://europepmc.org/articles/pmc1081291?pdf=render",
         null,
         null,
         null,
         null,
         null,
         null,
         null
        ],
        [
         "14",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "Laboratory Exposures to Brucellae and Implications for Bioterrorism",
         "Laboratory exposures to brucellae indicate unfamiliarity with organism.",
         "Laboratory-acquired Brucellosis Acute hepatitis due to brucellosis in a laboratory technician. Prevention of Laboratory-Acquired Brucellosis Analysis of risk factors for laboratory-acquired brucella infections. Use of the Isolator 1.5 Microbial Tube for Detection of Brucella melitensis in Synovial Fluid Bacterial Pathogens as Biological Weapons and Agents of Bioterrorism Unusual Complication of Breast Implants: Brucella Infection Brucellosis in laboratory workers at a Saudi Arabian hospital. Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases Brucella abortus Infection Acquired in Microbiology Laboratories Posttreatment Follow-Up of Brucellosis by PCR Assay Detection of Brucellae in Blood Cultures Rapid diagnosis of human brucellosis by peripheral-blood PCR assay Failure of short antimicrobial treatments for human brucellosis Comparison of BACTEC 9240 Peds Plus medium and isolator 1.5 microbial tube for detection of Brucella melitensis from blood cultures Exposure to Brucella in the laboratory Brucellosis from sniffing bacteriological cultures Grand Rounds—Hammersmith Hospital: A case of laboratory acquired brucellosis Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases Live Brucella spp. fail to induce tumor necrosis factor alpha excretion upon infection of U937-derived phagocytes Outbreak of Brucella melitensis among microbiology laboratory workers Unsuspected brucellosis diagnosed in a child as a result of an outbreak of laboratory-acquired brucellosis. Brucellosis: imported and laboratory-acquired cases, and an overview of treatment trials. Biochemical mis-identification of Brucella melitensis and subsequent laboratory-acquired infections. Laboratory diagnosis of brucella infection: some pitfalls Outbreak of Brucella melitensis among microbiology laboratory workers in a community hospital. An outbreak of Brucella melitensis infection by airborne transmission among laboratory workers. Laboratory-acquired meningitis caused by Brucella abortus strain 19. Past and present hazards of working with infectious agents. From the Center for Disease Control Brucella canis: A cause of undulant fever An unusual outbreak of brucellosis. Brucellosis: I. Laboratory-Acquired Acute Infection Brucellosis in the United States. Brucella species Brucellosis: An occupational hazard for medical laboratory personnel. Report of five cases Beer-Sheva 84101, Israel; fax: 972-8640-3541 Prevention of laboratory-acquired brucellosis. (revised). Geneva: World Health Organization Basic protocols for level A laboratories for the presumptive identification of Brucella species. Washington: American Society for Microbiology Exposure of hospital personnel to Brucella melitensis and occurrence of laboratory-acquired disease in an endemic area. Biosafety in microbiological and biomedical laboratories Laboratory-associated infections: incidence, fatalities, causes, and prevention. Principles and practice of infectious diseases. Vols 1 and 2. Brucellosis in the United States, 1965-1974. Laboratory Infections Due to Brucella",
         "https://wwwnc.cdc.gov/eid/article/11/8/pdfs/04-1197.pdf",
         "Brucellae are class 3 organisms and potential agents\nof  bioterrorism.  Because  of  effective  public  health  meas-\nures, brucellosis has become a rare disease in industrial-\nized  countries,  and  clinical  microbiology  laboratories  are\nfrequently unfamiliar with the genus. A low index of suspi-\ncion by physicians or failure to notify the laboratory, equiv-\nocal  Gram-stain  results,  misidentification  of  the  organism\nby  commercial  systems,  unsafe  laboratory  practices,  and\nlaboratory accidents have been responsible for numerous\ncases of exposure to the organism and laboratory-acquired\ndisease in recent years. Discovery of a laboratory exposure\nto  brucellae  should  prompt  an  exhaustive  investigation  of\nthe event and its circumstances, definition of the population\nat  risk,  enforcement  of  safe  laboratory  practices,  and\nantimicrobial  drug  prophylaxis  for  exposed  persons.\nInadvertent exposures to brucellae in the clinical laboratory\nindicate  a  widespread  lack  of  preparedness  to  cope  with\neventual biologic threats involving use of the organism.",
         "by contact with infected animals and consumption of\ncontaminated animal products (1,2). Because of compulso-\nry pasteurization of milk products and strict control of the\ndisease  in  dairy  cattle,  the  incidence  of  brucellosis  has\nsteadily  declined  in  most  industrialized  countries  during\nthe last 50 years. However, the disease remains among the\nmost commonly recognized causes of laboratory-transmit-\nted infections; 2% of all brucellosis cases are laboratory-\nacquired (1,3–9).\nThe Organism\nSeveral  biologic  characteristics  make  brucellae  easily\ntransmissible within the close confinement of the clinical\nmicrobiology laboratory, including the facts that the infect-\ning dose for humans is low, and the organism may enter the\nbody in many ways relevant to laboratory practices (e.g.,\nthrough  the  respiratory  mucosa,  conjunctivae,  gastroin-\ntestinal tract, or abraded skin) (1,2). Rare cases of acquisi-\ntion of the organism through organ transplantation, sexual\ncontact,  breastfeeding,  or  the  transplacental  route  have\nalso been reported (1). Because person-to-person transmis-\nsion does not occur, infected persons do not pose a threat\nto their surroundings.\nreproducing\nSoon  after  entry  into  the  body  through  the  skin  or\nmucous  membranes,  brucellae  are  ingested  by  polymor-\nphonuclear and mononuclear phagocytes. The organism is\nable to escape phagocytic killing by inhibiting the phago-\nsome-lysosome\ninside\nmacrophages  (1,10).  After  a  variable  incubation  period\nranging  from  <1  week  to  several  months  (usually  2–4\nweeks),  nonspecific  systemic  symptoms  such  as  fever,\nheadache,  malaise,  night  sweats,  and  arthralgia  follow,\nresembling a flulike disease (1,2). During the early stages\nof  the  disease,  patients  are  frequently  bacteremic.  This\nbacteremia  has  a  continuous  pattern,  making  circulating\nbrucellae  easily  detectable  by  blood  culture.  Once  in  the\nbloodstream, the organism is seeded to multiple organ sys-\ntems and especially to those rich in reticuloendothelial tis-\nsue,  such  as  the  liver,  spleen,  and  the  skeletal  and\nhematopoietic systems, where it may cause localized dis-\nease such as hepatitis or arthritis (1,2).\nBecause  of  the  variable  manifestations  of  human  bru-\ncellosis, a wide array of different clinical specimens may\ncontain viable brucellae, including pus, blood, bone mar-\nrow,  synovial  fluid  and  tissues,  and  more  rarely,  cere-\nbrospinal  fluid,  urine,  and  genital  exudates.  The\nconcentration  of Brucella organisms  in  the  blood  (11,12)\nand synovial fluid (13) of patients with brucellosis is usu-\nally low, and therefore, these clinical specimens probably\npose  a  low  risk  for  contagion  for  laboratory  personnel.\nHowever,  the  danger  of  clinically  relevant  exposure\nincreases exponentially after incubation of both solid and\nliquid media. Seeded media harbor considerable amounts\nof  viable Brucella organisms,  and  routine  bacteriologic\nprocedures  such  as  preparing,  centrifuging,  and  vigorous\nagitation (vortexing) of bacterial suspensions, performing\nsubcultures  and  biochemical  testing,  and  particularly  the\ncatalase test, may create dangerous aerosols and the poten-\ntial for accidental spillage (14).\nMost  cases  of  laboratory-acquired  brucellosis  have\nbeen  caused  by  the  more  virulent  Brucella  melitensis\nspecies  (3,4,6,8,14–29).  B.  suis (4,6),  B.  abortus (3,6,8),\nand B. canis (30) have also been implicated, and transmis-\nsion of the attenuated B. abortus 19 and B. melitensis Rev-\n1 vaccine strains has also been reported (31,32).\nClinical Manifestations of\nLaboratory-acquired Infections\nPatients involved in laboratory outbreaks of brucellosis\nhave shown almost the entire range of clinical manifesta-\ntions of the disease, ranging from the common prolonged\nfebrile  syndrome  (undulant  fever)  (14)  and  a  flulike  dis-\nease (25,27), to focal signs and symptoms, such as hepati-\ntis (17,33), lymphadenopathy (17,25), uveitis (14), breast\nabscess  (28)  epididymitis  (29),  arthritis  (17),  discitis\n(29,34), pneumonitis (17), deep vein thrombosis (29), and\nmeningitis  (31).  Deaths  are  rare  (<5%)  even  among\nuntreated  patients  with  brucellosis,  but  5  fatal  cases\noccurred among the 426 persons with laboratory-acquired\ndisease summarized by Pike in 1978 (5).\nLaboratory Exposure to Brucella Organisms\nBecause  of  effective  control  measures  in  animals  and\nanimal  products,  brucellosis  has  been  almost  eradicated\nfrom  most  industrialized  countries,  where  the  disease  is\nusually limited to persons who have traveled to developing\ncountries  or  ingested  imported  contaminated  food  (14).\nHuman brucellosis has become so rare in the United States\nthat <300 cases have been reported annually in the last 4\ndecades (9,15) and ≈100 per year in the last 10 years (2).\nBecause of the rare occurrence of brucellosis, technicians\nworking in industrialized countries have become unfamil-\niar with the staining and other phenotypic characteristics of\nthe  organism.  In  addition,  physicians  frequently  do  not\nconsider  the  diagnosis  of  brucellosis  or  fail  to  communi-\ncate their suspicion to the clinical microbiology laborato-\nry, which results in inadvertent handling of cultures on an\nopen bench.\nAlthough  the  identification  of  the  genus  Brucella is\nstraightforward  (small  gram-negative  coccobacilli;  posi-\ntive oxidase, catalase, and urease test results; no sugar fer-\nmentation;  and  requirement  of  aerobic  conditions  and\nadded CO2 for its growth), laboratory-acquired disease has\nfrequently resulted from misidentification of the organism\n(35). In 2 published outbreaks, the Brucella isolate resisted\ndecoloration  and  appeared  as  a  gram-positive  or  gram-\nvariable  coccobacillus  and  thus  was  misidentified  as\nmicrococcus  or  a  coryneform  bacillus  (14,29).  In  other\ncases,  B.  melitensis organisms  tested  with  the  API20NE\n(bioMérieux,  Marcy  l’Etoile,  France)  identification  kit\nproduced  the  1200004  or  1000004  biochemical  profile\nleading  to  misidentification  of  the  isolate  as  Moraxella\nphenylpyruvica at  a  good  identification  level  (90.5%)\n(8,18–21). The  frequent  failure  of  clinical  laboratories  to\ncorrectly identify isolates as brucellae is particularly wor-\nrisome because these organisms are regarded as potential\nagents for bioterrorism. Brucellae are inexpensive to pro-\nduce and disperse, and transmission to humans may result\nin prolonged illness and long-term sequelae. These organ-\nisms  are  considered  category  B  select  agents  (2). The\noccurrence in the last few years of inadvertent exposures to\nbrucellae  in  laboratories  in  industrialized  countries\n(7,8,14,17–26)  indicates  a  lack  of  preparedness  to  deal\nwith a real biologic threat. Lack of recognition of an iso-\nlate as a Brucella sp. by a laboratory may enable a bioter-\nrorism-related  attack  to  go  undetected,  whereas  a\nfalse-positive identification may cause unnecessary alarm.\nIn laboratories serving countries in which human bru-\ncellosis is still endemic, the degree of potential exposure to\nthe  organism  may  be  extremely  high.  In  a  microbiology\nlaboratory in Ankara, Turkey, the average annual number\nof cultures positive for Brucella spp. reaches 400 (34). A\nlaboratory serving a brucellosis-endemic area in southern\nIsrael processes ≈150 per year; at the peak of the Brucella\nseason  (April–June),  10%  of  all  positive  blood  cultures\ngrow B. melitensis (27). The risk for exposure in develop-\ning  countries  is  frequently  aggravated  by  lack  of  safety\nequipment and inadequate laboratory facilities (34). In the\naforementioned Turkish laboratory, the disease affected 10\n(18%) of 55 laboratory workers, and the calculated hazard\nwas 8% per employee-year (34).\nthe\nThe probable source of the infection is usually apparent\nwhen disease occurs in laboratories where isolation of bru-\ncellae  is  rare  (21).  In  laboratories  serving  brucellosis-\ntime  and  circumstances  of\nendemic  areas,\ncontamination  are  more  difficult  to  trace  because  of  the\nexistence  of  multiple  potential  sources  (22,27). Although\nthe exact form of transmission often remains speculative,\naerosols have been implicated in most cases (24,27,29,32).\nThis  assumption  has  been  strengthened  by  spread  of  the\ndisease  to  distant  areas  through  common  ventilation\n(27,32).\nLaboratory  accidents  such  as  breakage  of  centrifuge\ntubes (7) or blood-culture vials (16) play a minor role in\nlaboratory-acquired  disease  and  are  responsible  for  only\n20% of cases (6). More commonly, exposures are the result\nof  unsafe  laboratory  practices,  such  as  sniffing  plates\n(8,24,26,29,34);  working  on  an  open  bench  with  viable\norganisms  (14,16,17,20,22);  not  using  protective  equip-\nment such as gloves, masks, and goggles (34); or ingesting\nsuspensions of living brucellae during mouth pipetting (3).\nPERSPECTIVE\nHowever, even when no apparent breach in safety proce-\ndures  is  apparent,  transmission  may  occur  (17).  Rare\nevents—such as self-inoculation of brucellae by syringes\nloaded with a suspension of the organism or with synovial\nfluid from an infected patient (3,16), injury to the conjunc-\ntiva with a broken tube that had contained a living culture\n(3), or participation in a laboratory exercise in which still-\nviable  organisms  were  inadvertently  used  (15)—are  also\nresponsible  for  a  small  number  of  laboratory  exposures\nand outbreaks of disease.\nMost  cases  of  laboratory-acquired  brucellosis  have\noccurred  in  clinical  laboratories,  but  transmission  of  the\ndisease  in  research  facilities  (4,25)  and  laboratories  that\nmanufactured Brucella vaccines has also been document-\ned  (21,22).  A laboratory  technician  at  the  Centers  for\nDisease  Control  and  Prevention  (CDC)  became  infected\nwhile working in a safety cabinet with a B. melitensis iso-\nlate that had originally caused an outbreak of laboratory-\nacquired  disease  in  a  community  hospital  (17).  In  a\nseparate episode, the organism isolated from a microbiolo-\ngy technologist with laboratory-acquired disease infected a\nlaboratory worker at the hospital to which the technologist\nhad been admitted (14).\nThe attack rate of laboratory-associated infections has\nranged  from  30%  to  100%  depending,  among  other  fac-\ntors, on the location of workers, whether aerosol-generat-\ning  procedures  have  been  performed,  and\nthe\nconcentration  of  microorganisms  in  the  contaminated\nmedia  (7,8).  However,  the  hazard  of  transmission  is  not\nlimited to persons who worked with the isolate. Among the\n74 reported cases of laboratory-acquired brucellosis from\n1897  to  1939  reviewed  by  Meyer  and  Eddie,  the  disease\nalso affected janitors and occasional visitors (3). In a large\noutbreak of laboratory-acquired brucellosis, 3 of 7 cases of\ndisease  occurred  among  persons  who  briefly  visited  the\nfacility but did not enter the room where Brucella cultures\nwere processed (27).\nInvestigating Outbreaks of Brucellosis\nRecognizing sporadic cases and even outbreaks of lab-\noratory-acquired brucellosis is not always easy because the\ndisease lacks distinctive clinical features (14,17) and has a\nwidely variable incubation period (7,17,19,32). In a cluster\nof disease that originated in a single exposure, the onset of\nsymptoms of the 7 affected laboratory workers spanned 5\nmonths (17). Although a temporal clustering of cases usu-\nally  suggests  a  large  exposure  from  a  pinpoint  source\n(7,32),  this  is  not  always  the  case.  When  biotyping  was\nperformed in the isolates recovered within a 5-week peri-\nod  from  7  infected  hospital  workers  in  southern  Israel,  3\ndistinct  strains  were  recognized,  demonstrating  that  the\noutbreak was caused by at least 3 separate exposures (27).\nGuidelines for Investigating Outbreaks\nIn-depth  investigation  of  laboratory-acquired  cases  of\nbrucellosis may lead to the identification of unsafe bacte-\nriologic  practices,  suitable  for  correction  by  educational\nand technical measures. To guide the investigation, the fol-\nlowing  recommendations  are  made.  1)  Send  isolates  to  a\nreference  laboratory  for  confirmation.  Organisms  should\nbe shipped as “dangerous goods,” according to the guide-\nlines established by the World Health Organization and the\nOffice  of  Biosafety  at  CDC  (36).  In  the  United  States,\norganisms presumptively identified as brucellae should be\nsent to CDC or another public health laboratory following\nthe specific guidelines for transferring “select agents.” 2)\nInform infection control services and public health author-\nities,  who  may  choose  to  involve  CDC.  3)  Conduct  a\nmeticulous epidemiologic investigation. 4) Determine the\ndate  and  circumstances  of  the  exposure  (17).  5)  Exclude\nother  potential  sources  of  transmission,  such  as  previous\nlaboratory exposures, travels to brucellosis-endemic areas,\nconsumption  of  unpasteurized  dairy  products,  and  han-\ndling  of  farm  or  laboratory  animals  (14,15,17,27,29).  6)\nKeep all relevant data and records. 7) Define the exposed\npopulation. 8) Determine the level of risk on the basis of\ntype of laboratory procedures performed, proximity to the\nsource,  duration  of  the  exposure,  and  the  like,  to  define\npersons  to  whom  postexposure  prophylaxis  should  be\noffered (8,32). 9) Check biologic safety cabinets (27,29).\n10) Check the ventilation system (27). 11) Collect baseline\nserum  samples  from  all  known  potentially  exposed  per-\nsons  (17,27).  12)  Freeze  isolates  for  future  typing,  espe-\ncially in brucellosis-endemic areas (23,27).\nPrevention of Laboratory-acquired Cases\nBrucellae  are  considered  class  3  organisms.  CDC  has\nstrongly recommended that live Brucella cultures and sus-\npicious  organisms  be  manipulated  in  a  class  II  biologic\nsafety  cabinet  (37).  This  recommendation,  however,  is\nclearly insufficient for preventing laboratory-acquired dis-\nease because by the time the organism is suspected or con-\nfirmed as Brucella, exposure of laboratory personnel may\nhave occurred (27). On the other hand, converting a large\nand busy clinical microbiology laboratory into a biosafety\nlevel III facility, where all specimens are handled in biolog-\nic safety cabinets (32), is both impractical and unnecessary,\nespecially in laboratories in areas where the disease is not\nendemic. However, in regions where brucellosis is highly\nprevalent, an enhanced safety policy should be adopted. All\nblood  culture  vials  detected  as  positive  by  the  automated\nblood culture system, as well as all bone marrow and syn-\novial  fluid  specimens,  should  be  manipulated  in  biologic\nsafety hoods until the isolated microorganisms are defini-\ntively determined to be other than Brucella sp. (27).\nFor  laboratories  serving  areas  where  brucellosis  is\nuncommon, the following recommendations are made. 1)\nBecause of the low incidence of brucellosis, physicians in\nareas not endemic for the disease are unfamiliar with the\nclinical and epidemiologic features of human brucellosis,\nand the possibility of brucellosis is rarely considered in the\ndifferential diagnosis. Periodic education of physicians on\nthis  subject  is  indicated.  2)  Communication  between\nattending  physicians  and  the  laboratory  should  be\nimproved. The clinical microbiology laboratory should be\ninformed  in  advance  when  clinical  specimens  had  been\nobtained  from  patients  with  risk  factors  for  brucellosis,\nsuch  as  recent  history  of  travel  to  brucellosis-endemic\nareas,  consumption  of  local  or  imported  unpasteurized\ndairy products, or professional exposures in veterinarians,\nshepherds,  slaughterhouse  employees,  and  laboratory\nworkers. 3) The use of automated, continuous monitoring\nblood culture systems for patients with suspected brucel-\nlosis  should  be  preferred  over  the  lysis-centrifugation\nmethod  because  the  latter  involves  centrifuging  clinical\nspecimens  and  visually  inspecting  plates  to  detect  the\norganism and probably increases the risk for transmission\n(11,27). Although the use of blood lysis-based methods has\nbeen  advocated  in  the  past  for  improving  detection  of\nBrucella bacteremia, modern automated blood culture sys-\ntems are faster and more sensitive (11,12). 4) The familiar-\nity of laboratory technicians with the characteristics of the\norganism,  as  well  as  with  the  safe  handling  of  cultures,\nshould be improved and maintained through periodic edu-\ncation.  The  reader  is  referred  to  the  excellent  review  by\nGilligan  and York  for  handling  and  identifying  presump-\ntive Brucella organisms (35). 5) Standard precautions and\nstrict adherence to good laboratory practices must be com-\npletely  adopted,  reinforced,  and  regularly  monitored.  6)\nAll work with gram-negative or gram-variable small rods\nor coccobacilli isolated from tissues, blood, bone marrow,\nbone, or synovial fluid exudates should be carried out in a\nbiosafety  cabinet  until  Brucella has  been  ruled  out.  7)\nPlates  should  be  sealed  for  safety  when  not  in  use  and\nappropriately disposed and sterilized as soon as they are no\nlonger being actively used (17). 8) Because brucellae are\nrelatively slow-growing bacteria, cultures for the organism\nhave been traditionally kept for several weeks. However,\nmodern  blood  culture  systems  enable  brucellae  to  be\ndetected within the routine 5-day incubation period insti-\ntuted  in  most  clinical  laboratories  (12). Therefore,  safety\nprecautions should not be limited to organisms that tend to\ngrow slowly (8,18,20,26). 9) Antimicrobial drug–suscepti-\nbility  testing  of  Brucella  organisms  is  not  indicated\nbecause the therapeutic regimen for brucellosis is standard,\nand  the  organism  does  not  usually  acquire  antimicrobial\nresistance.  Performance  of  this  and  other  unnecessary\ntests,  and  especially  of  laboratory  procedures  known  to\nproduce aerosols, should be strongly discouraged (27). 10)\nIf a suspension of living brucellae is spilled and the organ-\nism is recognized, the entire laboratory should be immedi-\nately  evacuated,  doors  should  be  shut,  and  an  effective\ngermicide  such  as  3%  phenol  or  10%  bleach  should  be\napplied  by  a  trained  person  wearing  a  safety  mask,  gog-\ngles, an impermeable laboratory gown, and gloves (7).\nPostexposure Prophylaxis\nBecause  of  ethical  considerations,  the  heterogeneous\nnature of the events leading to Brucella exposure, difficul-\nties in determining the actual risk for individual workers,\nlate recognition of outbreaks, and the small number of per-\nsons involved in each outbreak, no controlled studies have\nbeen performed to assess the value of administering post-\nexposure  prophylaxis  to  persons  at  risk.  However,  anec-\ndotal  evidence  suggests  that  administering  prophylactic\nantimicrobial drug therapy may reduce the risk of develop-\ning clinical disease (7,8,26).\nIn a recently reported event, an isolate from a chest wall\nexudate  culture  from  an  Indian  patient,  which  was  origi-\nnally identified as M. phenylpyruvica, was correctly recog-\nnized  as  B.  melitensis 22  days  after  the  specimen  was\nobtained (8). By that time, 26 laboratory workers had been\npotentially  exposed,  and  6  had  actually  manipulated  the\norganism. These 6 workers were considered to be at high\nrisk and offered a 3-week prophylactic course of combined\ndoxycycline-rifampin  or  trimethoprim-sulfamethoxazole\ntherapy.  None  of  the  5  laboratory  technologists  who\nreceived  postexposure  antimicrobial  drugs  became  ill  or\ndeveloped an antibody response. However, Brucella bac-\nteremia  developed  in  the  only  laboratory  worker  who\nrefused therapy, and she seroconverted 10 weeks after the\nspecimen  was  received  (8).  Neither  clinical  disease  nor\nseroconversion developed in the remaining 19 laboratory\nworkers who were only present in the laboratory but were\nnot in direct contact with the organism (8).\nIn  another  event,  3  laboratory  technologists  who  had\ninadvertently  worked  with  a  B.  melitensis isolate  on  an\nopen bench and sniffed the plates were given 1 week of pro-\nphylactic  doxycycline  within  24  hours  of  the  exposure;\nnone of them became ill or seroconverted (26). In a large\nexposure caused by breakage of a tube containing B. abor-\ntus, patients who started therapy immediately after serocon-\nversion was detected and before symptoms developed (10\nweeks after the laboratory accident) had a benign clinical\ncourse, suggesting that even if the disease was not prevent-\ned, a certain degree of attenuation probably occurred (7).\nBecause  of  the  high  attack  rate  of  brucellosis  among\nexposed  workers,  the  unpredictable  and  often  chronic\ncourse of the disease, and the difficulties in eradicating the\norganism  once  a  symptomatic  infection  has  been  estab-\nlished, postexposure prophylaxis is probably indicated for\nall  persons  after  an  obvious  exposure  to  living  brucellae\n(26). Antimicrobial  drug  prophylaxis  with  a  combination\nof oral doxycycline 100 mg twice a day plus rifampin 600\nmg 4 × per day for 3 weeks should be started as soon as the\nexposure  to  confirmed  Brucella organisms  is  recognized\n(8). For pregnant women, administration of trimethoprim-\nsulfamethoxazole 160/800 mg 2 × per day for 3 weeks has\nbeen advocated (8).\nPostexposure Follow-up\nWhether  exposed  persons  have  received  prophylactic\ntherapy or not, increased surveillance for clinical signs of\ndisease should be conducted for at least 6 months. In addi-\ntion,  exposed  laboratory  workers  should  be  followed  for\npossible subclinical infections or early signs of disease by\nperiodic serologic testing. Weekly or semiweekly serolog-\nic surveillance is recommended for the first 3 months and\nonce  a  month  thereafter  for  3  to  9  additional  months  to\ndetect late infections resulting from prolonged incubation\n(7,8).\nHuman brucellosis has variable clinical manifestations\nand may mimic other infections, particularly influenza, as\nwell as noninfectious conditions. Therefore, the diagnosis\nof  the  disease  requires  a  high  index  of  suspicion.\nEducation  of  exposed  personnel  on  the  symptoms  of  the\ndisease and the need for periodic and timely serologic fol-\nlow-up are particularly important. Increased vigilance dur-\ning  the  flu  season  employing  a  broad  case  definition  is\nclearly  needed,  and  administration  of  influenza  vaccine\nshould be strongly recommended to all Brucella-exposed\npersons. If symptoms develop, blood cultures and cultures\nof other normally sterile body fluids, as clinically indicat-\ned, should be obtained, and a new antibody titer should be\ndetermined.  Although  use  of  nucleic  acid  amplification\nmethods  has  been  shown  to  enable  early  detection  of\ninfected persons (38), these tests are not yet commercially\navailable.\nTreating Infected Persons and\nPosttreatment Follow-up\nPersons  in  whom  culture-confirmed  or  serologically\nproven disease develops should receive therapy with oral\ndoxycycline and rifampin for 6 weeks, or a combination of\noral doxycycline for 6 weeks and an intramuscular amino-\nglycoside  (gentamicin  or  streptomycin)  for  the  first  2\nweeks (2,7). Use of 3 drugs is usually reserved for compli-\ncated cases or life-threatening clinical manifestations, such\nas endocarditis or meningitis (1). Pregnant women should\nreceive  trimethoprim-sulfamethoxazole  160/800  mg  2  ×\nper day for 6 weeks (1).\nEven  when  patients  are  appropriately  treated,  the  risk\nfor  relapse  remains  high  (≈20%)  (39,40).  Therefore,\npatients who have completed a full therapeutic antimicro-\nbial course should be followed clinically and serologically\nfor  1  year.  Failure  to  show  declining  antibody  titers  may\nindicate incomplete cure. If symptoms consistent with bru-\ncellosis develop, blood cultures and serologic tests should\nbe performed to detect relapses of the disease (39). Use of\nnucleic acid amplification methods in the future may fur-\nther improve the detection of patients in whom the organ-\nism was not eliminated (40).\nAlthough human brucellosis has been eradicated from\nmost  industrialized  countries,  isolated  cases  of  disease,\nusually  related  to  travel  or  import  of  contaminated  food\nfrom disease-endemic areas, continue to occur. Because of\nthe  low  incidence  of  disease  in  industrialized  countries,\nclinical  laboratory  technologists  have  become  unfamiliar\nwith  identifying  and  handling  Brucella species.  Unsafe\nlaboratory  practices  while  manipulating  Brucella isolates\nhave resulted in inadvertent exposure to the organism and\nmany cases of laboratory-acquired disease, indicating lack\nof preparedness to cope with bioterrorism threats involv-\ning  brucellae.  Education  of  laboratory  personnel  on  the\nidentification  of  Brucella  species,  adherence  to  and\nenforcement  of  standard  precautions,  thorough  investiga-\ntion of laboratory exposures, administration of prophylac-\ntic  antimicrobial  drug  therapy,  and  close  follow-up  of\ndirectly exposed persons are strongly recommended.\nDr.  Yagupsky  is  the  director  of  the  Clinical  Microbiology\nLaboratory  of  the  Soroka  University  Medical  Center,  Beer-\nSheva, Israel. His main research areas are Kingella kingae infec-\ntions and diagnosis of human brucellosis.\nDr.  Baron  is  director  of  the  Clinical  Microbiology\nLaboratory at Stanford University Medical Center and professor\nof pathology at Stanford University Medical College. Her inter-\nests include the development of rapid infectious disease diagnos-\ntic  tests  and  educating  both  scientists  and  the  public  about\nrelevant aspects of infectious diseases.",
         "",
         "",
         "",
         "Brucellae are class 3 organisms and potential agents\nof  bioterrorism.  Because  of  effective  public  health  meas-\nures, brucellosis has become a rare disease in industrial-\nized  countries,  and  clinical  microbiology  laboratories  are\nfrequently unfamiliar with the genus. A low index of suspi-\ncion by physicians or failure to notify the laboratory, equiv-\nocal  Gram-stain  results,  misidentification  of  the  organism\nby  commercial  systems,  unsafe  laboratory  practices,  and\nlaboratory accidents have been responsible for numerous\ncases of exposure to the organism and laboratory-acquired\ndisease in recent years. Discovery of a laboratory exposure\nto  brucellae  should  prompt  an  exhaustive  investigation  of\nthe event and its circumstances, definition of the population\nat  risk,  enforcement  of  safe  laboratory  practices,  and\nantimicrobial  drug  prophylaxis  for  exposed  persons.\nInadvertent exposures to brucellae in the clinical laboratory\nindicate  a  widespread  lack  of  preparedness  to  cope  with\neventual biologic threats involving use of the organism.\nby contact with infected animals and consumption of\ncontaminated animal products (1,2). Because of compulso-\nry pasteurization of milk products and strict control of the\ndisease  in  dairy  cattle,  the  incidence  of  brucellosis  has\nsteadily  declined  in  most  industrialized  countries  during\nthe last 50 years. However, the disease remains among the\nmost commonly recognized causes of laboratory-transmit-\nted infections; 2% of all brucellosis cases are laboratory-\nacquired (1,3–9).\nThe Organism\nSeveral  biologic  characteristics  make  brucellae  easily\ntransmissible within the close confinement of the clinical\nmicrobiology laboratory, including the facts that the infect-\ning dose for humans is low, and the organism may enter the\nbody in many ways relevant to laboratory practices (e.g.,\nthrough  the  respiratory  mucosa,  conjunctivae,  gastroin-\ntestinal tract, or abraded skin) (1,2). Rare cases of acquisi-\ntion of the organism through organ transplantation, sexual\ncontact,  breastfeeding,  or  the  transplacental  route  have\nalso been reported (1). Because person-to-person transmis-\nsion does not occur, infected persons do not pose a threat\nto their surroundings.\nreproducing\nSoon  after  entry  into  the  body  through  the  skin  or\nmucous  membranes,  brucellae  are  ingested  by  polymor-\nphonuclear and mononuclear phagocytes. The organism is\nable to escape phagocytic killing by inhibiting the phago-\nsome-lysosome\ninside\nmacrophages  (1,10).  After  a  variable  incubation  period\nranging  from  <1  week  to  several  months  (usually  2–4\nweeks),  nonspecific  systemic  symptoms  such  as  fever,\nheadache,  malaise,  night  sweats,  and  arthralgia  follow,\nresembling a flulike disease (1,2). During the early stages\nof  the  disease,  patients  are  frequently  bacteremic.  This\nbacteremia  has  a  continuous  pattern,  making  circulating\nbrucellae  easily  detectable  by  blood  culture.  Once  in  the\nbloodstream, the organism is seeded to multiple organ sys-\ntems and especially to those rich in reticuloendothelial tis-\nsue,  such  as  the  liver,  spleen,  and  the  skeletal  and\nhematopoietic systems, where it may cause localized dis-\nease such as hepatitis or arthritis (1,2).\nBecause  of  the  variable  manifestations  of  human  bru-\ncellosis, a wide array of different clinical specimens may\ncontain viable brucellae, including pus, blood, bone mar-\nrow,  synovial  fluid  and  tissues,  and  more  rarely,  cere-\nbrospinal  fluid,  urine,  and  genital  exudates.  The\nconcentration  of Brucella organisms  in  the  blood  (11,12)\nand synovial fluid (13) of patients with brucellosis is usu-\nally low, and therefore, these clinical specimens probably\npose  a  low  risk  for  contagion  for  laboratory  personnel.\nHowever,  the  danger  of  clinically  relevant  exposure\nincreases exponentially after incubation of both solid and\nliquid media. Seeded media harbor considerable amounts\nof  viable Brucella organisms,  and  routine  bacteriologic\nprocedures  such  as  preparing,  centrifuging,  and  vigorous\nagitation (vortexing) of bacterial suspensions, performing\nsubcultures  and  biochemical  testing,  and  particularly  the\ncatalase test, may create dangerous aerosols and the poten-\ntial for accidental spillage (14).\nMost  cases  of  laboratory-acquired  brucellosis  have\nbeen  caused  by  the  more  virulent  Brucella  melitensis\nspecies  (3,4,6,8,14–29).  B.  suis (4,6),  B.  abortus (3,6,8),\nand B. canis (30) have also been implicated, and transmis-\nsion of the attenuated B. abortus 19 and B. melitensis Rev-\n1 vaccine strains has also been reported (31,32).\nClinical Manifestations of\nLaboratory-acquired Infections\nPatients involved in laboratory outbreaks of brucellosis\nhave shown almost the entire range of clinical manifesta-\ntions of the disease, ranging from the common prolonged\nfebrile  syndrome  (undulant  fever)  (14)  and  a  flulike  dis-\nease (25,27), to focal signs and symptoms, such as hepati-\ntis (17,33), lymphadenopathy (17,25), uveitis (14), breast\nabscess  (28)  epididymitis  (29),  arthritis  (17),  discitis\n(29,34), pneumonitis (17), deep vein thrombosis (29), and\nmeningitis  (31).  Deaths  are  rare  (<5%)  even  among\nuntreated  patients  with  brucellosis,  but  5  fatal  cases\noccurred among the 426 persons with laboratory-acquired\ndisease summarized by Pike in 1978 (5).\nLaboratory Exposure to Brucella Organisms\nBecause  of  effective  control  measures  in  animals  and\nanimal  products,  brucellosis  has  been  almost  eradicated\nfrom  most  industrialized  countries,  where  the  disease  is\nusually limited to persons who have traveled to developing\ncountries  or  ingested  imported  contaminated  food  (14).\nHuman brucellosis has become so rare in the United States\nthat <300 cases have been reported annually in the last 4\ndecades (9,15) and ≈100 per year in the last 10 years (2).\nBecause of the rare occurrence of brucellosis, technicians\nworking in industrialized countries have become unfamil-\niar with the staining and other phenotypic characteristics of\nthe  organism.  In  addition,  physicians  frequently  do  not\nconsider  the  diagnosis  of  brucellosis  or  fail  to  communi-\ncate their suspicion to the clinical microbiology laborato-\nry, which results in inadvertent handling of cultures on an\nopen bench.\nAlthough  the  identification  of  the  genus  Brucella is\nstraightforward  (small  gram-negative  coccobacilli;  posi-\ntive oxidase, catalase, and urease test results; no sugar fer-\nmentation;  and  requirement  of  aerobic  conditions  and\nadded CO2 for its growth), laboratory-acquired disease has\nfrequently resulted from misidentification of the organism\n(35). In 2 published outbreaks, the Brucella isolate resisted\ndecoloration  and  appeared  as  a  gram-positive  or  gram-\nvariable  coccobacillus  and  thus  was  misidentified  as\nmicrococcus  or  a  coryneform  bacillus  (14,29).  In  other\ncases,  B.  melitensis organisms  tested  with  the  API20NE\n(bioMérieux,  Marcy  l’Etoile,  France)  identification  kit\nproduced  the  1200004  or  1000004  biochemical  profile\nleading  to  misidentification  of  the  isolate  as  Moraxella\nphenylpyruvica at  a  good  identification  level  (90.5%)\n(8,18–21). The  frequent  failure  of  clinical  laboratories  to\ncorrectly identify isolates as brucellae is particularly wor-\nrisome because these organisms are regarded as potential\nagents for bioterrorism. Brucellae are inexpensive to pro-\nduce and disperse, and transmission to humans may result\nin prolonged illness and long-term sequelae. These organ-\nisms  are  considered  category  B  select  agents  (2). The\noccurrence in the last few years of inadvertent exposures to\nbrucellae  in  laboratories  in  industrialized  countries\n(7,8,14,17–26)  indicates  a  lack  of  preparedness  to  deal\nwith a real biologic threat. Lack of recognition of an iso-\nlate as a Brucella sp. by a laboratory may enable a bioter-\nrorism-related  attack  to  go  undetected,  whereas  a\nfalse-positive identification may cause unnecessary alarm.\nIn laboratories serving countries in which human bru-\ncellosis is still endemic, the degree of potential exposure to\nthe  organism  may  be  extremely  high.  In  a  microbiology\nlaboratory in Ankara, Turkey, the average annual number\nof cultures positive for Brucella spp. reaches 400 (34). A\nlaboratory serving a brucellosis-endemic area in southern\nIsrael processes ≈150 per year; at the peak of the Brucella\nseason  (April–June),  10%  of  all  positive  blood  cultures\ngrow B. melitensis (27). The risk for exposure in develop-\ning  countries  is  frequently  aggravated  by  lack  of  safety\nequipment and inadequate laboratory facilities (34). In the\naforementioned Turkish laboratory, the disease affected 10\n(18%) of 55 laboratory workers, and the calculated hazard\nwas 8% per employee-year (34).\nthe\nThe probable source of the infection is usually apparent\nwhen disease occurs in laboratories where isolation of bru-\ncellae  is  rare  (21).  In  laboratories  serving  brucellosis-\ntime  and  circumstances  of\nendemic  areas,\ncontamination  are  more  difficult  to  trace  because  of  the\nexistence  of  multiple  potential  sources  (22,27). Although\nthe exact form of transmission often remains speculative,\naerosols have been implicated in most cases (24,27,29,32).\nThis  assumption  has  been  strengthened  by  spread  of  the\ndisease  to  distant  areas  through  common  ventilation\n(27,32).\nLaboratory  accidents  such  as  breakage  of  centrifuge\ntubes (7) or blood-culture vials (16) play a minor role in\nlaboratory-acquired  disease  and  are  responsible  for  only\n20% of cases (6). More commonly, exposures are the result\nof  unsafe  laboratory  practices,  such  as  sniffing  plates\n(8,24,26,29,34);  working  on  an  open  bench  with  viable\norganisms  (14,16,17,20,22);  not  using  protective  equip-\nment such as gloves, masks, and goggles (34); or ingesting\nsuspensions of living brucellae during mouth pipetting (3).\nPERSPECTIVE\nHowever, even when no apparent breach in safety proce-\ndures  is  apparent,  transmission  may  occur  (17).  Rare\nevents—such as self-inoculation of brucellae by syringes\nloaded with a suspension of the organism or with synovial\nfluid from an infected patient (3,16), injury to the conjunc-\ntiva with a broken tube that had contained a living culture\n(3), or participation in a laboratory exercise in which still-\nviable  organisms  were  inadvertently  used  (15)—are  also\nresponsible  for  a  small  number  of  laboratory  exposures\nand outbreaks of disease.\nMost  cases  of  laboratory-acquired  brucellosis  have\noccurred  in  clinical  laboratories,  but  transmission  of  the\ndisease  in  research  facilities  (4,25)  and  laboratories  that\nmanufactured Brucella vaccines has also been document-\ned  (21,22).  A laboratory  technician  at  the  Centers  for\nDisease  Control  and  Prevention  (CDC)  became  infected\nwhile working in a safety cabinet with a B. melitensis iso-\nlate that had originally caused an outbreak of laboratory-\nacquired  disease  in  a  community  hospital  (17).  In  a\nseparate episode, the organism isolated from a microbiolo-\ngy technologist with laboratory-acquired disease infected a\nlaboratory worker at the hospital to which the technologist\nhad been admitted (14).\nThe attack rate of laboratory-associated infections has\nranged  from  30%  to  100%  depending,  among  other  fac-\ntors, on the location of workers, whether aerosol-generat-\ning  procedures  have  been  performed,  and\nthe\nconcentration  of  microorganisms  in  the  contaminated\nmedia  (7,8).  However,  the  hazard  of  transmission  is  not\nlimited to persons who worked with the isolate. Among the\n74 reported cases of laboratory-acquired brucellosis from\n1897  to  1939  reviewed  by  Meyer  and  Eddie,  the  disease\nalso affected janitors and occasional visitors (3). In a large\noutbreak of laboratory-acquired brucellosis, 3 of 7 cases of\ndisease  occurred  among  persons  who  briefly  visited  the\nfacility but did not enter the room where Brucella cultures\nwere processed (27).\nInvestigating Outbreaks of Brucellosis\nRecognizing sporadic cases and even outbreaks of lab-\noratory-acquired brucellosis is not always easy because the\ndisease lacks distinctive clinical features (14,17) and has a\nwidely variable incubation period (7,17,19,32). In a cluster\nof disease that originated in a single exposure, the onset of\nsymptoms of the 7 affected laboratory workers spanned 5\nmonths (17). Although a temporal clustering of cases usu-\nally  suggests  a  large  exposure  from  a  pinpoint  source\n(7,32),  this  is  not  always  the  case.  When  biotyping  was\nperformed in the isolates recovered within a 5-week peri-\nod  from  7  infected  hospital  workers  in  southern  Israel,  3\ndistinct  strains  were  recognized,  demonstrating  that  the\noutbreak was caused by at least 3 separate exposures (27).\nGuidelines for Investigating Outbreaks\nIn-depth  investigation  of  laboratory-acquired  cases  of\nbrucellosis may lead to the identification of unsafe bacte-\nriologic  practices,  suitable  for  correction  by  educational\nand technical measures. To guide the investigation, the fol-\nlowing  recommendations  are  made.  1)  Send  isolates  to  a\nreference  laboratory  for  confirmation.  Organisms  should\nbe shipped as “dangerous goods,” according to the guide-\nlines established by the World Health Organization and the\nOffice  of  Biosafety  at  CDC  (36).  In  the  United  States,\norganisms presumptively identified as brucellae should be\nsent to CDC or another public health laboratory following\nthe specific guidelines for transferring “select agents.” 2)\nInform infection control services and public health author-\nities,  who  may  choose  to  involve  CDC.  3)  Conduct  a\nmeticulous epidemiologic investigation. 4) Determine the\ndate  and  circumstances  of  the  exposure  (17).  5)  Exclude\nother  potential  sources  of  transmission,  such  as  previous\nlaboratory exposures, travels to brucellosis-endemic areas,\nconsumption  of  unpasteurized  dairy  products,  and  han-\ndling  of  farm  or  laboratory  animals  (14,15,17,27,29).  6)\nKeep all relevant data and records. 7) Define the exposed\npopulation. 8) Determine the level of risk on the basis of\ntype of laboratory procedures performed, proximity to the\nsource,  duration  of  the  exposure,  and  the  like,  to  define\npersons  to  whom  postexposure  prophylaxis  should  be\noffered (8,32). 9) Check biologic safety cabinets (27,29).\n10) Check the ventilation system (27). 11) Collect baseline\nserum  samples  from  all  known  potentially  exposed  per-\nsons  (17,27).  12)  Freeze  isolates  for  future  typing,  espe-\ncially in brucellosis-endemic areas (23,27).\nPrevention of Laboratory-acquired Cases\nBrucellae  are  considered  class  3  organisms.  CDC  has\nstrongly recommended that live Brucella cultures and sus-\npicious  organisms  be  manipulated  in  a  class  II  biologic\nsafety  cabinet  (37).  This  recommendation,  however,  is\nclearly insufficient for preventing laboratory-acquired dis-\nease because by the time the organism is suspected or con-\nfirmed as Brucella, exposure of laboratory personnel may\nhave occurred (27). On the other hand, converting a large\nand busy clinical microbiology laboratory into a biosafety\nlevel III facility, where all specimens are handled in biolog-\nic safety cabinets (32), is both impractical and unnecessary,\nespecially in laboratories in areas where the disease is not\nendemic. However, in regions where brucellosis is highly\nprevalent, an enhanced safety policy should be adopted. All\nblood  culture  vials  detected  as  positive  by  the  automated\nblood culture system, as well as all bone marrow and syn-\novial  fluid  specimens,  should  be  manipulated  in  biologic\nsafety hoods until the isolated microorganisms are defini-\ntively determined to be other than Brucella sp. (27).\nFor  laboratories  serving  areas  where  brucellosis  is\nuncommon, the following recommendations are made. 1)\nBecause of the low incidence of brucellosis, physicians in\nareas not endemic for the disease are unfamiliar with the\nclinical and epidemiologic features of human brucellosis,\nand the possibility of brucellosis is rarely considered in the\ndifferential diagnosis. Periodic education of physicians on\nthis  subject  is  indicated.  2)  Communication  between\nattending  physicians  and  the  laboratory  should  be\nimproved. The clinical microbiology laboratory should be\ninformed  in  advance  when  clinical  specimens  had  been\nobtained  from  patients  with  risk  factors  for  brucellosis,\nsuch  as  recent  history  of  travel  to  brucellosis-endemic\nareas,  consumption  of  local  or  imported  unpasteurized\ndairy products, or professional exposures in veterinarians,\nshepherds,  slaughterhouse  employees,  and  laboratory\nworkers. 3) The use of automated, continuous monitoring\nblood culture systems for patients with suspected brucel-\nlosis  should  be  preferred  over  the  lysis-centrifugation\nmethod  because  the  latter  involves  centrifuging  clinical\nspecimens  and  visually  inspecting  plates  to  detect  the\norganism and probably increases the risk for transmission\n(11,27). Although the use of blood lysis-based methods has\nbeen  advocated  in  the  past  for  improving  detection  of\nBrucella bacteremia, modern automated blood culture sys-\ntems are faster and more sensitive (11,12). 4) The familiar-\nity of laboratory technicians with the characteristics of the\norganism,  as  well  as  with  the  safe  handling  of  cultures,\nshould be improved and maintained through periodic edu-\ncation.  The  reader  is  referred  to  the  excellent  review  by\nGilligan  and York  for  handling  and  identifying  presump-\ntive Brucella organisms (35). 5) Standard precautions and\nstrict adherence to good laboratory practices must be com-\npletely  adopted,  reinforced,  and  regularly  monitored.  6)\nAll work with gram-negative or gram-variable small rods\nor coccobacilli isolated from tissues, blood, bone marrow,\nbone, or synovial fluid exudates should be carried out in a\nbiosafety  cabinet  until  Brucella has  been  ruled  out.  7)\nPlates  should  be  sealed  for  safety  when  not  in  use  and\nappropriately disposed and sterilized as soon as they are no\nlonger being actively used (17). 8) Because brucellae are\nrelatively slow-growing bacteria, cultures for the organism\nhave been traditionally kept for several weeks. However,\nmodern  blood  culture  systems  enable  brucellae  to  be\ndetected within the routine 5-day incubation period insti-\ntuted  in  most  clinical  laboratories  (12). Therefore,  safety\nprecautions should not be limited to organisms that tend to\ngrow slowly (8,18,20,26). 9) Antimicrobial drug–suscepti-\nbility  testing  of  Brucella  organisms  is  not  indicated\nbecause the therapeutic regimen for brucellosis is standard,\nand  the  organism  does  not  usually  acquire  antimicrobial\nresistance.  Performance  of  this  and  other  unnecessary\ntests,  and  especially  of  laboratory  procedures  known  to\nproduce aerosols, should be strongly discouraged (27). 10)\nIf a suspension of living brucellae is spilled and the organ-\nism is recognized, the entire laboratory should be immedi-\nately  evacuated,  doors  should  be  shut,  and  an  effective\ngermicide  such  as  3%  phenol  or  10%  bleach  should  be\napplied  by  a  trained  person  wearing  a  safety  mask,  gog-\ngles, an impermeable laboratory gown, and gloves (7).\nPostexposure Prophylaxis\nBecause  of  ethical  considerations,  the  heterogeneous\nnature of the events leading to Brucella exposure, difficul-\nties in determining the actual risk for individual workers,\nlate recognition of outbreaks, and the small number of per-\nsons involved in each outbreak, no controlled studies have\nbeen performed to assess the value of administering post-\nexposure  prophylaxis  to  persons  at  risk.  However,  anec-\ndotal  evidence  suggests  that  administering  prophylactic\nantimicrobial drug therapy may reduce the risk of develop-\ning clinical disease (7,8,26).\nIn a recently reported event, an isolate from a chest wall\nexudate  culture  from  an  Indian  patient,  which  was  origi-\nnally identified as M. phenylpyruvica, was correctly recog-\nnized  as  B.  melitensis 22  days  after  the  specimen  was\nobtained (8). By that time, 26 laboratory workers had been\npotentially  exposed,  and  6  had  actually  manipulated  the\norganism. These 6 workers were considered to be at high\nrisk and offered a 3-week prophylactic course of combined\ndoxycycline-rifampin  or  trimethoprim-sulfamethoxazole\ntherapy.  None  of  the  5  laboratory  technologists  who\nreceived  postexposure  antimicrobial  drugs  became  ill  or\ndeveloped an antibody response. However, Brucella bac-\nteremia  developed  in  the  only  laboratory  worker  who\nrefused therapy, and she seroconverted 10 weeks after the\nspecimen  was  received  (8).  Neither  clinical  disease  nor\nseroconversion developed in the remaining 19 laboratory\nworkers who were only present in the laboratory but were\nnot in direct contact with the organism (8).\nIn  another  event,  3  laboratory  technologists  who  had\ninadvertently  worked  with  a  B.  melitensis isolate  on  an\nopen bench and sniffed the plates were given 1 week of pro-\nphylactic  doxycycline  within  24  hours  of  the  exposure;\nnone of them became ill or seroconverted (26). In a large\nexposure caused by breakage of a tube containing B. abor-\ntus, patients who started therapy immediately after serocon-\nversion was detected and before symptoms developed (10\nweeks after the laboratory accident) had a benign clinical\ncourse, suggesting that even if the disease was not prevent-\ned, a certain degree of attenuation probably occurred (7).\nBecause  of  the  high  attack  rate  of  brucellosis  among\nexposed  workers,  the  unpredictable  and  often  chronic\ncourse of the disease, and the difficulties in eradicating the\norganism  once  a  symptomatic  infection  has  been  estab-\nlished, postexposure prophylaxis is probably indicated for\nall  persons  after  an  obvious  exposure  to  living  brucellae\n(26). Antimicrobial  drug  prophylaxis  with  a  combination\nof oral doxycycline 100 mg twice a day plus rifampin 600\nmg 4 × per day for 3 weeks should be started as soon as the\nexposure  to  confirmed  Brucella organisms  is  recognized\n(8). For pregnant women, administration of trimethoprim-\nsulfamethoxazole 160/800 mg 2 × per day for 3 weeks has\nbeen advocated (8).\nPostexposure Follow-up\nWhether  exposed  persons  have  received  prophylactic\ntherapy or not, increased surveillance for clinical signs of\ndisease should be conducted for at least 6 months. In addi-\ntion,  exposed  laboratory  workers  should  be  followed  for\npossible subclinical infections or early signs of disease by\nperiodic serologic testing. Weekly or semiweekly serolog-\nic surveillance is recommended for the first 3 months and\nonce  a  month  thereafter  for  3  to  9  additional  months  to\ndetect late infections resulting from prolonged incubation\n(7,8).\nHuman brucellosis has variable clinical manifestations\nand may mimic other infections, particularly influenza, as\nwell as noninfectious conditions. Therefore, the diagnosis\nof  the  disease  requires  a  high  index  of  suspicion.\nEducation  of  exposed  personnel  on  the  symptoms  of  the\ndisease and the need for periodic and timely serologic fol-\nlow-up are particularly important. Increased vigilance dur-\ning  the  flu  season  employing  a  broad  case  definition  is\nclearly  needed,  and  administration  of  influenza  vaccine\nshould be strongly recommended to all Brucella-exposed\npersons. If symptoms develop, blood cultures and cultures\nof other normally sterile body fluids, as clinically indicat-\ned, should be obtained, and a new antibody titer should be\ndetermined.  Although  use  of  nucleic  acid  amplification\nmethods  has  been  shown  to  enable  early  detection  of\ninfected persons (38), these tests are not yet commercially\navailable.\nTreating Infected Persons and\nPosttreatment Follow-up\nPersons  in  whom  culture-confirmed  or  serologically\nproven disease develops should receive therapy with oral\ndoxycycline and rifampin for 6 weeks, or a combination of\noral doxycycline for 6 weeks and an intramuscular amino-\nglycoside  (gentamicin  or  streptomycin)  for  the  first  2\nweeks (2,7). Use of 3 drugs is usually reserved for compli-\ncated cases or life-threatening clinical manifestations, such\nas endocarditis or meningitis (1). Pregnant women should\nreceive  trimethoprim-sulfamethoxazole  160/800  mg  2  ×\nper day for 6 weeks (1).\nEven  when  patients  are  appropriately  treated,  the  risk\nfor  relapse  remains  high  (≈20%)  (39,40).  Therefore,\npatients who have completed a full therapeutic antimicro-\nbial course should be followed clinically and serologically\nfor  1  year.  Failure  to  show  declining  antibody  titers  may\nindicate incomplete cure. If symptoms consistent with bru-\ncellosis develop, blood cultures and serologic tests should\nbe performed to detect relapses of the disease (39). Use of\nnucleic acid amplification methods in the future may fur-\nther improve the detection of patients in whom the organ-\nism was not eliminated (40).\nAlthough human brucellosis has been eradicated from\nmost  industrialized  countries,  isolated  cases  of  disease,\nusually  related  to  travel  or  import  of  contaminated  food\nfrom disease-endemic areas, continue to occur. Because of\nthe  low  incidence  of  disease  in  industrialized  countries,\nclinical  laboratory  technologists  have  become  unfamiliar\nwith  identifying  and  handling  Brucella species.  Unsafe\nlaboratory  practices  while  manipulating  Brucella isolates\nhave resulted in inadvertent exposure to the organism and\nmany cases of laboratory-acquired disease, indicating lack\nof preparedness to cope with bioterrorism threats involv-\ning  brucellae.  Education  of  laboratory  personnel  on  the\nidentification  of  Brucella  species,  adherence  to  and\nenforcement  of  standard  precautions,  thorough  investiga-\ntion of laboratory exposures, administration of prophylac-\ntic  antimicrobial  drug  therapy,  and  close  follow-up  of\ndirectly exposed persons are strongly recommended.\nDr.  Yagupsky  is  the  director  of  the  Clinical  Microbiology\nLaboratory  of  the  Soroka  University  Medical  Center,  Beer-\nSheva, Israel. His main research areas are Kingella kingae infec-\ntions and diagnosis of human brucellosis.\nDr.  Baron  is  director  of  the  Clinical  Microbiology\nLaboratory at Stanford University Medical Center and professor\nof pathology at Stanford University Medical College. Her inter-\nests include the development of rapid infectious disease diagnos-\ntic  tests  and  educating  both  scientists  and  the  public  about\nrelevant aspects of infectious diseases.",
         null
        ],
        [
         "15",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "Safe motion planning in dynamic environments",
         "This paper addresses the problem of motion planning (MP) in dynamic environments. It is first argued that dynamic environments impose a real-time constraint upon MP: it has a limited time only to compute a motion, the time available being a function of the dynamicity of the environment. Now, given the intrinsic complexity of MP, computing a complete motion to the goal within the time available is impossible to achieve in most real situations. Partial motion planning (PMP) is the answer to this problem proposed in this paper. PMP is a motion planning scheme with an anytime flavor: when the time available is over, PMP returns the best partial motion to the goal computed so far. Like reactive navigation scheme, PMP faces a safety issue: what guarantee is there that the system will never end up in a critical situation yielding an inevitable collision? The answer proposed in this paper to this safety issue relies upon the concept of inevitable collision states (ICS). ICS takes into account the dynamics of both the system and the moving obstacles. By computing ICS-free partial motion, the system safety can be guaranteed. Application of PMP to the case of a car-like system in a dynamic environment is presented.",
         "Motion prediction for moving objects: a statistical approach Inevitable collision states. A step towards safer robots? Towards real-time global motion planning in a dynamic environment using the NLVO concept An integrated approach to goal-directed obstacle avoidance under dynamic constraints for dynamic environments Real-time randomized path planning for robot navigation Improved fast replanning for robot navigation in unknown terrain Learning motion patterns of persons for mobile service robots Reactive navigation for non-holonomic robots using the ego-kinematic space Randomized Kinodynamic Motion Planning with Moving Obstacles Real-time motion planning for agile autonomous vehicles Kinodynamic motion planning amidst moving obstacles VFH/sup */: local obstacle avoidance with look-ahead verification Randomized Kinodynamic Planning High-speed navigation using the global dynamic window approach Motion Planning in Dynamic Environments Using Velocity Obstacles The dynamic window approach to collision avoidance Probabilistic roadmaps for path planning in high-dimensional configuration spaces The curvature-velocity method for local obstacle avoidance The Focussed D* Algorithm for Real-Time Replanning Reflexive collision avoidance: a generalized approach Motion planning in the presence of moving obstacles Robot Motion Planning Sensor-Based Motion Control for Mobile Robots Kinodynamic planning in a structured and time-varying 2-D workspace",
         "https://hal.inria.fr/inria-00182046/file/2005-petti-fraichard-iros.pdf",
         "Abstract— This paper addresses the problem of motion plan-\nning (MP) in dynamic environments. It is ﬁrst argued that\ndynamic environments impose a real-time constraint upon MP:\nit has a limited time only to compute a motion, the time available\nbeing a function of the dynamicity of the environment. Now,\ngiven the intrinsic complexity of MP, computing a complete\nmotion to the goal within the time available is impossible to\nachieve in most real situations. Partial Motion Planning (PMP)\nis the answer to this problem proposed in this paper. PMP is a\nmotion planning scheme with an anytime ﬂavor: when the time\navailable is over, PMP returns the best partial motion to the goal\ncomputed so far. Like reactive navigation scheme, PMP faces a\nsafety issue: what guarantee is there that the system will never\nend up in a critical situation yielding an inevitable collision?\nThe answer proposed in this paper to this safety issue relies\nupon the concept of Inevitable Collision States (ICS). ICS takes\ninto account the dynamics of both the system and the moving\nobstacles. By computing ICS-free partial motion, the system\nsafety can be guaranteed. Application of PMP to the case of a\ncar-like system in a dynamic environment is presented.",
         "The problem of autonomous navigation has attracted a lot\nof interest for various robotic systems during last decades.\nTwo main paradigms have arisen to tackle this problem, the\ndeliberative approaches resulting in global motion planning\nschemes, i.e. the determination of a complete plan based on a\npriori known information and the reactive approaches taking\ndecision from real-time data by means of exteroceptive sen-\nsors, while moving. Though ﬁrst problems involved mostly\nsimple systems evolving among a stationary environment,\nit became clear that for real applications, dynamic systems\nas well as a dynamic environment had to be considered.\nthe system has\nWithin a dynamic environment however,\nthe obligation to make a decision, within a bounded time,\notherwise it might be in danger by the sole fact of being\npassive. This limited available time for the system to make\na decision, i.e. plan a motion, depends on the nature and\ndynamicity of the environment and is a hard real\ntime\nconstraint. Unfortunately, global motion planning schemes\nbased on techniques from computational geometry that were\nused for the ﬁrst problems [1], did not give much hope to\nfulﬁl such a timing constraint due to their inherent NPHard\ncomplexity [2]. Hence, reactive methods have been preferred\nand several schemes have been presented [3], [4], [5]. Nev-\nertheless, reactive approaches exhibit strong limitations. In\norder to take fast decisions, these methods explore locally\nthe velocity space of the system from which one admissible\ncontrol is selected at a time. As a consequence these methods\nexhibit a lack of lookahead, conducting the robot\nto be\ntrapped in local minima during its trip, and a weak goal\ndirectedness keeping the robot from reaching the objective.\nDespite recent modiﬁcations on a few schemes in order to\nimprove these techniques, once systems with kinematic or\ndynamic constraints are considered, speciﬁc schemes are pro-\nposed [3], [6], [7] but cannot handle such complex systems\nin a general form. Fortunately, in the mid 90’s, probabilistic\nplanners appeared [8] and brought a new powerful\ntool\nfor rapid exploration of high dimensional state-time space,\nframework presented in [9] used for complex systems and\ndynamic environment description. Some recent work could\ndemonstrate fast planning within a dynamic environment\nusing probabilistic techniques [10] and [11].\nHowever, this work does not take into consideration the\nfact that a complete trajectory to the goal might not be\nfound before the available allotted time has elapsed. In fact,\nas [12], we believe that for complex systems or environment,\na complete trajectory to the goal cannot be found over a\nlimited time, in general. In such a case, it becomes of the\nutmost importance to consider the behaviour of the system\nat the end of the trajectory. What if a car ends its trajectory\nin front of a wall at high speed? It becomes clear that strong\nguarantees should be given to this trajectory in order to\nhandle the safety issues raised by such a partial planning.\nA few papers only have addressed the issue of safety\nfor partial or incomplete trajectory by providing, in another\ncontext braking policies [13] or more recently by calculating\nevasive plan [14] or by insuring the system to be collision\nfree during an empirical period of time τ [12]. We believe\nthat these methods propose partial answers to the problem of\nsafety. In Fig. 1 we consider a selected milestone of a point\nmass robot with non zero velocity moving to the right (a state\nof P is therefore characterised by its position (x, y) and its\nspeed v). Depending upon its speed there is a region of states\n(in grey) for which P, even though it is not in collision and\nis τ -safe, will not have the time to brake and avoid collision.\nThese states for which no matter what the future trajectory of\nthe system is, a collision with the obstacle occurs, have been\ncharacterized by [15] as Inevitable Collision States (ICS).\nThe framework of ICS provides a suitable framework to\nestablish the relation of the collision states and the dynamic\nconstraints in order to have strong safety guarantees.\nB. Contribution of the Paper\nIn this paper, we ﬁrst detail the mechanism of a Partial Mo-\ntion Planner (PMP) designed to handle the partial plans that\nwould result from planning under hard real time constraint\nwithin a dynamic environment. Then, we address the safety\nissue related to such a scheme by calculating trajectories\nof safe states using the framework of ICS. We provide a\nsufﬁcient condition for hard safety guarantee for the PMP,\nallowing discrete calculation of the ICS, thus reducing the\ncomputational complexity of the scheme while insuring the\nsafety of the whole trajectory.\nThe paper is organised as follows: §2 brieﬂy reviews\nthe work related to trajectory planning within a dynamic\nenvironment and detail the existing safety guarantees that\nhave been provided in recent approaches. After introducing\nin §3 useful notations and deﬁnitions, we detail in §4 the\nmechanism of the PMP algorithm for which we provide\nand demonstrate in §5 a sufﬁcient hard safety condition. We\npresent ﬁnally in §6 our preliminary results for a car-like\nrobot and draw our conclusion in §7.",
         "D. Preliminary Results\nIn this implementation, the duration of a cycle δc is 1s,\nand the integration step of the differential equation is 0.5s.\nIn Fig. 7 we can see an example of a navigation from a\nstill starting state to a still goal state. The environment is\ncluttered by moving and static obstacles. In 7(a) one can\nobserve in front of the car how the safe partial trajectory is\ncalculated and planned within the time-state space in order\nto avoid the obstacle moving upward. The states behind\nthe car, deﬁne the trajectory, built from partial trajectories\nfrom the previous PMP cycles and (ideally) executed by\nthe robot. In 7(b) we can observe that the car was obliged\nto slow down at the intersection of several obstacles, since\nno other safe trajectories could be found, before to re-\naccelerate. In 7(d) the system has planned a partial trajectory\nthat avoids the last static obstacle. Videos of this simula-\ntion can be found at http://emotion.inrialpes.fr\n/film-gallery.php.\nThis work is currently being integrated on a real platform\n(ﬁg. 8(a)) moving within a parking lot. External cameras\nabove the parking (ﬁg. 8(b)) have been installed in order to\nfully observe the environment and thus localize our system\nand detect static (parked cars) as well as moving obstacles\n(pedestrians and cars) whereas the trajectories of the moving\nobstacles will be predicted using the work of [23].\nThe exploration method used is the well known Rapidly-\nExploring Random Tree method (RRT) [24]. RRT incre-\nmentally builds a tree in the state space of A. The basic\nprinciple of RRT is depicted in Fig. 6. A state sr is randomly\nselected ﬁrst. Then, the closest node in the tree, say sc,\nis determined. Constant controls selected from U={ (α, ˙ξ)\nVII. DISCUSSION AND CONCLUSION\nIn this paper we tackled the problem of motion planning\nwithin a dynamic environment and proposed a Partial Motion\nPlanning scheme (PMP) which handles the real-time con-\nstraint inherent to such environment. The PMP algorithm\nconsists in iteratively exploring the state-time space during\na ﬁxed limited time, by building a tree using probabilistic\ntechniques. During a cycle, a complete trajectory calculation\nto the goal can not be guaranteed in general, which raises\nthe issue of the safety of our system. We use the formalism\nof the Inevitable collision States (ICS) as the theoretical\nanswer to this safety problem. A trajectory of ICS-free states\nguarantees that our system always have a possibility to escape\ncritical situations. ICS computation remains however a very\ncomplicated task in its general form and we have to rely\non conservative approximation for practical ICS calculation.\nIn this paper, we also demonstrate a property that simpliﬁes\nthe safety checking of a trajectory. Finally, we present an\nimplementation of the PMP algorithm for a car-like robot\nand provide simulation results of the safe trajectory found\nby PMP for our system.\nFuture work includes the coupling of the PMP algorithm\nwith a closed loop control and its integration on an exper-\nimental vehicle. Our goal is to perform experimentations\nwithin a real environment, for which a model of the future\nobstacles’ behaviour will be determined thanks to a predic-\ntion technique.\nACKNOWLEDGMENT\nThis work was partially supported by Aisin AW Corp. and\nthe French CNRS Parknav and Predit Mobivip projects.",
         "II. PREVIOUS WORKS\nRecently, a few reactive approaches have been modiﬁed\nin order to increase their lookahead and goal directedness.\nBasically, these modiﬁed methods rely on a map, built at\nexecution-time, like an occupancy grid, or known a priori.\nThese modiﬁed reactive methods use techniques stemming\nin order to ex-\nfrom global motion planning techniques,\nplore the velocity space of the system and select a discrete\nsequence of velocities, introducing in a sense the concept\nof partial planning. The trajectory to be followed, ie this\ndiscrete sequence of velocities, is determined by mean of\na navigation function [16] or an incremental algorithm that\nbuilds a tree within the velocity space [17], [18], [19].\nHowever, complex systems or environments remain difﬁcult\nto handle in a general form.\nthe environment, resulting thus in safer motion. Beside early\nwork based on dynamic programming, like the dynamic A*\nalgorithm (D*) [20], [21],\nthe approach of motion plan-\nning has been mainly reconsidered with the new tools of\nprobabilistic techniques based on graph or tree construction,\nshowing impressive results [10], [11]. Only latest work on\nthis issue, recognise the possibility of complete trajectory\nplanning failure [12], [14] and attempts to provide safety\nguarantee. However the guarantees apply over speciﬁc, em-\npirical time period without obvious physical relation with the\nconsidered dynamic system. In [14] the authors complement\nthe work presented in [10] and deﬁne safe planning as\nthe capacity of the planner to calculate an escape plan in\ncase it has failed ﬁnding a complete trajectory to the goal\nduring the allotted time. But no details with respect to the\nvalid time length of the escape plan is given. In the work\nof [12] the authors do not attempt to plan at each step a\ncomplete trajectory, the computational constraints being to\nhigh, hence considering the eventuality of partial planning.\nThey present the concept of τ -safety as a guarantee of no\ncollision during τ seconds for each milestone of the tree.\nSince the planner builds a tree between primary milestones\nat the equilibrium state, such a criteria might sufﬁce in the\npresented scheme, nevertheless it is too weak for a general\nkinodynamic framework.\nIII. NOTATIONS AND DEFINITIONS\nLet A denote a robotic system placed in a workspace\nW. The motion model of A is described by a differential\nequation of the form ˙s = f (s, u) where s ∈ S is the state\nof the system, ˙s its time derivative and u ∈ U a control. S\nis the state space and U the control space of A. Let φ ∈ Φ:\n[t0, tf ] −→ U denote a control input, ie. a time-sequence\nof controls. Starting from an initial state s0, at time t0, and\nunder the action of a control input φ, the state of the system\nA at time t is denoted by s(t) = φ(s0, t). An initial state and\na control input deﬁne a trajectory for A, ie a time sequence of\nstates. The environment is cluttered with a set of obstacles.\nAn obstacle B is a closed subset of W. The deﬁnition of\na moving obstacle is time-dependent and denoted by B(t).\nLet B(t0, ∞) denotes the obstacle B from time t0 to ∞, it\nmodels the future behaviour of B. A state s is a collision\nstate at time t if and only if ∃B such that A(s) ∩ B(t) = ∅.\nAs per [15], let us recall the formal deﬁnition of an Inevitable\nCollision State.\nDEFINITION 1 (INEVITABLE COLLISION STATE)\nA state s is an Inevitable Collision State (ICS) if and only if\n∀φ, ∃tc (for a given φ) such that φ(s, tc) is a collision state.\nA safe trajectory is deﬁned as:\nGlobal motion planning schemes have been modiﬁed as\nwell in order to gain some reactivity toward changes within\nDEFINITION 2 (SAFE TRAJECTORY)\nA trajectory, deﬁned by the initial state s0 at time t0 and the\ncontrol input φ over [t0, tf ], is safe if and only if ∀t ∈ [t0, tf ],\ns(t) = φ(s0, t) is not an ICS.\nUsing these notations and deﬁnitions,\ndetails the Partial Motion Planning (PMP) algorithm.\nIV. PARTIAL MOTION PLANNER\nIn general, a robotic system cannot safely stand still in\na dynamic environment (it might be collided by a moving\nobstacle). It has to plan a motion within a bounded time and\nthen execute it in order to remain safe. The time δc available\nto calculate a new motion is function of the nature and\ndynamicity of the environment. Today, the only motion plan-\nning techniques able to deal with reasonably complex robotic\nsystems are randomized. Unfortunately, these techniques do\nnot have any running-time upper-bound: there is no guarantee\nwhatsoever that a complete motion can be computed within\nthe time δc available. Like motion planning, partial motion\nplanning requires a model of the environment, the ﬁrst step\nis aimed at getting this model. This paper does not address\nhowever the problem of model construction. This model can\nbe given a priori or built using sensor observations. In space\napplications for instance, most of the moving obstacles follow\nKepler’s law, which provides a means to compute a full a\npriori model of the future [12]. In many other applications\nhowever, the moving obstacles have their own free will and\ntheir future behaviour is only partially predictable (if at all).\nIn this kind of situations,\nthe model of the future must\nbe predicted using prediction techniques such as the ones\npresented in [22] or [23].\nNote that when predictions are used, it is likely that the\nmodel of the future that is obtained will have a limited\nvalidity duration δv. This is an additional argument for\npartial motion planning: what is the point of a complete\nexploration of the future if the model of the future used is\nlikely to be grossly inaccurate? It is better to iterate a partial\nmotion planning process taking as input a regularly updated\npredicted model of the future. The periodic iterative PMP\nscheme proposed in this paper therefore accounts for both\nthe planning time constraints and the validity duration of the\nmodel of the environment.\nThus, the PMP algorithm iterates over a cycle of duration\nδc ≤ δv as depicted in Fig. 2. It is assumed that the initial\nstate of A is ICS-free. Let us focus on the planning iteration\nstarting at time ti:\n1) An updated model of the future is acquired, ie B(ti, ∞)\nfor each moving obstacles.\n2) The state-time space of A is searched using an ran-\ndomised tree rooted at the state s(ti+1) with ti+1 =\nti + δc.\n3) At time ti+1, the current iteration is over, the best safe\npartial trajectory φi in the tree is selected according to a\ngiven criterion and is fed to the robot that will execute\n]\nit from now on. φi is deﬁned over [ti+1, ti+1 + δhi\nwith δhi the trajectory duration.\nThe algorithm operates until the last state of the planned\ntrajectory reaches a neighbourhood of the goal state.\nV. SAFETY ISSUES\nLike every method that computes partial motion only, PMP\nhas to face a safety issue: since PMP has no control over\nthe duration of the partial trajectory that is computed, what\nguarantee do we have that A will never end up in a critical\nsituations yielding an inevitable collision? The answer to that\nproblem lies in the very fact that the partial trajectory that is\ncomputed is ICS-free. Meaning that, even in the worst case\nscenario where the duration δhi of the partial trajectory is\nshorter than the cycle time δc, A can always execute one of\nthe existing safe trajectory. The overall safety is guaranteed\nas long as the initial state is ICS-free (which is something\nthat has been assumed).\nNow, determining whether a given state of A is an ICS\nor not is a complex task since it requires to consider all\npossible future trajectories for A. However, it i possible to\ntake advantage of the approximation property demonstrated\nin [15] in order to compute a conservative approximation of\nthe set of ICS. This is done by considering only a subset of\nthe full set of possible future trajectories (see §VI.B).\nBesides, in order to further reduce the complexity of the\nPMP algorithm, we present a property that simpliﬁes the\nsafety checking of a trajectory. To begin with, we demon-\nstrate a property that states that for a given control input, all\nthe states between an ICS and the corresponding collision\nstate, are ICS.\nPROPERTY 1\nLet s be an ICS at time t0. For a given control input φ, let\ntc denote the time at which a collision occurs. Then ∀t ∈\n[t0, tc], s(t) = φ(s, t) is also an ICS.\nProof: suppose that a state si = φ(s, ti), with ti ∈ [0, tc],\nis not an ICS. By deﬁnition, ∃φj that yields no collision when\napplied to si. Let φi denote the part of φ deﬁned over [t0, ti].\nClearly, the combination of φi and φj also yields no collision\nwhen applied to s  contradiction. \nWe can now provide a sufﬁcient safety condition for a\npartial\ntrajectory that states that provided a trajectory is\ncollision free, if the last state of the trajectory is not an\ninevitable collision state then none of the states of the\ntrajectory are inevitable collision states.\nPROPERTY 2 (PMP SUFFICIENT SAFETY CONDITION)\nGiven a trajectory deﬁned over [t0, tf ], if\n(H1) the trajectory is collision-free and\n(H2) s(tf ) is not an ICS\nthen ∀t ∈ [t0, tf ], s(t) is not an ICS.\nProof: Suppose that ∃ti ∈ [t0, tf ] such that si = s(ti) is\nan ICS. Then, by deﬁnition, ∀φ, ∃tc such that φ(si, tc) is a\ncollision state. If t0 ≤ tc ≤ tf then collision occurs before\ntf  contradiction with H1. Now, if tc > tf then by previous\nproperty P1, we must have s(tf ) is an ICS  contradiction\nwith H2. \ny\nx\nIn this section we present the application of PMP to the\ncase of a car-like vehicle A moving on a planar surface\nW and within a fully observable environment cluttered with\nstationary and dynamic obstacles.\nA. Vehicle Model\nA moves like a car-like vehicle and its dynamics follows\nthe bicycle model. A state of A is deﬁned by the 5-tuple\ns = (x, y, θ, v, ξ) where (x, y) are the coordinates of the\nrear wheel, θ is the main orientation of A, v is the linear\nvelocity of the rear wheel, and ξ is the orientation of the\nfront wheels (Fig. 3). A control of A is deﬁned by the couple\n(α, γ) where α is the rear wheel linear acceleration. and γ\nthe steering velocity. The motion of A is governed by the\nfollowing differential equations:\n\n\n\n\n\n\n˙x\n˙y\n˙θ\n˙vr\n˙ξ\n\n\n\n\n\n\n\n\n\n\nvr cos θ\nvr sin θ\ntan ξvr\nL\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\nwith α ∈ [αmin, αmax]\n(acceleration bounds), γ ∈\n[γmin, γmax] (steering velocity bounds), and |ξ| ≤ ξmax\n(steering angle bounds). L is the wheelbase of A.\nB. ICS Calculation\nIn general, computing the ICS for a given system is an\nintricate problem since it would require to consider the set\nof all the possible future trajectories. To compute in practice\nthe ICS for a system such as A, it is taken advantage of\nthe approximation property established in [15] showing that\na conservative approximation of the ICS can be obtained\nby considering only a ﬁnite subset I of the whole set of\npossible future trajectories. For practical reasons, the duration\nof the trajectories of I has to be limited to a given time\nhorizon that determines the overall level of safety of A.\nthis horizon can be\nIn partially predictable environment,\nset up to δv, thus providing the maximum possible safety\nguarantee. In our case, the subset I considered includes\nthe braking trajectories with a constant control selected\nfrom [(αmin, ˙ξmax), (αmin, 0), (αmin, ˙ξmin)], and applied\nover the time necessary for A to stop.\nFor instance, the shaded regions in subﬁgure (a) corre-\nspond to states with (θ, v) = (0.9, 1.0) for which the braking\ntrajectory of control (αmin, ˙ξmin) yields a collision with one\nof the obstacles, ﬁxed or moving. In the bottom subﬁgures,\nthe three braking trajectories are considered.\nIn PMP, checking whether a state is an ICS or not is carried\nout by testing if all the braking trajectories yield a collision\nwith one of the moving obstacles. In case all the trajectories\nappear to be in collision in the future, this state is an ICS\nand is not selected. Fig. 5 illustrates how a state of the partial\ntrajectory is checked to be an ICS or not. The collision states\nrepresent the collision that will occur in the future from this\nstate for all trajectories of I. In this case, since all trajectories\ncollide in the future, this state is an ICS.\n| α ∈(αmin, 0, αmax) and ˙ξ ∈( ˙ξmax, 0, ˙ξmin) } are then\napplied to sc for a duration , they yield a set of candi-\ndate trajectories ending in given states sni. These candidate\ntrajectories are pruned out: only are kept the trajectories\nthat are collision-free and whose ﬁnal state is ICS-free (as\nper property 2, such trajectories are ICS-free). Finally, the\ntrajectory whose ﬁnal state is closer to sr is selected and\nadded up to the tree. This process is repeated until the end of\nthe time available where the best partial trajectory extracted\nfrom the tree is returned.",
         "",
         "Abstract— This paper addresses the problem of motion plan-\nning (MP) in dynamic environments. It is ﬁrst argued that\ndynamic environments impose a real-time constraint upon MP:\nit has a limited time only to compute a motion, the time available\nbeing a function of the dynamicity of the environment. Now,\ngiven the intrinsic complexity of MP, computing a complete\nmotion to the goal within the time available is impossible to\nachieve in most real situations. Partial Motion Planning (PMP)\nis the answer to this problem proposed in this paper. PMP is a\nmotion planning scheme with an anytime ﬂavor: when the time\navailable is over, PMP returns the best partial motion to the goal\ncomputed so far. Like reactive navigation scheme, PMP faces a\nsafety issue: what guarantee is there that the system will never\nend up in a critical situation yielding an inevitable collision?\nThe answer proposed in this paper to this safety issue relies\nupon the concept of Inevitable Collision States (ICS). ICS takes\ninto account the dynamics of both the system and the moving\nobstacles. By computing ICS-free partial motion, the system\nsafety can be guaranteed. Application of PMP to the case of a\ncar-like system in a dynamic environment is presented.\nThe problem of autonomous navigation has attracted a lot\nof interest for various robotic systems during last decades.\nTwo main paradigms have arisen to tackle this problem, the\ndeliberative approaches resulting in global motion planning\nschemes, i.e. the determination of a complete plan based on a\npriori known information and the reactive approaches taking\ndecision from real-time data by means of exteroceptive sen-\nsors, while moving. Though ﬁrst problems involved mostly\nsimple systems evolving among a stationary environment,\nit became clear that for real applications, dynamic systems\nas well as a dynamic environment had to be considered.\nthe system has\nWithin a dynamic environment however,\nthe obligation to make a decision, within a bounded time,\notherwise it might be in danger by the sole fact of being\npassive. This limited available time for the system to make\na decision, i.e. plan a motion, depends on the nature and\ndynamicity of the environment and is a hard real\ntime\nconstraint. Unfortunately, global motion planning schemes\nbased on techniques from computational geometry that were\nused for the ﬁrst problems [1], did not give much hope to\nfulﬁl such a timing constraint due to their inherent NPHard\ncomplexity [2]. Hence, reactive methods have been preferred\nand several schemes have been presented [3], [4], [5]. Nev-\nertheless, reactive approaches exhibit strong limitations. In\norder to take fast decisions, these methods explore locally\nthe velocity space of the system from which one admissible\ncontrol is selected at a time. As a consequence these methods\nexhibit a lack of lookahead, conducting the robot\nto be\ntrapped in local minima during its trip, and a weak goal\ndirectedness keeping the robot from reaching the objective.\nDespite recent modiﬁcations on a few schemes in order to\nimprove these techniques, once systems with kinematic or\ndynamic constraints are considered, speciﬁc schemes are pro-\nposed [3], [6], [7] but cannot handle such complex systems\nin a general form. Fortunately, in the mid 90’s, probabilistic\nplanners appeared [8] and brought a new powerful\ntool\nfor rapid exploration of high dimensional state-time space,\nframework presented in [9] used for complex systems and\ndynamic environment description. Some recent work could\ndemonstrate fast planning within a dynamic environment\nusing probabilistic techniques [10] and [11].\nHowever, this work does not take into consideration the\nfact that a complete trajectory to the goal might not be\nfound before the available allotted time has elapsed. In fact,\nas [12], we believe that for complex systems or environment,\na complete trajectory to the goal cannot be found over a\nlimited time, in general. In such a case, it becomes of the\nutmost importance to consider the behaviour of the system\nat the end of the trajectory. What if a car ends its trajectory\nin front of a wall at high speed? It becomes clear that strong\nguarantees should be given to this trajectory in order to\nhandle the safety issues raised by such a partial planning.\nA few papers only have addressed the issue of safety\nfor partial or incomplete trajectory by providing, in another\ncontext braking policies [13] or more recently by calculating\nevasive plan [14] or by insuring the system to be collision\nfree during an empirical period of time τ [12]. We believe\nthat these methods propose partial answers to the problem of\nsafety. In Fig. 1 we consider a selected milestone of a point\nmass robot with non zero velocity moving to the right (a state\nof P is therefore characterised by its position (x, y) and its\nspeed v). Depending upon its speed there is a region of states\n(in grey) for which P, even though it is not in collision and\nis τ -safe, will not have the time to brake and avoid collision.\nThese states for which no matter what the future trajectory of\nthe system is, a collision with the obstacle occurs, have been\ncharacterized by [15] as Inevitable Collision States (ICS).\nThe framework of ICS provides a suitable framework to\nestablish the relation of the collision states and the dynamic\nconstraints in order to have strong safety guarantees.\nB. Contribution of the Paper\nIn this paper, we ﬁrst detail the mechanism of a Partial Mo-\ntion Planner (PMP) designed to handle the partial plans that\nwould result from planning under hard real time constraint\nwithin a dynamic environment. Then, we address the safety\nissue related to such a scheme by calculating trajectories\nof safe states using the framework of ICS. We provide a\nsufﬁcient condition for hard safety guarantee for the PMP,\nallowing discrete calculation of the ICS, thus reducing the\ncomputational complexity of the scheme while insuring the\nsafety of the whole trajectory.\nThe paper is organised as follows: §2 brieﬂy reviews\nthe work related to trajectory planning within a dynamic\nenvironment and detail the existing safety guarantees that\nhave been provided in recent approaches. After introducing\nin §3 useful notations and deﬁnitions, we detail in §4 the\nmechanism of the PMP algorithm for which we provide\nand demonstrate in §5 a sufﬁcient hard safety condition. We\npresent ﬁnally in §6 our preliminary results for a car-like\nrobot and draw our conclusion in §7.\nII. PREVIOUS WORKS\nRecently, a few reactive approaches have been modiﬁed\nin order to increase their lookahead and goal directedness.\nBasically, these modiﬁed methods rely on a map, built at\nexecution-time, like an occupancy grid, or known a priori.\nThese modiﬁed reactive methods use techniques stemming\nin order to ex-\nfrom global motion planning techniques,\nplore the velocity space of the system and select a discrete\nsequence of velocities, introducing in a sense the concept\nof partial planning. The trajectory to be followed, ie this\ndiscrete sequence of velocities, is determined by mean of\na navigation function [16] or an incremental algorithm that\nbuilds a tree within the velocity space [17], [18], [19].\nHowever, complex systems or environments remain difﬁcult\nto handle in a general form.\nthe environment, resulting thus in safer motion. Beside early\nwork based on dynamic programming, like the dynamic A*\nalgorithm (D*) [20], [21],\nthe approach of motion plan-\nning has been mainly reconsidered with the new tools of\nprobabilistic techniques based on graph or tree construction,\nshowing impressive results [10], [11]. Only latest work on\nthis issue, recognise the possibility of complete trajectory\nplanning failure [12], [14] and attempts to provide safety\nguarantee. However the guarantees apply over speciﬁc, em-\npirical time period without obvious physical relation with the\nconsidered dynamic system. In [14] the authors complement\nthe work presented in [10] and deﬁne safe planning as\nthe capacity of the planner to calculate an escape plan in\ncase it has failed ﬁnding a complete trajectory to the goal\nduring the allotted time. But no details with respect to the\nvalid time length of the escape plan is given. In the work\nof [12] the authors do not attempt to plan at each step a\ncomplete trajectory, the computational constraints being to\nhigh, hence considering the eventuality of partial planning.\nThey present the concept of τ -safety as a guarantee of no\ncollision during τ seconds for each milestone of the tree.\nSince the planner builds a tree between primary milestones\nat the equilibrium state, such a criteria might sufﬁce in the\npresented scheme, nevertheless it is too weak for a general\nkinodynamic framework.\nIII. NOTATIONS AND DEFINITIONS\nLet A denote a robotic system placed in a workspace\nW. The motion model of A is described by a differential\nequation of the form ˙s = f (s, u) where s ∈ S is the state\nof the system, ˙s its time derivative and u ∈ U a control. S\nis the state space and U the control space of A. Let φ ∈ Φ:\n[t0, tf ] −→ U denote a control input, ie. a time-sequence\nof controls. Starting from an initial state s0, at time t0, and\nunder the action of a control input φ, the state of the system\nA at time t is denoted by s(t) = φ(s0, t). An initial state and\na control input deﬁne a trajectory for A, ie a time sequence of\nstates. The environment is cluttered with a set of obstacles.\nAn obstacle B is a closed subset of W. The deﬁnition of\na moving obstacle is time-dependent and denoted by B(t).\nLet B(t0, ∞) denotes the obstacle B from time t0 to ∞, it\nmodels the future behaviour of B. A state s is a collision\nstate at time t if and only if ∃B such that A(s) ∩ B(t) = ∅.\nAs per [15], let us recall the formal deﬁnition of an Inevitable\nCollision State.\nDEFINITION 1 (INEVITABLE COLLISION STATE)\nA state s is an Inevitable Collision State (ICS) if and only if\n∀φ, ∃tc (for a given φ) such that φ(s, tc) is a collision state.\nA safe trajectory is deﬁned as:\nGlobal motion planning schemes have been modiﬁed as\nwell in order to gain some reactivity toward changes within\nDEFINITION 2 (SAFE TRAJECTORY)\nA trajectory, deﬁned by the initial state s0 at time t0 and the\ncontrol input φ over [t0, tf ], is safe if and only if ∀t ∈ [t0, tf ],\ns(t) = φ(s0, t) is not an ICS.\nUsing these notations and deﬁnitions,\ndetails the Partial Motion Planning (PMP) algorithm.\nIV. PARTIAL MOTION PLANNER\nIn general, a robotic system cannot safely stand still in\na dynamic environment (it might be collided by a moving\nobstacle). It has to plan a motion within a bounded time and\nthen execute it in order to remain safe. The time δc available\nto calculate a new motion is function of the nature and\ndynamicity of the environment. Today, the only motion plan-\nning techniques able to deal with reasonably complex robotic\nsystems are randomized. Unfortunately, these techniques do\nnot have any running-time upper-bound: there is no guarantee\nwhatsoever that a complete motion can be computed within\nthe time δc available. Like motion planning, partial motion\nplanning requires a model of the environment, the ﬁrst step\nis aimed at getting this model. This paper does not address\nhowever the problem of model construction. This model can\nbe given a priori or built using sensor observations. In space\napplications for instance, most of the moving obstacles follow\nKepler’s law, which provides a means to compute a full a\npriori model of the future [12]. In many other applications\nhowever, the moving obstacles have their own free will and\ntheir future behaviour is only partially predictable (if at all).\nIn this kind of situations,\nthe model of the future must\nbe predicted using prediction techniques such as the ones\npresented in [22] or [23].\nNote that when predictions are used, it is likely that the\nmodel of the future that is obtained will have a limited\nvalidity duration δv. This is an additional argument for\npartial motion planning: what is the point of a complete\nexploration of the future if the model of the future used is\nlikely to be grossly inaccurate? It is better to iterate a partial\nmotion planning process taking as input a regularly updated\npredicted model of the future. The periodic iterative PMP\nscheme proposed in this paper therefore accounts for both\nthe planning time constraints and the validity duration of the\nmodel of the environment.\nThus, the PMP algorithm iterates over a cycle of duration\nδc ≤ δv as depicted in Fig. 2. It is assumed that the initial\nstate of A is ICS-free. Let us focus on the planning iteration\nstarting at time ti:\n1) An updated model of the future is acquired, ie B(ti, ∞)\nfor each moving obstacles.\n2) The state-time space of A is searched using an ran-\ndomised tree rooted at the state s(ti+1) with ti+1 =\nti + δc.\n3) At time ti+1, the current iteration is over, the best safe\npartial trajectory φi in the tree is selected according to a\ngiven criterion and is fed to the robot that will execute\n]\nit from now on. φi is deﬁned over [ti+1, ti+1 + δhi\nwith δhi the trajectory duration.\nThe algorithm operates until the last state of the planned\ntrajectory reaches a neighbourhood of the goal state.\nV. SAFETY ISSUES\nLike every method that computes partial motion only, PMP\nhas to face a safety issue: since PMP has no control over\nthe duration of the partial trajectory that is computed, what\nguarantee do we have that A will never end up in a critical\nsituations yielding an inevitable collision? The answer to that\nproblem lies in the very fact that the partial trajectory that is\ncomputed is ICS-free. Meaning that, even in the worst case\nscenario where the duration δhi of the partial trajectory is\nshorter than the cycle time δc, A can always execute one of\nthe existing safe trajectory. The overall safety is guaranteed\nas long as the initial state is ICS-free (which is something\nthat has been assumed).\nNow, determining whether a given state of A is an ICS\nor not is a complex task since it requires to consider all\npossible future trajectories for A. However, it i possible to\ntake advantage of the approximation property demonstrated\nin [15] in order to compute a conservative approximation of\nthe set of ICS. This is done by considering only a subset of\nthe full set of possible future trajectories (see §VI.B).\nBesides, in order to further reduce the complexity of the\nPMP algorithm, we present a property that simpliﬁes the\nsafety checking of a trajectory. To begin with, we demon-\nstrate a property that states that for a given control input, all\nthe states between an ICS and the corresponding collision\nstate, are ICS.\nPROPERTY 1\nLet s be an ICS at time t0. For a given control input φ, let\ntc denote the time at which a collision occurs. Then ∀t ∈\n[t0, tc], s(t) = φ(s, t) is also an ICS.\nProof: suppose that a state si = φ(s, ti), with ti ∈ [0, tc],\nis not an ICS. By deﬁnition, ∃φj that yields no collision when\napplied to si. Let φi denote the part of φ deﬁned over [t0, ti].\nClearly, the combination of φi and φj also yields no collision\nwhen applied to s  contradiction. \nWe can now provide a sufﬁcient safety condition for a\npartial\ntrajectory that states that provided a trajectory is\ncollision free, if the last state of the trajectory is not an\ninevitable collision state then none of the states of the\ntrajectory are inevitable collision states.\nPROPERTY 2 (PMP SUFFICIENT SAFETY CONDITION)\nGiven a trajectory deﬁned over [t0, tf ], if\n(H1) the trajectory is collision-free and\n(H2) s(tf ) is not an ICS\nthen ∀t ∈ [t0, tf ], s(t) is not an ICS.\nProof: Suppose that ∃ti ∈ [t0, tf ] such that si = s(ti) is\nan ICS. Then, by deﬁnition, ∀φ, ∃tc such that φ(si, tc) is a\ncollision state. If t0 ≤ tc ≤ tf then collision occurs before\ntf  contradiction with H1. Now, if tc > tf then by previous\nproperty P1, we must have s(tf ) is an ICS  contradiction\nwith H2. \ny\nx\nIn this section we present the application of PMP to the\ncase of a car-like vehicle A moving on a planar surface\nW and within a fully observable environment cluttered with\nstationary and dynamic obstacles.\nA. Vehicle Model\nA moves like a car-like vehicle and its dynamics follows\nthe bicycle model. A state of A is deﬁned by the 5-tuple\ns = (x, y, θ, v, ξ) where (x, y) are the coordinates of the\nrear wheel, θ is the main orientation of A, v is the linear\nvelocity of the rear wheel, and ξ is the orientation of the\nfront wheels (Fig. 3). A control of A is deﬁned by the couple\n(α, γ) where α is the rear wheel linear acceleration. and γ\nthe steering velocity. The motion of A is governed by the\nfollowing differential equations:\n\n\n\n\n\n\n˙x\n˙y\n˙θ\n˙vr\n˙ξ\n\n\n\n\n\n\n\n\n\n\nvr cos θ\nvr sin θ\ntan ξvr\nL\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\nwith α ∈ [αmin, αmax]\n(acceleration bounds), γ ∈\n[γmin, γmax] (steering velocity bounds), and |ξ| ≤ ξmax\n(steering angle bounds). L is the wheelbase of A.\nB. ICS Calculation\nIn general, computing the ICS for a given system is an\nintricate problem since it would require to consider the set\nof all the possible future trajectories. To compute in practice\nthe ICS for a system such as A, it is taken advantage of\nthe approximation property established in [15] showing that\na conservative approximation of the ICS can be obtained\nby considering only a ﬁnite subset I of the whole set of\npossible future trajectories. For practical reasons, the duration\nof the trajectories of I has to be limited to a given time\nhorizon that determines the overall level of safety of A.\nthis horizon can be\nIn partially predictable environment,\nset up to δv, thus providing the maximum possible safety\nguarantee. In our case, the subset I considered includes\nthe braking trajectories with a constant control selected\nfrom [(αmin, ˙ξmax), (αmin, 0), (αmin, ˙ξmin)], and applied\nover the time necessary for A to stop.\nFor instance, the shaded regions in subﬁgure (a) corre-\nspond to states with (θ, v) = (0.9, 1.0) for which the braking\ntrajectory of control (αmin, ˙ξmin) yields a collision with one\nof the obstacles, ﬁxed or moving. In the bottom subﬁgures,\nthe three braking trajectories are considered.\nIn PMP, checking whether a state is an ICS or not is carried\nout by testing if all the braking trajectories yield a collision\nwith one of the moving obstacles. In case all the trajectories\nappear to be in collision in the future, this state is an ICS\nand is not selected. Fig. 5 illustrates how a state of the partial\ntrajectory is checked to be an ICS or not. The collision states\nrepresent the collision that will occur in the future from this\nstate for all trajectories of I. In this case, since all trajectories\ncollide in the future, this state is an ICS.\n| α ∈(αmin, 0, αmax) and ˙ξ ∈( ˙ξmax, 0, ˙ξmin) } are then\napplied to sc for a duration , they yield a set of candi-\ndate trajectories ending in given states sni. These candidate\ntrajectories are pruned out: only are kept the trajectories\nthat are collision-free and whose ﬁnal state is ICS-free (as\nper property 2, such trajectories are ICS-free). Finally, the\ntrajectory whose ﬁnal state is closer to sr is selected and\nadded up to the tree. This process is repeated until the end of\nthe time available where the best partial trajectory extracted\nfrom the tree is returned.\nD. Preliminary Results\nIn this implementation, the duration of a cycle δc is 1s,\nand the integration step of the differential equation is 0.5s.\nIn Fig. 7 we can see an example of a navigation from a\nstill starting state to a still goal state. The environment is\ncluttered by moving and static obstacles. In 7(a) one can\nobserve in front of the car how the safe partial trajectory is\ncalculated and planned within the time-state space in order\nto avoid the obstacle moving upward. The states behind\nthe car, deﬁne the trajectory, built from partial trajectories\nfrom the previous PMP cycles and (ideally) executed by\nthe robot. In 7(b) we can observe that the car was obliged\nto slow down at the intersection of several obstacles, since\nno other safe trajectories could be found, before to re-\naccelerate. In 7(d) the system has planned a partial trajectory\nthat avoids the last static obstacle. Videos of this simula-\ntion can be found at http://emotion.inrialpes.fr\n/film-gallery.php.\nThis work is currently being integrated on a real platform\n(ﬁg. 8(a)) moving within a parking lot. External cameras\nabove the parking (ﬁg. 8(b)) have been installed in order to\nfully observe the environment and thus localize our system\nand detect static (parked cars) as well as moving obstacles\n(pedestrians and cars) whereas the trajectories of the moving\nobstacles will be predicted using the work of [23].\nThe exploration method used is the well known Rapidly-\nExploring Random Tree method (RRT) [24]. RRT incre-\nmentally builds a tree in the state space of A. The basic\nprinciple of RRT is depicted in Fig. 6. A state sr is randomly\nselected ﬁrst. Then, the closest node in the tree, say sc,\nis determined. Constant controls selected from U={ (α, ˙ξ)\nVII. DISCUSSION AND CONCLUSION\nIn this paper we tackled the problem of motion planning\nwithin a dynamic environment and proposed a Partial Motion\nPlanning scheme (PMP) which handles the real-time con-\nstraint inherent to such environment. The PMP algorithm\nconsists in iteratively exploring the state-time space during\na ﬁxed limited time, by building a tree using probabilistic\ntechniques. During a cycle, a complete trajectory calculation\nto the goal can not be guaranteed in general, which raises\nthe issue of the safety of our system. We use the formalism\nof the Inevitable collision States (ICS) as the theoretical\nanswer to this safety problem. A trajectory of ICS-free states\nguarantees that our system always have a possibility to escape\ncritical situations. ICS computation remains however a very\ncomplicated task in its general form and we have to rely\non conservative approximation for practical ICS calculation.\nIn this paper, we also demonstrate a property that simpliﬁes\nthe safety checking of a trajectory. Finally, we present an\nimplementation of the PMP algorithm for a car-like robot\nand provide simulation results of the safe trajectory found\nby PMP for our system.\nFuture work includes the coupling of the PMP algorithm\nwith a closed loop control and its integration on an exper-\nimental vehicle. Our goal is to perform experimentations\nwithin a real environment, for which a model of the future\nobstacles’ behaviour will be determined thanks to a predic-\ntion technique.\nACKNOWLEDGMENT\nThis work was partially supported by Aisin AW Corp. and\nthe French CNRS Parknav and Predit Mobivip projects.",
         null
        ],
        [
         "16",
         "0012aecca19e4d3e41741599b1dbd00fb46d0c22",
         "Glutathione S-transferase T1 polymorphisms are associated with outcome in colorectal cancer.",
         "Colorectal cancer (CRC) remains a significant cause of mortality accounting for approximately 10% of all deaths from malignancy in the western world. Polymorphism in the glutathione S-transferase GSTT1 gene has been associated with CRC risk in some but not all studies. In this study, we examined associations between GSTT1 genotypes and CRC risk, and prognosis in 361 cases and 881 unrelated controls. GSTT1 null was associated with a small but significant increase in risk (P = 0.0006, odds ratio (OR) = 1.65, 95% confidence interval (CI) = 1.22-2.24). GSTT1 null was also associated with a significantly younger age at diagnosis (mean 65.2 years) compared with GSTT1 A (mean 67.6 years, P = 0.031). There were no significant associations between GSTT1 genotypes and clinical factors (e.g. Dukes stage, differentiation and tumour node metastasis classification) in the total case group. However, following stratification by age (<70 versus > or =70 years at diagnosis), in the patients diagnosed <70 years of age, GSTT1 null was more common in Dukes grade A/B tumours (P = 0.046), stage T1/T2 tumours (P = 0.053) and those with a pushing margin (P = 0.066). We also identified associations between GSTT1 null and increased prevalence of host lymphocyte response, particularly in the younger patients (P = 0.036). Furthermore, GSTT1 null was associated with improved survival in younger patients (P = 0.017, hazards ratio (HR) = 0.52, 95% CI = 0.31-0.89) but poorer survival in older patients (P = 0.017, HR = 1.89, 95% CI = 1.12-3.20). We proposed a model based on the dual functionality of GSTT1 to explain these contrasting results. We suggest that the null genotype is associated with improved immune response in younger patients, but poorer detoxification in older patients. These findings may also provide an explanation for the contrasting finding of other studies on the role of this gene in CRC.",
         "",
         "https://academic.oup.com/carcin/article-pdf/26/12/2157/7088641/bgi195.pdf",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "17",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "The N‐terminus of B96Bom, a Bombyx mori G‐protein‐coupled receptor, is N‐myristoylated and translocated across the membrane",
         "In eukaryotic cellular proteins, protein N‐myristoylation has been recognized as a protein modification that occurs mainly on cytoplasmic or nucleoplasmic proteins. In this study, to search for a eukaryotic N‐myristoylated transmembrane protein, the susceptibility of the N‐terminus of several G‐protein‐coupled receptors (GPCRs) to protein N‐myristoylation was evaluated by in vitro and in vivo metabolic labeling. It was found that the N‐terminal 10 residues of B96Bom, a Bombyx mori GPCR, efficiently directed the protein N‐myristoylation. Analysis of a tumor necrosis factor (TNF) fusion protein with the N‐terminal 90 residues of B96Bom at its N‐terminus revealed that (a) transmembrane domain 1 of B96Bom functioned as a type I signal anchor sequence, (b) the N‐myristoylated N‐terminal domain (58 residues) was translocated across the membrane, and (c) two N‐glycosylation motifs located in this domain were efficiently N‐glycosylated. In addition, when Ala4 in the N‐myristoylation motif of B96Bom90‐TNF, Met‐Gly‐Gln‐Ala‐Ala‐Thr(1–6), was replaced with Asn to generate a new N‐glycosylation motif, Asn‐Ala‐Thr(4–6), efficient N‐glycosylation was observed on this newly introduced N‐glycosylation site in the expressed protein. These results indicate that the N‐myristoylated N‐terminus of B96Bom is translocated across the membrane and exposed to the extracellular surface. To our knowledge, this is the first report showing that a eukaryotic transmembrane protein can be N‐myristoylated and that the N‐myristoylated N‐terminus of the protein can be translocated across the membrane.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2004.04487.x",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "18",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "Uptake of denatured collagen into hepatic stellate cells: evidence for the involvement of urokinase plasminogen activator receptor-associated protein/Endo180.",
         "Tissue remodelling is dependent on the integration of signals that control turnover of ECM (extracellular matrix). Breakdown and endocytosis of collagen, a major component of the ECM, is central to this process. Whereas controlled secretion of matrix-degrading enzymes (such as matrix metalloproteinases) has long been known to mediate ECM breakdown, it is becoming clear that uPARAP/Endo180 (where uPARAP stands for urokinase plasminogen activator receptor-associated protein) serves as a receptor that mediates endocytosis of collagen by several types of cells. In the liver, the stellate cells play a major role in turnover of ECM including collagens. These cells synthesize various collagens and also produce matrix metalloproteinases. In the present study, we investigated the capacity of rat hepatic stellate cells to endocytose and degrade 125I-labelled heat-denatured collagen I. It was found that the collagen is efficiently taken up and degraded by these cells. Degradation was inhibited by inhibitors of lysosomal proteases (leupeptin and E-64d) and the vacuolar proton pump (concanamycin A), indicating that it takes place in lysosomes. Furthermore, endocytosed FITC-labelled collagen was shown to reach late endocytic compartments in which it colocalized with LysoTracker (a marker of late endocytic compartments). Competition experiments showed that uPA and unlabelled collagen are capable of inhibiting binding and uptake of [125I]collagen in a dose-dependent manner. Moreover, Western-blot analysis of cell lysate (using a polyclonal rabbit human-Endo180 antiserum) revealed a single band at 180 kDa. In addition, the antiserum was capable of reducing [125I]collagen binding to the cell surface. Finally, using two primers designed from the human uPARAP/Endo180 mRNA sequence, the expression of uPARAP/Endo180 mRNA was detected by reverse transcriptase-PCR. These results together suggest that uPARAP/Endo180 mediates endocytosis of collagen in rat liver stellate cells.",
         "uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV. Identification and characterization of the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor. A targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen uptake Liver extracellular matrix in health and disease Hepatic stellate cells: unique characteristics in cell biology and phenotype. uPARAP/Endo180 is essential for cellular uptake of collagen and promotes fibroblast collagen adhesion Characterization of Sugar Binding by the Mannose Receptor Family Member, Endo180* The mannose receptor family. The C-type Lectin Receptor Endo180 Displays Internalization and Recycling Properties Distinct from Other Members of the Mannose Receptor Family* Collagen-receptor signaling in health and disease. The Urokinase Plasminogen Activator Receptor–Associated Protein/Endo180 Is Coexpressed with Its Interaction Partners Urokinase Plasminogen Activator Receptor and Matrix Metalloprotease-13 during Osteogenesis Hepatic Blood Flow Regulation by Stellate Cells in Normal and Injured Liver Extracellular Matrix Degradation and the Role of Hepatic Stellate Cells Wortmannin-sensitive trafficking steps in the endocytic pathway in rat liver endothelial cells. Effects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy. Endo180, an endocytic recycling glycoprotein related to the macrophage mannose receptor is expressed on fibroblasts, endothelial cells and macrophages and functions as a lectin receptor. Selective adhesion of macrophages to denatured forms of type I collagen is mediated by scavenger receptors. A Urokinase Receptor-associated Protein with Specific Collagen Binding Properties* The C‐type lectin superfamily in the immune system An orphan receptor tyrosine kinase family whose members serve as nonintegrin collagen receptors. The discoidin domain receptor tyrosine kinases are activated by collagen. Vascular Smooth Muscle αvβ3 Integrin Mediates Arteriolar Vasodilation in Response to RGD Peptides Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis. Cell adhesion receptors for native and denatured type I collagens and fibronectin in rabbit arterial smooth muscle cells in culture. An endothelial cell receptor for plasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of t-PA-dependent plasminogen activation. Differential expression of matrix-metalloproteinase-1 and -2 genes in normal and fibrotic human liver. Transfer of retinol-binding protein from HepG2 human hepatoma cells to cocultured rat stellate cells. Fat-storing cells as liver-specific pericytes. Spatial dynamics of agonist-stimulated intracellular calcium transients. Affinity of integrins for damaged extracellular matrix: αvβ3 binds to denatured collagen type I through RGD sites Identification of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin in collagen. Internalization of retinol-binding protein in parenchymal and stellate cells of rat liver. p180, a novel recycling transmembrane glycoprotein with restricted cell type expression The alpha 2 beta 1 integrin cell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen. Activation of cultured rat hepatic lipocytes by Kupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation of cell proliferation via induction of platelet-derived growth factor receptors. Characterization of desmin‐positive rat liver sinusoidal cells Hepatic lipocytes: the principal collagen-producing cells of normal rat liver. A radioiodinated, intracellularly trapped ligand for determining the sites of plasma protein degradation in vivo. The effect of leupeptin on intracellular digestion of asialofetuin in rat hepatocytes. The structures of the galactose-containing sugar chains of ovalbumin. HIGH-YIELD PREPARATION OF ISOLATED RAT LIVER PARENCHYMAL CELLS Distribution of Endo180 receptor and ligand in developing articular cartilage. Endocytic uptake of advanced glycation end products by mouse liver sinusoidal endothelial cells is mediated by a scavenger receptor distinct from the macrophage scavenger receptor class A. Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-α and TGF-β1 Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1. Vascular smooth muscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD peptides. Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis. Affinity of integrins for damaged extracellular matrix: alpha v beta 3 binds to denatured collagen type I through RGD sites. Expression of the gene of the α-smooth muscle-actin isoform in rat liver and in rat fat-storing (ITO) cells Acidification of the endocytic and exocytic pathways. Collagen production by rat liver fat-storing cells in primary culture. Newly administered [3H]retinol is transferred from hepatocytes to stellate cells in liver for storage. Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes), their related structure in and around the liver sinusoids, and vitamin A-storing cells in extrahepatic organs. Preparation of isolated rat liver cells. Hepatic stellate cells: morphology, function, and regulation Isolation of perisinusoidal stellate cells (vitamin A-storting cells, fat-storting cells) of the rat liver",
         "https://europepmc.org/articles/pmc1134930?pdf=render",
         "",
         "",
         "",
         "",
         "Materials",
         "Materials",
         null
        ],
        [
         "19",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "ISEcp1B-Mediated Transposition of blaCTX-M in Escherichia coli",
         "ABSTRACT Several expanded-spectrum β-lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enterobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional mechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.",
         "",
         "https://europepmc.org/articles/pmc538921?pdf=render",
         "Several expanded-spectrum -lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enter-\nobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional\nmechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.",
         "An increasing number of reports of expanded-spectrum\n-lactamases of the CTX-M type and plasmid-mediated cepha-\nlosporinases of the AmpC type in Enterobacteriaceae raise the\nquestion of their acquisition (5, 24, 26). The blaCTX-M genes\nbelonging to three out of the ﬁve known clusters (the CTX-\nM-1, -M-2, and -M-9 clusters) are associated with ISEcp1-like\ninsertion sequences (IS) (2, 5–7, 11, 12, 18, 21, 23). Neverthe-\nless, the distance that separates the -lactamase gene from\nISEcp1 varies within a given cluster of CTX-M-type genes,\nindicating that different insertion events have occurred (13,\n19). Analysis of GenBank databases reveals that some plasmid-\nencoded cephalosporinase genes, such as the blaCMY-type\ngenes, may also be associated with ISEcp1 (3, 10, 17).\nISEcp1 is an IS weakly related to other IS elements and\nbelonging to the IS1380 family (IS Database HomePage\n[http://www-is.biotoul.fr/page-is.html]) (8). Nothing is known\nabout the mobilization properties of ISEcp1. Recently, we\ncharacterized ISEcp1B associated with a blaCTX-M-19 gene in a\nKlebsiella pneumoniae ILT-3 clinical\nisolate from Vietnam\n(21). ISEcp1B differs from ISEcp1 (GenBank accession no.\nAJ242809) by three nucleotide substitutions. Their inverted\nrepeat (IR) sequences are identical, and their transposases\ndiffer by a single amino acid change (18). Since ISEcp1-like\nelements are located upstream of several -lactamase genes,\nanalysis of the variable sequences separating these IS elements\nfrom the initiation codons of these genes allowed us to deter-\nmine its boundaries.\nRecently, it has been shown that ISEcp1B brought promoter\nsequences for high-level expression of the blaCTX-M-14/-18,\n-lactamase genes that belong to\nblaCTX-M-17, and blaCTX-M-19\nthe same blaCTX-M-9 subgroup (5, 6, 18) and might be also\nresponsible for their mobilization. The aim of this study was to\nanalyze the mobilization process mediated by ISEcp1B.\nISEcp1B possesses two imperfect IRs likely made of 14 bp,\npreviously estimated to be 18 bp (18), with 12 of these 14 bp\nbeing complementary (Table 1). A detailed analysis of a ca.\n9.5-kb DNA fragment from natural plasmid pILT-3 from K.\npneumoniae ILT-3 revealed that the gene encoding ORF1 in\ntransposon Tn1721 was interrupted by a 4.8-kb potential trans-\nposon bracketed on the left side by the left IR (IRL) of\nISEcp1B and on the right side by an imperfect right IR (IRR)\nnamed IRR1 made of a 14-bp sequence belonging to a gene\nencoding a putative iron transport protein (Fig. 1) (18). This\npotential transposon comprises ISEcp1B, blaCTX-M-19, IS903D,\nand part of the gene encoding an iron transport protein. We\nhave hypothesized that ISEcp1B could mobilize adjacent se-\nquences by transposition.\nSeveral transposition experiments were performed to deter-\nmine (i) the mobility of this potential transposon, (ii) the role\nof ISEcp1B in mobilizing the adjacent -lactamase gene, and\n(iii) the role of IS903D. Different recombinant plasmids were\nconstructed by inserting PCR products that encompassed ei-\nther one or both ISs that bracketed the blaCTX-M-19 gene in\nplasmid pILT-3 into the EcoRV site of plasmid pACYC184\n(Table 2) and then transformed into Escherichia coli reference\nstrain DH10B. Recombinant plasmids were constructed with\nthe pACYC184 backbone to prevent overexpression of the\nISEcp1B transposase. Plasmid pC-1 contained the sequences\nof the potential transposon identiﬁed in plasmid pILT-3,\nwhereas the following constructs lacked the right part of it\n(Fig. 1). Plasmid pC-2 possessed the ISEcp1B sequence but\nlacked part of IS903D, and plasmid pC-3 possessed the entire\nIS903D sequence and a truncated ISEcp1B transposase gene\n(Fig. 1; Table 2). E. coli RR1023 (pOX38) was transformed by\nelectroporation with these recombinant plasmids (Table 2) (9).\nPlasmid pOX38 is a self-conjugative and IS-free plasmid.\nTransposition events were searched between inserts of several\nrecombinant plasmids and recipient plasmid pOX38 after 24 h\nof growth in Trypticase soy broth. Transfer of the recombinant\nplasmids with the pOX38 backbone into streptomycin-resistant\nE. coli reference strain DH10B (Table 2) was then performed by\nconjugation as described previously (20), and transconjugants\nwere selected on agar plates containing 100 g of amoxicillin per\nml (plasmid resistance marker) and 20 g of streptomycin per ml\n(chromosomal marker). Several amoxicillin-resistant transconju-\ngants were extracted and sequenced in part.\nSusceptibility to chloramphenicol was always observed in\ntransconjugant strains, thus ruling out full integration (chlor-\namphenicol resistance) of the recombinant plasmids derived\nfrom pACYC184 into plasmid pOX38. Detection of ISEcp1B\nand IS903D was performed using two sets of primers (primer\nISEcp1A [Table 2] in combination with primer ISEcp1B [5-T\nTTCCGCAGCACCGTTTGC-3] and primer IS903A [5-CA\nIRL of ISEcp1B\nDeduced perfect IRR of ISEcp1B\nIRR of ISEcp1B\nIRR1 at the potential transposon end of pILT-3\nIRR2 in plasmid pACYC184\nIRR3 in plasmid pBK-CMV\nIRR4 in the tnpA gene of IS903D\nIRR5 in the iron gene of pILT-3\nIRR6 in the iron gene of pILT-3\nIRR7 in the iron gene of pILT-3\nIRR8 in the iron gene of pILT-3\nCATGAAATCATCTGCGC-3] in combination with primer\nIS903B [Table 2], respectively).\ntransconjugants were not obtained\nwhen E. coli RR1023 (pC-3) lacking ISEcp1B (Fig. 1) was used\nas a donor, suggesting that the IS903D element located down-\nstream of the -lactamase gene did not mobilize this latter\ngene. On the other hand, amoxicillin-resistant transconjugants\nwere obtained when E. coli RR1023 (pC-1) and E. coli RR1023\n(pC-2) were used as donors with a transposition efﬁciency of\nca. 107 per donor. These results indicated that ISEcp1B was\nessential for movement of the blaCTX-M-19 gene.\nUsing E. coli RR1023 (pC-1) as a donor, all amoxicillin-\nresistant transconjugants except one contained ISEcp1B and\nIS903D elements, with the blaCTX-M-19 gene and DNA se-\nquences being located on the right part of IS903D in plasmid\npILT-3 (Fig. 1). Surprisingly, none of the 11 transconjugants\nthat were sequenced had the entire potential transposon iden-\ntiﬁed in natural plasmid pILT-3. Nevertheless, the transposi-\ntion mechanism mediated by ISEcp1B was evident, since 5-bp\nduplications were observed at the insertion sites in plasmid\npOX38 (see below). These data suggested that the IRR se-\nquences recognized by ISEcp1B were located inside the poten-\ntial transposon. Analysis of the boundaries of the transposed\nfragments identiﬁed sequences, similar to that of the IRR of\nISEcp1B and deﬁned as IRR-like sequences, that had been\nrecognized as such by the transposase of ISEcp1B during a\nmobilization process.\nFive different IRRs (IRR4 to IRR8) (Table 1) were identi-\nﬁed in ﬁve amoxicillin-resistant transconjugants that were an-\nalyzed, four of them being located in the gene encoding the\niron transport protein and one of them being located in the\ntnpA gene of IS903D (Table 2; Fig. 1). The number of identical\nbase pairs among the sequence deﬁned as IRR varied from\nthree to six, corresponding to less than 50%.\nAnalysis of E. coli RR1023 (pOX38) transconjugants resis-\ntant to amoxicillin, obtained with plasmid pC-2 as a donor, and\nthus lacking the sequence used as IRR in pILT-3, also showed\nthat transposition had occurred using nearby sequences of\npACYC184. The IRR-like sequences bracketing the trans-\nposed fragment in these transconjugants remained unchanged.\nThey shared 9 of 14 bp of the perfect IRR sequence (Table 1)\nand corresponded to a segment belonging to the cloning vec-\ntor. In order to rule out\nthe speciﬁc sequence of\nthat\npACYC184 may play a role in this transposition process, the\nsame inserted PCR product of pC-2 was cloned into the SmaI\nsite of pBK-CMV, giving rise to pC-2K (Table 2; Fig. 1).\nAnalysis of several amoxicillin-resistant transconjugants ob-\ntained with plasmid pC-2K as a donor also showed that trans-\nposition had occurred by involving sequences of pBK-CMV.\nThe IRR-like sequences bracketing the transposed fragment in\nthese transconjugants were always the same and shared 9 of\nthe 14 bp of a perfect IRR (Table 1). These results indicated\nthat sequences of pACYC184 (IRR2) or of pBK-CMV (IRR3)\nmight be used as IRRs by ISEcp1B, but further repeated ex-\nperiments are needed to assess this hypothesis. The reason why\nthe IRR1 of the potential transposon of pILT-3 was not the\nboundary of the transposed fragments in these in vitro exper-\niments remains intriguing. One explanation could be that\nIRR2 and IRR3 share greater nucleotide identity in their 3\nextremities (7 and 8, respectively, of last 10 bp) to a perfect\nIRR than IRR1 (only 6 of last 10 bp).\nNo consensus sequence could be determined by comparing\nthese 14-bp-long IRR sequences of recombinant plasmids and\nthe IRR sequences of ISEcp1B in pILT-3 (Table 1). Neverthe-\nless, a guanosine residue located at the 3 end of these IRRs\nwas always found, likely indicating that this nucleotide was\nnecessary in the transposition process. In addition, the last two\nnucleotides of these IRR sequences were either GG or CG.\nThese data indicated that ISEcp1B was involved in a transpo-\nsition process with weakly related IRRs.\nAnalysis of the insertion sites of ISEcp1B-mediated trans-\nposition revealed variable sequences in recombinant plasmids\n(AAGAA, TATAA, TATTA, GCTGA, ATATC, TAATA,\nTTTAC, AATGA, TTATA, ATAAG, and TGATT). No con-\nsensus sequence was identiﬁed among the 5-bp duplicated\nsites, whereas an AT-rich content that may target ISEcp1-\nmediated transposition as observed for IS1 was noticed (16,\n27). In silico analysis of six ISEcp1-like sequences associated\nwith blaCTX-M-like or blaCMY-like -lactamase genes available\nin the GenBank databases did not identify target site duplica-\ntions just bracketing the insertion sites of ISEcp1-like se-\nlikely indicates that after insertion,\nquences. This result\nISEcp1-like elements may be responsible for mobilization of\nthe adjacent sequences. In addition, this in silico analysis con-\nﬁrmed that the 5-bp sequences adjacent to the IRLs and IRRs\nof ISEcp1 elements were always AT rich.\nISEcp1B may use a wide range of DNA sequences as IRRs\nduring a mobilization process that involves its adjacent se-\nquences, therefore explaining its ability to mobilize structurally\nunrelated -lactamase genes. Similar observations have been\nmade for IS1247 of Xanthobacter autotrophicus, which also\nbelongs to the IS1380 family (25). A single copy of IS1247 may\nmobilize sequences located at its right-end extremity in what is\nconsidered a one-ended transposition process since only a sin-\ngle IS is involved.\nThese results may also be compared to those obtained with\nother IS elements, such as IS91 and IS911, that are responsible\nfor mobilization of adjacent sequences by a one-ended trans-\nposition mechanism (4, 22). In addition, transposition of other\nelements belonging to the Tn3 family, such as Tn21, generates\nvariable endpoints as a consequence of one-ended transposi-\ntion (1, 15). This peculiar process is the consequence of rec-\nognition of several different IR sequences by the Tn21 trans-\nposase, generating novel joint molecules (14). However, in our\ncase, some homology between the different IRRs used by\nISEcp1B is noticed, whereas one-ended transposition (for ex-\nample, with Tn21) requires one ﬁxed end and another random\none.\nIn the present study, we have demonstrated the ability for\n-lactamase gene. In addi-\nISEcp1B to mobilize the blaCTX-M\ntion, the IS903D element present in the potential transposon in\npILT-3 had no role in the mobilization process.\nIn conclusion, it remains to be determined why the emerging\nexpanded-spectrum -lactamases of the CTX-M type that are\nstructurally related to several chromosomally located -lacta-\nmase genes of enterobacterial species are associated speciﬁ-\ncally with ISEcp1-like sequences.",
         "",
         "",
         "",
         "Several expanded-spectrum -lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enter-\nobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional\nmechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.\nAn increasing number of reports of expanded-spectrum\n-lactamases of the CTX-M type and plasmid-mediated cepha-\nlosporinases of the AmpC type in Enterobacteriaceae raise the\nquestion of their acquisition (5, 24, 26). The blaCTX-M genes\nbelonging to three out of the ﬁve known clusters (the CTX-\nM-1, -M-2, and -M-9 clusters) are associated with ISEcp1-like\ninsertion sequences (IS) (2, 5–7, 11, 12, 18, 21, 23). Neverthe-\nless, the distance that separates the -lactamase gene from\nISEcp1 varies within a given cluster of CTX-M-type genes,\nindicating that different insertion events have occurred (13,\n19). Analysis of GenBank databases reveals that some plasmid-\nencoded cephalosporinase genes, such as the blaCMY-type\ngenes, may also be associated with ISEcp1 (3, 10, 17).\nISEcp1 is an IS weakly related to other IS elements and\nbelonging to the IS1380 family (IS Database HomePage\n[http://www-is.biotoul.fr/page-is.html]) (8). Nothing is known\nabout the mobilization properties of ISEcp1. Recently, we\ncharacterized ISEcp1B associated with a blaCTX-M-19 gene in a\nKlebsiella pneumoniae ILT-3 clinical\nisolate from Vietnam\n(21). ISEcp1B differs from ISEcp1 (GenBank accession no.\nAJ242809) by three nucleotide substitutions. Their inverted\nrepeat (IR) sequences are identical, and their transposases\ndiffer by a single amino acid change (18). Since ISEcp1-like\nelements are located upstream of several -lactamase genes,\nanalysis of the variable sequences separating these IS elements\nfrom the initiation codons of these genes allowed us to deter-\nmine its boundaries.\nRecently, it has been shown that ISEcp1B brought promoter\nsequences for high-level expression of the blaCTX-M-14/-18,\n-lactamase genes that belong to\nblaCTX-M-17, and blaCTX-M-19\nthe same blaCTX-M-9 subgroup (5, 6, 18) and might be also\nresponsible for their mobilization. The aim of this study was to\nanalyze the mobilization process mediated by ISEcp1B.\nISEcp1B possesses two imperfect IRs likely made of 14 bp,\npreviously estimated to be 18 bp (18), with 12 of these 14 bp\nbeing complementary (Table 1). A detailed analysis of a ca.\n9.5-kb DNA fragment from natural plasmid pILT-3 from K.\npneumoniae ILT-3 revealed that the gene encoding ORF1 in\ntransposon Tn1721 was interrupted by a 4.8-kb potential trans-\nposon bracketed on the left side by the left IR (IRL) of\nISEcp1B and on the right side by an imperfect right IR (IRR)\nnamed IRR1 made of a 14-bp sequence belonging to a gene\nencoding a putative iron transport protein (Fig. 1) (18). This\npotential transposon comprises ISEcp1B, blaCTX-M-19, IS903D,\nand part of the gene encoding an iron transport protein. We\nhave hypothesized that ISEcp1B could mobilize adjacent se-\nquences by transposition.\nSeveral transposition experiments were performed to deter-\nmine (i) the mobility of this potential transposon, (ii) the role\nof ISEcp1B in mobilizing the adjacent -lactamase gene, and\n(iii) the role of IS903D. Different recombinant plasmids were\nconstructed by inserting PCR products that encompassed ei-\nther one or both ISs that bracketed the blaCTX-M-19 gene in\nplasmid pILT-3 into the EcoRV site of plasmid pACYC184\n(Table 2) and then transformed into Escherichia coli reference\nstrain DH10B. Recombinant plasmids were constructed with\nthe pACYC184 backbone to prevent overexpression of the\nISEcp1B transposase. Plasmid pC-1 contained the sequences\nof the potential transposon identiﬁed in plasmid pILT-3,\nwhereas the following constructs lacked the right part of it\n(Fig. 1). Plasmid pC-2 possessed the ISEcp1B sequence but\nlacked part of IS903D, and plasmid pC-3 possessed the entire\nIS903D sequence and a truncated ISEcp1B transposase gene\n(Fig. 1; Table 2). E. coli RR1023 (pOX38) was transformed by\nelectroporation with these recombinant plasmids (Table 2) (9).\nPlasmid pOX38 is a self-conjugative and IS-free plasmid.\nTransposition events were searched between inserts of several\nrecombinant plasmids and recipient plasmid pOX38 after 24 h\nof growth in Trypticase soy broth. Transfer of the recombinant\nplasmids with the pOX38 backbone into streptomycin-resistant\nE. coli reference strain DH10B (Table 2) was then performed by\nconjugation as described previously (20), and transconjugants\nwere selected on agar plates containing 100 g of amoxicillin per\nml (plasmid resistance marker) and 20 g of streptomycin per ml\n(chromosomal marker). Several amoxicillin-resistant transconju-\ngants were extracted and sequenced in part.\nSusceptibility to chloramphenicol was always observed in\ntransconjugant strains, thus ruling out full integration (chlor-\namphenicol resistance) of the recombinant plasmids derived\nfrom pACYC184 into plasmid pOX38. Detection of ISEcp1B\nand IS903D was performed using two sets of primers (primer\nISEcp1A [Table 2] in combination with primer ISEcp1B [5-T\nTTCCGCAGCACCGTTTGC-3] and primer IS903A [5-CA\nIRL of ISEcp1B\nDeduced perfect IRR of ISEcp1B\nIRR of ISEcp1B\nIRR1 at the potential transposon end of pILT-3\nIRR2 in plasmid pACYC184\nIRR3 in plasmid pBK-CMV\nIRR4 in the tnpA gene of IS903D\nIRR5 in the iron gene of pILT-3\nIRR6 in the iron gene of pILT-3\nIRR7 in the iron gene of pILT-3\nIRR8 in the iron gene of pILT-3\nCATGAAATCATCTGCGC-3] in combination with primer\nIS903B [Table 2], respectively).\ntransconjugants were not obtained\nwhen E. coli RR1023 (pC-3) lacking ISEcp1B (Fig. 1) was used\nas a donor, suggesting that the IS903D element located down-\nstream of the -lactamase gene did not mobilize this latter\ngene. On the other hand, amoxicillin-resistant transconjugants\nwere obtained when E. coli RR1023 (pC-1) and E. coli RR1023\n(pC-2) were used as donors with a transposition efﬁciency of\nca. 107 per donor. These results indicated that ISEcp1B was\nessential for movement of the blaCTX-M-19 gene.\nUsing E. coli RR1023 (pC-1) as a donor, all amoxicillin-\nresistant transconjugants except one contained ISEcp1B and\nIS903D elements, with the blaCTX-M-19 gene and DNA se-\nquences being located on the right part of IS903D in plasmid\npILT-3 (Fig. 1). Surprisingly, none of the 11 transconjugants\nthat were sequenced had the entire potential transposon iden-\ntiﬁed in natural plasmid pILT-3. Nevertheless, the transposi-\ntion mechanism mediated by ISEcp1B was evident, since 5-bp\nduplications were observed at the insertion sites in plasmid\npOX38 (see below). These data suggested that the IRR se-\nquences recognized by ISEcp1B were located inside the poten-\ntial transposon. Analysis of the boundaries of the transposed\nfragments identiﬁed sequences, similar to that of the IRR of\nISEcp1B and deﬁned as IRR-like sequences, that had been\nrecognized as such by the transposase of ISEcp1B during a\nmobilization process.\nFive different IRRs (IRR4 to IRR8) (Table 1) were identi-\nﬁed in ﬁve amoxicillin-resistant transconjugants that were an-\nalyzed, four of them being located in the gene encoding the\niron transport protein and one of them being located in the\ntnpA gene of IS903D (Table 2; Fig. 1). The number of identical\nbase pairs among the sequence deﬁned as IRR varied from\nthree to six, corresponding to less than 50%.\nAnalysis of E. coli RR1023 (pOX38) transconjugants resis-\ntant to amoxicillin, obtained with plasmid pC-2 as a donor, and\nthus lacking the sequence used as IRR in pILT-3, also showed\nthat transposition had occurred using nearby sequences of\npACYC184. The IRR-like sequences bracketing the trans-\nposed fragment in these transconjugants remained unchanged.\nThey shared 9 of 14 bp of the perfect IRR sequence (Table 1)\nand corresponded to a segment belonging to the cloning vec-\ntor. In order to rule out\nthe speciﬁc sequence of\nthat\npACYC184 may play a role in this transposition process, the\nsame inserted PCR product of pC-2 was cloned into the SmaI\nsite of pBK-CMV, giving rise to pC-2K (Table 2; Fig. 1).\nAnalysis of several amoxicillin-resistant transconjugants ob-\ntained with plasmid pC-2K as a donor also showed that trans-\nposition had occurred by involving sequences of pBK-CMV.\nThe IRR-like sequences bracketing the transposed fragment in\nthese transconjugants were always the same and shared 9 of\nthe 14 bp of a perfect IRR (Table 1). These results indicated\nthat sequences of pACYC184 (IRR2) or of pBK-CMV (IRR3)\nmight be used as IRRs by ISEcp1B, but further repeated ex-\nperiments are needed to assess this hypothesis. The reason why\nthe IRR1 of the potential transposon of pILT-3 was not the\nboundary of the transposed fragments in these in vitro exper-\niments remains intriguing. One explanation could be that\nIRR2 and IRR3 share greater nucleotide identity in their 3\nextremities (7 and 8, respectively, of last 10 bp) to a perfect\nIRR than IRR1 (only 6 of last 10 bp).\nNo consensus sequence could be determined by comparing\nthese 14-bp-long IRR sequences of recombinant plasmids and\nthe IRR sequences of ISEcp1B in pILT-3 (Table 1). Neverthe-\nless, a guanosine residue located at the 3 end of these IRRs\nwas always found, likely indicating that this nucleotide was\nnecessary in the transposition process. In addition, the last two\nnucleotides of these IRR sequences were either GG or CG.\nThese data indicated that ISEcp1B was involved in a transpo-\nsition process with weakly related IRRs.\nAnalysis of the insertion sites of ISEcp1B-mediated trans-\nposition revealed variable sequences in recombinant plasmids\n(AAGAA, TATAA, TATTA, GCTGA, ATATC, TAATA,\nTTTAC, AATGA, TTATA, ATAAG, and TGATT). No con-\nsensus sequence was identiﬁed among the 5-bp duplicated\nsites, whereas an AT-rich content that may target ISEcp1-\nmediated transposition as observed for IS1 was noticed (16,\n27). In silico analysis of six ISEcp1-like sequences associated\nwith blaCTX-M-like or blaCMY-like -lactamase genes available\nin the GenBank databases did not identify target site duplica-\ntions just bracketing the insertion sites of ISEcp1-like se-\nlikely indicates that after insertion,\nquences. This result\nISEcp1-like elements may be responsible for mobilization of\nthe adjacent sequences. In addition, this in silico analysis con-\nﬁrmed that the 5-bp sequences adjacent to the IRLs and IRRs\nof ISEcp1 elements were always AT rich.\nISEcp1B may use a wide range of DNA sequences as IRRs\nduring a mobilization process that involves its adjacent se-\nquences, therefore explaining its ability to mobilize structurally\nunrelated -lactamase genes. Similar observations have been\nmade for IS1247 of Xanthobacter autotrophicus, which also\nbelongs to the IS1380 family (25). A single copy of IS1247 may\nmobilize sequences located at its right-end extremity in what is\nconsidered a one-ended transposition process since only a sin-\ngle IS is involved.\nThese results may also be compared to those obtained with\nother IS elements, such as IS91 and IS911, that are responsible\nfor mobilization of adjacent sequences by a one-ended trans-\nposition mechanism (4, 22). In addition, transposition of other\nelements belonging to the Tn3 family, such as Tn21, generates\nvariable endpoints as a consequence of one-ended transposi-\ntion (1, 15). This peculiar process is the consequence of rec-\nognition of several different IR sequences by the Tn21 trans-\nposase, generating novel joint molecules (14). However, in our\ncase, some homology between the different IRRs used by\nISEcp1B is noticed, whereas one-ended transposition (for ex-\nample, with Tn21) requires one ﬁxed end and another random\none.\nIn the present study, we have demonstrated the ability for\n-lactamase gene. In addi-\nISEcp1B to mobilize the blaCTX-M\ntion, the IS903D element present in the potential transposon in\npILT-3 had no role in the mobilization process.\nIn conclusion, it remains to be determined why the emerging\nexpanded-spectrum -lactamases of the CTX-M type that are\nstructurally related to several chromosomally located -lacta-\nmase genes of enterobacterial species are associated speciﬁ-\ncally with ISEcp1-like sequences.",
         null
        ],
        [
         "20",
         "0018ece872bb78b06eb26ea35d5ed27f53702ce6",
         "Protein function prediction using the Protein Link EXplorer (PLEX)",
         "UNLABELLED\nWe introduce the Protein Link EXplorer (PLEX), a web-based environment that allows the construction of a phylogenetic profile for any given amino acid sequence, and its comparison with profiles of approximately 350,000 predicted genes from 89 genomes, as a means of interactively identifying functionally linked genes and predicting protein function. PLEX can be searched iteratively and also enables searches for chromosomal gene neighbors and Rosetta Stone linkages. PLEX search results are accompanied by quantitative estimates of linkage confidence, enabling users to take advantage of coinheritance, operon and gene fusion-based methods for inferring gene function and reconstructing cellular systems and pathways.\n\n\nAVAILABILITY\nhttp://bioinformatics.icmb.utexas.edu/plex",
         "",
         "https://academic.oup.com/bioinformatics/article-pdf/21/10/2558/48972005/bioinformatics_21_10_2558.pdf",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "21",
         "001a3ef673ee3e7703c7db597386a2162a2dc402",
         "How and When Selection Experiments Might Actually be Useful1",
         "Abstract Laboratory natural selection and artificial selection are vital tools for addressing specific questions about evolutionary patterns of variation. Laboratory natural selection can illuminate whether a putative selective agent is capable of generating long-term, sustained changes in individual traits and suites of traits. Artificial selection is the essential tool for understanding the general evolvability of traits and the extent to which genetic correlations constrain evolution. We review the contexts in which each type of experiment seems capable of offering key insights into important evolutionary issues. We also discuss theoretical and methodological considerations that play critical roles in designing selection experiments that are relevant to evolutionary patterns of trait variation. In particular, we focus on the critical role of selection intensity and the consequences of experiments with different intensities. While selection experiments are not practical in many cases, sophisticated selection experiments—designed with careful consideration of the theory of selection—should be taken beyond model organisms and used in well-chosen natural systems to understand natural patterns of variation.",
         "",
         "https://academic.oup.com/icb/article-pdf/45/3/391/1674343/i1540-7063-045-03-0391.pdf",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "22",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "Phylogenetic relationships of Irkut and West Caucasian bat viruses within the Lyssavirus genus and suggested quantitative criteria based on the N gene sequence for lyssavirus genotype definition.",
         "None",
         "Novel Lyssaviruses Isolated from Bats in Russia Bat lyssaviruses (Aravan and Khujand) from Central Asia: phylogenetic relationships according to N, P and G gene sequences. A comparison of DNA vaccines for the rabies-related virus, Mokola. New Lyssavirus Genotype from the Lesser Mouse-eared Bat (Myotis blythi), Kyrghyzstan A molecular epidemiological study of Australian bat lyssavirus. Competition Commission inquiry into POMs. Characterisation of an Australian bat lyssavirus variant isolated from an insectivorous bat. Phylogenetic comparison of the genus Lyssavirus using distal coding sequences of the glycoprotein and nucleoprotein genes Lyssavirus P gene characterisation provides insights into the phylogeny of the genus and identifies structural similarities and diversity within the encoded phosphoprotein. Increased positive selection pressure in persistent (SSPE) versus acute measles virus infections. European Bat Lyssavirus Infection in Spanish Bat Populations Serologic Evidence of Lyssavirus Infections among Bats, the Philippines Genetic constraints and the adaptive evolution of rabies virus in nature. Expression of the B-cell and T-cell epitopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG and induction of an humoral response in mice. MEGA2: molecular evolutionary genetics analysis software Extensive Attenuation of Rabies Virus by Simultaneously Modifying the Dynein Light Chain Binding Site in the P Protein and Replacing Arg333 in the G Protein Molecular basis for the interaction between rabies virus phosphoprotein P and the dynein light chain LC8: dissociation of dynein-binding properties and transcriptional functionality of P. Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains Host Switching in Lyssavirus History from the Chiroptera to the Carnivora Orders Evidence of Two Lyssavirus Phylogroups with Distinct Pathogenicity and Immunogenicity Antigenic and genetic divergence of rabies viruses from bat species indigenous to Canada. Codon-substitution models for heterogeneous selection pressure at amino acid sites. Sequencing and position analysis of the glycoprotein gene of four Chinese rabies viruses Ecology and evolution of rabies virus in Europe. Phylogeographic patterns exhibited by Ontario rabies virus variants Chimeric Lyssavirus Glycoproteins with Increased Immunological Potential Multiple sequence alignment with Clustal X. Rabies virus quasispecies: implications for pathogenesis. DNA-based immunization for exploring the enlargement of immunological cross-reactivity against the lyssaviruses. PAML: a program package for phylogenetic analysis by maximum likelihood Evolution of European bat lyssaviruses. The complete Mokola virus genome sequence: structure of the RNA-dependent RNA polymerase. A Biographical History of Veterinary Pathology Ten-year survey of British bats for the existence of rabies Encephalitis caused by a Lyssavirus in fruit bats in Australia. Characterization of a unique variant of bat rabies virus responsible for newly emerging human cases in North America. The design of strain-specific polymerase chain reactions for discrimination of the racoon rabies virus strain from indigenous rabies viruses of Ontario. Genetic polymorphism in the rabies virus nucleoprotein gene. Molecular epidemiology of lyssaviruses: focus on the glycoprotein and pseudogenes. Molecular diversity of the Lyssavirus genus. Isolation and characterization of 115 street rabies virus isolates from Ethiopia by using monoclonal antibodies: identification of 2 isolates as Mokola and Lagos bat viruses. [A lyssavirus with an unusual antigenic structure isolated from a bat in southern Kyrgyzstan]. Rapid sequence evolution of street rabies glycoprotein is related to the highly heterogeneous nature of the viral population. PCR technique as an alternative method for diagnosis and molecular epidemiology of rabies virus. T helper cell epitope of rabies virus nucleoprotein defined by tri‐ and tetrapeptides Molecular cloning and complete nucleotide sequence of the attenuated rabies virus SAD B19. Rabies-related Yuli virus; identification with a panel of monoclonal antibodies. Efficacy of rabies vaccines against Duvenhage virus isolated from European house bats (Eptesicus serotinus), classic rabies and rabies-related viruses. Immunity against the European bat rabies (Duvenhage) virus induced by rabies vaccines: an experimental study in mice. Localization and immunological characterization of antigenic domains of the rabies virus internal N and NS proteins. Walking along the rabies genome: is the large G-L intergenic region a remnant gene? HUMAN RABIES OF BAT ORIGIN IN EUROPE Rabies in the Tropics Rabies-related viruses. Rabies virus infection of a flying fox bat, Pteropus policephalus in Chandigarh, Northern India. Giulio Alfredo Maccacaro. Experimental infection of dogs and monkeys with two rabies serogroup viruses, Lagos bat and Mokola (IbAn 27377): clinical, serologic, virologic, and fluorescent-antibody studies. An unusual case of human rabies thought to be of chiropteran origin. Two African Viruses Serologically and Morphologically Related to Rabies Virus Isolation of Rabies Virus from Fruit Bats in Thailand Isolation of a virus from Nigerian fruit bats. New Cases of Mokola Virus Infection in South Africa: A Genotypic Comparison of Southern African Virus Isolates BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT MODELTEST: testing the model of DNA substitution Immunoprotection by rabies virus nucleoprotein. Nucleotide and deduced amino acid sequences of the nominal nonstructural phosphoprotein of the ERA, PM and CVS-11 strains of rabies virus. Rabies and Rabies-related viruses in Zimbabwe historical, virological and ecological aspects. Rabies-Related Viruses Characterization of an antigenic determinant of the glycoprotein that correlates with pathogenicity of rabies virus. Antigenic characterization of the bangui strain (ANCB-672d) of lagos bat virus",
         "https://zenodo.org/record/1259441/files/article.pdf",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "23",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         "Neuronal Signals in the Monkey Basolateral Amygdala during Reward Schedules",
         "The amygdala is critical for connecting emotional reactions with environmental events. We recorded neurons from the basolateral complex of two monkeys while they performed visually cued schedules of sequential color discrimination trials, with both valid and random cues. When the cues were valid, the visual cue, which was present throughout each trial, indicated how many trials remained to be successfully completed before a reward. Seventy-six percent of recorded neurons showed response selectivity, with the selectivity depending on some aspects of the current schedule. After a reward, when the monkeys knew that the upcoming cue would be valid, 88 of 246 (36%) neurons responded between schedules, seemingly anticipating the receiving information about the upcoming schedule length. When the cue appeared, 102 of 246 (41%) neurons became selective, at this point encoding information about whether the current trial was the only trial required or how many more trials are needed to obtain a reward. These cue-related responses had a median latency of 120 ms (just between the latencies in inferior temporal visual area TE and perirhinal cortex). When the monkey was releasing a touch bar to complete the trial correctly, 71 of 246 (29%) neurons responded, with responses in the rewarded trials being similar no matter which schedule was ending, thus being sensitive to the reward contingency. Finally, 39 of 246 (16%) neurons responded around the reward. We suggest that basolateral amygdala, by anticipating and then delineating the schedule and representing reward contingency, provide contextual information that is important for adjusting motivational level as a function of immediate behavior goals.",
         "",
         "https://www.jneurosci.org/content/jneuro/25/48/11071.full.pdf",
         "",
         "1. Introduction",
         "",
         "4. Discussion",
         "",
         "1. Introduction\n4. Discussion",
         null
        ],
        [
         "24",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "The role of microRNA genes in papillary thyroid carcinoma.",
         "Apart from alterations in the RET/PTC-RAS-BRAF pathway, comparatively little is known about the genetics of papillary thyroid carcinoma (PTC). We show that numerous microRNAs (miRNAs) are transcriptionally up-regulated in PTC tumors compared with unaffected thyroid tissue. A set of five miRNAs, including the three most up-regulated ones (miR-221, -222, and -146), distinguished unequivocally between PTC and normal thyroid. Additionally, miR-221 was up-regulated in unaffected thyroid tissue in several PTC patients, presumably an early event in carcinogenesis. Tumors in which the up-regulation (11- to 19-fold) of miR-221, -222, and -146 was strongest showed dramatic loss of KIT transcript and Kit protein. In 5 of 10 such cases, this down expression was associated with germline single-nucleotide changes in the two recognition sequences in KIT for these miRNAs. We conclude that up-regulation of several miRs and regulation of KIT are involved in PTC pathogenesis, and that sequence changes in genes targeted by miRNAs can contribute to their regulation.",
         "MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation. MicroRNA gene expression deregulation in human breast cancer. MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells. miRNAs, Cancer, and Stem Cell Division MicroRNA expression profiles classify human cancers Cancer genomics: Small RNAs with big impacts A microRNA polycistron as a potential human oncogene Small regulatory RNAs in mammals. Combinatorial microRNA target predictions The RET/PTC-RAS-BRAF linear signaling cascade mediates the motile and mitogenic phenotype of thyroid cancer cells. MicroRNAs: a developing story. RAS Is Regulated by the let-7 MicroRNA Family Accumulation of miR-155 and BIC RNA in human B cell lymphomas. A new mechanism of BRAF activation in human thyroid papillary carcinomas. Detection of Overexpressed and Phosphorylated Wild-Type Kit Receptor in Surgical Specimens of Small Cell Lung Cancer Kit as a human oncogenic tyrosine kinase. Oncogenic protein tyrosine kinases Human MicroRNA Targets Fast and effective prediction of microRNA/target duplexes. Genetic analysis of genome-wide variation in human gene expression MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias. Annual report to the nation on the status of cancer, 1975–2001, with a special feature regarding survival An oligonucleotide microchip for genome-wide microRNA profiling in human and mouse tissues. Reduced Expression of the let-7 MicroRNAs in Human Lung Cancers in Association with Shortened Postoperative Survival MicroRNAs Genomics, Biogenesis, Mechanism, and Function Prediction of Mammalian MicroRNA Targets BRAF mutations in thyroid tumors are restricted to papillary carcinomas and anaplastic or poorly differentiated carcinomas arising from papillary carcinomas. Reduced accumulation of specific microRNAs in colorectal neoplasia. Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma BRAF mutation in papillary thyroid carcinoma. High prevalence of BRAF mutations in thyroid cancer: genetic evidence for constitutive activation of the RET/PTC-RAS-BRAF signaling pathway in papillary thyroid carcinoma. Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia Environmental and heritable causes of cancer among 9.6 million individuals in the Swedish family‐cancer database Gene expression in papillary thyroid carcinoma reveals highly consistent profiles Localization of a susceptibility gene for familial nonmedullary thyroid carcinoma to chromosome 2q21. Papillary thyroid carcinoma associated with papillary renal neoplasia: genetic linkage analysis of a distinct heritable tumor syndrome. The biology of stem cell factor and its receptor C-kit. A gene predisposing to familial thyroid tumors with cell oxyphilia maps to chromosome 19p13.2. Paraffin section detection of the c-kit gene product (CD117) in human tissues: value in the diagnosis of mast cell disorders. c-Kit proto-oncogene is more likely to lose expression in differentiated thyroid carcinoma than three thyroid-specific genes: thyroid peroxidase, thyroglobulin, and thyroid stimulating hormone receptor. Transformation of thyroid epithelium is associated with loss of c-kit receptor. Systematic population-based assessment of cancer risk in first-degree relatives of cancer probands. Expression of c-kit receptor in normal and transformed human nonlymphoid tissues. RNA silencing. M  ED  IC  A  L  SC  IE  N  CE Loss of heterozygosity in papillary and follicular thyroid carcinoma: a mini review. de la Chapelle, A. Age at diagnosis Gender Race Diagnosis",
         "https://europepmc.org/articles/pmc1323209?pdf=render",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "25",
         "001ce484190e733e0d1dee181f9ee37fd0b15581",
         "Lethality of chlorine, chlorine dioxide, and a commercial fruit and vegetable sanitizer to vegetative cells and spores of Bacillus cereus and spores of Bacillus thuringiensis",
         "None",
         "Sanitizers: Halogens, Surface-Active Agents, and Peroxides Mechanisms of killing of Bacillus subtilis spores by hypochlorite and chlorine dioxide Bacillus anthracis: current knowledge in relation to contamination of food. Peracetic acid as an alternative wastewater disinfectant to chlorine dioxide Mechanisms of killing spores of Bacillus subtilis by acid, alkali and ethanol Differential efficacy of a chlorine dioxide‐containing sanitizer against single species and binary biofilms of a dairy‐associated Bacillus cereus and a Pseudomonas fluorescens isolate Reduction of listeria monocytogenes on green peppers (Capsicum annuum L.) by gaseous and aqueous chlorine dioxide and water washing and its growth at 7 degrees C. A study on the effects of high pressure and heat on Bacillus subtilis spores at low pH. Inactivation of Escherichia coli O157:H7 on surface-uninjured and -injured green pepper (Capsicum annuum L.) by chlorine dioxide gas as demonstrated by confocal laser scanning microscopy Resistance of Bacillus Endospores to Extreme Terrestrial and Extraterrestrial Environments Antiseptics and Disinfectants: Activity, Action, and Resistance Sensitivity of Bacillus coagulans spores to combinations of high hydrostatic pressure, heat, acidity and nisin The effects of various disinfectants againstListeria monocytogeneson fresh-cut vegetables Comparative sporicidal effects of liquid chemical agents Microbiological Control of Cucumber Hydrocooling Water with Chlorine Dioxide †. Interaction of Bacillus subtilis spores with sodium hypochlorite, sodium dichloroisocyanurate and chloramine-T. Bacterial spores and chemical sporicidal agents Bacillus Species Effects of Organic Acids on Thermal Inactivation of Bacillus stearothermophilus and Bacillus coagulans Spores in Frankfurter Emulsion Slurry. Sporicidal activity of mixtures of alcohol and hypochlorite. Ultrastructural and chemical changes in spores of Bacillus cereus after action of disinfectants. A sporulation medium for Bacillus anthracis. Sporicidal properties of some halogens. Kinetics and mechanism of bacterial disinfection by chlorine dioxide. Resistance of bacterial spores. In: Storz G, Hengge-Aronis R (eds) Bacterial stress Principles and practices of disinfection, reservation and sterilization Surface decontamination of fruits and vegetables eaten raw: a review. Food Safety Issues, Food Safety Unit, World Health Organization These include: Evicting an unwelcome tenant: anthrax",
         "https://academic.oup.com/jimb/article-pdf/32/7/301/34739283/jimb0301.pdf",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "26",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "Recent advances in understanding the cell death pathways activated by anticancer therapy",
         "Over the past two decades, the role of apoptosis in the cytotoxicity of anticancer drugs has become clear. Apoptosis may occur via a death receptor‐dependent (extrinsic) or independent (intrinsic or mitochondrial) pathway. Mitochondria play a central role in cell death in response to DNA damage, and mediate the interaction(s) of various cytoplasmic organelles, including the endoplasmic reticulum, Golgi apparatus, and lysosomes. The mitochondrial pathway of cell death is mediated by Bcl‐2 family proteins, a group of antiapoptotic and proapoptotic proteins that regulate the passage of small molecules, such as cytochrome c, Smac/Diablo, and apoptosis‐inducing factor, which activates caspase cascades, through the mitochondrial transition pore. In addition, apoptosis can induce autophagic cell death via crosstalk between the two pathways upon treatment with anticancer drugs. The current review focused on recent advances surrounding the mechanism(s) of cell death induced by anticancer agents and discussed potential molecular targets for enhancing the chemotherapeutic effect(s) of anticancer agents. Cancer 2005. © 2005 American Cancer Society.",
         "Neutralization of Smac/Diablo by Inhibitors of Apoptosis (IAPs) Aging as a catabolic malfunction. Apoptosis, autophagy, and more. Dynamic evolution of the adenine nucleotide translocase interactome during chemotherapy-induced apoptosis Bax is an important determinant for radiation sensitivity in esophageal carcinoma cells. Apollon ubiquitinates SMAC and caspase-9, and has an essential cytoprotection function A Phase I Pharmacokinetic and Biological Correlative Study of Oblimersen Sodium (Genasense, G3139), an Antisense Oligonucleotide to the Bcl-2 mRNA, and of Docetaxel in Patients with Hormone-Refractory Prostate Cancer DAP-kinase as a target for drug design in cancer and diseases associated with accelerated cell death. Endogenously released Smac is insufficient to mediate cell death of human lung carcinoma in response to etoposide. Smac induces cytochrome c release and apoptosis independently from Bax/Bcl-xL in a strictly caspase-3-dependent manner in human carcinoma cells Nuclear and mitochondrial conversations in cell death: PARP-1 and AIF signaling. Ceramide-mediated Macroautophagy Involves Inhibition of Protein Kinase B and Up-regulation of Beclin 1* JNK promotes Bax translocation to mitochondria through phosphorylation of 14‐3‐3 proteins Multiple cell death pathways as regulators of tumour initiation and progression Apoptosis defects and chemotherapy resistance: molecular interaction maps and networks Changes in Therapy for Solid Tumors: Potential for Overcoming Drug Resistance In Vivo with Molecular Targeting Agents Molecular therapy for peritoneal dissemination of xenotransplanted human MKN-45 gastric cancer cells with adenovirus mediated Bax gene transfer Mitochondrial protease Omi/HtrA2 enhances caspase activation through multiple pathways Apoptosis pathways in cancer and cancer therapy In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker. Endonuclease G is required for early embryogenesis and normal apoptosis in mice Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor Phosphoinositide 3-kinase accelerates autophagic cell death during glucose deprivation in the rat cardiomyocyte-derived cell line H9c2 Endostatin induces autophagic cell death in EAhy926 human endothelial cells. HSpin1, a transmembrane protein interacting with Bcl-2/Bcl-xL, induces a caspase-independent autophagic cell death Direct Interaction between Survivin and Smac/DIABLO Is Essential for the Anti-apoptotic Activity of Survivin during Taxol-induced Apoptosis* Omi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP) irreversibly inactivates IAPs and facilitates caspase activity in apoptosis. Transformation-selective apoptotic program triggered by farnesyltransferase inhibitors requires Bin1 The CD95 type I/type II model. Amino Acids Interfere with the ERK1/2-dependent Control of Macroautophagy by Controlling the Activation of Raf-1 in Human Colon Cancer HT-29 Cells* Induction of autophagic cell death in malignant glioma cells by arsenic trioxide. Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs. Chemotherapy enhances TNF-related apoptosis-inducing ligand DISC assembly in HT29 human colon cancer cells Molecular response of human glioblastoma multiforme cells to ionizing radiation: cell cycle arrest, modulation of the expression of cyclin-dependent kinase inhibitors, and autophagy. Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death? Current status of the molecular mechanisms of anticancer drug-induced apoptosis Hypoxia and Nitric Oxide Treatment Confer Tolerance to Glucose Starvation in a 5′-AMP-activated Protein Kinase-dependent Manner* Mediation of Poly(ADP-Ribose) Polymerase-1-Dependent Cell Death by Apoptosis-Inducing Factor DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death Bcl-2 and Bcl-xL Inhibit CD95-mediated Apoptosis by Preventing Mitochondrial Release of Smac/DIABLO and Subsequent Inactivation of X-linked Inhibitor-of-Apoptosis Protein* Increased Ras expression and caspase-independent neuroblastoma cell death: possible mechanism of spontaneous neuroblastoma regression. Paclitaxel triggers cell death primarily via caspase-independent routes in the non-small cell lung cancer cell line NCI-H460. The Apaf-1 apoptosome: a large caspase-activating complex. Apoptosis-inducing factor (AIF): a novel caspase-independent death effector released from mitochondria. Identification of Omi/HtrA2 as a Mitochondrial Apoptotic Serine Protease That Disrupts Inhibitor of Apoptosis Protein-Caspase Interaction* The Tumor Suppressor PTEN Positively Regulates Macroautophagy by Inhibiting the Phosphatidylinositol 3-Kinase/Protein Kinase B Pathway* Autophagy delays sulindac sulfide-induced apoptosis in the human intestinal colon cancer cell line HT-29. Bax translocation is crucial for the sensitivity of leukaemic cells to etoposide-induced apoptosis The target of rapamycin (TOR) proteins Proapoptotic BAX and BAK: A Requisite Gateway to Mitochondrial Dysfunction and Death Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function. PTEN controls tumor-induced angiogenesis Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis Bcl-2 down-regulation causes autophagy in a caspase-independent manner in human leukemic HL60 cells Molecular Determinants of the Caspase-promoting Activity of Smac/DIABLO and Its Role in the Death Receptor Pathway* BNIP3 and Genetic Control of Necrosis-Like Cell Death through the Mitochondrial Permeability Transition Pore Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments. Distinct Classes of Phosphatidylinositol 3′-Kinases Are Involved in Signaling Pathways That Control Macroautophagy in HT-29 Cells* Induction of autophagy and inhibition of tumorigenesis by beclin 1 Autophagy Is Activated by Apoptotic Signalling in Sympathetic Neurons: An Alternative Mechanism of Death Execution Cloning and genomic organization of beclin 1, a candidate tumor suppressor gene on chromosome 17q21. Oncogenic Ras triggers cell suicide through the activation of a caspase-independent cell death program in human cancer cells BIN1 is a novel MYC–interacting protein with features of a tumour suppressor Ultrastructural and immunocytochemical characterization of autophagic vacuoles in isolated hepatocytes: effects of vinblastine and asparagine on vacuole distributions. Studies on the mechanisms of autophagy: formation of the autophagic vacuole Vinblastine-induced autophagic vacuoles in mouse liver and Ehrlich ascites tumor cells as assessed by freeze-fracture electron microscopy. The morphology of various types of cell death in prenatal tissues. Inducing cancer cell death by targeting transcription factors. Mitochondrial dysfunction is an essential step for killing of non-small cell lung carcinomas resistant to conventional treatment",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/cncr.20947",
         "plays an important role in cellular homeostasis.1 Three types of\ncell death are known: apoptosis (type I), autophagy (type II), and\nnecrosis (type III).2 Apoptosis is induced by a number of stimuli,\nincluding anticancer drugs, growth factor deprivation, and irradia-\ntion. The molecular mechanism by which apoptosis occurs has been\nextensively examined. Apoptosis in response to chemotherapy is\nknown to correlate with a reduction in tumor volume, and resistance\nto apoptosis causes drug resistance.3–5 Introduction of a proapoptotic\ngene such as Bax,6,7 and inhibition of an antiapoptotic gene such as\nBcl-2,8,9 enhances the effects of chemotherapy. Thus, apoptosis\nclearly inﬂuences the antitumor effect(s) of chemotherapeutic agents.\nIn solid tumors, hypoxia and glucose deprivation enable tumor\nprogression,10 whereas depletion of adenosine 5-triphosphate (ATP)\nresults in necrosis. The production of lactic acid under anaerobic\nconditions does not necessarily induce necrosis, rather, autophagic\ncell death may depend on susceptibility to apoptosis, which can be\ninhibited by activation of Akt by environmental stimuli.10,11 Drug\nresistance might involve resistance to autophagic cell death, as well as\napoptosis. In fact, treatment with tamoxifen causes autophagy of\nMCF-7 breast carcinoma cells, as well as apoptotic cell death via a\ncaspase 3-independent pathway.12 Autophagy is also induced by au-\nrintricarboxylic acid via activation of the ERK1/2 signaling pathway in\nRas-mutated HT-29 colon carcinoma cells.13 In addition, a human\ntumor suppressor gene, known as Beclin 1, induces autophagy.14,15\nCANCER April 15, 2005 / Volume 103 / Number 8\nReduced expression of this gene occurs in breast and\novarian carcinoma due to a 40 –75% deletion of the\ngene.16 Although resistance to autophagy and apopto-\nsis is likely involved in tumor progression and acqui-\nsition of drug resistance by solid tumors, the exact\nmolecular mechanism(s) by which this occurs, and the\nrelation between autophagy and apoptosis, remain\nunknown. Here, we review evidence regarding the\nmolecular mechanism(s) of anticancer drug-induced\ncell death and discuss potential therapeutic targets by\nwhich signal transduction pathways leading to cell\ndeath might be modulated by cancer therapy.",
         "APOPTOTIC PATHWAYS OF CELL DEATH\nCaspase-Dependent Pathway\nGiven that a number of anticancer agents induce ap-\noptosis, a lot of research has focused on the various\npathways that mediate apoptosis. Two are known to\nmediate anticancer drug-induced apoptosis, a death\nreceptor-dependent (extrinsic) and a mitochondria-\ndependent (intrinsic) pathway.17 The death receptor-\ndependent pathway involves activation of death re-\nceptors, such as Fas and TRAIL receptors (DR4, DR5).\nDeath receptor activation is mediated by a death-\ninducing signaling complex, which is formed by re-\ncruitment of a Fas-associated death domain and pro-\ncaspase 8 to the death receptor, thereby activating\ncaspase 8.3,18 Caspase 8 directly activates caspase 3,\nleading to apoptosis (type I cell death).19 The mito-\nchondria-dependent pathway involves cleavage of a\nproapoptotic protein, Bid, resulting in the production\nof truncated Bid (tBid), by caspase 8, within the mito-\nchondria. Heterodimerization of tBid and Bak results\nin release of cytochrome c from mitochondria and\nactivation of caspase 3 and caspase 9 (type II cell\ndeath).19 The predominance of one pathway over an-\nother depends on cell type.\nUnlike the death receptor-dependent pathway,\nthe mitochondria-dependent pathway is mediated by\nBax/Bak and involves the release of cytochrome c.3,18\nIn general, Bax is associated with 14-3-3 protein,\nwhich anchors Bax in the cytoplasm. However, in the\npresence of DNA damage, c-Jun NH(2)-terminal ki-\nnase phosphorylates 14-3-3, resulting in dissociation\nof Bax from this protein.20 Homodimerization of Bax,\nor heterodimerization of Bax ank Bak, results in trans-\nlocation of Bax from the cytoplasm to mitochondria.21\nBax homodimers and heterodimers interact with a\nvoltage-dependent anion channel (VDAC) in the mi-\ntochondrial outer membrane to release cytochrome c\nby increasing mitochondrial membrane permeability\n(MMP) via opening of the mitochondrial permeability\ntransition pore.22 The mitochondrial permeability\ntransition pore complex (PTPC) mediates permeability\nof the mitochondrial membrane during apoptosis, ne-\ncrosis, and autophagy. PTPC is composed of an ade-\nnine nucleotide translocator and VDAC, both of which\nare targeted by a variety of proapoptotic inducers. The\nrelease of cytochrome c results in the formation of an\napoptosome, consisting of apoptosis activating factor\n1 and procaspase 9, after which dATP activates\ncaspase 9, leading to apoptosis.23 The activated initi-\nator caspase, caspase 9, leads to activation of one of\nthe effector caspases, caspase 3 or caspase 7, which\ncleaves an inhibitor of caspase-activated DNAase and\nproduces poly (ADP-ribose) polymerase (PARP), re-\nsulting in DNA fragmentation. The release of cyto-\nchrome c through the VDAC-mediated permeability\ntransition pore is inhibited by Bcl-2 and Bcl-xL. In\naddition to cytochrome c, Smac/Diablo is released\nfrom mitochondria when there is a loss of membrane\npotential. Smac/Diablo binds to X chromosome-en-\ncoded IAP, cellular IAP-1 (cIAP-1), and cellular IAP-2,\nas well as survivin, to inhibit apoptosis,24 and it di-\nrectly activates caspase 3. Inhibitors of apoptosis pro-\nteins (IAPs) block apoptosis by binding to and inhib-\niting caspases, as well as by activating caspase-\nindependent mechanisms.25\ninhibit\nactivation of executioner caspases activated by extrin-\nsic or intrinsic pathways. The Smac/Diablo complex\ninduces apoptosis via the apoptosome-dependent\npathway,26 and release of Smac/Diablo is inhibited by\nBcl-2 and Bcl-xL.27 There is also a report that Smac\ninduces cytochrome c release and apoptosis in the\nabsence of Bax/Bcl-xL via activation of caspase 3 in\nhuman HCT116 and DU145 colon carcinoma cells.28\nSmac is capable of circumventing defects in mito-\nchondrial apoptosis signaling, such as loss of Bax or\noverexpression of Bcl-xL, which frequently occurs in\ntumor cells resistant to anticancer therapy. A recent\nstudy suggests that an IAP family protein, Apollon,\nbinds to, ubiquitinates, and facilitates the proteasomal\ndegradation of Smac and caspase 9, thereby prevent-\ning Smac-induced apoptosis.29 The role of Apollon in\ndrug resistance remains to be clariﬁed. Translocation\nof endogenous Smac into the cytosol, and release of\nSmac/Diablo during anticancer drug-induced apopto-\nsis, does not appear to play a major role in cell death\nafter treatment of human lung carcinoma with etopo-\nside,30 because cytochrome c and mitochondrial pro-\ntease Omi/HtrA2 are still detectable in the cytosol in\nthe absence of Smac. Thus, the signiﬁcance of Smac/\nDiablo in anticancer drug-induced mitochondrial\npathway-mediated apoptosis remains unclear.\ncan\nCaspase-Independent Pathway\nAlthough caspase cysteine proteases execute apopto-\nsis, a mitochondrial serine protease released into the\nCell Death with Chemotherapy/Kim\ncytosol during apoptosis, Omi/HtrA2, is required to\nantagonize inhibitors of apoptosis (IAPs), thereby en-\nabling apoptosis to occur via an indirect mechanism.31\nCleavage of c-IAP-1 by Smac/Diablo and Omi/HtrA2 is\nirreversible, thus, signiﬁcant inactivation of IAPs oc-\ncurs, enabling increased caspase activity after tumor\nnecrosis factor apoptosis related ligand (TRAIL)-in-\nduced apoptosis.32 IAP cleavage by Omi is indepen-\ndent of caspase activity. In addition, Omi/HtrA2 might\nactivate more than one pathway of caspase activation\nbecause extramitochondrial expression of Omi-HtrA2\nresults in increased mitochondrial permeability and\ncytochrome c-induced caspase activation in HeLa\ncells.33\nApoptosis-inducing factor (AIF) is a phylogeneti-\ncally conserved ﬂavoprotein within the mitochondrial\nmembrane, which has the ability to induce apoptosis\nvia a caspase-independent pathway.34 AIF induces nu-\nclear chromatin condensation and large-scale DNA\nfragmentation (producing approximately 50-kilobase\n[kb] fragments), and is essential for programmed cell\ndeath. On lethal signaling, AIF translocates through\nthe cytosol to the nucleus where it binds to DNA and\nprovokes caspase-independent chromatin condensa-\ntion. Responses to injection of anti-AIF antibodies, or\nknockout of the AIF gene, suggest that AIF may be\nrequired for cell death in response to certain stimuli.35\nStaurosporine (STS) induces mitochondrial dysfunc-\ntion and translocation of AIF into the nucleus after\nactivation of nuclear apoptosis in nonsmall cell lung\ncarcinoma cells resistant to radiotherapy or chemo-\ntherapy.36 This suggests that, although caspase-de-\npendent and independent pathways coexist,\nthe\ncaspase-dependent pathway might be less efﬁcient.\nFurthermore, resistant cells might be made more sus-\nceptible to treatment by activation of the AIF-medi-\nated caspase-independent pathway. Moreover, recent\nreports indicate that AIF is necessary for PARP-1– de-\npendent cell death.37,38 PARP-1 generates a number of\nlong, branched PARPS after DNA damage. Activation\nof PARP-1 initiates a nuclear signal that triggers the\nrelease of AIF from mitochondria. AIF then shuttles\nfrom mitochondria to the nucleus and induces periph-\neral chromatin condensation and large-scale DNA\nfragmentation.38 Endonuclease G (Endo G) is a nucle-\nar-encoded mitochondrial protein believed to be im-\nportant for nuclear DNA fragmentation during apo-\nptosis, as well as mitochondrial DNA replication. Cells\nfrom Endo G heterozygous mutant mice are resistant\nto tumor necrosis factor alpha and STS-induced cell\ndeath.39\nThus, increased MMP in response to DNA damage\nresults in the release of a number of small molecules,\nsuch as cytochrome c, Smac/Diablo, and AIF from\nmitochondria. This results in activation of caspase-\ndependent and independent apoptotic pathways, de-\npending on which triggers and cell types are involved.\nCANCER April 15, 2005 / Volume 103 / Number 8\nOur current understanding regarding the role of mi-\ntochondria in cell death is summarized in Figure 1.\nAUTOPHAGIC PATHWAY OF CELL DEATH\nAutophagic cell death, otherwise known as type II cell\ndeath, is characterized by the appearance of large auto-\nphagic vacuoles in the cytoplasm. It occurs during em-\nbryogenesis and differs from type I programmed cell\ndeath.40 Autophagy is a bulk protein degradation system\nthat is essential for normal cell activity and survival when\nnutrients are scarce.41 The ﬁrst step that occurs in auto-\nphagic cell death is the formation of a double-mem-\nbrane vacuole, the autophagosome, which is derived\nfrom part of the endoplasmic reticulum42 or from the\ncytoplasmic lipid pool.43 The autophagosome fuses with\na lysosome, after which its contents are degraded by\nlysosomal hydrolytic proteases. Despite recent advances\nregarding the molecular mechanism(s) underlying auto-\nphagy, many details remain poorly understood. Of note,\nclass I phosphoinositide 3-kinase (PI 3-kinase) inhibits,\nwhereas Class III PI 3-kinase enhances, autophagy in\nHT-29 human colon carcinoma cells.44 Class I PI 3-ki-\nnase mediates various cellular functions, including insu-\nlin and glycogen synthesis, and suppresses apoptosis\nthrough the Akt-signaling pathway, whereas Class III PI\n3-kinase inﬂuences membrane trafﬁcking. Despite the\nﬁnding that insulin-like growth factor I (IGF-I) normally\ninhibits apoptosis, IGF-I accelerates cell death when glu-\ncose and amino acids are limited, leading to accumula-\ntion of autophagic vacuoles within the cytoplasm and\nevidence of chromosome condensation.40 Autophagic\ncell death is inhibited by 3-methyladenine (3-MA) and\nLY294002, which inhibit PI 3-kinase, but not by the pan-\ncaspase inhibitor, z-VAD-fmk. Of importance, LY294002\nhas no effect on cell death in PC12 and HepG2 carci-\nnoma cells during glucose deprivation,45 suggesting that\ninterference with the PI 3-kinase/Akt signaling pathway\ncauses autophagy of cancer cells but not normal cells.\nA mammalian gene capable of inducing autoph-\nagy, Beclin 1, is a component of the PI 3-kinase com-\nplex responsible for autophagic vesicle nucleation.16\nThis gene is deleted in 75% of ovarian, 50% of breast,\nand 40% of prostate carcinoma cases.16 Frequent al-\nlelic deletions are observed in breast carcinoma cell\nlines in association with signiﬁcant decreases in Beclin\n1 protein levels. Beclin 1 functions as a tumor sup-\npressor, because there is an attenuation of MCF-7–\nmediated tumorigenicity in nude mice overexpressing\nBeclin 1.46 Although Beclin 1 is known to interact with\nBcl-2 in yeast, the relation between this association\nand tumor suppression and autophagy is not yet clear.\nDownregulation of Bcl-2 by antisense Bcl-2 in HL-60\ncells induces autophagy independent of caspase-de-\npendent and mitochondria-dependent pathways,47\nsuggesting a potential role of Bcl-2 in autophagic cell\ndeath, as well as apoptosis. Other autophagy-inducing\ngenes, including bridging integrator 1 (Bin 1) and\ndeath-associated protein kinase (DAPK), exist. Bin 1\ninteracts with c-Myc and inhibits its transformation\nactivity.48 Bin 1 is regulated by alternate splicing, and\nthe splice forms that interact with c-Myc have tumor\nsuppressor properties. This gene is frequently missing\nor functionally inactivated in malignant melanoma,\nbreast, and prostate carcinoma cells, and loss of Bin 1\nexpression may promote tumor progression, while\nlimiting susceptibility to farnesyltransferase inhibitors\n(FTIs). Bin 1 adapter proteins act downstream or par-\nallel to RhoB to exert their effect on cell signaling.49\nDAPK is a calcium-regulated serine/threonine kinase,\nand an important mediator of autophagy induced by\nnutrient deprivation, antiestrogens, and tumor necro-\nsis factor.50 Cancer and cancer cell lines frequently\ndemonstrate decreased DAPK expression due to DNA\nmethylation.51 Transfection of Beclin 1, Bin 1, or an\nactive form of DAPK, into cancer cells induces auto-\nphagic cell death. A human homolog of the Drosophila\nspin gene product (HSpin 1) exists, which binds to\nBcl-2 and Bcl-xL. HSpin 1-induced cell death results in\nformation of autophagic vacuoles and release of a\nmature form of cathepsin D, suggesting a novel\ncaspase-independent cell death, pathway resulting in\nautophagy.52\nThe tumor suppressor phosphatase and tensin\nhomolog deleted on chromosome10 (PTEN), which is\nan inhibitor of the PI 3-kinase/Akt signaling pathway,\npromotes autophagy in HT-29 colon carcinoma\ncells.53 PTEN PI phosphatase activity inhibits Akt ac-\ntivity, resulting in induction of autophagy. Loss of\nfunction of PTEN increases the expression of Class I PI\n3-kinase and Akt in human cancer cells. Activation of\nAkt and constitutive expression of active Akt block\nautophagic cell death, leading to inhibition of apopto-\nsis and activation of a mammalian target of rapamycin\n(mTor) protein. It is likely that activation of Akt and\nthe mTor signaling pathway, due to loss of PTEN,\ncontributes to malignant transformation by simulta-\nneous inhibition of autophagic and apoptotic cell\ndeath. Another oncoprotein, Ha-Ras, induces autoph-\nagic cell death in glioma and gastric carcinoma cells54\nvia a p53-independent pathway that is not blocked by\nBcl-2. Another Ras family, including K-Ras and N-Ras\nproteins, also induces autophagic cell death.\nIn-\ncreased Ras expression is observed in spontaneous\nregression of neuroblastoma, which is characterized\nby autophagic cell death and a lack of apoptosis.55 In\naddition, Ha-Ras overexpression induces autophagic\ncell death in neuroblastoma cell\nlines. Ha-Ras–in-\nduced autophagy in HT-29 colon carcinoma cells has\nCell Death with Chemotherapy/Kim\nsome overlap with the Raf 1/ERK signaling pathway of\nautophagic cell death.56 The ability of Ras to cause\nautophagy might involve activation of ErbB2 and\nErbB3. It is interesting to note that one of the signaling\npathways regulated by Ras, the Raf1/ERK pathway,\ntriggers autophagy, whereas the PI 3-kinase/Akt path-\nway, which is downstream of Ras signaling, inhibits it.\nRas-induced increases in the expression and trafﬁck-\ning of lysosomal cathepsins might contribute to Ras-\ninduced autophagy. A proposed molecular model of\nautophagy is illustrated in Figure 2.\nA death-promoting “BH3-only” family protein, Bcl-2/\nadenovirus E1B 19-kilodalton interacting protein 3\n(BNIP3), differs from other proteins of this class.\nBNIP3 can directly integrate into the mitochondrial\nouter membrane via its BH3 domain in the absence of\nBax/Bak, thereby disrupting the MMP, generating re-\nactive oxygen species (ROS) and promoting autoph-\nagic degeneration.57 Overexpression of BNIP3, a hy-\npoxia-inducible protein with a BH3 domain, results in\nautophagic cell death in cancer cell lines, after cyto-\nchrome c or AIF nuclear translocation. BNIP3-induced\nautophagy can be attenuated by overexpression of Bcl-2,\nwhich interacts with BNIP3 protein. Autophagic cell\ndeath can be activated in cancer cells in response to\nvarious anticancer drugs. Treatment of MCF-7 breast\ncarcinoma cells with tamoxifen induces autophagy be-\nfore apoptotic cell death.12 Treatment with estradiol and\n3-MA inhibits tamoxifen-induced cell death. Similarly,\nother anticancer drugs, such as vinblastin, paclitaxel,\ninduce autophagic cell death.58 – 60\nand rapamycin,\nTreatment of malignant glioma cells with arsenic triox-\nide induces autophagic cell death in association with\nG2/M arrest.61 In addition, radiotherapy induces auto-\nphagic cell death in several cancer cell lines, including\nbreast, colon, and prostate carcinoma, as well as glio-\nblastoma multiforme.62 Despite several reports of induc-\ntion of autophagy by treatment with anticancer drugs\nand/or irradiation, the clinical signiﬁcance of autophagy\nas a result of chemotherapy remains unclear.\nINTERACTION BETWEEN APOPTOTIC AND\nAUTOPHAGIC PATHWAYS OF CELL DEATH\nDifferent types of cell death differ in their morphologic\ncharacteristics (Fig. 3 and Table 1). In fact, ceramide,\nwhich is believed to activate apoptosis, also induces\nCANCER April 15, 2005 / Volume 103 / Number 8\nChromatin condensation\nDNA laddering and fragmentation\nPyknosis\nCytoplasmic condensation\nFragmentation to apoptotic bodies\nIncrease in MMP\nActivation of caspase cascade\nPotential release of lysosomal enzyme\nOrganelles do not swell\nBlebbing\nCaspases such as caspase 3\nYes\nz-VAD-fmk\nXIAP\nBcl-2/Bcl-xL\nSometimes with actinomycin D\nSometimes with cyclophosphamide\nDNA laddering test\nCaspase activation and the substrate\nTUNEL and annexin V staining\nFACS analysis\nElectron microscopy\nautophagy.63 Similarly, proteins like DAPK are capable\nof inducing both apoptosis and autophagy depending\non the cell type involved, suggesting that they may\nfunction as molecular switches or integrators of these\ntwo types of programmed cell death. Commitment to\nautophagy or apoptosis may depend on the cell type\ninvolved, as well as the microenvironment. Depending\non the setting, initiation of autophagy may induce,\ndelay, or antagonize apoptosis. Three scenarios must\nbe considered when contemplating the interaction be-\ntween apoptosis and autophagic cell death.\nFirst, autophagy may precede apoptosis. Treat-\nment of sympathetic neurons with cytosine arabino-\nside, thereby depleting neural growth factor, results in\na 30-fold increase in autophagy, followed by DNA\ndegradation and apoptosis.64 Delayed induction of ap-\noptosis occurs with 3-MA, which inhibits cytochrome\nc release and caspase activation. These ﬁndings may\nmodel cell death after treatment with anticancer drugs\nor growth factor deprivation.\nSecond, autophagy may antagonize apoptosis.\nConversely, a reduction in autophagy might increase\nthe susceptibility of cells to apoptotic stimuli. Accel-\nerated sulindac sulﬁde-induced apoptosis occurs in\nmutant clone HT-29 colon carcinoma cells, which lack\nautophagic capability due to overexpression of a mu-\ntant G (alpha i3) protein.65 In contrast, treatment of\nparent HT-29 cells with 3-MA increases sensitivity to\nsulindac sulﬁde-induced apoptosis. Even though\nthere are no differences in the expression patterns of\nCOX-2, Bcl-2, Bcl-xL, Bax, and Akt/PKB among mutant\nand parent HT-29 cells, greater cytochrome c release\noccurs in mutant G (alpha i3) protein overexpressing\ncells, compared with parent cells. It is possible that\nautophagy retards apoptosis in colon carcinoma cells\nby sequestering mitochondrial death-promoting fac-\ntors, such as cytochrome c.\nThird, apoptosis and autophagy may be mutually\nexclusive. Inhibition of autophagy may lead to apo-\nptotic cell death. A recent report reveals that treat-\nment of malignant glioma cells with arsenic trioxide\ninduces autophagic cell death. However, inhibition of\nautophagy by treatment with baﬁlomycin, a H-AT-\nleads to apoptosis.61 However, al-\nPase inhibitor,\nthough treatment of neurons and HeLa cells with a\npancaspase inhibitor prevents apoptosis, selective mi-\nimpairment results in autophagy.66 In\ntochondrial\nlight of several reports indicating that treatment with\ncaspase inhibitors causes a shift from apoptotic cell\ndeath to autophagy, it is possible that this also occurs\nunder other circumstances. For example, inhibition of\napoptosis in Bax/Bak double-deﬁcient DKO cells in-\nduces autophagic cell death.67 The ability of cells to\nrevert between apoptosis and autophagy might be\nclinically signiﬁcant because drug resistance to apo-\nptosis might be overcome by activation of autophagic\ncell death. Along this line, multidrug-resistant MCF-7\ncells can be made susceptible to doxorubicin by treat-\nment with verapamil, after which autophagic cell\ndeath occurs, suggesting that the autophagic cell\ndeath pathway remains viable after inhibition of the\napoptotic pathway (unpublished data). The molecular\nCell Death with Chemotherapy/Kim\nmechanism(s) by which reversion from apoptosis to\nautophagy occurs remains unclear. Although overex-\npression of Bcl-2 or Bcl-xL may inhibit autophagic cell\ndeath, the role of these antiapoptotic proteins in au-\ntophagic cell death requires further clariﬁcation.\nThe role of Bcl-2 in apoptosis in response to mi-\ntochondrial damage is well established. However, sig-\nnaling as a result of mitochondrial damage might also\nactivate the autophagic pathway of cell death. It ap-\npears that mitochondria are involved in integrating\nthe apoptotic and autophagic pathways of cell death.\nLysosomal enzymes might activate apoptosis because\nlysosome rupture activates apoptosis. It is interesting\nto note that apoptosis after lysosome rupture is pre-\nceded by autophagy. Sequestration of mitochondria\nwithin autophagic vacuoles might protect cells from\napoptosis by delaying the release of cytochrome c.\nThus, autophagy might regulate apoptosis through ly-\nsosome formation and enzyme activity. With regard to\nthe possibility of signaling between lysosomes and\nmitochondria, a recent study reveals that production\nof H2O2 from ROS in response to DNA damage acti-\nvates lysosomal enzymes, such as cathepsin D and B.68\nCathepsin B directly or indirectly results in transloca-\ntion of proapoptotic proteins, such as Bid and Bax,\nfrom the cytoplasm to mitochondria. This alters MMP\nand results in caspase activation subsequent to the\nrelease of cytochrome c, Smac/Diablo, and AIF from\nthe mitochondria. Conversely, under different circum-\nstances, production of H2O2 from ROS in response to\nDNA damage may inhibit the release of small mole-\ncules, such as cytochrome c, Smac/Diablo, and AIF,\nfrom the mitochondrial membrane and activate lyso-\nsomal activity. Thus, crosstalk might occur between\nthe pathways mediating autophagy and apoptosis by\nway of lysosomes and mitochondria. Further research\nis required to clarify the signaling that occurs between\nthese pathways.\nFUTURE PERSPECTIVES REGARDING CELL DEATH\nAND CANCER THERAPY\nIncreasing information is known regarding the molec-\nular mechanism(s) by which cell death is mediated by\nanticancer drugs. Proapoptotic and antiapoptotic pro-\nteins are now being targeted to enhance the effect(s)\nof chemotherapy. In general, the antitumor effect of\nan anticancer agent is measured in terms of its ability\nto induce apoptotic cell death within tumors, which is\nwell correlated with outcome in most preclinical and\nclinical studies. Nevertheless, despite signiﬁcant re-\nductions in tumor volume, high rates of apoptosis do\nnot always occur within solid tumors after chemother-\napy. This suggests that other types of cell death, such\nas autophagy and necrosis, might also occur as a result\nCANCER April 15, 2005 / Volume 103 / Number 8\nof chemotherapy. Research into other types of cell\ndeath, particularly autophagic cell death, is important\nto better understand the effects of anticancer drugs. In\naddition, the relation between autophagic cell death\nand apoptosis needs to be examined, because several\nautophagy-related proteins, including Beclin 1, BNIP3,\nDAPK, and HSpin 1, may inﬂuence apoptosis. Because\napoptosis and autophagy are mutually exclusive in\nsome cell lines, molecular switches between the two\nprocesses must exist.\nResistance to autophagy, as well as apoptosis,\nmight lend resistance to anticancer drugs. Likewise,\ninduction of autophagic cell death in cancer cells may\nbe of therapeutic beneﬁt. Cells transformed with the\ntumor suppressor gene Beclin 1 are less likely to un-\ndergo autophagy and are more susceptible to onco-\ngenesis. Alterations in autophagic genes reduce the\nincidence of autophagy in cancer cells. However, even\nin cells resistant to apoptosis or cells in which apo-\nptosis is inhibited, nonapoptotic cell death pathways,\nsuch as autophagy, may be activated. Targeting alter-\nnative pathways of cell death might overcome drug\nresistance or explain why the effects of chemotherapy\nand irradiation complement each other in the clinical\nsetting. Alterations or defects in apoptotic signal\ntransduction pathways might result in drug resistance,\nhowever, irradiation induces autophagic cell death.\nAlterations of one or both pathways (autophagic or\napoptotic) might determine susceptibility to cell death\nafter chemotherapy because anticancer drugs might\ntrigger both apoptotic caspase-dependent and auto-\nphagic caspase-independent pathways at the same\ntime.\nAutophagic cell death might also inhibit angio-\ngenesis.69 The angiogenesis inhibitor, endostatin, in-\nduces endothelial cell death via the caspase-indepen-\ndent autophagic pathway. In addition, the tumor\nsuppressor gene, PTEN, which induces autophagy, in-\nhibits tumor angiogenesis in nude mice with ortho-\ntopic brain tumors.70 Reconstitution of wild-type\nPTEN has no effect on in vitro proliferation, but dra-\nmatically decreases tumor growth in vivo and pro-\nlongs survival in mice, suggesting that PTEN regulates\ntumor-induced angiogenesis via regulation of PI-de-\npendent signals. The role of autophagy in tumor an-\ngiogenesis might be an important means of inhibiting\ntumor progression and increasing the effectiveness of\nanticancer therapy.\nIt is now clear that a number of different types of\ncell death, including apoptosis and autophagy, play a\nrole in the antitumor effect(s) of treatment with anti-\ncancer drugs. Targeting therapy at genes related to\napoptosis might increase the therapeutic efﬁcacy of\nanticancer agents by modulating signal transduction\npathways, and other cell death pathways might be\ninﬂuenced by changes in the microenvironment and\nby crosstalk between the various cell death pathways.\nFurther research into the molecular mechanism(s) be-\nhind autophagic cell death as a result of chemother-\napy might provide a new strategy for overcoming drug\nresistance by exploring this alternative pathway of cell\ndeath.",
         "",
         "",
         "",
         "plays an important role in cellular homeostasis.1 Three types of\ncell death are known: apoptosis (type I), autophagy (type II), and\nnecrosis (type III).2 Apoptosis is induced by a number of stimuli,\nincluding anticancer drugs, growth factor deprivation, and irradia-\ntion. The molecular mechanism by which apoptosis occurs has been\nextensively examined. Apoptosis in response to chemotherapy is\nknown to correlate with a reduction in tumor volume, and resistance\nto apoptosis causes drug resistance.3–5 Introduction of a proapoptotic\ngene such as Bax,6,7 and inhibition of an antiapoptotic gene such as\nBcl-2,8,9 enhances the effects of chemotherapy. Thus, apoptosis\nclearly inﬂuences the antitumor effect(s) of chemotherapeutic agents.\nIn solid tumors, hypoxia and glucose deprivation enable tumor\nprogression,10 whereas depletion of adenosine 5-triphosphate (ATP)\nresults in necrosis. The production of lactic acid under anaerobic\nconditions does not necessarily induce necrosis, rather, autophagic\ncell death may depend on susceptibility to apoptosis, which can be\ninhibited by activation of Akt by environmental stimuli.10,11 Drug\nresistance might involve resistance to autophagic cell death, as well as\napoptosis. In fact, treatment with tamoxifen causes autophagy of\nMCF-7 breast carcinoma cells, as well as apoptotic cell death via a\ncaspase 3-independent pathway.12 Autophagy is also induced by au-\nrintricarboxylic acid via activation of the ERK1/2 signaling pathway in\nRas-mutated HT-29 colon carcinoma cells.13 In addition, a human\ntumor suppressor gene, known as Beclin 1, induces autophagy.14,15\nCANCER April 15, 2005 / Volume 103 / Number 8\nReduced expression of this gene occurs in breast and\novarian carcinoma due to a 40 –75% deletion of the\ngene.16 Although resistance to autophagy and apopto-\nsis is likely involved in tumor progression and acqui-\nsition of drug resistance by solid tumors, the exact\nmolecular mechanism(s) by which this occurs, and the\nrelation between autophagy and apoptosis, remain\nunknown. Here, we review evidence regarding the\nmolecular mechanism(s) of anticancer drug-induced\ncell death and discuss potential therapeutic targets by\nwhich signal transduction pathways leading to cell\ndeath might be modulated by cancer therapy.\nAPOPTOTIC PATHWAYS OF CELL DEATH\nCaspase-Dependent Pathway\nGiven that a number of anticancer agents induce ap-\noptosis, a lot of research has focused on the various\npathways that mediate apoptosis. Two are known to\nmediate anticancer drug-induced apoptosis, a death\nreceptor-dependent (extrinsic) and a mitochondria-\ndependent (intrinsic) pathway.17 The death receptor-\ndependent pathway involves activation of death re-\nceptors, such as Fas and TRAIL receptors (DR4, DR5).\nDeath receptor activation is mediated by a death-\ninducing signaling complex, which is formed by re-\ncruitment of a Fas-associated death domain and pro-\ncaspase 8 to the death receptor, thereby activating\ncaspase 8.3,18 Caspase 8 directly activates caspase 3,\nleading to apoptosis (type I cell death).19 The mito-\nchondria-dependent pathway involves cleavage of a\nproapoptotic protein, Bid, resulting in the production\nof truncated Bid (tBid), by caspase 8, within the mito-\nchondria. Heterodimerization of tBid and Bak results\nin release of cytochrome c from mitochondria and\nactivation of caspase 3 and caspase 9 (type II cell\ndeath).19 The predominance of one pathway over an-\nother depends on cell type.\nUnlike the death receptor-dependent pathway,\nthe mitochondria-dependent pathway is mediated by\nBax/Bak and involves the release of cytochrome c.3,18\nIn general, Bax is associated with 14-3-3 protein,\nwhich anchors Bax in the cytoplasm. However, in the\npresence of DNA damage, c-Jun NH(2)-terminal ki-\nnase phosphorylates 14-3-3, resulting in dissociation\nof Bax from this protein.20 Homodimerization of Bax,\nor heterodimerization of Bax ank Bak, results in trans-\nlocation of Bax from the cytoplasm to mitochondria.21\nBax homodimers and heterodimers interact with a\nvoltage-dependent anion channel (VDAC) in the mi-\ntochondrial outer membrane to release cytochrome c\nby increasing mitochondrial membrane permeability\n(MMP) via opening of the mitochondrial permeability\ntransition pore.22 The mitochondrial permeability\ntransition pore complex (PTPC) mediates permeability\nof the mitochondrial membrane during apoptosis, ne-\ncrosis, and autophagy. PTPC is composed of an ade-\nnine nucleotide translocator and VDAC, both of which\nare targeted by a variety of proapoptotic inducers. The\nrelease of cytochrome c results in the formation of an\napoptosome, consisting of apoptosis activating factor\n1 and procaspase 9, after which dATP activates\ncaspase 9, leading to apoptosis.23 The activated initi-\nator caspase, caspase 9, leads to activation of one of\nthe effector caspases, caspase 3 or caspase 7, which\ncleaves an inhibitor of caspase-activated DNAase and\nproduces poly (ADP-ribose) polymerase (PARP), re-\nsulting in DNA fragmentation. The release of cyto-\nchrome c through the VDAC-mediated permeability\ntransition pore is inhibited by Bcl-2 and Bcl-xL. In\naddition to cytochrome c, Smac/Diablo is released\nfrom mitochondria when there is a loss of membrane\npotential. Smac/Diablo binds to X chromosome-en-\ncoded IAP, cellular IAP-1 (cIAP-1), and cellular IAP-2,\nas well as survivin, to inhibit apoptosis,24 and it di-\nrectly activates caspase 3. Inhibitors of apoptosis pro-\nteins (IAPs) block apoptosis by binding to and inhib-\niting caspases, as well as by activating caspase-\nindependent mechanisms.25\ninhibit\nactivation of executioner caspases activated by extrin-\nsic or intrinsic pathways. The Smac/Diablo complex\ninduces apoptosis via the apoptosome-dependent\npathway,26 and release of Smac/Diablo is inhibited by\nBcl-2 and Bcl-xL.27 There is also a report that Smac\ninduces cytochrome c release and apoptosis in the\nabsence of Bax/Bcl-xL via activation of caspase 3 in\nhuman HCT116 and DU145 colon carcinoma cells.28\nSmac is capable of circumventing defects in mito-\nchondrial apoptosis signaling, such as loss of Bax or\noverexpression of Bcl-xL, which frequently occurs in\ntumor cells resistant to anticancer therapy. A recent\nstudy suggests that an IAP family protein, Apollon,\nbinds to, ubiquitinates, and facilitates the proteasomal\ndegradation of Smac and caspase 9, thereby prevent-\ning Smac-induced apoptosis.29 The role of Apollon in\ndrug resistance remains to be clariﬁed. Translocation\nof endogenous Smac into the cytosol, and release of\nSmac/Diablo during anticancer drug-induced apopto-\nsis, does not appear to play a major role in cell death\nafter treatment of human lung carcinoma with etopo-\nside,30 because cytochrome c and mitochondrial pro-\ntease Omi/HtrA2 are still detectable in the cytosol in\nthe absence of Smac. Thus, the signiﬁcance of Smac/\nDiablo in anticancer drug-induced mitochondrial\npathway-mediated apoptosis remains unclear.\ncan\nCaspase-Independent Pathway\nAlthough caspase cysteine proteases execute apopto-\nsis, a mitochondrial serine protease released into the\nCell Death with Chemotherapy/Kim\ncytosol during apoptosis, Omi/HtrA2, is required to\nantagonize inhibitors of apoptosis (IAPs), thereby en-\nabling apoptosis to occur via an indirect mechanism.31\nCleavage of c-IAP-1 by Smac/Diablo and Omi/HtrA2 is\nirreversible, thus, signiﬁcant inactivation of IAPs oc-\ncurs, enabling increased caspase activity after tumor\nnecrosis factor apoptosis related ligand (TRAIL)-in-\nduced apoptosis.32 IAP cleavage by Omi is indepen-\ndent of caspase activity. In addition, Omi/HtrA2 might\nactivate more than one pathway of caspase activation\nbecause extramitochondrial expression of Omi-HtrA2\nresults in increased mitochondrial permeability and\ncytochrome c-induced caspase activation in HeLa\ncells.33\nApoptosis-inducing factor (AIF) is a phylogeneti-\ncally conserved ﬂavoprotein within the mitochondrial\nmembrane, which has the ability to induce apoptosis\nvia a caspase-independent pathway.34 AIF induces nu-\nclear chromatin condensation and large-scale DNA\nfragmentation (producing approximately 50-kilobase\n[kb] fragments), and is essential for programmed cell\ndeath. On lethal signaling, AIF translocates through\nthe cytosol to the nucleus where it binds to DNA and\nprovokes caspase-independent chromatin condensa-\ntion. Responses to injection of anti-AIF antibodies, or\nknockout of the AIF gene, suggest that AIF may be\nrequired for cell death in response to certain stimuli.35\nStaurosporine (STS) induces mitochondrial dysfunc-\ntion and translocation of AIF into the nucleus after\nactivation of nuclear apoptosis in nonsmall cell lung\ncarcinoma cells resistant to radiotherapy or chemo-\ntherapy.36 This suggests that, although caspase-de-\npendent and independent pathways coexist,\nthe\ncaspase-dependent pathway might be less efﬁcient.\nFurthermore, resistant cells might be made more sus-\nceptible to treatment by activation of the AIF-medi-\nated caspase-independent pathway. Moreover, recent\nreports indicate that AIF is necessary for PARP-1– de-\npendent cell death.37,38 PARP-1 generates a number of\nlong, branched PARPS after DNA damage. Activation\nof PARP-1 initiates a nuclear signal that triggers the\nrelease of AIF from mitochondria. AIF then shuttles\nfrom mitochondria to the nucleus and induces periph-\neral chromatin condensation and large-scale DNA\nfragmentation.38 Endonuclease G (Endo G) is a nucle-\nar-encoded mitochondrial protein believed to be im-\nportant for nuclear DNA fragmentation during apo-\nptosis, as well as mitochondrial DNA replication. Cells\nfrom Endo G heterozygous mutant mice are resistant\nto tumor necrosis factor alpha and STS-induced cell\ndeath.39\nThus, increased MMP in response to DNA damage\nresults in the release of a number of small molecules,\nsuch as cytochrome c, Smac/Diablo, and AIF from\nmitochondria. This results in activation of caspase-\ndependent and independent apoptotic pathways, de-\npending on which triggers and cell types are involved.\nCANCER April 15, 2005 / Volume 103 / Number 8\nOur current understanding regarding the role of mi-\ntochondria in cell death is summarized in Figure 1.\nAUTOPHAGIC PATHWAY OF CELL DEATH\nAutophagic cell death, otherwise known as type II cell\ndeath, is characterized by the appearance of large auto-\nphagic vacuoles in the cytoplasm. It occurs during em-\nbryogenesis and differs from type I programmed cell\ndeath.40 Autophagy is a bulk protein degradation system\nthat is essential for normal cell activity and survival when\nnutrients are scarce.41 The ﬁrst step that occurs in auto-\nphagic cell death is the formation of a double-mem-\nbrane vacuole, the autophagosome, which is derived\nfrom part of the endoplasmic reticulum42 or from the\ncytoplasmic lipid pool.43 The autophagosome fuses with\na lysosome, after which its contents are degraded by\nlysosomal hydrolytic proteases. Despite recent advances\nregarding the molecular mechanism(s) underlying auto-\nphagy, many details remain poorly understood. Of note,\nclass I phosphoinositide 3-kinase (PI 3-kinase) inhibits,\nwhereas Class III PI 3-kinase enhances, autophagy in\nHT-29 human colon carcinoma cells.44 Class I PI 3-ki-\nnase mediates various cellular functions, including insu-\nlin and glycogen synthesis, and suppresses apoptosis\nthrough the Akt-signaling pathway, whereas Class III PI\n3-kinase inﬂuences membrane trafﬁcking. Despite the\nﬁnding that insulin-like growth factor I (IGF-I) normally\ninhibits apoptosis, IGF-I accelerates cell death when glu-\ncose and amino acids are limited, leading to accumula-\ntion of autophagic vacuoles within the cytoplasm and\nevidence of chromosome condensation.40 Autophagic\ncell death is inhibited by 3-methyladenine (3-MA) and\nLY294002, which inhibit PI 3-kinase, but not by the pan-\ncaspase inhibitor, z-VAD-fmk. Of importance, LY294002\nhas no effect on cell death in PC12 and HepG2 carci-\nnoma cells during glucose deprivation,45 suggesting that\ninterference with the PI 3-kinase/Akt signaling pathway\ncauses autophagy of cancer cells but not normal cells.\nA mammalian gene capable of inducing autoph-\nagy, Beclin 1, is a component of the PI 3-kinase com-\nplex responsible for autophagic vesicle nucleation.16\nThis gene is deleted in 75% of ovarian, 50% of breast,\nand 40% of prostate carcinoma cases.16 Frequent al-\nlelic deletions are observed in breast carcinoma cell\nlines in association with signiﬁcant decreases in Beclin\n1 protein levels. Beclin 1 functions as a tumor sup-\npressor, because there is an attenuation of MCF-7–\nmediated tumorigenicity in nude mice overexpressing\nBeclin 1.46 Although Beclin 1 is known to interact with\nBcl-2 in yeast, the relation between this association\nand tumor suppression and autophagy is not yet clear.\nDownregulation of Bcl-2 by antisense Bcl-2 in HL-60\ncells induces autophagy independent of caspase-de-\npendent and mitochondria-dependent pathways,47\nsuggesting a potential role of Bcl-2 in autophagic cell\ndeath, as well as apoptosis. Other autophagy-inducing\ngenes, including bridging integrator 1 (Bin 1) and\ndeath-associated protein kinase (DAPK), exist. Bin 1\ninteracts with c-Myc and inhibits its transformation\nactivity.48 Bin 1 is regulated by alternate splicing, and\nthe splice forms that interact with c-Myc have tumor\nsuppressor properties. This gene is frequently missing\nor functionally inactivated in malignant melanoma,\nbreast, and prostate carcinoma cells, and loss of Bin 1\nexpression may promote tumor progression, while\nlimiting susceptibility to farnesyltransferase inhibitors\n(FTIs). Bin 1 adapter proteins act downstream or par-\nallel to RhoB to exert their effect on cell signaling.49\nDAPK is a calcium-regulated serine/threonine kinase,\nand an important mediator of autophagy induced by\nnutrient deprivation, antiestrogens, and tumor necro-\nsis factor.50 Cancer and cancer cell lines frequently\ndemonstrate decreased DAPK expression due to DNA\nmethylation.51 Transfection of Beclin 1, Bin 1, or an\nactive form of DAPK, into cancer cells induces auto-\nphagic cell death. A human homolog of the Drosophila\nspin gene product (HSpin 1) exists, which binds to\nBcl-2 and Bcl-xL. HSpin 1-induced cell death results in\nformation of autophagic vacuoles and release of a\nmature form of cathepsin D, suggesting a novel\ncaspase-independent cell death, pathway resulting in\nautophagy.52\nThe tumor suppressor phosphatase and tensin\nhomolog deleted on chromosome10 (PTEN), which is\nan inhibitor of the PI 3-kinase/Akt signaling pathway,\npromotes autophagy in HT-29 colon carcinoma\ncells.53 PTEN PI phosphatase activity inhibits Akt ac-\ntivity, resulting in induction of autophagy. Loss of\nfunction of PTEN increases the expression of Class I PI\n3-kinase and Akt in human cancer cells. Activation of\nAkt and constitutive expression of active Akt block\nautophagic cell death, leading to inhibition of apopto-\nsis and activation of a mammalian target of rapamycin\n(mTor) protein. It is likely that activation of Akt and\nthe mTor signaling pathway, due to loss of PTEN,\ncontributes to malignant transformation by simulta-\nneous inhibition of autophagic and apoptotic cell\ndeath. Another oncoprotein, Ha-Ras, induces autoph-\nagic cell death in glioma and gastric carcinoma cells54\nvia a p53-independent pathway that is not blocked by\nBcl-2. Another Ras family, including K-Ras and N-Ras\nproteins, also induces autophagic cell death.\nIn-\ncreased Ras expression is observed in spontaneous\nregression of neuroblastoma, which is characterized\nby autophagic cell death and a lack of apoptosis.55 In\naddition, Ha-Ras overexpression induces autophagic\ncell death in neuroblastoma cell\nlines. Ha-Ras–in-\nduced autophagy in HT-29 colon carcinoma cells has\nCell Death with Chemotherapy/Kim\nsome overlap with the Raf 1/ERK signaling pathway of\nautophagic cell death.56 The ability of Ras to cause\nautophagy might involve activation of ErbB2 and\nErbB3. It is interesting to note that one of the signaling\npathways regulated by Ras, the Raf1/ERK pathway,\ntriggers autophagy, whereas the PI 3-kinase/Akt path-\nway, which is downstream of Ras signaling, inhibits it.\nRas-induced increases in the expression and trafﬁck-\ning of lysosomal cathepsins might contribute to Ras-\ninduced autophagy. A proposed molecular model of\nautophagy is illustrated in Figure 2.\nA death-promoting “BH3-only” family protein, Bcl-2/\nadenovirus E1B 19-kilodalton interacting protein 3\n(BNIP3), differs from other proteins of this class.\nBNIP3 can directly integrate into the mitochondrial\nouter membrane via its BH3 domain in the absence of\nBax/Bak, thereby disrupting the MMP, generating re-\nactive oxygen species (ROS) and promoting autoph-\nagic degeneration.57 Overexpression of BNIP3, a hy-\npoxia-inducible protein with a BH3 domain, results in\nautophagic cell death in cancer cell lines, after cyto-\nchrome c or AIF nuclear translocation. BNIP3-induced\nautophagy can be attenuated by overexpression of Bcl-2,\nwhich interacts with BNIP3 protein. Autophagic cell\ndeath can be activated in cancer cells in response to\nvarious anticancer drugs. Treatment of MCF-7 breast\ncarcinoma cells with tamoxifen induces autophagy be-\nfore apoptotic cell death.12 Treatment with estradiol and\n3-MA inhibits tamoxifen-induced cell death. Similarly,\nother anticancer drugs, such as vinblastin, paclitaxel,\ninduce autophagic cell death.58 – 60\nand rapamycin,\nTreatment of malignant glioma cells with arsenic triox-\nide induces autophagic cell death in association with\nG2/M arrest.61 In addition, radiotherapy induces auto-\nphagic cell death in several cancer cell lines, including\nbreast, colon, and prostate carcinoma, as well as glio-\nblastoma multiforme.62 Despite several reports of induc-\ntion of autophagy by treatment with anticancer drugs\nand/or irradiation, the clinical signiﬁcance of autophagy\nas a result of chemotherapy remains unclear.\nINTERACTION BETWEEN APOPTOTIC AND\nAUTOPHAGIC PATHWAYS OF CELL DEATH\nDifferent types of cell death differ in their morphologic\ncharacteristics (Fig. 3 and Table 1). In fact, ceramide,\nwhich is believed to activate apoptosis, also induces\nCANCER April 15, 2005 / Volume 103 / Number 8\nChromatin condensation\nDNA laddering and fragmentation\nPyknosis\nCytoplasmic condensation\nFragmentation to apoptotic bodies\nIncrease in MMP\nActivation of caspase cascade\nPotential release of lysosomal enzyme\nOrganelles do not swell\nBlebbing\nCaspases such as caspase 3\nYes\nz-VAD-fmk\nXIAP\nBcl-2/Bcl-xL\nSometimes with actinomycin D\nSometimes with cyclophosphamide\nDNA laddering test\nCaspase activation and the substrate\nTUNEL and annexin V staining\nFACS analysis\nElectron microscopy\nautophagy.63 Similarly, proteins like DAPK are capable\nof inducing both apoptosis and autophagy depending\non the cell type involved, suggesting that they may\nfunction as molecular switches or integrators of these\ntwo types of programmed cell death. Commitment to\nautophagy or apoptosis may depend on the cell type\ninvolved, as well as the microenvironment. Depending\non the setting, initiation of autophagy may induce,\ndelay, or antagonize apoptosis. Three scenarios must\nbe considered when contemplating the interaction be-\ntween apoptosis and autophagic cell death.\nFirst, autophagy may precede apoptosis. Treat-\nment of sympathetic neurons with cytosine arabino-\nside, thereby depleting neural growth factor, results in\na 30-fold increase in autophagy, followed by DNA\ndegradation and apoptosis.64 Delayed induction of ap-\noptosis occurs with 3-MA, which inhibits cytochrome\nc release and caspase activation. These ﬁndings may\nmodel cell death after treatment with anticancer drugs\nor growth factor deprivation.\nSecond, autophagy may antagonize apoptosis.\nConversely, a reduction in autophagy might increase\nthe susceptibility of cells to apoptotic stimuli. Accel-\nerated sulindac sulﬁde-induced apoptosis occurs in\nmutant clone HT-29 colon carcinoma cells, which lack\nautophagic capability due to overexpression of a mu-\ntant G (alpha i3) protein.65 In contrast, treatment of\nparent HT-29 cells with 3-MA increases sensitivity to\nsulindac sulﬁde-induced apoptosis. Even though\nthere are no differences in the expression patterns of\nCOX-2, Bcl-2, Bcl-xL, Bax, and Akt/PKB among mutant\nand parent HT-29 cells, greater cytochrome c release\noccurs in mutant G (alpha i3) protein overexpressing\ncells, compared with parent cells. It is possible that\nautophagy retards apoptosis in colon carcinoma cells\nby sequestering mitochondrial death-promoting fac-\ntors, such as cytochrome c.\nThird, apoptosis and autophagy may be mutually\nexclusive. Inhibition of autophagy may lead to apo-\nptotic cell death. A recent report reveals that treat-\nment of malignant glioma cells with arsenic trioxide\ninduces autophagic cell death. However, inhibition of\nautophagy by treatment with baﬁlomycin, a H-AT-\nleads to apoptosis.61 However, al-\nPase inhibitor,\nthough treatment of neurons and HeLa cells with a\npancaspase inhibitor prevents apoptosis, selective mi-\nimpairment results in autophagy.66 In\ntochondrial\nlight of several reports indicating that treatment with\ncaspase inhibitors causes a shift from apoptotic cell\ndeath to autophagy, it is possible that this also occurs\nunder other circumstances. For example, inhibition of\napoptosis in Bax/Bak double-deﬁcient DKO cells in-\nduces autophagic cell death.67 The ability of cells to\nrevert between apoptosis and autophagy might be\nclinically signiﬁcant because drug resistance to apo-\nptosis might be overcome by activation of autophagic\ncell death. Along this line, multidrug-resistant MCF-7\ncells can be made susceptible to doxorubicin by treat-\nment with verapamil, after which autophagic cell\ndeath occurs, suggesting that the autophagic cell\ndeath pathway remains viable after inhibition of the\napoptotic pathway (unpublished data). The molecular\nCell Death with Chemotherapy/Kim\nmechanism(s) by which reversion from apoptosis to\nautophagy occurs remains unclear. Although overex-\npression of Bcl-2 or Bcl-xL may inhibit autophagic cell\ndeath, the role of these antiapoptotic proteins in au-\ntophagic cell death requires further clariﬁcation.\nThe role of Bcl-2 in apoptosis in response to mi-\ntochondrial damage is well established. However, sig-\nnaling as a result of mitochondrial damage might also\nactivate the autophagic pathway of cell death. It ap-\npears that mitochondria are involved in integrating\nthe apoptotic and autophagic pathways of cell death.\nLysosomal enzymes might activate apoptosis because\nlysosome rupture activates apoptosis. It is interesting\nto note that apoptosis after lysosome rupture is pre-\nceded by autophagy. Sequestration of mitochondria\nwithin autophagic vacuoles might protect cells from\napoptosis by delaying the release of cytochrome c.\nThus, autophagy might regulate apoptosis through ly-\nsosome formation and enzyme activity. With regard to\nthe possibility of signaling between lysosomes and\nmitochondria, a recent study reveals that production\nof H2O2 from ROS in response to DNA damage acti-\nvates lysosomal enzymes, such as cathepsin D and B.68\nCathepsin B directly or indirectly results in transloca-\ntion of proapoptotic proteins, such as Bid and Bax,\nfrom the cytoplasm to mitochondria. This alters MMP\nand results in caspase activation subsequent to the\nrelease of cytochrome c, Smac/Diablo, and AIF from\nthe mitochondria. Conversely, under different circum-\nstances, production of H2O2 from ROS in response to\nDNA damage may inhibit the release of small mole-\ncules, such as cytochrome c, Smac/Diablo, and AIF,\nfrom the mitochondrial membrane and activate lyso-\nsomal activity. Thus, crosstalk might occur between\nthe pathways mediating autophagy and apoptosis by\nway of lysosomes and mitochondria. Further research\nis required to clarify the signaling that occurs between\nthese pathways.\nFUTURE PERSPECTIVES REGARDING CELL DEATH\nAND CANCER THERAPY\nIncreasing information is known regarding the molec-\nular mechanism(s) by which cell death is mediated by\nanticancer drugs. Proapoptotic and antiapoptotic pro-\nteins are now being targeted to enhance the effect(s)\nof chemotherapy. In general, the antitumor effect of\nan anticancer agent is measured in terms of its ability\nto induce apoptotic cell death within tumors, which is\nwell correlated with outcome in most preclinical and\nclinical studies. Nevertheless, despite signiﬁcant re-\nductions in tumor volume, high rates of apoptosis do\nnot always occur within solid tumors after chemother-\napy. This suggests that other types of cell death, such\nas autophagy and necrosis, might also occur as a result\nCANCER April 15, 2005 / Volume 103 / Number 8\nof chemotherapy. Research into other types of cell\ndeath, particularly autophagic cell death, is important\nto better understand the effects of anticancer drugs. In\naddition, the relation between autophagic cell death\nand apoptosis needs to be examined, because several\nautophagy-related proteins, including Beclin 1, BNIP3,\nDAPK, and HSpin 1, may inﬂuence apoptosis. Because\napoptosis and autophagy are mutually exclusive in\nsome cell lines, molecular switches between the two\nprocesses must exist.\nResistance to autophagy, as well as apoptosis,\nmight lend resistance to anticancer drugs. Likewise,\ninduction of autophagic cell death in cancer cells may\nbe of therapeutic beneﬁt. Cells transformed with the\ntumor suppressor gene Beclin 1 are less likely to un-\ndergo autophagy and are more susceptible to onco-\ngenesis. Alterations in autophagic genes reduce the\nincidence of autophagy in cancer cells. However, even\nin cells resistant to apoptosis or cells in which apo-\nptosis is inhibited, nonapoptotic cell death pathways,\nsuch as autophagy, may be activated. Targeting alter-\nnative pathways of cell death might overcome drug\nresistance or explain why the effects of chemotherapy\nand irradiation complement each other in the clinical\nsetting. Alterations or defects in apoptotic signal\ntransduction pathways might result in drug resistance,\nhowever, irradiation induces autophagic cell death.\nAlterations of one or both pathways (autophagic or\napoptotic) might determine susceptibility to cell death\nafter chemotherapy because anticancer drugs might\ntrigger both apoptotic caspase-dependent and auto-\nphagic caspase-independent pathways at the same\ntime.\nAutophagic cell death might also inhibit angio-\ngenesis.69 The angiogenesis inhibitor, endostatin, in-\nduces endothelial cell death via the caspase-indepen-\ndent autophagic pathway. In addition, the tumor\nsuppressor gene, PTEN, which induces autophagy, in-\nhibits tumor angiogenesis in nude mice with ortho-\ntopic brain tumors.70 Reconstitution of wild-type\nPTEN has no effect on in vitro proliferation, but dra-\nmatically decreases tumor growth in vivo and pro-\nlongs survival in mice, suggesting that PTEN regulates\ntumor-induced angiogenesis via regulation of PI-de-\npendent signals. The role of autophagy in tumor an-\ngiogenesis might be an important means of inhibiting\ntumor progression and increasing the effectiveness of\nanticancer therapy.\nIt is now clear that a number of different types of\ncell death, including apoptosis and autophagy, play a\nrole in the antitumor effect(s) of treatment with anti-\ncancer drugs. Targeting therapy at genes related to\napoptosis might increase the therapeutic efﬁcacy of\nanticancer agents by modulating signal transduction\npathways, and other cell death pathways might be\ninﬂuenced by changes in the microenvironment and\nby crosstalk between the various cell death pathways.\nFurther research into the molecular mechanism(s) be-\nhind autophagic cell death as a result of chemother-\napy might provide a new strategy for overcoming drug\nresistance by exploring this alternative pathway of cell\ndeath.",
         null
        ],
        [
         "27",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "Biochemical and spectroscopic characterization of the bacterial phytochrome of Pseudomonas aeruginosa",
         "Phytochromes are photochromic biliproteins found in plants as well as in some cyanotrophic, photoautotrophic and heterotrophic bacteria. In many bacteria, their function is largely unknown. Here we describe the biochemical and spectroscopic characterization of recombinant bacterial phytochrome from the opportunistic pathogen Pseudomonas aeruginosa (PaBphP). The recombinant protein displays all the characteristic features of a bonafide phytochrome. In contrast with cyanobacteria and plants, the chromophore of this bacterial phytochrome is biliverdin IXα, which is produced by the heme oxygenase BphO in P. aeruginosa. This chromophore was shown to be covalently attached via its A‐ring endo‐vinyl group to a cysteine residue outside the defined bilin lyase domain of plant and cyanobacterial phytochromes. Site‐directed mutagenesis identified Cys12 and His247 as being important for chromophore binding and photoreversibility, respectively. PaBphP is synthesized in the dark in the red‐light‐absorbing Pr form and immediately converted into a far‐red‐light‐absorbing Pfr‐enriched form. It shows the characteristic red/far‐red‐light‐induced photoreversibility of phytochromes. A chromophore analog that lacks the C15/16 double bond was used to show that this photoreversibility is due to a 15Z/15E isomerization of the biliverdin chromophore. Autophosphorylation of PaBphP was demonstrated, confirming its role as a sensor kinase of a bacterial two‐component signaling system.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2005.04623.x",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "28",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "Mycobacteria use their surface-exposed glycolipids to infect human macrophages through a receptor-dependent process Published, JLR Papers in Press, December 1, 2004. DOI 10.1194/jlr.M400308-JLR200",
         "Two subfamilies of the polar glycopeptidolipids (GPLs) located on the surface of Mycobacterium smegmatis, along with unknown phospholipids, were recently shown to participate in the nonopsonic phagocytosis of mycobacteria by human macrophages (Villeneuve, C., G. Etienne, V. Abadie, H. Montrozier, C. Bordier, F. Laval, M. Daffe, I. Maridonneau-Parini, and C. Astarie-Dequeker. 2003. Surface-exposed glycopeptidolipids of Mycobacterium smegmatis specifically inhibit the phagocytosis of mycobacteria by human macrophages. Identification of a novel family of glycopeptidolipids. J. Biol. Chem. 278: 51291–51300). As demonstrated herein, a phospholipid mixture that derived from the methanol-insoluble fraction inhibited the phagocytosis of M. smegmatis. Inhibition was essentially attributable to phosphatidylinositol mannosides (PIMs), namely PIM2 and PIM6, because the purified phosphatidylethanolamine, phosphatidylglycerol, and phosphatidylinositol were inactive. This was further confirmed using purified PIM2 and PIM6 from M. bovis BCG that decreased by half the internalization of M. smegmatis. Both compounds also inhibited the uptake of M. tuberculosis and M. avium but had no effect on the internalization of zymosan used as a control particle of the phagocytic process. When coated on latex beads, PIM2 and polar GPL (GPL III) favored the particle entry through complement receptor 3. GPL III, but not PIM2, also directed particle entry through the mannose receptor. Therefore, surface-exposed mycobacterial PIM and polar GPL participate in the receptor-dependent internalization of mycobacteria in human macrophages.",
         "",
         "http://www.jlr.org/article/S0022227520340323/pdf",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "29",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "Insights from the developing world: thrifty genotypes and thrifty phenotypes",
         "Few researchers would dispute that the pandemic of obesity is caused by a profound mismatch between humanity's present environmental circumstances and those that have moulded evolutionary selection. This concept was first articulated when gestational diabetes was described as being the result of a ‘thrifty genotype rendered detrimental by progress’. More recently, this hypothesis has been extended to the concept of a ‘thrifty phenotype’ to describe the metabolic adaptations adopted as a survival strategy by a malnourished fetus; changes that may also be inappropriate to deal with a later life of affluence. Both the thrifty genotype and the thrifty phenotype hypotheses would predict that populations in some areas of the developing world would be at greater risk of obesity and its co-morbidities; a proposition to be explored in the present paper. To date thrifty genes remain little more than a nebulous concept propagated by the intuitive logic that man has been selected to survive episodic famine and seasonal hungry periods. Under such conditions those individuals who could lay down extra energy stores and use them most efficiently would have a survival advantage. The search for candidate thrifty genes needs to cover every aspect of human energy balance from food-seeking behaviour to the coupling efficiency of oxidative phosphorylation. The present paper will describe examples of attempts to find thrifty genes in three selected candidate areas: maternally-transmitted mitochondrial genes; the uncoupling proteins; apoE4, whose geographical distribution has been linked to a possible thrifty role in lipoprotein and cholesterol metabolism.",
         "Fetal origins of insulin resistance and obesity Variation at the insulin gene VNTR (variable number tandem repeat) polymorphism and early growth: studies in a large Finnish birth cohort. Apolipoprotein E polymorphism and fertility: a study in pre-industrial populations. Opposing influences of prenatal and postnatal weight gain on adrenarche in normal boys and girls. Maternal-fetal interactions and birth order influence insulin variable number of tandem repeats allele class associations with head size at birth and childhood weight gain. The human obesity gene map: the 2003 update. Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA Intrauterine factors, adiposity, and hyperinsulinaemia Global haplotype diversity in the human insulin gene region. Birthweight and perinatal mortality: paradoxes, social class, and sibling dependencies. The double puzzle of diabetes The epidemiology of adverse pregnancy outcomes: an overview. Neonatal anthropometry: the thin–fat Indian baby. The Pune Maternal Nutrition Study Adiposity and hyperinsulinemia in Indians are present at birth. The role of uncoupling proteins in pathophysiological states. Racial differences in the relation between uncoupling protein genes and resting energy expenditure. An overview on the nutrition transition and its health implications: the Bellagio meeting An adipose-specific control of thermogenesis in body weight regulation Insulin gene variable number of tandem repeat genotype and the low birth weight, precocious pubarche, and hyperinsulinism sequence. On the importance--and the unimportance--of birthweight. The thrifty phenotype hypothesis. Obesity and undernutrition and cardiovascular risk factors in rural and urban Gambian communities. Glucose, insulin and lipid metabolism in rural Gambians exposed to early malnutrition Uncoupling proteins: functional characteristics and role in the pathogenesis of obesity and Type II diabetes Uncoupling proteins: their roles in adaptive thermogenesis and substrate metabolism reconsidered Fires of life: the struggles of an ancient metabolism in a modern world The nutrition transition and obesity in the developing world. Thrifty Genotypes and Phenotypes in the Pathogenesis of Type 2 Diabetes Mellitus Uncoupling to survive? The role of mitochondrial inefficiency in ageing The Fetal and Childhood Growth of Persons Who Develop Type 2 Diabetes Energy adaptations in human pregnancy: limits and long-term consequences. Interactions of perturbations in intrauterine growth and growth during childhood on the risk of adult-onset disease Insulin resistance syndrome in 8-year-old Indian children: small at birth, big at 8 years, or both? Transmission ratio distortion at the INS-IGF2 VNTR Apolipoprotein E (APOE) allele distribution in the world. Is APOE*4 a ‘thrifty’ allele? The insulin gene VNTR, type 2 diabetes and birth weight The thrifty genotype in type 2 diabetes The INS 5′ Variable Number of Tandem Repeats Is Associated with IGF2 Expression in Humans* Association of the INS VNTR with size at birth The Great Famine: Northern Europe in the Early Fourteenth Century Bergmann's rule and the thrifty genotype. Imprinted and genotype-specific expression of genes at the IDDM2 locus in pancreas and leucocytes. Susceptibility to human type 1 diabetes at IDDM2 is determined by tandem repeat variation at the insulin gene minisatellite locus Biochemical control logic and the metabolism of glutamine. Type 2 (non-insulin-dependent) diabetes mellitus: the thrifty phenotype hypothesis The possible role of glutamine substrate cycles in skeletal muscle. Maternal weight gain, infant birth weight, and diet: causal sequences. Role of triglyceride-fatty acid cycle in controlling fat metabolism in humans during and after exercise. A Polymorphic Locus Near the Human Insulin Gene Is Associated with Insulin-dependent Diabetes Melliitus Poverty and famine: An essay on entitlement and deprivation : , Oxford, New York: Oxford University Press, 1981. xi + 257 pp., index. $17.95 Malnutrition and fertility. Poverty and famines. An essay on entitlement and deprivation [incl. case studies of famines in Bengal, Ethiopia, Sahel, and Bangladesh]. Famine in Tudor and Stuart England. The high prevalence of diabetes mellitus on a Central Pacific island Famine and Human Development: The Dutch Hunger Winter 1944-1945 Faminee of history and of today Diabetes in American Indians and Other Native Populations of the New World The determinants of growth and form. Review lecture - The determinants of growth and form Diabetes mellitus: a \"thrifty\" genotype rendered detrimental by \"progress\"? Biology of Human Starvation A critical evaluation of the fetal origins hypothesis and its implications for developing countries. Lack of support for a role of the insulin gene variable number of tandem repeats minisatellite (INS-VNTR) locus in fetal growth or type 2 diabetes-related intermediate traits in United Kingdom populations. Differential reproductive efficiency associated with common apolipoprotein e alleles in postreproductive-aged subjects. Eating, exercise, and \"thrifty\" genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases. The insulin gene variable number tandem repeat class I/III polymorphism is in linkage disequilibrium with birth weight but not Type 2 diabetes in the Pima population. Floods, Famines and Emperors. London: Pimlico PPARgamma, the ultimate thrifty gene. Fetal origins of adult disease-the hypothesis revisited. Genomic imprinting of fetal growth The Demography of Famine: An Indian Historical Perspective Biochemical control logic and the metabolism of glutamine. Agriculture’s mixed blessings The Third Chimpanzee Determinants of low birth weight: methodological assessment and meta-analysis. Origin of Species",
         "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D4334A1A0F9096B08FAE923BAB0CA36D/S0029665105000194a.pdf/div-class-title-insights-from-the-developing-world-thrifty-genotypes-and-thrifty-phenotypes-div.pdf",
         "",
         "",
         "",
         "DISCUSSION",
         "",
         "DISCUSSION",
         null
        ],
        [
         "30",
         "00206ca6a32791adcca5effdfed6eecd035075f4",
         "Sparse basis selection: new results and application to adaptive prediction of video source traffic",
         "Real-time prediction of video source traffic is an important step in many network management tasks such as dynamic bandwidth allocation and end-to-end quality-of-service (QoS) control strategies. In this paper, an adaptive prediction model for MPEG-coded traffic is developed. A novel technology is used, first developed in the signal processing community, called sparse basis selection. It is based on selecting a small subset of inputs (basis) from among a large dictionary of possible inputs. A new sparse basis selection algorithm is developed that is based on efficiently updating the input selection adaptively. When a new measurement is received, the proposed algorithm updates the selected inputs in a recursive manner. Thus, adaptability is not only in the weight adjustment, but also in the dynamic update of the inputs. The algorithm is applied to the problem of single-step-ahead prediction of MPEG-coded video source traffic, and the developed method achieves improved results, as compared to the published results in the literature. The present analysis indicates that the adaptive feature of the developed algorithm seems to add significant overall value.",
         "",
         "http://www.vision.caltech.edu/malaa/publications/atiya05sparse.pdf",
         "",
         "Introduction",
         "",
         "",
         "",
         "Introduction",
         null
        ],
        [
         "31",
         "0020d5bdfc5f13026e7a1aca78e94724e8342b62",
         "A mutation in the Icsbp1 gene causes susceptibility to infection and a chronic myeloid leukemia–like syndrome in BXH-2 mice",
         "BXH-2 mice develop a fatal myeloid leukemia by a two-step mutagenic process. First, a BXH-2–specific recessive mutation causes a myeloproliferative syndrome. Second, retroviral insertions alter oncogenes or tumor suppressors, resulting in clonal expansion of leukemic cells. We have identified a recessive locus on chromosome 8 (Myls) that is responsible for myeloproliferation in BXH-2. This Myls interval has been narrowed down to 2 Mb and found to contain several positional candidates, including the interferon consensus sequence–binding protein 1 gene (Icsbp, also known as interferon regulatory factor 8 [IRF8]). We show that BXH-2 mice carry a mutation (915 C to T) resulting in an arginine-to-cysteine substitution at position 294 within the predicted IRF association domain of the protein. Although expression of Icsbp1 mRNA transcripts is normal in BXH-2 splenocytes, these cells are unable to produce interleukin 12 and interferon-γ in response to activating stimuli, confirming that R294C behaves as a loss-of-function mutation. Myeloproliferation in BXH-2 mice is concomitant to increased susceptibility to Mycobacterium bovis (BCG) despite the presence of resistance alleles at the Nramp1 locus. These results suggest a two-step model for chronic myeloid leukemia in BXH-2, in which inactivation of Icsbp1 predisposes to myeloproliferation and immunodeficiency. This event is required for retroviral replication, and subsequent insertional mutagenesis that causes leukemia in BXH-2 mice.",
         "",
         "https://rupress.org/jem/article-pdf/201/6/881/1153938/jem2016881.pdf",
         "",
         "",
         "RESULTS",
         "DISCUSSION",
         "MATERIALS AND METHODS",
         "MATERIALS AND METHODS\nRESULTS\nDISCUSSION",
         null
        ],
        [
         "32",
         "0022e70878c2f2a7c3c679a3ef3160a5cb803e20",
         "Binding of estrogen receptor beta to estrogen response element in situ is independent of estradiol and impaired by its amino terminus.",
         "The functions of 17beta-estradiol (E2) are mediated by estrogen receptor (ER) alpha and beta. ERs display similar DNA- and ligand-binding properties in vitro. However, ERbeta shows lower transcriptional activity than ERalpha from the estrogen response element (ERE)-dependent signaling. We predicted that distinct amino termini contribute to differences in transcription efficacies of ERs by affecting in situ ER-ERE interactions. We used chromatin immunoprecipitation and a novel in situ ERE competition assay, which is based on the ability of ER to compete for ERE binding with a designer activator that constitutively induces transcription from an ERE-driven reporter construct. Interference of activator-mediated transcription by unliganded or liganded ERs was taken as an indication of ER-ERE interaction. Results revealed that ERs interacted with ERE similarly in the absence of E2. However, E2 enhanced the ERE binding of ERalpha but not that of ERbeta. The removal of the amino terminus increased the ERbeta-ERE interaction independent of E2. The ERbeta amino terminus also prevented E2-mediated enhancement of the chimeric ERalpha-ERE interaction. Thus, the amino terminus of ERbeta impairs the binding of ERbeta to ERE. The abrogation of ligand-dependent activation function 2 of the amino-terminally truncated ERbeta resulted in the manifestation of E2 effect on ERbeta-ERE interaction. This implies that E2-mediated enhancement of ERbeta-ERE interaction is masked by the activation function 2, whereas the intact amino terminus is a dominant region that decreases the binding of ERbeta to ERE. Thus, ERbeta-ERE interaction is independent of E2 and is impaired by its amino terminus. These findings provide an additional explanation for differences between ERalpha and ERbeta functions that could differentially affect the physiology and pathophysiology of E2 signaling.",
         "",
         "https://academic.oup.com/mend/article-pdf/19/11/2696/10717141/mend2696.pdf",
         "",
         "",
         "",
         "",
         "MATERIALS AND METHODS",
         "MATERIALS AND METHODS",
         null
        ],
        [
         "33",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "Sensor node localization using mobile acoustic beacons",
         "We present a mobile acoustic beacon based sensor node localization method. Our technique is passive in that the sensor nodes themselves do not need to generate an acoustic signal for ranging. This saves cost, power and provides stealthy operation. Furthermore, the beacon can generate much more acoustic energy than a severely resource constrained sensor node, thereby significantly increasing the range. The acoustic ranging method uses a linear frequency modulated signal that can be accurately detected by matched filtering. This provides longer range and higher accuracy than the current state-of-the-art. The localization algorithm was especially designed to work in such acoustically reverberant environment, as urban terrain. The algorithm presented handles non-Gaussian ranging errors caused by echoes. Node locations are computed centrally by solving a global non-linear optimization problem in an iterative and incremental fashion",
         "Resilient Localization for Sensor Networks in Outdoor Environments Mobile-assisted localization in wireless sensor networks Sensor network-based countersniper system Robust distributed network localization with noisy range measurements The flooding time synchronization protocol Localization of wireless sensor networks with a mobile beacon Smart sensors to network the world. Range-free localization schemes for large scale sensor networks The n-Hop Multilateration Primitive for Node Localization Problems Ad hoc positioning system (APS) using AOA Mica: A Wireless Platform for Deeply Embedded Networks Middleware challenges for wireless sensor networks The bits and flops of the n-hop multilateration primitive for node localization problems Wireless sensor networks for habitat monitoring A survey on position-based routing in mobile ad hoc networks Robust range estimation using acoustic and multimodal sensing Location Systems for Ubiquitous Computing Smart kindergarten: sensor-based wireless networks for smart developmental problem-solving environments Exposure in wireless Ad-Hoc sensor networks Dynamic fine-grained localization in Ad-Hoc networks of sensors The cricket compass for context-aware mobile applications GPS-free Positioning in Mobile Ad Hoc Networks GPS-less low-cost outdoor localization for very small devices Embedding the Internet: introduction Acoustic Ranging in Resource-Constrained Sensor Networks IEEE Computer Graphics & Applications: Projects In VR Pervasive Pose-Aware Applications and Infrastructure VR Pervasive Pose-Aware Applications and Infrastructure. IEEE Distributed Systems Online Range-free localization schemes in large scale sensor network This copyright notice must be included in the reproduced paper. USENIX acknowledges all trademarks herein. Robust Positioning Algorithms for Distributed Ad-Hoc Wireless Sensor Networks The Design of Calamari : an Ad-hoc Localization System for Sensor Networks Complex Behavior at Scale: An Experimental Study of Low-Power Wireless Sensor Networks Convex Optimization Methods for Sensor Node Position Estimation SpotON: An Indoor 3D Location Sensing Technology Based on RF Signal Strength The Cricket location-support system Global Positioning System Overview (last modified 5.1.2000) Numerical Optimization, pages 250–275 Theory and application of digital signal processing If both | µ i | and σ i are large then again location might be affected by echo and we follow the same procedure as in case 2 above If µ i is close to zero but σ i is large then either the range deviations are spread around zero or few large deviations caused σ i to be large If | µ i | is large but σ i is small then all elements of ∆ i are large i.e. the node location is deﬁnitely incorrect",
         "https://ir.vanderbilt.edu/bitstream/1803/13704/1/ms-thesis-manish-kushwaha.pdf",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "34",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "Impact of Software Vulnerability Announcements on the Market Value of Software Vendors - an Empirical Investigation",
         "Researchers in the area of information security have mainly been concerned with tools, techniques and policies that firms can use to protect themselves against security breaches. However, information security is as much about security software as it is about secure software. Software is not secure when it has defects or flaws which can be exploited by hackers to cause attacks such as unauthorized intrusion or denial of service attacks. Any public announcement makes about a software defect is termed as 'vulnerability disclosure'. Although research in software economics have studied firms' incentive to improve overall quality, there have been no studies to show that software vendors have an incentive to invest in building more secure software. In this paper, we use the event study methodology to examine software vendors' incentives to build more secure software. We collect data from leading national newspapers and industry sources like CERT by searching for reports on published software vulnerabilities. We show that vulnerability disclosures lead to a negative and significant change in market value for a software vendor. On average, a vendor loses around 0.6% value in stock price when a vulnerability is reported. This is equivalent to a loss in market capitalization values of $0.86 billion per vulnerability announcement. To provide further insight, we use the information content of the disclosure announcement to classify vulnerabilities into various types. We find that the change in stock price is more negative if the vendor fails to provide a patch at the time of disclosure. Moreover, vulnerabilities which cause a confidentiality related breach cause a greater decline in the market value for a vendor than the vulnerabilities which cause non-confidentiality related breaches. Also, more severe flaws have a significantly greater impact than flaws with low or moderate severity. Finally, we find that the markets do not punish a software vendor more severely if a third party discovers a flaw in its product than if the vendor itself discovers the flaw. Our analysis provides many interesting implications for software vendors as well as policy makers.",
         "",
         "http://archive.nyu.edu/bitstream/2451/14996/2/Infosec_Book%2bWattal%2bTelang.pdf",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "35",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "Real-Time Polymerase Chain Reaction Quantification of Phytophthora capsici in Different Pepper Genotypes.",
         "ABSTRACT Reliable and sensitive quantification of Phytophthora capsici in pepper plants is of crucial importance in managing the multiple syndromes caused by this pathogen. A real-time polymerase chain reaction (PCR) assay was developed for the determination of P. capsici in pepper tissues. DNA levels of a highly virulent and a less virulent isolate were measured in different pepper genotypes with varying degrees of resistance. Using SYBR Green and specific primers for P. capsici, the minimal amount of pathogen DNA quantified was 10 pg. Pathogen DNA was recorded as early as 8 h postinoculation. Thereafter, the increase was rapid in susceptible cultivars and slower in resistant ones. The amount of pathogen DNA quantified in each pepper genotype correlated with susceptibility to Phytophthora root rot. Likewise, there was a relationship between the virulence of the pathogen and the degree of colonization. Differences also were found in oomycete amount among pepper tissues, with maximal pathogen biomass occurring in stems. The real-time PCR technique developed in this study was sensitive and robust enough to assess both pathogen development and resistance to Phytophthora root rot in different pepper genotypes.",
         "",
         "https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-95-1423",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "36",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "Wing-to-Leg Homeosis by Spineless Causes Apoptosis Regulated by Fish-lips, a Novel Leucine-Rich Repeat Transmembrane Protein",
         "ABSTRACT Growth, patterning, and apoptosis are mutually interactive during development. For example, cells that select an abnormal fate in a developing field are frequently removed by apoptosis. An important issue in this process that needs to be resolved is the mechanism used by cells to discern their correct fate from an abnormal fate. In order to examine this issue, we developed an animal model that expresses the dioxin receptor homolog Spineless (Ss) ectopically in the Drosophila wing. The presence of mosaic clones ectopically expressing ss results in a local transformation of organ identity, homeosis, from wing into a leg or antenna. The cells with misspecified fates subsequently activate c-Jun N-terminal kinase to undergo apoptosis in an autonomous or nonautonomous manner depending on their position within the wing, suggesting that a cell-cell interaction is, at least in some cases, involved in the detection of misspecified cells. Similar position dependence is commonly observed when various homeotic genes controlling the body segments are ectopically expressed. The autonomous and nonautonomous apoptosis caused by ss is regulated by a novel leucine-rich repeat family transmembrane protein, Fish-lips (Fili) that interacts with surrounding normal cells. These data support a mechanism in which the lack of some membrane proteins helps to recognize the presence of different cell types and direct these cells to an apoptotic fate in order to exclude them from the normal developing field.",
         "",
         "https://europepmc.org/articles/pmc1069588?pdf=render",
         null,
         null,
         null,
         null,
         null,
         null,
         null
        ],
        [
         "37",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "The sentinel within: exploiting the immune system for cancer biomarkers.",
         "The release of proteins from tumors triggers an immune response in cancer patients. These tumor antigens arise from several mechanisms including tumor-specific alterations in protein expression, mutation, folding, degradation, or intracellular localization. Responses to most tumor antigens are rarely observed in healthy individuals, making the response itself a biomarker that betrays the presence of underlying cancer. Antibody immune responses show promise as clinical biomarkers because antibodies have long half-lives in serum, are easy to measure, and are stable in blood samples. However, our understanding of the specificity and the impact of the immune response in early stages of cancer is limited. The immune response to cancer, whether endogenous or driven by vaccines, involves highly specific T lymphocytes (which target tumor-derived peptides bound to self-MHC proteins) and B lymphocytes (which generate antibodies to tumor-derived proteins). T cell target antigens have been identified either by expression cloning from tumor cDNA libraries, or by prediction based on patterns of antigen expression (\"reverse immunology\"). B cell targets have been similarly identified using the antibodies in patient sera to screen cDNA libraries derived from tumor cell lines. This review focuses on the application of recent advances in proteomics for the identification of tumor antigens. These advances are opening the door for targeted vaccine development, and for using immune response signatures as biomarkers for cancer diagnosis and monitoring.",
         "T Cell–Dependent Antibody Responses against Aberrantly Expressed Cyclin B1 Protein in Patients with Cancer and Premalignant Disease Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast Cancer A phase I trial of DNA vaccination with a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate cancer Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant induces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans. Self-Assembling Protein Microarrays Protein microarray technology Minor Histocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem Cell Transplantation An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in association with HLA-DR. Development of natural protein microarrays for diagnosing cancer based on an antibody response to tumor antigens. NY-ESO-1 Expression and Immunogenicity in Malignant and Benign Breast Tumors Clinical proteomics: revolutionizing disease detection and patient tailoring therapy. The clinical value of intracellular autoantigens B-cell epitopes in systemic rheumatic diseases. Serological cloning of cancer/testis antigens expressed in prostate cancer using cDNA phage surface display Complete response to donor lymphocyte infusion in multiple myeloma is associated with antibody responses to highly expressed antigens. Humoral immune response against melanoma antigens induced by vaccination with cytokine gene‐modified autologous tumor cells Identification of a shared epitope recognized by melanoma-specific, HLA-A3-restricted cytotoxic T lymphocytes. Genome-wide Characterization of a Viral Cytotoxic T Lymphocyte Epitope Repertoire* Microarrays of tumor cell derived proteins uncover a distinct pattern of prostate cancer serum immunoreactivity Protein arrays for autoantibody profiling and fine‐specificity mapping Protein microarrays: Promising tools for proteomic research Recursive partitioning as an approach to selection of immune markers for tumor diagnosis. Antigen-specific immunity in neuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen. Expression of cancer/testis (CT) antigens in lung cancer. Unique conformation of cancer autoantigen B23 in hepatoma: A mechanism for specificity in the autoimmune response NY-ESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial ovarian cancer. Identification of tumor‐associated antigens by screening phage‐displayed human cDNA libraries with sera from tumor patients Tumor-specific immunological recognition of frameshift-mutated peptides in colon cancer with microsatellite instability. CD1d-restricted T cells regulate dendritic cell function and antitumor immunity in a granulocyte–macrophage colony-stimulating factor-dependent fashion Melanoma inhibitor of apoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction The use of HPLC-MS in T-cell epitope identification. Immunomic analysis of human sarcoma Enhancement of antibody detection in cancer using panel of recombinant tumor-associated antigens. Immunoscreening of phage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease. Integrated functional genomics approach for the design of patient-individual antitumor vaccines. Proteomics Approaches to Uncover the Repertoire of Circulating Biomarkers for Breast Cancer CML28 is a broadly immunogenic antigen, which is overexpressed in tumor cells. Towards patient‐specific tumor antigen selection for vaccination Cancer/testis antigens: an expanding family of targets for cancer immunotherapy Serological identification of antigens associated with renal cell carcinoma. Characterisation of tumour-associated antigens in colon cancer Solid-Phase Epitope Recovery: A High Throughput Method for Antigen Identification and Epitope Optimization Enhanced humoral immune response correlates with improved disease-free and overall survival in American Joint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine. Identification of Breast Cancer-Restricted Antigens by Antibody Screening of SKBR3 cDNA Library Using a Preselected Patient's Serum Posttranslational protein modifications: new flavors in the menu of autoantigens ATP6S1 elicits potent humoral responses associated with immune-mediated tumor destruction Identification of cancer/testis genes by database mining and mRNA expression analysis Autoantigen microarrays for multiplex characterization of autoantibody responses Miniaturised multiplexed immunoassays. Identification of Cyclin B1 as a Shared Human Epithelial Tumor-Associated Antigen Recognized by T Cells Trends in biomarker research for cancer detection. Autoimmune diseases: are markers ready for prediction? Post-translational protein modifications in antigen recognition and autoimmunity. Identification of a tissue-specific putative transcription factor in breast tissue by serological screening of a breast cancer library. Profiling the immune response in patients with breast cancer by phage‐displayed cDNA libraries Tumor-associated antigen TA90 immune complex assay predicts recurrence and survival after surgical treatment of stage I-III melanoma. Evidence that specific T lymphocytes may participate in the elimination of chronic myelogenous leukemia Pre-existent immunity to the HER-2/neu oncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer Profiling the immune responses in patient sera with peptide and cDNA display libraries. Reactivity of natural and induced human antibodies to MUC1 mucin with MUC1 peptides and n‐acetylgalactosamine (GalNAc) peptides Rapid degradation of a large fraction of newly synthesized proteins by proteasomes Serological identification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung cancer. p53 Antibodies in the sera of patients with various types of cancer: a review. Expression of cancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a new CT gene, CT9. CT10: A new cancer‐testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational‐difference analysis p53 gene mutation, tumor p53 protein overexpression, and serum p53 autoantibody generation in patients with breast cancer. Cancer-testis antigens and ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-specific ING1 transcripts and a homologue gene. Antigens recognized by autologous antibody in patients with renal‐cell carcinoma SYFPEITHI: database for MHC ligands and peptide motifs Cleavage by Granzyme B Is Strongly Predictive of Autoantigen Status Humoral immune responses of cancer patients against \"Cancer-Testis\" antigen NY-ESO-1: correlation with clinical events. The telomerase catalytic subunit is a widely expressed tumor-associated antigen recognized by cytotoxic T lymphocytes. Isoforms of the human PDZ-73 protein exhibit differential tissue expression. The 72-kDa Component of Signal Recognition Particle Is Cleaved during Apoptosis* The Central Role of CD4+ T Cells in the Antitumor Immune Response Characterization of human colon cancer antigens recognized by autologous antibodies A Survey of the Humoral Immune Response of Cancer Patients to a Panel of Human Tumor Antigens Human lung cancer antigens recognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor gene locus on chromosome 3p21.3. High-titer HER-2/neu protein-specific antibody can be detected in patients with early-stage breast cancer. SSX: A multigene family with several members transcribed in normal testis and human cancer Autoantibodies to p53 in ovarian cancer patients and healthy women: a comparison between whole p53 protein and 18-mer peptides for screening purposes. A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening. Humoral immune response against p53 protein in patients with colorectal carcinoma DNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to detect all known p53 alterations in human malignancies. Anti-p53 antibodies in sera from patients with chronic obstructive pulmonary disease can predate a diagnosis of cancer. Anti-p53 antibodies in sera of workers occupationally exposed to vinyl chloride. Absence of p53 autoantibodies in sera from glioma patients. A motif in human histidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-coil that is essential for enzymatic activity and contains the major autoantigenic epitope. Clinical utility of the immunocytochemical detection of p53 protein in cytological specimens. Role of bone marrow-derived cells in presenting MHC class I-restricted tumor antigens. Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains. Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-macrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor immunity. Development of antibodies against p53 in lung cancer patients appears to be dependent on the type of p53 mutation. HLA-A2 molecules in an antigen-processing mutant cell contain signal sequence-derived peptides Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry. Very long charge runs in systemic lupus erythematosus-associated autoantigens. Characterization of naturally occurring minor histocompatibility peptides including H-4 and H-Y. The fine specificity of anti-La antibodies induced in mice by immunization with recombinant human La autoantigen. Post-translational modifications of proteins: some problems left to solve. CA125 Antigen Levels in Obstetric and Gynecologic Patients Purification of a human prostate specific antigen. Detection of a transformation-related antigen in chemically induced sarcomas and other transformed cells of the mouse. SPECIFIC CARCINOEMBRYONIC ANTIGENS OF THE HUMAN DIGESTIVE SYSTEM Immunotherapy (APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3 randomized placebo-controlled trial Efficient purification of unique antibodies using peptide affinity-matrix columns. Endogenous immune response to early- and intermediate-stage melanoma is correlated with outcomes and is independent of locoregional relapse and standard prognostic factors. Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem cell transplantation and in healthy female donors. Recombinant antigen expression on yeast surface (RAYS) for the detection of serological immune responses in cancer patients. Immunotherapy of hematologic malignancy. p53 autoantibodies can be indicative of the development of breast cancer relapse. The autoantibody repertoire: searching for order MUC1-like tandem repeat proteins are broadly immunogenic in cancer patients. Epitope identification and vaccine design for cancer immunotherapy. Immunotherapeutic Targets Cancer : Identification of Potential Diagnostic and Cancer-related Serological Recognition of Human Colon Updated Identification of tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by serological screening of a testicular library with breast cancer serum. Humoral immunity to human breast cancer: antigen definition and quantitative analysis of mRNA expression. Monitoring of p53 autoantibodies after resection of colorectal cancer: relationship to operative curability. Gure, A. O. et al. CT10: a new cancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-difference analysis. Int. J. Cancer 85, 726-732 MHCPEP, a database of MHC-binding peptides: update 1997 Cancer,Facts and Figures about.",
         "https://europepmc.org/articles/pmc2522321?pdf=render",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "38",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "Molecular Mechanisms of Cation Transport by the Renal Na+-K+-Cl- Cotransporter",
         "Two variants of the renal Na+-K+-Cl- cotransporter (NKCC2), called NKCC2A and NKCC2F, display marked differences in Na+, Rb+, and Cl- affinities, yet are identical to one another except for a 23-residue membrane-associated domain that is derived from alternatively spliced exons. The proximal portion of these exons is predicted to encode the second transmembrane domain (tm2) in the form of an α-helix, and the distal portion, part of the following connecting segment (cs1a). In recent studies, we have taken advantage of the A–F differences in kinetic behavior to determine which regions in tm2-cs1a are involved in ion transport. Functional characterizations of chimeras in which tm2 or cs1a were interchanged between the variants showed that both regions are important in specifying ion affinities, but did not allow delineating the contribution of individual residues. Here, we have extended these structure-function analyses by studying additional mutants in which variant residues between A and F were interchanged individually in the tm2-cs1a region (amino acid number 216, 220, 223, 229, or 233 in NKCC2). None of the substitutions were found to affect Km (C1-), suggesting that the affinity difference for anion transport is conveyed by a combination of variant residues in this domain. However, 2 substitutions in the tm2 of F were found to affect cation constants specifically; interestingly, one of these mutations (residue 216) only affected Km (Rb+) while the other (residue 220) only affected Km (Na+). We have thus identified two novel residues in NKCC2 that play a key role in cation transport. Because such residues should be adjacent to one another on the vertical axis of the tm2 α-helix, our results imply, furthermore, that the ion transport sites in NKCC2 could be physically linked.",
         "",
         "http://www.jbc.org/article/S0021925820792379/pdf",
         "",
         "Introduction",
         "",
         "",
         "",
         "Introduction",
         null
        ],
        [
         "39",
         "00295936451c8a6b8a982d6ba9ab64cb647370ae",
         "Choline deficiency in mice and humans is associated with increased plasma homocysteine concentration after a methionine load.",
         "BACKGROUND\nElevated concentrations of homocysteine in blood may be an independent risk factor for the development of atherosclerosis. Elevated homocysteine concentrations can be caused by decreased methylation of homocysteine to form methionine, as occurs in folate deficiency. A parallel pathway exists for methylation of homocysteine, in which choline, by way of betaine, is the methyl donor.\n\n\nOBJECTIVE\nOur goal was to determine whether choline deficiency results in a decreased capacity to methylate homocysteine.\n\n\nDESIGN\nC57BL/6J mice were fed diets containing 0, 10, or 35 mmol choline/kg diet for 3 wk. We then administered an oral methionine load to the animals and measured plasma homocysteine concentrations. Also, in a pilot study, we examined 8 men who were fed a diet providing 550 mg choline/d per 70 kg body weight for 10 d, followed by a diet providing almost no choline, until the subjects were clinically judged to be choline deficient or for <or=42 d. A methionine load was administered at the end of each dietary phase.\n\n\nRESULTS\nTwo hours after the methionine load, choline-deficient mice had plasma homocysteine concentrations twice those of choline-fed mice. Four hours after the methionine load, clinically choline-depleted men had plasma homocysteine concentrations that were 35% greater than those in men not choline depleted.\n\n\nCONCLUSION\nThese results suggest that choline, like folate, plays an important role in the metabolism of homocysteine in humans and that response to a methionine load may be useful when assessing choline nutriture.",
         "Choline- and betaine-defined diets for use in clinical research and for the management of trimethylaminuria. Elevated serum creatine phosphokinase in choline-deficient humans: mechanistic studies in C2C12 mouse myoblasts. Betaine as a Determinant of Postmethionine Load Total Plasma Homocysteine Before and After B-Vitamin Supplementation Folate status response to controlled folate intake is affected by the methylenetetrahydrofolate reductase 677C-->T polymorphism in young women. Effect of folic acid on fenofibrate-induced elevation of homocysteine and cysteine. Betaine supplementation lowers plasma homocysteine in healthy men and women. Concentrations of choline-containing compounds and betaine in common foods. Phosphatidylethanolamine N-methyltransferase (PEMT) knockout mice have hepatic steatosis and abnormal hepatic choline metabolite concentrations despite ingesting a recommended dietary intake of choline. Effects of cytidine 5'-diphosphocholine on plasma homocysteine levels in rat. Quantitation of choline and its metabolites in tissues and foods by liquid chromatography/electrospray ionization-isotope dilution mass spectrometry. Hyperhomocysteinemia due to methionine synthase deficiency, cblG: structure of the MTR gene, genotype diversity, and recognition of a common mutation, P1173L. Choline deficiency causes reversible hepatic abnormalities in patients receiving parenteral nutrition: proof of a human choline requirement: a placebo-controlled trial. The 1298A-->C polymorphism in methylenetetrahydrofolate reductase (MTHFR): in vitro expression and association with homocysteine. Determinants of plasma total homocysteine concentration in the Framingham Offspring cohort. Betaine-homocysteine methyltransferase expression in porcine and human tissues and chromosomal localization of the human gene. The use of homocysteine and other metabolites in the specific diagnosis of vitamin B-12 deficiency. Choline deficiency: A cause of hepatic steatosis during parenteral nutrition that can be reversed with intravenous choline supplementation Evidence that homocysteine is an independent risk factor for atherosclerosis in hyperlipidemic patients. Vitamin B-6 deficiency vs folate deficiency: comparison of responses to methionine loading in rats. AIN-93 purified diets for laboratory rodents: final report of the American Institute of Nutrition ad hoc writing committee on the reformulation of the AIN-76A rodent diet. Low plasma free choline is prevalent in patients receiving long term parenteral nutrition and is associated with hepatic aminotransferase abnormalities. Accumulation of 1,2-sn-diradylglycerol with increased membrane-associated protein kinase C may be the mechanism for spontaneous hepatocarcinogenesis in choline-deficient rats. Rapid high-performance liquid chromatographic assay for total homocysteine levels in human serum. Choline, an essential nutrient for humans Lactobacillus casei microbiological assay of folic acid derivatives in 96-well microtiter plates. The active synthesis of phosphatidylcholine is required for very low density lipoprotein secretion from rat hepatocytes. Plasma choline concentration in humans fed parenterally. Betaine in the treatment of homocystinuria due to 5,10-methylenetetrahydrofolate reductase deficiency A rapid method of total lipid extraction and purification. Effect of methotrexate on homocysteine and other sulfur compounds in tissues of rats fed a normal or a defined, choline-deficient diet Pathways and Regulation of Homocysteine Metabolism in Mammals Institute of Medicine, National Academy of Sciences USA. Dietary reference intakes for folate, thiamin, riboflavin, niacin, vitamin B-12, panthothenic acid, biotin, and choline The biosynthesis of choline and its relation to phospholipid metabolism.",
         "https://academic.oup.com/ajcn/article-pdf/81/2/440/23875105/440.pdf",
         "",
         "",
         "RESULTS",
         "DISCUSSION",
         "MATERIALS AND METHODS",
         "MATERIALS AND METHODS\nRESULTS\nDISCUSSION",
         null
        ],
        [
         "40",
         "002a108f32eb157c0b88aea3c6f29d4b614c3b78",
         "Altered gene expression during rat Wolffian duct development following di(n-butyl) phthalate exposure.",
         "Di(n-butyl) phthalate (DBP) is a common plasticizer and solvent that disrupts androgen-dependent male reproductive development in rats. In utero exposure to 500 mg/kg/day DBP on gestation days (GD) 12 to 21 decreases androgen biosynthetic enzymes, resulting in decreased fetal testicular testosterone levels. One consequence of prenatal DBP exposure is malformed epididymides in adult rats. Reduced fetal testosterone levels may be responsible for the malformation, since testosterone is required for Wolffian duct stabilization and their development into epididymides. Currently, little is understood about the molecular mechanisms of Wolffian duct differentiation. The objective of this study was to identify changes in gene expression associated with altered morphology of the proximal Wolffian duct following in utero exposure to DBP. Pregnant Crl:CD(R) (SD) rats were gavaged with corn oil vehicle or 500 mg/kg/day DBP from GD 12 to GD 19 or 21. There were only small morphological differences between control and DBP-exposed Wolffian ducts on GD 19. On GD 21, 89% of male fetuses in the DBP dose group showed marked underdevelopment of Wolffian ducts, characterized by decreased coiling. RNA was isolated from Wolffian ducts on GD 19 and 21. Together with empirical information, cDNA microarrays were used to help identify candidate genes that could be associated with the morphological changes observed on GD 21. These candidate genes were analyzed by real-time RT-PCR. Changes in mRNA expression were observed in genes within the insulin-like growth factor (IGF) pathway, the matrix metalloproteinase (MMP) family, the extracellular matrix, and in other developmentally conserved signaling pathways. On GD 19, immunolocalization of IGF-1 receptor protein demonstrated an increase in cytoplasmic expression in the mesenchymal and epithelial cells. There was also a variable decrease in androgen receptor protein in ductal epithelial cells on GD 19. This study provides insight into the effects of antiandrogens on the molecular mechanisms involved in Wolffian duct development. The altered morphology and changes in gene expression following DBP exposure are suggestive of altered paracrine interactions between ductal epithelial cells and the surrounding mesenchyme during Wolffian duct differentiation due to lowered testosterone production.",
         "",
         "https://academic.oup.com/toxsci/article-pdf/86/1/161/4657319/kfi172.pdf",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "41",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         "Prospective grading of neoplastic change in rat esophagus epithelium using angle-resolved low-coherence interferometry.",
         "Angle-resolved low-coherence interferometry (a/LCI) is used to obtain quantitative, depth-resolved nuclear morphology measurements. We compare the average diameter and texture of cell nuclei in rat esophagus epithelial tissue to grading criteria established in a previous a/LCI study to prospectively grade neoplastic progression. We exploit the depth resolution of a/LCI to exclusively examine the basal layer of the epithelium, approximately 50 to 100 microm beneath the tissue surface, without the need for exogenous contrast agents, tissue sectioning, or fixation. The results of two studies are presented that compare the performance of two a/LCI modalities. Overall, the combined studies show 91% sensitivity and 97% specificity for detecting dysplasia, using histopathology as the standard. In addition, the studies enable the effects of dietary chemopreventive agents, difluoromethylornithine (DFMO) and curcumin, to be assessed by observing modulation in the incidence of neoplastic change. We demonstrate that a/LCI is highly effective for monitoring neoplastic change and can be applied to assessing the efficacy of chemopreventive agents in the rat esophagus.",
         "",
         "https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-10/issue-5/051604/Prospective-grading-of-neoplastic-change-in-rat-esophagus-epithelium-using/10.1117/1.2102767.pdf",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "42",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "The role of fear-relevant stimuli in visual search: a comparison of phylogenetic and ontogenetic stimuli.",
         "It has been argued that phylogenetic fear-relevant stimuli elicit preattentive capture of attention. To distinguish between fear relevance and time of appearance in evolutionary history, the authors compare phylogenetic and ontogenetic fear-relevant and fear-irrelevant stimuli in a visual search task. The authors found no evidence for a special role of phylogenetic fear-relevant stimuli; it seems that fear relevance in general is more important than is the evolutionary age. The pattern of results indicates that attention toward threatening stimuli is mainly affected by a late component that prolongs the disengagement of attention.",
         "Attentional control of the processing of neural and emotional stimuli. Searching for threat Emotion drives attention: detecting the snake in the grass. The face in the crowd revisited: a threat advantage with schematic stimuli. Fear and anxiety: animal models and human cognitive psychophysiology. Facial Expressions of Emotion: Are Angry Faces Detected More Efficiently? A subcortical pathway to the right amygdala mediating \"unseen\" fear. It Takes a Confounded Face to Pop Out of a Crowd On the automatic nature of phobic fear: conditioned electrodermal responses to masked fear-relevant stimuli. Orienting and schizophrenia: stimulus significance, attention, and distraction in a signaled reaction time task. A feature-integration theory of attention The premise of equipotentiality in human classical conditioning: conditioned electrodermal responses to potentially phobic stimuli. One-trial learning and superior resistance to extinction of autonomic responses conditioned to potentially phobic stimuli. Phobias and preparedness The cognitive neuroscience of emotion Fears, phobias, and preparedness: toward an evolved module of fear and fear learning. Unconscious emotion : Evolutionary perspectives, psychophysiological data, and neuropsychological mechanisms A threat imminence approach to human fear responding : direction of threath, aversive contexts, and electrodermal responses Preparedness and electrodermal fear-conditioning: ontogenetic vs phylogenetic explanations. Finding the face in the crowd: an anger superiority effect. Feature analysis in early vision: evidence from search asymmetries. The emotional brain. E-Prime 32306-1270), as editor of the Clinician's Research Digest newsletter for",
         "https://access.archive-ouverte.unige.ch/access/metadata/414fbfe1-4481-4240-92c1-e62a3593f0f0/download",
         null,
         null,
         null,
         null,
         null,
         null,
         "captcha"
        ],
        [
         "43",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "WBN/Kob-Ht rats spontaneously develop dermatitis under conventional conditions: another possible model for atopic dermatitis.",
         "WBN/Kob-Ht rats (Ht-rats) raised under conventional conditions spontaneously developed dermatitis. In this study, we carried out histopathological analysis to elucidate the pathological features of the dermatitis in Ht-rats. We then tried to detect Staphylococcus species recovered from the skin lesions of Ht-rats. We also measured the serum levels of total IgE, IL-4 and IFN-gamma in these rats. The histopathological data indicated that inflammatory cells had infiltrated the skin lesions. Staphylococcus aureus was recovered from the skin lesions, and the serum levels of total IgE and IL-4 were elevated in Ht-rats with dermatitis. These results suggest that dermatitis in Ht-rats is similar to that in the DS-Nh mice, which has recently been proposed as an animal model for human atopic dermatitis.",
         "Role of the Nh (Non-hair) mutation in the development of dermatitis and hyperproduction of IgE in DS-Nh mice. DS–Nh as an experimental model of atopic dermatitis induced by Staphylococcus aureus producing staphylococcal enterotoxin C Characterization of dermatitis arising spontaneously in DS-Nh mice maintained under conventional conditions: another possible model for atopic dermatitis. Atopic dermatitis: new insights and opportunities for therapeutic intervention. Locus of dominant hairless gene (Ht) causing abnormal hair and keratinization maps to rat chromosome 10. Characteristics of spontaneous erythema appeared in hairless rats. IgE production by normal human lymphocytes is induced by interleukin 4 and suppressed by interferons gamma and alpha and prostaglandin E2.",
         "https://www.jstage.jst.go.jp/article/expanim/54/5/54_5_461/_pdf",
         "",
         "1 Introduction",
         "",
         "",
         "",
         "1 Introduction",
         null
        ],
        [
         "44",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "Structural Analyses Reveal Phosphatidyl Inositols as Ligands for the NR5 Orphan Receptors SF-1 and LRH-1",
         "None",
         "The DEAD-Box Protein DP103 (Ddx20 or Gemin-3) Represses Orphan Nuclear Receptor Activity via SUMO Modification Amplification of the steroidogenic factor 1 gene in childhood adrenocortical tumors. Identification of liver receptor homolog-1 as a novel regulator of apolipoprotein AI gene transcription. Synergy between LRH-1 and beta-catenin induces G1 cyclin-mediated cell proliferation. Quantitative Proteomics of the Thyroid Hormone Receptor-Coregulator Interactions*[boxs] LRH-1: an orphan nuclear receptor involved in development, metabolism and steroidogenesis. Sex determination: a ‘window’ of DAX1 activity Generation and Characterization of Human Steroidogenic Factor 1 LBD Crystals with and without Bound Cofactor Peptide SHIP-2 and PTEN Are Expressed and Active in Vascular Smooth Muscle Cell Nuclei, but Only SHIP-2 Is Associated with Nuclear Speckles* Resurrecting the Ancestral Steroid Receptor: Ancient Origin of Estrogen Signaling MrBayes 3: Bayesian phylogenetic inference under mixed models Activation of nuclear receptors: a perspective from structural genomics. StAR-related Lipid Transfer (START) Proteins: Mediators of Intracellular Lipid Metabolism* The Drosophila Orphan Nuclear Receptor DHR38 Mediates an Atypical Ecdysteroid Signaling Pathway Structural basis for ligand-independent activation of the orphan nuclear receptor LRH-1. Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors Nuclear lipid signalling Nuclear lipids: New functions for old molecules? X-Ray Structure of the hRORα LBD at 1.63 Å Elevation of plasma phospholipid transfer protein in transgenic mice increases VLDL secretion Published, JLR Papers in Press, August 16, 2002. DOI 10.1194/jlr.M200166-JLR200 Crystal Structure of the HNF4α Ligand Binding Domain in Complex with Endogenous Fatty Acid Ligand* Phosphorylation and Intramolecular Stabilization of the Ligand Binding Domain in the Nuclear Receptor Steroidogenic Factor 1 Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids. Inhibition of androgen receptor (AR) function by the reproductive orphan nuclear receptor DAX-1. Control of Cholesterol Turnover in the Mouse* Phosphatidic Acid-Mediated Mitogenic Activation of mTOR Signaling The PITP family of phosphatidylinositol transfer proteins Characterization of a G Protein-activated Phosphoinositide 3-Kinase in Vascular Smooth Muscle Cell Nuclei* Binding of ligands and activation of transcription by nuclear receptors. Structure of a Multifunctional Protein Crystal Structure of the Ligand-binding Domain of the Ultraspiracle Protein USP, the Ortholog of Retinoid X Receptors in Insects* Haploinsufficiency of steroidogenic factor-1 in mice disrupts adrenal development leading to an impaired stress response. Nuclear Lipid Signaling Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors. Ligand binding and nuclear receptor evolution A new method to localize and test the significance of incongruence: detecting domain shuffling in the nuclear receptor superfamily. Inferring the historical patterns of biological evolution Oxysterol Regulation of Steroidogenic Acute Regulatory Protein Gene Expression Structure and specificity of nuclear receptor-coactivator interactions. 25-Hydroxycholesterol is not a ligand for the orphan nuclear receptor steroidogenic factor-1 (SF-1). Wilms' Tumor 1 and Dax-1 Modulate the Orphan Nuclear Receptor SF-1 in Sex-Specific Gene Expression Activation of the orphan nuclear receptor steroidogenic factor 1 by oxysterols. DAX-1 inhibits SF-1-mediated transactivation via a carboxy-terminal domain that is deleted in adrenal hypoplasia congenita A new method of inference of ancestral nucleotide and amino acid sequences. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. The nuclear receptor steroidogenic factor 1 acts at multiple levels of the reproductive axis. A cell-specific nuclear receptor is essential for adrenal and gonadal development and sexual differentiation Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology Repression of orphan nuclear recep - or activity via SUMO - modification and the Dead - Box Protein P 103 ( Ddx 20 , Gemin - 3 ) Nuclear lipids: key signaling effectors in the nervous system and other tissues X-ray structure of the hRORalpha LBD at 1.63 A: structural and functional data that cholesterol or a cholesterol derivative is the natural ligand of RORalpha. Nuclear-receptor ligands and ligand-binding domains. Standard Fmoc protocols. Methods Enzymol",
         "http://www.cell.com/article/S009286740500098X/pdf",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "45",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "Rat Strain Effects of AT1 Receptor Activation on D1 Dopamine Receptors in Immortalized Renal Proximal Tubule Cells",
         "The dopaminergic and renin-angiotensin systems regulate blood pressure, in part, by affecting sodium transport in renal proximal tubules (RPTs). We have reported that activation of a D1-like receptor decreases AT1 receptor expression in the mouse kidney and in immortalized RPT cells from Wistar-Kyoto (WKY) rats. The current studies were designed to test the hypothesis that activation of the AT1 receptor can also regulate the D1 receptor in RPT cells, and this regulation is aberrant in spontaneously hypertensive rats (SHRs). Long-term (24 hours) stimulation of RPT cells with angiotensin II, via AT1 receptors increased total cellular D1 receptor protein in a time- and concentration-dependent manner in WKY but not in SHR cells. Short-term stimulation (15 minutes) with angiotensin II did not affect total cellular D1 receptor protein in either rat strain. However, in the short-term experiments, angiotensin II decreased cell surface membrane D1 receptor protein in WKY but not in SHR cells. D1 and AT1 receptors colocalized (confocal microscopy) and their coimmunoprecipitation was greater in WKY than in SHRs. However, AT1/D1 receptor coimmunoprecipitation was decreased by angiotensin II (10−8M/24 hours) to a similar extent in WKY (−22±8%) and SHRs (−22±12%). In summary, these studies show that AT1 and D1 receptors interact differently in RPT cells from WKY and SHRs. It is possible that an angiotensin II-mediated increase in D1 receptors and dissociation of AT1 from D1 receptors serve to counter regulate the long-term action of angiotensin II in WKY rats; different effects are seen in SHRs.",
         "",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.HYP.0000184251.01159.72",
         "",
         "Introduction",
         "",
         "",
         "",
         "Introduction",
         null
        ],
        [
         "46",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         "The value of c-kit mutational analysis in a cytokeratin positive gastrointestinal stromal tumour",
         "The expression of cytokeratins in gastrointestinal stromal tumours (GISTs) is rare and may lead to diagnostic confusion when it occurs. This report describes a metastatic GIST that stained strongly for cytokeratins, CD117, and CD34 in a patient who was previously diagnosed with gastric epithelioid angiosarcoma. A review of both tumours showed the same histological and immunohistochemical profiles, and c-kit molecular analysis revealed an insertional mutation at codon 558 of exon 11 in both tumours. Thus, pathologists should be aware that GISTs can occasionally express cytokeratins, and that c-kit mutational investigations may have a key diagnostic role and may prevent diagnostic mistakes that could have important clinical implications.",
         "",
         "https://jcp.bmj.com/content/jclinpath/58/9/991.full.pdf",
         "",
         "",
         "",
         "",
         "Methods",
         "Methods",
         null
        ],
        [
         "47",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "Contribution of high p34cdc2 kinase activity to premature chromosome condensation of injected somatic cell nuclei in rat oocytes.",
         "The present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated rat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes were placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-dependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15-45 min rather than 46-120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity transiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-phase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed oocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat oocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent level of aged (120 min) or enucleated (+60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2 kinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were treated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated oocytes. A significantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control oocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus cell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage. These results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for their inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemicals such as MG132 is in part effective for rat oocytes to promote PCC and further development.",
         "",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/129/2/1290171.pdf",
         "1National Institute for Physiological Sciences, Okazaki 444-8787, Japan, 2The Graduate University of\nAdvanced Studies, Okazaki 444-8787, Japan, 3CREST of Japan Science and Technology Agency, Kawaguchi\n332-0012, Japan, 4Faculty of Textile Science and Technology, Shinshu University, Ueda 386-8567, Japan and\n5Graduate School of Biosphere Science, Hiroshima University, Higashi-Hiroshima 739-8528, Japan\nAbstract\nThe present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated\nrat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes\nwere placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-\ndependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15– 45 min\nrather than 46– 120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity tran-\nsiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-\nphase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed\noocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat\noocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent\nlevel of aged (120 min) or enucleated (1 60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2\nkinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were\ntreated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated\noocytes. A signiﬁcantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control\noocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus\ncell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage.\nThese results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for\ntheir inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemi-\ncals such as MG132 is in part effective for rat oocytes to promote PCC and further development.\nReproduction (2005) 129 171–180\nSuccessful animal cloning by somatic cell nuclear transfer\n(NT) has been reported in large domestic species (Wilmut\net al. 1997, Kato et al. 1998, Baguisi et al. 1999, Onishi\net al. 2000, Galli et al. 2003). As for small rodents,\nWakayama et al. (1998) were the ﬁrst to produce cloned\nmice using G0/G1-phase cumulus cells, and very recently\nZhou et al. (2003) reported the production of fertile cloned\nrats using M-phase fetal ﬁbroblast cells. However, the suc-\ncess rates of producing cloned rodents by NT are still low.",
         "Introduction\nresulting in premature chromosome condensation (PCC),\nwith subsequent multiple pseudo-pronuclei formation fol-\nlowing activation (Wakayama et al. 1998, 1999, Ogura\net al. 2000). It is considered in rodents, at least, that PCC\nis a critical aspect contributing to successful cloning\n(Wakayama et al. 1999, Ogura et al. 2000). However,\nPCC is not essential for somatic cell cloning in all species,\nsince telophase-II\n(TII) oocytes have promoted embryo\ndevelopment in goats (Baguisi et al. 1999) and cattle (Kur-\nosaka et al. 2002).\nThere is a major difference between mouse and rat, in\nthe behavior of somatic nuclei upon exposure to oocyte\ncytoplasm. We have reported that the PCC of both mouse\nand rat cumulus cell nuclei occurred in enucleated mouse\noocytes but not at all\nin enucleated rat oocytes\n(Hirabayashi et al. 2003a). Even if cumulus nuclei were\ninjected into the rat oocytes without enucleation, the rate\nof PCC formation was less than half (Hirabayashi et al.\n2003b).\nWe also reported that\nthe ability of rat oocytes to\npromote PCC of the injected nuclei was dependent on\nthe characteristics of the oocytes, such as age (4- to 5-\nweek-old, superior to .10-week-old rats) or strain of the\ndonor rats (Wistar and LEW, superior to the Donryu and\nF344 strains) and the timing of oocyte recovery (14 h post-\nhuman chorionic gonadotropin (hCG), superior to 17 h\npost hCG) (Hirabayashi et al. 2003b). In the same study,\nthe time management of NT completion (less than 45 min\nafter donor animal killing) and addition to the oocyte-\nhandling media\nof N-acetylleucylleucylnorleucinal\n(ALLN), a neutral cysteine protease inhibitor, were also\nfound important for intact rat oocytes to promote PCC.\nThe successful production of cloned rats (Zhou et al.\n2003) may be attributed to injecting the metaphase-\narrested fetal ﬁbroblasts into oocytes treated with MG132,\na proteasome inhibitor, followed by quick removal of the\nrecipient metaphase-plate. Therefore, it is a possibility that\nrecipient oocytes arrested at the metaphase-II (MII) stage\nby the inhibition of protein degradation have essential\nroles in nuclear reprogramming including PCC formation.\nOvulated rat oocytes are known to activate spon-\ntaneously but abortively during in vitro culture (Zernicka-\nGoetz 1991). We also conﬁrmed that more than half of\nthe oocytes from Wistar rats progressed to anaphase-II\n(AII)/TII stages at 70 min after killing of\nthe donor rats\n(Kato et al. 2001). In vertebrate oocytes, it has been con-\nsidered that high activity of maturation promoting factor\n(MPF) affects the chromosome condensation and the\nmaintenance of the meiotic spindle, both of which are\ninvolved in the meiotic arrest at\nthe MII stage (Jones\n2004). MPF is a heterodimer composed of p34cdc2 kinase\nand cyclin B1, controlling the cell cycle. When the\nactivity was inactivated by the degradation of cyclin B\n(Nurse 1990), oocytes were released from the arrest at the\nMII stage. The p34cdc2 kinase in oocytes was maintained\nat a high level until ovulation in many species (Kubiak\net al. 1993, Naito et al. 1995, Abrieu et al. 2001, Ito &\nShimada 2005), suggesting that p34cdc2 kinase activity in\nrat ovulated oocytes was rapidly decreased in vitro. More-\nover, destruction of\nthe nuclear envelope in mitotic\nsomatic cells occurred during the G2/M-phase transition\nby p34cdc2 kinase-related phosphorylation of lamin, which\nis one of\nthe nuclear membrane proteins (Peter et al.\n1990). It has been reported that ALLN, which was used in\nour previous study (Hirabayashi et al. 2003b), has an\neffect on the inhibition of cyclin B destruction (Sherwood\net al. 1993). Therefore, it is possible that the low rate of\nPCC of injected somatic cell nuclei resulted from rapidly\ndecreasing p34cdc2 kinase activity in rat ovulated oocytes,\nwhile direct evidence that p34cdc2 kinase activity is\ninvolved in the PCC incidence has not yet been shown.\nClarifying the relationship between p34cdc2 kinase activity\nand PCC incidence would contribute to understanding the\nreprogramming mechanism in mammalian\nnuclear\noocytes.\nIn the present study, two experiments were conducted\nto examine the effects of in vitro aging and enucleation of\nrat oocytes on the kinetics of p34cdc2 kinase and the inci-\ndence of PCC of microinjected cumulus cell nuclei, with\nsome comparisons with mouse oocytes. In addition, two\nmore experiments were conducted to explain a possible\nrole of p34cdc2 kinase for promoting PCC, by treatment of\nthe rat oocytes with roscovitine, an inhibitor of p34cdc2\nkinase (Lazar et al. 2002), and MG132, a proteasome\ninhibitor which has an effect on the inhibition of cyclin\nB1 degradation (Josefsberg et al. 2000).\nMaterials and Methods\nOocyte preparation",
         "The effect of in vitro aging on the p34cdc2 kinase activity\nand the PCC incidence was investigated using rat oocytes.\nTen oocytes for each test were sampled for\nin vitro\np34cdc2 kinase assay immediately after recovery (15 min\ngroup), and after culture for 30 min (45 min group), 60 min\n(75 min group) or 105 min (120 min group) in mKRB at\n37 8C in 5% CO2 in air. Nuclear conﬁguration of\nthe\noocytes was also examined immediately after recovery\nand every subsequent 30 min up to 120 min. The other\noocytes were placed in Hepes-mKRB, and the cumulus\ncell nuclei were then injected into the oocytes up to\n120 min. The injected oocytes were divided into two\ngroups based on the time from the animal killing to NT\ncompletion (15 –45 min group and 46 – 120 min group).\nThe incidence of PCC was evaluated 1 h after the last\ninjection among each group.\nExperiment 2: effect of enucleation\nThe effect of oocyte enucleation on the p34cdc2 kinase\nactivity and the PCC incidence was investigated using\nboth rat and mouse oocytes. For the in vitro p34cdc2\nkinase assay, oocytes derived from the same donor rats\nand mice were allocated to the non-enucleation group\nand the enucleation group, and the non-enucleated and\nenucleated oocytes were sampled 45 min after the animal\nkilling. Enucleation was performed by aspiration of the\nmetaphase-plate with a part of the volume (,10–20%) of\ncytoplasm in Hepes-R1ECM containing\nsurrounding\n5 mg/ml cytochalasin B (Sigma-Aldrich). In addition to the\nnon-enucleation and enucleation groups, denuded rat\noocytes were allocated to a cytoplasm-removal group\n(a part of the cytoplasm was mechanically aspirated as a\nsham control.). All the treatments were completed within\n45 min. The oocyte samples were harvested at 45, 75, 90\nand 105 min (0, 30, 45 and 60 min after enucleation\nrespectively). For PCC evaluation, nuclear injection into\nnon-enucleated and enucleated mouse and rat oocytes,\nand into cytoplasm-removed rat oocytes, was completed\nwithin 45 min after animal killing.\nExperiment 3: effect of roscovitine treatment\nWe investigated whether the decreased level of p34cdc2\nkinase activity in non-enucleated rat oocytes negatively\naffected the PCC incidence of injected cell nuclei. Ros-\ncovitine (Sigma-Aldrich) dissolved at 10 mM in DMSO\nand stored at 2 80 8C, a speciﬁc inhibitor of p34cdc2\nkinase, was added to all media throughout the process of\noocyte recovery and the in vitro culture at the ﬁnal con-\ncentration of 150 mM, as reported by Lazar et al. (2002).\nOocytes were sampled for in vitro p34cdc2 kinase assay at\n15 (immediately after recovery), 45, 75 and 120 min as in\nExperiment 1. Because the 45 min treatment of rat oocytes\nwith 150 mM roscovitine decreased their p34cdc2 kinase\nactivity to an equivalent level of oocytes 120 min after in\nvitro aging or 60 min after enucleation, nuclear injection\ninto the roscovitine-treated oocytes was performed within\n45 min, and the incidence of PCC was then evaluated\nafter a further 1 h.\nExperiment 4: effect of MG132 treatment\nWe investigated whether the treatment of rat oocytes with a\nproteasome inhibitor, by which the p34cdc2 kinase activity\nwould be maintained, could support the PCC incidence of\nthe injected donor cell nuclei. MG132 (Sigma-Aldrich), an\ninhibitor which can block cyclin B1 degradation, was used\nat the ﬁnal concentration of 5 mM, as reported by Josefsberg\net al. (2000). Non-enucleated oocytes were exposed to\nMG132,\nthe process of oocyte recovery to\nin vitro culture or nuclear injection. Oocyte sampling for\nin vitro p34cdc2 kinase assay was performed at 15 min\n(immediately after recovery), 45, 75 and 120 min. Nuclear\ninjection into the MG132-treated oocytes and PCC evalu-\nation were performed as in Experiment 3.\nthroughout\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nIn the ﬁrst group, oocytes exposed to MG132 from oocyte\nrecovery to just before enucleation were used as recipi-\nents. From the result of Experiment 4, MG132 treatment is\nexpected to support PCC even in enucleated oocytes,\nbecause the p34cdc2 kinase activity was maintained at a\nhigh level during the treatment. The procedure of enuclea-\ntion was as described above. The cytoplasts were then\ninjected with cumulus cell nuclei. One hour later, a high\nincidence of PCC was expected.\nIn the second group,\noocytes with the extrusion of a second polar body were\nselected from 2 h cultures of non-treated oocytes, and\nthen enucleated and nuclear injected as well. The reduced\nincidence of PCC under a lowered MPF was expected. For\nboth groups, the oocytes 1 h after nuclear injection were\nactivated by two direct current pulses (100 V/mm, 99 ms)\nand 4 h treatment with 2 mM 6-dimethylaminopurine\n(6-DMAP)\nin mR1ECM medium. The\nreconstructed oocytes were cultured in 100 ml microdrops\nof mR1ECM at 37 8C in 5% CO2 in air, and number of\noocytes forming one or two pseudo-pronuclei and cleav-\ning to the two-cell stage was recorded at 6 and 24 h of\nculture respectively.\n(Sigma-Aldrich)\nStatistical analysis\nEach experiment had at least three replicates. Data on\np34cdc2 kinase activity were compared by ANOVA and\nFisher’s protected least signiﬁcant difference test using the\nStatView program (Abacus Concepts, Inc., Berkeley, CA,\nUSA). Data on PCC incidence and NT embryo develop-\nment were compared by Fisher’s exact probability test. A\nvalue of P , 0.05 was chosen as an indication of statisti-\ncal signiﬁcance.\nResults\nExperiment 1: effect of in vitro aging\nThe time-dependent change in p34cdc2 kinase activity in\nrat oocytes is shown in Fig. 1. The p34cdc2 kinase activity\nof\nrat oocytes decreased signiﬁcantly at 45 min (after\n30 min cultivation) and more signiﬁcantly at 75 min (after\n60 min cultivation). The decreased activity was maintained\nuntil 120 min (after 105 min cultivation). Probably due to\nthe decrease of p34cdc2 kinase activity, the release from\nmeiotic arrest of ovulated oocytes occurred in a time-\ndependent manner, as shown in Fig. 2. Within 75 min\nafter animal killing, more than half of the oocytes pro-\ngressed beyond the MII stage. At 135 min, most of the\noocytes were at either the AII or the TII stage. The pro-\nportion of in vitro-aged rat oocytes promoting PCC of the\ninjected cumulus cell nuclei is shown in Table 1. When\nnuclear injection was completed within 45 min from ani-\nmal killing, a positive sign of PCC was observed in 32.5%\nof the NT oocytes. On the other hand, PCC occurred in\nonly 2.6% of the oocytes into which cumulus cell nuclei\nwere injected between 46 and 120 min.\nExperiment 2: effect of oocyte enucleation\nThe activities of p34cdc2 kinase in non-enucleated and\nenucleated oocytes within 45 min were compared in rats\nand mice, as shown in Fig. 3. In both species, there was\na slight decrease in the p34cdc2 kinase activity after enu-\ncleation in both rat and mouse oocytes. A considerable\nRole of MPF in PCC induction of rat oocytes 175\ntreatment (enucleation or cytoplasm removal) is shown in\nTable 2. In rats, despite little reduction of p34cdc2 kinase\nactivity, few of the oocytes that either had a small portion\nof cytoplasm-removed (17.1%) or the non-treated controls\n(28.2%) could support PCC at 105 min after animal\nin the\nkilling. However, PCC was not observed at all\nfound in the p34cdc2 kinase activity\ndifference was\nbetween rat and mouse oocytes, because the mouse\noocytes carried a p34cdc2 kinase activity 2.5 times that of\nrat oocytes either immediately or up to 60 min after enu-\ncleation or cytoplasm removal and compared with non-\ntreated, age-matched controls, as shown in Fig. 4. The\nactivities of p34cdc2 kinase were measured in rat oocytes\nimmediately after or 60 min after one of three treatments\n(enucleated, cytoplasm-removed and non-treated), as\nshown in Fig. 4. Surprisingly, a higher p34cdc2 kinase\nactivity was detected in the oocytes immediately after\nenucleation, although the p34cdc2 kinase activity of\noocytes immediately after cytoplasm removal was not\nfrom that of non-treated oocytes. The p34cdc2\ndifferent\nin enucleated oocytes dramatically\nactivity\nkinase\ndecreased within 30 min after enucleation. At 45 min\nafter enucleation, the activity was further decreased and\nthereafter maintained until 60 min. However, the activity\nin non-treated or cytoplasm-removed oocytes did not sig-\nniﬁcantly differ by 60 min. The proportion of rat and\nmouse oocytes promoting PCC in relation to mechanical\nenucleated oocytes where p34cdc2 kinase activity was sig-\nniﬁcantly decreased at 60 min after completion of enu-\ncleation. In contrast, PCC of mouse cumulus cell nuclei\noccurred in most of the mouse oocytes before enucleation\n(96.7%) and even after enucleation (93.1%).\nExperiment 3: effect of roscovitine treatment\nThe time-dependent decrease of p34cdc2 kinase activity in\nroscovitine-treated rat oocytes is shown in Fig. 5. The\nactivity\nlinearly\nin roscovitine-treated oocytes was\ndecreased until 75 min and maintained until 120 min.\nAt 45, 75 and 120 min, each activity in the roscovitine-\ntreated oocytes was signiﬁcantly lower than that of non-\ntreated control oocytes. The result of PCC incidence in the\noocytes treated with or without roscovitine for 45 min is\nshown in Table 3. None of the roscovitine-treated oocytes\npromoted PCC, although 41.7% of non-treated oocytes\npromoted it.\nExperiment 4: effect of MG132 treatment\nThe activities of p34cdc2 kinase in MG132-treated rat\noocytes are shown in Fig. 6. The level of p34cdc2 kinase\nactivity still decreased in the MG132-treated oocytes\nafter 30 min of culture, but were signiﬁcantly greater\nthan in control oocytes. The p34cdc2 kinase activity was\nmaintained during the further incubation up to 105 min.\nThe kinase activities of MG132-treated oocytes at 60\nand 105 min cultivation were signiﬁcantly greater than\nthose of control oocytes. As shown in Table 4, the pro-\nportion of MG132-treated oocytes promoting PCC\n(66.0%) was higher than that of the non-treated oocytes\n(46.5%) when the nuclear\ninjection was completed\nwithin 45 min.\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nthe reconstructed embryos\nAfter oocyte activation with the electrical stimulus plus 6-\nDMAP,\nfrom both groups\n(MG132-treated MII oocytes and non-treated TII oocytes\nat the enucleation) formed one or two pseudo-pronuclei\n(91.7 and 87.5% in total respectively). After activation, a\ngreater proportion of oocytes enucleated at the MII stage\ntended to form two pseudo-pronuclei compared with\nthose enucleated at the TII stage. A signiﬁcantly higher\nproportion of the NT embryos in the MG132-treated group\nvs the non-treated group (59.1 vs 7.1%) developed to the\ntwo-cell stage.",
         "Discussion\nRat oocytes have poor potential\nfor promoting PCC of\ninjected somatic cell nuclei, and the rate was markedly\ndecreased in aged or enucleated oocytes (Hirabayashi\nIn the present study, rat oocytes into\net al. 2003a,b).\nwhich cumulus cell nuclei were injected within 45 min\nRole of MPF in PCC induction of rat oocytes 177\nthe non-treated oocytes during the same period, may\ninﬂuence the outcome of PCC. The high concentration of\nroscovitine affects the other cyclin-dependent protein\nkinases such as cdk2 (De Azevado et al. 1997). The\nunknown pathway and/or chemical toxicity of roscovitine\ncannot be ruled out\nfor complete suppression of PCC\ninduction. Moreover, the level of activity in rat oocytes\nwas less than half compared with that in mouse oocytes,\nwhich had a high potential for promoting PCC. Judging\nfrom these results, we estimated that the high activity of\np34cdc2 kinase is associated with promoting PCC of\ninjected somatic cell nuclei, and the low level of p34cdc2\nkinase activity is one of the reasons for the poor potential\nto promote PCC in rat oocytes compared with that of\nmouse oocytes. The reason why the potential\nlevel of\np34cdc2 kinase activity is different between mouse and rat\noocytes remains unclear. Our previous study indicated, at\nleast, that the PCC-inducing potential is determined by\nthe origin of the recipient cytoplasm, not by the origin of\nin the interspecies NT experiment\nthe donor nucleus,\nbetween mouse and rat\nIn\n(Hirabayashi et al. 2003a).\nsomatic cell division, dissolution of the nuclear envelope\noccurs during G2/M transition in mitosis (Nurse 1990) and\nis induced by phosphorylation of lamin, one of the major\nnuclear membrane proteins (Peter et al. 1990). In oocytes,\np34cdc2 kinase had the potential to phosphorylate lamins,\ninducing meiotic breakdown of the nuclear envelope and\nthe meiotic resumption (Dessev & Goldman 1988). There-\nfore, it is possible that p34cdc2 kinase plays an important\nrole in the destruction of the nuclear membrane of the\ninjected somatic cell through phosphorylation of lamin.\nOne possible pathway known for inactivation of p34cdc2\nkinase is the calmodulin-dependent pathway via Ca2þ.\nWe have reported that the Ca2þ-dependent pathway trig-\ngers the degradation of cyclin B and results in inactivation\nof p34cdc2 kinase in porcine oocytes (Ito et al. 2003,\n2004a). In Xenopus oocytes (Lorca et al. 1994, Morin et al.\n1994), activation of calmodulin-dependent protein kinase\nII induces the degradation of cyclin B via the ubiquitin\nproteasome pathway. Based on these literature sources\nand preliminary data, we have investigated the effect of a\nproteasome inhibitor MG132 on p34cdc2 kinase activity\nand PCC incidence. The PCC incidence in rat oocytes was\nsigniﬁcantly improved by MG132 treatment (Table 4). This\nresult matched well with the report by Josefsberg et al.\n(2000), who demonstrated that MG132-treated rat oocytes\ncontained a high amount of cyclin B1.\nIn the present\nstudy, MG132 also provided a suppressive effect on inacti-\nvation of p34cdc2 kinase (Fig. 6). Use of ALLN, a neutral\ncysteine protease inhibitor, was also slightly effective for\nimproving the PCC incidence in rat oocytes (Hirabayashi\net al. 2003b). Thus, the Ca2þ-dependent degradation path-\nway, which may induce a decrease in the cyclin B1 level,\nis activated in aged rat oocytes, which results in a low\nlevel of p34cdc2 kinase activity, and a low potential\nto promote PCC. Our unpublished data (M Hirabayashi\nand J Ito, unpublished data) showing that the decreased\nIn mouse oocytes, PCC was initiated at\npromoted PCC (frequency 28–47% in Tables 1 –4), but\nthe incidence of PCC decreased with oocyte aging with a\nsigniﬁcant decrease in p34cdc2 kinase activity (Table 1,\nFig. 1).\nleast\n30 min after nuclear injection and lasted until 2 h (Chen\net al. 2004), therefore the timing of PCC observation in\nthe present study (60 min after nuclear injection) might be\noptimal. When oocytes are treated with roscovitine,\nwhich decreases the level of p34cdc2 kinase activity, PCC\nformation was never observed even though nuclear injec-\ntion is completed within 45 min. Roscovitine is widely\nused as cyclin-dependent kinase inhibitor\nto suppress\np34cdc2 kinase in rodent oocytes (Phillips et al. 2002).\nTherefore, the rapid decrease of p34cdc2 kinase activity in\nthe roscovitine-treated oocytes found between 45 and\n75 min (Fig. 5), when compared with the slow decrease in\na Oocytes were treated with 5 mM MG132 until just before enucleation after oocyte recovery.\nb Oocytes were cultured for about 2 h and only oocytes with a completely extruded polar body (the TII stage) were used.\nc Percentage of reconstructed embryos with 1 or 2 pronuclei.\nd – e Different superscripts denote signiﬁcant difference (P , 0.05).\nconcentration of extracellular free calcium in the handling\nmedium partially inhibits spontaneous activation of rat\noocytes, support this hypothesis.\nThe other pathway for inactivation by phosphorylation\nof 14Thr and 15Tyr of p34cdc2 kinase has been reported in\naged porcine oocytes (Kikuchi et al. 2000). In the same\nstudy, it has been reported that caffeine suppressed the\nfrom MPF to pre-MPF (14Thr/15Tyr-phosphorylated\nshift\np34cdc2 kinase þ cyclin B1) in the aged porcine oocytes.\nOkadaic acid is also known as a protein phosphatase\ninhibitor which prevents phosphorylation of 14Thr and\n15Tyr of p34cdc2 kinase (Rime et al. 1995). In our prelimi-\nnary study, PCC incidence of injected cumulus cell nuclei\nwas slightly improved in rat oocytes treated with okadaic\nacid (M Hirabayashi and J Ito, unpublished data). There-\nfore, the phosphorylation of p34cdc2 kinase as well as the\ncyclin degradation may be involved in spontaneous acti-\nvation of rat oocytes during in vitro culture.\nThe level of p34cdc2 kinase activity in non-treated rat\noocytes was similar to that of cytoplasm-removed or enu-\ncleated oocytes in the case where the oocytes were\nsampled immediately after the treatment\n(Fig. 4). The\nmechanical puncture of the oocyte membrane and the\naspiration of the cytoplasm or karyoplast did not inﬂuence\nthe p34cdc2 kinase activity immediately after the enuclea-\ntion procedure beyond the level expected from the rela-\ntive volume of aspirated cytoplasm (, 10–20%). These\nresults strongly suggest that active p34cdc2 kinase is not\naccumulated near the metaphase spindles in rat oocytes.\nThere was signiﬁcant difference in the p34cdc2 kinase\nactivity between 45 and 75 min in both non-treated and\nenucleated groups (Fig. 4). A dramatic decrease in the\nactivity of p34cdc2 kinase was observed in the rat oocytes\nenucleated and cultured for 30 min compared with the\nnon-treated group (Fig. 4), suggesting that\nthe oocyte\nmetaphases per se are involved in the maintenance of\np34cdc2 kinase activity. We have previously reported in\nporcine oocytes that activation of p34cdc2 kinase was\ndependent on the synthesis of cyclin B and that the syn-\nthesis was induced by the activation of mitogen-activated\nprotein (MAP) kinase (Ito et al. 2004b). The MAP kinase\nsuppressed Myt1 activity that phosphorylated p34cdc2\nkinase at 14Thr and 15Tyr (Mueller et al. 1995, Palmer et al.\n1998). Because activated MAP kinase is localized on the\nmeiotic spindle in porcine oocytes (Lee et al. 2000), it is\npossible that enucleation results in the loss of MAP kinase\nfrom the recipient oocytes.\ninjection into the MG132-treated, MII-enu-\ncleated rat oocytes resulted in a higher development to\nthe two-cell stage when compared with non-treated, TII-\nenucleated rat oocytes after 24 h cultivation (Table 5).\nThe favorable circumstance of recipient cytoplasm with\na high MPF activity must induce PCC and promote sub-\nsequent development of NT embryos. Wakayama &\nYanagimachi\nthe injected\n(2001) reported in mice that\ncumulus cell nuclei undergoing PCC transformation to\nthe two pseudo-pronuclei and such NT embryos are able\nto develop to full-term. In contrast, a different situation\nhas been reported in the cloning of large domestic ani-\nmals. The somatic cell NT in the livestock species is suc-\ncessfully reproducible when enucleated and activated\noocytes with a low MPF activity are used as recipient\ncytoplasm (Baguisi et al. 1999, Kurosaka et al. 2002,\nShin et al. 2002). Such a difference between rodents\nand large domestic animals remains unclear, but our\nthat\npresent data suggest\ninduction of PCC in recon-\nstructed embryos\nrelated to the high activity of\nis\np34cdc2 kinase, which seems to play a key role in the\nsuccessful rat cloning.\nthe decreased level of p34cdc2 kinase\nactivity in aged or enucleated rat oocytes is responsible\nfor their inability to promote PCC of microinjected donor\ncell nuclei. Inhibition of p34cdc2 kinase inactivation by\nchemicals such as MG132 is in part effective for rat\noocytes to promote PCC and cleavage. These data may\nexpand our existing knowledge on characteristics of rat\noocytes and contribute to the development of a rat NT\nprotocol using G0/G1-stage donor cells.",
         "Speciﬁc-pathogen-free Crj:Wistar female rats (3 –5 weeks\nold) or Crj:BDF1 female mice (7 weeks old) were pur-\nchased from Charles River Japan, Inc. (Kanagawa, Japan).\nThey were housed under controlled lighting (lights on at\ntemperature (23 ^ 2 8C) and humidity\n0600 –1800 h),\n(55 ^ 5%), with free access\n(MF;\nOriental Yeast Co., Tokyo, Japan) and water. Rats were\nsuperovulated by i.p. injections of 30 IU equine chorio-\nnic gonadotropin (eCG) (Nippon Zenyaku, Co., Fukush-\nima, Japan) and 30 IU hCG (Sankyo Yell Yakuhin, Co.,\nTokyo, Japan) at 48 h intervals (Hirabayashi et al. 2001).\nMice were superovulated as well with 5 IU eCG and 5\nIU hCG.\nto laboratory diet\nFourteen hours after the hCG injection, cumulus –oocyte\ncomplexes (COCs) were collected from the oviductal\nampullae of donor rats or mice with modiﬁed Krebs–\nRinger bicarbonate solution (mKRB) in Experiments 1 and\n3 (Toyoda & Chang 1974) and modiﬁed rat one-cell\nembryo culture medium (mR1ECM) in Experiments 2, 4\nand 5 (Oh et al. 1998) supplemented with 0.1% hyaluroni-\ndase (Sigma-Aldrich Corp., St Louis, MO, USA) respect-\nively. Two to ﬁve minutes later, the denuded oocytes were\nwashed three times with fresh mKRB or R1ECM and kept in\nthe same medium at 37 8C until being subjected to the\ntreatments. The time of killing of the donor animals was\ndeﬁned as 0 min, and it usually took 10 –15 min from the\nanimal killing to the preparation of denuded oocytes.\nMeasurement of p34cdc2 kinase activity\nThe p34cdc2 kinase assay was performed according to the\nmethod as described by Ito et al. (2001) with some modiﬁ-\ncations. In brief, the oocytes were washed several times\nwith PBS and placed into cell lysis buffer composed of\n20 mM Tris, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1%\nTriton X-100, 2.5 mM sodium pyrophosphate, 1 mM b-gly-\ncerophosphate, 1 mM Na3VO4, 1 mg/ml\nleupeptin (all\nchemicals from Cell Signaling Technology, Beverly, MA,\nUSA) and 1 mM phenylmethylsulfonyl ﬂuoride (Sigma-\nAldrich). The oocyte lysates were stored at 2 80 8C until\nthe ELISA assay.\n(MBL, Nagoya,\nThe lysate (ten oocytes/5 ml cell lysis buffer) was mixed\nwith 45 ml of kinase assay buffer-A composed of 25 mM\nHepes buffer\nJapan), 10 mM\nMgCl2 (MBL), 10% (v/v) mouse vimentin peptide solution\n(SLYSSPGGAYC) (MBL) and 0.1 mM ATP (Sigma-Aldrich),\nand the mixture was incubated for 30 min at 30 8C. The\nreaction was terminated by the addition of 200 ml PBS\ncontaining 50 mM EGTA (MBL). The phosphorylation of\nmouse vimentin peptides was detected using an ELISA\nanalysis (MESACUP cdc2 kinase assay kit\n(code no.\n5234); MBL). Data were expressed in terms of the strength\nof p34cdc2 kinase activity in denuded rat oocytes 15 min\n(Experiments 1, 3 and 4) or 45 min (Experiment 2) after\nanimal killing.\nAssessment of nuclear conﬁguration\nRat oocytes immediately after recovery and 30, 60, 90\nand 120 min after culture in mKRB were stained with\n5 mg/ml Hoechst 33342 (Sigma-Aldrich). Their nuclear\nconﬁguration was then assessed under UV light at 330–\n380 nm, according to the classiﬁcations reported pre-\nviously (Kato et al. 2001): MII, characterized by a meta-\nphase spindle with the ﬁrst polar body; AII, characterized\nby elongated spindle without the second polar body; and\nTII, characterized by elongated spindle with extrusion of\nthe second polar body.\nNuclear injection and PCC evaluation\nCumulus cells dispersed from COCs by the hyaluronidase\ntreatment were placed in mKRB (Experiments 1 and 3) or\nR1ECM (Experiments 2, 4 and 5) containing 22 mM Hepes\nand 5 mM NaHCO3 (abbreviated hereafter as Hepes-\nmKRB and Hepes-R1ECM respectively). An aliquot (2 ml)\nof the cumulus cell suspension was transferred to 10 ml of\nHepes-mKRB or Hepes-R1ECM supplemented with 12%\nw/v 360 kDa polyvinylpyrrolidone (ICN Pharmaceuticals,\nInc., Costa Mesa, CA, USA). Nuclei were removed from\ncumulus cells and gently aspirated in and out of the injec-\ntion pipette (7 –10 mm) until\ntheir nuclei were largely\ndevoid of visible cytoplasmic materials. Each nucleus was\ninjected into a separate enucleated oocyte within 5 min of\nits isolation, as described by Kimura & Yanagimachi\n(1995). One hour after the nuclear injection, the oocytes\nwere stained with 5 mg/ml Hoechst 33342 and evaluated\nfor incidence of PCC under UV light at 330–380 nm, as\ndescribed previously (Hirabayashi et al. 2003a,b). The\ninjected somatic cell nucleus was distinguished from the\noocyte nucleus by the position of the opening in the zona\npellucida at the nuclear microinjection site.\nRole of MPF in PCC induction of rat oocytes 173\nExperimental design\nExperiment 1: effect of in vitro aging\nThe effect of in vitro aging on the p34cdc2 kinase activity\nand the PCC incidence was investigated using rat oocytes.\nTen oocytes for each test were sampled for\nin vitro\np34cdc2 kinase assay immediately after recovery (15 min\ngroup), and after culture for 30 min (45 min group), 60 min\n(75 min group) or 105 min (120 min group) in mKRB at\n37 8C in 5% CO2 in air. Nuclear conﬁguration of\nthe\noocytes was also examined immediately after recovery\nand every subsequent 30 min up to 120 min. The other\noocytes were placed in Hepes-mKRB, and the cumulus\ncell nuclei were then injected into the oocytes up to\n120 min. The injected oocytes were divided into two\ngroups based on the time from the animal killing to NT\ncompletion (15 –45 min group and 46 – 120 min group).\nThe incidence of PCC was evaluated 1 h after the last\ninjection among each group.\nExperiment 2: effect of enucleation\nThe effect of oocyte enucleation on the p34cdc2 kinase\nactivity and the PCC incidence was investigated using\nboth rat and mouse oocytes. For the in vitro p34cdc2\nkinase assay, oocytes derived from the same donor rats\nand mice were allocated to the non-enucleation group\nand the enucleation group, and the non-enucleated and\nenucleated oocytes were sampled 45 min after the animal\nkilling. Enucleation was performed by aspiration of the\nmetaphase-plate with a part of the volume (,10–20%) of\ncytoplasm in Hepes-R1ECM containing\nsurrounding\n5 mg/ml cytochalasin B (Sigma-Aldrich). In addition to the\nnon-enucleation and enucleation groups, denuded rat\noocytes were allocated to a cytoplasm-removal group\n(a part of the cytoplasm was mechanically aspirated as a\nsham control.). All the treatments were completed within\n45 min. The oocyte samples were harvested at 45, 75, 90\nand 105 min (0, 30, 45 and 60 min after enucleation\nrespectively). For PCC evaluation, nuclear injection into\nnon-enucleated and enucleated mouse and rat oocytes,\nand into cytoplasm-removed rat oocytes, was completed\nwithin 45 min after animal killing.\nExperiment 3: effect of roscovitine treatment\nWe investigated whether the decreased level of p34cdc2\nkinase activity in non-enucleated rat oocytes negatively\naffected the PCC incidence of injected cell nuclei. Ros-\ncovitine (Sigma-Aldrich) dissolved at 10 mM in DMSO\nand stored at 2 80 8C, a speciﬁc inhibitor of p34cdc2\nkinase, was added to all media throughout the process of\noocyte recovery and the in vitro culture at the ﬁnal con-\ncentration of 150 mM, as reported by Lazar et al. (2002).\nOocytes were sampled for in vitro p34cdc2 kinase assay at\n15 (immediately after recovery), 45, 75 and 120 min as in\nExperiment 1. Because the 45 min treatment of rat oocytes\nwith 150 mM roscovitine decreased their p34cdc2 kinase\nactivity to an equivalent level of oocytes 120 min after in\nvitro aging or 60 min after enucleation, nuclear injection\ninto the roscovitine-treated oocytes was performed within\n45 min, and the incidence of PCC was then evaluated\nafter a further 1 h.\nExperiment 4: effect of MG132 treatment\nWe investigated whether the treatment of rat oocytes with a\nproteasome inhibitor, by which the p34cdc2 kinase activity\nwould be maintained, could support the PCC incidence of\nthe injected donor cell nuclei. MG132 (Sigma-Aldrich), an\ninhibitor which can block cyclin B1 degradation, was used\nat the ﬁnal concentration of 5 mM, as reported by Josefsberg\net al. (2000). Non-enucleated oocytes were exposed to\nMG132,\nthe process of oocyte recovery to\nin vitro culture or nuclear injection. Oocyte sampling for\nin vitro p34cdc2 kinase assay was performed at 15 min\n(immediately after recovery), 45, 75 and 120 min. Nuclear\ninjection into the MG132-treated oocytes and PCC evalu-\nation were performed as in Experiment 3.\nthroughout\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nIn the ﬁrst group, oocytes exposed to MG132 from oocyte\nrecovery to just before enucleation were used as recipi-\nents. From the result of Experiment 4, MG132 treatment is\nexpected to support PCC even in enucleated oocytes,\nbecause the p34cdc2 kinase activity was maintained at a\nhigh level during the treatment. The procedure of enuclea-\ntion was as described above. The cytoplasts were then\ninjected with cumulus cell nuclei. One hour later, a high\nincidence of PCC was expected.\nIn the second group,\noocytes with the extrusion of a second polar body were\nselected from 2 h cultures of non-treated oocytes, and\nthen enucleated and nuclear injected as well. The reduced\nincidence of PCC under a lowered MPF was expected. For\nboth groups, the oocytes 1 h after nuclear injection were\nactivated by two direct current pulses (100 V/mm, 99 ms)\nand 4 h treatment with 2 mM 6-dimethylaminopurine\n(6-DMAP)\nin mR1ECM medium. The\nreconstructed oocytes were cultured in 100 ml microdrops\nof mR1ECM at 37 8C in 5% CO2 in air, and number of\noocytes forming one or two pseudo-pronuclei and cleav-\ning to the two-cell stage was recorded at 6 and 24 h of\nculture respectively.\n(Sigma-Aldrich)\nStatistical analysis\nEach experiment had at least three replicates. Data on\np34cdc2 kinase activity were compared by ANOVA and\nFisher’s protected least signiﬁcant difference test using the\nStatView program (Abacus Concepts, Inc., Berkeley, CA,\nUSA). Data on PCC incidence and NT embryo develop-\nment were compared by Fisher’s exact probability test. A\nvalue of P , 0.05 was chosen as an indication of statisti-\ncal signiﬁcance.\nResults\nExperiment 1: effect of in vitro aging",
         "1National Institute for Physiological Sciences, Okazaki 444-8787, Japan, 2The Graduate University of\nAdvanced Studies, Okazaki 444-8787, Japan, 3CREST of Japan Science and Technology Agency, Kawaguchi\n332-0012, Japan, 4Faculty of Textile Science and Technology, Shinshu University, Ueda 386-8567, Japan and\n5Graduate School of Biosphere Science, Hiroshima University, Higashi-Hiroshima 739-8528, Japan\nAbstract\nThe present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated\nrat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes\nwere placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-\ndependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15– 45 min\nrather than 46– 120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity tran-\nsiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-\nphase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed\noocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat\noocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent\nlevel of aged (120 min) or enucleated (1 60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2\nkinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were\ntreated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated\noocytes. A signiﬁcantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control\noocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus\ncell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage.\nThese results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for\ntheir inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemi-\ncals such as MG132 is in part effective for rat oocytes to promote PCC and further development.\nReproduction (2005) 129 171–180\nIntroduction\nSuccessful animal cloning by somatic cell nuclear transfer\n(NT) has been reported in large domestic species (Wilmut\net al. 1997, Kato et al. 1998, Baguisi et al. 1999, Onishi\net al. 2000, Galli et al. 2003). As for small rodents,\nWakayama et al. (1998) were the ﬁrst to produce cloned\nmice using G0/G1-phase cumulus cells, and very recently\nZhou et al. (2003) reported the production of fertile cloned\nrats using M-phase fetal ﬁbroblast cells. However, the suc-\ncess rates of producing cloned rodents by NT are still low.\nresulting in premature chromosome condensation (PCC),\nwith subsequent multiple pseudo-pronuclei formation fol-\nlowing activation (Wakayama et al. 1998, 1999, Ogura\net al. 2000). It is considered in rodents, at least, that PCC\nis a critical aspect contributing to successful cloning\n(Wakayama et al. 1999, Ogura et al. 2000). However,\nPCC is not essential for somatic cell cloning in all species,\nsince telophase-II\n(TII) oocytes have promoted embryo\ndevelopment in goats (Baguisi et al. 1999) and cattle (Kur-\nosaka et al. 2002).\nThere is a major difference between mouse and rat, in\nthe behavior of somatic nuclei upon exposure to oocyte\ncytoplasm. We have reported that the PCC of both mouse\nand rat cumulus cell nuclei occurred in enucleated mouse\noocytes but not at all\nin enucleated rat oocytes\n(Hirabayashi et al. 2003a). Even if cumulus nuclei were\ninjected into the rat oocytes without enucleation, the rate\nof PCC formation was less than half (Hirabayashi et al.\n2003b).\nWe also reported that\nthe ability of rat oocytes to\npromote PCC of the injected nuclei was dependent on\nthe characteristics of the oocytes, such as age (4- to 5-\nweek-old, superior to .10-week-old rats) or strain of the\ndonor rats (Wistar and LEW, superior to the Donryu and\nF344 strains) and the timing of oocyte recovery (14 h post-\nhuman chorionic gonadotropin (hCG), superior to 17 h\npost hCG) (Hirabayashi et al. 2003b). In the same study,\nthe time management of NT completion (less than 45 min\nafter donor animal killing) and addition to the oocyte-\nhandling media\nof N-acetylleucylleucylnorleucinal\n(ALLN), a neutral cysteine protease inhibitor, were also\nfound important for intact rat oocytes to promote PCC.\nThe successful production of cloned rats (Zhou et al.\n2003) may be attributed to injecting the metaphase-\narrested fetal ﬁbroblasts into oocytes treated with MG132,\na proteasome inhibitor, followed by quick removal of the\nrecipient metaphase-plate. Therefore, it is a possibility that\nrecipient oocytes arrested at the metaphase-II (MII) stage\nby the inhibition of protein degradation have essential\nroles in nuclear reprogramming including PCC formation.\nOvulated rat oocytes are known to activate spon-\ntaneously but abortively during in vitro culture (Zernicka-\nGoetz 1991). We also conﬁrmed that more than half of\nthe oocytes from Wistar rats progressed to anaphase-II\n(AII)/TII stages at 70 min after killing of\nthe donor rats\n(Kato et al. 2001). In vertebrate oocytes, it has been con-\nsidered that high activity of maturation promoting factor\n(MPF) affects the chromosome condensation and the\nmaintenance of the meiotic spindle, both of which are\ninvolved in the meiotic arrest at\nthe MII stage (Jones\n2004). MPF is a heterodimer composed of p34cdc2 kinase\nand cyclin B1, controlling the cell cycle. When the\nactivity was inactivated by the degradation of cyclin B\n(Nurse 1990), oocytes were released from the arrest at the\nMII stage. The p34cdc2 kinase in oocytes was maintained\nat a high level until ovulation in many species (Kubiak\net al. 1993, Naito et al. 1995, Abrieu et al. 2001, Ito &\nShimada 2005), suggesting that p34cdc2 kinase activity in\nrat ovulated oocytes was rapidly decreased in vitro. More-\nover, destruction of\nthe nuclear envelope in mitotic\nsomatic cells occurred during the G2/M-phase transition\nby p34cdc2 kinase-related phosphorylation of lamin, which\nis one of\nthe nuclear membrane proteins (Peter et al.\n1990). It has been reported that ALLN, which was used in\nour previous study (Hirabayashi et al. 2003b), has an\neffect on the inhibition of cyclin B destruction (Sherwood\net al. 1993). Therefore, it is possible that the low rate of\nPCC of injected somatic cell nuclei resulted from rapidly\ndecreasing p34cdc2 kinase activity in rat ovulated oocytes,\nwhile direct evidence that p34cdc2 kinase activity is\ninvolved in the PCC incidence has not yet been shown.\nClarifying the relationship between p34cdc2 kinase activity\nand PCC incidence would contribute to understanding the\nreprogramming mechanism in mammalian\nnuclear\noocytes.\nIn the present study, two experiments were conducted\nto examine the effects of in vitro aging and enucleation of\nrat oocytes on the kinetics of p34cdc2 kinase and the inci-\ndence of PCC of microinjected cumulus cell nuclei, with\nsome comparisons with mouse oocytes. In addition, two\nmore experiments were conducted to explain a possible\nrole of p34cdc2 kinase for promoting PCC, by treatment of\nthe rat oocytes with roscovitine, an inhibitor of p34cdc2\nkinase (Lazar et al. 2002), and MG132, a proteasome\ninhibitor which has an effect on the inhibition of cyclin\nB1 degradation (Josefsberg et al. 2000).\nMaterials and Methods\nOocyte preparation\nSpeciﬁc-pathogen-free Crj:Wistar female rats (3 –5 weeks\nold) or Crj:BDF1 female mice (7 weeks old) were pur-\nchased from Charles River Japan, Inc. (Kanagawa, Japan).\nThey were housed under controlled lighting (lights on at\ntemperature (23 ^ 2 8C) and humidity\n0600 –1800 h),\n(55 ^ 5%), with free access\n(MF;\nOriental Yeast Co., Tokyo, Japan) and water. Rats were\nsuperovulated by i.p. injections of 30 IU equine chorio-\nnic gonadotropin (eCG) (Nippon Zenyaku, Co., Fukush-\nima, Japan) and 30 IU hCG (Sankyo Yell Yakuhin, Co.,\nTokyo, Japan) at 48 h intervals (Hirabayashi et al. 2001).\nMice were superovulated as well with 5 IU eCG and 5\nIU hCG.\nto laboratory diet\nFourteen hours after the hCG injection, cumulus –oocyte\ncomplexes (COCs) were collected from the oviductal\nampullae of donor rats or mice with modiﬁed Krebs–\nRinger bicarbonate solution (mKRB) in Experiments 1 and\n3 (Toyoda & Chang 1974) and modiﬁed rat one-cell\nembryo culture medium (mR1ECM) in Experiments 2, 4\nand 5 (Oh et al. 1998) supplemented with 0.1% hyaluroni-\ndase (Sigma-Aldrich Corp., St Louis, MO, USA) respect-\nively. Two to ﬁve minutes later, the denuded oocytes were\nwashed three times with fresh mKRB or R1ECM and kept in\nthe same medium at 37 8C until being subjected to the\ntreatments. The time of killing of the donor animals was\ndeﬁned as 0 min, and it usually took 10 –15 min from the\nanimal killing to the preparation of denuded oocytes.\nMeasurement of p34cdc2 kinase activity\nThe p34cdc2 kinase assay was performed according to the\nmethod as described by Ito et al. (2001) with some modiﬁ-\ncations. In brief, the oocytes were washed several times\nwith PBS and placed into cell lysis buffer composed of\n20 mM Tris, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1%\nTriton X-100, 2.5 mM sodium pyrophosphate, 1 mM b-gly-\ncerophosphate, 1 mM Na3VO4, 1 mg/ml\nleupeptin (all\nchemicals from Cell Signaling Technology, Beverly, MA,\nUSA) and 1 mM phenylmethylsulfonyl ﬂuoride (Sigma-\nAldrich). The oocyte lysates were stored at 2 80 8C until\nthe ELISA assay.\n(MBL, Nagoya,\nThe lysate (ten oocytes/5 ml cell lysis buffer) was mixed\nwith 45 ml of kinase assay buffer-A composed of 25 mM\nHepes buffer\nJapan), 10 mM\nMgCl2 (MBL), 10% (v/v) mouse vimentin peptide solution\n(SLYSSPGGAYC) (MBL) and 0.1 mM ATP (Sigma-Aldrich),\nand the mixture was incubated for 30 min at 30 8C. The\nreaction was terminated by the addition of 200 ml PBS\ncontaining 50 mM EGTA (MBL). The phosphorylation of\nmouse vimentin peptides was detected using an ELISA\nanalysis (MESACUP cdc2 kinase assay kit\n(code no.\n5234); MBL). Data were expressed in terms of the strength\nof p34cdc2 kinase activity in denuded rat oocytes 15 min\n(Experiments 1, 3 and 4) or 45 min (Experiment 2) after\nanimal killing.\nAssessment of nuclear conﬁguration\nRat oocytes immediately after recovery and 30, 60, 90\nand 120 min after culture in mKRB were stained with\n5 mg/ml Hoechst 33342 (Sigma-Aldrich). Their nuclear\nconﬁguration was then assessed under UV light at 330–\n380 nm, according to the classiﬁcations reported pre-\nviously (Kato et al. 2001): MII, characterized by a meta-\nphase spindle with the ﬁrst polar body; AII, characterized\nby elongated spindle without the second polar body; and\nTII, characterized by elongated spindle with extrusion of\nthe second polar body.\nNuclear injection and PCC evaluation\nCumulus cells dispersed from COCs by the hyaluronidase\ntreatment were placed in mKRB (Experiments 1 and 3) or\nR1ECM (Experiments 2, 4 and 5) containing 22 mM Hepes\nand 5 mM NaHCO3 (abbreviated hereafter as Hepes-\nmKRB and Hepes-R1ECM respectively). An aliquot (2 ml)\nof the cumulus cell suspension was transferred to 10 ml of\nHepes-mKRB or Hepes-R1ECM supplemented with 12%\nw/v 360 kDa polyvinylpyrrolidone (ICN Pharmaceuticals,\nInc., Costa Mesa, CA, USA). Nuclei were removed from\ncumulus cells and gently aspirated in and out of the injec-\ntion pipette (7 –10 mm) until\ntheir nuclei were largely\ndevoid of visible cytoplasmic materials. Each nucleus was\ninjected into a separate enucleated oocyte within 5 min of\nits isolation, as described by Kimura & Yanagimachi\n(1995). One hour after the nuclear injection, the oocytes\nwere stained with 5 mg/ml Hoechst 33342 and evaluated\nfor incidence of PCC under UV light at 330–380 nm, as\ndescribed previously (Hirabayashi et al. 2003a,b). The\ninjected somatic cell nucleus was distinguished from the\noocyte nucleus by the position of the opening in the zona\npellucida at the nuclear microinjection site.\nRole of MPF in PCC induction of rat oocytes 173\nExperimental design\nExperiment 1: effect of in vitro aging\nThe effect of in vitro aging on the p34cdc2 kinase activity\nand the PCC incidence was investigated using rat oocytes.\nTen oocytes for each test were sampled for\nin vitro\np34cdc2 kinase assay immediately after recovery (15 min\ngroup), and after culture for 30 min (45 min group), 60 min\n(75 min group) or 105 min (120 min group) in mKRB at\n37 8C in 5% CO2 in air. Nuclear conﬁguration of\nthe\noocytes was also examined immediately after recovery\nand every subsequent 30 min up to 120 min. The other\noocytes were placed in Hepes-mKRB, and the cumulus\ncell nuclei were then injected into the oocytes up to\n120 min. The injected oocytes were divided into two\ngroups based on the time from the animal killing to NT\ncompletion (15 –45 min group and 46 – 120 min group).\nThe incidence of PCC was evaluated 1 h after the last\ninjection among each group.\nExperiment 2: effect of enucleation\nThe effect of oocyte enucleation on the p34cdc2 kinase\nactivity and the PCC incidence was investigated using\nboth rat and mouse oocytes. For the in vitro p34cdc2\nkinase assay, oocytes derived from the same donor rats\nand mice were allocated to the non-enucleation group\nand the enucleation group, and the non-enucleated and\nenucleated oocytes were sampled 45 min after the animal\nkilling. Enucleation was performed by aspiration of the\nmetaphase-plate with a part of the volume (,10–20%) of\ncytoplasm in Hepes-R1ECM containing\nsurrounding\n5 mg/ml cytochalasin B (Sigma-Aldrich). In addition to the\nnon-enucleation and enucleation groups, denuded rat\noocytes were allocated to a cytoplasm-removal group\n(a part of the cytoplasm was mechanically aspirated as a\nsham control.). All the treatments were completed within\n45 min. The oocyte samples were harvested at 45, 75, 90\nand 105 min (0, 30, 45 and 60 min after enucleation\nrespectively). For PCC evaluation, nuclear injection into\nnon-enucleated and enucleated mouse and rat oocytes,\nand into cytoplasm-removed rat oocytes, was completed\nwithin 45 min after animal killing.\nExperiment 3: effect of roscovitine treatment\nWe investigated whether the decreased level of p34cdc2\nkinase activity in non-enucleated rat oocytes negatively\naffected the PCC incidence of injected cell nuclei. Ros-\ncovitine (Sigma-Aldrich) dissolved at 10 mM in DMSO\nand stored at 2 80 8C, a speciﬁc inhibitor of p34cdc2\nkinase, was added to all media throughout the process of\noocyte recovery and the in vitro culture at the ﬁnal con-\ncentration of 150 mM, as reported by Lazar et al. (2002).\nOocytes were sampled for in vitro p34cdc2 kinase assay at\n15 (immediately after recovery), 45, 75 and 120 min as in\nExperiment 1. Because the 45 min treatment of rat oocytes\nwith 150 mM roscovitine decreased their p34cdc2 kinase\nactivity to an equivalent level of oocytes 120 min after in\nvitro aging or 60 min after enucleation, nuclear injection\ninto the roscovitine-treated oocytes was performed within\n45 min, and the incidence of PCC was then evaluated\nafter a further 1 h.\nExperiment 4: effect of MG132 treatment\nWe investigated whether the treatment of rat oocytes with a\nproteasome inhibitor, by which the p34cdc2 kinase activity\nwould be maintained, could support the PCC incidence of\nthe injected donor cell nuclei. MG132 (Sigma-Aldrich), an\ninhibitor which can block cyclin B1 degradation, was used\nat the ﬁnal concentration of 5 mM, as reported by Josefsberg\net al. (2000). Non-enucleated oocytes were exposed to\nMG132,\nthe process of oocyte recovery to\nin vitro culture or nuclear injection. Oocyte sampling for\nin vitro p34cdc2 kinase assay was performed at 15 min\n(immediately after recovery), 45, 75 and 120 min. Nuclear\ninjection into the MG132-treated oocytes and PCC evalu-\nation were performed as in Experiment 3.\nthroughout\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nIn the ﬁrst group, oocytes exposed to MG132 from oocyte\nrecovery to just before enucleation were used as recipi-\nents. From the result of Experiment 4, MG132 treatment is\nexpected to support PCC even in enucleated oocytes,\nbecause the p34cdc2 kinase activity was maintained at a\nhigh level during the treatment. The procedure of enuclea-\ntion was as described above. The cytoplasts were then\ninjected with cumulus cell nuclei. One hour later, a high\nincidence of PCC was expected.\nIn the second group,\noocytes with the extrusion of a second polar body were\nselected from 2 h cultures of non-treated oocytes, and\nthen enucleated and nuclear injected as well. The reduced\nincidence of PCC under a lowered MPF was expected. For\nboth groups, the oocytes 1 h after nuclear injection were\nactivated by two direct current pulses (100 V/mm, 99 ms)\nand 4 h treatment with 2 mM 6-dimethylaminopurine\n(6-DMAP)\nin mR1ECM medium. The\nreconstructed oocytes were cultured in 100 ml microdrops\nof mR1ECM at 37 8C in 5% CO2 in air, and number of\noocytes forming one or two pseudo-pronuclei and cleav-\ning to the two-cell stage was recorded at 6 and 24 h of\nculture respectively.\n(Sigma-Aldrich)\nStatistical analysis\nEach experiment had at least three replicates. Data on\np34cdc2 kinase activity were compared by ANOVA and\nFisher’s protected least signiﬁcant difference test using the\nStatView program (Abacus Concepts, Inc., Berkeley, CA,\nUSA). Data on PCC incidence and NT embryo develop-\nment were compared by Fisher’s exact probability test. A\nvalue of P , 0.05 was chosen as an indication of statisti-\ncal signiﬁcance.\nResults\nExperiment 1: effect of in vitro aging\nThe time-dependent change in p34cdc2 kinase activity in\nrat oocytes is shown in Fig. 1. The p34cdc2 kinase activity\nof\nrat oocytes decreased signiﬁcantly at 45 min (after\n30 min cultivation) and more signiﬁcantly at 75 min (after\n60 min cultivation). The decreased activity was maintained\nuntil 120 min (after 105 min cultivation). Probably due to\nthe decrease of p34cdc2 kinase activity, the release from\nmeiotic arrest of ovulated oocytes occurred in a time-\ndependent manner, as shown in Fig. 2. Within 75 min\nafter animal killing, more than half of the oocytes pro-\ngressed beyond the MII stage. At 135 min, most of the\noocytes were at either the AII or the TII stage. The pro-\nportion of in vitro-aged rat oocytes promoting PCC of the\ninjected cumulus cell nuclei is shown in Table 1. When\nnuclear injection was completed within 45 min from ani-\nmal killing, a positive sign of PCC was observed in 32.5%\nof the NT oocytes. On the other hand, PCC occurred in\nonly 2.6% of the oocytes into which cumulus cell nuclei\nwere injected between 46 and 120 min.\nExperiment 2: effect of oocyte enucleation\nThe activities of p34cdc2 kinase in non-enucleated and\nenucleated oocytes within 45 min were compared in rats\nand mice, as shown in Fig. 3. In both species, there was\na slight decrease in the p34cdc2 kinase activity after enu-\ncleation in both rat and mouse oocytes. A considerable\nRole of MPF in PCC induction of rat oocytes 175\ntreatment (enucleation or cytoplasm removal) is shown in\nTable 2. In rats, despite little reduction of p34cdc2 kinase\nactivity, few of the oocytes that either had a small portion\nof cytoplasm-removed (17.1%) or the non-treated controls\n(28.2%) could support PCC at 105 min after animal\nin the\nkilling. However, PCC was not observed at all\nfound in the p34cdc2 kinase activity\ndifference was\nbetween rat and mouse oocytes, because the mouse\noocytes carried a p34cdc2 kinase activity 2.5 times that of\nrat oocytes either immediately or up to 60 min after enu-\ncleation or cytoplasm removal and compared with non-\ntreated, age-matched controls, as shown in Fig. 4. The\nactivities of p34cdc2 kinase were measured in rat oocytes\nimmediately after or 60 min after one of three treatments\n(enucleated, cytoplasm-removed and non-treated), as\nshown in Fig. 4. Surprisingly, a higher p34cdc2 kinase\nactivity was detected in the oocytes immediately after\nenucleation, although the p34cdc2 kinase activity of\noocytes immediately after cytoplasm removal was not\nfrom that of non-treated oocytes. The p34cdc2\ndifferent\nin enucleated oocytes dramatically\nactivity\nkinase\ndecreased within 30 min after enucleation. At 45 min\nafter enucleation, the activity was further decreased and\nthereafter maintained until 60 min. However, the activity\nin non-treated or cytoplasm-removed oocytes did not sig-\nniﬁcantly differ by 60 min. The proportion of rat and\nmouse oocytes promoting PCC in relation to mechanical\nenucleated oocytes where p34cdc2 kinase activity was sig-\nniﬁcantly decreased at 60 min after completion of enu-\ncleation. In contrast, PCC of mouse cumulus cell nuclei\noccurred in most of the mouse oocytes before enucleation\n(96.7%) and even after enucleation (93.1%).\nExperiment 3: effect of roscovitine treatment\nThe time-dependent decrease of p34cdc2 kinase activity in\nroscovitine-treated rat oocytes is shown in Fig. 5. The\nactivity\nlinearly\nin roscovitine-treated oocytes was\ndecreased until 75 min and maintained until 120 min.\nAt 45, 75 and 120 min, each activity in the roscovitine-\ntreated oocytes was signiﬁcantly lower than that of non-\ntreated control oocytes. The result of PCC incidence in the\noocytes treated with or without roscovitine for 45 min is\nshown in Table 3. None of the roscovitine-treated oocytes\npromoted PCC, although 41.7% of non-treated oocytes\npromoted it.\nExperiment 4: effect of MG132 treatment\nThe activities of p34cdc2 kinase in MG132-treated rat\noocytes are shown in Fig. 6. The level of p34cdc2 kinase\nactivity still decreased in the MG132-treated oocytes\nafter 30 min of culture, but were signiﬁcantly greater\nthan in control oocytes. The p34cdc2 kinase activity was\nmaintained during the further incubation up to 105 min.\nThe kinase activities of MG132-treated oocytes at 60\nand 105 min cultivation were signiﬁcantly greater than\nthose of control oocytes. As shown in Table 4, the pro-\nportion of MG132-treated oocytes promoting PCC\n(66.0%) was higher than that of the non-treated oocytes\n(46.5%) when the nuclear\ninjection was completed\nwithin 45 min.\nExperiment 5: effect of PCC on NT embryo\ndevelopment\nthe reconstructed embryos\nAfter oocyte activation with the electrical stimulus plus 6-\nDMAP,\nfrom both groups\n(MG132-treated MII oocytes and non-treated TII oocytes\nat the enucleation) formed one or two pseudo-pronuclei\n(91.7 and 87.5% in total respectively). After activation, a\ngreater proportion of oocytes enucleated at the MII stage\ntended to form two pseudo-pronuclei compared with\nthose enucleated at the TII stage. A signiﬁcantly higher\nproportion of the NT embryos in the MG132-treated group\nvs the non-treated group (59.1 vs 7.1%) developed to the\ntwo-cell stage.\nDiscussion\nRat oocytes have poor potential\nfor promoting PCC of\ninjected somatic cell nuclei, and the rate was markedly\ndecreased in aged or enucleated oocytes (Hirabayashi\nIn the present study, rat oocytes into\net al. 2003a,b).\nwhich cumulus cell nuclei were injected within 45 min\nRole of MPF in PCC induction of rat oocytes 177\nthe non-treated oocytes during the same period, may\ninﬂuence the outcome of PCC. The high concentration of\nroscovitine affects the other cyclin-dependent protein\nkinases such as cdk2 (De Azevado et al. 1997). The\nunknown pathway and/or chemical toxicity of roscovitine\ncannot be ruled out\nfor complete suppression of PCC\ninduction. Moreover, the level of activity in rat oocytes\nwas less than half compared with that in mouse oocytes,\nwhich had a high potential for promoting PCC. Judging\nfrom these results, we estimated that the high activity of\np34cdc2 kinase is associated with promoting PCC of\ninjected somatic cell nuclei, and the low level of p34cdc2\nkinase activity is one of the reasons for the poor potential\nto promote PCC in rat oocytes compared with that of\nmouse oocytes. The reason why the potential\nlevel of\np34cdc2 kinase activity is different between mouse and rat\noocytes remains unclear. Our previous study indicated, at\nleast, that the PCC-inducing potential is determined by\nthe origin of the recipient cytoplasm, not by the origin of\nin the interspecies NT experiment\nthe donor nucleus,\nbetween mouse and rat\nIn\n(Hirabayashi et al. 2003a).\nsomatic cell division, dissolution of the nuclear envelope\noccurs during G2/M transition in mitosis (Nurse 1990) and\nis induced by phosphorylation of lamin, one of the major\nnuclear membrane proteins (Peter et al. 1990). In oocytes,\np34cdc2 kinase had the potential to phosphorylate lamins,\ninducing meiotic breakdown of the nuclear envelope and\nthe meiotic resumption (Dessev & Goldman 1988). There-\nfore, it is possible that p34cdc2 kinase plays an important\nrole in the destruction of the nuclear membrane of the\ninjected somatic cell through phosphorylation of lamin.\nOne possible pathway known for inactivation of p34cdc2\nkinase is the calmodulin-dependent pathway via Ca2þ.\nWe have reported that the Ca2þ-dependent pathway trig-\ngers the degradation of cyclin B and results in inactivation\nof p34cdc2 kinase in porcine oocytes (Ito et al. 2003,\n2004a). In Xenopus oocytes (Lorca et al. 1994, Morin et al.\n1994), activation of calmodulin-dependent protein kinase\nII induces the degradation of cyclin B via the ubiquitin\nproteasome pathway. Based on these literature sources\nand preliminary data, we have investigated the effect of a\nproteasome inhibitor MG132 on p34cdc2 kinase activity\nand PCC incidence. The PCC incidence in rat oocytes was\nsigniﬁcantly improved by MG132 treatment (Table 4). This\nresult matched well with the report by Josefsberg et al.\n(2000), who demonstrated that MG132-treated rat oocytes\ncontained a high amount of cyclin B1.\nIn the present\nstudy, MG132 also provided a suppressive effect on inacti-\nvation of p34cdc2 kinase (Fig. 6). Use of ALLN, a neutral\ncysteine protease inhibitor, was also slightly effective for\nimproving the PCC incidence in rat oocytes (Hirabayashi\net al. 2003b). Thus, the Ca2þ-dependent degradation path-\nway, which may induce a decrease in the cyclin B1 level,\nis activated in aged rat oocytes, which results in a low\nlevel of p34cdc2 kinase activity, and a low potential\nto promote PCC. Our unpublished data (M Hirabayashi\nand J Ito, unpublished data) showing that the decreased\nIn mouse oocytes, PCC was initiated at\npromoted PCC (frequency 28–47% in Tables 1 –4), but\nthe incidence of PCC decreased with oocyte aging with a\nsigniﬁcant decrease in p34cdc2 kinase activity (Table 1,\nFig. 1).\nleast\n30 min after nuclear injection and lasted until 2 h (Chen\net al. 2004), therefore the timing of PCC observation in\nthe present study (60 min after nuclear injection) might be\noptimal. When oocytes are treated with roscovitine,\nwhich decreases the level of p34cdc2 kinase activity, PCC\nformation was never observed even though nuclear injec-\ntion is completed within 45 min. Roscovitine is widely\nused as cyclin-dependent kinase inhibitor\nto suppress\np34cdc2 kinase in rodent oocytes (Phillips et al. 2002).\nTherefore, the rapid decrease of p34cdc2 kinase activity in\nthe roscovitine-treated oocytes found between 45 and\n75 min (Fig. 5), when compared with the slow decrease in\na Oocytes were treated with 5 mM MG132 until just before enucleation after oocyte recovery.\nb Oocytes were cultured for about 2 h and only oocytes with a completely extruded polar body (the TII stage) were used.\nc Percentage of reconstructed embryos with 1 or 2 pronuclei.\nd – e Different superscripts denote signiﬁcant difference (P , 0.05).\nconcentration of extracellular free calcium in the handling\nmedium partially inhibits spontaneous activation of rat\noocytes, support this hypothesis.\nThe other pathway for inactivation by phosphorylation\nof 14Thr and 15Tyr of p34cdc2 kinase has been reported in\naged porcine oocytes (Kikuchi et al. 2000). In the same\nstudy, it has been reported that caffeine suppressed the\nfrom MPF to pre-MPF (14Thr/15Tyr-phosphorylated\nshift\np34cdc2 kinase þ cyclin B1) in the aged porcine oocytes.\nOkadaic acid is also known as a protein phosphatase\ninhibitor which prevents phosphorylation of 14Thr and\n15Tyr of p34cdc2 kinase (Rime et al. 1995). In our prelimi-\nnary study, PCC incidence of injected cumulus cell nuclei\nwas slightly improved in rat oocytes treated with okadaic\nacid (M Hirabayashi and J Ito, unpublished data). There-\nfore, the phosphorylation of p34cdc2 kinase as well as the\ncyclin degradation may be involved in spontaneous acti-\nvation of rat oocytes during in vitro culture.\nThe level of p34cdc2 kinase activity in non-treated rat\noocytes was similar to that of cytoplasm-removed or enu-\ncleated oocytes in the case where the oocytes were\nsampled immediately after the treatment\n(Fig. 4). The\nmechanical puncture of the oocyte membrane and the\naspiration of the cytoplasm or karyoplast did not inﬂuence\nthe p34cdc2 kinase activity immediately after the enuclea-\ntion procedure beyond the level expected from the rela-\ntive volume of aspirated cytoplasm (, 10–20%). These\nresults strongly suggest that active p34cdc2 kinase is not\naccumulated near the metaphase spindles in rat oocytes.\nThere was signiﬁcant difference in the p34cdc2 kinase\nactivity between 45 and 75 min in both non-treated and\nenucleated groups (Fig. 4). A dramatic decrease in the\nactivity of p34cdc2 kinase was observed in the rat oocytes\nenucleated and cultured for 30 min compared with the\nnon-treated group (Fig. 4), suggesting that\nthe oocyte\nmetaphases per se are involved in the maintenance of\np34cdc2 kinase activity. We have previously reported in\nporcine oocytes that activation of p34cdc2 kinase was\ndependent on the synthesis of cyclin B and that the syn-\nthesis was induced by the activation of mitogen-activated\nprotein (MAP) kinase (Ito et al. 2004b). The MAP kinase\nsuppressed Myt1 activity that phosphorylated p34cdc2\nkinase at 14Thr and 15Tyr (Mueller et al. 1995, Palmer et al.\n1998). Because activated MAP kinase is localized on the\nmeiotic spindle in porcine oocytes (Lee et al. 2000), it is\npossible that enucleation results in the loss of MAP kinase\nfrom the recipient oocytes.\ninjection into the MG132-treated, MII-enu-\ncleated rat oocytes resulted in a higher development to\nthe two-cell stage when compared with non-treated, TII-\nenucleated rat oocytes after 24 h cultivation (Table 5).\nThe favorable circumstance of recipient cytoplasm with\na high MPF activity must induce PCC and promote sub-\nsequent development of NT embryos. Wakayama &\nYanagimachi\nthe injected\n(2001) reported in mice that\ncumulus cell nuclei undergoing PCC transformation to\nthe two pseudo-pronuclei and such NT embryos are able\nto develop to full-term. In contrast, a different situation\nhas been reported in the cloning of large domestic ani-\nmals. The somatic cell NT in the livestock species is suc-\ncessfully reproducible when enucleated and activated\noocytes with a low MPF activity are used as recipient\ncytoplasm (Baguisi et al. 1999, Kurosaka et al. 2002,\nShin et al. 2002). Such a difference between rodents\nand large domestic animals remains unclear, but our\nthat\npresent data suggest\ninduction of PCC in recon-\nstructed embryos\nrelated to the high activity of\nis\np34cdc2 kinase, which seems to play a key role in the\nsuccessful rat cloning.\nthe decreased level of p34cdc2 kinase\nactivity in aged or enucleated rat oocytes is responsible\nfor their inability to promote PCC of microinjected donor\ncell nuclei. Inhibition of p34cdc2 kinase inactivation by\nchemicals such as MG132 is in part effective for rat\noocytes to promote PCC and cleavage. These data may\nexpand our existing knowledge on characteristics of rat\noocytes and contribute to the development of a rat NT\nprotocol using G0/G1-stage donor cells.",
         null
        ],
        [
         "48",
         "002cc4c2e457c3cc0fb0f704d0483a003e8a0575",
         "Comparison of automated microbroth dilution and agar dilution for antimicrobial susceptibility of Campylobacter jejuni isolated from dairy sources.",
         "OBJECTIVES\nTo compare the agreement between microbroth dilution and agar dilution for antimicrobial susceptibility testing of Campylobacter jejuni.\n\n\nMETHODS\nUtilizing commercially prepared antimicrobial panels, microbroth dilution was compared with agar dilution for determining antimicrobial susceptibility in C. jejuni isolates. To assess the performance of both techniques for ampicillin, 190 C. jejuni isolates from dairy cattle were utilized. A group of 172 C. jejuni isolates from dairy sources were used to compare the susceptibility to ciprofloxacin, erythromycin, nalidixic acid and tetracycline.\n\n\nRESULTS\nOur results indicate that microbroth dilution and agar dilution agree within +/-1 log2 dilution for 86.7% of the isolates tested. Ciprofloxacin had the highest level of agreement for isolates tested by both techniques, resulting in a kappa of 0.886 and 97.1% agreement +/-1 log2 dilution. The least agreement was observed in determining the susceptibility of isolates to ampicillin and erythromycin (82.1 and 79.7% agreement +/-1 log2 dilution). However, kappa statistics were considered to have good agreement for these antimicrobials. There were no significant differences in the summary statistics for any of the five antimicrobials evaluated for the isolates analysed by the percentage of resistant isolates, MIC50, MIC75 or MIC90 beyond +/-1 log2 dilution. There was no association in the classification of resistance by the testing methods employed. We also demonstrated that the quality control strain of C. jejuni ATCC 33650 performed in a consistent manner for both agar dilution and microbroth dilution.\n\n\nCONCLUSIONS\nMicrobroth dilution may be an acceptable alternative to agar dilution for determining susceptibility of C. jejuni in research or surveillance where flow of samples, labour efficiency and cost may restrict the use of agar dilution.",
         "",
         "https://academic.oup.com/jac/article-pdf/56/4/686/2308205/dki309.pdf",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ],
        [
         "49",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "Numerical Periodic Normalization for Codim 1 Bifurcations of Limit Cycles",
         "Explicit computational formulas for the coefficients of the periodic normal forms for all codim 1 bifurcations of limit cycles in generic autonomous ODEs are derived. They include second-order coefficients for the fold (limit point) bifurcation, as well as third-order coefficients for the flip (period-doubling) and Neimark--Sacker (torus) bifurcations. The formulas are independent of the dimension of the phase space and involve solutions of certain boundary-value problems on the interval [0,T], where T is the period of the critical cycle, as well as multilinear functions from the Taylor expansion of the right-hand sides near the cycle. The formulas allow us to distinguish between sub- and supercritical bifurcations, in agreement with earlier asymptotic expansions of the bifurcating solutions. Our formulation makes it possible to use robust numerical boundary-value algorithms based on orthogonal collocation, rather than shooting techniques, which greatly expands its applicability. The actual implementation is described in detail. We include three numerical examples, in which codim 2 singularities are detected along branches of codim 1 bifurcations of limit cycles as zeros of the periodic normal form coefficients.",
         "Numerical Methods for Two-Parameter Local Bifurcation Analysis of Maps Newton's descent method for the determination of invariant tori. A Newton Method for Locating Invariant Tori of Maps Sensitivity Analysis of ODES/DAES Using the Taylor Series Method Numerical Normal Forms for Codim 2 Bifurcations of Fixed Points with at Most Two Critical Eigenvalues Numerical Methods for the Continuation of Invariant Tori MATCONT: A MATLAB package for numerical bifurcation analysis of ODEs Cl_matcont: a continuation toolbox in Matlab Computation of Periodic Solution Bifurcations in ODEs Using Bordered Systems Chapter 4 – Numerical Continuation, and Computation of Normal Forms Computing Periodic Orbits and their Bifurcations with Automatic Differentiation Numerical Normalization Techniques for All Codim 2 Bifurcations of Equilibria in ODE's Topics in bifurcation theory and applications Elements of applied bifurcation theory (2nd ed.) Harmonic balance analysis of period-doubling bifurcations with implications for control of nonlinear dynamics Algorithm 755: ADOL-C: a package for the automatic differentiation of algorithms written in C/C++ Bifurcation and predictability analysis of a low-order atmospheric circulation model Models of complex dynamics in nonlinear systems Bifurcation sequences at 1:4 resonance: an inventory Harmonic balance methods for the analysis of chaotic dynamics in nonlinear systems BIFURCATIONS AND CHAOS IN A PERIODIC PREDATOR-PREY MODEL NUMERICAL ANALYSIS AND CONTROL OF BIFURCATION PROBLEMS (II): BIFURCATION IN INFINITE DIMENSIONS Numerical Calculation of Invariant Tori THE QUASIPERIODIC ROUTE TO CHAOS IN A MODEL OF THE PEROXIDASE-OXIDASE REACTION Global Characterization of the Normal Form for a Vector Field Near a Closed Orbit A simple global characterization for normal forms of singular vector fields Normal form reduction for time-periodically driven differential equations Numerical computation of invariant circles of maps NONLINEAR OSCILLATIONS, DYNAMICAL SYSTEMS, AND BIFURCATIONS OF VECTOR FIELDS (Applied Mathematical Sciences, 42) Irregularity: a fundamental property of the atmosphere* Nonlinear Oscillations, Dynamical Systems, and Bifurcations of Vector Fields Collocation at Gaussian Points Continuation of Quasi-periodic Invariant Tori The fold-flip bifurcation On the computation of characteristic multipliers for predicting limit cycle bifurcations Elements of applied bifurcation theory AUTO 2000 : CONTINUATION AND BIFURCATION SOFTWARE FOR ORDINARY DIFFERENTIAL EQUATIONS (with HomCont) CONTENT: A multiplatform environment for analyzing dynamical systems Normal forms of bifurcating periodic orbits Numerical computation of periodic solution branches and oscillatory dynamics of the stirred tank reactor with A → B → C reactions Geometrical Methods in the Theory of Ordinary Differential Equations Elementary stability and bifurcation theory Bifurcation of maps and applications Topics in bifurcation theory Numerical solution of boundary value problems in ordinary differential equations Numerical Continuation, And Computation Of Normal Forms CAPD: Computer assisted proofs in dynamics",
         "http://arxiv.org/pdf/1111.4445.pdf",
         "",
         "",
         "",
         "",
         "",
         "",
         null
        ]
       ],
       "shape": {
        "columns": 12,
        "rows": 69988
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>referenceTitles</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>pdf_abstract</th>\n",
       "      <th>pdf_introduction</th>\n",
       "      <th>pdf_results</th>\n",
       "      <th>pdf_discussion</th>\n",
       "      <th>pdf_methods</th>\n",
       "      <th>pdf_text</th>\n",
       "      <th>problem</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>Disruption of Fibroblast Growth Factor Signal ...</td>\n",
       "      <td>Purpose: Synovial sarcoma is a soft tissue sar...</td>\n",
       "      <td></td>\n",
       "      <td>https://aacrjournals.org/clincancerres/article...</td>\n",
       "      <td>Abstract Purpose: Synovial sarcoma is a soft t...</td>\n",
       "      <td>Synovial sarcoma is the most frequent soft-tis...</td>\n",
       "      <td>Results\\nSynovial sarcoma cell lines expressed...</td>\n",
       "      <td>Discussion\\nFGFs may activate genetic programs...</td>\n",
       "      <td>Materials and Methods\\nTumor tissues were obta...</td>\n",
       "      <td>Abstract Purpose: Synovial sarcoma is a soft t...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>Shuffled iterative decoding</td>\n",
       "      <td>Shuffled versions of iterative decoding of low...</td>\n",
       "      <td>Low-Density Parity-Check Codes Turbo decoder a...</td>\n",
       "      <td>http://www.merl.com/publications/docs/TR2005-0...</td>\n",
       "      <td>Replica shufﬂed versions of interactive decode...</td>\n",
       "      <td>This work may not be copied or reproduced in w...</td>\n",
       "      <td>D. Simulation results\\nIII. ITERATIVE DECODING...</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>Replica shufﬂed versions of interactive decode...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>Diphasic effects of Astragalus membranaceus BU...</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.jstage.jst.go.jp/article/bpb/28/8/...</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td>The  roots  of  Astragalus  membranaceus BUNGE...</td>\n",
       "      <td>M)  induced  a  similar  sustained  contractio...</td>\n",
       "      <td>DISCUSSION\\nThe present study ﬁrst and detaile...</td>\n",
       "      <td>MATERIALS AND METHODS\\nPlant  Material  and  P...</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>Critical roles of CD146 in zebrafish vascular ...</td>\n",
       "      <td>In this report, we use zebrafish as a model sy...</td>\n",
       "      <td></td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/pdfdirect/...</td>\n",
       "      <td>In this report, we use zebraﬁsh as a model sys...</td>\n",
       "      <td>INTRODUCTION\\nCD146, also known as MUC18, A32,...</td>\n",
       "      <td>A cDNA clone (clone number, 3512;\\nGenBank acc...</td>\n",
       "      <td></td>\n",
       "      <td>Total RNA was collected from ze-\\nbraﬁsh embry...</td>\n",
       "      <td>In this report, we use zebraﬁsh as a model sys...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>Differential Contribution of Troponin I Phosph...</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td></td>\n",
       "      <td>http://www.jbc.org/article/S0021925820589394/pdf</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td>Protein kinase C (PKC)2 activation is an impor...</td>\n",
       "      <td>EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...</td>\n",
       "      <td>DISCUSSION\\nResults from the present study dem...</td>\n",
       "      <td>EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69983</th>\n",
       "      <td>fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6</td>\n",
       "      <td>Irregular cycles and steroid hormones in polyc...</td>\n",
       "      <td>BACKGROUND\\nThis cross-sectional study was und...</td>\n",
       "      <td>Predictors for treatment failure after laparos...</td>\n",
       "      <td>https://academic.oup.com/humrep/article-pdf/20...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69984</th>\n",
       "      <td>fffbc9b37f20515de2ff4fe60353d52dc37e65f5</td>\n",
       "      <td>An organizational grid of federated MOSIX clus...</td>\n",
       "      <td>MOSIX is a cluster management system that uses...</td>\n",
       "      <td>From sandbox to playground: dynamic virtual en...</td>\n",
       "      <td>http://www.cs.virginia.edu/~jl3aq/courses/CS85...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69985</th>\n",
       "      <td>fffd2d6d54b4023ece28bfdebffdfb00502a09ec</td>\n",
       "      <td>Mutations in the RNA Polymerase III Subunit Rp...</td>\n",
       "      <td>ABSTRACT Termination by RNA polymerase III (Po...</td>\n",
       "      <td>Nuclear surveillance and degradation of hypomo...</td>\n",
       "      <td>https://europepmc.org/articles/pmc543423?pdf=r...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69986</th>\n",
       "      <td>ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3</td>\n",
       "      <td>The Core Histone N-terminal Tail Domains Funct...</td>\n",
       "      <td>Salt-dependent oligomerization of nucleosomal ...</td>\n",
       "      <td>Histone modifications: combinatorial complexit...</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820789519/pdf</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69987</th>\n",
       "      <td>ffffeaedbe7931d161ab532015de3be15f99b180</td>\n",
       "      <td>Of mice and men: should we extrapolate rodent ...</td>\n",
       "      <td>None</td>\n",
       "      <td>Developmental neurotoxicity of ketamine: morph...</td>\n",
       "      <td>https://pubs.asahq.org/anesthesiology/article-...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>69988 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paperId  \\\n",
       "0      0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1      000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2      000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3      0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4      00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "...                                         ...   \n",
       "69983  fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6   \n",
       "69984  fffbc9b37f20515de2ff4fe60353d52dc37e65f5   \n",
       "69985  fffd2d6d54b4023ece28bfdebffdfb00502a09ec   \n",
       "69986  ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3   \n",
       "69987  ffffeaedbe7931d161ab532015de3be15f99b180   \n",
       "\n",
       "                                                   title  \\\n",
       "0      Disruption of Fibroblast Growth Factor Signal ...   \n",
       "1                            Shuffled iterative decoding   \n",
       "2      Diphasic effects of Astragalus membranaceus BU...   \n",
       "3      Critical roles of CD146 in zebrafish vascular ...   \n",
       "4      Differential Contribution of Troponin I Phosph...   \n",
       "...                                                  ...   \n",
       "69983  Irregular cycles and steroid hormones in polyc...   \n",
       "69984  An organizational grid of federated MOSIX clus...   \n",
       "69985  Mutations in the RNA Polymerase III Subunit Rp...   \n",
       "69986  The Core Histone N-terminal Tail Domains Funct...   \n",
       "69987  Of mice and men: should we extrapolate rodent ...   \n",
       "\n",
       "                                                abstract  \\\n",
       "0      Purpose: Synovial sarcoma is a soft tissue sar...   \n",
       "1      Shuffled versions of iterative decoding of low...   \n",
       "2      This study was designed to investigate the eff...   \n",
       "3      In this report, we use zebrafish as a model sy...   \n",
       "4      Cardiac troponin I is a phosphorylation target...   \n",
       "...                                                  ...   \n",
       "69983  BACKGROUND\\nThis cross-sectional study was und...   \n",
       "69984  MOSIX is a cluster management system that uses...   \n",
       "69985  ABSTRACT Termination by RNA polymerase III (Po...   \n",
       "69986  Salt-dependent oligomerization of nucleosomal ...   \n",
       "69987                                               None   \n",
       "\n",
       "                                         referenceTitles  \\\n",
       "0                                                          \n",
       "1      Low-Density Parity-Check Codes Turbo decoder a...   \n",
       "2                                                          \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "69983  Predictors for treatment failure after laparos...   \n",
       "69984  From sandbox to playground: dynamic virtual en...   \n",
       "69985  Nuclear surveillance and degradation of hypomo...   \n",
       "69986  Histone modifications: combinatorial complexit...   \n",
       "69987  Developmental neurotoxicity of ketamine: morph...   \n",
       "\n",
       "                                           openAccessPdf  \\\n",
       "0      https://aacrjournals.org/clincancerres/article...   \n",
       "1      http://www.merl.com/publications/docs/TR2005-0...   \n",
       "2      https://www.jstage.jst.go.jp/article/bpb/28/8/...   \n",
       "3      https://onlinelibrary.wiley.com/doi/pdfdirect/...   \n",
       "4       http://www.jbc.org/article/S0021925820589394/pdf   \n",
       "...                                                  ...   \n",
       "69983  https://academic.oup.com/humrep/article-pdf/20...   \n",
       "69984  http://www.cs.virginia.edu/~jl3aq/courses/CS85...   \n",
       "69985  https://europepmc.org/articles/pmc543423?pdf=r...   \n",
       "69986   http://www.jbc.org/article/S0021925820789519/pdf   \n",
       "69987  https://pubs.asahq.org/anesthesiology/article-...   \n",
       "\n",
       "                                            pdf_abstract  \\\n",
       "0      Abstract Purpose: Synovial sarcoma is a soft t...   \n",
       "1      Replica shufﬂed versions of interactive decode...   \n",
       "2      This study was designed to investigate the eff...   \n",
       "3      In this report, we use zebraﬁsh as a model sys...   \n",
       "4      Cardiac troponin I is a phosphorylation target...   \n",
       "...                                                  ...   \n",
       "69983                                                NaN   \n",
       "69984                                                NaN   \n",
       "69985                                                NaN   \n",
       "69986                                                NaN   \n",
       "69987                                                NaN   \n",
       "\n",
       "                                        pdf_introduction  \\\n",
       "0      Synovial sarcoma is the most frequent soft-tis...   \n",
       "1      This work may not be copied or reproduced in w...   \n",
       "2      The  roots  of  Astragalus  membranaceus BUNGE...   \n",
       "3      INTRODUCTION\\nCD146, also known as MUC18, A32,...   \n",
       "4      Protein kinase C (PKC)2 activation is an impor...   \n",
       "...                                                  ...   \n",
       "69983                                                NaN   \n",
       "69984                                                NaN   \n",
       "69985                                                NaN   \n",
       "69986                                                NaN   \n",
       "69987                                                NaN   \n",
       "\n",
       "                                             pdf_results  \\\n",
       "0      Results\\nSynovial sarcoma cell lines expressed...   \n",
       "1      D. Simulation results\\nIII. ITERATIVE DECODING...   \n",
       "2      M)  induced  a  similar  sustained  contractio...   \n",
       "3      A cDNA clone (clone number, 3512;\\nGenBank acc...   \n",
       "4      EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...   \n",
       "...                                                  ...   \n",
       "69983                                                NaN   \n",
       "69984                                                NaN   \n",
       "69985                                                NaN   \n",
       "69986                                                NaN   \n",
       "69987                                                NaN   \n",
       "\n",
       "                                          pdf_discussion  \\\n",
       "0      Discussion\\nFGFs may activate genetic programs...   \n",
       "1                                                          \n",
       "2      DISCUSSION\\nThe present study ﬁrst and detaile...   \n",
       "3                                                          \n",
       "4      DISCUSSION\\nResults from the present study dem...   \n",
       "...                                                  ...   \n",
       "69983                                                NaN   \n",
       "69984                                                NaN   \n",
       "69985                                                NaN   \n",
       "69986                                                NaN   \n",
       "69987                                                NaN   \n",
       "\n",
       "                                             pdf_methods  \\\n",
       "0      Materials and Methods\\nTumor tissues were obta...   \n",
       "1                                                          \n",
       "2      MATERIALS AND METHODS\\nPlant  Material  and  P...   \n",
       "3      Total RNA was collected from ze-\\nbraﬁsh embry...   \n",
       "4      EXPERIMENTAL PROCEDURES\\nMutagenesis Strategy—...   \n",
       "...                                                  ...   \n",
       "69983                                                NaN   \n",
       "69984                                                NaN   \n",
       "69985                                                NaN   \n",
       "69986                                                NaN   \n",
       "69987                                                NaN   \n",
       "\n",
       "                                                pdf_text problem  \n",
       "0      Abstract Purpose: Synovial sarcoma is a soft t...    None  \n",
       "1      Replica shufﬂed versions of interactive decode...    None  \n",
       "2      This study was designed to investigate the eff...    None  \n",
       "3      In this report, we use zebraﬁsh as a model sys...    None  \n",
       "4      Cardiac troponin I is a phosphorylation target...    None  \n",
       "...                                                  ...     ...  \n",
       "69983                                                NaN     NaN  \n",
       "69984                                                NaN     NaN  \n",
       "69985                                                NaN     NaN  \n",
       "69986                                                NaN     NaN  \n",
       "69987                                                NaN     NaN  \n",
       "\n",
       "[69988 rows x 12 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(metadata)\n",
    "display(textdata_pdf)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "'https://www.cell.com/article/S009286740500098X/pdf'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pdf_data, pdf_sections, pdf_tags = extract_pdf_sections(os.path.join(path_to_data, 'pdf', '0', 'PIIS009286740500098X (1).pdf'),'I. Krylova,E. Sablin,Jamie M. R. Moore,R. Xu,G. Waitt,J. A. MacKay,D. Juzumiene,J. Bynum,K. Madauss,V. Montana,L. Lebedeva,M. Suzawa,Jon D. Williams,Shawn P. Williams,R. Guy,J. W. Thornton,R. Fletterick,T. Willson,H. Ingraham', 0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Krylova',\n",
       " 'Sablin',\n",
       " 'Moore',\n",
       " 'Xu',\n",
       " 'Waitt',\n",
       " 'MacKay',\n",
       " 'Juzumiene',\n",
       " 'Bynum',\n",
       " 'Madauss',\n",
       " 'Montana',\n",
       " 'Lebedeva',\n",
       " 'Suzawa',\n",
       " 'Williams',\n",
       " 'Williams',\n",
       " 'Guy',\n",
       " 'Thornton',\n",
       " 'Fletterick',\n",
       " 'Willson',\n",
       " 'Ingraham']"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[name.split(' ')[-1] for name in data4pdf['authorName'].iloc[0].split(',')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "UNDEFINED \n",
      " Summary\n",
      "\n",
      "ABSTRACT \n",
      " Vertebrate members of the nuclear receptor NR5A sub-\n",
      "family, which includes steroidogenic factor 1 (SF-1) and\n",
      "liver receptor homolog 1 (LRH-1), regulate crucial as-\n",
      "pects of development, endocrine homeostasis, and\n",
      "metabolism. Mouse LRH-1 is believed to be a ligand-\n",
      "independent transcription factor with a large and\n",
      "empty hydrophobic pocket. Here we present struc-\n",
      "tural and biochemical data for three other NR5A mem-\n",
      "bers—mouse and human SF-1 and human LRH-1—\n",
      "which reveal that these receptors bind phosphatidyl\n",
      "inositol second messengers and that ligand binding\n",
      "is required for maximal activity. Evolutionary analysis\n",
      "of structure-function relationships across the SF-1/\n",
      "LRH-1 subfamily indicates that ligand binding is the\n",
      "ancestral state of NR5A receptors and was uniquely\n",
      "diminished or altered in the rodent LRH-1 lineage. We\n",
      "propose that phospholipids regulate gene expression\n",
      "by directly binding to NR5A nuclear receptors.\n",
      "\n",
      "INTRODUCTION \n",
      " Introduction\n",
      "\n",
      "INTRODUCTION \n",
      " Unlike most nuclear receptors, which are regulated by\n",
      "steroid, retinoid, and other nonpolar ligands, NR sub-\n",
      "family 5A are constitutively active in many cell types,\n",
      "and obvious ligand candidates have not been forth-\n",
      "coming. SF-1 (NR5A1) orchestrates endocrine tissue\n",
      "development and male sexual differentiation and con-\n",
      "\n",
      "INTRODUCTION \n",
      " trols steroid biosynthesis (Ingraham et al., 1994; Luo et\n",
      "al., 1994). LRH-1 (NR5A2) is essential in early develop-\n",
      "ment and regulates bile acid synthesis, cholesterol\n",
      "transport, and ovarian function (Fayard et al., 2004; Lu\n",
      "et al., 2000). This subfamily is distinct in that members\n",
      "are able to bind as monomers to DNA, and they pos-\n",
      "sess an unusually large hinge domain that is subject to\n",
      "posttranslational modifications, including phosphoryla-\n",
      "tion and sumoylation (Desclozeaux et al., 2002; Lee et\n",
      "al., 2005). Although physiological roles for these recep-\n",
      "tors have emerged, ligand candidates have remained\n",
      "controversial. The proposed SF-1 ligand 25-hydro-\n",
      "xycholesterol, while appealing because of the recep-\n",
      "tor's regulation of steroid synthesis, appears to be in-\n",
      "correct because oxysterols fail to activate SF-1 in most\n",
      "cell lines (Christenson et al., 1998; Lala et al., 1997; Mel-\n",
      "lon and Bair, 1998). Further, the notion that ligands\n",
      "might be dispensable for this subfamily of receptors\n",
      "was bolstered by the crystal structure of mouse LRH-1\n",
      "ligand binding domain (LBD), showing an active confor-\n",
      "mation in the absence of either ligand or coregulator\n",
      "peptide. Unlike other orphan nuclear receptors, which\n",
      "either lack a ligand binding pocket (e.g., Nurr1, DHR38,\n",
      "or ERRα) or have a constitutively bound lipid (e.g.,\n",
      "HNF4α, HNF4γ, or USP), mLRH-1 was found to possess\n",
      "a large unoccupied hydrophobic pocket of w830 Å (re-\n",
      "viewed in Li et al. [2003]). Remarkably, mutations that\n",
      "filled the mLRH-1 pocket failed to reduce transcrip-\n",
      "tional activity (Sablin et al., 2003).\n",
      "\n",
      "INTRODUCTION \n",
      " Whereas an elegant structural paradigm has devel-\n",
      "oped to explain the activation of ligand-regulated re-\n",
      "ceptors, no uniform mechanism has been proposed to\n",
      "account for the modulation of orphan nuclear receptor\n",
      "activity (Steinmetz et al., 2001; Weatherman et al.,\n",
      "1999). The presence of fortuitous lipid molecules re-\n",
      "vealed in crystallization experiments can be associated\n",
      "with an inactive LBD conformation as noted for the\n",
      "dipterian USP LBD (Billas et al., 2001). On the other\n",
      "hand, lipid components have been proposed to stabi-\n",
      "lize an active LBD conformation in constitutively active\n",
      "nuclear receptors, with HNF4 binding fatty acids (Dhe-\n",
      "Paganon et al., 2002; Wisely et al., 2002) and RORα\n",
      "binding cholesterol (Kallen et al., 2002). In this context,\n",
      "the structure of the murine LRH-1 with its empty ligand\n",
      "binding pocket remains a puzzle. To further explore\n",
      "how NR5A orphan receptors are regulated, we obtained\n",
      "the LBD crystal structures of other members of this\n",
      "subfamily, including mSF-1, hSF-1, and hLRH-1. Here\n",
      "we provide structural and biochemical data supporting\n",
      "ligand regulation for NR5A orphan nuclear receptors.\n",
      "\n",
      "INTRODUCTION \n",
      " Results and Discussion\n",
      "\n",
      "INTRODUCTION \n",
      " Cell\n",
      "\n",
      "INTRODUCTION \n",
      " hLRH-1/TIF-2\n",
      "\n",
      "INTRODUCTION \n",
      " a (Å)\n",
      "b (Å)\n",
      "c (Å)\n",
      "Space group\n",
      "Molecules per asymmetric unit\n",
      "Resolution (Å)\n",
      "Number of unique reflections\n",
      "Data redundancy\n",
      "Completeness (%)\n",
      "Rsymm (%)\n",
      "< I/s(I) >\n",
      "\n",
      "INTRODUCTION \n",
      " Refinement\n",
      "\n",
      "INTRODUCTION \n",
      " R\n",
      "Rfree\n",
      "Rms deviation from ideality\n",
      "\n",
      "INTRODUCTION \n",
      " Bond length (Å)\n",
      "Bond angle (0)\n",
      "Average B factor (A2)\n",
      "\n",
      "INTRODUCTION \n",
      " All atoms\n",
      "Protein atoms\n",
      "Water molecules\n",
      "\n",
      "INTRODUCTION \n",
      " 73.9\n",
      "73.9\n",
      "117.0\n",
      "P41212\n",
      "\n",
      "25–1.2\n",
      "100,431\n",
      "13.1\n",
      "99.9\n",
      "6.1\n",
      "39.2\n",
      "\n",
      "INTRODUCTION \n",
      " 128.4\n",
      "66.0\n",
      "141.0\n",
      "P21\n",
      "\n",
      "20–2.5\n",
      "71,698\n",
      "\n",
      "97.9\n",
      "4.5\n",
      "28.0\n",
      "\n",
      "INTRODUCTION \n",
      " 73.1\n",
      "73.1\n",
      "139.4\n",
      "P41212\n",
      "\n",
      "20–2.9\n",
      "15,614\n",
      "\n",
      "99.8\n",
      "5.8\n",
      "30.0\n",
      "\n",
      "INTRODUCTION \n",
      " 59.9\n",
      "67.2\n",
      "79.6\n",
      "P212121\n",
      "\n",
      "40–2.5\n",
      "10,787\n",
      "\n",
      "91.9\n",
      "3.7\n",
      "44.0\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " were obtained, with and without TIF-2 peptide, and\n",
      "were determined by MAD and SAD methods using sele-\n",
      "nomethionine substituted proteins, respectively (Mad-\n",
      "auss et al., 2004). The structure of hLRH-1 was det-\n",
      "ermined by molecular replacement using the atomic\n",
      "coordinates for hSF-1. The current best structures are\n",
      "refined to 1.2 Å, 2.5 Å, and 2.5 Å for mSF-1, hLRH-1, and\n",
      "hSF-1, respectively. Relevant features of data collection\n",
      "and refinement statistics for each protein crystal are\n",
      "provided in Table 1. All three LBDs exhibit a character-\n",
      "istic protein fold noted for other LBDs, with 12 α heli-\n",
      "ces, two β strands, and helix H12 positioned in an\n",
      "active conformation (Figure 1). A fourth layer is pro-\n",
      "vided by a structured helix H2, as previously shown for\n",
      "mLRH-1, although the position and length of this helix\n",
      "are the most varied features as judged by comparison\n",
      "of mSF-1 with hSF-1 LBD (Figure 1C) and mLRH-1 with\n",
      "hLRH-1 (Figure 1D). Upon refinement of these three\n",
      "LBD structures, additional electron density was present\n",
      "in the ligand binding pockets consistent with the pres-\n",
      "ence of a phospholipid; the presence of ligand con-\n",
      "trasts the reported structure of mLRH-1 showing no\n",
      "density inside or at the opening of the pocket.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Phospholipids as NR5A Receptor Ligands\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " (A) Nondenaturing mass spectrometry was carried out on all four LBD proteins used for crystallization (mSF-1, hSF-1 and mLRH-1, hLRH-1).\n",
      "Spectra are shown for mLRH-1 (left panel) and mSF-1 (right panel), with the apo [A], and the bound [B] species indicated. The molecular\n",
      "weight difference between the two peaks is also shown (arrow).\n",
      "(B) The concentrations of all bound LBD species, including the mSF-1 A270W pocket mutant, were determined as described and are listed\n",
      "for equal amounts of LBD protein (w30 µM), with the exception of hSF-1, where more protein was used due to the poorer quality of protein\n",
      "(50 µM).\n",
      "(C) Product ions following collision-induced decomposition using denaturing electrospray tandem mass spectrometry (ESI MS/MS) on m/z\n",
      "745 (blue diamond) are shown for hSF-1 and reveal a phosphatidyl glycerol (chemical structure) ligand with two acyl chains of C16 and C18.\n",
      "(D) Electron density for mSF-1 is from a 2Fo−Fc map contoured at 1.2 sigma that shows the phospholipid ligand flanked by surrounding\n",
      "residues (sticks). Three residues are labeled: Gly342, Tyr437, and Ala270, and the phosphate in the PG ligand is indicated (P).\n",
      "(E) Electron density found for both mSF-1 (blue) and hSF-1 ± TIF peptide (white or gray) ligands are superposed showing an identical position\n",
      "in the hydrophobic ligand binding pocket.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Phospholipids as NR5A Receptor Ligands\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Cell\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " In the upper left corner, a flow diagram is shown for ligand analyses of mSF-1 LBD protein following incubation with different liposomes, as\n",
      "described in Experimental Procedures. Mass spectra profiles (ESI-LC/MS) are shown for the basal state (No Liposomes, panel A), and after\n",
      "incubations with the neutral liposomes of phosphatidyl choline (PC, panel B), with phosphatidic acid liposomes (PC:PA [9:1], panel C), and\n",
      "with phosphatidyl inositol liposomes PC:PI(3,4)P2 [9:1] (PIP2, panel D) or PC:PI(3,4,5)P3 [9:1] (PIP3, panel E). In (B), (D), and (E), the small\n",
      "peaks at mass 28730 Da are PC liganded receptor (PC), and in (C), the peak at mass 28646 is PA liganded receptor (PA). Direct mass\n",
      "spectrometry analysis (without gel filtration) on the PC:PA-receptor sample revealed a strong peak of free phosphatidic acid (data not shown),\n",
      "showing that ample PA was present in the PC:PA liposomes.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " The binding of phospholipids was further explored\n",
      "by mass spectrometry analysis of purified LBD proteins\n",
      "after incubation with liposomes containing phosphati-\n",
      "dyl choline (PC), mixed with either phosphatidic acid\n",
      "(PA), PI(3,4)P2, or PI(3,4,5)P3 in a 9:1 ratio. Remarkably,\n",
      "incubation of the mSF-1 LBD with the neutral PC lipo-\n",
      "some showed an almost complete loss of the bacterial\n",
      "PG phospholipid with very little exchange or binding by\n",
      "PC; similar results were found with the PC:PA liposome\n",
      "(Figures 4B and 4C) with both profiles differing greatly\n",
      "from spectra observed with the PG bound state of the\n",
      "mSF-1 LBD (Figure 4A, No Liposomes). In contrast, effi-\n",
      "cient binding was observed for PI(3,4)P2 or PI(3,4,5)P3\n",
      "as evidenced by the increase mass of 970 and 1050 Da,\n",
      "respectively, with only a small uptake of PC observed\n",
      "(Figures 4D and 4E). A similar experiment carried out\n",
      "with mLRH-1 LBD revealed less selectivity compared\n",
      "to that of mSF-1, with equal exchange of PC, PI(3,4)P2,\n",
      "or PI(3,4,5)P3, as well as some liganded PA detected\n",
      "(see Supplemental Figure S2). These data taken to-\n",
      "gether with the PIP strip binding data suggest that SF-\n",
      "1s and hLRH-1 bind much more robustly and selec-\n",
      "tively to phosphorylated PIs than does mLRH-1. The\n",
      "structural basis for this species difference in selective\n",
      "phospholipid binding is discussed below.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " ure 5A). These orphan nuclear receptors contain a pu-\n",
      "tative LBD but no DBD; their capacity to repress SF-1,\n",
      "LRH-1, and steroid hormone receptors is mediated by\n",
      "regions within the very N- and C-terminal LBD (Holter\n",
      "et al., 2002; Ito et al., 1997; Nachtigal et al., 1998). Un-\n",
      "like NR4 orphan receptors (Nurr1, DHR38) that have\n",
      "lost their coactivator cleft (Baker et al., 2003; Wang et\n",
      "al., 2003), all four LBD structures of SF-1 and LRH-1\n",
      "exhibit a well-formed docking site for coregulators.\n",
      "Peptide binding assays were carried out\n",
      "for both\n",
      "mLRH-1 and mSF-1 LBD proteins using C-terminal flu-\n",
      "orescently tagged SHP and Dax-1 peptides and fluo-\n",
      "rescence anisotropy, as described (Moore et al., 2004).\n",
      "Selective high-affinity binding was found for both\n",
      "mLRH-1 and mSF-1, with SF-1 binding more tightly to\n",
      "peptides from Dax-1; only the N-terminal LXXLL motif\n",
      "in the Dax-1 LBD region is able to bind both mSF-1 and\n",
      "mLRH-1 (Figures 5A and 5B).\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Phospholipids as NR5A Receptor Ligands\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " (A) Schematics of SHP and Dax-1 are shown\n",
      "with colored boxes corresponding to LXXLL\n",
      "peptides used in the fluorescence anisot-\n",
      "ropy assay, as depicted and listed in the\n",
      "table.\n",
      "(B) Relative binding affinities for labeled pep-\n",
      "tides are indicated by color-coded affinities\n",
      "(Kd values) ranging from 0.5–10.0, 10.1–30.0,\n",
      "and >30.0 µM and were determined as de-\n",
      "scribed in Experimental Procedures. The\n",
      "mSHP-1 peptide used in crystallization of\n",
      "mSF-1 is RPTILYALLSPSPR (mSHP-1).\n",
      "(C) Ribbon depiction of the mSHP-1 peptide\n",
      "in the coactivator cleft of mSF-1 (orange rib-\n",
      "bon) or superposed with the mLRH-1 struc-\n",
      "ture (beige ribbon). Residues are numbered\n",
      "according to the database entry Q62227 for\n",
      "mSHP.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " LRH-1 regulate Dax-1 and SHP in a negative feedback\n",
      "regulatory mechanism (Lu et al., 2000).\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Cell\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " In the top panel, ribbon diagrams of partial structures of mLRH-1\n",
      "and mSF-1 are shown. Helices H2 (red) and H3 (pink) of mSF-1 are\n",
      "superposed with the helices H2 and H3 of mLRH-1 (light yellow),\n",
      "and the phosphatidyl glycerol ligand is shown (blue stick). The H2\n",
      "of mSF-1 is approximately half the length of helix H2 in mLRH-1\n",
      "H2. The bottom panel shows reporter activity in HepG2 cells with\n",
      "increasing concentrations of wild-type and helix H5 mutants for\n",
      "both mLRH-1 and mSF-1. Relative luciferase activities are shown.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " SF-1. Further analysis shows residues within this clus-\n",
      "ter to be crucial for phospholipid binding. Indeed, bind-\n",
      "ing of this ligand is optimized by coordination of the\n",
      "phosphate head group at the base of the pocket, with\n",
      "hydrogen bonding provided by Gly342 at the beginning\n",
      "of H6, and Tyr437 and Lys441 at the end of H11, which\n",
      "are 2.8 Å, 2.6 Å, and 2.8 Å from the ligand, respectively\n",
      "(Figure 7B). These interactions effectively seal the bot-\n",
      "tom of the ligand binding pocket with the phosphate\n",
      "group from the ligand contributing to this closure (Fig-\n",
      "ure 7C).\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " A different scenario occurs in mLRH-1, where Glu440\n",
      "forms a strong ionic interaction with the opposing\n",
      "Lys539 in helix H11 (analogous to Lys441 in mSF-1),\n",
      "sealing the pocket and stabilizing the mLRH-1 LBD in\n",
      "the absence of the ligand (Figure 7C). The other unique\n",
      "residues spanning helices H6-H7 in mLRH-1 (Thr439\n",
      "and Phe443, Figure 7A) are also predicted to minimize\n",
      "ligand binding by clashing with both the phosphate\n",
      "group and aliphatic chains, respectively. To test the hy-\n",
      "pothesis that key residues interacting with the lipid’s\n",
      "phosphate group are critical to ligand binding, mLRH-1\n",
      "was “humanized” by an E440G mutation. Wild-type and\n",
      "mutant receptors were assayed before and after reduc-\n",
      "ing cellular pools of phosphatidic acid and subsequent\n",
      "phosphatidyl\n",
      "inositols with a competitive inhibitor of\n",
      "phospholipase D (Fang et al., 2001). N-butanol treat-\n",
      "ment had a marginal effect on the activity of mLRH-1\n",
      "but substantially reduced activation by the humanized\n",
      "E440G mLRH-1, and to a lesser extent wild-type mSF-1\n",
      "(Figure 7D). Collectively, these data are consistent with\n",
      "the lowered selectively observed for mLRH-1 phospho-\n",
      "lipid binding and are predicted by structures of the\n",
      "LRH-1 and SF-1 orthologs.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Recent Divergence in the Rodent LRH-1\n",
      "Evolutionary analysis of structure-function relations in\n",
      "the LRH-1/SF-1 subfamily indicates that ligand binding\n",
      "is the ancestral state for these proteins where ligand\n",
      "binding was subsequently reduced or altered in the lin-\n",
      "eage leading to the rodent LRH-1. We coded the pres-\n",
      "ence or absence of ligand binding as a phylogenetic\n",
      "character and reconstructed its evolution on the phy-\n",
      "logeny of the NR5A gene family (Figure 8). As described\n",
      "in Experimental Procedures, both maximum parsimony\n",
      "and maximum likelihoods were used to evaluate two\n",
      "hypotheses concerning the function of the common an-\n",
      "cestor of the entire NR5A subfamily—that the ancestor\n",
      "was liganded (H1) and that it was unliganded (H0). Un-\n",
      "der H1, a single loss of ligand binding is required on the\n",
      "branch leading to the rodent LRH-1, whereas H0 re-\n",
      "quires ligand binding to have been gained indepen-\n",
      "dently in the SF-1 and human LRH-1 lineages. H1 is\n",
      "therefore the most parsimonious hypothesis. A maxi-\n",
      "mum likelihood analysis assuming that ligand binding\n",
      "can be gained or lost under a simple reversible Markov\n",
      "process also supports H1, albeit nonsignificantly, indi-\n",
      "cating a loss of ligand binding in rodent LRH-1s (likeli-\n",
      "hood ratio = 3.05).\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Phospholipids as NR5A Receptor Ligands\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " (A) Sequence alignment of the region span-\n",
      "ning helices H6 and H7 is shown for a limited\n",
      "number of species. Rodent LRH-1-specific\n",
      "changes are highlighted in red.\n",
      "(B) Electron density showing coordination of\n",
      "the phosphate group by three residues in\n",
      "mSF-1. These include Gly342 (H7) and Tyr437\n",
      "and Lys441 (H11) with hydrogen bonds indi-\n",
      "cated (dashed line). The phosphate is la-\n",
      "beled (P).\n",
      "(C) Interactions between helices H7 and H11\n",
      "are depicted for mSF-1, with phospholipid li-\n",
      "gand (white stick) and for mLRH-1 without\n",
      "ligand. The salt bridge between Glu440 and\n",
      "Lys539 in mLRH-1 is depicted.\n",
      "(D) Reporter activity is shown after depletion\n",
      "of phosphatidic acid by butanol with wild-type\n",
      "mLRH-1, mSF-1, and “humanized” mLRH-1\n",
      "E440G mutant transfected into HepG2 cells.\n",
      "Activity is expressed as fold activation using\n",
      "the aromatase luciferase reporter.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " are known for only four family members, sequences are\n",
      "available for dozens. We used maximum likelihood\n",
      "(Yang et al., 1995) to determine ancestral amino acid\n",
      "states and evolutionary changes in the H6-H7 residues\n",
      "corresponding to Thr439, Glu440, and Phe443 in rodent\n",
      "LRH-1 (TEF, Figure 8). These sites were chosen be-\n",
      "cause, as discussed above, structural and experimen-\n",
      "tal evidence indicates that they exclude or diminish\n",
      "phospholipid binding in mLRH-1, but the homologous\n",
      "residues, Ala, Gly, and Leu, confer ligand binding on\n",
      "hLRH-1 and the SF-1s (AGL, Figure 8). The NR5A an-\n",
      "cestral sequence has all three ligand-associated states\n",
      "(AGL), and this reconstruction has extremely high confi-\n",
      "dence (posterior probability 100% for each site). The\n",
      "derived TEF residues that close the pocket in the ab-\n",
      "sence of ligand all evolved much later on than the\n",
      "branch leading to the rodent LRH-1 (posterior prob-\n",
      "ability 100% for each site). Examination of the genetic\n",
      "code indicates that each of these replacements can be\n",
      "conferred by a single base-pair mutation.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " acid replacements. The lost or altered ligand binding\n",
      "state in rodent LRH-1 is the second example of a taxon-\n",
      "specific nuclear receptor losing the capacity to bind li-\n",
      "gand from a liganded ancestor; the other is the estro-\n",
      "gen receptor ortholog of the mollusc Aplysia californica\n",
      "(Thornton et al., 2003). These findings cast doubt on\n",
      "the hypothesis that ligand binding in the nuclear recep-\n",
      "tor superfamily was independently gained numerous\n",
      "times from an ancestral orphan receptor (Escriva et al.,\n",
      "2000). If the loss of ligand dependence is common and\n",
      "requires only a few sequence changes, the opposite\n",
      "scenario—repeated loss of ligand binding from a li-\n",
      "ganded ancestor—may better characterize NR evo-\n",
      "lution.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Cell\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " The maximum posterior probability phylogeny of the NR5 gene\n",
      "family is shown, including LRH-1s (red), SF-1s (blue), and outgroup\n",
      "sequences (black, with number of sequences analyzed shown in\n",
      "parentheses). Residues homologous to mLRH-1 positions 439, 440,\n",
      "and 443 are shown for each LRH-1/SF-1 sequence (aa). Maximum\n",
      "likelihood reconstructions of ancestral amino acid states are shown\n",
      "for SF-1 and LRH-1 proteins in the last common ancestor of verte-\n",
      "brates (squares), of tetrapods (triangles), and of mammals (circles),\n",
      "as well as for the common ancestral gene preceding the duplica-\n",
      "tion that split the SF-1 and LRH-1 lineages (star). All ancestral re-\n",
      "constructions have 100% posterior probability. Green arrowhead\n",
      "shows acquisition of amino acids that exclude ligand in the rodent\n",
      "LRH-1 lineage and loss of ligand binding on the same branch. Spe-\n",
      "cies names and accession numbers are provided in Supplemental\n",
      "Table S2.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " implies that a distinct pool of cellular lipids signal via\n",
      "modulation of the NR5A nuclear receptors. Indeed, the\n",
      "literature suggests that nuclear phosphatidyl\n",
      "inositol\n",
      "lipids exist as well as enzymes affecting their metabo-\n",
      "lism, such as phosphatidyl inositol 3-kinase (PI-3K) (Ir-\n",
      "vine, 2002; Ledeen and Wu, 2004; Martelli et al., 2003).\n",
      "Additionally, both PI-3 and PI-5 phosphatases exhibit\n",
      "prominent nuclear staining (Bacqueville et al., 2001; De-\n",
      "leris et al., 2003), thus providing a mechanism to down-\n",
      "regulate PI nuclear signaling. Although we found that\n",
      "Wortmannin (PI-3 kinase inhibitor) and the PI-3 phos-\n",
      "phatase PTEN affected the activities of wild-type and\n",
      "pocket mutant SF-1 receptors differently (I.N.K., unpub-\n",
      "lished data), further studies and additional pharmaco-\n",
      "logical tools are needed to selectively disrupt nuclear\n",
      "phospholipid signaling pathways. Nonetheless, the link\n",
      "between PI-3 kinase signaling and NR5A receptor ac-\n",
      "tivity is intriguing given the roles that both SF-1 and\n",
      "LRH-1 play in cell proliferation. Indeed, haploinsuffi-\n",
      "ciency of both sf-1 and lrh-1 leads to decreased cell\n",
      "proliferation (Bland et al., 2000; Botrugno et al., 2004),\n",
      "while increased gene dosage of SF-1 is reported to be\n",
      "associated with human adrenal cortical tumors (de Fi-\n",
      "gueiredo et al., 2004). Because PI-3 kinase pathways\n",
      "are known to regulate cell cycle progression and cell\n",
      "growth, it is tempting to speculate that extracellular\n",
      "events that trigger this pathway might also elevate\n",
      "nuclear PIs, and thus modulate SF-1/LRH-1 activities.\n",
      "Finally, it is worth considering how phospholipids are\n",
      "delivered to nuclear receptors. Cytoplasmic shuttling of\n",
      "PIs to the plasma membrane is carried out by phospha-\n",
      "tidyl inositol transport proteins (PITP) in an ATP-inde-\n",
      "pendent manner; these proteins are also present in the\n",
      "nucleus. Crystal structures of phospholipid binding\n",
      "proteins show a large hydrophobic pocket or tunnel\n",
      "able to bind lipids (Yoder et al., 2001; Hsuan and Cock-\n",
      "croft, 2001; Soccio and Breslow, 2003). Whether PIs are\n",
      "directly taken up by SF-1/LRH-1 from membranes (due\n",
      "to transient receptor shuttling between the cytoplasm\n",
      "and nucleus) or are delivered by nuclear phospholipid\n",
      "binding proteins remains unclear. While many ques-\n",
      "tions are still to be answered, our current study leads\n",
      "us to hypothesize phosphatidyl inositols as bona fide\n",
      "ligands for NR5A receptors and, as such, provide a di-\n",
      "rect link between transcription and phospholipid sig-\n",
      "naling.\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Experimental Procedures\n",
      "\n",
      "RESULTS AND DISCUSSION \n",
      " Phospholipids as NR5A Receptor Ligands\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " diffracted to 1.1–1.2 Å. Data were integrated using DENZO and\n",
      "scaled with SCALEPACK. For hLRH-1, the protein and TIF-2 pep-\n",
      "tide were mixed in equal volumes with 1.2 M sodium potassium\n",
      "phosphate (pH 4.9), 4% (v/v) glycerol, 4% (v/v) ethylene glycol.\n",
      "Crystals were grown at 22°C and appeared in one week and\n",
      "reached 100–200 m in each dimension over a period of 3 weeks.\n",
      "Prior to data collection, hLRH-1 crystals were dipped in cryopro-\n",
      "tectant (reservoir solution containing 11.5% glycerol and 11.5%\n",
      "ethylene glycol) and flash-frozen in liquid nitrogen. The TIF-2 pep-\n",
      "tide sequence is KENALLRYLLDKDD and was used for both hLRH-1\n",
      "and hSF-1 crystallization. X-ray data were collected at IMCA 17-ID\n",
      "line (Argonne, Illinois), and data were processed and scaled with\n",
      "HKL2000. The hLRH-1 model was built by molecular replacement\n",
      "with Amore program using hSF-1 as a search model and refined\n",
      "with CNS. In this model, a second TIF-2 peptide is found in the\n",
      "asymmetric unit and forms loose contacts with the tail of the first\n",
      "TIF-2 peptide positioned in the coactivator cleft formed by H3, H4,\n",
      "and H12 of the hLRH-1 receptor. Rebuilding between rounds of\n",
      "refinement was performed with the graphics program QUANTA.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " Cell Culture and Transfections\n",
      "Human HepG2 liver carcinoma cells were plated the day of trans-\n",
      "fection in 24 well plates and transfected with FUGENE 6 with\n",
      "amounts of DNA indicated in figures and legends. For all experi-\n",
      "ments, 200 ng/well of luciferase reporter were used and pCI-Neo-\n",
      "β-gal (50 ng/well) were transfected along with wild-type or mutant\n",
      "receptors. Luciferase activity was measured after 48 hr, as pre-\n",
      "viously described (Sablin et al., 2003). Expression of all mutants\n",
      "was tested by Western blot analyses using anti-HA antibody (Co-\n",
      "vance) directed against an N-terminal tag. In all cases, mutant re-\n",
      "ceptors were expressed at equivalent or higher levels than wild-\n",
      "type protein. For N-butanol experiments, 0.1% 1-butanol was\n",
      "added for 2 hr. All transfection experiments were done in triplicates\n",
      "and repeated at least three times; representative experiments are\n",
      "shown with standard errors indicated by bars.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " Ligand Analyses and Phospholipid Binding Assays\n",
      "Protein samples were desalted and injected onto a column packed\n",
      "with P6 gel (BioRad) to exchange nonvolatile buffers with 20 mM\n",
      "ammonium acetate (pH 6.8). Buffer-exchanged protein was col-\n",
      "lected and back flushed at a flow rate of 0.75–1 µl/min into a na-\n",
      "noflow electrospray ionization source fitted on a Waters/Micromass\n",
      "QTOF-1 (Beverly, Massachusetts) mass spectrometer. Data were col-\n",
      "lected in positive ion mode over a mass range of 1000–4000 m/z. For\n",
      "structural determination of ligands, a similar in-gel filtration appara-\n",
      "tus was connected to an orthogonal acceleration time-of-flight\n",
      "mass spectrometer (LCMSD-TOF, Agilent) and a quadrupole ion\n",
      "trap mass spectrometer (LCMSD Trap SL, Agilent). Samples were\n",
      "examined utilizing negative ion electrospray using conditions that\n",
      "denatured the protein-ligand complex.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " these conditions, PI and PA lipids should mix completely into sta-\n",
      "ble bilayers and consist primarily of PC. LBD proteins (2 mg/ml)\n",
      "were exchanged into liposome buffer and combined with the lipo-\n",
      "somes to produce a 25:1 molar ratio of PC to protein and a 2.5:1\n",
      "molar ratio of PA, PI(3,4)P2, or PI(3,4,5)P3 to protein. Mixtures were\n",
      "incubated for 20 hr at room temperature, and protein was sepa-\n",
      "rated from liposomes by gel-filtration on Superdex 75 (Pharmacia),\n",
      "concentrated, subjected to another round of gel filtration, and ana-\n",
      "lyzed by mass spectrometry as described above. The amount of\n",
      "SF-1 LBD protein associated with free liposomes (void volume) fol-\n",
      "liposome\n",
      "lowing gel filtration was determined to be <1% for all\n",
      "species.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " Peptide Binding Assays\n",
      "LBD proteins were serially diluted from 150 µM to 0.002 µM in bind-\n",
      "ing buffer (50 mM NaPO4, 150 mM NaCl, pH 7.2, 1 mM DTT, 1 mM\n",
      "EDTA, 0.01% NP40, 10% glycerol). Diluted protein was added to\n",
      "fluorescent coregulator peptide (20 nM) in 384 well plates yielding\n",
      "final protein concentrations of 75–0.001 µM and 10 nM of fluores-\n",
      "cent peptide. Samples were analyzed as previously described\n",
      "(Moore et al., 2004). Construction of 20 aa coregulator peptides\n",
      "with the general motif of CXXXXXXXLXXL/AL/AXXXXXXX were as\n",
      "follows: peptides were synthesized in parallel using standard fluor-\n",
      "enylmethoxycarbonyl (Fmoc) chemistry in 48 well synthesis blocks\n",
      "(FlexChem System, Robbins) (Wellings and Atherton, 1997). Cou-\n",
      "pling efficiency was monitored and the completed peptides were\n",
      "cleaved from the resin, followed by reversed-phase chromatogra-\n",
      "phy and mass spectrometry (MALDI-TOF/ESI) to purify the pep-\n",
      "tides, and are described in Moore et al. (2004).\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " Evolutionary Analysis\n",
      "Protein sequences of 48 SF-1/LRH-1/FtzF1 family members with\n",
      "full-length ligand binding domains were obtained from GenBank by\n",
      "BLAST search using Homo sapiens SF-1 as a query. Twenty addi-\n",
      "tional sequences from the closely related ERR, GCNF/GRF, and\n",
      "RXR groups (Thornton and DeSalle, 2000) were also obtained to\n",
      "serve as outgroup sequences. (See Supplemental Table S2 for ac-\n",
      "cession numbers and species names). These sequences were\n",
      "aligned using ClustalX 1.83 software (Thompson et al., 1994) as-\n",
      "suming a gap:change cost ratio of 10. The phylogeny of the gene\n",
      "family was then inferred from the aligned sequences using the\n",
      "Bayesian Markov Chain Monte Carlo method (Ronquist and\n",
      "Huelsenbeck, 2003) implemented in MrBayes v. 3.0 software. Tree\n",
      "and parameter space were explored with four simultaneous chains\n",
      "(three heated) for 300,000 generations each, using the Jones model\n",
      "of protein evolution with a four-category discrete γ model of\n",
      "among-site rate variation. Priors were equal for tree topologies,\n",
      "uniform (0,10) for branch lengths, and uniform (0.1, 5) for the α\n",
      "shape parameter of the γ distribution. The initial 100,000 trees, a\n",
      "point well past stationarity, were discarded as burn-in.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " Phylogenetic reconstruction of the ancestral states at internal\n",
      "nodes for ligand binding or ligand independence, coded as a binary\n",
      "character, were conducted by parsimony and maximum likelihood.\n",
      "For the former, we used Fitch’s parsimony algorithm (Fitch, 1971)\n",
      "on a simple tree consisting of the four taxa for which ligand binding\n",
      "states are known (Sablin et al., 2003; this paper). For maximum\n",
      "likelihood reconstruction, we calculated the likelihood of each an-\n",
      "cestral state 0 (no ligand binding) and 1 (ligand binding) as the\n",
      "probability of evolving the terminal states in mLRH-1, hLRH-1,\n",
      "mSF-1, and hSF-1, given the phylogeny that relates these genes,\n",
      "and a simple reversible Markov process for transitions between\n",
      "states 0 and 1 (Pagel, 1999). The probability Pij of observing a state\n",
      "change on a tree branch of length k is Pij =\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " e−k.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " Branch lengths were assumed equal a priori and scaled to their\n",
      "maximum likelihood values by hill climbing; unknown states at in-\n",
      "ternal nodes other than the NR5A ancestor were treated as nui-\n",
      "sance parameters and optimized by maximum likelihood.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " assuming a Jones model of amino acid evolution and a four-cate-\n",
      "gory γ model of rate variation among sites (α optimized by ML). The\n",
      "tree for ancestral state reconstruction was the maximum posterior\n",
      "probability tree from the Bayesian analysis, adjusted at three nodes\n",
      "that had posterior probability <0.95 to make them consistent with\n",
      "known taxonomic relationships among species (see Supplemental\n",
      "Figure S3). Branch lengths for the ancestral reconstruction were\n",
      "optimized by maximum likelihood using PAML software.\n",
      "\n",
      "EXPERIMENTAL PROCEDURES \n",
      " Supplemental Data\n",
      "Supplemental Data include three figures and two tables and can\n",
      "be found with this article online at http://www.cell.com/cgi/\n",
      "content/full/120/3/343/DC1/.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "[print(tag, '\\n', txt, end='\\n\\n') for txt, tag in zip(pdf_sections, pdf_tags)];"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pdf_text = extract_text(os.path.join(config.path_to_data, 'pdf', '002dea70291555c99aa2d88b87db7e8d3ccad66a.pdf'))\n",
    "\n",
    "def process_pdf(pdf_text):\n",
    "    \n",
    "    # Find sections. We’ll guess some common next headings for each section:\n",
    "    abstract_text = find_section(pdf_text, \"abstract\", \n",
    "                                 [\"\\n\\n\"], max_chars=3000)\n",
    "    introduction_text = find_section(pdf_text, \"introduction\", \n",
    "                                     [\"methods\", \"methodology\", \"materials and methods\", \"results\", \"discussion\"], max_chars=5000)\n",
    "    discussion_text = find_section(pdf_text, \"discussion\", \n",
    "                                   [\"conclusion\", \"references\", \"acknowledgements\"], max_chars=5000)\n",
    "    \n",
    "    all_text = find_section(pdf_text, \" \", \n",
    "                                   [\"conclusion\", \"references\", \"acknowledgements\"], max_chars=5000)\n",
    "    \n",
    "    return abstract_text, introduction_text, discussion_text, all_text\n",
    "\n",
    "abstract_text, introduction_text, discussion_text, all_text = process_pdf(pdf_text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 620,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "L. Poirel,M. Lartigue,J. Decousser,P. Nordmann\n",
      "0016746ad49041402c38da533049ebe0fe9539d1\n",
      "Several expanded-spectrum -lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enter-\n",
      "obacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional\n",
      "mechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.\n"
     ]
    }
   ],
   "source": [
    "idx = 19\n",
    "print(metadata['authorName'].iloc[idx])\n",
    "sections, text_blocks, tag_blocks = extract_pdf_sections(os.path.join(config.path_to_data, 'pdf', metadata['paperId'].iloc[idx] + '.pdf'), metadata['authorName'].iloc[idx])\n",
    "print(metadata['paperId'].iloc[idx])\n",
    "# [print(tag + '\\n', txt, end='\\n\\n') for txt, tag in zip(text_blocks, tag_blocks)];\n",
    "\n",
    "print(sections['abstract'])\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 594,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Abstract Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which\n",
      "are unknown.We investigated the involvement of fibroblast growth factor (FGF) signals in synovial\n",
      "sarcoma and evaluated the therapeutic effect of inhibiting the FGF signal.\n",
      "Experimental Design:The expression of 22 FGF and 4 FGF receptor (FGFR) genes in18 prima-\n",
      "ry tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR.\n",
      "Effects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein\n",
      "kinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory\n",
      "effects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo.\n",
      "Results: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed\n",
      "in neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma\n",
      "cell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal ^\n",
      "regulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the\n",
      "FGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle ar-\n",
      "rest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the\n",
      "FGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not\n",
      "p38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sar-\n",
      "coma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2.\n",
      "Conclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibi-\n",
      "tory molecules will be of potential use for molecular target therapy in synovial sarcoma.\n",
      "Synovial sarcoma is the most frequent soft-tissue sarcoma\n",
      "(STS) among patients in the third to fourth decade of life (1)\n",
      "and accounts for about 7% to 10% of all human STSs (2). It\n",
      "predominantly affects the lower extremities but can occur in\n",
      "any part of the body. Surgical resection with an adequate\n",
      "for\n",
      "surgical margin is the definitive choice of\n",
      "primary tumors and has been shown to control\n",
      "local\n",
      "recurrence (3, 4). Disseminated distant metastasis is the major\n",
      "cause of poor outcome, and several reports describing the\n",
      "results of current therapy showed a 5-year survival rate of\n",
      "around 50% to 60% (3, 4). The efficacy of adjuvant\n",
      "chemotherapy is still a matter of debate, and the development\n",
      "of a new therapeutic approach is required to improve the\n",
      "prognosis.\n",
      "the reciprocal\n",
      "Despite little progress in clinical treatment during the last\n",
      "20 years, cytogenetic and molecular genetic analyses have greatly\n",
      "improved the understanding of this type of tumor, especially\n",
      "with the discovery of\n",
      "translocation\n",
      "t(18;X)(q11;p11) creating an SYT-SSX fusion gene as a synovial\n",
      "sarcoma – specific genetic alteration (5, 6). Thus far, three SSX\n",
      "genes (SSX1, SSX2, and SSX4) have been identified as a partner\n",
      "of the SYT gene, and >95% of synovial sarcoma tumors carried\n",
      "one of these fusion genes (7). Although the precise function and\n",
      "the mechanism of oncogenesis are not yet clearly shown, the\n",
      "high sensitivity and specificity of the SYT-SSX fusion gene in\n",
      "synovial sarcoma have proven to be useful for molecular\n",
      "diagnosis (8). In particular types of tumors with a specific\n",
      "reciprocal translocation such as PML-RARa in acute promyelo-\n",
      "cytic leukemia (9) and BCR-ABL in chronic myelogenous\n",
      "leukemia (10), fusion gene products themselves serve as targets\n",
      "for the therapy. Although immunotherapy using a peptide\n",
      "derived from SYT-SSX protein as a specific vaccine has been\n",
      "investigated (11), no therapeutic approach has been discovered\n",
      "for directly targeting the fusion protein or its function.\n",
      "Gene expression profiling of tumors has been shown a\n",
      "powerful tool with which to isolate a molecular target for\n",
      "therapy (12). We have done a gene expression analysis of\n",
      "synovial sarcoma using a genome-wide cDNA microarray and\n",
      "found that synovial sarcoma shared its molecular signature with\n",
      "malignant peripheral nerve sheath tumor, of which the\n",
      "precursors were neural crest – derived cells, and also identified\n",
      "a set of genes commonly up-regulated in synovial sarcoma\n",
      "including the fibroblast growth factor 18 (FGF18) gene (13). The\n",
      "FGF signaling pathway seems to play significant roles in tumor\n",
      "development and progression (14 – 16), and recently FGF18 was\n",
      "identified as an autocrine growth factor involved in colon\n",
      "cancers (17). Thus far, 22 genes have been identified as members\n",
      "of the FGF family, and our cDNA microarray contained 10 FGF\n",
      "genes (FGF2, FGF3, FGF4, FGF7, FGF9, FGF11, FGF12, FGF13,\n",
      "FGF18, and FGF19). We found that some FGF genes other\n",
      "than FGF18 were also expressed in synovial sarcoma (data\n",
      "shown). Based on these results, we have done an\n",
      "not\n",
      "intensive analysis of FGF and its receptor (FGFR) genes in\n",
      "synovial sarcoma and also investigated whether inhibition of\n",
      "the FGF signal is a new therapeutic modality for synovial\n",
      "sarcoma.\n",
      "Materials and Methods\n",
      "Tumor tissues were obtained at either\n",
      "biopsy or resection surgery and kept at 80jC. Informed consent was\n",
      "obtained from each patient, and tumor samples were approved for\n",
      "analysis by the Ethics Committee of the Faculty of Medicine, Kyoto\n",
      "University. Five human synovial sarcoma cell lines (YaFuSS, HS-SY-II,\n",
      "SYO-1, Fuji, and 1273/99) were used in this study. YaFuSS and HS-SY-II\n",
      "cells have the SYT-SSX1 fusion gene and the others have the SYT-SSX2\n",
      "fusion gene (data not shown). YaFuSS was established in our laboratory\n",
      "from a monophasic synovial sarcoma in a 28-year-old male. HS-SY-II\n",
      "was a gift from H. Sonobe (Kochi University, Japan; ref. 18), SYO-1\n",
      "from A. Kawai (Okayama University, Japan; ref. 19), Fuji from S.\n",
      "Tanaka (Hokkaido University, Japan; ref. 20), and 1273/99 from\n",
      "O. Larsson (Karolinska Institute, Sweden). Among control cell lines,\n",
      "NMS-2 (malignant peripheral nerve sheath tumor; ref. 21) was\n",
      "provided by A. Ogose (Niigata University,\n",
      "Japan), and Saos2\n",
      "(osteosarcoma), HT1080 (fibrosarcoma), COLO205 (colon cancer),\n",
      "and SW480 (colon cancer) cells were purchased from American Type\n",
      "Culture Collection (Manassas, VA). Cells were maintained in RPMI\n",
      "1640 (Sigma-Aldrich, St. Louis, MI) for Fuji, SW480, and COLO205\n",
      "with 10% fetal bovine serum (FBS, HyClone, Road Logan, UT) or\n",
      "DMEM (Sigma, St. Louis, MO) for other cells with 10% FBS. OPTI-\n",
      "MEM I, which contains insulin and transferrin as protein supplements\n",
      "(Invitrogen, Carlsberg, CA), was used in serum-free cultures.\n",
      "The FGFR-specific tyrosine kinase\n",
      "inhibitor SU5402 was purchased from Calbiochem (La Jolla, CA;\n",
      "ref. 22). PD166866 was provided by Pfizer Global Research and\n",
      "Development (Groton, CT; ref. 23). The MAPK/extracellular signal –\n",
      "regulated kinase (ERK) kinase (MEK) inhibitor U0126 was purchased\n",
      "from Promega (Madison, WI; ref. 24). Recombinant human FGF18\n",
      "(rhFGF18) was purchased from Wako Pure Chemical Industry (Osaka,\n",
      "Japan),\n",
      "rhFGF8 from PeproTech (London, United Kingdom),\n",
      "and rhFGF2 from Oncogene Research Products (San Diego, CA).\n",
      "Anti-ERK1 (M12320), anti-phosphorylated-p38 (P39520), anti-pan-\n",
      "p38 (P19820), anti-phosphorylated-JNK1/SAPK1 (S37220), and anti-\n",
      "pan-JNK1/SAPK1 (M54920) antibodies were purchased from\n",
      "BD Biosciences PharMingen (San Diego, CA); anti-phosphorylated-\n",
      "ERK1/2 (sc-7383), anti-FGFR3 (sc-123), and peroxidase-conjugated\n",
      "anti-mouse IgG (sc-2005) from Santa Cruz Biotechnology (Santa\n",
      "Cruz, CA); anti-phosphotyrosine (PY20) from Zymed Laboratories\n",
      "(South San Francisco, CA); anti-FGF18 (MAB667) from R&D Systems\n",
      "(Minneapolis, MN); peroxidase-conjugated anti-rabbitimmunoglobu-\n",
      "lin (P0399) from DakoCytomation (Glostrup, Denmark).\n",
      "Total RNA was extracted using TRIzol\n",
      "reagent (Invitrogen) following the manufacturer’s instructions. After\n",
      "treatment with DNase I (Nippon Gene, Osaka, Japan), 1 Ag of total\n",
      "RNA was reverse transcribed for single-stranded cDNAs using oligo(dT)\n",
      "primer and Superscript II reverse transcriptase (Invitrogen). PCR was\n",
      "done using 1 AL of RT product in a final volume of 25 AL containing\n",
      "20 pmol each of the sense and antisense primers, 2.5 mmol/L MgCl2,\n",
      "0.2 mmol/L of each deoxynucleotide triphosphate, and 1 unit of rTaq\n",
      "polymerase (TOYOBO, Osaka, Japan). All PCRs were done using\n",
      "GeneAmp 9700 (PE Applied Biosystems, Foster City, CA). Information\n",
      "of the primers are available upon request.\n",
      "Quantitative reverse transcription-PCR (RT-PCR) analyses were done\n",
      "in selected FGF and FGFR genes with ABI PRISM 7700 Sequence\n",
      "Detection System (PE Applied Biosystems). One microliter of RT\n",
      "product in a final volume of 25 AL containing 12.5 AL of 2 SYBR\n",
      "Green Mastermix (PE Applied Biosystems). Information for primers are\n",
      "available upon requests. The mean of triplicated data was used to\n",
      "calculate the ratio of target gene/b-actin expression. Statistical analysis\n",
      "was done by t test after logarithmic transformation.\n",
      "To detect FGF18 protein in the supernatant,\n",
      "cells were cultured up to 80% confluency in 100-mm dish. After\n",
      "washing the dish, cells were further incubated in 5 mL OPTI-MEM I for\n",
      "4 days. The supernatant was harvested, mixed with 8 mL ice-cold\n",
      "acetone, and kept at 80jC for 1 hour. The mixture was centrifuged at\n",
      "10,000  g for 15 minutes, and the precipitate was suspended by lysis\n",
      "buffer (100 AL) containing aprotinin (1 Ag/mL), leupeptin (1 Ag/mL),\n",
      "pepstatin A (1 Ag/mL), and phenylmethylsulfonyl fluoride (1 mmol/L)\n",
      "followed by sonication and centrifugation. Twenty microliters of the\n",
      "supernatant were electrophoresed on a 10% SDS-polyacrylamide gel\n",
      "and transferred onto a polyvinylidene difluoride membrane (Millipore,\n",
      "Bedford, MA). After blocking with 3% skim milk, membranes were\n",
      "probed with anti-FGF18 antibody at 1:1,000 dilutions for 1 hour and\n",
      "with peroxidase-conjugated anti-mouse IgG 1:2,000 for 1 hour.\n",
      "Immunoreactive bands were detected with Enhanced Chemilumines-\n",
      "cence Plus (Amersham Biosciences, Little Chalfont Buckinghamshire,\n",
      "United Kingdom).\n",
      "To detect FGFR3 protein in cell lysate, cells were harvested by the\n",
      "same lysis buffer mentioned above. The lysate (40 Ag) was electro-\n",
      "phoresed using 8% SDS-polyacrylamide gel, blotted, and blocked by\n",
      "5% skim milk. The membrane was probed with anti-FGFR3 antibody at\n",
      "1:500 dilutions and with peroxidase-conjugated anti-rabbit immuno-\n",
      "globulin at 1:2,000.\n",
      "To evaluate the phosphorylation status\n",
      "of FGFR3, serum-starved cells were treated by 20 Amol/L SU5402 for\n",
      "30 minutes followed by the treatment with rhFGF18 or vehicle. The\n",
      "cells were harvested by 1 mL modified radioimmunoprecipitation assay\n",
      "buffer containing protease inhibitors and sodium orthovanadate,\n",
      "sonicated, and centrifuged at 15,000  g for 10 minutes. After\n",
      "preclearing by 100 AL protein A agarose (Upstate, Lake Placid, NY),\n",
      "1 mg of the protein was added with 4 Ag anti-FGFR3 antibody and\n",
      "rotated for 2 hours at 4jC, and the immunocomplex was mixed with\n",
      "100 AL protein A agarose and rotated for another 2 hours. The mixture\n",
      "was washed twice by PBS, and the precipitate was boiled for 5 minutes\n",
      "with 60 AL of 2 sample buffer, centrifuged, and the supernatant was\n",
      "used for Western blotting using anti-FGFR3 antibody (1:500) or anti-\n",
      "phosphotyrosine (1:500).\n",
      "To evaluate the phosphorylation of MAPKs, cells (4  105) were\n",
      "seeded on 60-mm dishes and incubated overnight with medium\n",
      "containing 10% FBS followed by serum starvation for 24 hours. After\n",
      "treatment with each reagent for 15 minutes in the serum-free medium,\n",
      "cells were lysed in a buffer (100 AL) containing sodium orthovanadate\n",
      "(5 mmol/L) and protease inhibitors, sonicated, and centrifuged.\n",
      "Proteins (15 Ag) were used for Western blotting.\n",
      "Cell cycle analysis. Cells (1  105) were incubated overnight in\n",
      "60-mm dishes\n",
      "followed by serum\n",
      "starvation for 24 hours. After treatment with SU5402 (20 Amol/L)\n",
      "for 30 minutes, FBS (1%) was added to the medium and cells were\n",
      "further incubated for 48 hours. Cells were then harvested, washed\n",
      "once with PBS, and fixed in 70% ethanol overnight. After an\n",
      "incubation in PBS containing propidium iodide (10 Ag/mL) and\n",
      "RNaseA (10 Ag/mL) for 30 minutes at 37jC, the DNA content of\n",
      "these cells was analyzed using FACScan and CELL Quest Software\n",
      "In vivo growth assay. All animal studies were approved by the\n",
      "Animal Research Committee, Graduate School of Medicine, Kyoto\n",
      "University and done according to the Guideline for Animal Experi-\n",
      "ments of Kyoto University. First,\n",
      "the effect of PD166866 on\n",
      "phosphorylated ERK1/2 in vivo was investigated. SYO-1 (5  106)\n",
      "cells suspended in 100 AL of PBS were s.c. injected into the hind\n",
      "flank region of male BALB/c nu/nu athymic mice at 5 weeks of age\n",
      "(Japan SLC, Hamamatsu,\n",
      "Japan). PD166866 (0.1 or 0.5 mg)\n",
      "suspended in 50 AL of DMSO was given i.p. when the tumor\n",
      "volume was f1,500 mm3. Tumors were dissected 30 minutes after\n",
      "the injection, and tissue blocks (5  5  5 mm) were prepared from\n",
      "the central and peripheral portion of tumors. Whole cell lysates were\n",
      "prepared from each tissue block and used for Western blot. Growth\n",
      "inhibition experiments were done using SYO-1 and HT1080. Cells\n",
      "(5  106) were inoculated as described above. When the tumor\n",
      "volume reached about 50 mm3 (usually 11-12 days after inocula-\n",
      "tion), PD166866 was given i.p., a treatment which was repeated\n",
      "thereafter once a day, 6 days a week, for 2 weeks. Tumor size was\n",
      "measured with vernier calipers, and the volume was calculated as k/\n",
      "6  length  width  height.\n",
      "Immunohistochemical\n",
      "staining of proliferating cell nuclear antigen (PCNA) was done using\n",
      "EPOS anti-PCNA/horseradish peroxidase system (DakoCytomation).\n",
      "On day 13 (the day of the last administration), tumor tissues were\n",
      "dissected, fixed in 10% formalin, and embedded in paraffin. Sections\n",
      "(5-Am-thick) were probed with anti-PCNA antibody for 1 hour and\n",
      "counterstained with hematoxylin. To quantify the PCNA-positive area,\n",
      "Results\n",
      "Synovial sarcoma cell lines expressed multiple fibroblast growth\n",
      "factor and fibroblast growth factor receptor genes. The mRNA\n",
      "expression of 22 subtypes of FGF genes was analyzed by RT-\n",
      "PCR in five synovial sarcoma and five other cell lines (Fig. 1A).\n",
      "five synovial sarcoma cell\n",
      "None of the five synovial sarcoma cell lines expressed the\n",
      "FGF5, FGF6, FGF12, FGF14, FGF16, FGF17, FGF22, or FGF23\n",
      "gene, whereas the FGF2, FGF8, FGF9, FGF11, and FGF18 genes\n",
      "were expressed in all\n",
      "lines.\n",
      "Expression of the remaining nine FGF genes varied among\n",
      "cell\n",
      "lines. The expression profile in NMS-2 (malignant\n",
      "peripheral nerve sheath tumor) was similar to that in synovial\n",
      "sarcoma, which may relate to the overall similarity of the gene\n",
      "expression profile between synovial sarcoma and malignant\n",
      "peripheral nerve sheath tumor (13). HT1080 (fibrosarcoma)\n",
      "expressed a similar set of FGF genes to malignant peripheral\n",
      "nerve sheath tumor but at much lower levels, and the\n",
      "expression profile of Saos2 (osteosarcoma) was quite different\n",
      "being positive only for FGF1 and FGF13. Interestingly, synovial\n",
      "sarcoma shared with colon carcinoma cell lines the expression\n",
      "of some FGF genes such as FGF3 and FGF18 genes, which were\n",
      "not expressed in mesenchymal cell lines. Secretion of these FGF\n",
      "proteins by synovial sarcoma cell lines was confirmed by the\n",
      "Western blotting of culture supernatant. FGF18 proteins were\n",
      "clearly detected in culture supernatant of synovial sarcoma cell\n",
      "lines, although the level of protein was not completely agreed\n",
      "with the level of RNA expression (Fig. 1C).\n",
      "In contrast with the heterogeneous expression pattern of\n",
      "the expression profile of receptors (FGFR) was\n",
      "ligands,\n",
      "relatively homogeneous among synovial sarcoma cell lines,\n",
      "being positive for all subtypes analyzed except FGFR2b, which\n",
      "is an epithelia-specific FGFR expressed in colon carcinomas\n",
      "(Fig. 1B). In addition to FGFR2b, NMS-2 was negative for\n",
      "FGFR3, and HT1080 was negative for FGFR2c. Expression of\n",
      "FGFR3 in synovial sarcoma cell lines was confirmed at the\n",
      "protein level by the Western blotting using cell lysates of each\n",
      "cell line (Fig. 1D).\n",
      "Expression of fibroblast growth factor and fibroblast growth\n",
      "factor receptor genes in tumor tissues.\n",
      "The FGF2, FGF8,\n",
      "FGF9, FGF11, and FGF18 genes, which were commonly ex-\n",
      "pressed in the five synovial sarcoma cell lines, were inves-\n",
      "tigated in tumor tissues of 18 synovial sarcomas and 11 STSs\n",
      "of other types (Fig. 2A). In accordance with the results in the\n",
      "cell lines, almost all 18 synovial sarcoma tumors expressed\n",
      "all of these FGF genes (Fig. 2A). Among five FGF genes, the\n",
      "FGF2, FGF9, and FGF11 genes were also expressed in some\n",
      "tumors other than synovial sarcoma. In contrast, expression of\n",
      "FGF8 and FGF18 genes was only weakly detected in other\n",
      "tumors. Expression profiles of these five FGF genes were further\n",
      "analyzed by the quantitative RT-PCR (Fig. 2B). The level of\n",
      "expression of FGF8 and FGF18 genes in synovial sarcoma\n",
      "tumors were significantly higher than in tumor tissues of other\n",
      "type of tumors, whereas those of FGF2, FGF9, and FGF11 genes\n",
      "showed no difference.\n",
      "As for receptor genes, synovial sarcoma tumors expressed all\n",
      "types of FGFR genes analyzed including the FGFR2b gene\n",
      "Growth stimulatory effects of\n",
      "fibroblast growth factors in\n",
      "synovial sarcoma cell lines. Expression of FGF and FGFR genes\n",
      "and proteins suggested the involvement of FGFs as autocrine/\n",
      "paracrine growth factors in synovial sarcoma. Among FGF genes\n",
      "expressed in synovial sarcoma both in vitro and in vivo, we\n",
      "focused on the FGF2, FGF8, and FGF18 genes in subsequent\n",
      "experiments. The FGF2 gene was selected as a gene commonly\n",
      "up-regulated in STS (Fig. 2A). FGF8 and FGF18 genes were\n",
      "selected as synovial sarcoma – specific up-regulated FGF genes\n",
      "(Fig. 2A).\n",
      "Growth stimulatory effects of each rhFGF protein were\n",
      "serum-starved conditions. Recombinant\n",
      "analyzed under\n",
      "hFGF8 showed a dose-dependent growth-promoting effect\n",
      "in all synovial sarcoma cell lines (Fig. 3A). On the treatment\n",
      "with rhFGF18, only HS-SY-II\n",
      "showed a dose-dependent\n",
      "increase (Fig. 3B). A heterogeneous response to FGF was\n",
      "more clearly observed in the case of rhFGF2, in which the\n",
      "growth of HS-SY-II\n",
      "increase,\n",
      "whereas a growth inhibitory effect was observed in the case\n",
      "of YaFuSS (Fig. 3C).\n",
      "showed a dose-dependent\n",
      "Activation of mitogen-activated protein kinases by fibroblast\n",
      "growth factor signals. One of major signal pathways located\n",
      "downstream of FGFR is the MAPKs, for which three subtypes are\n",
      "known: ERK, p38 MAPKs, and c-Jun NH2-terminal kinases\n",
      "(JNK; ref. 25). To investigate the events downstream of activated\n",
      "FGFRs, the phosphorylation of these MAPKs was investigated. In\n",
      "all synovial sarcoma cell lines analyzed, ERK1/2 was phosphor-\n",
      "ylated under serum-free conditions, suggesting the presence of\n",
      "endogenous stimulatory signals (Fig. 4A). The amount of\n",
      "phosphorylated ERK1/2 (pERK1/2) increased on treatment\n",
      "with FGF8 in a dose-dependent manner in all four cell lines\n",
      "analyzed, with the response of HS-SY-II being the highest\n",
      "levels. A dose-dependent\n",
      "Fibroblast growth factor receptor – specific inhibitors reduce the\n",
      "lines in accordance with\n",
      "growth of synovial sarcoma cell\n",
      "inactivation of fibroblast growth factor receptor. MAPKs in\n",
      "synovial sarcoma are activated by intrinsic pathways, and\n",
      "exogenous FGFs can stimulate the growth of synovial sarcoma,\n",
      "suggesting that inhibition of the intrinsic FGF signal pathway\n",
      "may inhibit the growth of synovial sarcoma. SU5402 is known\n",
      "to inhibit FGFR autophosphorylation at an IC50 of 10 to\n",
      "20 Amol/L (22). To confirm the effect of SU5402 on synovial\n",
      "sarcoma cell lines, the phosphorylation status of FGFR3 was\n",
      "analyzed by the immunoprecipitation and Western blotting in\n",
      "three synovial sarcoma cell lines (YaFuSS, SYO-1, and 1273/\n",
      "99; Fig. 5A). Phosphorylated FGFR3 was detected in these cell\n",
      "lines with serum-starved condition, and the treatment with\n",
      "rhFGF18 protein increased the amount of phosphorylated\n",
      "FGFR3 (Fig. 5A), which was compatible with the autocrine/\n",
      "paracrine model of FGFs in synovial sarcoma cells. Treatment\n",
      "with SU5402 effectively decreased the phosphorylation of\n",
      "FGFR3 into almost undetectable level (Fig. 5A), indicating that\n",
      "SU5402 can be used as an inhibitor of FGFR in synovial\n",
      "sarcoma cells. By the treatment with SU5402, the growth of all\n",
      "five synovial sarcoma cells was inhibited in a dose-dependent\n",
      "manner (Fig. 5B), and the IC50 value ranged from 8.5 to 17.2\n",
      "Amol/L (Table 1). The growth inhibitory effect of SU5402 for\n",
      "cell lines other than synovial sarcoma was not remarkable. The\n",
      "same experiments were done under low-serum conditions (1%\n",
      "FBS, Fig. 5C), resulting in similar growth inhibition in all\n",
      "synovial sarcoma cell lines. The IC50 was almost identical in\n",
      "the three cell lines (YaFuSS, HS-SY-II, and SYO-1), whereas the\n",
      "lines (Fuji and 1273/99) required a higher\n",
      "other two cell\n",
      "concentration of SU5402 (Table 1). Growth inhibitory effects\n",
      "of another FGFR-specific inhibitor, PD166866 (23), was also\n",
      "investigated under the low-serum conditions. All five synovial\n",
      "sarcoma cell lines had an IC50 value (0.08-4.7 Amol/L) much\n",
      "lower than those of other cell lines (>10 Amol/L, Table 1).\n",
      "Similar to the data obtained using SU5402, IC50 values in\n",
      "YaFuSS, HS-SY-II, and SYO-1 were much lower than those in\n",
      "Fuji and 1273/99.\n",
      "11.5 F 1.8\n",
      "15.1 F3.4\n",
      "8.5 F 2.5\n",
      "17.2 F 1.8\n",
      "12.8 F 6.1\n",
      "46.4 F 6.9\n",
      "65.4 F 2.6\n",
      ">80\n",
      ">80\n",
      "54.4 F 21.5\n",
      "11.4 F 3.6\n",
      "4.9 F 0.5\n",
      "8.6 F 1.6\n",
      "25.3 F 3.6\n",
      "29.9 F 16.6\n",
      "60.2 F 20.6\n",
      "63.7 F 27.8\n",
      "55.8 F 19.1\n",
      ">80\n",
      "78.1 F20.3\n",
      "0.08 F 0.04\n",
      "0.77 F 0.52\n",
      "0.18 F 0.11\n",
      "4.0 F 1.6\n",
      "2.7 F 0.4\n",
      ">10\n",
      ">10\n",
      ">10\n",
      ">10\n",
      ">10\n",
      "Growth inhibition of fibroblast growth factor receptor inhibitor\n",
      "associated with the reduction of phosphorylation of extracellular\n",
      "signal – regulated kinase 1/2 but not of p38.\n",
      "To investigate\n",
      "which signals may contribute to the growth of synovial\n",
      "sarcoma, the phosphorylation of ERK1/2 and p38 before and\n",
      "after the treatment with SU5402 was investigated. In three\n",
      "synovial sarcoma cell lines (YaFuSS, HS-SY-II, and SYO-1),\n",
      "which were relatively sensitive to the growth inhibitory effect of\n",
      "SU5402, phosphorylation of ERK1/2 under low-serum con-\n",
      "ditions was completely inhibited by SU5402, whereas the\n",
      "amount of phosphorylated p38 showed no change (Fig. 6A). A\n",
      "small amount of pERK1/2 was still observed after SU5402\n",
      "treatment in 1273/99, whereas no significant decrease was\n",
      "observed in Fuji, both of which showed no significant change in\n",
      "the pp38 (Fig. 6A). In cell lines other than synovial sarcoma,\n",
      "SU5402 showed no significant changes in the amount of\n",
      "pERK1/2 or pp38 (data not shown). These results suggested that\n",
      "the signal to induce the phosphorylation of ERK1/2 in synovial\n",
      "sarcoma was sent mainly through the FGF and FGFR pathway\n",
      "and that although the FGF signal induced the phosphorylation\n",
      "of both ERK1/2 and p38, the growth stimulatory effects were\n",
      "transmitted through ERK1/2. To further confirm that the growth\n",
      "inhibition of SU5402 was due to the reduction in pERK1/2, the\n",
      "function of MAPK/ERK kinase 1/2 in synovial sarcoma, which is\n",
      "a kinase of ERK1/2, was inhibited by U0216. Phosphorylation\n",
      "of ERK1/2 was effectively inhibited in all five cell lines, in which\n",
      "1273/99 still had a small amount of pERK1/2 at the lower\n",
      "concentration (Fig. 6B). BrdUrd uptake was inhibited in a dose-\n",
      "dependent manner in all five cell lines, in which 1273/99 was\n",
      "relatively resistant at the lower concentration, consistent with\n",
      "the status of pERK1/2 (Fig. 6C). These results further indicated\n",
      "that the growth inhibition of SU5402 in synovial sarcoma was\n",
      "achieved through the inhibition of signals through ERK1/2.\n",
      "Growth inhibition by fibroblast growth factor receptor inhibitor\n",
      "in synovial sarcoma is associated with cell cycle arrest. The\n",
      "fraction of each cell cycle phase was analyzed before and after\n",
      "the treatment with SU5402 in YaFuSS cells. SU5402 signifi-\n",
      "cantly increased the G1 and subG1 fractions, and decreased the\n",
      "S fraction (Fig. 6D). These data suggested that the growth\n",
      "inhibition of synovial sarcoma by SU5402 was due to its\n",
      "induction of G1 arrest as previously reported for other tyrosine\n",
      "kinase inhibitors (26).\n",
      "Fibroblast growth factor receptor inhibitors reduced the\n",
      "For in vivo study,\n",
      "growth of synovial sarcoma cells in vivo.\n",
      "we used SYO-1 because of\n",
      "its consistency in developing\n",
      "tumor, and PD166866 was used because of its lower IC50\n",
      "value. To confirm its efficacy in vivo, tumor tissue was taken\n",
      "30 minutes after the injection of PD166866. We found that\n",
      "0.1 mg of PD166866 was enough to reduce the phosphory-\n",
      "lated ERK1/2 in the tumor (Fig. 7A). Based on this result,\n",
      "the growth inhibitory effect of PD166866 (0.1 and 0.5 mg/\n",
      "body) was analyzed in vivo. Although PD166866 was unable\n",
      "to stop the growth of the tumor completely, a significant\n",
      "inhibition of growth was observed during the period of\n",
      "administration (P < 0.01), and the effect was retained even at\n",
      "day 21 at the higher dose (P = 0.02, Fig. 7B). No definite\n",
      "reduction of body weight or pathologic changes in vital\n",
      "organs such as liver and kidney was observed (data not\n",
      "shown). No growth inhibitory effect was observed when\n",
      "HT1080 was used instead of SYO-1 even at the higher dose\n",
      "(data not shown).\n",
      "Histologic examination of tumor tissues showed no signifi-\n",
      "cant morphologic changes or focal necrosis. The number of\n",
      "PCNA-positive cells, however, was clearly smaller in PD166866-\n",
      "treated tumors (19.3 F 6.6%, Fig. 7D) than control tumors\n",
      "(53.0 F 2.0%, Fig. 7C), indicating that PD166866 reduced the\n",
      "number of cells in S phase. No such difference was observed\n",
      "when HT1080 was used instead of SYO-1 (data not shown).\n",
      "Discussion\n",
      "FGFs may activate genetic programs which promote cell\n",
      "growth by at least one of three general mechanisms: first, as\n",
      "mitogens\n",
      "second, by\n",
      "promoting angiogenesis to supply a growing tumor; and\n",
      "third, by inhibiting apoptosis and allowing tumor cells to\n",
      "continue to grow beyond normal constraints. We found that\n",
      "synovial sarcoma cells expressed a number of FGF genes, of\n",
      "which FGF2 , FGF8 , FGF9 , FGF11 , and FGF18 were\n",
      "commonly expressed. The involvement of FGF2 in malignant\n",
      "tumors as an autocrine growth factor and/or a paracrine\n",
      "angiogenic factor was first shown in brain tumors (27), and\n",
      "subsequently in non – small cell carcinoma (14). FGF8 was\n",
      "originally identified as an androgen-induced growth factor\n",
      "found in conditioned medium of androgen-dependent mouse\n",
      "mammary carcinoma SC-3 cells and reported to have a\n",
      "growth stimulatory effect (28). The human FGF8 gene was\n",
      "expressed in breast and prostate cancers (15, 16, 29) and\n",
      "associated with anchorage-independent proliferation and\n",
      "invasion (30, 31). FGF9 was originally purified from the\n",
      "conditioned medium of\n",
      "line NMC-G1 and\n",
      "designated as glia-activating factor due to its mitogenic\n",
      "activity toward glial cells (32). FGF11 was one of four FGF\n",
      "homology factors, which shared structural homology with\n",
      "failed to activate FGFRs (33). Recently,\n",
      "classic FGFs but\n",
      "FGF11 was identified as one of the gene products enriched in\n",
      "neuronal precursors during the development of the nervous\n",
      "system (34). FGF18 is cloned by its homology to FGF8 and\n",
      "FGF9 (35, 36). FGF18, in association with FGF8, is expressed\n",
      "at various times and places during embryogenesis, especially\n",
      "neurogenesis (33, 37). Mitotic activity was shown in colon\n",
      "carcinomas,\n",
      "in which the expression of FGF18 was up-\n",
      "regulated by WNT signals (17). Overall, we found that\n",
      "synovial sarcoma expressed several FGF genes, particularly\n",
      "those expressed in neural tissues, which may further suggest\n",
      "the neural origin of precursor cells of synovial sarcoma as we\n",
      "previously proposed (13). Mitotic activity of exogeneous FGFs\n",
      "was not remarkable in some cell lines (Fig. 3). The serum-\n",
      "free medium used in this study contained insulin, which is\n",
      "known to potentiate the effect of FGF, and the further\n",
      "addition of insulin showed no increase of the BrdUrd uptake\n",
      "(data not shown). It is possible that the extent of autocrine\n",
      "stimulation may already be saturating so that further FGF\n",
      "stimulation does not lead to additional growth.\n",
      "The expression of FGFR1, FGFR2c, FGFR3, and FGFR4, to\n",
      "which FGF2 (1c, 3c and 4), FGF8 (2c, 3c, 4), and FGF18 (2c,\n",
      "3c, 4) preferentially bind (38, 39), was confirmed in both tumor\n",
      "tissues and cell lines. The expression of FGFR2b gene, which was\n",
      "detected in tumor tissues but not in cell lines, is intriguing.\n",
      "FGFR2b is a receptor for keratinocyte growth factor (FGF7), and\n",
      "expressed in epithelial cells (40). As expected, all biphasic\n",
      "synovial sarcomas expressed the FGFR2b gene, whereas the\n",
      "expression was low in most of the monophasic synovial\n",
      "sarcomas and other STSs. It is, however, intriguing that some\n",
      "monophasic synovial sarcomas with no apparent epithelial\n",
      "structures expressed the FGFR2b gene at a level comparable with\n",
      "that in biphasic synovial sarcoma (Fig. 2D). All five synovial\n",
      "sarcoma cell lines used in this study including SYO-1, which was\n",
      "established from a biphasic synovial sarcoma, were negative for\n",
      "the FGFR2b gene. These results suggested that synovial sarcoma\n",
      "tumor cells may have an intrinsic mechanism to express FGFR2b\n",
      "in vivo, which was somehow inhibited or disappeared in vitro.\n",
      "Alternatively, synovial sarcoma cells positive for FGFR2b may\n",
      "have growth disadvantages, and therefore only negative cells\n",
      "were established as cell\n",
      "lines. Recently, single nucleotide\n",
      "polymorphism in FGFR4 is reported to associate with poor\n",
      "prognosis in high-grade soft tissue sarcomas including synovial\n",
      "sarcomas (41) and both FGF8 and FGF18 bind to FGFR4.\n",
      "Although the status of polymorphism was not investigated in our\n",
      "samples, FGF signals through FGFR4 may contribute to the\n",
      "aggressiveness of synovial sarcoma.\n",
      "The binding of FGF to FGFR leads to receptor dimerization\n",
      "and tyrosine autophosphorylation, followed by downstream\n",
      "activation of PLCg, Crk, and SNT-1/FRS2 signaling pathways\n",
      "among which the SNT-1/FRS2 is the major pathway for\n",
      "mitogenic signals (25). Phosphorylated SNT-1/FRS2 directly\n",
      "binds to the GRB2-SOS complex, and then membrane-\n",
      "associated RAS recruits RAF-1, which in turn activates MAPKs\n",
      "(42). MAPKs are composed of three well-characterized sub-\n",
      "families, ERK, p38 MAPK, and JNK (25), and we found that the\n",
      "growth-promoting signal of FGF was transmitted mainly\n",
      "through ERK1/2. Although ERK1/2 are activated by signals\n",
      "through other receptor tyrosine kinases (43), phosphorylation\n",
      "of ERK1/2 was completely inhibited by FGFR inhibitors, which\n",
      "indicated that the major signal to activate ERK1/2 in synovial\n",
      "sarcoma is FGF signaling, suggesting the important role of FGF\n",
      "in synovial sarcoma.\n",
      "We have no clear explanation for the heterogeneous response\n",
      "in synovial sarcoma cell lines to FGF2, which stimulated the\n",
      "growth of HS-SY-II, but inhibited the growth of YaFuSS. In\n",
      "contrast with many studies describing the role of FGF2 as a\n",
      "mitogen, several reports showed that FGF2 acted as a growth\n",
      "inhibitor of tumors of Ewing’s sarcoma family, which are\n",
      "supposed to be of neuroectodermal origin (44, 45). Multiple\n",
      "FGF signals are involved in the biological process in synovial\n",
      "sarcoma, and the response may depend on the cell specificity.\n",
      "Nevertheless, the disruption of FGF signals was found to cause\n",
      "growth inhibition in all synovial sarcoma cell lines, suggesting\n",
      "that an inhibitory molecule for FGFR will be a promising tool\n",
      "for molecular target therapy in synovial sarcoma.\n",
      "Molecular therapy targeted to FGFR has been investigated\n",
      "with hematologic malignancies carrying a translocation involv-\n",
      "ing the FGFR gene such as chronic myeloid leukemia with the\n",
      "BCR-FGFR1 gene (46) and multiple myeloma with the MMSET-\n",
      "FGFR3 fusion gene (47). In both instances, SU5402 was found\n",
      "to effectively inhibit the growth of cell lines with an IC50\n",
      "equivalent to that in synovial sarcoma. For experiments in vivo,\n",
      "we used PD166866 instead of SU5402 because of its lower IC50\n",
      "value and found that i.p. given PD166866 efficiently inhibited\n",
      "tumor causing an\n",
      "the phosphorylation of ERK1/2 in s.c.\n",
      "inhibition of growth without significant side effects. As in the\n",
      "case of other tyrosine kinase inhibitors, PD166866 did not\n",
      "strongly induce apoptosis or necrosis, and therefore no\n",
      "complete tumor remission was observed. However, cell cycle\n",
      "arrest caused by the treatment with PD166866 may enhance\n",
      "the cytotoxic effects of anticancer drugs as previously shown in\n",
      "some instances (48, 49), and therefore further in vivo studies\n",
      "using the treatment combined with suitable anticancer drugs\n",
      "may provide a promising approach for clinical application.\n"
     ]
    }
   ],
   "source": [
    "print(sections['all'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "recoRAG",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
