{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append(\"..\")\n",
    "sys.path.append(\"../src\")\n",
    "\n",
    "from dataset import load_dataset, push_data_to_drive, extract_pdf_sections, get_valid_pages\n",
    "from dataset import download_and_extract_PDFs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "fieldsOfStudy",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "venue",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorName",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "citationCount",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "influentialCitationCount",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "publicationDate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "publicationYear",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "referenceIds",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "77bba53f-d93e-458e-8919-d61bed3ad5a6",
       "rows": [
        [
         "0",
         "000a3e6667afb4ca8476ce15555b05244c3b458a",
         "https://www.nature.com/articles/s41598-025-86087-8.pdf",
         "Medicine",
         "Scientific Reports",
         "Fenghui Lian,Yingjie Sun,Meiyu Li",
         "2064292642,2340550389,2109040952",
         "0",
         "0.0",
         "2025-01-15",
         "2025",
         "",
         ""
        ],
        [
         "1",
         "00139e24da15df3f2f4ef70a92a781ccf315ef47",
         "https://www.biorxiv.org/content/biorxiv/early/2025/02/19/2025.02.19.639066.full.pdf",
         "Biology",
         "bioRxiv",
         "Jana Jung,Timo Glatter,Marco Herfurth,L. Søgaard‐Andersen",
         "2346560940,2277617384,1992769227,1398654276",
         "0",
         "0.0",
         "2025-02-19",
         "2025",
         "",
         ""
        ],
        [
         "2",
         "0027ea27f5986f9c6ee9776e10289c7f2aa089ca",
         "http://arxiv.org/pdf/2501.13553",
         "Computer Science",
         "International Conference on Compiler Construction",
         "Robert Szafarczyk,S. Nabi,W. Vanderbauwhede",
         "2168878518,1714681,1692663",
         "1",
         "0.0",
         "2025-01-23",
         "2025",
         "",
         ""
        ],
        [
         "3",
         "00291c982d2d9e69d8fe8c89a7db1fbb25f15687",
         "https://www.biorxiv.org/content/biorxiv/early/2024/06/25/2024.06.22.600087.full.pdf",
         "Biology",
         "bioRxiv",
         "Affiong I. Oqua,Kin Chao,Liliane El Eid,Lisa Casteller,Billy P Baxter,Alba Miguéns-Gómez,Sebastian Barg,Ben Jones,Jorge Bernadino de la Serna,Sarah L Rouse,Alejandra Tomas",
         "30455986,2283562588,2182492272,2308111697,2338816502,2338815388,2243227743,2296667017,2338816370,2283585238,2256102695",
         "0",
         "0.0",
         "2025-01-02",
         "2025",
         "346ef7fdf555fc0396f9e954bcda723a2fb1b356",
         "Imperial"
        ],
        [
         "4",
         "00317a3206df00f0c3c91595a349bc81b7118929",
         "http://arxiv.org/pdf/2502.03676",
         "Computer Science",
         "IEEE Robotics and Automation Letters",
         "Yeping Wang,M. Gleicher",
         "2115738494,2066395412",
         "1",
         "0.0",
         "2025-02-05",
         "2025",
         "",
         ""
        ],
        [
         "5",
         "0034e238be40924c815d89333e50afeb8fde846e",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/17/2024.10.17.618923.full.pdf",
         "Biology",
         "bioRxiv",
         "Meghan M. Moran,Jun Li,Quan Shen,Sheona P. Drummond,C. Milner,Anthony J. Day,A. Naqib,D. R. Sumner,Anna Plaas",
         "7167369,2327626722,2069020677,2279798536,143647897,2327037019,3528292,2298230556,2294582175",
         "0",
         "0.0",
         "2025-02-19",
         "2025",
         "",
         ""
        ],
        [
         "6",
         "0082b6dd5b95334df816971d506e22a0f0da9cc4",
         "https://figshare.com/articles/journal_contribution/Two_new_amino_acid-derived_oximes_from_the_mangrove-sediment-derived_fungus_i_Lecanicillium_kalimantanense_i_SCSIO_41702/28323314/1/files/52058141.pdf",
         "Medicine",
         "Natural Product Research",
         "Xu-Meng Ren,Lin-Fang Zhong,Ke-Yue Wu,Xiao Liang,Shu-Hua Qi",
         "2334697739,2263771937,2334438251,2153398053,2053993487",
         "0",
         "0.0",
         "2025-01-31",
         "2025",
         "",
         ""
        ],
        [
         "7",
         "008b67207bda1f15d92c5c6d9283c2fae1ace158",
         "https://www.researchsquare.com/article/rs-4132920/latest.pdf",
         "Computer Science",
         "Displays (Guildford)",
         "Samirah Altukhaim,Naoko Sakabe,Kirubananthan Nagaratnam,Neelima Mannava,Toshiyuki Kondo,Yoshikatsu Hayashi",
         "2111514858,2111523944,2339793923,13163310,2288956458,2249229457",
         "1",
         "0.0",
         "2025-01-01",
         "2025",
         "6f6e700e37721ea89d058b326ea256c0cf8fa8d9;955f24b7aaccb45c242b9a9b0aa4160b5a109eb4;8bcc4c37694307385527aed48fe740b504bec608;b40e32c020366a26744ea157b393c392da4bf0fe;4914701c3e312fc76be338b30ff83be4229942c1;f33c66df439ed0d0fa38844c54de9c4267b80ecd;57e5dc0c444b47f460531e8004541017207112d9;95823a2bab2321f0d5c12ba55081511e271d52dc;85f8cfa361aebb5c633c2e82d34fa4b3c286e096;304d72b8a1d4cb2ea1dea37c325f824405671e19;1463f8a0f2990694a49263029f9c89878be4ac60;1091ff983b034bbf7d9fc5bf4b10f204ac60dc88;224a82608f57bcf8568159a337ba911c613299e0;834d478732bf78fed106b5fbe6a0a4bc18a1107a;b7eb1db2b92d382778dc818b099dd6491ceea9d8;93d90114efa1863f94bb1f8cfe929197fccb43b0;91b657fae7258eb8dab5845327a03d084fefcbf0;d9d87cc217ce00ce1be0ecf411dbd88f04448861;d48e1cbb4d9ec501acedc2dd6a3456ef2fea77f6;39870d36edcf62f774a2c836e3868462c755579e;8af42ac53127e8deb2d0ce258b74a4b2d2906330;6e23b5a3677e518508a116e0734101af917f7570;11d66ab0b7019f1a51605cdfc86554ce8192e927;aa41d824bbc21a761228c9dd961d8fb1973725ef;054ff45ac81b9477aaaa3d7d9883581030adec0a;6d804ae534d19b0cbde26ec3cc7340bc38072c04;322f2a23f27383922b4c0602954171841f7f4068;40484fea5a2dab3f8a7d207fff38adaca3c59601;0672617f1d7111b9ea7757bfff2c2c4325395375;44c0860feb26ad4b6761feeec26340f888d0d502;21bd2841cc11f2df294279c0b5dda45e1e8ad76e;33d732245e04e38f793da6a02929fc3bbde02e33;e8e181e3c1e77ebcd088a0001c0f2801f55dce0e;1a1a1379d4fa10bddcf52c0aa2f72de8c7d23c49;058986a51b4e46e1efc62b39a51ec33515720f94;0a41d85e192ee0c236c188685e070e6137570acb;631e1183a51bc7fc0445d94413ae0368e07cd736;1b61dc05db02fb00349a5f809e161f2578dbdbc2;69854e21f2ba0eefb0b468a7f7a77f9d8ee726d3;1290baa19d0917e30faddd385e28dcec7b5af800;bace6d72540f41699205edf057af16cbe635431a;2675bac8f3ee81918603fef5b38d1cb3baf1cc28;fa72748b82c6d68c15b8c6bb4a14649a8458cb02;e8215844d84bcdd03104eb9753a31ebbd17dced9;f9fa613e8e53f3da9317993311bf9c286d94ddbe;45aa95e3668b67f1708c9d719233fab1840fcf7d;4c41e1c71404d71698c5dd300ac46ad8e07b6102;377f431de70b9027bf353dbcd93024c2178bcbb7;da28b0ebd337efa846ac7857d34db71ee7221ecd;3ead86286f338886798ccbda98cebc46d90f192a;75554749c0f696088957f14ea620a5ff956cbad4;6d44f492a8bd891e986d68af86c91145669f6c22;71c4a1f734bef36b9ad0a291bdcc29adc5748b7a;98f38f235375b846d71287e1a807dbbbce3f0431;95469539792004d038f94830948f21461cb49c7e;75b4966859bde87d86db57aaabe90bc9a24d4fdd;d0a238f400ff9ea7038f63b982781b0e2c6bf5ca;2c20d97ff237a1882de426237ba84d77917a0e8d;63658167b1fc618ff70be21bdfb264324d696f3e;163f868c756260221edeb4693b8fb320688f1f7b;880abb21380c79fbd89db9fb6d685b9894f48afe;5c576d779839ffd6ef54467bac7bdc0186641d7e;a73f093add8e85bd974bb729ff231ff4f925b8bd;2a7b68006306b8e42676db90aa3fb03818f89d8a;c493f94a24056f4d215d98f28923d7e5566d6709;b03bcca3a5f7b5ac9547b5c49a3ccc9402603385;ab30e1514660f5880ad14a8d93668570e1189f14;1d59e9a675ed8618f72b4a5c3fd851f48549386f;865e1105a843c2d57d7fa8bb0c4dfcf3e02c2dae;7ac7a228b3cbb08349212e4bbe4c90d9c243cf58;f8531c0ad04c3f850abc88c705224cf21f31c827;ed09074b546ea3e827683415f54a1353ecf901a7;047b58d265e5fffb321f96f034a3c2bdd8883544;172ed1332c0ba51af4c91d3fdc4b244f71fc05a5;e232ede843c9b3f93b6a5d719bfae6043e039953;551f062060c062a026a559d8d5d83ba708bc587b;a459bf376c8fc12884b5024f3fdd24c2c9818a74;1a35527a1956d893601f6f7f4b8e96db51cd69ee;b99269ca05dd571445255e1a5fc8b1f0ca095218;acd68210b5585f70d989370c92cca1a6d4675deb;7f4c64ee8a19336e08fad856353d29ea79b39fa7;6cf57a73ab400ad65df77663187c8296a783700f;a5df9bc637dafe1599619b6c06928680932981b8;1603072459bcf4a40a182460de144b08408d5a7c;c8bc3cf8f506d093452707a189f66d53ac8307b0;907d72f62967272b04b59224c262a4aae95bc234;e0db01812fe8ce272c64ac3f88633f64e19ee168;634c9fde5f1c411e4487658ac738dcf18d98ea8d;7a2422e7cf038232dca8fa1a930249e226382e0a;bb5dc4d4d67d9c472083846567934af7fde9a93c;db166a794c6c4c961e3b25fc688f700684a8e5e0;df4fc9cf0162133758f55424b8ca64856b364afa;021f29dd49fd359178064a1874467627c07d7e11;e995514530d12552a41ecdc279ff07ea00ef212f;8b34b1b83fd819506eb8edb29bf007c034cab70f;5bb7b79c29034c515ee423daffadd8364d2526c4;02edf22980611f433084a112b490221a3500f4b1",
         "Immersive Virtual Reality in Post-Stroke Rehabilitation: A Systematic Review;Effectiveness, safety and patients’ perceptions of an immersive virtual reality–based exercise system for poststroke upper limb motor rehabilitation: A proof-of-concept and feasibility randomized controlled trial;New technologies promoting active upper limb rehabilitation after stroke: an overview and network meta-analysis;Effectiveness of Using Virtual Reality–Supported Exercise Therapy for Upper Extremity Motor Rehabilitation in Patients With Stroke: Systematic Review and Meta-analysis of Randomized Controlled Trials;Enhanced Visual Feedback Using Immersive VR Affects Decision Making Regarding Hand Use With a Simulated Impaired Limb;A novel fully immersive virtual reality environment for upper extremity rehabilitation in patients with stroke;Neuroplasticity;Virtual reality games for rehabilitation of upper extremities in stroke patients.;On Shooting Stars;Post-stroke depression: A 2020 updated review.;Virtual reality therapy for upper limb rehabilitation in patients with stroke: a meta-analysis of randomized clinical trials;Principles of Neurorehabilitation After Stroke Based on Motor Learning and Brain Plasticity Mechanisms;Exergaming for stroke rehabilitation: Lessons learned for future implementation strategies;Elements virtual rehabilitation improves motor, cognitive, and functional outcomes in adult stroke: evidence from a randomized controlled pilot study;Comparison of Kinect2Scratch game-based training and therapist-based training for the improvement of upper extremity functions of patients with chronic stroke: a randomized controlled single-blinded trial.;Evaluating the effect and mechanism of upper limb motor function recovery induced by immersive virtual-reality-based rehabilitation for subacute stroke subjects: study protocol for a randomized controlled trial;Virtual Reality in Upper Extremity Rehabilitation of Stroke Patients: A Randomized Controlled Trial.;Project Star Catcher;Rehabilitation via HOMe Based gaming exercise for the Upper-limb post Stroke (RHOMBUS): protocol of an intervention feasibility trial;Virtual Reality Clinical Research: Promises and Challenges;Combining the benefits of tele-rehabilitation and virtual reality-based balance training: a systematic review on feasibility and effectiveness;Towards an Immersive Virtual Reality Game for Smarter Post-Stroke Rehabilitation;Effectiveness of Wii-based rehabilitation in stroke: A randomized controlled study.;Virtual Reality for Upper Limb Rehabilitation in Subacute and Chronic Stroke: A Randomized Controlled Trial.;Game-Based Virtual Reality Canoe Paddling Training to Improve Postural Balance and Upper Extremity Function: A Preliminary Randomized Controlled Study of 30 Patients with Subacute Stroke;What do randomized controlled trials say about virtual rehabilitation in stroke? A systematic literature review and meta-analysis of upper-limb and cognitive outcomes;In-Home Delivery of Constraint-Induced Movement Therapy via Virtual Reality Gaming.;Effects of Kinect-based virtual reality game training on upper extremity motor recovery in chronic stroke;Increasing upper limb training intensity in chronic stroke using embodied virtual reality: a pilot study;Leap Motion-based virtual reality training for improving motor functional recovery of upper limbs and neural reorganization in subacute stroke patients;Stroke recovery and rehabilitation in 2016: a year in review of basic science and clinical science;Effects of virtual reality for stroke individuals based on the International Classification of Functioning and Health: a systematic review;Does the use of Nintendo Wii SportsTM improve arm function? Trial of WiiTM in Stroke: a randomized controlled trial and economics analysis;What Is the Sense of Agency and Why Does it Matter?;Counteracting learned non-use in chronic stroke patients with reinforcement-induced movement therapy;Coaching or gaming? Implications of strategy choice for home based stroke rehabilitation;Post-stroke fatigue: a review on prevalence, correlates, measurement, and management;Constraint-induced movement therapy after stroke;CI Therapy is Beneficial to Patients with Chronic Low-Functioning Hemiparesis after Stroke;Virtual Reality Therapy for Adults Post-Stroke: A Systematic Review and Meta-Analysis Exploring Virtual Environments and Commercial Games in Therapy;What Is the Evidence for Physical Therapy Poststroke? A Systematic Review and Meta-Analysis;Plasticity in the Injured Brain;Connectome: How the Brain’s Wiring Makes Us Who We Are;Mechanism of Kinect-based virtual reality training for motor functional recovery of upper limbs after subacute stroke;Virtual reality for the rehabilitation of the upper limb motor function after stroke: a prospective controlled trial;Depression, activities of daily living and quality of life in patients with stroke;Arm Motor Recovery Using a Virtual Reality Intervention in Chronic Stroke;Maladaptive Plasticity for Motor Recovery after Stroke: Mechanisms and Approaches;Neurorehabilitation of stroke;Virtual reality in neuroscience research and therapy;Virtual Reality for Stroke Rehabilitation;The Stroke Association;Stroke rehabilitation;Potential for new technologies in clinical practice.;Creating an Interface Between the International Classification of Functioning, Disability and Health and Physical Therapist Practice;Apathy following Stroke;The neural basis of constraint-induced movement therapy;Exercises for paretic upper limb after stroke: a combined virtual-reality and telemedicine approach.;Observation of amounts of movement practice provided during stroke rehabilitation.;Repetitive Task Training for Improving Functional Ability After Stroke;Sense of Agency Primes Manual Motor Responses;Training and exercise to drive poststroke recovery;Principles of experience-dependent neural plasticity: implications for rehabilitation after brain damage.;Poststroke shoulder pain: its relationship to motor impairment, activity limitation, and quality of life.;The learned nonuse phenomenon: implications for rehabilitation.;Textbook of Neural Repair and Rehabilitation: Virtual reality in neurorehabilitation;Ambulatory monitoring of arm movement using accelerometry: an objective measure of upper-extremity rehabilitation in persons with chronic stroke.;A Virtual RealityBased Exercise System for Hand Rehabilitation Post-Stroke;Motor Impairment and Recovery in the Upper Limb After Stroke: Behavioral and Neuroanatomical Correlates;A Theory of Fun for Game Design;Mixed reality environments in stroke rehabilitation: Development as rehabilitation tools;Rules of play: game design fundamentals;The Fugl-Meyer Assessment of Motor Recovery after Stroke: A Critical Review of Its Measurement Properties;What are the components of effective stroke unit care?;Stroke patients' and therapists' opinions of constraint-induced movement therapy;Research on Presence in Virtual Reality: A Survey;Neurological deterioration in acute ischemic stroke: potential predictors and associated factors in the European cooperative acute stroke study (ECASS) I.;Impairment and disability: their relation during stroke rehabilitation.;From apoplexy to stroke.;Assessment and psychologic factors in stroke rehabilitation.;Motor learning after recovery from hemiparesis;Reliability of the Fugl-Meyer assessment for testing motor performance in patients following stroke.;RECOVERY TIME OF INDEPENDENT FUNCTION POST-STROKE;Neuropsychological impairments associated with lesions caused by tumor or stroke.;A Motor Learning Model for Stroke Rehabilitation;Reliability of the Fugl-Meyer assessment of sensorimotor recovery following cerebrovascular accident.;Long‐lasting potentiation of synaptic transmission in the dentate area of the anaesthetized rabbit following stimulation of the perforant path;The information capacity of the human motor system in controlling the amplitude of movement.;Virtual Activities of Daily Living for Recovery of Upper Extremity Motor Function;Neurobiology of Sleep and Circadian Rhythms The role of sleep in recovery following ischemic stroke: A review of human and animal data;Motor learning and motor recovery in poststroke individuals: a review of the evidence;Virtual reality training for upper extremity in subacute stroke (VIRTUES): a multicenter RCT;Stroke Rehabilitation A Function Based Approach;The brain's way of healing : stories of remarkable recoveries and discoveries;Febr 25. Stroke deaths in England halved in the �rst decade of the 21st century;Using the international classi�cation of functioning, disability and health in physiotherapy in multidisciplinary vocational rehabilitation: a case study of low back pain;Contribution of the shaping and restraint components of Constraint-Induced Movement therapy to treatment outcome.;Random.org.;Contemporary management of motor control problems;Improving the sensitivity of the Barthel Index for stroke rehabilitation.;The Barthel ADL Index: a reliability study.;Cerebrovascular disease in the community: results of a WHO collaborative study.;Journal of Neuroengineering and Rehabilitation Open Access Reaching in Reality and Virtual Reality: a Comparison of Movement Kinematics in Healthy Subjects and in Adults with Hemiparesis;ET019) who stayed longer in the hospital were evaluated three times. Nevertheless, data from only the �rst and �fth sessions were analyzed (Table 5A)"
        ],
        [
         "8",
         "008dbf8a0db8fa34cd847d7937b8ddbd2c0b37df",
         "https://biblio.ugent.be/publication/01JD53YTP2RKAS44E416FSWMXW/file/01JD54148HSWY3634VH3MMMV6V.pdf",
         "Medicine",
         "Journal of Experimental Psychology: Human Perception and Performance",
         "Maura Nevejans,J. Wiersema,J. de Houwer,Emiel Cracco",
         "2151524023,123060968,3602422,4288646",
         "0",
         "0.0",
         "2025-01-23",
         "2025",
         "",
         ""
        ],
        [
         "9",
         "00a9a436e7eb5e57de9231fc28b57e8452f45f4c",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/21/2024.08.20.608462.full.pdf",
         "Biology",
         "bioRxiv",
         "Jinru He,A. Klimovich,Sabine Kock,Linus Dahmke,Sören Franzenburg,Thomas C. G. Bosch",
         "2289486984,34337069,2316858491,2316857892,2294337866,2267763616",
         "0",
         "0.0",
         "2025-03-07",
         "2025",
         "4edec8a87a12ae490c5514d826ba87c102acb818;86350230568af77a534b914a2a9dff7cdeb6bbb4;b150ffd0dfbaafc78f5a4bca1cdfa8a309edf05b;bc2ff5c0668b1c6d3b8849e472f8a106895cfef5;78b6f1dc0b633d47805a3422c61bf5dc42df1267",
         "The Role of Lung and Gut Microbiota in the Pathology of Asthma;Mouse Microbiota Models: Comparing Germ-Free Mice and Antibiotics Treatment as Tools for Modifying Gut Bacteria;Punctuated Emergences of Genetic and Phenotypic Innovations in Eumetazoan, Bilaterian, Euteleostome, and Hominidae Ancestors;EST Analysis of the Cnidarian Acropora millepora Reveals Extensive Gene Loss and Rapid Sequence Divergence in the Model Invertebrates;Elimination by Hydra interstitial and nerve cells by means of colchicine."
        ],
        [
         "10",
         "00f1e24ed9abc1d314d58bad653da4de8e95539e",
         "https://www.biorxiv.org/content/biorxiv/early/2024/12/09/2024.12.05.627021.full.pdf",
         "Biology",
         "bioRxiv",
         "Felix Langschied,M. Leisegang,Stefan Günther,F. Hahner,Ralf Peter Brandes,Ingo Ebersberger",
         "51066936,2239858785,2265653135,30005641,2243713122,2250574407",
         "0",
         "0.0",
         "2025-02-07",
         "2025",
         "",
         ""
        ],
        [
         "11",
         "013cb58c3526d64e0f8e0d18d71c57250eaa0db2",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/20/2024.08.20.608769.full.pdf",
         "Biology",
         "bioRxiv",
         "A. Perry,Joan K. Beaton,J. Stockan,G. Iason,J. Cottrell,S. Cavers",
         "3083562,40259332,4993063,6294722,2261744330,5202463",
         "1",
         "0.0",
         "2025-02-11",
         "2025",
         "68f2bbbb6c4a7a47f62123dc5ede6dcab06185f1;634193909d96a5f49fd2e32aa3efbcba99d0a145;6bf7b936954f45c6bae49d0a8ed9767bf59dd549;545dc5fe906c9c06a8f5aa76b97465e0edc5dbaf;a533606f0001dd5c7ab65863ea871d451102eca2;e34a6f34395f06631fa10ea9e1c2074faff0ff90;44a7d15987553e17a12d5c41fe6ac552881c20eb;6ac638500a919ad8d35f6c2d056b27b760f89723;7d7a83bb31c17511bf3af92f2bfc41b69ce4ce39;c4efa6be19614d943031d0ed4963712406f69ad3;2eda87bd20b3c7a7925bd8d46beb74cfdc6760bb;ce2e127a76809a170fb3e497ea035315ed1aec1a;fc5ef641cff64275fca067cf493ce5122fc07036;a0652d2c46744c9ad7a9ab36885b7d09f9ca4fb0;cb45eb725eee0e9dfd7749c24641afda94289419;165ab0bcde1f80965563a10e968424b176773f76;11f4e8a9ba7b480600703998e54f0f6e7a293231;69ebbd1299ac9b1db121b45dac14b943cdaa2fe7;6e0369af30c8a52fe40a4066b5233416a920f031;94560b886e674d2111eda6600646598bfd03af88;67edcf8e927f39268778be1176ba4737d08def11;3650f71f5b1c307336946e8173769d1601a7b96c;0d53e6631a48a864496b98a9c438230fe1412906;b1efaad71fdc2b532203ddd73fb16b6f6d0e6ef5;d6c72391f565b973032b13fd14d0b7d66c57757c;b3af3c33a32c69333a997a4e27cfffdc7912d305;36629961a3e68527ad128fae880626491a91f3ad;cd42d586dc8eb605e322d2c2d87dc7c1ce50b575;a63f6e60cd55c4e4fefe9f378e3feded2edac8c6;524f9b31dc53fca19846f5a15da3afd63c3ca3ec;2c0c7327e3c373ca0852f7721d28fd6cb8fd43c5;fe6f1e6a8fce05f0d717b9924172d7cbf4c08864;b42b639358029f5724b0b6aea2df0743ed0c3e4b;2a030eba8aa438fd2c93ee1b58dcf75e0c95a735;42435a57486bc4ef5c8606ab84acd2ff0fcd87c7;47ea95c4f1f39fd443d5bba361f92a444ce255e7;9cfd46949708c08bd052a3324e025cb50b32d153;a860a438904acb644ae9ff1c30a8db55e5459286;c850ef64b05740c911951f9527415aee1b632ed3;3ef577e60ddf5e9f596d68b2f61e11482bc395a2;b6834a414f1c9aac1b32ee35c922971f0a8b435e;e86ca8d829ae00253b72836f5a291d9dff4addca;f77afd6a4d437a824fcdb2700b8c5adf72833f13",
         "Meeting tree planting targets on the UK's path to net-zero: A review of lessons learnt from 100 years of land use policies;Phenotypic trait variation in a long-term multisite common garden experiment of Scots pine in Scotland;Explaining Extreme Events of 2020 from a Climate Perspective;Phenotypes of Pinus sylvestris are more coordinated under local harsher conditions across Europe;Selection patterns on early-life phenotypic traits in Pinus sylvestris are associated with precipitation and temperature along a climatic gradient in Europe.;Reaching Natural Growth: The Significance of Light and Temperature Fluctuations in Plant Performance in Indoor Growth Facilities;Guidance for successful tree planting initiatives;Cryptic genetic variation and adaptation to waterlogging in Caledonian Scots pine, Pinus sylvestris L.;Age–age correlations and early selection for growth traits in 40 half-sib families of Larix principis-rupprechtii;Pampered inside, pestered outside? Differences and similarities between plants growing in controlled conditions and in the field.;Has Scots pine (Pinus sylvestris) co‐evolved with Dothistroma septosporum in Scotland? Evidence for spatial heterogeneity in the susceptibility of native provenances;Bud flush phenology and nursery carryover effect of paper birch provenances;Age trend of heritability, genetic correlation, and efficiency of early selection for wood quality traits in Scots pine;Plant growth and mortality under climatic extremes: An overview;What controls tropical forest architecture: testing environmental, structural and floristic drivers;Cultivation of Norway spruce and Scots pine on organic nitrogen improves seedling morphology and field performance;Seasonal patterns of photochemical capacity and spring phenology reveal genetic differentiation amon;A Large and Persistent Carbon Sink in the World’s Forests;Efficiency of the indirect selection and the evaluation of the genotype by environment interaction using Pilodyn for the genetic improvement of wood density in Cryptomeria japonica;A method to construct dose-response curves for a wide range of environmental factors and plant traits by means of a meta-analysis of phenotypic data.;Evaluation of leaf traits for indirect selection of high yielding poplar hybrids;The physiological basis of containerised tree seedling ‘transplant shock’: a review;Age–age and trait–trait correlations for Eucalyptus grandis Hill ex Maiden and their implications for optimal selection age and design of clonal trials;Effects of soil temperature on biomass and carbohydrate allocation in Scots pine (Pinus sylvestris) seedlings at the beginning of the growing season.;Effect of Peat-based Container Media on Establishment of Scots Pine, Norway Spruce and Silver Birch Seedlings after Transplanting in Contrasting Water Conditions;Current and future status of Scots pine (Pinus sylvestris L.) forests in Europe;Heritable variation and evolution under favourable and unfavourable conditions.;The nitrogen economy of mountain birch seedlings: implications for winter survival;Effect of diurnal temperature alternations on plant morphology in some greenhouse crops. A mini review;Seasonal and geographical variation of terpenes, resin acids and total phenolics in nursery grown seedlings of Scots pine (Pinus sylvestris L.);Effects of root temperature on growth and photosynthesis in conifer seedlings during shoot elongation.;Nestling growth in the Great Tit I. Heritability estimates under different environmental conditions;Accelerated short-term genetic testing for loblolly pine families;Effects of Photoperiod on Growth of Trees;Autumn versus spring planting: the initiation of root growth and subsequent field performance of Scots pine and Norway spruce seedlings;Photoperiod- and temperature-mediated control of phenology in trees - a molecular perspective.;GENETIC SURVEY OF PINUS RADIATA. 5: BETWEEN-TRAIT AND AGE-AGE CORRELATIONS FOR GROWTH RATE, MORPHOLOGY, AND DISEASE RESISTANCE;Plant Growth and Climate Change.;Low temperature, but not photoperiod, controls growth cessation and dormancy induction and release in apple and pear.;Selection of parents for the Scots pine breeding population in Britain;Using Native Stock for Planting Native Trees and Shrubs;On the sampling variance of interclass correlations and genetic 786 correlations;Genotype by environment interaction and genetic 795 correlation of greenhouse and field performance in Pinus contorta ssp. latifolia . 796 Silvae;Forest nursery practice.;Effect of fertilization and watering of Scots pine seedlings on the feeding preference of the pine weevil (Hylobius abietis L.).;Tree planting: not a simple solution;Applied Tree Improvement;Growth response of woody plants to photoperiodic stimuli;Table 1. Traits measured in individual seedlings, grouped into ‘Growth’, ‘Form’, ‘Phenology’ and ‘Survival’. Units for;age of the trees increased and as the number of years between comparisons decreased;through to maturity. The trees in this study have now been transplanted to three field;organisms grow should lead to higher or lower heritability of quantitative traits;Introduction to Quantitative Genetics . 4th ed. Essex, UK: Longman, 1996.;variances when data from 2007 are not included. For traits measured in multiple years, only the most recent year is shown;transplanted to their field locations, as measurements can be made over many more years"
        ],
        [
         "12",
         "0142fc80cb50bb72904e314218eaebac59ddfa68",
         "https://www.biorxiv.org/content/biorxiv/early/2024/05/03/2024.05.01.591994.full.pdf",
         "Biology",
         "bioRxiv",
         "A. Luca,Tine Swartenbroekx,H. Seelaar,J. V. van Swieten,Suheyla Cetin Karayumak,Y. Rathi,O. Pasternak,L. Jiskoot,A. Leemans",
         "40557803,2300078674,4854131,152563647,40067986,1766623,2240377,3501875,2238425153",
         "0",
         "0.0",
         "2025-02-28",
         "2025",
         "f936262abe82472e654a020573d897df67032b73;587b07794826ba1a259b8a81c081be2e45a9d399;a0916d400cf6bbeaa0183e4284c254a3b03dfeaf;f80c6027ff2c0909c3b312dc88c8d582c7bafa46;a5119a9c4e77a55a02fe36ea8ade90bb347d26d2;4e5c9eb2663855ed9ddc2595f909b2562b2abdae;a1f06e9a11c6ce9a4aeb0b16111013c533722b74;9ef5e445034d499713ab66f59bd72ecb8684f408;246e03f202de45ea583e14ad55ce5a5de16a66a8;92a2cf9910a4b9a951450a5f9bd052afb77b3731;01021c0be1f4f5949dd735b9b6d16f7cf3fe4eb2;1ed8e2134e45dace29969012e73d32c66d3b2d43;d6dd530802cbb4e49e785b27163f3055a97a9925;9ec1caf9435cb43a8043da965b194d9bf40c87b6;cacfffc6b93abfd8d83ff12f1f578de240ecf6c3;03a360e29a0388e3aa24e713e57815a4177ccf73;bb94aeae150a91408a83d9d6dd8f4d7653c258ff;27f9fd4a9940f68da8ae507c8fdea8f87d8cb0a2;e83c898be03948e7e2fdf0e489fdb0b835e1bb6f;a48ee96ec29683f8d135985219d6b6c10f643033;5cb233988e355605d3deb74de3055447f239ae29;c795a295222bb65cdc3474b48a4350fe2224a09e;2fb7811eed842bf44eb28491f052f3cd37237249;262293db5668fca6b547d579dfbc1b7929e75cb4;5f60ecf17abd6a1983768bf567257126669aaeb0;87fa2bc1a94d9dc5c0fe57a449f18a606275ecce;37d20f6992bba95a418d60ea32c9c7c141672289;eade28d9ee7bd8acc855d3dbcb5161181a430193;21e4b4fbdcaa90b32c56a87a1896751b559e9b64;2dd32ff6608812d9b5d3877340df1cf9fceda08f;453464f815d29dbec5ff7a5afee5100aafa2baa0;7f5fc390ddbcb8cfdeb9ec7d26a32651ad253d21;dc46f1bb1073d5f533ff2141e650267b9e2124d1;66287e72a366ae6f154b3d76001b5641e65f3750;9a8a061834350b7434bbbdd6ef75aefb39eea38b;f92f4418d4a4523a5760414144ecce8ab6472fde;427eaafcd47b3abc620b953ff021f25925ff0e62;2bab19d59e953df2e28cd9d868e769dfc448cc7f;8cb01b154ae81d71a82bfcbacd6de6761891b208;da586e64082236180bb3f48d664d2c5b7fccaebb",
         "Lifespan reference curves for harmonizing multi-site regional brain white matter metrics from diffusion MRI;Neuroimaging standards for research into small vessel disease—advances since 2013;Abnormal white matter changes in Alzheimer's disease based on diffusion tensor imaging: A systematic review;Improved sensitivity and precision in multicentre diffusion MRI network analysis using thresholding and harmonization;Multimodal tract-based MRI metrics outperform whole brain markers in determining cognitive impact of small vessel disease-related brain injury;Cross-site harmonization of multi-shell diffusion MRI measures based on rotational invariant spherical harmonics (RISH);Diffusion MRI harmonization enables joint-analysis of multicentre data of patients with cerebral small vessel disease;Towards multicentre diffusion MRI studies in cerebral small vessel disease;MarkVCID cerebral small vessel consortium: I. Enrollment, clinical, fluid protocols;Cross-scanner and cross-protocol multi-shell diffusion MRI data harmonization: Algorithms and results;A prospective harmonized multicenter DTI study of cerebral white matter degeneration in ALS;Generalized Richardson-Lucy (GRL) for analyzing multi-shell diffusion MRI data;Cross-scanner and cross-protocol diffusion MRI data harmonisation: A benchmark database and evaluation of algorithms;Scanner invariant representations for diffusion MRI harmonization;Retrospective harmonization of multi-site diffusion MRI data acquired with different acquisition parameters;Inter-Scanner Harmonization of High Angular Resolution DW-MRI using Null Space Deep Learning;Presymptomatic white matter integrity loss in familial frontotemporal dementia in the GENFI cohort: A cross‐sectional diffusion tensor imaging study;On modeling;Current Clinical Applications of Diffusion-Tensor Imaging in Neurological Disorders;Harmonization of cortical thickness measurements across scanners and sites;Harmonization of multi-site diffusion tensor imaging data;The importance of correcting for signal drift in diffusion MRI;Denoising of diffusion MRI using random matrix theory;Inter-site and inter-scanner diffusion MRI data harmonization;The effect of Gibbs ringing artifacts on measures derived from diffusion MRI;REKINDLE: Robust extraction of kurtosis INDices with linear estimation;Lifespan maturation and degeneration of human brain white matter;Faculty Opinions recommendation of Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data.;Diffusion MRI at 25: Exploring brain tissue structure and function;A reproducible evaluation of ANTs similarity metric performance in brain image registration;Microstructural maturation of the human brain from childhood to adulthood;Empirical Bayes Methods;Harmonization;Toward a quantitative assessment of diffusion anisotropy;Matter;multimodal brain 3 T MRI harmonisation approaches;presymptomatic familial frontotemporal dementia;ExploreDTI: a graphical toolbox for processing, analyzing, and visualizing diffusion MR data;Adjusting batch effects in microarray expression data using empirical Bayes methods.;Microstructural and physiological features of tissues elucidated by quantitative-diffusion-tensor MRI.;MR diffusion tensor spectroscopy and imaging.;Spin diffusion measurements : spin echoes in the presence of a time-dependent field gradient;Submitted to Magnetic Resonance in Medicine Corresponding author: Alberto De Luca (a.deluca-2@umcutrecht.nl) Word count: 3924 words;Discussion: RISH-GLM can learn cross-site harmonization both from matched and unmatched groups training subjects, and can effectively be used to harmonize data of multiple sites in;This study is funded by the Bluefield Project to cure FTD;determined with RISH-GLM on data from all three sites in Experiment 3"
        ],
        [
         "13",
         "0155e40ceb6ddc83a93fb431cc86ebe135079cab",
         "http://arxiv.org/pdf/2408.03797",
         "Computer Science",
         "Quantum Information Processing",
         "Yao-Kun Wang,Li-Zhu Ge,Tinggui Zhang,Shao-Ming Fei,Yufeng Gao,Zhixi Wang",
         "3398784,2152266603,2258620721,2238169673,2342246044,3398895",
         "0",
         "0.0",
         "2025-01-24",
         "2025",
         "",
         ""
        ],
        [
         "14",
         "01891feafb883a81f4c532208e34dcb5806d8ca5",
         "https://www.researchsquare.com/article/rs-5304452/latest.pdf",
         "Medicine",
         "International Microbiology",
         "Mehmet Aytar,Demet Yalçın Bingül,M. Touray,D. A. Uygun,Gamze Başbülbül",
         "2281168561,2344980752,89314229,8050600,13894937",
         "0",
         "0.0",
         "2025-02-12",
         "2025",
         "",
         ""
        ],
        [
         "15",
         "01ae6d81c72cb5b14ae95519f29011609a777351",
         "https://link.springer.com/content/pdf/10.1007/s00442-025-05677-5.pdf",
         "Medicine",
         "Oecologia",
         "David P Gregovich,Gretchen H. Roffler,Christina M Prokopenko",
         "2311799755,3952003,2347526811",
         "0",
         "0.0",
         "2025-02-27",
         "2025",
         "",
         ""
        ],
        [
         "16",
         "01c65b584d35362abc5f2425273ec2eefc579594",
         "https://www.biorxiv.org/content/biorxiv/early/2023/08/19/2023.08.11.552961.full.pdf",
         "Biology",
         "bioRxiv",
         "Germán Robert,Alejandro Enet,Laura Saavedra,R. Lascano",
         "120334882,2197261055,3851119,4069239",
         "0",
         "0.0",
         "2025-01-02",
         "2025",
         "",
         ""
        ],
        [
         "17",
         "01e2c0cb2a8adced75528452c58dc12ae704f755",
         "http://arxiv.org/pdf/2501.02118",
         "Computer Science",
         "Asia and South Pacific Design Automation Conference",
         "Kevin Lopez,Amin Rezaei",
         "2338832390,2338833028",
         "1",
         "0.0",
         "2025-01-03",
         "2025",
         "16b3258cd43cfb939d486a99f40fe6657dfe488f;176a3b504fcd7a0ccbc932c0eba393602abb96eb;4b5979a035917e9538db26cf1f0b39e84cfb90dc;889a05294bd1246876a3befd64e3b2ab7474ddac;ab735a16709a3d96e9fc101b380e0db9f30afa90;a1b4c387fcfc80b22fd588b464d656701f95cef1;40ceecc03c580c9eecd0906589ce868832b99d78",
         "STATION: State Encoding-Based Attack-Resilient Sequential Obfuscation;Machine Learning-Based Security Evaluation and Overhead Analysis of Logic Locking;CoLA: Convolutional Neural Network Model for Secure Low Overhead Logic Locking Assignment;CAS-Lock: A Security-Corruptibility Trade-off Resilient Logic Locking Scheme;Anti-SAT: Mitigating SAT Attack on Logic Locking;CycSAT-Unresolvable Cyclic Logic Encryption Using Unreachable States;Unveiling the ISCAS-85 Benchmarks: A Case Study in Reverse Engineering"
        ],
        [
         "18",
         "01ee76d50524c8842f3ce68ef656f630485cbec1",
         "https://journalofbigdata.springeropen.com/counter/pdf/10.1186/s40537-025-01063-3",
         "Computer Science",
         "Journal of Big Data",
         "Thompson Stephan,Padma Priya Dharishini Paramana,Chia-Chen Lin,Saurabh Agarwal,Rajan Verma",
         "2148717073,2345748099,2345756660,2343606498,2345723398",
         "0",
         "0.0",
         "2025-02-15",
         "2025",
         "",
         ""
        ],
        [
         "19",
         "02005c9e65bcf62c329533a372895f68b739c991",
         "https://www.biorxiv.org/content/biorxiv/early/2024/12/04/2024.11.28.625676.full.pdf",
         "Biology",
         "bioRxiv",
         "Luca Manciocchi,Alexandre Bianchi,Valérie Mazan,Mark Potapov,Katharina M. Fromm,Martin Spichty",
         "2279404745,2333606643,2255647838,2333597038,2249337375,2245105965",
         "0",
         "0.0",
         "2025-02-23",
         "2025",
         "8236cad76ac9ced26ffbc7469121d101ad37e15c;6518433cc4cd3a1dcb757d838ee4854a52249425;b044ffb2a8b548f5b6c8c2e0eb12dcec9fc36e29;59ec593117b385383de56fbfb9850c174f01725f;3c58224850bc9cefbad20e634846278f02de7c73;1c8ccd1488f6d592a65bd4d022eef2969c0aa668;8169ff5bc49e482ef70153dc7b2a56b44eea6d53;4ff4ccc14e1de6440cff57367a4d0ea71ea058a3;01cc10a58d743140037b11b1f33a6a4a842b7d2c;0cbea48e04da728afdf0591c30055e2562bade95;d0ca22248bc835be5d7e9d0af8278f8f994e5ec8;2012be16842b142c3868ebab8e6caa70a05e91c3;bb673759c421a48a9ee9a070d984aebf09f96fed;d8a8be809f95068a8a5e2839666d45209a666b76;87c6156fb290cf3914cd3ed18590200b9a72627a;58172dc45aaacf8a221767401be96aa8c9803ef0;5c15d0ecf392ebe4790e03882f7982709ac712b8;4f2556e717327d18bf6e74ad5beeddfa80382d16;2b9c74e76f86c896db9936e2d0546ffd9dac0fb3;12111256cc53f82a65b8d1c284309754f8a0d596;b08092845d60952fd5d79daf7ee72397f9c35d47;2e4e2fb0d08e93ee597381b3c7225a0f49c18ec1;8a258fe707d1cf444ba506d9e818a1329dd91ea6;4fedae8329960a9b1d1a5f71776a11269bb75583;4a3f9a1d66cb41689b2c8226e643d2eb92b06be4;6028582733d89bbaf0be2ea28dd954fd0f64ebc8;f29fa405bf383ca96df2bc78f13d713f8841a443;3555543153d42636bc967a93ede43e4f0f982391;3390f1f37196a2fafebdab8434a5e4f52456d441;1829d212c1a0f73a34b70ec87ae76123e91a0308;ab06bf6c7c3f7d6db066fcffcd4e06e9b0045b93;96b806a4a3909be212c3d2d14df20665e29ec01b;3e91f8caa53385dc4256634638431d10a8c9173a;ef513693e0885c35483c2264fe6b410f4c07b5b5;d3064425f4110faef5efbfbdc06df968ed91a414;149dc47112d54760cd7c14ef78090816ed56eb46;5a8b899b8edaae8c4c7e662c431360be986ad84d",
         "Thermodynamics of Metal-Acetate Interactions.;The battle for silver binding: How the interplay between the SilE, SilF, and SilB proteins contributes to the silver efflux pump mechanism;Accurate Metal-Imidazole Interactions.;Constant pH molecular dynamics simulations: Current status and recent applications.;Re-sensitization of mcr carrying multidrug resistant bacteria to colistin by silver;Scalable Constant pH Molecular Dynamics in GROMACS;Computational strategies to model the interaction and the reactivity of biologically-relevant transition metal complexes;Alpha-helical folding of SilE models upon Ag(His)(Met) motif formation.;Configuration-Sampling-Based Surrogate Models for Rapid Parameterization of Non-Bonded Interactions.;Model peptide studies of Ag+ binding sites from the silver resistance protein SilE.;CHARMM36m: an improved force field for folded and intrinsically disordered proteins;SilE is an intrinsically disordered periplasmic “molecular sponge” involved in bacterial silver resistance;CHARMM TIP3P Water Model Suppresses Peptide Folding by Solvating the Unfolded State.;Enhanced sampling techniques in molecular dynamics simulations of biological systems.;Systematic Parameterization of Monovalent Ions Employing the Nonbonded Model.;Recent Advances in Polarizable Force Fields for Macromolecules: Microsecond Simulations of Proteins Using the Classical Drude Oscillator Model;Force Field Independent Metal Parameters Using a Nonbonded Dummy Model;Nanobio silver: its interactions with peptides and bacteria, and its uses in medicine.;Force field for monovalent, divalent, and trivalent cations developed under the solvent boundary potential.;Syntheses, structures, and antimicrobial activities of remarkably light-stable and water-soluble silver complexes with amino acid derivatives, silver(I) N-acetylmethioninates.;Constant pH Molecular Dynamics in Explicit Solvent with λ-Dynamics;Current status of the AMOEBA polarizable force field.;Antibacterial Activity and Mechanism of Action of the Silver Ion in Staphylococcus aureus and Escherichia coli;Atomistic Simulation Studies of Polymers and Water;The Simple Yet Elusive Crystal Structure of Silver Acetate and the Role of the Ag−Ag Bond in the Formation of Silver Nanoparticles during the Thermally Induced Reduction of Silver Carboxylates;Calculation of absolute protein-ligand binding free energy from computer simulations.;Bacterial silver resistance: molecular biology and uses and misuses of silver compounds.;A mechanistic study of the antibacterial effect of silver ions on Escherichia coli and Staphylococcus aureus.;Synthesis and characterization of water-soluble silver(I) complexes with L-histidine (H2his) and (S)-(-)-2-pyrrolidone-5-carboxylic acid (H2pyrrld) showing a wide spectrum of effective antibacterial and antifungal activities. Crystal structures of chiral helical polymers [Ag(Hhis)]n and ([Ag(Hpyrrld;Critical evaluation of stability constants of metal-imidazole and metal-histamine systems (Technical Report);THE weighted histogram analysis method for free‐energy calculations on biomolecules. I. The method;STEREOREGULAR COORDINATION POLYMERS FORMED ON BINDING OF PEPTIDE-BASED POLYDENTATE LIGANDS TO SILVER(I) AND COPPER(I) - X-RAY STRUCTURE OF ([AG(N-[N-((5-METHYL-2-THIENYL)METHYLIDENE)-L-METHIONYL]HISTAMINE)]+[O3SCF3]-.MEOH)INFINITY AND A SOLUTION STRUCTURE STUDY;Comparison of simple potential functions for simulating liquid water;Nonphysical sampling distributions in Monte Carlo free-energy estimation: Umbrella sampling;NIST SRD 46. Critically Selected Stability Constants of Metal Complexes: Version 8.0 for Windows;unpublished results;Theoretical Studies of Hydrogen Bonding;Comparison of the ligating properties of disulphides and thioethers: dimethyl disulphide, dimethyl sulphide, and related ligands;Sur Le Mélange Des Gaz;Ueber die Anwendung des Satzes vom Virial in der kinetischen Theorie der Gase;Using the procedure given in the section 2.2, the PMF and binding constant are calculated;We performed MD based Umbrella Sampling (US) simulations in explicit water to sample along the reaction coordinate of binding, i.e. the distance between silver(I) ion and the coordinated atom;The Potential of Mean Force (PMF) was determined along the binding reaction coordinate through WHAM;From the PMF, the binding constant log 10 ( K bind , calc ) was calculated (and, consequently, the binding free energy ∆ G bind,calc );Toward Predicting Silver Ion Binding in Proteins, under Revision;If the minimum of the PMF matches the target distance"
        ],
        [
         "20",
         "022103bcb2f73b22a217aaa1bb92d4f8df4a75c2",
         "https://www.biorxiv.org/content/biorxiv/early/2022/10/12/2022.10.10.511559.full.pdf",
         "Biology",
         "bioRxiv",
         "J. Bunce,Catalina I. Fernández,Caissa Revilla-Minaya",
         "33949687,144155243,1405019368",
         "0",
         "0.0",
         "2025-01-02",
         "2025",
         "",
         ""
        ],
        [
         "21",
         "0223a10fa0f31b13cd2ae9f43bad026285be5b85",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/09/2024.08.07.606870.full.pdf",
         "Biology",
         "bioRxiv",
         "Nima Zargarnezhad,B. Mesquita,E. Macpherson,Ingrid Johnsrude",
         "2315918382,2236443166,8602213,2265497669",
         "0",
         "0.0",
         "2025-01-31",
         "2025",
         "7de002adc188b5465f319a13c28a9e25de0a115d;b5f814c704b2206690a4d526a3dda24e3dc3dcbc;91163586f5c1ea0370ef912c9fbc78c72e9ccd8f;a28d35f6e2af7347861a808eeb1346f13ac9bf9c;592e51212d752a646c7ea3e823510e9380524da5;adea7833fa0a1d7db10111ddab8bcb3fdc0f6a51;88b56468d959caa3eb242b9fa344a83955ac7f70;37ac61c227e1a5bde47189422880d1a56eff4f6e;59ab8c9ef50cd0aad46a39f57472e8ec21b18280;d38d9b8a5f4ce487e110a8769324e242f7856405;6fdada7d918cc216c433231b5b78763a98c496f5",
         "Spectral weighting functions for lateralization and localization of complex sound.;Spatial variation in signal and sensory precision both constrain auditory acuity at high frequencies;Investigation on localisation accuracy for first and higher order ambisonics reproduced sound sources;Head Circumference in Canadian Male Adults: Development of a Normalized Chart;Rapid Formation of Robust Auditory Memories: Insights from Noise;Listener weighting of cues for lateral angle: the duplex theory of sound localization revisited.;Reproduction of a plane-wave sound field using an array of loudspeakers;Uniform spreading of amplitude panned virtual sources;Spectral cues used in the localization of sound sources on the median plane.;The sense of hearing , (Third edition. | Abingdon, Oxon, New York, NY: Routledge, 2018.;Fundamentals of binaural technology;MINIMUM AUDIBLE ANGLES FOR HORIZONTAL, VERTICAL, AND OBLIQUE ORIENTATIONS : LATERAL AND DORSAL PLANES;higher-frequency ITDs are not relevant for perception and so these high-frequency;block keep them focused on the task. The data were collected within one session of 1 hour duration;at the spectral power peaks at ~8 kHz in FIG 6.A).;reported the burst with less power (i.e., the ambisonically rendered burst) as lower in elevation;average low-frequency ILD was -0.55 dB when the source was at the center (reference point C);even higher order systems may not perfectly reproduce sound locations: errors in 66;to frontal HRTF magnitudes previously reported 1 . At all locations, within each channel;its values are around 0 dB confirms the equivalence of VBAP and SC for locations;0 everywhere, confirming the equivalence of VBAP and SC presentation methods for veridical LSP locations;higher, we could still ask whether there is a difference in the degree to which wideband 708"
        ],
        [
         "22",
         "0230eb8423f14aa38144acb3a3d1fb105857fdd2",
         "https://link.springer.com/content/pdf/10.1007/s11357-025-01564-2.pdf",
         "Medicine",
         "GeroScience",
         "Zachary D. Von Ruff,Matthew J Miller,Tatiana Moro,P. Reidy,Scott M. Ebert,E. Volpi,Christopher M Adams,Blake B. Rasmussen",
         "2256963685,2257132225,2252932126,6441573,3681898,6567558,2256968103,2256965936",
         "0",
         "0.0",
         "2025-02-27",
         "2025",
         "",
         ""
        ],
        [
         "23",
         "023287f776585ce5ae13285b8949f1ba4cde871a",
         "https://www.mdpi.com/2076-328X/15/2/137/pdf?version=1737888395",
         "Medicine",
         "Behavioral Science",
         "F. Costello,Cheong-Gi Kim",
         "1391664016,122552573",
         "0",
         "0.0",
         "2025-01-26",
         "2025",
         "",
         ""
        ],
        [
         "24",
         "02365f56e2e4b54038afd9a17e398ee90c297288",
         "https://www.biorxiv.org/content/biorxiv/early/2023/10/23/2023.10.20.563281.full.pdf",
         "Biology",
         "bioRxiv",
         "Florence D. Hulot,C. Hanot,S. Nélieu,Isabelle Lamy,Sara Karolak,Ghislaine Delarue,Emmanuelle Baudry",
         "2261896957,153014326,11791277,2258625228,9871307,49367724,2261896913",
         "0",
         "0.0",
         "2025-01-24",
         "2025",
         "",
         ""
        ],
        [
         "25",
         "02405247da19656e1027e2575504ba54a65dc7fa",
         "https://www.biorxiv.org/content/biorxiv/early/2024/04/01/2024.04.01.587545.full.pdf",
         "Biology",
         "bioRxiv",
         "Kerem Uzel,C. Grossen,F. G. Çilingir",
         "1394883938,4896019,12808627",
         "0",
         "0.0",
         "2025-01-18",
         "2025",
         "",
         ""
        ],
        [
         "26",
         "0248ab98531b074b916d3f3a0e63bd84c51cadce",
         "https://www.researchsquare.com/article/rs-2168273/latest.pdf",
         "Medicine",
         "Oncology Research",
         "Zhengyang Zhou,Lei Qiao,Tongtong Wang,Wen Pan,J. Duan,Haiyang Zhang,Ting Deng,Yi Ba,Yi He",
         "2118666646,2308329182,2278987538,2262731575,1750651,48212725,2244637962,2241258736,2242391301",
         "0",
         "0.0",
         "2025-01-16",
         "2025",
         "b949f8505e56ee01b05a97a7d072f1d83c699f41;f5c17cea915ddb4bb37336a08191a0f29dd4d2d7;4a24a7622b26fab273c8c08d46ca5d7a519468f4;373d8b09597d02521669fec8c6cbff8d638000c0;d4e50273afe92e9eb1b4710453e896e4c6bd5133;566594cd02ce4efca6bd42f6c2fd0ac8c36fc0a2;8e3f43da5bc6e859b0ee0c59410bb2a272a53cf5;0266144687df5c7c6f07c3c10364893b7179e7e1;6d3d9772dcd42fe7a8e72b9d424b4ec03960de42;9ceba45eb0ed07dd9b89f4e8dd337da064f4b60d;493ef66411dd833b8a083dd0ca84473bc59a98b4;474f691172792ba6b47c8a3010b7b6db8a4c9a48;7812d3667a8c7afaaaf2eb1f1d639d534392a81a;69216c23b3ffeab70a49d16967c4079efcf7e061;f460960774b5ca66d05cbce3f8cb263d329aef23;21b09b8427a0e784bf4e68920cef5099384646c3;67156cb733bf5bffe687e739835299a3318bac2a;b28893da7a1169fa48eff01f6649536bf26d9187;ac5154c68f01e1618031b82ea5ef48f061d8e14b;a92fb26a0f0ef1a02f76d7921eb31a3c15e8c9e1;c4a059db8d2960c02f4f4aeaf77fabe9ef792b7e;a22ea4956cd8af90fde710edceda3264af1bec3b;9e51f7a51e1be156639884f4502f9f0ff58b65d1;319a2c8807749b532b88cc712ca3af04ef33185e;74a6c3fc4f06669076dda2f9bd5b8ce444f6c6cc;5846f751701ac6096b7d2990daf08ef7cc15a997;a1ecfe4c3845890f63d34b828ded715f68843647;1e86143310006a70825c9d6657f4f9c5d8d3693b;1f85c025854714b693372447edf42d55c18770c8;23792a8b24be500d4a4cec391edcf416bf0f68ed;da2fe3219ce3d1c8515224c31f4e4afa8b258bd5;92492584265828b81bc821b0cadb04ff16f2d3fc;99a47a2ad804e536d28fd6dabacb5e7484ee8648;29645e32c226a94f9e8a5bfbaf0cc399538ad2f7;e44e817a35b39bfb3b90d3c65a97b3838c39ec77;1e4a5b9f2340c734db82df76fa0d48ea946dc7bc;ea3b02f90fdfc085562126d1768f35b748f92802;159c1a738bcc3b310b3e2abcf0331d7d6afacbb2;5243c2a5e9f844f2066a343b34a10a4fd48113e0;1071ea1d3f24f939f09122856dfe3ba50c611f99;94a768196ff741b4682db5d47e1d8ddfbdf73d2a;129430f9316981cede2c185bededbe31b0de79e2;7eed78bb3f352288bef67e4872549cc4b200e832;26c76c04f1261f79c788be80331b3560b4702b03;0e8e44b4a3b278f024dd1be8aa55de23a0958e1d;739f00971cb4c47c8882fbf8eb8b19dfa21c3060;7582f7ee49601271ab92c2e987bcd2c7115829ac;8ed9b235694ce240dad068d6b7664fc3b18e167e;713595be11d256112e4cbb3b19df55a2d432b416;989b9bd66fecdf62494a1aef6c721dea257b169b;0a6c4dd0670f75e661f3e5d0e39a6f41f9099b9d;c2145d4aec4c6f7ac0272cef320fffee65ff78b7;2b14d0743cda875e9eaa4e28cf38543b4f87ace8;57fee5a78fff9f528569e4af8f377adcac189e01;2168f52d04ad2def712991ee5d147b17cd47bf73;bc24b34428ba6b080cf097bfec1c16452db4ad1e;06354c7890d943c72898399c3d0af9e16645cc9a;40657abb9c4b32e70e50bfd5049b7d66f9ccc19e;7ca98dcdc37fc140d36a731e337a489059813ae8;2f65c8e07b7088ffb532d6bbaf6c212275e87628;2af09abd200cbdd11e1b0e9fd69fafe08e77415a;08cf5d31e84756276460afbfc50bcc24e8ea4eb0",
         "SLC6A14 Depletion Contributes to Amino Acid Starvation to Suppress EMT-Induced Metastasis in Gastric Cancer by Perturbing the PI3K/AKT/mTORC1 Pathway;Low Expression of miR-491-3p Is Correlated with Lymph Node Metastasis in Gastric Cancer;β-Catenin signaling in hepatocellular carcinoma;The role of Exosomal miRNAs in cancer;Lymph Node Metastases from Visceral Peritoneal Colorectal Metastases are Associated with Systemic Recurrence;The Chinese Society of Clinical Oncology (CSCO): Clinical guidelines for the diagnosis and treatment of gastric cancer, 2021;Role of lymphatic endothelial cells in the tumor microenvironment—a narrative review of recent advances;Current status of lymph node dissection in gastric cancer;The impact of thyroid tumor features on lymph node metastasis in papillary thyroid carcinoma patients in head and neck department at KAMC: A retrospective cross-sectional study;Lymph node metastasis-derived gastric cancer cells educate bone marrow-derived mesenchymal stem cells via YAP signaling activation by exosomal Wnt5a;Current treatment and recent progress in gastric cancer;Global Cancer Statistics 2020: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries;RNA packaging into extracellular vesicles: An orchestra of RNA‐binding proteins?;The Roles of Non-Coding RNAs in Tumor-Associated Lymphangiogenesis;Ectosomal PKM2 Promotes HCC by Inducing Macrophage Differentiation and Remodeling the Tumor Microenvironment.;Exosome-mediated transfer of miR-93-5p from cancer-associated fibroblasts confer radioresistance in colorectal cancer cells by downregulating FOXA1 and upregulating TGFB3;PCDHGA9 represses epithelial-mesenchymal transition and metastatic potential in gastric cancer cells by reducing β-catenin transcriptional activity;CAF secreted miR-522 suppresses ferroptosis and promotes acquired chemo-resistance in gastric cancer;The biology, function, and biomedical applications of exosomes;Exosomal long noncoding RNA LNMAT2 promotes lymphatic metastasis in bladder cancer.;Extracellular miRNAs: From Biomarkers to Mediators of Physiology and Disease.;A novel role of HIF-1α/PROX-1/LYVE-1 axis on tissue regeneration after renal ischaemia/reperfusion in mice;Glycogen synthase kinase 3-β inhibition induces lymphangiogenesis through β-catenin-dependent and mTOR-independent pathways;Deciphering role of FGFR signalling pathway in pancreatic cancer;Ki-67 protein as a tumour proliferation marker.;Exosomes in gastric cancer: roles, mechanisms, and applications;TRAF6 inhibits colorectal cancer metastasis through regulating selective autophagic CTNNB1/β-catenin degradation and is targeted for GSK3B/GSK3β-mediated phosphorylation and degradation;CCL27/CCL28-CCR10 Chemokine Signaling Mediates Migration of Lymphatic Endothelial Cells.;Exosomal miR-196a derived from cancer-associated fibroblasts confers cisplatin resistance in head and neck cancer through targeting CDKN1B and ING5;Stabilization of miRNAs in esophageal cancer contributes to radioresistance and limits efficacy of therapy.;Circulating exosomal noncoding RNAs as prognostic biomarkers in human hepatocellular carcinoma;Cancer-derived exosomal miR-25-3p promotes pre-metastatic niche formation by inducing vascular permeability and angiogenesis;PROX1 Is Associated with Cancer Progression and Prognosis in Gastric Cancer;Effect of exosomal miRNA on cancer biology and clinical applications;Complementary Wnt Sources Regulate Lymphatic Vascular Development via PROX1-Dependent Wnt/β-Catenin Signaling;Cervical squamous cell carcinoma-secreted exosomal miR-221-3p promotes lymphangiogenesis and lymphatic metastasis by targeting VASH1;The chromosome 11q13.3 amplification associated lymph node metastasis is driven by miR-548k through modulating tumor microenvironment;Tumor-derived exosomal miR-1247-3p induces cancer-associated fibroblast activation to foster lung metastasis of liver cancer;Lymph node staging systems in patients with gastric cancer treated with D2 resection plus adjuvant chemotherapy;Exosomal transfer of tumor-associated macrophage-derived miR-21 confers cisplatin resistance in gastric cancer cells;Pyruvate kinase type M2 promotes tumour cell exosome release via phosphorylating synaptosome-associated protein 23;Positive regulation of β-catenin–PROX1 signaling axis by DBC1 in colon cancer progression;Cell-based approach for 3D reconstruction of lymphatic capillaries in vitro reveals distinct functions of HGF and VEGF-C in lymphangiogenesis.;Advanced gastric cancer: What we know and what we still have to learn.;Glycogen synthase kinase-3 (GSK3): regulation, actions, and diseases.;VEGFR signaling during lymphatic vascular development: From progenitor cells to functional vessels;Exosome and Exosomal MicroRNA: Trafficking, Sorting, and Function;Transcriptional regulation of podoplanin expression by Prox1 in lymphatic endothelial cells.;Gastric Cancer: Descriptive Epidemiology, Risk Factors, Screening, and Prevention;Sumoylated hnRNPA2B1 controls the sorting of miRNAs into exosomes through binding to specific motifs;GSK3: a multifaceted kinase in Wnt signaling.;Lymphangiogenesis: Molecular Mechanisms and Future Promise;A GSK-3/TSC2/mTOR pathway regulates glucose uptake and GLUT1 glucose transporter expression.;Clinicopathological evaluation of biological behavior of submucosal invasive gastric carcinomas: relationship among lymph node metastasis, mucin phenotype and proliferative activity.;TSC2 Integrates Wnt and Energy Signals via a Coordinated Phosphorylation by AMPK and GSK3 to Regulate Cell Growth;Vascular endothelial growth factor-a promotes peritumoral lymphangiogenesis and lymphatic metastasis.;PDGF-BB induces intratumoral lymphangiogenesis and promotes lymphatic metastasis.;An essential role for Prox1 in the induction of the lymphatic endothelial cell phenotype;complementary and alternative medicine: eCAM;A MicroRNA signature identi ﬁ es pancreatic ductal adenocarcinoma patients at risk for lymph node metastases;An unexpected role for EGF in lymphangiogenesis-mediated melanoma metastasis to sentinel lymph nodes.;Lymphangiogenesis and cancer metastasis.;Accurate Lymph Node Staging is of Greater Prognostic Importance Than Subclassification of the T2 Category for Gastric Adenocarcinoma;[Gastric cancer]."
        ],
        [
         "27",
         "0296022d83ce83107b2e66ba8bf405e567b47269",
         "http://arxiv.org/pdf/2502.09118",
         "Engineering",
         "IEEE Communications Surveys &amp; Tutorials",
         "Qinwen Deng,Yao Ge,Zhi Ding",
         "30581393,2288399468,2147265182",
         "1",
         "0.0",
         "2025-02-13",
         "2025",
         "",
         ""
        ],
        [
         "28",
         "02a6b974397afa97ee32a32b7a655f2aed513313",
         "https://www.biorxiv.org/content/biorxiv/early/2023/01/04/2022.10.26.513870.full.pdf",
         "Biology",
         "bioRxiv",
         "Giuseppe Deganutti,Ludovico Pipitò,Roxana-Maria Rujan,T. Weizmann,Peter Griffin,Antonella Ciancetta,S. Moro,C. Reynolds",
         "3408926,2175571400,67145659,3756944,2189229376,2395552,145581962,2248226941",
         "3",
         "0.0",
         "2025-02-17",
         "2025",
         "a993c48dece2cd10252d05ca3c7708b0408754b1;151918d04abf8d14784ed9a37e39f39d9d76b497;f48b04b5fae34ec7a66990759e327e91531d030f;438fcc621c0370750bc7db61e972c266f6cbbc45;bc3570ed7b8cc02dc061c11f75fad9055b47fb52;e5eca5192c58486f7a7bf869e79ffcecb52c47c6;1ee21fb0597973a6a3cd9379b00b6e30b09cf6d4;5d3d90b19775c64fd00a4728cf1415e0adab2192;d4422caeb69acf5a4be7c861318def610fd35b4a;dc32a984b651256a8ec282be52310e6bd33d9815;430ba46e178c491225c1dd673ae97742a3fab9f1;f81423add18c9e2886eb334b642e8f3f47785efa;56ac21b131a132784b1635464d2f86ec897e5170;a4607a83a2479e2654ea490b5c106b631feddf5d;2c5e20b7d84910ed0414972ed2b64c9be648d225;a72ed79136a937999209eb9ed654b166fcd8c95b;6a33a787d61545591aca4430e8f17ce068cff58c;f40f5dcbdf1c074a1b46233db209e38403c0f040;2ca5e766f1539fd70137c5365cdb8e98dea0a8f7;ac85cf3f36a7eb7b3b9897352d1180e2860df1bf;db18cbf211ba42ec9c9a22361c7501f84100c450;b42d5d66bfb3253cb6f3ab3d8261e3aba2103cdb;2a9249d73d461ed9b72d97b19c65d610a23e1736;7b8e69dfd8f5774b98e7adb42e36f95187f7787e;77ad45eac173fdafa9210696fcf61c4edd0feb07;4678414beaeb3100934c8e5aa76438223e6e570d;4453c8bc88a326988ec742d4ffa9d75d24f3334c;b82648533c62326ef616c46d065cd08b680f6bdb;bf426148bb70f4757a12534f1faf3661689149e8;b8228578c44d89889c8e6f680964e6dcabe86636;9d320e5e60f966335f51445a605de8412c9ffc06;9de06a34f09f77d984337821f1abd04dba2173fc;4e888df7af42d205b27e5d10c9b5b57e32d47db3;88258494e34880631bbfe8d9f11cc0324830f360;42434ae4f03476a6c09c01e5d1fe82512fc2d5a9;adba1a42c43f8d5cf9fe68fdf71c9d10b4b825c4;9363333f4da5ae501fc7f6816ad1dc71b9f7c53a;877221ceb4652438dc20a0895373f01cb43543ec;4bcd6f1ca99842b401c194c5b21dd9ca548b9c13;d6a49514a5163bdd997dfc5c12e13d1866767c46;8177381a29ee8f0133072d0995bc0a8393db7d19;61dd01731b4888eaf5071336bcf56f087cbeb08e;7bd5beeabac59fae3636592dd9845561f5d00b1f;94dd7a2ac0b7ce0e7637f07419efd5cec42db3d1;59b2f3d4737ed246c8abdc3917c384e32615178b;5310476b7688850b1c163cc88b15b2190661defd;aaa66e8729fbd071781f23c4239eff9995054019;5e65b9e79eb72718eb83d6a3b45999948cf9a2b3;260f212da0cf8b3309f0ab3a130a0ad22993892e;49d40250c1bf1eb95b5c8caa3c2f663f336f117d;a4468501c28802c6bf295001e92d825c6d59963e;2456c2bab5c3591518db6c4d8f81719240413bf8;a16930753f0f12d04a108f3afc2e72c23670ad01;4d03e2e39386ff933830af152e9185d33b5db98b;f2555cb8ee6147e18416ef7c37883ab09a9a027c;6a1e5f5a24015b5db0e21be6604274b1a57f37e0;14f67e7d8854b472ca6435fc6a23fa7ce7e3cac1;91d4c8e1167f50b90a4f3f936af5a57bec7a3058;d0ca22248bc835be5d7e9d0af8278f8f994e5ec8;40a1d9b4bf0d36f112ee0ef46f6c7bd9f9a66401;0b638137484590551b94db3164ac3092fd9190b3;f22f0ba76c1b5d2e85250dbb8833a654b02e39e8;c90d4fecf86fdb1115cc77113c8581c498edeeda;0a1d473b76802804a5c5af08d53d8c8c89bbe78a;e2d0b376ebbda89b90fe90882906d2e43e14f664;e0dcbf2f1e47bee3da8524305dcdc9eb4a1a44e7;1ff445708caa52b9232b6851cff28c12caf81d8a;20ba7ceccf717b056a036754be93ca247430de9a;04164358bcefd96ac953824e1de9f95de69d72be;8bbcdbcf8f589715e06778283dc1e2cd623fe6ad;22a2a8eb65c1bb217fc2e335fe2f6bce43c1615f;ca0fbeeab0ef281be949d75e8439d92f2a13378a;a56eae356b1acdc7884cd0a4d6c915d9a5351ff4;888d0c64db910cdf339447036830d93ae7dbd4b8;2be2c501e8803d18d9c07997485fee54b89cf55f;f3f775a832129840066b92c46ce1d0cf13b13cf6;8ea40128aabed0ecd5108cd34eefaf161e923d37;cc04a4a2970d3dcaf22ba469feddc40ad696fe32;e0674c46f6d2cdc3c5a005176d4f5a13a016e10f;b98cfb73824ae44676a61cbe321a118821512fa3;8f84236748e7480f473a15b25c7cec5a1b0f3b5b;d7ba2f187db8cbfcd3266fb232f7fed4198d71c7;c1dfe256cca07edbb255c4674bf57ff3c4ad6a05;5693ac9e2cc3a4af6294a45d921c666377355309;37f656c902e2c8cbfcc6c52b299722b25eb600ff;0a03f2cf01537d20c42f5173177888f8a1890716;cd863c68bf75abf5783fdbb52a3776e7152308fb;1fb4d85ebae0b7daf59456daed7b1143e2df007a;c8fbd2e8a5b818cd3ee0748795282be58647b883;c1551fd631c26d783059c1c2c51af7b97e02fd22;9a0557d4cd6506ff5903670f75bb6df7fbed58e4;9c006b7d672f2908e32303d7b11de05b9fe3562e;d7003bd90a399d7997bffc8531495d8160abc682;42a7c3a80cf6ab712b461502fbff37700e2ea74c;101dba7d3c94f59f34eacd6893a02ba647b1ebd5;7fcd5fe69ddd9ba4175c95c71fa6e04dfa4e483b;3606a711c414b3058344e344a68675d08d4da354;874a2afd75a5855d144fce12cae9f9a9d288b386;9fb9d52747b5937a31a67a76cd3d7a50ecbe1a82;bb60cb3697cb2c365c8f057b690a5b43145e0210;e3f9ed5a6c1c2045adfb5f66845338d9352149d3;04c69e602946dc56b985ad51aeeae403fbf04802;acd8aefad9e84192bb12c855d5e2fb164501a457;895860c6083736508d2541900cdf0960eb11592f;6ff68e8af2fb9a03ab3035628929d9f970772b65;05647876ef826f0e28b75b1ee111ea631160a9aa;0d743061cbb5d454947fb76d5a11f39e754f9175;2111193ae2067ff3190ff16e69fb8ac573db3217;a4ffdf18203baa73a983a2a3dc0b6adbe764c72a;0cfd8229fcd87c6f9545d9cce08d882f93e9eacd;995719871e89a8ab7f6a3ebca13b61aaf6f1f7b8;28e6284c59a9bce5059def34ae9ece76fe8e2b5f;5a7928df576c828fdb065fe33338a79e0b6a5d99;3e979d47c5f9fc4b876831af20e8574bfaf5c055;6b309e645e740f96392f5432fa3be4187ae6d6b0;8612a028cafd8be801c830708ae0ef1ea6ad582e;adff5f3d9e42e0e853b065cbf720f5fedadbc102;521027c14a7ef622a4f03106799706d0824e192a;70302e07a5d51a7e330c11e61424f6a3beffa0ab;625ddfe164e3a25405672ecc1ca17d05353ff182;7690072298d0071074745c23ec6452d26b66c10a;c41c73c346652c2780d56372e2a332a480a2b332;3e91f8caa53385dc4256634638431d10a8c9173a;b5d3e7103d3637ec2441d10d087ec15a29f4a8f4;8207aba298da335fec4cb90d180f053fdf26843a;062a1004ac8e5640c249aff635957ed16aee5ebd;217c9ab4ed9c699a40e1b3025e4056d8995d8a59;76ff34826c14c9979e7795e1202dfc59dd12009c;5c822ada591dfd76cde0ab18e59e2ebf78737ccb;8a2f1a287a0ae2379f0961089d491923f62775b4;56c6f93bafd8fe1c82ae42851cf350cb846840e8",
         "Mechanistic insights into G-protein coupling with an agonist-bound G-protein-coupled receptor.;The dynamics of agonist-β_2-adrenergic receptor activation induced by binding of GDP-bound Gs protein;Conformational Dynamics of the Activated GLP-1 Receptor-Gs Complex Revealed by Cross-Linking Mass Spectrometry and Integrative Structure Modeling;The full activation mechanism of the adenosine A1 receptor revealed by GaMD and Su-GaMD simulations;Kinetic model of GPCR-G protein interactions reveals allokairic modulation of signaling;Structural basis of peptidomimetic agonism revealed by small-molecule GLP-1R agonists Boc5 and WB4-24;Heterobivalent Ligand for the Adenosine A2A-Dopamine D2 Receptor Heteromer.;Structural perspective of class B1 GPCR signaling.;Exploring Ligand Binding to Calcitonin Gene-Related Peptide Receptors;Highly accurate protein structure prediction with AlphaFold;Molecular insights into ago-allosteric modulation of the human glucagon-like peptide-1 receptor;Structural and Functional Diversity among Agonist-Bound States of the GLP-1 Receptor;Molecular basis of ligand recognition and activation of human V2 vasopressin receptor;Evolving cryo-EM structural approaches for GPCR drug discovery;Conformational switch that induces GDP release from Gi.;Structural basis for GLP-1 receptor activation by LY3502970, an orally active nonpeptide agonist;Deciphering the Agonist Binding Mechanism to the Adenosine A1 Receptor;A Multisite Model of Allosterism for the Adenosine A1 Receptor;Addressing free fatty acid receptor 1 (FFAR1) activation using supervised molecular dynamics;Supervised molecular dynamics for exploring the druggability of the SARS-CoV-2 spike protein;Structure and dynamics of the active Gs-coupled human secretin receptor;Differential GLP-1R binding and activation by peptide and non-peptide agonists;Structural insights into differences in G protein activation by family A and family B GPCRs;Structural insights into the activation of GLP-1R by a small molecule agonist;A combined activation mechanism for the glucagon receptor;Selective activation of Gαob by an adenosine A1 receptor agonist elicits analgesia without cardiorespiratory depression;Structure and Dynamics of Adrenomedullin Receptors AM1 and AM2 Reveal Key Mechanisms in the Control of Receptor Phenotype by Receptor Activity-Modifying Proteins.;Full-length human GLP-1 receptor structure without orthosteric ligands;Using metadynamics to explore complex free-energy landscapes;A Supervised Molecular Dynamics Approach to Unbiased Ligand-Protein Unbinding;Haloperidol bound D2 dopamine receptor structure inspired the discovery of subtype selective ligands;Energy Landscapes Reveal Agonist Control of G Protein-Coupled Receptor Activation via Microswitches;Activation of the GLP-1 receptor by a non-peptidic agonist;Reconstruction of apo A2A receptor activation pathways reveal ligand-competent intermediates and state-dependent cholesterol hotspots;Gγ and Gα Identity Dictate a G-Protein Heterotrimer Plasma Membrane Targeting;End-Point Binding Free Energy Calculation with MM/PBSA and MM/GBSA: Strategies and Applications in Drug Design.;Machine Learning Analysis of τRAMD Trajectories to Decipher Molecular Determinants of Drug-Target Residence Times;Structural Insights into the Process of GPCR-G Protein Complex Formation;Assembly of a GPCR-G Protein Complex;Rules of Engagement: GPCRs and G Proteins.;Molecular Dynamics Simulation for All;Mapping the Interface of a GPCR Dimer: A Structural Model of the A2A Adenosine and D2 Dopamine Receptor Heteromer;Biased Ligands of G Protein-Coupled Receptors (GPCRs): Structure-Functional Selectivity Relationships (SFSRs) and Therapeutic Potential.;Molecular Signature for Receptor Engagement in the Metabolic Peptide Hormone Amylin.;Simulation of spontaneous G protein activation reveals a new intermediate driving GDP unbinding;G Protein-Coupled Receptors as Targets for Approved Drugs: How Many Targets and How Many Drugs?;Simulations meet machine learning in structural biology.;Mechanism of the G-protein mimetic nanobody binding to a muscarinic G-protein-coupled receptor;Markov State Models: From an Art to a Science.;Dynamic Docking: A Paradigm Shift in Computational Drug Discovery;Serial millisecond crystallography for routine room-temperature structure determination at synchrotrons;Weighted Ensemble Simulation: Review of Methodology, Applications, and Software.;Selectivity determinants of GPCR–G-protein binding;Single-molecule analysis of ligand efficacy in β2AR-G protein activation;Exploring Protein-Peptide Recognition Pathways Using a Supervised Molecular Dynamics Approach.;Estimation of kinetic and thermodynamic ligand-binding parameters using computational strategies.;Kinetic Aspects of the Interaction between Ligand and G Protein-Coupled Receptor: The Case of the Adenosine Receptors.;OpenMM 7: Rapid development of high performance algorithms for molecular dynamics;CHARMM36m: an improved force field for folded and intrinsically disordered proteins;Regulation, Signaling, and Physiological Functions of G-Proteins.;Cholesterol-dependent Conformational Plasticity in GPCR Dimers;A Conserved Hydrophobic Core in Gαi1 Regulates G Protein Activation and Release from Activated Receptor *;Deciphering the Complexity of Ligand-Protein Recognition Pathways Using Supervised Molecular Dynamics (SuMD) Simulations;The pathway of ligand entry from the membrane bilayer to a lipid G protein-coupled receptor;A Hydrogen-Bonded Polar Network in the Core of the Glucagon-Like Peptide-1 Receptor Is a Fulcrum for Biased Agonism: Lessons from Class B Crystal Structures;The Role of Target Binding Kinetics in Drug Discovery;Gaussian Accelerated Molecular Dynamics: Unconstrained Enhanced Sampling and Free Energy Calculation;Universal allosteric mechanism for Gα activation by GPCRs;Structural basis for nucleotide exchange in heterotrimeric G proteins;Dopamine heteroreceptor complexes as therapeutic targets in Parkinson’s disease;Entry from the Lipid Bilayer: A Possible Pathway for Inhibition of a Peptide G Protein-Coupled Receptor by a Lipophilic Small Molecule.;Computational approaches for modeling GPCR dimerization.;MDTraj: a modern, open library for the analysis of molecular dynamics trajectories;Using caffeine and other adenosine receptor antagonists and agonists as therapeutic tools against neurodegenerative diseases: a review.;The Evolution of the GPCR Signaling System in Eukaryotes: Modularity, Conservation, and the Transition to Metazoan Multicellularity;WExplore: hierarchical exploration of high-dimensional spaces using the weighted ensemble algorithm.;Supervised Molecular Dynamics (SuMD) as a Helpful Tool To Depict GPCR-Ligand Recognition Pathway in a Nanosecond Time Scale;Rapid parameterization of small molecules using the force field toolkit;GPCRDB: an information system for G protein-coupled receptors;PLUMED 2: New feathers for an old bird;CHARMM36 all‐atom additive protein force field: Validation based on comparison to NMR data;Adrenaline-activated structure of the β2-adrenoceptor stabilized by an engineered nanobody;Polar transmembrane interactions drive formation of ligand-specific and signal pathway-biased family B G protein-coupled receptor conformations;Membrane Packing Problems: A short Review on computational Membrane Modeling Methods and Tools;Extension of the CHARMM general force field to sulfonyl‐containing compounds and its utility in biomolecular simulations;Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges;Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing;Protein data bank;MMPBSA.py: An Efficient Program for End-State Free Energy Calculations.;Molecular dynamics simulations and drug discovery;MDAnalysis: A toolkit for the analysis of molecular dynamics simulations;Crystal Structure of the β2Adrenergic Receptor-Gs protein complex;PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions.;Lateral diffusion of membrane proteins.;Adenosine A2A receptor-antagonist/dopamine D2 receptor-agonist bivalent ligands as pharmacological tools to detect A2A-D2 receptor heteromers.;ACEMD: Accelerating Biomolecular Dynamics in the Microsecond Time Scale.;CHARMM‐GUI: A web‐based graphical user interface for CHARMM;Membrane interactions of G proteins and other related proteins.;Well-tempered metadynamics: a smoothly converging and tunable free-energy method.;Vasopressin and disorders of water balance: the physiology and pathophysiology of vasopressin;OPM: Orientations of Proteins in Membranes database;Scalable molecular dynamics with NAMD;The GRAFS classification system of G-protein coupled receptors in comparative perspective.;UCSF Chimera—A visualization system for exploratory research and analysis;How a G Protein Binds a Membrane*;PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations;Accelerated molecular dynamics: a promising and efficient simulation method for biomolecules.;Adenosine A2A-Dopamine D2 Receptor-Receptor Heteromerization;Crystal Structure of Rhodopsin: A G‐Protein‐Coupled Receptor;Escaping free-energy minima;A fast SHAKE algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations;Vasopressin Receptors;Restrained electrostatic potential atomic partial charges for condensed-phase simulations of carbohydrates.;SHAKE, rattle, and roll: Efficient constraint algorithms for linked rigid bodies;Replica-exchange molecular dynamics method for protein folding;Adiabatic bias molecular dynamics: A method to navigate the conformational space of complex molecular systems;Adenosine A2A receptor antagonists as new agents for the treatment of Parkinson's disease.;VMD: visual molecular dynamics.;A smooth particle mesh Ewald method;GROMACS: A message-passing parallel molecular dynamics implementation;Langevin dynamics of peptides: The frictional dependence of isomerization rates of N‐acetylalanyl‐N′‐methylamide;Molecular dynamics with coupling to an external bath;Comparison of simple potential functions for simulating liquid water;The Adaptive Path Collective Variable: A Versatile Biasing Approach to Compute the Average Transition Path and Free Energy of Molecular Transitions.;Heteroreceptor Complexes and their Allosteric Receptor–Receptor Interactions as a Novel Biological Principle for Integration of Communication in the CNS: Targets for Drug Development;Lipid-dependent GPCR dimerization.;High-resolution crystal structure of an engineered human beta2-adrenergic G protein-coupled receptor.;Modeller: generation and refinement of homology-based protein structure models.;Kr (cid:0);SHAKE, rattle, and roll: Efficient constraint algorithms for linked rigid bodies;[19] Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors;Lipid Modification of G Proteins: alpha Subunits are Palmitoylated;G protein-coupled receptors | G protein-coupled receptors | IUPHAR/BPS Guide to PHARMACOLOGY"
        ],
        [
         "29",
         "02abac2e5c388dd967a6b99def97cbca4d64d315",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/05/2024.08.02.606316.full.pdf",
         "Biology",
         "bioRxiv",
         "Zhou Yu,Andrian Gutu,Namsoo Kim,Erin K. O’Shea",
         "2314948403,6373079,2315813699,2314920602",
         "0",
         "0.0",
         "2025-02-17",
         "2025",
         "",
         ""
        ],
        [
         "30",
         "02cabd03b27e19ee87f97b1cd9c7d9d05649ea85",
         "https://www.biorxiv.org/content/biorxiv/early/2024/01/22/2024.01.18.576181.full.pdf",
         "Biology",
         "bioRxiv",
         "Michael T. Pinner,Hendrik Dietz",
         "2280904284,2210418674",
         "1",
         "0.0",
         "2025-02-27",
         "2025",
         "",
         ""
        ],
        [
         "31",
         "02f332a476ad77bda3627220c4f1f6fc6e3e0b03",
         "https://www.biorxiv.org/content/biorxiv/early/2024/06/01/2024.05.31.596594.full.pdf",
         "Biology",
         "bioRxiv",
         "Hisashi Endo,Yuki Yamagishi,Thi Tuyen Nguyen,Hiroyuki Ogata",
         "51106178,2304556789,2304688307,2241789570",
         "0",
         "0.0",
         "2025-02-21",
         "2025",
         "c71f9d753626217f2e599e0b0c115c1a11c8af88;1f03aa0a1c771f73f4ee5308a6f5e09d9d0d19a1;3a4914603d93b26553e9765c50d20d00b1c084f6;a83db063fcd2d67b31190f52f3fdf1f5dc5bbe7b;20edebff123932779901a5934495447772a94e61;0eb8e0b53fb023486f6ef03d53e27a5f4ca5e877;e6612a8d239a294fcf728a3ad814b368fc8c14bc;b72d14201d65e788a094e820059ece24c022cb5a;61848f4d99616ee182ee96a581e6a528b17afd7d;8ca5f64f8bd5043ba7fa688e7a38f358ee1fff29;5f90f273c8058c415643d5bbbd0ce620f7c0a3cd;3f9a5258ac2324355eab7bbb5bb3e804ffee794e;d05247f1bc21b5da4a17c60d0fead11491e9448b;220ac5e17ab565a62cc8f0e54067d62856650bbd;3fb84644a0eb1e60c355f4570b2bd13c35169fb3;d9aa573fc2ad239395216532d49805272b872254;b31d0f8aa7cb8fe5676e60109de4385a5244129f;e4cda0ec06d985fdef23c72b10fc58cf0de63036;f3626f38fedea3818603d0d6b2d39cacee9a0f44;d19ba399bc668b0a54f5e65cdd9935e738c97c27;ebfcaf6439ea3007f974335ed968919ae66e5109;54bd4c1360c3aedb6cfdeabe43dc37d8cec787ed;3ba264b872dac0bf918565bef0553bb01f0717b7;8f8bded8c09ec81e2529d5c99a5506e144dd8b5e;6d778c96f31f9ffc6bdb948e94964cb0c44b143b;fb863912acd9f59a2a3547c898ce68093aa566b6;133bcd7488a3c07cb0f493a87564c30e5433768c;763c15ac95d28be6fde37bcc9709dfc1a7043cd7;70a5effb32fae2a245b656ac70aba2686fb1a3e8;ef7afb91e6b40fb2fc5b24e0bc79d01b40b62a23;0d6c714135071ebdcc0b05422e52ef8b820af378;75f6d8079d154ac48e192fda7951218f8322a087;0fa56e9ad5cc3da874ce5d1d9a1e868f6528db1b;3e56119cfd0feffdb022987cf6c7c5e828275bd8;ce2be8897484a5a0eb37382e8a9797e111a2de4f;9efd70d2c06733704220313fb67720aa45c6362a;4c44196efc4f641e125529f80e56c81f41918541;5e7ad9e5fda4fc235543ff8d34d3dc63f2f1bfd4;32f60b371197caed135f829d78632658f641bbc2;c123627e60f1be3e8a2f3511c147a6055c371de7;633ee02c8fa5214465e362134db4408b694cd552;96795eaacf185bd244f5ceedace82b5c3b0cafd8;c2173bd67e6ed56457f2ae3e90c42ecffd80101d;4c1fd933264bc730865a66de7a1074e175e9d9f4;eb3832ba9a5af0d6bcb6cca4ed137e0f14466ff3;38b15a5d2481179de7b22c9d4458727b30552127;8bb6cc4057c0f76e963f19918a79697acbd2bc41;f8b60bb6c8e7bfec84d31fe49482b79811604bcd;63825973f020269e522f87d4785f405ec9f93292;224eb6271b2e375cc47605729e5b86bf6bc998ef;3abad0b62777ee2f1180ff2b87ce2377e72b49c6;3e691ea51b7616d7b6a5636497348bdac2d6c8de;f877e863cb0bd4fbcca9720ccc3c42049bc664b8;7d94c4cbc8352b6bcf745be8333eb5c0b67d8444",
         "Functions of extracellular polymeric substances in partitioning suspended and sinking particles in the upper oceans of two open ocean systems;The microbial carbon pump and climate change.;Taxon-specific contributions of microeukaryotes to biological carbon pump in the Oyashio region;Selective cell lysis pressure on rare and abundant prokaryotic taxa across a shelf-to-slope continuum in the Northern South China Sea;Characterization of sinking and suspended microeukaryotic communities in spring Oyashio waters;Vertical imbalance in organic carbon budgets is indicative of a missing vertical transfer during a phytoplankton bloom near South Georgia (COMICS);Genomic adaptation of giant viruses in polar oceans;Viral infection switches the balance between bacterial and eukaryotic recyclers of organic matter during coccolithophore blooms;Viral infection in the ocean—A journey across scales;Haptophyte communities along the Kuroshio current reveal their geographical sources and ecological traits;Mortality by ribosomal sequencing (MoRS) provides a window into taxon-specific cell lysis;Cryptic and abundant marine viruses at the evolutionary origins of Earth’s RNA virome;The Ocean's labile DOC supply chain;Patterns of eukaryotic diversity from the surface to the deep-ocean sediment;Environmental vulnerability of the global ocean epipelagic plankton community interactome;A direct capture method for purification and detection of viral nucleic acid enables epidemiological surveillance of SARS-CoV-2;A thermal trade-off between viral production and degradation drives virus-phytoplankton population dynamics.;Impaired viral infection and reduced mortality of diatoms in iron-limited oceanic regions;Environmental (e)RNA advances the reliability of eDNA by predicting its age;A method for characterizing dissolved DNA and its application to the North Pacific Subtropical Gyre;A dataset on trophic modes of aquatic protists;Biogeography of marine giant viruses reveals their interplay with eukaryotes and ecological functions;Resolving marine dissolved organic phosphorus (DOP) composition in a coastal estuary;Visualizing active viral infection reveals diverse cell fates in synchronized algal bloom demise;Metabolic and biogeochemical consequences of viral infection in aquatic ecosystems;Molecular properties are a primary control on the microbial utilization of dissolved organic matter in the ocean;Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2;Eukaryotic virus composition can predict the efficiency of carbon export in the global ocean;Spatial Variations in Community Structure of Haptophytes Across the Kuroshio Front in the Tokara Strait;Aggregate Formation During the Viral Lysis of a Marine Diatom;Carbon Budgets of Mesozooplankton Copepod Communities in the Eastern Atlantic Ocean—Regional and Vertical Patterns Between 24°N and 21°S;Coccolithovirus facilitation of carbon export in the North Atlantic;fastp: an ultra-fast all-in-one FASTQ preprocessor;Optimizing taxonomic classification of marker-gene amplicon sequences with QIIME 2’s q2-feature-classifier plugin;Unveiling the role and life strategies of viruses from the surface to the dark ocean;DADA2: High resolution sample inference from Illumina amplicon data;Plankton networks driving carbon export in the oligotrophic ocean;Eukaryotic plankton diversity in the sunlit ocean;Rethinking the marine carbon cycle: Factoring in the multifarious lifestyles of microbes;Rising to the challenge: accelerated pace of discovery transforms marine virology;Effects of sea surface warming on marine plankton.;Reconciliation of the carbon budget in the ocean’s twilight zone;Dissolved organic matter (DOM) release by phytoplankton in the contemporary and future ocean;Microbial control of the dark end of the biological pump.;Arctic Ocean Microbial Community Structure before and after the 2007 Record Sea Ice Minimum;The relative importance of sloppy feeding, excretion, and fecal pellet leaching in the release of dissolved carbon and nitrogen by Acartia tonsa copepods;edgeR: a Bioconductor package for differential expression analysis of digital gene expression data;Estimates of virus- vs. grazing induced mortality of picophytoplankton in the North Sea during summer;Marine viruses — major players in the global ecosystem;Production of dissolved organic carbon by sloppy feeding in the copepods Acartia tonsa, Centropages typicus, and Temora longicornis;Determination of dissolved nucleic acids in seawater by the fluorescence dye, ethidium bromide;The measurement and distribution of dissolved nucleic acids in aquatic environments;Dynamics of Dissolved Organic Phosphorus;Corresponding author.;mesopelagic layer showed a habitat preference for the epipelagic layer"
        ],
        [
         "32",
         "03003bda6891a727a43bbe0ff847b40748a8187e",
         "https://www.biorxiv.org/content/biorxiv/early/2023/09/15/2023.09.02.555672.full.pdf",
         "Biology",
         "bioRxiv",
         "D. Amorim,S. S. Oliveira,M. Balbi,Y. Ang,Ambrosio Torres,Darren Yeo,A. Srivathsan,Rudolf Meier",
         "48848980,2809096,122467597,38735837,2341885613,2238219849,144870729,2238219980",
         "2",
         "0.0",
         "2025-01-21",
         "2025",
         "",
         ""
        ],
        [
         "33",
         "031550851c1139950a93f08ec62552c8ebbca5e3",
         "https://www.biorxiv.org/content/biorxiv/early/2023/08/08/2023.02.23.529673.full.pdf",
         "Biology",
         "bioRxiv",
         "Zezhen Wang,Weihao Mai,Yuming Chai,Kexin Qi,Hongtai Ren,Chen Shen,Shiwu Zhang,Guodong Tan,Yu Hu,Quan Wen",
         "2162347465,145510274,6213274,2209893237,2152103760,2209944330,2552347,2146720593,2143988959,2058043932",
         "3",
         "0.0",
         "2025-01-06",
         "2025",
         "",
         ""
        ],
        [
         "34",
         "0316bec2c50dbd379f8c217b638bd35b213b66fc",
         "https://www.biorxiv.org/content/biorxiv/early/2023/10/21/2023.10.18.562953.full.pdf",
         "Biology",
         "bioRxiv",
         "Arnaud N’Guessan,Wen Yuan Tong,Hamed Heydari,Alex N. Nguyen Ba",
         "2261479825,2261483266,2261517554,2422462",
         "1",
         "0.0",
         "2025-02-20",
         "2025",
         "f4a4139a264e26ded6730bb2ba66a1979f9da53a;23359eb2fd778ead3fae41a97c0fbe3f14002518;6d5cebe2a7e72618f49ac419fd0c0398a5897981;ecad8506c584aebbbbdfac9a3aa580caf2ca120b;de683c1a563ed329f4f58548bfe50c0a4c0a4505;e4a1421aa7016a1ffe344daa854a73497c4eabda;dae2170fece4d6db80c4476d94cecda4198457fd;c9b2076ec0c6e6d0379aee76ff45bc2a12a5b21d;552f676b10ffb713424650655e75ee3829e693c7;920b9d11910d3e1c5d2dad2ac24ae62eb67a4a87;6a58da8e0073a34bd0822d6e2a09a6969ae7b980;0ab52b28ed16c7a38a6dc164a5623647ca3b634d;9369c9cb70cd93b98909c2cc805c4b94132bb7d6;2d7390cc6cd787e2111aaf8754ee812022893301;3109393967ed9e1b3c5a2e994728d0465ef07995;003a3ff4b5adf544423b1249512615025a5d0667;d783d19ccb503c56819ddd2590859b6f2df6de04;e7db05c36efd6aabb0bd793fda02e6f82d5a0d1d;8a270829fc20daaf46f10bb42f317068c258b7f1;463e2ac71d549b18a91a24cd9bc2e641b3a4542c;219bee9fde1303f423fac52ff373f19552c60d53;8932f29b432ca127ce0df3469b6e67990deb9a1f;2b4d0a90fb392cdfbdefb8e96a606d64368a3012;b547b5af9bad50387bdadf2964628ea816faae83;aa728f3e82b8a171cd1417905b4d6bb3a0a574b1;b573996695773e5ce9545025208a0feccddb3161;17422a9fcf3f05408f7f2f270b127aa812e9b2b3;3cb0ed79692dfac07b3ee23127125d49cce40054;e748f21252ce5d96f42bc508374659fbeb6f97ff;e9499fd4926ffd990c7b4d64fbca2147cd205c89;17c7a0ffc53643f089959b019fd034f24ab7e429;05043cc44c382c4c1e5a0cddc313d4c7120cd646;fb5b33799f391510f48ff30965234ebd5e02d841;8e3002bd2d398254092da5ff74616cbe93dcad16;98ac9cc09fd81741408ad5eba2c951bd86006ffc;964a74eb563dd2d1294edba87730e5ad99549553;8895fe55aa0b88799b8193113638a08e237c9feb;33de7877a9b6debcff71b7e100d1cefa61e55bfc;599dceeed1c8178a1da7d608db803488d6a467dc;7fcabcb1dff389a3c8750c983b1b34753cbce59e;3967ce20ca9d212db341bf943ec824b8b74ed701;7e4325660b960a398e6408e83add5e244272a1bd;2fad434cad40e5fbf5f852bd443a15e53aeafa2d;766d42c4196a9e1b69494d86d10aabc056514e60;d5b5aa46cff84bac25f60e87474006adb12a4868;f33846ab567e5be19280b80f17a712e71b02f074;f2f6c02b41331c6f5c4b0151dea10b4b028eff02;4ccfac91b2c66be7ac5b741f2f7bd88bc8d28f1a;49a4b0131e313b93394046d4ac4d7e2f3ece65db;236fe437af4b7d0bdfac70142782a904ee5c6921;a681150c05e0244054f6c86ec7702ccb2dad39a8;236f4275a7a2db3db8e901aea5d6469c601b0732;fb575405f25f13876ba4dc5898f911d3c1d2870d;e806647ac8f0ccc546934de20d536f14a2719738;2a22f6ae51161c6f4c40fc736ac90a999f403a9e;7eeac509af9de50900a362c3824f29ef0596e9c7;21ac65c06822e0e25956edf9461f5ed16f1db1cf;d0c463f3cbf54cf98a1be8109e1376259c014dfc;cc9344f7ef22f27ec864fdc492e781cdfc81b407",
         "Trans-eQTL mapping in gene sets identifies network effects of genetic variants;Molecular quantitative trait loci;DiSCERN - Deep Single Cell Expression ReconstructioN for improved cell clustering and cell subtype and state detection;Pleiotropic effects of trans-regulatory mutations on fitness and gene expression;An introduction to spatial transcriptomics for biomedical research;PANTHER: Making genome‐scale phylogenetics accessible to all;Barcoded bulk QTL mapping reveals highly polygenic and epistatic architecture of complex traits in yeast;Powerful eQTL mapping through low coverage RNA sequencing;Aggregative trans-eQTL analysis detects trait-specific target gene sets in whole blood;Phenotypic and molecular evolution across 10,000 generations in laboratory budding yeast populations;Cis and trans effects differentially contribute to the evolution of promoters and enhancers;Efficient and effective control of confounding in eQTL mapping studies through joint differential expression and Mendelian randomization analyses;A new protocol for single-cell RNA-seq reveals stochastic gene expression during lag phase in budding yeast;Size-Dependent Increase in RNA Polymerase II Initiation Rates Mediates Gene Expression Scaling with Cell Size;RNA sequencing: the teenage years;Genome-wide association and transcriptome studies identify target genes and risk loci for breast cancer;Rare variants contribute disproportionately to quantitative trait variation in yeast;Opportunities and challenges for transcriptome-wide association studies;Protocol Update for large-scale genome and gene function analysis with the PANTHER classification system (v.14.0);Trans Effects on Gene Expression Can Drive Omnigenic Inheritance;Single-cell RNA sequencing technologies and bioinformatics pipelines;Genetics of trans-regulatory variation in gene expression;The Dynamics of Molecular Evolution Over 60,000 Generations;Transcriptome Profiling in Human Diseases: New Advances and Perspectives;Genome-Wide Association Studies In Plant Pathosystems: Toward an Ecological Genomics Approach;Rare cell variability and drug-induced reprogramming as a mode of cancer drug resistance;Massively parallel digital transcriptional profiling of single cells;Evolution: Tinkering within Gene Regulatory Landscapes;Assessing Glucose Uptake through the Yeast Hexose Transporter 1 (Hxt1);A High-Definition View of Functional Genetic Variation from Natural Yeast Genomes;Systematic identification of trans eQTLs as putative drivers of known disease associations;Chapter 11: Genome-Wide Association Studies;Coordinating genome expression with cell size.;Small- and Large-Effect Quantitative Trait Locus Interactions Underlie Variation in Yeast Sporulation Efficiency;Systematic Localization of Common Disease-Associated Variation in Regulatory DNA;Linking disease associations with regulatory information in the human genome;Finding the sources of missing heritability in a yeast cross;Comparative studies of gene expression and the evolution of gene regulation;GCTA: a tool for genome-wide complex trait analysis.;Trait-Associated SNPs Are More Likely to Be eQTLs: Annotation to Enhance Discovery from GWAS;Dissection of genetically complex traits with extremely large pools of yeast segregants;THE LOCUS OF EVOLUTION: EVO DEVO AND THE GENETICS OF ADAPTATION;A survey of essential gene function in the yeast cell division cycle.;Mapping Novel Traits by Array-Assisted Bulk Segregant Analysis in Saccharomyces cerevisiae;Cell-cycle control of gene expression in budding and fission yeast.;The YJR127C/ZMS1 gene product is involved in glycerol-based respiratory growth of the yeast Saccharomyces cerevisiae;Genetic interactions between polymorphisms that affect gene expression in yeast;Enp1, a yeast protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis.;Genetic Dissection of Transcriptional Regulation in Budding Yeast;Regression Modeling Strategies: With Applications to Linear Models, Logistic Regression, and Survival Analysis;The core meiotic transcriptome in budding yeasts;Manifold anomalies in gene expression in a vineyard isolate of Saccharomyces cerevisiae revealed by DNA microarray analysis.;ENP1, an essential gene encoding a nuclear protein that is highly conserved from yeast to humans.;Evolution and tinkering.;Evolution at two levels in humans and chimpanzees.;Single-Cell RNA Sequencing in Yeast Using the 10× Genomics Chromium Device.;Dynamic Role of trans Regulation of Gene Expression in Relation to Complex Traits.;Forward genetic approaches to understanding complex behaviors.;KEGG: Kyoto Encyclopedia of Genes and Genomes;Genomics: Methods and Protocols;was often selected by the cross-validation procedure when tested on a few genes and thus we do 684 not believe that this approach will significantly change our results;Department of Molecular Genetics, University of Toronto;Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, 160 College 11 Street, Room 230, M5S3E1, Toronto, Ontario, Canada;Department of Biology, University of Toronto at Mississauga, 3359 Mississauga Rd, L5L 1C5, 7;Transcriptome: Connecting the Genome to Gene Function | Learn Science at Scitable"
        ],
        [
         "35",
         "033a24661b6865c08c45c49383378f4bd80c5310",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/16/2024.10.14.618165.full.pdf",
         "Biology",
         "bioRxiv",
         "Haiteng Wang,Qihong Zou,Jinbo Zhang,Jiahong Gao,Yunzhe Liu",
         "2247832508,2262267181,2326323825,2274195082,2318367719",
         "0",
         "0.0",
         "2025-02-17",
         "2025",
         "412a79c2b1cf1ca97ab8703352fe2d1657ee8b1e;6557cb3a028549f263f50cc8287285e685de5d42;157c253e40060fb09555fd5d23ba33d1bf6a61b3;6bba0495b3fecb8101de29135a75f53d88026b96;fbe24e2717389c4ad5ddba922938b71c303299ec;aa97b49d2febaa0b8e88f046abce40d59a3ad162;aac33395437c9eba5310daf27e83a56ec50a4c87;0c4202286cf0f986c7e9919d8198845f9e08c8c8;b5617ff697fa1c04fc8c7ff07289b393345532aa;3e892fb175db4556e027e0f6fdd568911e7c0a62;8e557932685d74b99d485eb4e7caccac4dc1394a;b7ba5e72e4930e40edcbdf69ec4062a09dc1efce;b27746e73f0dba363ad406fd61dfc8e12e11ba2d;719d196179be616a8acf8b2979f63ed4519e7222;e09e67cffbea7be63d2c494490ca0347b0330275;c36cd75a5e1334daa0015a5244cef095bd2ec1c8;37e2687c7de1e1ae96f7ae043596ce25b05d5fad;731f1fc2d0763155de59f3958856871e6627f828;bfda62d4b638c5db9611b440142d40f595f13ec3;bdf26032c2141733d6493f864a03a52780e993cf;ad95d4e46c0fd2d49264c189d2ac16c87f37b17a;03b32e32c868e7106df11cd5390756379241d583;d51aeba2dbe428cd580a84890fd4beb8f39b963c;3a03606dff18977871c777f07233dee59f836241;afddf9fe7c033c18282ed34ad86210c1ff16aeb6;d51701868e12ecc1f8c68b4a6c3e7e07c8e3d918;5d83dd7bc824f76ec5d88cc05488331ae9359794;5da3a19d55078f76031a56de60f8ee2d6f4878df;a349feb94fe2b1b0520710a36c475b98e52ce96f;11deb8897c8a31afc78ebe4d297ada9a0f33c9f6;8874390fcd4d37657b1c4308820582f00899ba34;f15367ed93c3505b1d62d802f3f4b769ae0f4ba5;a3f132288f4797236098766cb2893b5d3afe69f6;be6638e641c5e993474703de6e0261357da71736;a775181ffabe0a797734d510f8219633184d392e;722d274df9d63f65b9c403ba7632627b56982d26;13846f6879521f901e94b4c15c32cc7c703f2319;cbb8a961ea6cef922e47cc32bf7771b175b80147;63a604e73af861640cbf711cef8b75fc6d5ccd1e;1f3f89b25d7e7c1dbe159aeac11c950941e070c3;f3a2f04772c560d4f5ccef21f544a7e5837dedb6;46064fe6c3539f623234ad98b2b57cff56fcb85d;3bda8b8dd09117f7d5abc12cf4581ac50b6412ee;10773590b90ac24d56d2a1ab4ae221ee773e2573;c65d341e1313e153cbce97dc9de38bbf6b94bcfc;a43fb952240a3042629aa9187e380d39828b9d96;c05fa9c64aa19ae67ea7cda5fb21018745d50b85;b122d45f2f1d78bf996296d6adc54a924062d67b;66b498a54843e14270b75f0c3c8fc65a0b2e1e98;75f6236262d934d50aa61ad3a3e70808b1591e6d;694509dccd2831bc7afc13e510ad6a081726e354;ca60e5ff6a49cd23e89dc99d2cc905b917d76f8a;ad4e281293237bdb8efbbd639ac823536815f42b;c11ff925a3d829436f15a26aafb8594c6386e477;2ad260db5dd6c0fc3cb3938ffa11e256c91d194a;76ad159a2887b008e5a7335d124c148c13e65465;291c9d66cf886ed27d89dc03d42a4612b66ab007;ad9d07c86f861fde7d0618e38f54f7ba9b317c57;7d6365839a069042c79f1cb6471045c61566eaa0;29980b90f83dcf26eb5b4a3dc282ae987c42a834;cae30359d3e2e10bc4e586db006be4584d7ec9b0;851d8a4f1d891a45b0aa529b6fcda23934ac2a06;60b68edc039324bfb7092414a8210ab118db9438;059e2f18d333b464493314ec518abea04c49bc89;7d5aedecdfc4e8f83638bac47eb7cf2f860ec51c;5c5eeb1e82490eeb1291c2da2e17e0cf9256abd2;9166397d44abf720639e8a3abc14ac56ef615f44",
         "Memory reactivation during sleep does not act holistically on object memory;Replay-triggered brain-wide activation in humans;How coupled slow oscillations, spindles and ripples coordinate neuronal processing and communication during human sleep;Targeted memory reactivation during sleep influences social bias as a function of slow-oscillation phase and delta power.;Decoding cognition from spontaneous neural activity;A model of autonomous interactions between hippocampus and neocortex driving sleep-dependent memory consolidation;The human thalamus orchestrates neocortical oscillations during NREM sleep;An open-source, high-performance tool for automated sleep staging;The default mode network in cognition: a topographical perspective;The nucleus reuniens, a thalamic relay for cortico-hippocampal interaction in recent and remote memory consolidation;The default mode network: where the idiosyncratic self meets the shared social world;Endogenous memory reactivation during sleep in humans is clocked by slow oscillation-spindle complexes;Sleep spindles mediate hippocampal-neocortical coupling during long-duration ripples;Inferences on a multidimensional social hierarchy use a grid-like code;Promoting memory consolidation during sleep: A meta-analysis of targeted memory reactivation.;Sleep Spindles: Mechanisms and Functions.;Bidirectional prefrontal-hippocampal dynamics organize information transfer during sleep in humans;Analysis of task-based functional MRI data preprocessed with fMRIPrep;Increased neuronal signatures of targeted memory reactivation during slow-wave up states;The hippocampal sharp wave–ripple in memory retrieval for immediate use and consolidation;FMRIPrep: a robust preprocessing pipeline for functional MRI;Representation of Real-World Event Schemas during Narrative Perception;Dual function of thalamic low-vigilance state oscillations: rhythm-regulation and plasticity;Thalamic Spindles Promote Memory Formation during Sleep through Triple Phase-Locking of Cortical, Thalamic, and Hippocampal Rhythms;Coordination of cortical and thalamic activity during non-REM sleep in humans;Situating the default-mode network along a principal gradient of macroscale cortical organization;Organizing conceptual knowledge in humans with a gridlike code;Hippocampo-cortical coupling mediates memory consolidation during sleep;Cued Reactivation of Motor Learning during Sleep Leads to Overnight Changes in Functional Brain Activity and Connectivity;Hippocampal sharp wave‐ripple: A cognitive biomarker for episodic memory and planning;Hierarchical nesting of slow oscillations, spindles and ripples in the human hippocampus during sleep;Machine learning for neuroimaging with scikit-learn;Methods to detect, characterize, and remove motion artifact in resting state fMRI;MEG and EEG data analysis with MNE-Python;Auditory Closed-Loop Stimulation of the Sleep Slow Oscillation Enhances Memory;Upgrading the sleeping brain with targeted memory reactivation;Fast and slow spindles during the sleep slow oscillation: disparate coalescence and engagement in memory processing.;The sleep relay—the role of the thalamus in central and decentral sleep regulation;Nipype: A Flexible, Lightweight and Extensible Neuroimaging Data Processing Framework in Python;Selective optical drive of thalamic reticular nucleus generates thalamic bursts & cortical spindles;Regional Slow Waves and Spindles in Human Sleep;FieldTrip: Open Source Software for Advanced Analysis of MEG, EEG, and Invasive Electrophysiological Data;N4ITK: Improved N3 Bias Correction;The memory function of sleep;Accurate and robust brain image alignment using boundary-based registration;Unbiased nonlinear average age-appropriate brain templates from birth to adulthood;The Human K-Complex Represents an Isolated Cortical Down-State;Spontaneous neural activity during human slow wave sleep;Symmetric diffeomorphic image registration with cross-correlation: Evaluating automated labeling of elderly and neurodegenerative brain;Forward and reverse hippocampal place-cell sequences during ripples;A component based noise correction method (CompCor) for BOLD and perfusion based fMRI;Odor Cues During Slow-Wave Sleep Prompt Declarative Memory Consolidation;An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest;The Sleep Slow Oscillation as a Traveling Wave;Normal human sleep: an overview.;EEGLAB: an open source toolbox for analysis of single-trial EEG dynamics including independent component analysis;Improved Optimization for the Robust and Accurate Linear Registration and Motion Correction of Brain Images;The functional significance of K-complexes.;A global optimisation method for robust affine registration of brain images;Psychophysiological and Modulatory Interactions in Neuroimaging;Software tools for analysis and visualization of fMRI data;Thalamocortical oscillations in the sleeping and aroused brain.;Spindle density in sleep of normal subjects.;cortex (Coulon, Budde, & Pape,;NeuroSynth: a new platform for large-scale automated synthesis of human functional neuroimaging data;Segmentation of brain MR images through a hidden Markov random field model and the expectation-maximization algorithm;Evaluation of Noisy Data;� Correspondence;Keywords : slow oscillation (SO), spindle, hippocampus, thalamus, simultaneous EEG-fMRI;The analysis code will be;sleep rhythms align. Understanding these mechanisms paves the way for developing neuromodulation interventions to enhance cognitive function during sleep and treat sleep-related disorders;all other FC pathways ( Fig. 4e );generating and coordinating sleep rhythms, the implications of reduced and the temporal dynamics of hippocampal-neocortical communication. brain-wide activation patterns"
        ],
        [
         "36",
         "0347d45ca1cd3e88a43d9b988242ed908b6634aa",
         "https://www.biorxiv.org/content/biorxiv/early/2024/01/15/2024.01.12.575186.full.pdf",
         "Biology",
         "bioRxiv",
         "Esra Öztürk,B. Temiz,A. Karahan",
         "2260678655,2126517257,15769885",
         "0",
         "0.0",
         "2025-01-08",
         "2025",
         "5ffde96faef8b42162df76aa6d11421526e2a4f7;f7189f9a5ae633292333e475dbf3a6e8e03a4f46;2ecc79a2c17a51e2f6fc0e78e223014692305741;0ccc82b74890478854505cd95ddd30e51c6cf661;7e9687c927e9d0be81861de5ea0f4d0c379cff02;64c9d3bc2f28b1e579586cdcff6ec71897305f0c;d6cc423001f17b7dae5595972c3654e7b27af960;2876bb25c62047f4897bee699dade4b1399c5bf2;3d7c4f03bbe0f96322bcf6cb2c023eb730e9d842;8125ff270d4c6cb2b78a09e4522be4c712887d26;8e5f52ed1540ee934312f77c091c9ed17761f43c;4a2f0d3d08ab0e0c140f325413ae26b8820e9077;cfbe72a4ee4fea8fe103c9204a40e4f0efd8e3d0;83744a39e78297de15570bf7877782ceb8b719ae;98d3667d3ff37e8c66523ad01f71c291c9fc1b90;7d1f979e999aaa72939b8e2eff830e654d99a615;485ab05a5a4b16fa7df22554f9066beb3716ff4b;e134cceaf71d6e2b025ad41b32164067d29d2712;4f149a015b3a5cae16bd223549cf7c17daad162e;f63436fd05c3d0ad17dc22e5d1d88d3873cecdee;8527ddaaa151f352e45c8cb0789f2387f0bf1ea8;5fa412f03ff08cd5b4c24503e685875cae48e461;943d38b9d96f8222e883604822bcafb7930ca6da;8fc9f73877a6e9bd2b7fdf173314fa23d726faa8;fad3cd73f2e5927595640d97375ea302b68caff1;c94922a6541da3df35809c993519fba677c1c3b3;397669ff0c623113f692d7015e8a7c6713470f4f;59a98cbce4200c807cfad783380ec7be955e7b6f;cf2bfaf3ac10b888a4c7c1981acbc3b064692b4a;b0fad63eeb3b627f16c5284a5af0dc54aaa9f109;01a0330ef297429cf2826ac5bb0794490a58a431;07cb3d16b0824bab28938618b46bd4fbfc1acdae;9304e349c08d04f7dd722984eba16530b61d7810;2396b101a344a539c7c8a50ce3dcda7204fbd19c;ea0d088835e68c5ef05556a03e8812e26a2d8380;e6b031ad350b991f7957baa6307a79a152d6161f",
         "Phylomitogenomics bolsters the high-level classification of Demospongiae (phylum Porifera);Shedding light on an East-Mediterranean mesophotic sponge ground community and the regional sponge fauna;Who’s there? – First morphological and DNA barcoding catalogue of the shallow Hawai’ian sponge fauna;MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization;Employing DNA barcoding as taxonomy and conservation tools for fish species censuses at the southeastern Mediterranean, a hot-spot area for biological invasion;Molecular phylogenies confirm the presence of two cryptic Hemimycale species in the Mediterranean and reveal the polyphyly of the genera Crella and Hemimycale (Demospongiae: Poecilosclerida);First record of Terpios fugax Duchassaing and Michelotti, 1864 (Demospongiae: Suberitidae) in the Eastern Pacific Ocean;Porifera of Greece: an updated checklist;Effects of phylogenetic reconstruction method on the robustness of species delimitation using single-locus data;Stability of the sponge assemblage of Mediterranean coralligenous concretions along a millennial time span;Particularly Sea-Sensitive Areas and Marine Protection Zones. A Controversial Issue that needs Interpretation;Molecular phylogenies support homoplasy of multiple morphological characters used in the taxonomy of Heteroscleromorpha (Porifera: Demospongiae).;An overview of the submerged sea caves of Sagres (South of Portugal-Algarve);Cryptic speciation and phylogeographic relationships in the elephant ear sponge Ianthella basta (Porifera, Ianthellidae) from northern Australia;Barcoding Sponges: An Overview Based on Comprehensive Sampling;Global Diversity of Sponges (Porifera);How Many Species Are There on Earth and in the Ocean?;DNA taxonomy of sponges—progress and perspectives;Phylogenetic position of sponges in early metazoan evolution and bionic applications of siliceous sponge spicules;CO1 phylogenies in diploblasts and the 'Barcoding of Life' — are we sequencing a suboptimal partition?;Intragenomic variation of the rDNA internal transcribed spacers in sponges (Phylum Porifera): implications for phylogenetic studies.;Keeping taxonomy based in morphology;DNA primers for amplification of mitochondrial cytochrome c oxidase subunit I from diverse metazoan invertebrates.;Genetic differentiation between morphotypes of the marine sponge Suberites ficus (Demospongiae: Hadromerida);A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences;The detection of disease clustering and a generalized regression approach.;Systema Porifera: A Guide To The Classification Of Sponges;Spatial patterns in sponge biodiversity in a Caribbean coral reef;Sponge ecology in the molecular era.;Polyphyly and rapid diversification in the marine sponge suborder Haplosclerina;Mitochondrial diversity of early-branching metazoa is revealed by the complete mt genome of a haplosclerid demosponge.;DNA barcoding: How it complements taxonomy, molecular phylogenetics and genetics;An evaluation of support for order-level monophyly and interrelationships within the class Demospongiae using partial data from the large subunit rDNA and cytochrome oxidase subunit I.;Family Suberitidae Schmidt, 1870;Phylogeography and taxonomy of the Indo-Pacific reef cave dwelling coralline demosponge Astrosclera 'willeyana': new data from nuclear internal transcribed spacer sequences;A molecular framework for the phylogeny of Sponges (Porifera);Phylogenetic relationships of tetractinellid sponges (Porifera: Demospongiae: Astrophorida) inferred from mitochondrial DNA sequences;Nuclear markers in evolutionary and population genetic studies of scleractinian corals and sponges;Suberitidae (Demospongiae, Hadromerida) from the North Aegean Sea;Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics;The sponges of the west-central Pacific"
        ],
        [
         "37",
         "035c95b92e2b71202ebcc9e53350745dd7a95007",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/24/2024.10.22.619592.full.pdf",
         "Medicine,Biology",
         "bioRxiv",
         "Grace N. Hibshman,David W. Taylor",
         "1627784745,144467204",
         "0",
         "0.0",
         "2025-02-11",
         "2025",
         "",
         ""
        ],
        [
         "38",
         "036ab69ea914ce7d5da1f991b9ee3f75109c8d6e",
         "https://www.biorxiv.org/content/biorxiv/early/2024/02/21/2024.02.18.580880.full.pdf",
         "Biology,Medicine",
         "bioRxiv",
         "Christina L. Lebonville,J. Rinker,Krysten O’Hara,Christopher S McMahan,Michaela Hoffman,Howard C. Becker,Patrick J. Mulholland",
         "2284991527,3527797,147240235,2214047354,47979106,2254022763,2238432947",
         "0",
         "0.0",
         "2025-02-06",
         "2025",
         "",
         ""
        ],
        [
         "39",
         "03a298b7781e6da3e58d530bf8f3e70b6952836d",
         "https://www.biorxiv.org/content/biorxiv/early/2024/03/13/2024.03.11.584453.full.pdf",
         "Biology",
         "bioRxiv",
         "Yari Cerruti,Daniela Fusco,Mahdis Jahanbin,Davide Marnetto,Paolo Provero",
         "2291363099,2256788247,2291367081,4015732,2281595923",
         "0",
         "0.0",
         "2025-01-22",
         "2025",
         "",
         ""
        ],
        [
         "40",
         "03bf8a25798de436ab666b3c42754c2de89ff482",
         "https://www.researchsquare.com/article/rs-5417187/latest.pdf",
         "Computer Science",
         "Journal of Real-Time Image Processing",
         "Xingyu Wang,Nini Hao,Yu Yun,Mengchao Zhang,Yuan Zhang,Zeqing Zhong",
         "2338668577,153238869,2338588110,47474450,2307435756,2338694403",
         "0",
         "0.0",
         "2025-01-01",
         "2025",
         "a15a38de3164fc4cb34b5fa3d89ffcbd64b71aea;3925f29aea2d29050f10eed02977d38272ba8cfc;27b44b66fdbdeebe3284a48587428dc439a33d79;a88c7d29a82dda65b0a7c6312d80398eecc2bd66;3973dc32900f1579796a7004ed3b04f01a7fbc46;7793cd7bef06781e7cff5f3611a20735549d272a;d0ae9e9404393d6860402ddb5e686bdff7d4bb3b;a2c7c57f87697c8f77d55b7bf254b2a1e0514ecc;d0f5beb6a0cd23c128514679ed9173904769b337;c52129611e204dc8dea3afdb60cc37ec5ad416c0;62208b9e88f081ed60531c92b20273d081085c91",
         "A novel combined method for conveyor belt deviation discrimination under complex operational scenarios;FastBeltNet: a dual-branch light-weight network for real-time conveyor belt edge detection;SCTNet: Single-Branch CNN with Transformer Semantic Information for Real-Time Segmentation;Machine vision based damage detection for conveyor belt safety using Fusion knowledge distillation;Real-time and accurate defect segmentation of aluminum strip surface via a lightweight network;Coal Mine Belt Conveyor Foreign Objects Recognition Method of Improved YOLOv5 Algorithm with Defogging and Deblurring;An intelligent belt wear fault diagnosis method based on deep learning;Research on Deviation Detection of Belt Conveyor Based on Inspection Robot and Deep Learning;Research in placement of measuring sensors on hexagonal idler housing with regard to requirements of pipe conveyor failure analysis;OSED: Object-specific edge detection;The conveyor belt longitudinal tear on-line detection based on improved SSR algorithm☆"
        ],
        [
         "41",
         "03d6f4500d4696a207d73b7b5c905492d5420acb",
         "https://www.researchsquare.com/article/rs-4951492/latest.pdf",
         "Medicine",
         "Parasitology Research",
         "Safiye Merve Bostancioglu,O. Mutlu",
         "35299140,19270514",
         "0",
         "0.0",
         "2025-01-01",
         "2025",
         "1165f51b633ccaba951bfd521a91695e619efd64;e78abfefd0a969338f7d6edf4d6c43a5fa2ef9d3;db75067e9e5b32f436a75d635b86dd1e6e4fb44b;043430cc5fb8747b4b90c837e21bbf77cf90cb4e;dc32a984b651256a8ec282be52310e6bd33d9815;339a8e4cb0eba77675711ac255ac2a5d7ede1d53;fd95a48cbffb5e93f90f85844cd03ca01a59cded;526788055f6b459d996931a23d3eb3f907b653e0;235b82169c6bfab170e24f31f85854264e4a07b7;eb1c560b55430d24758ccfd3718b858408da9e5d;f7bc06d4944c5577db0a99868f96d6902f79e928;36abaca9db1327fdff3dfdd07a58aae61eede7d2;afccd2f65a262cb86b96a181fa51a25b454edd94;5325a5846f0c1a48033ebf56b56dd15117ed2423;ca3e6d92f6f2f86b618dc0a45e37cdb8013d20d1;d6e1bd7af910a96502a9d62c04e696c6e635666c;184047d8d1c7e31b2a2b3316e90f03284916bd15;72fb9db6f5ebd2ad093e4fe7ab20da524c48a61f;29a7ffbcbe2069a274531b0471595deefad91623;a6e847ccaca59340e1693fbbde85fb87bd75d6a7;d81327ef1cf1e9b32adcbcc7b5bca32d4e8ebb64;67ff5a2652e8e794d25cf278fffe2695a07ea1a3;68b3140e974ddf7f49eb4286e76ca61889d2df1a;d5654bc484fc08301fbc7e2710e40a2d6266367e;6fc1dc70c76980ff172ff8736c817c2a21c2ed63;70bbdc8cd60ece56b94c29bc7e52f1619d2cf2c3;8f5c082bf8cefce865d5aa2a70b5117769b70b93;89e74df7bc331399a4277e2b4100f899213dba1e;48f64a1ed40cf53ccdc06f1b847845d7586e2d96;af61d8c484e0b4527b84ddaf0c314a92121265be;895860c6083736508d2541900cdf0960eb11592f;9c83e6a75f008e10b8f0be5a57069675c3c492e6;fc43d39e9ff298271329f1ce656076e0d41b38e9;5e33907a9505d001542085ddcb826270a0c224c3;2e571af231168185d7fe24c7766d6cfd4c5f044d;cf422922561f8ae328fe4d5fff0ba48453129bd6;f7bc3ec84dae259d627bec42d54d21b0530c7813;0f64423355445da736c70ba784d56de48116bfeb;6c908a9d0d235da8f79412813d8f7594f0f52f6d;307b46bfb2ae98d38e1f1e8e2a136214f60a301d;521027c14a7ef622a4f03106799706d0824e192a;3a324c0037acd51f8e745681ea9b441b99640167;778f21ebec86722b1b98638d790c7272ab9b0eea;579de0a16edb6b1ebb14adb720933527881cc18f;98a91496ba69b92b8b301c9ee4edecdcfee6c03c;548fe25043edbe4539c68550383875466ed0d777;0d61aeac8cef27fc5e14160f8a0b88b975b66e3c;56f4309aabb4ad1a2a25fbfa25112e44d16e450c",
         "Anti-parasitic drug discovery against Babesia microti by natural compounds: an extensive computational drug design approach;Case report: First report on human infection by tick-borne Babesia bigemina in the Amazon region of Ecuador;Lactate Dehydrogenase as a Potential Therapeutic Drug Target to Control Babesia bigemina;The Structural Basis of Babesia orientalis Lactate Dehydrogenase;Highly accurate protein structure prediction with AlphaFold;MEGA11: Molecular Evolutionary Genetics Analysis Version 11;Chemoprophylaxis for babesiosis and anaplasmosis in cattle: case report.;BRENDA, the ELIXIR core data resource in 2021: new developments and updates;Scalable molecular dynamics on CPU and GPU architectures with NAMD.;Development of unstable resistance to diminazene aceturate in Babesia bovis;Functional and structural characterization of the pentapeptide insertion of Theileria annulata lactate dehydrogenase by site-directed mutagenesis, comparative modeling and molecular dynamics simulations.;Comparative Protein Structure Modeling Using MODELLER;CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field;Structure-Based Virtual Screening for Drug Discovery: Principles, Applications and Recent Advances;Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences;Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments;Babesia: a world emerging.;DoGSiteScorer: a web server for automatic binding site prediction, analysis and druggability assessment;Kinetic analysis of the amino terminal end of active site loop of lactate deyhdrogenase from Plasmodium vivax.;Current Advances in Detection and Treatment of Babesiosis;Investigations Into the Nature, Causation, and Prevention of Texas or Southern Cattle Fever;Open Babel: An open chemical toolbox;Emerging perspectives in the research of bovine babesiosis and anaplasmosis.;CHARMM general force field: A force field for drug‐like molecules compatible with the CHARMM all‐atom additive biological force fields;QMEAN: A comprehensive scoring function for model quality assessment;Epik: a software program for pKa prediction and protonation state generation for drug-like molecules;Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes.;Chemotherapy against babesiosis.;Mapping the binding site for gossypol-like inhibitors of Plasmodium falciparum lactate dehydrogenase.;Importance of accurate charges in molecular docking: Quantum mechanical/molecular mechanical (QM/MM) approach;UCSF Chimera—A visualization system for exploratory research and analysis;Identification of Babesia bovis L-lactate dehydrogenase as a potential chemotherapeutical target against bovine babesiosis.;Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy.;Structure of Toxoplasma gondii LDH1: active-site differences from human lactate dehydrogenases and the structural basis for efficient APAD+ use.;Evolution of Cryptosporidium parvum lactate dehydrogenase from malate dehydrogenase by a very recent event of gene duplication.;Toward a pharmacophore for drugs inducing the long QT syndrome: insights from a CoMFA study of HERG K(+) channel blockers.;Cheminformatic models to predict binding affinities to human serum albumin.;Chloroquine Binds in the Cofactor Binding Site ofPlasmodium falciparum Lactate Dehydrogenase*;The structure of lactate dehydrogenase from Plasmodium falciparum reveals a new target for anti-malarial design;Thirty years of environmental health research--and growing.;VMD: visual molecular dynamics.;Recognition of errors in three‐dimensional structures of proteins;Verification of protein structures: Patterns of nonbonded atomic interactions;The neighbor-joining method: a new method for reconstructing phylogenetic trees.;Conversion of glucose of lactate by intraerythrocytic Plasmodium berghei.;The Protein Data Bank;Prediction of the Brain-Blood Distribution of a Large Set of Drugs from Structurally Derived Descriptors Using Partial Least-Squares (PLS) Modeling;A method to identify protein sequences that fold into a known three-dimensional structure."
        ],
        [
         "42",
         "03ea505917144ae863c916e9aece54480f264a49",
         "https://researchonline.ljmu.ac.uk/id/eprint/22956/1/COMST-Energy%20Conservation%20in%20Passive%20Optical%20Networks-Accepted.pdf",
         "Computer Science",
         "IEEE Communications Surveys and Tutorials",
         "S. H. Shah Newaz,E. Ahvar,Md. Shamim Ahsan,J. Kamruzzaman,Gour C. Karmakar,Gyu Myoung Lee",
         "2276809949,3106336,1792022,1683337,2261581625,2345377884",
         "0",
         "0.0",
         "2025-02-01",
         "2025",
         "",
         ""
        ],
        [
         "43",
         "04180d1bfe001f4460d0d5e57104fba7bec7f5da",
         "https://www.biorxiv.org/content/biorxiv/early/2024/02/28/2024.02.27.582377.full.pdf",
         "Biology",
         "bioRxiv",
         "Yan-Xue Li,Xin-Le Kang,Yan-Li Li,Xiao-Pei Wang,Qiao Yan,Jin-Xing Wang,Xiao-Fan Zhao",
         "2125247950,16305120,2170502373,2144675201,2288581870,79800101,2324965721",
         "1",
         "0.0",
         "2025-01-15",
         "2025",
         "63715c5b15a91079f146f6ad6f2cedc87725d655;32c717642948dbb2f6615685a7917ba837b78189;5bfef3ebb5d03f183b21d934e5378ad1a5e0ece8;d6e13ae4a7172647cde9de046ba7f3a19dd184b5;9429e61387fab5cb2d40a80c2b42db6a706de0b0;c440d5d6f7de5c5d61654599dd6039bd4f067102;a2cb21422e18d5550dea0f43e7d68a7a4a4d3fbe;537cb881f571c5c56dc49cb96a35069c6f0375bc;7115e4c3305fa1e364411c773f7d3f3b1368f4b8;5931bca902f48f1422685e7a81559ce04114cd99;dcc6dc372d215ed18c239649ab9a60e1eee191e6;a2514e8b71d14863d51c73dc346176990c0ab800;3671c34c4d471f1205c0f5614615cc8d1154598f;6d0f80c077b256cdd40313c3cef5414e012fe9e2;2e247f74c50e5579f6edd08ed8cffaa9a92b5396;8497c9cc97cd4e9bc4fcb7e017bae1bddb191022;409559893134d972327a18982361017c5247ec93;e7c8aa2cb2223f17615c1b1ae3b33095466e95cc;ec6832035dea84b44a45cadc0ce7af4dc8c7787b;334b6127ed5a74d0d9b02a5d22b7469bde3703c2;a838c336ba73206dc75b68e24f6d38f2f4969eca;73373a156cb24b578938096b18012b1f0ea1f574;6a964b931019ac30d03570824bd4fdd8b1ecdf20;9edf24bcde752b584bccf36d0bf2ddd648a38172;15aa036f9761495ca881a3f21b09ceab90821104;a47fa8362c5b19b3426d737c00743556456fe5fe;fbf6199b2a430ebf1471e6b99ce814fed7f9d5e1;17e965223e9fbf39657d5e7f1e34d7eab9628d42;5e68d6444fe32b07895bfb03c222560a14171447;732158d3001cd14af2c8efc4df3696698ae78b6d;9a37767c0a038b42b14c8f4dd7359ef040107596;786ef6bd537acba324f8f265009845cea9b0992c;746aaa51db498010b3528a34c655813b847bdadc;80386d294405a3078ac25dd10f3575faf2a37050;96ec0727355cc2f12d66d3e5efb39acea2555b49;dc824817727a71d5330f9eccddc9dc612bdd734d;08f32d5024755875dd347a6cd9e58a7af904702a",
         "Krüppel-homolog 1 exerts anti-metamorphic and vitellogenic functions in insects via phosphorylation-mediated recruitment of specific cofactors;The vitellogenin receptor functionality of the migratory locust depends on its phosphorylation by juvenile hormone;Neuropeptide ACP facilitates lipid oxidation and utilization during long-term flight in locusts;Fibroblast Growth Factor Receptor, a Novel Receptor for Vegetative Insecticidal Protein Vip3Aa;Feedback regulation of cytoneme-mediated transport shapes a tissue-specific FGF morphogen gradient;Identification of G protein-coupled receptors required for vitellogenin uptake into the oocytes of the red flour beetle, Tribolium castaneum;Protein kinase C modulates transcriptional activation by the juvenile hormone receptor methoprene-tolerant.;Reassessing ecdysteroidogenic cells from the cell membrane receptors’ perspective;Juvenile hormone-activated phospholipase C pathway enhances transcriptional activation by the methoprene-tolerant protein;Krüppel-homolog 1 mediates juvenile hormone action to promote vitellogenesis and oocyte maturation in the migratory locust.;Two-Tiered Control of Epithelial Growth and Autophagy by the Insulin Receptor and the Ret-Like Receptor, Stitcher;Receptor tyrosine kinases in Drosophila development.;Functions and Mechanisms of Fibroblast Growth Factor (FGF) Signalling in Drosophila melanogaster;Ligand-binding properties of a juvenile hormone receptor, Methoprene-tolerant;Drosophila Met and Gce are partially redundant in transducing juvenile hormone action.;Establishment of a New Cell Line from Lepidopteran Epidermis and Hormonal Regulation on the Genes;Posttranscriptional control of the competence factor βFTZ-F1 by juvenile hormone in the mosquito Aedes aegypti;Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.;Multiple actions of steroid hormones--a focus on rapid, nongenomic effects.;The modes of action of juvenile hormones: some questions we ought to ask.;The isolation of two juvenile hormone-inducible genes in Drosophila melanogaster.;RNAi Double-Stranded RNA Directs the ATP-Dependent Cleavage of mRNA at 21 to 23 Nucleotide Intervals;Fast induction of vitellogenin gene expression by juvenile hormone III in the cockroach Blattella germanica (L.) (Dictyoptera, Blattellidae).;Spruce budworm (Choristoneura fumiferana) juvenile hormone esterase: hormonal regulation, developmental expression and cDNA cloning;Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans;The Disulfide Bonds in the C-terminal Domains of the Human Insulin Receptor Ectodomain*;Juvenile hormone controls early trypsin gene transcription in the midgut of\n Aedes aegypti;heartless encodes a fibroblast growth factor receptor (DFR1/DFGF-R2) involved in the directional migration of early mesodermal cells in the Drosophila embryo.;Juvenile hormone regulation of an insect gene: a specific transcription factor and a DNA response element;Murine FGFR-1 is required for early postimplantation growth and axial organization.;Regulation of juvenile hormone esterase gene expression in the tobacco budworm (Heliothis virescens).;Microscale Thermophoresis (MST) to Detect the Interaction Between Purified Protein and Small Molecule.;A decade with the juvenile hormone receptor;Determination by LC-745 MS of juvenile hormone titers in hemolymph of the silkworm, Bombyx mori;952 The tyrosine kinase Stitcher activates Grainy head and epidermal wound healing in Drosophila . 953;Isothermal titration calorimetry.;Evidence that autophosphorylation of 820 solubilized receptors for epidermal growth factor is mediated by intermolecular cross-821 phosphorylation;Research;Juvenile Hormones O, I, and II Exist Only in Lepidoptera;Primate Atlas;The hormonal mechanism of larval molting and metamorphosis in the silkworm, 807 Bombyx mori;characteristic ( Figure 6 ). This study presents a platform to identify the agonist or inhibitor of JH 392 cell membrane receptors to develop an environmental friend insect growth regulator;receptor tyrosine kinase, juvenile hormone, cell membrane receptor, methoprene protein 1;induced gene expression;2022. Egfr signaling 866 promotes juvenile hormone biosynthesis in the German cockroach;2022. Juvenile 812 hormone membrane signaling phosphorylates USP and thus potentiates 20-hydroxyecdysone 813 action in Drosophila"
        ],
        [
         "44",
         "04194fd0c5a6cfc5decf792900c5711fd7a98753",
         "https://www.biorxiv.org/content/biorxiv/early/2024/01/23/2024.01.10.574820.full.pdf",
         "Biology",
         "bioRxiv",
         "A. Sukys,Ramon Grima",
         "1746173807,2237143604",
         "2",
         "0.0",
         "2025-01-30",
         "2025",
         "",
         ""
        ],
        [
         "45",
         "0419bc34536d0f4a3f60accd0745fa7ab3eb6d6b",
         "http://arxiv.org/pdf/2203.09430",
         "Computer Science",
         "Displays (Guildford)",
         "Erkang Chen,L. Tong,Tian Ye,Sixiang Chen,Yunchen Zhang,Yun Liu",
         "2150974076,2159671912,2265679826,2111636335,2140010411,2144443867",
         "0",
         "0.0",
         "2025-04-01",
         "2025",
         "5d0880e662c293df497975266ada86f7492142ff;a03f2b80e2d77a801c2f79af31fc520878805ae2;e29fdb064718ec1fcb963b07495de76f926602e0;d0f8169b2ad089f565eb5859d55534e1f8c303aa;b8c9b1948a233e68f1fbef8b8ca4bf90c262efff;2c269c3d128375dba7b2fe7215a1c3c615ff76f6;07bbbb13d34a77e2e4a001044e5c4fd03dfd7b35;ad5467dae28696cd5ebc6d0a662aac1ee884c7fb;29a5b82d4fc061746f31951edc071751e4c45f2b;36169ad27b5ef7a7c664bf9b01d334a4fc0bdf52;37246e26163cdd0f5ddd1ea47a5a5019dead8abb;7b3cd13b5d5dad3e9204674bab80ce3578e552d8;f9295ac4abc2235414b9f34752f1cf745dbfaa9a;7d9296222ac17bfcd242a27b29556baa02eedafa;d3faa88ddedd47d5d2acb05a706dc5c6f5d14743;439f47cf271c9e503064bfbc8442d9245c647867;e35aceb36036889b50659a6d5ab4612f170fed1d;22418b7b7d5d6d18b81f232f60c22a15d1c8a38a;818dfb10c898a77a0a84a1eddcdedd069d71763b;abf75b81b96ff2cd262086675deba55f907108db;896e160b98d52d13a97caa664038e37e86075ee4;a127899f71b6ebc03521d2f9538335ebf1f29d78;042e12e44ad99e55f509b2128d9a5bea2212e0c7;a2cad4e4fd946adf6cc87e483b2ba18579de1264;3360c3ccdf73539200491fe958295302fef04d6e;eae2e0fa72e898c289365c0af16daf57a7a6cf40;870fdfe89c440f35ce83b915519001ea0da518ec;2ed176a7d062b4d02e3e08f8b0a8cf7058fe7f98;c2aa46af887264490350b11ed585d74381f8ee2f",
         "Visibility restoration for real-world hazy images via improved physical model and Gaussian total variation;Degradation-adaptive neural network for jointly single image dehazing and desnowing;Frequency-guidance Collaborative Triple-branch Network for single image dehazing;Depth color correlation-guided dark channel prior for underwater image enhancement;DHIQA: Quality assessment of dehazed images based on attentive multi-scale feature fusion and rank learning;MFID-Net: Multi-scaled feature-fused image dehazing via dynamic weights;Single Image Dehazing Using Saturation Line Prior;NightHazeFormer: Single Nighttime Haze Removal Using Prior Query Transformer;Multi-Purpose Oriented Single Nighttime Image Haze Removal Based on Unified Variational Retinex Model;Image Dehazing Transformer with Transmission-Aware 3D Position Embedding;Vision Transformers for Single Image Dehazing;IDRLP: Image Dehazing Using Region Line Prior;From Synthetic to Real: Image Dehazing Collaborating with Unlabeled Real Data;Effective Meta-Attention Dehazing Networks for Vision-Based Outdoor Industrial Systems;Towards Domain Invariant Single Image Dehazing;Effective Data-Driven Technology for Efficient Vision-Based Outdoor Industrial Systems;Single Image Dehazing Using Haze-Lines;FFA-Net: Feature Fusion Attention Network for Single Image Dehazing;Semi-Supervised Image Dehazing;Benchmarking Single-Image Dehazing and Beyond;NIMA: Neural Image Assessment;DehazeNet: An End-to-End System for Single Image Haze Removal;A Fast Single Image Haze Removal Algorithm Using Color Attenuation Prior;No-Reference Image Quality Assessment in the Spatial Domain;Single Image Haze Removal Using Dark Channel Prior;Image quality assessment: from error visibility to structural similarity;Two deterministic half-quadratic regularization algorithms for computed imaging;Remote sensing image dehazing method based on global and local advantageous feature fusion;An algorithm for enhancing low-light images at sea based on improved dark channel priors;Multiscale Depth Fusion With Contextual Hybrid Enhancement Network for Image Dehazing;International Journal of Computer Vision 48(3), 233--254, 2002"
        ],
        [
         "46",
         "042cde9aa3dc3d91bb528dfff54349708df45e8a",
         "https://www.researchsquare.com/article/rs-3850792/latest.pdf",
         "Medicine",
         "eLife",
         "Flavie Coquel,Sing-Zong Ho,K. Tsai,Chun-Yen Yang,Antoine Aze,Julie Devin,Ting-Hsiang Chang,Marie Kong-Hap,Audrey Bioteau,J. Moreaux,Domenico Maiorano,Philippe Pourquier,Wen-Chin Yang,Yea-Lih Lin,Philippe Pasero",
         "2343069081,2343510294,6448562,2343406411,2321532058,144153902,2343795799,2343069760,2343068935,2298264205,2285342209,2343069758,2301845715,2260606772,2248688576",
         "0",
         "0.0",
         "2025-01-31",
         "2025",
         "",
         ""
        ],
        [
         "47",
         "044b3471e2d2bb3d2ae6fa4f80e1a2836e113caa",
         "https://www.biorxiv.org/content/biorxiv/early/2024/11/03/2024.10.29.620862.full.pdf",
         "Biology",
         "bioRxiv",
         "Alexander J. Petri,Kristoffer Sahlin",
         "2220871413,38100195",
         "0",
         "0.0",
         "2025-02-19",
         "2025",
         "f8af1516ccb36ea12b39b477107f42b3fb5db165;0511bfa93681aaa19e494bcb03d56f398b4ea9cd;05cbfea0c46cb7f22b8a4bd61a841f94987237ec;e3270311b660a605a65c5f0e32e9e6a15e6a5d58;f48444e115bc57b2a0097dcc7b5c7a2fb601f9d7;916023acbfee776d83ca6dc9f0a80d967ccba545;bfa4726627cdfb1f345e0fa43ed8d37e8cc6f6f1;1de401be490484a6a967366742899a8229428b2d;10729403e3560dd4d7298253ece4a3a7ddbee10f;fa1656233ace990aaac9caef380b277f8b829fac;ca63fa1382ad118993eadb386bfe9250f379c71f;bff639f7b02be17dca93ee0d8515144fbd79944e;35732c17dcf18f8f6363630a085a0566535265c3;a26cc7fd405bdff9435a251c3ab5946a512099cd;f11e91018c8cfd511ea71e8cd35cb126beff797c;15183caa56fd8eba5c05e95a7adc444cf2379598;d95c49edac7cacbd859d9559956270944bad8964;9e4310a109f0f2c672d75bdbc51328510a7e6d94;dfc1b32f47fed561e7c8f31ad5c2ff9c89e14d55;48c717002dc54a6fd58d06f8e00e2ba0e894a914;371782e43eb31762f25cd88492cf2a28f55b7f2d;7550021266b4fc1fa958bacd1c209eb8e19c2361;8d5b967fca549dad50e2ac7b6643fb504cfeb554;a47483dca56d93dabc54c8d49a5d03c6fb1e9771;3658413fe0008d7e4a4decaabcac0a3033f56d43;f9692de1e3f63d4bf2bea0a8dfbf1b28b07f1b43;ac808fbdf6d6f87ff12078849a6766d46f5b53e3;f2f85ec20bcae8d67035fd446b855a75b9a0aa8d;eefc01476aea1ff1a9d06f9837124ea13479e7c3;5421dbcb7e14766eb3d951910ae8d7892d735a01;584fd593dc69720afad8b1b5663cd9aea4da9c45;4d6dc037b6cf768393f35aa5726a8cda2ff736a9;fd3e3907e4888c264916e473f74c98f2f4ad9b02;866b90d114e591b88d042c105d949133241d0e65",
         "Assembly Arena: Benchmarking RNA isoform reconstruction algorithms for nanopore sequencing;isONform: reference-free transcriptome reconstruction from Oxford Nanopore data;Ecological genomics in the Northern krill uncovers loci for local adaptation across ocean basins;Benchmarking of Nanopore R10.4 and R9.4.1 flow cells in single-cell whole-genome amplification and whole-genome shotgun sequencing.;Accurate isoform discovery with IsoQuant using long reads;Method of the year: long-read sequencing;Freddie: annotation-independent detection and discovery of transcriptomic alternative splicing isoforms using long-read sequencing;Reference-free assembly of long-read transcriptome sequencing data with RNA-Bloom2;Rapid in situ identification of biological specimens via DNA amplicon sequencing using miniaturized laboratory equipment;ClusTrast: a short read de novo transcript isoform assembler guided by clustered contigs;Molecular phylogeny of fucoxanthin-chlorophyll a/c proteins from Chaetoceros gracilis and Lhcq/Lhcf diversity;Sustainable data analysis with Snakemake;NGSpeciesID: DNA barcode and amplicon consensus generation from long‐read sequencing data;Syncmers are more sensitive than minimizers for selecting conserved k‑mers in biological sequences;RATTLE: reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing;Error correction enables use of Oxford Nanopore technology for reference-free transcriptome analysis;Quantifying the benefit offered by transcript assembly with Scallop-LR on single-molecule long reads;Transcriptome assembly from long-read RNA-seq alignments with StringTie2;De novo clustering of long-read transcriptome data using a greedy, quality-value based algorithm;Deciphering highly similar multigene family transcripts from Iso-Seq data with IsoCon;CARNAC-LR : Clustering coefficient-based Acquisition of RNA Communities in Long Reads;MeShClust: an intelligent tool for clustering DNA sequences;MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets;Minimap2: pairwise alignment for nucleotide sequences;Clustering huge protein sequence sets in linear time;Rust-Bio: a fast and safe bioinformatics library;Widespread Polycistronic Transcripts in Fungi Revealed by Single-Molecule mRNA Sequencing;On comparing partitions;DNACLUST: accurate and efficient clustering of phylogenetic marker genes;V-Measure: A Conditional Entropy-Based External Cluster Evaluation Measure;Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences;Reducing storage requirements for biological sequence comparison;Spectral clustering of protein sequences;Base-calling of automated sequencer traces using phred. I. Accuracy assessment.;Highlyefficientclusteringoflong-readtranscriptomic data with geluster;Discovering novel reproductive genesinanon-modelflyusingdenovogridiontranscriptomics;Effectivesequencesimilaritydetectionwithstrobemers;Search and clustering orders of magnitude faster than blast;Minimizer-iter: Iterate over minimizers of a dna sequence"
        ],
        [
         "48",
         "044c7ebb747f02dc4f0a7e518030141baa15fa26",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/13/2024.10.12.617732.full.pdf",
         "Biology",
         "bioRxiv",
         "Hanwen Zhou,Yue Hu,Yulong Zheng,Jiefu Li,Jielong Peng,Jiang Hu,Yun Yang,Guoqing Zhang,Zefeng Wang",
         "2268724581,2323433025,2326271976,2155871719,2326612459,2332289515,2330362785,2293901008,2297462933",
         "0",
         "0.0",
         "2025-03-04",
         "2025",
         "",
         ""
        ],
        [
         "49",
         "047fe4832170430bc42950c5a249f42cabcacf6b",
         "http://arxiv.org/pdf/2502.12401",
         "Computer Science",
         "2025 IEEE PES Grid Edge Technologies Conference & Exposition (Grid Edge)",
         "Saeed Nematshahi,Amin Khodaei,Ali Arabnya",
         "2332220554,2256906534,2198000388",
         "1",
         "0.0",
         "2025-01-21",
         "2025",
         "",
         ""
        ]
       ],
       "shape": {
        "columns": 12,
        "rows": 2239
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>fieldsOfStudy</th>\n",
       "      <th>venue</th>\n",
       "      <th>authorName</th>\n",
       "      <th>authorId</th>\n",
       "      <th>citationCount</th>\n",
       "      <th>influentialCitationCount</th>\n",
       "      <th>publicationDate</th>\n",
       "      <th>publicationYear</th>\n",
       "      <th>referenceIds</th>\n",
       "      <th>referenceTitles</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>000a3e6667afb4ca8476ce15555b05244c3b458a</td>\n",
       "      <td>https://www.nature.com/articles/s41598-025-860...</td>\n",
       "      <td>Medicine</td>\n",
       "      <td>Scientific Reports</td>\n",
       "      <td>Fenghui Lian,Yingjie Sun,Meiyu Li</td>\n",
       "      <td>2064292642,2340550389,2109040952</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-01-15</td>\n",
       "      <td>2025</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>00139e24da15df3f2f4ef70a92a781ccf315ef47</td>\n",
       "      <td>https://www.biorxiv.org/content/biorxiv/early/...</td>\n",
       "      <td>Biology</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Jana Jung,Timo Glatter,Marco Herfurth,L. Søgaa...</td>\n",
       "      <td>2346560940,2277617384,1992769227,1398654276</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-02-19</td>\n",
       "      <td>2025</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0027ea27f5986f9c6ee9776e10289c7f2aa089ca</td>\n",
       "      <td>http://arxiv.org/pdf/2501.13553</td>\n",
       "      <td>Computer Science</td>\n",
       "      <td>International Conference on Compiler Construction</td>\n",
       "      <td>Robert Szafarczyk,S. Nabi,W. Vanderbauwhede</td>\n",
       "      <td>2168878518,1714681,1692663</td>\n",
       "      <td>1</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-01-23</td>\n",
       "      <td>2025</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>00291c982d2d9e69d8fe8c89a7db1fbb25f15687</td>\n",
       "      <td>https://www.biorxiv.org/content/biorxiv/early/...</td>\n",
       "      <td>Biology</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Affiong I. Oqua,Kin Chao,Liliane El Eid,Lisa C...</td>\n",
       "      <td>30455986,2283562588,2182492272,2308111697,2338...</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-01-02</td>\n",
       "      <td>2025</td>\n",
       "      <td>346ef7fdf555fc0396f9e954bcda723a2fb1b356</td>\n",
       "      <td>Imperial</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00317a3206df00f0c3c91595a349bc81b7118929</td>\n",
       "      <td>http://arxiv.org/pdf/2502.03676</td>\n",
       "      <td>Computer Science</td>\n",
       "      <td>IEEE Robotics and Automation Letters</td>\n",
       "      <td>Yeping Wang,M. Gleicher</td>\n",
       "      <td>2115738494,2066395412</td>\n",
       "      <td>1</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-02-05</td>\n",
       "      <td>2025</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2234</th>\n",
       "      <td>fed4e1c03728d1747984ccc035e9e52d4f576666</td>\n",
       "      <td>https://www.biorxiv.org/content/biorxiv/early/...</td>\n",
       "      <td>Biology</td>\n",
       "      <td>bioRxiv</td>\n",
       "      <td>Niko Kasalo,Mirjana Domazet-Lošo,Tomislav Doma...</td>\n",
       "      <td>2145558181,1414104312,1397755049</td>\n",
       "      <td>2</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-01-16</td>\n",
       "      <td>2025</td>\n",
       "      <td>519ea2ff9af9e86c6bc85d1e07ef5c337bcfbac4;4aa83...</td>\n",
       "      <td>The Origin of Metazoan Multicellularity: A Pot...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2235</th>\n",
       "      <td>fee975b0352fbf35fdc1926b25a16d8531ebb4e6</td>\n",
       "      <td>https://www.researchsquare.com/article/rs-4129...</td>\n",
       "      <td>Computer Science</td>\n",
       "      <td>Multim. Syst.</td>\n",
       "      <td>Liang Yang,Qi Yang,Jingjie Zeng,Tao Peng,Zhiha...</td>\n",
       "      <td>2143920912,2308550856,2000361118,2068930271,15...</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-02-06</td>\n",
       "      <td>2025</td>\n",
       "      <td>ec1967121c969690b60d5e6d3bd1480553e59dda;ae01b...</td>\n",
       "      <td>Turn-Level Active Learning for Dialogue State ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2236</th>\n",
       "      <td>ff622ab9a90f9881749716d70893b0f850247a4c</td>\n",
       "      <td>http://arxiv.org/pdf/2309.00093</td>\n",
       "      <td>Computer Science</td>\n",
       "      <td>at - Automatisierungstechnik</td>\n",
       "      <td>Ala' Alalabi,Kirsten Morris</td>\n",
       "      <td>2295510488,2237423222</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-04-01</td>\n",
       "      <td>2025</td>\n",
       "      <td>f0ca0783030328fca2d43745b255d01e37681741;3aa0c...</td>\n",
       "      <td>Stabilization of a Parabolic-Elliptic System v...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2237</th>\n",
       "      <td>ff8fd5f30461bb788c6a32f37fe0f7821eafebf6</td>\n",
       "      <td>http://arxiv.org/pdf/2501.00799</td>\n",
       "      <td>Computer Science,Mathematics</td>\n",
       "      <td>IEEE Signal Processing Letters</td>\n",
       "      <td>S. Mukhopadhyay,Debasmita Mukherjee</td>\n",
       "      <td>46287665,2338269614</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-01-01</td>\n",
       "      <td>2025</td>\n",
       "      <td>2cfe7dd4b34ab75165c911c8a62edd0036bc2d77;c3855...</td>\n",
       "      <td>Improved RIP-based bounds for guaranteed perfo...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2238</th>\n",
       "      <td>ffd00ce568f98d0febe6aeb407b54f9075ea732b</td>\n",
       "      <td>http://arxiv.org/pdf/2501.04448</td>\n",
       "      <td>Physics</td>\n",
       "      <td>Proceedings of 42nd International Conference o...</td>\n",
       "      <td>Pietro Vischia</td>\n",
       "      <td>2339007853</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2025-01-07</td>\n",
       "      <td>2025</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2239 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                       paperId  \\\n",
       "0     000a3e6667afb4ca8476ce15555b05244c3b458a   \n",
       "1     00139e24da15df3f2f4ef70a92a781ccf315ef47   \n",
       "2     0027ea27f5986f9c6ee9776e10289c7f2aa089ca   \n",
       "3     00291c982d2d9e69d8fe8c89a7db1fbb25f15687   \n",
       "4     00317a3206df00f0c3c91595a349bc81b7118929   \n",
       "...                                        ...   \n",
       "2234  fed4e1c03728d1747984ccc035e9e52d4f576666   \n",
       "2235  fee975b0352fbf35fdc1926b25a16d8531ebb4e6   \n",
       "2236  ff622ab9a90f9881749716d70893b0f850247a4c   \n",
       "2237  ff8fd5f30461bb788c6a32f37fe0f7821eafebf6   \n",
       "2238  ffd00ce568f98d0febe6aeb407b54f9075ea732b   \n",
       "\n",
       "                                          openAccessPdf  \\\n",
       "0     https://www.nature.com/articles/s41598-025-860...   \n",
       "1     https://www.biorxiv.org/content/biorxiv/early/...   \n",
       "2                       http://arxiv.org/pdf/2501.13553   \n",
       "3     https://www.biorxiv.org/content/biorxiv/early/...   \n",
       "4                       http://arxiv.org/pdf/2502.03676   \n",
       "...                                                 ...   \n",
       "2234  https://www.biorxiv.org/content/biorxiv/early/...   \n",
       "2235  https://www.researchsquare.com/article/rs-4129...   \n",
       "2236                    http://arxiv.org/pdf/2309.00093   \n",
       "2237                    http://arxiv.org/pdf/2501.00799   \n",
       "2238                    http://arxiv.org/pdf/2501.04448   \n",
       "\n",
       "                     fieldsOfStudy  \\\n",
       "0                         Medicine   \n",
       "1                          Biology   \n",
       "2                 Computer Science   \n",
       "3                          Biology   \n",
       "4                 Computer Science   \n",
       "...                            ...   \n",
       "2234                       Biology   \n",
       "2235              Computer Science   \n",
       "2236              Computer Science   \n",
       "2237  Computer Science,Mathematics   \n",
       "2238                       Physics   \n",
       "\n",
       "                                                  venue  \\\n",
       "0                                    Scientific Reports   \n",
       "1                                               bioRxiv   \n",
       "2     International Conference on Compiler Construction   \n",
       "3                                               bioRxiv   \n",
       "4                  IEEE Robotics and Automation Letters   \n",
       "...                                                 ...   \n",
       "2234                                            bioRxiv   \n",
       "2235                                      Multim. Syst.   \n",
       "2236                       at - Automatisierungstechnik   \n",
       "2237                     IEEE Signal Processing Letters   \n",
       "2238  Proceedings of 42nd International Conference o...   \n",
       "\n",
       "                                             authorName  \\\n",
       "0                     Fenghui Lian,Yingjie Sun,Meiyu Li   \n",
       "1     Jana Jung,Timo Glatter,Marco Herfurth,L. Søgaa...   \n",
       "2           Robert Szafarczyk,S. Nabi,W. Vanderbauwhede   \n",
       "3     Affiong I. Oqua,Kin Chao,Liliane El Eid,Lisa C...   \n",
       "4                               Yeping Wang,M. Gleicher   \n",
       "...                                                 ...   \n",
       "2234  Niko Kasalo,Mirjana Domazet-Lošo,Tomislav Doma...   \n",
       "2235  Liang Yang,Qi Yang,Jingjie Zeng,Tao Peng,Zhiha...   \n",
       "2236                        Ala' Alalabi,Kirsten Morris   \n",
       "2237                S. Mukhopadhyay,Debasmita Mukherjee   \n",
       "2238                                     Pietro Vischia   \n",
       "\n",
       "                                               authorId  citationCount  \\\n",
       "0                      2064292642,2340550389,2109040952              0   \n",
       "1           2346560940,2277617384,1992769227,1398654276              0   \n",
       "2                            2168878518,1714681,1692663              1   \n",
       "3     30455986,2283562588,2182492272,2308111697,2338...              0   \n",
       "4                                 2115738494,2066395412              1   \n",
       "...                                                 ...            ...   \n",
       "2234                   2145558181,1414104312,1397755049              2   \n",
       "2235  2143920912,2308550856,2000361118,2068930271,15...              0   \n",
       "2236                              2295510488,2237423222              0   \n",
       "2237                                46287665,2338269614              0   \n",
       "2238                                         2339007853              0   \n",
       "\n",
       "      influentialCitationCount publicationDate  publicationYear  \\\n",
       "0                          0.0      2025-01-15             2025   \n",
       "1                          0.0      2025-02-19             2025   \n",
       "2                          0.0      2025-01-23             2025   \n",
       "3                          0.0      2025-01-02             2025   \n",
       "4                          0.0      2025-02-05             2025   \n",
       "...                        ...             ...              ...   \n",
       "2234                       0.0      2025-01-16             2025   \n",
       "2235                       0.0      2025-02-06             2025   \n",
       "2236                       0.0      2025-04-01             2025   \n",
       "2237                       0.0      2025-01-01             2025   \n",
       "2238                       0.0      2025-01-07             2025   \n",
       "\n",
       "                                           referenceIds  \\\n",
       "0                                                         \n",
       "1                                                         \n",
       "2                                                         \n",
       "3              346ef7fdf555fc0396f9e954bcda723a2fb1b356   \n",
       "4                                                         \n",
       "...                                                 ...   \n",
       "2234  519ea2ff9af9e86c6bc85d1e07ef5c337bcfbac4;4aa83...   \n",
       "2235  ec1967121c969690b60d5e6d3bd1480553e59dda;ae01b...   \n",
       "2236  f0ca0783030328fca2d43745b255d01e37681741;3aa0c...   \n",
       "2237  2cfe7dd4b34ab75165c911c8a62edd0036bc2d77;c3855...   \n",
       "2238                                                      \n",
       "\n",
       "                                        referenceTitles  \n",
       "0                                                        \n",
       "1                                                        \n",
       "2                                                        \n",
       "3                                              Imperial  \n",
       "4                                                        \n",
       "...                                                 ...  \n",
       "2234  The Origin of Metazoan Multicellularity: A Pot...  \n",
       "2235  Turn-Level Active Learning for Dialogue State ...  \n",
       "2236  Stabilization of a Parabolic-Elliptic System v...  \n",
       "2237  Improved RIP-based bounds for guaranteed perfo...  \n",
       "2238                                                     \n",
       "\n",
       "[2239 rows x 12 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "543b145c-bffc-463f-954e-8c5c31586119",
       "rows": [
        [
         "0",
         "000a3e6667afb4ca8476ce15555b05244c3b458a",
         "Extracting organs of interest from medical images based on convolutional neural network with auxiliary and refined constraints",
         "None",
         "",
         "https://www.nature.com/articles/s41598-025-86087-8.pdf"
        ],
        [
         "1",
         "00139e24da15df3f2f4ef70a92a781ccf315ef47",
         "DdiA, an XRE family transcriptional regulator, regulates a LexA-independent DNA damage response in Myxococcus xanthus",
         "Repair of DNA damage is essential for genome integrity. DNA damage elicits a DNA damage response (DDR) that includes error-free and error-prone, i.e. mutagenic, repair. The SOS response is a widely conserved system in bacteria that regulates the DDR and depends on the recombinase RecA and the transcriptional repressor LexA. However, RecA/LexA-independent DDRs have been identified in several bacterial species. Here, using a whole-cell, label-free quantitative proteomics approach, we map the proteomic response in Myxococcus xanthus to mitomycin C treatment and the lack of LexA. In doing so, we confirm a LexA-independent DDR in M. xanthus. Using a candidate approach, we identify DdiA, a transcriptional regulator of the XRE family, and demonstrate that it regulates a subset of the LexA-independent DDR genes. ddiA is expressed heterogeneously in a subpopulation of cells in the absence of exogenous genotoxic stress and reversibly induced population-wide in response to such stress. DdiA, indirectly or directly, activates the expression of dnaE2, which encodes the DnaE2 error-prone DNA polymerase, and inhibits the expression of recX, which encodes RecX, a negative regulator of RecA. Accordingly, the ΔddiA mutant has a lower mutation frequency than the wild-type but also a fitness defect, suggesting that DdiA mediates a trade-off between fitness and mutagenesis. We speculate that the DdiA-dependent response is tailored to counter replication stress, thereby preventing the induction of the complete RecA/LexA-dependent DDR in the absence of exogenous genotoxic stress. Importance DNA damage repair is essential for genome integrity and depends on the DNA damage response (DDR). While the RecA/LexA-dependent SOS response is widely conserved in bacteria, there are also RecA/LexA-independent DDRs. Here, we identify the DNA damage-induced transcriptional regulator DdiA in Myxococcus xanthus and demonstrate that it regulates part of the RecA/LexA-independent DDR. DdiA activates the expression of dnaE2, which encodes the DnaE2 error-prone DNA polymerase, and inhibits the expression of recX, which encodes RecX, a negative regulator of RecA. Because the ΔddiA mutant has a lower mutation frequency than the wild-type but also a fitness defect, we suggest that DdiA mediates a trade-off between fitness and mutagenesis and that the DdiA-dependent DDR is specifically tailored to counter replication stress.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2025/02/19/2025.02.19.639066.full.pdf"
        ],
        [
         "2",
         "0027ea27f5986f9c6ee9776e10289c7f2aa089ca",
         "Compiler Support for Speculation in Decoupled Access/Execute Architectures",
         "Irregular codes are bottlenecked by memory and communication latency. Decoupled access/execute (DAE) is a common technique to tackle this problem. It relies on the compiler to separate memory address generation from the rest of the program, however, such a separation is not always possible due to control and data dependencies between the access and execute slices, resulting in a loss of decoupling. In this paper, we present compiler support for speculation in DAE architectures that preserves decoupling in the face of control dependencies. We speculate memory requests in the access slice and poison mis-speculations in the execute slice without the need for replays or synchronization. Our transformation works on arbitrary, reducible control flow and is proven to preserve sequential consistency. We show that our approach applies to a wide range of architectural work on CPU/GPU prefetchers, CGRAs, and accelerators, enabling DAE on a wider range of codes than before.",
         "",
         "http://arxiv.org/pdf/2501.13553"
        ],
        [
         "3",
         "00291c982d2d9e69d8fe8c89a7db1fbb25f15687",
         "Molecular mapping and functional validation of GLP-1R cholesterol binding sites in pancreatic beta cells",
         "G protein-coupled receptors (GPCRs) are integral membrane proteins which closely interact with their plasma membrane lipid microenvironment. Cholesterol is a lipid enriched at the plasma membrane with pivotal roles in the control of membrane fluidity and maintenance of membrane microarchitecture, directly impacting on GPCR stability, dynamics and function. Cholesterol extraction from pancreatic beta cells has previously been shown to disrupt the internalisation, clustering and cAMP responses of the glucagon-like peptide-1 receptor (GLP-1R), a class B1 GPCR with key roles in the control of blood glucose levels via the potentiation of insulin secretion in beta cells and weight reduction via the modulation of brain appetite control centres. Here, we unveil the detrimental effect of a high cholesterol diet on GLP-1R-dependent glucoregulation in vivo, and the improvement in GLP-1R function that a reduction in cholesterol synthesis using simvastatin exerts in pancreatic islets. We next identify and map sites of cholesterol high occupancy and residence time on active versus inactive GLP-1Rs using coarse-grained molecular dynamics (cgMD) simulations, followed by a screen of key residues selected from these sites and detailed analyses of the effects of mutating one of these residues, Val229, to alanine on GLP-1R interactions with cholesterol, plasma membrane behaviours, clustering, trafficking and signalling in pancreatic beta cells and primary islets, unveiling an improved insulin secretion profile for the V229A mutant receptor. This study 1) highlights the role of cholesterol in regulating GLP-1R responses in vivo; 2) provides a detailed map of GLP-1R - cholesterol binding sites in model membranes; 3) validates their functional relevance in beta cells; and 4) highlights their potential as locations for the rational design of novel allosteric modulators with the capacity to fine-tune GLP-1R responses.",
         "Imperial",
         "https://www.biorxiv.org/content/biorxiv/early/2024/06/25/2024.06.22.600087.full.pdf"
        ],
        [
         "4",
         "00317a3206df00f0c3c91595a349bc81b7118929",
         "Anytime Planning for End-Effector Trajectory Tracking",
         "End-effector trajectory tracking algorithms find joint motions that drive robot manipulators to track reference trajectories. In practical scenarios, anytime algorithms are preferred for their ability to quickly generate initial motions and continuously refine them over time. In this letter, we present an algorithmic framework that adapts common graph-based trajectory tracking algorithms to be anytime and enhances their efficiency and effectiveness. Our key insight is to identify guide paths that approximately track the reference trajectory and strategically bias sampling toward the guide paths. We demonstrate the effectiveness of the proposed framework by restructuring two existing graph-based trajectory tracking algorithms and evaluating the updated algorithms in three experiments.",
         "",
         "http://arxiv.org/pdf/2502.03676"
        ],
        [
         "5",
         "0034e238be40924c815d89333e50afeb8fde846e",
         "Evidence of an Allostatic Response by Intestinal Tissues Following Induction of Joint Inflammation",
         "Disrupted intestinal epithelial barrier function has been proposed to be integral to rheumatoid arthritis (RA) progression and pathogenesis. To further define the molecular pathways in synovial inflammation and the response of the intestinal tissues, we have used a rat model of mono-joint inflammatory arthritis, induced by intra-articular injection of Complete Freund’s adjuvant (CFA). The predominant inflammatory response of a single injection of the adjuvant into the knee joint resulted in rapid and reproducible formation of a fibrotic myeloid-infiltrated synovial pannus. Our aim was to determine how intestinal tissues, including the proximal and distal ileum and distal colon, responded to inflammatory changes in the synovium in a temporally coordinated manner by comparing their transcriptomic landscapes using RNASeq analyses. We confirmed the timeline of joint inflammation by knee joint swelling measurement, increased synovial fluid levels of bikunin (a component of both the acute phase protein pre-alpha-inhibitor and inter-alpha-inhibitor) and demonstrated a self-correcting response of trabecular and cortical bone to the CFA challenge. Intestine-specific responses were monitored by 16S microbiome amplicon sequencing, histopathology for mucus layer integrity, and immune cell immunohistochemistry. We present data that shows the intestinal tissue displays an allostatic response to the acute joint inflammation and was region specific. The ileum primarily responded with increased mucus secretion and silencing of T-cell specific pathways, whereas the colon showed a transient upregulation of macrophages, with a broader suppression of immune related and metabolic pathway related transcripts. Interestingly, many neuropathways were activated early but then suppressed later in both the ileum and colon. There were only insignificant changes in the fecal microbiome composition in ileum or colon post-CFA administration. In summary, our data show for the first time a suppression of intestinal inflammatory and immune responses following the induction of joint inflammation and only minimal and transient changes in the microbiome. The results help clarify the molecular responses of intestinal tissues to inflammatory stresses that accompany the pathogenesis of inflammatory joint diseases.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/17/2024.10.17.618923.full.pdf"
        ],
        [
         "6",
         "0082b6dd5b95334df816971d506e22a0f0da9cc4",
         "Two new amino acid-derived oximes from the mangrove-sediment-derived fungus Lecanicillium kalimantanense SCSIO 41702.",
         "Two new amino acid-derived oximes N-(2-hydroxyimino-4-methyl-pentanoyl)-L-isoleucine (1) and N-(2-hydroxyimino-4-methyl-pentanoyl)-L-leucine (2), along with two known analogues (E)-N-(2-hydroxyimino-3-phenylpropanoyl)-L-phenylalanine (3) and methyl (E)-N- (2-hydroxyimino-3-phenylpropanoyl)-L-phenylalaninate (4), were isolated from the mangrove-sediment-derived fungus Lecanicillium kalimantanense SCSIO 41702. Their structures were determined by spectroscopic analysis. The absolute configurations of 1 and 2 were determined by Marfey's method. Compounds 1 and 2 showed medium inhibitory activity against LPS-induced NO production.",
         "",
         "https://figshare.com/articles/journal_contribution/Two_new_amino_acid-derived_oximes_from_the_mangrove-sediment-derived_fungus_i_Lecanicillium_kalimantanense_i_SCSIO_41702/28323314/1/files/52058141.pdf"
        ],
        [
         "7",
         "008b67207bda1f15d92c5c6d9283c2fae1ace158",
         "Immersive virtual reality enhanced reinforcement induced physical therapy (EVEREST)",
         "None",
         "Immersive Virtual Reality in Post-Stroke Rehabilitation: A Systematic Review;Effectiveness, safety and patients’ perceptions of an immersive virtual reality–based exercise system for poststroke upper limb motor rehabilitation: A proof-of-concept and feasibility randomized controlled trial;New technologies promoting active upper limb rehabilitation after stroke: an overview and network meta-analysis;Effectiveness of Using Virtual Reality–Supported Exercise Therapy for Upper Extremity Motor Rehabilitation in Patients With Stroke: Systematic Review and Meta-analysis of Randomized Controlled Trials;Enhanced Visual Feedback Using Immersive VR Affects Decision Making Regarding Hand Use With a Simulated Impaired Limb;A novel fully immersive virtual reality environment for upper extremity rehabilitation in patients with stroke;Neuroplasticity;Virtual reality games for rehabilitation of upper extremities in stroke patients.;On Shooting Stars;Post-stroke depression: A 2020 updated review.;Virtual reality therapy for upper limb rehabilitation in patients with stroke: a meta-analysis of randomized clinical trials;Principles of Neurorehabilitation After Stroke Based on Motor Learning and Brain Plasticity Mechanisms;Exergaming for stroke rehabilitation: Lessons learned for future implementation strategies;Elements virtual rehabilitation improves motor, cognitive, and functional outcomes in adult stroke: evidence from a randomized controlled pilot study;Comparison of Kinect2Scratch game-based training and therapist-based training for the improvement of upper extremity functions of patients with chronic stroke: a randomized controlled single-blinded trial.;Evaluating the effect and mechanism of upper limb motor function recovery induced by immersive virtual-reality-based rehabilitation for subacute stroke subjects: study protocol for a randomized controlled trial;Virtual Reality in Upper Extremity Rehabilitation of Stroke Patients: A Randomized Controlled Trial.;Project Star Catcher;Rehabilitation via HOMe Based gaming exercise for the Upper-limb post Stroke (RHOMBUS): protocol of an intervention feasibility trial;Virtual Reality Clinical Research: Promises and Challenges;Combining the benefits of tele-rehabilitation and virtual reality-based balance training: a systematic review on feasibility and effectiveness;Towards an Immersive Virtual Reality Game for Smarter Post-Stroke Rehabilitation;Effectiveness of Wii-based rehabilitation in stroke: A randomized controlled study.;Virtual Reality for Upper Limb Rehabilitation in Subacute and Chronic Stroke: A Randomized Controlled Trial.;Game-Based Virtual Reality Canoe Paddling Training to Improve Postural Balance and Upper Extremity Function: A Preliminary Randomized Controlled Study of 30 Patients with Subacute Stroke;What do randomized controlled trials say about virtual rehabilitation in stroke? A systematic literature review and meta-analysis of upper-limb and cognitive outcomes;In-Home Delivery of Constraint-Induced Movement Therapy via Virtual Reality Gaming.;Effects of Kinect-based virtual reality game training on upper extremity motor recovery in chronic stroke;Increasing upper limb training intensity in chronic stroke using embodied virtual reality: a pilot study;Leap Motion-based virtual reality training for improving motor functional recovery of upper limbs and neural reorganization in subacute stroke patients;Stroke recovery and rehabilitation in 2016: a year in review of basic science and clinical science;Effects of virtual reality for stroke individuals based on the International Classification of Functioning and Health: a systematic review;Does the use of Nintendo Wii SportsTM improve arm function? Trial of WiiTM in Stroke: a randomized controlled trial and economics analysis;What Is the Sense of Agency and Why Does it Matter?;Counteracting learned non-use in chronic stroke patients with reinforcement-induced movement therapy;Coaching or gaming? Implications of strategy choice for home based stroke rehabilitation;Post-stroke fatigue: a review on prevalence, correlates, measurement, and management;Constraint-induced movement therapy after stroke;CI Therapy is Beneficial to Patients with Chronic Low-Functioning Hemiparesis after Stroke;Virtual Reality Therapy for Adults Post-Stroke: A Systematic Review and Meta-Analysis Exploring Virtual Environments and Commercial Games in Therapy;What Is the Evidence for Physical Therapy Poststroke? A Systematic Review and Meta-Analysis;Plasticity in the Injured Brain;Connectome: How the Brain’s Wiring Makes Us Who We Are;Mechanism of Kinect-based virtual reality training for motor functional recovery of upper limbs after subacute stroke;Virtual reality for the rehabilitation of the upper limb motor function after stroke: a prospective controlled trial;Depression, activities of daily living and quality of life in patients with stroke;Arm Motor Recovery Using a Virtual Reality Intervention in Chronic Stroke;Maladaptive Plasticity for Motor Recovery after Stroke: Mechanisms and Approaches;Neurorehabilitation of stroke;Virtual reality in neuroscience research and therapy;Virtual Reality for Stroke Rehabilitation;The Stroke Association;Stroke rehabilitation;Potential for new technologies in clinical practice.;Creating an Interface Between the International Classification of Functioning, Disability and Health and Physical Therapist Practice;Apathy following Stroke;The neural basis of constraint-induced movement therapy;Exercises for paretic upper limb after stroke: a combined virtual-reality and telemedicine approach.;Observation of amounts of movement practice provided during stroke rehabilitation.;Repetitive Task Training for Improving Functional Ability After Stroke;Sense of Agency Primes Manual Motor Responses;Training and exercise to drive poststroke recovery;Principles of experience-dependent neural plasticity: implications for rehabilitation after brain damage.;Poststroke shoulder pain: its relationship to motor impairment, activity limitation, and quality of life.;The learned nonuse phenomenon: implications for rehabilitation.;Textbook of Neural Repair and Rehabilitation: Virtual reality in neurorehabilitation;Ambulatory monitoring of arm movement using accelerometry: an objective measure of upper-extremity rehabilitation in persons with chronic stroke.;A Virtual RealityBased Exercise System for Hand Rehabilitation Post-Stroke;Motor Impairment and Recovery in the Upper Limb After Stroke: Behavioral and Neuroanatomical Correlates;A Theory of Fun for Game Design;Mixed reality environments in stroke rehabilitation: Development as rehabilitation tools;Rules of play: game design fundamentals;The Fugl-Meyer Assessment of Motor Recovery after Stroke: A Critical Review of Its Measurement Properties;What are the components of effective stroke unit care?;Stroke patients' and therapists' opinions of constraint-induced movement therapy;Research on Presence in Virtual Reality: A Survey;Neurological deterioration in acute ischemic stroke: potential predictors and associated factors in the European cooperative acute stroke study (ECASS) I.;Impairment and disability: their relation during stroke rehabilitation.;From apoplexy to stroke.;Assessment and psychologic factors in stroke rehabilitation.;Motor learning after recovery from hemiparesis;Reliability of the Fugl-Meyer assessment for testing motor performance in patients following stroke.;RECOVERY TIME OF INDEPENDENT FUNCTION POST-STROKE;Neuropsychological impairments associated with lesions caused by tumor or stroke.;A Motor Learning Model for Stroke Rehabilitation;Reliability of the Fugl-Meyer assessment of sensorimotor recovery following cerebrovascular accident.;Long‐lasting potentiation of synaptic transmission in the dentate area of the anaesthetized rabbit following stimulation of the perforant path;The information capacity of the human motor system in controlling the amplitude of movement.;Virtual Activities of Daily Living for Recovery of Upper Extremity Motor Function;Neurobiology of Sleep and Circadian Rhythms The role of sleep in recovery following ischemic stroke: A review of human and animal data;Motor learning and motor recovery in poststroke individuals: a review of the evidence;Virtual reality training for upper extremity in subacute stroke (VIRTUES): a multicenter RCT;Stroke Rehabilitation A Function Based Approach;The brain's way of healing : stories of remarkable recoveries and discoveries;Febr 25. Stroke deaths in England halved in the �rst decade of the 21st century;Using the international classi�cation of functioning, disability and health in physiotherapy in multidisciplinary vocational rehabilitation: a case study of low back pain;Contribution of the shaping and restraint components of Constraint-Induced Movement therapy to treatment outcome.;Random.org.;Contemporary management of motor control problems;Improving the sensitivity of the Barthel Index for stroke rehabilitation.;The Barthel ADL Index: a reliability study.;Cerebrovascular disease in the community: results of a WHO collaborative study.;Journal of Neuroengineering and Rehabilitation Open Access Reaching in Reality and Virtual Reality: a Comparison of Movement Kinematics in Healthy Subjects and in Adults with Hemiparesis;ET019) who stayed longer in the hospital were evaluated three times. Nevertheless, data from only the �rst and �fth sessions were analyzed (Table 5A)",
         "https://www.researchsquare.com/article/rs-4132920/latest.pdf"
        ],
        [
         "8",
         "008dbf8a0db8fa34cd847d7937b8ddbd2c0b37df",
         "The impact of model eyesight and social reward on automatic imitation in virtual reality.",
         "Motivational theories of imitation state that we imitate because this led to positive social consequences in the past. Because movement imitation typically only leads to these consequences when perceived by the imitated person, it should increase when the interaction partner sees the imitator. Current evidence for this hypothesis is mixed, potentially due to the low ecological validity in previous studies. We conducted two experiments (NExperiment 1 = 94, NExperiment 2 = 110) in which we resolved this limitation by placing participants in a virtual environment with a seeing and a blindfolded virtual agent, where they reacted to auditory cues with a head movement to the left or right, while the agent(s) also made a left or right head movement. We tested the effect of model eyesight (Experiments 1 and 2) and social reward on imitation (Experiment 2). Data were collected in 2023 and 2024. As expected, participants tended to imitate the agents. However, we found only limited evidence for the effect of model eyesight on automatic imitation in Experiment 1 and no evidence for the effect of model eyesight or social reward in Experiment 2. These findings challenge claims made by motivational theories. (PsycInfo Database Record (c) 2025 APA, all rights reserved).",
         "",
         "https://biblio.ugent.be/publication/01JD53YTP2RKAS44E416FSWMXW/file/01JD54148HSWY3634VH3MMMV6V.pdf"
        ],
        [
         "9",
         "00a9a436e7eb5e57de9231fc28b57e8452f45f4c",
         "The microbiota affects stem cell decision making in Hydra",
         "Research on microbial communities colonizing animals has revealed that the microbiota, despite its typical containment to surfaces, influences virtually all organ systems of the host. In absence of a natural microbiota, the host’s development can be disturbed, but how developmental programs are affected by the microbiota is still poorly understood. Removing the microbiota from Hydra, a classic model animal in developmental biology, causes drastic developmental malformations and leads to polyps that temporarily lack the ability to bud. Recolonizing non-budding germfree Hydra with bacteria reverses this budding inhibition. Single-nucleus ATAC-seq detected a unique chromatin landscape associated with the non-budding phenotype. Single-cell RNA-seq and trajectory-based differential expression analysis showed that epithelial stem cell decision making is disturbed in non-budding polyps, whereby key developmental regulators are not expressed. This process is reversible by adding back bacteria. Transcriptionally silencing of one of the genes that failed to be activated in non-budding animals, GAPR1, led to polyps that have a significantly reduced budding capacity. The results show that maintaining a species-specific microbiota may enable the animal host to maintain its developmental program. Significance Statement Animal developmental programs work within the context of coevolved associations with microbes. Here, we provide mechanistic evidence of the involvement of the microbiota in maintaining the pattern formation program of Hydra with the asexual formation of buds in the lower part of the body column. We demonstrate that in the absence of bacteria both the epigenetic and transcriptomic landscape is changed and that key regulatory factors are not expressed, causing changes in stem cell trajectories that result in loss of budding capacity. This study provides a new perspective on the role that microbiota plays during animal development and evolution. One Sentence Summary Microbiota interfere with Hydra’s asexual reproduction via modulating its stem cell differentiation programs.",
         "The Role of Lung and Gut Microbiota in the Pathology of Asthma;Mouse Microbiota Models: Comparing Germ-Free Mice and Antibiotics Treatment as Tools for Modifying Gut Bacteria;Punctuated Emergences of Genetic and Phenotypic Innovations in Eumetazoan, Bilaterian, Euteleostome, and Hominidae Ancestors;EST Analysis of the Cnidarian Acropora millepora Reveals Extensive Gene Loss and Rapid Sequence Divergence in the Model Invertebrates;Elimination by Hydra interstitial and nerve cells by means of colchicine.",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/21/2024.08.20.608462.full.pdf"
        ],
        [
         "10",
         "00f1e24ed9abc1d314d58bad653da4de8e95539e",
         "Loss of multiple micro-RNAs uncovers multi-level restructuring of gene regulation in rodents",
         "Background The regulatory network that coordinates gene expression ultimately determines the phenotype of an organism. Micro-RNAs (miRNAs) are post-transcriptional regulators involved in key biological processes. Lineage-specific losses of multiple miRNA families are rare, and reported cases of multiple miRNA losses coincide with significant changes in gene regulation resulting in body plan modifications. Recently, 15 mammalian miRNA families were found to be missing in the Eumuroidea, the rodent lineage that includes the model organisms mouse and rat. However, the impact of their absence on the gene regulatory networks in this lineage remains unknown. Results The in silico characterization of all 15 miRNAs revealed that their absence is best explained by multiple independent losses. Analyzing their target genes in humans reveals a significant enrichment of GO- terms linked to cellular and developmental processes. Overexpressing two of the co-absent miRNAs, Mir-197 and Mir-769, in human and mouse inducible pluripotent stem cells (iPSCs) resulted in significantly perturbed expression patterns in both species. In silico target site prediction revealed a significant enrichment of direct targets exclusively in the down-regulated genes. Four genes were down-regulated in both mouse and human and maximum parsimony suggests that the corresponding miRNA target sites were already present in the last common ancestor of mammals. The response of these genes to miRNA overexpression in mice, therefore, unveils remnants of the ancient gene regulatory network that have persisted until today. The evolutionary age of these regulatory connections provides initial evidence that the miRNA losses in Eumuroidea must have had consequences for the regulation of gene expression. Notably, we show that the miRNA loss in the Eumuroidea coincides with the largest lineage-specific loss of transcription factors within the mammals. Conclusions The impact of miRNA losses has thus far been investigated on a gene-by-gene basis. Our findings indicate that cooperative effects between miRNAs should be considered when assessing the impact of miRNA loss. We provide evidence that the Eumuroidea have modified their gene regulatory networks on two levels, transcriptionally and post-transcriptionally. It will now be interesting to precisely chart the differences in gene regulation and assess their combined impact on the suitability of mice and rats as model systems for human disease.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/12/09/2024.12.05.627021.full.pdf"
        ],
        [
         "11",
         "013cb58c3526d64e0f8e0d18d71c57250eaa0db2",
         "Tree nursery environments and their effect on early trait variation",
         "Despite the major role of nurseries in raising young plants and trees prior to transplantation, not enough is known about how the nursery climate impacts the growth and development of plants from germination through to maturity. It is important for forestry practitioners to understand the effect that different nursery environments may have on early stage growth as these may exceed differences due to genetic variation and can confound the use of early stage traits for selection. Here, a replicated progeny-provenance experiment of the economically and ecologically important species Scots pine (Pinus sylvestris L.) was established in three environmentally distinct nurseries in Scotland and traits including survival, growth, form and phenology were measured. Temperature variation and photoperiod were the only uncontrolled environmental variables during this period, and their effect on measured traits was found to be significant among nurseries from the first growing season onwards. Trait interactions were not consistent between nurseries, indicating that the effectiveness of using proxy traits to select for desirable characteristics may depend on the environment in which the trees are grown. This study is the first in a series that will examine trait variation in Scots pine from seedlings to mature trees and highlights the importance of carefully considering and accounting for the nursery environment when growing trees for subsequent transplantation.",
         "Meeting tree planting targets on the UK's path to net-zero: A review of lessons learnt from 100 years of land use policies;Phenotypic trait variation in a long-term multisite common garden experiment of Scots pine in Scotland;Explaining Extreme Events of 2020 from a Climate Perspective;Phenotypes of Pinus sylvestris are more coordinated under local harsher conditions across Europe;Selection patterns on early-life phenotypic traits in Pinus sylvestris are associated with precipitation and temperature along a climatic gradient in Europe.;Reaching Natural Growth: The Significance of Light and Temperature Fluctuations in Plant Performance in Indoor Growth Facilities;Guidance for successful tree planting initiatives;Cryptic genetic variation and adaptation to waterlogging in Caledonian Scots pine, Pinus sylvestris L.;Age–age correlations and early selection for growth traits in 40 half-sib families of Larix principis-rupprechtii;Pampered inside, pestered outside? Differences and similarities between plants growing in controlled conditions and in the field.;Has Scots pine (Pinus sylvestris) co‐evolved with Dothistroma septosporum in Scotland? Evidence for spatial heterogeneity in the susceptibility of native provenances;Bud flush phenology and nursery carryover effect of paper birch provenances;Age trend of heritability, genetic correlation, and efficiency of early selection for wood quality traits in Scots pine;Plant growth and mortality under climatic extremes: An overview;What controls tropical forest architecture: testing environmental, structural and floristic drivers;Cultivation of Norway spruce and Scots pine on organic nitrogen improves seedling morphology and field performance;Seasonal patterns of photochemical capacity and spring phenology reveal genetic differentiation amon;A Large and Persistent Carbon Sink in the World’s Forests;Efficiency of the indirect selection and the evaluation of the genotype by environment interaction using Pilodyn for the genetic improvement of wood density in Cryptomeria japonica;A method to construct dose-response curves for a wide range of environmental factors and plant traits by means of a meta-analysis of phenotypic data.;Evaluation of leaf traits for indirect selection of high yielding poplar hybrids;The physiological basis of containerised tree seedling ‘transplant shock’: a review;Age–age and trait–trait correlations for Eucalyptus grandis Hill ex Maiden and their implications for optimal selection age and design of clonal trials;Effects of soil temperature on biomass and carbohydrate allocation in Scots pine (Pinus sylvestris) seedlings at the beginning of the growing season.;Effect of Peat-based Container Media on Establishment of Scots Pine, Norway Spruce and Silver Birch Seedlings after Transplanting in Contrasting Water Conditions;Current and future status of Scots pine (Pinus sylvestris L.) forests in Europe;Heritable variation and evolution under favourable and unfavourable conditions.;The nitrogen economy of mountain birch seedlings: implications for winter survival;Effect of diurnal temperature alternations on plant morphology in some greenhouse crops. A mini review;Seasonal and geographical variation of terpenes, resin acids and total phenolics in nursery grown seedlings of Scots pine (Pinus sylvestris L.);Effects of root temperature on growth and photosynthesis in conifer seedlings during shoot elongation.;Nestling growth in the Great Tit I. Heritability estimates under different environmental conditions;Accelerated short-term genetic testing for loblolly pine families;Effects of Photoperiod on Growth of Trees;Autumn versus spring planting: the initiation of root growth and subsequent field performance of Scots pine and Norway spruce seedlings;Photoperiod- and temperature-mediated control of phenology in trees - a molecular perspective.;GENETIC SURVEY OF PINUS RADIATA. 5: BETWEEN-TRAIT AND AGE-AGE CORRELATIONS FOR GROWTH RATE, MORPHOLOGY, AND DISEASE RESISTANCE;Plant Growth and Climate Change.;Low temperature, but not photoperiod, controls growth cessation and dormancy induction and release in apple and pear.;Selection of parents for the Scots pine breeding population in Britain;Using Native Stock for Planting Native Trees and Shrubs;On the sampling variance of interclass correlations and genetic 786 correlations;Genotype by environment interaction and genetic 795 correlation of greenhouse and field performance in Pinus contorta ssp. latifolia . 796 Silvae;Forest nursery practice.;Effect of fertilization and watering of Scots pine seedlings on the feeding preference of the pine weevil (Hylobius abietis L.).;Tree planting: not a simple solution;Applied Tree Improvement;Growth response of woody plants to photoperiodic stimuli;Table 1. Traits measured in individual seedlings, grouped into ‘Growth’, ‘Form’, ‘Phenology’ and ‘Survival’. Units for;age of the trees increased and as the number of years between comparisons decreased;through to maturity. The trees in this study have now been transplanted to three field;organisms grow should lead to higher or lower heritability of quantitative traits;Introduction to Quantitative Genetics . 4th ed. Essex, UK: Longman, 1996.;variances when data from 2007 are not included. For traits measured in multiple years, only the most recent year is shown;transplanted to their field locations, as measurements can be made over many more years",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/20/2024.08.20.608769.full.pdf"
        ],
        [
         "12",
         "0142fc80cb50bb72904e314218eaebac59ddfa68",
         "Cross-site harmonization of diffusion MRI data without matched training subjects",
         "Purpose Diffusion MRI (dMRI) data typically suffer of significant cross-site variability, which prevents naively performing pooled analyses. To attenuate cross-site variability, harmonization methods such as the rotational invariant spherical harmonics (RISH) have been introduced to harmonize the dMRI data at the signal level. A common requirement of the RISH method, is the availability of healthy individuals who are matched at the group level, which may not always be readily available, particularly retrospectively. In this work, we propose a framework to harmonize dMRI without matched training groups. Methods Our framework learns harmonization features while controlling for potential covariates using a voxel-based generalized linear model (RISH-GLM). RISH-GLM allows to simultaneously harmonize data from any number of sites while also accounting for covariates of interest, thus not requiring matched training subjects. Additionally, RISH-GLM can harmonize data from multiple sites in a single step, whereas RISH is performed for each site independently. Results We considered data of training subjects from retrospective cohorts acquired with 3 different scanners and performed 3 harmonization experiments of increasing complexity. First, we demonstrate that RISH-GLM is equivalent to conventional RISH when trained with data of matched training subjects. Secondly, we demonstrate that RISH-GLM can effectively learn harmonization with two groups of highly unmatched subjects. Thirdly, we evaluate the ability of RISH-GLM to simultaneously harmonize data from 3 different sites. Discussion RISH-GLM can learn cross-site harmonization both from matched and unmatched groups of training subjects, and can effectively be used to harmonize data of multiple sites in one single step.",
         "Lifespan reference curves for harmonizing multi-site regional brain white matter metrics from diffusion MRI;Neuroimaging standards for research into small vessel disease—advances since 2013;Abnormal white matter changes in Alzheimer's disease based on diffusion tensor imaging: A systematic review;Improved sensitivity and precision in multicentre diffusion MRI network analysis using thresholding and harmonization;Multimodal tract-based MRI metrics outperform whole brain markers in determining cognitive impact of small vessel disease-related brain injury;Cross-site harmonization of multi-shell diffusion MRI measures based on rotational invariant spherical harmonics (RISH);Diffusion MRI harmonization enables joint-analysis of multicentre data of patients with cerebral small vessel disease;Towards multicentre diffusion MRI studies in cerebral small vessel disease;MarkVCID cerebral small vessel consortium: I. Enrollment, clinical, fluid protocols;Cross-scanner and cross-protocol multi-shell diffusion MRI data harmonization: Algorithms and results;A prospective harmonized multicenter DTI study of cerebral white matter degeneration in ALS;Generalized Richardson-Lucy (GRL) for analyzing multi-shell diffusion MRI data;Cross-scanner and cross-protocol diffusion MRI data harmonisation: A benchmark database and evaluation of algorithms;Scanner invariant representations for diffusion MRI harmonization;Retrospective harmonization of multi-site diffusion MRI data acquired with different acquisition parameters;Inter-Scanner Harmonization of High Angular Resolution DW-MRI using Null Space Deep Learning;Presymptomatic white matter integrity loss in familial frontotemporal dementia in the GENFI cohort: A cross‐sectional diffusion tensor imaging study;On modeling;Current Clinical Applications of Diffusion-Tensor Imaging in Neurological Disorders;Harmonization of cortical thickness measurements across scanners and sites;Harmonization of multi-site diffusion tensor imaging data;The importance of correcting for signal drift in diffusion MRI;Denoising of diffusion MRI using random matrix theory;Inter-site and inter-scanner diffusion MRI data harmonization;The effect of Gibbs ringing artifacts on measures derived from diffusion MRI;REKINDLE: Robust extraction of kurtosis INDices with linear estimation;Lifespan maturation and degeneration of human brain white matter;Faculty Opinions recommendation of Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data.;Diffusion MRI at 25: Exploring brain tissue structure and function;A reproducible evaluation of ANTs similarity metric performance in brain image registration;Microstructural maturation of the human brain from childhood to adulthood;Empirical Bayes Methods;Harmonization;Toward a quantitative assessment of diffusion anisotropy;Matter;multimodal brain 3 T MRI harmonisation approaches;presymptomatic familial frontotemporal dementia;ExploreDTI: a graphical toolbox for processing, analyzing, and visualizing diffusion MR data;Adjusting batch effects in microarray expression data using empirical Bayes methods.;Microstructural and physiological features of tissues elucidated by quantitative-diffusion-tensor MRI.;MR diffusion tensor spectroscopy and imaging.;Spin diffusion measurements : spin echoes in the presence of a time-dependent field gradient;Submitted to Magnetic Resonance in Medicine Corresponding author: Alberto De Luca (a.deluca-2@umcutrecht.nl) Word count: 3924 words;Discussion: RISH-GLM can learn cross-site harmonization both from matched and unmatched groups training subjects, and can effectively be used to harmonize data of multiple sites in;This study is funded by the Bluefield Project to cure FTD;determined with RISH-GLM on data from all three sites in Experiment 3",
         "https://www.biorxiv.org/content/biorxiv/early/2024/05/03/2024.05.01.591994.full.pdf"
        ],
        [
         "13",
         "0155e40ceb6ddc83a93fb431cc86ebe135079cab",
         "Dynamics of quantum battery capacity under Markovian channels",
         "None",
         "",
         "http://arxiv.org/pdf/2408.03797"
        ],
        [
         "14",
         "01891feafb883a81f4c532208e34dcb5806d8ca5",
         "Larvicidal and cytotoxic activities of exopolysaccharides produced by thermophilic bacteria.",
         "None",
         "",
         "https://www.researchsquare.com/article/rs-5304452/latest.pdf"
        ],
        [
         "15",
         "01ae6d81c72cb5b14ae95519f29011609a777351",
         "Vegetation influences wolf fine-scale habitat selection and movement rate in a logged coastal rainforest",
         "None",
         "",
         "https://link.springer.com/content/pdf/10.1007/s00442-025-05677-5.pdf"
        ],
        [
         "16",
         "01c65b584d35362abc5f2425273ec2eefc579594",
         "Redox regulation of autophagy in Arabidopsis: differential effects of reactive oxygen species",
         "Autophagy plays a key role in the responses to different stress condition in plants. Reactive oxygen species (ROS) are common modulators of stress responses, having both toxic and signaling functions. In this context, the relationship between ROS and autophagy regulation remains unclear, and in some aspects, contradictory. In this study, we employed pharmacological and genetic approaches to investigate the effects of different ROS on the cytoplastic redox state and autophagic flux in Arabidopsis thaliana. Ours results demonstrated that oxidative treatments with H2O2 and MV, which drastically increased the oxidized state of the cytoplasm, reduced the autophagic flux. Conversely, singlet oxygen, which did not have significant effects on the cytoplasmic redox state, increased the autophagic flux. Additionally, our findings indicated that after H2O2 and high light treatments and during the recovery period, the cytoplasm returned to its reduced state, while autophagy was markedly induced. In summary, our study unveils the differential effects of ROS on the autophagic flux, establishing a correlation with the redox state of the cytoplasm. Moreover, it emphasizes the dynamic nature of autophagy in response to oxidative stress and the subsequent recovery period. HIGHLIGHTS This research shows the differential effects of reactive oxygen species on autophagic modulation, highlighting their impact on the cytoplasmic redox state. The relationship between ROS and autophagy regulation remains unclear, and in some aspect’s contradictory. Here, we present a comprehensive investigation characterizing the effects of different ROS, such as hydrogen peroxide and singlet oxygen, on the modulation of autophagy in Arabidopsis. In brief, our findings reveal differential impacts on cytoplasmic redox states and autophagic flux, providing insight into the dynamic nature of autophagy, especially in stress and post-stress conditions. To the best of our knowledge, our work is the first to evaluate autophagic flux both during and after oxidative stress in plants. Our results indicate that this differentiation is crucial when analyzing the effects of oxidative stress on autophagy.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2023/08/19/2023.08.11.552961.full.pdf"
        ],
        [
         "17",
         "01e2c0cb2a8adced75528452c58dc12ae704f755",
         "K-Gate Lock: Multi-Key Logic Locking Using Input Encoding Against Oracle-Guided Attacks",
         "Logic locking has emerged to prevent piracy and overproduction of integrated circuits ever since the split of the design house and manufacturing foundry was established. While there has been a lot of research using a single global key to lock the circuit, even the most sophisticated single-key locking methods have been shown to be vulnerable to powerful SAT-based oracle-guided attacks that can extract the correct key with the help of an activated chip bought off the market and the locked netlist leaked from the untrusted foundry. To address this challenge, we propose, implement, and evaluate a novel logic locking method called K-Gate Lock that encodes input patterns using multiple keys that are applied to one set of key inputs at different operational times. Our comprehensive experimental results confirm that using multiple keys will make the circuit secure against oracle-guided attacks and increase attacker efforts to an exponentially time-consuming brute force search. K-Gate Lock has reasonable power and performance overheads, making it a practical solution for real-world hardware intellectual property protection.",
         "STATION: State Encoding-Based Attack-Resilient Sequential Obfuscation;Machine Learning-Based Security Evaluation and Overhead Analysis of Logic Locking;CoLA: Convolutional Neural Network Model for Secure Low Overhead Logic Locking Assignment;CAS-Lock: A Security-Corruptibility Trade-off Resilient Logic Locking Scheme;Anti-SAT: Mitigating SAT Attack on Logic Locking;CycSAT-Unresolvable Cyclic Logic Encryption Using Unreachable States;Unveiling the ISCAS-85 Benchmarks: A Case Study in Reverse Engineering",
         "http://arxiv.org/pdf/2501.02118"
        ],
        [
         "18",
         "01ee76d50524c8842f3ce68ef656f630485cbec1",
         "Federated learning-driven IoT system for automated freshness monitoring in resource-constrained vending carts",
         "None",
         "",
         "https://journalofbigdata.springeropen.com/counter/pdf/10.1186/s40537-025-01063-3"
        ],
        [
         "19",
         "02005c9e65bcf62c329533a372895f68b739c991",
         "Modelling the interaction between silver(I) ion and proteins with the 12-6 Lennard-Jones potential: a bottom-up parameterization approach",
         "Silver(I) ions and organometallic complexes thereof are well established antimicrobial agents. They have been employed in medical applications for centuries. It is also known that some bacteria can resist to silver(I) treatments through an efflux mechanism. However, the exact mechanism of action remains unclear. All-atom force-field simulations can provide valuable structural and thermodynamic insights on the molecular processes of the underlying mechanism. Lennard-Jones parameters of silver(I) have been available for quite some time; their applicability to describe properly the binding properties (affinity, binding distance) between silver(I) and peptide-based binding motifs is, however, still an open question. Here, we demonstrate that the standard 12-6 Lennard-Jones parameters (previously developed to describe the hydration free energy with the TIP3P water model) significantly underestimate the interaction strength between silver(I) and both methionine and histidine. These are two key amino-acid residues in silver(I)-binding motifs of proteins involved in the efflux process. Using free-energy calculations, we calibrated non-bonded fix (NBFIX) parameters for the CHARMM36m force field to reproduce the experimental binding constant between amino acid sidechain fragments and silver(I) ions. We then successfully validated the new parameters on a set of small silver-binding peptides with experimentally known binding constants. In addition, we could monitor how silver(I) ions increase the α-helical content of the LP1 oligopeptide, in agreement with previously reported Circular Dichroism (CD) experiments. Future improvements are outlined. The implementation of these new parameters is straightforward in all simulation packages that can use the CHARMM36m force field. It sets the stage for the modeling community to study more complex silver(I)-binding processes such as the interaction with silver(I)-binding-transporter proteins.",
         "Thermodynamics of Metal-Acetate Interactions.;The battle for silver binding: How the interplay between the SilE, SilF, and SilB proteins contributes to the silver efflux pump mechanism;Accurate Metal-Imidazole Interactions.;Constant pH molecular dynamics simulations: Current status and recent applications.;Re-sensitization of mcr carrying multidrug resistant bacteria to colistin by silver;Scalable Constant pH Molecular Dynamics in GROMACS;Computational strategies to model the interaction and the reactivity of biologically-relevant transition metal complexes;Alpha-helical folding of SilE models upon Ag(His)(Met) motif formation.;Configuration-Sampling-Based Surrogate Models for Rapid Parameterization of Non-Bonded Interactions.;Model peptide studies of Ag+ binding sites from the silver resistance protein SilE.;CHARMM36m: an improved force field for folded and intrinsically disordered proteins;SilE is an intrinsically disordered periplasmic “molecular sponge” involved in bacterial silver resistance;CHARMM TIP3P Water Model Suppresses Peptide Folding by Solvating the Unfolded State.;Enhanced sampling techniques in molecular dynamics simulations of biological systems.;Systematic Parameterization of Monovalent Ions Employing the Nonbonded Model.;Recent Advances in Polarizable Force Fields for Macromolecules: Microsecond Simulations of Proteins Using the Classical Drude Oscillator Model;Force Field Independent Metal Parameters Using a Nonbonded Dummy Model;Nanobio silver: its interactions with peptides and bacteria, and its uses in medicine.;Force field for monovalent, divalent, and trivalent cations developed under the solvent boundary potential.;Syntheses, structures, and antimicrobial activities of remarkably light-stable and water-soluble silver complexes with amino acid derivatives, silver(I) N-acetylmethioninates.;Constant pH Molecular Dynamics in Explicit Solvent with λ-Dynamics;Current status of the AMOEBA polarizable force field.;Antibacterial Activity and Mechanism of Action of the Silver Ion in Staphylococcus aureus and Escherichia coli;Atomistic Simulation Studies of Polymers and Water;The Simple Yet Elusive Crystal Structure of Silver Acetate and the Role of the Ag−Ag Bond in the Formation of Silver Nanoparticles during the Thermally Induced Reduction of Silver Carboxylates;Calculation of absolute protein-ligand binding free energy from computer simulations.;Bacterial silver resistance: molecular biology and uses and misuses of silver compounds.;A mechanistic study of the antibacterial effect of silver ions on Escherichia coli and Staphylococcus aureus.;Synthesis and characterization of water-soluble silver(I) complexes with L-histidine (H2his) and (S)-(-)-2-pyrrolidone-5-carboxylic acid (H2pyrrld) showing a wide spectrum of effective antibacterial and antifungal activities. Crystal structures of chiral helical polymers [Ag(Hhis)]n and ([Ag(Hpyrrld;Critical evaluation of stability constants of metal-imidazole and metal-histamine systems (Technical Report);THE weighted histogram analysis method for free‐energy calculations on biomolecules. I. The method;STEREOREGULAR COORDINATION POLYMERS FORMED ON BINDING OF PEPTIDE-BASED POLYDENTATE LIGANDS TO SILVER(I) AND COPPER(I) - X-RAY STRUCTURE OF ([AG(N-[N-((5-METHYL-2-THIENYL)METHYLIDENE)-L-METHIONYL]HISTAMINE)]+[O3SCF3]-.MEOH)INFINITY AND A SOLUTION STRUCTURE STUDY;Comparison of simple potential functions for simulating liquid water;Nonphysical sampling distributions in Monte Carlo free-energy estimation: Umbrella sampling;NIST SRD 46. Critically Selected Stability Constants of Metal Complexes: Version 8.0 for Windows;unpublished results;Theoretical Studies of Hydrogen Bonding;Comparison of the ligating properties of disulphides and thioethers: dimethyl disulphide, dimethyl sulphide, and related ligands;Sur Le Mélange Des Gaz;Ueber die Anwendung des Satzes vom Virial in der kinetischen Theorie der Gase;Using the procedure given in the section 2.2, the PMF and binding constant are calculated;We performed MD based Umbrella Sampling (US) simulations in explicit water to sample along the reaction coordinate of binding, i.e. the distance between silver(I) ion and the coordinated atom;The Potential of Mean Force (PMF) was determined along the binding reaction coordinate through WHAM;From the PMF, the binding constant log 10 ( K bind , calc ) was calculated (and, consequently, the binding free energy ∆ G bind,calc );Toward Predicting Silver Ion Binding in Proteins, under Revision;If the minimum of the PMF matches the target distance",
         "https://www.biorxiv.org/content/biorxiv/early/2024/12/04/2024.11.28.625676.full.pdf"
        ],
        [
         "20",
         "022103bcb2f73b22a217aaa1bb92d4f8df4a75c2",
         "A causal model of human growth and its estimation using temporally-sparse data",
         "Existing models of human growth provide little insight into underlying mechanisms responsible for inter-individual and inter-population variation in children’s growth trajectories. Building on general theories linking growth to metabolic rates, we develop a causal parametric model of height and weight growth incorporating a representation of human body allometry and a process-partitioned representation of ontogeny. This model permits separation of metabolic causes of growth variation, potentially influenced by nutrition and disease, from allometric factors, potentially under stronger genetic control. We estimate model parameters using a Bayesian multilevel statistical design applied to temporally-dense height and weight measurements of U.S. children, and temporally-sparse measurements of Indigenous Amazonian children. This facilitates a comparison of the contributions of metabolism and allometry to observed cross-cultural variation in the growth trajectories of the two populations, and permits simulation of the effects of healthcare interventions on growth. This theoretical model provides a new framework for exploring the causes of growth variation in our species, while potentially guiding the development of appropriate, and desired, healthcare interventions in societies confronting growth-related health challenges, such as malnutrition and stunting.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2022/10/12/2022.10.10.511559.full.pdf"
        ],
        [
         "21",
         "0223a10fa0f31b13cd2ae9f43bad026285be5b85",
         "Focality of sound source placement by higher (9th) order ambisonics and perceptual effects of spectral reproduction errors",
         "Higher-order ambisonic rendering is an increasingly common soundscape reproduction technique that, in theory, enables presentation of virtual sounds at nearly any location in 3D space, relatively unconstrained by the veridical locations of loudspeakers. We evaluated whether 3D sound reproduction through a 9th-order ambisonic loudspeaker array was indeed sufficiently accurate to probe the limits of human spatial perception. We first estimated minimum audible angles for human listeners for a variety of reference points on the horizontal plane. We demonstrated that the system can reproduce sounds with a spatial resolution that is equal or superior to the limits of human acuity, at least on the horizontal plane at the front. Importantly, the resolution of ambisonic reproduction appeared equivalent for regions of the system with high and low loudspeaker density. We also estimated localization cues at the same locations and showed that although localization cues for low-frequency components were well preserved, they were somewhat distorted for components above 4000 Hz. Finally, we provide evidence that these high-frequency distortions can serve elevation cues by human listeners. In summary, we showed that a 9th order ambisonic system is able to render highly focal sound sources, and that high-frequency reproduction distortions may introduce unwanted localization cues.",
         "Spectral weighting functions for lateralization and localization of complex sound.;Spatial variation in signal and sensory precision both constrain auditory acuity at high frequencies;Investigation on localisation accuracy for first and higher order ambisonics reproduced sound sources;Head Circumference in Canadian Male Adults: Development of a Normalized Chart;Rapid Formation of Robust Auditory Memories: Insights from Noise;Listener weighting of cues for lateral angle: the duplex theory of sound localization revisited.;Reproduction of a plane-wave sound field using an array of loudspeakers;Uniform spreading of amplitude panned virtual sources;Spectral cues used in the localization of sound sources on the median plane.;The sense of hearing , (Third edition. | Abingdon, Oxon, New York, NY: Routledge, 2018.;Fundamentals of binaural technology;MINIMUM AUDIBLE ANGLES FOR HORIZONTAL, VERTICAL, AND OBLIQUE ORIENTATIONS : LATERAL AND DORSAL PLANES;higher-frequency ITDs are not relevant for perception and so these high-frequency;block keep them focused on the task. The data were collected within one session of 1 hour duration;at the spectral power peaks at ~8 kHz in FIG 6.A).;reported the burst with less power (i.e., the ambisonically rendered burst) as lower in elevation;average low-frequency ILD was -0.55 dB when the source was at the center (reference point C);even higher order systems may not perfectly reproduce sound locations: errors in 66;to frontal HRTF magnitudes previously reported 1 . At all locations, within each channel;its values are around 0 dB confirms the equivalence of VBAP and SC for locations;0 everywhere, confirming the equivalence of VBAP and SC presentation methods for veridical LSP locations;higher, we could still ask whether there is a difference in the degree to which wideband 708",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/09/2024.08.07.606870.full.pdf"
        ],
        [
         "22",
         "0230eb8423f14aa38144acb3a3d1fb105857fdd2",
         "Resistance exercise training in older men reduces ATF4-activated and senescence-associated mRNAs in skeletal muscle.",
         "None",
         "",
         "https://link.springer.com/content/pdf/10.1007/s11357-025-01564-2.pdf"
        ],
        [
         "23",
         "023287f776585ce5ae13285b8949f1ba4cde871a",
         "Leveraging Sentiment–Topic Analysis for Understanding the Psychological Role of Hype in Emerging Technologies—A Case Study of Electric Vehicles",
         "This study presents a novel approach to examining the psychological impact of emerging technologies through the development of a Hype Cycle Model (HCM), utilizing sentiment analysis and topic modeling. Focusing on electric vehicles, we investigate how public sentiment—captured via social media comments—reflects the psychological effects of technology adoption and hype. Our model integrates both qualitative and quantitative analyses, utilizing sentiment scoring and topic modeling to explore thematic psychological trends. An analysis of approximately 43,000 social media comments on electric vehicles demonstrated that the integration of expert knowledge with public sentiment provides a comprehensive understanding of technology hype dynamics. The results revealed that sentiment analysis enables real-time tracking of emotional responses to emerging technologies, while Correlated Topic Modeling (CTM) offers contextual insights into the positioning of technologies within the HCM. These findings demonstrate that understanding public sentiment towards emerging technologies can provide valuable insights for both organizations and policymakers in technology forecasting and adoption planning. Our approach transforms the traditional black box implementation by Gartner Inc. into a transparent framework that illuminates the psychological underpinnings of technology hype, revealing how collective excitement, expectations, and emotional responses shape the trajectory of emerging technology adoption.",
         "",
         "https://www.mdpi.com/2076-328X/15/2/137/pdf?version=1737888395"
        ],
        [
         "24",
         "02365f56e2e4b54038afd9a17e398ee90c297288",
         "Do macroinvertebrate abundance and community structure depend on the quality of ponds located in peri-urban areas?",
         "Contamination is one of the major threats to freshwater biodiversity. Compared to other aquatic ecosystems, peri-urban ponds are unique because they are embedded in human-dominated areas. However, it is poorly understood how different land uses such as urban or agricultural contribute multiple pollutants to ponds and thus affect pond biodiversity. In this work, 12 ponds located in a peri-urban area (Ile-de-France region, France) were monitored for 2 consecutive years in spring and fall. We surveyed macroinvertebrates and measured the physicochemical parameters and contaminants of different classes (trace elements, pharmaceuticals, pesticides and polycyclic aromatic hydrocarbons) in both water and sediment. The objective was twofold: (1) to explore local and regional macroinvertebrate spatiotemporal diversity and (2) to understand the effects of contaminants on community structure. We observed 236 macroinvertebrate morphotaxa, none of which were rare or sensitive to pollutants. Morphotaxa richness showed small differences between ponds but no difference between there was no effect of field campaign. There was no effect of ponds and field campaign on morphotaxa diversity and equitability. We did not observe a relationship between land use around the pond (agricultural, urban, or semi-natural) and diversity indices with the exception of the proportion of agricultural land in the vicinity of the pond on equitability. Regional beta diversity (between ponds) showed that differences in morphotaxa composition reflected species replacement more than differences in species richness; these were primarily due to the high abundance of pollutant-tolerant species in some of the ponds. The effects of environmental parameters on community structure were studied using partial redundancy analysis based on the presence-absence of morphotaxa, showing that community assemblages are shaped by sediment levels of pharmaceuticals, water conductivity and ammonium concentration. In conclusion, ponds in peri-urban areas are exposed to various human activities, with our results suggesting that this exposure leads to chronic and diverse contaminations that affect morphotaxa communities.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2023/10/23/2023.10.20.563281.full.pdf"
        ],
        [
         "25",
         "02405247da19656e1027e2575504ba54a65dc7fa",
         "lcUMAPtSNE: Use of non-linear dimensionality reduction techniques with genotype likelihoods",
         "Understanding population structure is essential for conservation genetics, as it provides insights into population connectivity and supports the development of targeted strategies to preserve genetic diversity and adaptability. T-distributed stochastic neighbor embedding (t-SNE) and uniform manifold approximation and projection (UMAP) have proven effective for revealing population genetic structures in human and model organisms using hard-called genotypes, but their application in wild species using genotype likelihoods from low coverage sequencing (as a cost-saving measure) remains underexplored. Here, we present a Jupyter Notebook-based workflow that facilitates the use of UMAP and t-SNE on genotype likelihood-derived principal components. This workflow is demonstrated using medium to low-coverage whole-genome sequencing data from scimitar-horned oryx, which has been reintroduced into the wild and faces multiple conservation challenges. Detailed guidance on hyperparameter tuning and practical implementation is also provided, enhancing the application of these methods in wildlife genetics to potentially support biodiversity conservation.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/04/01/2024.04.01.587545.full.pdf"
        ],
        [
         "26",
         "0248ab98531b074b916d3f3a0e63bd84c51cadce",
         "Exosomal miR-224-3p promotes lymphangiogenesis and lymph node metastasis by targeting GSK3B in gastric cancer",
         "Background Patients with gastric cancer (GC) are prone to lymph node metastasis (LNM), which is an important factor for recurrence and poor prognosis of GC. Nowadays, more and more studies have confirmed that exosomes can participate in tumor lymphangiogenesis. An in-depth exploration of the pathological mechanism in the process of LNM in GC may provide effective targets and improve the diagnosis and treatment effect. Materials and Methods We used sequencing analysis of collected serum to screen out exo-miRNA related to LNM in GC. ELISA, qRT-PCR, Western Blot, RNA pull-down assay, Transwell assay, animal experiments, and other experiments were used to verify the results. Results In this study, we screened out miR-224-3p related to GC progression and LNM in a vascular endothelial growth Factor C (VEGFC)-independent manner. We found that exo-miR-224-3p derived from GC cells could enter human lymphatic endothelial cells (HLECs) and promote the tube formation and migration of HLECs. In addition, it was revealed that miR-224-3p could bind to the 3′UTR region of GSK3B mRNA. Then, we proved that inhibiting the expression of GSK3B could suppress the phosphorylation of β-catenin and promote the transcription of PROX1, thus leading to tumor lymphangiogenesis. Furthermore, it was also found that hnRNPA1 mediated the sorting of miR-224-3p into exosomes, and the high expression of PKM2 promoted the secretion of exo-miR-224-3p. Conclusions Our discovery of the exo-miR-224-3p/GSK3B/β-catenin/PROX1 axis may provide a new direction for the clinical treatment of GC.",
         "SLC6A14 Depletion Contributes to Amino Acid Starvation to Suppress EMT-Induced Metastasis in Gastric Cancer by Perturbing the PI3K/AKT/mTORC1 Pathway;Low Expression of miR-491-3p Is Correlated with Lymph Node Metastasis in Gastric Cancer;β-Catenin signaling in hepatocellular carcinoma;The role of Exosomal miRNAs in cancer;Lymph Node Metastases from Visceral Peritoneal Colorectal Metastases are Associated with Systemic Recurrence;The Chinese Society of Clinical Oncology (CSCO): Clinical guidelines for the diagnosis and treatment of gastric cancer, 2021;Role of lymphatic endothelial cells in the tumor microenvironment—a narrative review of recent advances;Current status of lymph node dissection in gastric cancer;The impact of thyroid tumor features on lymph node metastasis in papillary thyroid carcinoma patients in head and neck department at KAMC: A retrospective cross-sectional study;Lymph node metastasis-derived gastric cancer cells educate bone marrow-derived mesenchymal stem cells via YAP signaling activation by exosomal Wnt5a;Current treatment and recent progress in gastric cancer;Global Cancer Statistics 2020: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries;RNA packaging into extracellular vesicles: An orchestra of RNA‐binding proteins?;The Roles of Non-Coding RNAs in Tumor-Associated Lymphangiogenesis;Ectosomal PKM2 Promotes HCC by Inducing Macrophage Differentiation and Remodeling the Tumor Microenvironment.;Exosome-mediated transfer of miR-93-5p from cancer-associated fibroblasts confer radioresistance in colorectal cancer cells by downregulating FOXA1 and upregulating TGFB3;PCDHGA9 represses epithelial-mesenchymal transition and metastatic potential in gastric cancer cells by reducing β-catenin transcriptional activity;CAF secreted miR-522 suppresses ferroptosis and promotes acquired chemo-resistance in gastric cancer;The biology, function, and biomedical applications of exosomes;Exosomal long noncoding RNA LNMAT2 promotes lymphatic metastasis in bladder cancer.;Extracellular miRNAs: From Biomarkers to Mediators of Physiology and Disease.;A novel role of HIF-1α/PROX-1/LYVE-1 axis on tissue regeneration after renal ischaemia/reperfusion in mice;Glycogen synthase kinase 3-β inhibition induces lymphangiogenesis through β-catenin-dependent and mTOR-independent pathways;Deciphering role of FGFR signalling pathway in pancreatic cancer;Ki-67 protein as a tumour proliferation marker.;Exosomes in gastric cancer: roles, mechanisms, and applications;TRAF6 inhibits colorectal cancer metastasis through regulating selective autophagic CTNNB1/β-catenin degradation and is targeted for GSK3B/GSK3β-mediated phosphorylation and degradation;CCL27/CCL28-CCR10 Chemokine Signaling Mediates Migration of Lymphatic Endothelial Cells.;Exosomal miR-196a derived from cancer-associated fibroblasts confers cisplatin resistance in head and neck cancer through targeting CDKN1B and ING5;Stabilization of miRNAs in esophageal cancer contributes to radioresistance and limits efficacy of therapy.;Circulating exosomal noncoding RNAs as prognostic biomarkers in human hepatocellular carcinoma;Cancer-derived exosomal miR-25-3p promotes pre-metastatic niche formation by inducing vascular permeability and angiogenesis;PROX1 Is Associated with Cancer Progression and Prognosis in Gastric Cancer;Effect of exosomal miRNA on cancer biology and clinical applications;Complementary Wnt Sources Regulate Lymphatic Vascular Development via PROX1-Dependent Wnt/β-Catenin Signaling;Cervical squamous cell carcinoma-secreted exosomal miR-221-3p promotes lymphangiogenesis and lymphatic metastasis by targeting VASH1;The chromosome 11q13.3 amplification associated lymph node metastasis is driven by miR-548k through modulating tumor microenvironment;Tumor-derived exosomal miR-1247-3p induces cancer-associated fibroblast activation to foster lung metastasis of liver cancer;Lymph node staging systems in patients with gastric cancer treated with D2 resection plus adjuvant chemotherapy;Exosomal transfer of tumor-associated macrophage-derived miR-21 confers cisplatin resistance in gastric cancer cells;Pyruvate kinase type M2 promotes tumour cell exosome release via phosphorylating synaptosome-associated protein 23;Positive regulation of β-catenin–PROX1 signaling axis by DBC1 in colon cancer progression;Cell-based approach for 3D reconstruction of lymphatic capillaries in vitro reveals distinct functions of HGF and VEGF-C in lymphangiogenesis.;Advanced gastric cancer: What we know and what we still have to learn.;Glycogen synthase kinase-3 (GSK3): regulation, actions, and diseases.;VEGFR signaling during lymphatic vascular development: From progenitor cells to functional vessels;Exosome and Exosomal MicroRNA: Trafficking, Sorting, and Function;Transcriptional regulation of podoplanin expression by Prox1 in lymphatic endothelial cells.;Gastric Cancer: Descriptive Epidemiology, Risk Factors, Screening, and Prevention;Sumoylated hnRNPA2B1 controls the sorting of miRNAs into exosomes through binding to specific motifs;GSK3: a multifaceted kinase in Wnt signaling.;Lymphangiogenesis: Molecular Mechanisms and Future Promise;A GSK-3/TSC2/mTOR pathway regulates glucose uptake and GLUT1 glucose transporter expression.;Clinicopathological evaluation of biological behavior of submucosal invasive gastric carcinomas: relationship among lymph node metastasis, mucin phenotype and proliferative activity.;TSC2 Integrates Wnt and Energy Signals via a Coordinated Phosphorylation by AMPK and GSK3 to Regulate Cell Growth;Vascular endothelial growth factor-a promotes peritumoral lymphangiogenesis and lymphatic metastasis.;PDGF-BB induces intratumoral lymphangiogenesis and promotes lymphatic metastasis.;An essential role for Prox1 in the induction of the lymphatic endothelial cell phenotype;complementary and alternative medicine: eCAM;A MicroRNA signature identi ﬁ es pancreatic ductal adenocarcinoma patients at risk for lymph node metastases;An unexpected role for EGF in lymphangiogenesis-mediated melanoma metastasis to sentinel lymph nodes.;Lymphangiogenesis and cancer metastasis.;Accurate Lymph Node Staging is of Greater Prognostic Importance Than Subclassification of the T2 Category for Gastric Adenocarcinoma;[Gastric cancer].",
         "https://www.researchsquare.com/article/rs-2168273/latest.pdf"
        ],
        [
         "27",
         "0296022d83ce83107b2e66ba8bf405e567b47269",
         "A Unifying View of OTFS and Its Many Variants",
         "High mobility environment leads to severe Doppler effects and poses serious challenges to the conventional physical layer based on the widely popular orthogonal frequency division multiplexing (OFDM). The recent emergence of orthogonal time frequency space (OTFS) modulation, along with its many related variants, presents a promising solution to overcome such channel Doppler effects. This paper aims to clearly establish the relationships among the various manifestations of OTFS. Among these related modulations, we identify their connections, common features, and distinctions. Building on existing works, this work provides a general overview of various OTFS-related detection schemes and performance comparisons. We first provide an overview of OFDM and filter bank multi-carrier (FBMC) by demonstrating OTFS as a precoded FBMC through the introduction of inverse symplectic finite Fourier transform (ISFFT). We explore the relationship between OTFS and related modulation schemes with similar characteristics. We provide an effective channel model for high-mobility channels and offer a unified detection representation. We provide numerical comparisons of power spectrum density (PSD) and bit error rate (BER) to underscore the benefit of these modulation schemes in high-mobility scenarios. We also evaluate various detection schemes, revealing insights into their efficacies. We discuss opportunities and challenges for OTFS in high mobility, setting the stage for future research and development in this field.",
         "",
         "http://arxiv.org/pdf/2502.09118"
        ],
        [
         "28",
         "02a6b974397afa97ee32a32b7a655f2aed513313",
         "Hidden GPCR structural transitions addressed by multiple walker supervised molecular dynamics (mwSuMD)",
         "The structural basis for the pharmacology of G protein-coupled receptors (GPCRs), the most abundant membrane proteins and the target of about 35% of approved drugs, is still a matter of intense study. What makes GPCRs challenging to study is the inherent flexibility and the metastable nature of interaction with extra-and intracellular partners that drive their effects. Here, we present a molecular dynamics (MD) adaptive sampling algorithm, namely multiple walker supervised molecular dynamics (mwSuMD), to address complex structural transitions involving GPCRs without energy input. We first report the binding and unbinding of the vasopressin peptide from its receptor V2. Successively, we present the complete transition of the glucagon-like peptide-1 receptor (GLP-1R) from inactive to active, agonist and Gs-bound state, and the GDP release from Gs. To our knowledge, this is the first time the whole sequence of events leading from an inactive GPCR to the GDP release is simulated without any energy bias. We demonstrate that mwSuMD can address complex binding processes intrinsically linked to protein dynamics out of reach of classic MD.",
         "Mechanistic insights into G-protein coupling with an agonist-bound G-protein-coupled receptor.;The dynamics of agonist-β_2-adrenergic receptor activation induced by binding of GDP-bound Gs protein;Conformational Dynamics of the Activated GLP-1 Receptor-Gs Complex Revealed by Cross-Linking Mass Spectrometry and Integrative Structure Modeling;The full activation mechanism of the adenosine A1 receptor revealed by GaMD and Su-GaMD simulations;Kinetic model of GPCR-G protein interactions reveals allokairic modulation of signaling;Structural basis of peptidomimetic agonism revealed by small-molecule GLP-1R agonists Boc5 and WB4-24;Heterobivalent Ligand for the Adenosine A2A-Dopamine D2 Receptor Heteromer.;Structural perspective of class B1 GPCR signaling.;Exploring Ligand Binding to Calcitonin Gene-Related Peptide Receptors;Highly accurate protein structure prediction with AlphaFold;Molecular insights into ago-allosteric modulation of the human glucagon-like peptide-1 receptor;Structural and Functional Diversity among Agonist-Bound States of the GLP-1 Receptor;Molecular basis of ligand recognition and activation of human V2 vasopressin receptor;Evolving cryo-EM structural approaches for GPCR drug discovery;Conformational switch that induces GDP release from Gi.;Structural basis for GLP-1 receptor activation by LY3502970, an orally active nonpeptide agonist;Deciphering the Agonist Binding Mechanism to the Adenosine A1 Receptor;A Multisite Model of Allosterism for the Adenosine A1 Receptor;Addressing free fatty acid receptor 1 (FFAR1) activation using supervised molecular dynamics;Supervised molecular dynamics for exploring the druggability of the SARS-CoV-2 spike protein;Structure and dynamics of the active Gs-coupled human secretin receptor;Differential GLP-1R binding and activation by peptide and non-peptide agonists;Structural insights into differences in G protein activation by family A and family B GPCRs;Structural insights into the activation of GLP-1R by a small molecule agonist;A combined activation mechanism for the glucagon receptor;Selective activation of Gαob by an adenosine A1 receptor agonist elicits analgesia without cardiorespiratory depression;Structure and Dynamics of Adrenomedullin Receptors AM1 and AM2 Reveal Key Mechanisms in the Control of Receptor Phenotype by Receptor Activity-Modifying Proteins.;Full-length human GLP-1 receptor structure without orthosteric ligands;Using metadynamics to explore complex free-energy landscapes;A Supervised Molecular Dynamics Approach to Unbiased Ligand-Protein Unbinding;Haloperidol bound D2 dopamine receptor structure inspired the discovery of subtype selective ligands;Energy Landscapes Reveal Agonist Control of G Protein-Coupled Receptor Activation via Microswitches;Activation of the GLP-1 receptor by a non-peptidic agonist;Reconstruction of apo A2A receptor activation pathways reveal ligand-competent intermediates and state-dependent cholesterol hotspots;Gγ and Gα Identity Dictate a G-Protein Heterotrimer Plasma Membrane Targeting;End-Point Binding Free Energy Calculation with MM/PBSA and MM/GBSA: Strategies and Applications in Drug Design.;Machine Learning Analysis of τRAMD Trajectories to Decipher Molecular Determinants of Drug-Target Residence Times;Structural Insights into the Process of GPCR-G Protein Complex Formation;Assembly of a GPCR-G Protein Complex;Rules of Engagement: GPCRs and G Proteins.;Molecular Dynamics Simulation for All;Mapping the Interface of a GPCR Dimer: A Structural Model of the A2A Adenosine and D2 Dopamine Receptor Heteromer;Biased Ligands of G Protein-Coupled Receptors (GPCRs): Structure-Functional Selectivity Relationships (SFSRs) and Therapeutic Potential.;Molecular Signature for Receptor Engagement in the Metabolic Peptide Hormone Amylin.;Simulation of spontaneous G protein activation reveals a new intermediate driving GDP unbinding;G Protein-Coupled Receptors as Targets for Approved Drugs: How Many Targets and How Many Drugs?;Simulations meet machine learning in structural biology.;Mechanism of the G-protein mimetic nanobody binding to a muscarinic G-protein-coupled receptor;Markov State Models: From an Art to a Science.;Dynamic Docking: A Paradigm Shift in Computational Drug Discovery;Serial millisecond crystallography for routine room-temperature structure determination at synchrotrons;Weighted Ensemble Simulation: Review of Methodology, Applications, and Software.;Selectivity determinants of GPCR–G-protein binding;Single-molecule analysis of ligand efficacy in β2AR-G protein activation;Exploring Protein-Peptide Recognition Pathways Using a Supervised Molecular Dynamics Approach.;Estimation of kinetic and thermodynamic ligand-binding parameters using computational strategies.;Kinetic Aspects of the Interaction between Ligand and G Protein-Coupled Receptor: The Case of the Adenosine Receptors.;OpenMM 7: Rapid development of high performance algorithms for molecular dynamics;CHARMM36m: an improved force field for folded and intrinsically disordered proteins;Regulation, Signaling, and Physiological Functions of G-Proteins.;Cholesterol-dependent Conformational Plasticity in GPCR Dimers;A Conserved Hydrophobic Core in Gαi1 Regulates G Protein Activation and Release from Activated Receptor *;Deciphering the Complexity of Ligand-Protein Recognition Pathways Using Supervised Molecular Dynamics (SuMD) Simulations;The pathway of ligand entry from the membrane bilayer to a lipid G protein-coupled receptor;A Hydrogen-Bonded Polar Network in the Core of the Glucagon-Like Peptide-1 Receptor Is a Fulcrum for Biased Agonism: Lessons from Class B Crystal Structures;The Role of Target Binding Kinetics in Drug Discovery;Gaussian Accelerated Molecular Dynamics: Unconstrained Enhanced Sampling and Free Energy Calculation;Universal allosteric mechanism for Gα activation by GPCRs;Structural basis for nucleotide exchange in heterotrimeric G proteins;Dopamine heteroreceptor complexes as therapeutic targets in Parkinson’s disease;Entry from the Lipid Bilayer: A Possible Pathway for Inhibition of a Peptide G Protein-Coupled Receptor by a Lipophilic Small Molecule.;Computational approaches for modeling GPCR dimerization.;MDTraj: a modern, open library for the analysis of molecular dynamics trajectories;Using caffeine and other adenosine receptor antagonists and agonists as therapeutic tools against neurodegenerative diseases: a review.;The Evolution of the GPCR Signaling System in Eukaryotes: Modularity, Conservation, and the Transition to Metazoan Multicellularity;WExplore: hierarchical exploration of high-dimensional spaces using the weighted ensemble algorithm.;Supervised Molecular Dynamics (SuMD) as a Helpful Tool To Depict GPCR-Ligand Recognition Pathway in a Nanosecond Time Scale;Rapid parameterization of small molecules using the force field toolkit;GPCRDB: an information system for G protein-coupled receptors;PLUMED 2: New feathers for an old bird;CHARMM36 all‐atom additive protein force field: Validation based on comparison to NMR data;Adrenaline-activated structure of the β2-adrenoceptor stabilized by an engineered nanobody;Polar transmembrane interactions drive formation of ligand-specific and signal pathway-biased family B G protein-coupled receptor conformations;Membrane Packing Problems: A short Review on computational Membrane Modeling Methods and Tools;Extension of the CHARMM general force field to sulfonyl‐containing compounds and its utility in biomolecular simulations;Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges;Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing;Protein data bank;MMPBSA.py: An Efficient Program for End-State Free Energy Calculations.;Molecular dynamics simulations and drug discovery;MDAnalysis: A toolkit for the analysis of molecular dynamics simulations;Crystal Structure of the β2Adrenergic Receptor-Gs protein complex;PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions.;Lateral diffusion of membrane proteins.;Adenosine A2A receptor-antagonist/dopamine D2 receptor-agonist bivalent ligands as pharmacological tools to detect A2A-D2 receptor heteromers.;ACEMD: Accelerating Biomolecular Dynamics in the Microsecond Time Scale.;CHARMM‐GUI: A web‐based graphical user interface for CHARMM;Membrane interactions of G proteins and other related proteins.;Well-tempered metadynamics: a smoothly converging and tunable free-energy method.;Vasopressin and disorders of water balance: the physiology and pathophysiology of vasopressin;OPM: Orientations of Proteins in Membranes database;Scalable molecular dynamics with NAMD;The GRAFS classification system of G-protein coupled receptors in comparative perspective.;UCSF Chimera—A visualization system for exploratory research and analysis;How a G Protein Binds a Membrane*;PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations;Accelerated molecular dynamics: a promising and efficient simulation method for biomolecules.;Adenosine A2A-Dopamine D2 Receptor-Receptor Heteromerization;Crystal Structure of Rhodopsin: A G‐Protein‐Coupled Receptor;Escaping free-energy minima;A fast SHAKE algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations;Vasopressin Receptors;Restrained electrostatic potential atomic partial charges for condensed-phase simulations of carbohydrates.;SHAKE, rattle, and roll: Efficient constraint algorithms for linked rigid bodies;Replica-exchange molecular dynamics method for protein folding;Adiabatic bias molecular dynamics: A method to navigate the conformational space of complex molecular systems;Adenosine A2A receptor antagonists as new agents for the treatment of Parkinson's disease.;VMD: visual molecular dynamics.;A smooth particle mesh Ewald method;GROMACS: A message-passing parallel molecular dynamics implementation;Langevin dynamics of peptides: The frictional dependence of isomerization rates of N‐acetylalanyl‐N′‐methylamide;Molecular dynamics with coupling to an external bath;Comparison of simple potential functions for simulating liquid water;The Adaptive Path Collective Variable: A Versatile Biasing Approach to Compute the Average Transition Path and Free Energy of Molecular Transitions.;Heteroreceptor Complexes and their Allosteric Receptor–Receptor Interactions as a Novel Biological Principle for Integration of Communication in the CNS: Targets for Drug Development;Lipid-dependent GPCR dimerization.;High-resolution crystal structure of an engineered human beta2-adrenergic G protein-coupled receptor.;Modeller: generation and refinement of homology-based protein structure models.;Kr (cid:0);SHAKE, rattle, and roll: Efficient constraint algorithms for linked rigid bodies;[19] Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors;Lipid Modification of G Proteins: alpha Subunits are Palmitoylated;G protein-coupled receptors | G protein-coupled receptors | IUPHAR/BPS Guide to PHARMACOLOGY",
         "https://www.biorxiv.org/content/biorxiv/early/2023/01/04/2022.10.26.513870.full.pdf"
        ],
        [
         "29",
         "02abac2e5c388dd967a6b99def97cbca4d64d315",
         "Activity-dependent synapse elimination requires caspase-3 activation",
         "During brain development, synapses are initially formed in excess and are later eliminated in an activity-dependent manner. Weak synapses are preferentially removed, but the mechanism linking neuronal activity to synapse removal is unclear. Here we show that, in the developing mouse visual pathway, inhibiting synaptic transmission induces postsynaptic activation of caspase-3. Caspase-3 deficiency results in defects in synapse elimination driven by both spontaneous and experience-dependent neural activity. Notably, caspase-3 deficiency blocks activity-dependent synapse elimination, as evidenced by reduced engulfment of inactive synapses by microglia. Furthermore, in a mouse model of Alzheimer’s disease, caspase-3 deficiency protects against synapse loss induced by amyloid-β deposition. Our results reveal caspase-3 activation as a key step in activity-dependent synapse elimination during development and synapse loss in neurodegeneration.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/08/05/2024.08.02.606316.full.pdf"
        ],
        [
         "30",
         "02cabd03b27e19ee87f97b1cd9c7d9d05649ea85",
         "Programmable DNA shell scaffolds for directional membrane budding",
         "In the pursuit of replicating biological processes at the nanoscale, controlling cellular membrane dynamics has emerged as a key area of interest. Here, we report a system mimicking virus assembly to control directional membrane budding. We employ three-dimensional DNA origami techniques to construct cholesterol-modified triangles that self-assemble into polyhedral shells on lipid vesicles, resulting in gradual curvature induction, bud formation and spontaneous neck scission. Strategic positioning of cholesterols on the triangle surface provides control over the directionality of bud growth and yields daughter vesicles with DNA endo- or exoskeletons reminiscent of clathrin-coated vesicles. This process occurs with rapid kinetics and across various lipid compositions. When combined into a two-step process, nested bivesicular objects with DNA shells encapsulated between lipid vesicles could be produced. Our work replicates key aspects of natural endocytic and exocytic pathways, opening new avenues for exploring membrane mechanics and applications in targeted drug delivery and synthetic biology.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/01/22/2024.01.18.576181.full.pdf"
        ],
        [
         "31",
         "02f332a476ad77bda3627220c4f1f6fc6e3e0b03",
         "Extracellular rRNA profiling reveals the sinking and cell lysis dynamics of marine microeukaryotes",
         "Marine plankton communities consist of numerous species, and their composition and physiological states are closely linked to ecosystem functions. Understanding biogeochemical cycles requires measuring taxon-specific mortality due to viral lysis or sloppy feeding, as the dissolved organic matter released contributes to rapid nutrient recycling and long-term carbon sequestration following microbial transformation. This study introduces a quantitative and comprehensive analysis of microeukaryotes in the dissolved constituents of seawater by using Mortality by Ribosomal Sequencing (MoRS) method. Our experimental pipeline successfully recovered 83% of cell-free rRNA. The ratio of cell-free rRNA to cell-associated rRNA was more than 10-fold higher in the mesopelagic layer than in the upper epipelagic layer, suggesting the mesopelagic zone as a hotspot for eukaryotic cell lysis. Many protist lineages, including phytoplankton such as haptophytes, are less susceptible to cell lysis in the epipelagic layer yet are actively lysed in the mesopelagic zone. Notably, over 86% of the significantly lysed species in the mesopelagic layer showed a habitat preference for the epipelagic layer. These findings indicate that sinking from the surface and lysis in the mesopelagic are prevalent dynamics for various eukaryotes.",
         "Functions of extracellular polymeric substances in partitioning suspended and sinking particles in the upper oceans of two open ocean systems;The microbial carbon pump and climate change.;Taxon-specific contributions of microeukaryotes to biological carbon pump in the Oyashio region;Selective cell lysis pressure on rare and abundant prokaryotic taxa across a shelf-to-slope continuum in the Northern South China Sea;Characterization of sinking and suspended microeukaryotic communities in spring Oyashio waters;Vertical imbalance in organic carbon budgets is indicative of a missing vertical transfer during a phytoplankton bloom near South Georgia (COMICS);Genomic adaptation of giant viruses in polar oceans;Viral infection switches the balance between bacterial and eukaryotic recyclers of organic matter during coccolithophore blooms;Viral infection in the ocean—A journey across scales;Haptophyte communities along the Kuroshio current reveal their geographical sources and ecological traits;Mortality by ribosomal sequencing (MoRS) provides a window into taxon-specific cell lysis;Cryptic and abundant marine viruses at the evolutionary origins of Earth’s RNA virome;The Ocean's labile DOC supply chain;Patterns of eukaryotic diversity from the surface to the deep-ocean sediment;Environmental vulnerability of the global ocean epipelagic plankton community interactome;A direct capture method for purification and detection of viral nucleic acid enables epidemiological surveillance of SARS-CoV-2;A thermal trade-off between viral production and degradation drives virus-phytoplankton population dynamics.;Impaired viral infection and reduced mortality of diatoms in iron-limited oceanic regions;Environmental (e)RNA advances the reliability of eDNA by predicting its age;A method for characterizing dissolved DNA and its application to the North Pacific Subtropical Gyre;A dataset on trophic modes of aquatic protists;Biogeography of marine giant viruses reveals their interplay with eukaryotes and ecological functions;Resolving marine dissolved organic phosphorus (DOP) composition in a coastal estuary;Visualizing active viral infection reveals diverse cell fates in synchronized algal bloom demise;Metabolic and biogeochemical consequences of viral infection in aquatic ecosystems;Molecular properties are a primary control on the microbial utilization of dissolved organic matter in the ocean;Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2;Eukaryotic virus composition can predict the efficiency of carbon export in the global ocean;Spatial Variations in Community Structure of Haptophytes Across the Kuroshio Front in the Tokara Strait;Aggregate Formation During the Viral Lysis of a Marine Diatom;Carbon Budgets of Mesozooplankton Copepod Communities in the Eastern Atlantic Ocean—Regional and Vertical Patterns Between 24°N and 21°S;Coccolithovirus facilitation of carbon export in the North Atlantic;fastp: an ultra-fast all-in-one FASTQ preprocessor;Optimizing taxonomic classification of marker-gene amplicon sequences with QIIME 2’s q2-feature-classifier plugin;Unveiling the role and life strategies of viruses from the surface to the dark ocean;DADA2: High resolution sample inference from Illumina amplicon data;Plankton networks driving carbon export in the oligotrophic ocean;Eukaryotic plankton diversity in the sunlit ocean;Rethinking the marine carbon cycle: Factoring in the multifarious lifestyles of microbes;Rising to the challenge: accelerated pace of discovery transforms marine virology;Effects of sea surface warming on marine plankton.;Reconciliation of the carbon budget in the ocean’s twilight zone;Dissolved organic matter (DOM) release by phytoplankton in the contemporary and future ocean;Microbial control of the dark end of the biological pump.;Arctic Ocean Microbial Community Structure before and after the 2007 Record Sea Ice Minimum;The relative importance of sloppy feeding, excretion, and fecal pellet leaching in the release of dissolved carbon and nitrogen by Acartia tonsa copepods;edgeR: a Bioconductor package for differential expression analysis of digital gene expression data;Estimates of virus- vs. grazing induced mortality of picophytoplankton in the North Sea during summer;Marine viruses — major players in the global ecosystem;Production of dissolved organic carbon by sloppy feeding in the copepods Acartia tonsa, Centropages typicus, and Temora longicornis;Determination of dissolved nucleic acids in seawater by the fluorescence dye, ethidium bromide;The measurement and distribution of dissolved nucleic acids in aquatic environments;Dynamics of Dissolved Organic Phosphorus;Corresponding author.;mesopelagic layer showed a habitat preference for the epipelagic layer",
         "https://www.biorxiv.org/content/biorxiv/early/2024/06/01/2024.05.31.596594.full.pdf"
        ],
        [
         "32",
         "03003bda6891a727a43bbe0ff847b40748a8187e",
         "An integrative taxonomic treatment of the Mycetophilidae (Diptera: Bibionomorpha) from Singapore reveals 115 new species on 730km2",
         "“Open ended” or “dark taxa” are species-rich clades that are so abundant and diverse that conventional taxonomic methods tend to struggle with the onslaught of specimens and species. New approaches based on presorting specimens to putative species with low-cost DNA barcodes may make tackling these taxa manageable. However, this will still require limiting the geographic scope of taxonomic revisions, given that most countries and biogeographic regions will have too many specimens and species for comprehensive coverage. We demonstrate the power of this approach by carrying out a revision of the Mycetophilidae fungus gnats (Diptera) of Singapore. The material revised here was obtained from 496 samples collected with 71 Malaise traps placed at 107 sites in different habitats in Singapore: mangroves, swamp forests, freshwater swamps, primary rainforests, and different types of secondary forests (old, maturing, young, urban). Based on molecular and morphological data for 1,454 specimens, we delimit 120 species of 23 genera. Of these, only five were species previously described. The remaining 115 species are new to science and described here. We name, however, only 98 of these species since 14 species are currently only known from females and we cannot prepare a fully satisfying morphological diagnosis (Manota spp. A–G and Neoempheria spp. A–G), and three species lack molecular data (Epicypta sp. A, Epicypta sp. B, and Neoempheria sp. H). To assess congruence between species delimited with DNA barcodes (3% clusters) and morphology, we determined a match ratio and found it to be overall high (95%) with even higher match ratios (99%) observed for MOTUs clustered at 5% with Objective Clustering and MOTUs obtained with ABGD set to (P=0.060). Overall, the ratio of undescribed to described is an astonishing 20:1. Only revising the Singapore fauna increases the number of described species of Oriental Mycetophilidae by about 25%, highlighting the size of the taxonomic impediment for fungus gnats. Most of the Singapore Mycetophilidae diversity belongs to three genera—Neoempheria Osten-Sacken (31 species), Epicypta Winnertz (29 species) and Manota Williston (14 species), but we also describe a new genus— Integricypta, gen. n., which is the putative sistergroup of Aspidionia Colless—belonging to the Mycetophilinae Mycetophilini based on three species. The species sequenced, illustrated, and named are: Leptomorphus rafflesi, sp. n.; Monoclona simhapura, sp. n.; Azana demeijeri, sp. n.; Azana leekongchiani, sp. n. (Sciophilinae); Tetragoneura crawfurdi, sp. n.; Tetragoneura chola, sp. n.; Tetragoneura dayuan, sp. n.; Tetragoneura farquhari, sp. n.; Ectrepesthoneura johor, sp. n. (Tetragoneurinae); Mohelia zubirsaidi, sp. n.; Allactoneura tumasik, sp. n.; Allactoneura limbosengi, sp. n.; Manota banzu, sp. n.; Manota tantocksengi, sp. n.; Manota bukittimah, sp. n.; Manota chiamassie, sp. n.; Manota danmaxi, sp. n.; Manota mahuan, sp. n.; Manota temenggong, sp. n.; Clastobasis sritribuana, sp. n.; Clastobasis bugis, sp. n.; Clastobasis oranglaut, sp. n. (Leiinae); Parempheriella mait, sp. n.; Parempheriella longyamen, sp. n.; Parempheriella peranakan, sp. n.; Mycomya sachmatich, sp. n.; Neoempheria merlio, sp. n.; Neoempheria sabana, sp. n.; Neoempheria sangabo, sp. n.; Neoempheria shicheng, sp. n.; Neoempheria ujong, sp. n.; Neoempheria subaraji, sp. n.; Neoempheria kokoiyeeae, sp. n.; Neoempheria mandai, sp. n.; Neoempheria malacca, sp. n.; Neoempheria sinkapho, sp. n.; Neoempheria singapura, sp. n.; Neoempheria xinjiapo, sp. n.; Neoempheria puluochung, sp. n.; Neoempheria merdeka, sp. n.; Neoempheria neesoon, sp. n.; Neoempheria pulau, sp. n.; Neoempheria cinkappur, sp. n.; Neoempheria temasek, sp. n.; Neoempheria polunini, sp. n.; Neoempheria fajar, sp. n.; Neoempheria riatanae, sp. n. (Mycomyinae); Brachycampta glorialimae, sp. n.; Brachycampta murphyi, sp. n.; Brachycampta limtzepengi, sp. n.; Rymosia teopohlengi, sp. n.; Exechia tanswiehiani, sp. n.; Exechia alinewongae, sp. n.; Mycetophila chngseoktinae, sp. n.; Mycetophila georgettechenae, sp. n.; Mycetophila aishae, sp. n.; Platyprosthiogyne phanwaithongae, sp. n.; Platyprosthiogyne gohsookhimae, sp. n.; Platyprosthiogyne rahimahae, sp. n.; Platyprosthiogyne lynetteseahae, sp. n.; Platyprosthiogyne neilaae, sp. n.; Platyprosthiogyne snehalethaae, sp. n.; Platurocypta adeleneweeae, sp. n.; Platurocypta tanhoweliangi, sp. n.; Epicypta constancesingamae, sp. n.; Epicypta jennylauae, sp. n.; Epicypta limchiumeiae, sp. n.; Epicypta janetyeeae, sp. n.; Epicypta kohkhenglianae, sp. n.; Epicypta daintoni, sp. n.; Epicypta holltumi, sp.n.; Epicypta ridleyi, sp. n.; Epicypta chezaharaae, sp. n.; Epicypta tanjiakkimi, sp. n.; Epicypta gehminae, sp. n.; Epicypta jackieyingae, sp. n.; Epicypta khatijunae, sp. n.; Epicypta purchoni, sp. n.; Epicypta foomaoshengi, sp. n.; Epicypta ganengsengi, sp. n.; Epicypta nanyangu, sp. n.; Epicypta nus, sp. n.; Epicypta peterngi, sp. n.; Epicypta maggielimae, sp. n.; Epicypta yupeigaoae, sp. n.; Epicypta annwee, sp. n.; Epicypta wallacei, sp. n.; Epicypta lamtoongjini, sp. n.; Epicypta catherinelimae, sp. n.; Epicypta grootaerti, sp. n.; Epicypta joaquimae, sp. n.; Aspidionia cheesweeleeae, sp. n.; Aspidionia janetjesudasonae, sp. n.; Aspidionia fatimahae, sp. n.; Integricypta fergusondavie, sp. n.; Integricypta teosoonkimae, sp. n.; Integricypta shirinae, sp. n.; Integricypta hoyuenhoeae, sp. n. (Mycetophilinae). The previously described species are Metanepsia malaysiana Kallweit, Eumanota racola Søli, Parempheriella defectiva (Edwards), Neoempheria dizonalis (Edwards) (all known from Sumatra and/or the Malaysian peninsula), and Chalastonepsia hokkaidensis Kallweit a species spread in east Asia. The mycomyine genus Vecella Wu & Yang is here proposed as a new synonym to Parempheriella, with P. guadunana (Wu & Yang), n. comb. corresponding to an additional Palearctic species of Parempheriella. Barcodes for a second set of 1,493 Singapore mycetophilid specimens suggest the presence of an additional 18 MOTUs. We thus estimate that approximately 85% of all the Singapore species that routinely enter Malaise traps are identified or described here. The revision concludes with a discussion of the biogeography and generic composition of the mycetophilid fauna at the southern end of the Malay Peninsula. Zusammenfassung “Open-ended” oder “Dark Taxa” sind artenreiche Klade, die so abundant und arteinreich sind, dass die herkömmlichen taxonomischen Methoden angesichts der großen Anzahl von Exemplaren und Arten nicht gut funktionieren. Neue Ansätze, die auf das Vorsortierung von Exemplaren mit DNA Barcodes bis zum Artniveau basieren, erlauben es jetzt aber solche Taxa zu revidieren. Allerdings kann auch mit DNA Barcodes, nur die Fauna eines vergleichsweise kleinen Gebietes bearbeiten werden, weil für die meisten Länder und biogeografischen Regionen zu viele Exemplare und Arten abgedeckt werden müssten. Wir demonstrieren hier wie eine solche Revision durchgeführt werden kann. Wir revidieren hier die Pilzmücken (Diptera: Mycetophilidae) von Singapur, die mit Malaisefallen gefangen werden können. Das hier überarbeitete Material stammt aus 496 Malaise-Fallenproben, die mit 71 Malaise-Fallen an 107 Sammelstellen in verschiedenen Habitaten gefangen wurden: Mangroven, Sumpfwälder, Süßwassersümpfe, Primärregenwälder und Sekundärwälder. Basierend auf molekularen und morphologischen Daten für mehr als 1454 Tiere grenzen wir 120 Arten mit molekularen und morphologischen Daten ab, wobei nur 5 dieser Arten bereits beschrieben sind. Die verbleibenden 115 werden hier beschrieben. Allerdings benennen wir nur 98 Arten, da für zwei Arten molekulare Daten fehlen und für 14 weitere Arten derzeit nur Weibchen bekannt sind. Daher können wir derzeit keine zufriedenstellende morphologische Diagnose erstellen. Darüber hinaus fehlen molekulare Daten für drei Arten (Epicypta sp. A, Epicypta sp. B und Neoempheria sp. H). Was die Artgrenzen betrifft, so stimmen die molekularen und morphologischen Daten in den meisten Fällen überein (match ratio: 95% für 3% MOTUs). Eine noch höhere „match ratio“ von 99% wird für 5% und ABGD MOTUs (P=0,060) beobachtet. Insgesamt ist das Verhältnis zwischen unbeschrieben und beschrieben erstaunlich hoch (20:1), und die Überarbeitung der singapurer Fauna erhöht die Anzahl der beschriebenen Arten in der Orientalischen Region um über 25%. Dies unterstreicht den Ausmaß des „taxonomic impediments“ für Pilzmücken. Die meisten der Mycetophilidenarten in Singapur gehören zu drei von 22 Gattungen - Neoempheria Osten-Sacken (31 Arten), Epicypta Winnertz (29 Arten) und Manota Williston (14 Arten), aber wir beschreiben hier auch eine neue Gattung, Integricypta, gen. n. basierend auf drei Arten. Die Gattung gehört zu den Mycetophilinae Mycetophilini und ist die mutmaßliche Schwestergruppe von Aspidionia Colless. Die sequenzierten, illustrierten und benannten Arten sind: Leptomorphus rafflesi, sp. n.; Monoclona simhapura, sp. n.; Azana demeijeri, sp. n.; Azana leekongchiani, sp. n. (Sciophilinae); Tetragoneura crawfurdi, sp. n.; Tetragoneura chola, sp. n.; Tetragoneura dayuan, sp. n.; Tetragoneura farquhari, sp. n.; Ectrepesthoneura johor, sp. n. (Tetragoneurinae); Mohelia zubirsaidi, sp. n.; Allactoneura tumasik, sp. n.; Allactoneura limbosengi, sp. n.; Manota banzu, sp. n.; Manota tantocksengi, sp. n.; Manota bukittimah, sp. n.; Manota chiamassie, sp. n.; Manota danmaxi, sp. n.; Manota mahuan, sp. n.; Manota temenggong, sp. n.; Clastobasis sritribuana, sp. n.; Clastobasis bugis, sp. n.; Clastobasis oranglaut, sp. n. (Leiinae); Parempheriella mait, sp. n.; Parempheriella longyamen, sp. n.; Parempheriella peranakan, sp. n.; Mycomya sachmatich, sp. n.; Neoempheria merlio, sp. n.; Neoempheria sabana, sp. n.; Neoempheria sangabo, sp. n.; Neoempheria shicheng, sp. n.; Neoempheria ujong, sp. n.; Neoempheria subaraji, sp. n.; Neoempheria kokoiyeeae, sp. n.; Neoempheria mandai, sp",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2023/09/15/2023.09.02.555672.full.pdf"
        ],
        [
         "33",
         "031550851c1139950a93f08ec62552c8ebbca5e3",
         "The Geometry and Dimensionality of Brain-wide Activity",
         "Understanding neural activity organization is vital for deciphering brain function. By recording whole-brain calcium activity in larval zebrafish during hunting and spontaneous behaviors, we find that the shape of the neural activity space, described by the neural covariance spectrum, is scale-invariant: a smaller, randomly sampled cell assembly resembles the entire brain. This phenomenon can be explained by Euclidean Random Matrix theory, where neurons are reorganized from anatomical to functional positions based on their correlations. Three factors contribute to the observed scale invariance: slow neural correlation decay, higher functional space dimension, and neural activity heterogeneity. In addition to matching data from zebrafish and mice, our theory and analysis demonstrate how the geometry of neural activity space evolves with population sizes and sampling methods, thus revealing an organizing principle of brain-wide activity.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2023/08/08/2023.02.23.529673.full.pdf"
        ],
        [
         "34",
         "0316bec2c50dbd379f8c217b638bd35b213b66fc",
         "Refining the resolution of the yeast genotype-phenotype map using single-cell RNA-sequencing",
         "Genotype-phenotype mapping (GPM), or the association of trait variation to genetic variation, has been a long-lasting problem in biology. The existing approaches to this problem allowed researchers to partially understand within– and between-species variation as well as the emergence or evolution of phenotypes. However, traditional GPM methods typically ignore the transcriptome or have low statistical power due to challenges related to dataset scale. Thus, it is not clear to what extent selection modulates transcriptomes and whether cis-or trans-regulatory elements are more important. To overcome these challenges, we leveraged the cost efficiency and scalability of single-cell RNA sequencing (scRNA-seq) by collecting data from 18,233 yeast cells from 4,489 F2 segregants derived from an F1 cross between the laboratory strain BY4741 and the vineyard strain RM11-1a. More precisely, we performed eQTL mapping with the scRNA-seq data to identify single-cell eQTL (sc-eQTL) and transcriptome variation patterns associated with fitness variation inferred from the segregant bulk fitness assay. Due to the larger scale of our dataset and its multidimensionality, we could recapitulate results from decades of work in GPM from yeast bulk assays while revealing new associations between phenotypic and transcriptomic variations at a broad scale. We evaluated the strength of the association between phenotype variation and expression variation, revealed new hotspots of gene expression regulation associated to trait variation, revealed new gene function with high expression heritability and highlighted the larger aggregate effect of trans-regulation compared to cis-regulation Altogether these results suggest that integrating large-scale scRNA-seq data into GPM improves our understanding of trait variation in the context of transcriptomic regulation.",
         "Trans-eQTL mapping in gene sets identifies network effects of genetic variants;Molecular quantitative trait loci;DiSCERN - Deep Single Cell Expression ReconstructioN for improved cell clustering and cell subtype and state detection;Pleiotropic effects of trans-regulatory mutations on fitness and gene expression;An introduction to spatial transcriptomics for biomedical research;PANTHER: Making genome‐scale phylogenetics accessible to all;Barcoded bulk QTL mapping reveals highly polygenic and epistatic architecture of complex traits in yeast;Powerful eQTL mapping through low coverage RNA sequencing;Aggregative trans-eQTL analysis detects trait-specific target gene sets in whole blood;Phenotypic and molecular evolution across 10,000 generations in laboratory budding yeast populations;Cis and trans effects differentially contribute to the evolution of promoters and enhancers;Efficient and effective control of confounding in eQTL mapping studies through joint differential expression and Mendelian randomization analyses;A new protocol for single-cell RNA-seq reveals stochastic gene expression during lag phase in budding yeast;Size-Dependent Increase in RNA Polymerase II Initiation Rates Mediates Gene Expression Scaling with Cell Size;RNA sequencing: the teenage years;Genome-wide association and transcriptome studies identify target genes and risk loci for breast cancer;Rare variants contribute disproportionately to quantitative trait variation in yeast;Opportunities and challenges for transcriptome-wide association studies;Protocol Update for large-scale genome and gene function analysis with the PANTHER classification system (v.14.0);Trans Effects on Gene Expression Can Drive Omnigenic Inheritance;Single-cell RNA sequencing technologies and bioinformatics pipelines;Genetics of trans-regulatory variation in gene expression;The Dynamics of Molecular Evolution Over 60,000 Generations;Transcriptome Profiling in Human Diseases: New Advances and Perspectives;Genome-Wide Association Studies In Plant Pathosystems: Toward an Ecological Genomics Approach;Rare cell variability and drug-induced reprogramming as a mode of cancer drug resistance;Massively parallel digital transcriptional profiling of single cells;Evolution: Tinkering within Gene Regulatory Landscapes;Assessing Glucose Uptake through the Yeast Hexose Transporter 1 (Hxt1);A High-Definition View of Functional Genetic Variation from Natural Yeast Genomes;Systematic identification of trans eQTLs as putative drivers of known disease associations;Chapter 11: Genome-Wide Association Studies;Coordinating genome expression with cell size.;Small- and Large-Effect Quantitative Trait Locus Interactions Underlie Variation in Yeast Sporulation Efficiency;Systematic Localization of Common Disease-Associated Variation in Regulatory DNA;Linking disease associations with regulatory information in the human genome;Finding the sources of missing heritability in a yeast cross;Comparative studies of gene expression and the evolution of gene regulation;GCTA: a tool for genome-wide complex trait analysis.;Trait-Associated SNPs Are More Likely to Be eQTLs: Annotation to Enhance Discovery from GWAS;Dissection of genetically complex traits with extremely large pools of yeast segregants;THE LOCUS OF EVOLUTION: EVO DEVO AND THE GENETICS OF ADAPTATION;A survey of essential gene function in the yeast cell division cycle.;Mapping Novel Traits by Array-Assisted Bulk Segregant Analysis in Saccharomyces cerevisiae;Cell-cycle control of gene expression in budding and fission yeast.;The YJR127C/ZMS1 gene product is involved in glycerol-based respiratory growth of the yeast Saccharomyces cerevisiae;Genetic interactions between polymorphisms that affect gene expression in yeast;Enp1, a yeast protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis.;Genetic Dissection of Transcriptional Regulation in Budding Yeast;Regression Modeling Strategies: With Applications to Linear Models, Logistic Regression, and Survival Analysis;The core meiotic transcriptome in budding yeasts;Manifold anomalies in gene expression in a vineyard isolate of Saccharomyces cerevisiae revealed by DNA microarray analysis.;ENP1, an essential gene encoding a nuclear protein that is highly conserved from yeast to humans.;Evolution and tinkering.;Evolution at two levels in humans and chimpanzees.;Single-Cell RNA Sequencing in Yeast Using the 10× Genomics Chromium Device.;Dynamic Role of trans Regulation of Gene Expression in Relation to Complex Traits.;Forward genetic approaches to understanding complex behaviors.;KEGG: Kyoto Encyclopedia of Genes and Genomes;Genomics: Methods and Protocols;was often selected by the cross-validation procedure when tested on a few genes and thus we do 684 not believe that this approach will significantly change our results;Department of Molecular Genetics, University of Toronto;Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, 160 College 11 Street, Room 230, M5S3E1, Toronto, Ontario, Canada;Department of Biology, University of Toronto at Mississauga, 3359 Mississauga Rd, L5L 1C5, 7;Transcriptome: Connecting the Genome to Gene Function | Learn Science at Scitable",
         "https://www.biorxiv.org/content/biorxiv/early/2023/10/21/2023.10.18.562953.full.pdf"
        ],
        [
         "35",
         "033a24661b6865c08c45c49383378f4bd80c5310",
         "Human Brain-Wide Activation of Sleep Rhythms",
         "During sleep, our brain undergoes highly synchronized activity, orchestrated by distinct neural rhythms. Little is known about the associated brain activation during these sleep rhythms, and even less about their functional implications. In this study, we investigated the brain-wide activation underlying human sleep rhythms by employing simultaneous electroencephalography (EEG) and functional magnetic resonance imaging (fMRI) in 107 participants during nocturnal nap (first half of the night). We identified robust coupling between slow oscillations (SOs) and fast spindles during deep non-rapid eye movement (NREM) sleep (N2/3 stages), with spindle peaks consistently occurring just before the SO UP-state. This SO-spindle coupling was linked to elevated activation in both the thalamus and hippocampus, alongside increased functional connectivity from the hippocampus to the thalamus and from the thalamus to the medial prefrontal cortex (mPFC). An open-ended cognitive state decoding analysis suggested that these activations may relate to episodic memory processes, yet were distinct from task-related networks. Together, these findings highlight the thalamus as a key coordinator of hippocampal-cortical communication during sleep and provide new insights into the mechanisms by which synchronized sleep rhythms may support memory consolidation.",
         "Memory reactivation during sleep does not act holistically on object memory;Replay-triggered brain-wide activation in humans;How coupled slow oscillations, spindles and ripples coordinate neuronal processing and communication during human sleep;Targeted memory reactivation during sleep influences social bias as a function of slow-oscillation phase and delta power.;Decoding cognition from spontaneous neural activity;A model of autonomous interactions between hippocampus and neocortex driving sleep-dependent memory consolidation;The human thalamus orchestrates neocortical oscillations during NREM sleep;An open-source, high-performance tool for automated sleep staging;The default mode network in cognition: a topographical perspective;The nucleus reuniens, a thalamic relay for cortico-hippocampal interaction in recent and remote memory consolidation;The default mode network: where the idiosyncratic self meets the shared social world;Endogenous memory reactivation during sleep in humans is clocked by slow oscillation-spindle complexes;Sleep spindles mediate hippocampal-neocortical coupling during long-duration ripples;Inferences on a multidimensional social hierarchy use a grid-like code;Promoting memory consolidation during sleep: A meta-analysis of targeted memory reactivation.;Sleep Spindles: Mechanisms and Functions.;Bidirectional prefrontal-hippocampal dynamics organize information transfer during sleep in humans;Analysis of task-based functional MRI data preprocessed with fMRIPrep;Increased neuronal signatures of targeted memory reactivation during slow-wave up states;The hippocampal sharp wave–ripple in memory retrieval for immediate use and consolidation;FMRIPrep: a robust preprocessing pipeline for functional MRI;Representation of Real-World Event Schemas during Narrative Perception;Dual function of thalamic low-vigilance state oscillations: rhythm-regulation and plasticity;Thalamic Spindles Promote Memory Formation during Sleep through Triple Phase-Locking of Cortical, Thalamic, and Hippocampal Rhythms;Coordination of cortical and thalamic activity during non-REM sleep in humans;Situating the default-mode network along a principal gradient of macroscale cortical organization;Organizing conceptual knowledge in humans with a gridlike code;Hippocampo-cortical coupling mediates memory consolidation during sleep;Cued Reactivation of Motor Learning during Sleep Leads to Overnight Changes in Functional Brain Activity and Connectivity;Hippocampal sharp wave‐ripple: A cognitive biomarker for episodic memory and planning;Hierarchical nesting of slow oscillations, spindles and ripples in the human hippocampus during sleep;Machine learning for neuroimaging with scikit-learn;Methods to detect, characterize, and remove motion artifact in resting state fMRI;MEG and EEG data analysis with MNE-Python;Auditory Closed-Loop Stimulation of the Sleep Slow Oscillation Enhances Memory;Upgrading the sleeping brain with targeted memory reactivation;Fast and slow spindles during the sleep slow oscillation: disparate coalescence and engagement in memory processing.;The sleep relay—the role of the thalamus in central and decentral sleep regulation;Nipype: A Flexible, Lightweight and Extensible Neuroimaging Data Processing Framework in Python;Selective optical drive of thalamic reticular nucleus generates thalamic bursts & cortical spindles;Regional Slow Waves and Spindles in Human Sleep;FieldTrip: Open Source Software for Advanced Analysis of MEG, EEG, and Invasive Electrophysiological Data;N4ITK: Improved N3 Bias Correction;The memory function of sleep;Accurate and robust brain image alignment using boundary-based registration;Unbiased nonlinear average age-appropriate brain templates from birth to adulthood;The Human K-Complex Represents an Isolated Cortical Down-State;Spontaneous neural activity during human slow wave sleep;Symmetric diffeomorphic image registration with cross-correlation: Evaluating automated labeling of elderly and neurodegenerative brain;Forward and reverse hippocampal place-cell sequences during ripples;A component based noise correction method (CompCor) for BOLD and perfusion based fMRI;Odor Cues During Slow-Wave Sleep Prompt Declarative Memory Consolidation;An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest;The Sleep Slow Oscillation as a Traveling Wave;Normal human sleep: an overview.;EEGLAB: an open source toolbox for analysis of single-trial EEG dynamics including independent component analysis;Improved Optimization for the Robust and Accurate Linear Registration and Motion Correction of Brain Images;The functional significance of K-complexes.;A global optimisation method for robust affine registration of brain images;Psychophysiological and Modulatory Interactions in Neuroimaging;Software tools for analysis and visualization of fMRI data;Thalamocortical oscillations in the sleeping and aroused brain.;Spindle density in sleep of normal subjects.;cortex (Coulon, Budde, & Pape,;NeuroSynth: a new platform for large-scale automated synthesis of human functional neuroimaging data;Segmentation of brain MR images through a hidden Markov random field model and the expectation-maximization algorithm;Evaluation of Noisy Data;� Correspondence;Keywords : slow oscillation (SO), spindle, hippocampus, thalamus, simultaneous EEG-fMRI;The analysis code will be;sleep rhythms align. Understanding these mechanisms paves the way for developing neuromodulation interventions to enhance cognitive function during sleep and treat sleep-related disorders;all other FC pathways ( Fig. 4e );generating and coordinating sleep rhythms, the implications of reduced and the temporal dynamics of hippocampal-neocortical communication. brain-wide activation patterns",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/16/2024.10.14.618165.full.pdf"
        ],
        [
         "36",
         "0347d45ca1cd3e88a43d9b988242ed908b6634aa",
         "Species delimitation analysis indicates cryptic speciation for Terpios gelatinosus (Porifera, Demospongiae) from coastal regions of the northeastern Mediterranean Sea",
         "Sponges, comprising over 9000 recorded species, represent one of the most primitive groups of metazoans. Given the importance of species distribution records and the identification of new species in scientific research, these endeavors play a crucial role in enhancing ecological insights, conserving biodiversity, facilitating a better understanding of the relationships between various groups of organisms, advancing our knowledge of evolution, and potentially expanding biomedical implications. In this study, we used basic morphological data, mitochondrial Cytochrome Oxidase I (COI) gene, and Internal Transcribed Spacer two (ITS2) regions to evaluate the Terpios samples collected from four coastal sites along the northeastern Mediterranean Sea, covering a distance of approximately 450 km. For this purpose, 81 COI and 11 ITS2 records of the order Suberitida were mined from GenBank, and species delimitation analysis was performed using both the Automatic Barcode Gap Discovery method and the Poisson Tree Process (PTP), along with 11 samples from the present study. While we observed slight differences in spicule sizes, the overall morphology of all samples from our study was similar. Within the scope of this research, we report the first-ever presence record of T. gelatinosus in the northeastern Mediterranean Sea. Furthermore, we document evidence pointing to the potential existence of cryptic speciation in the region.",
         "Phylomitogenomics bolsters the high-level classification of Demospongiae (phylum Porifera);Shedding light on an East-Mediterranean mesophotic sponge ground community and the regional sponge fauna;Who’s there? – First morphological and DNA barcoding catalogue of the shallow Hawai’ian sponge fauna;MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization;Employing DNA barcoding as taxonomy and conservation tools for fish species censuses at the southeastern Mediterranean, a hot-spot area for biological invasion;Molecular phylogenies confirm the presence of two cryptic Hemimycale species in the Mediterranean and reveal the polyphyly of the genera Crella and Hemimycale (Demospongiae: Poecilosclerida);First record of Terpios fugax Duchassaing and Michelotti, 1864 (Demospongiae: Suberitidae) in the Eastern Pacific Ocean;Porifera of Greece: an updated checklist;Effects of phylogenetic reconstruction method on the robustness of species delimitation using single-locus data;Stability of the sponge assemblage of Mediterranean coralligenous concretions along a millennial time span;Particularly Sea-Sensitive Areas and Marine Protection Zones. A Controversial Issue that needs Interpretation;Molecular phylogenies support homoplasy of multiple morphological characters used in the taxonomy of Heteroscleromorpha (Porifera: Demospongiae).;An overview of the submerged sea caves of Sagres (South of Portugal-Algarve);Cryptic speciation and phylogeographic relationships in the elephant ear sponge Ianthella basta (Porifera, Ianthellidae) from northern Australia;Barcoding Sponges: An Overview Based on Comprehensive Sampling;Global Diversity of Sponges (Porifera);How Many Species Are There on Earth and in the Ocean?;DNA taxonomy of sponges—progress and perspectives;Phylogenetic position of sponges in early metazoan evolution and bionic applications of siliceous sponge spicules;CO1 phylogenies in diploblasts and the 'Barcoding of Life' — are we sequencing a suboptimal partition?;Intragenomic variation of the rDNA internal transcribed spacers in sponges (Phylum Porifera): implications for phylogenetic studies.;Keeping taxonomy based in morphology;DNA primers for amplification of mitochondrial cytochrome c oxidase subunit I from diverse metazoan invertebrates.;Genetic differentiation between morphotypes of the marine sponge Suberites ficus (Demospongiae: Hadromerida);A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences;The detection of disease clustering and a generalized regression approach.;Systema Porifera: A Guide To The Classification Of Sponges;Spatial patterns in sponge biodiversity in a Caribbean coral reef;Sponge ecology in the molecular era.;Polyphyly and rapid diversification in the marine sponge suborder Haplosclerina;Mitochondrial diversity of early-branching metazoa is revealed by the complete mt genome of a haplosclerid demosponge.;DNA barcoding: How it complements taxonomy, molecular phylogenetics and genetics;An evaluation of support for order-level monophyly and interrelationships within the class Demospongiae using partial data from the large subunit rDNA and cytochrome oxidase subunit I.;Family Suberitidae Schmidt, 1870;Phylogeography and taxonomy of the Indo-Pacific reef cave dwelling coralline demosponge Astrosclera 'willeyana': new data from nuclear internal transcribed spacer sequences;A molecular framework for the phylogeny of Sponges (Porifera);Phylogenetic relationships of tetractinellid sponges (Porifera: Demospongiae: Astrophorida) inferred from mitochondrial DNA sequences;Nuclear markers in evolutionary and population genetic studies of scleractinian corals and sponges;Suberitidae (Demospongiae, Hadromerida) from the North Aegean Sea;Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics;The sponges of the west-central Pacific",
         "https://www.biorxiv.org/content/biorxiv/early/2024/01/15/2024.01.12.575186.full.pdf"
        ],
        [
         "37",
         "035c95b92e2b71202ebcc9e53350745dd7a95007",
         "Structural basis for a dual-function type II-B CRISPR-Cas9",
         "Cas9 from Streptococcus pyogenes (SpCas9) revolutionized genome editing by enabling programmable DNA cleavage guided by an RNA. However, SpCas9 tolerates mismatches in the DNA-RNA duplex, which can lead to deleterious off-target editing. Here, we reveal that Cas9 from Francisella novicida (FnCas9) possesses a unique structural feature—the REC3 clamp—that underlies its intrinsic high-fidelity DNA targeting. Through kinetic and structural analyses, we show that the REC3 clamp forms critical contacts with the PAM-distal region of the R-loop, thereby imposing a novel checkpoint during enzyme activation. Notably, F. novicida encodes a non-canonical small CRISPR-associated RNA (scaRNA) that enables FnCas9 to repress an endogenous bacterial lipoprotein gene, subverting host immune detection. Structures of FnCas9 with scaRNA illustrate how partial R-loop complementarity hinders REC3 clamp docking and prevents cleavage in favor of transcriptional repression. The REC3 clamp is conserved in type II-B CRISPR-Cas9 systems, pointing to a potential path for engineering precise genome editors or developing novel antibacterial strategies. These findings reveal the dual mechanisms of high specificity and virulence by FnCas9, with broad implications for biotechnology and therapeutic development. Graphical Abstract",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/24/2024.10.22.619592.full.pdf"
        ],
        [
         "38",
         "036ab69ea914ce7d5da1f991b9ee3f75109c8d6e",
         "Alcohol drinking is associated with greater calcium activity in mouse central amygdala dynorphin-expressing neurons",
         "Alcohol use disorder (AUD) is a chronic disease that poses significant economic burden and health risks. It is pivotal to better understand brain mechanisms engaged by alcohol that promote misuse. The central amygdala (CeA) has emerged as a key mediator of excessive preclinical alcohol consumption. A dynorphin-expressing subpopulation within the CeA (CeADyn) has been implicated in excessive alcohol drinking, yet how cellular activity of CeADyn neurons relates to ongoing alcohol drinking is not well-understood. The current study interrogated the engagement of CeADyn neurons in male and female mice during voluntary alcohol consumption using fiber photometry and compared this cellular response with that of other solutions having similar motivational and/or taste characteristics. Activity of a calcium sensor, GCaMP7f, expressed in mouse CeADyn neurons was recorded and time-locked to bouts of drinking. Multilevel linear mixed modeling was applied to better resolve focal effects from complex data. These analyses revealed a relatively large increase in CeADyn neuron calcium transients after bouts of alcohol drinking compared to water or sucrose drinking, indicating these neurons are uniquely engaged during alcohol consumption. Drinking behavior unique to alcohol (i.e., longer bout durations) did not fully explain signal differences between alcohol and other solutions nor did the relatively increased alcohol response diminish over time. No other conditions or solutions tested reproduced the pronounced change in CeADyn activity associated with alcohol drinking. These findings, collectively, support the presence of a unique functional signature for alcohol in a cell population known to control excessive alcohol drinking. Highlights Central amygdala dynorphin cells (CeADyn) are firmly implicated in alcohol misuse. CeADyn neuron activity was higher when mice drank alcohol versus other solutions. Neither how mice drank alcohol nor motivational states could explain this activity. CeADyn neurons having uniquely high alcohol responses may underlie AUD development.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/02/21/2024.02.18.580880.full.pdf"
        ],
        [
         "39",
         "03a298b7781e6da3e58d530bf8f3e70b6952836d",
         "Sequence composition and conservation predict the phenotypic relevance of transposable elements",
         "Transposable elements (TEs) are powerful drivers of genome evolution, in part through their ability to rapidly rewire the host regulatory network by creating transcription factor binding sites that can potentially be turned to the host’s advantage in a process called exaptation. However, the effects on the host phenotype vary widely among different TEs. Here, we classify TEs based on their contribution to the human host phenotypes at both molecular and macroscopic scales. TE contributions to chromatin accessibility, gene expression, and the heritability of complex traits are strongly correlated to each other, confirming that the main mechanism through which TEs affect the host phenotype is through the rewiring of the regulatory network. TE sequence and evolutionary features are able to explain a large fraction of the variance in phenotypic relevance, and in particular more than 50% of the variance of their contribution to the heritability of complex traits. A conspicuous exception to this pattern is represented by TEs of the ERV1 family, whose phenotypic impact cannot be explained by our model: In particular, this family includes a set of relatively young TEs whose phenotypic relevance is much larger than would be expected based on their sequence and evolutionary parameters. These TEs are involved in fast-evolving biological processes related to the interaction of the organism with its environment. In conclusion, our results confirm quantitatively that TE insertions affect the host phenotype mostly through the rewiring of its regulatory network; identify a signature of phenotypic relevance based on sequence and conservation properties; and highlight several TEs as promising candidates for functional studies.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/03/13/2024.03.11.584453.full.pdf"
        ],
        [
         "40",
         "03bf8a25798de436ab666b3c42754c2de89ff482",
         "D2net: a dual-branch lightweight network for conveyor belt rotation detection in pipe belt conveyors",
         "None",
         "A novel combined method for conveyor belt deviation discrimination under complex operational scenarios;FastBeltNet: a dual-branch light-weight network for real-time conveyor belt edge detection;SCTNet: Single-Branch CNN with Transformer Semantic Information for Real-Time Segmentation;Machine vision based damage detection for conveyor belt safety using Fusion knowledge distillation;Real-time and accurate defect segmentation of aluminum strip surface via a lightweight network;Coal Mine Belt Conveyor Foreign Objects Recognition Method of Improved YOLOv5 Algorithm with Defogging and Deblurring;An intelligent belt wear fault diagnosis method based on deep learning;Research on Deviation Detection of Belt Conveyor Based on Inspection Robot and Deep Learning;Research in placement of measuring sensors on hexagonal idler housing with regard to requirements of pipe conveyor failure analysis;OSED: Object-specific edge detection;The conveyor belt longitudinal tear on-line detection based on improved SSR algorithm☆",
         "https://www.researchsquare.com/article/rs-5417187/latest.pdf"
        ],
        [
         "41",
         "03d6f4500d4696a207d73b7b5c905492d5420acb",
         "Exploring novel inhibitors for Babesia bigemina lactate dehydrogenase: a computational structural biology perspective.",
         "None",
         "Anti-parasitic drug discovery against Babesia microti by natural compounds: an extensive computational drug design approach;Case report: First report on human infection by tick-borne Babesia bigemina in the Amazon region of Ecuador;Lactate Dehydrogenase as a Potential Therapeutic Drug Target to Control Babesia bigemina;The Structural Basis of Babesia orientalis Lactate Dehydrogenase;Highly accurate protein structure prediction with AlphaFold;MEGA11: Molecular Evolutionary Genetics Analysis Version 11;Chemoprophylaxis for babesiosis and anaplasmosis in cattle: case report.;BRENDA, the ELIXIR core data resource in 2021: new developments and updates;Scalable molecular dynamics on CPU and GPU architectures with NAMD.;Development of unstable resistance to diminazene aceturate in Babesia bovis;Functional and structural characterization of the pentapeptide insertion of Theileria annulata lactate dehydrogenase by site-directed mutagenesis, comparative modeling and molecular dynamics simulations.;Comparative Protein Structure Modeling Using MODELLER;CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field;Structure-Based Virtual Screening for Drug Discovery: Principles, Applications and Recent Advances;Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences;Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments;Babesia: a world emerging.;DoGSiteScorer: a web server for automatic binding site prediction, analysis and druggability assessment;Kinetic analysis of the amino terminal end of active site loop of lactate deyhdrogenase from Plasmodium vivax.;Current Advances in Detection and Treatment of Babesiosis;Investigations Into the Nature, Causation, and Prevention of Texas or Southern Cattle Fever;Open Babel: An open chemical toolbox;Emerging perspectives in the research of bovine babesiosis and anaplasmosis.;CHARMM general force field: A force field for drug‐like molecules compatible with the CHARMM all‐atom additive biological force fields;QMEAN: A comprehensive scoring function for model quality assessment;Epik: a software program for pKa prediction and protonation state generation for drug-like molecules;Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes.;Chemotherapy against babesiosis.;Mapping the binding site for gossypol-like inhibitors of Plasmodium falciparum lactate dehydrogenase.;Importance of accurate charges in molecular docking: Quantum mechanical/molecular mechanical (QM/MM) approach;UCSF Chimera—A visualization system for exploratory research and analysis;Identification of Babesia bovis L-lactate dehydrogenase as a potential chemotherapeutical target against bovine babesiosis.;Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy.;Structure of Toxoplasma gondii LDH1: active-site differences from human lactate dehydrogenases and the structural basis for efficient APAD+ use.;Evolution of Cryptosporidium parvum lactate dehydrogenase from malate dehydrogenase by a very recent event of gene duplication.;Toward a pharmacophore for drugs inducing the long QT syndrome: insights from a CoMFA study of HERG K(+) channel blockers.;Cheminformatic models to predict binding affinities to human serum albumin.;Chloroquine Binds in the Cofactor Binding Site ofPlasmodium falciparum Lactate Dehydrogenase*;The structure of lactate dehydrogenase from Plasmodium falciparum reveals a new target for anti-malarial design;Thirty years of environmental health research--and growing.;VMD: visual molecular dynamics.;Recognition of errors in three‐dimensional structures of proteins;Verification of protein structures: Patterns of nonbonded atomic interactions;The neighbor-joining method: a new method for reconstructing phylogenetic trees.;Conversion of glucose of lactate by intraerythrocytic Plasmodium berghei.;The Protein Data Bank;Prediction of the Brain-Blood Distribution of a Large Set of Drugs from Structurally Derived Descriptors Using Partial Least-Squares (PLS) Modeling;A method to identify protein sequences that fold into a known three-dimensional structure.",
         "https://www.researchsquare.com/article/rs-4951492/latest.pdf"
        ],
        [
         "42",
         "03ea505917144ae863c916e9aece54480f264a49",
         "Energy Conservation in Passive Optical Networks: A Tutorial and Survey",
         "The Passive Optical Network (PON) has been evolving continuously in terms of architecture and capacity to keep up with the demand for high-speed Internet access in the access network segment. Recently, integration of Software-Defined Networking (SDN), which provides programmable and (logically) centralized network control, with PON has attracted intensive research interest to further enhance PON performance and reduce operational and capital expenditure. Although PON is regarded as an energy-efficient access network solution, it would be a major contributor for increasing energy consumption in the access network segment because of its higher penetration rate than other access network technologies. Over the past several years, the major standardization bodies like IEEE and ITU-T, and research communities have engaged in introducing energy-efficient PON solutions. This article familiarizes readers with PON evolution in terms of capacity, architecture, and its integration with virtualization and SDN based control. We present a comprehensive survey of the energy conservation research efforts in PON starting from conventional PON to SDN based PON leveraging virtual and physical network functions. This article also presents contemporary energy-efficient standardization activities in IEEE and ITU-T. To the best of our knowledge, to date, this article is the first most comprehensive survey on energy saving research and standardization on PON. We summarize the lessons learned from the recent advancements, identify important challenges ahead and outline several future research directions that can contribute to further advancement of energy-efficient PON.",
         "",
         "https://researchonline.ljmu.ac.uk/id/eprint/22956/1/COMST-Energy%20Conservation%20in%20Passive%20Optical%20Networks-Accepted.pdf"
        ],
        [
         "43",
         "04180d1bfe001f4460d0d5e57104fba7bec7f5da",
         "Receptor tyrosine kinases CAD96CA and FGFR1 function as the cell membrane receptors of insect juvenile hormone",
         "Juvenile hormone (JH) is important to maintain insect larval status; however, its cell membrane receptor has not been identified. Using the lepidopteran insect Helicoverpa armigera (cotton bollworm), a serious agricultural pest, as a model, we determined that receptor tyrosine kinases (RTKs) cadherin 96ca (CAD96CA) and fibroblast growth factor receptor homologue (FGFR1) function as JH cell membrane receptors by their roles in JH-regulated gene expression, larval status maintaining, rapid intracellular calcium increase, phosphorylation of JH intracellular receptor MET1 and cofactor Taiman, and high affinity to JH III. Gene knockout of Cad96ca and Fgfr1 by CRISPR/Cas9 in embryo and knockdown in various insect cells, and overexpression of CAD96CA and FGFR1 in mammalian HEK-293T cells all supported CAD96CA and FGFR1 transmitting JH signal as JH cell membrane receptors.",
         "Krüppel-homolog 1 exerts anti-metamorphic and vitellogenic functions in insects via phosphorylation-mediated recruitment of specific cofactors;The vitellogenin receptor functionality of the migratory locust depends on its phosphorylation by juvenile hormone;Neuropeptide ACP facilitates lipid oxidation and utilization during long-term flight in locusts;Fibroblast Growth Factor Receptor, a Novel Receptor for Vegetative Insecticidal Protein Vip3Aa;Feedback regulation of cytoneme-mediated transport shapes a tissue-specific FGF morphogen gradient;Identification of G protein-coupled receptors required for vitellogenin uptake into the oocytes of the red flour beetle, Tribolium castaneum;Protein kinase C modulates transcriptional activation by the juvenile hormone receptor methoprene-tolerant.;Reassessing ecdysteroidogenic cells from the cell membrane receptors’ perspective;Juvenile hormone-activated phospholipase C pathway enhances transcriptional activation by the methoprene-tolerant protein;Krüppel-homolog 1 mediates juvenile hormone action to promote vitellogenesis and oocyte maturation in the migratory locust.;Two-Tiered Control of Epithelial Growth and Autophagy by the Insulin Receptor and the Ret-Like Receptor, Stitcher;Receptor tyrosine kinases in Drosophila development.;Functions and Mechanisms of Fibroblast Growth Factor (FGF) Signalling in Drosophila melanogaster;Ligand-binding properties of a juvenile hormone receptor, Methoprene-tolerant;Drosophila Met and Gce are partially redundant in transducing juvenile hormone action.;Establishment of a New Cell Line from Lepidopteran Epidermis and Hormonal Regulation on the Genes;Posttranscriptional control of the competence factor βFTZ-F1 by juvenile hormone in the mosquito Aedes aegypti;Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.;Multiple actions of steroid hormones--a focus on rapid, nongenomic effects.;The modes of action of juvenile hormones: some questions we ought to ask.;The isolation of two juvenile hormone-inducible genes in Drosophila melanogaster.;RNAi Double-Stranded RNA Directs the ATP-Dependent Cleavage of mRNA at 21 to 23 Nucleotide Intervals;Fast induction of vitellogenin gene expression by juvenile hormone III in the cockroach Blattella germanica (L.) (Dictyoptera, Blattellidae).;Spruce budworm (Choristoneura fumiferana) juvenile hormone esterase: hormonal regulation, developmental expression and cDNA cloning;Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans;The Disulfide Bonds in the C-terminal Domains of the Human Insulin Receptor Ectodomain*;Juvenile hormone controls early trypsin gene transcription in the midgut of\n Aedes aegypti;heartless encodes a fibroblast growth factor receptor (DFR1/DFGF-R2) involved in the directional migration of early mesodermal cells in the Drosophila embryo.;Juvenile hormone regulation of an insect gene: a specific transcription factor and a DNA response element;Murine FGFR-1 is required for early postimplantation growth and axial organization.;Regulation of juvenile hormone esterase gene expression in the tobacco budworm (Heliothis virescens).;Microscale Thermophoresis (MST) to Detect the Interaction Between Purified Protein and Small Molecule.;A decade with the juvenile hormone receptor;Determination by LC-745 MS of juvenile hormone titers in hemolymph of the silkworm, Bombyx mori;952 The tyrosine kinase Stitcher activates Grainy head and epidermal wound healing in Drosophila . 953;Isothermal titration calorimetry.;Evidence that autophosphorylation of 820 solubilized receptors for epidermal growth factor is mediated by intermolecular cross-821 phosphorylation;Research;Juvenile Hormones O, I, and II Exist Only in Lepidoptera;Primate Atlas;The hormonal mechanism of larval molting and metamorphosis in the silkworm, 807 Bombyx mori;characteristic ( Figure 6 ). This study presents a platform to identify the agonist or inhibitor of JH 392 cell membrane receptors to develop an environmental friend insect growth regulator;receptor tyrosine kinase, juvenile hormone, cell membrane receptor, methoprene protein 1;induced gene expression;2022. Egfr signaling 866 promotes juvenile hormone biosynthesis in the German cockroach;2022. Juvenile 812 hormone membrane signaling phosphorylates USP and thus potentiates 20-hydroxyecdysone 813 action in Drosophila",
         "https://www.biorxiv.org/content/biorxiv/early/2024/02/28/2024.02.27.582377.full.pdf"
        ],
        [
         "44",
         "04194fd0c5a6cfc5decf792900c5711fd7a98753",
         "Cell-cycle dependence of bursty gene expression: insights from fitting mechanistic models to single-cell RNA-seq data",
         "Bursty gene expression is characterised by two intuitive parameters, burst frequency and burst size, the cell-cycle dependence of which has not been extensively profiled at the transcriptome level. In this study, we estimate the burst parameters per allele in the G1 and G2/M cell-cycle phases for thousands of mouse genes by fitting mechanistic models of gene expression to mRNA count data, obtained by sequencing of single cells whose cell-cycle position has been inferred using a deep-learning method. We find that upon DNA replication, the median burst frequency approximately halves, while the burst size remains mostly unchanged. Genome-wide distributions of the burst parameter ratios between the G2/M and G1 phases are wide, indicating substantial heterogeneity in transcriptional regulation patterns. We also observe a significant negative correlation between the burst frequency and size ratios, suggesting that regulatory processes do not independently control the burst parameters. Finally, we argue that to accurately estimate the burst parameter ratios, mechanistic models must explicitly account for gene copy number variation and extrinsic noise due to the coupling of transcription to cell age within the cell cycle, but corrections for technical noise due to imperfect capture of RNA molecules in sequencing experiments are less critical.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/01/23/2024.01.10.574820.full.pdf"
        ],
        [
         "45",
         "0419bc34536d0f4a3f60accd0745fa7ab3eb6d6b",
         "Mutual learning for domain adaptation: Self-distillation image dehazing network with sample-cycle",
         "None",
         "Visibility restoration for real-world hazy images via improved physical model and Gaussian total variation;Degradation-adaptive neural network for jointly single image dehazing and desnowing;Frequency-guidance Collaborative Triple-branch Network for single image dehazing;Depth color correlation-guided dark channel prior for underwater image enhancement;DHIQA: Quality assessment of dehazed images based on attentive multi-scale feature fusion and rank learning;MFID-Net: Multi-scaled feature-fused image dehazing via dynamic weights;Single Image Dehazing Using Saturation Line Prior;NightHazeFormer: Single Nighttime Haze Removal Using Prior Query Transformer;Multi-Purpose Oriented Single Nighttime Image Haze Removal Based on Unified Variational Retinex Model;Image Dehazing Transformer with Transmission-Aware 3D Position Embedding;Vision Transformers for Single Image Dehazing;IDRLP: Image Dehazing Using Region Line Prior;From Synthetic to Real: Image Dehazing Collaborating with Unlabeled Real Data;Effective Meta-Attention Dehazing Networks for Vision-Based Outdoor Industrial Systems;Towards Domain Invariant Single Image Dehazing;Effective Data-Driven Technology for Efficient Vision-Based Outdoor Industrial Systems;Single Image Dehazing Using Haze-Lines;FFA-Net: Feature Fusion Attention Network for Single Image Dehazing;Semi-Supervised Image Dehazing;Benchmarking Single-Image Dehazing and Beyond;NIMA: Neural Image Assessment;DehazeNet: An End-to-End System for Single Image Haze Removal;A Fast Single Image Haze Removal Algorithm Using Color Attenuation Prior;No-Reference Image Quality Assessment in the Spatial Domain;Single Image Haze Removal Using Dark Channel Prior;Image quality assessment: from error visibility to structural similarity;Two deterministic half-quadratic regularization algorithms for computed imaging;Remote sensing image dehazing method based on global and local advantageous feature fusion;An algorithm for enhancing low-light images at sea based on improved dark channel priors;Multiscale Depth Fusion With Contextual Hybrid Enhancement Network for Image Dehazing;International Journal of Computer Vision 48(3), 233--254, 2002",
         "http://arxiv.org/pdf/2203.09430"
        ],
        [
         "46",
         "042cde9aa3dc3d91bb528dfff54349708df45e8a",
         "Synergistic effect of inhibiting CHK2 and DNA replication on cancer cell growth",
         "Cancer cells display high levels of oncogene-induced replication stress (RS) and rely on DNA damage checkpoint for viability. This feature is exploited by cancer therapies to either increase RS to unbearable levels or inhibit checkpoint kinases involved in the DNA damage response. Thus far, treatments that combine these two strategies have shown promise but also have severe adverse effects. To identify novel, better-tolerated anticancer combinations, we screened a collection of plant extracts and found two natural compounds from the plant, Psoralea corylifolia, that synergistically inhibit cancer cell proliferation. Bakuchiol inhibited DNA replication and activated the checkpoint kinase CHK1 by targeting DNA polymerases. Isobavachalcone interfered with DNA double-strand break repair by inhibiting the checkpoint kinase CHK2 and DNA end resection. The combination of bakuchiol and isobavachalcone synergistically inhibited cancer cell proliferation in vitro. Importantly, it also prevented tumor development in xenografted NOD/SCID mice. The synergistic effect of inhibiting DNA replication and CHK2 signaling identifies a vulnerability of cancer cells that might be exploited by using clinically approved inhibitors in novel combination therapies.",
         "",
         "https://www.researchsquare.com/article/rs-3850792/latest.pdf"
        ],
        [
         "47",
         "044b3471e2d2bb3d2ae6fa4f80e1a2836e113caa",
         "De novo clustering of large long-read transcriptome datasets with isONclust3",
         "Long-read sequencing techniques can sequence transcripts from end to end, greatly improving our ability to study the transcription process. Although there are several well-established tools for long-read transcriptome analysis, most are reference-based. This limits the analysis of organisms without high-quality reference genomes and samples or genes with high variability (e.g., cancer samples or some gene families). In such settings, analysis using a reference-free method is favorable. The computational problem of clustering long reads by region of common origin is well-established for reference-free transcriptome analysis pipelines. Such clustering enables large datasets to be split up roughly by gene family and, therefore, an independent analysis of each cluster. There exist tools for this. However, none of those tools can efficiently process the large amount of reads that are now generated by long-read sequencing technologies. We present isONclust3, an improved algorithm over isONclust and isONclust2, to cluster massive long-read transcriptome datasets into gene families. Like isONclust, IsONclust3 represents each cluster with a set of minimizers. However, unlike other approaches, isONclust3 dynamically updates the cluster representation during clustering by adding high-confidence minimizers from new reads assigned to the cluster and employs an iterative cluster-merging step. We show that isONclust3 yields results with higher or comparable quality to state-of-the-art algorithms but is 10-100 times faster on large datasets. Also, using a 256Gb computing node, isONclust3 was the only tool that could cluster 37 million PacBio reads, which is a typical throughput of the recent PacBio Revio sequencing machine. Availability https://github.com/aljpetri/isONclust3 Contact kristoffer.sahlin@math.su.se",
         "Assembly Arena: Benchmarking RNA isoform reconstruction algorithms for nanopore sequencing;isONform: reference-free transcriptome reconstruction from Oxford Nanopore data;Ecological genomics in the Northern krill uncovers loci for local adaptation across ocean basins;Benchmarking of Nanopore R10.4 and R9.4.1 flow cells in single-cell whole-genome amplification and whole-genome shotgun sequencing.;Accurate isoform discovery with IsoQuant using long reads;Method of the year: long-read sequencing;Freddie: annotation-independent detection and discovery of transcriptomic alternative splicing isoforms using long-read sequencing;Reference-free assembly of long-read transcriptome sequencing data with RNA-Bloom2;Rapid in situ identification of biological specimens via DNA amplicon sequencing using miniaturized laboratory equipment;ClusTrast: a short read de novo transcript isoform assembler guided by clustered contigs;Molecular phylogeny of fucoxanthin-chlorophyll a/c proteins from Chaetoceros gracilis and Lhcq/Lhcf diversity;Sustainable data analysis with Snakemake;NGSpeciesID: DNA barcode and amplicon consensus generation from long‐read sequencing data;Syncmers are more sensitive than minimizers for selecting conserved k‑mers in biological sequences;RATTLE: reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing;Error correction enables use of Oxford Nanopore technology for reference-free transcriptome analysis;Quantifying the benefit offered by transcript assembly with Scallop-LR on single-molecule long reads;Transcriptome assembly from long-read RNA-seq alignments with StringTie2;De novo clustering of long-read transcriptome data using a greedy, quality-value based algorithm;Deciphering highly similar multigene family transcripts from Iso-Seq data with IsoCon;CARNAC-LR : Clustering coefficient-based Acquisition of RNA Communities in Long Reads;MeShClust: an intelligent tool for clustering DNA sequences;MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets;Minimap2: pairwise alignment for nucleotide sequences;Clustering huge protein sequence sets in linear time;Rust-Bio: a fast and safe bioinformatics library;Widespread Polycistronic Transcripts in Fungi Revealed by Single-Molecule mRNA Sequencing;On comparing partitions;DNACLUST: accurate and efficient clustering of phylogenetic marker genes;V-Measure: A Conditional Entropy-Based External Cluster Evaluation Measure;Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences;Reducing storage requirements for biological sequence comparison;Spectral clustering of protein sequences;Base-calling of automated sequencer traces using phred. I. Accuracy assessment.;Highlyefficientclusteringoflong-readtranscriptomic data with geluster;Discovering novel reproductive genesinanon-modelflyusingdenovogridiontranscriptomics;Effectivesequencesimilaritydetectionwithstrobemers;Search and clustering orders of magnitude faster than blast;Minimizer-iter: Iterate over minimizers of a dna sequence",
         "https://www.biorxiv.org/content/biorxiv/early/2024/11/03/2024.10.29.620862.full.pdf"
        ],
        [
         "48",
         "044c7ebb747f02dc4f0a7e518030141baa15fa26",
         "A foundation language model to decipher diverse regulation of RNAs",
         "RNA metabolism is tightly regulated by cis-elements and trans-acting factors. Most information guiding such regulation is encoded in RNA sequences. Considering the similarities in semantic and syntactic features between RNAs and human language, we developed LAMAR, a transformer-based foundation language model for RNA regulation, to decipher general rules underlying RNA processing. The model was pretrained on approximately 15 million sequences from both genome and transcriptome of 225 mammals and 1569 viruses, and further fine-tuned with labeled datasets for various tasks. The resulting fine-tuned models outperformed the state-of-the-art methods in predicting mRNA translation efficiency and mRNA half-life, while achieving comparable accuracy to specifically designed methods in predicting splice sites of pre-mRNAs and internal ribosome entry sites. Our results indicated that a single foundation language model is applicable in the comprehensive analysis of different aspects of RNA regulation, providing new insight into the design and optimization of RNA drugs.",
         "",
         "https://www.biorxiv.org/content/biorxiv/early/2024/10/13/2024.10.12.617732.full.pdf"
        ],
        [
         "49",
         "047fe4832170430bc42950c5a249f42cabcacf6b",
         "Risk Assessment of Transmission Lines Against Grid-Ignited Wildfires",
         "Wildfires ignited by the power lines have become increasingly common over the past decade. Enhancing the operational and financial resilience of power grids against wildfires involves a multifaceted approach. Key proactive measures include meticulous vegetation management, strategic grid hardening such as infrastructure undergrounding, preemptive de-energization, and disaster risk financing, among others. Each measure should be tailored to prioritize efforts in mitigating the consequences of wildfires. This paper proposes a transmission line risk assessment method for grid-ignited wildfires, identifying the transmission lines that could potentially lead to damage to the natural and built environment and to other transmission lines if igniting a wildfire. Grid, meteorological, and topological datasets are combined to enable a comprehensive analysis. Numerical analysis on the standard IEEE 30-bus system demonstrates the effectiveness of the proposed method.",
         "",
         "http://arxiv.org/pdf/2502.12401"
        ]
       ],
       "shape": {
        "columns": 5,
        "rows": 2239
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>referenceTitles</th>\n",
       "      <th>openAccessPdf</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>000a3e6667afb4ca8476ce15555b05244c3b458a</td>\n",
       "      <td>Extracting organs of interest from medical ima...</td>\n",
       "      <td>None</td>\n",
       "      <td></td>\n",
       "      <td>https://www.nature.com/articles/s41598-025-860...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>00139e24da15df3f2f4ef70a92a781ccf315ef47</td>\n",
       "      <td>DdiA, an XRE family transcriptional regulator,...</td>\n",
       "      <td>Repair of DNA damage is essential for genome i...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.biorxiv.org/content/biorxiv/early/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0027ea27f5986f9c6ee9776e10289c7f2aa089ca</td>\n",
       "      <td>Compiler Support for Speculation in Decoupled ...</td>\n",
       "      <td>Irregular codes are bottlenecked by memory and...</td>\n",
       "      <td></td>\n",
       "      <td>http://arxiv.org/pdf/2501.13553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>00291c982d2d9e69d8fe8c89a7db1fbb25f15687</td>\n",
       "      <td>Molecular mapping and functional validation of...</td>\n",
       "      <td>G protein-coupled receptors (GPCRs) are integr...</td>\n",
       "      <td>Imperial</td>\n",
       "      <td>https://www.biorxiv.org/content/biorxiv/early/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00317a3206df00f0c3c91595a349bc81b7118929</td>\n",
       "      <td>Anytime Planning for End-Effector Trajectory T...</td>\n",
       "      <td>End-effector trajectory tracking algorithms fi...</td>\n",
       "      <td></td>\n",
       "      <td>http://arxiv.org/pdf/2502.03676</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2234</th>\n",
       "      <td>fed4e1c03728d1747984ccc035e9e52d4f576666</td>\n",
       "      <td>Massive outsourcing of energetically costly am...</td>\n",
       "      <td>Animals are generally capable of synthesizing ...</td>\n",
       "      <td>The Origin of Metazoan Multicellularity: A Pot...</td>\n",
       "      <td>https://www.biorxiv.org/content/biorxiv/early/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2235</th>\n",
       "      <td>fee975b0352fbf35fdc1926b25a16d8531ebb4e6</td>\n",
       "      <td>Dialogue sentiment analysis based on dialogue ...</td>\n",
       "      <td>None</td>\n",
       "      <td>Turn-Level Active Learning for Dialogue State ...</td>\n",
       "      <td>https://www.researchsquare.com/article/rs-4129...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2236</th>\n",
       "      <td>ff622ab9a90f9881749716d70893b0f850247a4c</td>\n",
       "      <td>Boundary control and observer design via backs...</td>\n",
       "      <td>None</td>\n",
       "      <td>Stabilization of a Parabolic-Elliptic System v...</td>\n",
       "      <td>http://arxiv.org/pdf/2309.00093</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2237</th>\n",
       "      <td>ff8fd5f30461bb788c6a32f37fe0f7821eafebf6</td>\n",
       "      <td>Follow the Approximate Sparse Leader for No-Re...</td>\n",
       "      <td>We consider the problem of online sparse linea...</td>\n",
       "      <td>Improved RIP-based bounds for guaranteed perfo...</td>\n",
       "      <td>http://arxiv.org/pdf/2501.00799</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2238</th>\n",
       "      <td>ffd00ce568f98d0febe6aeb407b54f9075ea732b</td>\n",
       "      <td>AI-assisted design of experiments at the front...</td>\n",
       "      <td>Designing the next generation colliders and de...</td>\n",
       "      <td></td>\n",
       "      <td>http://arxiv.org/pdf/2501.04448</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2239 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                       paperId  \\\n",
       "0     000a3e6667afb4ca8476ce15555b05244c3b458a   \n",
       "1     00139e24da15df3f2f4ef70a92a781ccf315ef47   \n",
       "2     0027ea27f5986f9c6ee9776e10289c7f2aa089ca   \n",
       "3     00291c982d2d9e69d8fe8c89a7db1fbb25f15687   \n",
       "4     00317a3206df00f0c3c91595a349bc81b7118929   \n",
       "...                                        ...   \n",
       "2234  fed4e1c03728d1747984ccc035e9e52d4f576666   \n",
       "2235  fee975b0352fbf35fdc1926b25a16d8531ebb4e6   \n",
       "2236  ff622ab9a90f9881749716d70893b0f850247a4c   \n",
       "2237  ff8fd5f30461bb788c6a32f37fe0f7821eafebf6   \n",
       "2238  ffd00ce568f98d0febe6aeb407b54f9075ea732b   \n",
       "\n",
       "                                                  title  \\\n",
       "0     Extracting organs of interest from medical ima...   \n",
       "1     DdiA, an XRE family transcriptional regulator,...   \n",
       "2     Compiler Support for Speculation in Decoupled ...   \n",
       "3     Molecular mapping and functional validation of...   \n",
       "4     Anytime Planning for End-Effector Trajectory T...   \n",
       "...                                                 ...   \n",
       "2234  Massive outsourcing of energetically costly am...   \n",
       "2235  Dialogue sentiment analysis based on dialogue ...   \n",
       "2236  Boundary control and observer design via backs...   \n",
       "2237  Follow the Approximate Sparse Leader for No-Re...   \n",
       "2238  AI-assisted design of experiments at the front...   \n",
       "\n",
       "                                               abstract  \\\n",
       "0                                                  None   \n",
       "1     Repair of DNA damage is essential for genome i...   \n",
       "2     Irregular codes are bottlenecked by memory and...   \n",
       "3     G protein-coupled receptors (GPCRs) are integr...   \n",
       "4     End-effector trajectory tracking algorithms fi...   \n",
       "...                                                 ...   \n",
       "2234  Animals are generally capable of synthesizing ...   \n",
       "2235                                               None   \n",
       "2236                                               None   \n",
       "2237  We consider the problem of online sparse linea...   \n",
       "2238  Designing the next generation colliders and de...   \n",
       "\n",
       "                                        referenceTitles  \\\n",
       "0                                                         \n",
       "1                                                         \n",
       "2                                                         \n",
       "3                                              Imperial   \n",
       "4                                                         \n",
       "...                                                 ...   \n",
       "2234  The Origin of Metazoan Multicellularity: A Pot...   \n",
       "2235  Turn-Level Active Learning for Dialogue State ...   \n",
       "2236  Stabilization of a Parabolic-Elliptic System v...   \n",
       "2237  Improved RIP-based bounds for guaranteed perfo...   \n",
       "2238                                                      \n",
       "\n",
       "                                          openAccessPdf  \n",
       "0     https://www.nature.com/articles/s41598-025-860...  \n",
       "1     https://www.biorxiv.org/content/biorxiv/early/...  \n",
       "2                       http://arxiv.org/pdf/2501.13553  \n",
       "3     https://www.biorxiv.org/content/biorxiv/early/...  \n",
       "4                       http://arxiv.org/pdf/2502.03676  \n",
       "...                                                 ...  \n",
       "2234  https://www.biorxiv.org/content/biorxiv/early/...  \n",
       "2235  https://www.researchsquare.com/article/rs-4129...  \n",
       "2236                    http://arxiv.org/pdf/2309.00093  \n",
       "2237                    http://arxiv.org/pdf/2501.00799  \n",
       "2238                    http://arxiv.org/pdf/2501.04448  \n",
       "\n",
       "[2239 rows x 5 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "paperdata = load_dataset(dataset_source='s2', \n",
    "                                  subsets=2025, \n",
    "                                  data_types=['text', 'metadata'],\n",
    "                                  filters=None, #[('abstract', '==', 'None')],\n",
    "                                  n_jobs=21)\n",
    "display(paperdata[\"metadata\"])\n",
    "display(paperdata[\"text\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-23 18:58:58,315 - dataset.extractPDFs - INFO - Downloading pdfs for year 2006\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [00:00<?, ?batch/s]\n",
      "2025-03-23 18:59:00,882 - dataset.utils_Download - INFO - Closed 0 Chrome processes\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [01:41<?, ?batch/s, download tasks=136 left, extraction tasks=142 left]\n",
      "2025-03-23 19:00:42,896 - dataset.utils_PDF - WARNING - Font size extraction failed for /home/jul/DST/asXiv/data/tmp/00b4b239286e92774b807c21f13a614886615f8c/The Plant Journal - 2006 - Gargantini - A CDPK isoform participates in the regulation of nodule number in Medicago.pdf: Invalid dictionary construct: [/'OP', /b'fals', /b'e', /'op', False, /'Type', /'ExtGState', /'SM', 0.02, /'OPM', 1, /'SA', False]\n",
      "\n",
      "2025-03-23 19:00:42,901 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/00b4b239286e92774b807c21f13a614886615f8c/The Plant Journal - 2006 - Gargantini - A CDPK isoform participates in the regulation of nodule number in Medicago.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [03:16<?, ?batch/s, download tasks=117 left, extraction tasks=128 left]\n",
      "2025-03-23 19:02:17,431 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/020d670d9554c0c2f06f94cdc1de68e0191c3768/medvedev-et-al-2006-invited-review-tolerance-to-microbial-tlr-ligands-molecular-mechanisms-and-relevance-to-disease.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [05:13<?, ?batch/s, download tasks=90 left, extraction tasks=100 left] \n",
      "2025-03-23 19:04:14,934 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/03b9f851ff5f9045221a4e74cd6fcc43e84456f4/20060012301.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [07:48<?, ?batch/s, download tasks=59 left, extraction tasks=69 left] \n",
      "2025-03-23 19:06:50,159 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/06196c1b6e85a148552df8ec2cb59a5247258e9e/00085472can060365-sup-can_9-1-06_synowitz.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [08:00<?, ?batch/s, download tasks=54 left, extraction tasks=64 left]\n",
      "2025-03-23 19:07:01,799 - dataset.utils_PDF - WARNING - Font size extraction failed for /home/jul/DST/asXiv/data/tmp/0646aaa662a8e0b2a0a2bbcad938d68cd45102f7/strobl_2006iros.pdf: Invalid dictionary construct: [/'Matrix', [1, 0, 0, 1, 225.73438, 407.03516], /'Vertical', /b'fa', /b'lse']\n",
      "\n",
      "2025-03-23 19:07:01,810 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/0646aaa662a8e0b2a0a2bbcad938d68cd45102f7/strobl_2006iros.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [08:12<?, ?batch/s, download tasks=51 left, extraction tasks=64 left]\n",
      "2025-03-23 19:07:13,748 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/059d477efde5f20b13f53c7d54aa64657e7941f5/00085472can061163-sup-supplementary_figure_s1.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [08:37<?, ?batch/s, download tasks=47 left, extraction tasks=60 left]\n",
      "2025-03-23 19:07:38,782 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/06ae22b1dbd1f8998ca234e877b7797d24d58df2/69_341.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [08:40<?, ?batch/s, download tasks=45 left, extraction tasks=59 left]\n",
      "2025-03-23 19:07:42,534 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/069828e26acc3313d0c98fda92c08e49b2f27550/05-40_fin.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [11:41<?, ?batch/s, download tasks=6 left, extraction tasks=16 left] \n",
      "2025-03-23 19:10:42,794 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/091486a021b09f987cfbce1042ae56e18b734cc1/00085472can052850-sup-supplementary_figure_1.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [11:52<?, ?batch/s, download tasks=2 left, extraction tasks=12 left]\n",
      "2025-03-23 19:10:53,616 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/08f096e099322ca7afd913d4650800d971640e57/Tellez_JMB_2006.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [11:59<?, ?batch/s, download tasks=1 left, extraction tasks=10 left]\n",
      "2025-03-23 19:11:00,396 - dataset.utils_PDF - WARNING - PDF /home/jul/DST/asXiv/data/tmp/08f2d51806af94338c4cc35024d553b362da797b/BotvinickPlaut06PR.serialOrder.pdf likely corrupted\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [12:08<?, ?batch/s, extraction tasks=9 still running,                                                         4.0 seconds left]              \n",
      "2025-03-23 19:11:11,367 - dataset.extractPDFs - INFO - 9 extractions and 0 downloads out of 150 timed out. Terminating the pool.\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [12:10<?, ?batch/s, status=checking valid pages for 6articles that timed out...]                                               \n",
      "2025-03-23 19:11:12,800 - dataset.utils_PDF - WARNING - Timeout on page 3 out of 18 for /home/jul/DST/asXiv/data/tmp/06821186194e8279a1872d0bd6e7517979df2ea4/1471-2105-7-100.pdf\n",
      "\n",
      "2025-03-23 19:11:18,702 - dataset.utils_PDF - WARNING - Timeout on page 6 out of 23 for /home/jul/DST/asXiv/data/tmp/08e75a3571ca244b8ef3f76a7d05f462d1f7aa8c/0005-06.pdf\n",
      "Year 2006 / 1 years (3000 papers):   0%|          | 0/1 [12:19<?, ?batch/s, status=2268 articles extracted out of 3000]                 "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2268\n",
      "final_results collected\n",
      "collecting download results\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Year 2006 / 1 years (3000 papers): 100%|██████████| 1/1 [12:20<00:00, 740.28s/batch, status=2268 articles extracted out of 3000]\n",
      "\n",
      "2025-03-23 19:11:21,322 - dataset.utils_Download - INFO - Closed 206 Chrome processes\n"
     ]
    }
   ],
   "source": [
    "textdata = download_and_extract_PDFs(\"s2\", years=2006, n_jobs=[10, 20])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorName",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "author_list",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_discussion",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_introduction",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_methods",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_results",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_status",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_text",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "valid_pages",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "af0d36b5-1d5c-4110-b973-dc7c8637d9dc",
       "rows": [
        [
         "0",
         "00009f98f5ff962beb322e6c0e78e4b77dd2d968",
         "Tungiasis is a parasitic skin disease widespread in resource-poor urban and rural communities in Brazil. Inhabitants of an urban slum in Northeast Brazil were examined for the presence of tungiasis lesions and followed-up twice a week for a period of three weeks. Each time the number, stages, and topographic localization of lesions were recorded on a documentation sheet. The infestation rate (number of newly embedded sand fleas per individual and day) remained stable during the observation period. The infestation rate was significantly related to the intensity of infestation (total number of lesions present) (rho = 0.70, p < 0.0001) and the proportion of viable lesions (rho = 0.28, p < 0.0001). The results indicate that in an endemic area the infestation intensity and the proportion of viable lesions can be used as a proxy to assess the exposure of individuals at risk for tungiasis. Persistently high infestation rates during the transmission season favour the use of prevention measures against invading sand fleas (such as a repellent) rather than a drug to kill already embedded parasites.",
         "H. Feldmeier,J. D. Kehr,G. Poggensee,J. Heukelbach",
         "H. Feldmeier,J. D. Kehr,G. Poggensee,J. Heukelbach",
         "https://www.scielo.br/j/mioc/a/csnpsRLq5FqY5MkQLf8GRjG/?lang=en&format=pdf",
         "Department of Medical Microbiology and Immunology of Infection, Institute of Infection Medicine, Charité-University Medi-\ncine Berlin, Germany *Department of Infectious Disease Epidemiology, Robert Koch-Institute, Berlin, Germany **Departmento\nde Saúde Comunitária, Faculdade de Medicina, Universidade Federal do Ceará e Fundação Mandacaru, Rua Prof. Costa Mendes\n1608, 5°andar, 60430-140 Fortaleza, CE, Brasil\nTungiasis  is  a  parasitic  skin  disease  widespread  in  resource-poor  urban  and  rural  communities  in  Brazil.\nInhabitants of an urban slum in Northeast Brazil were examined for the presence of tungiasis lesions and followed-\nup twice a week for a period of three weeks. Each time the number, stages, and topographic localization of lesions\nwere recorded on a documentation sheet. The infestation rate (number of newly embedded sand fleas per individual\nand  day)  remained  stable  during  the  observation  period.  The  infestation  rate  was  significantly  related  to  the\nintensity  of  infestation  (total  number  of  lesions  present)  (rho  =  0.70,  p  <  0.0001)  and  the  proportion  of  viable\nlesions  (rho  =  0.28,  p  <  0.0001).  The  results  indicate  that  in  an  endemic  area  the  infestation  intensity  and  the\nproportion  of  viable  lesions  can  be  used  as  a  proxy  to  assess  the  exposure  of  individuals  at  risk  for  tungiasis.\nPersistently high infestation rates during the transmission season favour the use of prevention measures against\ninvading sand fleas (such as a repellent) rather than a drug to kill already embedded parasites.",
         "DISCUSSION\nTungiasis occurs in Central and South America, the\nCaribbean,  and  sub-Saharan Africa  (Heukelbach  et  al.\n2001). Prevalence in the general population may be 50%\nand more in the general population and even higher in\nchildren and old people (Carvalho et al. 2003, Muehlen et\nal. 2003, Heukelbach et al. 2005a). The ectoparasitosis is\npainful, particularly when lesions are located along the\nnail rim. Pain and itching impede people from sleeping\nwell. In resource-poor settings bacterial superinfection is\na constant finding (Feldmeier et al. 2002). Deformation\nand auto-amputation of digits, deformation and loss of\nnails, development of chronic ulcers, gangrene, difficulty\nwalking, and gripping (when lesions are located at the\nhands) are known sequels (Heukelbach et al. 2001).\ncorded on a visual documentation sheet, and newly em-\nbedded sand fleas were counted.\nThe  results  show  that  in  the  group  studied  on  the\naverage each individual experienced two successful pen-\netrations of female T. penetrans per day. As supposedly\nthe infestation rate does not vary much during the trans-\nmission season of T. penetrans in Northeast Brazil, i.e.\nfrom May to December, each participant of the study popu-\nlation will experience at least 480 sand flea lesions per\nyear.\nTo the best of our knowledge, infestation rates have\nnever been determined in parasitic skin diseases. Thus,\ncomparison can only be made with inoculation rates of\ndiseases transmitted by biting insects. For example, in a\nhyperendemic malaria area the highest inoculation rates\n(the number of sporozoites of P. falciparum transmitted\nby anopheles mosquitoes per person per year) observed,\nranged from 34 to 630 (van der Kolk et al.  2003, Appawu et\nal.  2004). For other important arthropod-borne infections,\nthese  rates  are  much  lower. Although  inoculation  rates\nare not directly comparable with the infestation rate used\nhere,  it  is  obvious  that  the  transmission  potential  of T.\npenetrans is similar to that of P. falciparum in a hyperen-\ndemic malaria area.\nAs indicated by an intra class correlation coefficient\nof 0.2 there was little variation in the infestation rate within\naffected  individuals  as  compared  to  variation  between\nindividuals. Stable infestation rates during the observa-\ntion period were confirmed by a similar proportion of vi-\nable lesions at the beginning and the end of the study.\nThere was a highly significant correlation between the\ninfestation  rate  and  the  infestation  intensity  during  the\nstudy period (rho = 0.70, P < 0.001). Thus, the infestation\nintensity can be used as a proxy for the infestation rate.\nThis  has  practical  consequences  as  the  assessment  of\nthe infestation rate is laborious, needs repeated examina-\ntions and can only be performed by an experienced health\nworker.  In  contrast,  counting  once  the  total  number  of\nlesions can be rapidly performed by an auxiliary, a com-\nmunity health worker or even by the affected individual.\nAs the infestation rate did not vary considerably in\nour  study  and  because  the  development  and  resolution\nof sand flea lesions follow a typical time pattern (Eisele et\nal.  2003), it seemed conceivable that the proportion of\nviable lesions (stage I to III) observed in an individual at\nany given point of time also reflects individual exposure.\nThis assumption was confirmed by a highly significant\ncorrelation between the infestation rate and the propor-\ntion of viable lesions.\nThe almost constant re-infestation during three weeks\nof  the  transmission  season  is  a  good  argument  against\nchemotherapy. Even if a highly effective compound was\nat hand, which actually is not the case (Heukelbach et al.\n2003a,  2004), it would have to be administered repeatedly\nin order to kill newly embedded sand fleas. By conse-\nquence,  in  order  to  keep  the  parasite  load  low  and  the\ndevelopment of morbidity at bay, the use of a repellent\nseems to be a more rational alternative (Schwalfenberg et\nal.  2004).\nOur  study  has  several  limitations. As  study  partici-\npants were selected according to a minimal number of\nlesions present at baseline, the selection procedure caused\na disproportionally high number of moderately to highly\ninfested individuals to be included in the study. There-\nfore, one cannot infer on the true distribution of infesta-\ntion rates in the community from these data. Moreover,\nthe  correlation  between  infestation  rate  and  infestation\nintensity might change, if many individuals with a low\ninfestation rate are included in the regression analysis.\nSecond,  we  measured  the  infestation  rate  during  a\nperiod of three weeks. The variation of this measure might\nbe higher if individuals are followed-up for a longer pe-\nriod of time.\nIn conclusion, our data show for the first time that it is\npossible  to  determine  the  degree  of  the  exposure  to  T.\npenetrans in individuals of an endemic community. The\ninfestation rate only varied little over time and predicted\nthe number of sand flea lesions present in an individual.",
         "Tungiasis is a neglected parasitic skin disease of im-\npoverished populations (Heukelbach et al.  2001). The\nectoparasitosis occurs in Central and South America, the\nCaribbean and sub-Saharan Africa (Ade-Serrano & Ejezie\n1981, Arene 1984, Matias 1989, Chadee 1994, 1998, Carvalho\net al.  2003). In Brazil it is widespread in urban squatter\nsettlements, villages in the rural hinterland and in tradi-\ntional fishing communities along the coast (Wilcke et al.\n2002, Carvalho et al.  2003).\nCommunity-based  studies  have  consistently  shown\nprevalences between 16 and 55% in typical endemic areas\nwith a peak of age-specific disease occurrence in children\n5  to  14  years  and  the  elderly,  and  a  preponderance  of\ninfestation in the male sex (Ade-Serrano & Ejezie 1981,\nArene 1984, Matias 1989, Chadee 1994, Wilcke et al.  2002,\nMuehlen et al.  2003).\nThere is a considerable seasonal variation in the in-\ntensity of infestation indicating that transmission varies\nduring the year (Heukelbach et al.  2005a). In the semi-arid\nNortheast Brazil, attack rates decrease as soon as the rainy\nseason begins and re-increase when the weather becomes\ndryer.  Environmental  determined  patterns  in  off-host\npropagation  and  development  of  T.  penetrans  seem  to\nunderlie the seasonal variation of attack rates.\nIn  contrast  to  other  ectoparasitoses  such  as  scabies\nand pediculocis, tungiasis is a self-limiting disease with\nduration of four to six weeks (Eisele et al.  2003). As re-\ninfestation is the rule and new lesions may appear close\nto already embedded fleas, cross-sectional surveys can-\nnot be used to infer on local transmission dynamics. How-\never, anecdotal observations from colonial times suggest\nthat transmission dynamics vary considerably from loca-\ntion to location and that attack rates differ between indi-\nviduals living in the same area (Karsten 1865, Jolly 1926,\nGordon 1941).\nIn fact, the number of embedded sand fleas per indi-\nvidual varies considerably. Whereas in travellers only a\nfew lesions are observed, individuals in the endemic area\nmay carry several hundred parasites in different stages of\ndevelopment (Bezerra 1994, Mashek et al.  1997, Franck et\nal.  2003). There is convincing evidence that severe mor-\nbidity is the result of a high infestation (Feldmeier et al.\n2003).\nAs  a  first  step  towards  the  assessment  and  under-\nstanding of local transmission dynamics in endemic popu-\nlations we developed a method to assess the infestation\nrate (number of newly embedded sand fleas per individual\nand day) and correlated this measure with infestation in-\ntensity (total number of lesions present) and the propor-\ntion of viable lesions. The results show that infestation\nrates remained relatively stable during a period of three\nweeks and were significantly correlated to the parasite\nburden and the proportion of viable lesions.",
         "MATERIALS  AND  METHODS\nThe  study  was  conducted  in  Luxou,  Morro  do\nSandra’s and Novo Rumo, three neighbourhoods of the\nfavela Vicente Pinzón, an agglomeration of shantytowns\nin Fortaleza, the capital of the state of Ceará in Northeast\nBrazil. The shantytown has a total population of about\n15,000 inhabitants, and the three neighbourhoods of  3400,\n1500, and 4000, respectively. The three neighbourhoods\nare located on a dune with only a few paved roads. There\nis one Primary Health Care Centre for the whole favela.\nThe  prevalence  of  the  parasitic  skin  diseases  scabies,\ncutaneous larva migrans, pediculosis, and tungiasis is high\n(Heukelbach et al. 2003b, 2005a,b). There is no systemic\nwaste collection, public sewage disposal does not exist,\nand hygiene conditions are precarious. Many stray dogs\nroam through the favela and rats are seen even at day-\ntime. Unemployment and illiteracy is high, and alcohol-\nism, crime, and domestic violence are important social prob-\nlems. Sixty percent of the population have a monthly fam-\nily income of less than two minimum wages (1 minimum\nwage H = 100 Euro) (Family Health Program 1999).\nIn cooperation with community leaders and members\nof the neighbourhood associations, households affected\nby  tungiasis  were  identified.  In  these  households  each\nfamily member was thoroughly examined for the presence\nof embedded fleas.\nThe following findings were considered to be pathog-\nnomonic for tungiasis: an itching red-brownish spot with\na diameter of 1 to 3 mm, a circular lesion presenting as a\nwhite patch with a diameter of 4 to 10 mm with a central\nblack dot, black crust surrounded by necrotic tissue, as\nwell as partially or totally removed fleas leaving a charac-\nteristic sore in the skin (Eisele et al.  2003). Lesions were\ndocumented on a visual record sheet. They were counted\nand  staged  according  to  the  “Fortaleza  Classification”\n(Eisele et al.  2003), in which stages I to III correspond to\nviable sand fleas.\nAny individual with a minimum of seven embedded\nfleas (or lesions altered through manipulation) was eli-\ngible for the study. Individuals expected not to be regu-\nlarly at home in the coming three weeks, children less\nthan one year and individuals with severely inflamed le-\nsions were not eligible. The latter were immediately trans-\nferred to the next Primary Health Care Centre.\nIndividuals admitted to the study were examined twice\na week from July 17th to August 11th  2003. Seven points\nof time of examination were available for the assessment\nof  the  infestation  intensity  (total  number  of  embedded\nsand fleas) and the proportion of viable lesions, and six\nobservation periods for the determination of the infesta-\ntion rate (number of newly embedded sand fleas per day\nof observation).\nIndividual infestation rates were calculated by divid-\ning the number of newly embedded sand fleas diagnosed\nat the examination through the person – days of observa-\ntion, i.e. the number of days having passed since the last\nexamination.\nThe intra class correlation coefficient (ICC) was cal-\nculated to assess the variation of the infestation rate within\nand  between  individuals  during  the  study  period.  The\nICC can take a value between +1.0 and –1.0. The ICC ap-\nproaches 1.0 when the variation between individuals is\nvery large in relation to the variation within individuals\nover time and approaches –1.0 when the contrary is the\ncase. Differences of measurements between male and fe-\nmale study members were compared with the Wilcoxon\nsigned rank test. The Spearman rank correlation coeffi-\ncient was calculated to assess the significance of asso-\nciation between the parameters. Significance calculations\nwere performed with SPSS 11.0 software.\nPrior to the studies, meetings with community health\nworkers and community leaders were held and the objec-\ntives were explained. Community associations of the town-\nship (associações dos moradores) gave their consent to\nthe study. The study protocol was approved by the Ethi-\ncal Committee of the Federal University of Ceará, Brazil.\nInformed consent was obtained from all adult participants\nand from the parents or legal guardians of minors. At the\nend of the study, all participants and household members\nwith tungiasis were treated twice daily for a period of four\nweeks with a plant-based repellent which effectively re-\nduces the infestation with T. penetrans (Schwalfenberg\net al.  2004).",
         "RESULTS\nOf the 77 subjects included in the study, nine were\nexcluded from data analysis, because they were not en-\ncountered in at least one of the six follow-up examina-\ntions. Of the remaining 68 individuals, 36 were males and\n32 females. Median age was 7.4 years (range 1 to 44 years).\nThe parasitological data stratified by sex are depicted in\nthe Table. The infestation intensity varied from 7 to 144\nembedded sand fleas. The infestation rate ranged from\n0.8 to 10.75 newly embedded sand fleas per individual\nand day. It showed a highly skewed distribution with the\nmajority of subjects experiencing one or two new lesions\nMedian\n(interquartile range)\nMinimum\nParameter\nInfestation rate a\nMales\nFemales\nTotal\nIntensity of infestation b\nMales\nFemales\nTotal\na: based on the median number of newly penetrated fleas per individual and day at six observations periods; b: based on the median\nnumber of total number of lesions present at seven examinations.\n34.0 (24.8 - 47.1)\n22.0 (16.0 - 38.3)\n27.3 (18.0 - 40.9)\n2.4 (1.9 - 3.2)\n1.7 (1.4 - 2.4)\n2.0 (1.6 - 2.9)\nper day (Fig. 1). Neither the infestation rate nor the infes-\ntation intensity differed between male and female partici-\npants (p = 0.09 and p = 0.19, respectively).\nThe  infestation  rate  and  the  intensity  of  infestation\ndid not vary considerably during the observation period\n(Figs 2A, B). An intra class correlation coefficient of 0.19\n(95% CI: 0.02 –1.00) for the infestation rate indicated that\nvariation within individuals was relatively small as com-\npared to variation between individuals (p = 0.04).\nAt the beginning of the study 53% of the lesions con-\nsisted of viable sand fleas (stage I to III), 30% were in\nstage IV (dead and decaying parasites), and 17% had been\nmanipulated with a perforating instrument by the patient\nor his/her carer in an attempt to extract the embedded flea.\nThe  proportions  did  not  vary  significantly  during  the\nobservation time and were 59, 20, and 21%, respectively,\nat the last examination.\nStudies performed so far indicate that the ectoparasi-\ntosis is unevenly distributed within the community, with\na  few  individuals  bearing  a  high  number  of  embedded\nsand fleas (Feldmeier et al.  2003, Muehlen et al.  2003).\nCertain risk factors such as age, type of housing, level of\neducation, and a low socio-economic profile predispose\nindividuals  to  develop  a  high  intensity  of  infestation\n(Muehlen et al.  2005).\nThe shortcomings of the existing studies are that they\nare cross-sectional examinations and do not allow to as-\nsess to which degree individuals of a community have\nbeen  exposed.  However,  the  understanding  of  factors\ngoverning local transmission is crucial for the planning of\ncontrol  measures.  Precise  assessment  of  transmission\nwould  also  be  helpful  in  identifying  individuals  who\ndisproportionably  contribute  to  the  spread  of  the  ecto-\nparasites and/or who are most vulnerable for the devel-\nopment of severe disease.\nTo assess exposure on the individual level we devel-\noped a method to determine the number of newly embed-\nded sand fleas in an individual per unit of time. Individu-\nals with tungiasis were carefully examined twice a week.\nLocation of lesions and stages of development were re-",
         "downloading",
         "Department of Medical Microbiology and Immunology of Infection, Institute of Infection Medicine, Charité-University Medi-\ncine Berlin, Germany *Department of Infectious Disease Epidemiology, Robert Koch-Institute, Berlin, Germany **Departmento\nde Saúde Comunitária, Faculdade de Medicina, Universidade Federal do Ceará e Fundação Mandacaru, Rua Prof. Costa Mendes\n1608, 5°andar, 60430-140 Fortaleza, CE, Brasil\nTungiasis  is  a  parasitic  skin  disease  widespread  in  resource-poor  urban  and  rural  communities  in  Brazil.\nInhabitants of an urban slum in Northeast Brazil were examined for the presence of tungiasis lesions and followed-\nup twice a week for a period of three weeks. Each time the number, stages, and topographic localization of lesions\nwere recorded on a documentation sheet. The infestation rate (number of newly embedded sand fleas per individual\nand  day)  remained  stable  during  the  observation  period.  The  infestation  rate  was  significantly  related  to  the\nintensity  of  infestation  (total  number  of  lesions  present)  (rho  =  0.70,  p  <  0.0001)  and  the  proportion  of  viable\nlesions  (rho  =  0.28,  p  <  0.0001).  The  results  indicate  that  in  an  endemic  area  the  infestation  intensity  and  the\nproportion  of  viable  lesions  can  be  used  as  a  proxy  to  assess  the  exposure  of  individuals  at  risk  for  tungiasis.\nPersistently high infestation rates during the transmission season favour the use of prevention measures against\ninvading sand fleas (such as a repellent) rather than a drug to kill already embedded parasites.\nTungiasis is a neglected parasitic skin disease of im-\npoverished populations (Heukelbach et al.  2001). The\nectoparasitosis occurs in Central and South America, the\nCaribbean and sub-Saharan Africa (Ade-Serrano & Ejezie\n1981, Arene 1984, Matias 1989, Chadee 1994, 1998, Carvalho\net al.  2003). In Brazil it is widespread in urban squatter\nsettlements, villages in the rural hinterland and in tradi-\ntional fishing communities along the coast (Wilcke et al.\n2002, Carvalho et al.  2003).\nCommunity-based  studies  have  consistently  shown\nprevalences between 16 and 55% in typical endemic areas\nwith a peak of age-specific disease occurrence in children\n5  to  14  years  and  the  elderly,  and  a  preponderance  of\ninfestation in the male sex (Ade-Serrano & Ejezie 1981,\nArene 1984, Matias 1989, Chadee 1994, Wilcke et al.  2002,\nMuehlen et al.  2003).\nThere is a considerable seasonal variation in the in-\ntensity of infestation indicating that transmission varies\nduring the year (Heukelbach et al.  2005a). In the semi-arid\nNortheast Brazil, attack rates decrease as soon as the rainy\nseason begins and re-increase when the weather becomes\ndryer.  Environmental  determined  patterns  in  off-host\npropagation  and  development  of  T.  penetrans  seem  to\nunderlie the seasonal variation of attack rates.\nIn  contrast  to  other  ectoparasitoses  such  as  scabies\nand pediculocis, tungiasis is a self-limiting disease with\nduration of four to six weeks (Eisele et al.  2003). As re-\ninfestation is the rule and new lesions may appear close\nto already embedded fleas, cross-sectional surveys can-\nnot be used to infer on local transmission dynamics. How-\never, anecdotal observations from colonial times suggest\nthat transmission dynamics vary considerably from loca-\ntion to location and that attack rates differ between indi-\nviduals living in the same area (Karsten 1865, Jolly 1926,\nGordon 1941).\nIn fact, the number of embedded sand fleas per indi-\nvidual varies considerably. Whereas in travellers only a\nfew lesions are observed, individuals in the endemic area\nmay carry several hundred parasites in different stages of\ndevelopment (Bezerra 1994, Mashek et al.  1997, Franck et\nal.  2003). There is convincing evidence that severe mor-\nbidity is the result of a high infestation (Feldmeier et al.\n2003).\nAs  a  first  step  towards  the  assessment  and  under-\nstanding of local transmission dynamics in endemic popu-\nlations we developed a method to assess the infestation\nrate (number of newly embedded sand fleas per individual\nand day) and correlated this measure with infestation in-\ntensity (total number of lesions present) and the propor-\ntion of viable lesions. The results show that infestation\nrates remained relatively stable during a period of three\nweeks and were significantly correlated to the parasite\nburden and the proportion of viable lesions.\nMATERIALS  AND  METHODS\nThe  study  was  conducted  in  Luxou,  Morro  do\nSandra’s and Novo Rumo, three neighbourhoods of the\nfavela Vicente Pinzón, an agglomeration of shantytowns\nin Fortaleza, the capital of the state of Ceará in Northeast\nBrazil. The shantytown has a total population of about\n15,000 inhabitants, and the three neighbourhoods of  3400,\n1500, and 4000, respectively. The three neighbourhoods\nare located on a dune with only a few paved roads. There\nis one Primary Health Care Centre for the whole favela.\nThe  prevalence  of  the  parasitic  skin  diseases  scabies,\ncutaneous larva migrans, pediculosis, and tungiasis is high\n(Heukelbach et al. 2003b, 2005a,b). There is no systemic\nwaste collection, public sewage disposal does not exist,\nand hygiene conditions are precarious. Many stray dogs\nroam through the favela and rats are seen even at day-\ntime. Unemployment and illiteracy is high, and alcohol-\nism, crime, and domestic violence are important social prob-\nlems. Sixty percent of the population have a monthly fam-\nily income of less than two minimum wages (1 minimum\nwage H = 100 Euro) (Family Health Program 1999).\nIn cooperation with community leaders and members\nof the neighbourhood associations, households affected\nby  tungiasis  were  identified.  In  these  households  each\nfamily member was thoroughly examined for the presence\nof embedded fleas.\nThe following findings were considered to be pathog-\nnomonic for tungiasis: an itching red-brownish spot with\na diameter of 1 to 3 mm, a circular lesion presenting as a\nwhite patch with a diameter of 4 to 10 mm with a central\nblack dot, black crust surrounded by necrotic tissue, as\nwell as partially or totally removed fleas leaving a charac-\nteristic sore in the skin (Eisele et al.  2003). Lesions were\ndocumented on a visual record sheet. They were counted\nand  staged  according  to  the  “Fortaleza  Classification”\n(Eisele et al.  2003), in which stages I to III correspond to\nviable sand fleas.\nAny individual with a minimum of seven embedded\nfleas (or lesions altered through manipulation) was eli-\ngible for the study. Individuals expected not to be regu-\nlarly at home in the coming three weeks, children less\nthan one year and individuals with severely inflamed le-\nsions were not eligible. The latter were immediately trans-\nferred to the next Primary Health Care Centre.\nIndividuals admitted to the study were examined twice\na week from July 17th to August 11th  2003. Seven points\nof time of examination were available for the assessment\nof  the  infestation  intensity  (total  number  of  embedded\nsand fleas) and the proportion of viable lesions, and six\nobservation periods for the determination of the infesta-\ntion rate (number of newly embedded sand fleas per day\nof observation).\nIndividual infestation rates were calculated by divid-\ning the number of newly embedded sand fleas diagnosed\nat the examination through the person – days of observa-\ntion, i.e. the number of days having passed since the last\nexamination.\nThe intra class correlation coefficient (ICC) was cal-\nculated to assess the variation of the infestation rate within\nand  between  individuals  during  the  study  period.  The\nICC can take a value between +1.0 and –1.0. The ICC ap-\nproaches 1.0 when the variation between individuals is\nvery large in relation to the variation within individuals\nover time and approaches –1.0 when the contrary is the\ncase. Differences of measurements between male and fe-\nmale study members were compared with the Wilcoxon\nsigned rank test. The Spearman rank correlation coeffi-\ncient was calculated to assess the significance of asso-\nciation between the parameters. Significance calculations\nwere performed with SPSS 11.0 software.\nPrior to the studies, meetings with community health\nworkers and community leaders were held and the objec-\ntives were explained. Community associations of the town-\nship (associações dos moradores) gave their consent to\nthe study. The study protocol was approved by the Ethi-\ncal Committee of the Federal University of Ceará, Brazil.\nInformed consent was obtained from all adult participants\nand from the parents or legal guardians of minors. At the\nend of the study, all participants and household members\nwith tungiasis were treated twice daily for a period of four\nweeks with a plant-based repellent which effectively re-\nduces the infestation with T. penetrans (Schwalfenberg\net al.  2004).\nRESULTS\nOf the 77 subjects included in the study, nine were\nexcluded from data analysis, because they were not en-\ncountered in at least one of the six follow-up examina-\ntions. Of the remaining 68 individuals, 36 were males and\n32 females. Median age was 7.4 years (range 1 to 44 years).\nThe parasitological data stratified by sex are depicted in\nthe Table. The infestation intensity varied from 7 to 144\nembedded sand fleas. The infestation rate ranged from\n0.8 to 10.75 newly embedded sand fleas per individual\nand day. It showed a highly skewed distribution with the\nmajority of subjects experiencing one or two new lesions\nMedian\n(interquartile range)\nMinimum\nParameter\nInfestation rate a\nMales\nFemales\nTotal\nIntensity of infestation b\nMales\nFemales\nTotal\na: based on the median number of newly penetrated fleas per individual and day at six observations periods; b: based on the median\nnumber of total number of lesions present at seven examinations.\n34.0 (24.8 - 47.1)\n22.0 (16.0 - 38.3)\n27.3 (18.0 - 40.9)\n2.4 (1.9 - 3.2)\n1.7 (1.4 - 2.4)\n2.0 (1.6 - 2.9)\nper day (Fig. 1). Neither the infestation rate nor the infes-\ntation intensity differed between male and female partici-\npants (p = 0.09 and p = 0.19, respectively).\nThe  infestation  rate  and  the  intensity  of  infestation\ndid not vary considerably during the observation period\n(Figs 2A, B). An intra class correlation coefficient of 0.19\n(95% CI: 0.02 –1.00) for the infestation rate indicated that\nvariation within individuals was relatively small as com-\npared to variation between individuals (p = 0.04).\nAt the beginning of the study 53% of the lesions con-\nsisted of viable sand fleas (stage I to III), 30% were in\nstage IV (dead and decaying parasites), and 17% had been\nmanipulated with a perforating instrument by the patient\nor his/her carer in an attempt to extract the embedded flea.\nThe  proportions  did  not  vary  significantly  during  the\nobservation time and were 59, 20, and 21%, respectively,\nat the last examination.\nStudies performed so far indicate that the ectoparasi-\ntosis is unevenly distributed within the community, with\na  few  individuals  bearing  a  high  number  of  embedded\nsand fleas (Feldmeier et al.  2003, Muehlen et al.  2003).\nCertain risk factors such as age, type of housing, level of\neducation, and a low socio-economic profile predispose\nindividuals  to  develop  a  high  intensity  of  infestation\n(Muehlen et al.  2005).\nThe shortcomings of the existing studies are that they\nare cross-sectional examinations and do not allow to as-\nsess to which degree individuals of a community have\nbeen  exposed.  However,  the  understanding  of  factors\ngoverning local transmission is crucial for the planning of\ncontrol  measures.  Precise  assessment  of  transmission\nwould  also  be  helpful  in  identifying  individuals  who\ndisproportionably  contribute  to  the  spread  of  the  ecto-\nparasites and/or who are most vulnerable for the devel-\nopment of severe disease.\nTo assess exposure on the individual level we devel-\noped a method to determine the number of newly embed-\nded sand fleas in an individual per unit of time. Individu-\nals with tungiasis were carefully examined twice a week.\nLocation of lesions and stages of development were re-\nDISCUSSION\nTungiasis occurs in Central and South America, the\nCaribbean,  and  sub-Saharan Africa  (Heukelbach  et  al.\n2001). Prevalence in the general population may be 50%\nand more in the general population and even higher in\nchildren and old people (Carvalho et al. 2003, Muehlen et\nal. 2003, Heukelbach et al. 2005a). The ectoparasitosis is\npainful, particularly when lesions are located along the\nnail rim. Pain and itching impede people from sleeping\nwell. In resource-poor settings bacterial superinfection is\na constant finding (Feldmeier et al. 2002). Deformation\nand auto-amputation of digits, deformation and loss of\nnails, development of chronic ulcers, gangrene, difficulty\nwalking, and gripping (when lesions are located at the\nhands) are known sequels (Heukelbach et al. 2001).\ncorded on a visual documentation sheet, and newly em-\nbedded sand fleas were counted.\nThe  results  show  that  in  the  group  studied  on  the\naverage each individual experienced two successful pen-\netrations of female T. penetrans per day. As supposedly\nthe infestation rate does not vary much during the trans-\nmission season of T. penetrans in Northeast Brazil, i.e.\nfrom May to December, each participant of the study popu-\nlation will experience at least 480 sand flea lesions per\nyear.\nTo the best of our knowledge, infestation rates have\nnever been determined in parasitic skin diseases. Thus,\ncomparison can only be made with inoculation rates of\ndiseases transmitted by biting insects. For example, in a\nhyperendemic malaria area the highest inoculation rates\n(the number of sporozoites of P. falciparum transmitted\nby anopheles mosquitoes per person per year) observed,\nranged from 34 to 630 (van der Kolk et al.  2003, Appawu et\nal.  2004). For other important arthropod-borne infections,\nthese  rates  are  much  lower. Although  inoculation  rates\nare not directly comparable with the infestation rate used\nhere,  it  is  obvious  that  the  transmission  potential  of T.\npenetrans is similar to that of P. falciparum in a hyperen-\ndemic malaria area.\nAs indicated by an intra class correlation coefficient\nof 0.2 there was little variation in the infestation rate within\naffected  individuals  as  compared  to  variation  between\nindividuals. Stable infestation rates during the observa-\ntion period were confirmed by a similar proportion of vi-\nable lesions at the beginning and the end of the study.\nThere was a highly significant correlation between the\ninfestation  rate  and  the  infestation  intensity  during  the\nstudy period (rho = 0.70, P < 0.001). Thus, the infestation\nintensity can be used as a proxy for the infestation rate.\nThis  has  practical  consequences  as  the  assessment  of\nthe infestation rate is laborious, needs repeated examina-\ntions and can only be performed by an experienced health\nworker.  In  contrast,  counting  once  the  total  number  of\nlesions can be rapidly performed by an auxiliary, a com-\nmunity health worker or even by the affected individual.\nAs the infestation rate did not vary considerably in\nour  study  and  because  the  development  and  resolution\nof sand flea lesions follow a typical time pattern (Eisele et\nal.  2003), it seemed conceivable that the proportion of\nviable lesions (stage I to III) observed in an individual at\nany given point of time also reflects individual exposure.\nThis assumption was confirmed by a highly significant\ncorrelation between the infestation rate and the propor-\ntion of viable lesions.\nThe almost constant re-infestation during three weeks\nof  the  transmission  season  is  a  good  argument  against\nchemotherapy. Even if a highly effective compound was\nat hand, which actually is not the case (Heukelbach et al.\n2003a,  2004), it would have to be administered repeatedly\nin order to kill newly embedded sand fleas. By conse-\nquence,  in  order  to  keep  the  parasite  load  low  and  the\ndevelopment of morbidity at bay, the use of a repellent\nseems to be a more rational alternative (Schwalfenberg et\nal.  2004).\nOur  study  has  several  limitations. As  study  partici-\npants were selected according to a minimal number of\nlesions present at baseline, the selection procedure caused\na disproportionally high number of moderately to highly\ninfested individuals to be included in the study. There-\nfore, one cannot infer on the true distribution of infesta-\ntion rates in the community from these data. Moreover,\nthe  correlation  between  infestation  rate  and  infestation\nintensity might change, if many individuals with a low\ninfestation rate are included in the regression analysis.\nSecond,  we  measured  the  infestation  rate  during  a\nperiod of three weeks. The variation of this measure might\nbe higher if individuals are followed-up for a longer pe-\nriod of time.\nIn conclusion, our data show for the first time that it is\npossible  to  determine  the  degree  of  the  exposure  to  T.\npenetrans in individuals of an endemic community. The\ninfestation rate only varied little over time and predicted\nthe number of sand flea lesions present in an individual.",
         "",
         "High exposure to Tunga penetrans (Linnaeus, 1758) correlates with intensity of infestation.",
         "None"
        ],
        [
         "1",
         "0000e1833852eb3271356c493d59e83da0c4c7db",
         "With the burgeoning immunological data in the scientific literature, scientists must increasingly rely on Internet resources to inform and enhance their work. Here we provide a brief overview of the adaptive immune response and summaries of immunoinformatics resources, emphasizing those with Web interfaces. These resources include searchable databases of epitopes and immune-related molecules, and analysis tools for T cell and B cell epitope prediction, vaccine design, and protein structure comparisons. There is an agreeable synergy between the growing collections in immune-related databases and the growing sophistication of analysis software; the databases provide the foundation for developing predictive computational tools, which in turn enable more rapid identification of immune responses to populate the databases. Collectively, these resources contribute to improved understanding of immune responses and escape, and evolution of pathogens under immune pressure. The public health implications are vast, including designing vaccines, understanding autoimmune diseases, and defining the correlates of immune protection.",
         "B. Korber,M. LaBute,K. Yusim",
         "B. Korber,M. LaBute,K. Yusim",
         "https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.0020071&type=printable",
         "scientiﬁc literature, scientists must increasingly\nrely on Internet resources to inform and enhance\ntheir work. Here we provide a brief overview of the adaptive\nimmune response and summaries of immunoinformatics\nresources, emphasizing those with Web interfaces. These\nresources include searchable databases of epitopes and\nimmune-related molecules, and analysis tools for T cell and B\ncell epitope prediction, vaccine design, and protein structure\ncomparisons. There is an agreeable synergy between the\ngrowing collections in immune-related databases and the\ngrowing sophistication of analysis software; the databases\nprovide the foundation for developing predictive\ncomputational tools, which in turn enable more rapid\nidentiﬁcation of immune responses to populate the\ndatabases. Collectively, these resources contribute to\nimproved understanding of immune responses and escape,\nand evolution of pathogens under immune pressure. The\npublic health implications are vast, including designing\nvaccines, understanding autoimmune diseases, and deﬁning\nthe correlates of immune protection.",
         "Discussion\nThis review is intended as a portal to some of the most useful\nonline immunological software and searchable databases. This\nis a rapidly expanding area—experimental advances have\nmoved immunology into population-based studies and\nsimultaneously have brought us to the brink of\ncomprehensively characterizing an individual’s immune\nresponse to infection. Extensive listings of T cell epitopes and\nHLA-binding peptides, as well as peptides that do not bind,\nhave been an invaluable resource for motif resolution and\nepitope prediction. Epitope prediction in turn facilitates\ndetection of new epitopes, vaccine design, site-directed\nmutagenesis (to make proteins less immunogenic), potential\nautoantigen identiﬁcation, and the design of immune-based\ncancer therapies. Given the compelling nature of the problem\nand its suitability for computational methods, many scientists\nhave developed interesting alternative approaches to epitope\nprediction in silico, and have made their methods freely\navailable through the Web (Table 1). We applaud this effort,\nbut have the nagging concern that as the number of epitopes\ndeﬁned after an initial computational prediction prescreening\ngrows, the resulting sets of experimentally deﬁned epitopes\nmay bias subsequent predictors in ways that traditional\nprotein scanning with overlapping peptides would not.\nPromiscuous HLA presentation and epitope prediction\noffers one sensible strategy for the creation of T cell vaccines\nQuantitative binding data for B cell epitopes\nB cell epitope database\nAlign the nucleotide sequence of a rearranged V gene to\nthe closet V, D, and J segment germline counterparts\nBeta-version of biothreat pathogen B cell epitope database\nCalculates antigenic propensity scores based on\namino acid properties\nImmunoglobulin structures and annotated sequences\nA listing of haptens, structural similarity searches,\nand antibody and biological information\nB cell HIV epitopes, pathogen proteome linear epitope\nmaps, extensive literature citations regarding monoclonal\nantibodies, curated epitope alignments\nB cell HCV epitopes, pathogen proteome linear epitope\nmaps, extensive literature citations regarding monoclonal\nantibodies, curated epitope alignments\nThe most comprehensive listing of crystal structures\navailable, including antibodies and HLA, and TCRs\nSummary of antibody structures\n[69,73,74]. Alternatively, a rational epitope-informed peptide\nvaccine design can utilize the data in specialized pathogen-\nspeciﬁc databases to focus on epitopes with the most\nbiological promise to be beneﬁcial [94]. Finally, for a highly\nvariable pathogen, we are trying approaches intended to\nimprove the coverage of potential epitopes in the population,\nfor example by using a single consensus or ancestral sequence\n[95–97] or a computationally designed polyvalent vaccine that\nwill maximize epitope coverage [75].\nUnderstanding the impact of host immune-pathogen\ninteractions on pathogen evolution, pathogenesis, and\nimmunogen design depends on coordinated global efforts to\ngather and share data and requires the combined expertise of\nexperimental and computational scientists. Only through this\ntype of cooperation will we fully harvest the knowledge\nimplicit in the data. The computational tools presented here\nare not yet ready to supplant experiment, rather they should\nassist in experimental design and interpretation of data. We\nclearly do not know all of the rules yet, for instance in\npeptide-MHC binding, and key questions such as what\ndetermines immunodominance in T and B cell responses are\nstill unanswered. Yet the range and power of the tools already\navailable through the Internet, many representing global\nnetworks and collaboration, is a testimony to the substantial\nprogress we have made in facing emerging infectious diseases\nand potential biothreats with broader and deeper collective\nknowledge. &\nSupporting Information\nAccession Numbers\nThe Protein Data Bank (http://www.rcsb.org/pdb) accession number of\nHIV-1 p24 capsid protein is 2BVO.",
         "Introduction\nThe adaptive immune response. The immune system is the\nbody’s defense against infectious organisms and other foreign\nagents. The ﬁrst line of defense is innate immunity, rapid\nnonspeciﬁc responses that allow recognition of conserved\nsignature structures present in many microorganisms, such as\nlipopolysaccharides in bacterial cell walls or proteins in\nﬂagella [1]. The second line of defense is the adaptive immune\nresponse, tailored to an individual threat. An infected host\nmounts an immune response speciﬁc to an infectious agent;\nafter the infection is resolved, memory cells persist that\nenable a more rapid and potent response if the infectious\nagent is encountered again.\nThe adaptive immune response has two major arms: the\ncellular immune response of T lymphocytes, and the humoral\nimmune response of antibody-secreting B lymphocytes. In\nboth cases the immune response is stimulated by receptor\nrecognition of a speciﬁc small part of an antigen known as an\nepitope. Antibodies generally recognize intact proteins. B cell\nepitopes can be linear, contiguous amino acids, or they can be\ndiscontinuous amino acids that are brought together spatially\nin folded proteins. Discontinuous epitopes are deﬁned\nthrough mutagenesis, competition experiments, modeling, or\nthrough cocrystallization or modeling of protein structure\nand docking [2]. Even linear B cell epitopes are often\nconformation-dependent, and antibody-antigen interactions\nare improved when the epitope is displayed in the context of\nthe folded protein.\nIn contrast, T cell epitopes are short linear peptides that\nare cleaved from antigenic proteins, although T cell epitope\ngeneration by protein splicing is also observed [3]. T cell\nepitopes are presented in the context of major\nhistocompatibility complex (MHC) proteins, or, in case of\nhumans, human leukocyte antigen (HLA) class I or class II\nmolecules. Epitope presentation depends on both MHC-\npeptide binding and T cell receptor (TCR) interactions [4,5].\nMHC proteins are highly polymorphic, and each binds to a\nlimited set of peptides. Thus the particular combination of\nMHC alleles present in a host limits the range of potential\nepitopes recognized during an infection. The conformation\nof a T cell epitope embedded in an MHC protein is critical\nfor TCR recognition [6,7].\nTwo fundamental types of T cells are distinguished by\nexpression of CD8 and CD4 proteins, which dictate whether a\nT cell will recognize epitopes presented by class I or class II\nmolecules, respectively. Underlying this high-level bifurcation\nis a complex array of other functional markers. A key effector\nfunction of CD8þ T cells is cytolytic activity resulting in\napoptosis of virally infected cells [8], which depends upon the\nCD8þ T cell’s previous exposure to antigen and activation\nstate [9]. The primary function of CD4þ T cells is to produce\ncytokines that regulate the rest of the immune response.\nThese functions are not exclusive, however—CD4þ T cells can\ninduce cytolysis [10], and CD8þ T cells can secrete\nimmunoregulatory factors.\nCD4þ T cell epitopes are processed after encapsulation by\nantigen-presenting cells in membrane-bound vesicles, where\nthey are degraded by proteases into the peptide fragments\nthat bind to MHC class II proteins. Then they are delivered to\nthe cell surface, where class II-peptide complexes can be\nrecognized by the CD4þ TCRs [5]. In contrast, CD8þ T cells\ngenerally recognize viral or self antigens expressed from\nwithin a cell [11], proteins that are cleaved into short peptides\nin the cytosol by the immunoproteasome [12] at the C-\nterminal end of the peptide [13]. The N terminus is later\ntrimmed by proteases in endoplasmic reticulum [14]. After\ncleavage, peptides are translocated by the transporter\nassociated with antigen processing (TAP) into the\nendoplasmic reticulum for loading onto HLA class I\nmolecules [12,15], although other transport pathways can be\nused [16]. The MHC class I-peptide complex is then presented\non the cell surface, allowing recognition by epitope-speciﬁc\nTCRs on CD8þ T cells [5,12].\nBoth B cell and T cell epitopes are constrained by sequence\nspeciﬁcity, and mutations within and external to epitopes can\nresult in immune escape. Obviously, mutations within an\nepitope can directly impact antibody-antigen interactions or\nepitope-MHC and TCR interactions. Mutations outside of the\nepitope can inhibit antibody binding through\nconformational changes, or inhibit proper cleavage and\nprocessing of T cell epitopes [17,18]. TAP also binds peptides\nsomewhat selectively [19]. While there is a predilection for\ncertain peptides to be processed for MHC binding and\npresentation, processing steps must be general enough to\naccommodate a wide variety of potential epitopes so as to not\nexcessively constrain T cell immunity.\nPathogen- and cancer-related immune responses are being\ncharacterized at a remarkable pace, with precise mapping of\nwell-characterized epitopes and increasing use of full genetic\ntyping of HLA-epitope presenting molecules,\ncharacterization of accompanying crystal structures, and\ndeﬁnitions of escape mutations. As these elements are\ndeﬁned piece-by-piece in the literature, it becomes\nincreasingly valuable to assemble the data into searchable\ndatabases and to provide computational tools to assist in\ninterpretation of this complex information. Deﬁning epitope\nsequence speciﬁcity (including cleavage and transport signals\nand MHC binding) presents a tantalizing problem for\ncomputational biologists. The predictive amino acid patterns\nassociated with these events are subtle, requiring\nsophisticated pattern recognition methods to infer directly\nfrom protein sequences which peptides have the potential to\nbecome epitopes. The complexity is compounded by the fact\nthat recognition patterns might not be encoded by the\ncontiguous primary sequence, but rather in local three-\ndimensional structure. The response to this challenging\nproblem has resulted in an abundance of Web-based methods\nenabling the exploration of immunologically relevant data\nfrom a variety of perspectives. This review summarizes a\nsampling of particularly useful and user-friendly Web-based\ncomputational tools and searchable databases. The\ncomputational methods and databases are described and\nreferenced in the text, and Web links are provided in\nsummary tables. As a cautionary note, the authors have not\ndirectly tested that the functions contained in these resources\nwill produce meaningful results, nor have we done systematic\ncomparisons of the output of the different analysis tools;\nusers would beneﬁt by reading the primary literature\nregarding the different analyses methods if they decide to use\none or more in their own work.\nTools for predicting potential T cell epitopes in protein\nsequences. The most thoroughly studied step of T cell epitope\ngeneration is peptide binding to MHC molecules, and the\nWeb-based databases that include peptide-MHC data enable\nbinding predictions. The MHCPEP database [20], for\nexample, contains 13,000 MHC-binding peptides. Each entry\ncontains the peptide sequence, its MHC speciﬁcity and, when\navailable, experimental methods, observed activity, binding\nafﬁnity, source protein, anchor positions, and references.\nThis database, however, has been static since 1998. MHCBN\n[21] includes 18,790 MHC-binding peptides, 3,227 MHC-\nnonbinding peptides, 1,053 TAP binders and nonbinders, and\n6,548 T cell epitopes. A beta-version of the new Immune\nEpitope Database and Analysis Resource (IEDB) has recently\ncome online that will focus on epitopes in potential\nbioterrorism agents or emerging infectious diseases [22].\nMore databases are available, and some are discussed below\ntogether with relevant prediction tools.\nPeptide-MHC binding is the most predictable aspect of T\ncell epitope generation. MHC class I and class II genes are\nhighly polymorphic, and the majority of their variable\npositions are located in binding pockets that restrict peptide\ninteractions to those with particular amino acids at\ncharacteristic positions (Figure 1); the set of amino acids that\nare well tolerated in these binding pockets are called anchor\nmotifs. The search for epitopes in full-length proteins or\nwithin the context of a reactive peptide can be narrowed\nthrough a search for MHC-appropriate anchor motifs.\nPrimary HLA class I anchor positions are generally located at\nthe C terminus and a middle position of a peptide; as optimal\nepitope lengths vary between 8 and 12 amino acids long, the\nspacing between these two positions varies [23,24]. The ﬁrst\nMHC allele-speciﬁc motifs were deﬁned for murine class II\nmolecules [25]. Tracking anchor motifs patterns alone was\nsoon found to be of limited predictive value [26], while\nincluding more extensive binding patterns using quantitative\nmatrices representing the frequency and weight of every\namino acid in every position enabled the prediction of\nepitope locations in protein sequences with somewhat\ngreater [24,27–31], although still limited [32], accuracy.\nFor many MHC alleles, both simple and extended motifs\nare characterized and used to predict potential epitopes. For\nexample, the SYFPEITHI database [33] contains extensive\ninformation on MHC class I and class II anchor motifs and\nbinding speciﬁcity, and includes more than 4,500 entries of\nMHC proteins and aligned sequences of their epitopes and\nnatural ligands, with source proteins, organisms, and\npublication references for each peptide. The SYFPEITHI\nepitope prediction server [33] uses a frequency-based scoring\nsystem for every amino acid position within a peptide. The\nSYFPEITHI database allows, through examination of aligned\npeptides known to bind the HLA molecules, appreciation of\nthe relative level of conservation of anchor motifs, as well as\nthe number of peptides that bind despite imperfect motifs.\nThe Los Alamos HIV/HCV databases offer a simple tool\n(MotifScan) for identifying HLA anchor-binding motifs in\nquery proteins, highlighting them on a protein or protein\nalignment [34,35]. This tool is based on motif libraries\nincluded at the SYFPEITHI site, assembled by S. Marsh and\ncolleagues [23,24], and motifs extracted from the primary\nliterature. The more sophisticated MHC-peptide binding\nprediction approaches have generally been applied to limited\nnumbers of MHC proteins, so MotifScan provides a more\ncomprehensive, but less reliable, exploration of potential\nHLA-binding peptides. The input protein sequences can be\nautomatically uploaded from predeﬁned sets of HIV or HCV\nproteins, or the user can input any protein sequence or\nsequence alignment. MotifScan is taken one step further for\nHIV and HCV through the Epitope Location Finder (ELF)\n[36], where HLA anchor motifs are mapped onto proteins or\npeptides in conjunction with known epitopes taken from\nextensive database listings of class I HIV and HCV T cell\nepitopes and their presenting HLAs [37,38]. Currently the\nHIV CD8þ T cell epitope database contains 3,150 entries\ndescribing 1,600 distinct MHC class I-epitope combinations\n(a single epitope can have multiple entries); the HCV\ndatabase contains 510 entries describing 250 distinct MHC\nclass I–epitope combinations. These databases include\ndetailed biological information regarding the response to the\nepitope, including its impact on long term survival, common\nescape mutations, and whether an epitope is recognized in\nearly infection; links to the primary literature; and curated\nalignments summarizing the epitope’s global variability.\nA central assumption of the traditional prediction methods\nbased on motif frequencies is that each position contributes\nindependently to binding. Interactions at one site, however,\ncan affect interactions in another site [27,39]. Statistical\nclassiﬁers such as Hidden Markov Models have better success\nrates at MHC-binding predictions, and machine learning\nmethods such as artiﬁcial neural networks and support vector\nmachines can recognize nonlinear sequence-dependent\ncorrelated effects in MHC binding. Machine learning\nmethods as well as statistical methods are also useful for\ndeﬁning characteristic sequences related to TAP binding, and\nfor addressing the complexity of proteasome cleavage [40–\n47]. These methods, however, require large numbers of well-\ncharacterized peptides as training sets [32]. One comparative\nanalysis suggested that motifs gave the most accurate MHC-\nbinding predictions with limited data, but as the data\nincreases, machine learning methods become more reliable\npredictors [48]. In another comparative study, a support\nvector machine outperformed other methods [40]. Both\nmotif-based and machine learning methods for prediction of\ndifferent steps of T cell epitope generation are available\n(Table 1) [49], often offered in combination with databases of\nMHC-ligand interactions (Table 2). Below we discuss some of\nthe Web sites that are particularly helpful for T cell epitope\nprediction, many of which incorporate all three elements:\nimmunoproteasome cleavage, TAP binding, and MHC\nbinding.\nThe Edward Jenner Institute for Vaccine Research\nmaintains the AntiJen database, which contains quantitative\nexperimental binding data for peptides that bind to MHC,\nTAP, TCR-MHC complexes, T cell epitopes, and B cell\nepitopes; it also offers data on immunological protein-\nprotein interactions. It includes more than 24,000 entries.\nThe MHCPred [50,51] tool predicts the energetics of protein-\nligand interactions related to the free energy of binding, and\ntakes into account individual amino acids and contributions\nfrom side chain-side chain interactions, allowing peptide-\nMHC and peptide-TAP binding predictions. This site also\nallows the prediction of high afﬁnity peptides by comparing\nthe predicted binding afﬁnities of the original and the\nmutated peptides. PREDEPP [52,53] relies on the structural\nconservation and interactions observed in crystal structures\nof peptide-MHC complexes. A peptide’s compatibility for\nbinding is evaluated statistically by pairwise potentials. The\nWeb site also predicts proteasomal cleavage sites [54].\nThe BIMAS tool [31,55] ranks potential peptides based on a\npredicted half-time of disassociation from HLA class I\nmolecules, based on coefﬁcient tables deduced from the\npublished literature. The Max Planck Institute for Infection\nBiology offers MAPPP software [56] that combines either\nBIMAS or SYFPEITHI MHC-binding prediction with the\nproteasome cleavage software FRAGPREDICT [57].\nFRAGPREDICT predicts potential proteasomal cleavage sites\nbased on a combination of two algorithms. A statistical\nanalysis of cleavage-determining amino acid patterns is\nperformed [57], followed by predictions of major proteolytic\nfragments based on a kinetic model of the 20S proteasome\ndescribing the time-dependent digestion of smaller (up to 40\nresidues long) peptide substrates [58].\nThe following three suites of tools allow MHC/class I\nepitope prediction through a combination of cleavage\nprediction, TAP binding, and MHC binding. The Center for\nBiological Sequence Analysis offers the NetChop tool [44,59]\nfor predicting proteasomal or immunoproteasomal cleavage\nusing a nonlinear neural network, trained on in vitro\nexperimental cleavage data or MHC class I ligand data,\nrespectively. NetMHC [60–62] predicts binding of peptides to\nHLA supertypes (groups of HLA proteins that are likely to\ncross-present epitopes because of similarity in allowed\nbinding motifs) or to 120 individual HLA alleles, using\nCD8þ Tcell epitope prediction based on IEDB MHC class-I peptide binding\nprediction, proteasomal/immunoproteasomal cleavage, TAP binding\nAll predicted epitopes based on FRAGPREDICT and SYFPEITHI\nor BIMAS and predicted MHC-peptide binding propensity\nPredicted MHC/peptide or TAP/peptide IC50 binding values\nPrediction of promiscuous MHC class II binders\nPrediction of promiscuous MHC class I binders and prediction\nof mutations that will allow high affinity binding\nSummary and location of anchor motifs\nPredicted proteasome or immunoproteasome cleavage\nCTL/HLA-supertype epitope prediction, predicting NetMHC suptertype\nbinding, TAP binding, and NetChop proteasome cleavage scores\nPredicts MHC binding propensity of peptides\nHuman and yeast proteasome, or immunoproteasome cleavage\nPredicted proteasome or immunoproteasome cleavage\nPredicted MHC-peptide binding based on structure\nPredicted MHC class II-peptide binding (there are other\nrelated tools at the imtech Web site)\nPredicted MHC c lass I-peptide binding, optional proteasome/\nimmunoproteasome cleavage filter\nPredicted epitopes, binding motifs, epitope alignments for MHC proteins\nPrediction of binding affinity of TAP proteins\nPrediction of promiscuous class II epitopes\nFor the corresponding published references, please see the descriptions of the tools in the text. To use the imtech resources you must first register at: http://www.imtech.res.in/errors/\nnoauth.html.\naLast known update as of 02 May 2006. Empty cells indicate that updates were not readily available.\nbThese tools do not have a Web-based interface, but access may be requested by contacting the design companies.\nDOI: 10.1371/journal.pcbi.0020071.t001\nartiﬁcial neural networks. NetCTL [63,64] predicts epitopes\nby combining predictions of peptide-HLA-supertype binding\n(NetMHC), proteasomal C-terminal cleavage (NetChop), and\nTAP transport efﬁciency using a weight-matrix based method\n[65]. The Bioinformatics Centre Institute of Microbial\nTechnology has also developed a suite of servers [21,40–\n42,66,67] designed for predicting immunologically interesting\nfeatures in antigen sequences. ProPred1 and ProPred, along\nwith a series of related programs using different strategies,\npredicts speciﬁc MHC-binding peptides in proteins [67,68].\nPromiscuous binders can be predicted using a support vector\nmachine by MHC2Pred for MHC class II, or quantitative\nmatrices by MMBPred for MHC class I [69]. Pcleavage uses a\nsupport vector machine to predict proteasomal cleavage\nbased on in vitro data, or immunoproteasomal cleavage data\nbased on MHC class I ligand data [42]. TAPPred predicts\nbinding to TAP [41]. CTLpred predicts CTL epitopes in an\nantigen sequence by combining the processing and binding\nprediction methods [40]. IEDP also offers a suite of tools for T\ncell epitope prediction. Their peptide-MHC class I binding\nprediction tool allows the options of using an artiﬁcial neural\nnet, average relative binding [70], or a stabilized matrix\nmethod [71]. A comparison of the accuracy of these methods\nis underway by the IEDP team. These three methods also use\nthe average binding method for the prediction of MHC class\nII peptide binding [70]. Their MHC class I-peptide binding\nprediction can be combined with immunoproteasome\ncleavage [72] and TAP transport predictions [65], to predict\nMHC class I epitopes.\nMany of the sites listed are convenient for large-scale\ncalculations. Some, for example SYFPEITHI and MHCPred,\nallow one to incorporate multiple HLA alleles for epitope\nprediction, while others, such as NetChop, NetMHC, NetCTL,\nFRAGPREDICT, and IEDP tools allow one to upload protein\nalignments. MotifScan, MAPPP, and the ProPred series allow\nboth. These methods are currently being applied to peptide\nvaccine design and can be used to identify epitopes that have\nthe desirable properties of promiscuous presentation by\nmany HLAs and relative conservation [69,73,74]. We have\nrecently taken a very different approach to T cell vaccine\ndesign and developed a computational method for designing\npolyvalent protein cocktails that provide maximum peptide\nHLA frequencies in worldwide populations and\npolymorphism frequencies in immunologically.\nImportant alleles like KIR and different cytokines\nQuantitative binding data for MHC-ligand interactions,\nTCR-MHC complexes, TAP\nHLA alignments and tools for typing\nHLA anthropology database with many populations represented\nby genetic typing, and quick views of HLA frequencies\nCD8þ and CD4þ T cell HIV epitopes, proteome epitope maps\nCD8þ and CD4þ T cell HCV epitopes, proteome epitope maps\nBeta-version of biothreat pathogen T cell epitope database\nAligned and annotated HLA sequences following the\nWorld Health Organization nomenclature\nAligned and annotated T cell receptor sequences\nMHC-peptide binders and nonbinders, TAP-peptide\nbinders and nonbinders, T cell epitopes\nThe haplotype of MHC-linked-diseases, with complete\ngenomic sequences, variations (SNPs and DIPs) and\nancestral relationships.\nMHC-presented epitopes\nGenotype/serotype/supertype conversion tables\nHLA-specific primary anchor motifs\nStructural database and viewing tools, MHC/peptide/\nTCR combinations\nMHC-presented epitopes, MHC-specific anchor and auxiliary motifs\ncoverage (where peptides are set to a user-speciﬁed length,\nfor example nine amino acids) in a population of diverse\nproteins [75]. The mosaic proteins we create resemble real\nproteins, as they are assembled using a genetic algorithm by\nin silico homologous recombination of natural strains, and\nsets of mosaics are created based on the optimizing their\ncombined population coverage. While no Web interface has\nyet been built for this code, the two related programs are\nfreely available. One program enables an exploration of the\npeptide coverage in any set of natural proteins by a prototype\nvaccine strain or combinations of strains, while the other\ndesigns sets of mosaic proteins for a polyvalent vaccine that\nwill maximize population coverage. These tools could be\napplied to any variable pathogen for vaccine design, or used\nto design sets of reagents to probe the immune response.\nHLA-related databases and Web services. The number of\ngenetically deﬁned MHC and HLA alleles continues to\nexpand, with a corresponding evolving and expanding\nnomenclature. The European Bioinformatics Institute\nmaintains the IMGT/HLA sequence database [76], which\nincludes HLA allele listings as deﬁned in the World Health\nOrganization Nomenclature Committee Reports. The reports\ninclude previous designations, accession numbers, references,\nand information on the source of the allele. This Web site has\nsequences and alignments from HLA class I and II loci, from\nthe related MICA and MICB loci and from TAP1 and TAP2.\nTo ﬁnd Protein Data Bank (PDB) structures of MHC alleles in\ncomplex with peptides and/or the TCR domain, one easy\nmethod is to perform a BLAST search using the MHC alpha\nchain on PDB itself. There are about 100 available structures\nof MHCs in complex with peptides (mostly A alleles for MHC\nclass I), and 20 of MHC, peptide, and TCR complexes (mostly\ninvolving HLA A2-related alleles).\nThe National Center for Biotechnology Information (NCBI)\nmaintains dbMHC [77], which includes summaries of the\ngenetic organization of the HLA region, genetic sequence\nalignments, and tools for HLA typing. It also houses the HLA\nanthropology database, where individual allele and haplotype\nfrequencies can be retrieved from many different populations,\nnations, or geographic areas. The Allele Frequencies in\nWorldwide Populations project also offers summaries of HLA\nfrequencies, as well as polymorphisms in cytokines and KIR\nalleles. The Sanger MHC haplotype project offers information\non MHC related disease haplotypes, sequences,\npolymorphisms, and ancestral relationships [78,79].\nTools to assist the experimental T cell immunologist.\nExperimental T cell response mapping efforts recently have\nbeen scaling up, including additions of variant peptides to\nbetter probe responses to variable pathogens and extensions of\nT cell response mapping studies to span the full proteome of\npathogens for large study populations (for one example of a\npopulation study incorporating Elisot mapping of T cell\nresponses to HIV, see [80]). Complete datasets for several of\nthese large T cell peptide response studies for HIV are available\n(http://www.hiv.lanl.gov/content/immunology/hlatem/index.\nhtml). These efforts have led the HIV/HCV database team to\ndevelop computational tools to facilitate study design and\nanalyses of experimental data of this nature. These tools could,\nfor the most part, be applied to any pathogen or protein.\nPeptGen [81] enables a user to design overlapping peptide sets\nof any length and overlap, using a single sequence or an\nalignment if a variable pathogen is being studied and peptide\nvariants are desired. If an alignment is used, insertions or\ndeletions in the sequence are handled sensibly, and a ready-for-\nordering peptide list is created, organized so that identical\npeptides between need only be ordered once. If a population\nwith known HLA typing is screened, for example by EliSpot,\nHepitope allows a rapid search for HLAs that are enriched\namong people that react with each peptide in the study, and\nprovides anchor motif searches for the enriched HLAs. For\nHIV- or HCV-related studies, ELF [36] can be combined with\nHepitope to map previously described CD8þ T cell epitopes\nonto a reactive peptide.\nThere is growing interest in deﬁning and comparing HLA\nallele frequencies in study populations where vaccine trials\nare planned; thus we have made a suite of tools to compare\nHLA frequencies in two populations, to identify alleles in\nlinkage disequilibrium, and to ﬁll in estimates of missing HLA\ninformation if full genetic typing is not feasible (Note: we will\nadd a URL if the beta-version is ready in time, and delete this\nsection otherwise). Because of the high cost of genetic HLA\ntyping, although it is desirable, the reality is that often only\npartial HLA genetic typing of key alleles is available. A\npartially described data set could provide the basis for\ninformed guesses of the HLA genotypes superimposed onto\ntwo-digit typing—for example, by utilizing available four-\ndigit data genetic typing data at dbMHC for different\npopulations or a cohort subset that is fully genetically typed.\nThus we have created a computational tool in which four-\ndigit HLA allele designations are estimated from a\ncombination of two-digit and four-digit HLA typing data. A\nmaximum likelihood probability is assigned to each four-digit\nestimate, based on a combination of allele frequencies in the\npopulation and linkage disequilibrium patterns.\nTools for predicting B cell epitopes and related Internet\nresources. The conformational aspects of antibody binding\ncomplicates the problem of B cell epitope prediction, making\nit less tractable than T cell epitope prediction. Indeed, Blythe\nand Flower [82] recently undertook an exhaustive assessment\nof amino acid propensity scales using the AntiJen B cell\nepitope database, and even the best combinations performed\nonly marginally better than random [83]. If one wishes to\nexplore antigenic propensity using traditional methods,\nhowever, IEDB provides tools for predicting ﬁve features that\nhave been proposed to relate to B cell antigenicity, including\nbeta turn prediction [84], surface accessibility [85], ﬂexibility\n[86], and hydrophilicity [87]; it also includes an antigenicity\npredictor based on amino acid frequencies in antigenic\ndomains and chemistry [88]. An alternative strategy for\npredicting linear B cell epitopes, ABCpred, uses a neural\nnetwork trained and tested on the BCIPEP B cell epitope\ndatabase [66].\nAlthough antibody epitope prediction is difﬁcult, many\nother antibody-speciﬁc resources are available on the Web\n(Table 3). If the variable region sequence of a monoclonal\nantibody is obtained, ABcheck [89] enables a rapid crosscheck\nagainst the Kabat antibody database to identify unusual\nresidues that might be a sequencing artifact. (As a historical\naside, the Kabat database was an early immunological\ndatabase compiled to provide researchers with a\ncomprehensive comparison of antibody sequences. It was\navailable as a book long before the Internet enabled Web-\nsearchable molecular databases, at a time when GenBank, a\nresource that originated at Los Alamos National Laboratory,\nwas still in its early, groundbreaking stages. GenBank\neventually moved to the National Library of Medicine.\nSimilarly, the Los Alamos HIV database, the ﬁrst pathogen-\nspeciﬁc sequence database, was initially available only as a\nbook of aligned viral sequences.) The sequence could then be\nsubmitted to DNAPLOT, alignment software that enables\nrearranged V genes to be reliably assigned to their closest V,\nD, and J segment germline counterparts. The most\ncomprehensive data for crystallographic structures can be\nfound at the molecular modeling database (MMDB) [90],\nsummaries of antibody crystal structures are maintained at\nSACS [91], and both structures and alignments are available\nthrough the antibody group (ABG). The ImMunoGeneTics\n(IMGT) database provides annotated listings and alignments\nof both immunoglobulins and TCR binding regions [92,93] .\nWe maintain comprehensive Web-searchable databases of\npathogen-speciﬁc HIV [37] and HCV antibodies [38]. These\nare listings of monoclonal and polyclonal responses to the\nproteomes of these pathogens, including information\nregarding epitope location and variation, escape mutations,\nstructure, biological impact of antibody responses, keywords,\nand links to PubMed. The HIV database currently contains\n1,273, and the HCV database 120, unique antibody entries.\nAntibody entries are associated with multiple publications;\nfor some of the more intensively studied HIV neutralizing\nmonoclonal antibodies, more than 130 papers are cited, each\nwith a brief summary of what was learned about the speciﬁc\nantibody in that paper. It is difﬁcult to track a given\nmonoclonal antibody in the literature by other means, as\noften many antibodies are used in a single study so are not\nnamed in an abstract. To compound the problem, the name\nof a monoclonal antibody often ‘‘mutates’’ as it is exchanged\nbetween different labs, so is not readily searchable by\ntraditional means.",
         "",
         "",
         "downloading",
         "scientiﬁc literature, scientists must increasingly\nrely on Internet resources to inform and enhance\ntheir work. Here we provide a brief overview of the adaptive\nimmune response and summaries of immunoinformatics\nresources, emphasizing those with Web interfaces. These\nresources include searchable databases of epitopes and\nimmune-related molecules, and analysis tools for T cell and B\ncell epitope prediction, vaccine design, and protein structure\ncomparisons. There is an agreeable synergy between the\ngrowing collections in immune-related databases and the\ngrowing sophistication of analysis software; the databases\nprovide the foundation for developing predictive\ncomputational tools, which in turn enable more rapid\nidentiﬁcation of immune responses to populate the\ndatabases. Collectively, these resources contribute to\nimproved understanding of immune responses and escape,\nand evolution of pathogens under immune pressure. The\npublic health implications are vast, including designing\nvaccines, understanding autoimmune diseases, and deﬁning\nthe correlates of immune protection.\nIntroduction\nThe adaptive immune response. The immune system is the\nbody’s defense against infectious organisms and other foreign\nagents. The ﬁrst line of defense is innate immunity, rapid\nnonspeciﬁc responses that allow recognition of conserved\nsignature structures present in many microorganisms, such as\nlipopolysaccharides in bacterial cell walls or proteins in\nﬂagella [1]. The second line of defense is the adaptive immune\nresponse, tailored to an individual threat. An infected host\nmounts an immune response speciﬁc to an infectious agent;\nafter the infection is resolved, memory cells persist that\nenable a more rapid and potent response if the infectious\nagent is encountered again.\nThe adaptive immune response has two major arms: the\ncellular immune response of T lymphocytes, and the humoral\nimmune response of antibody-secreting B lymphocytes. In\nboth cases the immune response is stimulated by receptor\nrecognition of a speciﬁc small part of an antigen known as an\nepitope. Antibodies generally recognize intact proteins. B cell\nepitopes can be linear, contiguous amino acids, or they can be\ndiscontinuous amino acids that are brought together spatially\nin folded proteins. Discontinuous epitopes are deﬁned\nthrough mutagenesis, competition experiments, modeling, or\nthrough cocrystallization or modeling of protein structure\nand docking [2]. Even linear B cell epitopes are often\nconformation-dependent, and antibody-antigen interactions\nare improved when the epitope is displayed in the context of\nthe folded protein.\nIn contrast, T cell epitopes are short linear peptides that\nare cleaved from antigenic proteins, although T cell epitope\ngeneration by protein splicing is also observed [3]. T cell\nepitopes are presented in the context of major\nhistocompatibility complex (MHC) proteins, or, in case of\nhumans, human leukocyte antigen (HLA) class I or class II\nmolecules. Epitope presentation depends on both MHC-\npeptide binding and T cell receptor (TCR) interactions [4,5].\nMHC proteins are highly polymorphic, and each binds to a\nlimited set of peptides. Thus the particular combination of\nMHC alleles present in a host limits the range of potential\nepitopes recognized during an infection. The conformation\nof a T cell epitope embedded in an MHC protein is critical\nfor TCR recognition [6,7].\nTwo fundamental types of T cells are distinguished by\nexpression of CD8 and CD4 proteins, which dictate whether a\nT cell will recognize epitopes presented by class I or class II\nmolecules, respectively. Underlying this high-level bifurcation\nis a complex array of other functional markers. A key effector\nfunction of CD8þ T cells is cytolytic activity resulting in\napoptosis of virally infected cells [8], which depends upon the\nCD8þ T cell’s previous exposure to antigen and activation\nstate [9]. The primary function of CD4þ T cells is to produce\ncytokines that regulate the rest of the immune response.\nThese functions are not exclusive, however—CD4þ T cells can\ninduce cytolysis [10], and CD8þ T cells can secrete\nimmunoregulatory factors.\nCD4þ T cell epitopes are processed after encapsulation by\nantigen-presenting cells in membrane-bound vesicles, where\nthey are degraded by proteases into the peptide fragments\nthat bind to MHC class II proteins. Then they are delivered to\nthe cell surface, where class II-peptide complexes can be\nrecognized by the CD4þ TCRs [5]. In contrast, CD8þ T cells\ngenerally recognize viral or self antigens expressed from\nwithin a cell [11], proteins that are cleaved into short peptides\nin the cytosol by the immunoproteasome [12] at the C-\nterminal end of the peptide [13]. The N terminus is later\ntrimmed by proteases in endoplasmic reticulum [14]. After\ncleavage, peptides are translocated by the transporter\nassociated with antigen processing (TAP) into the\nendoplasmic reticulum for loading onto HLA class I\nmolecules [12,15], although other transport pathways can be\nused [16]. The MHC class I-peptide complex is then presented\non the cell surface, allowing recognition by epitope-speciﬁc\nTCRs on CD8þ T cells [5,12].\nBoth B cell and T cell epitopes are constrained by sequence\nspeciﬁcity, and mutations within and external to epitopes can\nresult in immune escape. Obviously, mutations within an\nepitope can directly impact antibody-antigen interactions or\nepitope-MHC and TCR interactions. Mutations outside of the\nepitope can inhibit antibody binding through\nconformational changes, or inhibit proper cleavage and\nprocessing of T cell epitopes [17,18]. TAP also binds peptides\nsomewhat selectively [19]. While there is a predilection for\ncertain peptides to be processed for MHC binding and\npresentation, processing steps must be general enough to\naccommodate a wide variety of potential epitopes so as to not\nexcessively constrain T cell immunity.\nPathogen- and cancer-related immune responses are being\ncharacterized at a remarkable pace, with precise mapping of\nwell-characterized epitopes and increasing use of full genetic\ntyping of HLA-epitope presenting molecules,\ncharacterization of accompanying crystal structures, and\ndeﬁnitions of escape mutations. As these elements are\ndeﬁned piece-by-piece in the literature, it becomes\nincreasingly valuable to assemble the data into searchable\ndatabases and to provide computational tools to assist in\ninterpretation of this complex information. Deﬁning epitope\nsequence speciﬁcity (including cleavage and transport signals\nand MHC binding) presents a tantalizing problem for\ncomputational biologists. The predictive amino acid patterns\nassociated with these events are subtle, requiring\nsophisticated pattern recognition methods to infer directly\nfrom protein sequences which peptides have the potential to\nbecome epitopes. The complexity is compounded by the fact\nthat recognition patterns might not be encoded by the\ncontiguous primary sequence, but rather in local three-\ndimensional structure. The response to this challenging\nproblem has resulted in an abundance of Web-based methods\nenabling the exploration of immunologically relevant data\nfrom a variety of perspectives. This review summarizes a\nsampling of particularly useful and user-friendly Web-based\ncomputational tools and searchable databases. The\ncomputational methods and databases are described and\nreferenced in the text, and Web links are provided in\nsummary tables. As a cautionary note, the authors have not\ndirectly tested that the functions contained in these resources\nwill produce meaningful results, nor have we done systematic\ncomparisons of the output of the different analysis tools;\nusers would beneﬁt by reading the primary literature\nregarding the different analyses methods if they decide to use\none or more in their own work.\nTools for predicting potential T cell epitopes in protein\nsequences. The most thoroughly studied step of T cell epitope\ngeneration is peptide binding to MHC molecules, and the\nWeb-based databases that include peptide-MHC data enable\nbinding predictions. The MHCPEP database [20], for\nexample, contains 13,000 MHC-binding peptides. Each entry\ncontains the peptide sequence, its MHC speciﬁcity and, when\navailable, experimental methods, observed activity, binding\nafﬁnity, source protein, anchor positions, and references.\nThis database, however, has been static since 1998. MHCBN\n[21] includes 18,790 MHC-binding peptides, 3,227 MHC-\nnonbinding peptides, 1,053 TAP binders and nonbinders, and\n6,548 T cell epitopes. A beta-version of the new Immune\nEpitope Database and Analysis Resource (IEDB) has recently\ncome online that will focus on epitopes in potential\nbioterrorism agents or emerging infectious diseases [22].\nMore databases are available, and some are discussed below\ntogether with relevant prediction tools.\nPeptide-MHC binding is the most predictable aspect of T\ncell epitope generation. MHC class I and class II genes are\nhighly polymorphic, and the majority of their variable\npositions are located in binding pockets that restrict peptide\ninteractions to those with particular amino acids at\ncharacteristic positions (Figure 1); the set of amino acids that\nare well tolerated in these binding pockets are called anchor\nmotifs. The search for epitopes in full-length proteins or\nwithin the context of a reactive peptide can be narrowed\nthrough a search for MHC-appropriate anchor motifs.\nPrimary HLA class I anchor positions are generally located at\nthe C terminus and a middle position of a peptide; as optimal\nepitope lengths vary between 8 and 12 amino acids long, the\nspacing between these two positions varies [23,24]. The ﬁrst\nMHC allele-speciﬁc motifs were deﬁned for murine class II\nmolecules [25]. Tracking anchor motifs patterns alone was\nsoon found to be of limited predictive value [26], while\nincluding more extensive binding patterns using quantitative\nmatrices representing the frequency and weight of every\namino acid in every position enabled the prediction of\nepitope locations in protein sequences with somewhat\ngreater [24,27–31], although still limited [32], accuracy.\nFor many MHC alleles, both simple and extended motifs\nare characterized and used to predict potential epitopes. For\nexample, the SYFPEITHI database [33] contains extensive\ninformation on MHC class I and class II anchor motifs and\nbinding speciﬁcity, and includes more than 4,500 entries of\nMHC proteins and aligned sequences of their epitopes and\nnatural ligands, with source proteins, organisms, and\npublication references for each peptide. The SYFPEITHI\nepitope prediction server [33] uses a frequency-based scoring\nsystem for every amino acid position within a peptide. The\nSYFPEITHI database allows, through examination of aligned\npeptides known to bind the HLA molecules, appreciation of\nthe relative level of conservation of anchor motifs, as well as\nthe number of peptides that bind despite imperfect motifs.\nThe Los Alamos HIV/HCV databases offer a simple tool\n(MotifScan) for identifying HLA anchor-binding motifs in\nquery proteins, highlighting them on a protein or protein\nalignment [34,35]. This tool is based on motif libraries\nincluded at the SYFPEITHI site, assembled by S. Marsh and\ncolleagues [23,24], and motifs extracted from the primary\nliterature. The more sophisticated MHC-peptide binding\nprediction approaches have generally been applied to limited\nnumbers of MHC proteins, so MotifScan provides a more\ncomprehensive, but less reliable, exploration of potential\nHLA-binding peptides. The input protein sequences can be\nautomatically uploaded from predeﬁned sets of HIV or HCV\nproteins, or the user can input any protein sequence or\nsequence alignment. MotifScan is taken one step further for\nHIV and HCV through the Epitope Location Finder (ELF)\n[36], where HLA anchor motifs are mapped onto proteins or\npeptides in conjunction with known epitopes taken from\nextensive database listings of class I HIV and HCV T cell\nepitopes and their presenting HLAs [37,38]. Currently the\nHIV CD8þ T cell epitope database contains 3,150 entries\ndescribing 1,600 distinct MHC class I-epitope combinations\n(a single epitope can have multiple entries); the HCV\ndatabase contains 510 entries describing 250 distinct MHC\nclass I–epitope combinations. These databases include\ndetailed biological information regarding the response to the\nepitope, including its impact on long term survival, common\nescape mutations, and whether an epitope is recognized in\nearly infection; links to the primary literature; and curated\nalignments summarizing the epitope’s global variability.\nA central assumption of the traditional prediction methods\nbased on motif frequencies is that each position contributes\nindependently to binding. Interactions at one site, however,\ncan affect interactions in another site [27,39]. Statistical\nclassiﬁers such as Hidden Markov Models have better success\nrates at MHC-binding predictions, and machine learning\nmethods such as artiﬁcial neural networks and support vector\nmachines can recognize nonlinear sequence-dependent\ncorrelated effects in MHC binding. Machine learning\nmethods as well as statistical methods are also useful for\ndeﬁning characteristic sequences related to TAP binding, and\nfor addressing the complexity of proteasome cleavage [40–\n47]. These methods, however, require large numbers of well-\ncharacterized peptides as training sets [32]. One comparative\nanalysis suggested that motifs gave the most accurate MHC-\nbinding predictions with limited data, but as the data\nincreases, machine learning methods become more reliable\npredictors [48]. In another comparative study, a support\nvector machine outperformed other methods [40]. Both\nmotif-based and machine learning methods for prediction of\ndifferent steps of T cell epitope generation are available\n(Table 1) [49], often offered in combination with databases of\nMHC-ligand interactions (Table 2). Below we discuss some of\nthe Web sites that are particularly helpful for T cell epitope\nprediction, many of which incorporate all three elements:\nimmunoproteasome cleavage, TAP binding, and MHC\nbinding.\nThe Edward Jenner Institute for Vaccine Research\nmaintains the AntiJen database, which contains quantitative\nexperimental binding data for peptides that bind to MHC,\nTAP, TCR-MHC complexes, T cell epitopes, and B cell\nepitopes; it also offers data on immunological protein-\nprotein interactions. It includes more than 24,000 entries.\nThe MHCPred [50,51] tool predicts the energetics of protein-\nligand interactions related to the free energy of binding, and\ntakes into account individual amino acids and contributions\nfrom side chain-side chain interactions, allowing peptide-\nMHC and peptide-TAP binding predictions. This site also\nallows the prediction of high afﬁnity peptides by comparing\nthe predicted binding afﬁnities of the original and the\nmutated peptides. PREDEPP [52,53] relies on the structural\nconservation and interactions observed in crystal structures\nof peptide-MHC complexes. A peptide’s compatibility for\nbinding is evaluated statistically by pairwise potentials. The\nWeb site also predicts proteasomal cleavage sites [54].\nThe BIMAS tool [31,55] ranks potential peptides based on a\npredicted half-time of disassociation from HLA class I\nmolecules, based on coefﬁcient tables deduced from the\npublished literature. The Max Planck Institute for Infection\nBiology offers MAPPP software [56] that combines either\nBIMAS or SYFPEITHI MHC-binding prediction with the\nproteasome cleavage software FRAGPREDICT [57].\nFRAGPREDICT predicts potential proteasomal cleavage sites\nbased on a combination of two algorithms. A statistical\nanalysis of cleavage-determining amino acid patterns is\nperformed [57], followed by predictions of major proteolytic\nfragments based on a kinetic model of the 20S proteasome\ndescribing the time-dependent digestion of smaller (up to 40\nresidues long) peptide substrates [58].\nThe following three suites of tools allow MHC/class I\nepitope prediction through a combination of cleavage\nprediction, TAP binding, and MHC binding. The Center for\nBiological Sequence Analysis offers the NetChop tool [44,59]\nfor predicting proteasomal or immunoproteasomal cleavage\nusing a nonlinear neural network, trained on in vitro\nexperimental cleavage data or MHC class I ligand data,\nrespectively. NetMHC [60–62] predicts binding of peptides to\nHLA supertypes (groups of HLA proteins that are likely to\ncross-present epitopes because of similarity in allowed\nbinding motifs) or to 120 individual HLA alleles, using\nCD8þ Tcell epitope prediction based on IEDB MHC class-I peptide binding\nprediction, proteasomal/immunoproteasomal cleavage, TAP binding\nAll predicted epitopes based on FRAGPREDICT and SYFPEITHI\nor BIMAS and predicted MHC-peptide binding propensity\nPredicted MHC/peptide or TAP/peptide IC50 binding values\nPrediction of promiscuous MHC class II binders\nPrediction of promiscuous MHC class I binders and prediction\nof mutations that will allow high affinity binding\nSummary and location of anchor motifs\nPredicted proteasome or immunoproteasome cleavage\nCTL/HLA-supertype epitope prediction, predicting NetMHC suptertype\nbinding, TAP binding, and NetChop proteasome cleavage scores\nPredicts MHC binding propensity of peptides\nHuman and yeast proteasome, or immunoproteasome cleavage\nPredicted proteasome or immunoproteasome cleavage\nPredicted MHC-peptide binding based on structure\nPredicted MHC class II-peptide binding (there are other\nrelated tools at the imtech Web site)\nPredicted MHC c lass I-peptide binding, optional proteasome/\nimmunoproteasome cleavage filter\nPredicted epitopes, binding motifs, epitope alignments for MHC proteins\nPrediction of binding affinity of TAP proteins\nPrediction of promiscuous class II epitopes\nFor the corresponding published references, please see the descriptions of the tools in the text. To use the imtech resources you must first register at: http://www.imtech.res.in/errors/\nnoauth.html.\naLast known update as of 02 May 2006. Empty cells indicate that updates were not readily available.\nbThese tools do not have a Web-based interface, but access may be requested by contacting the design companies.\nDOI: 10.1371/journal.pcbi.0020071.t001\nartiﬁcial neural networks. NetCTL [63,64] predicts epitopes\nby combining predictions of peptide-HLA-supertype binding\n(NetMHC), proteasomal C-terminal cleavage (NetChop), and\nTAP transport efﬁciency using a weight-matrix based method\n[65]. The Bioinformatics Centre Institute of Microbial\nTechnology has also developed a suite of servers [21,40–\n42,66,67] designed for predicting immunologically interesting\nfeatures in antigen sequences. ProPred1 and ProPred, along\nwith a series of related programs using different strategies,\npredicts speciﬁc MHC-binding peptides in proteins [67,68].\nPromiscuous binders can be predicted using a support vector\nmachine by MHC2Pred for MHC class II, or quantitative\nmatrices by MMBPred for MHC class I [69]. Pcleavage uses a\nsupport vector machine to predict proteasomal cleavage\nbased on in vitro data, or immunoproteasomal cleavage data\nbased on MHC class I ligand data [42]. TAPPred predicts\nbinding to TAP [41]. CTLpred predicts CTL epitopes in an\nantigen sequence by combining the processing and binding\nprediction methods [40]. IEDP also offers a suite of tools for T\ncell epitope prediction. Their peptide-MHC class I binding\nprediction tool allows the options of using an artiﬁcial neural\nnet, average relative binding [70], or a stabilized matrix\nmethod [71]. A comparison of the accuracy of these methods\nis underway by the IEDP team. These three methods also use\nthe average binding method for the prediction of MHC class\nII peptide binding [70]. Their MHC class I-peptide binding\nprediction can be combined with immunoproteasome\ncleavage [72] and TAP transport predictions [65], to predict\nMHC class I epitopes.\nMany of the sites listed are convenient for large-scale\ncalculations. Some, for example SYFPEITHI and MHCPred,\nallow one to incorporate multiple HLA alleles for epitope\nprediction, while others, such as NetChop, NetMHC, NetCTL,\nFRAGPREDICT, and IEDP tools allow one to upload protein\nalignments. MotifScan, MAPPP, and the ProPred series allow\nboth. These methods are currently being applied to peptide\nvaccine design and can be used to identify epitopes that have\nthe desirable properties of promiscuous presentation by\nmany HLAs and relative conservation [69,73,74]. We have\nrecently taken a very different approach to T cell vaccine\ndesign and developed a computational method for designing\npolyvalent protein cocktails that provide maximum peptide\nHLA frequencies in worldwide populations and\npolymorphism frequencies in immunologically.\nImportant alleles like KIR and different cytokines\nQuantitative binding data for MHC-ligand interactions,\nTCR-MHC complexes, TAP\nHLA alignments and tools for typing\nHLA anthropology database with many populations represented\nby genetic typing, and quick views of HLA frequencies\nCD8þ and CD4þ T cell HIV epitopes, proteome epitope maps\nCD8þ and CD4þ T cell HCV epitopes, proteome epitope maps\nBeta-version of biothreat pathogen T cell epitope database\nAligned and annotated HLA sequences following the\nWorld Health Organization nomenclature\nAligned and annotated T cell receptor sequences\nMHC-peptide binders and nonbinders, TAP-peptide\nbinders and nonbinders, T cell epitopes\nThe haplotype of MHC-linked-diseases, with complete\ngenomic sequences, variations (SNPs and DIPs) and\nancestral relationships.\nMHC-presented epitopes\nGenotype/serotype/supertype conversion tables\nHLA-specific primary anchor motifs\nStructural database and viewing tools, MHC/peptide/\nTCR combinations\nMHC-presented epitopes, MHC-specific anchor and auxiliary motifs\ncoverage (where peptides are set to a user-speciﬁed length,\nfor example nine amino acids) in a population of diverse\nproteins [75]. The mosaic proteins we create resemble real\nproteins, as they are assembled using a genetic algorithm by\nin silico homologous recombination of natural strains, and\nsets of mosaics are created based on the optimizing their\ncombined population coverage. While no Web interface has\nyet been built for this code, the two related programs are\nfreely available. One program enables an exploration of the\npeptide coverage in any set of natural proteins by a prototype\nvaccine strain or combinations of strains, while the other\ndesigns sets of mosaic proteins for a polyvalent vaccine that\nwill maximize population coverage. These tools could be\napplied to any variable pathogen for vaccine design, or used\nto design sets of reagents to probe the immune response.\nHLA-related databases and Web services. The number of\ngenetically deﬁned MHC and HLA alleles continues to\nexpand, with a corresponding evolving and expanding\nnomenclature. The European Bioinformatics Institute\nmaintains the IMGT/HLA sequence database [76], which\nincludes HLA allele listings as deﬁned in the World Health\nOrganization Nomenclature Committee Reports. The reports\ninclude previous designations, accession numbers, references,\nand information on the source of the allele. This Web site has\nsequences and alignments from HLA class I and II loci, from\nthe related MICA and MICB loci and from TAP1 and TAP2.\nTo ﬁnd Protein Data Bank (PDB) structures of MHC alleles in\ncomplex with peptides and/or the TCR domain, one easy\nmethod is to perform a BLAST search using the MHC alpha\nchain on PDB itself. There are about 100 available structures\nof MHCs in complex with peptides (mostly A alleles for MHC\nclass I), and 20 of MHC, peptide, and TCR complexes (mostly\ninvolving HLA A2-related alleles).\nThe National Center for Biotechnology Information (NCBI)\nmaintains dbMHC [77], which includes summaries of the\ngenetic organization of the HLA region, genetic sequence\nalignments, and tools for HLA typing. It also houses the HLA\nanthropology database, where individual allele and haplotype\nfrequencies can be retrieved from many different populations,\nnations, or geographic areas. The Allele Frequencies in\nWorldwide Populations project also offers summaries of HLA\nfrequencies, as well as polymorphisms in cytokines and KIR\nalleles. The Sanger MHC haplotype project offers information\non MHC related disease haplotypes, sequences,\npolymorphisms, and ancestral relationships [78,79].\nTools to assist the experimental T cell immunologist.\nExperimental T cell response mapping efforts recently have\nbeen scaling up, including additions of variant peptides to\nbetter probe responses to variable pathogens and extensions of\nT cell response mapping studies to span the full proteome of\npathogens for large study populations (for one example of a\npopulation study incorporating Elisot mapping of T cell\nresponses to HIV, see [80]). Complete datasets for several of\nthese large T cell peptide response studies for HIV are available\n(http://www.hiv.lanl.gov/content/immunology/hlatem/index.\nhtml). These efforts have led the HIV/HCV database team to\ndevelop computational tools to facilitate study design and\nanalyses of experimental data of this nature. These tools could,\nfor the most part, be applied to any pathogen or protein.\nPeptGen [81] enables a user to design overlapping peptide sets\nof any length and overlap, using a single sequence or an\nalignment if a variable pathogen is being studied and peptide\nvariants are desired. If an alignment is used, insertions or\ndeletions in the sequence are handled sensibly, and a ready-for-\nordering peptide list is created, organized so that identical\npeptides between need only be ordered once. If a population\nwith known HLA typing is screened, for example by EliSpot,\nHepitope allows a rapid search for HLAs that are enriched\namong people that react with each peptide in the study, and\nprovides anchor motif searches for the enriched HLAs. For\nHIV- or HCV-related studies, ELF [36] can be combined with\nHepitope to map previously described CD8þ T cell epitopes\nonto a reactive peptide.\nThere is growing interest in deﬁning and comparing HLA\nallele frequencies in study populations where vaccine trials\nare planned; thus we have made a suite of tools to compare\nHLA frequencies in two populations, to identify alleles in\nlinkage disequilibrium, and to ﬁll in estimates of missing HLA\ninformation if full genetic typing is not feasible (Note: we will\nadd a URL if the beta-version is ready in time, and delete this\nsection otherwise). Because of the high cost of genetic HLA\ntyping, although it is desirable, the reality is that often only\npartial HLA genetic typing of key alleles is available. A\npartially described data set could provide the basis for\ninformed guesses of the HLA genotypes superimposed onto\ntwo-digit typing—for example, by utilizing available four-\ndigit data genetic typing data at dbMHC for different\npopulations or a cohort subset that is fully genetically typed.\nThus we have created a computational tool in which four-\ndigit HLA allele designations are estimated from a\ncombination of two-digit and four-digit HLA typing data. A\nmaximum likelihood probability is assigned to each four-digit\nestimate, based on a combination of allele frequencies in the\npopulation and linkage disequilibrium patterns.\nTools for predicting B cell epitopes and related Internet\nresources. The conformational aspects of antibody binding\ncomplicates the problem of B cell epitope prediction, making\nit less tractable than T cell epitope prediction. Indeed, Blythe\nand Flower [82] recently undertook an exhaustive assessment\nof amino acid propensity scales using the AntiJen B cell\nepitope database, and even the best combinations performed\nonly marginally better than random [83]. If one wishes to\nexplore antigenic propensity using traditional methods,\nhowever, IEDB provides tools for predicting ﬁve features that\nhave been proposed to relate to B cell antigenicity, including\nbeta turn prediction [84], surface accessibility [85], ﬂexibility\n[86], and hydrophilicity [87]; it also includes an antigenicity\npredictor based on amino acid frequencies in antigenic\ndomains and chemistry [88]. An alternative strategy for\npredicting linear B cell epitopes, ABCpred, uses a neural\nnetwork trained and tested on the BCIPEP B cell epitope\ndatabase [66].\nAlthough antibody epitope prediction is difﬁcult, many\nother antibody-speciﬁc resources are available on the Web\n(Table 3). If the variable region sequence of a monoclonal\nantibody is obtained, ABcheck [89] enables a rapid crosscheck\nagainst the Kabat antibody database to identify unusual\nresidues that might be a sequencing artifact. (As a historical\naside, the Kabat database was an early immunological\ndatabase compiled to provide researchers with a\ncomprehensive comparison of antibody sequences. It was\navailable as a book long before the Internet enabled Web-\nsearchable molecular databases, at a time when GenBank, a\nresource that originated at Los Alamos National Laboratory,\nwas still in its early, groundbreaking stages. GenBank\neventually moved to the National Library of Medicine.\nSimilarly, the Los Alamos HIV database, the ﬁrst pathogen-\nspeciﬁc sequence database, was initially available only as a\nbook of aligned viral sequences.) The sequence could then be\nsubmitted to DNAPLOT, alignment software that enables\nrearranged V genes to be reliably assigned to their closest V,\nD, and J segment germline counterparts. The most\ncomprehensive data for crystallographic structures can be\nfound at the molecular modeling database (MMDB) [90],\nsummaries of antibody crystal structures are maintained at\nSACS [91], and both structures and alignments are available\nthrough the antibody group (ABG). The ImMunoGeneTics\n(IMGT) database provides annotated listings and alignments\nof both immunoglobulins and TCR binding regions [92,93] .\nWe maintain comprehensive Web-searchable databases of\npathogen-speciﬁc HIV [37] and HCV antibodies [38]. These\nare listings of monoclonal and polyclonal responses to the\nproteomes of these pathogens, including information\nregarding epitope location and variation, escape mutations,\nstructure, biological impact of antibody responses, keywords,\nand links to PubMed. The HIV database currently contains\n1,273, and the HCV database 120, unique antibody entries.\nAntibody entries are associated with multiple publications;\nfor some of the more intensively studied HIV neutralizing\nmonoclonal antibodies, more than 130 papers are cited, each\nwith a brief summary of what was learned about the speciﬁc\nantibody in that paper. It is difﬁcult to track a given\nmonoclonal antibody in the literature by other means, as\noften many antibodies are used in a single study so are not\nnamed in an abstract. To compound the problem, the name\nof a monoclonal antibody often ‘‘mutates’’ as it is exchanged\nbetween different labs, so is not readily searchable by\ntraditional means.\nDiscussion\nThis review is intended as a portal to some of the most useful\nonline immunological software and searchable databases. This\nis a rapidly expanding area—experimental advances have\nmoved immunology into population-based studies and\nsimultaneously have brought us to the brink of\ncomprehensively characterizing an individual’s immune\nresponse to infection. Extensive listings of T cell epitopes and\nHLA-binding peptides, as well as peptides that do not bind,\nhave been an invaluable resource for motif resolution and\nepitope prediction. Epitope prediction in turn facilitates\ndetection of new epitopes, vaccine design, site-directed\nmutagenesis (to make proteins less immunogenic), potential\nautoantigen identiﬁcation, and the design of immune-based\ncancer therapies. Given the compelling nature of the problem\nand its suitability for computational methods, many scientists\nhave developed interesting alternative approaches to epitope\nprediction in silico, and have made their methods freely\navailable through the Web (Table 1). We applaud this effort,\nbut have the nagging concern that as the number of epitopes\ndeﬁned after an initial computational prediction prescreening\ngrows, the resulting sets of experimentally deﬁned epitopes\nmay bias subsequent predictors in ways that traditional\nprotein scanning with overlapping peptides would not.\nPromiscuous HLA presentation and epitope prediction\noffers one sensible strategy for the creation of T cell vaccines\nQuantitative binding data for B cell epitopes\nB cell epitope database\nAlign the nucleotide sequence of a rearranged V gene to\nthe closet V, D, and J segment germline counterparts\nBeta-version of biothreat pathogen B cell epitope database\nCalculates antigenic propensity scores based on\namino acid properties\nImmunoglobulin structures and annotated sequences\nA listing of haptens, structural similarity searches,\nand antibody and biological information\nB cell HIV epitopes, pathogen proteome linear epitope\nmaps, extensive literature citations regarding monoclonal\nantibodies, curated epitope alignments\nB cell HCV epitopes, pathogen proteome linear epitope\nmaps, extensive literature citations regarding monoclonal\nantibodies, curated epitope alignments\nThe most comprehensive listing of crystal structures\navailable, including antibodies and HLA, and TCRs\nSummary of antibody structures\n[69,73,74]. Alternatively, a rational epitope-informed peptide\nvaccine design can utilize the data in specialized pathogen-\nspeciﬁc databases to focus on epitopes with the most\nbiological promise to be beneﬁcial [94]. Finally, for a highly\nvariable pathogen, we are trying approaches intended to\nimprove the coverage of potential epitopes in the population,\nfor example by using a single consensus or ancestral sequence\n[95–97] or a computationally designed polyvalent vaccine that\nwill maximize epitope coverage [75].\nUnderstanding the impact of host immune-pathogen\ninteractions on pathogen evolution, pathogenesis, and\nimmunogen design depends on coordinated global efforts to\ngather and share data and requires the combined expertise of\nexperimental and computational scientists. Only through this\ntype of cooperation will we fully harvest the knowledge\nimplicit in the data. The computational tools presented here\nare not yet ready to supplant experiment, rather they should\nassist in experimental design and interpretation of data. We\nclearly do not know all of the rules yet, for instance in\npeptide-MHC binding, and key questions such as what\ndetermines immunodominance in T and B cell responses are\nstill unanswered. Yet the range and power of the tools already\navailable through the Internet, many representing global\nnetworks and collaboration, is a testimony to the substantial\nprogress we have made in facing emerging infectious diseases\nand potential biothreats with broader and deeper collective\nknowledge. &\nSupporting Information\nAccession Numbers\nThe Protein Data Bank (http://www.rcsb.org/pdb) accession number of\nHIV-1 p24 capsid protein is 2BVO.",
         "Prediction of the secondary structure of proteins from their amino acid sequence.;How TCRs bind MHCs, peptides, and coreceptors.;Genetic Analysis of Completely Sequenced Disease-Associated MHC Haplotypes Identifies Shuffling of Segments in Recent Human History;Human Immunodeficiency Virus Type 1 Subtype B Ancestral Envelope Protein Is Functional and Elicits Neutralizing Antibodies in Rabbits Similar to Those Elicited by a Circulating Subtype B Envelope;Pcleavage: an SVM based method for prediction of constitutive proteasome and immunoproteasome cleavage sites in antigenic sequences;Generating quantitative models describing the sequence specificity of biological processes with the stabilized matrix method;Bcipep: A database of B-cell epitopes;Automated generation and evaluation of specific MHC binding predictive tools: ARB matrix applications;Modeling the MHC class I pathway by combining predictions of proteasomal cleavage,TAP transport and MHC class I binding;HIV vaccine development by computer assisted design: the GAIA vaccine.;The role of the proteasome in generating cytotoxic T-cell epitopes: insights obtained from improved predictions of proteasomal cleavage;Sensor molecules in intestinal innate immunity against bacterial infections;The Immune Epitope Database and Analysis Resource: From Vision to Blueprint;Antigenicity and Immunogenicity of a Synthetic Human Immunodeficiency Virus Type 1 Group M Consensus Envelope Glycoprotein;Benchmarking B cell epitope prediction: Underperformance of existing methods;IMGT, the international ImMunoGeneTics information system®;Dominant influence of HLA-B in mediating the potential co-evolution of HIV and HLA;Computational methods for prediction of T-cell epitopes--a framework for modelling, testing, and applications.;Prediction of CTL epitopes using QM, SVM and ANN techniques.;Selection, Transmission, and Reversion of an Antigen-Processing Cytotoxic T-Lymphocyte Escape Mutation in Human Immunodeficiency Virus Type 1 Infection;Improved prediction of MHC class I and class II epitopes using a novel Gibbs sampling approach;Immediate Cytotoxicity But Not Degranulation Distinguishes Effector and Memory Subsets of CD8+ T Cells;Analysis and prediction of affinity of TAP binding peptides using cascade SVM;Definition of supertypes for HLA molecules using clustering of specificity matrices;Immune recognition of a human renal cancer antigen through post-translational protein splicing;Sensitive quantitative predictions of peptide-MHC binding by a 'Query by Committee' artificial neural network approach.;Mapping cross-clade HIV-1 vaccine epitopes using a bioinformatics approach.;Identifying MHC Class I Epitopes by Predicting the TAP Transport Efficiency of Epitope Precursors;Prediction of promiscuous and high-affinity mutated MHC binders.;MHCPred: a server for quantitative prediction of peptide-MHC binding;JenPep: A Novel Computational Information Resource for Immunobiology and Vaccinology;ProPred1: Prediction of Promiscuous MHC Class-I Binding Sites;MHCBN: a comprehensive database of MHC binding and non-binding peptides;Diversity Considerations in HIV-1 Vaccine Selection;Large‐scale computational identification of HIV T‐cell epitopes;The MHC haplotype project: a resource for HLA-linked association studies.;Development of a DNA-MVA/HIVA vaccine for Kenya.;Prediction of proteasome cleavage motifs by neural networks.;ProPred: prediction of HLA-DR binding sites;From genome to vaccine: in silico predictions, ex vivo verification.;Crystal Structure of a Neutralizing Human IgG Against HIV-1: A Template for Vaccine Design;Functional Evidence That Conserved TCR CDRα3 Loop Docking Governs the Cross-Recognition of Closely Related Peptide:Class I Complexes1;Quantitative predictions of peptide binding to MHC class I molecules using specificity matrices and anchor-stratified calibrations.;A kinetic model of vertebrate 20S proteasome accounting for the generation of major proteolytic fragments from oligomeric peptide substrates.;Sequence signals for generation of antigenic peptides by the proteasome: implications for proteasomal cleavage mechanism.;The HLA FactsBook;Function of the transport complex TAP in cellular immune recognition.;The specificity of proteasomes: impact on MHC class I processing and presentation of antigens;MHC superfamily structure and the immune system.;Distinct proteolytic processes generate the C and N termini of MHC class I-binding peptides.;Nine major HLA class I supertypes account for the vast preponderance of HLA-A and -B polymorphism;SYFPEITHI: database for MHC ligands and peptide motifs;Description and prediction of peptide-MHC binding: the 'human MHC project'.;A theoretical approach towards the identification of cleavage-determining amino acid motifs of the 20 S proteasome.;The importance of pairwise interactions between peptide residues in the delineation of TCR specificity.;Application of an artificial neural network to predict specific class I MHC binding peptide sequences;Relationship between peptide selectivities of human transporters associated with antigen processing and HLA class I molecules.;MHC Ligands and Peptide Motifs;A structure-based algorithm to predict potential binding peptides to MHC molecules with hydrophobic binding pockets.;Two distinct proteolytic processes in the generation of a major histocompatibility complex class I-presented peptide.;Two complementary methods for predicting peptides binding major histocompatibility complex molecules.;Accessing the Kabat antibody sequence database by computer;An epitope-selective, transporter associated with antigen presentation (TAP)-1/2-independent pathway and a more general TAP-1/2-dependent antigen-processing pathway allow recognition of the HIV-1 envelope glycoprotein by CD8+ CTL.;Cytotoxicity mediated by T cells and natural killer cells is greatly impaired in perforin-deficient mice;Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains.;Identification of peptide sequences that potentially trigger HLA‐A2.1‐restricted cytotoxic T lymphocytes;Emerging principles for the recognition of peptide antigens by MHC class I molecules.;Query by committee;Characterization of a conserved T cell epitope in HIV-1 gp41 recognized by vaccine-induced human cytolytic T cells.;A semi‐empirical method for prediction of antigenic determinants on protein antigens;Prediction of major histocompatibility complex binding regions of protein antigens by sequence pattern analysis.;New hydrophilicity scale derived from high-performance liquid chromatography peptide retention data: correlation of predicted surface residues with antigenicity and X-ray-derived accessible sites.;Induction of hepatitis A virus-neutralizing antibody by a virus-specific synthetic peptide;Prediction of chain flexibility in proteins;Polyvalent vaccines for optimal coverage of potential T-cell epitopes in global HIV-1 variants;PLoS Computational Biology | www.ploscompbiol;HIV Molecular Immunology 2005;Los Alamos Hepatitis C Immunology Database;Identification of immunodominant epitopes of Schistosoma mansoni vaccine candidate antigens using human T cells.;IMGT-ONTOLOGY and IMGT databases, tools and Web resources for immunogenetics and immunoinformatics.;MHCPred: bringing a quantitative dimension to the online prediction of MHC binding.;IMGT/HLA and IMGT/MHC: sequence databases for the study of the major histocompatibility complex;MAPPP: MHC class I antigenic peptide processing prediction.;MMDB: Entrez's 3D-structure database;IMGT , the international ImMunoGeneTics information system;Enhanced motif scan: A tool to scan for HLA anchor residues in proteins. HIV Immunology and HIV/SIV Vaccine Databases Publication number LA-UR 04-8162;SACS-Self-maintaining database of antibody crystal structure information;Reliable prediction of T-cell epitopes using neural networks with novel sequence representations;Web-based Tools for Vaccine Design;Methods for Prediction of Peptide Binding to MHC Molecules: A Comparative Study;ELF: An analysis tool for HIV-1 peptides and HLA types;Structure‐based prediction of binding peptides to MHC class I molecules: Application to a broad range of MHC alleles;PeptGen: Designing Peptides for Immunological Studies and Application to HIV Consensus Sequences;Part 5 – HLA-F;Motifscan: A Web-based tool to find HLA anchor residues in proteins or peptides;MMDB: Entrez's 3D structure database;Mechanisms of MHC class I--restricted antigen processing.;MHCPEP, a database of MHC-binding peptides: update 1997;MHCPEP, a database of MHC-binding peptides: update 1996;Peptide binding to MHC class I molecules: Implications for antigenic peptide prediction;An Artificial Neural Network Approach;Publication number LA-UR 06-0036",
         "Immunoinformatics Comes of Age",
         "None"
        ],
        [
         "2",
         "00016363b9a128e01a17e9b831249967a0fee15f",
         "14-3-3 proteins modulate the plant inward rectifier K+ channel KAT1 heterologously expressed in Xenopus oocytes. Injection of recombinant plant 14-3-3 proteins into oocytes shifted the activation curve of KAT1 by +11 mV and increased the τon. KAT1 was also modulated by 14-3-3 proteins of Xenopus oocytes. Titration of the endogenous 14-3-3 proteins by injection of the peptide Raf 621p resulted in a strong decrease in KAT1 current (∼70% at –150 mV). The mutation K56E performed on plant protein 14-3-3 in a highly conserved recognition site prevented channel activation. Because the maximal conductance of KAT1 was unaffected by 14-3-3, we can exclude that they act by increasing the number of channels, thus ruling out any effect of these proteins on channel trafficking and/or insertion into the oocyte membrane. 14-3-3 proteins also increased KAT1 current in inside-out patches, suggesting a direct interaction with the channel. Direct interaction was confirmed by overlay experiments with radioactive 14-3-3 on oocyte membranes expressing KAT1.",
         "B. Sottocornola,S. Visconti,Sara A. Orsi,S. Gazzarrini,S. Giacometti,C. Olivari,L. Camoni,P. Aducci,M. Marra,A. Abenavoli,G. Thiel,A. Moroni",
         "B. Sottocornola,S. Visconti,Sara A. Orsi,S. Gazzarrini,S. Giacometti,C. Olivari,L. Camoni,P. Aducci,M. Marra,A. Abenavoli,G. Thiel,A. Moroni",
         "http://www.jbc.org/article/S0021925820681212/pdf",
         "14-3-3 proteins modulate the plant inward rectifier K chan-\nnel KAT1 heterologously expressed in Xenopus oocytes. Injec-\ntion of recombinant plant 14-3-3 proteins into oocytes shifted\nthe activation curve of KAT1 by 11 mV and increased the \non.\nKAT1 was also modulated by 14-3-3 proteins of Xenopus\noocytes. Titration of the endogenous 14-3-3 proteins by injec-\ntion of the peptide Raf 621p resulted in a strong decrease in\nKAT1 current (70% at 150 mV). The mutation K56E per-\nformed on plant protein 14-3-3 in a highly conserved recogni-\ntion site prevented channel activation. Because the maximal\nconductance of KAT1 was unaffected by 14-3-3, we can exclude\nthat they act by increasing the number of channels, thus ruling\nout any effect of these proteins on channel trafficking and/or\ninsertion into the oocyte membrane. 14-3-3 proteins also\nincreased KAT1 current in inside-out patches, suggesting a\ndirect interaction with the channel. Direct interaction was con-\nfirmed by overlay experiments with radioactive 14-3-3 on\noocyte membranes expressing KAT1.",
         "RESULTS AND DISCUSSION\nRecombinant 14-3-3 Proteins Increase the Activity of KAT1\nChannels Expressed in Xenopus Oocytes—Fig. 1A shows the\ncurrent traces recorded from KAT1 channels expressed in a\nXenopus oocyte. Clamping the oocyte with a voltage step pro-\ntocol ranging from 60 mV (holding potential) to 150 mV,\nelicited the slowly activating, inward K currents IKAT1 typical\nof this channel (5). The measured current values were stable\nover time; when monitored over a period of 30 min they var-\nied by 1% (not shown).\nTo examine the effect of 14-3-3 proteins on KAT1 currents\nthe same oocyte was removed from the voltage clamp apparatus\nand injected with 46 nl of a buffer containing 4 mg/ml of puri-\nfied recombinant 14-3-3 proteins from maize GF14-6 (final\nconcentration in the oocyte 1 M). The oocyte was kept 10\nmin at room temperature to recover from the injection and to\nallow 14-3-3 diffusion and challenged with the same voltage\nprotocol used prior to injection. Fig. 1A shows a strong increase\nin KAT1 current after 14-3-3 injection. The steady state cur-\nrent/voltage (I/V)2 relationship of the currents from Fig. 1A are\nshown in 1B. Fig. 1C shows the mean I/V relationship from 10\noocytes from three different batches. The increase in current\ninduced by 14-3-3 is voltage dependent; at 110 mV the cur-\nrent is increased by 200% (control, 0.36  0.02 A; 14-3-3,\n1.1  0.24 A), whereas at 140 mV is increased by 100%\n(control, 5.1  0.7 A; 14-3-3, 10.3  2.3 A).\nTo test the specificity of the 14-3-3 effect on IKAT1 we per-\nformed several control experiments, summarized in Fig. 2. The\neffect of injection of 14-3-3 on IKAT1, at the reference voltage of\n120 mV, is compared with that of heat-denatured 14-3-3 pro-\nteins, buffer only, and 14-3-3 proteins bearing the K56E muta-\ntion. This mutation, lysine 56 replaced by glutamic acid, is\nknown to disrupt the interaction with target proteins, as shown\nwith the plant proton pump (25). In our experiments, K56E\n14-3-3 proteins as well as heat-denatured wild-type proteins\nand resuspension buffer did not enhance IKAT1, indicating that\nthe effect observed after 14-3-3 injection is specific and not\nrelated to injection artifacts.\nBecause 14-3-3 proteins have been reported to modulate the\nendogenous Cl channels in Xenopus oocytes (29), we tested\nwhether the increase in inward current was entirely due to a\nstimulation of KAT1 channels. Fig. 3 shows that addition of 5\nmM Cs, a well known blocker of KAT1 channels (5) with no\nknown effect on Cl channels, completely blocked the inward\ncurrent after 14-3-3 stimulation. This shows that the stimu-\nlated current is KAT1 specific and is not due to a nonspecific\neffect on endogenous channels of the oocyte.\n14-3-3 Proteins Activate KAT1 Channel\nother hand, it has also been shown\nthat 14-3-3 proteins regulate the\ngating of animal and plant chan-\nnels by shifting the activation\ncurve, that is the plot of open\nprobability over voltage (20, 32).\nTo discriminate between these\ntwo possible modes of action, it was\nnecessary to check whether the\nmaximal value of IKAT1 was affected\nby the 14-3-3 proteins. Therefore\nwe had to measure the current of the\nfully activated channel, in the case of\nKAT1 at about 200 mV. Such neg-\native voltages can be deleterious for\nthe stability of the membrane; fur-\nthermore, they can trigger random\nactivation of endogenous currents.\nBecause the speed of IKAT1 activa-\ntion increases at negative voltages\n(33), we made use of a protocol\n(shown in Fig. 4A) that progres-\nsively reduces the length of the volt-\nage steps at negative potentials; in\nthis case we obtained all the essen-\ntial kinetic information without any\nunnecessary long and potentially\ndeleterious polarization of\nthe\nmembrane. Fig. 4B shows a control\nexperiment in which the full activa-\ntion of KAT1 was achieved. The\namplitude of the tail current from\nthe last preconditioning voltage\n(196 mV) is the same as that from\nthe preconditioning\nvoltage of\n186 mV. This indicates that KAT1\nchannels had reached full open\nprobability at 186 mV and that\nour protocol that steps at 200 mV\nserves the purpose of measuring the\ncurrent at full activation.\n14-3-3 Proteins Activate KAT1 Channels by Shifting the Acti-\nvation Curve—KAT1 is a voltage-gated channel, and its open\nprobability strictly depends on voltage. The increase in current\nobserved after injection of the 14-3-3 proteins can, in principle,\nresult from a change in conductance or from a change in open\nprobability.\nAn increase in conductance due to an increase in the number\nof active channels at the plasma membrane is a means by which\n14-3-3 proteins regulate ion channels in plants (19) and animal\ncells; in particular, for animal channels it has been shown that\n14-3-3 proteins modulate protein trafficking (30, 31). On the\n14-3-3 Proteins Activate KAT1 Channel\nability curve (V1⁄2) of155 mV and for the inverse slope factor\ncoefficient (s) of13 mV. The rather negative V1⁄2 value was in\nthe range reported previously for KAT1 in oocytes (33, 34).\nAfter 14-3-3 protein injection, the V1⁄2 shifted by 12 mV to 143\nmV and the inverse slope factor coefficient was 14 mV. The\nmaximal current value obtained by the Boltzmann fit was 3.57\nA in both cases, again confirming that 14-3-3 did not affect the\nmaximal conductance but only the open probability.\nAveraging normalized open probability curves (Fig. 4F) (n \n15 oocytes from seven independent experiments) shows that\n14-3-3 shifted the V1⁄2 by 11 mV, from 146 to 135 mV while\ns was unchanged (13 mV). The increase in current observed\nin response to 14-3-3 protein injection can hence be explained\nby a positive shift of the activation curve of the channel.\nThe findings that 14-3-3 proteins act directly on a channel\nprotein by altering its voltage dependence is in agreement with\nthat reported for the native inward rectifier from radicle proto-\nplasts (20). Also in this case, an elevation of 14-3-3 protein\nconcentration on the cytosolic side of the membrane resulted in\na positive shift in the activation curve of the channel by 9 mV.\nTo further test whether a positive shift of 11 mV in the\nchannel open probability is sufficient to explain the reported\nincrease in KAT1 current in response to elevation of 14-3-3\nproteins, we used the aforementioned Boltzmann parameters\nV1⁄2 and s to reconstruct the measured current/voltage relation.\nAs shown in Equation 2, the cell current (I) is\nI  i  N  Po\nwhere i is the single channel current amplitude of KAT1, N the\nnumber of channels in the membrane, and Po the open proba-\nbility. The curves in Fig. 1C were calculated using a unitary\nKAT1 channel conductance of 10 pS (36) with a reversal voltage\nat 18 mV and the respective values V1⁄2 and s from the Boltz-\nmann fit reported in Fig. 4F. The curves were jointly fitted to the\nmean current/voltage relations obtained before and after injec-\ntion of 14-3-3 proteins into oocytes (Fig. 1C); only the parame-\nter N was used as a fitting variable. A very good agreement\nbetween data and calculation was obtained with an N of 1.3 106\nchannels. This means that the entire increase in current\nobserved in response to 14-3-3 injection can be explained on\nthe basis of a shift in the voltage dependence; an increase in the\nnumber of channels as a consequence of a 14-3-3 effect on\nchannel trafficking was not needed to reproduce the data.\nWe then checked whether the shift in the activation curve\nwas accompanied by a change in channel kinetics. As expected,\nfollowing 14-3-3 treatment IKAT1 activation was accelerated.\nFig. 5A shows as an example current traces recorded at 130\nmV before and after injection of 14-3-3. Traces have been nor-\nmalized to show, qualitatively, current acceleration due to\n14-3-3. Fig. 5B shows the time constant of activation (\non) plot-\nted against voltage. The \non were estimated by fitting a single\nexponential function to current activation kinetics. At 130\nmV mean \non (n  6) was 486  39 and 363  18 ms for control\nand 14-3-3s, respectively. No apparent effect was observed on\nthe \noff, obtained from the deactivation kinetics. The accelera-\ntion of activation kinetics together with the positive shift of the\nactivation curve confirms that 14-3-3 proteins act by modifying\nthe kinetic properties of IKAT1.\n14-3-3 Proteins Act on KAT1 in a Membrane-delimited Way—\nTo test whether KAT1 is modulated by 14-3-3 proteins inde-\npendently of regulatory cellular pathways, we recorded IKAT1\nfrom inside-out macropatches. In this way we could access the\ncytosolic side of the channel and at the same time remove the\nchannel from the cytosolic environment. In excised patches\nIKAT1 undergoes a fast run down process, which shifts the volt-\nage dependence of the channel (37). Run down of the KAT1\nchannel could be recovered by the addition of 3 mM ATP to the\nbath solution as described by Hoshi (37, 38). In this condition\nwe were able to record KAT1 macrocurrents that were stable\nfor the length of the experiment (several minutes) and allowed\ntesting of the effect of 14-3-3 proteins (1 M final concentra-\ntion). Fig. 6A shows that addition of 14-3-3 increased IKAT1, as\n14-3-3 Proteins Activate KAT1 Channel\nalso shown by the steady state I/V relationship in Fig. 6B. A\nsimilar stimulation was observed in three of four patches; the\nmean I/V relationship of the positive responding patches is\nreported in Fig. 6C.\nThe occurrence of a run down process in excised patches\nalready indicates that KAT1 is regulated by cytoplasmic factors.\nTang and Hoshi (37) reported that a protein kinase A-mediated\nphosphorylation is responsible for the shift in the voltage-de-\npendent gating observed on patch excision. In contrast, non-\nspecific alkaline phosphatase accelerated the run down time\ncourse. Among others, we can probably add 14-3-3 proteins as\nfactors that seem to modulate the channel in a membrane-de-\nlimited pathway, given that the channel is in its active phospho-\nrylated form.\nEndogenous 14-3-3 Modulate KAT1 Channels—Working\nwith KAT1 channels expressed in oocytes we observed that the\nposition of the activation curve along the voltage axis can vary\ngreatly between experiments. The measured values of V1⁄2 var-\nied in the range 150 to 120 mV. This variability is suggestive\nof an endogenous regulation of the channel, and therefore we\ntested the possibility that endogenous 14-3-3 proteins could\nmodulate KAT1.\nTo examine this possibility we compared currents from\nKAT1-expressing oocytes before (control) and after the injec-\ntion of peptide Raf 621p (Fig. 7A). This phosphorylated peptide\nmimics the mode I recognition site of 14-3-3 proteins (16).\nWhen provided in large excess, it presumably titrates the\nendogenous 14-3-3 (see the use of this peptide in Refs. 15, 20,\n39). The injection of this peptide strongly decreased the level of\nIKAT1 in oocytes (70%  6 at150 mV, n  4); in control\nexperiments (data not shown) heat-inactivated Raf 621p\ndecreased the current at 150 mV of 38%  4 (n  4). This is\npresumably because it is not possible to efficiently inactivate\nthis short peptide by heating. As is apparent from the single\nexperiment and the mean I/V curves of Fig. 7, B and C, the\npeptide acts by shifting the voltage dependence of the channel\nto more negative values. For this reason it is difficult to reach\nfull activation of the channel in the Raf 621p-injected oocytes.\nOnly in one experiment (not shown) was it possible to measure\nthe current at full activation, and the measured shift in the\nactivation curve was 12 mV. We can conclude that endoge-\nnous 14-3-3s modulate KAT1 channel in oocytes and presum-\nably influence the position of the activation curve of this chan-\nnel. This finding could partly explain the strong run down\nphenomenon observed in excised patches of this channel.\nKAT1 Interacts in Vitro with the 14-3-3 Proteins—To test\nwhether the KAT1 protein interacts directly with the 14-3-3\nproteins, we have performed overlay experiments with radioac-\ntive 14-3-3s on total protein extract obtained from oocytes\ninjected with KAT1 mRNA and tested for KAT1 expression 3\ndays from injection. Membrane fraction from control or KAT1-\nexpressing oocytes was purified, subjected to SDS-PAGE, and\nblotted onto nitrocellulose membrane. Membranes were incu-\nbated with recombinant 32P-labeled 14-3-3 isoform from maize\nGF14-6. Autoradiography shows (Fig. 8) that the radioactive\nprobe in the crude membrane fraction purified from KAT1-\nexpressing oocytes recognizes only a single band of 80 kDa\n(lane 3), corresponding to the expected KAT1 molecular mass,\nwhereas no signal is detected in control membrane purified\nfrom oocytes not expressing KAT1 (lane 2). As expected, the\n14-3-3 probe efficiently recognizes the H-ATPase in the\nmaize plasma membrane fraction loaded as positive control\n(lane 1).\nControl experiments included the addition of Raf 621p pep-\ntide to inhibit the interaction of 14-3-3s with KAT1 and the\nsubstitution of wild-type 14-3-3 with the non-functional K56E\n14-3-3 mutant. In both cases we obtained no interaction (data\nnot shown).\nCONCLUSIONS\nRecombinant 14-3-3 proteins from maize activate KAT1\nchannels expressed in Xenopus oocytes. The 14-3-3-induced\nincrease in current is due to a positive shift in the voltage\ndependence of the channel and not to an increase in the num-\nber of channels. This effect is disrupted by mutation K56E in the\ncanonical interaction site of maize 14-3-3s. Excised patch\nexperiments and in vitro data suggest a direct interaction\nbetween KAT1 and the 14-3-3 proteins. Endogenous 14-3-3\nfrom Xenopus oocytes can control the activation curve of this\nchannel and represent a possible source of variability in this\nwell known expression system for ion channels.\nPhysiological Considerations—KAT1-like K channels mediate\nK uptake in many plant cells. The activity of these channels is\ncontrolled by the proton pump that establishes the negative\nmembrane voltage that opens the channels and thermodynam-\nically favors K uptake (40). It is also known that the two trans-\nport systems operate in concert, in as much as conditions cre-\nated by the proton pump, such as external acidification,\nenhance the activity of KAT1 channels and that the activity of\nthe channel is required to depolarize the negative potential that\nthermodynamically inhibits the proton pump (41). The finding\nthat 14-3-3 proteins, well known as proton pump activators,\ndirectly stimulate KAT1 channels constitutes strong molecular\nevidence that these two transporters can operate in concert in\nregulating K uptake.",
         "Voltage-dependent (Kv) potassium channels mediate an\ninward K current (IKin) at the plasma membrane of plant cells.\nIKin serves several functions: controlling membrane voltage,\nnutrient uptake, and osmotic regulation. In guard cells onset of\nthis current typically leads to K accumulation triggering sto-\nmatal opening (1–3). The molecular characterization of guard\ncell IKin started in 1992 with the cloning of KAT1 (4, 5), the first\nKv channel subunit from Arabidopsis thaliana, and still is a\nsubject of intensive research. One unsolved problem in this\nfield is the rationale for the existence of a large number of K\nchannel genes with apparently similar characteristics. Five\ngenes coding for inward K channel subunits have so far been\nfound to be expressed in guard cells, KAT1, KAT2, AKT1,\nAKT3/2, and AtKC1 (6). When heterologously expressed, none\nof these genes alone forms a channel that fully matches the\nproperties of native IKin with respect to pH, Ca2 dependence,\n* This work was supported in part by Ministero dell’Istruzione, dell’Universita`\ne della Ricerca PRIN 2005 (to A. M.). The costs of publication of this article\nwere defrayed in part by the payment of page charges. This article must\ntherefore be hereby marked “advertisement” in accordance with 18 U.S.C.\nSection 1734 solely to indicate this fact.\nand block by Cs (7). This suggests that the native channel\nmight be composed of different subunits and/or that additional\nregulatory proteins contribute to native IKin current. KAT1 can\nform heteromultimers with each of the four other K channel\nsubunits expressed in guard cells (8 –10). At least in the case of\nKAT1 and KAT2, two very similar proteins (displaying 85%\nidentity in their transmembrane region), the heteromeric\nchannel does not have properties distinguishable from the\nhomomeric channels. It is relevant that the C-terminal cyto-\nplasmic regions of the two channels display the lowest level\nof sequence similarities; these regions are typically regula-\ntory domains in Kv channels and often provide specific inter-\naction sites with regulatory proteins (11).\nKnown protein partners of plant Kv channels are  sub-\nunits (12), protein kinases, and phosphatases (13). Recent\ncontributions have now extended information on the regu-\nlatory effect of 14-3-3 proteins. These are a family of highly\nconserved proteins with molecular mass of 30 kDa,\nexpressed in all eukaryotic organisms. They exist in a num-\nber of isoforms and form homo- and heterodimers. 14-3-3\nproteins play a central role in the regulation of many cellular\nprocesses, such as cell cycle, differentiation, apoptosis, and\nmitogenic signal transduction (14). The common feature of\n14-3-3s is their ability to bind phosphorylated consensus\nmotifs on target proteins (15, 16); this accounts for their\ndiverse regulatory functions. In plants it has been shown that\n14-3-3 proteins accomplish peculiar functions, such as reg-\nulation of ion transport, through interaction with the plasma\nmembrane H-ATPase (17). Recently, increasing evidence\nsuggests that besides the H-ATPase 14-3-3s can also regulate\nK channels. In fact, they have been shown to interact with\nplant K outward rectifiers (18 –20), SV channels (21), and K\ninward rectifiers (20).\nThe effect of 14-3-3 proteins on K inward rectifiers has so\nfar only been reported for native channels. Because these stud-\nies have not yet exploited the question of subunit specificity, we\nhave tested the possibility that 14-3-3 proteins interact with the\nguard cell subunit KAT1.\nHere we show that the homotetrameric KAT1 channel\nexpressed in Xenopus oocytes is modulated by 14-3-3 proteins\nvia a direct interaction with the channel protein and that this\ninteraction changes the voltage-dependent properties and the\nactivation kinetics of the channel. We also show that KAT1 is\n14-3-3 Proteins Activate KAT1 Channel\nregulated by the interaction with endogenous 14-3-3 proteins\npresent in the oocyte.",
         "EXPERIMENTAL PROCEDURES\nExpression of KAT1 in Oocytes—KAT1 cDNA was cloned\ninto pSGEM vector (a modified version of pGEM-HE). cRNA\nwas transcribed in vitro using T7 RNA polymerase (Promega)\nand injected (25 ng/oocyte) into Xenopus laevis oocytes pre-\npared according to standard methods (22). Measurements were\nperformed 2– 4 days after injection. The average diameter of\nthe oocytes (stage IV and V) was considered to be 1 mm, and on\nthis basis we have calculated the final concentration of injected\n14-3-3 proteins and peptides when mentioned in the text.\nElectrophysiology—Two-electrode voltage clamp recordings\nwere made using a two-electrode voltage clamp amplifier\n(GeneClamp 500; Axon Instruments, Foster City, CA) under\ncontrol of pCLAMP8 software (Axon Instruments). Electrodes\nwere filled with 3 M KCl and had a resistance of 0.4 – 0.8 M in\n50 mM KCl. Experiments were performed at room temperature\nin a standard bath solution containing (mM): 50 KCl, 1.8 CaCl2,\n1 MgCl2, 5 Hepes/KOH, pH 7.4, at a rate of 2 ml min1. Man-\nnitol was used to adjust the osmolarity of the solution to 215\nmosmol. The standard clamp protocol consisted of a holding\nvoltage of 40 mV and steps from 60 to 180 mV. Current/\nvoltage relations report the experimentally measured voltages,\nwhich can slightly deviate from command voltages.\nPatch clamp recordings were made using an Axopatch 200A\namplifier (Axon Instruments) with borosilicate pipettes coated\nwith paraffin wax. Data acquisition and analysis were per-\nformed with pCLAMP 5.5 software (Axon Instruments). Linear\nleak subtraction was performed manually after acquisition.\nExperiments were performed at room temperature (20 –25 °C).\nThe external solution contained (mM): 100 KCl, 2 CaCl2, and 10\nHEPES (N-methyl-D-glucosamine), pH 7.2. The internal solu-\ntion contained (mM): 100 KCl, 2 MgCl2, 2 ATP (disodium salt),\n10 EGTA/KOH, pH 7.2, and 10 HEPES (N-methyl-D-glucosa-\nmine), pH 7.2.\nData Analysis—Activation curves were fitted by a Boltzmann\nequation as shown in Equation 1\npo  1/1  expV  V1/ 2/s\nwhere po is fractional activation (equivalent to the channel open\nprobability normalized to the maximum open probability), V is\nvoltage, V1⁄2 is the half-activation voltage, and s the inverse slope\ncoefficient (23). In Fig. 4E the tail current values are plotted as\nabsolute.\nEscherichia coli Strains—E. coli strain DH5 was used for\nplasmid propagation, and strain BL21 (DE3) pLysS was used for\nprotein expression. E. coli strains were grown at 37 °C in Luria\nBertani liquid medium or agar plates (15 g/liter). Ampicillin was\nused at a concentration of 100 g/ml.\nExpression and Purification of Recombinant 14-3-3 Proteins—\nThe maize 14-3-3 isoform GF14-6 was expressed in E. coli as\nGST fusion protein as already described (24). K56E GF14-6\nmutant was produced and expressed as previously described\n(25).\nFor production of His-tagged GF14-6, a cDNA fragment was\nremoved from pGEX-2TK, using a 5-NcoI restriction site and\na 3-EcoRI restriction site, and inserted into NcoI-EcoRI-cut\npET-32B vector (Novagen, Madison, WI), previously deprived\nof the thioredoxin coding region by NdeI digestion. Transfor-\nmation of E. coli was performed according to standard\nmethods.\nInduction of protein expression was performed by adding 1\nmM isopropyl-1-thio--D-galactopyranoside to liquid cultures.\nAfter 3 h ofinduction, cells were collected by centrifugation\nand stored at 80 °C. Cells were lysed in a buffer containing 10\nmM Tris-HCl, pH 8, 1 mM EDTA, 10 mM dithiothreitol, 20\ng/ml leupeptin, 0.5 mM phenylmethylsulfonyl fluoride, and\n0.75% N-lauryl sarcosine and sonicated until a clear, non-vis-\ncous solution was obtained (26). Particulate matter was re-\nmoved by centrifugation (15 min at 12,000  g), and the soluble\nfraction was used for protein purification.\nHis-GF14-6 was purified by affinity chromatography on\nNi2-nitriloacetic acid-agarose (Qiagen, Hilden, Germany)\nequilibrated with 50 mM sodium phosphate, pH 8, 300 mM\nNaCl, 5 mM -mercaptoethanol, and 20 mM imidazole; after\nextensive washing, protein was eluted by addition of increasing\nconcentrations (50, 100, 200, 400 mM) of imidazole to the above\ndescribed buffer. Fractions containing the purified fusion pro-\ntein were extensively dialyzed against 5 mM bis-tris propane-\nHepes, pH 7, in 10% (v/v) glycerol and stored at 80 °C.\nRaf 621p—Raf 621p is 15 amino acids long (LPKINR-\nSASpEPSLHR), corresponding to amino acids 613– 627 of Raf 1\n(15). Serine 621 is phosphorylated. The sequence highlighted in\nbold mimics the mode I recognition site of 14-3-3 proteins (16).\nIsolation of Oocyte Membranes—Membrane fractions were\nobtained according to De Jong et al. with minor modifications\n(27). 40 control or KAT1-expressing oocytes were homoge-\nnized in 800 l of 20 mM Tris-HCl, pH 7.4, containing 5 mM\nMgCl2, 5 mM NaH2PO4, 1 mM EDTA, 80 mM sucrose, 1 mM\nphenylmethylsulfonyl fluoride, 5 g/ml leupeptin, and 5 g/ml\npepstatin and centrifuged two times for 10 min at 3000  g at\n4 °C to remove yolk proteins. Membranes were pelleted by cen-\ntrifugation for 30 min at 14,000  g at 4 °C and resuspended in\n20 mM Tris-HCl, pH 7.4, containing 1 mM EDTA, 1 mM phen-\nylmethylsulfonyl fluoride, 5 g/ml leupeptin and 5 g/ml pep-\nstatin, and 20% (v/v) glycerol.\nSDS-PAGE and Overlay Assay—SDS-PAGE was performed\nas described by Laemmli (28) in a Mini Protean apparatus\n(Bio-Rad). The system used for expression of GF14-6 pro-\nteins produces a glutathione S-transferase-fused 14-3-3 con-\ntaining a cAMP-dependent protein kinase phosphorylation\nsite and a thrombin site between the two polypeptides. The\n32P-labeled GF14-6 was obtained as described (24). Specific\nactivity of the labeled protein was 3 MBq/mg. The overlay\nassay was carried out according to Ref. 24 with minor mod-\nifications. Briefly, 20 g of membrane fractions from KAT1-\nexpressing Xenopus oocytes were subjected to SDS-PAGE\nand blotted onto nitrocellulose membrane, using a semidry\nLKB apparatus (2 h, 0.8 mA cm2). The membrane was\nblocked with 5% fatty acid-free milk in 25 mM Hepes-OH, 75\nmM KCl, 5 mM MgCl2, 1 mM dithiothreitol, 0.1 mM EDTA,\n0.05% Tween 20, pH 7.5 (buffer H) and then incubated over-\nnight at 4 °C in the same buffer containing 2% fatty acid-free\nmilk, 3 g of 32P-labeled GF14-6 (corresponding to 9 kBq/\n14-3-3 Proteins Activate KAT1 Channel\nml). After incubation, the membrane was washed three times\nwith buffer H, dried, and subjected to autoradiography at\n80 °C. The overlay experiment was performed three times,\nand similar results were obtained.",
         "EXPERIMENTAL PROCEDURES\nExpression of KAT1 in Oocytes—KAT1 cDNA was cloned\ninto pSGEM vector (a modified version of pGEM-HE). cRNA\nwas transcribed in vitro using T7 RNA polymerase (Promega)\nand injected (25 ng/oocyte) into Xenopus laevis oocytes pre-\npared according to standard methods (22). Measurements were\nperformed 2– 4 days after injection. The average diameter of\nthe oocytes (stage IV and V) was considered to be 1 mm, and on\nthis basis we have calculated the final concentration of injected\n14-3-3 proteins and peptides when mentioned in the text.\nElectrophysiology—Two-electrode voltage clamp recordings\nwere made using a two-electrode voltage clamp amplifier\n(GeneClamp 500; Axon Instruments, Foster City, CA) under\ncontrol of pCLAMP8 software (Axon Instruments). Electrodes\nwere filled with 3 M KCl and had a resistance of 0.4 – 0.8 M in\n50 mM KCl. Experiments were performed at room temperature\nin a standard bath solution containing (mM): 50 KCl, 1.8 CaCl2,\n1 MgCl2, 5 Hepes/KOH, pH 7.4, at a rate of 2 ml min1. Man-\nnitol was used to adjust the osmolarity of the solution to 215\nmosmol. The standard clamp protocol consisted of a holding\nvoltage of 40 mV and steps from 60 to 180 mV. Current/\nvoltage relations report the experimentally measured voltages,\nwhich can slightly deviate from command voltages.\nPatch clamp recordings were made using an Axopatch 200A\namplifier (Axon Instruments) with borosilicate pipettes coated\nwith paraffin wax. Data acquisition and analysis were per-\nformed with pCLAMP 5.5 software (Axon Instruments). Linear\nleak subtraction was performed manually after acquisition.\nExperiments were performed at room temperature (20 –25 °C).\nThe external solution contained (mM): 100 KCl, 2 CaCl2, and 10\nHEPES (N-methyl-D-glucosamine), pH 7.2. The internal solu-\ntion contained (mM): 100 KCl, 2 MgCl2, 2 ATP (disodium salt),\n10 EGTA/KOH, pH 7.2, and 10 HEPES (N-methyl-D-glucosa-\nmine), pH 7.2.\nData Analysis—Activation curves were fitted by a Boltzmann\nequation as shown in Equation 1\npo  1/1  expV  V1/ 2/s\nwhere po is fractional activation (equivalent to the channel open\nprobability normalized to the maximum open probability), V is\nvoltage, V1⁄2 is the half-activation voltage, and s the inverse slope\ncoefficient (23). In Fig. 4E the tail current values are plotted as\nabsolute.\nEscherichia coli Strains—E. coli strain DH5 was used for\nplasmid propagation, and strain BL21 (DE3) pLysS was used for\nprotein expression. E. coli strains were grown at 37 °C in Luria\nBertani liquid medium or agar plates (15 g/liter). Ampicillin was\nused at a concentration of 100 g/ml.\nExpression and Purification of Recombinant 14-3-3 Proteins—\nThe maize 14-3-3 isoform GF14-6 was expressed in E. coli as\nGST fusion protein as already described (24). K56E GF14-6\nmutant was produced and expressed as previously described\n(25).\nFor production of His-tagged GF14-6, a cDNA fragment was\nremoved from pGEX-2TK, using a 5-NcoI restriction site and\na 3-EcoRI restriction site, and inserted into NcoI-EcoRI-cut\npET-32B vector (Novagen, Madison, WI), previously deprived\nof the thioredoxin coding region by NdeI digestion. Transfor-\nmation of E. coli was performed according to standard\nmethods.\nInduction of protein expression was performed by adding 1\nmM isopropyl-1-thio--D-galactopyranoside to liquid cultures.\nAfter 3 h ofinduction, cells were collected by centrifugation\nand stored at 80 °C. Cells were lysed in a buffer containing 10\nmM Tris-HCl, pH 8, 1 mM EDTA, 10 mM dithiothreitol, 20\ng/ml leupeptin, 0.5 mM phenylmethylsulfonyl fluoride, and\n0.75% N-lauryl sarcosine and sonicated until a clear, non-vis-\ncous solution was obtained (26). Particulate matter was re-\nmoved by centrifugation (15 min at 12,000  g), and the soluble\nfraction was used for protein purification.\nHis-GF14-6 was purified by affinity chromatography on\nNi2-nitriloacetic acid-agarose (Qiagen, Hilden, Germany)\nequilibrated with 50 mM sodium phosphate, pH 8, 300 mM\nNaCl, 5 mM -mercaptoethanol, and 20 mM imidazole; after\nextensive washing, protein was eluted by addition of increasing\nconcentrations (50, 100, 200, 400 mM) of imidazole to the above\ndescribed buffer. Fractions containing the purified fusion pro-\ntein were extensively dialyzed against 5 mM bis-tris propane-\nHepes, pH 7, in 10% (v/v) glycerol and stored at 80 °C.\nRaf 621p—Raf 621p is 15 amino acids long (LPKINR-\nSASpEPSLHR), corresponding to amino acids 613– 627 of Raf 1\n(15). Serine 621 is phosphorylated. The sequence highlighted in\nbold mimics the mode I recognition site of 14-3-3 proteins (16).\nIsolation of Oocyte Membranes—Membrane fractions were\nobtained according to De Jong et al. with minor modifications\n(27). 40 control or KAT1-expressing oocytes were homoge-\nnized in 800 l of 20 mM Tris-HCl, pH 7.4, containing 5 mM\nMgCl2, 5 mM NaH2PO4, 1 mM EDTA, 80 mM sucrose, 1 mM\nphenylmethylsulfonyl fluoride, 5 g/ml leupeptin, and 5 g/ml\npepstatin and centrifuged two times for 10 min at 3000  g at\n4 °C to remove yolk proteins. Membranes were pelleted by cen-\ntrifugation for 30 min at 14,000  g at 4 °C and resuspended in\n20 mM Tris-HCl, pH 7.4, containing 1 mM EDTA, 1 mM phen-\nylmethylsulfonyl fluoride, 5 g/ml leupeptin and 5 g/ml pep-\nstatin, and 20% (v/v) glycerol.\nSDS-PAGE and Overlay Assay—SDS-PAGE was performed\nas described by Laemmli (28) in a Mini Protean apparatus\n(Bio-Rad). The system used for expression of GF14-6 pro-\nteins produces a glutathione S-transferase-fused 14-3-3 con-\ntaining a cAMP-dependent protein kinase phosphorylation\nsite and a thrombin site between the two polypeptides. The\n32P-labeled GF14-6 was obtained as described (24). Specific\nactivity of the labeled protein was 3 MBq/mg. The overlay\nassay was carried out according to Ref. 24 with minor mod-\nifications. Briefly, 20 g of membrane fractions from KAT1-\nexpressing Xenopus oocytes were subjected to SDS-PAGE\nand blotted onto nitrocellulose membrane, using a semidry\nLKB apparatus (2 h, 0.8 mA cm2). The membrane was\nblocked with 5% fatty acid-free milk in 25 mM Hepes-OH, 75\nmM KCl, 5 mM MgCl2, 1 mM dithiothreitol, 0.1 mM EDTA,\n0.05% Tween 20, pH 7.5 (buffer H) and then incubated over-\nnight at 4 °C in the same buffer containing 2% fatty acid-free\nmilk, 3 g of 32P-labeled GF14-6 (corresponding to 9 kBq/\n14-3-3 Proteins Activate KAT1 Channel\nml). After incubation, the membrane was washed three times\nwith buffer H, dried, and subjected to autoradiography at\n80 °C. The overlay experiment was performed three times,\nand similar results were obtained.\nRESULTS AND DISCUSSION\nRecombinant 14-3-3 Proteins Increase the Activity of KAT1\nChannels Expressed in Xenopus Oocytes—Fig. 1A shows the\ncurrent traces recorded from KAT1 channels expressed in a\nXenopus oocyte. Clamping the oocyte with a voltage step pro-\ntocol ranging from 60 mV (holding potential) to 150 mV,\nelicited the slowly activating, inward K currents IKAT1 typical\nof this channel (5). The measured current values were stable\nover time; when monitored over a period of 30 min they var-\nied by 1% (not shown).\nTo examine the effect of 14-3-3 proteins on KAT1 currents\nthe same oocyte was removed from the voltage clamp apparatus\nand injected with 46 nl of a buffer containing 4 mg/ml of puri-\nfied recombinant 14-3-3 proteins from maize GF14-6 (final\nconcentration in the oocyte 1 M). The oocyte was kept 10\nmin at room temperature to recover from the injection and to\nallow 14-3-3 diffusion and challenged with the same voltage\nprotocol used prior to injection. Fig. 1A shows a strong increase\nin KAT1 current after 14-3-3 injection. The steady state cur-\nrent/voltage (I/V)2 relationship of the currents from Fig. 1A are\nshown in 1B. Fig. 1C shows the mean I/V relationship from 10\noocytes from three different batches. The increase in current\ninduced by 14-3-3 is voltage dependent; at 110 mV the cur-\nrent is increased by 200% (control, 0.36  0.02 A; 14-3-3,\n1.1  0.24 A), whereas at 140 mV is increased by 100%\n(control, 5.1  0.7 A; 14-3-3, 10.3  2.3 A).\nTo test the specificity of the 14-3-3 effect on IKAT1 we per-\nformed several control experiments, summarized in Fig. 2. The\neffect of injection of 14-3-3 on IKAT1, at the reference voltage of\n120 mV, is compared with that of heat-denatured 14-3-3 pro-\nteins, buffer only, and 14-3-3 proteins bearing the K56E muta-\ntion. This mutation, lysine 56 replaced by glutamic acid, is\nknown to disrupt the interaction with target proteins, as shown\nwith the plant proton pump (25). In our experiments, K56E\n14-3-3 proteins as well as heat-denatured wild-type proteins\nand resuspension buffer did not enhance IKAT1, indicating that\nthe effect observed after 14-3-3 injection is specific and not\nrelated to injection artifacts.\nBecause 14-3-3 proteins have been reported to modulate the\nendogenous Cl channels in Xenopus oocytes (29), we tested\nwhether the increase in inward current was entirely due to a\nstimulation of KAT1 channels. Fig. 3 shows that addition of 5\nmM Cs, a well known blocker of KAT1 channels (5) with no\nknown effect on Cl channels, completely blocked the inward\ncurrent after 14-3-3 stimulation. This shows that the stimu-\nlated current is KAT1 specific and is not due to a nonspecific\neffect on endogenous channels of the oocyte.\n14-3-3 Proteins Activate KAT1 Channel\nother hand, it has also been shown\nthat 14-3-3 proteins regulate the\ngating of animal and plant chan-\nnels by shifting the activation\ncurve, that is the plot of open\nprobability over voltage (20, 32).\nTo discriminate between these\ntwo possible modes of action, it was\nnecessary to check whether the\nmaximal value of IKAT1 was affected\nby the 14-3-3 proteins. Therefore\nwe had to measure the current of the\nfully activated channel, in the case of\nKAT1 at about 200 mV. Such neg-\native voltages can be deleterious for\nthe stability of the membrane; fur-\nthermore, they can trigger random\nactivation of endogenous currents.\nBecause the speed of IKAT1 activa-\ntion increases at negative voltages\n(33), we made use of a protocol\n(shown in Fig. 4A) that progres-\nsively reduces the length of the volt-\nage steps at negative potentials; in\nthis case we obtained all the essen-\ntial kinetic information without any\nunnecessary long and potentially\ndeleterious polarization of\nthe\nmembrane. Fig. 4B shows a control\nexperiment in which the full activa-\ntion of KAT1 was achieved. The\namplitude of the tail current from\nthe last preconditioning voltage\n(196 mV) is the same as that from\nthe preconditioning\nvoltage of\n186 mV. This indicates that KAT1\nchannels had reached full open\nprobability at 186 mV and that\nour protocol that steps at 200 mV\nserves the purpose of measuring the\ncurrent at full activation.\n14-3-3 Proteins Activate KAT1 Channels by Shifting the Acti-\nvation Curve—KAT1 is a voltage-gated channel, and its open\nprobability strictly depends on voltage. The increase in current\nobserved after injection of the 14-3-3 proteins can, in principle,\nresult from a change in conductance or from a change in open\nprobability.\nAn increase in conductance due to an increase in the number\nof active channels at the plasma membrane is a means by which\n14-3-3 proteins regulate ion channels in plants (19) and animal\ncells; in particular, for animal channels it has been shown that\n14-3-3 proteins modulate protein trafficking (30, 31). On the\n14-3-3 Proteins Activate KAT1 Channel\nability curve (V1⁄2) of155 mV and for the inverse slope factor\ncoefficient (s) of13 mV. The rather negative V1⁄2 value was in\nthe range reported previously for KAT1 in oocytes (33, 34).\nAfter 14-3-3 protein injection, the V1⁄2 shifted by 12 mV to 143\nmV and the inverse slope factor coefficient was 14 mV. The\nmaximal current value obtained by the Boltzmann fit was 3.57\nA in both cases, again confirming that 14-3-3 did not affect the\nmaximal conductance but only the open probability.\nAveraging normalized open probability curves (Fig. 4F) (n \n15 oocytes from seven independent experiments) shows that\n14-3-3 shifted the V1⁄2 by 11 mV, from 146 to 135 mV while\ns was unchanged (13 mV). The increase in current observed\nin response to 14-3-3 protein injection can hence be explained\nby a positive shift of the activation curve of the channel.\nThe findings that 14-3-3 proteins act directly on a channel\nprotein by altering its voltage dependence is in agreement with\nthat reported for the native inward rectifier from radicle proto-\nplasts (20). Also in this case, an elevation of 14-3-3 protein\nconcentration on the cytosolic side of the membrane resulted in\na positive shift in the activation curve of the channel by 9 mV.\nTo further test whether a positive shift of 11 mV in the\nchannel open probability is sufficient to explain the reported\nincrease in KAT1 current in response to elevation of 14-3-3\nproteins, we used the aforementioned Boltzmann parameters\nV1⁄2 and s to reconstruct the measured current/voltage relation.\nAs shown in Equation 2, the cell current (I) is\nI  i  N  Po\nwhere i is the single channel current amplitude of KAT1, N the\nnumber of channels in the membrane, and Po the open proba-\nbility. The curves in Fig. 1C were calculated using a unitary\nKAT1 channel conductance of 10 pS (36) with a reversal voltage\nat 18 mV and the respective values V1⁄2 and s from the Boltz-\nmann fit reported in Fig. 4F. The curves were jointly fitted to the\nmean current/voltage relations obtained before and after injec-\ntion of 14-3-3 proteins into oocytes (Fig. 1C); only the parame-\nter N was used as a fitting variable. A very good agreement\nbetween data and calculation was obtained with an N of 1.3 106\nchannels. This means that the entire increase in current\nobserved in response to 14-3-3 injection can be explained on\nthe basis of a shift in the voltage dependence; an increase in the\nnumber of channels as a consequence of a 14-3-3 effect on\nchannel trafficking was not needed to reproduce the data.\nWe then checked whether the shift in the activation curve\nwas accompanied by a change in channel kinetics. As expected,\nfollowing 14-3-3 treatment IKAT1 activation was accelerated.\nFig. 5A shows as an example current traces recorded at 130\nmV before and after injection of 14-3-3. Traces have been nor-\nmalized to show, qualitatively, current acceleration due to\n14-3-3. Fig. 5B shows the time constant of activation (\non) plot-\nted against voltage. The \non were estimated by fitting a single\nexponential function to current activation kinetics. At 130\nmV mean \non (n  6) was 486  39 and 363  18 ms for control\nand 14-3-3s, respectively. No apparent effect was observed on\nthe \noff, obtained from the deactivation kinetics. The accelera-\ntion of activation kinetics together with the positive shift of the\nactivation curve confirms that 14-3-3 proteins act by modifying\nthe kinetic properties of IKAT1.\n14-3-3 Proteins Act on KAT1 in a Membrane-delimited Way—\nTo test whether KAT1 is modulated by 14-3-3 proteins inde-\npendently of regulatory cellular pathways, we recorded IKAT1\nfrom inside-out macropatches. In this way we could access the\ncytosolic side of the channel and at the same time remove the\nchannel from the cytosolic environment. In excised patches\nIKAT1 undergoes a fast run down process, which shifts the volt-\nage dependence of the channel (37). Run down of the KAT1\nchannel could be recovered by the addition of 3 mM ATP to the\nbath solution as described by Hoshi (37, 38). In this condition\nwe were able to record KAT1 macrocurrents that were stable\nfor the length of the experiment (several minutes) and allowed\ntesting of the effect of 14-3-3 proteins (1 M final concentra-\ntion). Fig. 6A shows that addition of 14-3-3 increased IKAT1, as\n14-3-3 Proteins Activate KAT1 Channel\nalso shown by the steady state I/V relationship in Fig. 6B. A\nsimilar stimulation was observed in three of four patches; the\nmean I/V relationship of the positive responding patches is\nreported in Fig. 6C.\nThe occurrence of a run down process in excised patches\nalready indicates that KAT1 is regulated by cytoplasmic factors.\nTang and Hoshi (37) reported that a protein kinase A-mediated\nphosphorylation is responsible for the shift in the voltage-de-\npendent gating observed on patch excision. In contrast, non-\nspecific alkaline phosphatase accelerated the run down time\ncourse. Among others, we can probably add 14-3-3 proteins as\nfactors that seem to modulate the channel in a membrane-de-\nlimited pathway, given that the channel is in its active phospho-\nrylated form.\nEndogenous 14-3-3 Modulate KAT1 Channels—Working\nwith KAT1 channels expressed in oocytes we observed that the\nposition of the activation curve along the voltage axis can vary\ngreatly between experiments. The measured values of V1⁄2 var-\nied in the range 150 to 120 mV. This variability is suggestive\nof an endogenous regulation of the channel, and therefore we\ntested the possibility that endogenous 14-3-3 proteins could\nmodulate KAT1.\nTo examine this possibility we compared currents from\nKAT1-expressing oocytes before (control) and after the injec-\ntion of peptide Raf 621p (Fig. 7A). This phosphorylated peptide\nmimics the mode I recognition site of 14-3-3 proteins (16).\nWhen provided in large excess, it presumably titrates the\nendogenous 14-3-3 (see the use of this peptide in Refs. 15, 20,\n39). The injection of this peptide strongly decreased the level of\nIKAT1 in oocytes (70%  6 at150 mV, n  4); in control\nexperiments (data not shown) heat-inactivated Raf 621p\ndecreased the current at 150 mV of 38%  4 (n  4). This is\npresumably because it is not possible to efficiently inactivate\nthis short peptide by heating. As is apparent from the single\nexperiment and the mean I/V curves of Fig. 7, B and C, the\npeptide acts by shifting the voltage dependence of the channel\nto more negative values. For this reason it is difficult to reach\nfull activation of the channel in the Raf 621p-injected oocytes.\nOnly in one experiment (not shown) was it possible to measure\nthe current at full activation, and the measured shift in the\nactivation curve was 12 mV. We can conclude that endoge-\nnous 14-3-3s modulate KAT1 channel in oocytes and presum-\nably influence the position of the activation curve of this chan-\nnel. This finding could partly explain the strong run down\nphenomenon observed in excised patches of this channel.\nKAT1 Interacts in Vitro with the 14-3-3 Proteins—To test\nwhether the KAT1 protein interacts directly with the 14-3-3\nproteins, we have performed overlay experiments with radioac-\ntive 14-3-3s on total protein extract obtained from oocytes\ninjected with KAT1 mRNA and tested for KAT1 expression 3\ndays from injection. Membrane fraction from control or KAT1-\nexpressing oocytes was purified, subjected to SDS-PAGE, and\nblotted onto nitrocellulose membrane. Membranes were incu-\nbated with recombinant 32P-labeled 14-3-3 isoform from maize\nGF14-6. Autoradiography shows (Fig. 8) that the radioactive\nprobe in the crude membrane fraction purified from KAT1-\nexpressing oocytes recognizes only a single band of 80 kDa\n(lane 3), corresponding to the expected KAT1 molecular mass,\nwhereas no signal is detected in control membrane purified\nfrom oocytes not expressing KAT1 (lane 2). As expected, the\n14-3-3 probe efficiently recognizes the H-ATPase in the\nmaize plasma membrane fraction loaded as positive control\n(lane 1).\nControl experiments included the addition of Raf 621p pep-\ntide to inhibit the interaction of 14-3-3s with KAT1 and the\nsubstitution of wild-type 14-3-3 with the non-functional K56E\n14-3-3 mutant. In both cases we obtained no interaction (data\nnot shown).",
         "downloading",
         "14-3-3 proteins modulate the plant inward rectifier K chan-\nnel KAT1 heterologously expressed in Xenopus oocytes. Injec-\ntion of recombinant plant 14-3-3 proteins into oocytes shifted\nthe activation curve of KAT1 by 11 mV and increased the \non.\nKAT1 was also modulated by 14-3-3 proteins of Xenopus\noocytes. Titration of the endogenous 14-3-3 proteins by injec-\ntion of the peptide Raf 621p resulted in a strong decrease in\nKAT1 current (70% at 150 mV). The mutation K56E per-\nformed on plant protein 14-3-3 in a highly conserved recogni-\ntion site prevented channel activation. Because the maximal\nconductance of KAT1 was unaffected by 14-3-3, we can exclude\nthat they act by increasing the number of channels, thus ruling\nout any effect of these proteins on channel trafficking and/or\ninsertion into the oocyte membrane. 14-3-3 proteins also\nincreased KAT1 current in inside-out patches, suggesting a\ndirect interaction with the channel. Direct interaction was con-\nfirmed by overlay experiments with radioactive 14-3-3 on\noocyte membranes expressing KAT1.\nVoltage-dependent (Kv) potassium channels mediate an\ninward K current (IKin) at the plasma membrane of plant cells.\nIKin serves several functions: controlling membrane voltage,\nnutrient uptake, and osmotic regulation. In guard cells onset of\nthis current typically leads to K accumulation triggering sto-\nmatal opening (1–3). The molecular characterization of guard\ncell IKin started in 1992 with the cloning of KAT1 (4, 5), the first\nKv channel subunit from Arabidopsis thaliana, and still is a\nsubject of intensive research. One unsolved problem in this\nfield is the rationale for the existence of a large number of K\nchannel genes with apparently similar characteristics. Five\ngenes coding for inward K channel subunits have so far been\nfound to be expressed in guard cells, KAT1, KAT2, AKT1,\nAKT3/2, and AtKC1 (6). When heterologously expressed, none\nof these genes alone forms a channel that fully matches the\nproperties of native IKin with respect to pH, Ca2 dependence,\n* This work was supported in part by Ministero dell’Istruzione, dell’Universita`\ne della Ricerca PRIN 2005 (to A. M.). The costs of publication of this article\nwere defrayed in part by the payment of page charges. This article must\ntherefore be hereby marked “advertisement” in accordance with 18 U.S.C.\nSection 1734 solely to indicate this fact.\nand block by Cs (7). This suggests that the native channel\nmight be composed of different subunits and/or that additional\nregulatory proteins contribute to native IKin current. KAT1 can\nform heteromultimers with each of the four other K channel\nsubunits expressed in guard cells (8 –10). At least in the case of\nKAT1 and KAT2, two very similar proteins (displaying 85%\nidentity in their transmembrane region), the heteromeric\nchannel does not have properties distinguishable from the\nhomomeric channels. It is relevant that the C-terminal cyto-\nplasmic regions of the two channels display the lowest level\nof sequence similarities; these regions are typically regula-\ntory domains in Kv channels and often provide specific inter-\naction sites with regulatory proteins (11).\nKnown protein partners of plant Kv channels are  sub-\nunits (12), protein kinases, and phosphatases (13). Recent\ncontributions have now extended information on the regu-\nlatory effect of 14-3-3 proteins. These are a family of highly\nconserved proteins with molecular mass of 30 kDa,\nexpressed in all eukaryotic organisms. They exist in a num-\nber of isoforms and form homo- and heterodimers. 14-3-3\nproteins play a central role in the regulation of many cellular\nprocesses, such as cell cycle, differentiation, apoptosis, and\nmitogenic signal transduction (14). The common feature of\n14-3-3s is their ability to bind phosphorylated consensus\nmotifs on target proteins (15, 16); this accounts for their\ndiverse regulatory functions. In plants it has been shown that\n14-3-3 proteins accomplish peculiar functions, such as reg-\nulation of ion transport, through interaction with the plasma\nmembrane H-ATPase (17). Recently, increasing evidence\nsuggests that besides the H-ATPase 14-3-3s can also regulate\nK channels. In fact, they have been shown to interact with\nplant K outward rectifiers (18 –20), SV channels (21), and K\ninward rectifiers (20).\nThe effect of 14-3-3 proteins on K inward rectifiers has so\nfar only been reported for native channels. Because these stud-\nies have not yet exploited the question of subunit specificity, we\nhave tested the possibility that 14-3-3 proteins interact with the\nguard cell subunit KAT1.\nHere we show that the homotetrameric KAT1 channel\nexpressed in Xenopus oocytes is modulated by 14-3-3 proteins\nvia a direct interaction with the channel protein and that this\ninteraction changes the voltage-dependent properties and the\nactivation kinetics of the channel. We also show that KAT1 is\n14-3-3 Proteins Activate KAT1 Channel\nregulated by the interaction with endogenous 14-3-3 proteins\npresent in the oocyte.\nEXPERIMENTAL PROCEDURES\nExpression of KAT1 in Oocytes—KAT1 cDNA was cloned\ninto pSGEM vector (a modified version of pGEM-HE). cRNA\nwas transcribed in vitro using T7 RNA polymerase (Promega)\nand injected (25 ng/oocyte) into Xenopus laevis oocytes pre-\npared according to standard methods (22). Measurements were\nperformed 2– 4 days after injection. The average diameter of\nthe oocytes (stage IV and V) was considered to be 1 mm, and on\nthis basis we have calculated the final concentration of injected\n14-3-3 proteins and peptides when mentioned in the text.\nElectrophysiology—Two-electrode voltage clamp recordings\nwere made using a two-electrode voltage clamp amplifier\n(GeneClamp 500; Axon Instruments, Foster City, CA) under\ncontrol of pCLAMP8 software (Axon Instruments). Electrodes\nwere filled with 3 M KCl and had a resistance of 0.4 – 0.8 M in\n50 mM KCl. Experiments were performed at room temperature\nin a standard bath solution containing (mM): 50 KCl, 1.8 CaCl2,\n1 MgCl2, 5 Hepes/KOH, pH 7.4, at a rate of 2 ml min1. Man-\nnitol was used to adjust the osmolarity of the solution to 215\nmosmol. The standard clamp protocol consisted of a holding\nvoltage of 40 mV and steps from 60 to 180 mV. Current/\nvoltage relations report the experimentally measured voltages,\nwhich can slightly deviate from command voltages.\nPatch clamp recordings were made using an Axopatch 200A\namplifier (Axon Instruments) with borosilicate pipettes coated\nwith paraffin wax. Data acquisition and analysis were per-\nformed with pCLAMP 5.5 software (Axon Instruments). Linear\nleak subtraction was performed manually after acquisition.\nExperiments were performed at room temperature (20 –25 °C).\nThe external solution contained (mM): 100 KCl, 2 CaCl2, and 10\nHEPES (N-methyl-D-glucosamine), pH 7.2. The internal solu-\ntion contained (mM): 100 KCl, 2 MgCl2, 2 ATP (disodium salt),\n10 EGTA/KOH, pH 7.2, and 10 HEPES (N-methyl-D-glucosa-\nmine), pH 7.2.\nData Analysis—Activation curves were fitted by a Boltzmann\nequation as shown in Equation 1\npo  1/1  expV  V1/ 2/s\nwhere po is fractional activation (equivalent to the channel open\nprobability normalized to the maximum open probability), V is\nvoltage, V1⁄2 is the half-activation voltage, and s the inverse slope\ncoefficient (23). In Fig. 4E the tail current values are plotted as\nabsolute.\nEscherichia coli Strains—E. coli strain DH5 was used for\nplasmid propagation, and strain BL21 (DE3) pLysS was used for\nprotein expression. E. coli strains were grown at 37 °C in Luria\nBertani liquid medium or agar plates (15 g/liter). Ampicillin was\nused at a concentration of 100 g/ml.\nExpression and Purification of Recombinant 14-3-3 Proteins—\nThe maize 14-3-3 isoform GF14-6 was expressed in E. coli as\nGST fusion protein as already described (24). K56E GF14-6\nmutant was produced and expressed as previously described\n(25).\nFor production of His-tagged GF14-6, a cDNA fragment was\nremoved from pGEX-2TK, using a 5-NcoI restriction site and\na 3-EcoRI restriction site, and inserted into NcoI-EcoRI-cut\npET-32B vector (Novagen, Madison, WI), previously deprived\nof the thioredoxin coding region by NdeI digestion. Transfor-\nmation of E. coli was performed according to standard\nmethods.\nInduction of protein expression was performed by adding 1\nmM isopropyl-1-thio--D-galactopyranoside to liquid cultures.\nAfter 3 h ofinduction, cells were collected by centrifugation\nand stored at 80 °C. Cells were lysed in a buffer containing 10\nmM Tris-HCl, pH 8, 1 mM EDTA, 10 mM dithiothreitol, 20\ng/ml leupeptin, 0.5 mM phenylmethylsulfonyl fluoride, and\n0.75% N-lauryl sarcosine and sonicated until a clear, non-vis-\ncous solution was obtained (26). Particulate matter was re-\nmoved by centrifugation (15 min at 12,000  g), and the soluble\nfraction was used for protein purification.\nHis-GF14-6 was purified by affinity chromatography on\nNi2-nitriloacetic acid-agarose (Qiagen, Hilden, Germany)\nequilibrated with 50 mM sodium phosphate, pH 8, 300 mM\nNaCl, 5 mM -mercaptoethanol, and 20 mM imidazole; after\nextensive washing, protein was eluted by addition of increasing\nconcentrations (50, 100, 200, 400 mM) of imidazole to the above\ndescribed buffer. Fractions containing the purified fusion pro-\ntein were extensively dialyzed against 5 mM bis-tris propane-\nHepes, pH 7, in 10% (v/v) glycerol and stored at 80 °C.\nRaf 621p—Raf 621p is 15 amino acids long (LPKINR-\nSASpEPSLHR), corresponding to amino acids 613– 627 of Raf 1\n(15). Serine 621 is phosphorylated. The sequence highlighted in\nbold mimics the mode I recognition site of 14-3-3 proteins (16).\nIsolation of Oocyte Membranes—Membrane fractions were\nobtained according to De Jong et al. with minor modifications\n(27). 40 control or KAT1-expressing oocytes were homoge-\nnized in 800 l of 20 mM Tris-HCl, pH 7.4, containing 5 mM\nMgCl2, 5 mM NaH2PO4, 1 mM EDTA, 80 mM sucrose, 1 mM\nphenylmethylsulfonyl fluoride, 5 g/ml leupeptin, and 5 g/ml\npepstatin and centrifuged two times for 10 min at 3000  g at\n4 °C to remove yolk proteins. Membranes were pelleted by cen-\ntrifugation for 30 min at 14,000  g at 4 °C and resuspended in\n20 mM Tris-HCl, pH 7.4, containing 1 mM EDTA, 1 mM phen-\nylmethylsulfonyl fluoride, 5 g/ml leupeptin and 5 g/ml pep-\nstatin, and 20% (v/v) glycerol.\nSDS-PAGE and Overlay Assay—SDS-PAGE was performed\nas described by Laemmli (28) in a Mini Protean apparatus\n(Bio-Rad). The system used for expression of GF14-6 pro-\nteins produces a glutathione S-transferase-fused 14-3-3 con-\ntaining a cAMP-dependent protein kinase phosphorylation\nsite and a thrombin site between the two polypeptides. The\n32P-labeled GF14-6 was obtained as described (24). Specific\nactivity of the labeled protein was 3 MBq/mg. The overlay\nassay was carried out according to Ref. 24 with minor mod-\nifications. Briefly, 20 g of membrane fractions from KAT1-\nexpressing Xenopus oocytes were subjected to SDS-PAGE\nand blotted onto nitrocellulose membrane, using a semidry\nLKB apparatus (2 h, 0.8 mA cm2). The membrane was\nblocked with 5% fatty acid-free milk in 25 mM Hepes-OH, 75\nmM KCl, 5 mM MgCl2, 1 mM dithiothreitol, 0.1 mM EDTA,\n0.05% Tween 20, pH 7.5 (buffer H) and then incubated over-\nnight at 4 °C in the same buffer containing 2% fatty acid-free\nmilk, 3 g of 32P-labeled GF14-6 (corresponding to 9 kBq/\n14-3-3 Proteins Activate KAT1 Channel\nml). After incubation, the membrane was washed three times\nwith buffer H, dried, and subjected to autoradiography at\n80 °C. The overlay experiment was performed three times,\nand similar results were obtained.\nRESULTS AND DISCUSSION\nRecombinant 14-3-3 Proteins Increase the Activity of KAT1\nChannels Expressed in Xenopus Oocytes—Fig. 1A shows the\ncurrent traces recorded from KAT1 channels expressed in a\nXenopus oocyte. Clamping the oocyte with a voltage step pro-\ntocol ranging from 60 mV (holding potential) to 150 mV,\nelicited the slowly activating, inward K currents IKAT1 typical\nof this channel (5). The measured current values were stable\nover time; when monitored over a period of 30 min they var-\nied by 1% (not shown).\nTo examine the effect of 14-3-3 proteins on KAT1 currents\nthe same oocyte was removed from the voltage clamp apparatus\nand injected with 46 nl of a buffer containing 4 mg/ml of puri-\nfied recombinant 14-3-3 proteins from maize GF14-6 (final\nconcentration in the oocyte 1 M). The oocyte was kept 10\nmin at room temperature to recover from the injection and to\nallow 14-3-3 diffusion and challenged with the same voltage\nprotocol used prior to injection. Fig. 1A shows a strong increase\nin KAT1 current after 14-3-3 injection. The steady state cur-\nrent/voltage (I/V)2 relationship of the currents from Fig. 1A are\nshown in 1B. Fig. 1C shows the mean I/V relationship from 10\noocytes from three different batches. The increase in current\ninduced by 14-3-3 is voltage dependent; at 110 mV the cur-\nrent is increased by 200% (control, 0.36  0.02 A; 14-3-3,\n1.1  0.24 A), whereas at 140 mV is increased by 100%\n(control, 5.1  0.7 A; 14-3-3, 10.3  2.3 A).\nTo test the specificity of the 14-3-3 effect on IKAT1 we per-\nformed several control experiments, summarized in Fig. 2. The\neffect of injection of 14-3-3 on IKAT1, at the reference voltage of\n120 mV, is compared with that of heat-denatured 14-3-3 pro-\nteins, buffer only, and 14-3-3 proteins bearing the K56E muta-\ntion. This mutation, lysine 56 replaced by glutamic acid, is\nknown to disrupt the interaction with target proteins, as shown\nwith the plant proton pump (25). In our experiments, K56E\n14-3-3 proteins as well as heat-denatured wild-type proteins\nand resuspension buffer did not enhance IKAT1, indicating that\nthe effect observed after 14-3-3 injection is specific and not\nrelated to injection artifacts.\nBecause 14-3-3 proteins have been reported to modulate the\nendogenous Cl channels in Xenopus oocytes (29), we tested\nwhether the increase in inward current was entirely due to a\nstimulation of KAT1 channels. Fig. 3 shows that addition of 5\nmM Cs, a well known blocker of KAT1 channels (5) with no\nknown effect on Cl channels, completely blocked the inward\ncurrent after 14-3-3 stimulation. This shows that the stimu-\nlated current is KAT1 specific and is not due to a nonspecific\neffect on endogenous channels of the oocyte.\n14-3-3 Proteins Activate KAT1 Channel\nother hand, it has also been shown\nthat 14-3-3 proteins regulate the\ngating of animal and plant chan-\nnels by shifting the activation\ncurve, that is the plot of open\nprobability over voltage (20, 32).\nTo discriminate between these\ntwo possible modes of action, it was\nnecessary to check whether the\nmaximal value of IKAT1 was affected\nby the 14-3-3 proteins. Therefore\nwe had to measure the current of the\nfully activated channel, in the case of\nKAT1 at about 200 mV. Such neg-\native voltages can be deleterious for\nthe stability of the membrane; fur-\nthermore, they can trigger random\nactivation of endogenous currents.\nBecause the speed of IKAT1 activa-\ntion increases at negative voltages\n(33), we made use of a protocol\n(shown in Fig. 4A) that progres-\nsively reduces the length of the volt-\nage steps at negative potentials; in\nthis case we obtained all the essen-\ntial kinetic information without any\nunnecessary long and potentially\ndeleterious polarization of\nthe\nmembrane. Fig. 4B shows a control\nexperiment in which the full activa-\ntion of KAT1 was achieved. The\namplitude of the tail current from\nthe last preconditioning voltage\n(196 mV) is the same as that from\nthe preconditioning\nvoltage of\n186 mV. This indicates that KAT1\nchannels had reached full open\nprobability at 186 mV and that\nour protocol that steps at 200 mV\nserves the purpose of measuring the\ncurrent at full activation.\n14-3-3 Proteins Activate KAT1 Channels by Shifting the Acti-\nvation Curve—KAT1 is a voltage-gated channel, and its open\nprobability strictly depends on voltage. The increase in current\nobserved after injection of the 14-3-3 proteins can, in principle,\nresult from a change in conductance or from a change in open\nprobability.\nAn increase in conductance due to an increase in the number\nof active channels at the plasma membrane is a means by which\n14-3-3 proteins regulate ion channels in plants (19) and animal\ncells; in particular, for animal channels it has been shown that\n14-3-3 proteins modulate protein trafficking (30, 31). On the\n14-3-3 Proteins Activate KAT1 Channel\nability curve (V1⁄2) of155 mV and for the inverse slope factor\ncoefficient (s) of13 mV. The rather negative V1⁄2 value was in\nthe range reported previously for KAT1 in oocytes (33, 34).\nAfter 14-3-3 protein injection, the V1⁄2 shifted by 12 mV to 143\nmV and the inverse slope factor coefficient was 14 mV. The\nmaximal current value obtained by the Boltzmann fit was 3.57\nA in both cases, again confirming that 14-3-3 did not affect the\nmaximal conductance but only the open probability.\nAveraging normalized open probability curves (Fig. 4F) (n \n15 oocytes from seven independent experiments) shows that\n14-3-3 shifted the V1⁄2 by 11 mV, from 146 to 135 mV while\ns was unchanged (13 mV). The increase in current observed\nin response to 14-3-3 protein injection can hence be explained\nby a positive shift of the activation curve of the channel.\nThe findings that 14-3-3 proteins act directly on a channel\nprotein by altering its voltage dependence is in agreement with\nthat reported for the native inward rectifier from radicle proto-\nplasts (20). Also in this case, an elevation of 14-3-3 protein\nconcentration on the cytosolic side of the membrane resulted in\na positive shift in the activation curve of the channel by 9 mV.\nTo further test whether a positive shift of 11 mV in the\nchannel open probability is sufficient to explain the reported\nincrease in KAT1 current in response to elevation of 14-3-3\nproteins, we used the aforementioned Boltzmann parameters\nV1⁄2 and s to reconstruct the measured current/voltage relation.\nAs shown in Equation 2, the cell current (I) is\nI  i  N  Po\nwhere i is the single channel current amplitude of KAT1, N the\nnumber of channels in the membrane, and Po the open proba-\nbility. The curves in Fig. 1C were calculated using a unitary\nKAT1 channel conductance of 10 pS (36) with a reversal voltage\nat 18 mV and the respective values V1⁄2 and s from the Boltz-\nmann fit reported in Fig. 4F. The curves were jointly fitted to the\nmean current/voltage relations obtained before and after injec-\ntion of 14-3-3 proteins into oocytes (Fig. 1C); only the parame-\nter N was used as a fitting variable. A very good agreement\nbetween data and calculation was obtained with an N of 1.3 106\nchannels. This means that the entire increase in current\nobserved in response to 14-3-3 injection can be explained on\nthe basis of a shift in the voltage dependence; an increase in the\nnumber of channels as a consequence of a 14-3-3 effect on\nchannel trafficking was not needed to reproduce the data.\nWe then checked whether the shift in the activation curve\nwas accompanied by a change in channel kinetics. As expected,\nfollowing 14-3-3 treatment IKAT1 activation was accelerated.\nFig. 5A shows as an example current traces recorded at 130\nmV before and after injection of 14-3-3. Traces have been nor-\nmalized to show, qualitatively, current acceleration due to\n14-3-3. Fig. 5B shows the time constant of activation (\non) plot-\nted against voltage. The \non were estimated by fitting a single\nexponential function to current activation kinetics. At 130\nmV mean \non (n  6) was 486  39 and 363  18 ms for control\nand 14-3-3s, respectively. No apparent effect was observed on\nthe \noff, obtained from the deactivation kinetics. The accelera-\ntion of activation kinetics together with the positive shift of the\nactivation curve confirms that 14-3-3 proteins act by modifying\nthe kinetic properties of IKAT1.\n14-3-3 Proteins Act on KAT1 in a Membrane-delimited Way—\nTo test whether KAT1 is modulated by 14-3-3 proteins inde-\npendently of regulatory cellular pathways, we recorded IKAT1\nfrom inside-out macropatches. In this way we could access the\ncytosolic side of the channel and at the same time remove the\nchannel from the cytosolic environment. In excised patches\nIKAT1 undergoes a fast run down process, which shifts the volt-\nage dependence of the channel (37). Run down of the KAT1\nchannel could be recovered by the addition of 3 mM ATP to the\nbath solution as described by Hoshi (37, 38). In this condition\nwe were able to record KAT1 macrocurrents that were stable\nfor the length of the experiment (several minutes) and allowed\ntesting of the effect of 14-3-3 proteins (1 M final concentra-\ntion). Fig. 6A shows that addition of 14-3-3 increased IKAT1, as\n14-3-3 Proteins Activate KAT1 Channel\nalso shown by the steady state I/V relationship in Fig. 6B. A\nsimilar stimulation was observed in three of four patches; the\nmean I/V relationship of the positive responding patches is\nreported in Fig. 6C.\nThe occurrence of a run down process in excised patches\nalready indicates that KAT1 is regulated by cytoplasmic factors.\nTang and Hoshi (37) reported that a protein kinase A-mediated\nphosphorylation is responsible for the shift in the voltage-de-\npendent gating observed on patch excision. In contrast, non-\nspecific alkaline phosphatase accelerated the run down time\ncourse. Among others, we can probably add 14-3-3 proteins as\nfactors that seem to modulate the channel in a membrane-de-\nlimited pathway, given that the channel is in its active phospho-\nrylated form.\nEndogenous 14-3-3 Modulate KAT1 Channels—Working\nwith KAT1 channels expressed in oocytes we observed that the\nposition of the activation curve along the voltage axis can vary\ngreatly between experiments. The measured values of V1⁄2 var-\nied in the range 150 to 120 mV. This variability is suggestive\nof an endogenous regulation of the channel, and therefore we\ntested the possibility that endogenous 14-3-3 proteins could\nmodulate KAT1.\nTo examine this possibility we compared currents from\nKAT1-expressing oocytes before (control) and after the injec-\ntion of peptide Raf 621p (Fig. 7A). This phosphorylated peptide\nmimics the mode I recognition site of 14-3-3 proteins (16).\nWhen provided in large excess, it presumably titrates the\nendogenous 14-3-3 (see the use of this peptide in Refs. 15, 20,\n39). The injection of this peptide strongly decreased the level of\nIKAT1 in oocytes (70%  6 at150 mV, n  4); in control\nexperiments (data not shown) heat-inactivated Raf 621p\ndecreased the current at 150 mV of 38%  4 (n  4). This is\npresumably because it is not possible to efficiently inactivate\nthis short peptide by heating. As is apparent from the single\nexperiment and the mean I/V curves of Fig. 7, B and C, the\npeptide acts by shifting the voltage dependence of the channel\nto more negative values. For this reason it is difficult to reach\nfull activation of the channel in the Raf 621p-injected oocytes.\nOnly in one experiment (not shown) was it possible to measure\nthe current at full activation, and the measured shift in the\nactivation curve was 12 mV. We can conclude that endoge-\nnous 14-3-3s modulate KAT1 channel in oocytes and presum-\nably influence the position of the activation curve of this chan-\nnel. This finding could partly explain the strong run down\nphenomenon observed in excised patches of this channel.\nKAT1 Interacts in Vitro with the 14-3-3 Proteins—To test\nwhether the KAT1 protein interacts directly with the 14-3-3\nproteins, we have performed overlay experiments with radioac-\ntive 14-3-3s on total protein extract obtained from oocytes\ninjected with KAT1 mRNA and tested for KAT1 expression 3\ndays from injection. Membrane fraction from control or KAT1-\nexpressing oocytes was purified, subjected to SDS-PAGE, and\nblotted onto nitrocellulose membrane. Membranes were incu-\nbated with recombinant 32P-labeled 14-3-3 isoform from maize\nGF14-6. Autoradiography shows (Fig. 8) that the radioactive\nprobe in the crude membrane fraction purified from KAT1-\nexpressing oocytes recognizes only a single band of 80 kDa\n(lane 3), corresponding to the expected KAT1 molecular mass,\nwhereas no signal is detected in control membrane purified\nfrom oocytes not expressing KAT1 (lane 2). As expected, the\n14-3-3 probe efficiently recognizes the H-ATPase in the\nmaize plasma membrane fraction loaded as positive control\n(lane 1).\nControl experiments included the addition of Raf 621p pep-\ntide to inhibit the interaction of 14-3-3s with KAT1 and the\nsubstitution of wild-type 14-3-3 with the non-functional K56E\n14-3-3 mutant. In both cases we obtained no interaction (data\nnot shown).\nCONCLUSIONS\nRecombinant 14-3-3 proteins from maize activate KAT1\nchannels expressed in Xenopus oocytes. The 14-3-3-induced\nincrease in current is due to a positive shift in the voltage\ndependence of the channel and not to an increase in the num-\nber of channels. This effect is disrupted by mutation K56E in the\ncanonical interaction site of maize 14-3-3s. Excised patch\nexperiments and in vitro data suggest a direct interaction\nbetween KAT1 and the 14-3-3 proteins. Endogenous 14-3-3\nfrom Xenopus oocytes can control the activation curve of this\nchannel and represent a possible source of variability in this\nwell known expression system for ion channels.\nPhysiological Considerations—KAT1-like K channels mediate\nK uptake in many plant cells. The activity of these channels is\ncontrolled by the proton pump that establishes the negative\nmembrane voltage that opens the channels and thermodynam-\nically favors K uptake (40). It is also known that the two trans-\nport systems operate in concert, in as much as conditions cre-\nated by the proton pump, such as external acidification,\nenhance the activity of KAT1 channels and that the activity of\nthe channel is required to depolarize the negative potential that\nthermodynamically inhibits the proton pump (41). The finding\nthat 14-3-3 proteins, well known as proton pump activators,\ndirectly stimulate KAT1 channels constitutes strong molecular\nevidence that these two transporters can operate in concert in\nregulating K uptake.",
         "",
         "The Potassium Channel KAT1 Is Activated by Plant and Animal 14-3-3 Proteins*",
         "None"
        ],
        [
         "3",
         "00034b38a5273b760824f3223fa0d4fdb349d6df",
         "The natively disordered N-terminal 83-aa translocation (T) domain of E group nuclease colicins recruits OmpF to a colicin-receptor complex in the outer membrane (OM) as well as TolB in the periplasm of Escherichia coli, the latter triggering translocation of the toxin across the OM. We have identified the 16-residue TolB binding epitope in the natively disordered T-domain of the nuclease colicin E9 (ColE9) and solved the crystal structure of the complex. ColE9 folds into a distorted hairpin within a canyon of the six-bladed β-propeller of TolB, using two tryptophans to bolt the toxin to the canyon floor and numerous intramolecular hydrogen bonds to stabilize the bound conformation. This mode of binding enables colicin side chains to hydrogen-bond TolB residues in and around the channel that runs through the β-propeller and that constitutes the binding site of peptidoglycan-associated lipoprotein (Pal). Pal is a globular binding partner of TolB, and their association is known to be important for OM integrity. The structure is therefore consistent with translocation models wherein the colicin disrupts the TolB–Pal complex causing local instability of the OM as a prelude to toxin import. Intriguingly, Ca2+ ions, which bind within the β-propeller channel and switch the surface electrostatics from negative to positive, are needed for the negatively charged T-domain to bind TolB with an affinity equivalent to that of Pal and competitively displace it. Our study demonstrates that natively disordered proteins can compete with globular proteins for binding to folded scaffolds but that this can require cofactors such as metal ions to offset unfavorable interactions.",
         "S. Loftus,D. Walker,M. Maté,D. Bonsor,R. James,G. Moore,C. Kleanthous",
         "S. Loftus,D. Walker,M. Maté,D. Bonsor,R. James,G. Moore,C. Kleanthous",
         "https://www.pnas.org/doi/pdf/10.1073/pnas.0603433103",
         "The natively disordered N-terminal 83-aa translocation (T) domain\nof E group nuclease colicins recruits OmpF to a colicin-receptor\ncomplex in the outer membrane (OM) as well as TolB in the\nperiplasm of Escherichia coli, the latter triggering translocation of\nthe toxin across the OM. We have identiﬁed the 16-residue TolB\nbinding epitope in the natively disordered T-domain of the nucle-\nase colicin E9 (ColE9) and solved the crystal structure of the\ncomplex. ColE9 folds into a distorted hairpin within a canyon of\nthe six-bladed -propeller of TolB, using two tryptophans to bolt\nthe toxin to the canyon ﬂoor and numerous intramolecular hydro-\ngen bonds to stabilize the bound conformation. This mode of\nbinding enables colicin side chains to hydrogen-bond TolB residues\nin and around the channel that runs through the -propeller and\nthat constitutes the binding site of peptidoglycan-associated li-\npoprotein (Pal). Pal is a globular binding partner of TolB, and their\nassociation is known to be important for OM integrity. The struc-\nture is therefore consistent with translocation models wherein the\ncolicin disrupts the TolB–Pal complex causing local instability of the\nOM as a prelude to toxin import. Intriguingly, Ca2 ions, which bind\nwithin the -propeller channel and switch the surface electrostatics\nfrom negative to positive, are needed for the negatively charged\nT-domain to bind TolB with an afﬁnity equivalent to that of Pal and\ncompetitively displace it. Our study demonstrates that natively\ndisordered proteins can compete with globular proteins for bind-\ning to folded scaffolds but that this can require cofactors such as\nmetal ions to offset unfavorable interactions.",
         "",
         "proteins contain stretches of 30 aa that are natively (intrin-\nsically) disordered (1), many with functions yet to be defined.\nNative disorder in proteins occurs in regions of low sequence\ncomplexity, typically with a high glycine content and a preva-\nlence of charged or polar residues and few bulky hydrophobic\namino acids (2, 3). Their composition precludes the formation of\na hydrophobic core or a stable three-dimensional fold, the\nprotein instead remaining partially or completely unstructured\nunder physiological conditions. The extent of native disorder in\na protein can vary. For example, the entire interaction domain\nof the activator for thyroid hormone retinoid receptor is natively\ndisordered (4), whereas native disorder is restricted to the\nN-terminal region of the eukaryotic translation initiation factor\neIF4E (5).\nNatively disordered states are important in many biological\nand disease processes; including, transcription, translation, cell\nintracellular signaling, host–\ncycle regulation, endocytosis,\npathogen interactions, and cancer. For the most part, the role of\nnatively disordered proteins is to bind other macromolecules\n(nucleic acids, membranes, and proteins) often expediting the\nassembly of larger molecular complexes. As studies of natively\ndisordered proteins have accumulated, so the biological advan-\ntages inherent to such states have begun to emerge (2, 6). (i)\nNatively unfolded proteins undergo disorder–order transitions\non binding to folded scaffolds that have characteristic thermo-\ndynamic signatures, where the significant entropic penalties\nassociated with binding-induced folding are counterbalanced by\na favorable enthalpy resulting in weak but specific complexation.\n(ii) Natively unfolded proteins can contain many linear binding\nepitopes allowing a relatively small domain to engage in multiple\nassociations that if recapitulated by a folded protein would\nrequire a much larger polypeptide chain (7). (iii) ‘‘Fly-casting’’\nfacilitates recruitment of binding partners as natively unfolded\nproteins have a larger Stokes radius than a similarly sized\nglobular protein (8). Through the study of the cellular uptake\nmechanism of colicins, we identify a functional paradigm for a\nnatively disordered state that we term ‘‘competitive recruit-\nment,’’ in which the protein folds on binding its target in the\nprocess competing with a prefolded globular protein.\nColicins are plasmid-encoded, SOS-induced protein antibiot-\nics that specifically target Escherichia coli cells (9), with most\nGram-negative microorganisms producing such bacteriocins.\nTypically killing cells by depolarizing the inner membrane (IM)\nor degrading nucleic acid in the cytoplasm, colicins first bind an\nouter membrane (OM) receptor before translocating into the\ncell via Tol or Ton proteins within the periplasm (10). We have\nbeen investigating the 60-kDa nuclease colicins E2–E9 that bind\nto the vitamin B12 receptor BtuB in the OM and translocate to\nthe cytoplasm via the Tol system (11). E colicins share a high\ndegree of sequence conservation (90%) in the regions involved\nin receptor binding and translocation (460 aa), organized as a\ncentral coiled-coil receptor binding domain and an N-terminal\ntranslocation (T) domain, with the cytotoxic RNase or DNase\nlocated at the C terminus (12, 13). The N-terminal 83 aa of an\nE colicin T-domain are natively disordered as shown by three\nobservations. First, this region is predicted to be disordered by\npredictive algorithms such as PONDR (14), Foldindex (15), and\nDISOPRED2 (1). Second, the crystal structure of ColE3 failed\nto resolve electron density for this region (13). Third, hetero-\nnuclear NMR studies of the homologous ColE9 have demon-\nstrated this region to be unstructured and highly flexible (16, 17).\nWe showed recently that the natively disordered T-domain of\nE colicins recruits the porin OmpF in the OM to a preformed,\nhigh-affinity colicin–BtuB complex (18). It is thought that the\nT-domain is then lowered into the periplasm through the lumen\nof OmpF from where it must locate the 45-kDa translocation\nportal TolB to trigger cell entry (19–21). In the present study, we\nset out to determine the molecular basis for TolB recruitment by\nColE9 in the periplasm.\nResults\nResidues 32– 47 in the Natively Unfolded T-Domain of ColE9 Constitute\nthe TolB Binding Epitope. A ‘‘TolB box’’ (35DGSGW39) has pre-\nviously been described that is part of the natively unfolded\nT-domain where mutations (of non-glycine residues) abolish\ncolicin toxicity (21, 22). More recently, the potential TolB\nbinding site has been extended by mutagenesis and NMR\nexperiments (17, 23), but it is still not clear whether the site is a\ncontiguous or discontiguous epitope or whether regions outside\nthe natively disordered domain are involved. Two complemen-\ntary approaches were taken to delineate precisely the binding\nsite in ColE9. The first involved photocrosslinking using the\nheterobifunctional crosslinker 4-(N-maleimido)benzophenone\n(MBP) (24). MBP was attached to unique cysteines that replaced\nresidues in the disordered T-domain (Arg-7, Ser-15, Thr-25,\nVal-29, Ser-34, Gly-38, Ser-40, Glu-42, Trp-46, Gly-51, Trp-56,\nAsn-64, Ser-75, and Ser-82) and irradiated with UV light when\nmixed with TolB in vitro. MBP-labeled ColE9 mutants could be\nreadily crosslinked to TolB, but surprisingly these were distrib-\nuted on either side of the previously identified TolB box\nsequence, appearing instead to form a photocrosslink footprint\nthat spanned residues 34–46 (see Fig. 5a, which is published as\nsupporting information on the PNAS web site).\nThe second approach utilized deletion analysis, in combina-\ntion with gel-filtration chromatography and isothermal titration\ncalorimetry (ITC) as assays for TolB binding (see Materials and\nMethods for details). From these studies, we deduced that (i) the\nfirst 61 aa of the disordered T-domain are sufficient for stoichi-\nometric TolB binding; (ii) of the four deletions that were\ngenerated (T31– 61, T1– 44, T1– 49, and T1–54), only one (T1– 44) did\nnot bind TolB (data not shown); and (iii) two synthetic peptides,\nTpep30 –50 and Tpep32– 47, aimed at narrowing down the binding site\non ColE9 yet further, bound TolB with the same equilibrium\ndissociation constant as intact colicin (Kd  1 M) and with very\nsimilar thermodynamic parameters. ITC data for Tpep32– 47 bind-\ning TolB are shown in Fig. 5b and compared with wild-type\nColE9 in Table 1. The data show that complex formation is\nenthalpically driven and has a negative entropy, which is typical\nof binding-induced disorder–order transitions of natively disor-\ndered proteins. Critically, this sequence approximates the region\nof the colicin where MBP photocrosslinking to TolB was lost.\nHence, the TolB binding site on the enzymatic ColE9 is con-\ntained within a linear 16-residue epitope spanning residues\n32–47.\nStructure of the ColE9 Tpep32– 47–TolB Complex. The structure of E.\ncoli TolB has been described (25, 26), but as yet no high-\nresolution structure for this or any other Tol or Ton protein\nbound to a bacteriocin has been reported. To understand how\nColE9 binds TolB in the periplasm, the structure of the ColE9\nTpep32– 47–TolB complex was solved to a resolution of 2.0 Å (for\ndata refinement statistics, see Table 2 and Supporting Materials\nand Methods, which are published as supporting information on\nthe PNAS web site). The overall structure of the complex is\nshown in Fig. 1a, with the electron density of the bound ColE9\nTpep32– 47 peptide shown in Fig. 1b. TolB is composed of two\ndomains, an N-terminal  domain (residues 34–165) and a\nC-terminal, six-bladed -propeller domain (residues 166–431).\nColE9 Tpep32– 47 binds to the ‘‘top’’ side of the -propeller\ndomain, distal to the N-terminal domain, with 15 of the 16\nresidues (32–46) contacting TolB. Peptide binding does not\nresult in any major structural changes to TolB, the rms deviation\nfor C atoms relative to the native structure of TolB is 0.64 Å,\nand the accessible surface area that is buried on complex\nformation is 1,408 Å2. As with most -propellers, a narrow\nchannel runs through the domain, which in the case of TolB\ncontains two Ca2 ions (Fig. 1a); calcium had been included in\nthe crystallization conditions. Only three TolB residues are\ninvolved in binding the Ca2 ions (the side chains of Asp-337 and\nAsp-339 and the carbonyl of Val-340), with water molecules\noccupying the remaining coordination sites. Although neither\nCa2 ion is in direct contact with ColE9 Tpep32– 47, they never-\ntheless influence colicin binding, a point we return to below.\nThe colicin binds within a canyon in the -propeller domain\nblocking the central channel (Fig. 1c). Searches of the Dali server\n(27) failed to identify structures similar to that of ColE9 Tpep32–47,\nwhich adopts a distorted hairpin conformation. The peptide is\nanchored to the TolB canyon floor by the indole side chains of\nthe two tryptophans (Trp-39 and Trp-46) with additional hy-\ndrophobic contacts made by Pro-45 and Ala-33 (Fig. 1c). The gap\nseparating the hydrophobic docking sites for the tryptophans can\nonly accommodate three residues in extended conformation. As\nsix amino acids separate the two tryptophans, the chain becomes\ncompressed, the resulting turn pushing colicin residues toward\nthe channel at the base of the canyon. Comparing the protein–\nprotein interaction of the ColE9 Tpep32– 47–TolB complex with\nother -propeller complexes suggests that its architecture is\nnovel (28). -propellers usually bind other proteins along their\nexternal surfaces, whereas the central cavity is generally where\nenzyme prosthetic groups are located. ColE9 Tpep32– 47 binding is\na combination of these binding modes, contacting both the\nexternal surface and the central cavity of the -propeller.\nAnother unusual feature of the complex is that relatively few\ndirect hydrogen bonds stabilize the ColE9 Tpep32– 47–TolB com-\nplex. Discounting solvating waters, the colicin forms 21 hydrogen\nbonds; only 5 are direct intermolecular hydrogen bonds, a\nfurther 4 involve a bridging water molecule, and 12 are intramo-\nlecular. The importance of intramolecular contacts is most\napparent around the TolB box sequence (35DGSGW39) of the\ncolicin T-domain (Fig. 2a). Only Trp-39 from this sequence motif\nHis-246. Glu-42 also contacts Thr-292 indirectly through a\nwater-mediated hydrogen bond. A network of water-mediated\ninteractions helps stabilize the orientation of colicin side chains\nwithin the channel; this includes TolB Ser-205 in the wall of the\nchannel, the backbone atoms of ColE9 Ser-41 and Glu-42, and\nthe side chain of Ser-40, which hydrogen-bonds the amide\nnitrogen of Glu-42.\nThe results of previous ColE9 T-domain mutagenesis data can\nnow be rationalized in the context of the TolB-ColE9 Tpep32– 47\ncomplex structure (21–23). Mutations fall into two categories,\nthose that abolish binding and those that result in weak binding\nto TolB. The first group includes Asp-35, Ser-37, Trp-39, Glu-42,\nand Trp-46, all of which the structure shows play critical roles in\nstabilizing the bound conformation of the colicin, either through\nhydrophobic interactions or by forming multiple hydrogen\nbonds. Mutations at these sites also render the colicin completely\ninactive as a toxin. The second group includes Ser-40, Ser-41,\nAsn-43, and Asn-44, of which all but Ser-41 are involved in\nforming single hydrogen bonds and where mutations of some\n(Ser-41 and Asn-43) retain partial biological activity.\nThe Natively Disordered ColE9 T-Domain Binds at the Pal Site on TolB.\nIn vivo crosslinking has established that TolB and Pal form a\ncomplex at the OM in the bacterial periplasm (29), where Pal is\nlipoyl tether. Using a\nlocalized by virtue of its N-terminal\nsuppressor screen, Lazzaroni and coworkers (30) identified a\nnumber of residues in TolB that potentially comprise a Pal\nbinding site. When mapped onto the structure of the ColE9\nTpep32– 47–TolB complex, it becomes clear that there is significant\noverlap with the ColE9 T-domain binding site (Fig. 3), which\nis involved in hydrogen bonding to residues in TolB, Asp-308 and\nGln-336, the latter mediated via a bridging water molecule to the\nbackbone carbonyl of the tryptophan. Instead, the TolB box\nforms a network of intramolecular hydrogen bonds that holds the\npeptide in its distinctive conformation within the confines of the\n-propeller canyon. Colicin E9 Asp-35, in particular, is critical in\nthis role because it forms three intramolecular hydrogen bonds,\ntwo to Ser-37 and one to Ser-40. Similarly, Ser-37 is hydrogen-\nbonded to the main-chain of Ser-40, whereas Asn-43 from\nfurther along the peptide forms a single hydrogen bond with the\namide nitrogen of Gly-38 (Fig. 2a). Finally, the side chain of\nAsn-44 hydrogen-bonds the backbone nitrogen of the N-\nterminal glycine, Gly-32, effectively cyclizing the peptide in the\ncombining site (Fig. 1c).\nA consequence of the turn between the two tryptophans in\nColE9 Tpep32– 47 is that Ser-40, Ser-41, and Glu-42 project into the\nchannel at the base of the canyon and it is here that the majority\nof hydrogen bonds with TolB are formed (Fig. 2b). Ser-41\nhydrogen bonds to TolB Glu-293, whereas Glu-42 forms hydro-\ngen bonds with three TolB residues, Ser-205, Ala-249, and\nincludes residues within the TolB -propeller channel that form\nhydrogen bonds with the colicin (for example, His-246 and\nThr-292). However, there are no biochemical or structural data\nshowing that the Pal and E colicin binding sites on TolB overlap,\nand thus we investigated this in more detail using mutagenesis.\nTo evaluate the impact of any TolB mutations on the ability to\nbind Pal, the soluble 13-kDa periplasmic domain of E. coli Pal\n(residues 65–173) was overexpressed and purified (see Materials\nand Methods). Far-UV circular dichroism spectroscopy showed\nthat the protein contained 40% -helix and 15% -sheet,\nwhich is in good agreement with the x-ray structure of E. coli Pal\n(PDB entry 1OAP). Initially, we used analytical gel-filtration\nchromatography to confirm the domain bound TolB in a 1:1\ncomplex (data not shown). ITC established the Kd for the\ncomplex as 47 nM with binding being enthalpically and entropi-\ncally favorable (Table 1), which contrasts with the energetics of\nthe disordered ColE9 T-domain binding to TolB under the same\nconditions.\nA TolB Thr292Ile mutation was isolated in the study of Ray\net al. (30) that had a large effect in the phenotypic screens that\nwere used. We generated this mutation as well as an alanine\nmutant at this position. We also substituted His-246 for alanine.\nMutant TolB proteins were assayed for ColE9 T-domain and Pal\nbinding in vitro by ITC, and bacterial strains carrying these\nmutations were tested for their susceptibility toward ColE9 and\nSDS (see Fig. 6, which is published as supporting information on\nthe PNAS web site). Growth in the presence of 2% SDS provides\na convenient assay for OM integrity and hence TolB function in\nvivo. Our phenotypic assays made use of an E. coli K-12 tolB\nstrain that is unable to grow in the presence of SDS and is\nresistant to enzymatic colicins. The strain was transformed with\nan arabinose-inducible plasmid encoding wild-type and mutant\nTolBs targeted to the periplasm by the gIII signal sequence.\nBoth His246Ala and Thr292Ile TolB-expressing bacteria were\ncompletely resistant to the colicin and showed no toxin binding,\nwhereas Thr292Ala TolB showed 100-fold reduced binding and\ndiminished colicin toxicity relative to the wild-type control (Fig.\n6). The TolB mutations had an even greater impact on Pal\nbinding because none were able to bind Pal in vitro (Fig. 6), which\nis unlikely to be due to misfolding of the mutants because all of\nthe purified proteins had far-UV CD spectra identical to wild-\ntype TolB (data not shown). Moreover, unlike wild-type TolB-\ncontaining E. coli, the three mutant strains showed little growth\nCa2 ions modulate the binding of the ColE9 T-domain to TolB. (a)\nFig. 4.\nMBP-photocrosslinking of ColE9 Trp56Cys T-domain is inhibited by Pal (2 M)\nin the absence of Ca2 ions. TolB and ColE9 (2 M) were crosslinked as\ndescribed in Materials and Methods. The addition of 1 mM Ca2 ions abolished\nthe ability of Pal to inhibit ColE9 T-domain crosslinking to TolB. The arrow\nindicates the migration position of the crosslinked adduct. The ﬁgure also\nhighlights how the electrostatic potentials of the TolB -propeller canyon are\ninﬂuenced by calcium ions. (b) The surface is negatively charged in the absence\nof Ca2 ions weakening the afﬁnity for ColE9 T-domain. (c) Ca2 binding\nwithin the channel results in a more positively charged surface, which binds\nColE9 Tpep32– 47 10-fold more tightly (see text for details). Bound ColE9\nTpep32– 47 peptide (shown as yellow ribbon in b and c) has a net charge of 2\nat neutral pH due to Asp-35 and Glu-42.\nin the presence of 2% SDS (Fig. 6) demonstrating that all had\ncompromised TolB function in vivo. We conclude that the\nbinding sites for ColE9 and Pal on TolB must overlap to a\nsignificant degree and that, as anticipated, disruption of the\nTol–Pal interaction destabilizes the bacterial OM.\nCa2 Ions Enhance the Association of the Natively Disordered T-\nDomain of ColE9 with the TolB -Propeller. The data in Fig. 3 show\nhow TolB-dependent colicins associate at the Pal site, supporting\na competitive binding mode for these bacteriocins. With the\nrealization that the -propeller channel of TolB binds two Ca2\nions, we investigated whether calcium (or magnesium) could\ninfluence the binding of ColE9 T-domain (Tpep30 –50) and Pal for\nTolB by ITC. Because of buffermetal ion compatibility issues,\nwe opted for Hepes buffer at 20°C, pH 7.5, 50 mM NaCl, and\ncontaining either 5 mM EDTA or 1 mM CaCl2: the measured\nKds for the TolB–Pal complex were 27 and 90 nM, respectively,\nwhereas those for TolB-ColE9 T-domain were 900 and 84 nM,\nrespectively (data not shown). Ca2 ions therefore weaken Pal\nbinding by 3-fold but enhance ColE9 T-domain binding by\n10-fold, with similar changes seen for Mg2 ions. The ability of\nthe colicin T-domain to compete directly with Pal for binding to\nTolB was shown by MBP-photocrosslinking experiments. When\nPal was present in the incubations, T-domain could only be\ncrosslinked to TolB in the presence of Ca2 ions (Fig. 4a). We\nconclude that ColE9 T-domain can compete with Pal for binding\nto TolB but that this requires Ca2 or Mg2 ions to be bound\nwithin the -propeller channel, which render their affinities for\nTolB essentially equivalent.\nDiscussion\nColicin Recruitment of the Translocation Portal TolB Requires a\nNatively Unfolded N Terminus. The 16-residue TolB binding site in\nColE9 has been shown by NMR to be unstructured and flexible\nin solution but also to contain regions of reduced mobility\nrelative to other parts of the disordered T-domain that take the\nform of local hydrophobic clusters centering on the tryptophan\nside chains of Trp-39 and Trp-46 (16, 17). Similar clustering is\nseen around bulky hydrophobic residues in urea-denatured\nstates of proteins (31, 32). NMR has further demonstrated that\nassociation of the T-domain with TolB perturbs these hydro-\nphobic clusters, with sequences flanking these regions remaining\nunaffected (16, 33). The present structure now shows how the\ntwo tryptophans act as the main attachment points of the colicin\non the TolB surface. With the exception of the C-terminal\nglycine, every residue of the ColE9 T-domain peptide is involved\nin stabilizing the bound conformation, most by intra- or inter-\nmolecular hydrogen bonds (Fig. 2). Consequently, the underly-\ning thermodynamics of binding reflect not just binding-induced\nfolding within the TolB canyon but also disruption of the\nhydrophobic clusters in water. Although binding is weak, specific\nrecognition of TolB by the natively disordered T-domain is\nemphasized by the fact that no crosslinking could be detected\nwithin the binding site, whereas adjacent sequences were readily\ncrosslinked (Fig. 5a), indicating that MBP-labeling likely abol-\nished binding to TolB.\nBecause the soluble periplasmic domain of Pal is a globular\nprotein, the question arises as to why the colicin T-domain\nshould be natively unfolded before binding TolB. This require-\nment for native disorder is undoubtedly a consequence of the\ncolicin cellular uptake mechanism because the narrow lumen of\nan OmpF monomer is thought to be the entry point for the\nunfolded polypeptide into the periplasm before translocation\nacross the OM (19). Thereafter, association with TolB would be\nexpedited because OmpF is known to be closely associated with\nTol proteins (34). The role of native disorder in colicin trans-\nlocation is further illustrated by the pore-forming colicin N,\nwhich uses OmpF both as a receptor and translocator to the\nperiplasm where the toxin binds TolA. The first 90 aa of ColN\nare natively disordered, of which 27 residues comprise the TolA\nbinding site (35, 36). Native disorder is therefore a mechanistic\nprerequisite for Tol-dependent colicin translocation, allowing\nthe toxin to penetrate the OM via trimeric porins and contact Tol\nproteins in the periplasm.\nColicins Disrupt Host Protein–Protein Interactions in the Periplasm.\nTol and Ton proteins are essential periplasmic systems in\nGram-negative bacteria that have been subverted by group A\n(such as ColN and ColE2-E9) and group B (such as ColB and\nColIa) colicins, respectively, for translocation across the OM.\nTolAQR are orthologues of TonBExbBExbD; both are IM\nsystems coupled to the proton-motive force, and each (TolA and\nTonB) traverse the periplasm. The physiological role of the Ton\ncomplex is to energize the OM for active transport of ligands\nthrough nutrient receptors, such as FepA (37), whereas that of\nTol\nis less well defined but involves maintenance of outer-\nenvelope integrity. It is thought that group B colicins disrupt\nprotein–protein interactions in the periplasm of the host because\nthey possess a ‘‘TonB box’’ sequence at their N terminus\nequivalent to a sequence at the C terminus of OM nutrient\nreceptors that signals to TonB its liganded status (37). The\npresent work shows that such disruption extends to the group A\ncolicin ColE9 where the disordered T-domain associates at the\nPal site on TolB. The TonB box of group B colicins forms a\ncontinuous -sheet with the C-terminal domain of TonB, most\nlikely equivalent to that of a ligand-bound OM receptor (38, 39).\nIn contrast, ColE9 adopts a hairpin conformation in the confines\nof the -propeller domain of TolB. We conclude that both Ton-\nand Tol-dependent colicins interfere with host protein–protein\ninteractions but in very different ways. The outcome is ostensibly\nthe same, however, because each results in the disruption of a\nprotein–protein complex near the OM allowing the toxin to tap\ninto the proton-motive force across the IM via proteins that\ntraverse the periplasm.\nTol-Dependent Colicin Translocation Across the OM. TolB and Pal\nform a subcomplex that connects essential OM proteins such as\nLpp and OmpA to the IM via the TolAQR complex (40).\nDisruption of any of these associations through mutation causes\ninstability of the OM, with the cells becoming periplasmically\nleaky and hypersensitive to detergents and antibiotics (41). Such\nmutations also display characteristic tol phenotypes whereby\nthey are tolerant of colicin intoxication (42), the same phenotype\nexhibited by the TolB mutations generated in this study (Fig. 6).\nBe´ne´detti and coworkers (43, 44) have shown that colicin\nT-domain sequences secreted directly into the periplasm (via the\nsec pathway) generate the same pleiotropic effects as tol muta-\ntions and that this results in decreased in vivo crosslinking\nbetween TolB and Pal. This finding suggested that Tol-\ndependent colicins might perturb host protein–protein interac-\ntions to cause local disruption of the OM and so expedite their\nentry into the periplasm (45). Our study provides the first\nstructural and biophysical data in support of such a mechanism.\nThe colicin thus bound to TolB might then be able to traverse the\ndestabilized membrane by ‘‘hitching a ride’’ on the energized\nTolAQR complex, which is known to bind TolB at its N-terminal\n domain (46).\nCompetitive Recruitment and the Role of Divalent Cations. Compe-\ntition between natively disordered proteins for a common scaf-\nfold has been reported previously. The transcriptional-adapter\nzinc finger-1 (TAZ1) domain of the transcriptional regulator\nCBP is bound by disordered domains of HIF1a and CITED2,\nwhich fold around TAZ1 (reviewed in ref. 6). To our knowledge,\nthere have been no previous reports of a natively disordered\nprotein competing with a globular protein for binding to a folded\nscaffold. The degree of overlap awaits structural resolution of\nthe TolB–Pal complex but is likely to be substantial given the\ndramatic in vitro and in vivo effects of the single TolB -propeller\nmutations on both ColE9 and Pal binding. Competition exper-\niments reinforce this view because the ColE9 T-domain can\ndislodge Pal from its complex with TolB but only in the presence\nof Ca2 ions (Fig. 4a), which increase its binding affinity for TolB\nby 10-fold. Ca2 ions switch the electrostatic surface potential\nof the -propeller channel from negative to positive, favoring\nbinding of the negatively charged ColE9 T-domain peptide (pI \n3.7) (Fig. 4 b and c) while having minor impact on the binding\nof Pal, which has no net charge at neutral pH.\nThese observations raise the question of whether the effects of\ndivalent cations are physiologically relevant. Because the func-\ntion of TolB has yet to be established, this remains unclear, but\nwe note that Ca2 and Mg2 ions are known to be concentrated\nin the periplasm relative to the cytoplasm and external environ-\nment (47, 48), implying that alkaline earth metal ions are bound\nto TolB under physiological conditions. We speculate that these\nmetal sites are involved in native TolB function or stability, their\npresence having the secondary effect of expediting its compet-\nitive recruitment by the natively disordered T-domain of a\ntranslocating colicin.\nMaterials and Methods\nBacterial Strains and Plasmids. Plasmid pRJ379 encoding TolB\nwith a C-terminal His6-tag but lacking its 23-aa signal sequence\nis described in ref. 26. All other plasmids were constructed by\nstandard methods (for further details, see Supporting Materials\nand Methods). E. coli strain JM83 was used as a general host\nstrain for cloning, and BL21 (DE3) was used for protein\nexpression. Cultures were grown in LB broth or on LB agar\nplates and supplemented with ampicillin at 100 gml1 or\nkanamycin at 50 gml1, when required.\nProtein Expression and Purification. Proteins were overexpressed\nfrom BL21 (DE3) and purified by nickel affinity chromatogra-\nphy and gel filtration. Synthetic peptides were obtained from\nPepceuticals (Nottingham, United Kingdom) (for further de-\ntails, see Supporting Materials and Methods).\nLabeling and Crosslinking. N-His6ColE9 (16 M) in 50 mM\nMops3 M GnHCl (pH 7.0) was incubated for 30 min at room\ntemperature in the presence of 50 M DTT. MBP, from a stock\nsolution of 10 mM in DMSO, was added to a final concentration\nof 120 M, mixed by pipetting, and immediately quenched by the\naddition of 240 M DTT. The labeled proteins were dialyzed\novernight against 50 mM Mops100 mM NaCl (pH 7.0).\nCrosslinking reactions were performed in 0.5-ml Eppendorf\ntubes. MBP-labeled N-His6ColE9 (1 M) and TolB (4 M) were\nmixed and placed directly on a transilluminator for 10 min, and\nthe products were run on an SDS10% PAGE gel and visualized\nby Coomassie staining.\nCrystallization and Structure Determination. A 1:1 TolB–ColE9\nTpep32– 47 complex (10 mgml1 in water) was prepared by adding\nequimolar amounts of both components and used for crystalli-\nzation trials without further purification. Crystals of the TolB–\nColE9 Tpep32– 47 complex were obtained by the hanging-drop\nvapor-diffusion method. One microliter of TolB–Tpep32– 47 com-\nplex was mixed with the same volume of 24% polyethylene glycol\nmonomethyl ether 500080 mM CaCl2100 mM Hepes (pH 7.5)\nand equilibrated against a reservoir of 500 l containing this\nsolution. Crystals were flash-cooled by using paratone as a\ncryoprotectant, and a data set was collected at 100 K on\nbeamline ID23 at the European Synchrotron Radiation Facility\n(Grenoble, France). Crystals belonged to the space group P21\nand contained four molecules of the complex (TolBColE9;\nA–DE–H) in the asymmetric unit giving a Matthews coefficient\nof 2.1 for a 40.7% solvent content. The structure was solved by\nmolecular replacement with the program MOLREP and the\nstructure of TolB (PDB entry 1C5K) as the search probe. For\ndetails of the structure determination, see Supporting Materials\nand Methods. Molecular graphics images were produced by\nusing the UCSF Chimera package (Resource for Biocompu-\nting, Visualization, and Informatics, University of California,\nSan Francisco, CA).\nWe thank Andrew Leech and Berni Strongitharm (York Technology\nFacility) for expert assistance with many of the biophysical measure-\nments conducted in this study and Nick Housden and Jennifer Potts\n(University of York) for comments on the manuscript. We acknowledge\nthe European Synchrotron Radiation Facility for provision of synchro-\ntron radiation facilities and thank Didier Nurizzo for assistance in using\nbeamline ID23. This work was supported by the Wellcome Trust and the\nBiotechnology and Biological Sciences Research Council of the United\nKingdom.\n45. Lazzaroni, J. C., Dubuisson, J. F. & Vianney, A. (2002) Biochimie 84, 391–397.\n46. Walburger, A., Lazdunski, C. & Corda, Y. (2002) Mol. Microbiol. 44, 695–708.\n47. Jones, H. E., Holland, I. B. & Campbell, A. K. (2002) Cell Calcium 32, 182–192.\n48. Nikaido, H. (2003) Mol. Biol. Rev. 67, 593–656.",
         "",
         "",
         "downloading",
         "The natively disordered N-terminal 83-aa translocation (T) domain\nof E group nuclease colicins recruits OmpF to a colicin-receptor\ncomplex in the outer membrane (OM) as well as TolB in the\nperiplasm of Escherichia coli, the latter triggering translocation of\nthe toxin across the OM. We have identiﬁed the 16-residue TolB\nbinding epitope in the natively disordered T-domain of the nucle-\nase colicin E9 (ColE9) and solved the crystal structure of the\ncomplex. ColE9 folds into a distorted hairpin within a canyon of\nthe six-bladed -propeller of TolB, using two tryptophans to bolt\nthe toxin to the canyon ﬂoor and numerous intramolecular hydro-\ngen bonds to stabilize the bound conformation. This mode of\nbinding enables colicin side chains to hydrogen-bond TolB residues\nin and around the channel that runs through the -propeller and\nthat constitutes the binding site of peptidoglycan-associated li-\npoprotein (Pal). Pal is a globular binding partner of TolB, and their\nassociation is known to be important for OM integrity. The struc-\nture is therefore consistent with translocation models wherein the\ncolicin disrupts the TolB–Pal complex causing local instability of the\nOM as a prelude to toxin import. Intriguingly, Ca2 ions, which bind\nwithin the -propeller channel and switch the surface electrostatics\nfrom negative to positive, are needed for the negatively charged\nT-domain to bind TolB with an afﬁnity equivalent to that of Pal and\ncompetitively displace it. Our study demonstrates that natively\ndisordered proteins can compete with globular proteins for bind-\ning to folded scaffolds but that this can require cofactors such as\nmetal ions to offset unfavorable interactions.\nproteins contain stretches of 30 aa that are natively (intrin-\nsically) disordered (1), many with functions yet to be defined.\nNative disorder in proteins occurs in regions of low sequence\ncomplexity, typically with a high glycine content and a preva-\nlence of charged or polar residues and few bulky hydrophobic\namino acids (2, 3). Their composition precludes the formation of\na hydrophobic core or a stable three-dimensional fold, the\nprotein instead remaining partially or completely unstructured\nunder physiological conditions. The extent of native disorder in\na protein can vary. For example, the entire interaction domain\nof the activator for thyroid hormone retinoid receptor is natively\ndisordered (4), whereas native disorder is restricted to the\nN-terminal region of the eukaryotic translation initiation factor\neIF4E (5).\nNatively disordered states are important in many biological\nand disease processes; including, transcription, translation, cell\nintracellular signaling, host–\ncycle regulation, endocytosis,\npathogen interactions, and cancer. For the most part, the role of\nnatively disordered proteins is to bind other macromolecules\n(nucleic acids, membranes, and proteins) often expediting the\nassembly of larger molecular complexes. As studies of natively\ndisordered proteins have accumulated, so the biological advan-\ntages inherent to such states have begun to emerge (2, 6). (i)\nNatively unfolded proteins undergo disorder–order transitions\non binding to folded scaffolds that have characteristic thermo-\ndynamic signatures, where the significant entropic penalties\nassociated with binding-induced folding are counterbalanced by\na favorable enthalpy resulting in weak but specific complexation.\n(ii) Natively unfolded proteins can contain many linear binding\nepitopes allowing a relatively small domain to engage in multiple\nassociations that if recapitulated by a folded protein would\nrequire a much larger polypeptide chain (7). (iii) ‘‘Fly-casting’’\nfacilitates recruitment of binding partners as natively unfolded\nproteins have a larger Stokes radius than a similarly sized\nglobular protein (8). Through the study of the cellular uptake\nmechanism of colicins, we identify a functional paradigm for a\nnatively disordered state that we term ‘‘competitive recruit-\nment,’’ in which the protein folds on binding its target in the\nprocess competing with a prefolded globular protein.\nColicins are plasmid-encoded, SOS-induced protein antibiot-\nics that specifically target Escherichia coli cells (9), with most\nGram-negative microorganisms producing such bacteriocins.\nTypically killing cells by depolarizing the inner membrane (IM)\nor degrading nucleic acid in the cytoplasm, colicins first bind an\nouter membrane (OM) receptor before translocating into the\ncell via Tol or Ton proteins within the periplasm (10). We have\nbeen investigating the 60-kDa nuclease colicins E2–E9 that bind\nto the vitamin B12 receptor BtuB in the OM and translocate to\nthe cytoplasm via the Tol system (11). E colicins share a high\ndegree of sequence conservation (90%) in the regions involved\nin receptor binding and translocation (460 aa), organized as a\ncentral coiled-coil receptor binding domain and an N-terminal\ntranslocation (T) domain, with the cytotoxic RNase or DNase\nlocated at the C terminus (12, 13). The N-terminal 83 aa of an\nE colicin T-domain are natively disordered as shown by three\nobservations. First, this region is predicted to be disordered by\npredictive algorithms such as PONDR (14), Foldindex (15), and\nDISOPRED2 (1). Second, the crystal structure of ColE3 failed\nto resolve electron density for this region (13). Third, hetero-\nnuclear NMR studies of the homologous ColE9 have demon-\nstrated this region to be unstructured and highly flexible (16, 17).\nWe showed recently that the natively disordered T-domain of\nE colicins recruits the porin OmpF in the OM to a preformed,\nhigh-affinity colicin–BtuB complex (18). It is thought that the\nT-domain is then lowered into the periplasm through the lumen\nof OmpF from where it must locate the 45-kDa translocation\nportal TolB to trigger cell entry (19–21). In the present study, we\nset out to determine the molecular basis for TolB recruitment by\nColE9 in the periplasm.\nResults\nResidues 32– 47 in the Natively Unfolded T-Domain of ColE9 Constitute\nthe TolB Binding Epitope. A ‘‘TolB box’’ (35DGSGW39) has pre-\nviously been described that is part of the natively unfolded\nT-domain where mutations (of non-glycine residues) abolish\ncolicin toxicity (21, 22). More recently, the potential TolB\nbinding site has been extended by mutagenesis and NMR\nexperiments (17, 23), but it is still not clear whether the site is a\ncontiguous or discontiguous epitope or whether regions outside\nthe natively disordered domain are involved. Two complemen-\ntary approaches were taken to delineate precisely the binding\nsite in ColE9. The first involved photocrosslinking using the\nheterobifunctional crosslinker 4-(N-maleimido)benzophenone\n(MBP) (24). MBP was attached to unique cysteines that replaced\nresidues in the disordered T-domain (Arg-7, Ser-15, Thr-25,\nVal-29, Ser-34, Gly-38, Ser-40, Glu-42, Trp-46, Gly-51, Trp-56,\nAsn-64, Ser-75, and Ser-82) and irradiated with UV light when\nmixed with TolB in vitro. MBP-labeled ColE9 mutants could be\nreadily crosslinked to TolB, but surprisingly these were distrib-\nuted on either side of the previously identified TolB box\nsequence, appearing instead to form a photocrosslink footprint\nthat spanned residues 34–46 (see Fig. 5a, which is published as\nsupporting information on the PNAS web site).\nThe second approach utilized deletion analysis, in combina-\ntion with gel-filtration chromatography and isothermal titration\ncalorimetry (ITC) as assays for TolB binding (see Materials and\nMethods for details). From these studies, we deduced that (i) the\nfirst 61 aa of the disordered T-domain are sufficient for stoichi-\nometric TolB binding; (ii) of the four deletions that were\ngenerated (T31– 61, T1– 44, T1– 49, and T1–54), only one (T1– 44) did\nnot bind TolB (data not shown); and (iii) two synthetic peptides,\nTpep30 –50 and Tpep32– 47, aimed at narrowing down the binding site\non ColE9 yet further, bound TolB with the same equilibrium\ndissociation constant as intact colicin (Kd  1 M) and with very\nsimilar thermodynamic parameters. ITC data for Tpep32– 47 bind-\ning TolB are shown in Fig. 5b and compared with wild-type\nColE9 in Table 1. The data show that complex formation is\nenthalpically driven and has a negative entropy, which is typical\nof binding-induced disorder–order transitions of natively disor-\ndered proteins. Critically, this sequence approximates the region\nof the colicin where MBP photocrosslinking to TolB was lost.\nHence, the TolB binding site on the enzymatic ColE9 is con-\ntained within a linear 16-residue epitope spanning residues\n32–47.\nStructure of the ColE9 Tpep32– 47–TolB Complex. The structure of E.\ncoli TolB has been described (25, 26), but as yet no high-\nresolution structure for this or any other Tol or Ton protein\nbound to a bacteriocin has been reported. To understand how\nColE9 binds TolB in the periplasm, the structure of the ColE9\nTpep32– 47–TolB complex was solved to a resolution of 2.0 Å (for\ndata refinement statistics, see Table 2 and Supporting Materials\nand Methods, which are published as supporting information on\nthe PNAS web site). The overall structure of the complex is\nshown in Fig. 1a, with the electron density of the bound ColE9\nTpep32– 47 peptide shown in Fig. 1b. TolB is composed of two\ndomains, an N-terminal  domain (residues 34–165) and a\nC-terminal, six-bladed -propeller domain (residues 166–431).\nColE9 Tpep32– 47 binds to the ‘‘top’’ side of the -propeller\ndomain, distal to the N-terminal domain, with 15 of the 16\nresidues (32–46) contacting TolB. Peptide binding does not\nresult in any major structural changes to TolB, the rms deviation\nfor C atoms relative to the native structure of TolB is 0.64 Å,\nand the accessible surface area that is buried on complex\nformation is 1,408 Å2. As with most -propellers, a narrow\nchannel runs through the domain, which in the case of TolB\ncontains two Ca2 ions (Fig. 1a); calcium had been included in\nthe crystallization conditions. Only three TolB residues are\ninvolved in binding the Ca2 ions (the side chains of Asp-337 and\nAsp-339 and the carbonyl of Val-340), with water molecules\noccupying the remaining coordination sites. Although neither\nCa2 ion is in direct contact with ColE9 Tpep32– 47, they never-\ntheless influence colicin binding, a point we return to below.\nThe colicin binds within a canyon in the -propeller domain\nblocking the central channel (Fig. 1c). Searches of the Dali server\n(27) failed to identify structures similar to that of ColE9 Tpep32–47,\nwhich adopts a distorted hairpin conformation. The peptide is\nanchored to the TolB canyon floor by the indole side chains of\nthe two tryptophans (Trp-39 and Trp-46) with additional hy-\ndrophobic contacts made by Pro-45 and Ala-33 (Fig. 1c). The gap\nseparating the hydrophobic docking sites for the tryptophans can\nonly accommodate three residues in extended conformation. As\nsix amino acids separate the two tryptophans, the chain becomes\ncompressed, the resulting turn pushing colicin residues toward\nthe channel at the base of the canyon. Comparing the protein–\nprotein interaction of the ColE9 Tpep32– 47–TolB complex with\nother -propeller complexes suggests that its architecture is\nnovel (28). -propellers usually bind other proteins along their\nexternal surfaces, whereas the central cavity is generally where\nenzyme prosthetic groups are located. ColE9 Tpep32– 47 binding is\na combination of these binding modes, contacting both the\nexternal surface and the central cavity of the -propeller.\nAnother unusual feature of the complex is that relatively few\ndirect hydrogen bonds stabilize the ColE9 Tpep32– 47–TolB com-\nplex. Discounting solvating waters, the colicin forms 21 hydrogen\nbonds; only 5 are direct intermolecular hydrogen bonds, a\nfurther 4 involve a bridging water molecule, and 12 are intramo-\nlecular. The importance of intramolecular contacts is most\napparent around the TolB box sequence (35DGSGW39) of the\ncolicin T-domain (Fig. 2a). Only Trp-39 from this sequence motif\nHis-246. Glu-42 also contacts Thr-292 indirectly through a\nwater-mediated hydrogen bond. A network of water-mediated\ninteractions helps stabilize the orientation of colicin side chains\nwithin the channel; this includes TolB Ser-205 in the wall of the\nchannel, the backbone atoms of ColE9 Ser-41 and Glu-42, and\nthe side chain of Ser-40, which hydrogen-bonds the amide\nnitrogen of Glu-42.\nThe results of previous ColE9 T-domain mutagenesis data can\nnow be rationalized in the context of the TolB-ColE9 Tpep32– 47\ncomplex structure (21–23). Mutations fall into two categories,\nthose that abolish binding and those that result in weak binding\nto TolB. The first group includes Asp-35, Ser-37, Trp-39, Glu-42,\nand Trp-46, all of which the structure shows play critical roles in\nstabilizing the bound conformation of the colicin, either through\nhydrophobic interactions or by forming multiple hydrogen\nbonds. Mutations at these sites also render the colicin completely\ninactive as a toxin. The second group includes Ser-40, Ser-41,\nAsn-43, and Asn-44, of which all but Ser-41 are involved in\nforming single hydrogen bonds and where mutations of some\n(Ser-41 and Asn-43) retain partial biological activity.\nThe Natively Disordered ColE9 T-Domain Binds at the Pal Site on TolB.\nIn vivo crosslinking has established that TolB and Pal form a\ncomplex at the OM in the bacterial periplasm (29), where Pal is\nlipoyl tether. Using a\nlocalized by virtue of its N-terminal\nsuppressor screen, Lazzaroni and coworkers (30) identified a\nnumber of residues in TolB that potentially comprise a Pal\nbinding site. When mapped onto the structure of the ColE9\nTpep32– 47–TolB complex, it becomes clear that there is significant\noverlap with the ColE9 T-domain binding site (Fig. 3), which\nis involved in hydrogen bonding to residues in TolB, Asp-308 and\nGln-336, the latter mediated via a bridging water molecule to the\nbackbone carbonyl of the tryptophan. Instead, the TolB box\nforms a network of intramolecular hydrogen bonds that holds the\npeptide in its distinctive conformation within the confines of the\n-propeller canyon. Colicin E9 Asp-35, in particular, is critical in\nthis role because it forms three intramolecular hydrogen bonds,\ntwo to Ser-37 and one to Ser-40. Similarly, Ser-37 is hydrogen-\nbonded to the main-chain of Ser-40, whereas Asn-43 from\nfurther along the peptide forms a single hydrogen bond with the\namide nitrogen of Gly-38 (Fig. 2a). Finally, the side chain of\nAsn-44 hydrogen-bonds the backbone nitrogen of the N-\nterminal glycine, Gly-32, effectively cyclizing the peptide in the\ncombining site (Fig. 1c).\nA consequence of the turn between the two tryptophans in\nColE9 Tpep32– 47 is that Ser-40, Ser-41, and Glu-42 project into the\nchannel at the base of the canyon and it is here that the majority\nof hydrogen bonds with TolB are formed (Fig. 2b). Ser-41\nhydrogen bonds to TolB Glu-293, whereas Glu-42 forms hydro-\ngen bonds with three TolB residues, Ser-205, Ala-249, and\nincludes residues within the TolB -propeller channel that form\nhydrogen bonds with the colicin (for example, His-246 and\nThr-292). However, there are no biochemical or structural data\nshowing that the Pal and E colicin binding sites on TolB overlap,\nand thus we investigated this in more detail using mutagenesis.\nTo evaluate the impact of any TolB mutations on the ability to\nbind Pal, the soluble 13-kDa periplasmic domain of E. coli Pal\n(residues 65–173) was overexpressed and purified (see Materials\nand Methods). Far-UV circular dichroism spectroscopy showed\nthat the protein contained 40% -helix and 15% -sheet,\nwhich is in good agreement with the x-ray structure of E. coli Pal\n(PDB entry 1OAP). Initially, we used analytical gel-filtration\nchromatography to confirm the domain bound TolB in a 1:1\ncomplex (data not shown). ITC established the Kd for the\ncomplex as 47 nM with binding being enthalpically and entropi-\ncally favorable (Table 1), which contrasts with the energetics of\nthe disordered ColE9 T-domain binding to TolB under the same\nconditions.\nA TolB Thr292Ile mutation was isolated in the study of Ray\net al. (30) that had a large effect in the phenotypic screens that\nwere used. We generated this mutation as well as an alanine\nmutant at this position. We also substituted His-246 for alanine.\nMutant TolB proteins were assayed for ColE9 T-domain and Pal\nbinding in vitro by ITC, and bacterial strains carrying these\nmutations were tested for their susceptibility toward ColE9 and\nSDS (see Fig. 6, which is published as supporting information on\nthe PNAS web site). Growth in the presence of 2% SDS provides\na convenient assay for OM integrity and hence TolB function in\nvivo. Our phenotypic assays made use of an E. coli K-12 tolB\nstrain that is unable to grow in the presence of SDS and is\nresistant to enzymatic colicins. The strain was transformed with\nan arabinose-inducible plasmid encoding wild-type and mutant\nTolBs targeted to the periplasm by the gIII signal sequence.\nBoth His246Ala and Thr292Ile TolB-expressing bacteria were\ncompletely resistant to the colicin and showed no toxin binding,\nwhereas Thr292Ala TolB showed 100-fold reduced binding and\ndiminished colicin toxicity relative to the wild-type control (Fig.\n6). The TolB mutations had an even greater impact on Pal\nbinding because none were able to bind Pal in vitro (Fig. 6), which\nis unlikely to be due to misfolding of the mutants because all of\nthe purified proteins had far-UV CD spectra identical to wild-\ntype TolB (data not shown). Moreover, unlike wild-type TolB-\ncontaining E. coli, the three mutant strains showed little growth\nCa2 ions modulate the binding of the ColE9 T-domain to TolB. (a)\nFig. 4.\nMBP-photocrosslinking of ColE9 Trp56Cys T-domain is inhibited by Pal (2 M)\nin the absence of Ca2 ions. TolB and ColE9 (2 M) were crosslinked as\ndescribed in Materials and Methods. The addition of 1 mM Ca2 ions abolished\nthe ability of Pal to inhibit ColE9 T-domain crosslinking to TolB. The arrow\nindicates the migration position of the crosslinked adduct. The ﬁgure also\nhighlights how the electrostatic potentials of the TolB -propeller canyon are\ninﬂuenced by calcium ions. (b) The surface is negatively charged in the absence\nof Ca2 ions weakening the afﬁnity for ColE9 T-domain. (c) Ca2 binding\nwithin the channel results in a more positively charged surface, which binds\nColE9 Tpep32– 47 10-fold more tightly (see text for details). Bound ColE9\nTpep32– 47 peptide (shown as yellow ribbon in b and c) has a net charge of 2\nat neutral pH due to Asp-35 and Glu-42.\nin the presence of 2% SDS (Fig. 6) demonstrating that all had\ncompromised TolB function in vivo. We conclude that the\nbinding sites for ColE9 and Pal on TolB must overlap to a\nsignificant degree and that, as anticipated, disruption of the\nTol–Pal interaction destabilizes the bacterial OM.\nCa2 Ions Enhance the Association of the Natively Disordered T-\nDomain of ColE9 with the TolB -Propeller. The data in Fig. 3 show\nhow TolB-dependent colicins associate at the Pal site, supporting\na competitive binding mode for these bacteriocins. With the\nrealization that the -propeller channel of TolB binds two Ca2\nions, we investigated whether calcium (or magnesium) could\ninfluence the binding of ColE9 T-domain (Tpep30 –50) and Pal for\nTolB by ITC. Because of buffermetal ion compatibility issues,\nwe opted for Hepes buffer at 20°C, pH 7.5, 50 mM NaCl, and\ncontaining either 5 mM EDTA or 1 mM CaCl2: the measured\nKds for the TolB–Pal complex were 27 and 90 nM, respectively,\nwhereas those for TolB-ColE9 T-domain were 900 and 84 nM,\nrespectively (data not shown). Ca2 ions therefore weaken Pal\nbinding by 3-fold but enhance ColE9 T-domain binding by\n10-fold, with similar changes seen for Mg2 ions. The ability of\nthe colicin T-domain to compete directly with Pal for binding to\nTolB was shown by MBP-photocrosslinking experiments. When\nPal was present in the incubations, T-domain could only be\ncrosslinked to TolB in the presence of Ca2 ions (Fig. 4a). We\nconclude that ColE9 T-domain can compete with Pal for binding\nto TolB but that this requires Ca2 or Mg2 ions to be bound\nwithin the -propeller channel, which render their affinities for\nTolB essentially equivalent.\nDiscussion\nColicin Recruitment of the Translocation Portal TolB Requires a\nNatively Unfolded N Terminus. The 16-residue TolB binding site in\nColE9 has been shown by NMR to be unstructured and flexible\nin solution but also to contain regions of reduced mobility\nrelative to other parts of the disordered T-domain that take the\nform of local hydrophobic clusters centering on the tryptophan\nside chains of Trp-39 and Trp-46 (16, 17). Similar clustering is\nseen around bulky hydrophobic residues in urea-denatured\nstates of proteins (31, 32). NMR has further demonstrated that\nassociation of the T-domain with TolB perturbs these hydro-\nphobic clusters, with sequences flanking these regions remaining\nunaffected (16, 33). The present structure now shows how the\ntwo tryptophans act as the main attachment points of the colicin\non the TolB surface. With the exception of the C-terminal\nglycine, every residue of the ColE9 T-domain peptide is involved\nin stabilizing the bound conformation, most by intra- or inter-\nmolecular hydrogen bonds (Fig. 2). Consequently, the underly-\ning thermodynamics of binding reflect not just binding-induced\nfolding within the TolB canyon but also disruption of the\nhydrophobic clusters in water. Although binding is weak, specific\nrecognition of TolB by the natively disordered T-domain is\nemphasized by the fact that no crosslinking could be detected\nwithin the binding site, whereas adjacent sequences were readily\ncrosslinked (Fig. 5a), indicating that MBP-labeling likely abol-\nished binding to TolB.\nBecause the soluble periplasmic domain of Pal is a globular\nprotein, the question arises as to why the colicin T-domain\nshould be natively unfolded before binding TolB. This require-\nment for native disorder is undoubtedly a consequence of the\ncolicin cellular uptake mechanism because the narrow lumen of\nan OmpF monomer is thought to be the entry point for the\nunfolded polypeptide into the periplasm before translocation\nacross the OM (19). Thereafter, association with TolB would be\nexpedited because OmpF is known to be closely associated with\nTol proteins (34). The role of native disorder in colicin trans-\nlocation is further illustrated by the pore-forming colicin N,\nwhich uses OmpF both as a receptor and translocator to the\nperiplasm where the toxin binds TolA. The first 90 aa of ColN\nare natively disordered, of which 27 residues comprise the TolA\nbinding site (35, 36). Native disorder is therefore a mechanistic\nprerequisite for Tol-dependent colicin translocation, allowing\nthe toxin to penetrate the OM via trimeric porins and contact Tol\nproteins in the periplasm.\nColicins Disrupt Host Protein–Protein Interactions in the Periplasm.\nTol and Ton proteins are essential periplasmic systems in\nGram-negative bacteria that have been subverted by group A\n(such as ColN and ColE2-E9) and group B (such as ColB and\nColIa) colicins, respectively, for translocation across the OM.\nTolAQR are orthologues of TonBExbBExbD; both are IM\nsystems coupled to the proton-motive force, and each (TolA and\nTonB) traverse the periplasm. The physiological role of the Ton\ncomplex is to energize the OM for active transport of ligands\nthrough nutrient receptors, such as FepA (37), whereas that of\nTol\nis less well defined but involves maintenance of outer-\nenvelope integrity. It is thought that group B colicins disrupt\nprotein–protein interactions in the periplasm of the host because\nthey possess a ‘‘TonB box’’ sequence at their N terminus\nequivalent to a sequence at the C terminus of OM nutrient\nreceptors that signals to TonB its liganded status (37). The\npresent work shows that such disruption extends to the group A\ncolicin ColE9 where the disordered T-domain associates at the\nPal site on TolB. The TonB box of group B colicins forms a\ncontinuous -sheet with the C-terminal domain of TonB, most\nlikely equivalent to that of a ligand-bound OM receptor (38, 39).\nIn contrast, ColE9 adopts a hairpin conformation in the confines\nof the -propeller domain of TolB. We conclude that both Ton-\nand Tol-dependent colicins interfere with host protein–protein\ninteractions but in very different ways. The outcome is ostensibly\nthe same, however, because each results in the disruption of a\nprotein–protein complex near the OM allowing the toxin to tap\ninto the proton-motive force across the IM via proteins that\ntraverse the periplasm.\nTol-Dependent Colicin Translocation Across the OM. TolB and Pal\nform a subcomplex that connects essential OM proteins such as\nLpp and OmpA to the IM via the TolAQR complex (40).\nDisruption of any of these associations through mutation causes\ninstability of the OM, with the cells becoming periplasmically\nleaky and hypersensitive to detergents and antibiotics (41). Such\nmutations also display characteristic tol phenotypes whereby\nthey are tolerant of colicin intoxication (42), the same phenotype\nexhibited by the TolB mutations generated in this study (Fig. 6).\nBe´ne´detti and coworkers (43, 44) have shown that colicin\nT-domain sequences secreted directly into the periplasm (via the\nsec pathway) generate the same pleiotropic effects as tol muta-\ntions and that this results in decreased in vivo crosslinking\nbetween TolB and Pal. This finding suggested that Tol-\ndependent colicins might perturb host protein–protein interac-\ntions to cause local disruption of the OM and so expedite their\nentry into the periplasm (45). Our study provides the first\nstructural and biophysical data in support of such a mechanism.\nThe colicin thus bound to TolB might then be able to traverse the\ndestabilized membrane by ‘‘hitching a ride’’ on the energized\nTolAQR complex, which is known to bind TolB at its N-terminal\n domain (46).\nCompetitive Recruitment and the Role of Divalent Cations. Compe-\ntition between natively disordered proteins for a common scaf-\nfold has been reported previously. The transcriptional-adapter\nzinc finger-1 (TAZ1) domain of the transcriptional regulator\nCBP is bound by disordered domains of HIF1a and CITED2,\nwhich fold around TAZ1 (reviewed in ref. 6). To our knowledge,\nthere have been no previous reports of a natively disordered\nprotein competing with a globular protein for binding to a folded\nscaffold. The degree of overlap awaits structural resolution of\nthe TolB–Pal complex but is likely to be substantial given the\ndramatic in vitro and in vivo effects of the single TolB -propeller\nmutations on both ColE9 and Pal binding. Competition exper-\niments reinforce this view because the ColE9 T-domain can\ndislodge Pal from its complex with TolB but only in the presence\nof Ca2 ions (Fig. 4a), which increase its binding affinity for TolB\nby 10-fold. Ca2 ions switch the electrostatic surface potential\nof the -propeller channel from negative to positive, favoring\nbinding of the negatively charged ColE9 T-domain peptide (pI \n3.7) (Fig. 4 b and c) while having minor impact on the binding\nof Pal, which has no net charge at neutral pH.\nThese observations raise the question of whether the effects of\ndivalent cations are physiologically relevant. Because the func-\ntion of TolB has yet to be established, this remains unclear, but\nwe note that Ca2 and Mg2 ions are known to be concentrated\nin the periplasm relative to the cytoplasm and external environ-\nment (47, 48), implying that alkaline earth metal ions are bound\nto TolB under physiological conditions. We speculate that these\nmetal sites are involved in native TolB function or stability, their\npresence having the secondary effect of expediting its compet-\nitive recruitment by the natively disordered T-domain of a\ntranslocating colicin.\nMaterials and Methods\nBacterial Strains and Plasmids. Plasmid pRJ379 encoding TolB\nwith a C-terminal His6-tag but lacking its 23-aa signal sequence\nis described in ref. 26. All other plasmids were constructed by\nstandard methods (for further details, see Supporting Materials\nand Methods). E. coli strain JM83 was used as a general host\nstrain for cloning, and BL21 (DE3) was used for protein\nexpression. Cultures were grown in LB broth or on LB agar\nplates and supplemented with ampicillin at 100 gml1 or\nkanamycin at 50 gml1, when required.\nProtein Expression and Purification. Proteins were overexpressed\nfrom BL21 (DE3) and purified by nickel affinity chromatogra-\nphy and gel filtration. Synthetic peptides were obtained from\nPepceuticals (Nottingham, United Kingdom) (for further de-\ntails, see Supporting Materials and Methods).\nLabeling and Crosslinking. N-His6ColE9 (16 M) in 50 mM\nMops3 M GnHCl (pH 7.0) was incubated for 30 min at room\ntemperature in the presence of 50 M DTT. MBP, from a stock\nsolution of 10 mM in DMSO, was added to a final concentration\nof 120 M, mixed by pipetting, and immediately quenched by the\naddition of 240 M DTT. The labeled proteins were dialyzed\novernight against 50 mM Mops100 mM NaCl (pH 7.0).\nCrosslinking reactions were performed in 0.5-ml Eppendorf\ntubes. MBP-labeled N-His6ColE9 (1 M) and TolB (4 M) were\nmixed and placed directly on a transilluminator for 10 min, and\nthe products were run on an SDS10% PAGE gel and visualized\nby Coomassie staining.\nCrystallization and Structure Determination. A 1:1 TolB–ColE9\nTpep32– 47 complex (10 mgml1 in water) was prepared by adding\nequimolar amounts of both components and used for crystalli-\nzation trials without further purification. Crystals of the TolB–\nColE9 Tpep32– 47 complex were obtained by the hanging-drop\nvapor-diffusion method. One microliter of TolB–Tpep32– 47 com-\nplex was mixed with the same volume of 24% polyethylene glycol\nmonomethyl ether 500080 mM CaCl2100 mM Hepes (pH 7.5)\nand equilibrated against a reservoir of 500 l containing this\nsolution. Crystals were flash-cooled by using paratone as a\ncryoprotectant, and a data set was collected at 100 K on\nbeamline ID23 at the European Synchrotron Radiation Facility\n(Grenoble, France). Crystals belonged to the space group P21\nand contained four molecules of the complex (TolBColE9;\nA–DE–H) in the asymmetric unit giving a Matthews coefficient\nof 2.1 for a 40.7% solvent content. The structure was solved by\nmolecular replacement with the program MOLREP and the\nstructure of TolB (PDB entry 1C5K) as the search probe. For\ndetails of the structure determination, see Supporting Materials\nand Methods. Molecular graphics images were produced by\nusing the UCSF Chimera package (Resource for Biocompu-\nting, Visualization, and Informatics, University of California,\nSan Francisco, CA).\nWe thank Andrew Leech and Berni Strongitharm (York Technology\nFacility) for expert assistance with many of the biophysical measure-\nments conducted in this study and Nick Housden and Jennifer Potts\n(University of York) for comments on the manuscript. We acknowledge\nthe European Synchrotron Radiation Facility for provision of synchro-\ntron radiation facilities and thank Didier Nurizzo for assistance in using\nbeamline ID23. This work was supported by the Wellcome Trust and the\nBiotechnology and Biological Sciences Research Council of the United\nKingdom.\n45. Lazzaroni, J. C., Dubuisson, J. F. & Vianney, A. (2002) Biochimie 84, 391–397.\n46. Walburger, A., Lazdunski, C. & Corda, Y. (2002) Mol. Microbiol. 44, 695–708.\n47. Jones, H. E., Holland, I. B. & Campbell, A. K. (2002) Cell Calcium 32, 182–192.\n48. Nikaido, H. (2003) Mol. Biol. Rev. 67, 593–656.",
         "",
         "Competitive recruitment of the periplasmic translocation portal TolB by a natively disordered domain of colicin E9",
         "None"
        ],
        [
         "4",
         "00036a3493967932ac256f38c0c2419415f04feb",
         "Cyclophosphamide (CY) is a cytostatic agent that produces systemic toxicity especially on cells with high proliferative capacity, while polysaccharides from Angelica sinensis (AP) have been shown to increase the turnover of gastrointestinal mucosal and hemopoietic stem cells. It is not known whether AP has an effect on CY-induced cytotoxicity on bone marrow and gastrointestinal tract. In this study, we assessed the protective actions of AP on CY-induced leukopenia and proliferative arrest in the gastroduodenal mucosa in mice. Subcutaneous injection of CY (200 mg/kg) provoked dramatic decrease in white blood cell (WBC) count and number of blood vessels and proliferating cells in both the gastric and duodenal mucosae. Subcutaneous injection of AP significantly promoted the recovery from leukopenia and increased number of blood vessels and proliferating cells in both the gastric and duodenal tissues. Western blotting revealed that CY significantly down-regulated the protein expression of vascular endothelial growth factor (VEGF), c-Myc and ornithine decarboxylase (ODC) in gastric mucosae but had no effect on epidermal growth factor (EGF) expression. AP also reversed the dampening effect of CY on VEGF expression in the gastric mucosa. These data suggest that AP is a cytoprotective agent which can protect against the cytotoxicity of CY on hematopoietic and gastrointestinal tissues when the polysaccharide is co-administered with CY in cancer patients during treatment regimen.",
         "M. K. C. Hui,W. Wu,V. Shin,W. H. So,C. Cho",
         "M. K. C. Hui,W. Wu,V. Shin,W. H. So,C. Cho",
         "https://www.medsci.org/v03p0001.pdf",
         "The  major  side  effect  of  anticancer  drugs,  e.g.\ncyclophosphamide,  is  the  non-specific  cytostatic  action  on\nnormal  healthy\nthose  with  high\nproliferating capacity like the hematopoietic and GI tissues\n[1].    The  extensive  death  of  the  immune  cells  results  in\nleukopenia which severely weakens the immune system of\ncancer  patients  and  therefore  greatly  increases  the  chance\nof  disseminated  infections  which  could  be  fetal.      As  a\nresult,  drug-free  period  is  always  clinically  necessary  in\ncancer  patients  receiving  chemotherapy,  so  as  to  allow\ntheir immune systems to restore function [2].  On the other\nhand,  the  death  of  GI  cells  breaks  down  the  physical\ndefence  of  GI  system  in  the  host  who  will  become  more\nsusceptible  to  antigen  originated  from  GI  systems  and\ntherefore further  increases  death  rate  due  to  opportunistic\ninfection  [3].    In  addition,  emesis  due  to  the  release  of\nserotonin  from  enterochromaffin  cells  is  also  discouraging\nto cancer patients [4].  All of these are the main reasons for\ndiscontinuation of cancer chemotherapy, which lowers the\nchance of a successful and complete treatment regimen.",
         "this  study,  CY  produced  myelosuppression\nmanifested  as  leukopenia  (Fig.  1).    It  also  significantly\nreduced the blood supply and proliferating cell number in\nboth  the  gastric  and  duodenal  mucosae.    Subcutaneous\nadministration  of  AP  at  the  dose  of  5  mg/kg  daily\nsignificantly promoted the recovery rate of immune system\nin mice in a 14-day treatment (Fig. 1).  It also significantly\nincreased  the  number  of  blood  vessel  and  PCNA-positive\ncell  in  both  the  gastric  (Fig.  2A  and  2B  respectively)  and\nduodenal  tissues  (Fig.  3A  and  3B  respectively).    Dose-\ndependent effects were observed in general.  Western blot\nanalysis implicated the reduction by CY and normalization\nby  AP  of  blood  vessel  count  was  VEGF  dependent  in\ngastric tissue (Fig. 4A).  On the other hand, the decrease in\nin  gastric  mucosa  by  CY\nproliferating  cell  number\nto  be  c-Myc  and  ODC-\nadministration  was\ndependent (Fig 4B and 4C respectively).\nsuggest\nInt. J. Med. Sci. 2006, 3\nis  known  to\ncell  proliferation  in  which  mitogenic  stimulation  leads  to\nincreased expression and drives the cell cycle from G0 to G1\ninduce  the\n[27].\ntranscriptional activity of ODC gene [29], which is involved\nin  polyamine  synthesis.    It  is  likely  that  the  cell  cycle-\narresting action of CY in the stomach in this study was due\nto  the  activation  of  p53  and  therefore  leading  to  the\nAll  these\nsuppression  of  the  c-Myc/ODC  pathway.\nfindings could explain the mechanism of CY on repression\nof    cell  proliferation  in  the  gastroduodenal  mucosae.    In\ngeneral, dose-dependent effect on the reversal action of AP\non this suppressive effect could be observed.  Although our\nprevious  study  has  shown  that  ODC  was involved  in  AP-\ninduced normal gastric epithelial cell proliferation [10], the\nincreased  proliferating rate  in  this study  was  independent\nof\nthe\ncorresponding  protein  levels  in  Western  Blot  analysis.\nHowever  this  could  be  partly  contributed  to  the  effect  of\nAP  on  angiogenesis  that  increased  the  blood  supply  to\ntissues for growth and repairment.\nthe  c-Myc,  ODC  or  EGF  as\nindicated  by\nTo  conclude,  the  above  findings  not  only  provide  a\nfundamental  insight  into  the  mechanism  of  CY-induced\nsystemic  cytotoxicity,  particularly  in  the  gastrointestinal\nsystem,  but  also  propose  a  role  for  polysaccharides  from\nAngelica  sinensis  as  a  cytoprotective  agent  to  spare  the\nhemopoietic and gastrointestinal toxicities of CY. Whether\nor  not  the  present  study  can  be  translated  into  practical\nbenefits, warrants further investigation.\nAcknowledgments\nThis  study  is  supported  by  the  Research  Grants\nthe\n(HKU  7397/03M)  and\nCouncil  of  Hong  Kong\nUniversity of Hong Kong.\nConflict of interests\nThe authors have declared that no conflict of interest",
         "residue was boiled again with water for another four-hour\nperiod.  All extracts were finally pooled and mixed with a\nconcentrated  ethanol  solution  (final  concentration  75%\nv/v)  to  precipitate  the  polysaccharide-enriched  fraction.\nTwo  kinds  of  high  performance  liquid  chromatography\n(HPLC)  methods  including  the  high  performance  anion\nexchange\nchromatography,\nrespectively  [13],  were  employed  to  concentrate  and\ndetermine  the  molecular  size  of  the  polysaccharide-rich\nfraction.    The  molecular  sizes  of  polysaccharides  were\ndetermined  in  HPLC  (gel  filtration  column,  Biosep  SEC-\nS3000,  Phenomenex,  USA;  mobile  phase  0.15  mol/L  NaCl\nsolution;  detector  wavelength  220  nm)  combined  with  the\nphenol-sulfuric  acid  method  [14,  15].    The  amounts  of\nuronic acids and proteins were also determined [16, 17].\nfiltration\nThe  Angelica  polysaccharide  fraction  was  found  to\nconsist  of  5  main  polysaccharide  sub-fractions  with  the\nfollowing  moleculard  weights:  >670.00,  433.72,  167.55,\n82.10  and  15.54  kD  respectively.    The  total  extracted\nfraction  consisted  of  97%  carbohydrates  (about  30%  of\nthem uronic acids) and 3% proteins.  This polysaccharide-\nenriched fraction from Angelica sinensis (AP) was dissolved\nin  normal  saline  (0.9%,  w/v,  NaCl)  before  subcutaneous\ninjection to animals.\nExperimental animals and drug administration\nThis  study  was  conducted  with  the  consent  of  the\nCommittee  on  the  Use  of  Live  Animals  in  Teaching  and\nResearch of the University of Hong Kong.  Male ICR mice\n(weighing  25–30  g)  were  reared  on  a  standard  laboratory\ndiet (Ralston Purina, Chicago, Illinois, USA) and given tap\nwater  ad  libitum.    Mice  were  randomly  allocated  into  5\ntreatment  groups  (n  =  8  -  15  in  each  group)  which  were\nsubject  to  a  14-day  treatment.    Group  1  was  the  normal\nuntreated  control  (Nor)  while  groups  2  to  5  received  a\nsingle  dose  of  CY  200  mg/kg  daily  by  subcutaneous\ninjection  on  day  0  and  day  7.    In  addition,  group  2  (NS)\nmice received daily dose of normal saline while groups 3 to\n5 mice received daily dose of AP at 5 (AP5), 10 (AP10) or 25\n(AP25)  mg/kg,  respectively.    Mice  were  sacrificed  on  day\n14 and the gastric and duodenal tissues were collected for\nbiochemical and histological assessments.\nAssessment of white blood cell (WBC) count\nBlood samples were collected from the tail arteries on\nday 0, 4, 7, 11 and 14 to monitor the toxicity of CY on bone\nmarrow  by  measuring  WBC  number  in  the  peripheral\nblood.  Twenty microliters of blood was mixed with 380μl\nof Randolph’s solution. WBC counting was then performed\nimproved  Neubauer  hematocytometer\nby  using  an\n(Reichert, U.S.A.).\nAssessment of angiogenesis\nfactor  were  counted  with  Leica\nsections  were\nthe\nthen\nInt. J. Med. Sci. 2006, 3\nNS  group,  the  time  needed  for  WBC  number  to  recover\nback  to  normal  level  was  7  days.    Upon  administration  of\nAP 5mg/kg once daily, the WBC number could recover in\n5-day period (Fig. 1).\ntreatment  only  significantly  reversed\nInt. J. Med. Sci. 2006, 3\nexpression.    AP  treatment  did  not  have  any  effect  on  the\nexpression  of  c-Myc,  ODC  and  EGF  in  the  gastric  mucosa\n(Fig. 4B, 4C and 4D respectively).\nInt. J. Med. Sci. 2006, 3\n4.  Discussion",
         "",
         "",
         "downloading",
         "The  major  side  effect  of  anticancer  drugs,  e.g.\ncyclophosphamide,  is  the  non-specific  cytostatic  action  on\nnormal  healthy\nthose  with  high\nproliferating capacity like the hematopoietic and GI tissues\n[1].    The  extensive  death  of  the  immune  cells  results  in\nleukopenia which severely weakens the immune system of\ncancer  patients  and  therefore  greatly  increases  the  chance\nof  disseminated  infections  which  could  be  fetal.      As  a\nresult,  drug-free  period  is  always  clinically  necessary  in\ncancer  patients  receiving  chemotherapy,  so  as  to  allow\ntheir immune systems to restore function [2].  On the other\nhand,  the  death  of  GI  cells  breaks  down  the  physical\ndefence  of  GI  system  in  the  host  who  will  become  more\nsusceptible  to  antigen  originated  from  GI  systems  and\ntherefore further  increases  death  rate  due  to  opportunistic\ninfection  [3].    In  addition,  emesis  due  to  the  release  of\nserotonin  from  enterochromaffin  cells  is  also  discouraging\nto cancer patients [4].  All of these are the main reasons for\ndiscontinuation of cancer chemotherapy, which lowers the\nchance of a successful and complete treatment regimen.\nresidue was boiled again with water for another four-hour\nperiod.  All extracts were finally pooled and mixed with a\nconcentrated  ethanol  solution  (final  concentration  75%\nv/v)  to  precipitate  the  polysaccharide-enriched  fraction.\nTwo  kinds  of  high  performance  liquid  chromatography\n(HPLC)  methods  including  the  high  performance  anion\nexchange\nchromatography,\nrespectively  [13],  were  employed  to  concentrate  and\ndetermine  the  molecular  size  of  the  polysaccharide-rich\nfraction.    The  molecular  sizes  of  polysaccharides  were\ndetermined  in  HPLC  (gel  filtration  column,  Biosep  SEC-\nS3000,  Phenomenex,  USA;  mobile  phase  0.15  mol/L  NaCl\nsolution;  detector  wavelength  220  nm)  combined  with  the\nphenol-sulfuric  acid  method  [14,  15].    The  amounts  of\nuronic acids and proteins were also determined [16, 17].\nfiltration\nThe  Angelica  polysaccharide  fraction  was  found  to\nconsist  of  5  main  polysaccharide  sub-fractions  with  the\nfollowing  moleculard  weights:  >670.00,  433.72,  167.55,\n82.10  and  15.54  kD  respectively.    The  total  extracted\nfraction  consisted  of  97%  carbohydrates  (about  30%  of\nthem uronic acids) and 3% proteins.  This polysaccharide-\nenriched fraction from Angelica sinensis (AP) was dissolved\nin  normal  saline  (0.9%,  w/v,  NaCl)  before  subcutaneous\ninjection to animals.\nExperimental animals and drug administration\nThis  study  was  conducted  with  the  consent  of  the\nCommittee  on  the  Use  of  Live  Animals  in  Teaching  and\nResearch of the University of Hong Kong.  Male ICR mice\n(weighing  25–30  g)  were  reared  on  a  standard  laboratory\ndiet (Ralston Purina, Chicago, Illinois, USA) and given tap\nwater  ad  libitum.    Mice  were  randomly  allocated  into  5\ntreatment  groups  (n  =  8  -  15  in  each  group)  which  were\nsubject  to  a  14-day  treatment.    Group  1  was  the  normal\nuntreated  control  (Nor)  while  groups  2  to  5  received  a\nsingle  dose  of  CY  200  mg/kg  daily  by  subcutaneous\ninjection  on  day  0  and  day  7.    In  addition,  group  2  (NS)\nmice received daily dose of normal saline while groups 3 to\n5 mice received daily dose of AP at 5 (AP5), 10 (AP10) or 25\n(AP25)  mg/kg,  respectively.    Mice  were  sacrificed  on  day\n14 and the gastric and duodenal tissues were collected for\nbiochemical and histological assessments.\nAssessment of white blood cell (WBC) count\nBlood samples were collected from the tail arteries on\nday 0, 4, 7, 11 and 14 to monitor the toxicity of CY on bone\nmarrow  by  measuring  WBC  number  in  the  peripheral\nblood.  Twenty microliters of blood was mixed with 380μl\nof Randolph’s solution. WBC counting was then performed\nimproved  Neubauer  hematocytometer\nby  using  an\n(Reichert, U.S.A.).\nAssessment of angiogenesis\nfactor  were  counted  with  Leica\nsections  were\nthe\nthen\nInt. J. Med. Sci. 2006, 3\nNS  group,  the  time  needed  for  WBC  number  to  recover\nback  to  normal  level  was  7  days.    Upon  administration  of\nAP 5mg/kg once daily, the WBC number could recover in\n5-day period (Fig. 1).\ntreatment  only  significantly  reversed\nInt. J. Med. Sci. 2006, 3\nexpression.    AP  treatment  did  not  have  any  effect  on  the\nexpression  of  c-Myc,  ODC  and  EGF  in  the  gastric  mucosa\n(Fig. 4B, 4C and 4D respectively).\nInt. J. Med. Sci. 2006, 3\n4.  Discussion\nthis  study,  CY  produced  myelosuppression\nmanifested  as  leukopenia  (Fig.  1).    It  also  significantly\nreduced the blood supply and proliferating cell number in\nboth  the  gastric  and  duodenal  mucosae.    Subcutaneous\nadministration  of  AP  at  the  dose  of  5  mg/kg  daily\nsignificantly promoted the recovery rate of immune system\nin mice in a 14-day treatment (Fig. 1).  It also significantly\nincreased  the  number  of  blood  vessel  and  PCNA-positive\ncell  in  both  the  gastric  (Fig.  2A  and  2B  respectively)  and\nduodenal  tissues  (Fig.  3A  and  3B  respectively).    Dose-\ndependent effects were observed in general.  Western blot\nanalysis implicated the reduction by CY and normalization\nby  AP  of  blood  vessel  count  was  VEGF  dependent  in\ngastric tissue (Fig. 4A).  On the other hand, the decrease in\nin  gastric  mucosa  by  CY\nproliferating  cell  number\nto  be  c-Myc  and  ODC-\nadministration  was\ndependent (Fig 4B and 4C respectively).\nsuggest\nInt. J. Med. Sci. 2006, 3\nis  known  to\ncell  proliferation  in  which  mitogenic  stimulation  leads  to\nincreased expression and drives the cell cycle from G0 to G1\ninduce  the\n[27].\ntranscriptional activity of ODC gene [29], which is involved\nin  polyamine  synthesis.    It  is  likely  that  the  cell  cycle-\narresting action of CY in the stomach in this study was due\nto  the  activation  of  p53  and  therefore  leading  to  the\nAll  these\nsuppression  of  the  c-Myc/ODC  pathway.\nfindings could explain the mechanism of CY on repression\nof    cell  proliferation  in  the  gastroduodenal  mucosae.    In\ngeneral, dose-dependent effect on the reversal action of AP\non this suppressive effect could be observed.  Although our\nprevious  study  has  shown  that  ODC  was involved  in  AP-\ninduced normal gastric epithelial cell proliferation [10], the\nincreased  proliferating rate  in  this study  was  independent\nof\nthe\ncorresponding  protein  levels  in  Western  Blot  analysis.\nHowever  this  could  be  partly  contributed  to  the  effect  of\nAP  on  angiogenesis  that  increased  the  blood  supply  to\ntissues for growth and repairment.\nthe  c-Myc,  ODC  or  EGF  as\nindicated  by\nTo  conclude,  the  above  findings  not  only  provide  a\nfundamental  insight  into  the  mechanism  of  CY-induced\nsystemic  cytotoxicity,  particularly  in  the  gastrointestinal\nsystem,  but  also  propose  a  role  for  polysaccharides  from\nAngelica  sinensis  as  a  cytoprotective  agent  to  spare  the\nhemopoietic and gastrointestinal toxicities of CY. Whether\nor  not  the  present  study  can  be  translated  into  practical\nbenefits, warrants further investigation.\nAcknowledgments\nThis  study  is  supported  by  the  Research  Grants\nthe\n(HKU  7397/03M)  and\nCouncil  of  Hong  Kong\nUniversity of Hong Kong.\nConflict of interests\nThe authors have declared that no conflict of interest",
         "",
         "Polysaccharides from the root of Angelica sinensis protect bone marrow and gastrointestinal tissues against the cytotoxicity of cyclophosphamide in mice",
         "None"
        ],
        [
         "5",
         "0003825e87b4ccccdc75f7014d1d6917804adced",
         "Cellular oxalate, widely distributed in many plants, is implicated to play important roles in various functions and is also known to affect food qualities adversely in fruits and vegetables. How oxalate is regulated in plants is currently not well understood. Glycolate oxidase (GLO) has long been considered as an important player in oxalate accumulation in plants. To gain further insight into the biochemical and molecular mechanisms, the possible roles of GLO in the process were studied. Drastically different levels of oxalate could be achieved by treating rice with various nitrogen forms (nitrate versus ammonium). While nitrate stimulated oxalate accumulation, ammonium reduced its level. Such treatments resulted in similar pattern changes for some other related organic acids, such as glycolate, oxaloacetate, and malate. By feeding plants with exogenous glycolate it was possible almost completely to restore the ammonium-decreased oxalate level. Under the two treatments few differences were observed for GLO mRNA levels, protein levels, and in vitro activities. Both K(m) for glycolate/glyoxylate and K(i) for oxalate remained almost the same for GLO purified from either nitrate- or ammonium-fed leaves. A further in vivo study, with transgenic plants carrying an estradiol-inducible GLO antisense gene, showed that, while the estradiol-induced antisense expression remarkably reduced both GLO protein levels and activities, oxalate levels were not significantly altered in the estradiol-treated transgenic plants. Taken together, it is suggested that oxalate accumulation and regulation is independent of GLO in rice leaves.",
         "Hua-Wei Xu,X. Ji,Zheng-Hui He,Wei Shi,G. Zhu,Jian-Kang Niu,Bao-Sheng Li,Xinxiang Peng",
         "None",
         "https://academic.oup.com/jxb/article-pdf/57/9/1899/1715826/erj131.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://academic.oup.com/crawlprevention/governor?content=%2fjxb%2fOUP%2fRunPdfHandler%3fjournal%3djxb%26volume%3d57%26issue%3d9%26page%3d1899%26multimediaId%3d1715826",
         "None",
         "",
         "Oxalate accumulation and regulation is independent of glycolate oxidase in rice leaves.",
         "None"
        ],
        [
         "6",
         "000468a0bca628e7d42ebc516606d7a1874e1491",
         "A simultaneous embedding of two planar graphs G1 and G2 is a pair of drawings of G1\nand G2 such that each drawing is planar and each vertex common to G1 and G2 is repre-\nsented by the same point in both drawings. Unfortunately, if one wishes to visualize the\nedges of G1 and G2 as rectilinear segments (the so called geometric simultaneous em-\nbedding), not all pairs of graphs can be embedded simultaneously. Erten and Kobourov\n([4]), Brass et al. ([1]), and Geyer et al. ([7]) have shown that it is not always possible\nto embed simultaneously with straight-line edges a planar graph and a path, three paths,\nand two trees, respectively. On the other hand, if one permits that each edge of a graph\nis displayed as a different Jordan curve (the so called simultaneous embedding), then\nby the results of Pach and Wenger ([9]) any number of planar graphs can be embedded\nsimultaneously. Restricting the last constraints, one could permit that each edge is rep-\nresented by a Jordan curve, but could force edges common to more graphs to have the\nsame representation in the drawing of each graph (the so called simultaneous embed-\nding with ﬁxed edges). Di Giacomo and Liotta ([3]) showed that an outerplanar graph\nand a cycle can always be simultaneously embedded with ﬁxed edges, improving the\nresults in [4], where it is shown how to embed simultaneously with ﬁxed (“consistent”)\nedges a tree and a path. In [4] and [3] the problem of ﬁnding simultaneous embeddings\nwith ﬁxed edges of pairs of trees and of pairs of planar graphs is explicitly mentioned.\nIn this paper we improve the results on simultaneous embedding with ﬁxed edges\nof graphs, by showing that there exist two outerplanar graphs that cannot be simulta-\nneously embedded with ﬁxed edges (Section 3) and that a planar graph and a tree can\nalways be simultaneously embedded with ﬁxed edges (Section 4). Then in Section 5 we\ngive conclusions and suggest some open problems.",
         "Fabrizio Frati",
         "Fabrizio Frati",
         "https://link.springer.com/content/pdf/10.1007%2F978-3-540-70904-6_12.pdf",
         "A simultaneous embedding of two planar graphs G1 and G2 is a pair of drawings of G1\nand G2 such that each drawing is planar and each vertex common to G1 and G2 is repre-\nsented by the same point in both drawings. Unfortunately, if one wishes to visualize the\nedges of G1 and G2 as rectilinear segments (the so called geometric simultaneous em-\nbedding), not all pairs of graphs can be embedded simultaneously. Erten and Kobourov\n([4]), Brass et al. ([1]), and Geyer et al. ([7]) have shown that it is not always possible\nto embed simultaneously with straight-line edges a planar graph and a path, three paths,\nand two trees, respectively. On the other hand, if one permits that each edge of a graph\nis displayed as a different Jordan curve (the so called simultaneous embedding), then\nby the results of Pach and Wenger ([9]) any number of planar graphs can be embedded\nsimultaneously. Restricting the last constraints, one could permit that each edge is rep-\nresented by a Jordan curve, but could force edges common to more graphs to have the\nsame representation in the drawing of each graph (the so called simultaneous embed-\nding with ﬁxed edges). Di Giacomo and Liotta ([3]) showed that an outerplanar graph\nand a cycle can always be simultaneously embedded with ﬁxed edges, improving the\nresults in [4], where it is shown how to embed simultaneously with ﬁxed (“consistent”)\nedges a tree and a path. In [4] and [3] the problem of ﬁnding simultaneous embeddings\nwith ﬁxed edges of pairs of trees and of pairs of planar graphs is explicitly mentioned.\nIn this paper we improve the results on simultaneous embedding with ﬁxed edges\nof graphs, by showing that there exist two outerplanar graphs that cannot be simulta-\nneously embedded with ﬁxed edges (Section 3) and that a planar graph and a tree can\nalways be simultaneously embedded with ﬁxed edges (Section 4). Then in Section 5 we\ngive conclusions and suggest some open problems.",
         "5 Conclusions\nIn this paper we have shown that two outerplanar graphs can not always be simulta-\nneously embedded with ﬁxed edges, while a planar graph and a tree can. Observe that\nafter the last execution of Add Edge (see Section 4) the dual graph D of the augmented\ntriangular graph G is still connected even after the removal from D of the edges dual\nto edges of the tree. Hence an other execution of Add Edge is still possible and so the\nalgorithm proposed in Section 4 works more generally when the ﬁrst graph is planar\nand the second is a tree augmented by an edge. This straightforwardly implies that the\nalgorithm works also for a planar graph and a cycle. A drawback of the algorithm in\nSection 4 is that of using a large number of bends, and so it remains an open problem\nto ﬁnd an algorithm for drawing a planar graph and a tree with ﬁxed edges with only\na small number of bends and within a small area. As far as we know, it is also still\nopen the geometric simultaneous embedding of a tree and a path and the simultaneous\nembedding without mapping ([1]) of two planar graphs.",
         "1 Introduction\nA drawing of a graph is a mapping of each vertex to a distinct point in the plane and\nof each edge to a Jordan curve between the endpoints of the edge. A planar drawing is\n Work partially supported by EC - Fet Project DELIS - Contract no 001907, by “Project ALGO-\nNEXT: Algorithms for the Next Generation Internet and Web: Methodologies, Design, and Ex-\nperiments”, and by MIUR Programmi di Ricerca Scientiﬁca di Rilevante Interesse Nazionale.\nEmbedding Graphs Simultaneously with Fixed Edges\nsuch that no two edges intersect. A planar graph is a graph that admits a planar drawing.\nAn embedding of a graph G is a circular ordering of the edges incident on each vertex of\nG. An embedding of a graph speciﬁes what are its faces in any drawing respecting such\nembedding and so it speciﬁes the dual graph of G that is the graph with one vertex for\neach face of G and with one edge between two vertices if the corresponding faces share\nan edge in G. A poly-line drawing is such that the edges are sequences of rectilinear\nsegments. A straight-line drawing is such that all edges are rectilinear segments. It has\nbeen shown in [5] that every planar graph admits a planar straight-line drawing.\nA simultaneous embedding with ﬁxed edges of two graphs G1 = (V1, E1) and\nG2 = (V2, E2) with a bijective mapping γ : V1 → V2 between their vertices is a\npair of drawings Γ1 and Γ2 of G1 and of G2, respectively, such that: (i) each of Γ1 and\nΓ2 is a planar drawing, (ii) each vertex v2 = γ(v1), with v1 ∈ V1 and v2 ∈ V2, is\nmapped in Γ2 to the same point where v1 is mapped in Γ1, and (iii) an edge belonging\nto both E1 and E2 is represented by the same simple Jordan curve in Γ1 and Γ2. In this\npaper we only deal with simultaneous embedding with ﬁxed edges, so in the following,\nunless otherwise speciﬁed, simultaneous embedding will always stand for simultaneous\nembedding with ﬁxed edges.\n3 Simultaneous Embeddings of Outerplanar Graphs\nIn this section we show that there exist two outerplanar graphs that cannot be simulta-\nneously embedded. This result was already obtained in [1] for geometric simultaneous\nembedding. Although we believe that the pair of outerplanar graphs presented in [1] can\nnot be simultaneously embedded even in the ﬁxed edges setting, this was never pointed\nup and also the proof in [1] exploits the fact that the edges are drawn as segments.\nTheorem 1. There exist two outerplanar graphs that can not be simultaneously em-\nbedded with ﬁxed edges.\nTo prove Theorem 1 we ﬁrst show the topologies of two outerplanar graphs G1 and G2\nand a bijective mapping γ between their vertices. We use the same name vi for a vertex\nu of G1 and a vertex v of G2 to mean that v = γ(u).\nLet G1 = (V1, E1) and G2 = (V2, E2) be two outerplanar graphs with six vertices\neach and with E1 = {(v1, v2), (v1, v3), (v2, v3), (v1, v4), (v2, v4), (v2, v5), (v4, v5),\n(v2, v6), (v3, v6)}, and E2 = {(v1, v2), (v1, v3), (v2, v3), (v1, v6), (v2, v6), (v2, v5),\n(v6, v5), (v2, v4), (v3, v4)} (see Fig. 1). To show that G1 and G2 admit no simultaneous\n(b)\nembedding with mapping γ between their vertices, we try to construct embeddings E1\nof G1 and E2 of G2, proving that at least one between E1 and E2 must be non planar.\nFirst we embed vertices v1, v2, and v3. These vertices form a cycle C1, that is com-\nmon to E1 and E2, and that divides the plane in two parts, one inside and one out-\nside C1. Then, after having drawn v4 and its incident edges, the plane is subdivided\nin four regions F1, F2, F3, and F4, delimited by cycles between vertices (v1, v2, v3),\n(v1, v2, v4), (v1, v3, v4), and (v2, v3, v4), respectively, each one with the fourth vertex\noutside. This is because drawing v4 inside or outside C1 and changing the clockwise\norder of the edges incident in v4 only permit to choose the external face of the simulta-\nneous embedding, as shown in Fig. 2 (a) – (d). Since only the edges (v1, v4) and (v3, v4)\ncan intersect, regions F2, F3 and F4 can overlap, while region F1 can not intersect any\nother region, as shown in Fig. 2 (e).\n(e)\nNow we embed v6 and its incident edges. It is easy to observe that v6 must be placed\ninside region F1. In fact if v6 is placed inside F2, then (v3, v6) intersects the cycle\n(v1, v2, v4) in E1; if v6 is placed inside F4, then (v1, v6) intersects the cycle (v2, v3, v4)\nin E2; if v6 is placed inside F3, then (v2, v6) either intersects an edge between (v1, v3)\nand (v1, v4) in E1 or intersects an edge between (v1, v3) and (v3, v4) in E2. We have\nshown that v4 and v6 must be placed one inside and the other outside C1. Note that v4\nis adjacent to v5 in G1 and v6 is adjacent to v5 in G2. Hence, embedding v5 anywhere\nin the plane creates an edge e from inside to outside C1. Since e intersects C1, that is\ncommon to G1 and G2, there is not a placement for v5 preserving the planarity of both\nE1 and E2 and this concludes the proof.\nEmbedding Graphs Simultaneously with Fixed Edges\n4 Simultaneous Embedding of a Planar Graph and a Tree\nIn this section we prove the following theorem:\nTheorem 2. A planar graph and a tree can be simultaneously embedded with ﬁxed\nedges with any established mapping between their vertices.\nTo prove Theorem 2 we show how to construct a simultaneous drawing of a planar\ngraph G and of a tree T with any mapping γ between their vertices.\nAugment G to a triangular graph G, by adding edges that will be removed later\nand then, by using one of the well-known methods to draw with straight-line edges a\ntriangular graph ([5,6,10]), construct a straight-line drawing of G with dual graph D.\nNote that the edges of T that are common to edges of G have been already drawn. Let\nET denote the set of edges of T that have still to be drawn. From now on, we call Γ the\ncurrent simultaneous drawing of G and T . Let also SN denote the set of already drawn\nsegments representing straight-line edges of T or representing parts of poly-line edges\nof T . Now, until all the edges in ET have been drawn, choose one of them, say (u, v),\nand draw it by using the following procedure that we call Add Edge.\n1. Select a simple path p = (f1, f2, . . . , fk) in D with the following properties: (i) f1\nis dual to a face of G that contains u, (ii) fk is dual to a face of G that contains v,\nand (iii) each edge (fi, fi+1), 1 ≤ i < k, is not dual to an edge of G represented\nby a segment in SN . Then, for every face Fi of G that is dual to a vertex fi of p,\n1 ≤ i ≤ k, choose a point pi in the interior of the region representing Fi in Γ . For\nevery edge (w1, w2) of G dual to an edge (fi, fi+1) of p, 1 ≤ i < k, choose a\npoint ei in the interior of the segment representing (w1, w2) in Γ (see Fig. 3 (a)).\n2. Augment G by adding to it a vertex ui for each vertex fi of p, 1 ≤ i ≤ k, and a\nvertex vi for each edge (fi, fi+1) of p, 1 ≤ i < k. Add also to G the following\nedges: for each face Fi dual to fi ∈ p, 1 < i < k, let (wi,1, wi,2) and (wi,2, wi,3) be\nthe edges of Fi dual to (fi−1, fi) and (fi, fi+1), respectively. Add to G the edges\n(wi,1, ui), (wi,2, ui), (wi,3, ui), (vi−1, ui), and (vi, ui). Split (wi,1, wi,2) in two\nedges (wi,1, vi−1) and (wi,2, vi−1) and split (wi,2, wi,3) in two edges (wi,2, vi) and\n(wi,3, vi). For the face F1 dual to f1 ∈ p let a and b be the vertices of F1 distinct\nfrom u, and let (w1,1, w1,2) be the edge of F1 dual to (f1, f2). Add to G edges\n(u, u1), (a, u1), (b, u1), and (v1, u1). Split (w1,1, w1,2) in two edges (w1,1, v1)\nand (w1,2, v1). For the face Fk dual to fk ∈ p let c and d be the vertices of Fk\ndistinct from v and let (wk,1, wk,2) be the edge of Fk dual to (fk−1, fk). Add to\nG edges (v, uk), (c, uk), (d, uk), and (vk−1, uk). Split (wk,1, wk,2) in two edges\n(wk,1, vk−1) and (wk,2, vk−1).\n3. As shown in Fig. 3 (b), map each ui to pi, 1 ≤ i ≤ k and each vi to ei, 1 ≤ i < k.\nDraw the edges added to G as straight lines. Draw the edge (u, v) of T in Γ as\na polygonal line epol passing through points u, p1, e1, . . . , pi, ei, . . . , pk−1, ek−1,\npk, v.\n4. Remove (u, v) from ET and add to SN every segment of epol.\nWhen ET = ∅, the ﬁnal drawing of G is obtained by deleting from the current G\nall the edges not belonging to G. Note that it is possible that some edges of G are now\nrepresented by a polygonal line obtained by repeatedly splitting the starting straight-line\nedge. The ﬁnal drawing of T is formed by all the segments in SN .\nWe now show that the above described algorithm constructs a simultaneous embed-\nding of G and T . To check this, we show that: (i) the construction of the simultaneous\ndrawing of G and T starts from a partial simultaneous planar drawing, (ii) the planarity\nof the drawing of G is preserved after each application of Add Edge, (iii) the planarity\nof the drawing of T is preserved after each application of Add Edge, (iv) one can always\napply Add Edge until all the edges in ET have been drawn, and (v) each edge common\nto G and T has the same representation in both the drawings of G and of T .\n(i) The planarity of the starting simultaneous drawing Γ is a consequence of the\nplanarity of the straight-line drawings obtained by applications of the algorithms in\n[5,6,10].\n(ii) Each time one applies Add Edge to draw an edge of ET , G is augmented by\nadding new vertices and new edges to it. This is done in such a way that both the\ntriangulation of G and the planarity of its current drawing are preserved, as can be\neasily checked (see Figure 3).\n(iii) After each execution of Add Edge the subgraph of T that has been already drawn\nis a subgraph of G. By construction, the last assertion is true after triangulating G and\nafter straight-line drawing G and it remains true also after each application of Add\nEdge. To prove this, observe that at step 2 of Add Edge one augments G by adding\nsome new vertices and edges to it, then at step 3 one draws these new vertices and\nedges. Then an edge of T is drawn as a polygonal line whose bends coincide with the\nnew vertices of G and whose edges coincide with some of the new edges of G. So the\nplanarity of the drawing of T is a consequence of the planarity of the drawing of G.\n(iv) Suppose that when we are starting a new execution of Add Edge the dual graph\nD of G is not connected after the removal of the edges that are dual to edges of T\nrepresented by segments in SN . This is equivalent of saying that the removed edges\nform a cutset for D. From [8] we know that:\nLemma 1. Let G be a planar graph and D be a geometric dual of G, then a set of\nedges in G forms a cycle (or cutset) in G if and only if the corresponding set of edges\nof D forms a cutset (res. cycle) in D.\nEmbedding Graphs Simultaneously with Fixed Edges\nSo the set of edges of T already drawn forms a cycle and this gives us a contradiction,\nsince T is a tree. So D is connected even after removing from it the edges that are\ndual to edges of T already drawn and this permits us to select a path p in D with the\nproperties described at step 1 of Add Edge and to apply Add Edge.\n(v) After triangulating G and after drawing the resulting triangular graph G, the\nedges that are common to both graphs are drawn as straight-line segments between the\nsame end-points and while applying Add Edge they are splitted in the same way. So the\nedges that are common to G and T have the same drawing in Γ .",
         "",
         "",
         "https://academic.oup.com/crawlprevention/governor?content=%2fjxb%2fOUP%2fRunPdfHandler%3fjournal%3djxb%26volume%3d57%26issue%3d9%26page%3d1899%26multimediaId%3d1715826",
         "1 Introduction\nA simultaneous embedding of two planar graphs G1 and G2 is a pair of drawings of G1\nand G2 such that each drawing is planar and each vertex common to G1 and G2 is repre-\nsented by the same point in both drawings. Unfortunately, if one wishes to visualize the\nedges of G1 and G2 as rectilinear segments (the so called geometric simultaneous em-\nbedding), not all pairs of graphs can be embedded simultaneously. Erten and Kobourov\n([4]), Brass et al. ([1]), and Geyer et al. ([7]) have shown that it is not always possible\nto embed simultaneously with straight-line edges a planar graph and a path, three paths,\nand two trees, respectively. On the other hand, if one permits that each edge of a graph\nis displayed as a different Jordan curve (the so called simultaneous embedding), then\nby the results of Pach and Wenger ([9]) any number of planar graphs can be embedded\nsimultaneously. Restricting the last constraints, one could permit that each edge is rep-\nresented by a Jordan curve, but could force edges common to more graphs to have the\nsame representation in the drawing of each graph (the so called simultaneous embed-\nding with ﬁxed edges). Di Giacomo and Liotta ([3]) showed that an outerplanar graph\nand a cycle can always be simultaneously embedded with ﬁxed edges, improving the\nresults in [4], where it is shown how to embed simultaneously with ﬁxed (“consistent”)\nedges a tree and a path. In [4] and [3] the problem of ﬁnding simultaneous embeddings\nwith ﬁxed edges of pairs of trees and of pairs of planar graphs is explicitly mentioned.\nIn this paper we improve the results on simultaneous embedding with ﬁxed edges\nof graphs, by showing that there exist two outerplanar graphs that cannot be simulta-\nneously embedded with ﬁxed edges (Section 3) and that a planar graph and a tree can\nalways be simultaneously embedded with ﬁxed edges (Section 4). Then in Section 5 we\ngive conclusions and suggest some open problems.\nA drawing of a graph is a mapping of each vertex to a distinct point in the plane and\nof each edge to a Jordan curve between the endpoints of the edge. A planar drawing is\n Work partially supported by EC - Fet Project DELIS - Contract no 001907, by “Project ALGO-\nNEXT: Algorithms for the Next Generation Internet and Web: Methodologies, Design, and Ex-\nperiments”, and by MIUR Programmi di Ricerca Scientiﬁca di Rilevante Interesse Nazionale.\nEmbedding Graphs Simultaneously with Fixed Edges\nsuch that no two edges intersect. A planar graph is a graph that admits a planar drawing.\nAn embedding of a graph G is a circular ordering of the edges incident on each vertex of\nG. An embedding of a graph speciﬁes what are its faces in any drawing respecting such\nembedding and so it speciﬁes the dual graph of G that is the graph with one vertex for\neach face of G and with one edge between two vertices if the corresponding faces share\nan edge in G. A poly-line drawing is such that the edges are sequences of rectilinear\nsegments. A straight-line drawing is such that all edges are rectilinear segments. It has\nbeen shown in [5] that every planar graph admits a planar straight-line drawing.\nA simultaneous embedding with ﬁxed edges of two graphs G1 = (V1, E1) and\nG2 = (V2, E2) with a bijective mapping γ : V1 → V2 between their vertices is a\npair of drawings Γ1 and Γ2 of G1 and of G2, respectively, such that: (i) each of Γ1 and\nΓ2 is a planar drawing, (ii) each vertex v2 = γ(v1), with v1 ∈ V1 and v2 ∈ V2, is\nmapped in Γ2 to the same point where v1 is mapped in Γ1, and (iii) an edge belonging\nto both E1 and E2 is represented by the same simple Jordan curve in Γ1 and Γ2. In this\npaper we only deal with simultaneous embedding with ﬁxed edges, so in the following,\nunless otherwise speciﬁed, simultaneous embedding will always stand for simultaneous\nembedding with ﬁxed edges.\n3 Simultaneous Embeddings of Outerplanar Graphs\nIn this section we show that there exist two outerplanar graphs that cannot be simulta-\nneously embedded. This result was already obtained in [1] for geometric simultaneous\nembedding. Although we believe that the pair of outerplanar graphs presented in [1] can\nnot be simultaneously embedded even in the ﬁxed edges setting, this was never pointed\nup and also the proof in [1] exploits the fact that the edges are drawn as segments.\nTheorem 1. There exist two outerplanar graphs that can not be simultaneously em-\nbedded with ﬁxed edges.\nTo prove Theorem 1 we ﬁrst show the topologies of two outerplanar graphs G1 and G2\nand a bijective mapping γ between their vertices. We use the same name vi for a vertex\nu of G1 and a vertex v of G2 to mean that v = γ(u).\nLet G1 = (V1, E1) and G2 = (V2, E2) be two outerplanar graphs with six vertices\neach and with E1 = {(v1, v2), (v1, v3), (v2, v3), (v1, v4), (v2, v4), (v2, v5), (v4, v5),\n(v2, v6), (v3, v6)}, and E2 = {(v1, v2), (v1, v3), (v2, v3), (v1, v6), (v2, v6), (v2, v5),\n(v6, v5), (v2, v4), (v3, v4)} (see Fig. 1). To show that G1 and G2 admit no simultaneous\n(b)\nembedding with mapping γ between their vertices, we try to construct embeddings E1\nof G1 and E2 of G2, proving that at least one between E1 and E2 must be non planar.\nFirst we embed vertices v1, v2, and v3. These vertices form a cycle C1, that is com-\nmon to E1 and E2, and that divides the plane in two parts, one inside and one out-\nside C1. Then, after having drawn v4 and its incident edges, the plane is subdivided\nin four regions F1, F2, F3, and F4, delimited by cycles between vertices (v1, v2, v3),\n(v1, v2, v4), (v1, v3, v4), and (v2, v3, v4), respectively, each one with the fourth vertex\noutside. This is because drawing v4 inside or outside C1 and changing the clockwise\norder of the edges incident in v4 only permit to choose the external face of the simulta-\nneous embedding, as shown in Fig. 2 (a) – (d). Since only the edges (v1, v4) and (v3, v4)\ncan intersect, regions F2, F3 and F4 can overlap, while region F1 can not intersect any\nother region, as shown in Fig. 2 (e).\n(e)\nNow we embed v6 and its incident edges. It is easy to observe that v6 must be placed\ninside region F1. In fact if v6 is placed inside F2, then (v3, v6) intersects the cycle\n(v1, v2, v4) in E1; if v6 is placed inside F4, then (v1, v6) intersects the cycle (v2, v3, v4)\nin E2; if v6 is placed inside F3, then (v2, v6) either intersects an edge between (v1, v3)\nand (v1, v4) in E1 or intersects an edge between (v1, v3) and (v3, v4) in E2. We have\nshown that v4 and v6 must be placed one inside and the other outside C1. Note that v4\nis adjacent to v5 in G1 and v6 is adjacent to v5 in G2. Hence, embedding v5 anywhere\nin the plane creates an edge e from inside to outside C1. Since e intersects C1, that is\ncommon to G1 and G2, there is not a placement for v5 preserving the planarity of both\nE1 and E2 and this concludes the proof.\nEmbedding Graphs Simultaneously with Fixed Edges\n4 Simultaneous Embedding of a Planar Graph and a Tree\nIn this section we prove the following theorem:\nTheorem 2. A planar graph and a tree can be simultaneously embedded with ﬁxed\nedges with any established mapping between their vertices.\nTo prove Theorem 2 we show how to construct a simultaneous drawing of a planar\ngraph G and of a tree T with any mapping γ between their vertices.\nAugment G to a triangular graph G, by adding edges that will be removed later\nand then, by using one of the well-known methods to draw with straight-line edges a\ntriangular graph ([5,6,10]), construct a straight-line drawing of G with dual graph D.\nNote that the edges of T that are common to edges of G have been already drawn. Let\nET denote the set of edges of T that have still to be drawn. From now on, we call Γ the\ncurrent simultaneous drawing of G and T . Let also SN denote the set of already drawn\nsegments representing straight-line edges of T or representing parts of poly-line edges\nof T . Now, until all the edges in ET have been drawn, choose one of them, say (u, v),\nand draw it by using the following procedure that we call Add Edge.\n1. Select a simple path p = (f1, f2, . . . , fk) in D with the following properties: (i) f1\nis dual to a face of G that contains u, (ii) fk is dual to a face of G that contains v,\nand (iii) each edge (fi, fi+1), 1 ≤ i < k, is not dual to an edge of G represented\nby a segment in SN . Then, for every face Fi of G that is dual to a vertex fi of p,\n1 ≤ i ≤ k, choose a point pi in the interior of the region representing Fi in Γ . For\nevery edge (w1, w2) of G dual to an edge (fi, fi+1) of p, 1 ≤ i < k, choose a\npoint ei in the interior of the segment representing (w1, w2) in Γ (see Fig. 3 (a)).\n2. Augment G by adding to it a vertex ui for each vertex fi of p, 1 ≤ i ≤ k, and a\nvertex vi for each edge (fi, fi+1) of p, 1 ≤ i < k. Add also to G the following\nedges: for each face Fi dual to fi ∈ p, 1 < i < k, let (wi,1, wi,2) and (wi,2, wi,3) be\nthe edges of Fi dual to (fi−1, fi) and (fi, fi+1), respectively. Add to G the edges\n(wi,1, ui), (wi,2, ui), (wi,3, ui), (vi−1, ui), and (vi, ui). Split (wi,1, wi,2) in two\nedges (wi,1, vi−1) and (wi,2, vi−1) and split (wi,2, wi,3) in two edges (wi,2, vi) and\n(wi,3, vi). For the face F1 dual to f1 ∈ p let a and b be the vertices of F1 distinct\nfrom u, and let (w1,1, w1,2) be the edge of F1 dual to (f1, f2). Add to G edges\n(u, u1), (a, u1), (b, u1), and (v1, u1). Split (w1,1, w1,2) in two edges (w1,1, v1)\nand (w1,2, v1). For the face Fk dual to fk ∈ p let c and d be the vertices of Fk\ndistinct from v and let (wk,1, wk,2) be the edge of Fk dual to (fk−1, fk). Add to\nG edges (v, uk), (c, uk), (d, uk), and (vk−1, uk). Split (wk,1, wk,2) in two edges\n(wk,1, vk−1) and (wk,2, vk−1).\n3. As shown in Fig. 3 (b), map each ui to pi, 1 ≤ i ≤ k and each vi to ei, 1 ≤ i < k.\nDraw the edges added to G as straight lines. Draw the edge (u, v) of T in Γ as\na polygonal line epol passing through points u, p1, e1, . . . , pi, ei, . . . , pk−1, ek−1,\npk, v.\n4. Remove (u, v) from ET and add to SN every segment of epol.\nWhen ET = ∅, the ﬁnal drawing of G is obtained by deleting from the current G\nall the edges not belonging to G. Note that it is possible that some edges of G are now\nrepresented by a polygonal line obtained by repeatedly splitting the starting straight-line\nedge. The ﬁnal drawing of T is formed by all the segments in SN .\nWe now show that the above described algorithm constructs a simultaneous embed-\nding of G and T . To check this, we show that: (i) the construction of the simultaneous\ndrawing of G and T starts from a partial simultaneous planar drawing, (ii) the planarity\nof the drawing of G is preserved after each application of Add Edge, (iii) the planarity\nof the drawing of T is preserved after each application of Add Edge, (iv) one can always\napply Add Edge until all the edges in ET have been drawn, and (v) each edge common\nto G and T has the same representation in both the drawings of G and of T .\n(i) The planarity of the starting simultaneous drawing Γ is a consequence of the\nplanarity of the straight-line drawings obtained by applications of the algorithms in\n[5,6,10].\n(ii) Each time one applies Add Edge to draw an edge of ET , G is augmented by\nadding new vertices and new edges to it. This is done in such a way that both the\ntriangulation of G and the planarity of its current drawing are preserved, as can be\neasily checked (see Figure 3).\n(iii) After each execution of Add Edge the subgraph of T that has been already drawn\nis a subgraph of G. By construction, the last assertion is true after triangulating G and\nafter straight-line drawing G and it remains true also after each application of Add\nEdge. To prove this, observe that at step 2 of Add Edge one augments G by adding\nsome new vertices and edges to it, then at step 3 one draws these new vertices and\nedges. Then an edge of T is drawn as a polygonal line whose bends coincide with the\nnew vertices of G and whose edges coincide with some of the new edges of G. So the\nplanarity of the drawing of T is a consequence of the planarity of the drawing of G.\n(iv) Suppose that when we are starting a new execution of Add Edge the dual graph\nD of G is not connected after the removal of the edges that are dual to edges of T\nrepresented by segments in SN . This is equivalent of saying that the removed edges\nform a cutset for D. From [8] we know that:\nLemma 1. Let G be a planar graph and D be a geometric dual of G, then a set of\nedges in G forms a cycle (or cutset) in G if and only if the corresponding set of edges\nof D forms a cutset (res. cycle) in D.\nEmbedding Graphs Simultaneously with Fixed Edges\nSo the set of edges of T already drawn forms a cycle and this gives us a contradiction,\nsince T is a tree. So D is connected even after removing from it the edges that are\ndual to edges of T already drawn and this permits us to select a path p in D with the\nproperties described at step 1 of Add Edge and to apply Add Edge.\n(v) After triangulating G and after drawing the resulting triangular graph G, the\nedges that are common to both graphs are drawn as straight-line segments between the\nsame end-points and while applying Add Edge they are splitted in the same way. So the\nedges that are common to G and T have the same drawing in Γ .\n5 Conclusions\nIn this paper we have shown that two outerplanar graphs can not always be simulta-\nneously embedded with ﬁxed edges, while a planar graph and a tree can. Observe that\nafter the last execution of Add Edge (see Section 4) the dual graph D of the augmented\ntriangular graph G is still connected even after the removal from D of the edges dual\nto edges of the tree. Hence an other execution of Add Edge is still possible and so the\nalgorithm proposed in Section 4 works more generally when the ﬁrst graph is planar\nand the second is a tree augmented by an edge. This straightforwardly implies that the\nalgorithm works also for a planar graph and a cycle. A drawback of the algorithm in\nSection 4 is that of using a large number of bends, and so it remains an open problem\nto ﬁnd an algorithm for drawing a planar graph and a tree with ﬁxed edges with only\na small number of bends and within a small area. As far as we know, it is also still\nopen the geometric simultaneous embedding of a tree and a path and the simultaneous\nembedding without mapping ([1]) of two planar graphs.",
         "",
         "Embedding Graphs Simultaneously with Fixed Edges",
         "None"
        ],
        [
         "7",
         "0004f9cbd5224cab85d3da4c93ba7013245647b4",
         "Poly‐γ‐glutamate (PGA), a natural polymer, is synthesized by several bacteria (all Gram‐positive), one archaea and one eukaryote. PGA has diverse biochemical properties, enabling it to play different roles, depending on the organism and its environment. Indeed, PGA allows bacteria to survive at high salt concentrations and may also be involved in virulence. The minimal gene sets required for PGA synthesis were recently defined. There are currently two nomenclatures depending on the PGA final status: cap, for ‘capsule’, when PGA is surface associated or pgs, for ‘polyglutamate synthase’, when PGA is released. The minimal gene sets contain four genes termed cap or pgs B, C, A and E. The PGA synthesis complex is membrane‐anchored and uses glutamate and ATP as substrates. Schematically, the reaction may be divided into two steps, PGA synthesis and PGA transport through the membrane. PGA synthesis depends primarily on CapB‐CapC (or PgsB‐PgsC), whereas PGA transport requires the presence, or the addition, of CapA‐CapE (or PgsAA‐PgsE). The synthesis complex is probably responsible for the stereochemical specificity of PGA composition. Finally, PGA may be anchored to the bacterial surface or released. An additional enzyme is involved in this reaction: either CapD, a γ‐glutamyl‐transpeptidase that catalyses anchorage of the PGA, or PgsS, a hydrolase that facilitates release. The anchoring of PGA to the bacterial surface is important for virulence. All cap genes are therefore potential targets for inhibitors specifically blocking PGA synthesis or anchorage.",
         "T. Candela,A. Fouet",
         "T. Candela,A. Fouet",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1365-2958.2006.05179.x",
         "-glutamate (PGA), a natural polymer, is synthe-\nPoly-\ng\nsized  by  several  bacteria  (all  Gram-positive),  one\narchaea  and  one  eukaryote.  PGA  has  diverse  bio-\nchemical properties, enabling it to play different roles,\ndepending  on  the  organism  and  its  environment.\nIndeed,  PGA  allows  bacteria  to  survive  at  high  salt\nconcentrations and may also be involved in virulence.\nThe  minimal  gene  sets  required  for  PGA  synthesis\nwere recently deﬁned. There are currently two nomen-\nclatures  depending  on  the  PGA  ﬁnal  status:\n,  for\n‘capsule’, when PGA is surface associated or\n, for\n‘polyglutamate synthase’, when PGA is released. The\nor\nminimal gene sets contain four genes termed\n.  The  PGA  synthesis  complex  is\npgs  B\nmembrane-anchored  and  uses  glutamate  and  ATP\nas  substrates.  Schematically,  the  reaction  may  be\ndivided into two steps, PGA synthesis and PGA trans-\nport through the membrane. PGA synthesis depends\nprimarily  on  CapB-CapC  (or  PgsB-PgsC),  whereas\nPGA transport requires the presence, or the addition,\nof CapA-CapE (or PgsAA-PgsE). The synthesis com-\nplex  is  probably  responsible  for  the  stereochemical\nspeciﬁcity of PGA composition. Finally, PGA may be\nanchored  to  the  bacterial  surface  or  released.  An\nadditional enzyme is involved in this reaction: either\nCapD,  a\n-glutamyl-transpeptidase  that  catalyses\ng\nanchorage of the PGA, or PgsS, a hydrolase that facil-\nitates release. The anchoring of PGA to the bacterial\nsurface is important for virulence. All\ngenes are\ntherefore  potential  targets  for  inhibitors  speciﬁcally\nblocking PGA synthesis or anchorage.",
         "",
         "Have you ever tasted the Japanese speciality natto? If you\nhave, you might like to know that it consists largely of poly-\nγ\n-glutamate (PGA). Indeed, PGA is a fermentation product\ngrown  on  soybean.  PGA  was\nmade  by\nﬁrst described in\n, at the start of the 20th\nCentury.\n, a Gram-positive sporulating bacte-\nrium, is the causal agent of anthrax. This disease mostly\naffects  herbivores  but  all  mammals,  including  humans,\nmay be affected (for review, see Mock and Fouet, 2001).\nB. anthracis\nvirulence was found to be correlated with the\npresence of a capsule composed solely of PGA (Tomcsik\nand  Szongott,  1933).\nalso\nStaphylococcus  epidermidis\nsynthesizes  surface-associated  PGA  (Kocianova\n.,\net al\n2005).\nmay  cause  severe  infection  after\npenetrating the protective epidermal barriers of the human\nbody.  PGA  enables  both  of  these  bacteria  to  escape\nphagocytosis,  and  therefore  acts  as  a  virulence  factor\n(Zwartouw and Smith, 1956; Kocianova\n., 2005). Fur-\net al\ncapsule  is  composed  exclu-\nB. anthracis\nthermore,  the\nsively of the\nenantiomer (PDGA), making it particularly\nD\nnon-immunogenic (Zwartouw and Smith, 1956). This cap-\nsule also prevents antibodies from gaining access to the\nB. anthracis\nbacterium (Mesnage\nbacilli  against  phage  infections  and\ncocci\nagainst  antimicrobial  peptides  (McCloy,  1951;  Kocianova\n., 2005), probably by acting as a passive barrier.\net al\nSeveral  organisms  (bacteria,  one  archaea  and  one\neukaryotic  organism;  Table 1)  produce  non-anchored\nPGA. Interestingly, all the bacteria known to produce PGA\nare  Gram-positive  and  more  precisely  members  of  the\nBacillales  order,  belonging  to  the  Bacilli  class,  thus  they\nare all phylogenetically closely related. PGA has diverse\nfunctions that vary according to the species synthesizing\nit  and  their  environment.  Soil  bacteria  (mostly  from  the\ngenus\n)  use  released\nPGA for the sequestration of toxic metal ions, increasing\ntheir resistance to adverse environments (McLean\n.,\n1990). PGA may also be a source of glutamate for bacte-\nria  in  a  starvation  state  during  late  stationary  phase\nSporosa-\n(Kimura\n,\n,  the  only  archaea\nNatrialba  asiatica\nrcina  halophila\nknown to produce PGA, use it to decrease high local salt\n., 1998). It protects\n,  but  excluding\nS. epidermidis\nB. anthracis\net al\nT. Candela and A. Fouet",
         "",
         "",
         "https://academic.oup.com/crawlprevention/governor?content=%2fjxb%2fOUP%2fRunPdfHandler%3fjournal%3djxb%26volume%3d57%26issue%3d9%26page%3d1899%26multimediaId%3d1715826",
         "-glutamate (PGA), a natural polymer, is synthe-\nPoly-\ng\nsized  by  several  bacteria  (all  Gram-positive),  one\narchaea  and  one  eukaryote.  PGA  has  diverse  bio-\nchemical properties, enabling it to play different roles,\ndepending  on  the  organism  and  its  environment.\nIndeed,  PGA  allows  bacteria  to  survive  at  high  salt\nconcentrations and may also be involved in virulence.\nThe  minimal  gene  sets  required  for  PGA  synthesis\nwere recently deﬁned. There are currently two nomen-\nclatures  depending  on  the  PGA  ﬁnal  status:\n,  for\n‘capsule’, when PGA is surface associated or\n, for\n‘polyglutamate synthase’, when PGA is released. The\nor\nminimal gene sets contain four genes termed\n.  The  PGA  synthesis  complex  is\npgs  B\nmembrane-anchored  and  uses  glutamate  and  ATP\nas  substrates.  Schematically,  the  reaction  may  be\ndivided into two steps, PGA synthesis and PGA trans-\nport through the membrane. PGA synthesis depends\nprimarily  on  CapB-CapC  (or  PgsB-PgsC),  whereas\nPGA transport requires the presence, or the addition,\nof CapA-CapE (or PgsAA-PgsE). The synthesis com-\nplex  is  probably  responsible  for  the  stereochemical\nspeciﬁcity of PGA composition. Finally, PGA may be\nanchored  to  the  bacterial  surface  or  released.  An\nadditional enzyme is involved in this reaction: either\nCapD,  a\n-glutamyl-transpeptidase  that  catalyses\ng\nanchorage of the PGA, or PgsS, a hydrolase that facil-\nitates release. The anchoring of PGA to the bacterial\nsurface is important for virulence. All\ngenes are\ntherefore  potential  targets  for  inhibitors  speciﬁcally\nblocking PGA synthesis or anchorage.\nHave you ever tasted the Japanese speciality natto? If you\nhave, you might like to know that it consists largely of poly-\nγ\n-glutamate (PGA). Indeed, PGA is a fermentation product\ngrown  on  soybean.  PGA  was\nmade  by\nﬁrst described in\n, at the start of the 20th\nCentury.\n, a Gram-positive sporulating bacte-\nrium, is the causal agent of anthrax. This disease mostly\naffects  herbivores  but  all  mammals,  including  humans,\nmay be affected (for review, see Mock and Fouet, 2001).\nB. anthracis\nvirulence was found to be correlated with the\npresence of a capsule composed solely of PGA (Tomcsik\nand  Szongott,  1933).\nalso\nStaphylococcus  epidermidis\nsynthesizes  surface-associated  PGA  (Kocianova\n.,\net al\n2005).\nmay  cause  severe  infection  after\npenetrating the protective epidermal barriers of the human\nbody.  PGA  enables  both  of  these  bacteria  to  escape\nphagocytosis,  and  therefore  acts  as  a  virulence  factor\n(Zwartouw and Smith, 1956; Kocianova\n., 2005). Fur-\net al\ncapsule  is  composed  exclu-\nB. anthracis\nthermore,  the\nsively of the\nenantiomer (PDGA), making it particularly\nD\nnon-immunogenic (Zwartouw and Smith, 1956). This cap-\nsule also prevents antibodies from gaining access to the\nB. anthracis\nbacterium (Mesnage\nbacilli  against  phage  infections  and\ncocci\nagainst  antimicrobial  peptides  (McCloy,  1951;  Kocianova\n., 2005), probably by acting as a passive barrier.\net al\nSeveral  organisms  (bacteria,  one  archaea  and  one\neukaryotic  organism;  Table 1)  produce  non-anchored\nPGA. Interestingly, all the bacteria known to produce PGA\nare  Gram-positive  and  more  precisely  members  of  the\nBacillales  order,  belonging  to  the  Bacilli  class,  thus  they\nare all phylogenetically closely related. PGA has diverse\nfunctions that vary according to the species synthesizing\nit  and  their  environment.  Soil  bacteria  (mostly  from  the\ngenus\n)  use  released\nPGA for the sequestration of toxic metal ions, increasing\ntheir resistance to adverse environments (McLean\n.,\n1990). PGA may also be a source of glutamate for bacte-\nria  in  a  starvation  state  during  late  stationary  phase\nSporosa-\n(Kimura\n,\n,  the  only  archaea\nNatrialba  asiatica\nrcina  halophila\nknown to produce PGA, use it to decrease high local salt\n., 1998). It protects\n,  but  excluding\nS. epidermidis\nB. anthracis\net al\nT. Candela and A. Fouet",
         "",
         "Poly‐gamma‐glutamate in bacteria",
         "None"
        ],
        [
         "8",
         "0007428e145f375cd67c1a4cef3aa1c5af931e98",
         "In this study, three strains of Trypanosoma cruzi were isolated at the same time and in the same endemic region in Mexico from a human patient with chronic chagasic cardiomyopathy (RyC-H); vector (Triatoma barberi) (RyC-V); and rodent reservoir (Peromyscus peromyscus) (RyC-R). The three strains were characterized by multilocus enzyme electrophoresis, random amplified polymorphic DNA, and by pathological profiles in experimental animals (biodemes). Based on the analysis of genetic markers the three parasite strains were typed as belonging to T. cruzi I major group, discrete typing unit 1. The pathological profile of RyC-H and RyC-V strains indicated medium virulence and low mortality and, accordingly, the strains should be considered as belonging to biodeme Type III. On the other hand, the parasites from RyC-R strain induced more severe inflammatory processes and high mortality (> 40%) and were considered as belonging to biodeme Type II. The relationship between genotypes and biological characteristics in T. cruzi strains is still debated and not clearly understood. An expert committee recommended in 1999 that Biodeme Type III would correspond to T. cruzi I group, whereas Biodeme Type II, to T. cruzi II group. Our findings suggest that, at least for Mexican isolates, this correlation does not stand and that biological characteristics such as pathogenicity and virulence could be determined by factors different from those identified in the genotypic characterization.",
         "M. C. Sánchez-guillén,Christopher Bernabe,M. Tibayrenc,J. Zavala-Castro,José-Luis Totolhua,Julio Méndez-López,M. Gonzalez-Mejia,E. Torres-Rasgado,A. López-Colombo,R. Pérez-Fuentes",
         "M. C. Sánchez-guillén,Christopher Bernabe,M. Tibayrenc,J. Zavala-Castro,José-Luis Totolhua,Julio Méndez-López,M. Gonzalez-Mejia,E. Torres-Rasgado,A. López-Colombo,R. Pérez-Fuentes",
         "https://www.scielo.br/j/mioc/a/hkBS94wNKVZsJqmD8CxYq3t/?lang=en&format=pdf",
         "Laboratorio de Fisiopatología de Enfermedades Crónicas, Centro de Investigación Biomédica de Oriente, Instituto Mexicano del\nSeguro Social, Km 4.5 Carretera Federal Atlixco-Metepec, Atlixco, Puebla, México *Centre d’Etudes sur le Polymorphisme des\nMicro-Organismes, Montpellier, France **Facultad de Medicina, Benemérita Universidad Autónoma de Puebla, Puebla, México\n***Laboratorio de Parasitología, Centro de Investigaciones Regionales, Dr. Hideyo Noguchi, Mérida, Yucatán, México\nIn this study, three strains of  Trypanosoma cruzi were isolated at the same time and in the same endemic region\nin Mexico from a human patient with chronic chagasic cardiomyopathy (RyC-H); vector (Triatoma barberi) (RyC-V);\nand rodent reservoir (Peromyscus peromyscus) (RyC-R). The three strains were characterized by multilocus enzyme\nelectrophoresis,  random  amplified  polymorphic  DNA,  and  by  pathological  profiles  in  experimental  animals\n(biodemes). Based on the analysis of genetic markers the three parasite strains were typed as belonging to T. cruzi\nI  major  group,  discrete  typing  unit  1.  The  pathological  profile  of  RyC-H  and  RyC-V  strains  indicated  medium\nvirulence and low mortality and, accordingly, the strains should be considered as belonging to biodeme Type III. On\nthe other hand, the parasites from RyC-R strain induced more severe inflammatory processes and high mortality\n(> 40%) and were considered as belonging to biodeme Type II. The relationship between genotypes and biological\ncharacteristics in T. cruzi strains is still debated and not clearly understood. An expert committee recommended in\n1999 that Biodeme Type III would correspond to T. cruzi I group, whereas Biodeme Type II, to T. cruzi II group. Our\nfindings suggest that, at least for Mexican isolates, this correlation does not stand and that biological characteris-\ntics  such  as  pathogenicity  and  virulence  could  be  determined  by  factors  different  from  those  identified  in  the\ngenotypic  characterization\nKey words: Trypanosoma cruzi - genetic characterization - major groups - discrete typing units - pathogenic profile - biodemes\nTrypanosoma cruzi is the etiologic agent of Chagas\ndisease affecting approximately 20 million people in Cen-\ntral and South America (Moncayo 1997). T. cruzi is a para-\nsitic protozoan that consists of a heterogeneous popula-\ntion composed of a pool of strains circulating in both the\ndomestic and sylvatic cycles in humans, vectors, and ani-\nmal reservoirs (Souto et al. 1996).",
         "DISCUSSION\nGenetic and biological characteristics of the T. cruzi\nstrains represent a tool for the understanding of different\naspects of the epidemiology Chagas disease. Populations\nof T. cruzi are highly heterogeneous and by using genetic\nmarkers were distributed into two major phylogenetic lin-\neages (Tibayrenc 1995, Souto et al. 1996), which were\nnamed as T. cruzi I and T. cruzi II (Anonymous 1999).\nStudies carried out in Southern Cone Countries of South\nAmerica indicate that T. cruzi I is dominant in the sylvatic\ncycle and T. cruzi II is dominant in the domestic cycle of\nthe parasite transmission (Zingales et al. 1998, Diosque et\nal. 2003). Very few data on the epidemiological distribu-\ntion of the two T. cruzi major groups are available for the\nother countries of the Americas e.g. Mexico (Zavala Castro\net al. 1992, Espinosa et al. 1998) and Central America (Higo\net al. 2000). In this study, three Mexican T. cruzi parasites\nfrom human, vector and reservoir were genetically identi-\nfied  as  belonging  to  T.  cruzi  I;  moreover,  high  genetic\nidentity was shown between them (Jaccard’s distances <\n0.20). This observation is in agreement with the data show-\ning the predominance of T. cruzi I in stocks isolated from\n8 of the 32 states of the Mexican Republic (Bosseno et al.\n2002).\nmononuclear infiltrates and low parasitism of myocytes.\nBy the 21st day of infection, a progressive parasitism of\nmyocardiocytes and destruction of myocells with dense\ninflammatory infiltration was seen, with focal myocyte\nnecrosis and diffuse mononuclear cell infiltration. Tissue\nparasitism was prominent in skeletal muscles and progres-\nsive until 24 days of infection.\nHistopathological lesions determined by human and\nvector  T.  cruzi  strains  from  14  to  30  days  of  infection\nshowed the same evolution and histopathological aspects.\nIn general, the parasitism and inflammatory infiltration\nwere present at 14 and 20 days pi, varying from slight to\nmoderate. Lesions were more prominent in cardiac muscles\nand intensified up to the 28th day of infection. Analysis\nof the severity of tissues lesions showed an intense in-\nName\nDate\nBiological source\nRyC-H\nVII/98\nPatient with chronic\nchagasic cardiomyopathy\nRyC-V\nIV-98\nRyC-R\nVIII/98\nTriatoma barberi\nPeromyscus peromyscus\nGeographical source\nMolcaxac, Puebla, Mexico\nMolcaxac, Puebla, Mexico\n21 days\nBroad forms (70%)\nBroad forms (70%)\nSlender forms (70%)\nMortality rate in mice\nPeak of growth\nMorphology\nTissue  tropism\nHistopathogenicity\nBiodemes\nIII\nSkeletal muscle\nCardiac muscle\nCardiac muscle\nCardiac muscle\nIII\nCardiac muscle\nCardiac muscle\nII\nGenetic characterization\nLineage1, DTU-1\nLineage 1, DTU-1\nLineage 1, DTU-1\nStudy of the various biological parameters in response to infection by strains of T. cruzi show clear differences in response to\ninfection by strains from different sources. T. cruzi parasites from human (with chronic chagasic cardiomyopathy) and vector did not\ninduce a significant inflammatory process in the heart and muscles. In contrast, T. cruzi parasites from reservoir induce an intense\ninflammatory process in the heart and skeletal muscles.\nT. cruzi II has been shown to be heterogeneous and\ndifferent genetic subdivisions have been described, named\nDTUs (Brisse et al. 1998). It was suggested that different\nDTUs correspond to distinct epidemiological pictures and\nare proposed as a reference framework for genetic vari-\nability and for biological characterization studies of T. cruzi\nstrains (Barnabé et al. 2000). The geographical distribu-\ntion of the six DTUs is present in the entire range of Chagas\ndisease. On the other hand, DTU 2b has been mainly iso-\nlated in Brazil and Chile, while DTU 2d was mainly en-\ncountered in Chile and Bolivia (Barnabé et al. 2000). In\nthis study, the three T. cruzi strains isolated in the same\nregion from human, domestic and wild transmission cycles\nwere characterized by multigene typing as belonging to\nDTU 1 and exhibited strong levels of linkage. T. cruzi I\n(DTU 1) has been considered a homogeneous group. The\nfindings of the present study show clear biological differ-\nences between the three strains regarding the profile of\nparasitemia,  tissue  tropism,  pathogenicity  pattern,  and\nmortality. In fact, T. cruzi parasites from human and vec-\ntor (belonging to biodeme Type III) did not induce a sig-\nnificant inflammatory process in the heart and muscles. In\ncontrast, T. cruzi parasites from the reservoir (belonging\nto biodeme Type II) induced an intense inflammatory pro-\ncess in the heart and skeletal muscles.\nAccording  to  Andrade  and  Magalhães  (1997),  al-\nthough  genetic  studies  are  important  to  clarify  the  in-\ntraspecific heterogeneity of the parasite, only the study\nof the biological behavior could clarify the importance of\ndifferent strains in the determination of clinicopathologi-\ncal manifestations of Chagas disease. Our results could\nbe explained, in part, by the fact that many studies have\nbeen performed using T. cruzi strains isolated for more\nthan  20  years  and  that  genetic  intrinsic  characteristics\nand clonal selection could be influenced by handling in\nthe laboratory. Although experimental evidence of the sta-\nbility of strain behavior has been obtained by biochemi-\ncal  and  biological  characterization  after  parasites  were\nsubjected to different conditions of maintenance and cul-\ntivation, the main biological characteristics of the strains,\nsuch as infectivity and inoculum size, tended to decrease\nin association with maintenance and cultivation condi-\ntions (Magalhaes et al. 1985). Decrease in virulence has\nbeen observed when the culture forms were used or when\nthe infection with low inoculum was employed. Addition-\nally, the passage of T. cruzi parasites through the vector\nhas a positive influence on the virulence (higher levels of\nparasitemia and predominance of slender forms in mice)\n(Magalhaes et al. 1996). In this work we employed T. cruzi\nparasites recently isolated from the same endemic area.\nThis procedure would guarantee the reliability of the com-\nparative studies.\nInfection with T. cruzi is an enzootic disease extend-\ning from the US to South America (sylvatic cycle), that\ncan  lead  to  human  infection  when  the  insect  vector\n(triatomine  bugs)  adapts  to  human  dwelling  (domestic\ncycles). This adaptation occurs in different manners and\nat different times (Prata 2001). The presence of different\ntypes of T. cruzi strains in endemic areas may bear influ-\nence on the biological characteristics as well as pathoge-\nnicity. Studies with cloned populations of natural strains\nhave demonstrated either homogeneity or heterogeneity\nof several clones including differences in virulence and\npathogenicity. Thus, in an epidemiological study, cloned\npopulations  cannot  be  taken  as  representative  of  the\nstrains isolated from different geographical areas.\nThe expert committee in 1999 recommended that iso-\nlates typed as Biodeme Type III (Andrade 1974) are equiva-\nlent to T. cruzi I, and isolates typed as Biodeme Type II\nare equivalente to T. cruzi II (Anonymous 1999), our data\nclearly show differences in the biological behavior of three\nstrains classified as T. cruzi I, although T. cruzi I is con-\nsidered a homogeneous group based on the analysis of\ngenetic markers. Therefore, our T. cruzi I isolates were\ntyped as Biodeme Type II and Type III.\nIn conclusion, our findings suggest that for Mexican\nisolates, characteristics such as pathogenicity and viru-\nlence of T. cruzi strains must be described by a combina-\ntion of genetic and biological characteristics. Reliance on\nthe genotype alone may confer significance of little bio-\nlogical relevance and could be determined by factors dif-\nferent from those identified in the genotypic characteriza-\ntion.",
         "In Chagas disease, different clinical pictures predomi-\nnate in different areas, e.g., in Brazil, the asymptomatic or\nindeterminate form is the most common (60-70%), followed\nby the cardiac and digestive forms (20-30 and 8-10%, re-\nspectively); however, in central Brazil and Chile, the di-\ngestive form of Chagas disease predominates, whereas it\nis practically non-existent in Venezuela and Central America\n(Luquetti et al. 1986, Dias 1992). The reason for this geo-\ngraphical  heterogeneity  and  why  different  patients  de-\nFinancial support: Fosiza-Conacyt project 9980802019\n+Corresponding author: rycardoperez@yahoo.com.mx\n++Sponsor from Institut de Recherche pour le Développement\nand the Centre National de Recherche Scientifique\nReceived 7 November 2005\nAccepted 19 July 2006\nvelop  different  clinical  forms  poses  the  question  of  its\nrelationship to the genetic heterogeneity of T. cruzi popu-\nlations in different areas.\nhavior in animal models has indeed shown a correlation\nbetween T. cruzi natural strain behavior and the genetic\nvariability  in  different  geographical  areas  (Andrade  &\nMagalhaes 1997). Biodeme Type III would correspond to\nT. cruzi major I group, whereas Biodeme Type II, to T.\ncruzi II major group (Anonymous 1999).\nGenetic studies are important to clarify the intraspe-\ncific heterogeneity of the parasite; however, the ability to\nstudy the biological behavior and the host-parasite rela-\ntionship is crucial for the evaluation of the impact of the\ngenetic  diversity  of  the  parasite  regarding  its  relevant\nmedical properties such as pathogenicity and virulence.\nIn  this  work  we  attempt  to  understand  the  genetic\nstructure,  pathogenic  pattern,  and  epidemiological  and\nclinical implications of T. cruzi parasites from an endemic\narea in Mexico. Mexican strains of T. cruzi isolated from\nhuman, vector, and animal reservoir at the same time and\nin the same endemic area were genetically characterized\nby multilocus enzyme electrophoresis (MLEE), random\namplified polymorphic DNA (RAPD), and pathological pro-\nfiles by biological in vitro and in vivo analysis (biodemes).\nMATERIALS  AND  METHODS\nAnimals",
         "Mice - A total of 48 BALB/c female mice weighing 25-\n30 g were used at 10 to 12 weeks of age for each isolate.\nMice were maintained in a temperature-controlled envi-\nronment and provided with a balanced mouse ration and\nwater ad libitum. All animals were maintained in the same\nfacility for a minimum of two weeks before infection.\nParasites - Three T. cruzi isolates were obtained, re-\nspectively, from feces of adult Triatoma barberi (RyC-V),\nperipheral blood of Peromyscus peromyscus (RyC-R), a\nrodent from the Muridae family, and by xenodiagnostics\nof  human  hosts  with  chronic  Chagas  disease  (RyC-H).\nThe strains were isolated during epidemiological studies\nin Puebla, Mexico. Isolates were maintained by serial in-\ntraperitoneal (ip) passages in 10-week-old BALB/c mice\nand were grown axenically as epimastigote forms (culture\nforms) at 28°C in liver infusion-tryptose medium (LIT)\n(Difco Laboratories, Detroit, MI) supplemented with 10%\nheat-inactived fetal calf serum (Gibco, Grand Island, NY).\nGenetic characterization\nMLEE - Methods for isolating, growing in LIT me-\ndium, harvesting, and storing the stocks have been previ-\nously described (Tibayrenc & Le Ray 1984). The condi-\ntions for electrophoresis on cellulose acetate were per-\nformed as described by Ben Abde-rrazak et al. (1993) with\nslight  modifications. Twenty  enzyme  systems  were  as-\nsayed (Barnabé et al. 2000), namely, aconitase (EC 4.2.1.3,\nACON), alanine aminotransferase (EC 2.6.1.2, ALAT), dia-\nphorase (EC 1.6.99.2, DIA), glyceraldehyde-3-phosphate\ndehydrogenase (EC 1.2.1.12, GAPD), glutamate dehydro-\ngenase NAD+ (EC 1.4.1.2, GDH-NAD+), glutamate dehy-\ndrogenase NADP+ (EC 1.4.1.4. GDH-NADP+), aspartate\namino transferase (EC 2.6.1.1, GOT), glucose-6-phosphate\ndehydrogenase (EC 1.1.1.49, G6PD), glucose-6-phosphate\nisomerase (EC 5.3.1.9, GPI), isocitrate dehydrogenase (EC\n1.1.1.42, IDH), leucine aminopeptidase (cytosol aminopep-\ntidase) (EC 3.4.11.1, LAP), malate dehydrogenase (EC\n1.1.1.37, MDH), malate dehydrogenase (oxaloacetate de-\ncarboxylating, NADP+) or malic enzyme (EC 1.1.1.40, ME),\nmannose-phosphate isomerase (EC 5.3.1.8, MPI), nucleo-\nside hydrolase (EC 2.4.2.1, NHi); substrate: inosine, pep-\ntidase 1 (EC 3.4.22.3) (formerly EC 3.4.4.12), (PEP-1); sub-\nstrate: leucyl-leucyl-leucine, peptidase 2 (EC 3.4.22.4) (for-\nmerly EC 3.3.3.24), PEP-2; substrate: leucyl-L-alanine), 6-\nphosphoglucomutase dehydrogenase (EC 1.1.1.44, 6PGD),\nphosphoglucomutase (EC 5.4.2.2) (formerly EC 2.7.5.1.),\nPGM), and superoxide dismutase (EC 1.15.1.1, SOD).\nRAPD - was performed according to Tibayrenc et al.\n(1993) with slight modifications. Ten different decamer\nprimers from the A-kit of Operon Technologies were used.\nBiological characterization\nGrowth kinetics of epimastigote forms - A total of 25 ×\n106 parasites was inoculated in a final volume of 50 ml of\nLIT medium. Cultures were incubated at 27ºC. Kinetics\nwere followed for 24 days. The culture concentration was\nestimated  three  times  every  two  days  by  counting  in  a\nThomas chamber. Analysis of growth kinetics takes into\naccount the doubling time, estimated in hours, calculated\nfrom the log phase and the parasite concentration in 10–6\nparasites per ml (10–6 p/ml) and the end of the log phase.\nExperimental infection in mice (biodeme) - Experi-\nmental infection was performed in 4- or 5-week-old BALB/\nc female mice. A total of 144 mice were inoculated intrader-\nmally with 1 × 104 bloodstream trypomastigotes from in-\nfected  mice  for  each  strain.  Parasitemia  was  evaluated\nevery three days during the infection by microscopic ex-\namination of fresh blood samples obtained from the tail.\nParasitemic levels were expressed as logarithms of the\nmedia of parasites in peripheral blood of six mice for each\ngroup. Mortality rate in relation to infection was evalu-\nated  from  7  to  48  days  after  infection.  Morphology  of\ntrypomastigotes in peripheral blood was evaluated by the\npercentage of slender and broad forms, as seen in smears\nstained with the Giemsa method.\nHistopathological  analyses  -  These  analyses  were\nperformed on 144 of the infected mice corresponding to\neach T. cruzi strain and on 48 uninfected control mice.\nMice were killed by cervical dislocation and tissues were\nobtained each week for 7 weeks. Histological studies were\ncarried out in infected and control animals from skeletal\nmuscle, heart muscle, esophagus, colon, spleen, and liver.\nSamples were fixed in 10% neutral-buffered formalin and\nembedded in paraffin. Sections were stained with hema-\ntoxylin and eosin and examined by light microscope.\nTwo serial sections cut from three different parts of\neach organ were examined. The total surface of the sec-\ntions was at least 2 cm, and the lesions were recorded and\nclassified according to their severity. The severity of the\nlesions was generally evaluated with respect to histologi-\ncal characteristics rather than to the number of lesions,\nwhich varied between the sections. Each type of lesion\nwas assigned a severity index (scored from 1 to 3) accord-\ning  to  its  severity  and  potential  pathological  conse-\nquences.\nStatistical analysis - For both MLEE and RAPD data,\nJaccard’s distance (Jaccard 1908) was used. Each RAPD\nand MLEE gel band was coded with a number, starting\nwith 1 for the slowest band. The distance was estimated\naccording to the formula:\nD = 1 – [a/(a + b + c)]\nwhere a is the number of bands that are common to the\ntwo  compared  genotypes,  b  is  the  number  of  bands\npresent  in  the  first  genotype  and  absent  in  the  second,\nand c is the number of bands absent in the first genotype\nand present in the second.\nThe UPGMA method (unweighted pair-group method\nwith arithmetic averages) (Sneath & Sokal 1973) was used\nand for comparative evaluation of pathological profile of\nmice infected with human, vector and reservoir T. cruzi\nstrains a non-parametric Fisher test was applied.\nData analysis of the biological parameters - In a first\napproach, the average values of all biological parameters\nwere compared by Student’s t-test between each of the\ndifferent  strains  (human,  vector  and  reservoir  T.  cruzi\nstrains)  genetically  characterized  by  MLEE/RAPD\nanalysis.",
         "RESULTS\nGenetic diversity - The phylogenetic picture obtained\nfrom 22 isoenzyme loci for the three Mexican strains and\nseven T. cruzi standard strains type into different DTUs\nare shown in Fig. 1. Parasite strains from human (patient\nwith  chronic  Chagas  disease)  RyC-H;  reservoir  (P.\nperomyscus) RyC-R; and vector (T. barberi) RyC-V all be-\nlong to T. cruzi I, DTU 1 (DTU-1). Thus, T. cruzi parasites\nfrom human were characterized as HUM/ME/1997/MEX/\nRyC-H (TC 1); from vector-like VCT/ME/1997/MEX/RyC-\nV (TC 1); and from reservoir-like RES/ME/1997/MEX/ RyC-\nR (TC 1).\nBiological characterization - Genetic characteristics\nof T. cruzi parasites isolated from human, reservoir, and\nvector were analyzed in comparison with the pathological\nprofile in in vitro and in vivo analysis. Several parameters\nwere taken into account and included (a) growth kinetics\nof  epimastigotes  forms;  (b)  curves  of  parasitemia;  (c)\nmorphology of the parasite in the peripheral blood; (d)\ntissue tropism; (e) histopathological lesions (virulence);\nand (f) mortality rate of the infected animals.\nThe growth kinetics of the strains in LIT medium are\nshown  in  Fig.  2.  By  taking  the  average  of  six  mice,\nparasitemic peaks were considered as “high” above 500\ntrypomastigotes  and  as  “low”  from  100  to  500  trypo-\nmastigotes. Thus, parasites from human and vector hosts\nshowed slow multiplication in LIT medium and low para-\nsitemia.  The  morphological  study  of  peripheral  blood\ntrypomastigotes showed that the human and vector strains\npresented a predominance of broad forms (70%) by day\n28 post-inoculation (pi) and a low percentage of slender\nforms, while parasites isolated from reservoir show a high\nmultiplication rate with predominance of slender forms\n(60%) (Table I).\nThe reservoir strain which showed high parasitemia\nand predominance of slender forms, determined high mor-\ntality rates (31.25%) whereas this parameter was 5 to 10%\nfor the human and vector strains (Fig. 3).\nHistopathological study - The study of the tissue le-\nsions  and  parasite  tropism  indicated  that  the  reservoir\nstrain determined progressive skeletal muscle and myo-\ncardium lesions from the 14th day of infection with slight\nDays\nHuman strain\n(parasites × 104)\nBroad\nforms (%)\n3.53\n(70.6)\n15.32\n(74)\n40.39\n(73)\n\n(65)\n29.58\n(71)\n13.52\n(69)\n1.47\n(29.4)\n5.37\n(26)\n14.94\n(27)\n41.99\n(35)\n12.08\n(29)\n6.08\n(31)\nVector strain\n(parasites × 104)\nBroad\nforms (%)\n2.96\n(74.2)\n14.22\n(73)\n33.26\n(74)\n67.38\n(69)\n27.78\n(70)\n12.13\n(67)\n1.03\n(25.8 )\n5.25\n(27)\n11.69\n(26)\n30.27\n(31)\n11.90\n(30)\n5.96\n(33)\n18.09\nReservoir strain\n(parasites × 104)\nBroad\nforms (%)\n5.88\n(55)\n42.03\n(55)\n70.02\n(52)\n34.04\n(53)\n12.16\n(58)\n5.92\n(57)\n4.79\n(45)\n34.38\n(45)\n64.63\n(48)\n30.36\n(47)\n8.80\n(42)\n4.47\n(43)\n10.39\n19.6\nExperimental  infection  from  infected  mice  for  each  strain  of  T.  cruzi  from  different  sources.  T.  cruzi  parasites  from  reservoir\n(Peromyscus peromyscus) RyC-R show highest percentage of slender forms associated with mortality.\nflammatory infiltrate, which was clearly more severe in\nBALB/c mice infected with parasites from reservoir RyC-\nR than human RyC-H and vector RyC-V, T. cruzi strains.\nIn summary, the biological profiles show a different\npathogenic  pattern  among  T.  cruzi  strains.  The  human\nand vector T. cruzi strains show low mortality and a slow\nincrease of parasitemia with peaks of parasitemia between\n26 and 28 days pi associated with a profile characteristic\nof the biodeme type III (Table II). Parasites isolated from\nthe reservoir show parasitemic profiles of the biodeme\ntype II with a mild multiplication of parasites and with\npeaks of parasitemia between 18 and 21 days pi and also\nshow high mortality (Table II).",
         "https://academic.oup.com/crawlprevention/governor?content=%2fjxb%2fOUP%2fRunPdfHandler%3fjournal%3djxb%26volume%3d57%26issue%3d9%26page%3d1899%26multimediaId%3d1715826",
         "Laboratorio de Fisiopatología de Enfermedades Crónicas, Centro de Investigación Biomédica de Oriente, Instituto Mexicano del\nSeguro Social, Km 4.5 Carretera Federal Atlixco-Metepec, Atlixco, Puebla, México *Centre d’Etudes sur le Polymorphisme des\nMicro-Organismes, Montpellier, France **Facultad de Medicina, Benemérita Universidad Autónoma de Puebla, Puebla, México\n***Laboratorio de Parasitología, Centro de Investigaciones Regionales, Dr. Hideyo Noguchi, Mérida, Yucatán, México\nIn this study, three strains of  Trypanosoma cruzi were isolated at the same time and in the same endemic region\nin Mexico from a human patient with chronic chagasic cardiomyopathy (RyC-H); vector (Triatoma barberi) (RyC-V);\nand rodent reservoir (Peromyscus peromyscus) (RyC-R). The three strains were characterized by multilocus enzyme\nelectrophoresis,  random  amplified  polymorphic  DNA,  and  by  pathological  profiles  in  experimental  animals\n(biodemes). Based on the analysis of genetic markers the three parasite strains were typed as belonging to T. cruzi\nI  major  group,  discrete  typing  unit  1.  The  pathological  profile  of  RyC-H  and  RyC-V  strains  indicated  medium\nvirulence and low mortality and, accordingly, the strains should be considered as belonging to biodeme Type III. On\nthe other hand, the parasites from RyC-R strain induced more severe inflammatory processes and high mortality\n(> 40%) and were considered as belonging to biodeme Type II. The relationship between genotypes and biological\ncharacteristics in T. cruzi strains is still debated and not clearly understood. An expert committee recommended in\n1999 that Biodeme Type III would correspond to T. cruzi I group, whereas Biodeme Type II, to T. cruzi II group. Our\nfindings suggest that, at least for Mexican isolates, this correlation does not stand and that biological characteris-\ntics  such  as  pathogenicity  and  virulence  could  be  determined  by  factors  different  from  those  identified  in  the\ngenotypic  characterization\nKey words: Trypanosoma cruzi - genetic characterization - major groups - discrete typing units - pathogenic profile - biodemes\nTrypanosoma cruzi is the etiologic agent of Chagas\ndisease affecting approximately 20 million people in Cen-\ntral and South America (Moncayo 1997). T. cruzi is a para-\nsitic protozoan that consists of a heterogeneous popula-\ntion composed of a pool of strains circulating in both the\ndomestic and sylvatic cycles in humans, vectors, and ani-\nmal reservoirs (Souto et al. 1996).\nIn Chagas disease, different clinical pictures predomi-\nnate in different areas, e.g., in Brazil, the asymptomatic or\nindeterminate form is the most common (60-70%), followed\nby the cardiac and digestive forms (20-30 and 8-10%, re-\nspectively); however, in central Brazil and Chile, the di-\ngestive form of Chagas disease predominates, whereas it\nis practically non-existent in Venezuela and Central America\n(Luquetti et al. 1986, Dias 1992). The reason for this geo-\ngraphical  heterogeneity  and  why  different  patients  de-\nFinancial support: Fosiza-Conacyt project 9980802019\n+Corresponding author: rycardoperez@yahoo.com.mx\n++Sponsor from Institut de Recherche pour le Développement\nand the Centre National de Recherche Scientifique\nReceived 7 November 2005\nAccepted 19 July 2006\nvelop  different  clinical  forms  poses  the  question  of  its\nrelationship to the genetic heterogeneity of T. cruzi popu-\nlations in different areas.\nhavior in animal models has indeed shown a correlation\nbetween T. cruzi natural strain behavior and the genetic\nvariability  in  different  geographical  areas  (Andrade  &\nMagalhaes 1997). Biodeme Type III would correspond to\nT. cruzi major I group, whereas Biodeme Type II, to T.\ncruzi II major group (Anonymous 1999).\nGenetic studies are important to clarify the intraspe-\ncific heterogeneity of the parasite; however, the ability to\nstudy the biological behavior and the host-parasite rela-\ntionship is crucial for the evaluation of the impact of the\ngenetic  diversity  of  the  parasite  regarding  its  relevant\nmedical properties such as pathogenicity and virulence.\nIn  this  work  we  attempt  to  understand  the  genetic\nstructure,  pathogenic  pattern,  and  epidemiological  and\nclinical implications of T. cruzi parasites from an endemic\narea in Mexico. Mexican strains of T. cruzi isolated from\nhuman, vector, and animal reservoir at the same time and\nin the same endemic area were genetically characterized\nby multilocus enzyme electrophoresis (MLEE), random\namplified polymorphic DNA (RAPD), and pathological pro-\nfiles by biological in vitro and in vivo analysis (biodemes).\nMATERIALS  AND  METHODS\nAnimals\nMice - A total of 48 BALB/c female mice weighing 25-\n30 g were used at 10 to 12 weeks of age for each isolate.\nMice were maintained in a temperature-controlled envi-\nronment and provided with a balanced mouse ration and\nwater ad libitum. All animals were maintained in the same\nfacility for a minimum of two weeks before infection.\nParasites - Three T. cruzi isolates were obtained, re-\nspectively, from feces of adult Triatoma barberi (RyC-V),\nperipheral blood of Peromyscus peromyscus (RyC-R), a\nrodent from the Muridae family, and by xenodiagnostics\nof  human  hosts  with  chronic  Chagas  disease  (RyC-H).\nThe strains were isolated during epidemiological studies\nin Puebla, Mexico. Isolates were maintained by serial in-\ntraperitoneal (ip) passages in 10-week-old BALB/c mice\nand were grown axenically as epimastigote forms (culture\nforms) at 28°C in liver infusion-tryptose medium (LIT)\n(Difco Laboratories, Detroit, MI) supplemented with 10%\nheat-inactived fetal calf serum (Gibco, Grand Island, NY).\nGenetic characterization\nMLEE - Methods for isolating, growing in LIT me-\ndium, harvesting, and storing the stocks have been previ-\nously described (Tibayrenc & Le Ray 1984). The condi-\ntions for electrophoresis on cellulose acetate were per-\nformed as described by Ben Abde-rrazak et al. (1993) with\nslight  modifications. Twenty  enzyme  systems  were  as-\nsayed (Barnabé et al. 2000), namely, aconitase (EC 4.2.1.3,\nACON), alanine aminotransferase (EC 2.6.1.2, ALAT), dia-\nphorase (EC 1.6.99.2, DIA), glyceraldehyde-3-phosphate\ndehydrogenase (EC 1.2.1.12, GAPD), glutamate dehydro-\ngenase NAD+ (EC 1.4.1.2, GDH-NAD+), glutamate dehy-\ndrogenase NADP+ (EC 1.4.1.4. GDH-NADP+), aspartate\namino transferase (EC 2.6.1.1, GOT), glucose-6-phosphate\ndehydrogenase (EC 1.1.1.49, G6PD), glucose-6-phosphate\nisomerase (EC 5.3.1.9, GPI), isocitrate dehydrogenase (EC\n1.1.1.42, IDH), leucine aminopeptidase (cytosol aminopep-\ntidase) (EC 3.4.11.1, LAP), malate dehydrogenase (EC\n1.1.1.37, MDH), malate dehydrogenase (oxaloacetate de-\ncarboxylating, NADP+) or malic enzyme (EC 1.1.1.40, ME),\nmannose-phosphate isomerase (EC 5.3.1.8, MPI), nucleo-\nside hydrolase (EC 2.4.2.1, NHi); substrate: inosine, pep-\ntidase 1 (EC 3.4.22.3) (formerly EC 3.4.4.12), (PEP-1); sub-\nstrate: leucyl-leucyl-leucine, peptidase 2 (EC 3.4.22.4) (for-\nmerly EC 3.3.3.24), PEP-2; substrate: leucyl-L-alanine), 6-\nphosphoglucomutase dehydrogenase (EC 1.1.1.44, 6PGD),\nphosphoglucomutase (EC 5.4.2.2) (formerly EC 2.7.5.1.),\nPGM), and superoxide dismutase (EC 1.15.1.1, SOD).\nRAPD - was performed according to Tibayrenc et al.\n(1993) with slight modifications. Ten different decamer\nprimers from the A-kit of Operon Technologies were used.\nBiological characterization\nGrowth kinetics of epimastigote forms - A total of 25 ×\n106 parasites was inoculated in a final volume of 50 ml of\nLIT medium. Cultures were incubated at 27ºC. Kinetics\nwere followed for 24 days. The culture concentration was\nestimated  three  times  every  two  days  by  counting  in  a\nThomas chamber. Analysis of growth kinetics takes into\naccount the doubling time, estimated in hours, calculated\nfrom the log phase and the parasite concentration in 10–6\nparasites per ml (10–6 p/ml) and the end of the log phase.\nExperimental infection in mice (biodeme) - Experi-\nmental infection was performed in 4- or 5-week-old BALB/\nc female mice. A total of 144 mice were inoculated intrader-\nmally with 1 × 104 bloodstream trypomastigotes from in-\nfected  mice  for  each  strain.  Parasitemia  was  evaluated\nevery three days during the infection by microscopic ex-\namination of fresh blood samples obtained from the tail.\nParasitemic levels were expressed as logarithms of the\nmedia of parasites in peripheral blood of six mice for each\ngroup. Mortality rate in relation to infection was evalu-\nated  from  7  to  48  days  after  infection.  Morphology  of\ntrypomastigotes in peripheral blood was evaluated by the\npercentage of slender and broad forms, as seen in smears\nstained with the Giemsa method.\nHistopathological  analyses  -  These  analyses  were\nperformed on 144 of the infected mice corresponding to\neach T. cruzi strain and on 48 uninfected control mice.\nMice were killed by cervical dislocation and tissues were\nobtained each week for 7 weeks. Histological studies were\ncarried out in infected and control animals from skeletal\nmuscle, heart muscle, esophagus, colon, spleen, and liver.\nSamples were fixed in 10% neutral-buffered formalin and\nembedded in paraffin. Sections were stained with hema-\ntoxylin and eosin and examined by light microscope.\nTwo serial sections cut from three different parts of\neach organ were examined. The total surface of the sec-\ntions was at least 2 cm, and the lesions were recorded and\nclassified according to their severity. The severity of the\nlesions was generally evaluated with respect to histologi-\ncal characteristics rather than to the number of lesions,\nwhich varied between the sections. Each type of lesion\nwas assigned a severity index (scored from 1 to 3) accord-\ning  to  its  severity  and  potential  pathological  conse-\nquences.\nStatistical analysis - For both MLEE and RAPD data,\nJaccard’s distance (Jaccard 1908) was used. Each RAPD\nand MLEE gel band was coded with a number, starting\nwith 1 for the slowest band. The distance was estimated\naccording to the formula:\nD = 1 – [a/(a + b + c)]\nwhere a is the number of bands that are common to the\ntwo  compared  genotypes,  b  is  the  number  of  bands\npresent  in  the  first  genotype  and  absent  in  the  second,\nand c is the number of bands absent in the first genotype\nand present in the second.\nThe UPGMA method (unweighted pair-group method\nwith arithmetic averages) (Sneath & Sokal 1973) was used\nand for comparative evaluation of pathological profile of\nmice infected with human, vector and reservoir T. cruzi\nstrains a non-parametric Fisher test was applied.\nData analysis of the biological parameters - In a first\napproach, the average values of all biological parameters\nwere compared by Student’s t-test between each of the\ndifferent  strains  (human,  vector  and  reservoir  T.  cruzi\nstrains)  genetically  characterized  by  MLEE/RAPD\nanalysis.\nRESULTS\nGenetic diversity - The phylogenetic picture obtained\nfrom 22 isoenzyme loci for the three Mexican strains and\nseven T. cruzi standard strains type into different DTUs\nare shown in Fig. 1. Parasite strains from human (patient\nwith  chronic  Chagas  disease)  RyC-H;  reservoir  (P.\nperomyscus) RyC-R; and vector (T. barberi) RyC-V all be-\nlong to T. cruzi I, DTU 1 (DTU-1). Thus, T. cruzi parasites\nfrom human were characterized as HUM/ME/1997/MEX/\nRyC-H (TC 1); from vector-like VCT/ME/1997/MEX/RyC-\nV (TC 1); and from reservoir-like RES/ME/1997/MEX/ RyC-\nR (TC 1).\nBiological characterization - Genetic characteristics\nof T. cruzi parasites isolated from human, reservoir, and\nvector were analyzed in comparison with the pathological\nprofile in in vitro and in vivo analysis. Several parameters\nwere taken into account and included (a) growth kinetics\nof  epimastigotes  forms;  (b)  curves  of  parasitemia;  (c)\nmorphology of the parasite in the peripheral blood; (d)\ntissue tropism; (e) histopathological lesions (virulence);\nand (f) mortality rate of the infected animals.\nThe growth kinetics of the strains in LIT medium are\nshown  in  Fig.  2.  By  taking  the  average  of  six  mice,\nparasitemic peaks were considered as “high” above 500\ntrypomastigotes  and  as  “low”  from  100  to  500  trypo-\nmastigotes. Thus, parasites from human and vector hosts\nshowed slow multiplication in LIT medium and low para-\nsitemia.  The  morphological  study  of  peripheral  blood\ntrypomastigotes showed that the human and vector strains\npresented a predominance of broad forms (70%) by day\n28 post-inoculation (pi) and a low percentage of slender\nforms, while parasites isolated from reservoir show a high\nmultiplication rate with predominance of slender forms\n(60%) (Table I).\nThe reservoir strain which showed high parasitemia\nand predominance of slender forms, determined high mor-\ntality rates (31.25%) whereas this parameter was 5 to 10%\nfor the human and vector strains (Fig. 3).\nHistopathological study - The study of the tissue le-\nsions  and  parasite  tropism  indicated  that  the  reservoir\nstrain determined progressive skeletal muscle and myo-\ncardium lesions from the 14th day of infection with slight\nDays\nHuman strain\n(parasites × 104)\nBroad\nforms (%)\n3.53\n(70.6)\n15.32\n(74)\n40.39\n(73)\n\n(65)\n29.58\n(71)\n13.52\n(69)\n1.47\n(29.4)\n5.37\n(26)\n14.94\n(27)\n41.99\n(35)\n12.08\n(29)\n6.08\n(31)\nVector strain\n(parasites × 104)\nBroad\nforms (%)\n2.96\n(74.2)\n14.22\n(73)\n33.26\n(74)\n67.38\n(69)\n27.78\n(70)\n12.13\n(67)\n1.03\n(25.8 )\n5.25\n(27)\n11.69\n(26)\n30.27\n(31)\n11.90\n(30)\n5.96\n(33)\n18.09\nReservoir strain\n(parasites × 104)\nBroad\nforms (%)\n5.88\n(55)\n42.03\n(55)\n70.02\n(52)\n34.04\n(53)\n12.16\n(58)\n5.92\n(57)\n4.79\n(45)\n34.38\n(45)\n64.63\n(48)\n30.36\n(47)\n8.80\n(42)\n4.47\n(43)\n10.39\n19.6\nExperimental  infection  from  infected  mice  for  each  strain  of  T.  cruzi  from  different  sources.  T.  cruzi  parasites  from  reservoir\n(Peromyscus peromyscus) RyC-R show highest percentage of slender forms associated with mortality.\nflammatory infiltrate, which was clearly more severe in\nBALB/c mice infected with parasites from reservoir RyC-\nR than human RyC-H and vector RyC-V, T. cruzi strains.\nIn summary, the biological profiles show a different\npathogenic  pattern  among  T.  cruzi  strains.  The  human\nand vector T. cruzi strains show low mortality and a slow\nincrease of parasitemia with peaks of parasitemia between\n26 and 28 days pi associated with a profile characteristic\nof the biodeme type III (Table II). Parasites isolated from\nthe reservoir show parasitemic profiles of the biodeme\ntype II with a mild multiplication of parasites and with\npeaks of parasitemia between 18 and 21 days pi and also\nshow high mortality (Table II).\nDISCUSSION\nGenetic and biological characteristics of the T. cruzi\nstrains represent a tool for the understanding of different\naspects of the epidemiology Chagas disease. Populations\nof T. cruzi are highly heterogeneous and by using genetic\nmarkers were distributed into two major phylogenetic lin-\neages (Tibayrenc 1995, Souto et al. 1996), which were\nnamed as T. cruzi I and T. cruzi II (Anonymous 1999).\nStudies carried out in Southern Cone Countries of South\nAmerica indicate that T. cruzi I is dominant in the sylvatic\ncycle and T. cruzi II is dominant in the domestic cycle of\nthe parasite transmission (Zingales et al. 1998, Diosque et\nal. 2003). Very few data on the epidemiological distribu-\ntion of the two T. cruzi major groups are available for the\nother countries of the Americas e.g. Mexico (Zavala Castro\net al. 1992, Espinosa et al. 1998) and Central America (Higo\net al. 2000). In this study, three Mexican T. cruzi parasites\nfrom human, vector and reservoir were genetically identi-\nfied  as  belonging  to  T.  cruzi  I;  moreover,  high  genetic\nidentity was shown between them (Jaccard’s distances <\n0.20). This observation is in agreement with the data show-\ning the predominance of T. cruzi I in stocks isolated from\n8 of the 32 states of the Mexican Republic (Bosseno et al.\n2002).\nmononuclear infiltrates and low parasitism of myocytes.\nBy the 21st day of infection, a progressive parasitism of\nmyocardiocytes and destruction of myocells with dense\ninflammatory infiltration was seen, with focal myocyte\nnecrosis and diffuse mononuclear cell infiltration. Tissue\nparasitism was prominent in skeletal muscles and progres-\nsive until 24 days of infection.\nHistopathological lesions determined by human and\nvector  T.  cruzi  strains  from  14  to  30  days  of  infection\nshowed the same evolution and histopathological aspects.\nIn general, the parasitism and inflammatory infiltration\nwere present at 14 and 20 days pi, varying from slight to\nmoderate. Lesions were more prominent in cardiac muscles\nand intensified up to the 28th day of infection. Analysis\nof the severity of tissues lesions showed an intense in-\nName\nDate\nBiological source\nRyC-H\nVII/98\nPatient with chronic\nchagasic cardiomyopathy\nRyC-V\nIV-98\nRyC-R\nVIII/98\nTriatoma barberi\nPeromyscus peromyscus\nGeographical source\nMolcaxac, Puebla, Mexico\nMolcaxac, Puebla, Mexico\n21 days\nBroad forms (70%)\nBroad forms (70%)\nSlender forms (70%)\nMortality rate in mice\nPeak of growth\nMorphology\nTissue  tropism\nHistopathogenicity\nBiodemes\nIII\nSkeletal muscle\nCardiac muscle\nCardiac muscle\nCardiac muscle\nIII\nCardiac muscle\nCardiac muscle\nII\nGenetic characterization\nLineage1, DTU-1\nLineage 1, DTU-1\nLineage 1, DTU-1\nStudy of the various biological parameters in response to infection by strains of T. cruzi show clear differences in response to\ninfection by strains from different sources. T. cruzi parasites from human (with chronic chagasic cardiomyopathy) and vector did not\ninduce a significant inflammatory process in the heart and muscles. In contrast, T. cruzi parasites from reservoir induce an intense\ninflammatory process in the heart and skeletal muscles.\nT. cruzi II has been shown to be heterogeneous and\ndifferent genetic subdivisions have been described, named\nDTUs (Brisse et al. 1998). It was suggested that different\nDTUs correspond to distinct epidemiological pictures and\nare proposed as a reference framework for genetic vari-\nability and for biological characterization studies of T. cruzi\nstrains (Barnabé et al. 2000). The geographical distribu-\ntion of the six DTUs is present in the entire range of Chagas\ndisease. On the other hand, DTU 2b has been mainly iso-\nlated in Brazil and Chile, while DTU 2d was mainly en-\ncountered in Chile and Bolivia (Barnabé et al. 2000). In\nthis study, the three T. cruzi strains isolated in the same\nregion from human, domestic and wild transmission cycles\nwere characterized by multigene typing as belonging to\nDTU 1 and exhibited strong levels of linkage. T. cruzi I\n(DTU 1) has been considered a homogeneous group. The\nfindings of the present study show clear biological differ-\nences between the three strains regarding the profile of\nparasitemia,  tissue  tropism,  pathogenicity  pattern,  and\nmortality. In fact, T. cruzi parasites from human and vec-\ntor (belonging to biodeme Type III) did not induce a sig-\nnificant inflammatory process in the heart and muscles. In\ncontrast, T. cruzi parasites from the reservoir (belonging\nto biodeme Type II) induced an intense inflammatory pro-\ncess in the heart and skeletal muscles.\nAccording  to  Andrade  and  Magalhães  (1997),  al-\nthough  genetic  studies  are  important  to  clarify  the  in-\ntraspecific heterogeneity of the parasite, only the study\nof the biological behavior could clarify the importance of\ndifferent strains in the determination of clinicopathologi-\ncal manifestations of Chagas disease. Our results could\nbe explained, in part, by the fact that many studies have\nbeen performed using T. cruzi strains isolated for more\nthan  20  years  and  that  genetic  intrinsic  characteristics\nand clonal selection could be influenced by handling in\nthe laboratory. Although experimental evidence of the sta-\nbility of strain behavior has been obtained by biochemi-\ncal  and  biological  characterization  after  parasites  were\nsubjected to different conditions of maintenance and cul-\ntivation, the main biological characteristics of the strains,\nsuch as infectivity and inoculum size, tended to decrease\nin association with maintenance and cultivation condi-\ntions (Magalhaes et al. 1985). Decrease in virulence has\nbeen observed when the culture forms were used or when\nthe infection with low inoculum was employed. Addition-\nally, the passage of T. cruzi parasites through the vector\nhas a positive influence on the virulence (higher levels of\nparasitemia and predominance of slender forms in mice)\n(Magalhaes et al. 1996). In this work we employed T. cruzi\nparasites recently isolated from the same endemic area.\nThis procedure would guarantee the reliability of the com-\nparative studies.\nInfection with T. cruzi is an enzootic disease extend-\ning from the US to South America (sylvatic cycle), that\ncan  lead  to  human  infection  when  the  insect  vector\n(triatomine  bugs)  adapts  to  human  dwelling  (domestic\ncycles). This adaptation occurs in different manners and\nat different times (Prata 2001). The presence of different\ntypes of T. cruzi strains in endemic areas may bear influ-\nence on the biological characteristics as well as pathoge-\nnicity. Studies with cloned populations of natural strains\nhave demonstrated either homogeneity or heterogeneity\nof several clones including differences in virulence and\npathogenicity. Thus, in an epidemiological study, cloned\npopulations  cannot  be  taken  as  representative  of  the\nstrains isolated from different geographical areas.\nThe expert committee in 1999 recommended that iso-\nlates typed as Biodeme Type III (Andrade 1974) are equiva-\nlent to T. cruzi I, and isolates typed as Biodeme Type II\nare equivalente to T. cruzi II (Anonymous 1999), our data\nclearly show differences in the biological behavior of three\nstrains classified as T. cruzi I, although T. cruzi I is con-\nsidered a homogeneous group based on the analysis of\ngenetic markers. Therefore, our T. cruzi I isolates were\ntyped as Biodeme Type II and Type III.\nIn conclusion, our findings suggest that for Mexican\nisolates, characteristics such as pathogenicity and viru-\nlence of T. cruzi strains must be described by a combina-\ntion of genetic and biological characteristics. Reliance on\nthe genotype alone may confer significance of little bio-\nlogical relevance and could be determined by factors dif-\nferent from those identified in the genotypic characteriza-\ntion.",
         "",
         "Trypanosoma cruzi strains isolated from human, vector, and animal reservoir in the same endemic region in Mexico and typed as T. cruzi I, discrete typing unit 1 exhibit considerable biological diversity.",
         "None"
        ],
        [
         "9",
         "0007fad20e444cb908ccea0cb97c2e31ea7212c6",
         "Phospholipase C-type beta2 (PLCbeta2) is expressed in a subset of cells within mammalian taste buds. This enzyme is involved in the transduction of sweet, bitter, and umami stimuli and thus is believed to be a marker for gustatory sensory receptor cells. We have developed transgenic mice expressing green fluorescent protein (GFP) under the control of the PLCbeta2 promoter to enable one to identify these cells and record their physiological activity in living preparations. Expression of GFP (especially in lines with more than one copy integrated) is strong enough to be detected in intact tissue preparations using epifluorescence microscopy. By immunohistochemistry, we confirmed that the overwhelming majority of cells expressing GFP are those that endogenously express PLCbeta2. Expression of the GFP transgene in circumvallate papillae occurs at about the same time during development as endogenous PLCbeta2 expression. When loaded with a calcium-sensitive dye in situ, GFP-positive taste cells produce typical Ca2+ responses to a taste stimulus, the bitter compound cycloheximide. These PLCbeta2 promoter-GFP transgenic lines promise to be useful for studying taste transduction, sensory signal processing, and taste bud development.",
         "J. W. Kim,C. Roberts,Y. Maruyama,Stephanie Berg,S. Roper,Nirupa Chaudhari",
         "None",
         "https://academic.oup.com/chemse/article-pdf/31/3/213/979519/bjj021.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://academic.oup.com/chemse/article-pdf/31/3/213/979519/bjj021.pdf",
         "None",
         "Liposome-mediated transfection of mature taste cells.;PLCbeta2-independent behavioral avoidance of prototypical bitter-tasting ligands.;Morphologic characterization of rat taste receptor cells that express components of the phospholipase C signaling pathway;Electrophysiological Characterization of Voltage-Gated Currents in Defined Taste Cell Types of Mice;Sour Taste Stimuli Evoke Ca2+ and pH Responses in Mouse Taste Cells;Coding of Sweet, Bitter, and Umami Tastes Different Receptor Cells Sharing Similar Signaling Pathways;Individual mouse taste cells respond to multiple chemical stimuli;Taste receptor cell responses to the bitter stimulus denatonium involve Ca2+ influx via store-operated channels.;An amino-acid taste receptor;“Type III” cells of rat taste buds: Immunohistochemical and ultrastructural studies of neuron‐specific enolase, protein gene product 9.5, and serotonin;Bitter taste transduced by PLC-β2-dependent rise in IP3 and α-gustducin-dependent fall in cyclic nucleotides;IP(3) receptor type 3 and PLCbeta2 are co-expressed with taste receptors T1R and T2R in rat taste bud cells.;Mouse taste cells with glialike membrane properties.;G Protein βγ Complexes in Circumvallate Taste Cells Involved in Bitter Transduction;T2Rs Function as Bitter Taste Receptors;A metabotropic glutamate receptor variant functions as a taste receptor;Gγ13 colocalizes with gustducin in taste receptor cells and mediates IP3 responses to bitter denatonium;Directing Gene Expression to Gustducin-Positive Taste Receptor Cells;Identification of a phospholipase C beta subtype in rat taste cells.;Characterization and solubilization of bitter-responsive receptors that couple to gustducin.;Methylation of an ETS site in the intron enhancer of the keratin 18 gene participates in tissue-specific repression;Roles of phospholipase C beta2 in chemoattractant-elicited responses.;Role of methylation in maintenance of positionally restricted transgene expression in developing muscle.;The 5' flanking region of the rat LAP (C/EBP beta) gene can direct high-level, position-independent, copy number-dependent expression in multiple tissues in transgenic mice.;High-level expression of the rat whey acidic protein gene is mediated by elements in the promoter and 3' untranslated region",
         "Faithful expression of GFP from the PLCbeta2 promoter in a functional class of taste receptor cells.",
         "None"
        ],
        [
         "10",
         "0007fc19b59d115d1d89d19af7aaffcd72ff4013",
         "The self‐controlled case series method was developed to investigate associations between acute outcomes and transient exposures, using only data on cases, that is, on individuals who have experienced the outcome of interest. Inference is within individuals, and hence fixed covariates effects are implicitly controlled for within a proportional incidence framework. We describe the origins, assumptions, limitations, and uses of the method. The rationale for the model and the derivation of the likelihood are explained in detail using a worked example on vaccine safety. Code for fitting the model in the statistical package STATA is described. Two further vaccine safety data sets are used to illustrate a range of modelling issues and extensions of the basic model. Some brief pointers on the design of case series studies are provided. The data sets, STATA code, and further implementation details in SAS, GENSTAT and GLIM are available from an associated website. Copyright © 2005 John Wiley & Sons, Ltd.",
         "H. Whitaker,C. Paddy Farrington,B. Spiessens,P. Musonda",
         "H. Whitaker,C. Paddy Farrington,B. Spiessens,P. Musonda",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/sim.2302",
         "The self-controlled case series method was developed to investigate associations between acute out-\ncomes and transient exposures, using only data on cases, that is, on individuals who have experienced\nthe outcome of interest. Inference is within individuals, and hence xed covariates eects are implic-\nitly controlled for within a proportional incidence framework. We describe the origins, assumptions,\nlimitations, and uses of the method. The rationale for the model and the derivation of the likelihood\nare explained in detail using a worked example on vaccine safety. Code for tting the model in the\nstatistical package STATA is described. Two further vaccine safety data sets are used to illustrate a\nrange of modelling issues and extensions of the basic model. Some brief pointers on the design of\ncase series studies are provided. The data sets, STATA code, and further implementation details in\nSAS, GENSTAT and GLIM are available from an associated website. Copyright ? 2005 John Wiley &\nSons, Ltd.",
         "",
         "a time-varying exposure and an outcome event. It was developed to investigate associations\nbetween vaccination and acute potential adverse events [1, 2]. Subsequently the method was\napplied in other settings, for example, to investigate non-acute events such as autism, and has\nbeen used more widely in pharmacoepidemiology and other areas of epidemiology.\nThis tutorial has three main aims. First, we provide an account of the background to the\nmethod, and explain how it works by means of a worked example. Second, we describe\nhow the method may be implemented in commercial software packages. The software and\nseveral of the data sets are available online from http:==statistics.open.ac.uk=sccs. Third, we\ndescribe in detail how the method is applied, the assumptions it requires and its limitations,\nthrough further practical examples. We do not aim to present the most general version of\nthe model, which can readily be extended to handle time-varying continuous exposures, and\nto non-parametric modelling of the baseline incidence; pointers are given where appropriate\nto these more general versions. Instead we concentrate on transient exposures and parametric\nmodelling.\nThe paper is organized as follows. In Section 2 we describe the background of the method,\nbriey discuss alternative approaches, set out the main advantages and limitations of the case\nseries method, and describe how it has been used. In Section 3, we work through an example,\nsmall enough to allow all calculations to be done by hand, and set out the more general theory,\nincluding the likelihood and the associated regression model. In Section 4 we describe how\nthe model is tted using STATA. In Section 5 we present two other data sets, which we use\nfor illustrative purposes in Section 6, where we discuss at some length the assumptions and\nlimitations of the method, as well as issues in the analysis of case series studies. In Section 7\nwe briey discuss the design of case series studies. Section 8 is a short discussion, including\nareas for further research.\n2. THE SELF-CONTROLLED CASE SERIES METHOD\n2.1. Background\nThe method described in this tutorial was developed to analyse record linkage data on MMR\nvaccination and aseptic meningitis [3, 4]. Data on episodes of aseptic meningitis arising in\nchildren aged 1–2 years over a dened calendar time period (the observation period) were\nobtained from laboratory and hospital records. Throughout the paper, the term ‘case’ refers to\nan individual who has experienced one or more events of interest over the observation period.\nThese cases were linked to vaccination records: the resulting data set thus consisted of cases\nand their exposures. No precise denominators were available, nor was it wholly clear from\nwhat population the cases arose, since the catchment areas of the hospitals from which the\ncases were obtained were not clearly dened. Thus population-based cohort and case-control\nstudies would have required some ingenuity to avoid confounding, as vaccine coverage is not\nuniform. Furthermore, answers about a possible association with MMR vaccine were required\nquickly. The case series method was developed in response to these requirements. A positive\nassociation between vaccination with the Urabe mumps strain and aseptic meningitis in the\nperiod 15–25 days post-vaccination was conrmed, and the composition of MMR vaccines\nused in the UK was changed.\n2.2. Relation to other methods\nIn a case series analysis, only cases are sampled. To take account of this, the likelihood\nis conditional on an event having occurred during the observation period. The case series\nlikelihood is thus based on the probability density that an event occurred when it did in relation\nto the individual’s time of exposure, given that the event occurred during the observation\nperiod. Note that this derives from cohort logic: event times are regarded as random, whereas\nexposure times are regarded as xed. The focus of estimation, as in a cohort study, is the\nrelative incidence, or relative hazard of an event. This is the ratio of the rate (or hazard) of\nevents in a given post-exposure period, to the rate of events in the absence of the exposure.\nAn early approach to analysing case-only data was proposed by Aalen et al. [5]. This was\nlater generalized by Prentice [6], who suggested applying Cox regression to the cohort of\ncases, eectively ignoring the fact that only cases are sampled. This provides a valid test of\nthe null hypothesis of no association, but does not yield a useful eect estimate. This method\nwas used by Miller et al. [3] to analyse the MMR vaccination and aseptic meningitis data\ndescribed above, since the case series method had not yet been published at this time.\nMaclure [7] proposed a case-control method involving only cases, known as the case-cross-\nover method. In this method, exposures in the period immediately preceding the event time\nare compared to exposures at earlier ‘control’ times in the history of the case. There are\nseveral variants of this method, reviewed by Greenland [8], and the case-crossover approach\nhas been used in many settings [9]. However, a necessary condition for the case-crossover\nmethod to apply is that the exposure distribution is stationary. Indeed, a stronger assumption is\nrequired in many settings, namely that the exposure distribution in successive time periods is\nexchangeable [10]. Without this condition, which mimics the tacit assumption in case-control\nstudies that the ordering of the controls is immaterial, multiple logistic regression applied to\ncase-crossover data with several control periods does not produce consistent estimates. The\ncase series method does not require such an assumption. In particular, age or time eects can\nbe allowed for in much the same way as in a cohort study.\nFeldmann [11, 12] proposed an approach similar to the case series method insofar as it is\nalso based on cohort logic, conditioning on the probability that one or more events occur\nwithin each individual’s observation period. For rare events, Feldmann’s method coincides\napproximately with the case series model without adjustment for age. However, in general,\nthe method does not implicitly control for xed covariates.\nThe case series method incorporates features of these other methods: like Prentice’s, it\ncontrols for age; like Maclure’s, it controls for xed confounders; and like Feldmann’s, it\nallows for multiple events.\n2.3. Advantages and limitations\nThe main advantages of the case series method over other methods of analysis can be sum-\nmarized as follows.\n1. It is based only on cases, and provides consistent estimates of the relative incidence.\n2. It controls implicitly for all xed confounders, that is to say, confounders that do\nnot vary with time over the observation period, such as variables relating to genetics,\nlocation, socio-economic status, gender, individual frailty, severity of underlying disease,\netc.",
         "",
         "",
         "https://academic.oup.com/chemse/article-pdf/31/3/213/979519/bjj021.pdf",
         "The self-controlled case series method was developed to investigate associations between acute out-\ncomes and transient exposures, using only data on cases, that is, on individuals who have experienced\nthe outcome of interest. Inference is within individuals, and hence xed covariates eects are implic-\nitly controlled for within a proportional incidence framework. We describe the origins, assumptions,\nlimitations, and uses of the method. The rationale for the model and the derivation of the likelihood\nare explained in detail using a worked example on vaccine safety. Code for tting the model in the\nstatistical package STATA is described. Two further vaccine safety data sets are used to illustrate a\nrange of modelling issues and extensions of the basic model. Some brief pointers on the design of\ncase series studies are provided. The data sets, STATA code, and further implementation details in\nSAS, GENSTAT and GLIM are available from an associated website. Copyright ? 2005 John Wiley &\nSons, Ltd.\na time-varying exposure and an outcome event. It was developed to investigate associations\nbetween vaccination and acute potential adverse events [1, 2]. Subsequently the method was\napplied in other settings, for example, to investigate non-acute events such as autism, and has\nbeen used more widely in pharmacoepidemiology and other areas of epidemiology.\nThis tutorial has three main aims. First, we provide an account of the background to the\nmethod, and explain how it works by means of a worked example. Second, we describe\nhow the method may be implemented in commercial software packages. The software and\nseveral of the data sets are available online from http:==statistics.open.ac.uk=sccs. Third, we\ndescribe in detail how the method is applied, the assumptions it requires and its limitations,\nthrough further practical examples. We do not aim to present the most general version of\nthe model, which can readily be extended to handle time-varying continuous exposures, and\nto non-parametric modelling of the baseline incidence; pointers are given where appropriate\nto these more general versions. Instead we concentrate on transient exposures and parametric\nmodelling.\nThe paper is organized as follows. In Section 2 we describe the background of the method,\nbriey discuss alternative approaches, set out the main advantages and limitations of the case\nseries method, and describe how it has been used. In Section 3, we work through an example,\nsmall enough to allow all calculations to be done by hand, and set out the more general theory,\nincluding the likelihood and the associated regression model. In Section 4 we describe how\nthe model is tted using STATA. In Section 5 we present two other data sets, which we use\nfor illustrative purposes in Section 6, where we discuss at some length the assumptions and\nlimitations of the method, as well as issues in the analysis of case series studies. In Section 7\nwe briey discuss the design of case series studies. Section 8 is a short discussion, including\nareas for further research.\n2. THE SELF-CONTROLLED CASE SERIES METHOD\n2.1. Background\nThe method described in this tutorial was developed to analyse record linkage data on MMR\nvaccination and aseptic meningitis [3, 4]. Data on episodes of aseptic meningitis arising in\nchildren aged 1–2 years over a dened calendar time period (the observation period) were\nobtained from laboratory and hospital records. Throughout the paper, the term ‘case’ refers to\nan individual who has experienced one or more events of interest over the observation period.\nThese cases were linked to vaccination records: the resulting data set thus consisted of cases\nand their exposures. No precise denominators were available, nor was it wholly clear from\nwhat population the cases arose, since the catchment areas of the hospitals from which the\ncases were obtained were not clearly dened. Thus population-based cohort and case-control\nstudies would have required some ingenuity to avoid confounding, as vaccine coverage is not\nuniform. Furthermore, answers about a possible association with MMR vaccine were required\nquickly. The case series method was developed in response to these requirements. A positive\nassociation between vaccination with the Urabe mumps strain and aseptic meningitis in the\nperiod 15–25 days post-vaccination was conrmed, and the composition of MMR vaccines\nused in the UK was changed.\n2.2. Relation to other methods\nIn a case series analysis, only cases are sampled. To take account of this, the likelihood\nis conditional on an event having occurred during the observation period. The case series\nlikelihood is thus based on the probability density that an event occurred when it did in relation\nto the individual’s time of exposure, given that the event occurred during the observation\nperiod. Note that this derives from cohort logic: event times are regarded as random, whereas\nexposure times are regarded as xed. The focus of estimation, as in a cohort study, is the\nrelative incidence, or relative hazard of an event. This is the ratio of the rate (or hazard) of\nevents in a given post-exposure period, to the rate of events in the absence of the exposure.\nAn early approach to analysing case-only data was proposed by Aalen et al. [5]. This was\nlater generalized by Prentice [6], who suggested applying Cox regression to the cohort of\ncases, eectively ignoring the fact that only cases are sampled. This provides a valid test of\nthe null hypothesis of no association, but does not yield a useful eect estimate. This method\nwas used by Miller et al. [3] to analyse the MMR vaccination and aseptic meningitis data\ndescribed above, since the case series method had not yet been published at this time.\nMaclure [7] proposed a case-control method involving only cases, known as the case-cross-\nover method. In this method, exposures in the period immediately preceding the event time\nare compared to exposures at earlier ‘control’ times in the history of the case. There are\nseveral variants of this method, reviewed by Greenland [8], and the case-crossover approach\nhas been used in many settings [9]. However, a necessary condition for the case-crossover\nmethod to apply is that the exposure distribution is stationary. Indeed, a stronger assumption is\nrequired in many settings, namely that the exposure distribution in successive time periods is\nexchangeable [10]. Without this condition, which mimics the tacit assumption in case-control\nstudies that the ordering of the controls is immaterial, multiple logistic regression applied to\ncase-crossover data with several control periods does not produce consistent estimates. The\ncase series method does not require such an assumption. In particular, age or time eects can\nbe allowed for in much the same way as in a cohort study.\nFeldmann [11, 12] proposed an approach similar to the case series method insofar as it is\nalso based on cohort logic, conditioning on the probability that one or more events occur\nwithin each individual’s observation period. For rare events, Feldmann’s method coincides\napproximately with the case series model without adjustment for age. However, in general,\nthe method does not implicitly control for xed covariates.\nThe case series method incorporates features of these other methods: like Prentice’s, it\ncontrols for age; like Maclure’s, it controls for xed confounders; and like Feldmann’s, it\nallows for multiple events.\n2.3. Advantages and limitations\nThe main advantages of the case series method over other methods of analysis can be sum-\nmarized as follows.\n1. It is based only on cases, and provides consistent estimates of the relative incidence.\n2. It controls implicitly for all xed confounders, that is to say, confounders that do\nnot vary with time over the observation period, such as variables relating to genetics,\nlocation, socio-economic status, gender, individual frailty, severity of underlying disease,\netc.",
         "Semiparametric analysis of case series data;General population based study of the impact of tricyclic and selective serotonin reuptake inhibitor antidepressants on the risk of acute myocardial infarction;No evidence of an association between MMR vaccine and gait disturbance;Risk of myocardial infarction and stroke after acute infection or vaccination.;Safety of the trivalent inactivated influenza vaccine among children: a population-based study.;Control without separate controls: evaluation of vaccine safety using case-only methods.;The effect of transient exposures on the risk of an acute illness with low hazard rate.;Use of the inactivated intranasal influenza vaccine and the risk of Bell's palsy in Switzerland.;Does influenza vaccination increase consultations, corticosteroid prescriptions, or exacerbations in subjects with asthma or chronic obstructive pulmonary disease?;Exposure to tricyclic and selective serotonin reuptake inhibitor antidepressants and the risk of hip fracture.;Bacterial infections, immune overload, and MMR vaccine;Within‐subject exposure dependency in case‐crossover studies;Statistical assessment of the association between vaccination and rare adverse events post-licensure.;MMR and autism: further evidence against a causal association.;Idiopathic thrombocytopenic purpura and MMR vaccine;Intussusception among infants given an oral rotavirus vaccine.;Bias in the case – crossover design: implications for studies of air pollution;Does influenza vaccination exacerbate asthma? Analysis of a large cohort of children with asthma. Vaccine Safety Datalink Team.;Outbreak of aseptic meningitis associated with mass vaccination with a urabe-containing measles-mumps-rubella vaccine: implications for immunization programs.;Autism and measles, mumps, and rubella vaccine: no epidemiological evidence for a causal association;A unified approach to the analysis of case-distribution (case-only) studies.;Does Influenza Vaccination Exacerbate Asthma?;Bidirectional case-crossover designs for exposures with time trends.;Case series analysis of adverse reactions to vaccines: a comparative evaluation.;A new method for active surveillance of adverse events from diphtheria/tetanus/pertussis and measles/mumps/rubella vaccines;Epidemiologic assessment of risks of adverse reactions associated with intermittent exposure.;Risk of aseptic meningitis after measles, mumps,and rubella vaccine in UK children;Design and analysis of drug safety studies, with special reference to sporadic drug use and acute adverse reactions.;The case-crossover design: a method for studying transient effects on the risk of acute events.;A case-cohort design for epidemiologic cohort studies and disease prevention trials;Copyright Ltd. Statist. Med;Does oral polio vaccine cause intussusception in infants? Evidence from a sequence of three self-controlled cases series studies in the united kingdom;Lack of association between intussusception and oral polio vaccine in Cuban children;GenStat release 7.1. VSN international Ltd;Stata Statistical Software: Release 8.0. College Station, TX: Stata Corporation;The guide to GenStat release;Wheezing lower respiratory disease and vaccination of full‐term infants;Stata Statistical Software: Release 7.;Should we use a case-crossover design?;Relative incidence estimation from case series for vaccine safety evaluation.;GLIM release 4. Royal Statistical Society;Interaction between life history events, Nonparametric analysis for prospective and retrospective data in the presence of censoring",
         "Tutorial in biostatistics: the self‐controlled case series method",
         "None"
        ],
        [
         "11",
         "00088986c30f3b32724562d84b4e9be87262bc21",
         "Semantic Web (SW), commonly regarded as the next generation of the Web, is an emerging vision of the new Web from the Knowledge Representation and the Web communities. To realize this vision, a series of techniques has been proposed. Semantic Web Ontology Langauge (OWL) and its extension Semantic Web rule Language (SWRL) and Semantic Web Logic Language (SWRL-FOL) are some of the most important outputs from the SW activities. However the existing reasoning and consistency checking tools for those languages are primitive. This paper proposes using the existing formal modelling tool, in particular Alloy, to provide an automatic reasoning service for the Semantic Web ontology family languages (OWL/SWRL/SWRL-FOL).",
         "Hai H. Wang,J. Dong,Jing Sun,Jun Sun",
         "Hai H. Wang,J. Dong,Jing Sun,Jun Sun",
         "http://www.comp.nus.edu.sg/~dongjs/papers/reasoning-alloy06.pdf",
         "Semantic Web (SW), commonly regarded as the next generation of the Web, is\nan emerging vision of the new Web from the Knowledge Representation and the\nWeb communities. To realize this vision, a series of techniques has been proposed.\nSemantic Web Ontology Langauge (OWL) and its extension Semantic Web rule\nLanguage (SWRL) and Semantic Web Logic Language (SWRL-FOL) are some of\nthe most important outputs from the SW activities. However the existing reason-\ning and consistency checking tools for those languages are primitive. This paper\nproposes using the existing formal modelling tool, in particular Alloy, to provide\nan automatic reasoning service for the Semantic Web ontology family languages\n(OWL/SWRL/SWRL-FOL). 1\nkeywords: Semantic Web, Alloy, OWL, SWRL, FOL.\nThe power of the Semantic Web [1], as the next generation of the Web, will be realized\nwhen software agents are able to understand the Web content, process the informa-\ntion and exchange the results with other software agents. Adding logic to the Web is\none of the key requirements. This logic must be powerful enough to describe com-\nplex properties of web resources but not so complicated that agents could be tricked by\nbeing asked to consider a paradox. To achieve these two contradictory requirements,\nresearchers attempt to adopt the layered approach, where the upper layer is extended\nfrom the lower layer with enhanced expressive power. This allows that different appli-\ncations can choose the logic language suiting their needs most.",
         "5.3 Discussion\nThe correctness of the translation has been veriﬁed by many different test cases. A\nsame problem has been sent to existing SW tools, theorem provers and Alloy; the same\nconclusions are drawn. Furthermore, the OWL has well deﬁned semantics in ﬁrst order\nlogic and Alloy is also based on the ﬁrst-order logic. The soundness of the translation\ncan also be proved easily. In the early work [13], it shows that the consistence between\nthe Alloy Semantic for the Semantic Web languages and the original OWL semantic\n(Alloy has been regarded as a subset of Z). Formal proving this consistence is beyond\nthe scope of this paper.\n6 Related works and conclusion\nThis paper presented a reasoning environment for the Semantic Web ontology family\nlanguages (OWL/SWRL/SWRL-FOL). There are four main contributions of the paper.\nFirstly, it deﬁnes a semantic encoding for the OWL/SWRL/SWRL-FOL constructs in\nthe Alloy ﬁrst-order language. Secondly, it presents a systematic transformation tool\nfrom the OWL/SWRL/SWRL-FOL ontology (in XML) into its corresponding Alloy\nmodel. Thirdly, with the assistance of Alloy Analyzer, it has been demonstrated that\nthe consistency of an ontology model can be checked automatically and different kinds\nof reasoning tasks can be supported. Our approach complements with existing OWL\nreasoners by providing full automatic debugging aids and instance level reasoning.\nFurthermore, SWRL-FOL is a newly proposed extension to OWL, and to our best of\nknowledge, so far there is no existing reasoning support for SWRL-FOL prior to this\nwork. Finally, the paper also demonstrates a light-weight formal methods approach to\nthe web ontology domain. Alloy was chosen over other reasoning tools because it is\nbased on ﬁrst-order relational logic and relations between Web resources are the focus\nissues in the Semantic Web context. Furthermore, Alloy has an impressive automatic\ntool support, the Alloy Analyzer, where automated generation of ﬁnite set of ontology\ninstances, creation of counter-examples on assertions, and identifying the source of\ninconsistencies in the model are made available. Usual ontology tools such as FACT\nand RASER can detect errors in an ontology model, but may not be able to point out\nwhere the error is. Alloy approach provides the ontology “surgery” like capability to\npin point the errors in the model with counter-examples or contradictory constraints.\nThis is a highly complementary approach to Semantic Web reasoning. The approach\nhas been successfully applied to a recent military ontology [5].\nIt has indeed been realized that there is a limitation on the scalability of the current\nAlloy Analyzer in reasoning large ontology models. The approach presented here can\nonly deal with the ontologies with relatively small size. Based on the same idea, authors\nalso attempt to use the theorem prover, i.e. Z/EVES, to reason the SW ontology [4].\nThe theorem prover can handle large sized ontologies, but it requires the user’s interac-\ntion. Here authors do not claim that Alloy is the only and best formal tool to reason over\nSW ontologies, but authors do claim that it is an effective attempt with certain novel\nand irreplaceable advantages like full automation and promising debugging assistance.\nIn fact, it is unlikely in the near future that both expressive and automatic tool will be\ndeveloped. Currently, it is desirable if the strength from different ontology reasoning\ntools can be integrated. [3] presented the methodology of checking ontologies using\ntools RACER, Z/EVES and AA in conjunction. This approach has been successfully\napplied for reasoning a real life military ontology.\nIn the future, it has been planned to integrate the current Alloy Analyzer reasoning\nfacilities into our OWL/SWRL/SWRL-FOL transformation tool by connecting it to the\nAlloy API interfaces. In addition, we also plan to extend the transformation tool with\nthe editing and designing functions for the ontology models, so that it will become an\nintegrated development environment for the web ontology modelling, which includes\ndesign, transformation and reasoning functions in one coherent tool support.",
         "1 Introduction\nThe bottom layer is Web Ontology Language (OWL) [14]. OWL, a recommen-\ndation by the World Wide Web Consortium (W3C), is the latest standard to deﬁne\nthe ontology. It is based on Description Logic (DL). Although OWL adds consider-\nable expressive power to the Semantic Web, to retain the decidability of key inference\nproblems in OWL DL and OWL Lite, OWL has its expressive limitations. Certain\ndesired properties can not be expressed for some applications. Semantic Web Rule\nLanguage (SWRL) [9] extends OWL by combining the OWL DL and OWL Lite with\nthe Unary/Binary Catalog sub-languages of the Rule Markup Language. It introduces\na new kind of axiom, named Horn clause rules, to OWL DL. Recently, Semantic Web\nRule Language First Order Logic (SWRL-FOL) [15] has been proposed to further ex-\ntend the SWRL to handle unary/binary ﬁrst-oder logic.\nReasoning can be useful at many stages during the design, maintenance and deploy-\nment of ontology. Using the reasoning service provided by Semantic Web reasoners,\nsoftware agents can autonomously infer new knowledges from the given knowledges\nand perform different tasks. For example, Pizzaﬁnder 2, as a small application devel-\noped by our research group, demonstrates how the reasoner could be used by a web\nsoftware agent for their application. Pizzaﬁnder uses the Pizza Ontology, and a rea-\nsoner to dynamically generate pizza toppings and pizza topping categories. The user\ncan include toppings that they would like on their pizza and exclude any toppings that\nthey do not want on their pizza. The description logic reasoner is used to determine\nif the choices that have been made contradict each other - for example, choosing to\ninclude Jalapeno Pepper topping, but choosing to exclude all hot toppings - the choices\nare automatically adjusted to modify any decisions that could potentially lead to con-\ntradictions and inconsistent results.\nBecause autonomous software agents may perform their reasoning and come to\nconclusions without human supervision, it is essential that the shared ontology is con-\nsistent. However, since the Semantic Web technology is still in the early stage, the\nreasoning and consistency checking tools are primitive. The existing OWL reasoning\ntools such as FaCT [8] and RACER [6] have been developed speciﬁcally for the de-\ncidable description logic, which are based on tableaux algorithm. They are far from\nperfect. Furthermore, currently there does not exist a tableaux algorithm that can sup-\nport the reasoning of SWRL-FOL, or even SWRL. Hence, it would take some effort\nand time for people to research into new algorithms and build new tools to support\nSWRL and SWRL-FOL reasoning. However, as it can been foreseen that it is critical\nand urgent to provide some reasoning service to SWRL and SWRL-FOL in order to\nmake them to be integrated into ontology languages hierarchy and to have their impacts\non the practical web applications.\nAlternatively, rather than developing new algorithms and tools, a light-weight ap-\nproach to provide reasoning service which can complement existing OWL reasoners\nand support SWRL and SWRL-FOL is to customize and reuse some existing tools.\nAfter decades of research and development, some mature formal modelling/reasoning\ntools have been established successfully. These tools could well be adopted to reason-\ning about OWL, SWRL and SWRL-FOL.\nThis paper proposes to develop a reasoning environment using the software mod-\nelling language Alloy and its Analyzer [11] for web ontology families language. It\ncomplements the existing OWL reasoning tools like RACER, and also supports the\nnewly extended SWRL and SWRL-FOL.\nThe rest of the paper is organized as follows. Section 2 brieﬂy introduces the OWL,\nSWRL, SWRL-FOL and Alloy. In section 3, we present the Alloy semantics for the\nOWL/SWRL/SWRL-FOL language and the transformation from the ontologies into\ntheir corresponding Alloy models. Section 4 presents a case study to demonstrate the\nreasoning processes of SWRL-FOL ontology models in the Alloy Analyzer. Section 5\nconcludes the paper and discusses the future work.\nThis paper is substantially extended and revised from the early conference paper\n‘Reasoning Support for SWRL-FOL Using Alloy’ [19].\n2.1 Semantic web overview\n2.1.1 Semantic web and OWL\nThe Semantic Web is a vision for a new kind of Web with enhanced functionality\nwhich will require semantic-based representation and processing of Web information.\nW3C has proposed a series of technologies that can be applied to achieve this vision.\nThe Semantic Web extends the current Web by giving the web content a well-deﬁned\nmeaning, better enabling computers and people to work in cooperation. XML is aimed\nat delivering data to systems that can understand and interpret the information. XML\nis focused on the syntax (deﬁned by the XML schema or DTD) of a document and it\nprovides essentially a mechanism to declare and use simple data structures. However\nthere is no way for a program to actually understand the knowledge contained in the\nXML documents.\nResource Description Framework (RDF) [12] is a foundation for processing meta-\ndata; it provides interoperability between applications that exchange machine-understandable\ninformation on the Web. RDF uses XML to exchange descriptions of Web resources\nand emphasizes facilities to enable automated processing. The RDF descriptions pro-\nvide a simple ontology system to support the exchange of knowledge and semantic\ninformation on the Web. RDF Schema [2] provides the basic vocabulary to describe\nRDF documents. RDF Schema can be used to deﬁne properties and types of the web\nOWL constructs\nOWL class\nOWL property\nOWL subclass[C ]\nOWL subproperty[P ]\ninstanceof [C ]\nDescription\nclasses\nproperties\nsubclasses of C\nsubproperties of P\ninstances of the OWL class C\nresources. The advent of RDF Schema represented an early attempt at an SW ontology\nlanguage based on RDF.\nOWL [14] is a standard (W3C recommendation) for expressing ontologies in the\nSemantic Web. The OWL language facilitates greater machine understandability of\nWeb resources than that supported by RDFS by providing additional constructors for\nbuilding class and property descriptions (vocabulary) and new axioms (constraints),\nalong with a formal semantics. OWL consists of three sub-languages: OWL Lite, OWL\nDL and OWL Full, with increasing expressiveness. OWL Lite and DL are decidable,\nbut OWL Full is generally not. An OWL ontology consists of classes, properties and\nindividuals. Classes are interpreted as sets of objects that represent the individuals in\nthe domain of discourse. Properties are binary relations that link individuals, and are\ninterpreted as sets of tuples, which are the subsets of the cross product of the objects in\nthe domain of discourse. OWL classes fall into two main categories – named classes\nand anonymous classes. Anonymous classes are formed from logical statements. They\ncontain the individuals that satisfy the logical description. Anonymous classes may be\nfurther sub-divided into restrictions and logical class expressions. We summarize some\nessential OWL constructs in Table 1. To be simpler, informal OWL syntax has been\nused here. For example, OWL class denotes the OWL construct “Class” .\n2.1.2 SWRL\nAlthough OWL includes a relatively rich set of class constructors, the language pro-\nvided for expressing properties is much weaker. SWRL [9] intends to overcome the\nexpressive restriction of OWL properties by extending OWL with some form of “rule\nlanguage”. SWRL is based on a combination of the OWL DL and OWL Lite sub-\nlanguages of the OWL Web Ontology Language with the Unary/Binary Catalog sub-\nlanguages of the Rule Markup Language. SWRL introduces a high-level abstract syn-\ntax for Horn-like rules in both the OWL DL and OWL Lite sub-languages of OWL.\nSWRL extends OWL by also allowing rule axioms, i.e., by adding the construct:\naxiom ::= rule\nA rule axiom consists of an antecedent and a consequent, each of which consists of\na set of atoms which could be class membership (C(x)), property membership (P(x,y))\nor individual in/equalities(differentFrom(x,y)/sameAs(x, y)). Informally, a rule means\nthat if the antecedent holds (is “true”), then the consequent must also hold. A simple\nexample of the rules could be used to express the knowledge that “if ?x 1 is a child of\n?x 2 and ?x 2 is a brother of ?x 3, then ?x 3 is an uncle of ?x 1”. Informally, this rule\ncould be written as:\nhasChild (?x 2, ?x 1) ∧ hasBrother (?x 2, ?x 3)\n⇒ hasUncle(?x 1, ?x 3)\n2.1.3 SWRL-FOL\nSWRL-FOL [15] extends SWRL axiom to arbitrary ﬁrst-order formula over unary and\nbinary predicates. It extends SWRL with “assertion” axioms that contain ﬁrst-order\nsentences , i.e.\naxiom ::= assertion\nAssertions assert ﬁrst-order sentences, where no free variables are allowed in the\nformulae. For example an axiom could be used to express the knowledge that “for all\nthe person ?x 1 if s/he is a ‘wealthy Parent’, then s/he has at least one child ?x 2 who is\na millionnaire.”. Informally, this axiom could be written as:\n∀?x 1 | wealthyParent (?x 1) ⇒\n∃?x 2 | hasChild (?x 1, ?x 2) ∧ millionnaire(?x 2)\nWe can see from the above example that by introducing ﬁrst-order formulas, more\ncomplex logical statements can be expressed in SWRL.\n2.1.4 Existing reasoning tools for Semantic Web\nOntology reasoning tools have been built alongside the development of ontology lan-\nguages. The rest of this subsection will introduce a few of these tools.\nCwm (Closed world machine) [18] is a general-purpose data processor for the Se-\nmantic Web. Implemented in Python and command-line based, it is a forward chaining\nreasoner for RDF.\nTriple [17] is a RDF query, inference and transformation language. It does not\nhave a built-in semantics for RDF Schema, but it allows semantics of languages to be\ndeﬁned with rules on top of RDF. This feature of Triple facilitates data aggregation as\nusers can perform RDF reasoning and transformation under different semantics. The\nTriple tool supports OWL through external OWL reasoners such as FaCT and RACER.\nFaCT (Fast Classiﬁcation of Terminologies) [8], developed at University of Manch-\nester, is a TBox (concept-level) reasoner that supports automated concept-level reason-\ning, namely class subsumption and consistency reasoning. It does not support ABox\n(instance-level) reasoning. It is implemented in Common Lisp and comes with a FaCT\nserver, which can be accessed across network via its CORBA interface. Given an OWL\nontology, it can classify the ontology (performs subsumption reasoning) so as to reduce\nredundancy and detect any inconsistency within it.\nRACER, the Renamed ABox and Concept Expression Reasoner [6], implements\na TBox and partial ABox reasoner for the description logic ALCQHIR+ (D)− [7]. It\ncan be regarded as (a) a Semantic Web inference engine, (b) a description logic reason-\ning system capable of both TBox and ABox reasoning and (c) a prover for modal logic\nKm. In the Semantic Web domain, RACER’s functionalities include developing on-\ntologies (creating, maintaining and deleting concepts, roles and individuals); querying,\nretrieving and evaluating the knowledge base, etc. It supports OWL and RDF.\nThe FaCT and RACER are the most well accepted OWL reasoners. However,\nthey still have many limitations, such as both of them can only ﬂag an OWL class\nis inconsistent without providing any explanation. The debugging task is left to the\nuser. Furthermore, there is very limited datatype support, such as integer and string.\nAlso FaCT does not provide any ABox reasoning. RACER can only partially support\nABox reasoning. Alloy approach proposed in this paper can complement FaCT and\nRACER [3].\nCurrently, there is not a well accepted system supporting SWRL, and only few\nprototypes has been developed. There is no reasoning tool supporting SWRL-FOL yet.\n2.2 Alloy overview\nAlloy [11] is a structural modelling language based on ﬁrst-order logic, for express-\ning complex structural and behavioral constraints. Alloy treats relations as ﬁrst class\ncitizens and uses relational composition as a powerful operator to combine various\nstructured entities. The essential constructs of Alloy are as follows:\n• Signature: A signature (sig) paragraph introduces a basic type, a collection of\nrelations (called ﬁeld), and a set of constraints on their values. A signature may\ninherit ﬁelds and constraints from another signature.\n• Function: A function (fun) captures behavior constraints. It is a parameterized\nformula that can be “applied” elsewhere.\n• Fact: Fact (fact ) imposes global constraints on the relations and objects. A fact\nis a formula that takes no arguments and needs not to be invoked explicitly.\n• Assertion: An assertion (assert ) speciﬁes an intended property. It is a formula\nwhose correctness needs to be checked, assuming the facts in the model.\nThe Alloy Analyzer is a tool for analyzing models written in Alloy. Given a ﬁ-\nnite scope for a speciﬁcation, Alloy Analyzer translates it into a propositional formula\nand uses SAT solving technology to generate instances that can satisfy the facts and\nproperties expressed in the speciﬁcation.\n3 Alloy semantics for OWL/SWRL/SWRL-FOL\nThis section presents the Alloy semantics for OWL, SWRL and SWRL-FOL lan-\nguages, which forms the foundation of the reasoning environment. Due to limited\nspace, only part of semantic model has been presented here. A complete Alloy seman-\ntics for these languages can be found at http://www.cs.man.ac.uk/∼hwang/\nswrlfol.als.\n3.1 Alloy semantic for OWL constructs\n3.1.1 Basic concepts\nThe semantic model for OWL is encoded in the module OWL. Users only need to\nimport this module to reason about OWL ontology in Alloy.\nmodule OWL\nAll the things described in the Semantic web context are referred to as web re-\nsources. A basic type Resource is deﬁned as:\nsig Resource {}\nOther concepts such as classes and properties deﬁned later are extended from the\nResource. Property, which is a kind of Resource itself, relates Resource to Re-\nsource.\ndisj sig Property extends Resource\n{sub_val: Resource -> Resource}\n“disj” is a keyward from Alloy for denoting the disjointness. Each Property has a\nrelation sub val from set <Property, Resource, Resource> with type <Resource,\nResource, Resource> (since in Alloy subsignature does not introduce a new type).\nThis relation can be regarded as a RDF statement, i.e., a triple of the form <property(or\npredicate), subject, value(or object)>.\nThe class corresponds to the generic concept of type or category of resource. Each\nClass maps a set of resources via the relation instances, which contains all the in-\nstance resources. The keyword disj is used to indicate the Class and Property are\ndisjoint.\ndisj sig Class extends Resource {instances: set Resource}\nThe OWL also allows the use of XML Schema datatypes to describe (or deﬁne)\npart of the datatype domain. Alloy supports Integer and String. Apart from these there\nare no predeﬁned types in Alloy, Datatype has been treated as a special Class, which\ncontains all the possible datatype values in the instances relation.\ndisj sig Datatype extends Class {}\n3.1.2 Class elements\nThe subClassOf is a relation between classes. The instances in a subclass are also in\nthe super-classes. A parameterized formula (a function in Alloy) is used to represent\nthis concept.\nfun subClassOf(csup, csub: Class)\n{csub.instances in csup.instances}\nThe disjointWith is a relation between classes. It asserts that there are no instances\ncommon with each other.\nfun disjointWith (c1, c2: Class) {no c1.instances & c2.instances}\n3.1.3 Property restrictions\nThe allValuesFrom construct states that all instances of the class c1 that have the\nvalues of property P all belong to the class c2.\n(p: Property, c1: Class, c2: Class)\n{all r1, r2: Resource |\nr1 in c1.instances =>\nr2 in r1.(p.sub_val) =>\nr2 in c2.instances}\nA hasValue function states that all instances of the class c1 have the values of\nproperty P as resource r. The r could be an individual object or a datatype value.\nfun hasValue (p: Property, c1: Class, r: Resource)\n{all r1: Resource | r1 in c1.instances => r1.(p.sub_val) = r}\nA MaxCardinality function states that all instances of the class c1 have at most N\ndistinct values for the property P. Alloy supports some integer operations.\nfun maxCardinality (p: Property, c1: Class, N: Int)\n{all r1: Resource| r1 in c1.instances <=>\n# r1.(p.sub_val) <= int N }\n3.2 Boolean combination of class expressions\nThe intersectionOf function deﬁnes a relation between a class c1 and a list of classes\nclist. The List is deﬁned in the Alloy library. The class c1 consists of exactly all the\nobjects that are common to all class expressions from the list clist.\nfun intersectionOf (clist: List, c1: Class)\n{all r: Resource| r in c1.instances <=>\nall ca: clist.*next.val | r in ca.instances}\nThe unionOf function deﬁnes a relation between a class c1 and a list of classes\nclist. The class c1 consists of exactly all the objects that belong to at least one of the\nclass expressions from the list clist. It is analogous to logical disjunction;\nfun unionOf (clist: List, c1: Class)\n{all r: Resource| r in c1.instances <=>\nsome ca: clist.*next.val| r in ca.instances}\n3.2.1 Property elements\nThe subPropertyOf construct states that psub is a sub-property of the property psup.\nThis means that every pair (subject,value) that is in psup is also in the psub.\nfun subPropertyOf (psup, psub: Property)\n{psub.sub_val in psup.sub_val}\nThe domain function asserts that the property P only applies to instances of the\nfun domain (p: Property, c: Class)\n{(p.sub_val).Resource in c.instances}\nThe inverseOf function shows two properties are inverse.\nfun inverseOf (p1, p2: Property) {p1.sub_val = ˜(p2.sub_val)}\nAll other OWL constructs can be deﬁned in a similar manner. Please refer to the\ncomplete OWL Alloy semantics online.\n3.3 Alloy semantic for SWRL extension\nSWRL extends OWL by adding the rule axioms. A rule axiom consists of an antecedent\nand a consequent, each of which consists of a set of atoms. Atoms can be of the\nfollowing forms, where C is an OWL description, P is an OWL property, and x,y are\neither variables, OWL individuals or OWL data values.\n• C(x): Informally, it holds if x is an instance of the class description C.\n• P(x,y): It holds if x is related to y by property P.\n• sameAs(x,y): It holds if x is interpreted as the same object as y.\n• differentFrom(x,y): It holds if x and y are interpreted as different objects.\nAtom\nC (x )\nP (x , y)\nsameAs(x , y)\ndiﬀerentFrom(x , y)\nAlloy representation\n‘x in C.instances’\n‘(x->y) in P.sub val’\n‘x = y’\n‘x != y’\nAs mentioned before, a rule means that if the antecedent holds, the consequent must\nalso hold. It can be modelled as a universally quantiﬁed fact in the form of implication.\nFor example the following rule axiom (where a0 ... an are atoms)\nImplies(Antecedent (a1, ..., an ) Consequent (a0))\nwill be modelled as:\nfact { a_1 && ... && a_n => a_0 }\nSWRL-FOL formula Alloy semantics\nand (C1...Cn )\nor (C1...Cn )\nneg(C )\nimplies(C1 C2)\nequivalent(C1 C2)\nforall (V1...Vn C )\nexists(V1...Vn C )\nfact {C1 && ... && Cn }\nfact {C1 || ... || Cn }\nfact {not C}\nfact {C1 => C2}\nfact {C1 <=> C2}\nfact {all V1, ..., Vn : Resource | C}\nfact {some V1, ..., Vn : Resource | C}\n3.4 Alloy semantic for SWRL-FOL extension\nSWRL-FOL extends SWRL with assertion axioms that contain ﬁrst-order formulas.\nTable 3 presents the Alloy semantic for different SWRL-FOL formulas.\nThe above deﬁnes the basic transformation guidelines from the SWRL-FOL into\ntheir corresponding Alloy semantics. We will demonstrate the actual transformation\nprocess in the following section.\n4 OWL/SWRL/SWRL-FOL to Alloy transformation\nThe previous section presented Alloy semantics for OWL, SWRL and SWRL-FOL,\nwhich forms the foundation for the reasoning environment. To be able to perform the\nautomatic reasoning task using Alloy Analyzer, a Java program has been developed\nfor the automatic transformation from an OWL/SWRL/SWRL-FOL knowledge ﬁle (in\nXML format) into its corresponding Alloy model.\nA set of translation rules are developed in the following presentation.\n4.1 OWL class translation\nC ∈ OWL class\nstatic disj sig C extends Class{}\nAn OWL class C will be transferred into a scalar C, constrained to be an element of\nthe signature Class.\n4.2 OWL property translation\nP ∈ OWL property\nstatic disj sig P extends Property{}\nAn OWL property p will be translated into a scalar P, constrained to be an element of\nthe signature Property.\nx ∈ instancesof [Y ]\nstatic disj sig x extends Resource{}\nfact { x in Y .instances}\nAn OWL individual x of class Y will be translated into a scalar x, constrained to be an\nelement of the signature Resource. x is a subset of Y.instances.\n4.4 Other OWL translations\nOther OWL constructs can be easily translated into the Alloy functions deﬁned in the\nprevious section. For example the following rule shows how to translate the OWL\nsubclass relation into Alloy code.\nsubclass[X , Y ], X ∈ OWL class, Y ∈ OWL class\nfact {subClassOf (X , Y )}\n4.5 SWRL and SWRL-FOL translation\nThe transformation of SWRL rules follows the semantics deﬁned in Table 2. The\nvariable x and y will be bound by some universal quantiﬁers. The SWRL rule can be\nmodelled as a universally quantiﬁed fact in the form of implication.\nSimilarly, the transformation of SWRL-FOL follows the semantics presented in\nTable 3. More translation rules can be found from [19] and the web site http:\n//nt-appn.comp.nus.edu.sg/fm/alloy/introduction.htm.\n4.6 Translation example\nThe translation rules have been implemented in a Java program. The following OWL\nontology deﬁnes two classes animal and plant which are disjoint. The eats and\neaten by are two properties, which are inverse to each other. The domain of eats\nis animal. The carnivore is a subclass of animal which can only eat animals. The\nontology is given in a syntax similar to the “DL syntax” given in [10].\nClass (animal)\nClass (plant)\nDisjointClasses( animal plant )\nObjectProperty(eaten by)\nObjectProperty(eats\ndomain (animal))\nInverseProperties(eats eaten by)\nClass (carnivore complete animal\nrestriction(eat allValuesFrom animal))\nClass (herbivore complete animal\nrestriction(eat allValuesFrom plant))\nThis fragment ontology can be transformed by the tool into the following Alloy\nmodule animal\n/*import the defined library module */\nopen SWRL-FOL\n/* plant and animal are translated to two class instances. The key\nword static is used to a signature containing exactly one element.*/\nstatic disj sig plant, animal extends Class {}\n/* The disjoin element was translated into fact in Alloy */\nfact {disjointWith(plant, animal)}\n/* eats, eaten_by are translated to two property instances */\nstatic disj sig eats, eaten_by extends Property {}\nfact {inverseOf(eats, eaten_by)}\nfact {domain(eats, animal)}\nstatic disj sig carnivore extends Class{}\nfact{subClass(animal, carnivore)}\nfact{allValuesFrom(eats, carnivore, animal)}\nstatic disj sig herbivore extends Class{}\nfact{subClass(animal, herbivore)}\nfact{allValuesFrom(eats, herbivore, plant)}\nThe transformation of SWRL rules follows the semantics deﬁned in Table 2. The\nvariable x and y will be bound by some universal quantiﬁers. The SWRL rule can be\nmodelled as a universally quantiﬁed fact in the form of implication. For example the\nfollowing rule axiom\nhasParent (?x 1, ?x 2) ∧ hasBrother (?x 2, ?x 3) ⇒\nhasUncle(?x 1, ?x 3)\nwill be modelled as:\nfact {all x1, x2, x3: Resource |\n(x1->x2) in hasParent.sub_val &&\n(x2->x3) in hasBrother.sub_val =>\n(x1->x3) in hasUncle.sub_val}\nThe transformation of SWRL-FOL follows the semantic presented in Table 3. After\ntransforming the ontologties to the Alloy model, the consistency of the OWL/SWRL/SWRL-\nFOL ontology can be checked and some reasoning can be done readily.\n5 Reasoning OWL/SWRL/SWRL-FOL ontology with\nAlloy Analyzer\nReasoning is one of the key tasks for Semantic Web applications. It can be useful\nat many stages during the design, veriﬁcation, maintenance and deployment of web\nontology. In this section, we show that different Semantic Web reasoning tasks can be\naccomplished by using the Alloy Analyzer.\n5.1 Standard OWL reasoning tasks\nThere are two different levels of checking and reasoning in OWL, the conceptual level\nand the instance level. At the conceptual level, the class properties and subclass rela-\ntionships can be reasoned. At the instance level, the membership checking (instantia-\ntion) and instance property reasoning can be done.\n5.1.1 Class property checking\nIt is essential that the ontology shared between autonomous software agents is con-\nceptually consistent. Reasoning with inconsistent ontologies may lead to erroneous\nconclusions. An OWL class is deemed to be unsatisﬁable (inconsistent) if, because of\nits description, it cannot possibly have any instances. This section gives some exam-\nples of inconsistent ontology that can arise in ontology development, and demonstrate\nhow these inconsistencies can be detected by the Alloy Analyzer. For example, another\nclass tastyPlant which is a subclass of plant and eaten by the carnivore is deﬁned.\nThere is an inconsistency since by the ontology deﬁnition carnivores can only eat ani-\nmals. Animals and plants are disjoint.\nClass (tastyPlant partial plant\nrestriction(eat by allValuesFrom(carnivore)))\nWe translate the ontology into an Alloy program, add some facts to remove the triv-\nial models (like every type is empty set) and load the program into the Alloy Analyzer.\nThe Alloy Analyzer will automatically check the consistency. AA attempts to ﬁnd a\nmodel – a binding of the variables to values – that makes for the formulas (the formulas\ntranslated from the OWL model) true. If no such model can be build, it means that the\nmodel has been over constrained, i.e, there are some contradiction (inconsistency) in\nthe model.\nIn the example, it can be concluded that there is an inconsistency in the animal\nontology since Alloy can not ﬁnd any solutions satisfying all facts within the scope\n(Figure 1). Note that when Alloy can not ﬁnd a solution, it may be due to the scope\nbeing too small. By picking a large enough scope, “no solution found’ is very likely\nto mean that an inconsistency has occurred. AA tried to constructs a model which\nsatisﬁed all asserted axioms. If no such a model could be build (“no solution found”),\nthen there are some contradicted axioms in the model.\nBesides discovering the existence of an inconsistency in ontology, tracing where\nthe inconsistency arises from is also crucial for a reasoning tool to be practical. The\nexisting OWL reasoners like FaCT and RACER can only ﬂag the inconsistent class\nwithout providing any explanation. The debugging process is left to users. Without any\ntool support, identifying the conﬂicting knowledge could be frustrating. One possible\nsystematic technique for ﬁnding the causes of inconsistent ontology is to manually\nremove individual knowledge information until the culprit is identiﬁed. This task can\nbe lengthy and dangerous.\nIn Alloy, the “unsatisﬁed core” [16] functionality of recent SAT solvers was utilized\nand it supports core extraction, a new analysis technique that helps to discover over-\nconstraint in declarative models. This functionality can provide some assistance for the\nuser to trace the inconsistency.\nExtracting the unsatisﬁable core of a CNF formula, that is a subset of the clause\nset sufﬁcient to cause a contradiction, has been developed recently by satisﬁability\nsolvers [16]. In the latest version of Alloy, the declarative model analysis has been\ncast as satisﬁability instances and the unsatisﬁable core has been mapped back onto the\nmodel. In other words, a user can identify the parts of model responsible for producing\nthe unsatisﬁable CNF core. Those parts, by themselves, sufﬁce to produce an over-\nconstraint, and their identiﬁcation can help the user ﬁnd the over-constraint. Using this\nfunctionality, the portions of the ontology which contradict each other can be traced\nreadily. In the animal example, suppose a new class named funnything was deﬁned to\nbe a subclass of both animal and plant classes. It is easy to see that there is an incon-\nsistency since the class animal and plant are disjoint. Alloy can automatically identify\na set of knowledge which makes the ontology unsatisﬁable (Figure 2). The unsatisﬁ-\nability maybe due to the fact that funnything is a subclass of animal, funnything is a\nsubclass of plant or animal and plant are disjoint classes, and so on.\n5.1.2 Subsumption reasoning\nThe task of subsumption reasoning is to infer an OWL class is the subclass of another\nOWL class. That is for every instances of one OWL class, it is an instance of another\nOWL class as well. Using AA, the subsumption relationship between classes can be\nchecked automatically. The relationship between the ﬁsh, shark and dolphin has been\nused as an example to demonstrate this kind of reasoning task. In the animal ontol-\nogy a property breathe by is deﬁned. The ﬁsh is a subclass of the animal which\nbreathe by the gill.\nObjectProperty(breathe by)\nClass (gill)\nClass (ﬁsh complete animal\nrestriction(breathe by allValuesFrom gill))\nSince the purpose of this paper is to demonstrate ideas, the ontology has been kept\nsimple. In reality there are some animals such as frogs and toads, which can respire\nby use of gills when they are young and by lungs when they reach adult stage. Also\ncases like that the animals which respire by use of the pharyngeal lining or skin, like\nnewborn Julia Creek dunnarts have not been considered. A class shark, a subclass of\ncarnivore which breathe by the gill, has also been deﬁned.\nClass (shark)\nClass (ﬁsh complete animal\nrestriction(breathe by allValuesFrom gill))\nSeveral of the classes were upgraded to be deﬁned when their deﬁnitions consti-\ntuted both necessary and sufﬁcient conditions for class membership, e.g., an animal is\na ﬁsh if and only if it breathes by the gill. Additional subclass relationships can be in-\nferred, i.e., the shark is also a subclass of ﬁsh. We transfer this ontology into an Alloy\nprogram and make an assertion that the shark is a subclass of ﬁsh. The Alloy ana-\nlyzer will check the correctness of this assertion automatically (Figure 3). The Alloy\nAnalyzer checks whether an assertion holds by trying to ﬁnd a counterexample. Note\nthat “no solution” means no counterexample found, in this case, it strongly suggests\nthat the assertion is sound. To make it more interesting, classes dolphin and lung are\ndeﬁned. Dolphins are a kind of animal which breathe by lungs. The classes gill and\nlung are disjoint.\nClass (lung)\nDisjointClasses( lung gill )\nClass (dolphin complete animal\nrestriction(breathe by allValuesFrom lung))\nSuppose an assertion that the dolphin is a kind of ﬁsh is made, the Alloy Analyzer\nwill refute it since some counterexample was found (Figure 4). If the fact that dolphin\nis a ﬁsh is added in the module, the AA will conclude that an inconsistency has arisen.\n5.1.3 Debugging uncompleted ontology\nInformation in OWL is gathered into ontologies, which can then be from different\nparties and stored as documents in the World Wide Web. Some knowledge may be\nmissing in the ontology. Reasoning about uncompleted ontologies may lead to some\nunexpected results. We refer to the situation that because of some unavailable knowl-\nedges, the reasoners had inferred some unexpected knowledges which is different with\nnatural facts. We need some tools to help the users to trace what is the missed knowl-\nedge causing the untrue conclusion that has been drew. AA checks the assertion by\ngenerating counterexamples – structures or behaviors for which an expected property\ndoes not hold; from a counterexample, it is usually not too hard to ﬁgure out what is\nwrong. Looking at the counterexamples may provide some hints to the user on why the\nexpected result does not hold and what knowledge is missing. For example, to show\nthe OWL class dolphin and shark are disjoint, Intuitively, this is a correct statement\nsince dolphin breathes by the gill while shark breathes by the lung. Gill and lung are\ndisjoint. When the following assertion is added to Alloy, surprisingly AA concludes it\nis wrong.\n{disjointWith(shark, dolphin)}\nBy looking at the counterexamples graph, it has been noticed that all the counterex-\namples (an animal which is both a shark and a dolphin) generated by AA have empty\nvalues for the property breath by. In fact this unexpected result comes from the se-\nmantic of allValuesFrom construct in OWL. An OWL semantic can not deduce from\na allValuesFrom restriction alone that there actually is at least one value for the prop-\nerty. An allValuesFrom restriction for a property is trivially satisﬁed for an instance\nthat has no value for that property at all. The allValuesFrom restriction demands that\nall values of the property belong to a class, and if no such values exist, the restriction\nis trivially true. That is the reason why AA ﬁnds out the common instance, which does\nnot breathe at all, for the class dolphin and class shark. To remove this expected result,\nextra knowledge needs to be added, e.g., an animal must breathe by something.\n5.1.4 Instantiation\nInstantiation is one of the main contributions for reasoning over OWL ontology using\nAlloy. Currently some successful OWL reasoners like FaCT are designed for descrip-\ntion logic (DL) T-box reasoning, which lacks support for instances. In Alloy every\nexpression denotes relations. The scalars will be represented by singleton unary rela-\ntions - that is, relations with one column and one row. The instance level reasoning can\nbe supported readily in Alloy.\nInstantiation is a reasoning task which tries to check if an individual is an instance\nof a class. For example, two resources aFeralAnimal and aMeekAminal are deﬁned\nas the instances of class animal. aGill is an instance of class gill. aFeralAnimal eats\naMeekAnimal and breathes by aGill. People may want to check if aFeralAnimal is a\ncarnivore and a ﬁsh.\nIndividual(aMeekAnimal type(animal))\nIndividual(aGill type(gill))\nIndividual(aFeralAnimal type(animal) value(breathe by aGill)\nvalue(eats aMeekAnimal))\nWe translate the ontology into an Alloy program and make an assertion as follow-\nstatic disj sig aFeralAnimal, aMeekAnimal extends Resource{}\nstatic disj sig aGill extends Resource{}\nfact {aFeralAnimal in animal.instances &&\nfact {aGill in gill.instances}\nfact {(aFeralAnimal->aMeekAnimal) in eats.sub_val}\nfact {(aFeralAnimal->aGill) in breathe_by.sub_val}\nassert isFishCarnivore\n&& (aFeralAnimal in carnivore.instances)}\ncheck isFishCarnivore for 15\nAA concludes that this assertion is correct.\n5.1.5 Instance property reasoning\nInstance property reasoning (often regarded as knowledge querying) is important in\nSemantic Web applications.\nIt is a task to query some properties with individuals.\nSince one of the promising strengths of Semantic Web technology is that it gives the\nagents the capability to do more accurate and more meaningful searches. The agent can\nanswer some questions for which the answers are not explicitly stored in the knowledge\nbase.\nFor example, the emerge early and emerge later are two properties, which are\ninverse to each other. Animal A emerges earlier than B if the species of A emerge earlier\nthan the species of B on the earth. emerge early is transitive. Three animal instances\nﬁrstDinosaur, ﬁrstApe and ﬁrstHuman are deﬁned. ﬁrstDinosaur emerge early\nthan ﬁrstApe and ﬁrstApe emerge early than ﬁrstHuman. One possible question\npeople may ask is whether ﬁrstHuman is emerge later than ﬁrstDinosaur. With the\nassistance of Alloy reasoner, such questions can be answered.\nfact{TransitiveProperty(emerge_early)}\nstatic disj sig firstDinosaur, firstApe,\nfact { firstDinosaur in animal.instances\n&& firstApe in animal.instances\n&& firstHuman in animal.instances}\nfact {(firstDinosaur->firstApe) in emerge_early.sub_val}\nfact {(firstApe->firstHuman) in emerge_early.sub_val}\nassert hum {(firstHuman->firstDinosaur) in emerge_later.sub_val}\ncheck hum for 14\nAA concludes that this assertion is correct.\n5.2 SWRL/SWRL-FOL related reasoning\nBesides of being capable to support the standard reasoning tasks on OWL, such as per-\nforming consistency checking, subsumption and instantiation reasoning automatically,\nmoreover, Alloy can also check more complicated ontology properties expressed by the\nnewly extended languages such as SWRL/SWRL-FOL. In this section, we demonstrate\nhow Alloy can be used to reasoning the SWRL-FOL ontologies.\nA family relationship web ontology example is used here to illustrate the rea-\nsoning process. The following fragment of ontology ﬁrst deﬁnes two OWL classes,\nPerson and twinParent that represents the set of person who are the parents\nof twins, and three OWL object properties, i.e., hasChild, brotherSister and\nsameBirthTime. Secondly, the ontology class wealthyParent introduces the\nset of parents who have a child who is a millionnaire. Thirdly, two SWRL-FOL\naxiomatic assertions are deﬁned to provide inference for the brotherSister and\ntwinParent relationships. Lastly, the ontology class wealthyTwinParent is\ndeﬁned as a parent being both wealthyParent and twinParent.\nClass (Person partial)\nClass (twinParent partial Person)\nClass (millionnaire partial Person)\nObjectProperty(hasChild)\nObjectProperty(brotherSister)\nObjectProperty(sameBirthTime)\nClass (wealthyParent complete Person\nrestriction(hasChild someValuesFrom(millionnaire)))\nAssertion(forall I-variable(x1) I-variable(x2)\n(equivalent (exists (I-variable(x3)\n(and(hasChild(x3,x1) hasChild(x3,x2)\ndifferentFrom(x1,x2))))\n(brotherSister(x1, x2)))))\n(equivalent (exists (I-variable(x2)\n(exists (I-variable(x3)\n(and(brotherSister(x2, x3) sameBirthTime(x2, x3)\nhasChild(x1, x2))))))\n(twinParent(x1)))))\nClass (wealthyTwinParent complete wealthyParent twinParent)\nFrom the above, it is noticed that two SWRL-FOL axioms were asserted. The ﬁrst\nassertion shows that if two distinct people have a same parent, then they are brothers\nor sisters. The second assertion in the above ontology shows that if two people are\nbrothers or sisters, and they have the same birth time, then their parents are twin-\nparents. Furthermore, suppose some instances of the above ontology are asserted into\nthe knowledge base as follows.\nIndividual(Tom type(person)\ntype(complementOf(wealthyTwinParent))\nvalue(hasChild Jerry)\nvalue(hasChild Jim))\nIndividual(Jerry type(millionnaire) value(sameBirthTime Jim))\nIndividual(Jim type(person))\nDisjointWith(Jim Jerry)\nWe transform the above ontology (in XML format) into its Alloy model3 using our\nSimilar as reasoning OWL, the Alloy Analyzer can automatically perform different\nreasoning tasks for SWRL/SWRL-FOL. For example, it can detect that there is an\ninconsistency in the above ontology example, as the Alloy Analyzer can not ﬁnd any\nontology instances (solutions) satisfying all facts within the scope.\nIn this family ontology example, the inconsistency comes from the fact that Tom\nhas been inferred as an instance of both the class wealthyParent and the class\ntwinParent. However, there is a piece of knowledge in the model that explicitly\nindicates that Tom is not an instance of the wealthyTwinParent class, which con-\ntradicts to the inferred conclusion. As discussed before, with the assistance of Alloy\nAnalyzer’s “unsatisﬁable core” functionality, the debugging process of identifying the\nsource of inconsistency in the ontology becomes much more handy to the users.",
         "",
         "",
         "https://academic.oup.com/chemse/article-pdf/31/3/213/979519/bjj021.pdf",
         "Semantic Web (SW), commonly regarded as the next generation of the Web, is\nan emerging vision of the new Web from the Knowledge Representation and the\nWeb communities. To realize this vision, a series of techniques has been proposed.\nSemantic Web Ontology Langauge (OWL) and its extension Semantic Web rule\nLanguage (SWRL) and Semantic Web Logic Language (SWRL-FOL) are some of\nthe most important outputs from the SW activities. However the existing reason-\ning and consistency checking tools for those languages are primitive. This paper\nproposes using the existing formal modelling tool, in particular Alloy, to provide\nan automatic reasoning service for the Semantic Web ontology family languages\n(OWL/SWRL/SWRL-FOL). 1\nkeywords: Semantic Web, Alloy, OWL, SWRL, FOL.\n1 Introduction\nThe power of the Semantic Web [1], as the next generation of the Web, will be realized\nwhen software agents are able to understand the Web content, process the informa-\ntion and exchange the results with other software agents. Adding logic to the Web is\none of the key requirements. This logic must be powerful enough to describe com-\nplex properties of web resources but not so complicated that agents could be tricked by\nbeing asked to consider a paradox. To achieve these two contradictory requirements,\nresearchers attempt to adopt the layered approach, where the upper layer is extended\nfrom the lower layer with enhanced expressive power. This allows that different appli-\ncations can choose the logic language suiting their needs most.\nThe bottom layer is Web Ontology Language (OWL) [14]. OWL, a recommen-\ndation by the World Wide Web Consortium (W3C), is the latest standard to deﬁne\nthe ontology. It is based on Description Logic (DL). Although OWL adds consider-\nable expressive power to the Semantic Web, to retain the decidability of key inference\nproblems in OWL DL and OWL Lite, OWL has its expressive limitations. Certain\ndesired properties can not be expressed for some applications. Semantic Web Rule\nLanguage (SWRL) [9] extends OWL by combining the OWL DL and OWL Lite with\nthe Unary/Binary Catalog sub-languages of the Rule Markup Language. It introduces\na new kind of axiom, named Horn clause rules, to OWL DL. Recently, Semantic Web\nRule Language First Order Logic (SWRL-FOL) [15] has been proposed to further ex-\ntend the SWRL to handle unary/binary ﬁrst-oder logic.\nReasoning can be useful at many stages during the design, maintenance and deploy-\nment of ontology. Using the reasoning service provided by Semantic Web reasoners,\nsoftware agents can autonomously infer new knowledges from the given knowledges\nand perform different tasks. For example, Pizzaﬁnder 2, as a small application devel-\noped by our research group, demonstrates how the reasoner could be used by a web\nsoftware agent for their application. Pizzaﬁnder uses the Pizza Ontology, and a rea-\nsoner to dynamically generate pizza toppings and pizza topping categories. The user\ncan include toppings that they would like on their pizza and exclude any toppings that\nthey do not want on their pizza. The description logic reasoner is used to determine\nif the choices that have been made contradict each other - for example, choosing to\ninclude Jalapeno Pepper topping, but choosing to exclude all hot toppings - the choices\nare automatically adjusted to modify any decisions that could potentially lead to con-\ntradictions and inconsistent results.\nBecause autonomous software agents may perform their reasoning and come to\nconclusions without human supervision, it is essential that the shared ontology is con-\nsistent. However, since the Semantic Web technology is still in the early stage, the\nreasoning and consistency checking tools are primitive. The existing OWL reasoning\ntools such as FaCT [8] and RACER [6] have been developed speciﬁcally for the de-\ncidable description logic, which are based on tableaux algorithm. They are far from\nperfect. Furthermore, currently there does not exist a tableaux algorithm that can sup-\nport the reasoning of SWRL-FOL, or even SWRL. Hence, it would take some effort\nand time for people to research into new algorithms and build new tools to support\nSWRL and SWRL-FOL reasoning. However, as it can been foreseen that it is critical\nand urgent to provide some reasoning service to SWRL and SWRL-FOL in order to\nmake them to be integrated into ontology languages hierarchy and to have their impacts\non the practical web applications.\nAlternatively, rather than developing new algorithms and tools, a light-weight ap-\nproach to provide reasoning service which can complement existing OWL reasoners\nand support SWRL and SWRL-FOL is to customize and reuse some existing tools.\nAfter decades of research and development, some mature formal modelling/reasoning\ntools have been established successfully. These tools could well be adopted to reason-\ning about OWL, SWRL and SWRL-FOL.\nThis paper proposes to develop a reasoning environment using the software mod-\nelling language Alloy and its Analyzer [11] for web ontology families language. It\ncomplements the existing OWL reasoning tools like RACER, and also supports the\nnewly extended SWRL and SWRL-FOL.\nThe rest of the paper is organized as follows. Section 2 brieﬂy introduces the OWL,\nSWRL, SWRL-FOL and Alloy. In section 3, we present the Alloy semantics for the\nOWL/SWRL/SWRL-FOL language and the transformation from the ontologies into\ntheir corresponding Alloy models. Section 4 presents a case study to demonstrate the\nreasoning processes of SWRL-FOL ontology models in the Alloy Analyzer. Section 5\nconcludes the paper and discusses the future work.\nThis paper is substantially extended and revised from the early conference paper\n‘Reasoning Support for SWRL-FOL Using Alloy’ [19].\n2.1 Semantic web overview\n2.1.1 Semantic web and OWL\nThe Semantic Web is a vision for a new kind of Web with enhanced functionality\nwhich will require semantic-based representation and processing of Web information.\nW3C has proposed a series of technologies that can be applied to achieve this vision.\nThe Semantic Web extends the current Web by giving the web content a well-deﬁned\nmeaning, better enabling computers and people to work in cooperation. XML is aimed\nat delivering data to systems that can understand and interpret the information. XML\nis focused on the syntax (deﬁned by the XML schema or DTD) of a document and it\nprovides essentially a mechanism to declare and use simple data structures. However\nthere is no way for a program to actually understand the knowledge contained in the\nXML documents.\nResource Description Framework (RDF) [12] is a foundation for processing meta-\ndata; it provides interoperability between applications that exchange machine-understandable\ninformation on the Web. RDF uses XML to exchange descriptions of Web resources\nand emphasizes facilities to enable automated processing. The RDF descriptions pro-\nvide a simple ontology system to support the exchange of knowledge and semantic\ninformation on the Web. RDF Schema [2] provides the basic vocabulary to describe\nRDF documents. RDF Schema can be used to deﬁne properties and types of the web\nOWL constructs\nOWL class\nOWL property\nOWL subclass[C ]\nOWL subproperty[P ]\ninstanceof [C ]\nDescription\nclasses\nproperties\nsubclasses of C\nsubproperties of P\ninstances of the OWL class C\nresources. The advent of RDF Schema represented an early attempt at an SW ontology\nlanguage based on RDF.\nOWL [14] is a standard (W3C recommendation) for expressing ontologies in the\nSemantic Web. The OWL language facilitates greater machine understandability of\nWeb resources than that supported by RDFS by providing additional constructors for\nbuilding class and property descriptions (vocabulary) and new axioms (constraints),\nalong with a formal semantics. OWL consists of three sub-languages: OWL Lite, OWL\nDL and OWL Full, with increasing expressiveness. OWL Lite and DL are decidable,\nbut OWL Full is generally not. An OWL ontology consists of classes, properties and\nindividuals. Classes are interpreted as sets of objects that represent the individuals in\nthe domain of discourse. Properties are binary relations that link individuals, and are\ninterpreted as sets of tuples, which are the subsets of the cross product of the objects in\nthe domain of discourse. OWL classes fall into two main categories – named classes\nand anonymous classes. Anonymous classes are formed from logical statements. They\ncontain the individuals that satisfy the logical description. Anonymous classes may be\nfurther sub-divided into restrictions and logical class expressions. We summarize some\nessential OWL constructs in Table 1. To be simpler, informal OWL syntax has been\nused here. For example, OWL class denotes the OWL construct “Class” .\n2.1.2 SWRL\nAlthough OWL includes a relatively rich set of class constructors, the language pro-\nvided for expressing properties is much weaker. SWRL [9] intends to overcome the\nexpressive restriction of OWL properties by extending OWL with some form of “rule\nlanguage”. SWRL is based on a combination of the OWL DL and OWL Lite sub-\nlanguages of the OWL Web Ontology Language with the Unary/Binary Catalog sub-\nlanguages of the Rule Markup Language. SWRL introduces a high-level abstract syn-\ntax for Horn-like rules in both the OWL DL and OWL Lite sub-languages of OWL.\nSWRL extends OWL by also allowing rule axioms, i.e., by adding the construct:\naxiom ::= rule\nA rule axiom consists of an antecedent and a consequent, each of which consists of\na set of atoms which could be class membership (C(x)), property membership (P(x,y))\nor individual in/equalities(differentFrom(x,y)/sameAs(x, y)). Informally, a rule means\nthat if the antecedent holds (is “true”), then the consequent must also hold. A simple\nexample of the rules could be used to express the knowledge that “if ?x 1 is a child of\n?x 2 and ?x 2 is a brother of ?x 3, then ?x 3 is an uncle of ?x 1”. Informally, this rule\ncould be written as:\nhasChild (?x 2, ?x 1) ∧ hasBrother (?x 2, ?x 3)\n⇒ hasUncle(?x 1, ?x 3)\n2.1.3 SWRL-FOL\nSWRL-FOL [15] extends SWRL axiom to arbitrary ﬁrst-order formula over unary and\nbinary predicates. It extends SWRL with “assertion” axioms that contain ﬁrst-order\nsentences , i.e.\naxiom ::= assertion\nAssertions assert ﬁrst-order sentences, where no free variables are allowed in the\nformulae. For example an axiom could be used to express the knowledge that “for all\nthe person ?x 1 if s/he is a ‘wealthy Parent’, then s/he has at least one child ?x 2 who is\na millionnaire.”. Informally, this axiom could be written as:\n∀?x 1 | wealthyParent (?x 1) ⇒\n∃?x 2 | hasChild (?x 1, ?x 2) ∧ millionnaire(?x 2)\nWe can see from the above example that by introducing ﬁrst-order formulas, more\ncomplex logical statements can be expressed in SWRL.\n2.1.4 Existing reasoning tools for Semantic Web\nOntology reasoning tools have been built alongside the development of ontology lan-\nguages. The rest of this subsection will introduce a few of these tools.\nCwm (Closed world machine) [18] is a general-purpose data processor for the Se-\nmantic Web. Implemented in Python and command-line based, it is a forward chaining\nreasoner for RDF.\nTriple [17] is a RDF query, inference and transformation language. It does not\nhave a built-in semantics for RDF Schema, but it allows semantics of languages to be\ndeﬁned with rules on top of RDF. This feature of Triple facilitates data aggregation as\nusers can perform RDF reasoning and transformation under different semantics. The\nTriple tool supports OWL through external OWL reasoners such as FaCT and RACER.\nFaCT (Fast Classiﬁcation of Terminologies) [8], developed at University of Manch-\nester, is a TBox (concept-level) reasoner that supports automated concept-level reason-\ning, namely class subsumption and consistency reasoning. It does not support ABox\n(instance-level) reasoning. It is implemented in Common Lisp and comes with a FaCT\nserver, which can be accessed across network via its CORBA interface. Given an OWL\nontology, it can classify the ontology (performs subsumption reasoning) so as to reduce\nredundancy and detect any inconsistency within it.\nRACER, the Renamed ABox and Concept Expression Reasoner [6], implements\na TBox and partial ABox reasoner for the description logic ALCQHIR+ (D)− [7]. It\ncan be regarded as (a) a Semantic Web inference engine, (b) a description logic reason-\ning system capable of both TBox and ABox reasoning and (c) a prover for modal logic\nKm. In the Semantic Web domain, RACER’s functionalities include developing on-\ntologies (creating, maintaining and deleting concepts, roles and individuals); querying,\nretrieving and evaluating the knowledge base, etc. It supports OWL and RDF.\nThe FaCT and RACER are the most well accepted OWL reasoners. However,\nthey still have many limitations, such as both of them can only ﬂag an OWL class\nis inconsistent without providing any explanation. The debugging task is left to the\nuser. Furthermore, there is very limited datatype support, such as integer and string.\nAlso FaCT does not provide any ABox reasoning. RACER can only partially support\nABox reasoning. Alloy approach proposed in this paper can complement FaCT and\nRACER [3].\nCurrently, there is not a well accepted system supporting SWRL, and only few\nprototypes has been developed. There is no reasoning tool supporting SWRL-FOL yet.\n2.2 Alloy overview\nAlloy [11] is a structural modelling language based on ﬁrst-order logic, for express-\ning complex structural and behavioral constraints. Alloy treats relations as ﬁrst class\ncitizens and uses relational composition as a powerful operator to combine various\nstructured entities. The essential constructs of Alloy are as follows:\n• Signature: A signature (sig) paragraph introduces a basic type, a collection of\nrelations (called ﬁeld), and a set of constraints on their values. A signature may\ninherit ﬁelds and constraints from another signature.\n• Function: A function (fun) captures behavior constraints. It is a parameterized\nformula that can be “applied” elsewhere.\n• Fact: Fact (fact ) imposes global constraints on the relations and objects. A fact\nis a formula that takes no arguments and needs not to be invoked explicitly.\n• Assertion: An assertion (assert ) speciﬁes an intended property. It is a formula\nwhose correctness needs to be checked, assuming the facts in the model.\nThe Alloy Analyzer is a tool for analyzing models written in Alloy. Given a ﬁ-\nnite scope for a speciﬁcation, Alloy Analyzer translates it into a propositional formula\nand uses SAT solving technology to generate instances that can satisfy the facts and\nproperties expressed in the speciﬁcation.\n3 Alloy semantics for OWL/SWRL/SWRL-FOL\nThis section presents the Alloy semantics for OWL, SWRL and SWRL-FOL lan-\nguages, which forms the foundation of the reasoning environment. Due to limited\nspace, only part of semantic model has been presented here. A complete Alloy seman-\ntics for these languages can be found at http://www.cs.man.ac.uk/∼hwang/\nswrlfol.als.\n3.1 Alloy semantic for OWL constructs\n3.1.1 Basic concepts\nThe semantic model for OWL is encoded in the module OWL. Users only need to\nimport this module to reason about OWL ontology in Alloy.\nmodule OWL\nAll the things described in the Semantic web context are referred to as web re-\nsources. A basic type Resource is deﬁned as:\nsig Resource {}\nOther concepts such as classes and properties deﬁned later are extended from the\nResource. Property, which is a kind of Resource itself, relates Resource to Re-\nsource.\ndisj sig Property extends Resource\n{sub_val: Resource -> Resource}\n“disj” is a keyward from Alloy for denoting the disjointness. Each Property has a\nrelation sub val from set <Property, Resource, Resource> with type <Resource,\nResource, Resource> (since in Alloy subsignature does not introduce a new type).\nThis relation can be regarded as a RDF statement, i.e., a triple of the form <property(or\npredicate), subject, value(or object)>.\nThe class corresponds to the generic concept of type or category of resource. Each\nClass maps a set of resources via the relation instances, which contains all the in-\nstance resources. The keyword disj is used to indicate the Class and Property are\ndisjoint.\ndisj sig Class extends Resource {instances: set Resource}\nThe OWL also allows the use of XML Schema datatypes to describe (or deﬁne)\npart of the datatype domain. Alloy supports Integer and String. Apart from these there\nare no predeﬁned types in Alloy, Datatype has been treated as a special Class, which\ncontains all the possible datatype values in the instances relation.\ndisj sig Datatype extends Class {}\n3.1.2 Class elements\nThe subClassOf is a relation between classes. The instances in a subclass are also in\nthe super-classes. A parameterized formula (a function in Alloy) is used to represent\nthis concept.\nfun subClassOf(csup, csub: Class)\n{csub.instances in csup.instances}\nThe disjointWith is a relation between classes. It asserts that there are no instances\ncommon with each other.\nfun disjointWith (c1, c2: Class) {no c1.instances & c2.instances}\n3.1.3 Property restrictions\nThe allValuesFrom construct states that all instances of the class c1 that have the\nvalues of property P all belong to the class c2.\n(p: Property, c1: Class, c2: Class)\n{all r1, r2: Resource |\nr1 in c1.instances =>\nr2 in r1.(p.sub_val) =>\nr2 in c2.instances}\nA hasValue function states that all instances of the class c1 have the values of\nproperty P as resource r. The r could be an individual object or a datatype value.\nfun hasValue (p: Property, c1: Class, r: Resource)\n{all r1: Resource | r1 in c1.instances => r1.(p.sub_val) = r}\nA MaxCardinality function states that all instances of the class c1 have at most N\ndistinct values for the property P. Alloy supports some integer operations.\nfun maxCardinality (p: Property, c1: Class, N: Int)\n{all r1: Resource| r1 in c1.instances <=>\n# r1.(p.sub_val) <= int N }\n3.2 Boolean combination of class expressions\nThe intersectionOf function deﬁnes a relation between a class c1 and a list of classes\nclist. The List is deﬁned in the Alloy library. The class c1 consists of exactly all the\nobjects that are common to all class expressions from the list clist.\nfun intersectionOf (clist: List, c1: Class)\n{all r: Resource| r in c1.instances <=>\nall ca: clist.*next.val | r in ca.instances}\nThe unionOf function deﬁnes a relation between a class c1 and a list of classes\nclist. The class c1 consists of exactly all the objects that belong to at least one of the\nclass expressions from the list clist. It is analogous to logical disjunction;\nfun unionOf (clist: List, c1: Class)\n{all r: Resource| r in c1.instances <=>\nsome ca: clist.*next.val| r in ca.instances}\n3.2.1 Property elements\nThe subPropertyOf construct states that psub is a sub-property of the property psup.\nThis means that every pair (subject,value) that is in psup is also in the psub.\nfun subPropertyOf (psup, psub: Property)\n{psub.sub_val in psup.sub_val}\nThe domain function asserts that the property P only applies to instances of the\nfun domain (p: Property, c: Class)\n{(p.sub_val).Resource in c.instances}\nThe inverseOf function shows two properties are inverse.\nfun inverseOf (p1, p2: Property) {p1.sub_val = ˜(p2.sub_val)}\nAll other OWL constructs can be deﬁned in a similar manner. Please refer to the\ncomplete OWL Alloy semantics online.\n3.3 Alloy semantic for SWRL extension\nSWRL extends OWL by adding the rule axioms. A rule axiom consists of an antecedent\nand a consequent, each of which consists of a set of atoms. Atoms can be of the\nfollowing forms, where C is an OWL description, P is an OWL property, and x,y are\neither variables, OWL individuals or OWL data values.\n• C(x): Informally, it holds if x is an instance of the class description C.\n• P(x,y): It holds if x is related to y by property P.\n• sameAs(x,y): It holds if x is interpreted as the same object as y.\n• differentFrom(x,y): It holds if x and y are interpreted as different objects.\nAtom\nC (x )\nP (x , y)\nsameAs(x , y)\ndiﬀerentFrom(x , y)\nAlloy representation\n‘x in C.instances’\n‘(x->y) in P.sub val’\n‘x = y’\n‘x != y’\nAs mentioned before, a rule means that if the antecedent holds, the consequent must\nalso hold. It can be modelled as a universally quantiﬁed fact in the form of implication.\nFor example the following rule axiom (where a0 ... an are atoms)\nImplies(Antecedent (a1, ..., an ) Consequent (a0))\nwill be modelled as:\nfact { a_1 && ... && a_n => a_0 }\nSWRL-FOL formula Alloy semantics\nand (C1...Cn )\nor (C1...Cn )\nneg(C )\nimplies(C1 C2)\nequivalent(C1 C2)\nforall (V1...Vn C )\nexists(V1...Vn C )\nfact {C1 && ... && Cn }\nfact {C1 || ... || Cn }\nfact {not C}\nfact {C1 => C2}\nfact {C1 <=> C2}\nfact {all V1, ..., Vn : Resource | C}\nfact {some V1, ..., Vn : Resource | C}\n3.4 Alloy semantic for SWRL-FOL extension\nSWRL-FOL extends SWRL with assertion axioms that contain ﬁrst-order formulas.\nTable 3 presents the Alloy semantic for different SWRL-FOL formulas.\nThe above deﬁnes the basic transformation guidelines from the SWRL-FOL into\ntheir corresponding Alloy semantics. We will demonstrate the actual transformation\nprocess in the following section.\n4 OWL/SWRL/SWRL-FOL to Alloy transformation\nThe previous section presented Alloy semantics for OWL, SWRL and SWRL-FOL,\nwhich forms the foundation for the reasoning environment. To be able to perform the\nautomatic reasoning task using Alloy Analyzer, a Java program has been developed\nfor the automatic transformation from an OWL/SWRL/SWRL-FOL knowledge ﬁle (in\nXML format) into its corresponding Alloy model.\nA set of translation rules are developed in the following presentation.\n4.1 OWL class translation\nC ∈ OWL class\nstatic disj sig C extends Class{}\nAn OWL class C will be transferred into a scalar C, constrained to be an element of\nthe signature Class.\n4.2 OWL property translation\nP ∈ OWL property\nstatic disj sig P extends Property{}\nAn OWL property p will be translated into a scalar P, constrained to be an element of\nthe signature Property.\nx ∈ instancesof [Y ]\nstatic disj sig x extends Resource{}\nfact { x in Y .instances}\nAn OWL individual x of class Y will be translated into a scalar x, constrained to be an\nelement of the signature Resource. x is a subset of Y.instances.\n4.4 Other OWL translations\nOther OWL constructs can be easily translated into the Alloy functions deﬁned in the\nprevious section. For example the following rule shows how to translate the OWL\nsubclass relation into Alloy code.\nsubclass[X , Y ], X ∈ OWL class, Y ∈ OWL class\nfact {subClassOf (X , Y )}\n4.5 SWRL and SWRL-FOL translation\nThe transformation of SWRL rules follows the semantics deﬁned in Table 2. The\nvariable x and y will be bound by some universal quantiﬁers. The SWRL rule can be\nmodelled as a universally quantiﬁed fact in the form of implication.\nSimilarly, the transformation of SWRL-FOL follows the semantics presented in\nTable 3. More translation rules can be found from [19] and the web site http:\n//nt-appn.comp.nus.edu.sg/fm/alloy/introduction.htm.\n4.6 Translation example\nThe translation rules have been implemented in a Java program. The following OWL\nontology deﬁnes two classes animal and plant which are disjoint. The eats and\neaten by are two properties, which are inverse to each other. The domain of eats\nis animal. The carnivore is a subclass of animal which can only eat animals. The\nontology is given in a syntax similar to the “DL syntax” given in [10].\nClass (animal)\nClass (plant)\nDisjointClasses( animal plant )\nObjectProperty(eaten by)\nObjectProperty(eats\ndomain (animal))\nInverseProperties(eats eaten by)\nClass (carnivore complete animal\nrestriction(eat allValuesFrom animal))\nClass (herbivore complete animal\nrestriction(eat allValuesFrom plant))\nThis fragment ontology can be transformed by the tool into the following Alloy\nmodule animal\n/*import the defined library module */\nopen SWRL-FOL\n/* plant and animal are translated to two class instances. The key\nword static is used to a signature containing exactly one element.*/\nstatic disj sig plant, animal extends Class {}\n/* The disjoin element was translated into fact in Alloy */\nfact {disjointWith(plant, animal)}\n/* eats, eaten_by are translated to two property instances */\nstatic disj sig eats, eaten_by extends Property {}\nfact {inverseOf(eats, eaten_by)}\nfact {domain(eats, animal)}\nstatic disj sig carnivore extends Class{}\nfact{subClass(animal, carnivore)}\nfact{allValuesFrom(eats, carnivore, animal)}\nstatic disj sig herbivore extends Class{}\nfact{subClass(animal, herbivore)}\nfact{allValuesFrom(eats, herbivore, plant)}\nThe transformation of SWRL rules follows the semantics deﬁned in Table 2. The\nvariable x and y will be bound by some universal quantiﬁers. The SWRL rule can be\nmodelled as a universally quantiﬁed fact in the form of implication. For example the\nfollowing rule axiom\nhasParent (?x 1, ?x 2) ∧ hasBrother (?x 2, ?x 3) ⇒\nhasUncle(?x 1, ?x 3)\nwill be modelled as:\nfact {all x1, x2, x3: Resource |\n(x1->x2) in hasParent.sub_val &&\n(x2->x3) in hasBrother.sub_val =>\n(x1->x3) in hasUncle.sub_val}\nThe transformation of SWRL-FOL follows the semantic presented in Table 3. After\ntransforming the ontologties to the Alloy model, the consistency of the OWL/SWRL/SWRL-\nFOL ontology can be checked and some reasoning can be done readily.\n5 Reasoning OWL/SWRL/SWRL-FOL ontology with\nAlloy Analyzer\nReasoning is one of the key tasks for Semantic Web applications. It can be useful\nat many stages during the design, veriﬁcation, maintenance and deployment of web\nontology. In this section, we show that different Semantic Web reasoning tasks can be\naccomplished by using the Alloy Analyzer.\n5.1 Standard OWL reasoning tasks\nThere are two different levels of checking and reasoning in OWL, the conceptual level\nand the instance level. At the conceptual level, the class properties and subclass rela-\ntionships can be reasoned. At the instance level, the membership checking (instantia-\ntion) and instance property reasoning can be done.\n5.1.1 Class property checking\nIt is essential that the ontology shared between autonomous software agents is con-\nceptually consistent. Reasoning with inconsistent ontologies may lead to erroneous\nconclusions. An OWL class is deemed to be unsatisﬁable (inconsistent) if, because of\nits description, it cannot possibly have any instances. This section gives some exam-\nples of inconsistent ontology that can arise in ontology development, and demonstrate\nhow these inconsistencies can be detected by the Alloy Analyzer. For example, another\nclass tastyPlant which is a subclass of plant and eaten by the carnivore is deﬁned.\nThere is an inconsistency since by the ontology deﬁnition carnivores can only eat ani-\nmals. Animals and plants are disjoint.\nClass (tastyPlant partial plant\nrestriction(eat by allValuesFrom(carnivore)))\nWe translate the ontology into an Alloy program, add some facts to remove the triv-\nial models (like every type is empty set) and load the program into the Alloy Analyzer.\nThe Alloy Analyzer will automatically check the consistency. AA attempts to ﬁnd a\nmodel – a binding of the variables to values – that makes for the formulas (the formulas\ntranslated from the OWL model) true. If no such model can be build, it means that the\nmodel has been over constrained, i.e, there are some contradiction (inconsistency) in\nthe model.\nIn the example, it can be concluded that there is an inconsistency in the animal\nontology since Alloy can not ﬁnd any solutions satisfying all facts within the scope\n(Figure 1). Note that when Alloy can not ﬁnd a solution, it may be due to the scope\nbeing too small. By picking a large enough scope, “no solution found’ is very likely\nto mean that an inconsistency has occurred. AA tried to constructs a model which\nsatisﬁed all asserted axioms. If no such a model could be build (“no solution found”),\nthen there are some contradicted axioms in the model.\nBesides discovering the existence of an inconsistency in ontology, tracing where\nthe inconsistency arises from is also crucial for a reasoning tool to be practical. The\nexisting OWL reasoners like FaCT and RACER can only ﬂag the inconsistent class\nwithout providing any explanation. The debugging process is left to users. Without any\ntool support, identifying the conﬂicting knowledge could be frustrating. One possible\nsystematic technique for ﬁnding the causes of inconsistent ontology is to manually\nremove individual knowledge information until the culprit is identiﬁed. This task can\nbe lengthy and dangerous.\nIn Alloy, the “unsatisﬁed core” [16] functionality of recent SAT solvers was utilized\nand it supports core extraction, a new analysis technique that helps to discover over-\nconstraint in declarative models. This functionality can provide some assistance for the\nuser to trace the inconsistency.\nExtracting the unsatisﬁable core of a CNF formula, that is a subset of the clause\nset sufﬁcient to cause a contradiction, has been developed recently by satisﬁability\nsolvers [16]. In the latest version of Alloy, the declarative model analysis has been\ncast as satisﬁability instances and the unsatisﬁable core has been mapped back onto the\nmodel. In other words, a user can identify the parts of model responsible for producing\nthe unsatisﬁable CNF core. Those parts, by themselves, sufﬁce to produce an over-\nconstraint, and their identiﬁcation can help the user ﬁnd the over-constraint. Using this\nfunctionality, the portions of the ontology which contradict each other can be traced\nreadily. In the animal example, suppose a new class named funnything was deﬁned to\nbe a subclass of both animal and plant classes. It is easy to see that there is an incon-\nsistency since the class animal and plant are disjoint. Alloy can automatically identify\na set of knowledge which makes the ontology unsatisﬁable (Figure 2). The unsatisﬁ-\nability maybe due to the fact that funnything is a subclass of animal, funnything is a\nsubclass of plant or animal and plant are disjoint classes, and so on.\n5.1.2 Subsumption reasoning\nThe task of subsumption reasoning is to infer an OWL class is the subclass of another\nOWL class. That is for every instances of one OWL class, it is an instance of another\nOWL class as well. Using AA, the subsumption relationship between classes can be\nchecked automatically. The relationship between the ﬁsh, shark and dolphin has been\nused as an example to demonstrate this kind of reasoning task. In the animal ontol-\nogy a property breathe by is deﬁned. The ﬁsh is a subclass of the animal which\nbreathe by the gill.\nObjectProperty(breathe by)\nClass (gill)\nClass (ﬁsh complete animal\nrestriction(breathe by allValuesFrom gill))\nSince the purpose of this paper is to demonstrate ideas, the ontology has been kept\nsimple. In reality there are some animals such as frogs and toads, which can respire\nby use of gills when they are young and by lungs when they reach adult stage. Also\ncases like that the animals which respire by use of the pharyngeal lining or skin, like\nnewborn Julia Creek dunnarts have not been considered. A class shark, a subclass of\ncarnivore which breathe by the gill, has also been deﬁned.\nClass (shark)\nClass (ﬁsh complete animal\nrestriction(breathe by allValuesFrom gill))\nSeveral of the classes were upgraded to be deﬁned when their deﬁnitions consti-\ntuted both necessary and sufﬁcient conditions for class membership, e.g., an animal is\na ﬁsh if and only if it breathes by the gill. Additional subclass relationships can be in-\nferred, i.e., the shark is also a subclass of ﬁsh. We transfer this ontology into an Alloy\nprogram and make an assertion that the shark is a subclass of ﬁsh. The Alloy ana-\nlyzer will check the correctness of this assertion automatically (Figure 3). The Alloy\nAnalyzer checks whether an assertion holds by trying to ﬁnd a counterexample. Note\nthat “no solution” means no counterexample found, in this case, it strongly suggests\nthat the assertion is sound. To make it more interesting, classes dolphin and lung are\ndeﬁned. Dolphins are a kind of animal which breathe by lungs. The classes gill and\nlung are disjoint.\nClass (lung)\nDisjointClasses( lung gill )\nClass (dolphin complete animal\nrestriction(breathe by allValuesFrom lung))\nSuppose an assertion that the dolphin is a kind of ﬁsh is made, the Alloy Analyzer\nwill refute it since some counterexample was found (Figure 4). If the fact that dolphin\nis a ﬁsh is added in the module, the AA will conclude that an inconsistency has arisen.\n5.1.3 Debugging uncompleted ontology\nInformation in OWL is gathered into ontologies, which can then be from different\nparties and stored as documents in the World Wide Web. Some knowledge may be\nmissing in the ontology. Reasoning about uncompleted ontologies may lead to some\nunexpected results. We refer to the situation that because of some unavailable knowl-\nedges, the reasoners had inferred some unexpected knowledges which is different with\nnatural facts. We need some tools to help the users to trace what is the missed knowl-\nedge causing the untrue conclusion that has been drew. AA checks the assertion by\ngenerating counterexamples – structures or behaviors for which an expected property\ndoes not hold; from a counterexample, it is usually not too hard to ﬁgure out what is\nwrong. Looking at the counterexamples may provide some hints to the user on why the\nexpected result does not hold and what knowledge is missing. For example, to show\nthe OWL class dolphin and shark are disjoint, Intuitively, this is a correct statement\nsince dolphin breathes by the gill while shark breathes by the lung. Gill and lung are\ndisjoint. When the following assertion is added to Alloy, surprisingly AA concludes it\nis wrong.\n{disjointWith(shark, dolphin)}\nBy looking at the counterexamples graph, it has been noticed that all the counterex-\namples (an animal which is both a shark and a dolphin) generated by AA have empty\nvalues for the property breath by. In fact this unexpected result comes from the se-\nmantic of allValuesFrom construct in OWL. An OWL semantic can not deduce from\na allValuesFrom restriction alone that there actually is at least one value for the prop-\nerty. An allValuesFrom restriction for a property is trivially satisﬁed for an instance\nthat has no value for that property at all. The allValuesFrom restriction demands that\nall values of the property belong to a class, and if no such values exist, the restriction\nis trivially true. That is the reason why AA ﬁnds out the common instance, which does\nnot breathe at all, for the class dolphin and class shark. To remove this expected result,\nextra knowledge needs to be added, e.g., an animal must breathe by something.\n5.1.4 Instantiation\nInstantiation is one of the main contributions for reasoning over OWL ontology using\nAlloy. Currently some successful OWL reasoners like FaCT are designed for descrip-\ntion logic (DL) T-box reasoning, which lacks support for instances. In Alloy every\nexpression denotes relations. The scalars will be represented by singleton unary rela-\ntions - that is, relations with one column and one row. The instance level reasoning can\nbe supported readily in Alloy.\nInstantiation is a reasoning task which tries to check if an individual is an instance\nof a class. For example, two resources aFeralAnimal and aMeekAminal are deﬁned\nas the instances of class animal. aGill is an instance of class gill. aFeralAnimal eats\naMeekAnimal and breathes by aGill. People may want to check if aFeralAnimal is a\ncarnivore and a ﬁsh.\nIndividual(aMeekAnimal type(animal))\nIndividual(aGill type(gill))\nIndividual(aFeralAnimal type(animal) value(breathe by aGill)\nvalue(eats aMeekAnimal))\nWe translate the ontology into an Alloy program and make an assertion as follow-\nstatic disj sig aFeralAnimal, aMeekAnimal extends Resource{}\nstatic disj sig aGill extends Resource{}\nfact {aFeralAnimal in animal.instances &&\nfact {aGill in gill.instances}\nfact {(aFeralAnimal->aMeekAnimal) in eats.sub_val}\nfact {(aFeralAnimal->aGill) in breathe_by.sub_val}\nassert isFishCarnivore\n&& (aFeralAnimal in carnivore.instances)}\ncheck isFishCarnivore for 15\nAA concludes that this assertion is correct.\n5.1.5 Instance property reasoning\nInstance property reasoning (often regarded as knowledge querying) is important in\nSemantic Web applications.\nIt is a task to query some properties with individuals.\nSince one of the promising strengths of Semantic Web technology is that it gives the\nagents the capability to do more accurate and more meaningful searches. The agent can\nanswer some questions for which the answers are not explicitly stored in the knowledge\nbase.\nFor example, the emerge early and emerge later are two properties, which are\ninverse to each other. Animal A emerges earlier than B if the species of A emerge earlier\nthan the species of B on the earth. emerge early is transitive. Three animal instances\nﬁrstDinosaur, ﬁrstApe and ﬁrstHuman are deﬁned. ﬁrstDinosaur emerge early\nthan ﬁrstApe and ﬁrstApe emerge early than ﬁrstHuman. One possible question\npeople may ask is whether ﬁrstHuman is emerge later than ﬁrstDinosaur. With the\nassistance of Alloy reasoner, such questions can be answered.\nfact{TransitiveProperty(emerge_early)}\nstatic disj sig firstDinosaur, firstApe,\nfact { firstDinosaur in animal.instances\n&& firstApe in animal.instances\n&& firstHuman in animal.instances}\nfact {(firstDinosaur->firstApe) in emerge_early.sub_val}\nfact {(firstApe->firstHuman) in emerge_early.sub_val}\nassert hum {(firstHuman->firstDinosaur) in emerge_later.sub_val}\ncheck hum for 14\nAA concludes that this assertion is correct.\n5.2 SWRL/SWRL-FOL related reasoning\nBesides of being capable to support the standard reasoning tasks on OWL, such as per-\nforming consistency checking, subsumption and instantiation reasoning automatically,\nmoreover, Alloy can also check more complicated ontology properties expressed by the\nnewly extended languages such as SWRL/SWRL-FOL. In this section, we demonstrate\nhow Alloy can be used to reasoning the SWRL-FOL ontologies.\nA family relationship web ontology example is used here to illustrate the rea-\nsoning process. The following fragment of ontology ﬁrst deﬁnes two OWL classes,\nPerson and twinParent that represents the set of person who are the parents\nof twins, and three OWL object properties, i.e., hasChild, brotherSister and\nsameBirthTime. Secondly, the ontology class wealthyParent introduces the\nset of parents who have a child who is a millionnaire. Thirdly, two SWRL-FOL\naxiomatic assertions are deﬁned to provide inference for the brotherSister and\ntwinParent relationships. Lastly, the ontology class wealthyTwinParent is\ndeﬁned as a parent being both wealthyParent and twinParent.\nClass (Person partial)\nClass (twinParent partial Person)\nClass (millionnaire partial Person)\nObjectProperty(hasChild)\nObjectProperty(brotherSister)\nObjectProperty(sameBirthTime)\nClass (wealthyParent complete Person\nrestriction(hasChild someValuesFrom(millionnaire)))\nAssertion(forall I-variable(x1) I-variable(x2)\n(equivalent (exists (I-variable(x3)\n(and(hasChild(x3,x1) hasChild(x3,x2)\ndifferentFrom(x1,x2))))\n(brotherSister(x1, x2)))))\n(equivalent (exists (I-variable(x2)\n(exists (I-variable(x3)\n(and(brotherSister(x2, x3) sameBirthTime(x2, x3)\nhasChild(x1, x2))))))\n(twinParent(x1)))))\nClass (wealthyTwinParent complete wealthyParent twinParent)\nFrom the above, it is noticed that two SWRL-FOL axioms were asserted. The ﬁrst\nassertion shows that if two distinct people have a same parent, then they are brothers\nor sisters. The second assertion in the above ontology shows that if two people are\nbrothers or sisters, and they have the same birth time, then their parents are twin-\nparents. Furthermore, suppose some instances of the above ontology are asserted into\nthe knowledge base as follows.\nIndividual(Tom type(person)\ntype(complementOf(wealthyTwinParent))\nvalue(hasChild Jerry)\nvalue(hasChild Jim))\nIndividual(Jerry type(millionnaire) value(sameBirthTime Jim))\nIndividual(Jim type(person))\nDisjointWith(Jim Jerry)\nWe transform the above ontology (in XML format) into its Alloy model3 using our\nSimilar as reasoning OWL, the Alloy Analyzer can automatically perform different\nreasoning tasks for SWRL/SWRL-FOL. For example, it can detect that there is an\ninconsistency in the above ontology example, as the Alloy Analyzer can not ﬁnd any\nontology instances (solutions) satisfying all facts within the scope.\nIn this family ontology example, the inconsistency comes from the fact that Tom\nhas been inferred as an instance of both the class wealthyParent and the class\ntwinParent. However, there is a piece of knowledge in the model that explicitly\nindicates that Tom is not an instance of the wealthyTwinParent class, which con-\ntradicts to the inferred conclusion. As discussed before, with the assistance of Alloy\nAnalyzer’s “unsatisﬁable core” functionality, the debugging process of identifying the\nsource of inconsistency in the ontology becomes much more handy to the users.\n5.3 Discussion\nThe correctness of the translation has been veriﬁed by many different test cases. A\nsame problem has been sent to existing SW tools, theorem provers and Alloy; the same\nconclusions are drawn. Furthermore, the OWL has well deﬁned semantics in ﬁrst order\nlogic and Alloy is also based on the ﬁrst-order logic. The soundness of the translation\ncan also be proved easily. In the early work [13], it shows that the consistence between\nthe Alloy Semantic for the Semantic Web languages and the original OWL semantic\n(Alloy has been regarded as a subset of Z). Formal proving this consistence is beyond\nthe scope of this paper.\n6 Related works and conclusion\nThis paper presented a reasoning environment for the Semantic Web ontology family\nlanguages (OWL/SWRL/SWRL-FOL). There are four main contributions of the paper.\nFirstly, it deﬁnes a semantic encoding for the OWL/SWRL/SWRL-FOL constructs in\nthe Alloy ﬁrst-order language. Secondly, it presents a systematic transformation tool\nfrom the OWL/SWRL/SWRL-FOL ontology (in XML) into its corresponding Alloy\nmodel. Thirdly, with the assistance of Alloy Analyzer, it has been demonstrated that\nthe consistency of an ontology model can be checked automatically and different kinds\nof reasoning tasks can be supported. Our approach complements with existing OWL\nreasoners by providing full automatic debugging aids and instance level reasoning.\nFurthermore, SWRL-FOL is a newly proposed extension to OWL, and to our best of\nknowledge, so far there is no existing reasoning support for SWRL-FOL prior to this\nwork. Finally, the paper also demonstrates a light-weight formal methods approach to\nthe web ontology domain. Alloy was chosen over other reasoning tools because it is\nbased on ﬁrst-order relational logic and relations between Web resources are the focus\nissues in the Semantic Web context. Furthermore, Alloy has an impressive automatic\ntool support, the Alloy Analyzer, where automated generation of ﬁnite set of ontology\ninstances, creation of counter-examples on assertions, and identifying the source of\ninconsistencies in the model are made available. Usual ontology tools such as FACT\nand RASER can detect errors in an ontology model, but may not be able to point out\nwhere the error is. Alloy approach provides the ontology “surgery” like capability to\npin point the errors in the model with counter-examples or contradictory constraints.\nThis is a highly complementary approach to Semantic Web reasoning. The approach\nhas been successfully applied to a recent military ontology [5].\nIt has indeed been realized that there is a limitation on the scalability of the current\nAlloy Analyzer in reasoning large ontology models. The approach presented here can\nonly deal with the ontologies with relatively small size. Based on the same idea, authors\nalso attempt to use the theorem prover, i.e. Z/EVES, to reason the SW ontology [4].\nThe theorem prover can handle large sized ontologies, but it requires the user’s interac-\ntion. Here authors do not claim that Alloy is the only and best formal tool to reason over\nSW ontologies, but authors do claim that it is an effective attempt with certain novel\nand irreplaceable advantages like full automation and promising debugging assistance.\nIn fact, it is unlikely in the near future that both expressive and automatic tool will be\ndeveloped. Currently, it is desirable if the strength from different ontology reasoning\ntools can be integrated. [3] presented the methodology of checking ontologies using\ntools RACER, Z/EVES and AA in conjunction. This approach has been successfully\napplied for reasoning a real life military ontology.\nIn the future, it has been planned to integrate the current Alloy Analyzer reasoning\nfacilities into our OWL/SWRL/SWRL-FOL transformation tool by connecting it to the\nAlloy API interfaces. In addition, we also plan to extend the transformation tool with\nthe editing and designing functions for the ontology models, so that it will become an\nintegrated development environment for the web ontology modelling, which includes\ndesign, transformation and reasoning functions in one coherent tool support.",
         "",
         "Reasoning support for Semantic Web ontology family languages using Alloy",
         "None"
        ],
        [
         "12",
         "0009cc65e12ef353daf3bcc34fa2e24bbe15c7c8",
         "To better elucidate the role of vascular endothelial growth factor (VEGF)(165) in soft tissue sarcoma (STS) growth, metastasis, and chemoresistance, we generated stably transfected human STS cell lines with VEGF(165) to study the effect of VEGF(165) on STS cells in vitro and the effect of culture medium from these cells on human umbilical vascular endothelial cells. Severe combined immunodeficient mice bearing xenografts of transfected cell lines were used to assess the effect of VEGF overexpression and the effect of VEGF receptor (VEGFR) 2 inhibition on STS growth, metastasis, and response to doxorubicin. VEGF(165)-transfected xenografts formed highly vascular tumors with shorter latency, accelerated growth, enhanced chemoresistance, and increased incidence of pulmonary metastases. Blockade of VEGFR2 signaling using DC101 anti-VEGFR2 monoclonal antibody enhanced doxorubicin chemoresponse; this combined biochemotherapy inhibited tumor growth and decreased pulmonary metastases without overt toxicity. Combined therapy reduced microvessel counts while increasing vessel maturation index. VEGF overexpression did not affect on the sarcoma cells per se; however, conditioned medium from VEGF transfectants caused increased endothelial cell proliferation, migration, and chemoresistance. Addition of DC101 induced endothelial cell sensitivity to doxorubicin and suppressed the activity of matrix metalloproteinases secreted by endothelial cells. We therefore conclude that VEGF is a critical determinant of STS growth and metastasis and that STS chemoresistance, in our model, is a process induced by the interplay between STS cells and tumor-associated endothelial cells. STS growth and metastasis can be interrupted by combined low-dose doxorubicin and anti-VEGFR2, a strategy that attacks STS-associated endothelial cells. In the future, such therapeutic approaches may be useful in treating STS before the development of clinically apparent metastases.",
         "Lianglin Zhang,J. Hannay,Juehui Liu,P. Das,Maocheng Zhan,Theresa D. Nguyen,D. Hicklin,Dihua Yu,R. Pollock,D. Lev",
         "Lianglin Zhang,J. Hannay,Juehui Liu,P. Das,Maocheng Zhan,Theresa D. Nguyen,D. Hicklin,Dihua Yu,R. Pollock,D. Lev",
         "https://aacrjournals.org/cancerres/article-pdf/66/17/8770/2554504/8770.pdf",
         "To better elucidate the role of vascular endothelial growth\nfactor (VEGF)165 in soft\ntissue sarcoma (STS) growth,\nmetastasis, and chemoresistance, we generated stably trans-\nfected human STS cell lines with VEGF165 to study the effect\nof VEGF165 on STS cells in vitro and the effect of culture\nmedium from these cells on human umbilical vascular\nendothelial cells. Severe combined immunodeficient mice\nbearing xenografts of transfected cell\nlines were used to\nassess the effect of VEGF overexpression and the effect of\nVEGF receptor (VEGFR) 2 inhibition on STS growth,\nmetastasis, and response to doxorubicin. VEGF165-transfected\nxenografts formed highly vascular tumors with shorter\nlatency, accelerated growth, enhanced chemoresistance, and\nincreased incidence of pulmonary metastases. Blockade of\nVEGFR2 signaling using DC101 anti-VEGFR2 monoclonal\nantibody enhanced doxorubicin chemoresponse; this com-\nbined biochemotherapy inhibited tumor growth and de-\ncreased pulmonary metastases without overt\ntoxicity.\nCombined therapy reduced microvessel counts while increas-\ning vessel maturation index. VEGF overexpression did not\naffect on the sarcoma cells per se; however, conditioned\nmedium from VEGF transfectants caused increased endothe-\nlial cell proliferation, migration, and chemoresistance.\nAddition of DC101 induced endothelial cell sensitivity to\ndoxorubicin and suppressed the activity of matrix metal-\nloproteinases secreted by endothelial cells. We therefore\nconclude that VEGF is a critical determinant of STS growth\nand metastasis and that STS chemoresistance, in our model,\nis a process induced by the interplay between STS cells and\ntumor-associated endothelial cells. STS growth and metasta-\nsis can be interrupted by combined low-dose doxorubicin\nand anti-VEGFR2, a strategy that attacks STS-associated\nendothelial cells. In the future, such therapeutic approaches\nmay be useful in treating STS before the development of\nclinically apparent metastases. (Cancer Res 2006; 66(17): 8770-8)\nFive-year survival for soft tissue sarcoma (STS) patients is f50%\ndepending on tumor size, grade, location, the presence of regional\nor distant disease, and response to therapy (1). Metastases, mainly\nto the lung, are particularly ominous. The need to successfully\ncontrol STS dissemination by systemic approaches that can reach\ntotal body disease loci remains compelling. However, the use of\nchemotherapy in STS is problematic due to toxicity, expense, and\nmarked chemoresistance. Even doxorubicin, the single most active\nagent in STS, has a disappointing overall 30% response rate, with\nbreakthrough tumor progression and frequent recurrence after\ninitial sarcoma chemoresponsiveness (2). Although progression,\nmetastasis, and chemoresistance in STS have been characterized\nclinically,\nthe molecular mechanisms regulating these tumor\nprocesses are less well understood.",
         "Discussion\nIn this study, we examined the effects of VEGF165 overexpression\non human STS progression and chemotherapeutic efficacy. S.C.\nimplantation of VEGF165-overexpressing sarcoma cell lines in SCID\nmice produced highly vascular tumors that exhibited a much\nshorter latency, significantly accelerated tumor growth, angiogen-\nesis, and metastasis. However, no effect on sarcoma cell growth\nand invasion could be seen in vitro. We learned that the inhibitory\neffects of doxorubicin alone on tumor growth were minimal in\nVEGF165-overexpressing xenografts, whereas treatment with the\nanti-VEGFR2 monoclonal antibody combined with doxorubicin\nmarkedly improved xenograft responsiveness to doxorubicin.\nThe significance of VEGF expression in STS patients has been\nrecognized previously in patient-based descriptive studies. In a\nstudy of 85 STS patients, serum VEGF levels correlated significantly\nwith grade (25). Others have observed that the magnitude of\nSTS patient circulating or tumor VEGF levels is an important\nprognostic factor (10, 26). Mutation of p53 is very frequently found\nin STS and this may also play a role in up-regulating VEGF as we\nhave shown before (12). To the best of our knowledge, the current\nto show that overexpression of VEGF165\nstudy is the first\naccelerates STS angiogenesis, growth, metastasis, and chemo-\nresistance to doxorubicin. The effect of VEGF165 overexpression on\ntumor progression in different\ntumor types is not entirely\nconsistent. For example, overexpression of VEGF165 enhanced both\ntumor angiogenesis and growth in a MCF-7 human breast\ncarcinoma cell system (27) but only stimulated tumor angiogenesis\nwithout increasing the growth rate of OVCAR-3 human ovarian\ncancer xenografts (28). Varying degrees of effect were also noticed\nin preclinical models of h-cell carcinogenesis (29), melanoma (30),\nand glioma (31). Some of the apparent discrepancies observed in\nVEGF function in cancer development may be due to different ways\nVEGF impinges on the tumor-versus-microenvironment interplay\nand suggest that tumor type and tumor cell variability are\nimportant determinants for the role of VEGF.\nThis report also facilitates an awareness that chemoresistance\nmust be considered in the context of both tumor cells as well as\ncells recruited from the microenvironment. Examining both of\nthese cellular components in tandem has enabled an initial\nidentification of the critical role of endothelial cell chemoresistance\nin STS progression and dissemination. We have shown that STS\nchemoresistance is determined not only by tumor cell response to\ndrugs but also by host cells, although genetically stable, which can\nalso be adapted to become resistant to the toxic effect of\nchemotherapy. VEGF produced by tumor cells can mediate\nendothelial cell protection against the effects of chemotherapy by\npromoting the production and/or activation of cytoprotectors,\nsuch as survivin (32). In a similar manner, Kim et al. (33) have\nshown that the phosphatidylinositol 3-kinase/Akt pathway is\nactivated in endothelial cells exposed to radiation, which in turn\ncan lead to the activation of other prosurvival pathways. Targeting\nendothelial cell–based survival pathway components may prevent\ntheir production or activation and may be an important strategy to\novercome therapy resistance. In the context of tumor angiogenesis,\nthis conceptual framework suggests that abrogating tumor VEGF\nexpression-mediated chemoresistance might be augmented by\nstrategies that attack both the tumor cell as a source of VEGF as\nwell as the microenvironment endothelial cells recruited by the\nelaboration of VEGF.\nAnti-VEGF monotherapy efficacy has been shown in STS animal\nmodels (34). We have shown previously that combining anti-VEGF\ntherapy with chemotherapy results in superior STS inhibition (5).\nThe current report extends our previous studies to encompass the\nmore clinically significant issue of controlling metastasis, the most\ncommon lethal event in STS. Moreover, our study design uses a\npreclinical model of supranormal VEGF production that more\naccurately mirrors the human STS tumor-bearing continuum.\nA perceived concern with our model system might be that it\nlacks true relevance to the clinical situation in patients with STS.\nScientific and clinical oncology literature exists, which shows that\nVEGF expression is significantly elevated in human STS: in tumor\nspecimens at the RNA (35) and protein levels (10, 36, 37) as well as\nin patient serum samples (25, 26, 35, 38). Elevated serum VEGF\nlevels are reported to correlate with STS tumor grade (25, 36) and\nsize (25, 26), and immunohistochemical assessment of VEGF\nlevels in STS tumors correlates high-intensity VEGF staining with\nadvanced tumor grade (36, 37). These reports indicate clearly that\nVEGF is overexpressed in STS. In several series studying circulating\nVEGF levels in patients with STS, VEGF levels were found to be\nelevated in the vast majority of patients compared with healthy\ncontrols (25, 26, 35, 38). For example, Graeven et al. (25) reported\nmean serum VEGF levels to be 137 pg/mL (22-404 pg/mL; n = 18)\nfor healthy controls compared with 672 pg/mL (44-2,500 pg/mL;\nn = 62) in STS patients with macroscopic tumor lesions. In\ngenerating the model system used in our article, parental STS cell\nline VEGF secretion levels were compared with those of an\nimmortalized normal human fibroblast cell line. Two cell lines,\nwith low to moderate expression levels (by Western blot compa-\nrison among the cell lines; Fig. 1A), were stably transfected with\nVEGF to assess the effect of VEGF on STS biology with the\nminimum of genetic disruption. We reported previously quantitat-\ned VEGF levels secreted by at least one of these STS cell lines into\ntissue culture medium (12) using the same ELISA methodology\nused for patient sample analysis in the previously referred\npublished study. The tissue culture VEGF levels indicated that\nparental cell line VEGF secretion approximated to moderate serum\nlevels in patient samples. Tissue culture VEGF levels are,\nadmittedly, a debatable surrogate for equivalency to patient serum\nlevels; however, the shortcoming in the comparison is that the\nmeasured levels in the medium would be an overestimate\ncompared with in vivo-secreted VEGF levels. Taking this into\nconsideration, combined with our ELISA measurements falling into\nthe same range as moderate patient serum levels and Western blot\nresults showing secreted VEGF levels to be the same or less than\nthat released by immortalized fibroblast cell lines, we generated\nVEGF-overexpressing lines to more closely mimic the clinical STS\nscenario because elsewhere serum concentrations of VEGF have\nbeen reported as being 10 (35) and 12 (26) times higher than\nnormal in patients with STS. The need for a model that expresses\nabundant VEGF to mimic the clinical situation additionally stems\nfrom the reality that the high expression of VEGF observed in\nclinical STS is most likely secondary to the hypoxia generated in\nthese tumors, which frequently grow to very large dimensions\nbefore detection. A recently published report supports the need for\nsuch a model in STS by showing that STS cells in culture, when\nexposed to hypoxic conditions, significantly increase VEGF\nexpression compared with normoxic controls (39). These issues,\ntaken together with the previously mentioned reports, indicate that\nSTS overexpress VEGF and that our laboratory-generated model\nrepresents an acceptable investigative model of the actual clinical\nsituation.\nVulnerability of established tumor vasculature to therapeutic\nmanipulation of VEGF/VEGFR2 seems to occur selectively only in\nthose vessels that are immature and lack the contact or presence of\npericytes (40). Conversely, the presence of pericytes as an integral\ncomponent of mature tumor neovasculature may provide a basis\nfor surviving antiangiogenic treatments. These considerations may\nhelp explain the results of our experiment where the percentage of\ncells that stained positive for SMA, a marker of pericytes, was\nhighest in VEGF transfectants treated with the combined regimen.\nThis treatment, compared with single-agent or no treatment\ncontrols, was much more effective in eliminating SMA-negative\nimmature cells, leaving a much higher percentage of cells that were\nSMA positive. A similar finding has been reported in orthotopic\nmodels of human gastric (41) and pancreatic cancers (42, 43)\ntreated with VEGFR tyrosine kinase inhibitors. Although immature\nblood vessels rely on VEGF-induced survival signals (44), inhibition\nof both VEGF and platelet-derived growth factor signaling was\nfound to be superior for the elimination of pericytes (40). It has\nbeen suggested previously that eliminating immature and ‘‘leaky’’\nneovasculature while leaving intact the more mature vasculature\ncan provide a more effective conduit for chemotherapy to reach the\ntumor cells (45). However, apparently, combinations of treatments\nthat will also target endothelial support cells, such as pericytes,\nmight result in even better tumor control and thus merit further\ninvestigation.\nthan as stand-alone therapy. An additional\nTwo clinical trials have been conducted in STS using antiangio-\ngenic systemic treatments (46, 47). A phase II study examined the\neffect of SU5416, a tyrosine kinase VEGFR2 inhibitor, on 13 patients\nwith locally advanced or metastatic STS (46). Although no objective\ntumor responses were observed, the drug was generally well\ntolerated. Pretreatment and post-treatment tumor biopsies showed\nno significant alteration in VEGFR2 phosphorylation, suggesting\nthat therapeutic levels of the drug were not achieved. Furthermore,\nthe drug was used as monotherapy, and per our observations\nabove, VEGFR inhibition seems to work better as a chemosensitiz-\ntrial\ning agent\ncombined doxorubicin with bevacizumab, a human monoclonal\nantibody that binds VEGF (47). A 12% response rate was observed,\nwhich was not greater than the single-agent doxorubicin response\nrate. Although 65% of\nthe study patients enjoyed disease\nstabilization, this came at the expense of increased cardiomyop-\nathy. Apparently, the discrepancy between this clinical study and\nour own preclinical results has to do with the extent of disease at\ntime of therapy initiation. The patients in the clinical trial already\nhad extensive metastatic disease when treated. Established\nmetastases contain mature blood vessels against which the\ndescribed antiangiogenic therapy may be ineffective as we have\nshown above. In contrast, the animals in our study were treated\n(i.e., before the development of\nwith a preventive intent\nmetastases). A second difference is that the clinical trial used\nconventional dose of chemotherapy, resulting in major toxicity. In\nour studies, low-dose chemotherapy was effective as conventional\ndoses but with much less toxicity. Taken together, these results\nsuggest the possibility that combining anti-VEGF biotherapy with\nlow-dose doxorubicin might be most useful in stage III high VEGF-\nsecreting STS patients in whom there is a high risk of eventual\nmetastatic spread. Clinical trials pertaining to this question may be\na relevant next step.\nIn conclusion, combination chemotherapy/antiangiogensis\ntreatment algorithms targeting both endothelial and tumor cells\nshould be considered for stage III STS patients so that both\nprimary tumor and subclinical metastases can be addressed. Such\ncombinations, if successful, may offer new vistas of tumor control\nfor STS patients burdened by this devastating malignancy.",
         "Introduction\nPreviously, we have examined various molecular mechanisms of\nchemoresistance in STS (3–5). These efforts have primarily\nconsidered processes within the STS cell per se and have not\ndirectly addressed the reality that STS are not composed solely of\nmalignant cells but additionally include a diverse ensemble of host\ncells that are recruited into the tumor. It is likely that therapies\ntargeting both sarcoma cells and otherwise normal\ntumor-\nassociated cells may lead to improved STS treatment. The\nlikelihood of greater genetic stability in co-opted microenviron-\nment cells than in the STS tumor cells would presumably lead to\ntheir being less likely to acquire chemoresistance during the toxic\nstress selection of chemotherapy (6).\nEndothelial cells play a crucial role in tumor development and\nprogression. It is now well established that a tumor must initiate\nand sustain new blood vessels from preexisting vessels (angiogen-\nesis) and from the recruitment of circulating endothelial progenitor\ncells (vasculogenesis; refs. 7, 8) if it is to grow beyond a few\nmillimeters (9). This multistep process is regulated by the local\nbalance of endogenous proangiogenic and antiangiogenic factors,\nwhere vascular endothelial growth factor (VEGF) plays a critical\nrole. The biological relevance of VEGF in STS has been suggested\nby clinical studies showing impaired prognosis in patients who\nhave elevated VEGF levels (10). However, the relationship between\ntumor VEGF levels and either progression of human STS or the\ntherapeutic efficacy of chemotherapy remains elusive.\nIn light of these considerations, we studied the effect of VEGF\nexpression levels on (a) the progression of human STS and (b)\nchemotherapeutic efficacy in STS, by generation of VEGF165-\noverexpressing stable transfectants capable of forming metastasiz-\ning STS xenografts in severe combined immunodeficient (SCID)\nmice. We have been able to show the remarkable effect of VEGF\nexpression levels on both metastatic STS activity and doxorubicin\nchemoresistance, the latter process involving VEGF effects on\nendothelial cell sensitivity to doxorubicin rather than on STS tumor\ncells per se. An increased understanding of STS tumor cell:\nmicroenvironment cell interactions may be critical if improved and\nclinically relevant therapeutics are to be developed.\nresected on reaching 1.5 cm in diameter, weighed and frozen at 140jC, or\nfixed in a 10% formalin solution and paraffin embedded. Following tumor\nresection, mice were kept alive for up to 13 weeks to allow lung metastasis\ndevelopment. Metastases were enumerated after India ink instillation (15);\nonly nodules that exceeded 0.5 mm in diameter were counted. Animals",
         "lines and stable transfectant generation. Human SKLMS-1\nleiomyosarcoma, RD rhabdomyosarcoma, and the human umbilical\nvascular endothelial cell (HUVEC) lines were purchased from the American\nType Culture Collection (ATCC; Rockville, MD) and cultured according to\nATCC recommendations. The full-length human VEGF165 cDNA was cloned\ninto pcDNA3 expression vector with transcription driven by a cytomega-\nlovirus enhancer-promoter (kindly provided by Dr. Lee M. Ellis, M.D.\nAnderson Cancer Center, Houston, TX) and also contained the neomycin\nresistance gene. Subconfluent SKLMS-1 or RD cells were transfected;\nindividual G418-resistant colonies were cloned and expanded as described\npreviously (11). Sarcoma cell conditioned medium was prepared and stored\nas described previously (12).\nNorthern blot and Western blot analyses. Northern and Western blot\nanalyses were done as described previously (13). Western blot primary\nantibodies included monoclonal mouse anti-human-VEGF (R&D Systems,\nMinneapolis, MN) and goat polyclonal anti-human angiopoietin-2 (Ang-2;\nSanta Cruz Biotechnology, Santa Cruz, CA). To detect Ang-2 expression in\nHUVECs, cells were treated with VEGF (10 ng/mL) and CoCl2 (50 Amol/L)\nfor 24 hours.\nProliferation, chemosensitivity, migration, and chemoinvasion\nassays. Endothelial cell growth assays were done as described previously\n(5). For in vitro treatment experiments, HUVECs were treated with control\nrat IgG (10 Ag/mL), doxorubicin (0.0625 Amol/L), DC101 (10 Ag/mL), or\nDC101 (10 Ag/mL) plus doxorubicin (0.0625 Amol/L)\nfor 1 hour.\nConditioned media from VEGF165 transfectants and controls were added\nto the HUVECs; migratory and proliferative abilities were determined.\nTumor cell growth assays were done using 96-well plates (2,000 cells per\nwell). Cell growth was quantitated using crystal violet staining (5). Growth\nrate was analyzed 24, 48, and 72 hours after treatment with doxorubicin\n(doses range, 0-500 Amol/L), DC101 (10 Ag/mL), or DC101 (10 Ag/mL) plus\ndoxorubicin (0.1 Amol/L). Tumor cell invasion was assessed as described\npreviously (14).\ncDNA microarray. Total RNA was extracted from SKLMS-1, SKLneo,\nand SKLSE-10 cells using Trizol reagent (Life Technologies, Gaithersburg,\nMD). RNA quality was checked and then used to synthesize cDNA as a\ntemplate for generating biotin-16-UTP (nonradioactive)–labeled cRNA\ntarget using TrueLabelingAMP linear RNA amplification kit (SuperArray\nBioscience Corp., Frederick, MD). Oligo GEArray Human Toxicology and\nDrug Resistance Microarray (SuperArray Bioscience) was used to charac-\nterize expression of 263 genes. Prehybridization (2 hours) and hybridization\n(overnight) was done at 60jC in a hybridization oven using 6-Ag labeled\ncRNA target. SDS membranes (0.5%) underwent chemiluminescent detec-\ntion. Array data were analyzed using the GEArray Expression Analysis Suite.\nZymography. HUVECs were incubated with no therapy, IgG, DC101,\nand/or doxorubicin in VEGF-enriched serum-free medium for 48 hours.\nCulture supernatants were collected, and equimolar amounts of concen-\ntrated samples were subjected to 10% SDS-PAGE gel. After electrophoresis\nand 16 to 24 hours of incubation, the gels were stained with Coomassie\nbrilliant blue and destained. Equimolar loading was verified by comparisons\nof staining patterns on the gel.\nAnimal studies. Female SCID mice were obtained from Taconic Farms\n(Germantown, NY). Animals received humane care as per the Animal Welfare\nAct and the NIH ‘‘Guide for the Care and Use of Laboratory Animals.’’\nExperiments were carried out under a protocol approved by the Institutional\nAnimal Care and Use Committee at The University of Texas M.D. Anderson\nCancer Center (Houston, TX). Trypan blue staining–confirmed viable\nsarcoma cells, 1  106 in 0.1 mL HBSS, were injected s.c. into the flank of\n5- to 6-week-old SCID mice. Tumors were measured twice weekly and\nbearing sarcoma xenografts (eight mice per group) began receiving treat-\nment when the tumor diameters reached 5 mm. DC101 (ImClone Systems,\nInc., New York, NY), an anti-VEGF receptor (VEGFR) 2 rat IgG1 mono-\nclonal antibody (16), was given i.p. at a dose of 400 Ag every 3 days for\na total of seven times. Doxorubicin (6 mg/kg; American Pharmaceutical\nPartners, Los Angeles, CA; conventional schedule) was given twice every\n2 weeks, and doxorubicin (1.2 mg/kg; continuous low-dose schedule) was\ngiven every 3 days for a total of eight times. Combined treatment (400 Ag\nDC101 plus 1.2 mg/kg doxorubicin) was given once every 3 days for a total\nof eight times. Control animals bearing s.c. flank tumors were treated with\nnormal rat IgG (400 Ag) or saline following the same schedules.\nHistologic and immunohistochemical analysis. Conventional H&E\nstaining was done and examined by light microscopy. Immunohistochem-\nistry was done using anti-mouse CD31 (1:50; PharMingen, San Diego, CA),\nanti-mouse a-smooth muscle actin (a-SMA; 1:80; Neomarkers, Fremont,\nCA), anti-mouse Ang-2 (1:50), and anti-Ki67 (1:500; DAKO Carpinteria,\nCA). For microvessel counts, frozen tissues were sectioned and stained\nwith anti-CD31 antibody and then counted at 200 magnification. For\npericyte analysis, frozen sections were stained for a-SMA. Blood vessels\nassociated with a-SMA-positive cells were considered mature (17).\nSections were stained with a-SMA and with anti-CD31, which stains\nboth mature and immature vessels. The CD31+ and a-SMA-positive versus\nCD31+ vessels were scored as percent SMA-positive vessels (vessel\nmaturation index).",
         "Results\nSKLMS-1 and RD cell lines expressing low and medium levels of\nendogenous VEGF, respectively (Fig. 1A), were stably transfected\nwith VEGF165 and resultant cell lines were designated SKLSE and\nRDSE. Transfection with the pcDNA3 plasmid containing a\nneomycin-resistant selection marker gene alone resulted in SKLneo\nand RDneo used as controls.\nNorthern blots of VEGF165 mRNA expression (Fig. 1B) and\nWestern blots for VEGF165 protein expression (Fig. 1C) showed\nincreased VEGF165 mRNA and protein levels in the selected\ntransfectants.\nConditioned medium from the parental cells, vector control\ncells, and the VEGF165-overexpressing transfectants was compared\nfor ability to stimulate HUVEC proliferation (Fig. 1D). HUVEC\nproliferation stimulated by the conditioned medium of VEGF165-\noverexpressing cells was significantly higher, showing that the\noverexpressed VEGF165 from the transfectants was biologically\nactive.\nXenografts from VEGF165-transfected cell\nlines SKLSE10,\nSKLSE13, SKLSE23, and RDSE14, RDSE17, and RDSE26 had average\nlatency periods of 14, 16, 17, days, and 17, 15, and 18 days,\nrespectively. In contrast, xenografts of control and parental cell\nlines SKLneo, SKLMS-1, RDneo, and RD had much longer average\nlatency periods of 26, 27, 29, and 31 days. Tumors from VEGF165-\noverexpressing SKLSE and RDSE cells grew faster than those from\nvector control or parental cells (P < 0.001; Fig. 2A) as can also be\nseen visually (Fig. 2B). During tumor removal, we noted that\nVEGF165-overexpressing xenografts adhered more tightly to adja-\ncent normal tissues. H&E staining of primary tumor margins\nrevealed that parental and control cell\nline xenografts abutted\nnormal tissues, whereas VEGF165-overexpressing tumors directly\ninvaded adjacent fat and muscle tissue (Fig. 2C).\nBecause high levels of VEGF are prognostic for chemoresistance\nin patients bearing certain malignancies (18), we asked if high\nVEGF165 expression levels could negatively affect doxorubicin\nefficacy in the human STS xenograft model. Doxorubicin inhibited\nthe xenograft growth of controls much more than the growth of\nSKLSE10 (Fig. 2D, left) and RDSE17 (Fig. 2D, right); response rates\nfor SKLSE10 and RDSE17 tumors were 58.4% and 66.3% of those for\nSKLneo and RDneo, respectively.\nAnimal body weights were measured at regular intervals to\nevaluate chemotoxicity (19). Significant weight loss was observed in\nall STS xenograft–bearing mice treated with conventional schedule\ndoxorubicin (data not shown).\nAs shown in Table 1, pulmonary metastases were not detected in\nany of the mice bearing parental or control cell xenografts. In\ncontrast, significant metastatic activity was detected in mice\nbearing VEGF165-overexpressing xenografts, suggesting that in-\ncreased VEGF165 expression in SKLSE and RDSE cells enhanced\ntheir metastatic potential in vivo.\nNext, we investigated whether the accelerated tumor growth and\nmetastasis could be due to tumor angiogenesis stimulated by\nVEGF165. Xenografts from VEGF165-overexpressing cell lines had\nmuch higher microvessel counts (P < 0.001; Fig. 3A) and\nmicrovessel density (data not shown) than did control or parental\ncell line xenografts. Figure 3B depicts representative examples of\nxenograft microvessel appearance in tumors from VEGF165-over-\nexpressing SKLSE10 cells and control SKLneo cells, showing that\nenhanced VEGF165 expression increased intratumoral angiogenesis\nof human STS.\nAng-2, a potent proangiogenic switch factor, can be induced by\ntumor-derived VEGF signaling through endothelial cell VEGFR2,\nresulting in destabilized host vasculature (20, 21). Consistent with\nthese reports, we treated HUVECs with VEGF165, showing\ninduction of Ang-2 expression by these cells (Fig. 3C). Immuno-\nhistochemistry studies showed that Ang-2 expression was much\ngreater in the intratumoral endothelium of VEGF165-overexpressing\nindicating that\ntumors than in the controls (Fig. 3D, bottom),\nVEGF165 overexpression in sarcoma xenografts enhanced the\nproangiogenic state in tumor vasculature.\nAssays were done to determine whether the enhanced in vivo\ngrowth of SKLSE and RDSE cell xenografts resulted from\nalterations in their in vitro properties. The growth and invasion\nrates of VEGF165-transfected cells and control cells were similar\n(data not shown),\nindicating that the increased growth and\nmetastasis of SKLSE and RDSE cells was not a result of an\nalteration in the intrinsic properties of these cells. Finally, we\ncompared the expression of a panel of known chemoresistance\ngenes in the parental and transfected SKLMS-1 cells using a cDNA\narray. No difference in gene expression was detected (data not\nshown), suggesting that VEGF165 does not act in an autocrine-like\nmanner to affect the tumor cell chemoresistant effectors directly.\nProliferation, migration, and chemosensitivity of HUVECs were\nassessed after exposure to conditioned medium derived from STS\nparental cells, vector control cells, and VEGF165-overexpressing\ntransfectants. The proliferation (Fig. 1D) of HUVECs stimulated by\nVEGF165-overexpressing cell conditioned medium was significantly\nhigher than that stimulated by conditioned medium of controls.\nSimilarly, HUVEC migration was significantly enhanced by\nVEGF165-overexpressing cell conditioned medium (Fig. 4A). Inter-\nestingly, HUVECs exhibited much greater resistance to doxorubicin\ncytotoxic effects after exposure to VEGF transfectant conditioned\nmedium (Fig. 4B). To the best of our knowledge, the effect of VEGF\non endothelial cell chemoresistance has not been described\npreviously; our data show that the increased chemoresistance of\nVEGF165 STS xenografts is due to the induced chemoresistance of\nthe tumor-associated endothelial cells rather than the tumor cells\nper se.\nlow-dose chemotherapy is an\nattractive therapeutic strategy to minimize chemotherapy-related\ntoxicity (22). Because signs of conventional-dose doxorubicin\ntoxicity developed in all mice treated, low-dose doxorubicin was\ncombined with DC101 as a chemosensitizer to treat mice bearing\nVEGF165-overexpressing sarcoma xenografts. Whereas treatment\nwith either continuous low-dose doxorubicin or DC101 alone did\nnot significantly affect growth of SKLSE10 and RDSE17 xenografts\n(Fig. 5A, left and right), combined continuous low-dose doxorubicin\nand DC101 was markedly inhibitory. In addition, although animals\nreceiving low-dose doxorubicin experienced some weight loss, the\nloss was significantly less than that of the conventional-schedule\ngroup. Moreover, combined therapy caused no additional host\nanimal toxicity above that seen in the low-dose doxorubicin alone\ngroup (data not shown). Taken together, these data suggest that\nDC101 blockade of VEGFR2 signaling can reverse the endothelial\ncell chemoresistance caused by human STS xenograft over-\nexpression of VEGF165.\nMicrovessel counts were significantly lower in tumors treated\nwith DC101 plus doxorubicin than in those treated with either\nagent alone (Fig. 5B). The percentage of SMA-positive vessels was\nsignificantly greater in VEGF165-overexpressing xenografts from\nanimals treated with the DC101 and low-dose doxorubicin\ncompared with either drug alone or control treatments (Fig. 5C),\nsuggesting that antiangiogenic therapy increased the percentage of\nmature vessels in VEGF165-overexpressing xenografts.\nTo determine whether combined therapy more effectively\ninhibited xenograft proliferation in vivo, xenograft sections were\nstained for Ki67, a nuclear marker of proliferation. As shown in\nFig. 5D, combination therapy suppressed proliferation in the\nVEGF165-overexpressing tumors much more than did either\nmonotherapy or mock therapy, suggesting that the decreased\nvascularity leads to decreased tumor cell proliferation.\nBecause inhibition of angiogenesis has been shown before to\nsuppress breast cancer metastasis (23), we were interested to\nexamine whether antiangiogenic treatment with DC101 and low-\ndose doxorubicin could inhibit lung metastasis of STS. Lungs of\nmice from the various treatment groups were examined for\nsarcoma metastases. Metastatic burden was significantly reduced\nin mice receiving the combined regimen than in those treated by\neither agent alone (Table 2).\nTo further elucidate the mechanisms by which the combined\ntherapy was superior in inhibiting tumor growth and metastasis,\nin vitro endothelial cell migration and proliferation assays were\ndone. Although single-agent doxorubicin or DC101 inhibited the\nmigration and proliferation of HUVECs exposed to control cell\nconditioned medium, doxorubicin or DC101 alone did not affect\nthese activities in HUVECs exposed to high VEGF165-containing\nconditioned medium from SKLSE10 and RDSE17 cells (Fig. 6A\nand B). In addition, combined therapy inhibited endothelial cells\nmore than single agent alone or mock treatments.\nEndothelial cell matrix metalloproteinase (MMP) secretion has\nbeen associated with tumor metastasis (24). Interestingly, we found\nthat endothelial cell MMP activity was markedly inhibited in the\nconditioned medium of HUVECs treated with combined DC101\nand doxorubicin but not in the conditioned medium treated with\neither agent alone (Fig. 6C), implying that DC101 and doxorubicin\nin combination may help decrease STS pulmonary metastasis via\nthe inhibition of MMP secretion by tumor-associated endothelial\ncells.",
         "https://aacrjournals.org/cancerres/article-pdf/66/17/8770/2554504/8770.pdf",
         "To better elucidate the role of vascular endothelial growth\nfactor (VEGF)165 in soft\ntissue sarcoma (STS) growth,\nmetastasis, and chemoresistance, we generated stably trans-\nfected human STS cell lines with VEGF165 to study the effect\nof VEGF165 on STS cells in vitro and the effect of culture\nmedium from these cells on human umbilical vascular\nendothelial cells. Severe combined immunodeficient mice\nbearing xenografts of transfected cell\nlines were used to\nassess the effect of VEGF overexpression and the effect of\nVEGF receptor (VEGFR) 2 inhibition on STS growth,\nmetastasis, and response to doxorubicin. VEGF165-transfected\nxenografts formed highly vascular tumors with shorter\nlatency, accelerated growth, enhanced chemoresistance, and\nincreased incidence of pulmonary metastases. Blockade of\nVEGFR2 signaling using DC101 anti-VEGFR2 monoclonal\nantibody enhanced doxorubicin chemoresponse; this com-\nbined biochemotherapy inhibited tumor growth and de-\ncreased pulmonary metastases without overt\ntoxicity.\nCombined therapy reduced microvessel counts while increas-\ning vessel maturation index. VEGF overexpression did not\naffect on the sarcoma cells per se; however, conditioned\nmedium from VEGF transfectants caused increased endothe-\nlial cell proliferation, migration, and chemoresistance.\nAddition of DC101 induced endothelial cell sensitivity to\ndoxorubicin and suppressed the activity of matrix metal-\nloproteinases secreted by endothelial cells. We therefore\nconclude that VEGF is a critical determinant of STS growth\nand metastasis and that STS chemoresistance, in our model,\nis a process induced by the interplay between STS cells and\ntumor-associated endothelial cells. STS growth and metasta-\nsis can be interrupted by combined low-dose doxorubicin\nand anti-VEGFR2, a strategy that attacks STS-associated\nendothelial cells. In the future, such therapeutic approaches\nmay be useful in treating STS before the development of\nclinically apparent metastases. (Cancer Res 2006; 66(17): 8770-8)\nIntroduction\nFive-year survival for soft tissue sarcoma (STS) patients is f50%\ndepending on tumor size, grade, location, the presence of regional\nor distant disease, and response to therapy (1). Metastases, mainly\nto the lung, are particularly ominous. The need to successfully\ncontrol STS dissemination by systemic approaches that can reach\ntotal body disease loci remains compelling. However, the use of\nchemotherapy in STS is problematic due to toxicity, expense, and\nmarked chemoresistance. Even doxorubicin, the single most active\nagent in STS, has a disappointing overall 30% response rate, with\nbreakthrough tumor progression and frequent recurrence after\ninitial sarcoma chemoresponsiveness (2). Although progression,\nmetastasis, and chemoresistance in STS have been characterized\nclinically,\nthe molecular mechanisms regulating these tumor\nprocesses are less well understood.\nPreviously, we have examined various molecular mechanisms of\nchemoresistance in STS (3–5). These efforts have primarily\nconsidered processes within the STS cell per se and have not\ndirectly addressed the reality that STS are not composed solely of\nmalignant cells but additionally include a diverse ensemble of host\ncells that are recruited into the tumor. It is likely that therapies\ntargeting both sarcoma cells and otherwise normal\ntumor-\nassociated cells may lead to improved STS treatment. The\nlikelihood of greater genetic stability in co-opted microenviron-\nment cells than in the STS tumor cells would presumably lead to\ntheir being less likely to acquire chemoresistance during the toxic\nstress selection of chemotherapy (6).\nEndothelial cells play a crucial role in tumor development and\nprogression. It is now well established that a tumor must initiate\nand sustain new blood vessels from preexisting vessels (angiogen-\nesis) and from the recruitment of circulating endothelial progenitor\ncells (vasculogenesis; refs. 7, 8) if it is to grow beyond a few\nmillimeters (9). This multistep process is regulated by the local\nbalance of endogenous proangiogenic and antiangiogenic factors,\nwhere vascular endothelial growth factor (VEGF) plays a critical\nrole. The biological relevance of VEGF in STS has been suggested\nby clinical studies showing impaired prognosis in patients who\nhave elevated VEGF levels (10). However, the relationship between\ntumor VEGF levels and either progression of human STS or the\ntherapeutic efficacy of chemotherapy remains elusive.\nIn light of these considerations, we studied the effect of VEGF\nexpression levels on (a) the progression of human STS and (b)\nchemotherapeutic efficacy in STS, by generation of VEGF165-\noverexpressing stable transfectants capable of forming metastasiz-\ning STS xenografts in severe combined immunodeficient (SCID)\nmice. We have been able to show the remarkable effect of VEGF\nexpression levels on both metastatic STS activity and doxorubicin\nchemoresistance, the latter process involving VEGF effects on\nendothelial cell sensitivity to doxorubicin rather than on STS tumor\ncells per se. An increased understanding of STS tumor cell:\nmicroenvironment cell interactions may be critical if improved and\nclinically relevant therapeutics are to be developed.\nresected on reaching 1.5 cm in diameter, weighed and frozen at 140jC, or\nfixed in a 10% formalin solution and paraffin embedded. Following tumor\nresection, mice were kept alive for up to 13 weeks to allow lung metastasis\ndevelopment. Metastases were enumerated after India ink instillation (15);\nonly nodules that exceeded 0.5 mm in diameter were counted. Animals\nlines and stable transfectant generation. Human SKLMS-1\nleiomyosarcoma, RD rhabdomyosarcoma, and the human umbilical\nvascular endothelial cell (HUVEC) lines were purchased from the American\nType Culture Collection (ATCC; Rockville, MD) and cultured according to\nATCC recommendations. The full-length human VEGF165 cDNA was cloned\ninto pcDNA3 expression vector with transcription driven by a cytomega-\nlovirus enhancer-promoter (kindly provided by Dr. Lee M. Ellis, M.D.\nAnderson Cancer Center, Houston, TX) and also contained the neomycin\nresistance gene. Subconfluent SKLMS-1 or RD cells were transfected;\nindividual G418-resistant colonies were cloned and expanded as described\npreviously (11). Sarcoma cell conditioned medium was prepared and stored\nas described previously (12).\nNorthern blot and Western blot analyses. Northern and Western blot\nanalyses were done as described previously (13). Western blot primary\nantibodies included monoclonal mouse anti-human-VEGF (R&D Systems,\nMinneapolis, MN) and goat polyclonal anti-human angiopoietin-2 (Ang-2;\nSanta Cruz Biotechnology, Santa Cruz, CA). To detect Ang-2 expression in\nHUVECs, cells were treated with VEGF (10 ng/mL) and CoCl2 (50 Amol/L)\nfor 24 hours.\nProliferation, chemosensitivity, migration, and chemoinvasion\nassays. Endothelial cell growth assays were done as described previously\n(5). For in vitro treatment experiments, HUVECs were treated with control\nrat IgG (10 Ag/mL), doxorubicin (0.0625 Amol/L), DC101 (10 Ag/mL), or\nDC101 (10 Ag/mL) plus doxorubicin (0.0625 Amol/L)\nfor 1 hour.\nConditioned media from VEGF165 transfectants and controls were added\nto the HUVECs; migratory and proliferative abilities were determined.\nTumor cell growth assays were done using 96-well plates (2,000 cells per\nwell). Cell growth was quantitated using crystal violet staining (5). Growth\nrate was analyzed 24, 48, and 72 hours after treatment with doxorubicin\n(doses range, 0-500 Amol/L), DC101 (10 Ag/mL), or DC101 (10 Ag/mL) plus\ndoxorubicin (0.1 Amol/L). Tumor cell invasion was assessed as described\npreviously (14).\ncDNA microarray. Total RNA was extracted from SKLMS-1, SKLneo,\nand SKLSE-10 cells using Trizol reagent (Life Technologies, Gaithersburg,\nMD). RNA quality was checked and then used to synthesize cDNA as a\ntemplate for generating biotin-16-UTP (nonradioactive)–labeled cRNA\ntarget using TrueLabelingAMP linear RNA amplification kit (SuperArray\nBioscience Corp., Frederick, MD). Oligo GEArray Human Toxicology and\nDrug Resistance Microarray (SuperArray Bioscience) was used to charac-\nterize expression of 263 genes. Prehybridization (2 hours) and hybridization\n(overnight) was done at 60jC in a hybridization oven using 6-Ag labeled\ncRNA target. SDS membranes (0.5%) underwent chemiluminescent detec-\ntion. Array data were analyzed using the GEArray Expression Analysis Suite.\nZymography. HUVECs were incubated with no therapy, IgG, DC101,\nand/or doxorubicin in VEGF-enriched serum-free medium for 48 hours.\nCulture supernatants were collected, and equimolar amounts of concen-\ntrated samples were subjected to 10% SDS-PAGE gel. After electrophoresis\nand 16 to 24 hours of incubation, the gels were stained with Coomassie\nbrilliant blue and destained. Equimolar loading was verified by comparisons\nof staining patterns on the gel.\nAnimal studies. Female SCID mice were obtained from Taconic Farms\n(Germantown, NY). Animals received humane care as per the Animal Welfare\nAct and the NIH ‘‘Guide for the Care and Use of Laboratory Animals.’’\nExperiments were carried out under a protocol approved by the Institutional\nAnimal Care and Use Committee at The University of Texas M.D. Anderson\nCancer Center (Houston, TX). Trypan blue staining–confirmed viable\nsarcoma cells, 1  106 in 0.1 mL HBSS, were injected s.c. into the flank of\n5- to 6-week-old SCID mice. Tumors were measured twice weekly and\nbearing sarcoma xenografts (eight mice per group) began receiving treat-\nment when the tumor diameters reached 5 mm. DC101 (ImClone Systems,\nInc., New York, NY), an anti-VEGF receptor (VEGFR) 2 rat IgG1 mono-\nclonal antibody (16), was given i.p. at a dose of 400 Ag every 3 days for\na total of seven times. Doxorubicin (6 mg/kg; American Pharmaceutical\nPartners, Los Angeles, CA; conventional schedule) was given twice every\n2 weeks, and doxorubicin (1.2 mg/kg; continuous low-dose schedule) was\ngiven every 3 days for a total of eight times. Combined treatment (400 Ag\nDC101 plus 1.2 mg/kg doxorubicin) was given once every 3 days for a total\nof eight times. Control animals bearing s.c. flank tumors were treated with\nnormal rat IgG (400 Ag) or saline following the same schedules.\nHistologic and immunohistochemical analysis. Conventional H&E\nstaining was done and examined by light microscopy. Immunohistochem-\nistry was done using anti-mouse CD31 (1:50; PharMingen, San Diego, CA),\nanti-mouse a-smooth muscle actin (a-SMA; 1:80; Neomarkers, Fremont,\nCA), anti-mouse Ang-2 (1:50), and anti-Ki67 (1:500; DAKO Carpinteria,\nCA). For microvessel counts, frozen tissues were sectioned and stained\nwith anti-CD31 antibody and then counted at 200 magnification. For\npericyte analysis, frozen sections were stained for a-SMA. Blood vessels\nassociated with a-SMA-positive cells were considered mature (17).\nSections were stained with a-SMA and with anti-CD31, which stains\nboth mature and immature vessels. The CD31+ and a-SMA-positive versus\nCD31+ vessels were scored as percent SMA-positive vessels (vessel\nmaturation index).\nResults\nSKLMS-1 and RD cell lines expressing low and medium levels of\nendogenous VEGF, respectively (Fig. 1A), were stably transfected\nwith VEGF165 and resultant cell lines were designated SKLSE and\nRDSE. Transfection with the pcDNA3 plasmid containing a\nneomycin-resistant selection marker gene alone resulted in SKLneo\nand RDneo used as controls.\nNorthern blots of VEGF165 mRNA expression (Fig. 1B) and\nWestern blots for VEGF165 protein expression (Fig. 1C) showed\nincreased VEGF165 mRNA and protein levels in the selected\ntransfectants.\nConditioned medium from the parental cells, vector control\ncells, and the VEGF165-overexpressing transfectants was compared\nfor ability to stimulate HUVEC proliferation (Fig. 1D). HUVEC\nproliferation stimulated by the conditioned medium of VEGF165-\noverexpressing cells was significantly higher, showing that the\noverexpressed VEGF165 from the transfectants was biologically\nactive.\nXenografts from VEGF165-transfected cell\nlines SKLSE10,\nSKLSE13, SKLSE23, and RDSE14, RDSE17, and RDSE26 had average\nlatency periods of 14, 16, 17, days, and 17, 15, and 18 days,\nrespectively. In contrast, xenografts of control and parental cell\nlines SKLneo, SKLMS-1, RDneo, and RD had much longer average\nlatency periods of 26, 27, 29, and 31 days. Tumors from VEGF165-\noverexpressing SKLSE and RDSE cells grew faster than those from\nvector control or parental cells (P < 0.001; Fig. 2A) as can also be\nseen visually (Fig. 2B). During tumor removal, we noted that\nVEGF165-overexpressing xenografts adhered more tightly to adja-\ncent normal tissues. H&E staining of primary tumor margins\nrevealed that parental and control cell\nline xenografts abutted\nnormal tissues, whereas VEGF165-overexpressing tumors directly\ninvaded adjacent fat and muscle tissue (Fig. 2C).\nBecause high levels of VEGF are prognostic for chemoresistance\nin patients bearing certain malignancies (18), we asked if high\nVEGF165 expression levels could negatively affect doxorubicin\nefficacy in the human STS xenograft model. Doxorubicin inhibited\nthe xenograft growth of controls much more than the growth of\nSKLSE10 (Fig. 2D, left) and RDSE17 (Fig. 2D, right); response rates\nfor SKLSE10 and RDSE17 tumors were 58.4% and 66.3% of those for\nSKLneo and RDneo, respectively.\nAnimal body weights were measured at regular intervals to\nevaluate chemotoxicity (19). Significant weight loss was observed in\nall STS xenograft–bearing mice treated with conventional schedule\ndoxorubicin (data not shown).\nAs shown in Table 1, pulmonary metastases were not detected in\nany of the mice bearing parental or control cell xenografts. In\ncontrast, significant metastatic activity was detected in mice\nbearing VEGF165-overexpressing xenografts, suggesting that in-\ncreased VEGF165 expression in SKLSE and RDSE cells enhanced\ntheir metastatic potential in vivo.\nNext, we investigated whether the accelerated tumor growth and\nmetastasis could be due to tumor angiogenesis stimulated by\nVEGF165. Xenografts from VEGF165-overexpressing cell lines had\nmuch higher microvessel counts (P < 0.001; Fig. 3A) and\nmicrovessel density (data not shown) than did control or parental\ncell line xenografts. Figure 3B depicts representative examples of\nxenograft microvessel appearance in tumors from VEGF165-over-\nexpressing SKLSE10 cells and control SKLneo cells, showing that\nenhanced VEGF165 expression increased intratumoral angiogenesis\nof human STS.\nAng-2, a potent proangiogenic switch factor, can be induced by\ntumor-derived VEGF signaling through endothelial cell VEGFR2,\nresulting in destabilized host vasculature (20, 21). Consistent with\nthese reports, we treated HUVECs with VEGF165, showing\ninduction of Ang-2 expression by these cells (Fig. 3C). Immuno-\nhistochemistry studies showed that Ang-2 expression was much\ngreater in the intratumoral endothelium of VEGF165-overexpressing\nindicating that\ntumors than in the controls (Fig. 3D, bottom),\nVEGF165 overexpression in sarcoma xenografts enhanced the\nproangiogenic state in tumor vasculature.\nAssays were done to determine whether the enhanced in vivo\ngrowth of SKLSE and RDSE cell xenografts resulted from\nalterations in their in vitro properties. The growth and invasion\nrates of VEGF165-transfected cells and control cells were similar\n(data not shown),\nindicating that the increased growth and\nmetastasis of SKLSE and RDSE cells was not a result of an\nalteration in the intrinsic properties of these cells. Finally, we\ncompared the expression of a panel of known chemoresistance\ngenes in the parental and transfected SKLMS-1 cells using a cDNA\narray. No difference in gene expression was detected (data not\nshown), suggesting that VEGF165 does not act in an autocrine-like\nmanner to affect the tumor cell chemoresistant effectors directly.\nProliferation, migration, and chemosensitivity of HUVECs were\nassessed after exposure to conditioned medium derived from STS\nparental cells, vector control cells, and VEGF165-overexpressing\ntransfectants. The proliferation (Fig. 1D) of HUVECs stimulated by\nVEGF165-overexpressing cell conditioned medium was significantly\nhigher than that stimulated by conditioned medium of controls.\nSimilarly, HUVEC migration was significantly enhanced by\nVEGF165-overexpressing cell conditioned medium (Fig. 4A). Inter-\nestingly, HUVECs exhibited much greater resistance to doxorubicin\ncytotoxic effects after exposure to VEGF transfectant conditioned\nmedium (Fig. 4B). To the best of our knowledge, the effect of VEGF\non endothelial cell chemoresistance has not been described\npreviously; our data show that the increased chemoresistance of\nVEGF165 STS xenografts is due to the induced chemoresistance of\nthe tumor-associated endothelial cells rather than the tumor cells\nper se.\nlow-dose chemotherapy is an\nattractive therapeutic strategy to minimize chemotherapy-related\ntoxicity (22). Because signs of conventional-dose doxorubicin\ntoxicity developed in all mice treated, low-dose doxorubicin was\ncombined with DC101 as a chemosensitizer to treat mice bearing\nVEGF165-overexpressing sarcoma xenografts. Whereas treatment\nwith either continuous low-dose doxorubicin or DC101 alone did\nnot significantly affect growth of SKLSE10 and RDSE17 xenografts\n(Fig. 5A, left and right), combined continuous low-dose doxorubicin\nand DC101 was markedly inhibitory. In addition, although animals\nreceiving low-dose doxorubicin experienced some weight loss, the\nloss was significantly less than that of the conventional-schedule\ngroup. Moreover, combined therapy caused no additional host\nanimal toxicity above that seen in the low-dose doxorubicin alone\ngroup (data not shown). Taken together, these data suggest that\nDC101 blockade of VEGFR2 signaling can reverse the endothelial\ncell chemoresistance caused by human STS xenograft over-\nexpression of VEGF165.\nMicrovessel counts were significantly lower in tumors treated\nwith DC101 plus doxorubicin than in those treated with either\nagent alone (Fig. 5B). The percentage of SMA-positive vessels was\nsignificantly greater in VEGF165-overexpressing xenografts from\nanimals treated with the DC101 and low-dose doxorubicin\ncompared with either drug alone or control treatments (Fig. 5C),\nsuggesting that antiangiogenic therapy increased the percentage of\nmature vessels in VEGF165-overexpressing xenografts.\nTo determine whether combined therapy more effectively\ninhibited xenograft proliferation in vivo, xenograft sections were\nstained for Ki67, a nuclear marker of proliferation. As shown in\nFig. 5D, combination therapy suppressed proliferation in the\nVEGF165-overexpressing tumors much more than did either\nmonotherapy or mock therapy, suggesting that the decreased\nvascularity leads to decreased tumor cell proliferation.\nBecause inhibition of angiogenesis has been shown before to\nsuppress breast cancer metastasis (23), we were interested to\nexamine whether antiangiogenic treatment with DC101 and low-\ndose doxorubicin could inhibit lung metastasis of STS. Lungs of\nmice from the various treatment groups were examined for\nsarcoma metastases. Metastatic burden was significantly reduced\nin mice receiving the combined regimen than in those treated by\neither agent alone (Table 2).\nTo further elucidate the mechanisms by which the combined\ntherapy was superior in inhibiting tumor growth and metastasis,\nin vitro endothelial cell migration and proliferation assays were\ndone. Although single-agent doxorubicin or DC101 inhibited the\nmigration and proliferation of HUVECs exposed to control cell\nconditioned medium, doxorubicin or DC101 alone did not affect\nthese activities in HUVECs exposed to high VEGF165-containing\nconditioned medium from SKLSE10 and RDSE17 cells (Fig. 6A\nand B). In addition, combined therapy inhibited endothelial cells\nmore than single agent alone or mock treatments.\nEndothelial cell matrix metalloproteinase (MMP) secretion has\nbeen associated with tumor metastasis (24). Interestingly, we found\nthat endothelial cell MMP activity was markedly inhibited in the\nconditioned medium of HUVECs treated with combined DC101\nand doxorubicin but not in the conditioned medium treated with\neither agent alone (Fig. 6C), implying that DC101 and doxorubicin\nin combination may help decrease STS pulmonary metastasis via\nthe inhibition of MMP secretion by tumor-associated endothelial\ncells.\nDiscussion\nIn this study, we examined the effects of VEGF165 overexpression\non human STS progression and chemotherapeutic efficacy. S.C.\nimplantation of VEGF165-overexpressing sarcoma cell lines in SCID\nmice produced highly vascular tumors that exhibited a much\nshorter latency, significantly accelerated tumor growth, angiogen-\nesis, and metastasis. However, no effect on sarcoma cell growth\nand invasion could be seen in vitro. We learned that the inhibitory\neffects of doxorubicin alone on tumor growth were minimal in\nVEGF165-overexpressing xenografts, whereas treatment with the\nanti-VEGFR2 monoclonal antibody combined with doxorubicin\nmarkedly improved xenograft responsiveness to doxorubicin.\nThe significance of VEGF expression in STS patients has been\nrecognized previously in patient-based descriptive studies. In a\nstudy of 85 STS patients, serum VEGF levels correlated significantly\nwith grade (25). Others have observed that the magnitude of\nSTS patient circulating or tumor VEGF levels is an important\nprognostic factor (10, 26). Mutation of p53 is very frequently found\nin STS and this may also play a role in up-regulating VEGF as we\nhave shown before (12). To the best of our knowledge, the current\nto show that overexpression of VEGF165\nstudy is the first\naccelerates STS angiogenesis, growth, metastasis, and chemo-\nresistance to doxorubicin. The effect of VEGF165 overexpression on\ntumor progression in different\ntumor types is not entirely\nconsistent. For example, overexpression of VEGF165 enhanced both\ntumor angiogenesis and growth in a MCF-7 human breast\ncarcinoma cell system (27) but only stimulated tumor angiogenesis\nwithout increasing the growth rate of OVCAR-3 human ovarian\ncancer xenografts (28). Varying degrees of effect were also noticed\nin preclinical models of h-cell carcinogenesis (29), melanoma (30),\nand glioma (31). Some of the apparent discrepancies observed in\nVEGF function in cancer development may be due to different ways\nVEGF impinges on the tumor-versus-microenvironment interplay\nand suggest that tumor type and tumor cell variability are\nimportant determinants for the role of VEGF.\nThis report also facilitates an awareness that chemoresistance\nmust be considered in the context of both tumor cells as well as\ncells recruited from the microenvironment. Examining both of\nthese cellular components in tandem has enabled an initial\nidentification of the critical role of endothelial cell chemoresistance\nin STS progression and dissemination. We have shown that STS\nchemoresistance is determined not only by tumor cell response to\ndrugs but also by host cells, although genetically stable, which can\nalso be adapted to become resistant to the toxic effect of\nchemotherapy. VEGF produced by tumor cells can mediate\nendothelial cell protection against the effects of chemotherapy by\npromoting the production and/or activation of cytoprotectors,\nsuch as survivin (32). In a similar manner, Kim et al. (33) have\nshown that the phosphatidylinositol 3-kinase/Akt pathway is\nactivated in endothelial cells exposed to radiation, which in turn\ncan lead to the activation of other prosurvival pathways. Targeting\nendothelial cell–based survival pathway components may prevent\ntheir production or activation and may be an important strategy to\novercome therapy resistance. In the context of tumor angiogenesis,\nthis conceptual framework suggests that abrogating tumor VEGF\nexpression-mediated chemoresistance might be augmented by\nstrategies that attack both the tumor cell as a source of VEGF as\nwell as the microenvironment endothelial cells recruited by the\nelaboration of VEGF.\nAnti-VEGF monotherapy efficacy has been shown in STS animal\nmodels (34). We have shown previously that combining anti-VEGF\ntherapy with chemotherapy results in superior STS inhibition (5).\nThe current report extends our previous studies to encompass the\nmore clinically significant issue of controlling metastasis, the most\ncommon lethal event in STS. Moreover, our study design uses a\npreclinical model of supranormal VEGF production that more\naccurately mirrors the human STS tumor-bearing continuum.\nA perceived concern with our model system might be that it\nlacks true relevance to the clinical situation in patients with STS.\nScientific and clinical oncology literature exists, which shows that\nVEGF expression is significantly elevated in human STS: in tumor\nspecimens at the RNA (35) and protein levels (10, 36, 37) as well as\nin patient serum samples (25, 26, 35, 38). Elevated serum VEGF\nlevels are reported to correlate with STS tumor grade (25, 36) and\nsize (25, 26), and immunohistochemical assessment of VEGF\nlevels in STS tumors correlates high-intensity VEGF staining with\nadvanced tumor grade (36, 37). These reports indicate clearly that\nVEGF is overexpressed in STS. In several series studying circulating\nVEGF levels in patients with STS, VEGF levels were found to be\nelevated in the vast majority of patients compared with healthy\ncontrols (25, 26, 35, 38). For example, Graeven et al. (25) reported\nmean serum VEGF levels to be 137 pg/mL (22-404 pg/mL; n = 18)\nfor healthy controls compared with 672 pg/mL (44-2,500 pg/mL;\nn = 62) in STS patients with macroscopic tumor lesions. In\ngenerating the model system used in our article, parental STS cell\nline VEGF secretion levels were compared with those of an\nimmortalized normal human fibroblast cell line. Two cell lines,\nwith low to moderate expression levels (by Western blot compa-\nrison among the cell lines; Fig. 1A), were stably transfected with\nVEGF to assess the effect of VEGF on STS biology with the\nminimum of genetic disruption. We reported previously quantitat-\ned VEGF levels secreted by at least one of these STS cell lines into\ntissue culture medium (12) using the same ELISA methodology\nused for patient sample analysis in the previously referred\npublished study. The tissue culture VEGF levels indicated that\nparental cell line VEGF secretion approximated to moderate serum\nlevels in patient samples. Tissue culture VEGF levels are,\nadmittedly, a debatable surrogate for equivalency to patient serum\nlevels; however, the shortcoming in the comparison is that the\nmeasured levels in the medium would be an overestimate\ncompared with in vivo-secreted VEGF levels. Taking this into\nconsideration, combined with our ELISA measurements falling into\nthe same range as moderate patient serum levels and Western blot\nresults showing secreted VEGF levels to be the same or less than\nthat released by immortalized fibroblast cell lines, we generated\nVEGF-overexpressing lines to more closely mimic the clinical STS\nscenario because elsewhere serum concentrations of VEGF have\nbeen reported as being 10 (35) and 12 (26) times higher than\nnormal in patients with STS. The need for a model that expresses\nabundant VEGF to mimic the clinical situation additionally stems\nfrom the reality that the high expression of VEGF observed in\nclinical STS is most likely secondary to the hypoxia generated in\nthese tumors, which frequently grow to very large dimensions\nbefore detection. A recently published report supports the need for\nsuch a model in STS by showing that STS cells in culture, when\nexposed to hypoxic conditions, significantly increase VEGF\nexpression compared with normoxic controls (39). These issues,\ntaken together with the previously mentioned reports, indicate that\nSTS overexpress VEGF and that our laboratory-generated model\nrepresents an acceptable investigative model of the actual clinical\nsituation.\nVulnerability of established tumor vasculature to therapeutic\nmanipulation of VEGF/VEGFR2 seems to occur selectively only in\nthose vessels that are immature and lack the contact or presence of\npericytes (40). Conversely, the presence of pericytes as an integral\ncomponent of mature tumor neovasculature may provide a basis\nfor surviving antiangiogenic treatments. These considerations may\nhelp explain the results of our experiment where the percentage of\ncells that stained positive for SMA, a marker of pericytes, was\nhighest in VEGF transfectants treated with the combined regimen.\nThis treatment, compared with single-agent or no treatment\ncontrols, was much more effective in eliminating SMA-negative\nimmature cells, leaving a much higher percentage of cells that were\nSMA positive. A similar finding has been reported in orthotopic\nmodels of human gastric (41) and pancreatic cancers (42, 43)\ntreated with VEGFR tyrosine kinase inhibitors. Although immature\nblood vessels rely on VEGF-induced survival signals (44), inhibition\nof both VEGF and platelet-derived growth factor signaling was\nfound to be superior for the elimination of pericytes (40). It has\nbeen suggested previously that eliminating immature and ‘‘leaky’’\nneovasculature while leaving intact the more mature vasculature\ncan provide a more effective conduit for chemotherapy to reach the\ntumor cells (45). However, apparently, combinations of treatments\nthat will also target endothelial support cells, such as pericytes,\nmight result in even better tumor control and thus merit further\ninvestigation.\nthan as stand-alone therapy. An additional\nTwo clinical trials have been conducted in STS using antiangio-\ngenic systemic treatments (46, 47). A phase II study examined the\neffect of SU5416, a tyrosine kinase VEGFR2 inhibitor, on 13 patients\nwith locally advanced or metastatic STS (46). Although no objective\ntumor responses were observed, the drug was generally well\ntolerated. Pretreatment and post-treatment tumor biopsies showed\nno significant alteration in VEGFR2 phosphorylation, suggesting\nthat therapeutic levels of the drug were not achieved. Furthermore,\nthe drug was used as monotherapy, and per our observations\nabove, VEGFR inhibition seems to work better as a chemosensitiz-\ntrial\ning agent\ncombined doxorubicin with bevacizumab, a human monoclonal\nantibody that binds VEGF (47). A 12% response rate was observed,\nwhich was not greater than the single-agent doxorubicin response\nrate. Although 65% of\nthe study patients enjoyed disease\nstabilization, this came at the expense of increased cardiomyop-\nathy. Apparently, the discrepancy between this clinical study and\nour own preclinical results has to do with the extent of disease at\ntime of therapy initiation. The patients in the clinical trial already\nhad extensive metastatic disease when treated. Established\nmetastases contain mature blood vessels against which the\ndescribed antiangiogenic therapy may be ineffective as we have\nshown above. In contrast, the animals in our study were treated\n(i.e., before the development of\nwith a preventive intent\nmetastases). A second difference is that the clinical trial used\nconventional dose of chemotherapy, resulting in major toxicity. In\nour studies, low-dose chemotherapy was effective as conventional\ndoses but with much less toxicity. Taken together, these results\nsuggest the possibility that combining anti-VEGF biotherapy with\nlow-dose doxorubicin might be most useful in stage III high VEGF-\nsecreting STS patients in whom there is a high risk of eventual\nmetastatic spread. Clinical trials pertaining to this question may be\na relevant next step.\nIn conclusion, combination chemotherapy/antiangiogensis\ntreatment algorithms targeting both endothelial and tumor cells\nshould be considered for stage III STS patients so that both\nprimary tumor and subclinical metastases can be addressed. Such\ncombinations, if successful, may offer new vistas of tumor control\nfor STS patients burdened by this devastating malignancy.",
         "Angiogenic profile of soft tissue sarcomas based on analysis of circulating factors and microarray gene expression.;Phase II study of doxorubicin and bevacizumab for patients with metastatic soft-tissue sarcomas.;Expression of vascular endothelial growth factor and its receptor, KDR/Flk-1, in soft tissue sarcomas.;Soft-tissue sarcomas in adults.;Analysis of hypoxia-related gene expression in sarcomas and effect of hypoxia on RNA interference of vascular endothelial cell growth factor A.;Therapeutic targeting of the tumor microenvironment.;Transcriptional Repression of Protein Kinase Cα via Sp1 by Wild Type p53 Is Involved in Inhibition of Multidrug Resistance 1 P-Glycoprotein Phosphorylation*;Antiangiogenic versus cytotoxic therapeutic approaches to human pancreas cancer: an experimental study with a vascular endothelial growth factor receptor-2 tyrosine kinase inhibitor and gemcitabine.;ZD6474, a vascular endothelial growth factor receptor tyrosine kinase inhibitor with additional activity against epidermal growth factor receptor tyrosine kinase, inhibits orthotopic growth and angiogenesis of gastric cancer.;Angiopoietins in tumours: the angiogenic switch;Phase II Study of the Antiangiogenic Agent SU5416 in Patients with Advanced Soft Tissue Sarcomas;Circulating angiogenic factor levels correlate with extent of disease and risk of recurrence in patients with soft tissue sarcoma.;Combined inhibition of VEGF‐ and PDGF‐signaling enforces tumor vessel regression by interfering with pericyte‐mediated endothelial cell survival mechanisms;Blood vessels and nerves: common signals, pathways and diseases;Doxorubicin-based chemotherapy for the palliative treatment of adult patients with locally advanced or metastatic soft tissue sarcoma.;Tumor-derived vascular endothelial growth factor up-regulates angiopoietin-2 in host endothelium and destabilizes host vasculature, supporting angiogenesis in ovarian cancer.;Protracted low-dose effects on human endothelial cell proliferation and survival in vitro reveal a selective antiangiogenic window for various chemotherapeutic drugs.;Vascular and haematopoietic stem cells: novel targets for anti-angiogenesis therapy?;Anti-vascular endothelial growth factor receptor 2 antibody reduces tumorigenicity and metastasis in orthotopic prostate cancer xenografts via induction of endothelial cell apoptosis and reduction of endothelial cell matrix metalloproteinase type 9 production.;Combined anti-fetal liver kinase 1 monoclonal antibody and continuous low-dose doxorubicin inhibits angiogenesis and growth of human soft tissue sarcoma xenografts by induction of endothelial cell apoptosis.;A role for survivin in chemoresistance of endothelial cells mediated by VEGF;VEGF(121), VEGF(165) overexpression enhances tumorigenicity in U251 MG but not in NG-1 glioma cells.;Vascular endothelial growth factor-165 overexpression stimulates angiogenesis and induces cyst formation and macrophage infiltration in human ovarian cancer xenografts.;Vascular endothelial growth factor-A(165) induces progression of melanoma brain metastases without induction of sprouting angiogenesis.;Vascular endothelial growth factor (VEGF) is an autocrine growth factor for VEGF receptor-positive human tumors.;Wild type p53 sensitizes soft tissue sarcoma cells to doxorubicin by down‐regulating multidrug resistance‐1 expression;Normalizing tumor vasculature with anti-angiogenic therapy: A new paradigm for combination therapy;High tumor levels of vascular endothelial growth factor predict poor response to systemic therapy in advanced breast cancer.;Angiogenesis and antiangiogenic approaches to sarcomas;Concentration of vascular endothelial growth factor in the tumour tissue as a prognostic factor of soft tissue sarcomas;Vascular Endothelial Growth Factor and Soft Tissue Sarcomas: Tumor Expression Correlates With Grade;Up-regulation of vascular endothelial growth factor in breast cancer cells by the heregulin-beta1-activated p38 signaling pathway enhances endothelial cell migration.;Wild-type p53 suppresses angiogenesis in human leiomyosarcoma and synovial sarcoma by transcriptional suppression of vascular endothelial growth factor expression.;Blockade of the epidermal growth factor receptor signaling by a novel tyrosine kinase inhibitor leads to apoptosis of endothelial cells and therapy of human pancreatic carcinoma.;Serum levels of vascular endothelial growth factor and basic fibroblast growth factor in patients with soft-tissue sarcoma;Selective ablation of immature blood vessels in established human tumors follows vascular endothelial growth factor withdrawal.;Vascular Endothelial Growth Factor Regulates Endothelial Cell Survival through the Phosphatidylinositol 3′-Kinase/Akt Signal Transduction Pathway;Overexpression of c-erbB-2/neu in breast cancer cells confers increased resistance to Taxol via mdr-1-independent mechanisms.;Patterns and Emerging Mechanisms of the Angiogenic Switch during Tumorigenesis;In vitro neutralization of vascular endothelial growth factor activation of Flk-1 by a monoclonal antibody;c-erbB-2/neu overexpression enhances metastatic potential of human lung cancer cells by induction of metastasis-associated properties.;Effects of verapamil on the acute toxicity of doxorubicin in vivo.;Accurate identification of experimental pulmonary metastases.;Overexpression of Vascular Endothelial Growth Factor-a165 Updated Version Cited Articles Citing Articles E-mail Alerts Overexpression of Vascular Endothelial Growth Factor-a165 Enhances Tumor Angiogenesis but Not Metastasis during ␤-cell Carcinogenesis;Molecular strategies targeting the host component of cancer to enhance tumor response to radiation therapy.;Cancer Research Cancer Res;Patterns of expression and secretion of vascular endothelial growth factor in malignant soft-tissue tumours;Dormancy of micrometastases: Balanced proliferation and apoptosis in the presence of angiogenesis suppression",
         "Vascular endothelial growth factor overexpression by soft tissue sarcoma cells: implications for tumor growth, metastasis, and chemoresistance.",
         "None"
        ],
        [
         "13",
         "000a0424b18c417d0e86d2f7896613f58306a5b3",
         "Pitch perception is crucial for vocal communication, music perception, and auditory object\nprocessing in a complex acoustic environment. The neural representation of pitch in the cerebral\ncortex has long been an outstanding question in auditory neuroscience. Several lines of evidence\nnow point to a distinct non-primary region of auditory cortex in primates that contains a cortical\nrepresentation of pitch.",
         "D. Bendor,Xiaoqin Wang",
         "D. Bendor,Xiaoqin Wang",
         "https://europepmc.org/articles/pmc4325365?pdf=render",
         "Pitch perception is crucial for vocal communication, music perception, and auditory object\nprocessing in a complex acoustic environment. The neural representation of pitch in the cerebral\ncortex has long been an outstanding question in auditory neuroscience. Several lines of evidence\nnow point to a distinct non-primary region of auditory cortex in primates that contains a cortical\nrepresentation of pitch.",
         "",
         "Introduction\nOur ability to hear pitch allows us to determine if an acoustic signal sounds “higher” or\n“lower” than another acoustic signal. We can track pitch changes over time to hear a musical\nmelody or to recognize intonations in tonal languages like Chinese, and do so robustly\nacross different musical instruments or speakers. Two musical instruments have dissimilar\nsounds because of how they spectrally shape the acoustic energy they produce (i.e., which\nfrequencies are amplified or attenuated). Yet they can still play the same musical note, and\nthus the same pitch, if the fundamental frequencies of the acoustic waveforms match. For a\nstring instrument such as a violin, the fundamental frequency is equal to the vibration\nfrequency of the a plucked or bowed string. In a more general sense, this fundamental\nfrequency is related to the temporal periodicity of the sound’s acoustic waveform [1].\nHow does the auditory system extract the fundamental frequency from the complex\nspectrum of a sound in order to generate a percept of pitch? Several recent findings indicate\nthat there is a specialized region in the auditory cortex of primates that is involved in the\nrepresentation of pitch [2-10]. Here we review several key studies in the identification of a\npitch processing center in primate auditory cortex, and discuss issues concerning the neural\nsubstrate of pitch perception.\n1. What is pitch?\nPitch is defined as “that attribute of auditory sensation in terms of which sounds may be\nordered on a musical scale” (American Standards Association). The key distinction between\nfrequency and pitch is that the former is a physical description of a sound, whereas the latter\nis a perceptual attribute of a sound. Although frequency and pitch are typically similar for\nsounds composed of a single spectral component (pure tones), this relationship becomes\nmore complicated when sounds are composed of multiple spectral components (e.g.,\nharmonic complex sounds). For such sounds, the pitch is related to their fundamental\nfrequency. Spectrally, the fundamental frequency can be thought of as the highest frequency\nfor which the spectral components of the sound are integer multiples. Thus a harmonic\ncomplex sound consisting of 100, 200 and 300 Hz tones has a pitch of 100 Hz (Figure 1A).\nTemporally, pitch is typically related to the repetition rate of periodic amplitude envelope\nchanges in a complex acoustic signal. For example, a sequence of brief bursts of broadband\nnoise repeated periodically at 100 times per second has a pitch of 100 Hz (Figure 1B).\nBoth speech and animal vocalizations contain spectrally complex, harmonically structured\nsounds. Pitch acts to group these harmonics together into a single percept that is related to\nthe periodicity of the source generating a sound (the vibration of the vocal apparatus). Pitch\nperception, a necessary feature of our auditory system, allows us to hear two speech sounds\nas distinct from each other when both have similar spectrums and source locations but differ\nin their fundamental frequencies, such as the same word spoken by a male and a female\nspeaker standing next to each other. One of the remarkable features of the pitch perception is\nthat a spectral component does not need to be present at the fundamental frequency in order\nfor a pitch equal to the fundamental frequency to be perceived [1]. In other words, although\nindividual components of a harmonic complex sound consisting of 200, 300, and 400 Hz\ntones do not generate a pitch of 100 Hz when they are played one at a time, a listener would\nhear a pitch of 100 Hz when these components are played together, even though there is no\nenergy at 100 Hz (Figure 1C). This is a well-known phenomenon called “missing\nfundamental pitch” and is a hallmark of pitch perception [1]. Our ability to perceive pitch\nfrom missing fundamental harmonic complex sounds explains why we are able to hear the\npitch of someone’s voice over the telephone system that effectively cuts out lower\nfrequencies where fundamental frequencies of human voices are normally present.\n2. Potential neural representation schemes of pitch in the brain\nAcoustic signals can be mathematically analyzed by the Fourier transform which\ndecomposes a sound into its spectral components. The auditory system performs a similar\ntransformation in the cochlea where low frequency sounds or components excite the apical\nend of the basilar membrane, while the basal end responds best to high frequency sounds or\ncomponents [11]. This tonotopic representation is preserved at each level of the auditory\nsystem, up to and including auditory cortex, such that neurons typically respond to a\nrestricted range of frequencies, and are physically ordered within a brain area by the\nfrequency to which each neuron is most sensitive (the “tonotopicity” or “tonotopic map”).\nWhile tonotopicity provides an explicit representation of frequency, it does not provide an\nexplicit representation of pitch for complex sounds. Previous MEG experiments in humans\nhave suggested that a pitch map coexists with the tonotopic map in primary auditory cortex\n(AI) [12-13]. Pitch map topographies both parallel [12] and orthogonal [13] to AI’s\ntonotopic map have been proposed, but these topographies have not been confirmed using\nimaging techniques that directly measure spatial locations of neural activity with sufficient\nresolution. Another possibility is that pitch is processed in parallel with frequency but in\nseparate brain regions such that frequency is represented topographically in AI and pitch is\nrepresented in a non-primary area. In the pursuit of neural mechanisms for pitch perception,\nhuman and animal studies have been complementary to each other. In order to bridge pitch-\nrelated research between animal and human studies, it is important to establish similarities in\nhow pitch is perceived as well as the anatomical and physiological properties of the brain\nareas studied. In this regard, non-human primate models play an essential role to help reveal\nneural mechanisms of pitch perception.\n3. The perception of pitch is not unique to humans\nAn advantage of using non-human primates as experimental models for studying pitch-\nrelated questions is that the frequency range of their hearing is similar to that of humans\n[14]. The ability of monkeys to hear the pitch of the missing fundamental was first\ndemonstrated in a behavioral study by Tomlinson and Schwarz (1988) [15]. Rhesus\nmonkeys were trained to push a button if the two complex tones presented sequentially had\nthe same fundamental frequency. The monkeys were able to perform the task even when a\nmissing fundamental complex tone was used, demonstrating their ability to hear the missing\nfundamental pitch. The ability to perceive this missing fundamental is not unique to\nprimates, and has also been shown in several other animal species including birds [16] and\ncats [17]. In addition, monkeys have been shown to be capable of spectral pitch\ndiscrimination [18], melody recognition [19-20] and octave generalization [21], each of\nwhich requires the perception of pitch. One important difference between humans and some\nanimal species is the size of their cochlea. For smaller cochleas, excitations on the basilar\nmembrane caused by individual components of a harmonic complex tone are more closely\nspaced than those in larger cochleas. Different pitch processing mechanisms have been\npostulated for sounds with harmonics producing segregated regions of excitation on the\nbasilar membrane (“resolved harmonics”) and for sounds with harmonics that are too close\ntogether and only produce a single region of excitation (“unresolved harmonics”) [1,11].\nThus animals with smaller cochleas may need to rely more on the pitch processing\nmechanism that utilizes unresolved harmonics than do humans.\n4. Anatomical and physiological similarity of auditory cortex among\nprimates\nThe anatomical study by Brodmann (1909) [22] suggested that the structure of the temporal\nlobe is largely preserved across primate species (New World monkeys, Old World monkeys,\nand humans). Studies in recent years have revealed distinctions between various auditory\ncortical areas using anatomical [23-29] and physiological criteria [23,24,30-39]. Humans\nand monkeys appear to share a similar organization of primary, primary-like, and secondary\ncortical areas based on accumulating evidence [26,27], suggesting a generalizable structure\nand function of auditory cortex among primates.\nPrimate auditory cortex is divided into a core region of primary and primary-like areas\nsurrounded by a belt of multiple secondary areas. In humans and monkeys, the core areas of\nauditory cortex can be distinguished from belt areas by their cytoarchitecture, namely a\nmore prominent granular layer [23,24,27,29]. In monkeys the core areas have been shown to\nreceive thalamic inputs from the principal (ventral) nucleus of the medial geniculate body\n(MGB) [26]. In contrast, the belt areas receive more dominant thalamic inputs from the non-\nleminscal divisions (dorsal and medial) of the MGB [26]. In both monkeys and humans,\nneurons in core areas respond strongly to narrowband sounds such as tones, whereas\nneurons in belt areas respond better to more complex sounds (e.g., noise, frequency\nmodulation, low-contrast spectrums, and vocalizations) [32,36-39]. Within the core areas,\nfor both monkeys [23,24] and humans [33], two mirror symmetric tonotopic maps sharing a\nlow-frequency border have been identified, corresponding to AI and the rostral field (R). In\nhumans, the core areas are generally confined to Heschl’s gyrus [27], with AI located\nmedially to R (Figure 2A). However, this is difficult to determine precisely by anatomical\nlandmarks due to substantial intra-subject variability. Monkeys do not possess an anatomical\nlandmark for the location of AI, and the location of AI must be determined physiologically\nor histologically. AI is typically buried within the lateral sulcus in monkeys, except for a few\nNew World species (e.g. marmosets, owl monkeys) for which the lateral portion of AI is\nlocated on the surface of the superior temporal gyrus [7,23]. There is anatomical [23] and\nphysiological evidence [7,23] of a third core area (RT) that lies rostral to R (Figure 2B).\nKaas and Hackett [26] have postulated that each core area is connected to a medial and\nlateral neighboring belt area, with additional belt areas located on the rostral and caudal ends\nof the core region of auditory cortex (AI, R and RT). Three of these lateral belt areas (CL,\nML, and AL) have been mapped electrophysiologically, and have been found to possess\nsimilar mirror symmetric tonotopic maps as their adjacent core areas [38,39]. Additional\nhigher auditory areas such as parabelt [40-41] and the rostral pole [42] have been defined\nanatomically, but little is known about their physiological properties.\n5. A pitch-processing center in primate auditory cortex\nWhere is a sound’s pitch encoded in the brain? The information needed to extract the pitch\nof a complex sound is contained in both the discharge rates and temporal firing patterns of\nthe population of auditory nerve fibers [43-45]. There is no clear evidence that subcortical\nstations of the ascending auditory pathway explicitly encode pitch. In cats, bilateral lesions\nof the entire auditory cortex impair the discrimination of changes in the pitch of the missing\nfundamental, but not changes in frequency alone, demonstrating the necessity of auditory\ncortex in pitch perception [46]. Humans with partial auditory cortex lesions have also been\nshown to have poorer pitch discrimination [47], with lesions of the right auditory cortex\nshowing larger deficits [48-49]. Behavioral studies in patients with auditory cortex lesions\nhave further suggested that lesions anterior to primary auditory cortex result in more\npronounced deficits in pitch discrimination [49]. These studies point to a possible cortical\nsource of pitch representation. Several recent studies have identified a specific region in\nprimate auditory cortex that appears to be involved in representing a sound’s pitch [2-10]. A\nschematic showing the location of this pitch-processing center in relation to its neighboring\ncortical areas for both humans and monkeys is shown in Figure 2.\nEvidence from human studies\nIn an fMRI study, Patterson and colleagues [3] identified a specific region of human\nauditory cortex (lateral Heschl’s gyrus), which was preferentially activated by temporally\nregular sounds with a pitch. The acoustic stimulus used in this study is iterated rippled noise\n(IRN), which is generated by iteratively adding delayed broadband noise [1,50]. By\nadjusting the delay and the number of iterations, the fundamental frequency and pitch\nsalience of the resulting sound can be changed, respectively. For IRN sounds with low\nfrequency pitches, auditory filters in the high frequency range cannot distinguish between\nnoise and IRN stimuli, due to the spectral resolvability limitation, even though these two\nsounds have physically different spectrums. Yet, subjects hear a pitch from IRN sounds but\nnot from noise because of the temporally regular acoustic structure of the IRN sound\n(extracted presumably by temporal pitch mechanisms). By subtracting the Blood\nOxygenation Level Dependent (BOLD) signal originating from the IRN sound from that\nevoked by noise, Patterson and colleagues determined that only lateral Heschl’s gyrus, a\nnon-primary auditory region anterolateral to primary auditory cortex, responded to the\ntemporal regularity or pitch of the acoustic stimuli (this occurred bilaterally) [3].\nIn another imaging study by Penagos, Melcher, and Oxenham [4], the BOLD signal was\ncompared between four harmonic complex sounds that had either a low or high pitch and\noccupied either a low or high spectral range. Of these four sounds, only the harmonic\ncomplex sound with the low pitch and high spectral components had unresolved harmonics.\nSounds with unresolved harmonics have a weaker pitch salience than sounds containing\nresolved harmonics, for which spectral cues are available [1]. Thus, Penagos et al. [4] were\nable to compare BOLD signals between sounds evoking a strong or weak pitch salience, but\nmatched in their fundamental frequency or spectral range. Bilaterally, a restricted region of\nnon-primary auditory cortex, anterolateral to AI, was found more weakly responsive to the\nsound with low pitch salience than the other three sounds with high pitch salience [4]. This\nstudy therefore confirms the location of the pitch-processing center identified by Patterson et\nal. [3] using a different type of acoustic stimuli and extends the earlier finding to\ndemonstrate the sensitivity for pitch salience within this pitch-processing center.\nThe significance of the lateral Heschl’s gyrus in pitch representation was further\ninvestigated by Schneider et al. [6] using a combination of psychophysics, anatomical MRI\nscans and MEG measurements. For harmonic complex tones with a few components, a\nperson may hear the pitch increasing or decreasing when the frequencies of harmonics are\ndecreased while the fundamental frequency is increased [1]. Different subjects showed a\nbias towards using the fundamental frequency or spectrum frequency when discriminating\npitch changes. These biases were found to be highly correlated with hemispheric asymmetry\nin relative size between the right and left lateral Heschl’s gyrus (and not with other regions\nof auditory cortex) [6]. Subjects that relied more on fundamental frequency to discriminate\npitch had a larger left lateral Heschl’s gyrus, while subjects using spectrum frequency had a\nlarger right lateral Heschl’s gyrus. In addition, MEG responses recorded from a source\nestimated as lateral Heschl’s gyrus showed a similar asymmetry between hemispheres.\nStronger responses were obtained from the left hemisphere for subjects with a bias towards\nusing fundamental frequency in pitch discrimination. In contrast, subjects with a bias\ntowards using spectrum frequency had larger MEG responses recorded from the right\nhemisphere.\nEvidence from non-human primate studies\nGiven the evidence for a pitch-processing center in human auditory cortex and the\nanatomical and physiological similarities of auditory cortex between humans and monkeys,\nit is reasonable to expect that monkeys possess a pitch-processing center in their auditory\ncortex as well. Schwarz and Tomlinson [51] searched for single-unit responses to the\nfundamental frequency of missing fundamental harmonic complex sounds in AI of three\nawake macaque monkeys previously trained on a pitch discrimination task, but failed to find\nany neurons responsive to the fundamental frequency centered at neuron’s characteristics\nfrequency (CF). Schwarz and Tomlinson concluded that pitch is either represented implicitly\nacross a population of neurons in AI or an explicit representation exists outside of AI.\nHowever, Fishman et al. [52] were unable to find an implicit representation for the missing\nfundamental in AI based on population neuronal responses, using multi-unit recordings in\nawake macaque monkeys.\nIn a recent study in awake marmoset monkeys (a New World primate species) [7], Bendor\nand Wang searched for single-unit responses to the missing fundamental in AI and\nsurrounding non-primary areas and identified a restricted region anterolateral to AI\ncontaining pitch-selective neurons. Neurons were identified as pitch-selective if they\nresponded to missing fundamental sounds (with harmonics outside the neuron’s excitatory\nfrequency response area) and pure tones with a similar pitch. A typical pitch-selective\nneuron responded to an array of spectrally dissimilar sounds (harmonic complex tones, click\ntrains, iterated ripple noise) when the pitch was near the neuron’s preferred fundamental\nfrequency, which was found similar to the neuron’s CF determined by pure tones. Relative\nto AI, this newly identified region containing pitch-selective neurons in marmoset monkeys\nis in a similar location to the pitch-processing center found in humans [2-6,8-10] (Figure 2).\nIn addition, Bendor and Wang [7] found that pitch-selective neurons preferred temporally\nregular sounds and were sensitive to pitch salience changes due to harmonic frequency and\norder, in agreement with the imaging studies by Patterson et al. [3] and Penagos et al. [4].\nThe microelectrode recording study by Bendor and Wang [7] also showed that the pitch-\nprocessing region found in marmosets contained non-pitch-selective neurons (spanning a\nsimilar range of CFs) that responded to the spectral frequency of the sound, rather than the\nfundamental frequency. Potentially, these two classes of neurons co-localized within the\npitch-processing center respectively encode spectral and missing fundamental pitch\npercepts. For ambiguous pitch changes, in which the fundamental frequency and spectrum\nshift in opposite directions, as in the study by Schneider et al. [6], these two types of neurons\nwould provide conflicting information. Unequal weighting towards one of these neuron\ntypes within the pitch-processing center (and/or between hemispheres) may be the cause of a\nsubject’s perceptual bias of hearing pitch changes based on the fundamental or spectral\nfrequency. No topography has yet been identified within the primate’s pitch-processing\ncenter in above cited studies [2-10], possibly due to spatial resolution constraints.\nMany marmoset AI neurons located outside the pitch-processing region are tuned to the\nmodulation frequency (repetition rate) of an amplitude- or frequency-modulated sound.\nHowever, in sharp contrast to pitch-selective neurons, these responses require that the\nacoustic signal’s spectral components be within the neuron’s frequency response area [57].\nIn awake marmoset AI, neural responses to stimulus repetition rates below ~40 Hz (near the\nlower limit of pitch [61,62]) are represented temporally by stimulus-synchronized\ndischarges, while a monotonically tuned rate code is used by another population of neurons\nto represent higher repetition rates [58]. Neurons with harmonically related multi-peaked\nfrequency response areas have previously been observed in marmoset AI [59], although they\ndiffer from pitch-selective neurons in that their primary frequency tuning and harmonic\nresponses are most commonly outside the frequency range of pitch perception, and that they\nrespond to each component of a complex harmonic tone. Together, the findings from\nmarmoset auditory cortex suggest that pitch-processing is a specialized function by a\nsubpopulation of neurons in a restricted region of non-primary auditory cortex, whereas\nprocessing of temporally modulated or harmonically rich signals is a general function by\nother auditory neurons located across the tonotopic axis.\nDifferences between observations from studies in primate and non-primate species\nSeveral previous studies investigated how missing fundamental sounds are represented in\nthe auditory cortex of non-primates species. Microelectrode recordings in gerbils suggested\nthat AI neurons could respond to the periodicity of amplitude-modulated tones with the\nspectral components located outside neuron’s excitatory frequency response area [53], in\ncontrast to a putative non-primary auditory cortical area encoding pitch in primates [2-10].\nSchulze et el. [54] has also found a semi-circularly shaped map of best fundamental\nfrequency in gerbil auditory cortex using optical imaging techniques. Differences in the\nresults from these studies in primate and non-primate species could be due to an\nevolutionary divergence of pitch-processing strategies within auditory cortex, or may result\nfrom methodological differences. Combination tones at the fundamental frequency are\nproduced within the cochlea by missing fundamental sounds, creating a potential ambiguity\nregarding whether a missing fundamental or combination tone is the source of the neuron’s\nevoked response [1]. The use of appropriate sound levels (to ensure that combination tones\nare below a neuron’s pure tone response threshold) [7], as well as the use of noise maskers\n[4,7] or a cancellation tone [55] are among the necessary control conditions to confirm that a\ntrue missing fundamental response is observed.\n6. Further questions on pitch processing in auditory cortex\nThe discovery of a pitch-processing center in human (Figure 2A) and monkey (Figure 2B)\nauditory cortex is only the first step in understanding the physiological mechanisms of pitch\nperception. Several important questions remain unanswered. First, what is the source of\ninputs to pitch-selective neurons (corticocortical or thalamocortical)? The location of the\npitch center appears to be overlapping low-frequency portions of field R (primary-like) and\nlateral belt areas AL and ML. This suggests that the pitch-selective neurons may receive\ninputs from both ventral and dorsal divisions of the auditory thalamus (MGB) that respond\nto narrowband and wideband sounds, respectively [26]. This could be a possible source of\nthe response to pure tones and broadband missing fundamental sounds. Alternatively, given\nthe extensive connectivity between AI, R, and neighboring belt areas surrounding the pitch-\nprocessing center, the pitch-selective neurons may extract the fundamental frequency using\ninputs from neighboring cortical areas (including AI). Second, do pitch-selective neurons\nuse a spectral and/or temporal mechanism to extract the fundamental frequency of complex\nsounds? This is an issue that has been at the center of debate among auditory researchers for\nthe last over 60 years. Computational models and auditory nerve data support the possibility\nof both a purely temporal mechanism and a hybrid mechanism using both spectral and\ntemporal information [1,43-45]. In monkeys, AI neurons with temporal and spectral\nresponse properties potentially useful for these pitch models have been observed [56-59],\nbut whether they provide an input to pitch-selective neurons is unknown.",
         "",
         "",
         "broken link",
         "Pitch perception is crucial for vocal communication, music perception, and auditory object\nprocessing in a complex acoustic environment. The neural representation of pitch in the cerebral\ncortex has long been an outstanding question in auditory neuroscience. Several lines of evidence\nnow point to a distinct non-primary region of auditory cortex in primates that contains a cortical\nrepresentation of pitch.\nIntroduction\nOur ability to hear pitch allows us to determine if an acoustic signal sounds “higher” or\n“lower” than another acoustic signal. We can track pitch changes over time to hear a musical\nmelody or to recognize intonations in tonal languages like Chinese, and do so robustly\nacross different musical instruments or speakers. Two musical instruments have dissimilar\nsounds because of how they spectrally shape the acoustic energy they produce (i.e., which\nfrequencies are amplified or attenuated). Yet they can still play the same musical note, and\nthus the same pitch, if the fundamental frequencies of the acoustic waveforms match. For a\nstring instrument such as a violin, the fundamental frequency is equal to the vibration\nfrequency of the a plucked or bowed string. In a more general sense, this fundamental\nfrequency is related to the temporal periodicity of the sound’s acoustic waveform [1].\nHow does the auditory system extract the fundamental frequency from the complex\nspectrum of a sound in order to generate a percept of pitch? Several recent findings indicate\nthat there is a specialized region in the auditory cortex of primates that is involved in the\nrepresentation of pitch [2-10]. Here we review several key studies in the identification of a\npitch processing center in primate auditory cortex, and discuss issues concerning the neural\nsubstrate of pitch perception.\n1. What is pitch?\nPitch is defined as “that attribute of auditory sensation in terms of which sounds may be\nordered on a musical scale” (American Standards Association). The key distinction between\nfrequency and pitch is that the former is a physical description of a sound, whereas the latter\nis a perceptual attribute of a sound. Although frequency and pitch are typically similar for\nsounds composed of a single spectral component (pure tones), this relationship becomes\nmore complicated when sounds are composed of multiple spectral components (e.g.,\nharmonic complex sounds). For such sounds, the pitch is related to their fundamental\nfrequency. Spectrally, the fundamental frequency can be thought of as the highest frequency\nfor which the spectral components of the sound are integer multiples. Thus a harmonic\ncomplex sound consisting of 100, 200 and 300 Hz tones has a pitch of 100 Hz (Figure 1A).\nTemporally, pitch is typically related to the repetition rate of periodic amplitude envelope\nchanges in a complex acoustic signal. For example, a sequence of brief bursts of broadband\nnoise repeated periodically at 100 times per second has a pitch of 100 Hz (Figure 1B).\nBoth speech and animal vocalizations contain spectrally complex, harmonically structured\nsounds. Pitch acts to group these harmonics together into a single percept that is related to\nthe periodicity of the source generating a sound (the vibration of the vocal apparatus). Pitch\nperception, a necessary feature of our auditory system, allows us to hear two speech sounds\nas distinct from each other when both have similar spectrums and source locations but differ\nin their fundamental frequencies, such as the same word spoken by a male and a female\nspeaker standing next to each other. One of the remarkable features of the pitch perception is\nthat a spectral component does not need to be present at the fundamental frequency in order\nfor a pitch equal to the fundamental frequency to be perceived [1]. In other words, although\nindividual components of a harmonic complex sound consisting of 200, 300, and 400 Hz\ntones do not generate a pitch of 100 Hz when they are played one at a time, a listener would\nhear a pitch of 100 Hz when these components are played together, even though there is no\nenergy at 100 Hz (Figure 1C). This is a well-known phenomenon called “missing\nfundamental pitch” and is a hallmark of pitch perception [1]. Our ability to perceive pitch\nfrom missing fundamental harmonic complex sounds explains why we are able to hear the\npitch of someone’s voice over the telephone system that effectively cuts out lower\nfrequencies where fundamental frequencies of human voices are normally present.\n2. Potential neural representation schemes of pitch in the brain\nAcoustic signals can be mathematically analyzed by the Fourier transform which\ndecomposes a sound into its spectral components. The auditory system performs a similar\ntransformation in the cochlea where low frequency sounds or components excite the apical\nend of the basilar membrane, while the basal end responds best to high frequency sounds or\ncomponents [11]. This tonotopic representation is preserved at each level of the auditory\nsystem, up to and including auditory cortex, such that neurons typically respond to a\nrestricted range of frequencies, and are physically ordered within a brain area by the\nfrequency to which each neuron is most sensitive (the “tonotopicity” or “tonotopic map”).\nWhile tonotopicity provides an explicit representation of frequency, it does not provide an\nexplicit representation of pitch for complex sounds. Previous MEG experiments in humans\nhave suggested that a pitch map coexists with the tonotopic map in primary auditory cortex\n(AI) [12-13]. Pitch map topographies both parallel [12] and orthogonal [13] to AI’s\ntonotopic map have been proposed, but these topographies have not been confirmed using\nimaging techniques that directly measure spatial locations of neural activity with sufficient\nresolution. Another possibility is that pitch is processed in parallel with frequency but in\nseparate brain regions such that frequency is represented topographically in AI and pitch is\nrepresented in a non-primary area. In the pursuit of neural mechanisms for pitch perception,\nhuman and animal studies have been complementary to each other. In order to bridge pitch-\nrelated research between animal and human studies, it is important to establish similarities in\nhow pitch is perceived as well as the anatomical and physiological properties of the brain\nareas studied. In this regard, non-human primate models play an essential role to help reveal\nneural mechanisms of pitch perception.\n3. The perception of pitch is not unique to humans\nAn advantage of using non-human primates as experimental models for studying pitch-\nrelated questions is that the frequency range of their hearing is similar to that of humans\n[14]. The ability of monkeys to hear the pitch of the missing fundamental was first\ndemonstrated in a behavioral study by Tomlinson and Schwarz (1988) [15]. Rhesus\nmonkeys were trained to push a button if the two complex tones presented sequentially had\nthe same fundamental frequency. The monkeys were able to perform the task even when a\nmissing fundamental complex tone was used, demonstrating their ability to hear the missing\nfundamental pitch. The ability to perceive this missing fundamental is not unique to\nprimates, and has also been shown in several other animal species including birds [16] and\ncats [17]. In addition, monkeys have been shown to be capable of spectral pitch\ndiscrimination [18], melody recognition [19-20] and octave generalization [21], each of\nwhich requires the perception of pitch. One important difference between humans and some\nanimal species is the size of their cochlea. For smaller cochleas, excitations on the basilar\nmembrane caused by individual components of a harmonic complex tone are more closely\nspaced than those in larger cochleas. Different pitch processing mechanisms have been\npostulated for sounds with harmonics producing segregated regions of excitation on the\nbasilar membrane (“resolved harmonics”) and for sounds with harmonics that are too close\ntogether and only produce a single region of excitation (“unresolved harmonics”) [1,11].\nThus animals with smaller cochleas may need to rely more on the pitch processing\nmechanism that utilizes unresolved harmonics than do humans.\n4. Anatomical and physiological similarity of auditory cortex among\nprimates\nThe anatomical study by Brodmann (1909) [22] suggested that the structure of the temporal\nlobe is largely preserved across primate species (New World monkeys, Old World monkeys,\nand humans). Studies in recent years have revealed distinctions between various auditory\ncortical areas using anatomical [23-29] and physiological criteria [23,24,30-39]. Humans\nand monkeys appear to share a similar organization of primary, primary-like, and secondary\ncortical areas based on accumulating evidence [26,27], suggesting a generalizable structure\nand function of auditory cortex among primates.\nPrimate auditory cortex is divided into a core region of primary and primary-like areas\nsurrounded by a belt of multiple secondary areas. In humans and monkeys, the core areas of\nauditory cortex can be distinguished from belt areas by their cytoarchitecture, namely a\nmore prominent granular layer [23,24,27,29]. In monkeys the core areas have been shown to\nreceive thalamic inputs from the principal (ventral) nucleus of the medial geniculate body\n(MGB) [26]. In contrast, the belt areas receive more dominant thalamic inputs from the non-\nleminscal divisions (dorsal and medial) of the MGB [26]. In both monkeys and humans,\nneurons in core areas respond strongly to narrowband sounds such as tones, whereas\nneurons in belt areas respond better to more complex sounds (e.g., noise, frequency\nmodulation, low-contrast spectrums, and vocalizations) [32,36-39]. Within the core areas,\nfor both monkeys [23,24] and humans [33], two mirror symmetric tonotopic maps sharing a\nlow-frequency border have been identified, corresponding to AI and the rostral field (R). In\nhumans, the core areas are generally confined to Heschl’s gyrus [27], with AI located\nmedially to R (Figure 2A). However, this is difficult to determine precisely by anatomical\nlandmarks due to substantial intra-subject variability. Monkeys do not possess an anatomical\nlandmark for the location of AI, and the location of AI must be determined physiologically\nor histologically. AI is typically buried within the lateral sulcus in monkeys, except for a few\nNew World species (e.g. marmosets, owl monkeys) for which the lateral portion of AI is\nlocated on the surface of the superior temporal gyrus [7,23]. There is anatomical [23] and\nphysiological evidence [7,23] of a third core area (RT) that lies rostral to R (Figure 2B).\nKaas and Hackett [26] have postulated that each core area is connected to a medial and\nlateral neighboring belt area, with additional belt areas located on the rostral and caudal ends\nof the core region of auditory cortex (AI, R and RT). Three of these lateral belt areas (CL,\nML, and AL) have been mapped electrophysiologically, and have been found to possess\nsimilar mirror symmetric tonotopic maps as their adjacent core areas [38,39]. Additional\nhigher auditory areas such as parabelt [40-41] and the rostral pole [42] have been defined\nanatomically, but little is known about their physiological properties.\n5. A pitch-processing center in primate auditory cortex\nWhere is a sound’s pitch encoded in the brain? The information needed to extract the pitch\nof a complex sound is contained in both the discharge rates and temporal firing patterns of\nthe population of auditory nerve fibers [43-45]. There is no clear evidence that subcortical\nstations of the ascending auditory pathway explicitly encode pitch. In cats, bilateral lesions\nof the entire auditory cortex impair the discrimination of changes in the pitch of the missing\nfundamental, but not changes in frequency alone, demonstrating the necessity of auditory\ncortex in pitch perception [46]. Humans with partial auditory cortex lesions have also been\nshown to have poorer pitch discrimination [47], with lesions of the right auditory cortex\nshowing larger deficits [48-49]. Behavioral studies in patients with auditory cortex lesions\nhave further suggested that lesions anterior to primary auditory cortex result in more\npronounced deficits in pitch discrimination [49]. These studies point to a possible cortical\nsource of pitch representation. Several recent studies have identified a specific region in\nprimate auditory cortex that appears to be involved in representing a sound’s pitch [2-10]. A\nschematic showing the location of this pitch-processing center in relation to its neighboring\ncortical areas for both humans and monkeys is shown in Figure 2.\nEvidence from human studies\nIn an fMRI study, Patterson and colleagues [3] identified a specific region of human\nauditory cortex (lateral Heschl’s gyrus), which was preferentially activated by temporally\nregular sounds with a pitch. The acoustic stimulus used in this study is iterated rippled noise\n(IRN), which is generated by iteratively adding delayed broadband noise [1,50]. By\nadjusting the delay and the number of iterations, the fundamental frequency and pitch\nsalience of the resulting sound can be changed, respectively. For IRN sounds with low\nfrequency pitches, auditory filters in the high frequency range cannot distinguish between\nnoise and IRN stimuli, due to the spectral resolvability limitation, even though these two\nsounds have physically different spectrums. Yet, subjects hear a pitch from IRN sounds but\nnot from noise because of the temporally regular acoustic structure of the IRN sound\n(extracted presumably by temporal pitch mechanisms). By subtracting the Blood\nOxygenation Level Dependent (BOLD) signal originating from the IRN sound from that\nevoked by noise, Patterson and colleagues determined that only lateral Heschl’s gyrus, a\nnon-primary auditory region anterolateral to primary auditory cortex, responded to the\ntemporal regularity or pitch of the acoustic stimuli (this occurred bilaterally) [3].\nIn another imaging study by Penagos, Melcher, and Oxenham [4], the BOLD signal was\ncompared between four harmonic complex sounds that had either a low or high pitch and\noccupied either a low or high spectral range. Of these four sounds, only the harmonic\ncomplex sound with the low pitch and high spectral components had unresolved harmonics.\nSounds with unresolved harmonics have a weaker pitch salience than sounds containing\nresolved harmonics, for which spectral cues are available [1]. Thus, Penagos et al. [4] were\nable to compare BOLD signals between sounds evoking a strong or weak pitch salience, but\nmatched in their fundamental frequency or spectral range. Bilaterally, a restricted region of\nnon-primary auditory cortex, anterolateral to AI, was found more weakly responsive to the\nsound with low pitch salience than the other three sounds with high pitch salience [4]. This\nstudy therefore confirms the location of the pitch-processing center identified by Patterson et\nal. [3] using a different type of acoustic stimuli and extends the earlier finding to\ndemonstrate the sensitivity for pitch salience within this pitch-processing center.\nThe significance of the lateral Heschl’s gyrus in pitch representation was further\ninvestigated by Schneider et al. [6] using a combination of psychophysics, anatomical MRI\nscans and MEG measurements. For harmonic complex tones with a few components, a\nperson may hear the pitch increasing or decreasing when the frequencies of harmonics are\ndecreased while the fundamental frequency is increased [1]. Different subjects showed a\nbias towards using the fundamental frequency or spectrum frequency when discriminating\npitch changes. These biases were found to be highly correlated with hemispheric asymmetry\nin relative size between the right and left lateral Heschl’s gyrus (and not with other regions\nof auditory cortex) [6]. Subjects that relied more on fundamental frequency to discriminate\npitch had a larger left lateral Heschl’s gyrus, while subjects using spectrum frequency had a\nlarger right lateral Heschl’s gyrus. In addition, MEG responses recorded from a source\nestimated as lateral Heschl’s gyrus showed a similar asymmetry between hemispheres.\nStronger responses were obtained from the left hemisphere for subjects with a bias towards\nusing fundamental frequency in pitch discrimination. In contrast, subjects with a bias\ntowards using spectrum frequency had larger MEG responses recorded from the right\nhemisphere.\nEvidence from non-human primate studies\nGiven the evidence for a pitch-processing center in human auditory cortex and the\nanatomical and physiological similarities of auditory cortex between humans and monkeys,\nit is reasonable to expect that monkeys possess a pitch-processing center in their auditory\ncortex as well. Schwarz and Tomlinson [51] searched for single-unit responses to the\nfundamental frequency of missing fundamental harmonic complex sounds in AI of three\nawake macaque monkeys previously trained on a pitch discrimination task, but failed to find\nany neurons responsive to the fundamental frequency centered at neuron’s characteristics\nfrequency (CF). Schwarz and Tomlinson concluded that pitch is either represented implicitly\nacross a population of neurons in AI or an explicit representation exists outside of AI.\nHowever, Fishman et al. [52] were unable to find an implicit representation for the missing\nfundamental in AI based on population neuronal responses, using multi-unit recordings in\nawake macaque monkeys.\nIn a recent study in awake marmoset monkeys (a New World primate species) [7], Bendor\nand Wang searched for single-unit responses to the missing fundamental in AI and\nsurrounding non-primary areas and identified a restricted region anterolateral to AI\ncontaining pitch-selective neurons. Neurons were identified as pitch-selective if they\nresponded to missing fundamental sounds (with harmonics outside the neuron’s excitatory\nfrequency response area) and pure tones with a similar pitch. A typical pitch-selective\nneuron responded to an array of spectrally dissimilar sounds (harmonic complex tones, click\ntrains, iterated ripple noise) when the pitch was near the neuron’s preferred fundamental\nfrequency, which was found similar to the neuron’s CF determined by pure tones. Relative\nto AI, this newly identified region containing pitch-selective neurons in marmoset monkeys\nis in a similar location to the pitch-processing center found in humans [2-6,8-10] (Figure 2).\nIn addition, Bendor and Wang [7] found that pitch-selective neurons preferred temporally\nregular sounds and were sensitive to pitch salience changes due to harmonic frequency and\norder, in agreement with the imaging studies by Patterson et al. [3] and Penagos et al. [4].\nThe microelectrode recording study by Bendor and Wang [7] also showed that the pitch-\nprocessing region found in marmosets contained non-pitch-selective neurons (spanning a\nsimilar range of CFs) that responded to the spectral frequency of the sound, rather than the\nfundamental frequency. Potentially, these two classes of neurons co-localized within the\npitch-processing center respectively encode spectral and missing fundamental pitch\npercepts. For ambiguous pitch changes, in which the fundamental frequency and spectrum\nshift in opposite directions, as in the study by Schneider et al. [6], these two types of neurons\nwould provide conflicting information. Unequal weighting towards one of these neuron\ntypes within the pitch-processing center (and/or between hemispheres) may be the cause of a\nsubject’s perceptual bias of hearing pitch changes based on the fundamental or spectral\nfrequency. No topography has yet been identified within the primate’s pitch-processing\ncenter in above cited studies [2-10], possibly due to spatial resolution constraints.\nMany marmoset AI neurons located outside the pitch-processing region are tuned to the\nmodulation frequency (repetition rate) of an amplitude- or frequency-modulated sound.\nHowever, in sharp contrast to pitch-selective neurons, these responses require that the\nacoustic signal’s spectral components be within the neuron’s frequency response area [57].\nIn awake marmoset AI, neural responses to stimulus repetition rates below ~40 Hz (near the\nlower limit of pitch [61,62]) are represented temporally by stimulus-synchronized\ndischarges, while a monotonically tuned rate code is used by another population of neurons\nto represent higher repetition rates [58]. Neurons with harmonically related multi-peaked\nfrequency response areas have previously been observed in marmoset AI [59], although they\ndiffer from pitch-selective neurons in that their primary frequency tuning and harmonic\nresponses are most commonly outside the frequency range of pitch perception, and that they\nrespond to each component of a complex harmonic tone. Together, the findings from\nmarmoset auditory cortex suggest that pitch-processing is a specialized function by a\nsubpopulation of neurons in a restricted region of non-primary auditory cortex, whereas\nprocessing of temporally modulated or harmonically rich signals is a general function by\nother auditory neurons located across the tonotopic axis.\nDifferences between observations from studies in primate and non-primate species\nSeveral previous studies investigated how missing fundamental sounds are represented in\nthe auditory cortex of non-primates species. Microelectrode recordings in gerbils suggested\nthat AI neurons could respond to the periodicity of amplitude-modulated tones with the\nspectral components located outside neuron’s excitatory frequency response area [53], in\ncontrast to a putative non-primary auditory cortical area encoding pitch in primates [2-10].\nSchulze et el. [54] has also found a semi-circularly shaped map of best fundamental\nfrequency in gerbil auditory cortex using optical imaging techniques. Differences in the\nresults from these studies in primate and non-primate species could be due to an\nevolutionary divergence of pitch-processing strategies within auditory cortex, or may result\nfrom methodological differences. Combination tones at the fundamental frequency are\nproduced within the cochlea by missing fundamental sounds, creating a potential ambiguity\nregarding whether a missing fundamental or combination tone is the source of the neuron’s\nevoked response [1]. The use of appropriate sound levels (to ensure that combination tones\nare below a neuron’s pure tone response threshold) [7], as well as the use of noise maskers\n[4,7] or a cancellation tone [55] are among the necessary control conditions to confirm that a\ntrue missing fundamental response is observed.\n6. Further questions on pitch processing in auditory cortex\nThe discovery of a pitch-processing center in human (Figure 2A) and monkey (Figure 2B)\nauditory cortex is only the first step in understanding the physiological mechanisms of pitch\nperception. Several important questions remain unanswered. First, what is the source of\ninputs to pitch-selective neurons (corticocortical or thalamocortical)? The location of the\npitch center appears to be overlapping low-frequency portions of field R (primary-like) and\nlateral belt areas AL and ML. This suggests that the pitch-selective neurons may receive\ninputs from both ventral and dorsal divisions of the auditory thalamus (MGB) that respond\nto narrowband and wideband sounds, respectively [26]. This could be a possible source of\nthe response to pure tones and broadband missing fundamental sounds. Alternatively, given\nthe extensive connectivity between AI, R, and neighboring belt areas surrounding the pitch-\nprocessing center, the pitch-selective neurons may extract the fundamental frequency using\ninputs from neighboring cortical areas (including AI). Second, do pitch-selective neurons\nuse a spectral and/or temporal mechanism to extract the fundamental frequency of complex\nsounds? This is an issue that has been at the center of debate among auditory researchers for\nthe last over 60 years. Computational models and auditory nerve data support the possibility\nof both a purely temporal mechanism and a hybrid mechanism using both spectral and\ntemporal information [1,43-45]. In monkeys, AI neurons with temporal and spectral\nresponse properties potentially useful for these pitch models have been observed [56-59],\nbut whether they provide an input to pitch-selective neurons is unknown.",
         "",
         "Cortical representations of pitch in monkeys and humans",
         "None"
        ],
        [
         "14",
         "000a0840f6a41c30cd2b60f262f8167be74077da",
         "We reduce the problem of impulse elimination via state feedback in singular differential equations to algebra. Our results are developed for systems over an arbitrary Hermite domain. We show that the established theories for the time-invariant and the real analytic time-varying settings can be unified in this way. Besides the constant and real analytic functions, several other function rings are considered. Our algebraic theory is applied to these cases, providing solutions to the impulse elimination problem for classes of systems not previously studied. In particular, our work allows the restriction of the feedback matrix to certain function rings",
         "D. Cobb",
         "None",
         "http://www.ece.wisc.edu/~cobb/papers/SICON6(2).pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "download too long",
         "None",
         "Analysis and Numerical Solution of Control Problems in Descriptor Form;Singular Control Systems;Linear Systems over Commutative Rings;A survey of linear singular systems;Controllability, observability, and duality in singular systems;Linear Algebra over Commutative Rings;The Theory of Matrices;CANONICAL FORMS AND SOLVABLE SINGULAR SYSTEMS OF DIFFERENTIAL EQUATIONS;A further interpretation of inconsistent initial conditions in descriptor-variable systems;A generalized state-space for singular systems;Feedback and pole placement in descriptor variable Systems;Introduction to the Theory and Application of the Laplace Transformation;Generalizations of a theorem of doležal;The Theory of Matrices;Elementary divisors and modules;Functions of One Complex Variable;Regularization of Linear Descriptor Systems with Linear Coefficients;State feedback impulse elimination of linear time-varying singular systems;Control System Synthesis : A Factorization Approach;Singular systems of differential equations II;The theory of matrices;Functions of a Complex Variable",
         "State Feedback Impulse Elimination for Singular Systems over a Hermite Domain",
         "None"
        ],
        [
         "15",
         "000a1c397de7a66ef20202693e14cf9251d471d8",
         "In case‐control association studies unobserved population stratification may act as a confounder, leading to an increased number of false positive results. Methods accounting for population structure by using additional genetic markers broadly follow one of two concepts: Genomic Control (GC) and Structured Association (SA). While extending existing methods of Structured Association we show that it is necessary to incorporate phenotypic information when inferring population structure, otherwise a systematic bias is introduced. Moreover, for moderate population stratification a Wald test statistic should be preferred as a Structured Association test statistic in comparison to a likelihood ratio test. The introduced extensions are compared to existing methods of Structured Association, as well as to Genomic Control, in a simulation study which is based on realistic situations of large case‐control studies with moderate population stratification. A disadvantage of Genomic Control turns out to be the large variation in estimating the variance inflation factor, as well as the power loss if population structure increases. We come to the overall conclusion that Structured Association, if applied correctly, is superior to Genomic Control, at least in the case of simple population structure as simulated here.",
         "Karola Köhler,J. Pritchard",
         "Karola Köhler,J. Pritchard",
         "https://ediss.uni-goettingen.de/bitstream/11858/00-1735-0000-000D-F273-A/1/koehler.pdf",
         "",
         "",
         "",
         "",
         "",
         "downloading",
         "",
         "",
         "Case‐Control Association Tests Correcting for Population Stratification",
         "None"
        ],
        [
         "16",
         "000a5535c7f5a8b73a8ec93354a41153b87a5ab2",
         "ABSTRACT NK cells are key effectors of innate immunity and host survival during cytomegalovirus (CMV) infection. Innate murine CMV (MCMV) resistance in MA/My mice requires Ly49H/m157-independent H-2k-linked NK cell control. Here we show that replacement of MA/My H-2k with C57L H-2b susceptibility genes led to a remarkable loss of innate virus immunity, though NK gamma interferon was induced in H-2b and H-2k strains shortly after infection. Thus, H-2b genes expressed in C57L or MA/My.L-H2b are sufficient in alerting NK cells to intrusion but fail to support NK restraint of viral infection. In addition, novel H-2 recombinant strains were produced and utilized in a further refinement of a critical genetic interval controlling innate H-2k-linked MCMV resistance. Importantly, this analysis excluded the gene interval from Kk class I through class II. The responsible gene(s) therefore resides in an interval spanning Dk class Ia and more-distal major histocompatibility complex (MHC) nonclassical class Ib genes. Recently, the NK activation receptor Ly49P and MHC class I Dk proteins were genetically implicated in MCMV resistance, in part because Ly49P-expressing reporter T cells could specifically bind Dk molecules on MCMV-infected mouse embryonic fibroblasts (MEFs). However, as we found that H-2k innate resistance differs in the C57L or MA/My backgrounds and because MCMV very efficiently downregulates H-2k class I proteins in L929 cells and primary MEFs shortly after infection, a Ly49P/Dk model should not fully explain H-2k-linked MCMV resistance.",
         "Xuefang Xie,Abhijit S. Dighe,Patricia Clark,Pearl Sabastian,Sarah N. Buss,Michael G. Brown",
         "Xuefang Xie,Abhijit S. Dighe,Patricia Clark,Pearl Sabastian,Sarah N. Buss,Michael G. Brown",
         "https://europepmc.org/articles/pmc1797260?pdf=render",
         "NK cells are key effectors of innate immunity and host survival during cytomegalovirus (CMV) infection.\nInnate murine CMV (MCMV) resistance in MA/My mice requires Ly49H/m157-independent H-2k-linked NK\ncell control. Here we show that replacement of MA/My H-2k with C57L H-2b susceptibility genes led to a\nremarkable loss of innate virus immunity, though NK gamma interferon was induced in H-2b and H-2k strains\nshortly after infection. Thus, H-2b genes expressed in C57L or MA/My.L-H2b are sufﬁcient in alerting NK cells\nto intrusion but fail to support NK restraint of viral infection. In addition, novel H-2 recombinant strains were\nproduced and utilized in a further reﬁnement of a critical genetic interval controlling innate H-2k-linked\nMCMV resistance. Importantly, this analysis excluded the gene interval from Kk class I through class II. The\nresponsible gene(s) therefore resides in an interval spanning Dk class Ia and more-distal major histocompat-\nibility complex (MHC) nonclassical class Ib genes. Recently, the NK activation receptor Ly49P and MHC class\nI Dk proteins were genetically implicated in MCMV resistance, in part because Ly49P-expressing reporter T\ncells could speciﬁcally bind Dk molecules on MCMV-infected mouse embryonic ﬁbroblasts (MEFs). However,\nas we found that H-2k innate resistance differs in the C57L or MA/My backgrounds and because MCMV very\nefﬁciently downregulates H-2k class I proteins in L929 cells and primary MEFs shortly after infection, a\nLy49P/Dk model should not fully explain H-2k-linked MCMV resistance.",
         "DISCUSSION\nAn H-2k association with host survival following MCMV\ninfection was long ago recognized, though a mechanistic un-\nderstanding is still elusive. We show that dominant innate virus\nresistance was converted to profound susceptibility in MA/\nwhen H-2k is available, perhaps through augmentation of NK\nsensing, but this factor by itself is not sufﬁcient in innate virus\nresistance because C57L NK cells fail to thwart a comparable\nMCMV infection. We further show that proximal H-2 Kk class\nI and class II genes do not explain the observed genetic vari-\nation, since our novel H-2 recombinant strain is fully compe-\ntent to limit virus replication in the spleen at early times after\ninfection. Thus, a genetic locus distal to the MHC class II gene\nregion and overlapping with Dk class Ia and more-distal non-\nclassical class Ib genes should contain the responsible gene(s)\nrequired in H-2k-linked innate MCMV resistance through NK\ncells. Because multiple H-2 loci were previously implicated in\nthe control of MCMV-induced mortality (15), we were also\nprompted to study mortality in the novel strains. Recent pre-\nliminary studies with sublethal or lethal doses of MCMV have\nso far indicated similar mortality rates for our H-2k recombi-\nnant strain and littermate control tested as shown in Fig. 1E\n(X. Xie and M. G. Brown, unpublished data).\nTo further examine the role of NK cells in innate H-2k\ncontrol, IFN- was quantiﬁed for comparison in H-2k and H-2b\nstrains over a 3.5-d time course following infection. Deﬁcient\nNK control in C57L and M.L-H2b is not due to an intrinsic\ndefect in virus sensing, since active viral replication alerted\ntheir NK cells to enhance IFN- production to levels similar to\nthose for NK cells in MA/My or L.M-H2k shortly after infec-\ntion. Similar diminished responses in all strains were also\nnoted by later time points and did not directly correlate with\ninnate virus immunity. Whether this response is due to virus\nsensing through innate receptors (i.e., TLR3 and TLR9) lead-\ning to enhanced cytokine stimulation of NK function (4, 34) or\ndue to direct NK recognition of virus-infected targets is an\nimportant question. Recent reports indicate that NK IFN-\nrelease is severely impaired in mice without TLR9 or MyD88\nand that dendritic cell-derived interleukin 18 (IL-18) is also\nimportant (1, 9). Here we show similar numbers of NK cells\nproducing IFN- and comparable induction levels after\nMCMV infection; thus, these data suggest that an MCMV 3\nTLR9 3 MyD88 3 IL-18 axis in dendritic cells and an IL-18 3\nIFN- axis in NK cells during the nonspeciﬁc phase of NK\nactivation are fully competent in the strains tested in Fig. 3.\nNevertheless, H-2b-linked gene expression fails to support NK\ncontrol of viral replication on the C57L or MA/My genetic\nbackgrounds.\nWhile both MCMV and HCMV convergently acquired key\ngene products to manipulate MHC class I display (27), genetic\nvariance in NK control may not be altogether surprising, since\nthese cells readily respond to MHC-compatible targets with\ndeﬁcient class I expression. Because Wagner et al. noted be-\nfore that class I Kb molecules are somewhat refractory to\nMCMV control (38), we reasoned that class I gene- or allele-\nspeciﬁc regulation by MCMV might explain H-2k protection,\nat least in part. However, minor variations in class I display\nmight best be observed only under certain circumstances, since\nKk (decreased by 95%) and Dk (decreased by 50%) pro-\nteins were differently downregulated on MCMV-infected L929\ncells. Dk display, on the other hand, was affected more than Kk\nby MCMV in IFN--induced F1 MEFs, since a substantially\ngreater number of infected (GFP) cells continued to display\nKk proteins even after Dk had seemingly been lost from the cell\nsurface. IFN- may conceivably have an impact on Kk induc-\nMy.L-H2b congenic mice, since infectious MCMV freely rep-\nlicates to very high levels in M.L-H2b spleens within days after\ninfection. Thus, MA/My H-2k-linked loci are indeed necessary\ncomponents of innate viral immunity. This difference cannot\nbe explained by H-2-controlled differences in infectivity or\nreplication, as we observe similar MCMV growth kinetics in\nMEFs derived from embryos from resistant or susceptible\nstrains. Another intriguing and unexpected ﬁnding in the cur-\nrent report is seen in Fig. 1, since H-2k-linked resistance is fully\ncompetent in H-2 heterozygous or homozygous mice on the\nC57L background, whereas a single H-2k haplotype in H-2\nheterozygous mice on the MA/My background did not result in\nfull resistance. An additional polymorphic genetic factor(s) in\nC57L therefore further limits MCMV spread and replication\ntion greater than that of other class I proteins. Alternately,\nMCMV proteins known to bind and downregulate class I dis-\nplay may more avidly bind Dk when class I synthesis is strongly\ninduced by type I interferons. Taken together, our ﬁndings\nsuggest that only very minimal levels of Dk ligands that could\npotentially stimulate Ly49P receptors on NK cells in the course\nof MCMV infection should be available on infected host cells\nand that even fewer might be expected to hold a particular\nrelevant virus peptide, as suggested recently (10).\nAlternately, MCMV might stabilize some cell surface class I\ndisplays, including that of Dk proteins, while at the same time\ninterfering with cytotoxic T-lymphocyte recognition. MCMV\ngp34 is notable in this regard, since it binds endoplasmic re-\nticulum-resident class I proteins destined for expression at the\ncell surface. Interestingly, Dk but not Kk surface expression is\nselectively affected by gp34 in infected L929 cells (Xie and\nBrown, unpublished data). An intriguing possibility is that\nLy49P NK cells might speciﬁcally recognize gp34-associated\nDk proteins. However, innate H-2k resistance is likely more\ncomplex, since signiﬁcant protection has been observed even\non the BALB/c background, where Ly49P receptors are not\nexpressed (32). Finally, it is possible that NK recognition and\ncontrol become fully competent only in mice that express H-2k\nclass I ligands during development. While NK cells in C57L\nand M.L-H2b mice do recognize viral intrusion and IFN-\nexpression is stimulated shortly afterward, these cells could be\nhyporesponsive (14) or unlicensed (19) in terms of their ca-\npacity to attack infected cells. NK receptors in H-2k mice, on\nthe other hand, might recognize viral signatures more efﬁ-\nciently than those in other H-2 haplotypes, in addition to un-\ndergoing potential stimulation through Ly49P/Dk interactions\nduring NK encounters with MCMV-infected cells. While in-\nhibitory KIR/MHC interactions also insure functional compe-\ntency in human NK cells (2) and can regulate antiviral immune\nresponses in infected individuals (13, 18), this model should\nenhance our understanding of the role of NK cells in the early\ndetection and control of viral pathogens.",
         "Cytomegaloviruses (CMV) are betaherpesviruses that dis-\nplay species-speciﬁc tropism. Because human CMV (HCMV)\nand murine CMV (MCMV) cause severe infections in immu-\nnodeﬁcient or immunologically immature hosts and share\nmany biological features in their natural host settings, MCMV\nis an important model system for studies of innate viral immu-\nnity (27). Additionally, because NK cells provide critical innate\nimmune defenses in CMV infections (25), the MCMV infec-\ntion model is also useful for understanding the role of NK cells\nand their capacity to recognize virus targets.\nChalmer et al. noted a survival role for H-2k after high-dose\nMCMV infection in C3H, CBA, and BALB.K inbred mice (7).\nFurther study revealed similar levels of MCMV infection and\nreplication in mouse embryonic ﬁbroblasts (MEFs) of different\nH-2 haplotypes, including H-2b and H-2k (7). NK cells were\nalso implicated, since enhanced NK cytotoxicity is a general\nfeature of MCMV infection in more-resistant H-2k mouse\nstrains (3). Scalzo et al. (32) later documented a primary role\nfor NK cells in innate MCMV immunity in MA/My through\nimmunodepleting NK cells with anti-NK1.1 monoclonal anti-\nbody (mAb) before infection. Elsewhere, we conﬁrmed the\nimportance of NK cells in innate MA/My MCMV immunity\nand extended our previous ﬁndings by demonstrating that H-2k\nintrogression onto the C57L genetic background in C57L.M-\nH2k (L.M-H2k) congenic mice was sufﬁcient\nto convert\nMCMV susceptibility to resistance through NK control (11).\nAn epistatic genetic interaction between Cmv3 (Ly49P) and\nH-2k was recently noted, and because Ly49P-transduced re-\nporter T cells speciﬁcally bind class I Dk proteins on MCMV-\ninfected MEFs, the receptor-ligand interaction is implicated in\nMA/My resistance (10). However, other unknown genetic fac-\ntors clearly contribute, since H-2k substantially reduces spleen\nvirus replication (32) and mortality (3) even when expressed on\nthe BALB/c (Ly49P) background. Further study is thus re-\nquired to elucidate the mechanisms that govern the effectual\nNK recognition of MCMV-infected targets leading to innate\nmajor histocompatibility complex (MHC)-linked control\nin\nH-2k mice.\nBecause NK cells also express inhibitory receptors for MHC\nclass I molecules and their effector functions are determined\nthrough integration of stimulatory and inhibitory signals (22),\nan alternate hypothesis also involves NK recognition of MHC\nclass I deﬁciency, as often occurs in viral infection. Cell surface\nexpression of H-2b and H-2d class I proteins are speciﬁcally\nand dramatically altered by MCMV in vitro (28, 38, 42).\nMCMV gp48 hijacks nascent MHC class I proteins for lysoso-\nmal degradation (28), gp40 blocks endoplasmic reticulum-\nGolgi MHC class I export (42), and gp34 escorts class I\nmolecules to the cell surface to interfere with cytotoxic T-\nlymphocyte recognition (17, 20). Although the mechanisms are\ndistinct, HCMV likewise utilizes several gene products to spe-\nciﬁcally alter HLA class I expression (27). However, HCMV\nalso stabilizes HLA-E proteins with class I signal peptide-\nrelated UL40 leader peptide (35), presumably to bind inhibi-\ntory CD94/NKG2A receptors on NK cells and evade attack (5,\n37, 39). Additionally, HCMV utilizes UL141 and UL142 to\ndirectly interfere with NK recognition and attack (36, 41).\nTaken together, the data show that H-2k resistance may re-\nquire altered MHC class I or class I-related protein recognition\nthrough inhibitory receptor sensing in addition to direct stim-\nulation.\nTo assess the role of H-2k in MCMV resistance, we replaced\nit in MA/My mice with the H-2b susceptibility locus of C57L.\nWe examined MCMV infectivity and replication in MEFs de-\nrived from the resistant or susceptible strains, and innate\nMCMV immunity phenotypes were also analyzed. To also ex-\namine the role of NK cells, intracellular gamma interferon\n(IFN-) was quantiﬁed for comparison in H-2k and H-2b\nstrains over a 3.5-day (d) time course following infection. Fur-\nther, since the regulation of H-2k class I proteins by MCMV\nhas not been previously characterized, we also examined Kk\nand Dk class I proteins on L929 cells and primary (C57L \nMA/My)F1 MEFs shortly after virus infection. We provide\nevidence indicating that a Ly49P/Dk model is inadequate to\nfully account for MA/My MCMV resistance.\nMice. MA/My and C57L breeder pairs were purchased from The Jackson\nLaboratory (Bar Harbor, ME) and housed in the MR-5 speciﬁc-pathogen-free\nvivarium at the University of Virginia, which is fully accredited by the American\nAssociation for Accreditation of Laboratory Animal Care. A simple sequence\nlength polymorphism (SSLP) marker-assisted genome selection method was\nused to produce MA/My.L-H2b (M.L-H2b) mice as we have described previously\n(11, 30, 33). Brieﬂy, (C57L  MA/My)F1 hybrid mice were ﬁrst backcrossed to\nMA/My. Backcross offspring were selected for H-2b-linked D17Mit16 and\nD17Mit10 donor alleles on a genetic background enriched for MA/My alleles.\nSelected backcross mice were then backcrossed to MA/My, and further selective\nbackcrossing followed by brother-sister crossing for H-2b homozygosity yielded\nthe M.L-H2b (N6 [sixth backcross generation]) mice tested as shown in Fig. 1.\nNotably, homozygous M.L-H2b (N9) mice display comparable MCMV suscepti-\nbilities (not shown). (C57L  MA/My)F1 hybrids, L.M-H2k (N7), and MA/My.L-\nH2bR1 (M.L-H2bR1 [N9]) recombinant congenic mice were bred in the same\nfacility. All animal studies were approved by and conducted in accordance with\nAnimal Care and Use Committee oversight.\nCell lines and tissue culture. NIH 3T12 (ATCC CCL-164), NIH 3T3 (ATCC\nCRL1658), and L929 (ATCC CCL-1, H-2k) were grown in Dulbecco’s modiﬁed\nEagle medium supplemented with 10% (vol/vol) newborn calf serum, penicillin-\nstreptomycin (100 U/ml:100 g/ml), and glutamine (2.0 mM). C57L, MA/My,\n(C57L  MA/My)F1, and C57L.M-H2k MEFs were prepared on d 16 to 18\npostcoitus as described previously (31). After expansion in Dulbecco’s modiﬁed\nEagle medium plus 10% fetal bovine serum in T175 ﬂasks, passage 2 MEFs were\nstored under liquid nitrogen. Only passage 2 to 5 MEFs were utilized in exper-\niments. For IFN induction of MHC class I cell surface expression, MEFs were\nincubated in recombinant mouse IFN- (Research Diagnostics, Inc.) or IFN-\n(Fitzgerald Industries, Concord, MA) for 24 to 30 h before virus infection.\nAntibodies and ﬂow cytometry. Anti-mouse CD3ε (145-2C11) peridinin chlo-\nrophyll protein (PerCP), CD49b (DX5) phycoerythrin (PE), H-2Db (28-14-8)\nPE, H-2-Kb-Db (20-8-4S) biotinylated, control immunoglobulin G2a (IgG2a)\n(eBM2a), and rat anti-mouse CD71 and IgG2a were purchased from eBio-\nscience. Anti-mouse NK1.1 (PK136) PE, IFN- ﬂuorescein isothiocyanate,\nH-2Kk (36-7-5) PE, H-2Dk (15-5-5) biotinylated and H-2Kb (AF6-88.5) PE, and\nstreptavidin-PerCP were purchased from BD Pharmingen. Anti-mouse H-2Dk\n(15-5-5) PE was purchased from BioLegend. Anti-H-2-Kb-Db (20-8-4S; kindly\nprovided by V. Engelhard, University of Virginia, Charlottesville, VA) antibod-\nies were puriﬁed from spent hybridoma supernatant by use of a HiTrap Protein\nG Hp column (Amersham Pharmacia Biotech, Sweden) and subsequently bio-\ntinylated using EZ-Link sulfo-N-hydroxysuccinimide–LC–biotin, 2-(4-hydroxy-\nazobenzene) benzoic acid (HABA), and avidin (Pierce). All antibody staining\nwas performed on ice. For analysis of spleen leukocytes, cells were ﬁrst blocked\nwith mAb 2.4G2 (5 g/ml) on ice for 30 min before primary antibody staining.\nFor intracellular IFN- staining, cells were incubated for 2 h at 37°C in media\ncontaining either monensin (2.5 M) or brefeldin A (5 g/ml). Afterwards, cells\nwere ﬁrst stained for cell surface markers and then ﬁxed and permeabilized using\na CytoFix/CytoPerm kit (BD Pharmingen). Labeled cells were analyzed by ﬂow\ncytometry on a FACScan instrument (BD Biosciences), and data were subse-\nquently analyzed using FlowJo (TreeStar, version 4.3.1).\nVirus assays. MCMV (Smith strain, ATCC VR 194) salivary gland stock virus\nwas prepared after serial passage in BALB/c as described previously (29). Ex-\nperimental mice (8 to 12 weeks of age) were intraperitoneally infected with\nMCMV (1  105 PFU). On d 3.5 postinfection (84 to 90 h), spleen and liver virus\nlevels were quantiﬁed using quantitative real-time PCR (QPCR) as described\npreviously (29, 40). All sample measurements were performed in triplicate.\nSpleen and liver homogenate infectious MCMV PFU were also measured on\nNIH 3T3 monolayers essentially as described previously (6, 40), but with a slight\nmodiﬁcation. Brieﬂy, serially 10-fold-diluted tissue homogenates (100 l) were\nplated on NIH 3T3 cell monolayers (70 to 90% conﬂuent) in duplicate in 24-well\ntissue culture plates for 1 h at 37°C with frequent rocking. Afterwards, MCMV-\ninfected cell monolayers were fully resuspended in D10 medium (1 ml). Virus\nplaques were counted microscopically on d 3. In conﬁrmation, infectious virus\nparticles can be efﬁciently and reliably distributed through limiting dilution based\non viral titers determined using this strategy. In particular, when NIH 3T3\nmonolayers are infected with 0.67 PFU per well (10 wells of a 24-well plate), 20\nto 70% of wells contain at least one plaque by d 3 after cellular infection. Most\nwells without viral plaques on d 3 after infection remain plaque free through d 7\nafter infection.\nFor in vitro MCMV infectivity and replication studies, MEFs growing in\n24-well plates were infected with MCMV (multiplicity of infection [MOI] \n0.08). Supernatants were collected daily through d 8 and subsequently titered on\n3T12 cells. DNA was also isolated from the remaining cell monolayers by use of\na Puregene DNA isolation kit as described previously (34). Recombinant green\nﬂuorescent protein (GFP)-expressing m157-MCMV (11), wild-type K181, and\nSmith strain MCMV viruses were utilized after passage and thorough titer\ndeterminations with NIH 3T3 cells. Cell lines or primary MEFs were infected\nwith a range of MCMV PFU (MOI, 0.01 to 10).",
         "",
         "RESULTS\nH-2b linked genes fail to support innate MCMV immunity\nthrough NK cells. We previously noted that multiple genes\ncontribute vital Ly49H-independent immunity shortly after\nMCMV infection in New Zealand White or MA/My mice (11,\n29). Because H-2k-linked genes were shown to contribute sub-\nstantially to MA/My virus resistance (10, 11) and also con-\nverted susceptibility to full resistance in C57L.M-H2k congenic\nmice (11), we examined whether they are also essential to\ninnate immunity in this model system. We replaced H-2k in\nMA/My with H-2b from MCMV-susceptible C57L by use of\nSSLP marker-assisted selection to obtain H-2b donor alleles on\na MA/My background (Fig. 1A). H-2b introgression was con-\nﬁrmed using allele-speciﬁc anti-MHC class I mAbs to stain\nM.L-H2b splenocytes (not shown).\nWe assessed innate MCMV immunity in M.L-H2b by mea-\nsuring virus levels in spleens and livers. These mice displayed\nremarkable susceptibility that was even more severe than that\nof C57L (Fig. 1B and C). By comparison, similarly infected\nL.M-H2k/b and control MA/My mice were fully protected by\nH-2k. Thus, H-2k-linked gene expression is essential for innate\nMCMV protection through NK cells in MA/My, whereas H-2b\nlinked gene expression is not adequate for effective NK con-\ntrol. While previous work indicated that an additional genetic\nfactor beyond the MHC or the NK gene complex can contrib-\nute to innate MCMV control (11), we also evaluated immunity\nin (MA/My  C57L)F1 hybrids and their heterozygous con-\ngenics, M.L-H2b/k mice (Fig. 1D). As expected, virus levels in\nlittermate control (H-2k) spleens were comparable with those\nfor MA/My. Intriguingly, however, virus levels in M.L-H2b/k\nspleens were actually intermediate between those of the pro-\ngenitor strains. This ﬁnding indicates that H-2k is not fully\nprotective on the MA/My genetic background. Instead, H-2k\nhomozygosity is required to fully establish innate MCMV re-\nsistance on the MA/My background. Moreover, since differ-\nences in MCMV control in H-2 heterozygous mice strictly\ncorrelated with their genetic background (compare L.M-H2k/b\nand M.L-H2b/k in Fig. 1B and D, respectively), an additional\nfactor(s) coming from C57L must increase the extent of innate\nvirus resistance afforded by the H-2k haplotype.\nTo further delimit an H-2k-linked critical region, novel H-2\nrecombinant mice were produced from the M.L-H2b and L.M-\nH2k strains. One strain, M.L-H2bR1, was identiﬁed as contain-\ning an informative recombination breakpoint between the H-2\nK and D genes. Importantly, this strain retains the expression\nof Kb and C57L-derived MHC class II proteins (not depicted)\nbut also expresses MA/My-derived class I Dk molecules. We\ncrossed (MA/My  M.L-H2bR1)N9 backcross mice with C57L\nand studied MCMV control in their hybrid offspring. As ex-\npected, a 1:1 Mendelian distribution of homozygous (Kb to\nI-Eb region) and fully heterozygous (H-2k,b) progeny on F1\ngenetic backgrounds was observed. The corresponding innate\nMCMV immunities in the two groups shown in Fig. 1E indi-\ncate that Kk expression is not needed to limit MCMV in the\nspleen. The H-2k-linked critical region should therefore reside\ndistal to SSLP marker 17Uva12 and the class II genes while\noverlapping Dk class I and more-distal H-2 nonclassical class Ib\ngenes.\nSimilar levels of MCMV infectivity and replication in sus-\nceptible C57L and resistant C57L.M-H2k strain-derived MEFs.\nThough previous work has shown that MCMV infectivity and\nreplication kinetics are similar in H-2b- and H-2k-derived\nMEFs (7), Harnett and Shellam (16) noted high MCMV sus-\nceptibility in BALB/c (H-2d) and B6 (H-2b) strain-derived\nMEFs compared with that for MEFs from H-2k strains C3H\nand CBA. Thus, we also analyzed viral growth kinetics in\nMEFs prepared from embryos of MCMV-resistant and -sus-\nceptible strains. Figure 2A shows that MCMV grew exponen-\ntially from d 1 to d 5 with similar kinetics in MEFs derived\nfrom either strain. In addition, comparable levels of infectious\nvirions were released into the MEF supernatants after d 2 and\nthrough d 6. Furthermore, we did not observe signiﬁcant dif-\nferences in viral replication, plaque size, or viral cytopathic\neffect in comparison with MA/My MEFs (not shown). In\nagreement with Chalmer et al. (7), we conclude that H-2-\nlinked genetic differences in innate MCMV immunity are not\ncontrolled at the level of MCMV infectivity or replication\nwithin host cells.\nNK cell effector functions in M.L-H-2b and L.M-H-2k. To\ndelineate whether an inherent defect in H-2b NK virus sensing\nmight explain an inability to restrain viral replication, we ex-\namined NK IFN- shortly after infection, as this cytokine is\nrapidly induced in NK cells of MCMV-resistant C57BL/6 mice\nduring MCMV infection (26). Spleen cellularity generally cor-\nresponded with either the MA/My or the C57L genetic back-\nground through the ﬁrst 39 h of infection (Fig. 3). By 90 h after\ninfection, however, splenocyte numbers dropped precipitously\nin homozygous H-2b strains and also in M.L-H2b/k littermate\ncontrols (Fig. 3A). Thus, splenocyte numbers were not abso-\nlutely correlated with high or low virus levels, but in mice\nwithout the H-2k haplotype, substantial losses were always\nnoted. Also consistent with previous ﬁndings (8, 11, 12), NK\ncell numbers drop similarly through the ﬁrst 39 h after infec-\ntion in resistant and susceptible strains (not depicted). More-\nover, similar numbers of spleen NK cells were producing\nIFN- with maximal induction in strains studied by the same\ntime point (Fig. 3B). Notably, NK cells did not produce IFN-\nafter injection of heat- or UV-inactivated MCMV into the\nH-2k or H-2b strains under study (not depicted). After 90 h,\nspleen NK cell IFN- levels were diminished considerably in\nresistant and susceptible mice and did not correlate with\nMCMV control traits.\nRole of H-2k class I expression in MCMV infection. H-2-\nlinked control\nin innate MCMV immunity suggested that\nMHC class I molecules might have a role in NK cell control. As\nregulation of Kk and Dk class I proteins by MCMV has not\nbeen previously characterized, we also examined their expres-\nsion on L929 (H-2k) cells and primary (C57L  MA/My)F1\nMEFs shortly after virus infection. We utilized a reporter\nMCMV virus (MCMV-m157/EGFP [11]) to directly visualize\ninfected cells by ﬂow cytometry. Although more virus is re-\nquired to infect L929 (H-2k) than NIH 3T3 cells, ﬂow cyto-\nmetric analysis revealed that 24% of L929 cells (MOI  10)\nwere GFP MCMV-infected cells by 24 h after infection (not\nshown). Under these conditions, Kk and Dk proteins on L929\ncells were signiﬁcantly decreased only on infected (GFP) cells\n(Fig. 4A). Similar decreases in class I display were observed\nwhen L929 cells were infected with the wild-type K181 strain\n(not depicted). Fine differences in Kk or Dk proteins on\nMCMV-infected L929 prompted us to further investigate po-\ntential gene- or allele-speciﬁc viral regulation of MHC class I\nexpression. We measured class I expression on primary (C57L \nMA/My)F1 MEFs as an independent means to establish that\nthe diminishment of class I proteins of either haplotype is a\ngeneral feature of MCMV infection. Due to low constitutive\nclass I levels on the MEFs, IFN- induction was utilized to\nboost expression before MCMV infection. In the same way, we\nfound that MCMV downregulated Kk, Dk, and H-2b class I\nmolecules by 90% on F1 MEFs (Fig. 4B). MCMV control of\nclass I expression is speciﬁc, since CD71, a common surface\nprotein, was not affected (Fig. 4C). We obtained similar results\nusing Smith strain MCMV (data not shown). H-2b and H-2k\nclass I proteins are therefore efﬁciently and speciﬁcally de-\ncreased by MCMV on cells expressing alleles of either haplo-\ntype.\nBecause type I interferons are indispensable in innate\nimmunity and MCMV resistance, we also studied class I down-\nregulation by MCMV on IFN--stimulated F1 MEFs. Though\nIFN- directly affected the percentage of cells infected in a\ndose-dependent manner, presumably through direct inhibition\nof viral gene expression (24), MCMV again efﬁciently de-\ncreased H-2k and H-2b class I proteins on F1 MEFs (Fig. 5A).\nAdditionally, though uninfected F1 MEFs have similar levels of\nKk and Dk display, the amount of Dk class I proteins was\nconsistently less than half that of Kk proteins on infected cells.\nSimilar results were obtained over a large range of IFN-\ndoses (Fig. 5B). Furthermore, Dk mean ﬂuorescence intensity\ndecreased sharply, whereas Kk mean ﬂuorescence intensity\nlevels remained fairly constant throughout the study, indicating\nthat MCMV infection under certain conditions can selectively\nregulate MHC class I proteins in a gene-speciﬁc manner.",
         "downloading",
         "NK cells are key effectors of innate immunity and host survival during cytomegalovirus (CMV) infection.\nInnate murine CMV (MCMV) resistance in MA/My mice requires Ly49H/m157-independent H-2k-linked NK\ncell control. Here we show that replacement of MA/My H-2k with C57L H-2b susceptibility genes led to a\nremarkable loss of innate virus immunity, though NK gamma interferon was induced in H-2b and H-2k strains\nshortly after infection. Thus, H-2b genes expressed in C57L or MA/My.L-H2b are sufﬁcient in alerting NK cells\nto intrusion but fail to support NK restraint of viral infection. In addition, novel H-2 recombinant strains were\nproduced and utilized in a further reﬁnement of a critical genetic interval controlling innate H-2k-linked\nMCMV resistance. Importantly, this analysis excluded the gene interval from Kk class I through class II. The\nresponsible gene(s) therefore resides in an interval spanning Dk class Ia and more-distal major histocompat-\nibility complex (MHC) nonclassical class Ib genes. Recently, the NK activation receptor Ly49P and MHC class\nI Dk proteins were genetically implicated in MCMV resistance, in part because Ly49P-expressing reporter T\ncells could speciﬁcally bind Dk molecules on MCMV-infected mouse embryonic ﬁbroblasts (MEFs). However,\nas we found that H-2k innate resistance differs in the C57L or MA/My backgrounds and because MCMV very\nefﬁciently downregulates H-2k class I proteins in L929 cells and primary MEFs shortly after infection, a\nLy49P/Dk model should not fully explain H-2k-linked MCMV resistance.\nCytomegaloviruses (CMV) are betaherpesviruses that dis-\nplay species-speciﬁc tropism. Because human CMV (HCMV)\nand murine CMV (MCMV) cause severe infections in immu-\nnodeﬁcient or immunologically immature hosts and share\nmany biological features in their natural host settings, MCMV\nis an important model system for studies of innate viral immu-\nnity (27). Additionally, because NK cells provide critical innate\nimmune defenses in CMV infections (25), the MCMV infec-\ntion model is also useful for understanding the role of NK cells\nand their capacity to recognize virus targets.\nChalmer et al. noted a survival role for H-2k after high-dose\nMCMV infection in C3H, CBA, and BALB.K inbred mice (7).\nFurther study revealed similar levels of MCMV infection and\nreplication in mouse embryonic ﬁbroblasts (MEFs) of different\nH-2 haplotypes, including H-2b and H-2k (7). NK cells were\nalso implicated, since enhanced NK cytotoxicity is a general\nfeature of MCMV infection in more-resistant H-2k mouse\nstrains (3). Scalzo et al. (32) later documented a primary role\nfor NK cells in innate MCMV immunity in MA/My through\nimmunodepleting NK cells with anti-NK1.1 monoclonal anti-\nbody (mAb) before infection. Elsewhere, we conﬁrmed the\nimportance of NK cells in innate MA/My MCMV immunity\nand extended our previous ﬁndings by demonstrating that H-2k\nintrogression onto the C57L genetic background in C57L.M-\nH2k (L.M-H2k) congenic mice was sufﬁcient\nto convert\nMCMV susceptibility to resistance through NK control (11).\nAn epistatic genetic interaction between Cmv3 (Ly49P) and\nH-2k was recently noted, and because Ly49P-transduced re-\nporter T cells speciﬁcally bind class I Dk proteins on MCMV-\ninfected MEFs, the receptor-ligand interaction is implicated in\nMA/My resistance (10). However, other unknown genetic fac-\ntors clearly contribute, since H-2k substantially reduces spleen\nvirus replication (32) and mortality (3) even when expressed on\nthe BALB/c (Ly49P) background. Further study is thus re-\nquired to elucidate the mechanisms that govern the effectual\nNK recognition of MCMV-infected targets leading to innate\nmajor histocompatibility complex (MHC)-linked control\nin\nH-2k mice.\nBecause NK cells also express inhibitory receptors for MHC\nclass I molecules and their effector functions are determined\nthrough integration of stimulatory and inhibitory signals (22),\nan alternate hypothesis also involves NK recognition of MHC\nclass I deﬁciency, as often occurs in viral infection. Cell surface\nexpression of H-2b and H-2d class I proteins are speciﬁcally\nand dramatically altered by MCMV in vitro (28, 38, 42).\nMCMV gp48 hijacks nascent MHC class I proteins for lysoso-\nmal degradation (28), gp40 blocks endoplasmic reticulum-\nGolgi MHC class I export (42), and gp34 escorts class I\nmolecules to the cell surface to interfere with cytotoxic T-\nlymphocyte recognition (17, 20). Although the mechanisms are\ndistinct, HCMV likewise utilizes several gene products to spe-\nciﬁcally alter HLA class I expression (27). However, HCMV\nalso stabilizes HLA-E proteins with class I signal peptide-\nrelated UL40 leader peptide (35), presumably to bind inhibi-\ntory CD94/NKG2A receptors on NK cells and evade attack (5,\n37, 39). Additionally, HCMV utilizes UL141 and UL142 to\ndirectly interfere with NK recognition and attack (36, 41).\nTaken together, the data show that H-2k resistance may re-\nquire altered MHC class I or class I-related protein recognition\nthrough inhibitory receptor sensing in addition to direct stim-\nulation.\nTo assess the role of H-2k in MCMV resistance, we replaced\nit in MA/My mice with the H-2b susceptibility locus of C57L.\nWe examined MCMV infectivity and replication in MEFs de-\nrived from the resistant or susceptible strains, and innate\nMCMV immunity phenotypes were also analyzed. To also ex-\namine the role of NK cells, intracellular gamma interferon\n(IFN-) was quantiﬁed for comparison in H-2k and H-2b\nstrains over a 3.5-day (d) time course following infection. Fur-\nther, since the regulation of H-2k class I proteins by MCMV\nhas not been previously characterized, we also examined Kk\nand Dk class I proteins on L929 cells and primary (C57L \nMA/My)F1 MEFs shortly after virus infection. We provide\nevidence indicating that a Ly49P/Dk model is inadequate to\nfully account for MA/My MCMV resistance.\nMice. MA/My and C57L breeder pairs were purchased from The Jackson\nLaboratory (Bar Harbor, ME) and housed in the MR-5 speciﬁc-pathogen-free\nvivarium at the University of Virginia, which is fully accredited by the American\nAssociation for Accreditation of Laboratory Animal Care. A simple sequence\nlength polymorphism (SSLP) marker-assisted genome selection method was\nused to produce MA/My.L-H2b (M.L-H2b) mice as we have described previously\n(11, 30, 33). Brieﬂy, (C57L  MA/My)F1 hybrid mice were ﬁrst backcrossed to\nMA/My. Backcross offspring were selected for H-2b-linked D17Mit16 and\nD17Mit10 donor alleles on a genetic background enriched for MA/My alleles.\nSelected backcross mice were then backcrossed to MA/My, and further selective\nbackcrossing followed by brother-sister crossing for H-2b homozygosity yielded\nthe M.L-H2b (N6 [sixth backcross generation]) mice tested as shown in Fig. 1.\nNotably, homozygous M.L-H2b (N9) mice display comparable MCMV suscepti-\nbilities (not shown). (C57L  MA/My)F1 hybrids, L.M-H2k (N7), and MA/My.L-\nH2bR1 (M.L-H2bR1 [N9]) recombinant congenic mice were bred in the same\nfacility. All animal studies were approved by and conducted in accordance with\nAnimal Care and Use Committee oversight.\nCell lines and tissue culture. NIH 3T12 (ATCC CCL-164), NIH 3T3 (ATCC\nCRL1658), and L929 (ATCC CCL-1, H-2k) were grown in Dulbecco’s modiﬁed\nEagle medium supplemented with 10% (vol/vol) newborn calf serum, penicillin-\nstreptomycin (100 U/ml:100 g/ml), and glutamine (2.0 mM). C57L, MA/My,\n(C57L  MA/My)F1, and C57L.M-H2k MEFs were prepared on d 16 to 18\npostcoitus as described previously (31). After expansion in Dulbecco’s modiﬁed\nEagle medium plus 10% fetal bovine serum in T175 ﬂasks, passage 2 MEFs were\nstored under liquid nitrogen. Only passage 2 to 5 MEFs were utilized in exper-\niments. For IFN induction of MHC class I cell surface expression, MEFs were\nincubated in recombinant mouse IFN- (Research Diagnostics, Inc.) or IFN-\n(Fitzgerald Industries, Concord, MA) for 24 to 30 h before virus infection.\nAntibodies and ﬂow cytometry. Anti-mouse CD3ε (145-2C11) peridinin chlo-\nrophyll protein (PerCP), CD49b (DX5) phycoerythrin (PE), H-2Db (28-14-8)\nPE, H-2-Kb-Db (20-8-4S) biotinylated, control immunoglobulin G2a (IgG2a)\n(eBM2a), and rat anti-mouse CD71 and IgG2a were purchased from eBio-\nscience. Anti-mouse NK1.1 (PK136) PE, IFN- ﬂuorescein isothiocyanate,\nH-2Kk (36-7-5) PE, H-2Dk (15-5-5) biotinylated and H-2Kb (AF6-88.5) PE, and\nstreptavidin-PerCP were purchased from BD Pharmingen. Anti-mouse H-2Dk\n(15-5-5) PE was purchased from BioLegend. Anti-H-2-Kb-Db (20-8-4S; kindly\nprovided by V. Engelhard, University of Virginia, Charlottesville, VA) antibod-\nies were puriﬁed from spent hybridoma supernatant by use of a HiTrap Protein\nG Hp column (Amersham Pharmacia Biotech, Sweden) and subsequently bio-\ntinylated using EZ-Link sulfo-N-hydroxysuccinimide–LC–biotin, 2-(4-hydroxy-\nazobenzene) benzoic acid (HABA), and avidin (Pierce). All antibody staining\nwas performed on ice. For analysis of spleen leukocytes, cells were ﬁrst blocked\nwith mAb 2.4G2 (5 g/ml) on ice for 30 min before primary antibody staining.\nFor intracellular IFN- staining, cells were incubated for 2 h at 37°C in media\ncontaining either monensin (2.5 M) or brefeldin A (5 g/ml). Afterwards, cells\nwere ﬁrst stained for cell surface markers and then ﬁxed and permeabilized using\na CytoFix/CytoPerm kit (BD Pharmingen). Labeled cells were analyzed by ﬂow\ncytometry on a FACScan instrument (BD Biosciences), and data were subse-\nquently analyzed using FlowJo (TreeStar, version 4.3.1).\nVirus assays. MCMV (Smith strain, ATCC VR 194) salivary gland stock virus\nwas prepared after serial passage in BALB/c as described previously (29). Ex-\nperimental mice (8 to 12 weeks of age) were intraperitoneally infected with\nMCMV (1  105 PFU). On d 3.5 postinfection (84 to 90 h), spleen and liver virus\nlevels were quantiﬁed using quantitative real-time PCR (QPCR) as described\npreviously (29, 40). All sample measurements were performed in triplicate.\nSpleen and liver homogenate infectious MCMV PFU were also measured on\nNIH 3T3 monolayers essentially as described previously (6, 40), but with a slight\nmodiﬁcation. Brieﬂy, serially 10-fold-diluted tissue homogenates (100 l) were\nplated on NIH 3T3 cell monolayers (70 to 90% conﬂuent) in duplicate in 24-well\ntissue culture plates for 1 h at 37°C with frequent rocking. Afterwards, MCMV-\ninfected cell monolayers were fully resuspended in D10 medium (1 ml). Virus\nplaques were counted microscopically on d 3. In conﬁrmation, infectious virus\nparticles can be efﬁciently and reliably distributed through limiting dilution based\non viral titers determined using this strategy. In particular, when NIH 3T3\nmonolayers are infected with 0.67 PFU per well (10 wells of a 24-well plate), 20\nto 70% of wells contain at least one plaque by d 3 after cellular infection. Most\nwells without viral plaques on d 3 after infection remain plaque free through d 7\nafter infection.\nFor in vitro MCMV infectivity and replication studies, MEFs growing in\n24-well plates were infected with MCMV (multiplicity of infection [MOI] \n0.08). Supernatants were collected daily through d 8 and subsequently titered on\n3T12 cells. DNA was also isolated from the remaining cell monolayers by use of\na Puregene DNA isolation kit as described previously (34). Recombinant green\nﬂuorescent protein (GFP)-expressing m157-MCMV (11), wild-type K181, and\nSmith strain MCMV viruses were utilized after passage and thorough titer\ndeterminations with NIH 3T3 cells. Cell lines or primary MEFs were infected\nwith a range of MCMV PFU (MOI, 0.01 to 10).\nRESULTS\nH-2b linked genes fail to support innate MCMV immunity\nthrough NK cells. We previously noted that multiple genes\ncontribute vital Ly49H-independent immunity shortly after\nMCMV infection in New Zealand White or MA/My mice (11,\n29). Because H-2k-linked genes were shown to contribute sub-\nstantially to MA/My virus resistance (10, 11) and also con-\nverted susceptibility to full resistance in C57L.M-H2k congenic\nmice (11), we examined whether they are also essential to\ninnate immunity in this model system. We replaced H-2k in\nMA/My with H-2b from MCMV-susceptible C57L by use of\nSSLP marker-assisted selection to obtain H-2b donor alleles on\na MA/My background (Fig. 1A). H-2b introgression was con-\nﬁrmed using allele-speciﬁc anti-MHC class I mAbs to stain\nM.L-H2b splenocytes (not shown).\nWe assessed innate MCMV immunity in M.L-H2b by mea-\nsuring virus levels in spleens and livers. These mice displayed\nremarkable susceptibility that was even more severe than that\nof C57L (Fig. 1B and C). By comparison, similarly infected\nL.M-H2k/b and control MA/My mice were fully protected by\nH-2k. Thus, H-2k-linked gene expression is essential for innate\nMCMV protection through NK cells in MA/My, whereas H-2b\nlinked gene expression is not adequate for effective NK con-\ntrol. While previous work indicated that an additional genetic\nfactor beyond the MHC or the NK gene complex can contrib-\nute to innate MCMV control (11), we also evaluated immunity\nin (MA/My  C57L)F1 hybrids and their heterozygous con-\ngenics, M.L-H2b/k mice (Fig. 1D). As expected, virus levels in\nlittermate control (H-2k) spleens were comparable with those\nfor MA/My. Intriguingly, however, virus levels in M.L-H2b/k\nspleens were actually intermediate between those of the pro-\ngenitor strains. This ﬁnding indicates that H-2k is not fully\nprotective on the MA/My genetic background. Instead, H-2k\nhomozygosity is required to fully establish innate MCMV re-\nsistance on the MA/My background. Moreover, since differ-\nences in MCMV control in H-2 heterozygous mice strictly\ncorrelated with their genetic background (compare L.M-H2k/b\nand M.L-H2b/k in Fig. 1B and D, respectively), an additional\nfactor(s) coming from C57L must increase the extent of innate\nvirus resistance afforded by the H-2k haplotype.\nTo further delimit an H-2k-linked critical region, novel H-2\nrecombinant mice were produced from the M.L-H2b and L.M-\nH2k strains. One strain, M.L-H2bR1, was identiﬁed as contain-\ning an informative recombination breakpoint between the H-2\nK and D genes. Importantly, this strain retains the expression\nof Kb and C57L-derived MHC class II proteins (not depicted)\nbut also expresses MA/My-derived class I Dk molecules. We\ncrossed (MA/My  M.L-H2bR1)N9 backcross mice with C57L\nand studied MCMV control in their hybrid offspring. As ex-\npected, a 1:1 Mendelian distribution of homozygous (Kb to\nI-Eb region) and fully heterozygous (H-2k,b) progeny on F1\ngenetic backgrounds was observed. The corresponding innate\nMCMV immunities in the two groups shown in Fig. 1E indi-\ncate that Kk expression is not needed to limit MCMV in the\nspleen. The H-2k-linked critical region should therefore reside\ndistal to SSLP marker 17Uva12 and the class II genes while\noverlapping Dk class I and more-distal H-2 nonclassical class Ib\ngenes.\nSimilar levels of MCMV infectivity and replication in sus-\nceptible C57L and resistant C57L.M-H2k strain-derived MEFs.\nThough previous work has shown that MCMV infectivity and\nreplication kinetics are similar in H-2b- and H-2k-derived\nMEFs (7), Harnett and Shellam (16) noted high MCMV sus-\nceptibility in BALB/c (H-2d) and B6 (H-2b) strain-derived\nMEFs compared with that for MEFs from H-2k strains C3H\nand CBA. Thus, we also analyzed viral growth kinetics in\nMEFs prepared from embryos of MCMV-resistant and -sus-\nceptible strains. Figure 2A shows that MCMV grew exponen-\ntially from d 1 to d 5 with similar kinetics in MEFs derived\nfrom either strain. In addition, comparable levels of infectious\nvirions were released into the MEF supernatants after d 2 and\nthrough d 6. Furthermore, we did not observe signiﬁcant dif-\nferences in viral replication, plaque size, or viral cytopathic\neffect in comparison with MA/My MEFs (not shown). In\nagreement with Chalmer et al. (7), we conclude that H-2-\nlinked genetic differences in innate MCMV immunity are not\ncontrolled at the level of MCMV infectivity or replication\nwithin host cells.\nNK cell effector functions in M.L-H-2b and L.M-H-2k. To\ndelineate whether an inherent defect in H-2b NK virus sensing\nmight explain an inability to restrain viral replication, we ex-\namined NK IFN- shortly after infection, as this cytokine is\nrapidly induced in NK cells of MCMV-resistant C57BL/6 mice\nduring MCMV infection (26). Spleen cellularity generally cor-\nresponded with either the MA/My or the C57L genetic back-\nground through the ﬁrst 39 h of infection (Fig. 3). By 90 h after\ninfection, however, splenocyte numbers dropped precipitously\nin homozygous H-2b strains and also in M.L-H2b/k littermate\ncontrols (Fig. 3A). Thus, splenocyte numbers were not abso-\nlutely correlated with high or low virus levels, but in mice\nwithout the H-2k haplotype, substantial losses were always\nnoted. Also consistent with previous ﬁndings (8, 11, 12), NK\ncell numbers drop similarly through the ﬁrst 39 h after infec-\ntion in resistant and susceptible strains (not depicted). More-\nover, similar numbers of spleen NK cells were producing\nIFN- with maximal induction in strains studied by the same\ntime point (Fig. 3B). Notably, NK cells did not produce IFN-\nafter injection of heat- or UV-inactivated MCMV into the\nH-2k or H-2b strains under study (not depicted). After 90 h,\nspleen NK cell IFN- levels were diminished considerably in\nresistant and susceptible mice and did not correlate with\nMCMV control traits.\nRole of H-2k class I expression in MCMV infection. H-2-\nlinked control\nin innate MCMV immunity suggested that\nMHC class I molecules might have a role in NK cell control. As\nregulation of Kk and Dk class I proteins by MCMV has not\nbeen previously characterized, we also examined their expres-\nsion on L929 (H-2k) cells and primary (C57L  MA/My)F1\nMEFs shortly after virus infection. We utilized a reporter\nMCMV virus (MCMV-m157/EGFP [11]) to directly visualize\ninfected cells by ﬂow cytometry. Although more virus is re-\nquired to infect L929 (H-2k) than NIH 3T3 cells, ﬂow cyto-\nmetric analysis revealed that 24% of L929 cells (MOI  10)\nwere GFP MCMV-infected cells by 24 h after infection (not\nshown). Under these conditions, Kk and Dk proteins on L929\ncells were signiﬁcantly decreased only on infected (GFP) cells\n(Fig. 4A). Similar decreases in class I display were observed\nwhen L929 cells were infected with the wild-type K181 strain\n(not depicted). Fine differences in Kk or Dk proteins on\nMCMV-infected L929 prompted us to further investigate po-\ntential gene- or allele-speciﬁc viral regulation of MHC class I\nexpression. We measured class I expression on primary (C57L \nMA/My)F1 MEFs as an independent means to establish that\nthe diminishment of class I proteins of either haplotype is a\ngeneral feature of MCMV infection. Due to low constitutive\nclass I levels on the MEFs, IFN- induction was utilized to\nboost expression before MCMV infection. In the same way, we\nfound that MCMV downregulated Kk, Dk, and H-2b class I\nmolecules by 90% on F1 MEFs (Fig. 4B). MCMV control of\nclass I expression is speciﬁc, since CD71, a common surface\nprotein, was not affected (Fig. 4C). We obtained similar results\nusing Smith strain MCMV (data not shown). H-2b and H-2k\nclass I proteins are therefore efﬁciently and speciﬁcally de-\ncreased by MCMV on cells expressing alleles of either haplo-\ntype.\nBecause type I interferons are indispensable in innate\nimmunity and MCMV resistance, we also studied class I down-\nregulation by MCMV on IFN--stimulated F1 MEFs. Though\nIFN- directly affected the percentage of cells infected in a\ndose-dependent manner, presumably through direct inhibition\nof viral gene expression (24), MCMV again efﬁciently de-\ncreased H-2k and H-2b class I proteins on F1 MEFs (Fig. 5A).\nAdditionally, though uninfected F1 MEFs have similar levels of\nKk and Dk display, the amount of Dk class I proteins was\nconsistently less than half that of Kk proteins on infected cells.\nSimilar results were obtained over a large range of IFN-\ndoses (Fig. 5B). Furthermore, Dk mean ﬂuorescence intensity\ndecreased sharply, whereas Kk mean ﬂuorescence intensity\nlevels remained fairly constant throughout the study, indicating\nthat MCMV infection under certain conditions can selectively\nregulate MHC class I proteins in a gene-speciﬁc manner.\nDISCUSSION\nAn H-2k association with host survival following MCMV\ninfection was long ago recognized, though a mechanistic un-\nderstanding is still elusive. We show that dominant innate virus\nresistance was converted to profound susceptibility in MA/\nwhen H-2k is available, perhaps through augmentation of NK\nsensing, but this factor by itself is not sufﬁcient in innate virus\nresistance because C57L NK cells fail to thwart a comparable\nMCMV infection. We further show that proximal H-2 Kk class\nI and class II genes do not explain the observed genetic vari-\nation, since our novel H-2 recombinant strain is fully compe-\ntent to limit virus replication in the spleen at early times after\ninfection. Thus, a genetic locus distal to the MHC class II gene\nregion and overlapping with Dk class Ia and more-distal non-\nclassical class Ib genes should contain the responsible gene(s)\nrequired in H-2k-linked innate MCMV resistance through NK\ncells. Because multiple H-2 loci were previously implicated in\nthe control of MCMV-induced mortality (15), we were also\nprompted to study mortality in the novel strains. Recent pre-\nliminary studies with sublethal or lethal doses of MCMV have\nso far indicated similar mortality rates for our H-2k recombi-\nnant strain and littermate control tested as shown in Fig. 1E\n(X. Xie and M. G. Brown, unpublished data).\nTo further examine the role of NK cells in innate H-2k\ncontrol, IFN- was quantiﬁed for comparison in H-2k and H-2b\nstrains over a 3.5-d time course following infection. Deﬁcient\nNK control in C57L and M.L-H2b is not due to an intrinsic\ndefect in virus sensing, since active viral replication alerted\ntheir NK cells to enhance IFN- production to levels similar to\nthose for NK cells in MA/My or L.M-H2k shortly after infec-\ntion. Similar diminished responses in all strains were also\nnoted by later time points and did not directly correlate with\ninnate virus immunity. Whether this response is due to virus\nsensing through innate receptors (i.e., TLR3 and TLR9) lead-\ning to enhanced cytokine stimulation of NK function (4, 34) or\ndue to direct NK recognition of virus-infected targets is an\nimportant question. Recent reports indicate that NK IFN-\nrelease is severely impaired in mice without TLR9 or MyD88\nand that dendritic cell-derived interleukin 18 (IL-18) is also\nimportant (1, 9). Here we show similar numbers of NK cells\nproducing IFN- and comparable induction levels after\nMCMV infection; thus, these data suggest that an MCMV 3\nTLR9 3 MyD88 3 IL-18 axis in dendritic cells and an IL-18 3\nIFN- axis in NK cells during the nonspeciﬁc phase of NK\nactivation are fully competent in the strains tested in Fig. 3.\nNevertheless, H-2b-linked gene expression fails to support NK\ncontrol of viral replication on the C57L or MA/My genetic\nbackgrounds.\nWhile both MCMV and HCMV convergently acquired key\ngene products to manipulate MHC class I display (27), genetic\nvariance in NK control may not be altogether surprising, since\nthese cells readily respond to MHC-compatible targets with\ndeﬁcient class I expression. Because Wagner et al. noted be-\nfore that class I Kb molecules are somewhat refractory to\nMCMV control (38), we reasoned that class I gene- or allele-\nspeciﬁc regulation by MCMV might explain H-2k protection,\nat least in part. However, minor variations in class I display\nmight best be observed only under certain circumstances, since\nKk (decreased by 95%) and Dk (decreased by 50%) pro-\nteins were differently downregulated on MCMV-infected L929\ncells. Dk display, on the other hand, was affected more than Kk\nby MCMV in IFN--induced F1 MEFs, since a substantially\ngreater number of infected (GFP) cells continued to display\nKk proteins even after Dk had seemingly been lost from the cell\nsurface. IFN- may conceivably have an impact on Kk induc-\nMy.L-H2b congenic mice, since infectious MCMV freely rep-\nlicates to very high levels in M.L-H2b spleens within days after\ninfection. Thus, MA/My H-2k-linked loci are indeed necessary\ncomponents of innate viral immunity. This difference cannot\nbe explained by H-2-controlled differences in infectivity or\nreplication, as we observe similar MCMV growth kinetics in\nMEFs derived from embryos from resistant or susceptible\nstrains. Another intriguing and unexpected ﬁnding in the cur-\nrent report is seen in Fig. 1, since H-2k-linked resistance is fully\ncompetent in H-2 heterozygous or homozygous mice on the\nC57L background, whereas a single H-2k haplotype in H-2\nheterozygous mice on the MA/My background did not result in\nfull resistance. An additional polymorphic genetic factor(s) in\nC57L therefore further limits MCMV spread and replication\ntion greater than that of other class I proteins. Alternately,\nMCMV proteins known to bind and downregulate class I dis-\nplay may more avidly bind Dk when class I synthesis is strongly\ninduced by type I interferons. Taken together, our ﬁndings\nsuggest that only very minimal levels of Dk ligands that could\npotentially stimulate Ly49P receptors on NK cells in the course\nof MCMV infection should be available on infected host cells\nand that even fewer might be expected to hold a particular\nrelevant virus peptide, as suggested recently (10).\nAlternately, MCMV might stabilize some cell surface class I\ndisplays, including that of Dk proteins, while at the same time\ninterfering with cytotoxic T-lymphocyte recognition. MCMV\ngp34 is notable in this regard, since it binds endoplasmic re-\nticulum-resident class I proteins destined for expression at the\ncell surface. Interestingly, Dk but not Kk surface expression is\nselectively affected by gp34 in infected L929 cells (Xie and\nBrown, unpublished data). An intriguing possibility is that\nLy49P NK cells might speciﬁcally recognize gp34-associated\nDk proteins. However, innate H-2k resistance is likely more\ncomplex, since signiﬁcant protection has been observed even\non the BALB/c background, where Ly49P receptors are not\nexpressed (32). Finally, it is possible that NK recognition and\ncontrol become fully competent only in mice that express H-2k\nclass I ligands during development. While NK cells in C57L\nand M.L-H2b mice do recognize viral intrusion and IFN-\nexpression is stimulated shortly afterward, these cells could be\nhyporesponsive (14) or unlicensed (19) in terms of their ca-\npacity to attack infected cells. NK receptors in H-2k mice, on\nthe other hand, might recognize viral signatures more efﬁ-\nciently than those in other H-2 haplotypes, in addition to un-\ndergoing potential stimulation through Ly49P/Dk interactions\nduring NK encounters with MCMV-infected cells. While in-\nhibitory KIR/MHC interactions also insure functional compe-\ntency in human NK cells (2) and can regulate antiviral immune\nresponses in infected individuals (13, 18), this model should\nenhance our understanding of the role of NK cells in the early\ndetection and control of viral pathogens.",
         "",
         "Deficient Major Histocompatibility Complex-Linked Innate Murine Cytomegalovirus Immunity in MA/My.L-H2b Mice and Viral Downregulation of H-2k Class I Proteins",
         "None"
        ],
        [
         "17",
         "000a55c6a88b8c1413e6bb0281ea4e6a8bac4588",
         "Maintenance is recognised as the most difficult and expansive activity of the software development process. Numerous techniques and processes have been proposed to ease the maintenance of software. In particular, several authors published design defects formalising \"bad\" solutions to recurring design problems (e.g., anti-patterns, code smells). We propose a language and a framework to express design defects synthetically and to generate detection algorithms automatically. We show that this language is sufficient to describe some design defects and to generate detection algorithms, which have a good precision. We validate the generated algorithms on several programs",
         "Naouel Moha,Yann-Gaël Guéhéneuc,Pierre Leduc",
         "None",
         "http://www.archipel.uqam.ca/5168/1/Moha06-ASE.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://www.archipel.uqam.ca/5168/1/Moha06-ASE.pdf",
         "None",
         "",
         "Automatic Generation of Detection Algorithms for Design Defects",
         "None"
        ],
        [
         "18",
         "000b0ebf13045810dbd2aeae7423b10ce306dd76",
         "Decay-accelerating factor (CD55) is a complement regulatory protein, which is expressed by most cells to protect them from complement-mediated attack. CD55 also binds CD97, an EGF-TM7 receptor constitutively expressed on granulocytes and monocytes and rapidly up-regulated on T and B cells upon activation. Early results suggested that CD55 could further enhance T cell proliferation induced by phorbol ester treatment. The present study demonstrates that coengagement of CD55, using either cross-linking mAbs or its natural ligand CD97, and CD3 results in enhanced proliferation of human peripheral blood CD4+ T cells, expression of the activation markers CD69 and CD25, and secretion of IL-10 and GM-CSF. Recently, an increase in T cell responsiveness in CD55−/− mice was shown to be mediated by a lack of complement regulation. In this study, we show that direct stimulation of CD55 on CD4+ T cells with CD97 can modulate T cell activation but does not interfere with CD55-mediated complement regulation. Our results support a multifaceted role for CD55 in human T cell activation, constituting a further link between innate and adaptive immunity.",
         "M. Capasso,L. Durrant,M. Stacey,S. Gordon,J. Ramage,I. Spendlove",
         "None",
         "https://journals.aai.org/jimmunol/article-pdf/177/2/1070/1220551/zim01406001070.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1220551%26volume%3d177%26issue%3d2%26page%3d1070%26filename%3dzim01406001070.pdf%26languageId%3d",
         "None",
         "Holding T cells in check--a new role for complement regulators?;Regulatory T cells: development, function and role in autoimmunity.;Decay-accelerating factor modulates induction of T cell immunity;CD97, an adhesion receptor on inflammatory cells, stimulates angiogenesis through binding integrin counterreceptors on endothelial cells.;The complement inhibitory protein DAF (CD55) suppresses T cell immunity in vivo;CD46 is a cellular receptor for group B adenoviruses;The epidermal growth factor-like domains of the human EMR2 receptor mediate cell attachment through chondroitin sulfate glycosaminoglycans.;Activation of human CD4+ cells with CD3 and CD46 induces a T-regulatory cell 1 phenotype;Cutting Edge: B7/CD28 Interactions Regulate Cell Cycle Progression Independent of the Strength of TCR Signaling1;Expression and regulation of CD97 in colorectal carcinoma cell lines and tumor tissues.;The role of CD55 in protecting the tumour environment from complement attack.;Molecular Analysis of the Epidermal Growth Factor-like Short Consensus Repeat Domain-mediated Protein-Protein Interactions;Tissue distribution of the human CD97 EGF-TM7 receptor.;Family of Escherichia coli Dr adhesins: decay-accelerating factor receptor recognition and invasiveness.;Membrane complement regulatory proteins: insight from animal studies and relevance to human diseases.;Crry/p65, a Membrane Complement Regulatory Protein, Has Costimulatory Properties on Mouse T Cells1;CD46 Is a Cellular Receptor for Human Herpesvirus 6;Mapping the binding domains on decay accelerating factor (DAF) for haemagglutinating enteroviruses: implications for the evolution of a DAF-binding phenotype.;Expression characteristics of the human CD97 antigen.;Membrane cofactor protein (MCP or CD46) is a cellular pilus receptor for pathogenic Neisseria;CD97 is a processed, seven-transmembrane, heterodimeric receptor associated with inflammation.;The seven-span transmembrane receptor CD97 has a cellular ligand (CD55, DAF);EGF-TM7: a novel subfamily of seven-transmembrane-region leukocyte cell-surface molecules.;Membrane cofactor protein (CD46) is a keratinocyte receptor for the M protein of the group A streptococcus.;Coxsackievirus B3 adapted to growth in RD cells binds to decay-accelerating factor (CD55);Decay-accelerating factor (CD55), a glycosylphosphatidylinositol-anchored complement regulatory protein, is a receptor for several echoviruses.;Characterization of an Early Activation‐Dependent Antigen on Lymphocytes Defined by the Monoclonal Antibody BL‐Ac(F2);Human membrane cofactor protein (CD46) acts as a cellular receptor for measles virus;Signal transduction through decay-accelerating factor. Interaction of glycosyl-phosphatidylinositol anchor and protein tyrosine kinases p56lck and p59fyn 1.;Mapping of epitopes, glycosylation sites, and complement regulatory domains in human decay accelerating factor.;Distinct receptor and regulatory properties of recombinant mouse complement receptor 1 (CR1) and Crry, the two genetic homologues of human CR1;GPI-anchored cell-surface molecules complexed to protein tyrosine kinases.;The Dr hemagglutinin, afimbrial adhesins AFA-I and AFA-III, and F1845 fimbriae of uropathogenic and diarrhea-associated Escherichia coli belong to a family of hemagglutinins with Dr receptor recognition;Decay-accelerating factor functions as a signal transducing molecule for human T cells.;Induction of T cell activation by monoclonal anti-Thy-1 antibodies.;Antitumour reactions of monoclonal antibody against a human osteogenic-sarcoma cell line.;Natural versus adaptive regulatory T cells.;Activation of T cells via CD55: recruitment of early components of the CD3-TCR pathway is required for IL-2 secretion.;Decay-accelerating factor: biochemistry, molecular biology, and function.;The epidermal growth factor-seven transmembrane (EGF-TM7) receptor CD97 is required for neutrophil migration and host defense",
         "Costimulation via CD55 on Human CD4+ T Cells Mediated by CD97",
         "None"
        ],
        [
         "19",
         "000f0b4a5545457a688f5ab07ac7e179ca881b82",
         "Analysis of molecular data sets has provided new insights into higher-level relationships of living Eutheria, including the recognition of Afrotheria as a novel taxon. This offers an opportunity to take a fresh look at the evolution of organ systems, including some that are little used in traditional systematics. In the present study, we attempted a reconstruction of the evolution of characters associated with placentation, the fetal membranes and the female reproductive tract. The evolutionary history of 21 characters has been traced, based on a current hypothesis of eutherian relationships, by applying the computer program MacClade. Accordingly, the analysis provides a first comprehensive interpretation of the stem species pattern of Eutheria. Of particular note, this pattern includes an endotheliochorial chorioallantoic placenta. The reconstructed pattern of Eutheria does not change in the basal nodes of the group. Thus, no character transformations occur on the stem lineages of Laurasiatheria or Euarchontoglires, and even Afrotheria has mostly plesiomorphic character conditions. However, two character transformations occur on the common stem lineage of Afrotheria and its sister taxon Xenarthra, i.e., amniogenesis by cavitation instead of folding and the precocial state of the newborn. In addition, we recognized one character transformation on the stem lineage of Afrotheria, i.e., the occurrence of a four-lobed allantoic sac. Thus, contrary to previous assertions, it is possible to identify morphological characters that could be synapomorphic for this novel taxon.",
         "A. Mess,A. Carter",
         "A. Mess,A. Carter",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/jez.b.21079",
         "ABSTRACT\nAnalysis of molecular data sets has provided new insights into higher-level\nrelationships of living Eutheria, including the recognition of Afrotheria as a novel taxon. This offers\nan opportunity to take a fresh look at the evolution of organ systems, including some that are little\nused in traditional systematics. In the present study, we attempted a reconstruction of the evolution\nof characters associated with placentation, the fetal membranes and the female reproductive tract.\nThe evolutionary history of 21 characters has been traced, based on a current hypothesis of\neutherian relationships, by applying the computer program MacClade. Accordingly, the analysis\nprovides a ﬁrst comprehensive interpretation of the stem species pattern of Eutheria. Of particular\nnote, this pattern includes an endotheliochorial chorioallantoic placenta. The reconstructed pattern\nof Eutheria does not change in the basal nodes of the group. Thus, no character transformations\noccur on the stem lineages of Laurasiatheria or Euarchontoglires, and even Afrotheria has mostly\nplesiomorphic character conditions. However, two character transformations occur on the common\nstem lineage of Afrotheria and its sister taxon Xenarthra, i.e., amniogenesis by cavitation instead\nof folding and the precocial state of the newborn. In addition, we recognized one character\ntransformation on the stem lineage of Afrotheria, i.e., the occurrence of a four-lobed allantoic\nsac. Thus, contrary to previous assertions, it is possible to identify morphological characters that\ncould be synapomorphic for this novel taxon. J. Exp. Zool. (Mol. Dev. Evol.) 306B:140– 163, 2006.\nr 2005 Wiley-Liss, Inc.",
         "DISCUSSION\nA large amount of information is available on\nthe character complex of placentation, fetal mem-\nbranes and the female reproductive tract. As a\nresult, we were able to identify 21 characters\nsuitable for analysis and trace their evolution on\nthe basis of a pre-existing hypothesis of taxon\nrelationships (Douady et al., 2002b). For most of\nthese characters it was possible to determine the\nstem species pattern of living Eutheria and of the\nmain superordinal taxa, including Afrotheria. Of\nparticular note, we found one character that had\nbeen evolved on the stem lineage of Afrotheria,\nand two others that are shared with their sister\ngroup Xenarthra. Thus, contrary to previous\nassertions (e.g. Yang et al., 2003), it should be\npossible to identify morphological characters that\nare synapomorphic for this novel taxon.\nStem species pattern of Eutheria\nFirst of all, based on distributional data, the\nanalysis indicates that the stem species pattern of\nliving Eutheria (Placentalia) is as follows. The\ndeﬁnitive chorioallantoic placenta is discoid in\nshape and labyrinthine in structure. It is supplied\nby two umbilical arteries and a single umbilical\nvein. The interhemal barrier separating maternal\nand fetal blood is of the endotheliochorial type and\ncontains syncytiotrophoblast but not cytotropho-\nblast, at least in the deﬁnitive or late placenta. The\nendometrium possesses decidual cells. Special\nstructures such as hemophagous organs, areolae\nand trophoblastic fenestrae are not present in the\nstem species pattern of Eutheria. Implantation is\nantimesometrial and amniogenesis occurs by fold-\ning. The bilaminar and trilaminar omphalopleure\nare temporary structures. At term, the yolk sac is\neither absent or has lost all contact with the\nuterine wall and chorion, so that there is no\nfunctional choriovitelline placenta. In contrast,\nthe allantoic vesicle is permanent, medium to\nlarge in size and comprises a single chamber. The\npregnancy is established in a bicornuate uterus\nand the newborn are altricial.\nOf particular note, the current analysis appears\nto resolve a long-standing controversy about the\nancient condition of the interhemal barrier of the\nplacenta. Ever since the classiﬁcation made by\nGrosser (’09), authorities on placentation have\ndisputed whether a non-invasive (epitheliochorial)\nor an invasive (hemochorial) type was primitive\nfor eutherians (inter alia Hubrecht, ’09; Wislocki,\n’29; Starck, ’59; Mossman, ’87; Luckett, ’93). Only\nMossman (’37) took into account the possibility\nthat an endotheliochorial condition could be\nprimitive for this group. But, after observing\nepitheliochorial placentation in Scalopus (Prasad\net al., ’79), an insectivore, Mossman changed his\nmind and argued for the epitheliochorial condition\nas the primitive state (Mossman, ’87). However,\naccording to our results, the stem species pattern\nis indeed characterized by the endotheliochorial\nplacental type (Fig. 1A). This result is in accor-\ndance with other recent evolutionary interpreta-\ntions (Martin, 2003; Carter and Enders, 2004;\nVogel, 2005).\nThe interhemal membrane is of interest in the\ncontext of maternal–fetal exchange of respiratory\ngasses and nutrients, sometimes referred to as\nhemotrophic nutrition (Mess et al., 2003). Of equal\ninterest is the ingestion by trophoblast of gland-\nular secretions, cell debris and erythrocytes,\nknown as histiotrophic nutrition (for a recent\ndiscussion see Vogel, 2005). Neither of the specia-\nlizations considered here, hemophagous regions\nand areolae, appear in the stem species pattern of\nEutheria (see Fig. 1C). Other specializations\nmentioned by Mossman (’87, his Table 14.4)\ninclude absorptive plaques, arcades and marginal\nfolds. We attempted to incorporate these into our\nanalysis but did not obtain resolution (data not\nshown).\nThe\npattern\nfor the chorioallantoic placenta,\nincluding the\nendotheliochorial barrier, occurs in Bradypus,\nreﬂecting its basal position in the tree of Douady\net al. (2002b). Thus, the reconstructed condition\nof chorioallantic placentation is present in an\nextant species.\nreconstructed\nAs an additional feature, we were able to gain\nmore precise insight into the evolution of yolk sac\ncharacters (cf. extensive discussion in Mossman,\n’87). Formerly, the presence in several species of a\ntrilaminar omphalopleure or a true yolk sac\nplacenta was regarded as a plesiomorphic condi-\ntion of Eutheria, retained from the common\nancestor of Theria, Mammalia or even Amniota\n(i.e. Luckett, ’77, ’85). Up to a point our analysis\nsupports this interpretation. However,\nit also\nshows that both the bilaminar and trilaminar\nomphalopleure exist only as temporary structures\nin the stem species pattern of Eutheria and their\nmajor clades (Fig. 3C, cf. Fig. 5). Consequently, the\nabsence of a true yolk sac placenta at term is\nlikewise a plesiomorphic condition of Eutheria.\nThus yolk sac placentation is more different\nbetween Eutheria and Metatheria than indicated\nby common text books. Interestingly, a phyloge-\nnetic analysis of marsupial placentation (Freyer\net al., 2003) led to the conclusion that a chor-\nioallantoic placenta was part of the stem species\npattern of marsupials but that the yolk sac had\nevolved to become the principal organ of physio-\nlogical exchange.\nAs a consequence of our analysis, Tupaia with\nits permanent yolk sac placenta or members of\nLipotyphla with a permanent bilaminar omphalo-\npleure can no longer serve as models that have\nretained ancient conditions. The eutherian stem\nspecies pattern with regard to yolk sac organiza-\ntion can be recognized in quite different taxa, i.e.,\nin lower primates such as Galago or members\nof Tenrecoidea.\nWe separately examined the consequences of re-\nrooting the current tree between Afrotheria and\nall other placentals (Murphy et al., 2001a,b;\nAmrine-Madsen et al., 2003). Brieﬂy, the stem\nspecies pattern for Eutheria remained much as\ndescribed above (data not shown), i.e., 17 of 21\ncharacters were not at all\ninﬂuenced by the\ndifferent position of the basal node within Eu-\nthe results contradicted our\ntheria. None of\nfour characters were now\nanalysis. However,\nunresolved: gross form of\nthe placenta, ﬁrst\nattachment of the blastocyst, amniogenesis and\nnewborn.\nIn the following, we deal with character trans-\nformations—compared with the stem species\npattern of Eutheria—that occur on the stem\nlineages of the higher order clades of Eutheria\nand discuss important transformations that took\nplace within these groups.\nStem species pattern of Afrotheria\nAfrotheria presents some interesting transfor-\nmations compared with the stem species pattern of\nEutheria. Two of these—amniogenesis by cavita-\ntion (Fig. 3A) and precocial young (Fig. 4C)—are\nshared with Bradypus, which is representative of\nthe sister group Xenarthra. There are few other\ninstances in the tree of amnion formation by\ncavitation. Often, as in Erinaceus, Cavia and\nHomo, this character state is associated with\ninterstitial implantation (Luckett, ’76). However,\nthis is not the case in Afrotheria or Bradypus.\nPrecociality requires a longer gestation and\nusually occurs in relatively large animals such as\nCetartiodactyla, Hystricognathi (Mess, 2003) and\nhigher primates (Martin, 2003), where it\nis\napomorphic.\nThe presence of a large, four-chambered allan-\ntoic sac in representatives of every order of\nAfrotheria, but not in Bradypus, represents a\nhitherto unrecognized synapomorphy for\nthe\nsuperorder (Fig. 4A). A similar arrangement is\n’32), but\nknown only for Strepsirhini\nconvergently\naccording to our analysis was\nachieved.\n(Hill,\nWe did not obtain resolution in Afrotheria\nfor the gross form of the placenta, which is\ndiscoidal in Tenrecoidea and Elephantulus, but\nzonary in the rock hyrax–elephant–sea cow\nclade (Paenungulata) and Orycteropus. Hemopha-\ngous regions are a prominent feature of the\nplacenta in several genera, but apparently the\nplesiomorphic state, i.e., absence of the organ, is\ncharacteristic for the stem species pattern of\nAfrotheria (cf. Fig. 2).\nEvolutionary transformations\nwithin Afrotheria\nThere is a transformation from a separate\nurogenital opening to a ‘‘cloaca’’ in Tenrecoidea\n(Fig. 4B). Our analysis fails to support\nthe\nassumption that this is a primitive rather than a\nderived character. As extensively discussed by\nShoshani and McKenna (’98; their Appendix 1),\ninclusion of the anal and urogenital openings in a\ncommon skin fold, as occurs in marsupials and\nsome eutherians, may not be homologous with the\ncloaca found in reptiles, birds and monotremes.\nIn addition to the establishment of a cloaca, there\nare two further character transformations on the\nstem lineage of Tenrecoidea: amniogenesis by\nfolding (Fig. 3B) and the altricial state of the\nnewborn (Fig. 4C).\nAbove, it was noted that a large, four-chambered\nallantoic sac is a synapomorphy for Afrotheria\n(Fig. 4A). This is secondarily transformed to a\nsingle sac in Echinops, although the associated\nearly division of cord vessels is retained in this\nspecies (Carter et al., 2004). Even within Tenre-\ncoidea, a four-chambered sac is found in Micro-\npotamogale and the related genus Potamogale\n(Carter et al., 2005b).\nRetention of a trilaminar omphalopleure to term\nhas been recorded in Eremitalpa (Gabie,\n’60),\nwhich represents a derived rather than a plesio-\nmorphic pattern within Eutheria (see above).\nOn the stem lineage of Tenrecoidea plus their\nsister group Macroscelidea, here represented by\nElephantulus, a transformation from the endothe-\nliochorial to the hemochorial type of placental\nbarrier took place (Fig. 1A). Such a transformation\noccurs twice in Afrotheria, the second example\nbeing Procavia. Until recently it was thought that\nthe interhemal membrane of Trichechus was\n’35), but this has been\nhemochorial (Wislocki,\nrefuted by studies at the ultrastructural\nlevel\n(Carter, Enders, Miglino and Da Silva, unpub-\nlished observations). Within Tenrecoidea and\nTenrecidae, a reversal of the interhemal mem-\nbrane structure to the endotheliochorial condition\nis recognized for Micropotamogale (Carter et al.,\n2005b).\nThe stem lineage of Orycteropus, Elephantulus\nand the Tenrecoidea does not show any character\ntransformations. Orycteropus has a duplex uterus.\nThis is clearly depicted by van der Horst (’49) and\nmust be regarded as a transformation from the\nbicornuate state, which represents the condition\nin the stem species pattern of Afrotheria. More-\nover, Orycteropus possesses areolae as an addi-\ntional placental structure (Fig. 1C) and decidual\ncells are absent. Both of\nthese are derived\ncharacter conditions within Afrotheria that are\nconvergently developed in Tenrec as well as\nLoxodonta.\ntrophoblast\nFinally, the stem species pattern of Paenungu-\nlata (i.e. hyrax, sea cow and elephant) is char-\nin the\nacterized by the type of\ninterhemal barrier of\nthe deﬁnitive placenta,\nwhich shows a transformation from syncytiotro-\nphoblast to cytotrophoblast (Fig. 1B). In addition,\nPaenungulata is characterized by a second derived\ncondition, in that implantation is central rather\nthan antimesometrial (Fig. 3A). The ﬁrst attach-\nment of the blastocyst evidences several other\ntransformations within Afrotheria. Tenrec and\nEremitalpa are among the few genera with\nimplantation on a lateral wall of the uterus.\nFinally, implantation in Elephantulus is within\nan implantation chamber on the mesometrial side\nof the uterus, associated with secondarily inter-\nstitial implantation as another derived character\ncondition of this genus.\nto Bradypus,\nThe tree contains only one representative of\nXenarthra and this is not\nthe place for an\nextensive analysis of the clade. It may be observed\nthe placenta\nthat,\nof armadillos is both villous and hemochorial\n(Enders, ’60). Fenestration of the trophoblast in\nBradypus is comparable to that in shrews (King\net al., ’82), but is not known for other Xenarthra.\nThe ‘‘cloaca’’\n’98),\nsimplex uterus (Wislocki, ’28), rudimentary allan-\n’87) and precocial young\ntoic sac (Mossman,\n’70) are characters\n(Naaktgeboren and Slijper,\nshared by armadillos, anteaters and sloths.\n(Shoshani and McKenna,\nEuarchontoglires\nNot a single character transformation is indi-\ncated on the stem lineage of Euarchontoglires.\nWithin that group, Tupaia retains the bicornuate\nuterus and endotheliochorial placenta (Fig. 1A)\nthat are part of the stem pattern of Eutheria yet\nlost in other representatives of Euarchontoglires.\nThere is lateral implantation in Tupaia (Fig. 3A),\nresulting in a double discoid placenta. A trilami-\nnar omphalopleure or yolk sac placenta is retained\nuntil\n’68).\nlate gestation in Tupaia (Luckett,\nAlthough this has been regarded as retention of\na primitive character (Luckett, ’68, ’77), it is clear\nfrom our analysis (Fig. 3C) that it is derived\nrelative to the common ancestor of Eutheria.\nThe common stem for Primates, Cynocephalus\nand Glires shows a transformation to hemochorial\nplacentation. In Galago, however, the placenta is\ndiffuse and epitheliochorial with cytotrophoblast\nin the interhemal membrane, all representing\nderived conditions of that group. Moreover, as\nfurther apomorphic conditions of Galago there are\nareolae and there are no decidual cells (cf. Figs. 1C\nand 2). This pattern is convergent with that in\nFerungulata. Hill (’32) regards a four-lobed allan-\ntoic sac, described for Loris and Nycticebus, as\ntypical for lorises, resulting as a derived pattern\nthat was independently established compared to\nAfrotheria (Fig. 4A).\nCynocephalus shares the following character\ntransformations with the three representatives of\nGlires: duplex uterus, inclusion of both types of\ntrophoblast in the interhemal membrane, and an\ninverted yolk sac. It must be noted, however, that\nthe position of the ﬂying lemurs is weakly supported\nin molecular analyses, resulting in very different\ntopologies. Short internodes and long branch lengths\ncontribute to this unstable topology (Douady, 2001).\n(a)\nWe are aware of ﬁve alterative topologies:\nDermoptera with Scandentia, as sister to primates\n(Murphy et al., 2001b, see Springer et al., 2005a);\n(b) Dermoptera as a sister group to Anthropoidea,\nthus ruling out primate monophyly (Murphy et al.,\n2001a; Arnason et al., 2002; but see Schmitz et al.,\n2002; Schmitz and Zischler, 2003); (c) Dermoptera\nas the sister group to Primates (Douady, 2001);\n(d) Dermoptera as sister to Glires (Douady et al.,\n2002b) and (e) Dermoptera as sister to Glires plus\nScandentia (Amrine-Madsen et al., 2003). In view of\nthis uncertainty,\nthe character transformations\nshared by Cynocephalus and Glires are of limited\nimportance.\nTransformations within Glires include reversion\nto syncytiotrophoblast in Cavia and other hystri-\ncognaths, which also have precocial newborns\n(Mess, 2003). In addition Cavia, as a typical\nhystricognath rodent, has primary interstitial\nimplantation; there is no true yolk sac placenta-\ntion and the allantoic sac is rudimentary. Thus,\nhystricognath rodents are especially characterized\nby derived conditions of\nfetal membrane and\nplacentation characters (see Luckett, ’85; Mess,\n2003). The aforementioned character states of\nCavia, and hystricognath rodents in general, often\noccur together (Luckett, ’76) and are convergently\nshared by Homo. Primates have a trabecular to\nvillous placenta and precocial young.\nLaurasiatheria, Ferungulata\nand Lipotyphla\nSimilar to Euarchontoglires, and in clear con-\ntrast to Afrotheria and/or Xenarthra, no character\ntransformations can be recognized on the stem\nlineage of Laurasiatheria in comparison with the\nEutherian stem species pattern. Transformations\nwithin this clade mostly affect Ferungulata (see\nbelow) and Lipotyphla.\nWithin Laurasiatheria, analyses of molecular\ndata provide robust support for a clade comprising\nCarnivora, Pholidota, Perissodactyla and Cetar-\ntiodactyla (Waddell et al., 2001). Archibald (2003)\nrecommends retaining the appellation Ferungula-\nta for this clade, notwithstanding that it differs\nfrom that originally proposed by Simpson (’45).\nA Ferungulata clade is consistent with our\ncharacter optimization. The stem pattern of\nFerungulata exhibits the following transforma-\ntions from that of Laurasiatheria or Eutheria: the\nplacenta is diffuse, villous and epitheliochorial\n(Fig. 1A); areolae are present (Fig. 1C); decidual\ncells are absent; and the newborn are precocial\n(Fig. 4C). Thus our analysis adds further support\nto the view (Vogel, 2005) that this character\ncomplex is not primitive but derived relative to\nEutheria. Ferungulata shares two further trans-\nformations with the bats, namely cytotrophoblast\nin the interhemal membrane of the deﬁnitive\nplacenta (Fig. 1B) and mesometrial attachment of\nthe blastocyst (Fig. 3A). There are reversals within\nFerungulata. In particular, carnivores have a\nzonary, labyrinthine, endotheliochorial placenta;\nantimesometrial attachment; and altricial young\nin their stem species pattern.\nA second point of interest within Laurasiatheria\nis the status of Lipotyphla following removal of the\ntenrecs and golden moles. Many transformations\nare indicated within Lipotyphla, but only two of\nthem are common to all genera included in the\ndata set. These are the presence of a permanent,\njuxtauterine omphalopleure together with a per-\nmanent,\ninverted, visceral yolk sac (cf. Fig. 5;\nMossman, ’87; Asher, 2005). Solenodon is not part\nof the set. However, a recent analysis (Roca et al.,\n2004) found Solenodon to be basal to the Lipo-\ntyphla with the topology for the remainder of the\norder being similar to that used here. Therefore,\nin a separate analysis (data not shown), we added\nSolenodon to the tree and reanalyzed those\ncharacters for which data are available (Wislocki,\n’40). This reduced the transformations common\nto Lipotyphla since Solenodon does not have a\npermanent juxtauterine bilaminar omphalopleure.\nMoreover, Solenodon has hemochorial placenta-\ntion, resulting in loss of resolution of the inter-\nhemal membrane not only for Lipotyphla, but for\nEuarchontoglires1Laurasiatheria (Boreoeutheria)\nand Eutheria too.\nThere are derived characters in every family of\nLipotyphla. Erinaceus exhibits interstitial implan-\ntation and has a hemochorial placenta. The shrews\nhave a unique hemophagous organ formed by the\ntrophoblastic annulus and trophoblastic ‘‘curtain’’\n(Brambell and Perry, ’45), the trophoblast of the\ninterhemal membrane is extremely thin and\nfenestrated, and there is a rudimentary allantoic\nvesicle. Moles have both types of trophoblast in the\ninterhemal membrane at term as well as promi-\nnent areolae. In addition, Scalopus has a diffuse\nplacenta that may or may not be epitheliochorial\n(Carter, 2005). As Symonds (2005) recently con-\ncluded, molecular analyses of Lipotyphla agree\nonly on the removal of tenrecs and golden moles,\nand there are competing hypotheses for the\nphylogeny of the residual families. Future ana-\nlyses combining molecular and morphological data\nmight beneﬁt from inclusion of characters from\nthe current set.\nConclusions\nWe traced the evolutionary history of 21 char-\nacters associated with placentation, fetal mem-\nbranes and female reproductive organs within\nliving Eutheria on the basis of a current hypoth-\nesis of eutherian relationships derived from\nDouady et al. (2002b). We recognized one char-\nacter transformation on the stem lineage of\ni.e., the occurrence of a four-lobed\nAfrotheria,\nallantoic sac, and two additional ones that occur\non the common stem lineage of Afrotheria and its\nsister taxon Xenarthra,\ni.e., amniogenesis by\ncavitation and the precocial state of the newborn.\nFinally, the analysis provides a comprehensive\ninterpretation of the ancient condition or stem\nspecies pattern of Eutheria. Apart from the above-\nmentioned character\ntowards\nAfrotheria, the reconstructed pattern of Eutheria\ndoes not change in the basal nodes of the group.\nThus, no character transformation occurred on\nthe stem lineages of Laurasiatheria and Euarch-\nontoglires, and even Afrotheria possesses mostly\nplesiomorphic character conditions of Eutheria.\nTransformations of the chosen set of characters\ncommonly took place within subgroups of the\nsuperordinal taxa of Eutheria such as Ferungulata\nand Tenrecoidea.",
         "In recent years, analysis of molecular data sets\nhas led to a revolution in mammalian systematics\nand revealed completely new higher-level relation-\nships within Eutheria (Springer et al., ’97; Stan-\nhope et al., ’98). Of particular note is the view that\nall extant eutherian species belong to one of four\nsuperordinal clades (Madsen et al., 2001; Murphy\net al., 2001a,b; Waddell et al., 2001; Springer et al.,\n2005a,b). These ideas about mammalian evolution\nhave provoked fruitful debate and stimulated\nresearch to test them by using morphological data\n(Asher, 2001; Symonds, 2005).\nThe orders of Eutheria recognized by molecular\nphylogeneticists are quite stable in relation to the\ntraditional morphological\nsystem (Archibald,\n2003), but there are two exceptions. The ﬁrst,\nthe inclusion of whales within Artiodactyla (as\nCetartiodactyla), is not controversial since these\norders are grouped close together in the tradi-\n’88; Shoshani and McKenna,\ntional system (Novacek and Wyss, ’86; Novacek\net al.,\n’98). The\nsecond, exclusion of tenrecs and golden moles from\nLipotyphla,\nis more contentious (Asher, 2003).\nThey are placed in a new order that has been\ncalled Afrosoricida (Stanhope et al., ’98), although\nTenrecoidea (McDowell,\n’58) takes precedence\n(Asher, 2005; Symonds, 2005).\nThe main outcome of molecular systematics is\nthe linkage of tenrecs and golden moles to other\norders originating in Africa, and the establish-\nment of a novel clade called Afrotheria (Stanhope\net al.,\n’98). This includes Proboscidea, Sirenia,\nHyracoidea, Macroscelidea, Tubulidentata and\nTenrecoidea. Although the composition of Lipo-\ntyphla has been enigmatic (Simpson, ’45; MacPhee\nand Novacek, ’93; Symonds, 2005), inclusion of\ntenrecoids with taxa such as Paenungulata (Simp-\nson,\n’45), was never before suggested. Recent\ninvestigations to test the Afrotheria hypothesis\nwith morphological data have failed to support it\n(Asher, 2001; Whidden, 2002). Therefore, there\nmay be good reason to include morphological data\nsets from other systems, which are less used in\ntraditional systematics.\nOne such character complex comprises placenta-\ntion, fetal membranes and the female reproductive\ntract. There is great variation in gross and\nﬁne morphology of the placenta in eutherians\n(Hubrecht, ’09; Starck, ’59; Mossman, ’87) and\nthere is currently speculation about the evolu-\ntionary forces leading to such diversity (Crespi and\nSemeniuk, 2004; Vogel, 2005). Much detailed\ninformation is available from original studies.\nNevertheless, very few of these characters have\nbeen incorporated in phylogenetic analyses of\n’86;\nmorphological data (Novacek and Wyss,\nNovacek et al., ’88; Luckett, ’93; Shoshani and\nMcKenna, ’98; Asher et al., 2003). A comprehen-\nsive evaluation of the large amount of data within\na phylogenetic framework seems very promising.\nTherefore,\nin the present study, we have at-\ntempted a reconstruction of the evolution of such\ncharacters\nin living Eutheria. The analysis\nis based on a pre-existing hypothesis of inter-\nordinal relationships (Douady et al., 2002b) and\nthe tracing of character evolution according\nto Afrotheria and other superordinal clades.\nMoreover, a reconstruction of the stem species\npattern of extant Eutheria (Placentalia) is given\nsince for several of these characters,\ni.e., the\ninterhemal barrier within the chorioallantoic\nplacenta,\nthe ancient condition of eutherians\nhas so far not been resolved.",
         "MATERIALS AND METHODS\nThe goal of the current analysis was to reveal\nthe evolution of placental characters within living\nEutheria and their higher-level clades. Accord-\ningly, we reconstructed the stem species pattern\nor ancient condition of relevant taxa, such as\nAfrotheria, in order to discover transformations\nbetween subsequent nodes or stem species pat-\nterns. A stem species pattern is deﬁned as the\n’92, 2000; Cunningham et al.,\ncharacter set of the last common ancestor of\na monophyletic taxon,\nincluding plesiomorphic\nand derived or apomorphic characters (Ko¨nigs-\nmann, ’75; Lieven and Sudhaus, 2000). For this\npurpose, the distribution of selected characters\nwas mapped on a current hypothesis of phyloge-\nnetic relationships within Eutheria (see below).\nThe evolutionary history of characters is revealed\nby character optimization (see Maddison and\nMaddison,\n’98;\nRieppel,\ncomputer program\nthe\nMacCladeTM (Version 4.0: David R. Maddison\nand Wayne P. Maddison, University of Arizona,\nSinauer Associates). MacClade is a widely accepted\ntool for analyzing phylogenetic hypotheses and\nreconstructing and displaying character evolution\non a phylogenetic tree. Character optimization is\nan approach previously used to trace character\nevolution in fetal membranes (Freyer et al., 2003;\nMess, 2003; Asher, 2005; Vogel, 2005). Moreover,\nthe parsimony-based method of character analysis\nin MacClade easily allows handling of missing\ndata. This is an enormous advantage in compar-\nison with maximum likelihood analysis (e.g. the\nprogram MesquiteTM, designed by the same\nauthors, which includes maximum likelihood op-\ntions adapted for morphological characters). In-\ndeed, for about half the characters in our study,\ndata are missing for several species, and thus they\nare not suited for such treatment.\nThough molecular analyses are currently con-\nverging in a well-resolved pattern of recent\nEutheria, characterized by four major groups\n(see Springer et al., 2005a,b), the individual papers\ndiffer in respect of taxa and molecules as well as in\nsome aspects of tree topology. We are well aware\nthat any topology chosen for analysis will have\nsome advantages and some shortcomings. Thus,\nsome background information on the chosen\ntopology and possible alternatives is needed.\nAnalyses that include molecular data are agreed\non an afrotherian clade including tenrecs and\ngolden moles; nonetheless, there are competing\ntopologies as summarized by Asher (2005) and\nSymonds (2005). We have chosen to focus on a tree\n(Fig. 2 in Douady et al., 2002b) that was based on a\nmaximum likelihood analysis of exon sequences\nfrom three nuclear genes: alpha-2B adrenergic\nreceptor, von Willebrand factor and breast-cancer-\nassociated gene 1. This is a relatively extensive\nmolecular data set in contrast to topologies that\nrely exclusively on mitochondrial DNA (e.g.\nArnason et al., 2002). Secondly, this tree (Douady\net al., 2002b) includes eight core insectivores,\nthree tenrecs and a golden mole, species that are\nregarded to be critical to an understanding of\nplacental evolution. Other topologies (Murphy\net al., 2001a,b; Waddell et al., 2001; Malia et al.,\n2002; Amrine-Madsen et al., 2003) have rather too\nfew of the species of interest and/or species for\nwhich insufﬁcient data on the placenta are\navailable. Two alternative topologies that are rich\nin insectivoran grade species (Douady et al.,\n2002a; Asher et al., 2003) ﬁnd a basal position\nfor murid rodents. As shown by Douzery and\nHuchon (2004), this likely is the result of basing\nthe analyses on models of sequence evolution that\nﬁt the data poorly and the use of outgroups that\nare too divergent. The tree of Douady et al.\n(2002b) does not suffer from this problem.\nAs recently reviewed by Springer et al. (2005a),\nthere are different possible locations for the root of\nthe placental tree. In Douady et al. (2002b), this is\nbetween Xenarthra1Afrotheria and Euarchonto-\nglires1Laurasiatheria (Boreoeutheria). Other mo-\nlecular studies, including supertree analysis, favor\na rooting between Afrotheria on the one hand and\nall other placentals on the other (Murphy et al.,\n2001a,b; Amrine-Madsen et al., 2003). Since these\ndifferences in basal position could be crucial for\nthe analysis, we have examined the consequences\nof re-rooting the current tree in the second\nlocation (see discussion, stem species pattern of\nEutheria). The available molecular phylogenies\ndiffer in some other points, as acknowledged\nwithin the discussion. Last but not least, Soleno-\ndon is not part of the set of Douady et al. (2002b)\nand other topologies. However, a recent analysis\n(Roca et al., 2004) found Solenodon to be basal\nto the Lipotyphla with the topology for the re-\nmainder of the order being similar to that used\nin a separate analysis, we\nhere. Therefore,\nadded Solenodon to the tree (see discussion,\nLaurasiatheria).\nIn order to map our characters on the selected\nphylogeny, we next searched the literature for\ndata on placentation in the 301 genera on which\nthe tree is based (we are grateful to Dr. Christophe\nJ. Douady for supplying us with a complete list of\nexamined species). In some instances, Douady\net al. (2002b) took sequences from two or three\nrelated genera, but we have chosen a single genus\nfor each branch of the tree. As an example, the\nmicrobats are represented by Tadarida in our\nanalysis, since placentation in this species is better\nknown than in Macrotus and Emballonura, which\nare also among the microbats in Douady’s data set.\nIn a few cases, data on placentation were missing\nor greatly deﬁcient. It was therefore decided to\nmake the following omissions and substitutions:\n(a) Cetacea: Douady et al. (2002b) used sequences\nfrom the\nbaleen whales Balaenoptera,\nMegaptera and Eschrichtius. We have substi-\ntuted a toothed whale, Tursiops truncatus,\nas this is the species for which placentation is\nbest known (Wislocki and Enders,\n’41).\nThe sparse data available for Balaenoptera\n(Turner, 1870) and on early development\n’60) are in\nin Megaptera (Stump et al.,\nagreement with the more complete study of\nTursiops.\n(b) Erinaceidae: In addition to Erinaceus, Douady\net al. (2002b) included the gymnure Hylomys\nin their analysis. A little is known about\nplacentation in a related genus, Echinosorex\n(Meister and Davis, ’53), but the study has\nbeen so severely criticized by Mossman (’87)\nthat we thought it preferable to omit the\ngymnures altogether.\n(c) Sirenia: Douady et al. (2002b) used sequences\nfrom Dugong. Although Turner (1889) de-\nscribed a poorly preserved placenta from this\nspecies, much better information is available\nfor the West Indian manatee, Trichechus\nmanatus (Wislocki, ’35); in addition, we have\nunpublished data on placental ultrastructure\nin the Amazonian manatee T. inunguis (Car-\nter, Enders, Miglino and Da Silva, unpublished\nobservations).\n(d) Proboscidea: Douady et al.\n(2002b) relied\nmainly on sequences from Elephas. Informa-\ntion is much more thorough on placentation in\nLoxodonta, and includes ultrastructural stu-\ndies (Allen et al., 2003; Wooding et al., 2005).\nData on the delivered placenta of Elephas\n(Cooper et al., ’64) are consistent with what is\nknown about Loxodonta.\n(e) Chrysochloridae: Douady et al. (2002b) used\nfrom Amblysomus. Nothing is\nsequences\nknown of placentation in this genus and we\nhave substituted Eremitalpa granti (Gabie,\n’59, ’60).\n(f) Metatheria: Douady et al.\n(2002b) used a\nkangaroo, Macropus giganteus. Information\non placentation is available for the tammar\nwallaby M. eugenii (e.g. Freyer et al., 2002),\nbut we have substituted Didelphis virginiana\nfor which more complete data are accessible.\nHowever, data are consistent between these\ntwo marsupials.\nA complete list of the species and principal\nliterature consulted is given in Table 1. In addition\nto these primary sources, we made use of reviews\nof placentation (De Lange, ’33; Starck, ’59; Steven,\nfemale reproductive tract\n’75; Mossman,\n(Starck, ’82) and newborns (Naaktgeboren and\n’70) and of standard reference works\nSlijper,\n(Grzimek,\n’99). Moreover, we fre-\nquently referred to a recent survey of comparative\nplacentation by Benirschke on the internet\n(Benirschke, 2005).\n’87),\nDespite the considerable amount of information\navailable on these organ systems, the number of\ncharacters that can be coded for the 34 genera in\nthe set is insufﬁcient for an independent cladistic\nanalysis, e.g., by PAUP.\nRESULTS\nCharacters associated with the\nchorioallantoic placenta",
         "A chorioallantoic placenta is not present in most\nincluding Didelphis and Macropus.\nmarsupials,\nThus,\nfor all characters associated with the\nchorioallantoic placenta, Didelphis has been as-\nsigned state 0, signifying that no chorioallantoic\nplacenta is present (Table 2). Some eutherian\nmammals exhibit different character states ac-\ncording to the stage of development. Since the\nmaterial available for the different species varies,\nbut usually includes the late or deﬁnitive placenta,\ncharacters 1–9 all refer to the deﬁnitive placenta\nfound near term (Table 2, Figs. 1 and 2). Some\nmammals, including tenrecs, elephant shrews and\nsome bats, have accessory placentae with char-\nacter states different from those of the main\nplacenta; these have been disregarded. Plotting\nthe character distribution on the selected tree of\nDouady et al. (2002b), the following information\non stem species patterns and character transfor-\nmation within Eutheria resulted.\nGross form of the placenta\nThere is considerable variation in the gross\nmorphology of the placenta (character 1; Steven,\n’87). In the diffuse type, the\n’75; Mossman,\nplacenta forms a sac covered with small villi or\nfolds, which interdigitate with corresponding\ndepressions on the uterine epithelium (state 1,\ne.g. Equus). In a cotyledonary placenta, the villi\nare concentrated in cotyledons, each of which is a\nsmall disk (state 3, e.g. Bos). In a zonary placenta,\nthe trophoblast forms an equatorial band of tissue\n(state 2, e.g. Canis). A single discoid placenta\noccurs in many species including man (state 4, e.g.\nCavia). Tree shrews and New World and Old\nWorld monkeys have a secondary disk and thus a\ndouble discoid placenta (state 5, e.g. Tupaia).\nA discoid placenta is present in the stem species\npattern of Eutheria, Euarchontoglires and Laur-\nasiatheria and is accordingly regarded as the\nplesiomorphic condition of those groups. This\ncharacter cannot be resolved for Afrotheria. With-\nin all superordinal taxa, a remarkable number of\ncharacter transformations occurs, indicating that\nthe gross form of the placenta possesses a high\ndegree of plasticity.\nOverall structure of the placenta\nAttention is often given to variations in the\noverall structure of the placenta (character 2;\nSteven, ’75; Mossman, ’87). In many placentae,\nthe maternal vessels or maternal blood channels of\nthe exchange area are completely enclosed in\ntrophoblast, forming a labyrinth (state 1, e.g.\nElephantulus). In a villous placenta, the fetal\nsurface is thrown into blind-ending projections or\nvilli which are bathed in maternal blood (human)\nor invested by the epithelial walls of maternal\ncrypts (cow). Intermediate forms occur with a\nlooser or trabecular structure. Because authors\ntend to differ in their deﬁnition of the trabecular\ntype, we have chosen to pool trabecular and villous\nplacentae (state 2, e.g. Tursiops).\nRepresenting the plesiomorphic condition of\neutherians, the labyrinthine pattern is found in\nthe stem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria. Transfor-\nmations toward the trabecular to villous type\noccur within the latter two clades, in particular,\nin the primates, including human beings, and in\nFerungulata.\nNumber of umbilical vessels\nEarly in development there are paired umbilical\narteries and veins. In many species, however, a\nreduction occurs so that the number of vessels in\nthe cord (character 3) varies from a single artery\nand vein (state 1, e.g. Mus), through two arteries\nand a vein (state 2, e.g. Cavia) to two arteries and\ntwo veins (state 3, e.g. Tursiops).\nTwo arteries and one vein are found in the stem\nspecies of Eutheria and this is a very regular\npattern, characterizing all superordinal taxa of\nthat group. Transformations to a pattern with two\nveins (and likewise two arteries) occur within\nLaurasiatheria and a reduction to one artery and\none vein is recognized for Mus. The later condition\nseems likewise to occur in Rattus and other\nmurids such as Phodopus (see Benirschke, 2005).\nInterhemal barrier\nThe number of tissue layers separating fetal\nfrom maternal blood varies (character 4; Grosser,\n’09; Enders, ’65; Mossman, ’87). In epitheliochorial\nplacentae, these include fetal capillary endothe-\nlium, trophoblast, uterine epithelium and mater-\nnal capillary endothelium (state 1, e.g. Sus). In\nendotheliochorial placentae, the uterine epithelial\nlayer is absent (state 2, e.g. Felis). In hemochorial\nplacentae, the trophoblast is bathed by maternal\nblood (state 3, e.g. Oryctolagus). In both states\n2 and 3, the fetal parts of the placenta,\ni.e.,\ntrophoblast and capillary endothelium, remain\nintact.\nCharacter 1: Gross form – State 1 5 diffuse; state 2 5 zonary; state 3 5 cotyledonary; state 4 5 discoid; state 5 5 double discoid.\nCharacter 2: Overall structure – State 1 5 labyrinthine; state 2 5 trabecular to villous.\nCharacter 3: Umbilical vessels – State 1 5 one artery and one vein; state 2 5 two arteries and one vein; state 3 5 two arteries and two veins.\nCharacter 4: Interhemal barrier – State 1 5 epitheliochorial; state 2 5 endotheliochorial; state 3 5 hemochorial.\nCharacter 5: Type of trophoblast in barrier – State 1 5 cytotrophoblast; state 2 5 syncytiotrophoblast; state 3 5 both.\nCharacter 6: Trophoblastic fenestrae – State 1 5 absent; state 2 5 present.\nCharacter 7: Hemophagous regions – State 1 5 present; state 2 5 absent.\nCharacter 8: Areolae – State 1 5 present; state 2 5 absent.\nCharacter 9: Differentiation of endometrial stroma – State 1 5 decidual cells atypical or absent; state 2 5 decidual cells present.\ncharacter\nas\nendotheliochorial for the stem species pattern\nof Eutheria and of Afrotheria, Euarchontoglires\nand Laurasiatheria,\nis the\nplesiomorphic condition for Eutherians (Fig. 1A).\nIn contrast, the epitheliochorial type is present as\nan apomorphic character condition in the stem\nspecies of Ferungulata. Transformations to the\nhemochorial state occur in all major clades, e.g., in\nit\nGlires plus relatives as well as in tenrecs, golden\nmoles and Elephantulus.\nType of trophoblast in interhemal barrier\nThe type of trophoblast present in the inter-\nhemal barrier differs within Eutheria (character 5;\nEnders, ’65). Trophoblast occurs as discrete cells,\ncytotrophoblasts, or as syncytiotrophoblast. Both\ntypes of trophoblast may be present as discrete\nlayers in the interhemal barrier (state 3, e.g. Mus).\nCommonly, only syncytiotrophoblast is found in\nthe interhemal membrane, at least in the late or\ndeﬁnitive placenta, even when cytotrophoblasts\nare present elsewhere (state 2, e.g. Homo). Less\nfrequently, only cytotrophoblast is found in the\ninterhemal region (state 1, e.g. Loxodonta). The\ncow, which possesses binucleate trophoblast cells,\nhas been coded as state 1, because the interhemal\nbarrier is likewise characterized by discrete cells.\nIn the cat and dog, remnants of cytotrophoblast\noccur in the interhemal membrane at term, but do\nnot form a continuous layer; we have scored them\nas state 2. In contrast, according to the detailed\ndocumentation of De Lange (’33), Manis possesses\na one-layered trophoblast in the late placenta,\nconsisting generally of cytotrophoblast, although\nwith some fused cells in between (which form a\nseparate layer of syncytiotrophoblast in earlier\nstages). Since a true layer of syncytiotrophoblast is\nabsent in the deﬁnitive placenta, we have coded\nManis as state 1.\nThe interhemal membrane contains syncytio-\ntrophoblast to the exclusion of cytotrophoblast in\nthe stem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria (Fig. 1B).\nTransformations occur within all major clades,\ne.g., the exclusive occurrence of cytotrophoblast as\na derived trait of Ferungulata and chiropterans,\nand the more layered condition within Glires and\nCynocephalus as well as in the moles.\nFenestrated trophoblast\nThere is usually some reduction or thinning of\nfetal tissue in order to reduce the interhemal\ndistance. In some species, however, the tropho-\nblast is also fenestrated (character 6, state 2, e.g.\nSuncus). In most eutherians such fenestrae are\nabsent (state 1, e.g. Talpa).\nFenestrae are a specialization not present in the\nstem species pattern of Eutheria, Afrotheria,\nEuarchontoglires or Laurasiatheria. Thus, non-\nfenestrated trophoblast is the plesiomorphic con-\ndition for those taxa. Transformations occur\nwithin Laurasiatheria in the shrews and conver-\ngently in Bradypus as a member of Xenarthra.\nHemophagous regions\nMany placentae have areas in which extrava-\nsated maternal erythrocytes are engulfed by\nspecialized trophoblast cells (Burton, ’82). These\nare sometimes referred to as hemotomes, but the\npreferred term is hemophagous regions (character\n7, state 1, e.g. Tenrec). Hemophagous regions\nusually are obvious because of their accumulation\nof pigments. Other placentae lack a dedicated\nhemophagous region (state 2, e.g. Cavia). In some\ninstances, the placenta has columnar trophoblast\nthat is well adapted for phagocytosis and will\ningest cell or tissue fragments or glandular\nsecretions. In such heterophagous regions, ery-\nthrocytes will be ingested if available, although\nthis does not seem to be their primary function.\nAn example discussed by Prasad et al. (’79) is the\nareola of the mole placenta. Similarly, we have\nseen little evidence to support Wislocki’s (’35)\ninterpretation of the columnar trophoblast in\nTrichechus as being primarily a hemophagous\norgan. Both mole species and Trichechus have\naccordingly been scored as state 2.\nHemophagous regions are not part of the stem\nspecies pattern of Eutheria, Laurasiatheria or\nEuarchontoglires, although transformations occur\nwithin the last two clades. According to the\nstates,\npresumed\na hemophagous organ is likewise absent in the\nstem species pattern of Afrotheria, and evolution-\nary transformations occur towards the tenrecs and\nthe elephant.\ncharacter\nAreolae\nVarious other accessory structures within the\nchorioallantoic placenta are concerned with ab-\nsorption of maternal secretions and cell debris, so-\ncalled histiotrophic nutrition (Mossman, ’87; Mess\net al., 2003). Typical of these are areolae associated\nwith uterine glands (character 8, state 1, e.g.\nTalpa, Sus), i.e., small, usually circular, domed,\npouched or plicated areas lined by columnar\nabsorptive and phagocytic trophoblast (Mossman,\n’87). Most placentae lack areolae (state 2, e.g.\nSorex). Mossman (’87) provided a catalog of\nadditional structures functioning as accessory\nplacental structures, including absorptive plaques,\narcades, absorptive villi and marginal folds. In a\nnumber of genera included in our analysis, some of\nthese additional structures occur either in addi-\ntion to areolae or in their absence. We were unable\nto obtain substantial resolution on the evolution of\nthese structures, however, and therefore chose to\nfocus on areolae. Thus, genera coded as state\n2 may have different arrangements for ingesting\nhistiotrophe.\nAreolae are another specialization missing from\nthe stem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria (see Fig. 1C).\nThey are, however, found in the stem of Ferungu-\nlata as well as convergently in moles and Galago.\nWithin Afrotheria they occur as transformations\nin Tenrec and Orycteropus.\nDifferentiation of endometrial stroma\nDecidual cells are found in the pregnant or\npseudopregnant uterus of some mammals. They\nare differentiated from the stroma by hypertrophy\nand functional modiﬁcation of connective tissue\ncells (Mossman, ’87). Typical decidual cells occur\nin many eutherians including anthropoid primates\n(character 9, state 2, e.g. Homo). In other cases,\nincluding strepsirhine primates, decidualization\ndoes not take place (state 1, e.g. Galago). Taxa\nreported to have atypical decidual cells, including\nelephants and carnivores (Mossman,\nbeen scored as state 1.\n’87), have\nDecidual cells occur in the stem species pattern\nof Eutheria, Afrotheria, Euarchontoglires and\nLaurasiatheria, signifying the plesiomorphic char-\nacter condition. Transformations toward the apo-\nmorphic condition, considered currently as a loss\nof decidual cells during pregnancy, occur in\nFerungulata, Scalopus and Galago. Moreover,\nwithin Afrotheria, decidual cells are judged to be\nlost in Echinops and Orycteropus; and conver-\ngently they are only atypically developed in\nLoxodonta (cf. Mossman, ’87).\nCharacters associated with implantation\nand the fetal membranes\nUnlike the chorioallantoic placenta, these are\ncharacters that can be coded both for marsupials\nand for further outgroups (not treated here). They\nare tabulated separately, together with the type of\nthe newborn (Table 3,\nuterus and state of\nFigs. 3–5).\nImplantation: ﬁrst attachment\nMossman (’87) remarked that the site of ﬁrst\nattachment (character 10) was constant within\norders. Central attachment of a relatively ex-\npanded blastocyst is sometimes found (state 1,\ne.g. Canis). Otherwise, attachment is on the\nmesometrial side of the uterus (state 2, e.g. Sus),\nor more commonly on the antimesometrial side\n(state 3, e.g. Sorex). Less frequently, ﬁrst attach-\nment is to either or both of the lateral walls (state\n4, e.g. Eremitalpa). In shrews the trophoblastic\nannulus ﬁrst attaches to the lateral walls of a\npocket on the antimesometrial side of the uterus.\nWe chose to score this as state 3.\nAccording to our analysis, antimesometrial\nattachment of the blastocyst during implantation\nremains as the plesiomorphic condition, achieved\nin the stem species pattern of Eutheria, Afrother-\nia, Euarchontoglires and Laurasiatheria (Fig. 3A).\nTransformations occur in each of the major clades,\ne.g., the occurrence of mesometrial attachment in\nFerungulata plus Chiroptera.\nImplantation: depth of nidation\nSome variation occurs in the depth of implanta-\ntion (character 11; Mossman,\n’37). Superﬁcial\nattachment of the blastocyst to the endometrium\n(state 1, e.g. Echinops) is far more common than\n(primary) interstitial attachment, where the blas-\ntocyst penetrates deep into the endometrium\nCharacter 10: Implantation (ﬁrst attachment) – State 1 5 central; state 2 5 mesometrial; state 3 5 antimesometrial; state 4 5 lateral.\nCharacter 11: Implantation (depth of nidation) – State 1 5 superﬁcial; state 2 5 secondarily interstitial; state 3 5 primarily interstitial.\nCharacter 12: Amniogenesis – State 1 5 cavitation; state 2 5 folding.\nCharacter 13: Bilaminar omphalopleure – State 1 5 permanent; state 2 5 temporary; state 3 5 only in blastocyst.\nCharacter 14: Trilaminar omphalopleure – State 1 5 permanent; state 2 5 temporary; state 3 5 absent.\nCharacter 15: Deﬁnitive yolk sac – State 1 5 free or absent; state 2 5 trilaminar; state 3 5 inverted.\nCharacter 16: Allantoic vesicle – State 1 5 permanent; state 2 5 temporary; state 3 5 rudimentary or absent.\nCharacter 17: Allantoic vesicle size – State 1 5 large to medium; state 2 5 small; state 3 5 rudimentary or absent.\nCharacter 18: Allantoic vesicle chambers – State 1 5 one; state 2 5 four; state 3 5 two; state 4 5 absent.\nCharacter 19: Urogenital opening – State 1 5 ‘‘cloaca’’; state 2 5 separate.\nCharacter 20: Type of uterus – State 1 5 unfused horns; state 2 5 duplex; state 3 5 bicornuate; state 4 5 simplex.\nCharacter 21: Newborn – State 1 5 eyes open at birth (precocial); state 2 5 eyes closed at birth (altricial).\n(state 3, e.g. Cavia). Sometimes the blastocyst\nattaches superﬁcially in a pocket that subse-\nquently closes over it; this is known as secondary\ninterstitial implantation (state 2, e.g. Erinaceus).\nSuperﬁcial attachment is the plesiomorphic con-\ndition and part of the stem species pattern of\nEutheria. Within Eutheria, this character is highly\nconserved, occurring in the stem species pattern\nof all higher systematic clades. Within our data\nset,\ninterstitial\nimplantation occur only in Cavia and Homo,\nwhilst secondary interstitial implantation is seen\nin Erinaceus, Suncus, Mus and Elephantulus.\nthese apomorphic\nAccording to our analysis,\nconditions are convergently established in most of\nthe taxa included. Even for Rodentia, where the\npattern is not resolved, we know about several\ntaxa with superﬁcial implantation, indicating an\nindependent achievement of more invasive blas-\ntocyst attachment (e.g. Luckett,\n’85; Mossman,\n’87; Mess, 2003).\nAmniogenesis\nMuch signiﬁcance has been attached to the way\nin which the amnion is formed (character 12;\nMossman, ’87). In marsupials and most orders of\neutherian mammals, the amnion develops as folds\nof the extraembryonic somatopleure at the cepha-\nlic end and sides of the early embryo. The folds\nmeet and fuse dorsal to the embryo (state 2; e.g.\nSus). Amnion formation by folding also occurs in\nreptiles, birds and monotremes. Several mammals\nhave no amniotic folds or only rudimentary ones.\nIn these the amnion forms by cavitation,\ni.e.,\nintercellular spaces arise in the inner cell mass\nand coalesce to form the amniotic cavity (state 1,\ne.g. Cavia). According to Moghe (’56), in Cynop-\nterus a primary amniotic cavity is formed by\ncavitation, but later folds arise and form another\ncavity which fuses with the ﬁrst. Therefore, we\nhave chosen not to code Cynopterus for this\ncharacter. Similar considerations apply to Mus\n(Luckett, ’85).\nThe stem species pattern for Eutheria, Euarch-\nontoglires and Laurasiatheria is amniogenesis by\nfolding as the plesiomorphic character condition of\nthe group (Fig. 3B). Transformations to the\napomorphic condition, i.e., cavitation, occur con-\nvergently in Erinaceus, Cavia, Cynocephalus and\nHomo.\nindependently from the\naforementioned taxa, the stem species pattern\nfor Afrotheria likewise is amniogenesis by cavita-\ntion. This character state occurs in the armadillos\n(Enders, 2002) and probably in Bradypus (Moss-\nman, ’87), indicating a character transformation\non the stem lineage of Afrotheria plus Xenarthra.\nBilaminar omphalopleure\nThe arrangement of the yolk sac wall varies\nmarkedly, and different regions—characterized by\nthe germ layers involved—exist during pregnancy.\nBecause of their distinct structural design, we\nhave decided to treat the presence and absence of\nthe most important regions as discrete characters.\nFirst of all, in its earliest developmental stage, the\nyolk sac is composed of trophoblastic ectoderm\nlined by extraembryonic endoderm: a bilaminar\nomphalopleure (character 13; Steven, ’75; Perry,\n’81; Mossman, ’87). This condition may occur only\nin the blastocyst (state 3, e.g. Homo). More\ncommonly it is retained into later development,\nbut as a temporary structure (state 2, e.g. Tenrec).\nOccasionally, however, a portion of juxtauterine\nbilaminar omphalopleure is maintained until term\n(state 1, e.g. Erinaceus). It should be noted that in\nrodents with inversion of the yolk sac (see below),\nthe periplacental part of the bilaminar omphalo-\npleure with its Reichert’s membrane persists as\nthe covering of the disk of the chorioallantoic\nplacenta. However, a juxtauterine bilaminar om-\nphalopleure occurs only in the blastocyst; conse-\nquently, these species (Mus, Cavia) have been\ncoded as state 3.\nA temporary bilaminar omphalopleure is pre-\nsent in the stem species pattern of Eutheria,\nAfrotheria, Euarchontoglires and Laurasiatheria,\nbut apparently does not occur in Xenarthra as\nindicated by the single genus Bradypus. Several\ntransformations to a permanent bilaminar om-\nphalopleure occur, most notably in all representa-\ntives of Lipotyphla. In addition, others such as\nHomo are characterized by another apomorphic\ncondition,\ni.e., the restriction of the bilaminar\nomphalopleure to the blastocyst stage only.\nTrilaminar omphalopleure\nA trilaminar omphalopleure is formed when\nextraembryonic mesoderm is interposed between\nthe trophoblast and endoderm, bringing with it\nfetal blood vessels (character 14; Perry, ’81). If it is\napplied to the uterine wall, allowing maternal–\nit constitutes a choriovitelline\nfetal exchange,\n’87, see character 15). In\nplacenta (Mossman,\nmost orders of Eutheria, the trilaminar omphalo-\npleure is a temporary structure (state 2, e.g.\nEchinops). Very few eutherian mammals, but all\nmarsupials, retain the trilaminar omphalopleure\nto term (state 1, e.g. Tupaia). In some mammals a\ntrilaminar omphalopleure is never formed (state 3,\ne.g. Homo, Cavia).\nSimilar to the previous character, a temporary\ntrilaminar omphalopleure is present in the stem\nspecies pattern of Eutheria, Afrotheria, Euarch-\nontoglires and Laurasiatheria (Fig. 3C). Character\ntransformation towards complete absence of a\ntrilaminar omphalopleure or choriovitelline pla-\ncenta took place convergently in Bradypus, Homo\nand the two representatives of Rodentia.\nIn\ncontrast, Tupaia and Eremitalpa are the only\nspecies in the set with a permanent trilaminar\nomphalopleure, representing an independently\nachieved derived character condition. Moreover,\nthe marsupial Didelphis likewise possesses a\npermanent trilaminar omphalopeure.\nDeﬁnitive yolk sac\nFinally, the association of the yolk sac to the\nmaternal uterine epithelium will be considered.\nTo function as a placenta, the trilaminar yolk sac\nmust maintain contact with the uterine wall, in\norder to take up histiotrophe (Mossman, ’87; Mess\net al., 2003). All marsupials rely on yolk sac\nplacentation. In Eutheria, such a yolk sac placenta\nis only rarely present during the later stages of\npregnancy when fetomaternal exchange has to be\nmost efﬁcient. In these cases, the trilaminar\nomphalopleure is retained until term in close\nassociation with the maternal epithelium (char-\nacter 15, state 2, e.g. Tupaia). However, this may\nnot be the case at term, either because the yolk sac\nis not formed or because it has been displaced and\nnow lies free in the exocelom (state 1, e.g.\nBradypus). More often there is partial or complete\ninversion of the germ layers (state 3, e.g. Cavia);\npartial\ninversion is seen in insectivores and\ncomplete inversion occurs in many rodents. This\ntype could serve as a so-called inverted yolk sac\nplacenta, characterized by an inverse organization\nof layers that are apposed to the maternal uterine\nepithelium. However, the situation is so different\nfrom that of state 2 that we have decided to treat it\nas a separate character state.\nAt term, in nearly all orders the yolk sac either\nhas not formed, or it has formed but subsequently\nregressed or lost contact with the uterine wall.\nConsequently, the absence of a choriovitelline\nplacenta is a plesiomorphic character condition\noccurring in the stem species pattern of Eutheria,\nAfrotheria, Euarchontoglires and Laurasiatheria.\nIt is also present in Bradypus. Partial or complete\ninversion of the yolk sac occurs as an apomorphic\ncondition in Lipotyphla, and is independently\nachieved by Glires and the ﬂying lemur, Cynoce-\nphalus. A true yolk sac placenta at term is\nretained only in Tupaia and Eremitalpa, and\nconsequently must be regarded as a derived\ncharacter condition within Eutheria.\nAllantoic vesicle\nIn most mammals the allantois encloses a sac,\nthe allantoic vesicle (character 16), which is\nretained throughout pregnancy (state 1, e.g.\nEquus). In some, the vesicle is a temporary\nstructure (state 2, e.g. Tadarida). In others there\nis an allantoic stalk bearing blood vessels to the\nchorioallantoic placenta, but the vesicle is rudi-\nmentary or absent (state 3, e.g. Sorex).\nA permanent allantoic vesicle is a feature of the\nstem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria. Some trans-\nformations occur, e.g., the allantoic vesicle is\nrudimentary or absent in shrews, rodents and\nHomo as well as in Bradypus.\nAllantoic vesicle size\nThere are considerable variations in the size of\nthe allantoic vesicle (character 17), that have been\nsigniﬁcance (Portmann,\nascribed phylogenetic\n’38). We have combined medium and large sizes\n(state 1, e.g. Equus) as the descriptions available\ndo not enable us to make the distinction given for\nselected species by Portmann (’38). Some euther-\nians have only a small allantoic vesicle (state 2, e.g.\nOryctolagus) and in others it is rudimentary or\nabsent (state 3, e.g. Homo).\nA medium or large allantoic vesicle is another\nfeature of the stem species pattern of Eutheria,\nAfrotheria, Euarchontoglires and Laurasiatheria.\nTransformation to a small vesicle occurs in\nTadarida, Oryctolagus and Cynocephalus and to\na rudimentary or absent vesicle in shrews,\nrodents, Homo and Bradypus.\nAllantoic vesicle chambers\nThe allantoic vesicle is usually a single sac\n(character 18, state 1, e.g. Felis) or is bilobed\n(state 3, e.g. Tursiops). In some instances, how-\never, it forms four lobes (state 2, e.g. Orycteropus);\nthis is always associated with early division of the\numbilical cord giving rise to four sets of vessels\nthat subtend the septal folds of the allantois. As\nalready noted, the vesicle is absent in some species\n(state 4, e.g. Mus).\nRepresenting the plesiomorphic condition of\nEutheria and Metatheria,\nin the stem species\npattern of Eutheria, Euarchontoglires and Laur-\nasiatheria, as well as in Bradypus, the allantoic\nvesicle is a single chamber (Fig. 4A). Transforma-\ntion to a bilobed state occurs within Cetartiodac-\ntyla, i.e., in Tursiops and Hippopotamus. Of note,\nthe stem species pattern of Afrotheria is a four-\nlobed allantoic sac, this being a synapomorphic\ntrait of Afrotheria. A separate, i.e., convergent\ntransformation to a four-lobed conformation likely\noccurs in Otolemur, as Hill (’32) considered this to\nbe typical of the strepsirhine allantois.\nCharacters associated with the female\nreproductive tract and newborn\nUrogenital opening\nIn reptiles, birds and monotremes, the anal and\nurogenital tracts open into a common recess, the\ncloaca. In marsupials and some eutherians, they\nopen into a common skin fold that Starck (’82)\nregards as a secondary cloaca (character 19, state\n1). This condition occurs in Xenarthra (e.g.\ntenrecs, otter\nBradypus), many insectivores,\nshrews (P. Vogel, personal communication) and\nin the lagomorph, Ochotona. Shoshani and\nMcKenna (’98, char. 215) assigned this state to\nmarsupials but not, for example, to lagomorphs, a\ndistinction that does not seem justiﬁed based on\ntheir description of Macropus. In most eutherians,\nthe anal and urogenital tracts have separate\nperineal openings (state 2, e.g. Homo).\nWe were unable to resolve the stem species\npattern for Eutheria, particularly because of\ndifferent conditions in basal branches and in the\nmarsupial (Fig. 4B). However, separate openings\nare the norm and this state is present in the stem\nspecies pattern of Laurasiatheria, Euarchonto-\nglires and Afrotheria.\nType of uterus\nThere are several possible arrangements of the\nuterine lumen (character 20; Mossman,\n’87;\nShoshani and McKenna, ’98; character 223). In\nmarsupials the uterine tubes are not fused, each\ncommunicating with the urogenital sinus through\na separate vagina (state 1). This represents a true\nmetatherian–eutherian dichotomy (Harder et al.,\n’93). In a duplex uterus, the uterine tubes open\nindependently into the vagina (state 2, e.g.\nOrycteropus). In a bicornuate uterus, the uterine\nhorns join internally to form the body (corpus)\nwhich communicates with the vagina through\na common cervical canal\n(state 3, e.g. Sus).\nA simplex uterus with a single unpaired corpus\nbut only rudimentary horns (state 4, e.g. Homo)\nis rare among mammals (Martin, 2003). A more\ndetailed analysis of uterine types is given by\nMossman (’87; Table 9.1).\nA bicornuate uterus is present in the stem\nspecies pattern of Eutheria and in three of the\nmajor clades (Afrotheria, Euarchontoglires and\nLaurasiatheria). The tree contains only one\nrepresentative of Xenarthra (Bradypus), but a\nin armadillos and\nsimplex uterus is present\nanteaters as well as sloths (Wislocki, ’28). Thus,\ndata suggest that Xenarthra seems to be the only\nsuperordinal taxon that is characterized by an\napomorphic condition, i.e., the uterus simplex.\nNewborn\nThe developmental condition of the newborn\n(character 21) likewise varies in Eutheria. In\naltricial species, the newborn young are relatively\nimmobile, have closed eyes, lack hair and require\nparental care for a comparatively long time. In\nthe newborn are relatively\nprecocial species,\nmature and mobile from the moment of birth.\nWe follow Shoshani and McKenna (’98, char. 249)\nin coding for eyes open at birth (state 1, e.g.\nCavia), which usually implies precociality, or eyes\nclosed at birth (state 2, e.g. Mus), which usually\ncharacterizes altricial young. Clearly this is a\nsimpliﬁcation. For example,\nthe state of eye\nopening in chiropterans is not the most reliable\nindicator of developmental state (Kunz and Hood,\n2000). In addition, the human newborn is some-\ntimes characterized as secondarily altricial (Mar-\ntin, 2003), but is born with open eyes and is coded\nas state 1. Naaktgeboren and Slijper (’70) was a\nuseful source for this character. Dr. Peter Vogel\n(personal communication) provided information\non Potamogale and Dr. Thomas Kunz (personal\ncommunication) on Tadarida.\nThis character has an interesting distribution.\nRepresenting a plesiomorphic character condition\nfor the Eutheria, the stem species pattern of\nEutheria is characterized by altricial newborns, as\nfor Euarchontoglires and Laurasiatheria (Fig. 4C).\nWithin the latter clade, however, precocial new-\nborns are part of the stem pattern of Ferungulata\nand the same transformation occurs in Cavia (and\nother hystricognath rodents; Mess, 2003) and in\nprimates. Precocial young are part of the stem\nspecies pattern of Xenarthra as indicated by their\nsingle representative Bradypus as well as of the\nstem species pattern of Afrotheria, but with\nsecondary transformation to altricial newborns\nin Tenrecoidea.",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1220551%26volume%3d177%26issue%3d2%26page%3d1070%26filename%3dzim01406001070.pdf%26languageId%3d",
         "ABSTRACT\nAnalysis of molecular data sets has provided new insights into higher-level\nrelationships of living Eutheria, including the recognition of Afrotheria as a novel taxon. This offers\nan opportunity to take a fresh look at the evolution of organ systems, including some that are little\nused in traditional systematics. In the present study, we attempted a reconstruction of the evolution\nof characters associated with placentation, the fetal membranes and the female reproductive tract.\nThe evolutionary history of 21 characters has been traced, based on a current hypothesis of\neutherian relationships, by applying the computer program MacClade. Accordingly, the analysis\nprovides a ﬁrst comprehensive interpretation of the stem species pattern of Eutheria. Of particular\nnote, this pattern includes an endotheliochorial chorioallantoic placenta. The reconstructed pattern\nof Eutheria does not change in the basal nodes of the group. Thus, no character transformations\noccur on the stem lineages of Laurasiatheria or Euarchontoglires, and even Afrotheria has mostly\nplesiomorphic character conditions. However, two character transformations occur on the common\nstem lineage of Afrotheria and its sister taxon Xenarthra, i.e., amniogenesis by cavitation instead\nof folding and the precocial state of the newborn. In addition, we recognized one character\ntransformation on the stem lineage of Afrotheria, i.e., the occurrence of a four-lobed allantoic\nsac. Thus, contrary to previous assertions, it is possible to identify morphological characters that\ncould be synapomorphic for this novel taxon. J. Exp. Zool. (Mol. Dev. Evol.) 306B:140– 163, 2006.\nr 2005 Wiley-Liss, Inc.\nIn recent years, analysis of molecular data sets\nhas led to a revolution in mammalian systematics\nand revealed completely new higher-level relation-\nships within Eutheria (Springer et al., ’97; Stan-\nhope et al., ’98). Of particular note is the view that\nall extant eutherian species belong to one of four\nsuperordinal clades (Madsen et al., 2001; Murphy\net al., 2001a,b; Waddell et al., 2001; Springer et al.,\n2005a,b). These ideas about mammalian evolution\nhave provoked fruitful debate and stimulated\nresearch to test them by using morphological data\n(Asher, 2001; Symonds, 2005).\nThe orders of Eutheria recognized by molecular\nphylogeneticists are quite stable in relation to the\ntraditional morphological\nsystem (Archibald,\n2003), but there are two exceptions. The ﬁrst,\nthe inclusion of whales within Artiodactyla (as\nCetartiodactyla), is not controversial since these\norders are grouped close together in the tradi-\n’88; Shoshani and McKenna,\ntional system (Novacek and Wyss, ’86; Novacek\net al.,\n’98). The\nsecond, exclusion of tenrecs and golden moles from\nLipotyphla,\nis more contentious (Asher, 2003).\nThey are placed in a new order that has been\ncalled Afrosoricida (Stanhope et al., ’98), although\nTenrecoidea (McDowell,\n’58) takes precedence\n(Asher, 2005; Symonds, 2005).\nThe main outcome of molecular systematics is\nthe linkage of tenrecs and golden moles to other\norders originating in Africa, and the establish-\nment of a novel clade called Afrotheria (Stanhope\net al.,\n’98). This includes Proboscidea, Sirenia,\nHyracoidea, Macroscelidea, Tubulidentata and\nTenrecoidea. Although the composition of Lipo-\ntyphla has been enigmatic (Simpson, ’45; MacPhee\nand Novacek, ’93; Symonds, 2005), inclusion of\ntenrecoids with taxa such as Paenungulata (Simp-\nson,\n’45), was never before suggested. Recent\ninvestigations to test the Afrotheria hypothesis\nwith morphological data have failed to support it\n(Asher, 2001; Whidden, 2002). Therefore, there\nmay be good reason to include morphological data\nsets from other systems, which are less used in\ntraditional systematics.\nOne such character complex comprises placenta-\ntion, fetal membranes and the female reproductive\ntract. There is great variation in gross and\nﬁne morphology of the placenta in eutherians\n(Hubrecht, ’09; Starck, ’59; Mossman, ’87) and\nthere is currently speculation about the evolu-\ntionary forces leading to such diversity (Crespi and\nSemeniuk, 2004; Vogel, 2005). Much detailed\ninformation is available from original studies.\nNevertheless, very few of these characters have\nbeen incorporated in phylogenetic analyses of\n’86;\nmorphological data (Novacek and Wyss,\nNovacek et al., ’88; Luckett, ’93; Shoshani and\nMcKenna, ’98; Asher et al., 2003). A comprehen-\nsive evaluation of the large amount of data within\na phylogenetic framework seems very promising.\nTherefore,\nin the present study, we have at-\ntempted a reconstruction of the evolution of such\ncharacters\nin living Eutheria. The analysis\nis based on a pre-existing hypothesis of inter-\nordinal relationships (Douady et al., 2002b) and\nthe tracing of character evolution according\nto Afrotheria and other superordinal clades.\nMoreover, a reconstruction of the stem species\npattern of extant Eutheria (Placentalia) is given\nsince for several of these characters,\ni.e., the\ninterhemal barrier within the chorioallantoic\nplacenta,\nthe ancient condition of eutherians\nhas so far not been resolved.\nMATERIALS AND METHODS\nThe goal of the current analysis was to reveal\nthe evolution of placental characters within living\nEutheria and their higher-level clades. Accord-\ningly, we reconstructed the stem species pattern\nor ancient condition of relevant taxa, such as\nAfrotheria, in order to discover transformations\nbetween subsequent nodes or stem species pat-\nterns. A stem species pattern is deﬁned as the\n’92, 2000; Cunningham et al.,\ncharacter set of the last common ancestor of\na monophyletic taxon,\nincluding plesiomorphic\nand derived or apomorphic characters (Ko¨nigs-\nmann, ’75; Lieven and Sudhaus, 2000). For this\npurpose, the distribution of selected characters\nwas mapped on a current hypothesis of phyloge-\nnetic relationships within Eutheria (see below).\nThe evolutionary history of characters is revealed\nby character optimization (see Maddison and\nMaddison,\n’98;\nRieppel,\ncomputer program\nthe\nMacCladeTM (Version 4.0: David R. Maddison\nand Wayne P. Maddison, University of Arizona,\nSinauer Associates). MacClade is a widely accepted\ntool for analyzing phylogenetic hypotheses and\nreconstructing and displaying character evolution\non a phylogenetic tree. Character optimization is\nan approach previously used to trace character\nevolution in fetal membranes (Freyer et al., 2003;\nMess, 2003; Asher, 2005; Vogel, 2005). Moreover,\nthe parsimony-based method of character analysis\nin MacClade easily allows handling of missing\ndata. This is an enormous advantage in compar-\nison with maximum likelihood analysis (e.g. the\nprogram MesquiteTM, designed by the same\nauthors, which includes maximum likelihood op-\ntions adapted for morphological characters). In-\ndeed, for about half the characters in our study,\ndata are missing for several species, and thus they\nare not suited for such treatment.\nThough molecular analyses are currently con-\nverging in a well-resolved pattern of recent\nEutheria, characterized by four major groups\n(see Springer et al., 2005a,b), the individual papers\ndiffer in respect of taxa and molecules as well as in\nsome aspects of tree topology. We are well aware\nthat any topology chosen for analysis will have\nsome advantages and some shortcomings. Thus,\nsome background information on the chosen\ntopology and possible alternatives is needed.\nAnalyses that include molecular data are agreed\non an afrotherian clade including tenrecs and\ngolden moles; nonetheless, there are competing\ntopologies as summarized by Asher (2005) and\nSymonds (2005). We have chosen to focus on a tree\n(Fig. 2 in Douady et al., 2002b) that was based on a\nmaximum likelihood analysis of exon sequences\nfrom three nuclear genes: alpha-2B adrenergic\nreceptor, von Willebrand factor and breast-cancer-\nassociated gene 1. This is a relatively extensive\nmolecular data set in contrast to topologies that\nrely exclusively on mitochondrial DNA (e.g.\nArnason et al., 2002). Secondly, this tree (Douady\net al., 2002b) includes eight core insectivores,\nthree tenrecs and a golden mole, species that are\nregarded to be critical to an understanding of\nplacental evolution. Other topologies (Murphy\net al., 2001a,b; Waddell et al., 2001; Malia et al.,\n2002; Amrine-Madsen et al., 2003) have rather too\nfew of the species of interest and/or species for\nwhich insufﬁcient data on the placenta are\navailable. Two alternative topologies that are rich\nin insectivoran grade species (Douady et al.,\n2002a; Asher et al., 2003) ﬁnd a basal position\nfor murid rodents. As shown by Douzery and\nHuchon (2004), this likely is the result of basing\nthe analyses on models of sequence evolution that\nﬁt the data poorly and the use of outgroups that\nare too divergent. The tree of Douady et al.\n(2002b) does not suffer from this problem.\nAs recently reviewed by Springer et al. (2005a),\nthere are different possible locations for the root of\nthe placental tree. In Douady et al. (2002b), this is\nbetween Xenarthra1Afrotheria and Euarchonto-\nglires1Laurasiatheria (Boreoeutheria). Other mo-\nlecular studies, including supertree analysis, favor\na rooting between Afrotheria on the one hand and\nall other placentals on the other (Murphy et al.,\n2001a,b; Amrine-Madsen et al., 2003). Since these\ndifferences in basal position could be crucial for\nthe analysis, we have examined the consequences\nof re-rooting the current tree in the second\nlocation (see discussion, stem species pattern of\nEutheria). The available molecular phylogenies\ndiffer in some other points, as acknowledged\nwithin the discussion. Last but not least, Soleno-\ndon is not part of the set of Douady et al. (2002b)\nand other topologies. However, a recent analysis\n(Roca et al., 2004) found Solenodon to be basal\nto the Lipotyphla with the topology for the re-\nmainder of the order being similar to that used\nin a separate analysis, we\nhere. Therefore,\nadded Solenodon to the tree (see discussion,\nLaurasiatheria).\nIn order to map our characters on the selected\nphylogeny, we next searched the literature for\ndata on placentation in the 301 genera on which\nthe tree is based (we are grateful to Dr. Christophe\nJ. Douady for supplying us with a complete list of\nexamined species). In some instances, Douady\net al. (2002b) took sequences from two or three\nrelated genera, but we have chosen a single genus\nfor each branch of the tree. As an example, the\nmicrobats are represented by Tadarida in our\nanalysis, since placentation in this species is better\nknown than in Macrotus and Emballonura, which\nare also among the microbats in Douady’s data set.\nIn a few cases, data on placentation were missing\nor greatly deﬁcient. It was therefore decided to\nmake the following omissions and substitutions:\n(a) Cetacea: Douady et al. (2002b) used sequences\nfrom the\nbaleen whales Balaenoptera,\nMegaptera and Eschrichtius. We have substi-\ntuted a toothed whale, Tursiops truncatus,\nas this is the species for which placentation is\nbest known (Wislocki and Enders,\n’41).\nThe sparse data available for Balaenoptera\n(Turner, 1870) and on early development\n’60) are in\nin Megaptera (Stump et al.,\nagreement with the more complete study of\nTursiops.\n(b) Erinaceidae: In addition to Erinaceus, Douady\net al. (2002b) included the gymnure Hylomys\nin their analysis. A little is known about\nplacentation in a related genus, Echinosorex\n(Meister and Davis, ’53), but the study has\nbeen so severely criticized by Mossman (’87)\nthat we thought it preferable to omit the\ngymnures altogether.\n(c) Sirenia: Douady et al. (2002b) used sequences\nfrom Dugong. Although Turner (1889) de-\nscribed a poorly preserved placenta from this\nspecies, much better information is available\nfor the West Indian manatee, Trichechus\nmanatus (Wislocki, ’35); in addition, we have\nunpublished data on placental ultrastructure\nin the Amazonian manatee T. inunguis (Car-\nter, Enders, Miglino and Da Silva, unpublished\nobservations).\n(d) Proboscidea: Douady et al.\n(2002b) relied\nmainly on sequences from Elephas. Informa-\ntion is much more thorough on placentation in\nLoxodonta, and includes ultrastructural stu-\ndies (Allen et al., 2003; Wooding et al., 2005).\nData on the delivered placenta of Elephas\n(Cooper et al., ’64) are consistent with what is\nknown about Loxodonta.\n(e) Chrysochloridae: Douady et al. (2002b) used\nfrom Amblysomus. Nothing is\nsequences\nknown of placentation in this genus and we\nhave substituted Eremitalpa granti (Gabie,\n’59, ’60).\n(f) Metatheria: Douady et al.\n(2002b) used a\nkangaroo, Macropus giganteus. Information\non placentation is available for the tammar\nwallaby M. eugenii (e.g. Freyer et al., 2002),\nbut we have substituted Didelphis virginiana\nfor which more complete data are accessible.\nHowever, data are consistent between these\ntwo marsupials.\nA complete list of the species and principal\nliterature consulted is given in Table 1. In addition\nto these primary sources, we made use of reviews\nof placentation (De Lange, ’33; Starck, ’59; Steven,\nfemale reproductive tract\n’75; Mossman,\n(Starck, ’82) and newborns (Naaktgeboren and\n’70) and of standard reference works\nSlijper,\n(Grzimek,\n’99). Moreover, we fre-\nquently referred to a recent survey of comparative\nplacentation by Benirschke on the internet\n(Benirschke, 2005).\n’87),\nDespite the considerable amount of information\navailable on these organ systems, the number of\ncharacters that can be coded for the 34 genera in\nthe set is insufﬁcient for an independent cladistic\nanalysis, e.g., by PAUP.\nRESULTS\nCharacters associated with the\nchorioallantoic placenta\nA chorioallantoic placenta is not present in most\nincluding Didelphis and Macropus.\nmarsupials,\nThus,\nfor all characters associated with the\nchorioallantoic placenta, Didelphis has been as-\nsigned state 0, signifying that no chorioallantoic\nplacenta is present (Table 2). Some eutherian\nmammals exhibit different character states ac-\ncording to the stage of development. Since the\nmaterial available for the different species varies,\nbut usually includes the late or deﬁnitive placenta,\ncharacters 1–9 all refer to the deﬁnitive placenta\nfound near term (Table 2, Figs. 1 and 2). Some\nmammals, including tenrecs, elephant shrews and\nsome bats, have accessory placentae with char-\nacter states different from those of the main\nplacenta; these have been disregarded. Plotting\nthe character distribution on the selected tree of\nDouady et al. (2002b), the following information\non stem species patterns and character transfor-\nmation within Eutheria resulted.\nGross form of the placenta\nThere is considerable variation in the gross\nmorphology of the placenta (character 1; Steven,\n’87). In the diffuse type, the\n’75; Mossman,\nplacenta forms a sac covered with small villi or\nfolds, which interdigitate with corresponding\ndepressions on the uterine epithelium (state 1,\ne.g. Equus). In a cotyledonary placenta, the villi\nare concentrated in cotyledons, each of which is a\nsmall disk (state 3, e.g. Bos). In a zonary placenta,\nthe trophoblast forms an equatorial band of tissue\n(state 2, e.g. Canis). A single discoid placenta\noccurs in many species including man (state 4, e.g.\nCavia). Tree shrews and New World and Old\nWorld monkeys have a secondary disk and thus a\ndouble discoid placenta (state 5, e.g. Tupaia).\nA discoid placenta is present in the stem species\npattern of Eutheria, Euarchontoglires and Laur-\nasiatheria and is accordingly regarded as the\nplesiomorphic condition of those groups. This\ncharacter cannot be resolved for Afrotheria. With-\nin all superordinal taxa, a remarkable number of\ncharacter transformations occurs, indicating that\nthe gross form of the placenta possesses a high\ndegree of plasticity.\nOverall structure of the placenta\nAttention is often given to variations in the\noverall structure of the placenta (character 2;\nSteven, ’75; Mossman, ’87). In many placentae,\nthe maternal vessels or maternal blood channels of\nthe exchange area are completely enclosed in\ntrophoblast, forming a labyrinth (state 1, e.g.\nElephantulus). In a villous placenta, the fetal\nsurface is thrown into blind-ending projections or\nvilli which are bathed in maternal blood (human)\nor invested by the epithelial walls of maternal\ncrypts (cow). Intermediate forms occur with a\nlooser or trabecular structure. Because authors\ntend to differ in their deﬁnition of the trabecular\ntype, we have chosen to pool trabecular and villous\nplacentae (state 2, e.g. Tursiops).\nRepresenting the plesiomorphic condition of\neutherians, the labyrinthine pattern is found in\nthe stem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria. Transfor-\nmations toward the trabecular to villous type\noccur within the latter two clades, in particular,\nin the primates, including human beings, and in\nFerungulata.\nNumber of umbilical vessels\nEarly in development there are paired umbilical\narteries and veins. In many species, however, a\nreduction occurs so that the number of vessels in\nthe cord (character 3) varies from a single artery\nand vein (state 1, e.g. Mus), through two arteries\nand a vein (state 2, e.g. Cavia) to two arteries and\ntwo veins (state 3, e.g. Tursiops).\nTwo arteries and one vein are found in the stem\nspecies of Eutheria and this is a very regular\npattern, characterizing all superordinal taxa of\nthat group. Transformations to a pattern with two\nveins (and likewise two arteries) occur within\nLaurasiatheria and a reduction to one artery and\none vein is recognized for Mus. The later condition\nseems likewise to occur in Rattus and other\nmurids such as Phodopus (see Benirschke, 2005).\nInterhemal barrier\nThe number of tissue layers separating fetal\nfrom maternal blood varies (character 4; Grosser,\n’09; Enders, ’65; Mossman, ’87). In epitheliochorial\nplacentae, these include fetal capillary endothe-\nlium, trophoblast, uterine epithelium and mater-\nnal capillary endothelium (state 1, e.g. Sus). In\nendotheliochorial placentae, the uterine epithelial\nlayer is absent (state 2, e.g. Felis). In hemochorial\nplacentae, the trophoblast is bathed by maternal\nblood (state 3, e.g. Oryctolagus). In both states\n2 and 3, the fetal parts of the placenta,\ni.e.,\ntrophoblast and capillary endothelium, remain\nintact.\nCharacter 1: Gross form – State 1 5 diffuse; state 2 5 zonary; state 3 5 cotyledonary; state 4 5 discoid; state 5 5 double discoid.\nCharacter 2: Overall structure – State 1 5 labyrinthine; state 2 5 trabecular to villous.\nCharacter 3: Umbilical vessels – State 1 5 one artery and one vein; state 2 5 two arteries and one vein; state 3 5 two arteries and two veins.\nCharacter 4: Interhemal barrier – State 1 5 epitheliochorial; state 2 5 endotheliochorial; state 3 5 hemochorial.\nCharacter 5: Type of trophoblast in barrier – State 1 5 cytotrophoblast; state 2 5 syncytiotrophoblast; state 3 5 both.\nCharacter 6: Trophoblastic fenestrae – State 1 5 absent; state 2 5 present.\nCharacter 7: Hemophagous regions – State 1 5 present; state 2 5 absent.\nCharacter 8: Areolae – State 1 5 present; state 2 5 absent.\nCharacter 9: Differentiation of endometrial stroma – State 1 5 decidual cells atypical or absent; state 2 5 decidual cells present.\ncharacter\nas\nendotheliochorial for the stem species pattern\nof Eutheria and of Afrotheria, Euarchontoglires\nand Laurasiatheria,\nis the\nplesiomorphic condition for Eutherians (Fig. 1A).\nIn contrast, the epitheliochorial type is present as\nan apomorphic character condition in the stem\nspecies of Ferungulata. Transformations to the\nhemochorial state occur in all major clades, e.g., in\nit\nGlires plus relatives as well as in tenrecs, golden\nmoles and Elephantulus.\nType of trophoblast in interhemal barrier\nThe type of trophoblast present in the inter-\nhemal barrier differs within Eutheria (character 5;\nEnders, ’65). Trophoblast occurs as discrete cells,\ncytotrophoblasts, or as syncytiotrophoblast. Both\ntypes of trophoblast may be present as discrete\nlayers in the interhemal barrier (state 3, e.g. Mus).\nCommonly, only syncytiotrophoblast is found in\nthe interhemal membrane, at least in the late or\ndeﬁnitive placenta, even when cytotrophoblasts\nare present elsewhere (state 2, e.g. Homo). Less\nfrequently, only cytotrophoblast is found in the\ninterhemal region (state 1, e.g. Loxodonta). The\ncow, which possesses binucleate trophoblast cells,\nhas been coded as state 1, because the interhemal\nbarrier is likewise characterized by discrete cells.\nIn the cat and dog, remnants of cytotrophoblast\noccur in the interhemal membrane at term, but do\nnot form a continuous layer; we have scored them\nas state 2. In contrast, according to the detailed\ndocumentation of De Lange (’33), Manis possesses\na one-layered trophoblast in the late placenta,\nconsisting generally of cytotrophoblast, although\nwith some fused cells in between (which form a\nseparate layer of syncytiotrophoblast in earlier\nstages). Since a true layer of syncytiotrophoblast is\nabsent in the deﬁnitive placenta, we have coded\nManis as state 1.\nThe interhemal membrane contains syncytio-\ntrophoblast to the exclusion of cytotrophoblast in\nthe stem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria (Fig. 1B).\nTransformations occur within all major clades,\ne.g., the exclusive occurrence of cytotrophoblast as\na derived trait of Ferungulata and chiropterans,\nand the more layered condition within Glires and\nCynocephalus as well as in the moles.\nFenestrated trophoblast\nThere is usually some reduction or thinning of\nfetal tissue in order to reduce the interhemal\ndistance. In some species, however, the tropho-\nblast is also fenestrated (character 6, state 2, e.g.\nSuncus). In most eutherians such fenestrae are\nabsent (state 1, e.g. Talpa).\nFenestrae are a specialization not present in the\nstem species pattern of Eutheria, Afrotheria,\nEuarchontoglires or Laurasiatheria. Thus, non-\nfenestrated trophoblast is the plesiomorphic con-\ndition for those taxa. Transformations occur\nwithin Laurasiatheria in the shrews and conver-\ngently in Bradypus as a member of Xenarthra.\nHemophagous regions\nMany placentae have areas in which extrava-\nsated maternal erythrocytes are engulfed by\nspecialized trophoblast cells (Burton, ’82). These\nare sometimes referred to as hemotomes, but the\npreferred term is hemophagous regions (character\n7, state 1, e.g. Tenrec). Hemophagous regions\nusually are obvious because of their accumulation\nof pigments. Other placentae lack a dedicated\nhemophagous region (state 2, e.g. Cavia). In some\ninstances, the placenta has columnar trophoblast\nthat is well adapted for phagocytosis and will\ningest cell or tissue fragments or glandular\nsecretions. In such heterophagous regions, ery-\nthrocytes will be ingested if available, although\nthis does not seem to be their primary function.\nAn example discussed by Prasad et al. (’79) is the\nareola of the mole placenta. Similarly, we have\nseen little evidence to support Wislocki’s (’35)\ninterpretation of the columnar trophoblast in\nTrichechus as being primarily a hemophagous\norgan. Both mole species and Trichechus have\naccordingly been scored as state 2.\nHemophagous regions are not part of the stem\nspecies pattern of Eutheria, Laurasiatheria or\nEuarchontoglires, although transformations occur\nwithin the last two clades. According to the\nstates,\npresumed\na hemophagous organ is likewise absent in the\nstem species pattern of Afrotheria, and evolution-\nary transformations occur towards the tenrecs and\nthe elephant.\ncharacter\nAreolae\nVarious other accessory structures within the\nchorioallantoic placenta are concerned with ab-\nsorption of maternal secretions and cell debris, so-\ncalled histiotrophic nutrition (Mossman, ’87; Mess\net al., 2003). Typical of these are areolae associated\nwith uterine glands (character 8, state 1, e.g.\nTalpa, Sus), i.e., small, usually circular, domed,\npouched or plicated areas lined by columnar\nabsorptive and phagocytic trophoblast (Mossman,\n’87). Most placentae lack areolae (state 2, e.g.\nSorex). Mossman (’87) provided a catalog of\nadditional structures functioning as accessory\nplacental structures, including absorptive plaques,\narcades, absorptive villi and marginal folds. In a\nnumber of genera included in our analysis, some of\nthese additional structures occur either in addi-\ntion to areolae or in their absence. We were unable\nto obtain substantial resolution on the evolution of\nthese structures, however, and therefore chose to\nfocus on areolae. Thus, genera coded as state\n2 may have different arrangements for ingesting\nhistiotrophe.\nAreolae are another specialization missing from\nthe stem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria (see Fig. 1C).\nThey are, however, found in the stem of Ferungu-\nlata as well as convergently in moles and Galago.\nWithin Afrotheria they occur as transformations\nin Tenrec and Orycteropus.\nDifferentiation of endometrial stroma\nDecidual cells are found in the pregnant or\npseudopregnant uterus of some mammals. They\nare differentiated from the stroma by hypertrophy\nand functional modiﬁcation of connective tissue\ncells (Mossman, ’87). Typical decidual cells occur\nin many eutherians including anthropoid primates\n(character 9, state 2, e.g. Homo). In other cases,\nincluding strepsirhine primates, decidualization\ndoes not take place (state 1, e.g. Galago). Taxa\nreported to have atypical decidual cells, including\nelephants and carnivores (Mossman,\nbeen scored as state 1.\n’87), have\nDecidual cells occur in the stem species pattern\nof Eutheria, Afrotheria, Euarchontoglires and\nLaurasiatheria, signifying the plesiomorphic char-\nacter condition. Transformations toward the apo-\nmorphic condition, considered currently as a loss\nof decidual cells during pregnancy, occur in\nFerungulata, Scalopus and Galago. Moreover,\nwithin Afrotheria, decidual cells are judged to be\nlost in Echinops and Orycteropus; and conver-\ngently they are only atypically developed in\nLoxodonta (cf. Mossman, ’87).\nCharacters associated with implantation\nand the fetal membranes\nUnlike the chorioallantoic placenta, these are\ncharacters that can be coded both for marsupials\nand for further outgroups (not treated here). They\nare tabulated separately, together with the type of\nthe newborn (Table 3,\nuterus and state of\nFigs. 3–5).\nImplantation: ﬁrst attachment\nMossman (’87) remarked that the site of ﬁrst\nattachment (character 10) was constant within\norders. Central attachment of a relatively ex-\npanded blastocyst is sometimes found (state 1,\ne.g. Canis). Otherwise, attachment is on the\nmesometrial side of the uterus (state 2, e.g. Sus),\nor more commonly on the antimesometrial side\n(state 3, e.g. Sorex). Less frequently, ﬁrst attach-\nment is to either or both of the lateral walls (state\n4, e.g. Eremitalpa). In shrews the trophoblastic\nannulus ﬁrst attaches to the lateral walls of a\npocket on the antimesometrial side of the uterus.\nWe chose to score this as state 3.\nAccording to our analysis, antimesometrial\nattachment of the blastocyst during implantation\nremains as the plesiomorphic condition, achieved\nin the stem species pattern of Eutheria, Afrother-\nia, Euarchontoglires and Laurasiatheria (Fig. 3A).\nTransformations occur in each of the major clades,\ne.g., the occurrence of mesometrial attachment in\nFerungulata plus Chiroptera.\nImplantation: depth of nidation\nSome variation occurs in the depth of implanta-\ntion (character 11; Mossman,\n’37). Superﬁcial\nattachment of the blastocyst to the endometrium\n(state 1, e.g. Echinops) is far more common than\n(primary) interstitial attachment, where the blas-\ntocyst penetrates deep into the endometrium\nCharacter 10: Implantation (ﬁrst attachment) – State 1 5 central; state 2 5 mesometrial; state 3 5 antimesometrial; state 4 5 lateral.\nCharacter 11: Implantation (depth of nidation) – State 1 5 superﬁcial; state 2 5 secondarily interstitial; state 3 5 primarily interstitial.\nCharacter 12: Amniogenesis – State 1 5 cavitation; state 2 5 folding.\nCharacter 13: Bilaminar omphalopleure – State 1 5 permanent; state 2 5 temporary; state 3 5 only in blastocyst.\nCharacter 14: Trilaminar omphalopleure – State 1 5 permanent; state 2 5 temporary; state 3 5 absent.\nCharacter 15: Deﬁnitive yolk sac – State 1 5 free or absent; state 2 5 trilaminar; state 3 5 inverted.\nCharacter 16: Allantoic vesicle – State 1 5 permanent; state 2 5 temporary; state 3 5 rudimentary or absent.\nCharacter 17: Allantoic vesicle size – State 1 5 large to medium; state 2 5 small; state 3 5 rudimentary or absent.\nCharacter 18: Allantoic vesicle chambers – State 1 5 one; state 2 5 four; state 3 5 two; state 4 5 absent.\nCharacter 19: Urogenital opening – State 1 5 ‘‘cloaca’’; state 2 5 separate.\nCharacter 20: Type of uterus – State 1 5 unfused horns; state 2 5 duplex; state 3 5 bicornuate; state 4 5 simplex.\nCharacter 21: Newborn – State 1 5 eyes open at birth (precocial); state 2 5 eyes closed at birth (altricial).\n(state 3, e.g. Cavia). Sometimes the blastocyst\nattaches superﬁcially in a pocket that subse-\nquently closes over it; this is known as secondary\ninterstitial implantation (state 2, e.g. Erinaceus).\nSuperﬁcial attachment is the plesiomorphic con-\ndition and part of the stem species pattern of\nEutheria. Within Eutheria, this character is highly\nconserved, occurring in the stem species pattern\nof all higher systematic clades. Within our data\nset,\ninterstitial\nimplantation occur only in Cavia and Homo,\nwhilst secondary interstitial implantation is seen\nin Erinaceus, Suncus, Mus and Elephantulus.\nthese apomorphic\nAccording to our analysis,\nconditions are convergently established in most of\nthe taxa included. Even for Rodentia, where the\npattern is not resolved, we know about several\ntaxa with superﬁcial implantation, indicating an\nindependent achievement of more invasive blas-\ntocyst attachment (e.g. Luckett,\n’85; Mossman,\n’87; Mess, 2003).\nAmniogenesis\nMuch signiﬁcance has been attached to the way\nin which the amnion is formed (character 12;\nMossman, ’87). In marsupials and most orders of\neutherian mammals, the amnion develops as folds\nof the extraembryonic somatopleure at the cepha-\nlic end and sides of the early embryo. The folds\nmeet and fuse dorsal to the embryo (state 2; e.g.\nSus). Amnion formation by folding also occurs in\nreptiles, birds and monotremes. Several mammals\nhave no amniotic folds or only rudimentary ones.\nIn these the amnion forms by cavitation,\ni.e.,\nintercellular spaces arise in the inner cell mass\nand coalesce to form the amniotic cavity (state 1,\ne.g. Cavia). According to Moghe (’56), in Cynop-\nterus a primary amniotic cavity is formed by\ncavitation, but later folds arise and form another\ncavity which fuses with the ﬁrst. Therefore, we\nhave chosen not to code Cynopterus for this\ncharacter. Similar considerations apply to Mus\n(Luckett, ’85).\nThe stem species pattern for Eutheria, Euarch-\nontoglires and Laurasiatheria is amniogenesis by\nfolding as the plesiomorphic character condition of\nthe group (Fig. 3B). Transformations to the\napomorphic condition, i.e., cavitation, occur con-\nvergently in Erinaceus, Cavia, Cynocephalus and\nHomo.\nindependently from the\naforementioned taxa, the stem species pattern\nfor Afrotheria likewise is amniogenesis by cavita-\ntion. This character state occurs in the armadillos\n(Enders, 2002) and probably in Bradypus (Moss-\nman, ’87), indicating a character transformation\non the stem lineage of Afrotheria plus Xenarthra.\nBilaminar omphalopleure\nThe arrangement of the yolk sac wall varies\nmarkedly, and different regions—characterized by\nthe germ layers involved—exist during pregnancy.\nBecause of their distinct structural design, we\nhave decided to treat the presence and absence of\nthe most important regions as discrete characters.\nFirst of all, in its earliest developmental stage, the\nyolk sac is composed of trophoblastic ectoderm\nlined by extraembryonic endoderm: a bilaminar\nomphalopleure (character 13; Steven, ’75; Perry,\n’81; Mossman, ’87). This condition may occur only\nin the blastocyst (state 3, e.g. Homo). More\ncommonly it is retained into later development,\nbut as a temporary structure (state 2, e.g. Tenrec).\nOccasionally, however, a portion of juxtauterine\nbilaminar omphalopleure is maintained until term\n(state 1, e.g. Erinaceus). It should be noted that in\nrodents with inversion of the yolk sac (see below),\nthe periplacental part of the bilaminar omphalo-\npleure with its Reichert’s membrane persists as\nthe covering of the disk of the chorioallantoic\nplacenta. However, a juxtauterine bilaminar om-\nphalopleure occurs only in the blastocyst; conse-\nquently, these species (Mus, Cavia) have been\ncoded as state 3.\nA temporary bilaminar omphalopleure is pre-\nsent in the stem species pattern of Eutheria,\nAfrotheria, Euarchontoglires and Laurasiatheria,\nbut apparently does not occur in Xenarthra as\nindicated by the single genus Bradypus. Several\ntransformations to a permanent bilaminar om-\nphalopleure occur, most notably in all representa-\ntives of Lipotyphla. In addition, others such as\nHomo are characterized by another apomorphic\ncondition,\ni.e., the restriction of the bilaminar\nomphalopleure to the blastocyst stage only.\nTrilaminar omphalopleure\nA trilaminar omphalopleure is formed when\nextraembryonic mesoderm is interposed between\nthe trophoblast and endoderm, bringing with it\nfetal blood vessels (character 14; Perry, ’81). If it is\napplied to the uterine wall, allowing maternal–\nit constitutes a choriovitelline\nfetal exchange,\n’87, see character 15). In\nplacenta (Mossman,\nmost orders of Eutheria, the trilaminar omphalo-\npleure is a temporary structure (state 2, e.g.\nEchinops). Very few eutherian mammals, but all\nmarsupials, retain the trilaminar omphalopleure\nto term (state 1, e.g. Tupaia). In some mammals a\ntrilaminar omphalopleure is never formed (state 3,\ne.g. Homo, Cavia).\nSimilar to the previous character, a temporary\ntrilaminar omphalopleure is present in the stem\nspecies pattern of Eutheria, Afrotheria, Euarch-\nontoglires and Laurasiatheria (Fig. 3C). Character\ntransformation towards complete absence of a\ntrilaminar omphalopleure or choriovitelline pla-\ncenta took place convergently in Bradypus, Homo\nand the two representatives of Rodentia.\nIn\ncontrast, Tupaia and Eremitalpa are the only\nspecies in the set with a permanent trilaminar\nomphalopleure, representing an independently\nachieved derived character condition. Moreover,\nthe marsupial Didelphis likewise possesses a\npermanent trilaminar omphalopeure.\nDeﬁnitive yolk sac\nFinally, the association of the yolk sac to the\nmaternal uterine epithelium will be considered.\nTo function as a placenta, the trilaminar yolk sac\nmust maintain contact with the uterine wall, in\norder to take up histiotrophe (Mossman, ’87; Mess\net al., 2003). All marsupials rely on yolk sac\nplacentation. In Eutheria, such a yolk sac placenta\nis only rarely present during the later stages of\npregnancy when fetomaternal exchange has to be\nmost efﬁcient. In these cases, the trilaminar\nomphalopleure is retained until term in close\nassociation with the maternal epithelium (char-\nacter 15, state 2, e.g. Tupaia). However, this may\nnot be the case at term, either because the yolk sac\nis not formed or because it has been displaced and\nnow lies free in the exocelom (state 1, e.g.\nBradypus). More often there is partial or complete\ninversion of the germ layers (state 3, e.g. Cavia);\npartial\ninversion is seen in insectivores and\ncomplete inversion occurs in many rodents. This\ntype could serve as a so-called inverted yolk sac\nplacenta, characterized by an inverse organization\nof layers that are apposed to the maternal uterine\nepithelium. However, the situation is so different\nfrom that of state 2 that we have decided to treat it\nas a separate character state.\nAt term, in nearly all orders the yolk sac either\nhas not formed, or it has formed but subsequently\nregressed or lost contact with the uterine wall.\nConsequently, the absence of a choriovitelline\nplacenta is a plesiomorphic character condition\noccurring in the stem species pattern of Eutheria,\nAfrotheria, Euarchontoglires and Laurasiatheria.\nIt is also present in Bradypus. Partial or complete\ninversion of the yolk sac occurs as an apomorphic\ncondition in Lipotyphla, and is independently\nachieved by Glires and the ﬂying lemur, Cynoce-\nphalus. A true yolk sac placenta at term is\nretained only in Tupaia and Eremitalpa, and\nconsequently must be regarded as a derived\ncharacter condition within Eutheria.\nAllantoic vesicle\nIn most mammals the allantois encloses a sac,\nthe allantoic vesicle (character 16), which is\nretained throughout pregnancy (state 1, e.g.\nEquus). In some, the vesicle is a temporary\nstructure (state 2, e.g. Tadarida). In others there\nis an allantoic stalk bearing blood vessels to the\nchorioallantoic placenta, but the vesicle is rudi-\nmentary or absent (state 3, e.g. Sorex).\nA permanent allantoic vesicle is a feature of the\nstem species pattern of Eutheria, Afrotheria,\nEuarchontoglires and Laurasiatheria. Some trans-\nformations occur, e.g., the allantoic vesicle is\nrudimentary or absent in shrews, rodents and\nHomo as well as in Bradypus.\nAllantoic vesicle size\nThere are considerable variations in the size of\nthe allantoic vesicle (character 17), that have been\nsigniﬁcance (Portmann,\nascribed phylogenetic\n’38). We have combined medium and large sizes\n(state 1, e.g. Equus) as the descriptions available\ndo not enable us to make the distinction given for\nselected species by Portmann (’38). Some euther-\nians have only a small allantoic vesicle (state 2, e.g.\nOryctolagus) and in others it is rudimentary or\nabsent (state 3, e.g. Homo).\nA medium or large allantoic vesicle is another\nfeature of the stem species pattern of Eutheria,\nAfrotheria, Euarchontoglires and Laurasiatheria.\nTransformation to a small vesicle occurs in\nTadarida, Oryctolagus and Cynocephalus and to\na rudimentary or absent vesicle in shrews,\nrodents, Homo and Bradypus.\nAllantoic vesicle chambers\nThe allantoic vesicle is usually a single sac\n(character 18, state 1, e.g. Felis) or is bilobed\n(state 3, e.g. Tursiops). In some instances, how-\never, it forms four lobes (state 2, e.g. Orycteropus);\nthis is always associated with early division of the\numbilical cord giving rise to four sets of vessels\nthat subtend the septal folds of the allantois. As\nalready noted, the vesicle is absent in some species\n(state 4, e.g. Mus).\nRepresenting the plesiomorphic condition of\nEutheria and Metatheria,\nin the stem species\npattern of Eutheria, Euarchontoglires and Laur-\nasiatheria, as well as in Bradypus, the allantoic\nvesicle is a single chamber (Fig. 4A). Transforma-\ntion to a bilobed state occurs within Cetartiodac-\ntyla, i.e., in Tursiops and Hippopotamus. Of note,\nthe stem species pattern of Afrotheria is a four-\nlobed allantoic sac, this being a synapomorphic\ntrait of Afrotheria. A separate, i.e., convergent\ntransformation to a four-lobed conformation likely\noccurs in Otolemur, as Hill (’32) considered this to\nbe typical of the strepsirhine allantois.\nCharacters associated with the female\nreproductive tract and newborn\nUrogenital opening\nIn reptiles, birds and monotremes, the anal and\nurogenital tracts open into a common recess, the\ncloaca. In marsupials and some eutherians, they\nopen into a common skin fold that Starck (’82)\nregards as a secondary cloaca (character 19, state\n1). This condition occurs in Xenarthra (e.g.\ntenrecs, otter\nBradypus), many insectivores,\nshrews (P. Vogel, personal communication) and\nin the lagomorph, Ochotona. Shoshani and\nMcKenna (’98, char. 215) assigned this state to\nmarsupials but not, for example, to lagomorphs, a\ndistinction that does not seem justiﬁed based on\ntheir description of Macropus. In most eutherians,\nthe anal and urogenital tracts have separate\nperineal openings (state 2, e.g. Homo).\nWe were unable to resolve the stem species\npattern for Eutheria, particularly because of\ndifferent conditions in basal branches and in the\nmarsupial (Fig. 4B). However, separate openings\nare the norm and this state is present in the stem\nspecies pattern of Laurasiatheria, Euarchonto-\nglires and Afrotheria.\nType of uterus\nThere are several possible arrangements of the\nuterine lumen (character 20; Mossman,\n’87;\nShoshani and McKenna, ’98; character 223). In\nmarsupials the uterine tubes are not fused, each\ncommunicating with the urogenital sinus through\na separate vagina (state 1). This represents a true\nmetatherian–eutherian dichotomy (Harder et al.,\n’93). In a duplex uterus, the uterine tubes open\nindependently into the vagina (state 2, e.g.\nOrycteropus). In a bicornuate uterus, the uterine\nhorns join internally to form the body (corpus)\nwhich communicates with the vagina through\na common cervical canal\n(state 3, e.g. Sus).\nA simplex uterus with a single unpaired corpus\nbut only rudimentary horns (state 4, e.g. Homo)\nis rare among mammals (Martin, 2003). A more\ndetailed analysis of uterine types is given by\nMossman (’87; Table 9.1).\nA bicornuate uterus is present in the stem\nspecies pattern of Eutheria and in three of the\nmajor clades (Afrotheria, Euarchontoglires and\nLaurasiatheria). The tree contains only one\nrepresentative of Xenarthra (Bradypus), but a\nin armadillos and\nsimplex uterus is present\nanteaters as well as sloths (Wislocki, ’28). Thus,\ndata suggest that Xenarthra seems to be the only\nsuperordinal taxon that is characterized by an\napomorphic condition, i.e., the uterus simplex.\nNewborn\nThe developmental condition of the newborn\n(character 21) likewise varies in Eutheria. In\naltricial species, the newborn young are relatively\nimmobile, have closed eyes, lack hair and require\nparental care for a comparatively long time. In\nthe newborn are relatively\nprecocial species,\nmature and mobile from the moment of birth.\nWe follow Shoshani and McKenna (’98, char. 249)\nin coding for eyes open at birth (state 1, e.g.\nCavia), which usually implies precociality, or eyes\nclosed at birth (state 2, e.g. Mus), which usually\ncharacterizes altricial young. Clearly this is a\nsimpliﬁcation. For example,\nthe state of eye\nopening in chiropterans is not the most reliable\nindicator of developmental state (Kunz and Hood,\n2000). In addition, the human newborn is some-\ntimes characterized as secondarily altricial (Mar-\ntin, 2003), but is born with open eyes and is coded\nas state 1. Naaktgeboren and Slijper (’70) was a\nuseful source for this character. Dr. Peter Vogel\n(personal communication) provided information\non Potamogale and Dr. Thomas Kunz (personal\ncommunication) on Tadarida.\nThis character has an interesting distribution.\nRepresenting a plesiomorphic character condition\nfor the Eutheria, the stem species pattern of\nEutheria is characterized by altricial newborns, as\nfor Euarchontoglires and Laurasiatheria (Fig. 4C).\nWithin the latter clade, however, precocial new-\nborns are part of the stem pattern of Ferungulata\nand the same transformation occurs in Cavia (and\nother hystricognath rodents; Mess, 2003) and in\nprimates. Precocial young are part of the stem\nspecies pattern of Xenarthra as indicated by their\nsingle representative Bradypus as well as of the\nstem species pattern of Afrotheria, but with\nsecondary transformation to altricial newborns\nin Tenrecoidea.\nDISCUSSION\nA large amount of information is available on\nthe character complex of placentation, fetal mem-\nbranes and the female reproductive tract. As a\nresult, we were able to identify 21 characters\nsuitable for analysis and trace their evolution on\nthe basis of a pre-existing hypothesis of taxon\nrelationships (Douady et al., 2002b). For most of\nthese characters it was possible to determine the\nstem species pattern of living Eutheria and of the\nmain superordinal taxa, including Afrotheria. Of\nparticular note, we found one character that had\nbeen evolved on the stem lineage of Afrotheria,\nand two others that are shared with their sister\ngroup Xenarthra. Thus, contrary to previous\nassertions (e.g. Yang et al., 2003), it should be\npossible to identify morphological characters that\nare synapomorphic for this novel taxon.\nStem species pattern of Eutheria\nFirst of all, based on distributional data, the\nanalysis indicates that the stem species pattern of\nliving Eutheria (Placentalia) is as follows. The\ndeﬁnitive chorioallantoic placenta is discoid in\nshape and labyrinthine in structure. It is supplied\nby two umbilical arteries and a single umbilical\nvein. The interhemal barrier separating maternal\nand fetal blood is of the endotheliochorial type and\ncontains syncytiotrophoblast but not cytotropho-\nblast, at least in the deﬁnitive or late placenta. The\nendometrium possesses decidual cells. Special\nstructures such as hemophagous organs, areolae\nand trophoblastic fenestrae are not present in the\nstem species pattern of Eutheria. Implantation is\nantimesometrial and amniogenesis occurs by fold-\ning. The bilaminar and trilaminar omphalopleure\nare temporary structures. At term, the yolk sac is\neither absent or has lost all contact with the\nuterine wall and chorion, so that there is no\nfunctional choriovitelline placenta. In contrast,\nthe allantoic vesicle is permanent, medium to\nlarge in size and comprises a single chamber. The\npregnancy is established in a bicornuate uterus\nand the newborn are altricial.\nOf particular note, the current analysis appears\nto resolve a long-standing controversy about the\nancient condition of the interhemal barrier of the\nplacenta. Ever since the classiﬁcation made by\nGrosser (’09), authorities on placentation have\ndisputed whether a non-invasive (epitheliochorial)\nor an invasive (hemochorial) type was primitive\nfor eutherians (inter alia Hubrecht, ’09; Wislocki,\n’29; Starck, ’59; Mossman, ’87; Luckett, ’93). Only\nMossman (’37) took into account the possibility\nthat an endotheliochorial condition could be\nprimitive for this group. But, after observing\nepitheliochorial placentation in Scalopus (Prasad\net al., ’79), an insectivore, Mossman changed his\nmind and argued for the epitheliochorial condition\nas the primitive state (Mossman, ’87). However,\naccording to our results, the stem species pattern\nis indeed characterized by the endotheliochorial\nplacental type (Fig. 1A). This result is in accor-\ndance with other recent evolutionary interpreta-\ntions (Martin, 2003; Carter and Enders, 2004;\nVogel, 2005).\nThe interhemal membrane is of interest in the\ncontext of maternal–fetal exchange of respiratory\ngasses and nutrients, sometimes referred to as\nhemotrophic nutrition (Mess et al., 2003). Of equal\ninterest is the ingestion by trophoblast of gland-\nular secretions, cell debris and erythrocytes,\nknown as histiotrophic nutrition (for a recent\ndiscussion see Vogel, 2005). Neither of the specia-\nlizations considered here, hemophagous regions\nand areolae, appear in the stem species pattern of\nEutheria (see Fig. 1C). Other specializations\nmentioned by Mossman (’87, his Table 14.4)\ninclude absorptive plaques, arcades and marginal\nfolds. We attempted to incorporate these into our\nanalysis but did not obtain resolution (data not\nshown).\nThe\npattern\nfor the chorioallantoic placenta,\nincluding the\nendotheliochorial barrier, occurs in Bradypus,\nreﬂecting its basal position in the tree of Douady\net al. (2002b). Thus, the reconstructed condition\nof chorioallantic placentation is present in an\nextant species.\nreconstructed\nAs an additional feature, we were able to gain\nmore precise insight into the evolution of yolk sac\ncharacters (cf. extensive discussion in Mossman,\n’87). Formerly, the presence in several species of a\ntrilaminar omphalopleure or a true yolk sac\nplacenta was regarded as a plesiomorphic condi-\ntion of Eutheria, retained from the common\nancestor of Theria, Mammalia or even Amniota\n(i.e. Luckett, ’77, ’85). Up to a point our analysis\nsupports this interpretation. However,\nit also\nshows that both the bilaminar and trilaminar\nomphalopleure exist only as temporary structures\nin the stem species pattern of Eutheria and their\nmajor clades (Fig. 3C, cf. Fig. 5). Consequently, the\nabsence of a true yolk sac placenta at term is\nlikewise a plesiomorphic condition of Eutheria.\nThus yolk sac placentation is more different\nbetween Eutheria and Metatheria than indicated\nby common text books. Interestingly, a phyloge-\nnetic analysis of marsupial placentation (Freyer\net al., 2003) led to the conclusion that a chor-\nioallantoic placenta was part of the stem species\npattern of marsupials but that the yolk sac had\nevolved to become the principal organ of physio-\nlogical exchange.\nAs a consequence of our analysis, Tupaia with\nits permanent yolk sac placenta or members of\nLipotyphla with a permanent bilaminar omphalo-\npleure can no longer serve as models that have\nretained ancient conditions. The eutherian stem\nspecies pattern with regard to yolk sac organiza-\ntion can be recognized in quite different taxa, i.e.,\nin lower primates such as Galago or members\nof Tenrecoidea.\nWe separately examined the consequences of re-\nrooting the current tree between Afrotheria and\nall other placentals (Murphy et al., 2001a,b;\nAmrine-Madsen et al., 2003). Brieﬂy, the stem\nspecies pattern for Eutheria remained much as\ndescribed above (data not shown), i.e., 17 of 21\ncharacters were not at all\ninﬂuenced by the\ndifferent position of the basal node within Eu-\nthe results contradicted our\ntheria. None of\nfour characters were now\nanalysis. However,\nunresolved: gross form of\nthe placenta, ﬁrst\nattachment of the blastocyst, amniogenesis and\nnewborn.\nIn the following, we deal with character trans-\nformations—compared with the stem species\npattern of Eutheria—that occur on the stem\nlineages of the higher order clades of Eutheria\nand discuss important transformations that took\nplace within these groups.\nStem species pattern of Afrotheria\nAfrotheria presents some interesting transfor-\nmations compared with the stem species pattern of\nEutheria. Two of these—amniogenesis by cavita-\ntion (Fig. 3A) and precocial young (Fig. 4C)—are\nshared with Bradypus, which is representative of\nthe sister group Xenarthra. There are few other\ninstances in the tree of amnion formation by\ncavitation. Often, as in Erinaceus, Cavia and\nHomo, this character state is associated with\ninterstitial implantation (Luckett, ’76). However,\nthis is not the case in Afrotheria or Bradypus.\nPrecociality requires a longer gestation and\nusually occurs in relatively large animals such as\nCetartiodactyla, Hystricognathi (Mess, 2003) and\nhigher primates (Martin, 2003), where it\nis\napomorphic.\nThe presence of a large, four-chambered allan-\ntoic sac in representatives of every order of\nAfrotheria, but not in Bradypus, represents a\nhitherto unrecognized synapomorphy for\nthe\nsuperorder (Fig. 4A). A similar arrangement is\n’32), but\nknown only for Strepsirhini\nconvergently\naccording to our analysis was\nachieved.\n(Hill,\nWe did not obtain resolution in Afrotheria\nfor the gross form of the placenta, which is\ndiscoidal in Tenrecoidea and Elephantulus, but\nzonary in the rock hyrax–elephant–sea cow\nclade (Paenungulata) and Orycteropus. Hemopha-\ngous regions are a prominent feature of the\nplacenta in several genera, but apparently the\nplesiomorphic state, i.e., absence of the organ, is\ncharacteristic for the stem species pattern of\nAfrotheria (cf. Fig. 2).\nEvolutionary transformations\nwithin Afrotheria\nThere is a transformation from a separate\nurogenital opening to a ‘‘cloaca’’ in Tenrecoidea\n(Fig. 4B). Our analysis fails to support\nthe\nassumption that this is a primitive rather than a\nderived character. As extensively discussed by\nShoshani and McKenna (’98; their Appendix 1),\ninclusion of the anal and urogenital openings in a\ncommon skin fold, as occurs in marsupials and\nsome eutherians, may not be homologous with the\ncloaca found in reptiles, birds and monotremes.\nIn addition to the establishment of a cloaca, there\nare two further character transformations on the\nstem lineage of Tenrecoidea: amniogenesis by\nfolding (Fig. 3B) and the altricial state of the\nnewborn (Fig. 4C).\nAbove, it was noted that a large, four-chambered\nallantoic sac is a synapomorphy for Afrotheria\n(Fig. 4A). This is secondarily transformed to a\nsingle sac in Echinops, although the associated\nearly division of cord vessels is retained in this\nspecies (Carter et al., 2004). Even within Tenre-\ncoidea, a four-chambered sac is found in Micro-\npotamogale and the related genus Potamogale\n(Carter et al., 2005b).\nRetention of a trilaminar omphalopleure to term\nhas been recorded in Eremitalpa (Gabie,\n’60),\nwhich represents a derived rather than a plesio-\nmorphic pattern within Eutheria (see above).\nOn the stem lineage of Tenrecoidea plus their\nsister group Macroscelidea, here represented by\nElephantulus, a transformation from the endothe-\nliochorial to the hemochorial type of placental\nbarrier took place (Fig. 1A). Such a transformation\noccurs twice in Afrotheria, the second example\nbeing Procavia. Until recently it was thought that\nthe interhemal membrane of Trichechus was\n’35), but this has been\nhemochorial (Wislocki,\nrefuted by studies at the ultrastructural\nlevel\n(Carter, Enders, Miglino and Da Silva, unpub-\nlished observations). Within Tenrecoidea and\nTenrecidae, a reversal of the interhemal mem-\nbrane structure to the endotheliochorial condition\nis recognized for Micropotamogale (Carter et al.,\n2005b).\nThe stem lineage of Orycteropus, Elephantulus\nand the Tenrecoidea does not show any character\ntransformations. Orycteropus has a duplex uterus.\nThis is clearly depicted by van der Horst (’49) and\nmust be regarded as a transformation from the\nbicornuate state, which represents the condition\nin the stem species pattern of Afrotheria. More-\nover, Orycteropus possesses areolae as an addi-\ntional placental structure (Fig. 1C) and decidual\ncells are absent. Both of\nthese are derived\ncharacter conditions within Afrotheria that are\nconvergently developed in Tenrec as well as\nLoxodonta.\ntrophoblast\nFinally, the stem species pattern of Paenungu-\nlata (i.e. hyrax, sea cow and elephant) is char-\nin the\nacterized by the type of\ninterhemal barrier of\nthe deﬁnitive placenta,\nwhich shows a transformation from syncytiotro-\nphoblast to cytotrophoblast (Fig. 1B). In addition,\nPaenungulata is characterized by a second derived\ncondition, in that implantation is central rather\nthan antimesometrial (Fig. 3A). The ﬁrst attach-\nment of the blastocyst evidences several other\ntransformations within Afrotheria. Tenrec and\nEremitalpa are among the few genera with\nimplantation on a lateral wall of the uterus.\nFinally, implantation in Elephantulus is within\nan implantation chamber on the mesometrial side\nof the uterus, associated with secondarily inter-\nstitial implantation as another derived character\ncondition of this genus.\nto Bradypus,\nThe tree contains only one representative of\nXenarthra and this is not\nthe place for an\nextensive analysis of the clade. It may be observed\nthe placenta\nthat,\nof armadillos is both villous and hemochorial\n(Enders, ’60). Fenestration of the trophoblast in\nBradypus is comparable to that in shrews (King\net al., ’82), but is not known for other Xenarthra.\nThe ‘‘cloaca’’\n’98),\nsimplex uterus (Wislocki, ’28), rudimentary allan-\n’87) and precocial young\ntoic sac (Mossman,\n’70) are characters\n(Naaktgeboren and Slijper,\nshared by armadillos, anteaters and sloths.\n(Shoshani and McKenna,\nEuarchontoglires\nNot a single character transformation is indi-\ncated on the stem lineage of Euarchontoglires.\nWithin that group, Tupaia retains the bicornuate\nuterus and endotheliochorial placenta (Fig. 1A)\nthat are part of the stem pattern of Eutheria yet\nlost in other representatives of Euarchontoglires.\nThere is lateral implantation in Tupaia (Fig. 3A),\nresulting in a double discoid placenta. A trilami-\nnar omphalopleure or yolk sac placenta is retained\nuntil\n’68).\nlate gestation in Tupaia (Luckett,\nAlthough this has been regarded as retention of\na primitive character (Luckett, ’68, ’77), it is clear\nfrom our analysis (Fig. 3C) that it is derived\nrelative to the common ancestor of Eutheria.\nThe common stem for Primates, Cynocephalus\nand Glires shows a transformation to hemochorial\nplacentation. In Galago, however, the placenta is\ndiffuse and epitheliochorial with cytotrophoblast\nin the interhemal membrane, all representing\nderived conditions of that group. Moreover, as\nfurther apomorphic conditions of Galago there are\nareolae and there are no decidual cells (cf. Figs. 1C\nand 2). This pattern is convergent with that in\nFerungulata. Hill (’32) regards a four-lobed allan-\ntoic sac, described for Loris and Nycticebus, as\ntypical for lorises, resulting as a derived pattern\nthat was independently established compared to\nAfrotheria (Fig. 4A).\nCynocephalus shares the following character\ntransformations with the three representatives of\nGlires: duplex uterus, inclusion of both types of\ntrophoblast in the interhemal membrane, and an\ninverted yolk sac. It must be noted, however, that\nthe position of the ﬂying lemurs is weakly supported\nin molecular analyses, resulting in very different\ntopologies. Short internodes and long branch lengths\ncontribute to this unstable topology (Douady, 2001).\n(a)\nWe are aware of ﬁve alterative topologies:\nDermoptera with Scandentia, as sister to primates\n(Murphy et al., 2001b, see Springer et al., 2005a);\n(b) Dermoptera as a sister group to Anthropoidea,\nthus ruling out primate monophyly (Murphy et al.,\n2001a; Arnason et al., 2002; but see Schmitz et al.,\n2002; Schmitz and Zischler, 2003); (c) Dermoptera\nas the sister group to Primates (Douady, 2001);\n(d) Dermoptera as sister to Glires (Douady et al.,\n2002b) and (e) Dermoptera as sister to Glires plus\nScandentia (Amrine-Madsen et al., 2003). In view of\nthis uncertainty,\nthe character transformations\nshared by Cynocephalus and Glires are of limited\nimportance.\nTransformations within Glires include reversion\nto syncytiotrophoblast in Cavia and other hystri-\ncognaths, which also have precocial newborns\n(Mess, 2003). In addition Cavia, as a typical\nhystricognath rodent, has primary interstitial\nimplantation; there is no true yolk sac placenta-\ntion and the allantoic sac is rudimentary. Thus,\nhystricognath rodents are especially characterized\nby derived conditions of\nfetal membrane and\nplacentation characters (see Luckett, ’85; Mess,\n2003). The aforementioned character states of\nCavia, and hystricognath rodents in general, often\noccur together (Luckett, ’76) and are convergently\nshared by Homo. Primates have a trabecular to\nvillous placenta and precocial young.\nLaurasiatheria, Ferungulata\nand Lipotyphla\nSimilar to Euarchontoglires, and in clear con-\ntrast to Afrotheria and/or Xenarthra, no character\ntransformations can be recognized on the stem\nlineage of Laurasiatheria in comparison with the\nEutherian stem species pattern. Transformations\nwithin this clade mostly affect Ferungulata (see\nbelow) and Lipotyphla.\nWithin Laurasiatheria, analyses of molecular\ndata provide robust support for a clade comprising\nCarnivora, Pholidota, Perissodactyla and Cetar-\ntiodactyla (Waddell et al., 2001). Archibald (2003)\nrecommends retaining the appellation Ferungula-\nta for this clade, notwithstanding that it differs\nfrom that originally proposed by Simpson (’45).\nA Ferungulata clade is consistent with our\ncharacter optimization. The stem pattern of\nFerungulata exhibits the following transforma-\ntions from that of Laurasiatheria or Eutheria: the\nplacenta is diffuse, villous and epitheliochorial\n(Fig. 1A); areolae are present (Fig. 1C); decidual\ncells are absent; and the newborn are precocial\n(Fig. 4C). Thus our analysis adds further support\nto the view (Vogel, 2005) that this character\ncomplex is not primitive but derived relative to\nEutheria. Ferungulata shares two further trans-\nformations with the bats, namely cytotrophoblast\nin the interhemal membrane of the deﬁnitive\nplacenta (Fig. 1B) and mesometrial attachment of\nthe blastocyst (Fig. 3A). There are reversals within\nFerungulata. In particular, carnivores have a\nzonary, labyrinthine, endotheliochorial placenta;\nantimesometrial attachment; and altricial young\nin their stem species pattern.\nA second point of interest within Laurasiatheria\nis the status of Lipotyphla following removal of the\ntenrecs and golden moles. Many transformations\nare indicated within Lipotyphla, but only two of\nthem are common to all genera included in the\ndata set. These are the presence of a permanent,\njuxtauterine omphalopleure together with a per-\nmanent,\ninverted, visceral yolk sac (cf. Fig. 5;\nMossman, ’87; Asher, 2005). Solenodon is not part\nof the set. However, a recent analysis (Roca et al.,\n2004) found Solenodon to be basal to the Lipo-\ntyphla with the topology for the remainder of the\norder being similar to that used here. Therefore,\nin a separate analysis (data not shown), we added\nSolenodon to the tree and reanalyzed those\ncharacters for which data are available (Wislocki,\n’40). This reduced the transformations common\nto Lipotyphla since Solenodon does not have a\npermanent juxtauterine bilaminar omphalopleure.\nMoreover, Solenodon has hemochorial placenta-\ntion, resulting in loss of resolution of the inter-\nhemal membrane not only for Lipotyphla, but for\nEuarchontoglires1Laurasiatheria (Boreoeutheria)\nand Eutheria too.\nThere are derived characters in every family of\nLipotyphla. Erinaceus exhibits interstitial implan-\ntation and has a hemochorial placenta. The shrews\nhave a unique hemophagous organ formed by the\ntrophoblastic annulus and trophoblastic ‘‘curtain’’\n(Brambell and Perry, ’45), the trophoblast of the\ninterhemal membrane is extremely thin and\nfenestrated, and there is a rudimentary allantoic\nvesicle. Moles have both types of trophoblast in the\ninterhemal membrane at term as well as promi-\nnent areolae. In addition, Scalopus has a diffuse\nplacenta that may or may not be epitheliochorial\n(Carter, 2005). As Symonds (2005) recently con-\ncluded, molecular analyses of Lipotyphla agree\nonly on the removal of tenrecs and golden moles,\nand there are competing hypotheses for the\nphylogeny of the residual families. Future ana-\nlyses combining molecular and morphological data\nmight beneﬁt from inclusion of characters from\nthe current set.\nConclusions\nWe traced the evolutionary history of 21 char-\nacters associated with placentation, fetal mem-\nbranes and female reproductive organs within\nliving Eutheria on the basis of a current hypoth-\nesis of eutherian relationships derived from\nDouady et al. (2002b). We recognized one char-\nacter transformation on the stem lineage of\ni.e., the occurrence of a four-lobed\nAfrotheria,\nallantoic sac, and two additional ones that occur\non the common stem lineage of Afrotheria and its\nsister taxon Xenarthra,\ni.e., amniogenesis by\ncavitation and the precocial state of the newborn.\nFinally, the analysis provides a comprehensive\ninterpretation of the ancient condition or stem\nspecies pattern of Eutheria. Apart from the above-\nmentioned character\ntowards\nAfrotheria, the reconstructed pattern of Eutheria\ndoes not change in the basal nodes of the group.\nThus, no character transformation occurred on\nthe stem lineages of Laurasiatheria and Euarch-\nontoglires, and even Afrotheria possesses mostly\nplesiomorphic character conditions of Eutheria.\nTransformations of the chosen set of characters\ncommonly took place within subgroups of the\nsuperordinal taxa of Eutheria such as Ferungulata\nand Tenrecoidea.",
         "",
         "Evolutionary transformations of fetal membrane characters in Eutheria with special reference to Afrotheria.",
         "None"
        ],
        [
         "20",
         "000f70100783b16315ff748f23387076fc91d876",
         "Address: 1Department of Computer Science and Information Engineering, National Taiwan University, Taipei, 106, Taiwan, R.O.C, 2Department\nof Bio-industrial Mechatronics Engineering, National Taiwan University, Taipei, 106, Taiwan, R.O.C, 3Graduate School of Biotechnology and\nBioinformatics, Yuan Ze University, Chung-Li, 320, Taiwan, R.O.C and 4Department of Computer Science and Engineering, Yuan Ze University,\nChung-Li, 320, Taiwan, R.O.C\nAbstract\nBackground: More and more disordered regions have been discovered in protein sequences, and\nmany  of  them  are  found  to  be  functionally  significant.  Previous  studies  reveal  that  disordered\nregions  of  a  protein  can  be  predicted  by  its  primary  structure,  the  amino  acid  sequence.  One\nobservation that has been widely accepted is that ordered regions usually have compositional bias\ntoward hydrophobic amino acids, and disordered regions are toward charged amino acids. Recent\nstudies  further  show  that  employing  evolutionary  information  such  as  position  specific  scoring\nmatrices  (PSSMs)  improves  the  prediction  accuracy  of  protein  disorder.  As  more  and  more\nmachine learning techniques have been introduced to protein disorder detection, extracting more\nuseful features with biological insights attracts more attention.",
         "Chung-Tsai Su,Chien-Yu Chen,Yu-Yen Ou",
         "Chung-Tsai Su,Chien-Yu Chen,Yu-Yen Ou",
         "https://bmcbioinformatics.biomedcentral.com/counter/pdf/10.1186/1471-2105-7-319",
         "Address: 1Department of Computer Science and Information Engineering, National Taiwan University, Taipei, 106, Taiwan, R.O.C, 2Department\nof Bio-industrial Mechatronics Engineering, National Taiwan University, Taipei, 106, Taiwan, R.O.C, 3Graduate School of Biotechnology and\nBioinformatics, Yuan Ze University, Chung-Li, 320, Taiwan, R.O.C and 4Department of Computer Science and Engineering, Yuan Ze University,\nChung-Li, 320, Taiwan, R.O.C\nAbstract\nBackground: More and more disordered regions have been discovered in protein sequences, and\nmany  of  them  are  found  to  be  functionally  significant.  Previous  studies  reveal  that  disordered\nregions  of  a  protein  can  be  predicted  by  its  primary  structure,  the  amino  acid  sequence.  One\nobservation that has been widely accepted is that ordered regions usually have compositional bias\ntoward hydrophobic amino acids, and disordered regions are toward charged amino acids. Recent\nstudies  further  show  that  employing  evolutionary  information  such  as  position  specific  scoring\nmatrices  (PSSMs)  improves  the  prediction  accuracy  of  protein  disorder.  As  more  and  more\nmachine learning techniques have been introduced to protein disorder detection, extracting more\nuseful features with biological insights attracts more attention.",
         "",
         "Results: This paper first studies the effect of a condensed position specific scoring matrix with\nrespect to physicochemical properties (PSSMP) on the prediction accuracy, where the PSSMP is\nderived by merging several amino acid columns of a PSSM belonging to a certain property into a\nsingle column. Next, we decompose each conventional physicochemical property of amino acids\ninto two disjoint groups which have a propensity for order and disorder respectively, and show by\nexperiments  that  some  of  the  new  properties  perform  better  than  their  parent  properties  in\npredicting protein disorder. In order to get an effective and compact feature set on this problem,\nwe propose a hybrid feature selection method that inherits the efficiency of uni-variant analysis and\nthe effectiveness of the stepwise feature selection that explores combinations of multiple features.\nThe  experimental  results  show  that  the  selected  feature  set  improves  the  performance  of  a\nclassifier  built  with  Radial  Basis  Function  Networks  (RBFN)  in  comparison  with  the  feature  set\nconstructed with PSSMs or PSSMPs that adopt simply the conventional physicochemical properties.\nConclusion:  Distinguishing  disordered  regions  from  ordered  regions  in  protein  sequences\nfacilitates the exploration of protein structures and functions. Results based on independent testing\ndata reveal that the proposed predicting model DisPSSMP performs the best among several of the\nexisting  packages  doing  similar  tasks,  without  either  under-predicting  or  over-predicting  the\ndisordered regions. Furthermore, the selected properties are demonstrated to be useful in finding\ndiscriminating patterns for order/disorder classification.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nBackground\nA large proportion of proteins are found to be intrinsically\ndisordered, which refers to segments of protein sequences\nthat  do  not  form  well-defined  three-dimensional  struc-\ntures  in  their  native  states  [1-6].  Many  proteins  contain\nlocal  regions  of  such  disorder,  and  in  some  cases  entire\nproteins  appear  to  exist  as  ensembles  of  structures  [1-\n3,7,8]. Since protein subsequences that lack the ability to\nform a well-defined three-dimensional structure may still\nbe  functionally  related,  identifying  such  disordered\nregions  is  getting  more  and  more  important  for  both\nunderstanding  protein  functions  and  conducting  struc-\ntural analyses [6].\nProtein disorders have been observed in a variety of bio-\nlogical  functions,  including  molecular  recognition,  cell\nsignaling pathways, and enzyme catalysis [3,9]. Taking the\nyeast  proteome  as  an  example,  the  proteins  containing\ndisorder  are  often  located  in  the  cell  nucleus  and  are\ninvolved  in  transcription  regulation  and  cell  signalling\n[4]. The disordered regions that remain as flexible ensem-\nbles under physiological conditions allow the disordered\nbinding  sites  to  interact  with  several  different  targets\n[10,11].  The  disorder-to-order  transition  upon  binding\nprovides  high  specificity  with  modest  binding  affinities\n[7,12].\nIt has been shown in many studies that protein disorders\ncan be predicted from their primary sequences [10,13-18].\nThe prediction methods developed in recent years initiate\nthe possibility of identifying such disordered binding sites\nautomatically [14,17]. A more general concept is that all\nthe necessary information for the correct folding of a pro-\ntein  is  included  in  its  amino  acid  sequence  [19].  Disor-\ndered regions are comprised of a category of amino acids\ndistinct from that of ordered protein structures [20]. For\nexample,  amino  acids  of  aromatic  hydrophobic  groups\nare  known  to  be  good  for  the  general  stabilization  of\norder, and thus are less found in the disordered regions\n[13]. Incorporating information of the biased amino acid\ncomposition in a neural network predicts the locations of\ndisorder with accuracy better than random guesses [13].\nIn 1998, Romero et al. showed that more than 15,000 pro-\nteins in the Swiss-Prot database contain long disordered\nsegments (40 or more residues) based on their predictions\n[14,21].  Studies  on  some  of  these  disordered  regions\nreveal that they are evolutionarily conserved and possess\nbiological functions [10].\nSeveral machine learning approaches, such as neural net-\nlogistic  regression  (LR)\nworks  (NNs)  [6,10,13,22],\n[22,23],  discriminant  analysis  (DA)  [23],  ordinary  least\nsquares (OLS) [22], and support vector machines (SVM)\n[4,23,24]  have  been  introduced  to  protein  disorder  pre-\ndiction.  Since  different  classifiers  deliver  similar  predic-\ntion  accuracy  based  on  the  same  feature  set  [23],\nextracting more useful features with biological insights to\nimprove the quality of prediction attracts more attention\nin recent studies [10,24]. As amino acid composition has\nbeen demonstrated as a useful feature for detecting disor-\ndered regions, Jones et al. showed in their paper that using\nthe  position  specific  scoring  matrices  (PSSMs)  within  a\nspecific length of window centred at a given residue can\nimprove  the  accuracy  of  predicting  its  disorder  attribute\n[10]. The values in a position specific scoring matrix indi-\ncate the level of conservation of a position and the prop-\nerties  of  the  substituted  residues,  which  can  be  derived\ndirectly from executing PSI-BLAST for each target protein\nsequence. PSSMs have been demonstrated to be powerful\nin constructing feature sets for prediction of single-residue\nproperties from an amino acid sequence, such as category\nof  secondary  structures  or  solvent  accessibility  [10].  The\nevolutionary  information  summarized  in  a  PSSM  table\ngeneralizes  the  attribute  of  each  position  in  a  protein\nsequence, and thus improves the sensitivity of the predict-\ning model.\nA  development  of  this  approach  employs  a  condensed\nposition  specific  scoring  matrix  with  respect  to  physico-\nchemical properties (it will be called PSSMP in this paper)\nin predicting protein disorder, where the PSSMP is derived\nby  merging  several  amino  acid  columns  of  a  PSSM  that\nbelong to a certain property into a single column [24]. As\na  PSSM  brings  in  the  evolutionary  information  on  each\nposition, a PSSMP summarizes this information as prop-\nerty  attributes.  The  improvement  achieved  by  PSSMP\ndemonstrates that property attributes are more informa-\ntive than amino acid attributes in distinguishing ordered/\ndisordered  regions.  A  more  comprehensive  study  con-\nducted  in  this  paper  reveals  that  PSSMPs  outperform\nPSSMs  especially  when  the  employed  window  size  is\nlarge.\nWhen employing PSSMP tables as the feature set in pro-\ntein disorder prediction, two questions arise: (1) if all the\namino acids in one physicochemical property group con-\ntribute to the predicting power; and (2) if all the amino\nacids  in  one  physicochemical  property  group  result  in\nconsistent effect on prediction. It has been widely studied\nin previous works that the propensity for order or disorder\nof several amino acids is clear. Hydrophobic amino acids\nare more frequently observed in ordered regions than dis-\nordered  regions  [18,19].  Among  them  the  aromatic\namino  acids  are  present  in  different  locations  to  the\naliphatic amino acids [25]. On the other hand, the amino\nacids  with  charge  imbalance  are  often  present  in  disor-\ndered regions. In this paper, we argue that the propensity\nfor order or disorder of each amino acid should be consid-\nered when constructing PSSMP. After examining the statis-\ntics  derived  by  comparing  the  sequence  segments  in\nBMC Bioinformatics 2006, 7:319\nordered regions and disordered regions, we observe that\nnot all the hydrophobic amino acids possess a propensity\nfor order. Thus we suggest that each conventional physic-\nochemical  property  should  be  divided  into  two  smaller\ngroups  with  propensities  for  order  and  disorder  respec-\ntively,  such  as  hydrophobic  with  an  order  propensity\n(HydrophobicO) and hydrophobic with a disorder propen-\nsity  (HydrophobicD).  The  experiments  conducted  in  this\nwork  reveal  that  some  newly  derived  properties  provide\nmore accurate information regarding order or disorder.\nIncorporating  the  propensity  for  order/disorder  with\nphysicochemical properties in PSSMP produces informa-\ntive features for protein disorder prediction. However, the\nnumber of candidate features becomes larger than in the\ncase  when  only twenty  amino acids  are  considered.  The\nsize of the feature set gets even larger when a large window\nsize is employed, which might cause the performance of\nthe learning algorithms to be degraded due to abundant\nnoise.  Thus,  we  present  in  this  paper  a  feature  selection\nmechanism that considers both the size and effectiveness\nissues when determining a feature set on protein disorder\nprediction.  A  wrapper  approach  of  feature  selection  is\nemployed during training period that invokes the adopted\nRadial Basis Function Networks (RBFN) classifier to eval-\nuate  the  predicting  power  of  a  candidate  feature  set.  A\ncluster-based redundancy removal procedure is incorpo-\nrated to speed up the stepwise feature selection process,\nwhere two levels of redundancy among features are con-\nsidered.\nAs far as the experimental materials are concerned, a new\ndataset  PDB693  organized  from  the  Protein  Data  Bank\n(PDB) [26] database is coined in this work to benefit the\nstudy  on  protein  disorder.  PDB693  and  another  dataset\nD184 collected from Database of Protein Disorder (Dis-\nProt)  [27]  constitute  the  training  data  of  our  classifier\nDisPSSMP.  The  performance  of  DisPSSMP  is  compared\nwith twelve existing disorder prediction packages, where\nthe blind testing data comes from a recent study [6]. The\nexperimental results demonstrate that the selected prop-\nerty  features  are  informative  in  protein  disorder  predic-\ntion and can be used to derive discriminating patterns for\norder and disorder classification.\nResults and discussion\nIn  this  section,  we  first  describe  how  the  datasets  have\nbeen prepared and how the performance of prediction is\nevaluated. Next, we show the results of the feature selec-\ntion  after  conducting  cross-validation  on  the  training\ndata. At this stage, we also discuss the effect of the window\nsize  employed  in  constructing  PSSMP.  After  that,  the\nresultant feature set is employed in constructing the final\npredicting  model  DisPSSMP.  Finally,  the  testing  results\nare evaluated based on the bind testing data, and are com-\npared  with  other  existing  packages  performing  similar\ntasks. At the end of the section, we show the derived prop-\nerty sets can be used to discover patterns that distinguish\nordered and disordered regions.\nDatasets\nIn this study, five  datasets  have been collected  or newly\ncreated for training and validating processes. The detailed\nstatistics  about  each  dataset  are  provided  in  Table  1,\nincluding  the  number  of  chains,  ordered/disordered\nregions, and residues in ordered/disordered regions. The\ntraining data used in constructing the predictor DisPSSMP\nis  composed  of  datasets  PDB693  and  D184,  which  are\nrespectively  organized  from  PDB  and  DisProt  database\nbased on the procedures described in the following para-\ngraphs. Meanwhile, three datasets named R80, U79, and\nP80, which are taken from two related studies [6,19], con-\nstitute  an  independent  testing  data.  This  blind  dataset\nserves as a platform for comparing the performance of the\nproposed method with some other existing packages per-\nforming protein disorder prediction.\nThe  dataset  PDB693  contains  693  partially  disordered\nproteins,  where  the  locations  of  disordered  regions  are\nidentified by looking for the missing residues in a protein\nstructure  from  PDB  database  (28-Aug-2005  version).\nThere  are  originally  32204  structures  in  this  version  of\nPDB database, and those structures are filtered by a clus-\ntering  program  Cd-Hit  [28,29]  such  that  the  resultant\nBMC Bioinformatics 2006, 7:319\nnonredundant set containing no pair of protein sequences\nwith similarity identity of more than 70%. The so-called\nmissing residues are those present in the SEQRES records\nbut  not  in  the  ATOM  records  with  their  alpha-carbon\ncoordinates.  A  protein  sequence  is  considered  in  this\nstudy  only  if  it  contains  at  least  one  disordered  region\nwith  more  than  30  consecutive  residues.  Furthermore,\nprotein sequences of similarity identity of more than 70%\nagainst any protein sequence in the independent testing\ndata  have  been  removed,  resulting  in  693  protein\nsequences in the PDB693 dataset.\nAnother training set D184 is extracted from DisProt data-\nbase. DisProt is a curated database that provides informa-\ntion about proteins that entirely or partially lack a fixed\nthree-dimensional structure under putatively native con-\nditions [27]. The DisProt release 2.2 consists of 202 pro-\nteins,  including  431  distinct  disordered  regions  in  total.\nAmong the 202 proteins, there are 157 proteins that con-\ntain at least one disordered region longer than 30 consec-\nutive residues. There are more than 50 wholly disordered\nproteins in DisProt database which are annotated as serv-\ning certain  functions. D184  is  also  filtered  by  Cd-Hit  to\nremove  redundant  proteins which have more  than  70%\nidentity with some other proteins inside it or in any of the\nthree testing datasets.\nThe dataset R80 was prepared by Yang et al. in 2005 [6].\nThe 80 protein chains in this dataset are collected from the\nPDB database, and each protein chain contains a region of\nat least 21 consecutive disordered residues. Additionally,\nthe dataset U79 organized by Uversky et al. in 2000 [19]\nand  the  dataset  P80  provided  by  PONDR®  web  site\n(retrieved  in  February  2003)  are  also  compiled  into  the\nblind  testing  set,  where  the  dataset  U79  contains  79\ntotally disordered proteins and the dataset P80 contains\n80  completely  ordered  proteins.  By  using  Cd-Hit  again,\nwe observed that two sequences in P80 are subsequences\nof  a  protein  in  R80  and  a  pair  of  proteins  in  U79  have\nidentity  of  73%.  Like  Yang  et  al.  did  in  their  paper  [6],\nthese three datasets are employed as a platform for com-\nparison of our approach to some other present packages\ntargeting at protein disorder prediction. Thus, we did not\nchange the contents of these three datasets such that the\ncomparison can be carried out. In particular, the datasets\nU79  (fully  disordered  proteins)  and  P80  (globular  pro-\nteins)  together  suggest  whether  the  proposed  method  is\nunder- or over-predicting protein disorder.\nEvaluation measures\nPredicting a residue in the given protein sequence as order\nor disorder is a binary classification problem, and many\nmeasures  have  been  introduced  for  validation  issues\n[30,31]. Table 2 lists four widely used indices defined by\nprevious related works [6,18,24,30-32]. We employ these\nmeasures in this study to evaluate the performance of dif-\nferent feature sets or different packages. Sensitivity repre-\nsents  the  fraction  of  disordered  residues  correctly\nidentified  in  a  prediction  method,  while  specificity  indi-\ncates the fraction of ordered residues correctly identified.\nThe Matthews' correlation coefficient is a popular measure in\nmany bioinformatics problems [33-35]. However, sensitiv-\nity,  specificity,  and  the  Matthews'  correlation  coefficient  are\nseriously  affected  by  the  relative  frequency  of  the  target\nclass. Therefore, the above three measures are not suitable\nfor evaluating the performance in isolation. The probability\nexcess is independent of the relative class frequency, and\nthis measure can be reduced to sensitivity + specificity - 1\nconcisely  [6].  In  addition,  some  other  indices  including\nthe CASP S score, product, and probability excess are recom-\nmended and advised by CASP6 [31] and Yang et al. [6] for\nevaluating  the  performance  of  prediction.  Since  these\nthree  measures  have  the  same  tendency  with  probability\nexcess, we adopt only the probability excess in this paper for\nsimplicity and show the results of other measures in the\nsupplementary [See Additional file 1].\nFeature selection by cross-validation\nIn order to conduct a five-fold cross validation, the chains\nin datasets PDB693 and D184 are randomly split into five\nsubsets of approximately equal size. The results of uni-var-\niant analysis on each property feature are shown in Table\n3, in which the properties oriented from the same physic-\nochemical  group  are  put  together  for  the  following\ndependency analysis. The dependency analysis of feature\nselection aims to answer if a subset of a property group\nperforms better than the original one.\nBMC Bioinformatics 2006, 7:319\nIt  is  observed  in  Table  3  that  the  performance  of  some\nphysicochemical properties has been improved after they\nare  split  into  order/disorder-based  properties.  In  other\nwords, purifying the physicochemical properties by con-\nsidering the propensity for order or disorder contributes\nto  the  predicting  power  of  the  classifier.  HydrophobicO is\nthe best property among all of them and gets an explicit\nimprovement over Hydrophobic. On the other hand, nei-\nther PolarD nor PolarO get a better performance than Polar.\nIn  summary,  the  decomposition  of  some  conventional\nproperties  by  considering  the  order/disorder  propensity\nbrings explicit benefit in terms of the uni-variant analysis.\nAfter  the  best  property  for  each  group  has  been  deter-\nmined,  a  second  level  of  dependency  analysis  is  per-\nformed  by\nrelations  between\nphysicochemical  properties.  The  selected  features  are\nshown  in  Figure  1,  and  the  relation  of  these  features  is\nderived  by  incorporating  the  inheritance  relationships\nbetween the child properties and their parent properties.\nThat  is,  Aliphatic  and  AromaticO are  subsets  of  Hydropho-\nbicO, Tiny is a subset of Small, Positive and Negative are sub-\nsets  of  Charged,  which  recursively  is  a  subset  of  Polar.\nBased on these hierarchies, we aim to investigate if a com-\nbination  of  two  subproperties  performs  better  than  the\noriginal  one.  According  to  the  results  shown  in  Table  4\nand  Table  5,  property  features  Aliphatic+AromaticO  per-\nforms  better  than  HydrophobicO,  but  Positive+Negative  is\nnot superior to Polar.\nAfter  the  dependency  analysis,  the  redundancy  removal\nstep selects the best property from each cluster for the next\nstep of feature selection. The selected representative prop-\nerties  are  sorted  by  their  performance  in  the  uni-variant\nanalysis, resulting in the following order: Aliphatic+Aro-\nmaticO, Polar, SmallD, and Proline. The stepwise feature\nselection is preformed by adding one candidate property\nin each iteration until the predicting performance cannot\nBMC Bioinformatics 2006, 7:319\n47 and thus employs in total 4 × 47 = 188 attributes in the\nfeature vector for a query residue.\nThe relation of the selected properties after the first level of\nFigure 1\nredundancy analysis\nThe relation of the selected properties after the first level of\nredundancy analysis.\nbe improved. The results of the stepwise feature selection\nare shown in Table 6, indicating that the four properties,\nAliphatic, AromaticO, Polar, and SmallD, will be used in\nconstructing the final RBFN classifier for predicting pro-\ntein disorder. We name the final feature set of PSSMP with\nfour properties as FS-PSSMP-4, the feature set of ten con-\nventional  physicochemical  properties  as  FS-PSSMP-10,\nand  the  feature  set  employing  the  original  PSSM  as  FS-\nPSSM.\nSuggestion of window size\nAll the experiments described above have been conducted\nwith  different  window  sizes  of  11,  35,  and  59,  and  the\nresulted  feature  sets  are  the  same  as  reported.  However,\nthough they turn out the same result on feature selection,\nit is observed that larger window sizes such as 35 and 59\nare  favourable  when  prediction  accuracy  is  considered.\nThe current version of DisPSSMP adopts a window size of\nResults on testing data\nIn this subsection, we compare the performance of three\nfeature sets, FS-PSSM, FS-PSSMP-10, and the proposed FS-\nPSSMP-4 on the independent testing data, which consists\nof three datasets, R80, U79, and P80. In the following dis-\ncussions, the results on datasets U79 and P80 are always\ncombined when they are reported, because U79 contains\nonly  fully  disordered  proteins  and  P80  comprises  only\ncompletely ordered proteins. The results are shown in Fig-\nure 2 and Figure 3, with the window size changes from 11\nto  59  systematically.  It  is  clearly  shown  in  Figure  2  that\nlarger window sizes deliver better performance for all the\nfeature sets on the dataset R80, and the performance of FS-\nPSSMP-4 and FS-PSSMP-10 are generally better than FS-\nPSSM.\nOn the other hand, the probability excesses of all the feature\nsets in Figure 3 decrease considerably when the window\nsize is more than 39 due to the over-prediction in terminal\nregions.  In  fact,  FS-PSSMP-4  performs  worse  than  FS-\nPSSMP-10 and FS-PSSM when the window size is smaller\nthan 19, but better than both of them when the window\nsize is larger than 23. Figure 4 shows the overall perform-\nance by combining these three testing datasets. The aver-\nage  performance  of  FS-PSSMP-4  delivers  the  highest\nprobability  excess  among  the  three  feature  sets.  Also,  FS-\nPSSMP-4 has the fewest number of features among all. To\nsum up, FS-PSSMP-4 is superior to the others for its suc-\ncess of feature reduction and the improvement of accuracy\nand  efficiency,  and  it  delivers  roughly  the  same  level  of\naccuracy when the window size is larger than 35.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nComparison with existing packages\nIn  this  subsection  we  investigate  the  performance  of\ntwelve web servers or packages in protein disorder predic-\ntion,  some  of  which  were  included  in  comparison  with\nthe work of Yang et al. in their paper [6]. The predictors for\ncomparison  here  are  RONN  [6],  IUPred(short)  [36,37],\nIUPred(long)  [36,37],  DISpro  [38],  DISOPRED2  [4,32],\nPONDR® [21],  DisEMBL(hot)  [39],  DisEMBL(465)  [39],\nFoldIndex [19,40], PreLink [18], GlobPlot [41], and Dis-\nEMBL(coils) [39]. DISOPRED2 has a limit of 1000 resi-\ndues  per  protein,  so  1HN0,  1FO4,  and  1PS3  in  dataset\nR80  and  the  u15  protein  in  dataset  U79  have  been\nremoved  from  the  blind  testing  data  when  testing\nDISOPRED2.  IUPred provides  two choices  of  predicting\nshort or long disordered regions, and DisEMBL provides\nthree  choices:  DisEMBL(hot),  DisEMBL(465),  and  Dis-\nEMBL(coils). The plots in Figure 5, Figure 6, and Figure 7\nshow the results in the way of specificity versus sensitivity,\nand the plots are rotated anticlockwise by 45° to be equiv-\nalent to the plot of probability excess = sensitivity + specificity\n- 1.\nWhen compared with the other packages, DisPSSMP per-\nforms the best when probability excess is considered (with\na probability excess of 0.60). In Figure 5, DisPSSMP shows\nits  ability  in  identifying  the  boundaries  of  ordered  and\ndisordered regions. The predictors IUPred(long), DISpro,\nDISOPRED2, DisEMBL(465), and PreLink have a specifi-\ncity of more than 95% but a sensitivity of less than 50%,\nwhich show the tendency of predicting order more than\ndisorder. In contrast, the predictor DisEMBL(coils) with a\nsensitivity of less than 50% but a specificity of more than\n70%  has  the  tendency  of  predicting  disorder  more  than\norder.  It  depends  on  the  users  to  select  the  predictors\nIUPred(long), DISpro, DISOPRED2, DisEMBL(465), and\nPreLink  for  under-prediction  of  disorder  and  Dis-\nEMBL(coils) for over-prediction.\nThe main purpose of the experiment on datasets U79 and\nP80 is checking whether a method is under-predicting or\nover-predicting  protein  disorder.  As  shown  in  Figure  6,\nthe  results  of  all  the  methods  except  IUPred(long)  and\nFoldIndex  are  similar  to  that  in  the  main  blind  testing\ndataset R80 in Figure 5. The sensitivity of IUPred(long) and\nFoldIndex have an improvement of more then 20% in this\nexperiment, and they are ranked as the first and the fourth\namong  all  methods.  Since  IUPred(long)  has  been\ndesigned for predicting context-independent global disor-\nder that encompasses at least 30 consecutive residues in\nthe predicted disordered regions and adopts a large win-\ndow size like 101 [36,37], it is suitable for the recognition\nof the fully globular proteins and the totally unstructured\nproteins. On the other hand, the training data of FoldIn-\ndex contains 91 totally unfolded proteins and 275 globu-\nlar  proteins,  resulting  in  its  good  performance  in\nComparison of using different feature sets on testing data\nR80Figure 2\nComparison of using different feature sets on testing data\nR80.\nBMC Bioinformatics 2006, 7:319\nComparison of using different feature sets on testing data\nFigure 4\nR80, U79, and P80\nComparison of using different feature sets on testing data\nR80, U79, and P80.\ndiscriminating  fully  ordered  proteins  from  fully  disor-\ndered  proteins  [19,40].  Nevertheless,  due  to  the  lack  of\ninformation  about  the  boundaries  between  ordered\nregions and disordered regions, FoldIndex does not have\na good performance in R80.\nThe overall comparison is provided in Figure 7 by com-\nbining the results of Figure 5 and Figure 6. There are only\nthree methods that have a probability excess of more than\n0.50.  Although  IUPred(long)  has  an  distinguishing  per-\nformance in the combined blind test, this method cannot\npredict short disordered regions correctly due to the con-\nstraint of its algorithm. DisPSSMP and RONN have com-\nparable  performances  in  identifying  disordered  residues\nor ordered residues when no prior information is availa-\nble.\nProperty-based sequential patterns\nIn  the  previous  subsections  we  have  demonstrated  that\nthe  newly  defined  property  groups  perform  better  than\nthe  original  physicochemical  properties  when  incorpo-\nrated with position specific scoring matrices in predicting\nprotein disorder. In this subsection, we show the potential\nof the newly defined property features from another point\nof  view.  As  most  approaches  of  protein  disorder  predic-\ntion use the subsequences centered at the target residue as\ndiscriminative  features,  it  is  generally  believed  that\nsequential  patterns  of  amino  acids  may  provide  useful\nand  disorder\nabout  protein  order\ninformation\n[4,10,17,23,39,42].  Lise  et  al.  also  found  some  reliable\nand significant sequence patterns that characterize disor-\ndered  segments  [5].  Here,  we  show  by  examples  of\nsequence  patterns  that  the  selected  feature  properties\nindeed serve as better units in characterizing both ordered\nand disordered regions.\nIn this experiment, the ordered and disordered regions are\nextracted from both the training and testing datasets and\nare  organized  as  two  datasets  respectively.  The  scoring\nfunction  employed  by  Lise  et  al.  [5]  is  adopted  here  for\nevaluating the discriminating power of patterns. The score\nSo or an alternative score Sd are defined as in Eq. (1) to rep-\nresent the ordered and disordered preference of a pattern.\nProb.\nSens. Spec. MCC\nExcess\nMethod\n0.767 0.848 0.463 0.615\nDisPSSMP\n0.603 0.878 0.395 0.481\nRONN\n0.517 0.945 0.469 0.462\nIUPred(short)\n0.418 0.993 0.578 0.411\nDISpro\n0.436 0.961 0.449 0.397\nIUPred(long)\n0.405 0.972 0.470 0.377\nDISOPRED2*\nPONDR\n0.557 0.816 0.278 0.373\nDisEMBL(hot)  0.492 0.840 0.260 0.332\nDisEMBL(465)  0.334 0.981 0.437 0.315\n0.488 0.811 0.224 0.299\nFoldIndex\n0.237 0.947 0.219 0.183\nPreLink\nGlobPlot\n0.372 0.811 0.140 0.183\nDisEMBL(coils) 0.740 0.424 0.104 0.165\n*  For  DISOPRED2, the public  web  server  has  a  sequence  length  limit\nof 1000 residues; therefore, 1HN0, 1FO4, and 1PS3 in R80 cannot be\npredicted.\nComparing the performance of thirteen disorder prediction packages on testing data R80\nFigure 5\nComparing the performance of thirteen disorder prediction packages on testing data R80.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nProb.\nSens. Spec. MCC\nExcess\nMethod\n0.678 0.942 0.650 0.620\nIUPred(long)\n0.825 0.765 0.589 0.590\nDisPSSMP\n0.675 0.888 0.580 0.563\nRONN\n0.722 0.815 0.540 0.536\nFoldIndex\n0.556 0.915 0.511 0.472\nIUPred(short)\n0.469 0.981 0.543 0.449\nDISPRED2*\n0.632 0.782 0.420 0.414\nPONDR\nDISpro\n0.383 0.982 0.467 0.365\nDisEMBL(465)  0.348 0.978 0.430 0.327\nPreLink\n0.319 0.991 0.430 0.310\nDisEMBL(hot)  0.502 0.749 0.260 0.251\nDisEMBL(coils) 0.719 0.446 0.170 0.165\n0.308 0.821 0.151 0.129\nGlobPlot\n*  For  DISOPRED2, the public  web  server  has  a  sequence  length  limit\nof  1000  residues;  therefore,  the  u15  protein  in  U79  cannot  be\npredicted.\nComparing the performance of thirteen disorder prediction packages on testing data U79 and P80\nFigure 6\nComparing the performance of thirteen disorder prediction packages on testing data U79 and P80.\n=\nand\n= −\nwhere Mo and Md are the occurrences of a given pattern in\nthe ordered and disordered regions, and No and Nd are the\ntotal numbers of k-residue-long segments in the ordered\nregions and disordered regions. It is clearly that both So\nand Sd fall within 0 and 1, and if So = Sd = 0.5, the pattern\nhas  no  preference  for  order  or  disorder.  On  the  other\nhand, patterns with So = 0 and Sd = 1 and patterns with So\n= 1 and Sd = 0 are only observed in disordered and ordered\nregions respectively, and thus are considered as the most\ndifferential sequence patterns.\nSens. Spec. MCC\nMethod\n0.814 0.818 0.592\nDisPSSMP\n0.629 0.954 0.644\nIUPred(long)\n0.661 0.882 0.549\nRONN\n0.675 0.812 0.467\nFoldIndex\nIUPred(short)\n0.549 0.934 0.541\nDISOPRED2*  0.455 0.976 0.550\n0.617 0.804 0.407\nPONDR\nDISpro\n0.390 0.989 0.530\nDisEMBL(465)  0.345 0.980 0.465\nDisEMBL(hot)  0.500 0.807 0.308\nPreLink\n0.302 0.963 0.378\nDisEMBL(coils) 0.723 0.432 0.143\n0.321 0.814 0.146\nGlobPlot\nProb.\nExcess\n0.632\n0.583\n0.542\n0.487\n0.483\n0.430\n0.420\n0.379\n0.326\n0.308\n0.265\n0.155\n0.136\n*  For DISOPRED2, the public web server has a sequence length limit of\n1000 residues; therefore, 1HN0, 1FO4, and 1PS3 in R80 and the u15\nprotein in U79 cannot be predicted.\nComparing the performance of thirteen disorder prediction packages on testing data R80, U79, and P80\nFigure 7\nComparing the performance of thirteen disorder prediction packages on testing data R80, U79, and P80.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nIn this paper, only property patterns with three elements\nare  considered  since  the  occurrences  of  longer  patterns\nmight  not  be  frequent  enough  to  have  statistical  mean-\nings. Table 7 lists the occurrences and scores for each pat-\ntern  that  is  composed  of  three  identical  elements  of  a\nproperty group. Four clusters of property groups that are\ndemonstrated to be useful in protein order/disorder pre-\ndiction are considered here. In Table 7, the occurrences of\nthe synthetic His purification tags present in both training\nand testing data have been excluded because those succes-\nsive Hs bias the statistics of the observed patterns in this\nexperiment, especially in the case of property group Aro-\nmatic. The results reveal that both the patterns composed\nof  HydrophobicO  and  HydrophobicD  are  more  differential\nthan the pattern of three successive Hydrophobic residues.\nFor  the  property  groups  Polar,  Small  and  Aromatic,  it  is\nobserved that at least one of the two child properties in\neach property group deliver the patterns that are more dis-\ncriminative than the pattern of the parent property. This\nreveals  the  benefit  of  decomposing  the  conventional\nphysicochemical  properties  when  detecting  differential\npatterns.\nConclusion\nMore and more disordered regions are discovered in pro-\ntein sequences, and many of them are found to be func-\ntionally  significant.  Distinguishing  disordered  regions\nfrom ordered regions in protein sequences facilitates the\nexploration of protein structures and functions. The evo-\nlutionary information embedded in the PSSM tables has\nbeen demonstrated useful in many problems that predict\nthe functional or structural properties of a given residue in\nprotein sequences. Our results in this paper demonstrate\nthat considering the condensed PSSM with physicochem-\nical properties and large window sizes further benefits the\nperformance of protein disorder prediction.\nThe second and the most remarkable contribution of this\npaper is it removes some amino acid members from the\nwidely used physicochemical properties that are not use-\nful in predicting the disordered regions. This is achieved\nby decomposing each original physicochemical property\ninto two disjoint sets that are with a propensity for order\nand disorder, respectively. In addition, a hybrid wrapped\nfeature selection method that employs a uni-variant anal-\nysis and a cluster-based redundancy removal procedure is\nproposed  to  derive  a  satisfied  feature  set  efficiently.\nResults  on  the  independent  testing  data  reveal  that  the\nproposed  predicting  model  DisPSSMP  outperforms  the\nexisting packages performing similar tasks without either\nunder-predicting  or  over-predicting\nthe  disordered\nregions. Furthermore, the selected properties can be used\nto  derive  more  informative  patterns  that  facilitate  the\nstudy of protein disorder. As more and more disordered\nregions are found to be functionally significant, combin-\ning  predicted  information  of  secondary  structures  and\nconversed regions for predicting disordered regions with\nbinding ability deserves further studies.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nMethods\nIn  this  section,  we  provide  the  details  about  the  proce-\ndures of constructing PSSMPs, calculating the propensities\nfor order/disorder of an amino acid, training a predicting\nmodel, and selecting useful feature sets respectively.\nThe concept of building PSSMP is also exemplified in Fig-\nure 8, and we call the derived feature set as FS-PSSMP. As\nwill be explained the following subsection, different FS-\nPSSMPs can be generated when different property groups\nare specified in constructing FS-PSSMP.\nConstruction of PSSMP\nFor  each  protein  in  the  training  and  testing  data,  we\nemploy the PSI-BLAST program [43] to construct its posi-\ntion specific scoring matrix (PSSM). More specifically, the\nderived PSSM table is a position specific scoring matrix of\n20 amino acids, which provides the evolutionary informa-\ntion  about  the  protein  at  the  level  of  residue  types.  We\nname  the  feature  set  created  based  on  PSSMs  FS-PSSM,\nwhich is considered as the baseline of employing evolu-\ntionary  information  in  protein  disorder  prediction.  The\nprocedure of constructing FS-PSSM is shown in Figure 8.\nThe values in PSSMs, which each represents the likelihood\nof a particular residue substitution at specific position, are\nfirst rescaled to be within 0 and 1 using the logistic func-\ntion as suggested in [44]:\nConsidering propensity for order or disorder\nTable  8  lists  the  ten  physicochemical  groups  that  are\nwidely  used  in  analyzing  protein  sequences  [5,45].  This\npaper  proposes  considering  the  propensity  for  order  or\ndisorder  of  each  amino  acid  when  designing  a  property\ngroup  in  construction  of  PSSMPs.  The  propensities  for\norder/disorder of different amino acids have been widely\ndiscussed in the previous studies [13,17,19,21,22,39-42].\nSome of them specifically provide a measure of propen-\nsity based on the occurrences of each amino acid in differ-\nent  regions  of  the  datasets  they  collected  [22,39,41].  In\nthis  work,  we  recalculate  the  propensity  for  each  amino\nacid based our training data. The propensity P(AAi) of an\namino  acid  i  toward  ordered  or  disordered  regions  is\ndefined as follows:\n( )\nf x\n(\nP AA\ni\n)\n(\nF AA\ni\nO\n(\nF AA\ni\n(\nF AA\nD\ni\n)\nwhere x is the raw value in profile matrices and f(x) is the\nrescaled  value  of  x.  After  that,  the  rescaled  profiles  are\norganized into a number of w × 20 dimensional vectors,\neach of which serves as the feature vector of a target resi-\ndue  as  the  learning  or  predicting  instances.  When  w  is\nodd, which is always the case in our experiments, the slid-\ning  window  of  size  w  for  acquiring  the  feature  set  of  a\ngiven residue is centred at the target residue.\nWe  next  construct  the  feature  set  of  PSSMP  as  follows.\nFirst, columns in the original PSSMs are grouped by the\nuser defined property groups and the raw values from dif-\nferent columns are summed up as a new feature column.\nIn a PSSMP table, the entry yik of position i for property k\nis defined as follows:\n(1) i is the index of a position;\n(2) Akj = 1, if the j-th type of amino acid belong in the k-th\nproperty, and Akj = 0, otherwise;\n(3) xij is the raw value of the j-th type of amino acid in the\nposition i of the PSSM.\nwhere F(AAi) is the frequency of amino acid i in the train-\ning data and FO(AAi) and FD(AAi) are the frequencies of\namino acid i in the ordered and disordered regions of the\ntraining  data.  We  say  amino  acid  i  has  a  propensity  for\norder if P(AAi) > 0, and verse visa.\nWe provide in Figure 9 the frequency of each amino acid\nin the training data, and the propensity for order P(AAi).\nThe records shown in Figure 9 are compared with several\nprevious studies [6,22,39,41]. For most amino acids, the\npropensity  adopted  in  this  paper  is  consistent  with  the\npreference  of  at  least  one  of  the  previous  studies.  How-\never, for the amino acids H, A, and M, the propensity for\norder/disorder is not certain according to the experimen-\ntal  results  and  the  information  collected  at  hand.  More\nattention has been paid on the amino acid H, since some\nsynthetic His tags present in the training data somewhat\nbias the statistics provided in Figure 9. Even though, we\nstill adopt that H has a propensity for disorder because of\nits charged characteristic.\nBased  on  the  records  shown  in  Figure  9,  each  physico-\nchemical property in Table 8 can be decomposed into two\ndisjoint set as new order/disorder-based property features,\nas  shown  in  Table  9.  Three  exceptions  are:  Aliphatic  has\nonly  three  types  of  amino  acid  toward  ordered  regions,\nand  Negative  and  Proline  have  only  the  disorder  type  of\namino acids. It is noticed that there are some new proper-\nties which only comprise a single type of amino acid, such\nas AromaticD, PositiveO, ChargedO, and TinyO. All the prop-\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nSequence\n>1A0OB\nRQLALEAKGETPSAVTRLSVVAKSEPQDEQSRSQSPRRIILSRLKAGEVD\nLLEEELGHLTTLTDVVKGADSLSAILPGDIAEDDITAVLCFVIEADQITFE\nTVEVSPKISTPPVLKLAAEQAPTGRVEREKTTR\nPSSM\n-2R\n-16\n-1Q\n-10\n-2\n-4\nL\n-3\n-25A\n-2\n-2\nL\n-2\n-3\n…\n-2\n-1\n-4\n-2\n-3\n-4\n-4\n-2\n-1\n-2\n-303\n-217\n-3\n-3\n-4\n-2\n-2\n-1\n-22\n-4\n-1\n-4\n-30\n-3\n-2\n-3\n-3\n-3\n-351\n-2\n-2\n-1\n-241\n-4\n-3\n-4\n-2\n-401\n-2\n-2\n-3\n-11\n-3\n-1\n-2\n-1\n-1\n-3\n-2\n-10\n-2\n-3\n-3\n-3\n-2\n-3\n-3\n-2\n-2\n-2\n-2\n-2\n-10\n-10\n-1\n-1\n-2\n-1\n-1\n-2\n-201\n-2\n-1\n-2\n-1\n-30\n-1\n-1\n-1\n-1\n-12\n-251\n-251\n-3\n-1\n-1\n20 amino acids\nCreating feature set for each\nresidue by using a sliding window\nTransforming PSSM of amino\nacids into PSSM of properties\n(PSSMP), using m properties as\nan example\nPSSMP\nThe w×20 features of PSSM\nFeature-wise normalization\nThe feature set of PSSM\n(FS-PSSM)\nR\nQ\nL\nA\nL\nT\nT\n-32\n-26\n-13\n-16\n-13\n-2\n\n-32\n-19\n-23\n-12\n-10\n-8\n\n-6\n-10\n-9\n-7\n-10\n-9\n-2\n\n-7\n-4\n-5\n-3\n-2\n-4\n-2\n-3\nm properties\nCreating feature set for each\nresidue by using a sliding window\nw residues\nThe w×m features of PSSMP\nFeature-wise normalization\nThe feature set of PSSMP\n(FS-PSSMP)\nThe procedure of preparing feature set for training and testing data\nFigure 8\nThe procedure of preparing feature set for training and testing data.\nBMC Bioinformatics 2006, 7:319\nerty features will be considered in constructing the PSSMP\nfeature set for protein disorder prediction. Since the size of\nthe feature set is quite large and we do not expect all the\nproperty features are useful in predicting protein disorder,\na  feature  selection  method  will  be  conducted  to  find  a\ncombination of property features that benefits protein dis-\norder prediction.\nClassifier\nIn this study, the Radial Basis Function Network (RBFN)\nis used as the classifier for predicting protein disorder. The\nemployed QuickRBF package [46] is an efficient tool for\nconstructing  RBFN  classifiers,  which  uses  the  Cholesky\ndecomposition technique to resolve the least mean square\nerror  optimization  problem  when  constructing  a  RBFN\nclassifier. We rely on the efficiency of QuickRBF such that\na wrapped method of feature selection can be used in con-\nstructing  our  predictor  DisPSSMP,  where  the  'wrapped'\nmeans that the classifier is employed in feature selection\nprocess for evaluating the predicting power of the candi-\ndate feature set [47].\nThe propensity for order and the frequency of each amino\nFigure 9\nacid in the training set\nThe propensity for order and the frequency of each amino\nacid in the training set.\nAccording to the statistics provided in Table 1, the ratios\nof disorder residues to order residues in datasets PDB693\nand D184 are 1:3.83 and 1:2.03, respectively. In order to\ntackle  the  problem  of  the  skewed  datasets  with  unbal-\nanced  number  of  positive  and  negative  instances,  equal\nquantity of residues from ordered and disordered regions\nis used in constructing the classifier. In other words, the\nsame amount of ordered residues as that of the disordered\nresidues in the training sets is randomly selected and the\nothers are removed before the training process.\nFeature selection\nIt is doubted that all of the properties described in Table 8\nand Table 9 are useful in the problem of disorder predic-\ntion. Thus, it is suggested to conduct a procedure of fea-\nture selection on the training data to find a combination\nof features that perform the best in this problem. Feature\nselection is a common optimization problem for finding\nthe smallest subset of the features with the best classifica-\ntion performance [48]. However, finding the optimal fea-\nture  subset  is  not  easy,  since  there  are  2n  possible\ncombinations  when  given  n  features.  The  algorithm  of\nevaluating all subsets such as exhaustive search is imprac-\ntical for large n. Therefore, an alternative stepwise feature\nselection is presented in this paper that takes the charac-\nteristics of features into account to improve the computa-\ntional  efficiency.  Three  factors  are  frequently  used  in\nevaluating  the  performance  of  a  feature  selection\napproach: classification accuracy, size of the subset, and\ncomputational  efficiency  [48].  The  proposed  hybrid\nmethod  employs  the  efficient  uni-variant  analysis  first,\nand  uses  a  cluster-based  redundancy  removal  procedure\nto  speed  up  the  tedious  stepwise  feature  selection  that\nexplores the predicting power of combinations of multi-\nple features simultaneously.\nBMC Bioinformatics 2006, 7:319\nthe members of properties listed in Table 8 and Table 9 are\nclearly specified, it is easy to put the related features in one\ncluster. The first level considers the dependency between\nthe child properties in Table 9 and their parent properties\nin Table 8, and the second level considers the hierarchy\ndependency  between  the  physicochemical  properties\nlisted in Table 8. After the dependency analysis, the redun-\ndancy removal step brings one feature or the combination\nof two features that performs the best in each cluster to the\nnext  step,  stepwise  feature  selection.  The  representative\nproperties from each cluster are sorted by their perform-\nance, and the final subset is constructed by adding prop-\nerty  features  one  by  one  until  the  performance  of  cross\nvalidation on the training data cannot be improved.\nAvailability and requirements\nThe  executable  file:  http://biominer.bime.ntu.edu.tw/\ndispssmp or http://bioinfo.mc.ntu.edu.tw/dispssmp\nUni-variant analysis\nOperating system(s): Linux\nDependency analysis\nRedundancy removal\nStepwise feature selection\nThe flowchart of the hybrid feature selection mechanism\nFigure 10\nThe flowchart of the hybrid feature selection mechanism.\nProgramming language: C/Perl\nDependant packages: BLAST\nLicence: None\nAny restrictions to use by non-academics: None\nAbbreviations\nPSSM,  position  specific  scoring  matrix;  PSSMP,  con-\ndensed  position  specific  scoring  matrix  with  respect  to\nphysicochemical properties; PDB, Protein Data Bank; NN,\nneural networks; LR, logistic regression; DA, discriminant\nanalysis;  CASP,  the  critical  assessment  of  techniques  for\nprotein  structure  prediction;  TP,  true  positive;  FP,  false\npositive;  TN,  true  negative;  FN,  false  negative;  RBFN,\nradial basis function networks.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nAuthors' contributions\nCTS  initialized  the study and performed all calculations\nand  analysis,  and  drafted  the  manuscript.  CYC  aided  in\ndesign of the methodology, interpretation of the data, and\nmanuscript preparation. YYO participated in the integra-\ntion of the RBFN classifier. All authors read and approved\nthe manuscript.\nAdditional material\nAdditional File 1\nThis supplement provides the complete version of Table 2, 3, 4, 5, 6 , the\nrecords involved in Figure 5, 6, 7, and the protein list of the training data.\nClick here for file\n[http://www.biomedcentral.com/content/supplementary/1471-\n2105-7-319-S1.doc]",
         "",
         "",
         "downloading",
         "Address: 1Department of Computer Science and Information Engineering, National Taiwan University, Taipei, 106, Taiwan, R.O.C, 2Department\nof Bio-industrial Mechatronics Engineering, National Taiwan University, Taipei, 106, Taiwan, R.O.C, 3Graduate School of Biotechnology and\nBioinformatics, Yuan Ze University, Chung-Li, 320, Taiwan, R.O.C and 4Department of Computer Science and Engineering, Yuan Ze University,\nChung-Li, 320, Taiwan, R.O.C\nAbstract\nBackground: More and more disordered regions have been discovered in protein sequences, and\nmany  of  them  are  found  to  be  functionally  significant.  Previous  studies  reveal  that  disordered\nregions  of  a  protein  can  be  predicted  by  its  primary  structure,  the  amino  acid  sequence.  One\nobservation that has been widely accepted is that ordered regions usually have compositional bias\ntoward hydrophobic amino acids, and disordered regions are toward charged amino acids. Recent\nstudies  further  show  that  employing  evolutionary  information  such  as  position  specific  scoring\nmatrices  (PSSMs)  improves  the  prediction  accuracy  of  protein  disorder.  As  more  and  more\nmachine learning techniques have been introduced to protein disorder detection, extracting more\nuseful features with biological insights attracts more attention.\nResults: This paper first studies the effect of a condensed position specific scoring matrix with\nrespect to physicochemical properties (PSSMP) on the prediction accuracy, where the PSSMP is\nderived by merging several amino acid columns of a PSSM belonging to a certain property into a\nsingle column. Next, we decompose each conventional physicochemical property of amino acids\ninto two disjoint groups which have a propensity for order and disorder respectively, and show by\nexperiments  that  some  of  the  new  properties  perform  better  than  their  parent  properties  in\npredicting protein disorder. In order to get an effective and compact feature set on this problem,\nwe propose a hybrid feature selection method that inherits the efficiency of uni-variant analysis and\nthe effectiveness of the stepwise feature selection that explores combinations of multiple features.\nThe  experimental  results  show  that  the  selected  feature  set  improves  the  performance  of  a\nclassifier  built  with  Radial  Basis  Function  Networks  (RBFN)  in  comparison  with  the  feature  set\nconstructed with PSSMs or PSSMPs that adopt simply the conventional physicochemical properties.\nConclusion:  Distinguishing  disordered  regions  from  ordered  regions  in  protein  sequences\nfacilitates the exploration of protein structures and functions. Results based on independent testing\ndata reveal that the proposed predicting model DisPSSMP performs the best among several of the\nexisting  packages  doing  similar  tasks,  without  either  under-predicting  or  over-predicting  the\ndisordered regions. Furthermore, the selected properties are demonstrated to be useful in finding\ndiscriminating patterns for order/disorder classification.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nBackground\nA large proportion of proteins are found to be intrinsically\ndisordered, which refers to segments of protein sequences\nthat  do  not  form  well-defined  three-dimensional  struc-\ntures  in  their  native  states  [1-6].  Many  proteins  contain\nlocal  regions  of  such  disorder,  and  in  some  cases  entire\nproteins  appear  to  exist  as  ensembles  of  structures  [1-\n3,7,8]. Since protein subsequences that lack the ability to\nform a well-defined three-dimensional structure may still\nbe  functionally  related,  identifying  such  disordered\nregions  is  getting  more  and  more  important  for  both\nunderstanding  protein  functions  and  conducting  struc-\ntural analyses [6].\nProtein disorders have been observed in a variety of bio-\nlogical  functions,  including  molecular  recognition,  cell\nsignaling pathways, and enzyme catalysis [3,9]. Taking the\nyeast  proteome  as  an  example,  the  proteins  containing\ndisorder  are  often  located  in  the  cell  nucleus  and  are\ninvolved  in  transcription  regulation  and  cell  signalling\n[4]. The disordered regions that remain as flexible ensem-\nbles under physiological conditions allow the disordered\nbinding  sites  to  interact  with  several  different  targets\n[10,11].  The  disorder-to-order  transition  upon  binding\nprovides  high  specificity  with  modest  binding  affinities\n[7,12].\nIt has been shown in many studies that protein disorders\ncan be predicted from their primary sequences [10,13-18].\nThe prediction methods developed in recent years initiate\nthe possibility of identifying such disordered binding sites\nautomatically [14,17]. A more general concept is that all\nthe necessary information for the correct folding of a pro-\ntein  is  included  in  its  amino  acid  sequence  [19].  Disor-\ndered regions are comprised of a category of amino acids\ndistinct from that of ordered protein structures [20]. For\nexample,  amino  acids  of  aromatic  hydrophobic  groups\nare  known  to  be  good  for  the  general  stabilization  of\norder, and thus are less found in the disordered regions\n[13]. Incorporating information of the biased amino acid\ncomposition in a neural network predicts the locations of\ndisorder with accuracy better than random guesses [13].\nIn 1998, Romero et al. showed that more than 15,000 pro-\nteins in the Swiss-Prot database contain long disordered\nsegments (40 or more residues) based on their predictions\n[14,21].  Studies  on  some  of  these  disordered  regions\nreveal that they are evolutionarily conserved and possess\nbiological functions [10].\nSeveral machine learning approaches, such as neural net-\nlogistic  regression  (LR)\nworks  (NNs)  [6,10,13,22],\n[22,23],  discriminant  analysis  (DA)  [23],  ordinary  least\nsquares (OLS) [22], and support vector machines (SVM)\n[4,23,24]  have  been  introduced  to  protein  disorder  pre-\ndiction.  Since  different  classifiers  deliver  similar  predic-\ntion  accuracy  based  on  the  same  feature  set  [23],\nextracting more useful features with biological insights to\nimprove the quality of prediction attracts more attention\nin recent studies [10,24]. As amino acid composition has\nbeen demonstrated as a useful feature for detecting disor-\ndered regions, Jones et al. showed in their paper that using\nthe  position  specific  scoring  matrices  (PSSMs)  within  a\nspecific length of window centred at a given residue can\nimprove  the  accuracy  of  predicting  its  disorder  attribute\n[10]. The values in a position specific scoring matrix indi-\ncate the level of conservation of a position and the prop-\nerties  of  the  substituted  residues,  which  can  be  derived\ndirectly from executing PSI-BLAST for each target protein\nsequence. PSSMs have been demonstrated to be powerful\nin constructing feature sets for prediction of single-residue\nproperties from an amino acid sequence, such as category\nof  secondary  structures  or  solvent  accessibility  [10].  The\nevolutionary  information  summarized  in  a  PSSM  table\ngeneralizes  the  attribute  of  each  position  in  a  protein\nsequence, and thus improves the sensitivity of the predict-\ning model.\nA  development  of  this  approach  employs  a  condensed\nposition  specific  scoring  matrix  with  respect  to  physico-\nchemical properties (it will be called PSSMP in this paper)\nin predicting protein disorder, where the PSSMP is derived\nby  merging  several  amino  acid  columns  of  a  PSSM  that\nbelong to a certain property into a single column [24]. As\na  PSSM  brings  in  the  evolutionary  information  on  each\nposition, a PSSMP summarizes this information as prop-\nerty  attributes.  The  improvement  achieved  by  PSSMP\ndemonstrates that property attributes are more informa-\ntive than amino acid attributes in distinguishing ordered/\ndisordered  regions.  A  more  comprehensive  study  con-\nducted  in  this  paper  reveals  that  PSSMPs  outperform\nPSSMs  especially  when  the  employed  window  size  is\nlarge.\nWhen employing PSSMP tables as the feature set in pro-\ntein disorder prediction, two questions arise: (1) if all the\namino acids in one physicochemical property group con-\ntribute to the predicting power; and (2) if all the amino\nacids  in  one  physicochemical  property  group  result  in\nconsistent effect on prediction. It has been widely studied\nin previous works that the propensity for order or disorder\nof several amino acids is clear. Hydrophobic amino acids\nare more frequently observed in ordered regions than dis-\nordered  regions  [18,19].  Among  them  the  aromatic\namino  acids  are  present  in  different  locations  to  the\naliphatic amino acids [25]. On the other hand, the amino\nacids  with  charge  imbalance  are  often  present  in  disor-\ndered regions. In this paper, we argue that the propensity\nfor order or disorder of each amino acid should be consid-\nered when constructing PSSMP. After examining the statis-\ntics  derived  by  comparing  the  sequence  segments  in\nBMC Bioinformatics 2006, 7:319\nordered regions and disordered regions, we observe that\nnot all the hydrophobic amino acids possess a propensity\nfor order. Thus we suggest that each conventional physic-\nochemical  property  should  be  divided  into  two  smaller\ngroups  with  propensities  for  order  and  disorder  respec-\ntively,  such  as  hydrophobic  with  an  order  propensity\n(HydrophobicO) and hydrophobic with a disorder propen-\nsity  (HydrophobicD).  The  experiments  conducted  in  this\nwork  reveal  that  some  newly  derived  properties  provide\nmore accurate information regarding order or disorder.\nIncorporating  the  propensity  for  order/disorder  with\nphysicochemical properties in PSSMP produces informa-\ntive features for protein disorder prediction. However, the\nnumber of candidate features becomes larger than in the\ncase  when  only twenty  amino acids  are  considered.  The\nsize of the feature set gets even larger when a large window\nsize is employed, which might cause the performance of\nthe learning algorithms to be degraded due to abundant\nnoise.  Thus,  we  present  in  this  paper  a  feature  selection\nmechanism that considers both the size and effectiveness\nissues when determining a feature set on protein disorder\nprediction.  A  wrapper  approach  of  feature  selection  is\nemployed during training period that invokes the adopted\nRadial Basis Function Networks (RBFN) classifier to eval-\nuate  the  predicting  power  of  a  candidate  feature  set.  A\ncluster-based redundancy removal procedure is incorpo-\nrated to speed up the stepwise feature selection process,\nwhere two levels of redundancy among features are con-\nsidered.\nAs far as the experimental materials are concerned, a new\ndataset  PDB693  organized  from  the  Protein  Data  Bank\n(PDB) [26] database is coined in this work to benefit the\nstudy  on  protein  disorder.  PDB693  and  another  dataset\nD184 collected from Database of Protein Disorder (Dis-\nProt)  [27]  constitute  the  training  data  of  our  classifier\nDisPSSMP.  The  performance  of  DisPSSMP  is  compared\nwith twelve existing disorder prediction packages, where\nthe blind testing data comes from a recent study [6]. The\nexperimental results demonstrate that the selected prop-\nerty  features  are  informative  in  protein  disorder  predic-\ntion and can be used to derive discriminating patterns for\norder and disorder classification.\nResults and discussion\nIn  this  section,  we  first  describe  how  the  datasets  have\nbeen prepared and how the performance of prediction is\nevaluated. Next, we show the results of the feature selec-\ntion  after  conducting  cross-validation  on  the  training\ndata. At this stage, we also discuss the effect of the window\nsize  employed  in  constructing  PSSMP.  After  that,  the\nresultant feature set is employed in constructing the final\npredicting  model  DisPSSMP.  Finally,  the  testing  results\nare evaluated based on the bind testing data, and are com-\npared  with  other  existing  packages  performing  similar\ntasks. At the end of the section, we show the derived prop-\nerty sets can be used to discover patterns that distinguish\nordered and disordered regions.\nDatasets\nIn this study, five  datasets  have been collected  or newly\ncreated for training and validating processes. The detailed\nstatistics  about  each  dataset  are  provided  in  Table  1,\nincluding  the  number  of  chains,  ordered/disordered\nregions, and residues in ordered/disordered regions. The\ntraining data used in constructing the predictor DisPSSMP\nis  composed  of  datasets  PDB693  and  D184,  which  are\nrespectively  organized  from  PDB  and  DisProt  database\nbased on the procedures described in the following para-\ngraphs. Meanwhile, three datasets named R80, U79, and\nP80, which are taken from two related studies [6,19], con-\nstitute  an  independent  testing  data.  This  blind  dataset\nserves as a platform for comparing the performance of the\nproposed method with some other existing packages per-\nforming protein disorder prediction.\nThe  dataset  PDB693  contains  693  partially  disordered\nproteins,  where  the  locations  of  disordered  regions  are\nidentified by looking for the missing residues in a protein\nstructure  from  PDB  database  (28-Aug-2005  version).\nThere  are  originally  32204  structures  in  this  version  of\nPDB database, and those structures are filtered by a clus-\ntering  program  Cd-Hit  [28,29]  such  that  the  resultant\nBMC Bioinformatics 2006, 7:319\nnonredundant set containing no pair of protein sequences\nwith similarity identity of more than 70%. The so-called\nmissing residues are those present in the SEQRES records\nbut  not  in  the  ATOM  records  with  their  alpha-carbon\ncoordinates.  A  protein  sequence  is  considered  in  this\nstudy  only  if  it  contains  at  least  one  disordered  region\nwith  more  than  30  consecutive  residues.  Furthermore,\nprotein sequences of similarity identity of more than 70%\nagainst any protein sequence in the independent testing\ndata  have  been  removed,  resulting  in  693  protein\nsequences in the PDB693 dataset.\nAnother training set D184 is extracted from DisProt data-\nbase. DisProt is a curated database that provides informa-\ntion about proteins that entirely or partially lack a fixed\nthree-dimensional structure under putatively native con-\nditions [27]. The DisProt release 2.2 consists of 202 pro-\nteins,  including  431  distinct  disordered  regions  in  total.\nAmong the 202 proteins, there are 157 proteins that con-\ntain at least one disordered region longer than 30 consec-\nutive residues. There are more than 50 wholly disordered\nproteins in DisProt database which are annotated as serv-\ning certain  functions. D184  is  also  filtered  by  Cd-Hit  to\nremove  redundant  proteins which have more  than  70%\nidentity with some other proteins inside it or in any of the\nthree testing datasets.\nThe dataset R80 was prepared by Yang et al. in 2005 [6].\nThe 80 protein chains in this dataset are collected from the\nPDB database, and each protein chain contains a region of\nat least 21 consecutive disordered residues. Additionally,\nthe dataset U79 organized by Uversky et al. in 2000 [19]\nand  the  dataset  P80  provided  by  PONDR®  web  site\n(retrieved  in  February  2003)  are  also  compiled  into  the\nblind  testing  set,  where  the  dataset  U79  contains  79\ntotally disordered proteins and the dataset P80 contains\n80  completely  ordered  proteins.  By  using  Cd-Hit  again,\nwe observed that two sequences in P80 are subsequences\nof  a  protein  in  R80  and  a  pair  of  proteins  in  U79  have\nidentity  of  73%.  Like  Yang  et  al.  did  in  their  paper  [6],\nthese three datasets are employed as a platform for com-\nparison of our approach to some other present packages\ntargeting at protein disorder prediction. Thus, we did not\nchange the contents of these three datasets such that the\ncomparison can be carried out. In particular, the datasets\nU79  (fully  disordered  proteins)  and  P80  (globular  pro-\nteins)  together  suggest  whether  the  proposed  method  is\nunder- or over-predicting protein disorder.\nEvaluation measures\nPredicting a residue in the given protein sequence as order\nor disorder is a binary classification problem, and many\nmeasures  have  been  introduced  for  validation  issues\n[30,31]. Table 2 lists four widely used indices defined by\nprevious related works [6,18,24,30-32]. We employ these\nmeasures in this study to evaluate the performance of dif-\nferent feature sets or different packages. Sensitivity repre-\nsents  the  fraction  of  disordered  residues  correctly\nidentified  in  a  prediction  method,  while  specificity  indi-\ncates the fraction of ordered residues correctly identified.\nThe Matthews' correlation coefficient is a popular measure in\nmany bioinformatics problems [33-35]. However, sensitiv-\nity,  specificity,  and  the  Matthews'  correlation  coefficient  are\nseriously  affected  by  the  relative  frequency  of  the  target\nclass. Therefore, the above three measures are not suitable\nfor evaluating the performance in isolation. The probability\nexcess is independent of the relative class frequency, and\nthis measure can be reduced to sensitivity + specificity - 1\nconcisely  [6].  In  addition,  some  other  indices  including\nthe CASP S score, product, and probability excess are recom-\nmended and advised by CASP6 [31] and Yang et al. [6] for\nevaluating  the  performance  of  prediction.  Since  these\nthree  measures  have  the  same  tendency  with  probability\nexcess, we adopt only the probability excess in this paper for\nsimplicity and show the results of other measures in the\nsupplementary [See Additional file 1].\nFeature selection by cross-validation\nIn order to conduct a five-fold cross validation, the chains\nin datasets PDB693 and D184 are randomly split into five\nsubsets of approximately equal size. The results of uni-var-\niant analysis on each property feature are shown in Table\n3, in which the properties oriented from the same physic-\nochemical  group  are  put  together  for  the  following\ndependency analysis. The dependency analysis of feature\nselection aims to answer if a subset of a property group\nperforms better than the original one.\nBMC Bioinformatics 2006, 7:319\nIt  is  observed  in  Table  3  that  the  performance  of  some\nphysicochemical properties has been improved after they\nare  split  into  order/disorder-based  properties.  In  other\nwords, purifying the physicochemical properties by con-\nsidering the propensity for order or disorder contributes\nto  the  predicting  power  of  the  classifier.  HydrophobicO is\nthe best property among all of them and gets an explicit\nimprovement over Hydrophobic. On the other hand, nei-\nther PolarD nor PolarO get a better performance than Polar.\nIn  summary,  the  decomposition  of  some  conventional\nproperties  by  considering  the  order/disorder  propensity\nbrings explicit benefit in terms of the uni-variant analysis.\nAfter  the  best  property  for  each  group  has  been  deter-\nmined,  a  second  level  of  dependency  analysis  is  per-\nformed  by\nrelations  between\nphysicochemical  properties.  The  selected  features  are\nshown  in  Figure  1,  and  the  relation  of  these  features  is\nderived  by  incorporating  the  inheritance  relationships\nbetween the child properties and their parent properties.\nThat  is,  Aliphatic  and  AromaticO are  subsets  of  Hydropho-\nbicO, Tiny is a subset of Small, Positive and Negative are sub-\nsets  of  Charged,  which  recursively  is  a  subset  of  Polar.\nBased on these hierarchies, we aim to investigate if a com-\nbination  of  two  subproperties  performs  better  than  the\noriginal  one.  According  to  the  results  shown  in  Table  4\nand  Table  5,  property  features  Aliphatic+AromaticO  per-\nforms  better  than  HydrophobicO,  but  Positive+Negative  is\nnot superior to Polar.\nAfter  the  dependency  analysis,  the  redundancy  removal\nstep selects the best property from each cluster for the next\nstep of feature selection. The selected representative prop-\nerties  are  sorted  by  their  performance  in  the  uni-variant\nanalysis, resulting in the following order: Aliphatic+Aro-\nmaticO, Polar, SmallD, and Proline. The stepwise feature\nselection is preformed by adding one candidate property\nin each iteration until the predicting performance cannot\nBMC Bioinformatics 2006, 7:319\n47 and thus employs in total 4 × 47 = 188 attributes in the\nfeature vector for a query residue.\nThe relation of the selected properties after the first level of\nFigure 1\nredundancy analysis\nThe relation of the selected properties after the first level of\nredundancy analysis.\nbe improved. The results of the stepwise feature selection\nare shown in Table 6, indicating that the four properties,\nAliphatic, AromaticO, Polar, and SmallD, will be used in\nconstructing the final RBFN classifier for predicting pro-\ntein disorder. We name the final feature set of PSSMP with\nfour properties as FS-PSSMP-4, the feature set of ten con-\nventional  physicochemical  properties  as  FS-PSSMP-10,\nand  the  feature  set  employing  the  original  PSSM  as  FS-\nPSSM.\nSuggestion of window size\nAll the experiments described above have been conducted\nwith  different  window  sizes  of  11,  35,  and  59,  and  the\nresulted  feature  sets  are  the  same  as  reported.  However,\nthough they turn out the same result on feature selection,\nit is observed that larger window sizes such as 35 and 59\nare  favourable  when  prediction  accuracy  is  considered.\nThe current version of DisPSSMP adopts a window size of\nResults on testing data\nIn this subsection, we compare the performance of three\nfeature sets, FS-PSSM, FS-PSSMP-10, and the proposed FS-\nPSSMP-4 on the independent testing data, which consists\nof three datasets, R80, U79, and P80. In the following dis-\ncussions, the results on datasets U79 and P80 are always\ncombined when they are reported, because U79 contains\nonly  fully  disordered  proteins  and  P80  comprises  only\ncompletely ordered proteins. The results are shown in Fig-\nure 2 and Figure 3, with the window size changes from 11\nto  59  systematically.  It  is  clearly  shown  in  Figure  2  that\nlarger window sizes deliver better performance for all the\nfeature sets on the dataset R80, and the performance of FS-\nPSSMP-4 and FS-PSSMP-10 are generally better than FS-\nPSSM.\nOn the other hand, the probability excesses of all the feature\nsets in Figure 3 decrease considerably when the window\nsize is more than 39 due to the over-prediction in terminal\nregions.  In  fact,  FS-PSSMP-4  performs  worse  than  FS-\nPSSMP-10 and FS-PSSM when the window size is smaller\nthan 19, but better than both of them when the window\nsize is larger than 23. Figure 4 shows the overall perform-\nance by combining these three testing datasets. The aver-\nage  performance  of  FS-PSSMP-4  delivers  the  highest\nprobability  excess  among  the  three  feature  sets.  Also,  FS-\nPSSMP-4 has the fewest number of features among all. To\nsum up, FS-PSSMP-4 is superior to the others for its suc-\ncess of feature reduction and the improvement of accuracy\nand  efficiency,  and  it  delivers  roughly  the  same  level  of\naccuracy when the window size is larger than 35.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nComparison with existing packages\nIn  this  subsection  we  investigate  the  performance  of\ntwelve web servers or packages in protein disorder predic-\ntion,  some  of  which  were  included  in  comparison  with\nthe work of Yang et al. in their paper [6]. The predictors for\ncomparison  here  are  RONN  [6],  IUPred(short)  [36,37],\nIUPred(long)  [36,37],  DISpro  [38],  DISOPRED2  [4,32],\nPONDR® [21],  DisEMBL(hot)  [39],  DisEMBL(465)  [39],\nFoldIndex [19,40], PreLink [18], GlobPlot [41], and Dis-\nEMBL(coils) [39]. DISOPRED2 has a limit of 1000 resi-\ndues  per  protein,  so  1HN0,  1FO4,  and  1PS3  in  dataset\nR80  and  the  u15  protein  in  dataset  U79  have  been\nremoved  from  the  blind  testing  data  when  testing\nDISOPRED2.  IUPred provides  two choices  of  predicting\nshort or long disordered regions, and DisEMBL provides\nthree  choices:  DisEMBL(hot),  DisEMBL(465),  and  Dis-\nEMBL(coils). The plots in Figure 5, Figure 6, and Figure 7\nshow the results in the way of specificity versus sensitivity,\nand the plots are rotated anticlockwise by 45° to be equiv-\nalent to the plot of probability excess = sensitivity + specificity\n- 1.\nWhen compared with the other packages, DisPSSMP per-\nforms the best when probability excess is considered (with\na probability excess of 0.60). In Figure 5, DisPSSMP shows\nits  ability  in  identifying  the  boundaries  of  ordered  and\ndisordered regions. The predictors IUPred(long), DISpro,\nDISOPRED2, DisEMBL(465), and PreLink have a specifi-\ncity of more than 95% but a sensitivity of less than 50%,\nwhich show the tendency of predicting order more than\ndisorder. In contrast, the predictor DisEMBL(coils) with a\nsensitivity of less than 50% but a specificity of more than\n70%  has  the  tendency  of  predicting  disorder  more  than\norder.  It  depends  on  the  users  to  select  the  predictors\nIUPred(long), DISpro, DISOPRED2, DisEMBL(465), and\nPreLink  for  under-prediction  of  disorder  and  Dis-\nEMBL(coils) for over-prediction.\nThe main purpose of the experiment on datasets U79 and\nP80 is checking whether a method is under-predicting or\nover-predicting  protein  disorder.  As  shown  in  Figure  6,\nthe  results  of  all  the  methods  except  IUPred(long)  and\nFoldIndex  are  similar  to  that  in  the  main  blind  testing\ndataset R80 in Figure 5. The sensitivity of IUPred(long) and\nFoldIndex have an improvement of more then 20% in this\nexperiment, and they are ranked as the first and the fourth\namong  all  methods.  Since  IUPred(long)  has  been\ndesigned for predicting context-independent global disor-\nder that encompasses at least 30 consecutive residues in\nthe predicted disordered regions and adopts a large win-\ndow size like 101 [36,37], it is suitable for the recognition\nof the fully globular proteins and the totally unstructured\nproteins. On the other hand, the training data of FoldIn-\ndex contains 91 totally unfolded proteins and 275 globu-\nlar  proteins,  resulting  in  its  good  performance  in\nComparison of using different feature sets on testing data\nR80Figure 2\nComparison of using different feature sets on testing data\nR80.\nBMC Bioinformatics 2006, 7:319\nComparison of using different feature sets on testing data\nFigure 4\nR80, U79, and P80\nComparison of using different feature sets on testing data\nR80, U79, and P80.\ndiscriminating  fully  ordered  proteins  from  fully  disor-\ndered  proteins  [19,40].  Nevertheless,  due  to  the  lack  of\ninformation  about  the  boundaries  between  ordered\nregions and disordered regions, FoldIndex does not have\na good performance in R80.\nThe overall comparison is provided in Figure 7 by com-\nbining the results of Figure 5 and Figure 6. There are only\nthree methods that have a probability excess of more than\n0.50.  Although  IUPred(long)  has  an  distinguishing  per-\nformance in the combined blind test, this method cannot\npredict short disordered regions correctly due to the con-\nstraint of its algorithm. DisPSSMP and RONN have com-\nparable  performances  in  identifying  disordered  residues\nor ordered residues when no prior information is availa-\nble.\nProperty-based sequential patterns\nIn  the  previous  subsections  we  have  demonstrated  that\nthe  newly  defined  property  groups  perform  better  than\nthe  original  physicochemical  properties  when  incorpo-\nrated with position specific scoring matrices in predicting\nprotein disorder. In this subsection, we show the potential\nof the newly defined property features from another point\nof  view.  As  most  approaches  of  protein  disorder  predic-\ntion use the subsequences centered at the target residue as\ndiscriminative  features,  it  is  generally  believed  that\nsequential  patterns  of  amino  acids  may  provide  useful\nand  disorder\nabout  protein  order\ninformation\n[4,10,17,23,39,42].  Lise  et  al.  also  found  some  reliable\nand significant sequence patterns that characterize disor-\ndered  segments  [5].  Here,  we  show  by  examples  of\nsequence  patterns  that  the  selected  feature  properties\nindeed serve as better units in characterizing both ordered\nand disordered regions.\nIn this experiment, the ordered and disordered regions are\nextracted from both the training and testing datasets and\nare  organized  as  two  datasets  respectively.  The  scoring\nfunction  employed  by  Lise  et  al.  [5]  is  adopted  here  for\nevaluating the discriminating power of patterns. The score\nSo or an alternative score Sd are defined as in Eq. (1) to rep-\nresent the ordered and disordered preference of a pattern.\nProb.\nSens. Spec. MCC\nExcess\nMethod\n0.767 0.848 0.463 0.615\nDisPSSMP\n0.603 0.878 0.395 0.481\nRONN\n0.517 0.945 0.469 0.462\nIUPred(short)\n0.418 0.993 0.578 0.411\nDISpro\n0.436 0.961 0.449 0.397\nIUPred(long)\n0.405 0.972 0.470 0.377\nDISOPRED2*\nPONDR\n0.557 0.816 0.278 0.373\nDisEMBL(hot)  0.492 0.840 0.260 0.332\nDisEMBL(465)  0.334 0.981 0.437 0.315\n0.488 0.811 0.224 0.299\nFoldIndex\n0.237 0.947 0.219 0.183\nPreLink\nGlobPlot\n0.372 0.811 0.140 0.183\nDisEMBL(coils) 0.740 0.424 0.104 0.165\n*  For  DISOPRED2, the public  web  server  has  a  sequence  length  limit\nof 1000 residues; therefore, 1HN0, 1FO4, and 1PS3 in R80 cannot be\npredicted.\nComparing the performance of thirteen disorder prediction packages on testing data R80\nFigure 5\nComparing the performance of thirteen disorder prediction packages on testing data R80.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nProb.\nSens. Spec. MCC\nExcess\nMethod\n0.678 0.942 0.650 0.620\nIUPred(long)\n0.825 0.765 0.589 0.590\nDisPSSMP\n0.675 0.888 0.580 0.563\nRONN\n0.722 0.815 0.540 0.536\nFoldIndex\n0.556 0.915 0.511 0.472\nIUPred(short)\n0.469 0.981 0.543 0.449\nDISPRED2*\n0.632 0.782 0.420 0.414\nPONDR\nDISpro\n0.383 0.982 0.467 0.365\nDisEMBL(465)  0.348 0.978 0.430 0.327\nPreLink\n0.319 0.991 0.430 0.310\nDisEMBL(hot)  0.502 0.749 0.260 0.251\nDisEMBL(coils) 0.719 0.446 0.170 0.165\n0.308 0.821 0.151 0.129\nGlobPlot\n*  For  DISOPRED2, the public  web  server  has  a  sequence  length  limit\nof  1000  residues;  therefore,  the  u15  protein  in  U79  cannot  be\npredicted.\nComparing the performance of thirteen disorder prediction packages on testing data U79 and P80\nFigure 6\nComparing the performance of thirteen disorder prediction packages on testing data U79 and P80.\n=\nand\n= −\nwhere Mo and Md are the occurrences of a given pattern in\nthe ordered and disordered regions, and No and Nd are the\ntotal numbers of k-residue-long segments in the ordered\nregions and disordered regions. It is clearly that both So\nand Sd fall within 0 and 1, and if So = Sd = 0.5, the pattern\nhas  no  preference  for  order  or  disorder.  On  the  other\nhand, patterns with So = 0 and Sd = 1 and patterns with So\n= 1 and Sd = 0 are only observed in disordered and ordered\nregions respectively, and thus are considered as the most\ndifferential sequence patterns.\nSens. Spec. MCC\nMethod\n0.814 0.818 0.592\nDisPSSMP\n0.629 0.954 0.644\nIUPred(long)\n0.661 0.882 0.549\nRONN\n0.675 0.812 0.467\nFoldIndex\nIUPred(short)\n0.549 0.934 0.541\nDISOPRED2*  0.455 0.976 0.550\n0.617 0.804 0.407\nPONDR\nDISpro\n0.390 0.989 0.530\nDisEMBL(465)  0.345 0.980 0.465\nDisEMBL(hot)  0.500 0.807 0.308\nPreLink\n0.302 0.963 0.378\nDisEMBL(coils) 0.723 0.432 0.143\n0.321 0.814 0.146\nGlobPlot\nProb.\nExcess\n0.632\n0.583\n0.542\n0.487\n0.483\n0.430\n0.420\n0.379\n0.326\n0.308\n0.265\n0.155\n0.136\n*  For DISOPRED2, the public web server has a sequence length limit of\n1000 residues; therefore, 1HN0, 1FO4, and 1PS3 in R80 and the u15\nprotein in U79 cannot be predicted.\nComparing the performance of thirteen disorder prediction packages on testing data R80, U79, and P80\nFigure 7\nComparing the performance of thirteen disorder prediction packages on testing data R80, U79, and P80.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nIn this paper, only property patterns with three elements\nare  considered  since  the  occurrences  of  longer  patterns\nmight  not  be  frequent  enough  to  have  statistical  mean-\nings. Table 7 lists the occurrences and scores for each pat-\ntern  that  is  composed  of  three  identical  elements  of  a\nproperty group. Four clusters of property groups that are\ndemonstrated to be useful in protein order/disorder pre-\ndiction are considered here. In Table 7, the occurrences of\nthe synthetic His purification tags present in both training\nand testing data have been excluded because those succes-\nsive Hs bias the statistics of the observed patterns in this\nexperiment, especially in the case of property group Aro-\nmatic. The results reveal that both the patterns composed\nof  HydrophobicO  and  HydrophobicD  are  more  differential\nthan the pattern of three successive Hydrophobic residues.\nFor  the  property  groups  Polar,  Small  and  Aromatic,  it  is\nobserved that at least one of the two child properties in\neach property group deliver the patterns that are more dis-\ncriminative than the pattern of the parent property. This\nreveals  the  benefit  of  decomposing  the  conventional\nphysicochemical  properties  when  detecting  differential\npatterns.\nConclusion\nMore and more disordered regions are discovered in pro-\ntein sequences, and many of them are found to be func-\ntionally  significant.  Distinguishing  disordered  regions\nfrom ordered regions in protein sequences facilitates the\nexploration of protein structures and functions. The evo-\nlutionary information embedded in the PSSM tables has\nbeen demonstrated useful in many problems that predict\nthe functional or structural properties of a given residue in\nprotein sequences. Our results in this paper demonstrate\nthat considering the condensed PSSM with physicochem-\nical properties and large window sizes further benefits the\nperformance of protein disorder prediction.\nThe second and the most remarkable contribution of this\npaper is it removes some amino acid members from the\nwidely used physicochemical properties that are not use-\nful in predicting the disordered regions. This is achieved\nby decomposing each original physicochemical property\ninto two disjoint sets that are with a propensity for order\nand disorder, respectively. In addition, a hybrid wrapped\nfeature selection method that employs a uni-variant anal-\nysis and a cluster-based redundancy removal procedure is\nproposed  to  derive  a  satisfied  feature  set  efficiently.\nResults  on  the  independent  testing  data  reveal  that  the\nproposed  predicting  model  DisPSSMP  outperforms  the\nexisting packages performing similar tasks without either\nunder-predicting  or  over-predicting\nthe  disordered\nregions. Furthermore, the selected properties can be used\nto  derive  more  informative  patterns  that  facilitate  the\nstudy of protein disorder. As more and more disordered\nregions are found to be functionally significant, combin-\ning  predicted  information  of  secondary  structures  and\nconversed regions for predicting disordered regions with\nbinding ability deserves further studies.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nMethods\nIn  this  section,  we  provide  the  details  about  the  proce-\ndures of constructing PSSMPs, calculating the propensities\nfor order/disorder of an amino acid, training a predicting\nmodel, and selecting useful feature sets respectively.\nThe concept of building PSSMP is also exemplified in Fig-\nure 8, and we call the derived feature set as FS-PSSMP. As\nwill be explained the following subsection, different FS-\nPSSMPs can be generated when different property groups\nare specified in constructing FS-PSSMP.\nConstruction of PSSMP\nFor  each  protein  in  the  training  and  testing  data,  we\nemploy the PSI-BLAST program [43] to construct its posi-\ntion specific scoring matrix (PSSM). More specifically, the\nderived PSSM table is a position specific scoring matrix of\n20 amino acids, which provides the evolutionary informa-\ntion  about  the  protein  at  the  level  of  residue  types.  We\nname  the  feature  set  created  based  on  PSSMs  FS-PSSM,\nwhich is considered as the baseline of employing evolu-\ntionary  information  in  protein  disorder  prediction.  The\nprocedure of constructing FS-PSSM is shown in Figure 8.\nThe values in PSSMs, which each represents the likelihood\nof a particular residue substitution at specific position, are\nfirst rescaled to be within 0 and 1 using the logistic func-\ntion as suggested in [44]:\nConsidering propensity for order or disorder\nTable  8  lists  the  ten  physicochemical  groups  that  are\nwidely  used  in  analyzing  protein  sequences  [5,45].  This\npaper  proposes  considering  the  propensity  for  order  or\ndisorder  of  each  amino  acid  when  designing  a  property\ngroup  in  construction  of  PSSMPs.  The  propensities  for\norder/disorder of different amino acids have been widely\ndiscussed in the previous studies [13,17,19,21,22,39-42].\nSome of them specifically provide a measure of propen-\nsity based on the occurrences of each amino acid in differ-\nent  regions  of  the  datasets  they  collected  [22,39,41].  In\nthis  work,  we  recalculate  the  propensity  for  each  amino\nacid based our training data. The propensity P(AAi) of an\namino  acid  i  toward  ordered  or  disordered  regions  is\ndefined as follows:\n( )\nf x\n(\nP AA\ni\n)\n(\nF AA\ni\nO\n(\nF AA\ni\n(\nF AA\nD\ni\n)\nwhere x is the raw value in profile matrices and f(x) is the\nrescaled  value  of  x.  After  that,  the  rescaled  profiles  are\norganized into a number of w × 20 dimensional vectors,\neach of which serves as the feature vector of a target resi-\ndue  as  the  learning  or  predicting  instances.  When  w  is\nodd, which is always the case in our experiments, the slid-\ning  window  of  size  w  for  acquiring  the  feature  set  of  a\ngiven residue is centred at the target residue.\nWe  next  construct  the  feature  set  of  PSSMP  as  follows.\nFirst, columns in the original PSSMs are grouped by the\nuser defined property groups and the raw values from dif-\nferent columns are summed up as a new feature column.\nIn a PSSMP table, the entry yik of position i for property k\nis defined as follows:\n(1) i is the index of a position;\n(2) Akj = 1, if the j-th type of amino acid belong in the k-th\nproperty, and Akj = 0, otherwise;\n(3) xij is the raw value of the j-th type of amino acid in the\nposition i of the PSSM.\nwhere F(AAi) is the frequency of amino acid i in the train-\ning data and FO(AAi) and FD(AAi) are the frequencies of\namino acid i in the ordered and disordered regions of the\ntraining  data.  We  say  amino  acid  i  has  a  propensity  for\norder if P(AAi) > 0, and verse visa.\nWe provide in Figure 9 the frequency of each amino acid\nin the training data, and the propensity for order P(AAi).\nThe records shown in Figure 9 are compared with several\nprevious studies [6,22,39,41]. For most amino acids, the\npropensity  adopted  in  this  paper  is  consistent  with  the\npreference  of  at  least  one  of  the  previous  studies.  How-\never, for the amino acids H, A, and M, the propensity for\norder/disorder is not certain according to the experimen-\ntal  results  and  the  information  collected  at  hand.  More\nattention has been paid on the amino acid H, since some\nsynthetic His tags present in the training data somewhat\nbias the statistics provided in Figure 9. Even though, we\nstill adopt that H has a propensity for disorder because of\nits charged characteristic.\nBased  on  the  records  shown  in  Figure  9,  each  physico-\nchemical property in Table 8 can be decomposed into two\ndisjoint set as new order/disorder-based property features,\nas  shown  in  Table  9.  Three  exceptions  are:  Aliphatic  has\nonly  three  types  of  amino  acid  toward  ordered  regions,\nand  Negative  and  Proline  have  only  the  disorder  type  of\namino acids. It is noticed that there are some new proper-\nties which only comprise a single type of amino acid, such\nas AromaticD, PositiveO, ChargedO, and TinyO. All the prop-\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nSequence\n>1A0OB\nRQLALEAKGETPSAVTRLSVVAKSEPQDEQSRSQSPRRIILSRLKAGEVD\nLLEEELGHLTTLTDVVKGADSLSAILPGDIAEDDITAVLCFVIEADQITFE\nTVEVSPKISTPPVLKLAAEQAPTGRVEREKTTR\nPSSM\n-2R\n-16\n-1Q\n-10\n-2\n-4\nL\n-3\n-25A\n-2\n-2\nL\n-2\n-3\n…\n-2\n-1\n-4\n-2\n-3\n-4\n-4\n-2\n-1\n-2\n-303\n-217\n-3\n-3\n-4\n-2\n-2\n-1\n-22\n-4\n-1\n-4\n-30\n-3\n-2\n-3\n-3\n-3\n-351\n-2\n-2\n-1\n-241\n-4\n-3\n-4\n-2\n-401\n-2\n-2\n-3\n-11\n-3\n-1\n-2\n-1\n-1\n-3\n-2\n-10\n-2\n-3\n-3\n-3\n-2\n-3\n-3\n-2\n-2\n-2\n-2\n-2\n-10\n-10\n-1\n-1\n-2\n-1\n-1\n-2\n-201\n-2\n-1\n-2\n-1\n-30\n-1\n-1\n-1\n-1\n-12\n-251\n-251\n-3\n-1\n-1\n20 amino acids\nCreating feature set for each\nresidue by using a sliding window\nTransforming PSSM of amino\nacids into PSSM of properties\n(PSSMP), using m properties as\nan example\nPSSMP\nThe w×20 features of PSSM\nFeature-wise normalization\nThe feature set of PSSM\n(FS-PSSM)\nR\nQ\nL\nA\nL\nT\nT\n-32\n-26\n-13\n-16\n-13\n-2\n\n-32\n-19\n-23\n-12\n-10\n-8\n\n-6\n-10\n-9\n-7\n-10\n-9\n-2\n\n-7\n-4\n-5\n-3\n-2\n-4\n-2\n-3\nm properties\nCreating feature set for each\nresidue by using a sliding window\nw residues\nThe w×m features of PSSMP\nFeature-wise normalization\nThe feature set of PSSMP\n(FS-PSSMP)\nThe procedure of preparing feature set for training and testing data\nFigure 8\nThe procedure of preparing feature set for training and testing data.\nBMC Bioinformatics 2006, 7:319\nerty features will be considered in constructing the PSSMP\nfeature set for protein disorder prediction. Since the size of\nthe feature set is quite large and we do not expect all the\nproperty features are useful in predicting protein disorder,\na  feature  selection  method  will  be  conducted  to  find  a\ncombination of property features that benefits protein dis-\norder prediction.\nClassifier\nIn this study, the Radial Basis Function Network (RBFN)\nis used as the classifier for predicting protein disorder. The\nemployed QuickRBF package [46] is an efficient tool for\nconstructing  RBFN  classifiers,  which  uses  the  Cholesky\ndecomposition technique to resolve the least mean square\nerror  optimization  problem  when  constructing  a  RBFN\nclassifier. We rely on the efficiency of QuickRBF such that\na wrapped method of feature selection can be used in con-\nstructing  our  predictor  DisPSSMP,  where  the  'wrapped'\nmeans that the classifier is employed in feature selection\nprocess for evaluating the predicting power of the candi-\ndate feature set [47].\nThe propensity for order and the frequency of each amino\nFigure 9\nacid in the training set\nThe propensity for order and the frequency of each amino\nacid in the training set.\nAccording to the statistics provided in Table 1, the ratios\nof disorder residues to order residues in datasets PDB693\nand D184 are 1:3.83 and 1:2.03, respectively. In order to\ntackle  the  problem  of  the  skewed  datasets  with  unbal-\nanced  number  of  positive  and  negative  instances,  equal\nquantity of residues from ordered and disordered regions\nis used in constructing the classifier. In other words, the\nsame amount of ordered residues as that of the disordered\nresidues in the training sets is randomly selected and the\nothers are removed before the training process.\nFeature selection\nIt is doubted that all of the properties described in Table 8\nand Table 9 are useful in the problem of disorder predic-\ntion. Thus, it is suggested to conduct a procedure of fea-\nture selection on the training data to find a combination\nof features that perform the best in this problem. Feature\nselection is a common optimization problem for finding\nthe smallest subset of the features with the best classifica-\ntion performance [48]. However, finding the optimal fea-\nture  subset  is  not  easy,  since  there  are  2n  possible\ncombinations  when  given  n  features.  The  algorithm  of\nevaluating all subsets such as exhaustive search is imprac-\ntical for large n. Therefore, an alternative stepwise feature\nselection is presented in this paper that takes the charac-\nteristics of features into account to improve the computa-\ntional  efficiency.  Three  factors  are  frequently  used  in\nevaluating  the  performance  of  a  feature  selection\napproach: classification accuracy, size of the subset, and\ncomputational  efficiency  [48].  The  proposed  hybrid\nmethod  employs  the  efficient  uni-variant  analysis  first,\nand  uses  a  cluster-based  redundancy  removal  procedure\nto  speed  up  the  tedious  stepwise  feature  selection  that\nexplores the predicting power of combinations of multi-\nple features simultaneously.\nBMC Bioinformatics 2006, 7:319\nthe members of properties listed in Table 8 and Table 9 are\nclearly specified, it is easy to put the related features in one\ncluster. The first level considers the dependency between\nthe child properties in Table 9 and their parent properties\nin Table 8, and the second level considers the hierarchy\ndependency  between  the  physicochemical  properties\nlisted in Table 8. After the dependency analysis, the redun-\ndancy removal step brings one feature or the combination\nof two features that performs the best in each cluster to the\nnext  step,  stepwise  feature  selection.  The  representative\nproperties from each cluster are sorted by their perform-\nance, and the final subset is constructed by adding prop-\nerty  features  one  by  one  until  the  performance  of  cross\nvalidation on the training data cannot be improved.\nAvailability and requirements\nThe  executable  file:  http://biominer.bime.ntu.edu.tw/\ndispssmp or http://bioinfo.mc.ntu.edu.tw/dispssmp\nUni-variant analysis\nOperating system(s): Linux\nDependency analysis\nRedundancy removal\nStepwise feature selection\nThe flowchart of the hybrid feature selection mechanism\nFigure 10\nThe flowchart of the hybrid feature selection mechanism.\nProgramming language: C/Perl\nDependant packages: BLAST\nLicence: None\nAny restrictions to use by non-academics: None\nAbbreviations\nPSSM,  position  specific  scoring  matrix;  PSSMP,  con-\ndensed  position  specific  scoring  matrix  with  respect  to\nphysicochemical properties; PDB, Protein Data Bank; NN,\nneural networks; LR, logistic regression; DA, discriminant\nanalysis;  CASP,  the  critical  assessment  of  techniques  for\nprotein  structure  prediction;  TP,  true  positive;  FP,  false\npositive;  TN,  true  negative;  FN,  false  negative;  RBFN,\nradial basis function networks.\nBMC Bioinformatics 2006, 7:319\nhttp://www.biomedcentral.com/1471-2105/7/319\nAuthors' contributions\nCTS  initialized  the study and performed all calculations\nand  analysis,  and  drafted  the  manuscript.  CYC  aided  in\ndesign of the methodology, interpretation of the data, and\nmanuscript preparation. YYO participated in the integra-\ntion of the RBFN classifier. All authors read and approved\nthe manuscript.\nAdditional material\nAdditional File 1\nThis supplement provides the complete version of Table 2, 3, 4, 5, 6 , the\nrecords involved in Figure 5, 6, 7, and the protein list of the training data.\nClick here for file\n[http://www.biomedcentral.com/content/supplementary/1471-\n2105-7-319-S1.doc]",
         "Accurate Prediction of Protein Disordered Regions by Mining Protein Structure Data;RONN: the bio-basis function neural network technique applied to the detection of natively disordered regions in proteins;IUPred: web server for the prediction of intrinsically unstructured regions of proteins based on estimated energy content;FoldIndex©: a simple tool to predict whether a given protein sequence is intrinsically unfolded",
         "Protein disorder prediction by condensed PSSM considering propensity for order or disorder",
         "None"
        ],
        [
         "21",
         "00117e0d2c89e54b24f9ee6d1b78b479cfc8b357",
         "Abstract One of the central goals of mathematical epidemiology is to predict dis-\nease transmission patterns in populations. Two models are commonly used to pre-\ndict spatial spread of a disease. The ﬁrst is the distributed-contacts model, often\ndescribed by a contact distribution among stationary individuals. The second is\nthe distributed-infectives model, often described by the diffusion of infected indi-\nviduals. However, neither approach is ideal when individuals move within home\nranges. This paper presents a uniﬁed modeling hypothesis, called the restricted-\nmovement model. We use this model to predict spatial spread in settings where\ninfected individuals move within overlapping home ranges. Using mathematical\nand computational approaches, we show that our restricted-movement model has\nthree limits: the distributed-contacts model, the distributed-infectives model, and a\nthird, less studied advective distributed-infectives limit. We also calculate approx-\nimate upper bounds for the rates of an epidemic’s spatial spread. Guidelines are\nsuggested for determining which limit is most appropriate for a speciﬁc disease.",
         "Timothy C. Reluga,J. Medlock,A. Galvani",
         "Timothy C. Reluga,J. Medlock,A. Galvani",
         "https://link.springer.com/content/pdf/10.1007/s11538-005-9027-y.pdf",
         "Abstract One of the central goals of mathematical epidemiology is to predict dis-\nease transmission patterns in populations. Two models are commonly used to pre-\ndict spatial spread of a disease. The ﬁrst is the distributed-contacts model, often\ndescribed by a contact distribution among stationary individuals. The second is\nthe distributed-infectives model, often described by the diffusion of infected indi-\nviduals. However, neither approach is ideal when individuals move within home\nranges. This paper presents a uniﬁed modeling hypothesis, called the restricted-\nmovement model. We use this model to predict spatial spread in settings where\ninfected individuals move within overlapping home ranges. Using mathematical\nand computational approaches, we show that our restricted-movement model has\nthree limits: the distributed-contacts model, the distributed-infectives model, and a\nthird, less studied advective distributed-infectives limit. We also calculate approx-\nimate upper bounds for the rates of an epidemic’s spatial spread. Guidelines are\nsuggested for determining which limit is most appropriate for a speciﬁc disease.",
         "existing models by describing individual movement relative to a “home” location\nto which individuals return sporadically. The restricted-movement model also uni-\nﬁes the distributed-infectives and distributed-contacts models in a common frame-\nwork and further illuminates their relationships. When infected individuals re-\nturn home rapidly, speeds in the restricted-movement model coincide with those\nof the distributed-contacts model. Conversely, when infected individuals return\nhome very slowly relative to both diffusion and transmission, rates of spread in\nthe restricted-movement model coincide with those of the diffusive distributed-\ninfectives model. In the distributed-contacts and diffusive distributed-infectives\nlimits, the asymptotic rates of spread can be calculated from the simple expres-\nsions we have formulated.\nAn imprecise approximation in the advective distributed-infectives limit is also\ngiven, but this limit requires further study. In the advective limit, the dispersal\nprocess is closely related to the “velocity jump process” of Othmer et al. (1988).\nThe velocity jump process is motivated by the “run and tumble movement” of\nﬂagellated bacteria such as E. coli. This process describes the movement of an or-\nganism that alternately “runs” at a constant velocity and then “tumbles” to a new\nvelocity. Alt (1980) derives a special case of the velocity jump process from bio-\nlogical assumptions and studies a diffusion approximation. Othmer et al. (1988)\nderives the velocity jump process from a generalization of the telegrapher’s equa-\ntion and then analyzes some special cases. Schwetlick (2000) shows the existence\nof traveling-wave solutions for the velocity jump process when the velocities of\nindividuals are bounded and gives an expression for the minimum wave speed.\nHillen and Othmer (2000) and Othmer and Hillen (2002) give a detailed study of\ndiffusion approximations for the velocity jump process.\nIndividual-based models of social networks (Eubank et al., 2004) contain both\nstatic components, such as family contacts, and transient components, such as peo-\nple sharing a crowded subway car. Under the restricted-movement model, tran-\nsient components will govern the rate of spatial epidemic spread when trans-\nmission frequently occurs before the movement of an individual has relaxed to\nits equilibrium distributions. When many transmissions occur during relaxation,\ndistributed-infectives models are most appropriate for describing spread. If trans-\nmissions seldom occur before relaxation, the distributed-contacts approach is the\nmost appropriate. The boundary between these two limits can be approximated\nby equating Eqs. (24) and (20). In the speciﬁc case where removal is slow (µ ≈ 0),\ninfection does not alter individual behavior (Dsαi = Di αs), and individuals return\nhome every night (αi = 1), the distributed-infectives model must be considered\nwhen there are more than 0.7 transmissions per day per individual. For example,\nsmallpox and measles are transmitted more than once a day on average, and are\nlikely to require distributed-infectives models, while scarlet fever is transmitted\nless than 0.5 times a day, suggesting a distributed-contacts model may be adequate\n(Anderson and May, 1991; Fenner et al., 1988).\nOur results for the restricted-movement model are robust to a greater or lesser\nextent when the underlying assumptions are relaxed. For instance, the general-\nization from 1 to 2 spatial dimensions may prolong the transient dynamics, but is\nnot expected to effect our asymptotic results. The mean-ﬁeld description of dis-\nease transmission implicitly assumes a large, dense population. Disease spread is\nlikely to be slower in a sparse population with fewer contacts between suscepti-\nble and infected individuals. The accumulation of immune individuals may further\nslow the spread of disease. We have also assumed that transmission can only oc-\ncur through close contact between individuals. We have not explored scenarios\nwhere transmission is facilitated exogenously by vectors or other environmental\nfactors.\nThe Ornstein–Uhlenbeck process is a convenient assumption which is reason-\nable for many animal species. Our qualitative results should also hold for other\nempirically derived movement models. Consider alternative cases where infected\nindividuals advect away from their point of infection for a ﬁxed time, after which\nthey are stationary for the duration of infection. In such cases, we can expect epi-\ndemic spread to correspond to an advective distributed-infectives model for fast\ntransmission, but a distributed-contacts model for slow transmission.\nGreater biological realism can be achieved from the restricted-movement model\nby relaxing the assumptions of temporal, spatial, and demographic homogeneity.\nTemporal homogeneity can be relaxed to allow for daily and weekly movement\ncycles or aperiodic transients. Spatial homogeneity and isotropy can be loosened,\nparticularly in two dimensions, to allow for spatial variation in population den-\nsity and individual movement patterns. The current model captures some demo-\ngraphic heterogeneity by allowing infected and susceptible individuals to behave\ndifferently, but these classes themselves are assumed to be homogeneous. De-\nmographic heterogeneity may be further explored by allowing population age\nstructure and correlated movement of individuals. Each of these generalizations\ngreatly increases the amount of biological information that can be incorporated\ninto the restricted-movement model.\nOur methods may also be of use in ecological studies of animal populations\nwhere individuals can exhibit behavioral biases in their movement. Density-\ndependence can be included by adding non-linear growth and mortality terms to\nEq. (9). Such models provide an extension of diffusion models and might provide\nuseful analytic results.\nIn conclusion, the restricted-movement model provides an improved approach\nto spatial epidemic modeling, and elucidates the relationship between popular\ndistributed-contacts and distributed-infectives approaches. Along with the issues\ndiscussed above, future work may include applications of this theory to speciﬁc\nepidemic control problems.",
         "1. Introduction\nA fundamental challenge in mathematical epidemiology is determining how the\nstructure of a population inﬂuences disease transmission. One important aspect is\nthe spatial structure. For instance, the SARS epidemic spread through 12 coun-\ntries within a few weeks. Projections of the spatial spread of an epidemic and the\ninteractions of human movement at multiple levels with a response protocol will\nfacilitate the assessment of policy alternatives. Spatially-explicit models are neces-\nsary to evaluate the efﬁcacy of movement controls (Riley et al., 2003; Eubank et al.,\n2004). Models that ignore spatial structure can lead to inaccuracy in the prediction\nof population dynamics (Durrett and Levin, 1994).\nA wide variety of methods have been used for the study of spatially struc-\ntured epidemics. Some examples include cellular automata (Doran and Laffan,\n2005; Fuks and Lawniczak, 2001), networks (Bauch and Galvani, 2003; Newman,\n2002), metapopulations (Lloyd and Jansen, 2004; Arino and van den Driessche,\n2003; Keeling and Gilligan, 2000), individual-based models (Bian, 2004), moment-\nclosure approximations (Thomson and Ellner, 2003; Filipe and Maule, 2003),\ninteracting particle systems (Durrett, 1999; Schinazi, 1996), diffusion equations\n(Caraco et al., 2002; M ´endez, 1998), integro-differential equations (Medlock and\nKot, 2003), and integrodifference equations (Allen and Ernest, 2002). Two partic-\nularly useful approaches for describing a disease’s rate of spatial spread are the\ndistributed-contacts and distributed-infectives models.\nKendall (1965) and Mollison (1972) developed the theory of asymptotic spread\nrates in distributed-contacts models. Their family of integral-equation models as-\nsumed that each individual is stationary and has a distribution of contacts over\nspace (Mollison, 1972). Distributed-contacts models are particularly appropriate\nfor the study of plant diseases (van den Bosch et al., 1988; Metz and van den Bosch,\n1995; van den Bosch et al., 1999), but researchers are also using the closely related\nframework of contact networks to study disease transmission in human popula-\ntions (Newman, 2002; Meyers et al., 2005; Read and Keeling, 2003).\nShortly after Mollison’s ﬁrst work, Noble (1974) formulated the closely related\ntheory of asymptotic spread rates under the distributed-infectives model. Building\non earlier work tracing back to Fisher and Kolmogorov, Noble applied diffusion\ntheory to the spread of bubonic plague in Europe (Noble, 1974). Noble’s model\nrelies on the assumptions that disease is transmitted through interactions between\ndispersing individuals, and that infected individuals move in uncorrelated random\nwalks. Medlock and Kot (2003) recently developed a distributed-infectives frame-\nwork that uses a ﬂexible kernel-based approach similar to that employed in earlier\ndistributed-contact models. They found that inappropriate application of either\nthe distributed-contact or distributed-infectives approaches can generate inaccu-\nrate projections of epidemic spread.\nIn many epidemiological contexts, the transmission process involves compo-\nnents of both distributed contacts and distributed infectives. Hybrid models have\nbeen proposed for this situation. For instance, the equation of Bailey (1975),\n\n\nn + DDC∇ 2n\n+ DDI∇ 2n − µn,\nallows for both diffusive contacts and diffusive movement. An analogous spatially\nnon-local model,\nkDI(x, y) [n(y, t) − n(x, t)] dy − µn,\nincorporates both a movement kernel kDI and a contact kernel kDC. Busenberg and\nTravis (1983) study a related model with diffusive movement of individuals and\ndistributed contacts. Recent study of a model where individuals transition between\nstationary and motile states (Hadeler, 2003) demonstrates that both distributed-\ninfectives and distributed-contacts models can appear as limiting cases of a single\nmodel.\nSpatially structured epidemic models are useful tools in the study of geographic\nepidemic spread. In particular, spatial models can be used to estimate the speed of\ngeographic spread. Estimates of rapidity of disease dissemination can, in turn, be\nused to guide policy decisions. For many linear models, such as the one discussed\nin this paper, researchers have shown that there is a minimum speed c∗ for trav-\neling wave solutions and that in many biologically realistic settings, solutions tend\nto approach advancing fronts that travel no faster than c∗. The speed of advancing\nfronts in non-linear deterministic and linear stochastic models has been shown in\na number of instances to be given by the minimum speed c∗ from a corresponding\nlinear model (Mollison, 1991; Kot et al., 2004). The connection between speeds\nof non-linear models with their corresponding linear models holding more gener-\nally is the subject of the “linear conjecture” (van den Bosch et al., 1990; Mollison,\n1991). Moreover, the minimum speed from a linear model often provides a good\nupper bound for the speeds observed in non-linear stochastic models (Mollison,\n1972; McKean, 1975; Mollison, 1977; Lewis, 2000; Lewis and Pacala, 2000; Clark\net al., 2001; Snyder, 2003).\nHere we consider a scenario that does not neatly conform to the assumptions\nof either the distributed-contacts or distributed-infectives hypotheses: the case\nof individuals moving within overlapping home ranges. Many animal species live\nwithin a “home range” and periodically return to their “home.” We develop a\nrestricted-movement model that describes movement relative to a “home” loca-\ntion, and study the rate of spatial spread of disease using mathematical analysis and\nstochastic simulations. We show that the restricted-movement model possesses\nlimits corresponding to the distributed-contact and distributed-infective models,\nbut also possesses a third limit, which we call the advective distributed-infectives\nmodel. Approximations to the rate of spread are provided in all three limits. These\napproximations agree with the speeds observed in individual-based Monte Carlo\nsimulations in all but the advective distributed-infectives limit.\n2. Restricted-movement model\nConsider a large population distributed uniformly over a one-dimensional world.\nEvery individual has a spatial position, denoted by x, and a “home” location,\ndenoted by xh. An individual’s home remains constant, but the individual’s posi-\ntion changes according to a biased random walk over time. Speciﬁcally, we assume\nthat an individual is attracted to its home with a force proportional to its current\ndisplacement from that home, x − xh. Given an initial position y, the probability\ndensity p(x|y, xh, t) for the individual’s position at time t satisﬁes the partial dif-\nferential equation\n∂ 2 p\n∂ x2\n[(x − xh) p] ,\nwith delta-function initial condition\np(x|y, xh, 0) = δ(x − y),\nwhere D is the rate of diffusion, and α is the strength of attraction of the individuals\ntowards their homes. Eqs. (3) and (4) have solution\np(x|y, xh, t) =\n2π(1 − e−2αt )D/α\n−α [(x − xh) − (y − xh) e−αt ]2\n2D(1 − e−2αt )\n(5)\nAs time passes, the probability distribution of the individual’s position approaches\nthe stationary Gauss distribution\np(x|y, xh, t = ∞) =\n−α(x − xh)2\n2D\n(6)\nEvery individual is restricted to a neighborhood with variance D/α. This general\ndescription of the random motion of an elastically conﬁned particle is given by an\nOrnstein–Uhlenbeck process (Uhlenbeck and Ornstein, 1930; Goel and Richter-\nDyn, 1974).\nNow consider the introduction and transmission of an infectious disease. Let\nn(x, xh, t) be the density of infected individuals at position x with home xh at time t.\nIndividuals may behave differently depending on whether they are infected or sus-\nceptible. Therefore, let Di and αi describe the diffusion and attraction of infected\nindividuals, and let Ds and αs describe the diffusion and attraction of susceptible\nindividuals. Let transmission of infection occur only among individuals occupying\nthe same position x. If susceptible individuals have been dispersing for a long time\n( 1/αs) prior to the introduction of infection, the density of susceptible individu-\nals with homes xh, as given by Eq. (6), is\n(7)\nIt follows that the rate at which newly infected individuals with home xh appear at\nposition x at time t is\nn(x, xh, t) dxh,\nwhere r is the rate of transmission per infection. This is an approximation\nvalid for the earliest phase of an epidemic when the number of suscepti-\nbles is not signiﬁcantly reduced by infection. Infected individuals are removed\nfrom the disease dynamics by death, recovery, vaccination or quarantine at a\nrate µn.\nThe rate of change in the density n(x, xh, t) is the sum of movement under the\nOrnstein–Uhlenbeck process and new transmissions minus removal,\n∂ 2n\n∂ x2\n[(x − xh)n] + r\nn dxh − µn.\n(9)\nThe restricted-movement model for epidemic spread is posed explicitly by\nEq. (9).\nAlternatively, the transmission pattern of the restricted-movement model can\nbe posed as a convolution over the density of newly infected individuals b(y, t).\nEquation (5) provides the probability that an infected individual will be located\nat position x, conditioned on the location at the time of infection, home location,\nand time. The conditional dependence on home location can be removed using the\ndeﬁnition of conditional probability,\n ∞\np(x|y, xh, t) p(xh|y) dxh.\nFrom Eq.(7), we obtain the conditioned distribution of home locations\np(xh|y) = e−αs(y−xh)2/2Ds\n2π Ds/αs\n(11)\nThus, Eq. (10) is a convolution of Gauss distributions. After integration, we ﬁnd\np(x|y, t) = A(x − y, t), where A(x, t) is also a Gauss distribution in x with mean 0\nand time-dependent variance given by\n(1 − e−2αi t )\n+ (1 − e−αi t )2 Ds\nαs\n(12)\nThe density of newly infected individuals can now be constructed by generalizing\nthe Lotka integral equation (Kot, 2001) to include space. The rate at which all\nindividuals infected at location y at time t − τ produce new infections at location\ny at time t is proportional to the fraction of individuals occupying location y and\nremaining infected at time t. Integrating over all locations of infection and past\ntimes, the density of individuals b(y, t) newly infected at location y at time t will be\nb(y, t) = r\nA(y − y, τ )b(y, t − τ ) e−µτ dy dτ + G(y, t),\nwhere G(y, t) is an inhomogeneity corresponding to initial conditions.\n3. Analysis\nAlthough the restricted-movement model is linear, spatially-dependent coefﬁ-\ncients and non-local terms make Eq. (9) difﬁcult to analyze. Fortunately, the\nconvolution form of Eq. (13) is amenable to analysis using standard transform\nmethods (Metz et al., 1999). The minimum speed c∗ of solutions to the restricted-\nmovement model must satisfy the system\nˆb(c∗, ω∗) = 1,\n∂ ˆb\n∂ω (c∗, ω∗) = 0,\ne−ω(y+ct)b(y, t) dy dt\nafter converting the inﬁnite-domain integral into a ﬁnite-domain integral for nu-\nmerical computation. Heuristically, this approach is equivalent to assuming trav-\nelling wave solutions of the form b(y, t) = exp[−ω(y + ct)], deriving a dispersion\nrelation c(ω), and then using comparison theory (Fife, 1979) to argue that solu-\ntions originating from compact initial conditions will travel with asymptotic speed\nc∗ = minω c(ω).\nNumerical solutions of Eqs. (14) and (15) can describe the rate of spread under\ndifferent parameter regimes. For instance, Fig. 1 shows that the qualitative effect\nof removal on the speed of epidemic spread is independent of α. When µ  r , the\ndecrease in the asymptotic speed of spread is proportional to µ, and as removal\nrate µ approaches the transmission rate r , the speed vanishes with c∗ ∝\n1 − µ/r .\nThis relation can be derived analytically by using a two-term Taylor approximation\nfor the exponential in the integrand of Eq. (16) when ω is small. This yields\n1 − µ\nDs\nαs\nr\n \n\n1 + r\nαi\n+ Di\nαi\n2 + r\nαi\n+ r Di\nα2\n\ni\n(17)\nTo better understand the mechanisms involved, we will study the rate of spread in\nthe remaining three limiting cases depicted in Fig. 2. The heuristic arguments we\nprovide can also be made in a more formal manner by the careful study of Eq. (13).\n3.1. Distributed-contacts limit\nWhen the homeward attraction is much larger than the transmission rate (r  αi ),\nthe position of each infected individual is reasonably approximated by the time-\nindependent solution of the Ornstein–Uhlenbeck equation. Using a separation-of-\nvariables ansatz, we let\nn(x, xh, t) = e−αi (x−xh)2/2Di\n2π Di /αi\n˜n (xh, t) .\n(18)\nSubstituting Eq. (18) into Eq. (9) and integrating over all space x, we ﬁnd ˜n is\ngoverned by the integro-differential equation\n∂ ˜n(xh, t)\n∂t\n˜n(u, t) du − µ ˜n.\n(19)\nUsing the methods described in Daniels (1975) and Medlock and Kot (2003), the\nrate of spread c∗\n\nDC given compact initial conditions is\nθ eθ\n(1 − θ ) eθ/2 =\n.\nIn the special case of µ = 0,\n\nwhere e ≈ 2.7.\n3.2. Diffusive distributed-infectives limit\nWhen disease transmission is rapid (r  αi ), transmission often occurs before an\nindividual’s position has relaxed to equilibrium. If diffusion is much faster than\nhomeward attraction (Di αs/Ds  αi ), the movement of each individual is approx-\nimately independent of its home. If we apply this assumption and integrate over\nall homes, we ﬁnd\n∂ 2n\n∂ x2\n+ (r − µ)n, where n(x, t) =\nn(x, xh, t) dxh.\n(23)\nThis diffusion equation is a common description of the distributed-infectives\nmodel (Medlock and Kot, 2003). The asymptotic speed given compact initial con-\nditions is\n= 2\nDi (r − µ).\n3.3. Advective distributed-infectives limit\n(24)\nWhen disease transmission is rapid (r  αi ) but diffusion effects are weak\n(Di αs/Ds  αi ), advection governs dispersal. Because of the dominance of advec-\ntion, we refer to this as the advective distributed-infectives model. The calculation\nof an asymptotic spread rate c∗\nADI is more challenging in the advective distributed-\ninfectives case than in the previously studied cases. Dimensional analysis in the\ncase of Di = 0 shows\n= αi\nwhere f is an unknown function. One ad hoc solution method that seems to give\nreasonable estimates in the absence of removal (µ = 0) is to solve Eq. (14) us-\ning the ansatz ω∗ = 4c∗. This ansatz is the asymptotic condition for there to be a\nsingle-critical point of the integrand of Eq. (16) within the interval (0, 1), and is an\napproximate condition for minimization of Eq. (16) when ω is large. This leads to\nthe solution\n.\n(26)\nHowever, Fig. 3 suggests that this approximation is strictly greater than c∗ in the\nlimit of rapid transmission (r  αi ). The difﬁculty in computing the speed of the\nadvective distributed-infectives limit may be because the non-local term appears\nto make non-vanishing contributions to ˆb(c, ω) in regions of solution of Eqs. (14)\nand (15).\n4. Individual-based simulation\nHere we compare the results of our mathematical analysis to individual-based sim-\nulations of the transmission process. The simulation methods are described in the\nAppendix.\nThe median spatial position of the epidemic at a given time was approximated\nusing the empirical distribution of the furthest-forward observed individual from\nan ensemble of 1000 simulation runs. The speed is calculated by least-squares in-\nterpolation through the median epidemic positions (see Fig. 4). The simluation\ncode is available on request.\nObservations of spatial spread by individual-based simulation experiments gen-\nerally coincided with the minimum speeds calculated from Eqs. (14) and (15) (see\nFigs. 5 and 6). Observed speeds were around 90% of the minimum speeds c∗ by\nthe time the infected population reached 2 million individuals. This prediction\nagrees with the previous simulation results of Snyder (2003) and Kot et al. (2004).\nApproaches based on Eqs. (14) and (15) are far more computationally efﬁcient\nthan individual-based simulations, and can obtain solutions in a matter of seconds\nrather than hours.\nThere is a signiﬁcant discrepancy between the mathematical and simulation\nmethods in the advective distributed-infectives limit. Simulation experiments\ndemonstrated that speeds could be orders of magnitude slower than asymptotic\nprediction of the minimum wave speed c∗ (see Fig. 3). Observed speeds decayed\nat a rate proportional to αi . This did not accord with the the square-root depen-\ndence predicted by Eq. (26), which was derived using an ad hoc ansatz. Careful\ninspection of simulation results, however, suggests that epidemics were still accel-\nerating at the end of simulation runs. It was likely that fronts had not reached their\nasymptotic spread rates before time and memory constraints stopped simulations\nat 2 million infected individuals. Nonetheless, if the spread is still accelerating after\nmore than 2 million individuals have been infected, the asymptotic speed is prob-\nably not a practical upper bound in any realistic epidemiological scenario. Further\nresearch with particular emphasis on accelerating fronts is needed. When an ad-\nvective distributed-infectives limit is considered most appropriate, we currently\nrecommend that individual-based models be used, at least until a more efﬁcient\nalternative is developed.\n5. Discussion\nIn this paper we have formulated the restricted-movement model to describe spa-\ntial patterns of disease transmission. The restricted-movement model differs from",
         "",
         "",
         "downloading",
         "Abstract One of the central goals of mathematical epidemiology is to predict dis-\nease transmission patterns in populations. Two models are commonly used to pre-\ndict spatial spread of a disease. The ﬁrst is the distributed-contacts model, often\ndescribed by a contact distribution among stationary individuals. The second is\nthe distributed-infectives model, often described by the diffusion of infected indi-\nviduals. However, neither approach is ideal when individuals move within home\nranges. This paper presents a uniﬁed modeling hypothesis, called the restricted-\nmovement model. We use this model to predict spatial spread in settings where\ninfected individuals move within overlapping home ranges. Using mathematical\nand computational approaches, we show that our restricted-movement model has\nthree limits: the distributed-contacts model, the distributed-infectives model, and a\nthird, less studied advective distributed-infectives limit. We also calculate approx-\nimate upper bounds for the rates of an epidemic’s spatial spread. Guidelines are\nsuggested for determining which limit is most appropriate for a speciﬁc disease.\n1. Introduction\nA fundamental challenge in mathematical epidemiology is determining how the\nstructure of a population inﬂuences disease transmission. One important aspect is\nthe spatial structure. For instance, the SARS epidemic spread through 12 coun-\ntries within a few weeks. Projections of the spatial spread of an epidemic and the\ninteractions of human movement at multiple levels with a response protocol will\nfacilitate the assessment of policy alternatives. Spatially-explicit models are neces-\nsary to evaluate the efﬁcacy of movement controls (Riley et al., 2003; Eubank et al.,\n2004). Models that ignore spatial structure can lead to inaccuracy in the prediction\nof population dynamics (Durrett and Levin, 1994).\nA wide variety of methods have been used for the study of spatially struc-\ntured epidemics. Some examples include cellular automata (Doran and Laffan,\n2005; Fuks and Lawniczak, 2001), networks (Bauch and Galvani, 2003; Newman,\n2002), metapopulations (Lloyd and Jansen, 2004; Arino and van den Driessche,\n2003; Keeling and Gilligan, 2000), individual-based models (Bian, 2004), moment-\nclosure approximations (Thomson and Ellner, 2003; Filipe and Maule, 2003),\ninteracting particle systems (Durrett, 1999; Schinazi, 1996), diffusion equations\n(Caraco et al., 2002; M ´endez, 1998), integro-differential equations (Medlock and\nKot, 2003), and integrodifference equations (Allen and Ernest, 2002). Two partic-\nularly useful approaches for describing a disease’s rate of spatial spread are the\ndistributed-contacts and distributed-infectives models.\nKendall (1965) and Mollison (1972) developed the theory of asymptotic spread\nrates in distributed-contacts models. Their family of integral-equation models as-\nsumed that each individual is stationary and has a distribution of contacts over\nspace (Mollison, 1972). Distributed-contacts models are particularly appropriate\nfor the study of plant diseases (van den Bosch et al., 1988; Metz and van den Bosch,\n1995; van den Bosch et al., 1999), but researchers are also using the closely related\nframework of contact networks to study disease transmission in human popula-\ntions (Newman, 2002; Meyers et al., 2005; Read and Keeling, 2003).\nShortly after Mollison’s ﬁrst work, Noble (1974) formulated the closely related\ntheory of asymptotic spread rates under the distributed-infectives model. Building\non earlier work tracing back to Fisher and Kolmogorov, Noble applied diffusion\ntheory to the spread of bubonic plague in Europe (Noble, 1974). Noble’s model\nrelies on the assumptions that disease is transmitted through interactions between\ndispersing individuals, and that infected individuals move in uncorrelated random\nwalks. Medlock and Kot (2003) recently developed a distributed-infectives frame-\nwork that uses a ﬂexible kernel-based approach similar to that employed in earlier\ndistributed-contact models. They found that inappropriate application of either\nthe distributed-contact or distributed-infectives approaches can generate inaccu-\nrate projections of epidemic spread.\nIn many epidemiological contexts, the transmission process involves compo-\nnents of both distributed contacts and distributed infectives. Hybrid models have\nbeen proposed for this situation. For instance, the equation of Bailey (1975),\n\n\nn + DDC∇ 2n\n+ DDI∇ 2n − µn,\nallows for both diffusive contacts and diffusive movement. An analogous spatially\nnon-local model,\nkDI(x, y) [n(y, t) − n(x, t)] dy − µn,\nincorporates both a movement kernel kDI and a contact kernel kDC. Busenberg and\nTravis (1983) study a related model with diffusive movement of individuals and\ndistributed contacts. Recent study of a model where individuals transition between\nstationary and motile states (Hadeler, 2003) demonstrates that both distributed-\ninfectives and distributed-contacts models can appear as limiting cases of a single\nmodel.\nSpatially structured epidemic models are useful tools in the study of geographic\nepidemic spread. In particular, spatial models can be used to estimate the speed of\ngeographic spread. Estimates of rapidity of disease dissemination can, in turn, be\nused to guide policy decisions. For many linear models, such as the one discussed\nin this paper, researchers have shown that there is a minimum speed c∗ for trav-\neling wave solutions and that in many biologically realistic settings, solutions tend\nto approach advancing fronts that travel no faster than c∗. The speed of advancing\nfronts in non-linear deterministic and linear stochastic models has been shown in\na number of instances to be given by the minimum speed c∗ from a corresponding\nlinear model (Mollison, 1991; Kot et al., 2004). The connection between speeds\nof non-linear models with their corresponding linear models holding more gener-\nally is the subject of the “linear conjecture” (van den Bosch et al., 1990; Mollison,\n1991). Moreover, the minimum speed from a linear model often provides a good\nupper bound for the speeds observed in non-linear stochastic models (Mollison,\n1972; McKean, 1975; Mollison, 1977; Lewis, 2000; Lewis and Pacala, 2000; Clark\net al., 2001; Snyder, 2003).\nHere we consider a scenario that does not neatly conform to the assumptions\nof either the distributed-contacts or distributed-infectives hypotheses: the case\nof individuals moving within overlapping home ranges. Many animal species live\nwithin a “home range” and periodically return to their “home.” We develop a\nrestricted-movement model that describes movement relative to a “home” loca-\ntion, and study the rate of spatial spread of disease using mathematical analysis and\nstochastic simulations. We show that the restricted-movement model possesses\nlimits corresponding to the distributed-contact and distributed-infective models,\nbut also possesses a third limit, which we call the advective distributed-infectives\nmodel. Approximations to the rate of spread are provided in all three limits. These\napproximations agree with the speeds observed in individual-based Monte Carlo\nsimulations in all but the advective distributed-infectives limit.\n2. Restricted-movement model\nConsider a large population distributed uniformly over a one-dimensional world.\nEvery individual has a spatial position, denoted by x, and a “home” location,\ndenoted by xh. An individual’s home remains constant, but the individual’s posi-\ntion changes according to a biased random walk over time. Speciﬁcally, we assume\nthat an individual is attracted to its home with a force proportional to its current\ndisplacement from that home, x − xh. Given an initial position y, the probability\ndensity p(x|y, xh, t) for the individual’s position at time t satisﬁes the partial dif-\nferential equation\n∂ 2 p\n∂ x2\n[(x − xh) p] ,\nwith delta-function initial condition\np(x|y, xh, 0) = δ(x − y),\nwhere D is the rate of diffusion, and α is the strength of attraction of the individuals\ntowards their homes. Eqs. (3) and (4) have solution\np(x|y, xh, t) =\n2π(1 − e−2αt )D/α\n−α [(x − xh) − (y − xh) e−αt ]2\n2D(1 − e−2αt )\n(5)\nAs time passes, the probability distribution of the individual’s position approaches\nthe stationary Gauss distribution\np(x|y, xh, t = ∞) =\n−α(x − xh)2\n2D\n(6)\nEvery individual is restricted to a neighborhood with variance D/α. This general\ndescription of the random motion of an elastically conﬁned particle is given by an\nOrnstein–Uhlenbeck process (Uhlenbeck and Ornstein, 1930; Goel and Richter-\nDyn, 1974).\nNow consider the introduction and transmission of an infectious disease. Let\nn(x, xh, t) be the density of infected individuals at position x with home xh at time t.\nIndividuals may behave differently depending on whether they are infected or sus-\nceptible. Therefore, let Di and αi describe the diffusion and attraction of infected\nindividuals, and let Ds and αs describe the diffusion and attraction of susceptible\nindividuals. Let transmission of infection occur only among individuals occupying\nthe same position x. If susceptible individuals have been dispersing for a long time\n( 1/αs) prior to the introduction of infection, the density of susceptible individu-\nals with homes xh, as given by Eq. (6), is\n(7)\nIt follows that the rate at which newly infected individuals with home xh appear at\nposition x at time t is\nn(x, xh, t) dxh,\nwhere r is the rate of transmission per infection. This is an approximation\nvalid for the earliest phase of an epidemic when the number of suscepti-\nbles is not signiﬁcantly reduced by infection. Infected individuals are removed\nfrom the disease dynamics by death, recovery, vaccination or quarantine at a\nrate µn.\nThe rate of change in the density n(x, xh, t) is the sum of movement under the\nOrnstein–Uhlenbeck process and new transmissions minus removal,\n∂ 2n\n∂ x2\n[(x − xh)n] + r\nn dxh − µn.\n(9)\nThe restricted-movement model for epidemic spread is posed explicitly by\nEq. (9).\nAlternatively, the transmission pattern of the restricted-movement model can\nbe posed as a convolution over the density of newly infected individuals b(y, t).\nEquation (5) provides the probability that an infected individual will be located\nat position x, conditioned on the location at the time of infection, home location,\nand time. The conditional dependence on home location can be removed using the\ndeﬁnition of conditional probability,\n ∞\np(x|y, xh, t) p(xh|y) dxh.\nFrom Eq.(7), we obtain the conditioned distribution of home locations\np(xh|y) = e−αs(y−xh)2/2Ds\n2π Ds/αs\n(11)\nThus, Eq. (10) is a convolution of Gauss distributions. After integration, we ﬁnd\np(x|y, t) = A(x − y, t), where A(x, t) is also a Gauss distribution in x with mean 0\nand time-dependent variance given by\n(1 − e−2αi t )\n+ (1 − e−αi t )2 Ds\nαs\n(12)\nThe density of newly infected individuals can now be constructed by generalizing\nthe Lotka integral equation (Kot, 2001) to include space. The rate at which all\nindividuals infected at location y at time t − τ produce new infections at location\ny at time t is proportional to the fraction of individuals occupying location y and\nremaining infected at time t. Integrating over all locations of infection and past\ntimes, the density of individuals b(y, t) newly infected at location y at time t will be\nb(y, t) = r\nA(y − y, τ )b(y, t − τ ) e−µτ dy dτ + G(y, t),\nwhere G(y, t) is an inhomogeneity corresponding to initial conditions.\n3. Analysis\nAlthough the restricted-movement model is linear, spatially-dependent coefﬁ-\ncients and non-local terms make Eq. (9) difﬁcult to analyze. Fortunately, the\nconvolution form of Eq. (13) is amenable to analysis using standard transform\nmethods (Metz et al., 1999). The minimum speed c∗ of solutions to the restricted-\nmovement model must satisfy the system\nˆb(c∗, ω∗) = 1,\n∂ ˆb\n∂ω (c∗, ω∗) = 0,\ne−ω(y+ct)b(y, t) dy dt\nafter converting the inﬁnite-domain integral into a ﬁnite-domain integral for nu-\nmerical computation. Heuristically, this approach is equivalent to assuming trav-\nelling wave solutions of the form b(y, t) = exp[−ω(y + ct)], deriving a dispersion\nrelation c(ω), and then using comparison theory (Fife, 1979) to argue that solu-\ntions originating from compact initial conditions will travel with asymptotic speed\nc∗ = minω c(ω).\nNumerical solutions of Eqs. (14) and (15) can describe the rate of spread under\ndifferent parameter regimes. For instance, Fig. 1 shows that the qualitative effect\nof removal on the speed of epidemic spread is independent of α. When µ  r , the\ndecrease in the asymptotic speed of spread is proportional to µ, and as removal\nrate µ approaches the transmission rate r , the speed vanishes with c∗ ∝\n1 − µ/r .\nThis relation can be derived analytically by using a two-term Taylor approximation\nfor the exponential in the integrand of Eq. (16) when ω is small. This yields\n1 − µ\nDs\nαs\nr\n \n\n1 + r\nαi\n+ Di\nαi\n2 + r\nαi\n+ r Di\nα2\n\ni\n(17)\nTo better understand the mechanisms involved, we will study the rate of spread in\nthe remaining three limiting cases depicted in Fig. 2. The heuristic arguments we\nprovide can also be made in a more formal manner by the careful study of Eq. (13).\n3.1. Distributed-contacts limit\nWhen the homeward attraction is much larger than the transmission rate (r  αi ),\nthe position of each infected individual is reasonably approximated by the time-\nindependent solution of the Ornstein–Uhlenbeck equation. Using a separation-of-\nvariables ansatz, we let\nn(x, xh, t) = e−αi (x−xh)2/2Di\n2π Di /αi\n˜n (xh, t) .\n(18)\nSubstituting Eq. (18) into Eq. (9) and integrating over all space x, we ﬁnd ˜n is\ngoverned by the integro-differential equation\n∂ ˜n(xh, t)\n∂t\n˜n(u, t) du − µ ˜n.\n(19)\nUsing the methods described in Daniels (1975) and Medlock and Kot (2003), the\nrate of spread c∗\n\nDC given compact initial conditions is\nθ eθ\n(1 − θ ) eθ/2 =\n.\nIn the special case of µ = 0,\n\nwhere e ≈ 2.7.\n3.2. Diffusive distributed-infectives limit\nWhen disease transmission is rapid (r  αi ), transmission often occurs before an\nindividual’s position has relaxed to equilibrium. If diffusion is much faster than\nhomeward attraction (Di αs/Ds  αi ), the movement of each individual is approx-\nimately independent of its home. If we apply this assumption and integrate over\nall homes, we ﬁnd\n∂ 2n\n∂ x2\n+ (r − µ)n, where n(x, t) =\nn(x, xh, t) dxh.\n(23)\nThis diffusion equation is a common description of the distributed-infectives\nmodel (Medlock and Kot, 2003). The asymptotic speed given compact initial con-\nditions is\n= 2\nDi (r − µ).\n3.3. Advective distributed-infectives limit\n(24)\nWhen disease transmission is rapid (r  αi ) but diffusion effects are weak\n(Di αs/Ds  αi ), advection governs dispersal. Because of the dominance of advec-\ntion, we refer to this as the advective distributed-infectives model. The calculation\nof an asymptotic spread rate c∗\nADI is more challenging in the advective distributed-\ninfectives case than in the previously studied cases. Dimensional analysis in the\ncase of Di = 0 shows\n= αi\nwhere f is an unknown function. One ad hoc solution method that seems to give\nreasonable estimates in the absence of removal (µ = 0) is to solve Eq. (14) us-\ning the ansatz ω∗ = 4c∗. This ansatz is the asymptotic condition for there to be a\nsingle-critical point of the integrand of Eq. (16) within the interval (0, 1), and is an\napproximate condition for minimization of Eq. (16) when ω is large. This leads to\nthe solution\n.\n(26)\nHowever, Fig. 3 suggests that this approximation is strictly greater than c∗ in the\nlimit of rapid transmission (r  αi ). The difﬁculty in computing the speed of the\nadvective distributed-infectives limit may be because the non-local term appears\nto make non-vanishing contributions to ˆb(c, ω) in regions of solution of Eqs. (14)\nand (15).\n4. Individual-based simulation\nHere we compare the results of our mathematical analysis to individual-based sim-\nulations of the transmission process. The simulation methods are described in the\nAppendix.\nThe median spatial position of the epidemic at a given time was approximated\nusing the empirical distribution of the furthest-forward observed individual from\nan ensemble of 1000 simulation runs. The speed is calculated by least-squares in-\nterpolation through the median epidemic positions (see Fig. 4). The simluation\ncode is available on request.\nObservations of spatial spread by individual-based simulation experiments gen-\nerally coincided with the minimum speeds calculated from Eqs. (14) and (15) (see\nFigs. 5 and 6). Observed speeds were around 90% of the minimum speeds c∗ by\nthe time the infected population reached 2 million individuals. This prediction\nagrees with the previous simulation results of Snyder (2003) and Kot et al. (2004).\nApproaches based on Eqs. (14) and (15) are far more computationally efﬁcient\nthan individual-based simulations, and can obtain solutions in a matter of seconds\nrather than hours.\nThere is a signiﬁcant discrepancy between the mathematical and simulation\nmethods in the advective distributed-infectives limit. Simulation experiments\ndemonstrated that speeds could be orders of magnitude slower than asymptotic\nprediction of the minimum wave speed c∗ (see Fig. 3). Observed speeds decayed\nat a rate proportional to αi . This did not accord with the the square-root depen-\ndence predicted by Eq. (26), which was derived using an ad hoc ansatz. Careful\ninspection of simulation results, however, suggests that epidemics were still accel-\nerating at the end of simulation runs. It was likely that fronts had not reached their\nasymptotic spread rates before time and memory constraints stopped simulations\nat 2 million infected individuals. Nonetheless, if the spread is still accelerating after\nmore than 2 million individuals have been infected, the asymptotic speed is prob-\nably not a practical upper bound in any realistic epidemiological scenario. Further\nresearch with particular emphasis on accelerating fronts is needed. When an ad-\nvective distributed-infectives limit is considered most appropriate, we currently\nrecommend that individual-based models be used, at least until a more efﬁcient\nalternative is developed.\n5. Discussion\nIn this paper we have formulated the restricted-movement model to describe spa-\ntial patterns of disease transmission. The restricted-movement model differs from\nexisting models by describing individual movement relative to a “home” location\nto which individuals return sporadically. The restricted-movement model also uni-\nﬁes the distributed-infectives and distributed-contacts models in a common frame-\nwork and further illuminates their relationships. When infected individuals re-\nturn home rapidly, speeds in the restricted-movement model coincide with those\nof the distributed-contacts model. Conversely, when infected individuals return\nhome very slowly relative to both diffusion and transmission, rates of spread in\nthe restricted-movement model coincide with those of the diffusive distributed-\ninfectives model. In the distributed-contacts and diffusive distributed-infectives\nlimits, the asymptotic rates of spread can be calculated from the simple expres-\nsions we have formulated.\nAn imprecise approximation in the advective distributed-infectives limit is also\ngiven, but this limit requires further study. In the advective limit, the dispersal\nprocess is closely related to the “velocity jump process” of Othmer et al. (1988).\nThe velocity jump process is motivated by the “run and tumble movement” of\nﬂagellated bacteria such as E. coli. This process describes the movement of an or-\nganism that alternately “runs” at a constant velocity and then “tumbles” to a new\nvelocity. Alt (1980) derives a special case of the velocity jump process from bio-\nlogical assumptions and studies a diffusion approximation. Othmer et al. (1988)\nderives the velocity jump process from a generalization of the telegrapher’s equa-\ntion and then analyzes some special cases. Schwetlick (2000) shows the existence\nof traveling-wave solutions for the velocity jump process when the velocities of\nindividuals are bounded and gives an expression for the minimum wave speed.\nHillen and Othmer (2000) and Othmer and Hillen (2002) give a detailed study of\ndiffusion approximations for the velocity jump process.\nIndividual-based models of social networks (Eubank et al., 2004) contain both\nstatic components, such as family contacts, and transient components, such as peo-\nple sharing a crowded subway car. Under the restricted-movement model, tran-\nsient components will govern the rate of spatial epidemic spread when trans-\nmission frequently occurs before the movement of an individual has relaxed to\nits equilibrium distributions. When many transmissions occur during relaxation,\ndistributed-infectives models are most appropriate for describing spread. If trans-\nmissions seldom occur before relaxation, the distributed-contacts approach is the\nmost appropriate. The boundary between these two limits can be approximated\nby equating Eqs. (24) and (20). In the speciﬁc case where removal is slow (µ ≈ 0),\ninfection does not alter individual behavior (Dsαi = Di αs), and individuals return\nhome every night (αi = 1), the distributed-infectives model must be considered\nwhen there are more than 0.7 transmissions per day per individual. For example,\nsmallpox and measles are transmitted more than once a day on average, and are\nlikely to require distributed-infectives models, while scarlet fever is transmitted\nless than 0.5 times a day, suggesting a distributed-contacts model may be adequate\n(Anderson and May, 1991; Fenner et al., 1988).\nOur results for the restricted-movement model are robust to a greater or lesser\nextent when the underlying assumptions are relaxed. For instance, the general-\nization from 1 to 2 spatial dimensions may prolong the transient dynamics, but is\nnot expected to effect our asymptotic results. The mean-ﬁeld description of dis-\nease transmission implicitly assumes a large, dense population. Disease spread is\nlikely to be slower in a sparse population with fewer contacts between suscepti-\nble and infected individuals. The accumulation of immune individuals may further\nslow the spread of disease. We have also assumed that transmission can only oc-\ncur through close contact between individuals. We have not explored scenarios\nwhere transmission is facilitated exogenously by vectors or other environmental\nfactors.\nThe Ornstein–Uhlenbeck process is a convenient assumption which is reason-\nable for many animal species. Our qualitative results should also hold for other\nempirically derived movement models. Consider alternative cases where infected\nindividuals advect away from their point of infection for a ﬁxed time, after which\nthey are stationary for the duration of infection. In such cases, we can expect epi-\ndemic spread to correspond to an advective distributed-infectives model for fast\ntransmission, but a distributed-contacts model for slow transmission.\nGreater biological realism can be achieved from the restricted-movement model\nby relaxing the assumptions of temporal, spatial, and demographic homogeneity.\nTemporal homogeneity can be relaxed to allow for daily and weekly movement\ncycles or aperiodic transients. Spatial homogeneity and isotropy can be loosened,\nparticularly in two dimensions, to allow for spatial variation in population den-\nsity and individual movement patterns. The current model captures some demo-\ngraphic heterogeneity by allowing infected and susceptible individuals to behave\ndifferently, but these classes themselves are assumed to be homogeneous. De-\nmographic heterogeneity may be further explored by allowing population age\nstructure and correlated movement of individuals. Each of these generalizations\ngreatly increases the amount of biological information that can be incorporated\ninto the restricted-movement model.\nOur methods may also be of use in ecological studies of animal populations\nwhere individuals can exhibit behavioral biases in their movement. Density-\ndependence can be included by adding non-linear growth and mortality terms to\nEq. (9). Such models provide an extension of diffusion models and might provide\nuseful analytic results.\nIn conclusion, the restricted-movement model provides an improved approach\nto spatial epidemic modeling, and elucidates the relationship between popular\ndistributed-contacts and distributed-infectives approaches. Along with the issues\ndiscussed above, future work may include applications of this theory to speciﬁc\nepidemic control problems.",
         "",
         "A Model of Spatial Epidemic Spread When Individuals Move Within Overlapping Home Ranges",
         "None"
        ],
        [
         "22",
         "00122b0f976ecd6b26a8eece8c1d6a89c7cea69b",
         "In type 1 diabetes mellitus (T1DM), T cell-mediated destruction of insulin-producing pancreatic β cells leads to the acute onset of hyperglycemia. The nonobese diabetic mouse model of human T1DM reveals that T cells capable of inducing diabetes can escape normal central tolerance, and can cause T1DM if left unchecked. However, several regulatory T cell subsets can temper autoaggressive T cells, although it remains undetermined when and how, and by which subset, homeostatic control of diabetogenic T cells is normally achieved in vivo. Using a cotransfer model, we find that NKT cells efficiently dampen the action of diabetogenic CD4+ T cells, and do so in an indirect manner by modifying the host environment. Moreover, the NKT cell-containing population modifies the host via production of IFN-γ that is necessary for driving the inhibition of diabetogenic T cells in vivo.",
         "J. A. Cain,Judith A. Smith,Jennifer K. Ondr,Bo Wang,J. Katz",
         "None",
         "https://www.jimmunol.org/content/jimmunol/176/3/1645.full.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1217413%26volume%3d176%26issue%3d3%26page%3d1645%26filename%3d1645.pdf%26languageId%3d",
         "None",
         "",
         "NKT Cells and IFN-γ Establish the Regulatory Environment for the Control of Diabetogenic T Cells in the Nonobese Diabetic Mouse1",
         "None"
        ],
        [
         "23",
         "00123890feb5ddd7d2a734aae13f7927aa9e8477",
         "Autosomal-dominant polycystic kidney disease (ADPKD) is a common genetic disease caused by mutations in the PKD1 gene. This adult-onset disease results in the accumulation of destructive kidney cysts, leading to progressive loss of renal function and eventually to renal failure. A need for kidney transplantation and dialysis are common outcomes. There are currently no treatment options to prevent or delay the disease onset. Although the PKD1 gene was identified more than a decade ago, the development of treatment strategies has been hampered by a lack of understanding of the function of polycystin-1 (PC1), the protein encoded by the PKD1 gene (1, 2). Work by Shillingford et al. (3) in this issue of PNAS now identifies a new function of PC1, which immediately suggests a possibility for future treatment options.\n\nPC1 is a very large integral membrane protein with a much smaller C-terminal cytoplasmic tail. Previous work had implicated this tail in numerous signaling events, but their relevance to ADPKD had remained largely unclear. Shillingford et al. (3) now report that the PC1 tail interacts with a very interesting player by the name of tuberin, the product of the TSC2 gene. Tuberin mutations lead to the complex disease tuberous sclerosis (TSC), which is ≈10 times less common than ADPKD. Three earlier observations had already suggested that tuberin and PC1 could be functionally linked. First, in addition to benign tumors in multiple organs, TSC patients also exhibit kidney cysts. Second, the TSC2 gene is located only a handful of base pairs away from the PKD1 gene. A subset of patients has larger chromosomal deletions that affect both the TSC2 and the PKD1 genes at the same time. These patients suffer from very severe, early onset polycystic kidney disease. Third, previous results using tuberin null cells had suggested that tuberin may play … \n\n*Correspondence should be addressed: Department of Anatomy, University of California School of Medicine, Genentech Hall, Room N212B, Box 2140, 600 16th Street, San Francisco, CA 94107. E-mail: \n\nmostov{at}itsa.ucsf.edu",
         "K. Mostov",
         "K. Mostov",
         "https://europepmc.org/articles/pmc1459339?pdf=render",
         "kidney disease (ADPKD) is a\ncommon genetic disease caused\nby mutations in the PKD1 gene.\nThis adult-onset disease results in the ac-\ncumulation of destructive kidney cysts,\nleading to progressive loss of renal func-\ntion and eventually to renal failure. A\nneed for kidney transplantation and dialy-\nsis are common outcomes. There are cur-\nrently no treatment options to prevent or\ndelay the disease onset. Although the\nPKD1 gene was identified more than a\ndecade ago, the development of treatment\nstrategies has been hampered by a lack of\nunderstanding of the function of polycys-\ntin-1 (PC1), the protein encoded by the\nPKD1 gene (1, 2). Work by Shillingford\net al. (3) in this issue of PNAS now identi-\nfies a new function of PC1, which im-\nmediately suggests a possibility for future\ntreatment options.",
         "",
         "protein with a much smaller C-terminal\ncytoplasmic tail. Previous work had impli-\ncated this tail in numerous signaling\nevents, but their relevance to ADPKD\nhad remained largely unclear. Shillingford\net al. (3) now report that the PC1 tail in-\nteracts with a very interesting player by\nthe name of tuberin, the product of the\nTSC2 gene. Tuberin mutations lead to the\ncomplex disease tuberous sclerosis (TSC),\nwhich is 10 times less common than\nADPKD. Three earlier observations had\nalready suggested that tuberin and PC1\ncould be functionally linked. First, in addi-\ntion to benign tumors in multiple organs,\nTSC patients also exhibit kidney cysts.\nSecond, the TSC2 gene is located only a\nhandful of base pairs away from the PKD1\ngene. A subset of patients has larger chro-\nmosomal deletions that affect both the\nTSC2 and the PKD1 genes at the same\ntime. These patients suffer from very se-\nvere, early onset polycystic kidney disease.\nThird, previous results using tuberin null\ncells had suggested that tuberin may play\na role in intracellular trafficking of PC1\n(4). These observations led Shillingford\net al. (3) to investigate the possibility that\ntuberin may interact with the PC1 tail.\nThis possibility could indeed be shown by\nforced colocalization experiments in vivo\nand binding experiments in vitro. To eluci-\ndate functional consequences of this inter-\naction, the authors capitalized on a recent\nwealth of information on the function of\ntuberin. Its main function appears to be to\ninactivate the SerThr kinase mTOR (5,\n6). mTOR, in turn, promotes translation\nvia phosphorylation of two proteins,\nS6-kinase and 4E-BP1 (Fig. 1). mTOR\nactivity has been linked to increased cell\ngrowth, proliferation, apoptosis, and\nchanges in differentiation (7).\nReasoning that the interaction between\nPC1 and tuberin may have functional con-\nsequences for mTOR activity, Shillingford\net al. (3) tested whether mTOR activity is\naltered in ADPKD or a PC1-inactivated\nmouse model. Indeed, in both cases cyst-\nlining epithelial cells exhibited very high\nmTOR activity. Furthermore, forced colo-\ncalization experiments suggested that\nmTOR may also be part of the PC1–\ntuberin complex. These results suggest\nthat PC1 normally suppresses mTOR\nactivity and that defects in PC1 conse-\nquently lead to aberrant mTOR activa-\ntion. A working model consistent with\nthese findings is shown in Fig. 1.\nInterestingly, the authors also found\nexcessive mTOR activity in kidney cysts of\nmouse models with defects in proteins\nother than PC1. A host of proteins has\nrecently emerged that, when mutated, re-\nsults in renal cystic diseases in humans or\nanimal models. Although the function of\nnone of these proteins is clear, they all\nhave something in common: they all local-\nize to primary cilia of renal epithelial cells\nor to the basal bodies from which cilia\nemanate. This finding has led to the\nwidely held view that loss of cilia function\nleads to cyst formation in the kidney\n(8, 9). But what is the function of cilia in\nthe kidney? These nonmotile organelles\nproject from the apical plasma membrane\nof epithelial cells into the lumen of the\nrenal tubules. After many decades of\naccusing primary cilia of being vestigial\nstructures, at least one function has re-\ncently emerged when they were shown to\nact as mechanosensors of intralumenal\nfluid flow: bending of cilia results in the\nelevation of cytoplasmic calcium (10).\nPC1-null cells have been found to be de-\nfective in this flow response (11), which\nsuggests that (i) PC1 is required for mech-\nanotransduction and (ii) loss of ciliary\nmechanotransduction somehow leads to\nthe growth of renal cysts. The finding by\nShillingford et al. (3) that mTOR is inap-\npropriately activated not only in PC1-\ndefective cysts but also in polycystic\nmouse models with defects in different\nproteins (namely, polaris and MAL) sug-\ngests that mTOR activation may be a\ncommon consequence of the loss of cilia\nfunction. If mTOR is indeed such a con-\nverging point, then it would be of great\ninterest as a possible drug target for treat-\nment of renal cystic diseases.\nmTOR (for ‘‘mammalian target of\nrapamycin’’) is a protein that is named\nafter the drug that inhibits it. Rapamycin\nis a compound originally discovered in the\n1970s in soil from Easter Island. It very\nspecifically and effectively inhibits mTOR\n(12). Rapamycin is clinically approved as\nan immunosuppressant and is mostly used\nin kidney transplant patients. When Shil-\nlingford et al. (3) treated two different\npolycystic mouse models with rapamycin,\nthe results were stunning. In a mild, late-\nonset mouse model, rapamycin treatment\nfor 1 month not only stopped kidney\ngrowth but resulted in a regression of\nkidney size. This result was shown to be\nlikely because of the induction of pro-\ngrammed cell death (apoptosis) specifi-\ncally in cyst lining epithelial cells. In\ncontrast, rapamycin seemed to have no\neffect in the kidneys of normal mice.\nTreatment of an aggressive, early onset\nmouse model with rapamycin for 2 weeks\nresulted in a dramatic reduction of kidney\nsize and prevented the loss of kidney\nfunction. Because inhibition of mTOR\nalone had these dramatic beneficial ef-\nfects, this result suggests that the inappro-\npriate activation of mTOR is of central\nimportance for the growth of renal cysts\nand further supports the idea that mTOR\nlies at a converging point of signaling\npathways that lead to cyst formation.\nDrugs that work well in animal models\noften fail to be effective in human trials.\nTo obtain preliminary information on the\npossible effectiveness of rapamycin in hu-\nman ADPKD patients, Shillingford et al.\n(3) made use of the facts that this drug is\nclinically approved to immunosuppress\nkidney transplant patients and that\nADPKD patients frequently undergo kid-\nney transplantation. Typically, ADPKD\npatients receive the transplant in addition\nto their remaining native kidneys. These\npatients therefore have three kidneys, two\nof which are polycystic. Some of these\npatients are being treated with rapamycin.\nIn their retrospective study, Shillingford\net al. (3) identified a group of such pa-\ntients who also had computed tomography\n(CT) scans performed at the beginning of\nthe treatment and 2 years later. In this\nrapamycin group, the kidney volumes de-\ncreased by 25%, whereas there was no\neffect in a nonrapamycin control group.\nAlthough the patient numbers are proba-\nbly too small for this result to be definite,\nthe data are highly encouraging because\nthey point in the right direction.\nOverall, the functional link between\nmTOR and PC1 and the effects of rapa-\nmycin on animal models and patients sug-\ngest that rapamycin may be a promising\ndrug with the potential to become the\nfirst available treatment for ADPKD\npatients. The fact that this drug is already\nclinically approved should greatly facilitate\nclinical trials.\nBesides these exciting clinical implica-\ntions, this work also raises many impor-\ntant basic science questions. Which\nother regulatory proteins (such as Rheb\nand hamartin; see Fig. 1) are involved in\nthe PC1–tuberin–mTOR complex? Is\nthere a cilia connection? If so, is mTOR\nactivity regulated by fluid flow and cilia\nbending? Very recent work by the same\nlaboratory identified another novel\npathway involving the PC1 tail (13).\nThese results suggested that another\nfunction of PC1 is to sequester the tran-\nscription factor signal transducers and\nactivators of transcription 6 (STAT6) in\ncilia under normal conditions. In the\nabsence of fluid flow through the kid-\nney, however, the tail of PC1 is cleaved\nby a protease and translocates together\nwith STAT6 to the nucleus, where it\nactivates gene expression (13). This\nnovel ciliary mechanotransduction path-\nway may be the long-sought-after mech-\nanism of cilia function in the kidney, but\nhow does it relate to the activation of\nmTOR? It will be exciting to find\nanswers to these questions.\n3. Shillingford, J. M., Murcia, N. S., Larson, C. H., Low,\nS. H., Hedgepeth, R., Brown, N., Flask, C. A., Novick,\nA. C., Goldfarb, D. A., Kramer-Zucker, A., et al.\n(2006) Proc. Natl. Acad. Sci. USA 103, 5466–5471.\n4. Kleymenova, E., Ibraghimov-Beskrovnaya, O., Kugoh,\nH., Everitt, J., Xu, H., Kiguchi, K., Landes, G., Harris,\nP. & Walker, C. (2001) Mol. Cell 7, 823–832.",
         "",
         "",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1217413%26volume%3d176%26issue%3d3%26page%3d1645%26filename%3d1645.pdf%26languageId%3d",
         "kidney disease (ADPKD) is a\ncommon genetic disease caused\nby mutations in the PKD1 gene.\nThis adult-onset disease results in the ac-\ncumulation of destructive kidney cysts,\nleading to progressive loss of renal func-\ntion and eventually to renal failure. A\nneed for kidney transplantation and dialy-\nsis are common outcomes. There are cur-\nrently no treatment options to prevent or\ndelay the disease onset. Although the\nPKD1 gene was identified more than a\ndecade ago, the development of treatment\nstrategies has been hampered by a lack of\nunderstanding of the function of polycys-\ntin-1 (PC1), the protein encoded by the\nPKD1 gene (1, 2). Work by Shillingford\net al. (3) in this issue of PNAS now identi-\nfies a new function of PC1, which im-\nmediately suggests a possibility for future\ntreatment options.\nprotein with a much smaller C-terminal\ncytoplasmic tail. Previous work had impli-\ncated this tail in numerous signaling\nevents, but their relevance to ADPKD\nhad remained largely unclear. Shillingford\net al. (3) now report that the PC1 tail in-\nteracts with a very interesting player by\nthe name of tuberin, the product of the\nTSC2 gene. Tuberin mutations lead to the\ncomplex disease tuberous sclerosis (TSC),\nwhich is 10 times less common than\nADPKD. Three earlier observations had\nalready suggested that tuberin and PC1\ncould be functionally linked. First, in addi-\ntion to benign tumors in multiple organs,\nTSC patients also exhibit kidney cysts.\nSecond, the TSC2 gene is located only a\nhandful of base pairs away from the PKD1\ngene. A subset of patients has larger chro-\nmosomal deletions that affect both the\nTSC2 and the PKD1 genes at the same\ntime. These patients suffer from very se-\nvere, early onset polycystic kidney disease.\nThird, previous results using tuberin null\ncells had suggested that tuberin may play\na role in intracellular trafficking of PC1\n(4). These observations led Shillingford\net al. (3) to investigate the possibility that\ntuberin may interact with the PC1 tail.\nThis possibility could indeed be shown by\nforced colocalization experiments in vivo\nand binding experiments in vitro. To eluci-\ndate functional consequences of this inter-\naction, the authors capitalized on a recent\nwealth of information on the function of\ntuberin. Its main function appears to be to\ninactivate the SerThr kinase mTOR (5,\n6). mTOR, in turn, promotes translation\nvia phosphorylation of two proteins,\nS6-kinase and 4E-BP1 (Fig. 1). mTOR\nactivity has been linked to increased cell\ngrowth, proliferation, apoptosis, and\nchanges in differentiation (7).\nReasoning that the interaction between\nPC1 and tuberin may have functional con-\nsequences for mTOR activity, Shillingford\net al. (3) tested whether mTOR activity is\naltered in ADPKD or a PC1-inactivated\nmouse model. Indeed, in both cases cyst-\nlining epithelial cells exhibited very high\nmTOR activity. Furthermore, forced colo-\ncalization experiments suggested that\nmTOR may also be part of the PC1–\ntuberin complex. These results suggest\nthat PC1 normally suppresses mTOR\nactivity and that defects in PC1 conse-\nquently lead to aberrant mTOR activa-\ntion. A working model consistent with\nthese findings is shown in Fig. 1.\nInterestingly, the authors also found\nexcessive mTOR activity in kidney cysts of\nmouse models with defects in proteins\nother than PC1. A host of proteins has\nrecently emerged that, when mutated, re-\nsults in renal cystic diseases in humans or\nanimal models. Although the function of\nnone of these proteins is clear, they all\nhave something in common: they all local-\nize to primary cilia of renal epithelial cells\nor to the basal bodies from which cilia\nemanate. This finding has led to the\nwidely held view that loss of cilia function\nleads to cyst formation in the kidney\n(8, 9). But what is the function of cilia in\nthe kidney? These nonmotile organelles\nproject from the apical plasma membrane\nof epithelial cells into the lumen of the\nrenal tubules. After many decades of\naccusing primary cilia of being vestigial\nstructures, at least one function has re-\ncently emerged when they were shown to\nact as mechanosensors of intralumenal\nfluid flow: bending of cilia results in the\nelevation of cytoplasmic calcium (10).\nPC1-null cells have been found to be de-\nfective in this flow response (11), which\nsuggests that (i) PC1 is required for mech-\nanotransduction and (ii) loss of ciliary\nmechanotransduction somehow leads to\nthe growth of renal cysts. The finding by\nShillingford et al. (3) that mTOR is inap-\npropriately activated not only in PC1-\ndefective cysts but also in polycystic\nmouse models with defects in different\nproteins (namely, polaris and MAL) sug-\ngests that mTOR activation may be a\ncommon consequence of the loss of cilia\nfunction. If mTOR is indeed such a con-\nverging point, then it would be of great\ninterest as a possible drug target for treat-\nment of renal cystic diseases.\nmTOR (for ‘‘mammalian target of\nrapamycin’’) is a protein that is named\nafter the drug that inhibits it. Rapamycin\nis a compound originally discovered in the\n1970s in soil from Easter Island. It very\nspecifically and effectively inhibits mTOR\n(12). Rapamycin is clinically approved as\nan immunosuppressant and is mostly used\nin kidney transplant patients. When Shil-\nlingford et al. (3) treated two different\npolycystic mouse models with rapamycin,\nthe results were stunning. In a mild, late-\nonset mouse model, rapamycin treatment\nfor 1 month not only stopped kidney\ngrowth but resulted in a regression of\nkidney size. This result was shown to be\nlikely because of the induction of pro-\ngrammed cell death (apoptosis) specifi-\ncally in cyst lining epithelial cells. In\ncontrast, rapamycin seemed to have no\neffect in the kidneys of normal mice.\nTreatment of an aggressive, early onset\nmouse model with rapamycin for 2 weeks\nresulted in a dramatic reduction of kidney\nsize and prevented the loss of kidney\nfunction. Because inhibition of mTOR\nalone had these dramatic beneficial ef-\nfects, this result suggests that the inappro-\npriate activation of mTOR is of central\nimportance for the growth of renal cysts\nand further supports the idea that mTOR\nlies at a converging point of signaling\npathways that lead to cyst formation.\nDrugs that work well in animal models\noften fail to be effective in human trials.\nTo obtain preliminary information on the\npossible effectiveness of rapamycin in hu-\nman ADPKD patients, Shillingford et al.\n(3) made use of the facts that this drug is\nclinically approved to immunosuppress\nkidney transplant patients and that\nADPKD patients frequently undergo kid-\nney transplantation. Typically, ADPKD\npatients receive the transplant in addition\nto their remaining native kidneys. These\npatients therefore have three kidneys, two\nof which are polycystic. Some of these\npatients are being treated with rapamycin.\nIn their retrospective study, Shillingford\net al. (3) identified a group of such pa-\ntients who also had computed tomography\n(CT) scans performed at the beginning of\nthe treatment and 2 years later. In this\nrapamycin group, the kidney volumes de-\ncreased by 25%, whereas there was no\neffect in a nonrapamycin control group.\nAlthough the patient numbers are proba-\nbly too small for this result to be definite,\nthe data are highly encouraging because\nthey point in the right direction.\nOverall, the functional link between\nmTOR and PC1 and the effects of rapa-\nmycin on animal models and patients sug-\ngest that rapamycin may be a promising\ndrug with the potential to become the\nfirst available treatment for ADPKD\npatients. The fact that this drug is already\nclinically approved should greatly facilitate\nclinical trials.\nBesides these exciting clinical implica-\ntions, this work also raises many impor-\ntant basic science questions. Which\nother regulatory proteins (such as Rheb\nand hamartin; see Fig. 1) are involved in\nthe PC1–tuberin–mTOR complex? Is\nthere a cilia connection? If so, is mTOR\nactivity regulated by fluid flow and cilia\nbending? Very recent work by the same\nlaboratory identified another novel\npathway involving the PC1 tail (13).\nThese results suggested that another\nfunction of PC1 is to sequester the tran-\nscription factor signal transducers and\nactivators of transcription 6 (STAT6) in\ncilia under normal conditions. In the\nabsence of fluid flow through the kid-\nney, however, the tail of PC1 is cleaved\nby a protease and translocates together\nwith STAT6 to the nucleus, where it\nactivates gene expression (13). This\nnovel ciliary mechanotransduction path-\nway may be the long-sought-after mech-\nanism of cilia function in the kidney, but\nhow does it relate to the activation of\nmTOR? It will be exciting to find\nanswers to these questions.\n3. Shillingford, J. M., Murcia, N. S., Larson, C. H., Low,\nS. H., Hedgepeth, R., Brown, N., Flask, C. A., Novick,\nA. C., Goldfarb, D. A., Kramer-Zucker, A., et al.\n(2006) Proc. Natl. Acad. Sci. USA 103, 5466–5471.\n4. Kleymenova, E., Ibraghimov-Beskrovnaya, O., Kugoh,\nH., Everitt, J., Xu, H., Kiguchi, K., Landes, G., Harris,\nP. & Walker, C. (2001) Mol. Cell 7, 823–832.",
         "The mTOR pathway is regulated by polycystin-1, and its inhibition reverses renal cystogenesis in polycystic kidney disease.;Rapamycin: an anti-cancer immunosuppressant?;Intraflagellar transport and cilia-dependent renal disease: the ciliary hypothesis of polycystic kidney disease.;Autosomal dominant polycystic kidney disease: molecular genetics and pathophysiology.;Polycystins 1 and 2 mediate mechanosensation in the primary cilium of kidney cells;Tuberin, the tuberous sclerosis complex 2 tumor suppressor gene product, regulates Rho activation, cell adhesion and migration;Bending the MDCK Cell Primary Cilium Increases Intracellular Calcium;Tuberin-dependent membrane localization of polycystin-1: a functional link between polycystic kidney disease and the TSC2 tumor suppressor gene.;Polycystin-1, STAT6, and P100 function in a pathway that transduces ciliary mechanosensation and is activated in polycystic kidney disease.;TSC2: filling the GAP in the mTOR signaling pathway.;Dysregulation of the TSC-mTOR pathway in human disease;www.pnas.org cgi doi 10.1073 pnas.0601352103 Mostov D ow nl oa de d by g ue st o n Ja nu ar y 10",
         "mTOR is out of control in polycystic kidney disease.",
         "None"
        ],
        [
         "24",
         "0015c20c3ac2a207674c3bad17bf7487b6e9863d",
         "A discrete model of pedestrian motion is presented that is imple-\nmented in the Floor ﬁeld- and Agentbased Simulation Tool (F.A.S.T.)\nwhich has already been applicated to a variety of real life scenarios.",
         "T. Kretz,M. Schreckenberg",
         "T. Kretz,M. Schreckenberg",
         "http://arxiv.org/pdf/0804.1893",
         "A discrete model of pedestrian motion is presented that is imple-\nmented in the Floor ﬁeld- and Agentbased Simulation Tool (F.A.S.T.)\nwhich has already been applicated to a variety of real life scenarios.",
         "",
         "The F.A.S.T. model is discrete in space and time with an orthogonal lattice.\nIt can be classiﬁed as probabilistic CA with extensions demanded by reality.\nThere is a hard-core exlusion between the agents, of which at maximum one\ncan stand at a cell at a certain point in time. An agent needs the space\nof one cell. This implies a cell size of roughly 40 · 40 cm2, the minimum\nspace a pedestrian occupies [Dre67]. So far the model follows earlier models\n[Klu03, Kes01]. In fact this model is in many aspects an extension - mainly\nrelated to speeds larger one cell per round - of the model presented in [Kir02]\nwhich itself had predecessors [Bur01a, Sch01a, Bur01b].\nThere are three levels of decision making in this model: 1) The choice of\nan exit, 2) the choice of a destination cell, 3) the path between the current\nand the destination cell. The ﬁrst two are probabilistic processes. The third\none is deterministic, except for the order in which the agents carry out their\nsteps to reach the destination cell. The process of choosing a destination\ncell is done completely in parallel by all agents, while the actual motion is\na totally sequential process.\nIn the following a round includes the decision for an exit as well as for a\ndestination cell and all steps, while a step is the movement of an agent from\none cell to one of the nearest neighbour cells i.e. a part of the path from the\ncurrent towards the destination cell.\n1.1 Choosing an Exit\nAt the beginning of each round all agents choose one of the exits with the\nprobability pA\nE = N (1+ δAEkE(A))/S(A, E)2, with A numbering the agents,\nE numbering the exits agent A is allowed to use, δAE = 1 if agent A chose\nexit E during the last round and δAE = 0 otherwise, kE(A) being agent A’s\npersistance to stick with a once taken decision for one of the exits, S(A, E)\nbeing the distance between the exit and the current position of agent A,\nand N as normalization constant guaranteeing PE pE = 1. The distance\nis squared so the probability is proportional to the inverse of the area of a\ncircle around the exit with radius S(A, E). Given a homogeneous density of\nagents all over a scenario with high symmetry this area is proportional to\nthe number of agents which are closer to the exit than agent A. Therefore\nthis is a measure of a possible queue before agent A at exit E.\n1.2 Choosing a Destination Cell\nIn a model which is spatially and temporally discrete an agent’s (dimension-\nless) velocity is the number of cells which he is allowed to move during one\nround. As the real-world interpretation of the size of a cell is ﬁxed by the\nscale of the discretization, the real-time interpretation of one round ﬁxes\nthe real-world interpretation of such a dimensionless velocity. One round is\nchosen to equal the typical reaction time of one secound. Typical maximal\nvelocities vmax of the agents therefore are three to six cells per round.\nIn the F.A.S.T. model an agent chooses one cell (the destination cell) he\nwants to move to out of all cells he would be able to reach during one round,\nexcept for those that are occupied. Which cells are part of the neighbour-\nhood that belongs to a certain vmax (i.e. the shape of such a neighbourhood)\nis described in [Kre05].\n1.2.1 Probabilities for the possible Destination Cells:\nProbabilities get assigned to each free and unoccupied cell in the neighbour-\nhood of an agent that corresponds to the maximum velocity of that agent,\nthat that particular cell is chosen as destination cell. The probability that\nan agent chooses cell (x, y) is\np = N pS\n(1)\nWhile N is a normalization constant all pX\nthe diﬀerent inﬂuences on the movement of an agent.\nxy are partial probabilities from\nxy is the inﬂuence of the static ﬂoor ﬁeld which contains the informa-\ntion on the distance towards the exit.\nxy is the inﬂuence of the dynamic ﬂoor ﬁeld [Sch01b] which contains\nthe information of the motion of the other agents.\nxy is the inﬂuence of inertia eﬀects.\nxy is the inﬂuence of nearby walls.\nxy is the inﬂuence of the density of nearby agents.\nThese ﬁve inﬂuences will be introduced in more detail now.\nMoving towards the Exit - Following the Static Floor Field: Before\nthe simulation begins, the distance from each cell to each exit is calculated\nusing Dijkstra’s algorithm [Dij59] and stored in the static ﬂoor ﬁeld. With\nthe static ﬂoor ﬁeld pS is calculated for a certain cell at (x, y) as pS\nxy =\ne−kSSxy , with kS being the coupling strength of an agent to the static ﬂoor\nﬁeld knowledge as well as will to move are parametrized. All of the ﬁve\ninﬂuences are weighted against each other in their relative strengths by\ncoupling constants kX and all coupling constants are individual parameters\nof the agents.\nHerding Behaviour - Following Others: Asides the main CA where\nthe agents move, there is another CA - the dynamic ﬂoor ﬁeld - where agents\nleave a virtual trace whenever they move. This trace decays and diﬀuses\nwith time.\nIn the F.A.S.T. model the dynamic ﬂoor ﬁeld is a vectorial\nﬁeld. So an agent who has moved from (a, b) to (x, y) changes the dynamic\nﬂoor ﬁeld (Dx, Dy) at (a, b) by (x − a, y − b) after all agents have moved.\nRight after that all values of both components of D decay with probability\nδ and diﬀuse with probability α to one of the (von Neumann) neighbouring\ncells. Since the vector components can be negative, decay means a reduction\nof the absolute value. Diﬀusion is only possible from x- to x- and from\ny- to y-component. The inﬂuence on the motion of the agents is pD\nxy =\nekD(Dx(x,y)(x−a)+Dy(x,y)(y−b)) where (a, b) is the current position of the agent.\nInertia: Contrary to Newtonian physics pedestrians experience de- and\naccelerating in motion direction as being less ardous than walking through\ncurves. Due to the shape and functionality of the human movement appa-\nratus pedestrians can de- and accelerate from and to their preferred walking\nvelocities almost instantaneously compared to a timescale of one secound.\nHowever deviating quickly by e.g. 90◦ from a certain direction while keep-\ning up the walking speed is far more diﬃcult. So only centrifugal forces are\nconsidered to have an inﬂuence on the motion of the agents.\nOn a perfect circle the centrifugal force - which in the F.A.S.T. model is\nthe measure for inertia inﬂuences - is Fc ∝ v2/r. This assumption after a\nfew steps [Kre06] leads to the following inertia dependence of the movement\n−kI (vt+1+vt) sin |φ|\nprobability pI(xt+1, yt+1) = e\nfrom the former direction of motion and t counting the timesteps.\n2 with φ as angle of deviation\nxy = e(−kW Wxy)\nSafety Distance towards Walls. This is considered via pW\nwhere Wxy is the distance of the cell (x, y) towards the closest wall. For dis-\ntances larger than a certain Wmax the eﬀect vanishes completely and pW\nxy = 1.\nStaying Polite - Keeping a Distance Towards other Agents. After\neach round for each cell (x, y) the number NP (x, y) of agents in its Moore\nneighbourhood is counted. The more agents are immediately neighboured,\nthe less another agent might want to choose this cell as his destination:\nxy = e−kP NP (x,y). Then the density at the border of a crowd changes less\npP\nrapidly, while in the center of the crowd the density remains high since all\nfree cells which an agent can reach during one round are surrounded by\nagents.\n1.3 Moving towards the Destination Cell\nOnce all agents have chosen their destination cell the agents start moving\ntowards them. The sequence in which the agents execute their steps is chosen\nrandomly in a way, that agent D executes a step, then agent B, then again\nagent D, then agent K. During one step an agent moves deterministically\nonto that cell within the Moore neighbourhood of his current cell, that lies\nclosest to his destination cell. To represent the dynamic space consumption\n[Wei93] a cell which has been occupied once during this process remains\nblocked for all other agents until the end of the round. So if the cell closest\nto the destination cell is blocked the agent moves to the secound closest and\nso on. If there is no cell left that is unblocked and closer to the destination\ncell than the current cell the round ends for that particular agent.",
         "",
         "",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1217413%26volume%3d176%26issue%3d3%26page%3d1645%26filename%3d1645.pdf%26languageId%3d",
         "A discrete model of pedestrian motion is presented that is imple-\nmented in the Floor ﬁeld- and Agentbased Simulation Tool (F.A.S.T.)\nwhich has already been applicated to a variety of real life scenarios.\nThe F.A.S.T. model is discrete in space and time with an orthogonal lattice.\nIt can be classiﬁed as probabilistic CA with extensions demanded by reality.\nThere is a hard-core exlusion between the agents, of which at maximum one\ncan stand at a cell at a certain point in time. An agent needs the space\nof one cell. This implies a cell size of roughly 40 · 40 cm2, the minimum\nspace a pedestrian occupies [Dre67]. So far the model follows earlier models\n[Klu03, Kes01]. In fact this model is in many aspects an extension - mainly\nrelated to speeds larger one cell per round - of the model presented in [Kir02]\nwhich itself had predecessors [Bur01a, Sch01a, Bur01b].\nThere are three levels of decision making in this model: 1) The choice of\nan exit, 2) the choice of a destination cell, 3) the path between the current\nand the destination cell. The ﬁrst two are probabilistic processes. The third\none is deterministic, except for the order in which the agents carry out their\nsteps to reach the destination cell. The process of choosing a destination\ncell is done completely in parallel by all agents, while the actual motion is\na totally sequential process.\nIn the following a round includes the decision for an exit as well as for a\ndestination cell and all steps, while a step is the movement of an agent from\none cell to one of the nearest neighbour cells i.e. a part of the path from the\ncurrent towards the destination cell.\n1.1 Choosing an Exit\nAt the beginning of each round all agents choose one of the exits with the\nprobability pA\nE = N (1+ δAEkE(A))/S(A, E)2, with A numbering the agents,\nE numbering the exits agent A is allowed to use, δAE = 1 if agent A chose\nexit E during the last round and δAE = 0 otherwise, kE(A) being agent A’s\npersistance to stick with a once taken decision for one of the exits, S(A, E)\nbeing the distance between the exit and the current position of agent A,\nand N as normalization constant guaranteeing PE pE = 1. The distance\nis squared so the probability is proportional to the inverse of the area of a\ncircle around the exit with radius S(A, E). Given a homogeneous density of\nagents all over a scenario with high symmetry this area is proportional to\nthe number of agents which are closer to the exit than agent A. Therefore\nthis is a measure of a possible queue before agent A at exit E.\n1.2 Choosing a Destination Cell\nIn a model which is spatially and temporally discrete an agent’s (dimension-\nless) velocity is the number of cells which he is allowed to move during one\nround. As the real-world interpretation of the size of a cell is ﬁxed by the\nscale of the discretization, the real-time interpretation of one round ﬁxes\nthe real-world interpretation of such a dimensionless velocity. One round is\nchosen to equal the typical reaction time of one secound. Typical maximal\nvelocities vmax of the agents therefore are three to six cells per round.\nIn the F.A.S.T. model an agent chooses one cell (the destination cell) he\nwants to move to out of all cells he would be able to reach during one round,\nexcept for those that are occupied. Which cells are part of the neighbour-\nhood that belongs to a certain vmax (i.e. the shape of such a neighbourhood)\nis described in [Kre05].\n1.2.1 Probabilities for the possible Destination Cells:\nProbabilities get assigned to each free and unoccupied cell in the neighbour-\nhood of an agent that corresponds to the maximum velocity of that agent,\nthat that particular cell is chosen as destination cell. The probability that\nan agent chooses cell (x, y) is\np = N pS\n(1)\nWhile N is a normalization constant all pX\nthe diﬀerent inﬂuences on the movement of an agent.\nxy are partial probabilities from\nxy is the inﬂuence of the static ﬂoor ﬁeld which contains the informa-\ntion on the distance towards the exit.\nxy is the inﬂuence of the dynamic ﬂoor ﬁeld [Sch01b] which contains\nthe information of the motion of the other agents.\nxy is the inﬂuence of inertia eﬀects.\nxy is the inﬂuence of nearby walls.\nxy is the inﬂuence of the density of nearby agents.\nThese ﬁve inﬂuences will be introduced in more detail now.\nMoving towards the Exit - Following the Static Floor Field: Before\nthe simulation begins, the distance from each cell to each exit is calculated\nusing Dijkstra’s algorithm [Dij59] and stored in the static ﬂoor ﬁeld. With\nthe static ﬂoor ﬁeld pS is calculated for a certain cell at (x, y) as pS\nxy =\ne−kSSxy , with kS being the coupling strength of an agent to the static ﬂoor\nﬁeld knowledge as well as will to move are parametrized. All of the ﬁve\ninﬂuences are weighted against each other in their relative strengths by\ncoupling constants kX and all coupling constants are individual parameters\nof the agents.\nHerding Behaviour - Following Others: Asides the main CA where\nthe agents move, there is another CA - the dynamic ﬂoor ﬁeld - where agents\nleave a virtual trace whenever they move. This trace decays and diﬀuses\nwith time.\nIn the F.A.S.T. model the dynamic ﬂoor ﬁeld is a vectorial\nﬁeld. So an agent who has moved from (a, b) to (x, y) changes the dynamic\nﬂoor ﬁeld (Dx, Dy) at (a, b) by (x − a, y − b) after all agents have moved.\nRight after that all values of both components of D decay with probability\nδ and diﬀuse with probability α to one of the (von Neumann) neighbouring\ncells. Since the vector components can be negative, decay means a reduction\nof the absolute value. Diﬀusion is only possible from x- to x- and from\ny- to y-component. The inﬂuence on the motion of the agents is pD\nxy =\nekD(Dx(x,y)(x−a)+Dy(x,y)(y−b)) where (a, b) is the current position of the agent.\nInertia: Contrary to Newtonian physics pedestrians experience de- and\naccelerating in motion direction as being less ardous than walking through\ncurves. Due to the shape and functionality of the human movement appa-\nratus pedestrians can de- and accelerate from and to their preferred walking\nvelocities almost instantaneously compared to a timescale of one secound.\nHowever deviating quickly by e.g. 90◦ from a certain direction while keep-\ning up the walking speed is far more diﬃcult. So only centrifugal forces are\nconsidered to have an inﬂuence on the motion of the agents.\nOn a perfect circle the centrifugal force - which in the F.A.S.T. model is\nthe measure for inertia inﬂuences - is Fc ∝ v2/r. This assumption after a\nfew steps [Kre06] leads to the following inertia dependence of the movement\n−kI (vt+1+vt) sin |φ|\nprobability pI(xt+1, yt+1) = e\nfrom the former direction of motion and t counting the timesteps.\n2 with φ as angle of deviation\nxy = e(−kW Wxy)\nSafety Distance towards Walls. This is considered via pW\nwhere Wxy is the distance of the cell (x, y) towards the closest wall. For dis-\ntances larger than a certain Wmax the eﬀect vanishes completely and pW\nxy = 1.\nStaying Polite - Keeping a Distance Towards other Agents. After\neach round for each cell (x, y) the number NP (x, y) of agents in its Moore\nneighbourhood is counted. The more agents are immediately neighboured,\nthe less another agent might want to choose this cell as his destination:\nxy = e−kP NP (x,y). Then the density at the border of a crowd changes less\npP\nrapidly, while in the center of the crowd the density remains high since all\nfree cells which an agent can reach during one round are surrounded by\nagents.\n1.3 Moving towards the Destination Cell\nOnce all agents have chosen their destination cell the agents start moving\ntowards them. The sequence in which the agents execute their steps is chosen\nrandomly in a way, that agent D executes a step, then agent B, then again\nagent D, then agent K. During one step an agent moves deterministically\nonto that cell within the Moore neighbourhood of his current cell, that lies\nclosest to his destination cell. To represent the dynamic space consumption\n[Wei93] a cell which has been occupied once during this process remains\nblocked for all other agents until the end of the round. So if the cell closest\nto the destination cell is blocked the agent moves to the secound closest and\nso on. If there is no cell left that is unblocked and closer to the destination\ncell than the current cell the round ends for that particular agent.",
         "",
         "The F.A.S.T.-Model",
         "None"
        ],
        [
         "25",
         "0016d2f869fda0a40404330902d5f4a8c361bad8",
         "In the northern hemisphere, there is compelling evidence for climate-related advances of spring events, but no such long-term biological time series exist for the southern hemisphere. We have studied a unique data set of dates of first arrival and laying of first eggs over a 55-year period for the entire community of Antarctic seabirds in East Antarctica. The records over this long period show a general unexpected tendency toward later arrival and laying, an inverse trend to those observed in the northern hemisphere. Overall, species now arrive at their colonies 9.1 days later, on average, and lay eggs an average of 2.1 days later than in the early 1950s. Furthermore, these delays are linked to a decrease in sea ice extent that has occurred in eastern Antarctica, which underlies the contrasted effects of global climate change on species in Antarctica.",
         "C. Barbraud,H. Weimerskirch",
         "C. Barbraud,H. Weimerskirch",
         "https://europepmc.org/articles/pmc1458863?pdf=render",
         "In the northern hemisphere, there is compelling evidence for\nclimate-related advances of spring events, but no such long-term\nbiological time series exist for the southern hemisphere. We have\nstudied a unique data set of dates of ﬁrst arrival and laying of ﬁrst\neggs over a 55-year period for the entire community of Antarctic\nseabirds in East Antarctica. The records over this long period show\na general unexpected tendency toward later arrival and laying, an\ninverse trend to those observed in the northern hemisphere.\nOverall, species now arrive at their colonies 9.1 days later, on\naverage, and lay eggs an average of 2.1 days later than in the early\n1950s. Furthermore, these delays are linked to a decrease in sea\nice extent that has occurred in eastern Antarctica, which underlies\nthe contrasted effects of global climate change on species in\nAntarctica.",
         "",
         "ogy (the study of the timing of seasonal activities of biolog-\nical events) of organisms living in seasonal environments (1, 2).\nAnalyses of long-term phenological data sets have indeed re-\nvealed significant phenological trends linked to large-scale cli-\nmatic signals, such as the North Atlantic Oscillation, and weather\nconditions such as spring temperatures (3–5). In general, these\ntrends indicate that spring activities such as first arrival of\nmigrant birds or earlier sprouting of plants have occurred\nprogressively earlier since the mid-20th century in response to\nincreasing temperatures. However, there is a major geographical\nbias among these phenological studies. Indeed, the overwhelm-\ning majority of the studies have been conducted in the northern\nhemisphere, particularly in Europe and North America. To our\nknowledge, no such long-term biological time series exist for the\nsouthern hemisphere and particularly not for the Antarctic\nfauna. This bias is problematic because observed climate changes\ndiffer between both hemispheres (6), and, consequently, the\nphenological responses might differ.\nHere, we analyze the only long-term record of phenological\nevents for all species of seabirds breeding in continental Ant-\narctica. This record offers an unusual and unique opportunity to\nobserve long-term changes of arrival and egg-laying dates in\nrelation to climate trends in the southern hemisphere.\nResults and Discussion\nOf nine seabird species for which dates of first arrival and first\negg laying have been recorded at the Dumont d’Urville Station\n(lat 66°70S, long 140°00E) in Ade´lie Land (Fig. 1) over the\nperiod 1950–2004, four species tended to arrive later, with shifts\nranging from 5.2 to 30.6 days, and two species bred later, with\nshifts ranging from 2.8 to 3.7 days (Fig. 2). All other species also\ntended to arrive and breed later, but the trends were not\nstatistically significant at the 0.05 level. Overall, species now\narrive at their colonies an average of 9.1 days (SE of 3.6 days)\nlater and lay eggs an average of 2.1 days (SE of 1.4 days) later\nthan in the early 1950s. In contrast, the only significant trend\ntoward earlier laying (3.8-day shift) was for the south polar skua\n(Catharacta maccormicki), which mainly predates eggs and\nchicks of Ade´lie penguins (Pygoscelis adeliae), whereas the other\nspecies forage at sea for fish, crustaceans, and squid (7).\nWe found no significant relationship between phenological\nevents and average air temperatures for the month preceding\nspecies-specific arrival and laying dates. Large-scale climatic\nindices that quantify Antarctic climatic conditions [i.e., the\nSouthern Annular Mode (Fig. 3A) and the Southern Oscillation\nIndex (Fig. 3B) (8, 9)] were also not related to changes in\nphenological events, except for the Cape petrel (Daption cap-\nense), for which dates of first arrival and first egg laying were\nnegatively related to the Southern Oscillation Index (r2  0.196,\nP  0.0047 and r2  0.242, P  0.0058, respectively). Because sea\nice extent has a profound influence on marine productivity and\nseabird dynamics (10–12), we used the only existing long-term\nproxy for sea ice extent in East Antarctica before the satellite era\n(Fig. 3D) (13). Regression analyses between sea ice extent and\nphenological variables indicate that the date of first arrival was\nnegatively related to sea ice extent for four species and that the\ndate of first egg laying was negatively related to sea ice extent for\nthree species (Fig. 4). A metaanalysis in which the slopes of\nregressions were considered as measures of effect size confirmed\nthat the mean slope of arrival date versus sea ice extent signif-\nicantly departed from the null model that assumes no change as\na function of sea ice extent [one sample Student’s t test of mean\nslope against 0; t(16)  2.577; P (two-tailed)  0.02].\nOur results contrast with phenological changes observed in\nthe northern hemisphere, where increasing spring tempera-\ntures are likely to have had pronounced effects on nearly all\nphenophases. For birds, spring warming is likely to have\nincreased the availability of food supplies, resulting in earlier\narrival and laying dates (2). In East (and continental) Ant-\narctica, no major warming or cooling has occurred since the\nearly 1950s, contrary to what has been observed in the\nnorthern hemisphere or in western Antarctica (6, 14). How-\never, the large (12–20%) reduction in sea ice extent since the\n1950s (13) has been associated with a decline in abundances of\nkrill and other marine organisms (10, 11, 15), which are major\nfood resources for most Antarctic seabirds (7). Their decline\nmay be driving the observed delays in bird arrival and breed-\ning. In addition, the duration of the sea ice season tended to\nincrease by 40 days in eastern Antarctica since the late 1970s\n(Fig. 3E) (16). Although these contrasting trends seem coun-\nterintuitive, recent research suggests that sea ice extent is\nconnected to large-scale features of the global climate system\n(such as the Southern Oscillation Index or the Southern\nAnnular Mode), whereas the length of the sea ice season is\nconnected to more regional features such as temperature (17).\nBecause late sea ice breakup is known to delay access to\ncolonies (18, 19) and food resources at sea, it may also explain\nAntarctic seabirds’ tendency to arrive and reproduce later.\nTherefore, we suggest that decreasing sea ice extent and\nPhenological changes. (A) Dates of ﬁrst arrival for the emperor\nFig. 2.\npenguin, Aptenodytes forsteri (EMPE) (r2  0.001; P  0.81); Ade´ lie penguin,\nP. adeliae (ADPE) (r2  0.03; P  0.30); southern giant petrel, M. giganteus\n(SGPE) (r2  0.06; P  0.20); southern fulmar, F. glacialoides (SOFU) (r2  0.38;\nP  0.001); Antarctic petrel, T. antarctica (ANPE) (r2  0.21; P  0.04); Cape\npetrel, D. capense (CAPE) (r2  0.17; P  0.01); snow petrel, Pagodroma nivea\n(SNPE) (r2  0.001; P  0.83); Wilson’s storm petrel, O. oceanicus (WSPE) (r2 \n0.11; P  0.05); and south polar skua C. maccormicki (SPSK) (r2  0.07; P  0.09).\n(B) Dates of laying of ﬁrst eggs. Data are shown for EMPE (r2  0.12; P  0.11),\nADPE (r2  0.19; P  0.01), CAPE (r2  0.16; P  0.03), SNPE (r2  0.004; P  0.73),\nand SPSK (r2  0.11; P  0.05). Regression lines indicate signiﬁcance of the\ntrends (solid line, P  0.05; dotted line, P  0.05).\nincreasing sea ice season duration have reduced the quantity\nand accessibility of the food supplies available in early spring\nand may partly explain the delays observed in arrival and laying\ndates, with seabirds needing more time to build up the reserves\nnecessary for breeding (20). However, sea ice extent only\nexplained up to 24% of the variance in arrival or egg-laying\ndates. Given that none of the other climatic factors examined\nwere related to arrival date or to laying date of first eggs, there\nare other factor(s) that are more important than sea ice extent\nin determining arrival and laying dates.\nThis unique data set for the southern hemisphere shows\nclearly that responses of Antarctic seabirds are opposite to\nthose observed in the northern hemisphere and the Arctic but\nconsistent with climatic and oceanographic changes in eastern\nAntarctica. The consequences of delayed phenology could be\nserious for these top predators if they become less synchro-\nnized with the phenology of their food supplies (21). Inter-\nestingly, the 9.1-day delay in arrival at the colonies and the\n2.1-day delay in the date of laying of the first egg represent, on\naverage, a 7-day compression of the prelaying period when\nbirds set up territories and court and females make their\negg(s). The time needed by females to make their egg is\nprobably under strong physiological constraints, and thus,\npresumably, the duration needed for setting up territories\nandor courting decreased, suggesting some plasticity in the\ntiming of these activities. The fact that Antarctic top predators\nalready respond to environmental changes by shifts in their\npopulation dynamics (12, 22, 23) and phenology raises some\nfascinating questions about how they will be affected by\nongoing climatic trends. Because southern shifts in the distri-\nbution of Antarctic species are extremely limited by the\npresence of the Antarctic ice cap, unless microevolutionary\nresponses to climate change progress with sufficient speed,\nthese species may be unable to respond appropriately to\nchanges occurring in their breeding area and therefore may be\nClimate changes. (A) Southern Annular Mode (SAM) (r2  0.65; P \nFig. 3.\n0.001) as deﬁned in ref. 9. (B) Southern Oscillation Index (SOI) (r2  0.08; P \n0.03). (C) Average annual air temperatures recorded at the Dumont d’Urville\nmeteorological station (1956 –2002) (r2  0.01; P  0.47). (D) Methanesulfonic\nacid concentration (MSA) (r2  0.32; P  0.001) as a proxy for sea ice extent\nfrom an ice core in East Antarctica (13). Values were redrawn from ref. 13. (E)\nSea ice season length (SIL) for the sector 136°E to 142°E and north of 65°S (r2 \n0.13; P  0.09). Data are from ref. 16; updates were kindly provided by C. L.\nParkinson (NASA Goddard Space Flight Center, Greenbelt, MD). Regression\nlines indicate signiﬁcance of the trends (solid line, P  0.05; dotted line, P \n0.05).\ndisproportionately negatively affected by current climate\nchanges.\nMaterials and Methods\nPhenological Data. Since 1950, the dates of first arrival of the nine\nseabird species breeding in Antarctica and the dates of laying of first\neggs from five of these species have been recorded by ornithologists\nwintering almost each year at the Dumont d’Urville Station in\nAde´lie Land, Antarctica, generating the longest phenological data\nset available for Antarctic species. First arrival dates and first\negg-laying dates were recorded as part of a long-term study of top\npredators at Ade´lie Land. The Antarctic petrel (Thalassoica ant-\narctica) is the only Antarctic seabird species not breeding at\nDumont d’Urville. However, a large breeding colony (nearly 4,000\nbreeding pairs) is situated 110 km from Dumont d’Urville (24),\nand the species is regularly seen in the surroundings of the station.\nConsequently, first arrival dates for all of the nine species of\nAntarctic seabirds were used in the analysis. For three species\n(southern giant petrel, Macronectes giganteus; southern fulmar,\nFulmarus glacialoides; and Wilson’s storm petrel, Oceanites oceani-\ncus), egg-laying dates were not recorded either because these\nspecies breed in low numbers, and access to colonies was limited to\nminimize disturbance, or because of their cryptic nesting. Because\nall breeding colonies are situated in the vicinity of the research\nstation and because the observer was permanently in the field\n(weather permitting), we are very confident in the precision of\narrival and egg-laying dates. Although the methodological uncer-\ntainties for the observations have not been quantified, the precision\nof the observations is probably 1 day.\nStatistical Analyses. To correct for fluctuations in the calendar\ndate of the vernal equinox over the study period, phenological\nevents were reported and expressed as the time elapsed since the\nvernal equinox for each year of the data set (25). In the\nmetaanalysis, the slopes of regressions were considered as\nmeasures of effect size, and Kolmogorov–Smirnov tests con-\nfirmed that the slopes were distributed normally (P  0.66).\nWe thank all of the people involved in the monitoring programs of\nseabirds in Terre Ade´lie since 1950; C. L. Parkinson for providing\nupdates of the sea ice season length data; Me´te´o France for the\nmeteorological data of Dumont d’Urville; A. Lagarde and D. Besson for\ndata management; and J. Croxall, C.-A. Bost, O. Chastel, Y. Cherel, M.\nNevoux, and D. Pinaud for comments on the manuscript. This work was\nsupported over the past 55 years by Expe´ditions Polaires Franc¸aises,\nInstitut Polaire Paul Emile Victor (Research Program 109), and Terres\nAustrales et Antarctiques Franc¸aises.",
         "",
         "",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1217413%26volume%3d176%26issue%3d3%26page%3d1645%26filename%3d1645.pdf%26languageId%3d",
         "In the northern hemisphere, there is compelling evidence for\nclimate-related advances of spring events, but no such long-term\nbiological time series exist for the southern hemisphere. We have\nstudied a unique data set of dates of ﬁrst arrival and laying of ﬁrst\neggs over a 55-year period for the entire community of Antarctic\nseabirds in East Antarctica. The records over this long period show\na general unexpected tendency toward later arrival and laying, an\ninverse trend to those observed in the northern hemisphere.\nOverall, species now arrive at their colonies 9.1 days later, on\naverage, and lay eggs an average of 2.1 days later than in the early\n1950s. Furthermore, these delays are linked to a decrease in sea\nice extent that has occurred in eastern Antarctica, which underlies\nthe contrasted effects of global climate change on species in\nAntarctica.\nogy (the study of the timing of seasonal activities of biolog-\nical events) of organisms living in seasonal environments (1, 2).\nAnalyses of long-term phenological data sets have indeed re-\nvealed significant phenological trends linked to large-scale cli-\nmatic signals, such as the North Atlantic Oscillation, and weather\nconditions such as spring temperatures (3–5). In general, these\ntrends indicate that spring activities such as first arrival of\nmigrant birds or earlier sprouting of plants have occurred\nprogressively earlier since the mid-20th century in response to\nincreasing temperatures. However, there is a major geographical\nbias among these phenological studies. Indeed, the overwhelm-\ning majority of the studies have been conducted in the northern\nhemisphere, particularly in Europe and North America. To our\nknowledge, no such long-term biological time series exist for the\nsouthern hemisphere and particularly not for the Antarctic\nfauna. This bias is problematic because observed climate changes\ndiffer between both hemispheres (6), and, consequently, the\nphenological responses might differ.\nHere, we analyze the only long-term record of phenological\nevents for all species of seabirds breeding in continental Ant-\narctica. This record offers an unusual and unique opportunity to\nobserve long-term changes of arrival and egg-laying dates in\nrelation to climate trends in the southern hemisphere.\nResults and Discussion\nOf nine seabird species for which dates of first arrival and first\negg laying have been recorded at the Dumont d’Urville Station\n(lat 66°70S, long 140°00E) in Ade´lie Land (Fig. 1) over the\nperiod 1950–2004, four species tended to arrive later, with shifts\nranging from 5.2 to 30.6 days, and two species bred later, with\nshifts ranging from 2.8 to 3.7 days (Fig. 2). All other species also\ntended to arrive and breed later, but the trends were not\nstatistically significant at the 0.05 level. Overall, species now\narrive at their colonies an average of 9.1 days (SE of 3.6 days)\nlater and lay eggs an average of 2.1 days (SE of 1.4 days) later\nthan in the early 1950s. In contrast, the only significant trend\ntoward earlier laying (3.8-day shift) was for the south polar skua\n(Catharacta maccormicki), which mainly predates eggs and\nchicks of Ade´lie penguins (Pygoscelis adeliae), whereas the other\nspecies forage at sea for fish, crustaceans, and squid (7).\nWe found no significant relationship between phenological\nevents and average air temperatures for the month preceding\nspecies-specific arrival and laying dates. Large-scale climatic\nindices that quantify Antarctic climatic conditions [i.e., the\nSouthern Annular Mode (Fig. 3A) and the Southern Oscillation\nIndex (Fig. 3B) (8, 9)] were also not related to changes in\nphenological events, except for the Cape petrel (Daption cap-\nense), for which dates of first arrival and first egg laying were\nnegatively related to the Southern Oscillation Index (r2  0.196,\nP  0.0047 and r2  0.242, P  0.0058, respectively). Because sea\nice extent has a profound influence on marine productivity and\nseabird dynamics (10–12), we used the only existing long-term\nproxy for sea ice extent in East Antarctica before the satellite era\n(Fig. 3D) (13). Regression analyses between sea ice extent and\nphenological variables indicate that the date of first arrival was\nnegatively related to sea ice extent for four species and that the\ndate of first egg laying was negatively related to sea ice extent for\nthree species (Fig. 4). A metaanalysis in which the slopes of\nregressions were considered as measures of effect size confirmed\nthat the mean slope of arrival date versus sea ice extent signif-\nicantly departed from the null model that assumes no change as\na function of sea ice extent [one sample Student’s t test of mean\nslope against 0; t(16)  2.577; P (two-tailed)  0.02].\nOur results contrast with phenological changes observed in\nthe northern hemisphere, where increasing spring tempera-\ntures are likely to have had pronounced effects on nearly all\nphenophases. For birds, spring warming is likely to have\nincreased the availability of food supplies, resulting in earlier\narrival and laying dates (2). In East (and continental) Ant-\narctica, no major warming or cooling has occurred since the\nearly 1950s, contrary to what has been observed in the\nnorthern hemisphere or in western Antarctica (6, 14). How-\never, the large (12–20%) reduction in sea ice extent since the\n1950s (13) has been associated with a decline in abundances of\nkrill and other marine organisms (10, 11, 15), which are major\nfood resources for most Antarctic seabirds (7). Their decline\nmay be driving the observed delays in bird arrival and breed-\ning. In addition, the duration of the sea ice season tended to\nincrease by 40 days in eastern Antarctica since the late 1970s\n(Fig. 3E) (16). Although these contrasting trends seem coun-\nterintuitive, recent research suggests that sea ice extent is\nconnected to large-scale features of the global climate system\n(such as the Southern Oscillation Index or the Southern\nAnnular Mode), whereas the length of the sea ice season is\nconnected to more regional features such as temperature (17).\nBecause late sea ice breakup is known to delay access to\ncolonies (18, 19) and food resources at sea, it may also explain\nAntarctic seabirds’ tendency to arrive and reproduce later.\nTherefore, we suggest that decreasing sea ice extent and\nPhenological changes. (A) Dates of ﬁrst arrival for the emperor\nFig. 2.\npenguin, Aptenodytes forsteri (EMPE) (r2  0.001; P  0.81); Ade´ lie penguin,\nP. adeliae (ADPE) (r2  0.03; P  0.30); southern giant petrel, M. giganteus\n(SGPE) (r2  0.06; P  0.20); southern fulmar, F. glacialoides (SOFU) (r2  0.38;\nP  0.001); Antarctic petrel, T. antarctica (ANPE) (r2  0.21; P  0.04); Cape\npetrel, D. capense (CAPE) (r2  0.17; P  0.01); snow petrel, Pagodroma nivea\n(SNPE) (r2  0.001; P  0.83); Wilson’s storm petrel, O. oceanicus (WSPE) (r2 \n0.11; P  0.05); and south polar skua C. maccormicki (SPSK) (r2  0.07; P  0.09).\n(B) Dates of laying of ﬁrst eggs. Data are shown for EMPE (r2  0.12; P  0.11),\nADPE (r2  0.19; P  0.01), CAPE (r2  0.16; P  0.03), SNPE (r2  0.004; P  0.73),\nand SPSK (r2  0.11; P  0.05). Regression lines indicate signiﬁcance of the\ntrends (solid line, P  0.05; dotted line, P  0.05).\nincreasing sea ice season duration have reduced the quantity\nand accessibility of the food supplies available in early spring\nand may partly explain the delays observed in arrival and laying\ndates, with seabirds needing more time to build up the reserves\nnecessary for breeding (20). However, sea ice extent only\nexplained up to 24% of the variance in arrival or egg-laying\ndates. Given that none of the other climatic factors examined\nwere related to arrival date or to laying date of first eggs, there\nare other factor(s) that are more important than sea ice extent\nin determining arrival and laying dates.\nThis unique data set for the southern hemisphere shows\nclearly that responses of Antarctic seabirds are opposite to\nthose observed in the northern hemisphere and the Arctic but\nconsistent with climatic and oceanographic changes in eastern\nAntarctica. The consequences of delayed phenology could be\nserious for these top predators if they become less synchro-\nnized with the phenology of their food supplies (21). Inter-\nestingly, the 9.1-day delay in arrival at the colonies and the\n2.1-day delay in the date of laying of the first egg represent, on\naverage, a 7-day compression of the prelaying period when\nbirds set up territories and court and females make their\negg(s). The time needed by females to make their egg is\nprobably under strong physiological constraints, and thus,\npresumably, the duration needed for setting up territories\nandor courting decreased, suggesting some plasticity in the\ntiming of these activities. The fact that Antarctic top predators\nalready respond to environmental changes by shifts in their\npopulation dynamics (12, 22, 23) and phenology raises some\nfascinating questions about how they will be affected by\nongoing climatic trends. Because southern shifts in the distri-\nbution of Antarctic species are extremely limited by the\npresence of the Antarctic ice cap, unless microevolutionary\nresponses to climate change progress with sufficient speed,\nthese species may be unable to respond appropriately to\nchanges occurring in their breeding area and therefore may be\nClimate changes. (A) Southern Annular Mode (SAM) (r2  0.65; P \nFig. 3.\n0.001) as deﬁned in ref. 9. (B) Southern Oscillation Index (SOI) (r2  0.08; P \n0.03). (C) Average annual air temperatures recorded at the Dumont d’Urville\nmeteorological station (1956 –2002) (r2  0.01; P  0.47). (D) Methanesulfonic\nacid concentration (MSA) (r2  0.32; P  0.001) as a proxy for sea ice extent\nfrom an ice core in East Antarctica (13). Values were redrawn from ref. 13. (E)\nSea ice season length (SIL) for the sector 136°E to 142°E and north of 65°S (r2 \n0.13; P  0.09). Data are from ref. 16; updates were kindly provided by C. L.\nParkinson (NASA Goddard Space Flight Center, Greenbelt, MD). Regression\nlines indicate signiﬁcance of the trends (solid line, P  0.05; dotted line, P \n0.05).\ndisproportionately negatively affected by current climate\nchanges.\nMaterials and Methods\nPhenological Data. Since 1950, the dates of first arrival of the nine\nseabird species breeding in Antarctica and the dates of laying of first\neggs from five of these species have been recorded by ornithologists\nwintering almost each year at the Dumont d’Urville Station in\nAde´lie Land, Antarctica, generating the longest phenological data\nset available for Antarctic species. First arrival dates and first\negg-laying dates were recorded as part of a long-term study of top\npredators at Ade´lie Land. The Antarctic petrel (Thalassoica ant-\narctica) is the only Antarctic seabird species not breeding at\nDumont d’Urville. However, a large breeding colony (nearly 4,000\nbreeding pairs) is situated 110 km from Dumont d’Urville (24),\nand the species is regularly seen in the surroundings of the station.\nConsequently, first arrival dates for all of the nine species of\nAntarctic seabirds were used in the analysis. For three species\n(southern giant petrel, Macronectes giganteus; southern fulmar,\nFulmarus glacialoides; and Wilson’s storm petrel, Oceanites oceani-\ncus), egg-laying dates were not recorded either because these\nspecies breed in low numbers, and access to colonies was limited to\nminimize disturbance, or because of their cryptic nesting. Because\nall breeding colonies are situated in the vicinity of the research\nstation and because the observer was permanently in the field\n(weather permitting), we are very confident in the precision of\narrival and egg-laying dates. Although the methodological uncer-\ntainties for the observations have not been quantified, the precision\nof the observations is probably 1 day.\nStatistical Analyses. To correct for fluctuations in the calendar\ndate of the vernal equinox over the study period, phenological\nevents were reported and expressed as the time elapsed since the\nvernal equinox for each year of the data set (25). In the\nmetaanalysis, the slopes of regressions were considered as\nmeasures of effect size, and Kolmogorov–Smirnov tests con-\nfirmed that the slopes were distributed normally (P  0.66).\nWe thank all of the people involved in the monitoring programs of\nseabirds in Terre Ade´lie since 1950; C. L. Parkinson for providing\nupdates of the sea ice season length data; Me´te´o France for the\nmeteorological data of Dumont d’Urville; A. Lagarde and D. Besson for\ndata management; and J. Croxall, C.-A. Bost, O. Chastel, Y. Cherel, M.\nNevoux, and D. Pinaud for comments on the manuscript. This work was\nsupported over the past 55 years by Expe´ditions Polaires Franc¸aises,\nInstitut Polaire Paul Emile Victor (Research Program 109), and Terres\nAustrales et Antarctiques Franc¸aises.",
         "Southern Ocean sea ice and its wider linkages: insights revealed from models and observations;Long-term decline in krill stock and increase in salps within the Southern Ocean;Ice Core Evidence for Antarctic Sea Ice Decline Since the 1950s;The Adélie Penguin: Bellwether of Climate Change;A globally coherent fingerprint of climate change impacts across natural systems;Fingerprints of global warming on wild animals and plants;Environmental Change and Antarctic Seabird Populations;Interpretation of Recent Southern Hemisphere Climate Change;Ecological responses to recent climate change;Brood parasitism: Female ducks can double their reproduction;Devil in the Detail;Emperor penguins and climate change;Ocean circulation off east Antarctica affects ecosystem structure and sea-ice extent;Effects of elevated temperature on multi‐species interactions: the case of Pedunculate Oak, Winter Moth and Tits;First census of breeding seabirds between Cap Bienvenue (Terre Adélie) and Moyes Islands (King George V Land), Antarctica: new records for Antarctic seabird populations;The effect of ice conditions in northern Hudson Bay on breeding by Thick-billed Murres (Uria lomvia);Effects of sea-ice extent and krill or salp dominance on the Antarctic food web;Body Condition and Seabird Reproductive Performance: A Study of Three Petrel Species;Amphibian breeding and climate;The diets of five summer breeding seabirds in Adélie Land, Antarctica;Climate change 2001 : the scientific basis",
         "Antarctic birds breed later in response to climate change.",
         "None"
        ],
        [
         "26",
         "0017b6ca03ef7cff14e2bc573673ff91ea8caf24",
         "Classical cadherins such as E‐, P‐ and N‐cadherin are transmembrane proteins that mediate cell–cell adhesion, and are important in embryogenesis, maintenance of tissue integrity and cancer. Proteolytic shedding of the extracellular domain results in the generation of soluble E‐, P‐ or N‐cadherin ectodomains. Circulating soluble E‐ and P‐cadherin have been described in the serum, and elevated levels were detected in cancer patients when compared with healthy persons. Here we report the presence of soluble N‐cadherin, a 90‐kD protein fragment, in the serum of both healthy persons and cancer patients, using a direct ELISA and immunoprecipitation. A correlation was found between prostate specific antigen and soluble N‐cadherin, and significantly elevated levels were detected in prostate cancer follow‐up patients. The N‐cadherin protein is neo‐expressed by carcinomas of the prostate, and is responsible for epithelial to fibroblastic transition. This is reflected by the higher concentrations of soluble N‐cadherin in prostate cancer patients than in healthy persons. © 2006 Wiley‐Liss, Inc.",
         "L. Derycke,O. De Wever,V. Stove,B. Vanhoecke,J. Delanghe,H. Depypere,M. Bracke",
         "L. Derycke,O. De Wever,V. Stove,B. Vanhoecke,J. Delanghe,H. Depypere,M. Bracke",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/ijc.22219",
         "Classical cadherins such as E-, P- and N-cadherin are transmem-\nbrane proteins that mediate cell–cell adhesion, and are important\nin embryogenesis, maintenance of tissue integrity and cancer.\nProteolytic shedding of the extracellular domain results in the\ngeneration of soluble E-, P- or N-cadherin ectodomains. Circulat-\ning soluble E- and P-cadherin have been described in the serum,\nand elevated levels were detected in cancer patients when com-\npared with healthy persons. Here we report the presence of solu-\nble N-cadherin, a 90-kD protein fragment, in the serum of both\nhealthy persons and cancer patients, using a direct ELISA and\nimmunoprecipitation. A correlation was found between prostate\nspeciﬁc antigen and soluble N-cadherin, and signiﬁcantly elevated\nlevels were detected in prostate cancer follow-up patients. The N-\ncadherin protein is neo-expressed by carcinomas of the prostate,\nand is responsible for epithelial to ﬁbroblastic transition. This is\nreﬂected by the higher concentrations of soluble N-cadherin in\nprostate cancer patients than in healthy persons.\n' 2006 Wiley-Liss, Inc.",
         "Discussion\nTumor invasion is characterized in part by the ability of cancer\ncells to downregulate cell–cell adhesion and to invade into the sur-\nrounding tissues. In this respect, the progression of a carcinoma is\noften characterized by a switch from E- to N-cadherin. This switch\nin cadherin type expression has been observed in different malig-\nnant tissues, like breast, prostate, bladder, skin and colon carci-\nnoma.1,3,22,23 This switch also coincides with the transition of the\ncells from an epithelial to a ﬁbroblastic phenotype, and invasive-\nness of the cancer cells. By extension in the tumor microenviron-\nment, N-cadherin expression is not exclusively found on the cancer\ncells, but is also reactively overexpressed on the surface of sur-\nrounding ﬁbroblasts and endothelial cells.24 During cancer pro-\ngression a multitude of extracellular proteinases are present and\nto serum samples at a ﬁnal concentration of 1 mM dramatically\nreduced the amount of sN-CAD to 50%. So, this shows that we\ndetect the cadherin protein. This effect was also observed when\nblood from the same individual was collected in tubes containing\nEDTA (EDTA K2 or EDTA K3, 8%) or citrate (3.2% sodium ci-\nare now known as the cancer degradome.27,28 These proteinases\nare responsible for the shedding of several transmembrane protein\nectodomains, including cadherins and tyrosine kinase receptors.29\nIn the eighties, Damsky et al.30 reported the cleavage of E-\ncadherin in cell culture experiments, with the release of an 80-kD\nextracellular fragment (sE-CAD). Later, this fragment was also\nfound in different biological ﬂuids, like serum14 and urine,17 and\nelevated concentrations were described in the serum of gastric car-\ncinoma patients.14,31,32 In recent studies the serum sE-CAD con-\ncentration was considered as an independent predictive factor for\nlong term survival33 and can predict tumor recurrency after cancer\nresection.15\nsP-CAD, also an 80-kD fragment, was measurable in different\nbiological ﬂuids like semen, milk and serum. The concentrations\nobserved in serum were ~20-fold lower than those of sE-CAD.\nHowever, no correlation was found with the sE-CAD levels or the\noccurrence of breast cancer.\nParadies and Grunwald9 reported the ﬁnding of a 90-kD frag-\nment of N-cadherin during retinal development of the chick, but\nuntil now no data were reported on sN-CAD in other biological\nﬂuids. Here we describe the presence of sN-CAD in serum by\nusing a direct ELISA or via immunoprecipitation and western blot.\nsN-CAD is a relatively stable molecule, and repeated freezing and\nthawing of the samples does not affect the levels. However, EDTA\nat concentrations higher then 0.5 mM reduces the serum concen-\ntration of sN-CAD dramatically, which conﬁrms the cadherin na-\nture of the measured protein. Measuring sN-CAD in a population\nwith no evidence of disease resulted in a median value of 99 ng/\nml which is ~10 times lower than the levels of sE-CAD and in the\nsame range as the sP-CAD value reported for a similar popula-\ntion.19 This could be explained by the number of cells expressing\neither E-, N- or P-cadherin, E-cadherin being the more abundant\nprotein. It is no surprise that low levels of soluble cadherins are\nfound in the control population, as the normal turn over of the\ncells will be responsible for the shedding of these soluble cadher-\nins. However when people are suffering from a disease, like diabe-\ntes or liver cirrhosis, higher amounts of sN-CAD are measured\n(data not shown), probably because more protease is shed. The\nsame has been observed for sE-CAD.34\nIn a population of cancer follow-up patients selected for ele-\nvated serum concentration of different tumor markers, we could\nﬁnd an almost 6-fold higher median concentration. In our popula-\ntion various types of cancers at different stages were included. In\nthe future a more stratiﬁed study on tumor types, grades and stages\nis mandatory to elucidate the clinical performance of sN-CAD as\na potential tumor marker. Therefore, we have started collecting\ndocumented serum samples from cancer patients. Prostate cancer\npatients are of particular interest, because we found a correlation\nbetween PSA and sN-CAD, and a recent report demonstrated that\nN-cadherin expression occurs in high grade prostate cancer and\ncorrelates signiﬁcantly with increasing Gleason patterns.1\nIt is currently not clear whether the circulating sN-CAD pos-\nsesses a biological function. We found recently that sN-CAD stim-\nulates angiogenesis in the chorioallantoic membrane assay and in\nthe rabbit cornea assay.10 Moreover, it exerts a motogenic action\non cancer cells in vitro, and we found evidence that the ﬁbroblast\ngrowth factor receptor is involved in both signalling processes.\nOur results indicate that sN-CAD is present in human serum, and\nthat signiﬁcantly higher amounts are present in cancer patients than\nin persons with no evidence of disease. Further clinical studies are\nneeded to evaluate whether this ﬁnding can be the basis for the de-\nvelopment of potentially useful tumor marker for cancer follow-up.",
         "Cell–cell adhesion molecules play an important role during\nembryogenesis, tumor invasion and metastasis. The cadherins,\nCa21-dependent cell–cell adhesion molecules, are essential for\nintercellular connections. E (epithelial)-cadherin is involved in\nnormal epithelial cell–cell interactions, but is often downregulated\nin epithelioid cancer cells. These cancer cells can switch cadherin\nexpression from E- to N- (neural),1 E- to P- (placental)2 or E- to\nOB- (osteoblast)-cadherin.3 This coincides with the transition\nfrom an epithelioid to a ﬁbroblastic phenotype, and is often corre-\nlated with invasiveness.4\nProteolysis can contribute to the impairment of the cadherin\nfunction. Several enzymes have been shown to be responsible for\nthe shedding of the extracellular domain. Cleavage of E-cadherin\nand shedding of an 80-kD fragment can be mediated by matrilysin,\nstromelysin-1,5,6 plasmin7 and a disintegrin and metalloproteinase\n10 (ADAM10).8 For N-cadherin, a 90-kD fragment can be\nreleased by the enzymes matrix metalloproteinase (MMP),9 plas-\nmin,10 ADAM10,11 membrane type 1-MMP (MT1-MMP)12 and\nMT5-MMP.13\nSoluble E-cadherin (sE-CAD) has been described in the circula-\ntion of cancer patients.14 In that study, the mean sE-CAD level,\ndetected by enzyme linked immunosorbent assay (ELISA), in se-\nrum was signiﬁcantly higher in the studied cancer patients (3.8 6\n2.36 lg/ml) when compared with the healthy controls (1.99 6 0.5\nlg/ml). In addition, sE-CAD was detected in serum of patients\nwith diabetes or hepatitis, but these values were not signiﬁcantly\ndifferent from the healthy controls. As a result, sE-CAD is now\npropagated as a marker for early prediction of tumor recurrence of\ngastric cancer.15 sE-CAD is also found in urine16 and the mean\nlevels in the urine of patients with bladder cancer are signiﬁcantly\nhigher than in the urine of healthy persons (1.6 vs. 0.9 lg/ml).17\nHigh levels of sE-CAD appear to be associated with positive cy-\ntology results and with bladder muscle invasion of transitional cell\ncarcinoma.18\nSoluble P-cadherin (sP-CAD) is found in serum19 of healthy\npersons as well as in patients with breast cancer, but no signiﬁcant\ndifference was found between these 2 populations. Remarkably,\nthe sP-CAD levels are 20-fold lower than the sE-CAD levels in se-\nrum. In addition, sP-CAD is also detected in milk20 of lactating\nwomen (in the range, 100–400 lg/ml, while sE-CAD was not\nfound in milk) and in semen of fertile and nonfertile men.21\nThree major observations made us speculate that soluble N-\ncadherin (sN-CAD) might be present in extracellular ﬂuids of can-\ncer patients: its upregulation in epithelioid cancers, for example in\nbreast, prostate and bladder carcinoma,1,3,22,23 its reactive upregu-\nlation in stromal cells of malignant tumors24 and the presence of\nmultiple extracellular proteases such as MMP25 that are reported\nto cleave the extracellular part of cadherins. Apart from its pres-\nence in the embryonic vitreous humor,9 no reports of sN-CAD in\nbiological ﬂuids are available. Here we report the presence of sN-\nCAD in biological ﬂuids. Moreover, we developed an ELISA\nmethod to quantify sN-CAD levels. We demonstrate the presence\nof sN-CAD in serum and found signiﬁcantly higher levels in can-\ncer patients when compared with individuals without evidence of\ndisease. sN-CAD was also present in seminal ﬂuid and urine but\nwas undetectable in synovial ﬂuid and spinal ﬂuid.\nMaterial and methods\nCell lines and preparation of medium containing\nsoluble N-cadherin\nS180-N-cadherin (ARM), mouse sarcoma cells (a gift from\nR.M. Me`ge, INSERM, Paris, France)26 were grown in D-MEM\n(Invitrogen, Merelbeke, Belgium) supplemented with 10% foetal\nbovine serum and penicillin, streptomycin and Fungizone (Invi-\ntrogen). The S180-NCAD cells, used as a source of sN-CAD, are\nS180 cells transfected with chicken cDNA encoding for N-cad-\nherin. The cells were incubated in a 100% water-saturated atmos-\nphere of 10% CO2. All cells were routinely tested for mycoplasma\ncontamination by staining with 40,60-diamidino-2-phenylindole\n(DAPI) and were found to be negative.\nSubconﬂuent monolayers were washed 3 times with phosphate\nbuffered saline (PBS) and were incubated with serum-free me-\ndium for 24 hr, washed another 3 times with PBS and incubated\nfor 48 hr with serum-free medium. The medium containing\nsN-CAD was harvested and centrifuged for 5 min at 250g and\nfor 15 min at 2,000g. The supernatant was ﬁltered through a\n0.22-lm ﬁlter and was concentrated 10 times (Amicon Ultra\n50 kD; Millipore, Bedford, MA) before use in ELISA or western\nblot.\nReagents and antibodies\nRecombinant human N-cadherin/Fc chimera was purchased\nfrom R&D Systems (Abingdon, UK) and used to set up a standard\ncurve with the ELISA method. Mouse GC-4 antibody (Sigma,\nSt. Louis, MO) against the extracellular part of N-cadherin was\nused for ELISA, immunoprecipitation and immunostaining of the\nwestern blot. The secondary antibodies were goat anti-mouse\nlinked to alkaline phosphatase (Sigma) for ELISA and anti-mouse\nlinked to horseradish peroxidase (Amersham Pharmacia Biotech,\nUppsala, Sweden) for western blot.\nCollection of human biological ﬂuids\nSerum samples were collected from 193 healthy subjects (with\nno evidence of disease, NED, and having C-reactive protein val-\nues below 0.6 mg/dl) and 179 patients during follow-up of cancer.\nThe selection of the cancer patients was based on the presence of\nhigh serum levels of an established tumor marker: prostate speciﬁc\nantigen (PSA) was above 50 ng/ml (prostate carcinoma, n 5 54),\nepisialin (CA15-3) above 50 U/ml (breast carcinoma, n 5 71), car-\ncino embryonic antigen (CEA) above 100 ng/ml (various carci-\nnoma, n 5 15), CA 125 above 50 U/ml (ovarium carcinoma, n 5 12)\nor CA19-9 above 1,000 U/ml (gastrointestinal carcinoma, n 5 12).\nAlso 8 sera samples from patients with chronic lymphocytic leu-\nkemia (CLL) were included. Venous blood samples were collected\nand centrifuged (after clotting) at 1,500g for 10 min. All sera were\nstored at 220C until sN-CAD assessment. Sera were residual\nsamples with an approval from the ethical committee of the Ghent\nUniversity Hospital (No. 2001/333). Other human biological ﬂuids\ncollected were urine (n 5 33), seminal ﬂuid (n 5 15), synovial\nﬂuid (n 5 2) and spinal ﬂuid (n 5 18). After reception of these\nsamples, they were centrifuged to remove cells and stored at\n220C.\nImmunoenzymometric assay for soluble N-cadherin\nsN-CAD levels were measured with a home-made ELISA.\nA dilution series of recombinant N-cadherin from 1 to 1,000 ng/ml\nwas prepared. All serum samples were diluted 5 times in PBS con-\ntaining 0.1% bovine serum albumin (BSA). A 96-well immuno-\nplate (Nunc) was coated with 75 ll of the diluted sample overnight\nat 4C. The wells were washed with PBS/0.05% Tween-20 and\nquenched at 37C with PBS/1% BSA for 1 hr. Next, the plates\nwere washed again (4 times) and incubated with the primary anti-\nbody (GC-4, 1/200 in PBS/0.1% BSA) at 37C for 2 hr. The plates\nwere washed (4 times) and subsequently incubated with a mouse\nsecondary antibody linked to alkaline phosphatase (Sigma; 1/3,000).\nThe substrate p-nitrophenylphosphate (Sigma, St. Louis, MO) was\nadded to the plates and after 30 min the optical density of each\nwell was determined with a microplate reader (Molecular Devices,\nWokingham, UK) at 405 nm. In control experiments we omitted\nthe primary antibody. Measurements were done at least in dupli-\ncate for each sample, and the mean value was calculated.\nImmunoprecipitation and western blotting\nSerum samples were depleted of albumin and immunoglobulins\nusing the AurumTM Serum Protein Mini kit (Bio-RAD, Hercules,\nCA). The eluted fractions were used to start the immunoprecipita-\ntions. For the immunoprecipitation, equal amounts of serum sam-\nples were incubated with Dynabeads Protein G (Dynal Biotech,\nOslon, Norway) for 30 min. After discarding the beads, the super-\nnatant was incubated with primary antibody for 3 hr at 4C, fol-\nlowed by incubation with Dynabeads Protein G for 1 hr. Next,\nsample buffer (Laemmli) with 5% 2-mercaptoethanol and 0.012%\nbromophenol blue was added and supernatant was separated on\n8% SDS-PAGE and blotted onto a nitrocellulose membrane\n(Amersham Pharmacia Biotech). Quenching and immunostaining\nwere done in 5% nonfat dry milk in PBS containing 0.5% Tween-\n20. The membranes were quenched for 30 min, incubated with pri-\nmary antibody (1/1,000) for 1 hr, washed 4 times for 10 min, incu-\nbated with horseradish peroxidase-conjugated secondary antibody\n(1/3,000) for 45 min, and washed 6 times for 5 min. Detection\nwas carried out using enhanced chemiluminescence reagent\n(Amersham Pharmacia Biotech) as a substrate.\nStatistical analysis\nData were collected and analyzed with SPSS (SPSS, Chicago).\nComparisons were performed with the nonparametric Mann–\nWhitney U-test because the populations were not normally distrib-\nuted. Differences were considered signiﬁcant at p values <0.001.\nSpearman correlation coefﬁcients were used to examine the corre-\nlation between sN-CAD and the tumor markers like PSA, CA15-3\nand CEA.\nResults\nELISA for the detection of soluble N-cadherin in serum\nsamples and other biological ﬂuids\nA recombinant N-cadherin chimera (RECN), which consists of\namino acids 1–724 of human N-cadherin fused to the carboxyter-\nminal end of the Fc region of human IgG1, migrated at 135 kD in\nSDS-PAGE, and was reactive with the GC-4 antibody in ELISA\nand immunostaining of western blot. This recombinant protein\nwas used as an assay standard for quantiﬁcation of antigen levels\nin several biological ﬂuids. A dilution series of RECN was made\nfrom 1 to 1,000 ng/ml and 75 ll of each dilution was incubated at\n4C overnight in an immunoplate in order to coat the wells. On\nthe next day, the immunoplate was quenched with 1% BSA, fol-\nlowed by incubation at 37C with the primary antibody GC-4 for\n2 hr. Next, the immunoplate was incubated with the goat anti-\nmouse antibody linked to alkaline phosphatase, and ﬁnally the\nchromogen p-nitrophenylphosphate was added to the plate. After\n30 min, the optical density was read at 405 nm (Fig. 1). Dilution\ncurves from 2 serum samples were linear, suggesting that the same\nimmunoreactive substance is measured at the different dilutions\n(Fig. 2). We found that a dilution factor of 5 of all samples gave\nthe best read out when compared with the standard curve. The pre-\ncision of the ELISA was tested assaying serum samples 8 times\nfrom 5 healthy donors, with a mean sN-CAD value between 50\nand 300 ng/ml. Intra-assay and inter-assay coefﬁcients of variation\nwere 6 and 9%, respectively.\nSerum samples from 193 people with no evidence of disease\n(NED) were collected and tested in the ELISA. sN-CAD concen-\ntrations had a median value of 99 ng/ml (range, 0–1,130 ng/ml).\nSerum samples (179) from cancer follow-up patients were selected\non the basis of elevated tumor maker concentration of CA15-3,\nCEA, PSA, CA19-9 and CA 125 and CLL patients. The results are\nshown in a box plot, and compared with the NED population\n(Fig. 3a). Signiﬁcantly elevated levels were observed in the cancer\npatient group with a median value of 584 ng/ml (range, 0–2,713\nng/ml) (Mann–Witney U-test, p < 0.001). Neither the frequency\ndistribution curve of the NED population (Fig. 3b) nor that of the\ncancer patients group (Fig. 3c) showed a normal distribution as\nassessed by the Kolmogorov–Smirnov test.\nUsing the nonparametric Spearman correlation coefﬁcient, the\nsN-CAD serum concentrations showed no correlation with the\ntumor markers CA15-3 (r 5 20.086) and CEA (r 5 20.273).\nA signiﬁcant, but weak correlation was found between PSA and\nsN-CAD levels (r 5 0.254, p < 0.05) (Fig. 4a). The PSA posi-\ntive population (n 5 54) had signiﬁcant higher sN-CAD values\n(range, 0–1,445 ng/ml; median, 675 ng/ml) when compared with\nthe NED population (range, 0–1,130 ng/ml; median, 99 ng/ml)\n(p < 0.001) (Fig. 4b).\nOther biological ﬂuids were also tested for the presence of\nsN-CAD: we found sN-CAD in seminal ﬂuid samples (range,\n1,063–2,553 ng/ml) and in urine (range, 0–655 ng/ml), but none in\nsynovial ﬂuid or spinal ﬂuid (data not shown).\nImmunoblot analysis of serum soluble N-cadherin\nIn immunoblot analysis after 8% SDS-PAGE, we separated the\nimmuno-isolated proteins, reactive with the GC-4 antibody, from\nserum samples of 3 subjects with no evidence of disease and 4\ncancer patients (2 breast cancer patients and 2 prostate cancer\npatients). Serum samples were ﬁrst depleted of albumin and\nimmunoglobulins. As positive control we used medium containing\nsN-CAD, which was harvested from S180 cells transfected with\ncDNA of N-cadherin (S180-NCAD). Various amounts of the 90-\nkD N-cadherin fragment (sN-CAD) were detected in the serum\nsamples by the GC-4 antibody (Fig. 5). Moreover, these could be\ncorrelated with the concentrations measured by ELISA. The mo-\nlecular weight of the sN-CAD was in all samples identical. The\nupper band in lane 6 is an aspeciﬁc band, recognized by the sec-\nondary antibody (anti-mouse linked to HRP) alone.\nStability of the soluble N-cadherin in serum\nTo investigate the role of calcium in the stabilization of sN-\nCAD, S180-NCAD cells were treated with EDTA (0.125–3 mM)\nfor 24 hr. The release of sN-CAD remained the same as with\nuntreated cells up to a concentration of 0.5 mM of EDTA in the\nculture medium (Fig. 6a). Treatment with higher concentrations of\nEDTA made sN-CAD less detectable in the culture medium. Con-\ncentrations of EDTA higher than 0.75 mM are known to chelate\nall divalent cations (for e.g. Ca21) in the medium. Adding EDTA\ntrate). The plasma concentrations of sN-CAD were reduced to 30\nand 45% respectively (Fig. 6b) when compared with serum con-\ncentrations. For this reason, we used serum from clotted blood to\nmeasure sN-CAD.\nSerum with a known concentration of sN-CAD was mixed (1:1)\nwith serum containing high concentrations of hemoglobin (615 g/\ndl), bilirubin (610 mg/dl) or triglycerides (61,500 mg/l), affect-\ning the aspect of the serum. These factors did not interfere with\nthe detection of sN-CAD (data not shown). Repeated freezing and\nthawing (5 times) of the biological ﬂuids did not affect the stabil-\nity of sN-CAD in serum samples. Storage of the samples at 4C\ndid not change the amount of sN-CAD. However, storage of the\nsamples for 2 days at room temperature diminished the concentra-\ntion of sN-CAD (49% to control) (Fig. 6c).",
         "",
         "",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1217413%26volume%3d176%26issue%3d3%26page%3d1645%26filename%3d1645.pdf%26languageId%3d",
         "Classical cadherins such as E-, P- and N-cadherin are transmem-\nbrane proteins that mediate cell–cell adhesion, and are important\nin embryogenesis, maintenance of tissue integrity and cancer.\nProteolytic shedding of the extracellular domain results in the\ngeneration of soluble E-, P- or N-cadherin ectodomains. Circulat-\ning soluble E- and P-cadherin have been described in the serum,\nand elevated levels were detected in cancer patients when com-\npared with healthy persons. Here we report the presence of solu-\nble N-cadherin, a 90-kD protein fragment, in the serum of both\nhealthy persons and cancer patients, using a direct ELISA and\nimmunoprecipitation. A correlation was found between prostate\nspeciﬁc antigen and soluble N-cadherin, and signiﬁcantly elevated\nlevels were detected in prostate cancer follow-up patients. The N-\ncadherin protein is neo-expressed by carcinomas of the prostate,\nand is responsible for epithelial to ﬁbroblastic transition. This is\nreﬂected by the higher concentrations of soluble N-cadherin in\nprostate cancer patients than in healthy persons.\n' 2006 Wiley-Liss, Inc.\nCell–cell adhesion molecules play an important role during\nembryogenesis, tumor invasion and metastasis. The cadherins,\nCa21-dependent cell–cell adhesion molecules, are essential for\nintercellular connections. E (epithelial)-cadherin is involved in\nnormal epithelial cell–cell interactions, but is often downregulated\nin epithelioid cancer cells. These cancer cells can switch cadherin\nexpression from E- to N- (neural),1 E- to P- (placental)2 or E- to\nOB- (osteoblast)-cadherin.3 This coincides with the transition\nfrom an epithelioid to a ﬁbroblastic phenotype, and is often corre-\nlated with invasiveness.4\nProteolysis can contribute to the impairment of the cadherin\nfunction. Several enzymes have been shown to be responsible for\nthe shedding of the extracellular domain. Cleavage of E-cadherin\nand shedding of an 80-kD fragment can be mediated by matrilysin,\nstromelysin-1,5,6 plasmin7 and a disintegrin and metalloproteinase\n10 (ADAM10).8 For N-cadherin, a 90-kD fragment can be\nreleased by the enzymes matrix metalloproteinase (MMP),9 plas-\nmin,10 ADAM10,11 membrane type 1-MMP (MT1-MMP)12 and\nMT5-MMP.13\nSoluble E-cadherin (sE-CAD) has been described in the circula-\ntion of cancer patients.14 In that study, the mean sE-CAD level,\ndetected by enzyme linked immunosorbent assay (ELISA), in se-\nrum was signiﬁcantly higher in the studied cancer patients (3.8 6\n2.36 lg/ml) when compared with the healthy controls (1.99 6 0.5\nlg/ml). In addition, sE-CAD was detected in serum of patients\nwith diabetes or hepatitis, but these values were not signiﬁcantly\ndifferent from the healthy controls. As a result, sE-CAD is now\npropagated as a marker for early prediction of tumor recurrence of\ngastric cancer.15 sE-CAD is also found in urine16 and the mean\nlevels in the urine of patients with bladder cancer are signiﬁcantly\nhigher than in the urine of healthy persons (1.6 vs. 0.9 lg/ml).17\nHigh levels of sE-CAD appear to be associated with positive cy-\ntology results and with bladder muscle invasion of transitional cell\ncarcinoma.18\nSoluble P-cadherin (sP-CAD) is found in serum19 of healthy\npersons as well as in patients with breast cancer, but no signiﬁcant\ndifference was found between these 2 populations. Remarkably,\nthe sP-CAD levels are 20-fold lower than the sE-CAD levels in se-\nrum. In addition, sP-CAD is also detected in milk20 of lactating\nwomen (in the range, 100–400 lg/ml, while sE-CAD was not\nfound in milk) and in semen of fertile and nonfertile men.21\nThree major observations made us speculate that soluble N-\ncadherin (sN-CAD) might be present in extracellular ﬂuids of can-\ncer patients: its upregulation in epithelioid cancers, for example in\nbreast, prostate and bladder carcinoma,1,3,22,23 its reactive upregu-\nlation in stromal cells of malignant tumors24 and the presence of\nmultiple extracellular proteases such as MMP25 that are reported\nto cleave the extracellular part of cadherins. Apart from its pres-\nence in the embryonic vitreous humor,9 no reports of sN-CAD in\nbiological ﬂuids are available. Here we report the presence of sN-\nCAD in biological ﬂuids. Moreover, we developed an ELISA\nmethod to quantify sN-CAD levels. We demonstrate the presence\nof sN-CAD in serum and found signiﬁcantly higher levels in can-\ncer patients when compared with individuals without evidence of\ndisease. sN-CAD was also present in seminal ﬂuid and urine but\nwas undetectable in synovial ﬂuid and spinal ﬂuid.\nMaterial and methods\nCell lines and preparation of medium containing\nsoluble N-cadherin\nS180-N-cadherin (ARM), mouse sarcoma cells (a gift from\nR.M. Me`ge, INSERM, Paris, France)26 were grown in D-MEM\n(Invitrogen, Merelbeke, Belgium) supplemented with 10% foetal\nbovine serum and penicillin, streptomycin and Fungizone (Invi-\ntrogen). The S180-NCAD cells, used as a source of sN-CAD, are\nS180 cells transfected with chicken cDNA encoding for N-cad-\nherin. The cells were incubated in a 100% water-saturated atmos-\nphere of 10% CO2. All cells were routinely tested for mycoplasma\ncontamination by staining with 40,60-diamidino-2-phenylindole\n(DAPI) and were found to be negative.\nSubconﬂuent monolayers were washed 3 times with phosphate\nbuffered saline (PBS) and were incubated with serum-free me-\ndium for 24 hr, washed another 3 times with PBS and incubated\nfor 48 hr with serum-free medium. The medium containing\nsN-CAD was harvested and centrifuged for 5 min at 250g and\nfor 15 min at 2,000g. The supernatant was ﬁltered through a\n0.22-lm ﬁlter and was concentrated 10 times (Amicon Ultra\n50 kD; Millipore, Bedford, MA) before use in ELISA or western\nblot.\nReagents and antibodies\nRecombinant human N-cadherin/Fc chimera was purchased\nfrom R&D Systems (Abingdon, UK) and used to set up a standard\ncurve with the ELISA method. Mouse GC-4 antibody (Sigma,\nSt. Louis, MO) against the extracellular part of N-cadherin was\nused for ELISA, immunoprecipitation and immunostaining of the\nwestern blot. The secondary antibodies were goat anti-mouse\nlinked to alkaline phosphatase (Sigma) for ELISA and anti-mouse\nlinked to horseradish peroxidase (Amersham Pharmacia Biotech,\nUppsala, Sweden) for western blot.\nCollection of human biological ﬂuids\nSerum samples were collected from 193 healthy subjects (with\nno evidence of disease, NED, and having C-reactive protein val-\nues below 0.6 mg/dl) and 179 patients during follow-up of cancer.\nThe selection of the cancer patients was based on the presence of\nhigh serum levels of an established tumor marker: prostate speciﬁc\nantigen (PSA) was above 50 ng/ml (prostate carcinoma, n 5 54),\nepisialin (CA15-3) above 50 U/ml (breast carcinoma, n 5 71), car-\ncino embryonic antigen (CEA) above 100 ng/ml (various carci-\nnoma, n 5 15), CA 125 above 50 U/ml (ovarium carcinoma, n 5 12)\nor CA19-9 above 1,000 U/ml (gastrointestinal carcinoma, n 5 12).\nAlso 8 sera samples from patients with chronic lymphocytic leu-\nkemia (CLL) were included. Venous blood samples were collected\nand centrifuged (after clotting) at 1,500g for 10 min. All sera were\nstored at 220C until sN-CAD assessment. Sera were residual\nsamples with an approval from the ethical committee of the Ghent\nUniversity Hospital (No. 2001/333). Other human biological ﬂuids\ncollected were urine (n 5 33), seminal ﬂuid (n 5 15), synovial\nﬂuid (n 5 2) and spinal ﬂuid (n 5 18). After reception of these\nsamples, they were centrifuged to remove cells and stored at\n220C.\nImmunoenzymometric assay for soluble N-cadherin\nsN-CAD levels were measured with a home-made ELISA.\nA dilution series of recombinant N-cadherin from 1 to 1,000 ng/ml\nwas prepared. All serum samples were diluted 5 times in PBS con-\ntaining 0.1% bovine serum albumin (BSA). A 96-well immuno-\nplate (Nunc) was coated with 75 ll of the diluted sample overnight\nat 4C. The wells were washed with PBS/0.05% Tween-20 and\nquenched at 37C with PBS/1% BSA for 1 hr. Next, the plates\nwere washed again (4 times) and incubated with the primary anti-\nbody (GC-4, 1/200 in PBS/0.1% BSA) at 37C for 2 hr. The plates\nwere washed (4 times) and subsequently incubated with a mouse\nsecondary antibody linked to alkaline phosphatase (Sigma; 1/3,000).\nThe substrate p-nitrophenylphosphate (Sigma, St. Louis, MO) was\nadded to the plates and after 30 min the optical density of each\nwell was determined with a microplate reader (Molecular Devices,\nWokingham, UK) at 405 nm. In control experiments we omitted\nthe primary antibody. Measurements were done at least in dupli-\ncate for each sample, and the mean value was calculated.\nImmunoprecipitation and western blotting\nSerum samples were depleted of albumin and immunoglobulins\nusing the AurumTM Serum Protein Mini kit (Bio-RAD, Hercules,\nCA). The eluted fractions were used to start the immunoprecipita-\ntions. For the immunoprecipitation, equal amounts of serum sam-\nples were incubated with Dynabeads Protein G (Dynal Biotech,\nOslon, Norway) for 30 min. After discarding the beads, the super-\nnatant was incubated with primary antibody for 3 hr at 4C, fol-\nlowed by incubation with Dynabeads Protein G for 1 hr. Next,\nsample buffer (Laemmli) with 5% 2-mercaptoethanol and 0.012%\nbromophenol blue was added and supernatant was separated on\n8% SDS-PAGE and blotted onto a nitrocellulose membrane\n(Amersham Pharmacia Biotech). Quenching and immunostaining\nwere done in 5% nonfat dry milk in PBS containing 0.5% Tween-\n20. The membranes were quenched for 30 min, incubated with pri-\nmary antibody (1/1,000) for 1 hr, washed 4 times for 10 min, incu-\nbated with horseradish peroxidase-conjugated secondary antibody\n(1/3,000) for 45 min, and washed 6 times for 5 min. Detection\nwas carried out using enhanced chemiluminescence reagent\n(Amersham Pharmacia Biotech) as a substrate.\nStatistical analysis\nData were collected and analyzed with SPSS (SPSS, Chicago).\nComparisons were performed with the nonparametric Mann–\nWhitney U-test because the populations were not normally distrib-\nuted. Differences were considered signiﬁcant at p values <0.001.\nSpearman correlation coefﬁcients were used to examine the corre-\nlation between sN-CAD and the tumor markers like PSA, CA15-3\nand CEA.\nResults\nELISA for the detection of soluble N-cadherin in serum\nsamples and other biological ﬂuids\nA recombinant N-cadherin chimera (RECN), which consists of\namino acids 1–724 of human N-cadherin fused to the carboxyter-\nminal end of the Fc region of human IgG1, migrated at 135 kD in\nSDS-PAGE, and was reactive with the GC-4 antibody in ELISA\nand immunostaining of western blot. This recombinant protein\nwas used as an assay standard for quantiﬁcation of antigen levels\nin several biological ﬂuids. A dilution series of RECN was made\nfrom 1 to 1,000 ng/ml and 75 ll of each dilution was incubated at\n4C overnight in an immunoplate in order to coat the wells. On\nthe next day, the immunoplate was quenched with 1% BSA, fol-\nlowed by incubation at 37C with the primary antibody GC-4 for\n2 hr. Next, the immunoplate was incubated with the goat anti-\nmouse antibody linked to alkaline phosphatase, and ﬁnally the\nchromogen p-nitrophenylphosphate was added to the plate. After\n30 min, the optical density was read at 405 nm (Fig. 1). Dilution\ncurves from 2 serum samples were linear, suggesting that the same\nimmunoreactive substance is measured at the different dilutions\n(Fig. 2). We found that a dilution factor of 5 of all samples gave\nthe best read out when compared with the standard curve. The pre-\ncision of the ELISA was tested assaying serum samples 8 times\nfrom 5 healthy donors, with a mean sN-CAD value between 50\nand 300 ng/ml. Intra-assay and inter-assay coefﬁcients of variation\nwere 6 and 9%, respectively.\nSerum samples from 193 people with no evidence of disease\n(NED) were collected and tested in the ELISA. sN-CAD concen-\ntrations had a median value of 99 ng/ml (range, 0–1,130 ng/ml).\nSerum samples (179) from cancer follow-up patients were selected\non the basis of elevated tumor maker concentration of CA15-3,\nCEA, PSA, CA19-9 and CA 125 and CLL patients. The results are\nshown in a box plot, and compared with the NED population\n(Fig. 3a). Signiﬁcantly elevated levels were observed in the cancer\npatient group with a median value of 584 ng/ml (range, 0–2,713\nng/ml) (Mann–Witney U-test, p < 0.001). Neither the frequency\ndistribution curve of the NED population (Fig. 3b) nor that of the\ncancer patients group (Fig. 3c) showed a normal distribution as\nassessed by the Kolmogorov–Smirnov test.\nUsing the nonparametric Spearman correlation coefﬁcient, the\nsN-CAD serum concentrations showed no correlation with the\ntumor markers CA15-3 (r 5 20.086) and CEA (r 5 20.273).\nA signiﬁcant, but weak correlation was found between PSA and\nsN-CAD levels (r 5 0.254, p < 0.05) (Fig. 4a). The PSA posi-\ntive population (n 5 54) had signiﬁcant higher sN-CAD values\n(range, 0–1,445 ng/ml; median, 675 ng/ml) when compared with\nthe NED population (range, 0–1,130 ng/ml; median, 99 ng/ml)\n(p < 0.001) (Fig. 4b).\nOther biological ﬂuids were also tested for the presence of\nsN-CAD: we found sN-CAD in seminal ﬂuid samples (range,\n1,063–2,553 ng/ml) and in urine (range, 0–655 ng/ml), but none in\nsynovial ﬂuid or spinal ﬂuid (data not shown).\nImmunoblot analysis of serum soluble N-cadherin\nIn immunoblot analysis after 8% SDS-PAGE, we separated the\nimmuno-isolated proteins, reactive with the GC-4 antibody, from\nserum samples of 3 subjects with no evidence of disease and 4\ncancer patients (2 breast cancer patients and 2 prostate cancer\npatients). Serum samples were ﬁrst depleted of albumin and\nimmunoglobulins. As positive control we used medium containing\nsN-CAD, which was harvested from S180 cells transfected with\ncDNA of N-cadherin (S180-NCAD). Various amounts of the 90-\nkD N-cadherin fragment (sN-CAD) were detected in the serum\nsamples by the GC-4 antibody (Fig. 5). Moreover, these could be\ncorrelated with the concentrations measured by ELISA. The mo-\nlecular weight of the sN-CAD was in all samples identical. The\nupper band in lane 6 is an aspeciﬁc band, recognized by the sec-\nondary antibody (anti-mouse linked to HRP) alone.\nStability of the soluble N-cadherin in serum\nTo investigate the role of calcium in the stabilization of sN-\nCAD, S180-NCAD cells were treated with EDTA (0.125–3 mM)\nfor 24 hr. The release of sN-CAD remained the same as with\nuntreated cells up to a concentration of 0.5 mM of EDTA in the\nculture medium (Fig. 6a). Treatment with higher concentrations of\nEDTA made sN-CAD less detectable in the culture medium. Con-\ncentrations of EDTA higher than 0.75 mM are known to chelate\nall divalent cations (for e.g. Ca21) in the medium. Adding EDTA\ntrate). The plasma concentrations of sN-CAD were reduced to 30\nand 45% respectively (Fig. 6b) when compared with serum con-\ncentrations. For this reason, we used serum from clotted blood to\nmeasure sN-CAD.\nSerum with a known concentration of sN-CAD was mixed (1:1)\nwith serum containing high concentrations of hemoglobin (615 g/\ndl), bilirubin (610 mg/dl) or triglycerides (61,500 mg/l), affect-\ning the aspect of the serum. These factors did not interfere with\nthe detection of sN-CAD (data not shown). Repeated freezing and\nthawing (5 times) of the biological ﬂuids did not affect the stabil-\nity of sN-CAD in serum samples. Storage of the samples at 4C\ndid not change the amount of sN-CAD. However, storage of the\nsamples for 2 days at room temperature diminished the concentra-\ntion of sN-CAD (49% to control) (Fig. 6c).\nDiscussion\nTumor invasion is characterized in part by the ability of cancer\ncells to downregulate cell–cell adhesion and to invade into the sur-\nrounding tissues. In this respect, the progression of a carcinoma is\noften characterized by a switch from E- to N-cadherin. This switch\nin cadherin type expression has been observed in different malig-\nnant tissues, like breast, prostate, bladder, skin and colon carci-\nnoma.1,3,22,23 This switch also coincides with the transition of the\ncells from an epithelial to a ﬁbroblastic phenotype, and invasive-\nness of the cancer cells. By extension in the tumor microenviron-\nment, N-cadherin expression is not exclusively found on the cancer\ncells, but is also reactively overexpressed on the surface of sur-\nrounding ﬁbroblasts and endothelial cells.24 During cancer pro-\ngression a multitude of extracellular proteinases are present and\nto serum samples at a ﬁnal concentration of 1 mM dramatically\nreduced the amount of sN-CAD to 50%. So, this shows that we\ndetect the cadherin protein. This effect was also observed when\nblood from the same individual was collected in tubes containing\nEDTA (EDTA K2 or EDTA K3, 8%) or citrate (3.2% sodium ci-\nare now known as the cancer degradome.27,28 These proteinases\nare responsible for the shedding of several transmembrane protein\nectodomains, including cadherins and tyrosine kinase receptors.29\nIn the eighties, Damsky et al.30 reported the cleavage of E-\ncadherin in cell culture experiments, with the release of an 80-kD\nextracellular fragment (sE-CAD). Later, this fragment was also\nfound in different biological ﬂuids, like serum14 and urine,17 and\nelevated concentrations were described in the serum of gastric car-\ncinoma patients.14,31,32 In recent studies the serum sE-CAD con-\ncentration was considered as an independent predictive factor for\nlong term survival33 and can predict tumor recurrency after cancer\nresection.15\nsP-CAD, also an 80-kD fragment, was measurable in different\nbiological ﬂuids like semen, milk and serum. The concentrations\nobserved in serum were ~20-fold lower than those of sE-CAD.\nHowever, no correlation was found with the sE-CAD levels or the\noccurrence of breast cancer.\nParadies and Grunwald9 reported the ﬁnding of a 90-kD frag-\nment of N-cadherin during retinal development of the chick, but\nuntil now no data were reported on sN-CAD in other biological\nﬂuids. Here we describe the presence of sN-CAD in serum by\nusing a direct ELISA or via immunoprecipitation and western blot.\nsN-CAD is a relatively stable molecule, and repeated freezing and\nthawing of the samples does not affect the levels. However, EDTA\nat concentrations higher then 0.5 mM reduces the serum concen-\ntration of sN-CAD dramatically, which conﬁrms the cadherin na-\nture of the measured protein. Measuring sN-CAD in a population\nwith no evidence of disease resulted in a median value of 99 ng/\nml which is ~10 times lower than the levels of sE-CAD and in the\nsame range as the sP-CAD value reported for a similar popula-\ntion.19 This could be explained by the number of cells expressing\neither E-, N- or P-cadherin, E-cadherin being the more abundant\nprotein. It is no surprise that low levels of soluble cadherins are\nfound in the control population, as the normal turn over of the\ncells will be responsible for the shedding of these soluble cadher-\nins. However when people are suffering from a disease, like diabe-\ntes or liver cirrhosis, higher amounts of sN-CAD are measured\n(data not shown), probably because more protease is shed. The\nsame has been observed for sE-CAD.34\nIn a population of cancer follow-up patients selected for ele-\nvated serum concentration of different tumor markers, we could\nﬁnd an almost 6-fold higher median concentration. In our popula-\ntion various types of cancers at different stages were included. In\nthe future a more stratiﬁed study on tumor types, grades and stages\nis mandatory to elucidate the clinical performance of sN-CAD as\na potential tumor marker. Therefore, we have started collecting\ndocumented serum samples from cancer patients. Prostate cancer\npatients are of particular interest, because we found a correlation\nbetween PSA and sN-CAD, and a recent report demonstrated that\nN-cadherin expression occurs in high grade prostate cancer and\ncorrelates signiﬁcantly with increasing Gleason patterns.1\nIt is currently not clear whether the circulating sN-CAD pos-\nsesses a biological function. We found recently that sN-CAD stim-\nulates angiogenesis in the chorioallantoic membrane assay and in\nthe rabbit cornea assay.10 Moreover, it exerts a motogenic action\non cancer cells in vitro, and we found evidence that the ﬁbroblast\ngrowth factor receptor is involved in both signalling processes.\nOur results indicate that sN-CAD is present in human serum, and\nthat signiﬁcantly higher amounts are present in cancer patients than\nin persons with no evidence of disease. Further clinical studies are\nneeded to evaluate whether this ﬁnding can be the basis for the de-\nvelopment of potentially useful tumor marker for cancer follow-up.",
         "Soluble N-cadherin fragment promotes angiogenesis;Membrane Localization of Membrane Type 5 Matrix Metalloproteinase by AMPA Receptor Binding Protein and Cleavage of Cadherins;N‐cadherin switching occurs in high Gleason grade prostate cancer;Differential exoprotease activities confer tumor-specific serum peptidome patterns.;N-cadherin Expression in Breast Cancer: Correlation with an Aggressive Histologic Variant – Invasive Micropapillary Carcinoma;Early prediction of tumor recurrence after curative resection of gastric carcinoma by measuring soluble E‐cadherin;Urinary levels of soluble e-cadherin in the detection of transitional cell carcinoma of the urinary bladder.;ADAM10 mediates E-cadherin shedding and regulates epithelial cell-cell adhesion, migration, and beta-catenin translocation.;Identification of degradome components associated with prostate cancer progression by expression analysis of human prostatic tissues;ADAM10 cleavage of N‐cadherin and regulation of cell–cell adhesion and β‐catenin nuclear signalling;Soluble p-cadherin found in human semen.;Regulation of matrix biology by matrix metalloproteinases.;Critical role of N-cadherin in myofibroblast invasion and migration in vitro stimulated by colon-cancer-cell-derived TGF-β or wounding;N-cadherin in the spotlight of cell-cell adhesion, differentiation, embryogenesis, invasion and signalling.;Soluble E-cadherin is an independent pretherapeutic factor for long-term survival in gastric cancer.;New functions for the matrix metalloproteinases in cancer progression;Plasmin Produces an E-Cadherin Fragment That Stimulates Cancer Cell Invasion;Matrilysin mediates extracellular cleavage of E-cadherin from prostate cancer cells: a key mechanism in hepatocyte growth factor/scatter factor-induced cell-cell dissociation and in vitro invasion.;Soluble E-cadherin is a valid prognostic marker in gastric carcinoma;Lack of correlation between serum levels of E- and P-cadherin fragments and the presence of breast cancer.;Cadherin switching in human prostate cancer progression.;Urinary concentrations of the soluble adhesion molecule E-cadherin and total protein in patients with bladder cancer;Characterization of soluble E-cadherin as a disease marker in gastric cancer patients.;Soluble E-cadherin concentrations in patients with systemic inflammatory response syndrome and multiorgan dysfunction syndrome.;Soluble forms of the adhesion molecule E-cadherin in urine.;Soluble E-cadherin fragments increased in circulation of cancer patients.;Purification and characterization of NCAD90, a Soluble endogenous form of N‐cadherin, which is generated by proteolysis during retinal development and retains adhesive and neurite‐promoting function;Construction of epithelioid sheets by transfection of mouse sarcoma cells with cDNAs for chicken cell adhesion molecules.;Identification and purification of a cell surface glycoprotein mediating intercellular adhesion in embryonic and adult tissue;Ischemia-induced cleavage of cadherins in NRK cells requires MT1-MMP (MMP-14).;Soluble fragment of P‐cadherin adhesion protein found in human milk;Expression of classic cadherins type I in urothelial neoplastic progression.;The E-cadherin/catenin complex in invasion: the role of ectodomain shedding.;Cancer Metastasis, Molecular and Cellular Mechanisms and Clinical Intervention;Plasmin produces an Ecadherin fragment that stimulates cancer cell invasion;Monden M. Characterization of soluble E-cadherin as a disease marker in gastric cancer patients;nary concentrations of the soluble adhesion molecule E - cadherin and total protein in patients with bladder cancer;Identification and purification of a",
         "Soluble N‐cadherin in human biological fluids",
         "None"
        ],
        [
         "27",
         "0018adec76441851263e5f7d8195a50ae353707b",
         "Severe acute respiratory syndrome (SARS) is an infectious disease caused by the human coronavirus, SARS-CoV. The main viral protease, SARS 3CLpro, is a validated target for the development of antiviral therapies. Since the enzyme is a homodimer and the individual monomers are inactive, two approaches are being used to develop inhibitors: enzyme activity inhibitors that target the active site and dimerization inhibitors. Dimerization inhibitors are usually targeted to the dimerization interface and need to compete with the attractive forces between subunits to be effective. In this paper, we show that the dimerization of SARS 3CLpro is also under allosteric control and that additional and energetically more favorable target sites away from the dimerization interface may also lead to subunit dissociation. We previously identified a cluster of conserved serine residues (Ser139, Ser144, and Ser147) located adjacent to the active site of 3CLpro that could effectively be targeted to inactivate the protease [Bacha, U et al. (2004) Biochemistry 43, 4906-4912]. Mutation of any of these serine residues to alanine had a debilitating effect on the catalytic activity of 3CLpro. In particular, the mutation of Ser147, which does not make any contact with the opposing subunit and is located approximately 9 A away from the dimer interface, totally inhibited dimerization and resulted in a complete loss of enzymatic activity. The finding that residues away from the dimer interface are able to control dimerization defines alternative targets for the design of dimerization inhibitors.",
         "Jennifer Barrila,Usman M. Bacha,E. Freire",
         "Jennifer Barrila,Usman M. Bacha,E. Freire",
         "https://europepmc.org/articles/pmc2570436?pdf=render",
         "Severe acute respiratory syndrome (SARS) is an infectious disease caused by the human coronavirus,\nSARS-CoV. The main viral protease, SARS 3CLpro is a validated target for the development of\nantiviral therapies. Since the enzyme is a homodimer and the individual monomers are inactive, two\napproaches are being used to develop inhibitors: enzyme activity inhibitors that target the active site\nand dimerization inhibitors. Dimerization inhibitors are usually targeted to the dimerization interface\nand need to compete with the attractive forces between subunits in order to be effective. In this paper,\nwe show that the dimerization of SARS 3CLpro is also under allosteric control and that additional\nand energetically more favorable target sites away from the dimerization interface may also lead to\nsubunit dissociation. We previously identified a cluster of conserved serine residues (Ser139, Ser144\nand Ser147) located adjacent to the active site of 3CLpro that could effectively be targeted to inactivate\nthe protease. Mutation of any of these serine residues to alanine had a debilitating effect on the\ncatalytic activity of 3CLpro. In particular, the mutation of Ser147, which does not make any contact\nwith the opposing subunit and is located approximately 9Å away from the dimer interface, totally\ninhibited dimerization and resulted in a complete loss of enzymatic activity. The finding that residues\naway from the dimer interface are able to control dimerization, defines alternative targets for the\ndesign of dimerization inhibitors.",
         "We previously identified a highly conserved cluster of serine residues: Ser139, Ser144 and\nSer147, which lie adjacent to the active site of SARS 3CLpro (5). The location of this serine\ncluster in the structure of 3CLpro is shown in Figure 1A. Serines 139 and 147 are absolutely\nconserved in all known coronavirus 3CL proteases, while Ser144 is conserved in 55% of the\nreported sequences.\nThe role played by the three serine residues in the catalytic activity of SARS 3CLpro was\nevaluated by generating serine to alanine mutants at the three positions. The catalytic activities\nof wild type 3CLpro and the mutants S139A, S144A and S147A were measured at 1 μM enzyme\nconcentration using a fluorescence-based cleavage assay in order to obtain the catalytic rate\nconstants (kcat) and the Michaelis constant (Km). The results are shown in Figure 2. The wild\ntype 3CLpro has a Km of 56.42 ± 9.18 μM and a kcat of 0.643 ± 0.051 sec−1. The resulting\nKm and kcat are of the same magnitude as those previously reported with the same substrate\n(24). Table 1 shows the kinetic parameters obtained for wild type, S139A, S144A and S147A.\nAll of the proteases exhibited very similar Km's, suggesting that their ability to bind the substrate\nwith similar affinities was not impaired. S139A had a catalytic constant, kcat, similar to the\nwild type while the S144A value was about half. The most significant effect was observed for\nS147A which had a 160-fold drop in kcat with respect to the wild type enzyme. Consistent with\npreviously published observations(23,33), the activity of the wild type SARS 3CLpro and the\nthree mutants showed a linear correlation with increasing enzyme concentration as shown in\nFigure 3.\nStructural Stability of Wild Type 3CLpro and Serine Mutants\nTo verify that the mutants were properly folded and to assess the effect of the serine to alanine\nmutations, the structural stability of the wild type and mutant proteases was measured by\ndifferential scanning calorimetry. Figure 4 shows the temperature dependence of the heat\ncapacity function of wild type 3CLpro, S139A, S144A and S147A at pH 7.4 and 10 mM NaCl.\nThe denaturation transition for wild type 3CLpro is centered at 56.50 ± 0.1°C at a protein\nconcentration of 0.2 mg/mL (5.9 μM) and temperature scanning rate (1°C/min) used in these\nexperiments. This value is approximately 4°C higher than our previously reported value (5)\nwhich was measured using a construct containing five additional amino acids at the N-terminus\nof the protein. The transition temperature is similar to the value obtained by circular dichroism\n(CD)(27), but lower than others reported values (~60°C) obtained at faster scanning rates\n(29), as expected for irreversible denaturation. The S139A mutant had a slight effect on the\nstability on the protease with a downward shift in Tm of 0.9°C. The S144A mutation resulted\nin a 1.9°C upward shift in the transition temperature. The S147A mutation resulted in a\ndownward shift in the transition temperature of approximately 1.7°C. Besides the changes in\ndenaturation temperature, these experiments confirmed that all four proteins were properly\nfolded and indicate that the serine to alanine mutations did not significantly affect the overall\nstability of the recombinant proteins. Furthermore, similar results were obtained throughout\nthe length of time required for other experiments (e.g. sedimentation equilibrium\nmeasurements) confirming the structural integrity of the proteins in all experiments presented\nhere. The areas under the curves, ΔH, were also similar for all proteins, indicating the absence\nof any significant structural changes. A full thermodynamic analysis could not be performed\nwith any of the proteins due to precipitation once the transitions were complete which lead to\na lack of reversibility in the microcalorimetric scans. This irreversibility was observed\nregardless of buffer conditions used (pH values between 3 and 9; NaCl concentrations between\n0 and 3M).\nQuaternary Structure of Wild Type 3CLpro and Serine Mutants\nTo investigate the effects of the serine to alanine mutations on the quaternary structure of SARS\n3CLpro, sedimentation velocity ultracentrifugation was performed at 20°C on all constructs at\nconcentrations between 0.25 mg/ml (7.4 μM) and 1 mg/ml (29.5 μM). Figure 5 shows typical\nsedimentation velocity profiles at 0.5mg/ml (14.8 μM) obtained using direct boundary\nmodeling with the program SEDFIT (30). The experimental data is well accounted for by the\ncontinuous sedimentation coefficient model, c(s), as indicated by the plot of the residuals in\nFigure 5B. The sedimentation data for wild type 3CLpro in Figure 5C shows the presence of\ntwo peaks, at S1 = 2.5 and S2 = 4.0. These results are consistent with previous reported values\nin the literature (16,26,28). There did not appear to be a concentration dependence of the peak\npositions of c(s) in the concentration range tested. Similar results were obtained for the S139A\nand S144A proteins, shown in figures 5D and 5E. Despite being located directly at the dimer\ninterface, Serine 139 does not play a dominating role in enzyme dimerization. Similarly, even\nthough kinetic measurements show that S144A is less active than the wild type, the\nsedimentation profile was similar, albeit with a slight decrease in dimer population, suggesting\nthat this mutation directly affects catalysis. Like the wild type, neither of these mutants showed\na concentration dependence on the peak position of c(s).\nAnalysis of the S147A sedimentation velocity data unexpectedly produced a primary peak at\n2.7S, as shown in Figure 5F. An additional small, broad peak can be seen at ~4.0S,\ncorresponding to the formation of a minimal amounts of dimer. This peak can be seen to\nincrease slightly with increasing protein concentration. The peak positions of this mutant also\nremained constant within the range of the protein concentrations tested. The presence of the\nS147A dimer is short lived however, as ultracentrifugation experiments performed later than\ntwenty four hours after purification yielded solely a monomeric species. This correlates well\nwith the lack of enzymatic activity observed in the kinetics measurements.\nTo obtain dimer dissociation constants for the four constructs, sedimentation equilibrium\nexperiments were performed with wild type 3CLpro, S139A, S144A and S147A. Global\nanalysis for each of the proteins at multiple concentrations between 0.1 mg/ml (2.9 μM) and\n2 mg/ml (59.1 μM), and at multiple rotor speeds (15,000 rpm, 20,000 rpm, and 25,000 rpm)\nwas carried out with the program SEDPHAT using the monomer-dimer self association model\n(32). Figure 6 shows the results of the global fitting of each set of data at 0.5 mg/ml (14.8\nμM). All sets of experimental data fit well to the monomer-dimer model; the molecular weight\nobtained for the wild type monomeric species was 33,889 ± 441 daltons, which correlates\nexceedingly well with the expected monomeric weight of 33,846 kDa as estimated by the\nprotein sequence. The dissociation constant for the wild type 3CLpro dimer was determined to\nbe 1.3 ± 1.8 μM, which is slightly higher than previously reported values of 0.19 μM (26) and\n0.28 μM (28) obtained using sedimentation velocity, and slightly lower than a recently reported\nKd value of 14 μM obtained also by sedimentation equilibrium ultracentrifugation (16). The\nslight discrepancy between the two sedimentation equilibrium measurements may be due to\nthe extra amino acid on the N-terminus of the protein used in the previously published work.\nIt has been observed that additional amino acids on the N-terminus of the protease can increase\nthe measured Kd value or interfere with dimerization completely [(34) and unpublished data\nfrom this laboratory]. If additional amino acids corresponding to a 3CLpro cleavage site are\nadded to the N-terminus of the protein, the protein is able to cleave them off through trans-\nprocessing, albeit with a lower dimerization Kd. If the added residues do not correspond to a\ncleavage site, as with the previously published sedimentation equilibrium experiments, then\nthe extra residue cannot be cleaved off and may have an effect on the dimerization Kd.\nDissociation constants for the three mutants were also obtained, the results of which are shown\nin Figure 6 and Table 1. S139A showed a slight decrease in its ability to dimerize with a Kd\nof 12.9 ± 1.7 μM, and a molecular weight of the monomeric species calculated as 32,737 ±\n342 daltons. The decrease in catalytic efficiency is less than the one expected from the weaker\ndimerization constant, suggesting a better intrinsic catalytic activity of the mutant over the wild\ntype. On the other hand, the decrease in catalytic activity observed with S144A was not\nmirrored by a large decrease in dimerization, confirming the sedimentation velocity results.\nThe dimerization Kd obtained for the mutant was 3.3 ± 1.4 μM and the molecular weight\nobtained for the monomer was 33,466 ± 1121 daltons. The reduction in S144A activity is\napparently not due to a weaker dimerization, but to a direct effect on the catalytic activity of\nthe enzyme. It must be noted that this enzyme is structurally more stable than the wild type\n(Table 1) and that this added stability might reflect itself in a lower catalytic efficiency.\nThe sedimentation equilibrium profile of S147A fit extremely well to a monomeric distribution,\nas evidenced by the well dispersed residuals seen in Figure 6D. The molecular weight obtained\nfor the monomer was 32,641 ± 210 daltons, a value consistent with that obtained from the\nsedimentation velocity experiments. If we assume that only the dimer is active and that the\nS147A mutant dimer has an intrinsic kcat similar to that of the wild type, the Kd for S147A\ncould be expected to be around 300 μM; i.e. more than two orders of magnitude weaker than\nthe wild type.\nIn summary, we have carried out mutational analysis on a cluster of conserved serine residues\n(Ser139, Ser144 and Ser147) found adjacent to the active site of SARS 3CLpro. We examined\nthe effects of separately mutating each of these residues to alanine on the activity, stability and\noligomeric state of the protease using a fluorescence-based cleavage assay, differential\nscanning calorimetry, sedimentation velocity ultracentrifugation and sedimentation\nequilibrium sedimentation. Ser139, which makes interactions at the dimer interface, does not\nappear to play a significant role in maintaining the dimeric state of the protease, showing only\nslight decrease in activity, thermal stability and dimerization as compared to wild type\n3CLpro when mutated to alanine. Ser144, located closer to the active site cavity, showed a two-\nfold decrease in catalytic efficiency when mutated to alanine compared to that of the wild type,\nbut maintained a similar dimeric state and showed an increase in thermal stability by 1.9°C.\nMutation of Ser147, located approximately 9 Å away from the dimer interface, to alanine\nresulted in an approximately 150-fold loss in catalytic efficiency in comparison to the wild\ntype, and an inability of the protease to dimerize. This demonstrates that dimerization in SARS\n3CLpro can be controlled by long range cooperative interactions, suggesting that targeting\ncavities nearby Ser147 with small molecule inhibitors may provide an alternative way of\ninactivating the protease.\nStructural Analysis\nAccording to the existing crystallographic structures, serine 139 is the only residue of the three\nconsidered in this paper directly located at the dimerization interface. Serines 144 and 147 are\ncloser to the active site cavity and by visual inspection do not appear to play a key role in the\ndimerization of the enzyme. The interactions made by Ser139 are not bidirectional. Figure 1B\nshows the residues surrounding Ser139 of chain A at pH 7.5 [pdb structure 1UK3 (18)]. In this\nstructure, Ser139 of chain A makes hydrogen bonds to the backbone of Gly2 from chain B.\nConversely, no side chain interactions are made by Ser139 of chain B with residues in chain\nA. Mutation of Ser139 to Ala disrupts the dimerization interface and weakens the dimerization\nconstant by about 10-fold but does not interfere directly in the catalytic activity of the enzyme.\nSerine 144 is located closer to the active site in comparison to Ser139. Figure 1C shows the\nresidues surrounding Ser144. The side chain hydroxyl of Ser144 forms hydrogen bonds with\nthe side chain and backbone of Asn142. The backbone carbonyl of Ser144 interacts with the\nbackbone amide of Ser147, and the side chain of Asn28. Mutation of Ser 144 to Ala results in\na more structurally stable enzyme as reflected in the calorimetric data. The Km and the\ndimerization constant remain essentially the same while the catalytic constant, kcat, is reduced\nby half, suggesting that the structural rearrangement induced by the mutant reduces the catalytic\nefficiency of the enzyme.\nThe Ser147 to Ala mutation induces the most dramatic activity and structural effects even\nthough this residue is 9Å away from the dimer interface. Ser147 makes several backbone and\nside chain interactions with residues which are involved in an extensive hydrogen bonding\nnetwork within domains I and II in the protein. The backbone of Ser147 interacts with the\nbackbones of residues Ser144 and His163. Incidentally, His163 is important for catalytic\nactivity, since it comprises the S1 subsite that contributes to the specificity of substrate binding\n(18). The side chain of Ser147 makes two interactions with surrounding residues. The side\nchain hydroxyl of Ser147 hydrogen bonds to the backbones of Cys117 and Ser144. Cys117\nforms two additional hydrogen bonds with Asn28 and Leu115. Asn28 may be important in\nmaintaining the structural integrity in and around the active site region of the protease. This\nresidue makes several key interactions, including two interactions with the backbone carbonyl\nof the catalytic residue Cys145. The hydrogen bonds made by Asn28 to the backbones of\nCys117 and Gly120 may be critical in maintaining the position of the beta strand on which\nthey reside, which is connected to and runs anti-parallel to the beta strand containing Tyr126.\nIn turn, Tyr126 forms a hydrophobic pocket with Phe140 of the same chain and Met6 of the\nopposing chain. Met6 is the closest residue of the opposing monomer to Ser147, which is\npositioned approximately 9Å away. As previously mentioned, there is evidence suggesting that\nMet6 plays a role in maintaining dimerization, potentially through this hydrophobic interaction\nwith Tyr126 and Phe140 (16). The loss of Asn28 interactions with Cys117 and Gly120 may\nresult in the repositioning of this beta sheet and the disruption of this hydrophobic interaction.\nAdditionally, structural rearrangements due to the loss of these interactions may also interfere\nwith the hydrogen bond that Asn28 makes with the backbone carbonyl Gly143. Gly143 lies at\nthe top of the loop which contains Phe140 which is part of the hydrophobic pocket.\nFurthermore, the elimination of the hydroxyl group in the S147A mutation not only eliminates\nthe hydrogen bonds made by this group but also allows some rearrangements due to the smaller\nvan der Waals volume of the Ala mutation.\nConclusions\nThese studies reveal the possibility of new strategies for engineering inhibitors of SARS\n3CLpro by targeting sites that are away from the dimer interface but involved in the allosteric\ncontrol of dimerization. Since Ser147 is conserved in all known coronavirus 3CL proteases,\nthis observation opens the intriguing possibility that small molecules targeting this site may\nhave a broad antiviral spectrum against infections caused by different coronaviruses.",
         "Severe acute respiratory syndrome (SARS) is an upper respiratory tract disease caused by a\nnovel human coronavirus, SARS-CoV (1-3). Since the emergence of the virus in late 2002,\nover 8,000 people have been infected, resulting in a mortality exceeding 50% for patients over\nage 65 with an estimated 10% mortality overall. Currently there are no effective treatments\navailable against SARS. The recent identification of a SARS-like coronavirus (SL-CoV)\ncirculating in the Chinese horseshoe bat population emphasizes the possibility of a reemergence\nof infection, calling for the development of antiviral therapies targeting SARS-CoV (4). The\nSARS-CoV main protease, 3CLpro (also called Mpro) has been identified as a key target for\ndrug development. SARS 3CLpro plays a pivotal role in the life cycle of the coronavirus by the\nextensive proteolytic processing of two highly conserved polyproteins: pp1a (486 kDa) and\npp1ab (790 kDa), which encode for essential viral replicase proteins. The functional importance\nof 3CLpro in the coronavirus life cycle makes it a highly attractive target for drug design.\nSeveral types of small molecule inhibitors targeting 3CLpro have been identified either by high-\nthroughput screening, computational docking or structure-based drug design, some of which\ninclude bifunctional aryl boronic acid compounds, substrate analogue inhibitors, HIV protease\ninhibitors and dimerization inhibitors based on the N-terminal residues of 3CLpro (5-18).\nIn all available crystal structures SARS 3CLpro is a 68 kDa homodimer, with the two protomers\nof the dimer oriented almost perpendicular to one another, as shown in Figure 1A (18-21).\nEach monomer consists of three domains and contains a catalytic dyad of His41 and Cys145\nlocated in a cleft between the first two domains. A water molecule is hydrogen bonded to His41;\na position usually occupied by the third member of the catalytic triad in other cysteine proteases\n(18, 20, 21). The catalytic residues lie within a chymotrypsin-like double β-barrel fold\ncomprised of the first two domains of the protein (residues 1-184). The catalytic domains are\nconnected by a long loop region (residues 185-200) to a third globular domain composed of\nfive anti-parallel α-helices (residues 201-306). A surface area of ~2300 Å (~1250Å apolar and\n950Å polar) is buried at the dimer interface. The N-terminus, or “N-finger”, comprised of\nresidues 1-8 makes extensive interactions with the second domain of the parent protomer and\nthe third domain of the opposing monomer near the active site, thereby preserving the shape\nof the S1 pocket through hydrogen bond interactions with the oxyanion loop (18, 19). The\npositioning of these residues prompted speculation that the N-terminus is likely to be important\nin mediating dimerization and enzymatic activity of 3CLpro (18-21). The third alpha helical\ndomain has also been proposed to mediate dimerization and substrate recognition either\nthrough positioning of the N-terminus or by holding domain two and the long loop connected\nto the third domain in a catalytically competent orientation (18, 20-22).\nKinetic studies have demonstrated that the specific activity of SARS 3CLpro increases with\nprotein concentration, indicating that the dimer is the functional form of the protein [(23,24),\nthis work]. The enzyme exists as a dimer in solution over a wide range of concentrations (12,\n24-27). Reported dimerization dissociation constants (Kd) for SARS 3CLpro range from low\nnanomolar affinities (~0.35 nM) to high micromolar affinities (~230 μM). Differences in the\nvalues depend on the technique used to measure the Kd as well as the presence of N- or C-\nterminal extensions or affinity tags. In addition, handling of the protease during purification\nand storage appears to significantly affect the overall stability of the protease, thereby affecting\nactivity and dimerization measurements.\nResidues at the dimer interface that are essential for SARS 3CLpro dimerization and activity\nhave been identified by several groups. The N- and C-termini of the protease are important for\nmaintaining the oligomeric state of 3CLpro, with the majority of interactions being either ionic\nor hydrophobic. Chou et al (26) observed that both enzyme dimerization and activity decrease\nat higher salt concentrations and at lower pH, and that a salt bridge between Arg4 and Glu290\nwas important for these interactions. Mutation of Glu290 to Ala resulted in a complete loss of\ndimerization and activity while a mutation in Arg4 resulted in an approximately five-fold loss\nin dimerization and only a moderate loss of activity. Another study showed that deletion of the\nfirst three residues of the N-terminus resulted in only a moderate loss in activity and\ndimerization, while deletion of the first four residues of the N-terminus resulted in a dramatic\nloss in both areas, supporting the importance of Arg4 (28). Furthermore, it has been observed\nthat deletion of the first seven residues at the N-terminus of 3CLpro interfered with protease\nactivity and dimerization (16,28), though one group reported diminished activity but no change\nin oligomeric state when the same mutation was made (29). Mutation of Met6 to Ala completely\ninterferes with both dimerization and enzymatic activity (16). Met6 is proposed to form\nhydrophobic interactions with Tyr126 and Phe140 of the opposing chain. In addition, two\npeptides derived from the N-terminus of the protein have been shown to inhibit the activity of\n3CLpro, one of which has been shown to prevent dimerization, supporting the idea that the N-\nterminus plays a role in maintaining the oligomeric state of 3CLpro (8,16). Deletion of the third\nalpha helical domain also interferes with dimerization and activity [(27-29)and unpublished\ndata from this laboratory]. The third domain alone has been observed to dimerize on its own,\nleading to the hypothesis that the role of this domain may be the regulation of enzymatic activity\nthrough dimerization (27).\nAll of the mutations affecting the dimerization of SARS 3CLpro reported so far involve residues\nwhich are located directly at the dimer interface. Here we provide data which demonstrates\nthat dimerization is also controlled by long range cooperative interactions in 3CLpro. Mutation\nof conserved Ser147 to Ala, located approximately 9 Å away from the dimer interface, inhibited\ndimerization and resulted in an inactive enzyme. The location of Ser147 suggests alternative\nsites for SARS 3CLpro dimerization inhibitors.\nEXPERIMENTAL PROCEDURES",
         "cDNA encoding full-length SARS 3CLpro (Tor2 strain, GenBank entry AY274119) was\npreviously cloned into a pET 100 vector (Champion pET Directional TOPO Expression and\nCloning Kit, Invitrogen) which carries an N-terminal polyhistidine tag, an enterokinase\ncleavage site, and ampicillin resistance (5). Mutations at selected positions (S139A, S144A\nand S147A) were introduced using an in vitro site directed mutagenesis kit (QuickChange,\nStratagene) with the pET-SARS 3CLpro vector as the template. The sequences of the primers\nused to generate the mutants were as follows: S139A: 5′ CCA TTA AAG GTG CTT TCC TTA\nATG G 3′; S144A: TTC CTT AAT GGA GCA TGT GGT AGT G 3′; and S147A: GGA TCA\nTGT GGT GCT GTT GGT TTT AAC ATT G 3′. Following the PCR reaction carried out with\nPfu turbo DNA polymerase (Stratagene), the parent template was degraded with a DpnI\n(Stratagene) digestion reaction for 1 hour at 37 °C. One shot TOPO10 competent cells\n(Invitrogen) were transformed with the PCR product for plasmid amplification. The plasmid\nwas then purified and mutations confirmed by DNA sequencing. This procedure was used to\ngenerate single, double and triple mutations of the serines to alanine using plasmids and primers\ncorresponding to the residue change.\nProtein Expression and Purification\nRecombinant SARS 3CLpro was expressed as a soluble fraction in BL21 Star DE3 E. coli\ncompetent cells (Invitrogen). The construct begins with residue Ser1, and therefore does not\ncontain the full N-terminal auto-cleavage site of the protein. Cells were grown in LB\nsupplemented with ampicillin (50 μg/mL) at 37 °C, induced with 1 mM IPTG when the optical\ndensity (as determined by absorbance at 600 nm) was 0.8 or greater, and harvested after 4\nhours. Cells were resuspended in lysis buffer (50 mM potassium phosphate (pH 7.8), 400 mM\nsodium chloride, 100 mM potassium chloride, 10% glycerol, 0.5% Triton-X, and 10 mM\nimidazole). The cells were broken by sonicating on ice for short pulses of one second followed\nby three seconds off for a total of 16 minutes. Cell debris was collected by centrifugation\n(20,000g at 4 °C for 45 min). The supernatant was filtered using a 0.45 μm pore size filter\n(Millipore) and applied directly to a nickel affinity column (HiTrap Chelating HP, Amersham\nBiosciences) that had been pre-equilibrated with binding buffer (50 mM sodium phosphate,\n0.3 M sodium chloride, 10 mM imidazole, pH 8.0). The protease was eluted with a linear\ngradient of 50 mM sodium phosphate, 0.3 M sodium chloride, 250 mM imidazole, pH 8.0.\nAfter elution, the protein was buffer exchanged into 10 mM Tris-HCl pH 7.5, and loaded onto\na Q-sepharose anion exchange column (Amersham Biosciences). The protease was eluted with\na gradient of 10 mM Tris-HCl, 1 M NaCl, pH 7.5. The pooled fractions containing 3CLpro\nwere buffer exchanged into storage buffer (10 mM sodium phosphate, 10 mM sodium chloride,\n1 mM TCEP, 1 mM EDTA, pH 7.4) and digested for 24 h at 4 °C with enterokinase (Invitrogen,\n0.1 units per 112 μg of protease) to remove the N-terminal polyhistidine tag. The enterokinase\nwas removed by incubation with EK-away resin (Invitrogen). The reaction mixture was passed\nthrough a nickel affinity column to remove undigested protease. The protease was buffer\nexchanged into storage buffer, concentrated to 10 mg/mL and used immediately for\nexperiments. The sample was more than 95% pure, as assessed by SDS-PAGE.\nKinetics\nThe catalytic activities of wild type and mutant SARS 3CLpro were determined using a\nfluorescence-based peptide cleavage assay with a commercially available fluorogenic\nsubstrate, Dabcyl-KTSAVLQSGFRKME-Edans (Genesis Biotech, Taiwan), which\ncorresponds to the N-terminal auto-cleavage site of the protease (24). The change in\nfluorescence intensity was monitored in a Cary Eclipse fluorescence spectrophotometer\n(Varian) with 355 nm and 538 nm excitation and emission wavelengths respectively. The\nexperiments were performed in 10 mM sodium phosphate, 10 mM sodium chloride, 1 mM\nEDTA, 1 mM TCEP, pH 7.4. Kinetic parameters such as Km and kcat were determined by initial\nrate measurement of substrate cleavage at 25 °C. The reaction was initiated by the addition of\nsubstrate such that final concentration varied from 1-60 μM.\nAnalytical Ultracentrifugation\nSedimentation velocity and equilibrium experiments were conducted using a Beckman-Coulter\nXL-I analytical ultracentrifuge. Wild type and mutant samples were prepared by exhaustive\ndialysis into 10 mM Tris-HCl, 0.1 M NaCl, 1 mM TCEP, 1 mM EDTA, pH 7.5 and then\nconcentrated to 1 mg/mL. For sedimentation velocity experiments, reference (420 μL) and\nsample (400 μL) solutions were loaded into double-sector centerpieces and mounted in a\nBeckman An60Ti rotor. Protein concentrations for these experiments were between 0.25mg/\nml (7.4 μM) and 1 mg/ml (29.5 μM). Experiments were performed at 20 °C with a rotor speed\nof 50,000 rpm. Sample absorbance at 280 nm was monitored in a continuous mode with no\ndelay and a step size of 0.003 cm without averaging. Multiple scans at different time points\nwere fit to a continuous size distribution using SEDFIT version 9.3b (30). The c(s) distribution\nin SEDFIT was exported to SEDPHAT where experimental values were corrected to S20, w\n(31). A partial specific volume of 0.7331 cm3/g for wild type 3CLpro and 0.7334 cm3/g for the\nS147A mutant were used in all calculations based on the amino acid sequence of the protein.\nThe solvent density and viscosity were calculated using SEDNTERP (Philo, J. website\nhttp://www.jphilo.mailway.com/default.htm). Sedimentation equilibrium experiments were\nperformed in the same buffer as sedimentation velocity experiments at concentrations between\n0.1 and 2 mg/ml (between 2.9 μM and 59.1 μM). Reference (120 μl) and sample (110 μl)\nsolutions were loaded into six sector centerpieces and mounted in a Beckman An60Ti rotor.\nSamples were equilibrated at 20 °C and three speeds: 15,000 rpm, 20,000 rpm and 25,000 rpm.\nSample absorbance at 280 or 250 nm was collected in step mode with a step size of 0.001 cm\nand 10 replicates at each radial position. The program WinMatch (Biotechnology-Bioservices\nCenter, University of Connecticut) was used to determine the point at which samples reached\nequilibrium. Equilibrium was typically attained for all samples between 24 and 30 hours. All\nsamples were visually inspected for clarity to ensure no precipitation had occurred during the\nruns. Dissociation constants were determined by global fitting of multiple concentrations and\nrotor speeds in SEDPHAT using the monomer-dimer equilibrium model (32).\nDifferential Scanning Calorimetry\nThe heat capacities of wild type 3CLpro and the serine to alanine mutants were measured as a\nfunction of temperature with a high-precision differential scanning VP-DSC microcalorimeter\n(Microcal Inc., Northampton, MA). Protein samples and reference solutions were properly\ndegassed and carefully loaded into the cells to avoid bubble formation. Thermal denaturation\nscans were performed with freshly prepared buffer-exchanged protease solutions using a\ntemperature scan rate of 1°C/min. Data was analyzed by software developed in this laboratory.\nRESULTS AND DISCUSSION\nEnzymatic Activity",
         "cDNA encoding full-length SARS 3CLpro (Tor2 strain, GenBank entry AY274119) was\npreviously cloned into a pET 100 vector (Champion pET Directional TOPO Expression and\nCloning Kit, Invitrogen) which carries an N-terminal polyhistidine tag, an enterokinase\ncleavage site, and ampicillin resistance (5). Mutations at selected positions (S139A, S144A\nand S147A) were introduced using an in vitro site directed mutagenesis kit (QuickChange,\nStratagene) with the pET-SARS 3CLpro vector as the template. The sequences of the primers\nused to generate the mutants were as follows: S139A: 5′ CCA TTA AAG GTG CTT TCC TTA\nATG G 3′; S144A: TTC CTT AAT GGA GCA TGT GGT AGT G 3′; and S147A: GGA TCA\nTGT GGT GCT GTT GGT TTT AAC ATT G 3′. Following the PCR reaction carried out with\nPfu turbo DNA polymerase (Stratagene), the parent template was degraded with a DpnI\n(Stratagene) digestion reaction for 1 hour at 37 °C. One shot TOPO10 competent cells\n(Invitrogen) were transformed with the PCR product for plasmid amplification. The plasmid\nwas then purified and mutations confirmed by DNA sequencing. This procedure was used to\ngenerate single, double and triple mutations of the serines to alanine using plasmids and primers\ncorresponding to the residue change.\nProtein Expression and Purification\nRecombinant SARS 3CLpro was expressed as a soluble fraction in BL21 Star DE3 E. coli\ncompetent cells (Invitrogen). The construct begins with residue Ser1, and therefore does not\ncontain the full N-terminal auto-cleavage site of the protein. Cells were grown in LB\nsupplemented with ampicillin (50 μg/mL) at 37 °C, induced with 1 mM IPTG when the optical\ndensity (as determined by absorbance at 600 nm) was 0.8 or greater, and harvested after 4\nhours. Cells were resuspended in lysis buffer (50 mM potassium phosphate (pH 7.8), 400 mM\nsodium chloride, 100 mM potassium chloride, 10% glycerol, 0.5% Triton-X, and 10 mM\nimidazole). The cells were broken by sonicating on ice for short pulses of one second followed\nby three seconds off for a total of 16 minutes. Cell debris was collected by centrifugation\n(20,000g at 4 °C for 45 min). The supernatant was filtered using a 0.45 μm pore size filter\n(Millipore) and applied directly to a nickel affinity column (HiTrap Chelating HP, Amersham\nBiosciences) that had been pre-equilibrated with binding buffer (50 mM sodium phosphate,\n0.3 M sodium chloride, 10 mM imidazole, pH 8.0). The protease was eluted with a linear\ngradient of 50 mM sodium phosphate, 0.3 M sodium chloride, 250 mM imidazole, pH 8.0.\nAfter elution, the protein was buffer exchanged into 10 mM Tris-HCl pH 7.5, and loaded onto\na Q-sepharose anion exchange column (Amersham Biosciences). The protease was eluted with\na gradient of 10 mM Tris-HCl, 1 M NaCl, pH 7.5. The pooled fractions containing 3CLpro\nwere buffer exchanged into storage buffer (10 mM sodium phosphate, 10 mM sodium chloride,\n1 mM TCEP, 1 mM EDTA, pH 7.4) and digested for 24 h at 4 °C with enterokinase (Invitrogen,\n0.1 units per 112 μg of protease) to remove the N-terminal polyhistidine tag. The enterokinase\nwas removed by incubation with EK-away resin (Invitrogen). The reaction mixture was passed\nthrough a nickel affinity column to remove undigested protease. The protease was buffer\nexchanged into storage buffer, concentrated to 10 mg/mL and used immediately for\nexperiments. The sample was more than 95% pure, as assessed by SDS-PAGE.\nKinetics\nThe catalytic activities of wild type and mutant SARS 3CLpro were determined using a\nfluorescence-based peptide cleavage assay with a commercially available fluorogenic\nsubstrate, Dabcyl-KTSAVLQSGFRKME-Edans (Genesis Biotech, Taiwan), which\ncorresponds to the N-terminal auto-cleavage site of the protease (24). The change in\nfluorescence intensity was monitored in a Cary Eclipse fluorescence spectrophotometer\n(Varian) with 355 nm and 538 nm excitation and emission wavelengths respectively. The\nexperiments were performed in 10 mM sodium phosphate, 10 mM sodium chloride, 1 mM\nEDTA, 1 mM TCEP, pH 7.4. Kinetic parameters such as Km and kcat were determined by initial\nrate measurement of substrate cleavage at 25 °C. The reaction was initiated by the addition of\nsubstrate such that final concentration varied from 1-60 μM.\nAnalytical Ultracentrifugation\nSedimentation velocity and equilibrium experiments were conducted using a Beckman-Coulter\nXL-I analytical ultracentrifuge. Wild type and mutant samples were prepared by exhaustive\ndialysis into 10 mM Tris-HCl, 0.1 M NaCl, 1 mM TCEP, 1 mM EDTA, pH 7.5 and then\nconcentrated to 1 mg/mL. For sedimentation velocity experiments, reference (420 μL) and\nsample (400 μL) solutions were loaded into double-sector centerpieces and mounted in a\nBeckman An60Ti rotor. Protein concentrations for these experiments were between 0.25mg/\nml (7.4 μM) and 1 mg/ml (29.5 μM). Experiments were performed at 20 °C with a rotor speed\nof 50,000 rpm. Sample absorbance at 280 nm was monitored in a continuous mode with no\ndelay and a step size of 0.003 cm without averaging. Multiple scans at different time points\nwere fit to a continuous size distribution using SEDFIT version 9.3b (30). The c(s) distribution\nin SEDFIT was exported to SEDPHAT where experimental values were corrected to S20, w\n(31). A partial specific volume of 0.7331 cm3/g for wild type 3CLpro and 0.7334 cm3/g for the\nS147A mutant were used in all calculations based on the amino acid sequence of the protein.\nThe solvent density and viscosity were calculated using SEDNTERP (Philo, J. website\nhttp://www.jphilo.mailway.com/default.htm). Sedimentation equilibrium experiments were\nperformed in the same buffer as sedimentation velocity experiments at concentrations between\n0.1 and 2 mg/ml (between 2.9 μM and 59.1 μM). Reference (120 μl) and sample (110 μl)\nsolutions were loaded into six sector centerpieces and mounted in a Beckman An60Ti rotor.\nSamples were equilibrated at 20 °C and three speeds: 15,000 rpm, 20,000 rpm and 25,000 rpm.\nSample absorbance at 280 or 250 nm was collected in step mode with a step size of 0.001 cm\nand 10 replicates at each radial position. The program WinMatch (Biotechnology-Bioservices\nCenter, University of Connecticut) was used to determine the point at which samples reached\nequilibrium. Equilibrium was typically attained for all samples between 24 and 30 hours. All\nsamples were visually inspected for clarity to ensure no precipitation had occurred during the\nruns. Dissociation constants were determined by global fitting of multiple concentrations and\nrotor speeds in SEDPHAT using the monomer-dimer equilibrium model (32).\nDifferential Scanning Calorimetry\nThe heat capacities of wild type 3CLpro and the serine to alanine mutants were measured as a\nfunction of temperature with a high-precision differential scanning VP-DSC microcalorimeter\n(Microcal Inc., Northampton, MA). Protein samples and reference solutions were properly\ndegassed and carefully loaded into the cells to avoid bubble formation. Thermal denaturation\nscans were performed with freshly prepared buffer-exchanged protease solutions using a\ntemperature scan rate of 1°C/min. Data was analyzed by software developed in this laboratory.\nRESULTS AND DISCUSSION\nEnzymatic Activity\nWe previously identified a highly conserved cluster of serine residues: Ser139, Ser144 and\nSer147, which lie adjacent to the active site of SARS 3CLpro (5). The location of this serine\ncluster in the structure of 3CLpro is shown in Figure 1A. Serines 139 and 147 are absolutely\nconserved in all known coronavirus 3CL proteases, while Ser144 is conserved in 55% of the\nreported sequences.\nThe role played by the three serine residues in the catalytic activity of SARS 3CLpro was\nevaluated by generating serine to alanine mutants at the three positions. The catalytic activities\nof wild type 3CLpro and the mutants S139A, S144A and S147A were measured at 1 μM enzyme\nconcentration using a fluorescence-based cleavage assay in order to obtain the catalytic rate\nconstants (kcat) and the Michaelis constant (Km). The results are shown in Figure 2. The wild\ntype 3CLpro has a Km of 56.42 ± 9.18 μM and a kcat of 0.643 ± 0.051 sec−1. The resulting\nKm and kcat are of the same magnitude as those previously reported with the same substrate\n(24). Table 1 shows the kinetic parameters obtained for wild type, S139A, S144A and S147A.\nAll of the proteases exhibited very similar Km's, suggesting that their ability to bind the substrate\nwith similar affinities was not impaired. S139A had a catalytic constant, kcat, similar to the\nwild type while the S144A value was about half. The most significant effect was observed for\nS147A which had a 160-fold drop in kcat with respect to the wild type enzyme. Consistent with\npreviously published observations(23,33), the activity of the wild type SARS 3CLpro and the\nthree mutants showed a linear correlation with increasing enzyme concentration as shown in\nFigure 3.\nStructural Stability of Wild Type 3CLpro and Serine Mutants\nTo verify that the mutants were properly folded and to assess the effect of the serine to alanine\nmutations, the structural stability of the wild type and mutant proteases was measured by\ndifferential scanning calorimetry. Figure 4 shows the temperature dependence of the heat\ncapacity function of wild type 3CLpro, S139A, S144A and S147A at pH 7.4 and 10 mM NaCl.\nThe denaturation transition for wild type 3CLpro is centered at 56.50 ± 0.1°C at a protein\nconcentration of 0.2 mg/mL (5.9 μM) and temperature scanning rate (1°C/min) used in these\nexperiments. This value is approximately 4°C higher than our previously reported value (5)\nwhich was measured using a construct containing five additional amino acids at the N-terminus\nof the protein. The transition temperature is similar to the value obtained by circular dichroism\n(CD)(27), but lower than others reported values (~60°C) obtained at faster scanning rates\n(29), as expected for irreversible denaturation. The S139A mutant had a slight effect on the\nstability on the protease with a downward shift in Tm of 0.9°C. The S144A mutation resulted\nin a 1.9°C upward shift in the transition temperature. The S147A mutation resulted in a\ndownward shift in the transition temperature of approximately 1.7°C. Besides the changes in\ndenaturation temperature, these experiments confirmed that all four proteins were properly\nfolded and indicate that the serine to alanine mutations did not significantly affect the overall\nstability of the recombinant proteins. Furthermore, similar results were obtained throughout\nthe length of time required for other experiments (e.g. sedimentation equilibrium\nmeasurements) confirming the structural integrity of the proteins in all experiments presented\nhere. The areas under the curves, ΔH, were also similar for all proteins, indicating the absence\nof any significant structural changes. A full thermodynamic analysis could not be performed\nwith any of the proteins due to precipitation once the transitions were complete which lead to\na lack of reversibility in the microcalorimetric scans. This irreversibility was observed\nregardless of buffer conditions used (pH values between 3 and 9; NaCl concentrations between\n0 and 3M).\nQuaternary Structure of Wild Type 3CLpro and Serine Mutants\nTo investigate the effects of the serine to alanine mutations on the quaternary structure of SARS\n3CLpro, sedimentation velocity ultracentrifugation was performed at 20°C on all constructs at\nconcentrations between 0.25 mg/ml (7.4 μM) and 1 mg/ml (29.5 μM). Figure 5 shows typical\nsedimentation velocity profiles at 0.5mg/ml (14.8 μM) obtained using direct boundary\nmodeling with the program SEDFIT (30). The experimental data is well accounted for by the\ncontinuous sedimentation coefficient model, c(s), as indicated by the plot of the residuals in\nFigure 5B. The sedimentation data for wild type 3CLpro in Figure 5C shows the presence of\ntwo peaks, at S1 = 2.5 and S2 = 4.0. These results are consistent with previous reported values\nin the literature (16,26,28). There did not appear to be a concentration dependence of the peak\npositions of c(s) in the concentration range tested. Similar results were obtained for the S139A\nand S144A proteins, shown in figures 5D and 5E. Despite being located directly at the dimer\ninterface, Serine 139 does not play a dominating role in enzyme dimerization. Similarly, even\nthough kinetic measurements show that S144A is less active than the wild type, the\nsedimentation profile was similar, albeit with a slight decrease in dimer population, suggesting\nthat this mutation directly affects catalysis. Like the wild type, neither of these mutants showed\na concentration dependence on the peak position of c(s).\nAnalysis of the S147A sedimentation velocity data unexpectedly produced a primary peak at\n2.7S, as shown in Figure 5F. An additional small, broad peak can be seen at ~4.0S,\ncorresponding to the formation of a minimal amounts of dimer. This peak can be seen to\nincrease slightly with increasing protein concentration. The peak positions of this mutant also\nremained constant within the range of the protein concentrations tested. The presence of the\nS147A dimer is short lived however, as ultracentrifugation experiments performed later than\ntwenty four hours after purification yielded solely a monomeric species. This correlates well\nwith the lack of enzymatic activity observed in the kinetics measurements.\nTo obtain dimer dissociation constants for the four constructs, sedimentation equilibrium\nexperiments were performed with wild type 3CLpro, S139A, S144A and S147A. Global\nanalysis for each of the proteins at multiple concentrations between 0.1 mg/ml (2.9 μM) and\n2 mg/ml (59.1 μM), and at multiple rotor speeds (15,000 rpm, 20,000 rpm, and 25,000 rpm)\nwas carried out with the program SEDPHAT using the monomer-dimer self association model\n(32). Figure 6 shows the results of the global fitting of each set of data at 0.5 mg/ml (14.8\nμM). All sets of experimental data fit well to the monomer-dimer model; the molecular weight\nobtained for the wild type monomeric species was 33,889 ± 441 daltons, which correlates\nexceedingly well with the expected monomeric weight of 33,846 kDa as estimated by the\nprotein sequence. The dissociation constant for the wild type 3CLpro dimer was determined to\nbe 1.3 ± 1.8 μM, which is slightly higher than previously reported values of 0.19 μM (26) and\n0.28 μM (28) obtained using sedimentation velocity, and slightly lower than a recently reported\nKd value of 14 μM obtained also by sedimentation equilibrium ultracentrifugation (16). The\nslight discrepancy between the two sedimentation equilibrium measurements may be due to\nthe extra amino acid on the N-terminus of the protein used in the previously published work.\nIt has been observed that additional amino acids on the N-terminus of the protease can increase\nthe measured Kd value or interfere with dimerization completely [(34) and unpublished data\nfrom this laboratory]. If additional amino acids corresponding to a 3CLpro cleavage site are\nadded to the N-terminus of the protein, the protein is able to cleave them off through trans-\nprocessing, albeit with a lower dimerization Kd. If the added residues do not correspond to a\ncleavage site, as with the previously published sedimentation equilibrium experiments, then\nthe extra residue cannot be cleaved off and may have an effect on the dimerization Kd.\nDissociation constants for the three mutants were also obtained, the results of which are shown\nin Figure 6 and Table 1. S139A showed a slight decrease in its ability to dimerize with a Kd\nof 12.9 ± 1.7 μM, and a molecular weight of the monomeric species calculated as 32,737 ±\n342 daltons. The decrease in catalytic efficiency is less than the one expected from the weaker\ndimerization constant, suggesting a better intrinsic catalytic activity of the mutant over the wild\ntype. On the other hand, the decrease in catalytic activity observed with S144A was not\nmirrored by a large decrease in dimerization, confirming the sedimentation velocity results.\nThe dimerization Kd obtained for the mutant was 3.3 ± 1.4 μM and the molecular weight\nobtained for the monomer was 33,466 ± 1121 daltons. The reduction in S144A activity is\napparently not due to a weaker dimerization, but to a direct effect on the catalytic activity of\nthe enzyme. It must be noted that this enzyme is structurally more stable than the wild type\n(Table 1) and that this added stability might reflect itself in a lower catalytic efficiency.\nThe sedimentation equilibrium profile of S147A fit extremely well to a monomeric distribution,\nas evidenced by the well dispersed residuals seen in Figure 6D. The molecular weight obtained\nfor the monomer was 32,641 ± 210 daltons, a value consistent with that obtained from the\nsedimentation velocity experiments. If we assume that only the dimer is active and that the\nS147A mutant dimer has an intrinsic kcat similar to that of the wild type, the Kd for S147A\ncould be expected to be around 300 μM; i.e. more than two orders of magnitude weaker than\nthe wild type.\nIn summary, we have carried out mutational analysis on a cluster of conserved serine residues\n(Ser139, Ser144 and Ser147) found adjacent to the active site of SARS 3CLpro. We examined\nthe effects of separately mutating each of these residues to alanine on the activity, stability and\noligomeric state of the protease using a fluorescence-based cleavage assay, differential\nscanning calorimetry, sedimentation velocity ultracentrifugation and sedimentation\nequilibrium sedimentation. Ser139, which makes interactions at the dimer interface, does not\nappear to play a significant role in maintaining the dimeric state of the protease, showing only\nslight decrease in activity, thermal stability and dimerization as compared to wild type\n3CLpro when mutated to alanine. Ser144, located closer to the active site cavity, showed a two-\nfold decrease in catalytic efficiency when mutated to alanine compared to that of the wild type,\nbut maintained a similar dimeric state and showed an increase in thermal stability by 1.9°C.\nMutation of Ser147, located approximately 9 Å away from the dimer interface, to alanine\nresulted in an approximately 150-fold loss in catalytic efficiency in comparison to the wild\ntype, and an inability of the protease to dimerize. This demonstrates that dimerization in SARS\n3CLpro can be controlled by long range cooperative interactions, suggesting that targeting\ncavities nearby Ser147 with small molecule inhibitors may provide an alternative way of\ninactivating the protease.\nStructural Analysis\nAccording to the existing crystallographic structures, serine 139 is the only residue of the three\nconsidered in this paper directly located at the dimerization interface. Serines 144 and 147 are\ncloser to the active site cavity and by visual inspection do not appear to play a key role in the\ndimerization of the enzyme. The interactions made by Ser139 are not bidirectional. Figure 1B\nshows the residues surrounding Ser139 of chain A at pH 7.5 [pdb structure 1UK3 (18)]. In this\nstructure, Ser139 of chain A makes hydrogen bonds to the backbone of Gly2 from chain B.\nConversely, no side chain interactions are made by Ser139 of chain B with residues in chain\nA. Mutation of Ser139 to Ala disrupts the dimerization interface and weakens the dimerization\nconstant by about 10-fold but does not interfere directly in the catalytic activity of the enzyme.\nSerine 144 is located closer to the active site in comparison to Ser139. Figure 1C shows the\nresidues surrounding Ser144. The side chain hydroxyl of Ser144 forms hydrogen bonds with\nthe side chain and backbone of Asn142. The backbone carbonyl of Ser144 interacts with the\nbackbone amide of Ser147, and the side chain of Asn28. Mutation of Ser 144 to Ala results in\na more structurally stable enzyme as reflected in the calorimetric data. The Km and the\ndimerization constant remain essentially the same while the catalytic constant, kcat, is reduced\nby half, suggesting that the structural rearrangement induced by the mutant reduces the catalytic\nefficiency of the enzyme.\nThe Ser147 to Ala mutation induces the most dramatic activity and structural effects even\nthough this residue is 9Å away from the dimer interface. Ser147 makes several backbone and\nside chain interactions with residues which are involved in an extensive hydrogen bonding\nnetwork within domains I and II in the protein. The backbone of Ser147 interacts with the\nbackbones of residues Ser144 and His163. Incidentally, His163 is important for catalytic\nactivity, since it comprises the S1 subsite that contributes to the specificity of substrate binding\n(18). The side chain of Ser147 makes two interactions with surrounding residues. The side\nchain hydroxyl of Ser147 hydrogen bonds to the backbones of Cys117 and Ser144. Cys117\nforms two additional hydrogen bonds with Asn28 and Leu115. Asn28 may be important in\nmaintaining the structural integrity in and around the active site region of the protease. This\nresidue makes several key interactions, including two interactions with the backbone carbonyl\nof the catalytic residue Cys145. The hydrogen bonds made by Asn28 to the backbones of\nCys117 and Gly120 may be critical in maintaining the position of the beta strand on which\nthey reside, which is connected to and runs anti-parallel to the beta strand containing Tyr126.\nIn turn, Tyr126 forms a hydrophobic pocket with Phe140 of the same chain and Met6 of the\nopposing chain. Met6 is the closest residue of the opposing monomer to Ser147, which is\npositioned approximately 9Å away. As previously mentioned, there is evidence suggesting that\nMet6 plays a role in maintaining dimerization, potentially through this hydrophobic interaction\nwith Tyr126 and Phe140 (16). The loss of Asn28 interactions with Cys117 and Gly120 may\nresult in the repositioning of this beta sheet and the disruption of this hydrophobic interaction.\nAdditionally, structural rearrangements due to the loss of these interactions may also interfere\nwith the hydrogen bond that Asn28 makes with the backbone carbonyl Gly143. Gly143 lies at\nthe top of the loop which contains Phe140 which is part of the hydrophobic pocket.\nFurthermore, the elimination of the hydroxyl group in the S147A mutation not only eliminates\nthe hydrogen bonds made by this group but also allows some rearrangements due to the smaller\nvan der Waals volume of the Ala mutation.",
         "https://journals.aai.org/crawlprevention/governor?content=%2fjimmunol%2farticle-pdf-handler%3fjournal%3djimmunol%26multimediaId%3d1217413%26volume%3d176%26issue%3d3%26page%3d1645%26filename%3d1645.pdf%26languageId%3d",
         "Severe acute respiratory syndrome (SARS) is an infectious disease caused by the human coronavirus,\nSARS-CoV. The main viral protease, SARS 3CLpro is a validated target for the development of\nantiviral therapies. Since the enzyme is a homodimer and the individual monomers are inactive, two\napproaches are being used to develop inhibitors: enzyme activity inhibitors that target the active site\nand dimerization inhibitors. Dimerization inhibitors are usually targeted to the dimerization interface\nand need to compete with the attractive forces between subunits in order to be effective. In this paper,\nwe show that the dimerization of SARS 3CLpro is also under allosteric control and that additional\nand energetically more favorable target sites away from the dimerization interface may also lead to\nsubunit dissociation. We previously identified a cluster of conserved serine residues (Ser139, Ser144\nand Ser147) located adjacent to the active site of 3CLpro that could effectively be targeted to inactivate\nthe protease. Mutation of any of these serine residues to alanine had a debilitating effect on the\ncatalytic activity of 3CLpro. In particular, the mutation of Ser147, which does not make any contact\nwith the opposing subunit and is located approximately 9Å away from the dimer interface, totally\ninhibited dimerization and resulted in a complete loss of enzymatic activity. The finding that residues\naway from the dimer interface are able to control dimerization, defines alternative targets for the\ndesign of dimerization inhibitors.\nSevere acute respiratory syndrome (SARS) is an upper respiratory tract disease caused by a\nnovel human coronavirus, SARS-CoV (1-3). Since the emergence of the virus in late 2002,\nover 8,000 people have been infected, resulting in a mortality exceeding 50% for patients over\nage 65 with an estimated 10% mortality overall. Currently there are no effective treatments\navailable against SARS. The recent identification of a SARS-like coronavirus (SL-CoV)\ncirculating in the Chinese horseshoe bat population emphasizes the possibility of a reemergence\nof infection, calling for the development of antiviral therapies targeting SARS-CoV (4). The\nSARS-CoV main protease, 3CLpro (also called Mpro) has been identified as a key target for\ndrug development. SARS 3CLpro plays a pivotal role in the life cycle of the coronavirus by the\nextensive proteolytic processing of two highly conserved polyproteins: pp1a (486 kDa) and\npp1ab (790 kDa), which encode for essential viral replicase proteins. The functional importance\nof 3CLpro in the coronavirus life cycle makes it a highly attractive target for drug design.\nSeveral types of small molecule inhibitors targeting 3CLpro have been identified either by high-\nthroughput screening, computational docking or structure-based drug design, some of which\ninclude bifunctional aryl boronic acid compounds, substrate analogue inhibitors, HIV protease\ninhibitors and dimerization inhibitors based on the N-terminal residues of 3CLpro (5-18).\nIn all available crystal structures SARS 3CLpro is a 68 kDa homodimer, with the two protomers\nof the dimer oriented almost perpendicular to one another, as shown in Figure 1A (18-21).\nEach monomer consists of three domains and contains a catalytic dyad of His41 and Cys145\nlocated in a cleft between the first two domains. A water molecule is hydrogen bonded to His41;\na position usually occupied by the third member of the catalytic triad in other cysteine proteases\n(18, 20, 21). The catalytic residues lie within a chymotrypsin-like double β-barrel fold\ncomprised of the first two domains of the protein (residues 1-184). The catalytic domains are\nconnected by a long loop region (residues 185-200) to a third globular domain composed of\nfive anti-parallel α-helices (residues 201-306). A surface area of ~2300 Å (~1250Å apolar and\n950Å polar) is buried at the dimer interface. The N-terminus, or “N-finger”, comprised of\nresidues 1-8 makes extensive interactions with the second domain of the parent protomer and\nthe third domain of the opposing monomer near the active site, thereby preserving the shape\nof the S1 pocket through hydrogen bond interactions with the oxyanion loop (18, 19). The\npositioning of these residues prompted speculation that the N-terminus is likely to be important\nin mediating dimerization and enzymatic activity of 3CLpro (18-21). The third alpha helical\ndomain has also been proposed to mediate dimerization and substrate recognition either\nthrough positioning of the N-terminus or by holding domain two and the long loop connected\nto the third domain in a catalytically competent orientation (18, 20-22).\nKinetic studies have demonstrated that the specific activity of SARS 3CLpro increases with\nprotein concentration, indicating that the dimer is the functional form of the protein [(23,24),\nthis work]. The enzyme exists as a dimer in solution over a wide range of concentrations (12,\n24-27). Reported dimerization dissociation constants (Kd) for SARS 3CLpro range from low\nnanomolar affinities (~0.35 nM) to high micromolar affinities (~230 μM). Differences in the\nvalues depend on the technique used to measure the Kd as well as the presence of N- or C-\nterminal extensions or affinity tags. In addition, handling of the protease during purification\nand storage appears to significantly affect the overall stability of the protease, thereby affecting\nactivity and dimerization measurements.\nResidues at the dimer interface that are essential for SARS 3CLpro dimerization and activity\nhave been identified by several groups. The N- and C-termini of the protease are important for\nmaintaining the oligomeric state of 3CLpro, with the majority of interactions being either ionic\nor hydrophobic. Chou et al (26) observed that both enzyme dimerization and activity decrease\nat higher salt concentrations and at lower pH, and that a salt bridge between Arg4 and Glu290\nwas important for these interactions. Mutation of Glu290 to Ala resulted in a complete loss of\ndimerization and activity while a mutation in Arg4 resulted in an approximately five-fold loss\nin dimerization and only a moderate loss of activity. Another study showed that deletion of the\nfirst three residues of the N-terminus resulted in only a moderate loss in activity and\ndimerization, while deletion of the first four residues of the N-terminus resulted in a dramatic\nloss in both areas, supporting the importance of Arg4 (28). Furthermore, it has been observed\nthat deletion of the first seven residues at the N-terminus of 3CLpro interfered with protease\nactivity and dimerization (16,28), though one group reported diminished activity but no change\nin oligomeric state when the same mutation was made (29). Mutation of Met6 to Ala completely\ninterferes with both dimerization and enzymatic activity (16). Met6 is proposed to form\nhydrophobic interactions with Tyr126 and Phe140 of the opposing chain. In addition, two\npeptides derived from the N-terminus of the protein have been shown to inhibit the activity of\n3CLpro, one of which has been shown to prevent dimerization, supporting the idea that the N-\nterminus plays a role in maintaining the oligomeric state of 3CLpro (8,16). Deletion of the third\nalpha helical domain also interferes with dimerization and activity [(27-29)and unpublished\ndata from this laboratory]. The third domain alone has been observed to dimerize on its own,\nleading to the hypothesis that the role of this domain may be the regulation of enzymatic activity\nthrough dimerization (27).\nAll of the mutations affecting the dimerization of SARS 3CLpro reported so far involve residues\nwhich are located directly at the dimer interface. Here we provide data which demonstrates\nthat dimerization is also controlled by long range cooperative interactions in 3CLpro. Mutation\nof conserved Ser147 to Ala, located approximately 9 Å away from the dimer interface, inhibited\ndimerization and resulted in an inactive enzyme. The location of Ser147 suggests alternative\nsites for SARS 3CLpro dimerization inhibitors.\nEXPERIMENTAL PROCEDURES\ncDNA encoding full-length SARS 3CLpro (Tor2 strain, GenBank entry AY274119) was\npreviously cloned into a pET 100 vector (Champion pET Directional TOPO Expression and\nCloning Kit, Invitrogen) which carries an N-terminal polyhistidine tag, an enterokinase\ncleavage site, and ampicillin resistance (5). Mutations at selected positions (S139A, S144A\nand S147A) were introduced using an in vitro site directed mutagenesis kit (QuickChange,\nStratagene) with the pET-SARS 3CLpro vector as the template. The sequences of the primers\nused to generate the mutants were as follows: S139A: 5′ CCA TTA AAG GTG CTT TCC TTA\nATG G 3′; S144A: TTC CTT AAT GGA GCA TGT GGT AGT G 3′; and S147A: GGA TCA\nTGT GGT GCT GTT GGT TTT AAC ATT G 3′. Following the PCR reaction carried out with\nPfu turbo DNA polymerase (Stratagene), the parent template was degraded with a DpnI\n(Stratagene) digestion reaction for 1 hour at 37 °C. One shot TOPO10 competent cells\n(Invitrogen) were transformed with the PCR product for plasmid amplification. The plasmid\nwas then purified and mutations confirmed by DNA sequencing. This procedure was used to\ngenerate single, double and triple mutations of the serines to alanine using plasmids and primers\ncorresponding to the residue change.\nProtein Expression and Purification\nRecombinant SARS 3CLpro was expressed as a soluble fraction in BL21 Star DE3 E. coli\ncompetent cells (Invitrogen). The construct begins with residue Ser1, and therefore does not\ncontain the full N-terminal auto-cleavage site of the protein. Cells were grown in LB\nsupplemented with ampicillin (50 μg/mL) at 37 °C, induced with 1 mM IPTG when the optical\ndensity (as determined by absorbance at 600 nm) was 0.8 or greater, and harvested after 4\nhours. Cells were resuspended in lysis buffer (50 mM potassium phosphate (pH 7.8), 400 mM\nsodium chloride, 100 mM potassium chloride, 10% glycerol, 0.5% Triton-X, and 10 mM\nimidazole). The cells were broken by sonicating on ice for short pulses of one second followed\nby three seconds off for a total of 16 minutes. Cell debris was collected by centrifugation\n(20,000g at 4 °C for 45 min). The supernatant was filtered using a 0.45 μm pore size filter\n(Millipore) and applied directly to a nickel affinity column (HiTrap Chelating HP, Amersham\nBiosciences) that had been pre-equilibrated with binding buffer (50 mM sodium phosphate,\n0.3 M sodium chloride, 10 mM imidazole, pH 8.0). The protease was eluted with a linear\ngradient of 50 mM sodium phosphate, 0.3 M sodium chloride, 250 mM imidazole, pH 8.0.\nAfter elution, the protein was buffer exchanged into 10 mM Tris-HCl pH 7.5, and loaded onto\na Q-sepharose anion exchange column (Amersham Biosciences). The protease was eluted with\na gradient of 10 mM Tris-HCl, 1 M NaCl, pH 7.5. The pooled fractions containing 3CLpro\nwere buffer exchanged into storage buffer (10 mM sodium phosphate, 10 mM sodium chloride,\n1 mM TCEP, 1 mM EDTA, pH 7.4) and digested for 24 h at 4 °C with enterokinase (Invitrogen,\n0.1 units per 112 μg of protease) to remove the N-terminal polyhistidine tag. The enterokinase\nwas removed by incubation with EK-away resin (Invitrogen). The reaction mixture was passed\nthrough a nickel affinity column to remove undigested protease. The protease was buffer\nexchanged into storage buffer, concentrated to 10 mg/mL and used immediately for\nexperiments. The sample was more than 95% pure, as assessed by SDS-PAGE.\nKinetics\nThe catalytic activities of wild type and mutant SARS 3CLpro were determined using a\nfluorescence-based peptide cleavage assay with a commercially available fluorogenic\nsubstrate, Dabcyl-KTSAVLQSGFRKME-Edans (Genesis Biotech, Taiwan), which\ncorresponds to the N-terminal auto-cleavage site of the protease (24). The change in\nfluorescence intensity was monitored in a Cary Eclipse fluorescence spectrophotometer\n(Varian) with 355 nm and 538 nm excitation and emission wavelengths respectively. The\nexperiments were performed in 10 mM sodium phosphate, 10 mM sodium chloride, 1 mM\nEDTA, 1 mM TCEP, pH 7.4. Kinetic parameters such as Km and kcat were determined by initial\nrate measurement of substrate cleavage at 25 °C. The reaction was initiated by the addition of\nsubstrate such that final concentration varied from 1-60 μM.\nAnalytical Ultracentrifugation\nSedimentation velocity and equilibrium experiments were conducted using a Beckman-Coulter\nXL-I analytical ultracentrifuge. Wild type and mutant samples were prepared by exhaustive\ndialysis into 10 mM Tris-HCl, 0.1 M NaCl, 1 mM TCEP, 1 mM EDTA, pH 7.5 and then\nconcentrated to 1 mg/mL. For sedimentation velocity experiments, reference (420 μL) and\nsample (400 μL) solutions were loaded into double-sector centerpieces and mounted in a\nBeckman An60Ti rotor. Protein concentrations for these experiments were between 0.25mg/\nml (7.4 μM) and 1 mg/ml (29.5 μM). Experiments were performed at 20 °C with a rotor speed\nof 50,000 rpm. Sample absorbance at 280 nm was monitored in a continuous mode with no\ndelay and a step size of 0.003 cm without averaging. Multiple scans at different time points\nwere fit to a continuous size distribution using SEDFIT version 9.3b (30). The c(s) distribution\nin SEDFIT was exported to SEDPHAT where experimental values were corrected to S20, w\n(31). A partial specific volume of 0.7331 cm3/g for wild type 3CLpro and 0.7334 cm3/g for the\nS147A mutant were used in all calculations based on the amino acid sequence of the protein.\nThe solvent density and viscosity were calculated using SEDNTERP (Philo, J. website\nhttp://www.jphilo.mailway.com/default.htm). Sedimentation equilibrium experiments were\nperformed in the same buffer as sedimentation velocity experiments at concentrations between\n0.1 and 2 mg/ml (between 2.9 μM and 59.1 μM). Reference (120 μl) and sample (110 μl)\nsolutions were loaded into six sector centerpieces and mounted in a Beckman An60Ti rotor.\nSamples were equilibrated at 20 °C and three speeds: 15,000 rpm, 20,000 rpm and 25,000 rpm.\nSample absorbance at 280 or 250 nm was collected in step mode with a step size of 0.001 cm\nand 10 replicates at each radial position. The program WinMatch (Biotechnology-Bioservices\nCenter, University of Connecticut) was used to determine the point at which samples reached\nequilibrium. Equilibrium was typically attained for all samples between 24 and 30 hours. All\nsamples were visually inspected for clarity to ensure no precipitation had occurred during the\nruns. Dissociation constants were determined by global fitting of multiple concentrations and\nrotor speeds in SEDPHAT using the monomer-dimer equilibrium model (32).\nDifferential Scanning Calorimetry\nThe heat capacities of wild type 3CLpro and the serine to alanine mutants were measured as a\nfunction of temperature with a high-precision differential scanning VP-DSC microcalorimeter\n(Microcal Inc., Northampton, MA). Protein samples and reference solutions were properly\ndegassed and carefully loaded into the cells to avoid bubble formation. Thermal denaturation\nscans were performed with freshly prepared buffer-exchanged protease solutions using a\ntemperature scan rate of 1°C/min. Data was analyzed by software developed in this laboratory.\nRESULTS AND DISCUSSION\nEnzymatic Activity\nWe previously identified a highly conserved cluster of serine residues: Ser139, Ser144 and\nSer147, which lie adjacent to the active site of SARS 3CLpro (5). The location of this serine\ncluster in the structure of 3CLpro is shown in Figure 1A. Serines 139 and 147 are absolutely\nconserved in all known coronavirus 3CL proteases, while Ser144 is conserved in 55% of the\nreported sequences.\nThe role played by the three serine residues in the catalytic activity of SARS 3CLpro was\nevaluated by generating serine to alanine mutants at the three positions. The catalytic activities\nof wild type 3CLpro and the mutants S139A, S144A and S147A were measured at 1 μM enzyme\nconcentration using a fluorescence-based cleavage assay in order to obtain the catalytic rate\nconstants (kcat) and the Michaelis constant (Km). The results are shown in Figure 2. The wild\ntype 3CLpro has a Km of 56.42 ± 9.18 μM and a kcat of 0.643 ± 0.051 sec−1. The resulting\nKm and kcat are of the same magnitude as those previously reported with the same substrate\n(24). Table 1 shows the kinetic parameters obtained for wild type, S139A, S144A and S147A.\nAll of the proteases exhibited very similar Km's, suggesting that their ability to bind the substrate\nwith similar affinities was not impaired. S139A had a catalytic constant, kcat, similar to the\nwild type while the S144A value was about half. The most significant effect was observed for\nS147A which had a 160-fold drop in kcat with respect to the wild type enzyme. Consistent with\npreviously published observations(23,33), the activity of the wild type SARS 3CLpro and the\nthree mutants showed a linear correlation with increasing enzyme concentration as shown in\nFigure 3.\nStructural Stability of Wild Type 3CLpro and Serine Mutants\nTo verify that the mutants were properly folded and to assess the effect of the serine to alanine\nmutations, the structural stability of the wild type and mutant proteases was measured by\ndifferential scanning calorimetry. Figure 4 shows the temperature dependence of the heat\ncapacity function of wild type 3CLpro, S139A, S144A and S147A at pH 7.4 and 10 mM NaCl.\nThe denaturation transition for wild type 3CLpro is centered at 56.50 ± 0.1°C at a protein\nconcentration of 0.2 mg/mL (5.9 μM) and temperature scanning rate (1°C/min) used in these\nexperiments. This value is approximately 4°C higher than our previously reported value (5)\nwhich was measured using a construct containing five additional amino acids at the N-terminus\nof the protein. The transition temperature is similar to the value obtained by circular dichroism\n(CD)(27), but lower than others reported values (~60°C) obtained at faster scanning rates\n(29), as expected for irreversible denaturation. The S139A mutant had a slight effect on the\nstability on the protease with a downward shift in Tm of 0.9°C. The S144A mutation resulted\nin a 1.9°C upward shift in the transition temperature. The S147A mutation resulted in a\ndownward shift in the transition temperature of approximately 1.7°C. Besides the changes in\ndenaturation temperature, these experiments confirmed that all four proteins were properly\nfolded and indicate that the serine to alanine mutations did not significantly affect the overall\nstability of the recombinant proteins. Furthermore, similar results were obtained throughout\nthe length of time required for other experiments (e.g. sedimentation equilibrium\nmeasurements) confirming the structural integrity of the proteins in all experiments presented\nhere. The areas under the curves, ΔH, were also similar for all proteins, indicating the absence\nof any significant structural changes. A full thermodynamic analysis could not be performed\nwith any of the proteins due to precipitation once the transitions were complete which lead to\na lack of reversibility in the microcalorimetric scans. This irreversibility was observed\nregardless of buffer conditions used (pH values between 3 and 9; NaCl concentrations between\n0 and 3M).\nQuaternary Structure of Wild Type 3CLpro and Serine Mutants\nTo investigate the effects of the serine to alanine mutations on the quaternary structure of SARS\n3CLpro, sedimentation velocity ultracentrifugation was performed at 20°C on all constructs at\nconcentrations between 0.25 mg/ml (7.4 μM) and 1 mg/ml (29.5 μM). Figure 5 shows typical\nsedimentation velocity profiles at 0.5mg/ml (14.8 μM) obtained using direct boundary\nmodeling with the program SEDFIT (30). The experimental data is well accounted for by the\ncontinuous sedimentation coefficient model, c(s), as indicated by the plot of the residuals in\nFigure 5B. The sedimentation data for wild type 3CLpro in Figure 5C shows the presence of\ntwo peaks, at S1 = 2.5 and S2 = 4.0. These results are consistent with previous reported values\nin the literature (16,26,28). There did not appear to be a concentration dependence of the peak\npositions of c(s) in the concentration range tested. Similar results were obtained for the S139A\nand S144A proteins, shown in figures 5D and 5E. Despite being located directly at the dimer\ninterface, Serine 139 does not play a dominating role in enzyme dimerization. Similarly, even\nthough kinetic measurements show that S144A is less active than the wild type, the\nsedimentation profile was similar, albeit with a slight decrease in dimer population, suggesting\nthat this mutation directly affects catalysis. Like the wild type, neither of these mutants showed\na concentration dependence on the peak position of c(s).\nAnalysis of the S147A sedimentation velocity data unexpectedly produced a primary peak at\n2.7S, as shown in Figure 5F. An additional small, broad peak can be seen at ~4.0S,\ncorresponding to the formation of a minimal amounts of dimer. This peak can be seen to\nincrease slightly with increasing protein concentration. The peak positions of this mutant also\nremained constant within the range of the protein concentrations tested. The presence of the\nS147A dimer is short lived however, as ultracentrifugation experiments performed later than\ntwenty four hours after purification yielded solely a monomeric species. This correlates well\nwith the lack of enzymatic activity observed in the kinetics measurements.\nTo obtain dimer dissociation constants for the four constructs, sedimentation equilibrium\nexperiments were performed with wild type 3CLpro, S139A, S144A and S147A. Global\nanalysis for each of the proteins at multiple concentrations between 0.1 mg/ml (2.9 μM) and\n2 mg/ml (59.1 μM), and at multiple rotor speeds (15,000 rpm, 20,000 rpm, and 25,000 rpm)\nwas carried out with the program SEDPHAT using the monomer-dimer self association model\n(32). Figure 6 shows the results of the global fitting of each set of data at 0.5 mg/ml (14.8\nμM). All sets of experimental data fit well to the monomer-dimer model; the molecular weight\nobtained for the wild type monomeric species was 33,889 ± 441 daltons, which correlates\nexceedingly well with the expected monomeric weight of 33,846 kDa as estimated by the\nprotein sequence. The dissociation constant for the wild type 3CLpro dimer was determined to\nbe 1.3 ± 1.8 μM, which is slightly higher than previously reported values of 0.19 μM (26) and\n0.28 μM (28) obtained using sedimentation velocity, and slightly lower than a recently reported\nKd value of 14 μM obtained also by sedimentation equilibrium ultracentrifugation (16). The\nslight discrepancy between the two sedimentation equilibrium measurements may be due to\nthe extra amino acid on the N-terminus of the protein used in the previously published work.\nIt has been observed that additional amino acids on the N-terminus of the protease can increase\nthe measured Kd value or interfere with dimerization completely [(34) and unpublished data\nfrom this laboratory]. If additional amino acids corresponding to a 3CLpro cleavage site are\nadded to the N-terminus of the protein, the protein is able to cleave them off through trans-\nprocessing, albeit with a lower dimerization Kd. If the added residues do not correspond to a\ncleavage site, as with the previously published sedimentation equilibrium experiments, then\nthe extra residue cannot be cleaved off and may have an effect on the dimerization Kd.\nDissociation constants for the three mutants were also obtained, the results of which are shown\nin Figure 6 and Table 1. S139A showed a slight decrease in its ability to dimerize with a Kd\nof 12.9 ± 1.7 μM, and a molecular weight of the monomeric species calculated as 32,737 ±\n342 daltons. The decrease in catalytic efficiency is less than the one expected from the weaker\ndimerization constant, suggesting a better intrinsic catalytic activity of the mutant over the wild\ntype. On the other hand, the decrease in catalytic activity observed with S144A was not\nmirrored by a large decrease in dimerization, confirming the sedimentation velocity results.\nThe dimerization Kd obtained for the mutant was 3.3 ± 1.4 μM and the molecular weight\nobtained for the monomer was 33,466 ± 1121 daltons. The reduction in S144A activity is\napparently not due to a weaker dimerization, but to a direct effect on the catalytic activity of\nthe enzyme. It must be noted that this enzyme is structurally more stable than the wild type\n(Table 1) and that this added stability might reflect itself in a lower catalytic efficiency.\nThe sedimentation equilibrium profile of S147A fit extremely well to a monomeric distribution,\nas evidenced by the well dispersed residuals seen in Figure 6D. The molecular weight obtained\nfor the monomer was 32,641 ± 210 daltons, a value consistent with that obtained from the\nsedimentation velocity experiments. If we assume that only the dimer is active and that the\nS147A mutant dimer has an intrinsic kcat similar to that of the wild type, the Kd for S147A\ncould be expected to be around 300 μM; i.e. more than two orders of magnitude weaker than\nthe wild type.\nIn summary, we have carried out mutational analysis on a cluster of conserved serine residues\n(Ser139, Ser144 and Ser147) found adjacent to the active site of SARS 3CLpro. We examined\nthe effects of separately mutating each of these residues to alanine on the activity, stability and\noligomeric state of the protease using a fluorescence-based cleavage assay, differential\nscanning calorimetry, sedimentation velocity ultracentrifugation and sedimentation\nequilibrium sedimentation. Ser139, which makes interactions at the dimer interface, does not\nappear to play a significant role in maintaining the dimeric state of the protease, showing only\nslight decrease in activity, thermal stability and dimerization as compared to wild type\n3CLpro when mutated to alanine. Ser144, located closer to the active site cavity, showed a two-\nfold decrease in catalytic efficiency when mutated to alanine compared to that of the wild type,\nbut maintained a similar dimeric state and showed an increase in thermal stability by 1.9°C.\nMutation of Ser147, located approximately 9 Å away from the dimer interface, to alanine\nresulted in an approximately 150-fold loss in catalytic efficiency in comparison to the wild\ntype, and an inability of the protease to dimerize. This demonstrates that dimerization in SARS\n3CLpro can be controlled by long range cooperative interactions, suggesting that targeting\ncavities nearby Ser147 with small molecule inhibitors may provide an alternative way of\ninactivating the protease.\nStructural Analysis\nAccording to the existing crystallographic structures, serine 139 is the only residue of the three\nconsidered in this paper directly located at the dimerization interface. Serines 144 and 147 are\ncloser to the active site cavity and by visual inspection do not appear to play a key role in the\ndimerization of the enzyme. The interactions made by Ser139 are not bidirectional. Figure 1B\nshows the residues surrounding Ser139 of chain A at pH 7.5 [pdb structure 1UK3 (18)]. In this\nstructure, Ser139 of chain A makes hydrogen bonds to the backbone of Gly2 from chain B.\nConversely, no side chain interactions are made by Ser139 of chain B with residues in chain\nA. Mutation of Ser139 to Ala disrupts the dimerization interface and weakens the dimerization\nconstant by about 10-fold but does not interfere directly in the catalytic activity of the enzyme.\nSerine 144 is located closer to the active site in comparison to Ser139. Figure 1C shows the\nresidues surrounding Ser144. The side chain hydroxyl of Ser144 forms hydrogen bonds with\nthe side chain and backbone of Asn142. The backbone carbonyl of Ser144 interacts with the\nbackbone amide of Ser147, and the side chain of Asn28. Mutation of Ser 144 to Ala results in\na more structurally stable enzyme as reflected in the calorimetric data. The Km and the\ndimerization constant remain essentially the same while the catalytic constant, kcat, is reduced\nby half, suggesting that the structural rearrangement induced by the mutant reduces the catalytic\nefficiency of the enzyme.\nThe Ser147 to Ala mutation induces the most dramatic activity and structural effects even\nthough this residue is 9Å away from the dimer interface. Ser147 makes several backbone and\nside chain interactions with residues which are involved in an extensive hydrogen bonding\nnetwork within domains I and II in the protein. The backbone of Ser147 interacts with the\nbackbones of residues Ser144 and His163. Incidentally, His163 is important for catalytic\nactivity, since it comprises the S1 subsite that contributes to the specificity of substrate binding\n(18). The side chain of Ser147 makes two interactions with surrounding residues. The side\nchain hydroxyl of Ser147 hydrogen bonds to the backbones of Cys117 and Ser144. Cys117\nforms two additional hydrogen bonds with Asn28 and Leu115. Asn28 may be important in\nmaintaining the structural integrity in and around the active site region of the protease. This\nresidue makes several key interactions, including two interactions with the backbone carbonyl\nof the catalytic residue Cys145. The hydrogen bonds made by Asn28 to the backbones of\nCys117 and Gly120 may be critical in maintaining the position of the beta strand on which\nthey reside, which is connected to and runs anti-parallel to the beta strand containing Tyr126.\nIn turn, Tyr126 forms a hydrophobic pocket with Phe140 of the same chain and Met6 of the\nopposing chain. Met6 is the closest residue of the opposing monomer to Ser147, which is\npositioned approximately 9Å away. As previously mentioned, there is evidence suggesting that\nMet6 plays a role in maintaining dimerization, potentially through this hydrophobic interaction\nwith Tyr126 and Phe140 (16). The loss of Asn28 interactions with Cys117 and Gly120 may\nresult in the repositioning of this beta sheet and the disruption of this hydrophobic interaction.\nAdditionally, structural rearrangements due to the loss of these interactions may also interfere\nwith the hydrogen bond that Asn28 makes with the backbone carbonyl Gly143. Gly143 lies at\nthe top of the loop which contains Phe140 which is part of the hydrophobic pocket.\nFurthermore, the elimination of the hydroxyl group in the S147A mutation not only eliminates\nthe hydrogen bonds made by this group but also allows some rearrangements due to the smaller\nvan der Waals volume of the Ala mutation.\nConclusions\nThese studies reveal the possibility of new strategies for engineering inhibitors of SARS\n3CLpro by targeting sites that are away from the dimer interface but involved in the allosteric\ncontrol of dimerization. Since Ser147 is conserved in all known coronavirus 3CL proteases,\nthis observation opens the intriguing possibility that small molecules targeting this site may\nhave a broad antiviral spectrum against infections caused by different coronaviruses.",
         "",
         "Long-range cooperative interactions modulate dimerization in SARS 3CLpro.",
         "None"
        ],
        [
         "28",
         "001a3a8f9bcd143329d679da7b04980946bf1c78",
         "In an attempt to decrease mortality in dialysis patients, the care for end-stage renal disease patients has been a focus of nephrologists for many years. Good pre-dialysis care has been found to improve mortality numbers in several studies. However, it is still not exactly clear, when the best moment has arrived to start renal replacement therapy. By illustrating two totally different patient cases the effect of outcome and quality of life are discussed-demonstrating on the one hand the role of clinical parameters, and on the other hand the influence of biochemical parameters.",
         "C. Douma,W. Smit",
         "None",
         "https://academic.oup.com/ndt/article-pdf/21/suppl_2/ii20/6835586/gfl139.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://academic.oup.com/crawlprevention/governor?content=%2fndt%2fOUP%2fRunPdfHandler%3fjournal%3dndt%26volume%3d21%26issue%3dsuppl_2%26page%3dii20%26multimediaId%3d6835586",
         "None",
         "",
         "When to start dialysis?",
         "None"
        ],
        [
         "29",
         "001b4ffaab6cdc560e071cb0dcb2680269480829",
         "None",
         "C. Gualtieri,L. Johnson",
         "None",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "None",
         "Efficient Allocation of Attentional Resources in Patients With ADHD;Neurocognitive testing supports a broader concept of mild cognitive impairment;Validity of a novel computerized cognitive battery for mild cognitive impairment;Test-retest reliability, practice effects and reliable change indices for the recognition memory test.;The reliability and stability of verbal working memory measures;Further standardization of some NES3 tests.;Development and validation of a web-based neuropsychological test protocol for sports-related return-to-play decision-making.;The effects of practice on the cognitive test performance of neurologically normal individuals assessed at brief test-retest intervals.;A Verbal Learning and Memory Test for English- and Spanish-speaking Older Mexican-American Adults;Factor Structure, Normative Data and Retest-Reliability of a Test of Fine Motor Functions in Patients With Idiopathic Parkinson's Disease;Note on Comparability of Microcog Test Forms;Development of a general ability index for the Wechsler Adult Intelligence Scale--Third Edition.;Computerized Neurocognitive Scanning: I. Methodology and Validation in Healthy People;Attentional Function as Measured by a Continuous Performance Task in Children with Dyscalculia;Convergent Validity and Neuropsychological Correlates of the Schedule for the Assessment of Negative Symptoms (SANS) Attention Subscale;Effects of stimulants on the continuous performance test (CPT): implications for CPT use and interpretation.;Test–retest bias, reliability, and regression equations for neuropsychological measures repeated over a 12–16-week period;MicroCog: Assessment of Cognitive Functioning;Neuropsychological Assessment of Response Inhibition in Adults With ADHD;Continuous Performance Tests Are Sensitive to ADHD in Adults but Lack Specificity;Effects of chronic nicotine and methylphenidate in adults with attention deficit/hyperactivity disorder.;Effects of moderate and high doses of alcohol on attention, impulsivity, discriminability, and response bias in immediate and delayed memory task performance.;Establishing stable test performance in tests from the Behavioral Assessment and Research System (BARS).;Does the Conners' Continuous Performance Test Aid in ADHD Diagnosis?;Validity of a computer-assisted neurobehavioral test battery in toxicant encephalopathy.;Relationship of CogScreen-AE to flight simulator performance and pilot age.;Recognition Memory for Faces: Reliability and Validity of the Warrington Recognition Memory Test (RMT) in a Neurological Sample;Test–retest reliability and practice effects of Expanded Halstead–Reitan Neuropsychological Test Battery;Handbook of Normative Data for Neuropsychological Assessment;Test\\re-test reliability of the CANTAB and ISPOCD neuropsychological batteries: theoretical and practical issues;Neural activity during attention shifts between object features;Computerized self‐assessment in psychiatric in‐patients: acceptability, feasibility and influence of computer attitude;Clinical utility of the Test of Variables of Attention (TOVA) in the diagnosis of attention-deficit/hyperactivity disorder.;4 T-fMRI study of nonspatial shifting of selective attention: cerebellar and parietal contributions.;Smith's Recognizable Patterns of Human Malformation;A comparison of NES2 and traditional neuropsychological tests in a neurologic patient sample.;Screening for Cognitive Impairment in Older Individuals: Validation Study of a Computer-Based Test;Gender- and Age-Specific Changes in Motor Speed and Eye-Hand Coordination in Adults: Normative Values for the Finger Tapping and Grooved Pegboard Tests;A Compendium of Neuropsychological Tests: Administration, Norms, and Commentary;A methodological review of “method skeptic” reports;Computerized assessment in neuropsychology: A review of tests and test batteries;Concise Guide to Child and Adolescent Psychiatry;Performance of four age groups of normal elderly on the Rey Auditory-Verbal Learning Test.;Neurobehavioral effects of phenytoin prophylaxis of posttraumatic seizures.;The cognitive drug research computerized assessment system for demented patients: A validation study;Test-retest reliability of the halstead-reitan battery and the WAIS in a neuropsychiatric population;Neuropsychological deficit and emotional disturbance in head-injured patients.;Finger Tapping: Effects of Trials and Sessions;A computer-based neurobehavioral evaluation system for occupational and environmental epidemiology: methodology and validation studies.;Use of Memory Tests in Differentiating Organic Disorder from Depression;The computerized continuous performance task: A new measure of inattention;Psychomotor function and psychoactive drugs.;A review of stimulant drug research with hyperactive children.;EFFECTS OF REPEATED ADMINISTRATIONS OF A COMPREHENSIVE NEUROPSYCHOLOGICAL BATTERY AMONG CHRONIC EPILEPTICS;Sustained attention in remitted schizophrenics.;Attention in hyperactive children and the effect of methylphenidate (ritalin).;Perceptual‐Motor Behaviour in Relation to Blood Phenobarbitone Level: A Preliminary Report;ON THE NATURE OF THE \"ABSENCE\" IN CENTRENCEPHALIC EPILEPSY: A STUDY OF SOME BEHAVIORAL, ELECTROENCEPHALOGRAPHIC AND AUTONOMIC FACTORS.;L’examen Clinique en Psychologie;The effect on delayed-alternation test performance of stimulating or destroying electrically structures within the frontal lobes of the monkey's brain.;Self-administered screening for mild cognitive impairment: initial validation of a computerized test battery.;Pointing to an allocentric and egocentric remembered target.;RELIABILITY OF A NOVEL COMPUTERIZED NEUROPSYCHOLOGICAL BATTERY FOR MILD COGNITIVE IMPAIRMENT;Neuropsychological testing of high school athletes. Preliminary norms and test-retest indices.;Relationship of CogScreen-AE to flight simulator performance and pilot age.;Validity of a cognitive computerized assessment system in brain-injured patients.;Domains of behavior from a neuropsychological perspective: the whole story.;Alternate forms of the AVLT: a procedure and test of form equivalency.;An investigation of the test-retest reliability of the Stroop Color-Word Test across two intervals.;Symbol Digit Modalities Test (SDMT). Manual (Revised). Los Angeles;Stroop Color and Word Test: Manual for Clinical and Experimental Uses.;Recognizable patterns of human malformation.;The appraisal of children with cerebral deficits;The effect on delayed-alternation test performance of stimulating or destroying electrically structures within the frontal lobes of the monkey's brain.",
         "Reliability and validity of a computerized neurocognitive test battery, CNS Vital Signs.",
         "None"
        ],
        [
         "30",
         "001b56d111a9c856159f0d08738d8158aabfe621",
         "Recent studies show that plant organ positioning may be mediated by localized concentrations of the plant hormone auxin. Auxin patterning in the shoot apical meristem is in turn brought about by the subcellular polar distribution of the putative auxin efflux mediator, PIN1. However, the question of what signals determine PIN1 polarization and how this gives rise to regular patterns of auxin concentration remains unknown. Here we address these questions by using mathematical modeling combined with confocal imaging. We propose a model that is based on the assumption that auxin influences the polarization of its own efflux within the meristem epidermis. We show that such a model is sufficient to create regular spatial patterns of auxin concentration on systems with static and dynamic cellular connectivities, the latter governed by a mechanical model. We also optimize parameter values for the PIN1 dynamics by using a detailed auxin transport model, for which parameter values are taken from experimental estimates, together with a template consisting of cell and wall compartments as well as PIN1 concentrations quantitatively extracted from confocal data. The model shows how polarized transport can drive the formation of regular patterns.",
         "H. Jönsson,M. Heisler,B. Shapiro,E. Meyerowitz,E. Mjolsness",
         "H. Jönsson,M. Heisler,B. Shapiro,E. Meyerowitz,E. Mjolsness",
         "https://europepmc.org/articles/pmc1326488?pdf=render",
         "Recent studies show that plant organ positioning may be mediated\nby localized concentrations of the plant hormone auxin. Auxin\npatterning in the shoot apical meristem is in turn brought about by\nthe subcellular polar distribution of the putative auxin efﬂux\nmediator, PIN1. However, the question of what signals determine\nPIN1 polarization and how this gives rise to regular patterns of\nauxin concentration remains unknown. Here we address these\nquestions by using mathematical modeling combined with confo-\ncal imaging. We propose a model that is based on the assumption\nthat auxin inﬂuences the polarization of its own efﬂux within the\nmeristem epidermis. We show that such a model is sufﬁcient to\ncreate regular spatial patterns of auxin concentration on systems\nwith static and dynamic cellular connectivities, the latter governed\nby a mechanical model. We also optimize parameter values for the\nPIN1 dynamics by using a detailed auxin transport model, for which\nparameter values are taken from experimental estimates, together\nwith a template consisting of cell and wall compartments as well\nas PIN1 concentrations quantitatively extracted from confocal\ndata. The model shows how polarized transport can drive the\nformation of regular patterns.",
         "",
         "emerge at well defined positions, resulting in strikingly regular\npatterns (1). These phyllotactic patterns can be whorled (more\nthan one new primordium develops simultaneously) or spiral\n(single primordia are created sequentially). Spiral phyllotaxis is\noften connected to the Fibonacci series because the numbers of\nparastichies (visible spirals) in each direction around the axis are\ncommonly consecutive Fibonacci numbers. Also, consecutive\nprimordia in the spiral often appear at a divergence angle close\nto the golden angle. The beautiful symmetries apparent in\nphyllotaxis and its connection to mathematics have inspired\nscientists to create theories and models to explain these patterns.\nOne important finding from mathematical analysis and physical\nsimulation (2, 3) is that many of the seemingly complex phyllo-\ntactic patterns and transitions found in plants can probably be\nexplained to a large degree by any regular spacing mechanism\nsuperimposed on a gradually enlarging generative region. This\nfinding is important because it suggests that the problem can be\nreduced to two potentially independent and smaller questions.\nThe first question regards how meristem size is determined\nduring plant development. The second question involves how a\nregularly spaced pattern of primordial position is specified.\nTraditionally, models have concentrated on the second question,\nand they can be divided into molecular and mechanical ideas\n(4–6). Because we investigate a model based on molecular\nexperiments, here we discuss some molecular-based ideas in\nmore detail. Schoute (7) first proposed the idea of lateral\ninhibition based on a diffusible chemical produced by each\ndeveloping primordium that inhibits the initiation of primordia\nnearby. The inhibition hypothesis has been studied extensively in\niterative models (3, 8) and has been shown to be capable of\nproducing common phyllotactic patterns for which different\nparameter values specifying inhibition range relative to a gen-\nerative region lead to different stable phyllotactic patterns (3).\nContinuous changes in these parameters were used to investigate\ntransient phases between these patterns relevant to a growing\nplant (8). Relating back to the work of Turing (9), Meinhardt\n(10) introduced a reaction–diffusion version of the inhibitor\nmodel by incorporating an activator molecule as well as an\ninhibitor, allowing for a continuous dynamical model. These\nmodels exhibit robust and dynamic pattern generation, including\nthe initiation of new concentration peaks corresponding to\nprimordia (10).\nAn alternative idea is based on competition or depletion of a\nprimordium-promoting factor and was proposed by Preistley and\nScott in 1933 (11) and by Mitchison in 1977 (2) and Chapman\nand Perry in 1987 (12). Recent (and old) findings have provided\nconsiderable support for such a depletion scenario. In a series of\nelegant experiments, it has been shown that the plant hormone\nauxin (in the form of indole-3-acetic acid) is an essential\nactivator for primordium formation (13, 14). Plants in which\nauxin transport is blocked (either chemically or in the pin1\nmutant) exhibit a pin-formed morphological phenotype charac-\nterized by a lack of primordium development and a bare\nmeristem. This phenotype can be rescued by local application of\nauxin in the form of a lanolin paste, showing that localized auxin\nis both necessary and sufficient for primordial development. In\nthe wild type, auxin transport is mediated by the Arabidopsis PIN\n(PINFORMED) family of putative auxin efflux mediators (15)\nas well as the auxin import mediators AUX1 and its relatives\n(16). However, the principle protein required for primordium\ndevelopment appears to be PIN1 because pin1 mutants lack\nfloral primordia in contrast to the relatively mild phenotypes, so\nfar, of other auxin transport mutants. In the shoot apical\nmeristem (SAM), the PIN1 protein is expressed mainly in the\nepidermal (L1) layer of cells and it is polarized toward newly\nforming primordia (Fig. 1) (14). In young primordia, PIN1 is\npolarized downwards into the subepidermal layers, presumably\ninitiating vascular differentiation. In the epidermis below the\nSAM, PIN1 is polarized upwards toward the SAM, and this\npolarization depends on the PID (PINOID) protein (17).\nConsidering these data, Reinhardt et al. (14) have proposed\nthat auxin is a primordium activator that is depleted from\nprimordial regions via PIN1-dependent auxin transport so that\nauxin reaches the next critical threshold for initiation at the point\nfarthest from the previous point of depletion. One limitation of\nthis class of model would appear to be in generating whorled\npatterns for which multiple positions are specified simulta-\nneously. Another equally important issue not explained by the\nReinhardt et al. (14) proposal is how auxin transport itself is\npatterned. The objective of this work is to propose a model for\nprimordial positioning that is based on the assumption that PIN1\npolarity is regulated by relative auxin concentrations in neigh-\nboring cells within a two-dimensional space, such as the epider-\nmis. The main elements of our model include (i) passive and\nactive auxin transport, in which PIN1 mediates auxin efflux; (ii)\na model for PIN1 cycling between internal and membrane\ncompartments, in which auxin regulates the polarization of PIN1\nby modulating the cycling parameters in different directions; and\n(iii) changes in cell adjacency relationships determined by cel-\nlular growth and mechanics. The basis for the regulatory model\nis a feedback mechanism by which cells with relatively high auxin\nconcentrations increase their auxin content by influencing PIN1\npolarity in neighboring cells, which as a result become auxin-\ndepleted. Thus, from a close to homogeneous state, auxin peaks\nemerge at regular distances and determine the locations of early\nprimordia. These mechanisms were simulated on a meristem-\nlike surface topology by using a cell-based model including\ncellular growth and an elastic mechanics model (18). This model\nrealizes one direction of the two-way interaction between reg-\nulatory and mechanical networks outlined in ref. 19. The result-\ning auxin model generates spiral and whorled phyllotactic-like\npatterns as found in plants. A more detailed look at the PIN1\npolarization dynamics around the formation of a new peak\n(primordium) shows that the model is capable of predicting the\npolarization reversal in cells in between the new and older\nprimordia, which has recently been shown in experiments (20).\nWe also include a detailed auxin transport description based\non the chemiosmotic hypothesis of auxin transport using exper-\nimental estimates for auxin transport parameter values (21–23).\nUsing this model, we introduce a schema for evaluating and\noptimizing PIN1 cycling model parameters that compares the\nmodel predictions to quantified confocal microscopy data of\nfunctional PIN1 protein fused to GFP.\nResults\nAuxin Transport Model Combined with Extracted PIN1 Localization.\nWe first set out to test the behavior of the detailed auxin transport\nmodel (Eq. 3) by using the geometry and PIN1 localization ex-\ntracted from a confocal image (Fig. 1). The expectation is that cells\nforming new primordia have a relatively high auxin concentration.\nIn the model, most of the parameters are taken from experimental\nestimates, but the strength of active transport and the explicit PIN1\nconcentrations are unknown. In ref. 21, PIN1 is not explicitly\nmodeled and an anion permeability constant is inferred. We used\nthis value together with a maximal membrane PIN1 concentration\nset to 1 mol per unit of area. In addition, production and\ndegradation of auxin is allowed for. The details of the model and\nparameter values used are provided in Supporting Methods (see also\nFigs. 7 and 8 and Tables 1, 2, and 5, which are published as\nsupporting information on the PNAS web site). The equilibrium\nauxin concentrations resulting from these parameter values are\nshown in Fig. 2A. As can be seen, the newest visible primordium\n(P1) has a clear peak in auxin concentration. There is also a low\nconcentration region between the SAM and the next-older visible\nprimordium (P2), but there is no major peak of auxin concentration\nat this primordium, most likely because the primordium peak is\noutside the two-dimensional template. It can also be seen that the\ntwo primordial positions that form next have small peaks of auxin\nconcentrations (I1 and I2). Thus, the model seems to behave\nqualitatively as expected.\nTo investigate how robust this behavior is to different param-\neter changes, we simulated the model using a range of parameter\nvalues around the estimated values (Fig. 2 B and C; see also\nSupporting Methods and Figs. 9–11, which are published as\nsupporting information on the PNAS web site). The results show\nthat the model exhibits considerable stability and the qualitative\nresults are similar to Fig. 2 A. As an example, varying the wall pH\nresults in the auxin concentrations shown in Figs. 2 B and C.\nLowering the wall pH can be interpreted as an approximation to\nhow a symmetric influx mediator may influence the model. The\nconclusions drawn from these simulations are that the auxin\ntransport model combined with the extracted PIN1 localizations\nqualitatively behaves as expected and that this behavior is robust.\nAuxin-Driven PIN1 Cycling. Our central hypothesis is that the\nrelative concentrations of auxin in neighboring cells differen-\ntially drive the polarization of PIN1 to the corresponding portion\nof the membrane between each cell and its neighbors. To analyze\nthe dynamics of auxin transport in such a model, we simplify the\nmodel to a single parameter description. We use the cell-based\nmodel and simplify the auxin transport description (Eq. 2) by\nassuming that the PIN1-mediated transport is unsaturated,\nwhich leads to a model described by\n[1]\nAi(Ak) is the auxin concentration in compartment i(k),\nwhereas Pik and Pki are the PIN1 concentrations on the mem-\nbrane toward the neighboring compartment. D is the strength of\npassive transport, and T is the strength of the PIN1-dependent\nactive transport. The summation is over the set of neighbors, Ni,\nand the only parameters present are the strengths of the passive\nand active transports. We assume that the total amount of PIN1\nin the cell and its membrane is constant and equal for all cells\n(P  Pi\ntot  Pi  kPik, @i). We use a linear polarization feedback\n[f(Aj)  k1Aj] in Eq. 4 and assume that the PIN1 is in its\nequilibrium polarization state at any given time, which leads to\nPij  PAj[(k2k1)  kAk]. Finally, we assume that most PIN1\nis situated at the membrane (k2  k1), which leads to Pij \nPAj(k Ak) and a model in Eq. 1 only depending on auxin\nconcentrations and the three parameters D, T, and P, which can\nbe treated as a single parameter DTP.\nOptimizing the PIN1-Cycling Model. As described in Supporting\nMethods (see also Fig. 13, which is published as supporting\ninformation on the PNAS web site), the optimization is per-\nformed by simulating the detailed auxin model with extracted\nvalues of PIN1 concentration; it then uses the equilibrium auxin\nvalues for optimizing the PIN1 cycling model parameters [in Eq.\n4 using a Hill description for f(Aj)] to fit the extracted values. The\nobjective function landscape is quite smooth, resulting in a\n‘‘simple’’ optimization problem for which a local search algo-\nrithm with multiple restarts is sufficient to find good solutions.\nIn Fig. 4A the resulting PIN1 concentrations for the optimized\nmodel are plotted directly on the template to be compared with\nFig. 1D. Fig. 4B shows a quantitative comparison between\nextracted and optimized PIN1 concentrations, with a mean\nsquared error of 0.015. It should be stressed that we use extracted\nvalues for the total amount of PIN1 in each cell and that these\nvalues are different for individual cells. This variation introduces\na bias in the optimization measure. An indication of this bias is\ngiven by having each cell uniformly distribute its PIN1 content,\nwhich leads to a mean squared error of 0.031. To better\nappreciate the information in these numbers, more data should\nbe used together with comparisons between different kinds of\nPIN1 polarization models. In addition, optimization of some of\nthe more uncertain parameters within the auxin transport model\ncan be included in such an approach.\nPhyllotaxis Model on a Simplified Shoot Topology with Growth. To\ninvestigate the patterning dynamics of our model on a growing\nshoot, we simulate the cell-based auxin model in a system where\ncells are confined to a half-sphere connected to a cylinder. The\ncellular growth is uniform, the definition of the central zone is\ndynamic, and the auxin peaks are not fixed. Hence, these\nsimulations should be viewed as an initial test simply to see\nwhether patterning can be maintained in such a situation. Model\ndetails and parameter values used are given in the Supporting\nMethods. Time points from simulations for two different param-\neter sets are shown in Fig. 5. Illustrations of the complete\nsimulations are provided as Movies 1 and 2, which are published\nas supporting information on the PNAS web site. Fig. 5A shows\na simulation leading to a spiral-like phyllotaxis, with new peaks\nforming one at a time, and Fig. 5B shows a pattern in which the\npeaks form in pairs in a whorled-like (decussate) pattern. The\nparameter difference between the simulations is the size of the\ndefined central zone. It should be noted that the continuation of\nthese patterns is not extremely stable. Typically the temporal\norder in which new primordia form in the spiral-like pattern can\nsometimes reverse, and the radial position of the appearance of\npeaks and the time between the formation of two consecutive\npeaks are not constant, which can lead to switching between\ndifferent quasistable patterns in a single simulation. To a large\nextent, this instability is due to the noise introduced by the\ngrowthmechanical model, which allows for discrete jumps at\ncell division and local rearrangements of cells (leading to\nappearance and disappearance of neighbor pairs and the sliding\nof cells in respect to each other), that is not present in a growing\nplant. In addition, to achieve more stability, a more appropriate\ncell-based version of the model on a growing shoot-like topology,\nthis model is capable of producing both spiral and whorled\npatterns and recapitulates observed polarization reversals in the\nneighborhood of newly formed auxin peaks. These results show\nthat regulation of polarized auxin transport by auxin, together\nwith the mechanics of cell growth and neighborhood change,\nmay provide the underlying mechanisms for phyllotactic pat-\nterning. We also present a methodology that bridges the gap\nbetween theoretical models and experiments. Unlike previous\nmodels for generating phyllotaxis, this model is based on de-\ntailed experimental data. Auxin is now established as an essen-\ntial molecule required for primordium initiation. In turn, its\ndistribution in the shoot apical meristem depends on the activity\nof the PIN1 putative auxin efflux mediator. Our model includes\nthe chemiosmotic hypothesis of auxin transport, where transport\ndepends on the differential permeability of the protonated and\nneutral forms of the acid as well as the pH difference between\ncytoplasm and wall regions (24, 25). This study provides a\nsimulation of our present conception of auxin transport mech-\nanisms on a cellular template derived directly from real tissue.\nAlthough PIN1 has not been shown to directly transport auxin,\nits activity is essential for this process. Thus, we feel that we are\njustified in using its relative concentrations as a proxy for auxin\nefflux mediator concentrations. The main limitation of the\npresent technique is that it is limited to two dimensions because\nof the lack of resolution in the confocal z axis. For the purposes\nis\nof this study, our data are adequate because our model\ntwo-dimensional and is proposed to occur in the meristem\nepidermis. Although a model\nlimited to two dimensions is\ninadequate for describing all of the processes that occur in a\nthree-dimensional structure, such as the meristem, data suggest\nthat the L1 layer of the meristem may play a special role in auxin\ntransport and that phyllotactic patterning may occur as an\nessentially two-dimensional process. First, PIN1 expression is\nmuch higher in the epidermis than in underlying cells. Second,\nPIN1 polarity in these cells is predominantly lateral with basal\nlocalization occurring only after localized expression has been\nestablished (20, 26). The expression of the auxin influx mediator\nAUX1 is also limited predominantly to the L1, and removal of\nthe L1 layer essentially abolishes primordial development (14).\nThese data suggest that it is possible that auxin patterning of\nprimordial positions occurs in two dimensions; hence, we feel\njustified in proposing a two-dimensional model. Recently, it has\nbeen shown that auxin can influence PIN1 polarization by\nreducing endocytosis (27). It is interesting to note that a model\nwith a constant PIN1 exocytosis and an endocytosis reduced by\nauxin in the neighboring cell can lead to a similar pattern-\ngenerating behavior as the model presented. The two models do\nlead to differences in the detailed dynamics, but these discrep-\nancies are unnoticeable in the current investigation. Elsewhere,\nit has been reported that auxin can regulate the transcription of\nboth efflux and influx carriers (20, 28, 29). Although we have not\nSimulation of the phyllotaxis model on a half-sphere cylinder surface\nFig. 5.\nincluding cellular growth and proliferation. The main images show a top view\n(with the insets showing a side view), and time is increasing from top to\nbottom. The two simulations have different values for the size of the deﬁned\ncentral zone. (A) Peaks formed in a spiral pattern. (B) Peaks formed in a\nwhorled (decussate) pattern.\ndefinition of the central or peripheral zone would probably be\nrequired along with fixation of the auxin concentrations (or their\neffects) in cells once they move out of the peripheral zone.\nDespite these caveats, the dynamics of the simulations and the\nresulting patterns show that the model has the potential for\ngenerating phyllotactic patterns on a growing tissue such as the\nplant apical meristem.\nPIN1 Polarization Reversal. A characteristic feature of PIN1 be-\nhavior during Arabidopsis primordium development is that, in\ncells adaxial to a developing primordium, PIN1 polarity under-\ngoes a reversal from being directed toward the primordium to\nbeing directed away from it and toward younger primordia as\nthey are specified in adjacent cells (20). To test whether our\nmodel can recapitulate these polarity reversals we examined the\ndynamics of PIN1 polarization in our cell-based model when a\nnew peak forms by using a two-dimensional system of growing\nand proliferating cells. Fig. 6 shows cells from such a simulation\nthat are located near to where a new peak forms. It can be seen\nthat polarization reverses in cells located between the older\npeaks and the newly formed peak, confirming that our model can\nrecapitulate reversals in PIN1 polarity similar to those that are\nobserved in the living plant.\nDiscussion\nWe have presented a model for phyllotactic patterning based on\na feedback loop between relative auxin concentrations in adja-\ncent cells and auxin efflux direction. When simulating the\nyet included this aspect of regulation into our model, such\nregulation may strengthen the feedback process leading to\nlocalized maxima. If the conditions were such that the addition\nof this type of feedback were necessary for patterning to occur,\nregulation of transcriptional response may provide a means to\ndemarcate a peripheral zone where organogenesis occurs exclu-\nsively. We note that transcriptional responses to auxin are\napparently suppressed in the central zone (30), consistent with\nthis proposal.\nAnother model for auxin-influenced auxin transport is the\nwell known flux-based canalization model for venation patterns\n(31, 32). In that model, rather than polarity being determined by\nauxin concentrations in neighboring cells, polarity is established\naccording to the flux of auxin passing through a cell’s mem-\nbranes. It would seem that once established, a flux-based trans-\nport pathway might be hard to reverse. Because polarity reversals\nare a regular phenomenon associated with primordium devel-\nopment, a flux-based model might require an additional com-\nponent mediating these reversals, such as the activity of the\nprotein kinase PID, which is known to function as a PIN1\npolarity switch (17). It also remains to be shown whether such a\nmodel is capable of producing regular patterns, such as those\nrequired for organ positioning. Mechanical buckling models, on\nthe other hand, are capable of generating phyllotactic patterns\nand can explain transitions between patterns and the different\nprimordial shapes that occur in plants (4–6). A role for auxin\ntransport in this type of model has recently been proposed in\nwhich stress patterns dictate transport patterns, which then alter\nauxin levels to further feedback on stress patterns (6). Although\nan interesting idea, primordial initiation appears surprisingly\nrobust after meristem tissues are disrupted by laser treatments\nthat presumably change stress patterns to a large degree (33, 34).\nThe degree to which such treatments also disrupt auxin levels\nremains to be determined. Live imaging of PIN1–GFP-\nexpressing meristems after mechanical perturbations should\nenable an assessment of this proposal.\nLastly, we would like to stress that the presented polarized\ntransport model is useful not only as a model for phyllotaxis. It\nprovides a variant of a pattern-forming, reaction-transport\nmodel for which feedback through polarized transport is the\nunderlying mechanism that creates regular patterning with a\nparameter-dependent length scale. This mechanism could be a\npotential explanation for other biological systems where pat-\nterning appears.\nA future challenge will be to incorporate more accurate\nestimates for passive and carrier-mediated membrane perme-\nabilities and to extend the model to include tissue below the L1\nlayer. We would also like to find conditions under which the\nmodel may give all of the phyllotactic patterns and transitions\nobserved in plants, which will involve understanding the role of\nthe central zone more thoroughly, including how its size changes\nover time. Perhaps the most important task ahead is to exper-\nimentally test the underlying assumption, which is that PIN1\npolarizes up auxin concentration gradients between cells of the\nmeristem epidermis.\nMethods\nData Template Extraction. To compare and optimize the models\nwith data from real meristems, we extracted relative PIN1\nconcentrations in cellular and membranewall compartments by\nquantifying GFP f luorescence emitted by a functional\nPIN1::GFP fusion protein expressed under the PIN1 promoter\nand imaged by confocal microscopy (for details, see Supporting\nMethods, which is published as supporting information on the\nPNAS web site). Because we are interested in PIN1 localization\nin the epidermal layer, we used a two-dimensional horizontal\nsection covering the L1 layer at the apex and the newly forming\nprimordia (Fig. 1 A). Cell and wall compartments were extracted\nwith a watershed-type algorithm (Fig. 1B) (35). PIN1 concen-\ntrations were then estimated from the average GFP intensities in\nthe individual extracted compartments (Fig. 1 C and D).\nAuxin Transport Model. Models for chemiosmotic auxin transport\nin plant cells were developed in refs. 21–23, and we incorporated\nPIN1 dependence on auxin efflux into a similar model including\ncellular and wallmembrane compartments. We also applied the\nmodel on a system of cellular compartments, in which case we\nused PIN1 polarization from analytically calculated equilibrium\nvalues. We will refer to these model descriptions as the ‘‘de-\ntailed’’ and the ‘‘cell-based’’ models, respectively. In both cases\nwe appreciated the fact that auxin can cross a cellular membrane\npassively and actively. The net auxin flux between two compart-\ntot  Ji3j \nments (i, j) separated by a membrane is defined by Ji3j\nJj3i, where the individual terms include passive and active\ntransport.\nIn our cell-based simulations, we have transport between\ncellular compartments i and j, resulting in a net flux of (Table 1,\nrows 4 and 5, which is published as supporting information on the\nPNAS web site)\ntot  DA i\nJi3j\n  T P ij\nA i\n A i\n P ji\nA j\n A j\nwhere Ai (Aj) is the auxin concentration in compartment i (j),\nwhereas Pij and Pji are the PIN1 concentrations on the membrane\ntoward the neighboring compartment. D is the strength of the\npassive transport, and T is the strength of the PIN1-dependent\nactive transport. We allow the active transport to be saturable,\nmodeled in a Michaelis–Menten formalism, where KA is the\nMichaelis–Menten constant. The cell-to-cell transport is to be\ninterpreted as the cellular efflux combined with a symmetrical\ninflux.\nIn the detailed model, simulated directly on the geometry of the\nexperimental template, we use a more elaborate description of\ncompartment-specific transport parameters where the parameter\nvalues are experimental estimates. We use a compartmentalization\nwhere cellular (cytosol) compartments are surrounded by wall\ncompartments toward each neighbor separated by a membrane.\nThe model is a development of the models originally proposed by\nGoldsmith et al. (21) and Mitchison (22), and it explicitly accounts\nfor the anion and weak acid forms of auxin, and that the PIN1-\nmediated anion transport depends on the electrochemical gradient\nacross the plasma membrane. The net auxin flux between a cellular\ncompartment, i, and its neighboring wall compartment, ij is given by\n(Table 2, rows 4, 5, 7, and 8, which is published as supporting\ninformation on the PNAS web site)\ntot  D eff lux A i\nJi3ij\n D inf lux A ij\n P ij T eff lux\nA i\n A i\n T inf lux\nA ij\n A ij\nwhere Pij is the PIN1 located at the membrane and the transport\nparameters are compartment-type-dependent. In addition, we\nhave apoplastic auxin transport modeled as diffusion between\nneighboring wall compartments. The details of the auxin trans-\nport models are given in Supporting Methods (see also Figs. 7 and\n8, which are published as supporting information on the PNAS\nweb site).\nPIN1 Cycling Model. PIN1 is known to cycle between the plasma\nmembrane and internal cellular compartments (36). The sig-\nnals that govern the rates and direction of this cycling are to\na large extent unknown. Here we propose that relative auxin\nconcentrations in neighboring cells are communicated back to\na cell to differentially drive PIN1 polarization, creating a\nfeedback mechanism whereby auxin regulates its own trans-\nport. This feedback leads to regular spatial patterns in auxin\nconcentrations, where we assume that the peaks correspond to\nsites of primordium initiation.\nSpecifically, the hypothesis for PIN1 cycling is that auxin in a\nneighboring cell (Aj) induces the cycling from the cellular\ncompartment (Pi) into the membrane located toward the neigh-\nboring cell (Pij), which together with a constant internalization\nis described by (Table 2, rows 9 and 10)\nUntil now there has been a gap between molecular models and\nexperiments for explaining phyllotaxis. New confocal imaging\ntechniques allow a quantitative comparison between models and\nexperiments for essential molecules, which allows the optimiza-\ntion of unknown model parameters by fitting the model to\nexperiments. We used these techniques to infer parameters in\nthe PIN1 cycling model, given the detailed auxin transport model\nthat uses experimental estimates of the parameters (details of the\noptimization procedure are available in Supporting Methods).\n fAj\n \nwhere f(Aj) encodes the feedback from auxin in the neighboring\ncell and should be an increasing function of Aj, whereas k2 is a\nconstant. The summation is over the set of cell neighbors, Ni, for\nthe cellular compartment i. We use a linear description, f(Aj) \nk1Aj, as well as a saturable form allowing for a nonlinear\nn\nfeedback and described by a Hill-type equation, f(Aj)  k1[Aj\n(Kn  Aj\nn)], where k1 is the maximal rate and K and n are the Hill\nconstant and coefficient, respectively. This saturable form can be\nderived from the hypothesis of a combination of quickly equil-\nibrating reactions including a cell–cell communication pathway\n(Tables 3 and 4, which are published as supporting information\non the PNAS web site). Higher auxin in a neighboring cell leads\nto an increased PIN1 localization at the membrane toward that\ncell, resulting in a higher auxin transport into that cell. We find\nthat the strength of active transport compared with passive\ntransport determines whether the feedback is strong enough for\ncreating spatial patterns in auxin concentration. When a local\nauxin maximum is formed, the cells in the maximum deplete\nauxin from the surrounding cells. However, at a parameter-\ndependent distance from the maximum, cells tend to have less\nauxin in them than neighbors on either side. These cells start to\ntransport auxin away from the initial maximum into new max-\nima, with the end result being the formation of peaks and troughs\nof auxin concentration in a spatially regular pattern.\nCell-Based Model with Growing and Proliferating Cells. To be able to\ninvestigate the model in a growing shoot-like system, we study\nthe cell-based auxin model in a system where individual cells\ngrow and proliferate and are constrained to exist on a half-\nspherical surface connected to a cylindrical surface below. The\ncells are modeled as spheres with radial growth, and neighbors\nare defined as cell pairs with spatial overlap. For simplicity, we\nused a single growth rate for all cells at the apex. The mechanical\ninteractions are modeled with a repulsive spring force between\nneighboring cells (18). Cells in the apical region divide as they\nreach a threshold size. The initial division direction is random on\nthe surface, and then the positions are adjusted by the mechan-\nical neighbor interactions. Although this model provides a\nsimplified growth model for the epidermal layer of the SAM, it\ndoes include individual proliferating cells, and it has the property\nthat cells are displaced out of the apical region toward the\nperiphery, just as they are for real meristems. The growth and\nmechanical models are discussed in more detail in Supporting\nMethods. To avoid peaks forming at the very apex, a central zone\nand peripheral zone are defined by an additional auxin produc-\ntion term outside the central zone. This implementation is used\nto break the symmetry of patterning rather than as a valid\nbiological mechanism for defining these zones.\nWe thank Christophe Godin, Przemyslaw Prusinkiewicz, and Jan Traas\nfor discussions and Jan Traas for sharing unpublished results. This\nresearch was supported by U.S. National Science Foundation FIBR\nAward EF-0330786 and U.S. Department of Energy Award FG02–\n88ER13873. H.J. was supported in part by the Knut and Alice Wallen-\nberg Foundation through Swegene (Lund, Sweden).\n5. Shipman, P. D. & Newell, A. C. (2004) Phys. Rev. Lett. 92, art. no. 168102.\n6. Shipman, P. D. & Newell, A. C. (2005) J. Theor. Biol. 236, 154–197.\n7. Schoute, J. C. (1913) Re´. Trav. Bot. Ne´erl. 10, 153–235.\n8. Douady, S. & Couder, Y. (1996) J. Theor. Biol. 178, 295–312.\n9. Turing, A. M. (1952) Philos. Trans. R. Soc. London B 237, 37–72.\n10. Meinhardt, H. (1982) Models of Biological Pattern Formation (Academic,\n11. Preistley, J. H. & Scott, L. (1933) Biol. Rev. Cambridge Philos. Soc. 8, 241–268.\n12. Chapman, J. M. & Perry, R. (1987) Ann. Bot. (London) 60, 377–389.\n13. Reinhardt, D., Mandel, T. & Kuhlemeier, C. (2000) Plant Cell 12, 507–518.\n14. Reinhardt, D., Pesce, E. R., Stieger, P., Mandel, T., Baltensperger, K., Bennett,\n16. Bennett, M. J., Marchant, A., Green, H. G., May, S. T., Ward, S. P., Millner,\nP. A., Walker, A. R., Schulz, B. & Feldmann, K. A. (1996) Science 273, 948–950.\n17. Friml, J., Yang, X., Michniewicz, M., Weijers, D., Quint, A., Tietz, O.,\nBenjamins, R., Ouwerkerk, P. B., Ljung, K., Sandberg, G., et al. (2005) Science\n306, 862–865.\n18. Shapiro, B. E. & Mjolsness, E. (2001) in Proceedings of the Second International\nConference on Systems Biology (Omnipress, Madison, WI), pp. 342–351.\n19. Mjolsness, E., Sharp, D. H. & Reinitz, J. (1991) J. Theor. Biol. 152, 429–454.\n20. Heisler, M., Ohno, C., Das, P., Sieber, P., Long, J. A., Reddy, G. V. &\n22. Mitchison, G. J. (1980) Proc. R. Soc. London Ser. B 209, 489–511.\n23. Kramer, E. M. (2004) Trends Plant Sci. 9, 578–582.\n24. Rubery, P. H. & Sheldrake, A. R. (1974) Planta 118, 101–121.\n25. Raven, J. A. (1975) New Phytol. 74, 163–172.\n26. Benkova´, E., Michniewicz, M., Sauer, M., Teichmann, T., Seifertova´, D.,",
         "",
         "",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "Recent studies show that plant organ positioning may be mediated\nby localized concentrations of the plant hormone auxin. Auxin\npatterning in the shoot apical meristem is in turn brought about by\nthe subcellular polar distribution of the putative auxin efﬂux\nmediator, PIN1. However, the question of what signals determine\nPIN1 polarization and how this gives rise to regular patterns of\nauxin concentration remains unknown. Here we address these\nquestions by using mathematical modeling combined with confo-\ncal imaging. We propose a model that is based on the assumption\nthat auxin inﬂuences the polarization of its own efﬂux within the\nmeristem epidermis. We show that such a model is sufﬁcient to\ncreate regular spatial patterns of auxin concentration on systems\nwith static and dynamic cellular connectivities, the latter governed\nby a mechanical model. We also optimize parameter values for the\nPIN1 dynamics by using a detailed auxin transport model, for which\nparameter values are taken from experimental estimates, together\nwith a template consisting of cell and wall compartments as well\nas PIN1 concentrations quantitatively extracted from confocal\ndata. The model shows how polarized transport can drive the\nformation of regular patterns.\nemerge at well defined positions, resulting in strikingly regular\npatterns (1). These phyllotactic patterns can be whorled (more\nthan one new primordium develops simultaneously) or spiral\n(single primordia are created sequentially). Spiral phyllotaxis is\noften connected to the Fibonacci series because the numbers of\nparastichies (visible spirals) in each direction around the axis are\ncommonly consecutive Fibonacci numbers. Also, consecutive\nprimordia in the spiral often appear at a divergence angle close\nto the golden angle. The beautiful symmetries apparent in\nphyllotaxis and its connection to mathematics have inspired\nscientists to create theories and models to explain these patterns.\nOne important finding from mathematical analysis and physical\nsimulation (2, 3) is that many of the seemingly complex phyllo-\ntactic patterns and transitions found in plants can probably be\nexplained to a large degree by any regular spacing mechanism\nsuperimposed on a gradually enlarging generative region. This\nfinding is important because it suggests that the problem can be\nreduced to two potentially independent and smaller questions.\nThe first question regards how meristem size is determined\nduring plant development. The second question involves how a\nregularly spaced pattern of primordial position is specified.\nTraditionally, models have concentrated on the second question,\nand they can be divided into molecular and mechanical ideas\n(4–6). Because we investigate a model based on molecular\nexperiments, here we discuss some molecular-based ideas in\nmore detail. Schoute (7) first proposed the idea of lateral\ninhibition based on a diffusible chemical produced by each\ndeveloping primordium that inhibits the initiation of primordia\nnearby. The inhibition hypothesis has been studied extensively in\niterative models (3, 8) and has been shown to be capable of\nproducing common phyllotactic patterns for which different\nparameter values specifying inhibition range relative to a gen-\nerative region lead to different stable phyllotactic patterns (3).\nContinuous changes in these parameters were used to investigate\ntransient phases between these patterns relevant to a growing\nplant (8). Relating back to the work of Turing (9), Meinhardt\n(10) introduced a reaction–diffusion version of the inhibitor\nmodel by incorporating an activator molecule as well as an\ninhibitor, allowing for a continuous dynamical model. These\nmodels exhibit robust and dynamic pattern generation, including\nthe initiation of new concentration peaks corresponding to\nprimordia (10).\nAn alternative idea is based on competition or depletion of a\nprimordium-promoting factor and was proposed by Preistley and\nScott in 1933 (11) and by Mitchison in 1977 (2) and Chapman\nand Perry in 1987 (12). Recent (and old) findings have provided\nconsiderable support for such a depletion scenario. In a series of\nelegant experiments, it has been shown that the plant hormone\nauxin (in the form of indole-3-acetic acid) is an essential\nactivator for primordium formation (13, 14). Plants in which\nauxin transport is blocked (either chemically or in the pin1\nmutant) exhibit a pin-formed morphological phenotype charac-\nterized by a lack of primordium development and a bare\nmeristem. This phenotype can be rescued by local application of\nauxin in the form of a lanolin paste, showing that localized auxin\nis both necessary and sufficient for primordial development. In\nthe wild type, auxin transport is mediated by the Arabidopsis PIN\n(PINFORMED) family of putative auxin efflux mediators (15)\nas well as the auxin import mediators AUX1 and its relatives\n(16). However, the principle protein required for primordium\ndevelopment appears to be PIN1 because pin1 mutants lack\nfloral primordia in contrast to the relatively mild phenotypes, so\nfar, of other auxin transport mutants. In the shoot apical\nmeristem (SAM), the PIN1 protein is expressed mainly in the\nepidermal (L1) layer of cells and it is polarized toward newly\nforming primordia (Fig. 1) (14). In young primordia, PIN1 is\npolarized downwards into the subepidermal layers, presumably\ninitiating vascular differentiation. In the epidermis below the\nSAM, PIN1 is polarized upwards toward the SAM, and this\npolarization depends on the PID (PINOID) protein (17).\nConsidering these data, Reinhardt et al. (14) have proposed\nthat auxin is a primordium activator that is depleted from\nprimordial regions via PIN1-dependent auxin transport so that\nauxin reaches the next critical threshold for initiation at the point\nfarthest from the previous point of depletion. One limitation of\nthis class of model would appear to be in generating whorled\npatterns for which multiple positions are specified simulta-\nneously. Another equally important issue not explained by the\nReinhardt et al. (14) proposal is how auxin transport itself is\npatterned. The objective of this work is to propose a model for\nprimordial positioning that is based on the assumption that PIN1\npolarity is regulated by relative auxin concentrations in neigh-\nboring cells within a two-dimensional space, such as the epider-\nmis. The main elements of our model include (i) passive and\nactive auxin transport, in which PIN1 mediates auxin efflux; (ii)\na model for PIN1 cycling between internal and membrane\ncompartments, in which auxin regulates the polarization of PIN1\nby modulating the cycling parameters in different directions; and\n(iii) changes in cell adjacency relationships determined by cel-\nlular growth and mechanics. The basis for the regulatory model\nis a feedback mechanism by which cells with relatively high auxin\nconcentrations increase their auxin content by influencing PIN1\npolarity in neighboring cells, which as a result become auxin-\ndepleted. Thus, from a close to homogeneous state, auxin peaks\nemerge at regular distances and determine the locations of early\nprimordia. These mechanisms were simulated on a meristem-\nlike surface topology by using a cell-based model including\ncellular growth and an elastic mechanics model (18). This model\nrealizes one direction of the two-way interaction between reg-\nulatory and mechanical networks outlined in ref. 19. The result-\ning auxin model generates spiral and whorled phyllotactic-like\npatterns as found in plants. A more detailed look at the PIN1\npolarization dynamics around the formation of a new peak\n(primordium) shows that the model is capable of predicting the\npolarization reversal in cells in between the new and older\nprimordia, which has recently been shown in experiments (20).\nWe also include a detailed auxin transport description based\non the chemiosmotic hypothesis of auxin transport using exper-\nimental estimates for auxin transport parameter values (21–23).\nUsing this model, we introduce a schema for evaluating and\noptimizing PIN1 cycling model parameters that compares the\nmodel predictions to quantified confocal microscopy data of\nfunctional PIN1 protein fused to GFP.\nResults\nAuxin Transport Model Combined with Extracted PIN1 Localization.\nWe first set out to test the behavior of the detailed auxin transport\nmodel (Eq. 3) by using the geometry and PIN1 localization ex-\ntracted from a confocal image (Fig. 1). The expectation is that cells\nforming new primordia have a relatively high auxin concentration.\nIn the model, most of the parameters are taken from experimental\nestimates, but the strength of active transport and the explicit PIN1\nconcentrations are unknown. In ref. 21, PIN1 is not explicitly\nmodeled and an anion permeability constant is inferred. We used\nthis value together with a maximal membrane PIN1 concentration\nset to 1 mol per unit of area. In addition, production and\ndegradation of auxin is allowed for. The details of the model and\nparameter values used are provided in Supporting Methods (see also\nFigs. 7 and 8 and Tables 1, 2, and 5, which are published as\nsupporting information on the PNAS web site). The equilibrium\nauxin concentrations resulting from these parameter values are\nshown in Fig. 2A. As can be seen, the newest visible primordium\n(P1) has a clear peak in auxin concentration. There is also a low\nconcentration region between the SAM and the next-older visible\nprimordium (P2), but there is no major peak of auxin concentration\nat this primordium, most likely because the primordium peak is\noutside the two-dimensional template. It can also be seen that the\ntwo primordial positions that form next have small peaks of auxin\nconcentrations (I1 and I2). Thus, the model seems to behave\nqualitatively as expected.\nTo investigate how robust this behavior is to different param-\neter changes, we simulated the model using a range of parameter\nvalues around the estimated values (Fig. 2 B and C; see also\nSupporting Methods and Figs. 9–11, which are published as\nsupporting information on the PNAS web site). The results show\nthat the model exhibits considerable stability and the qualitative\nresults are similar to Fig. 2 A. As an example, varying the wall pH\nresults in the auxin concentrations shown in Figs. 2 B and C.\nLowering the wall pH can be interpreted as an approximation to\nhow a symmetric influx mediator may influence the model. The\nconclusions drawn from these simulations are that the auxin\ntransport model combined with the extracted PIN1 localizations\nqualitatively behaves as expected and that this behavior is robust.\nAuxin-Driven PIN1 Cycling. Our central hypothesis is that the\nrelative concentrations of auxin in neighboring cells differen-\ntially drive the polarization of PIN1 to the corresponding portion\nof the membrane between each cell and its neighbors. To analyze\nthe dynamics of auxin transport in such a model, we simplify the\nmodel to a single parameter description. We use the cell-based\nmodel and simplify the auxin transport description (Eq. 2) by\nassuming that the PIN1-mediated transport is unsaturated,\nwhich leads to a model described by\n[1]\nAi(Ak) is the auxin concentration in compartment i(k),\nwhereas Pik and Pki are the PIN1 concentrations on the mem-\nbrane toward the neighboring compartment. D is the strength of\npassive transport, and T is the strength of the PIN1-dependent\nactive transport. The summation is over the set of neighbors, Ni,\nand the only parameters present are the strengths of the passive\nand active transports. We assume that the total amount of PIN1\nin the cell and its membrane is constant and equal for all cells\n(P  Pi\ntot  Pi  kPik, @i). We use a linear polarization feedback\n[f(Aj)  k1Aj] in Eq. 4 and assume that the PIN1 is in its\nequilibrium polarization state at any given time, which leads to\nPij  PAj[(k2k1)  kAk]. Finally, we assume that most PIN1\nis situated at the membrane (k2  k1), which leads to Pij \nPAj(k Ak) and a model in Eq. 1 only depending on auxin\nconcentrations and the three parameters D, T, and P, which can\nbe treated as a single parameter DTP.\nOptimizing the PIN1-Cycling Model. As described in Supporting\nMethods (see also Fig. 13, which is published as supporting\ninformation on the PNAS web site), the optimization is per-\nformed by simulating the detailed auxin model with extracted\nvalues of PIN1 concentration; it then uses the equilibrium auxin\nvalues for optimizing the PIN1 cycling model parameters [in Eq.\n4 using a Hill description for f(Aj)] to fit the extracted values. The\nobjective function landscape is quite smooth, resulting in a\n‘‘simple’’ optimization problem for which a local search algo-\nrithm with multiple restarts is sufficient to find good solutions.\nIn Fig. 4A the resulting PIN1 concentrations for the optimized\nmodel are plotted directly on the template to be compared with\nFig. 1D. Fig. 4B shows a quantitative comparison between\nextracted and optimized PIN1 concentrations, with a mean\nsquared error of 0.015. It should be stressed that we use extracted\nvalues for the total amount of PIN1 in each cell and that these\nvalues are different for individual cells. This variation introduces\na bias in the optimization measure. An indication of this bias is\ngiven by having each cell uniformly distribute its PIN1 content,\nwhich leads to a mean squared error of 0.031. To better\nappreciate the information in these numbers, more data should\nbe used together with comparisons between different kinds of\nPIN1 polarization models. In addition, optimization of some of\nthe more uncertain parameters within the auxin transport model\ncan be included in such an approach.\nPhyllotaxis Model on a Simplified Shoot Topology with Growth. To\ninvestigate the patterning dynamics of our model on a growing\nshoot, we simulate the cell-based auxin model in a system where\ncells are confined to a half-sphere connected to a cylinder. The\ncellular growth is uniform, the definition of the central zone is\ndynamic, and the auxin peaks are not fixed. Hence, these\nsimulations should be viewed as an initial test simply to see\nwhether patterning can be maintained in such a situation. Model\ndetails and parameter values used are given in the Supporting\nMethods. Time points from simulations for two different param-\neter sets are shown in Fig. 5. Illustrations of the complete\nsimulations are provided as Movies 1 and 2, which are published\nas supporting information on the PNAS web site. Fig. 5A shows\na simulation leading to a spiral-like phyllotaxis, with new peaks\nforming one at a time, and Fig. 5B shows a pattern in which the\npeaks form in pairs in a whorled-like (decussate) pattern. The\nparameter difference between the simulations is the size of the\ndefined central zone. It should be noted that the continuation of\nthese patterns is not extremely stable. Typically the temporal\norder in which new primordia form in the spiral-like pattern can\nsometimes reverse, and the radial position of the appearance of\npeaks and the time between the formation of two consecutive\npeaks are not constant, which can lead to switching between\ndifferent quasistable patterns in a single simulation. To a large\nextent, this instability is due to the noise introduced by the\ngrowthmechanical model, which allows for discrete jumps at\ncell division and local rearrangements of cells (leading to\nappearance and disappearance of neighbor pairs and the sliding\nof cells in respect to each other), that is not present in a growing\nplant. In addition, to achieve more stability, a more appropriate\ncell-based version of the model on a growing shoot-like topology,\nthis model is capable of producing both spiral and whorled\npatterns and recapitulates observed polarization reversals in the\nneighborhood of newly formed auxin peaks. These results show\nthat regulation of polarized auxin transport by auxin, together\nwith the mechanics of cell growth and neighborhood change,\nmay provide the underlying mechanisms for phyllotactic pat-\nterning. We also present a methodology that bridges the gap\nbetween theoretical models and experiments. Unlike previous\nmodels for generating phyllotaxis, this model is based on de-\ntailed experimental data. Auxin is now established as an essen-\ntial molecule required for primordium initiation. In turn, its\ndistribution in the shoot apical meristem depends on the activity\nof the PIN1 putative auxin efflux mediator. Our model includes\nthe chemiosmotic hypothesis of auxin transport, where transport\ndepends on the differential permeability of the protonated and\nneutral forms of the acid as well as the pH difference between\ncytoplasm and wall regions (24, 25). This study provides a\nsimulation of our present conception of auxin transport mech-\nanisms on a cellular template derived directly from real tissue.\nAlthough PIN1 has not been shown to directly transport auxin,\nits activity is essential for this process. Thus, we feel that we are\njustified in using its relative concentrations as a proxy for auxin\nefflux mediator concentrations. The main limitation of the\npresent technique is that it is limited to two dimensions because\nof the lack of resolution in the confocal z axis. For the purposes\nis\nof this study, our data are adequate because our model\ntwo-dimensional and is proposed to occur in the meristem\nepidermis. Although a model\nlimited to two dimensions is\ninadequate for describing all of the processes that occur in a\nthree-dimensional structure, such as the meristem, data suggest\nthat the L1 layer of the meristem may play a special role in auxin\ntransport and that phyllotactic patterning may occur as an\nessentially two-dimensional process. First, PIN1 expression is\nmuch higher in the epidermis than in underlying cells. Second,\nPIN1 polarity in these cells is predominantly lateral with basal\nlocalization occurring only after localized expression has been\nestablished (20, 26). The expression of the auxin influx mediator\nAUX1 is also limited predominantly to the L1, and removal of\nthe L1 layer essentially abolishes primordial development (14).\nThese data suggest that it is possible that auxin patterning of\nprimordial positions occurs in two dimensions; hence, we feel\njustified in proposing a two-dimensional model. Recently, it has\nbeen shown that auxin can influence PIN1 polarization by\nreducing endocytosis (27). It is interesting to note that a model\nwith a constant PIN1 exocytosis and an endocytosis reduced by\nauxin in the neighboring cell can lead to a similar pattern-\ngenerating behavior as the model presented. The two models do\nlead to differences in the detailed dynamics, but these discrep-\nancies are unnoticeable in the current investigation. Elsewhere,\nit has been reported that auxin can regulate the transcription of\nboth efflux and influx carriers (20, 28, 29). Although we have not\nSimulation of the phyllotaxis model on a half-sphere cylinder surface\nFig. 5.\nincluding cellular growth and proliferation. The main images show a top view\n(with the insets showing a side view), and time is increasing from top to\nbottom. The two simulations have different values for the size of the deﬁned\ncentral zone. (A) Peaks formed in a spiral pattern. (B) Peaks formed in a\nwhorled (decussate) pattern.\ndefinition of the central or peripheral zone would probably be\nrequired along with fixation of the auxin concentrations (or their\neffects) in cells once they move out of the peripheral zone.\nDespite these caveats, the dynamics of the simulations and the\nresulting patterns show that the model has the potential for\ngenerating phyllotactic patterns on a growing tissue such as the\nplant apical meristem.\nPIN1 Polarization Reversal. A characteristic feature of PIN1 be-\nhavior during Arabidopsis primordium development is that, in\ncells adaxial to a developing primordium, PIN1 polarity under-\ngoes a reversal from being directed toward the primordium to\nbeing directed away from it and toward younger primordia as\nthey are specified in adjacent cells (20). To test whether our\nmodel can recapitulate these polarity reversals we examined the\ndynamics of PIN1 polarization in our cell-based model when a\nnew peak forms by using a two-dimensional system of growing\nand proliferating cells. Fig. 6 shows cells from such a simulation\nthat are located near to where a new peak forms. It can be seen\nthat polarization reverses in cells located between the older\npeaks and the newly formed peak, confirming that our model can\nrecapitulate reversals in PIN1 polarity similar to those that are\nobserved in the living plant.\nDiscussion\nWe have presented a model for phyllotactic patterning based on\na feedback loop between relative auxin concentrations in adja-\ncent cells and auxin efflux direction. When simulating the\nyet included this aspect of regulation into our model, such\nregulation may strengthen the feedback process leading to\nlocalized maxima. If the conditions were such that the addition\nof this type of feedback were necessary for patterning to occur,\nregulation of transcriptional response may provide a means to\ndemarcate a peripheral zone where organogenesis occurs exclu-\nsively. We note that transcriptional responses to auxin are\napparently suppressed in the central zone (30), consistent with\nthis proposal.\nAnother model for auxin-influenced auxin transport is the\nwell known flux-based canalization model for venation patterns\n(31, 32). In that model, rather than polarity being determined by\nauxin concentrations in neighboring cells, polarity is established\naccording to the flux of auxin passing through a cell’s mem-\nbranes. It would seem that once established, a flux-based trans-\nport pathway might be hard to reverse. Because polarity reversals\nare a regular phenomenon associated with primordium devel-\nopment, a flux-based model might require an additional com-\nponent mediating these reversals, such as the activity of the\nprotein kinase PID, which is known to function as a PIN1\npolarity switch (17). It also remains to be shown whether such a\nmodel is capable of producing regular patterns, such as those\nrequired for organ positioning. Mechanical buckling models, on\nthe other hand, are capable of generating phyllotactic patterns\nand can explain transitions between patterns and the different\nprimordial shapes that occur in plants (4–6). A role for auxin\ntransport in this type of model has recently been proposed in\nwhich stress patterns dictate transport patterns, which then alter\nauxin levels to further feedback on stress patterns (6). Although\nan interesting idea, primordial initiation appears surprisingly\nrobust after meristem tissues are disrupted by laser treatments\nthat presumably change stress patterns to a large degree (33, 34).\nThe degree to which such treatments also disrupt auxin levels\nremains to be determined. Live imaging of PIN1–GFP-\nexpressing meristems after mechanical perturbations should\nenable an assessment of this proposal.\nLastly, we would like to stress that the presented polarized\ntransport model is useful not only as a model for phyllotaxis. It\nprovides a variant of a pattern-forming, reaction-transport\nmodel for which feedback through polarized transport is the\nunderlying mechanism that creates regular patterning with a\nparameter-dependent length scale. This mechanism could be a\npotential explanation for other biological systems where pat-\nterning appears.\nA future challenge will be to incorporate more accurate\nestimates for passive and carrier-mediated membrane perme-\nabilities and to extend the model to include tissue below the L1\nlayer. We would also like to find conditions under which the\nmodel may give all of the phyllotactic patterns and transitions\nobserved in plants, which will involve understanding the role of\nthe central zone more thoroughly, including how its size changes\nover time. Perhaps the most important task ahead is to exper-\nimentally test the underlying assumption, which is that PIN1\npolarizes up auxin concentration gradients between cells of the\nmeristem epidermis.\nMethods\nData Template Extraction. To compare and optimize the models\nwith data from real meristems, we extracted relative PIN1\nconcentrations in cellular and membranewall compartments by\nquantifying GFP f luorescence emitted by a functional\nPIN1::GFP fusion protein expressed under the PIN1 promoter\nand imaged by confocal microscopy (for details, see Supporting\nMethods, which is published as supporting information on the\nPNAS web site). Because we are interested in PIN1 localization\nin the epidermal layer, we used a two-dimensional horizontal\nsection covering the L1 layer at the apex and the newly forming\nprimordia (Fig. 1 A). Cell and wall compartments were extracted\nwith a watershed-type algorithm (Fig. 1B) (35). PIN1 concen-\ntrations were then estimated from the average GFP intensities in\nthe individual extracted compartments (Fig. 1 C and D).\nAuxin Transport Model. Models for chemiosmotic auxin transport\nin plant cells were developed in refs. 21–23, and we incorporated\nPIN1 dependence on auxin efflux into a similar model including\ncellular and wallmembrane compartments. We also applied the\nmodel on a system of cellular compartments, in which case we\nused PIN1 polarization from analytically calculated equilibrium\nvalues. We will refer to these model descriptions as the ‘‘de-\ntailed’’ and the ‘‘cell-based’’ models, respectively. In both cases\nwe appreciated the fact that auxin can cross a cellular membrane\npassively and actively. The net auxin flux between two compart-\ntot  Ji3j \nments (i, j) separated by a membrane is defined by Ji3j\nJj3i, where the individual terms include passive and active\ntransport.\nIn our cell-based simulations, we have transport between\ncellular compartments i and j, resulting in a net flux of (Table 1,\nrows 4 and 5, which is published as supporting information on the\nPNAS web site)\ntot  DA i\nJi3j\n  T P ij\nA i\n A i\n P ji\nA j\n A j\nwhere Ai (Aj) is the auxin concentration in compartment i (j),\nwhereas Pij and Pji are the PIN1 concentrations on the membrane\ntoward the neighboring compartment. D is the strength of the\npassive transport, and T is the strength of the PIN1-dependent\nactive transport. We allow the active transport to be saturable,\nmodeled in a Michaelis–Menten formalism, where KA is the\nMichaelis–Menten constant. The cell-to-cell transport is to be\ninterpreted as the cellular efflux combined with a symmetrical\ninflux.\nIn the detailed model, simulated directly on the geometry of the\nexperimental template, we use a more elaborate description of\ncompartment-specific transport parameters where the parameter\nvalues are experimental estimates. We use a compartmentalization\nwhere cellular (cytosol) compartments are surrounded by wall\ncompartments toward each neighbor separated by a membrane.\nThe model is a development of the models originally proposed by\nGoldsmith et al. (21) and Mitchison (22), and it explicitly accounts\nfor the anion and weak acid forms of auxin, and that the PIN1-\nmediated anion transport depends on the electrochemical gradient\nacross the plasma membrane. The net auxin flux between a cellular\ncompartment, i, and its neighboring wall compartment, ij is given by\n(Table 2, rows 4, 5, 7, and 8, which is published as supporting\ninformation on the PNAS web site)\ntot  D eff lux A i\nJi3ij\n D inf lux A ij\n P ij T eff lux\nA i\n A i\n T inf lux\nA ij\n A ij\nwhere Pij is the PIN1 located at the membrane and the transport\nparameters are compartment-type-dependent. In addition, we\nhave apoplastic auxin transport modeled as diffusion between\nneighboring wall compartments. The details of the auxin trans-\nport models are given in Supporting Methods (see also Figs. 7 and\n8, which are published as supporting information on the PNAS\nweb site).\nPIN1 Cycling Model. PIN1 is known to cycle between the plasma\nmembrane and internal cellular compartments (36). The sig-\nnals that govern the rates and direction of this cycling are to\na large extent unknown. Here we propose that relative auxin\nconcentrations in neighboring cells are communicated back to\na cell to differentially drive PIN1 polarization, creating a\nfeedback mechanism whereby auxin regulates its own trans-\nport. This feedback leads to regular spatial patterns in auxin\nconcentrations, where we assume that the peaks correspond to\nsites of primordium initiation.\nSpecifically, the hypothesis for PIN1 cycling is that auxin in a\nneighboring cell (Aj) induces the cycling from the cellular\ncompartment (Pi) into the membrane located toward the neigh-\nboring cell (Pij), which together with a constant internalization\nis described by (Table 2, rows 9 and 10)\nUntil now there has been a gap between molecular models and\nexperiments for explaining phyllotaxis. New confocal imaging\ntechniques allow a quantitative comparison between models and\nexperiments for essential molecules, which allows the optimiza-\ntion of unknown model parameters by fitting the model to\nexperiments. We used these techniques to infer parameters in\nthe PIN1 cycling model, given the detailed auxin transport model\nthat uses experimental estimates of the parameters (details of the\noptimization procedure are available in Supporting Methods).\n fAj\n \nwhere f(Aj) encodes the feedback from auxin in the neighboring\ncell and should be an increasing function of Aj, whereas k2 is a\nconstant. The summation is over the set of cell neighbors, Ni, for\nthe cellular compartment i. We use a linear description, f(Aj) \nk1Aj, as well as a saturable form allowing for a nonlinear\nn\nfeedback and described by a Hill-type equation, f(Aj)  k1[Aj\n(Kn  Aj\nn)], where k1 is the maximal rate and K and n are the Hill\nconstant and coefficient, respectively. This saturable form can be\nderived from the hypothesis of a combination of quickly equil-\nibrating reactions including a cell–cell communication pathway\n(Tables 3 and 4, which are published as supporting information\non the PNAS web site). Higher auxin in a neighboring cell leads\nto an increased PIN1 localization at the membrane toward that\ncell, resulting in a higher auxin transport into that cell. We find\nthat the strength of active transport compared with passive\ntransport determines whether the feedback is strong enough for\ncreating spatial patterns in auxin concentration. When a local\nauxin maximum is formed, the cells in the maximum deplete\nauxin from the surrounding cells. However, at a parameter-\ndependent distance from the maximum, cells tend to have less\nauxin in them than neighbors on either side. These cells start to\ntransport auxin away from the initial maximum into new max-\nima, with the end result being the formation of peaks and troughs\nof auxin concentration in a spatially regular pattern.\nCell-Based Model with Growing and Proliferating Cells. To be able to\ninvestigate the model in a growing shoot-like system, we study\nthe cell-based auxin model in a system where individual cells\ngrow and proliferate and are constrained to exist on a half-\nspherical surface connected to a cylindrical surface below. The\ncells are modeled as spheres with radial growth, and neighbors\nare defined as cell pairs with spatial overlap. For simplicity, we\nused a single growth rate for all cells at the apex. The mechanical\ninteractions are modeled with a repulsive spring force between\nneighboring cells (18). Cells in the apical region divide as they\nreach a threshold size. The initial division direction is random on\nthe surface, and then the positions are adjusted by the mechan-\nical neighbor interactions. Although this model provides a\nsimplified growth model for the epidermal layer of the SAM, it\ndoes include individual proliferating cells, and it has the property\nthat cells are displaced out of the apical region toward the\nperiphery, just as they are for real meristems. The growth and\nmechanical models are discussed in more detail in Supporting\nMethods. To avoid peaks forming at the very apex, a central zone\nand peripheral zone are defined by an additional auxin produc-\ntion term outside the central zone. This implementation is used\nto break the symmetry of patterning rather than as a valid\nbiological mechanism for defining these zones.\nWe thank Christophe Godin, Przemyslaw Prusinkiewicz, and Jan Traas\nfor discussions and Jan Traas for sharing unpublished results. This\nresearch was supported by U.S. National Science Foundation FIBR\nAward EF-0330786 and U.S. Department of Energy Award FG02–\n88ER13873. H.J. was supported in part by the Knut and Alice Wallen-\nberg Foundation through Swegene (Lund, Sweden).\n5. Shipman, P. D. & Newell, A. C. (2004) Phys. Rev. Lett. 92, art. no. 168102.\n6. Shipman, P. D. & Newell, A. C. (2005) J. Theor. Biol. 236, 154–197.\n7. Schoute, J. C. (1913) Re´. Trav. Bot. Ne´erl. 10, 153–235.\n8. Douady, S. & Couder, Y. (1996) J. Theor. Biol. 178, 295–312.\n9. Turing, A. M. (1952) Philos. Trans. R. Soc. London B 237, 37–72.\n10. Meinhardt, H. (1982) Models of Biological Pattern Formation (Academic,\n11. Preistley, J. H. & Scott, L. (1933) Biol. Rev. Cambridge Philos. Soc. 8, 241–268.\n12. Chapman, J. M. & Perry, R. (1987) Ann. Bot. (London) 60, 377–389.\n13. Reinhardt, D., Mandel, T. & Kuhlemeier, C. (2000) Plant Cell 12, 507–518.\n14. Reinhardt, D., Pesce, E. R., Stieger, P., Mandel, T., Baltensperger, K., Bennett,\n16. Bennett, M. J., Marchant, A., Green, H. G., May, S. T., Ward, S. P., Millner,\nP. A., Walker, A. R., Schulz, B. & Feldmann, K. A. (1996) Science 273, 948–950.\n17. Friml, J., Yang, X., Michniewicz, M., Weijers, D., Quint, A., Tietz, O.,\nBenjamins, R., Ouwerkerk, P. B., Ljung, K., Sandberg, G., et al. (2005) Science\n306, 862–865.\n18. Shapiro, B. E. & Mjolsness, E. (2001) in Proceedings of the Second International\nConference on Systems Biology (Omnipress, Madison, WI), pp. 342–351.\n19. Mjolsness, E., Sharp, D. H. & Reinitz, J. (1991) J. Theor. Biol. 152, 429–454.\n20. Heisler, M., Ohno, C., Das, P., Sieber, P., Long, J. A., Reddy, G. V. &\n22. Mitchison, G. J. (1980) Proc. R. Soc. London Ser. B 209, 489–511.\n23. Kramer, E. M. (2004) Trends Plant Sci. 9, 578–582.\n24. Rubery, P. H. & Sheldrake, A. R. (1974) Planta 118, 101–121.\n25. Raven, J. A. (1975) New Phytol. 74, 163–172.\n26. Benkova´, E., Michniewicz, M., Sauer, M., Teichmann, T., Seifertova´, D.,",
         "Computer simulations reveal properties of the cell-cell signaling network at the shoot apex in Arabidopsis;Patterns of Auxin Transport and Gene Expression during Primordium Development Revealed by Live Imaging of the Arabidopsis Inflorescence Meristem;Functional redundancy of PIN proteins is accompanied by auxin-dependent cross-regulation of PIN expression;Polygonal planforms and phyllotaxis on plants.;Auxin inhibits endocytosis and promotes its own efflux from cells;PIN and AUX/LAX proteins: their role in auxin accumulation.;Microsurgical and laser ablation analysis of leaf positioning and dorsoventral patterning in tomato;A PINOID-Dependent Binary Switch in Apical-Basal PIN Polar Targeting Directs Auxin Efflux;Local, Efflux-Dependent Auxin Gradients as a Common Module for Plant Organ Formation;Regulation of phyllotaxis by polar auxin transport;Microsurgical and laser ablation analysis of interactions between the zones and layers of the tomato shoot apical meristem;Polar auxin transport in the wood-forming tissues of hybrid aspen is under simultaneous control of developmental and environmental signals;Cellerator: extending a computer algebra system to include biochemical arrows for signal transduction simulations;Auxin transport inhibitors block PIN1 cycling and vesicle trafficking;Nonlinear Dynamics and Chaos;Auxin Regulates the Initiation and Radial Position of Plant Lateral Organs;Regulation of polar auxin transport by AtPIN1 in Arabidopsis vascular tissue.;Snakes, shapes, and gradient vector flow;Symmetry in Plants;Arabidopsis AUX1 Gene: A Permease-Like Regulator of Root Gravitropism;Development;Phyllotaxis as a Dynamical Self Organizing Process Part III: The Simulation of the Transient Regimes of Ontogeny;Phyllotaxis as a physical self-organized growth process.;A connectionist model of development.;Numerical Recipes in C: The Art of Scientific Computing;A Diffusion Model of Phyllotaxis;Mathematical analysis of the chemosmotic polar diffusion of auxin through plant tissues.;The dynamics of auxin transport;A model for vein formation in higher plants;Phyllotaxis and the Fibonacci Series;TRANSPORT OF INDOLEACETIC ACID IN PLANT CELLS IN RELATION TO pH AND ELECTRICAL POTENTIAL GRADIENTS, AND ITS SIGNIFICANCE FOR POLAR IAA TRANSPORT;Carrier-mediated auxin transport;The chemical basis of morphogenesis;Studies of nuclear collisions involving 8 MeV deuterons by the photographic method III. Angular distribution of the charged particles produced by the bombardment of hydrogen and deuterium;PHYLLOTAXIS IN THE DICOTYLEDON FROM THE STANDPOINT OF DEVELOPMENTAL ANATOMY;New Phytol;The plant cell;An Explicit Spatial Model of Yeast Microcolony Growth;Modeling the organization of the WUSCHEL expression domain in the shoot apical meristem;Modeling Plant Development with Gene Regulation Networks Including Signaling and Cell Division;Proceedings of the second international conference on systems biology;Nature;Science;J. Theor. Biol;Phys. Rev. Lett;Ann. Bot. (London);Models of biological pattern formation;Proc. Natl. Acad. Sci. USA 78;The Control of the Patterned Differentiation of Vascular Tissues;Adv. Bot. Res;Digital image processing;Proc. R. Soc. London Ser. B;Patterns in plant development: Subject index;Philos. Trans. R. Soc. London B;Biol. Rev. Cambridge Philos. Soc;Ré. Trav. Bot. Néerl",
         "An auxin-driven polarized transport model for phyllotaxis",
         "None"
        ],
        [
         "31",
         "001bc274047c2f66701d4c2b951054af5560c464",
         "ABSTRACT For many enveloped viruses, cellular multivesicular body (MVB) sorting machinery has been reported to be utilized for efficient viral budding. Matrix and Gag proteins have been shown to contain one or two L-domain motifs (PPxY, PT/SAP, YPDL, and FPIV), some of which interact specifically with host cellular proteins involved in MVB sorting, which are recruited to the viral budding site. However, for many enveloped viruses, L-domain motifs have not yet been identified and the involvement of MVB sorting machinery in viral budding is still unknown. Here we show that both Sendai virus (SeV) matrix protein M and accessory protein C contribute to virus budding by physically interacting with Alix/AIP1. A YLDL sequence within the M protein showed L-domain activity, and its specific interaction with the N terminus of Alix/AIP1(1-211) was important for the budding of virus-like particles (VLPs) of M protein. In addition, M-VLP budding was inhibited by the overexpression of some deletion mutant forms of Alix/AIP1 and depletion of endogenous Alix/AIP1 with specific small interfering RNAs. The YLDL sequence was not replaceable by other L-domain motifs, such as PPxY and PT/SAP, and even YPxL. C protein was also able to physically interact with the N terminus of Alix/AIP1(212-357) and enhanced M-VLP budding independently of M-Alix/AIP1 interaction, although it was not released from the transfected cells itself. Our results suggest that the interaction of multiple viral proteins with Alix/AIP1 may enhance the efficiency of the utilization of cellular MVB sorting machinery for efficient SeV budding.",
         "T. Irie,Y. Shimazu,Tetsuya Yoshida,T. Sakaguchi",
         "T. Irie,Y. Shimazu,Tetsuya Yoshida,T. Sakaguchi",
         "https://europepmc.org/articles/pmc1865917?pdf=render",
         "",
         "",
         "",
         "",
         "",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "",
         "",
         "The YLDL Sequence within Sendai Virus M Protein Is Critical for Budding of Virus-Like Particles and Interacts with Alix/AIP1 Independently of C Protein",
         "None"
        ],
        [
         "32",
         "001c2ec8248cb8b5bc38ece17019ad0e133d7bec",
         "The aggregation of ataxin-3 is associated with spinocerebellar ataxia type 3, which is characterized by the formation of intraneuronal aggregates. However, the mechanism of aggregation is currently not well understood. Ataxin-3 consists of a folded Josephin domain followed by two ubiquitin-interacting motifs and a C-terminal polyglutamine tract, which in the non-pathological form is less than 45 residues in length. We demonstrate that ataxin-3 with 64 glutamines (at(Q64)) undergoes a two-stage aggregation. The first stage involves formation of SDS-soluble aggregates, and the second stage results in formation of SDS-insoluble aggregates via the poly(Q) region. Both these first and second stage aggregates display typical amyloid-like characteristics. Under the same conditions at(Q15) and at(QHQ) undergo a single step aggregation event resulting in SDS-soluble aggregates, which does not involve the polyglutamine tract. These aggregates do not convert to the SDS-insoluble form. These observations demonstrate that ataxin-3 has an inherent capacity to aggregate through its non-polyglutamine domains. However, the presence of a pathological length polyglutamine tract introduces an additional step resulting in formation of a highly stable amyloid-like aggregate.",
         "A. Ellisdon,Bronwen Thomas,S. Bottomley",
         "A. Ellisdon,Bronwen Thomas,S. Bottomley",
         "http://www.jbc.org/article/S0021925820557128/pdf",
         "The aggregation of ataxin-3 is associated with spinocerebellar\nataxia type 3, which is characterized by the formation of intraneu-\nronal aggregates. However, the mechanism of aggregation is cur-\nrently not well understood. Ataxin-3 consists of a folded Josephin\ndomain followed by two ubiquitin-interacting motifs and a C-ter-\nminal polyglutamine tract, which in the non-pathological form is\nless than 45 residues in length. We demonstrate that ataxin-3 with\n64 glutamines (at(Q64)) undergoes a two-stage aggregation. The\nfirst stage involves formation of SDS-soluble aggregates, and the\nsecond stage results in formation of SDS-insoluble aggregates via\nthe poly(Q) region. Both these first and second stage aggregates\ndisplay typical amyloid-like characteristics. Under the same condi-\ntions at(Q15) and at(QHQ) undergo a single step aggregation event\nresulting in SDS-soluble aggregates, which does not involve the\npolyglutamine tract. These aggregates do not convert to the SDS-\ninsoluble form. These observations demonstrate that ataxin-3 has\nan inherent capacity to aggregate through its non-polyglutamine\ndomains. However, the presence of a pathological length polyglu-\ntamine tract introduces an additional step resulting in formation of\na highly stable amyloid-like aggregate.",
         "DISCUSSION\nThe poly(Q) family represents a biomedically important class of pro-\nteins whose aggregation is intrinsically linked to the disease process. A\nnumber of reports have analyzed the aggregation properties of poly(Q)\npeptides (15, 18), fragments of poly(Q) proteins (16, 39, 45), or fusion\nproteins containing poly(Q) tracts (38, 52). However, little is known\nabout the aggregation of intact naturally occurring poly(Q) proteins.\nThis has become increasingly important as numerous studies have\nrecently highlighted the importance of protein context in the aggrega-\ntion of some poly(Q) proteins (34, 35). In this present study, we report\nthe kinetic and morphological changes of both non-expanded and\npathologically expanded ataxin-3 upon aggregation. In taking this\ncomparative approach, we have been able to explore both the role of\nprotein context, and the poly(Q) tract in defining the pathway of\nataxin-3 aggregation.\nThe major findings of this study arise from our ability to directly\ncompare the aggregation of at3(QHQ), at3(Q15), and at3(Q64). Our\ndetailed analysis of ataxin-3 aggregation showed that, independent of\nthe poly-Q tract length, the protein has an inherent propensity to aggre-\ngate. Aggregation of the shorter ataxin-3 variants (at3(QHQ) and\nat3(Q15)) occurred through a nucleation-dependent process that dis-\nplayed simultaneous changes in ThT fluorescence, intrinsic tryptophan\nfluorescence, and SEC. Furthermore, morphological analysis revealed\nthe formation of amyloid-like fibrils, which closely resemble those\nformed by the isolated Josephin domain (29). These data are consistent\nwith previous studies of non-expanded ataxin-3, under physiological\nand denaturing conditions, which have also suggested that this protein\nhas an intrinsic tendency to aggregate (32, 33). The biological relevance\nof this in-built aggregation propensity, displayed by both non-expanded\nataxin-3 and the Josephin domain (29), to the disease process is unclear.\nIt has been suggested that intracellular binding partners may strongly\nlimit the conformational flexibility of ataxin-3 and in turn its suscepti-\nbility to aggregation (29).\nConsistent with the poly(Q) tract length dependence of SCA3 (2)\nat3(Q64) has an increased rate of aggregation. The first stage of\nat3(Q64) aggregation is indistinguishable from that of the shorter vari-\nants. Strikingly, however, only at3(Q64) undergoes a second step, which\nis characterized by the formation of large SDS-insoluble aggregates.\nThese aggregates are dependent upon the presence of an expanded\npoly(Q) tract for their formation. The aggregates are highly stable and\nmorphologically resemble those formed in previous studies of expanded\npoly(Q) proteins and peptides (16, 39, 45, 53).\nOn the basis of our data we propose a minimal two-stage model of\nataxin-3 aggregation, which takes into account both protein context and\npoly(Q) expansion (Fig. 9). The first stage involves a conformational\nchange and self-association in regions other than the poly(Q) tract that\nresults in the formation of SDS-soluble aggregates; the second step that\noccurred solely in the protein with an expanded poly(Q) tract results in\nformation of an SDS-insoluble state. Previous aggregation analysis of\npoly(Q) peptides and huntingtin exon 1 identified a concerted single\nstage of aggregation, which is characterized by the formation of SDS-\ninsoluble aggregates and an increase in -sheet secondary structure\n(15–18, 53). Our data clearly demonstrate that for ataxin-3 this single\nstage model is not sufficient, due to the added complexity provided by\nregions other than the poly(Q) tract.\nThe early aggregate species formed during this first stage of aggrega-\ntion by all variants resemble the morphologies of pre-fibrillar aggregates\nimplicated in the toxicity of numerous neurodegenerative diseases (43–\n45, 49, 54). Similar spheroids and oligomers have previously been shown\nto form from non-expanded ataxin-3 and poly(Q) peptides, with anti-\nbody analysis indicating that such pre-fibrillar aggregates may share\nboth a common structure and toxic mechanism (33, 43, 54). Therefore,\nthe neuronal toxicity observed in SCA3 is potentially caused by the\nability of ataxin-3 to form these toxic structures. In keeping with this,\nrecent studies showed that overexpression of non-expanded poly(Q)\nproteins and other neurodegenerative-associated proteins in animal\nmodels can lead to toxicity (36, 55). Our observations suggest a toxic\nmechanism in agreement with this theory, in which expansion of the\npoly(Q) tract accelerates, but is not structurally involved in, the misfold-\ning of ataxin-3 into toxic intermediate structures. However, in contrast\nwith such a theory overexpression of wild-type ataxin-3 in a variety of\nanimal models has failed to reveal patterns of neural toxicity (56, 57).\nFurthermore, overexpression of wild-type ataxin-3 in a Drosophila\nmodel has recently been shown to be neuroprotective (57). Therefore,\nwhether such a proposed mechanism is relevant to disease-associated\ntoxicity requires further exploration. As such, current attempts are\nunderway to analyze the in vivo relevance of these structures to cellular\ntoxicity in ataxin-3.\nAlternatively, a wealth of data suggests that aggregates formed\nthrough poly(Q) tract interactions, whether they are spheroids, oli-\ngomers, or mature fibrils, are themselves the toxic species (58, 59). For\nexample, aggregates derived from poly(Q) peptides were found to be\ntoxic to mammalian cells when delivered to the nucleus (59). Further-\nmore, a mutagenesis approach has linked aggregation of the poly(Q)\ntract in huntingtin exon 1 to enhanced toxicity in vivo (58). In this\ncurrent study, the observation that only pathological length ataxin-3 is\nable to form poly(Q)-based fibrils is indeed compatible with such a\nmechanism. The increased stability of these poly(Q) aggregates would\nmost likely only serve to enhance their lifetime within the cell, and\ntherefore their chance of causing toxicity.\nIn this study, we have proposed a model of ataxin-3 aggregation in vitro\nthat takes into account both the role of protein context and the poly(Q)\ntract in modulating the aggregation pathway. In doing so, we have shown\nthe importance of taking a comparative approach to understand the com-\nplexity of poly(Q) aggregation. The aim now is to characterize these struc-\ntural changes and determine strategies to prevent them.",
         "Ataxin-3 is a 42-kDa multi-domain protein consisting of an N-termi-\nnal Josephin domain, two ubiquitin-interacting motifs, which are situ-\nated next to a polymorphous C-terminal polyglutamine (poly(Q))3 tract\n(1, 2). Ataxin-3 functions as a de-ubiquitinating enzyme (3, 4) and binds\npolyubiquitin chains through the ubiquitin-interacting motifs (5– 8).\nExpansion of the poly(Q) tract beyond 45 residues causes spinocerebel-\nlar ataxia type 3 (SCA3) also known as Machado-Joseph disease (9, 10).\nSimilar dynamic expansion of poly(Q) tracts within various other pro-\nteins causes a further eight autosomally dominant neurodegenerative\ndiseases, collectively termed poly(Q) diseases (2, 11).\nSeveral key observations have led to the conclusion that poly(Q) dis-\neases originate via a toxic gain of function, mediated by poly(Q) tract\nexpansion. Firstly, the manifestation of each poly(Q) disease is directly\nreliant on a threshold length of consecutive glutamine residues. In all of\nthe diseases, except for SCA6, this threshold is remarkably similar with\na tract length in excess of 40 residues associated with disease onset (2).\n* This work was supported in part by grants from the National Health and Medical\nResearch Council (NH&MRC). The costs of publication of this article were defrayed in\npart by the payment of page charges. This article must therefore be hereby marked\n“advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.\n1 An Australian Postgraduate scholar and recipient of a Monash University Faculty of\nSecondly, there is a non-linear correlation between an increasing gluta-\nmine tract length and an earlier age of disease onset (12). Thirdly, dif-\nferent proteins involved in each of the various poly(Q) diseases share no\nsequence homology except the presence of the glutamine tract (13).\nVarious studies have suggested that this toxic gain of function is caus-\nally linked to aberrant protein aggregation mediated by the extended\npoly(Q) tract (14). In vitro studies have shown that poly(Q) peptides,\nfragments, and proteins can form amyloid-like fibrillar aggregates and\nthat the aggregation rate increases with increasing glutamine tract\nlength (15–19). Various types of intraneuronal aggregates, including\nnuclear inclusions, of the poly(Q) protein in question are also evident\nupon post-mortem analysis (20 –22). These observations, among oth-\ners, suggest that poly(Q) diseases are conformational diseases, in which\ntoxicity is linked to the ability of the protein to self-associate and form\naggregates (23–27).\nWe and others have demonstrated that under physiologically rele-\nvant conditions expanded ataxin-3 can form amyloid-like fibrils (19, 28).\nFurthermore, under certain conditions non-expanded ataxin-3 can also\nform amyloid-like fibrils and their precursors (29 –33). The conclusion\nfrom these studies is that the poly(Q) tract is the main mediator of the\naggregation process, with other domains and surrounding residues\nplaying secondary roles in modulating aggregation (34).\nRecently however, it was shown that under destabilizing conditions\nthe Josephin domain alone is capable of forming amyloid-like fibrils,\nwith a similar rate to full-length ataxin-3 containing a 15-residue-long\npoly(Q) tract (29). The authors suggested that the Josephin domain is a\nmajor determinant in ataxin-3 aggregation, with the poly(Q) tract play-\ning a role in increasing the rate of fibril formation (29). Although it is\nclear that the poly(Q) tract is necessary for kinetic instigation of aggre-\ngation and, therefore, the disease process, these data and other work\nhave led to the hypothesis that non-poly(Q)-containing domains may\nplay a large role in determining the aggregation characteristics and tox-\nicity of poly(Q) proteins (35–37).\nWe have therefore studied a range of ataxin-3 variants in an attempt\nto understand how both the poly(Q) tract and protein context kineti-\ncally and structurally characterize the aggregation pathway. These vari-\nants include ataxin-3 without a poly(Q) tract (at3(QHQ)), a non-patho-\nlogical length variant with 15 glutamines (at3(Q15)), and pathological\nlength ataxin-3 with 64 glutamines (at3(Q64)). We firstly demonstrate\nthat full-length ataxin-3 has an intrinsic ability to form amyloid-like\nfibrils independent of the poly(Q) tract. However, we further show that\na pathological length poly(Q) tract introduces a second step into the\naggregation pathway, which dramatically alters the stability and mor-\nphology of the end-stage fibrils.\nEXPERIMENTAL PROCEDURES\nMaterials—Phenylmethylsulfonyl fluoride, thioflavin T (ThT), and\n-mercaptoethanol were all obtained from Sigma.",
         "Expression and Purification of Ataxin-3 Variants—All ataxin-3 vari-\nants were expressed and purified as described previously (19). Protein\nwas stored at 80 °C and run on a Superose 12 column prior to each\nexperiment to confirm it was monomeric.\nFibrillogenesis Time-course Assays—Ataxin-3 variants were incu-\nbated at a concentration of 30 M at pH 7.4 in Tris-buffered saline (TBS)\n(100 mM Tris, 80 mM NaCl) containing 10% (v/v) glycerol, phenylmeth-\nylsulfonyl fluoride (2 mM), EDTA (5 mM), and -mercaptoethanol (15\nmM). Reactions were performed at 37 °C without shaking and in air-\ntight containers to eliminate evaporation.\nMembrane Filter Trap Assay—The membrane filter-trap assay was\ncarried out as previously described with only minor modifications (16,\n17). An aliquot of protein (30 g) was removed from the fibrillogenesis\ntime-course incubations and diluted into 4% SDS (w/v) and 100 mM\ndithiothreitol at a 1:1 ratio. The samples were heated for 5 min at 100 °C\nbefore being diluted to 200 l with 0.1% (w/v) SDS. The samples were\nfiltered through a cellulose acetate membrane (Schleicher and Schuell,\n0.2 m) using a Bio-Rad Bio-Dot SF microfiltration unit. After applica-\ntion, the membrane was washed twice by filtering through 200 l of\n0.1% SDS (w/v). The membranes were then blotted with a hexahistidine\n(His6) antibody (Serotec).\nThT Measurements—ThT fluorescence was measured using a BMG\nLaboratories FLUOstar Optima fluorescence plate reader. Excitation\nand emission filters used were for 450 and 490 nm, respectively. All\nreactions were carried out in black 384-well clear bottom plates with\nboth excitation and emission read from the bottom. Reactions were\ncarried out at 37 °C, without shaking, and the top of the plate was sealed\nto prevent any evaporation. Plate reader assays contained 20 M ThT,\nwith all other buffering solutions and additives remaining the same as\npreviously described herein. Discontinuous assays, carried out without\nthe continual presence of ThT, displayed comparable kinetics to the\ncontinuous plate reader assays (data not shown).\nSDS Stability Assays—An aliquot of 15 g of end-stage fibrils was\nremoved from the fibril assays and immediately diluted into 200 l of\nSDS at the indicated concentration. Samples were incubated for 20 min\nat room temperature before being filtered through a 0.2-m cellulose\nacetate membrane (Schleicher & Schuell). The membrane was pre-\nequilibrated in TBS, and after filtration 200 l of TBS was filtered\nthrough the membrane as a washing stage. The membrane was then\nprobed as previously described for the membrane filter-trap assay.\nQBP1 Aggregation Assays—The QBP1 peptide was purchased from\nAusPep with the same sequence as described previously (38). The pep-\ntide was dissolved in Me2SO to a final concentration of 13.5 mM. The\npeptide was incubated with ataxin-3 aggregation assays at a concentra-\ntion of 120 M. Upon final dilution the concentration of Me2SO was\n1%. Control assays with identical final Me2SO concentrations were\nused for comparison with QBP1 assays.\nFluorescence Spectroscopy—All fluorescence measurements were\nperformed by diluting aliquots of sample into TBS (pH 7.4) to a final\nconcentration of 1 M. Fluorescence emission spectra were immedi-\nately recorded on a PerkinElmer Life Sciences LS50B spectrofluorom-\neter with a thermostatted cuvette holder at 25 °C, using a 1-cm path\nlength quartz cuvette. For intrinsic fluorescence measurements an exci-\ntation wavelength of 280 nm was used, with measurement of the emis-\nsion spectra from 300 to 450 nm. Emission and excitation slit widths\nwere set at 2.5 and 3.0 nm, respectively, and a scan speed of 25 nm/min\nwas used.\nSize Exclusion Analysis of Fibril Formation—Size exclusion chroma-\ntography (SEC) was carried out on an Amersham Biosciences fast pro-\ntein liquid chromatography system. Fibril formation was monitored by\nthe loss of area of the monomer peak on a Superose 12 column. At\nindicated time points 20 g of protein was injected onto the column,\nand its elution was monitored by absorbance at 214 nm. The area of the\npeak was calculated using the manufacturers supplied software.\nCircular Dichroism—CD analysis was carried out by removing ali-\nquots of protein from the incubating sample and diluting into TBS to\n21.4 M immediately prior to analysis. All far-UV CD spectra were\nmeasured on a Jasco-810 spectropolarimeter with a path length of 0.1\nmm. Spectra were recorded from 190 to 260 nm, using a 5-s/point signal\naveraging, and a scan speed of 20 nm/min.\nTransmission Electron Microscopy—TEM images were obtained\nusing a Jeol JEM-200CX transmission electron microscope. The accel-\neration voltage was 100 kV. Samples were adsorbed onto carbon-coated\ngrids and stained with 1% (w/v) uranyl acetate.\nAnalysis of Seeded Aggregation—End-stage aggregates were added to\nthe fibril reaction immediately prior to incubation at 10% of the total\nmolar concentration. A reaction containing seed only was analyzed, and\nthe result was used to correct kinetics of seeded samples. ThT and\nmembrane filter-trap assays were carried out as previously described\nherein.",
         "Expression and Purification of Ataxin-3 Variants—All ataxin-3 vari-\nants were expressed and purified as described previously (19). Protein\nwas stored at 80 °C and run on a Superose 12 column prior to each\nexperiment to confirm it was monomeric.\nFibrillogenesis Time-course Assays—Ataxin-3 variants were incu-\nbated at a concentration of 30 M at pH 7.4 in Tris-buffered saline (TBS)\n(100 mM Tris, 80 mM NaCl) containing 10% (v/v) glycerol, phenylmeth-\nylsulfonyl fluoride (2 mM), EDTA (5 mM), and -mercaptoethanol (15\nmM). Reactions were performed at 37 °C without shaking and in air-\ntight containers to eliminate evaporation.\nMembrane Filter Trap Assay—The membrane filter-trap assay was\ncarried out as previously described with only minor modifications (16,\n17). An aliquot of protein (30 g) was removed from the fibrillogenesis\ntime-course incubations and diluted into 4% SDS (w/v) and 100 mM\ndithiothreitol at a 1:1 ratio. The samples were heated for 5 min at 100 °C\nbefore being diluted to 200 l with 0.1% (w/v) SDS. The samples were\nfiltered through a cellulose acetate membrane (Schleicher and Schuell,\n0.2 m) using a Bio-Rad Bio-Dot SF microfiltration unit. After applica-\ntion, the membrane was washed twice by filtering through 200 l of\n0.1% SDS (w/v). The membranes were then blotted with a hexahistidine\n(His6) antibody (Serotec).\nThT Measurements—ThT fluorescence was measured using a BMG\nLaboratories FLUOstar Optima fluorescence plate reader. Excitation\nand emission filters used were for 450 and 490 nm, respectively. All\nreactions were carried out in black 384-well clear bottom plates with\nboth excitation and emission read from the bottom. Reactions were\ncarried out at 37 °C, without shaking, and the top of the plate was sealed\nto prevent any evaporation. Plate reader assays contained 20 M ThT,\nwith all other buffering solutions and additives remaining the same as\npreviously described herein. Discontinuous assays, carried out without\nthe continual presence of ThT, displayed comparable kinetics to the\ncontinuous plate reader assays (data not shown).\nSDS Stability Assays—An aliquot of 15 g of end-stage fibrils was\nremoved from the fibril assays and immediately diluted into 200 l of\nSDS at the indicated concentration. Samples were incubated for 20 min\nat room temperature before being filtered through a 0.2-m cellulose\nacetate membrane (Schleicher & Schuell). The membrane was pre-\nequilibrated in TBS, and after filtration 200 l of TBS was filtered\nthrough the membrane as a washing stage. The membrane was then\nprobed as previously described for the membrane filter-trap assay.\nQBP1 Aggregation Assays—The QBP1 peptide was purchased from\nAusPep with the same sequence as described previously (38). The pep-\ntide was dissolved in Me2SO to a final concentration of 13.5 mM. The\npeptide was incubated with ataxin-3 aggregation assays at a concentra-\ntion of 120 M. Upon final dilution the concentration of Me2SO was\n1%. Control assays with identical final Me2SO concentrations were\nused for comparison with QBP1 assays.\nFluorescence Spectroscopy—All fluorescence measurements were\nperformed by diluting aliquots of sample into TBS (pH 7.4) to a final\nconcentration of 1 M. Fluorescence emission spectra were immedi-\nately recorded on a PerkinElmer Life Sciences LS50B spectrofluorom-\neter with a thermostatted cuvette holder at 25 °C, using a 1-cm path\nlength quartz cuvette. For intrinsic fluorescence measurements an exci-\ntation wavelength of 280 nm was used, with measurement of the emis-\nsion spectra from 300 to 450 nm. Emission and excitation slit widths\nwere set at 2.5 and 3.0 nm, respectively, and a scan speed of 25 nm/min\nwas used.\nSize Exclusion Analysis of Fibril Formation—Size exclusion chroma-\ntography (SEC) was carried out on an Amersham Biosciences fast pro-\ntein liquid chromatography system. Fibril formation was monitored by\nthe loss of area of the monomer peak on a Superose 12 column. At\nindicated time points 20 g of protein was injected onto the column,\nand its elution was monitored by absorbance at 214 nm. The area of the\npeak was calculated using the manufacturers supplied software.\nCircular Dichroism—CD analysis was carried out by removing ali-\nquots of protein from the incubating sample and diluting into TBS to\n21.4 M immediately prior to analysis. All far-UV CD spectra were\nmeasured on a Jasco-810 spectropolarimeter with a path length of 0.1\nmm. Spectra were recorded from 190 to 260 nm, using a 5-s/point signal\naveraging, and a scan speed of 20 nm/min.\nTransmission Electron Microscopy—TEM images were obtained\nusing a Jeol JEM-200CX transmission electron microscope. The accel-\neration voltage was 100 kV. Samples were adsorbed onto carbon-coated\ngrids and stained with 1% (w/v) uranyl acetate.\nAnalysis of Seeded Aggregation—End-stage aggregates were added to\nthe fibril reaction immediately prior to incubation at 10% of the total\nmolar concentration. A reaction containing seed only was analyzed, and\nthe result was used to correct kinetics of seeded samples. ThT and\nmembrane filter-trap assays were carried out as previously described\nherein.\nRESULTS\nCharacterization of Ataxin-3 Variants—We expressed and purified\nthree ataxin-3 variants from Escherichia coli as previously described\n(19) (Fig. 1A). Using SEC (Fig. 1B) and mass spectrometry (data not\nshown) we confirmed that the final purified proteins were entirely\nmonomeric and intact. Furthermore, all the variants possessed ubiq-\nuitin-protease activity as previously described (data not shown) (3).\nOnly Expanded Ataxin-3 Forms SDS-resistant Aggregates—One\ncommon feature of poly(Q) aggregates is their ability to retain structure\nupon incubation with SDS (16, 17, 39). We initially analyzed the aggre-\ngation of all ataxin-3 variants using the membrane filter-trap assay rou-\ntinely used to detect SDS-resistant huntingtin exon 1 fibrils (16, 17, 39).\nUpon incubation of all variants, at a concentration of 30 M, we observe\nthat only pathological length at3(Q64) forms SDS-resistant aggregates\n(Fig. 2A). Further analysis of the aggregation with densitometry reveals\nthat at3(Q64) aggregation follows a classic nucleation-dependent time\ning conditions (19, 29 –31). Therefore, it is possible that the protein\nregion responsible for the formation of ThT-positive aggregates\ninvolves the N-terminal Josephin domain, which is common to all of the\nproteins, although further work is needed to clarify this (29, 33).\nInspection of the kinetics derived from the SDS-insolubility and ThT\nassays (Fig. 2) suggests that pathological length at3(Q64) aggregates in\ntwo stages. The first is a ThT-detectable stage common to all variants,\nand this is followed by a second poly(Q)-length dependent stage in\nwhich classic SDS-insoluble poly(Q) aggregates are formed. A small\ndegree of overlap is seen between the two phases, however the bulk of\nSDS-resistant aggregates do not appear to form until after a significant\nThT increase.\nTherefore, our data indicate that the three variants form different\naggregates based upon the length of their poly-Q tract. If this is the case\nthen the aggregates formed are likely to possess different stabilities. To\nexamine these differences we analyzed the resistance of these end-stage\naggregates to SDS. End-stage aggregates of the three proteins were incu-\nbated with increasing concentrations of SDS to determine any stability\ndifferences (Fig. 3). As can be seen, only at3(Q64) aggregates are stable\nupon incubation in up to 1% (w/v) SDS. The two shorter variants both\nlose the ability to be retained on the 0.2-m filter after incubation in a\nvery low 0.01% (w/v) SDS. This strongly suggests that the aggregates\nformed from expanded ataxin-3 are structurally different to those of the\nnon-expanded variants. Furthermore,\nthis increased stability of\nexpanded ataxin-3 aggregates may have ramifications in the pathogen-\nesis of the disease, as more stable aggregates are likely to be harder to\nbreak down and longer lived.\nQBP1 Inhibits the Second Phase of at3(Q64) Aggregation—The data\npresented above suggest a two-step mechanism of aggregation with only\nthe pathological length poly(Q) tract playing a role in formation of SDS-\nstable aggregates. To confirm this, we have used the QBP1 peptide as a\ntool to explore the role of the poly(Q) tract in defining the at3(Q64)\naggregation pathway. The QBP1 peptide is 11 amino acids in length and\nhas been shown to inhibit poly(Q) aggregation both in vitro and in vivo\n(38, 42). In Fig. 4 we followed the aggregation of 30 M at3(Q64) by the\ncombination of the filter-trap assay and the ThT assay. Upon incubation\nof the QBP1 peptide, at a molar ratio of 4:1 (peptide:protein), we still\nobserve the initial ThT increase (Fig. 4C). However, the second stage of\naggregation is completely eliminated, with no detectable SDS-insoluble\naggregates formed in the presence of QBP1 (Fig. 4, A and B). Taken\ntogether these data indicate that expanded ataxin-3 aggregates through\ncourse, with a defined lag period followed by a rapid growth phase (Fig.\n2B). Even upon incubation times of up to 200 h, at3(QHQ) and at3(Q15)\ndid not form SDS-resistant aggregates, suggesting that under physiolog-\nically relevant conditions ataxin-3 forms SDS-resistant aggregates in a\npoly(Q) length-dependent manner similar to that seen in other poly(Q)\nproteins (15–19).\nIn addition to the filter-trap assay we followed the aggregation reac-\ntion with the fluorescent dye ThT. We analyzed the aggregation prop-\nerties of the three ataxin-3 variants, under identical conditions (Fig. 2C).\nIn contrast to the filter-trap assay we observed that all the variants are\nable to form ThT-positive aggregates. The rate of aggregation was\npoly(Q)-length dependent, with at3(Q64) aggregating at a faster rate\nthan either at3(QHQ) or at3(Q15). However intriguingly, both\nat3(QHQ) and at3(Q15) aggregate with essentially identical kinetics\n(Table 1). The kinetics display time-squared dependence (data not\nshown) suggestive of a nucleation-dependent time course (40, 41). As\nat3(QHQ) contains no poly(Q) tract, it appears that the ataxin-3 protein\ncontains an inherent propensity to form ThT-positive aggregates inde-\npendently of the poly(Q) tract. In a recent study, the Josephin domain of\nataxin-3 has been shown to form amyloid-like aggregates upon thermal\nstress (29). Furthermore, numerous ataxin-3 variants with non-patho-\nlogical length poly(Q) tracts have been shown to form amyloid-like\naggregates under both near-physiological conditions (33) and denatur-\na two-stage pathway, with only the second stage reliant on the presence\nof an expanded glutamine tract. We then went on to further structurally\nand kinetically characterize this pathway using a range of techniques.\nAtaxin-3 Aggregation Monitored by Intrinsic Fluorescence and Size\nExclusion Chromatography—The two-step mechanism of aggregation\nproposed above suggests that the N-terminal domain of ataxin-3, which\nis common to all the proteins, is involved in the first step. To determine\nif this is the case we exploited the fact that ataxin-3 contains three\ntryptophan residues, all of which are located within the Josephin\ndomain, as such they act as a highly sensitive reporter groups for the\ntertiary structure of this domain. We therefore followed the changes in\nintrinsic tryptophan fluorescence during aggregation for all three pro-\nteins. We observe that, upon formation of the final aggregate conforma-\ntion, all three variants adopt a hyper-fluorescent state in which the\nemission maxima has shifted from 333 nm in the native state to 340 nm\nin the aggregate (Fig. 5, A–C). These data suggest that Josephin has\nundergone a conformational change during aggregation and is in a sim-\nilar non-native conformation in the final aggregation states for all three\nvariants. For at3(Q64) there was no difference in emission spectra\nbetween the SDS-soluble and SDS-stable aggregates, suggesting that the\nJosephin domain undergoes no detectable structural rearrangement\nupon the formation of the SDS-stable aggregates. We also used this\nchange in fluorescence to follow the kinetics of aggregation in reference\nto the conformational changes in the Josephin domain (Fig. 5, D–F).\nInterestingly, the fluorescence-derived kinetics correlates well with the\ngain of ThT signal (Table 1). These data further suggest that the Jose-\nphin domain plays a crucial structural role only in the first stage of\nataxin-3 aggregation.\nTo further analyze the aggregation pathway of ataxin-3 we used SEC\nto follow the loss of monomer from solution during aggregation (Fig. 5,\nD–F). For all three variants, monomer loss showed a good correlation\nwith both aggregation monitored by ThT and intrinsic fluorescence\n(Table 1). Our data are consistent with a model of aggregation, in which\nmonomer conversion into the growing fibril is characterized by a struc-\ntural rearrangement of the Josephin domain. During this first stage of\naggregation, the aggregates are soluble, based upon their ability to run\nbefore or within the void volume of the SEC column (data not shown),\nand the lack of any increase in light scatter (data not shown). The second\nstage of aggregation is accompanied by a dramatic loss of solubility of\nthe aggregates as they visibly precipitate from solution. This stage pre-\ndominantly occurs after the loss of monomer from solution, suggesting\nthat SDS-insoluble aggregates are formed through the summation of\nsmaller aggregates, rather than by monomer addition to the end of\ngrowing fibrils. More detailed analysis of these stages of aggregation is\ncurrently underway to further kinetically characterize the stages of\nataxin-3 aggregation.\nAggregate Formation Is Associated with an Increase in -Sheet Sec-\nondary Structure—The changes in secondary structure of the ataxin-3\nvariants during aggregation were monitored using far-UV CD spectros-\ncopy. At 0 h all variants display CD spectra similar to those previously\nreported (19), indicative of a predominantly -helical protein (Fig. 6).\nAfter completion of aggregation at 96 h, both at3(QHQ) (Fig. 6A) and\nat3(Q15) (Fig. 6B) have undergone similar changes, with a gain of signal\nat 220 nm causing a flattening of the CD spectra. These spectral changes\nare indicative of an increase in -sheet content, and a loss of -helix.\nThis is similar to that recently reported for fibril formation of the Jose-\nphin domain and non-pathological lengths of ataxin-3 under thermal\ndenaturation (29, 31).\nThe secondary structural changes of at3(Q64) during the same incu-\nbation are more complex (Fig. 6C). An initial change in secondary struc-\nture, comparable to that of the non-pathological length variants, is seen\nafter 24 h during the first phase of aggregation. Secondary structural\nchanges are also evident upon the formation of SDS-stable aggregates,\nhowever during this phase the CD spectrum undergoes a significant loss\nof signal, thought to be caused by a large increase in light scatter during\nthe second phase of aggregation (data not shown). This increase in light\nscatter unfortunately precludes us from obtaining detailed information\non the secondary structure of end-point at3(Q64) fibrils.\nPoly(Q) Tract Length Affects End-stage Aggregate Morphology—TEM\nhas been used in a range of studies to observe both intermediate assem-\nblies formed prior to fibril formation and end-stage aggregates (43– 45).\nHere, we have undertaken a detailed analysis of the intermediate species\nand fibril morphologies formed during aggregation of all ataxin-3 vari-\nants at 30 M. At 0 h, for all variants, the TEM grid has no prominent\nobservable structures consistent with the ataxin-3 monomer as the pre-\ndominant species (data not shown).\nThe first structures we observe, for all three variants, are a combina-\ntion of small spheroid-like aggregates and short curvilinear oligomeric\naggregates (Fig. 7, A–C). Consistent with the ThT kinetics, these struc-\ntures are formed at a faster rate in at3(Q64) than the other two shorter\nvariants. The diameters of the spheroids vary between 5 and 20 nm,\nwhich are similar to those observed recently with non-expanded\nataxin-3 (33), other poly(Q) proteins/peptides (39, 43, 45), and amyloi-\ndogenic proteins (43, 46 – 48). The oligomeric aggregates also mirror\nthose previously observed for non-expanded ataxin-3 (33), having a\nconsistent diameter 10 –12 nm and a length that increases with time.\nThese short curvilinear aggregates also strongly resemble the protofi-\nbrils observed during amyloid- aggregation (44, 49), -synuclein\naggregation (48, 50), and previous poly(Q) studies (45, 51). As aggrega-\ntion continues these oligomers elongate, up to many hundreds of\nnanometers in length, while their diameter remains the same (Fig. 7,\nD–F). These banded aggregates appear quite flexible and have a very\nsimilar morphology to that of thermally derived Josephin aggregates\n(29). For both at3(QHQ) and at3(Q15) this represents the end-stage\nmorphology of the aggregation pathway with no change in structure\nobserved at later time points (Fig. 7, G and H).\nAt3(Q64) aggregates further into a second type of morphology whose\noccurrence is concurrent with the second phase of aggregation (Fig. 7I).\nThese aggregates are larger and more fibrous than the banded aggre-\ngates formed during the previous stage, resembling those previously\nreported for at3(Q50) (19). Upon close analysis, some individual aggre-\ngates also resemble the beaded morphology previously reported for\nat3(Q78) (28). These fibrils tend to self-associate and form large insol-\nuble aggregates, making their characterization by TEM difficult. These\nlarge filamentous aggregates are usually 50 –100 nm in width and can be\nup to many microns in length. The banded aggregates that represent the\nend-stage morphology of the shorter variants no longer appear to be\npresent.\nCross-seeding Reveals a Model Pathway of Ataxin-3 Aggregation—\nOur data suggest that the conformation of the final aggregates of\nat3(Q15) and at3(QHQ) are structurally different from those of\nat3(Q64). One of the properties of fibrillizing proteins, including the\npoly(Q) family of proteins, is that propagation of the aggregates can be\naccomplished by the presence of “seed,” which catalyzes the aggregation\nof free protein. We next determined whether the different aggregates\nproduced by the different length repeat proteins could catalyze the\naggregation of each other or there is a structural barrier within this\nfamily of aggregates that limits its propagation. Cross-seeding of\neach of the three variants was analyzed initially by ThT assay (Fig. 8).\nAll of the aggregation assays monitored display no observable lag\nphase and follow a linear gradient, indicative of the abolishment of\nthe rate-limiting nucleation stage (Fig. 8, A–C). This confirms that\nataxin-3 aggregation is nucleation-dependent, in line with previous\nstudies of ataxin-3 (33), and kinetically is indicative of amyloid-like\nfibril formation (17). Furthermore, as each of the variants are able to\ncross-seed the aggregation pathways of the other, these data suggest\nthat the aggregation pathways of the three proteins are also struc-\nturally similar, with each variant able to act as a template for the\nelongation of the other variants.\nWe then examined the same cross-seeding assays by the membrane\nfilter-trap assay, which reports the formation of SDS-stable aggregates.\nIn Fig. 8 (D and E), we can see that none of the variants are able to\ncross-seed the shorter variants into forming SDS-stable aggregates. This\nis most likely because the poly(Q) tracts in each of the variants are either\nnon-existent or too short to readily form poly(Q)-dependent aggregates\nover the timeframe analyzed. Only at3(Q64) aggregates can noticeably\naccelerate the formation of poly(Q)-dependent\nfibrils from free\nat3(Q64) (Fig. 8F).",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "The aggregation of ataxin-3 is associated with spinocerebellar\nataxia type 3, which is characterized by the formation of intraneu-\nronal aggregates. However, the mechanism of aggregation is cur-\nrently not well understood. Ataxin-3 consists of a folded Josephin\ndomain followed by two ubiquitin-interacting motifs and a C-ter-\nminal polyglutamine tract, which in the non-pathological form is\nless than 45 residues in length. We demonstrate that ataxin-3 with\n64 glutamines (at(Q64)) undergoes a two-stage aggregation. The\nfirst stage involves formation of SDS-soluble aggregates, and the\nsecond stage results in formation of SDS-insoluble aggregates via\nthe poly(Q) region. Both these first and second stage aggregates\ndisplay typical amyloid-like characteristics. Under the same condi-\ntions at(Q15) and at(QHQ) undergo a single step aggregation event\nresulting in SDS-soluble aggregates, which does not involve the\npolyglutamine tract. These aggregates do not convert to the SDS-\ninsoluble form. These observations demonstrate that ataxin-3 has\nan inherent capacity to aggregate through its non-polyglutamine\ndomains. However, the presence of a pathological length polyglu-\ntamine tract introduces an additional step resulting in formation of\na highly stable amyloid-like aggregate.\nAtaxin-3 is a 42-kDa multi-domain protein consisting of an N-termi-\nnal Josephin domain, two ubiquitin-interacting motifs, which are situ-\nated next to a polymorphous C-terminal polyglutamine (poly(Q))3 tract\n(1, 2). Ataxin-3 functions as a de-ubiquitinating enzyme (3, 4) and binds\npolyubiquitin chains through the ubiquitin-interacting motifs (5– 8).\nExpansion of the poly(Q) tract beyond 45 residues causes spinocerebel-\nlar ataxia type 3 (SCA3) also known as Machado-Joseph disease (9, 10).\nSimilar dynamic expansion of poly(Q) tracts within various other pro-\nteins causes a further eight autosomally dominant neurodegenerative\ndiseases, collectively termed poly(Q) diseases (2, 11).\nSeveral key observations have led to the conclusion that poly(Q) dis-\neases originate via a toxic gain of function, mediated by poly(Q) tract\nexpansion. Firstly, the manifestation of each poly(Q) disease is directly\nreliant on a threshold length of consecutive glutamine residues. In all of\nthe diseases, except for SCA6, this threshold is remarkably similar with\na tract length in excess of 40 residues associated with disease onset (2).\n* This work was supported in part by grants from the National Health and Medical\nResearch Council (NH&MRC). The costs of publication of this article were defrayed in\npart by the payment of page charges. This article must therefore be hereby marked\n“advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.\n1 An Australian Postgraduate scholar and recipient of a Monash University Faculty of\nSecondly, there is a non-linear correlation between an increasing gluta-\nmine tract length and an earlier age of disease onset (12). Thirdly, dif-\nferent proteins involved in each of the various poly(Q) diseases share no\nsequence homology except the presence of the glutamine tract (13).\nVarious studies have suggested that this toxic gain of function is caus-\nally linked to aberrant protein aggregation mediated by the extended\npoly(Q) tract (14). In vitro studies have shown that poly(Q) peptides,\nfragments, and proteins can form amyloid-like fibrillar aggregates and\nthat the aggregation rate increases with increasing glutamine tract\nlength (15–19). Various types of intraneuronal aggregates, including\nnuclear inclusions, of the poly(Q) protein in question are also evident\nupon post-mortem analysis (20 –22). These observations, among oth-\ners, suggest that poly(Q) diseases are conformational diseases, in which\ntoxicity is linked to the ability of the protein to self-associate and form\naggregates (23–27).\nWe and others have demonstrated that under physiologically rele-\nvant conditions expanded ataxin-3 can form amyloid-like fibrils (19, 28).\nFurthermore, under certain conditions non-expanded ataxin-3 can also\nform amyloid-like fibrils and their precursors (29 –33). The conclusion\nfrom these studies is that the poly(Q) tract is the main mediator of the\naggregation process, with other domains and surrounding residues\nplaying secondary roles in modulating aggregation (34).\nRecently however, it was shown that under destabilizing conditions\nthe Josephin domain alone is capable of forming amyloid-like fibrils,\nwith a similar rate to full-length ataxin-3 containing a 15-residue-long\npoly(Q) tract (29). The authors suggested that the Josephin domain is a\nmajor determinant in ataxin-3 aggregation, with the poly(Q) tract play-\ning a role in increasing the rate of fibril formation (29). Although it is\nclear that the poly(Q) tract is necessary for kinetic instigation of aggre-\ngation and, therefore, the disease process, these data and other work\nhave led to the hypothesis that non-poly(Q)-containing domains may\nplay a large role in determining the aggregation characteristics and tox-\nicity of poly(Q) proteins (35–37).\nWe have therefore studied a range of ataxin-3 variants in an attempt\nto understand how both the poly(Q) tract and protein context kineti-\ncally and structurally characterize the aggregation pathway. These vari-\nants include ataxin-3 without a poly(Q) tract (at3(QHQ)), a non-patho-\nlogical length variant with 15 glutamines (at3(Q15)), and pathological\nlength ataxin-3 with 64 glutamines (at3(Q64)). We firstly demonstrate\nthat full-length ataxin-3 has an intrinsic ability to form amyloid-like\nfibrils independent of the poly(Q) tract. However, we further show that\na pathological length poly(Q) tract introduces a second step into the\naggregation pathway, which dramatically alters the stability and mor-\nphology of the end-stage fibrils.\nEXPERIMENTAL PROCEDURES\nMaterials—Phenylmethylsulfonyl fluoride, thioflavin T (ThT), and\n-mercaptoethanol were all obtained from Sigma.\nExpression and Purification of Ataxin-3 Variants—All ataxin-3 vari-\nants were expressed and purified as described previously (19). Protein\nwas stored at 80 °C and run on a Superose 12 column prior to each\nexperiment to confirm it was monomeric.\nFibrillogenesis Time-course Assays—Ataxin-3 variants were incu-\nbated at a concentration of 30 M at pH 7.4 in Tris-buffered saline (TBS)\n(100 mM Tris, 80 mM NaCl) containing 10% (v/v) glycerol, phenylmeth-\nylsulfonyl fluoride (2 mM), EDTA (5 mM), and -mercaptoethanol (15\nmM). Reactions were performed at 37 °C without shaking and in air-\ntight containers to eliminate evaporation.\nMembrane Filter Trap Assay—The membrane filter-trap assay was\ncarried out as previously described with only minor modifications (16,\n17). An aliquot of protein (30 g) was removed from the fibrillogenesis\ntime-course incubations and diluted into 4% SDS (w/v) and 100 mM\ndithiothreitol at a 1:1 ratio. The samples were heated for 5 min at 100 °C\nbefore being diluted to 200 l with 0.1% (w/v) SDS. The samples were\nfiltered through a cellulose acetate membrane (Schleicher and Schuell,\n0.2 m) using a Bio-Rad Bio-Dot SF microfiltration unit. After applica-\ntion, the membrane was washed twice by filtering through 200 l of\n0.1% SDS (w/v). The membranes were then blotted with a hexahistidine\n(His6) antibody (Serotec).\nThT Measurements—ThT fluorescence was measured using a BMG\nLaboratories FLUOstar Optima fluorescence plate reader. Excitation\nand emission filters used were for 450 and 490 nm, respectively. All\nreactions were carried out in black 384-well clear bottom plates with\nboth excitation and emission read from the bottom. Reactions were\ncarried out at 37 °C, without shaking, and the top of the plate was sealed\nto prevent any evaporation. Plate reader assays contained 20 M ThT,\nwith all other buffering solutions and additives remaining the same as\npreviously described herein. Discontinuous assays, carried out without\nthe continual presence of ThT, displayed comparable kinetics to the\ncontinuous plate reader assays (data not shown).\nSDS Stability Assays—An aliquot of 15 g of end-stage fibrils was\nremoved from the fibril assays and immediately diluted into 200 l of\nSDS at the indicated concentration. Samples were incubated for 20 min\nat room temperature before being filtered through a 0.2-m cellulose\nacetate membrane (Schleicher & Schuell). The membrane was pre-\nequilibrated in TBS, and after filtration 200 l of TBS was filtered\nthrough the membrane as a washing stage. The membrane was then\nprobed as previously described for the membrane filter-trap assay.\nQBP1 Aggregation Assays—The QBP1 peptide was purchased from\nAusPep with the same sequence as described previously (38). The pep-\ntide was dissolved in Me2SO to a final concentration of 13.5 mM. The\npeptide was incubated with ataxin-3 aggregation assays at a concentra-\ntion of 120 M. Upon final dilution the concentration of Me2SO was\n1%. Control assays with identical final Me2SO concentrations were\nused for comparison with QBP1 assays.\nFluorescence Spectroscopy—All fluorescence measurements were\nperformed by diluting aliquots of sample into TBS (pH 7.4) to a final\nconcentration of 1 M. Fluorescence emission spectra were immedi-\nately recorded on a PerkinElmer Life Sciences LS50B spectrofluorom-\neter with a thermostatted cuvette holder at 25 °C, using a 1-cm path\nlength quartz cuvette. For intrinsic fluorescence measurements an exci-\ntation wavelength of 280 nm was used, with measurement of the emis-\nsion spectra from 300 to 450 nm. Emission and excitation slit widths\nwere set at 2.5 and 3.0 nm, respectively, and a scan speed of 25 nm/min\nwas used.\nSize Exclusion Analysis of Fibril Formation—Size exclusion chroma-\ntography (SEC) was carried out on an Amersham Biosciences fast pro-\ntein liquid chromatography system. Fibril formation was monitored by\nthe loss of area of the monomer peak on a Superose 12 column. At\nindicated time points 20 g of protein was injected onto the column,\nand its elution was monitored by absorbance at 214 nm. The area of the\npeak was calculated using the manufacturers supplied software.\nCircular Dichroism—CD analysis was carried out by removing ali-\nquots of protein from the incubating sample and diluting into TBS to\n21.4 M immediately prior to analysis. All far-UV CD spectra were\nmeasured on a Jasco-810 spectropolarimeter with a path length of 0.1\nmm. Spectra were recorded from 190 to 260 nm, using a 5-s/point signal\naveraging, and a scan speed of 20 nm/min.\nTransmission Electron Microscopy—TEM images were obtained\nusing a Jeol JEM-200CX transmission electron microscope. The accel-\neration voltage was 100 kV. Samples were adsorbed onto carbon-coated\ngrids and stained with 1% (w/v) uranyl acetate.\nAnalysis of Seeded Aggregation—End-stage aggregates were added to\nthe fibril reaction immediately prior to incubation at 10% of the total\nmolar concentration. A reaction containing seed only was analyzed, and\nthe result was used to correct kinetics of seeded samples. ThT and\nmembrane filter-trap assays were carried out as previously described\nherein.\nRESULTS\nCharacterization of Ataxin-3 Variants—We expressed and purified\nthree ataxin-3 variants from Escherichia coli as previously described\n(19) (Fig. 1A). Using SEC (Fig. 1B) and mass spectrometry (data not\nshown) we confirmed that the final purified proteins were entirely\nmonomeric and intact. Furthermore, all the variants possessed ubiq-\nuitin-protease activity as previously described (data not shown) (3).\nOnly Expanded Ataxin-3 Forms SDS-resistant Aggregates—One\ncommon feature of poly(Q) aggregates is their ability to retain structure\nupon incubation with SDS (16, 17, 39). We initially analyzed the aggre-\ngation of all ataxin-3 variants using the membrane filter-trap assay rou-\ntinely used to detect SDS-resistant huntingtin exon 1 fibrils (16, 17, 39).\nUpon incubation of all variants, at a concentration of 30 M, we observe\nthat only pathological length at3(Q64) forms SDS-resistant aggregates\n(Fig. 2A). Further analysis of the aggregation with densitometry reveals\nthat at3(Q64) aggregation follows a classic nucleation-dependent time\ning conditions (19, 29 –31). Therefore, it is possible that the protein\nregion responsible for the formation of ThT-positive aggregates\ninvolves the N-terminal Josephin domain, which is common to all of the\nproteins, although further work is needed to clarify this (29, 33).\nInspection of the kinetics derived from the SDS-insolubility and ThT\nassays (Fig. 2) suggests that pathological length at3(Q64) aggregates in\ntwo stages. The first is a ThT-detectable stage common to all variants,\nand this is followed by a second poly(Q)-length dependent stage in\nwhich classic SDS-insoluble poly(Q) aggregates are formed. A small\ndegree of overlap is seen between the two phases, however the bulk of\nSDS-resistant aggregates do not appear to form until after a significant\nThT increase.\nTherefore, our data indicate that the three variants form different\naggregates based upon the length of their poly-Q tract. If this is the case\nthen the aggregates formed are likely to possess different stabilities. To\nexamine these differences we analyzed the resistance of these end-stage\naggregates to SDS. End-stage aggregates of the three proteins were incu-\nbated with increasing concentrations of SDS to determine any stability\ndifferences (Fig. 3). As can be seen, only at3(Q64) aggregates are stable\nupon incubation in up to 1% (w/v) SDS. The two shorter variants both\nlose the ability to be retained on the 0.2-m filter after incubation in a\nvery low 0.01% (w/v) SDS. This strongly suggests that the aggregates\nformed from expanded ataxin-3 are structurally different to those of the\nnon-expanded variants. Furthermore,\nthis increased stability of\nexpanded ataxin-3 aggregates may have ramifications in the pathogen-\nesis of the disease, as more stable aggregates are likely to be harder to\nbreak down and longer lived.\nQBP1 Inhibits the Second Phase of at3(Q64) Aggregation—The data\npresented above suggest a two-step mechanism of aggregation with only\nthe pathological length poly(Q) tract playing a role in formation of SDS-\nstable aggregates. To confirm this, we have used the QBP1 peptide as a\ntool to explore the role of the poly(Q) tract in defining the at3(Q64)\naggregation pathway. The QBP1 peptide is 11 amino acids in length and\nhas been shown to inhibit poly(Q) aggregation both in vitro and in vivo\n(38, 42). In Fig. 4 we followed the aggregation of 30 M at3(Q64) by the\ncombination of the filter-trap assay and the ThT assay. Upon incubation\nof the QBP1 peptide, at a molar ratio of 4:1 (peptide:protein), we still\nobserve the initial ThT increase (Fig. 4C). However, the second stage of\naggregation is completely eliminated, with no detectable SDS-insoluble\naggregates formed in the presence of QBP1 (Fig. 4, A and B). Taken\ntogether these data indicate that expanded ataxin-3 aggregates through\ncourse, with a defined lag period followed by a rapid growth phase (Fig.\n2B). Even upon incubation times of up to 200 h, at3(QHQ) and at3(Q15)\ndid not form SDS-resistant aggregates, suggesting that under physiolog-\nically relevant conditions ataxin-3 forms SDS-resistant aggregates in a\npoly(Q) length-dependent manner similar to that seen in other poly(Q)\nproteins (15–19).\nIn addition to the filter-trap assay we followed the aggregation reac-\ntion with the fluorescent dye ThT. We analyzed the aggregation prop-\nerties of the three ataxin-3 variants, under identical conditions (Fig. 2C).\nIn contrast to the filter-trap assay we observed that all the variants are\nable to form ThT-positive aggregates. The rate of aggregation was\npoly(Q)-length dependent, with at3(Q64) aggregating at a faster rate\nthan either at3(QHQ) or at3(Q15). However intriguingly, both\nat3(QHQ) and at3(Q15) aggregate with essentially identical kinetics\n(Table 1). The kinetics display time-squared dependence (data not\nshown) suggestive of a nucleation-dependent time course (40, 41). As\nat3(QHQ) contains no poly(Q) tract, it appears that the ataxin-3 protein\ncontains an inherent propensity to form ThT-positive aggregates inde-\npendently of the poly(Q) tract. In a recent study, the Josephin domain of\nataxin-3 has been shown to form amyloid-like aggregates upon thermal\nstress (29). Furthermore, numerous ataxin-3 variants with non-patho-\nlogical length poly(Q) tracts have been shown to form amyloid-like\naggregates under both near-physiological conditions (33) and denatur-\na two-stage pathway, with only the second stage reliant on the presence\nof an expanded glutamine tract. We then went on to further structurally\nand kinetically characterize this pathway using a range of techniques.\nAtaxin-3 Aggregation Monitored by Intrinsic Fluorescence and Size\nExclusion Chromatography—The two-step mechanism of aggregation\nproposed above suggests that the N-terminal domain of ataxin-3, which\nis common to all the proteins, is involved in the first step. To determine\nif this is the case we exploited the fact that ataxin-3 contains three\ntryptophan residues, all of which are located within the Josephin\ndomain, as such they act as a highly sensitive reporter groups for the\ntertiary structure of this domain. We therefore followed the changes in\nintrinsic tryptophan fluorescence during aggregation for all three pro-\nteins. We observe that, upon formation of the final aggregate conforma-\ntion, all three variants adopt a hyper-fluorescent state in which the\nemission maxima has shifted from 333 nm in the native state to 340 nm\nin the aggregate (Fig. 5, A–C). These data suggest that Josephin has\nundergone a conformational change during aggregation and is in a sim-\nilar non-native conformation in the final aggregation states for all three\nvariants. For at3(Q64) there was no difference in emission spectra\nbetween the SDS-soluble and SDS-stable aggregates, suggesting that the\nJosephin domain undergoes no detectable structural rearrangement\nupon the formation of the SDS-stable aggregates. We also used this\nchange in fluorescence to follow the kinetics of aggregation in reference\nto the conformational changes in the Josephin domain (Fig. 5, D–F).\nInterestingly, the fluorescence-derived kinetics correlates well with the\ngain of ThT signal (Table 1). These data further suggest that the Jose-\nphin domain plays a crucial structural role only in the first stage of\nataxin-3 aggregation.\nTo further analyze the aggregation pathway of ataxin-3 we used SEC\nto follow the loss of monomer from solution during aggregation (Fig. 5,\nD–F). For all three variants, monomer loss showed a good correlation\nwith both aggregation monitored by ThT and intrinsic fluorescence\n(Table 1). Our data are consistent with a model of aggregation, in which\nmonomer conversion into the growing fibril is characterized by a struc-\ntural rearrangement of the Josephin domain. During this first stage of\naggregation, the aggregates are soluble, based upon their ability to run\nbefore or within the void volume of the SEC column (data not shown),\nand the lack of any increase in light scatter (data not shown). The second\nstage of aggregation is accompanied by a dramatic loss of solubility of\nthe aggregates as they visibly precipitate from solution. This stage pre-\ndominantly occurs after the loss of monomer from solution, suggesting\nthat SDS-insoluble aggregates are formed through the summation of\nsmaller aggregates, rather than by monomer addition to the end of\ngrowing fibrils. More detailed analysis of these stages of aggregation is\ncurrently underway to further kinetically characterize the stages of\nataxin-3 aggregation.\nAggregate Formation Is Associated with an Increase in -Sheet Sec-\nondary Structure—The changes in secondary structure of the ataxin-3\nvariants during aggregation were monitored using far-UV CD spectros-\ncopy. At 0 h all variants display CD spectra similar to those previously\nreported (19), indicative of a predominantly -helical protein (Fig. 6).\nAfter completion of aggregation at 96 h, both at3(QHQ) (Fig. 6A) and\nat3(Q15) (Fig. 6B) have undergone similar changes, with a gain of signal\nat 220 nm causing a flattening of the CD spectra. These spectral changes\nare indicative of an increase in -sheet content, and a loss of -helix.\nThis is similar to that recently reported for fibril formation of the Jose-\nphin domain and non-pathological lengths of ataxin-3 under thermal\ndenaturation (29, 31).\nThe secondary structural changes of at3(Q64) during the same incu-\nbation are more complex (Fig. 6C). An initial change in secondary struc-\nture, comparable to that of the non-pathological length variants, is seen\nafter 24 h during the first phase of aggregation. Secondary structural\nchanges are also evident upon the formation of SDS-stable aggregates,\nhowever during this phase the CD spectrum undergoes a significant loss\nof signal, thought to be caused by a large increase in light scatter during\nthe second phase of aggregation (data not shown). This increase in light\nscatter unfortunately precludes us from obtaining detailed information\non the secondary structure of end-point at3(Q64) fibrils.\nPoly(Q) Tract Length Affects End-stage Aggregate Morphology—TEM\nhas been used in a range of studies to observe both intermediate assem-\nblies formed prior to fibril formation and end-stage aggregates (43– 45).\nHere, we have undertaken a detailed analysis of the intermediate species\nand fibril morphologies formed during aggregation of all ataxin-3 vari-\nants at 30 M. At 0 h, for all variants, the TEM grid has no prominent\nobservable structures consistent with the ataxin-3 monomer as the pre-\ndominant species (data not shown).\nThe first structures we observe, for all three variants, are a combina-\ntion of small spheroid-like aggregates and short curvilinear oligomeric\naggregates (Fig. 7, A–C). Consistent with the ThT kinetics, these struc-\ntures are formed at a faster rate in at3(Q64) than the other two shorter\nvariants. The diameters of the spheroids vary between 5 and 20 nm,\nwhich are similar to those observed recently with non-expanded\nataxin-3 (33), other poly(Q) proteins/peptides (39, 43, 45), and amyloi-\ndogenic proteins (43, 46 – 48). The oligomeric aggregates also mirror\nthose previously observed for non-expanded ataxin-3 (33), having a\nconsistent diameter 10 –12 nm and a length that increases with time.\nThese short curvilinear aggregates also strongly resemble the protofi-\nbrils observed during amyloid- aggregation (44, 49), -synuclein\naggregation (48, 50), and previous poly(Q) studies (45, 51). As aggrega-\ntion continues these oligomers elongate, up to many hundreds of\nnanometers in length, while their diameter remains the same (Fig. 7,\nD–F). These banded aggregates appear quite flexible and have a very\nsimilar morphology to that of thermally derived Josephin aggregates\n(29). For both at3(QHQ) and at3(Q15) this represents the end-stage\nmorphology of the aggregation pathway with no change in structure\nobserved at later time points (Fig. 7, G and H).\nAt3(Q64) aggregates further into a second type of morphology whose\noccurrence is concurrent with the second phase of aggregation (Fig. 7I).\nThese aggregates are larger and more fibrous than the banded aggre-\ngates formed during the previous stage, resembling those previously\nreported for at3(Q50) (19). Upon close analysis, some individual aggre-\ngates also resemble the beaded morphology previously reported for\nat3(Q78) (28). These fibrils tend to self-associate and form large insol-\nuble aggregates, making their characterization by TEM difficult. These\nlarge filamentous aggregates are usually 50 –100 nm in width and can be\nup to many microns in length. The banded aggregates that represent the\nend-stage morphology of the shorter variants no longer appear to be\npresent.\nCross-seeding Reveals a Model Pathway of Ataxin-3 Aggregation—\nOur data suggest that the conformation of the final aggregates of\nat3(Q15) and at3(QHQ) are structurally different from those of\nat3(Q64). One of the properties of fibrillizing proteins, including the\npoly(Q) family of proteins, is that propagation of the aggregates can be\naccomplished by the presence of “seed,” which catalyzes the aggregation\nof free protein. We next determined whether the different aggregates\nproduced by the different length repeat proteins could catalyze the\naggregation of each other or there is a structural barrier within this\nfamily of aggregates that limits its propagation. Cross-seeding of\neach of the three variants was analyzed initially by ThT assay (Fig. 8).\nAll of the aggregation assays monitored display no observable lag\nphase and follow a linear gradient, indicative of the abolishment of\nthe rate-limiting nucleation stage (Fig. 8, A–C). This confirms that\nataxin-3 aggregation is nucleation-dependent, in line with previous\nstudies of ataxin-3 (33), and kinetically is indicative of amyloid-like\nfibril formation (17). Furthermore, as each of the variants are able to\ncross-seed the aggregation pathways of the other, these data suggest\nthat the aggregation pathways of the three proteins are also struc-\nturally similar, with each variant able to act as a template for the\nelongation of the other variants.\nWe then examined the same cross-seeding assays by the membrane\nfilter-trap assay, which reports the formation of SDS-stable aggregates.\nIn Fig. 8 (D and E), we can see that none of the variants are able to\ncross-seed the shorter variants into forming SDS-stable aggregates. This\nis most likely because the poly(Q) tracts in each of the variants are either\nnon-existent or too short to readily form poly(Q)-dependent aggregates\nover the timeframe analyzed. Only at3(Q64) aggregates can noticeably\naccelerate the formation of poly(Q)-dependent\nfibrils from free\nat3(Q64) (Fig. 8F).\nDISCUSSION\nThe poly(Q) family represents a biomedically important class of pro-\nteins whose aggregation is intrinsically linked to the disease process. A\nnumber of reports have analyzed the aggregation properties of poly(Q)\npeptides (15, 18), fragments of poly(Q) proteins (16, 39, 45), or fusion\nproteins containing poly(Q) tracts (38, 52). However, little is known\nabout the aggregation of intact naturally occurring poly(Q) proteins.\nThis has become increasingly important as numerous studies have\nrecently highlighted the importance of protein context in the aggrega-\ntion of some poly(Q) proteins (34, 35). In this present study, we report\nthe kinetic and morphological changes of both non-expanded and\npathologically expanded ataxin-3 upon aggregation. In taking this\ncomparative approach, we have been able to explore both the role of\nprotein context, and the poly(Q) tract in defining the pathway of\nataxin-3 aggregation.\nThe major findings of this study arise from our ability to directly\ncompare the aggregation of at3(QHQ), at3(Q15), and at3(Q64). Our\ndetailed analysis of ataxin-3 aggregation showed that, independent of\nthe poly-Q tract length, the protein has an inherent propensity to aggre-\ngate. Aggregation of the shorter ataxin-3 variants (at3(QHQ) and\nat3(Q15)) occurred through a nucleation-dependent process that dis-\nplayed simultaneous changes in ThT fluorescence, intrinsic tryptophan\nfluorescence, and SEC. Furthermore, morphological analysis revealed\nthe formation of amyloid-like fibrils, which closely resemble those\nformed by the isolated Josephin domain (29). These data are consistent\nwith previous studies of non-expanded ataxin-3, under physiological\nand denaturing conditions, which have also suggested that this protein\nhas an intrinsic tendency to aggregate (32, 33). The biological relevance\nof this in-built aggregation propensity, displayed by both non-expanded\nataxin-3 and the Josephin domain (29), to the disease process is unclear.\nIt has been suggested that intracellular binding partners may strongly\nlimit the conformational flexibility of ataxin-3 and in turn its suscepti-\nbility to aggregation (29).\nConsistent with the poly(Q) tract length dependence of SCA3 (2)\nat3(Q64) has an increased rate of aggregation. The first stage of\nat3(Q64) aggregation is indistinguishable from that of the shorter vari-\nants. Strikingly, however, only at3(Q64) undergoes a second step, which\nis characterized by the formation of large SDS-insoluble aggregates.\nThese aggregates are dependent upon the presence of an expanded\npoly(Q) tract for their formation. The aggregates are highly stable and\nmorphologically resemble those formed in previous studies of expanded\npoly(Q) proteins and peptides (16, 39, 45, 53).\nOn the basis of our data we propose a minimal two-stage model of\nataxin-3 aggregation, which takes into account both protein context and\npoly(Q) expansion (Fig. 9). The first stage involves a conformational\nchange and self-association in regions other than the poly(Q) tract that\nresults in the formation of SDS-soluble aggregates; the second step that\noccurred solely in the protein with an expanded poly(Q) tract results in\nformation of an SDS-insoluble state. Previous aggregation analysis of\npoly(Q) peptides and huntingtin exon 1 identified a concerted single\nstage of aggregation, which is characterized by the formation of SDS-\ninsoluble aggregates and an increase in -sheet secondary structure\n(15–18, 53). Our data clearly demonstrate that for ataxin-3 this single\nstage model is not sufficient, due to the added complexity provided by\nregions other than the poly(Q) tract.\nThe early aggregate species formed during this first stage of aggrega-\ntion by all variants resemble the morphologies of pre-fibrillar aggregates\nimplicated in the toxicity of numerous neurodegenerative diseases (43–\n45, 49, 54). Similar spheroids and oligomers have previously been shown\nto form from non-expanded ataxin-3 and poly(Q) peptides, with anti-\nbody analysis indicating that such pre-fibrillar aggregates may share\nboth a common structure and toxic mechanism (33, 43, 54). Therefore,\nthe neuronal toxicity observed in SCA3 is potentially caused by the\nability of ataxin-3 to form these toxic structures. In keeping with this,\nrecent studies showed that overexpression of non-expanded poly(Q)\nproteins and other neurodegenerative-associated proteins in animal\nmodels can lead to toxicity (36, 55). Our observations suggest a toxic\nmechanism in agreement with this theory, in which expansion of the\npoly(Q) tract accelerates, but is not structurally involved in, the misfold-\ning of ataxin-3 into toxic intermediate structures. However, in contrast\nwith such a theory overexpression of wild-type ataxin-3 in a variety of\nanimal models has failed to reveal patterns of neural toxicity (56, 57).\nFurthermore, overexpression of wild-type ataxin-3 in a Drosophila\nmodel has recently been shown to be neuroprotective (57). Therefore,\nwhether such a proposed mechanism is relevant to disease-associated\ntoxicity requires further exploration. As such, current attempts are\nunderway to analyze the in vivo relevance of these structures to cellular\ntoxicity in ataxin-3.\nAlternatively, a wealth of data suggests that aggregates formed\nthrough poly(Q) tract interactions, whether they are spheroids, oli-\ngomers, or mature fibrils, are themselves the toxic species (58, 59). For\nexample, aggregates derived from poly(Q) peptides were found to be\ntoxic to mammalian cells when delivered to the nucleus (59). Further-\nmore, a mutagenesis approach has linked aggregation of the poly(Q)\ntract in huntingtin exon 1 to enhanced toxicity in vivo (58). In this\ncurrent study, the observation that only pathological length ataxin-3 is\nable to form poly(Q)-based fibrils is indeed compatible with such a\nmechanism. The increased stability of these poly(Q) aggregates would\nmost likely only serve to enhance their lifetime within the cell, and\ntherefore their chance of causing toxicity.\nIn this study, we have proposed a model of ataxin-3 aggregation in vitro\nthat takes into account both the role of protein context and the poly(Q)\ntract in modulating the aggregation pathway. In doing so, we have shown\nthe importance of taking a comparative approach to understand the com-\nplexity of poly(Q) aggregation. The aim now is to characterize these struc-\ntural changes and determine strategies to prevent them.",
         "",
         "The Two-stage Pathway of Ataxin-3 Fibrillogenesis Involves a Polyglutamine-independent Step*",
         "None"
        ],
        [
         "33",
         "001c4745f180ca22c9ceced3991fe1cc539677bc",
         "β-Catenin is a signaling molecule that promotes cell proliferation by the induction of gene transcription through the activation of T-cell factor (TCF)/lymphoid enhancer factor (LEF) transcription factors. The canonical mechanism of the regulation of β-catenin involves its phosphorylation by casein kinase 1 at the Ser-45 site and by glycogen synthase kinase 3 (GSK3) at the Thr-41, Ser-37, and Ser-33 sites. This phosphorylation targets β-catenin to ubiquitination and degradation by the proteasome system. Mitogenic factors promote β-catenin signaling through the inhibition of GSK3, resulting in reduced β-catenin phosphorylation, its stabilization, and subsequent accumulation in the nucleus, where it stimulates TCF/LEF-dependent gene transcription. In the present study, we have shown that (i) β-catenin can be phosphorylated by protein kinase A (PKA) in vitro and in intact cells at two novel sites, Ser-552 and Ser-675; (ii) phosphorylation by PKA promotes the transcriptional activity (TCF/LEF transactivation) ofβ-catenin; (iii) mutation of Ser-675 attenuates the promoting effect of PKA; (iv) phosphorylation by PKA does not affect the GSK3-dependent phosphorylation ofβ-catenin, its stability, or intracellular localization; and (v) phosphorylation at the Ser-675 site promotes the binding of β-catenin to its transcriptional coactivator, CREB-binding protein. In conclusion, this study identifies a novel, noncanonical mechanism of modulation of β-catenin signaling through direct phosphorylation of β-catenin by PKA, promoting its interaction with CREB-binding protein.",
         "S. Taurin,N. Sandbo,Yimin Qin,D. Browning,N. Dulin",
         "S. Taurin,N. Sandbo,Yimin Qin,D. Browning,N. Dulin",
         "http://www.jbc.org/article/S0021925819563438/pdf",
         "-Catenin is a signaling molecule that promotes cell proliferation by\nthe induction of gene transcription through the activation of T-cell\nfactor (TCF)/lymphoid enhancer factor (LEF) transcription factors.\nThe canonical mechanism of the regulation of -catenin involves its\nphosphorylation by casein kinase 1 at the Ser-45 site and by glycogen\nsynthase kinase 3 (GSK3) at the Thr-41, Ser-37, and Ser-33 sites. This\nphosphorylation targets -catenin to ubiquitination and degradation\nby the proteasome system. Mitogenic factors promote -catenin sig-\nnaling through the inhibition of GSK3, resulting in reduced -catenin\nphosphorylation, its stabilization, and subsequent accumulation in the\nnucleus, where it stimulates TCF/LEF-dependent gene transcription.\nIn the present study, we have shown that (i) -catenin can be phospho-\nrylated by protein kinase A (PKA) in vitro and in intact cells at two\nnovel sites, Ser-552 and Ser-675; (ii) phosphorylation by PKA pro-\nmotes the transcriptional activity (TCF/LEF transactivation) of -cate-\nnin; (iii) mutation of Ser-675 attenuates the promoting effect of PKA;\n(iv) phosphorylation by PKA does not affect the GSK3-dependent\nphosphorylation of -catenin, its stability, or intracellular localization;\nand (v) phosphorylation at the Ser-675 site promotes the binding of\n-catenin to its transcriptional coactivator, CREB-binding protein. In\nconclusion, this study identifies a novel, noncanonical mechanism of\nmodulation of -catenin signaling through direct phosphorylation of\n-catenin by PKA, promoting its interaction with CREB-binding\nprotein.",
         "DISCUSSION\nThe present study describes three major findings. (i) -Catenin can\nbe phosphorylated by PKA in vitro and in intact cells at novel sites\n(Ser-552 and Ser-675) distinct from those phosphorylated by CK1,\nGSK3, or Src kinases. (ii) Phosphorylation of -catenin by PKA at the\nSer-675 site promotes its transcriptional activity by a noncanonical\nmechanism that does not involve the stabilization of -catenin. (iii)\nPhosphorylation of -catenin at the Ser-675 site facilitates the interac-\ntion between -catenin and CBP, which may provide a molecular mech-\nanism by which PKA promotes the transcriptional activity of -catenin.\nThe role of the cAMP/PKA pathway in the modulation of cell growth\nPhosphorylation of -Catenin by PKA\nis highly complex and cell-specific; it stimulates cell growth in many cell\ntypes while inhibiting cell growth in others (27). Moreover, even in the\nsame cell type (vascular smooth muscle cells), PKA can promote or\ninhibit cell proliferation dependent on the agonist that stimulates its\nactivity (16). The present study suggests that, in COS7 and HEK293\ncells, PKA facilitates the signaling of -catenin, which is consistent with\nthe recent report demonstrating that prostaglandin E2, through PKA\nactivation, promotes -catenin signaling in colon cancer cells (28).\nHowever, in the latter report, the promoting effect of PKA is attributed\nto phosphorylation and inactivation of GSK3, whereas our results point\nto the role of a direct phosphorylation of -catenin by PKA at the Ser-\n675 site.\nRegarding the mechanism by which phosphorylation of -catenin by\nPKA promotes its transcriptional activity, we first showed that phos-\nphorylation of -catenin by PKA: (i) does not affect the GSK3-depend-\nent phosphorylation of endogenous or ectopic -catenin at the Ser-33/\nSer-37/Thr-41 sites (Figs. 6C and 11C), (ii) does not protect -catenin\nfrom degradation (Fig. 9A), (iii) does not affect the amount of endoge-\nnous -catenin (Fig. 6D), and (iv) does not affect the nuclear localization\nof ectopic -catenin (Fig. 9B). Thus, our data suggest that PKA pro-\nmotes -catenin signaling by a noncanonical mechanism. In this study,\nwe have proposed one potential mechanism wherein phosphorylation\nof -catenin by PKA at the Ser-675 site promotes the interaction\nbetween -catenin and its co-activator, CBP, whereas S675A mutation\nnearly abolishes this interaction.\nIt has been recently shown that the phosphoserine-binding protein\n14-3-3 interacts with -catenin and potentiates its transcriptional\nactivity by an unknown mechanism (29). Given that both PKA phos-\nphorylation sites on -catenin (RRTS552MGGT and KRLS675VELT)\n(Fig. 1) match the type-2 consensus 14-3-3 binding motif (RX1–2(pS/\npT)X2–3(pS/pT)) (30), we explored whether 14-3-3 interacts with\n-catenin through these sites. However, we have failed to coimmuno-\nprecipitate these two proteins despite multiple efforts using different\nconditions (data not shown). In contrast, a positive control interaction\nbetween the regulator of G protein signaling (RGS3) and 14-3-3 (exper-\niments performed in parallel) was readily detectable, as we have shown\npreviously (31).\nWhile this work was in progress, an independent study by Hino et al.\n(32) has been published showing that PKA can phosphorylate -catenin\nat the Ser-675 site. Consistent with our results, these authors show that\nphosphorylation of -catenin at this site promotes TCF-dependent\ngene transcription, whereas the S675A mutation attenuates this effect.\nHowever, Hino et al. (32) suggest that phosphorylation of -catenin at\nthe Ser-675 site leads to stabilization of -catenin, which is different\nfrom what our data, showing no stabilization and no protection of\n-catenin from degradation, suggest. Although acknowledging that\nsuch a discrepancy could be the result of multiple experimental factors,\nincluding cell-type specificity, cell growth conditions, density, etc., we\npropose here an additional possible mechanism by which PKA pro-\nmotes the transcriptional activity of -catenin, namely by facilitating its\ninteraction with CBP (Fig. 10).\nFinally, we also identified Ser-552 as a site of phosphorylation by PKA,\nwhich was not reported previously (Figs. 3 and 4). Even though its muta-\ntion has a lesser effect on 32P incorporation into -catenin, as does the\nS675A mutation (Figs. 3A and 4A), the Ser-552 site is surely phospho-\nrylated by PKA, because its mutation abolishes the recognition of\n-catenin by antibodies against phosphorylated PKA substrates (data\nnot shown). At this time, we do not know what the functional signifi-\ncance is of -catenin phosphorylation at the Ser-552 site. Few possibil-\nities exist. (i) It has no significance; this can be argued by the notion that\nevery modification of the protein is meaningful. (ii) We have missed the\neffect of the S552A mutation due to an overwhelming overexpression of\nthe mutant; this is unlikely, because under the same transfection condi-\ntions, the effect of the S675A mutation was readily detectable (Figs. 7, 8,\nand 10). (iii) Phosphorylation of -catenin at the Ser-552 site may be\nimportant for another previously unrecognized function of -catenin.\nThese unanswered questions are currently being addressed in our\nlaboratory.",
         "-Catenin plays a dual role in cell function. First, being a major com-\nponent of the cell-cell adherens junctions, -catenin links members of\nthe cadherin family of transmembrane cell-cell adhesion receptors to\nthe actin cytoskeleton (1). Second, -catenin acts as an inducer of gene\nexpression that controls an array of embryonic and adult processes (2,\n3). -Catenin is a key effector of Wnt signaling, a pathway first identified\nin Drosophila (4) and soon found to be highly conserved throughout the\nanimal kingdom and to affect cell growth and differentiation in many\ncell types (5, 6). In this role, -catenin functions as an activator of T-cell\nfactor (TCF)2/lymphoid enhancer factor (LEF) transcription factors to\n* This study was supported by National Institutes of Health Grant HL071755 (to N. O. D.),\nAmerican Heart Association Grant AHA0235405Z (to N. O. D.), and American Heart\nAssociation post-doctoral fellowship awards (to S. T. and N. S.). The costs of publica-\ntion of this article were defrayed in part by the payment of page charges. This article\nmust therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Sec-\ntion 1734 solely to indicate this fact.\nstimulate transcription of a variety of growth-related genes, including\nc-myc (7) and cyclin D1 (8). Mutations of -catenin or of its regulatory\nproteins are frequently found in various types of cancers, and they result\nin the accumulation of -catenin and in the activation of TCF/TCF-de-\npendent gene transcription (9, 10).\nIn quiescent cells, -catenin is maintained in the cytoplasm at low\nlevels. This is provided by its interaction with the scaffolding proteins,\nadenomatous polyposis coli and axin, and protein kinases, casein-kinase\n1(CK1) and glycogen synthase kinase 3 (GSK3), which phosphoryl-\nate -catenin on Ser-45 and then on Ser-33/Ser-37/Thr-41 sites,\nrespectively (11), leading to its ubiquitination and proteasomal degra-\ndation (12, 13). The Wnt and other growth stimuli lead to inhibition of\nGSK3, resulting in a decreased phosphorylation of -catenin at Ser-33/\nSer-37/Thr-41 sites, its stabilization, accumulation and subsequent\ntranslocation to the nucleus (14). In addition to phosphorylation by CK1\nand GSK3 at the N terminus, -catenin can also be phosphorylated at\nthe C terminus on Tyr-654 by Src tyrosine kinases, and this phospho-\nrylation affects the association of -catenin with cadherin, resulting in a\nloss of cell-cell adhesion (15).\nWhile analyzing the primary structure of -catenin, we found two\nputative sites of phosphorylation by PKA that are distinct from those\nphosphorylated by CK1, GSK3, or Src kinases (Fig. 1). In this study, we\nhave shown that (i) -catenin can be phosphorylated by PKA at these\nsites and (ii) phosphorylation at one of these sites promotes the tran-\nscriptional activity of -catenin by facilitating its interaction with a tran-\nscriptional co-activator, CREB-binding protein (CBP).",
         "EXPERIMENTAL PROCEDURES\nCell Culture, DNA Reagents, and Antibodies—COS7 cells and human\nembryonic kidney (HEK)293 cells were grown in Dulbecco’s modified\nEagle’s medium supplemented with 10% fetal bovine serum, 2 mM L-glu-\ntamine, 100 units/ml streptomycin, 250 ng/ml amphotericin B, and 100\nunits/ml penicillin. Twenty-four hours prior to stimulation, the cells\nwere serum-deprived using Dulbecco’s modified Eagle’s medium con-\ntaining 0.1% bovine serum albumin and 2 mM L-glutamine. Transient\ntransfections were performed using Lipofectamine-PLUS reagent\n(Invitrogen) following the manufacturer’s recommendations.\nThe cDNA for human -catenin was kindly provided by Dr. Ste-\nphen W. Byers, Georgetown University, Washington, D. C.). The\n-catenin mutants were generated by site-directed mutagenesis and\nconfirmed by sequencing. The TCF/LEF luciferase reporter constructs\n(TOPflash and FOPflash) were from Upstate Biotechnology. The repli-\ncation-deficient adenovirus (Ad) encoding a complete sequence for the\nPKA inhibitor protein (PKI) gene under the control of the cytomegalo-\nvirus promoter was described previously (16). The cDNA for Myc-\ntagged dominant negative PKA mutant was described previously (17).\nThe cDNA for the PKA catalytic subunit (cPKA) was from Stratagene.\nThe antibodies were from the following sources: anti-PKA substrate\n(PKAS, catalog number 9624), anti-phospho-pS33/pS37/pT41--catenin\nPhosphorylation of -Catenin by PKA\n(catalog number 9561), and anti-phospho-GSK-3/ (catalog number\n9331) were from Cell Signaling Technology; anti--catenin (catalog\nnumber sc-7939), anti-cPKA (catalog number sc-903), anti-CREB-\nbinding protein (CBP, catalog number sc-369), and anti-N-cadherin\n(catalog number sc-7939) were from Santa Cruz Biotechnology; anti-\nGSK-3/ (catalog number 44-610) was from BIOSOURCE Interna-\ntional; anti-FLAG M2 (catalog number F1804) and anti--actin (catalog\nnumber A5441) were from Sigma-Aldrich.\nNonradioactive in Vitro Assay for PKA Activity in Cell Lysates—Fol-\nlowing stimulation with the desired agonists, the cells (grown in 12-well\nplates) were lysed in 0.1 ml/well lysis buffer containing 25 mM HEPES\n(pH 7.5), 0.5% Nonidet P-40, protease inhibitors (1 mg/ml leupeptin, 1\nmg/ml aprotinin, 1 mM phenylmethylsulfonyl fluoride), and phospha-\ntase inhibitors (1 mM NaF, 200 mM Na-orthovanadate). The lysates were\ncleared from insoluble material by centrifugation at 20,000  g for 10\nmin, and 5 l of cleared lysates were subjected to a kinase reaction with\nthe fluorescence-labeled PKA substrate, Kemptide (Promega) following\nthe manufacturer’s protocol. The reaction was stopped by boiling the\nsamples for 10 min. The phosphorylated Kemptide was separated by\n0.8% agarose electrophoresis, and the fluorescence of the phosphoryl-\nated Kemptide was detected and quantified by a Luminescent Image\nAnalyzer LAS-3000 (Fujifilm).\nWestern Blotting—Unless indicated, cells were lysed in radioimmune\nprecipitation assay buffer (containing 25 mM HEPES (pH 7.5), 150 mM\nNaCl, 1% Triton X-100, 0.1% SDS, 2 mM EDTA, 2 mM EGTA, 10%\nglycerol, 1 mM NaF, 200 M sodium-orthovanadate, and protease inhib-\nitors (1 g/ml leupeptin, 1 g/ml aprotinin, 1 mM phenylmethylsulfonyl\nfluoride). The lysates were cleared from insoluble material by centrifu-\ngation at 20,000  g for 10 min, boiled in Laemmli buffer, subjected to\npolyacrylamide gel electrophoresis, and analyzed by Western blotting\nwith the desired primary antibodies, followed by horseradish peroxi-\ndase-conjugated secondary antibodies (Calbiochem), and developed by\nenhanced chemiluminescence reaction (Pierce). The digital chemilumi-\nnescence images were taken by a Luminescent Image Analyzer LAS-\n3000 (Fujifilm).\nImmunoprecipitation—Generally, the lysates from 2  106 trans-\nfected cells were used for one immunoprecipitation. The cells were\nwashed twice with ice-cold PBS and lysed in the extraction buffer (300\nl/106 cells) containing 150 mM NaCl, 25 mM HEPES (pH 7.4), 0.5%\nNonidet P-40, 2 mM EDTA, 2 mM EGTA, 100 M orthovanadate, and\nprotease inhibitors (200 M phenylmethylsulfonyl fluoride, 2 g/ml leu-\npeptin, and 2 g/ml aprotinin). The lysates were cleared by centrifuga-\ntion at 20,000  g for 5 min. The immunoprecipitation of endogenous\n-catenin was performed by overnight incubation of the cleared lysates\nwith agarose-conjugated goat anti--catenin antibodies (Santa Cruz\nBiotechnology, catalog number sc-1496), followed by three washes with\n1 ml of lysis buffer. Immunoprecipitation of FLAG-tagged -catenin\nproteins was performed using agarose-conjugated mouse anti-FLAG\nantibodies (Sigma-Aldrich, catalog number A2220). Immunoprecipita-\ntion of CBP was performed using anti-CBP antibodies followed by incu-\nbation with protein A/protein G-conjugated agarose beads (The\nimmune complexes were boiled in Laemmli buffer and subjected to\nWestern blotting with the desired antibodies.\nIn Vitro 32P Labeling of -Catenin—FLAG-tagged wild-type (WT)\n-catenin or its PKA phosphorylation mutants were expressed in COS7\ncells and purified using anti-FLAG-agarose beads. The beads were\nwashed once with PKA assay buffer (20 mM Tris-HCl (pH 7.4), 100 mM\nNaCl, 10 mM MgCl2, 1 mM ATP) and then incubated with 45 l of PKA\nassay buffer supplemented with 0.1 mM dithiothreitol, 5 units of purified\nprotein kinase A catalytic subunit (Promega, V5340), and 5 Ci of\n[-32P]ATP at 30 °C for 30 min. The reaction was stopped by the addi-\ntion of 15 l of 4 Laemmli buffer and heating for 5 min at 95 °C.\nProteins were subjected to SDS-PAGE, followed by autoradiography\nWestern blotting with the desired antibodies.\n32P Labeling of -Catenin in Intact Cells—COS7 cells were trans-\nfected with cDNAs for N-terminally truncated (N100) FLAG-tagged\nWT--catenin or its PKA phosphorylation mutants and serum-starved\nfor 24 h. The cells were incubated with 0.25 mCi/ml [32P]orthophos-\nphate in a phosphate-free and serum-free medium, followed by three\nwashes and stimulation with 10 M forskolin (FSK) for 10 min. The cells\nwere then lysed, and the cell lysates were subjected to immunoprecipi-\ntation with FLAG antibodies as described above, followed by electro-\nphoresis, autoradiography, or Western blotting with FLAG antibodies.\nTCF/LEF-Luciferase Reporter Assay—Cells grown on 24-well plates\nwere transfected in triplicates or quadruplicates with cDNAs (20\nng/well) for TCF-luciferase reporter (TOPflash) or its mutated control\nreporter (FOPflash) along with the Renilla reporter driven by the thy-\nmidine kinase promoter (tk-RL) and with the other plasmids of interest.\nThe total amount of DNA was equalized in all of the samples by using\nthe corresponding empty vector. Following stimulation with the desired\nagonists, the cells were lysed and the luciferase activity was measured\nand normalized to the corresponding Renilla activity using the dual-\nluciferase assay kit (Promega). The normalized FOPflash values were\nsubtracted from the corresponding TOPflash values and expressed as\nmean  S.D.\nIndirect\nImmunofluorescence Microscopy—Immunofluorescence\nwas performed as described previously (18). Cells grown on glass cham-\nber slides and transfected with FLAG-tagged -catenin cDNA were\nstimulated with or without 10 M forskolin for 15 min, washed twice\nwith ice-cold PBS, fixed in 4% paraformaldehyde in PBS for 15 min at\nroom temperature, washed again with PBS, and permeabilized in 0.2%\nTriton-X100 in PBS for 5 min, followed by incubation with 1% bovine\nserum albumin in PBS for 1 h. The cells were then incubated with\nmonoclonal anti-FLAG antibodies (5 g/ml in PBS/bovine serum albu-\nmin, as above) overnight at 4 °C for 1 h and washed four times with PBS,\nfollowed by incubation with fluorescein-conjugated goat anti-mouse\nIgG (10 g/ml in PBS/bovine serum albumin, as above) for 1 h atroom\ntemperature. The slides were additionally washed four times with PBS, and\nthe coverslips were mounted using Gel/Mount aqueous mounting medium\n(Fisher, Pittsburgh, PA). The confocal images (sections near the bottom of\nthe cell) were taken under a confocal fluorescent microscope.\nStatistics—The Bonferroni t test for multiple comparisons was uti-\nlized to test for statistically significant differences. The appropriate level\nof significance was determined for each analysis, according to the num-\nber of simultaneous comparisons tested against the reference group (p\nvalue cutoffs are indicated in the figure legends).\nRESULTS\nPhosphorylation of -Catenin by PKA—We first tested the possibility\nthat the endogenous -catenin can be phosphorylated by PKA using\nPKAS antibodies directed against the phosphorylated PKA substrates\nPhosphorylation of -Catenin by PKA",
         "EXPERIMENTAL PROCEDURES\nCell Culture, DNA Reagents, and Antibodies—COS7 cells and human\nembryonic kidney (HEK)293 cells were grown in Dulbecco’s modified\nEagle’s medium supplemented with 10% fetal bovine serum, 2 mM L-glu-\ntamine, 100 units/ml streptomycin, 250 ng/ml amphotericin B, and 100\nunits/ml penicillin. Twenty-four hours prior to stimulation, the cells\nwere serum-deprived using Dulbecco’s modified Eagle’s medium con-\ntaining 0.1% bovine serum albumin and 2 mM L-glutamine. Transient\ntransfections were performed using Lipofectamine-PLUS reagent\n(Invitrogen) following the manufacturer’s recommendations.\nThe cDNA for human -catenin was kindly provided by Dr. Ste-\nphen W. Byers, Georgetown University, Washington, D. C.). The\n-catenin mutants were generated by site-directed mutagenesis and\nconfirmed by sequencing. The TCF/LEF luciferase reporter constructs\n(TOPflash and FOPflash) were from Upstate Biotechnology. The repli-\ncation-deficient adenovirus (Ad) encoding a complete sequence for the\nPKA inhibitor protein (PKI) gene under the control of the cytomegalo-\nvirus promoter was described previously (16). The cDNA for Myc-\ntagged dominant negative PKA mutant was described previously (17).\nThe cDNA for the PKA catalytic subunit (cPKA) was from Stratagene.\nThe antibodies were from the following sources: anti-PKA substrate\n(PKAS, catalog number 9624), anti-phospho-pS33/pS37/pT41--catenin\nPhosphorylation of -Catenin by PKA\n(catalog number 9561), and anti-phospho-GSK-3/ (catalog number\n9331) were from Cell Signaling Technology; anti--catenin (catalog\nnumber sc-7939), anti-cPKA (catalog number sc-903), anti-CREB-\nbinding protein (CBP, catalog number sc-369), and anti-N-cadherin\n(catalog number sc-7939) were from Santa Cruz Biotechnology; anti-\nGSK-3/ (catalog number 44-610) was from BIOSOURCE Interna-\ntional; anti-FLAG M2 (catalog number F1804) and anti--actin (catalog\nnumber A5441) were from Sigma-Aldrich.\nNonradioactive in Vitro Assay for PKA Activity in Cell Lysates—Fol-\nlowing stimulation with the desired agonists, the cells (grown in 12-well\nplates) were lysed in 0.1 ml/well lysis buffer containing 25 mM HEPES\n(pH 7.5), 0.5% Nonidet P-40, protease inhibitors (1 mg/ml leupeptin, 1\nmg/ml aprotinin, 1 mM phenylmethylsulfonyl fluoride), and phospha-\ntase inhibitors (1 mM NaF, 200 mM Na-orthovanadate). The lysates were\ncleared from insoluble material by centrifugation at 20,000  g for 10\nmin, and 5 l of cleared lysates were subjected to a kinase reaction with\nthe fluorescence-labeled PKA substrate, Kemptide (Promega) following\nthe manufacturer’s protocol. The reaction was stopped by boiling the\nsamples for 10 min. The phosphorylated Kemptide was separated by\n0.8% agarose electrophoresis, and the fluorescence of the phosphoryl-\nated Kemptide was detected and quantified by a Luminescent Image\nAnalyzer LAS-3000 (Fujifilm).\nWestern Blotting—Unless indicated, cells were lysed in radioimmune\nprecipitation assay buffer (containing 25 mM HEPES (pH 7.5), 150 mM\nNaCl, 1% Triton X-100, 0.1% SDS, 2 mM EDTA, 2 mM EGTA, 10%\nglycerol, 1 mM NaF, 200 M sodium-orthovanadate, and protease inhib-\nitors (1 g/ml leupeptin, 1 g/ml aprotinin, 1 mM phenylmethylsulfonyl\nfluoride). The lysates were cleared from insoluble material by centrifu-\ngation at 20,000  g for 10 min, boiled in Laemmli buffer, subjected to\npolyacrylamide gel electrophoresis, and analyzed by Western blotting\nwith the desired primary antibodies, followed by horseradish peroxi-\ndase-conjugated secondary antibodies (Calbiochem), and developed by\nenhanced chemiluminescence reaction (Pierce). The digital chemilumi-\nnescence images were taken by a Luminescent Image Analyzer LAS-\n3000 (Fujifilm).\nImmunoprecipitation—Generally, the lysates from 2  106 trans-\nfected cells were used for one immunoprecipitation. The cells were\nwashed twice with ice-cold PBS and lysed in the extraction buffer (300\nl/106 cells) containing 150 mM NaCl, 25 mM HEPES (pH 7.4), 0.5%\nNonidet P-40, 2 mM EDTA, 2 mM EGTA, 100 M orthovanadate, and\nprotease inhibitors (200 M phenylmethylsulfonyl fluoride, 2 g/ml leu-\npeptin, and 2 g/ml aprotinin). The lysates were cleared by centrifuga-\ntion at 20,000  g for 5 min. The immunoprecipitation of endogenous\n-catenin was performed by overnight incubation of the cleared lysates\nwith agarose-conjugated goat anti--catenin antibodies (Santa Cruz\nBiotechnology, catalog number sc-1496), followed by three washes with\n1 ml of lysis buffer. Immunoprecipitation of FLAG-tagged -catenin\nproteins was performed using agarose-conjugated mouse anti-FLAG\nantibodies (Sigma-Aldrich, catalog number A2220). Immunoprecipita-\ntion of CBP was performed using anti-CBP antibodies followed by incu-\nbation with protein A/protein G-conjugated agarose beads (The\nimmune complexes were boiled in Laemmli buffer and subjected to\nWestern blotting with the desired antibodies.\nIn Vitro 32P Labeling of -Catenin—FLAG-tagged wild-type (WT)\n-catenin or its PKA phosphorylation mutants were expressed in COS7\ncells and purified using anti-FLAG-agarose beads. The beads were\nwashed once with PKA assay buffer (20 mM Tris-HCl (pH 7.4), 100 mM\nNaCl, 10 mM MgCl2, 1 mM ATP) and then incubated with 45 l of PKA\nassay buffer supplemented with 0.1 mM dithiothreitol, 5 units of purified\nprotein kinase A catalytic subunit (Promega, V5340), and 5 Ci of\n[-32P]ATP at 30 °C for 30 min. The reaction was stopped by the addi-\ntion of 15 l of 4 Laemmli buffer and heating for 5 min at 95 °C.\nProteins were subjected to SDS-PAGE, followed by autoradiography\nWestern blotting with the desired antibodies.\n32P Labeling of -Catenin in Intact Cells—COS7 cells were trans-\nfected with cDNAs for N-terminally truncated (N100) FLAG-tagged\nWT--catenin or its PKA phosphorylation mutants and serum-starved\nfor 24 h. The cells were incubated with 0.25 mCi/ml [32P]orthophos-\nphate in a phosphate-free and serum-free medium, followed by three\nwashes and stimulation with 10 M forskolin (FSK) for 10 min. The cells\nwere then lysed, and the cell lysates were subjected to immunoprecipi-\ntation with FLAG antibodies as described above, followed by electro-\nphoresis, autoradiography, or Western blotting with FLAG antibodies.\nTCF/LEF-Luciferase Reporter Assay—Cells grown on 24-well plates\nwere transfected in triplicates or quadruplicates with cDNAs (20\nng/well) for TCF-luciferase reporter (TOPflash) or its mutated control\nreporter (FOPflash) along with the Renilla reporter driven by the thy-\nmidine kinase promoter (tk-RL) and with the other plasmids of interest.\nThe total amount of DNA was equalized in all of the samples by using\nthe corresponding empty vector. Following stimulation with the desired\nagonists, the cells were lysed and the luciferase activity was measured\nand normalized to the corresponding Renilla activity using the dual-\nluciferase assay kit (Promega). The normalized FOPflash values were\nsubtracted from the corresponding TOPflash values and expressed as\nmean  S.D.\nIndirect\nImmunofluorescence Microscopy—Immunofluorescence\nwas performed as described previously (18). Cells grown on glass cham-\nber slides and transfected with FLAG-tagged -catenin cDNA were\nstimulated with or without 10 M forskolin for 15 min, washed twice\nwith ice-cold PBS, fixed in 4% paraformaldehyde in PBS for 15 min at\nroom temperature, washed again with PBS, and permeabilized in 0.2%\nTriton-X100 in PBS for 5 min, followed by incubation with 1% bovine\nserum albumin in PBS for 1 h. The cells were then incubated with\nmonoclonal anti-FLAG antibodies (5 g/ml in PBS/bovine serum albu-\nmin, as above) overnight at 4 °C for 1 h and washed four times with PBS,\nfollowed by incubation with fluorescein-conjugated goat anti-mouse\nIgG (10 g/ml in PBS/bovine serum albumin, as above) for 1 h atroom\ntemperature. The slides were additionally washed four times with PBS, and\nthe coverslips were mounted using Gel/Mount aqueous mounting medium\n(Fisher, Pittsburgh, PA). The confocal images (sections near the bottom of\nthe cell) were taken under a confocal fluorescent microscope.\nStatistics—The Bonferroni t test for multiple comparisons was uti-\nlized to test for statistically significant differences. The appropriate level\nof significance was determined for each analysis, according to the num-\nber of simultaneous comparisons tested against the reference group (p\nvalue cutoffs are indicated in the figure legends).\nRESULTS\nPhosphorylation of -Catenin by PKA—We first tested the possibility\nthat the endogenous -catenin can be phosphorylated by PKA using\nPKAS antibodies directed against the phosphorylated PKA substrates\nPhosphorylation of -Catenin by PKA\n((RXX(pS/pT), Cell Signaling Technology). Fig. 2A shows stimulation of\nPKA by a cAMP-rising agent, FSK, and inhibition by adenovirus (Ad)-\nmediated transduction of the PKA inhibitor PKI (AdPKI). The specific-\nity of AdPKI was previously confirmed by its inability to affect mitogen-\nactivated protein kinase activation induced by various agonists (16).\nWestern blotting of cell lysates with PKAS antibodies revealed a PKA-\ndependent phosphorylation of a number of proteins, and this PKAS\nrecognition was attenuated by AdPKI transduction (Fig. 2B). When\n-catenin was immunoprecipitated from cell lysates followed by West-\nern blotting with PKAS antibodies, a single band that co-migrated with\n-catenin was detected only after FSK stimulation (Fig. 2C). Overex-\npression of PKI prevented the recognition of immunoprecipitated\n-catenin by PKAS antibodies. The equal amount of -catenin in the\nimmunoprecipitation samples was confirmed by probing with -cate-\nnin antibodies (Fig. 2D). These data suggest that endogenous -catenin\ncan be phosphorylated in a PKA-dependent manner in response to FSK.\nWe identified two potential PKA phosphorylation sites on -catenin\n(serine 552, serine 675; Fig. 1) and generated alanine substitutions to\nexamine the function. We first examined whether PKA could phospho-\nrylate -catenin and the above mutants in vitro and found that PKA\nreadily phosphorylated the WT -catenin (Fig. 3). The S552A or S675A\nmutation partially attenuated this effect, whereas the double mutation\n(S552A/S675A) completely abolished the phosphorylation of -catenin\nby PKA (Fig. 3). We then examined whether FSK could stimulate PKA-\ndependent phosphorylation of -catenin in intact cells. To dissect the\nCK1- and GSK3-dependent phosphorylations, we generated the FLAG-\ntagged truncated versions of -catenin and of its PKA phosphorylation\nmutants by deleting the N-terminal (N100) region containing the\nphosphorylation sites for CK1 or GSK3. We then overexpressed these\ntruncated -catenin proteins in COS7 cells and examined their phos-\nphorylation in response to FSK by immunoprecipitation from 32P-la-\nbeled cells, followed by autoradiography. As shown in Fig. 4, FSK stim-\nulated phosphorylation of N100-WT--catenin; this phosphorylation\nwas partially attenuated by the S552A or S675A mutations and was\nabolished by mutation of both sites. The smear (Fig. 4A, above the\n-catenin band), nonspecifically precipitated by FLAG antibodies,\nserved as an internal control for the presence of 32P labeling in all of the\noriginal lysates. Together, these data convincingly demonstrate that\n-catenin can be phosphorylated by PKA at Ser-552 and Ser-675 sites\nboth in vitro and in intact cells.\nPhosphorylation of -Catenin Promotes Its Transcriptional Activity—\nTo understand the functional significance of -catenin phosphorylation\nby PKA, we first examined how the stimulation of cells with FSK affects\nthe signaling of endogenous -catenin, using TCF/LEF-luciferase\nreporter (TOPflash/FOPflash). As shown in Fig. 5A, FSK stimulated a\nsignificant induction of TCF/LEF reporter (TOPflash) without affecting\nthe control (mutant) reporter activity (FOPflash). Overexpression of\ndominant negative PKA mutant (dnPKA) abolished FSK-induced TCF/\nLEF activity, indicating the role of PKA for this effect (Fig. 5B). The\nefficiency and specificity of dnPKA in inhibiting PKA activity in intact\ncells was previously confirmed (17).\nIt has been suggested that PKA can phosphorylate and inactivate\nGSK3 resulting in decreased phosphorylation and increased stability of\n-catenin (19 –21). We examined whether this could explain the effect\nof FSK on TCF/LEF induction in our experiments. In COS7 cells, FSK\nstimulated a rapid and prolonged phosphorylation of GSK3 (a major\nGSK3 isoform in these cells), as assessed by phosphospecific antibodies\n(Fig. 6A). Nevertheless, the total levels of -catenin, as well as its GSK3-\ndependent phosphorylation, were not significantly affected by FSK, as\nassessed by Western blotting with antibodies against total -catenin\n(Fig. 6D), or with phospho-(pS33/pS37/pT41)-specific -catenin anti-\nPhosphorylation of -Catenin by PKA\nbodies (Fig. 6C). Together, these data suggest that, in COS7 cells, PKA\npromotes the signaling of endogenous -catenin without affecting the\nlevels of its expression.\nWe then examined how the PKA-dependent phosphorylation of\nectopically expressed -catenin affects its transcriptional activity. Over-\nexpression of FLAG-tagged WT--catenin on its own in COS7 cells\nresulted in a 5–7-fold activation of TCF/LEF-reporter, as expected (not\nshown), but this effect was further potentiated by FSK treatment (Fig.\n7A). Mutation of -catenin at S552A did not change the TCF/LEF-\nreporter activity, but the S675A or the S552A/S675A mutations\ndecreased the effect of FSK by half. Western blotting of cell lysates with\nFLAG antibodies revealed that FSK facilitated the expression of ectopic\n-catenin (Fig. 7B), which was probably the result of an increased tran-\nscription by the cytomegalovirus promoter (driving transcription of\n-catenin and its mutants). Likewise, FSK treatment also increased the\nexpression of cytomegalovirus-driven -galactosidase or green fluores-\ncent protein (data not shown). This may partially, but not entirely, con-\ntribute to a FSK-induced TCF/LEF activity in -catenin-transfected\ncells (note that FSK did not affect the expression of endogenous -cate-\nnin (Fig. 6D); whereas it still stimulated the TCF/LEF activity in non-\ntransfected cells (Fig. 5A)). Importantly, the expression of both WT and\nall three PKA phosphorylation mutants of -catenin was equally\nincreased upon FSK stimulation, whereas the FSK-induced TCF/LEF\nreporter activity was significantly reduced in cells transfected with\nS675A or S552A/S675A mutants. This suggests that the S675A muta-\ntion attenuates the ability of FSK to promote the -catenin-mediated\nTCF/LEF activity.\nTo test whether the results obtained in FSK-stimulated COS7 cells\ncould be reproduced in other cells and by other means of PKA stimula-\ntion, we overexpressed the PKA-catalytic subunit (cPKA) in HEK293\ncells. As shown in Fig. 8A, cPKA overexpression in HEK293 cells pro-\nmoted the signaling of WT--catenin or its S552A mutant to a much\ngreater extent than that of S675A or S552A/S675A mutants. Similar to\nour observations using FSK-stimulated COS7 cells (Fig. 7), cPKA over-\nexpression in HEK293 cells resulted in increased levels of -catenin and\nits mutants but all to the same extent (Fig. 8C). Thus, the very similar\nresults obtained by two different approaches strongly suggest that (i)\nPKA promotes the TCF/LEF activity induced by -catenin, at least in\npart, through phosphorylation of -catenin at the Ser-675 site and (ii)\nthis effect of PKA does not seem to involve stabilization of -catenin.\nTo further address the latter notion, we examined the rate of -cate-\nnin degradation following the inhibition of protein synthesis by cyclo-\nheximide. As shown in Fig. 9A, a 3-h treatment of cells with cyclohexi-\nmide resulted in a 60% decrease of exogenous -catenin but not of\n-actin. Importantly, (i) treatment of cells with FSK did not protect\n-catenin from degradation, and (ii) there was no significant difference\nbetween WT--catenin and its PKA phosphorylation mutants in the\ndegradation rate. We also examined how PKA affects the intracellular\nlocalization of -catenin. As shown on Fig. 9B, overexpressed -catenin\nlocalized mainly to the nucleus as expected. However, forskolin stimu-\nPhosphorylation of -Catenin by PKA\nlation for 15 min (which results in maximal phosphorylation at both the\nSer-552 and Ser-675 sites) did not affect the localization of WT--\ncatenin (Fig. 9B). Furthermore, the alanine substitution of either the\nSer-552 or Ser-675 sites also had no obvious effect on -catenin local-\nization under basal or forskolin-stimulated conditions; it still remained\nlargely in the nucleus (data not shown). Together, these data suggest\nthat PKA promotes -catenin signaling without affecting its stability or\nintracellular localization.\nGiven that the Ser-675 phosphorylation site (responsible for the pro-\nmoting effect of PKA) is located within the transactivation domain of\n-catenin, we then hypothesized that phosphorylation by PKA pro-\nmotes -catenin signaling at the level of its transcriptional activity,\nthrough modulation of its interaction with co-activators, such as CBP.\nCBP is an acetyltransferase that promotes the activity of several tran-\nscription factors (i.e. CREB) by their acetylation. CBP acetylates -cate-\nnin and increases its transcriptional activity (22–24). Therefore, we\nexamined whether phosphorylation of -catenin by PKA is important\nfor -catenin-CBP interaction by co-immunoprecipitation. As shown\non Fig. 10, WT--catenin and the S552A mutant bound specifically to\nCBP, and their binding was significantly increased by 50% after FSK\ntreatment. In contrast, the S675A mutation resulted in a dramatic\nreduction of -catenin binding to CBP, and, importantly, FSK no longer\npromoted this binding. Note that -catenin is partially phosphorylated\nat the Ser-675 site under basal conditions (as the S675A mutation abol-\nishes this basal phosphorylation) (Fig. 4C), which may explain an appre-\nciable binding of -catenin to CBP under control conditions (Fig. 10, no\nFSK ()). Together, these data suggest that phosphorylation of -cate-\nnin by PKA at the Ser-675 site is important for its interaction with CBP,\nwhich may explain how PKA promotes transcriptional activity of\n-catenin.\nFinally, we sought to examine whether the phosphorylation of\n-catenin by PKA affects its interaction with cadherin, for the following\nreasons: (i) the Ser-552 phosphorylation site does not seem important\nfor modulation of -catenin transcriptional activity, suggesting that\nphosphorylation at this site may modulate the other function of -cate-\nnin at cell-cell adherens junctions, (ii) the Ser-552 phosphorylation site\nis located within the 10th armadillo repeat of -catenin that may be\nimportant for interaction with cadherin (25), and (iii) the -catenin/\ncadherin interaction was reported to be regulated by PKA, albeit\nthrough phosphorylation of cadherin (26). As shown in Fig. 11A, endog-\nenous N-cadherin readily and specifically co-immunoprecipitated with\nFLAG-WT--catenin. However, stimulation of cells with FSK had no\neffect on the interaction of N-cadherin with either WT--catenin, or\nwith its PKA phosphorylation mutants. Thus, at least as assessed by\nco-immunoprecipitation approach, phosphorylation of -catenin by\nPKA does not seem to affect its interaction with N-cadherin. In addi-\ntion, Fig. 11C also shows that GSK3-dependent phosphorylation (at\nSer-33/Ser-37/Thr-41 sites) of ectopic -catenin is not affected by PKA-\ndependent phosphorylation of -catenin (at Ser-552 or Ser-675 sites),\nwhich is consistent with the results on the endogenous -catenin phos-\nphorylation by GSK3 (Fig. 6C).",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "-Catenin is a signaling molecule that promotes cell proliferation by\nthe induction of gene transcription through the activation of T-cell\nfactor (TCF)/lymphoid enhancer factor (LEF) transcription factors.\nThe canonical mechanism of the regulation of -catenin involves its\nphosphorylation by casein kinase 1 at the Ser-45 site and by glycogen\nsynthase kinase 3 (GSK3) at the Thr-41, Ser-37, and Ser-33 sites. This\nphosphorylation targets -catenin to ubiquitination and degradation\nby the proteasome system. Mitogenic factors promote -catenin sig-\nnaling through the inhibition of GSK3, resulting in reduced -catenin\nphosphorylation, its stabilization, and subsequent accumulation in the\nnucleus, where it stimulates TCF/LEF-dependent gene transcription.\nIn the present study, we have shown that (i) -catenin can be phospho-\nrylated by protein kinase A (PKA) in vitro and in intact cells at two\nnovel sites, Ser-552 and Ser-675; (ii) phosphorylation by PKA pro-\nmotes the transcriptional activity (TCF/LEF transactivation) of -cate-\nnin; (iii) mutation of Ser-675 attenuates the promoting effect of PKA;\n(iv) phosphorylation by PKA does not affect the GSK3-dependent\nphosphorylation of -catenin, its stability, or intracellular localization;\nand (v) phosphorylation at the Ser-675 site promotes the binding of\n-catenin to its transcriptional coactivator, CREB-binding protein. In\nconclusion, this study identifies a novel, noncanonical mechanism of\nmodulation of -catenin signaling through direct phosphorylation of\n-catenin by PKA, promoting its interaction with CREB-binding\nprotein.\n-Catenin plays a dual role in cell function. First, being a major com-\nponent of the cell-cell adherens junctions, -catenin links members of\nthe cadherin family of transmembrane cell-cell adhesion receptors to\nthe actin cytoskeleton (1). Second, -catenin acts as an inducer of gene\nexpression that controls an array of embryonic and adult processes (2,\n3). -Catenin is a key effector of Wnt signaling, a pathway first identified\nin Drosophila (4) and soon found to be highly conserved throughout the\nanimal kingdom and to affect cell growth and differentiation in many\ncell types (5, 6). In this role, -catenin functions as an activator of T-cell\nfactor (TCF)2/lymphoid enhancer factor (LEF) transcription factors to\n* This study was supported by National Institutes of Health Grant HL071755 (to N. O. D.),\nAmerican Heart Association Grant AHA0235405Z (to N. O. D.), and American Heart\nAssociation post-doctoral fellowship awards (to S. T. and N. S.). The costs of publica-\ntion of this article were defrayed in part by the payment of page charges. This article\nmust therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Sec-\ntion 1734 solely to indicate this fact.\nstimulate transcription of a variety of growth-related genes, including\nc-myc (7) and cyclin D1 (8). Mutations of -catenin or of its regulatory\nproteins are frequently found in various types of cancers, and they result\nin the accumulation of -catenin and in the activation of TCF/TCF-de-\npendent gene transcription (9, 10).\nIn quiescent cells, -catenin is maintained in the cytoplasm at low\nlevels. This is provided by its interaction with the scaffolding proteins,\nadenomatous polyposis coli and axin, and protein kinases, casein-kinase\n1(CK1) and glycogen synthase kinase 3 (GSK3), which phosphoryl-\nate -catenin on Ser-45 and then on Ser-33/Ser-37/Thr-41 sites,\nrespectively (11), leading to its ubiquitination and proteasomal degra-\ndation (12, 13). The Wnt and other growth stimuli lead to inhibition of\nGSK3, resulting in a decreased phosphorylation of -catenin at Ser-33/\nSer-37/Thr-41 sites, its stabilization, accumulation and subsequent\ntranslocation to the nucleus (14). In addition to phosphorylation by CK1\nand GSK3 at the N terminus, -catenin can also be phosphorylated at\nthe C terminus on Tyr-654 by Src tyrosine kinases, and this phospho-\nrylation affects the association of -catenin with cadherin, resulting in a\nloss of cell-cell adhesion (15).\nWhile analyzing the primary structure of -catenin, we found two\nputative sites of phosphorylation by PKA that are distinct from those\nphosphorylated by CK1, GSK3, or Src kinases (Fig. 1). In this study, we\nhave shown that (i) -catenin can be phosphorylated by PKA at these\nsites and (ii) phosphorylation at one of these sites promotes the tran-\nscriptional activity of -catenin by facilitating its interaction with a tran-\nscriptional co-activator, CREB-binding protein (CBP).\nEXPERIMENTAL PROCEDURES\nCell Culture, DNA Reagents, and Antibodies—COS7 cells and human\nembryonic kidney (HEK)293 cells were grown in Dulbecco’s modified\nEagle’s medium supplemented with 10% fetal bovine serum, 2 mM L-glu-\ntamine, 100 units/ml streptomycin, 250 ng/ml amphotericin B, and 100\nunits/ml penicillin. Twenty-four hours prior to stimulation, the cells\nwere serum-deprived using Dulbecco’s modified Eagle’s medium con-\ntaining 0.1% bovine serum albumin and 2 mM L-glutamine. Transient\ntransfections were performed using Lipofectamine-PLUS reagent\n(Invitrogen) following the manufacturer’s recommendations.\nThe cDNA for human -catenin was kindly provided by Dr. Ste-\nphen W. Byers, Georgetown University, Washington, D. C.). The\n-catenin mutants were generated by site-directed mutagenesis and\nconfirmed by sequencing. The TCF/LEF luciferase reporter constructs\n(TOPflash and FOPflash) were from Upstate Biotechnology. The repli-\ncation-deficient adenovirus (Ad) encoding a complete sequence for the\nPKA inhibitor protein (PKI) gene under the control of the cytomegalo-\nvirus promoter was described previously (16). The cDNA for Myc-\ntagged dominant negative PKA mutant was described previously (17).\nThe cDNA for the PKA catalytic subunit (cPKA) was from Stratagene.\nThe antibodies were from the following sources: anti-PKA substrate\n(PKAS, catalog number 9624), anti-phospho-pS33/pS37/pT41--catenin\nPhosphorylation of -Catenin by PKA\n(catalog number 9561), and anti-phospho-GSK-3/ (catalog number\n9331) were from Cell Signaling Technology; anti--catenin (catalog\nnumber sc-7939), anti-cPKA (catalog number sc-903), anti-CREB-\nbinding protein (CBP, catalog number sc-369), and anti-N-cadherin\n(catalog number sc-7939) were from Santa Cruz Biotechnology; anti-\nGSK-3/ (catalog number 44-610) was from BIOSOURCE Interna-\ntional; anti-FLAG M2 (catalog number F1804) and anti--actin (catalog\nnumber A5441) were from Sigma-Aldrich.\nNonradioactive in Vitro Assay for PKA Activity in Cell Lysates—Fol-\nlowing stimulation with the desired agonists, the cells (grown in 12-well\nplates) were lysed in 0.1 ml/well lysis buffer containing 25 mM HEPES\n(pH 7.5), 0.5% Nonidet P-40, protease inhibitors (1 mg/ml leupeptin, 1\nmg/ml aprotinin, 1 mM phenylmethylsulfonyl fluoride), and phospha-\ntase inhibitors (1 mM NaF, 200 mM Na-orthovanadate). The lysates were\ncleared from insoluble material by centrifugation at 20,000  g for 10\nmin, and 5 l of cleared lysates were subjected to a kinase reaction with\nthe fluorescence-labeled PKA substrate, Kemptide (Promega) following\nthe manufacturer’s protocol. The reaction was stopped by boiling the\nsamples for 10 min. The phosphorylated Kemptide was separated by\n0.8% agarose electrophoresis, and the fluorescence of the phosphoryl-\nated Kemptide was detected and quantified by a Luminescent Image\nAnalyzer LAS-3000 (Fujifilm).\nWestern Blotting—Unless indicated, cells were lysed in radioimmune\nprecipitation assay buffer (containing 25 mM HEPES (pH 7.5), 150 mM\nNaCl, 1% Triton X-100, 0.1% SDS, 2 mM EDTA, 2 mM EGTA, 10%\nglycerol, 1 mM NaF, 200 M sodium-orthovanadate, and protease inhib-\nitors (1 g/ml leupeptin, 1 g/ml aprotinin, 1 mM phenylmethylsulfonyl\nfluoride). The lysates were cleared from insoluble material by centrifu-\ngation at 20,000  g for 10 min, boiled in Laemmli buffer, subjected to\npolyacrylamide gel electrophoresis, and analyzed by Western blotting\nwith the desired primary antibodies, followed by horseradish peroxi-\ndase-conjugated secondary antibodies (Calbiochem), and developed by\nenhanced chemiluminescence reaction (Pierce). The digital chemilumi-\nnescence images were taken by a Luminescent Image Analyzer LAS-\n3000 (Fujifilm).\nImmunoprecipitation—Generally, the lysates from 2  106 trans-\nfected cells were used for one immunoprecipitation. The cells were\nwashed twice with ice-cold PBS and lysed in the extraction buffer (300\nl/106 cells) containing 150 mM NaCl, 25 mM HEPES (pH 7.4), 0.5%\nNonidet P-40, 2 mM EDTA, 2 mM EGTA, 100 M orthovanadate, and\nprotease inhibitors (200 M phenylmethylsulfonyl fluoride, 2 g/ml leu-\npeptin, and 2 g/ml aprotinin). The lysates were cleared by centrifuga-\ntion at 20,000  g for 5 min. The immunoprecipitation of endogenous\n-catenin was performed by overnight incubation of the cleared lysates\nwith agarose-conjugated goat anti--catenin antibodies (Santa Cruz\nBiotechnology, catalog number sc-1496), followed by three washes with\n1 ml of lysis buffer. Immunoprecipitation of FLAG-tagged -catenin\nproteins was performed using agarose-conjugated mouse anti-FLAG\nantibodies (Sigma-Aldrich, catalog number A2220). Immunoprecipita-\ntion of CBP was performed using anti-CBP antibodies followed by incu-\nbation with protein A/protein G-conjugated agarose beads (The\nimmune complexes were boiled in Laemmli buffer and subjected to\nWestern blotting with the desired antibodies.\nIn Vitro 32P Labeling of -Catenin—FLAG-tagged wild-type (WT)\n-catenin or its PKA phosphorylation mutants were expressed in COS7\ncells and purified using anti-FLAG-agarose beads. The beads were\nwashed once with PKA assay buffer (20 mM Tris-HCl (pH 7.4), 100 mM\nNaCl, 10 mM MgCl2, 1 mM ATP) and then incubated with 45 l of PKA\nassay buffer supplemented with 0.1 mM dithiothreitol, 5 units of purified\nprotein kinase A catalytic subunit (Promega, V5340), and 5 Ci of\n[-32P]ATP at 30 °C for 30 min. The reaction was stopped by the addi-\ntion of 15 l of 4 Laemmli buffer and heating for 5 min at 95 °C.\nProteins were subjected to SDS-PAGE, followed by autoradiography\nWestern blotting with the desired antibodies.\n32P Labeling of -Catenin in Intact Cells—COS7 cells were trans-\nfected with cDNAs for N-terminally truncated (N100) FLAG-tagged\nWT--catenin or its PKA phosphorylation mutants and serum-starved\nfor 24 h. The cells were incubated with 0.25 mCi/ml [32P]orthophos-\nphate in a phosphate-free and serum-free medium, followed by three\nwashes and stimulation with 10 M forskolin (FSK) for 10 min. The cells\nwere then lysed, and the cell lysates were subjected to immunoprecipi-\ntation with FLAG antibodies as described above, followed by electro-\nphoresis, autoradiography, or Western blotting with FLAG antibodies.\nTCF/LEF-Luciferase Reporter Assay—Cells grown on 24-well plates\nwere transfected in triplicates or quadruplicates with cDNAs (20\nng/well) for TCF-luciferase reporter (TOPflash) or its mutated control\nreporter (FOPflash) along with the Renilla reporter driven by the thy-\nmidine kinase promoter (tk-RL) and with the other plasmids of interest.\nThe total amount of DNA was equalized in all of the samples by using\nthe corresponding empty vector. Following stimulation with the desired\nagonists, the cells were lysed and the luciferase activity was measured\nand normalized to the corresponding Renilla activity using the dual-\nluciferase assay kit (Promega). The normalized FOPflash values were\nsubtracted from the corresponding TOPflash values and expressed as\nmean  S.D.\nIndirect\nImmunofluorescence Microscopy—Immunofluorescence\nwas performed as described previously (18). Cells grown on glass cham-\nber slides and transfected with FLAG-tagged -catenin cDNA were\nstimulated with or without 10 M forskolin for 15 min, washed twice\nwith ice-cold PBS, fixed in 4% paraformaldehyde in PBS for 15 min at\nroom temperature, washed again with PBS, and permeabilized in 0.2%\nTriton-X100 in PBS for 5 min, followed by incubation with 1% bovine\nserum albumin in PBS for 1 h. The cells were then incubated with\nmonoclonal anti-FLAG antibodies (5 g/ml in PBS/bovine serum albu-\nmin, as above) overnight at 4 °C for 1 h and washed four times with PBS,\nfollowed by incubation with fluorescein-conjugated goat anti-mouse\nIgG (10 g/ml in PBS/bovine serum albumin, as above) for 1 h atroom\ntemperature. The slides were additionally washed four times with PBS, and\nthe coverslips were mounted using Gel/Mount aqueous mounting medium\n(Fisher, Pittsburgh, PA). The confocal images (sections near the bottom of\nthe cell) were taken under a confocal fluorescent microscope.\nStatistics—The Bonferroni t test for multiple comparisons was uti-\nlized to test for statistically significant differences. The appropriate level\nof significance was determined for each analysis, according to the num-\nber of simultaneous comparisons tested against the reference group (p\nvalue cutoffs are indicated in the figure legends).\nRESULTS\nPhosphorylation of -Catenin by PKA—We first tested the possibility\nthat the endogenous -catenin can be phosphorylated by PKA using\nPKAS antibodies directed against the phosphorylated PKA substrates\nPhosphorylation of -Catenin by PKA\n((RXX(pS/pT), Cell Signaling Technology). Fig. 2A shows stimulation of\nPKA by a cAMP-rising agent, FSK, and inhibition by adenovirus (Ad)-\nmediated transduction of the PKA inhibitor PKI (AdPKI). The specific-\nity of AdPKI was previously confirmed by its inability to affect mitogen-\nactivated protein kinase activation induced by various agonists (16).\nWestern blotting of cell lysates with PKAS antibodies revealed a PKA-\ndependent phosphorylation of a number of proteins, and this PKAS\nrecognition was attenuated by AdPKI transduction (Fig. 2B). When\n-catenin was immunoprecipitated from cell lysates followed by West-\nern blotting with PKAS antibodies, a single band that co-migrated with\n-catenin was detected only after FSK stimulation (Fig. 2C). Overex-\npression of PKI prevented the recognition of immunoprecipitated\n-catenin by PKAS antibodies. The equal amount of -catenin in the\nimmunoprecipitation samples was confirmed by probing with -cate-\nnin antibodies (Fig. 2D). These data suggest that endogenous -catenin\ncan be phosphorylated in a PKA-dependent manner in response to FSK.\nWe identified two potential PKA phosphorylation sites on -catenin\n(serine 552, serine 675; Fig. 1) and generated alanine substitutions to\nexamine the function. We first examined whether PKA could phospho-\nrylate -catenin and the above mutants in vitro and found that PKA\nreadily phosphorylated the WT -catenin (Fig. 3). The S552A or S675A\nmutation partially attenuated this effect, whereas the double mutation\n(S552A/S675A) completely abolished the phosphorylation of -catenin\nby PKA (Fig. 3). We then examined whether FSK could stimulate PKA-\ndependent phosphorylation of -catenin in intact cells. To dissect the\nCK1- and GSK3-dependent phosphorylations, we generated the FLAG-\ntagged truncated versions of -catenin and of its PKA phosphorylation\nmutants by deleting the N-terminal (N100) region containing the\nphosphorylation sites for CK1 or GSK3. We then overexpressed these\ntruncated -catenin proteins in COS7 cells and examined their phos-\nphorylation in response to FSK by immunoprecipitation from 32P-la-\nbeled cells, followed by autoradiography. As shown in Fig. 4, FSK stim-\nulated phosphorylation of N100-WT--catenin; this phosphorylation\nwas partially attenuated by the S552A or S675A mutations and was\nabolished by mutation of both sites. The smear (Fig. 4A, above the\n-catenin band), nonspecifically precipitated by FLAG antibodies,\nserved as an internal control for the presence of 32P labeling in all of the\noriginal lysates. Together, these data convincingly demonstrate that\n-catenin can be phosphorylated by PKA at Ser-552 and Ser-675 sites\nboth in vitro and in intact cells.\nPhosphorylation of -Catenin Promotes Its Transcriptional Activity—\nTo understand the functional significance of -catenin phosphorylation\nby PKA, we first examined how the stimulation of cells with FSK affects\nthe signaling of endogenous -catenin, using TCF/LEF-luciferase\nreporter (TOPflash/FOPflash). As shown in Fig. 5A, FSK stimulated a\nsignificant induction of TCF/LEF reporter (TOPflash) without affecting\nthe control (mutant) reporter activity (FOPflash). Overexpression of\ndominant negative PKA mutant (dnPKA) abolished FSK-induced TCF/\nLEF activity, indicating the role of PKA for this effect (Fig. 5B). The\nefficiency and specificity of dnPKA in inhibiting PKA activity in intact\ncells was previously confirmed (17).\nIt has been suggested that PKA can phosphorylate and inactivate\nGSK3 resulting in decreased phosphorylation and increased stability of\n-catenin (19 –21). We examined whether this could explain the effect\nof FSK on TCF/LEF induction in our experiments. In COS7 cells, FSK\nstimulated a rapid and prolonged phosphorylation of GSK3 (a major\nGSK3 isoform in these cells), as assessed by phosphospecific antibodies\n(Fig. 6A). Nevertheless, the total levels of -catenin, as well as its GSK3-\ndependent phosphorylation, were not significantly affected by FSK, as\nassessed by Western blotting with antibodies against total -catenin\n(Fig. 6D), or with phospho-(pS33/pS37/pT41)-specific -catenin anti-\nPhosphorylation of -Catenin by PKA\nbodies (Fig. 6C). Together, these data suggest that, in COS7 cells, PKA\npromotes the signaling of endogenous -catenin without affecting the\nlevels of its expression.\nWe then examined how the PKA-dependent phosphorylation of\nectopically expressed -catenin affects its transcriptional activity. Over-\nexpression of FLAG-tagged WT--catenin on its own in COS7 cells\nresulted in a 5–7-fold activation of TCF/LEF-reporter, as expected (not\nshown), but this effect was further potentiated by FSK treatment (Fig.\n7A). Mutation of -catenin at S552A did not change the TCF/LEF-\nreporter activity, but the S675A or the S552A/S675A mutations\ndecreased the effect of FSK by half. Western blotting of cell lysates with\nFLAG antibodies revealed that FSK facilitated the expression of ectopic\n-catenin (Fig. 7B), which was probably the result of an increased tran-\nscription by the cytomegalovirus promoter (driving transcription of\n-catenin and its mutants). Likewise, FSK treatment also increased the\nexpression of cytomegalovirus-driven -galactosidase or green fluores-\ncent protein (data not shown). This may partially, but not entirely, con-\ntribute to a FSK-induced TCF/LEF activity in -catenin-transfected\ncells (note that FSK did not affect the expression of endogenous -cate-\nnin (Fig. 6D); whereas it still stimulated the TCF/LEF activity in non-\ntransfected cells (Fig. 5A)). Importantly, the expression of both WT and\nall three PKA phosphorylation mutants of -catenin was equally\nincreased upon FSK stimulation, whereas the FSK-induced TCF/LEF\nreporter activity was significantly reduced in cells transfected with\nS675A or S552A/S675A mutants. This suggests that the S675A muta-\ntion attenuates the ability of FSK to promote the -catenin-mediated\nTCF/LEF activity.\nTo test whether the results obtained in FSK-stimulated COS7 cells\ncould be reproduced in other cells and by other means of PKA stimula-\ntion, we overexpressed the PKA-catalytic subunit (cPKA) in HEK293\ncells. As shown in Fig. 8A, cPKA overexpression in HEK293 cells pro-\nmoted the signaling of WT--catenin or its S552A mutant to a much\ngreater extent than that of S675A or S552A/S675A mutants. Similar to\nour observations using FSK-stimulated COS7 cells (Fig. 7), cPKA over-\nexpression in HEK293 cells resulted in increased levels of -catenin and\nits mutants but all to the same extent (Fig. 8C). Thus, the very similar\nresults obtained by two different approaches strongly suggest that (i)\nPKA promotes the TCF/LEF activity induced by -catenin, at least in\npart, through phosphorylation of -catenin at the Ser-675 site and (ii)\nthis effect of PKA does not seem to involve stabilization of -catenin.\nTo further address the latter notion, we examined the rate of -cate-\nnin degradation following the inhibition of protein synthesis by cyclo-\nheximide. As shown in Fig. 9A, a 3-h treatment of cells with cyclohexi-\nmide resulted in a 60% decrease of exogenous -catenin but not of\n-actin. Importantly, (i) treatment of cells with FSK did not protect\n-catenin from degradation, and (ii) there was no significant difference\nbetween WT--catenin and its PKA phosphorylation mutants in the\ndegradation rate. We also examined how PKA affects the intracellular\nlocalization of -catenin. As shown on Fig. 9B, overexpressed -catenin\nlocalized mainly to the nucleus as expected. However, forskolin stimu-\nPhosphorylation of -Catenin by PKA\nlation for 15 min (which results in maximal phosphorylation at both the\nSer-552 and Ser-675 sites) did not affect the localization of WT--\ncatenin (Fig. 9B). Furthermore, the alanine substitution of either the\nSer-552 or Ser-675 sites also had no obvious effect on -catenin local-\nization under basal or forskolin-stimulated conditions; it still remained\nlargely in the nucleus (data not shown). Together, these data suggest\nthat PKA promotes -catenin signaling without affecting its stability or\nintracellular localization.\nGiven that the Ser-675 phosphorylation site (responsible for the pro-\nmoting effect of PKA) is located within the transactivation domain of\n-catenin, we then hypothesized that phosphorylation by PKA pro-\nmotes -catenin signaling at the level of its transcriptional activity,\nthrough modulation of its interaction with co-activators, such as CBP.\nCBP is an acetyltransferase that promotes the activity of several tran-\nscription factors (i.e. CREB) by their acetylation. CBP acetylates -cate-\nnin and increases its transcriptional activity (22–24). Therefore, we\nexamined whether phosphorylation of -catenin by PKA is important\nfor -catenin-CBP interaction by co-immunoprecipitation. As shown\non Fig. 10, WT--catenin and the S552A mutant bound specifically to\nCBP, and their binding was significantly increased by 50% after FSK\ntreatment. In contrast, the S675A mutation resulted in a dramatic\nreduction of -catenin binding to CBP, and, importantly, FSK no longer\npromoted this binding. Note that -catenin is partially phosphorylated\nat the Ser-675 site under basal conditions (as the S675A mutation abol-\nishes this basal phosphorylation) (Fig. 4C), which may explain an appre-\nciable binding of -catenin to CBP under control conditions (Fig. 10, no\nFSK ()). Together, these data suggest that phosphorylation of -cate-\nnin by PKA at the Ser-675 site is important for its interaction with CBP,\nwhich may explain how PKA promotes transcriptional activity of\n-catenin.\nFinally, we sought to examine whether the phosphorylation of\n-catenin by PKA affects its interaction with cadherin, for the following\nreasons: (i) the Ser-552 phosphorylation site does not seem important\nfor modulation of -catenin transcriptional activity, suggesting that\nphosphorylation at this site may modulate the other function of -cate-\nnin at cell-cell adherens junctions, (ii) the Ser-552 phosphorylation site\nis located within the 10th armadillo repeat of -catenin that may be\nimportant for interaction with cadherin (25), and (iii) the -catenin/\ncadherin interaction was reported to be regulated by PKA, albeit\nthrough phosphorylation of cadherin (26). As shown in Fig. 11A, endog-\nenous N-cadherin readily and specifically co-immunoprecipitated with\nFLAG-WT--catenin. However, stimulation of cells with FSK had no\neffect on the interaction of N-cadherin with either WT--catenin, or\nwith its PKA phosphorylation mutants. Thus, at least as assessed by\nco-immunoprecipitation approach, phosphorylation of -catenin by\nPKA does not seem to affect its interaction with N-cadherin. In addi-\ntion, Fig. 11C also shows that GSK3-dependent phosphorylation (at\nSer-33/Ser-37/Thr-41 sites) of ectopic -catenin is not affected by PKA-\ndependent phosphorylation of -catenin (at Ser-552 or Ser-675 sites),\nwhich is consistent with the results on the endogenous -catenin phos-\nphorylation by GSK3 (Fig. 6C).\nDISCUSSION\nThe present study describes three major findings. (i) -Catenin can\nbe phosphorylated by PKA in vitro and in intact cells at novel sites\n(Ser-552 and Ser-675) distinct from those phosphorylated by CK1,\nGSK3, or Src kinases. (ii) Phosphorylation of -catenin by PKA at the\nSer-675 site promotes its transcriptional activity by a noncanonical\nmechanism that does not involve the stabilization of -catenin. (iii)\nPhosphorylation of -catenin at the Ser-675 site facilitates the interac-\ntion between -catenin and CBP, which may provide a molecular mech-\nanism by which PKA promotes the transcriptional activity of -catenin.\nThe role of the cAMP/PKA pathway in the modulation of cell growth\nPhosphorylation of -Catenin by PKA\nis highly complex and cell-specific; it stimulates cell growth in many cell\ntypes while inhibiting cell growth in others (27). Moreover, even in the\nsame cell type (vascular smooth muscle cells), PKA can promote or\ninhibit cell proliferation dependent on the agonist that stimulates its\nactivity (16). The present study suggests that, in COS7 and HEK293\ncells, PKA facilitates the signaling of -catenin, which is consistent with\nthe recent report demonstrating that prostaglandin E2, through PKA\nactivation, promotes -catenin signaling in colon cancer cells (28).\nHowever, in the latter report, the promoting effect of PKA is attributed\nto phosphorylation and inactivation of GSK3, whereas our results point\nto the role of a direct phosphorylation of -catenin by PKA at the Ser-\n675 site.\nRegarding the mechanism by which phosphorylation of -catenin by\nPKA promotes its transcriptional activity, we first showed that phos-\nphorylation of -catenin by PKA: (i) does not affect the GSK3-depend-\nent phosphorylation of endogenous or ectopic -catenin at the Ser-33/\nSer-37/Thr-41 sites (Figs. 6C and 11C), (ii) does not protect -catenin\nfrom degradation (Fig. 9A), (iii) does not affect the amount of endoge-\nnous -catenin (Fig. 6D), and (iv) does not affect the nuclear localization\nof ectopic -catenin (Fig. 9B). Thus, our data suggest that PKA pro-\nmotes -catenin signaling by a noncanonical mechanism. In this study,\nwe have proposed one potential mechanism wherein phosphorylation\nof -catenin by PKA at the Ser-675 site promotes the interaction\nbetween -catenin and its co-activator, CBP, whereas S675A mutation\nnearly abolishes this interaction.\nIt has been recently shown that the phosphoserine-binding protein\n14-3-3 interacts with -catenin and potentiates its transcriptional\nactivity by an unknown mechanism (29). Given that both PKA phos-\nphorylation sites on -catenin (RRTS552MGGT and KRLS675VELT)\n(Fig. 1) match the type-2 consensus 14-3-3 binding motif (RX1–2(pS/\npT)X2–3(pS/pT)) (30), we explored whether 14-3-3 interacts with\n-catenin through these sites. However, we have failed to coimmuno-\nprecipitate these two proteins despite multiple efforts using different\nconditions (data not shown). In contrast, a positive control interaction\nbetween the regulator of G protein signaling (RGS3) and 14-3-3 (exper-\niments performed in parallel) was readily detectable, as we have shown\npreviously (31).\nWhile this work was in progress, an independent study by Hino et al.\n(32) has been published showing that PKA can phosphorylate -catenin\nat the Ser-675 site. Consistent with our results, these authors show that\nphosphorylation of -catenin at this site promotes TCF-dependent\ngene transcription, whereas the S675A mutation attenuates this effect.\nHowever, Hino et al. (32) suggest that phosphorylation of -catenin at\nthe Ser-675 site leads to stabilization of -catenin, which is different\nfrom what our data, showing no stabilization and no protection of\n-catenin from degradation, suggest. Although acknowledging that\nsuch a discrepancy could be the result of multiple experimental factors,\nincluding cell-type specificity, cell growth conditions, density, etc., we\npropose here an additional possible mechanism by which PKA pro-\nmotes the transcriptional activity of -catenin, namely by facilitating its\ninteraction with CBP (Fig. 10).\nFinally, we also identified Ser-552 as a site of phosphorylation by PKA,\nwhich was not reported previously (Figs. 3 and 4). Even though its muta-\ntion has a lesser effect on 32P incorporation into -catenin, as does the\nS675A mutation (Figs. 3A and 4A), the Ser-552 site is surely phospho-\nrylated by PKA, because its mutation abolishes the recognition of\n-catenin by antibodies against phosphorylated PKA substrates (data\nnot shown). At this time, we do not know what the functional signifi-\ncance is of -catenin phosphorylation at the Ser-552 site. Few possibil-\nities exist. (i) It has no significance; this can be argued by the notion that\nevery modification of the protein is meaningful. (ii) We have missed the\neffect of the S552A mutation due to an overwhelming overexpression of\nthe mutant; this is unlikely, because under the same transfection condi-\ntions, the effect of the S675A mutation was readily detectable (Figs. 7, 8,\nand 10). (iii) Phosphorylation of -catenin at the Ser-552 site may be\nimportant for another previously unrecognized function of -catenin.\nThese unanswered questions are currently being addressed in our\nlaboratory.",
         "Phosphorylation of β-Catenin by Cyclic AMP-Dependent Protein Kinase Stabilizes β-Catenin through Inhibition of Its Ubiquitination;Prostaglandin E2 Stimulates the β-Catenin/T Cell Factor-dependent Transcription in Colon Cancer*;Proteomic analysis identifies that 14-3-3ζ interacts with β-catenin and facilitates its activation by Akt;Dual role of PKA in phenotypic modulation of vascular smooth muscle cells by extracellular ATP;Functional significance of protein kinase A activation by endothelin-1 and ATP: negative regulation of SRF-dependent gene expression by PKA.;The Wnt signalling pathway;A-Kinase Anchoring Protein AKAP220 Binds to Glycogen Synthase Kinase-3β (GSK-3β) and Mediates Protein Kinase A-dependent Inhibition of GSK-3β*;Glycogen Synthase Kinase 3β Is Activated by cAMP and Plays an Active Role in the Regulation of Melanogenesis*;RGS3 interacts with 14-3-3 via the N-terminal region distinct from the RGS (regulator of G-protein signalling) domain.;Acetylation of β-Catenin by CREB-binding Protein (CBP)*;Crosstalk between cAMP and MAP kinase signaling in the regulation of cell proliferation.;Wnt Signaling Controls the Phosphorylation Status of β-Catenin*;Control of β-Catenin Phosphorylation/Degradation by a Dual-Kinase Mechanism;The Structure of the β-Catenin/E-Cadherin Complex and the Molecular Basis of Diverse Ligand Recognition by β-Catenin;Cyclic AMP Promotes Neuronal Survival by Phosphorylation of Glycogen Synthase Kinase 3β;Linking Colorectal Cancer to Wnt Signaling;Regulator of G Protein Signaling RGS3T Is Localized to the Nucleus and Induces Apoptosis*;The transcriptional coactivator CBP interacts with beta-catenin to activate gene expression.;The p300/CBP acetyltransferases function as transcriptional coactivators of β‐catenin in vertebrates;Mechanism and function of signal transduction by the Wnt/β-catenin and Wnt/Ca2+ pathways;Regulation of E-cadherin/Catenin Association by Tyrosine Phosphorylation*;β-Catenin regulates expression of cyclin D1 in colon carcinoma cells;The oncogenic activation of β-catenin;Identification of c-MYC as a target of the APC pathway.;β-catenin: a key mediator of Wnt signaling;Serine Phosphorylation-regulated Ubiquitination and Degradation of β-Catenin*;β‐catenin is a target for the ubiquitin–proteasome pathway;Activation of β-Catenin-Tcf Signaling in Colon Cancer by Mutations in β-Catenin or APC;Binding of GSK3β to the APC-β-Catenin Complex and Regulation of Complex Assembly;The cytoplasmic domain of the cell adhesion molecule uvomorulin associates with three independent proteins structurally related in different species.;Mutations affecting segment number and polarity in Drosophila;Cyclic AMP‐dependent protein kinase A negatively modulates adherens junction integrity and differentiation of intestinal epithelial cells;14-3-3 proteins: structure, function, and regulation.;hp://w w w .jb.org/  D;Phosphorylation of -Catenin by PKA 9976 JOURNAL OF BIOLOGICAL CHEMISTRY VOLUME",
         "Phosphorylation of β-Catenin by Cyclic AMP-dependent Protein Kinase*",
         "None"
        ],
        [
         "34",
         "001c80ded266b8d277cf9276003215e75637ceb5",
         "The ompF and ompC genes of Escherichia coli are reciprocally regulated by a single transcription factor, phosphorylated OmpR (OmpR-P), depending upon medium osmolarity. This regulation involves activation of ompF and its repression with concomitant activation of ompC. This occurs through OmpR-P binding to four (F1, F2, F3, and F4) and three (C1, C2, and C3) sites located upstream of the ompF and ompC promoters, respectively, through a novel mechanism. Here we show that there is a distinct OmpR-P binding hierarchy within F1, F2, and F3 sites as well as within C1, C2, and C3 sites. Each of these sites contains two tandem 10-bp OmpR-P-binding subsites, a-site and b-site (from 5′ to 3′ direction). OmpR-P has higher affinity to the downstream b-site than to the upstream a-site in each case. Six OmpR-P molecules bind to F and C sites two-by-two in a discontinuous “galloping” manner. We propose that this tight hierarchical binding of a transcription factor, OmpR, allows distinct stepwise regulation of ompF and ompC transcription, which minimizes their overlapping expression upon changes in the medium osmolarity to achieve the reciprocal expression of ompF and ompC.",
         "Takeshi Yoshida,L. Qin,L. Egger,M. Inouye",
         "Takeshi Yoshida,L. Qin,L. Egger,M. Inouye",
         "http://www.jbc.org/article/S0021925820557384/pdf",
         "The ompF and ompC genes of Escherichia coli are reciprocally\nregulated by a single transcription factor, phosphorylated OmpR\n(OmpR-P), depending upon medium osmolarity. This regulation\ninvolves activation of ompF and its repression with concomitant\nactivation of ompC. This occurs through OmpR-P binding to four\n(F1, F2, F3, and F4) and three (C1, C2, and C3) sites located\nupstream of the ompF and ompC promoters, respectively, through a\nnovel mechanism. Here we show that there is a distinct OmpR-P\nbinding hierarchy within F1, F2, and F3 sites as well as within C1,\nC2, and C3 sites. Each of these sites contains two tandem 10-bp\nOmpR-P-binding subsites, a-site and b-site (from 5 to 3 direction).\nOmpR-P has higher affinity to the downstream b-site than to the\nupstream a-site in each case. Six OmpR-P molecules bind to F and C\nsites two-by-two in a discontinuous “galloping” manner. We pro-\npose that this tight hierarchical binding of a transcription factor,\nOmpR, allows distinct stepwise regulation of ompF and ompC tran-\nscription, which minimizes their overlapping expression upon\nchanges in the medium osmolarity to achieve the reciprocal expres-\nsion of ompF and ompC.",
         "20-bp F1 site (11). This F1 sequence consists of two 10-bp direct repeats,\nF1a at the 5-end and F1b at the 3-end, which share a number of com-\nmon bases (Fig. 1A). In the present study, we demonstrated that\nOmpR-P binds more tightly to downstream F1b than to upstream F1a,\nbecause OmpR-P is able to bind to F1bF1b but not to F1aF1a (Fig. 1, B\nand C). This OmpR-P binding hierarchy is due to the fact that the F1b\nsite contains more conserved bases (eight bases; Fig. 1A) than the F1a\nsite (six bases) in the 10-base consensus sequence (G1)-T2-T3-A4-C5-\nA6-T7-X-T9-X, derived by PCR-based enrichment technology (Fig. 1A\nand Ref. 11). We showed that G1, A4, C5, and A6 residues in this con-\nsensus sequence play a key role for optimal OmpR-P binding (Figs. 2C\nand 3, A and B). Similarly, in all of the other OmpR-P-binding sites (F2,\nF3, C1, C2, and C3) except for F4, the b-sites have more consensus bases\nthan their cognate a-sites (Fig. 1A). Notably, all of the b-sites contain\nboth G1 and C5 residues. Consistently, the gel shift assay demonstrated\nthat OmpR-P has a higher affinity to the b-sites than to the a-sites (Fig.\n6). Thus, the following hierarchy was established for OmpR-P binding\naffinity within each OmpR-P-binding site; F1b  F1a, F2b  F2a, F3b \nF3a, C1b  C1a, C2b  C2a, and C3b  C3a. As for the F4 site, the G1\nresidue in F4b is absent, whereas F4a has the G1 residue. It is currently\nnot known whether F4b has higher affinity than that of F4a for binding\nto OmpR-P.\nStimulation of OmpR Cooperative Binding by Phosphorylation—Al-\nthough both OmpR and OmpR-P exist as monomers in solution, two\nOmpR-P molecules cooperatively bind to the F1 site (11, 12). There are\ntwo modes of cooperative binding of OmpR-P F and C sites: one\nbetween two OmpR-P molecules in each 20-bp OmpR-P binding unit\nand the other between OmpR-P dimer binding to F1 and F2, to F2 and\nF3, to C1 and C2, or to C2 and C3. It is not known at present how\ncooperative binding occurs between two OmpR-P dimers on the F and\nC sites. In the present study, we found that nonphosphorylated OmpR is\nalso capable of cooperatively binding to F1aF1b (F1) and F1bF1b (Fig. 4,\nA and B). The C-terminal DNA-binding domain (OmpRc) by itself was\ncapable of binding to the F1b site in F1, but it could not cooperatively\nbind to F1aF1b (Fig. 4C). This indicates that the N-terminal domain of\nOmpR is responsible for its cooperative binding to DNA, and its phos-\nphorylation is likely to play an important role in enhancing the interac-\ntion between two N-terminal domains of the OmpR-P dimer on DNA.\nRecent structural studies of proteins in the OmpR/PhoB family sug-\ngest that the N-terminal domain is important to cooperative OmpR-P\nDNA binding activity. The x-ray structures of DrrD and DrrB, OmpR\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nhomologues from Thermotoga maritima, revealed that the N-terminal\ndomain of the molecule interacts with its C-terminal domain (18, 19).\nBecause the 4-5-5 structure of the N-terminal domain that inter-\nacts with the C-terminal domain is highly conserved among the OmpR/\nPhoB family, it was proposed that this region may also serve as an inter-\nface to stabilize dimer formation on DNA (18). This proposal was\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nsupported by recent structural studies of the dimerized N-terminal\ndomain of ArcA (20) and PhoB (21), suggesting an intriguing insight\ninto the role of phosphorylation and subsequent interaction between\nOmpR-P molecules through the N-terminal 4-5-5 region.\nDNA-mediated Dimerization of OmpR-P—We previously showed\nthat certain substitution mutations in F1a completely abolished\nOmpR-P binding to the mutated F1 fragments even if F1b was kept\nintact, indicating that one OmpR-P molecule is unable to make a stable\ncomplex with DNA (11). In the present study, we showed that only two\nOmpR-P molecules bind to a 30-bp DNA fragment consisting of three\noptimized binding subsites (F1bF1bF1b),\nindicating that OmpR-P\ndimer formation is mediated by DNA.\nThis pair-wise binding of OmpR-P is not due to preformed OmpR-P\ndimers that subsequently bind to DNA, because switching the order of\nthe a- and b-sites in F1 significantly reduced cooperative OmpR-P bind-\ning to the F1bF1a-F2 fragment (Fig. 6E). It appears that OmpR-P dimer\nformation occurs only on DNA such that the binding of an OmpR-P\nmolecule to the downstream (stronger) b-site assists binding of a second\nOmpR-P molecule to the upstream (weaker) a-site. This DNA-medi-\nated OmpR-P dimer formation is highly cooperative and synergistic,\nbeing mediated by interaction between the N-terminal domains (possi-\nbly the 4-5-5 regions) of two OmpR-P molecules and the interac-\ntion of the C-terminal domain of the second OmpR-P molecule with\nDNA at the a-site. In this model, the C-terminal DNA-binding domains\nare tandemly (or in a head-to-tail manner) aligned on DNA, whereas the\nN-terminal domains interact in a head-to-head manner as proposed by\nStock and co-workers (12, 18, 20, 21). Thus, the order of OmpR-P inter-\naction with the two subsites appears to play a critical role in the forma-\ntion of the most stable OmpR-P/DNA complex.\nSignificance of Hierarchical OmpR-P Binding in Reciprocal Gene Reg-\nulation of ompF and ompC—On the basis of the data presented, we\ntermed the two-by-two binding mechanism of OmpR-P to DNA as a\ngalloping model because this OmpR-P binding mode is similar to the\nway the front and rear legs move when a horse gallops. The key feature\nof this model is that according to hierarchical arrangement of OmpR-\nP-binding sites at the regulatory elements, F1-F2-F3 and C1-C2-C3, six\nOmpR-P molecules bind discontinuously (two-by-two) because a\ndownstream b-subsite in individual F and C sites serve as the primary\nOmpR-P-binding site, and simultaneously a second OmpR-P molecule\nbinds to a weaker upstream a-subsite (Fig. 7A).\nThe tandem arrangement of two 10-bp sites is essential for two\nOmpR-P molecules to form a stable complex with DNA. If both sites\nhave a weak affinity for OmpR-P, it severely reduces the binding of\nOmpR-P to these sites, requiring a significantly high concentration of\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nOmpR-P to detect OmpR-P binding to DNA (Figs. 1 and 6). On the\nother hand, if both sites have a high affinity for OmpR-P, even nonphos-\nphorylated OmpR is able to bind to DNA (Fig. 4). Therefore, the\narrangement of a weaker (a-subsite) followed by a higher (b-subsite)\naffinity found in all OmpR-P-binding sites in F1-F2-F3 and C1-C2-C3 is\nimportant for ompF and ompC regulation. As proposed in a galloping\nmodel, discontinuous arrangement of the a-subsite with weak affinity\nand the b-subsite with high affinity probably ensures that one OmpR-P\nmolecule first binds to the b-subsite. This also regulates the final binding\nof two OmpR-P molecules by modulating the affinity of a second a-sub-\nsite. It is also important to note that the different affinity between F2a\nand C2a (which lacks C at 5th position) for OmpR-P, plays a key role in\nthe reciprocal regulation of ompF and ompC. This is because F2a has a\nhigher affinity for OmpR-P than C2a, because two OmpR-P molecules\nare able to bind to F2aF2a site at a high concentration of OmpR-P, but\nnot to the C2aC2a site. This results in the binding of four OmpR-P\nmolecules to F1-F2aF2a (Fig. 6A) but binding of only two OmpR-P\nmolecules to C1-C2aC2a (Fig. 6C). The same effect was also observed\nwith F3a and C3a (Fig. 6, B and D), so that only after OmpR-P occupies\nall F1-F2-F3 sites, will the C1-C2-C3 sites be occupied with OmpR-P,\nthus achieving the reciprocal regulation of ompF and ompC.\nEarlier footprinting studies established that OmpR-P binding to each\nof the regulatory elements for the ompF and ompC promoters occurs\ndifferentially in four steps (9, 14); (i) F1, C1 (ompF and ompC are not\nexpressed); (ii) F1-F2, C1 (ompF, but not ompC is expressed); (iii) F1-F2-\nF3, C1 (ompF, but not ompC is expressed); and (iv) F1-F2-F3 plus F4,\nC1-C2-C3 (ompF repressed and ompC is expressed). This suggests that\nthere are four critical cellular concentrations of OmpR-P that regulate\nthe induction and repression of ompF and the induction of ompC. Fig.\n7B illustrates the sequential regulation of ompF and ompC expression,\nwherein the cellular concentration of OmpR-P increases from KI (low)\nto KIV (high) upon medium osmolarity changes. At KI, only F1 and C1\nare occupied with two OmpR-P molecules. Note that two OmpR-P\nmolecules bind to both F1 and C1 sites in the order depicted by the\nnumbered molecules and according to the galloping model. Both ompF\nand ompC are not transcribed under this condition. When the OmpR-P\nconcentration increases from KI to KII, two more OmpR-P molecules\nbind to the F2 site again in the order from molecule 3 to molecule 4. At\nKIII, two more OmpR-P molecules (molecules 5 and 6) bind to the F3\nsite. On the other hand, the OmpR-P concentration of KII or KIII is not\nhigh enough to occupy C2 and C3 sites, allowing exclusive transcription\nof ompF. As medium osmolarity further increases, the OmpR-P concen-\ntration finally reaches KIV allowing the binding of two more OmpR-P\nmolecules (molecules 7 and 8) to F4 to repress ompF transcription. This\nrepression is assumed to be due to DNA loop formation between F4 and\nF1-F2-F3 mediated by OmpR-P bound to these regions (9). Six OmpR-P\nmolecules (molecules 1– 6) are now able to bind to C1-C2-C3 in a dis-\ncontinuous and cooperative manner to activate ompC transcription.\nThis well orchestrated gene regulation of ompF and ompC by\nOmpR-P is achieved by the combination of the two-domain structure of\nOmpR, the cellular concentration of OmpR-P, and the highly sophisti-\ncated hierarchical arrangement of OmpR-P-binding sites. Each N- and\nC-terminal domain of OmpR has a distinct role in the DNA binding of\nOmpR-P. We have demonstrated that E. coli cells contain a large pool of\nOmpR, whereas only a minor fraction is presumed to be phosphorylated\nby EnvZ, a transmembrane osmosensing histidine kinase (7, 8). The\ncellular level of OmpR-P is thus finely tuned by dual EnvZ functions,\nOmpR kinase and OmpR-P phosphatase, whose ratios are regulated by\nthe medium osmolarity. Finally, because of the hierarchical arrange-\nment of OmpR-P-binding sites, OmpR-P occupy differently the regula-\ntory element of ompF and ompC promoters depending on the concen-\ntration of OmpR-P, allowing a reciprocal expression of these genes to\navoid overlapping expression from both genes at the same time.\nConclusion—In this study, we demonstrated a regulatory mechanism\nfor transcription using a single transcription factor. The reciprocal tran-\nscription of ompF and ompC is regulated by OmpR-P and is finely tuned\nby virtue of hierarchical OmpR-P binding to DNA. This simple but\nhighly sophisticated transcriptional regulation performed by a single\ntranscription factor may be widely used in prokaryotes and is an inter-\nesting contrast to the eukaryotic systems, where a large number of tran-\nscription factors are involved in regulating the expression of a single\ngene.",
         "Escherichia coli contains two outer membrane porin proteins, OmpF\nand OmpC, which serve as passive diffusion pores across the outer\nmembrane. Expression of these two proteins is reciprocally regulated by\nmedium osmolarity. OmpF is preferentially produced at low osmolarity,\nwhereas OmpC is almost exclusively produced at high osmolarity.\nBecause OmpF forms a larger pore than OmpC (1), the regulation of the\nsize of the passive diffusion pores across the outer membrane appears to\nbe vital for E. coli, which is able to live in two distinctly different envi-\nronmental conditions: one under nutrient-rich high osmolarity condi-\ntions in animal gut at 37 °C and the other under poor nutritional and low\nosmolarity conditions outside animals and thus at low osmolarity at\nambient temperatures. It has been proposed that the larger OmpF pore\nsize is important for efficient nutrient uptake from nutritionally poor\nmedia, whereas the smaller OmpC pore size is important to exclude the\npassage of toxic bile salt across the outer membrane (2). E. coli has a\nhighly sophisticated regulatory system to regulate the reciprocal expres-\nsion of ompF and ompC. Both are controlled at the transcriptional level\nby the histidine kinase EnvZ, a transmembrane osmosensor, and the\nresponse regulator OmpR, a transcriptional factor (3), and also at the\ntranslational level by antisense RNAs, micF RNA for ompF mRNA (4)\nand micC RNA for ompC mRNA (5).\n* This work was supported by Grant GM076587 from the National Institutes of Health.\nThe costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked “advertisement” in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\nThe EnvZ-OmpR system is a typical two-component His-Asp phos-\nphorelay signal transduction system in E. coli (see review in Ref. 6).\nBecause EnvZ not only has kinase activity to phosphorylate OmpR but\nalso phosphatase activity toward phosphorylated OmpR (OmpR-P), it is\nable to regulate the level of OmpR-P in the cell depending upon the\nmedium osmolarity. At low medium osmolarity, the phosphatase activ-\nity is relatively higher than the kinase activity so that the cellular con-\ncentration of OmpR-P is maintained at a lower concentration, which is\nsufficient to activate expression of ompF but not of ompC. At high\nmedium osmolarity, the EnvZ phosphatase activity becomes relatively\nlower than its kinase activity, resulting in higher cellular concentration\nof OmpR-P. Under this condition, ompF expression is repressed,\nwhereas ompC expression is activated.\nIn our recent study on the interaction between the cytoplasmic\ndomain of EnvZ (EnvZc) and OmpR or OmpR-P (7), we proposed that\nthe reciprocal expression of ompF and ompC occurs through a finely\ntuned mechanism. In this model, osmotic signals regulate the levels of\nOmpR-P by modulating the ratio of kinase to phosphatase activity of\nEnvZ. Most importantly, of the large pool of OmpR (3,500 molecules/\ncell), only a very small portion is proposed to be phosphorylated (8). On\nthe other hand, the Kd value for OmpR-P binding to the regulatory\nregions of the ompF and ompC promoters (9, 10) is less than 1% of the\ntotal cellular concentration of OmpR and OmpR-P. This was proposed\nto be a critical factor in the reciprocal expression of ompF and ompC.\nFor example, even with a minor change in the ratio of the kinase to\nphosphatase activity of EnvZ, the resulting small shift in the cellular\nconcentration of OmpR-P would be sufficient to substantially change\nthe number of OmpR-P molecules bound to the regulatory regions of\nthe ompF and ompC promoters. There is approximately a few hundred-\nfold difference between the Kd value for OmpR-P binding to the regu-\nlatory regions of the ompF and ompC promoters versus the total cellular\nconcentration of OmpR and OmpR-P (7).\nThere are four F sites (F1, F2, F3, and F4) and three C sites (C1, C2,\nand C3) in the upstream region of the ompF and ompC promoters,\nrespectively. Each site consists of 20 base pairs, providing a tandem\nbinding site for two OmpR-P molecules (11, 12). The hierarchy of\nOmpR-P binding to these F and C sites was determined to be F1, C1 \nF2, F3  C2  C3 (9). At low osmolarity, OmpR-P cooperatively binds\nto F1-F2/F1-F2-F3 to activate ompF transcription. Under this condition,\nonly the C1 site is occupied by OmpR-P, which is not sufficient to\nactivate ompC transcription. Only when the OmpR-P concentration\nelevates in response to increased medium osmolarity do the C2 and C3\nsites become occupied by OmpR-P, resulting in ompC expression.\nUnder this condition, OmpR-P also binds to the F4 site (380 to 361),\na weak OmpR-P-binding site located 260-bp upstream of the F1 site\n(100 to 81). OmpR-P binding to the F4 site is proposed to form a\nloop that interacts with OmpR-P molecules binding to F1, F2, and F3,\nthereby blocking ompF transcription (9, 13, 14).\nHere, we investigated how two OmpR-P molecules cooperatively\nbind to F1, F2, and F3 as well as C1, C2, and C3. The 20-bp F1 site is\nHierarchical Binding of OmpR-P Regulates ompF and ompC\ncomposed of two 10-bp subsites, F1a and F1b (from 5 to 3). We found\nthat the affinity of OmpR-P for the upstream F1a is much weaker than\nthat for the downstream F1b, suggesting binding to the F1 site is initi-\nated by OmpR-P interaction with the F1b subsite. Similarly, OmpR-P\nhad a higher affinity for the downstream subsites, F2b and F3b for F2\nand F3 sites, respectively, than their cognate upstream subsites, F2a and\nF3a. Therefore, OmpR-P binding to F1-F2-F3 occurs discontinuously in\na stepwise manner in the order F1b to F1a, F2b to F2a, and F3b to F3a.\nSimilarly, OmpR-P binds to C1-C2-C3 in this discontinuous manner.\nWe termed this novel mode of OmpR-P binding to the regulatory\nregions as the “galloping model.” This mechanism is inherent to the distinct\nreciprocal expression of ompF and ompC minimizes their overlapping\nexpression under a given condition. Because OmpR belongs to the largest\nresponse regulator family (OmpR/PhoB) in E. coli, the DNA binding of\ntranscription factors in galloping mode may be widely found in E. coli.",
         "EXPERIMENTAL PROCEDURES\nOligonucleotides—The OmpR-binding sites of F (F1, F1-F2, and F1-F2-\nF3), C (C1, C1-C2, and C1-C2-C3) and various mutated sequences were\nsynthesized by Integrated DNA Technologies, Inc. Complimentary strands\nwere first annealed, and double-stranded DNA was purified for DNA gel\nmobility shift assay. DNA concentration was determined by UV absorb-\nance at 260 nm.\nPurification of Proteins—EnvZc[T247R] previously termed EnvZc11,\nOmpR, and PrS2-OmpR were purified as described previously (11, 15).\nThe C-terminal domain of OmpR (OmpRc) was purified as described\npreviously (16).\nDNA Gel Mobility Shift Assay—Each purified 20 pmol of double-\nstranded DNA was labeled by using T4 kinase and [-32P]ATP (3000\nCi/mmol and 10 mCi/ml) for 1 h at 37 °C.Free [ -32P]ATP was removed\nby phenol/chloroform extraction followed by ethanol precipitation.\nAfter air drying, labeled DNA was resuspended in 100 l of water con-\ntaining 70 pmol of unlabeled double-stranded DNA.\nThe binding reactions were carried out in 50 mM Tris-HCl (pH 7.2)\nbuffer containing 50 mM KCl, 5 mM CaCl2, 5% glycerol, 0.05% Nonidet\nP-40, and 100 ng of poly(dI-dC) at room temperature. First, EnvZc11\nwas phosphorylated using 1 mM ATP for 5 min. EnvZc11 mixture was\ntransferred to the tube containing different amounts of OmpR or PrS2-\nOmpR, and the reaction mixture was further incubated for 20 min to\ngenerate phosphorylated OmpR or PrS2-OmpR. Under this condition,\n100% of OmpR or PrS2-OmpR was phosphorylated as judged from C4\nreverse phase high pressure liquid chromatography. Fifteen l of the\nreaction mixture was mixed with 1.5 l of labeled DNA (45 fmol/l)\nand incubated for 30 min (final volume, 16.5 l). After the prerun for\n1 h, the samples were subjected to 5% acrylamide/bisacrylamide (40:1.2)\ngel. The gel was run at 4 °C in TE buffer (10 mM Tris-HCl, pH 7.2, 1 mM\nEDTA) at 110 V, and every 30 min the buffer in the upper chamber and\nthe buffer in the bottom chamber were exchanged. The dried gel was\nexposed for autoradiography and analyzed by phosphorimaging. The\nconcentration of OmpR-P required for 50% binding to DNA is pre-\nsented as the C50 value.\nDNase I Footprinting—Each purified 30-pmol sense oligonucleotide\nwas labeled at 5-end by using T4 kinase and [-32P]ATP (3000\nCi/mmol and 10 mCi/ml) for 1 h at 37 °C.Free [ -32P]ATP was removed\nby phenol/chloroform extraction followed by ethanol precipitation.\nLabeled oligonucleotide was annealed by mixing its complementary\nsequence (40 pmol) in 50 l of the annealing buffer (25 mM Tris-HCl,\npH 8.0, and 400 mM NaCl).\nThe DNA binding reaction was carried out as described in DNA gel\nmobility shift assay by mixing OmpR or OmpRc with 1 l of labeled\nDNA (600 fmol/l); however, 50 mM Tris-HCl (pH 8.0) buffer con-\ntaining 50 mM KCl, 10 mM MgCl2, and 1 mM ATP, was used. Partial\nDNase I digestion was carried out at room temperature by adding 1 l of\n0.2 unit/l DNase I (Promega) into the reaction mixture (final reaction\nvolume, 10 l). After 1-min incubation, the reaction was stopped by\nadding 1 l of 200 mM EDTA, followed by incubating in the boiling\nwater bath for 4 min. 6 l of the loading buffer (95% formamide, 20 mM\nEDTA, 0.05% bromphenol blue, 0.05% xylene cyanol FF) was added to\nthe sample. The samples were incubated in the boiling water bath for 4\nmin, and 2.5 l of the sample was subjected to 12% acrylamide/bisac-\nrylamide (19: 1), 8 M urea gel after prerun for 30 min. The gel was run in\nTBE buffer (50 mM Tris-HCl, 50 mM boric acid, and 1 mM EDTA) at\n45 W. The residues were positioned with the A  G ladders.",
         "EXPERIMENTAL PROCEDURES\nOligonucleotides—The OmpR-binding sites of F (F1, F1-F2, and F1-F2-\nF3), C (C1, C1-C2, and C1-C2-C3) and various mutated sequences were\nsynthesized by Integrated DNA Technologies, Inc. Complimentary strands\nwere first annealed, and double-stranded DNA was purified for DNA gel\nmobility shift assay. DNA concentration was determined by UV absorb-\nance at 260 nm.\nPurification of Proteins—EnvZc[T247R] previously termed EnvZc11,\nOmpR, and PrS2-OmpR were purified as described previously (11, 15).\nThe C-terminal domain of OmpR (OmpRc) was purified as described\npreviously (16).\nDNA Gel Mobility Shift Assay—Each purified 20 pmol of double-\nstranded DNA was labeled by using T4 kinase and [-32P]ATP (3000\nCi/mmol and 10 mCi/ml) for 1 h at 37 °C.Free [ -32P]ATP was removed\nby phenol/chloroform extraction followed by ethanol precipitation.\nAfter air drying, labeled DNA was resuspended in 100 l of water con-\ntaining 70 pmol of unlabeled double-stranded DNA.\nThe binding reactions were carried out in 50 mM Tris-HCl (pH 7.2)\nbuffer containing 50 mM KCl, 5 mM CaCl2, 5% glycerol, 0.05% Nonidet\nP-40, and 100 ng of poly(dI-dC) at room temperature. First, EnvZc11\nwas phosphorylated using 1 mM ATP for 5 min. EnvZc11 mixture was\ntransferred to the tube containing different amounts of OmpR or PrS2-\nOmpR, and the reaction mixture was further incubated for 20 min to\ngenerate phosphorylated OmpR or PrS2-OmpR. Under this condition,\n100% of OmpR or PrS2-OmpR was phosphorylated as judged from C4\nreverse phase high pressure liquid chromatography. Fifteen l of the\nreaction mixture was mixed with 1.5 l of labeled DNA (45 fmol/l)\nand incubated for 30 min (final volume, 16.5 l). After the prerun for\n1 h, the samples were subjected to 5% acrylamide/bisacrylamide (40:1.2)\ngel. The gel was run at 4 °C in TE buffer (10 mM Tris-HCl, pH 7.2, 1 mM\nEDTA) at 110 V, and every 30 min the buffer in the upper chamber and\nthe buffer in the bottom chamber were exchanged. The dried gel was\nexposed for autoradiography and analyzed by phosphorimaging. The\nconcentration of OmpR-P required for 50% binding to DNA is pre-\nsented as the C50 value.\nDNase I Footprinting—Each purified 30-pmol sense oligonucleotide\nwas labeled at 5-end by using T4 kinase and [-32P]ATP (3000\nCi/mmol and 10 mCi/ml) for 1 h at 37 °C.Free [ -32P]ATP was removed\nby phenol/chloroform extraction followed by ethanol precipitation.\nLabeled oligonucleotide was annealed by mixing its complementary\nsequence (40 pmol) in 50 l of the annealing buffer (25 mM Tris-HCl,\npH 8.0, and 400 mM NaCl).\nThe DNA binding reaction was carried out as described in DNA gel\nmobility shift assay by mixing OmpR or OmpRc with 1 l of labeled\nDNA (600 fmol/l); however, 50 mM Tris-HCl (pH 8.0) buffer con-\ntaining 50 mM KCl, 10 mM MgCl2, and 1 mM ATP, was used. Partial\nDNase I digestion was carried out at room temperature by adding 1 l of\n0.2 unit/l DNase I (Promega) into the reaction mixture (final reaction\nvolume, 10 l). After 1-min incubation, the reaction was stopped by\nadding 1 l of 200 mM EDTA, followed by incubating in the boiling\nwater bath for 4 min. 6 l of the loading buffer (95% formamide, 20 mM\nEDTA, 0.05% bromphenol blue, 0.05% xylene cyanol FF) was added to\nthe sample. The samples were incubated in the boiling water bath for 4\nmin, and 2.5 l of the sample was subjected to 12% acrylamide/bisac-\nrylamide (19: 1), 8 M urea gel after prerun for 30 min. The gel was run in\nTBE buffer (50 mM Tris-HCl, 50 mM boric acid, and 1 mM EDTA) at\n45 W. The residues were positioned with the A  G ladders.\nRESULTS\nHierarchical Binding of OmpR-P to F1 Subsites—Each 20-bp F- and\nC-binding site for OmpR-P is composed of two 10-bp repetitive sub-\nsites, the a-site and the b-site, as shown in Fig. 1A. Earlier PCR selection\nof nine randomized sequences of the 3-end of the F1 site revealed a\nconsensus sequence of 12TTACAXXXX20 (the numbers indicate the\nbase positions in the 20-bp F1 site from the 5-end), in which the 14AC15\nsequence is tightly conserved (11). In addition to this consensus\nsequence, it is also known that the highly conserved G at the 11th posi-\ntion is important for OmpR-P binding (11, 17). Compared with these\nconsensus sequences, the 10-bp F1b site, the 3-end of the F1 site\n(GTTACATATT), contains all six consensus bases, whereas the 10-bp\nF1a site, the 5-end of the F1 site (TTTACTTTTG), contains four of six\nconsensus bases (Fig. 1A), suggesting that OmpR-P may have higher\naffinity for the F1b site than for the F1a site. To examine this possibility,\ntwo 20-mer DNA fragments, F1aF1a and F1bF1b, were synthesized.\nNote that most of the short constructs in the present study have four\nC:G base pair extensions (CCCC) at both the 5- and 3-ends to stabilize\nduplex formation as shown in Fig. 1B.\nAs shown in Fig. 1B, binding to F1aF1a was barely detectable even at\nthe highest concentration of OmpR-P used (80 nM; lane 12), whereas\nalmost 90% of F1bF1b was bound at a concentration as low as 5 nM\nOmpR-P (lane 14). With the wild-type F1aF1b fragment, 50% binding to\nOmpR-P was observed between 5 and 10 nM OmpR-P (lanes 2 and 3,\nrespectively). These observations are consistent with the previously\nreported Kd value of OmpR-P binding to the F1 site (10). These results\nindicate that OmpR-P binding to the F1a site in the F1 site depends on\nthe initial binding of OmpR-P to the F1b site because OmpR-P is unable\nto bind to F1aF1a under the same condition. However, binding of one\nOmpR-P molecule to the F1b site is not enough to form a stable complex\nwith the F1 site, because the mutation of the F1a site abolishes the\nbinding of OmpR-P to the F1 site (11). Therefore, for OmpR-P to form\na stable complex with the F1 site, binding of two OmpR-P molecules to\nboth F1a and F1b sites is necessary. In this sense the binding of two\nOmpR-P molecules to the F1 site is cooperative.\nWe also carried out DNase I footprinting using the same 28-bp oli-\ngonucleotides used in the gel shift assay to monitor the binding of\nOmpR-P to the F1a and the F1b site. Because the amount of OmpR-P\nwas increased, both the F1a and F1b sites in the wild-type F1 site\n(F1aF1b) were protected from DNase I digestion, in a cooperative man-\nner (Fig. 1C, lanes 1– 6). The protection of F1aF1a by OmpR-P (lanes\n7–12) was significantly less compared with that of F1bF1b (lanes\n13–18). This supports the hierarchy in the binding affinity of OmpR-P\nfor these sequences (F1bF1b  F1aF1b  F1aF1a) consistent with the\nresults of the gel mobility shift assay (Fig. 1B).\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nG1 and 4ACA6 Are Essential for Optimal Binding of OmpR-P—We\nnoticed that the G residue is highly conserved and is located at every\n11th position of each F and C site except in the case of the F4 site (Fig.\n1A). Because G11 has been shown to be important for OmpR-P binding\nto the F1 site (11), we next examined its role in OmpR-P binding using\nthe F1 and C1 sites. Because the F1 site has two G residues in the center\nat positions 10 and 11 (Fig. 1A), we tested which G residue contributes\nmore to OmpR-P/F1 site interaction. Substitution of G10 (G10A; Fig.\n2A, lanes 13–18) hardly affected the binding of OmpR-P to the DNA\nfragment as compared with the wild-type F1 sequence (lanes 1– 6). On\nthe other hand, the G11A mutation weakened the binding with a C50\nvalue of 80 nM (lanes 7–12), 10 times greater than the C50 value\nobserved with the wild-type F1 fragment. Mutation of both G residues\nfurther destabilized the binding of\n(G10A/G11A;\nOmpR-P, resulting in a C50 value of higher than 80 nM. These results\nindicate that contribution of the G11 residue is more significant in\nOmpR-P/F1 site interaction than that of the G10 residue.\nlanes 19 –24)\nWe performed a similar experiment with the C1 site, which has a\nsingle G residue at position 11 (Fig. 1A). When G11 was substituted with\neither C (Fig. 2B, lanes 6 –10) or T (lanes 16 –19), OmpR-P binding was\nsignificantly affected. These results further support the importance of\nthe G11 residue both in F1 and C1 sites in OmpR-P binding. Interest-\ningly, in the absence of G at position 11, a second mutation at position\n10, T10G, elevated the affinity for OmpR-P (Fig. 2B, lanes 11–15), albeit\nweaker than wild type (lanes 1–5). This suggests that similar to the F1\nsite, the G residue at position 10 in the C1 site can also interact with\nOmpR-P.\nBoth the G1 and A6 residues are absent in the F1a subsite (Fig. 1A).\nTherefore, we examined whether OmpR-P can bind to F1aF1a, if G1,\nA6, or both are introduced to mimic the F1b subsite. Indeed, when the\nT16A mutation was introduced in to create the ACA consensus\nsequence in the second F1a subsite, OmpR-P binding was substantially\nimproved with a C50 value of 10 nM (Fig. 2C, lanes 7–12). Similarly, the\nT11G mutation also improved the C50 value to 10 nM (lanes 13–18).\nHowever, interestingly, if these mutations were combined together,\nthey exerted a synergistic effect as the C50 value was lowered to less than\n5 nM as can be seen with the T11G/T16A mutations (lanes 19 –24),\nreaching a C50 value similar to that with the F1bF1b fragment (Fig. 1B,\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nlanes 13–18). These results suggest that the G residue at position 1 of a\nsubsite and the ACA consensus sequence are key for OmpR-P binding\nand are sufficient to elevate the affinity of F1aF1a to that of F1aF1b (Fig.\n2A, lanes 1– 6).\nBecause the G11 residue is more important than G10, we speculated\nthat having the G residue as the first position in each tandem repeated\nsubsite is critical. Therefore, we further examined the role of the G\nresidue for OmpR-P binding using an F1bF1b fragment. This fragment\nhas the G residue at the first position of each subsite at G1 and G11,\nrespectively. As shown in Fig. 2D, the G11A mutation slightly increased\nthe C50 value. The G1A/G11A double mutant further increased the C50\nvalue to 20 nM (lanes 13–18). This result indicates that the G residue\nlocated at the first position of each subsite is important for OmpR-P\nbinding.\nImportance of the C5 and G11 Residues on the Upper Strand for\nOmpR-P Binding—In all F and C b-sites, the C:G pair at the fifth posi-\ntion is highly conserved, and it also exists in some of the a-sites (Fig. 1A).\nIt has been shown that mutation of this C:G pair abolishes the binding of\nOmpR-P (11). When the C:G pair at the 5th position in F1 (F1aF1b) was\nsubstituted with an A:T pair, binding to OmpR-P became undetectable\nunder the conditions used (700 nM OmpR-P) (Fig. 3A, compare lanes 2\nand 4). By exchanging one strand between the two duplexes used above,\ntwo other heteroduplexes having a C:T or A:G mismatched base pair at\nposition 5 in the F1 site were formed. OmpR-P was still able to bind to\nthe DNA fragment with the C:T mismatched pair (lane 6) but not with\nthe A:G pair (lane 8), indicating that in the conserved C:G pair at the 5th\nposition in the a-site, it is the C residue in the upper strand that is\nessential for OmpR-P binding; this is most likely the case for the b-site as\nwell.\nA similar experiment was carried out to elucidate which residue in\nthe G:C pair at the 11th position plays an important role for OmpR-P\nbinding using the C1 site. When the G:C pair was substituted with the\nC:G or T:A pair, binding was very weak, even at concentrations of\nOmpR-P as high as 190 nM (Fig. 2B), suggesting that the G residue in this\nG:C pair is important for binding. Using the wild-type and mutated\noligonucleotides, two heteroduplex DNA fragments were constructed.\nOmpR-P was still able to bind to the C1 fragment with the G:T mis-\nmatched pair at the 11th position (Fig. 3B, lanes 6 –10) but not with the\nA:C pair (lanes 16 –20). This confirmed that similar to the C:G pair at\nthe 5th position, the G residue on the upper strand of the G:C pair at\nthe 11th position is involved in OmpR-P binding. The replacement\nof the G residue with an Ile residue had little effect on OmpR-P\nbinding (lanes 11–15), suggesting that the carbonyl group at position 6\nof the purine ring may be directly involved in the interaction with\nOmpR-P.\nNonphosphorylated OmpR Is Capable of Binding to F1bF1b—Non-\nphosphorylated OmpR is capable of binding to the F1 sequence with low\naffinity (more than a 10-fold difference in the Kd value as compared with\ninterestingly, nonphosphorylated\nthat of OmpR-P (10)). However,\nOmpR was able to bind to the F1bF1b fragment with a C50 value of 10\nnM (Fig. 4A, lanes 19 –24), which is very similar to the binding of OmpR-P\nto F1 (lanes 1– 6). On the other hand, the affinity of OmpR to the F1 site was\nvery weak, and its C50 value appears to be higher than 80 nM (lanes 13–18).\nImportantly, the high affinity of OmpR for the F1bF1b fragment can be\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nfurther enhanced by its phosphorylation because more than 90% of the\nDNA is bound at 5 nM concentration of OmpR-P (lane 8).\nF1 fragment and are consistent with the fact that OmpR-P has higher\naffinity for the F1b site than for the F1a site.\nThe N-terminal Domain Is Required for Cooperative Binding of OmpR\nto the F1 Site—We next compared the DNA binding activity of the\nC-terminal DNA-binding domain of OmpR (OmpRc) with that of\nOmpR and OmpR-P by DNase I footprinting. OmpR was able to protect\nboth the F1a and F1b sites from DNase I digestion in a cooperative\nmanner (Fig. 4B, lanes 2– 6). When the F1bF1b fragment was used, the\nprotection of both F1b sites was dramatically increased (lanes 13–17)\n(compare the F1b site in the F1aF1b fragment and the upper F1b site in\nthe F1bF1b site). Phosphorylation of OmpR further enhanced protec-\ntion of the F1aF1b (lanes 7–11) and F1bF1b (lanes 18 –22) fragments.\nThese results using DNase I footprinting and a 28-bp DNA fragment are\nin accordance with those obtained using the gel shift assay for OmpR-P\nbinding to DNA (Fig. 4A).\nTherefore, DNase I footprinting experiments were then carried out\nusing the F1, F1aF1a, or F1bF1b fragment with OmpRc. Interestingly,\nthe F1b site in the F1 site was only protected at high concentrations of\nOmpRc (Fig. 4C, lanes 1– 6), and the F1a site was not protected at all\neven at the highest OmpRc concentration (lane 6). Consistent with this\nresult, both F1a sites in the F1aF1a fragment were poorly protected from\nDNase I digestion (lanes 7–12), whereas both F1b sites in the F1bF1b\nfragment were well protected (lanes 13–18), confirming that OmpRc\nhas higher affinity to F1b than to F1a. These results indicate that the\nN-terminal domain is required for OmpR to bind to the F1a site in the\nOmpR-P Binding to DNA: a Two-by-Two Mode—We previously\nshowed that only an even number of OmpR-P molecules can bind to the\nF1-F2-F3 fragment, because the OmpR-PDNA complexes detected\ncontained only two, four, or six OmpR-P molecules (11). To further\nexamine this two-by-two mode of OmpR-P binding to DNA, we syn-\nthesized an oligonucleotide consisting of three tandem F1b sequences,\nF1bF1bF1b. As a control, another oligonucleotide, F1-F2a (F1aF1bF2a),\nwas synthesized that was previously shown to bind only two OmpR-P\nmolecules (11).\nTo accurately estimate the number of OmpR-P molecules binding to\nthe DNA fragments, mixtures of OmpR-P and PrS2-OmpR-P were used\n(11). PrS2-OmpR is a hybrid protein, in which two tandemly repeated\nN-terminal domains (a total of 184 amino acid residues) from Myxococ-\ncus xanthus protein S is linked to the N-terminal end of the OmpR\nmolecule. The resulting PrS2-OmpR is 20 kDa larger than OmpR.\nImportantly, this hybrid protein exhibited binding activity similar to\nthat of OmpR-P upon phosphorylation with EnvZc (11).\nAs shown in lanes 1– 8 of Fig. 5, only three complexes were detectable\nwith increasing concentrations of PrS2-OmpR-P and concomitantly\ndecreasing concentrations of OmpR-P. The proteinDNA complex with\nthe fastest mobility contained only OmpR-P molecules, whereas the\nslowest mobility complex contained only PrS2-OmpR-P molecules.\nTherefore, the center complex should contain one molecule each of\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nOmpR-P and PrS2-OmpR-P. We conclude that only two OmpR-P mol-\necules are able to bind to the F1F2a fragment.\nined OmpR-P binding to all of the other F and C sites using the same\nstrategy used for the F1 site.\nWhen a similar experiment was carried out with the F1bF1bF1b frag-\nment, again only three complexes were detectable (Fig. 5, lanes 9 –16).\nThis result demonstrates that even if a DNA fragment contains three\nhigh affinity OmpR-P-binding sites, not more than two OmpR-P mol-\necules can bind to the fragment, likely in two different modes as illus-\ntrated in Fig. 5A.\nAll b-sites Have Higher Affinity for OmpR-P than Their Cognate\nUpstream a-sites—Thus far we demonstrated that there is a distinct\nOmpR-P binding hierarchy between F1a and F1b subsites consistent\nwith the F1b sequence having the consensus G1 and ACA residues\nrelative to F1a. A similar tendency is observed in all F and C sites without\nexception (Fig. 1A), indicating that all of the downstream b-sites have a\nhigher affinity than their cognate upstream a-sites. Thus, we next exam-\nWe synthesized 40-bp F1-F2aF2a and F1-F2bF2b for studying the F2\nsite and 60-bp F1-F2-F3aF3a and F1-F2-F3bF3b for studying the F3 site.\nNote that to detect the binding of OmpR-P to the F2 and F3 sites, its\nbinding to the F1 site and the F1-F2 site, respectively, was required.\nBinding of OmpR-P to the F1-F2aF2a fragment was significantly\nreduced (Fig. 6A, lanes 8 –14), because even at the highest concentration\nof OmpR-P used (lane 14), the binding of four OmpR-P molecules to\nthis fragment was marginal in comparison with that to the wild-type\nF1-F2 fragment (lane 7). Surprisingly, the binding of two OmpR-P mol-\necules to the F1 site in this F1-F2aF2a was also affected. However, this\nreduction of OmpR-P binding to the F1 site in F1-F2aF2a was abolished\nby a point mutation in either highly conserved C5 (in first F2a site) or\nC15 (in second F2a site) in F1-F2aF2a (data not shown), suggesting that\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nsomehow the weak binding of OmpR-P to F2aF2a negatively affects the\noverall stability of complex formation between four OmpR-P molecules\nand F1-F2aF2a. On the other hand, binding of OmpR-P to the\nF1-F2bF2b fragment was similar to (or slightly better than) its binding to\nthe F1-F2 fragment, resulting in highly cooperative binding of four\nOmpR-P molecules to this fragment (lanes 16 –21). These results dem-\nonstrate that F2b has a higher affinity to OmpR-P than to F2a. Similarly,\nsix molecules of OmpR-P exhibited greater cooperative binding to the\nF1-F2-F3bF3b fragment (Fig. 6B, lanes 17–24), relative to the F1-F2-\nF3aF3a fragment (lanes 9 –16), again supporting that OmpR-P has a\nhigher affinity to F3b than to F3a.\nSimilar experiments were carried out with C2 (Fig. 6C) and C3 (Fig. 6D)\nsites, demonstrating that OmpR-P has a higher affinity to the C2bC2b\nsequence or the C3bC3b sequence than to the C2aC2a sequence or the\nC3aC3a sequence, respectively. Interestingly, OmpR-P binding to either\nC2aC2a or C3aC3a sequences was completely abolished, because com-\nplexes with the slowest mobility were not detected at the positions indi-\ncated by “4 OmpR-P” for C1-C2aC2a (Fig. 6C, lanes 9 –14) and by “6\nOmpR-P” for C1-C2-C3aC3a (Fig. 6D, lanes 9 –16), respectively. In con-\ntrast, C2bC2b in C1-C2 (Fig. 6C, lanes 16 –21) significantly improved\ncooperative binding of OmpR-P compared with the C1-C2 (lanes 1–7).\nA similar effect was also observed with C3bC3b in C1-C2-C3 (Fig. 6D,\nlanes 17–24).\nInterestingly, if the order of F1a and F1b sites in F1-F2 was reversed to\nthat of F1bF1a-F2, four OmpR-P molecules were still able to bind to the\nDNA fragment; however their cooperative binding (Fig. 6E, lanes 8 –14)\nwas poorer relative to the wild-type F1-F2 fragment (lanes 1–7), indi-\ncating that having the site for which OmpR-P has weaker affinity (F1a)\nupstream of the site for which it has stronger affinity (F1b) is important\nfor the most effective two-by-two OmpR-P binding mechanism.\nThese results unambiguously established the hierarchies for OmpR-P\nbinding affinity in all F and C sites as follows: F1b  F1a, F2b  F2a,\nF3b  F3a, C1b  C1a, C2b  C2a, and C3b  C3a. Furthermore, the\npresent results indicate that OmpR-P has different affinities for a-sites\nin F2 and F3 than for the a-sites in C2 and C3. This may have important\nimplications in the reciprocal regulation of ompF and ompC genes as\ndiscussed below.\nDISCUSSION\nHierarchical Arrangement of Two OmpR-P-binding Subsites—It has\nbeen shown that two OmpR-P molecules cooperatively bind to the",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "The ompF and ompC genes of Escherichia coli are reciprocally\nregulated by a single transcription factor, phosphorylated OmpR\n(OmpR-P), depending upon medium osmolarity. This regulation\ninvolves activation of ompF and its repression with concomitant\nactivation of ompC. This occurs through OmpR-P binding to four\n(F1, F2, F3, and F4) and three (C1, C2, and C3) sites located\nupstream of the ompF and ompC promoters, respectively, through a\nnovel mechanism. Here we show that there is a distinct OmpR-P\nbinding hierarchy within F1, F2, and F3 sites as well as within C1,\nC2, and C3 sites. Each of these sites contains two tandem 10-bp\nOmpR-P-binding subsites, a-site and b-site (from 5 to 3 direction).\nOmpR-P has higher affinity to the downstream b-site than to the\nupstream a-site in each case. Six OmpR-P molecules bind to F and C\nsites two-by-two in a discontinuous “galloping” manner. We pro-\npose that this tight hierarchical binding of a transcription factor,\nOmpR, allows distinct stepwise regulation of ompF and ompC tran-\nscription, which minimizes their overlapping expression upon\nchanges in the medium osmolarity to achieve the reciprocal expres-\nsion of ompF and ompC.\nEscherichia coli contains two outer membrane porin proteins, OmpF\nand OmpC, which serve as passive diffusion pores across the outer\nmembrane. Expression of these two proteins is reciprocally regulated by\nmedium osmolarity. OmpF is preferentially produced at low osmolarity,\nwhereas OmpC is almost exclusively produced at high osmolarity.\nBecause OmpF forms a larger pore than OmpC (1), the regulation of the\nsize of the passive diffusion pores across the outer membrane appears to\nbe vital for E. coli, which is able to live in two distinctly different envi-\nronmental conditions: one under nutrient-rich high osmolarity condi-\ntions in animal gut at 37 °C and the other under poor nutritional and low\nosmolarity conditions outside animals and thus at low osmolarity at\nambient temperatures. It has been proposed that the larger OmpF pore\nsize is important for efficient nutrient uptake from nutritionally poor\nmedia, whereas the smaller OmpC pore size is important to exclude the\npassage of toxic bile salt across the outer membrane (2). E. coli has a\nhighly sophisticated regulatory system to regulate the reciprocal expres-\nsion of ompF and ompC. Both are controlled at the transcriptional level\nby the histidine kinase EnvZ, a transmembrane osmosensor, and the\nresponse regulator OmpR, a transcriptional factor (3), and also at the\ntranslational level by antisense RNAs, micF RNA for ompF mRNA (4)\nand micC RNA for ompC mRNA (5).\n* This work was supported by Grant GM076587 from the National Institutes of Health.\nThe costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked “advertisement” in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\nThe EnvZ-OmpR system is a typical two-component His-Asp phos-\nphorelay signal transduction system in E. coli (see review in Ref. 6).\nBecause EnvZ not only has kinase activity to phosphorylate OmpR but\nalso phosphatase activity toward phosphorylated OmpR (OmpR-P), it is\nable to regulate the level of OmpR-P in the cell depending upon the\nmedium osmolarity. At low medium osmolarity, the phosphatase activ-\nity is relatively higher than the kinase activity so that the cellular con-\ncentration of OmpR-P is maintained at a lower concentration, which is\nsufficient to activate expression of ompF but not of ompC. At high\nmedium osmolarity, the EnvZ phosphatase activity becomes relatively\nlower than its kinase activity, resulting in higher cellular concentration\nof OmpR-P. Under this condition, ompF expression is repressed,\nwhereas ompC expression is activated.\nIn our recent study on the interaction between the cytoplasmic\ndomain of EnvZ (EnvZc) and OmpR or OmpR-P (7), we proposed that\nthe reciprocal expression of ompF and ompC occurs through a finely\ntuned mechanism. In this model, osmotic signals regulate the levels of\nOmpR-P by modulating the ratio of kinase to phosphatase activity of\nEnvZ. Most importantly, of the large pool of OmpR (3,500 molecules/\ncell), only a very small portion is proposed to be phosphorylated (8). On\nthe other hand, the Kd value for OmpR-P binding to the regulatory\nregions of the ompF and ompC promoters (9, 10) is less than 1% of the\ntotal cellular concentration of OmpR and OmpR-P. This was proposed\nto be a critical factor in the reciprocal expression of ompF and ompC.\nFor example, even with a minor change in the ratio of the kinase to\nphosphatase activity of EnvZ, the resulting small shift in the cellular\nconcentration of OmpR-P would be sufficient to substantially change\nthe number of OmpR-P molecules bound to the regulatory regions of\nthe ompF and ompC promoters. There is approximately a few hundred-\nfold difference between the Kd value for OmpR-P binding to the regu-\nlatory regions of the ompF and ompC promoters versus the total cellular\nconcentration of OmpR and OmpR-P (7).\nThere are four F sites (F1, F2, F3, and F4) and three C sites (C1, C2,\nand C3) in the upstream region of the ompF and ompC promoters,\nrespectively. Each site consists of 20 base pairs, providing a tandem\nbinding site for two OmpR-P molecules (11, 12). The hierarchy of\nOmpR-P binding to these F and C sites was determined to be F1, C1 \nF2, F3  C2  C3 (9). At low osmolarity, OmpR-P cooperatively binds\nto F1-F2/F1-F2-F3 to activate ompF transcription. Under this condition,\nonly the C1 site is occupied by OmpR-P, which is not sufficient to\nactivate ompC transcription. Only when the OmpR-P concentration\nelevates in response to increased medium osmolarity do the C2 and C3\nsites become occupied by OmpR-P, resulting in ompC expression.\nUnder this condition, OmpR-P also binds to the F4 site (380 to 361),\na weak OmpR-P-binding site located 260-bp upstream of the F1 site\n(100 to 81). OmpR-P binding to the F4 site is proposed to form a\nloop that interacts with OmpR-P molecules binding to F1, F2, and F3,\nthereby blocking ompF transcription (9, 13, 14).\nHere, we investigated how two OmpR-P molecules cooperatively\nbind to F1, F2, and F3 as well as C1, C2, and C3. The 20-bp F1 site is\nHierarchical Binding of OmpR-P Regulates ompF and ompC\ncomposed of two 10-bp subsites, F1a and F1b (from 5 to 3). We found\nthat the affinity of OmpR-P for the upstream F1a is much weaker than\nthat for the downstream F1b, suggesting binding to the F1 site is initi-\nated by OmpR-P interaction with the F1b subsite. Similarly, OmpR-P\nhad a higher affinity for the downstream subsites, F2b and F3b for F2\nand F3 sites, respectively, than their cognate upstream subsites, F2a and\nF3a. Therefore, OmpR-P binding to F1-F2-F3 occurs discontinuously in\na stepwise manner in the order F1b to F1a, F2b to F2a, and F3b to F3a.\nSimilarly, OmpR-P binds to C1-C2-C3 in this discontinuous manner.\nWe termed this novel mode of OmpR-P binding to the regulatory\nregions as the “galloping model.” This mechanism is inherent to the distinct\nreciprocal expression of ompF and ompC minimizes their overlapping\nexpression under a given condition. Because OmpR belongs to the largest\nresponse regulator family (OmpR/PhoB) in E. coli, the DNA binding of\ntranscription factors in galloping mode may be widely found in E. coli.\nEXPERIMENTAL PROCEDURES\nOligonucleotides—The OmpR-binding sites of F (F1, F1-F2, and F1-F2-\nF3), C (C1, C1-C2, and C1-C2-C3) and various mutated sequences were\nsynthesized by Integrated DNA Technologies, Inc. Complimentary strands\nwere first annealed, and double-stranded DNA was purified for DNA gel\nmobility shift assay. DNA concentration was determined by UV absorb-\nance at 260 nm.\nPurification of Proteins—EnvZc[T247R] previously termed EnvZc11,\nOmpR, and PrS2-OmpR were purified as described previously (11, 15).\nThe C-terminal domain of OmpR (OmpRc) was purified as described\npreviously (16).\nDNA Gel Mobility Shift Assay—Each purified 20 pmol of double-\nstranded DNA was labeled by using T4 kinase and [-32P]ATP (3000\nCi/mmol and 10 mCi/ml) for 1 h at 37 °C.Free [ -32P]ATP was removed\nby phenol/chloroform extraction followed by ethanol precipitation.\nAfter air drying, labeled DNA was resuspended in 100 l of water con-\ntaining 70 pmol of unlabeled double-stranded DNA.\nThe binding reactions were carried out in 50 mM Tris-HCl (pH 7.2)\nbuffer containing 50 mM KCl, 5 mM CaCl2, 5% glycerol, 0.05% Nonidet\nP-40, and 100 ng of poly(dI-dC) at room temperature. First, EnvZc11\nwas phosphorylated using 1 mM ATP for 5 min. EnvZc11 mixture was\ntransferred to the tube containing different amounts of OmpR or PrS2-\nOmpR, and the reaction mixture was further incubated for 20 min to\ngenerate phosphorylated OmpR or PrS2-OmpR. Under this condition,\n100% of OmpR or PrS2-OmpR was phosphorylated as judged from C4\nreverse phase high pressure liquid chromatography. Fifteen l of the\nreaction mixture was mixed with 1.5 l of labeled DNA (45 fmol/l)\nand incubated for 30 min (final volume, 16.5 l). After the prerun for\n1 h, the samples were subjected to 5% acrylamide/bisacrylamide (40:1.2)\ngel. The gel was run at 4 °C in TE buffer (10 mM Tris-HCl, pH 7.2, 1 mM\nEDTA) at 110 V, and every 30 min the buffer in the upper chamber and\nthe buffer in the bottom chamber were exchanged. The dried gel was\nexposed for autoradiography and analyzed by phosphorimaging. The\nconcentration of OmpR-P required for 50% binding to DNA is pre-\nsented as the C50 value.\nDNase I Footprinting—Each purified 30-pmol sense oligonucleotide\nwas labeled at 5-end by using T4 kinase and [-32P]ATP (3000\nCi/mmol and 10 mCi/ml) for 1 h at 37 °C.Free [ -32P]ATP was removed\nby phenol/chloroform extraction followed by ethanol precipitation.\nLabeled oligonucleotide was annealed by mixing its complementary\nsequence (40 pmol) in 50 l of the annealing buffer (25 mM Tris-HCl,\npH 8.0, and 400 mM NaCl).\nThe DNA binding reaction was carried out as described in DNA gel\nmobility shift assay by mixing OmpR or OmpRc with 1 l of labeled\nDNA (600 fmol/l); however, 50 mM Tris-HCl (pH 8.0) buffer con-\ntaining 50 mM KCl, 10 mM MgCl2, and 1 mM ATP, was used. Partial\nDNase I digestion was carried out at room temperature by adding 1 l of\n0.2 unit/l DNase I (Promega) into the reaction mixture (final reaction\nvolume, 10 l). After 1-min incubation, the reaction was stopped by\nadding 1 l of 200 mM EDTA, followed by incubating in the boiling\nwater bath for 4 min. 6 l of the loading buffer (95% formamide, 20 mM\nEDTA, 0.05% bromphenol blue, 0.05% xylene cyanol FF) was added to\nthe sample. The samples were incubated in the boiling water bath for 4\nmin, and 2.5 l of the sample was subjected to 12% acrylamide/bisac-\nrylamide (19: 1), 8 M urea gel after prerun for 30 min. The gel was run in\nTBE buffer (50 mM Tris-HCl, 50 mM boric acid, and 1 mM EDTA) at\n45 W. The residues were positioned with the A  G ladders.\nRESULTS\nHierarchical Binding of OmpR-P to F1 Subsites—Each 20-bp F- and\nC-binding site for OmpR-P is composed of two 10-bp repetitive sub-\nsites, the a-site and the b-site, as shown in Fig. 1A. Earlier PCR selection\nof nine randomized sequences of the 3-end of the F1 site revealed a\nconsensus sequence of 12TTACAXXXX20 (the numbers indicate the\nbase positions in the 20-bp F1 site from the 5-end), in which the 14AC15\nsequence is tightly conserved (11). In addition to this consensus\nsequence, it is also known that the highly conserved G at the 11th posi-\ntion is important for OmpR-P binding (11, 17). Compared with these\nconsensus sequences, the 10-bp F1b site, the 3-end of the F1 site\n(GTTACATATT), contains all six consensus bases, whereas the 10-bp\nF1a site, the 5-end of the F1 site (TTTACTTTTG), contains four of six\nconsensus bases (Fig. 1A), suggesting that OmpR-P may have higher\naffinity for the F1b site than for the F1a site. To examine this possibility,\ntwo 20-mer DNA fragments, F1aF1a and F1bF1b, were synthesized.\nNote that most of the short constructs in the present study have four\nC:G base pair extensions (CCCC) at both the 5- and 3-ends to stabilize\nduplex formation as shown in Fig. 1B.\nAs shown in Fig. 1B, binding to F1aF1a was barely detectable even at\nthe highest concentration of OmpR-P used (80 nM; lane 12), whereas\nalmost 90% of F1bF1b was bound at a concentration as low as 5 nM\nOmpR-P (lane 14). With the wild-type F1aF1b fragment, 50% binding to\nOmpR-P was observed between 5 and 10 nM OmpR-P (lanes 2 and 3,\nrespectively). These observations are consistent with the previously\nreported Kd value of OmpR-P binding to the F1 site (10). These results\nindicate that OmpR-P binding to the F1a site in the F1 site depends on\nthe initial binding of OmpR-P to the F1b site because OmpR-P is unable\nto bind to F1aF1a under the same condition. However, binding of one\nOmpR-P molecule to the F1b site is not enough to form a stable complex\nwith the F1 site, because the mutation of the F1a site abolishes the\nbinding of OmpR-P to the F1 site (11). Therefore, for OmpR-P to form\na stable complex with the F1 site, binding of two OmpR-P molecules to\nboth F1a and F1b sites is necessary. In this sense the binding of two\nOmpR-P molecules to the F1 site is cooperative.\nWe also carried out DNase I footprinting using the same 28-bp oli-\ngonucleotides used in the gel shift assay to monitor the binding of\nOmpR-P to the F1a and the F1b site. Because the amount of OmpR-P\nwas increased, both the F1a and F1b sites in the wild-type F1 site\n(F1aF1b) were protected from DNase I digestion, in a cooperative man-\nner (Fig. 1C, lanes 1– 6). The protection of F1aF1a by OmpR-P (lanes\n7–12) was significantly less compared with that of F1bF1b (lanes\n13–18). This supports the hierarchy in the binding affinity of OmpR-P\nfor these sequences (F1bF1b  F1aF1b  F1aF1a) consistent with the\nresults of the gel mobility shift assay (Fig. 1B).\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nG1 and 4ACA6 Are Essential for Optimal Binding of OmpR-P—We\nnoticed that the G residue is highly conserved and is located at every\n11th position of each F and C site except in the case of the F4 site (Fig.\n1A). Because G11 has been shown to be important for OmpR-P binding\nto the F1 site (11), we next examined its role in OmpR-P binding using\nthe F1 and C1 sites. Because the F1 site has two G residues in the center\nat positions 10 and 11 (Fig. 1A), we tested which G residue contributes\nmore to OmpR-P/F1 site interaction. Substitution of G10 (G10A; Fig.\n2A, lanes 13–18) hardly affected the binding of OmpR-P to the DNA\nfragment as compared with the wild-type F1 sequence (lanes 1– 6). On\nthe other hand, the G11A mutation weakened the binding with a C50\nvalue of 80 nM (lanes 7–12), 10 times greater than the C50 value\nobserved with the wild-type F1 fragment. Mutation of both G residues\nfurther destabilized the binding of\n(G10A/G11A;\nOmpR-P, resulting in a C50 value of higher than 80 nM. These results\nindicate that contribution of the G11 residue is more significant in\nOmpR-P/F1 site interaction than that of the G10 residue.\nlanes 19 –24)\nWe performed a similar experiment with the C1 site, which has a\nsingle G residue at position 11 (Fig. 1A). When G11 was substituted with\neither C (Fig. 2B, lanes 6 –10) or T (lanes 16 –19), OmpR-P binding was\nsignificantly affected. These results further support the importance of\nthe G11 residue both in F1 and C1 sites in OmpR-P binding. Interest-\ningly, in the absence of G at position 11, a second mutation at position\n10, T10G, elevated the affinity for OmpR-P (Fig. 2B, lanes 11–15), albeit\nweaker than wild type (lanes 1–5). This suggests that similar to the F1\nsite, the G residue at position 10 in the C1 site can also interact with\nOmpR-P.\nBoth the G1 and A6 residues are absent in the F1a subsite (Fig. 1A).\nTherefore, we examined whether OmpR-P can bind to F1aF1a, if G1,\nA6, or both are introduced to mimic the F1b subsite. Indeed, when the\nT16A mutation was introduced in to create the ACA consensus\nsequence in the second F1a subsite, OmpR-P binding was substantially\nimproved with a C50 value of 10 nM (Fig. 2C, lanes 7–12). Similarly, the\nT11G mutation also improved the C50 value to 10 nM (lanes 13–18).\nHowever, interestingly, if these mutations were combined together,\nthey exerted a synergistic effect as the C50 value was lowered to less than\n5 nM as can be seen with the T11G/T16A mutations (lanes 19 –24),\nreaching a C50 value similar to that with the F1bF1b fragment (Fig. 1B,\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nlanes 13–18). These results suggest that the G residue at position 1 of a\nsubsite and the ACA consensus sequence are key for OmpR-P binding\nand are sufficient to elevate the affinity of F1aF1a to that of F1aF1b (Fig.\n2A, lanes 1– 6).\nBecause the G11 residue is more important than G10, we speculated\nthat having the G residue as the first position in each tandem repeated\nsubsite is critical. Therefore, we further examined the role of the G\nresidue for OmpR-P binding using an F1bF1b fragment. This fragment\nhas the G residue at the first position of each subsite at G1 and G11,\nrespectively. As shown in Fig. 2D, the G11A mutation slightly increased\nthe C50 value. The G1A/G11A double mutant further increased the C50\nvalue to 20 nM (lanes 13–18). This result indicates that the G residue\nlocated at the first position of each subsite is important for OmpR-P\nbinding.\nImportance of the C5 and G11 Residues on the Upper Strand for\nOmpR-P Binding—In all F and C b-sites, the C:G pair at the fifth posi-\ntion is highly conserved, and it also exists in some of the a-sites (Fig. 1A).\nIt has been shown that mutation of this C:G pair abolishes the binding of\nOmpR-P (11). When the C:G pair at the 5th position in F1 (F1aF1b) was\nsubstituted with an A:T pair, binding to OmpR-P became undetectable\nunder the conditions used (700 nM OmpR-P) (Fig. 3A, compare lanes 2\nand 4). By exchanging one strand between the two duplexes used above,\ntwo other heteroduplexes having a C:T or A:G mismatched base pair at\nposition 5 in the F1 site were formed. OmpR-P was still able to bind to\nthe DNA fragment with the C:T mismatched pair (lane 6) but not with\nthe A:G pair (lane 8), indicating that in the conserved C:G pair at the 5th\nposition in the a-site, it is the C residue in the upper strand that is\nessential for OmpR-P binding; this is most likely the case for the b-site as\nwell.\nA similar experiment was carried out to elucidate which residue in\nthe G:C pair at the 11th position plays an important role for OmpR-P\nbinding using the C1 site. When the G:C pair was substituted with the\nC:G or T:A pair, binding was very weak, even at concentrations of\nOmpR-P as high as 190 nM (Fig. 2B), suggesting that the G residue in this\nG:C pair is important for binding. Using the wild-type and mutated\noligonucleotides, two heteroduplex DNA fragments were constructed.\nOmpR-P was still able to bind to the C1 fragment with the G:T mis-\nmatched pair at the 11th position (Fig. 3B, lanes 6 –10) but not with the\nA:C pair (lanes 16 –20). This confirmed that similar to the C:G pair at\nthe 5th position, the G residue on the upper strand of the G:C pair at\nthe 11th position is involved in OmpR-P binding. The replacement\nof the G residue with an Ile residue had little effect on OmpR-P\nbinding (lanes 11–15), suggesting that the carbonyl group at position 6\nof the purine ring may be directly involved in the interaction with\nOmpR-P.\nNonphosphorylated OmpR Is Capable of Binding to F1bF1b—Non-\nphosphorylated OmpR is capable of binding to the F1 sequence with low\naffinity (more than a 10-fold difference in the Kd value as compared with\ninterestingly, nonphosphorylated\nthat of OmpR-P (10)). However,\nOmpR was able to bind to the F1bF1b fragment with a C50 value of 10\nnM (Fig. 4A, lanes 19 –24), which is very similar to the binding of OmpR-P\nto F1 (lanes 1– 6). On the other hand, the affinity of OmpR to the F1 site was\nvery weak, and its C50 value appears to be higher than 80 nM (lanes 13–18).\nImportantly, the high affinity of OmpR for the F1bF1b fragment can be\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nfurther enhanced by its phosphorylation because more than 90% of the\nDNA is bound at 5 nM concentration of OmpR-P (lane 8).\nF1 fragment and are consistent with the fact that OmpR-P has higher\naffinity for the F1b site than for the F1a site.\nThe N-terminal Domain Is Required for Cooperative Binding of OmpR\nto the F1 Site—We next compared the DNA binding activity of the\nC-terminal DNA-binding domain of OmpR (OmpRc) with that of\nOmpR and OmpR-P by DNase I footprinting. OmpR was able to protect\nboth the F1a and F1b sites from DNase I digestion in a cooperative\nmanner (Fig. 4B, lanes 2– 6). When the F1bF1b fragment was used, the\nprotection of both F1b sites was dramatically increased (lanes 13–17)\n(compare the F1b site in the F1aF1b fragment and the upper F1b site in\nthe F1bF1b site). Phosphorylation of OmpR further enhanced protec-\ntion of the F1aF1b (lanes 7–11) and F1bF1b (lanes 18 –22) fragments.\nThese results using DNase I footprinting and a 28-bp DNA fragment are\nin accordance with those obtained using the gel shift assay for OmpR-P\nbinding to DNA (Fig. 4A).\nTherefore, DNase I footprinting experiments were then carried out\nusing the F1, F1aF1a, or F1bF1b fragment with OmpRc. Interestingly,\nthe F1b site in the F1 site was only protected at high concentrations of\nOmpRc (Fig. 4C, lanes 1– 6), and the F1a site was not protected at all\neven at the highest OmpRc concentration (lane 6). Consistent with this\nresult, both F1a sites in the F1aF1a fragment were poorly protected from\nDNase I digestion (lanes 7–12), whereas both F1b sites in the F1bF1b\nfragment were well protected (lanes 13–18), confirming that OmpRc\nhas higher affinity to F1b than to F1a. These results indicate that the\nN-terminal domain is required for OmpR to bind to the F1a site in the\nOmpR-P Binding to DNA: a Two-by-Two Mode—We previously\nshowed that only an even number of OmpR-P molecules can bind to the\nF1-F2-F3 fragment, because the OmpR-PDNA complexes detected\ncontained only two, four, or six OmpR-P molecules (11). To further\nexamine this two-by-two mode of OmpR-P binding to DNA, we syn-\nthesized an oligonucleotide consisting of three tandem F1b sequences,\nF1bF1bF1b. As a control, another oligonucleotide, F1-F2a (F1aF1bF2a),\nwas synthesized that was previously shown to bind only two OmpR-P\nmolecules (11).\nTo accurately estimate the number of OmpR-P molecules binding to\nthe DNA fragments, mixtures of OmpR-P and PrS2-OmpR-P were used\n(11). PrS2-OmpR is a hybrid protein, in which two tandemly repeated\nN-terminal domains (a total of 184 amino acid residues) from Myxococ-\ncus xanthus protein S is linked to the N-terminal end of the OmpR\nmolecule. The resulting PrS2-OmpR is 20 kDa larger than OmpR.\nImportantly, this hybrid protein exhibited binding activity similar to\nthat of OmpR-P upon phosphorylation with EnvZc (11).\nAs shown in lanes 1– 8 of Fig. 5, only three complexes were detectable\nwith increasing concentrations of PrS2-OmpR-P and concomitantly\ndecreasing concentrations of OmpR-P. The proteinDNA complex with\nthe fastest mobility contained only OmpR-P molecules, whereas the\nslowest mobility complex contained only PrS2-OmpR-P molecules.\nTherefore, the center complex should contain one molecule each of\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nOmpR-P and PrS2-OmpR-P. We conclude that only two OmpR-P mol-\necules are able to bind to the F1F2a fragment.\nined OmpR-P binding to all of the other F and C sites using the same\nstrategy used for the F1 site.\nWhen a similar experiment was carried out with the F1bF1bF1b frag-\nment, again only three complexes were detectable (Fig. 5, lanes 9 –16).\nThis result demonstrates that even if a DNA fragment contains three\nhigh affinity OmpR-P-binding sites, not more than two OmpR-P mol-\necules can bind to the fragment, likely in two different modes as illus-\ntrated in Fig. 5A.\nAll b-sites Have Higher Affinity for OmpR-P than Their Cognate\nUpstream a-sites—Thus far we demonstrated that there is a distinct\nOmpR-P binding hierarchy between F1a and F1b subsites consistent\nwith the F1b sequence having the consensus G1 and ACA residues\nrelative to F1a. A similar tendency is observed in all F and C sites without\nexception (Fig. 1A), indicating that all of the downstream b-sites have a\nhigher affinity than their cognate upstream a-sites. Thus, we next exam-\nWe synthesized 40-bp F1-F2aF2a and F1-F2bF2b for studying the F2\nsite and 60-bp F1-F2-F3aF3a and F1-F2-F3bF3b for studying the F3 site.\nNote that to detect the binding of OmpR-P to the F2 and F3 sites, its\nbinding to the F1 site and the F1-F2 site, respectively, was required.\nBinding of OmpR-P to the F1-F2aF2a fragment was significantly\nreduced (Fig. 6A, lanes 8 –14), because even at the highest concentration\nof OmpR-P used (lane 14), the binding of four OmpR-P molecules to\nthis fragment was marginal in comparison with that to the wild-type\nF1-F2 fragment (lane 7). Surprisingly, the binding of two OmpR-P mol-\necules to the F1 site in this F1-F2aF2a was also affected. However, this\nreduction of OmpR-P binding to the F1 site in F1-F2aF2a was abolished\nby a point mutation in either highly conserved C5 (in first F2a site) or\nC15 (in second F2a site) in F1-F2aF2a (data not shown), suggesting that\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nsomehow the weak binding of OmpR-P to F2aF2a negatively affects the\noverall stability of complex formation between four OmpR-P molecules\nand F1-F2aF2a. On the other hand, binding of OmpR-P to the\nF1-F2bF2b fragment was similar to (or slightly better than) its binding to\nthe F1-F2 fragment, resulting in highly cooperative binding of four\nOmpR-P molecules to this fragment (lanes 16 –21). These results dem-\nonstrate that F2b has a higher affinity to OmpR-P than to F2a. Similarly,\nsix molecules of OmpR-P exhibited greater cooperative binding to the\nF1-F2-F3bF3b fragment (Fig. 6B, lanes 17–24), relative to the F1-F2-\nF3aF3a fragment (lanes 9 –16), again supporting that OmpR-P has a\nhigher affinity to F3b than to F3a.\nSimilar experiments were carried out with C2 (Fig. 6C) and C3 (Fig. 6D)\nsites, demonstrating that OmpR-P has a higher affinity to the C2bC2b\nsequence or the C3bC3b sequence than to the C2aC2a sequence or the\nC3aC3a sequence, respectively. Interestingly, OmpR-P binding to either\nC2aC2a or C3aC3a sequences was completely abolished, because com-\nplexes with the slowest mobility were not detected at the positions indi-\ncated by “4 OmpR-P” for C1-C2aC2a (Fig. 6C, lanes 9 –14) and by “6\nOmpR-P” for C1-C2-C3aC3a (Fig. 6D, lanes 9 –16), respectively. In con-\ntrast, C2bC2b in C1-C2 (Fig. 6C, lanes 16 –21) significantly improved\ncooperative binding of OmpR-P compared with the C1-C2 (lanes 1–7).\nA similar effect was also observed with C3bC3b in C1-C2-C3 (Fig. 6D,\nlanes 17–24).\nInterestingly, if the order of F1a and F1b sites in F1-F2 was reversed to\nthat of F1bF1a-F2, four OmpR-P molecules were still able to bind to the\nDNA fragment; however their cooperative binding (Fig. 6E, lanes 8 –14)\nwas poorer relative to the wild-type F1-F2 fragment (lanes 1–7), indi-\ncating that having the site for which OmpR-P has weaker affinity (F1a)\nupstream of the site for which it has stronger affinity (F1b) is important\nfor the most effective two-by-two OmpR-P binding mechanism.\nThese results unambiguously established the hierarchies for OmpR-P\nbinding affinity in all F and C sites as follows: F1b  F1a, F2b  F2a,\nF3b  F3a, C1b  C1a, C2b  C2a, and C3b  C3a. Furthermore, the\npresent results indicate that OmpR-P has different affinities for a-sites\nin F2 and F3 than for the a-sites in C2 and C3. This may have important\nimplications in the reciprocal regulation of ompF and ompC genes as\ndiscussed below.\nDISCUSSION\nHierarchical Arrangement of Two OmpR-P-binding Subsites—It has\nbeen shown that two OmpR-P molecules cooperatively bind to the\n20-bp F1 site (11). This F1 sequence consists of two 10-bp direct repeats,\nF1a at the 5-end and F1b at the 3-end, which share a number of com-\nmon bases (Fig. 1A). In the present study, we demonstrated that\nOmpR-P binds more tightly to downstream F1b than to upstream F1a,\nbecause OmpR-P is able to bind to F1bF1b but not to F1aF1a (Fig. 1, B\nand C). This OmpR-P binding hierarchy is due to the fact that the F1b\nsite contains more conserved bases (eight bases; Fig. 1A) than the F1a\nsite (six bases) in the 10-base consensus sequence (G1)-T2-T3-A4-C5-\nA6-T7-X-T9-X, derived by PCR-based enrichment technology (Fig. 1A\nand Ref. 11). We showed that G1, A4, C5, and A6 residues in this con-\nsensus sequence play a key role for optimal OmpR-P binding (Figs. 2C\nand 3, A and B). Similarly, in all of the other OmpR-P-binding sites (F2,\nF3, C1, C2, and C3) except for F4, the b-sites have more consensus bases\nthan their cognate a-sites (Fig. 1A). Notably, all of the b-sites contain\nboth G1 and C5 residues. Consistently, the gel shift assay demonstrated\nthat OmpR-P has a higher affinity to the b-sites than to the a-sites (Fig.\n6). Thus, the following hierarchy was established for OmpR-P binding\naffinity within each OmpR-P-binding site; F1b  F1a, F2b  F2a, F3b \nF3a, C1b  C1a, C2b  C2a, and C3b  C3a. As for the F4 site, the G1\nresidue in F4b is absent, whereas F4a has the G1 residue. It is currently\nnot known whether F4b has higher affinity than that of F4a for binding\nto OmpR-P.\nStimulation of OmpR Cooperative Binding by Phosphorylation—Al-\nthough both OmpR and OmpR-P exist as monomers in solution, two\nOmpR-P molecules cooperatively bind to the F1 site (11, 12). There are\ntwo modes of cooperative binding of OmpR-P F and C sites: one\nbetween two OmpR-P molecules in each 20-bp OmpR-P binding unit\nand the other between OmpR-P dimer binding to F1 and F2, to F2 and\nF3, to C1 and C2, or to C2 and C3. It is not known at present how\ncooperative binding occurs between two OmpR-P dimers on the F and\nC sites. In the present study, we found that nonphosphorylated OmpR is\nalso capable of cooperatively binding to F1aF1b (F1) and F1bF1b (Fig. 4,\nA and B). The C-terminal DNA-binding domain (OmpRc) by itself was\ncapable of binding to the F1b site in F1, but it could not cooperatively\nbind to F1aF1b (Fig. 4C). This indicates that the N-terminal domain of\nOmpR is responsible for its cooperative binding to DNA, and its phos-\nphorylation is likely to play an important role in enhancing the interac-\ntion between two N-terminal domains of the OmpR-P dimer on DNA.\nRecent structural studies of proteins in the OmpR/PhoB family sug-\ngest that the N-terminal domain is important to cooperative OmpR-P\nDNA binding activity. The x-ray structures of DrrD and DrrB, OmpR\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nhomologues from Thermotoga maritima, revealed that the N-terminal\ndomain of the molecule interacts with its C-terminal domain (18, 19).\nBecause the 4-5-5 structure of the N-terminal domain that inter-\nacts with the C-terminal domain is highly conserved among the OmpR/\nPhoB family, it was proposed that this region may also serve as an inter-\nface to stabilize dimer formation on DNA (18). This proposal was\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nsupported by recent structural studies of the dimerized N-terminal\ndomain of ArcA (20) and PhoB (21), suggesting an intriguing insight\ninto the role of phosphorylation and subsequent interaction between\nOmpR-P molecules through the N-terminal 4-5-5 region.\nDNA-mediated Dimerization of OmpR-P—We previously showed\nthat certain substitution mutations in F1a completely abolished\nOmpR-P binding to the mutated F1 fragments even if F1b was kept\nintact, indicating that one OmpR-P molecule is unable to make a stable\ncomplex with DNA (11). In the present study, we showed that only two\nOmpR-P molecules bind to a 30-bp DNA fragment consisting of three\noptimized binding subsites (F1bF1bF1b),\nindicating that OmpR-P\ndimer formation is mediated by DNA.\nThis pair-wise binding of OmpR-P is not due to preformed OmpR-P\ndimers that subsequently bind to DNA, because switching the order of\nthe a- and b-sites in F1 significantly reduced cooperative OmpR-P bind-\ning to the F1bF1a-F2 fragment (Fig. 6E). It appears that OmpR-P dimer\nformation occurs only on DNA such that the binding of an OmpR-P\nmolecule to the downstream (stronger) b-site assists binding of a second\nOmpR-P molecule to the upstream (weaker) a-site. This DNA-medi-\nated OmpR-P dimer formation is highly cooperative and synergistic,\nbeing mediated by interaction between the N-terminal domains (possi-\nbly the 4-5-5 regions) of two OmpR-P molecules and the interac-\ntion of the C-terminal domain of the second OmpR-P molecule with\nDNA at the a-site. In this model, the C-terminal DNA-binding domains\nare tandemly (or in a head-to-tail manner) aligned on DNA, whereas the\nN-terminal domains interact in a head-to-head manner as proposed by\nStock and co-workers (12, 18, 20, 21). Thus, the order of OmpR-P inter-\naction with the two subsites appears to play a critical role in the forma-\ntion of the most stable OmpR-P/DNA complex.\nSignificance of Hierarchical OmpR-P Binding in Reciprocal Gene Reg-\nulation of ompF and ompC—On the basis of the data presented, we\ntermed the two-by-two binding mechanism of OmpR-P to DNA as a\ngalloping model because this OmpR-P binding mode is similar to the\nway the front and rear legs move when a horse gallops. The key feature\nof this model is that according to hierarchical arrangement of OmpR-\nP-binding sites at the regulatory elements, F1-F2-F3 and C1-C2-C3, six\nOmpR-P molecules bind discontinuously (two-by-two) because a\ndownstream b-subsite in individual F and C sites serve as the primary\nOmpR-P-binding site, and simultaneously a second OmpR-P molecule\nbinds to a weaker upstream a-subsite (Fig. 7A).\nThe tandem arrangement of two 10-bp sites is essential for two\nOmpR-P molecules to form a stable complex with DNA. If both sites\nhave a weak affinity for OmpR-P, it severely reduces the binding of\nOmpR-P to these sites, requiring a significantly high concentration of\nHierarchical Binding of OmpR-P Regulates ompF and ompC\nOmpR-P to detect OmpR-P binding to DNA (Figs. 1 and 6). On the\nother hand, if both sites have a high affinity for OmpR-P, even nonphos-\nphorylated OmpR is able to bind to DNA (Fig. 4). Therefore, the\narrangement of a weaker (a-subsite) followed by a higher (b-subsite)\naffinity found in all OmpR-P-binding sites in F1-F2-F3 and C1-C2-C3 is\nimportant for ompF and ompC regulation. As proposed in a galloping\nmodel, discontinuous arrangement of the a-subsite with weak affinity\nand the b-subsite with high affinity probably ensures that one OmpR-P\nmolecule first binds to the b-subsite. This also regulates the final binding\nof two OmpR-P molecules by modulating the affinity of a second a-sub-\nsite. It is also important to note that the different affinity between F2a\nand C2a (which lacks C at 5th position) for OmpR-P, plays a key role in\nthe reciprocal regulation of ompF and ompC. This is because F2a has a\nhigher affinity for OmpR-P than C2a, because two OmpR-P molecules\nare able to bind to F2aF2a site at a high concentration of OmpR-P, but\nnot to the C2aC2a site. This results in the binding of four OmpR-P\nmolecules to F1-F2aF2a (Fig. 6A) but binding of only two OmpR-P\nmolecules to C1-C2aC2a (Fig. 6C). The same effect was also observed\nwith F3a and C3a (Fig. 6, B and D), so that only after OmpR-P occupies\nall F1-F2-F3 sites, will the C1-C2-C3 sites be occupied with OmpR-P,\nthus achieving the reciprocal regulation of ompF and ompC.\nEarlier footprinting studies established that OmpR-P binding to each\nof the regulatory elements for the ompF and ompC promoters occurs\ndifferentially in four steps (9, 14); (i) F1, C1 (ompF and ompC are not\nexpressed); (ii) F1-F2, C1 (ompF, but not ompC is expressed); (iii) F1-F2-\nF3, C1 (ompF, but not ompC is expressed); and (iv) F1-F2-F3 plus F4,\nC1-C2-C3 (ompF repressed and ompC is expressed). This suggests that\nthere are four critical cellular concentrations of OmpR-P that regulate\nthe induction and repression of ompF and the induction of ompC. Fig.\n7B illustrates the sequential regulation of ompF and ompC expression,\nwherein the cellular concentration of OmpR-P increases from KI (low)\nto KIV (high) upon medium osmolarity changes. At KI, only F1 and C1\nare occupied with two OmpR-P molecules. Note that two OmpR-P\nmolecules bind to both F1 and C1 sites in the order depicted by the\nnumbered molecules and according to the galloping model. Both ompF\nand ompC are not transcribed under this condition. When the OmpR-P\nconcentration increases from KI to KII, two more OmpR-P molecules\nbind to the F2 site again in the order from molecule 3 to molecule 4. At\nKIII, two more OmpR-P molecules (molecules 5 and 6) bind to the F3\nsite. On the other hand, the OmpR-P concentration of KII or KIII is not\nhigh enough to occupy C2 and C3 sites, allowing exclusive transcription\nof ompF. As medium osmolarity further increases, the OmpR-P concen-\ntration finally reaches KIV allowing the binding of two more OmpR-P\nmolecules (molecules 7 and 8) to F4 to repress ompF transcription. This\nrepression is assumed to be due to DNA loop formation between F4 and\nF1-F2-F3 mediated by OmpR-P bound to these regions (9). Six OmpR-P\nmolecules (molecules 1– 6) are now able to bind to C1-C2-C3 in a dis-\ncontinuous and cooperative manner to activate ompC transcription.\nThis well orchestrated gene regulation of ompF and ompC by\nOmpR-P is achieved by the combination of the two-domain structure of\nOmpR, the cellular concentration of OmpR-P, and the highly sophisti-\ncated hierarchical arrangement of OmpR-P-binding sites. Each N- and\nC-terminal domain of OmpR has a distinct role in the DNA binding of\nOmpR-P. We have demonstrated that E. coli cells contain a large pool of\nOmpR, whereas only a minor fraction is presumed to be phosphorylated\nby EnvZ, a transmembrane osmosensing histidine kinase (7, 8). The\ncellular level of OmpR-P is thus finely tuned by dual EnvZ functions,\nOmpR kinase and OmpR-P phosphatase, whose ratios are regulated by\nthe medium osmolarity. Finally, because of the hierarchical arrange-\nment of OmpR-P-binding sites, OmpR-P occupy differently the regula-\ntory element of ompF and ompC promoters depending on the concen-\ntration of OmpR-P, allowing a reciprocal expression of these genes to\navoid overlapping expression from both genes at the same time.\nConclusion—In this study, we demonstrated a regulatory mechanism\nfor transcription using a single transcription factor. The reciprocal tran-\nscription of ompF and ompC is regulated by OmpR-P and is finely tuned\nby virtue of hierarchical OmpR-P binding to DNA. This simple but\nhighly sophisticated transcriptional regulation performed by a single\ntranscription factor may be widely used in prokaryotes and is an inter-\nesting contrast to the eukaryotic systems, where a large number of tran-\nscription factors are involved in regulating the expression of a single\ngene.",
         "Mechanism of activation for transcription factor PhoB suggested by different modes of dimerization in the inactive and active states.;Structural Analysis and Solution Studies of the Activated Regulatory Domain of the Response Regulator ArcA: A Symmetric Dimer Mediated by the α4-β5-α5 Face;MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli;Structural Analysis of the Domain Interface in DrrB, a Response Regulator of the OmpR/PhoB Subfamily;Interaction of EnvZ, a sensory histidine kinase, with phosphorylated OmpR, the cognate response regulator;Histidine Kinases in Signal Transduction;EnvZ-OmpR Interaction and Osmoregulation in Escherichia coli *;Evidence of intradomain and interdomain flexibility in an OmpR/PhoB homolog from Thermotoga maritima.;The Critical Role of the Conserved Thr247 Residue in the Functioning of the Osmosensor EnvZ, a Histidine Kinase/Phosphatase, in Escherichia coli *;Orientation of OmpR monomers within an OmpR:DNA complex determined by DNA affinity cleaving.;Hierarchical and co‐operative binding of OmpR to a fusion construct containing the ompC and ompF upstream regulatory sequences of Escherichia coli;Relative binding affinities of OmpR and OmpR-phosphate at the ompF and ompC regulatory sites.;The DNA-binding domain of OmpR: crystal structures of a winged helix transcription factor.;Tandem Binding of Six OmpR Proteins to the ompF Upstream Regulatory Sequence of Escherichia coli(*);Fractal patterns for cells in culture;The OmpR protein of Escherichia coli binds to sites in the ompF promoter region in a hierarchical manner determined by its degree of phosphorylation.;A distant upstream site involved in the negative regulation of the Escherichia coli ompF gene;Characterization by deletion and localized mutagenesis in vitro of the promoter region of the Escherichia coli ompC gene and importance of the upstream DNA domain in positive regulation by the OmpR protein;A unique mechanism regulating gene expression: translational inhibition by a complementary RNA transcript (micRNA).;Porin channels in Escherichia coli: studies with liposomes reconstituted from purified proteins;Genetic analysis of the ompB locus in Escherichia coli K-12.;Regulation of Porins in Escherichia coli by the Osmosensing Histidine Kinase/Phosphatase EnvZ;Structure (Camb.) 13, 1353–1363 Hierarchical Binding of OmpR-P Regulates",
         "Transcription Regulation of ompF and ompC by a Single Transcription Factor, OmpR*",
         "None"
        ],
        [
         "35",
         "001cdeaaf9f7141b7a9598b26684c01ab199a311",
         "The combination of accessible embryology and forward genetic techniques has made zebraWsh a powerful model system for the study\nof vertebrate development. One limitation of genetic analysis is that the study of gene function is usually limited to the Wrst developmental\nevent aVected by a gene. In vivo electroporation has recently matured as a method for studying gene function at diVerent developmental\ntime points and in speciWc regions of the organism. The focal application of current allows macromolecules to be eYciently introduced\ninto a targeted region at any time in the life cycle. Here we describe a rapid protocol by which DNA, RNA and morpholinos can all be\nprecisely electroporated into zebraWsh in a temporally and spatially controlled manner. This versatile technique allows gene function to\nbe determined by both gain and loss of function analyses in speciWc regions at speciWc times. This is the Wrst report that describes the\nelectroporation of three diVerent molecules into embryonic and larval zebraWsh cells.",
         "Gustavo A. Cerda,Jeanne E. Thomas,M. Allende,R. Karlstrom,Verónica Palma",
         "Gustavo A. Cerda,Jeanne E. Thomas,M. Allende,R. Karlstrom,Verónica Palma",
         "http://repositorio.uchile.cl/bitstream/2250/118774/1/Cerda_Gustavo.pdf",
         "The combination of accessible embryology and forward genetic techniques has made zebraWsh a powerful model system for the study\nof vertebrate development. One limitation of genetic analysis is that the study of gene function is usually limited to the Wrst developmental\nevent aVected by a gene. In vivo electroporation has recently matured as a method for studying gene function at diVerent developmental\ntime points and in speciWc regions of the organism. The focal application of current allows macromolecules to be eYciently introduced\ninto a targeted region at any time in the life cycle. Here we describe a rapid protocol by which DNA, RNA and morpholinos can all be\nprecisely electroporated into zebraWsh in a temporally and spatially controlled manner. This versatile technique allows gene function to\nbe determined by both gain and loss of function analyses in speciWc regions at speciWc times. This is the Wrst report that describes the\nelectroporation of three diVerent molecules into embryonic and larval zebraWsh cells.",
         "",
         "1. Introduction\nGene  transfer  techniques  have  been  used  extensively  in\ncultured  cells,  tissues  and  whole  organisms  for  assaying\neukaryotic gene function. Traditionally, the introduction of\ngenes into cells, or into genomes, has relied on transfection,\nviral infection or microinjection. Ideally, one should be able\nto study the eVects of a gain or loss of gene function eVect\nin the intact, live organism to monitor the eVects of genetic\nmodiWcations in a physiological context. Transgenesis and\ngene  targeting  (knock  outs)  have  been  invaluable  in  the\nassignment  of  gene  function  but  they  present  problems\nwhen the gene of interest has multiple roles during develop-\nment. While tissue speciWc knock outs or inducible expres-\nsion systems are elegant ways to circumvent this problem,\nthey  are  expensive,  laborious,  and  limited  to  certain\norganisms.\nElectroporation  has  served  as  an  eVective  method  for\nintroducing DNA into bacteria, yeast, and mammalian cells\n[1]. This technique uses electric pulses to make small holes\nin  the  cell  membrane  through  which  DNA  molecules  can\nenter  the  cell.  The  application  has  been  extended  in  past\nyears to embryos of several species, most notably the chick,\nfacilitating analyses at the molecular level, which tradition-\nally  have  been  diYcult  to  tackle  in  many  model  systems\n[2,3]. Electroporation into mouse embryos is also possible,\nthough embryos must be cultured after the procedure, a sit-\nuation  that  can  be  extended  for  a  limited  time  [4].  More\nrecently,  Xenopus  tadpoles  have  been  electroporated  with\nminimal  cell  death  and  excellent  survival  [5].  Electropora-\ntion of embryos has been made feasible by altering voltage\nand current parameters to minimize damage. Most eVective\nare square wave pulses of low voltage and longer durations\ncompared to conditions used in cultured cells [6–8]. Electro-\nporation  of  DNA  and  RNA  into  embryos  of  diVerent\nstages has thus introduced the possibility of modifying gene\nactivity  late  in  development  in  organisms  that  are  not\ngenetically tractable. Even when mutations are available, it\nis often important to analyze the eVects of eliminating gene\nfunction  in  a  speciWc  tissue  while  the  rest  of  the  animal\ndevelops  unperturbed.  Electroporation  provides  a  means\nfor creating localized loss of function using overexpression\nof  dominant-negative  proteins,  siRNA  [2]  or  antisense\noligonucleotides such as morpholinos [9].\nElectroporation  of  DNA  into  the  zebraWsh  embryonic\nneural tube [10] and adult Wn [11] have been reported, and\nthis technique was recently used to manipulate Fgf signal-\nin  the  zebraWsh  midbrain/hindbrain  region\ning\n[12].\nHowever, the electroporation of DNA constructs into other\ntissues,  and  of  mRNA  and  antisense  morpholinos  (MOs)\ninto zebraWsh, have not been reported. Here we describe a\ngreatly simpliWed yet precise method for introducing these\nmolecules  into  neural,  retinal,  and  somitic  tissue  in  the\nzebraWsh  embryo.  These  techniques  will  extend  the  use  of\nelectroporation and facilitate the analysis of gene function\nin any tissue and at any time in development.\n2. Method for electroporation\n2.1. Embryo preparation and mounting\nZebraWsh  (Danio  rerio)  of  the  Tübingen  or  TL  strains\nwere maintained on a 14–10-h light–dark cycle and bred in\nour  laboratory  according  to  standard  conditions  [13].\nEmbryos  obtained  from  mass  matings  were  kept  in  E3\nmedium  (5 mM  NaCl,  0.17 mM  KCl,  0.33 mM  CaCl2,\n0.33 mM MgSO4, and 0.1% methylene blue), staged in hours\npostfertilization (hpf) [14], dechorionated, and anesthetized\nusing  MS222  (Sigma).  20–24  hour  embryos  were  dechori-\nnated by incubating in pronase (125 g/ml dissolved in E3\nmedium) for 2 min followed by three washes in E3 medium.\n16–20 hpf embryos were dechorionated by hand, as pronase\ntreatment tended to make them more fragile during mount-\ning.  Dechorionated  embryos  were  placed  in  Ringer’s\nsolution (116 mM NaCl, 2.9 mM KCl, 1.8 mM CaCl2, 5 mM\nHepes pH 7.2).\nTo  mount  embryos  for  electroporation,  a  1%  LMT\nagarose  solution  was  prepared  in  electroporation  Ringers\n(ER),  a  slightly  modiWed  Ringers  solution  that  facilitates\ncurrent Xow (180 mM NaCl, 5 mM KCl, 1.8 mM CaCl2, pH\n7.2).  The  agarose  was  made  in  a  15 ml  conical  tube  and\nmaintained at 50 °C in a water bath until needed. Embryos\nwere  pipetted  into  the  tube,  allowed  to  sink,  then  quickly\npipetted onto a 2% agarose (in ER) covered 100 mm petri\ndish  with  one  embryo  per  drop  of  agarose.  For  speed,\nembryos  were  not  oriented,  with  up  to  60  embryos  being\nplaced in a single dish. Orientation was not necessary when\ntwo  micromanipulators  were  used  to  position  the  elec-\ntrodes.  The  plate  was  then  Wlled  with  ER  and  positioned\nunder a stereomicroscope equipped with two micromanipu-\nlators,  one  controlling  each  electroporation  electrode\n(Fig. 1).\n2.2. Preparation for microinjection\nCircular  GFP  encoding  plasmid  DNA  (Gibco,  Cat.  #\n10642.015) and in vitro synthetized mRNA were diluted to\nthe  desired  concentration  in  injection  buVer  (0.5%  Fast\nGreen, 200 mM Tris–Cl pH 7.0, 20 mM EDTA). Morpholi-\nnos (Gene Tools, Inc.) were diluted in PBS or water without\nfast  green,  as  fast  green  was  shown  to  inhibit MO  uptake\n[9].  DNA  (0.7–2 g/l)  was  prepared  using  a  Qiagen  plas-\nmid Maxi kit, while mRNA (0.5–1.0 g/l) was synthesized\nusing  the  Message  Machine  kit  (Ambion).  Morpholinos\nwere synthesized with a lissamine side chain to allow visual-\nization  within  the  electroporated  embryo.  0.2–0.5 mM  of\nthe  ptc1  MO  (CATAGTCCAAACGGGAGGCAGAA\nGA) [15] was injected in these experiments. Embryos were\npressure  injected  using  a  MPPI  Pressure  injector  (ASI,\nEugene,  OR,  USA),  with  the  injection  needle  being  posi-\ntioned  by  hand.  Injection  needles  were  made  using  1 mm\ncapillaries with Wlament (Cat. # TW100F-4, WPI, Sarasota,\nFL, USA) and a Sutter P-97 electrode puller and backWlled\nby pipetting a small drop into the blunt end of the capillary.\nElectrode  tips  were  broken  to  the  desired  diameter  using\nWne  forceps.  10–30 nl  of  solution  was  microinjected  in  the\nsomites, eyes, or brain ventricles.\n2.3. Microinjection and electroporation\nTwo diVerent electroporation devices were used to gen-\nerate electric pulses for these experiments: (1) the Electro-\nSquare Porator ECM830 (BTX, San Diego, CA, USA) and\n(2)  the  TSS20  Ovodyne  Electroporator  (Intracel,  Roysten\nHerts,  UK)  without  the  accessory  current  ampliWer.  Both\nelectroporators  were  eVective,  however  we  found  that  the\nIntracel  unit  allowed  for  a  wider  range  of  electroporation\nconditions.  Tungsten  electrodes  (0.010”,  Cat.  #  573500,\nA-M Systems Inc., Carlsberg, WA, USA) were used for all\nexperiments.  To  avoid  current  leakage  it  was  crucial  to\ninsulate the electrodes completely using paraWlm, leaving a\nfew  millimeters  of  metal  exposed  at  the  tip  to  contact  the\ntarget tissue. Electrodes were positioned using micromanip-\nulators such that the resulting current would pass through\nthe desired tissue. For DNA and mRNA, the positive elec-\ntrode  was  placed  just  outside  the  embryo  in  the  desired\nlocation,  while  the  negative  electrode  was  positioned  so\nthat  it  could  quickly  be  inserted  into  the  tissue  near  the\ninjection  site.  For  lissamine-coupled  MOs,  the  reverse\npolarity was used. Embryos were then injected, the negative\nelectrode  was  inserted  into  the  tissue,  and  current  was\nquickly applied using a foot switch.\nOptimizing  the  electroporation  parameters  is  impera-\ntive  as  the  application  of  high-voltage  pulses  can\nkill many  cells.  Parameters  of  voltage,  pulse  length,\npulse intervals,  and  the  number  of  pulses  for  DNA,\nmRNA, and MO electroporation are speciWed in Table 1.\nThese parameters serve as a starting point and should be\noptimized for each electroporator and electrode set-up, as\nwell as for each tissue and developmental stage being ana-\nlyzed.  After  electroporation,  the  embryos  were  gently\nremoved  from  the  agarose  with  the  help  of  forceps  and\nplaced  in petri dishes  containing zebraWsh Ringer’s solu-\ntion with penicillin–streptomycin (50–100 U/ml) and incu-\nbated  at  28 °C.  GFP  and  lissamine  Xuorescence  were\nvisualized  in  live  embryos  using  both  dissecting  and\ncompound microscopes (Fig. 2).\n3. Concluding remarks\nThe ability to express or inactivate a gene in a directed\nfashion is a powerful means of analyzing its role in develop-\nment.  Electroporation  is  a  simple  procedure  for  spatially\nand  temporally  regulating  gene  function  that  can  be  per-\nformed more easily and quickly than any other transfection\nmethod. The electroporation protocol we describe here has\nseveral  advantages  over  those  previously  described  for\nzebraWsh.  First,  by  replacing  both  electrodes  with\nsharpened  tungsten  needles  we  were  able  to  narrow  and\nprecisely position the electric Weld, allowing for very precise\ntargeting  of  macromolecules  into  the  brain.  Second,  this\ntechnique allowed for highly eYcient introduction of mac-\nromolecules  while  maintaining  high  survival  rates.  Third,\nby  rapidly  mounting  embryos,  we  greatly  increased  the\nspeed of the protocol, allowing many more experiments to\nbe performed in a given time. Finally, we show that DNA,\nmRNA, and morpholinos can all be precisely targeted into\nzebraWsh tissue.\n‘Conditional  mutagenesis’  techniques  such  as  Cre–Lox\nrecombination in the mouse [16] and the Gal4–UAS system\nin  the  Xy  [17]  and  zebraWsh  [18]  allow  tissue  speciWc  and\ntime-restricted analysis, but the process can be complicated\nand  is  dependent  on  having  promoters  that  will  drive\nexpression  in the  correct tissue and/or at the desired time.\nFurther, the analysis of two or more genes is especially diY-\ncult  using  these  methods.  Using  electroporation  it  is  now\nrelatively  easy  to  perform  both  gain-of-function  and  loss-\nof-function experiments in any tissue. A key advantage of\nelectroporation is the ease with which two or more expres-\nsion constructs can be simultaneously introduced into cells.\nMorpholinos, dominant negative constructs, and/or consti-\ntutive  activator  constructs  for diVerent  genes  can be com-\nbined  to  simultaneously  manipulate  multiple  genes  in  the\nsame  cells.  Since  in vivo  electroporation  can  alter  gene\nexpression  in  a  small area, this method will be  helpful for\nthe  generation  of  region  speciWc  transient  knockouts  and\nwill  allow  analyses  in  embryonic  areas  for  which  speciWc\ntranscriptional control sequences have not been identiWed.\nOur studies illustrate that GFP-construct electroporated\ncells can be easily visualized in the zebraWsh, both in the live\nembryo (Fig. 2), as well as after Wxation and using the anti-\nGFP  antibody  (data  not  shown).  We  were  able  to  detect\nstrong Xuorescence from the GFP transgene product both\n12 h and 24 h after electroporation.\nBy co-electroporating constructs encoding GFP or other\ncellular markers it will thus be possible to track the develop-\nmental fates of cells with altered gene expression as well as to\ndetermine the eVects on surrounding cells. Importantly, these\nstudies  can  be  done  at  any  time  in  embryonic  or  larval\ndevelopment, or in adult tissue, greatly expanding the ability\nto assess gene function. This approach is especially useful for\nstudying postembryonic gene activity, as early essential gene\nfunction can be bypassed. This will greatly facilitate the study\nof processes such as regeneration, tumorigenesis, or response\nto injury in diVerentiated tissue.",
         "",
         "",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "The combination of accessible embryology and forward genetic techniques has made zebraWsh a powerful model system for the study\nof vertebrate development. One limitation of genetic analysis is that the study of gene function is usually limited to the Wrst developmental\nevent aVected by a gene. In vivo electroporation has recently matured as a method for studying gene function at diVerent developmental\ntime points and in speciWc regions of the organism. The focal application of current allows macromolecules to be eYciently introduced\ninto a targeted region at any time in the life cycle. Here we describe a rapid protocol by which DNA, RNA and morpholinos can all be\nprecisely electroporated into zebraWsh in a temporally and spatially controlled manner. This versatile technique allows gene function to\nbe determined by both gain and loss of function analyses in speciWc regions at speciWc times. This is the Wrst report that describes the\nelectroporation of three diVerent molecules into embryonic and larval zebraWsh cells.\n1. Introduction\nGene  transfer  techniques  have  been  used  extensively  in\ncultured  cells,  tissues  and  whole  organisms  for  assaying\neukaryotic gene function. Traditionally, the introduction of\ngenes into cells, or into genomes, has relied on transfection,\nviral infection or microinjection. Ideally, one should be able\nto study the eVects of a gain or loss of gene function eVect\nin the intact, live organism to monitor the eVects of genetic\nmodiWcations in a physiological context. Transgenesis and\ngene  targeting  (knock  outs)  have  been  invaluable  in  the\nassignment  of  gene  function  but  they  present  problems\nwhen the gene of interest has multiple roles during develop-\nment. While tissue speciWc knock outs or inducible expres-\nsion systems are elegant ways to circumvent this problem,\nthey  are  expensive,  laborious,  and  limited  to  certain\norganisms.\nElectroporation  has  served  as  an  eVective  method  for\nintroducing DNA into bacteria, yeast, and mammalian cells\n[1]. This technique uses electric pulses to make small holes\nin  the  cell  membrane  through  which  DNA  molecules  can\nenter  the  cell.  The  application  has  been  extended  in  past\nyears to embryos of several species, most notably the chick,\nfacilitating analyses at the molecular level, which tradition-\nally  have  been  diYcult  to  tackle  in  many  model  systems\n[2,3]. Electroporation into mouse embryos is also possible,\nthough embryos must be cultured after the procedure, a sit-\nuation  that  can  be  extended  for  a  limited  time  [4].  More\nrecently,  Xenopus  tadpoles  have  been  electroporated  with\nminimal  cell  death  and  excellent  survival  [5].  Electropora-\ntion of embryos has been made feasible by altering voltage\nand current parameters to minimize damage. Most eVective\nare square wave pulses of low voltage and longer durations\ncompared to conditions used in cultured cells [6–8]. Electro-\nporation  of  DNA  and  RNA  into  embryos  of  diVerent\nstages has thus introduced the possibility of modifying gene\nactivity  late  in  development  in  organisms  that  are  not\ngenetically tractable. Even when mutations are available, it\nis often important to analyze the eVects of eliminating gene\nfunction  in  a  speciWc  tissue  while  the  rest  of  the  animal\ndevelops  unperturbed.  Electroporation  provides  a  means\nfor creating localized loss of function using overexpression\nof  dominant-negative  proteins,  siRNA  [2]  or  antisense\noligonucleotides such as morpholinos [9].\nElectroporation  of  DNA  into  the  zebraWsh  embryonic\nneural tube [10] and adult Wn [11] have been reported, and\nthis technique was recently used to manipulate Fgf signal-\nin  the  zebraWsh  midbrain/hindbrain  region\ning\n[12].\nHowever, the electroporation of DNA constructs into other\ntissues,  and  of  mRNA  and  antisense  morpholinos  (MOs)\ninto zebraWsh, have not been reported. Here we describe a\ngreatly simpliWed yet precise method for introducing these\nmolecules  into  neural,  retinal,  and  somitic  tissue  in  the\nzebraWsh  embryo.  These  techniques  will  extend  the  use  of\nelectroporation and facilitate the analysis of gene function\nin any tissue and at any time in development.\n2. Method for electroporation\n2.1. Embryo preparation and mounting\nZebraWsh  (Danio  rerio)  of  the  Tübingen  or  TL  strains\nwere maintained on a 14–10-h light–dark cycle and bred in\nour  laboratory  according  to  standard  conditions  [13].\nEmbryos  obtained  from  mass  matings  were  kept  in  E3\nmedium  (5 mM  NaCl,  0.17 mM  KCl,  0.33 mM  CaCl2,\n0.33 mM MgSO4, and 0.1% methylene blue), staged in hours\npostfertilization (hpf) [14], dechorionated, and anesthetized\nusing  MS222  (Sigma).  20–24  hour  embryos  were  dechori-\nnated by incubating in pronase (125 g/ml dissolved in E3\nmedium) for 2 min followed by three washes in E3 medium.\n16–20 hpf embryos were dechorionated by hand, as pronase\ntreatment tended to make them more fragile during mount-\ning.  Dechorionated  embryos  were  placed  in  Ringer’s\nsolution (116 mM NaCl, 2.9 mM KCl, 1.8 mM CaCl2, 5 mM\nHepes pH 7.2).\nTo  mount  embryos  for  electroporation,  a  1%  LMT\nagarose  solution  was  prepared  in  electroporation  Ringers\n(ER),  a  slightly  modiWed  Ringers  solution  that  facilitates\ncurrent Xow (180 mM NaCl, 5 mM KCl, 1.8 mM CaCl2, pH\n7.2).  The  agarose  was  made  in  a  15 ml  conical  tube  and\nmaintained at 50 °C in a water bath until needed. Embryos\nwere  pipetted  into  the  tube,  allowed  to  sink,  then  quickly\npipetted onto a 2% agarose (in ER) covered 100 mm petri\ndish  with  one  embryo  per  drop  of  agarose.  For  speed,\nembryos  were  not  oriented,  with  up  to  60  embryos  being\nplaced in a single dish. Orientation was not necessary when\ntwo  micromanipulators  were  used  to  position  the  elec-\ntrodes.  The  plate  was  then  Wlled  with  ER  and  positioned\nunder a stereomicroscope equipped with two micromanipu-\nlators,  one  controlling  each  electroporation  electrode\n(Fig. 1).\n2.2. Preparation for microinjection\nCircular  GFP  encoding  plasmid  DNA  (Gibco,  Cat.  #\n10642.015) and in vitro synthetized mRNA were diluted to\nthe  desired  concentration  in  injection  buVer  (0.5%  Fast\nGreen, 200 mM Tris–Cl pH 7.0, 20 mM EDTA). Morpholi-\nnos (Gene Tools, Inc.) were diluted in PBS or water without\nfast  green,  as  fast  green  was  shown  to  inhibit MO  uptake\n[9].  DNA  (0.7–2 g/l)  was  prepared  using  a  Qiagen  plas-\nmid Maxi kit, while mRNA (0.5–1.0 g/l) was synthesized\nusing  the  Message  Machine  kit  (Ambion).  Morpholinos\nwere synthesized with a lissamine side chain to allow visual-\nization  within  the  electroporated  embryo.  0.2–0.5 mM  of\nthe  ptc1  MO  (CATAGTCCAAACGGGAGGCAGAA\nGA) [15] was injected in these experiments. Embryos were\npressure  injected  using  a  MPPI  Pressure  injector  (ASI,\nEugene,  OR,  USA),  with  the  injection  needle  being  posi-\ntioned  by  hand.  Injection  needles  were  made  using  1 mm\ncapillaries with Wlament (Cat. # TW100F-4, WPI, Sarasota,\nFL, USA) and a Sutter P-97 electrode puller and backWlled\nby pipetting a small drop into the blunt end of the capillary.\nElectrode  tips  were  broken  to  the  desired  diameter  using\nWne  forceps.  10–30 nl  of  solution  was  microinjected  in  the\nsomites, eyes, or brain ventricles.\n2.3. Microinjection and electroporation\nTwo diVerent electroporation devices were used to gen-\nerate electric pulses for these experiments: (1) the Electro-\nSquare Porator ECM830 (BTX, San Diego, CA, USA) and\n(2)  the  TSS20  Ovodyne  Electroporator  (Intracel,  Roysten\nHerts,  UK)  without  the  accessory  current  ampliWer.  Both\nelectroporators  were  eVective,  however  we  found  that  the\nIntracel  unit  allowed  for  a  wider  range  of  electroporation\nconditions.  Tungsten  electrodes  (0.010”,  Cat.  #  573500,\nA-M Systems Inc., Carlsberg, WA, USA) were used for all\nexperiments.  To  avoid  current  leakage  it  was  crucial  to\ninsulate the electrodes completely using paraWlm, leaving a\nfew  millimeters  of  metal  exposed  at  the  tip  to  contact  the\ntarget tissue. Electrodes were positioned using micromanip-\nulators such that the resulting current would pass through\nthe desired tissue. For DNA and mRNA, the positive elec-\ntrode  was  placed  just  outside  the  embryo  in  the  desired\nlocation,  while  the  negative  electrode  was  positioned  so\nthat  it  could  quickly  be  inserted  into  the  tissue  near  the\ninjection  site.  For  lissamine-coupled  MOs,  the  reverse\npolarity was used. Embryos were then injected, the negative\nelectrode  was  inserted  into  the  tissue,  and  current  was\nquickly applied using a foot switch.\nOptimizing  the  electroporation  parameters  is  impera-\ntive  as  the  application  of  high-voltage  pulses  can\nkill many  cells.  Parameters  of  voltage,  pulse  length,\npulse intervals,  and  the  number  of  pulses  for  DNA,\nmRNA, and MO electroporation are speciWed in Table 1.\nThese parameters serve as a starting point and should be\noptimized for each electroporator and electrode set-up, as\nwell as for each tissue and developmental stage being ana-\nlyzed.  After  electroporation,  the  embryos  were  gently\nremoved  from  the  agarose  with  the  help  of  forceps  and\nplaced  in petri dishes  containing zebraWsh Ringer’s solu-\ntion with penicillin–streptomycin (50–100 U/ml) and incu-\nbated  at  28 °C.  GFP  and  lissamine  Xuorescence  were\nvisualized  in  live  embryos  using  both  dissecting  and\ncompound microscopes (Fig. 2).\n3. Concluding remarks\nThe ability to express or inactivate a gene in a directed\nfashion is a powerful means of analyzing its role in develop-\nment.  Electroporation  is  a  simple  procedure  for  spatially\nand  temporally  regulating  gene  function  that  can  be  per-\nformed more easily and quickly than any other transfection\nmethod. The electroporation protocol we describe here has\nseveral  advantages  over  those  previously  described  for\nzebraWsh.  First,  by  replacing  both  electrodes  with\nsharpened  tungsten  needles  we  were  able  to  narrow  and\nprecisely position the electric Weld, allowing for very precise\ntargeting  of  macromolecules  into  the  brain.  Second,  this\ntechnique allowed for highly eYcient introduction of mac-\nromolecules  while  maintaining  high  survival  rates.  Third,\nby  rapidly  mounting  embryos,  we  greatly  increased  the\nspeed of the protocol, allowing many more experiments to\nbe performed in a given time. Finally, we show that DNA,\nmRNA, and morpholinos can all be precisely targeted into\nzebraWsh tissue.\n‘Conditional  mutagenesis’  techniques  such  as  Cre–Lox\nrecombination in the mouse [16] and the Gal4–UAS system\nin  the  Xy  [17]  and  zebraWsh  [18]  allow  tissue  speciWc  and\ntime-restricted analysis, but the process can be complicated\nand  is  dependent  on  having  promoters  that  will  drive\nexpression  in the  correct tissue and/or at the desired time.\nFurther, the analysis of two or more genes is especially diY-\ncult  using  these  methods.  Using  electroporation  it  is  now\nrelatively  easy  to  perform  both  gain-of-function  and  loss-\nof-function experiments in any tissue. A key advantage of\nelectroporation is the ease with which two or more expres-\nsion constructs can be simultaneously introduced into cells.\nMorpholinos, dominant negative constructs, and/or consti-\ntutive  activator  constructs  for diVerent  genes  can be com-\nbined  to  simultaneously  manipulate  multiple  genes  in  the\nsame  cells.  Since  in vivo  electroporation  can  alter  gene\nexpression  in  a  small area, this method will be  helpful for\nthe  generation  of  region  speciWc  transient  knockouts  and\nwill  allow  analyses  in  embryonic  areas  for  which  speciWc\ntranscriptional control sequences have not been identiWed.\nOur studies illustrate that GFP-construct electroporated\ncells can be easily visualized in the zebraWsh, both in the live\nembryo (Fig. 2), as well as after Wxation and using the anti-\nGFP  antibody  (data  not  shown).  We  were  able  to  detect\nstrong Xuorescence from the GFP transgene product both\n12 h and 24 h after electroporation.\nBy co-electroporating constructs encoding GFP or other\ncellular markers it will thus be possible to track the develop-\nmental fates of cells with altered gene expression as well as to\ndetermine the eVects on surrounding cells. Importantly, these\nstudies  can  be  done  at  any  time  in  embryonic  or  larval\ndevelopment, or in adult tissue, greatly expanding the ability\nto assess gene function. This approach is especially useful for\nstudying postembryonic gene activity, as early essential gene\nfunction can be bypassed. This will greatly facilitate the study\nof processes such as regeneration, tumorigenesis, or response\nto injury in diVerentiated tissue.",
         "",
         "Electroporation of DNA, RNA, and morpholinos into zebrafish embryos.",
         "None"
        ],
        [
         "36",
         "001ced5aee9fad2575bb01579ec7e703e8ff7386",
         "c-Fos, a component of the transcription factor AP-1, is rapidly imported into the nucleus after translation. We established an in vitro system using digitonin-permeabilized cells to analyze nuclear import of c-Fos in detail. Two import receptors of the importin β superfamily, importin β itself and transportin, promote import of c-Fos in vitro. Under conditions where importin β-dependent transport was blocked, c-Fos still accumulated in the nucleus in the presence of cytosol. Inhibition of the transportin-dependent pathway, in contrast, abolished import of c-Fos. Furthermore, c-Fos mutants that interact with transportin but not with importin β were efficiently imported in the presence of cytosol. Hence, transportin appears to be the predominant import receptor for c-Fos. A detailed biochemical characterization revealed that the interaction of transportin with c-Fos is distinct from the interaction with its established import cargoes, the M9 sequence of heterogeneous nuclear ribonucleoprotein A1 or the nuclear localization sequence of some basic proteins. Likewise, the binding sites on importin β for its classic import cargo and for c-Fos can be separated. In summary, c-Fos employs a novel mode of receptor-cargo interaction. Hence, transportin may be as versatile as importin β in recognizing different nuclear import cargoes.",
         "M. Arnold,A. Nath,D. Wohlwend,R. Kehlenbach",
         "M. Arnold,A. Nath,D. Wohlwend,R. Kehlenbach",
         "http://www.jbc.org/article/S0021925819766721/pdf",
         "c-Fos, a component of the transcription factor AP-1, is rapidly\nimported into the nucleus after translation. We established an in\nvitro system using digitonin-permeabilized cells to analyze nuclear\nimport of c-Fos in detail. Two import receptors of the importin \nsuperfamily, importin  itself and transportin, promote import of\nc-Fos in vitro. Under conditions where importin -dependent\ntransport was blocked, c-Fos still accumulated in the nucleus in the\npresence of cytosol. Inhibition of the transportin-dependent path-\nway, in contrast, abolished import of c-Fos. Furthermore, c-Fos\nmutants that interact with transportin but not with importin were\nefficiently imported in the presence of cytosol. Hence, transportin\nappears to be the predominant import receptor for c-Fos. A detailed\nbiochemical characterization revealed that the interaction of trans-\nportin with c-Fos is distinct from the interaction with its established\nimport cargoes, the M9 sequence of heterogeneous nuclear ribonu-\ncleoprotein A1 or the nuclear localization sequence of some basic\nproteins. Likewise, the binding sites on importin  for its classic\nimport cargo and for c-Fos can be separated. In summary, c-Fos\nemploys a novel mode of receptor-cargo interaction. Hence, trans-\nportin may be as versatile as importin  in recognizing different\nnuclear import cargoes.",
         "DISCUSSION\nTransportin Is the Major Import Receptor for c-Fos—Different regions\nin c-Fos required for its efficient import into the nucleus have been\nidentified (23, 24). Transport factors that promote nuclear import of\nc-Fos in the classic in vitro system, however, have not been described. In\nthe course of this study, we analyzed nuclear import of c-Fos in digito-\nnin-permeabilized cells. In this system, the import receptors transportin\nand importin  are able to promote specific import of c-Fos into the\nnucleus. Several lines of evidence, however, point to a dominant role of\ntransportin for nuclear import of c-Fos. (i) Transportin was highly\nenriched on GST-Fos beads incubated with a cytosolic extract. In con-\ntrast, only a small fraction of other import receptors like importin and\nimportin 7 bound to immobilized GST-Fos. (ii) Recombinant transpor-\ntin promoted nuclear import of GST-Fos more efficiently than importin\n. Importin 7 did not promote nuclear import of c-Fos. Like importin 8,\nimportin 7 interacts with importin (35). Thus, these two proteins may\nhave interacted with GST-Fos via importin (Fig. 2A). (iii) Competition\nexperiments revealed that GST-Fos interacts more efficiently with\ntransportin than importin , suggesting a much higher affinity. The\npublished dissociation constant of the c-Fos-importin  complex is\nrather high (65 nM (13)) compared with the dissociation constant of the\nPTHrP-importin  complex (3.5 nM (9)) or to the binding affinity of\nimportin to importin (5–18 nM (44)). (iv) Fragments or mutants of\nc-Fos that do not interact with importin (but with transportin) were\nefficiently imported into the nucleus in the presence of cytosol. (v) Most\nimportantly, under conditions where transportin-dependent import\nwas inhibited by a substrate-competitor, c-Fos did not enter the\nnucleus, whereas an importin /substrate was efficiently transported.\nIn contrast, inhibition of the importin -dependent pathway with an\nantibody against the receptor did abolish import of GST-NLS but not\nthat of c-Fos. Taken together, we believe that transportin is the major\nimport receptor for c-Fos. Under certain conditions of activation or in\ncertain cell types, however, importin  may serve as an importin for\nc-Fos as well. This interpretation is in agreement with earlier results that\nshowed that a 22-amino acid basic region (i.e. the importin -interac-\ntion domain, see below) can function as an NLS when fused to pyruvate\nkinase as a reporter protein (23). Deletion of this basic region, however,\ndid not affect the nuclear localization of c-Fos, suggesting alternative\nimport pathways.\nIn living cells, nuclear import of c-Fos does not occur by default,\nbecause the newly synthesized protein is retained in the cytoplasm in\nserum-starved cells (21). One example for a stimulus-dependent trans-\nport is the UV-induced phosphorylation of c-Fos by p38 kinases and the\nsubsequent import of the transcription factor into the nucleus (45). Our\nbacterially produced c-Fos (i.e. unphosphorylated protein) readily binds\nto transportin and importin , suggesting that phosphorylation of c-Fos\nmay rather regulate its cytoplasmic retention than its interaction with\ntransport receptors. Nevertheless, different signaling pathways could\nregulate nuclear import of c-Fos by different importins. Furthermore,\nnuclear import pathways of c-Fos could differ from cell type to cell type,\ndepending on relative expression levels of potential import receptors.\nInteraction of c-Fos with Importin  and Transportin—The basic\nregions in the DNA-binding domain of c-Fos have been implicated in\nnuclear import of the transcription factor (23). Indeed, deletions of one\nor more of these regions, as well as point mutations exchanging argin-\nines for alanines abolished the interaction of c-Fos with importin .\nThus, c-Fos contains a basic, arginine-rich NLS that can be recognized\nby importin , similar to other proteins (5–7). 19 HEAT repeats in\nimportin generate an extensive interaction surface that allows binding\nof a large variety of molecules, including RanGTP, nucleoporins, and\ndifferent import cargoes. Most prominent for cargo binding is the\nimportin -interaction domain in the C-terminal part of importin \n(43), comprising HEAT repeats 7–19 (46). In contrast, PTHrP binds to\nthe concave surface of the N-terminal part of importin , contacting\nHEAT repeats 2–11 (47). A third interaction mode is used by SREBP-2,\nwhich binds as a dimer to importin  (48). SREBP-2 has no typical\nconsecutive basic residues and the binding to importin  is primarily\nmediated by hydrophobic interactions, with HEAT repeats 7 and 17\nbeing of particular importance (48). c-Fos binds to the N-terminal por-\ntion of importin , possibly in a similar way as PTHrP. Unlike c-Fos,\nhowever, PTHrP can form a trimeric complex with importin  and\nimportin , at least in vitro (47), suggesting a somewhat different bind-\ning mode. A common feature of all importin -cargo complexes, includ-\ning the importin -c-Fos complex, is their sensitivity to RanGTP, which\nbinds to HEAT repeats 1– 8 at the N terminus of the receptor (49). Thus,\nregardless of the nature of the receptor-cargo complex, common mech-\nanism are at play for the dissociation of the complex by RanGTP in the\nnucleus.\nThe regions in c-Fos that are required for binding to transportin are\nclearly distinct from the basic regions needed for binding to importin ,\nbecause their deletion does not affect transportin binding. So far we\nhave been unable to generate point mutations that do not bind to trans-\nportin. Amino acids 111–124 of the c-Fos protein appear to be impor-\ntant for binding, because the interaction was lost when these residues\nwere deleted. This region, which is much shorter than the classic M9\nsequence and does not contain consecutive basic residues, is not suffi-\ncient for binding, as even longer fragments (e.g. Fos 81–124) did not\ninteract with transportin. Of course we cannot exclude problems result-\ning from misfolded proteins. Together, these results argue for a more\ncomplex mode of interaction, possibly involving residues at the N ter-\nminus of the transcription factor. By several criteria, however, the inter-\naction of transportin with c-Fos is distinct from its binding to its estab-\nlished import cargoes. First, the N terminus of transportin clearly\ninteracts specifically with c-Fos, in contrast to the classic import cargo,\nthe M9 sequence. We also detected binding of c-Fos to the C-terminal\npart of transportin, which also interacts with the M9 sequence (50).\nHence, there is probably more than one binding site on transportin for\nc-Fos, explaining the strong interaction of the importin with the tran-\nscription factor. Second, the transportin-binding fragment c-Fos 3–124\nhas no characteristic basic regions, unlike other import cargoes of trans-\nportin that do not contain an M9-like NLS (16). Therefore, the interac-\ntion of transportin with c-Fos is the first example of a third binding\nmode of the import receptor to a cargo molecule. The observation that\nbinding of the M9 sequence and of c-Fos to transportin are mutually\nexclusive argues for extensive conformational changes in the receptor\nmolecule upon cargo interaction, similar to what has been described for\nimportin (48, 51). Alternatively, the first cargo could mask the binding\nsite for the second cargo.\nIn summary, transportin may be as versatile as importin  when it\ncomes to the accommodation of a variety of import cargoes. A detailed\ncrystal structure analysis will be required to characterize the different\ntransportin-cargo complexes at the molecular level.",
         "Import of proteins into the nucleus is mediated by importins, mem-\nbers of the importin  superfamily. The “classic” signals for nuclear\nimport (nuclear localization signal; NLS)2 are short basic stretches of\namino acids with lysines as characteristic components, which may\noccur either as a single or a bipartite motif. These classic NLSs are\nrecognized by the adapter molecule importin , which forms a het-\nerodimer with the actual transport receptor, importin (for reviews see\nRefs. 1–3). The importin /-NLS-cargo complex is then translocated\nthrough the nuclear pore complex. Nucleoporins, the protein compo-\nnents of the nuclear pore complex, may interact with the transport\nreceptor, facilitating the translocation of the complex. Importin inter-\naction with RanGTP on the nuclear side of the nuclear pore complex\nresults in dissociation of the transport complex (4). Hundreds of indi-\nvidual nuclear proteins or proteins that shuttle between the nucleus and\nthe cytoplasm are thought to use this classic importin / nuclear\nimport pathway. Over the last couple of years, however, a number of\nproteins have been described that do not depend on the adapter protein\n* The work was partially supported by the Deutsche Forschungsgemeinschaft (Grant\nKe 660/2-2 to R. H. K.). The costs of publication of this article were defrayed in part by\nthe payment of page charges. This article must therefore be hereby marked “adver-\ntisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.\n2 The abbreviations used are: NLS, nuclear localization signal; AP-1, activator protein-1;\nBSA, bovine serum albumin; IBB, importin binding domain; LZ, leucine zipper; MBP,\nmaltose-binding protein; PTHrP, parathyroid hormone-related protein; SREBP-2, ste-\nrol regulatory element-binding protein 2; WGA, wheat germ agglutinin; GST, gluta-\nthione S-transferase; FITC, fluorescein isothiocyanate; HRP, horseradish peroxidase.\nimportin . Some of these bind directly to the classic import receptor,\nimportin , without requiring any adapter protein. These include the\nproteins Rev and Rex of the human immunodeficiency virus (5, 6) and\nthe human T-cell leukemia virus (7), respectively, the T-cell protein\ntyrosine phosphatase (8), the parathyroid hormone-related protein\n(PTHrP) (9), cyclin B1 (10), the sterol regulatory element-binding pro-\ntein 2 (SREBP-2 (11)), the zinc finger protein Snail (12), and the tran-\nscription factor CREB (13). Many of these proteins also contain basic\nNLSs. In contrast to the classic NLS, however, arginine is the predom-\ninant basic amino acid in these motives. Other import substrates do not\nbind to importin but to other members of the importin superfamily.\nThe best described member of this family (besides importin itself) is\ntransportin, which has originally been shown to serve as an import\nreceptor for the M9 sequence, a 38-amino acid domain in the heteroge-\nneous nuclear ribonucleoprotein A1 protein that functions as an NLS\n(14). Transportin is also an import receptor for other heterogeneous\nnuclear ribonucleoproteins (15) as well as for ribosomal proteins (16)\nand the nucleoporin Nup153 (17). Other members of the family are, for\nexample, importin 5 and importin 7, with ribosomal proteins and core\nhistones as import substrates (16), and importin 8, which mediates\nnuclear import of a protein of the signal recognition particle (18). Some\ncargo molecules like histones or certain ribosomal proteins can be\nimported by more than one receptor (16). For some importins, no\nimport cargo has been identified so far (3).\nAP-1 (activator protein-1), one of the first mammalian transcription\nfactors to be identified, converts short-term signals to long-lasting\nresponses in a large variety of complex cellular processes like transfor-\nmation, proliferation, differentiation, and apoptosis (for reviews see\nRefs. 19 and 20). The best characterized components of AP-1, which\nfunctions as a dimer of basic leucine zipper proteins, are c-Jun and\nc-Fos. c-Jun may form homodimers, but it interacts preferentially with\nc-Fos proteins,\nforming heterodimers. c-Fos, unable to form\nhomodimers, is a nuclear proto-oncogene that is expressed at low levels\nin most exponentially growing cells. Its expression level can be\nincreased by a variety of stimuli. After translation, c-Fos accumulates in\nthe nucleus very rapidly. Its nuclear import appears to be regulated by\nextracellular signals, as newly synthesized c-Fos accumulates in the\ncytoplasm in serum-deprived cells (21, 22). Different regions in the\nc-Fos protein are required for its efficient nuclear import (23, 24).\nNuclear transport of c-Fos and c-Jun appears to be independent of\nimportin , because both proteins interact directly with the transport\nreceptor importin . The apparent affinities of the transcription factors\nfor importin , however, are rather low (13), and importin -dependent\ntransport has not been demonstrated in a permeabilized cell system so\nfar.\nIn this study, we investigate nuclear import of c-Fos in detail, using\ndigitonin-permeabilized cells. Surprisingly, c-Fos still accumulated in\nthe nucleus, even when importin -dependent transport was strongly\ninhibited. We identified transportin as the major import receptor for\nc-Fos. Under certain conditions, importin  may also function as a\ntransport receptor for the transcription factor. A detailed biochemical\nanalysis revealed that the binding sites for c-Fos and classic import\ncargoes are distinct on both transportin and importin . Likewise, dis-\ntinct regions in c-Fos are involved in the interaction with either trans-\nport receptor. Regarding transportin, these results clearly point to a\nnovel mode of receptor-cargo interaction.",
         "MATERIALS AND METHODS\nCell Culture—Adherent HeLa cells were grown in Dulbecco’s modi-\nfied Eagle’s medium (Invitrogen) containing 4500 mg/liter glucose, 10%\nfetal calf serum, 2 mM glutamine, 100 units/ml penicillin, and 100 g/ml\nstreptomycin.\nMolecular Cloning—Plasmids for GST fusion proteins were gener-\nated as follows: c-Fos fragments were amplified by PCR from a codon-\noptimized sequence from rat (25, 26) and cloned into the BamHI and\nEcoRI sites (Fos 3–380, Fos 3–154, Fos 3–138, Fos 3–124, and Fos\n3–110) or the BamHI and HindIII sites (Fos 3–160, Fos 138 –160, and\nFos 81–160) of pGEX-KG. pGEX-KG-Fos R143A/R144A and pGEX-\nKG-Fos R157A/R158A were constructed by site-directed mutagenesis\nof pGEX-KG-Fos using standard protocols. The IBB domain of impor-\ntin (amino acids 1– 65) was amplified and cloned into the BamHI and\nHindIII sites of pGEX-KG. Transportin 1– 890, transportin 1–517, and\ntransportin 518 – 890 were amplified from pQE32-transportin (27) and\ncloned into the BamHI and SalI sites of pMal-c2 (New England Biolabs),\ngenerating pMal-c2-transportin F, pMal-c2-transportin N, and pMal-\nc2-transportin C, respectively. MBP-M9 was generated by digestion of\npGEX-M9 (14) with EcoRI and XhoI and blunt end ligation of the result-\ning insert into the EcoRI and HindIII sites of pMal-c2. MBP-IBB was\nconstructed by amplification of the IBB domain of importin and clon-\ning into the BamHI and HindIII sites of pMal-c2. All constructs were\nverified by DNA sequencing.\nProtein Purification and Labeling—Protein expression was induced\nwith 0.25 mM (MBP-tagged proteins) or 0.5 mM (all others) isopropyl-\nthio---galactoside at 25 °C for 2– 4 h. GST fusion proteins were\nexpressed in BL21(-DE3) cells and natively purified by single-step affin-\nity chromatography using glutathione-Sepharose beads (Amersham\nBiosciences), according to the instructions of the manufacturer. MBP\nfusion proteins were expressed in BL21 cells in the presence of 2% glu-\ncose. For MBP-transportin, MBP-transportin-N, MBP-transportin-C,\nand MBP-IBB, bacterial pellets were resuspended in buffer A (20 mM\nTris, pH 7.4, 200 mM NaCl, 5% glycerol) containing 1 mM dithiothreitol\nand 1 g/ml each of leupeptin, aprotinin, and pepstatin. After sonifica-\ntion, lysates were cleared by centrifugation at 100,000  g and incubated\nwith amylose beads (New England Biolabs) for 4 h at 4 °C. Thebeads\nwere washed with buffer A, and bound proteins were eluted with buffer\nA containing 10 mM maltose, 1 mM dithiothreitol, and protease inhibi-\ntors. For purification of MBP-M9, the bacterial pellet was resuspended\nin buffer B (20 mM Tris, pH 7.4, 50 mM NaCl, 5% glycerol) containing 1\nmM dithiothreitol and protease inhibitors as above. After sonification\nand centrifugation at 100,000  g, the lysate was incubated with DEAE-\nSepharose (Sigma). The flow-through was collected and subjected to a\n40 –75% saturation ammonium sulfate cut, followed by gel-filtration on\na S200 column (Amersham Pharmacia Biotech). His-Fos was expressed\nin BL21(-DE3) cells and natively purified using nickel-nitrilotriacetic\nacid-agarose (Qiagen), according to the instructions of the manufac-\nturer. Importin  1–396 and 304 – 876 (28) were expressed in BL21(-\nDE3) and purified via their N-terminal His tags using nickel-nitrilotri-\nacetic acid-agarose. Bound proteins were eluted with 300 mM imidazole\nand supplemented with 30 mM dithiothreitol, inducing cleavage of the\nTransportin-mediated Nuclear Import of c-Fos\nC-terminal intein tag. The tag was removed by addition of chitin beads\n(New England Biolabs). His-S-Importin  (43), His-transportin (27),\nHis-importin 5 (16), His-importin 7 (16), wild-type Ran, and RanQ69L\n(29) were purified as described. All GST fusion and MBP fusion pro-\nteins, except of MBP-M9, were dialyzed against 20 mM Tris, pH 7.4, 200\nmM NaCl, 5% glycerol. MBP-M9, transport receptors, Ran, and\nRanQ69L were dialyzed against transport buffer (20 mM Hepes, pH 7.3,\n110 mM KOAc, 2 mM Mg(OAc)2, 1 mM EGTA). For binding studies,\nRanQ69L was loaded with GTP as described (30). His-Fos was dialyzed\nagainst 100 mM sodium carbonate buffer, pH 9.3, and labeled with Cy3\n(Amersham Biosciences), according to the instructions of the manufac-\nturer. Preparation of FITC-BSA-NLS was essentially as described (31).\nAll proteins were frozen in liquid nitrogen and stored at 80 °C.\nBinding Studies—Binding of cytosolic proteins to c-Fos: 200 g of\nGST, GST-Fos, or GST-Fos3–160 were immobilized on 100 l of glu-\ntathione-Sepharose (Amersham Biosciences) and incubated with 600 l\nof HeLa cell cytosol (32); 10 mg/ml, in binding buffer (50 mM Tris pH\n7.4, 150 mM NaCl, 1 mM MgCl2, 5% glycerol), that had been preincu-\nbated with or without 10 M RanQ69LGTP. After incubation for 4 h at\n4 °C, beads were washed three times with binding buffer, and bound\nmaterial was eluted with 120 l of 2 SDS buffer. 5 l each were ana-\nlyzed by SDS-PAGE, followed by Western blotting or silver staining. 60\nl were analyzed by SDS-PAGE followed by Coomassie staining and\nmass spectrometry. Binding of recombinant transport receptors to\nc-Fos: GST or GST fusion proteins were immobilized on glutathione-\nSepharose beads that had been preincubated with 10 mg/ml BSA to a\nfinal concentration of 1 g/l. 5 g of recombinant His-tagged or MBP-\ntagged transport receptors in 400 l of binding buffer (50 mM Tris, pH\n7.4, 200 mM NaCl, 1 mM MgCl2, 5% glycerol, 2 mg/ml BSA) were added\nto 5 l of beads. In some reactions, transport receptors were preincu-\nbated with 2 M RanQ69LGTP or a 20-fold molar excess (relative to the\nimmobilized proteins) of MBP-IBB and MBP-M9 as competitors. After\nincubation for 2 h at 4 °C, beads were washed three times with binding\nbuffer without BSA, and bound proteins were eluted with 2 SDS\nbuffer. In experiments where relative affinities between c-Fos and\nimport receptors were measured, importin without an affinity tag was\nused (purified by ammonium sulfate precipitation and gel filtration; a\nkind gift from F. Melchior). In binding studies with His-S-importin , 5\ng of this import receptor was immobilized on 5 l of S-protein-agarose\n(Amersham Biosciences) and incubated with an equimolar amount of\nGST-Fos, preincubated with or without an equimolar amount or a\n10-fold excess of transportin.\nMass Spectrometry—Protein digestion was performed essentially as\nreported (33), and peptides were desalted and concentrated using C18\nextraction tips (34). The peptide mixture was analyzed with an Ultimate\nnanoHPLC system (LC Packings) coupled to an ion trap mass spec-\ntrometer (LCQ, Thermo Finnigan) using a C18 reversed-phase column\n(PepMap, LC Packings). Peak lists were searched against the Swiss-Prot\nProtein Sequence Data Base.\nIn Vitro Import Assays—For nuclear import assays (36), HeLa cells\nwere grown on 10-well slides (Roth), permeabilized with 30 g/ml dig-\nitonin (Calbiochem) in transport buffer on ice, and washed. In some\nexperiments, permeabilized cells were preincubated with 200 g/ml\nwheat germ agglutinin (WGA, Sigma) in transport buffer for 12 min on\nice. Each import mixture contained 250 –500 nM import substrates, 4\nM Ran or RanQ69L, 2 mM GTP, 3 mg/ml BSA, an ATP-regenerating\nsystem (32), and either 2 mg/ml HeLa cytosol (in transport buffer) or\n500 nM His-tagged importins. In competition experiments, HeLa\ncytosol was preincubated with 3 l of a monoclonal anti-importin -an-\ntibody (Affinity BioReagents, clone 3E9 (32)) or with 20 M of MBP-M9\nTransportin-mediated Nuclear Import of c-Fos\nas competitor. Import reactions were performed for 30 min at 25 °C for\nc-Fos or at 30 °C for GST-NLS, BSA-NLS, and GST-M9. After washing,\ncells incubated with Cy3-Fos or FITC-BSA-NLS were fixed with 3.7%\nformaldehyde and analyzed directly by fluorescence microscopy. Cells\nincubated with GST-tagged import substrates were first subjected to\nindirect immunostaining using an anti-GST-antibody (Amersham Bio-\nsciences) and Alexa 594 donkey-anti-goat (Molecular Probes). Cells\nwere analyzed by fluorescence microscopy using a Zeiss Axioskop2\nmicroscope. Pictures were processed using Adobe Photoshop.\nSDS-PAGE and Western Blotting—Proteins were separated by SDS-\nPAGE and subjected to Western blotting using standard methods. Rab-\nbit anti-importin and anti-importin 7 antibodies were a kind gift from\nD. Go¨rlich (35, 52). The rabbit-anti-transportin antibody was a kind gift\nfrom L. Gerace. Rabbit anti-maltose-binding protein (MBP) was pur-\nchased from New England Biolabs. The mouse anti-His antibody was\nfrom Covance. HRP-coupled goat anti rabbit or goat anti mouse IgG\n(Dianova) was used as secondary antibody. The ECL system (Pierce) was\nused for visualization of proteins.",
         "RESULTS\nImportin -independent Nuclear Import of c-Fos in Vitro—The tran-\nscription factor c-Fos has been suggested to be imported into the\nnucleus by the transport receptor importin , independent of the\nadapter protein importin (13). Importin -dependent import of c-Fos\nin permeabilized cells, however, has not been demonstrated so far. To\nanalyze nuclear transport of c-Fos in detail, we established an import\nassay in digitonin-permeabilized cells (36). As shown in Fig. 1A, c-Fos\nwas efficiently imported into nuclei of permeabilized cells at 25 °C in the\npresence of cytosol and wild-type Ran. Transport was energy-depend-\nent, as it was blocked at 4 °C, and required the addition of soluble fac-\ntors, as no nuclear accumulation was observed in cells incubated with\nbuffer alone. To test whether nuclear import of c-Fos depends on trans-\nport receptors of the importin -superfamily, a reaction was performed\nin the presence of RanQ69L, a Ran-mutant that is predominantly in the\nGTP-bound form (37), preventing the interaction of import substrates\nwith import receptors (4). RanQ69L strongly inhibited nuclear accumu-\nlation of c-Fos, demonstrating the specificity of the nuclear transport\nreaction. We next used our in vitro transport assay to test if import of\nc-Fos depends on the activity of importin . Importin -dependent\ntransport can be inhibited with a monoclonal antibody against this\ntransport receptor (38). Preincubation of cytosol with this antibody\nresulted in strong inhibition of nuclear import of GST-NLS, a substrate\nthat uses the importin / dimer as a transport receptor (Fig. 1B). In\ncontrast, nuclear import of GST-M9, a substrate for the import receptor\ntransportin, was not affected by the anti-importin antibody. Likewise,\nnuclear import of c-Fos was not significantly reduced upon preincuba-\ntion of cytosol with the anti-importin antibody. These results strongly\nsuggest that cytosolic transport factors distinct from importin  can\nmediate nuclear import of c-Fos.\nc-Fos Interacts with Different Import Receptors—To identify trans-\nport receptors that specifically interact with c-Fos, a GST fusion protein\nof the transcription factor was immobilized on Sepharose beads and\nincubated with cytosol in the absence or presence of RanQ69L. This\nRan-mutant inhibited nuclear import of c-Fos (Fig. 1A), and we there-\nfore expected it to release transport-relevant interaction partners from\nthe immobilized bait. As shown in Fig. 2A, two proteins 100 and 120\nkDa specifically bound to GST-Fos in the absence but not in the pres-\nence of RanQ69L. To identify these potential import receptors for c-Fos,\nthe bands were cut from the gel and tryptic digests of proteins were\nanalyzed by mass spectrometry. In each band, two proteins could\nunequivocally be identified. The strong band at 100 kDa contained\nimportin and transportin (specifically, transportin 1 (14)), whereas the\nweaker band at 120 kDa contained importins 7 and 8. We next used\nspecific antibodies against three of these four import receptors to verify\ntheir specific interaction with GST-Fos. Importin , importin 7, as well\nas transportin specifically bound to GST-Fos in the absence but not in\nthe presence of RanQ69L (Fig. 2B). No binding to immobilized GST was\ndetected. Strikingly, a strong accumulation of transportin on the GST-\nFos beads was observed, whereas only a rather small proportion of\nimportin or importin 7 was bound to the beads (compare “input” and\n“GST-Fos”). c-Fos contains a leucine-zipper and forms stable dimers\nwith proteins like c-Jun. To exclude the possibility that the identified\nimport receptors bound indirectly to c-Fos via proteins that interact\nwith its leucine-zipper, we expressed an N-terminal fragment of c-Fos,\nlacking this protein interaction domain (GST-Fos 3–160; see also Fig.\n5A). Binding of importin 7, importin , and transportin to this fragment\nwas as strong as to the full-length protein (Fig. 2B), suggesting a direct\ninteraction. To demonstrate such a direct interaction of import recep-\ntors with c-Fos, importin 7, importin , and transportin, as well as\nimportin 5 as a negative control, were expressed in bacteria and incu-\nbated with immobilized GST-Fos. Importin 5 did not interact with\nGST-Fos under these conditions (Fig. 2C). Importin and transportin\nbound efficiently to GST-Fos, whereas importin 7 bound rather weakly\nTransportin-mediated Nuclear Import of c-Fos\n(compare “GST-Fos” to “input”). Binding of importin and transportin\nwas abolished when RanQ69L was included in the reaction, demon-\nstrating the specificity of the interaction. Binding of importin 7 to GST-\nFos was less sensitive to RanQ69L, probably reflecting the lower affinity\nof this transport receptor for RanGTP, compared with importin (35).\nImportin 7 has been shown to form a heterodimer with importin (35).\nThis dimer serves as an import receptor for histone H1 (39). We there-\nfore combined importin  and importin 7 in the binding reaction and\nnow observed increased binding of importin 7 (together with importin\n) to GST-Fos (Fig. 2C). This result suggests that at least part of the\ncytosolic importin 7 that interacted with GST-Fos (Fig. 2A) was indi-\nrectly bound via importin . Taken together, different transport recep-\ntors of the importin  superfamily specifically interact with c-Fos and\nmay therefore function as physiological import receptors.\nTo test which of the identified import receptors promote efficient\nnuclear transport of c-Fos, we performed import reactions in digitonin-\npermeabilized cells, using recombinant proteins. Cells were incubated\neither at 4 °C or 25 °C with equimolar concentrations of the various\ntransport receptors. At 25 °C, transportin promoted the strongest accu-\nmulation of c-Fos in the nucleus (Fig. 3A). Importin  also mediated\nnuclear import of c-Fos, yet to a somewhat lower extent. Importins 5\nand 7 did not promote nuclear import of c-Fos, although they were\ntransport-competent for their established import cargo, the ribosomal\nprotein L23 (data not shown (16)). Combining importin with its bind-\ning partner importin 7 did not further promote nuclear import, com-\npared with a reaction with importin  alone. Thus, transportin and\nimportin  appear to be the major transport receptors for nuclear\nimport of c-Fos. Transport was clearly specific, because it was temper-\nature-dependent (Fig. 3A) and could be inhibited by RanQ69L and by\nWGA, a lectin that binds to O-linked nucleoporins (40) and blocks\nvarious nucleocytoplasmic transport pathways (41, 42) (Fig. 3B).\nCharacterization of the Interaction between c-Fos and Transportin or\nImportin —Having established that at least two importins, transportin\nand importin , may serve as import receptors for c-Fos, we next ana-\nlyzed the interaction of the transcription factor with these receptors in\ndetail. We first expressed N- and C-terminal fragments of both importin\nand transportin and tested them for interaction with either an estab-\nlished binding partner or with c-Fos. A GST fusion of the importin \nbinding (IBB-) domain of importin  was immobilized on beads and\nincubated with full-length importin , or with its N- or C-terminal\nTransportin-mediated Nuclear Import of c-Fos\nfragment. As expected (43), the IBB-domain preferentially bound to the\nC-terminal fragment of importin (Fig. 4A). In contrast, when GST-Fos\nwas immobilized, the N-terminal, but not the C-terminal fragment of\nimportin  was retained on the beads. No binding was observed when\nGST alone was used as a bait.\nIn a similar set of experiments, we immobilized GST, GST-M9, or\nGST-Fos on beads and incubated them with full-length transportin or\nits N- or C-terminal fragments (Fig. 4B). In these experiments, trans-\nportin fragments fused to the maltose binding protein (MBP) were used,\nbecause His-tagged fragments were insoluble. Binding of the MBP-\ntransportin fragments to GST beads was not observed. GST-M9 inter-\nacted with full-length transportin as well as with the C-terminal frag-\nment, but not with the N-terminal fragment. GST-Fos, in contrast,\ninteracted with all three proteins. Binding of free MBP to the beads was\nnot detected. Interaction of full-length transportin with GST-Fos was\nspecific, as it could be blocked by RanQ69L (see Fig. 2C), suggesting that\nthe interaction of c-Fos with the two individual fragments of transportin\nis specific, too. We could directly test this assumption for the N-termi-\nnal fragment, which contains the Ran-binding site of transportin. As\nshown in Fig. 4C, the interaction of GST-Fos with this fragment was\nlargely inhibited when RanQ69L was included in the reaction. These\nresults show that c-Fos interacts with different regions of both importin\n and transportin compared with their classic binding partners, the\nIBB-domain of importin and the M9 sequence, respectively.\nIn light of this differential binding, we next analyzed whether these\nclassic binding partners and c-Fos could interact simultaneously with\neither importin  or transportin. Immobilized GST-IBB or GST-Fos\nwas incubated with importin , in the absence or presence of an excess\nof the IBB-domain, coupled to the maltose binding protein (MBP-IBB).\nAs expected, no binding of importin to GST-IBB was observed when\nMBP-IBB was present (Fig. 4D, top panel). MBP-IBB also abolished the\ninteraction of importin with c-Fos, suggesting that binding of IBB and\nc-Fos to importin  are mutually exclusive. Similarly, an M9 sequence\ncoupled to MBP prevented the interaction of transportin with both\nGST-M9 and GST-Fos (Fig. 4D, bottom panel). These results show that\nthe two cargo binding sites on importin  and transportin cannot be\noccupied simultaneously by the classic cargoes of the import receptors\nand by c-Fos. This may result from conformational changes in the struc-\nture of the receptors upon binding one type of ligand, preventing simul-\ntaneous binding of a second ligand to a different binding site.\nTo characterize the regions in c-Fos required for interaction with\nimportin and transportin, we generated a number of fragments of the\ntranscription factor as well as two point mutations as GST fusion pro-\nteins (Fig. 5A). These proteins were immobilized on beads and incu-\nbated with either full-length importin  or full-length transportin. A\nfragment, where the C-terminal part of c-Fos, including the leucine-\nzipper was deleted, interacted with importin (Fig. 5B) and transportin\n(Fig. 5C), similar to the full-length protein (compare lanes 1 and 2).\nThus, the C-terminal part of c-Fos beyond the basic DNA-binding\ndomain (“ ” in Fig. 5A) is not required for interaction with the\ntransport receptors. The DNA-binding domain has been reported to\nfunction as an NLS (23). Indeed, when one, two or three of these basic\nregions were deleted, the corresponding c-Fos fragment failed to inter-\nact with importin (Fig. 5B, lanes 3–5). Similarly, when two arginines in\neither the second or the third basic region were mutated to alanines,\nfull-length c-Fos did not bind to importin  (Fig. 5B, lanes 9 and 10).\nStrikingly, however, all these fragments/mutants still interacted with\ntransportin (Fig. 5C, lanes 3–5, 9, and 10), demonstrating that different\nregions in c-Fos must be responsible for binding to the different trans-\nport receptors. Accordingly, a short fragment comprised of the second\nand third basic region of c-Fos bound to importin (Fig. 5B, lane 7) but\nhardly to transportin (Fig. 5C, lane 7). In an attempt to further narrow\ndown the binding site on c-Fos for transportin, we tested other frag-\nments of the transcription factor. Fragments comprising amino acids\n3–124 or 81–160 efficiently bound to transportin (Fig. 5, lanes 5 and 8),\nwhereas a fragment comprising amino acids 3–110 did not (Fig. 5, lane\n6). These results suggested that amino acids 81–124 (overlap between\nfragments 5 and 8) or 111–124 (deleted residues in fragment 6 com-\npared with fragment 5) are required for binding. A fragment comprising\namino acids 81–124, however, did not bind to transportin (data not\nshown), arguing for a more complex mechanism of interaction.\nAs importin and transportin bind to different regions of c-Fos, we\nnext asked whether the transcription factor could interact simulta-\nTransportin-mediated Nuclear Import of c-Fos\nneously with the two transport receptors. Importin was immobilized\non beads and incubated with GST-Fos in the absence or presence of\ntransportin. As shown in Fig. 5D, an equimolar concentration of trans-\nportin was sufficient to abolish binding of GST-Fos to importin , sug-\ngesting that the proteins cannot form a trimeric complex.\nTransportin Is the Major Import Receptor for c-Fos—Our results have\nshown that c-Fos can be imported into the nucleus by two transport\nreceptors, importin , and transportin. Is there any preference for one of\nthese factors? To answer this question, we first compared relative affin-\nities of importin and transportin for c-Fos. GST-Fos was immobilized\non beads and incubated either with transportin or importin alone, or\nwith transportin in combination with increasing concentrations of\nimportin . The amounts of GST-Fos and import receptors were cho-\nsen such that GST-Fos would be limiting for the binding reactions.\nTransportin and importin  efficiently interacted with GST-Fos when\nthey were added separately to the reaction (Fig. 6A). When the two\ntransport receptors were present at equimolar concentrations, trans-\nportin still bound efficiently to GST-Fos, whereas importin  was\nalmost undetectable, in agreement with the result presented in Fig. 5D.\nEven at a 20-fold molar excess of importin over transportin, there was\nstill substantial binding of transportin to GST-Fos, suggesting that\ntransportin has a much higher affinity for the transcription factor com-\npared with importin . We next tested whether fragments or mutants of\nc-Fos that interact with transportin but not with importin  could be\nimported efficiently into nuclei of digitonin-permeabilized cells in the\npresence of cytosol. As shown in Fig. 6 (B and C), the fragment 3–154 as\nwell as the mutant R157A/R158A (compare Fig. 5A) were imported at\n25 °C but not at 4 °C. Import was specific, as it could be blocked by\nRanQ69L or by WGA. Under conditions where importin -dependent\ntransport was inhibited, import of c-Fos still occurred (Fig. 1B). We now\nfinally tested, if inhibition of the transportin-dependent nuclear import\npathway would abolish transport of c-Fos into the nucleus. Permeabi-\nlized cells were incubated with the transport substrates GST-M9, BSA-\nNLS, or GST-Fos with cytosol as the source of transport factors. All\nsubstrates were imported efficiently when no inhibitor was present (Fig.\n6D). When we included MBP-M9 in the reaction, a substrate that com-\npetes with transportin-dependent cargoes for efficient import, we\nobserved strong inhibition of transport of GST-M9, as well as of c-Fos.\nThe importin /-dependent substrate BSA-NLS was not affected by\nMBP-M9, demonstrating the specificity of inhibition. Thus, under con-\nTransportin-mediated Nuclear Import of c-Fos\nditions where importin -dependent transport still occurs, inhibition of\ntransportin abolishes the nuclear import of c-Fos, suggesting that trans-\nportin is the major cytosolic transport receptor for the transcription\nfactor under our experimental conditions.",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf",
         "c-Fos, a component of the transcription factor AP-1, is rapidly\nimported into the nucleus after translation. We established an in\nvitro system using digitonin-permeabilized cells to analyze nuclear\nimport of c-Fos in detail. Two import receptors of the importin \nsuperfamily, importin  itself and transportin, promote import of\nc-Fos in vitro. Under conditions where importin -dependent\ntransport was blocked, c-Fos still accumulated in the nucleus in the\npresence of cytosol. Inhibition of the transportin-dependent path-\nway, in contrast, abolished import of c-Fos. Furthermore, c-Fos\nmutants that interact with transportin but not with importin were\nefficiently imported in the presence of cytosol. Hence, transportin\nappears to be the predominant import receptor for c-Fos. A detailed\nbiochemical characterization revealed that the interaction of trans-\nportin with c-Fos is distinct from the interaction with its established\nimport cargoes, the M9 sequence of heterogeneous nuclear ribonu-\ncleoprotein A1 or the nuclear localization sequence of some basic\nproteins. Likewise, the binding sites on importin  for its classic\nimport cargo and for c-Fos can be separated. In summary, c-Fos\nemploys a novel mode of receptor-cargo interaction. Hence, trans-\nportin may be as versatile as importin  in recognizing different\nnuclear import cargoes.\nImport of proteins into the nucleus is mediated by importins, mem-\nbers of the importin  superfamily. The “classic” signals for nuclear\nimport (nuclear localization signal; NLS)2 are short basic stretches of\namino acids with lysines as characteristic components, which may\noccur either as a single or a bipartite motif. These classic NLSs are\nrecognized by the adapter molecule importin , which forms a het-\nerodimer with the actual transport receptor, importin (for reviews see\nRefs. 1–3). The importin /-NLS-cargo complex is then translocated\nthrough the nuclear pore complex. Nucleoporins, the protein compo-\nnents of the nuclear pore complex, may interact with the transport\nreceptor, facilitating the translocation of the complex. Importin inter-\naction with RanGTP on the nuclear side of the nuclear pore complex\nresults in dissociation of the transport complex (4). Hundreds of indi-\nvidual nuclear proteins or proteins that shuttle between the nucleus and\nthe cytoplasm are thought to use this classic importin / nuclear\nimport pathway. Over the last couple of years, however, a number of\nproteins have been described that do not depend on the adapter protein\n* The work was partially supported by the Deutsche Forschungsgemeinschaft (Grant\nKe 660/2-2 to R. H. K.). The costs of publication of this article were defrayed in part by\nthe payment of page charges. This article must therefore be hereby marked “adver-\ntisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.\n2 The abbreviations used are: NLS, nuclear localization signal; AP-1, activator protein-1;\nBSA, bovine serum albumin; IBB, importin binding domain; LZ, leucine zipper; MBP,\nmaltose-binding protein; PTHrP, parathyroid hormone-related protein; SREBP-2, ste-\nrol regulatory element-binding protein 2; WGA, wheat germ agglutinin; GST, gluta-\nthione S-transferase; FITC, fluorescein isothiocyanate; HRP, horseradish peroxidase.\nimportin . Some of these bind directly to the classic import receptor,\nimportin , without requiring any adapter protein. These include the\nproteins Rev and Rex of the human immunodeficiency virus (5, 6) and\nthe human T-cell leukemia virus (7), respectively, the T-cell protein\ntyrosine phosphatase (8), the parathyroid hormone-related protein\n(PTHrP) (9), cyclin B1 (10), the sterol regulatory element-binding pro-\ntein 2 (SREBP-2 (11)), the zinc finger protein Snail (12), and the tran-\nscription factor CREB (13). Many of these proteins also contain basic\nNLSs. In contrast to the classic NLS, however, arginine is the predom-\ninant basic amino acid in these motives. Other import substrates do not\nbind to importin but to other members of the importin superfamily.\nThe best described member of this family (besides importin itself) is\ntransportin, which has originally been shown to serve as an import\nreceptor for the M9 sequence, a 38-amino acid domain in the heteroge-\nneous nuclear ribonucleoprotein A1 protein that functions as an NLS\n(14). Transportin is also an import receptor for other heterogeneous\nnuclear ribonucleoproteins (15) as well as for ribosomal proteins (16)\nand the nucleoporin Nup153 (17). Other members of the family are, for\nexample, importin 5 and importin 7, with ribosomal proteins and core\nhistones as import substrates (16), and importin 8, which mediates\nnuclear import of a protein of the signal recognition particle (18). Some\ncargo molecules like histones or certain ribosomal proteins can be\nimported by more than one receptor (16). For some importins, no\nimport cargo has been identified so far (3).\nAP-1 (activator protein-1), one of the first mammalian transcription\nfactors to be identified, converts short-term signals to long-lasting\nresponses in a large variety of complex cellular processes like transfor-\nmation, proliferation, differentiation, and apoptosis (for reviews see\nRefs. 19 and 20). The best characterized components of AP-1, which\nfunctions as a dimer of basic leucine zipper proteins, are c-Jun and\nc-Fos. c-Jun may form homodimers, but it interacts preferentially with\nc-Fos proteins,\nforming heterodimers. c-Fos, unable to form\nhomodimers, is a nuclear proto-oncogene that is expressed at low levels\nin most exponentially growing cells. Its expression level can be\nincreased by a variety of stimuli. After translation, c-Fos accumulates in\nthe nucleus very rapidly. Its nuclear import appears to be regulated by\nextracellular signals, as newly synthesized c-Fos accumulates in the\ncytoplasm in serum-deprived cells (21, 22). Different regions in the\nc-Fos protein are required for its efficient nuclear import (23, 24).\nNuclear transport of c-Fos and c-Jun appears to be independent of\nimportin , because both proteins interact directly with the transport\nreceptor importin . The apparent affinities of the transcription factors\nfor importin , however, are rather low (13), and importin -dependent\ntransport has not been demonstrated in a permeabilized cell system so\nfar.\nIn this study, we investigate nuclear import of c-Fos in detail, using\ndigitonin-permeabilized cells. Surprisingly, c-Fos still accumulated in\nthe nucleus, even when importin -dependent transport was strongly\ninhibited. We identified transportin as the major import receptor for\nc-Fos. Under certain conditions, importin  may also function as a\ntransport receptor for the transcription factor. A detailed biochemical\nanalysis revealed that the binding sites for c-Fos and classic import\ncargoes are distinct on both transportin and importin . Likewise, dis-\ntinct regions in c-Fos are involved in the interaction with either trans-\nport receptor. Regarding transportin, these results clearly point to a\nnovel mode of receptor-cargo interaction.\nMATERIALS AND METHODS\nCell Culture—Adherent HeLa cells were grown in Dulbecco’s modi-\nfied Eagle’s medium (Invitrogen) containing 4500 mg/liter glucose, 10%\nfetal calf serum, 2 mM glutamine, 100 units/ml penicillin, and 100 g/ml\nstreptomycin.\nMolecular Cloning—Plasmids for GST fusion proteins were gener-\nated as follows: c-Fos fragments were amplified by PCR from a codon-\noptimized sequence from rat (25, 26) and cloned into the BamHI and\nEcoRI sites (Fos 3–380, Fos 3–154, Fos 3–138, Fos 3–124, and Fos\n3–110) or the BamHI and HindIII sites (Fos 3–160, Fos 138 –160, and\nFos 81–160) of pGEX-KG. pGEX-KG-Fos R143A/R144A and pGEX-\nKG-Fos R157A/R158A were constructed by site-directed mutagenesis\nof pGEX-KG-Fos using standard protocols. The IBB domain of impor-\ntin (amino acids 1– 65) was amplified and cloned into the BamHI and\nHindIII sites of pGEX-KG. Transportin 1– 890, transportin 1–517, and\ntransportin 518 – 890 were amplified from pQE32-transportin (27) and\ncloned into the BamHI and SalI sites of pMal-c2 (New England Biolabs),\ngenerating pMal-c2-transportin F, pMal-c2-transportin N, and pMal-\nc2-transportin C, respectively. MBP-M9 was generated by digestion of\npGEX-M9 (14) with EcoRI and XhoI and blunt end ligation of the result-\ning insert into the EcoRI and HindIII sites of pMal-c2. MBP-IBB was\nconstructed by amplification of the IBB domain of importin and clon-\ning into the BamHI and HindIII sites of pMal-c2. All constructs were\nverified by DNA sequencing.\nProtein Purification and Labeling—Protein expression was induced\nwith 0.25 mM (MBP-tagged proteins) or 0.5 mM (all others) isopropyl-\nthio---galactoside at 25 °C for 2– 4 h. GST fusion proteins were\nexpressed in BL21(-DE3) cells and natively purified by single-step affin-\nity chromatography using glutathione-Sepharose beads (Amersham\nBiosciences), according to the instructions of the manufacturer. MBP\nfusion proteins were expressed in BL21 cells in the presence of 2% glu-\ncose. For MBP-transportin, MBP-transportin-N, MBP-transportin-C,\nand MBP-IBB, bacterial pellets were resuspended in buffer A (20 mM\nTris, pH 7.4, 200 mM NaCl, 5% glycerol) containing 1 mM dithiothreitol\nand 1 g/ml each of leupeptin, aprotinin, and pepstatin. After sonifica-\ntion, lysates were cleared by centrifugation at 100,000  g and incubated\nwith amylose beads (New England Biolabs) for 4 h at 4 °C. Thebeads\nwere washed with buffer A, and bound proteins were eluted with buffer\nA containing 10 mM maltose, 1 mM dithiothreitol, and protease inhibi-\ntors. For purification of MBP-M9, the bacterial pellet was resuspended\nin buffer B (20 mM Tris, pH 7.4, 50 mM NaCl, 5% glycerol) containing 1\nmM dithiothreitol and protease inhibitors as above. After sonification\nand centrifugation at 100,000  g, the lysate was incubated with DEAE-\nSepharose (Sigma). The flow-through was collected and subjected to a\n40 –75% saturation ammonium sulfate cut, followed by gel-filtration on\na S200 column (Amersham Pharmacia Biotech). His-Fos was expressed\nin BL21(-DE3) cells and natively purified using nickel-nitrilotriacetic\nacid-agarose (Qiagen), according to the instructions of the manufac-\nturer. Importin  1–396 and 304 – 876 (28) were expressed in BL21(-\nDE3) and purified via their N-terminal His tags using nickel-nitrilotri-\nacetic acid-agarose. Bound proteins were eluted with 300 mM imidazole\nand supplemented with 30 mM dithiothreitol, inducing cleavage of the\nTransportin-mediated Nuclear Import of c-Fos\nC-terminal intein tag. The tag was removed by addition of chitin beads\n(New England Biolabs). His-S-Importin  (43), His-transportin (27),\nHis-importin 5 (16), His-importin 7 (16), wild-type Ran, and RanQ69L\n(29) were purified as described. All GST fusion and MBP fusion pro-\nteins, except of MBP-M9, were dialyzed against 20 mM Tris, pH 7.4, 200\nmM NaCl, 5% glycerol. MBP-M9, transport receptors, Ran, and\nRanQ69L were dialyzed against transport buffer (20 mM Hepes, pH 7.3,\n110 mM KOAc, 2 mM Mg(OAc)2, 1 mM EGTA). For binding studies,\nRanQ69L was loaded with GTP as described (30). His-Fos was dialyzed\nagainst 100 mM sodium carbonate buffer, pH 9.3, and labeled with Cy3\n(Amersham Biosciences), according to the instructions of the manufac-\nturer. Preparation of FITC-BSA-NLS was essentially as described (31).\nAll proteins were frozen in liquid nitrogen and stored at 80 °C.\nBinding Studies—Binding of cytosolic proteins to c-Fos: 200 g of\nGST, GST-Fos, or GST-Fos3–160 were immobilized on 100 l of glu-\ntathione-Sepharose (Amersham Biosciences) and incubated with 600 l\nof HeLa cell cytosol (32); 10 mg/ml, in binding buffer (50 mM Tris pH\n7.4, 150 mM NaCl, 1 mM MgCl2, 5% glycerol), that had been preincu-\nbated with or without 10 M RanQ69LGTP. After incubation for 4 h at\n4 °C, beads were washed three times with binding buffer, and bound\nmaterial was eluted with 120 l of 2 SDS buffer. 5 l each were ana-\nlyzed by SDS-PAGE, followed by Western blotting or silver staining. 60\nl were analyzed by SDS-PAGE followed by Coomassie staining and\nmass spectrometry. Binding of recombinant transport receptors to\nc-Fos: GST or GST fusion proteins were immobilized on glutathione-\nSepharose beads that had been preincubated with 10 mg/ml BSA to a\nfinal concentration of 1 g/l. 5 g of recombinant His-tagged or MBP-\ntagged transport receptors in 400 l of binding buffer (50 mM Tris, pH\n7.4, 200 mM NaCl, 1 mM MgCl2, 5% glycerol, 2 mg/ml BSA) were added\nto 5 l of beads. In some reactions, transport receptors were preincu-\nbated with 2 M RanQ69LGTP or a 20-fold molar excess (relative to the\nimmobilized proteins) of MBP-IBB and MBP-M9 as competitors. After\nincubation for 2 h at 4 °C, beads were washed three times with binding\nbuffer without BSA, and bound proteins were eluted with 2 SDS\nbuffer. In experiments where relative affinities between c-Fos and\nimport receptors were measured, importin without an affinity tag was\nused (purified by ammonium sulfate precipitation and gel filtration; a\nkind gift from F. Melchior). In binding studies with His-S-importin , 5\ng of this import receptor was immobilized on 5 l of S-protein-agarose\n(Amersham Biosciences) and incubated with an equimolar amount of\nGST-Fos, preincubated with or without an equimolar amount or a\n10-fold excess of transportin.\nMass Spectrometry—Protein digestion was performed essentially as\nreported (33), and peptides were desalted and concentrated using C18\nextraction tips (34). The peptide mixture was analyzed with an Ultimate\nnanoHPLC system (LC Packings) coupled to an ion trap mass spec-\ntrometer (LCQ, Thermo Finnigan) using a C18 reversed-phase column\n(PepMap, LC Packings). Peak lists were searched against the Swiss-Prot\nProtein Sequence Data Base.\nIn Vitro Import Assays—For nuclear import assays (36), HeLa cells\nwere grown on 10-well slides (Roth), permeabilized with 30 g/ml dig-\nitonin (Calbiochem) in transport buffer on ice, and washed. In some\nexperiments, permeabilized cells were preincubated with 200 g/ml\nwheat germ agglutinin (WGA, Sigma) in transport buffer for 12 min on\nice. Each import mixture contained 250 –500 nM import substrates, 4\nM Ran or RanQ69L, 2 mM GTP, 3 mg/ml BSA, an ATP-regenerating\nsystem (32), and either 2 mg/ml HeLa cytosol (in transport buffer) or\n500 nM His-tagged importins. In competition experiments, HeLa\ncytosol was preincubated with 3 l of a monoclonal anti-importin -an-\ntibody (Affinity BioReagents, clone 3E9 (32)) or with 20 M of MBP-M9\nTransportin-mediated Nuclear Import of c-Fos\nas competitor. Import reactions were performed for 30 min at 25 °C for\nc-Fos or at 30 °C for GST-NLS, BSA-NLS, and GST-M9. After washing,\ncells incubated with Cy3-Fos or FITC-BSA-NLS were fixed with 3.7%\nformaldehyde and analyzed directly by fluorescence microscopy. Cells\nincubated with GST-tagged import substrates were first subjected to\nindirect immunostaining using an anti-GST-antibody (Amersham Bio-\nsciences) and Alexa 594 donkey-anti-goat (Molecular Probes). Cells\nwere analyzed by fluorescence microscopy using a Zeiss Axioskop2\nmicroscope. Pictures were processed using Adobe Photoshop.\nSDS-PAGE and Western Blotting—Proteins were separated by SDS-\nPAGE and subjected to Western blotting using standard methods. Rab-\nbit anti-importin and anti-importin 7 antibodies were a kind gift from\nD. Go¨rlich (35, 52). The rabbit-anti-transportin antibody was a kind gift\nfrom L. Gerace. Rabbit anti-maltose-binding protein (MBP) was pur-\nchased from New England Biolabs. The mouse anti-His antibody was\nfrom Covance. HRP-coupled goat anti rabbit or goat anti mouse IgG\n(Dianova) was used as secondary antibody. The ECL system (Pierce) was\nused for visualization of proteins.\nRESULTS\nImportin -independent Nuclear Import of c-Fos in Vitro—The tran-\nscription factor c-Fos has been suggested to be imported into the\nnucleus by the transport receptor importin , independent of the\nadapter protein importin (13). Importin -dependent import of c-Fos\nin permeabilized cells, however, has not been demonstrated so far. To\nanalyze nuclear transport of c-Fos in detail, we established an import\nassay in digitonin-permeabilized cells (36). As shown in Fig. 1A, c-Fos\nwas efficiently imported into nuclei of permeabilized cells at 25 °C in the\npresence of cytosol and wild-type Ran. Transport was energy-depend-\nent, as it was blocked at 4 °C, and required the addition of soluble fac-\ntors, as no nuclear accumulation was observed in cells incubated with\nbuffer alone. To test whether nuclear import of c-Fos depends on trans-\nport receptors of the importin -superfamily, a reaction was performed\nin the presence of RanQ69L, a Ran-mutant that is predominantly in the\nGTP-bound form (37), preventing the interaction of import substrates\nwith import receptors (4). RanQ69L strongly inhibited nuclear accumu-\nlation of c-Fos, demonstrating the specificity of the nuclear transport\nreaction. We next used our in vitro transport assay to test if import of\nc-Fos depends on the activity of importin . Importin -dependent\ntransport can be inhibited with a monoclonal antibody against this\ntransport receptor (38). Preincubation of cytosol with this antibody\nresulted in strong inhibition of nuclear import of GST-NLS, a substrate\nthat uses the importin / dimer as a transport receptor (Fig. 1B). In\ncontrast, nuclear import of GST-M9, a substrate for the import receptor\ntransportin, was not affected by the anti-importin antibody. Likewise,\nnuclear import of c-Fos was not significantly reduced upon preincuba-\ntion of cytosol with the anti-importin antibody. These results strongly\nsuggest that cytosolic transport factors distinct from importin  can\nmediate nuclear import of c-Fos.\nc-Fos Interacts with Different Import Receptors—To identify trans-\nport receptors that specifically interact with c-Fos, a GST fusion protein\nof the transcription factor was immobilized on Sepharose beads and\nincubated with cytosol in the absence or presence of RanQ69L. This\nRan-mutant inhibited nuclear import of c-Fos (Fig. 1A), and we there-\nfore expected it to release transport-relevant interaction partners from\nthe immobilized bait. As shown in Fig. 2A, two proteins 100 and 120\nkDa specifically bound to GST-Fos in the absence but not in the pres-\nence of RanQ69L. To identify these potential import receptors for c-Fos,\nthe bands were cut from the gel and tryptic digests of proteins were\nanalyzed by mass spectrometry. In each band, two proteins could\nunequivocally be identified. The strong band at 100 kDa contained\nimportin and transportin (specifically, transportin 1 (14)), whereas the\nweaker band at 120 kDa contained importins 7 and 8. We next used\nspecific antibodies against three of these four import receptors to verify\ntheir specific interaction with GST-Fos. Importin , importin 7, as well\nas transportin specifically bound to GST-Fos in the absence but not in\nthe presence of RanQ69L (Fig. 2B). No binding to immobilized GST was\ndetected. Strikingly, a strong accumulation of transportin on the GST-\nFos beads was observed, whereas only a rather small proportion of\nimportin or importin 7 was bound to the beads (compare “input” and\n“GST-Fos”). c-Fos contains a leucine-zipper and forms stable dimers\nwith proteins like c-Jun. To exclude the possibility that the identified\nimport receptors bound indirectly to c-Fos via proteins that interact\nwith its leucine-zipper, we expressed an N-terminal fragment of c-Fos,\nlacking this protein interaction domain (GST-Fos 3–160; see also Fig.\n5A). Binding of importin 7, importin , and transportin to this fragment\nwas as strong as to the full-length protein (Fig. 2B), suggesting a direct\ninteraction. To demonstrate such a direct interaction of import recep-\ntors with c-Fos, importin 7, importin , and transportin, as well as\nimportin 5 as a negative control, were expressed in bacteria and incu-\nbated with immobilized GST-Fos. Importin 5 did not interact with\nGST-Fos under these conditions (Fig. 2C). Importin and transportin\nbound efficiently to GST-Fos, whereas importin 7 bound rather weakly\nTransportin-mediated Nuclear Import of c-Fos\n(compare “GST-Fos” to “input”). Binding of importin and transportin\nwas abolished when RanQ69L was included in the reaction, demon-\nstrating the specificity of the interaction. Binding of importin 7 to GST-\nFos was less sensitive to RanQ69L, probably reflecting the lower affinity\nof this transport receptor for RanGTP, compared with importin (35).\nImportin 7 has been shown to form a heterodimer with importin (35).\nThis dimer serves as an import receptor for histone H1 (39). We there-\nfore combined importin  and importin 7 in the binding reaction and\nnow observed increased binding of importin 7 (together with importin\n) to GST-Fos (Fig. 2C). This result suggests that at least part of the\ncytosolic importin 7 that interacted with GST-Fos (Fig. 2A) was indi-\nrectly bound via importin . Taken together, different transport recep-\ntors of the importin  superfamily specifically interact with c-Fos and\nmay therefore function as physiological import receptors.\nTo test which of the identified import receptors promote efficient\nnuclear transport of c-Fos, we performed import reactions in digitonin-\npermeabilized cells, using recombinant proteins. Cells were incubated\neither at 4 °C or 25 °C with equimolar concentrations of the various\ntransport receptors. At 25 °C, transportin promoted the strongest accu-\nmulation of c-Fos in the nucleus (Fig. 3A). Importin  also mediated\nnuclear import of c-Fos, yet to a somewhat lower extent. Importins 5\nand 7 did not promote nuclear import of c-Fos, although they were\ntransport-competent for their established import cargo, the ribosomal\nprotein L23 (data not shown (16)). Combining importin with its bind-\ning partner importin 7 did not further promote nuclear import, com-\npared with a reaction with importin  alone. Thus, transportin and\nimportin  appear to be the major transport receptors for nuclear\nimport of c-Fos. Transport was clearly specific, because it was temper-\nature-dependent (Fig. 3A) and could be inhibited by RanQ69L and by\nWGA, a lectin that binds to O-linked nucleoporins (40) and blocks\nvarious nucleocytoplasmic transport pathways (41, 42) (Fig. 3B).\nCharacterization of the Interaction between c-Fos and Transportin or\nImportin —Having established that at least two importins, transportin\nand importin , may serve as import receptors for c-Fos, we next ana-\nlyzed the interaction of the transcription factor with these receptors in\ndetail. We first expressed N- and C-terminal fragments of both importin\nand transportin and tested them for interaction with either an estab-\nlished binding partner or with c-Fos. A GST fusion of the importin \nbinding (IBB-) domain of importin  was immobilized on beads and\nincubated with full-length importin , or with its N- or C-terminal\nTransportin-mediated Nuclear Import of c-Fos\nfragment. As expected (43), the IBB-domain preferentially bound to the\nC-terminal fragment of importin (Fig. 4A). In contrast, when GST-Fos\nwas immobilized, the N-terminal, but not the C-terminal fragment of\nimportin  was retained on the beads. No binding was observed when\nGST alone was used as a bait.\nIn a similar set of experiments, we immobilized GST, GST-M9, or\nGST-Fos on beads and incubated them with full-length transportin or\nits N- or C-terminal fragments (Fig. 4B). In these experiments, trans-\nportin fragments fused to the maltose binding protein (MBP) were used,\nbecause His-tagged fragments were insoluble. Binding of the MBP-\ntransportin fragments to GST beads was not observed. GST-M9 inter-\nacted with full-length transportin as well as with the C-terminal frag-\nment, but not with the N-terminal fragment. GST-Fos, in contrast,\ninteracted with all three proteins. Binding of free MBP to the beads was\nnot detected. Interaction of full-length transportin with GST-Fos was\nspecific, as it could be blocked by RanQ69L (see Fig. 2C), suggesting that\nthe interaction of c-Fos with the two individual fragments of transportin\nis specific, too. We could directly test this assumption for the N-termi-\nnal fragment, which contains the Ran-binding site of transportin. As\nshown in Fig. 4C, the interaction of GST-Fos with this fragment was\nlargely inhibited when RanQ69L was included in the reaction. These\nresults show that c-Fos interacts with different regions of both importin\n and transportin compared with their classic binding partners, the\nIBB-domain of importin and the M9 sequence, respectively.\nIn light of this differential binding, we next analyzed whether these\nclassic binding partners and c-Fos could interact simultaneously with\neither importin  or transportin. Immobilized GST-IBB or GST-Fos\nwas incubated with importin , in the absence or presence of an excess\nof the IBB-domain, coupled to the maltose binding protein (MBP-IBB).\nAs expected, no binding of importin to GST-IBB was observed when\nMBP-IBB was present (Fig. 4D, top panel). MBP-IBB also abolished the\ninteraction of importin with c-Fos, suggesting that binding of IBB and\nc-Fos to importin  are mutually exclusive. Similarly, an M9 sequence\ncoupled to MBP prevented the interaction of transportin with both\nGST-M9 and GST-Fos (Fig. 4D, bottom panel). These results show that\nthe two cargo binding sites on importin  and transportin cannot be\noccupied simultaneously by the classic cargoes of the import receptors\nand by c-Fos. This may result from conformational changes in the struc-\nture of the receptors upon binding one type of ligand, preventing simul-\ntaneous binding of a second ligand to a different binding site.\nTo characterize the regions in c-Fos required for interaction with\nimportin and transportin, we generated a number of fragments of the\ntranscription factor as well as two point mutations as GST fusion pro-\nteins (Fig. 5A). These proteins were immobilized on beads and incu-\nbated with either full-length importin  or full-length transportin. A\nfragment, where the C-terminal part of c-Fos, including the leucine-\nzipper was deleted, interacted with importin (Fig. 5B) and transportin\n(Fig. 5C), similar to the full-length protein (compare lanes 1 and 2).\nThus, the C-terminal part of c-Fos beyond the basic DNA-binding\ndomain (“ ” in Fig. 5A) is not required for interaction with the\ntransport receptors. The DNA-binding domain has been reported to\nfunction as an NLS (23). Indeed, when one, two or three of these basic\nregions were deleted, the corresponding c-Fos fragment failed to inter-\nact with importin (Fig. 5B, lanes 3–5). Similarly, when two arginines in\neither the second or the third basic region were mutated to alanines,\nfull-length c-Fos did not bind to importin  (Fig. 5B, lanes 9 and 10).\nStrikingly, however, all these fragments/mutants still interacted with\ntransportin (Fig. 5C, lanes 3–5, 9, and 10), demonstrating that different\nregions in c-Fos must be responsible for binding to the different trans-\nport receptors. Accordingly, a short fragment comprised of the second\nand third basic region of c-Fos bound to importin (Fig. 5B, lane 7) but\nhardly to transportin (Fig. 5C, lane 7). In an attempt to further narrow\ndown the binding site on c-Fos for transportin, we tested other frag-\nments of the transcription factor. Fragments comprising amino acids\n3–124 or 81–160 efficiently bound to transportin (Fig. 5, lanes 5 and 8),\nwhereas a fragment comprising amino acids 3–110 did not (Fig. 5, lane\n6). These results suggested that amino acids 81–124 (overlap between\nfragments 5 and 8) or 111–124 (deleted residues in fragment 6 com-\npared with fragment 5) are required for binding. A fragment comprising\namino acids 81–124, however, did not bind to transportin (data not\nshown), arguing for a more complex mechanism of interaction.\nAs importin and transportin bind to different regions of c-Fos, we\nnext asked whether the transcription factor could interact simulta-\nTransportin-mediated Nuclear Import of c-Fos\nneously with the two transport receptors. Importin was immobilized\non beads and incubated with GST-Fos in the absence or presence of\ntransportin. As shown in Fig. 5D, an equimolar concentration of trans-\nportin was sufficient to abolish binding of GST-Fos to importin , sug-\ngesting that the proteins cannot form a trimeric complex.\nTransportin Is the Major Import Receptor for c-Fos—Our results have\nshown that c-Fos can be imported into the nucleus by two transport\nreceptors, importin , and transportin. Is there any preference for one of\nthese factors? To answer this question, we first compared relative affin-\nities of importin and transportin for c-Fos. GST-Fos was immobilized\non beads and incubated either with transportin or importin alone, or\nwith transportin in combination with increasing concentrations of\nimportin . The amounts of GST-Fos and import receptors were cho-\nsen such that GST-Fos would be limiting for the binding reactions.\nTransportin and importin  efficiently interacted with GST-Fos when\nthey were added separately to the reaction (Fig. 6A). When the two\ntransport receptors were present at equimolar concentrations, trans-\nportin still bound efficiently to GST-Fos, whereas importin  was\nalmost undetectable, in agreement with the result presented in Fig. 5D.\nEven at a 20-fold molar excess of importin over transportin, there was\nstill substantial binding of transportin to GST-Fos, suggesting that\ntransportin has a much higher affinity for the transcription factor com-\npared with importin . We next tested whether fragments or mutants of\nc-Fos that interact with transportin but not with importin  could be\nimported efficiently into nuclei of digitonin-permeabilized cells in the\npresence of cytosol. As shown in Fig. 6 (B and C), the fragment 3–154 as\nwell as the mutant R157A/R158A (compare Fig. 5A) were imported at\n25 °C but not at 4 °C. Import was specific, as it could be blocked by\nRanQ69L or by WGA. Under conditions where importin -dependent\ntransport was inhibited, import of c-Fos still occurred (Fig. 1B). We now\nfinally tested, if inhibition of the transportin-dependent nuclear import\npathway would abolish transport of c-Fos into the nucleus. Permeabi-\nlized cells were incubated with the transport substrates GST-M9, BSA-\nNLS, or GST-Fos with cytosol as the source of transport factors. All\nsubstrates were imported efficiently when no inhibitor was present (Fig.\n6D). When we included MBP-M9 in the reaction, a substrate that com-\npetes with transportin-dependent cargoes for efficient import, we\nobserved strong inhibition of transport of GST-M9, as well as of c-Fos.\nThe importin /-dependent substrate BSA-NLS was not affected by\nMBP-M9, demonstrating the specificity of inhibition. Thus, under con-\nTransportin-mediated Nuclear Import of c-Fos\nditions where importin -dependent transport still occurs, inhibition of\ntransportin abolishes the nuclear import of c-Fos, suggesting that trans-\nportin is the major cytosolic transport receptor for the transcription\nfactor under our experimental conditions.\nDISCUSSION\nTransportin Is the Major Import Receptor for c-Fos—Different regions\nin c-Fos required for its efficient import into the nucleus have been\nidentified (23, 24). Transport factors that promote nuclear import of\nc-Fos in the classic in vitro system, however, have not been described. In\nthe course of this study, we analyzed nuclear import of c-Fos in digito-\nnin-permeabilized cells. In this system, the import receptors transportin\nand importin  are able to promote specific import of c-Fos into the\nnucleus. Several lines of evidence, however, point to a dominant role of\ntransportin for nuclear import of c-Fos. (i) Transportin was highly\nenriched on GST-Fos beads incubated with a cytosolic extract. In con-\ntrast, only a small fraction of other import receptors like importin and\nimportin 7 bound to immobilized GST-Fos. (ii) Recombinant transpor-\ntin promoted nuclear import of GST-Fos more efficiently than importin\n. Importin 7 did not promote nuclear import of c-Fos. Like importin 8,\nimportin 7 interacts with importin (35). Thus, these two proteins may\nhave interacted with GST-Fos via importin (Fig. 2A). (iii) Competition\nexperiments revealed that GST-Fos interacts more efficiently with\ntransportin than importin , suggesting a much higher affinity. The\npublished dissociation constant of the c-Fos-importin  complex is\nrather high (65 nM (13)) compared with the dissociation constant of the\nPTHrP-importin  complex (3.5 nM (9)) or to the binding affinity of\nimportin to importin (5–18 nM (44)). (iv) Fragments or mutants of\nc-Fos that do not interact with importin (but with transportin) were\nefficiently imported into the nucleus in the presence of cytosol. (v) Most\nimportantly, under conditions where transportin-dependent import\nwas inhibited by a substrate-competitor, c-Fos did not enter the\nnucleus, whereas an importin /substrate was efficiently transported.\nIn contrast, inhibition of the importin -dependent pathway with an\nantibody against the receptor did abolish import of GST-NLS but not\nthat of c-Fos. Taken together, we believe that transportin is the major\nimport receptor for c-Fos. Under certain conditions of activation or in\ncertain cell types, however, importin  may serve as an importin for\nc-Fos as well. This interpretation is in agreement with earlier results that\nshowed that a 22-amino acid basic region (i.e. the importin -interac-\ntion domain, see below) can function as an NLS when fused to pyruvate\nkinase as a reporter protein (23). Deletion of this basic region, however,\ndid not affect the nuclear localization of c-Fos, suggesting alternative\nimport pathways.\nIn living cells, nuclear import of c-Fos does not occur by default,\nbecause the newly synthesized protein is retained in the cytoplasm in\nserum-starved cells (21). One example for a stimulus-dependent trans-\nport is the UV-induced phosphorylation of c-Fos by p38 kinases and the\nsubsequent import of the transcription factor into the nucleus (45). Our\nbacterially produced c-Fos (i.e. unphosphorylated protein) readily binds\nto transportin and importin , suggesting that phosphorylation of c-Fos\nmay rather regulate its cytoplasmic retention than its interaction with\ntransport receptors. Nevertheless, different signaling pathways could\nregulate nuclear import of c-Fos by different importins. Furthermore,\nnuclear import pathways of c-Fos could differ from cell type to cell type,\ndepending on relative expression levels of potential import receptors.\nInteraction of c-Fos with Importin  and Transportin—The basic\nregions in the DNA-binding domain of c-Fos have been implicated in\nnuclear import of the transcription factor (23). Indeed, deletions of one\nor more of these regions, as well as point mutations exchanging argin-\nines for alanines abolished the interaction of c-Fos with importin .\nThus, c-Fos contains a basic, arginine-rich NLS that can be recognized\nby importin , similar to other proteins (5–7). 19 HEAT repeats in\nimportin generate an extensive interaction surface that allows binding\nof a large variety of molecules, including RanGTP, nucleoporins, and\ndifferent import cargoes. Most prominent for cargo binding is the\nimportin -interaction domain in the C-terminal part of importin \n(43), comprising HEAT repeats 7–19 (46). In contrast, PTHrP binds to\nthe concave surface of the N-terminal part of importin , contacting\nHEAT repeats 2–11 (47). A third interaction mode is used by SREBP-2,\nwhich binds as a dimer to importin  (48). SREBP-2 has no typical\nconsecutive basic residues and the binding to importin  is primarily\nmediated by hydrophobic interactions, with HEAT repeats 7 and 17\nbeing of particular importance (48). c-Fos binds to the N-terminal por-\ntion of importin , possibly in a similar way as PTHrP. Unlike c-Fos,\nhowever, PTHrP can form a trimeric complex with importin  and\nimportin , at least in vitro (47), suggesting a somewhat different bind-\ning mode. A common feature of all importin -cargo complexes, includ-\ning the importin -c-Fos complex, is their sensitivity to RanGTP, which\nbinds to HEAT repeats 1– 8 at the N terminus of the receptor (49). Thus,\nregardless of the nature of the receptor-cargo complex, common mech-\nanism are at play for the dissociation of the complex by RanGTP in the\nnucleus.\nThe regions in c-Fos that are required for binding to transportin are\nclearly distinct from the basic regions needed for binding to importin ,\nbecause their deletion does not affect transportin binding. So far we\nhave been unable to generate point mutations that do not bind to trans-\nportin. Amino acids 111–124 of the c-Fos protein appear to be impor-\ntant for binding, because the interaction was lost when these residues\nwere deleted. This region, which is much shorter than the classic M9\nsequence and does not contain consecutive basic residues, is not suffi-\ncient for binding, as even longer fragments (e.g. Fos 81–124) did not\ninteract with transportin. Of course we cannot exclude problems result-\ning from misfolded proteins. Together, these results argue for a more\ncomplex mode of interaction, possibly involving residues at the N ter-\nminus of the transcription factor. By several criteria, however, the inter-\naction of transportin with c-Fos is distinct from its binding to its estab-\nlished import cargoes. First, the N terminus of transportin clearly\ninteracts specifically with c-Fos, in contrast to the classic import cargo,\nthe M9 sequence. We also detected binding of c-Fos to the C-terminal\npart of transportin, which also interacts with the M9 sequence (50).\nHence, there is probably more than one binding site on transportin for\nc-Fos, explaining the strong interaction of the importin with the tran-\nscription factor. Second, the transportin-binding fragment c-Fos 3–124\nhas no characteristic basic regions, unlike other import cargoes of trans-\nportin that do not contain an M9-like NLS (16). Therefore, the interac-\ntion of transportin with c-Fos is the first example of a third binding\nmode of the import receptor to a cargo molecule. The observation that\nbinding of the M9 sequence and of c-Fos to transportin are mutually\nexclusive argues for extensive conformational changes in the receptor\nmolecule upon cargo interaction, similar to what has been described for\nimportin (48, 51). Alternatively, the first cargo could mask the binding\nsite for the second cargo.\nIn summary, transportin may be as versatile as importin  when it\ncomes to the accommodation of a variety of import cargoes. A detailed\ncrystal structure analysis will be required to characterize the different\ntransportin-cargo complexes at the molecular level.",
         "",
         "Transportin Is a Major Nuclear Import Receptor for c-Fos",
         "None"
        ],
        [
         "37",
         "001cf91b762ae802a1fc6fe610759b83b300d1d1",
         "None",
         "Yun-Hsin Wang,Yau-Hung Chen,Tai-Na Wu,Yu-Ju Lin,H. Tsai",
         "None",
         "http://ntur.lib.ntu.edu.tw/bitstream/246246/162249/1/35.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://ntur.lib.ntu.edu.tw/bitstream/246246/162249/1/35.pdf",
         "None",
         "Spatiotemporal expression of zebrafish keratin 18 during early embryogenesis and the establishment of a keratin 18:RFP transgenic line.;Arsenic exposure, urinary arsenic speciation, and peripheral vascular disease in blackfoot disease-hyperendemic villages in Taiwan.;Arsenic contamination in drinking water and skin manifestations in lowland Nepal: the first community-based survey.;Developmental consequences of in utero sodium arsenate exposure in mice with folate transport deficiencies.;Animal models for arsenic carcinogenesis: inorganic arsenic is a transplacental carcinogen in mice.;Mechanism of arsenic carcinogenesis: an integrated approach.;Induction of tumors of the liver, lung, ovary and adrenal in adult mice after brief maternal gestational exposure to inorganic arsenic: promotional effects of postnatal phorbol ester exposure on hepatic and pulmonary, but not dermal cancers.;Acute and chronic arsenic toxicity;A review of animal models for the study of arsenic carcinogenesis.;Arsenite is a cocarcinogen with solar ultraviolet radiation for mouse skin: an animal model for arsenic carcinogenesis.;Genetic toxicology of a paradoxical human carcinogen, arsenic: a review.;K6/ODC transgenic mice as a sensitive model for carcinogen identification.;Risk of internal cancers from arsenic in drinking water.;Arsenic enhancement of skin neoplasia by chronic stimulation of growth factors.;Stages of embryonic development of the zebrafish;Study on the carcinogenicity of lead arsenate and sodium arsenate and on the possible synergistic effect of diethylnitrosamine.;Life term studies on the effect of trace elements on spontaneous tumors in mice and rats.;cDNA sequences of the authentic keratins 8 and 18 in zebrafish.;Teneurin‐2 is expressed in tissues that regulate limb and somite pattern formation and is induced in vitro and in situ by FGF8;Carcinogenicity and co-carcinogenicity of sodium arsenite in P53+/− male mice;nitrosamine. Food Cosmer. Toxicol",
         "A keratin 18 transgenic zebrafish Tg(k18(2.9):RFP) treated with inorganic arsenite reveals visible overproliferation of epithelial cells.",
         "None"
        ],
        [
         "38",
         "001d0543c95a930756c172d9000b59ea2d82af78",
         "Immature oocytes benefit from nutrient modification of the follicular environment by the surrounding cumulus mass. However, the oxygen concentration that the oocyte may be exposed to could be lower than the antral follicular concentration due to the metabolism of surrounding cumulus cells. Using metabolic data previously determined, we have developed a mathematical model of O(2) diffusion across the bovine and murine cumulus-oocyte complex. From this we have determined that across a physiological range of external pO(2), less than 0.25% and 0.5% O(2) is removed by cumulus cells within the bovine and murine cumulus-oocyte complex respectively. Our model differs from others as it: incorporates a term that allows for nonlinear variation of the oxygen consumption rate with oxygen concentration; considers two regions (oocyte and cumulus) sharing a common boundary, both of which consume oxygen at different non linear rates. Cumulus cells therefore remove little O(2), thus sparing this essential gas for the oocyte, which is dependent on ATP generation via oxidative phosphorylation.",
         "Alys R. Clark,Yvonne M. Stokes,Michelle Lane,Jeremy G. Thompson",
         "Alys R. Clark,Yvonne M. Stokes,Michelle Lane,Jeremy G. Thompson",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/131/6/1310999.pdf",
         "Immature oocytes beneﬁt from nutrient modiﬁcation of the follicular environment by the surrounding cumulus mass. However,\nthe oxygen concentration that the oocyte may be exposed to could be lower than the antral follicular concentration due to the\nmetabolism of surrounding cumulus cells. Using metabolic data previously determined, we have developed a mathematical\nmodel of O2 diffusion across the bovine and murine cumulus–oocyte complex. From this we have determined that across a\nphysiological range of external pO2, less than 0.25% and 0.5% O2 is removed by cumulus cells within the bovine and murine\ncumulus–oocyte complex respectively. Our model differs from others as it: incorporates a term that allows for nonlinear variation\nof the oxygen consumption rate with oxygen concentration; considers two regions (oocyte and cumulus) sharing a common\nboundary, both of which consume oxygen at different non linear rates. Cumulus cells therefore remove little O2, thus sparing this\nessential gas for the oocyte, which is dependent on ATP generation via oxidative phosphorylation.\nReproduction (2006) 131 999–1006\nA quantitative understanding of nutrient supply to the\noocyte in the antral follicle is generally lacking. There is\ngood evidence that, initially, gap-junctional communi-\ncation between the oocyte and the surrounding cumulus\ncells is important in nutrient transfer (Herlands & Shultz\n1984), and the transfer of small molecular weight\nmolecules, such as cyclic AMP (Bornslaeger & Schultz\n1985), uridine (Herlands & Shultz 1984) and glucose\n(Saito et al. 1994) by this means has been described.\nOnce the gap-junction connection has been severed in\nresponse to the ovulatory stimulus (in vivo) or spon-\ntaneous meiotic maturation (in vitro),\nthe oocyte\npresumably relies on a histotrophic nutrient supply,\nalthough the close proximity to the expanding cumulus\nmass is likely to alter nutrient composition adjacent to\nthe oocyte relative to that in the follicular ﬂuid.",
         "",
         "Introduction\nOne component that appears to be important for the\nestablishment of developmental competence in the\noocyte within antral follicles is oxygen. In general, pO2\nlevels around 7–11% have been recorded in antral\nfollicles of humans and pigs (Knudsen et al. 1978,\nFischer et al. 1992). Several studies in humans (Van\nBlerkom et al. 1997, Van Blerkom 1998) indicate that\nperifollicular vascularity is associated with oocyte\ndevelopmental competence. This was subsequently\n(Van Blerkom\nrelated to the dissolved O2 content\n1998), although this association has been disputed\n(Huey et al. 1999). In contrast, there are many studies\nthat have examined the effect of O2 concentration during\nin vitro maturation of oocytes across several species.\nExamples of such studies include hamster (Gwatkin &\nHaidri 1974), squirrel monkey (Yeoman et al. 1999), cow\n(Hashimoto et al. 2000, Minami et al. 2000), pig (Park\net al. 2005) and mouse (Haidri et al. 1971, Eppig &\nWigglesworth 1995, Hu et al. 2001). Such studies have\nnot yielded any consensus over\nthe optimal O2\nconcentration during maturation, as there are conﬂicting\nresults, especially in regard to the beneﬁts of a low O2\natmosphere. Of central importance to these studies is the\nchoice of oxygen concentration used. Many have\ncompared atmosphere (approximately 20% O2) against\n5% O2 concentrations,\nthe latter being the widely\naccepted optimal value for the development of mouse\nembryos in vitro (Quinn & Harlow 1978).\nRecently we measured the uptake of key nutrients in\nbovine cumulus–oocyte complexes (COCs) to determine\nif oocyte-secreted factors inﬂuence bovine COC metab-\nolism (Sutton et al. 2003). In contrast to the mouse\n(Sugiura et al. 2005), oocyte-secreted factors appear\nto have little inﬂuence over the metabolism of\nthe\nsurrounding cumulus cells. However, as with the mouse\n(Sugiura et al. 2005), bovine COCs have signiﬁcant\nlevels of glycolysis, with lactic acid production account-\ning for much of the glucose taken up, especially prior\nto cumulus expansion (Sutton et al. 2003). These data\nsuggest that oxygen consumption by the bovine cumulus\ncell mass is minimal, thus sparing O2 for oxidative phos-\nphorylation by the oocyte. We have therefore developed\na mathematical model of the diffusion of oxygen through\nthe COC so as to assess how the O2 concentration\ndecreases from the follicular-ﬂuid cumulus oophorus\ninterface to the corona radiata oocyte interface. We have\napplied this model to data already described for both\nbovine and murine COCs.\nand does not consume oxygen, nor is it likely to impede\noxygen diffusion. Finally, we have assumed that the\nfollicular ﬂuid surrounding the COC is perfectly mixed,\nenabling us to take the concentration of oxygen in this\nﬂuid to be constant.\nMathematical model\nThe approximate spherical symmetry of the COC and our\ninterest in the very early stage of development justiﬁes our\nuse of a spherically symmetric steady-state model. The\nrate of oxygen consumption by both oocyte and cumulus\nis expected to vary with the concentration of oxygen. Thus\nwe solve the equation\nMaterials and Methods\nCumulus–oocyte complex parameters\n\nr 2 dC\ndr\nZ MðC Þ;\nData for COC diameter and O2 consumption was obtained\nfrom a variety of published sources, identiﬁed in Table 1.\nGeneral assumptions",
         "Prior to the luteinising hormone surge, cumulus cells are\ntightly packed around the oocyte and we have assumed\nthat, in the case of passive diffusion of oxygen, these\ncumulus cells are effectively a homogeneous mixture of\nintra- and extracellular matter (Jones 1986). The diffusion\ncoefﬁcient of oxygen in the cumulus mass can therefore\nbe taken to be constant. We have also assumed spherical\nsymmetry of the COC, and ignored that at least part of it\nis attached to the mural granulosa cell layer of the\nfollicle. The thickness of the zona pellucida (approxi-\nmately 10 mm) has been neglected, as this is acellular",
         "",
         "https://ntur.lib.ntu.edu.tw/bitstream/246246/162249/1/35.pdf",
         "Immature oocytes beneﬁt from nutrient modiﬁcation of the follicular environment by the surrounding cumulus mass. However,\nthe oxygen concentration that the oocyte may be exposed to could be lower than the antral follicular concentration due to the\nmetabolism of surrounding cumulus cells. Using metabolic data previously determined, we have developed a mathematical\nmodel of O2 diffusion across the bovine and murine cumulus–oocyte complex. From this we have determined that across a\nphysiological range of external pO2, less than 0.25% and 0.5% O2 is removed by cumulus cells within the bovine and murine\ncumulus–oocyte complex respectively. Our model differs from others as it: incorporates a term that allows for nonlinear variation\nof the oxygen consumption rate with oxygen concentration; considers two regions (oocyte and cumulus) sharing a common\nboundary, both of which consume oxygen at different non linear rates. Cumulus cells therefore remove little O2, thus sparing this\nessential gas for the oocyte, which is dependent on ATP generation via oxidative phosphorylation.\nReproduction (2006) 131 999–1006\nIntroduction\nA quantitative understanding of nutrient supply to the\noocyte in the antral follicle is generally lacking. There is\ngood evidence that, initially, gap-junctional communi-\ncation between the oocyte and the surrounding cumulus\ncells is important in nutrient transfer (Herlands & Shultz\n1984), and the transfer of small molecular weight\nmolecules, such as cyclic AMP (Bornslaeger & Schultz\n1985), uridine (Herlands & Shultz 1984) and glucose\n(Saito et al. 1994) by this means has been described.\nOnce the gap-junction connection has been severed in\nresponse to the ovulatory stimulus (in vivo) or spon-\ntaneous meiotic maturation (in vitro),\nthe oocyte\npresumably relies on a histotrophic nutrient supply,\nalthough the close proximity to the expanding cumulus\nmass is likely to alter nutrient composition adjacent to\nthe oocyte relative to that in the follicular ﬂuid.\nOne component that appears to be important for the\nestablishment of developmental competence in the\noocyte within antral follicles is oxygen. In general, pO2\nlevels around 7–11% have been recorded in antral\nfollicles of humans and pigs (Knudsen et al. 1978,\nFischer et al. 1992). Several studies in humans (Van\nBlerkom et al. 1997, Van Blerkom 1998) indicate that\nperifollicular vascularity is associated with oocyte\ndevelopmental competence. This was subsequently\n(Van Blerkom\nrelated to the dissolved O2 content\n1998), although this association has been disputed\n(Huey et al. 1999). In contrast, there are many studies\nthat have examined the effect of O2 concentration during\nin vitro maturation of oocytes across several species.\nExamples of such studies include hamster (Gwatkin &\nHaidri 1974), squirrel monkey (Yeoman et al. 1999), cow\n(Hashimoto et al. 2000, Minami et al. 2000), pig (Park\net al. 2005) and mouse (Haidri et al. 1971, Eppig &\nWigglesworth 1995, Hu et al. 2001). Such studies have\nnot yielded any consensus over\nthe optimal O2\nconcentration during maturation, as there are conﬂicting\nresults, especially in regard to the beneﬁts of a low O2\natmosphere. Of central importance to these studies is the\nchoice of oxygen concentration used. Many have\ncompared atmosphere (approximately 20% O2) against\n5% O2 concentrations,\nthe latter being the widely\naccepted optimal value for the development of mouse\nembryos in vitro (Quinn & Harlow 1978).\nRecently we measured the uptake of key nutrients in\nbovine cumulus–oocyte complexes (COCs) to determine\nif oocyte-secreted factors inﬂuence bovine COC metab-\nolism (Sutton et al. 2003). In contrast to the mouse\n(Sugiura et al. 2005), oocyte-secreted factors appear\nto have little inﬂuence over the metabolism of\nthe\nsurrounding cumulus cells. However, as with the mouse\n(Sugiura et al. 2005), bovine COCs have signiﬁcant\nlevels of glycolysis, with lactic acid production account-\ning for much of the glucose taken up, especially prior\nto cumulus expansion (Sutton et al. 2003). These data\nsuggest that oxygen consumption by the bovine cumulus\ncell mass is minimal, thus sparing O2 for oxidative phos-\nphorylation by the oocyte. We have therefore developed\na mathematical model of the diffusion of oxygen through\nthe COC so as to assess how the O2 concentration\ndecreases from the follicular-ﬂuid cumulus oophorus\ninterface to the corona radiata oocyte interface. We have\napplied this model to data already described for both\nbovine and murine COCs.\nand does not consume oxygen, nor is it likely to impede\noxygen diffusion. Finally, we have assumed that the\nfollicular ﬂuid surrounding the COC is perfectly mixed,\nenabling us to take the concentration of oxygen in this\nﬂuid to be constant.\nMathematical model\nThe approximate spherical symmetry of the COC and our\ninterest in the very early stage of development justiﬁes our\nuse of a spherically symmetric steady-state model. The\nrate of oxygen consumption by both oocyte and cumulus\nis expected to vary with the concentration of oxygen. Thus\nwe solve the equation\nMaterials and Methods\nCumulus–oocyte complex parameters\n\nr 2 dC\ndr\nZ MðC Þ;\nData for COC diameter and O2 consumption was obtained\nfrom a variety of published sources, identiﬁed in Table 1.\nGeneral assumptions\nPrior to the luteinising hormone surge, cumulus cells are\ntightly packed around the oocyte and we have assumed\nthat, in the case of passive diffusion of oxygen, these\ncumulus cells are effectively a homogeneous mixture of\nintra- and extracellular matter (Jones 1986). The diffusion\ncoefﬁcient of oxygen in the cumulus mass can therefore\nbe taken to be constant. We have also assumed spherical\nsymmetry of the COC, and ignored that at least part of it\nis attached to the mural granulosa cell layer of the\nfollicle. The thickness of the zona pellucida (approxi-\nmately 10 mm) has been neglected, as this is acellular",
         "High oxygen tension during in vitro oocyte maturation improves in vitro development of porcine oocytes after fertilization.;Effect of hexoses and gonadotrophin supplementation on bovine oocyte nuclear maturation during in vitro maturation in a synthetic follicle fluid medium.;Oocyte control of metabolic cooperativity between oocytes and companion granulosa cells: energy metabolism.;Mitochondria in human oogenesis and preimplantation embryogenesis: engines of metabolism, ionic regulation and developmental competence;Regulation of tissue oxygen levels in the mammalian lens;Cumulus expansion and glucose utilisation by bovine cumulus-oocyte complexes during in vitro maturation: the influence of glucosamine and follicle-stimulating hormone.;An Oxygen Transport Model for Human Bone Marrow Microcirculation;Influence of oocyte-secreted factors and culture duration on the metabolic activity of bovine cumulus cell complexes.;REDOX regulation of early embryo development.;Effects of low O2 and ageing on spindles and chromosomes in mouse oocytes from pre-antral follicle culture.;Low oxygen tension during in vitro maturation is beneficial for supporting the subsequent development of bovine cumulus–oocyte complexes;Perifollicular blood flow Doppler indices, but not follicular pO2, pCO2, or pH, predict oocyte developmental competence in in vitro fertilization.;Low Oxygen Inhibits but Complex High-Glucose Medium Facilitates In Vitro Maturation of Squirrel Monkey Oocyte–Granulosa Cell Complexes;Epigenetic Influences on Oocyte Developmental Competence: Perifollicular Vascularity and Intrafollicular Oxygen;The developmental potential of the human oocyte is related to the dissolved oxygen content of follicular fluid: association with vascular endothelial growth factor levels and perifollicular blood flow characteristics.;Substrate utilization and maturation of cumulus cell-enclosed mouse oocytes: evidence that pyruvate oxidation does not mediate meiotic induction.;Oxygen consumption and energy metabolism of the early mouse embryo;Oxygen uptake and carbohydrate metabolism by in vitro derived bovine embryos.;Factors affecting the developmental competence of mouse oocytes grown in vitro: Oxygen concentration;ATP content of human oocytes and developmental potential and outcome after in-vitro fertilization and embryo transfer.;Laboratory Production of Cattle Embryos;Development of glucose utilization studied in single oocytes and preimplantation embryos from mice.;Oxygen tension in follicular fluid falls with follicle maturation.;Effect of oxygen concentration on in-vitro development of preimplantation sheep and cattle embryos.;Intracellular diffusion gradients of O2 and ATP.;Oxygen concentration gradient across the ovarian follicular epithelium: model, predictions and implications.;Effect of mitochondrial clustering on O2 supply in hepatocytes.;Regulation of mouse oocyte growth: probable nutritional role for intercellular communication between follicle cells and oocytes in oocyte growth.;Intracellular oxygen supply during hypoxia.;Concentrations of hydrogen ions, oxygen, carbon dioxide and bicarbonate in porcine follicular fluid.;The effect of oxygen on the development of preimplantation mouse embryos in vitro.;Oxygen requirements for the maturation of hamster oocytes.;Oxygen transport in heterogeneous tissue.;Volume changes during the preimplantation stages of mouse egg development.;Culture of mouse oocytes in vitro, using a system without oil or protein.;Respiration of tissue as a medium of heterogeneous permeability.;The number and distribution of capillaries in muscles with calculations of the oxygen pressure head necessary for supplying the tissue;Mitochondria in human oogenesis and preimplantation embryogenesis: engines of metabolism, ionic regulation and developmental competence.;Oxygen balance for small organisms: An analytical model;The diffusive transport of gibberellins and abscisic acid through the aleurone layer of germinating barley grain: a mathematical model;Epigenetic Influences on Oocyte Developmental Competence: Perifollicular Vascularity and Intrafollicular Oxygen;An investigation by mathematical modelling of whether mouse and human preimplantation embryos in static culture can satisfy their demands for oxygen by diffusion.;Concentric oxygen diffusion in tissue with heterogeneous permeability and consumption.;Adenylate cyclase activity in zona-free mouse oocytes.;Versuche an uberlebendum carcinomgewebe;Molecular Reproduction and Development",
         "Mathematical modelling of oxygen concentration in bovine and murine cumulus-oocyte complexes.",
         "None"
        ],
        [
         "39",
         "001d60f67084573ca4336fc4afdf0b7184aa47e8",
         "The expression of systemic acquired resistance (SAR) in plants involves the upregulation of many Pathogenesis-Related (PR) genes, which work in concert to confer resistance to a broad spectrum of pathogens. Because SAR is a costly process, SAR-associated transcription must be tightly regulated. Arabidopsis thaliana SNI1 (for Suppressor of NPR1, Inducible) is a negative regulator of SAR required to dampen the basal expression of PR genes. Whole genome transcriptional profiling showed that in the sni1 mutant, Nonexpresser of PR genes (NPR1)–dependent benzothiadiazole S-methylester–responsive genes were specifically derepressed. Interestingly, SNI1 also repressed transcription when expressed in yeast, suggesting that it functions as an active transcriptional repressor through a highly conserved mechanism. Chromatin immunoprecipitation indicated that histone modification may be involved in SNI1-mediated repression. Sequence comparison with orthologs in other plant species and a saturating NAAIRS-scanning mutagenesis of SNI1 identified regions in SNI1 that are required for its activity. The structural similarity of SNI1 to Armadillo repeat proteins implies that SNI1 may form a scaffold for interaction with proteins that modulate transcription.",
         "R. Mosher,Wendy E. Durrant,Dong Wang,Junqi Song,Xinnian Dong",
         "None",
         "https://academic.oup.com/plcell/article-pdf/18/7/1750/36903817/plcell_v18_7_1750.pdf",
         "None",
         "None",
         "None",
         "None",
         "None",
         "https://academic.oup.com/plcell/article-pdf/18/7/1750/36903817/plcell_v18_7_1750.pdf",
         "None",
         "Systemic Acquired Resistance;Whole-Genome Analysis of the SHORT-ROOT Developmental Pathway in Arabidopsis;Faculty Opinions recommendation of A beta-catenin identified by functional rather than sequence criteria and its role in Wnt/MAPK signaling.;A β-Catenin Identified by Functional Rather Than Sequence Criteria and Its Role in Wnt/MAPK Signaling;Fitness Costs of Mutations Affecting the Systemic Acquired Resistance Pathway in Arabidopsis thaliana;Phylogenetic profiling of the Arabidopsis thaliana proteome: what proteins distinguish plants from other organisms?;Development and evaluation of an Arabidopsis whole genome Affymetrix probe array.;Comparison of Li-Wong and loglinear mixed models for the statistical analysis of oligonucleotide arrays;Knockout Analysis of Arabidopsis Transcription Factors TGA2, TGA5, and TGA6 Reveals Their Redundant and Essential Roles in Systemic Acquired Resistance Article, publication date, and citation information can be found at www.plantcell.org/cgi/doi/10.1105/tpc.014894.;Armadillo repeat proteins: beyond the animal kingdom.;Genetic and molecular evidence that the Pseudomonas syringae type III effector protein AvrRpt2 is a cysteine protease;DAVID: Database for Annotation, Visualization, and Integrated Discovery;Statistical significance for genomewide studies;Inducers of Plant Systemic Acquired Resistance Regulate NPR1 Function through Redox Changes;Transcriptional repression in eukaryotes: repressors and repression mechanisms;Arabidopsis : a laboratory manual;C-Terminal Regions of the Human Telomerase Catalytic Subunit Essential for In Vivo Enzyme Activity;Interplay between Two Epigenetic Marks DNA Methylation and Histone H3 Lysine 9 Methylation;Isolation and characterization of broad-spectrum disease-resistant Arabidopsis mutants.;N-Terminal Domains of the Human Telomerase Catalytic Subunit Required for Enzyme Activity in Vivo;Transcription regulation by histone methylation: interplay between different covalent modifications of the core histone tails.;Trans-dominant suppression of plant TGA factors reveals their negative and positive roles in plant defense responses.;The Structure of the β-Catenin/E-Cadherin Complex and the Molecular Basis of Diverse Ligand Recognition by β-Catenin;Constitutive disease resistance requires EDS1 in the Arabidopsis mutants cpr1 and cpr6 and is partially EDS1-dependent in cpr5.;Arabidopsis in planta transformation. Uses, mechanisms, and prospects for transformation of other species.;The transcriptome of Arabidopsis thaliana during systemic acquired resistance;Roles of Salicylic Acid, Jasmonic Acid, and Ethylene in cpr-Induced Resistance in Arabidopsis;Technical Focus:a guide to Agrobacterium binary Ti vectors.;Reduced growth and seed set following chemical induction of pathogen defence: does systemic acquired resistance (SAR) incur allocation costs?;Enhanced genome annotation using structural profiles in the program 3D-PSSM.;The Arabidopsis NPR1/NIM1 Protein Enhances the DNA Binding Activity of a Subgroup of the TGA Family of bZIP Transcription Factors;NPR1 differentially interacts with members of the TGA/OBF family of transcription factors that bind an element of the PR-1 gene required for induction by salicylic acid.;The language of covalent histone modifications;Identification and Cloning of a Negative Regulator of Systemic Acquired Resistance, SNI1, through a Screen for Suppressors of npr1-1;The families of pathogenesis-related proteins, their activities, and comparative analysis of PR-1 type proteins;A mini binary vector series for plant transformation;Topological characteristics of helical repeat proteins.;Interaction of NPR1 with basic leucine zipper protein transcription factors that bind sequences required for salicylic acid induction of the PR-1 gene.;Functional analysis of regulatory sequences controlling PR-1 gene expression in Arabidopsis.;Crystallographic Analysis of the Recognition of a Nuclear Localization Signal by the Nuclear Import Factor Karyopherin α;Uncoupling PR Gene Expression from NPR1 and Bacterial Resistance: Characterization of the Dominant Arabidopsis cpr6-1 Mutant;Regulation of Hypoxia-Inducible mRNAs by the von Hippel-Lindau Tumor Suppressor Protein Requires Binding to Complexes Containing Elongins B/C and Cul2;β-catenin: a key mediator of Wnt signaling;Three-Dimensional Structure of the Armadillo Repeat Region of β-Catenin;The cpr5 mutant of Arabidopsis expresses both NPR1-dependent and NPR1-independent resistance.;Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;Salicylate-independent lesion formation in Arabidopsis lsd mutants.;Arabidopsis signal transduction mutant defective in chemically and biologically induced disease resistance.;A mutation in Arabidopsis that leads to constitutive expression of systemic acquired resistance.;CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;Characterization of an Arabidopsis Mutant That Is Nonresponsive to Inducers of Systemic Acquired Resistance.;Enhanced Protection Against Fungal Attack by Constitutive Co–expression of Chitinase and Glucanase Genes in Transgenic Tobacco;Osmotin overexpression in potato delays development of disease symptoms.;New eukaryotic transcriptional repressers;Transgenic Plants with Enhanced Resistance to the Fungal Pathogen Rhizoctonia solani;The T/t common region of simian virus 40 large T antigen contains a distinct transformation-governing sequence;Basic local alignment search tool.;Basic local alignment search tool. Journal of Molecular Biology;Deletion analysis of GAL4 defines two transcriptional activating segments;Identical short peptide sequences in unrelated proteins can have different conformations: a testing ground for theories of immune recognition.;Structure of thermolysin refined at 1.6 A resolution.;Structure and control of phosphofructokinase from Bacillus stearothermophilus;A structure for thermolysin.;Prediction of Protein Structure;The cpr 5 Mutant of Arabidopsis Expresses Both NPRl-Dependent and NPRl-1 ndependent Resistance;The structure of the beta-catenin/E-cadherin complex and the molecular basis of diverse ligand recognition by beta-catenin.;BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT;Beta-catenin: a key mediator of Wnt signaling.;Stable binding to E2F is not required for the retinoblastoma protein to activate transcription, promote differentiation, and suppress tumor cell growth.;Three-dimensional structure of the armadillo repeat region of beta-catenin.;SWISS‐MODEL and the Swiss‐Pdb Viewer: An environment for comparative protein modeling;Repression versus activation in the control of gene transcription.",
         "A Comprehensive Structure–Function Analysis of Arabidopsis SNI1 Defines Essential Regions and Transcriptional Repressor Activity[W][OA]",
         "None"
        ],
        [
         "40",
         "001e6a0f509456cef8c74afa35cfdc1e9a07daf6",
         "Stuttering is a speech disorder long recognized to have a genetic component. Recent linkage studies mapped a susceptibility locus for stuttering to chromosome 12 in 46 highly inbred families ascertained in Pakistan. We report here on linkage studies in 100 families of European descent ascertained in the United States, Sweden, and Israel. These families included 252 individuals exhibiting persistent stuttering, 45 individuals classified as recovered from stuttering, and 19 individuals too young to classify. Primary analyses identified moderate evidence for linkage of the broader diagnosis of \"ever stuttered\" (including both persistent and recovered stuttering) on chromosome 9 (LOD = 2.3 at 60 cM) and of the narrower diagnosis of persistent stuttering on chromosome 15 (LOD = 1.95 at 23 cM). In contrast, sex-specific evidence for linkage on chromosome 7 at 153 cM in the male-only data subset (LOD = 2.99) and on chromosome 21 at 34 cM in the female-only data subset (LOD = 4.5) met genomewide criteria for significance. Secondary analyses revealed a significant increase in the evidence for linkage on chromosome 12, conditional on the evidence for linkage at chromosome 7, with the location of the increased signal congruent with the previously reported signal in families ascertained in Pakistan. In addition, a region on chromosome 2 (193 cM) showed a significant increase in the evidence for linkage conditional on either chromosome 9 (positive) or chromosome 7 (negative); this chromosome 2 region has been implicated elsewhere in studies on autism, with increased evidence for linkage observed when the sample is restricted to those with delayed onset of phrase speech. Our results support the hypothesis that the genetic component to stuttering has significant sex effects.",
         "R. Suresh,N. Ambrose,Cheryl A. Roe,A. Pluzhnikov,Jacqueline K. Wittke-Thompson,M. Ng,Xiaolin Wu,E. Cook,C. Lundstrom,M. Garsten,Ruth Ezrati,E. Yairi,N. Cox",
         "R. Suresh,N. Ambrose,Cheryl A. Roe,A. Pluzhnikov,Jacqueline K. Wittke-Thompson,M. Ng,Xiaolin Wu,E. Cook,C. Lundstrom,M. Garsten,Ruth Ezrati,E. Yairi,N. Cox",
         "http://www.cell.com/article/S0002929707636957/pdf",
         "1Department of Genetics and Development, Columbia University, New York; 2Department of Speech and Hearing Science, University of\nIllinois, Urbana-Champaign; Departments of 3Medicine and 4Human Genetics, The University of Chicago, and 5Institute for Juvenile\nResearch, Department of Psychiatry, University of Illinois at Chicago, Chicago; 6Helsingborg Hospital, Helsingborg, Sweden; and 7Tel Aviv\nUniversity School of Medicine, Tel Aviv\nStuttering is a speech disorder long recognized to have a genetic component. Recent linkage studies mapped a\nsusceptibility locus for stuttering to chromosome 12 in 46 highly inbred families ascertained in Pakistan. We report\nhere on linkage studies in 100 families of European descent ascertained in the United States, Sweden, and Israel.\nThese families included 252 individuals exhibiting persistent stuttering, 45 individuals classiﬁed as recovered from\nstuttering, and 19 individuals too young to classify. Primary analyses identiﬁed moderate evidence for linkage of\nthe broader diagnosis of “ever stuttered” (including both persistent and recovered stuttering) on chromosome 9\n(LOD p 2.3 at 60 cM) and of the narrower diagnosis of persistent stuttering on chromosome 15 (LOD p 1.95\nat 23 cM). In contrast, sex-speciﬁc evidence for linkage on chromosome 7 at 153 cM in the male-only data subset\n(LOD p 2.99) and on chromosome 21 at 34 cM in the female-only data subset (LOD p 4.5) met genomewide\ncriteria for signiﬁcance. Secondary analyses revealed a signiﬁcant increase in the evidence for linkage on chromosome\n12, conditional on the evidence for linkage at chromosome 7, with the location of the increased signal congruent\nwith the previously reported signal in families ascertained in Pakistan. In addition, a region on chromosome 2 (193\ncM) showed a signiﬁcant increase in the evidence for linkage conditional on either chromosome 9 (positive) or\nchromosome 7 (negative); this chromosome 2 region has been implicated elsewhere in studies on autism, with\nincreased evidence for linkage observed when the sample is restricted to those with delayed onset of phrase speech.\nOur results support the hypothesis that the genetic component to stuttering has signiﬁcant sex effects.",
         "Discussion\nGenomewide linkage analysis of 100 families segregating\nfor stuttering resulted in suggestive evidence for linkage\non chromosome 9, by use of the broad phenotype def-\ninition, with a LOD score of 2.3 at 60 cM. Modest\nevidence for linkage with the persistent-stuttering di-\nagnosis was detected on chromosomes 15 and 13, with\nLOD scores of 1.95 (23 cM) and 1.72 (14 cM), respec-\ntively. However, primary linkage mapping of the overall\ndata did not yield any LOD scores in the range of criteria\nestablished for genomewide signiﬁcance (Lander and\nKruglyak 1995). These results are not surprising, since\na large proportion of linkage studies designed to identify\nfor potential sex-related loci in our stuttering cohort was\nquite successful, leading to the identiﬁcation of several\nregions that merit further investigation. We observed ge-\nnomewide-signiﬁcant evidence for linkage (genomewide\nP ! .04\n) of stuttering in the male-only data set on chro-\nmosome 7 (153 cM). Evidence for linkage to this region\nis supported by replication of the ﬁnding (including the\nsex-speciﬁc effect) in the Hutterite stuttering cohort (J.\nK. Wittke-Thompson, unpublished data). Modest evi-\ndence for linkage to chromosome 7 (160 cM) was also\nobserved in families ascertained for speciﬁc language im-\npairment (SLI), a disorder characterized by impairments\nrestricted to the domain of language skills and in the\nabsence of other factors such as mental retardation and\nspeech-motor or sensory deﬁcits (Bartlett et al. 2004).\nThis region was also identiﬁed as harboring a language-\nrelated QTL for autism (Alarcon et al. 2005), a disorder\ncharacterized by disturbances in social, communicative,\nand behavioral functioning and that also shows a strik-\ning male bias in prevalence, with consistent estimates of\n4:1 affected males versus females (Veenstra-Vander-\nWeele and Cook 2004). A greater-than-expected prev-\nalence of SLI in families with autistic disorder and vice\nversa has been observed, as has the existence of over-\nlapping phenotypes for the two disorders (Folstein and\nMankoski 2000). It is, therefore, tempting to speculate\nthat there are one or more genes in this region of chro-\nmosome 7 that help to shape dimensions of speech and\nlanguage development and that may differ (e.g., in tim-\ning) between males and females, with genetic variation\nat the gene(s) affecting susceptibility to SLI, autism, and\nstuttering. We note, however, that this region does not\nencompass the FOXP2 gene identiﬁed elsewhere as the\ncausal gene for a rare speech and language disorder seg-\nregating in a large pedigree (Lai et al. 2001).\nWe also obtained genomewide-signiﬁcant evidence for\nlinkage of stuttering in the female-only subgroup on\nchromosome 21 (33.5 cM), with a LOD score of 4.5.\nAlthough the LOD score is likely to be poorly estimated\nin a sample this small, simulation studies conﬁrm that\na signal of this magnitude is very unlikely by chance\n(genomewide\n). Moreover, there is some sup-\nport for this ﬁnding, including the sex-speciﬁc nature of\nthe result, in an independent stuttering data set (J. K.\nWittke-Thompson, unpublished data).\nP p .005\nReplication of evidence for linkage in complex dis-\norders is limited, as previously mentioned. Two whole-\ngenome scans for stuttering have been published so far,\nwith only one signiﬁcant result on chromosome 12 at\n115 cM, with a LOD score of 4.61, observed to date\n(Riaz et al. 2005). Conditional analysis using families\nweighted on the basis of their evidence for linkage to\nchromosome 7 (153 cM), a genomewide-signiﬁcant link-\nage signal in our data, showed a marked increase in the\nevidence for linkage to chromosome 12, with a resulting\nthe underlying genetic variation of complex diseases\nhave met with limited success. Altmuller et al. (2001)\nanalyzed 1100 published whole-genome scans of com-\nplex human diseases and found that only 32% reported\nsigniﬁcant linkage (LOD 13.6) and that only 2% of the\nstudies showed replicated evidence for linkage. These\nresults can be attributed both to methodological limi-\ntations, such as small sample size, genotyping errors,\nand/or clinical heterogeneity, and to the underlying com-\nplexity of genetic models for common diseases arising\nfrom the actions and interactions of genetic and non-\ngenetic factors.\nRecently, several studies have advocated investigating\nthe sex-speciﬁc genetic architecture of complex disor-\nders, especially when the disorder exhibits a skewed sex\nratio (Stone et al. 2004; Weiss et al. 2005). The search\nLOD score of 2.66 (increased from a baseline LOD of\n0.57). The location of the chromosome 12 signal in our\nstuttering cohort overlaps with that reported by Riaz et\nal. (2005), providing support for the existence of a locus\naffecting susceptibility to stuttering on chromosome\n12q23-24 in our sample, which is quite historically dis-\ntinct from the Pakistani population studied in Riaz et\nal. (2005).\nChromosome 2q31-33 is a region of the genome that\nhas been repeatedly implicated in autism, with evidence\nfor linkage to this region substantially increasing when\nanalyses were performed using a narrow diagnosis of\nautism with phrase speech delay (Buxbaum et al. 2001;\nShao et al. 2002). Interestingly, this same region on chro-\nmosome 2 showed nominal evidence for linkage in the\nprimary analysis of the overall data set with use of the\never-stuttered phenotype and increased signiﬁcantly\nLOD p 1.63 P !\n(conditional\n.01\n) when families were weighted on the basis of their\nevidence for linkage to chromosome 9. A negative in-\nteraction between the chromosome 2 locus and the link-\nage signal on chromosome 7 was also detected.\n;\nThe appropriate criteria for assessing genomewide sig-\nniﬁcance in the context of sex-speciﬁc linkage analysis\nmerit some additional discussion. Because the sex-spe-\nciﬁc data used in such studies are subsets of the overall\ndata, evidence for linkage would not be considered sex-\nspeciﬁc when the signal in the subsets is merely pro-\nportional to the data included. Rather, evidence for link-\nage has generally been interpreted as being sex-speciﬁc\nwhen the evidence for linkage in the male or female\nsubset is at least as large as, and usually is substantially\nlarger than, the evidence for linkage in the overall sam-\nple. The ﬁxed structure of the sex-speciﬁc data subsets\npermits a straightforward estimation of the genomewide\nsigniﬁcance of sex-speciﬁc signals through the same sim-\nulation studies used to estimate genomewide signiﬁcance\nof the results for the overall data. That is, because the\nmale-only data subset is the same for each replicate, as\nis the female-only data subset, it is, therefore, possible\nto establish thresholds for the sex-speciﬁc signals and\nfor the magnitude of the observed increase (DLOD), to\nestimate genomewide signiﬁcance.\nIn contrast, estimating the genomewide signiﬁcance of\nan observed conditional linkage signal from this exper-\niment is difﬁcult because most simulations from the null\nmay not replicate the structure of the real data. In such\ncircumstances, conditioning on the largest observed sig-\nnal in each replicate will rarely generate a set of nonzero\nweights as large as that obtained for the real data and,\nthus, cannot appropriately estimate the genomewide sig-\nniﬁcance of a conditional increase. Although such sec-\nondary analyses must, therefore, be considered “hy-\npothesis generating,” they can be compelling when, as\nwe have shown here, they highlight regions that have\nbeen convincingly implicated in independent studies of\nthe same or related phenotypes.\nFinally, our studies emphasize the importance of con-\nsidering the sex-speciﬁc genetic architecture of complex\nphenotypes. Stuttering is one of many complex disorders\nshowing striking sex differences in severity and/or prev-\nalence, including inﬂammatory bowel disease (Fisher et\nal. 2002), mood disorders (Zubenko et al. 2003), and\nautism (Veenstra-VanderWeele and Cook 2004). Ap-\nproaches considering sex as a factor that may inﬂuence\ndisease risk in a genotype-speciﬁc manner have met with\nsuccess in identifying a variety of sex-speciﬁc suscepti-\nbility loci for different complex phenotypes. In the case\nof autism, two independent studies showed signiﬁcant\nmale-speciﬁc evidence for linkage (maximized LOD\nscore [MLS] 14.0) to chromosome 17q11, whereas anal-\nysis of the families with at least one affected female\nshowed little evidence for linkage in the same region\n(MLS 0.5) (Stone et al. 2004; Cantor et al. 2005).\nRecently, sex-speciﬁc heritability and genomewide link-\nages for a total of 17 quantitative traits associated with\ncommon human diseases, such as heart disease, hyper-\ntension, diabetes, asthma, and autoimmune disease,\nwere evaluated in the Hutterites (Weiss et al. 2006). Ten\nof these traits showed sex-speciﬁc evidence for linkage,\nwith two genomewide-signiﬁcant linkages detected in\nthe male-only subset and two genomewide-signiﬁcant\nlinkages detected in the female-only subset. Interestingly,\nof the 62 regions that showed suggestive evidence for\nlinkage in either the male-only, female-only, or complete\ndata set, only 4 regions reached suggestive signiﬁcance\nin both the sex-stratiﬁed and complete data set, whereas\nnone of the male-speciﬁc or female-speciﬁc signals over-\nlapped. Notably, none of the sex-speciﬁc signals that\nshowed genomewide signiﬁcance reached suggestive sig-\nniﬁcance in the complete data set. In light of these re-\nports, as well as the presence of sex-speciﬁc loci observed\nin this study, it seems crucial to explore sex-speciﬁc ge-\nnetic effects when conducting genomewide screens, es-\npecially in those disorders where sex differences in risk\nare observed.",
         "Developmental stuttering is a communication disorder\nthat begins in early childhood and is characterized by\ninvoluntary disruptions in the ﬂuency of verbal expres-\nsion. The most typical core behaviors are repetitions of\nsounds or syllables and articulatory ﬁxation, resulting\nin blocks or prolongations of sounds. When the disorder\ncontinues into later childhood and beyond, it may cause\nserious interference with both personal and professional\npursuits. Evidence from twin and family studies has\nclearly established a role for genetic factors in the de-\nvelopment of stuttering (Howie 1981; Kidd 1984; Yairi\net al. 1996). Increased concordance rates in MZ twins\nversus DZ twins have been consistently observed, with\npairwise concordance estimates ranging from 20% to\n83% for MZ twins and from 4% to 19% for DZ twins\n(Howie 1981; Andrews et al. 1991; Felsenfeld et al.\n2000), depending on the screening method used in the\nstudy. Both Andrews et al. (1991) and Felsenfeld et al.\n(2000) calculated that ∼70% of the variance in liability\nfor stuttering could be attributed to additive genetic ef-\nfects, whereas the remaining 30% was due to nonshared\nenvironmental effects. Familial aggregation of stuttering\nhas been extensively documented, with an increased in-\ncidence of ∼15% in ﬁrst-degree relatives of probands,\nas compared with a 5% lifetime risk in the general pop-\nulation (Kidd 1984; Ambrose et al. 1993). Segregation\nanalysis has suggested different genetic models for the\ninheritance of stuttering in families, including multifac-\ntorial/polygenic transmission (Cox et al. 1984), a major\nlocus with multifactorial/polygenic background (Am-\nbrose et al. 1993), and, recently, an autosomal dominant\nlocus (Viswanath et al. 2004).\nThe onset of stuttering usually occurs in childhood,\nbetween the ages of 3 and 6 years, with reported rates\nof natural, unassisted recovery of ∼75% (Yairi and Am-\nbrose 1999, 2005). So, whereas the lifetime incidence of\nstuttering is estimated at ∼4%–5%, its prevalence is\nlower, at 0.5%–1% (Bloodstein 1995; Felsenfeld 2002).\nAn investigation into the relationship between persis-\ntence and recovery in stuttering, with the use of pedigree\nand segregation analysis, suggested that the two phe-\nnomena were not genetically independent disorders,\nthough the persistence of stuttering may require the\ntransmission of additional genetic factors (Ambrose et\nal. 1997). In contrast, previous investigations of stut-\ntering severity (as measured by the frequency or length\nof disﬂuencies) showed no relationship between any\nmeasure of severity and genetic liability to stuttering\n(Kidd et al. 1980). It, therefore, seems prudent to use\nboth broad (any individual who ever stuttered) and nar-\nrow (those with persistent stuttering) deﬁnitions of af-\nfection status in linkage studies of stuttering, although\nthere is little justiﬁcation for using measures of stuttering\nseverity in genetic studies.\nThere is also a signiﬁcant sex bias in the incidence of\nstuttering, with a male-to-female ratio of ∼2:1 during\nchildhood increasing to 4:1 or 5:1 in adulthood (Blood-\nstein 1995; Yairi and Ambrose 2005). The increased po-\nlarity with age of affected males versus females suggests\nthat recovery from stuttering is considerably more fre-\nquent in girls than in boys (Yairi and Ambrose 1999).\nThus, investigations into the genetic basis underlying sex\neffects in stuttering are warranted.\nLOD p 4.61\nTo date, only two whole-genome scans designed to\ndetect genetic variation related to stuttering have been\npublished. Riaz et al. (2005) reported genomewide-sig-\nniﬁcant evidence for linkage of stuttering to chromo-\n) and suggestive evidence for\nsome 12q (\nlinkage to chromosome 1q (\n) in 46 highly\ninbred Pakistani families. Shugart et al. (2004) observed\nmodest evidence for linkage to chromosome 18q\nNPL p 1.51\n) in 68 families of European ancestry,\n(\nin which only individuals with persistent stuttering were\nconsidered as affected.\nLOD p 2.93\nBoth aforementioned studies used microsatellite mark-\ners at an average spacing of ∼10 cM. However, with the\nadvent of high-throughput genotyping methods that al-\nlow tens to hundreds of thousands of SNPs to be ge-\nnotyped at a relatively low cost, using high-density SNP\nmaps to detect linkage seems an attractive alternative to\nmicrosatellite mapping. In fact, there is growing evidence\nthat, compared with using traditional microsatellite\nmaps, using high-density SNP maps lends greater power\nto detect linkage (Kruglyak 1997; Evans and Cardon\n2004) and improved deﬁnition of underlying disease/\ntrait loci (John et al. 2004). For example, Evans and\nCardon (2004) showed that, when parental genotypes\nwere available, a 2-cM–dense SNP map was sufﬁcient\nto extract almost 100% of the inheritance information,\nwhereas a 10-cM–dense microsatellite map could extract\nonly ∼70%.\nHere, we present the results of a genomewide scan\nwith 110,000 SNPs genotyped in 100 families that have\nat least two relatives affected with stuttering. We have\nconsidered both susceptibility to stuttering and persis-\ntence of stuttering in our analyses, by using two phe-\nnotype deﬁnitions that distinguish between individuals\nwho have ever stuttered (broad) and those with persis-\ntent stuttering (narrow). The notable male bias in the\nprevalence of stuttering led us to investigate whether\nthere are sex-speciﬁc genes that underlie susceptibility to\nthe disorder, as has been suggested for autism, which\nalso shows a profound difference in risk by sex (Veen-\nstra-VanderWeele and Cook 2004). These studies led us\nto consider criteria for assessing genomewide signiﬁ-\ncance of sex-speciﬁc analyses. Finally, because stuttering\nis a complex disorder with multiple interacting genetic\nand environmental factors for susceptibility, we per-\nformed secondary analyses that allowed us to assess sta-\ntistical interactions of loci with the most signiﬁcant ef-\nfects in primary analyses.\nn p 16",
         "The sample comprised 110 families, each with at least two\nnon–parent/offspring relatives affected with stuttering. The\nfamilies were collected from the University of Illinois at Ur-\n); the Tel Aviv University School of\nbana-Champaign (\nMedicine, Israel (\n); and the Helsingborg Hospital, Swe-\nn p 18\nden (\n). All samples were collected with the approval\nof local institutional review boards and with informed consent.\nThe primary group of American families were identiﬁed (1)\nfrom pedigrees of 180 children previously evaluated for stut-\ntering through a longitudinal study (the Illinois Stuttering Re-\nsearch Program), (2) through a well-established, nationwide\nnetwork of speech-language pathologists specializing in stut-\ntering, and (3) through publications, announcements, and ﬂiers\nat meetings of organizations geared to provide support and\ninformation for people who stutter. Families from the Swedish\nand Israeli centers were identiﬁed and recruited by specialists\nin stuttering. Since the families were referred through a broad\nnetwork of pediatricians, day care centers, and speech-lan-\nguage pathologists and by word of mouth, they were consid-\nerably more representative of the population than a typical\nclinical sample. Blood samples from a total of 585 individuals\nwere collected. There were 362 males, 233 adult and 129 mi-\nnor, and 223 females, 182 adult and 41 minor. A total of 365\naffected-relative pairs were identiﬁed.\nDiagnosis\nIndividuals were categorized as having ever stuttered (cur-\nrently or in the past) if they met one or more of the following\ncriteria: (1) direct observation of stuttering by one of the speech\ninvestigators, (2) diagnosis and/or treatment by speech-lan-\nguage pathologists, or (3) detailed description by a participant\nof stuttering behaviors, judged sufﬁcient to positively identify\nthe disorder by a speech-language pathologist with expertise\nin stuttering. The stuttering was classiﬁed as persistent if the\nindividual stuttered for a minimum of 4 years and continued\nto stutter into later childhood and adulthood. Particular care\nwas taken to verify any cases of naturally recovered stuttering\n(see Yairi and Ambrose [1999] for full treatment of this issue).\nIf evidence was vague or conﬂicting, the stuttering was clas-\nsiﬁed as “status unknown.” Those reported as having ever\nstuttered also ﬁlled out a questionnaire regarding their history\nof stuttering. An earlier version of this questionnaire has been\nused in past research (Ambrose et al. 1997). Stuttering was\nclassiﬁed as recovered in 45 individuals. Of the remainder, 252\nexhibited persistent stuttering, and 19 were too young to de-\ntermine if they would recover or become persistent.\nGenotyping\nTen milliliters of blood was drawn from an antecubital vein\ninto two 5-ml plastic vacutainer tubes containing ethylenedi-\naminetetraacetic acid. Half of each blood sample was stored\nat 70C as a backup, and the other half was used for DNA\nextraction and genotyping. DNA was extracted from 3 ml of\nblood by use of the PureGene DNA extraction kit (Gentra),\nand the remainder of the blood was frozen. The Affymetrix\nMapping Array 10K set was used for genotyping as per the\nstandard protocol described elsewhere (Matsuzaki et al. 2004).\nThree pairs of MZ twins available in the data set were used\nto assess the discordance of genotype calls, with only one from\neach of the twin sets used in subsequent linkage analyses. The\nobserved level of genotype disagreement was low, at 0.00012–\n0.00037.\nQuality Control (QC)\nOf the 11,561 total markers, those that were not placed on\nthe deCODE map or that had a missing rate 110% were dis-\ncarded from analysis. Extensive relationship checking was per-\nformed using PREST (Sun et al. 2002) to assess the evidence\nfor the speciﬁed within-family relationships and using REL-\nPAIR (Epstein et al. 2000) to check across families for unre-\ncognized relationships and/or switched or duplicated samples.\nThree families were dropped because of unresolvable errors in\npedigree speciﬁcation. Mendelian incompatibilities were iden-\ntiﬁed using PEDCHECK (O’Connell and Weeks 1998). Any\nmarkers that were associated with Mendelian errors in more\nthan two families were dropped from subsequent analyses. All\nremaining incompatibilities were coded as “missing.”\nSince linkage studies are sensitive to the misspeciﬁcation of\nmarker-allele frequencies, we compared the marker-allele fre-\nquencies of the collected families on the basis of ethnicity. We\nobserved that the seven non-European families had allele fre-\nquencies for the included SNPs sufﬁciently different from those\nin the European families that results could be compromised,\nand we, therefore, excluded them from these analyses. SNPs\nwere also checked for departures from Hardy-Weinberg equi-\nlibrium (HWE). Signiﬁcant departures from HWE were ob-\nserved in only 0.171% of the markers (\n) and showed\nno clustering; therefore, all markers meeting the above-de-\nscribed QC standards were kept for our analyses. We note,\nhowever, that the stringent criteria we used for completion of\ngenotyping (we dropped markers with 110% missing data)\nhad largely eliminated the markers with highly signiﬁcant de-\npartures from HWE.\nP ! .00001\nIt has been shown elsewhere that failure to account for in-\ntermarker linkage disequilibrium (LD), especially when paren-\ntal genotypes are missing, can lead to inﬂated linkage scores\n(Huang et al. 2004). The effect of intermarker LD was ex-\n2r  0.8\n)\namined by removing markers that were in high LD (\nand conducting linkage analyses on the modiﬁed data set. The\naverage difference in LOD scores observed was low, at 0.025.\nThis is consistent with the previous reports, in that most fam-\nilies in our sample contain parental data, and the potential\ninﬂation in linkage evidence is more marked when parental\ndata are unavailable. When we eliminated the parental ge-\nnotypes in our data set, we did detect differences in the evi-\ndence for linkage across the genome, depending on whether\nmarkers in LD were included or excluded in the analysis. Given\nthat there appeared to be no detectable effect in our data of\nincluding markers in some LD (because of the presence of\nparental data), we report here the results of linkage analyses\nusing all markers passing the QC standards.\nAlthough the genotyping accuracy of SNPs is extremely high\n(Matsuzaki et al. 2004), the fact that they encode binary in-\nformation makes it harder to detect erroneous genotypes just\nby checking for Mendelian incompatibilities (Douglas et al.\n2002). MERLIN (Abecasis et al. 2002) was, therefore, used\nto identify any unlikely recombination events that were present\nin the data set. The baseline event rate was determined by\nrunning MERLIN on ﬁve sets of simulated data that were\nknown to have no errors. We observed a two- to threefold\nincrease in the number of events relative to the baseline, with\nthe exception of a few chromosomes (5, 18, 20, and 22) for\nwhich the number of observed events was similar to the base-\nline. All unlikely recombination events that were ﬂagged by\nMERLIN were coded as missing, and linkage analyses were\nconducted on the modiﬁed data set to determine the effect of\ngenotyping error on evidence for linkage across the genome.\nThe mean difference in LOD scores was negligible, at 0.048\nover the genome. We, therefore, report here on analyses using\nall markers passing QC standards. A total of 9,144 markers\nwere used in the ﬁnal analysis, with an average genomewide\ndensity of 0.3 cM.\nn p 100\nLinkage analysis was performed using ALLEGRO (Gu-\ndbjartsson et al. 2000) for both broad (\n) and narrow\nn p 86\n(\n) phenotype deﬁnitions. The exponential model was\nused to obtain nonparametric LOD scores (Kong and Cox\n1997), and a total LOD score for families under heterogeneity\n(HLOD) was also calculated using a parametric model derived\nfrom previous segregation analyses of stuttering (Ambrose et\nal. 1993).\nThe sex bias observed in stuttering led us to investigate\nwhether there was sex-speciﬁc evidence for linkage in these\nfamilies. The larger ever-stuttered data set was stratiﬁed on\nthe basis of sex, with the male data set generated by setting\nthe affection status of all affected females to “unknown,” and\nn p 74\n)\nvice versa for the female data set. The new male (\nand female (\n) data sets were used in linkage analyses as\ndescribed above.\nSimulations consisting of 1,000 replicates of the overall data\nset, as well as the male-only and female-only data subsets, were\ngenerated using MERLIN to determine the genomewide sig-\nniﬁcance of the linkage results, including the sex-speciﬁc ﬁnd-\nings. In assessing the genomewide signiﬁcance of sex-speciﬁc\nsignals, we tallied signals for the sex-speciﬁc subsets of rep-\nlicates at least as large as those observed in the real sex-speciﬁc\ndata that also had a sex-speciﬁc LOD difference, DLOD\n(DLOD p maximum local LOD score in sex-speciﬁc subset \nmaximum local LOD score in overall data), at least as large\nas that observed for the actual data. These criteria explicitly\nacknowledge that signals will be considered sex-speciﬁc only\nwhen the LOD score in a sex-speciﬁc data subset is larger than\nthat in the overall sample and that the signiﬁcance of signals\nwith the same LOD score for sex-speciﬁc data subsets will\ndiffer depending on the magnitude of the signal in the overall\nsample. Because the SNP maps are relatively dense, we ex-\npected a 5-cM window for calculating DLOD to be sufﬁcient\nbut examined the robustness of results when using a 5-cM\nwindow (LOD for the overall data is the maximum LOD score\nwithin 5 cM on either side of the location of the maximum\nLOD score for the largest sex-speciﬁc LOD score in that region)\nand a 10-cM window.\nTo assess potential interactions between linkage signals of\ninterest and the rest of the genome, conditional analyses were\nperformed. Three weighting functions (described below) were\nused to generate sets of family-speciﬁc weights based on the\nlargest of the primary signals obtained for the ever-stuttered\nand persistent-stutter diagnoses, as well as for the largest sex-\nspeciﬁc signal in males (chromosome 9, 60 cM; chromosome\n13, 14 cM; chromosome 15, 19 cM; chromosome 7, 153 cM).\nWeighted linkage analyses were then performed on the re-\nmainder of the genome, as described above. Weighting func-\ntions were as follows:\n1. Weight0–1: Families with a negative NPL score at the lo-\ncation under investigation were assigned a weight of 0\n(i.e., they were excluded from the subsequent weighted\nanalysis), whereas families with a positive NPL score\nwere assigned a weight of 1 (included in weighted\nanalysis).\n2. Weight0–NPL: Families with a negative NPL score received\na weight of 0, whereas families with a positive NPL were\nassigned a weight equivalent to the value of the NPL\nscore.\n3. Weight1–0: Families with a negative NPL score were as-\nsigned a weight of 1, whereas families with a positive\nNPL received a weight of 0.\nStatistical signiﬁcance was assessed by permuting the original\nweights ﬁles (1,000 total permutations) and by conducting\nlinkage analyses on the actual data set with use of each of the\n1,000 permuted weight ﬁles.\nThe family-based association test (FBAT) (v.1.5.5) (Rabi-\nnowitz and Laird 2000) was used to test the null hypothesis\nof no association and no linkage for each marker with the\npersistent-stuttering phenotype under an additive genetic\nmodel. Genomewide signiﬁcance was assessed by comparing\nthe number of signals exceeding a range of P value thresholds\nin the actual data set with the 1,000 null replicates used in the\nlinkage study. For associated markers showing strong LD\n2r  0.8\n(\n), only one marker was counted in the actual data set,\nsince linkage equilibrium was assumed under the null hypoth-\nesis. Overlapping signals for FBAT and linkage (nonparametric\nLOD score) were considered if a marker exceeded the speciﬁed\nsigniﬁcance thresholds for both analyses. Genomewide signif-\nicance for the overlapping signals was assessed by comparing\nthe actual and the 1,000 null replicates.",
         "Results\nThe genomewide nonparametric LOD scores (Kong and\nCox 1997) for the broad and narrow phenotype deﬁ-\nnitions are shown in ﬁgure 1. The HLOD results tracked\nremarkably well with the nonparametric LOD scores,\nwith a correlation over the genome of 0.784, and are,\nthus, not shown separately. For the ever-stuttered phe-\nnotype, a total of 100 families were included in the anal-\nysis, and the largest score was observed on chromosome\n9 (60 cM), with a LOD of 2.3 (ﬁg. 2a). Chromosomes\n2 and 7 also showed nominal evidence for linkage, with\nLOD scores of 1.72 and 1.69, respectively. For the per-\nsistent-stuttering phenotype, a total of 86 families were\nincluded in the analysis, and positive evidence for linkage\nspanning a large region on chromosome 15 (5–96 cM)\nand peaking at 23 cM, with a LOD score of 1.95, was\nobserved (ﬁg. 2b). Modest evidence for linkage to chro-\nmosome 13 (14 cM), with a LOD score of 1.72, was\nalso seen.\nSex-speciﬁc analyses resulted in increased evidence for\nlinkage in three regions across the genome. Genome-\nwide-signiﬁcant evidence for linkage on chromosome 21\n(LOD score of 4.5; DLOD p 4.46) was observed in the\nfemale-only families (ﬁg. 3a). Within the 1,000 female-\nonly simulated data sets, there was a peak that exceeded\nthe 4.5 LOD threshold and the 4.46 DLOD threshold\n(with use of either a 5-cM or a 10-cM window) only\nP p .005\nﬁve times (genomewide\n). Analysis of the male\ndata set showed increased evidence for linkage on chro-\nmosome 7 at 153 cM (ﬁg. 3b) and chromosome 20 at\n38 cM (ﬁg. 3c), with LOD scores of 2.99 and 2.18,\nrespectively, and DLOD values of 1.30 (for both window\nsizes) and 1.52 (for 10-cM window; DLOD p 1.39 for\n5-cM window), respectively. The genomewide signiﬁ-\ncance of the chromosome 7 linkage signal was estimated\n(for both window sizes), whereas that for\nas\nchromosome 20 was\nfor a 5-cM window and\n.115 for a 10-cM window.\nP p .096\nP p .04\nConditional analyses identiﬁed four regions across the\ngenome that showed a notable increase in evidence for\nlinkage, with LOD scores 12.5. The weighting functions\nenable us to detect both positive (weight0–1 and weight0–\nNPL) and negative (weight1–0) interactions, since families\nare assigned weights on the basis of the presence or\nabsence of linkage, respectively, to the signal of interest.\nWhen families were weighted on the basis of their evi-\ndence for linkage at chromosome 15 (weight0–1), a pos-\nitive interaction was observed with chromosome 13, for\nwhich the evidence for linkage increased from a LOD\n) (ﬁg. 4b). Conversely, a\nscore of 1.78 to 2.8 (\nP p .013\nnegative interaction was observed with chromosome 20\n(weight1–0) at 40 cM, with a LOD score of 4.5 (ﬁg. 4a).\nIn the 1,000 simulated data sets that were generated, a\n),\npeak crossing that threshold was not observed (\nmaking this interaction highly signiﬁcant. In addition,\nthe location of the peak overlapped the male-speciﬁc\nlinkage signal previously detected. When families were\nweighted on the basis of their evidence for linkage at\nchromosome 9, increased evidence for linkage on chro-\nmosome 2 at 193 cM was observed using the weight0–1\nfunction, with the original LOD score of 1.63 increasing\n). Interestingly, when families were\nto 3.16 (\nweighted on the basis of their evidence for linkage at\nchromosome 7 (153 cM), analysis with the weight1–0\nfunction led to the observation of a negative interaction\n) (ﬁg.\nin the same location on chromosome 2 (\n4c). Finally, a strong positive interaction (\n) with\nthe chromosome 7 linkage signal and chromosome 12\nat 120 cM was observed, with the LOD score increasing\nfrom 0.57 to 2.66 (ﬁg. 4d).\nP p .013\nP p .01\nP p .01\nFBAT did not reveal any marker showing genome-\nwide-signiﬁcant association for persistent stuttering.\nMoreover, there were no more overlapping signals be-\ntween FBAT and linkage analyses than expected by\nchance alone (data not shown).",
         "downloading",
         "1Department of Genetics and Development, Columbia University, New York; 2Department of Speech and Hearing Science, University of\nIllinois, Urbana-Champaign; Departments of 3Medicine and 4Human Genetics, The University of Chicago, and 5Institute for Juvenile\nResearch, Department of Psychiatry, University of Illinois at Chicago, Chicago; 6Helsingborg Hospital, Helsingborg, Sweden; and 7Tel Aviv\nUniversity School of Medicine, Tel Aviv\nStuttering is a speech disorder long recognized to have a genetic component. Recent linkage studies mapped a\nsusceptibility locus for stuttering to chromosome 12 in 46 highly inbred families ascertained in Pakistan. We report\nhere on linkage studies in 100 families of European descent ascertained in the United States, Sweden, and Israel.\nThese families included 252 individuals exhibiting persistent stuttering, 45 individuals classiﬁed as recovered from\nstuttering, and 19 individuals too young to classify. Primary analyses identiﬁed moderate evidence for linkage of\nthe broader diagnosis of “ever stuttered” (including both persistent and recovered stuttering) on chromosome 9\n(LOD p 2.3 at 60 cM) and of the narrower diagnosis of persistent stuttering on chromosome 15 (LOD p 1.95\nat 23 cM). In contrast, sex-speciﬁc evidence for linkage on chromosome 7 at 153 cM in the male-only data subset\n(LOD p 2.99) and on chromosome 21 at 34 cM in the female-only data subset (LOD p 4.5) met genomewide\ncriteria for signiﬁcance. Secondary analyses revealed a signiﬁcant increase in the evidence for linkage on chromosome\n12, conditional on the evidence for linkage at chromosome 7, with the location of the increased signal congruent\nwith the previously reported signal in families ascertained in Pakistan. In addition, a region on chromosome 2 (193\ncM) showed a signiﬁcant increase in the evidence for linkage conditional on either chromosome 9 (positive) or\nchromosome 7 (negative); this chromosome 2 region has been implicated elsewhere in studies on autism, with\nincreased evidence for linkage observed when the sample is restricted to those with delayed onset of phrase speech.\nOur results support the hypothesis that the genetic component to stuttering has signiﬁcant sex effects.\nDevelopmental stuttering is a communication disorder\nthat begins in early childhood and is characterized by\ninvoluntary disruptions in the ﬂuency of verbal expres-\nsion. The most typical core behaviors are repetitions of\nsounds or syllables and articulatory ﬁxation, resulting\nin blocks or prolongations of sounds. When the disorder\ncontinues into later childhood and beyond, it may cause\nserious interference with both personal and professional\npursuits. Evidence from twin and family studies has\nclearly established a role for genetic factors in the de-\nvelopment of stuttering (Howie 1981; Kidd 1984; Yairi\net al. 1996). Increased concordance rates in MZ twins\nversus DZ twins have been consistently observed, with\npairwise concordance estimates ranging from 20% to\n83% for MZ twins and from 4% to 19% for DZ twins\n(Howie 1981; Andrews et al. 1991; Felsenfeld et al.\n2000), depending on the screening method used in the\nstudy. Both Andrews et al. (1991) and Felsenfeld et al.\n(2000) calculated that ∼70% of the variance in liability\nfor stuttering could be attributed to additive genetic ef-\nfects, whereas the remaining 30% was due to nonshared\nenvironmental effects. Familial aggregation of stuttering\nhas been extensively documented, with an increased in-\ncidence of ∼15% in ﬁrst-degree relatives of probands,\nas compared with a 5% lifetime risk in the general pop-\nulation (Kidd 1984; Ambrose et al. 1993). Segregation\nanalysis has suggested different genetic models for the\ninheritance of stuttering in families, including multifac-\ntorial/polygenic transmission (Cox et al. 1984), a major\nlocus with multifactorial/polygenic background (Am-\nbrose et al. 1993), and, recently, an autosomal dominant\nlocus (Viswanath et al. 2004).\nThe onset of stuttering usually occurs in childhood,\nbetween the ages of 3 and 6 years, with reported rates\nof natural, unassisted recovery of ∼75% (Yairi and Am-\nbrose 1999, 2005). So, whereas the lifetime incidence of\nstuttering is estimated at ∼4%–5%, its prevalence is\nlower, at 0.5%–1% (Bloodstein 1995; Felsenfeld 2002).\nAn investigation into the relationship between persis-\ntence and recovery in stuttering, with the use of pedigree\nand segregation analysis, suggested that the two phe-\nnomena were not genetically independent disorders,\nthough the persistence of stuttering may require the\ntransmission of additional genetic factors (Ambrose et\nal. 1997). In contrast, previous investigations of stut-\ntering severity (as measured by the frequency or length\nof disﬂuencies) showed no relationship between any\nmeasure of severity and genetic liability to stuttering\n(Kidd et al. 1980). It, therefore, seems prudent to use\nboth broad (any individual who ever stuttered) and nar-\nrow (those with persistent stuttering) deﬁnitions of af-\nfection status in linkage studies of stuttering, although\nthere is little justiﬁcation for using measures of stuttering\nseverity in genetic studies.\nThere is also a signiﬁcant sex bias in the incidence of\nstuttering, with a male-to-female ratio of ∼2:1 during\nchildhood increasing to 4:1 or 5:1 in adulthood (Blood-\nstein 1995; Yairi and Ambrose 2005). The increased po-\nlarity with age of affected males versus females suggests\nthat recovery from stuttering is considerably more fre-\nquent in girls than in boys (Yairi and Ambrose 1999).\nThus, investigations into the genetic basis underlying sex\neffects in stuttering are warranted.\nLOD p 4.61\nTo date, only two whole-genome scans designed to\ndetect genetic variation related to stuttering have been\npublished. Riaz et al. (2005) reported genomewide-sig-\nniﬁcant evidence for linkage of stuttering to chromo-\n) and suggestive evidence for\nsome 12q (\nlinkage to chromosome 1q (\n) in 46 highly\ninbred Pakistani families. Shugart et al. (2004) observed\nmodest evidence for linkage to chromosome 18q\nNPL p 1.51\n) in 68 families of European ancestry,\n(\nin which only individuals with persistent stuttering were\nconsidered as affected.\nLOD p 2.93\nBoth aforementioned studies used microsatellite mark-\ners at an average spacing of ∼10 cM. However, with the\nadvent of high-throughput genotyping methods that al-\nlow tens to hundreds of thousands of SNPs to be ge-\nnotyped at a relatively low cost, using high-density SNP\nmaps to detect linkage seems an attractive alternative to\nmicrosatellite mapping. In fact, there is growing evidence\nthat, compared with using traditional microsatellite\nmaps, using high-density SNP maps lends greater power\nto detect linkage (Kruglyak 1997; Evans and Cardon\n2004) and improved deﬁnition of underlying disease/\ntrait loci (John et al. 2004). For example, Evans and\nCardon (2004) showed that, when parental genotypes\nwere available, a 2-cM–dense SNP map was sufﬁcient\nto extract almost 100% of the inheritance information,\nwhereas a 10-cM–dense microsatellite map could extract\nonly ∼70%.\nHere, we present the results of a genomewide scan\nwith 110,000 SNPs genotyped in 100 families that have\nat least two relatives affected with stuttering. We have\nconsidered both susceptibility to stuttering and persis-\ntence of stuttering in our analyses, by using two phe-\nnotype deﬁnitions that distinguish between individuals\nwho have ever stuttered (broad) and those with persis-\ntent stuttering (narrow). The notable male bias in the\nprevalence of stuttering led us to investigate whether\nthere are sex-speciﬁc genes that underlie susceptibility to\nthe disorder, as has been suggested for autism, which\nalso shows a profound difference in risk by sex (Veen-\nstra-VanderWeele and Cook 2004). These studies led us\nto consider criteria for assessing genomewide signiﬁ-\ncance of sex-speciﬁc analyses. Finally, because stuttering\nis a complex disorder with multiple interacting genetic\nand environmental factors for susceptibility, we per-\nformed secondary analyses that allowed us to assess sta-\ntistical interactions of loci with the most signiﬁcant ef-\nfects in primary analyses.\nn p 16\nThe sample comprised 110 families, each with at least two\nnon–parent/offspring relatives affected with stuttering. The\nfamilies were collected from the University of Illinois at Ur-\n); the Tel Aviv University School of\nbana-Champaign (\nMedicine, Israel (\n); and the Helsingborg Hospital, Swe-\nn p 18\nden (\n). All samples were collected with the approval\nof local institutional review boards and with informed consent.\nThe primary group of American families were identiﬁed (1)\nfrom pedigrees of 180 children previously evaluated for stut-\ntering through a longitudinal study (the Illinois Stuttering Re-\nsearch Program), (2) through a well-established, nationwide\nnetwork of speech-language pathologists specializing in stut-\ntering, and (3) through publications, announcements, and ﬂiers\nat meetings of organizations geared to provide support and\ninformation for people who stutter. Families from the Swedish\nand Israeli centers were identiﬁed and recruited by specialists\nin stuttering. Since the families were referred through a broad\nnetwork of pediatricians, day care centers, and speech-lan-\nguage pathologists and by word of mouth, they were consid-\nerably more representative of the population than a typical\nclinical sample. Blood samples from a total of 585 individuals\nwere collected. There were 362 males, 233 adult and 129 mi-\nnor, and 223 females, 182 adult and 41 minor. A total of 365\naffected-relative pairs were identiﬁed.\nDiagnosis\nIndividuals were categorized as having ever stuttered (cur-\nrently or in the past) if they met one or more of the following\ncriteria: (1) direct observation of stuttering by one of the speech\ninvestigators, (2) diagnosis and/or treatment by speech-lan-\nguage pathologists, or (3) detailed description by a participant\nof stuttering behaviors, judged sufﬁcient to positively identify\nthe disorder by a speech-language pathologist with expertise\nin stuttering. The stuttering was classiﬁed as persistent if the\nindividual stuttered for a minimum of 4 years and continued\nto stutter into later childhood and adulthood. Particular care\nwas taken to verify any cases of naturally recovered stuttering\n(see Yairi and Ambrose [1999] for full treatment of this issue).\nIf evidence was vague or conﬂicting, the stuttering was clas-\nsiﬁed as “status unknown.” Those reported as having ever\nstuttered also ﬁlled out a questionnaire regarding their history\nof stuttering. An earlier version of this questionnaire has been\nused in past research (Ambrose et al. 1997). Stuttering was\nclassiﬁed as recovered in 45 individuals. Of the remainder, 252\nexhibited persistent stuttering, and 19 were too young to de-\ntermine if they would recover or become persistent.\nGenotyping\nTen milliliters of blood was drawn from an antecubital vein\ninto two 5-ml plastic vacutainer tubes containing ethylenedi-\naminetetraacetic acid. Half of each blood sample was stored\nat 70C as a backup, and the other half was used for DNA\nextraction and genotyping. DNA was extracted from 3 ml of\nblood by use of the PureGene DNA extraction kit (Gentra),\nand the remainder of the blood was frozen. The Affymetrix\nMapping Array 10K set was used for genotyping as per the\nstandard protocol described elsewhere (Matsuzaki et al. 2004).\nThree pairs of MZ twins available in the data set were used\nto assess the discordance of genotype calls, with only one from\neach of the twin sets used in subsequent linkage analyses. The\nobserved level of genotype disagreement was low, at 0.00012–\n0.00037.\nQuality Control (QC)\nOf the 11,561 total markers, those that were not placed on\nthe deCODE map or that had a missing rate 110% were dis-\ncarded from analysis. Extensive relationship checking was per-\nformed using PREST (Sun et al. 2002) to assess the evidence\nfor the speciﬁed within-family relationships and using REL-\nPAIR (Epstein et al. 2000) to check across families for unre-\ncognized relationships and/or switched or duplicated samples.\nThree families were dropped because of unresolvable errors in\npedigree speciﬁcation. Mendelian incompatibilities were iden-\ntiﬁed using PEDCHECK (O’Connell and Weeks 1998). Any\nmarkers that were associated with Mendelian errors in more\nthan two families were dropped from subsequent analyses. All\nremaining incompatibilities were coded as “missing.”\nSince linkage studies are sensitive to the misspeciﬁcation of\nmarker-allele frequencies, we compared the marker-allele fre-\nquencies of the collected families on the basis of ethnicity. We\nobserved that the seven non-European families had allele fre-\nquencies for the included SNPs sufﬁciently different from those\nin the European families that results could be compromised,\nand we, therefore, excluded them from these analyses. SNPs\nwere also checked for departures from Hardy-Weinberg equi-\nlibrium (HWE). Signiﬁcant departures from HWE were ob-\nserved in only 0.171% of the markers (\n) and showed\nno clustering; therefore, all markers meeting the above-de-\nscribed QC standards were kept for our analyses. We note,\nhowever, that the stringent criteria we used for completion of\ngenotyping (we dropped markers with 110% missing data)\nhad largely eliminated the markers with highly signiﬁcant de-\npartures from HWE.\nP ! .00001\nIt has been shown elsewhere that failure to account for in-\ntermarker linkage disequilibrium (LD), especially when paren-\ntal genotypes are missing, can lead to inﬂated linkage scores\n(Huang et al. 2004). The effect of intermarker LD was ex-\n2r  0.8\n)\namined by removing markers that were in high LD (\nand conducting linkage analyses on the modiﬁed data set. The\naverage difference in LOD scores observed was low, at 0.025.\nThis is consistent with the previous reports, in that most fam-\nilies in our sample contain parental data, and the potential\ninﬂation in linkage evidence is more marked when parental\ndata are unavailable. When we eliminated the parental ge-\nnotypes in our data set, we did detect differences in the evi-\ndence for linkage across the genome, depending on whether\nmarkers in LD were included or excluded in the analysis. Given\nthat there appeared to be no detectable effect in our data of\nincluding markers in some LD (because of the presence of\nparental data), we report here the results of linkage analyses\nusing all markers passing the QC standards.\nAlthough the genotyping accuracy of SNPs is extremely high\n(Matsuzaki et al. 2004), the fact that they encode binary in-\nformation makes it harder to detect erroneous genotypes just\nby checking for Mendelian incompatibilities (Douglas et al.\n2002). MERLIN (Abecasis et al. 2002) was, therefore, used\nto identify any unlikely recombination events that were present\nin the data set. The baseline event rate was determined by\nrunning MERLIN on ﬁve sets of simulated data that were\nknown to have no errors. We observed a two- to threefold\nincrease in the number of events relative to the baseline, with\nthe exception of a few chromosomes (5, 18, 20, and 22) for\nwhich the number of observed events was similar to the base-\nline. All unlikely recombination events that were ﬂagged by\nMERLIN were coded as missing, and linkage analyses were\nconducted on the modiﬁed data set to determine the effect of\ngenotyping error on evidence for linkage across the genome.\nThe mean difference in LOD scores was negligible, at 0.048\nover the genome. We, therefore, report here on analyses using\nall markers passing QC standards. A total of 9,144 markers\nwere used in the ﬁnal analysis, with an average genomewide\ndensity of 0.3 cM.\nn p 100\nLinkage analysis was performed using ALLEGRO (Gu-\ndbjartsson et al. 2000) for both broad (\n) and narrow\nn p 86\n(\n) phenotype deﬁnitions. The exponential model was\nused to obtain nonparametric LOD scores (Kong and Cox\n1997), and a total LOD score for families under heterogeneity\n(HLOD) was also calculated using a parametric model derived\nfrom previous segregation analyses of stuttering (Ambrose et\nal. 1993).\nThe sex bias observed in stuttering led us to investigate\nwhether there was sex-speciﬁc evidence for linkage in these\nfamilies. The larger ever-stuttered data set was stratiﬁed on\nthe basis of sex, with the male data set generated by setting\nthe affection status of all affected females to “unknown,” and\nn p 74\n)\nvice versa for the female data set. The new male (\nand female (\n) data sets were used in linkage analyses as\ndescribed above.\nSimulations consisting of 1,000 replicates of the overall data\nset, as well as the male-only and female-only data subsets, were\ngenerated using MERLIN to determine the genomewide sig-\nniﬁcance of the linkage results, including the sex-speciﬁc ﬁnd-\nings. In assessing the genomewide signiﬁcance of sex-speciﬁc\nsignals, we tallied signals for the sex-speciﬁc subsets of rep-\nlicates at least as large as those observed in the real sex-speciﬁc\ndata that also had a sex-speciﬁc LOD difference, DLOD\n(DLOD p maximum local LOD score in sex-speciﬁc subset \nmaximum local LOD score in overall data), at least as large\nas that observed for the actual data. These criteria explicitly\nacknowledge that signals will be considered sex-speciﬁc only\nwhen the LOD score in a sex-speciﬁc data subset is larger than\nthat in the overall sample and that the signiﬁcance of signals\nwith the same LOD score for sex-speciﬁc data subsets will\ndiffer depending on the magnitude of the signal in the overall\nsample. Because the SNP maps are relatively dense, we ex-\npected a 5-cM window for calculating DLOD to be sufﬁcient\nbut examined the robustness of results when using a 5-cM\nwindow (LOD for the overall data is the maximum LOD score\nwithin 5 cM on either side of the location of the maximum\nLOD score for the largest sex-speciﬁc LOD score in that region)\nand a 10-cM window.\nTo assess potential interactions between linkage signals of\ninterest and the rest of the genome, conditional analyses were\nperformed. Three weighting functions (described below) were\nused to generate sets of family-speciﬁc weights based on the\nlargest of the primary signals obtained for the ever-stuttered\nand persistent-stutter diagnoses, as well as for the largest sex-\nspeciﬁc signal in males (chromosome 9, 60 cM; chromosome\n13, 14 cM; chromosome 15, 19 cM; chromosome 7, 153 cM).\nWeighted linkage analyses were then performed on the re-\nmainder of the genome, as described above. Weighting func-\ntions were as follows:\n1. Weight0–1: Families with a negative NPL score at the lo-\ncation under investigation were assigned a weight of 0\n(i.e., they were excluded from the subsequent weighted\nanalysis), whereas families with a positive NPL score\nwere assigned a weight of 1 (included in weighted\nanalysis).\n2. Weight0–NPL: Families with a negative NPL score received\na weight of 0, whereas families with a positive NPL were\nassigned a weight equivalent to the value of the NPL\nscore.\n3. Weight1–0: Families with a negative NPL score were as-\nsigned a weight of 1, whereas families with a positive\nNPL received a weight of 0.\nStatistical signiﬁcance was assessed by permuting the original\nweights ﬁles (1,000 total permutations) and by conducting\nlinkage analyses on the actual data set with use of each of the\n1,000 permuted weight ﬁles.\nThe family-based association test (FBAT) (v.1.5.5) (Rabi-\nnowitz and Laird 2000) was used to test the null hypothesis\nof no association and no linkage for each marker with the\npersistent-stuttering phenotype under an additive genetic\nmodel. Genomewide signiﬁcance was assessed by comparing\nthe number of signals exceeding a range of P value thresholds\nin the actual data set with the 1,000 null replicates used in the\nlinkage study. For associated markers showing strong LD\n2r  0.8\n(\n), only one marker was counted in the actual data set,\nsince linkage equilibrium was assumed under the null hypoth-\nesis. Overlapping signals for FBAT and linkage (nonparametric\nLOD score) were considered if a marker exceeded the speciﬁed\nsigniﬁcance thresholds for both analyses. Genomewide signif-\nicance for the overlapping signals was assessed by comparing\nthe actual and the 1,000 null replicates.\nResults\nThe genomewide nonparametric LOD scores (Kong and\nCox 1997) for the broad and narrow phenotype deﬁ-\nnitions are shown in ﬁgure 1. The HLOD results tracked\nremarkably well with the nonparametric LOD scores,\nwith a correlation over the genome of 0.784, and are,\nthus, not shown separately. For the ever-stuttered phe-\nnotype, a total of 100 families were included in the anal-\nysis, and the largest score was observed on chromosome\n9 (60 cM), with a LOD of 2.3 (ﬁg. 2a). Chromosomes\n2 and 7 also showed nominal evidence for linkage, with\nLOD scores of 1.72 and 1.69, respectively. For the per-\nsistent-stuttering phenotype, a total of 86 families were\nincluded in the analysis, and positive evidence for linkage\nspanning a large region on chromosome 15 (5–96 cM)\nand peaking at 23 cM, with a LOD score of 1.95, was\nobserved (ﬁg. 2b). Modest evidence for linkage to chro-\nmosome 13 (14 cM), with a LOD score of 1.72, was\nalso seen.\nSex-speciﬁc analyses resulted in increased evidence for\nlinkage in three regions across the genome. Genome-\nwide-signiﬁcant evidence for linkage on chromosome 21\n(LOD score of 4.5; DLOD p 4.46) was observed in the\nfemale-only families (ﬁg. 3a). Within the 1,000 female-\nonly simulated data sets, there was a peak that exceeded\nthe 4.5 LOD threshold and the 4.46 DLOD threshold\n(with use of either a 5-cM or a 10-cM window) only\nP p .005\nﬁve times (genomewide\n). Analysis of the male\ndata set showed increased evidence for linkage on chro-\nmosome 7 at 153 cM (ﬁg. 3b) and chromosome 20 at\n38 cM (ﬁg. 3c), with LOD scores of 2.99 and 2.18,\nrespectively, and DLOD values of 1.30 (for both window\nsizes) and 1.52 (for 10-cM window; DLOD p 1.39 for\n5-cM window), respectively. The genomewide signiﬁ-\ncance of the chromosome 7 linkage signal was estimated\n(for both window sizes), whereas that for\nas\nchromosome 20 was\nfor a 5-cM window and\n.115 for a 10-cM window.\nP p .096\nP p .04\nConditional analyses identiﬁed four regions across the\ngenome that showed a notable increase in evidence for\nlinkage, with LOD scores 12.5. The weighting functions\nenable us to detect both positive (weight0–1 and weight0–\nNPL) and negative (weight1–0) interactions, since families\nare assigned weights on the basis of the presence or\nabsence of linkage, respectively, to the signal of interest.\nWhen families were weighted on the basis of their evi-\ndence for linkage at chromosome 15 (weight0–1), a pos-\nitive interaction was observed with chromosome 13, for\nwhich the evidence for linkage increased from a LOD\n) (ﬁg. 4b). Conversely, a\nscore of 1.78 to 2.8 (\nP p .013\nnegative interaction was observed with chromosome 20\n(weight1–0) at 40 cM, with a LOD score of 4.5 (ﬁg. 4a).\nIn the 1,000 simulated data sets that were generated, a\n),\npeak crossing that threshold was not observed (\nmaking this interaction highly signiﬁcant. In addition,\nthe location of the peak overlapped the male-speciﬁc\nlinkage signal previously detected. When families were\nweighted on the basis of their evidence for linkage at\nchromosome 9, increased evidence for linkage on chro-\nmosome 2 at 193 cM was observed using the weight0–1\nfunction, with the original LOD score of 1.63 increasing\n). Interestingly, when families were\nto 3.16 (\nweighted on the basis of their evidence for linkage at\nchromosome 7 (153 cM), analysis with the weight1–0\nfunction led to the observation of a negative interaction\n) (ﬁg.\nin the same location on chromosome 2 (\n4c). Finally, a strong positive interaction (\n) with\nthe chromosome 7 linkage signal and chromosome 12\nat 120 cM was observed, with the LOD score increasing\nfrom 0.57 to 2.66 (ﬁg. 4d).\nP p .013\nP p .01\nP p .01\nFBAT did not reveal any marker showing genome-\nwide-signiﬁcant association for persistent stuttering.\nMoreover, there were no more overlapping signals be-\ntween FBAT and linkage analyses than expected by\nchance alone (data not shown).\nDiscussion\nGenomewide linkage analysis of 100 families segregating\nfor stuttering resulted in suggestive evidence for linkage\non chromosome 9, by use of the broad phenotype def-\ninition, with a LOD score of 2.3 at 60 cM. Modest\nevidence for linkage with the persistent-stuttering di-\nagnosis was detected on chromosomes 15 and 13, with\nLOD scores of 1.95 (23 cM) and 1.72 (14 cM), respec-\ntively. However, primary linkage mapping of the overall\ndata did not yield any LOD scores in the range of criteria\nestablished for genomewide signiﬁcance (Lander and\nKruglyak 1995). These results are not surprising, since\na large proportion of linkage studies designed to identify\nfor potential sex-related loci in our stuttering cohort was\nquite successful, leading to the identiﬁcation of several\nregions that merit further investigation. We observed ge-\nnomewide-signiﬁcant evidence for linkage (genomewide\nP ! .04\n) of stuttering in the male-only data set on chro-\nmosome 7 (153 cM). Evidence for linkage to this region\nis supported by replication of the ﬁnding (including the\nsex-speciﬁc effect) in the Hutterite stuttering cohort (J.\nK. Wittke-Thompson, unpublished data). Modest evi-\ndence for linkage to chromosome 7 (160 cM) was also\nobserved in families ascertained for speciﬁc language im-\npairment (SLI), a disorder characterized by impairments\nrestricted to the domain of language skills and in the\nabsence of other factors such as mental retardation and\nspeech-motor or sensory deﬁcits (Bartlett et al. 2004).\nThis region was also identiﬁed as harboring a language-\nrelated QTL for autism (Alarcon et al. 2005), a disorder\ncharacterized by disturbances in social, communicative,\nand behavioral functioning and that also shows a strik-\ning male bias in prevalence, with consistent estimates of\n4:1 affected males versus females (Veenstra-Vander-\nWeele and Cook 2004). A greater-than-expected prev-\nalence of SLI in families with autistic disorder and vice\nversa has been observed, as has the existence of over-\nlapping phenotypes for the two disorders (Folstein and\nMankoski 2000). It is, therefore, tempting to speculate\nthat there are one or more genes in this region of chro-\nmosome 7 that help to shape dimensions of speech and\nlanguage development and that may differ (e.g., in tim-\ning) between males and females, with genetic variation\nat the gene(s) affecting susceptibility to SLI, autism, and\nstuttering. We note, however, that this region does not\nencompass the FOXP2 gene identiﬁed elsewhere as the\ncausal gene for a rare speech and language disorder seg-\nregating in a large pedigree (Lai et al. 2001).\nWe also obtained genomewide-signiﬁcant evidence for\nlinkage of stuttering in the female-only subgroup on\nchromosome 21 (33.5 cM), with a LOD score of 4.5.\nAlthough the LOD score is likely to be poorly estimated\nin a sample this small, simulation studies conﬁrm that\na signal of this magnitude is very unlikely by chance\n(genomewide\n). Moreover, there is some sup-\nport for this ﬁnding, including the sex-speciﬁc nature of\nthe result, in an independent stuttering data set (J. K.\nWittke-Thompson, unpublished data).\nP p .005\nReplication of evidence for linkage in complex dis-\norders is limited, as previously mentioned. Two whole-\ngenome scans for stuttering have been published so far,\nwith only one signiﬁcant result on chromosome 12 at\n115 cM, with a LOD score of 4.61, observed to date\n(Riaz et al. 2005). Conditional analysis using families\nweighted on the basis of their evidence for linkage to\nchromosome 7 (153 cM), a genomewide-signiﬁcant link-\nage signal in our data, showed a marked increase in the\nevidence for linkage to chromosome 12, with a resulting\nthe underlying genetic variation of complex diseases\nhave met with limited success. Altmuller et al. (2001)\nanalyzed 1100 published whole-genome scans of com-\nplex human diseases and found that only 32% reported\nsigniﬁcant linkage (LOD 13.6) and that only 2% of the\nstudies showed replicated evidence for linkage. These\nresults can be attributed both to methodological limi-\ntations, such as small sample size, genotyping errors,\nand/or clinical heterogeneity, and to the underlying com-\nplexity of genetic models for common diseases arising\nfrom the actions and interactions of genetic and non-\ngenetic factors.\nRecently, several studies have advocated investigating\nthe sex-speciﬁc genetic architecture of complex disor-\nders, especially when the disorder exhibits a skewed sex\nratio (Stone et al. 2004; Weiss et al. 2005). The search\nLOD score of 2.66 (increased from a baseline LOD of\n0.57). The location of the chromosome 12 signal in our\nstuttering cohort overlaps with that reported by Riaz et\nal. (2005), providing support for the existence of a locus\naffecting susceptibility to stuttering on chromosome\n12q23-24 in our sample, which is quite historically dis-\ntinct from the Pakistani population studied in Riaz et\nal. (2005).\nChromosome 2q31-33 is a region of the genome that\nhas been repeatedly implicated in autism, with evidence\nfor linkage to this region substantially increasing when\nanalyses were performed using a narrow diagnosis of\nautism with phrase speech delay (Buxbaum et al. 2001;\nShao et al. 2002). Interestingly, this same region on chro-\nmosome 2 showed nominal evidence for linkage in the\nprimary analysis of the overall data set with use of the\never-stuttered phenotype and increased signiﬁcantly\nLOD p 1.63 P !\n(conditional\n.01\n) when families were weighted on the basis of their\nevidence for linkage to chromosome 9. A negative in-\nteraction between the chromosome 2 locus and the link-\nage signal on chromosome 7 was also detected.\n;\nThe appropriate criteria for assessing genomewide sig-\nniﬁcance in the context of sex-speciﬁc linkage analysis\nmerit some additional discussion. Because the sex-spe-\nciﬁc data used in such studies are subsets of the overall\ndata, evidence for linkage would not be considered sex-\nspeciﬁc when the signal in the subsets is merely pro-\nportional to the data included. Rather, evidence for link-\nage has generally been interpreted as being sex-speciﬁc\nwhen the evidence for linkage in the male or female\nsubset is at least as large as, and usually is substantially\nlarger than, the evidence for linkage in the overall sam-\nple. The ﬁxed structure of the sex-speciﬁc data subsets\npermits a straightforward estimation of the genomewide\nsigniﬁcance of sex-speciﬁc signals through the same sim-\nulation studies used to estimate genomewide signiﬁcance\nof the results for the overall data. That is, because the\nmale-only data subset is the same for each replicate, as\nis the female-only data subset, it is, therefore, possible\nto establish thresholds for the sex-speciﬁc signals and\nfor the magnitude of the observed increase (DLOD), to\nestimate genomewide signiﬁcance.\nIn contrast, estimating the genomewide signiﬁcance of\nan observed conditional linkage signal from this exper-\niment is difﬁcult because most simulations from the null\nmay not replicate the structure of the real data. In such\ncircumstances, conditioning on the largest observed sig-\nnal in each replicate will rarely generate a set of nonzero\nweights as large as that obtained for the real data and,\nthus, cannot appropriately estimate the genomewide sig-\nniﬁcance of a conditional increase. Although such sec-\nondary analyses must, therefore, be considered “hy-\npothesis generating,” they can be compelling when, as\nwe have shown here, they highlight regions that have\nbeen convincingly implicated in independent studies of\nthe same or related phenotypes.\nFinally, our studies emphasize the importance of con-\nsidering the sex-speciﬁc genetic architecture of complex\nphenotypes. Stuttering is one of many complex disorders\nshowing striking sex differences in severity and/or prev-\nalence, including inﬂammatory bowel disease (Fisher et\nal. 2002), mood disorders (Zubenko et al. 2003), and\nautism (Veenstra-VanderWeele and Cook 2004). Ap-\nproaches considering sex as a factor that may inﬂuence\ndisease risk in a genotype-speciﬁc manner have met with\nsuccess in identifying a variety of sex-speciﬁc suscepti-\nbility loci for different complex phenotypes. In the case\nof autism, two independent studies showed signiﬁcant\nmale-speciﬁc evidence for linkage (maximized LOD\nscore [MLS] 14.0) to chromosome 17q11, whereas anal-\nysis of the families with at least one affected female\nshowed little evidence for linkage in the same region\n(MLS 0.5) (Stone et al. 2004; Cantor et al. 2005).\nRecently, sex-speciﬁc heritability and genomewide link-\nages for a total of 17 quantitative traits associated with\ncommon human diseases, such as heart disease, hyper-\ntension, diabetes, asthma, and autoimmune disease,\nwere evaluated in the Hutterites (Weiss et al. 2006). Ten\nof these traits showed sex-speciﬁc evidence for linkage,\nwith two genomewide-signiﬁcant linkages detected in\nthe male-only subset and two genomewide-signiﬁcant\nlinkages detected in the female-only subset. Interestingly,\nof the 62 regions that showed suggestive evidence for\nlinkage in either the male-only, female-only, or complete\ndata set, only 4 regions reached suggestive signiﬁcance\nin both the sex-stratiﬁed and complete data set, whereas\nnone of the male-speciﬁc or female-speciﬁc signals over-\nlapped. Notably, none of the sex-speciﬁc signals that\nshowed genomewide signiﬁcance reached suggestive sig-\nniﬁcance in the complete data set. In light of these re-\nports, as well as the presence of sex-speciﬁc loci observed\nin this study, it seems crucial to explore sex-speciﬁc ge-\nnetic effects when conducting genomewide screens, es-\npecially in those disorders where sex differences in risk\nare observed.",
         "The sex-specific genetic architecture of quantitative traits in humans;Quantitative genome scan and Ordered-Subsets Analysis of autism endophenotypes support language QTLs;Replication of autism linkage: fine-mapping peak at 17q21.;Variation in ITGB3 has sex-specific associations with plasma lipoprotein(a) and whole blood serotonin levels in a population-based sample;Genomewide significant linkage to stuttering on chromosome 12.;Evidence for sex-specific risk alleles in autism spectrum disorder.;Ignoring linkage disequilibrium among tightly linked markers induces false-positive evidence of linkage for affected sib pair analysis.;Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays;Guidelines for genotyping in genomewide linkage studies: single-nucleotide-polymorphism maps versus microsatellite maps.;Molecular genetics of autism spectrum disorder;Evidence for a Major Gene Influence on Persistent Developmental Stuttering;Whole-genome scan, in a complex disease, using 11,245 single-nucleotide polymorphisms: comparison with microsatellites.;Examination of Potential Overlap in Autism and Language Loci on Chromosomes 2, 7, and 13 in Two Independent Samples Ascertained for Specific Language Impairment;Results of a genome‐wide linkage scan for stuttering;Genome‐wide linkage survey for genetic loci that influence the development of depressive disorders in families with recurrent, early‐onset, major depression;Finding susceptibility genes for developmental disorders of speech: the long and winding road.;Sex stratification of an inflammatory bowel disease genome search shows male-specific linkage to the HLA region of chromosome 6;Probability of detection of genotyping errors and mutations as inheritance inconsistencies in nuclear-family data.;Enhanced Pedigree Error Detection;Genomewide scans of complex human diseases: true linkage is hard to find.;Phenotypic homogeneity provides increased support for linkage on chromosome 2 in autistic disorder.;A forkhead-domain gene is mutated in a severe speech and language disorder;Evidence for a susceptibility gene for autism on chromosome 2 and for genetic heterogeneity.;Improved inference of relationship for pairs of individuals.;A Study of the Genetic and Environmental Etiology of Stuttering in a Selected Twin Sample;Chromosome 7q: where autism meets language disorder?;Erratum to “MSX1 mutation is associated with orofacial clefting and tooth agenesis in humans”;A Unified Approach to Adjusting Association Tests for Population Admixture with Arbitrary Pedigree Structure and Arbitrary Missing Marker Information;Early childhood stuttering I: persistency and recovery rates.;PedCheck: a program for identification of genotype incompatibilities in linkage analysis.;Allele-sharing models: LOD scores and accurate linkage tests.;The use of a genetic map of biallelic markers in linkage studies;The genetic basis of persistence and recovery in stuttering.;Genetics of stuttering: a critical review.;Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results;Genetic aspects of early childhood stuttering.;Early Childhood Stuttering.;Genetic factors in stuttering confirmed.;Concordance for stuttering in monozygotic and dizygotic twin pairs.;Familial stuttering patterns are not related to one measure of severity.;Nature and treatment of stuttering.;Merlin—rapid analysis of dense genetic maps using sparse gene flow trees;Segregation analyses of stuttering;Stuttering as a genetic disorder;A handbook on stuttering",
         "New complexities in the genetics of stuttering: significant sex-specific linkage signals.",
         "None"
        ],
        [
         "41",
         "001f28be8f0482a67e69907068a20f9960642a8c",
         "The ability to switch between multiple tasks is central to flexible behavior. Although switching between tasks is readily accomplished, a well established consequence of task switching (TS) is behavioral slowing. The source of this switch cost and the contribution of cognitive control to its resolution remain highly controversial. Here, we tested whether proactive interference arising from memory places fundamental constraints on flexible performance, and whether prefrontal control processes contribute to overcoming these constraints. Event-related functional MRI indexed neural responses during TS. The contributions of cognitive control and interference were made theoretically explicit in a computational model of task performance. Model estimates of two levels of proactive interference, \"conceptual conflict\" and \"response conflict,\" produced distinct preparation-related profiles. Left ventrolateral prefrontal cortical activation paralleled model estimates of conceptual conflict, dissociating from that in left inferior parietal cortex, which paralleled model estimates of response conflict. These computationally informed neural measures specify retrieved conceptual representations as a source of conflict during TS and suggest that left ventrolateral prefrontal cortex resolves this conflict to facilitate flexible performance.",
         "David Badre,A. Wagner",
         "David Badre,A. Wagner",
         "https://europepmc.org/articles/pmc1459038?pdf=render",
         "The ability to switch between multiple tasks is central to ﬂexible\nbehavior. Although switching between tasks is readily accom-\nplished, a well established consequence of task switching (TS) is\nbehavioral slowing. The source of this switch cost and the contri-\nbution of cognitive control to its resolution remain highly contro-\nversial. Here, we tested whether proactive interference arising\nfrom memory places fundamental constraints on ﬂexible perfor-\nmance, and whether prefrontal control processes contribute to\novercoming these constraints. Event-related functional MRI in-\ndexed neural responses during TS. The contributions of cognitive\ncontrol and interference were made theoretically explicit in a\ncomputational model of task performance. Model estimates of two\nlevels of proactive interference, ‘‘conceptual conﬂict’’ and ‘‘re-\nsponse conﬂict,’’ produced distinct preparation-related proﬁles.\nLeft ventrolateral prefrontal cortical activation paralleled model\nestimates of conceptual conﬂict, dissociating from that in left\ninferior parietal cortex, which paralleled model estimates of re-\nsponse conﬂict. These computationally informed neural measures\nspecify retrieved conceptual representations as a source of conﬂict\nduring TS and suggest that left ventrolateral prefrontal cortex\nresolves this conﬂict to facilitate ﬂexible performance.",
         "",
         "do not await our disposition but must be addressed immedi-\nately. The ability to reconfigure our cognitive system to meet\nshifting task demands is evident and remarkable. A fundamental\nproblem in the study of cognitive control is specification of the\npsychological and neural processes by which we achieve this flex-\nibility and successfully switch tasks. Task switching (TS) is studied\nby comparing episodes in which subjects switch between two simple\ntasks to those in which they repeat the same task (1–5). In such\ncomparisons, TS incurs slowing in response time (RT), termed a\nswitch cost. As a window onto the mechanisms of flexible perfor-\nmance, considerable attention has focused on characterizing the\nsource of the switch cost, although extant data have generated\ncontroversy rather than resolution (2, 4).\nTwo classes of theory have framed the debate over TS costs.\nReconfiguration theories posit time-consuming intentional con-\ntrol processes that initiate reconfiguration of the task set inde-\npendent of the presentation of a target stimulus (6). From this\nperspective, the switch cost reflects the time consumed by these\ncontrol processes, and their progress during a preparation\ninterval yields preparation-related reductions in the switch cost.\nAlternatively, interference theories propose that switch costs are\nsubstantially or wholly attributable to conflict arising from\nmemory due to the recent performance of a different task (2, 7,\n8). From one such perspective, performance of a given task\nprimes associations among available cues and task representa-\ntions (2, 9, 10). Subsequent encounter with these cues in the\ncontext of a new task results in facilitated retrieval of the primed\nbut irrelevant information. This proactive interference compro-\nmises retrieval of relevant information.\nConsideration of the relation between TS and the neural mech-\nanisms of retrieval and interference resolution in memory may bear\non these theories, because a central component of both the recon-\nfiguration and interference accounts is the activation of a task set\nfrom memory (2, 3, 9, 11). Outside the context of TS, ventrolateral\nprefrontal cortex (VLPFC) has been associated with the retrieval\nand selection of task-relevant representations (12–15). In particu-\nlar, left mid-VLPFC (BA 45; inferior frontal gyrus pars triangu-\nlaris) has been associated with resolution of proactive interference\n(15, 16) and with overcoming interference during semantic and\nepisodic retrieval (13, 14, 17). Within the context of TS, simple\ncomparisons of switch vs. repeat conditions have revealed a fronto-\nparietal network including VLPFC, in addition to supplementary\nmotor area (SMA) and inferiorsuperior parietal cortices (18–25).§\nImportantly, the switch-related activation in VLPFC has been\nmodulated in experiments that test both reconfiguration and in-\nterference theories of TS (20, 26, 27). We posit that these initial\nfindings, although potentially contradictory, when considered in\nthe context of the broader literature on left VLPFC function, motivate\nthe hypothesis that VLPFC is engaged to overcome interference\nbetween competing mnemonic representations retrieved during TS.\nHere, we sought to test this hypothesis directly. To provide an\nexplicit theoretical context, a computational model, termed control\nof associative memory during TS (CAM-TS), was evaluated based\non behavioral data (Experiment 1). Indices of two forms of\nproactive interference, among concepts and responses, were de-\nrived from the model, providing quantitative predictions for func-\ntional MRI (fMRI) measures (Experiment 2). In Experiment 2, we\nused fMRI to assess the changes in neural switch effects due to\npreparation time. Anticipating the results, this manipulation dis-\nsociated the pattern of activation in mid-VLPFC from other\nswitch-related regions and related this activation directly to the\nquantitative predictions of conceptual conflict from CAM-TS.\nResults\nExperiment 1: Preparation Effects on Behavioral Switch Costs. Exper-\niment 1 characterized the change in RT switch cost due to prepa-\nration time, independent of transient decay effects (7, 28), thus\nproviding data on which to evaluate CAM-TS (Fig. 1A). On each\ntrial, subjects were cued as to which task (vowelconsonant or\noddeven judgment) was to be performed with the next required\ntarget (a letterdigit pair). Half of the trials required a TS, and half\nrequired a repeat. Preparedness was manipulated by varying the\n1.5, P  0.13]. Moreover, RCI did not significantly impact the\nswitch cost, when collapsed across CSI [F(3,69)  1.4, P  0.24].\nHence, the observed changes in RT slowing due to CSI primarily\nreflect preparation-related declines, rather than passive decay. These\nbehavioral results informed evaluation of the CAM-TS model.\nCAM-TS. Model overview. CAM-TS is designed to model TS during\nthe explicit cueing task based on associative memory theories of TS,\nsuch as task-set priming (2) (for additional model details, see\nsupporting information, which is published on the PNAS web site).\nAs such, other factors, such as short-term transient carryover or a\ntime-consuming reconfiguration process, are not directly modeled.\nExcluding short-term carryover from the model should not be\ninterpreted as a strong theoretical position but rather reflects the\ngoal of restricting the possible factors that contribute to model\npredictions. The critical assumptions captured by CAM-TS are (i)\nexperience strengthens associations among representations coac-\ntive during performance of a given task, and (ii) after a TS, these\nassociations facilitate retrieval of competing task-irrelevant infor-\nmation to produce interference and a behavioral switch cost. It is\na further goal of CAM-TS to be explicit about the contribution of\ncontrol in resolving interference arising during TS.\nCAM-TS consists of three layers, associated through reciprocal\nconnections, representing the task (letter and number), concept\n(odd, even, vowel, and consonant), and response (left and right)\nalternatives in the explicit cueing paradigm (Fig. 2 A). Switch costs\nemerge from two features of the model. (i) Connections between\nlayers enable activated response and concept units to elicit activa-\ntion of task-irrelevant units that compete with the task-relevant\nunits. (ii) After each trial, the baseline connection weights between\nthe task and the concept and response layers change based on a\nsimple learning rule that strengthens associations among mutually\nactive nodes (7). Thus, on switch trials, irrelevant associations are\nprepotent relative to the task-relevant associations. This ‘‘priming’’\nresults in greater memory-induced conflict on switch relative to\nrepeat trials. Importantly, preparatory task control is modulated\nthrough a gain term (30) that up-regulates the influence of the task\nlayer on the concept layer during the preparatory interval of a\nswitch trial. Because of this bias, relevant concept units come to\ndominate the concept layer before target presentation. This acts\nprotectively to overcome switch-related conflict arising at target\ncue-to-stimulus interval (CSI) from 250 to 1,150 ms. Time from the\nprevious response until cue presentation (RCI) also varied from 50\nto 950 ms, decoupling decay from preparation time. Response\nrepetition (RR) interference was manipulated across all CSIRCI\ncombinations, as repetition of the same manual response [response\nsame (RS)] across a TS increases RT switch costs relative to a\ndifferent response [response different (RD)] (6, 29).\nTS resulted in RT slowing [Fig. 2; F(1,23)  30.6, P  0.0001],\nwith the switch cost being greater on RS than RD trials [TS  RR:\nF(1,23)  29.9, P  0.0001]. Switch costs declined with CSI [Fig. 2B;\nF(3,69)  5.4, P  0.005] and did not interact with RCI [F(9,207) \npresentation (e.g., ref. 31). Critically, we conceptualize this control\nprocess as directly analogous to that proposed for selection from\namong retrieved competitors during memory retrieval tasks and\nunder short-term proactive interference (13–16) (see supporting\ninformation for analysis).\nSimulated behavior and conflict estimates. Simulated TS effects from\nCAM-TS provided a strong fit to the behavioral results from\nExperiment 1 (Fig. 2 B and C), with the simulated CSI-dependent\ndecline in the RT switch cost closely paralleling the empirically\nderived decline (R  0.96) and also the impact of RS-enhanced\nproactive interference across CSI (R values  0.97). Enhanced\nconflict due to RR occurs because the irrelevant associations\nformed on the previous trial are strongest for the previously\nexecuted response pathway. In other words, the response itself is a\nhighly effective cue for the irrelevant task set.\nCritically, although switching tasks may be accomplished without\nup-regulation of control, there is no CSI-dependent decline in\nsimulated switch costs when up-regulation is absent (Fig. 2B; model\nfit, R  0.49).\nAs a source of interference, conflict emerges in CAM-TS due to\nactivation being spread over more competing units in a layer. A\nquantifiable index of this conflict was computed as Hopfield energy\n(32, 33) (Fig. 2 A). The computational properties of Hopfield\nenergy correspond to features of conflict (34), in that energy\nincreases exponentially with the numbers of units over which\nactivation is spread and with the strength of their mutually com-\npetitive activation levels. Energy is distinct from the overall level of\nactivation, however, because energy would be highest when multi-\nple nodes are equally active (i.e., competing) but relatively low when\nonly one node is highly active (i.e., not competing).\nStrikingly, in CAM-TS, switch-derived conflict in the concept\nlayer during a TS declines across CSI (Fig. 2 A). By contrast, the\nresponse layer shows a generally increasing conflict function with\nincreasing CSI (Fig. 2 A). Further priming of the irrelevant pathway\ndue to RR also increases conflict values, but in this case, this conflict\nincrease is observed in both the conceptual and response layers.\nThese patterns emerge from the dynamics of CAM-TS. As noted\nabove, given bias and enough preparation time, the relevant con-\ncept units come to dominate the concept layer before target\npresentation. When only relevant representations dominate, there\nis less conflict, hence a lower conflict signal at longer CSIs. By\ncontrast, the more the two relevant conceptual representations\ncome to dominate, the more activation feeds forward to the\nresponse layer and equates activation among these nodes before the\npresentation of the target. Hence, conflict may get transiently\nhigher within the response layer with greater preparation before a\nswitch in the concept layer (although enhanced response conflict at\na long CSI never reaches the value of conflict in the concept layer\nat a short CSI). These simulated levels of conflict provide quanti-\ntative predictions to guide fMRI analysis.\nExperiment 2: VLPFC and Interference Resolution During TS. An\nindependent sample underwent fMRI while performing an\nadapted version of the Experiment 1 paradigm (Fig. 1B). Again, a\nCSI manipulation varied preparation time (250–1,150 ms), and RR\nfurther varied interference. As in Experiment 1, TS incurred a RT\nswitch cost [F(1,9)  82.8, P  0.0001; Fig. 3]. This cost declined\nlinearly with increasing CSI [t(9)  2.4, P  0.05], and a residual cost\n(105 ms) was evident after a 1,150-ms CSI [F(1,9)  31.4, P  0.0001].\nEffects of preparation time. According to CAM-TS, conflict in the\nconceptual layer is maximal at the shortest CSI. Switch vs. repeat\ndifferences at the 250-ms CSI were reliable in left mid-VLPFC\n(BA 45; 51 27 6), left posterior VLPFC (BA 4469; 45 6 27),\nleft dorsolateral prefrontal cortex (BA 46; 36 24 30) and along the\nsuperior bank of the intraparietal sulcus bilaterally, inclusive of superior\nparietal cortex (BA 740; 36 54 51; 30, 54 39; Fig. 4A).\nImportantly, in region of interest (ROI) analyses, the only\nregion to show a CSI-dependent decline in the neural switch\ncost (switch  repeat) was mid-VLPFC [Fig. 4 A; TS  CSI:\nF(3,27)  3.8, P  0.05]. The magnitude of the neural switch\ncost was greater at the shortest (250 ms) relative to the longest\nCSI [1,150 ms; F(1,9)  7.8, P  0.01]. The pattern of change\nacross CSI was reliably characterized by a monotonic decline\n[t(9)  2.5, P  0.05]. In further support of our prediction that\nthis region would be sensitive to the level of interference from\nretrieved conceptual representations, the neural switch cost in\nleft mid-VLPFC was reliably characterized by the CSI-\ndependent decline in conf lict computed from the concept\nlayer of CAM-TS [t(9)  2.7, P  0.05; Fig. 4 A].\nCaudal to mid-VLPFC, the posterior VLPFC region show-\ning a neural switch cost, approximates what has been termed\nthe inferior frontal junction (35), a region that, together with\nmid-VLPFC, has been consistently implicated in TS. However,\nthe magnitude of the neural switch cost in posterior VLPFC\nacross CSI was poorly fit by a monotonically declining function\n[t(9)  0.12, P  0.9; Fig. 4 A], because the switch cost did not\ndiffer between the shortest and longest CSIs [F  1.0]. This\ninsensitivity to CSI reliably dissociated posterior VLPFC from\nmid-VLPFC [t(9)  2.6, P  0.05].\nVoxel-wise analysis of switch vs. repeat trials collapsed across CSI\nfurther characterized the neural correlates of TS (Fig. 4B). In PFC,\ngreater activation on switch trials was observed in mid-VLPFC\ndorsolateral prefrontal cortex (BA 4546; 54 18 24; 54 33 18;\n48 39 18) and posterior VLPFC (BA 44; 45 9 27), as well as\nin SMA (BA 8; 0 18 48). Neural switch costs were also evident in\nleft inferior (BA 3940; 33 63 42; 51 33 48) and bilateral\nsuperior parietal cortex (BA 7; 27 66 57; 16 63 60).\nAgain, the only region to show a CSI-dependent decline in the\nneural switch cost was the anterior and dorsal extent of mid-VLPFC\n(54 33 18; Fig. 4B). ROI analysis revealed (i) that the neural\nswitch cost was greater at the shortest relative to the longest CSI\n[F(1,9)  4.0, P  0.05], (ii) that the switch cost declined mono-\ntonically [t(9)  2.7, P  0.05], and (iii) that this decrease was\nreliably characterized by the CAM-TS index of conceptual conflict\n[t(9)  2.8, P  0.05].\nIn contrast to mid-VLPFC, the neural switch cost in inferior\nparietal cortex (51 33 48) tended to increase with longer CSIs,\nalthough not reliably [t(9)  1.0]. Qualitatively, this pattern ap-\npeared to correspond to the level of conflict arising in the Response\nlayer of CAM-TS, in that both showed a modest ramping response\n(Fig. 4B), although the fit was unreliable when using the standard\ncalculation of conflict [t(9)  1.4, P  0.19]. However, motivated\nby the recent hypothesis that inferior parietal neurons may accu-\nmulate evidence en route to response selection (36), we recom-\nputed the level of conflict in CAM-TS using accumulated activation\nin the response layer (for details, see supporting information). This\nindex of accumulated conflict fit the inferior parietal response\n[t(9)  2.9, P  0.05; Fig. 4B].\nfMRI results from Experiment 2. (A) Surface rendering of switch  repeat at the shortest CSI (250 ms). Plotted are neural switch costs across CSI from ROI analyses\nFig. 4.\nof left mid-VLPFC (51 27 6) and posterior VLPFC (45 6 27). (B) The contrast of switch vs. repeat collapsed across CSI. Plotted are changes in neural switch cost from\nROIs in left mid-VLPFC (54 33 18), inferior parietal cortex (51 33 48), and SMA (0 18 48). Also depicted are linearly scaled conﬂict signals (dashed lines) from\nconceptual (red) and response [blue (standard) and green (cumulative)] layers of CAM-TS. (C) Bar graphs depict enhancement of switch costs during RS vs. RD trials split\nby switch (red bar) and repeat (green bar) from ROIs in left mid-VLPFC (54 33 18), inferior parietal cortex (51 33 48), and SMA (0 18 48).\nOf central importance, the pattern over CSI in inferior parietal\ncortex dissociated from the decreasing response in left mid-VLPFC\n[t(9)  3.0, P  0.05]. Collectively, these data provide strong\nevidence that mid-VLPFC is recruited in the face of switch-related\ninterference, with CAM-TS suggesting that mid-VLPFC may re-\nsolve interference at the conceptual level.\nEffectsofRR.Proactive interference was also modulated through the\nmanipulation of RR. According to CAM-TS, RR induces increased\nconflict in multiple layers. Behaviorally, RR enhanced the RT\nswitch cost during fMRI [Fig. 3A; TS  RR: F(1,9)  58.5, P \n0.0001]. Neurally, a TS  RR interaction was marginally evident in\nSMA [Fig. 4C; F(1,9)  4.3, P  0.06], reflecting the fact that the\neffect of TS in SMA was entirely accounted for by RR-induced\ninterference: Switch was greater than Repeat on RS [F(1,9)  9.0,\nP  0.05] but not RD trials (F  1).\nIndicative of RR interference and consistent with the conceptual\nconflict signal in CAM-TS, planned contrasts also revealed that\nactivation in mid-VLPFC was greater for switch-RS than\nswitch-RD trials [F(1,9)  5.2, P  0.05]. However, in contrast to\nSMA, neural switch costs on RD trials were also evident in\nmid-VLPFC (Fig. 4C), activation in mid-VLPFC tended to dissociate\nfrom that in SMA, as evidenced by a marginal region [mid-VLPFC\nSMA]  RR  TS interaction [F(1,9)  4.3, P  0.06]. Finally,\nqualitatively similar to mid-VLPFC, inferior parietal cortex showed a\nmarginal switch-RS vs. switch-RD difference [F(1,9)  4.7, P  0.06].\nConclusion\nTS can be understood as an act of memory. Thus, the behavioral\nand psychological consequences of TS can be understood in terms\nof the structures, processes, and failures of memory. This propo-\nsition entails that the control processes contributing to TS are\nindistinguishable from the control processes engaged to overcome\ninterference arising during other acts of memory. Accordingly, the\nneural mechanisms supporting interference resolution during\nmemory retrieval, such as those subserved by mid-VLPFC, are\ncentral for successfully overcoming interference during a TS.\nOur results strongly support these conclusions. In particular, we\ndemonstrate that a simple connectionist model deriving its RT\nswitch cost entirely from experience-dependent changes in its\nassociative structure accounts for switch costs through proactive\ninterference at both the conceptual and response levels and ac-\ncomplishes preparation-related declines in the switch cost through\nprospective, endogenous control. Strikingly, the model’s signature\nof declining conflict among activated concepts after preparation\nwas characteristic of activity in left mid-VLPFC and dissociated this\nregion from other regions active during TS. Importantly, this\nmonotonic decline in mid-VLPFC did not simply track changes in\nRT, because both the form of the decline and the lack of a residual\ncost in mid-VLPFC differentiate these response profiles. Moreover,\ninferior parietal cortex was associated with a ramping response over\nCSI, a pattern qualitatively consistent with the increased conflict in\nthe response layer of CAM-TS at longer CSIs. A role for posterior\nparietal cortex in processing response alternatives finds additional\nsupport in the broader cognitive control literature (37, 38). A temporal\nshift from mid-VLPFC (conceptual) to inferior parietal (response)\ncortex may also parallel electroencephalogram evidence that TS is\naccompanied by an early frontal component (300–500 ms) followed\nby a subsequent parietal component (500–1,000 ms) (39–41).\nIt is important to note that, although the behavioral and neuro-\nimaging results confirm the predictions derived from our memory-\nbased model of TS, this does not preclude the possibility that other\nmodels based on different assumptions could make similar quan-\ntitative predictions. However, such a model must account for\nseveral empirical findings reported here that stand independent of\nthe validity of our specific theoretical framework: (i) the CSI-based\ndecline and the RR effect in mid-VLPFC, (ii) the dissociation of\nmid-VLPFC from sustained effects in posterior VLPFC and SMA,\nand (iii) the dissociation of mid-VLPFC from the ramping function\nin posterior parietal cortex.\nThis caveat notwithstanding, the present framework and sup-\nporting results are broadly appealing, because they provide a\npotential theoretical reconciliation to the long-standing debate over\nthe origins of TS costs and the contribution of cognitive control to\nprospectively switching tasks. CAM-TS is unique from other prom-\ninent frameworks (7, 42) in its exclusive reliance on learning-based\nchanges as a source of its switch cost, its locus of conflict in the\nconceptual and response layers as opposed to the control layer, and\nthe intervention of control in reducing costs. CAM-TS assumes that\na considerable portion of switch costs is attributable to proactive\ninterference from retrieved irrelevant information. This is consis-\ntent with interference accounts of the switch cost and accounts that\nminimize the direct contribution of control to the length of the\nswitch cost. However, our data also indicate that prospective\nengagement of control processes that regulate memory may help\nresolve interference, contributing to preparation-related declines.\nIn this respect, CAM-TS shares an active and strategic control\nprocess with reconfiguration models.\nA distinguishing feature of this synthesis is its potentially parsi-\nmonious account of the neuropsychological data on TS. Again,\ncontrol in CAM-TS reduces interference and facilitates retrieval\nrather than enacts a TS. Consequently, the model predicts that in\nthis explicit cueing variant, TS can proceed without endogenous\ncontrol, although under such circumstances performance is more\nvulnerable to interference (Fig. 2B). Lesion data indicate that\ndamage inclusive of left mid-VLPFC results in an enhanced switch\ncost but neither prohibits the ability to switch tasks nor results in\nperseveration (43). CAM-TS accurately predicts this pattern of en-\nhanced cost but not perseveration after mid-VLPFC insult, a point that\nmay favorably distinguish the present model from a related connec-\ntionist framework designed to model asymmetric costs (7) or other\nelegant mathematical models (42) that suggest minimal control.\nFinally, the present work does not preclude the contributions of\nadditional control processes or interference effects during TS. A\nprocess of goal setting is likely required in many TS contexts (3) and\nmay be associated with distinct regions of PFC, such as frontal polar\ncortex (22). Furthermore, the unique effects in SMA and posterior\nVLPFC, not directly accounted for by CAM-TS, point to the\nmulticomponent nature of TS. By contrast, it is of some interest that\nthe anterior cingulate cortex was not sensitive to conceptual or re-\nsponse-level conflict in this task, although this is not necessarily incon-\nsistent with the broader literature on TS (44) or proactive interference\n(45). Critically, however, the present computational framework and\nneuroimaging evidence argue that one critical component for flexible\nbehavior is interference resolution by mid-VLPFC.\nMethods\nSubjects. Twenty-four right-handed native English speakers (16\nfemale; ages 18–25 yrs) were remunerated $10 per hour for\nparticipation in Experiment 1, and 13 right-handed native English\nspeakers (8 female; ages 18–25 yrs) were remunerated $50 for\nparticipation in Experiment 2. Data from three subjects from\nExperiment 2 (one female) were excluded before fMRI analysis\nbecause of high nonresponse rates due to a difficulty with respond-\ning before the response deadline. Informed consent was obtained\nas approved by the human subjects committees at Massachusetts\nGeneral Hospital, Massachusetts Institute of Technology, and\nStanford University.\nExperiment 1: Design and Procedure. Stimuli consisted of number–\nletter pairs (e.g., ‘‘2b’’) presented centrally in 32-point Monaco font.\nPairs were constructed from a set of 10 letters, five consonants (‘‘p,’’\n‘‘f,’’ ‘‘n,’’ ‘‘k,’’ ‘‘s’’), and five vowels (‘‘a,’’ ‘‘e,’’ ‘‘i,’’ ‘‘o,’’ ‘‘u’’), and a\nset of 10 digits, five odd numbers (‘‘1,’’ ‘‘3,’’ ‘‘5,’’ ‘‘7,’’ ‘‘9’’), and five\neven numbers (‘‘0,’’ ‘‘2,’’ ‘‘4,’’ ‘‘6,’’ ‘‘8’’). The number and letter\npositions were counterbalanced across pairs (e.g., ‘‘2b’’ or ‘‘b2’’).\nSubjects performed one of two categorization tasks (Fig. 1 A)\nwith each stimulus pair. In the number task, subjects categorized the\nnumber as odd or even. In the letter task, subjects categorized the\nletter as vowel or consonant. Categorization decisions for both tasks\nwere reported by using one of the same two buttons (left or right)\nunder the right hand.\nDuring each block of the experiment, the number and letter tasks\nwere intermixed. An instruction cue (LETTER or NUMBER)\npreceded the onset of each target stimulus (Fig. 1) and signaled the\ntask to be performed for that target. The trial terminated once a\nresponse was made. The correct response could be the same as (RS)\nor different from (RD) the response emitted on the previous trial.\nThe RCI varied among four values (50, 226, 506, or 950 ms) that\nexpanded logarithmically. Likewise, the CSI varied among four\nvalues (50, 226, 506, or 950 ms, plus 200 ms for cue presentation).\nTo maximize our ability to detect switch-related declines (6), CSI\nwas blocked, and the order of blocks was fully counterbalanced\nbetween subjects.\nAfter four excluded warm-up trials, each CSI-defined block\nconsisted of 256 trials divided evenly among remaining experimen-\ntal conditions. In addition, trials were counterbalanced for (i) the\nmatch of the irrelevant flanker response to the correct response,\nand (ii) whether the correct target position switched from the\nprevious trial.\nAll behavioral testing was conducted on a Macintosh G4 com-\nputer in a darkened testing room at Stanford University. Before\ndata collection, subjects received extensive training. Subjects con-\ntinuously practiced one of the tasks (e.g., letter or number) alone\nand then practiced the other task (e.g., number or letter). Then\nsubjects practiced switching between the tasks in four short blocks\nat each CSI in experimental order.\nExperiment 2: Design and Behavioral Procedures. Stimuli for the\nfMRI experiment were identical to those used in Experiment 1. As\nin Experiment 1, subjects performed either the number or letter\ntask (Fig. 1B), although making responses with their left hand.\nBefore fMRI data collection, subjects received extensive training:\n(i) outside the scanner, subjects continuously performed the letter\ntask and then the number task, and (ii) subsequently, subjects\npracticed switching between the tasks.\nDuring fMRI scanning, the number and letter tasks were inter-\nmixed. An instruction cue preceded the onset of each stimulus (Fig.\n1B) and signaled the upcoming task. During the CSI within a trial\nand the intertrial interval (ITI) separating trials, a white fixation\ncross was presented centrally (as a preparatory warning, the fixation\ncross turned from white to red immediately before cuestimulus\npresentation). To allow for estimation and deconvolution of the\nhemodynamic response as a function of small changes in CSI\nduration, trials were grouped into pairs of two task events (T-1 and\nT-2), although to the subject, the experiment appeared as a\ncontinuous stream of task events. The onset of the first task event\n(T-1) in each pair followed a variable-duration null fixation period\n(2–16 s) that followed the prior pair. To permit event-related fMRI\nanalyses, T-1 always required performance of the same task that\nhad just been performed (task repeat) during the T-2 phase in the\nprior pair. Furthermore, the consonance of the T-1 manual re-\nsponse to the manual response emitted during the T-2 phase of the\nprior pair (RDRS) and the target and flanker relationship within\nT-1 were counterbalanced across experimental conditions at T-2.\nThe CSI duration for T-1 was fixed at 1,000 ms, and the ITI between\nT-1 and the T-2 cue was always 50 ms.\nThe critical experimental variables were manipulated during T-2.\nThe task for T-2 was either a repeat or switch from that during T-1.\nThe T-2 manual response (left or right) was either the same as (RS)\nor different from (RD) that required during T-1. Finally, the\nduration of the CSI (inclusive of 200-ms task cue) for the T-2 event\nvaried (250 s; 426, 706, or 1,150 ms). Target position was also\ncounterbalanced across T-1T-2 pairs. Collectively, this design\nallowed for analysis of fMRI signal differences at T-2 according to\nthe critical factor manipulations (repeat vs. switch, RS vs. RD, and\nCSI duration). For all events, a response deadline of 1,800 ms was\nimposed. For the imaging analysis, a pair was considered incorrect\nif a subject responded incorrectly or failed to respond before the\nresponse deadline on T-1 andor T-2 within the pair. Analysis of RT\nwas restricted to the trials included in the imaging analysis; error analysis\nwas based only on T-2 events and did not depend on T-1 accuracy.\nDuring fMRI scanning, subjects encountered 480 pairs of trials\nacross four scan epochs. Events were grouped into blocks of 30 pairs\non the basis of T-2 CSI duration. Subjects encountered a block of\neach CSI duration during each scan epoch. Response mappings and\ncondition order were counterbalanced across subjects.\nfMRI Acquisition and Analysis Procedures. Whole-brain imaging was\nperformed on a 3-T Siemens (Iselin, NJ) Trio MRI system.\nFunctional data were acquired by using a gradient-echo echo-\nplanar pulse sequence [repetition time (TR)  2 s, echo time (TE)\n 30 ms, 21 axial slices, 3.125  3.125  5 mm, 1-mm interslice gap,\nfour runs  756 volume acquisitions]. High-resolution T1-weighted\n[magnetization prepared rapid gradient echo (MP-RAGE)] struc-\ntural images were collected for anatomical visualization, during\nwhich subjects practiced the tasks. Head motion was restricted by\nusing firm padding surrounding the head. Projected visual stimuli\nwere viewed through a mirror attached to the standard head coil.\nData were preprocessed by using SPM99 (Wellcome Department\nof Cognitive Neurology, London). Functional images were cor-\nrected for differences in slice acquisition timing, followed by motion\ncorrection (using sinc interpolation). Structural and functional data\nwere spatially normalized to a template based on the Montreal\nNeurological Institute stereotactic space (46) using a 12-parameter\naffine transformation along with a nonlinear transformation using\ncosine basis functions. Images were resampled to 3-mm cubic voxels\nand spatially smoothed with an 8-mm full-width-at-half-maximum\nisotropic Gaussian kernel.\nStatistical models were constructed by using SPM99 under the\nassumptions of the general linear model. The units of analysis were\nthe task pairs described above (Fig. 1B). Because the T-1 phase of\neach pair was counterbalanced across the T-2 conditions, the\nunique contribution to the overall variance due to manipulations of\nT-2 could be estimated. Epochs of 6 s, beginning at the onset of the\nT-1 cue, were used to model each pair; the 6-s epochs were\nconvolved with a canonical hemodynamic response function. Cor-\nrect and incorrect trials were modeled separately, and subsequent\ncontrasts were restricted to correct trials. Effects were estimated by\nusing a subject-specific fixed-effects model, with session-specific\neffects and low-frequency signal components treated as confounds.\nLinear contrasts were used to obtain subject-specific estimates for\neach effect. These estimates were entered into a second-level\nanalysis treating subjects as a random effect, using a one-sample t\ntest against a contrast value of zero at each voxel. Effects in the\nwhole-brain analysis were considered reliable to the extent that they\nconsisted of at least five contiguous voxels that exceeded an\nuncorrected threshold of P  0.001. For the purpose of additional\nanatomical precision, group contrasts were also rendered on a\nMontreal Neurological Institute canonical brain that underwent\ncortical ‘‘inflation’’ using FREESURFER (CorTechs, Centreville, VA)\n(47, 48) and the SPM SURFREND toolbox (written by I. Kahn,\nHarvard University, Cambridge, MA).\nThe voxel-based contrasts were supplemented with ROI analyses\nto further characterize the effects of CSI and interference (RS vs.\nRD) in a priori expected regions, including mid-VLPFC. The ROI\nanalyses were performed by using a toolbox for use with SPM99\n(written by R. Poldrack, University of California, Los Angeles).\nROIs included all significant (uncorrected P  0.001) voxels within\na 6-mm radius of the chosen maximum. Deconvolution allowed\nassessment of the signal change associated with each condition.\nROI analyses were performed on measures of integrated percent\nsignal change (peak  2 TRs) subjected to repeated-measures\nANOVA. Finally, assessment of monotonic decay components over\nCSI in ROIs was performed by estimating a logarithmic decay in the\nswitch vs. repeat difference for each subject individually and then\nentering these estimates into a second-level analysis that treated\nsubject as a random variable, using a one-sample t test against a null\neffect value of 0. Decay in RT cost was assessed similarly, except\nthat a linear rather than a logarithmic model was used, after residual\nanalysis revealed that the distribution of errors deviated from\nnormal. Likewise, convergence between the conflict output of the\ncomputational model and fMRI signal was evaluated within sub-\nject, based on a linear predictive relationship.\n4. Monsell, S. (2003) Trends Cognit. Sci. 7, 134–140.\n5. Logan, G. D. (2003) Curr. Direct. Psychol. Sci. 12, 45–48.\n6. Rogers, R. D. & Monsell, S. (1995) J. Exp. Psychol. Gen. 124, 207–231.\n7. Gilbert, S. J. & Shallice, T. (2002) Cognit. Psychol. 44, 297–337.\n8. Yeung, N. & Monsell, S. (2003) J. Exp. Psychol. Hum. Percept. Perform. 29, 919–936.\n9. Waszak, F., Hommel, B. & Allport, A. (2003) Cognit. Psychol. 46, 361–413.\n10. Sohn, M.-H. & Anderson, J. R. (2003) Mem. Cognit. 31, 775–780.\n11. Mayr, U. & Kliegl, R. (2000) J. Exp. Psychol. Learn Mem. Cognit. 26, 1124–1140.\n12. Petersen, S. E., Fox, P. T., Posner, M. I., Mintun, M. & Raichle, M. E. (1988) Nature 331, 585–589.\n13. Thompson-Schill, S. L., D’Esposito, M., Aguirre, G. K. & Farah, M. J. (1997) Proc. Natl.\n& Moscovitch, M. (MIT Press, Cambridge, MA), Vol. XV, pp. 421–452.\n29. Meiran, N., Chorev, Z. & Sapir, A. (2000) Cognit. Psychol. 41, 211–253.\n30. Cohen, J. D. & Servan-Schreiber, D. (1992) Psychol. Rev. 99, 45–77.\n31. Ruge, H., Brass, M., Koch, I., Rubin, O., Meiran, N. & von Cramon, D. Y. (2005)\n33. Hopfield, J. J. (1982) Proc. Natl. Acad. Sci. USA 79, 2554–2558.\n34. Berlyne, D. E. (1957) Psychol. Rev. 64, 329–339.\n35. Brass, M., Derrfuss, J., Forstmann, B. & von Cramon, D. Y. (2005) Trends Cognit. Sci. 9, 314–316.\n36. Shadlen, M. N. & Newsome, W. T. (2001) J. Neurophysiol. 86, 1916–1936.\n37. Bunge, S. A., Hazeltine, E., Scanlon, M. D., Rosen, A. C. & Gabrieli, J. D. (2002)\n19. Brass, M. & von Cramon, D. Y. (2002) Cereb. Cortex 12, 908–914.\n20. Brass, M. & von Cramon, D. Y. (2004) J. Neurosci. 24, 8847–8852.\n21. Dreher, J.-C. & Grafman, J. (2003) Cereb. Cortex 13, 329–339.\n22. Braver, T. S., Reynolds, J. R. & Donaldson, D. I. (2003) Neuron 39, 713–726.\n23. Reynolds, J. R., Donaldson, D. I., Wagner, A. D. & Braver, T. S. (2004) NeuroImage 21, 1472–1483.\n24. Rushworth, M. F., Hadland, K. A., Paus, T. & Sipila, P. K. (2002) J. Neurophysiol. 87, 2577–2592.\n25. Crone, E. A., Wendelken, C., Donohue, S. E. & Bunge, S. A. (2006) Cereb. Cortex 16, 475–486.\n26. Luks, T. L., Simpson, G. V., Feiwell, R. J. & Miller, W. L. (2002) NeuroImage 17, 792–802.",
         "",
         "",
         "downloading",
         "The ability to switch between multiple tasks is central to ﬂexible\nbehavior. Although switching between tasks is readily accom-\nplished, a well established consequence of task switching (TS) is\nbehavioral slowing. The source of this switch cost and the contri-\nbution of cognitive control to its resolution remain highly contro-\nversial. Here, we tested whether proactive interference arising\nfrom memory places fundamental constraints on ﬂexible perfor-\nmance, and whether prefrontal control processes contribute to\novercoming these constraints. Event-related functional MRI in-\ndexed neural responses during TS. The contributions of cognitive\ncontrol and interference were made theoretically explicit in a\ncomputational model of task performance. Model estimates of two\nlevels of proactive interference, ‘‘conceptual conﬂict’’ and ‘‘re-\nsponse conﬂict,’’ produced distinct preparation-related proﬁles.\nLeft ventrolateral prefrontal cortical activation paralleled model\nestimates of conceptual conﬂict, dissociating from that in left\ninferior parietal cortex, which paralleled model estimates of re-\nsponse conﬂict. These computationally informed neural measures\nspecify retrieved conceptual representations as a source of conﬂict\nduring TS and suggest that left ventrolateral prefrontal cortex\nresolves this conﬂict to facilitate ﬂexible performance.\ndo not await our disposition but must be addressed immedi-\nately. The ability to reconfigure our cognitive system to meet\nshifting task demands is evident and remarkable. A fundamental\nproblem in the study of cognitive control is specification of the\npsychological and neural processes by which we achieve this flex-\nibility and successfully switch tasks. Task switching (TS) is studied\nby comparing episodes in which subjects switch between two simple\ntasks to those in which they repeat the same task (1–5). In such\ncomparisons, TS incurs slowing in response time (RT), termed a\nswitch cost. As a window onto the mechanisms of flexible perfor-\nmance, considerable attention has focused on characterizing the\nsource of the switch cost, although extant data have generated\ncontroversy rather than resolution (2, 4).\nTwo classes of theory have framed the debate over TS costs.\nReconfiguration theories posit time-consuming intentional con-\ntrol processes that initiate reconfiguration of the task set inde-\npendent of the presentation of a target stimulus (6). From this\nperspective, the switch cost reflects the time consumed by these\ncontrol processes, and their progress during a preparation\ninterval yields preparation-related reductions in the switch cost.\nAlternatively, interference theories propose that switch costs are\nsubstantially or wholly attributable to conflict arising from\nmemory due to the recent performance of a different task (2, 7,\n8). From one such perspective, performance of a given task\nprimes associations among available cues and task representa-\ntions (2, 9, 10). Subsequent encounter with these cues in the\ncontext of a new task results in facilitated retrieval of the primed\nbut irrelevant information. This proactive interference compro-\nmises retrieval of relevant information.\nConsideration of the relation between TS and the neural mech-\nanisms of retrieval and interference resolution in memory may bear\non these theories, because a central component of both the recon-\nfiguration and interference accounts is the activation of a task set\nfrom memory (2, 3, 9, 11). Outside the context of TS, ventrolateral\nprefrontal cortex (VLPFC) has been associated with the retrieval\nand selection of task-relevant representations (12–15). In particu-\nlar, left mid-VLPFC (BA 45; inferior frontal gyrus pars triangu-\nlaris) has been associated with resolution of proactive interference\n(15, 16) and with overcoming interference during semantic and\nepisodic retrieval (13, 14, 17). Within the context of TS, simple\ncomparisons of switch vs. repeat conditions have revealed a fronto-\nparietal network including VLPFC, in addition to supplementary\nmotor area (SMA) and inferiorsuperior parietal cortices (18–25).§\nImportantly, the switch-related activation in VLPFC has been\nmodulated in experiments that test both reconfiguration and in-\nterference theories of TS (20, 26, 27). We posit that these initial\nfindings, although potentially contradictory, when considered in\nthe context of the broader literature on left VLPFC function, motivate\nthe hypothesis that VLPFC is engaged to overcome interference\nbetween competing mnemonic representations retrieved during TS.\nHere, we sought to test this hypothesis directly. To provide an\nexplicit theoretical context, a computational model, termed control\nof associative memory during TS (CAM-TS), was evaluated based\non behavioral data (Experiment 1). Indices of two forms of\nproactive interference, among concepts and responses, were de-\nrived from the model, providing quantitative predictions for func-\ntional MRI (fMRI) measures (Experiment 2). In Experiment 2, we\nused fMRI to assess the changes in neural switch effects due to\npreparation time. Anticipating the results, this manipulation dis-\nsociated the pattern of activation in mid-VLPFC from other\nswitch-related regions and related this activation directly to the\nquantitative predictions of conceptual conflict from CAM-TS.\nResults\nExperiment 1: Preparation Effects on Behavioral Switch Costs. Exper-\niment 1 characterized the change in RT switch cost due to prepa-\nration time, independent of transient decay effects (7, 28), thus\nproviding data on which to evaluate CAM-TS (Fig. 1A). On each\ntrial, subjects were cued as to which task (vowelconsonant or\noddeven judgment) was to be performed with the next required\ntarget (a letterdigit pair). Half of the trials required a TS, and half\nrequired a repeat. Preparedness was manipulated by varying the\n1.5, P  0.13]. Moreover, RCI did not significantly impact the\nswitch cost, when collapsed across CSI [F(3,69)  1.4, P  0.24].\nHence, the observed changes in RT slowing due to CSI primarily\nreflect preparation-related declines, rather than passive decay. These\nbehavioral results informed evaluation of the CAM-TS model.\nCAM-TS. Model overview. CAM-TS is designed to model TS during\nthe explicit cueing task based on associative memory theories of TS,\nsuch as task-set priming (2) (for additional model details, see\nsupporting information, which is published on the PNAS web site).\nAs such, other factors, such as short-term transient carryover or a\ntime-consuming reconfiguration process, are not directly modeled.\nExcluding short-term carryover from the model should not be\ninterpreted as a strong theoretical position but rather reflects the\ngoal of restricting the possible factors that contribute to model\npredictions. The critical assumptions captured by CAM-TS are (i)\nexperience strengthens associations among representations coac-\ntive during performance of a given task, and (ii) after a TS, these\nassociations facilitate retrieval of competing task-irrelevant infor-\nmation to produce interference and a behavioral switch cost. It is\na further goal of CAM-TS to be explicit about the contribution of\ncontrol in resolving interference arising during TS.\nCAM-TS consists of three layers, associated through reciprocal\nconnections, representing the task (letter and number), concept\n(odd, even, vowel, and consonant), and response (left and right)\nalternatives in the explicit cueing paradigm (Fig. 2 A). Switch costs\nemerge from two features of the model. (i) Connections between\nlayers enable activated response and concept units to elicit activa-\ntion of task-irrelevant units that compete with the task-relevant\nunits. (ii) After each trial, the baseline connection weights between\nthe task and the concept and response layers change based on a\nsimple learning rule that strengthens associations among mutually\nactive nodes (7). Thus, on switch trials, irrelevant associations are\nprepotent relative to the task-relevant associations. This ‘‘priming’’\nresults in greater memory-induced conflict on switch relative to\nrepeat trials. Importantly, preparatory task control is modulated\nthrough a gain term (30) that up-regulates the influence of the task\nlayer on the concept layer during the preparatory interval of a\nswitch trial. Because of this bias, relevant concept units come to\ndominate the concept layer before target presentation. This acts\nprotectively to overcome switch-related conflict arising at target\ncue-to-stimulus interval (CSI) from 250 to 1,150 ms. Time from the\nprevious response until cue presentation (RCI) also varied from 50\nto 950 ms, decoupling decay from preparation time. Response\nrepetition (RR) interference was manipulated across all CSIRCI\ncombinations, as repetition of the same manual response [response\nsame (RS)] across a TS increases RT switch costs relative to a\ndifferent response [response different (RD)] (6, 29).\nTS resulted in RT slowing [Fig. 2; F(1,23)  30.6, P  0.0001],\nwith the switch cost being greater on RS than RD trials [TS  RR:\nF(1,23)  29.9, P  0.0001]. Switch costs declined with CSI [Fig. 2B;\nF(3,69)  5.4, P  0.005] and did not interact with RCI [F(9,207) \npresentation (e.g., ref. 31). Critically, we conceptualize this control\nprocess as directly analogous to that proposed for selection from\namong retrieved competitors during memory retrieval tasks and\nunder short-term proactive interference (13–16) (see supporting\ninformation for analysis).\nSimulated behavior and conflict estimates. Simulated TS effects from\nCAM-TS provided a strong fit to the behavioral results from\nExperiment 1 (Fig. 2 B and C), with the simulated CSI-dependent\ndecline in the RT switch cost closely paralleling the empirically\nderived decline (R  0.96) and also the impact of RS-enhanced\nproactive interference across CSI (R values  0.97). Enhanced\nconflict due to RR occurs because the irrelevant associations\nformed on the previous trial are strongest for the previously\nexecuted response pathway. In other words, the response itself is a\nhighly effective cue for the irrelevant task set.\nCritically, although switching tasks may be accomplished without\nup-regulation of control, there is no CSI-dependent decline in\nsimulated switch costs when up-regulation is absent (Fig. 2B; model\nfit, R  0.49).\nAs a source of interference, conflict emerges in CAM-TS due to\nactivation being spread over more competing units in a layer. A\nquantifiable index of this conflict was computed as Hopfield energy\n(32, 33) (Fig. 2 A). The computational properties of Hopfield\nenergy correspond to features of conflict (34), in that energy\nincreases exponentially with the numbers of units over which\nactivation is spread and with the strength of their mutually com-\npetitive activation levels. Energy is distinct from the overall level of\nactivation, however, because energy would be highest when multi-\nple nodes are equally active (i.e., competing) but relatively low when\nonly one node is highly active (i.e., not competing).\nStrikingly, in CAM-TS, switch-derived conflict in the concept\nlayer during a TS declines across CSI (Fig. 2 A). By contrast, the\nresponse layer shows a generally increasing conflict function with\nincreasing CSI (Fig. 2 A). Further priming of the irrelevant pathway\ndue to RR also increases conflict values, but in this case, this conflict\nincrease is observed in both the conceptual and response layers.\nThese patterns emerge from the dynamics of CAM-TS. As noted\nabove, given bias and enough preparation time, the relevant con-\ncept units come to dominate the concept layer before target\npresentation. When only relevant representations dominate, there\nis less conflict, hence a lower conflict signal at longer CSIs. By\ncontrast, the more the two relevant conceptual representations\ncome to dominate, the more activation feeds forward to the\nresponse layer and equates activation among these nodes before the\npresentation of the target. Hence, conflict may get transiently\nhigher within the response layer with greater preparation before a\nswitch in the concept layer (although enhanced response conflict at\na long CSI never reaches the value of conflict in the concept layer\nat a short CSI). These simulated levels of conflict provide quanti-\ntative predictions to guide fMRI analysis.\nExperiment 2: VLPFC and Interference Resolution During TS. An\nindependent sample underwent fMRI while performing an\nadapted version of the Experiment 1 paradigm (Fig. 1B). Again, a\nCSI manipulation varied preparation time (250–1,150 ms), and RR\nfurther varied interference. As in Experiment 1, TS incurred a RT\nswitch cost [F(1,9)  82.8, P  0.0001; Fig. 3]. This cost declined\nlinearly with increasing CSI [t(9)  2.4, P  0.05], and a residual cost\n(105 ms) was evident after a 1,150-ms CSI [F(1,9)  31.4, P  0.0001].\nEffects of preparation time. According to CAM-TS, conflict in the\nconceptual layer is maximal at the shortest CSI. Switch vs. repeat\ndifferences at the 250-ms CSI were reliable in left mid-VLPFC\n(BA 45; 51 27 6), left posterior VLPFC (BA 4469; 45 6 27),\nleft dorsolateral prefrontal cortex (BA 46; 36 24 30) and along the\nsuperior bank of the intraparietal sulcus bilaterally, inclusive of superior\nparietal cortex (BA 740; 36 54 51; 30, 54 39; Fig. 4A).\nImportantly, in region of interest (ROI) analyses, the only\nregion to show a CSI-dependent decline in the neural switch\ncost (switch  repeat) was mid-VLPFC [Fig. 4 A; TS  CSI:\nF(3,27)  3.8, P  0.05]. The magnitude of the neural switch\ncost was greater at the shortest (250 ms) relative to the longest\nCSI [1,150 ms; F(1,9)  7.8, P  0.01]. The pattern of change\nacross CSI was reliably characterized by a monotonic decline\n[t(9)  2.5, P  0.05]. In further support of our prediction that\nthis region would be sensitive to the level of interference from\nretrieved conceptual representations, the neural switch cost in\nleft mid-VLPFC was reliably characterized by the CSI-\ndependent decline in conf lict computed from the concept\nlayer of CAM-TS [t(9)  2.7, P  0.05; Fig. 4 A].\nCaudal to mid-VLPFC, the posterior VLPFC region show-\ning a neural switch cost, approximates what has been termed\nthe inferior frontal junction (35), a region that, together with\nmid-VLPFC, has been consistently implicated in TS. However,\nthe magnitude of the neural switch cost in posterior VLPFC\nacross CSI was poorly fit by a monotonically declining function\n[t(9)  0.12, P  0.9; Fig. 4 A], because the switch cost did not\ndiffer between the shortest and longest CSIs [F  1.0]. This\ninsensitivity to CSI reliably dissociated posterior VLPFC from\nmid-VLPFC [t(9)  2.6, P  0.05].\nVoxel-wise analysis of switch vs. repeat trials collapsed across CSI\nfurther characterized the neural correlates of TS (Fig. 4B). In PFC,\ngreater activation on switch trials was observed in mid-VLPFC\ndorsolateral prefrontal cortex (BA 4546; 54 18 24; 54 33 18;\n48 39 18) and posterior VLPFC (BA 44; 45 9 27), as well as\nin SMA (BA 8; 0 18 48). Neural switch costs were also evident in\nleft inferior (BA 3940; 33 63 42; 51 33 48) and bilateral\nsuperior parietal cortex (BA 7; 27 66 57; 16 63 60).\nAgain, the only region to show a CSI-dependent decline in the\nneural switch cost was the anterior and dorsal extent of mid-VLPFC\n(54 33 18; Fig. 4B). ROI analysis revealed (i) that the neural\nswitch cost was greater at the shortest relative to the longest CSI\n[F(1,9)  4.0, P  0.05], (ii) that the switch cost declined mono-\ntonically [t(9)  2.7, P  0.05], and (iii) that this decrease was\nreliably characterized by the CAM-TS index of conceptual conflict\n[t(9)  2.8, P  0.05].\nIn contrast to mid-VLPFC, the neural switch cost in inferior\nparietal cortex (51 33 48) tended to increase with longer CSIs,\nalthough not reliably [t(9)  1.0]. Qualitatively, this pattern ap-\npeared to correspond to the level of conflict arising in the Response\nlayer of CAM-TS, in that both showed a modest ramping response\n(Fig. 4B), although the fit was unreliable when using the standard\ncalculation of conflict [t(9)  1.4, P  0.19]. However, motivated\nby the recent hypothesis that inferior parietal neurons may accu-\nmulate evidence en route to response selection (36), we recom-\nputed the level of conflict in CAM-TS using accumulated activation\nin the response layer (for details, see supporting information). This\nindex of accumulated conflict fit the inferior parietal response\n[t(9)  2.9, P  0.05; Fig. 4B].\nfMRI results from Experiment 2. (A) Surface rendering of switch  repeat at the shortest CSI (250 ms). Plotted are neural switch costs across CSI from ROI analyses\nFig. 4.\nof left mid-VLPFC (51 27 6) and posterior VLPFC (45 6 27). (B) The contrast of switch vs. repeat collapsed across CSI. Plotted are changes in neural switch cost from\nROIs in left mid-VLPFC (54 33 18), inferior parietal cortex (51 33 48), and SMA (0 18 48). Also depicted are linearly scaled conﬂict signals (dashed lines) from\nconceptual (red) and response [blue (standard) and green (cumulative)] layers of CAM-TS. (C) Bar graphs depict enhancement of switch costs during RS vs. RD trials split\nby switch (red bar) and repeat (green bar) from ROIs in left mid-VLPFC (54 33 18), inferior parietal cortex (51 33 48), and SMA (0 18 48).\nOf central importance, the pattern over CSI in inferior parietal\ncortex dissociated from the decreasing response in left mid-VLPFC\n[t(9)  3.0, P  0.05]. Collectively, these data provide strong\nevidence that mid-VLPFC is recruited in the face of switch-related\ninterference, with CAM-TS suggesting that mid-VLPFC may re-\nsolve interference at the conceptual level.\nEffectsofRR.Proactive interference was also modulated through the\nmanipulation of RR. According to CAM-TS, RR induces increased\nconflict in multiple layers. Behaviorally, RR enhanced the RT\nswitch cost during fMRI [Fig. 3A; TS  RR: F(1,9)  58.5, P \n0.0001]. Neurally, a TS  RR interaction was marginally evident in\nSMA [Fig. 4C; F(1,9)  4.3, P  0.06], reflecting the fact that the\neffect of TS in SMA was entirely accounted for by RR-induced\ninterference: Switch was greater than Repeat on RS [F(1,9)  9.0,\nP  0.05] but not RD trials (F  1).\nIndicative of RR interference and consistent with the conceptual\nconflict signal in CAM-TS, planned contrasts also revealed that\nactivation in mid-VLPFC was greater for switch-RS than\nswitch-RD trials [F(1,9)  5.2, P  0.05]. However, in contrast to\nSMA, neural switch costs on RD trials were also evident in\nmid-VLPFC (Fig. 4C), activation in mid-VLPFC tended to dissociate\nfrom that in SMA, as evidenced by a marginal region [mid-VLPFC\nSMA]  RR  TS interaction [F(1,9)  4.3, P  0.06]. Finally,\nqualitatively similar to mid-VLPFC, inferior parietal cortex showed a\nmarginal switch-RS vs. switch-RD difference [F(1,9)  4.7, P  0.06].\nConclusion\nTS can be understood as an act of memory. Thus, the behavioral\nand psychological consequences of TS can be understood in terms\nof the structures, processes, and failures of memory. This propo-\nsition entails that the control processes contributing to TS are\nindistinguishable from the control processes engaged to overcome\ninterference arising during other acts of memory. Accordingly, the\nneural mechanisms supporting interference resolution during\nmemory retrieval, such as those subserved by mid-VLPFC, are\ncentral for successfully overcoming interference during a TS.\nOur results strongly support these conclusions. In particular, we\ndemonstrate that a simple connectionist model deriving its RT\nswitch cost entirely from experience-dependent changes in its\nassociative structure accounts for switch costs through proactive\ninterference at both the conceptual and response levels and ac-\ncomplishes preparation-related declines in the switch cost through\nprospective, endogenous control. Strikingly, the model’s signature\nof declining conflict among activated concepts after preparation\nwas characteristic of activity in left mid-VLPFC and dissociated this\nregion from other regions active during TS. Importantly, this\nmonotonic decline in mid-VLPFC did not simply track changes in\nRT, because both the form of the decline and the lack of a residual\ncost in mid-VLPFC differentiate these response profiles. Moreover,\ninferior parietal cortex was associated with a ramping response over\nCSI, a pattern qualitatively consistent with the increased conflict in\nthe response layer of CAM-TS at longer CSIs. A role for posterior\nparietal cortex in processing response alternatives finds additional\nsupport in the broader cognitive control literature (37, 38). A temporal\nshift from mid-VLPFC (conceptual) to inferior parietal (response)\ncortex may also parallel electroencephalogram evidence that TS is\naccompanied by an early frontal component (300–500 ms) followed\nby a subsequent parietal component (500–1,000 ms) (39–41).\nIt is important to note that, although the behavioral and neuro-\nimaging results confirm the predictions derived from our memory-\nbased model of TS, this does not preclude the possibility that other\nmodels based on different assumptions could make similar quan-\ntitative predictions. However, such a model must account for\nseveral empirical findings reported here that stand independent of\nthe validity of our specific theoretical framework: (i) the CSI-based\ndecline and the RR effect in mid-VLPFC, (ii) the dissociation of\nmid-VLPFC from sustained effects in posterior VLPFC and SMA,\nand (iii) the dissociation of mid-VLPFC from the ramping function\nin posterior parietal cortex.\nThis caveat notwithstanding, the present framework and sup-\nporting results are broadly appealing, because they provide a\npotential theoretical reconciliation to the long-standing debate over\nthe origins of TS costs and the contribution of cognitive control to\nprospectively switching tasks. CAM-TS is unique from other prom-\ninent frameworks (7, 42) in its exclusive reliance on learning-based\nchanges as a source of its switch cost, its locus of conflict in the\nconceptual and response layers as opposed to the control layer, and\nthe intervention of control in reducing costs. CAM-TS assumes that\na considerable portion of switch costs is attributable to proactive\ninterference from retrieved irrelevant information. This is consis-\ntent with interference accounts of the switch cost and accounts that\nminimize the direct contribution of control to the length of the\nswitch cost. However, our data also indicate that prospective\nengagement of control processes that regulate memory may help\nresolve interference, contributing to preparation-related declines.\nIn this respect, CAM-TS shares an active and strategic control\nprocess with reconfiguration models.\nA distinguishing feature of this synthesis is its potentially parsi-\nmonious account of the neuropsychological data on TS. Again,\ncontrol in CAM-TS reduces interference and facilitates retrieval\nrather than enacts a TS. Consequently, the model predicts that in\nthis explicit cueing variant, TS can proceed without endogenous\ncontrol, although under such circumstances performance is more\nvulnerable to interference (Fig. 2B). Lesion data indicate that\ndamage inclusive of left mid-VLPFC results in an enhanced switch\ncost but neither prohibits the ability to switch tasks nor results in\nperseveration (43). CAM-TS accurately predicts this pattern of en-\nhanced cost but not perseveration after mid-VLPFC insult, a point that\nmay favorably distinguish the present model from a related connec-\ntionist framework designed to model asymmetric costs (7) or other\nelegant mathematical models (42) that suggest minimal control.\nFinally, the present work does not preclude the contributions of\nadditional control processes or interference effects during TS. A\nprocess of goal setting is likely required in many TS contexts (3) and\nmay be associated with distinct regions of PFC, such as frontal polar\ncortex (22). Furthermore, the unique effects in SMA and posterior\nVLPFC, not directly accounted for by CAM-TS, point to the\nmulticomponent nature of TS. By contrast, it is of some interest that\nthe anterior cingulate cortex was not sensitive to conceptual or re-\nsponse-level conflict in this task, although this is not necessarily incon-\nsistent with the broader literature on TS (44) or proactive interference\n(45). Critically, however, the present computational framework and\nneuroimaging evidence argue that one critical component for flexible\nbehavior is interference resolution by mid-VLPFC.\nMethods\nSubjects. Twenty-four right-handed native English speakers (16\nfemale; ages 18–25 yrs) were remunerated $10 per hour for\nparticipation in Experiment 1, and 13 right-handed native English\nspeakers (8 female; ages 18–25 yrs) were remunerated $50 for\nparticipation in Experiment 2. Data from three subjects from\nExperiment 2 (one female) were excluded before fMRI analysis\nbecause of high nonresponse rates due to a difficulty with respond-\ning before the response deadline. Informed consent was obtained\nas approved by the human subjects committees at Massachusetts\nGeneral Hospital, Massachusetts Institute of Technology, and\nStanford University.\nExperiment 1: Design and Procedure. Stimuli consisted of number–\nletter pairs (e.g., ‘‘2b’’) presented centrally in 32-point Monaco font.\nPairs were constructed from a set of 10 letters, five consonants (‘‘p,’’\n‘‘f,’’ ‘‘n,’’ ‘‘k,’’ ‘‘s’’), and five vowels (‘‘a,’’ ‘‘e,’’ ‘‘i,’’ ‘‘o,’’ ‘‘u’’), and a\nset of 10 digits, five odd numbers (‘‘1,’’ ‘‘3,’’ ‘‘5,’’ ‘‘7,’’ ‘‘9’’), and five\neven numbers (‘‘0,’’ ‘‘2,’’ ‘‘4,’’ ‘‘6,’’ ‘‘8’’). The number and letter\npositions were counterbalanced across pairs (e.g., ‘‘2b’’ or ‘‘b2’’).\nSubjects performed one of two categorization tasks (Fig. 1 A)\nwith each stimulus pair. In the number task, subjects categorized the\nnumber as odd or even. In the letter task, subjects categorized the\nletter as vowel or consonant. Categorization decisions for both tasks\nwere reported by using one of the same two buttons (left or right)\nunder the right hand.\nDuring each block of the experiment, the number and letter tasks\nwere intermixed. An instruction cue (LETTER or NUMBER)\npreceded the onset of each target stimulus (Fig. 1) and signaled the\ntask to be performed for that target. The trial terminated once a\nresponse was made. The correct response could be the same as (RS)\nor different from (RD) the response emitted on the previous trial.\nThe RCI varied among four values (50, 226, 506, or 950 ms) that\nexpanded logarithmically. Likewise, the CSI varied among four\nvalues (50, 226, 506, or 950 ms, plus 200 ms for cue presentation).\nTo maximize our ability to detect switch-related declines (6), CSI\nwas blocked, and the order of blocks was fully counterbalanced\nbetween subjects.\nAfter four excluded warm-up trials, each CSI-defined block\nconsisted of 256 trials divided evenly among remaining experimen-\ntal conditions. In addition, trials were counterbalanced for (i) the\nmatch of the irrelevant flanker response to the correct response,\nand (ii) whether the correct target position switched from the\nprevious trial.\nAll behavioral testing was conducted on a Macintosh G4 com-\nputer in a darkened testing room at Stanford University. Before\ndata collection, subjects received extensive training. Subjects con-\ntinuously practiced one of the tasks (e.g., letter or number) alone\nand then practiced the other task (e.g., number or letter). Then\nsubjects practiced switching between the tasks in four short blocks\nat each CSI in experimental order.\nExperiment 2: Design and Behavioral Procedures. Stimuli for the\nfMRI experiment were identical to those used in Experiment 1. As\nin Experiment 1, subjects performed either the number or letter\ntask (Fig. 1B), although making responses with their left hand.\nBefore fMRI data collection, subjects received extensive training:\n(i) outside the scanner, subjects continuously performed the letter\ntask and then the number task, and (ii) subsequently, subjects\npracticed switching between the tasks.\nDuring fMRI scanning, the number and letter tasks were inter-\nmixed. An instruction cue preceded the onset of each stimulus (Fig.\n1B) and signaled the upcoming task. During the CSI within a trial\nand the intertrial interval (ITI) separating trials, a white fixation\ncross was presented centrally (as a preparatory warning, the fixation\ncross turned from white to red immediately before cuestimulus\npresentation). To allow for estimation and deconvolution of the\nhemodynamic response as a function of small changes in CSI\nduration, trials were grouped into pairs of two task events (T-1 and\nT-2), although to the subject, the experiment appeared as a\ncontinuous stream of task events. The onset of the first task event\n(T-1) in each pair followed a variable-duration null fixation period\n(2–16 s) that followed the prior pair. To permit event-related fMRI\nanalyses, T-1 always required performance of the same task that\nhad just been performed (task repeat) during the T-2 phase in the\nprior pair. Furthermore, the consonance of the T-1 manual re-\nsponse to the manual response emitted during the T-2 phase of the\nprior pair (RDRS) and the target and flanker relationship within\nT-1 were counterbalanced across experimental conditions at T-2.\nThe CSI duration for T-1 was fixed at 1,000 ms, and the ITI between\nT-1 and the T-2 cue was always 50 ms.\nThe critical experimental variables were manipulated during T-2.\nThe task for T-2 was either a repeat or switch from that during T-1.\nThe T-2 manual response (left or right) was either the same as (RS)\nor different from (RD) that required during T-1. Finally, the\nduration of the CSI (inclusive of 200-ms task cue) for the T-2 event\nvaried (250 s; 426, 706, or 1,150 ms). Target position was also\ncounterbalanced across T-1T-2 pairs. Collectively, this design\nallowed for analysis of fMRI signal differences at T-2 according to\nthe critical factor manipulations (repeat vs. switch, RS vs. RD, and\nCSI duration). For all events, a response deadline of 1,800 ms was\nimposed. For the imaging analysis, a pair was considered incorrect\nif a subject responded incorrectly or failed to respond before the\nresponse deadline on T-1 andor T-2 within the pair. Analysis of RT\nwas restricted to the trials included in the imaging analysis; error analysis\nwas based only on T-2 events and did not depend on T-1 accuracy.\nDuring fMRI scanning, subjects encountered 480 pairs of trials\nacross four scan epochs. Events were grouped into blocks of 30 pairs\non the basis of T-2 CSI duration. Subjects encountered a block of\neach CSI duration during each scan epoch. Response mappings and\ncondition order were counterbalanced across subjects.\nfMRI Acquisition and Analysis Procedures. Whole-brain imaging was\nperformed on a 3-T Siemens (Iselin, NJ) Trio MRI system.\nFunctional data were acquired by using a gradient-echo echo-\nplanar pulse sequence [repetition time (TR)  2 s, echo time (TE)\n 30 ms, 21 axial slices, 3.125  3.125  5 mm, 1-mm interslice gap,\nfour runs  756 volume acquisitions]. High-resolution T1-weighted\n[magnetization prepared rapid gradient echo (MP-RAGE)] struc-\ntural images were collected for anatomical visualization, during\nwhich subjects practiced the tasks. Head motion was restricted by\nusing firm padding surrounding the head. Projected visual stimuli\nwere viewed through a mirror attached to the standard head coil.\nData were preprocessed by using SPM99 (Wellcome Department\nof Cognitive Neurology, London). Functional images were cor-\nrected for differences in slice acquisition timing, followed by motion\ncorrection (using sinc interpolation). Structural and functional data\nwere spatially normalized to a template based on the Montreal\nNeurological Institute stereotactic space (46) using a 12-parameter\naffine transformation along with a nonlinear transformation using\ncosine basis functions. Images were resampled to 3-mm cubic voxels\nand spatially smoothed with an 8-mm full-width-at-half-maximum\nisotropic Gaussian kernel.\nStatistical models were constructed by using SPM99 under the\nassumptions of the general linear model. The units of analysis were\nthe task pairs described above (Fig. 1B). Because the T-1 phase of\neach pair was counterbalanced across the T-2 conditions, the\nunique contribution to the overall variance due to manipulations of\nT-2 could be estimated. Epochs of 6 s, beginning at the onset of the\nT-1 cue, were used to model each pair; the 6-s epochs were\nconvolved with a canonical hemodynamic response function. Cor-\nrect and incorrect trials were modeled separately, and subsequent\ncontrasts were restricted to correct trials. Effects were estimated by\nusing a subject-specific fixed-effects model, with session-specific\neffects and low-frequency signal components treated as confounds.\nLinear contrasts were used to obtain subject-specific estimates for\neach effect. These estimates were entered into a second-level\nanalysis treating subjects as a random effect, using a one-sample t\ntest against a contrast value of zero at each voxel. Effects in the\nwhole-brain analysis were considered reliable to the extent that they\nconsisted of at least five contiguous voxels that exceeded an\nuncorrected threshold of P  0.001. For the purpose of additional\nanatomical precision, group contrasts were also rendered on a\nMontreal Neurological Institute canonical brain that underwent\ncortical ‘‘inflation’’ using FREESURFER (CorTechs, Centreville, VA)\n(47, 48) and the SPM SURFREND toolbox (written by I. Kahn,\nHarvard University, Cambridge, MA).\nThe voxel-based contrasts were supplemented with ROI analyses\nto further characterize the effects of CSI and interference (RS vs.\nRD) in a priori expected regions, including mid-VLPFC. The ROI\nanalyses were performed by using a toolbox for use with SPM99\n(written by R. Poldrack, University of California, Los Angeles).\nROIs included all significant (uncorrected P  0.001) voxels within\na 6-mm radius of the chosen maximum. Deconvolution allowed\nassessment of the signal change associated with each condition.\nROI analyses were performed on measures of integrated percent\nsignal change (peak  2 TRs) subjected to repeated-measures\nANOVA. Finally, assessment of monotonic decay components over\nCSI in ROIs was performed by estimating a logarithmic decay in the\nswitch vs. repeat difference for each subject individually and then\nentering these estimates into a second-level analysis that treated\nsubject as a random variable, using a one-sample t test against a null\neffect value of 0. Decay in RT cost was assessed similarly, except\nthat a linear rather than a logarithmic model was used, after residual\nanalysis revealed that the distribution of errors deviated from\nnormal. Likewise, convergence between the conflict output of the\ncomputational model and fMRI signal was evaluated within sub-\nject, based on a linear predictive relationship.\n4. Monsell, S. (2003) Trends Cognit. Sci. 7, 134–140.\n5. Logan, G. D. (2003) Curr. Direct. Psychol. Sci. 12, 45–48.\n6. Rogers, R. D. & Monsell, S. (1995) J. Exp. Psychol. Gen. 124, 207–231.\n7. Gilbert, S. J. & Shallice, T. (2002) Cognit. Psychol. 44, 297–337.\n8. Yeung, N. & Monsell, S. (2003) J. Exp. Psychol. Hum. Percept. Perform. 29, 919–936.\n9. Waszak, F., Hommel, B. & Allport, A. (2003) Cognit. Psychol. 46, 361–413.\n10. Sohn, M.-H. & Anderson, J. R. (2003) Mem. Cognit. 31, 775–780.\n11. Mayr, U. & Kliegl, R. (2000) J. Exp. Psychol. Learn Mem. Cognit. 26, 1124–1140.\n12. Petersen, S. E., Fox, P. T., Posner, M. I., Mintun, M. & Raichle, M. E. (1988) Nature 331, 585–589.\n13. Thompson-Schill, S. L., D’Esposito, M., Aguirre, G. K. & Farah, M. J. (1997) Proc. Natl.\n& Moscovitch, M. (MIT Press, Cambridge, MA), Vol. XV, pp. 421–452.\n29. Meiran, N., Chorev, Z. & Sapir, A. (2000) Cognit. Psychol. 41, 211–253.\n30. Cohen, J. D. & Servan-Schreiber, D. (1992) Psychol. Rev. 99, 45–77.\n31. Ruge, H., Brass, M., Koch, I., Rubin, O., Meiran, N. & von Cramon, D. Y. (2005)\n33. Hopfield, J. J. (1982) Proc. Natl. Acad. Sci. USA 79, 2554–2558.\n34. Berlyne, D. E. (1957) Psychol. Rev. 64, 329–339.\n35. Brass, M., Derrfuss, J., Forstmann, B. & von Cramon, D. Y. (2005) Trends Cognit. Sci. 9, 314–316.\n36. Shadlen, M. N. & Newsome, W. T. (2001) J. Neurophysiol. 86, 1916–1936.\n37. Bunge, S. A., Hazeltine, E., Scanlon, M. D., Rosen, A. C. & Gabrieli, J. D. (2002)\n19. Brass, M. & von Cramon, D. Y. (2002) Cereb. Cortex 12, 908–914.\n20. Brass, M. & von Cramon, D. Y. (2004) J. Neurosci. 24, 8847–8852.\n21. Dreher, J.-C. & Grafman, J. (2003) Cereb. Cortex 13, 329–339.\n22. Braver, T. S., Reynolds, J. R. & Donaldson, D. I. (2003) Neuron 39, 713–726.\n23. Reynolds, J. R., Donaldson, D. I., Wagner, A. D. & Braver, T. S. (2004) NeuroImage 21, 1472–1483.\n24. Rushworth, M. F., Hadland, K. A., Paus, T. & Sipila, P. K. (2002) J. Neurophysiol. 87, 2577–2592.\n25. Crone, E. A., Wendelken, C., Donohue, S. E. & Bunge, S. A. (2006) Cereb. Cortex 16, 475–486.\n26. Luks, T. L., Simpson, G. V., Feiwell, R. J. & Miller, W. L. (2002) NeuroImage 17, 792–802.",
         "Neural evidence for dissociable components of task-switching.;Advance preparation and stimulus-induced interference in cued task switching: further insights from BOLD fMRI;Frontal lobe mechanisms that resolve proactive interference.;Domain-general and domain-sensitive prefrontal mechanisms for recollecting events and detecting novelty.;Dissociable Controlled Retrieval and Generalized Selection Mechanisms in Ventrolateral Prefrontal Cortex;Who Comes First? The Role of the Prefrontal and Parietal Cortex in Cognitive Control;The role of the inferior frontal junction area in cognitive control;Selection for Cognitive Control: A Functional Magnetic Resonance Imaging Study on the Selection of Task-Relevant Information;The effects of recent practice on task switching.;Dissociable neural mechanisms underlying response-based and familiarity-based conflict in working memory;Neural Mechanisms of Transient and Sustained Cognitive Control during Task Switching;Stimulus-related priming during task switching;Task-switching and long-term priming: Role of episodic stimulus–task bindings in task-shift costs;Clever homunculus: is there an endogenous act of control in the explicit task-cuing procedure?;Dissociating the roles of the rostral anterior cingulate and the lateral prefrontal cortices in performing two tasks simultaneously or successively.;The Effect of Cingulate Cortex Lesions on Task Switching and Working Memory;Dissociable Contributions of Prefrontal and Parietal Cortices to Response Selection;Components of Switching Intentional Set;Neural implementation of response selection in humans as revealed by localized effects of stimulus–response compatibility on brain activation;Evidence for Anterior Cingulate Cortex Involvement in Monitoring Preparatory Attentional Set;The role of the frontal cortex in task preparation.;Role of the human medial frontal cortex in task switching: a combined fMRI and TMS study.;Task Switching: A PDP Model;Neural basis of a perceptual decision in the parietal cortex (area LIP) of the rhesus monkey.;Executive control of cognitive processes in task switching.;Conflict monitoring and cognitive control.;The role of prefrontal cortex and posterior parietal cortex in task switching.;Component Processes in Task Switching;Control of Cognitive Processes: Attention and Performance XVIII;Task-set switching and long-term memory retrieval.;Mental fatigue and task control: planning and preparation.;Cortical Surface-Based Analysis II: Inflation, Flattening, and a Surface-Based Coordinate System;Cortical Surface-Based Analysis I. Segmentation and Surface Reconstruction;Inhibition in verbal working memory revealed by brain activation.;Dissociating executive mechanisms of task control following frontal lobe damage and Parkinson's disease.;Role of left inferior prefrontal cortex in retrieval of semantic knowledge: a reevaluation.;Costs of a predictible switch between simple cognitive tasks.;Positron emission tomographic studies of the cortical anatomy of single-word processing;Neural networks and physical systems with emergent collective computational abilities.;Neuropsychologia;Trends Cognit. Sci;Science;Behav. Res. Methods Instr. Comput;Psychol. Rev;Nature;Curr. Direct. Psychol. Sci;Item-and Task-Level Processes in Left Inferior Prefrontal Cortex : Positive and Negative Correlates of Encoding;Neuron;Proc. Natl. Acad. Sci. USA 100;Mem. Cognit;Cereb. Cortex;Brain Mapp;J. Cognit. Neurosci;Attention and performance.;J. Neurophysiol;J. Exp. Psychol. Hum. Percept. Perform;Brain Res. Cognit. Brain Res;Prefrontal cortex activation in task switching: an event-related fMRI study.;Cognit. Psychol;Context, cortex, and dopamine: a connectionist approach to behavior and biology in schizophrenia.",
         "Computational and neurobiological mechanisms underlying cognitive flexibility.",
         "None"
        ],
        [
         "42",
         "001f5ce70e459a2f9d909b9c0f15d9dd3e2f6d8c",
         "ABSTRACT Herpes simplex virus type 1 (HSV-1) and HSV-2 cause very similar acute infections but differ in their abilities to reactivate from trigeminal and dorsal root ganglia. To investigate differences in patterns of viral infection, we colabeled murine sensory ganglia for evidence of HSV infection and for the sensory neuron marker A5 or KH10. During acute infection, 7 to 10% of HSV-1 or HSV-2 antigen-positive neurons were A5 positive and 13 to 16% were KH10 positive, suggesting that both viruses reach each type of neuron in a manner proportional to their representation in uninfected ganglia. In murine trigeminal ganglia harvested during HSV latency, 25% of HSV-1 latency-associated transcript (LAT)- and 4% of HSV-2 LAT-expressing neurons were A5 positive, while 12% of HSV-1 LAT- and 42% of HSV-2 LAT-expressing neurons were KH10 positive. A similar difference was observed in murine dorsal root ganglia. These differences could not be attributed to differences in LAT expression levels in A5- versus KH10-positive neurons. Thus, HSV-1 demonstrated a preference for the establishment of latency in A5-positive neurons, while HSV-2 demonstrated a preference for the establishment of latency in KH10-positive neurons. A chimeric HSV-2 mutant that expresses the HSV-1 LAT exhibited an HSV-1 phenotype, preferentially establishing latency in A5-positive neurons. These data imply that the HSV-1 and HSV-2 LAT regions influence the ability of virus to establish latency in different neuronal subtypes. That the same chimeric virus has a characteristic HSV-1 reactivation phenotype further suggests that LAT-influenced establishment of latency in specific neuronal subtypes could be an important part of the mechanism by which LAT influences viral reactivation phenotypes.",
         "T. Margolis,Y. Imai,Li Yang,V. Vallas,P. Krause",
         "T. Margolis,Y. Imai,Li Yang,V. Vallas,P. Krause",
         "https://europepmc.org/articles/pmc1797553?pdf=render",
         "Herpes simplex virus type 1 (HSV-1) and HSV-2 cause very similar acute infections but differ in their\nabilities to reactivate from trigeminal and dorsal root ganglia. To investigate differences in patterns of viral\ninfection, we colabeled murine sensory ganglia for evidence of HSV infection and for the sensory neuron\nmarker A5 or KH10. During acute infection, 7 to 10% of HSV-1 or HSV-2 antigen-positive neurons were A5\npositive and 13 to 16% were KH10 positive, suggesting that both viruses reach each type of neuron in a manner\nproportional to their representation in uninfected ganglia. In murine trigeminal ganglia harvested during HSV\nlatency, 25% of HSV-1 latency-associated transcript (LAT)- and 4% of HSV-2 LAT-expressing neurons were A5\npositive, while 12% of HSV-1 LAT- and 42% of HSV-2 LAT-expressing neurons were KH10 positive. A similar\ndifference was observed in murine dorsal root ganglia. These differences could not be attributed to differences\nin LAT expression levels in A5- versus KH10-positive neurons. Thus, HSV-1 demonstrated a preference for the\nestablishment of latency in A5-positive neurons, while HSV-2 demonstrated a preference for the establishment\nof latency in KH10-positive neurons. A chimeric HSV-2 mutant that expresses the HSV-1 LAT exhibited an\nHSV-1 phenotype, preferentially establishing latency in A5-positive neurons. These data imply that the HSV-1\nand HSV-2 LAT regions inﬂuence the ability of virus to establish latency in different neuronal subtypes. That\nthe same chimeric virus has a characteristic HSV-1 reactivation phenotype further suggests that LAT-\ninﬂuenced establishment of latency in speciﬁc neuronal subtypes could be an important part of the mechanism\nby which LAT inﬂuences viral reactivation phenotypes.",
         "DISCUSSION\nPrimary ganglionic infection with HSV-1 is characterized by\ntwo very different patterns of viral gene expression. In some\nneurons, there is abundant expression of lytic cycle genes,\nresulting in the production of infectious virus, whereas in other\nneurons, a latent infection is established (14, 19, 28, 33). This\npattern is not random; primary sensory neurons identiﬁed by\nMAb KH10 are very permissive for productive infection with\nHSV-1, whereas neurons identiﬁed by MAb A5 serve as the\nprincipal reservoir of latent infection (34). In the current study,\nwe demonstrate that HSV-2 follows a very different pattern of\nganglionic infection, with KH10-positive neurons serving as the\nprincipal reservoir of latent infection in both the TG and lum-\nbar-sacral DRG. These data suggest that HSV-1 and HSV-2\ntake advantage of two different biological niches within the\nperipheral nervous system: niches that go beyond the simple\nboundaries of “above versus below” the waist.\nIt is almost certain that the different lactoseries glycoconju-\ngates recognized by MAbs A5 and KH10 (4, 6, 12) have little\na All mice were treated with acyclovir starting at 40 h postinoculation. Each\ndata point for the recombinant viruses reﬂects analysis of step sections taken\nthrough two blocks of infected tissue (20 ganglia). Each data point for the HSV-1\n(17) reﬂects analysis of step sections taken through one block of infected tissue\n(10 ganglia).\nthere are limits to this type of data acquisition and analysis, this\nobservation is consistent with the hypothesis that A5- and\nKH10-positive neurons are equally permissive for the accumu-\nlation of HSV-1 and HSV-2 LAT.\nSecond, we examined the trigeminal ganglia of transgenic\nmice that contain the HSV-1 LAT-expressing region (8). In\nthese mice, 7.7% (79/1,025) of the LAT-expressing neurons\nwere A5 positive and 15.4% (154/1,000) were KH10 positive,\nwhich was very similar to the proportion of all ganglionic neu-\nrons that were A5 (8.4%; 86/1,026) and KH10 positive (14.9%;\n150/1,007) in these tissues. Thus, these data are also consistent\nwith the hypothesis that A5- and KH10-positive neurons are\nequally permissive for the accumulation of HSV-1 and HSV-2\nLAT.\nTo determine whether different patterns of HSV-1 and\nHSV-2 latent infection were unique to the TG, we studied the\ndistribution of latent HSV infection in murine DRG following\nfootpad inoculation. As summarized in Table 2, the results\nwere very similar to those observed with the TG. Following\nfootpad inoculation, HSV-1 preferentially established latent\ninfection in A5-positive DRG neurons, whereas HSV-2 pref-\nerentially established latent infection in KH10-positive DRG\nto do with the differences in permissiveness of the neurons\nrecognized by these markers. It is more likely that the expres-\nsion of these markers simply correlates with differences in\ncellular protein expression that enable different neuronal pop-\nulations to differentially support or suppress the expression of\nkey viral genes, thus leading to different outcomes of infection.\nAs suggested by the immunostaining patterns in Table 1, A5-\nand KH10-positive neurons largely represent two different\nclasses of intensively studied nociceptors, small-diameter BSL-\nIB4-positive and BSL-IB4-negative ganglionic neurons; neu-\nrons with differential responsiveness to nerve growth factor\nand glial cell line-derived neurotrophic factor (20, 27). It is\ntherefore tempting to speculate that neurotrophic factor-re-\nsponsive signaling pathways may play a role in regulating the\nestablishment and maintenance of HSV latent infection. These\nconsiderations highlight the importance of studying the inter-\naction of HSV with different sensory neuronal subtypes, and\nnot just ganglia as whole, in understanding viral latency and\nreactivation.\nThe ability of acyclovir to reduce the tropism of HSV-1\nlatent infection for A5-positive neurons implies that the pref-\nerential establishment of latency in these cells is dependent\nupon viral replication. One possibility is that type-speciﬁc se-\nquences enhance viral spread to A5-positive neurons during\nestablishment of latency and that acyclovir inhibits this spread.\nAlternatively, by inhibiting viral DNA replication, acyclovir\nmight tip the course of infection in replication-permissive neu-\nrons toward latency, thus reducing the relative latent viral load\nin A5-positive neurons.\nThe HSV LAT is the only available reliable marker of la-\ntently infected neurons. While neurons may contain viral DNA\nas detected by PCR-based techniques, it is unclear whether\nviral DNA detected in this manner represents reactivation-\ncompetent latent virus or simply fragments of the viral ge-\nnome. While it is not known whether viral reactivation is re-\nstricted to LAT-producing neurons, animal data showing the\naltered recurrence phenotypes of viruses with impaired capac-\nity to produce LAT supports the conclusion that LAT produc-\ntion is a biologically relevant marker of viral latency (10, 13, 15,\n17, 22, 32).\nThe different distributions of latent HSV-1 and HSV-2 in\nmurine sensory ganglia cannot simply be a consequence of\ndifferential LAT accumulation in A5-positive and KH10-posi-\ntive neurons. First, we found no signiﬁcant difference in HSV-1\nor HSV-2 LAT signal intensity among A5- and KH10-positive\nneurons. Second, studies of HSV-1 LAT transgene expression\nin murine trigeminal ganglia revealed no evidence of differen-\ntial LAT accumulation in A5- and KH10-positive neurons.\nThird, the treatment of mice with acyclovir, a drug that inhibits\nproductive viral infection but should not affect the accumula-\ntion of LAT, shifted the distribution of LAT-positive, latently\ninfected neurons closer to the distribution of A5- and KH10-\npositive neurons in uninfected ganglia. Fourth, we previously\ndemonstrated that the ratio of LAT-expressing A5-positive\nand KH10-positive neurons was independent of the sensitivity\nof the assay used to measure LAT (34), an observation that\nwould not be expected if there were different levels of LAT\nexpression in these neuronal populations. And ﬁfth, following\nocular inoculation with a TK deletion virus where latency is the\nonly possible outcome of infection, the distribution of viral\nlatency in A5-positive and KH10-positive neurons, as assayed\nby LAT expression, closely matches the relative distribution of\nthese neurons in the TG; a result that demonstrates similar\naccumulation of LAT in A5- and KH10-positive neurons if the\nvirus has equal opportunity to establish latency in these two\nneuronal populations (Y. Imai et al., personal communica-\ntion).\nIn the course of these studies, we found that a chimeric\nHSV-2 mutant that expresses the HSV-1 LAT (HSV-2 333/\nLAT1) establishes latency primarily in A5-positive neurons,\nsimilar to HSV-1. This indicates that the difference in the\nlatency phenotype of the two viruses can be mapped to a small\nportion of the long repeat region of the viral genome, the same\nregion that codes for the stable LAT intron and which conveys\nthe phenotype of virus type-associated, site-speciﬁc recurrence\n(35). This viral function might be associated with any or all of\nthe substituted sequences, including the LAT promoter, the\nprimary 5 end sequences or the sequences coding for the LAT\nintron. It might also be associated with sequences coding in an\nantisense direction relative to LAT, such as those recently\ndescribed for AL (24) or with the overlapping 3 end of the\nICP0 coding region, although the close homology of the HSV-1\nand HSV-2 ICP0 3 amino acid sequences makes this unlikely.\nOur ﬁndings are particularly interesting in light of our pre-\nvious report of KOS 62, an HSV-1 LAT deletion virus that\nexpresses the lacZ gene under control of the HSV-1 LAT\npromoter. Although containing a 1.6-kb LAT region deletion,\nthis virus preferentially established latency (as assayed by -ga-\nlactosidase expression) in A5 neurons, similar to that with the\nwild-type virus. Together with the current ﬁndings, this sug-\ngests that critical sequences for neuron subtype-speciﬁc estab-\nlishment of latency may reside in regions not shared by the\n1.6-kb deletion in KOS 62 and the 2.8-kb substitution in HSV-2\n333/LAT1. Thus, sequences in the HSV-1 LAT promoter, the\nLAT 5 exon, or in the 3 half of the LAT intron seem likely to\nbe important for this phenotype. It also is possible that these\nﬁndings are inﬂuenced by other differences between the\nHSV-1 backbone of KOS 62 and the HSV-2 backbone of\nHSV-2 333/LAT1.\nImplication of the LAT region of the viral genome in neu-\nronal type-speciﬁc establishment of latency is intriguing since it\nappears to play an important role in both the establishment\n(25, 31) and reactivation of latent infection (13), and more\nrecent studies suggest that LAT inﬂuences neuronal survival,\npossibly via an effect on apoptosis (1, 3, 7, 23). None of the\nleading hypotheses of LAT function serve to explain the type-\nspeciﬁc role of LAT sequences either in viral reactivation phe-\nnotype or in establishment of latency. Thus, further investiga-\ntion into the mechanism of LAT action is clearly warranted.\nOur ﬁndings are consistent with a potential role for LAT\nsequences in selectively promoting latent infection in speciﬁc\ntypes of sensory neurons via an as-yet-undeﬁned mechanism.\nThe ability of LAT region sequences to direct latent infec-\ntion to speciﬁc sensory neuronal subtypes and the correspond-\ning differences in the abilities of each virus to recur from\ndifferent anatomical locations (16) imply that HSV derives an\nadvantage in recurrence frequency from establishing latency in\ndifferent neurons in each anatomical location, A5-positive neu-\nrons in trigeminal ganglia and KH10-positive neurons in lum-\nbar-sacral ganglia. The increased prevalence of KH10-positive\nneurons in lumbar-sacral dorsal root ganglia suggests one\nmechanism by which this phenotype could inﬂuence type-spe-\nciﬁc recurrence patterns.",
         "Primary infection of mice with herpes simplex virus type 1\n(HSV-1) and HSV-2 is characterized by viral replication at the\nsite of inoculation, followed by retrograde axonal transport of\nthe virus to corresponding sensory ganglia where infection\nfollows two very different pathways (14, 19, 28, 33). In some\nneurons, the virus expresses productive cycle genes, replicates,\nand causes host cell death, whereas in other neurons, the virus\nestablishes a latent infection characterized by limited viral\ntranscription except\nfor the latency-associated transcripts\n(LATs), which accumulate to high copy number in the nuclei\nof latently infected cells (29). The LATs code from the long\nrepeat region of the viral genome and run antisense to the\nimmediate-early transactivator ICP0, the protein kinase R in-\nhibitor ICP34.5, the 3 end of the immediate-early transacti-\nvator ICP4 and the AL gene (24). A unique feature of the\nmajor 2-kb LAT is that it is a stable intron, spliced from a much\nless stable primary transcript (5). Studies from multiple labs\nsuggest that the LAT region of the viral genome plays an\nimportant role in both the establishment (25, 31) and the\nreactivation of latent infection (13). The mechanisms respon-\nsible for this are poorly deﬁned. Hypotheses (reviewed in ref-\nerence 3) include that the HSV-1 LATs mediate latent infec-\ntion by the antisense regulation of expression of the key viral\nimmediate-early gene ICP0 (5) or by the inhibition of apop-\ntosis (1, 23), an action that appears to at least in part be\nmediated by a LAT region micro-RNA (7), and also might be\nmediated by the interferons (21). To date, no published studies\naddress the mechanism of HSV-2 LAT action.\nThe clinical diseases caused by HSV-1 and HSV-2 are very\nsimilar except for marked differences in site-speciﬁc reactiva-\ntion. Whereas HSV-1 reactivates most efﬁciently from trigem-\ninal ganglia (TG), giving rise to recurrent disease of the face,\neyes, and oropharynx, HSV-2 reactivates more efﬁciently from\nthe lumbar-sacral ganglia, giving rise to recurrent disease be-\nlow the waist, including genital disease (16). Studies of HSV-1\nand HSV-2 chimeric viruses in rabbit and guinea pig models\nindicate that a 2.8-kb region of the viral genome that codes for\nLAT plays an important role in determining the efﬁciency of\nthis site-speciﬁc reactivation. Speciﬁcally, an HSV-2 virus en-\ngineered to express HSV-1 LAT instead of the native LAT\n(HSV-2 333/LAT1) reactivated more efﬁciently than did\nHSV-2 from the trigeminal ganglion (35), and an HSV-1 virus\nengineered to express HSV-2 LAT instead of the native LAT\n(HSV-1 17syn/LAT2) reactivated less efﬁciently from the tri-\ngeminal ganglion (9).\nPrimary sensory neurons are a diverse population of cells\nthat can be classiﬁed according to cellular morphology, phys-\niological response properties, and patterns of gene expression,\nand in a recent study, we demonstrated that the distribution of\nproductive and latent HSV-1 infection among populations of\nganglionic neurons is not random. Speciﬁcally, we showed that\nalthough all neuronal populations are capable of supporting a\nproductive HSV-1 infection, some neuronal populations of the\ntrigeminal ganglion are more likely than others to be associ-\nated with latent infection. Using a battery of antisera capable\nof recognizing different populations of ganglionic neurons, we\nfound that neurons identiﬁed by monoclonal antibody (MAb)\nA5 (speciﬁc for a population of neurons expressing Gal1-\n4GlcNAc-R epitopes) were the principal reservoir of latent\ninfection, whereas latent infection was rarely found in neurons\nidentiﬁed by MAb KH10 (a different population of ganglionic\nneurons expressing Gal1-3Gal1-4NAc-R epitopes) (6, 34).\nThese ﬁndings highlight not only the key role that the host\nneuron may play in regulating the repertoire of viral gene\nexpression during establishment of HSV latent infection but\nalso the importance of considering the complex neuronal com-\nposition of primary sensory ganglia in interpreting the results\nof in vivo studies of HSV infection.\nIn the current study, we tested the hypothesis that HSV-1\nand HSV-2 establish latent infection in different populations of\nganglionic neurons. This hypothesis derives from the role that\nLAT plays in the establishment of HSV latency, the LAT-\ndependent, site-speciﬁc reactivation phenotypes of HSV-1\nand HSV-2, and differences in the HSV-1 and HSV-2 LAT\ncoding regions. In the course of this study, we found signif-\nicant differences in the latent distributions of HSV-1 and\nHSV-2, with KH10-positive neurons serving as the principal\nreservoir of HSV-2 latent infection. We also found that\nthese differences are attributable to the same 2.8-kb portion\nof the LAT coding region that controls the site-speciﬁc\nreactivation phenotype (9, 35).\nVirus and animal preparation. The recombinant viruses HSV-2 333/Lat1 and\nHSV-2 333/Lat1R have been described in detail previously (35). HSV-1 strains\nKOS and 17 and HSV-2 strain 333 and the recombinant viruses HSV-2 333/\nLat1 and HSV-2 333/Lat1R were propagated as previously described (34). Viral\ntiters of 1  108 PFU/ml were typically obtained. Protocols were approved by the\nUCSF Committee on Animal Research. Six-week-old female outbred Swiss\nWebster mice (Simonsen Laboratories, Gilroy, CA) were anesthetized and in-\noculated as previously described (11, 19). In brief, mice were anesthetized by\nintraperitoneal injection of pentobarbital. For ocular inoculations, this was fol-\nlowed by topical proparacaine (1%). Once adequate anesthesia was obtained, the\ncorneal epithelium was abraded with a sterile 27-gauge needle, and the ocular\nsurface was inoculated with about 106 PFU of viral stock. For rear footpad\ninoculations, the keratinized surface of the footpad was lightly abraded with an\nemery board, followed by the application of about 3  106 PFU of viral stock to\nthe abraded skin. Mice were then allowed to recover from anesthesia (about 45\nto 60 min) on their backs. In those experiments requiring the use of an antiviral\ndrug, acyclovir (1.2 mg/ml) was placed in the drinking water 40 h after viral\ninoculation. Uninfected trigeminal ganglia from LAT 3098 transgenic mice (8)\nwere a generous gift of David Bloom (Gainesville, FL).\nTissue preparation. Three or 21 days postinoculation, mice were euthanized\nby carbon dioxide inhalation and thoracotomy. Cardiac perfusion with 0.1 M\nphosphate-buffered saline (PBS, pH 7.4) was performed immediately, followed\nby perfusion with 4% paraformaldehyde (PFA) in 0.1 M phosphate buffer (PB,\npH 7.4). Dissected ganglia were immersion ﬁxed in 4% PFA at 4°C for 30 to 60\nmin, equilibrated with 30% sucrose in 0.1 M PBS at 4°C, pooled in groups of 10\n(TG) or 50 dorsal root ganglia (DRG), embedded in Tissue-Tek O.C.T. com-\npound (Sakura Finetechnical, Tokyo, Japan), and snap-frozen in liquid nitrogen.\nSerial sections (7 m) of these pooled blocks of infected ganglia were then\nDual ﬂuorescent staining. Dual ﬂuorescent staining of frozen tissue sections\nwas carried out as previously described (18, 34). The following antisera were\nused: mouse MAbs A5 and KH10 (Developmental Studies Hybridoma Bank,\nIowa City, IA), anti-TrkA (a gift from Louis Reichardt, UCSF), anti-c-ret (Am-\ngen, Inc., Thousand Oaks, CA), anti-pan Trk (Promega, Madison, WI), and\nanti-CGRP (Amersham, Arlington Heights, IL). Fluorescein isothiocyanate-con-\njugated rabbit anti-HSV-1 and anti-HSV-2 antisera were from DAKO, Glostrup,\nDenmark. Bandeiraea Simplicifolia isolectin B4 (BSL-IB4) was obtained from\nVector Labs (Burlingame, CA).\nPreparation of DIG-labeled RNA probes for ﬂuorescence in situ hybridization\n(FISH). Labeled riboprobes for the HSV-1 stable LAT intron were prepared\nusing plasmid pATD-19 as a template (29). This plasmid contains a 347-bp\nfragment of HSV-1 DNA (genomic nucleotides 119629 to 119975). Labeled\nriboprobes for the HSV-2 stable LAT intron were prepared using a 782-bp\n(StuI-XhoI) fragment of pBam/Xho4Z as a template. Plasmid Bam/Xho4Z con-\ntains a 1.8-kb BamHI-XhoI fragment of HSV-2 DNA (genomic nucleotides\n120489 to 122276) cloned into the multiple cloning site of pGEM-4Z (Promega).\nPlasmids were linearized with either HindIII (for HSV-1) or StuI (for HSV-2),\npuriﬁed by phenol-chloroform extraction and ethanol precipitation, and then\nincubated with digoxigenin (DIG) RNA labeling mix (Roche, Mannheim, Ger-\nmany) and either T3 (for HSV-1) or SP6 (for HSV-2) polymerase for 3 h at 37°C.\nAfter treatment with DNase I, probes were puriﬁed using G-50 Sephadex col-\numns (Roche) and dissolved into 50% formamide in diethylpyrocarbonate-\ntreated water. Probe concentrations were determined by spotting each probe\nonto a nylon membrane; visualizing with anti-DIG-AP Fab fragments (Roche),\nnitroblue tetrazolium (Roche), and 5-bromo-4-chloro-3-indolylphosphate tolui-\ndinium (Roche); and comparing the intensity of the generated signal to that of\nthe labeled control RNA (Roche).\nCombined staining by ﬂuorescent in situ hybridization and immunohisto-\nchemistry. Tissue sections were postﬁxed with 4% PFA in 0.1 M PB for 10 min\nand then sequentially washed in 2 SSC (1 SSC is 0.15 M NaCl plus 0.015 M\nsodium citrate), diethyl pyrocarbonate (DEPC)-treated water, and 0.1 M trieth-\nanolamine hydrochloride (TEA). Tissue sections were then treated with 0.25%\nacetic anhydride in 0.1 M TEA (3 min), washed in 2 SSC, and incubated with\nprehybridization buffer (50% formamide, 40 g/ml salmon sperm DNA, 5\nSSC) at 45°C for 2 h. Each riboprobe was heated to 80°C for 10 min, chilled on\nice, and then diluted to 1 g/ml in preheated (55°C) hybridization buffer (50%\nformamide, 1 Denhardt’s solution, 10 mM EDTA, 10% dextran sulfate, 0.5\nmg/ml yeast tRNA, 0.5 mg/ml salmon sperm DNA, 3 SSC). Prehybridized\ntissue was incubated with each probe at 55°C overnight. After hybridization,\ntissue was washed with 2 SSC (10 min) and treated with 20 g/ml of RNase A\nin 2 SSC at 37°C for 30 min, followed by serial washing with graduated dilutions\nof SSC at 55°C. Tissue was then equilibrated with 0.1 M PBS and incubated with\nanti-DIG-rhodamine Fab fragments (Roche) diluted in 0.1 M PBS with 1\nblocking solution (Roche) for 30 min. After washing with 0.1 M PBS for 10 min,\ntissue was blocked with 3% normal goat serum in 0.1 M PBS for 30 min and then\nincubated with either MAb A5 or KH10 at 4°C for 2 nights. Tissue was then\nwashed with 0.1 M PBS, blocked with 3% normal goat serum in 0.1 M PBS and\nincubated for 1 h with FITC-labeled goat anti-mouse immunoglobulin M (Bio-\nsource, Camarillo, CA) diluted 1:100 in 0.1 M PBS with 1% normal goat serum.\nStained tissue sections were then washed with 0.1 M PBS and cover slipped with\nVectashield mounting medium (Vector Labs).\nStatistical analysis. To determine whether there were signiﬁcant differences in\nthe pattern of ganglionic latent infection, pair-wise comparisons were made using\nthe 2 test. Since we performed multiple comparisons, a Bonferroni-adjusted P\nvalue of 0.00625 was used to assign signiﬁcance. The two-tailed Student’s t test\nand Kolmogorov-Smirnov test were used to determine whether there were sig-\nniﬁcant differences in LAT signal intensities among different neuronal cell types.",
         "",
         "RESULTS\nFollowing ocular inoculation with HSV-1 or HSV-2, we as-\nsayed latently infected murine TG by FISH for viral LAT\nexpression in order to determine the proportion of the latent\nviral load present in populations of neurons identiﬁable by\nMAbs A5 and KH10. In the mouse trigeminal ganglion, MAbs\nA5 and KH10 recognize neuronal populations with very little\noverlap (Table 1). Most A5-positive neurons are immunoreac-\ntive for the calcitonin gene-related peptide and the high-afﬁn-\nity nerve growth factor receptor (TrkA), whereas KH10-posi-\ntive neurons colabel with the lectin BSL-IB4, identifying them as\na population of small-diameter, glial cell line-derived neurotro-\nphic factor-responsive neurons (2, 4, 20, 26). These neurochem-\nical characteristics imply that both KH10- and A5-positive neu-\nrons transmit nociceptive stimuli but represent functionally\ndistinct sensory neuronal populations (27, 30).\nConsistent with our previous report (34), 21 days after ocular\ninoculation with HSV-1 (KOS strain), 50% of the LAT-express-\ning TG neurons were A5 positive, while only 7% expressed the\nKH10 marker (Table 2 and Fig. 1). This distribution cannot be\nexplained by the neuronal composition of the ganglion (Table 2),\nand our previous studies showed that the expression of these\nmarkers does not change following infection (34). This distribu-\ntion also cannot be explained by differential access of peripherally\ninoculated virus to the two cell types since these neurons become\nproductively infected in proportion to their representation in the\nganglion (Table 3 and see reference 34). Thus, these data suggest\nthat a greater proportion of A5-positive neurons survive acute\nganglionic infection with HSV-1 to become latently infected than\ndo KH10-positive neurons.\nWhen administered under the same experimental conditions\nas those of HSV-1, HSV-2 (333 strain) caused fatal encepha-\nlitis in 100% of infected mice. Decreasing the viral inoculum by\n5 logs prevented encephalitis but also largely prevented the\nestablishment of latent infection. Adding the viral DNA poly-\nmerase inhibitor acyclovir to the drinking water at 40 h posti-\nnoculation prevented fatal encephalitis without preventing\nganglionic infection. In mice infected with HSV-2 and treated\nwith acyclovir in this manner, 42% of LAT-expressing TG\nneurons at 21 days postinoculation were KH10 positive and\nonly 4% were A5 positive (Table 2 and Fig. 2). These data\nimply that HSV-2 latent infection of the TG is preferentially\nestablished in KH10-positive neurons, with a pattern of infec-\ntion signiﬁcantly different from that observed with HSV-1 (P \n0.000001).\nTo determine whether the two different patterns of latent\ninfection were attributable to our use of acyclovir during\nHSV-2 infection, we examined the effect of acyclovir on the\ndistribution of HSV-1 latent infection. As summarized in Table\n2, the treatment of mice with acyclovir during acute ganglionic\ninfection reduced the proportion of HSV-1 LAT-expressing\nneurons that were A5 positive (to 25%) and increased the\nproportion that were KH10 positive (to 12%), in effect shifting\nthe distribution of LAT-expressing neurons closer to the rep-\nresentation of A5- and KH10-positive neurons in uninfected\nganglia and to the distribution of productive HSV-1 infection\namong these two neuronal populations. However, the pattern\nof HSV-1 LAT expression in the TG of acyclovir-treated mice\nwas still signiﬁcantly different from that observed with HSV-2\n(P  0.000001), and from the distribution of neurons in the\nuninfected ganglion. Thus, treatment with acyclovir blunts but\ndoes not eliminate the preferential establishment of HSV-1\nlatency in A5-positive neurons.\nThe effect of acyclovir administration on the neuronal dis-\ntribution of latent HSV-1 infection argues strongly that the\ntropism of HSV-1 latent infection for A5-positive neurons is\ndependent upon viral replication and is not simply an artifact\nof differential accumulation of LAT in A5- and KH10-positive\nneurons. However, to examine this issue more closely we per-\nformed two experiments speciﬁcally addressing the question of\nwhether the differential distribution of HSV-1 and HSV-2 in\nA5- and KH10-positive neurons in latently infected ganglia was\ndue to differential accumulation of LAT in these cells.\nFirst, we carried out digital analysis of confocal microscopic\nimages of latently infected ganglionic tissue probed by immu-\nnoﬂuorescence and FISH. As seen in Fig. 3, we found no\nsigniﬁcant difference in LAT signal\nintensity (HSV-1 or\nHSV-2) among A5- and KH10-positive neurons. Although\nneurons. These differences were statistically signiﬁcant (P \n0.000001). We also noted that KH10-positive neurons are\nmore prevalent in DRG than in TG (26 versus 16%, respec-\ntively; P  0.000001).\nWe hypothesized that differences in the genotypes and phe-\nnotypes of LATs expressed by HSV-1 and HSV-2 may be\nresponsible for the different patterns of latent infection with\nthese two viruses. To explore a possible role for the viral LATs\nin the differential establishment of HSV-1 and HSV-2 latent\ninfection, we next examined the neuronal distribution of latent\nviral infection following ocular inoculation with HSV-2 333/\nLAT1. Mice infected with either the viral rescuant (HSV-2\n333/LAT1R) or the 17 strain of HSV-1 (source of the donor\nLAT) served as controls. As summarized in Table 4 HSV-2\n333/LAT1 established latent infection in the TG with an\nHSV-1 phenotype; 40% of the neurons latently infected with\nHSV-2 333/LAT1 were A5 positive, while only 2% expressed\nthe KH10 marker. In contrast, HSV-2 333/LAT1R established\nlatent infection with an HSV-2 phenotype, with a preference\nfor KH10-positive neurons. These data strongly suggest that\nthe different patterns of latent infection observed with HSV-1\nand HSV-2 are due to a viral function associated with a 2.8-kb\nportion of the LAT coding region.",
         "downloading",
         "Herpes simplex virus type 1 (HSV-1) and HSV-2 cause very similar acute infections but differ in their\nabilities to reactivate from trigeminal and dorsal root ganglia. To investigate differences in patterns of viral\ninfection, we colabeled murine sensory ganglia for evidence of HSV infection and for the sensory neuron\nmarker A5 or KH10. During acute infection, 7 to 10% of HSV-1 or HSV-2 antigen-positive neurons were A5\npositive and 13 to 16% were KH10 positive, suggesting that both viruses reach each type of neuron in a manner\nproportional to their representation in uninfected ganglia. In murine trigeminal ganglia harvested during HSV\nlatency, 25% of HSV-1 latency-associated transcript (LAT)- and 4% of HSV-2 LAT-expressing neurons were A5\npositive, while 12% of HSV-1 LAT- and 42% of HSV-2 LAT-expressing neurons were KH10 positive. A similar\ndifference was observed in murine dorsal root ganglia. These differences could not be attributed to differences\nin LAT expression levels in A5- versus KH10-positive neurons. Thus, HSV-1 demonstrated a preference for the\nestablishment of latency in A5-positive neurons, while HSV-2 demonstrated a preference for the establishment\nof latency in KH10-positive neurons. A chimeric HSV-2 mutant that expresses the HSV-1 LAT exhibited an\nHSV-1 phenotype, preferentially establishing latency in A5-positive neurons. These data imply that the HSV-1\nand HSV-2 LAT regions inﬂuence the ability of virus to establish latency in different neuronal subtypes. That\nthe same chimeric virus has a characteristic HSV-1 reactivation phenotype further suggests that LAT-\ninﬂuenced establishment of latency in speciﬁc neuronal subtypes could be an important part of the mechanism\nby which LAT inﬂuences viral reactivation phenotypes.\nPrimary infection of mice with herpes simplex virus type 1\n(HSV-1) and HSV-2 is characterized by viral replication at the\nsite of inoculation, followed by retrograde axonal transport of\nthe virus to corresponding sensory ganglia where infection\nfollows two very different pathways (14, 19, 28, 33). In some\nneurons, the virus expresses productive cycle genes, replicates,\nand causes host cell death, whereas in other neurons, the virus\nestablishes a latent infection characterized by limited viral\ntranscription except\nfor the latency-associated transcripts\n(LATs), which accumulate to high copy number in the nuclei\nof latently infected cells (29). The LATs code from the long\nrepeat region of the viral genome and run antisense to the\nimmediate-early transactivator ICP0, the protein kinase R in-\nhibitor ICP34.5, the 3 end of the immediate-early transacti-\nvator ICP4 and the AL gene (24). A unique feature of the\nmajor 2-kb LAT is that it is a stable intron, spliced from a much\nless stable primary transcript (5). Studies from multiple labs\nsuggest that the LAT region of the viral genome plays an\nimportant role in both the establishment (25, 31) and the\nreactivation of latent infection (13). The mechanisms respon-\nsible for this are poorly deﬁned. Hypotheses (reviewed in ref-\nerence 3) include that the HSV-1 LATs mediate latent infec-\ntion by the antisense regulation of expression of the key viral\nimmediate-early gene ICP0 (5) or by the inhibition of apop-\ntosis (1, 23), an action that appears to at least in part be\nmediated by a LAT region micro-RNA (7), and also might be\nmediated by the interferons (21). To date, no published studies\naddress the mechanism of HSV-2 LAT action.\nThe clinical diseases caused by HSV-1 and HSV-2 are very\nsimilar except for marked differences in site-speciﬁc reactiva-\ntion. Whereas HSV-1 reactivates most efﬁciently from trigem-\ninal ganglia (TG), giving rise to recurrent disease of the face,\neyes, and oropharynx, HSV-2 reactivates more efﬁciently from\nthe lumbar-sacral ganglia, giving rise to recurrent disease be-\nlow the waist, including genital disease (16). Studies of HSV-1\nand HSV-2 chimeric viruses in rabbit and guinea pig models\nindicate that a 2.8-kb region of the viral genome that codes for\nLAT plays an important role in determining the efﬁciency of\nthis site-speciﬁc reactivation. Speciﬁcally, an HSV-2 virus en-\ngineered to express HSV-1 LAT instead of the native LAT\n(HSV-2 333/LAT1) reactivated more efﬁciently than did\nHSV-2 from the trigeminal ganglion (35), and an HSV-1 virus\nengineered to express HSV-2 LAT instead of the native LAT\n(HSV-1 17syn/LAT2) reactivated less efﬁciently from the tri-\ngeminal ganglion (9).\nPrimary sensory neurons are a diverse population of cells\nthat can be classiﬁed according to cellular morphology, phys-\niological response properties, and patterns of gene expression,\nand in a recent study, we demonstrated that the distribution of\nproductive and latent HSV-1 infection among populations of\nganglionic neurons is not random. Speciﬁcally, we showed that\nalthough all neuronal populations are capable of supporting a\nproductive HSV-1 infection, some neuronal populations of the\ntrigeminal ganglion are more likely than others to be associ-\nated with latent infection. Using a battery of antisera capable\nof recognizing different populations of ganglionic neurons, we\nfound that neurons identiﬁed by monoclonal antibody (MAb)\nA5 (speciﬁc for a population of neurons expressing Gal1-\n4GlcNAc-R epitopes) were the principal reservoir of latent\ninfection, whereas latent infection was rarely found in neurons\nidentiﬁed by MAb KH10 (a different population of ganglionic\nneurons expressing Gal1-3Gal1-4NAc-R epitopes) (6, 34).\nThese ﬁndings highlight not only the key role that the host\nneuron may play in regulating the repertoire of viral gene\nexpression during establishment of HSV latent infection but\nalso the importance of considering the complex neuronal com-\nposition of primary sensory ganglia in interpreting the results\nof in vivo studies of HSV infection.\nIn the current study, we tested the hypothesis that HSV-1\nand HSV-2 establish latent infection in different populations of\nganglionic neurons. This hypothesis derives from the role that\nLAT plays in the establishment of HSV latency, the LAT-\ndependent, site-speciﬁc reactivation phenotypes of HSV-1\nand HSV-2, and differences in the HSV-1 and HSV-2 LAT\ncoding regions. In the course of this study, we found signif-\nicant differences in the latent distributions of HSV-1 and\nHSV-2, with KH10-positive neurons serving as the principal\nreservoir of HSV-2 latent infection. We also found that\nthese differences are attributable to the same 2.8-kb portion\nof the LAT coding region that controls the site-speciﬁc\nreactivation phenotype (9, 35).\nVirus and animal preparation. The recombinant viruses HSV-2 333/Lat1 and\nHSV-2 333/Lat1R have been described in detail previously (35). HSV-1 strains\nKOS and 17 and HSV-2 strain 333 and the recombinant viruses HSV-2 333/\nLat1 and HSV-2 333/Lat1R were propagated as previously described (34). Viral\ntiters of 1  108 PFU/ml were typically obtained. Protocols were approved by the\nUCSF Committee on Animal Research. Six-week-old female outbred Swiss\nWebster mice (Simonsen Laboratories, Gilroy, CA) were anesthetized and in-\noculated as previously described (11, 19). In brief, mice were anesthetized by\nintraperitoneal injection of pentobarbital. For ocular inoculations, this was fol-\nlowed by topical proparacaine (1%). Once adequate anesthesia was obtained, the\ncorneal epithelium was abraded with a sterile 27-gauge needle, and the ocular\nsurface was inoculated with about 106 PFU of viral stock. For rear footpad\ninoculations, the keratinized surface of the footpad was lightly abraded with an\nemery board, followed by the application of about 3  106 PFU of viral stock to\nthe abraded skin. Mice were then allowed to recover from anesthesia (about 45\nto 60 min) on their backs. In those experiments requiring the use of an antiviral\ndrug, acyclovir (1.2 mg/ml) was placed in the drinking water 40 h after viral\ninoculation. Uninfected trigeminal ganglia from LAT 3098 transgenic mice (8)\nwere a generous gift of David Bloom (Gainesville, FL).\nTissue preparation. Three or 21 days postinoculation, mice were euthanized\nby carbon dioxide inhalation and thoracotomy. Cardiac perfusion with 0.1 M\nphosphate-buffered saline (PBS, pH 7.4) was performed immediately, followed\nby perfusion with 4% paraformaldehyde (PFA) in 0.1 M phosphate buffer (PB,\npH 7.4). Dissected ganglia were immersion ﬁxed in 4% PFA at 4°C for 30 to 60\nmin, equilibrated with 30% sucrose in 0.1 M PBS at 4°C, pooled in groups of 10\n(TG) or 50 dorsal root ganglia (DRG), embedded in Tissue-Tek O.C.T. com-\npound (Sakura Finetechnical, Tokyo, Japan), and snap-frozen in liquid nitrogen.\nSerial sections (7 m) of these pooled blocks of infected ganglia were then\nDual ﬂuorescent staining. Dual ﬂuorescent staining of frozen tissue sections\nwas carried out as previously described (18, 34). The following antisera were\nused: mouse MAbs A5 and KH10 (Developmental Studies Hybridoma Bank,\nIowa City, IA), anti-TrkA (a gift from Louis Reichardt, UCSF), anti-c-ret (Am-\ngen, Inc., Thousand Oaks, CA), anti-pan Trk (Promega, Madison, WI), and\nanti-CGRP (Amersham, Arlington Heights, IL). Fluorescein isothiocyanate-con-\njugated rabbit anti-HSV-1 and anti-HSV-2 antisera were from DAKO, Glostrup,\nDenmark. Bandeiraea Simplicifolia isolectin B4 (BSL-IB4) was obtained from\nVector Labs (Burlingame, CA).\nPreparation of DIG-labeled RNA probes for ﬂuorescence in situ hybridization\n(FISH). Labeled riboprobes for the HSV-1 stable LAT intron were prepared\nusing plasmid pATD-19 as a template (29). This plasmid contains a 347-bp\nfragment of HSV-1 DNA (genomic nucleotides 119629 to 119975). Labeled\nriboprobes for the HSV-2 stable LAT intron were prepared using a 782-bp\n(StuI-XhoI) fragment of pBam/Xho4Z as a template. Plasmid Bam/Xho4Z con-\ntains a 1.8-kb BamHI-XhoI fragment of HSV-2 DNA (genomic nucleotides\n120489 to 122276) cloned into the multiple cloning site of pGEM-4Z (Promega).\nPlasmids were linearized with either HindIII (for HSV-1) or StuI (for HSV-2),\npuriﬁed by phenol-chloroform extraction and ethanol precipitation, and then\nincubated with digoxigenin (DIG) RNA labeling mix (Roche, Mannheim, Ger-\nmany) and either T3 (for HSV-1) or SP6 (for HSV-2) polymerase for 3 h at 37°C.\nAfter treatment with DNase I, probes were puriﬁed using G-50 Sephadex col-\numns (Roche) and dissolved into 50% formamide in diethylpyrocarbonate-\ntreated water. Probe concentrations were determined by spotting each probe\nonto a nylon membrane; visualizing with anti-DIG-AP Fab fragments (Roche),\nnitroblue tetrazolium (Roche), and 5-bromo-4-chloro-3-indolylphosphate tolui-\ndinium (Roche); and comparing the intensity of the generated signal to that of\nthe labeled control RNA (Roche).\nCombined staining by ﬂuorescent in situ hybridization and immunohisto-\nchemistry. Tissue sections were postﬁxed with 4% PFA in 0.1 M PB for 10 min\nand then sequentially washed in 2 SSC (1 SSC is 0.15 M NaCl plus 0.015 M\nsodium citrate), diethyl pyrocarbonate (DEPC)-treated water, and 0.1 M trieth-\nanolamine hydrochloride (TEA). Tissue sections were then treated with 0.25%\nacetic anhydride in 0.1 M TEA (3 min), washed in 2 SSC, and incubated with\nprehybridization buffer (50% formamide, 40 g/ml salmon sperm DNA, 5\nSSC) at 45°C for 2 h. Each riboprobe was heated to 80°C for 10 min, chilled on\nice, and then diluted to 1 g/ml in preheated (55°C) hybridization buffer (50%\nformamide, 1 Denhardt’s solution, 10 mM EDTA, 10% dextran sulfate, 0.5\nmg/ml yeast tRNA, 0.5 mg/ml salmon sperm DNA, 3 SSC). Prehybridized\ntissue was incubated with each probe at 55°C overnight. After hybridization,\ntissue was washed with 2 SSC (10 min) and treated with 20 g/ml of RNase A\nin 2 SSC at 37°C for 30 min, followed by serial washing with graduated dilutions\nof SSC at 55°C. Tissue was then equilibrated with 0.1 M PBS and incubated with\nanti-DIG-rhodamine Fab fragments (Roche) diluted in 0.1 M PBS with 1\nblocking solution (Roche) for 30 min. After washing with 0.1 M PBS for 10 min,\ntissue was blocked with 3% normal goat serum in 0.1 M PBS for 30 min and then\nincubated with either MAb A5 or KH10 at 4°C for 2 nights. Tissue was then\nwashed with 0.1 M PBS, blocked with 3% normal goat serum in 0.1 M PBS and\nincubated for 1 h with FITC-labeled goat anti-mouse immunoglobulin M (Bio-\nsource, Camarillo, CA) diluted 1:100 in 0.1 M PBS with 1% normal goat serum.\nStained tissue sections were then washed with 0.1 M PBS and cover slipped with\nVectashield mounting medium (Vector Labs).\nStatistical analysis. To determine whether there were signiﬁcant differences in\nthe pattern of ganglionic latent infection, pair-wise comparisons were made using\nthe 2 test. Since we performed multiple comparisons, a Bonferroni-adjusted P\nvalue of 0.00625 was used to assign signiﬁcance. The two-tailed Student’s t test\nand Kolmogorov-Smirnov test were used to determine whether there were sig-\nniﬁcant differences in LAT signal intensities among different neuronal cell types.\nRESULTS\nFollowing ocular inoculation with HSV-1 or HSV-2, we as-\nsayed latently infected murine TG by FISH for viral LAT\nexpression in order to determine the proportion of the latent\nviral load present in populations of neurons identiﬁable by\nMAbs A5 and KH10. In the mouse trigeminal ganglion, MAbs\nA5 and KH10 recognize neuronal populations with very little\noverlap (Table 1). Most A5-positive neurons are immunoreac-\ntive for the calcitonin gene-related peptide and the high-afﬁn-\nity nerve growth factor receptor (TrkA), whereas KH10-posi-\ntive neurons colabel with the lectin BSL-IB4, identifying them as\na population of small-diameter, glial cell line-derived neurotro-\nphic factor-responsive neurons (2, 4, 20, 26). These neurochem-\nical characteristics imply that both KH10- and A5-positive neu-\nrons transmit nociceptive stimuli but represent functionally\ndistinct sensory neuronal populations (27, 30).\nConsistent with our previous report (34), 21 days after ocular\ninoculation with HSV-1 (KOS strain), 50% of the LAT-express-\ning TG neurons were A5 positive, while only 7% expressed the\nKH10 marker (Table 2 and Fig. 1). This distribution cannot be\nexplained by the neuronal composition of the ganglion (Table 2),\nand our previous studies showed that the expression of these\nmarkers does not change following infection (34). This distribu-\ntion also cannot be explained by differential access of peripherally\ninoculated virus to the two cell types since these neurons become\nproductively infected in proportion to their representation in the\nganglion (Table 3 and see reference 34). Thus, these data suggest\nthat a greater proportion of A5-positive neurons survive acute\nganglionic infection with HSV-1 to become latently infected than\ndo KH10-positive neurons.\nWhen administered under the same experimental conditions\nas those of HSV-1, HSV-2 (333 strain) caused fatal encepha-\nlitis in 100% of infected mice. Decreasing the viral inoculum by\n5 logs prevented encephalitis but also largely prevented the\nestablishment of latent infection. Adding the viral DNA poly-\nmerase inhibitor acyclovir to the drinking water at 40 h posti-\nnoculation prevented fatal encephalitis without preventing\nganglionic infection. In mice infected with HSV-2 and treated\nwith acyclovir in this manner, 42% of LAT-expressing TG\nneurons at 21 days postinoculation were KH10 positive and\nonly 4% were A5 positive (Table 2 and Fig. 2). These data\nimply that HSV-2 latent infection of the TG is preferentially\nestablished in KH10-positive neurons, with a pattern of infec-\ntion signiﬁcantly different from that observed with HSV-1 (P \n0.000001).\nTo determine whether the two different patterns of latent\ninfection were attributable to our use of acyclovir during\nHSV-2 infection, we examined the effect of acyclovir on the\ndistribution of HSV-1 latent infection. As summarized in Table\n2, the treatment of mice with acyclovir during acute ganglionic\ninfection reduced the proportion of HSV-1 LAT-expressing\nneurons that were A5 positive (to 25%) and increased the\nproportion that were KH10 positive (to 12%), in effect shifting\nthe distribution of LAT-expressing neurons closer to the rep-\nresentation of A5- and KH10-positive neurons in uninfected\nganglia and to the distribution of productive HSV-1 infection\namong these two neuronal populations. However, the pattern\nof HSV-1 LAT expression in the TG of acyclovir-treated mice\nwas still signiﬁcantly different from that observed with HSV-2\n(P  0.000001), and from the distribution of neurons in the\nuninfected ganglion. Thus, treatment with acyclovir blunts but\ndoes not eliminate the preferential establishment of HSV-1\nlatency in A5-positive neurons.\nThe effect of acyclovir administration on the neuronal dis-\ntribution of latent HSV-1 infection argues strongly that the\ntropism of HSV-1 latent infection for A5-positive neurons is\ndependent upon viral replication and is not simply an artifact\nof differential accumulation of LAT in A5- and KH10-positive\nneurons. However, to examine this issue more closely we per-\nformed two experiments speciﬁcally addressing the question of\nwhether the differential distribution of HSV-1 and HSV-2 in\nA5- and KH10-positive neurons in latently infected ganglia was\ndue to differential accumulation of LAT in these cells.\nFirst, we carried out digital analysis of confocal microscopic\nimages of latently infected ganglionic tissue probed by immu-\nnoﬂuorescence and FISH. As seen in Fig. 3, we found no\nsigniﬁcant difference in LAT signal\nintensity (HSV-1 or\nHSV-2) among A5- and KH10-positive neurons. Although\nneurons. These differences were statistically signiﬁcant (P \n0.000001). We also noted that KH10-positive neurons are\nmore prevalent in DRG than in TG (26 versus 16%, respec-\ntively; P  0.000001).\nWe hypothesized that differences in the genotypes and phe-\nnotypes of LATs expressed by HSV-1 and HSV-2 may be\nresponsible for the different patterns of latent infection with\nthese two viruses. To explore a possible role for the viral LATs\nin the differential establishment of HSV-1 and HSV-2 latent\ninfection, we next examined the neuronal distribution of latent\nviral infection following ocular inoculation with HSV-2 333/\nLAT1. Mice infected with either the viral rescuant (HSV-2\n333/LAT1R) or the 17 strain of HSV-1 (source of the donor\nLAT) served as controls. As summarized in Table 4 HSV-2\n333/LAT1 established latent infection in the TG with an\nHSV-1 phenotype; 40% of the neurons latently infected with\nHSV-2 333/LAT1 were A5 positive, while only 2% expressed\nthe KH10 marker. In contrast, HSV-2 333/LAT1R established\nlatent infection with an HSV-2 phenotype, with a preference\nfor KH10-positive neurons. These data strongly suggest that\nthe different patterns of latent infection observed with HSV-1\nand HSV-2 are due to a viral function associated with a 2.8-kb\nportion of the LAT coding region.\nDISCUSSION\nPrimary ganglionic infection with HSV-1 is characterized by\ntwo very different patterns of viral gene expression. In some\nneurons, there is abundant expression of lytic cycle genes,\nresulting in the production of infectious virus, whereas in other\nneurons, a latent infection is established (14, 19, 28, 33). This\npattern is not random; primary sensory neurons identiﬁed by\nMAb KH10 are very permissive for productive infection with\nHSV-1, whereas neurons identiﬁed by MAb A5 serve as the\nprincipal reservoir of latent infection (34). In the current study,\nwe demonstrate that HSV-2 follows a very different pattern of\nganglionic infection, with KH10-positive neurons serving as the\nprincipal reservoir of latent infection in both the TG and lum-\nbar-sacral DRG. These data suggest that HSV-1 and HSV-2\ntake advantage of two different biological niches within the\nperipheral nervous system: niches that go beyond the simple\nboundaries of “above versus below” the waist.\nIt is almost certain that the different lactoseries glycoconju-\ngates recognized by MAbs A5 and KH10 (4, 6, 12) have little\na All mice were treated with acyclovir starting at 40 h postinoculation. Each\ndata point for the recombinant viruses reﬂects analysis of step sections taken\nthrough two blocks of infected tissue (20 ganglia). Each data point for the HSV-1\n(17) reﬂects analysis of step sections taken through one block of infected tissue\n(10 ganglia).\nthere are limits to this type of data acquisition and analysis, this\nobservation is consistent with the hypothesis that A5- and\nKH10-positive neurons are equally permissive for the accumu-\nlation of HSV-1 and HSV-2 LAT.\nSecond, we examined the trigeminal ganglia of transgenic\nmice that contain the HSV-1 LAT-expressing region (8). In\nthese mice, 7.7% (79/1,025) of the LAT-expressing neurons\nwere A5 positive and 15.4% (154/1,000) were KH10 positive,\nwhich was very similar to the proportion of all ganglionic neu-\nrons that were A5 (8.4%; 86/1,026) and KH10 positive (14.9%;\n150/1,007) in these tissues. Thus, these data are also consistent\nwith the hypothesis that A5- and KH10-positive neurons are\nequally permissive for the accumulation of HSV-1 and HSV-2\nLAT.\nTo determine whether different patterns of HSV-1 and\nHSV-2 latent infection were unique to the TG, we studied the\ndistribution of latent HSV infection in murine DRG following\nfootpad inoculation. As summarized in Table 2, the results\nwere very similar to those observed with the TG. Following\nfootpad inoculation, HSV-1 preferentially established latent\ninfection in A5-positive DRG neurons, whereas HSV-2 pref-\nerentially established latent infection in KH10-positive DRG\nto do with the differences in permissiveness of the neurons\nrecognized by these markers. It is more likely that the expres-\nsion of these markers simply correlates with differences in\ncellular protein expression that enable different neuronal pop-\nulations to differentially support or suppress the expression of\nkey viral genes, thus leading to different outcomes of infection.\nAs suggested by the immunostaining patterns in Table 1, A5-\nand KH10-positive neurons largely represent two different\nclasses of intensively studied nociceptors, small-diameter BSL-\nIB4-positive and BSL-IB4-negative ganglionic neurons; neu-\nrons with differential responsiveness to nerve growth factor\nand glial cell line-derived neurotrophic factor (20, 27). It is\ntherefore tempting to speculate that neurotrophic factor-re-\nsponsive signaling pathways may play a role in regulating the\nestablishment and maintenance of HSV latent infection. These\nconsiderations highlight the importance of studying the inter-\naction of HSV with different sensory neuronal subtypes, and\nnot just ganglia as whole, in understanding viral latency and\nreactivation.\nThe ability of acyclovir to reduce the tropism of HSV-1\nlatent infection for A5-positive neurons implies that the pref-\nerential establishment of latency in these cells is dependent\nupon viral replication. One possibility is that type-speciﬁc se-\nquences enhance viral spread to A5-positive neurons during\nestablishment of latency and that acyclovir inhibits this spread.\nAlternatively, by inhibiting viral DNA replication, acyclovir\nmight tip the course of infection in replication-permissive neu-\nrons toward latency, thus reducing the relative latent viral load\nin A5-positive neurons.\nThe HSV LAT is the only available reliable marker of la-\ntently infected neurons. While neurons may contain viral DNA\nas detected by PCR-based techniques, it is unclear whether\nviral DNA detected in this manner represents reactivation-\ncompetent latent virus or simply fragments of the viral ge-\nnome. While it is not known whether viral reactivation is re-\nstricted to LAT-producing neurons, animal data showing the\naltered recurrence phenotypes of viruses with impaired capac-\nity to produce LAT supports the conclusion that LAT produc-\ntion is a biologically relevant marker of viral latency (10, 13, 15,\n17, 22, 32).\nThe different distributions of latent HSV-1 and HSV-2 in\nmurine sensory ganglia cannot simply be a consequence of\ndifferential LAT accumulation in A5-positive and KH10-posi-\ntive neurons. First, we found no signiﬁcant difference in HSV-1\nor HSV-2 LAT signal intensity among A5- and KH10-positive\nneurons. Second, studies of HSV-1 LAT transgene expression\nin murine trigeminal ganglia revealed no evidence of differen-\ntial LAT accumulation in A5- and KH10-positive neurons.\nThird, the treatment of mice with acyclovir, a drug that inhibits\nproductive viral infection but should not affect the accumula-\ntion of LAT, shifted the distribution of LAT-positive, latently\ninfected neurons closer to the distribution of A5- and KH10-\npositive neurons in uninfected ganglia. Fourth, we previously\ndemonstrated that the ratio of LAT-expressing A5-positive\nand KH10-positive neurons was independent of the sensitivity\nof the assay used to measure LAT (34), an observation that\nwould not be expected if there were different levels of LAT\nexpression in these neuronal populations. And ﬁfth, following\nocular inoculation with a TK deletion virus where latency is the\nonly possible outcome of infection, the distribution of viral\nlatency in A5-positive and KH10-positive neurons, as assayed\nby LAT expression, closely matches the relative distribution of\nthese neurons in the TG; a result that demonstrates similar\naccumulation of LAT in A5- and KH10-positive neurons if the\nvirus has equal opportunity to establish latency in these two\nneuronal populations (Y. Imai et al., personal communica-\ntion).\nIn the course of these studies, we found that a chimeric\nHSV-2 mutant that expresses the HSV-1 LAT (HSV-2 333/\nLAT1) establishes latency primarily in A5-positive neurons,\nsimilar to HSV-1. This indicates that the difference in the\nlatency phenotype of the two viruses can be mapped to a small\nportion of the long repeat region of the viral genome, the same\nregion that codes for the stable LAT intron and which conveys\nthe phenotype of virus type-associated, site-speciﬁc recurrence\n(35). This viral function might be associated with any or all of\nthe substituted sequences, including the LAT promoter, the\nprimary 5 end sequences or the sequences coding for the LAT\nintron. It might also be associated with sequences coding in an\nantisense direction relative to LAT, such as those recently\ndescribed for AL (24) or with the overlapping 3 end of the\nICP0 coding region, although the close homology of the HSV-1\nand HSV-2 ICP0 3 amino acid sequences makes this unlikely.\nOur ﬁndings are particularly interesting in light of our pre-\nvious report of KOS 62, an HSV-1 LAT deletion virus that\nexpresses the lacZ gene under control of the HSV-1 LAT\npromoter. Although containing a 1.6-kb LAT region deletion,\nthis virus preferentially established latency (as assayed by -ga-\nlactosidase expression) in A5 neurons, similar to that with the\nwild-type virus. Together with the current ﬁndings, this sug-\ngests that critical sequences for neuron subtype-speciﬁc estab-\nlishment of latency may reside in regions not shared by the\n1.6-kb deletion in KOS 62 and the 2.8-kb substitution in HSV-2\n333/LAT1. Thus, sequences in the HSV-1 LAT promoter, the\nLAT 5 exon, or in the 3 half of the LAT intron seem likely to\nbe important for this phenotype. It also is possible that these\nﬁndings are inﬂuenced by other differences between the\nHSV-1 backbone of KOS 62 and the HSV-2 backbone of\nHSV-2 333/LAT1.\nImplication of the LAT region of the viral genome in neu-\nronal type-speciﬁc establishment of latency is intriguing since it\nappears to play an important role in both the establishment\n(25, 31) and reactivation of latent infection (13), and more\nrecent studies suggest that LAT inﬂuences neuronal survival,\npossibly via an effect on apoptosis (1, 3, 7, 23). None of the\nleading hypotheses of LAT function serve to explain the type-\nspeciﬁc role of LAT sequences either in viral reactivation phe-\nnotype or in establishment of latency. Thus, further investiga-\ntion into the mechanism of LAT action is clearly warranted.\nOur ﬁndings are consistent with a potential role for LAT\nsequences in selectively promoting latent infection in speciﬁc\ntypes of sensory neurons via an as-yet-undeﬁned mechanism.\nThe ability of LAT region sequences to direct latent infec-\ntion to speciﬁc sensory neuronal subtypes and the correspond-\ning differences in the abilities of each virus to recur from\ndifferent anatomical locations (16) imply that HSV derives an\nadvantage in recurrence frequency from establishing latency in\ndifferent neurons in each anatomical location, A5-positive neu-\nrons in trigeminal ganglia and KH10-positive neurons in lum-\nbar-sacral ganglia. The increased prevalence of KH10-positive\nneurons in lumbar-sacral dorsal root ganglia suggests one\nmechanism by which this phenotype could inﬂuence type-spe-\nciﬁc recurrence patterns.",
         "",
         "Herpes Simplex Virus Type 2 (HSV-2) Establishes Latent Infection in a Different Population of Ganglionic Neurons than HSV-1: Role of Latency-Associated Transcripts",
         "None"
        ],
        [
         "43",
         "001f88f4411f8465f0bd251668a585766474a2b5",
         "How do cell number and size determine brain size? Here, we show that, in the order Rodentia, increased size of the cerebral cortex, cerebellum, and remaining areas across six species is achieved through greater numbers of neurons of larger size, and much greater numbers of nonneuronal cells of roughly invariant size, such that the ratio between total neuronal and nonneuronal mass remains constant across species. Although relative cerebellar size remains stable among rodents, the number of cerebellar neurons increases with brain size more rapidly than in the cortex, such that the cerebellar fraction of total brain neurons increases with brain size. In contrast, although the relative cortical size increases with total brain size, the cortical fraction of total brain neurons remains constant. We propose that the faster increase in average neuronal size in the cerebral cortex than in the cerebellum as these structures gain neurons and the rapidly increasing glial numbers that generate glial mass to match total neuronal mass at a fixed glia/neuron total mass ratio are fundamental cellular constraints that lead to the relative expansion of cerebral cortical volume across species.",
         "S. Herculano‐Houzel,B. Mota,R. Lent",
         "S. Herculano‐Houzel,B. Mota,R. Lent",
         "https://europepmc.org/articles/pmc1567708?pdf=render",
         "How do cell number and size determine brain size? Here, we show\nthat, in the order Rodentia, increased size of the cerebral cortex,\ncerebellum, and remaining areas across six species is achieved\nthrough greater numbers of neurons of larger size, and much\ngreater numbers of nonneuronal cells of roughly invariant size,\nsuch that the ratio between total neuronal and nonneuronal mass\nremains constant across species. Although relative cerebellar size\nremains stable among rodents, the number of cerebellar neurons\nincreases with brain size more rapidly than in the cortex, such that\nthe cerebellar fraction of total brain neurons increases with brain\nsize. In contrast, although the relative cortical size increases with\ntotal brain size, the cortical fraction of total brain neurons remains\nconstant. We propose that the faster increase in average neuronal\nsize in the cerebral cortex than in the cerebellum as these structures\ngain neurons and the rapidly increasing glial numbers that gener-\nate glial mass to match total neuronal mass at a ﬁxed glianeuron\ntotal mass ratio are fundamental cellular constraints that lead to\nthe relative expansion of cerebral cortical volume across species.",
         "",
         "species (1, 2), and, although the cellular composition of the\nbrain is one of the major determinants of its computational\ncapacities (3), little is known about how the cellular composition\nvaries with brain size. What are the cellular scaling rules that\ndetermine brain allometry? How do numbers of neuronal and\nnonneuronal cells contribute to structure size? What are the\nrelative contributions of these cells across species of different brain\nsizes?\nGlia are said to be the most numerous cell type in the brain (4,\n5) and to be 10–50 times more numerous than neurons in humans\n(6). Evidence for this assertion, however, is scant. The ratio between\nthe total number of glial and neuronal cells (glianeuron ratio) in\nthe cerebral cortex has been shown to increase with brain size (1,\n7). However, the numeric expansion of glial cells relative to neurons\nseems to contradict the observation that the neuronal need for\nmetabolic support remains similar across species (8). Data on how\nneuronal and glial cell sizes scale with brain size might help solve\nthis discrepancy, but such data are lacking in the literature.\nNot much is known, either, about the total numbers of neuronal\nand nonneuronal cells in the brains of different species, because\nlimitations have largely restricted comparative\nmethodological\nstudies of brain anatomy to analyses of volumetric data published\nby a small number of laboratories. Strikingly, analyses of the same\ndata yield conflicting conclusions. For instance, although the\nneocortical fraction of brain volume increases from 14% in basal\ninsectivores to 80% in humans (9), the cerebellar fraction varies\nlittle across individuals of different mammalian orders (10), a\ndiscrepancy that the latter authors take to argue against the\nhypothesis that the cerebellum works in service of the neocortex.\nHowever, cerebellar and cerebral cortices increase concertedly in\nboth surface area (11) and volume (12), parameters used in the\nliterature to indicate computational capacity, and this evidence has\nbeen taken to suggest a functional dependence of one structure on\nthe other. A conciliatory view holds that the cerebellum and\nneocortex evolved together but with the cerebellum evolving more\nslowly than the neocortex (12).\nThese conflicting interpretations demonstrate that cortical vol-\nume and surface, although informative measurements and widely\nused in the literature, particularly in relation to intelligence, cog-\nnitive abilities, and versatility (13–15), are only indirect indicators of\ncomputational capacity. A comparison of direct estimates of num-\nbers of neurons in these structures might be a better tool to clarify\nthe issue of how cerebral and cerebellar cortices are structurally and\nfunctionally related.\nHere, we use the isotropic fractionator (16), a nonstereological\nmethod, to determine total numbers of neuronal and nonneuronal\ncells in the cerebral cortex, cerebellum, and remaining areas of the\nbrain and to examine how they scale across six species of the same\norder, Rodentia, from mouse to the giant Amazonian capybara.\nResults\nAcross the six rodent species we studied, body mass varies 1,000-\nfold, from 40 g in the mouse to 40 kg in the capybara, whereas\nbrain mass varies by a factor of 200, accompanied by a smaller\nincrease of 45 times in total number of cells, and an even smaller\n22-times increase in the total number of neurons (Table 1). We find\nthat body mass (MBO) in adult individuals relates to total brain mass\n(MBR) by a power function, such that MBR  MBO\n0.773 (see Fig. 5,\nwhich is published as supporting information on the PNAS web\nsite), in accordance with observations in rodents (17) and other\nmammalian orders (18), reporting that body size increases faster\nthan brain size in phylogeny. Total brain mass also increases as a\npower function of the total number of cells (NC) in the brain of these\nspecies (MBR  NC\n1.323) and can be expressed as a steeper power\nfunction of the total number of neurons with exponent 1.587.\nFractional Distribution of Cells and Neurons. The relative size of the\ncerebral cortex, expressed as the fraction of brain mass contained\nin this structure, increases significantly with brain size in these\nrodent species, whereas relative cerebellar mass remains constant,\nand relative mass of the remaining structures decreases, as de-\nscribed in the literature (10, 19) (Fig. 1). Interestingly, the fraction\nof neurons contained in each of these divisions behaves differently:\nRegardless of total brain size, the cerebral cortex in all six species\ncontains a relatively stable 17.8  3.4% of all brain neurons (Fig.\n1a,   0.171; P  0.5212, Spearman rank correlation). In\ncontrast, larger cerebella hold increasing fractions of total brain\nneurons, from 59.0  5.0% in the mouse to 72.4  3.3% in the\ncapybara (Fig. 1b,   0.739; P  0.0057), whereas the fraction of\ntotal brain neurons contained in the remaining areas decreases with\nincreasing brain size (Fig. 1c,  0.807; P  0.0025). These data\nsuggest that different neuronal scaling rules apply to the cerebral\ncortex, cerebellum, and remaining brain structures.\nStructure Size as a Function of the Number of Neuronal and Nonneu-\nronal Cells. We next assessed cellular scaling rules for these three\nbrain divisions by examining how tissue mass covaries with NC\nacross species. Tissue mass varies as a power function of total NC\n(Fig. 2a) and of the number of nonneuronal cells (Fig. 2b, NNN),\nwith similar exponents for the cerebral cortex, cerebellum, and\nremaining areas. In contrast, the mass of the cerebral cortex (MCX)\nand that of remaining areas (MRA) increase more steeply than\ncerebellar mass (MCB) as a power function of their respective\nnumbers of neurons (Fig. 2c, NN). This difference is simultaneous\nwith the addition of greater numbers of nonneuronal cells to the\ncerebral cortex and remaining areas than to the cerebellum as these\nstructures gain neurons (Fig. 2d). The latter finding is also evident\nas a prominent decrease in the percentage of cells that are neurons\n(and presumably as an increase in the glianeuron index) with\nincreasing mass of the cerebral cortex and remaining areas but not\nso prominent for the cerebellum (Fig. 2e).\nCell Densities. As expected from the larger increase in brain mass\nthan in the number of neurons, neuronal density decreases in these\nsix rodent species as a power function of brain size with an exponent\nthat is slightly more negative than those found in the literature (1,\n20, 21) and at rates that are similar in the cerebral cortex and\nremaining areas and larger in these than in the cerebellum (Fig. 2f ).\nSeparate analysis of total cellular and neuronal densities shows that\nsimilar values are found for the cerebral cortex and remaining areas\nof equivalent mass, and densities scale with structure mass, with\nexponents that are similar for the two structures (Fig. 3a). The\ncerebellum is distinctive in that it has larger cellular and neuronal\ndensities than the cerebral cortex or remaining areas of equivalent\nmass, and cerebellar neuronal density scales less steeply than that\nof the other structures (Fig. 3b). Remarkably, the cerebellum,\ncerebral cortex, and remaining areas share similar nonneuronal cell\ndensities, which vary very little with structure size (Fig. 3c). These\nfindings suggest that the addition of nonneuronal cells follows the\nsame principle and contributes equally in all structures and species\ntoward final structure size.\nNeuronal density scales less steeply than neuronal number across\nbrains of different sizes. Although the latter increases 23 times from\nthe mouse to the capybara, neuronal density in all structures\ndecreases by a factor 10-fold. An even more marked difference is\nobserved for nonneuronal cells, whose 86-fold increase in number\nfrom the mouse to the capybara is accompanied by a modest\ndecrease in nonneuronal cell density of 2-fold (see Table 2 and\nSupporting Appendix, which are published as supporting informa-\ntion on the PNAS web site). This difference in the steepness of\nscaling between neuronal and nonneuronal cells indicates that the\nlatter vary much less in size than do neuronal cells across species,\na possibility that we addressed next.\nRelative Contribution of Neuronal and Nonneuronal Cell Size to Brain\nMass. Methodological limitations make difficult the direct mea-\nsurement of cell sizes across species, but a mathematical analysis\noffers initial estimates of how neuronal and nonneuronal cell size\nscales as a function of cell number. If the mass of any given cell is\nconsidered to include all cellular processes and surrounding extra-\ncellular space, then the mass of any brain division can be expressed\nas the product of NC and the average cell mass (MC). We observe\nthat MC increases in the cerebral cortex, cerebellum, and remaining\nareas as similar power functions of NCCX, NCCB, and NCRA (slope\n  0.274, 0.301, and 0.266, respectively; see Methods).\nOther groups have shown that neuronal density (1, 20, 21)\nscales with brain mass, and our observation that it also scales\ndirectly as a power function of number of neurons suggests that\nneuronal size, as defined above, also scales with neuron number.\nIndeed, neuronal cell size and soma volume are known to scale\nas functions of brain mass (1, 20–22) and, thus, presumably, as\nwe show, with the number of neurons. Assuming, therefore, that\nthe average neuron (MN) and nonneuronal (MNN) cell mass scale\nas power functions of NN and NNN, respectively, their relative\ncontributions to the total size of each structure can be estimated\nmathematically (see Methods). Our data suggest that average\nneuronal size increases as similar functions of NN in the cerebral\ncortex and remaining areas (  0.760 and 0.772, respectively)\nand less rapidly with increasing NN in the cerebellum ( 0.370).\nIn contrast, average nonneuronal cell size is estimated to in-\ncrease very little, and by similar exponents, as a function of NNN\nin all three structures (  0.114, 0.063, and 0.176 for the\ncerebral cortex, cerebellum, and remaining areas, respectively).\nInterestingly, because we observe that NN and NNN are related by\na power function, it can be demonstrated mathematically that the\nratio between total neuronal (NN MN) and total nonneuronal\n(NNN MNN) mass for each structure, and for the whole brain, is\nconstant among species, regardless of brain size (see supporting\ninformation).\nBecause structure mass scales as the product of variations in NN\nand MN, the relative contribution of these two variables (NN\nMN) to the final size of any structure can be estimated as another\npower function of NN with exponent  1  , yielding N  0.240\nand 0.228 for cerebral cortex and remaining areas, respectively, and\nN  0.630 for cerebellum, where   0 indicates similar relative\ncontributions of neuronal number and average neuronal size, and\n  1 indicates no contribution of average neuronal size. The\ncoefficients obtained indicate that not only do changes in MN\ncontribute less than changes in NN to final structure size but also\nthat the larger the increase in NN between two species, the smaller\nthe relative contribution of MN to final structure size, particularly\nfor the cerebral cortex and remaining areas. When the same\nrationale is applied to nonneuronal cells, it is found that the relative\ncontribution of NNN and MNN (NNNMNN) to final structure size\nvaries as a power function of NNN in the structures with larger\nexponents (NN  0.886, 0.937, and 0.824 for the cerebral cortex,\ncerebellum, and remaining areas, respectively). In contrast to the N\nobtained, the larger values of NN indicate that increasing numbers\nof nonneuronal cells are added to all three structures in the absence\nof large changes in average nonneuronal cell size.\nComparative Scaling of Cerebral Cortex and Cerebellum. Direct\ncomparison across structures shows that, whereas the cerebral\ncortex gains mass faster than the other structures (Fig. 4a), the total\nnumber of neurons increases more rapidly in the cerebellum than\nin the cerebral cortex and remaining areas (Fig. 4b), as could be\nexpected from the fractional distribution of neurons across the\ncerebral cortex, cerebellum, and the remaining areas (Fig. 1).\nWhereas the total number of neurons in the cerebral cortex\nincreases as a power function of the number of neurons in all other\nbrain structures, with an exponent of 0.941 (P  0.0001), the\nexponent relating the total number of neurons in the cerebellum to\nthe number in all other brain structures is 1.181 (P  0.0001) (Fig.\n4c). Thus, although the cerebral cortex with the subjacent white\nmatter increases rapidly in fractional volume and becomes the\ndominant brain structure in relative volume as overall brain size\nincreases across species (10, 19), the cerebral cortex gains neurons,\nat most, isometrically with the remainder of the brain and maintains\na constant fraction of all brain neurons. In contrast, the cerebellum,\nalthough maintaining a constant fractional volume of the brain,\ngains neurons faster than both the cerebral cortex and the remain-\ning structures and encloses gradually larger fractions of all brain\nneurons as brain size increases.\nDiscussion\nThe cellular scaling rules for rodent brains that we describe here\nindicate that larger brains are built with more neurons and even\nlarger numbers of nonneuronal cells, particularly in the cerebral\ncortex and remaining areas;\nin addition, as their numbers\nincrease, neurons increase in size, but nonneuronal cells main-\ntain their average size relatively constant. Between mice and\ncapybaras, we show that the 278-fold difference in mass of the\ncerebral cortex is due to a 22-fold increase in the number of\nneurons and a 154-fold increase in the number of nonneuronal\ncells. Given the power laws that relate cell number and average\ncell size, we estimate that the 22 times more numerous cerebral\ncortical neurons are, on average, 10 times larger in the capybara\nthan in the mouse, whereas the 154 times more numerous\nnonneuronal cells are only 1.8 times larger. In the cerebellum, on\nthe other hand, the 28 times more numerous neurons are, on\naverage, only 3 times larger in capybara than in mouse, whereas\nthe 82 times more numerous nonneuronal cells become, on\naverage, only 1.3 times larger in the bigger rodent.\nIt will be interesting to see whether data on neuronal and\nnonneuronal cell size will match our estimates as they become\navailable by direct measurement. According to our estimates, a\nrodent brain with a human-sized cerebellum would be expected to\nhave 900 times more cerebellar nonneuronal cells that are, on\naverage, only 1.5 times larger than in the mouse cerebellum. Recent\nmeasurements of human astrocytes have shown that they are only\n3 times larger than mouse astrocytes,§ which seems to be good\nevidence that nonneuronal cell size indeed changes very little with\ncell number. Similarly, Purkinje cells are 50 times more numerous\nin the human (24) than in the rat cerebellum (25) and would,\ntherefore, be expected to be 4.2 times larger in the former,\naccording to our estimates; in the literature, these cells have been\nfound to have a 2.5 times bigger perykaryon (26), which falls close\nenough to the expected value, given that the arborizations were not\nconsidered in that study.\nSo far, we have been unable to analyze separate neuronal\nsubpopulations. Although our estimates refer to the total popula-\ntion of neurons, the literature suggests that similar scaling laws\napply to some aspects of different neuronal populations within each\nstructure. For example, neuronal density scales in all three cere-\nbellar layers, of strikingly different cellular compositions, as power\nfunctions of cerebellar weight with similar exponents, even though\nthe ratio between numbers of granular and Purkinje cells changes\n(27). To the extent that the organization of both cerebral (28) and\ncerebellar (29) cortices is modular, with the same basic laminar,\ntangential, and connective organization throughout in columnar\nfunctional units along the surface, some aspects of cellular allom-\netry can be expected to scale at similar rates among neuronal\nsubtypes as more modules are added, and it will be informative to\ndetermine which aspects do and which do not.\nOur data suggest that cell number and mass are regulated in such\na way that the ratio between total neuronal and total nonneuronal\nmass is kept constant in the brain, even though the glianeuron\nindex increases along with brain mass (30, 31). This finding means\nthat the significant increase in neuronal size as the number of\nneurons increases is compensated by the addition of much larger\nnumbers of nonneuronal cells of only slightly larger size, such that\na 2-fold increase in total neuronal mass is accompanied by an equal\n2-fold increase in total nonneuronal mass and yields a 2-fold\nincrease in brain size. The overall mass constraint suggested by our\ndata is compatible with the recent notion that glial cells serve as\ndynamic regulators of neuronal production, function, and pheno-\ntype and organize brain tissue into functional compartments (8). On\nthe other hand, an increase in the number of glial units would favor\na growing participation of these cells in neural computation, as has\nbeen proposed recently (32, 33), without compromising their role\nin regulatory and support functions. The constant neuronal\nnonneuronal mass ratio also settles the apparent discrepancy\nbetween the numeric expansion of glial cells compared with neu-\nrons, whereas the neuronal need for metabolic support remains\nsimilar across species (8). Because increasing numbers of neurons\nresult in larger brain size, these neurons can be expected to increase\nin size, with not only larger somata but, particularly, longer pro-\ncesses to maintain long-distance connectivity as brain size increases.\nCompared with neurons, glial cells act locally, so as the brain grows,\ntheir sustained small size relative to increasingly larger neurons\nwould be compensated by the addition of larger numbers of glial\ncells, thereby maintaining a constant balance between total neu-\nronal and nonneuronal mass in the brain, which we propose to be\na major mechanism in driving changes in brain size.\nThis constant balance could be achieved economically if the\nincreased neuronal proliferation that has been proposed to drive\ncortical growth across species (34) led, during the development of\neach individual, to the generation of appropriate numbers of glial\ncells through the regulation of gliogenesis, which is largely postnatal\n(35), according to the number of neurons generated in each\nstructure. Gliogenesis is known to be regulated by neuronal activity\n(31), and it has also been demonstrated that glial precursor prolif-\neration is density-dependent and ceases once a steady-state glial\ndensity has been achieved, most likely by cell–cell contact inhibition\n(36). In this direction, it is remarkable that the three brain regions\nin all of the rodent species that we examined share similar non-\nneuronal cell densities, despite containing very different numbers\nof neurons, which is what one should observe if gliogenesis were\nregulated by contact inhibition among proliferating precursors and\nnewly added glial cells as they invade structures that are, in early\ndevelopment, composed mostly of neurons. Thus, we suggest that\ncontinued gliogenesis until confluency is reached in a formerly\npurely neuronal tissue is a likely candidate mechanism by which\nneuronal and nonneuronal cell numbers are related and by which\nthe ratio between total neuronal and nonneuronal mass could be\nkept constant across species. It will be interesting to estimate total\nneuronal and nonneuronal mass in different brain structures and\nsee how manipulations of neuron number affect both the number\nand mass of nonneuronal cells.\nOur data indicate another unexpected trend in cellular scaling of\nthe brain, namely that the relative volumetric expansion of the\ncerebral cortex does not reflect a corresponding expansion of its\nnumber of neurons relative to the whole brain, because this\nrelationship seems to be fixed at 18% in all species examined. In\ncontrast, the stable relative cerebellar volume across species masks\nan increase in its relative number of neurons. We propose that this\ndifferential expansion of relative volume and number of neurons\nbetween the cerebral cortex and cerebellum has its origin in the\nlarger increase in the average size of neurons in the cerebral cortex\n(a power function of exponent 0.760 of neuron number) compared\nwith the increase in average cerebellar neuronal size (a power\nfunction of exponent 0.370 of neuron number), matched by a\ncorresponding increase in total nonneuronal mass. This finding is in\ngood agreement with the known architecture of cerebral and\ncerebellar cortices, the former composed of relatively large num-\nbers of neurons with large cell bodies and extensively arborized\nprocesses that span long distances (28) and the latter composed\nmostly of much smaller neurons with a single, long, and compar-\natively local arborization (29). Along this line, it is interesting to\nnote that the larger the increase in NN between two species, the\nsmaller the relative contribution of MN to final structure size,\nparticularly for the cerebral cortex and remaining areas, consistent\nwith a strong selective pressure against increased neuronal size, lest\nthe brain becomes too large too fast as it gains neurons (37).\nVolume and surface measurements of the cerebral cortex and\ncerebellum have been widely used in the literature as indicators of\nthe computational capacity of these structures (10–12), with the\ntacit assumption that they vary directly according to the number of\nneurons in these modular structures (28, 29), for instance, as more\ncolumns of a same total number of neurons are added (38).\nHowever, our results of a constant relative number of neurons in the\ncortex and an increasing number in the cerebellum show that\nchanges in volume cannot be used as proxy for changes in number\nof neurons or of nonneuronal cells. As long as neurons are\nconsidered the functional integrative units of nervous tissue, infer-\nences of computational power of brain structures should take their\nnumber into account, given that this should be a far more direct\nindicator of computational capacity than structure volume and\nsurface, which are inflated in different ways by nonneuronal cells\nand connecting fibers of larger caliber. The total number of neurons\nshould also be more direct an indicator of computational capacity\nthan total neuronal mass, because increasing soma size, the extent\nof arborizations, and the number of synapses allows neurons to\nintegrate more information but not to separate them as different\ncomputational units unless there were also more neurons in the\nbrain. Clearly, however, the number of neurons in a structure is not\nthe sole determinant of its computational capacity, because it must\nbe combined with synaptic numbers and density, variables that we\ndo not investigate here.\nIt could still be argued that the expected increase in computa-\ntional power of larger cerebral cortices does relate to an increase in\nabsolute number of neurons across species. However, this argument\nwould have to be logically extended to the cerebellum, which is\nmodular in its structure like the cortex. We find, nevertheless, that\nthe cerebellum actually gains more neurons than does the cerebral\ncortex as these structures become larger in rodents, a finding that\nwould, by extension,\nimply that the cerebellum actually gains\ncomputational power faster than the cerebral cortex as brains\nbecome larger.\nIt is important to realize that the current view of encephalization\nand neocorticalization as adaptive and selected traits in evolution\n(13) is based on volumetric relationships that we show not to hold\nat the cellular level of brain composition and, therefore, may not be\nreliable indicators of function. It is interesting to wonder how the\nview of neocorticalization would be different today if studies of\ncellular scaling rules had been available earlier. Given the increas-\ning concentration of neurons in the cerebellum, perhaps we would\ninstead deal today with concepts of cerebellarization of the brain.\nThe notion that, from a neuronal point of view, the cerebellum\nmight be favored with a greater increase in number of computa-\ntional units than the cerebral cortex itself as both structures increase\ncoordinately in size and in number of neurons is consistent with the\nnewly recognized role of the cerebellum as a structure that accu-\nmulates sensory, motor, emotional, and cognitive functions, in\naddition to on-line error analysis on inputs derived from the\ncerebral cortex (18, 39).\nRather than its number of neurons, we find that the increased size\nof the cerebral cortex reflects more closely the changes in its total\nnumbers of nonneuronal cells. In light of the recent discoveries on\nthe functional contribution of glial cells to intercellular signaling, in\nconcert with neurons (32, 33), this finding raises the possibility that\nit is the increasing numerical predominance of glial cells in the\ncerebral cortex that accounts for the expected increase in the\ncomputational power of larger brains.\nIn any case, given the discrepancies we show between the\nprevious volumetric and the cellular scaling rules for rodent brains,\nour findings suggest that the cognitive and ecological significance of\nspecies differences in brain size should be reevaluated by examining\ndirectly the numbers of neuronal and nonneuronal cells and not just\nvolume or surface area.\nMethods\nAnimals. Adult male Swiss mice (Mus musculus, n  4), golden\nhamsters (Mesocrycetus auratus, n  2), and Wistar rats (Rattus\nnorvegicus, n  4; data from this species appeared in a previous\nmethods paper) (16) were bred at the colony at the Instituto de\nCieˆncias Biome´dicas, Universidade Federal do Rio de Janeiro\nand were 2–5 months of age by the time of the experiments. Two\nyoung adult, 1-month-old male guinea pigs (Cavia porcellus)\nwere supplied by Carlos Virgı´nio Coda´ (Rio de Janeiro). Three\nadult agoutis (Dasyprocta primnolopha, one female) and two\nadult female capybaras (Hydrochoerus hydrochoeris) of undeter-\nmined age were donated by the Federal University of Para´ and\nthe Brazilian Institute for Environmental Protection (IBAMA,\nBele´m, Brazil), respectively.\nDissection. All animals were killed by inhalation of ether, weighed,\nand perfused transcardially with 0.9% saline, followed by 4%\nphosphate-buffered paraformaldehyde. The brains were removed\nfrom the skulls by using the foramen magnum as the lower limit and\ndissected free of dura mater and superficial blood vessels, weighed,\nand postfixed for 2 weeks to 12 months by immersion in 4%\nphosphate-buffered paraformaldehyde. The cerebellum was dis-\nsected by cutting the cerebellar peduncles at the surface of the\nbrainstem. The cerebral cortex in all animals was defined as all\ncortical regions lateral to the olfactory tract, including the hip-\npocampus, and was dissected from each hemisphere by peeling the\ncerebral cortex away from the striatum and other subcortical\nstructures under a stereomicroscope. All other brain structures,\nincluding the olfactory bulb, were pooled and processed together as\n‘‘remaining areas.’’ In mice, hamsters, rats, and guinea pigs, the two\nhemispheres were counted together; in agoutis and capybaras, only\none of the two hemispheres was processed, and results were\nmultiplied by 2 to estimate numbers for the whole brain.\nIsotropic Fractionator. Total numbers of cells, neurons, and non-\nneuronal cells were estimated as described by using the isotropic\nfractionator method (16). Briefly, each dissected brain division\nis turned into an isotropic suspension of isolated nuclei of known,\ndefined volume, kept homogeneous by agitation. The total\nnumber of nuclei in suspension and, therefore, the total number\nof cells in the original tissue, is estimated by determining the\ndensity of nuclei in small aliquots stained with the fluorescent\nDNA marker DAPI (4-6-diamidino-2-phenylindole dihydro-\nchloride) under the microscope. Once the total cell number was\nknown, the proportion of neurons was determined by immuno-\ncytochemical detection of neuronal nuclear antigen (NeuN),\nexpressed in all nuclei of most neuronal cell types and not in\nnonneuronal cells (23, 40). Estimates of the proportion of\nNeuN-positive nuclei are considered reliable, because the coef-\nficient of variation among animals of the same species is typically\n0.15 (see supporting information). Numbers of nonneuronal\ncells are derived by subtraction.\nData Analysis. All statistical analyses and regressions were per-\nformed in Statview (SAS, Cary, NC). Nonparametric Spearman\nrank correlations were calculated for fractional distributions of\nmass, number of cells, and number of neurons across species\nbecause there was no way to ascertain that the variables were\nnormally distributed.\nEstimates of Scaling of Average Cell Size. Considering that total brain\nmass (MBR) can be expressed as the product of the total number of\ncells (NcBR) and their average size (or mass, Mc), here considered\nto include all cellular processes and surrounding extracellular space,\nit is possible to estimate the relative contributions of cell number\nand average cell size to final brain mass. Because MBR  NcBR \nMc, the slopes () of the power function Mc  Nc\nfor each structure\ncan be inferred from the slopes () of the power functions relating\nM  Nc\n in Fig. 3, as     1.\nEstimates of Scaling of Average Neuronal and Nonneuronal Cell Size.\nThe contributions of average neuronal (MN) and nonneuronal\n(MNN) cell mass to the total size of each structure can also be\nestimated by assuming that MN and MNN vary as power functions\nof neuronal (NN) and nonneuronal (NNN) cell number, respectively,\nwhich is warranted by estimates of changes in cell size with brain\nmass (1, 19, 20) and by the finding that neuronal density decreases\nwith increasing brain mass (1, 20, 21). Based on this assumption, if\nM  NN  MN  NNN  MNN and the exponents relating M \nare known, the slopes () of the power functions\nNN\n can also be inferred from the slopes\nMN  NN\n() in Fig. 3, as     1 (see supporting information).\nand M  Nnn\nWe thank Jean-Christophe Houzel [Universidade Federal do Rio de\nJaneiro (UFRJ)], Cristo´vam Picanc¸o-Diniz, Ivanira Dias, Carlomagno\nBahia (Universidade Federal do Para´, Brazil), and the Instituto Brasileiro\ndo Meio Ambiente for providing agouti and capybara specimens; Cristiano\nNascimento Costa for help in some experiments; Adiel Nascimento for\nexpert animal care; Elizabeth de Moraes and Antoˆnia Lima Carvalho for\ntechnical assistance; Vivaldo Moura-Neto, Fla´via Gomes, and Jose´ Garcia\nAbreu (UFRJ) for providing materials; and Robert H. Miller for bringing\ndensity-dependent gliogenesis to our attention. This work was supported by\nthe Fundac¸a˜o Carlos Chagas Filho de Amparo a` Pesquisa do Estado do Rio\nde Janeiro Primeiros Projetos and CNPq-PRONEX.\n1. Stolzenburg, J. U., Reichenbach, A & Neumann, M. (1989) Glia 2, 78–84.\n2. Tower, D. B. (1954) J. Comp. Neurol. 101, 19–51.\n3. Williams, R. W. & Herrup, K. (1988) Annu. Rev. Neurosci. 11, 423–453.\n4. Doetsch, F. (2003) Nat. Neurosci. 6, 1127–1134.\n5. Nishiyama, A., Yang, Z. & Butt, A. (2005) J. Anat. 207, 687–693.\n6. Kandel, E., Schwartz, J. & Jessel, T. (2000) Principles of Neural Science (McGraw–Hill, New\n7. Hawkins, A. & Olszewski, J. (1957) Science 126, 76–77.\n8. Nedergaard, M., Ransom, B. & Goldman, S. A. (2003) Trends Neurosci. 26, 523–530.\n9. Frahm, H. D., Stephan, H. & Stephan, M. (1982) J. Hirnforsch. 23, 375–389.\n10. Clark, D. A., Mitra, P. P. & Wang, S. S. (2001) Nature 411, 189–193.\n11. Sultan, F. (2002) Nature 415, 133–134.\n12. Barton, R. A. (2002) Nature 415, 134–135.\n13. Jerison, H. J. (1985) Philos. Trans. R. Soc. London Ser. B 308, 21–35.\n14. Reader, S. M. & Laland, K. N. (2002) Proc. Natl. Acad. Sci. USA 99, 4436–4441.\n15. Sol, D., Duncan, R. P., Blackburn, T. M., Cassey, P. & Lefebvre, L. (2005) Proc. Natl. Acad.\n16. Herculano-Houzel, S. & Lent, R. (2005) J. Neurosci. 25, 2518–2521.\n17. Fox, J. H. & Wilczynski, W. (1986) Brain Behav. Evol. 28, 157–169.\n18. Martin, R. D. (1981) Nature 293, 57–60.\n19. Stephan, H., Frahm, H. & Baron, G. (1981) Folia Primatol. 35, 1–29.\n20. Haug, H. (1987) Am. J. Anat. 180, 126–142.\n21. Tower, D. B. & Elliot, K. A. C. (1952) Am. J. Physiol. 168, 747–759.\n22. Shariff, G. A. (1953) J. Comp. Neurol. 98, 381–400.\n23. Gittins, R. & Herrison, P. J. (2004) Brain Res. Bull. 63, 155–160.\n24. Andersen, B. B., Korbo, L. & Pakkenberg, B. (1992) J. Comp. Neurol. 326, 549–560.\n25. Korbo, L., Andersen, B. B., Ladefoged, O. & Moller, A. (1993) Brain Res. 609, 262–268.\n26. Korbo, L. & Andersen, B. B. (1995) Neuroscience 69, 151–158.\n27. Lange, W. (1975) Cell Tiss. Res. 157, 115–124.\n28. Douglas, R. J. & Martin, K. A. C. (2004) Annu. Rev. Neurosci. 27, 419–451.\n29. Leiner, H. C., Leiner A. L. & Dow, R. S. (1991) Behav. Brain Res. 44, 113–128.\n30. Tower, D. B. & Young, O. M. (1973) J. Neurochem. 20, 269–278.\n31. Reichenbach, A. (1989) Glia 2, 71–77.\n32. Allen, N. J. & Barres, B. A. (2005) Curr. Opin. Neurobiol. 15, 542–548.\n33. Volterra, A. & Meldolesi, J. (2005) Nat. Rev. Neurosci. 6, 626–640.\n34. Rakic, P. (1995) Trends Neurosci. 18, 383–388.\n35. Sauvageot, C. M. & Stiles, C. D. (2002) Curr. Opin. Neurobiol. 12, 244–249.\n36. Zhang, H. & Miller, R. H. (1996) J. Neurosci. 16, 6886–6895.\n37. Harrison, K. H., Hof, P. R. & Wang, S. S. (2002) J. Neurocytol. 31, 289–298.\n38. Rockel, A. J., Hiorns, R. W. & Powell, T. P. (1980) Brain 103, 221–244.\n39. Andreasen, N. C., O’Leary, D. S., Cizadlo, T., Arndt, S., Rezai, K., Boles Ponto, L. L.,",
         "",
         "",
         "broken link",
         "How do cell number and size determine brain size? Here, we show\nthat, in the order Rodentia, increased size of the cerebral cortex,\ncerebellum, and remaining areas across six species is achieved\nthrough greater numbers of neurons of larger size, and much\ngreater numbers of nonneuronal cells of roughly invariant size,\nsuch that the ratio between total neuronal and nonneuronal mass\nremains constant across species. Although relative cerebellar size\nremains stable among rodents, the number of cerebellar neurons\nincreases with brain size more rapidly than in the cortex, such that\nthe cerebellar fraction of total brain neurons increases with brain\nsize. In contrast, although the relative cortical size increases with\ntotal brain size, the cortical fraction of total brain neurons remains\nconstant. We propose that the faster increase in average neuronal\nsize in the cerebral cortex than in the cerebellum as these structures\ngain neurons and the rapidly increasing glial numbers that gener-\nate glial mass to match total neuronal mass at a ﬁxed glianeuron\ntotal mass ratio are fundamental cellular constraints that lead to\nthe relative expansion of cerebral cortical volume across species.\nspecies (1, 2), and, although the cellular composition of the\nbrain is one of the major determinants of its computational\ncapacities (3), little is known about how the cellular composition\nvaries with brain size. What are the cellular scaling rules that\ndetermine brain allometry? How do numbers of neuronal and\nnonneuronal cells contribute to structure size? What are the\nrelative contributions of these cells across species of different brain\nsizes?\nGlia are said to be the most numerous cell type in the brain (4,\n5) and to be 10–50 times more numerous than neurons in humans\n(6). Evidence for this assertion, however, is scant. The ratio between\nthe total number of glial and neuronal cells (glianeuron ratio) in\nthe cerebral cortex has been shown to increase with brain size (1,\n7). However, the numeric expansion of glial cells relative to neurons\nseems to contradict the observation that the neuronal need for\nmetabolic support remains similar across species (8). Data on how\nneuronal and glial cell sizes scale with brain size might help solve\nthis discrepancy, but such data are lacking in the literature.\nNot much is known, either, about the total numbers of neuronal\nand nonneuronal cells in the brains of different species, because\nlimitations have largely restricted comparative\nmethodological\nstudies of brain anatomy to analyses of volumetric data published\nby a small number of laboratories. Strikingly, analyses of the same\ndata yield conflicting conclusions. For instance, although the\nneocortical fraction of brain volume increases from 14% in basal\ninsectivores to 80% in humans (9), the cerebellar fraction varies\nlittle across individuals of different mammalian orders (10), a\ndiscrepancy that the latter authors take to argue against the\nhypothesis that the cerebellum works in service of the neocortex.\nHowever, cerebellar and cerebral cortices increase concertedly in\nboth surface area (11) and volume (12), parameters used in the\nliterature to indicate computational capacity, and this evidence has\nbeen taken to suggest a functional dependence of one structure on\nthe other. A conciliatory view holds that the cerebellum and\nneocortex evolved together but with the cerebellum evolving more\nslowly than the neocortex (12).\nThese conflicting interpretations demonstrate that cortical vol-\nume and surface, although informative measurements and widely\nused in the literature, particularly in relation to intelligence, cog-\nnitive abilities, and versatility (13–15), are only indirect indicators of\ncomputational capacity. A comparison of direct estimates of num-\nbers of neurons in these structures might be a better tool to clarify\nthe issue of how cerebral and cerebellar cortices are structurally and\nfunctionally related.\nHere, we use the isotropic fractionator (16), a nonstereological\nmethod, to determine total numbers of neuronal and nonneuronal\ncells in the cerebral cortex, cerebellum, and remaining areas of the\nbrain and to examine how they scale across six species of the same\norder, Rodentia, from mouse to the giant Amazonian capybara.\nResults\nAcross the six rodent species we studied, body mass varies 1,000-\nfold, from 40 g in the mouse to 40 kg in the capybara, whereas\nbrain mass varies by a factor of 200, accompanied by a smaller\nincrease of 45 times in total number of cells, and an even smaller\n22-times increase in the total number of neurons (Table 1). We find\nthat body mass (MBO) in adult individuals relates to total brain mass\n(MBR) by a power function, such that MBR  MBO\n0.773 (see Fig. 5,\nwhich is published as supporting information on the PNAS web\nsite), in accordance with observations in rodents (17) and other\nmammalian orders (18), reporting that body size increases faster\nthan brain size in phylogeny. Total brain mass also increases as a\npower function of the total number of cells (NC) in the brain of these\nspecies (MBR  NC\n1.323) and can be expressed as a steeper power\nfunction of the total number of neurons with exponent 1.587.\nFractional Distribution of Cells and Neurons. The relative size of the\ncerebral cortex, expressed as the fraction of brain mass contained\nin this structure, increases significantly with brain size in these\nrodent species, whereas relative cerebellar mass remains constant,\nand relative mass of the remaining structures decreases, as de-\nscribed in the literature (10, 19) (Fig. 1). Interestingly, the fraction\nof neurons contained in each of these divisions behaves differently:\nRegardless of total brain size, the cerebral cortex in all six species\ncontains a relatively stable 17.8  3.4% of all brain neurons (Fig.\n1a,   0.171; P  0.5212, Spearman rank correlation). In\ncontrast, larger cerebella hold increasing fractions of total brain\nneurons, from 59.0  5.0% in the mouse to 72.4  3.3% in the\ncapybara (Fig. 1b,   0.739; P  0.0057), whereas the fraction of\ntotal brain neurons contained in the remaining areas decreases with\nincreasing brain size (Fig. 1c,  0.807; P  0.0025). These data\nsuggest that different neuronal scaling rules apply to the cerebral\ncortex, cerebellum, and remaining brain structures.\nStructure Size as a Function of the Number of Neuronal and Nonneu-\nronal Cells. We next assessed cellular scaling rules for these three\nbrain divisions by examining how tissue mass covaries with NC\nacross species. Tissue mass varies as a power function of total NC\n(Fig. 2a) and of the number of nonneuronal cells (Fig. 2b, NNN),\nwith similar exponents for the cerebral cortex, cerebellum, and\nremaining areas. In contrast, the mass of the cerebral cortex (MCX)\nand that of remaining areas (MRA) increase more steeply than\ncerebellar mass (MCB) as a power function of their respective\nnumbers of neurons (Fig. 2c, NN). This difference is simultaneous\nwith the addition of greater numbers of nonneuronal cells to the\ncerebral cortex and remaining areas than to the cerebellum as these\nstructures gain neurons (Fig. 2d). The latter finding is also evident\nas a prominent decrease in the percentage of cells that are neurons\n(and presumably as an increase in the glianeuron index) with\nincreasing mass of the cerebral cortex and remaining areas but not\nso prominent for the cerebellum (Fig. 2e).\nCell Densities. As expected from the larger increase in brain mass\nthan in the number of neurons, neuronal density decreases in these\nsix rodent species as a power function of brain size with an exponent\nthat is slightly more negative than those found in the literature (1,\n20, 21) and at rates that are similar in the cerebral cortex and\nremaining areas and larger in these than in the cerebellum (Fig. 2f ).\nSeparate analysis of total cellular and neuronal densities shows that\nsimilar values are found for the cerebral cortex and remaining areas\nof equivalent mass, and densities scale with structure mass, with\nexponents that are similar for the two structures (Fig. 3a). The\ncerebellum is distinctive in that it has larger cellular and neuronal\ndensities than the cerebral cortex or remaining areas of equivalent\nmass, and cerebellar neuronal density scales less steeply than that\nof the other structures (Fig. 3b). Remarkably, the cerebellum,\ncerebral cortex, and remaining areas share similar nonneuronal cell\ndensities, which vary very little with structure size (Fig. 3c). These\nfindings suggest that the addition of nonneuronal cells follows the\nsame principle and contributes equally in all structures and species\ntoward final structure size.\nNeuronal density scales less steeply than neuronal number across\nbrains of different sizes. Although the latter increases 23 times from\nthe mouse to the capybara, neuronal density in all structures\ndecreases by a factor 10-fold. An even more marked difference is\nobserved for nonneuronal cells, whose 86-fold increase in number\nfrom the mouse to the capybara is accompanied by a modest\ndecrease in nonneuronal cell density of 2-fold (see Table 2 and\nSupporting Appendix, which are published as supporting informa-\ntion on the PNAS web site). This difference in the steepness of\nscaling between neuronal and nonneuronal cells indicates that the\nlatter vary much less in size than do neuronal cells across species,\na possibility that we addressed next.\nRelative Contribution of Neuronal and Nonneuronal Cell Size to Brain\nMass. Methodological limitations make difficult the direct mea-\nsurement of cell sizes across species, but a mathematical analysis\noffers initial estimates of how neuronal and nonneuronal cell size\nscales as a function of cell number. If the mass of any given cell is\nconsidered to include all cellular processes and surrounding extra-\ncellular space, then the mass of any brain division can be expressed\nas the product of NC and the average cell mass (MC). We observe\nthat MC increases in the cerebral cortex, cerebellum, and remaining\nareas as similar power functions of NCCX, NCCB, and NCRA (slope\n  0.274, 0.301, and 0.266, respectively; see Methods).\nOther groups have shown that neuronal density (1, 20, 21)\nscales with brain mass, and our observation that it also scales\ndirectly as a power function of number of neurons suggests that\nneuronal size, as defined above, also scales with neuron number.\nIndeed, neuronal cell size and soma volume are known to scale\nas functions of brain mass (1, 20–22) and, thus, presumably, as\nwe show, with the number of neurons. Assuming, therefore, that\nthe average neuron (MN) and nonneuronal (MNN) cell mass scale\nas power functions of NN and NNN, respectively, their relative\ncontributions to the total size of each structure can be estimated\nmathematically (see Methods). Our data suggest that average\nneuronal size increases as similar functions of NN in the cerebral\ncortex and remaining areas (  0.760 and 0.772, respectively)\nand less rapidly with increasing NN in the cerebellum ( 0.370).\nIn contrast, average nonneuronal cell size is estimated to in-\ncrease very little, and by similar exponents, as a function of NNN\nin all three structures (  0.114, 0.063, and 0.176 for the\ncerebral cortex, cerebellum, and remaining areas, respectively).\nInterestingly, because we observe that NN and NNN are related by\na power function, it can be demonstrated mathematically that the\nratio between total neuronal (NN MN) and total nonneuronal\n(NNN MNN) mass for each structure, and for the whole brain, is\nconstant among species, regardless of brain size (see supporting\ninformation).\nBecause structure mass scales as the product of variations in NN\nand MN, the relative contribution of these two variables (NN\nMN) to the final size of any structure can be estimated as another\npower function of NN with exponent  1  , yielding N  0.240\nand 0.228 for cerebral cortex and remaining areas, respectively, and\nN  0.630 for cerebellum, where   0 indicates similar relative\ncontributions of neuronal number and average neuronal size, and\n  1 indicates no contribution of average neuronal size. The\ncoefficients obtained indicate that not only do changes in MN\ncontribute less than changes in NN to final structure size but also\nthat the larger the increase in NN between two species, the smaller\nthe relative contribution of MN to final structure size, particularly\nfor the cerebral cortex and remaining areas. When the same\nrationale is applied to nonneuronal cells, it is found that the relative\ncontribution of NNN and MNN (NNNMNN) to final structure size\nvaries as a power function of NNN in the structures with larger\nexponents (NN  0.886, 0.937, and 0.824 for the cerebral cortex,\ncerebellum, and remaining areas, respectively). In contrast to the N\nobtained, the larger values of NN indicate that increasing numbers\nof nonneuronal cells are added to all three structures in the absence\nof large changes in average nonneuronal cell size.\nComparative Scaling of Cerebral Cortex and Cerebellum. Direct\ncomparison across structures shows that, whereas the cerebral\ncortex gains mass faster than the other structures (Fig. 4a), the total\nnumber of neurons increases more rapidly in the cerebellum than\nin the cerebral cortex and remaining areas (Fig. 4b), as could be\nexpected from the fractional distribution of neurons across the\ncerebral cortex, cerebellum, and the remaining areas (Fig. 1).\nWhereas the total number of neurons in the cerebral cortex\nincreases as a power function of the number of neurons in all other\nbrain structures, with an exponent of 0.941 (P  0.0001), the\nexponent relating the total number of neurons in the cerebellum to\nthe number in all other brain structures is 1.181 (P  0.0001) (Fig.\n4c). Thus, although the cerebral cortex with the subjacent white\nmatter increases rapidly in fractional volume and becomes the\ndominant brain structure in relative volume as overall brain size\nincreases across species (10, 19), the cerebral cortex gains neurons,\nat most, isometrically with the remainder of the brain and maintains\na constant fraction of all brain neurons. In contrast, the cerebellum,\nalthough maintaining a constant fractional volume of the brain,\ngains neurons faster than both the cerebral cortex and the remain-\ning structures and encloses gradually larger fractions of all brain\nneurons as brain size increases.\nDiscussion\nThe cellular scaling rules for rodent brains that we describe here\nindicate that larger brains are built with more neurons and even\nlarger numbers of nonneuronal cells, particularly in the cerebral\ncortex and remaining areas;\nin addition, as their numbers\nincrease, neurons increase in size, but nonneuronal cells main-\ntain their average size relatively constant. Between mice and\ncapybaras, we show that the 278-fold difference in mass of the\ncerebral cortex is due to a 22-fold increase in the number of\nneurons and a 154-fold increase in the number of nonneuronal\ncells. Given the power laws that relate cell number and average\ncell size, we estimate that the 22 times more numerous cerebral\ncortical neurons are, on average, 10 times larger in the capybara\nthan in the mouse, whereas the 154 times more numerous\nnonneuronal cells are only 1.8 times larger. In the cerebellum, on\nthe other hand, the 28 times more numerous neurons are, on\naverage, only 3 times larger in capybara than in mouse, whereas\nthe 82 times more numerous nonneuronal cells become, on\naverage, only 1.3 times larger in the bigger rodent.\nIt will be interesting to see whether data on neuronal and\nnonneuronal cell size will match our estimates as they become\navailable by direct measurement. According to our estimates, a\nrodent brain with a human-sized cerebellum would be expected to\nhave 900 times more cerebellar nonneuronal cells that are, on\naverage, only 1.5 times larger than in the mouse cerebellum. Recent\nmeasurements of human astrocytes have shown that they are only\n3 times larger than mouse astrocytes,§ which seems to be good\nevidence that nonneuronal cell size indeed changes very little with\ncell number. Similarly, Purkinje cells are 50 times more numerous\nin the human (24) than in the rat cerebellum (25) and would,\ntherefore, be expected to be 4.2 times larger in the former,\naccording to our estimates; in the literature, these cells have been\nfound to have a 2.5 times bigger perykaryon (26), which falls close\nenough to the expected value, given that the arborizations were not\nconsidered in that study.\nSo far, we have been unable to analyze separate neuronal\nsubpopulations. Although our estimates refer to the total popula-\ntion of neurons, the literature suggests that similar scaling laws\napply to some aspects of different neuronal populations within each\nstructure. For example, neuronal density scales in all three cere-\nbellar layers, of strikingly different cellular compositions, as power\nfunctions of cerebellar weight with similar exponents, even though\nthe ratio between numbers of granular and Purkinje cells changes\n(27). To the extent that the organization of both cerebral (28) and\ncerebellar (29) cortices is modular, with the same basic laminar,\ntangential, and connective organization throughout in columnar\nfunctional units along the surface, some aspects of cellular allom-\netry can be expected to scale at similar rates among neuronal\nsubtypes as more modules are added, and it will be informative to\ndetermine which aspects do and which do not.\nOur data suggest that cell number and mass are regulated in such\na way that the ratio between total neuronal and total nonneuronal\nmass is kept constant in the brain, even though the glianeuron\nindex increases along with brain mass (30, 31). This finding means\nthat the significant increase in neuronal size as the number of\nneurons increases is compensated by the addition of much larger\nnumbers of nonneuronal cells of only slightly larger size, such that\na 2-fold increase in total neuronal mass is accompanied by an equal\n2-fold increase in total nonneuronal mass and yields a 2-fold\nincrease in brain size. The overall mass constraint suggested by our\ndata is compatible with the recent notion that glial cells serve as\ndynamic regulators of neuronal production, function, and pheno-\ntype and organize brain tissue into functional compartments (8). On\nthe other hand, an increase in the number of glial units would favor\na growing participation of these cells in neural computation, as has\nbeen proposed recently (32, 33), without compromising their role\nin regulatory and support functions. The constant neuronal\nnonneuronal mass ratio also settles the apparent discrepancy\nbetween the numeric expansion of glial cells compared with neu-\nrons, whereas the neuronal need for metabolic support remains\nsimilar across species (8). Because increasing numbers of neurons\nresult in larger brain size, these neurons can be expected to increase\nin size, with not only larger somata but, particularly, longer pro-\ncesses to maintain long-distance connectivity as brain size increases.\nCompared with neurons, glial cells act locally, so as the brain grows,\ntheir sustained small size relative to increasingly larger neurons\nwould be compensated by the addition of larger numbers of glial\ncells, thereby maintaining a constant balance between total neu-\nronal and nonneuronal mass in the brain, which we propose to be\na major mechanism in driving changes in brain size.\nThis constant balance could be achieved economically if the\nincreased neuronal proliferation that has been proposed to drive\ncortical growth across species (34) led, during the development of\neach individual, to the generation of appropriate numbers of glial\ncells through the regulation of gliogenesis, which is largely postnatal\n(35), according to the number of neurons generated in each\nstructure. Gliogenesis is known to be regulated by neuronal activity\n(31), and it has also been demonstrated that glial precursor prolif-\neration is density-dependent and ceases once a steady-state glial\ndensity has been achieved, most likely by cell–cell contact inhibition\n(36). In this direction, it is remarkable that the three brain regions\nin all of the rodent species that we examined share similar non-\nneuronal cell densities, despite containing very different numbers\nof neurons, which is what one should observe if gliogenesis were\nregulated by contact inhibition among proliferating precursors and\nnewly added glial cells as they invade structures that are, in early\ndevelopment, composed mostly of neurons. Thus, we suggest that\ncontinued gliogenesis until confluency is reached in a formerly\npurely neuronal tissue is a likely candidate mechanism by which\nneuronal and nonneuronal cell numbers are related and by which\nthe ratio between total neuronal and nonneuronal mass could be\nkept constant across species. It will be interesting to estimate total\nneuronal and nonneuronal mass in different brain structures and\nsee how manipulations of neuron number affect both the number\nand mass of nonneuronal cells.\nOur data indicate another unexpected trend in cellular scaling of\nthe brain, namely that the relative volumetric expansion of the\ncerebral cortex does not reflect a corresponding expansion of its\nnumber of neurons relative to the whole brain, because this\nrelationship seems to be fixed at 18% in all species examined. In\ncontrast, the stable relative cerebellar volume across species masks\nan increase in its relative number of neurons. We propose that this\ndifferential expansion of relative volume and number of neurons\nbetween the cerebral cortex and cerebellum has its origin in the\nlarger increase in the average size of neurons in the cerebral cortex\n(a power function of exponent 0.760 of neuron number) compared\nwith the increase in average cerebellar neuronal size (a power\nfunction of exponent 0.370 of neuron number), matched by a\ncorresponding increase in total nonneuronal mass. This finding is in\ngood agreement with the known architecture of cerebral and\ncerebellar cortices, the former composed of relatively large num-\nbers of neurons with large cell bodies and extensively arborized\nprocesses that span long distances (28) and the latter composed\nmostly of much smaller neurons with a single, long, and compar-\natively local arborization (29). Along this line, it is interesting to\nnote that the larger the increase in NN between two species, the\nsmaller the relative contribution of MN to final structure size,\nparticularly for the cerebral cortex and remaining areas, consistent\nwith a strong selective pressure against increased neuronal size, lest\nthe brain becomes too large too fast as it gains neurons (37).\nVolume and surface measurements of the cerebral cortex and\ncerebellum have been widely used in the literature as indicators of\nthe computational capacity of these structures (10–12), with the\ntacit assumption that they vary directly according to the number of\nneurons in these modular structures (28, 29), for instance, as more\ncolumns of a same total number of neurons are added (38).\nHowever, our results of a constant relative number of neurons in the\ncortex and an increasing number in the cerebellum show that\nchanges in volume cannot be used as proxy for changes in number\nof neurons or of nonneuronal cells. As long as neurons are\nconsidered the functional integrative units of nervous tissue, infer-\nences of computational power of brain structures should take their\nnumber into account, given that this should be a far more direct\nindicator of computational capacity than structure volume and\nsurface, which are inflated in different ways by nonneuronal cells\nand connecting fibers of larger caliber. The total number of neurons\nshould also be more direct an indicator of computational capacity\nthan total neuronal mass, because increasing soma size, the extent\nof arborizations, and the number of synapses allows neurons to\nintegrate more information but not to separate them as different\ncomputational units unless there were also more neurons in the\nbrain. Clearly, however, the number of neurons in a structure is not\nthe sole determinant of its computational capacity, because it must\nbe combined with synaptic numbers and density, variables that we\ndo not investigate here.\nIt could still be argued that the expected increase in computa-\ntional power of larger cerebral cortices does relate to an increase in\nabsolute number of neurons across species. However, this argument\nwould have to be logically extended to the cerebellum, which is\nmodular in its structure like the cortex. We find, nevertheless, that\nthe cerebellum actually gains more neurons than does the cerebral\ncortex as these structures become larger in rodents, a finding that\nwould, by extension,\nimply that the cerebellum actually gains\ncomputational power faster than the cerebral cortex as brains\nbecome larger.\nIt is important to realize that the current view of encephalization\nand neocorticalization as adaptive and selected traits in evolution\n(13) is based on volumetric relationships that we show not to hold\nat the cellular level of brain composition and, therefore, may not be\nreliable indicators of function. It is interesting to wonder how the\nview of neocorticalization would be different today if studies of\ncellular scaling rules had been available earlier. Given the increas-\ning concentration of neurons in the cerebellum, perhaps we would\ninstead deal today with concepts of cerebellarization of the brain.\nThe notion that, from a neuronal point of view, the cerebellum\nmight be favored with a greater increase in number of computa-\ntional units than the cerebral cortex itself as both structures increase\ncoordinately in size and in number of neurons is consistent with the\nnewly recognized role of the cerebellum as a structure that accu-\nmulates sensory, motor, emotional, and cognitive functions, in\naddition to on-line error analysis on inputs derived from the\ncerebral cortex (18, 39).\nRather than its number of neurons, we find that the increased size\nof the cerebral cortex reflects more closely the changes in its total\nnumbers of nonneuronal cells. In light of the recent discoveries on\nthe functional contribution of glial cells to intercellular signaling, in\nconcert with neurons (32, 33), this finding raises the possibility that\nit is the increasing numerical predominance of glial cells in the\ncerebral cortex that accounts for the expected increase in the\ncomputational power of larger brains.\nIn any case, given the discrepancies we show between the\nprevious volumetric and the cellular scaling rules for rodent brains,\nour findings suggest that the cognitive and ecological significance of\nspecies differences in brain size should be reevaluated by examining\ndirectly the numbers of neuronal and nonneuronal cells and not just\nvolume or surface area.\nMethods\nAnimals. Adult male Swiss mice (Mus musculus, n  4), golden\nhamsters (Mesocrycetus auratus, n  2), and Wistar rats (Rattus\nnorvegicus, n  4; data from this species appeared in a previous\nmethods paper) (16) were bred at the colony at the Instituto de\nCieˆncias Biome´dicas, Universidade Federal do Rio de Janeiro\nand were 2–5 months of age by the time of the experiments. Two\nyoung adult, 1-month-old male guinea pigs (Cavia porcellus)\nwere supplied by Carlos Virgı´nio Coda´ (Rio de Janeiro). Three\nadult agoutis (Dasyprocta primnolopha, one female) and two\nadult female capybaras (Hydrochoerus hydrochoeris) of undeter-\nmined age were donated by the Federal University of Para´ and\nthe Brazilian Institute for Environmental Protection (IBAMA,\nBele´m, Brazil), respectively.\nDissection. All animals were killed by inhalation of ether, weighed,\nand perfused transcardially with 0.9% saline, followed by 4%\nphosphate-buffered paraformaldehyde. The brains were removed\nfrom the skulls by using the foramen magnum as the lower limit and\ndissected free of dura mater and superficial blood vessels, weighed,\nand postfixed for 2 weeks to 12 months by immersion in 4%\nphosphate-buffered paraformaldehyde. The cerebellum was dis-\nsected by cutting the cerebellar peduncles at the surface of the\nbrainstem. The cerebral cortex in all animals was defined as all\ncortical regions lateral to the olfactory tract, including the hip-\npocampus, and was dissected from each hemisphere by peeling the\ncerebral cortex away from the striatum and other subcortical\nstructures under a stereomicroscope. All other brain structures,\nincluding the olfactory bulb, were pooled and processed together as\n‘‘remaining areas.’’ In mice, hamsters, rats, and guinea pigs, the two\nhemispheres were counted together; in agoutis and capybaras, only\none of the two hemispheres was processed, and results were\nmultiplied by 2 to estimate numbers for the whole brain.\nIsotropic Fractionator. Total numbers of cells, neurons, and non-\nneuronal cells were estimated as described by using the isotropic\nfractionator method (16). Briefly, each dissected brain division\nis turned into an isotropic suspension of isolated nuclei of known,\ndefined volume, kept homogeneous by agitation. The total\nnumber of nuclei in suspension and, therefore, the total number\nof cells in the original tissue, is estimated by determining the\ndensity of nuclei in small aliquots stained with the fluorescent\nDNA marker DAPI (4-6-diamidino-2-phenylindole dihydro-\nchloride) under the microscope. Once the total cell number was\nknown, the proportion of neurons was determined by immuno-\ncytochemical detection of neuronal nuclear antigen (NeuN),\nexpressed in all nuclei of most neuronal cell types and not in\nnonneuronal cells (23, 40). Estimates of the proportion of\nNeuN-positive nuclei are considered reliable, because the coef-\nficient of variation among animals of the same species is typically\n0.15 (see supporting information). Numbers of nonneuronal\ncells are derived by subtraction.\nData Analysis. All statistical analyses and regressions were per-\nformed in Statview (SAS, Cary, NC). Nonparametric Spearman\nrank correlations were calculated for fractional distributions of\nmass, number of cells, and number of neurons across species\nbecause there was no way to ascertain that the variables were\nnormally distributed.\nEstimates of Scaling of Average Cell Size. Considering that total brain\nmass (MBR) can be expressed as the product of the total number of\ncells (NcBR) and their average size (or mass, Mc), here considered\nto include all cellular processes and surrounding extracellular space,\nit is possible to estimate the relative contributions of cell number\nand average cell size to final brain mass. Because MBR  NcBR \nMc, the slopes () of the power function Mc  Nc\nfor each structure\ncan be inferred from the slopes () of the power functions relating\nM  Nc\n in Fig. 3, as     1.\nEstimates of Scaling of Average Neuronal and Nonneuronal Cell Size.\nThe contributions of average neuronal (MN) and nonneuronal\n(MNN) cell mass to the total size of each structure can also be\nestimated by assuming that MN and MNN vary as power functions\nof neuronal (NN) and nonneuronal (NNN) cell number, respectively,\nwhich is warranted by estimates of changes in cell size with brain\nmass (1, 19, 20) and by the finding that neuronal density decreases\nwith increasing brain mass (1, 20, 21). Based on this assumption, if\nM  NN  MN  NNN  MNN and the exponents relating M \nare known, the slopes () of the power functions\nNN\n can also be inferred from the slopes\nMN  NN\n() in Fig. 3, as     1 (see supporting information).\nand M  Nnn\nWe thank Jean-Christophe Houzel [Universidade Federal do Rio de\nJaneiro (UFRJ)], Cristo´vam Picanc¸o-Diniz, Ivanira Dias, Carlomagno\nBahia (Universidade Federal do Para´, Brazil), and the Instituto Brasileiro\ndo Meio Ambiente for providing agouti and capybara specimens; Cristiano\nNascimento Costa for help in some experiments; Adiel Nascimento for\nexpert animal care; Elizabeth de Moraes and Antoˆnia Lima Carvalho for\ntechnical assistance; Vivaldo Moura-Neto, Fla´via Gomes, and Jose´ Garcia\nAbreu (UFRJ) for providing materials; and Robert H. Miller for bringing\ndensity-dependent gliogenesis to our attention. This work was supported by\nthe Fundac¸a˜o Carlos Chagas Filho de Amparo a` Pesquisa do Estado do Rio\nde Janeiro Primeiros Projetos and CNPq-PRONEX.\n1. Stolzenburg, J. U., Reichenbach, A & Neumann, M. (1989) Glia 2, 78–84.\n2. Tower, D. B. (1954) J. Comp. Neurol. 101, 19–51.\n3. Williams, R. W. & Herrup, K. (1988) Annu. Rev. Neurosci. 11, 423–453.\n4. Doetsch, F. (2003) Nat. Neurosci. 6, 1127–1134.\n5. Nishiyama, A., Yang, Z. & Butt, A. (2005) J. Anat. 207, 687–693.\n6. Kandel, E., Schwartz, J. & Jessel, T. (2000) Principles of Neural Science (McGraw–Hill, New\n7. Hawkins, A. & Olszewski, J. (1957) Science 126, 76–77.\n8. Nedergaard, M., Ransom, B. & Goldman, S. A. (2003) Trends Neurosci. 26, 523–530.\n9. Frahm, H. D., Stephan, H. & Stephan, M. (1982) J. Hirnforsch. 23, 375–389.\n10. Clark, D. A., Mitra, P. P. & Wang, S. S. (2001) Nature 411, 189–193.\n11. Sultan, F. (2002) Nature 415, 133–134.\n12. Barton, R. A. (2002) Nature 415, 134–135.\n13. Jerison, H. J. (1985) Philos. Trans. R. Soc. London Ser. B 308, 21–35.\n14. Reader, S. M. & Laland, K. N. (2002) Proc. Natl. Acad. Sci. USA 99, 4436–4441.\n15. Sol, D., Duncan, R. P., Blackburn, T. M., Cassey, P. & Lefebvre, L. (2005) Proc. Natl. Acad.\n16. Herculano-Houzel, S. & Lent, R. (2005) J. Neurosci. 25, 2518–2521.\n17. Fox, J. H. & Wilczynski, W. (1986) Brain Behav. Evol. 28, 157–169.\n18. Martin, R. D. (1981) Nature 293, 57–60.\n19. Stephan, H., Frahm, H. & Baron, G. (1981) Folia Primatol. 35, 1–29.\n20. Haug, H. (1987) Am. J. Anat. 180, 126–142.\n21. Tower, D. B. & Elliot, K. A. C. (1952) Am. J. Physiol. 168, 747–759.\n22. Shariff, G. A. (1953) J. Comp. Neurol. 98, 381–400.\n23. Gittins, R. & Herrison, P. J. (2004) Brain Res. Bull. 63, 155–160.\n24. Andersen, B. B., Korbo, L. & Pakkenberg, B. (1992) J. Comp. Neurol. 326, 549–560.\n25. Korbo, L., Andersen, B. B., Ladefoged, O. & Moller, A. (1993) Brain Res. 609, 262–268.\n26. Korbo, L. & Andersen, B. B. (1995) Neuroscience 69, 151–158.\n27. Lange, W. (1975) Cell Tiss. Res. 157, 115–124.\n28. Douglas, R. J. & Martin, K. A. C. (2004) Annu. Rev. Neurosci. 27, 419–451.\n29. Leiner, H. C., Leiner A. L. & Dow, R. S. (1991) Behav. Brain Res. 44, 113–128.\n30. Tower, D. B. & Young, O. M. (1973) J. Neurochem. 20, 269–278.\n31. Reichenbach, A. (1989) Glia 2, 71–77.\n32. Allen, N. J. & Barres, B. A. (2005) Curr. Opin. Neurobiol. 15, 542–548.\n33. Volterra, A. & Meldolesi, J. (2005) Nat. Rev. Neurosci. 6, 626–640.\n34. Rakic, P. (1995) Trends Neurosci. 18, 383–388.\n35. Sauvageot, C. M. & Stiles, C. D. (2002) Curr. Opin. Neurobiol. 12, 244–249.\n36. Zhang, H. & Miller, R. H. (1996) J. Neurosci. 16, 6886–6895.\n37. Harrison, K. H., Hof, P. R. & Wang, S. S. (2002) J. Neurocytol. 31, 289–298.\n38. Rockel, A. J., Hiorns, R. W. & Powell, T. P. (1980) Brain 103, 221–244.\n39. Andreasen, N. C., O’Leary, D. S., Cizadlo, T., Arndt, S., Rezai, K., Boles Ponto, L. L.,",
         "Astrocytes and NG2‐glia: what's in a name?;Signaling between glia and neurons: focus on synaptic plasticity;A novel splice mutation of HERG in a Chinese family with long QT syndrome;Big brains, enhanced cognition, and response of birds to novel environments.;Isotropic Fractionator: A Simple, Rapid Method for the Quantification of Total Cell and Neuron Numbers in the Brain;Neuronal circuits of the neocortex.;Neuronal density, size and shape in the human anterior cingulate cortex: a comparison of Nissl and NeuN staining;The glial identity of neural stem cells;\n New roles for astrocytes: Redefining the functional architecture of the brain;Molecular mechanisms controlling cortical gliogenesis;Social intelligence, innovation, and enhanced brain size in primates;Scaling laws in the mammalian neocortex: Does form provide clues to function?;Brain evolution (Communications arising): How did brains evolve?;Brain evolution (Communication arising): Analysis of mammalian brain architecture;Scalable architecture in mammalian brains;Density-Dependent Feedback Inhibition of Oligodendrocyte Precursor Expansion;Schizophrenia and cognitive dysmetria: a positron-emission tomography study of dysfunctional prefrontal-thalamic-cerebellar circuitry.;Development;The distributions of Purkinje cell perikaryon and nuclear volume in human and rat cerebellum with the nucleator method;A small step for the cell, a giant leap for mankind: a hypothesis of neocortical expansion during evolution;Total numbers of various cell types in rat cerebellar cortex estimated using an unbiased stereological method;A quantitative study of the human cerebellum with unbiased stereological techniques;NeuN, a neuronal specific nuclear protein in vertebrates.;The human cerebro-cerebellar system: its computing, cognitive, and language skills;Brain sizes, surfaces, and neuronal sizes of the cortex cerebri: a stereological investigation of man and his variability and a comparison with some mammals (primates, whales, marsupials, insectivores, and one elephant).;Animal intelligence as encephalization.;Relative brain size and basal metabolic rate in terrestrial vertebrates;Principles of Neural Science;The basic uniformity in structure of the neocortex.;THE ACTIVITIES OF BUTYRYLCHOLINESTERASE AND CARBONIC ANHYDRASE, THE RATE OF ANAEROBIC GLYCOLYSTS, AND THE QUESTION OF A CONSTANT DENSITY OF GLIAL CELLS IN CEREBRAL CORTICES OF VARIOUS MAMMALIAN SPECIES FROM MOUSE TO WHALE;Glia/nerve cell index for cortex of the whale.;Structural and functional organization of mammalian cerebral cortex: The correlation of neurone density with brain size. Cortical neurone density in the fin whale (Balaenoptera Physalus L.) with a note on the cortical neurone density in the Indian elephant;Cell counts in the primate cerebral cortex;Activity of acetylcholine system in cerebral cortex of various unanesthetized mammals.;Curr. Opin. Neurobiol;J. Anat;Proc. Natl. Acad. Sci. USA 102;Cell number and cell density in the cerebellar cortex of man and some other mammals;Nature;Glia:Neuron index: Review and hypothesis to account for different values in various mammals;Size and density of glial and neuronal cells within the cerebral neocortex of various insectivorian species;The control of neuron number.;Annu. Rev. Neurosci;Brain Behav. Evol;Importance de la standardisation dans la surveillance biologique des traitements par les anti-vitamines K.;Philos. Trans. R. Soc. London Ser. B;J. Hirnforsch;Comparison of brain structure volumes in Insectivora and Primates. I. Neocortex.;Folia Primatol;New and revised data on volumes of brain structures in insectivores and primates.;Nat. Rev. Neurosci. Trends Neurosci Curr. Opin. Neurobiol. J. Neurosci. J. Neurocytol;J. Comp. Neurol;Am. J. Physiol. J. Comp. Neurol. Brain Res. Bull. J. Comp. Neurol. Brain Res. Neuroscience Cell Tiss. Res. Annu. Rev. Neurosci. Behav. Brain Res. J. Neurochem",
         "Cellular scaling rules for rodent brains",
         "None"
        ],
        [
         "44",
         "002010074a1b72f890a62fdae5c3a16296071163",
         "The current model for pathogenesis of inhalation anthrax indicates that the uptake and fate of Bacillus anthracis spores in alveolar macrophages are critical to the infection process. We have employed primary macrophages, which are more efficient for spore uptake than the macrophage‐like cell line RAW264.7, to investigate spore uptake and survival. We found that at a multiplicity of infection (moi) of 5, greater than 80% of the spores of the Sterne strain containing only the pXO1 plasmid were internalized within 1 h. Within 4 h post infection, viability of internalized Sterne spores decreased to approximately 40%. Intracellular vegetative bacteria represented less than 1% of the total spore inoculum throughout the course of infection suggesting effective killing of germinated spores and/or vegetative bacteria. The Sterne spores trafficked quickly to phagolysosomes as indicated by colocalization with lysosome‐associated membrane protein 1 (LAMP1). Expression of a dominant‐negative Rab7 that blocked lysosome fusion enhanced Sterne spore survival. Addition of d‐alanine to the infection resulted in 75% inhibition of spore germination and increased survival of internalized spores of the Sterne strain and a pathogenic strain containing both the pXO1 and pXO2 plasmids. Inhibition was reversed by the addition of l‐alanine, which resumed spore germination and subsequent spore killing. Our data indicate that B. anthracis spores germinate in and are subsequently killed by primary macrophages.",
         "Haijing Hu,Q. Sa,T. Koehler,A. Aronson,Daoguo Zhou",
         "Haijing Hu,Q. Sa,T. Koehler,A. Aronson,Daoguo Zhou",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1462-5822.2006.00738.x",
         "The  current  model  for  pathogenesis  of  inhalation\nBacillus\nanthrax indicates that the uptake and fate of\nanthracis\nspores in alveolar macrophages are critical\nto  the  infection  process. We  have  employed  primary\nmacrophages,  which  are  more  efﬁcient  for  spore\nuptake than the macrophage-like cell line RAW264.7,\nto  investigate  spore  uptake  and  survival.  We  found\nthat  at  a  multiplicity  of  infection  (moi)  of  5,  greater\nthan 80% of the spores of the Sterne strain containing\nonly  the  pXO1  plasmid  were  internalized  within  1 h.\nWithin  4 h  post  infection,  viability  of  internalized\nSterne spores decreased to approximately 40%. Intra-\ncellular vegetative bacteria represented less than 1%\nof the total spore inoculum throughout the course of\ninfection  suggesting  effective  killing  of  germinated\nspores and/or vegetative bacteria. The Sterne spores\ntrafﬁcked quickly to phagolysosomes as indicated by\ncolocalization  with  lysosome-associated  membrane\nprotein 1 (LAMP1). Expression of a dominant-negative\nRab7 that blocked lysosome fusion enhanced Sterne\nspore  survival.  Addition  of\n-alanine  to  the  infection\nresulted  in  75%  inhibition  of  spore  germination  and\nincreased survival of internalized spores of the Sterne\nstrain  and  a  pathogenic  strain  containing  both  the\npXO1 and pXO2 plasmids. Inhibition was reversed by\nthe  addition  of\n-alanine,  which  resumed  spore  ger-\nL\nmination and subsequent spore killing. Our data indi-\ncate  that\nspores  germinate  in  and  are\nsubsequently killed by primary macrophages.",
         "Discussion\nMacrophages play a major role in host defence as part of\nthe  innate  immune  system  involved  in  engulﬁng  and\ndestroying  pathogenic  microorganisms  in  phagolysos-\nomes. Mice depleted of macrophages are more suscepti-\nble to B. anthracis spore infection (Cote et al., 2004; 2006)\nGerminated spores are inactivated in macrophages 1639\nand  resistance  was  restored  partially  by  injection  of\nRAW264.7 cells into the mice, indicating the essential role\nof  macrophages  in  limiting  the  anthrax  infection.  In  con-\ntrast,  depletion  of  neutrophils  had  no  such  effect  (Cote\net al., 2004; 2006).\nOur experiments were designed to examine the effec-\ntiveness  of  macrophages  in  controlling  an  infection  with\nspores. We  encountered  technical  difﬁculties  concerning\ninefﬁcient  spore  uptake  by  RAW264.7  macrophages\nincluding  signiﬁcant  spore  adhesion  to  the  outer  surface\nof  the  macrophages.  Extracellular  germination  in  the\nmedium  resulted  in  the  uptake  over  time  of  vegetative\nbacteria  making  accurate  assessment  of  intracellular\nspore  germination  and  subsequent  bacterial  outgrowth\ndifﬁcult. There are also potential problems with the use of\ngentamicin  which  enters  macrophages  (Drevets  et al.,\n1994;  Hamrick  et al.,  2003)  and  possibly  vacuoles  con-\ntaining spores.\nFor  these  reasons,  we  switched  to  mouse  primary\nmacrophages. We found the efﬁciency of spore uptake to\nbe greater than 80%, eliminating the need for either gen-\ntamicin  or  cytochalasin  D.  We  also  employed  an  inside/\noutside staining protocol which can speciﬁcally differenti-\nate  intracellular  spores/bacteria  from  those  outside. The\nstaining  and  viability  assays  were  used  to  demonstrate\nthe  rapid  trafﬁcking  of  spores,  germination  and  subse-\nquent killing.\nThe remaining viable spores at 6 h (Fig. 4, about 20%\nof the total inoculum that was still present after the addi-\ntion of L-alanine) may largely be due to those spores still\nungerminated in the medium or adhering to the macroph-\nages. If so, viable intracellular spores could be reduced to\nas low as 1–5% of the initial value. An increase of the D-\nalanine  concentration  to  5 mM  did  not  result  in  further\nprotection  of  spores,  i.e.  about  25%  of  the  spores  still\ngerminated. In contrast, 2 mM D-alanine completely inhib-\nited spore germination in DMEM medium containing 10%\nFBS (data not shown) suggesting the presence of special\ngerminants  (or  perhaps  limiting  concentrations  of  germi-\nnants)  or  conditions  within  the  macrophages. When  ger-\nmination was blocked by the addition of D-alanine, about\n75% of the spores remained dormant and viable even in\nphagolysosomes, suggesting that an L-alanine-dependent\ngermination  pathway  plays  a  major  role  within  macroph-\nages. Germination was totally inhibited when both 2 mM\nD-alanine  and  10 mM  D-histidine  were  present  (data  not\nshown). The D-histidine inhibition is consistent with a role\nfor  the gerH  operon  in  germination  within  macrophages.\nAromatic  amino  acids,  especially  L-histidine,  have  been\nreported to be required for spore germination in the pres-\nence of cultured macrophage-like cells or in macrophage-\nconditioned  medium  (Weiner  and  Hanna,  2003)  as  well\nas in primary mouse peritoneal macrophages (Kang et al.,\n2005). Our results indicate the function of more than one\n1640 H. Hu et al.\ngermination  operon  in  bone  marrow-derived  primary\nmacrophages.\nIt is not certain whether germinated spores, those that\noutgrow  to  small  vegetative  bacteria,  or  both  are  killed\ninside  macrophages. Vegetative  bacteria  added  to  mac-\nrophages were taken up rapidly, trafﬁcked to phagolysos-\nomes, and killed within 1 h (Fig. 3A). There were very few\nsmall  vegetative  bacteria  inside  infected  macrophages\nand  they  appeared  to  peak  a  few  hours  after  infection\nrather than reaching a rapid steady state value (Fig. 2). In\nvitro, outgrowth is slow relative to germination and as our\ndata  showed  that  loss  of  viability  was  detectable  within\n1 h,  it  is  most  likely  that  germinated  spores  were  the\nprimary targets. We also measured spore and vegetative\nbacteria killing in macrophages isolated from two knock-\nout  mice  (Jackson  laboratory,  Bar  Harbor,  Maine):  Cybb\nlacking NADH oxidase activity and Nos2 defective in NO\nproduction  (Lyons  et al.,  1992;  Pollock  et al.,  1995).  No\nsigniﬁcant  differences  in  spore  viability  were  observed\nwhen macrophages obtained from the parent and mutant\nmice were infected, suggesting that more than one factor\nwithin the phagolysosome is involved in spore killing.\nSpore  killing  by  primary  macrophages  was  quite  efﬁ-\ncient with less than 20% survival 4 h after the addition of\nL-alanine  (Fig. 4B).  As  mentioned  previously,  part  of  the\nsurviving spores may be outside of the macrophages so\nthat actual efﬁciency of killing with macrophages may be\neven  higher.  The  experiments  reported  here  were  done\nwith  macrophages  in  medium  containing  10%  FBS,  a\nnutrient  environment  suitable\nfor  sustaining  healthy\nmacrophages  and  for  providing  adequate  nutrients  for\nintracellular spore germination. In experiments employing\nmacrophages  suspended  in  buffer,  germination  was\ngreatly delayed (Cote et al., 2006) and thus spore survival\nwas enhanced. It would be important to determine which\nof  these  conditions  more  closely  resembles  those  in  the\nlung,  larynx  and  subsequently  in  the  lymph  nodes.  It  is\npossible that a few persistent spores may be sufﬁcient to\ninitiate an anthrax infection.\nExperimental procedures\nBacterial strains, spores and mammalian cell lines",
         "ingestion,  or  invasion  of  a  wound.\ninfection\noccurs when the dormant spores germinate and develop\ninto metabolically active vegetative bacteria which secrete\nthe  anthrax  toxin  proteins  (Dixon\n.,  1999;  Guidi-\nRontani\n., 2001; Agrawal and Pulendran, 2004). Early\nhistopathological studies showed that inhaled spores are\nefﬁciently taken up by phagocytes and transported to the\nregional lymph nodes (Barnes, 1947; Ross, 1957). Vege-\ntative bacteria eventually enter the bloodstream and may\n\nreach titers as high as 10\n., 1999). In\nper ml (Dixon\nmurine models, depletion of macrophages, in contrast to\nneutrophils, results in enhanced susceptibility to infection\nwith  spores.  Resistance  can  be  restored  by  increasing\nperitoneal macrophages levels, indicating the importance\ninfection\nof  macrophages  in  fending  off  a\n(Cote\nB. anthracis\n., 2005).\n., 2004; 2006; Bozue\ntrans\n;\ncya\nlef\nR; and the\n.,  1999).  Plasmid  pXO2  contains  the\nVirulent strains carry two large plasmids, pXO1 (186 kb)\nand pXO2 (96 kb). Plasmid pXO1 harbours a pathogenic-\nand\nity island containing the anthrax toxin genes;\npag\nX\nger\n., 1999a; Okinaka\ngermination operon (Guidi-Rontani\net al\ncapBCADE\noperon, encoding genes for biosynthesis and degradation\nof the poly-D-glutamic acid capsule (Drysdale\n., 2004;\nCandela  and  Fouet,  2005).  The  capsule  and  toxins  are\ntwo  major  virulence  factors  required  for  initiating  a  suc-\ncessful  anthrax  infection  (Leppla,  1982;  Guidi-Rontani\net al\n., 2005).\nThere have been several studies examining the fate of\nB. anthracis\nspores in both primary and cultured macroph-\n.  (2001)  reported  that\nage-like  cells.  Guidi-Rontani\nSterne  spores  (containing  only  the  pXO1\nB. anthracis\nplasmid)  germinated  but  did  not  replicate  in  either\nRAW264.7 macrophages or primary peritoneal macroph-\nages.  In  contrast,  Dixon\n.  (2000)  reported  extensive\ngermination  of  Sterne  spores  and  intracellular  growth\nfollowed  by  escape  of  the  vegetative  bacteria  from\nRAW264.7  macrophages.  Using\ntime-lapse  confocal\nmicroscopy,  Ruthel\n.  (2004)  found  that  10.8%  of  the\nSterne  spores  inside  RAW264.7  macrophages  germi-\nnated  and  grew  into  vegetative  bacteria.  Direct  viability\ntests of spores within RAW264.7 macrophages showed a\nrapid germination (of both the Sterne and Ames strains)\nand  thus  a  substantial  decline  in  heat-resistant  colony-\nforming  units  (cfu)  (Welkos\n.,  2002).  More  recently,\nthat  germination-\nKang\net al\nGerminated spores are inactivated in macrophages\nproﬁcient Sterne spores were killed much more efﬁciently\n∆\nthan germination-deﬁcient (\ngerH) spores by murine peri-\ntoneal  macrophages.  Overall,  these  studies  differed\nsomewhat  in  the  cell  lines  used  and  the  presence  of\ninhibitors such as gentamicin which was assumed to kill\nonly extracellular bacteria and cytochalasin D which was\nused  to  block  spore  and  vegetative  bacteria  uptake  into\nmacrophages. There  were  also  signiﬁcant  differences  in\nthe  methods  used  to  selectively  examine  intracellular\nspores or vegetative bacteria and these factors may have\ncontributed to the discrepancies in these reports.\net al\nWe  initially  attempted  to  resolve  these  issues  by\nemploying a very speciﬁc inside/outside staining protocol\n(Chen\n., 1996) for vegetative bacteria and spores in\ninvestigations  were\nRAW264.7  macrophages.  Our\nimpeded by inefﬁcient spore uptake and numerous tech-\nnical problems associated with internalization of extracel-\nlular  spores  and  vegetative  bacteria  by  RAW264.7\nmacrophages over the course of the experiments. We also\nhad concerns about the spore/bacterial killing capacity of\nRAW264.7 macrophages as compared with primary mac-\nrophages. We therefore switched to murine primary bone\nmarrow macrophages which proved to be very efﬁcient for\nspore  uptake. This  allowed  us  to  monitor,  without  inhibi-\ntors, intracellular events with less interference from extra-\ncellular  spores  and  bacteria  using  speciﬁc  ﬂuorescently\nlabelled antibodies. We report here evidence for the rapid\ntrafﬁcking of\nspores to phagolysosomes, fol-\nlowed by extensive germination and killing with very little\noutgrowth to vegetative bacteria.\nB. anthracis\nResults\nUptake and germination of\nRAW264.7 macrophages\nB. anthracis\nSterne spores in",
         "Bacillus anthracis Sterne 34F2 strain was obtained from P. Jack-\nson, Los Alamos Laboratories. Sterne 7702, the virulent UT500,\nand UT501 (atxA) have been described (Bourgogne et al., 2003;\nDrysdale  et al.,  2004)  and  were  used  for  the  experiments  in\nFig. 6.  To  produce  spores,  overnight  bacterial  culture  were\nstreaked  onto  NSM  (nutrient  sporulation  medium)  (Schaeffer\net al.,  1965)  agar  plates.  Spores  were  collected  2 days  after\nincubation at 37°C by washing the plates with 10 ml sterile 1 M\nKCl plus 5 mM EDTA (pH 8.0) and centrifuging at 10 000 rpm for\n10 min. The spores were washed once with 30 ml of water and\nthe pellets resuspended in water. Remaining vegetative bacteria\nwere lysed by sonication (Sterne spores only) of the suspensions\nfor  40 s,  diluted  to  30 ml  with  water  and  centrifuged.  Spores  of\nthe pathogenic strain containing both the pXO1 and pXO2 plas-\nmids were not sonicated. Spore suspensions were then loaded\nonto  a  38%  to  50%  Renograﬁn  (Braco  Diagnostics,  Princeton,\nNJ) step gradient (3 ml each) in 30 ml glass centrifuge tubes and\ncentrifuged at 8000 rpm for 50 min in a Sorvall HB4 rotor (Cieslak\net al., 1993). Spores were washed once with 30 ml of water and\nsuspended in water. Spore purity was assessed by examination\nunder  a  phase  contrast  microscope  and  was  routinely  greater\nthan 99%.\nFor infection with vegetative bacteria, B. anthracis was grown\novernight at 37°C in Brain Heart Infusion broth (BHI) (BD, Sparks,\nMD) supplemented with 0.5 M sorbitol (Sigma, St Louis, MO) (to\nkeep  vegetative  bacteria  as  short  rods).  Bacteria  were  washed\ntwice  with  an  equal  volume  of  Hank’s  balanced  salt  solution\n(HBSS) and resuspended in HBSS. Spores and vegetative bac-\nterial  titers  were  determined  with  a  Petroff  Hausser  chamber.\nSpores  were  stored  at  4°C. Vegetative  bacteria  were  prepared\nfresh for each experiment.\nThe murine macrophage-like cell line RAW264.7 (TIB-71) was\npurchased from the ATCC cell biology stock centre (Manassas,\nVA) and was maintained in DMEM containing 10% FBS (Invitro-\ngen,  Carlsbad,  CA)  at  37°C  with  5%  CO2  (Lyons  et al.,  1992).\nBMDM  were  obtained  from  femur  bone  marrow  exudates  of\nfemale A/J mice (Harland Laboratories, Indianapolis, IN) that are\nsusceptible to spores of B. anthracis Sterne strain (Welkos et al.,\n1986).  Bone  marrow  cells  were  cultured  in  RPMI  1640  supple-\nmented  with  10%  heat-inactivated  FBS,  30%  L-cell  conditioned\nmedium  and  100 U  penicillin-streptomycin  in  a  37°C  CO2  incu-\nbator  for  7 days  (Swanson  and  Isberg,  1995).  Cells  were  col-\nlected  and  washed  three  times  with  10 ml  each  of  ice-cold\nDulbecco’s  phosphate-buffered  saline  (DPBS,  Mediatech,  VA).\nCell suspensions were centrifuged at 1000 rpm for 5 min at 4°C\nbefore being seeded for infection.\nPlasmid construction and adenovirus preparation\nTo create a bicistronic viral expression vector, the DNA fragment\nencoding  the  EYFP  (Ormo  et al.,  1996)  from  pEYFP-C1  (Clon-\ntech,  Palo  Alto,  CA)  was  subcloned  into  the  BglII-KpnI  sites  of\npShuttle-CMV  (Stratagene,  La  Jolla,  CA)  to  create  pShuttle-\nCMV-EYFP. DNA fragments encoding the human wild type and\ndominant-negative  Rab7  (T22N)  were  cloned  into  the  KpnI-\nEcoRV  sites  of  pShuttle-CMV-EYFP.  Adenoviruses  expressing\nEYFP, wild-type Rab7 and Rab7 (T22N) were produced accord-\ning to the manufacturer’s instruction (Stratagene, La Jolla, CA).\nTransfection of AD293 cells (Stratagene, La Jolla, CA) was car-\nried out using the FuGENE 6 Transfection Reagent (Roche, Pen-\nzberg, Germany) following the manufacturer’s instructions.\nMacrophage infection and immunoﬂuorescent microscopy\nDifferentiated BMDM were seeded onto 24 well plates with glass\ncover slips at a density of 2 × 105 cells well−1. The following day,\nmacrophages were infected with spores or bacteria at a moi of\n5 in HBSS, centrifuged at 1000 rpm for 5 min to enhance adhe-\nsion and incubated for 1 h to facilitate uptake. Macrophages were\nthen  washed  three  times  with  phosphate-buffered  saline  (PBS)\nand incubated in DMEM supplemented with FBS. Six micromolar\ncytochalasin D (Sigma, St Louis, MO) was used only for the data\nin Fig. 4. The infection medium was changed at every time point\nto minimize the number of extracellular spores and bacteria. For\ninhibition of spore germination, the macrophages were pre-incu-\nbated  with  2 mM  D-alanine  (Sigma)  in  HBSS  for  1 h  and  main-\ntained  in  2 mM  D-alanine  during  the  infection.  Five  millimolar\nL-alanine was used to reverse the inhibition of germination.\nAfter infection, macrophages were processed for immunoﬂuo-\nrescent  staining  and  examined  with  a  Zeiss  Axiovert  200 M\ndeconvolution  microscope  as  described  previously  (Dai  et al.,\n2004).  Monoclonal  anti-spore  antibody,  EF12,  was  against  the\nmajor  exosporial  protein,  BclA  (Steichen  et al.,  2003).  Rabbit\npolyclonal  anti-bacteria  antibody,  1816,  was  produced  against\nformaldehyde-inactivated  Sterne  vegetative  bacteria  (Prosci)\n(Guidi-Rontani et al., 1999b). Anti-LAMP1 antibody, Alexa Fluor\n388, Alexa Fluor 350 and Texas Red-conjugated secondary anti-\nbodies  were  purchased  from  Molecular  Probes  (Eugene,  OR).\nImages  are  black  and  white  projections  of  z-section  slides  with\npseudocolours  added  by  Adobe  Photoshop.  Inside  and  outside\ndifferential  staining  of  spores  and  vegetative  bacteria  was  per-\nformed  as  described  (Chen  et al.,  1996).  Brieﬂy,  infected  mac-\nrophages were ﬁxed with 3.7% formaldehyde in PBS for 15 min.\nFixed cells were washed three times with PBS and blocked with\n3%  milk  in  PBS  for  30 min.  Extracellular  spores  and  bacteria\nwere stained with anti-spore or anti-bacteria antibodies followed\nby  AF488-conjugated  secondary  antibodies  for  30 min.  Stained\nsamples were then permeabilized with 0.2% Triton X-100 in PBS\nfor 5 min. Total spores and bacteria were stained with anti-spore\nor  anti-bacteria  antibodies  followed  by  Texas-Red-conjugated\nsecondary  antibodies  for  30 min.  Intracellular  spores/bacteria\nappeared  as  Red  while\nthe  extracellular  spores/bacteria\nappeared as green/yellow. When examining the LAMP1 colocal-\nization  with  spores  and  bacteria,  intracellular  spores/bacteria\nwere  stained  with  Texas-Red-conjugated  secondary  antibodies\nand extracellular spores/bacteria were identiﬁed with AF350-con-\njugated  secondary  antibodies.  LAMP1  was  stained  with  FITC\nanti-mouse  LAMP1  (BD).  Infection  media  were  changed  every\n15 min to further minimize the number of extracellular bacteria.\nAt least 1000 spores and bacteria were counted for each sample.\nValues presented are from three independent experiments with\nstandard errors shown.\nFor viability tests, macrophages (approximately 5 × 105) were\nlysed with 2.5% saponin (Mock and Fouet, 2001), and spore and\nbacterial cfu were assessed by plating dilutions on Luria–Bertani\nagar. Where indicated, samples were heated at 65°C for 30 min\nto inactivate geminated spores and vegetative bacteria. The total\nviability and heat resistant values are reported as cfu relative to\nthe cfu at time zero. All experiments were repeated three times\nwith at least two independently prepared batches of spores. Only\nexperiments  with  at  least  80%  spore  uptake  (examined  by\ninside–outside staining) are reported. Values presented are aver-\nages of the three independent experiments with standard errors\nshown.  A  two-level  Student’s  t-test  was  conducted  to  analyse\nvalues in Fig. 5 using software Orgin 7.0.",
         "We  initially  examined\nSterne  spore  uptake\nB. anthracis\nand  germination  in  the  RAW264.7  cell  line  using  an\ninside/outside staining protocol for both spores and veg-\netative  bacteria  as  described  in\nExperimental  proce-\n.  We  found  that  spore  uptake  was  very  inefﬁcient\ndures\nand  that  there  was  extensive  adherence  of  spores  and\nvegetative  cells  to  the  outside  of  the  macrophages.  In\nagreement  with  a  previous  report  (Welkos\n.,  2002),\nadherent  spores  and  vegetative  bacteria  were  taken  up\ncontinuously  over  several  hours  even  in  the  presence  of\ngentamicin (Fig. 1A). Initial spore uptake averaged about\n24%  of  the  total  spore  inoculum  with  a  multiplicity  of\ninfection  (moi)  of  5.  The  uptake  efﬁciency  increased  to\nabout 50% after 4–6 h (Fig. 1B). By 4 h, there was exten-\nsive  germination  and  vegetative  cell  outgrowth  with  long\nchains  of  bacteria  extending  across  several  macroph-\nages. These long chains of bacteria were located prima-\nrily  outside  the  RAW264.7  macrophages  and  in  some\ncases,  part  of  the  chain  appeared  to  be  inside  probably\ndue  to  uptake  after  germination  and  outgrowth  in  the\nmedium  (Fig. 1A  and  data  not  shown).  It  is  likely  that\nsome of the vegetative bacilli inside the RAW264.7 cells\nmay  have  germinated  in  the  medium  with  subsequent\nuptake  into  the  macrophages.  Addition  of  6\nM  cytocha-\nlasin  D  resulted  in  a  dramatic  reduction  in  intracellular\nvegetative bacteria (1–4% of the total inside spores plus\nvegetative  cells)  6 h  after  infection  (data  not  shown).\nThese  vegetative  bacteria  remained  very  small  with  no\nevidence  of  elongation  or  division.  The  lower  value  in\nthe  presence  of  cytochalasin  D  is  consistent  with  the\nnotion  that  there  had  been  continuous  uptake  of  spores\nand  vegetative  bacteria  by  RAW264.7  macrophages  in\nthe  absence  of  this  inhibitor.  We  therefore  switched  to\nprimary  murine  bone  marrow-derived  macrophages\n(BMDM)  from  A/J  mice  known  to  be  sensitive  to\nB. anthracis\net al\nUptake and germination of\nmurine BMDM\nB. anthracis\nSterne spores in\nMurine  BMDM  were  infected  with\n0.8\n0.7\n0.6\n0.5\n0.4\n0.3\n0.2\n0.1\nspores  (moi  of  5)  for  1 h  and  the  number  of  spores  and\nbacteria were enumerated using the inside–outside differ-\nential  staining  protocol  described  above  (Fig. 2A).  More\nthan  80%  of  the  spores  were  taken  up  by  the  primary\nmacrophages  after  this  incubation  period  (data  not\nshown). There appeared to be an initial drop in the number\nof  intracellular  spores  and  it  remained  relatively  stable\nthereafter (Fig. 2B). To determine if the germinated spores\ngrew into vegetative cells in primary macrophages, intra-\ncellular bacteria were monitored by the inside/outside dif-\nferential staining protocol. We observed very few and only\nsmall vegetative intracellular bacteria (Fig. 2A and C) and\n1% of the total intracellular spores plus\nthey peaked at\ncells 4–6 h after infection. All of the intracellular vegetative\nbacteria  remained  small  with  no  evidence  for  elongation\nor division (Fig. 2A).\nBacillus anthracis\nbacteria are trafﬁcked to the lysosomes\nspores and exogenously added\nB. anthracis\nThe decrease in intracellular spores can be attributed to\nthe  killing  of  intact  spores,  germinated  spores  or  out-\ngrown  bacteria.  Vegetative\nSterne  bacteria\nwere  efﬁciently  taken  up  by  BMDM  and  the  number\ndecreased  rapidly  (Fig. 3A)  indicating  that  these  primary\nB. anthracis\nmacrophages  were  very  efﬁcient  in  killing\nSterne  vegetative  bacteria. This  is  consistent  with  previ-\nous  ﬁndings  that  Sterne  vegetative  bacteria  are  highly\nattenuated  in  an  A/J  mouse  model  (Welkos  and  Fried-\nlander, 1988).\nB. anthracis\nthe  primary  compartment  where\ningested  particles  are  degraded  within  macrophages. To\ninvestigate whether exogenously added vegetative bacte-\nria  and  spores  reside  in  lysosomes,  we  monitored  their\ntrafﬁcking  in  BMDM  using  antibodies  speciﬁc  to  the\nlysosome-associated membrane protein 1 (LAMP1) (von\nFigura  and  Hasilik,  1986;  Fukuda,  1991).  Primary\nmacrophages were infected with\nSterne veg-\netative bacteria or spores for 30 min as described above.\nVegetative bacteria and spores were identiﬁed using the\ninside–outside  differential  staining  protocol.  As  shown  in\nFig. 3,  greater  than  90%  of  exogenously  added  vegeta-\ntive  bacteria  or  spores  were  associated  with  LAMP1  as\nearly as 30 min after infection, indicating a rapid trafﬁck-\ning to lysosomes. This result was validated by examining\ncolocalization  with  LysoTracker  Red  DND-99,  a  dye  that\nspeciﬁcally  accumulates\nin\n.,\n2000).  Consistent  with  the  LAMP1  colocalization  experi-\nment, more than 90% of bacteria and spores were asso-\nciated  with  LysoTracker  2 h  after  infection  (data  not\nshown). As shown earlier, the number of vegetative bac-\nteria  and  spores  decreased  over  time  but  the  remaining\nvegetative  bacteria  and  spores  were  all  associated  with\nLAMP1.\net al\nGerminated\nprimary macrophages\nB. anthracis\nSterne spores are killed in\nWe  next  sought  to  determine  whether  the  decrease  in\nintracellular  spore  number  was  due  to  the  killing  of  dor-\nmant or germinated spores. We used\n-alanine to inhibit\ngermination  during  infection  followed  by  viability  tests.\nWe  ﬁrst  determined  that  2 mM\n-alanine  was  sufﬁcient\nto  completely  inhibit  spore  germination  in  Dulbecco’s\nmodiﬁed  Eagle’s  medium  (DMEM)  media  containing\nGerminated spores are inactivated in macrophages\nB\ns\ne\ng\na\nh\np\no\nr\nc\na\nm\n\n\n\n/\na\ni\nr\ne\nt\nc\na\nB\na\ni\nr\ne\nt\nc\na\nb\n+\n\nP\nM\nA\nL\ns\ne\nr\no\np\ns\n+\n\nP\nM\nA\nL\n10%  fetal  bovine  serum  (FBS)  (data  not  shown).  Next,\nprimary  macrophages  were  pre-incubated  with  2 mM\n-\nD\nalanine followed by spore infection and subsequent incu-\nbation as described above. In this particular experiment,\nCytochalasin  D  was  used  during  the  incubation  so  that\nextracellular  germinated  spores  could  be  completely\nexcluded.  As  discussed  below,  similar  results  were\nobtained  in  the  absence  of  cytochalasin  D. The  number\nof heat-resistant spores and total viability were evaluated\nover time. Both values dropped in the ﬁrst hour and then\nremained relatively constant in the presence of\n-alanine\n-alanine to the medium at 2 h\n(Fig. 4). Addition of 5 mM\nL\nresulted in a decrease in heat-resistant spores and total\nviable count (Fig. 4) indicating that germinated spores or\nperhaps  newly  germinated  vegetative  bacteria  rather\nthan  dormant  spores  were  killed\nthe  primary\nmacrophages.\nin\nGerminated\nlysosomes\nspores are inactivated in\nTo investigate whether germinated spores and vegetative\nbacteria are indeed killed in lysosomes, we used the dom-\ninant-negative form of a Rab7 construct which blocks the\nfusion  of  the  late  endosomes  with  lysosomes  (Meresse\net al\n., 1995; 1999). We ﬁrst infected the primary macroph-\nages  with  adenoviruses  expressing  the  wild-type  Rab7\nand  the  dominant  negative  Rab7  (T22N)  fused  to  the\nenhanced yellow ﬂuorescent protein (EYFP), or EYFP as\na control. Macrophages expressing the appropriate Rab7\nconstructs were identiﬁed as green under the ﬂuorescent\nmicroscope with greater than 90% of the cells expressing\nthe  EYFP  fusions. The  macrophages  were  then  infected\nwith spores and intracellular spores were enumerated as\ndescribed  above.  The  number  of  intracellular  spores\nthe  genes  encoding  capsule  biosynthetic  proteins.  Con-\nsequently,  efﬁcient  killing  of  34F2  spores  and  vegetative\n+\nbacteria may not reﬂect the fate of a fully virulent (pXO1\n+\npXO2\n)  strain.  For  example,  capsule  biosynthesis  is\nbelieved  to  commence  soon  after  germination  of  Ames\nspores and has been shown to be important for anthrax\npathogenesis  (Bonhomme\n.,\n2005). There  is  also  evidence  for  toxin  gene  expression\n.,  1995).  We  sought  to\nwithin  macrophages  (Dai\n+\n+\ndetermine if spores of a virulent pXO1\nstrain could\npXO2\nsurvive  in  primary  macrophages.  Primary  macrophages\n+\nwere  infected  with  spores  of  Sterne  strain  7702  (pXO1\n+\n+\n–\npXO2\npXO2\n)  in\n)  and  the  virulent  strain  UT500  (pXO1\nthe  presence  or  absence  of  2 mM\n-alanine  without\ncytochalasin D. UT500 is isogenic to 7702, but harbours\npXO2  (Bourgogne\n.,  2003).  As  in  our  experiments\nwith\nB. anthracis Sterne strain 34F2, there were few intra-\ncellular vegetative bacteria in infected macrophages. Via-\nble numbers of UT500 dropped gradually in the absence\nof  D-alanine (Fig. 6). When spore germination was inhib-\nited  by  D-alanine,  there  was  an  initial  drop  in  the  viable\ncount during the ﬁrst hour and then the number remained\net al\nTime (Hr)\n=\ninside  macrophages\nremained\nexpressing  the  dominant  negative  Rab7  as  compared\nwith   those   in   macrophages   expressing   the   EYFP\n-test)  or  the  wild-type  Rab7\nt\nalone  (\n=\n-test)  (Fig. 5A). There  was  a  some-\nt\nP\n(\nwhat  greater  loss  of  spores  in  macrophages  expressing\nthe wild-type Rab7 as compared with that in macrophages\nexpressing the EYFP alone (Fig. 5A). The overexpression\nof  wild-type  Rab7  may  increase  the  maturation  of  endo-\nsomes  and  could  potentially  lead  to  increased  spore\ndestruction. This result indicates that a phagosome–lyso-\nsome  fusion  is  required  for  the  loss  of  spore  supporting\nthe  data  in  Fig. 3  that  spore  inactivation  occurred  in  the\nlysosome. Among the surviving spores, there was no sig-\nniﬁcant difference in viability (Fig. 5B).\nB. anthracis\nspores are inactivated in primary\nBacillus  anthracis\nSterne  lacks  plasmid  pXO2  and  thus\n0.2\nconstant for at least four additional hours. Spore inactiva-\ntion was restored when 5 mM L-alanine was added to the\ninfection  media  at  2 h.  Comparable  results  were  also\nobtained when macrophages were infected with spores of\nstrain UT501, an isogenic mutant of UT500 deleted for the\nvirulence regulator atxA and thus deﬁcient in the synthesis\nof toxins and highly attenuated in mice (Dai et al., 1995).\nThese  data  indicate  that  the  B. anthracis  toxin  proteins\nand  capsule  do  not  enhance  survival\nin  primary\nmacrophages.\nBacillus anthracis Sterne 34F2 strain was obtained from P. Jack-\nson, Los Alamos Laboratories. Sterne 7702, the virulent UT500,\nand UT501 (atxA) have been described (Bourgogne et al., 2003;\nDrysdale  et al.,  2004)  and  were  used  for  the  experiments  in\nFig. 6.  To  produce  spores,  overnight  bacterial  culture  were\nstreaked  onto  NSM  (nutrient  sporulation  medium)  (Schaeffer\net al.,  1965)  agar  plates.  Spores  were  collected  2 days  after\nincubation at 37°C by washing the plates with 10 ml sterile 1 M\nKCl plus 5 mM EDTA (pH 8.0) and centrifuging at 10 000 rpm for\n10 min. The spores were washed once with 30 ml of water and\nthe pellets resuspended in water. Remaining vegetative bacteria\nwere lysed by sonication (Sterne spores only) of the suspensions\nfor  40 s,  diluted  to  30 ml  with  water  and  centrifuged.  Spores  of\nthe pathogenic strain containing both the pXO1 and pXO2 plas-\nmids were not sonicated. Spore suspensions were then loaded\nonto  a  38%  to  50%  Renograﬁn  (Braco  Diagnostics,  Princeton,\nNJ) step gradient (3 ml each) in 30 ml glass centrifuge tubes and\ncentrifuged at 8000 rpm for 50 min in a Sorvall HB4 rotor (Cieslak\net al., 1993). Spores were washed once with 30 ml of water and\nsuspended in water. Spore purity was assessed by examination\nunder  a  phase  contrast  microscope  and  was  routinely  greater\nthan 99%.\nFor infection with vegetative bacteria, B. anthracis was grown\novernight at 37°C in Brain Heart Infusion broth (BHI) (BD, Sparks,\nMD) supplemented with 0.5 M sorbitol (Sigma, St Louis, MO) (to\nkeep  vegetative  bacteria  as  short  rods).  Bacteria  were  washed\ntwice  with  an  equal  volume  of  Hank’s  balanced  salt  solution\n(HBSS) and resuspended in HBSS. Spores and vegetative bac-\nterial  titers  were  determined  with  a  Petroff  Hausser  chamber.\nSpores  were  stored  at  4°C. Vegetative  bacteria  were  prepared\nfresh for each experiment.\nThe murine macrophage-like cell line RAW264.7 (TIB-71) was\npurchased from the ATCC cell biology stock centre (Manassas,\nVA) and was maintained in DMEM containing 10% FBS (Invitro-\ngen,  Carlsbad,  CA)  at  37°C  with  5%  CO2  (Lyons  et al.,  1992).\nBMDM  were  obtained  from  femur  bone  marrow  exudates  of\nfemale A/J mice (Harland Laboratories, Indianapolis, IN) that are\nsusceptible to spores of B. anthracis Sterne strain (Welkos et al.,\n1986).  Bone  marrow  cells  were  cultured  in  RPMI  1640  supple-\nmented  with  10%  heat-inactivated  FBS,  30%  L-cell  conditioned\nmedium  and  100 U  penicillin-streptomycin  in  a  37°C  CO2  incu-\nbator  for  7 days  (Swanson  and  Isberg,  1995).  Cells  were  col-\nlected  and  washed  three  times  with  10 ml  each  of  ice-cold\nDulbecco’s  phosphate-buffered  saline  (DPBS,  Mediatech,  VA).\nCell suspensions were centrifuged at 1000 rpm for 5 min at 4°C\nbefore being seeded for infection.\nPlasmid construction and adenovirus preparation\nTo create a bicistronic viral expression vector, the DNA fragment\nencoding  the  EYFP  (Ormo  et al.,  1996)  from  pEYFP-C1  (Clon-\ntech,  Palo  Alto,  CA)  was  subcloned  into  the  BglII-KpnI  sites  of\npShuttle-CMV  (Stratagene,  La  Jolla,  CA)  to  create  pShuttle-\nCMV-EYFP. DNA fragments encoding the human wild type and\ndominant-negative  Rab7  (T22N)  were  cloned  into  the  KpnI-\nEcoRV  sites  of  pShuttle-CMV-EYFP.  Adenoviruses  expressing\nEYFP, wild-type Rab7 and Rab7 (T22N) were produced accord-\ning to the manufacturer’s instruction (Stratagene, La Jolla, CA).\nTransfection of AD293 cells (Stratagene, La Jolla, CA) was car-\nried out using the FuGENE 6 Transfection Reagent (Roche, Pen-\nzberg, Germany) following the manufacturer’s instructions.\nMacrophage infection and immunoﬂuorescent microscopy\nDifferentiated BMDM were seeded onto 24 well plates with glass\ncover slips at a density of 2 × 105 cells well−1. The following day,\nmacrophages were infected with spores or bacteria at a moi of\n5 in HBSS, centrifuged at 1000 rpm for 5 min to enhance adhe-\nsion and incubated for 1 h to facilitate uptake. Macrophages were\nthen  washed  three  times  with  phosphate-buffered  saline  (PBS)\nand incubated in DMEM supplemented with FBS. Six micromolar\ncytochalasin D (Sigma, St Louis, MO) was used only for the data\nin Fig. 4. The infection medium was changed at every time point\nto minimize the number of extracellular spores and bacteria. For\ninhibition of spore germination, the macrophages were pre-incu-\nbated  with  2 mM  D-alanine  (Sigma)  in  HBSS  for  1 h  and  main-\ntained  in  2 mM  D-alanine  during  the  infection.  Five  millimolar\nL-alanine was used to reverse the inhibition of germination.\nAfter infection, macrophages were processed for immunoﬂuo-\nrescent  staining  and  examined  with  a  Zeiss  Axiovert  200 M\ndeconvolution  microscope  as  described  previously  (Dai  et al.,\n2004).  Monoclonal  anti-spore  antibody,  EF12,  was  against  the\nmajor  exosporial  protein,  BclA  (Steichen  et al.,  2003).  Rabbit\npolyclonal  anti-bacteria  antibody,  1816,  was  produced  against\nformaldehyde-inactivated  Sterne  vegetative  bacteria  (Prosci)\n(Guidi-Rontani et al., 1999b). Anti-LAMP1 antibody, Alexa Fluor\n388, Alexa Fluor 350 and Texas Red-conjugated secondary anti-\nbodies  were  purchased  from  Molecular  Probes  (Eugene,  OR).\nImages  are  black  and  white  projections  of  z-section  slides  with\npseudocolours  added  by  Adobe  Photoshop.  Inside  and  outside\ndifferential  staining  of  spores  and  vegetative  bacteria  was  per-\nformed  as  described  (Chen  et al.,  1996).  Brieﬂy,  infected  mac-\nrophages were ﬁxed with 3.7% formaldehyde in PBS for 15 min.\nFixed cells were washed three times with PBS and blocked with\n3%  milk  in  PBS  for  30 min.  Extracellular  spores  and  bacteria\nwere stained with anti-spore or anti-bacteria antibodies followed\nby  AF488-conjugated  secondary  antibodies  for  30 min.  Stained\nsamples were then permeabilized with 0.2% Triton X-100 in PBS\nfor 5 min. Total spores and bacteria were stained with anti-spore\nor  anti-bacteria  antibodies  followed  by  Texas-Red-conjugated\nsecondary  antibodies  for  30 min.  Intracellular  spores/bacteria\nappeared  as  Red  while\nthe  extracellular  spores/bacteria\nappeared as green/yellow. When examining the LAMP1 colocal-\nization  with  spores  and  bacteria,  intracellular  spores/bacteria\nwere  stained  with  Texas-Red-conjugated  secondary  antibodies\nand extracellular spores/bacteria were identiﬁed with AF350-con-\njugated  secondary  antibodies.  LAMP1  was  stained  with  FITC\nanti-mouse  LAMP1  (BD).  Infection  media  were  changed  every\n15 min to further minimize the number of extracellular bacteria.\nAt least 1000 spores and bacteria were counted for each sample.\nValues presented are from three independent experiments with\nstandard errors shown.\nFor viability tests, macrophages (approximately 5 × 105) were\nlysed with 2.5% saponin (Mock and Fouet, 2001), and spore and\nbacterial cfu were assessed by plating dilutions on Luria–Bertani\nagar. Where indicated, samples were heated at 65°C for 30 min\nto inactivate geminated spores and vegetative bacteria. The total\nviability and heat resistant values are reported as cfu relative to\nthe cfu at time zero. All experiments were repeated three times\nwith at least two independently prepared batches of spores. Only\nexperiments  with  at  least  80%  spore  uptake  (examined  by\ninside–outside staining) are reported. Values presented are aver-\nages of the three independent experiments with standard errors\nshown.  A  two-level  Student’s  t-test  was  conducted  to  analyse\nvalues in Fig. 5 using software Orgin 7.0.",
         "downloading",
         "The  current  model  for  pathogenesis  of  inhalation\nBacillus\nanthrax indicates that the uptake and fate of\nanthracis\nspores in alveolar macrophages are critical\nto  the  infection  process. We  have  employed  primary\nmacrophages,  which  are  more  efﬁcient  for  spore\nuptake than the macrophage-like cell line RAW264.7,\nto  investigate  spore  uptake  and  survival.  We  found\nthat  at  a  multiplicity  of  infection  (moi)  of  5,  greater\nthan 80% of the spores of the Sterne strain containing\nonly  the  pXO1  plasmid  were  internalized  within  1 h.\nWithin  4 h  post  infection,  viability  of  internalized\nSterne spores decreased to approximately 40%. Intra-\ncellular vegetative bacteria represented less than 1%\nof the total spore inoculum throughout the course of\ninfection  suggesting  effective  killing  of  germinated\nspores and/or vegetative bacteria. The Sterne spores\ntrafﬁcked quickly to phagolysosomes as indicated by\ncolocalization  with  lysosome-associated  membrane\nprotein 1 (LAMP1). Expression of a dominant-negative\nRab7 that blocked lysosome fusion enhanced Sterne\nspore  survival.  Addition  of\n-alanine  to  the  infection\nresulted  in  75%  inhibition  of  spore  germination  and\nincreased survival of internalized spores of the Sterne\nstrain  and  a  pathogenic  strain  containing  both  the\npXO1 and pXO2 plasmids. Inhibition was reversed by\nthe  addition  of\n-alanine,  which  resumed  spore  ger-\nL\nmination and subsequent spore killing. Our data indi-\ncate  that\nspores  germinate  in  and  are\nsubsequently killed by primary macrophages.\ningestion,  or  invasion  of  a  wound.\ninfection\noccurs when the dormant spores germinate and develop\ninto metabolically active vegetative bacteria which secrete\nthe  anthrax  toxin  proteins  (Dixon\n.,  1999;  Guidi-\nRontani\n., 2001; Agrawal and Pulendran, 2004). Early\nhistopathological studies showed that inhaled spores are\nefﬁciently taken up by phagocytes and transported to the\nregional lymph nodes (Barnes, 1947; Ross, 1957). Vege-\ntative bacteria eventually enter the bloodstream and may\n\nreach titers as high as 10\n., 1999). In\nper ml (Dixon\nmurine models, depletion of macrophages, in contrast to\nneutrophils, results in enhanced susceptibility to infection\nwith  spores.  Resistance  can  be  restored  by  increasing\nperitoneal macrophages levels, indicating the importance\ninfection\nof  macrophages  in  fending  off  a\n(Cote\nB. anthracis\n., 2005).\n., 2004; 2006; Bozue\ntrans\n;\ncya\nlef\nR; and the\n.,  1999).  Plasmid  pXO2  contains  the\nVirulent strains carry two large plasmids, pXO1 (186 kb)\nand pXO2 (96 kb). Plasmid pXO1 harbours a pathogenic-\nand\nity island containing the anthrax toxin genes;\npag\nX\nger\n., 1999a; Okinaka\ngermination operon (Guidi-Rontani\net al\ncapBCADE\noperon, encoding genes for biosynthesis and degradation\nof the poly-D-glutamic acid capsule (Drysdale\n., 2004;\nCandela  and  Fouet,  2005).  The  capsule  and  toxins  are\ntwo  major  virulence  factors  required  for  initiating  a  suc-\ncessful  anthrax  infection  (Leppla,  1982;  Guidi-Rontani\net al\n., 2005).\nThere have been several studies examining the fate of\nB. anthracis\nspores in both primary and cultured macroph-\n.  (2001)  reported  that\nage-like  cells.  Guidi-Rontani\nSterne  spores  (containing  only  the  pXO1\nB. anthracis\nplasmid)  germinated  but  did  not  replicate  in  either\nRAW264.7 macrophages or primary peritoneal macroph-\nages.  In  contrast,  Dixon\n.  (2000)  reported  extensive\ngermination  of  Sterne  spores  and  intracellular  growth\nfollowed  by  escape  of  the  vegetative  bacteria  from\nRAW264.7  macrophages.  Using\ntime-lapse  confocal\nmicroscopy,  Ruthel\n.  (2004)  found  that  10.8%  of  the\nSterne  spores  inside  RAW264.7  macrophages  germi-\nnated  and  grew  into  vegetative  bacteria.  Direct  viability\ntests of spores within RAW264.7 macrophages showed a\nrapid germination (of both the Sterne and Ames strains)\nand  thus  a  substantial  decline  in  heat-resistant  colony-\nforming  units  (cfu)  (Welkos\n.,  2002).  More  recently,\nthat  germination-\nKang\net al\nGerminated spores are inactivated in macrophages\nproﬁcient Sterne spores were killed much more efﬁciently\n∆\nthan germination-deﬁcient (\ngerH) spores by murine peri-\ntoneal  macrophages.  Overall,  these  studies  differed\nsomewhat  in  the  cell  lines  used  and  the  presence  of\ninhibitors such as gentamicin which was assumed to kill\nonly extracellular bacteria and cytochalasin D which was\nused  to  block  spore  and  vegetative  bacteria  uptake  into\nmacrophages. There  were  also  signiﬁcant  differences  in\nthe  methods  used  to  selectively  examine  intracellular\nspores or vegetative bacteria and these factors may have\ncontributed to the discrepancies in these reports.\net al\nWe  initially  attempted  to  resolve  these  issues  by\nemploying a very speciﬁc inside/outside staining protocol\n(Chen\n., 1996) for vegetative bacteria and spores in\ninvestigations  were\nRAW264.7  macrophages.  Our\nimpeded by inefﬁcient spore uptake and numerous tech-\nnical problems associated with internalization of extracel-\nlular  spores  and  vegetative  bacteria  by  RAW264.7\nmacrophages over the course of the experiments. We also\nhad concerns about the spore/bacterial killing capacity of\nRAW264.7 macrophages as compared with primary mac-\nrophages. We therefore switched to murine primary bone\nmarrow macrophages which proved to be very efﬁcient for\nspore  uptake. This  allowed  us  to  monitor,  without  inhibi-\ntors, intracellular events with less interference from extra-\ncellular  spores  and  bacteria  using  speciﬁc  ﬂuorescently\nlabelled antibodies. We report here evidence for the rapid\ntrafﬁcking of\nspores to phagolysosomes, fol-\nlowed by extensive germination and killing with very little\noutgrowth to vegetative bacteria.\nB. anthracis\nResults\nUptake and germination of\nRAW264.7 macrophages\nB. anthracis\nSterne spores in\nWe  initially  examined\nSterne  spore  uptake\nB. anthracis\nand  germination  in  the  RAW264.7  cell  line  using  an\ninside/outside staining protocol for both spores and veg-\netative  bacteria  as  described  in\nExperimental  proce-\n.  We  found  that  spore  uptake  was  very  inefﬁcient\ndures\nand  that  there  was  extensive  adherence  of  spores  and\nvegetative  cells  to  the  outside  of  the  macrophages.  In\nagreement  with  a  previous  report  (Welkos\n.,  2002),\nadherent  spores  and  vegetative  bacteria  were  taken  up\ncontinuously  over  several  hours  even  in  the  presence  of\ngentamicin (Fig. 1A). Initial spore uptake averaged about\n24%  of  the  total  spore  inoculum  with  a  multiplicity  of\ninfection  (moi)  of  5.  The  uptake  efﬁciency  increased  to\nabout 50% after 4–6 h (Fig. 1B). By 4 h, there was exten-\nsive  germination  and  vegetative  cell  outgrowth  with  long\nchains  of  bacteria  extending  across  several  macroph-\nages. These long chains of bacteria were located prima-\nrily  outside  the  RAW264.7  macrophages  and  in  some\ncases,  part  of  the  chain  appeared  to  be  inside  probably\ndue  to  uptake  after  germination  and  outgrowth  in  the\nmedium  (Fig. 1A  and  data  not  shown).  It  is  likely  that\nsome of the vegetative bacilli inside the RAW264.7 cells\nmay  have  germinated  in  the  medium  with  subsequent\nuptake  into  the  macrophages.  Addition  of  6\nM  cytocha-\nlasin  D  resulted  in  a  dramatic  reduction  in  intracellular\nvegetative bacteria (1–4% of the total inside spores plus\nvegetative  cells)  6 h  after  infection  (data  not  shown).\nThese  vegetative  bacteria  remained  very  small  with  no\nevidence  of  elongation  or  division.  The  lower  value  in\nthe  presence  of  cytochalasin  D  is  consistent  with  the\nnotion  that  there  had  been  continuous  uptake  of  spores\nand  vegetative  bacteria  by  RAW264.7  macrophages  in\nthe  absence  of  this  inhibitor.  We  therefore  switched  to\nprimary  murine  bone  marrow-derived  macrophages\n(BMDM)  from  A/J  mice  known  to  be  sensitive  to\nB. anthracis\net al\nUptake and germination of\nmurine BMDM\nB. anthracis\nSterne spores in\nMurine  BMDM  were  infected  with\n0.8\n0.7\n0.6\n0.5\n0.4\n0.3\n0.2\n0.1\nspores  (moi  of  5)  for  1 h  and  the  number  of  spores  and\nbacteria were enumerated using the inside–outside differ-\nential  staining  protocol  described  above  (Fig. 2A).  More\nthan  80%  of  the  spores  were  taken  up  by  the  primary\nmacrophages  after  this  incubation  period  (data  not\nshown). There appeared to be an initial drop in the number\nof  intracellular  spores  and  it  remained  relatively  stable\nthereafter (Fig. 2B). To determine if the germinated spores\ngrew into vegetative cells in primary macrophages, intra-\ncellular bacteria were monitored by the inside/outside dif-\nferential staining protocol. We observed very few and only\nsmall vegetative intracellular bacteria (Fig. 2A and C) and\n1% of the total intracellular spores plus\nthey peaked at\ncells 4–6 h after infection. All of the intracellular vegetative\nbacteria  remained  small  with  no  evidence  for  elongation\nor division (Fig. 2A).\nBacillus anthracis\nbacteria are trafﬁcked to the lysosomes\nspores and exogenously added\nB. anthracis\nThe decrease in intracellular spores can be attributed to\nthe  killing  of  intact  spores,  germinated  spores  or  out-\ngrown  bacteria.  Vegetative\nSterne  bacteria\nwere  efﬁciently  taken  up  by  BMDM  and  the  number\ndecreased  rapidly  (Fig. 3A)  indicating  that  these  primary\nB. anthracis\nmacrophages  were  very  efﬁcient  in  killing\nSterne  vegetative  bacteria. This  is  consistent  with  previ-\nous  ﬁndings  that  Sterne  vegetative  bacteria  are  highly\nattenuated  in  an  A/J  mouse  model  (Welkos  and  Fried-\nlander, 1988).\nB. anthracis\nthe  primary  compartment  where\ningested  particles  are  degraded  within  macrophages. To\ninvestigate whether exogenously added vegetative bacte-\nria  and  spores  reside  in  lysosomes,  we  monitored  their\ntrafﬁcking  in  BMDM  using  antibodies  speciﬁc  to  the\nlysosome-associated membrane protein 1 (LAMP1) (von\nFigura  and  Hasilik,  1986;  Fukuda,  1991).  Primary\nmacrophages were infected with\nSterne veg-\netative bacteria or spores for 30 min as described above.\nVegetative bacteria and spores were identiﬁed using the\ninside–outside  differential  staining  protocol.  As  shown  in\nFig. 3,  greater  than  90%  of  exogenously  added  vegeta-\ntive  bacteria  or  spores  were  associated  with  LAMP1  as\nearly as 30 min after infection, indicating a rapid trafﬁck-\ning to lysosomes. This result was validated by examining\ncolocalization  with  LysoTracker  Red  DND-99,  a  dye  that\nspeciﬁcally  accumulates\nin\n.,\n2000).  Consistent  with  the  LAMP1  colocalization  experi-\nment, more than 90% of bacteria and spores were asso-\nciated  with  LysoTracker  2 h  after  infection  (data  not\nshown). As shown earlier, the number of vegetative bac-\nteria  and  spores  decreased  over  time  but  the  remaining\nvegetative  bacteria  and  spores  were  all  associated  with\nLAMP1.\net al\nGerminated\nprimary macrophages\nB. anthracis\nSterne spores are killed in\nWe  next  sought  to  determine  whether  the  decrease  in\nintracellular  spore  number  was  due  to  the  killing  of  dor-\nmant or germinated spores. We used\n-alanine to inhibit\ngermination  during  infection  followed  by  viability  tests.\nWe  ﬁrst  determined  that  2 mM\n-alanine  was  sufﬁcient\nto  completely  inhibit  spore  germination  in  Dulbecco’s\nmodiﬁed  Eagle’s  medium  (DMEM)  media  containing\nGerminated spores are inactivated in macrophages\nB\ns\ne\ng\na\nh\np\no\nr\nc\na\nm\n\n\n\n/\na\ni\nr\ne\nt\nc\na\nB\na\ni\nr\ne\nt\nc\na\nb\n+\n\nP\nM\nA\nL\ns\ne\nr\no\np\ns\n+\n\nP\nM\nA\nL\n10%  fetal  bovine  serum  (FBS)  (data  not  shown).  Next,\nprimary  macrophages  were  pre-incubated  with  2 mM\n-\nD\nalanine followed by spore infection and subsequent incu-\nbation as described above. In this particular experiment,\nCytochalasin  D  was  used  during  the  incubation  so  that\nextracellular  germinated  spores  could  be  completely\nexcluded.  As  discussed  below,  similar  results  were\nobtained  in  the  absence  of  cytochalasin  D. The  number\nof heat-resistant spores and total viability were evaluated\nover time. Both values dropped in the ﬁrst hour and then\nremained relatively constant in the presence of\n-alanine\n-alanine to the medium at 2 h\n(Fig. 4). Addition of 5 mM\nL\nresulted in a decrease in heat-resistant spores and total\nviable count (Fig. 4) indicating that germinated spores or\nperhaps  newly  germinated  vegetative  bacteria  rather\nthan  dormant  spores  were  killed\nthe  primary\nmacrophages.\nin\nGerminated\nlysosomes\nspores are inactivated in\nTo investigate whether germinated spores and vegetative\nbacteria are indeed killed in lysosomes, we used the dom-\ninant-negative form of a Rab7 construct which blocks the\nfusion  of  the  late  endosomes  with  lysosomes  (Meresse\net al\n., 1995; 1999). We ﬁrst infected the primary macroph-\nages  with  adenoviruses  expressing  the  wild-type  Rab7\nand  the  dominant  negative  Rab7  (T22N)  fused  to  the\nenhanced yellow ﬂuorescent protein (EYFP), or EYFP as\na control. Macrophages expressing the appropriate Rab7\nconstructs were identiﬁed as green under the ﬂuorescent\nmicroscope with greater than 90% of the cells expressing\nthe  EYFP  fusions. The  macrophages  were  then  infected\nwith spores and intracellular spores were enumerated as\ndescribed  above.  The  number  of  intracellular  spores\nthe  genes  encoding  capsule  biosynthetic  proteins.  Con-\nsequently,  efﬁcient  killing  of  34F2  spores  and  vegetative\n+\nbacteria may not reﬂect the fate of a fully virulent (pXO1\n+\npXO2\n)  strain.  For  example,  capsule  biosynthesis  is\nbelieved  to  commence  soon  after  germination  of  Ames\nspores and has been shown to be important for anthrax\npathogenesis  (Bonhomme\n.,\n2005). There  is  also  evidence  for  toxin  gene  expression\n.,  1995).  We  sought  to\nwithin  macrophages  (Dai\n+\n+\ndetermine if spores of a virulent pXO1\nstrain could\npXO2\nsurvive  in  primary  macrophages.  Primary  macrophages\n+\nwere  infected  with  spores  of  Sterne  strain  7702  (pXO1\n+\n+\n–\npXO2\npXO2\n)  in\n)  and  the  virulent  strain  UT500  (pXO1\nthe  presence  or  absence  of  2 mM\n-alanine  without\ncytochalasin D. UT500 is isogenic to 7702, but harbours\npXO2  (Bourgogne\n.,  2003).  As  in  our  experiments\nwith\nB. anthracis Sterne strain 34F2, there were few intra-\ncellular vegetative bacteria in infected macrophages. Via-\nble numbers of UT500 dropped gradually in the absence\nof  D-alanine (Fig. 6). When spore germination was inhib-\nited  by  D-alanine,  there  was  an  initial  drop  in  the  viable\ncount during the ﬁrst hour and then the number remained\net al\nTime (Hr)\n=\ninside  macrophages\nremained\nexpressing  the  dominant  negative  Rab7  as  compared\nwith   those   in   macrophages   expressing   the   EYFP\n-test)  or  the  wild-type  Rab7\nt\nalone  (\n=\n-test)  (Fig. 5A). There  was  a  some-\nt\nP\n(\nwhat  greater  loss  of  spores  in  macrophages  expressing\nthe wild-type Rab7 as compared with that in macrophages\nexpressing the EYFP alone (Fig. 5A). The overexpression\nof  wild-type  Rab7  may  increase  the  maturation  of  endo-\nsomes  and  could  potentially  lead  to  increased  spore\ndestruction. This result indicates that a phagosome–lyso-\nsome  fusion  is  required  for  the  loss  of  spore  supporting\nthe  data  in  Fig. 3  that  spore  inactivation  occurred  in  the\nlysosome. Among the surviving spores, there was no sig-\nniﬁcant difference in viability (Fig. 5B).\nB. anthracis\nspores are inactivated in primary\nBacillus  anthracis\nSterne  lacks  plasmid  pXO2  and  thus\n0.2\nconstant for at least four additional hours. Spore inactiva-\ntion was restored when 5 mM L-alanine was added to the\ninfection  media  at  2 h.  Comparable  results  were  also\nobtained when macrophages were infected with spores of\nstrain UT501, an isogenic mutant of UT500 deleted for the\nvirulence regulator atxA and thus deﬁcient in the synthesis\nof toxins and highly attenuated in mice (Dai et al., 1995).\nThese  data  indicate  that  the  B. anthracis  toxin  proteins\nand  capsule  do  not  enhance  survival\nin  primary\nmacrophages.\nDiscussion\nMacrophages play a major role in host defence as part of\nthe  innate  immune  system  involved  in  engulﬁng  and\ndestroying  pathogenic  microorganisms  in  phagolysos-\nomes. Mice depleted of macrophages are more suscepti-\nble to B. anthracis spore infection (Cote et al., 2004; 2006)\nGerminated spores are inactivated in macrophages 1639\nand  resistance  was  restored  partially  by  injection  of\nRAW264.7 cells into the mice, indicating the essential role\nof  macrophages  in  limiting  the  anthrax  infection.  In  con-\ntrast,  depletion  of  neutrophils  had  no  such  effect  (Cote\net al., 2004; 2006).\nOur experiments were designed to examine the effec-\ntiveness  of  macrophages  in  controlling  an  infection  with\nspores. We  encountered  technical  difﬁculties  concerning\ninefﬁcient  spore  uptake  by  RAW264.7  macrophages\nincluding  signiﬁcant  spore  adhesion  to  the  outer  surface\nof  the  macrophages.  Extracellular  germination  in  the\nmedium  resulted  in  the  uptake  over  time  of  vegetative\nbacteria  making  accurate  assessment  of  intracellular\nspore  germination  and  subsequent  bacterial  outgrowth\ndifﬁcult. There are also potential problems with the use of\ngentamicin  which  enters  macrophages  (Drevets  et al.,\n1994;  Hamrick  et al.,  2003)  and  possibly  vacuoles  con-\ntaining spores.\nFor  these  reasons,  we  switched  to  mouse  primary\nmacrophages. We found the efﬁciency of spore uptake to\nbe greater than 80%, eliminating the need for either gen-\ntamicin  or  cytochalasin  D.  We  also  employed  an  inside/\noutside staining protocol which can speciﬁcally differenti-\nate  intracellular  spores/bacteria  from  those  outside. The\nstaining  and  viability  assays  were  used  to  demonstrate\nthe  rapid  trafﬁcking  of  spores,  germination  and  subse-\nquent killing.\nThe remaining viable spores at 6 h (Fig. 4, about 20%\nof the total inoculum that was still present after the addi-\ntion of L-alanine) may largely be due to those spores still\nungerminated in the medium or adhering to the macroph-\nages. If so, viable intracellular spores could be reduced to\nas low as 1–5% of the initial value. An increase of the D-\nalanine  concentration  to  5 mM  did  not  result  in  further\nprotection  of  spores,  i.e.  about  25%  of  the  spores  still\ngerminated. In contrast, 2 mM D-alanine completely inhib-\nited spore germination in DMEM medium containing 10%\nFBS (data not shown) suggesting the presence of special\ngerminants  (or  perhaps  limiting  concentrations  of  germi-\nnants)  or  conditions  within  the  macrophages. When  ger-\nmination was blocked by the addition of D-alanine, about\n75% of the spores remained dormant and viable even in\nphagolysosomes, suggesting that an L-alanine-dependent\ngermination  pathway  plays  a  major  role  within  macroph-\nages. Germination was totally inhibited when both 2 mM\nD-alanine  and  10 mM  D-histidine  were  present  (data  not\nshown). The D-histidine inhibition is consistent with a role\nfor  the gerH  operon  in  germination  within  macrophages.\nAromatic  amino  acids,  especially  L-histidine,  have  been\nreported to be required for spore germination in the pres-\nence of cultured macrophage-like cells or in macrophage-\nconditioned  medium  (Weiner  and  Hanna,  2003)  as  well\nas in primary mouse peritoneal macrophages (Kang et al.,\n2005). Our results indicate the function of more than one\n1640 H. Hu et al.\ngermination  operon  in  bone  marrow-derived  primary\nmacrophages.\nIt is not certain whether germinated spores, those that\noutgrow  to  small  vegetative  bacteria,  or  both  are  killed\ninside  macrophages. Vegetative  bacteria  added  to  mac-\nrophages were taken up rapidly, trafﬁcked to phagolysos-\nomes, and killed within 1 h (Fig. 3A). There were very few\nsmall  vegetative  bacteria  inside  infected  macrophages\nand  they  appeared  to  peak  a  few  hours  after  infection\nrather than reaching a rapid steady state value (Fig. 2). In\nvitro, outgrowth is slow relative to germination and as our\ndata  showed  that  loss  of  viability  was  detectable  within\n1 h,  it  is  most  likely  that  germinated  spores  were  the\nprimary targets. We also measured spore and vegetative\nbacteria killing in macrophages isolated from two knock-\nout  mice  (Jackson  laboratory,  Bar  Harbor,  Maine):  Cybb\nlacking NADH oxidase activity and Nos2 defective in NO\nproduction  (Lyons  et al.,  1992;  Pollock  et al.,  1995).  No\nsigniﬁcant  differences  in  spore  viability  were  observed\nwhen macrophages obtained from the parent and mutant\nmice were infected, suggesting that more than one factor\nwithin the phagolysosome is involved in spore killing.\nSpore  killing  by  primary  macrophages  was  quite  efﬁ-\ncient with less than 20% survival 4 h after the addition of\nL-alanine  (Fig. 4B).  As  mentioned  previously,  part  of  the\nsurviving spores may be outside of the macrophages so\nthat actual efﬁciency of killing with macrophages may be\neven  higher.  The  experiments  reported  here  were  done\nwith  macrophages  in  medium  containing  10%  FBS,  a\nnutrient  environment  suitable\nfor  sustaining  healthy\nmacrophages  and  for  providing  adequate  nutrients  for\nintracellular spore germination. In experiments employing\nmacrophages  suspended  in  buffer,  germination  was\ngreatly delayed (Cote et al., 2006) and thus spore survival\nwas enhanced. It would be important to determine which\nof  these  conditions  more  closely  resembles  those  in  the\nlung,  larynx  and  subsequently  in  the  lymph  nodes.  It  is\npossible that a few persistent spores may be sufﬁcient to\ninitiate an anthrax infection.\nExperimental procedures\nBacterial strains, spores and mammalian cell lines\nBacillus anthracis Sterne 34F2 strain was obtained from P. Jack-\nson, Los Alamos Laboratories. Sterne 7702, the virulent UT500,\nand UT501 (atxA) have been described (Bourgogne et al., 2003;\nDrysdale  et al.,  2004)  and  were  used  for  the  experiments  in\nFig. 6.  To  produce  spores,  overnight  bacterial  culture  were\nstreaked  onto  NSM  (nutrient  sporulation  medium)  (Schaeffer\net al.,  1965)  agar  plates.  Spores  were  collected  2 days  after\nincubation at 37°C by washing the plates with 10 ml sterile 1 M\nKCl plus 5 mM EDTA (pH 8.0) and centrifuging at 10 000 rpm for\n10 min. The spores were washed once with 30 ml of water and\nthe pellets resuspended in water. Remaining vegetative bacteria\nwere lysed by sonication (Sterne spores only) of the suspensions\nfor  40 s,  diluted  to  30 ml  with  water  and  centrifuged.  Spores  of\nthe pathogenic strain containing both the pXO1 and pXO2 plas-\nmids were not sonicated. Spore suspensions were then loaded\nonto  a  38%  to  50%  Renograﬁn  (Braco  Diagnostics,  Princeton,\nNJ) step gradient (3 ml each) in 30 ml glass centrifuge tubes and\ncentrifuged at 8000 rpm for 50 min in a Sorvall HB4 rotor (Cieslak\net al., 1993). Spores were washed once with 30 ml of water and\nsuspended in water. Spore purity was assessed by examination\nunder  a  phase  contrast  microscope  and  was  routinely  greater\nthan 99%.\nFor infection with vegetative bacteria, B. anthracis was grown\novernight at 37°C in Brain Heart Infusion broth (BHI) (BD, Sparks,\nMD) supplemented with 0.5 M sorbitol (Sigma, St Louis, MO) (to\nkeep  vegetative  bacteria  as  short  rods).  Bacteria  were  washed\ntwice  with  an  equal  volume  of  Hank’s  balanced  salt  solution\n(HBSS) and resuspended in HBSS. Spores and vegetative bac-\nterial  titers  were  determined  with  a  Petroff  Hausser  chamber.\nSpores  were  stored  at  4°C. Vegetative  bacteria  were  prepared\nfresh for each experiment.\nThe murine macrophage-like cell line RAW264.7 (TIB-71) was\npurchased from the ATCC cell biology stock centre (Manassas,\nVA) and was maintained in DMEM containing 10% FBS (Invitro-\ngen,  Carlsbad,  CA)  at  37°C  with  5%  CO2  (Lyons  et al.,  1992).\nBMDM  were  obtained  from  femur  bone  marrow  exudates  of\nfemale A/J mice (Harland Laboratories, Indianapolis, IN) that are\nsusceptible to spores of B. anthracis Sterne strain (Welkos et al.,\n1986).  Bone  marrow  cells  were  cultured  in  RPMI  1640  supple-\nmented  with  10%  heat-inactivated  FBS,  30%  L-cell  conditioned\nmedium  and  100 U  penicillin-streptomycin  in  a  37°C  CO2  incu-\nbator  for  7 days  (Swanson  and  Isberg,  1995).  Cells  were  col-\nlected  and  washed  three  times  with  10 ml  each  of  ice-cold\nDulbecco’s  phosphate-buffered  saline  (DPBS,  Mediatech,  VA).\nCell suspensions were centrifuged at 1000 rpm for 5 min at 4°C\nbefore being seeded for infection.\nPlasmid construction and adenovirus preparation\nTo create a bicistronic viral expression vector, the DNA fragment\nencoding  the  EYFP  (Ormo  et al.,  1996)  from  pEYFP-C1  (Clon-\ntech,  Palo  Alto,  CA)  was  subcloned  into  the  BglII-KpnI  sites  of\npShuttle-CMV  (Stratagene,  La  Jolla,  CA)  to  create  pShuttle-\nCMV-EYFP. DNA fragments encoding the human wild type and\ndominant-negative  Rab7  (T22N)  were  cloned  into  the  KpnI-\nEcoRV  sites  of  pShuttle-CMV-EYFP.  Adenoviruses  expressing\nEYFP, wild-type Rab7 and Rab7 (T22N) were produced accord-\ning to the manufacturer’s instruction (Stratagene, La Jolla, CA).\nTransfection of AD293 cells (Stratagene, La Jolla, CA) was car-\nried out using the FuGENE 6 Transfection Reagent (Roche, Pen-\nzberg, Germany) following the manufacturer’s instructions.\nMacrophage infection and immunoﬂuorescent microscopy\nDifferentiated BMDM were seeded onto 24 well plates with glass\ncover slips at a density of 2 × 105 cells well−1. The following day,\nmacrophages were infected with spores or bacteria at a moi of\n5 in HBSS, centrifuged at 1000 rpm for 5 min to enhance adhe-\nsion and incubated for 1 h to facilitate uptake. Macrophages were\nthen  washed  three  times  with  phosphate-buffered  saline  (PBS)\nand incubated in DMEM supplemented with FBS. Six micromolar\ncytochalasin D (Sigma, St Louis, MO) was used only for the data\nin Fig. 4. The infection medium was changed at every time point\nto minimize the number of extracellular spores and bacteria. For\ninhibition of spore germination, the macrophages were pre-incu-\nbated  with  2 mM  D-alanine  (Sigma)  in  HBSS  for  1 h  and  main-\ntained  in  2 mM  D-alanine  during  the  infection.  Five  millimolar\nL-alanine was used to reverse the inhibition of germination.\nAfter infection, macrophages were processed for immunoﬂuo-\nrescent  staining  and  examined  with  a  Zeiss  Axiovert  200 M\ndeconvolution  microscope  as  described  previously  (Dai  et al.,\n2004).  Monoclonal  anti-spore  antibody,  EF12,  was  against  the\nmajor  exosporial  protein,  BclA  (Steichen  et al.,  2003).  Rabbit\npolyclonal  anti-bacteria  antibody,  1816,  was  produced  against\nformaldehyde-inactivated  Sterne  vegetative  bacteria  (Prosci)\n(Guidi-Rontani et al., 1999b). Anti-LAMP1 antibody, Alexa Fluor\n388, Alexa Fluor 350 and Texas Red-conjugated secondary anti-\nbodies  were  purchased  from  Molecular  Probes  (Eugene,  OR).\nImages  are  black  and  white  projections  of  z-section  slides  with\npseudocolours  added  by  Adobe  Photoshop.  Inside  and  outside\ndifferential  staining  of  spores  and  vegetative  bacteria  was  per-\nformed  as  described  (Chen  et al.,  1996).  Brieﬂy,  infected  mac-\nrophages were ﬁxed with 3.7% formaldehyde in PBS for 15 min.\nFixed cells were washed three times with PBS and blocked with\n3%  milk  in  PBS  for  30 min.  Extracellular  spores  and  bacteria\nwere stained with anti-spore or anti-bacteria antibodies followed\nby  AF488-conjugated  secondary  antibodies  for  30 min.  Stained\nsamples were then permeabilized with 0.2% Triton X-100 in PBS\nfor 5 min. Total spores and bacteria were stained with anti-spore\nor  anti-bacteria  antibodies  followed  by  Texas-Red-conjugated\nsecondary  antibodies  for  30 min.  Intracellular  spores/bacteria\nappeared  as  Red  while\nthe  extracellular  spores/bacteria\nappeared as green/yellow. When examining the LAMP1 colocal-\nization  with  spores  and  bacteria,  intracellular  spores/bacteria\nwere  stained  with  Texas-Red-conjugated  secondary  antibodies\nand extracellular spores/bacteria were identiﬁed with AF350-con-\njugated  secondary  antibodies.  LAMP1  was  stained  with  FITC\nanti-mouse  LAMP1  (BD).  Infection  media  were  changed  every\n15 min to further minimize the number of extracellular bacteria.\nAt least 1000 spores and bacteria were counted for each sample.\nValues presented are from three independent experiments with\nstandard errors shown.\nFor viability tests, macrophages (approximately 5 × 105) were\nlysed with 2.5% saponin (Mock and Fouet, 2001), and spore and\nbacterial cfu were assessed by plating dilutions on Luria–Bertani\nagar. Where indicated, samples were heated at 65°C for 30 min\nto inactivate geminated spores and vegetative bacteria. The total\nviability and heat resistant values are reported as cfu relative to\nthe cfu at time zero. All experiments were repeated three times\nwith at least two independently prepared batches of spores. Only\nexperiments  with  at  least  80%  spore  uptake  (examined  by\ninside–outside staining) are reported. Values presented are aver-\nages of the three independent experiments with standard errors\nshown.  A  two-level  Student’s  t-test  was  conducted  to  analyse\nvalues in Fig. 5 using software Orgin 7.0.",
         "",
         "Inactivation of Bacillus anthracis spores in murine primary macrophages",
         "None"
        ],
        [
         "45",
         "0020212c1901cc54c623dd2638a2653bd6bfa869",
         "ABSTRACT The distinction between viable and dead bacterial cells poses a major challenge in microbial diagnostics. Due to the persistence of DNA in the environment after cells have lost viability, DNA-based quantification methods overestimate the number of viable cells in mixed populations or even lead to false-positive results in the absence of viable cells. On the other hand, RNA-based diagnostic methods, which circumvent this problem, are technically demanding and suffer from some drawbacks. A promising and easy-to-use alternative utilizing the DNA-intercalating dye ethidium monoazide bromide (EMA) was published recently. This chemical is known to penetrate only into “dead” cells with compromised cell membrane integrity. Subsequent photoinduced cross-linking was reported to inhibit PCR amplification of DNA from dead cells. We provide evidence here that in addition to inhibition of amplification, most of the DNA from dead cells is actually lost during the DNA extraction procedure, probably together with cell debris which goes into the pellet fraction. Exposure of bacteria to increasing stress and higher proportions of dead cells in defined populations led to increasing loss of genomic DNA. Experiments were performed using Escherichia coli O157:H7 and Salmonella enterica serovar Typhimurium as model pathogens and using real-time PCR for their quantification. Results showed that EMA treatment of mixed populations of these two species provides a valuable tool for selective removal of DNA of nonviable cells by using conventional extraction protocols. Furthermore, we provide evidence that prior to denaturing gradient gel electrophoresis, EMA treatment of a mature mixed-population drinking-water biofilm containing a substantial proportion of dead cells can result in community fingerprints dramatically different from those for an untreated biofilm. The interpretation of such fingerprints can have important implications in the field of microbial ecology.",
         "A. Nocker,A. Camper",
         "A. Nocker,A. Camper",
         "https://europepmc.org/articles/pmc1393219?pdf=render",
         "The distinction between viable and dead bacterial cells poses a major challenge in microbial diagnostics. Due\nto the persistence of DNA in the environment after cells have lost viability, DNA-based quantiﬁcation methods\noverestimate the number of viable cells in mixed populations or even lead to false-positive results in the\nabsence of viable cells. On the other hand, RNA-based diagnostic methods, which circumvent this problem, are\ntechnically demanding and suffer from some drawbacks. A promising and easy-to-use alternative utilizing the\nDNA-intercalating dye ethidium monoazide bromide (EMA) was published recently. This chemical is known to\npenetrate only into “dead” cells with compromised cell membrane integrity. Subsequent photoinduced cross-\nlinking was reported to inhibit PCR ampliﬁcation of DNA from dead cells. We provide evidence here that in\naddition to inhibition of ampliﬁcation, most of the DNA from dead cells is actually lost during the DNA\nextraction procedure, probably together with cell debris which goes into the pellet fraction. Exposure of\nbacteria to increasing stress and higher proportions of dead cells in deﬁned populations led to increasing loss\nof genomic DNA. Experiments were performed using Escherichia coli O157:H7 and Salmonella enterica serovar\nTyphimurium as model pathogens and using real-time PCR for their quantiﬁcation. Results showed that EMA\ntreatment of mixed populations of these two species provides a valuable tool for selective removal of DNA of\nnonviable cells by using conventional extraction protocols. Furthermore, we provide evidence that prior to\ndenaturing gradient gel electrophoresis, EMA treatment of a mature mixed-population drinking-water bioﬁlm\ncontaining a substantial proportion of dead cells can result in community ﬁngerprints dramatically different\nfrom those for an untreated bioﬁlm. The interpretation of such ﬁngerprints can have important implications\nin the ﬁeld of microbial ecology.",
         "DISCUSSION\nIn agreement with previous publications introducing the ap-\nplication of EMA in combination with quantitative PCR to\ndifferentiate between viable and dead bacterial cells (18, 24,\n25), the efﬁcacy of this DNA-intercalating dye in selecting\nagainst DNA from dead cells was conﬁrmed. EMA treatment\nof mixed bacterial communities might be a promising tool to\nselectively favor the analysis of the viable fraction.\nThe method might be an important step to mitigate one of\nthe major drawbacks from which all DNA-based molecular\ntechniques suffer, the inability to differentiate between viable\nand dead cells. In bacterial diagnostics, this leads to an over-\nestimation of viable cells or even to false-positive results, lim-\niting the suitability of these molecular tools for analysis of\nmicrobiological safety in water and food monitoring as well as\nfor clinical diagnostics (for reviews, see references 12 and 23).\nEMA pretreatment of samples might be a fast and elegant\nalternative to quantiﬁcation of RNA, which, due to the rapid\ndegradation of RNA after cell death, is currently the method\nof choice to avoid overestimation of bacterial numbers. Nev-\nertheless, working with RNA is still technically demanding,\ndue to the risk of RNA degradation and the instability of\nappropriate standards during storage. Practical problems of\nextracting detectable levels of intact RNA from small numbers\nof bacteria and the risk of DNA contamination add to the\ncomplexity of RNA-based approaches and are impediments to\nits utilization in routine diagnostics. Moreover, few studies\nhave investigated the relationship between detection of RNA\nand viability. Although most prokaryotic mRNA species have\nhalf-lives of only a few minutes in living cells (1, 2), Sheridan et\nal. (27) reported that some mRNA targets were detectable by\nreverse transcription-PCR 2 to 16 h after heat killing of E. coli\nand even longer after ethanol treatment. 16S rRNA was de-\ntected by reverse transcription-PCR in samples containing\ndead bacteria, and it did not disappear during a subsequent\nincubation of 16 h at room temperature. This was conﬁrmed by\nMcKillip et al. (16), who concluded that the presence of 16S\nrRNA up to 48 h after cell death was not correlated with\nviability following moderate heat inactivation or UV irradia-\ntion of E. coli O157:H7 and Staphylococcus aureus, raising\ndoubts about its potential as a useful indicator of viability.\nIn contrast to previous studies, we report that EMA-based\nselection against DNA originating from dead cells seems to\noccur primarily on the level of genomic DNA extraction. An\nunderstanding of the underlying principle might allow the com-\nbination of EMA treatment not only with real-time PCR as a\ndownstream analysis tool but also with hybridization-based\ntechniques such as microarray analysis. The selective loss of\nDNA from dead cells is presumably due to cross-linking of\nDNA to other cell components. EMA-DNA conjugates are\nlost during the extraction procedure and are pelleted together\nwith cell debris during the centrifugation step following cell\nlysis by bead beating. This correlated with qualitative observa-\ntions that an increasing number of dead cells resulted in a\nmore-intense red color of the pelleted cell debris (caused by\nthe EMA dye) after centrifugation. The observation holds true\nwhen different commercially available kits used routinely for\nextraction of genomic DNA are compared. Comparison with\nthe DNeasy tissue kit and the PrepMan Ultra sample prepa-\nration reagent conﬁrmed the results obtained with the Qbio-\ngene kit. Whereas the latter employs bead beating and binding\nof DNA to a resuspended binding matrix, which is added to the\nsolution, the DNeasy tissue kit relies on cell lysis using pro-\nteinase K treatment and selective binding of DNA to a silica\ngel membrane, followed by washing steps and elution. The\nPrepMan Ultra sample preparation procedure involves only\nboiling of samples in the lysis solution provided and removal of\ndebris by simple centrifugation; it does not employ any binding\nmatrix. Theoretically, the chemical properties of EMA could\naffect the binding efﬁcacy of DNA-EMA conjugates to the\nbinding matrix provided with the Qbiogene extraction kit, but\nthe results did not substantiate this. The molecular composi-\ntion of EMA conjugates has yet to be elucidated.\nOur results do not exclude the possibility that PCR ampli-\nﬁcation of remaining DNA from dead cells, which was copu-\nriﬁed along with DNA from viable cells, is inhibited due to\nEMA cross-linking. Indeed, we could conﬁrm previous reports\nthat treatment of puriﬁed DNA (not protected within a cell)\nresults in strong inhibition of ampliﬁcation (data not shown).\nDespite this fact, inhibition of PCR ampliﬁcation is very likely\nto play only a minor role, considering the good correlation\nbetween the proportion of viable cells, DNA template concen-\ntrations, and the signal detection thresholds in Q-PCR shown\nin this study.\nThe observation that EMA to some extent also decreases the\nDNA yield from growing unstressed cultures (mainly from E.\ncoli O157:H7 [Fig. 1A and 2B]) is not likely to be explained by\nthe presence of dead cells in the corresponding log-phase cul-\ntures and requires further examination. An intact cell mem-\nbrane should theoretically be an efﬁcient barrier to EMA due\nto the charge of the molecule. However, EMA might also\npenetrate viable cells to a certain extent, though not as efﬁ-\nciently as dead cells. Another speculation would be that the\nbright light used for inactivation of EMA would result in par-\ntial cell lysis and susceptibility to EMA. Special ﬁlters allowing\nthe transmission only of that part of the light spectrum relevant\nfor EMA inactivation (around 460 nm [3]) might be useful in\nthis regard. It also remains to be investigated whether the use\nof other chemicals for live-dead differentiation as an alterna-\ntive to ethidium monoazide would mitigate this problem. Im-\nprovement and further optimization of the method to avoid\nany loss of DNA from viable cells would allow better quanti-\nﬁcation of viable organisms. This is especially important in the\ncase of pathogens for which only viable cells are infectious. It\nis very encouraging, however, that Q-PCR could distinguish\nbetween deﬁned communities whose viable cell content\nshowed a difference of only 1% (Fig. 3). Automation of DNA\nisolation would further improve reproducibility and avoid vari-\nations due to manual sample processing. In order to achieve\nabsolute quantiﬁcation of bacterial pathogens, the use of stan-\ndards prepared from EMA-treated cells might be considered.\nEMA treatment of cells prior to PCR ampliﬁcation of se-\nlected targets may be a valuable method in microbial commu-\nnity analysis, a common procedure in molecular ecology. The\nstudy of environmental samples using DNA-based PCR inev-\nitably leads to ampliﬁcation of genetic material from both\nviable and dead cells. EMA might allow one to study speciﬁ-\ncally the viable and thus metabolically active fraction of these\nmixed communities. Conclusions from ﬁngerprints derived\nfrom the viable portion could differ substantially from those\noriginating from the study of total (viable and dead) cells (Fig.\n5). It must be mentioned, however, that the community ﬁnger-\nprint shown here should be interpreted with caution, because\nfurther studies are needed to answer the questions whether the\nEMA principle can be applied to a wide spectrum of bacteria\nand whether the same parameters can be applied to all mem-\nbers of a microbial community. So far EMA pretreatment has\nbeen successfully used only with E. coli O157:H7, Salmonella\nserovar Typhimurium, Campylobacter jejuni, and the gram-\npositive bacterium Listeria monocytogenes (18, 24, 25).\nIn summary, EMA treatment of mixed communities might\nimprove the targeting of genomic sequences for determination\nof the potential pathogenic threat originating from viable bac-\nteria, while avoiding less standardized, more troublesome, and\nmore expensive alternative techniques. The method presented\novercomes the currently missing relationship between viability\nand direct PCR detection of DNA targets without preenrich-\nments. Moreover, it could be an extremely useful method in\necological studies by limiting DNA-based community analysis\nto the viable fraction.",
         "Whether bacterial cells are dead or alive is an important\nquestion with many implications for monitoring of food and\nwater safety and for analysis of the sterility of pharmaceutical\ndrugs. Due to the relatively long persistence of DNA after cell\ndeath, in the range of several days to 3 weeks (10, 14), quan-\ntitative analysis of total DNA can lead to a substantial overes-\ntimation of the presence of living microorganisms and the\naccompanying pathogenic threats. The lack of differentiation\nbetween DNA from viable and dead bacterial cells is therefore\na major obstacle to broad-range application of DNA-based\nmolecular diagnostics (10, 12). The most commonly used strat-\negy to overcome this difﬁculty is to focus on the presence of the\nrapidly degrading RNA instead of the stable DNA (16, 27, 28,\n31). Due to its rapid turnover, the detection of RNA is far\nmore indicative of the presence of viable cells (1, 2). Never-\ntheless, working with RNA is technically demanding, and RNA\nis prone to contamination with RNA-degrading enzymes, re-\nsulting in problems of reproducibility. Moreover, the RNA\nexpression level depends on the physiological status of the cell,\nmaking accurate measurements of bacterial numbers difﬁcult.\nAnother strategy is the use of dyes for microscopic differenti-\nation between viable and dead cells (15, 13, 20, 29). Although\nthese methods are well established, quantiﬁcation by extrapo-\nlating counts of a limited number of bacteria is problematic\nand cannot be applied on a routine basis. Moreover, due to the\nlack of an ampliﬁcation step, the sensitivity of microscope-\nbased techniques is a limiting factor, as is the inability to\nspecify individual species. Culture-based techniques, on the\nother hand, are slow, providing results only after a few days,\nand are dependent on the medium and the incubation temper-\nature.\nA promising alternative has recently been described by\nNogva et al. (18) and Rudi et al. (24, 25), who have introduced\nethidium monoazide bromide (EMA)-PCR as a diagnostic\nDNA-based method combining the use of a live-dead staining\ndye with the speed and sensitivity of real-time PCR. EMA has\nbeen used as a dye for microscopic differentiation between\nviable and dead cells (4, 17, 19, 21, 22). Like ethidium bromide,\nthe structurally similar EMA is a phenanthridinium DNA/\nRNA-intercalating agent (30) with a binding constant of 2 \n105 to 3  105 M (3) that can enter only bacterial cells with\ncompromised cell walls and cell membranes (24). Following\nintercalation into the DNA of those cells, EMA can be co-\nvalently linked by photoactivation with high yields, up to 75%\n(3). Photolysis of EMA using visible light (maximum absor-\nbance at 460 nm) produces a nitrene that can form a covalent\nlinkage to DNA and other molecules (5, 6, 9). The unbound\nEMA, which remains free in solution, is simultaneously inac-\ntivated by reacting with water molecules (6). The resulting\nhydroxylamine is no longer capable of covalently binding to\nDNA (11). DNA from viable cells, protected from reactive\nEMA before light exposure by an intact cell membrane/cell\nwall, should thus not be affected by the inactivated EMA after\ncell lysis during the DNA extraction procedure.\nThe ampliﬁcation of DNA with bound EMA has been shown\nto be inhibited, leading to a strong signal reduction in subse-\nquent real-time PCR (24). However, the previous studies (18,\n24, 25), where this was shown, did not provide information\nabout the amount of genomic DNA added to the diagnostic\nPCR mixtures as a template following DNA extraction. In\norder to be able to attribute the viable-dead discrimination\nsolely to PCR inhibition, one needs to ensure that equal\namounts of genomic DNA are used as a template for quanti-\ntative PCR or that the template DNA is quantiﬁed. This point\nhas remained unaddressed.\nSurprisingly, we found a strong effect of EMA on the\ngenomic DNA yield when we started with the same bacterial\nbiomass for DNA extraction. The effect of EMA is selective in\nthat it reduces the DNA yield of stressed cultures signiﬁcantly\nmore than the DNA yield of unstressed cultures. As a conse-\nquence, equal volumes of genomic DNA preparations from\nstressed and unstressed cells used as templates for quantitative\nPCR result in different ampliﬁcation thresholds.\nWe share the view that EMA-PCR might be very useful for\nviable/dead discrimination of bacterial communities but sug-\ngest that the selection occurs mainly during DNA extraction\nand only to a lesser extent on the level of PCR. Signal reduc-\ntion may be due in part to PCR inhibition but is also due, more\nimportantly, to a selective loss of genomic DNA from dead\ncells during the extraction procedure. Understanding the un-\nderlying principle might allow the combination of EMA treat-\nment not only with real-time PCR but also with other down-\nstream analysis tools. Furthermore, we provide evidence that\nEMA can affect PCR-generated community ﬁngerprints. De-\nnaturing gradient gel electrophoresis (DGGE) of a mature\nmixed-population microbial drinking-water bioﬁlm produced\ndramatically different community structures when EMA-\ntreated and untreated aliquots of the same bioﬁlm were com-\npared.\nBacterial strains, culture conditions, and bioﬁlm sampling. Escherichia coli\nO157:H7 (strain 932) and Salmonella enterica serovar Typhimurium (environ-\nmental isolate; Department of Microbiology, Montana State University) were\nused as model organisms. Single colonies from Luria-Bertani (LB) agar streak\nplates were transferred to 50-ml culture tubes containing approximately 10 ml of\nLB medium. The cultures were incubated on a shaker at 280 rpm and 37°C and\nwere grown to log phase. Cultures were adjusted to an optical density at 600 nm\nof 1 by dilution with LB medium. Loss of viability was examined by plating 50 to\n100 l of serial dilutions (102 to 106 in LB medium) on LB agar. E. coli\nMach1-T1R (Invitrogen, Carlsbad, CA) transformed with a pCR2.1-TOPO plas-\nmid was grown in LB medium containing 50 g/ml kanamycin. A bioﬁlm for\nspiking experiments and community analysis was harvested from a water reser-\nvoir that was not light exposed. The bioﬁlm was resuspended in the original water\nby pipetting.\nStress conditions. Culture aliquots (500 l) of E. coli O157:H7 and Salmonella\nserovar Typhimurium were heat treated for 30 s to 15 min at 72°C using a\nstandard laboratory heat block and were immediately placed on ice after heat\ntreatment. Alternatively, cells were killed by exposure to isopropanol (ﬁnal\nconcentration, 70%) for 10 min. Isopropanol was removed by harvesting cells by\ncentrifugation at 5,000  g for 5 to 7 min prior to resuspension in 500 l of LB\nmedium. The number of CFU was determined in triplicate by plating 50 l of\nserial dilutions on LB agar, followed by overnight incubation at 37°C.\nEMA cross-linking. EMA bromide (phenanthridium, 3-amino-8-azido-5-ethyl-\n6-phenyl bromide; Molecular Probes Inc., Eugene, Oreg.) was dissolved in water\nto a stock concentration of 5 mg/ml and stored at 20°C in the dark. EMA was\nadded to cultures to a ﬁnal concentration of 100 g/ml, which was reported to be\nthe concentration resulting in the highest signal reduction (18). Following a\n5-min incubation in the dark with occasional ﬂipping, samples were light exposed\nfor 1 min using a 650-W halogen light source placed 20 cm from the sample\ntubes. During exposure, samples were placed on ice to avoid excessive heating.\nAfter photoinduced cross-linking, cells were pelleted at 5,000  g for 5 min prior\nto DNA isolation.\nDNA isolation and quantiﬁcation. Genomic DNA was extracted using the\nQbiogene (Carlsbad, CA) soil kit according to the manufacturer’s instructions.\nCell lysis of pure cultures was achieved by bead beating using a FastPrep machine\n(Qbiogene) for 25 s at a speed setting of 4.5 m/s. For bioﬁlm samples, the\nbead-beating time and speed were increased to 35 s and 5.5 m/s, respectively.\nAlternatively, for studying the effect of EMA on the DNA yield of heat- and\nisopropanol-stressed cultures (see Fig. 1), DNA was extracted using the DNeasy\ntissue kit (QIAGEN, Valencia, CA) and the PrepMan Ultra sample preparation\nreagent (Applied Biosystems, Foster City, CA) according to the manufacturer’s\ninstructions. DNA was quantiﬁed using the PicoGreen quantiﬁcation solution\n(Molecular Probes Inc.) and a TBS-380 ﬂuorometer (Turner BioSystems Inc.,\nSunnyvale, CA) using genomic DNA from E. coli O157:H7 as a standard. The\nquantiﬁcation results obtained with the ﬂuorometer were compared with band\nintensities of high-molecular-weight genomic DNA visualized on ethidium bro-\nmide-stained 1% agarose gels. Seven to ten percent of the total corresponding\neluate volumes was loaded onto the gels.\nQuantitative PCR. For relative PCR quantiﬁcation of Salmonella, 1 l of\nextracted genomic DNA was added to 24 l of PCR mixture containing Sybr\nGreen PCR master mix (Applied Biosystems, Foster City, CA) and 10 pmol of\ninvA2-F (5-GATTCTGGTACTAATGGTGATGATC-3)\nprimers\n(8) and\ninvA2-R (5-GCCAGGCTATCGCCAATAAC-3) (8). The cycling parameters\nwere as follows: 8 min at 95°C (initial polymerase activation and denaturation),\nfollowed by 50 cycles of 20 s at 95°C, 30 s at 60°C, and 25 s at 72°C. For melt curve\nanalysis, the temperature was increased in 0.2°C increments from 60 to 94°C.\nFor relative quantiﬁcation of E. coli O157:H7, Q-PCR was performed in a\ntotal volume of 25 l containing 1 l extracted genomic DNA and ﬁnal concen-\ntrations of 1 AmpliTaq GOLD buffer (Applied Biosystems), 5.5 mM MgCl2,\n0.3 M of primer stx1-forward (5-GACTGCAAAGACGTATGTAGATTCG-\n3), 0.3 M of primer stx1-reverse (5-ATCTATCCCTCTGACATCAACTGC-\n3) (26), 0.15 M of the stx1 probe (5-TGAATGTCATTCGCTCTGCAATAG\nGTACTC-3) (26), and 2.5 U of AmpliTaq Gold (Applied Biosystems). The\nstx1-probe had 6-carboxyﬂuorescein as the 5 reporter and BHQ1 as the 3\nquencher. The cycling parameters were as follows: 8 min at 95°C, followed by 50\ncycles of 20 s at 95°C, 30 s at 55°C, and 25 s at 72°C. The same cycling parameters\nwere applied for the ampliﬁcation of an internal 244-bp fragment of the pCR2.1\nplasmid (Invitrogen, Carlsbad, CA) using 10 pmol each of primers M13-for\n(5-GTAAAACGACGGCCAG-3) and M13-rev (5-CAGGAAACAGCTATG\nAC-3) and Sybr Green PCR master mix (Applied Biosystems, Foster City, CA).\nQ-PCR and data analysis were performed with a SmartCycler II (Cepheid,\nSunnyvale, CA). Cycle threshold (CT) values were automatically calculated by\nthe SmartCycler software using the second derivative method.\nCommunity ﬁngerprinting. Two milliliters of a resuspended bioﬁlm solution\nwas split into two identical aliquots, one of which was treated with EMA. Both\naliquots were exposed to light. Approximately 5 ng of extracted genomic DNA\nserved as a template for ampliﬁcation of an internal fragment of the 16S rRNA\ncoding gene using 10 pg each of primers 1070F (5-ATGGCTGTCGTCAGCT-\n3) (7) and 1392R (5-ACGGGCGGTGTGTAC-3) (7); 1392R had a GC clamp\nat its 5 end (5-CGCCCGCCGCGCCCCGCGCCCGGCCCGCCGCCCCCGC\nCCC-3) (7). The PCR mixture contained 4 mM MgCl2, 4 mM deoxynucleoside\ntriphosphates, 1 U rTth polymerase, and 1 of the buffer supplied with the\nenzyme (all reagents were from Applied Biosystems) in a total volume of 50 l.\nCycling parameters were as follows: 2 min at 95°C; 30 cycles of 40 s at 95°C, 40 s\nat 55°C, and 40 s at 72°C; and a 7-min ﬁnal elongation at 72°C. The 362-bp\nproduct was puriﬁed using the QIAgen PCR puriﬁcation kit (QIAGEN, Hilden,\nGermany). One hundred nanograms of PCR products was analyzed using\nDGGE. Gels had 8% polyacrylamide and a denaturation gradient of 30% to\n70%, where 100% denaturant is deﬁned as 7 M urea and 40% formamide (all\nreagents were from Sigma-Aldrich, St. Louis, MO). Electrophoresis was carried\nout at 60 V for 16 h using a DCode system (Bio-Rad, Hercules, CA). Gels were\nadding 1.5 l each of the dyes SYTO 9 and propidium iodide (both in anhydrous\ndimethyl sulfoxide) to the suspended bioﬁlm. SYTO 9 stain generally labels all\nbacteria in a population green, while propidium iodide penetrates only bacteria\nwith compromised membranes and labels them red. After mixing and a 1-h\nincubation in the dark, a 100-l sample was ﬁltered through 25-mm polycarbon-\nate ﬁlters (Osmonic Inc., Minnetonka, Minn.), washed with 3 ml of ﬁlter-steril-\nized nanopure water, and mounted on slides. Photomicrographs were taken on\na Nikon E800 microscope using a 1.4-numerical-aperture 100 oil objective and\nﬂuorescein isothiocyanate (480/30 excitation ﬁlter, DM505 dichroic mirror,\n535/40 emission ﬁlter) and tetramethyl rhodamine isocyanate (546/10 excitation\nﬁlter, DM575 dichroic mirror, 590 emission ﬁlter) ﬂuorescence ﬁlter sets. The\nsoftware used for visualization was MetaVue, version 6.1 (Universal Imaging,\nDownington, Pa.).",
         "",
         "RESULTS\nThroughout the Results and Discussion, the term “viable” is\nused. Although there is considerable controversy surrounding\nthe correlation between viable and culturable cells, for E. coli\nO157:H7 and Salmonella, whose culturability can be tested by\nplating on standard growth media, we use the term in an\noperational sense and do not differentiate between “viable”\nand “culturable.” However, when one is looking at undeﬁned,\nmixed-population samples from the environment, “viable” can-\nnot be interpreted as “culturable,” because it is well accepted\nthat only a small fraction of bacteria may be cultured. In the\ncase of these data, viability has been related to the relative\nproportion of live/dead cells as measured using the BacLight\nlive/dead stain.\nEffect of stress on DNA yield of EMA-treated cultures. E.\ncoli O157:H7 and Salmonella cultures were subjected to dif-\nferent stress conditions. Heat exposure at 72°C for 5 min or\ntreatment with 70% isopropanol for 10 min resulted in dra-\nmatic losses in DNA yield (below the detection limit of 1\nng/ml) for both species compared to the corresponding non-\nEMA-treated aliquots (Fig. 1A). In the case of E. coli O157:\nH7, EMA treatment resulted in a signiﬁcant decrease in the\ngenomic DNA yield of unstressed cells, but the stress-induced\nloss was far more pronounced. The loss of DNA correlated\nwith the qualitative observation that during DNA extraction\nonly stressed cultures produced a visible red pellet (caused by\nthe EMA dye) after centrifugation of cell debris. These results,\nobtained with the Qbiogene soil DNA extraction kit, were\nconﬁrmed using the DNeasy tissue kit and the PrepMan Ultra\nsample preparation reagent (used in previous publications [18,\n24, 25]). These methods employ different principles for cell\nlysis and cleanup, excluding the possibility that selective loss of\nDNA is seen only with a speciﬁc DNA extraction procedure.\nIndependently of the extraction method, stress and subsequent\nEMA treatment resulted in a dramatic loss of genomic DNA as\nvisualized on ethidium bromide-stained agarose gels (Fig. 1B,\nC, and D). Observations on the loss of DNA correlated with a\ncomplete loss of viability from heat and isopropanol treatment\nas conﬁrmed by plate counts. The stress-induced reduction in\nDNA yield was reﬂected in a Q-PCR signal reduction, as\ndetermined by subtraction of CT values for EMA-treated cells\nfrom those for non-EMA-treated cells (Fig. 1E). For example,\nthe difference in threshold values (CT) approximated 20 cy-\ncles in the case of heat-killed Salmonella.\nExposure of pathogens to a stress gradient. The next step\nwas to study the effect of increasingly long heat exposure on\nthe genomic DNA yield and threshold ampliﬁcation cycles of\nE. coli O157:H7 and Salmonella. Increasing stress resulted in\nan increasing loss in viability (Fig. 2A) as determined by plate\ncounts. Heat exposure of 1.5 to 2 min was the threshold, where\nno viable colonies could be detected. Genomic DNA extracted\nfrom EMA-treated or untreated cultures was normalized to\nthe genomic DNA yield of the corresponding non-EMA-\ntreated and unstressed samples (Fig. 2B and C). Whereas the\nDNA yields of non-EMA-treated cultures varied about 60%\nrelative to each other, the yields of EMA-treated aliquots\ndecreased to undetectable levels with prolonged heat expo-\nsure. Values obtained by ﬂuorometer quantiﬁcation of DNA\nfrom EMA-treated aliquots correlated nicely with band inten-\nsities of high-molecular-weight DNA on stained agarose gels\n(Fig. 2B and C, top). This was in agreement with qualitative\nobservations that an increasingly long heat exposure resulted\nin a more intense red color of the pelleted cell debris after\ncentrifugation.\nIdentical volumes of genomic DNA served as the template\nfor Q-PCR for detection of relative differences in stx1 (E. coli\nO157:H7) and invA (Salmonella) gene copies. The tendencies\nseen in plate counts and in DNA yields correlated with a\ngradual increase in the CT values with longer exposure times,\nmeaning that more cycles were needed for ampliﬁcation. Sub-\ntracting the CT values of EMA-treated aliquots from those of\nthe corresponding untreated aliquots showed that longer ex-\nposure times resulted in increasingly strong signal reductions\nfor both bacterial species (Fig. 2D).\nEffect of EMA on deﬁned ratios of viable and dead cells. To\nelucidate the relationship between the proportion of viable\ncells, the DNA yield, and the real-time PCR signals after EMA\ntreatment, mixtures with deﬁned ratios of viable and heat-\nkilled cells were used. An aliquot of E. coli O157:H7 was\nsubjected to heat treatment at 72°C for 15 min, resulting in a\ndecrease in culturable cell counts to zero. Heat-killed cells\nwere mixed with the untreated original culture in deﬁned ra-\ntios, with viable cells representing 0, 0.1, 1, 10, 50, or 100% of\nthe total bacterial cell concentration, respectively (Fig. 3A).\nEvery mixture was supplemented with 5% (vol/vol) E. coli\nMach1-T1R (Invitrogen, Carlsbad, CA) transformed with a\npCR2.1 plasmid as an ampliﬁcation control. Increasing pro-\nportions of unheated and viable cells led to a substantial in-\ncrease in the genomic DNA yield (Fig. 3B) and to decreasing\nCT values in Q-PCR targeting the stx1 gene (Fig. 3C). The\nhighest CT value was observed for sample I, containing only\nheat-killed E. coli O157:H7 cells. Although the control ampli-\nﬁcation of an internal sequence of the pCR2.1 plasmid using\nM13 primers followed a tendency similar to that of the stx1\nampliﬁcation, the range in CT values was less than 2 cycles\ncompared to a difference of at least 15 cycles for the detection\nof the stx1 target (Fig. 3C). This observation can be explained\nby a concentration-dependent coprecipitation of plasmid DNA\nwith cell debris.\nThe increase in CT values for increasing proportions of heat-\nkilled E. coli O157:H7 seems to be due mainly to a decrease in\nthe DNA template concentrations and not to the presence of\nEMA-cross-linked DNA as reported previously (24). Indeed,\nall of the DNA seems to be ampliﬁable, as reﬂected in an R2\nvalue of 0.9829 for a plot of the natural logarithm of template\nconcentrations versus CT values (Fig. 3D).\nEffect of EMA on deﬁned mixed two-species samples. En-\ncouraged by the results obtained from a mixture of viable and\nheat-killed cells of a single species, we extended the study to a\nmixture of two bacterial species (E. coli O157:H7 and Salmo-\nnella serovar Typhimurium), again with different ratios of via-\nble and heat-killed cells. Each bacterial species made up 50%\nof the total volume of 2 ml (Fig. 4A). Furthermore, the viable\nfraction from both species added up to 50%, as did the heat-\nkilled fraction from both species. Quantiﬁcation of\ntotal\ngenomic DNA showed relatively constant levels for non-EMA-\ntreated and EMA-treated samples, although the DNA yields of\nEMA-treated samples, as expected, were lower, considering\nthe fact that half of the cells were not viable (Fig. 4B). Species-\nspeciﬁc PCR ampliﬁcation of EMA-treated samples resulted\nin a signal that decreased as the percentage of heat-killed cells\nincreased (Fig. 4C). The total ranges in CT values between\nviable and heat-killed cells were 12.89 (0.6) and 15.6 (3.7)\ncycles for E. coli O157:H7 and Salmonella, respectively, com-\npared to 1.4 (0.3) and 2.1 (0.1) cycles for non-EMA-treated\nmixtures.\nDistinction between viable and dead pathogens in a spiked\nbioﬁlm. We were interested in whether the distinction between\nviable and dead would also work in a complex mixed-commu-\nnity bioﬁlm background. Aliquots of a resuspended natural\ndrinking-water bioﬁlm were spiked with either viable or heat-\nkilled E. coli O157:H7. The relative differences in ampliﬁcation\nefﬁcacies were determined by quantitative PCR followed by\nsubtraction of CT values derived from EMA-treated mixtures\nfrom the corresponding CT values from untreated mixtures.\nEMA treatment led to a signal reduction of 2.8 (0.2) cycles in\nthe case of unstressed viable cells compared to a signal reduc-\ntion of 11.8 (0.2) cycles in the case of heat-killed cells added\nto the bioﬁlm.\nEffect of EMA on community ﬁngerprinting. Community\nﬁngerprints from a mature bioﬁlm harvested from a water\nreservoir without light exposure were dramatically different\nwhen an EMA-treated aliquot was compared with an un-\ntreated aliquot (Fig. 5A). Whereas EMA treatment led to a\nsignal reduction of some bands, the intensities of other bands\nincreased as a result of EMA treatment using DGGE for\nseparation of 16S rRNA gene ampliﬁcation products. BacLight\nLIVE/DEAD ﬂuorescence microscopy showed that the major-\nity of the cells stained red or orange with propidium iodide and\nthus can be considered dead (Fig. 5B). Viable cells staining\ngreen with the SYTO 9 dye appeared to be signiﬁcantly less\nabundant. Quantiﬁcation was not attempted due to the pres-\nence of abiotic particles and the inability to distinguish be-\ntween bacteria and other organisms.",
         "downloading",
         "The distinction between viable and dead bacterial cells poses a major challenge in microbial diagnostics. Due\nto the persistence of DNA in the environment after cells have lost viability, DNA-based quantiﬁcation methods\noverestimate the number of viable cells in mixed populations or even lead to false-positive results in the\nabsence of viable cells. On the other hand, RNA-based diagnostic methods, which circumvent this problem, are\ntechnically demanding and suffer from some drawbacks. A promising and easy-to-use alternative utilizing the\nDNA-intercalating dye ethidium monoazide bromide (EMA) was published recently. This chemical is known to\npenetrate only into “dead” cells with compromised cell membrane integrity. Subsequent photoinduced cross-\nlinking was reported to inhibit PCR ampliﬁcation of DNA from dead cells. We provide evidence here that in\naddition to inhibition of ampliﬁcation, most of the DNA from dead cells is actually lost during the DNA\nextraction procedure, probably together with cell debris which goes into the pellet fraction. Exposure of\nbacteria to increasing stress and higher proportions of dead cells in deﬁned populations led to increasing loss\nof genomic DNA. Experiments were performed using Escherichia coli O157:H7 and Salmonella enterica serovar\nTyphimurium as model pathogens and using real-time PCR for their quantiﬁcation. Results showed that EMA\ntreatment of mixed populations of these two species provides a valuable tool for selective removal of DNA of\nnonviable cells by using conventional extraction protocols. Furthermore, we provide evidence that prior to\ndenaturing gradient gel electrophoresis, EMA treatment of a mature mixed-population drinking-water bioﬁlm\ncontaining a substantial proportion of dead cells can result in community ﬁngerprints dramatically different\nfrom those for an untreated bioﬁlm. The interpretation of such ﬁngerprints can have important implications\nin the ﬁeld of microbial ecology.\nWhether bacterial cells are dead or alive is an important\nquestion with many implications for monitoring of food and\nwater safety and for analysis of the sterility of pharmaceutical\ndrugs. Due to the relatively long persistence of DNA after cell\ndeath, in the range of several days to 3 weeks (10, 14), quan-\ntitative analysis of total DNA can lead to a substantial overes-\ntimation of the presence of living microorganisms and the\naccompanying pathogenic threats. The lack of differentiation\nbetween DNA from viable and dead bacterial cells is therefore\na major obstacle to broad-range application of DNA-based\nmolecular diagnostics (10, 12). The most commonly used strat-\negy to overcome this difﬁculty is to focus on the presence of the\nrapidly degrading RNA instead of the stable DNA (16, 27, 28,\n31). Due to its rapid turnover, the detection of RNA is far\nmore indicative of the presence of viable cells (1, 2). Never-\ntheless, working with RNA is technically demanding, and RNA\nis prone to contamination with RNA-degrading enzymes, re-\nsulting in problems of reproducibility. Moreover, the RNA\nexpression level depends on the physiological status of the cell,\nmaking accurate measurements of bacterial numbers difﬁcult.\nAnother strategy is the use of dyes for microscopic differenti-\nation between viable and dead cells (15, 13, 20, 29). Although\nthese methods are well established, quantiﬁcation by extrapo-\nlating counts of a limited number of bacteria is problematic\nand cannot be applied on a routine basis. Moreover, due to the\nlack of an ampliﬁcation step, the sensitivity of microscope-\nbased techniques is a limiting factor, as is the inability to\nspecify individual species. Culture-based techniques, on the\nother hand, are slow, providing results only after a few days,\nand are dependent on the medium and the incubation temper-\nature.\nA promising alternative has recently been described by\nNogva et al. (18) and Rudi et al. (24, 25), who have introduced\nethidium monoazide bromide (EMA)-PCR as a diagnostic\nDNA-based method combining the use of a live-dead staining\ndye with the speed and sensitivity of real-time PCR. EMA has\nbeen used as a dye for microscopic differentiation between\nviable and dead cells (4, 17, 19, 21, 22). Like ethidium bromide,\nthe structurally similar EMA is a phenanthridinium DNA/\nRNA-intercalating agent (30) with a binding constant of 2 \n105 to 3  105 M (3) that can enter only bacterial cells with\ncompromised cell walls and cell membranes (24). Following\nintercalation into the DNA of those cells, EMA can be co-\nvalently linked by photoactivation with high yields, up to 75%\n(3). Photolysis of EMA using visible light (maximum absor-\nbance at 460 nm) produces a nitrene that can form a covalent\nlinkage to DNA and other molecules (5, 6, 9). The unbound\nEMA, which remains free in solution, is simultaneously inac-\ntivated by reacting with water molecules (6). The resulting\nhydroxylamine is no longer capable of covalently binding to\nDNA (11). DNA from viable cells, protected from reactive\nEMA before light exposure by an intact cell membrane/cell\nwall, should thus not be affected by the inactivated EMA after\ncell lysis during the DNA extraction procedure.\nThe ampliﬁcation of DNA with bound EMA has been shown\nto be inhibited, leading to a strong signal reduction in subse-\nquent real-time PCR (24). However, the previous studies (18,\n24, 25), where this was shown, did not provide information\nabout the amount of genomic DNA added to the diagnostic\nPCR mixtures as a template following DNA extraction. In\norder to be able to attribute the viable-dead discrimination\nsolely to PCR inhibition, one needs to ensure that equal\namounts of genomic DNA are used as a template for quanti-\ntative PCR or that the template DNA is quantiﬁed. This point\nhas remained unaddressed.\nSurprisingly, we found a strong effect of EMA on the\ngenomic DNA yield when we started with the same bacterial\nbiomass for DNA extraction. The effect of EMA is selective in\nthat it reduces the DNA yield of stressed cultures signiﬁcantly\nmore than the DNA yield of unstressed cultures. As a conse-\nquence, equal volumes of genomic DNA preparations from\nstressed and unstressed cells used as templates for quantitative\nPCR result in different ampliﬁcation thresholds.\nWe share the view that EMA-PCR might be very useful for\nviable/dead discrimination of bacterial communities but sug-\ngest that the selection occurs mainly during DNA extraction\nand only to a lesser extent on the level of PCR. Signal reduc-\ntion may be due in part to PCR inhibition but is also due, more\nimportantly, to a selective loss of genomic DNA from dead\ncells during the extraction procedure. Understanding the un-\nderlying principle might allow the combination of EMA treat-\nment not only with real-time PCR but also with other down-\nstream analysis tools. Furthermore, we provide evidence that\nEMA can affect PCR-generated community ﬁngerprints. De-\nnaturing gradient gel electrophoresis (DGGE) of a mature\nmixed-population microbial drinking-water bioﬁlm produced\ndramatically different community structures when EMA-\ntreated and untreated aliquots of the same bioﬁlm were com-\npared.\nBacterial strains, culture conditions, and bioﬁlm sampling. Escherichia coli\nO157:H7 (strain 932) and Salmonella enterica serovar Typhimurium (environ-\nmental isolate; Department of Microbiology, Montana State University) were\nused as model organisms. Single colonies from Luria-Bertani (LB) agar streak\nplates were transferred to 50-ml culture tubes containing approximately 10 ml of\nLB medium. The cultures were incubated on a shaker at 280 rpm and 37°C and\nwere grown to log phase. Cultures were adjusted to an optical density at 600 nm\nof 1 by dilution with LB medium. Loss of viability was examined by plating 50 to\n100 l of serial dilutions (102 to 106 in LB medium) on LB agar. E. coli\nMach1-T1R (Invitrogen, Carlsbad, CA) transformed with a pCR2.1-TOPO plas-\nmid was grown in LB medium containing 50 g/ml kanamycin. A bioﬁlm for\nspiking experiments and community analysis was harvested from a water reser-\nvoir that was not light exposed. The bioﬁlm was resuspended in the original water\nby pipetting.\nStress conditions. Culture aliquots (500 l) of E. coli O157:H7 and Salmonella\nserovar Typhimurium were heat treated for 30 s to 15 min at 72°C using a\nstandard laboratory heat block and were immediately placed on ice after heat\ntreatment. Alternatively, cells were killed by exposure to isopropanol (ﬁnal\nconcentration, 70%) for 10 min. Isopropanol was removed by harvesting cells by\ncentrifugation at 5,000  g for 5 to 7 min prior to resuspension in 500 l of LB\nmedium. The number of CFU was determined in triplicate by plating 50 l of\nserial dilutions on LB agar, followed by overnight incubation at 37°C.\nEMA cross-linking. EMA bromide (phenanthridium, 3-amino-8-azido-5-ethyl-\n6-phenyl bromide; Molecular Probes Inc., Eugene, Oreg.) was dissolved in water\nto a stock concentration of 5 mg/ml and stored at 20°C in the dark. EMA was\nadded to cultures to a ﬁnal concentration of 100 g/ml, which was reported to be\nthe concentration resulting in the highest signal reduction (18). Following a\n5-min incubation in the dark with occasional ﬂipping, samples were light exposed\nfor 1 min using a 650-W halogen light source placed 20 cm from the sample\ntubes. During exposure, samples were placed on ice to avoid excessive heating.\nAfter photoinduced cross-linking, cells were pelleted at 5,000  g for 5 min prior\nto DNA isolation.\nDNA isolation and quantiﬁcation. Genomic DNA was extracted using the\nQbiogene (Carlsbad, CA) soil kit according to the manufacturer’s instructions.\nCell lysis of pure cultures was achieved by bead beating using a FastPrep machine\n(Qbiogene) for 25 s at a speed setting of 4.5 m/s. For bioﬁlm samples, the\nbead-beating time and speed were increased to 35 s and 5.5 m/s, respectively.\nAlternatively, for studying the effect of EMA on the DNA yield of heat- and\nisopropanol-stressed cultures (see Fig. 1), DNA was extracted using the DNeasy\ntissue kit (QIAGEN, Valencia, CA) and the PrepMan Ultra sample preparation\nreagent (Applied Biosystems, Foster City, CA) according to the manufacturer’s\ninstructions. DNA was quantiﬁed using the PicoGreen quantiﬁcation solution\n(Molecular Probes Inc.) and a TBS-380 ﬂuorometer (Turner BioSystems Inc.,\nSunnyvale, CA) using genomic DNA from E. coli O157:H7 as a standard. The\nquantiﬁcation results obtained with the ﬂuorometer were compared with band\nintensities of high-molecular-weight genomic DNA visualized on ethidium bro-\nmide-stained 1% agarose gels. Seven to ten percent of the total corresponding\neluate volumes was loaded onto the gels.\nQuantitative PCR. For relative PCR quantiﬁcation of Salmonella, 1 l of\nextracted genomic DNA was added to 24 l of PCR mixture containing Sybr\nGreen PCR master mix (Applied Biosystems, Foster City, CA) and 10 pmol of\ninvA2-F (5-GATTCTGGTACTAATGGTGATGATC-3)\nprimers\n(8) and\ninvA2-R (5-GCCAGGCTATCGCCAATAAC-3) (8). The cycling parameters\nwere as follows: 8 min at 95°C (initial polymerase activation and denaturation),\nfollowed by 50 cycles of 20 s at 95°C, 30 s at 60°C, and 25 s at 72°C. For melt curve\nanalysis, the temperature was increased in 0.2°C increments from 60 to 94°C.\nFor relative quantiﬁcation of E. coli O157:H7, Q-PCR was performed in a\ntotal volume of 25 l containing 1 l extracted genomic DNA and ﬁnal concen-\ntrations of 1 AmpliTaq GOLD buffer (Applied Biosystems), 5.5 mM MgCl2,\n0.3 M of primer stx1-forward (5-GACTGCAAAGACGTATGTAGATTCG-\n3), 0.3 M of primer stx1-reverse (5-ATCTATCCCTCTGACATCAACTGC-\n3) (26), 0.15 M of the stx1 probe (5-TGAATGTCATTCGCTCTGCAATAG\nGTACTC-3) (26), and 2.5 U of AmpliTaq Gold (Applied Biosystems). The\nstx1-probe had 6-carboxyﬂuorescein as the 5 reporter and BHQ1 as the 3\nquencher. The cycling parameters were as follows: 8 min at 95°C, followed by 50\ncycles of 20 s at 95°C, 30 s at 55°C, and 25 s at 72°C. The same cycling parameters\nwere applied for the ampliﬁcation of an internal 244-bp fragment of the pCR2.1\nplasmid (Invitrogen, Carlsbad, CA) using 10 pmol each of primers M13-for\n(5-GTAAAACGACGGCCAG-3) and M13-rev (5-CAGGAAACAGCTATG\nAC-3) and Sybr Green PCR master mix (Applied Biosystems, Foster City, CA).\nQ-PCR and data analysis were performed with a SmartCycler II (Cepheid,\nSunnyvale, CA). Cycle threshold (CT) values were automatically calculated by\nthe SmartCycler software using the second derivative method.\nCommunity ﬁngerprinting. Two milliliters of a resuspended bioﬁlm solution\nwas split into two identical aliquots, one of which was treated with EMA. Both\naliquots were exposed to light. Approximately 5 ng of extracted genomic DNA\nserved as a template for ampliﬁcation of an internal fragment of the 16S rRNA\ncoding gene using 10 pg each of primers 1070F (5-ATGGCTGTCGTCAGCT-\n3) (7) and 1392R (5-ACGGGCGGTGTGTAC-3) (7); 1392R had a GC clamp\nat its 5 end (5-CGCCCGCCGCGCCCCGCGCCCGGCCCGCCGCCCCCGC\nCCC-3) (7). The PCR mixture contained 4 mM MgCl2, 4 mM deoxynucleoside\ntriphosphates, 1 U rTth polymerase, and 1 of the buffer supplied with the\nenzyme (all reagents were from Applied Biosystems) in a total volume of 50 l.\nCycling parameters were as follows: 2 min at 95°C; 30 cycles of 40 s at 95°C, 40 s\nat 55°C, and 40 s at 72°C; and a 7-min ﬁnal elongation at 72°C. The 362-bp\nproduct was puriﬁed using the QIAgen PCR puriﬁcation kit (QIAGEN, Hilden,\nGermany). One hundred nanograms of PCR products was analyzed using\nDGGE. Gels had 8% polyacrylamide and a denaturation gradient of 30% to\n70%, where 100% denaturant is deﬁned as 7 M urea and 40% formamide (all\nreagents were from Sigma-Aldrich, St. Louis, MO). Electrophoresis was carried\nout at 60 V for 16 h using a DCode system (Bio-Rad, Hercules, CA). Gels were\nadding 1.5 l each of the dyes SYTO 9 and propidium iodide (both in anhydrous\ndimethyl sulfoxide) to the suspended bioﬁlm. SYTO 9 stain generally labels all\nbacteria in a population green, while propidium iodide penetrates only bacteria\nwith compromised membranes and labels them red. After mixing and a 1-h\nincubation in the dark, a 100-l sample was ﬁltered through 25-mm polycarbon-\nate ﬁlters (Osmonic Inc., Minnetonka, Minn.), washed with 3 ml of ﬁlter-steril-\nized nanopure water, and mounted on slides. Photomicrographs were taken on\na Nikon E800 microscope using a 1.4-numerical-aperture 100 oil objective and\nﬂuorescein isothiocyanate (480/30 excitation ﬁlter, DM505 dichroic mirror,\n535/40 emission ﬁlter) and tetramethyl rhodamine isocyanate (546/10 excitation\nﬁlter, DM575 dichroic mirror, 590 emission ﬁlter) ﬂuorescence ﬁlter sets. The\nsoftware used for visualization was MetaVue, version 6.1 (Universal Imaging,\nDownington, Pa.).\nRESULTS\nThroughout the Results and Discussion, the term “viable” is\nused. Although there is considerable controversy surrounding\nthe correlation between viable and culturable cells, for E. coli\nO157:H7 and Salmonella, whose culturability can be tested by\nplating on standard growth media, we use the term in an\noperational sense and do not differentiate between “viable”\nand “culturable.” However, when one is looking at undeﬁned,\nmixed-population samples from the environment, “viable” can-\nnot be interpreted as “culturable,” because it is well accepted\nthat only a small fraction of bacteria may be cultured. In the\ncase of these data, viability has been related to the relative\nproportion of live/dead cells as measured using the BacLight\nlive/dead stain.\nEffect of stress on DNA yield of EMA-treated cultures. E.\ncoli O157:H7 and Salmonella cultures were subjected to dif-\nferent stress conditions. Heat exposure at 72°C for 5 min or\ntreatment with 70% isopropanol for 10 min resulted in dra-\nmatic losses in DNA yield (below the detection limit of 1\nng/ml) for both species compared to the corresponding non-\nEMA-treated aliquots (Fig. 1A). In the case of E. coli O157:\nH7, EMA treatment resulted in a signiﬁcant decrease in the\ngenomic DNA yield of unstressed cells, but the stress-induced\nloss was far more pronounced. The loss of DNA correlated\nwith the qualitative observation that during DNA extraction\nonly stressed cultures produced a visible red pellet (caused by\nthe EMA dye) after centrifugation of cell debris. These results,\nobtained with the Qbiogene soil DNA extraction kit, were\nconﬁrmed using the DNeasy tissue kit and the PrepMan Ultra\nsample preparation reagent (used in previous publications [18,\n24, 25]). These methods employ different principles for cell\nlysis and cleanup, excluding the possibility that selective loss of\nDNA is seen only with a speciﬁc DNA extraction procedure.\nIndependently of the extraction method, stress and subsequent\nEMA treatment resulted in a dramatic loss of genomic DNA as\nvisualized on ethidium bromide-stained agarose gels (Fig. 1B,\nC, and D). Observations on the loss of DNA correlated with a\ncomplete loss of viability from heat and isopropanol treatment\nas conﬁrmed by plate counts. The stress-induced reduction in\nDNA yield was reﬂected in a Q-PCR signal reduction, as\ndetermined by subtraction of CT values for EMA-treated cells\nfrom those for non-EMA-treated cells (Fig. 1E). For example,\nthe difference in threshold values (CT) approximated 20 cy-\ncles in the case of heat-killed Salmonella.\nExposure of pathogens to a stress gradient. The next step\nwas to study the effect of increasingly long heat exposure on\nthe genomic DNA yield and threshold ampliﬁcation cycles of\nE. coli O157:H7 and Salmonella. Increasing stress resulted in\nan increasing loss in viability (Fig. 2A) as determined by plate\ncounts. Heat exposure of 1.5 to 2 min was the threshold, where\nno viable colonies could be detected. Genomic DNA extracted\nfrom EMA-treated or untreated cultures was normalized to\nthe genomic DNA yield of the corresponding non-EMA-\ntreated and unstressed samples (Fig. 2B and C). Whereas the\nDNA yields of non-EMA-treated cultures varied about 60%\nrelative to each other, the yields of EMA-treated aliquots\ndecreased to undetectable levels with prolonged heat expo-\nsure. Values obtained by ﬂuorometer quantiﬁcation of DNA\nfrom EMA-treated aliquots correlated nicely with band inten-\nsities of high-molecular-weight DNA on stained agarose gels\n(Fig. 2B and C, top). This was in agreement with qualitative\nobservations that an increasingly long heat exposure resulted\nin a more intense red color of the pelleted cell debris after\ncentrifugation.\nIdentical volumes of genomic DNA served as the template\nfor Q-PCR for detection of relative differences in stx1 (E. coli\nO157:H7) and invA (Salmonella) gene copies. The tendencies\nseen in plate counts and in DNA yields correlated with a\ngradual increase in the CT values with longer exposure times,\nmeaning that more cycles were needed for ampliﬁcation. Sub-\ntracting the CT values of EMA-treated aliquots from those of\nthe corresponding untreated aliquots showed that longer ex-\nposure times resulted in increasingly strong signal reductions\nfor both bacterial species (Fig. 2D).\nEffect of EMA on deﬁned ratios of viable and dead cells. To\nelucidate the relationship between the proportion of viable\ncells, the DNA yield, and the real-time PCR signals after EMA\ntreatment, mixtures with deﬁned ratios of viable and heat-\nkilled cells were used. An aliquot of E. coli O157:H7 was\nsubjected to heat treatment at 72°C for 15 min, resulting in a\ndecrease in culturable cell counts to zero. Heat-killed cells\nwere mixed with the untreated original culture in deﬁned ra-\ntios, with viable cells representing 0, 0.1, 1, 10, 50, or 100% of\nthe total bacterial cell concentration, respectively (Fig. 3A).\nEvery mixture was supplemented with 5% (vol/vol) E. coli\nMach1-T1R (Invitrogen, Carlsbad, CA) transformed with a\npCR2.1 plasmid as an ampliﬁcation control. Increasing pro-\nportions of unheated and viable cells led to a substantial in-\ncrease in the genomic DNA yield (Fig. 3B) and to decreasing\nCT values in Q-PCR targeting the stx1 gene (Fig. 3C). The\nhighest CT value was observed for sample I, containing only\nheat-killed E. coli O157:H7 cells. Although the control ampli-\nﬁcation of an internal sequence of the pCR2.1 plasmid using\nM13 primers followed a tendency similar to that of the stx1\nampliﬁcation, the range in CT values was less than 2 cycles\ncompared to a difference of at least 15 cycles for the detection\nof the stx1 target (Fig. 3C). This observation can be explained\nby a concentration-dependent coprecipitation of plasmid DNA\nwith cell debris.\nThe increase in CT values for increasing proportions of heat-\nkilled E. coli O157:H7 seems to be due mainly to a decrease in\nthe DNA template concentrations and not to the presence of\nEMA-cross-linked DNA as reported previously (24). Indeed,\nall of the DNA seems to be ampliﬁable, as reﬂected in an R2\nvalue of 0.9829 for a plot of the natural logarithm of template\nconcentrations versus CT values (Fig. 3D).\nEffect of EMA on deﬁned mixed two-species samples. En-\ncouraged by the results obtained from a mixture of viable and\nheat-killed cells of a single species, we extended the study to a\nmixture of two bacterial species (E. coli O157:H7 and Salmo-\nnella serovar Typhimurium), again with different ratios of via-\nble and heat-killed cells. Each bacterial species made up 50%\nof the total volume of 2 ml (Fig. 4A). Furthermore, the viable\nfraction from both species added up to 50%, as did the heat-\nkilled fraction from both species. Quantiﬁcation of\ntotal\ngenomic DNA showed relatively constant levels for non-EMA-\ntreated and EMA-treated samples, although the DNA yields of\nEMA-treated samples, as expected, were lower, considering\nthe fact that half of the cells were not viable (Fig. 4B). Species-\nspeciﬁc PCR ampliﬁcation of EMA-treated samples resulted\nin a signal that decreased as the percentage of heat-killed cells\nincreased (Fig. 4C). The total ranges in CT values between\nviable and heat-killed cells were 12.89 (0.6) and 15.6 (3.7)\ncycles for E. coli O157:H7 and Salmonella, respectively, com-\npared to 1.4 (0.3) and 2.1 (0.1) cycles for non-EMA-treated\nmixtures.\nDistinction between viable and dead pathogens in a spiked\nbioﬁlm. We were interested in whether the distinction between\nviable and dead would also work in a complex mixed-commu-\nnity bioﬁlm background. Aliquots of a resuspended natural\ndrinking-water bioﬁlm were spiked with either viable or heat-\nkilled E. coli O157:H7. The relative differences in ampliﬁcation\nefﬁcacies were determined by quantitative PCR followed by\nsubtraction of CT values derived from EMA-treated mixtures\nfrom the corresponding CT values from untreated mixtures.\nEMA treatment led to a signal reduction of 2.8 (0.2) cycles in\nthe case of unstressed viable cells compared to a signal reduc-\ntion of 11.8 (0.2) cycles in the case of heat-killed cells added\nto the bioﬁlm.\nEffect of EMA on community ﬁngerprinting. Community\nﬁngerprints from a mature bioﬁlm harvested from a water\nreservoir without light exposure were dramatically different\nwhen an EMA-treated aliquot was compared with an un-\ntreated aliquot (Fig. 5A). Whereas EMA treatment led to a\nsignal reduction of some bands, the intensities of other bands\nincreased as a result of EMA treatment using DGGE for\nseparation of 16S rRNA gene ampliﬁcation products. BacLight\nLIVE/DEAD ﬂuorescence microscopy showed that the major-\nity of the cells stained red or orange with propidium iodide and\nthus can be considered dead (Fig. 5B). Viable cells staining\ngreen with the SYTO 9 dye appeared to be signiﬁcantly less\nabundant. Quantiﬁcation was not attempted due to the pres-\nence of abiotic particles and the inability to distinguish be-\ntween bacteria and other organisms.\nDISCUSSION\nIn agreement with previous publications introducing the ap-\nplication of EMA in combination with quantitative PCR to\ndifferentiate between viable and dead bacterial cells (18, 24,\n25), the efﬁcacy of this DNA-intercalating dye in selecting\nagainst DNA from dead cells was conﬁrmed. EMA treatment\nof mixed bacterial communities might be a promising tool to\nselectively favor the analysis of the viable fraction.\nThe method might be an important step to mitigate one of\nthe major drawbacks from which all DNA-based molecular\ntechniques suffer, the inability to differentiate between viable\nand dead cells. In bacterial diagnostics, this leads to an over-\nestimation of viable cells or even to false-positive results, lim-\niting the suitability of these molecular tools for analysis of\nmicrobiological safety in water and food monitoring as well as\nfor clinical diagnostics (for reviews, see references 12 and 23).\nEMA pretreatment of samples might be a fast and elegant\nalternative to quantiﬁcation of RNA, which, due to the rapid\ndegradation of RNA after cell death, is currently the method\nof choice to avoid overestimation of bacterial numbers. Nev-\nertheless, working with RNA is still technically demanding,\ndue to the risk of RNA degradation and the instability of\nappropriate standards during storage. Practical problems of\nextracting detectable levels of intact RNA from small numbers\nof bacteria and the risk of DNA contamination add to the\ncomplexity of RNA-based approaches and are impediments to\nits utilization in routine diagnostics. Moreover, few studies\nhave investigated the relationship between detection of RNA\nand viability. Although most prokaryotic mRNA species have\nhalf-lives of only a few minutes in living cells (1, 2), Sheridan et\nal. (27) reported that some mRNA targets were detectable by\nreverse transcription-PCR 2 to 16 h after heat killing of E. coli\nand even longer after ethanol treatment. 16S rRNA was de-\ntected by reverse transcription-PCR in samples containing\ndead bacteria, and it did not disappear during a subsequent\nincubation of 16 h at room temperature. This was conﬁrmed by\nMcKillip et al. (16), who concluded that the presence of 16S\nrRNA up to 48 h after cell death was not correlated with\nviability following moderate heat inactivation or UV irradia-\ntion of E. coli O157:H7 and Staphylococcus aureus, raising\ndoubts about its potential as a useful indicator of viability.\nIn contrast to previous studies, we report that EMA-based\nselection against DNA originating from dead cells seems to\noccur primarily on the level of genomic DNA extraction. An\nunderstanding of the underlying principle might allow the com-\nbination of EMA treatment not only with real-time PCR as a\ndownstream analysis tool but also with hybridization-based\ntechniques such as microarray analysis. The selective loss of\nDNA from dead cells is presumably due to cross-linking of\nDNA to other cell components. EMA-DNA conjugates are\nlost during the extraction procedure and are pelleted together\nwith cell debris during the centrifugation step following cell\nlysis by bead beating. This correlated with qualitative observa-\ntions that an increasing number of dead cells resulted in a\nmore-intense red color of the pelleted cell debris (caused by\nthe EMA dye) after centrifugation. The observation holds true\nwhen different commercially available kits used routinely for\nextraction of genomic DNA are compared. Comparison with\nthe DNeasy tissue kit and the PrepMan Ultra sample prepa-\nration reagent conﬁrmed the results obtained with the Qbio-\ngene kit. Whereas the latter employs bead beating and binding\nof DNA to a resuspended binding matrix, which is added to the\nsolution, the DNeasy tissue kit relies on cell lysis using pro-\nteinase K treatment and selective binding of DNA to a silica\ngel membrane, followed by washing steps and elution. The\nPrepMan Ultra sample preparation procedure involves only\nboiling of samples in the lysis solution provided and removal of\ndebris by simple centrifugation; it does not employ any binding\nmatrix. Theoretically, the chemical properties of EMA could\naffect the binding efﬁcacy of DNA-EMA conjugates to the\nbinding matrix provided with the Qbiogene extraction kit, but\nthe results did not substantiate this. The molecular composi-\ntion of EMA conjugates has yet to be elucidated.\nOur results do not exclude the possibility that PCR ampli-\nﬁcation of remaining DNA from dead cells, which was copu-\nriﬁed along with DNA from viable cells, is inhibited due to\nEMA cross-linking. Indeed, we could conﬁrm previous reports\nthat treatment of puriﬁed DNA (not protected within a cell)\nresults in strong inhibition of ampliﬁcation (data not shown).\nDespite this fact, inhibition of PCR ampliﬁcation is very likely\nto play only a minor role, considering the good correlation\nbetween the proportion of viable cells, DNA template concen-\ntrations, and the signal detection thresholds in Q-PCR shown\nin this study.\nThe observation that EMA to some extent also decreases the\nDNA yield from growing unstressed cultures (mainly from E.\ncoli O157:H7 [Fig. 1A and 2B]) is not likely to be explained by\nthe presence of dead cells in the corresponding log-phase cul-\ntures and requires further examination. An intact cell mem-\nbrane should theoretically be an efﬁcient barrier to EMA due\nto the charge of the molecule. However, EMA might also\npenetrate viable cells to a certain extent, though not as efﬁ-\nciently as dead cells. Another speculation would be that the\nbright light used for inactivation of EMA would result in par-\ntial cell lysis and susceptibility to EMA. Special ﬁlters allowing\nthe transmission only of that part of the light spectrum relevant\nfor EMA inactivation (around 460 nm [3]) might be useful in\nthis regard. It also remains to be investigated whether the use\nof other chemicals for live-dead differentiation as an alterna-\ntive to ethidium monoazide would mitigate this problem. Im-\nprovement and further optimization of the method to avoid\nany loss of DNA from viable cells would allow better quanti-\nﬁcation of viable organisms. This is especially important in the\ncase of pathogens for which only viable cells are infectious. It\nis very encouraging, however, that Q-PCR could distinguish\nbetween deﬁned communities whose viable cell content\nshowed a difference of only 1% (Fig. 3). Automation of DNA\nisolation would further improve reproducibility and avoid vari-\nations due to manual sample processing. In order to achieve\nabsolute quantiﬁcation of bacterial pathogens, the use of stan-\ndards prepared from EMA-treated cells might be considered.\nEMA treatment of cells prior to PCR ampliﬁcation of se-\nlected targets may be a valuable method in microbial commu-\nnity analysis, a common procedure in molecular ecology. The\nstudy of environmental samples using DNA-based PCR inev-\nitably leads to ampliﬁcation of genetic material from both\nviable and dead cells. EMA might allow one to study speciﬁ-\ncally the viable and thus metabolically active fraction of these\nmixed communities. Conclusions from ﬁngerprints derived\nfrom the viable portion could differ substantially from those\noriginating from the study of total (viable and dead) cells (Fig.\n5). It must be mentioned, however, that the community ﬁnger-\nprint shown here should be interpreted with caution, because\nfurther studies are needed to answer the questions whether the\nEMA principle can be applied to a wide spectrum of bacteria\nand whether the same parameters can be applied to all mem-\nbers of a microbial community. So far EMA pretreatment has\nbeen successfully used only with E. coli O157:H7, Salmonella\nserovar Typhimurium, Campylobacter jejuni, and the gram-\npositive bacterium Listeria monocytogenes (18, 24, 25).\nIn summary, EMA treatment of mixed communities might\nimprove the targeting of genomic sequences for determination\nof the potential pathogenic threat originating from viable bac-\nteria, while avoiding less standardized, more troublesome, and\nmore expensive alternative techniques. The method presented\novercomes the currently missing relationship between viability\nand direct PCR detection of DNA targets without preenrich-\nments. Moreover, it could be an extremely useful method in\necological studies by limiting DNA-based community analysis\nto the viable fraction.",
         "Detection of viable and dead Listeria monocytogenes on gouda‐like cheeses by real‐time PCR;Use of Ethidium Monoazide and PCR in Combination for Quantification of Viable and Dead Cells in Complex Samples;Establishment of a Real-Time PCR-Based Approach for Accurate Quantification of Bacterial RNA Targets in Water, Using Salmonella as a Model Organism;Molecular Biology and DNA Microarray Technology for Microbial Quality Monitoring of Water;Detection of enterohemorrhagic Escherichia coli O157:H7 by using a multiplex real-time PCR assay for genes encoding intimin and Shiga toxins.;Ethidium monoazide for DNA-based differentiation of viable and dead bacteria by 5'-nuclease PCR.;Detection and enumeration of coliforms in drinking water: current methods and emerging approaches.;Assessment of viability of microorganisms employing fluorescence techniques.;β-Tubulin mRNA as a Marker ofCryptosporidium parvum Oocyst Viability;rRNA Stability in Heat-Killed and UV-Irradiated EnterotoxigenicStaphylococcus aureus and Escherichia coliO157:H7;The use of a ribosomal RNA targeted oligonucleotide probe for fluorescent labelling of viable Cryptosporidiumparvum oocysts;Assessment of bacterial viability status by flow cytometry and single cell sorting;Detection of mRNA by Reverse Transcription-PCR as an Indicator of Viability in Escherichia coliCells;Viability and activity in readily culturable bacteria: a review and discussion of the practical issues;Determination of Complement-Mediated Killing of Bacteria by Viability Staining and Bioluminescence;Assessment of E. coli and Salmonella viability and starvation by confocal laser microscopy and flow cytometry using rhodamine 123, DiBAC4(3), propidium iodide, and CTC.;Go with the flow – use of flow cytometry in environmental microbiology;Denaturing gradient gel electrophoresis profiles of 16S rRNA-defined populations inhabiting a hot spring microbial mat community;Comparison of cell viability probes compatible with fixation and permeabilization for combined surface and intracellular staining in flow cytometry.;Effect of stress treatments on the detection of Listeria monocytogenes and enterotoxigenic Escherichia coli by the polymerase chain reaction.;Polymerase chain reaction detection of nonviable bacterial pathogens;Demonstration of specific high affinity binding sites in plasmid DNA by photoaffinity labeling with an ethidium analog.;Rapid Membrane Filtration-Epifluorescent Microscopy Technique for Direct Enumeration of Bacteria in Raw Milk;Spectroscopic properties of ethidium monoazide: a fluorescent photoaffinity label for nucleic acids.;Characterization of azidobenzamidines as photoaffinity labels for trypsin.;Selective covalent binding of an ethidium analog to mitochondrial DNA with production of petite mutants in yeast by photoaffinity labelling.;Complex formation between ethidium bromide and nucleic acids.;Control of mRNA processing and decay in prokaryotes;rRNA stability in heatkilled and UV-irradiated enterotoxigenic Staphylococcus aureus and Escherichia coli O157:H7;Physiological assessment of bacteria using fluorochromes.;using rhodamine 123 , DiBAC 4 ( 3 ) , pro - pidium iodide and CTC;1 – mRNA Degradation in Prokaryotic Cells: An Overview;Control of messenger RNA stability.;Use of a photolabeling technique to identify nonviable cells in fixed homologous or heterologous cell populations.",
         "Selective Removal of DNA from Dead Cells of Mixed Bacterial Communities by Use of Ethidium Monoazide",
         "None"
        ],
        [
         "46",
         "002076637c0451e34516f26da871a209b0c201b7",
         "The crystal structure of the E1 component from the Escherichia coli pyruvate dehydrogenase multienzyme complex (PDHc) has been determined with phosphonolactylthiamin diphosphate (PLThDP) in its active site. PLThDP serves as a structural and electrostatic analogue of the natural intermediate α-lactylthiamin diphosphate (LThDP), in which the carboxylate from the natural substrate pyruvate is replaced by a phosphonate group. This represents the first example of an experimentally determined, three-dimensional structure of a thiamin diphosphate (ThDP)-dependent enzyme containing a covalently bound, pre-decarboxylation reaction intermediate analogue and should serve as a model for the corresponding intermediates in other ThDP-dependent decarboxylases. Regarding the PDHc-specific reaction, the presence of PLThDP induces large scale conformational changes in the enzyme. In conjunction with the E1-PLThDP and E1-ThDP structures, analysis of a H407A E1-PLThDP variant structure shows that an interaction between His-407 and PLThDP is essential for stabilization of two loop regions in the active site that are otherwise disordered in the absence of intermediate analogue. This ordering completes formation of the active site and creates a new ordered surface likely involved in interactions with the lipoyl domains of E2s within the PDHc complex. The tetrahedral intermediate analogue is tightly held in the active site through direct hydrogen bonds to residues His-407, Tyr-599, and His-640 and reveals a new, enzyme-induced, strain-related feature that appears to aid in the decarboxylation process. This feature is almost certainly present in all ThDP-dependent decarboxylases; thus its inclusion in our understanding of general thiamin catalysis is important.",
         "P. Arjunan,M. Sax,A. Brunskill,K. Chandrasekhar,N. Nemeria,S. Zhang,F. Jordan,W. Furey",
         "P. Arjunan,M. Sax,A. Brunskill,K. Chandrasekhar,N. Nemeria,S. Zhang,F. Jordan,W. Furey",
         "http://www.jbc.org/article/S0021925820559759/pdf",
         "The crystal structure of the E1 component from the Escherichia\ncoli pyruvate dehydrogenase multienzyme complex (PDHc) has\nbeen determined with phosphonolactylthiamin diphosphate\n(PLThDP) in its active site. PLThDP serves as a structural and elec-\ntrostatic analogue of the natural intermediate -lactylthiamin\ndiphosphate (LThDP), in which the carboxylate from the natural\nsubstrate pyruvate is replaced by a phosphonate group. This repre-\nsents the first example of an experimentally determined, three-di-\nmensional structure of a thiamin diphosphate (ThDP)-dependent\nenzyme containing a covalently bound, pre-decarboxylation reac-\ntion intermediate analogue and should serve as a model for the cor-\nresponding intermediates in other ThDP-dependent decarboxy-\nlases. Regarding the PDHc-specific reaction, the presence of\nPLThDP induces large scale conformational changes in the enzyme.\nIn conjunction with the E1-PLThDP and E1-ThDP structures, anal-\nysis of a H407A E1-PLThDP variant structure shows that an inter-\naction between His-407 and PLThDP is essential for stabilization of\ntwo loop regions in the active site that are otherwise disordered in\nthe absence of intermediate analogue. This ordering completes for-\nmation of the active site and creates a new ordered surface likely\ninvolved in interactions with the lipoyl domains of E2s within the\nPDHc complex. The tetrahedral intermediate analogue is tightly\nheld in the active site through direct hydrogen bonds to residues\nHis-407, Tyr-599, and His-640 and reveals a new, enzyme-induced,\nstrain-related feature that appears to aid in the decarboxylation\nprocess. This feature is almost certainly present in all ThDP-de-\npendent decarboxylases; thus its inclusion in our understanding of\ngeneral thiamin catalysis is important.",
         "DISCUSSION\nThe C2-C (carboxyl) bond in -LThDP is expected to be oriented\nrelative to the thiazolium ring nearly the same (roughly orthogonal) as\nthe C2-P bond seen in -PLThDP, and this orientation had been pre-\ndicted for both compounds (29). Given the structural and electronic\nsimilarities, active site residues adjacent to C2, OH(2), CH3(2), and\nsome of the phosphonyl and carboxylate oxygens are then also expected\nto have very nearly identical locations in both the catalytically active and\nthe analogue intermediates, and the short N4-O2 distance and out-\nof-plane deformation of the C2–C2bond observed in the E1-PLThDP\ncomplex should occur in the E1--LThDP complex as well. For exam-\nple, from Fig. 6, it is clear that the carboxyl group could form the same\nfavorable hydrogen bonds to His-407 and Tyr-599 without any changes\nin the residue positions, and only very subtle changes in these positions\nwould be required to make the interactions even stronger. Accordingly,\nthe E1-PLThDP structure should be an excellent model for the natural\nintermediate involving -LThDP, and conclusions drawn from the\nE1-PLThDP structure\ntransient\nintermediate.\nto the\nThe disorder-to-order transformation induced by the presence of the\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nintermediate analogue PLThDP (and presumably -LThDP) is particu-\nlarly important as it not only completes the interior of the active site but\nalso results in structural changes leading from it out to the protein\nexterior. This was shown in Fig. 2b. With this ordering, a tunnel that can\naccommodate the lipoyl-lysine side chain bound to an E2 component in\nPDHc is completely formed, and a new surface is created at the mouth of\nthe funnel, which, based on the distance from the active site and the\nknown length of the lipoyl-lysine side chain, is likely to bind to the\ncorresponding E2 lipoyl domain. These results are consistent with a\nprevious report indicating that E1-E2 lipoyl domain binding stability is\nincreased in the presence of the substrate pyruvate (31). Given the\nstrong structural similarity between substrate and analogue, we suggest\nthat the ordering observed in the E1-PLThDP complex would also be\npresent when the intermediate from the natural substrate pyruvate is\nformed, and the increased E1-E2 stability may be explained by E2-lipoyl\ndomain interactions with the resulting newly ordered E1 surface\nresidues.\nIt appears that nearly all of the induced structural changes begin with\nthe critical His-407-PLThDP interaction within the active site and then\npropagate outward, both to another previously disordered loop and to\nthe helical region that was always ordered but now has its location\nshifted. This can be understood as follows. The first newly ordered loop\n(401– 413) is initially stabilized by the direct and water-mediated hydro-\ngen bonds formed between a phosphonyl (or carboxyl in the case of\nLThDP) oxygen atom and His-407 and Gln-408 as seen in Fig. 3. The\nsecond newly ordered loop (541–557, located at the mouth of the fun-\nnel), although not forming any direct interactions with PLThDP/\nLThDP, is then initially stabilized through the formation of hydrogen\nbonds between Asn-404 in the (now ordered) first loop and the side\nchains of Gln-548 and Asp-549. Further stabilization of this segment\noccurs via interactions near the ends of the (always ordered) helical\nsegment Thr-525–Ile-535, which shifts by 2.7 Å relative to the native\nstructure. These interactions involve formation of hydrogen bonds\nbetween Ile-537 and Lys-557 and between Arg-524 and Ala-554, with\nthe latter being water-mediated. Although only the directed and specific\nhydrogen bonds were described, numerous favorable van der Waals\ncontacts are also present and contribute to the stabilization. The end\nresult is that the reaction intermediate, both newly ordered loops, and\nthe shifted helical segment are all interlinked through hydrogen-bond-\ning and van der Waals contacts to form a more tightly packed dimer\nwith a completed active site channel and a new surface near its entrance.\nThese results are consistent with the fact that the disorder-to-order\ntransformation occurs only when both His-407 and the intermediate\nanalogue PLThDP (or presumably LThDP) are present in the enzyme.\nOur previous biochemical studies on the H407A E1 variant clearly\nindicated its importance in post-decarboxylation steps for the PDHc\nreaction (19). The mutation was shown to dramatically decrease the rate\nof reductive acetylation, which could be caused either by direct partic-\nipation in the catalytic reaction at the active site or by interfering with\nproper E1-E2 association within the PDHc multienzyme complex. The\nevidence was that the specific activity of this variant in terms of the\noverall PDHc assay is less than 0.15%, whereas there are only minor\nactivity effects on the initial decarboxylation reaction step. The current\nresults and interpretation are consistent with this study and imply that\nHis-407 does indeed play a significant role in binding E1 to the E2-lipoyl\ndomains, although direct participation, at least in the later stages of\ncatalysis, cannot be ruled out. The observed change in the active site and\nits entrance pathway in the presence of substrate/analogue likely facili-\ntates binding with the E2-lipoyl domain and insertion of the lipoyl-\nlysine side chain, so that the required (post decarboxylation) reductive\nacetylation can occur.\nWe also previously suggested that His-407 functions as a proton\ndonor to a lipoamide sulfur prior to the addition of the enamine to the\ndisulfide (19). It is now tempting to suggest that this protonation occurs\nafter carbon dioxide release because of the availability of His-407 at that\ninstant. When pyruvate adds to form the pre-decarboxylation complex,\nHis-407 would form a hydrogen bond with the pyruvate adduct and\ntrigger the disorder-order transformation in preparation to receive the\nlipoamide of the lipoyl domain. As the interaction with E2 progresses\nand the C2-C (carboxyl) bond is severed, releasing CO2, the hydrogen\nbonds from the carboxyl group to His-407 and Tyr-599 obviously will\nno longer exist, and the lipoyl domain lipoamide can move into the\nspace vacated by the CO2 molecule in preparation to seize the acyl\ngroup. An intriguing question arises about events that occur during the\ninterim period between decarboxylation and reductive-acetylation.\nDoes a reversal of the conformational transformation (order-to-disor-\nder) take place upon severance of the hydrogen bond between His-407\nand the carboxylate group, or does the ordered structure remain intact,\nat least temporarily, until reductive acetylation is completed? In the first\ninstance, the implication is that the ordered structure is unnecessary for\nthe remaining reductive acetylation function to be completed in E1. In\nthe second case, His-407 would be required to form a hydrogen bond to\nanother group (on the lipoyl domain or on E1) until reductive acetyla-\ntion is completed. An answer to these questions hopefully will be\nobtained when structural information on a complete E1-E2 complex\nbecomes available.\nIt is noteworthy that the order-disorder transformation in this 2 E1\ndimer differs considerably from those that have been observed in certain\n22 E1 heterotetramers (14, 32). In the latter cases, the addition of the\ncofactor alone to the disordered apoenzyme produced an ordered\nholoenzyme. In the 2 homodimeric E1 protein, disorder occurs in both\nthe apoenzymes and holoenzymes, and it is only the formation of the\npre-decarboxylation reaction intermediate that triggers ordering. In\naddition, the loops in 2 E1 that exhibit transformation are not the same\nas those reported to undergo transformation in the 22 cases (14, 32,\n33).\nthe\ntransformation in\nhomodimeric 2 E1s induced by substrate binding apparently is to acti-\nvate the enzyme for reductive acetylation by preparing the channel in\nthe active site to receive the E2 lipoyl domain lipoamide group. Inter-\nestingly, the phosphorylation regulatory mechanism responsible for\ndeactivating an 22 heterotetrameric E1b component operating on\nbranched chains also involves an order/disorder transformation in the\ncorresponding sequence segment (34) but functions in reverse! In that\ncase, substrate binding is not required as the E1b holoenzyme structure\nis already fully ordered, and phosphorylation deactivates the enzyme by\ninducing disorder within the segment. It is noteworthy that the phos-\nphorylation site, Ser-292-, corresponds to Gln-408 in the 2 E1 struc-\nture, which is not only in the newly ordered segment but is adjacent to\nthe critical His-407 residue! Thus both dimeric and tetrameric E1s\nappear to effectively utilize disorder/order transformations in the same\nregions with similar trigger points but in different ways and in response\nto different stimuli.\nFinally, with regard to general, enzyme-mediated thiamin catalysis,\nwe now suggest that removal of strain created by the expected, corre-\nsponding, out-of-plane distortion in the -LThDP complex may be a\ncontributing force behind the accelerated decarboxylation that is seen\nwhen the thiamin-catalyzed reaction occurs in enzymes and that the\nstrained structure may represent a transition state analogue. We have\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nshown that the out-of-plane distortion is caused primarily by a short\nand repulsive N4-O2distance, which is the result of the V conforma-\ntion imposed on the ThDP cofactor by the enzyme. This is apparent\nsince in the absence of enzyme, the PLThDP structure has a C2–C2\nbond precisely in the plane of the thiazolium ring but has no short\ncontacts because it is not in the V conformation (29). Once forced in the\nV conformation by the enzyme, however, if left in the planar state, an\nextremely short and excessively repulsive contact of 2.2 Å would result\n(Fig. 7a). To reduce the highly undesirable effects of this short contact,\nthe non-planar distortion occurs, increasing the “bad” contact distance\nto a more tolerable 2.5 Å as is found in the E1-PLThDP structure and\nshown in Fig. 7b, although this still represents a strained molecule.\nComplete removal of the strain would occur only upon decarboxylation\nas then, the totally planar enamine form (Fig. 1, top right object) is cre-\nated in which the formerly bad contact distance is increased to a favor-\nable 2.8 Å as seen in Fig. 7c. The distorted, observed structure reported\nhere and shown in Fig. 7b is clearly shifted toward the product enamine\nstructure, both in terms of the N4-O2 distance and in terms of the\nmethyl carbon and O2locations, which are now close to being in the\nplane of the thiazolium ring. It could thus be described as a transition\nstate, in which the energy trapped in the C2out-of-plane distortion is\nutilized to help break the C-CO2 bond to form the enamine, in a manner\nrequiring minimal atomic motion for the remaining atoms to attain\ntheir positions in the product state. Note that if the alternative initial\ndecarboxylated form, the tetrahedral C2carbanion (Fig. 1, top, second\nobject from the right), were created, the same strained out-of-plane dis-\ntortion observed in the original pre-decarboxylation complex would\npersist, suggesting that the planar enamine form is indeed preferred.\nThis is consistent with the fact that in the published post-decarboxyla-\ntion crystal structures (11, 12, 14), the intermediates that were reported\nto best fit the electron density corresponded to planar enamines rather\nthan tetrahedral C2carbanions.",
         "* This work was supported by a grant from the Veterans Affairs Merit Review Program\nand National Institutes of Health Grant GM-61791 (to W. F.) and by National Institutes\nof Health Grant GM-62330 (to F. J.). The costs of publication of this article were\ndefrayed in part by the payment of page charges. This article must therefore be\nhereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to\nindicate this fact.\nof an aldehyde or an acyl group (1–5). Examples include the E1 compo-\nnents in pyruvate dehydrogenase complexes (PDHc),2 pyruvate decar-\nboxylase, transketolase, etc. Crystallographic studies (6 –12) have eluci-\ndated many of the structural, stereochemical, and biochemical details in\nthe mechanism of action of these enzymes and in the catalytic role of the\ncofactor ThDP (thiamin diphosphate, vitamin B1 diphosphate, Fig. 1,\ntop left). Despite the enormous contributions made by these and other\nstudies to our understanding of how such enzymes function, important\ndetails still remain obscure. There are, for example, no detailed struc-\ntural data on the first ThDP-bound intermediate in the presence of any\nenzyme (for example, -lactylthiamin diphosphate (-LThDP) in PDHc\nE1 and pyruvate decarboxylase), which is postulated to form in the\ncurrently accepted mechanism of thiamin catalysis (Fig. 1, top, third\nobject from the right). In an effort to obtain structural information per-\ntaining to this key intermediate, we have determined the crystal struc-\nture of PDHc E1 from Escherichia coli in complex with -phosphono-\nlactylthiamin diphosphate (PLThDP).\nPLThDP is the product of the reaction between ThDP and methyla-\ncetylphosphonate, with the latter being an analogue of the true substrate\npyruvate and a potent inhibitor of PDHc. The complex formed with\nPLThDP instead of ThDP therefore mimics the structure of the\nenzyme-bound, reactive tetrahedral intermediate -LThDP (13) in the\ndecarboxylation step of the PDHc E1 reaction. It differs from the com-\nplex formed with the true substrate only in the replacement of the car-\nboxylate group by a methyl phosphonate (PO3Me) group. However,\nunlike the C2-CO2 bond normally cleaved in the reaction with pyru-\nvate, the C2-PO3Me bond remains intact. The reaction is therefore\ntrapped in a pre-CO2 release-like state, and the structure represents a\ncovalently bound, pre-decarboxylation reaction intermediate analogue.\nThere have been three covalently bound reaction intermediate struc-\ntures reported for ThDP-dependent enzymes (11, 12, 14), but they all\nrepresented the planar enamine intermediate (Fig. 1, top right object)\nthat exists only after decarboxylation. The E1-PLThDP structure is thus\nthe first structural example of a covalently bound, pre-decarboxylation\nreaction intermediate analogue in any ThDP-dependent enzyme.\n2 The abbreviations used are: PDHc, pyruvate dehydrogenase multienzyme complex;\nThDP, thiamin diphosphate (vitamin B1 diphosphate); LThDP, lactylthiamin diphos-\nphate (reaction intermediate); PLThDP, phosphonolactylthiamin diphosphate (reac-\ntion intermediate analogue); E1, first enzymatic component of multienzyme com-\nplexes related to and including PDHc; parental E1, protein only component of E1;\nE1-ThDP, parental E1 with ThDP cofactor; E1-PLThDP, parental E1 with PLThDP;\nH407A E1-PLThDP, H407A parental E1 variant with PLThDP.\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nThe E1 component of the PDHc complex catalyzes the rate-limiting\nstep of the overall PDHc reaction and therefore also provides an ideal\ntarget for mechanistic structural investigation. Initial crystallographic\nresults for the E1-PLThDP complex have been reported previously (15),\nand complete, detailed results at 2.1 Å resolution are now presented\nwith the analysis revealing several unique structural features that are\nabsent in the native enzyme. 1) The enzyme-bound conformation of\nPLThDP is the V form, rather than the S form determined many years\nago for the same compound in the absence of enzyme, showing how the\nenzyme can enforce this highly strained V conformation even in the\nintermediate analogue. 2) The structure identifies a role for residue\nHis-407, explaining previous biochemical data, and shows the direct\ninteraction (and implied function) of several additional active center\nresidues with the covalently bound adduct. 3) Major reorganization of\nthe active center takes place and includes the ordering of two key loops\nnot seen in any of our other PDHc E1 structures before; thus the\nPLThDP dramatically diminishes their mobility. 4) A marked deviation\nfrom co-planarity of the C2–C2 bond with the thiazolium ring is\nclearly revealed and suggests likely mechanistic consequences. 5) The\nstructure shows strong interaction of the C2-OH with the N4 atom of\nthe 4-aminopyrimidine of ThDP, as evidenced by a short contact dis-\ntance. Elsewhere, we used PLThDP in solution to show that PLThDP\nexists with the 1,4-iminoThDP tautomeric form (16, 17) of the\n4-aminopyrimidine ring, the first instance in which this putative inter-\nmediate could be stabilized. The observed short O–N4 distance is fully\nconsistent with this tautomerization. The structural reorganization and\nordering resulting from the presence of PLThDP deep in the active site\nextends to the enzyme exterior, providing a new surface likely involved\nin interactions with other enzymatic components within the PDHc\nmultienzyme complex.",
         "EXPERIMENTAL PROCEDURES\nCrystallization and Data Collection—The PDHc E1 from E. coli was\npurified and assayed according to published procedures (18). ThDP,\nwhen present, was removed by gel filtration using an Amersham Bio-\nsciences G-25 column. For crystallization, the protein was dialyzed\nagainst 20 mM HEPES buffer at pH 7.0, containing 5 mM dithiothreitol,\n0.2% NaN3, 0.5 mM EDTA, and 1 M leupeptin. The H407A E1 variant\nwas purified and assayed as reported previously (19). The parental and\ncorresponding variant E1 proteins were independently co-crystallized\nwith PLThDP and Mg2 by the sitting drop vapor diffusion method\nunder conditions similar to that used for our earlier E1-ThDP crystal-\nlographic studies (8). The best crystals were obtained at a reservoir\nsolution with 15–20% polyethylene glycol 2000 monomethyl ether, 10%\npropanol, 0.2% NaN3, and 60 mM Hepes buffer (pH 7.05) at 22 °C. Drops\nwere 6 –10 l consisting of equal parts of reservoir and protein solution.\nCrystal growth took about 4 – 6 weeks with crystals typically having\ndimensions of 0.1  0.20  0.30 mm.\nX-ray diffraction data for the E1-PLThDP complex were collected\nfrom a single crystal flash-cooled at 180°C on a Bruker HiSTAR detec-\ntor, using a Rigaku RU200 rotating anode generator as source. An\nOsmic mirror system was used along with a thin nickel foil to provide a\nhighly focused, intense beam of CuKradiation. The crystals are mon-\noclinic with space group P21 and cell constants a  81.6, b  142.5, c \n82.1 Å, and  102.4°, with two molecules (subunits) in an asymmetric\nunit. To 2.1 Å resolution, 466,210 total observations were reduced to\nyield 99,699 unique reflections (94% complete) with an internal R factor\n(based on intensities) of 0.07. The data were processed with the XGEN\npackage (20) and scaled with a local version of Weissman’s Fourier\nscaling program FBSCALE (21). For the H407A E1-PLThDP complex,\nsynchrotron data were collected at SERCAT (Advanced Photon Source)\nand processed with the HKL2000 package (22).\nStructure Determination and Refinement—Since the previous\nE1-ThDP crystals are isomorphous with both E1-PLThDP and H407A\nE1-PLThDP crystals (cell constants differing by less than 0.8 Å), atomic\ncoordinates from the former structure were used as the starting model\nfor both refinements. The initial model included 1602 amino acids and\nomitted the ThDP cofactors. Following rigid body refinement, the\nE1-PLThDP model was further refined by simulated annealing using the\nprogram XPLOR (23) without imposing any non-crystallographic sym-\nmetry restraints. After the initial refinement, examination of electron\ndensity difference maps clearly showed strong density for the interme-\ndiate analogue, PLThDP, and also produced strong and interpretable\nelectron density for two of the three previously unobserved regions,\n401– 413 and 541–557. After including the PLThDP and the two miss-\ning regions, the model was refined with simulated annealing, and sub-\nsequent cycles consisted of positional and B factor refinement. The\nrefinement procedure included periodic examinations of simulated\nannealing omit and difference maps, as well as the introduction of water\nmolecules. A final examination of difference electron density maps still\ndid not produce interpretable electron density for the N-terminal resi-\ndues 1–55, and they are excluded from the model. Inspection of electron\ndensity maps, model building, and structural comparisons were carried\nout by using the program O (24). For all data to 2.1 Å resolution, the R\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nfactor is 0.203 (0.213 in the last shell), and the Rfree(based on 5.3% of\ndata) is 0.254 (0.291 in the last shell). The final structure has good ste-\nreochemistry with root mean square deviations from ideality of 0.006 Å\nand 1.3° for bond lengths and angles, respectively. The final model con-\ntains two independent subunits, each with 831 amino acids out of a\npossible 886, as well as two PLThDP-Mg2 cofactor pairs. This model\nalso includes 564 water molecules, five phosphate ions, and two cis-\nprolines (Pro-463) per asymmetric unit. The model was analyzed with\nPROCHECK (25), and 89% of the residues are in the most favored region\nin the Ramachandran plot (26). The Ramachandran plot shows no res-\nidues in disallowed regions in either subunit.\nThe structure of the H407A E1-PLThDP complex was determined\nand refined in a similar manner. However, in this structure, although the\nPLThDP was clearly present in strong density, the regions 1–55, 401–\n413, and 541–557 were completely disordered. The new structure was\nrefined using the program CNS (27). The final model, which included\n550 water molecules, resulted in an R factor of 21.6% and a Rfree of 24.1%\nto 1.85 Å resolution. The slightly higher R factor for the H407A\nE1-PLThDP variant may be attributed to the fact that this structure\ncontains 60 disordered residues absent from the model (the regions\n401– 413 and 541–557 in each molecule) that were ordered in the\nE1-PLThDP structure. Ramachandran statistics are similar to those for\nthe E1-PLThDP structure. All data collection and refinement statistics\nare given in Table 1. Graphical representations of protein models were\ngenerated by using the program RIBBONS (28). Coordinates and struc-\nture factors for the E1-PLThDP and H407A E1-PLThDP structures\nhave been deposited in the Protein Data Bank with the access codes\n2G25 and 2G28, respectively.",
         "EXPERIMENTAL PROCEDURES\nCrystallization and Data Collection—The PDHc E1 from E. coli was\npurified and assayed according to published procedures (18). ThDP,\nwhen present, was removed by gel filtration using an Amersham Bio-\nsciences G-25 column. For crystallization, the protein was dialyzed\nagainst 20 mM HEPES buffer at pH 7.0, containing 5 mM dithiothreitol,\n0.2% NaN3, 0.5 mM EDTA, and 1 M leupeptin. The H407A E1 variant\nwas purified and assayed as reported previously (19). The parental and\ncorresponding variant E1 proteins were independently co-crystallized\nwith PLThDP and Mg2 by the sitting drop vapor diffusion method\nunder conditions similar to that used for our earlier E1-ThDP crystal-\nlographic studies (8). The best crystals were obtained at a reservoir\nsolution with 15–20% polyethylene glycol 2000 monomethyl ether, 10%\npropanol, 0.2% NaN3, and 60 mM Hepes buffer (pH 7.05) at 22 °C. Drops\nwere 6 –10 l consisting of equal parts of reservoir and protein solution.\nCrystal growth took about 4 – 6 weeks with crystals typically having\ndimensions of 0.1  0.20  0.30 mm.\nX-ray diffraction data for the E1-PLThDP complex were collected\nfrom a single crystal flash-cooled at 180°C on a Bruker HiSTAR detec-\ntor, using a Rigaku RU200 rotating anode generator as source. An\nOsmic mirror system was used along with a thin nickel foil to provide a\nhighly focused, intense beam of CuKradiation. The crystals are mon-\noclinic with space group P21 and cell constants a  81.6, b  142.5, c \n82.1 Å, and  102.4°, with two molecules (subunits) in an asymmetric\nunit. To 2.1 Å resolution, 466,210 total observations were reduced to\nyield 99,699 unique reflections (94% complete) with an internal R factor\n(based on intensities) of 0.07. The data were processed with the XGEN\npackage (20) and scaled with a local version of Weissman’s Fourier\nscaling program FBSCALE (21). For the H407A E1-PLThDP complex,\nsynchrotron data were collected at SERCAT (Advanced Photon Source)\nand processed with the HKL2000 package (22).\nStructure Determination and Refinement—Since the previous\nE1-ThDP crystals are isomorphous with both E1-PLThDP and H407A\nE1-PLThDP crystals (cell constants differing by less than 0.8 Å), atomic\ncoordinates from the former structure were used as the starting model\nfor both refinements. The initial model included 1602 amino acids and\nomitted the ThDP cofactors. Following rigid body refinement, the\nE1-PLThDP model was further refined by simulated annealing using the\nprogram XPLOR (23) without imposing any non-crystallographic sym-\nmetry restraints. After the initial refinement, examination of electron\ndensity difference maps clearly showed strong density for the interme-\ndiate analogue, PLThDP, and also produced strong and interpretable\nelectron density for two of the three previously unobserved regions,\n401– 413 and 541–557. After including the PLThDP and the two miss-\ning regions, the model was refined with simulated annealing, and sub-\nsequent cycles consisted of positional and B factor refinement. The\nrefinement procedure included periodic examinations of simulated\nannealing omit and difference maps, as well as the introduction of water\nmolecules. A final examination of difference electron density maps still\ndid not produce interpretable electron density for the N-terminal resi-\ndues 1–55, and they are excluded from the model. Inspection of electron\ndensity maps, model building, and structural comparisons were carried\nout by using the program O (24). For all data to 2.1 Å resolution, the R\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nfactor is 0.203 (0.213 in the last shell), and the Rfree(based on 5.3% of\ndata) is 0.254 (0.291 in the last shell). The final structure has good ste-\nreochemistry with root mean square deviations from ideality of 0.006 Å\nand 1.3° for bond lengths and angles, respectively. The final model con-\ntains two independent subunits, each with 831 amino acids out of a\npossible 886, as well as two PLThDP-Mg2 cofactor pairs. This model\nalso includes 564 water molecules, five phosphate ions, and two cis-\nprolines (Pro-463) per asymmetric unit. The model was analyzed with\nPROCHECK (25), and 89% of the residues are in the most favored region\nin the Ramachandran plot (26). The Ramachandran plot shows no res-\nidues in disallowed regions in either subunit.\nThe structure of the H407A E1-PLThDP complex was determined\nand refined in a similar manner. However, in this structure, although the\nPLThDP was clearly present in strong density, the regions 1–55, 401–\n413, and 541–557 were completely disordered. The new structure was\nrefined using the program CNS (27). The final model, which included\n550 water molecules, resulted in an R factor of 21.6% and a Rfree of 24.1%\nto 1.85 Å resolution. The slightly higher R factor for the H407A\nE1-PLThDP variant may be attributed to the fact that this structure\ncontains 60 disordered residues absent from the model (the regions\n401– 413 and 541–557 in each molecule) that were ordered in the\nE1-PLThDP structure. Ramachandran statistics are similar to those for\nthe E1-PLThDP structure. All data collection and refinement statistics\nare given in Table 1. Graphical representations of protein models were\ngenerated by using the program RIBBONS (28). Coordinates and struc-\nture factors for the E1-PLThDP and H407A E1-PLThDP structures\nhave been deposited in the Protein Data Bank with the access codes\n2G25 and 2G28, respectively.\nRESULTS\nThe main chain folds in PDHc E1-ThDP (8) and in its E1-PLThDP\ncounterpart are very similar, with both structures containing two sub-\nunits in an asymmetric unit. The two independent subunits are almost\nidentical stereochemically and are related by a 2-fold non-crystallo-\ngraphic symmetry axis. The complete dimer is shown in Fig. 2a. Each\nE1-PLThDP subunit consists of a single polypeptide chain folded into\nthree domains: the N terminus (1– 470), the middle (471–700), and the\nC terminus (701– 886). Least-squares fitting of 801 common -carbon\natoms in the E1-ThDP and E1-PLThDP structures results in a root\nmean square deviation of 0.40 Å. However, significant main-chain devi-\nations (up to 2.7 Å) occur in the middle domain helix containing resi-\ndues 525–535, which are involved in close contacts with residues pres-\nent in the active site channel. Except for a few side chains at the surface\nof the protein, the electron density over the entire molecule is generally\nvery well defined, but as in the E1-ThDP structure, there is no interpret-\nable electron density for the N-terminal residues 1–55.\nA surprising observation in the E1-PLThDP map is the presence of\nwell defined electron density for loop residues 401– 413 and 541–557\nthat was absent in the map of the E1-ThDP structure due to complete\ndisorder. These two loops form part of the active site channel situated at\nthe dimer interface. The 401– 413 loop forms part of the active site, and\nresidue His-407 in this loop forms a critical hydrogen bond to an oxygen\natom in the substrate analogue phosphonyl group. This loop stabilizes\nthe other newly ordered loop 541–557, assisted by interactions created\nfrom the large movement in the helical region 525–535. The ordering\ncreates new interface interactions between residues Lys-403–Asn-404\nfrom one subunit and residues Gln-548 –Asp-549 from the other sub-\nunit. The two newly ordered regions thus interact with each other and\npropagate outward from the active site to the enzyme surface. The\nnewly ordered regions are shown in terms of the overall structure in Fig.\n2a and in greater clarity regarding the active site and enzyme surface in\nFig. 2b. Electron density for the 401– 413 loop and its interactions with\nPLThDP are shown in Fig. 3.\nThe average B value for main and side chain atoms is 9.5 and 11.2 Å2,\nrespectively. These values are significantly lower than those observed in the\nE1-ThDP structure (16.0 and 18.6 Å2, respectively) (8), implying a more\nordered assembly. Accordingly, the presence of the reaction intermediate\nanalogue in the active site supports the formation of a tighter E1 dimer. The\nnow ordered loop regions provide a new exterior surface that can conceiv-\nably interact with other subunits of the PDHc complex and also partially\nseal the active site entrance, leading to a more hydrophobic catalytic region.\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nThe H407A E1-PLThDP structure (not shown) reveals the active site to\nbe intact and the structure generally unchanged when compared with the\nE1-ThDP (8) and E1-PLThDP structures. As in E1-ThDP and unlike in the\nE1-PLThDP structure, however, a final difference electron density map did\nnot contain interpretable electron density for loop regions 401– 413 and\n541–557; the N-terminal region 1–55 is unobserved in all of the structures.\nWhen combined with the other structures, analysis of the H407A\nE1-PLThDP structure therefore confirmed the need for the His-407-\nPLThDP interaction in inducing the disorder-to-order transformation in\ntwo loops as the loops are ordered only when both the intermediate and\nHis-407 are present. Clearly, the ordering is induced by the presence of\nPLThDP via direct hydrogen-bonding to residue His-407 and results in\nproviding previously missing structural elements to complete both the\nexterior surface and the interior surface leading to the active sites.\nFor E1-PLThDP, a difference electron density map calculated with\nIt is important to note that the E1-ThDP and E1-PLThDP complex crystals\nare isomorphous; thus the ordering cannot be attributed to new packing\ncontacts in a different crystal form and arises only from the presence of the\nadduct now covalently bound to the cofactor.\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nthe protein-only model\nindicated strong positive density for the\nPLThDP (Fig. 4). Funnel-shaped channels leading to the active sites are\nabout 24 Å deep, with walls formed by residues from both the N-termi-\nnal domain and the middle domain. In the absence of enzyme, a prior\nPLThDP crystal structure analysis (29) revealed that the thiamin por-\n\ntion of the molecule assumes the S conformation (\nt\n173.6°), which is characteristic of other C2-substituted thiamins and\nplaces C2 and its bound adduct distant from N4 of the aminopyrimi-\ndine ring. The enzyme-bound PLThDP, however, adopts the V confor-\np of 100.5 and 67°, respec-\nmation with torsion angles (30) \ntively, and results in crowding near C2. The C2-PO3Me bond is\ndirected very nearly perpendicular to the thiazolium ring. The PLThDP\nconformation is stabilized by hydrogen-bonding interactions with Tyr-\n599, His-640, His-407, and Gln-408 (through a water). The active site\ncleft is lined with residues His-106, Ser-109, Gln-140, His-142, Tyr-177,\nMet-194, Asp-230, Glu-235, Asn-260, Leu-264, Lys-392, His-407, and\nGln-408 from one subunit and residues Asp-521, Thr-525, Ile-569, Glu-\n571, Tyr-599, Phe-602, Glu-636, and His-640 from the other subunit. A\nstereo view of the dimer interface environment around the catalytic\ncenter is shown in Fig. 5. The structure reveals residues on the enzyme\nthat are responsible for stabilization of the intermediate and that are\ncapable of participating in catalysis as well. On one side of the thiazo-\nlium ring, in the vicinity of the covalent adduct on C2, is a cluster of four\nhistidine residues. Of these, His-142 is involved in binding with the\ndiphosphate group of the cofactor, whereas His-407 and His-640 inter-\nact with oxygen atoms in the adduct. The fourth, His-106, is bound to a\nwater molecule. As seen earlier, His-407 is hydrogen-bonded to one of\nthe phosphonyl oxygens, and His-640 is hydrogen-bonded to the C2-\nhydroxyl oxygen atom. Tyr-599 also forms a hydrogen bond with one of\nthe phosphonyl oxygen atoms. Other residues (Tyr-177, Gln-408, and\nGlu-636) interact indirectly with the intermediate analogue by hydro-\ngen-bonding through water molecules. Leu-264 appears to interact with\nthe analogue through Van der Waals contacts, with a terminal methyl\ncarbon making a 3.52 Å contact distance to the one of the phosphonyl\noxygen atoms of the adduct.\nIn addition to inducing disorder-to-order transformations, the pres-\nence of PLThDP in the active site also induces conformational changes\nin nearby protein residues that were previously ordered. Relative to the\nE1-ThDP structure, a conformational rearrangement of protein resi-\ndues Leu-264, Glu-522, and Tyr-599 occurs in the E1-PLThDP struc-\nture. Residue Glu-522 shows a large conformational change as its side\nchain moved more than 4 Å. The movement of residues Leu-264 and\nTyr-599 and the presence of the phosphonyl oxygen atom close to the\nthiazolium ring play an important role in the movement of the side\nchain of Glu-522, which is no longer pointing toward the active site.\nAlthough it may not play a direct role in the mechanism of catalysis at\nthis stage, Glu-522 still interacts with the cofactor through water mol-\necules. The key interactions with PLThDP are all likely to be present in\nthe corresponding LThDP intermediate formed from the natural sub-\nstrate pyruvate, as indicated in Fig. 6, since the majority of good hydro-\ngen bonds can still be formed.\nPerhaps most surprisingly, analysis of the E1-PLThDP structure\nrevealed a significant distortion in planarity for the C2–C2bond con-\nnecting the substrate analogue to the planar thiazolium ring, as seen in\nFig. 4. This out-of-plane distortion implies considerable strain and per-\nsists despite incorporation of a planarity restraint during refinement,\neven after significantly increasing the restraint weight well beyond that\nnormally used. To determine whether this effect is real or simply\ninduced by other possibly incorrect refinement restraints, refinements\nand pure energy minimizations were carried out with a variety of stand-\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nard force fields. In all cases, the out-of-plane distortion persists (to vary-\ning but significant degrees), provided that any reasonable van der Waals\nradii are used for the aminopyrimidine N4 and adduct C2-hydroxyl O\natoms, and in any event, the distortion is required to fit the electron\ndensity. The resulting distance between these atoms (2.5 Å) is reasona-\nble, but it would be unacceptably short (2.2 Å), producing a strong\nrepulsive force in the absence of the distortion, thus explaining the\nstrained conformation that is observed. Furthermore, since only three\nessentially rigid groups (coordinates obtained from the highly accurate\nsmall molecule PLThDP crystal structure (29)) are involved, the short\ncontact is dictated by only the thiazolium and aminopyrimidine relative\nring orientations and by the fact that the C2-PO3Me bond is nearly\nperpendicular to the thiazolium ring plane. The \np torsion\nangles defining the ring orientations, as well as the S–C2–C2–P tor-\nsion angle, are clearly required to fit the strong and unambiguous elec-\ntron density, and the former are also consistent with those observed in\nall reported ThDP-containing enzyme structures. Therefore the short\ncontacts cannot be explained away by coordinate errors due to data\nresolution arguments. In addition, the distortion and short contacts are\nalso present in the higher resolution H407A E1-PLThDP variant\nstructure.\nt and \nIt is important to realize that the strain is removed immediately upon\ndecarboxylation if the enamine (Fig. 1, top right object) is formed as the\nformerly tetrahedral C2carbon then adopts sp2 hybridization, increas-\ning the distance between N4 and the oxygen. It thus appears that\nremoval of this strain may be a contributing force behind decarboxyla-\ntion. If this is the case, the cofactor V conformation imposed by the\nenzyme environment is important not only in creating the initial reac-\ntive ylide (6) (Fig. 1, third object from the left) but also in inducing stress\nto facilitate decarboxylation. This is a very important finding as the\nsame forces should be operating in all ThDP-dependent decarboxylases.\nThis view is strongly supported by the fact that when the PLThDP\nmolecule is free of the enzyme (and therefore in the S conformation), it\nshows no evidence of the strain that occurs when bound to the enzyme\nin the V conformation (29). For example, in the S form, the C2-C2\nexocyclic bond is coplanar with the thiazolium ring, the valence angles\nsubtended at C2are all near the ideal tetrahedral value, and O2and\nN4 are not in contact. Therefore the strain associated with the V con-\nformation imposed by the enzyme is avoided entirely in the isolated\nPLThDP molecule. Structural differences between a planar and severely\nthe observed partially\nstrained pre-decarboxylation intermediate,\nstrained, non-planar pre-decarboxylation intermediate, and totally\nrelaxed post-decarboxylation intermediate are shown in Fig. 7.",
         "downloading",
         "The crystal structure of the E1 component from the Escherichia\ncoli pyruvate dehydrogenase multienzyme complex (PDHc) has\nbeen determined with phosphonolactylthiamin diphosphate\n(PLThDP) in its active site. PLThDP serves as a structural and elec-\ntrostatic analogue of the natural intermediate -lactylthiamin\ndiphosphate (LThDP), in which the carboxylate from the natural\nsubstrate pyruvate is replaced by a phosphonate group. This repre-\nsents the first example of an experimentally determined, three-di-\nmensional structure of a thiamin diphosphate (ThDP)-dependent\nenzyme containing a covalently bound, pre-decarboxylation reac-\ntion intermediate analogue and should serve as a model for the cor-\nresponding intermediates in other ThDP-dependent decarboxy-\nlases. Regarding the PDHc-specific reaction, the presence of\nPLThDP induces large scale conformational changes in the enzyme.\nIn conjunction with the E1-PLThDP and E1-ThDP structures, anal-\nysis of a H407A E1-PLThDP variant structure shows that an inter-\naction between His-407 and PLThDP is essential for stabilization of\ntwo loop regions in the active site that are otherwise disordered in\nthe absence of intermediate analogue. This ordering completes for-\nmation of the active site and creates a new ordered surface likely\ninvolved in interactions with the lipoyl domains of E2s within the\nPDHc complex. The tetrahedral intermediate analogue is tightly\nheld in the active site through direct hydrogen bonds to residues\nHis-407, Tyr-599, and His-640 and reveals a new, enzyme-induced,\nstrain-related feature that appears to aid in the decarboxylation\nprocess. This feature is almost certainly present in all ThDP-de-\npendent decarboxylases; thus its inclusion in our understanding of\ngeneral thiamin catalysis is important.\n* This work was supported by a grant from the Veterans Affairs Merit Review Program\nand National Institutes of Health Grant GM-61791 (to W. F.) and by National Institutes\nof Health Grant GM-62330 (to F. J.). The costs of publication of this article were\ndefrayed in part by the payment of page charges. This article must therefore be\nhereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to\nindicate this fact.\nof an aldehyde or an acyl group (1–5). Examples include the E1 compo-\nnents in pyruvate dehydrogenase complexes (PDHc),2 pyruvate decar-\nboxylase, transketolase, etc. Crystallographic studies (6 –12) have eluci-\ndated many of the structural, stereochemical, and biochemical details in\nthe mechanism of action of these enzymes and in the catalytic role of the\ncofactor ThDP (thiamin diphosphate, vitamin B1 diphosphate, Fig. 1,\ntop left). Despite the enormous contributions made by these and other\nstudies to our understanding of how such enzymes function, important\ndetails still remain obscure. There are, for example, no detailed struc-\ntural data on the first ThDP-bound intermediate in the presence of any\nenzyme (for example, -lactylthiamin diphosphate (-LThDP) in PDHc\nE1 and pyruvate decarboxylase), which is postulated to form in the\ncurrently accepted mechanism of thiamin catalysis (Fig. 1, top, third\nobject from the right). In an effort to obtain structural information per-\ntaining to this key intermediate, we have determined the crystal struc-\nture of PDHc E1 from Escherichia coli in complex with -phosphono-\nlactylthiamin diphosphate (PLThDP).\nPLThDP is the product of the reaction between ThDP and methyla-\ncetylphosphonate, with the latter being an analogue of the true substrate\npyruvate and a potent inhibitor of PDHc. The complex formed with\nPLThDP instead of ThDP therefore mimics the structure of the\nenzyme-bound, reactive tetrahedral intermediate -LThDP (13) in the\ndecarboxylation step of the PDHc E1 reaction. It differs from the com-\nplex formed with the true substrate only in the replacement of the car-\nboxylate group by a methyl phosphonate (PO3Me) group. However,\nunlike the C2-CO2 bond normally cleaved in the reaction with pyru-\nvate, the C2-PO3Me bond remains intact. The reaction is therefore\ntrapped in a pre-CO2 release-like state, and the structure represents a\ncovalently bound, pre-decarboxylation reaction intermediate analogue.\nThere have been three covalently bound reaction intermediate struc-\ntures reported for ThDP-dependent enzymes (11, 12, 14), but they all\nrepresented the planar enamine intermediate (Fig. 1, top right object)\nthat exists only after decarboxylation. The E1-PLThDP structure is thus\nthe first structural example of a covalently bound, pre-decarboxylation\nreaction intermediate analogue in any ThDP-dependent enzyme.\n2 The abbreviations used are: PDHc, pyruvate dehydrogenase multienzyme complex;\nThDP, thiamin diphosphate (vitamin B1 diphosphate); LThDP, lactylthiamin diphos-\nphate (reaction intermediate); PLThDP, phosphonolactylthiamin diphosphate (reac-\ntion intermediate analogue); E1, first enzymatic component of multienzyme com-\nplexes related to and including PDHc; parental E1, protein only component of E1;\nE1-ThDP, parental E1 with ThDP cofactor; E1-PLThDP, parental E1 with PLThDP;\nH407A E1-PLThDP, H407A parental E1 variant with PLThDP.\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nThe E1 component of the PDHc complex catalyzes the rate-limiting\nstep of the overall PDHc reaction and therefore also provides an ideal\ntarget for mechanistic structural investigation. Initial crystallographic\nresults for the E1-PLThDP complex have been reported previously (15),\nand complete, detailed results at 2.1 Å resolution are now presented\nwith the analysis revealing several unique structural features that are\nabsent in the native enzyme. 1) The enzyme-bound conformation of\nPLThDP is the V form, rather than the S form determined many years\nago for the same compound in the absence of enzyme, showing how the\nenzyme can enforce this highly strained V conformation even in the\nintermediate analogue. 2) The structure identifies a role for residue\nHis-407, explaining previous biochemical data, and shows the direct\ninteraction (and implied function) of several additional active center\nresidues with the covalently bound adduct. 3) Major reorganization of\nthe active center takes place and includes the ordering of two key loops\nnot seen in any of our other PDHc E1 structures before; thus the\nPLThDP dramatically diminishes their mobility. 4) A marked deviation\nfrom co-planarity of the C2–C2 bond with the thiazolium ring is\nclearly revealed and suggests likely mechanistic consequences. 5) The\nstructure shows strong interaction of the C2-OH with the N4 atom of\nthe 4-aminopyrimidine of ThDP, as evidenced by a short contact dis-\ntance. Elsewhere, we used PLThDP in solution to show that PLThDP\nexists with the 1,4-iminoThDP tautomeric form (16, 17) of the\n4-aminopyrimidine ring, the first instance in which this putative inter-\nmediate could be stabilized. The observed short O–N4 distance is fully\nconsistent with this tautomerization. The structural reorganization and\nordering resulting from the presence of PLThDP deep in the active site\nextends to the enzyme exterior, providing a new surface likely involved\nin interactions with other enzymatic components within the PDHc\nmultienzyme complex.\nEXPERIMENTAL PROCEDURES\nCrystallization and Data Collection—The PDHc E1 from E. coli was\npurified and assayed according to published procedures (18). ThDP,\nwhen present, was removed by gel filtration using an Amersham Bio-\nsciences G-25 column. For crystallization, the protein was dialyzed\nagainst 20 mM HEPES buffer at pH 7.0, containing 5 mM dithiothreitol,\n0.2% NaN3, 0.5 mM EDTA, and 1 M leupeptin. The H407A E1 variant\nwas purified and assayed as reported previously (19). The parental and\ncorresponding variant E1 proteins were independently co-crystallized\nwith PLThDP and Mg2 by the sitting drop vapor diffusion method\nunder conditions similar to that used for our earlier E1-ThDP crystal-\nlographic studies (8). The best crystals were obtained at a reservoir\nsolution with 15–20% polyethylene glycol 2000 monomethyl ether, 10%\npropanol, 0.2% NaN3, and 60 mM Hepes buffer (pH 7.05) at 22 °C. Drops\nwere 6 –10 l consisting of equal parts of reservoir and protein solution.\nCrystal growth took about 4 – 6 weeks with crystals typically having\ndimensions of 0.1  0.20  0.30 mm.\nX-ray diffraction data for the E1-PLThDP complex were collected\nfrom a single crystal flash-cooled at 180°C on a Bruker HiSTAR detec-\ntor, using a Rigaku RU200 rotating anode generator as source. An\nOsmic mirror system was used along with a thin nickel foil to provide a\nhighly focused, intense beam of CuKradiation. The crystals are mon-\noclinic with space group P21 and cell constants a  81.6, b  142.5, c \n82.1 Å, and  102.4°, with two molecules (subunits) in an asymmetric\nunit. To 2.1 Å resolution, 466,210 total observations were reduced to\nyield 99,699 unique reflections (94% complete) with an internal R factor\n(based on intensities) of 0.07. The data were processed with the XGEN\npackage (20) and scaled with a local version of Weissman’s Fourier\nscaling program FBSCALE (21). For the H407A E1-PLThDP complex,\nsynchrotron data were collected at SERCAT (Advanced Photon Source)\nand processed with the HKL2000 package (22).\nStructure Determination and Refinement—Since the previous\nE1-ThDP crystals are isomorphous with both E1-PLThDP and H407A\nE1-PLThDP crystals (cell constants differing by less than 0.8 Å), atomic\ncoordinates from the former structure were used as the starting model\nfor both refinements. The initial model included 1602 amino acids and\nomitted the ThDP cofactors. Following rigid body refinement, the\nE1-PLThDP model was further refined by simulated annealing using the\nprogram XPLOR (23) without imposing any non-crystallographic sym-\nmetry restraints. After the initial refinement, examination of electron\ndensity difference maps clearly showed strong density for the interme-\ndiate analogue, PLThDP, and also produced strong and interpretable\nelectron density for two of the three previously unobserved regions,\n401– 413 and 541–557. After including the PLThDP and the two miss-\ning regions, the model was refined with simulated annealing, and sub-\nsequent cycles consisted of positional and B factor refinement. The\nrefinement procedure included periodic examinations of simulated\nannealing omit and difference maps, as well as the introduction of water\nmolecules. A final examination of difference electron density maps still\ndid not produce interpretable electron density for the N-terminal resi-\ndues 1–55, and they are excluded from the model. Inspection of electron\ndensity maps, model building, and structural comparisons were carried\nout by using the program O (24). For all data to 2.1 Å resolution, the R\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nfactor is 0.203 (0.213 in the last shell), and the Rfree(based on 5.3% of\ndata) is 0.254 (0.291 in the last shell). The final structure has good ste-\nreochemistry with root mean square deviations from ideality of 0.006 Å\nand 1.3° for bond lengths and angles, respectively. The final model con-\ntains two independent subunits, each with 831 amino acids out of a\npossible 886, as well as two PLThDP-Mg2 cofactor pairs. This model\nalso includes 564 water molecules, five phosphate ions, and two cis-\nprolines (Pro-463) per asymmetric unit. The model was analyzed with\nPROCHECK (25), and 89% of the residues are in the most favored region\nin the Ramachandran plot (26). The Ramachandran plot shows no res-\nidues in disallowed regions in either subunit.\nThe structure of the H407A E1-PLThDP complex was determined\nand refined in a similar manner. However, in this structure, although the\nPLThDP was clearly present in strong density, the regions 1–55, 401–\n413, and 541–557 were completely disordered. The new structure was\nrefined using the program CNS (27). The final model, which included\n550 water molecules, resulted in an R factor of 21.6% and a Rfree of 24.1%\nto 1.85 Å resolution. The slightly higher R factor for the H407A\nE1-PLThDP variant may be attributed to the fact that this structure\ncontains 60 disordered residues absent from the model (the regions\n401– 413 and 541–557 in each molecule) that were ordered in the\nE1-PLThDP structure. Ramachandran statistics are similar to those for\nthe E1-PLThDP structure. All data collection and refinement statistics\nare given in Table 1. Graphical representations of protein models were\ngenerated by using the program RIBBONS (28). Coordinates and struc-\nture factors for the E1-PLThDP and H407A E1-PLThDP structures\nhave been deposited in the Protein Data Bank with the access codes\n2G25 and 2G28, respectively.\nRESULTS\nThe main chain folds in PDHc E1-ThDP (8) and in its E1-PLThDP\ncounterpart are very similar, with both structures containing two sub-\nunits in an asymmetric unit. The two independent subunits are almost\nidentical stereochemically and are related by a 2-fold non-crystallo-\ngraphic symmetry axis. The complete dimer is shown in Fig. 2a. Each\nE1-PLThDP subunit consists of a single polypeptide chain folded into\nthree domains: the N terminus (1– 470), the middle (471–700), and the\nC terminus (701– 886). Least-squares fitting of 801 common -carbon\natoms in the E1-ThDP and E1-PLThDP structures results in a root\nmean square deviation of 0.40 Å. However, significant main-chain devi-\nations (up to 2.7 Å) occur in the middle domain helix containing resi-\ndues 525–535, which are involved in close contacts with residues pres-\nent in the active site channel. Except for a few side chains at the surface\nof the protein, the electron density over the entire molecule is generally\nvery well defined, but as in the E1-ThDP structure, there is no interpret-\nable electron density for the N-terminal residues 1–55.\nA surprising observation in the E1-PLThDP map is the presence of\nwell defined electron density for loop residues 401– 413 and 541–557\nthat was absent in the map of the E1-ThDP structure due to complete\ndisorder. These two loops form part of the active site channel situated at\nthe dimer interface. The 401– 413 loop forms part of the active site, and\nresidue His-407 in this loop forms a critical hydrogen bond to an oxygen\natom in the substrate analogue phosphonyl group. This loop stabilizes\nthe other newly ordered loop 541–557, assisted by interactions created\nfrom the large movement in the helical region 525–535. The ordering\ncreates new interface interactions between residues Lys-403–Asn-404\nfrom one subunit and residues Gln-548 –Asp-549 from the other sub-\nunit. The two newly ordered regions thus interact with each other and\npropagate outward from the active site to the enzyme surface. The\nnewly ordered regions are shown in terms of the overall structure in Fig.\n2a and in greater clarity regarding the active site and enzyme surface in\nFig. 2b. Electron density for the 401– 413 loop and its interactions with\nPLThDP are shown in Fig. 3.\nThe average B value for main and side chain atoms is 9.5 and 11.2 Å2,\nrespectively. These values are significantly lower than those observed in the\nE1-ThDP structure (16.0 and 18.6 Å2, respectively) (8), implying a more\nordered assembly. Accordingly, the presence of the reaction intermediate\nanalogue in the active site supports the formation of a tighter E1 dimer. The\nnow ordered loop regions provide a new exterior surface that can conceiv-\nably interact with other subunits of the PDHc complex and also partially\nseal the active site entrance, leading to a more hydrophobic catalytic region.\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nThe H407A E1-PLThDP structure (not shown) reveals the active site to\nbe intact and the structure generally unchanged when compared with the\nE1-ThDP (8) and E1-PLThDP structures. As in E1-ThDP and unlike in the\nE1-PLThDP structure, however, a final difference electron density map did\nnot contain interpretable electron density for loop regions 401– 413 and\n541–557; the N-terminal region 1–55 is unobserved in all of the structures.\nWhen combined with the other structures, analysis of the H407A\nE1-PLThDP structure therefore confirmed the need for the His-407-\nPLThDP interaction in inducing the disorder-to-order transformation in\ntwo loops as the loops are ordered only when both the intermediate and\nHis-407 are present. Clearly, the ordering is induced by the presence of\nPLThDP via direct hydrogen-bonding to residue His-407 and results in\nproviding previously missing structural elements to complete both the\nexterior surface and the interior surface leading to the active sites.\nFor E1-PLThDP, a difference electron density map calculated with\nIt is important to note that the E1-ThDP and E1-PLThDP complex crystals\nare isomorphous; thus the ordering cannot be attributed to new packing\ncontacts in a different crystal form and arises only from the presence of the\nadduct now covalently bound to the cofactor.\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nthe protein-only model\nindicated strong positive density for the\nPLThDP (Fig. 4). Funnel-shaped channels leading to the active sites are\nabout 24 Å deep, with walls formed by residues from both the N-termi-\nnal domain and the middle domain. In the absence of enzyme, a prior\nPLThDP crystal structure analysis (29) revealed that the thiamin por-\n\ntion of the molecule assumes the S conformation (\nt\n173.6°), which is characteristic of other C2-substituted thiamins and\nplaces C2 and its bound adduct distant from N4 of the aminopyrimi-\ndine ring. The enzyme-bound PLThDP, however, adopts the V confor-\np of 100.5 and 67°, respec-\nmation with torsion angles (30) \ntively, and results in crowding near C2. The C2-PO3Me bond is\ndirected very nearly perpendicular to the thiazolium ring. The PLThDP\nconformation is stabilized by hydrogen-bonding interactions with Tyr-\n599, His-640, His-407, and Gln-408 (through a water). The active site\ncleft is lined with residues His-106, Ser-109, Gln-140, His-142, Tyr-177,\nMet-194, Asp-230, Glu-235, Asn-260, Leu-264, Lys-392, His-407, and\nGln-408 from one subunit and residues Asp-521, Thr-525, Ile-569, Glu-\n571, Tyr-599, Phe-602, Glu-636, and His-640 from the other subunit. A\nstereo view of the dimer interface environment around the catalytic\ncenter is shown in Fig. 5. The structure reveals residues on the enzyme\nthat are responsible for stabilization of the intermediate and that are\ncapable of participating in catalysis as well. On one side of the thiazo-\nlium ring, in the vicinity of the covalent adduct on C2, is a cluster of four\nhistidine residues. Of these, His-142 is involved in binding with the\ndiphosphate group of the cofactor, whereas His-407 and His-640 inter-\nact with oxygen atoms in the adduct. The fourth, His-106, is bound to a\nwater molecule. As seen earlier, His-407 is hydrogen-bonded to one of\nthe phosphonyl oxygens, and His-640 is hydrogen-bonded to the C2-\nhydroxyl oxygen atom. Tyr-599 also forms a hydrogen bond with one of\nthe phosphonyl oxygen atoms. Other residues (Tyr-177, Gln-408, and\nGlu-636) interact indirectly with the intermediate analogue by hydro-\ngen-bonding through water molecules. Leu-264 appears to interact with\nthe analogue through Van der Waals contacts, with a terminal methyl\ncarbon making a 3.52 Å contact distance to the one of the phosphonyl\noxygen atoms of the adduct.\nIn addition to inducing disorder-to-order transformations, the pres-\nence of PLThDP in the active site also induces conformational changes\nin nearby protein residues that were previously ordered. Relative to the\nE1-ThDP structure, a conformational rearrangement of protein resi-\ndues Leu-264, Glu-522, and Tyr-599 occurs in the E1-PLThDP struc-\nture. Residue Glu-522 shows a large conformational change as its side\nchain moved more than 4 Å. The movement of residues Leu-264 and\nTyr-599 and the presence of the phosphonyl oxygen atom close to the\nthiazolium ring play an important role in the movement of the side\nchain of Glu-522, which is no longer pointing toward the active site.\nAlthough it may not play a direct role in the mechanism of catalysis at\nthis stage, Glu-522 still interacts with the cofactor through water mol-\necules. The key interactions with PLThDP are all likely to be present in\nthe corresponding LThDP intermediate formed from the natural sub-\nstrate pyruvate, as indicated in Fig. 6, since the majority of good hydro-\ngen bonds can still be formed.\nPerhaps most surprisingly, analysis of the E1-PLThDP structure\nrevealed a significant distortion in planarity for the C2–C2bond con-\nnecting the substrate analogue to the planar thiazolium ring, as seen in\nFig. 4. This out-of-plane distortion implies considerable strain and per-\nsists despite incorporation of a planarity restraint during refinement,\neven after significantly increasing the restraint weight well beyond that\nnormally used. To determine whether this effect is real or simply\ninduced by other possibly incorrect refinement restraints, refinements\nand pure energy minimizations were carried out with a variety of stand-\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nard force fields. In all cases, the out-of-plane distortion persists (to vary-\ning but significant degrees), provided that any reasonable van der Waals\nradii are used for the aminopyrimidine N4 and adduct C2-hydroxyl O\natoms, and in any event, the distortion is required to fit the electron\ndensity. The resulting distance between these atoms (2.5 Å) is reasona-\nble, but it would be unacceptably short (2.2 Å), producing a strong\nrepulsive force in the absence of the distortion, thus explaining the\nstrained conformation that is observed. Furthermore, since only three\nessentially rigid groups (coordinates obtained from the highly accurate\nsmall molecule PLThDP crystal structure (29)) are involved, the short\ncontact is dictated by only the thiazolium and aminopyrimidine relative\nring orientations and by the fact that the C2-PO3Me bond is nearly\nperpendicular to the thiazolium ring plane. The \np torsion\nangles defining the ring orientations, as well as the S–C2–C2–P tor-\nsion angle, are clearly required to fit the strong and unambiguous elec-\ntron density, and the former are also consistent with those observed in\nall reported ThDP-containing enzyme structures. Therefore the short\ncontacts cannot be explained away by coordinate errors due to data\nresolution arguments. In addition, the distortion and short contacts are\nalso present in the higher resolution H407A E1-PLThDP variant\nstructure.\nt and \nIt is important to realize that the strain is removed immediately upon\ndecarboxylation if the enamine (Fig. 1, top right object) is formed as the\nformerly tetrahedral C2carbon then adopts sp2 hybridization, increas-\ning the distance between N4 and the oxygen. It thus appears that\nremoval of this strain may be a contributing force behind decarboxyla-\ntion. If this is the case, the cofactor V conformation imposed by the\nenzyme environment is important not only in creating the initial reac-\ntive ylide (6) (Fig. 1, third object from the left) but also in inducing stress\nto facilitate decarboxylation. This is a very important finding as the\nsame forces should be operating in all ThDP-dependent decarboxylases.\nThis view is strongly supported by the fact that when the PLThDP\nmolecule is free of the enzyme (and therefore in the S conformation), it\nshows no evidence of the strain that occurs when bound to the enzyme\nin the V conformation (29). For example, in the S form, the C2-C2\nexocyclic bond is coplanar with the thiazolium ring, the valence angles\nsubtended at C2are all near the ideal tetrahedral value, and O2and\nN4 are not in contact. Therefore the strain associated with the V con-\nformation imposed by the enzyme is avoided entirely in the isolated\nPLThDP molecule. Structural differences between a planar and severely\nthe observed partially\nstrained pre-decarboxylation intermediate,\nstrained, non-planar pre-decarboxylation intermediate, and totally\nrelaxed post-decarboxylation intermediate are shown in Fig. 7.\nDISCUSSION\nThe C2-C (carboxyl) bond in -LThDP is expected to be oriented\nrelative to the thiazolium ring nearly the same (roughly orthogonal) as\nthe C2-P bond seen in -PLThDP, and this orientation had been pre-\ndicted for both compounds (29). Given the structural and electronic\nsimilarities, active site residues adjacent to C2, OH(2), CH3(2), and\nsome of the phosphonyl and carboxylate oxygens are then also expected\nto have very nearly identical locations in both the catalytically active and\nthe analogue intermediates, and the short N4-O2 distance and out-\nof-plane deformation of the C2–C2bond observed in the E1-PLThDP\ncomplex should occur in the E1--LThDP complex as well. For exam-\nple, from Fig. 6, it is clear that the carboxyl group could form the same\nfavorable hydrogen bonds to His-407 and Tyr-599 without any changes\nin the residue positions, and only very subtle changes in these positions\nwould be required to make the interactions even stronger. Accordingly,\nthe E1-PLThDP structure should be an excellent model for the natural\nintermediate involving -LThDP, and conclusions drawn from the\nE1-PLThDP structure\ntransient\nintermediate.\nto the\nThe disorder-to-order transformation induced by the presence of the\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nintermediate analogue PLThDP (and presumably -LThDP) is particu-\nlarly important as it not only completes the interior of the active site but\nalso results in structural changes leading from it out to the protein\nexterior. This was shown in Fig. 2b. With this ordering, a tunnel that can\naccommodate the lipoyl-lysine side chain bound to an E2 component in\nPDHc is completely formed, and a new surface is created at the mouth of\nthe funnel, which, based on the distance from the active site and the\nknown length of the lipoyl-lysine side chain, is likely to bind to the\ncorresponding E2 lipoyl domain. These results are consistent with a\nprevious report indicating that E1-E2 lipoyl domain binding stability is\nincreased in the presence of the substrate pyruvate (31). Given the\nstrong structural similarity between substrate and analogue, we suggest\nthat the ordering observed in the E1-PLThDP complex would also be\npresent when the intermediate from the natural substrate pyruvate is\nformed, and the increased E1-E2 stability may be explained by E2-lipoyl\ndomain interactions with the resulting newly ordered E1 surface\nresidues.\nIt appears that nearly all of the induced structural changes begin with\nthe critical His-407-PLThDP interaction within the active site and then\npropagate outward, both to another previously disordered loop and to\nthe helical region that was always ordered but now has its location\nshifted. This can be understood as follows. The first newly ordered loop\n(401– 413) is initially stabilized by the direct and water-mediated hydro-\ngen bonds formed between a phosphonyl (or carboxyl in the case of\nLThDP) oxygen atom and His-407 and Gln-408 as seen in Fig. 3. The\nsecond newly ordered loop (541–557, located at the mouth of the fun-\nnel), although not forming any direct interactions with PLThDP/\nLThDP, is then initially stabilized through the formation of hydrogen\nbonds between Asn-404 in the (now ordered) first loop and the side\nchains of Gln-548 and Asp-549. Further stabilization of this segment\noccurs via interactions near the ends of the (always ordered) helical\nsegment Thr-525–Ile-535, which shifts by 2.7 Å relative to the native\nstructure. These interactions involve formation of hydrogen bonds\nbetween Ile-537 and Lys-557 and between Arg-524 and Ala-554, with\nthe latter being water-mediated. Although only the directed and specific\nhydrogen bonds were described, numerous favorable van der Waals\ncontacts are also present and contribute to the stabilization. The end\nresult is that the reaction intermediate, both newly ordered loops, and\nthe shifted helical segment are all interlinked through hydrogen-bond-\ning and van der Waals contacts to form a more tightly packed dimer\nwith a completed active site channel and a new surface near its entrance.\nThese results are consistent with the fact that the disorder-to-order\ntransformation occurs only when both His-407 and the intermediate\nanalogue PLThDP (or presumably LThDP) are present in the enzyme.\nOur previous biochemical studies on the H407A E1 variant clearly\nindicated its importance in post-decarboxylation steps for the PDHc\nreaction (19). The mutation was shown to dramatically decrease the rate\nof reductive acetylation, which could be caused either by direct partic-\nipation in the catalytic reaction at the active site or by interfering with\nproper E1-E2 association within the PDHc multienzyme complex. The\nevidence was that the specific activity of this variant in terms of the\noverall PDHc assay is less than 0.15%, whereas there are only minor\nactivity effects on the initial decarboxylation reaction step. The current\nresults and interpretation are consistent with this study and imply that\nHis-407 does indeed play a significant role in binding E1 to the E2-lipoyl\ndomains, although direct participation, at least in the later stages of\ncatalysis, cannot be ruled out. The observed change in the active site and\nits entrance pathway in the presence of substrate/analogue likely facili-\ntates binding with the E2-lipoyl domain and insertion of the lipoyl-\nlysine side chain, so that the required (post decarboxylation) reductive\nacetylation can occur.\nWe also previously suggested that His-407 functions as a proton\ndonor to a lipoamide sulfur prior to the addition of the enamine to the\ndisulfide (19). It is now tempting to suggest that this protonation occurs\nafter carbon dioxide release because of the availability of His-407 at that\ninstant. When pyruvate adds to form the pre-decarboxylation complex,\nHis-407 would form a hydrogen bond with the pyruvate adduct and\ntrigger the disorder-order transformation in preparation to receive the\nlipoamide of the lipoyl domain. As the interaction with E2 progresses\nand the C2-C (carboxyl) bond is severed, releasing CO2, the hydrogen\nbonds from the carboxyl group to His-407 and Tyr-599 obviously will\nno longer exist, and the lipoyl domain lipoamide can move into the\nspace vacated by the CO2 molecule in preparation to seize the acyl\ngroup. An intriguing question arises about events that occur during the\ninterim period between decarboxylation and reductive-acetylation.\nDoes a reversal of the conformational transformation (order-to-disor-\nder) take place upon severance of the hydrogen bond between His-407\nand the carboxylate group, or does the ordered structure remain intact,\nat least temporarily, until reductive acetylation is completed? In the first\ninstance, the implication is that the ordered structure is unnecessary for\nthe remaining reductive acetylation function to be completed in E1. In\nthe second case, His-407 would be required to form a hydrogen bond to\nanother group (on the lipoyl domain or on E1) until reductive acetyla-\ntion is completed. An answer to these questions hopefully will be\nobtained when structural information on a complete E1-E2 complex\nbecomes available.\nIt is noteworthy that the order-disorder transformation in this 2 E1\ndimer differs considerably from those that have been observed in certain\n22 E1 heterotetramers (14, 32). In the latter cases, the addition of the\ncofactor alone to the disordered apoenzyme produced an ordered\nholoenzyme. In the 2 homodimeric E1 protein, disorder occurs in both\nthe apoenzymes and holoenzymes, and it is only the formation of the\npre-decarboxylation reaction intermediate that triggers ordering. In\naddition, the loops in 2 E1 that exhibit transformation are not the same\nas those reported to undergo transformation in the 22 cases (14, 32,\n33).\nthe\ntransformation in\nhomodimeric 2 E1s induced by substrate binding apparently is to acti-\nvate the enzyme for reductive acetylation by preparing the channel in\nthe active site to receive the E2 lipoyl domain lipoamide group. Inter-\nestingly, the phosphorylation regulatory mechanism responsible for\ndeactivating an 22 heterotetrameric E1b component operating on\nbranched chains also involves an order/disorder transformation in the\ncorresponding sequence segment (34) but functions in reverse! In that\ncase, substrate binding is not required as the E1b holoenzyme structure\nis already fully ordered, and phosphorylation deactivates the enzyme by\ninducing disorder within the segment. It is noteworthy that the phos-\nphorylation site, Ser-292-, corresponds to Gln-408 in the 2 E1 struc-\nture, which is not only in the newly ordered segment but is adjacent to\nthe critical His-407 residue! Thus both dimeric and tetrameric E1s\nappear to effectively utilize disorder/order transformations in the same\nregions with similar trigger points but in different ways and in response\nto different stimuli.\nFinally, with regard to general, enzyme-mediated thiamin catalysis,\nwe now suggest that removal of strain created by the expected, corre-\nsponding, out-of-plane distortion in the -LThDP complex may be a\ncontributing force behind the accelerated decarboxylation that is seen\nwhen the thiamin-catalyzed reaction occurs in enzymes and that the\nstrained structure may represent a transition state analogue. We have\nPre-decarboxylation Intermediate in PDHc Induces Ordering\nshown that the out-of-plane distortion is caused primarily by a short\nand repulsive N4-O2distance, which is the result of the V conforma-\ntion imposed on the ThDP cofactor by the enzyme. This is apparent\nsince in the absence of enzyme, the PLThDP structure has a C2–C2\nbond precisely in the plane of the thiazolium ring but has no short\ncontacts because it is not in the V conformation (29). Once forced in the\nV conformation by the enzyme, however, if left in the planar state, an\nextremely short and excessively repulsive contact of 2.2 Å would result\n(Fig. 7a). To reduce the highly undesirable effects of this short contact,\nthe non-planar distortion occurs, increasing the “bad” contact distance\nto a more tolerable 2.5 Å as is found in the E1-PLThDP structure and\nshown in Fig. 7b, although this still represents a strained molecule.\nComplete removal of the strain would occur only upon decarboxylation\nas then, the totally planar enamine form (Fig. 1, top right object) is cre-\nated in which the formerly bad contact distance is increased to a favor-\nable 2.8 Å as seen in Fig. 7c. The distorted, observed structure reported\nhere and shown in Fig. 7b is clearly shifted toward the product enamine\nstructure, both in terms of the N4-O2 distance and in terms of the\nmethyl carbon and O2locations, which are now close to being in the\nplane of the thiazolium ring. It could thus be described as a transition\nstate, in which the energy trapped in the C2out-of-plane distortion is\nutilized to help break the C-CO2 bond to form the enamine, in a manner\nrequiring minimal atomic motion for the remaining atoms to attain\ntheir positions in the product state. Note that if the alternative initial\ndecarboxylated form, the tetrahedral C2carbanion (Fig. 1, top, second\nobject from the right), were created, the same strained out-of-plane dis-\ntortion observed in the original pre-decarboxylation complex would\npersist, suggesting that the planar enamine form is indeed preferred.\nThis is consistent with the fact that in the published post-decarboxyla-\ntion crystal structures (11, 12, 14), the intermediates that were reported\nto best fit the electron density corresponded to planar enamines rather\nthan tetrahedral C2carbanions.",
         "Experimental observation of thiamin diphosphate-bound intermediates on enzymes and mechanistic information derived from these observations.;C2-α-Lactylthiamin Diphosphate Is an Intermediate on the Pathway of Thiamin Diphosphate-dependent Pyruvate Decarboxylation;Molecular mechanism for regulation of the human mitochondrial branched-chain alpha-ketoacid dehydrogenase complex by phosphorylation.;A Molecular Switch and Proton Wire Synchronize the Active Sites in Thiamine Enzymes;Cross-talk between Thiamin Diphosphate Binding and Phosphorylation Loop Conformation in Human Branched-chain α-Keto Acid Decarboxylase/Dehydrogenase*;Tetrahedral intermediates in thiamin diphosphate-dependent decarboxylations exist as a 1',4'-imino tautomeric form of the coenzyme, unlike the michaelis complex or the free coenzyme.;Ligand-induced conformational changes and a reaction intermediate in branched-chain 2-oxo acid dehydrogenase (E1) from Thermus thermophilus HB8, as revealed by X-ray crystallography.;Dual catalytic apparatus of the thiamin diphosphate coenzyme: acid-base via the 1',4'-iminopyrimidine tautomer along with its electrophilic role.;Current mechanistic understanding of thiamin diphosphate-dependent enzymatic reactions.;Histidine 407, a phantom residue in the E1 subunit of the Escherichia coli pyruvate dehydrogenase complex, activates reductive acetylation of lipoamide on the E2 subunit. An explanation for conservation of active sites between the E1 subunit and transketolase.;Structure of the pyruvate dehydrogenase multienzyme complex E1 component from Escherichia coli at 1.85 A resolution.;Snapshot of a key intermediate in enzymatic thiamin catalysis: Crystal structure of the α-carbanion of (α,β-dihydroxyethyl)-thiamin diphosphate in the active site of transketolase from Saccharomyces cerevisiae;Inhibition of the Escherichia coli Pyruvate Dehydrogenase Complex E1 Subunit and Its Tyrosine 177 Variants by Thiamin 2-Thiazolone and Thiamin 2-Thiothiazolone Diphosphates;Recognition of the lipoyl domain is the ultimate determinant of substrate channelling in the pyruvate dehydrogenase multienzyme complex.;Crystal structure of 2-oxoisovalerate and dehydrogenase and the architecture of 2-oxo acid dehydrogenase multienzyme complexes;Crystal structures of the key anaerobic enzyme pyruvate:ferredoxin oxidoreductase, free and in complex with pyruvate;Crystallography & NMR system: A new software suite for macromolecular structure determination.;Crystal structure of the thiamin diphosphate-dependent enzyme pyruvate decarboxylase from the yeast Saccharomyces cerevisiae at 2.3 A resolution.;A thiamin diphosphate binding fold revealed by comparison of the crystal structures of transketolase, pyruvate oxidase and pyruvate decarboxylase.;Catalytic centers in the thiamin diphosphate dependent enzyme pyruvate decarboxylase at 2.4-A resolution.;PROCHECK: a program to check the stereochemical quality of protein structures;Bioassay for carcinogenicity of rotenone in female Wistar rats.;RIBBONS 2.0;Improved methods for building protein models in electron density maps and the location of errors in these models.;Slow-cooling protocols for crystallographic refinement by simulated annealing.;Thiamin diphosphate: a mechanistic update on enzymic and nonenzymic catalysis of decarboxylation;Synthesis and crystal structure of an analog of 2-(.alpha.-lactyl)thiamin, racemic methyl 2-hydroxy-2-(2-thiamin)ethylphosphonate chloride trihydrate. A conformation for a least-motion, maximum-overlap mechanism for thiamin catalysis;Stereochemistry of intermediates in thiamine catalysis. 2. Crystal structure of DL-2-(alpha-hydroxybenzyl)thiamine chloride hydrochloride trihydrate.;NUMBER 22  JOURNAL OF BIOLOGICAL CHEMISTRY 15303  by gest  on July 14;Structure (Camb.) 12, 2185–2196 Pre-decarboxylation Intermediate in PDHc Induces;Thiamine: Catalytic Mechanisms and Role in Normal and Disease States;[20] Processing of X-ray diffraction data collected in oscillation mode.;Conformation of polypeptides and proteins.;Pyruvate Dehydrogenase E1 Subunit Induces Large Scale Disorder-to-Order A Thiamin-bound, Pre-decarboxylation Reaction Intermediate Analogue in the;Access the most updated version of this article at doi",
         "A Thiamin-bound, Pre-decarboxylation Reaction Intermediate Analogue in the Pyruvate Dehydrogenase E1 Subunit Induces Large Scale Disorder-to-Order Transformations in the Enzyme and Reveals Novel Structural Features in the Covalently Bound Adduct*",
         "None"
        ],
        [
         "47",
         "0020f876bc61a0a80a183829e838addcef4a7b8a",
         "The impact of plant-based factors on the population dynamics of mammalian herbivores has been the subject of much debate in ecology, but the role of antiherbivore defences in grasses has received relatively little attention. Silica has been proposed as the primary defence in grasses and is thought to lead to increased abrasiveness of foliage so deterring feeding, as well as reducing foliage digestibility and herbivore performance. However, at present there is little direct experimental evidence to support these ideas. In this study, we tested the effects of manipulating silica levels on the abrasiveness of grasses and on the feeding preference and growth performance of field voles, specialist grass-feeding herbivores. Elevated silica levels did increase the abrasiveness of grasses and deterred feeding by voles. We also demonstrated, for the first time, that silica reduced the growth rates of both juvenile and mature female voles by reducing the nitrogen they could absorb from the foliage. Furthermore, we found that vole feeding leads to increased levels of silica in leaves, suggesting a dynamic feedback between grasses and their herbivores. We propose that silica induction due to vole grazing reduces vole performance and hence could contribute to cyclic dynamics in vole populations.",
         "F. Massey,S. Hartley",
         "F. Massey,S. Hartley",
         "https://europepmc.org/articles/pmc1636089?pdf=render",
         "The impact of plant-based factors on the population dynamics of mammalian herbivores has been the\nsubject of much debate in ecology, but the role of antiherbivore defences in grasses has received relatively\nlittle attention. Silica has been proposed as the primary defence in grasses and is thought to lead to\nincreased abrasiveness of foliage so deterring feeding, as well as reducing foliage digestibility and herbivore\nperformance. However, at present there is little direct experimental evidence to support these ideas. In this\nstudy, we tested the effects of manipulating silica levels on the abrasiveness of grasses and on the feeding\npreference and growth performance of ﬁeld voles, specialist grass-feeding herbivores. Elevated silica levels\ndid increase the abrasiveness of grasses and deterred feeding by voles. We also demonstrated, for the ﬁrst\ntime, that silica reduced the growth rates of both juvenile and mature female voles by reducing the nitrogen\nthey could absorb from the foliage. Furthermore, we found that vole feeding leads to increased levels of\nsilica in leaves, suggesting a dynamic feedback between grasses and their herbivores. We propose that silica\ninduction due to vole grazing reduces vole performance and hence could contribute to cyclic dynamics in\nvole populations.",
         "",
         "1. INTRODUCTION\nGrasses dominate many terrestrial ecosystems and are of\ngreat ecological and economic importance, as are the grazing\nmammals associated with them. Although coevolution\nbetween grasses and grazers has received a great deal of\nattention (Stebbins 1981; McNaughton 1984; Jernvall &\nFortelius 2002; Prasad et al. 2005), the ecological\nmechanisms by which grasses defend themselves against\nherbivores remain poorly understood (O’Reagain & Mentis\n1989; Vicari & Bazely 1993). The silica content of grasses\ncan be considerably higher than in many other plants\nand has been proposed as an antiherbivore defence\nstrategy (McNaughton et al. 1985; Gali-Muhtasib et al.\n1992; Vicari & Bazely 1993). Silica is deposited as phytoliths\nin leaves. These are harder than tooth enamel (Baker et al.\n1959), so they could potentially increase the abrasiveness of\nleaves and deter feeding (Vicari & Bazely 1993). However,\ncurrently the evidence supporting the effects of silica on the\npalatability of grasses to vertebrate herbivores is limited:\nthere have been relatively few experimental studies and none\nwhich manipulate silica content and measure the sub-\nsequent changes in abrasiveness of leaves. For example,\nGali-Muhtasib et al. (1992) found prairie voles (Microtus\nochrogaster) preferred leaves of grasses with lower silica\ncontent when comparing cut leaves of two Bromus species,\nbut the effect of silica on abrasiveness was not tested. It has\nalso been proposed that phytoliths may disrupt microbial\naction in the gut of herbivores, reducing the digestibility of\ngrasses (Vicari & Bazely 1993), but almost nothing is known\nabout the effects of silica on either grass leaf digestibility or\nherbivore performance.\nMany small mammalian herbivores show strong\nselectivity in feeding preference both within and between\nplant species. In voles, these preferences have previously\nbeen shown to relate to both the nutrient content (e.g.\nnitrogen) of their food plants and the levels of physical and\nchemical antiherbivore defences (Batzli & Cole 1979;\nMarquis & Batzli 1989; Hartley et al. 1995; Hjalten et al.\n1996). Despite this evidence of the importance of food\nquality for vole behaviour, the effects of silica, the principal\ndefence in grasses, have not been fully investigated.\nFor small, fast growing and short lived mammalian\nfood quality is particularly\nherbivores such as voles,\nsigniﬁcant because of their relatively high metabolic rates\n(Demment & Van Soest 1985), and because they have a\nlimited capacity to increase rates of food consumption to\ncompensate for poor quality diets (Zynel & Wunder\n2002). In addition, the growth rates early in development\nare highly dependant upon nutrient intake. Therefore in\nvoles, food quality has the potential to dictate the time\ntaken to reach sexual maturity and the onset of breeding\neach year (Krebs & Myers 1974; Ergon et al. 2001). There\nhas been considerable interest in the factors inﬂuencing\nthe growth rates and reproductive output in voles because\nof the impact these parameters have on individual ﬁtness\nand potentially on vole population dynamics (Agrell et al.\n1995; Klemola et al. 2000a; Turchin & Batzli 2001; Ergon\net al. 2001, 2004). Changes in food quality, if they have\nsigniﬁcant effects on vole growth and reproduction, may\nplay a role in these population processes.\n(ii) ﬁeld silica levels\nFestuca ovina\nlow\nhigh\nlow\nhigh\nspecies\nsilica\nspecies!silica\n0.52G0.04\n2.44G0.17\n0.54G0.10\n4.68G0.34\n\n2.18G0.18\n2.18G0.18\n3.10G0.10\n3.12G0.10\n\n1.27G0.05\n1.25G0.11\n1.39G0.05\n1.38G0.05\nn.s.\nn.s.\nn.s.\n0.58–4.52\nHere, for the ﬁrst time, we test both the proposed\nmechanisms by which silica acts as a defence, namely its\nability to increase abrasion and its effect on foliage\ndigestibility and herbivore performance. We measure the\neffects of silica on the feeding preference and growth rates\nof ﬁeld voles (Microtus agrestis L.). We focus on the effects\nof silica on the growth of female ﬁeld voles as their\nreproductive performance, which is linked to their growth,\nis crucial to population growth and they are more likely to\nbe affected by changes in food quality than males (Ostfeld\n1985; Agrell et al. 1995). By manipulating the levels of\nsilica in two species of grass we tested whether silica is an\neffective defence against voles. We hypothesized that silica\nwill: (i) increase the abrasiveness of grass leaves reducing\nfeeding by voles; (ii) reduce the digestibility of grasses to\nvoles resulting in reduced growth rates of both juvenile\nand adult voles. We also tested whether vole feeding would\ninduce silica defences by measuring the effect of vole\ndamage on silica levels.\n2. MATERIAL AND METHODS\n(a) Study species and plant growth conditions\nShort-tailed ﬁeld voles (M. agrestis L.) are abundant in\ngrasslands across Europe (Corbet & Harris 1991) and feed\nselectively on a range of grass species. We selected two grass\nspecies (Festuca ovina L. and Lolium perenne L.) found\nabundantly in European grasslands, which vary in their\nnatural levels of foliar silica, nutrients and palatability to voles\n(table 1). Leaf samples of each grass species were collected\nfrom 10 replicate plants in 10 sites around East Sussex in\nAugust 2005, selected to represent a range of soil types and\nhabitats, to conﬁrm that our manipulated silica levels were\nwithin the range occurring in natural communities (table 1).\nGrasses were grown in inert growth media (perlite), under\nglasshouse conditions for 12–28 weeks (15–25 8C, 16/8 h\nL/D). Plants for preference trials were grown individually into\n5!5!5 cm plugs, while plants for vole growth performance\ntrials and leaf digestibility experiments were grown in sward\ntrays (20!30!5 cm). Grasses were watered every 3 days\nwith 25 ml (individual plants) or 300 ml (sward trays) of\nHoagland’s solution. To half of the plants, 150 mg lK1 of\nsoluble silica was added as NaSiO3.9H2O to the Hoagland’s\nsolution (Cid et al. 1990).",
         "",
         "",
         "broken link",
         "The impact of plant-based factors on the population dynamics of mammalian herbivores has been the\nsubject of much debate in ecology, but the role of antiherbivore defences in grasses has received relatively\nlittle attention. Silica has been proposed as the primary defence in grasses and is thought to lead to\nincreased abrasiveness of foliage so deterring feeding, as well as reducing foliage digestibility and herbivore\nperformance. However, at present there is little direct experimental evidence to support these ideas. In this\nstudy, we tested the effects of manipulating silica levels on the abrasiveness of grasses and on the feeding\npreference and growth performance of ﬁeld voles, specialist grass-feeding herbivores. Elevated silica levels\ndid increase the abrasiveness of grasses and deterred feeding by voles. We also demonstrated, for the ﬁrst\ntime, that silica reduced the growth rates of both juvenile and mature female voles by reducing the nitrogen\nthey could absorb from the foliage. Furthermore, we found that vole feeding leads to increased levels of\nsilica in leaves, suggesting a dynamic feedback between grasses and their herbivores. We propose that silica\ninduction due to vole grazing reduces vole performance and hence could contribute to cyclic dynamics in\nvole populations.\n1. INTRODUCTION\nGrasses dominate many terrestrial ecosystems and are of\ngreat ecological and economic importance, as are the grazing\nmammals associated with them. Although coevolution\nbetween grasses and grazers has received a great deal of\nattention (Stebbins 1981; McNaughton 1984; Jernvall &\nFortelius 2002; Prasad et al. 2005), the ecological\nmechanisms by which grasses defend themselves against\nherbivores remain poorly understood (O’Reagain & Mentis\n1989; Vicari & Bazely 1993). The silica content of grasses\ncan be considerably higher than in many other plants\nand has been proposed as an antiherbivore defence\nstrategy (McNaughton et al. 1985; Gali-Muhtasib et al.\n1992; Vicari & Bazely 1993). Silica is deposited as phytoliths\nin leaves. These are harder than tooth enamel (Baker et al.\n1959), so they could potentially increase the abrasiveness of\nleaves and deter feeding (Vicari & Bazely 1993). However,\ncurrently the evidence supporting the effects of silica on the\npalatability of grasses to vertebrate herbivores is limited:\nthere have been relatively few experimental studies and none\nwhich manipulate silica content and measure the sub-\nsequent changes in abrasiveness of leaves. For example,\nGali-Muhtasib et al. (1992) found prairie voles (Microtus\nochrogaster) preferred leaves of grasses with lower silica\ncontent when comparing cut leaves of two Bromus species,\nbut the effect of silica on abrasiveness was not tested. It has\nalso been proposed that phytoliths may disrupt microbial\naction in the gut of herbivores, reducing the digestibility of\ngrasses (Vicari & Bazely 1993), but almost nothing is known\nabout the effects of silica on either grass leaf digestibility or\nherbivore performance.\nMany small mammalian herbivores show strong\nselectivity in feeding preference both within and between\nplant species. In voles, these preferences have previously\nbeen shown to relate to both the nutrient content (e.g.\nnitrogen) of their food plants and the levels of physical and\nchemical antiherbivore defences (Batzli & Cole 1979;\nMarquis & Batzli 1989; Hartley et al. 1995; Hjalten et al.\n1996). Despite this evidence of the importance of food\nquality for vole behaviour, the effects of silica, the principal\ndefence in grasses, have not been fully investigated.\nFor small, fast growing and short lived mammalian\nfood quality is particularly\nherbivores such as voles,\nsigniﬁcant because of their relatively high metabolic rates\n(Demment & Van Soest 1985), and because they have a\nlimited capacity to increase rates of food consumption to\ncompensate for poor quality diets (Zynel & Wunder\n2002). In addition, the growth rates early in development\nare highly dependant upon nutrient intake. Therefore in\nvoles, food quality has the potential to dictate the time\ntaken to reach sexual maturity and the onset of breeding\neach year (Krebs & Myers 1974; Ergon et al. 2001). There\nhas been considerable interest in the factors inﬂuencing\nthe growth rates and reproductive output in voles because\nof the impact these parameters have on individual ﬁtness\nand potentially on vole population dynamics (Agrell et al.\n1995; Klemola et al. 2000a; Turchin & Batzli 2001; Ergon\net al. 2001, 2004). Changes in food quality, if they have\nsigniﬁcant effects on vole growth and reproduction, may\nplay a role in these population processes.\n(ii) ﬁeld silica levels\nFestuca ovina\nlow\nhigh\nlow\nhigh\nspecies\nsilica\nspecies!silica\n0.52G0.04\n2.44G0.17\n0.54G0.10\n4.68G0.34\n\n2.18G0.18\n2.18G0.18\n3.10G0.10\n3.12G0.10\n\n1.27G0.05\n1.25G0.11\n1.39G0.05\n1.38G0.05\nn.s.\nn.s.\nn.s.\n0.58–4.52\nHere, for the ﬁrst time, we test both the proposed\nmechanisms by which silica acts as a defence, namely its\nability to increase abrasion and its effect on foliage\ndigestibility and herbivore performance. We measure the\neffects of silica on the feeding preference and growth rates\nof ﬁeld voles (Microtus agrestis L.). We focus on the effects\nof silica on the growth of female ﬁeld voles as their\nreproductive performance, which is linked to their growth,\nis crucial to population growth and they are more likely to\nbe affected by changes in food quality than males (Ostfeld\n1985; Agrell et al. 1995). By manipulating the levels of\nsilica in two species of grass we tested whether silica is an\neffective defence against voles. We hypothesized that silica\nwill: (i) increase the abrasiveness of grass leaves reducing\nfeeding by voles; (ii) reduce the digestibility of grasses to\nvoles resulting in reduced growth rates of both juvenile\nand adult voles. We also tested whether vole feeding would\ninduce silica defences by measuring the effect of vole\ndamage on silica levels.\n2. MATERIAL AND METHODS\n(a) Study species and plant growth conditions\nShort-tailed ﬁeld voles (M. agrestis L.) are abundant in\ngrasslands across Europe (Corbet & Harris 1991) and feed\nselectively on a range of grass species. We selected two grass\nspecies (Festuca ovina L. and Lolium perenne L.) found\nabundantly in European grasslands, which vary in their\nnatural levels of foliar silica, nutrients and palatability to voles\n(table 1). Leaf samples of each grass species were collected\nfrom 10 replicate plants in 10 sites around East Sussex in\nAugust 2005, selected to represent a range of soil types and\nhabitats, to conﬁrm that our manipulated silica levels were\nwithin the range occurring in natural communities (table 1).\nGrasses were grown in inert growth media (perlite), under\nglasshouse conditions for 12–28 weeks (15–25 8C, 16/8 h\nL/D). Plants for preference trials were grown individually into\n5!5!5 cm plugs, while plants for vole growth performance\ntrials and leaf digestibility experiments were grown in sward\ntrays (20!30!5 cm). Grasses were watered every 3 days\nwith 25 ml (individual plants) or 300 ml (sward trays) of\nHoagland’s solution. To half of the plants, 150 mg lK1 of\nsoluble silica was added as NaSiO3.9H2O to the Hoagland’s\nsolution (Cid et al. 1990).",
         "Dinosaur Coprolites and the Early Evolution of Grasses and Grazers;Fading out of vole and predator cycles?;Long- and short-term induction of defences in seedlings of Shorea leprosula (Dipterocarpaceae): support for the carbon:nutrient balance hypothesis;VOLE‐DRIVEN SUCCESSION IN EXPERIMENTAL WET‐PRAIRIE RESTORATIONS;Optimal Body Size and Energy Expenditure during Winter: Why Are Voles Smaller in Declining Populations?;Does quality of winter food affect spring condition and breeding in female bank voles (Clethrionomys glareolus)?1;Seasonality, density dependence, and population cycles in Hokkaido voles;AUTOCORRELATED EXOGENOUS FACTORS AND THE DETECTION OF DELAYED DENSITY DEPENDENCE;Common mammals drive the evolutionary increase of hypsodonty in the Neogene;Limits to food intake by the Prairie Vole: effects of time for digestion;Symbiosis and the Regulation of Communities1;Life-history traits of voles in a fluctuating population respond to the immediate environment;AVAILABILITY OF FOOD AND THE POPULATION DYNAMICS OF ARVICOLINE RODENTS;Do delayed effects of overgrazing explain population cycles in voles;Experimental tests of predation and food hypotheses for population cycles of voles;Effect of grazing history, experimental defoliation, and genotype on patterns of silicification in Agrostis tenuis Sibth.;EFFECTS OF BIRDS AND RODENTS ON SYNTHETIC TALLGRASS COMMUNITIES;Food selection by two vole species in relation to plant growth strategies and plant chemistry;Handbook of British Mammals;Delayed density-dependence in a small-rodent population;The effect of fertiliser and shading on plant chemical composition and palatability to Orkney voles, Microtus arvalis orcadensis;Grazing by Food-Limited Microtine Rodents on a Productive Experimental Plant Community: Does the \"Green Desert\" Exist?;Population oscillations of boreal rodents: regulation by mustelid predators leads to chaos;Do grasses fight back? The case for antiherbivore defences.;The effect of silica in grasses on the feeding behavior of the prairie vole, Microtus ochrogaster;Silicon uptake and distribution in agropyron smithii as related to grazing history and defoliation.;Influence of Chemical Factors on Palatability of Forage to Voles;Leaf silicification in grasses — A review;ON THE MEANS WHEREBY MAMMALS ACHIEVE INCREASED FUNCTIONAL DURABILITY OF THEIR DENTITIONS, WITH SPECIAL REFERENCE TO LIMITING FACTORS;An Interpretation of Rodent Dynamics as Due to Trophic Interactions;How much do weasels shape microtine cycles in the northern Fennoscandian taiga;Silicon Concentration of Grasses Growing in Sites With Different Grazing Histories;Limiting Resources and Territoriality in Microtine Rodents;A Nutritional Explanation for Body-Size Patterns of Ruminant and Nonruminant Herbivores;Silica as a Defense against Herbivory and a Growth Promotor in African Grasses;Grazing Lawns: Animals in Herds, Plant Form, and Coevolution;Grass leaf silicification: Natural selection for an inducible defense against herbivores.;Nutritional Ecology of Microtine Rodents: Digestibility of Forage;NUTRITION AND POPULATION DYNAMICS OF THE PRAIRIE VOLE, MICROTUS OCHROGASTER, IN CENTRAL ILLINOIS;Chemical Analysis of Ecological Materials.;Cause of Wear in Sheeps' Teeth;Experimental demonstration of the antiherbivore effects of silica in grasses : impacts on foliage digestibility and vole growth rates;Mechanical testing of the abrasiveness of grass leaves. Third plant biomechanics conference proceedings;Interactions between woody plants and browsing mammals mediated by secondary metabolites;COEVOLUTION OF GRASSES AND HERBIVORES;Availability of Food;Population Cycles in Small Mammals;J. Mammal",
         "Experimental demonstration of the antiherbivore effects of silica in grasses: impacts on foliage digestibility and vole growth rates",
         "None"
        ],
        [
         "48",
         "00221c6a820c45208f266ee3276397aa16b5c5c4",
         "We present a method to infer the complete connectivity of a network from its stable response dynamics. As a paradigmatic example, we consider networks of coupled phase oscillators and explicitly study their long-term stationary response to temporally constant driving. For a given driving condition, measuring the phase differences and the collective frequency reveals information about how the units are interconnected. Sufficiently many repetitions for different driving conditions yield the entire network connectivity (the absence or presence of each connection) from measuring the response dynamics only. For sparsely connected networks, we obtain good predictions of the actual connectivity even for formally underdetermined problems.",
         "M. Timme",
         "M. Timme",
         "http://link.aps.org/pdf/10.1103/PhysRevLett.98.224101",
         "We present a method to infer the complete connectivity of a network from its stable response dynamics.\nAs a paradigmatic example, we consider networks of coupled phase oscillators and explicitly study their\nlong-term stationary response to temporally constant driving. For a given driving condition, measuring the\nphase differences and the collective frequency reveals information about how the units are interconnected.\nSufﬁciently many repetitions for different driving conditions yield the entire network connectivity (the\nabsence or presence of each connection) from measuring the response dynamics only. For sparsely\nconnected networks, we obtain good predictions of the actual connectivity even for formally under-\ndetermined problems.",
         "",
         "A central issue in current multidisciplinary research is to\nunderstand the relations between network structure and\nnetwork dynamics [1,2]. Given an idealized model of the\ndynamics of the individual units and of their interactions,\nwhat can we tell about features of the collective dynamics\ndepending on the network connectivity, say, a regular\nlattice, a random network, or some more intricately con-\nnected network [1–3]? For many biological systems, such\nas networks of neurons, interacting proteins or genes, and\necological foodwebs [4 –11], however, important aspects\nof the network structure are largely unknown such that\ninverse methods may prove useful. For\ninstance,\nRefs. [10,11] show recent advances relating functional\nand structural properties in neuronal networks using (pas-\nsive) observation of the dynamics. Originally, the inability\nto infer details of the connectivity of neural systems has led\nto the common notion of ‘‘effective connectivity’’ [12].\nMore generally, it is still not well understood how we can\ninfer details of the connectivity of a network, e.g., the\npresence and absence of individual links, from controlled\nmeasurements of its dynamics.\nHere we follow this reverse perspective for networks\nexhibiting stable dynamics using a complementary ap-\nproach: Applying external driving signals to actively mod-\nify the dynamics, the measured response in comparison to\nthe original, undriven dynamics depends on both the driv-\ning and the network connectivity. Thus, evaluating the\ncollective response dynamics in dependence on the driving\nsignals, we obtain information about the network connec-\ntivity. We assume that the network is strongly connected\n[13] (i.e., every unit can directly or indirectly communicate\nwith every other) such that each unit may respond to\ndriving signals applied to any other units in the network.\nApart from this, the network connectivity is completely\narbitrary.\nAs an explicit example, we here consider networks of\ncoupled phase oscillators, a paradigmatic model that has\nbeen successfully used to understand collective dynamical\nphenomena in engineering, physics, chemistry, biology,\nand medicine [14 –20]. Here the stationary response to\nany phase-locked dynamics reveals information about the\nspeciﬁc connectivity. We explicitly show that and how,\ngiven a network of N units, each experiment (consisting\nof driving and measuring) provides N restrictions onto the\nis deﬁned by N2 coupling\nnetwork connectivity that\nstrengths. Exploiting this, we reveal the entire network\nconnectivity by repeatedly performing measurements of\nthe dynamics only, under N independent driving condi-\ntions. Furthermore, assuming that real networks are sub-\nstantially more sparsely connected than all to all, we\nextend the method to reliably predict the entire connectiv-\nity of the network even by a number of experiments that is\nmuch smaller than the number of units in the network.\nThe oscillators are coupled on a directed network of\nunknown connectivity with their dynamics satisfying\n_ i  !i  X\nwhere it is the phase of oscillator i at time t, !i is its\nnatural frequency, Jij is the coupling strength from oscil-\nlator j to i (Jij  0 if this connection is absent), and fij are\nsmooth coupling functions. We drive one or more oscilla-\ntors i in the network by temporally constant input signals\nIi;m, i 2 f1; . . . ; Ng, that can be positive, negative, or zero\n(meaning that oscillator i is not driven). Here the index m\nspeciﬁes the driving condition. If the network is not driven,\nwe have m  0, and all input signals are identically zero:\nIi;0  0. We deﬁne the in-degree ki : jfJij  0jj 2\nf1; . . . ; Nggj as the number of incoming links to oscillator i.\nGiven driving condition m, consider the stationary dy-\nnamics on a phase-locked attractor that satisﬁes i;mt \nj;mt  ij;m, independent of time. The collective fre-\nquency m satisﬁes\n© 2007 The American Physical Society\nP H Y S I C A L R E V I E W L E T T E R S\nPRL 98, 224101 (2007)\nfor all i 2 f1; . . . ; Ng. The undriven (Ii;0  0) network has\ncollective frequency 0.\nThe driving inputs Ii;m effectively change the units’\nnatural frequencies. Keeping the signal strengths sufﬁ-\nciently small, we structurally perturb the phase-locked\nstate such that it stays phase-locked and close to the\noriginal, i.e.,\njj;m  i;m  j;0  i;0j  1;\ncf. Fig. 1. Such a driving signal results in a phase pattern of\nthe entire network that depends on the details of the con-\nnectivity of that network as well as on the driving signal\nitself [21–26].\nNow take the differences between the phase-locked\nconditions for the driven and the undriven system:\nJijfijj;m  i;m  fijj;0  i;0;\nwhere Di;m : m  0  Ii;m. For sufﬁciently small\nstructural perturbations, relation (3) holds, and we approxi-\nmate fijx  f0\nijji;0x  Ox2 and abbreviate the phase\nshifts j;m : j;m  j;0 yielding\nwhere ^J is the N  N Laplacian matrix given by\nijji;0\nJikf0\nfor i  j;\nfor i  j:\n(6)\nGiven one driving condition m, we measure N  1\nindependent phase shifts i;m and one collective frequency\nm to obtain N linearized equations (5) that restrict the\nN2-dimensional space of all possible network connectiv-\nities  ^Jiji;j2f1;...;Ng. This is the maximum number of re-\nstrictions one can deduce from one experiment. From\nrepeated measurements under linearly independent driving\nconditions, we obtain more and more information about the\nconnectivity: After performing M experiments [27], the\nspace of networks is restricted by MN equations\nwhere   i;mi2f1;...;Ng;m2f1;...;Mg is the N  M matrix of\ncolumn vectors of phase differences for each experiment\nm, and, analogously, D  Di;mi;m is the N  M matrix of\nthe effective frequency offsets. Thus, we are left with an\nN  MN-dimensional family of possible networks that\nare consistent with the M measured data sets. In particular,\nthis implies that, after M  N experiments, the network\nconnectivity is speciﬁed completely as given by ^J \nD1. During reconstruction, we substitute the diagonal\nentries of the reconstructed matrix by its off-diagonal sums\naccording to (6). In all numerical illustrations, we take the\nstandard Kuramoto model fij  sin for all i; j and approxi-\nmate the prefactors in (6) by f0\nijij;0  1. A reliable\nreconstruction of the connectivity from measurements of\nthe collective response dynamics is illustrated in Fig. 2.\nThis direct method is capable of revealing not only\nwhich links are present and which are absent but also gives\na good quantitative estimate of the actual effective link\nstrengths ^Jij. It has, however, also some drawbacks. The\nproblem of solving (7) can be ill-conditioned in the sense\nthat the ratio of the largest and the smallest singular value\nof T is large,\nleading to low-quality reconstruction.\nMoreover, the direct method might become impractical\nwhen studying real-world networks, which often consist\nof a large number N of units and thus would require a large\nnumber M  N of (possibly costly) experiments.\nCan we obtain the connectivity more efﬁciently, even\nwith M < N experiments? In many networks, such as net-\nworks of neurons in the brain, a substantial number of\npotential links are not present: Each node i typically has\na number ki  N of links. Here we exploit this fact and\nlook for that connectivity matrix J that has the least num-\nber of links (maximum number of Jij  0) but is still\nconsistent with all M measured data sets.\n224101-2\nPRL 98, 224101 (2007)\nP H Y S I C A L R E V I E W L E T T E R S\n Jderived\nij\nTo achieve this goal, we use the constraints (7) to pa-\nrametrize the family of admissible matrices by N  MN\nreal parameters Pij, i 2 f1; . . . ; Ng, j 2 fM  1; . . . ; Ng, in\na standard way using a singular value decomposition of\nT  USVT, where the M  N matrix S contains the\nsingular values on the diagonal Sij  iji  0. We re-\nwrite the set of all coupling matrices ^J  DU ~SVT  PV,\nsetting Pij  0 for all j  M and ~Sij  ij=i if i >\n104 and ~Sii  0 if i  104. Finally, we minimize the\n1-norms of the row vectors of ^J (input coupling strengths)\nk ^Jik1 : X\nwith respect to the parameters P, separately for all oscil-\nlators i. By this method, we ﬁnd the network with a\nminimal number of incoming links (maximal number of\nzero entries) [28]; thus, we ﬁnd a particularly sparse net-\nwork satisfying the measurement data. Reasonably good\nreconstructions can already be obtained with the number of\nexperiments M being substantially smaller than N, as\nillustrated in Fig. 3.\nHow reliable is such a reconstruction? This depends on\nthe details of the network connectivity and the realization\nof driving. We did a case study for random networks of dif-\nferent numbers N of identical oscillators each receiving\nfrom ki  k < N randomly chosen\ninput connections\nothers. Using Jmax  maxi0;j0fjJderived\njg, deﬁne\nthe elementwise relative difference as\nsuch that Jij 2 0; 1. After M experiments, the quality of\nreconstruction is deﬁned as the fraction\nQM :\nN2\nH1    Jij 2 0; 1\nof coupling strengths which are considered correct. Here\n  1 is the required accuracy of the coupling strengths\nand H the Heaviside step function; Hx  1 for x  0 and\nHx  0 for x < 0. Typically, the quality of reconstruc-\ntion increases with M (but depends also on the realizations\nof the experiments), becoming close to 1 already for M\nsubstantially smaller than N; see Fig. 4(a). We furthermore\nevaluated the minimum number of experiments\nMq; : minfMjQM  qg\nrequired for accurate reconstruction on quality level q.\nFigure 4(b) shows M0:98;0:95, the minimum number of ex-\nperiments required for having at least q  98% of the links\naccurate in strength on an accuracy level of at least  \n0:95, as a function of N. The numerics suggests that Mq;\ngenerally scales sublinearly (presumably logarithmically)\nwith network size N for reasonable 0 < 1    1 and\n0 < 1  q  1. In particular, it implies that the connec-\ntivity of a network can be revealed reliably even if M is\nmuch smaller than the network size N.\nIn the present study we took advantage of the fact that, in\nresponse to driving (cf. also [21–26]), networks with stable\ndynamics respond in a way characteristic of their connec-\ntivity (cf. also Fig. 1 and Refs. [25,26]). Thus, information\nabout the connectivity can be revealed from measuring the\nresponse dynamics. To achieve this, we exploited all avail-\nable information of the network dynamics (the N  1\nindependent phase differences and the collective fre-\nquency) rather than only statistical information such as\nin a recent study,\none order parameter. Interestingly,\n224101-3\nPRL 98, 224101 (2007)\nP H Y S I C A L R E V I E W L E T T E R S\nArenas et al. [20] also used more detailed information of\nthe dynamics and successfully inferred the hierarchical\nstructure of a network. The method presented here not\nonly identiﬁes where links are present and where they are\nabsent but also gives a good estimate for the strength of\neach connection. For networks with a substantial number\nof potential links absent, we furthermore showed how to\npredict the connectivity in a reliable way even by a number\nof experiments that is much smaller than the network size.\nThe relatively simple yet efﬁcient method presented here\nthus qualiﬁes as potentially practically useful also for real\nsystems of moderate or larger size where the number of\nexperiments might be desired as small as possible.\nThe method presented here is based on the standard form\nof Eq. (5) and thus is applicable not only to coupled phase\noscillators but also to all other systems that exhibit a line-\narization about some stable state analogous to Eq. (5). An\nimportant question for future research is how to extend this\nmethod further to networks of dynamical elements that are\ndescribed by more than one variable, are coupled via\nexchanging pulses, or exhibit delayed interactions. Each\nof these systems requires a separate and thorough study of\nthe technical details involved in the reconstruction analy-\nsis. The basic ideas underlying the reconstruction method\npresented here, however, are likely to transfer to a variety\nof systems because the main requirements —(i) stable dy-\nnamics and, as follows from the above analysis, (ii) ﬁrst\norder additivity of the couplings—are features ubiquitous\nin natural and artiﬁcial systems. For spiking neural net-\nworks, for instance, in which units are pulse-coupled and\ndelays can be signiﬁcant, preliminary studies show that,\nonce the delay times (that render the system formally\ninﬁnite dimensional) can be estimated independently, a\nreconstruction is possible along the lines presented here.\nThe multidisciplinary research community studying net-\nworks has recently seen signiﬁcant progress towards un-\nderstanding the implications of structural features for net-\nwork dynamics and function, in particular, in biological\nnetworks. Interesting examples [4 –11] include: (i) Net-\nwork motifs, small subnetworks that occur signiﬁcantly\nmore often than in randomized networks, have been iden-\ntiﬁed in a variety of complex systems and might be de-\nsigned for functionality; (ii) a small part of a genetic\npathway was successfully identiﬁed based on expression\nproﬁling; (iii) neural wiring in the brain appears to follow\noptimization rules. Together with such ﬁndings, our active\nmethod —exploiting dynamical response properties of net-\nworks—suggests a promising future direction of research:\nDeveloping new and reﬁning existing inverse methods\nshould, on the one hand, help to better understand\nstructure-dynamics relations from measuring (possibly\ncomplicated) dynamics; on the other hand, they could\nalso help to clarify structural questions in the ﬁrst place,\ne.g., by identifying functionally meaningful parts of a\nnetwork.\nI thank C. Kirst and S. Strogatz for helpful discussions\nand F. van Bussel for valuable help with numerical simu-\nlations and optimization. I acknowledge ﬁnancial support\nby the BMBF Germany via the BCCN Go¨ttingen under\nGrant No. 01GQ0430 and by the Max Planck Society.\n[1] S. H. Strogatz, Nature (London) 410, 268 (2001).\n[2] I. Stewart, Nature (London) 427, 601 (2004).\n[3] Handbook of Graphs and Networks, edited by S. Born-\nholdt and H. G. Schuster (Wiley-VCH, Weinheim, 2002).\n[4] R. Milo et al., Science 298, 824 (2002).\n[5] D. B. Chklovskii, T. Schikorski, and C. F. Stevens, Neuron\n[6] T. S. Gardner, D. di Bernardo, D. Lorenz, and J. J. Collins,\nScience 301, 102 (2003).\n[7] B. Drossel and A. McKane, in Ref. [3].\n[8] E. L. Berlow et al., J. Anim. Ecol. 73, 585 (2004).\n[9] R.-M. Memmesheimer\nPhysica\n(Amsterdam) 224D, 182 (2006); Phys. Rev. Lett. 97,\n188101 (2006).\n[10] O. Sporns and R. Koetter, PLoS Biology 2, e369\n[11] V. A. Makarov, F. Panetsos, and O. de Feo, J. Neurosci.\nMethods. 144, 265 (2005).\n[12] A. Aertsen et al., J. Neurophysiol. 61, 900 (1989).\n[13] A directed network is strongly connected if there is a\ndirected path between any ordered pair of nodes.\n[14] A. Pikovsky, M. Rosenblum, and J. Kurths, Synchro-\nin Nonlinear Sciences\nnization: A Universal Concept\n(Cambridge University Press, Cambridge, MA, 2001).\n[15] Y. Kuramoto, Chemical Oscillations, Waves,\nand\nTurbulence (Springer, Berlin, 1984).\n[16] S. H. Strogatz, Physica (Amsterdam) 143D, 1 (2000).\n[17] J. A. Acebro´n, L. L. Bonilla, C. J. Pe´rez Vicente, F. Ritort,\nand R. Spigler, Rev. Mod. Phys. 77, 137 (2005).\n[18] Yu. Maistrenko, O. Popovych, O. Burylko, and P. A. Tass,\nPhys. Rev. Lett. 93, 084102 (2004).\n[19] P. A. Tass, Phase Resetting in Medicine and Biology\n[20] A. Arenas, A. Dı´az-Guilera, and C. J. Pe´rez-Vicente, Phys.\nRev. Lett. 96, 114102 (2006).\n[21] H. Kori and A. S. Mikhailov, Phys. Rev. Lett. 93, 254101\n[22] M. Denker, M. Timme, M. Diesmann, F. Wolf, and\nT. Geisel, Phys. Rev. Lett. 92, 074103 (2004).\n[23] F. Radicchi and H. Meyer-Ortmanns, Phys. Rev. E 73,\n[24] D. H. Zanette, Europhys. Lett. 68, 356 (2004); Eur. Phys.\nJ. B 43, 97 (2005).\n[25] M. Blank and L. Bunimovich, Nonlinearity 19, 329\n[26] M. Timme, Europhys. Lett. 76, 367 (2006).\n[27] We discard outliers: They may originate from perturba-\ntions that create (i) no structurally stable response or\n(ii) responses that are close to linearly dependent on\nothers.\n[28] S. Boyd and L. Vandenberghe, Convex Optimization\nPress, Cambridge, United\n(Cambridge University\nKingdom, 2004).\n[29] Here and in the following, the driving signals are inde-\npendent random Ii;m 2 0; I, where I  0:2.\n224101-4",
         "",
         "",
         "downloading",
         "We present a method to infer the complete connectivity of a network from its stable response dynamics.\nAs a paradigmatic example, we consider networks of coupled phase oscillators and explicitly study their\nlong-term stationary response to temporally constant driving. For a given driving condition, measuring the\nphase differences and the collective frequency reveals information about how the units are interconnected.\nSufﬁciently many repetitions for different driving conditions yield the entire network connectivity (the\nabsence or presence of each connection) from measuring the response dynamics only. For sparsely\nconnected networks, we obtain good predictions of the actual connectivity even for formally under-\ndetermined problems.\nA central issue in current multidisciplinary research is to\nunderstand the relations between network structure and\nnetwork dynamics [1,2]. Given an idealized model of the\ndynamics of the individual units and of their interactions,\nwhat can we tell about features of the collective dynamics\ndepending on the network connectivity, say, a regular\nlattice, a random network, or some more intricately con-\nnected network [1–3]? For many biological systems, such\nas networks of neurons, interacting proteins or genes, and\necological foodwebs [4 –11], however, important aspects\nof the network structure are largely unknown such that\ninverse methods may prove useful. For\ninstance,\nRefs. [10,11] show recent advances relating functional\nand structural properties in neuronal networks using (pas-\nsive) observation of the dynamics. Originally, the inability\nto infer details of the connectivity of neural systems has led\nto the common notion of ‘‘effective connectivity’’ [12].\nMore generally, it is still not well understood how we can\ninfer details of the connectivity of a network, e.g., the\npresence and absence of individual links, from controlled\nmeasurements of its dynamics.\nHere we follow this reverse perspective for networks\nexhibiting stable dynamics using a complementary ap-\nproach: Applying external driving signals to actively mod-\nify the dynamics, the measured response in comparison to\nthe original, undriven dynamics depends on both the driv-\ning and the network connectivity. Thus, evaluating the\ncollective response dynamics in dependence on the driving\nsignals, we obtain information about the network connec-\ntivity. We assume that the network is strongly connected\n[13] (i.e., every unit can directly or indirectly communicate\nwith every other) such that each unit may respond to\ndriving signals applied to any other units in the network.\nApart from this, the network connectivity is completely\narbitrary.\nAs an explicit example, we here consider networks of\ncoupled phase oscillators, a paradigmatic model that has\nbeen successfully used to understand collective dynamical\nphenomena in engineering, physics, chemistry, biology,\nand medicine [14 –20]. Here the stationary response to\nany phase-locked dynamics reveals information about the\nspeciﬁc connectivity. We explicitly show that and how,\ngiven a network of N units, each experiment (consisting\nof driving and measuring) provides N restrictions onto the\nis deﬁned by N2 coupling\nnetwork connectivity that\nstrengths. Exploiting this, we reveal the entire network\nconnectivity by repeatedly performing measurements of\nthe dynamics only, under N independent driving condi-\ntions. Furthermore, assuming that real networks are sub-\nstantially more sparsely connected than all to all, we\nextend the method to reliably predict the entire connectiv-\nity of the network even by a number of experiments that is\nmuch smaller than the number of units in the network.\nThe oscillators are coupled on a directed network of\nunknown connectivity with their dynamics satisfying\n_ i  !i  X\nwhere it is the phase of oscillator i at time t, !i is its\nnatural frequency, Jij is the coupling strength from oscil-\nlator j to i (Jij  0 if this connection is absent), and fij are\nsmooth coupling functions. We drive one or more oscilla-\ntors i in the network by temporally constant input signals\nIi;m, i 2 f1; . . . ; Ng, that can be positive, negative, or zero\n(meaning that oscillator i is not driven). Here the index m\nspeciﬁes the driving condition. If the network is not driven,\nwe have m  0, and all input signals are identically zero:\nIi;0  0. We deﬁne the in-degree ki : jfJij  0jj 2\nf1; . . . ; Nggj as the number of incoming links to oscillator i.\nGiven driving condition m, consider the stationary dy-\nnamics on a phase-locked attractor that satisﬁes i;mt \nj;mt  ij;m, independent of time. The collective fre-\nquency m satisﬁes\n© 2007 The American Physical Society\nP H Y S I C A L R E V I E W L E T T E R S\nPRL 98, 224101 (2007)\nfor all i 2 f1; . . . ; Ng. The undriven (Ii;0  0) network has\ncollective frequency 0.\nThe driving inputs Ii;m effectively change the units’\nnatural frequencies. Keeping the signal strengths sufﬁ-\nciently small, we structurally perturb the phase-locked\nstate such that it stays phase-locked and close to the\noriginal, i.e.,\njj;m  i;m  j;0  i;0j  1;\ncf. Fig. 1. Such a driving signal results in a phase pattern of\nthe entire network that depends on the details of the con-\nnectivity of that network as well as on the driving signal\nitself [21–26].\nNow take the differences between the phase-locked\nconditions for the driven and the undriven system:\nJijfijj;m  i;m  fijj;0  i;0;\nwhere Di;m : m  0  Ii;m. For sufﬁciently small\nstructural perturbations, relation (3) holds, and we approxi-\nmate fijx  f0\nijji;0x  Ox2 and abbreviate the phase\nshifts j;m : j;m  j;0 yielding\nwhere ^J is the N  N Laplacian matrix given by\nijji;0\nJikf0\nfor i  j;\nfor i  j:\n(6)\nGiven one driving condition m, we measure N  1\nindependent phase shifts i;m and one collective frequency\nm to obtain N linearized equations (5) that restrict the\nN2-dimensional space of all possible network connectiv-\nities  ^Jiji;j2f1;...;Ng. This is the maximum number of re-\nstrictions one can deduce from one experiment. From\nrepeated measurements under linearly independent driving\nconditions, we obtain more and more information about the\nconnectivity: After performing M experiments [27], the\nspace of networks is restricted by MN equations\nwhere   i;mi2f1;...;Ng;m2f1;...;Mg is the N  M matrix of\ncolumn vectors of phase differences for each experiment\nm, and, analogously, D  Di;mi;m is the N  M matrix of\nthe effective frequency offsets. Thus, we are left with an\nN  MN-dimensional family of possible networks that\nare consistent with the M measured data sets. In particular,\nthis implies that, after M  N experiments, the network\nconnectivity is speciﬁed completely as given by ^J \nD1. During reconstruction, we substitute the diagonal\nentries of the reconstructed matrix by its off-diagonal sums\naccording to (6). In all numerical illustrations, we take the\nstandard Kuramoto model fij  sin for all i; j and approxi-\nmate the prefactors in (6) by f0\nijij;0  1. A reliable\nreconstruction of the connectivity from measurements of\nthe collective response dynamics is illustrated in Fig. 2.\nThis direct method is capable of revealing not only\nwhich links are present and which are absent but also gives\na good quantitative estimate of the actual effective link\nstrengths ^Jij. It has, however, also some drawbacks. The\nproblem of solving (7) can be ill-conditioned in the sense\nthat the ratio of the largest and the smallest singular value\nof T is large,\nleading to low-quality reconstruction.\nMoreover, the direct method might become impractical\nwhen studying real-world networks, which often consist\nof a large number N of units and thus would require a large\nnumber M  N of (possibly costly) experiments.\nCan we obtain the connectivity more efﬁciently, even\nwith M < N experiments? In many networks, such as net-\nworks of neurons in the brain, a substantial number of\npotential links are not present: Each node i typically has\na number ki  N of links. Here we exploit this fact and\nlook for that connectivity matrix J that has the least num-\nber of links (maximum number of Jij  0) but is still\nconsistent with all M measured data sets.\n224101-2\nPRL 98, 224101 (2007)\nP H Y S I C A L R E V I E W L E T T E R S\n Jderived\nij\nTo achieve this goal, we use the constraints (7) to pa-\nrametrize the family of admissible matrices by N  MN\nreal parameters Pij, i 2 f1; . . . ; Ng, j 2 fM  1; . . . ; Ng, in\na standard way using a singular value decomposition of\nT  USVT, where the M  N matrix S contains the\nsingular values on the diagonal Sij  iji  0. We re-\nwrite the set of all coupling matrices ^J  DU ~SVT  PV,\nsetting Pij  0 for all j  M and ~Sij  ij=i if i >\n104 and ~Sii  0 if i  104. Finally, we minimize the\n1-norms of the row vectors of ^J (input coupling strengths)\nk ^Jik1 : X\nwith respect to the parameters P, separately for all oscil-\nlators i. By this method, we ﬁnd the network with a\nminimal number of incoming links (maximal number of\nzero entries) [28]; thus, we ﬁnd a particularly sparse net-\nwork satisfying the measurement data. Reasonably good\nreconstructions can already be obtained with the number of\nexperiments M being substantially smaller than N, as\nillustrated in Fig. 3.\nHow reliable is such a reconstruction? This depends on\nthe details of the network connectivity and the realization\nof driving. We did a case study for random networks of dif-\nferent numbers N of identical oscillators each receiving\nfrom ki  k < N randomly chosen\ninput connections\nothers. Using Jmax  maxi0;j0fjJderived\njg, deﬁne\nthe elementwise relative difference as\nsuch that Jij 2 0; 1. After M experiments, the quality of\nreconstruction is deﬁned as the fraction\nQM :\nN2\nH1    Jij 2 0; 1\nof coupling strengths which are considered correct. Here\n  1 is the required accuracy of the coupling strengths\nand H the Heaviside step function; Hx  1 for x  0 and\nHx  0 for x < 0. Typically, the quality of reconstruc-\ntion increases with M (but depends also on the realizations\nof the experiments), becoming close to 1 already for M\nsubstantially smaller than N; see Fig. 4(a). We furthermore\nevaluated the minimum number of experiments\nMq; : minfMjQM  qg\nrequired for accurate reconstruction on quality level q.\nFigure 4(b) shows M0:98;0:95, the minimum number of ex-\nperiments required for having at least q  98% of the links\naccurate in strength on an accuracy level of at least  \n0:95, as a function of N. The numerics suggests that Mq;\ngenerally scales sublinearly (presumably logarithmically)\nwith network size N for reasonable 0 < 1    1 and\n0 < 1  q  1. In particular, it implies that the connec-\ntivity of a network can be revealed reliably even if M is\nmuch smaller than the network size N.\nIn the present study we took advantage of the fact that, in\nresponse to driving (cf. also [21–26]), networks with stable\ndynamics respond in a way characteristic of their connec-\ntivity (cf. also Fig. 1 and Refs. [25,26]). Thus, information\nabout the connectivity can be revealed from measuring the\nresponse dynamics. To achieve this, we exploited all avail-\nable information of the network dynamics (the N  1\nindependent phase differences and the collective fre-\nquency) rather than only statistical information such as\nin a recent study,\none order parameter. Interestingly,\n224101-3\nPRL 98, 224101 (2007)\nP H Y S I C A L R E V I E W L E T T E R S\nArenas et al. [20] also used more detailed information of\nthe dynamics and successfully inferred the hierarchical\nstructure of a network. The method presented here not\nonly identiﬁes where links are present and where they are\nabsent but also gives a good estimate for the strength of\neach connection. For networks with a substantial number\nof potential links absent, we furthermore showed how to\npredict the connectivity in a reliable way even by a number\nof experiments that is much smaller than the network size.\nThe relatively simple yet efﬁcient method presented here\nthus qualiﬁes as potentially practically useful also for real\nsystems of moderate or larger size where the number of\nexperiments might be desired as small as possible.\nThe method presented here is based on the standard form\nof Eq. (5) and thus is applicable not only to coupled phase\noscillators but also to all other systems that exhibit a line-\narization about some stable state analogous to Eq. (5). An\nimportant question for future research is how to extend this\nmethod further to networks of dynamical elements that are\ndescribed by more than one variable, are coupled via\nexchanging pulses, or exhibit delayed interactions. Each\nof these systems requires a separate and thorough study of\nthe technical details involved in the reconstruction analy-\nsis. The basic ideas underlying the reconstruction method\npresented here, however, are likely to transfer to a variety\nof systems because the main requirements —(i) stable dy-\nnamics and, as follows from the above analysis, (ii) ﬁrst\norder additivity of the couplings—are features ubiquitous\nin natural and artiﬁcial systems. For spiking neural net-\nworks, for instance, in which units are pulse-coupled and\ndelays can be signiﬁcant, preliminary studies show that,\nonce the delay times (that render the system formally\ninﬁnite dimensional) can be estimated independently, a\nreconstruction is possible along the lines presented here.\nThe multidisciplinary research community studying net-\nworks has recently seen signiﬁcant progress towards un-\nderstanding the implications of structural features for net-\nwork dynamics and function, in particular, in biological\nnetworks. Interesting examples [4 –11] include: (i) Net-\nwork motifs, small subnetworks that occur signiﬁcantly\nmore often than in randomized networks, have been iden-\ntiﬁed in a variety of complex systems and might be de-\nsigned for functionality; (ii) a small part of a genetic\npathway was successfully identiﬁed based on expression\nproﬁling; (iii) neural wiring in the brain appears to follow\noptimization rules. Together with such ﬁndings, our active\nmethod —exploiting dynamical response properties of net-\nworks—suggests a promising future direction of research:\nDeveloping new and reﬁning existing inverse methods\nshould, on the one hand, help to better understand\nstructure-dynamics relations from measuring (possibly\ncomplicated) dynamics; on the other hand, they could\nalso help to clarify structural questions in the ﬁrst place,\ne.g., by identifying functionally meaningful parts of a\nnetwork.\nI thank C. Kirst and S. Strogatz for helpful discussions\nand F. van Bussel for valuable help with numerical simu-\nlations and optimization. I acknowledge ﬁnancial support\nby the BMBF Germany via the BCCN Go¨ttingen under\nGrant No. 01GQ0430 and by the Max Planck Society.\n[1] S. H. Strogatz, Nature (London) 410, 268 (2001).\n[2] I. Stewart, Nature (London) 427, 601 (2004).\n[3] Handbook of Graphs and Networks, edited by S. Born-\nholdt and H. G. Schuster (Wiley-VCH, Weinheim, 2002).\n[4] R. Milo et al., Science 298, 824 (2002).\n[5] D. B. Chklovskii, T. Schikorski, and C. F. Stevens, Neuron\n[6] T. S. Gardner, D. di Bernardo, D. Lorenz, and J. J. Collins,\nScience 301, 102 (2003).\n[7] B. Drossel and A. McKane, in Ref. [3].\n[8] E. L. Berlow et al., J. Anim. Ecol. 73, 585 (2004).\n[9] R.-M. Memmesheimer\nPhysica\n(Amsterdam) 224D, 182 (2006); Phys. Rev. Lett. 97,\n188101 (2006).\n[10] O. Sporns and R. Koetter, PLoS Biology 2, e369\n[11] V. A. Makarov, F. Panetsos, and O. de Feo, J. Neurosci.\nMethods. 144, 265 (2005).\n[12] A. Aertsen et al., J. Neurophysiol. 61, 900 (1989).\n[13] A directed network is strongly connected if there is a\ndirected path between any ordered pair of nodes.\n[14] A. Pikovsky, M. Rosenblum, and J. Kurths, Synchro-\nin Nonlinear Sciences\nnization: A Universal Concept\n(Cambridge University Press, Cambridge, MA, 2001).\n[15] Y. Kuramoto, Chemical Oscillations, Waves,\nand\nTurbulence (Springer, Berlin, 1984).\n[16] S. H. Strogatz, Physica (Amsterdam) 143D, 1 (2000).\n[17] J. A. Acebro´n, L. L. Bonilla, C. J. Pe´rez Vicente, F. Ritort,\nand R. Spigler, Rev. Mod. Phys. 77, 137 (2005).\n[18] Yu. Maistrenko, O. Popovych, O. Burylko, and P. A. Tass,\nPhys. Rev. Lett. 93, 084102 (2004).\n[19] P. A. Tass, Phase Resetting in Medicine and Biology\n[20] A. Arenas, A. Dı´az-Guilera, and C. J. Pe´rez-Vicente, Phys.\nRev. Lett. 96, 114102 (2006).\n[21] H. Kori and A. S. Mikhailov, Phys. Rev. Lett. 93, 254101\n[22] M. Denker, M. Timme, M. Diesmann, F. Wolf, and\nT. Geisel, Phys. Rev. Lett. 92, 074103 (2004).\n[23] F. Radicchi and H. Meyer-Ortmanns, Phys. Rev. E 73,\n[24] D. H. Zanette, Europhys. Lett. 68, 356 (2004); Eur. Phys.\nJ. B 43, 97 (2005).\n[25] M. Blank and L. Bunimovich, Nonlinearity 19, 329\n[26] M. Timme, Europhys. Lett. 76, 367 (2006).\n[27] We discard outliers: They may originate from perturba-\ntions that create (i) no structurally stable response or\n(ii) responses that are close to linearly dependent on\nothers.\n[28] S. Boyd and L. Vandenberghe, Convex Optimization\nPress, Cambridge, United\n(Cambridge University\nKingdom, 2004).\n[29] Here and in the following, the driving signals are inde-\npendent random Ii;m 2 0; I, where I  0:2.\n224101-4",
         "Convex Optimization;Does Dynamics Reflect Topology in Directed Networks;Designing complex networks;Entrainment of coupled oscillators on regular networks by pacemakers.;Chemical oscillations;Designing the dynamics of spiking neural networks.;Synchronization reveals topological scales in complex networks.;A method for determining neural connectivity and inferring the underlying network dynamics using extracellular spike recordings;Long range action in networks of chaotic elements;The Kuramoto model: A simple paradigm for synchronization phenomena;Disturbing synchronization: Propagation of perturbations \nin networks of coupled oscillators;Motifs in Brain Networks;Propagation of small perturbations in synchronized oscillator networks;Mechanism of desynchronization in the finite-dimensional Kuramoto model.;Interaction strengths in food webs: issues and opportunities;Entrainment of randomly coupled oscillator networks by a pacemaker.;Breaking synchrony by heterogeneity in complex networks.;Inferring Genetic Networks and Identifying Compound Mode of Action via Expression Profiling;Network motifs: simple building blocks of complex networks.;Wiring Optimization in Cortical Circuits;Modelling Food Webs;Exploring complex networks;From Kuramoto to Crawford: exploring the onset of synchronization in populations of coupled oscillators;Dynamics of neuronal firing correlation: modulation of \"effective connectivity\".;Chemical Oscillations, Waves, and Turbulence;Nonlinearity 19;Physica (Amsterdam) 224D;and O;Rev. Mod. Phys;J. Anim. Ecol;PLoS Biology 2;Phys. Rev. Lett;Europhys. Lett. Eur. Phys. J. B;Nature (London) 427;Science;Neuron 34;Synchronization: A Universal Concept in Nonlinear Sciences;Synchronization: Phase locking and frequency entrainment;Physica D;Phase Resetting in Medicine and Biology;We discard outliers: they may originate from perturbations that create (i) no structurally stable response or (ii) responses that are close to linearly dependent on others;The reconstruction quality depends not only on the degree of inhomogeneity, but also, among other parameters, on the realization of it and of the driving;A directed network is strongly connected if there is a directed path between any ordered pair of nodes;Here and in the following, the driving signals are independent random I i,m ∈ [0, ∆I], where ∆I = 0",
         "Revealing network connectivity from response dynamics.",
         "None"
        ],
        [
         "49",
         "00221da1856a3265b29461d06b3a9054c814f401",
         "The study of semantic memory in patients with Alzheimer's disease (AD) has raised important questions about the representation of conceptual knowledge in the human brain. It is still unknown whether semantic memory impairments are caused by localized damage to specialized regions or by diffuse damage to distributed representations within nonspecialized brain areas. To our knowledge, there have been no direct correlations of neuroimaging of in vivo brain function in AD with performance on tasks differentially addressing visual and functional knowledge of living and nonliving concepts. We used a semantic verification task and resting 18-fluorodeoxyglucose positron emission tomography in a group of mild to moderate AD patients to investigate this issue. The four task conditions required semantic knowledge of (1) visual, (2) functional properties of living objects, and (3) visual or (4) functional properties of nonliving objects. Visual property verification of living objects was significantly correlated with left posterior fusiform gyrus metabolism (Brodmann's area [BA] 37/19). Effects of visual and functional property verification for non-living objects largely overlapped in the left anterior temporal (BA 38/20) and bilateral premotor areas (BA 6), with the visual condition extending more into left lateral precentral areas. There were no associations with functional property verification for living concepts. Our results provide strong support for anatomically separable representations of living and nonliving concepts, as well as visual feature knowledge of living objects, and against distributed accounts of semantic memory that view visual and functional features of living and nonliving objects as distributed across a common set of brain areas.",
         "R. Zahn,P. Garrard,Jochen Talazko,M. Gondan,Philine Bubrowski,Freimut Juengling, Jungling, Jüngling,H. Slawik,P. Dykierek,Bernd Koester,M. Hull",
         "R. Zahn,P. Garrard,Jochen Talazko,M. Gondan,Philine Bubrowski,Freimut Juengling, Jungling, Jüngling,H. Slawik,P. Dykierek,Bernd Koester,M. Hull",
         "http://discovery.ucl.ac.uk/6036/1/6036.pdf",
         "& The study of semantic memory in patients with Alzheimer’s\ndisease (AD) has raised important questions about the repre-\nsentation of conceptual knowledge in the human brain. It is still\nunknown whether semantic memory impairments are caused\nby localized damage to specialized regions or by diffuse dam-\nage to distributed representations within nonspecialized brain\nareas. To our knowledge, there have been no direct corre-\nlations of neuroimaging of in vivo brain function in AD with\nperformance on tasks differentially addressing visual and func-\ntional knowledge of living and nonliving concepts. We used\na semantic verification task and resting 18-fluorodeoxyglucose\npositron emission tomography in a group of mild to moder-\nate AD patients to investigate this issue. The four task condi-\ntions required semantic knowledge of (1) visual, (2) functional\nproperties of living objects, and (3) visual or (4) functional\nproperties of nonliving objects. Visual property verification of\nliving objects was significantly correlated with left posterior\nfusiform gyrus metabolism (Brodmann’s area [BA] 37/19). Ef-\nfects of visual and functional property verification for non-\nliving objects largely overlapped in the left anterior temporal\n(BA 38/20) and bilateral premotor areas (BA 6), with the\nvisual condition extending more into left lateral precentral\nareas. There were no associations with functional property\nverification for living concepts. Our results provide strong\nsupport for anatomically separable representations of living\nand nonliving concepts, as well as visual feature knowledge of\nliving objects, and against distributed accounts of semantic\nmemory that view visual and functional\nliving\nand nonliving objects as distributed across a common set of\nbrain areas. &\nTheoretical accounts of category-specific semantic\ndeficits—in which concepts from living and nonliving\ndomains are independently impaired—fall into two dis-\ntinct classes. One posits anatomically separable repre-\nsentations for different types of semantic information,\nwhereas the other assumes that all semantic knowledge\nis represented in a distributed fashion within the same\nregions, irrespective of type.",
         "DISCUSSION\nThe study reported here was designed to identify corre-\nlations between regions of cortical hypometabolism and\nperformance on a semantic feature verification task in\nwhich conceptual domain membership (living vs. non-\nliving) was crossed with type of knowledge probed\n(visual vs. functional). The principal motivation for the\nstudy was to test the contrasting predictions of a num-\nber of theoretical models of semantic organization, con-\ncerning the relationship between living and nonliving\nconcepts on the one hand, and visual and functional\nknowledge on the other. The study’s major findings\ncan be summarized as the association of (a) impaired\nknowledge of the visual properties of living concepts\nwith hypometabolism in the posterior fusiform gyrus\n(BA 37/19) and (b) impaired knowledge of the visual and\nfunctional properties of nonliving concepts with hypo-\nmetabolism in the left anterior temporal (BA 38/20) and\nmotor/premotor areas (BA 4/6). Both findings are in\nkeeping with a large body of previously published data\nlinking brain regions with semantic categories and dif-\nferent feature types.\nThe left posterior fusiform region has been linked to\nvisual property verification using functional MRI (Kan,\nBarsalou, Solomon, Minor, & Thompson-Schill, 2003),\nand to noun imageability using PET (Wise et al., 2000),\nalthough neither study contrasted living and nonliving\nconcepts. Thompson-Schill, Aguirre, Desposito, and Farah\n(1999) used a verbal property verification task to show\nactivation of the posterior fusiform gyrus (BA 37) for all\naspects of living things and for nonliving concepts when\nvisual property knowledge was explicitly required. As Price\net al. (1999) have pointed out, however, functional activa-\ntion in normal subjects is likely to include regions that are\ninvolved in but not necessary for task performance. Thus,\nalthough the posterior fusiform gyrus might be activated\nfor both living and nonliving concepts, the data reported\nhere would suggest that it may be necessary only for judg-\nments involving the visual properties of living things.\nThe association between knowledge of nonliving con-\nlobe accords with\ncepts and the anterior temporal\ndescriptions of patients with anterior temporal\nlobe\nresections who show selective naming impairments\nfor both human actions (Lu et al., 2002) and nonliving\nitems (Cappa, Frugoni, Pasquali, Perani, & Zorat, 1998;\nTippett, Glosser, & Farah, 1996). The importance of\nanterior temporal cortex for semantic memory is well\nestablished: Its supramodal role in semantic knowledge\ncan be inferred from the multimodal impairments char-\nacteristic of the syndrome of semantic dementia, which\nis associated with atrophy in anterior temporal regions\n(Davies et al., 2005; Rogers et al., 2004; Hodges, Bozeat,\nLambon Ralph, Patterson, & Spatt, 2000). Semantic\nknowledge impairments in semantic dementia also im-\npact on tool use, emphasizing the importance of the\nanterior temporal lobe for the representation of func-\ntional information (Hodges et al., 2000).\nThe importance of premotor areas for nonliving con-\ncepts accords with evidence from functional activation\nstudies in normal subjects pointing to the involvement of\nleft premotor cortex (BA 6) in the representation of tool\nand action knowledge (Martin & Chao, 2001) as well as\naction perception (Rizzolatti, Fogassi, & Gallese, 2002).\nAlso, primary motor cortex (BA 4) is activated during\nsemantic comprehension of verbs describing movement\nrelated actions (Hauk, Johnsrude, & Pulvermuller, 2004).\nFinally, the dependence of visual knowledge for nonliving\nconcepts on motor rather than visual areas is consistent\nwith the idea that imagery of motor acts is involved in\nretrieving visual information about artifacts (review of\n‘‘motor imagery’’ by Jeannerod & Frak, 1999).\nThe implications of these anatomical patterns for rival\ntheories of category specificity in AD are twofold. In the\nfirst place, it is clear that impairment of semantic knowl-\nedge for living and nonliving objects in AD patients\nresulted from regional dysfunction in anatomically sepa-\nrable brain areas. Second, higher order visual information\nin the left posterior fusiform region may relate specifi-\ncally to living concepts. Neither of these findings can be\naccounted for by theories based on the assumption of a\nwidely distributed supramodal semantic memory store.\nBy contrast, the predictions of Garrard et al. (1998,\n2001), based on the patterns of performance in tasks\nassessing semantic knowledge for living and nonliving\nconcepts in AD, were, at least partially, corroborated.\nAccording to this account, dissociations of semantic\nknowledge for living and nonliving concepts should be\nbecause of differences in the regional distribution of\npathology in AD. Temporal neocortical regions, which\nare most subject to degenerative damage in AD, are\ndifferentially important in the representation of living\nconcepts and give rise to the more typical nonliving ad-\nvantage in the earlier disease stages. Without in vivo ana-\ntomical support for a concomitant sensory–functional\ndifference, however, Garrard et al. conceded that the\npatterns of performance could also be explained by\nassuming ‘‘evolutionarily adapted domain-specific knowl-\nedge systems subserved by distinct neural mechanisms’’\n(Caramazza & Shelton, 1998).\nThe importance of inferior temporal dysfunction in\nthe impairment of living concept knowledge in AD is\nsupported here by the finding of such an association\nwithin the posterior fusiform gyrus (BA 37/19). It is also\nin accordance with the literature on category-specific\ndeficits for living things due to other causes (e.g., her-\npes simplex encephalitis, closed head injury) showing\nlobe involvement most frequently\ninferior temporal\n(Gainotti, 2000). The finding of a specific association\nof visual\nfeatures alone with left temporal changes\nsuggests in addition a link between visual feature knowl-\nedge and the impairments of living concept knowledge\nin AD, as predicted by the sensory–functional, but not\nthe domain-specific, theory. Our second finding of an\nassociation of nonliving but not living concept knowl-\nedge with anterior temporal and premotor cortical\nmetabolism, however, is more difficult to reconcile with\nthe sensory–functional theory. In this instance, the\noverlap of regions associated with visual and functional\nfeature knowledge of nonliving objects may be better\nexplained by the domain-specific hypothesis.\nOur seemingly contradictory findings with regard to\nthese two competing theoretical accounts of anatomically\nseparable subdivisions for different types of semantic\nknowledge can be best explained within a more recent\nformulation of the domain-specific account (Caramazza\n& Mahon, 2003). This hybrid model admits of the co-\nexistence of supramodal domain-specific regions for ani-\nmals, fruits/vegetables, and tools and additional unimodal\nfeature-specific semantic regions. Indeed, a growing body\nof evidence from the cognitive neuropsychological liter-\nature suggests that impairments of living and nonliving\nobject knowledge can occur in the absence of more\npronounced effects of either visual or functional fea-\ntures (Caramazza & Mahon, 2003; Laiacona, Capitani, &\nCaramazza, 2003; Zannino, Perri, Carlesimo, Pasqualetti,\n& Caltagirone, 2002; Lambon-Ralph, 1998). Moreover,\nvisual feature knowledge can be impaired in patients\nwithout leading to category-specific effects for living\nconcepts (Caramazza & Mahon, 2003; Lambon-Ralph,\n1998)—a pattern that, we note in passing, was exhibited\nby one of the patients in the present study. Our data thus\naccord with the heterogeneity of category-specific se-\nmantic disorders reported in the literature (Rosazza\net al., 2003).\nThe finding of regional dysfunction within the pre-\nmotor and anterior temporal cortex is in agreement with\npostmortem neurochemical studies that detected vari-\nable degrees of tau pathology in the anterior temporal\nareas (BA 20/38) and synaptic loss in the premotor area\n(BA 6) even at predementia stages of AD (Vermersch\net al., 1995; Clinton, Blackman, Royston, & Roberts,\n1994). Previous studies in AD patients using regression\nanalyses and global measures of semantic functions\n(naming and/or nonverbal impairment) have also shown\nassociations with premotor (Grossman et al., 2004)\nand anterior temporal (Zahn et al., 2004) areas using\nvoxel-based morphometry and FDG-PET, respectively.\nFunctional disconnection (diaschisis) of the anterior\ntemporal and motor/premotor regions caused by neuro-\ndegeneration within the posterior temporal or parietal\nassociation cortices also provide a plausible pathophys-\niological mechanism. The posterior temporal and parie-\ntal association cortex are highly connected to these\nregions (Creutzfeldt, 1983) and are severely and con-\nsistently affected by tangle neuropathology (Harasty,\nHalliday, Kril, & Code, 1999; Braak & Braak, 1991).\nOur negative finding with regard to significant associ-\nations for functional knowledge of living objects with\nany brain area points to greater variability in the associ-\nation of regional metabolism and task performance than\nin the other task conditions. Given the robust correla-\ntions of specific regional metabolism with performance\nin the other conditions, this negative finding is unlikely\nto be due simply to a lack of statistical power. An alter-\nnative explanation is that the knowledge required for\nthe task was represented across multiple regions. This\nwould be consistent with the heterogeneity characteristic\nof functional features of living objects, which may de-\nscribe their use by man or animal, characteristic motion,\nenvironmental context, or general ‘‘encyclopedic knowl-\nedge,’’ to name but a few. Thus, the a priori probability of\nlinking this diverse group of features to a specific brain\narea might be expected to be lower. Future studies using\ntasks with more differentiated subclassifications of fea-\ntures as proposed by, for example, Cree and McRae\n(2003) may well reveal specific brain regions linked to\nsubclasses of functional knowledge for living things.\nOur findings of regional hypometabolism that is asso-\nciated with semantic impairment in AD has to be inter-\npreted in conjunction with evidence from other sources\nto rule out a functional age-related origin of these ab-\nnormalities. Neuroimaging studies and neuropathology\nshowed abnormalities in AD patients compared with aged\nhealthy controls in those regions that were associated\nwith semantic memory dysfunction in our study (Zahn\net al., 2005; Grossman et al., 2004; Herholz et al., 2002;\nHarasty et al., 1999; Clinton et al., 1994; Braak & Braak,\n1991). PET studies showed age effects primarily outside of\ntemporal and premotor regions relevant to performance\nin our task (Herholz et al., 2002). Furthermore, our age-\nmatched control group performed almost perfectly on\nthe semantic task. Thus, normal aging does not markedly\naffect explicit semantic memory and associated brain\nregions. In summary, this supports the conclusion that\nregional effects demonstrated in our AD group did not\narise by functional age-related changes but by disease-\nrelated pathology. On a more cautious note, the robust-\nness of our findings would undoubtedly benefit from\nfurther studies replicating the regional associations seen\nin the present sample and thus confirming consistent\nregional effects across different AD groups.\nIn conclusion, therefore, the findings of the present\nstudy support the view that semantic organization is\ncharacterized both by category-specific supramodal and\nunimodal feature-specific regions. It should be noted\nthat these data neither demonstrate nor rule out the\npresence of category-specific subdivisions within the\nsupramodal anterior temporal area, where visual and\nfunctional feature knowledge for nonliving things over-\nlap. The findings would therefore be equally consistent\nwith theories positing supramodal semantic stores with-\nout category-specific subdivisions for living and non-\nliving concepts that allow for additional feature-specific\nregions with category-specific subdivisions. To the best\nof our knowledge, however, no single cases have yet\nbeen described that would be consistent with either of\nthese interpretations.",
         "INTRODUCTION\nAmong the advocates of the former, disagreement\npersists about how semantic information is subdivided:\nAccording to the sensory–functional theory (SFT), the\nperceptual and functional properties of concepts de-\npend on different brain regions, and category-specific\ndeficits are explained by the differential importance of\nthese two types of information for living and nonliving\nconcepts, together with selective lesions to visual or\nfunctional feature regions ( Warrington & Shallice, 1984).\nVariants of the sensory–functional theory have proposed\nthat domain-specific subdivisions may exist within such\nfeature-specific areas (McCarthy & Warrington, 1988)\nor, more recently, that conceptual knowledge arises\nfrom feature knowledge stored in the areas critical to\nlexical, as well as perceptual, and motor representa-\ntions (Barsalou, Kyle Simmons, Barbey, & Wilson, 2003;\nPulvermuller, 1999). In contrast, the domain-specific\naccount proposed by Caramazza and Mahon (2003)\nand Caramazza and Shelton (1998) assumes that evolu-\ntionary pressures have given rise to anatomically sepa-\nrable, generally amodal systems, representing semantic\nknowledge about animals, vegetables, and artifacts.\nDistributed theories do not postulate separable se-\nmantic stores for either categories or semantic features\n(Rogers et al., 2004; Moss & Tyler, 2000; Devlin,\nGonnerman, Andersen, & Seidenberg, 1998; Gonnerman,\nAndersen, Devlin, Kempler, & Seidenberg, 1997;\nCaramazza, Hillis, & Rapp, 1990) but account for category-\nspecific deficits by appealing to differences in concept\nstructure: the more co-occurring (‘‘intercorrelated’’)\nfeatures contributing to a concept’s representation, the\nless likely that concept is to become degraded by par-\ntial damage to distributed representations. A different\ncomposition of feature co-occurrences in living and non-\nliving concepts has been proposed as the basis of dif-\nferential impairments (Moss & Tyler, 2000; Devlin et al.,\n1998; Gonnerman et al., 1997; Caramazza et al., 1990).\nNeuropsychological studies of cohorts of patients with\nAlzheimer’s disease (AD) have shown that semantic\nprocessing is impaired more frequently and deteriorates\nmore rapidly for living concepts than for those in the\nnonliving domain. Garrard et al. (2001) and Garrard,\nPatterson, Watson, and Hodges (1998) argued that the\ncategory-specific patterns emerging at different stages\nof AD were because of the impact of regional distribu-\ntion of pathology on the representation of sensory and\nfunctional\ninformation. Apart from a small subset of\nindividual cases with available neuropathology, how-\never, no hard neuroanatomical evidence was available\nto back up this claim. More recently, correlations be-\ntween in vivo measures of regional pathology such as\n18-fluorodeoxyglucose positron emission tomography\n(FDG-PET) or quantitative magnetic resonance imaging\n(MRI) and functional indices of semantic memory integ-\nrity in AD have been used to identify regions important\nto picture naming and nonverbal semantic tasks (Zahn\net al., 2004; Hirono et al., 2001; Desgranges et al., 1998;\nGrossman et al., 1997, 2004). The technique provides a\nnovel opportunity to test Garrard et al.’s hypothesis on\na larger scale.\nThe aim of the present study was therefore to correlate\ninformation from FDG-PET—a sensitive and valid mea-\nsure of regional brain function in AD (Herholz et al.,\n2002)—with performance on an experimental semantic\ntask requiring knowledge of the sensory and functional\nfeature knowledge of living and nonliving concepts. Only\nmild to moderate AD patients were included, so that\nsemantic impairments would not be confounded with\noverall disease severity. We chose a feature verification\ntask, which required patients to answer simple questions\nabout the visual (e.g., ‘‘a zebra has stripes’’) and func-\ntional properties (e.g., ‘‘an axe chops’’) of object con-\ncepts. This task has a number of advantages over\nconventional semantic tasks such as picture naming:\nfirst, it may be more sensitive to category-specific impair-\nments than picture naming (Laws & Sartori, 2005); sec-\nond, it allows visual and functional knowledge of living\nand nonliving concepts to be specifically and separately\nprobed; finally, the fact that only the responses ‘‘true’’ or\n‘‘false’’ are required, reduces the likelihood of con-\nfounds between experimental conditions and difficulties\nof lexical retrieval or word production.\nThe central assumption was that regions in which me-\ntabolism correlates with performance on each of the\nfour conditions (living visual, living functional, nonliving\nvisual, nonliving functional) are critical to the represen-\ntation of that specific kind of semantic information. We\nwould further contend that although activation studies\nof normal subjects identify a set of brain regions that\nare involved in a particular task, the functional lesion\napproach reveals areas that are necessary for successful\nperformance (Price, Mummery, Moore, Frakowiak, &\nFriston, 1999) and, as such, represents an indispensable\nadjunct to this methodology.\nWe tested the following predictions from competing\ntheories of semantic memory organization and its im-\npairment in AD:\n(a) According to the sensory–functional theory, brain\nareas necessary for the representation of visual and\nfunctional feature knowledge are clearly separable.\nSubregions within each of\nthese areas may be\ndifferentially important\nfor living and nonliving\nconcepts.\n(b) The domain-specific account predicts the existence\nof separable brain areas underpinning the represen-\ntation of living and nonliving concepts with knowl-\nedge of\nfeatures\nthe perceptual and functional\nsubserved by the same brain regions for each\ncategory.\n(c) Theories of the distributed form would predict\nlargely overlapping brain regions necessary for all\ntypes of knowledge relating to both living and non-\nliving concepts, although damage to different types\nof feature may vary according to the overall extent\nof dysfunction.\nMETHODS\nPatients",
         "Sixteen mild to moderate AD patients (all right-handed;\n9 women; mean age = 67.7 years, SD = 8.5 years; mean\nyears of schooling, 9.4, SD = 2.0; mean Mini Mental State\nExamination [MMSE] score = 23.1, SD = 2.9; Clinical\nDementia Rating score = 1 in 11 patients and Clini-\ncal Dementia Rating score = 2 in 5 patients) fulfilling\nthe criteria of the National Institute of Neurological\nand Communicative Diseases and Stroke/Alzheimer’s\nDisease and Related Disorders Association (McKhann\net al., 1984) for probable AD and not fulfilling revised\nconsensus criteria for frontotemporal dementia (Neary\net al., 1998) were recruited through the Memory Clinic\nat the University of Freiburg. In one patient, a history of\ndelusions and cognitive fluctuation raised the possibility\nof comorbid Lewy body disease (McKeith et al., 1996).\nHis semantic memory test profile was within the range\n(mean ± 1.5 SD) of the other patients in the group.\nNeuropsychological evaluation revealed consistent im-\npairments of verbal and nonverbal delayed recall in all\npatients with additional naming or visuoconstructive\ndeficits in different subgroups of patients (see Table 1).\nMRI revealed changes of diffuse and symmetrical volume\nloss and bilateral hippocampal atrophy, together with\nminimal white matter lesions in some cases but no\nevidence of other pathology. Other causes of cognitive\ndecline had been excluded based on routine blood tests.\nNone of the patients had a history of stroke, other\nneurological disease, or major or recurrent psychiatric\nsymptomatology, including substance abuse, before on-\nset of dementia. All were native speakers of German.\nNone of the patients had received cholinesterase inhib-\nitors or cholinergic medication during the study; four\npatients were taking selective serotonin reuptake inhib-\nitors and two were taking an atypical neuroleptic drug\n(quetiapine).\nNormal Controls\nFourteen healthy right-handed volunteers (7 women;\nmean MMSE = 29.3, SD = 1.1) matched to the patient\ngroup for age and years of schooling ( p > .4, Mann–\nWhitney U test), without history of central nervous\nsystem or psychiatric disease, substance abuse, or psy-\nchotropic medication, provided control data for the\nexperimental semantic tasks described below. All per-\nformed at normal levels (z > 1) on the Consortium to\nEstablish a Registry for Alzheimer’s Disease (CERAD) test\nbattery (a comprehensive standardized neuropsycholog-\nical test battery for the diagnosis of dementia; German\nnorms from Berres, Monsch, Bernasconi, Thalmann, &\nStahelin, 2000). Because of local ethics regulations, no\nage-matched control group was available for the PET\nstudy, so data from a group of 12 healthy normal\nsubjects without psychiatric or neurological diseases\nor psychotropic medication (7 women; mean age =\n48 years, SD = 16 years) were used. The most important\naspect of the PET analysis (parametric regression),\nhowever, did not require control data.\nSemantic Verification Task\nA set of 192 simple propositions of the form ‘‘a zebra has\nstripes’’ or ‘‘an axe chops’’ were read aloud. Half of the\npropositions were true, and half were false, and subjects\nwere simply required to make a verbal ‘‘true-or-false’’\ndecision.\nThe properties used in this test were taken from\nresponses on a feature-generation task performed by a\nnonoverlapping group of 60 healthy volunteers (mean\nage = 27.1 years, SD = 10.9). For this task, words for\n120 concepts (60 living and 60 nonliving) were selected\nfrom Snodgrass and Vanderwart’s (1980) set. The words\nwere divided into two sublists of 60 words (30 living and\n30 nonliving), each of which was given to a subgroup of\n30 controls. Controls were asked first to perform a free\nforward word association task by writing down the first\nword that came to mind after reading each object name.\nThe percentage of subjects naming a word in response\nto a given object name yielded a measure of ‘‘forward\nassociativity’’. Subjects were then instructed to write\ndown between four and eight properties commonly\nattributed to each concept in turn. Each property was\nassigned a ‘‘dominance’’ value, equivalent to the pro-\nportion of subjects generating it in response to a word\n(Ashcraft, 1978). Feature distinctiveness was determined\nas the proportion of same-domain concepts for which\nthe feature was generated.\nProperties with dominance values of less than 15% were\ndiscarded, and 96 selected from the remainder such that\none visual and one functional property were associated\nwith each of 24 living and 24 nonliving concepts. Living\nand nonliving concept words and visual and functional\nfeature words were matched for spoken and written word\nfrequency and syllable length using the CELEX lexical\ndatabase (www.mpi.nl/world/celex/). Feature words were\nalso matched across domain and across type by domi-\nnance and distinctiveness. All concept-feature pairs had\nless than 15% associativity. The combination of feature\ndistinctiveness and feature dominance provides informa-\ntion comparable to the ‘‘feature relevance’’ measure of\nSartori and Lombardi (2004). An equal set of distracter\nstatements was formed by pseudorandom recombination\nof the selected properties with concepts to make a total\nof 192 statements in the task. Properties of living con-\ncepts were recombined with living concepts and nonliving\nproperties with nonliving concepts only.\nThe feature verification task was administered to 14\ncognitively normal age- and education-matched con-\ntrols. To equate the item difficulty across stimulus con-\nditions a subset of 128 statements with high response\nconsistency (>90% agreement) per item across nor-\nmal controls was selected. The properties of the final\n128 items used in the task are summarized in Table 2.\nSemantic Memory Test Battery\nSemantic memory integrity was documented using a\nGerman-language version of the semantic memory test\nbattery described by Hodges, Salmon, and Butters\n(1992). The battery, described in detail by Zahn et al.\n(2004), is based on a set of 48 line drawings and asso-\nciated concept names from Snodgrass and Vanderwart’s\n(1980) set. Half the items are drawn from the living and\nhalf from the nonliving domain, matched for proto-\ntypicality, familiarity, visual complexity, spoken frequen-\ncy, and syllable length. Subjects are presented with 48\nline drawings for naming and with 48 arrays containing a\ntarget and within-category foils, for spoken word-to-\npicture matching. Half of the items were presented\nwithin an array of 3, the other half within an array of\n6 items. Subjects were also asked to produce as many\nexemplars of the following categories in a fixed order for\n1 min each: land animals, clothing, vegetables, vehicles,\ninsects, kitchen items, fruits, and musical instruments.\nThe sum of correct, nonrepeated responses was re-\ncorded. The picture version of the Pyramids and Palm\nTrees (PPT) test (Howard & Patterson, 1992) was ad-\nministered as a nonlinguistic test of associative semantic\nknowledge.\nPositron Emission Tomography Imaging\nThe PET procedure was performed according to pre-\nviously defined standards ( Juengling, Kassubek, & Otte,\n2000). Patient and control participants were allowed to\nrest in an acoustically isolated and dimmed room for at\nleast 10 min before injection of 200 ± 20 MBq 18-FDG\nand for another 20 min during the uptake period.\nParticipants were then transferred to the scanner (Sie-\nmens, Erlangen, Germany, CTI ECAT exact tomograph,\n10.8 cm field of view, 6.8 mm full width half maximum),\nwith their heads positioned relative to the orbitomeatal\nline. Image acquisition began 30 min after injection. Six\ndynamic frames were acquired, each of 5-min duration.\nImages were reconstructed using filtered back-projec-\ntion (Shepp-Logan filter; cutoff = 0.35 cycles/pixel).\nAttenuation correction was performed using the stan-\ndard mathematical algorithm implemented in ECAT\nsoftware. The dynamic frames were then checked for\nmotion artifacts and summed to generate a single data\nset of 31 transaxial planes.\nFor analysis using Statistical Parametric Mapping (SPM2,\nimage data were converted to\nwww.fil.ion.ucl.ac.uk),\nANALYZE format, and automated spatial normalization\nwas performed to realign the data set to a common 3-D\nstereotaxic space using the Montreal Neurological Insti-\ntute template (Friston, Holmes, & Worsley, 1995). Be-\nfore voxel-based analysis, images were smoothed using\nan 18  18  18-mm Gaussian kernel to account for\ninterindividual anatomical variability. The global cerebral\nmetabolic rate for glucose (gCMRGlc) was normalized to\na mean of 50 Amol/100 mL/min. The normalized FDG-\nPET data of the patients were compared with healthy\nsubjects by computing pixel-by-pixel t statistics for detec-\ntion of hypometabolic areas (Signorini et al., 1999). For\nanatomical identification, the coordinates derived from\nthe Montreal Neurological Institute template were trans-\nformed using the appropriate algorithm (cf. www.mrc-\ncbu.cam.ac.uk/Imaging/mnispace.html) to comply with\nthe original grid by Talairach and Tournoux. All areas ex-\nceeding a voxel threshold level of z > 3.09 and p = .001,\nuncorrected, and a size of >15 voxels are reported in the\ngroup comparison.\nPercent correct scores on each condition of the se-\nmantic verification task were entered as covariates of\ninterest in the within-patient group parametric regres-\nsion analysis with the aim of identifying regions showing\ndecreased metabolism associated with low performance.\nFor covariance analysis, voxel clusters with t values\ncorresponding to p < .01 in a single test and a minimum\ncluster size of 15 voxels were displayed as projected on\nlateral views of a standard brain (Figure 3). Only regions\nwith a voxel significance level of p < .001, z > 3.09, and\na cluster size of >15 voxels at p = .01 are reported in\nTable 4 and discussed below.\nRESULTS\nNeuropsychology\nBackground Neuropsychology",
         "As a group, the AD patients showed severe impairments\nof direct and delayed verbal as well as nonverbal re-\ncall and verbal recognition in the CERAD test battery\n(Table 1, z < 2 SD of age- and education-matched\nnorm population mean). Boston Naming Test and figure\ndrawing group means were in the less severely impaired\nrange (z < 1). Individual case analyses showed most\nconsistent impairments in memory and fluency subtests;\nBoston Naming Test and figure drawing were impaired\nin different subgroups of patients.\nSemantic Memory Test Results\nsurvived a more stringent Bonferroni-corrected thresh-\nold ( p = .003; equivalent to an approximate Bonferroni-\ncorrected threshold of p = .05). On an individual case\nbasis, more than two thirds of the AD patients were\nbelow the normal cutoff score (below normal mean\n 2 SD and below normal range) in the following tests:\nsemantic word fluency, picture naming, and semantic\nverification. Word–picture matching was above the nor-\nmal cutoff score in 13 of 16 patients, and nonverbal\nsemantic memory (PPT test) was below the cutoff in 6\nof 16 patients.\nSemantic Verification Task Results\nAs a group, patients performed at a lower level than\ncontrols in all task conditions (Table 3). The scores of\n13 of 16 patients fell below the normal cutoff in the\nvisual conditions; of 16 patients, 10 were below cutoff\nin the living functional conditions and 9 in the nonliv-\ning functional condition (Table 3). Normal subjects per-\nformed at ceiling level all conditions. Figure 1 shows\nthe comparative levels of accuracy (percentage of cor-\nrect responses) achieved by patients in the four condi-\ntions of the semantic verification task. A nonparametric\none-way analysis of variance over the four task condi-\ntions revealed no significant effect of task condition\non performance [x2(3)= 7.5, p = .06, ns, Friedman test,\nasymptotic].\nSide maxima for large clusters are in italics. NC = normal controls. All areas exceeding a voxel threshold level of p = .001, uncorrected, and a cluster\nsize of >15 voxels are reported (SPM2). One patient had scores below the patient group mean  2 SD on three conditions of the verification task\n(see Figure 1). To rule out the possibility that our regression analyses were disproportionately inf luenced by this extreme case we repeated the\nanalysis after excluding the outlier. All reported regions were confirmed in the reanalysis.\npatterns of performance on word picture matching or\nfluency in any of the patients.\nWithin-Alzheimer’s Disease Patient Group\nRegression Analyses\n18-Fluorodeoxyglucose Positron\nEmission Tomography\nAlzheimer’s Disease versus Normal Control\nGroup Comparison\nThe group comparison shows a typical pattern of hy-\npometabolism in AD patients relative to normal con-\ntrols in the bilateral dorsolateral prefrontal cortices\n(Brodmann’s area [BA] 8/9), bilateral inferior temporal\ngyri (BA 20), the left anterior superior temporal gyrus\n(BA 22), the right posterior superior temporal sulcus\n(BA 22/21), bilateral angular gyri (BA 39/7), and bilateral\nposterior cingulate gyri/precuneus (BA 31) (Figure 3,\nTable 4).\nThe main focus of this article is the result of the within-\nAD patient group regression analyses, independent of\nnormal controls. These analyses identify brain regions in\nthe AD group where glucose metabolism was positively\ncorrelated with performance in the respective condi-\ntion of the semantic verification task (living visual,\nliving functional, nonliving visual, nonliving functional).\nFigure 3 shows 3-D projections of these significantly\ncorrelated regions on a more lenient level of significance\n( p = .01) to give an impression of the extension of\nareas. We will only report and discuss regions in the text\nand Table 4, however, that achieved a more stringent\nlevel of significance ( p = .001, 15 voxels).\nPerformance on verification of the visual properties of\nliving things was significantly correlated with metabolism\nin the left posterior fusiform gyrus (BA 37/19), whereas\nperformance with both visual and functional properties\nof the nonliving concepts was significantly associated with\nmetabolism in left anterior temporal (BA 20/38) and bi-\nlateral precentral/premotor regions (BA 4/6). The right\nfrontal pole (BA 10) was associated with the nonliving vi-\nsual properties only. No area exceeded threshold for the\nliving functional condition. The comparison of the ex-\ntensions of significantly associated areas (Figure 3) shows\na more pronounced extension within the left lateral\nprecentral cortex for the nonliving visual condition and\na larger extension of premotor hypometabolism into the\nright hemisphere for the nonliving functional condition.",
         "downloading",
         "& The study of semantic memory in patients with Alzheimer’s\ndisease (AD) has raised important questions about the repre-\nsentation of conceptual knowledge in the human brain. It is still\nunknown whether semantic memory impairments are caused\nby localized damage to specialized regions or by diffuse dam-\nage to distributed representations within nonspecialized brain\nareas. To our knowledge, there have been no direct corre-\nlations of neuroimaging of in vivo brain function in AD with\nperformance on tasks differentially addressing visual and func-\ntional knowledge of living and nonliving concepts. We used\na semantic verification task and resting 18-fluorodeoxyglucose\npositron emission tomography in a group of mild to moder-\nate AD patients to investigate this issue. The four task condi-\ntions required semantic knowledge of (1) visual, (2) functional\nproperties of living objects, and (3) visual or (4) functional\nproperties of nonliving objects. Visual property verification of\nliving objects was significantly correlated with left posterior\nfusiform gyrus metabolism (Brodmann’s area [BA] 37/19). Ef-\nfects of visual and functional property verification for non-\nliving objects largely overlapped in the left anterior temporal\n(BA 38/20) and bilateral premotor areas (BA 6), with the\nvisual condition extending more into left lateral precentral\nareas. There were no associations with functional property\nverification for living concepts. Our results provide strong\nsupport for anatomically separable representations of living\nand nonliving concepts, as well as visual feature knowledge of\nliving objects, and against distributed accounts of semantic\nmemory that view visual and functional\nliving\nand nonliving objects as distributed across a common set of\nbrain areas. &\nINTRODUCTION\nTheoretical accounts of category-specific semantic\ndeficits—in which concepts from living and nonliving\ndomains are independently impaired—fall into two dis-\ntinct classes. One posits anatomically separable repre-\nsentations for different types of semantic information,\nwhereas the other assumes that all semantic knowledge\nis represented in a distributed fashion within the same\nregions, irrespective of type.\nAmong the advocates of the former, disagreement\npersists about how semantic information is subdivided:\nAccording to the sensory–functional theory (SFT), the\nperceptual and functional properties of concepts de-\npend on different brain regions, and category-specific\ndeficits are explained by the differential importance of\nthese two types of information for living and nonliving\nconcepts, together with selective lesions to visual or\nfunctional feature regions ( Warrington & Shallice, 1984).\nVariants of the sensory–functional theory have proposed\nthat domain-specific subdivisions may exist within such\nfeature-specific areas (McCarthy & Warrington, 1988)\nor, more recently, that conceptual knowledge arises\nfrom feature knowledge stored in the areas critical to\nlexical, as well as perceptual, and motor representa-\ntions (Barsalou, Kyle Simmons, Barbey, & Wilson, 2003;\nPulvermuller, 1999). In contrast, the domain-specific\naccount proposed by Caramazza and Mahon (2003)\nand Caramazza and Shelton (1998) assumes that evolu-\ntionary pressures have given rise to anatomically sepa-\nrable, generally amodal systems, representing semantic\nknowledge about animals, vegetables, and artifacts.\nDistributed theories do not postulate separable se-\nmantic stores for either categories or semantic features\n(Rogers et al., 2004; Moss & Tyler, 2000; Devlin,\nGonnerman, Andersen, & Seidenberg, 1998; Gonnerman,\nAndersen, Devlin, Kempler, & Seidenberg, 1997;\nCaramazza, Hillis, & Rapp, 1990) but account for category-\nspecific deficits by appealing to differences in concept\nstructure: the more co-occurring (‘‘intercorrelated’’)\nfeatures contributing to a concept’s representation, the\nless likely that concept is to become degraded by par-\ntial damage to distributed representations. A different\ncomposition of feature co-occurrences in living and non-\nliving concepts has been proposed as the basis of dif-\nferential impairments (Moss & Tyler, 2000; Devlin et al.,\n1998; Gonnerman et al., 1997; Caramazza et al., 1990).\nNeuropsychological studies of cohorts of patients with\nAlzheimer’s disease (AD) have shown that semantic\nprocessing is impaired more frequently and deteriorates\nmore rapidly for living concepts than for those in the\nnonliving domain. Garrard et al. (2001) and Garrard,\nPatterson, Watson, and Hodges (1998) argued that the\ncategory-specific patterns emerging at different stages\nof AD were because of the impact of regional distribu-\ntion of pathology on the representation of sensory and\nfunctional\ninformation. Apart from a small subset of\nindividual cases with available neuropathology, how-\never, no hard neuroanatomical evidence was available\nto back up this claim. More recently, correlations be-\ntween in vivo measures of regional pathology such as\n18-fluorodeoxyglucose positron emission tomography\n(FDG-PET) or quantitative magnetic resonance imaging\n(MRI) and functional indices of semantic memory integ-\nrity in AD have been used to identify regions important\nto picture naming and nonverbal semantic tasks (Zahn\net al., 2004; Hirono et al., 2001; Desgranges et al., 1998;\nGrossman et al., 1997, 2004). The technique provides a\nnovel opportunity to test Garrard et al.’s hypothesis on\na larger scale.\nThe aim of the present study was therefore to correlate\ninformation from FDG-PET—a sensitive and valid mea-\nsure of regional brain function in AD (Herholz et al.,\n2002)—with performance on an experimental semantic\ntask requiring knowledge of the sensory and functional\nfeature knowledge of living and nonliving concepts. Only\nmild to moderate AD patients were included, so that\nsemantic impairments would not be confounded with\noverall disease severity. We chose a feature verification\ntask, which required patients to answer simple questions\nabout the visual (e.g., ‘‘a zebra has stripes’’) and func-\ntional properties (e.g., ‘‘an axe chops’’) of object con-\ncepts. This task has a number of advantages over\nconventional semantic tasks such as picture naming:\nfirst, it may be more sensitive to category-specific impair-\nments than picture naming (Laws & Sartori, 2005); sec-\nond, it allows visual and functional knowledge of living\nand nonliving concepts to be specifically and separately\nprobed; finally, the fact that only the responses ‘‘true’’ or\n‘‘false’’ are required, reduces the likelihood of con-\nfounds between experimental conditions and difficulties\nof lexical retrieval or word production.\nThe central assumption was that regions in which me-\ntabolism correlates with performance on each of the\nfour conditions (living visual, living functional, nonliving\nvisual, nonliving functional) are critical to the represen-\ntation of that specific kind of semantic information. We\nwould further contend that although activation studies\nof normal subjects identify a set of brain regions that\nare involved in a particular task, the functional lesion\napproach reveals areas that are necessary for successful\nperformance (Price, Mummery, Moore, Frakowiak, &\nFriston, 1999) and, as such, represents an indispensable\nadjunct to this methodology.\nWe tested the following predictions from competing\ntheories of semantic memory organization and its im-\npairment in AD:\n(a) According to the sensory–functional theory, brain\nareas necessary for the representation of visual and\nfunctional feature knowledge are clearly separable.\nSubregions within each of\nthese areas may be\ndifferentially important\nfor living and nonliving\nconcepts.\n(b) The domain-specific account predicts the existence\nof separable brain areas underpinning the represen-\ntation of living and nonliving concepts with knowl-\nedge of\nfeatures\nthe perceptual and functional\nsubserved by the same brain regions for each\ncategory.\n(c) Theories of the distributed form would predict\nlargely overlapping brain regions necessary for all\ntypes of knowledge relating to both living and non-\nliving concepts, although damage to different types\nof feature may vary according to the overall extent\nof dysfunction.\nMETHODS\nPatients\nSixteen mild to moderate AD patients (all right-handed;\n9 women; mean age = 67.7 years, SD = 8.5 years; mean\nyears of schooling, 9.4, SD = 2.0; mean Mini Mental State\nExamination [MMSE] score = 23.1, SD = 2.9; Clinical\nDementia Rating score = 1 in 11 patients and Clini-\ncal Dementia Rating score = 2 in 5 patients) fulfilling\nthe criteria of the National Institute of Neurological\nand Communicative Diseases and Stroke/Alzheimer’s\nDisease and Related Disorders Association (McKhann\net al., 1984) for probable AD and not fulfilling revised\nconsensus criteria for frontotemporal dementia (Neary\net al., 1998) were recruited through the Memory Clinic\nat the University of Freiburg. In one patient, a history of\ndelusions and cognitive fluctuation raised the possibility\nof comorbid Lewy body disease (McKeith et al., 1996).\nHis semantic memory test profile was within the range\n(mean ± 1.5 SD) of the other patients in the group.\nNeuropsychological evaluation revealed consistent im-\npairments of verbal and nonverbal delayed recall in all\npatients with additional naming or visuoconstructive\ndeficits in different subgroups of patients (see Table 1).\nMRI revealed changes of diffuse and symmetrical volume\nloss and bilateral hippocampal atrophy, together with\nminimal white matter lesions in some cases but no\nevidence of other pathology. Other causes of cognitive\ndecline had been excluded based on routine blood tests.\nNone of the patients had a history of stroke, other\nneurological disease, or major or recurrent psychiatric\nsymptomatology, including substance abuse, before on-\nset of dementia. All were native speakers of German.\nNone of the patients had received cholinesterase inhib-\nitors or cholinergic medication during the study; four\npatients were taking selective serotonin reuptake inhib-\nitors and two were taking an atypical neuroleptic drug\n(quetiapine).\nNormal Controls\nFourteen healthy right-handed volunteers (7 women;\nmean MMSE = 29.3, SD = 1.1) matched to the patient\ngroup for age and years of schooling ( p > .4, Mann–\nWhitney U test), without history of central nervous\nsystem or psychiatric disease, substance abuse, or psy-\nchotropic medication, provided control data for the\nexperimental semantic tasks described below. All per-\nformed at normal levels (z > 1) on the Consortium to\nEstablish a Registry for Alzheimer’s Disease (CERAD) test\nbattery (a comprehensive standardized neuropsycholog-\nical test battery for the diagnosis of dementia; German\nnorms from Berres, Monsch, Bernasconi, Thalmann, &\nStahelin, 2000). Because of local ethics regulations, no\nage-matched control group was available for the PET\nstudy, so data from a group of 12 healthy normal\nsubjects without psychiatric or neurological diseases\nor psychotropic medication (7 women; mean age =\n48 years, SD = 16 years) were used. The most important\naspect of the PET analysis (parametric regression),\nhowever, did not require control data.\nSemantic Verification Task\nA set of 192 simple propositions of the form ‘‘a zebra has\nstripes’’ or ‘‘an axe chops’’ were read aloud. Half of the\npropositions were true, and half were false, and subjects\nwere simply required to make a verbal ‘‘true-or-false’’\ndecision.\nThe properties used in this test were taken from\nresponses on a feature-generation task performed by a\nnonoverlapping group of 60 healthy volunteers (mean\nage = 27.1 years, SD = 10.9). For this task, words for\n120 concepts (60 living and 60 nonliving) were selected\nfrom Snodgrass and Vanderwart’s (1980) set. The words\nwere divided into two sublists of 60 words (30 living and\n30 nonliving), each of which was given to a subgroup of\n30 controls. Controls were asked first to perform a free\nforward word association task by writing down the first\nword that came to mind after reading each object name.\nThe percentage of subjects naming a word in response\nto a given object name yielded a measure of ‘‘forward\nassociativity’’. Subjects were then instructed to write\ndown between four and eight properties commonly\nattributed to each concept in turn. Each property was\nassigned a ‘‘dominance’’ value, equivalent to the pro-\nportion of subjects generating it in response to a word\n(Ashcraft, 1978). Feature distinctiveness was determined\nas the proportion of same-domain concepts for which\nthe feature was generated.\nProperties with dominance values of less than 15% were\ndiscarded, and 96 selected from the remainder such that\none visual and one functional property were associated\nwith each of 24 living and 24 nonliving concepts. Living\nand nonliving concept words and visual and functional\nfeature words were matched for spoken and written word\nfrequency and syllable length using the CELEX lexical\ndatabase (www.mpi.nl/world/celex/). Feature words were\nalso matched across domain and across type by domi-\nnance and distinctiveness. All concept-feature pairs had\nless than 15% associativity. The combination of feature\ndistinctiveness and feature dominance provides informa-\ntion comparable to the ‘‘feature relevance’’ measure of\nSartori and Lombardi (2004). An equal set of distracter\nstatements was formed by pseudorandom recombination\nof the selected properties with concepts to make a total\nof 192 statements in the task. Properties of living con-\ncepts were recombined with living concepts and nonliving\nproperties with nonliving concepts only.\nThe feature verification task was administered to 14\ncognitively normal age- and education-matched con-\ntrols. To equate the item difficulty across stimulus con-\nditions a subset of 128 statements with high response\nconsistency (>90% agreement) per item across nor-\nmal controls was selected. The properties of the final\n128 items used in the task are summarized in Table 2.\nSemantic Memory Test Battery\nSemantic memory integrity was documented using a\nGerman-language version of the semantic memory test\nbattery described by Hodges, Salmon, and Butters\n(1992). The battery, described in detail by Zahn et al.\n(2004), is based on a set of 48 line drawings and asso-\nciated concept names from Snodgrass and Vanderwart’s\n(1980) set. Half the items are drawn from the living and\nhalf from the nonliving domain, matched for proto-\ntypicality, familiarity, visual complexity, spoken frequen-\ncy, and syllable length. Subjects are presented with 48\nline drawings for naming and with 48 arrays containing a\ntarget and within-category foils, for spoken word-to-\npicture matching. Half of the items were presented\nwithin an array of 3, the other half within an array of\n6 items. Subjects were also asked to produce as many\nexemplars of the following categories in a fixed order for\n1 min each: land animals, clothing, vegetables, vehicles,\ninsects, kitchen items, fruits, and musical instruments.\nThe sum of correct, nonrepeated responses was re-\ncorded. The picture version of the Pyramids and Palm\nTrees (PPT) test (Howard & Patterson, 1992) was ad-\nministered as a nonlinguistic test of associative semantic\nknowledge.\nPositron Emission Tomography Imaging\nThe PET procedure was performed according to pre-\nviously defined standards ( Juengling, Kassubek, & Otte,\n2000). Patient and control participants were allowed to\nrest in an acoustically isolated and dimmed room for at\nleast 10 min before injection of 200 ± 20 MBq 18-FDG\nand for another 20 min during the uptake period.\nParticipants were then transferred to the scanner (Sie-\nmens, Erlangen, Germany, CTI ECAT exact tomograph,\n10.8 cm field of view, 6.8 mm full width half maximum),\nwith their heads positioned relative to the orbitomeatal\nline. Image acquisition began 30 min after injection. Six\ndynamic frames were acquired, each of 5-min duration.\nImages were reconstructed using filtered back-projec-\ntion (Shepp-Logan filter; cutoff = 0.35 cycles/pixel).\nAttenuation correction was performed using the stan-\ndard mathematical algorithm implemented in ECAT\nsoftware. The dynamic frames were then checked for\nmotion artifacts and summed to generate a single data\nset of 31 transaxial planes.\nFor analysis using Statistical Parametric Mapping (SPM2,\nimage data were converted to\nwww.fil.ion.ucl.ac.uk),\nANALYZE format, and automated spatial normalization\nwas performed to realign the data set to a common 3-D\nstereotaxic space using the Montreal Neurological Insti-\ntute template (Friston, Holmes, & Worsley, 1995). Be-\nfore voxel-based analysis, images were smoothed using\nan 18  18  18-mm Gaussian kernel to account for\ninterindividual anatomical variability. The global cerebral\nmetabolic rate for glucose (gCMRGlc) was normalized to\na mean of 50 Amol/100 mL/min. The normalized FDG-\nPET data of the patients were compared with healthy\nsubjects by computing pixel-by-pixel t statistics for detec-\ntion of hypometabolic areas (Signorini et al., 1999). For\nanatomical identification, the coordinates derived from\nthe Montreal Neurological Institute template were trans-\nformed using the appropriate algorithm (cf. www.mrc-\ncbu.cam.ac.uk/Imaging/mnispace.html) to comply with\nthe original grid by Talairach and Tournoux. All areas ex-\nceeding a voxel threshold level of z > 3.09 and p = .001,\nuncorrected, and a size of >15 voxels are reported in the\ngroup comparison.\nPercent correct scores on each condition of the se-\nmantic verification task were entered as covariates of\ninterest in the within-patient group parametric regres-\nsion analysis with the aim of identifying regions showing\ndecreased metabolism associated with low performance.\nFor covariance analysis, voxel clusters with t values\ncorresponding to p < .01 in a single test and a minimum\ncluster size of 15 voxels were displayed as projected on\nlateral views of a standard brain (Figure 3). Only regions\nwith a voxel significance level of p < .001, z > 3.09, and\na cluster size of >15 voxels at p = .01 are reported in\nTable 4 and discussed below.\nRESULTS\nNeuropsychology\nBackground Neuropsychology\nAs a group, the AD patients showed severe impairments\nof direct and delayed verbal as well as nonverbal re-\ncall and verbal recognition in the CERAD test battery\n(Table 1, z < 2 SD of age- and education-matched\nnorm population mean). Boston Naming Test and figure\ndrawing group means were in the less severely impaired\nrange (z < 1). Individual case analyses showed most\nconsistent impairments in memory and fluency subtests;\nBoston Naming Test and figure drawing were impaired\nin different subgroups of patients.\nSemantic Memory Test Results\nsurvived a more stringent Bonferroni-corrected thresh-\nold ( p = .003; equivalent to an approximate Bonferroni-\ncorrected threshold of p = .05). On an individual case\nbasis, more than two thirds of the AD patients were\nbelow the normal cutoff score (below normal mean\n 2 SD and below normal range) in the following tests:\nsemantic word fluency, picture naming, and semantic\nverification. Word–picture matching was above the nor-\nmal cutoff score in 13 of 16 patients, and nonverbal\nsemantic memory (PPT test) was below the cutoff in 6\nof 16 patients.\nSemantic Verification Task Results\nAs a group, patients performed at a lower level than\ncontrols in all task conditions (Table 3). The scores of\n13 of 16 patients fell below the normal cutoff in the\nvisual conditions; of 16 patients, 10 were below cutoff\nin the living functional conditions and 9 in the nonliv-\ning functional condition (Table 3). Normal subjects per-\nformed at ceiling level all conditions. Figure 1 shows\nthe comparative levels of accuracy (percentage of cor-\nrect responses) achieved by patients in the four condi-\ntions of the semantic verification task. A nonparametric\none-way analysis of variance over the four task condi-\ntions revealed no significant effect of task condition\non performance [x2(3)= 7.5, p = .06, ns, Friedman test,\nasymptotic].\nSide maxima for large clusters are in italics. NC = normal controls. All areas exceeding a voxel threshold level of p = .001, uncorrected, and a cluster\nsize of >15 voxels are reported (SPM2). One patient had scores below the patient group mean  2 SD on three conditions of the verification task\n(see Figure 1). To rule out the possibility that our regression analyses were disproportionately inf luenced by this extreme case we repeated the\nanalysis after excluding the outlier. All reported regions were confirmed in the reanalysis.\npatterns of performance on word picture matching or\nfluency in any of the patients.\nWithin-Alzheimer’s Disease Patient Group\nRegression Analyses\n18-Fluorodeoxyglucose Positron\nEmission Tomography\nAlzheimer’s Disease versus Normal Control\nGroup Comparison\nThe group comparison shows a typical pattern of hy-\npometabolism in AD patients relative to normal con-\ntrols in the bilateral dorsolateral prefrontal cortices\n(Brodmann’s area [BA] 8/9), bilateral inferior temporal\ngyri (BA 20), the left anterior superior temporal gyrus\n(BA 22), the right posterior superior temporal sulcus\n(BA 22/21), bilateral angular gyri (BA 39/7), and bilateral\nposterior cingulate gyri/precuneus (BA 31) (Figure 3,\nTable 4).\nThe main focus of this article is the result of the within-\nAD patient group regression analyses, independent of\nnormal controls. These analyses identify brain regions in\nthe AD group where glucose metabolism was positively\ncorrelated with performance in the respective condi-\ntion of the semantic verification task (living visual,\nliving functional, nonliving visual, nonliving functional).\nFigure 3 shows 3-D projections of these significantly\ncorrelated regions on a more lenient level of significance\n( p = .01) to give an impression of the extension of\nareas. We will only report and discuss regions in the text\nand Table 4, however, that achieved a more stringent\nlevel of significance ( p = .001, 15 voxels).\nPerformance on verification of the visual properties of\nliving things was significantly correlated with metabolism\nin the left posterior fusiform gyrus (BA 37/19), whereas\nperformance with both visual and functional properties\nof the nonliving concepts was significantly associated with\nmetabolism in left anterior temporal (BA 20/38) and bi-\nlateral precentral/premotor regions (BA 4/6). The right\nfrontal pole (BA 10) was associated with the nonliving vi-\nsual properties only. No area exceeded threshold for the\nliving functional condition. The comparison of the ex-\ntensions of significantly associated areas (Figure 3) shows\na more pronounced extension within the left lateral\nprecentral cortex for the nonliving visual condition and\na larger extension of premotor hypometabolism into the\nright hemisphere for the nonliving functional condition.\nDISCUSSION\nThe study reported here was designed to identify corre-\nlations between regions of cortical hypometabolism and\nperformance on a semantic feature verification task in\nwhich conceptual domain membership (living vs. non-\nliving) was crossed with type of knowledge probed\n(visual vs. functional). The principal motivation for the\nstudy was to test the contrasting predictions of a num-\nber of theoretical models of semantic organization, con-\ncerning the relationship between living and nonliving\nconcepts on the one hand, and visual and functional\nknowledge on the other. The study’s major findings\ncan be summarized as the association of (a) impaired\nknowledge of the visual properties of living concepts\nwith hypometabolism in the posterior fusiform gyrus\n(BA 37/19) and (b) impaired knowledge of the visual and\nfunctional properties of nonliving concepts with hypo-\nmetabolism in the left anterior temporal (BA 38/20) and\nmotor/premotor areas (BA 4/6). Both findings are in\nkeeping with a large body of previously published data\nlinking brain regions with semantic categories and dif-\nferent feature types.\nThe left posterior fusiform region has been linked to\nvisual property verification using functional MRI (Kan,\nBarsalou, Solomon, Minor, & Thompson-Schill, 2003),\nand to noun imageability using PET (Wise et al., 2000),\nalthough neither study contrasted living and nonliving\nconcepts. Thompson-Schill, Aguirre, Desposito, and Farah\n(1999) used a verbal property verification task to show\nactivation of the posterior fusiform gyrus (BA 37) for all\naspects of living things and for nonliving concepts when\nvisual property knowledge was explicitly required. As Price\net al. (1999) have pointed out, however, functional activa-\ntion in normal subjects is likely to include regions that are\ninvolved in but not necessary for task performance. Thus,\nalthough the posterior fusiform gyrus might be activated\nfor both living and nonliving concepts, the data reported\nhere would suggest that it may be necessary only for judg-\nments involving the visual properties of living things.\nThe association between knowledge of nonliving con-\nlobe accords with\ncepts and the anterior temporal\ndescriptions of patients with anterior temporal\nlobe\nresections who show selective naming impairments\nfor both human actions (Lu et al., 2002) and nonliving\nitems (Cappa, Frugoni, Pasquali, Perani, & Zorat, 1998;\nTippett, Glosser, & Farah, 1996). The importance of\nanterior temporal cortex for semantic memory is well\nestablished: Its supramodal role in semantic knowledge\ncan be inferred from the multimodal impairments char-\nacteristic of the syndrome of semantic dementia, which\nis associated with atrophy in anterior temporal regions\n(Davies et al., 2005; Rogers et al., 2004; Hodges, Bozeat,\nLambon Ralph, Patterson, & Spatt, 2000). Semantic\nknowledge impairments in semantic dementia also im-\npact on tool use, emphasizing the importance of the\nanterior temporal lobe for the representation of func-\ntional information (Hodges et al., 2000).\nThe importance of premotor areas for nonliving con-\ncepts accords with evidence from functional activation\nstudies in normal subjects pointing to the involvement of\nleft premotor cortex (BA 6) in the representation of tool\nand action knowledge (Martin & Chao, 2001) as well as\naction perception (Rizzolatti, Fogassi, & Gallese, 2002).\nAlso, primary motor cortex (BA 4) is activated during\nsemantic comprehension of verbs describing movement\nrelated actions (Hauk, Johnsrude, & Pulvermuller, 2004).\nFinally, the dependence of visual knowledge for nonliving\nconcepts on motor rather than visual areas is consistent\nwith the idea that imagery of motor acts is involved in\nretrieving visual information about artifacts (review of\n‘‘motor imagery’’ by Jeannerod & Frak, 1999).\nThe implications of these anatomical patterns for rival\ntheories of category specificity in AD are twofold. In the\nfirst place, it is clear that impairment of semantic knowl-\nedge for living and nonliving objects in AD patients\nresulted from regional dysfunction in anatomically sepa-\nrable brain areas. Second, higher order visual information\nin the left posterior fusiform region may relate specifi-\ncally to living concepts. Neither of these findings can be\naccounted for by theories based on the assumption of a\nwidely distributed supramodal semantic memory store.\nBy contrast, the predictions of Garrard et al. (1998,\n2001), based on the patterns of performance in tasks\nassessing semantic knowledge for living and nonliving\nconcepts in AD, were, at least partially, corroborated.\nAccording to this account, dissociations of semantic\nknowledge for living and nonliving concepts should be\nbecause of differences in the regional distribution of\npathology in AD. Temporal neocortical regions, which\nare most subject to degenerative damage in AD, are\ndifferentially important in the representation of living\nconcepts and give rise to the more typical nonliving ad-\nvantage in the earlier disease stages. Without in vivo ana-\ntomical support for a concomitant sensory–functional\ndifference, however, Garrard et al. conceded that the\npatterns of performance could also be explained by\nassuming ‘‘evolutionarily adapted domain-specific knowl-\nedge systems subserved by distinct neural mechanisms’’\n(Caramazza & Shelton, 1998).\nThe importance of inferior temporal dysfunction in\nthe impairment of living concept knowledge in AD is\nsupported here by the finding of such an association\nwithin the posterior fusiform gyrus (BA 37/19). It is also\nin accordance with the literature on category-specific\ndeficits for living things due to other causes (e.g., her-\npes simplex encephalitis, closed head injury) showing\nlobe involvement most frequently\ninferior temporal\n(Gainotti, 2000). The finding of a specific association\nof visual\nfeatures alone with left temporal changes\nsuggests in addition a link between visual feature knowl-\nedge and the impairments of living concept knowledge\nin AD, as predicted by the sensory–functional, but not\nthe domain-specific, theory. Our second finding of an\nassociation of nonliving but not living concept knowl-\nedge with anterior temporal and premotor cortical\nmetabolism, however, is more difficult to reconcile with\nthe sensory–functional theory. In this instance, the\noverlap of regions associated with visual and functional\nfeature knowledge of nonliving objects may be better\nexplained by the domain-specific hypothesis.\nOur seemingly contradictory findings with regard to\nthese two competing theoretical accounts of anatomically\nseparable subdivisions for different types of semantic\nknowledge can be best explained within a more recent\nformulation of the domain-specific account (Caramazza\n& Mahon, 2003). This hybrid model admits of the co-\nexistence of supramodal domain-specific regions for ani-\nmals, fruits/vegetables, and tools and additional unimodal\nfeature-specific semantic regions. Indeed, a growing body\nof evidence from the cognitive neuropsychological liter-\nature suggests that impairments of living and nonliving\nobject knowledge can occur in the absence of more\npronounced effects of either visual or functional fea-\ntures (Caramazza & Mahon, 2003; Laiacona, Capitani, &\nCaramazza, 2003; Zannino, Perri, Carlesimo, Pasqualetti,\n& Caltagirone, 2002; Lambon-Ralph, 1998). Moreover,\nvisual feature knowledge can be impaired in patients\nwithout leading to category-specific effects for living\nconcepts (Caramazza & Mahon, 2003; Lambon-Ralph,\n1998)—a pattern that, we note in passing, was exhibited\nby one of the patients in the present study. Our data thus\naccord with the heterogeneity of category-specific se-\nmantic disorders reported in the literature (Rosazza\net al., 2003).\nThe finding of regional dysfunction within the pre-\nmotor and anterior temporal cortex is in agreement with\npostmortem neurochemical studies that detected vari-\nable degrees of tau pathology in the anterior temporal\nareas (BA 20/38) and synaptic loss in the premotor area\n(BA 6) even at predementia stages of AD (Vermersch\net al., 1995; Clinton, Blackman, Royston, & Roberts,\n1994). Previous studies in AD patients using regression\nanalyses and global measures of semantic functions\n(naming and/or nonverbal impairment) have also shown\nassociations with premotor (Grossman et al., 2004)\nand anterior temporal (Zahn et al., 2004) areas using\nvoxel-based morphometry and FDG-PET, respectively.\nFunctional disconnection (diaschisis) of the anterior\ntemporal and motor/premotor regions caused by neuro-\ndegeneration within the posterior temporal or parietal\nassociation cortices also provide a plausible pathophys-\niological mechanism. The posterior temporal and parie-\ntal association cortex are highly connected to these\nregions (Creutzfeldt, 1983) and are severely and con-\nsistently affected by tangle neuropathology (Harasty,\nHalliday, Kril, & Code, 1999; Braak & Braak, 1991).\nOur negative finding with regard to significant associ-\nations for functional knowledge of living objects with\nany brain area points to greater variability in the associ-\nation of regional metabolism and task performance than\nin the other task conditions. Given the robust correla-\ntions of specific regional metabolism with performance\nin the other conditions, this negative finding is unlikely\nto be due simply to a lack of statistical power. An alter-\nnative explanation is that the knowledge required for\nthe task was represented across multiple regions. This\nwould be consistent with the heterogeneity characteristic\nof functional features of living objects, which may de-\nscribe their use by man or animal, characteristic motion,\nenvironmental context, or general ‘‘encyclopedic knowl-\nedge,’’ to name but a few. Thus, the a priori probability of\nlinking this diverse group of features to a specific brain\narea might be expected to be lower. Future studies using\ntasks with more differentiated subclassifications of fea-\ntures as proposed by, for example, Cree and McRae\n(2003) may well reveal specific brain regions linked to\nsubclasses of functional knowledge for living things.\nOur findings of regional hypometabolism that is asso-\nciated with semantic impairment in AD has to be inter-\npreted in conjunction with evidence from other sources\nto rule out a functional age-related origin of these ab-\nnormalities. Neuroimaging studies and neuropathology\nshowed abnormalities in AD patients compared with aged\nhealthy controls in those regions that were associated\nwith semantic memory dysfunction in our study (Zahn\net al., 2005; Grossman et al., 2004; Herholz et al., 2002;\nHarasty et al., 1999; Clinton et al., 1994; Braak & Braak,\n1991). PET studies showed age effects primarily outside of\ntemporal and premotor regions relevant to performance\nin our task (Herholz et al., 2002). Furthermore, our age-\nmatched control group performed almost perfectly on\nthe semantic task. Thus, normal aging does not markedly\naffect explicit semantic memory and associated brain\nregions. In summary, this supports the conclusion that\nregional effects demonstrated in our AD group did not\narise by functional age-related changes but by disease-\nrelated pathology. On a more cautious note, the robust-\nness of our findings would undoubtedly benefit from\nfurther studies replicating the regional associations seen\nin the present sample and thus confirming consistent\nregional effects across different AD groups.\nIn conclusion, therefore, the findings of the present\nstudy support the view that semantic organization is\ncharacterized both by category-specific supramodal and\nunimodal feature-specific regions. It should be noted\nthat these data neither demonstrate nor rule out the\npresence of category-specific subdivisions within the\nsupramodal anterior temporal area, where visual and\nfunctional feature knowledge for nonliving things over-\nlap. The findings would therefore be equally consistent\nwith theories positing supramodal semantic stores with-\nout category-specific subdivisions for living and non-\nliving concepts that allow for additional feature-specific\nregions with category-specific subdivisions. To the best\nof our knowledge, however, no single cases have yet\nbeen described that would be consistent with either of\nthese interpretations.",
         "",
         "Patterns of Regional Brain Hypometabolism Associated with Knowledge of Semantic Features and Categories in Alzheimer's Disease",
         "None"
        ]
       ],
       "shape": {
        "columns": 15,
        "rows": 76650
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>abstract</th>\n",
       "      <th>authorName</th>\n",
       "      <th>author_list</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>pdf_abstract</th>\n",
       "      <th>pdf_discussion</th>\n",
       "      <th>pdf_introduction</th>\n",
       "      <th>pdf_methods</th>\n",
       "      <th>pdf_results</th>\n",
       "      <th>pdf_status</th>\n",
       "      <th>pdf_text</th>\n",
       "      <th>referenceTitles</th>\n",
       "      <th>title</th>\n",
       "      <th>valid_pages</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>00009f98f5ff962beb322e6c0e78e4b77dd2d968</td>\n",
       "      <td>Tungiasis is a parasitic skin disease widespre...</td>\n",
       "      <td>H. Feldmeier,J. D. Kehr,G. Poggensee,J. Heukel...</td>\n",
       "      <td>H. Feldmeier,J. D. Kehr,G. Poggensee,J. Heukel...</td>\n",
       "      <td>https://www.scielo.br/j/mioc/a/csnpsRLq5FqY5Mk...</td>\n",
       "      <td>Department of Medical Microbiology and Immunol...</td>\n",
       "      <td>DISCUSSION\\nTungiasis occurs in Central and So...</td>\n",
       "      <td>Tungiasis is a neglected parasitic skin diseas...</td>\n",
       "      <td>MATERIALS  AND  METHODS\\nThe  study  was  cond...</td>\n",
       "      <td>RESULTS\\nOf the 77 subjects included in the st...</td>\n",
       "      <td>downloading</td>\n",
       "      <td>Department of Medical Microbiology and Immunol...</td>\n",
       "      <td></td>\n",
       "      <td>High exposure to Tunga penetrans (Linnaeus, 17...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0000e1833852eb3271356c493d59e83da0c4c7db</td>\n",
       "      <td>With the burgeoning immunological data in the ...</td>\n",
       "      <td>B. Korber,M. LaBute,K. Yusim</td>\n",
       "      <td>B. Korber,M. LaBute,K. Yusim</td>\n",
       "      <td>https://journals.plos.org/ploscompbiol/article...</td>\n",
       "      <td>scientiﬁc literature, scientists must increasi...</td>\n",
       "      <td>Discussion\\nThis review is intended as a porta...</td>\n",
       "      <td>Introduction\\nThe adaptive immune response. Th...</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>downloading</td>\n",
       "      <td>scientiﬁc literature, scientists must increasi...</td>\n",
       "      <td>Prediction of the secondary structure of prote...</td>\n",
       "      <td>Immunoinformatics Comes of Age</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>00016363b9a128e01a17e9b831249967a0fee15f</td>\n",
       "      <td>14-3-3 proteins modulate the plant inward rect...</td>\n",
       "      <td>B. Sottocornola,S. Visconti,Sara A. Orsi,S. Ga...</td>\n",
       "      <td>B. Sottocornola,S. Visconti,Sara A. Orsi,S. Ga...</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820681212/pdf</td>\n",
       "      <td>14-3-3 proteins modulate the plant inward rect...</td>\n",
       "      <td>RESULTS AND DISCUSSION\\nRecombinant 14-3-3 Pro...</td>\n",
       "      <td>Voltage-dependent (Kv) potassium channels medi...</td>\n",
       "      <td>EXPERIMENTAL PROCEDURES\\nExpression of KAT1 in...</td>\n",
       "      <td>EXPERIMENTAL PROCEDURES\\nExpression of KAT1 in...</td>\n",
       "      <td>downloading</td>\n",
       "      <td>14-3-3 proteins modulate the plant inward rect...</td>\n",
       "      <td></td>\n",
       "      <td>The Potassium Channel KAT1 Is Activated by Pla...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>00034b38a5273b760824f3223fa0d4fdb349d6df</td>\n",
       "      <td>The natively disordered N-terminal 83-aa trans...</td>\n",
       "      <td>S. Loftus,D. Walker,M. Maté,D. Bonsor,R. James...</td>\n",
       "      <td>S. Loftus,D. Walker,M. Maté,D. Bonsor,R. James...</td>\n",
       "      <td>https://www.pnas.org/doi/pdf/10.1073/pnas.0603...</td>\n",
       "      <td>The natively disordered N-terminal 83-aa trans...</td>\n",
       "      <td></td>\n",
       "      <td>proteins contain stretches of 30 aa that are n...</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>downloading</td>\n",
       "      <td>The natively disordered N-terminal 83-aa trans...</td>\n",
       "      <td></td>\n",
       "      <td>Competitive recruitment of the periplasmic tra...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00036a3493967932ac256f38c0c2419415f04feb</td>\n",
       "      <td>Cyclophosphamide (CY) is a cytostatic agent th...</td>\n",
       "      <td>M. K. C. Hui,W. Wu,V. Shin,W. H. So,C. Cho</td>\n",
       "      <td>M. K. C. Hui,W. Wu,V. Shin,W. H. So,C. Cho</td>\n",
       "      <td>https://www.medsci.org/v03p0001.pdf</td>\n",
       "      <td>The  major  side  effect  of  anticancer  drug...</td>\n",
       "      <td>this  study,  CY  produced  myelosuppression\\n...</td>\n",
       "      <td>residue was boiled again with water for anothe...</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>downloading</td>\n",
       "      <td>The  major  side  effect  of  anticancer  drug...</td>\n",
       "      <td></td>\n",
       "      <td>Polysaccharides from the root of Angelica sine...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76645</th>\n",
       "      <td>fffabc38e25edd5a95b48e2952322542317ea392</td>\n",
       "      <td>None</td>\n",
       "      <td>A. Marconi,M. Pistore,P. Traverso</td>\n",
       "      <td>None</td>\n",
       "      <td>https://link.springer.com/content/pdf/10.1007%...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Specifying Data-Flow Requirements for the Auto...</td>\n",
       "      <td>Implicit vs. Explicit Data-Flow Requirements i...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76646</th>\n",
       "      <td>fffae8f127ca823fecd1228e5131aa06ec98696a</td>\n",
       "      <td>ABSTRACT Enterococcus faecalis isolates of por...</td>\n",
       "      <td>N. Shankar,Arto S. Baghdayan,R. Willems,A. Ham...</td>\n",
       "      <td>None</td>\n",
       "      <td>https://europepmc.org/articles/pmc1698369?pdf=...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Multilocus Sequence Typing Scheme for Enteroco...</td>\n",
       "      <td>Presence of Pathogenicity Island Genes in Ente...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76647</th>\n",
       "      <td>fffe4813ee529996e47ec6c4f1732f0030d334c9</td>\n",
       "      <td>The accessory radula closer (ARC) muscle of Ap...</td>\n",
       "      <td>Yuriy Zhurov,V. Brezina</td>\n",
       "      <td>None</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/26/26...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Tight or loose coupling between components of ...</td>\n",
       "      <td>Variability of Motor Neuron Spike Timing Maint...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76648</th>\n",
       "      <td>ffff1780c2f0bb4d8ea115fdd820a518a1b33dd6</td>\n",
       "      <td>Chemical reaction systems operating in nonequi...</td>\n",
       "      <td>D. Beard,H. Qian</td>\n",
       "      <td>None</td>\n",
       "      <td>https://journals.plos.org/plosone/article/file...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Generalized Haldane equation and fluctuation t...</td>\n",
       "      <td>Relationship between Thermodynamic Driving For...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76649</th>\n",
       "      <td>ffffb1b949fe54eb8dda3238df0058b373ec3408</td>\n",
       "      <td>ABSTRACT A novel mutant of the marine oil-degr...</td>\n",
       "      <td>J. Sabirova,M. Ferrer,H. Lünsdorf,V. Wray,R. ...</td>\n",
       "      <td>None</td>\n",
       "      <td>https://europepmc.org/articles/pmc1698222?pdf=...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Genome sequence of the ubiquitous hydrocarbon-...</td>\n",
       "      <td>Mutation in a “tesB-Like” Hydroxyacyl-Coenzyme...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>76650 rows × 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paperId  \\\n",
       "0      00009f98f5ff962beb322e6c0e78e4b77dd2d968   \n",
       "1      0000e1833852eb3271356c493d59e83da0c4c7db   \n",
       "2      00016363b9a128e01a17e9b831249967a0fee15f   \n",
       "3      00034b38a5273b760824f3223fa0d4fdb349d6df   \n",
       "4      00036a3493967932ac256f38c0c2419415f04feb   \n",
       "...                                         ...   \n",
       "76645  fffabc38e25edd5a95b48e2952322542317ea392   \n",
       "76646  fffae8f127ca823fecd1228e5131aa06ec98696a   \n",
       "76647  fffe4813ee529996e47ec6c4f1732f0030d334c9   \n",
       "76648  ffff1780c2f0bb4d8ea115fdd820a518a1b33dd6   \n",
       "76649  ffffb1b949fe54eb8dda3238df0058b373ec3408   \n",
       "\n",
       "                                                abstract  \\\n",
       "0      Tungiasis is a parasitic skin disease widespre...   \n",
       "1      With the burgeoning immunological data in the ...   \n",
       "2      14-3-3 proteins modulate the plant inward rect...   \n",
       "3      The natively disordered N-terminal 83-aa trans...   \n",
       "4      Cyclophosphamide (CY) is a cytostatic agent th...   \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646  ABSTRACT Enterococcus faecalis isolates of por...   \n",
       "76647  The accessory radula closer (ARC) muscle of Ap...   \n",
       "76648  Chemical reaction systems operating in nonequi...   \n",
       "76649  ABSTRACT A novel mutant of the marine oil-degr...   \n",
       "\n",
       "                                              authorName  \\\n",
       "0      H. Feldmeier,J. D. Kehr,G. Poggensee,J. Heukel...   \n",
       "1                           B. Korber,M. LaBute,K. Yusim   \n",
       "2      B. Sottocornola,S. Visconti,Sara A. Orsi,S. Ga...   \n",
       "3      S. Loftus,D. Walker,M. Maté,D. Bonsor,R. James...   \n",
       "4             M. K. C. Hui,W. Wu,V. Shin,W. H. So,C. Cho   \n",
       "...                                                  ...   \n",
       "76645                  A. Marconi,M. Pistore,P. Traverso   \n",
       "76646  N. Shankar,Arto S. Baghdayan,R. Willems,A. Ham...   \n",
       "76647                            Yuriy Zhurov,V. Brezina   \n",
       "76648                                   D. Beard,H. Qian   \n",
       "76649  J. Sabirova,M. Ferrer,H. Lünsdorf,V. Wray,R. ...   \n",
       "\n",
       "                                             author_list  \\\n",
       "0      H. Feldmeier,J. D. Kehr,G. Poggensee,J. Heukel...   \n",
       "1                           B. Korber,M. LaBute,K. Yusim   \n",
       "2      B. Sottocornola,S. Visconti,Sara A. Orsi,S. Ga...   \n",
       "3      S. Loftus,D. Walker,M. Maté,D. Bonsor,R. James...   \n",
       "4             M. K. C. Hui,W. Wu,V. Shin,W. H. So,C. Cho   \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646                                               None   \n",
       "76647                                               None   \n",
       "76648                                               None   \n",
       "76649                                               None   \n",
       "\n",
       "                                           openAccessPdf  \\\n",
       "0      https://www.scielo.br/j/mioc/a/csnpsRLq5FqY5Mk...   \n",
       "1      https://journals.plos.org/ploscompbiol/article...   \n",
       "2       http://www.jbc.org/article/S0021925820681212/pdf   \n",
       "3      https://www.pnas.org/doi/pdf/10.1073/pnas.0603...   \n",
       "4                    https://www.medsci.org/v03p0001.pdf   \n",
       "...                                                  ...   \n",
       "76645  https://link.springer.com/content/pdf/10.1007%...   \n",
       "76646  https://europepmc.org/articles/pmc1698369?pdf=...   \n",
       "76647  https://www.jneurosci.org/content/jneuro/26/26...   \n",
       "76648  https://journals.plos.org/plosone/article/file...   \n",
       "76649  https://europepmc.org/articles/pmc1698222?pdf=...   \n",
       "\n",
       "                                            pdf_abstract  \\\n",
       "0      Department of Medical Microbiology and Immunol...   \n",
       "1      scientiﬁc literature, scientists must increasi...   \n",
       "2      14-3-3 proteins modulate the plant inward rect...   \n",
       "3      The natively disordered N-terminal 83-aa trans...   \n",
       "4      The  major  side  effect  of  anticancer  drug...   \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646                                               None   \n",
       "76647                                               None   \n",
       "76648                                               None   \n",
       "76649                                               None   \n",
       "\n",
       "                                          pdf_discussion  \\\n",
       "0      DISCUSSION\\nTungiasis occurs in Central and So...   \n",
       "1      Discussion\\nThis review is intended as a porta...   \n",
       "2      RESULTS AND DISCUSSION\\nRecombinant 14-3-3 Pro...   \n",
       "3                                                          \n",
       "4      this  study,  CY  produced  myelosuppression\\n...   \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646                                               None   \n",
       "76647                                               None   \n",
       "76648                                               None   \n",
       "76649                                               None   \n",
       "\n",
       "                                        pdf_introduction  \\\n",
       "0      Tungiasis is a neglected parasitic skin diseas...   \n",
       "1      Introduction\\nThe adaptive immune response. Th...   \n",
       "2      Voltage-dependent (Kv) potassium channels medi...   \n",
       "3      proteins contain stretches of 30 aa that are n...   \n",
       "4      residue was boiled again with water for anothe...   \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646                                               None   \n",
       "76647                                               None   \n",
       "76648                                               None   \n",
       "76649                                               None   \n",
       "\n",
       "                                             pdf_methods  \\\n",
       "0      MATERIALS  AND  METHODS\\nThe  study  was  cond...   \n",
       "1                                                          \n",
       "2      EXPERIMENTAL PROCEDURES\\nExpression of KAT1 in...   \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646                                               None   \n",
       "76647                                               None   \n",
       "76648                                               None   \n",
       "76649                                               None   \n",
       "\n",
       "                                             pdf_results   pdf_status  \\\n",
       "0      RESULTS\\nOf the 77 subjects included in the st...  downloading   \n",
       "1                                                         downloading   \n",
       "2      EXPERIMENTAL PROCEDURES\\nExpression of KAT1 in...  downloading   \n",
       "3                                                         downloading   \n",
       "4                                                         downloading   \n",
       "...                                                  ...          ...   \n",
       "76645                                               None         None   \n",
       "76646                                               None         None   \n",
       "76647                                               None         None   \n",
       "76648                                               None         None   \n",
       "76649                                               None         None   \n",
       "\n",
       "                                                pdf_text  \\\n",
       "0      Department of Medical Microbiology and Immunol...   \n",
       "1      scientiﬁc literature, scientists must increasi...   \n",
       "2      14-3-3 proteins modulate the plant inward rect...   \n",
       "3      The natively disordered N-terminal 83-aa trans...   \n",
       "4      The  major  side  effect  of  anticancer  drug...   \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646                                               None   \n",
       "76647                                               None   \n",
       "76648                                               None   \n",
       "76649                                               None   \n",
       "\n",
       "                                         referenceTitles  \\\n",
       "0                                                          \n",
       "1      Prediction of the secondary structure of prote...   \n",
       "2                                                          \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "76645  Specifying Data-Flow Requirements for the Auto...   \n",
       "76646  Multilocus Sequence Typing Scheme for Enteroco...   \n",
       "76647  Tight or loose coupling between components of ...   \n",
       "76648  Generalized Haldane equation and fluctuation t...   \n",
       "76649  Genome sequence of the ubiquitous hydrocarbon-...   \n",
       "\n",
       "                                                   title valid_pages  \n",
       "0      High exposure to Tunga penetrans (Linnaeus, 17...        None  \n",
       "1                         Immunoinformatics Comes of Age        None  \n",
       "2      The Potassium Channel KAT1 Is Activated by Pla...        None  \n",
       "3      Competitive recruitment of the periplasmic tra...        None  \n",
       "4      Polysaccharides from the root of Angelica sine...        None  \n",
       "...                                                  ...         ...  \n",
       "76645  Implicit vs. Explicit Data-Flow Requirements i...        None  \n",
       "76646  Presence of Pathogenicity Island Genes in Ente...        None  \n",
       "76647  Variability of Motor Neuron Spike Timing Maint...        None  \n",
       "76648  Relationship between Thermodynamic Driving For...        None  \n",
       "76649  Mutation in a “tesB-Like” Hydroxyacyl-Coenzyme...        None  \n",
       "\n",
       "[76650 rows x 15 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "textdata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "from pathlib import Path\n",
    "import pickle\n",
    "from science_parse_api.api import parse_pdf\n",
    "\n",
    "import sys\n",
    "sys.path.append(\"..\")\n",
    "sys.path.append(\"../src\")\n",
    "\n",
    "#Import config file. Update config.py according to your environment\n",
    "from config import DATA_PATH, DRIVE_DATA_PATH\n",
    "\n",
    "from dataset.utils import close_all_chrome_sessions\n",
    "from dataset import load_dataset, push_data_to_drive, extract_pdf_sections, get_valid_pages\n",
    "\n",
    "\n",
    "import os\n",
    "import glob\n",
    "import shutil\n",
    "import subprocess\n",
    "\n",
    "\n",
    "\n",
    "import pandas as pd\n",
    "from scipy import stats\n",
    "import math\n",
    "\n",
    "import re\n",
    "\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.chrome.service import Service\n",
    "import atexit\n",
    "\n",
    "from pdfminer.high_level import extract_pages\n",
    "from pdfminer.layout import LTTextContainer, LTChar, LTTextLine, LTTextBox\n",
    "\n",
    "from tqdm import tqdm\n",
    "import time\n",
    "from datetime import datetime\n",
    "import multiprocessing\n",
    "import operator\n",
    "import tempfile\n",
    "\n",
    "\n",
    "\n",
    "from pdfminer.pdfpage import PDFPage\n",
    "import logging\n",
    "pdfminer_logger = logging.getLogger('pdfminer')\n",
    "pdfminer_logger.setLevel(logging.ERROR)  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "fieldsOfStudy",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "venue",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorName",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "citationCount",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "influentialCitationCount",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "publicationDate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "publicationYear",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "referenceIds",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "672bdce7-e483-4759-9730-2c8da6473a4a",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "https://aacrjournals.org/clincancerres/article-pdf/11/7/2702/1962059/2702-2712.pdf",
         "Biology,Medicine",
         "Clinical Cancer Research",
         "T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. Nishijo,K. Shibata,Y. Shima,S. Nagayama,T. Katagiri,Yusuke Nakamura,Takashi Nakamura,J. Toguchida",
         "14839966,10239302,2000448727,3201672,7147569,2066435363,31907929,4360342,7367122,144863658,145103616,3630533",
         "88",
         "6.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "http://www.merl.com/publications/docs/TR2005-063.pdf",
         "Computer Science,Mathematics",
         "IEEE Transactions on Communications",
         "Juntan Zhang,M. Fossorier",
         "3193350,48891339",
         "319",
         "26.0",
         "2005-03-07",
         "2005",
         "17d0afe4f814e5abc6252e572e3b039082bf6f07;02464a444975fdfbef09c40e9e6d140b2f10d213;5adb4ba3b3717d86d507e57ec7b04ca91b0daa32;3870f98d6575420d9e3d02a5f299fa483123caaa;610cd4607f39edb48ce8d8dda982b04cb9cb1d75;4f773884c351152c49b1111d98c07446b1543dc2;e28fc01c3f8ca4151abf9b61296b56dc318d3b26;fbf85b75d9d3d8c224de034005d2f2cbc7c4fac2;64e7badd52b7841ebc25d09e6b38e6d7dbea5f38;293620136b122dbfb258fe95fd45ca85c814236c;0bf5d709b14dd074522d96d2ffb4474b3184b984;444e24fd9dff66dfafeb70914b6b3c83e9003853",
         "Low-Density Parity-Check Codes;Turbo decoder architectures for low-density parity-check codes;Shuffled belief propagation decoding;A parallel decoder for low latency decoding of turbo product codes;Parallel versus sequential updating for belief propagation decoding;High throughput low-density parity-check decoder architectures;Good error-correcting codes based on very sparse matrices;Near optimum error correcting coding and decoding: turbo-codes;Multiple turbo codes for deep-space communications;Optimal decoding of linear codes for minimizing symbol error rate (Corresp.);Generic Architecture for LDPC Codes Decoding;Optimal decoding of linear codes for minimizing symbol error rate"
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "https://www.jstage.jst.go.jp/article/bpb/28/8/28_8_1450/_pdf",
         "Medicine",
         "Biological and Pharmaceutical Bulletin",
         "Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,Jian Sun,Q. Xia,K. Bian",
         "11809650,6617669,123635715,11305673,2152146096,144130337,2144753879",
         "24",
         "2.0",
         "2005-08-01",
         "2005",
         "",
         ""
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/dvdy.20220",
         "Biology,Medicine",
         "Developmental Dynamics",
         "Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V. Sukhatme",
         "35690923,2114125286,2113252574,2134773,3658233",
         "53",
         "2.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "http://www.jbc.org/article/S0021925820589394/pdf",
         "Chemistry,Medicine",
         "Journal of Biological Chemistry",
         "M. Westfall,A. M. Lee,D. Robinson",
         "5025983,49249911,48706510",
         "70",
         "3.0",
         "2005-12-16",
         "2005",
         "",
         ""
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "http://dl.acm.org/ft_gateway.cfm?id=1047480&type=pdf",
         "Computer Science",
         "Technical Symposium on Computer Science Education",
         "Susan Bergin,R. Reilly",
         "145985158,2688685",
         "287",
         "26.0",
         "2005-02-23",
         "2005",
         "b8123b4807efdee5d3a408e6f862ff438c642127;afae3da8f9b6e9b644a5dc695b50796e020d4194;cbaddfc496cebf6dc683ed8aad2306a359aa58ce;839b94477a293d0bd10acd2411589fd0cad2512c;259ec7be9d7bbb5b633978c72cf2a93bd86b25f9;78033d2947f98bf7189c5522c78251d4f9bf2eec;499da8c546288d9b12dd5acddc6d1cba3c9a5f56;5453190a0090efe599f136df5a8a2a529ad6ac9a;978a0c742ebfcbb41b9b9b472519512d9759accd;4ea72d8b5f946427398474c32f4bcff74b5f871b;ef3cf9f8bf4aa787afbdb0eaec91b1cc808a0898;3a6651c691c57e1c5badd03bb9bcb2b565832159;500a21438777d026ea95489c5a77d1cb1ac5d5a8;dbae45bc40eeeac237f360c5006f40611360b856;0ae8f4656278b6c683984cd60f3c3a0606da5bc1;f60e8ea65110900e1f4c3816bc720591fcc41f79;fdc5a2d599b5f9c4990f3ab4dc3ce86ae8251627",
         "The impact of prior experience in an information technology programming course sequence;Predictors of success and failure in a CS1 course;A multi-national, multi-institutional study of assessment of programming skills of first-year CS students;The effect of student attributes on success in programming;Contributing to success in an introductory computer science course: a study of twelve factors;Does it help to have some programming experience before beginning a computing degree program?;Factors affecting performance in first-year computing;The effect of a constructivist learning environment for field-dependent/independent students on achievement in introductory computer programming;What best predicts computer proficiency?;Predicting student performance in a beginning computer science class;Predicting the success of freshmen in a computer science major;Predicting student success in an introductory programming course;Predicting success in a first programming course;Investigating the relationship between the development of abstract reasoning and performance in an introductory programming class;Factors affecting course outcomes in introductory programming;An Overview of the Integration of Problem Based Learning into an existing Computer Science Programming Module;Discovering the likely performance of novice programmers on an introductory programming module;Differential Performance by gender and achievement in the Certificate Examination Results 2000/2001: Final Report, Dublin: National Council for Curriculum and Assessment;Examining Gender;Cognitive skills needed in computer programming;speciﬁc cognitive skills: as measured by an in-house cognitive test;previous academic and computer experience: as measured by performance in the Irish Leaving Certiﬁcate (LC) examinations in"
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.ATV.0000168413.29874.d7",
         "Medicine,Biology",
         "Arteriosclerosis, Thrombosis and Vascular Biology",
         "E. Egorina,M. Sovershaev,G. Bjørkøy,F. Gruber,J. Olsen,B. Parhami-Seren,K. Mann,B. Østerud",
         "8887769,4962786,11414707,50148632,33502527,1398546666,3162218,5466077",
         "148",
         "2.0",
         "2005-07-01",
         "2005",
         "65eeaee9e552393e75a743c0344ebe4581a6e6a7;8bf514513122d0af4d0aa2b9789994b4211bd7cd;05d6048fa796174028c92893168a0773e927e454;8be482509b14152ffdb18fe3ae5dc7d1afb009a5;c847dd5549f0c843199c1a50718d0f139aaefcd7;ac3cc136968e89551fc3206b23c3bfb1cc150bbd;4498654bf9d23ded98cee2eb667944f9c27da787;832d3e34e1d44cc5c92702ff3e7e97fa6d43ee44;154f2466119853b7ba397eee95c14f010572275d;41f73ad3c17b7c021670dda4e2e1e0e948d5f90c;e6a10b6e88a706227967bae80080553b1e2b51f5;8527e6d03a0f79a038fc130a4344fa4853c8335a;49e92bb9ad78b8b307e99944834e6d1bc631297f;be3f99f27c1ad7a8cf0a8eca7e8b549ef1b6a6dd;4511c51ef9c78ef929a5c9766af97db701da82af;4ca60e84496caca48756bb5b4f6f24236ca17c8f;d24472339b51de01a39c49b3b201c8e43b16f242;01bdc51fe1630bc406cf0661f7ac67200b449966;0be3116d5897256dd59fef2dcf2ba5401359a10d;0db9a7f606d2539f5e1bd2587ba2c0cb7166d66d;2b4563c2e99ab1208d889c22ad7df414c183dd3e;886b5036def2f1c34f63a621738e9ebfe40332ce;18f594d1e008e6bc4e0eaaf7e771acd7f9b6c6ff;ed3087c11d9f60b01ce54b720df2a2420749e77c;516f18c8244c665e450172bbe183c4286bbaa1d4;25405d6b24240aa2adb1f7b7049b80efeb40ce3e;878ce5ee726c6651d087291f37cc046a9f980d54;4e7a537a06f4b75c72bfb551c9a18f99d9679cef;551e55506bd0f8cfc13f22816c9b20315a51970c;48cc95821fe443d86a1b63d0dbd1486a0b012ce1;e3acf2654d08a2006fd32dce370e4fa4e5653b22",
         "The production and availability of tissue thromboplastin in cellular populations of whole blood exposed to various concentrations of endotoxin. An assay for detection of endotoxin.;Tissue factor activity in whole blood.;Nicotinamide inhibits endotoxin‐induced monocyte tissue factor expression;Cellular activation responses in blood in relation to lipid pattern: healthy men and women in families with myocardial infarction or cancer;Tissue factor activity in human monocytes is regulated by plasma: implications for the high and low responder phenomenon;Role of Platelet P-Selectin and CD40 Ligand in the Induction of Monocytic Tissue Factor Expression;Eicosanoids and Iso-Eicosanoids: Constitutive, Inducible and Transcellular Biosynthesis in Vascular Disease;Cytokine gene polymorphism and heart transplant rejection.;Platelet-derived growth factor-BB and monocyte chemotactic protein-1 induce human peripheral blood monocytes to express tissue factor.;Association of smooth muscle cell tissue factor with caveolae.;Manifestation of cryptic fibroblast tissue factor occurs at detergent concentrations which dissolve the plasma membrane;A novel biological effect of platelet factor 4 (PF4): enhancement of LPS‐induced tissue factor activity in monocytes;Th1 and Th2 T-helper cells exert opposite regulatory effects on procoagulant activity and tissue factor production by human monocytes.;Granulocytes enhance LPS‐induced tissue factor activity in monocytes via an interaction with platelets;Cathepsin G-dependent platelet stimulation by activated polymorphonuclear leukocytes and its inhibition by antiproteinases: role of P-selectin-mediated cell-cell adhesion.;Relations between factor VIIa binding and expression of factor VIIa/tissue factor catalytic activity on cell surfaces.;Platelet activating factor enhancement of lipopolysaccharide‐induced tissue factor activity in monocytes: requirement of platelets and granulocytes;Dysregulation of in vitro cytokine production by monocytes during sepsis.;The role of arachidonic acid release and lipoxygenase pathway in lipopolysaccharide-induced thromboplastin activity in monocytes.;In vitro production of IL 1β, IL 1α, TNF and IL 2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation;Functional tissue factor is entirely cell surface expressed on lipopolysaccharide-stimulated human blood monocytes and a constitutively tissue factor-producing neoplastic cell line;Localization of tissue factor in the normal vessel wall and in the atherosclerotic plaque.;Cytokine-induced procoagulant activity in monocytes and endothelial cells. Further enhancement by cyclosporine.;Tumor necrosis factor induces tissue factor-like activity in human leukemia cell line U937 and peripheral blood monocytes.;Enhancement of mononuclear procoagulant activity by platelet 12-hydroxyeicosatetraenoic acid.;The stimulatory effect of platelets and platelet membranes on the procoagulant activity of leukocytes.;The Role of Protein Synthesis on the Generation of Tissue Factor Activity by Leukocytes;Monocytes.;The high responder phenomenon: enhancement of LPS induced tissue factor activity in monocytes by platelets and granulocytes.;Selective cellular expression of tissue factor in human tissues. Implications for disorders of hemostasis and thrombosis.;In vitro production of IL 1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation.;Mononuclear cell tissue factor: cell of origin and requirements for activation.;After blocking with 5% non-fat skimmed milk, membranes were incubated with mouse anti-human TF monoclonal antibodies;After Lymphoprep isolation, total RNA was extracted from MNC pellets with an;OR) anti-human TF monoclonal antibodies (10 µg/ml) for 30 min on ice. Mouse monoclonal antibodies recognizing extracellular domain of human TF were prepared by the Monoclonal Antibody Facility"
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "https://europepmc.org/articles/pmc549252?pdf=render",
         "Biology,Medicine",
         "Antimicrobial Agents and Chemotherapy",
         "T. Levin,B. Suh,P. Axelrod,A. Truant,T. Fekete",
         "24218070,1941629,144104608,4961128,49307125",
         "179",
         "10.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         "http://www.commsp.ee.ic.ac.uk/~ng1/pdf/gehrigD_CL_Jan05.pdf",
         "Computer Science,Mathematics",
         "IEEE Communications Letters",
         "N. Gehrig,P. Dragotti",
         "3017159,2752909",
         "59",
         "5.0",
         "2005-01-10",
         "2005",
         "a0de588554dc558fc9a7c968f02626444c917679;05c21f03eefa2d756229000ebe133b6bc6d08e1c;5e0eff55c4e3e3158b17e5452e213226fd8f21ac;33e627cb93a476a501308fb500015d282297ac03;d7cfbf051867bf3b35c3a2738a4731a148e06cc5;6e3da810748d174ccf2107591019ed318eaf6e40;f2b32222ec8955c4378fd2804757f4f8449d9241;29c907591b8be92bb535cc4e7453a4423d770238;c41ecfe2e42872cf03ced942f1d5daf918a00b5f;71c969f0a7204259cdd394283ab02d0fc34e5fc2",
         "Design of Slepian-Wolf codes by channel code partitioning;On some new approaches to practical Slepian-Wolf compression inspired by channel coding;Distributed code constructions for the entire Slepian-Wolf rate region for arbitrarily correlated sources;Compression of binary sources with side information at the decoder using LDPC codes;Compression with side information using turbo codes;Compression of correlated binary sources using turbo codes;The capacity of low-density parity-check codes under message-passing decoding;Distributed source coding: symmetric rates and applications to sensor networks;Distributed source coding using syndromes (DISCUS): design and construction;Noiseless coding of correlated information sources"
        ],
        [
         "9",
         "000a2234245d642c786a7355a15623444c33c1dc",
         "https://www.jimmunol.org/content/jimmunol/175/5/3140.full.pdf",
         "Biology,Medicine",
         "Journal of Immunology",
         "Suzel Davanture,J. Leignadier,Pascale Milani,P. Soubeyran,B. Malissen,M. Malissen,A. Schmitt-Verhulst,C. Boyer",
         "6084539,5629146,153940998,3941723,5534674,5153661,1400945749,1786102",
         "29",
         "0.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "10",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "http://www.cs.ucr.edu/%7Ekishore/papers/tencon.pdf",
         "Computer Science",
         "IEEE Region 10 Conference",
         "A. Negi,K. P. Kishore",
         "1728262,2341041934",
         "41",
         "3.0",
         "2005-11-01",
         "2005",
         "a6ce59fb59ba15a8833a12bfe6692dfcd436824a;aa03424e6c7e7c9415b554a68643c16085df0187;100ab9c177c6e0abdda1130eada7a3fe8015606c;25be4335206df53c769244cfce81ebc5914a46cf;1feb6fb70407c123de7af972a4271c07472fe3a6;1f0a71167544d4018089c51861b10365993ff151;956615751a21c1071f18ab90193f218d6319013e;6bc43977fb11cceed0b9aa55b23c6dd29dd9a132;9308cfdabf5303534b97d9ce5bfbb2c919a3f9cb;da7a3280ec5e77c007045eee8f6f657f567ab97d;3c520bf9d75f785f4d4935ef8bd9e4b38c7c5db2",
         "Performance Contracts: Predicting and Monitoring Grid Application Behavior;The design, implementation and initial evaluation of an advanced knowledge-based process scheduler;Understanding the Linux Kernel;Predicting Application Run Times Using Historical Information;A Historical Application Profiler for Use by Parallel Schedulers;Context Switching and IPC Performance Comparison between uClinux and Linux on the ARM9 based Processor;Characterizing Process Execution Behaviour Using Machine Learning Techniques;The Pearson Education;Linux Kernel Development , 1st ed;Correlation-based Feature Selection for Machine Learning;A Dynamically Adaptive CPU Scheduler;Linux Kernel 2.4 Internals;Co-rrelation based feature selection for Machine Learning, Master Thesis, Department of Computer Science, University of Waikato;Machine Learning , 1st ed, pp: 52-75, 154-183, 230-244, The;WEKA: The Waikato Environment for Knowledge Analysis;ELF: From The Programmer's Perspective;The Design of the UNIX Operating System;Build the knowledge base of static and dynamic characteristics of the programs from the run traces obtained in step 1 and train them with the C4.5 decision tree algorithm;is a Masters student in the Department of Computer and Informations Sciences;We send this STS information to modi(cid:2)ed scheduler through a system call;The scheduler instructs the CPU such that CPU allocates STS ticks to ’X’;decision tree will classify ’X’ and output the STS. per second.(cid:2)le;Aivazian , Tigran , Linux Kernel 2 . 4 Internals , The Linux Documentation Project , August , 2002"
        ],
        [
         "11",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "https://europepmc.org/articles/pmc1087730?pdf=render",
         "Biology,Medicine",
         "Molecular and Cellular Biology",
         "Takayoshi Bando,K. Sekine,Shizuka Kobayashi,A. M. Watabe,Armin Rump,Minoru Tanaka,Yoshikuni Suda,S. Kato,Y. Morikawa,T. Manabe,A. Miyajima",
         "24043805,2063038091,2110065840,40447710,32435082,2110038433,36341155,2568535,32275049,5658239,4789048",
         "38",
         "3.0",
         "2005-05-01",
         "2005",
         "",
         ""
        ],
        [
         "12",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "https://infoscience.epfl.ch/record/87259/files/Cavallaro2005_1275.pdf",
         "Computer Science",
         "IEEE transactions on circuits and systems for video technology (Print)",
         "A. Cavallaro,Olivier Steiger,T. Ebrahimi",
         "145440152,2359024,1681498",
         "120",
         "7.0",
         "2005-10-01",
         "2005",
         "bbb456acfec69d279b3f653b53dc1182d1b44eb6;cedf72be1fe814ef2ee9d65633dc3226f80f0785;dcdef9d1abf4be052e309058d695f19015868209;168033139bddbd1c02f761dd905375203cc2f73a;c69e28503a607c30f042fbd781de47c2a34b57fb;37eec7c527780a71578ecb32b5a48d563e651c7d;51a5d7d650ea5fdc12ae6e359ed052f67c3482c3;972fc844400ad601ca5c9274912ecb8007457790;d91e1b1ee1f83e2b3dc54ba7811a3c38f77ea701;48c7afd55f620cc665cabb52755968c5c57d9eda;ea3bcd105081726ab403513af553f5d0c32a2c6c;f494794fd30f451365aa69a96227ba8384ca492b;8b2f244b587207e8ee5415d507e432d5220d6163;a741ea5e483061f2ad1767290770ef35e74fcab4;3b9f60928a95709d90cf6b2f0705b16c07fca05b;d497613415c71d3ab544f3531871173749b31226;ce8883dbe8c695d8ef770eefe34e48efe7c0cf64;c1347daf84d6750a4fcb16cd04a1124ea3e7b775;100c788dea25ed53de43b1d5505afd19f4d85e30;8406c229176f56903f7de8e913081326341dcda9;89a05047a487ba4e9c32687ceb7fc00f59a834f2;5e52ecd0f0ab96bcb0b362e4ea4f0207ec996e66;292889c24c0fa339af5927bb4845b25b89a5e57a;86bbc82e2891d5baf900e40668f897e919d9ed87;332da8685270e4d39bdcededc7a00d1631c645d4;efa69b21767d15897e00f11bdf4625a8dcf1c1c4;febae7da65fbd5ec7d1227cf82e82cfbe48532dd;319b9413a46218c2832d6f3571880ac0d55adfbe;ba01ca8a463e294fc6a36473f6391095cbb164f2;e32c87769fef343508bd9bb2f7ee083417354245;dbc8987f77aaecdb058aac10c175124b06b1dbaf;cffb520ae667ed8737bbe3f5a05d86e8f85b9dba;0c646a8923d70ae9a93ee8945c05e47246707d80;cc032af2d72406bff21b34a473fdc3588c2fe920;770429367eb26731adc1abc31ba3693f7126e3a5",
         "Automatic foveation for video compression using a neurobiological model of visual attention;Text information extraction in images and video: a survey;DAVE: a system for quality driven adaptive video delivery;Video transmission adaptation on mobile devices;Visual attention for region of interest coding in JPEG 2000;Semantic segmentation and description for video transcoding;Resource adaptation based on MPEG-21 usage environment descriptions;Video transcoding architectures and techniques: an overview;Metadata-driven multimedia access;Semantic transcoding for live video server;Multiple video object tracking in complex scenes;Complexity-quality analysis of transcoding architectures for reduced spatial resolution;Instantly indexed multimedia databases of real world events;Face detection in color images;MPEG-7 description of generic video objects for scene reconstruction;A new content-based hybrid video transcoding method;Video Processing and Communications;Media conversions to support mobile users;Object-based transcoding for adaptable video content delivery;Overview of fine granularity scalability in MPEG-4 video standard;Video object extraction based on adaptive background and statistical change detection;Video transcoding for universal multimedia access;Heterogeneous Video Transcoding to Lower Spatio-Temporal Resolutions and Different Encoding Formats;Scalable multimedia delivery for pervasive computing;Adapting Multimedia Internet Content for Universal Access;A frequency-domain video transcoder for dynamic bit-rate reduction of MPEG-2 bit streams;Multimedia content description in the InfoPyramid;Objective methods for assessment of video quality : state of the art;On the methods and performances of rational downsizing video transcoding;Shadow detection in image sequences;Video Transcoding Architectures and Techniques : An Overview;Visual radio: should we paint pictures with workds, or pictures?;Dave: A system for qual ity driven adaptive video delivery,”Proceedings of the 5th ACM SIGMM international workshop on Multimedia information retriev;Processing and;Information technology – coding of audio-visual objects – part 2 visual–amendment 2: Streaming video profiles;Statistics, 3 ed;Continuous speech recognit  io;Submitted by;Information technology — Generic coding of moving pictures and associated audio information — Part 2: Video;SUBJECTIVE VIDEO QUALITY ASSESSMENT METHODS FOR MULTIMEDIA APPLICATIONS"
        ],
        [
         "13",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "https://europepmc.org/articles/pmc1081291?pdf=render",
         "Medicine,Biology",
         "Journal of Clinical Microbiology",
         "S. Nunes,R. Sá-Leão,João André Carriço,C. Alves,R. Mato,A. B. Avô,J. Saldanha,J. Almeida,I. Sanches,H. de Lencastre",
         "3688641,1398826424,4908451,2053407517,12533308,10268213,40282052,116697293,2452668,2290923",
         "67",
         "2.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "14",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "https://wwwnc.cdc.gov/eid/article/11/8/pdfs/04-1197.pdf",
         "Medicine",
         "Emerging Infectious Diseases",
         "P. Yagupsky,E. Baron",
         "6618694,39055987",
         "185",
         "2.0",
         "2005-08-01",
         "2005",
         "25f20e694999a6840a06702d74cfd37d95d667f3;1ff74908befd25fee4b2ab003de4f83a41a41e45;66cbab4b36bb6b4952ea76981c60f27ab3275898;bb816e36ad186f5dfde74d71885070d766e2d747;e8d3010e9528076bca3e5fba37538edaa1635eb1;2ae2e7ee84a9137b010e909f76fc936ff775ad6e;af5b6fcc6f7f7a1b4c60f0d7b74ca15e67a14519;7cd9837410cc2ef6f967852c5ffa0cbe336dbdea;256f57d10a7a759d7e388a7cbbaa2fd54613a54b;e59cf90691ee9a5bd3aa34c70030e27b623ffd14;c39da3e000e116a2ebf7e6b115799a90c4d1923f;3fdffab87e6d38e294b3d76c2df8568fcdf8a0c6;0d18a3b83e52b92ee2a0fb372eaa4346feb551ca;a162170411238cb7a93e9c93040982aa25454c19;c41cac3f6e8765bef847ec9b406aaacffe0e4fa1;39eb0675b38ca75201203941d13ba0f4a186e283;594b085d66a958b96aba9d1933972d3b34bb7678;e40f0229ec9bbe688f6356438951dfbb36fc59cb;0ca7f50704a2e0ab7da2896839f913a571b2e777;df8e94c95ce7f4d08c04cb2d562200b1eb0ddc59;4e4469c2a2132167fae10e5133dbd033b50d7471;f8b32e9384596d0545c70b5623e957a3aa7f60f9;1e0d986827359e89c42fdc145007c65b89a15ab5;db1a7e39ea496679bee7a5c6073402b6fd17e8d2;647d1a66e8764bdca4767f352d54f97fcd5252ac;cf4b52becaed752bb036d66d6cba6c0d1785af1d;24fab9ab18c998c12f7e99c14f31ae41c822402f;dd97f7c782c7ff9d1a98d2454f07dba3fc9e2906;29debcd880647e9d5ec1e1033228f5ca91753a8c;a03a55d8331f010683a46d3f2a808c56459961ad;1f7571e5b811f06b77bd16c7f277ed9849cd69e6;31a5d17ec14a53090757e7042ca707f4b8f3a35a;736a26b7dbf151a54974d500bad5c08b0b723ebb;f64fee05bc23ffc94ffec0e1061461c398ddf905;60fedd708bd15ec43d3e4c891b4f389db9c5fecf;e397ef4308fcf8f9cdc71dd5a4e43bb4b5d20887;ee5e98347d31991dd2b088ae627039ae24f0b10f;362c818a28759e5fca9a7f0d4e92ed695951a5fb;a01715bd830311a59ceda880f94f11a860224873;2b233f3fe9887477b9333af2751d2476954bc901;558e59a9972530acb403739cc7ef0239143f5a8f;ed002af5819415ec0a8b31a69ce941f6e7d53f64",
         "Laboratory-acquired Brucellosis;Acute hepatitis due to brucellosis in a laboratory technician.;Prevention of Laboratory-Acquired Brucellosis;Analysis of risk factors for laboratory-acquired brucella infections.;Use of the Isolator 1.5 Microbial Tube for Detection of Brucella melitensis in Synovial Fluid;Bacterial Pathogens as Biological Weapons and Agents of Bioterrorism;Unusual Complication of Breast Implants: Brucella Infection;Brucellosis in laboratory workers at a Saudi Arabian hospital.;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Brucella abortus Infection Acquired in Microbiology Laboratories;Posttreatment Follow-Up of Brucellosis by PCR Assay;Detection of Brucellae in Blood Cultures;Rapid diagnosis of human brucellosis by peripheral-blood PCR assay;Failure of short antimicrobial treatments for human brucellosis;Comparison of BACTEC 9240 Peds Plus medium and isolator 1.5 microbial tube for detection of Brucella melitensis from blood cultures;Exposure to Brucella in the laboratory;Brucellosis from sniffing bacteriological cultures;Grand Rounds—Hammersmith Hospital: A case of laboratory acquired brucellosis;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Live Brucella spp. fail to induce tumor necrosis factor alpha excretion upon infection of U937-derived phagocytes;Outbreak of Brucella melitensis among microbiology laboratory workers;Unsuspected brucellosis diagnosed in a child as a result of an outbreak of laboratory-acquired brucellosis.;Brucellosis: imported and laboratory-acquired cases, and an overview of treatment trials.;Biochemical mis-identification of Brucella melitensis and subsequent laboratory-acquired infections.;Laboratory diagnosis of brucella infection: some pitfalls;Outbreak of Brucella melitensis among microbiology laboratory workers in a community hospital.;An outbreak of Brucella melitensis infection by airborne transmission among laboratory workers.;Laboratory-acquired meningitis caused by Brucella abortus strain 19.;Past and present hazards of working with infectious agents.;From the Center for Disease Control;Brucella canis: A cause of undulant fever;An unusual outbreak of brucellosis.;Brucellosis: I. Laboratory-Acquired Acute Infection;Brucellosis in the United States.;Brucella species;Brucellosis: An occupational hazard for medical laboratory personnel. Report of five cases;Beer-Sheva 84101, Israel, fax: 972-8640-3541;Prevention of laboratory-acquired brucellosis.;(revised). Geneva: World Health Organization;Basic protocols for level A laboratories for the presumptive identification of Brucella species. Washington: American Society for Microbiology;Exposure of hospital personnel to Brucella melitensis and occurrence of laboratory-acquired disease in an endemic area.;Biosafety in microbiological and biomedical laboratories;Laboratory-associated infections: incidence, fatalities, causes, and prevention.;Principles and practice of infectious diseases. Vols 1 and 2.;Brucellosis in the United States, 1965-1974.;Laboratory Infections Due to Brucella"
        ],
        [
         "15",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "https://hal.inria.fr/inria-00182046/file/2005-petti-fraichard-iros.pdf",
         "Engineering,Computer Science",
         "2005 IEEE/RSJ International Conference on Intelligent Robots and Systems",
         "S. Petti,Thierry Fraichard",
         "40180963,1694234",
         "287",
         "17.0",
         "2005-12-05",
         "2005",
         "3d89fe08f16bf8ecabb7bb26115b8baccbdd8712;64be486150521eddcf83ef356a0719d1ef65ceb8;0cfc462c907a790ecc56fba3ffeade9f1fbae757;3c0ced2de89291e1e94a68217a2d27f4b8ba1b64;9cec1297bfce7ddd63390670467f1af6ccae4165;e4ed63ae63e6a8634e115925284f8c644b943623;9dddc932f553f31c14fdf941abe4f8961eef34f8;1281372e4b79bdb0d58a3d1dd2a813d23b2490c0;b0501adbca9917c19ae7b343e74b6e163753a227;62113b0fcb182853446679430aaefa9c367fe438;dc6e901b8c62d90c2d77877488b51c46f8bdf8a8;60f389d8def6c93aab17faf369f0727edbd51f47;afcd60559f9bc54bc2e0fd063fc0eb2bef87482e;9ba11955f7d8c93163288c33f9c3ed4cffc89a31;3f77f8b0e1a3e3dcedb0953a42018b6948b60617;ccfb06935ea9c8832ec6086000f71cf584a79679;7e81dac6260c4768af3a28ac21c78c5a38a5f7d0;2078d9a14df37d0992283c03cf42f591d71aeb3b;4d95e5c7c3b7d8a2a8a273e69a5b4fb98557b912;90c8e1ff97ffede807ce0e4bd20b984dd96725d9;ba40bb0b5d17afdab9adfcad9648a184b0a6c7f9;e8d60f2e1d51566953256020c71fee569865c948;09ddcaaf43aa3115ffa86aeb106604c9233717d5;6260d36c5fcb7fefb997ede20c2ba983adfa2470",
         "Motion prediction for moving objects: a statistical approach;Inevitable collision states. A step towards safer robots?;Towards real-time global motion planning in a dynamic environment using the NLVO concept;An integrated approach to goal-directed obstacle avoidance under dynamic constraints for dynamic environments;Real-time randomized path planning for robot navigation;Improved fast replanning for robot navigation in unknown terrain;Learning motion patterns of persons for mobile service robots;Reactive navigation for non-holonomic robots using the ego-kinematic space;Randomized Kinodynamic Motion Planning with Moving Obstacles;Real-time motion planning for agile autonomous vehicles;Kinodynamic motion planning amidst moving obstacles;VFH/sup */: local obstacle avoidance with look-ahead verification;Randomized Kinodynamic Planning;High-speed navigation using the global dynamic window approach;Motion Planning in Dynamic Environments Using Velocity Obstacles;The dynamic window approach to collision avoidance;Probabilistic roadmaps for path planning in high-dimensional configuration spaces;The curvature-velocity method for local obstacle avoidance;The Focussed D* Algorithm for Real-Time Replanning;Reflexive collision avoidance: a generalized approach;Motion planning in the presence of moving obstacles;Robot Motion Planning;Sensor-Based Motion Control for Mobile Robots;Kinodynamic planning in a structured and time-varying 2-D workspace"
        ],
        [
         "16",
         "0012aecca19e4d3e41741599b1dbd00fb46d0c22",
         "https://academic.oup.com/carcin/article-pdf/26/12/2157/7088641/bgi195.pdf",
         "Medicine",
         "Carcinogenesis",
         "R. Rajagopal,M. Deakin,A. Fawole,J. Elder,J. Elder,V. Smith,R. Strange,A. Fryer",
         "2573963,2004776,15074846,145503858,145503858,2056036524,1931855,34067155",
         "19",
         "1.0",
         "2005-12-01",
         "2005",
         "",
         ""
        ],
        [
         "17",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2004.04487.x",
         "Biology,Medicine",
         "The FEBS Journal",
         "T. Utsumi,H. Ohta,Y. Kayano,N. Sakurai,Y. Ozoe",
         "6319788,122141924,49256899,48638495,6146172",
         "22",
         "0.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "18",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "https://europepmc.org/articles/pmc1134930?pdf=render",
         "Medicine,Biology",
         "Biochemical Journal",
         "S. Mousavi,Mitsuru Sato,Marita Sporstøl,B. Smedsrød,T. Berg,N. Kojima,H. Senoo",
         "145278181,47702160,1402638735,1432387861,145476071,50662160,4710114",
         "37",
         "1.0",
         "2005-04-01",
         "2005",
         "a6d3bbeb84717deccff079bd319413fc54b50e16;100f45a6b0045379892054aa70a738df681c6e97;85f7f66dc1be37706f712d6616387d33bb209a44;f86dc51f49b3f3d30e8e008c327f8b5402cddaf5;00be5eb58ac778cc624178ac010099a0601b7296;7fa779abd761c9306c433a9e87c1d11f8df19c1b;eb97c3f3cf1881a3908a6908f5e5a37e6d9c2145;cd648ef152b1f5a5d2b0f4645192b1c9b0ef17ec;2120667e80c26b7240ccac884467e869f6ce4c5f;3ea0c614b0b3edc4cb61b22054bb4f35e2bbeb5c;1c54f9a2d4aed39b8ec37e71cc63424a47526376;1ac7d0b397db7e0260c7bcf5ae4b4736f44f09a2;252991b9bbe9a0eb815051a797ae3a154600b74a;2dbfc38f53fc5d1c5441892b50d6cb0e5dfd170d;961e7f9ed82dbb973b0db57f7103e3c867d158ef;50c37eea83465e0ac7ebca071c3b75dc559f81d1;ae03a6754c9d89bdeca4474383cb078429c3830f;2303a910b94222ed2388c2206fa36060bdfeaf28;116139463e233dfbe4e8b71aa1a858b0b1145c22;c65bae3717eda5fa4a9cb63a6bf7c09040773c00;e0400ec8e27dd9036ad838755534df4bdfad87d9;2170efb311a61e99b0c7a960aa715a39d237aed1;c76fd3c2bb59cd99649849e51b66fbb8ee75cd36;c7b72147f381caa4b35b44ca665b144474030d9c;0fa5ea6e6e3b8020866ffbd9b3a3235e295a1432;eca866fb91011bad32af30aab5638562bea0275b;c9d968f1085827fd0af82e323ec482e0f1caa8a3;da432fa0f66c6a0fe29a8032ed229112c6679365;075cf9dd18564794e48c3d4736a84a9c4433562d;b0d137da67e4e7ecd9887ee57a7260f18d4710d9;bdad5d7454e41e03771edff7c7c86a83d512094c;8a7b2a7500846faa86f207bf2c90aff7c07691b0;8aa62d7c8426b02ddafb2df88390688a92690c1f;14253d459b1d6329ff175d332ece39d6ef03b9ed;7269b727ca35ac19da971ed902ed32dd7b8785f4;f999493daa293f2d24782a9120dcf3d1d880b0fa;6a4b503f0840c26dab7ce2f0012715b47d054152;9c9ba006ad1b6be24f473a4a1ad36681f7fbc744;fb76c9e322005d0334146cfb3b248d32f9837f93;1c610c79254816af5845d65e4e4ff90213f00267;2d4cea38f64c8dcc06d2d1cee84f90f55821c32f;c3aacfc7fc9e06c6960d46653260edbee87858fc;b8a05a8ab5e142cc0348cf5a31a16d0aed1568f8;c99120417ac2adc583085b340766dbaf52d071c2;bc0ebe8a73c8efbdb7d0016eec8c513dcdd0a748;634668d47753c48db112e9377445fc37f9eeca22;26270ce70c5c8e8d40201fa3dd9fa717c2029329;f0d468bd28cbaa9d08f27278bcf6b372a6d846f9;e2b117fe55064f148332db4e1b36f52a55f2b3b0;38017c1032349f35c0b1efeafaf5861a9aa4c718;3cf858a3295d7b6f4b5fc7b234945ab25b65a51e;c568ad1bc32539c8bb0b8809873e7ff9fe73396e;1db7bf5ff49493dcfc46c66ed29a4b663cd78295",
         "uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV.;Identification and characterization of the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor.;A targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen uptake;Liver extracellular matrix in health and disease;Hepatic stellate cells: unique characteristics in cell biology and phenotype.;uPARAP/Endo180 is essential for cellular uptake of collagen and promotes fibroblast collagen adhesion;Characterization of Sugar Binding by the Mannose Receptor Family Member, Endo180*;The mannose receptor family.;The C-type Lectin Receptor Endo180 Displays Internalization and Recycling Properties Distinct from Other Members of the Mannose Receptor Family*;Collagen-receptor signaling in health and disease.;The Urokinase Plasminogen Activator Receptor–Associated Protein/Endo180 Is Coexpressed with Its Interaction Partners Urokinase Plasminogen Activator Receptor and Matrix Metalloprotease-13 during Osteogenesis;Hepatic Blood Flow Regulation by Stellate Cells in Normal and Injured Liver;Extracellular Matrix Degradation and the Role of Hepatic Stellate Cells;Wortmannin-sensitive trafficking steps in the endocytic pathway in rat liver endothelial cells.;Effects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy.;Endo180, an endocytic recycling glycoprotein related to the macrophage mannose receptor is expressed on fibroblasts, endothelial cells and macrophages and functions as a lectin receptor.;Selective adhesion of macrophages to denatured forms of type I collagen is mediated by scavenger receptors.;A Urokinase Receptor-associated Protein with Specific Collagen Binding Properties*;The C‐type lectin superfamily in the immune system;An orphan receptor tyrosine kinase family whose members serve as nonintegrin collagen receptors.;The discoidin domain receptor tyrosine kinases are activated by collagen.;Vascular Smooth Muscle αvβ3 Integrin Mediates Arteriolar Vasodilation in Response to RGD Peptides;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Cell adhesion receptors for native and denatured type I collagens and fibronectin in rabbit arterial smooth muscle cells in culture.;An endothelial cell receptor for plasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of t-PA-dependent plasminogen activation.;Differential expression of matrix-metalloproteinase-1 and -2 genes in normal and fibrotic human liver.;Transfer of retinol-binding protein from HepG2 human hepatoma cells to cocultured rat stellate cells.;Fat-storing cells as liver-specific pericytes. Spatial dynamics of agonist-stimulated intracellular calcium transients.;Affinity of integrins for damaged extracellular matrix: αvβ3 binds to denatured collagen type I through RGD sites;Identification of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin in collagen.;Internalization of retinol-binding protein in parenchymal and stellate cells of rat liver.;p180, a novel recycling transmembrane glycoprotein with restricted cell type expression;The alpha 2 beta 1 integrin cell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen.;Activation of cultured rat hepatic lipocytes by Kupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation of cell proliferation via induction of platelet-derived growth factor receptors.;Characterization of desmin‐positive rat liver sinusoidal cells;Hepatic lipocytes: the principal collagen-producing cells of normal rat liver.;A radioiodinated, intracellularly trapped ligand for determining the sites of plasma protein degradation in vivo.;The effect of leupeptin on intracellular digestion of asialofetuin in rat hepatocytes.;The structures of the galactose-containing sugar chains of ovalbumin.;HIGH-YIELD PREPARATION OF ISOLATED RAT LIVER PARENCHYMAL CELLS;Distribution of Endo180 receptor and ligand in developing articular cartilage.;Endocytic uptake of advanced glycation end products by mouse liver sinusoidal endothelial cells is mediated by a scavenger receptor distinct from the macrophage scavenger receptor class A.;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-α and TGF-β1;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1.;Vascular smooth muscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD peptides.;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Affinity of integrins for damaged extracellular matrix: alpha v beta 3 binds to denatured collagen type I through RGD sites.;Expression of the gene of the α-smooth muscle-actin isoform in rat liver and in rat fat-storing (ITO) cells;Acidification of the endocytic and exocytic pathways.;Collagen production by rat liver fat-storing cells in primary culture.;Newly administered [3H]retinol is transferred from hepatocytes to stellate cells in liver for storage.;Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes), their related structure in and around the liver sinusoids, and vitamin A-storing cells in extrahepatic organs.;Preparation of isolated rat liver cells.;Hepatic stellate cells: morphology, function, and regulation;Isolation of perisinusoidal stellate cells (vitamin A-storting cells, fat-storting cells) of the rat liver"
        ],
        [
         "19",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "https://europepmc.org/articles/pmc538921?pdf=render",
         "Medicine,Biology",
         "Antimicrobial Agents and Chemotherapy",
         "L. Poirel,M. Lartigue,J. Decousser,P. Nordmann",
         "6593096,40361075,46491845,5228247",
         "233",
         "8.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "20",
         "0018ece872bb78b06eb26ea35d5ed27f53702ce6",
         "https://academic.oup.com/bioinformatics/article-pdf/21/10/2558/48972005/bioinformatics_21_10_2558.pdf",
         "Computer Science,Medicine,Biology",
         "Bioinform.",
         "Shailesh V. Date,E. Marcotte",
         "2124202,1853998",
         "18",
         "0.0",
         "2005-05-15",
         "2005",
         "",
         ""
        ],
        [
         "21",
         "001a3ef673ee3e7703c7db597386a2162a2dc402",
         "https://academic.oup.com/icb/article-pdf/45/3/391/1674343/i1540-7063-045-03-0391.pdf",
         "Medicine,Biology",
         "Integrative and Comparative Biology",
         "R. Fuller,C. Baer,J. Travis",
         "7240006,2435258,145389296",
         "137",
         "8.0",
         "2005-06-01",
         "2005",
         "",
         ""
        ],
        [
         "22",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "https://zenodo.org/record/1259441/files/article.pdf",
         "Biology,Medicine",
         "Virus Research",
         "I. Kuzmin,G. Hughes,A. Botvinkin,L. Orciari,C. Rupprecht",
         "31991394,145378585,4167987,4216439,1727205",
         "187",
         "15.0",
         "2005-07-01",
         "2005",
         "f9833b6bdeb64d7b59c9f7d9b344ce25d5c90dba;2b97169a29a168ab41c3ec1a78164b96a6969a41;7a98ad002f81a2f7aab8d8491cadbc88aa8c3155;d1b3e4ff02d07be96c06fbfaf555406f17196741;33be5abf0ecb7dfdc6402e68dc86a1126c13adc8;1a9bc3e0a07a9d95d4f35323ab0f1c8c1413ef76;4001a31540fceaa58f5f393584a71522e1e31e4c;6b4c6ff3535d650d61c57ab38337d12b62314d01;7bcecb53c6e78442750e3e9b72ba3a80e769f70f;511a070c6dfd315dc7c4ae0e59c07957737618d8;dd06144588032875303e5ad4531de249890411a2;51a951e0338efd7eac40cb05a8d8bb1f9c0b2dc9;3e25fda5c88fe7969682f3203a9707ef42f0e551;6bb3b70653cf04a5b2c68240e740df7412f30ea2;725a3cbac5e12ee0dcd1dbd036929dc6d7783b58;ebb01f73c93b2d963c3dd09e607d85ac2921bd08;283f8d4a6b328ecf87aa6896307fa92a4159e282;a4bd32fbe148185f7f96bb6a8f057a8bee258e55;7696a27bd12a890797d339f730585d240ed59c1f;f63ea68c269895152c1aff73e5b1927b6fb32949;ad56575dae7f33f0a62df28b1bdc1c8344e4352a;8d54df6996acbb4c6758544b046278166afa4a9a;b0caf52d27797f3f413444bb15f5d6692b6375c9;cd2628e16a0c5978ccd8ab019308853ec712cc4b;aedc61581cafb2a48d3a0095d8694570f6d1e5d0;93260ef642602445acfac1a3130c86e85873cc38;2fa65c170ab7e679cbc0eecc5b03d3e25dac6556;5d8c958b183dad733d0e1003b74569b79a349478;1dfb94db76de9db1b86b2bd896a1991cdaf33c66;37815c1e6cce8b572a44892b360e619b10bfd28f;321bbf45869d401205db0123dd09aa5ddb78324b;f3b9fe7f78f8dd0673a811fec3b3741d652dcd8e;36f2bc416239289425d64110703dd037196571da;f1f369b6f62a2374dc6cba39c6ca58aa2544b016;1de7a444fac66d2c1f3a318d415714d524940c9d;e63bdf8efd8a01e5b53deee54a5837c8538b7cfe;970725ff0887cbdbf57e820588e4d7bb047383bf;c0f8a73ab829481eb8f97763d38a4d8eedaaf1d5;47c403f4295ab506971e1581475501908ac30e4e;385852642f3ada1e3dbf8ed90c005263a0be280a;dd20fc8f997d821dee710fe01a27049c12b5dfaa;54dfa2d112efd63b772c8c70795cd9cd7d3fc8dd;5685328fbbbf401d8b5b77924c616a825a56ea91;ba000c39195024b6f16e4eda48468db5be1506c0;f3ac4157691f496505db5c7b808b079f0e9d2363;682a5700489eec2f9c1f4f6246b73e5bcdb5fa97;f8d1d71b86d771cebd9061fba6e8459db171bbfc;096a93c9e0841bd3f99bfb486b170ed2c1b5d434;762cb1a4d0b79d9e9287cea102930294c87bf3ab;19b2eaa20f170259f94c5a84e1ff1952237bf92e;53bd1d94388d92e50aee35e9f5eb43a501b2a5e9;029463508fe0009869be5a3736075ab528d63664;adfd3c0cf96e20f5714a2deb0bb69772182b80da;2507c1996314911d9e998281f95f2a597358f74b;3842589a2100702965d4123501ff19858dc62cee;ec6f2977cb6c80aa845eafc14d46417a92ec3a00;17eab3eab209ab9b87425de3b51540b5efd9dd9e;b1d9cae15b280af1d4bc9001f066b757df14f5c4;e9038f1b41d390f5e6eeb9d8da000a94eec21e86;d3f4ba36f4de1d317078f90aa82b7e1526b5fb94;6993234eb3aba662d7d19b2b2c9b2f6f7304f830;a9c8b15829024098a9825321727fcd9e0d12c57d;0ae262d9cf78536754bc064e07113ab5e978f208;73679f1ba00de9e73492f3f5d53bdcef50af3fb4;85e77924e184aa659599cbc1675ac66142a3683c;924c6ec5ea4524ab6aa2e8f6e66bcdbcc4a7ba2b;94f848c5d0621d840bb13305ae945100c3551e77;cb592f5cbc4a9b871a79d9256404617ef6e5892f;2d10e6a2e1ffcb5608fa78ea47bab7927e53994a;0b8eb3ae0d8c670378d55f04ba69fa2f26ae2d31",
         "Novel Lyssaviruses Isolated from Bats in Russia;Bat lyssaviruses (Aravan and Khujand) from Central Asia: phylogenetic relationships according to N, P and G gene sequences.;A comparison of DNA vaccines for the rabies-related virus, Mokola.;New Lyssavirus Genotype from the Lesser Mouse-eared Bat (Myotis blythi), Kyrghyzstan;A molecular epidemiological study of Australian bat lyssavirus.;Competition Commission inquiry into POMs.;Characterisation of an Australian bat lyssavirus variant isolated from an insectivorous bat.;Phylogenetic comparison of the genus Lyssavirus using distal coding sequences of the glycoprotein and nucleoprotein genes;Lyssavirus P gene characterisation provides insights into the phylogeny of the genus and identifies structural similarities and diversity within the encoded phosphoprotein.;Increased positive selection pressure in persistent (SSPE) versus acute measles virus infections.;European Bat Lyssavirus Infection in Spanish Bat Populations;Serologic Evidence of Lyssavirus Infections among Bats, the Philippines;Genetic constraints and the adaptive evolution of rabies virus in nature.;Expression of the B-cell and T-cell epitopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG and induction of an humoral response in mice.;MEGA2: molecular evolutionary genetics analysis software;Extensive Attenuation of Rabies Virus by Simultaneously Modifying the Dynein Light Chain Binding Site in the P Protein and Replacing Arg333 in the G Protein;Molecular basis for the interaction between rabies virus phosphoprotein P and the dynein light chain LC8: dissociation of dynein-binding properties and transcriptional functionality of P.;Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains;Host Switching in Lyssavirus History from the Chiroptera to the Carnivora Orders;Evidence of Two Lyssavirus Phylogroups with Distinct Pathogenicity and Immunogenicity;Antigenic and genetic divergence of rabies viruses from bat species indigenous to Canada.;Codon-substitution models for heterogeneous selection pressure at amino acid sites.;Sequencing and position analysis of the glycoprotein gene of four Chinese rabies viruses;Ecology and evolution of rabies virus in Europe.;Phylogeographic patterns exhibited by Ontario rabies virus variants;Chimeric Lyssavirus Glycoproteins with Increased Immunological Potential;Multiple sequence alignment with Clustal X.;Rabies virus quasispecies: implications for pathogenesis.;DNA-based immunization for exploring the enlargement of immunological cross-reactivity against the lyssaviruses.;PAML: a program package for phylogenetic analysis by maximum likelihood;Evolution of European bat lyssaviruses.;The complete Mokola virus genome sequence: structure of the RNA-dependent RNA polymerase.;A Biographical History of Veterinary Pathology;Ten-year survey of British bats for the existence of rabies;Encephalitis caused by a Lyssavirus in fruit bats in Australia.;Characterization of a unique variant of bat rabies virus responsible for newly emerging human cases in North America.;The design of strain-specific polymerase chain reactions for discrimination of the racoon rabies virus strain from indigenous rabies viruses of Ontario.;Genetic polymorphism in the rabies virus nucleoprotein gene.;Molecular epidemiology of lyssaviruses: focus on the glycoprotein and pseudogenes.;Molecular diversity of the Lyssavirus genus.;Isolation and characterization of 115 street rabies virus isolates from Ethiopia by using monoclonal antibodies: identification of 2 isolates as Mokola and Lagos bat viruses.;[A lyssavirus with an unusual antigenic structure isolated from a bat in southern Kyrgyzstan].;Rapid sequence evolution of street rabies glycoprotein is related to the highly heterogeneous nature of the viral population.;PCR technique as an alternative method for diagnosis and molecular epidemiology of rabies virus.;T helper cell epitope of rabies virus nucleoprotein defined by tri‐ and tetrapeptides;Molecular cloning and complete nucleotide sequence of the attenuated rabies virus SAD B19.;Rabies-related Yuli virus, identification with a panel of monoclonal antibodies.;Efficacy of rabies vaccines against Duvenhage virus isolated from European house bats (Eptesicus serotinus), classic rabies and rabies-related viruses.;Immunity against the European bat rabies (Duvenhage) virus induced by rabies vaccines: an experimental study in mice.;Localization and immunological characterization of antigenic domains of the rabies virus internal N and NS proteins.;Walking along the rabies genome: is the large G-L intergenic region a remnant gene?;HUMAN RABIES OF BAT ORIGIN IN EUROPE;Rabies in the Tropics;Rabies-related viruses.;Rabies virus infection of a flying fox bat, Pteropus policephalus in Chandigarh, Northern India.;Giulio Alfredo Maccacaro.;Experimental infection of dogs and monkeys with two rabies serogroup viruses, Lagos bat and Mokola (IbAn 27377): clinical, serologic, virologic, and fluorescent-antibody studies.;An unusual case of human rabies thought to be of chiropteran origin.;Two African Viruses Serologically and Morphologically Related to Rabies Virus;Isolation of Rabies Virus from Fruit Bats in Thailand;Isolation of a virus from Nigerian fruit bats.;New Cases of Mokola Virus Infection in South Africa: A Genotypic Comparison of Southern African Virus Isolates;BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT;MODELTEST: testing the model of DNA substitution;Immunoprotection by rabies virus nucleoprotein.;Nucleotide and deduced amino acid sequences of the nominal nonstructural phosphoprotein of the ERA, PM and CVS-11 strains of rabies virus.;Rabies and Rabies-related viruses in Zimbabwe historical, virological and ecological aspects.;Rabies-Related Viruses;Characterization of an antigenic determinant of the glycoprotein that correlates with pathogenicity of rabies virus.;Antigenic characterization of the bangui strain (ANCB-672d) of lagos bat virus"
        ],
        [
         "23",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         "https://www.jneurosci.org/content/jneuro/25/48/11071.full.pdf",
         "Psychology,Medicine",
         "Journal of Neuroscience",
         "Y. Sugase-Miyamoto,B. Richmond",
         "1400827629,3007646",
         "110",
         "5.0",
         "2005-11-30",
         "2005",
         "",
         ""
        ],
        [
         "24",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "https://europepmc.org/articles/pmc1323209?pdf=render",
         "Biology,Medicine",
         "Proceedings of the National Academy of Sciences of the United States of America",
         "Huiling He,K. Jażdżewski,Wei Li,S. Liyanarachchi,R. Nagy,S. Volinia,G. Calin,Chang-gong Liu,K. Franssila,S. Suster,R. Kloos,C. Croce,A. de la Chapelle",
         "2328535,3744462,2157336158,2934528,32344884,1906416,2124909,16073087,4761141,3162753,3481071,144749275,3597724",
         "1251",
         "59.0",
         "2005-12-27",
         "2005",
         "5631e50fd39d9c7a54dec2bb5e5a016f8402c8b5;10bae8b93245299d0e4c4079e76f4d8c10458709;8de8119e007331a5b9e11c3ef99e15b2c7c8edce;e181f1874c85c6d42bf794f4e8aef5be2f77306a;c7497d6d32d3dd6f893609a17848c807134e1cc4;d8138beef46811bcb2ec68a4ae029b274ee4e1d1;77ef9aac7e0199184cb3169ac5e4de1d5fd7152b;94e90f0d2dba39b2357283e78f1a2dcb3f3ec358;240aa3fbe87b1fb085815baa87cca8bd04f52f70;631cf40b915e9f9b176d7a9be8a22c66a27a1acc;aab9e4d3502a20eb48d22ae8bb6ce29aabfe6a0a;01ae7d930255dd36a3308bd912d29d79859934db;62278466704924429db9b60bebc4ce8b4b7f2521;f60f9040c85eb47659079adf779a31bdb58014eb;c87c7646aa4273e73cdb1f2a7a8804281b478b4e;c8bb4cfcb04928f5312c66cc0239cca93839ff8b;72136fce3ef7aa30bf7ce640f44715daafe93c03;eab6ab732d7951e804769646a42a261665ed7297;8c1f06689340bbbd5f56d90511a338ebc958c517;2459b8e56f02c9d9c3b19ae8fcd75fedf05391d5;d577f656be9a1aa6d951dfd8cb02867d7194f4a8;3ed944a9b0fcd2a77298049f092a3b97b4633bc7;cde7b10619a54868cfbdacf99d051bc631bb86d7;97c4ead52f67e84d6ab55bb4f73a192533a95e1a;0442c89a09bfd5ba7f09340d0b23143f6c3f32b7;c81a5f7469419748daff6272230ed330001ee8f2;06cc16cdaf46fa657bb001faec28021b8ab5a3a1;5be20043c72560471655eba6f2a23329f5ea5f5e;b8f81ba06d314b8b112298c5b600557108c45872;92dcca877b55b85ceaa139bc5a6d4a563d598971;8259057fd4d7fb899894ff37c3fa9c2d1cca8798;726fa5af9a99f441cdddb2f86ffec6d6c968a6d5;80ce85acd05007231bbb58f41a4ee746ff5a6bd6;a96a1fee948a06d083e9d7bf57480666bd1653f3;4dac942817175d54e581803c90d20209cb0752b2;df292306c38ea75a87b649b3bd1833cdde89b128;f73bf5bf32c20821b00c854e60b21cff6f6a8e56;f5779281d7d71ae5a9d2f6a7ddc57699293d28aa;7a50462b065cdfddb9f96730fa68706174005e33;ab539ed05d3fc88082d19e3c5829da1d167ca066;234fd0f30e731e7d96e713d041c84fdc3bea8299;09b60a85059b98b1f3df8646742cf086edc512ea;b0d11c4e598c4c683cecac6640fae6e5330ce1ed;1bab97c3a3e9dc5c9f6545753bbd6fc7b58f6e68;bbefe294e85cb153d4bbf417b24b4073e0ec6999;cb9627792a808bf3921d00b62173fba1a2d2dfd3",
         "MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation.;MicroRNA gene expression deregulation in human breast cancer.;MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells.;miRNAs, Cancer, and Stem Cell Division;MicroRNA expression profiles classify human cancers;Cancer genomics: Small RNAs with big impacts;A microRNA polycistron as a potential human oncogene;Small regulatory RNAs in mammals.;Combinatorial microRNA target predictions;The RET/PTC-RAS-BRAF linear signaling cascade mediates the motile and mitogenic phenotype of thyroid cancer cells.;MicroRNAs: a developing story.;RAS Is Regulated by the let-7 MicroRNA Family;Accumulation of miR-155 and BIC RNA in human B cell lymphomas.;A new mechanism of BRAF activation in human thyroid papillary carcinomas.;Detection of Overexpressed and Phosphorylated Wild-Type Kit Receptor in Surgical Specimens of Small Cell Lung Cancer;Kit as a human oncogenic tyrosine kinase.;Oncogenic protein tyrosine kinases;Human MicroRNA Targets;Fast and effective prediction of microRNA/target duplexes.;Genetic analysis of genome-wide variation in human gene expression;MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias.;Annual report to the nation on the status of cancer, 1975–2001, with a special feature regarding survival;An oligonucleotide microchip for genome-wide microRNA profiling in human and mouse tissues.;Reduced Expression of the let-7 MicroRNAs in Human Lung Cancers in Association with Shortened Postoperative Survival;MicroRNAs Genomics, Biogenesis, Mechanism, and Function;Prediction of Mammalian MicroRNA Targets;BRAF mutations in thyroid tumors are restricted to papillary carcinomas and anaplastic or poorly differentiated carcinomas arising from papillary carcinomas.;Reduced accumulation of specific microRNAs in colorectal neoplasia.;Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma;BRAF mutation in papillary thyroid carcinoma.;High prevalence of BRAF mutations in thyroid cancer: genetic evidence for constitutive activation of the RET/PTC-RAS-BRAF signaling pathway in papillary thyroid carcinoma.;Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia;Environmental and heritable causes of cancer among 9.6 million individuals in the Swedish family‐cancer database;Gene expression in papillary thyroid carcinoma reveals highly consistent profiles;Localization of a susceptibility gene for familial nonmedullary thyroid carcinoma to chromosome 2q21.;Papillary thyroid carcinoma associated with papillary renal neoplasia: genetic linkage analysis of a distinct heritable tumor syndrome.;The biology of stem cell factor and its receptor C-kit.;A gene predisposing to familial thyroid tumors with cell oxyphilia maps to chromosome 19p13.2.;Paraffin section detection of the c-kit gene product (CD117) in human tissues: value in the diagnosis of mast cell disorders.;c-Kit proto-oncogene is more likely to lose expression in differentiated thyroid carcinoma than three thyroid-specific genes: thyroid peroxidase, thyroglobulin, and thyroid stimulating hormone receptor.;Transformation of thyroid epithelium is associated with loss of c-kit receptor.;Systematic population-based assessment of cancer risk in first-degree relatives of cancer probands.;Expression of c-kit receptor in normal and transformed human nonlymphoid tissues.;RNA silencing.;M  ED  IC  A  L  SC  IE  N  CE;Loss of heterozygosity in papillary and follicular thyroid carcinoma: a mini review.;de la Chapelle, A.;Age at diagnosis Gender Race Diagnosis"
        ],
        [
         "25",
         "001ce484190e733e0d1dee181f9ee37fd0b15581",
         "https://academic.oup.com/jimb/article-pdf/32/7/301/34739283/jimb0301.pdf",
         "Biology,Medicine",
         "Journal of Industrial Microbiology & Biotechnology",
         "L. Beuchat,C. Pettigrew,M. Tremblay,B. J. Roselle,A. Scouten",
         "5992117,5613293,2054407528,2080517093,144590251",
         "47",
         "6.0",
         "2005-04-22",
         "2005",
         "ebdc8132e63badc8486ea08b463ca45bbac920f2;bb9a03d7fc726a5caff85716756521682fa33599;542676cd70a4a13b6520be34840b3928627ac1af;26b975dc9e3dd5605b024be8d0b29612cf80fb95;c587419aa127e80e61e17ffcf11ad30862a792a3;e9d641ae53747c724f9e3de913bec18162003673;715ce6488cfee5aa05a74d05da1bed03c00190a7;a87d00cbd3d138f74205878feea606bfc1653091;f8129a6f2927c0cf33b078fef7268d1ee5f5a66b;6b65eceb0ec96f7d472afa7491ebbd0637a456da;65e28bd82cb982bbce54f201121d6f5c47165db3;aedb3b17c9021bd5fdfc8e2446d030bfb58180ad;24279652e866eff900e765a39514b4679dbfa608;d3ce20ca3d60bfb72e91c641e34546c85e7bb4fa;2564ab2faeab37139d6f3b40dcfac765a5278ca8;f1dddee8f7a3f47eaeed12e25f132a4798252989;27a2ef7c94d7f65bfe7ee99741741e0c81e2a618;e06fe7341ef360c9e8482ad305f282823feb41e4;7a77c028f8a5cb55c53132c85630ff110701a9ff;4ce9ddba67cc1dc2117dce76d8df8a9b3b8d691c;025f4f1eb58e140f52a303fbf4446c714b496db3;06fad055a38bb1e8446bff362de92e2a5612c866;f341765d3914f817359a4ebdeeb831b17ade67a5;576523cc2193662b0cac3a498e490b0bcc1dc7a7;0296aa37d73332d74117909893b81293d11e7ce0",
         "Sanitizers: Halogens, Surface-Active Agents, and Peroxides;Mechanisms of killing of Bacillus subtilis spores by hypochlorite and chlorine dioxide;Bacillus anthracis: current knowledge in relation to contamination of food.;Peracetic acid as an alternative wastewater disinfectant to chlorine dioxide;Mechanisms of killing spores of Bacillus subtilis by acid, alkali and ethanol;Differential efficacy of a chlorine dioxide‐containing sanitizer against single species and binary biofilms of a dairy‐associated Bacillus cereus and a Pseudomonas fluorescens isolate;Reduction of listeria monocytogenes on green peppers (Capsicum annuum L.) by gaseous and aqueous chlorine dioxide and water washing and its growth at 7 degrees C.;A study on the effects of high pressure and heat on Bacillus subtilis spores at low pH.;Inactivation of Escherichia coli O157:H7 on surface-uninjured and -injured green pepper (Capsicum annuum L.) by chlorine dioxide gas as demonstrated by confocal laser scanning microscopy;Resistance of Bacillus Endospores to Extreme Terrestrial and Extraterrestrial Environments;Antiseptics and Disinfectants: Activity, Action, and Resistance;Sensitivity of Bacillus coagulans spores to combinations of high hydrostatic pressure, heat, acidity and nisin;The effects of various disinfectants againstListeria monocytogeneson fresh-cut vegetables;Comparative sporicidal effects of liquid chemical agents;Microbiological Control of Cucumber Hydrocooling Water with Chlorine Dioxide †.;Interaction of Bacillus subtilis spores with sodium hypochlorite, sodium dichloroisocyanurate and chloramine-T.;Bacterial spores and chemical sporicidal agents;Bacillus Species;Effects of Organic Acids on Thermal Inactivation of Bacillus stearothermophilus and Bacillus coagulans Spores in Frankfurter Emulsion Slurry.;Sporicidal activity of mixtures of alcohol and hypochlorite.;Ultrastructural and chemical changes in spores of Bacillus cereus after action of disinfectants.;A sporulation medium for Bacillus anthracis.;Sporicidal properties of some halogens.;Kinetics and mechanism of bacterial disinfection by chlorine dioxide.;Resistance of bacterial spores. In: Storz G, Hengge-Aronis R (eds) Bacterial stress;Principles and practices of disinfection, reservation and sterilization;Surface decontamination of fruits and vegetables eaten raw: a review. Food Safety Issues, Food Safety Unit, World Health Organization;These include:;Evicting an unwelcome tenant: anthrax"
        ],
        [
         "26",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/cncr.20947",
         "Biology,Medicine",
         "Cancer",
         "R. Kim",
         "2592641",
         "315",
         "15.0",
         "2005-04-15",
         "2005",
         "376109aaa3cbc518162072d892982ee7d7c61716;ce3c0b44cd2216676a66ed0c658d2bc298883f47;ef47428ff7b435d697f530965245854043545ab3;aeeddeb465b14955c0c207ae5d8facd8d4e01ce8;f172956a2b5badf538115971e667d34bebd87d22;9e4b1bab7acb0526340bd10134c672d3db8d445d;cdf7719d569d5b5ee447d329a25bbf19fdcc37c3;1c624d6499bd865abc6f8e400b24c8b79f6b620f;529a77664ace0111bcc3e9220770f8897ef023e5;c548c49f6baee6772d6e9fe3cbb9dde108a707b3;bf5f409029a90feff7df4cacdbd467d8d1eb00d8;6d6a4eef1ac55af590484c713f2f576c85c78d98;f357979269b1757c92d0c092be99730778c09d24;b3a2c655c5e01b71f590b2151cff8f00a02e1226;c45854aadb1d8e2e411e87a467f9f065f4866966;92f639e758c61f702a9c00cbea9be0976b20d894;53a50ae0268994aed5cf952d6c99043de7e35ebd;d37d31faa3a00280d21f5e1a648f98b66c8e3432;11b3fec8df3f80b2bec1488a6f0df6c4d8441c71;799ab82ec856050d460526143b7d0718b96e14cb;b3fae1ae6bc52cfe558a594ba83f5dc8a700f43c;51525a9c8be7e63f959ac2bfa3158f9db5b10d7c;b5672e0714e39e7bda99ea73b0bcebfdc6b05571;6da399d8f4d3460dde33aa226c27b631d47734ba;e83e6c40a82928c05944bc56ab2ae4a7d8ccf302;05d42e39ca41624d718474dffb3bcdfd01359b4e;a675c38d04171783d9a60886cc54ece593194dfe;bd437570db7af2cab4d48a6ce8301493ced7d301;045bc283c609699990dff04aa9b4514027dfceca;ad8cba15117d0a7073e3638b8a22daaac3588483;0bb77a1432b8d5752838df435480e44e3c50fff0;2ba0fec7e8cf702393a1382ca856855a7837556f;0291f1c9ff0da520e91419ae2db08999ee399725;e54cab9b65df38aa1f0a9519923c159b51fb68b6;5df427a0bafeb5240b6f577879c0de8e63ee7417;bd0da4411086bad78115eaf6dd0b45004209ff44;a2b0a9954056c4b757c828e6b16144251a158da1;7b6d94906866ab52104a903207582f2436ea697d;4fe6cc56b37705ac433ce4cd16853d5fe9fceeac;3c49e9197ceb2728b27e913342fc321637cf6238;3c8a2cf8e6c03282842828cd36f189d835c21f6e;fd0960e7ef910dc32ca134e477f05dc9818f6351;1b7d30f14e6b49cc7f21c2392bd968bf0cfe625f;382eec2dc4340b1331798554b7a49218ccc478e3;e806d85d49e988ee33f5e36bffd2a984449cda39;1b4b00fb7e6f96fe6e94b2d2e274ee431d4796f3;27930771ff138d36d235401e67bc181c075ab02f;43325ff8d344de359f0f6ccc23ba171a7cc26748;65e6570ac8da2eab3e549c8d0fa921e6b56a442c;803b1672ed76dc13f50b020abd30e76a010be24b;75f6318d79eb3ef37dc5e6ef5069864829e4e1d0;8b3b03b3843bc2489aea05e5bd575719abdc31c3;367f3536f0b382b32d38034fcf9b2bc30304d6c6;6a0effba2107618e090d29a519bc8cc0d95574bc;2368dde26aa4ed09380222cfc8d8081de22ab899;4feb355910a6276d6bb1a790d1b76315bae25072;55617d0d12a80e300ee19353cb9234189d563685;6b5e978b6ee2b44ed9651d1a391eff867f9582b4;4af3826564c80d0e09c2c58b4919a6839dd544bc;c8c7d9d1cdefc34132dd7af4cf41042bc8142c77;9bb678d996f92800823bbb881c51c7ae5d1d6fe7;f50232e9fd7cb6b62e70966fc65676393a2ff40e;6a064731895d6c8b46146a46dd99be1f8fba54dd;607f07d507527a3765bceb8483c69486e1ff86a6;d211b8513a89e4274e69a40062724877c389d783;3a40fdacf4cd8ece6443f92033e8f6539ce19720;83c29f099cb279c1bee03261845205562f74fa29;be92bf9b710be989ba332bef5013ae49be45fb60;c2f7cf5511d33da829564a022e41742fc1255477",
         "Neutralization of Smac/Diablo by Inhibitors of Apoptosis (IAPs);Aging as a catabolic malfunction.;Apoptosis, autophagy, and more.;Dynamic evolution of the adenine nucleotide translocase interactome during chemotherapy-induced apoptosis;Bax is an important determinant for radiation sensitivity in esophageal carcinoma cells.;Apollon ubiquitinates SMAC and caspase-9, and has an essential cytoprotection function;A Phase I Pharmacokinetic and Biological Correlative Study of Oblimersen Sodium (Genasense, G3139), an Antisense Oligonucleotide to the Bcl-2 mRNA, and of Docetaxel in Patients with Hormone-Refractory Prostate Cancer;DAP-kinase as a target for drug design in cancer and diseases associated with accelerated cell death.;Endogenously released Smac is insufficient to mediate cell death of human lung carcinoma in response to etoposide.;Smac induces cytochrome c release and apoptosis independently from Bax/Bcl-xL in a strictly caspase-3-dependent manner in human carcinoma cells;Nuclear and mitochondrial conversations in cell death: PARP-1 and AIF signaling.;Ceramide-mediated Macroautophagy Involves Inhibition of Protein Kinase B and Up-regulation of Beclin 1*;JNK promotes Bax translocation to mitochondria through phosphorylation of 14‐3‐3 proteins;Multiple cell death pathways as regulators of tumour initiation and progression;Apoptosis defects and chemotherapy resistance: molecular interaction maps and networks;Changes in Therapy for Solid Tumors: Potential for Overcoming Drug Resistance In Vivo with Molecular Targeting Agents;Molecular therapy for peritoneal dissemination of xenotransplanted human MKN-45 gastric cancer cells with adenovirus mediated Bax gene transfer;Mitochondrial protease Omi/HtrA2 enhances caspase activation through multiple pathways;Apoptosis pathways in cancer and cancer therapy;In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker.;Endonuclease G is required for early embryogenesis and normal apoptosis in mice;Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor;Phosphoinositide 3-kinase accelerates autophagic cell death during glucose deprivation in the rat cardiomyocyte-derived cell line H9c2;Endostatin induces autophagic cell death in EAhy926 human endothelial cells.;HSpin1, a transmembrane protein interacting with Bcl-2/Bcl-xL, induces a caspase-independent autophagic cell death;Direct Interaction between Survivin and Smac/DIABLO Is Essential for the Anti-apoptotic Activity of Survivin during Taxol-induced Apoptosis*;Omi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP) irreversibly inactivates IAPs and facilitates caspase activity in apoptosis.;Transformation-selective apoptotic program triggered by farnesyltransferase inhibitors requires Bin1;The CD95 type I/type II model.;Amino Acids Interfere with the ERK1/2-dependent Control of Macroautophagy by Controlling the Activation of Raf-1 in Human Colon Cancer HT-29 Cells*;Induction of autophagic cell death in malignant glioma cells by arsenic trioxide.;Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs.;Chemotherapy enhances TNF-related apoptosis-inducing ligand DISC assembly in HT29 human colon cancer cells;Molecular response of human glioblastoma multiforme cells to ionizing radiation: cell cycle arrest, modulation of the expression of cyclin-dependent kinase inhibitors, and autophagy.;Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?;Current status of the molecular mechanisms of anticancer drug-induced apoptosis;Hypoxia and Nitric Oxide Treatment Confer Tolerance to Glucose Starvation in a 5′-AMP-activated Protein Kinase-dependent Manner*;Mediation of Poly(ADP-Ribose) Polymerase-1-Dependent Cell Death by Apoptosis-Inducing Factor;DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death;Bcl-2 and Bcl-xL Inhibit CD95-mediated Apoptosis by Preventing Mitochondrial Release of Smac/DIABLO and Subsequent Inactivation of X-linked Inhibitor-of-Apoptosis Protein*;Increased Ras expression and caspase-independent neuroblastoma cell death: possible mechanism of spontaneous neuroblastoma regression.;Paclitaxel triggers cell death primarily via caspase-independent routes in the non-small cell lung cancer cell line NCI-H460.;The Apaf-1 apoptosome: a large caspase-activating complex.;Apoptosis-inducing factor (AIF): a novel caspase-independent death effector released from mitochondria.;Identification of Omi/HtrA2 as a Mitochondrial Apoptotic Serine Protease That Disrupts Inhibitor of Apoptosis Protein-Caspase Interaction*;The Tumor Suppressor PTEN Positively Regulates Macroautophagy by Inhibiting the Phosphatidylinositol 3-Kinase/Protein Kinase B Pathway*;Autophagy delays sulindac sulfide-induced apoptosis in the human intestinal colon cancer cell line HT-29.;Bax translocation is crucial for the sensitivity of leukaemic cells to etoposide-induced apoptosis;The target of rapamycin (TOR) proteins;Proapoptotic BAX and BAK: A Requisite Gateway to Mitochondrial Dysfunction and Death;Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function.;PTEN controls tumor-induced angiogenesis;Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis;Bcl-2 down-regulation causes autophagy in a caspase-independent manner in human leukemic HL60 cells;Molecular Determinants of the Caspase-promoting Activity of Smac/DIABLO and Its Role in the Death Receptor Pathway*;BNIP3 and Genetic Control of Necrosis-Like Cell Death through the Mitochondrial Permeability Transition Pore;Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments.;Distinct Classes of Phosphatidylinositol 3′-Kinases Are Involved in Signaling Pathways That Control Macroautophagy in HT-29 Cells*;Induction of autophagy and inhibition of tumorigenesis by beclin 1;Autophagy Is Activated by Apoptotic Signalling in Sympathetic Neurons: An Alternative Mechanism of Death Execution;Cloning and genomic organization of beclin 1, a candidate tumor suppressor gene on chromosome 17q21.;Oncogenic Ras triggers cell suicide through the activation of a caspase-independent cell death program in human cancer cells;BIN1 is a novel MYC–interacting protein with features of a tumour suppressor;Ultrastructural and immunocytochemical characterization of autophagic vacuoles in isolated hepatocytes: effects of vinblastine and asparagine on vacuole distributions.;Studies on the mechanisms of autophagy: formation of the autophagic vacuole;Vinblastine-induced autophagic vacuoles in mouse liver and Ehrlich ascites tumor cells as assessed by freeze-fracture electron microscopy.;The morphology of various types of cell death in prenatal tissues.;Inducing cancer cell death by targeting transcription factors.;Mitochondrial dysfunction is an essential step for killing of non-small cell lung carcinomas resistant to conventional treatment"
        ],
        [
         "27",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2005.04623.x",
         "Biology,Medicine",
         "The FEBS Journal",
         "Ronja Tasler,T. Moises,N. Frankenberg‐Dinkel",
         "5801680,14847601,1399196272",
         "104",
         "6.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "28",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "http://www.jlr.org/article/S0022227520340323/pdf",
         "Biology,Medicine",
         "Journal of Lipid Research",
         "C. Villeneuve,M. Gilleron,I. Maridonneau-Parini,M. Daffé,C. Astarie-Dequeker,G. Etienne",
         "48980754,144505377,1397966565,5407662,1401604315,47231979",
         "107",
         "6.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "29",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D4334A1A0F9096B08FAE923BAB0CA36D/S0029665105000194a.pdf/div-class-title-insights-from-the-developing-world-thrifty-genotypes-and-thrifty-phenotypes-div.pdf",
         "Medicine,Biology",
         "Proceedings of the Nutrition Society",
         "A. Prentice,P. Rayco-Solon,S. Moore",
         "3674816,1401327164,48773910",
         "205",
         "7.0",
         "2005-05-01",
         "2005",
         "bc9ce03c2c4c174bd6c3b1d0be8900145edff1db;06acdfded5f54064fede109f07d9f36d8873834d;78d45a925322ae5987536246dd9a512b3d791ed7;c28997f8a141c0176ae7f5c06db2818ad40e220d;c57f0bf1bf34631a10d5f1d27191b88ed2f9b687;b3211005e2be14d595907435efc89659f76fa5c8;b02f1a164c91647b502be658d3c0e9e4c1f57f2c;27396abd2923770a9663d7ea65aa7a04ca6efc83;b23eef326baea71254d7c63656e97f638b9192ae;c137226bbdf92efc3da973dfd9e87a19dd042d68;eefefe28334b759b8b53e25caefca368a161dc98;2ae29359abae064e93f7d910e0c25aed49a88f4e;89c1180a10cd2d4f6b69b05faabcc6bb87929593;9ebf1a1f44e3658b2dd389b0526829653d05f0fd;de71e00c157f2a6252d6332c5a1e6063e6c4158f;f624515d506367c34202b1ccfb3a62566caf54b4;2f5d63fc5dd3fd93d6c8632642e9799cdb2dc667;8b025f19fc6c3a2565f2753b3d22219f0967e756;3b0189be73140b24df1fd28a3633c5debad6a958;4e4e8e7c72f63610dc20f83f07c02d220c438373;4a7f9ea4dd88c1d1793b79539dc887b93ee88c1c;a587a2b28dd5e8577fbd4d0a98dbf07506df40d6;d475e72c9d720f611b3728771f23082782877ebb;57784681819cecc1028626455fb79c57fe1030fd;95acbce1e13dea72bf4ecceef8070cd632c1905c;a1c3dfb03a261423e7f757718e61af931c1ac36d;0f4caae8c99e0d72292d71addb310524aec78c0d;a00002c79570f71907233f59273571ea44e67a27;7cfb934057f219e58da9c815e49c687cd5f3e305;c61d6adf94da35fa7b1ebd4332607ece26b12cff;0b69c9ba93b0e2d5d10c6b81cb65b179b0ce0e0e;00462a440fcfd0497a8769f2e3e94e40c50929c1;545f324e5db2615ddae40d7c685fd96ccb624f5d;e50748d4ac19be31f2f1ffe6591ef8b61a3bc136;864414f69552e3955b428c875a040785e2351d83;b7da62272a15b2eab222feeb806974255ffc6eb6;5c0dde78840d8c1c914f59179afc4c0993ce60fa;a37ef5d0f1eb4d9f2ee2c5e1ea49b3bceb1ca8eb;e946add8a70509c349ae15054c3c1052a981165d;5363b5010f319dc63abcdbfd1c868daebd725306;1ec845128a00e13b08eea726e8e8d784f844dfbd;795430c208a98348adb6fbbe7a962453d149c787;7454fa4f1eb0acfc56c67a288ac61bdebeb35970;8b6c6100fc489b4e951b9ee4f33f743d14b321e6;8246e8f6dbd713319cb843bf7e4990ad29df914e;f70b5135140b82250b8ccbcacde4973320f57b05;6ce812c3d9eb7a671e1a987517564a2b739870be;7d9c2c4768b5de3a533aa002b359449fabcdc151;b212cd8394f5fec259916490c9cf7668ed72160b;bc2e45abdfc013d33d5f1ba5c0934a8d4cc3574b;51c82b8dfc9ac7e33c7ecfaf023ddfe5c507de26;5522565260f1bd56280a2f5c50c5496b4ccad736;fb2b7baf210fb38b4a0501b2f5cd200abce4bb54;6a03fe8f413a6d83198f817eeaf8e1f27d9912fd;f83f5704a37c5776d21c4e950a45c27aaaecaac3;1036fd9c08f2f55be8974e20253875fd60ac16e0;323b16f39deeceae6fb5b285a9bd48c3e069a783;fc1819f465702c13883870784f9508e4054d00dc;935ec7c48c94bf948f1fbee7eb961255aa9eef24;c530eb353abee06e3b3cc5eea087d28aa0a800bb;6f38ee886f39daa656b1ac9ceb8a3c0785e100eb;d72154ce1a31048035a9ed91860acd6c895f735c;b5b8c35ff6c516a6106cfa930490e0779e8c43a4;d7d39f115e1b9b5482867486fcf2184576e8b72d;2a308549d0abed485e230c108a8bbe9338e2c926;90644eb817116ef816566ac87e3cc4b2804aaf26;94bdea56b64a2a3269c927af53a5f70a0abf389e;c49d0548da17d0e28c7d48fbfe7154cf5d523d7a;85d0d6037f14ad6bc592310d1f3350d87d424004;b79916ed1461af81a0559432924f559a13032c6f;9b47fdd07197bf012ba0b0bd9153b9dd70981bc1;8861cd3593c4334c59f75a859df8652cc4bb688d",
         "Fetal origins of insulin resistance and obesity;Variation at the insulin gene VNTR (variable number tandem repeat) polymorphism and early growth: studies in a large Finnish birth cohort.;Apolipoprotein E polymorphism and fertility: a study in pre-industrial populations.;Opposing influences of prenatal and postnatal weight gain on adrenarche in normal boys and girls.;Maternal-fetal interactions and birth order influence insulin variable number of tandem repeats allele class associations with head size at birth and childhood weight gain.;The human obesity gene map: the 2003 update.;Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA;Intrauterine factors, adiposity, and hyperinsulinaemia;Global haplotype diversity in the human insulin gene region.;Birthweight and perinatal mortality: paradoxes, social class, and sibling dependencies.;The double puzzle of diabetes;The epidemiology of adverse pregnancy outcomes: an overview.;Neonatal anthropometry: the thin–fat Indian baby. The Pune Maternal Nutrition Study;Adiposity and hyperinsulinemia in Indians are present at birth.;The role of uncoupling proteins in pathophysiological states.;Racial differences in the relation between uncoupling protein genes and resting energy expenditure.;An overview on the nutrition transition and its health implications: the Bellagio meeting;An adipose-specific control of thermogenesis in body weight regulation;Insulin gene variable number of tandem repeat genotype and the low birth weight, precocious pubarche, and hyperinsulinism sequence.;On the importance--and the unimportance--of birthweight.;The thrifty phenotype hypothesis.;Obesity and undernutrition and cardiovascular risk factors in rural and urban Gambian communities.;Glucose, insulin and lipid metabolism in rural Gambians exposed to early malnutrition;Uncoupling proteins: functional characteristics and role in the pathogenesis of obesity and Type II diabetes;Uncoupling proteins: their roles in adaptive thermogenesis and substrate metabolism reconsidered;Fires of life: the struggles of an ancient metabolism in a modern world;The nutrition transition and obesity in the developing world.;Thrifty Genotypes and Phenotypes in the Pathogenesis of Type 2 Diabetes Mellitus;Uncoupling to survive? The role of mitochondrial inefficiency in ageing;The Fetal and Childhood Growth of Persons Who Develop Type 2 Diabetes;Energy adaptations in human pregnancy: limits and long-term consequences.;Interactions of perturbations in intrauterine growth and growth during childhood on the risk of adult-onset disease;Insulin resistance syndrome in 8-year-old Indian children: small at birth, big at 8 years, or both?;Transmission ratio distortion at the INS-IGF2 VNTR;Apolipoprotein E (APOE) allele distribution in the world. Is APOE*4 a ‘thrifty’ allele?;The insulin gene VNTR, type 2 diabetes and birth weight;The thrifty genotype in type 2 diabetes;The INS 5′ Variable Number of Tandem Repeats Is Associated with IGF2 Expression in Humans*;Association of the INS VNTR with size at birth;The Great Famine: Northern Europe in the Early Fourteenth Century;Bergmann's rule and the thrifty genotype.;Imprinted and genotype-specific expression of genes at the IDDM2 locus in pancreas and leucocytes.;Susceptibility to human type 1 diabetes at IDDM2 is determined by tandem repeat variation at the insulin gene minisatellite locus;Biochemical control logic and the metabolism of glutamine.;Type 2 (non-insulin-dependent) diabetes mellitus: the thrifty phenotype hypothesis;The possible role of glutamine substrate cycles in skeletal muscle.;Maternal weight gain, infant birth weight, and diet: causal sequences.;Role of triglyceride-fatty acid cycle in controlling fat metabolism in humans during and after exercise.;A Polymorphic Locus Near the Human Insulin Gene Is Associated with Insulin-dependent Diabetes Melliitus;Poverty and famine: An essay on entitlement and deprivation : , Oxford, New York: Oxford University Press, 1981. xi + 257 pp., index. $17.95;Malnutrition and fertility.;Poverty and famines. An essay on entitlement and deprivation [incl. case studies of famines in Bengal, Ethiopia, Sahel, and Bangladesh].;Famine in Tudor and Stuart England.;The high prevalence of diabetes mellitus on a Central Pacific island;Famine and Human Development: The Dutch Hunger Winter 1944-1945;Faminee of history and of today;Diabetes in American Indians and Other Native Populations of the New World;The determinants of growth and form.;Review lecture - The determinants of growth and form;Diabetes mellitus: a \"thrifty\" genotype rendered detrimental by \"progress\"?;Biology of Human Starvation;A critical evaluation of the fetal origins hypothesis and its implications for developing countries.;Lack of support for a role of the insulin gene variable number of tandem repeats minisatellite (INS-VNTR) locus in fetal growth or type 2 diabetes-related intermediate traits in United Kingdom populations.;Differential reproductive efficiency associated with common apolipoprotein e alleles in postreproductive-aged subjects.;Eating, exercise, and \"thrifty\" genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases.;The insulin gene variable number tandem repeat class I/III polymorphism is in linkage disequilibrium with birth weight but not Type 2 diabetes in the Pima population.;Floods, Famines and Emperors. London: Pimlico;PPARgamma, the ultimate thrifty gene.;Fetal origins of adult disease-the hypothesis revisited.;Genomic imprinting of fetal growth;The Demography of Famine: An Indian Historical Perspective;Biochemical control logic and the metabolism of glutamine.;Agriculture’s mixed blessings;The Third Chimpanzee;Determinants of low birth weight: methodological assessment and meta-analysis.;Origin of Species"
        ],
        [
         "30",
         "00206ca6a32791adcca5effdfed6eecd035075f4",
         "http://www.vision.caltech.edu/malaa/publications/atiya05sparse.pdf",
         "Computer Science,Medicine",
         "IEEE Transactions on Neural Networks",
         "A. Atiya,Mohamed A. Aly,A. Parlos",
         "1686506,144933112,1838904",
         "35",
         "1.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "31",
         "0020d5bdfc5f13026e7a1aca78e94724e8342b62",
         "https://rupress.org/jem/article-pdf/201/6/881/1153938/jem2016881.pdf",
         "Biology,Medicine",
         "Journal of Experimental Medicine",
         "K. Turcotte,S. Gauthier,A. Tuite,A. Mullick,D. Malo,P. Gros",
         "2106108767,46813877,6563462,145705716,144605952,144829585",
         "109",
         "6.0",
         "2005-03-21",
         "2005",
         "",
         ""
        ],
        [
         "32",
         "0022e70878c2f2a7c3c679a3ef3160a5cb803e20",
         "https://academic.oup.com/mend/article-pdf/19/11/2696/10717141/mend2696.pdf",
         "Biology,Medicine",
         "Molecular Endocrinology",
         "Jing Huang,Xiaodong Li,C. Maguire,R. Hilf,R. Bambara,M. Muyan",
         "2145740413,2118899174,12605892,4551110,5522510,6431895",
         "45",
         "0.0",
         "2005-11-01",
         "2005",
         "",
         ""
        ],
        [
         "33",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "https://ir.vanderbilt.edu/bitstream/1803/13704/1/ms-thesis-manish-kushwaha.pdf",
         "Computer Science",
         "IEEE International Conference on Mobile Adhoc and Sensor Systems Conference, 2005.",
         "Manish Kushwaha,K. Molnár,J. Sallai,P. Völgyesi,M. Maróti,Á. Lédeczi",
         "2181068,2065280567,1717457,1740862,1692190,1721042",
         "70",
         "5.0",
         "2005-12-12",
         "2005",
         "813ffc490393534e9244924dcd0e4337b4bf5fa0;571eab8ea81d7c54bf5454c67c2d28f6aac3f086;35b225cf1cb2ff030eff1ffd9c554a87418b16ee;5e1c7fcad70403f2da76849eaf4551143c008116;a4ac5b5e63942b95c5ebbf51019869950282212d;cd608034cca0eb48676f3fc4f6082e4a504df42f;25a5f84ee5164162ad04e0a1d69c35525ce556de;26d7477c2f05594eb98a191826dcbdb836c9fb45;8b2815e893b0d4a476f27b15b69cec7f2a43d8e8;2cdd6e0dd343a94480adfeba247d3f33c429113a;d7d1ba245d7397c0cead0191c08aeb0da6fb2316;24741a5d686df667874f43d79e6359346444cd3e;6d5239e7d50c83dad3621b65a447ba8191b7694c;be8ac44dd2bb5f5d23cdb902dabd332969222c9a;869b5434530fd46dc84ca30661ebf8b3a697600b;2d4ad6826ed95a6b17709278b9f2b6c729b1163f;fba126954f354e24cc57d1159dfe7d5ae16d11ef;234efad9a8efb5d417743c70fc9f2d0ca6eac97f;6d6e4c92401f60682f00777238d4afeb729340bc;4f264407361a47c81695a03a8d96c078e1f588b0;0e4ade87ce468690926f153bc413d53a5034a9e4;22188b246b50e273b2f8de308c091d0c082aca03;15f705d41cfb39860c33d15435defaa4c6469d29;93172a2fb5c410498079361e806e9c43549c8d2b;fb5232fe382f4c801a0a0ecb4a6e92dab0f72889;fa9d7cdee95b6e780b5006606a8a96cdd62417d8;ba062f1deca36fdc287609771e47a92d26973ed9;0115f7ae387b4116acaafe84176f38bb2e7bab6f;96aa4d55a3870193b23c9c5d3b838e2be6f17c33;b2fe77a389e749359f12ba483664429e11182f62;46a1dbf20cd13dc3204eb0cc35fc34aaa3e2a70c;f5ed53760000b1851c10734f271a97b7017ecfb8",
         "Resilient Localization for Sensor Networks in Outdoor Environments;Mobile-assisted localization in wireless sensor networks;Sensor network-based countersniper system;Robust distributed network localization with noisy range measurements;The flooding time synchronization protocol;Localization of wireless sensor networks with a mobile beacon;Smart sensors to network the world.;Range-free localization schemes for large scale sensor networks;The n-Hop Multilateration Primitive for Node Localization Problems;Ad hoc positioning system (APS) using AOA;Mica: A Wireless Platform for Deeply Embedded Networks;Middleware challenges for wireless sensor networks;The bits and flops of the n-hop multilateration primitive for node localization problems;Wireless sensor networks for habitat monitoring;A survey on position-based routing in mobile ad hoc networks;Robust range estimation using acoustic and multimodal sensing;Location Systems for Ubiquitous Computing;Smart kindergarten: sensor-based wireless networks for smart developmental problem-solving environments;Exposure in wireless Ad-Hoc sensor networks;Dynamic fine-grained localization in Ad-Hoc networks of sensors;The cricket compass for context-aware mobile applications;GPS-free Positioning in Mobile Ad Hoc Networks;GPS-less low-cost outdoor localization for very small devices;Embedding the Internet: introduction;Acoustic Ranging in Resource-Constrained Sensor Networks;IEEE Computer Graphics & Applications: Projects In VR Pervasive Pose-Aware Applications and Infrastructure;VR Pervasive Pose-Aware Applications and Infrastructure. IEEE Distributed Systems Online;Range-free localization schemes in large scale sensor network;This copyright notice must be included in the reproduced paper. USENIX acknowledges all trademarks herein. Robust Positioning Algorithms for Distributed Ad-Hoc Wireless Sensor Networks;The Design of Calamari : an Ad-hoc Localization System for Sensor Networks;Complex Behavior at Scale: An Experimental Study of Low-Power Wireless Sensor Networks;Convex Optimization Methods for Sensor Node Position Estimation;SpotON: An Indoor 3D Location Sensing Technology Based on RF Signal Strength;The Cricket location-support system;Global Positioning System Overview (last modified 5.1.2000);Numerical Optimization, pages 250–275;Theory and application of digital signal processing;If both | µ i | and σ i are large then again location might be affected by echo and we follow the same procedure as in case 2 above;If µ i is close to zero but σ i is large then either the range deviations are spread around zero or few large deviations caused σ i to be large;If | µ i | is large but σ i is small then all elements of ∆ i are large i.e. the node location is deﬁnitely incorrect"
        ],
        [
         "34",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "http://archive.nyu.edu/bitstream/2451/14996/2/Infosec_Book%2bWattal%2bTelang.pdf",
         "Business,Computer Science",
         "Workshop on the Economics of Information Security",
         "Rahul Telang,S. Wattal",
         "145961354,2628487",
         "80",
         "3.0",
         "2005-02-01",
         "2005",
         "",
         ""
        ],
        [
         "35",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-95-1423",
         "Biology,Medicine",
         "Phytopathology",
         "C. Silvar,J. Diaz,F. Merino",
         "6248808,152435507,145620908",
         "63",
         "1.0",
         "2005-12-01",
         "2005",
         "",
         ""
        ],
        [
         "36",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "https://europepmc.org/articles/pmc1069588?pdf=render",
         "Biology,Medicine",
         "Molecular and Cellular Biology",
         "T. Adachi-Yamada,Toshiyuki Harumoto,Kayoko Sakurai,R. Ueda,K. Saigo,M. O’Connor,H. Nakato",
         "1399592175,6637499,3515490,34604795,33751317,7615853,3971771",
         "23",
         "1.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "37",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "https://europepmc.org/articles/pmc2522321?pdf=render",
         "Medicine,Biology",
         "Journal of Proteome Research",
         "K. Anderson,J. LaBaer",
         "2700838,2112063",
         "311",
         "12.0",
         "2005-06-18",
         "2005",
         "a5fec970c69680e7d7ec9fe90b057291b4e7773d;8ef30a4dcb0ef1af8522d2a4ac3922457ba62a8c;733b342cac9c3cf6fbb70715bc3ecca5b730b349;c3c2808e82d98a4159cabcad3802dd7dc1705f7f;58ee959e3fe9a781bf59ab64549810149f593474;0069e9285f277056500448265d34b03a44372a08;a9d867d185ae257493226d2a53a94b7b682328a2;13bb6ea8a95a04aff56f95a013c3219d3cb57ff3;c4130add7c4c78affcff33f330dec1019373ec22;a982595680c8bcce7fcea5f16ac98e00e1113d2a;13d7452b1f93ae19f9db16cbe60f681616d02198;6e2c87806883391644aaec82861a2d4e64d3a503;7cb9f8e54e2786196b7d0f7178af3e32a66a294f;405a39a2fbd43b5bb6a73f12e6d880a38c0ec469;00ce84e1bc0fb55d3e4ac1eafef11137b842b674;94cbbf1805238625054ddaee315edbe98db15b99;e37bd6ce0c355fbe970a72b5d5ab2c61e76f1e31;7933e84c6162015607118f650818a04029a123a9;1a9168875ee9115b0130a978779867d6c86c164d;d14f3e86ee66258fe9e194939b2eff10a26deac8;251328149f10c2264a5b81e1e2163518d83e7af2;8458139de08cf9af5a8a3db11f9f56b641e0f9ae;3dea83a9f695382d933eb3cd5e89df303b16383c;0c70af8c234cad6a3716ec219fde55a52bca5d8c;96cb8f7f09a8fde293b5675da621836cd4a25442;17c551aca1c09efe7308f3a370e14b6f43eed293;2ce4c74262c38b472473cd089b3f6e1546ba989b;617018364b7ad94d15b34c751700371258488a24;363c94c7ab5aa4baebc089c8a6697c2d6188f054;b9cd2802cfb0e4b3d9c01132f470a2daced6a6b0;9aecd5b116595b1355f5dad17c919dcf448a4d6c;49de046c674e6b1be12427b81126f921c66e06e3;09e506bad5617ee5c1c9fcb2fc1baff71d33c898;1de2e10678cd99ffc6c12a7c80b45300c1f74b9e;9411bbe9e8ba032141dbc5c1904c65b8dce131bd;2b7da19d167b41357fa323a83fec7f69db36a6f6;6285b7e9f8e7cdf299920247ca0e0e14b37b1e8b;640a97b4aeb462b135971ed4acc2a243b55f6b4c;13cadee9e67644d3729d471bc3757c70e9fe173f;ad7be2b4e10846bcd85bdbf69c3e6652d50af488;c0adf2095391d71f8e92eb33df92c66b45ff51f1;48d6fc3223842c715e80bf6096c43cb3a8730e76;02074b7a8b2742ce30eecff1fead09b735be7b58;576afd62b7e1da025747950bfc0031ac070e14d9;901bde6f666a8933f2a1631a28b3c2e85f552cd4;d18a6692c38fa5c91bbe586aaca0d79f7d73ab1f;7d56a74dd196c2e81415b5d95a55b232a60d52f6;c5e573e73615df34b8e755dcd37c71f51a2aef8b;d36b4e618097ff1f97fb54e4fe8338e6539b9419;906df3ac3c0efc4ae49abb03eccb8a1a56eb6d8d;cacf432fbf7361cfc1b5a21e7653f4abeeb608bc;89a91f216ed67b56dbd2e897888fb03d3b7689b4;b8d5884927255b8a1ee8666d42dce3c7d8edc8bc;adc0cef49de7640091c30c64b5b0971fa15f8cec;77c7be9fba69091cdd99b89ae78bc1224eb6427f;296a0473a4537088e01704a8af41c749916998de;dc2aba9d4f2ca0d1a87e3fc0c3d5dbd89347657f;45e8b0e8aba7bce41416d9dc50f1175b8e546537;faf3728220c32d50d6f6311096ec3d70b0efa2af;417636fc2fa5a79d0a9e151ca2fadf4d896c3a6a;06f479de8c40c51ccbe591c004ee358beea7e3cc;d41eb0fce92491576a05850c935f894bbf09bde8;75521ab6ea29e022f51c4ed91382365836db205e;1690493dc16a64b7103da134383d970e407576af;24b56291be5bfeec9b69ecda07e5bc8167b4b326;a566936accc6ba41396d83ab68232ad0083689ee;d028d2e7b78ceb3103cb6e51ac8866e32ca28952;a8df84e42991b3fe1eac759a9491d542673b7c12;ca6ac75d2408d9fbc3ff49e8d62341821574337b;f26551154538a2f1d5799b9540b24cb74fb89763;61dc4870984aa539428bd0fea54fda55149f0eeb;e92fe6ff345e23d95f5d4c263ef2b66f1cbaa947;cf9c0737484df260b4407aceca30c6dbd7e0c08e;24aacae008a93fc5f912537d90b3808da25245bd;1ca341b8f40b748249717f6b26db00517a65645b;9649c703b781c322fad5f5d2e44af14460049978;e29c64ec56f634049c8931616bbc34807e53ccbe;b76586c209648e3f66a48b075a5a3b85b780c8e2;6783882e887d1fd25d84d4ef22b7c4fb6d5982e6;2bc35ccc858ecff282da44ce7c971b3efe3f6edb;1557de0758d8e90f89806d5aac1e02a788941a0e;eb61c808a6b210f8134d3c3ecbb7eb4cec0df4a6;49790e49ee605e518c6c164d6dcd3709b3b3d366;4bd9e587ccb7e702f9b83ce79e9a14d5889a365b;0a7e447ffa5921c3cefbd0a4110d0b73fa24fae4;cc2f46e6bdf0681fd05b7b5d41a74f52aa131813;e4a27eac6551682770c0af50db31723ad34edd55;6ea42f29588a3699a8daaf03e613cdc385a07a98;6984707cd20cab11a337aae74d31aa28f28506f1;bfd9b207d5df59cb51ee9ed7cd70749a9abcb154;16f54f42f2b97dc6548eac350b67227bf4f24715;f09ce709ee7be9223c98fe2729ac39e3654eb432;6046d06aa295bfb20be8e952b5124375ac25d5bd;eec606cec01c5c86a6195abd33dcf9178e4c9dd6;ce54d815b5ea33a108a04557a2a5855c259bd5de;9d4028799d9b9b282a190b3e96498d824aa0fe31;2a0b10e367b0786d4600b336aefc5a15072e5d3e;31cc5c4ab5f488e2a50b692de8264c4e5886e3bc;477e6297459532d8a651d8947d471b69a10c0bee;a5f1d509eb95129272afe5db38d1fa08afc1cf82;0b3b00a3c3dcd8577c6594781da13e90d11a25da;1e672c6d143b318e918a8424e1026a9433fc5952;54e66e26f22f9573b1ab00ef154e4b9ec97662fb;df9f47703c6299a257cd25ef20c82a66d39fd355;dcb0df34754e316e80826416031bc497c2eec162;341d10aacb81f6688bd0eb8566f6b76b1a8ad7b3;fc612a59bce380077f4cb8080bf4f7dcfb3ee494;f7c6c19558fbfec06849749ffa2f18fc4de63455;8c140318c3811d98a91afc7e3bbec42f62dee579;432d239dac42a5263ed2010313d582cfc6c32018;478504ff6ae608efcfcc5d125196cd8e6df85d3e;28a9f8b464aeda844f2233a2da2a00ac86597c56;7e520b5bdc68f2520e402593aa7c5b56ef1a3ad4;33c9970d75ac2c8b0561cfb2c3a9db7f319ada71;c0e41bffc9acee371c0365d1a79d37475c27aff4;238954a402bc9a038f88f0cbde3d801628f64e59;df36002142e55b86221f4ec5ac2722edfb86f73c;c28b6762b25ff1f624cea7a775d7c6a2a4214832;61e58171fd3059916c2e021dee9683e5bcb901b3",
         "T Cell–Dependent Antibody Responses against Aberrantly Expressed Cyclin B1 Protein in Patients with Cancer and Premalignant Disease;Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles;Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast Cancer;A phase I trial of DNA vaccination with a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate cancer;Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant induces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans.;Self-Assembling Protein Microarrays;Protein microarray technology;Minor Histocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem Cell Transplantation;An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in association with HLA-DR.;Development of natural protein microarrays for diagnosing cancer based on an antibody response to tumor antigens.;NY-ESO-1 Expression and Immunogenicity in Malignant and Benign Breast Tumors;Clinical proteomics: revolutionizing disease detection and patient tailoring therapy.;The clinical value of intracellular autoantigens B-cell epitopes in systemic rheumatic diseases.;Serological cloning of cancer/testis antigens expressed in prostate cancer using cDNA phage surface display;Complete response to donor lymphocyte infusion in multiple myeloma is associated with antibody responses to highly expressed antigens.;Humoral immune response against melanoma antigens induced by vaccination with cytokine gene‐modified autologous tumor cells;Identification of a shared epitope recognized by melanoma-specific, HLA-A3-restricted cytotoxic T lymphocytes.;Genome-wide Characterization of a Viral Cytotoxic T Lymphocyte Epitope Repertoire*;Microarrays of tumor cell derived proteins uncover a distinct pattern of prostate cancer serum immunoreactivity;Protein arrays for autoantibody profiling and fine‐specificity mapping;Protein microarrays: Promising tools for proteomic research;Recursive partitioning as an approach to selection of immune markers for tumor diagnosis.;Antigen-specific immunity in neuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen.;Expression of cancer/testis (CT) antigens in lung cancer.;Unique conformation of cancer autoantigen B23 in hepatoma: A mechanism for specificity in the autoimmune response;NY-ESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial ovarian cancer.;Identification of tumor‐associated antigens by screening phage‐displayed human cDNA libraries with sera from tumor patients;Tumor-specific immunological recognition of frameshift-mutated peptides in colon cancer with microsatellite instability.;CD1d-restricted T cells regulate dendritic cell function and antitumor immunity in a granulocyte–macrophage colony-stimulating factor-dependent fashion;Melanoma inhibitor of apoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction;The use of HPLC-MS in T-cell epitope identification.;Immunomic analysis of human sarcoma;Enhancement of antibody detection in cancer using panel of recombinant tumor-associated antigens.;Immunoscreening of phage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease.;Integrated functional genomics approach for the design of patient-individual antitumor vaccines.;Proteomics Approaches to Uncover the Repertoire of Circulating Biomarkers for Breast Cancer;CML28 is a broadly immunogenic antigen, which is overexpressed in tumor cells.;Towards patient‐specific tumor antigen selection for vaccination;Cancer/testis antigens: an expanding family of targets for cancer immunotherapy;Serological identification of antigens associated with renal cell carcinoma.;Characterisation of tumour-associated antigens in colon cancer;Solid-Phase Epitope Recovery: A High Throughput Method for Antigen Identification and Epitope Optimization;Enhanced humoral immune response correlates with improved disease-free and overall survival in American Joint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine.;Identification of Breast Cancer-Restricted Antigens by Antibody Screening of SKBR3 cDNA Library Using a Preselected Patient's Serum;Posttranslational protein modifications: new flavors in the menu of autoantigens;ATP6S1 elicits potent humoral responses associated with immune-mediated tumor destruction;Identification of cancer/testis genes by database mining and mRNA expression analysis;Autoantigen microarrays for multiplex characterization of autoantibody responses;Miniaturised multiplexed immunoassays.;Identification of Cyclin B1 as a Shared Human Epithelial Tumor-Associated Antigen Recognized by T Cells;Trends in biomarker research for cancer detection.;Autoimmune diseases: are markers ready for prediction?;Post-translational protein modifications in antigen recognition and autoimmunity.;Identification of a tissue-specific putative transcription factor in breast tissue by serological screening of a breast cancer library.;Profiling the immune response in patients with breast cancer by phage‐displayed cDNA libraries;Tumor-associated antigen TA90 immune complex assay predicts recurrence and survival after surgical treatment of stage I-III melanoma.;Evidence that specific T lymphocytes may participate in the elimination of chronic myelogenous leukemia;Pre-existent immunity to the HER-2/neu oncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer;Profiling the immune responses in patient sera with peptide and cDNA display libraries.;Reactivity of natural and induced human antibodies to MUC1 mucin with MUC1 peptides and n‐acetylgalactosamine (GalNAc) peptides;Rapid degradation of a large fraction of newly synthesized proteins by proteasomes;Serological identification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung cancer.;p53 Antibodies in the sera of patients with various types of cancer: a review.;Expression of cancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a new CT gene, CT9.;CT10: A new cancer‐testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational‐difference analysis;p53 gene mutation, tumor p53 protein overexpression, and serum p53 autoantibody generation in patients with breast cancer.;Cancer-testis antigens and ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-specific ING1 transcripts and a homologue gene.;Antigens recognized by autologous antibody in patients with renal‐cell carcinoma;SYFPEITHI: database for MHC ligands and peptide motifs;Cleavage by Granzyme B Is Strongly Predictive of Autoantigen Status;Humoral immune responses of cancer patients against \"Cancer-Testis\" antigen NY-ESO-1: correlation with clinical events.;The telomerase catalytic subunit is a widely expressed tumor-associated antigen recognized by cytotoxic T lymphocytes.;Isoforms of the human PDZ-73 protein exhibit differential tissue expression.;The 72-kDa Component of Signal Recognition Particle Is Cleaved during Apoptosis*;The Central Role of CD4+ T Cells in the Antitumor Immune Response;Characterization of human colon cancer antigens recognized by autologous antibodies;A Survey of the Humoral Immune Response of Cancer Patients to a Panel of Human Tumor Antigens;Human lung cancer antigens recognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor gene locus on chromosome 3p21.3.;High-titer HER-2/neu protein-specific antibody can be detected in patients with early-stage breast cancer.;SSX: A multigene family with several members transcribed in normal testis and human cancer;Autoantibodies to p53 in ovarian cancer patients and healthy women: a comparison between whole p53 protein and 18-mer peptides for screening purposes.;A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening.;Humoral immune response against p53 protein in patients with colorectal carcinoma;DNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to detect all known p53 alterations in human malignancies.;Anti-p53 antibodies in sera from patients with chronic obstructive pulmonary disease can predate a diagnosis of cancer.;Anti-p53 antibodies in sera of workers occupationally exposed to vinyl chloride.;Absence of p53 autoantibodies in sera from glioma patients.;A motif in human histidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-coil that is essential for enzymatic activity and contains the major autoantigenic epitope.;Clinical utility of the immunocytochemical detection of p53 protein in cytological specimens.;Role of bone marrow-derived cells in presenting MHC class I-restricted tumor antigens.;Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains.;Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-macrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor immunity.;Development of antibodies against p53 in lung cancer patients appears to be dependent on the type of p53 mutation.;HLA-A2 molecules in an antigen-processing mutant cell contain signal sequence-derived peptides;Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry.;Very long charge runs in systemic lupus erythematosus-associated autoantigens.;Characterization of naturally occurring minor histocompatibility peptides including H-4 and H-Y.;The fine specificity of anti-La antibodies induced in mice by immunization with recombinant human La autoantigen.;Post-translational modifications of proteins: some problems left to solve.;CA125 Antigen Levels in Obstetric and Gynecologic Patients;Purification of a human prostate specific antigen.;Detection of a transformation-related antigen in chemically induced sarcomas and other transformed cells of the mouse.;SPECIFIC CARCINOEMBRYONIC ANTIGENS OF THE HUMAN DIGESTIVE SYSTEM;Immunotherapy (APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3 randomized placebo-controlled trial;Efficient purification of unique antibodies using peptide affinity-matrix columns.;Endogenous immune response to early- and intermediate-stage melanoma is correlated with outcomes and is independent of locoregional relapse and standard prognostic factors.;Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem cell transplantation and in healthy female donors.;Recombinant antigen expression on yeast surface (RAYS) for the detection of serological immune responses in cancer patients.;Immunotherapy of hematologic malignancy.;p53 autoantibodies can be indicative of the development of breast cancer relapse.;The autoantibody repertoire: searching for order;MUC1-like tandem repeat proteins are broadly immunogenic in cancer patients.;Epitope identification and vaccine design for cancer immunotherapy.;Immunotherapeutic Targets Cancer : Identification of Potential Diagnostic and Cancer-related Serological Recognition of Human Colon Updated;Identification of tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by serological screening of a testicular library with breast cancer serum.;Humoral immunity to human breast cancer: antigen definition and quantitative analysis of mRNA expression.;Monitoring of p53 autoantibodies after resection of colorectal cancer: relationship to operative curability.;Gure, A. O. et al. CT10: a new cancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-difference analysis. Int. J. Cancer 85, 726-732;MHCPEP, a database of MHC-binding peptides: update 1997;Cancer,Facts and Figures about."
        ],
        [
         "38",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "http://www.jbc.org/article/S0021925820792379/pdf",
         "Chemistry,Medicine",
         "Journal of Biological Chemistry",
         "E. Gagnon,M. Bergeron,Nikolas D. Daigle,Marie-Hélène Lefoll,P. Isenring",
         "117842903,5297299,40563519,4943153,144981400",
         "30",
         "1.0",
         "2005-09-16",
         "2005",
         "",
         ""
        ],
        [
         "39",
         "00295936451c8a6b8a982d6ba9ab64cb647370ae",
         "https://academic.oup.com/ajcn/article-pdf/81/2/440/23875105/440.pdf",
         "Medicine,Biology",
         "American Journal of Clinical Nutrition",
         "Kerry‐Ann da Costa,Christopher E. Gaffney,L. Fischer,S. Zeisel",
         "4038445,2054146022,38060603,2804284",
         "156",
         "1.0",
         "2005-02-01",
         "2005",
         "4d4669217589611bb901b17698fcb18a144740a7;0263d8843e92dc3e34506ad3fde379fad828f8a1;e2bd4ffcddd7af7c8b5f94cc49601aff57a7a21c;b72287e075cef4d3f7c8856498688358f61397da;9a4483b65e2c4c7b930f90b1d4a922a5dfee0ba6;d1395d688cec8093ddcccf1cc9d6ae04b0d3c23a;4efa51fbe900205a49bc7f6a68154ad4296eea0b;f204856fc5373437f814c0c2402778c27594ef2b;d2b625d574ca8fa283873237b1349fb49c785059;603601d2c4ef1e4412f1d89dea14c835213b83a0;b04ea587a39c3720482f3692ed743f98bd4bbee8;e9d74e7b48df8378510d1d6bd25a27f3fba5f150;a2e6970a571def1f90636db927b6ef9c59f69482;0873ef5d9de496dcdaa33415423647019752d873;eb9f6bcc3f5bdb4716c37c1e35c6db8997190097;ed6edc00cc4726200771bf11829207f2683462d9;0cad9d3316e0a9010cb0dacbb79e5f58e457c642;6a3444760b5ebe174e24c8d29fd97b5afa743ee6;b4c067e2e1b52ef152581aa4c2e9f40c072f6208;a957011ab44cca7976f56e23536c3d89fc14f6bd;49f5e5d64de07d62c6b2b0c8e03b20eaf1161d3a;9431c778f18d1f51cfb8e95c5035b4f3216c6998;e1ec7fbfe68d62be64eb8a7666d8ac6ec891ff13;008a3ea9d8e594a215ca16720893f456f7787cc5;a32c15814b6175e935ee3477514c4342b2242ccb;20cb3ad6e81475861e92b6d477d3d8c3386818c1;2e29ce6a2b65aa08a1bfe0a80b45b74d72edb6e7;569d20baf31b7d9cf72e621c30b0105bcb1abe6f;222b5b859bf2972cf7b74b1af5c1301a2c83a701;e426f902f9e694c64b430208317433c7f930d7f8;e24f25dba6659b9c7410676bd5a08b6b5caef2bf;9a395ca341ae350e2c374b3251865ccb1e4dd5c0",
         "Choline- and betaine-defined diets for use in clinical research and for the management of trimethylaminuria.;Elevated serum creatine phosphokinase in choline-deficient humans: mechanistic studies in C2C12 mouse myoblasts.;Betaine as a Determinant of Postmethionine Load Total Plasma Homocysteine Before and After B-Vitamin Supplementation;Folate status response to controlled folate intake is affected by the methylenetetrahydrofolate reductase 677C-->T polymorphism in young women.;Effect of folic acid on fenofibrate-induced elevation of homocysteine and cysteine.;Betaine supplementation lowers plasma homocysteine in healthy men and women.;Concentrations of choline-containing compounds and betaine in common foods.;Phosphatidylethanolamine N-methyltransferase (PEMT) knockout mice have hepatic steatosis and abnormal hepatic choline metabolite concentrations despite ingesting a recommended dietary intake of choline.;Effects of cytidine 5'-diphosphocholine on plasma homocysteine levels in rat.;Quantitation of choline and its metabolites in tissues and foods by liquid chromatography/electrospray ionization-isotope dilution mass spectrometry.;Hyperhomocysteinemia due to methionine synthase deficiency, cblG: structure of the MTR gene, genotype diversity, and recognition of a common mutation, P1173L.;Choline deficiency causes reversible hepatic abnormalities in patients receiving parenteral nutrition: proof of a human choline requirement: a placebo-controlled trial.;The 1298A-->C polymorphism in methylenetetrahydrofolate reductase (MTHFR): in vitro expression and association with homocysteine.;Determinants of plasma total homocysteine concentration in the Framingham Offspring cohort.;Betaine-homocysteine methyltransferase expression in porcine and human tissues and chromosomal localization of the human gene.;The use of homocysteine and other metabolites in the specific diagnosis of vitamin B-12 deficiency.;Choline deficiency: A cause of hepatic steatosis during parenteral nutrition that can be reversed with intravenous choline supplementation;Evidence that homocysteine is an independent risk factor for atherosclerosis in hyperlipidemic patients.;Vitamin B-6 deficiency vs folate deficiency: comparison of responses to methionine loading in rats.;AIN-93 purified diets for laboratory rodents: final report of the American Institute of Nutrition ad hoc writing committee on the reformulation of the AIN-76A rodent diet.;Low plasma free choline is prevalent in patients receiving long term parenteral nutrition and is associated with hepatic aminotransferase abnormalities.;Accumulation of 1,2-sn-diradylglycerol with increased membrane-associated protein kinase C may be the mechanism for spontaneous hepatocarcinogenesis in choline-deficient rats.;Rapid high-performance liquid chromatographic assay for total homocysteine levels in human serum.;Choline, an essential nutrient for humans;Lactobacillus casei microbiological assay of folic acid derivatives in 96-well microtiter plates.;The active synthesis of phosphatidylcholine is required for very low density lipoprotein secretion from rat hepatocytes.;Plasma choline concentration in humans fed parenterally.;Betaine in the treatment of homocystinuria due to 5,10-methylenetetrahydrofolate reductase deficiency;A rapid method of total lipid extraction and purification.;Effect of methotrexate on homocysteine and other sulfur compounds in tissues of rats fed a normal or a defined, choline-deficient diet;Pathways and Regulation of Homocysteine Metabolism in Mammals;Institute of Medicine, National Academy of Sciences USA. Dietary reference intakes for folate, thiamin, riboflavin, niacin, vitamin B-12, panthothenic acid, biotin, and choline;The biosynthesis of choline and its relation to phospholipid metabolism."
        ],
        [
         "40",
         "002a108f32eb157c0b88aea3c6f29d4b614c3b78",
         "https://academic.oup.com/toxsci/article-pdf/86/1/161/4657319/kfi172.pdf",
         "Biology,Medicine",
         "Toxicological Sciences",
         "C. Bowman,K. Turner,M. Sar,N. Barlow,K. Gaido,P. Foster",
         "5835636,121132152,4754930,39737755,4832338,41131499",
         "34",
         "0.0",
         "2005-07-01",
         "2005",
         "",
         ""
        ],
        [
         "41",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         "https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-10/issue-5/051604/Prospective-grading-of-neoplastic-change-in-rat-esophagus-epithelium-using/10.1117/1.2102767.pdf",
         "Medicine",
         "Journal of Biomedical Optics",
         "A. Wax,J. Pyhtila,R. Graf,R. Nines,C. Boone,R. Dasari,M. Feld,V. Steele,G. Stoner",
         "6601645,2596802,39855573,5362380,48807863,6398265,145506659,5086274,3306303",
         "65",
         "2.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "42",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "https://access.archive-ouverte.unige.ch/access/metadata/414fbfe1-4481-4240-92c1-e62a3593f0f0/download",
         "Psychology,Medicine",
         "Emotion",
         "T. Brosch,D. Sharma",
         "2256291,34546725",
         "153",
         "10.0",
         "2005-09-01",
         "2005",
         "804eeefaf41247d1927a7e9d4ab27f06c3917fab;24cea7f3a3d8e4d490725d55933a724421b8c086;e35f63ac711e7253049a4811a354b93478d4959e;240247d67cc26cd7474e02ed75de6682e181839d;494771f2b2bcff4d795dc4e974524c5a7ec5572f;2da6cce61dd4defa4c44e82d1e0bb9920ee193bb;3542ff0be48d6196677c85fb6b65dbf7180d0e46;bdfeee608d25196733e80800f43a57b768a039ab;fbe379683d148cdf82a03010f1e79f22d16657e6;4f3ef78375b4c147ab794ca9a8e9bca1fe4daf4e;76361a44e145732a39dbc68d9418871038c83be2;6871c3babd4f47895d616d15b17b7ba471afa89a;bc44f92d97e409449ce3af72f7d2cb4ac544245a;c26f4aef22b2682db192d60b932455df09702e92;d506a8fdc86aeb83c28d4b5a87cf35e3c0e14910;46c99f0a5935572dd8b19936afef8faa8de25fbf;05c2b29d567263cb034f28eefe078545ad357d73;bdeee63dc6398e2d9d1fe9571d23dcf141903ee1;2445bf8b2442a04e9a4db6819aaa267f8e45f4a7;9508efad2d667fa38e4c9a50efd91c117a7a7943;74257ff0d6645ab73650fdc9faaf8fe1a6a430b5;7f57c5462b5728661608c308e76be0e2f4a0408a",
         "Attentional control of the processing of neural and emotional stimuli.;Searching for threat;Emotion drives attention: detecting the snake in the grass.;The face in the crowd revisited: a threat advantage with schematic stimuli.;Fear and anxiety: animal models and human cognitive psychophysiology.;Facial Expressions of Emotion: Are Angry Faces Detected More Efficiently?;A subcortical pathway to the right amygdala mediating \"unseen\" fear.;It Takes a Confounded Face to Pop Out of a Crowd;On the automatic nature of phobic fear: conditioned electrodermal responses to masked fear-relevant stimuli.;Orienting and schizophrenia: stimulus significance, attention, and distraction in a signaled reaction time task.;A feature-integration theory of attention;The premise of equipotentiality in human classical conditioning: conditioned electrodermal responses to potentially phobic stimuli.;One-trial learning and superior resistance to extinction of autonomic responses conditioned to potentially phobic stimuli.;Phobias and preparedness;The cognitive neuroscience of emotion;Fears, phobias, and preparedness: toward an evolved module of fear and fear learning.;Unconscious emotion : Evolutionary perspectives, psychophysiological data, and neuropsychological mechanisms;A threat imminence approach to human fear responding : direction of threath, aversive contexts, and electrodermal responses;Preparedness and electrodermal fear-conditioning: ontogenetic vs phylogenetic explanations.;Finding the face in the crowd: an anger superiority effect.;Feature analysis in early vision: evidence from search asymmetries.;The emotional brain.;E-Prime;32306-1270), as editor of the Clinician's Research Digest newsletter for"
        ],
        [
         "43",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "https://www.jstage.jst.go.jp/article/expanim/54/5/54_5_461/_pdf",
         "Medicine",
         "Experimental animals",
         "M. Asakawa,T. Yoshioka,I. Hikita,T. Matsutani,T. Hirasawa,A. Arimura,T. Sakata,T. Horikawa",
         "50156427,2001345,5223435,5352485,34764605,47810324,48565673,143893766",
         "23",
         "1.0",
         "2005-10-01",
         "2005",
         "abd6726370ffc4c6e7ed4b64164bf5d2b42e1fcb;d1076a4457fff22f01c397ef080a731c23e5e807;a884242808a1ace5f72ba4d59c97a6360e3644e7;2571e34834098d5d1ec06e89c81458ec84a41b8e;6c644bd483d2639bcfddeaf8bcb696cd2df1ef77;e056d6b051a9582e3f3f1d81d84a99b329946b64;b3327aae68ba3bd0c99e3dc1504d0a118d9fa133",
         "Role of the Nh (Non-hair) mutation in the development of dermatitis and hyperproduction of IgE in DS-Nh mice.;DS–Nh as an experimental model of atopic dermatitis induced by Staphylococcus aureus producing staphylococcal enterotoxin C;Characterization of dermatitis arising spontaneously in DS-Nh mice maintained under conventional conditions: another possible model for atopic dermatitis.;Atopic dermatitis: new insights and opportunities for therapeutic intervention.;Locus of dominant hairless gene (Ht) causing abnormal hair and keratinization maps to rat chromosome 10.;Characteristics of spontaneous erythema appeared in hairless rats.;IgE production by normal human lymphocytes is induced by interleukin 4 and suppressed by interferons gamma and alpha and prostaglandin E2."
        ],
        [
         "44",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "http://www.cell.com/article/S009286740500098X/pdf",
         "Biology,Medicine",
         "Cell",
         "I. Krylova,E. Sablin,Jamie M. R. Moore,R. Xu,G. Waitt,J. A. MacKay,D. Juzumiene,J. Bynum,K. Madauss,V. Montana,L. Lebedeva,M. Suzawa,Jon D. Williams,Shawn P. Williams,R. Guy,J. W. Thornton,R. Fletterick,T. Willson,H. Ingraham",
         "34437421,5695081,2108562213,4605417,7978640,144066376,47866582,48218038,3502869,4051165,2106094788,40606710,51017082,23919632,144399402,3722546,4009116,5064590,3514504",
         "405",
         "18.0",
         "2005-02-11",
         "2005",
         "1d6bd7b0c8db990d8d67f496ede67ae2827cae6f;9603421bed7605ac327a47ae9982d0ab8c6eeb58;c84e4fce3b6dadc50ce3c174b3bc4759442e38e9;d91a2295136ab9944fcd756bddab143c84804b59;ce6902fec41ee6fd52c9e3eefa059bc6bb415eef;44ef03895ff4e14bf5d226ef3d05f0e097d80486;87f895a0ecdc49df01a9cbef037f898e90d2c91d;89544b7b1dfd705c9d5199e342e8cd0fa7221a51;926e9a38d4e6ee4ff925995e6625106a2513183a;cb4f09dcf20c7700c3a5c1c5e2ca4c981182ce68;f256f33b7a0cf35735fba4d5defa2c3da988e490;fe6adae44ba4ded648fe28fe3837a9d22eab5d08;63523500744c4a42596cef3a31049061b7999a0c;4bd5ee4101ea924889a1ea398fb3cdd39310ba74;7599925bf30ff65b9c4e63464a1d7d7d014c09a7;bf6d61b49f81eeea09540d0a5924f026230c8d14;69faa8f5d27ed9f6cb09a4ba5a251a52aa698fea;83011a2d9fafdebe71b685684d22e7d5e37af1f7;badcfc9340eee368b92961ba52fb94012a7f031f;3902088e094524712a09886b265e77cb3d5d4bb3;701fab8b80fffcd0765465a8ed7a523fa3429208;c740aba3971f2e536e4e70ae166ba46debd24086;52ca6d637dd35ebbe5659f31342260a24b354988;2a9dce8ca5350d4137a5debd70e144e145e0468e;bada3a161bb601bd7b0ef07a901379d8ff22298c;2f1bdce0bca9ff5556e0c237c232b92fc64fe1e5;ee5445492daa6341cab66a5ecf24aa041bb6129f;b3ac08c338b82741e97e0671aea2a3381cd0a57b;bac05ffabbbe20bbfd3f9ac7bf2bb0d627b364e0;6160c4d09e8745db0d669351d138547cfab8909c;885007d7cdff2b2f9330a93eb9d568d0bc76403a;2bb11f7aae32348cc314171336d2e27bebf32513;d615c6e4c2161a309b9c63972867db1adf646a7e;2a8fa95224b6a8a54368262f621e3bb36e6a62c6;eb4c2bdb02b2fbca400d8a645440fcf16d25b0e5;2749817f25ac423a826977acad0284405c163d0e;67c9984328066bf9019152a7c4c325941694bfa7;193b40d2589ce17655a376a82f32747b6edaafc0;91b1bd09d655da66f30a0b2b416be982383e4f93;363c04c59e1ccf958552fe68eed66e274977c70d;f9845ded6f23c4d2e620a88f7a765ec38dc22bef;8df18f32bb193c4c9cfae684ea4a80d616601331;20cc588436334fd28ca613a68cffe81c9ec29844;ecbf24f75bd3d9dd4b3b303994f77850f57e2ca7;847eab3db971b86aa434f2a60ffeef7317be5d42;0aef55484c9bcde366848592dd43b45aa85d55a5;4ab6abd506eb7b4b10836e83a03ff274278bb0c5;cbb0f3690661c288f1a383293a67d42218081e2c;1117b106700a5d022a3af1103c9c370f95ea0fcc;e08c7acadbbeada1f429d43231aa02d92c3b3ba1;6332dfdced172ca5ffa1971c1b285c5ab3119281",
         "The DEAD-Box Protein DP103 (Ddx20 or Gemin-3) Represses Orphan Nuclear Receptor Activity via SUMO Modification;Amplification of the steroidogenic factor 1 gene in childhood adrenocortical tumors.;Identification of liver receptor homolog-1 as a novel regulator of apolipoprotein AI gene transcription.;Synergy between LRH-1 and beta-catenin induces G1 cyclin-mediated cell proliferation.;Quantitative Proteomics of the Thyroid Hormone Receptor-Coregulator Interactions*[boxs];LRH-1: an orphan nuclear receptor involved in development, metabolism and steroidogenesis.;Sex determination: a ‘window’ of DAX1 activity;Generation and Characterization of Human Steroidogenic Factor 1 LBD Crystals with and without Bound Cofactor Peptide;SHIP-2 and PTEN Are Expressed and Active in Vascular Smooth Muscle Cell Nuclei, but Only SHIP-2 Is Associated with Nuclear Speckles*;Resurrecting the Ancestral Steroid Receptor: Ancient Origin of Estrogen Signaling;MrBayes 3: Bayesian phylogenetic inference under mixed models;Activation of nuclear receptors: a perspective from structural genomics.;StAR-related Lipid Transfer (START) Proteins: Mediators of Intracellular Lipid Metabolism*;The Drosophila Orphan Nuclear Receptor DHR38 Mediates an Atypical Ecdysteroid Signaling Pathway;Structural basis for ligand-independent activation of the orphan nuclear receptor LRH-1.;Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors;Nuclear lipid signalling;Nuclear lipids: New functions for old molecules?;X-Ray Structure of the hRORα LBD at 1.63 Å;Elevation of plasma phospholipid transfer protein in transgenic mice increases VLDL secretion Published, JLR Papers in Press, August 16, 2002. DOI 10.1194/jlr.M200166-JLR200;Crystal Structure of the HNF4α Ligand Binding Domain in Complex with Endogenous Fatty Acid Ligand*;Phosphorylation and Intramolecular Stabilization of the Ligand Binding Domain in the Nuclear Receptor Steroidogenic Factor 1;Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids.;Inhibition of androgen receptor (AR) function by the reproductive orphan nuclear receptor DAX-1.;Control of Cholesterol Turnover in the Mouse*;Phosphatidic Acid-Mediated Mitogenic Activation of mTOR Signaling;The PITP family of phosphatidylinositol transfer proteins;Characterization of a G Protein-activated Phosphoinositide 3-Kinase in Vascular Smooth Muscle Cell Nuclei*;Binding of ligands and activation of transcription by nuclear receptors.;Structure of a Multifunctional Protein;Crystal Structure of the Ligand-binding Domain of the Ultraspiracle Protein USP, the Ortholog of Retinoid X Receptors in Insects*;Haploinsufficiency of steroidogenic factor-1 in mice disrupts adrenal development leading to an impaired stress response.;Nuclear Lipid Signaling;Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors.;Ligand binding and nuclear receptor evolution;A new method to localize and test the significance of incongruence: detecting domain shuffling in the nuclear receptor superfamily.;Inferring the historical patterns of biological evolution;Oxysterol Regulation of Steroidogenic Acute Regulatory Protein Gene Expression;Structure and specificity of nuclear receptor-coactivator interactions.;25-Hydroxycholesterol is not a ligand for the orphan nuclear receptor steroidogenic factor-1 (SF-1).;Wilms' Tumor 1 and Dax-1 Modulate the Orphan Nuclear Receptor SF-1 in Sex-Specific Gene Expression;Activation of the orphan nuclear receptor steroidogenic factor 1 by oxysterols.;DAX-1 inhibits SF-1-mediated transactivation via a carboxy-terminal domain that is deleted in adrenal hypoplasia congenita;A new method of inference of ancestral nucleotide and amino acid sequences.;CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;The nuclear receptor steroidogenic factor 1 acts at multiple levels of the reproductive axis.;A cell-specific nuclear receptor is essential for adrenal and gonadal development and sexual differentiation;Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology;Repression of orphan nuclear recep - or activity via SUMO - modification and the Dead - Box Protein P 103 ( Ddx 20 , Gemin - 3 );Nuclear lipids: key signaling effectors in the nervous system and other tissues;X-ray structure of the hRORalpha LBD at 1.63 A: structural and functional data that cholesterol or a cholesterol derivative is the natural ligand of RORalpha.;Nuclear-receptor ligands and ligand-binding domains.;Standard Fmoc protocols.;Methods Enzymol"
        ],
        [
         "45",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.HYP.0000184251.01159.72",
         "Chemistry,Medicine",
         "HYPERTENSION",
         "C. Zeng,Z. Wang,U. Hopfer,L. Asico,G. Eisner,R. Felder,P. Jose",
         "49386778,48708059,4337525,6316449,38877762,1955072,3561892",
         "39",
         "1.0",
         "2005-10-01",
         "2005",
         "",
         ""
        ],
        [
         "46",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         "https://jcp.bmj.com/content/jclinpath/58/9/991.full.pdf",
         "Biology,Medicine",
         "Journal of Clinical Pathology",
         "Giulio Rossi,G. Sartori,Riccardo Valli,F. Bertolini,Nazzarena Bigiani,L. Schirosi,A. Cavazza,G. Luppi",
         "2248858530,143679786,2251605562,2251555357,12435222,48791235,3835552,2251360364",
         "23",
         "3.0",
         "2005-08-26",
         "2005",
         "",
         ""
        ],
        [
         "47",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/129/2/1290171.pdf",
         "Medicine,Biology",
         "Reproduction",
         "J. Ito,M. Hirabayashi,Megumi Kato,A. Takeuchi,Mayumi Ito,M. Shimada,S. Hochi",
         "38681521,34621985,32885513,152918649,50230871,10276380,5629712",
         "41",
         "3.0",
         "2005-02-01",
         "2005",
         "",
         ""
        ],
        [
         "48",
         "002cc4c2e457c3cc0fb0f704d0483a003e8a0575",
         "https://academic.oup.com/jac/article-pdf/56/4/686/2308205/dki309.pdf",
         "Medicine,Biology",
         "Journal of Antimicrobial Chemotherapy",
         "L. Halbert,J. Kaneene,L. Mansfield,P. Ruegg,L. Warnick,S. Wells,C. Fossler,A. Campbell,Angela M Geiger-Zwald",
         "12508766,5363293,3666808,6381510,5534287,32612292,12064236,144131845,1422352824",
         "18",
         "1.0",
         "2005-10-01",
         "2005",
         "",
         ""
        ],
        [
         "49",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "http://arxiv.org/pdf/1111.4445.pdf",
         "Mathematics,Computer Science",
         "SIAM Journal on Numerical Analysis",
         "Y. Kuznetsov,W. Govaerts,E. Doedel,Annick Dhooge",
         "1775504,1797969,2966430,2713848",
         "35",
         "2.0",
         "2005-04-01",
         "2005",
         "56ff3caaf8a0dd6ed415f8b84f6c8f9041d2a0fb;6ed02cffaaf644f63912f41fbfa00f6f4406c464;7838818b776dd024c885b857c56815ddf215cf60;435ab7306b96edfc3fad6fe165e22d6d6abe531b;526c44f9a4fdac6824d7b1813869649797defd08;3239959373f015e193b6493d7fd64b603547c6a2;8b6f7d0b305d6cba163ef568cdc361ada56e6b7a;cd060346f7d865fff39eb01922207776811cdfa4;b32aba832ca2be4a84c687b040bc39fc8a8413bb;09f52abb09d783865c4ef2cf5af6454802bd2b93;42763fcda07ec7c1a6a93aba8c8f83352c6299c3;cf2c8d80c7bcba7a507801c6e3da8cac70b85bd8;0cbe5c98f1b9c0273629c41745fbec5750b6cf9c;23a8fe3d34a78f2a2fe3f8b0ff32919586dbe96b;340d7ac3b6e13a63e0c47b46543e4377a55f6963;2b6adbad7d8f4c5f1ade291264df221e1b7e22eb;fc93cfbfe36648f105fd4034bbc2471c3367dff8;a594a834dd17b28e2b5ddd42cbd1d9e62efc1b7a;90c0d56498401779d928cd4d8d24c66705fad561;93b1a0baf18ec6b3278805931585392000877ff8;6df4bbd7d59ccce8b034ad9eca8ee1c5a4355592;54cce58d16bd817c766c9495c57eac4f77993b71;1cdcc6db6b8139eb1c1110ba44933727760a17dd;1c46bd1bfbfe4248aa642f5210841379d5923763;d84c4a0dc989082a28956c95ba8ff3a220182091;d7cbbcb3b90deef39fc3b1f4e2eecb80e78a3c08;17be3ad189a2e01f96648ab4b890ddc1c2cf641f;c2e294e7b709dcdae66b600a3c8d3fa459745d0f;2ce5b9b35b5941749cb614e72a69e7ee0de9184c;d35064f6159576e2d9a49a443f8abcdb681cc49a;70e3da6c426ca384f78f77474cbbf00a436038a2;52c0a963506ad2df44e4d5e4358b1e12f500b983;747fa5fcf394aeedc790488ce725a35e3351f310;0b41448a2d92cccfcf4a9ce1e0d68371ce200cfb;39f578060187f86bc80f645b39d735479e664d86;03894e8313ad4c1b36af338151a01baa88cc479e;eea110aa0ea13cc090cebe08f30c3178cc0ad359;7921f44af253340514c9c98af417b0cb07c75afd;4ae84da06e7dbb22fcf248ce4bfa31569dfbad6b;2f59d80b6cccd2b0a28a437d81f360c9fc701287;36ba81e158b864ed1576224f83da232b6031465c;6c1638a43f9bab7409de246138866814cffde0f2;b862a20ee79a56c79e03701fefd4c5b14d53a542",
         "Numerical Methods for Two-Parameter Local Bifurcation Analysis of Maps;Newton's descent method for the determination of invariant tori.;A Newton Method for Locating Invariant Tori of Maps;Sensitivity Analysis of ODES/DAES Using the Taylor Series Method;Numerical Normal Forms for Codim 2 Bifurcations of Fixed Points with at Most Two Critical Eigenvalues;Numerical Methods for the Continuation of Invariant Tori;MATCONT: A MATLAB package for numerical bifurcation analysis of ODEs;Cl_matcont: a continuation toolbox in Matlab;Computation of Periodic Solution Bifurcations in ODEs Using Bordered Systems;Chapter 4 – Numerical Continuation, and Computation of Normal Forms;Computing Periodic Orbits and their Bifurcations with Automatic Differentiation;Numerical Normalization Techniques for All Codim 2 Bifurcations of Equilibria in ODE's;Topics in bifurcation theory and applications;Elements of applied bifurcation theory (2nd ed.);Harmonic balance analysis of period-doubling bifurcations with implications for control of nonlinear dynamics;Algorithm 755: ADOL-C: a package for the automatic differentiation of algorithms written in C/C++;Bifurcation and predictability analysis of a low-order atmospheric circulation model;Models of complex dynamics in nonlinear systems;Bifurcation sequences at 1:4 resonance: an inventory;Harmonic balance methods for the analysis of chaotic dynamics in nonlinear systems;BIFURCATIONS AND CHAOS IN A PERIODIC PREDATOR-PREY MODEL;NUMERICAL ANALYSIS AND CONTROL OF BIFURCATION PROBLEMS (II): BIFURCATION IN INFINITE DIMENSIONS;Numerical Calculation of Invariant Tori;THE QUASIPERIODIC ROUTE TO CHAOS IN A MODEL OF THE PEROXIDASE-OXIDASE REACTION;Global Characterization of the Normal Form for a Vector Field Near a Closed Orbit;A simple global characterization for normal forms of singular vector fields;Normal form reduction for time-periodically driven differential equations;Numerical computation of invariant circles of maps;NONLINEAR OSCILLATIONS, DYNAMICAL SYSTEMS, AND BIFURCATIONS OF VECTOR FIELDS (Applied Mathematical Sciences, 42);Irregularity: a fundamental property of the atmosphere*;Nonlinear Oscillations, Dynamical Systems, and Bifurcations of Vector Fields;Collocation at Gaussian Points;Continuation of Quasi-periodic Invariant Tori;The fold-flip bifurcation;On the computation of characteristic multipliers for predicting limit cycle bifurcations;Elements of applied bifurcation theory;AUTO 2000 : CONTINUATION AND BIFURCATION SOFTWARE FOR ORDINARY DIFFERENTIAL EQUATIONS (with HomCont);CONTENT: A multiplatform environment for analyzing dynamical systems;Normal forms of bifurcating periodic orbits;Numerical computation of periodic solution branches and oscillatory dynamics of the stirred tank reactor with A → B → C reactions;Geometrical Methods in the Theory of Ordinary Differential Equations;Elementary stability and bifurcation theory;Bifurcation of maps and applications;Topics in bifurcation theory;Numerical solution of boundary value problems in ordinary differential equations;Numerical Continuation, And Computation Of Normal Forms;CAPD: Computer assisted proofs in dynamics"
        ]
       ],
       "shape": {
        "columns": 12,
        "rows": 69993
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>fieldsOfStudy</th>\n",
       "      <th>venue</th>\n",
       "      <th>authorName</th>\n",
       "      <th>authorId</th>\n",
       "      <th>citationCount</th>\n",
       "      <th>influentialCitationCount</th>\n",
       "      <th>publicationDate</th>\n",
       "      <th>publicationYear</th>\n",
       "      <th>referenceIds</th>\n",
       "      <th>referenceTitles</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>https://aacrjournals.org/clincancerres/article...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Clinical Cancer Research</td>\n",
       "      <td>T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...</td>\n",
       "      <td>14839966,10239302,2000448727,3201672,7147569,2...</td>\n",
       "      <td>88</td>\n",
       "      <td>6.0</td>\n",
       "      <td>2005-04-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>http://www.merl.com/publications/docs/TR2005-0...</td>\n",
       "      <td>Computer Science,Mathematics</td>\n",
       "      <td>IEEE Transactions on Communications</td>\n",
       "      <td>Juntan Zhang,M. Fossorier</td>\n",
       "      <td>3193350,48891339</td>\n",
       "      <td>319</td>\n",
       "      <td>26.0</td>\n",
       "      <td>2005-03-07</td>\n",
       "      <td>2005</td>\n",
       "      <td>17d0afe4f814e5abc6252e572e3b039082bf6f07;02464...</td>\n",
       "      <td>Low-Density Parity-Check Codes;Turbo decoder a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>https://www.jstage.jst.go.jp/article/bpb/28/8/...</td>\n",
       "      <td>Medicine</td>\n",
       "      <td>Biological and Pharmaceutical Bulletin</td>\n",
       "      <td>Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...</td>\n",
       "      <td>11809650,6617669,123635715,11305673,2152146096...</td>\n",
       "      <td>24</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2005-08-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/pdfdirect/...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Developmental Dynamics</td>\n",
       "      <td>Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....</td>\n",
       "      <td>35690923,2114125286,2113252574,2134773,3658233</td>\n",
       "      <td>53</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2005-01-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820589394/pdf</td>\n",
       "      <td>Chemistry,Medicine</td>\n",
       "      <td>Journal of Biological Chemistry</td>\n",
       "      <td>M. Westfall,A. M. Lee,D. Robinson</td>\n",
       "      <td>5025983,49249911,48706510</td>\n",
       "      <td>70</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2005-12-16</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69988</th>\n",
       "      <td>fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6</td>\n",
       "      <td>https://academic.oup.com/humrep/article-pdf/20...</td>\n",
       "      <td>Medicine,Biology</td>\n",
       "      <td>Human Reproduction</td>\n",
       "      <td>S. Doi,M. Al-Zaid,P. Towers,C. Scott,K. A. Al-...</td>\n",
       "      <td>40151220,1404195567,37556460,78745210,1403335938</td>\n",
       "      <td>46</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2005-09-01</td>\n",
       "      <td>2005</td>\n",
       "      <td>3d2aaa390e58ff4d10cb2b6bfb4813e48565998c;8abd1...</td>\n",
       "      <td>Predictors for treatment failure after laparos...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69989</th>\n",
       "      <td>fffbc9b37f20515de2ff4fe60353d52dc37e65f5</td>\n",
       "      <td>http://www.cs.virginia.edu/~jl3aq/courses/CS85...</td>\n",
       "      <td>Computer Science</td>\n",
       "      <td>CCGrid 2005. IEEE International Symposium on C...</td>\n",
       "      <td>A. Barak,A. Shiloh,Lior Amar</td>\n",
       "      <td>144214996,40399772,34848329</td>\n",
       "      <td>42</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2005-05-09</td>\n",
       "      <td>2005</td>\n",
       "      <td>1713c96f258a9925b3ca0d814fdd0641c07bb3f1;3342b...</td>\n",
       "      <td>From sandbox to playground: dynamic virtual en...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69990</th>\n",
       "      <td>fffd2d6d54b4023ece28bfdebffdfb00502a09ec</td>\n",
       "      <td>https://europepmc.org/articles/pmc543423?pdf=r...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Molecular and Cellular Biology</td>\n",
       "      <td>Ying Huang,R. Intine,Amy M Mozlin,Samuel A. Ha...</td>\n",
       "      <td>48355811,6534085,15003627,5662843,3935489</td>\n",
       "      <td>64</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2005-01-01</td>\n",
       "      <td>2005</td>\n",
       "      <td>49bb10549894f0596e527139cf4a3bfb28fd6f2c;ced74...</td>\n",
       "      <td>Nuclear surveillance and degradation of hypomo...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69991</th>\n",
       "      <td>ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820789519/pdf</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Journal of Biological Chemistry</td>\n",
       "      <td>Faye Gordon,K. Luger,J. Hansen</td>\n",
       "      <td>48200695,3864710,34597814</td>\n",
       "      <td>143</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2005-10-07</td>\n",
       "      <td>2005</td>\n",
       "      <td>83a02dee50f2f83987d5490c8c9501609fd8f2df;c2150...</td>\n",
       "      <td>Histone modifications: combinatorial complexit...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69992</th>\n",
       "      <td>ffffeaedbe7931d161ab532015de3be15f99b180</td>\n",
       "      <td>https://pubs.asahq.org/anesthesiology/article-...</td>\n",
       "      <td>Medicine</td>\n",
       "      <td>Anesthesiology</td>\n",
       "      <td>S. Soriano,K. Anand,C. Rovnaghi,P. Hickey</td>\n",
       "      <td>3744726,145876135,4577001,4185720</td>\n",
       "      <td>85</td>\n",
       "      <td>12.0</td>\n",
       "      <td>2005-04-01</td>\n",
       "      <td>2005</td>\n",
       "      <td>27205842511218fa3dd0576405f70622232d03ee;1caa8...</td>\n",
       "      <td>Developmental neurotoxicity of ketamine: morph...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>69993 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paperId  \\\n",
       "0      0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1      000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2      000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3      0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4      00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "...                                         ...   \n",
       "69988  fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6   \n",
       "69989  fffbc9b37f20515de2ff4fe60353d52dc37e65f5   \n",
       "69990  fffd2d6d54b4023ece28bfdebffdfb00502a09ec   \n",
       "69991  ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3   \n",
       "69992  ffffeaedbe7931d161ab532015de3be15f99b180   \n",
       "\n",
       "                                           openAccessPdf  \\\n",
       "0      https://aacrjournals.org/clincancerres/article...   \n",
       "1      http://www.merl.com/publications/docs/TR2005-0...   \n",
       "2      https://www.jstage.jst.go.jp/article/bpb/28/8/...   \n",
       "3      https://onlinelibrary.wiley.com/doi/pdfdirect/...   \n",
       "4       http://www.jbc.org/article/S0021925820589394/pdf   \n",
       "...                                                  ...   \n",
       "69988  https://academic.oup.com/humrep/article-pdf/20...   \n",
       "69989  http://www.cs.virginia.edu/~jl3aq/courses/CS85...   \n",
       "69990  https://europepmc.org/articles/pmc543423?pdf=r...   \n",
       "69991   http://www.jbc.org/article/S0021925820789519/pdf   \n",
       "69992  https://pubs.asahq.org/anesthesiology/article-...   \n",
       "\n",
       "                      fieldsOfStudy  \\\n",
       "0                  Biology,Medicine   \n",
       "1      Computer Science,Mathematics   \n",
       "2                          Medicine   \n",
       "3                  Biology,Medicine   \n",
       "4                Chemistry,Medicine   \n",
       "...                             ...   \n",
       "69988              Medicine,Biology   \n",
       "69989              Computer Science   \n",
       "69990              Biology,Medicine   \n",
       "69991              Biology,Medicine   \n",
       "69992                      Medicine   \n",
       "\n",
       "                                                   venue  \\\n",
       "0                               Clinical Cancer Research   \n",
       "1                    IEEE Transactions on Communications   \n",
       "2                 Biological and Pharmaceutical Bulletin   \n",
       "3                                 Developmental Dynamics   \n",
       "4                        Journal of Biological Chemistry   \n",
       "...                                                  ...   \n",
       "69988                                 Human Reproduction   \n",
       "69989  CCGrid 2005. IEEE International Symposium on C...   \n",
       "69990                     Molecular and Cellular Biology   \n",
       "69991                    Journal of Biological Chemistry   \n",
       "69992                                     Anesthesiology   \n",
       "\n",
       "                                              authorName  \\\n",
       "0      T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...   \n",
       "1                              Juntan Zhang,M. Fossorier   \n",
       "2      Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...   \n",
       "3      Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....   \n",
       "4                      M. Westfall,A. M. Lee,D. Robinson   \n",
       "...                                                  ...   \n",
       "69988  S. Doi,M. Al-Zaid,P. Towers,C. Scott,K. A. Al-...   \n",
       "69989                       A. Barak,A. Shiloh,Lior Amar   \n",
       "69990  Ying Huang,R. Intine,Amy M Mozlin,Samuel A. Ha...   \n",
       "69991                     Faye Gordon,K. Luger,J. Hansen   \n",
       "69992          S. Soriano,K. Anand,C. Rovnaghi,P. Hickey   \n",
       "\n",
       "                                                authorId  citationCount  \\\n",
       "0      14839966,10239302,2000448727,3201672,7147569,2...             88   \n",
       "1                                       3193350,48891339            319   \n",
       "2      11809650,6617669,123635715,11305673,2152146096...             24   \n",
       "3         35690923,2114125286,2113252574,2134773,3658233             53   \n",
       "4                              5025983,49249911,48706510             70   \n",
       "...                                                  ...            ...   \n",
       "69988   40151220,1404195567,37556460,78745210,1403335938             46   \n",
       "69989                        144214996,40399772,34848329             42   \n",
       "69990          48355811,6534085,15003627,5662843,3935489             64   \n",
       "69991                          48200695,3864710,34597814            143   \n",
       "69992                  3744726,145876135,4577001,4185720             85   \n",
       "\n",
       "       influentialCitationCount publicationDate  publicationYear  \\\n",
       "0                           6.0      2005-04-01             2005   \n",
       "1                          26.0      2005-03-07             2005   \n",
       "2                           2.0      2005-08-01             2005   \n",
       "3                           2.0      2005-01-01             2005   \n",
       "4                           3.0      2005-12-16             2005   \n",
       "...                         ...             ...              ...   \n",
       "69988                       0.0      2005-09-01             2005   \n",
       "69989                       1.0      2005-05-09             2005   \n",
       "69990                       1.0      2005-01-01             2005   \n",
       "69991                       3.0      2005-10-07             2005   \n",
       "69992                      12.0      2005-04-01             2005   \n",
       "\n",
       "                                            referenceIds  \\\n",
       "0                                                          \n",
       "1      17d0afe4f814e5abc6252e572e3b039082bf6f07;02464...   \n",
       "2                                                          \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "69988  3d2aaa390e58ff4d10cb2b6bfb4813e48565998c;8abd1...   \n",
       "69989  1713c96f258a9925b3ca0d814fdd0641c07bb3f1;3342b...   \n",
       "69990  49bb10549894f0596e527139cf4a3bfb28fd6f2c;ced74...   \n",
       "69991  83a02dee50f2f83987d5490c8c9501609fd8f2df;c2150...   \n",
       "69992  27205842511218fa3dd0576405f70622232d03ee;1caa8...   \n",
       "\n",
       "                                         referenceTitles  \n",
       "0                                                         \n",
       "1      Low-Density Parity-Check Codes;Turbo decoder a...  \n",
       "2                                                         \n",
       "3                                                         \n",
       "4                                                         \n",
       "...                                                  ...  \n",
       "69988  Predictors for treatment failure after laparos...  \n",
       "69989  From sandbox to playground: dynamic virtual en...  \n",
       "69990  Nuclear surveillance and degradation of hypomo...  \n",
       "69991  Histone modifications: combinatorial complexit...  \n",
       "69992  Developmental neurotoxicity of ketamine: morph...  \n",
       "\n",
       "[69993 rows x 12 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "8a0309b1-beda-4931-ae04-5fe1d40b3848",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "Disruption of Fibroblast Growth Factor Signal Pathway Inhibits the Growth of Synovial Sarcomas: Potential Application of Signal Inhibitors to Molecular Target Therapy",
         "Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which are unknown. We investigated the involvement of fibroblast growth factor (FGF) signals in synovial sarcoma and evaluated the therapeutic effect of inhibiting the FGF signal. Experimental Design: The expression of 22 FGF and 4 FGF receptor (FGFR) genes in 18 primary tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR. Effects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein kinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory effects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo. Results: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed in neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma cell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal–regulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the FGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle arrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the FGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not p38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sarcoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2. Conclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibitory molecules will be of potential use for molecular target therapy in synovial sarcoma.",
         "",
         "https://aacrjournals.org/clincancerres/article-pdf/11/7/2702/1962059/2702-2712.pdf"
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "Shuffled iterative decoding",
         "Shuffled versions of iterative decoding of low-density parity-check codes and turbo codes are presented. The proposed schemes have about the same computational complexity as the standard versions, and converge faster. Simulations show that the new schedules offer better performance/complexity tradeoffs, especially when the maximum number of iterations has to remain small.",
         "Low-Density Parity-Check Codes;Turbo decoder architectures for low-density parity-check codes;Shuffled belief propagation decoding;A parallel decoder for low latency decoding of turbo product codes;Parallel versus sequential updating for belief propagation decoding;High throughput low-density parity-check decoder architectures;Good error-correcting codes based on very sparse matrices;Near optimum error correcting coding and decoding: turbo-codes;Multiple turbo codes for deep-space communications;Optimal decoding of linear codes for minimizing symbol error rate (Corresp.);Generic Architecture for LDPC Codes Decoding;Optimal decoding of linear codes for minimizing symbol error rate",
         "http://www.merl.com/publications/docs/TR2005-063.pdf"
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "Diphasic effects of Astragalus membranaceus BUNGE (Leguminosae) on vascular tone in rat thoracic aorta.",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus BUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath. In endothelium-intact aortic rings, A. membranaceus extract induced a significant dose-dependent relaxation of the rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(omega)-nitro-arginine methyl ester or methylthioninium chloride. In endothelium-denuded ones, the extract could dose-dependently relax the rings contracted by phenylephrine, not by KCl; and it could also attenuate contractile response to phenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca(2+)-free medium; but it failed to affect the CaCl(2)-induced enhancement of contractile response to phenylephrine in Ca(2+)-free medium. These results indicate that nitric oxide signaling and Ca(2+)-handling pathway are involved in the A. membranaceus extract-induced vasodilatation.",
         "",
         "https://www.jstage.jst.go.jp/article/bpb/28/8/28_8_1450/_pdf"
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "Critical roles of CD146 in zebrafish vascular development",
         "In this report, we use zebrafish as a model system to understand the importance of CD146 in vascular development. Endothelial‐specific expression of CD146 was verified by whole‐mount in situ hybridization. Suppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly developed intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood flow through the ISV region, despite that fluorescence microangiography showed that the ISVs were present. This finding suggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally, remodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of CD146 protein expression resulted in a circulation shunt that caused the circulation to by‐pass part of the caudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino oligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N terminus deleted in an in‐frame manner. This region, therefore, is likely to contain elements critical for CD146 function. This study provides the first in vivo functional assessment of CD146 in embryonic development by showing that knockdown of CD146 protein expression severely hinders vascular development in zebrafish. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley‐Liss, Inc.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/dvdy.20220"
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "Differential Contribution of Troponin I Phosphorylation Sites to the Endothelin-modulated Contractile Response*",
         "Cardiac troponin I is a phosphorylation target for endothelin-activated protein kinase C. Earlier work in cardiac myocytes expressing nonphosphorylatable slow skeletal troponin I provided evidence that protein kinase C-mediated cardiac troponin I phosphorylation accelerates relaxation. However, replacement with the slow skeletal isoform also alters the myofilament pH response and the Ca2+ transient, which could influence endothelin-mediated relaxation. Here, differences in the Ca2+ transient could not explain the divergent relaxation response to endothelin in myocytes expressing cardiac versus slow skeletal troponin I nor could activation of Na+/H+ exchange. Three separate clusters within cardiac troponin I are phosphorylated by protein kinase C, and we set out to determine the contribution of the Thr144 and Ser23/Ser24 clusters to the endothelin-mediated contractile response. Myocyte replacement with a cardiac troponin I containing a Thr144 substituted with the Pro residue found in slow skeletal troponin I resulted in prolonged relaxation in response to acute endothelin compared with control myocytes. Ser23/Ser24 also is a target for protein kinase C phosphorylation of purified cardiac troponin I, and although this cluster was not acutely phosphorylated in intact myocytes, significant phosphorylation developed within 1 h after adding endothelin. Replacement of Ser23/Ser24 with Ala indicated that this cluster contributes significantly to relaxation during more prolonged endothelin stimulation. Overall, results with these mutants provide evidence that Thr144 plays an important role in the acute acceleration of relaxation, whereas Ser23/Ser24 contributes to relaxation during more prolonged activation of protein kinase C by endothelin.",
         "",
         "http://www.jbc.org/article/S0021925820589394/pdf"
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "Programming: factors that influence success",
         "This paper documents a study, carried out in the academic year 2003-2004, on fifteen factors that may influence performance on a first year object-oriented programming module. The factors included prior academic experience, prior computer experience, self-perception of programming performance and comfort level on the module and specific cognitive skills. The study found that a student's perception of their understanding of the module had the strongest correlation with programming performance, r=0.76, p‹0.01. In addition, Leaving Certificate (LC) mathematics and science scores were shown to have a strong correlation with performance. A regression module, based upon a student's perception of their understanding of the module, gender, LC mathematics score and comfort level was able to account for 79% of the variance in programming performance results.",
         "The impact of prior experience in an information technology programming course sequence;Predictors of success and failure in a CS1 course;A multi-national, multi-institutional study of assessment of programming skills of first-year CS students;The effect of student attributes on success in programming;Contributing to success in an introductory computer science course: a study of twelve factors;Does it help to have some programming experience before beginning a computing degree program?;Factors affecting performance in first-year computing;The effect of a constructivist learning environment for field-dependent/independent students on achievement in introductory computer programming;What best predicts computer proficiency?;Predicting student performance in a beginning computer science class;Predicting the success of freshmen in a computer science major;Predicting student success in an introductory programming course;Predicting success in a first programming course;Investigating the relationship between the development of abstract reasoning and performance in an introductory programming class;Factors affecting course outcomes in introductory programming;An Overview of the Integration of Problem Based Learning into an existing Computer Science Programming Module;Discovering the likely performance of novice programmers on an introductory programming module;Differential Performance by gender and achievement in the Certificate Examination Results 2000/2001: Final Report, Dublin: National Council for Curriculum and Assessment;Examining Gender;Cognitive skills needed in computer programming;speciﬁc cognitive skills: as measured by an in-house cognitive test;previous academic and computer experience: as measured by performance in the Irish Leaving Certiﬁcate (LC) examinations in",
         "http://dl.acm.org/ft_gateway.cfm?id=1047480&type=pdf"
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "Intracellular and Surface Distribution of Monocyte Tissue Factor: Application to Intersubject Variability",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced monocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and shedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs). Methods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of TF-positive monocytes in HRs (32.0±3.5%) versus LRs (11.2±1.2%; P≤0.05), along with a stronger mean fluorescence intensity of TF signal in HRs (7.1±0.5 arbitrary units [AU]) compared with LRs (5.4±0.4 AU; P≤0.05). The LPS-treated blood of the HR group contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular accumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs versus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of surface TF, whereas MNCs from LRs only had a minor increase in surface TF levels. Conclusions—The higher availability of surface TF antigen on MNCs from HRs and TF-containing microparticles might make these individuals more susceptible to hypercoagulation.",
         "The production and availability of tissue thromboplastin in cellular populations of whole blood exposed to various concentrations of endotoxin. An assay for detection of endotoxin.;Tissue factor activity in whole blood.;Nicotinamide inhibits endotoxin‐induced monocyte tissue factor expression;Cellular activation responses in blood in relation to lipid pattern: healthy men and women in families with myocardial infarction or cancer;Tissue factor activity in human monocytes is regulated by plasma: implications for the high and low responder phenomenon;Role of Platelet P-Selectin and CD40 Ligand in the Induction of Monocytic Tissue Factor Expression;Eicosanoids and Iso-Eicosanoids: Constitutive, Inducible and Transcellular Biosynthesis in Vascular Disease;Cytokine gene polymorphism and heart transplant rejection.;Platelet-derived growth factor-BB and monocyte chemotactic protein-1 induce human peripheral blood monocytes to express tissue factor.;Association of smooth muscle cell tissue factor with caveolae.;Manifestation of cryptic fibroblast tissue factor occurs at detergent concentrations which dissolve the plasma membrane;A novel biological effect of platelet factor 4 (PF4): enhancement of LPS‐induced tissue factor activity in monocytes;Th1 and Th2 T-helper cells exert opposite regulatory effects on procoagulant activity and tissue factor production by human monocytes.;Granulocytes enhance LPS‐induced tissue factor activity in monocytes via an interaction with platelets;Cathepsin G-dependent platelet stimulation by activated polymorphonuclear leukocytes and its inhibition by antiproteinases: role of P-selectin-mediated cell-cell adhesion.;Relations between factor VIIa binding and expression of factor VIIa/tissue factor catalytic activity on cell surfaces.;Platelet activating factor enhancement of lipopolysaccharide‐induced tissue factor activity in monocytes: requirement of platelets and granulocytes;Dysregulation of in vitro cytokine production by monocytes during sepsis.;The role of arachidonic acid release and lipoxygenase pathway in lipopolysaccharide-induced thromboplastin activity in monocytes.;In vitro production of IL 1β, IL 1α, TNF and IL 2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation;Functional tissue factor is entirely cell surface expressed on lipopolysaccharide-stimulated human blood monocytes and a constitutively tissue factor-producing neoplastic cell line;Localization of tissue factor in the normal vessel wall and in the atherosclerotic plaque.;Cytokine-induced procoagulant activity in monocytes and endothelial cells. Further enhancement by cyclosporine.;Tumor necrosis factor induces tissue factor-like activity in human leukemia cell line U937 and peripheral blood monocytes.;Enhancement of mononuclear procoagulant activity by platelet 12-hydroxyeicosatetraenoic acid.;The stimulatory effect of platelets and platelet membranes on the procoagulant activity of leukocytes.;The Role of Protein Synthesis on the Generation of Tissue Factor Activity by Leukocytes;Monocytes.;The high responder phenomenon: enhancement of LPS induced tissue factor activity in monocytes by platelets and granulocytes.;Selective cellular expression of tissue factor in human tissues. Implications for disorders of hemostasis and thrombosis.;In vitro production of IL 1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation.;Mononuclear cell tissue factor: cell of origin and requirements for activation.;After blocking with 5% non-fat skimmed milk, membranes were incubated with mouse anti-human TF monoclonal antibodies;After Lymphoprep isolation, total RNA was extracted from MNC pellets with an;OR) anti-human TF monoclonal antibodies (10 µg/ml) for 30 min on ice. Mouse monoclonal antibodies recognizing extracellular domain of human TF were prepared by the Monoclonal Antibody Facility",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.ATV.0000168413.29874.d7"
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "Potential Clindamycin Resistance in Clindamycin-Susceptible, Erythromycin-Resistant Staphylococcus aureus: Report of a Clinical Failure",
         "ABSTRACT The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin clinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-susceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of methicillin-resistant S. aureus strains were D-test positive.",
         "",
         "https://europepmc.org/articles/pmc549252?pdf=render"
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         "Symmetric and asymmetric Slepian-Wolf codes with systematic and nonsystematic linear codes",
         "We propose a constructive approach for distributed source coding of correlated binary sources using linear channel codes that can achieve any point of the Slepian-Wolf achievable rate region. Our approach is very intuitive and can be used with systematic and nonsystematic linear codes. Moreover, the proposed coding strategy can easily be extended to the case of more than two sources.",
         "Design of Slepian-Wolf codes by channel code partitioning;On some new approaches to practical Slepian-Wolf compression inspired by channel coding;Distributed code constructions for the entire Slepian-Wolf rate region for arbitrarily correlated sources;Compression of binary sources with side information at the decoder using LDPC codes;Compression with side information using turbo codes;Compression of correlated binary sources using turbo codes;The capacity of low-density parity-check codes under message-passing decoding;Distributed source coding: symmetric rates and applications to sensor networks;Distributed source coding using syndromes (DISCUS): design and construction;Noiseless coding of correlated information sources",
         "http://www.commsp.ee.ic.ac.uk/~ng1/pdf/gehrigD_CL_Jan05.pdf"
        ],
        [
         "9",
         "000a2234245d642c786a7355a15623444c33c1dc",
         "Selective Defect in Antigen-Induced TCR Internalization at the Immune Synapse of CD8 T Cells Bearing the ZAP-70(Y292F) Mutation1",
         "Cbl proteins have been implicated in ligand-induced TCR/CD3 down-modulation, but underlying mechanisms are unclear. We analyzed the effect of mutation of a cbl-binding site on ZAP-70 (ZAP-Y292F) on dynamics, internalization, and degradation of the TCR/CD3 complex in response to distinct stimuli. Naive CD8 T cells expressing the P14 transgenic TCR from ZAP-Y292F mice were selectively affected in TCR/CD3 down-modulation in response to antigenic stimulation, whereas neither anti-CD3 Ab-, and PMA-induced TCR down-modulation, nor constitutive receptor endocytosis/cycling were impaired. We further established that the defect in TCR/CD3 down-modulation in response to Ag was paralleled by an impaired TCR/CD3 internalization and CD3ζ degradation. Analysis of T/APC conjugates revealed that delayed redistribution of TCR at the T/APC contact zone was paralleled by a delay in TCR internalization in the synaptic zone in ZAP-Y292F compared with ZAP-wild-type T cells. Cbl recruitment to the synapse was also retarded in ZAP-Y292F T cells, although F-actin and LFA-1 redistribution was similar for both cell types. This study identifies a step involving ZAP-70/cbl interaction that is critical for rapid internalization of the TCR/CD3 complex at the CD8 T cell/APC synapse.",
         "",
         "https://www.jimmunol.org/content/jimmunol/175/5/3140.full.pdf"
        ],
        [
         "10",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "Applying Machine Learning Techniques to Improve Linux Process Scheduling",
         "In this work we use Machine Learning (ML) techniques to learn the CPU time-slice utilization behavior of known programs in a Linux system. Learning is done by an analysis of certain static and dynamic attributes of the processes while they are being run. Our objective was to discover the most important static and dynamic attributes of the processes that can help best in prediction of CPU burst times which minimize the process TaT (Turn-around-Time). In our experimentation we modify the Linux Kernel scheduler (version 2.4.20-8) to allow scheduling with customized time slices. The \"Waikato Environment for Knowledge Analysis\" (Weka), an open source machine-learning tool is used to find the most suitable ML method to characterize our programs. We experimentally find that the C'4.5 Decision Tree algorithm most effectively solved the problem. We find that predictive scheduling could reduce TaT in the range of 1.4% to 5.8%. This was due to a reduction in the number of context switches needed to complete the process execution. We find our result interesting in the context that generally operating systems presently never make use of a program's previous execution history in their scheduling behavior.",
         "Performance Contracts: Predicting and Monitoring Grid Application Behavior;The design, implementation and initial evaluation of an advanced knowledge-based process scheduler;Understanding the Linux Kernel;Predicting Application Run Times Using Historical Information;A Historical Application Profiler for Use by Parallel Schedulers;Context Switching and IPC Performance Comparison between uClinux and Linux on the ARM9 based Processor;Characterizing Process Execution Behaviour Using Machine Learning Techniques;The Pearson Education;Linux Kernel Development , 1st ed;Correlation-based Feature Selection for Machine Learning;A Dynamically Adaptive CPU Scheduler;Linux Kernel 2.4 Internals;Co-rrelation based feature selection for Machine Learning, Master Thesis, Department of Computer Science, University of Waikato;Machine Learning , 1st ed, pp: 52-75, 154-183, 230-244, The;WEKA: The Waikato Environment for Knowledge Analysis;ELF: From The Programmer's Perspective;The Design of the UNIX Operating System;Build the knowledge base of static and dynamic characteristics of the programs from the run traces obtained in step 1 and train them with the C4.5 decision tree algorithm;is a Masters student in the Department of Computer and Informations Sciences;We send this STS information to modi(cid:2)ed scheduler through a system call;The scheduler instructs the CPU such that CPU allocates STS ticks to ’X’;decision tree will classify ’X’ and output the STS. per second.(cid:2)le;Aivazian , Tigran , Linux Kernel 2 . 4 Internals , The Linux Documentation Project , August , 2002",
         "http://www.cs.ucr.edu/%7Ekishore/papers/tencon.pdf"
        ],
        [
         "11",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "Neuronal Leucine-Rich Repeat Protein 4 Functions in Hippocampus-Dependent Long-Lasting Memory",
         "ABSTRACT Neuronal leucine-rich repeat proteins (NLRRs) are type I transmembrane proteins and expressed in neuronal tissues, but their function remains unknown. Here, we describe the identification and characterization of a new member of the NLRR family, NLRR4, and its potential role in long-lasting memory. We generated NLRR4-deficient (NLRR4−/−) mice and found that they showed impaired memory retention. In hippocampus-dependent learning tasks, NLRR4−/− mice were able to learn and maintain the memories for one day but unable to retain the memories for four days after learning. In contrast, in a hippocampus-independent task, NLRR4−/− mice were able to retain the memory normally for at least seven days. These results suggest that NLRR4 plays a key role in hippocampus-dependent long-lasting memory.",
         "",
         "https://europepmc.org/articles/pmc1087730?pdf=render"
        ],
        [
         "12",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "Semantic video analysis for adaptive content delivery and automatic description",
         "We present an encoding framework which exploits semantics for video content delivery. The video content is organized based on the idea of main content message. In the work reported in this paper, the main content message is extracted from the video data through semantic video analysis, an application-dependent process that separates relevant information from non relevant information. We use here semantic analysis and the corresponding content annotation under a new perspective: the results of the analysis are exploited for object-based encoders, such as MPEG-4, as well as for frame-based encoders, such as MPEG-1. Moreover, the use of MPEG-7 content descriptors in conjunction with the video is used for improving content visualization for narrow channels and devices with limited capabilities. Finally, we analyze and evaluate the impact of semantic video analysis in video encoding and show that the use of semantic video analysis prior to encoding sensibly reduces the bandwidth requirements compared to traditional encoders not only for an object-based encoder but also for a frame-based encoder.",
         "Automatic foveation for video compression using a neurobiological model of visual attention;Text information extraction in images and video: a survey;DAVE: a system for quality driven adaptive video delivery;Video transmission adaptation on mobile devices;Visual attention for region of interest coding in JPEG 2000;Semantic segmentation and description for video transcoding;Resource adaptation based on MPEG-21 usage environment descriptions;Video transcoding architectures and techniques: an overview;Metadata-driven multimedia access;Semantic transcoding for live video server;Multiple video object tracking in complex scenes;Complexity-quality analysis of transcoding architectures for reduced spatial resolution;Instantly indexed multimedia databases of real world events;Face detection in color images;MPEG-7 description of generic video objects for scene reconstruction;A new content-based hybrid video transcoding method;Video Processing and Communications;Media conversions to support mobile users;Object-based transcoding for adaptable video content delivery;Overview of fine granularity scalability in MPEG-4 video standard;Video object extraction based on adaptive background and statistical change detection;Video transcoding for universal multimedia access;Heterogeneous Video Transcoding to Lower Spatio-Temporal Resolutions and Different Encoding Formats;Scalable multimedia delivery for pervasive computing;Adapting Multimedia Internet Content for Universal Access;A frequency-domain video transcoder for dynamic bit-rate reduction of MPEG-2 bit streams;Multimedia content description in the InfoPyramid;Objective methods for assessment of video quality : state of the art;On the methods and performances of rational downsizing video transcoding;Shadow detection in image sequences;Video Transcoding Architectures and Techniques : An Overview;Visual radio: should we paint pictures with workds, or pictures?;Dave: A system for qual ity driven adaptive video delivery,”Proceedings of the 5th ACM SIGMM international workshop on Multimedia information retriev;Processing and;Information technology – coding of audio-visual objects – part 2 visual–amendment 2: Streaming video profiles;Statistics, 3 ed;Continuous speech recognit  io;Submitted by;Information technology — Generic coding of moving pictures and associated audio information — Part 2: Video;SUBJECTIVE VIDEO QUALITY ASSESSMENT METHODS FOR MULTIMEDIA APPLICATIONS",
         "https://infoscience.epfl.ch/record/87259/files/Cavallaro2005_1275.pdf"
        ],
        [
         "13",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "Trends in Drug Resistance, Serotypes, and Molecular Types of Streptococcus pneumoniae Colonizing Preschool-Age Children Attending Day Care Centers in Lisbon, Portugal: a Summary of 4 Years of Annual Surveillance",
         "ABSTRACT Of the nasopharyngeal cultures recovered from 942 day care center (DCC) attendees in Lisbon, Portugal, 591 (62%) yielded Streptococcus pneumoniae during a surveillance performed in February and March of 1999. Forty percent of the isolates were resistant to one or more antimicrobial agents. In particular, 2% were penicillin resistant and 20% had intermediate penicillin resistance. Multidrug resistance to macrolides, lincosamides, and tetracycline was the most frequent antibiotype (17% of all isolates). Serotyping and molecular typing by pulsed-field gel electrophoresis were performed for 202 out of 237 drug-resistant pneumococci (DRPn). The most frequent serotypes were 6B (26%), 14 (22%), 19F (16%), 23F (10%), and nontypeable (12%). The majority (67%) of the DRPn strains were representatives of nine international clones included in the Pneumococcal Molecular Epidemiology Network; eight of them had been detected in previous studies. Fourteen novel clones were identified, corresponding to 26% of the DRPn strains. The remaining 7% of the strains were local clones detected in our previous studies. Comparison with studies conducted since 1996 in Portuguese DCCs identified several trends: (i) the rate of DRPn frequency has fluctuated between 40 and 50%; (ii) the serotypes most frequently recovered have remained the same; (iii) nontypeable strains appear to be increasing in frequency; and (iv) a clone of serotype 33F emerged in 1999. Together, our observations highlight that the nasopharynxes of children in DCCs are a melting pot of successful DRPn clones that are important to study and monitor if we aim to gain a better understanding on the epidemiology of this pathogen.",
         "",
         "https://europepmc.org/articles/pmc1081291?pdf=render"
        ],
        [
         "14",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "Laboratory Exposures to Brucellae and Implications for Bioterrorism",
         "Laboratory exposures to brucellae indicate unfamiliarity with organism.",
         "Laboratory-acquired Brucellosis;Acute hepatitis due to brucellosis in a laboratory technician.;Prevention of Laboratory-Acquired Brucellosis;Analysis of risk factors for laboratory-acquired brucella infections.;Use of the Isolator 1.5 Microbial Tube for Detection of Brucella melitensis in Synovial Fluid;Bacterial Pathogens as Biological Weapons and Agents of Bioterrorism;Unusual Complication of Breast Implants: Brucella Infection;Brucellosis in laboratory workers at a Saudi Arabian hospital.;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Brucella abortus Infection Acquired in Microbiology Laboratories;Posttreatment Follow-Up of Brucellosis by PCR Assay;Detection of Brucellae in Blood Cultures;Rapid diagnosis of human brucellosis by peripheral-blood PCR assay;Failure of short antimicrobial treatments for human brucellosis;Comparison of BACTEC 9240 Peds Plus medium and isolator 1.5 microbial tube for detection of Brucella melitensis from blood cultures;Exposure to Brucella in the laboratory;Brucellosis from sniffing bacteriological cultures;Grand Rounds—Hammersmith Hospital: A case of laboratory acquired brucellosis;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Live Brucella spp. fail to induce tumor necrosis factor alpha excretion upon infection of U937-derived phagocytes;Outbreak of Brucella melitensis among microbiology laboratory workers;Unsuspected brucellosis diagnosed in a child as a result of an outbreak of laboratory-acquired brucellosis.;Brucellosis: imported and laboratory-acquired cases, and an overview of treatment trials.;Biochemical mis-identification of Brucella melitensis and subsequent laboratory-acquired infections.;Laboratory diagnosis of brucella infection: some pitfalls;Outbreak of Brucella melitensis among microbiology laboratory workers in a community hospital.;An outbreak of Brucella melitensis infection by airborne transmission among laboratory workers.;Laboratory-acquired meningitis caused by Brucella abortus strain 19.;Past and present hazards of working with infectious agents.;From the Center for Disease Control;Brucella canis: A cause of undulant fever;An unusual outbreak of brucellosis.;Brucellosis: I. Laboratory-Acquired Acute Infection;Brucellosis in the United States.;Brucella species;Brucellosis: An occupational hazard for medical laboratory personnel. Report of five cases;Beer-Sheva 84101, Israel, fax: 972-8640-3541;Prevention of laboratory-acquired brucellosis.;(revised). Geneva: World Health Organization;Basic protocols for level A laboratories for the presumptive identification of Brucella species. Washington: American Society for Microbiology;Exposure of hospital personnel to Brucella melitensis and occurrence of laboratory-acquired disease in an endemic area.;Biosafety in microbiological and biomedical laboratories;Laboratory-associated infections: incidence, fatalities, causes, and prevention.;Principles and practice of infectious diseases. Vols 1 and 2.;Brucellosis in the United States, 1965-1974.;Laboratory Infections Due to Brucella",
         "https://wwwnc.cdc.gov/eid/article/11/8/pdfs/04-1197.pdf"
        ],
        [
         "15",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "Safe motion planning in dynamic environments",
         "This paper addresses the problem of motion planning (MP) in dynamic environments. It is first argued that dynamic environments impose a real-time constraint upon MP: it has a limited time only to compute a motion, the time available being a function of the dynamicity of the environment. Now, given the intrinsic complexity of MP, computing a complete motion to the goal within the time available is impossible to achieve in most real situations. Partial motion planning (PMP) is the answer to this problem proposed in this paper. PMP is a motion planning scheme with an anytime flavor: when the time available is over, PMP returns the best partial motion to the goal computed so far. Like reactive navigation scheme, PMP faces a safety issue: what guarantee is there that the system will never end up in a critical situation yielding an inevitable collision? The answer proposed in this paper to this safety issue relies upon the concept of inevitable collision states (ICS). ICS takes into account the dynamics of both the system and the moving obstacles. By computing ICS-free partial motion, the system safety can be guaranteed. Application of PMP to the case of a car-like system in a dynamic environment is presented.",
         "Motion prediction for moving objects: a statistical approach;Inevitable collision states. A step towards safer robots?;Towards real-time global motion planning in a dynamic environment using the NLVO concept;An integrated approach to goal-directed obstacle avoidance under dynamic constraints for dynamic environments;Real-time randomized path planning for robot navigation;Improved fast replanning for robot navigation in unknown terrain;Learning motion patterns of persons for mobile service robots;Reactive navigation for non-holonomic robots using the ego-kinematic space;Randomized Kinodynamic Motion Planning with Moving Obstacles;Real-time motion planning for agile autonomous vehicles;Kinodynamic motion planning amidst moving obstacles;VFH/sup */: local obstacle avoidance with look-ahead verification;Randomized Kinodynamic Planning;High-speed navigation using the global dynamic window approach;Motion Planning in Dynamic Environments Using Velocity Obstacles;The dynamic window approach to collision avoidance;Probabilistic roadmaps for path planning in high-dimensional configuration spaces;The curvature-velocity method for local obstacle avoidance;The Focussed D* Algorithm for Real-Time Replanning;Reflexive collision avoidance: a generalized approach;Motion planning in the presence of moving obstacles;Robot Motion Planning;Sensor-Based Motion Control for Mobile Robots;Kinodynamic planning in a structured and time-varying 2-D workspace",
         "https://hal.inria.fr/inria-00182046/file/2005-petti-fraichard-iros.pdf"
        ],
        [
         "16",
         "0012aecca19e4d3e41741599b1dbd00fb46d0c22",
         "Glutathione S-transferase T1 polymorphisms are associated with outcome in colorectal cancer.",
         "Colorectal cancer (CRC) remains a significant cause of mortality accounting for approximately 10% of all deaths from malignancy in the western world. Polymorphism in the glutathione S-transferase GSTT1 gene has been associated with CRC risk in some but not all studies. In this study, we examined associations between GSTT1 genotypes and CRC risk, and prognosis in 361 cases and 881 unrelated controls. GSTT1 null was associated with a small but significant increase in risk (P = 0.0006, odds ratio (OR) = 1.65, 95% confidence interval (CI) = 1.22-2.24). GSTT1 null was also associated with a significantly younger age at diagnosis (mean 65.2 years) compared with GSTT1 A (mean 67.6 years, P = 0.031). There were no significant associations between GSTT1 genotypes and clinical factors (e.g. Dukes stage, differentiation and tumour node metastasis classification) in the total case group. However, following stratification by age (<70 versus > or =70 years at diagnosis), in the patients diagnosed <70 years of age, GSTT1 null was more common in Dukes grade A/B tumours (P = 0.046), stage T1/T2 tumours (P = 0.053) and those with a pushing margin (P = 0.066). We also identified associations between GSTT1 null and increased prevalence of host lymphocyte response, particularly in the younger patients (P = 0.036). Furthermore, GSTT1 null was associated with improved survival in younger patients (P = 0.017, hazards ratio (HR) = 0.52, 95% CI = 0.31-0.89) but poorer survival in older patients (P = 0.017, HR = 1.89, 95% CI = 1.12-3.20). We proposed a model based on the dual functionality of GSTT1 to explain these contrasting results. We suggest that the null genotype is associated with improved immune response in younger patients, but poorer detoxification in older patients. These findings may also provide an explanation for the contrasting finding of other studies on the role of this gene in CRC.",
         "",
         "https://academic.oup.com/carcin/article-pdf/26/12/2157/7088641/bgi195.pdf"
        ],
        [
         "17",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "The N‐terminus of B96Bom, a Bombyx mori G‐protein‐coupled receptor, is N‐myristoylated and translocated across the membrane",
         "In eukaryotic cellular proteins, protein N‐myristoylation has been recognized as a protein modification that occurs mainly on cytoplasmic or nucleoplasmic proteins. In this study, to search for a eukaryotic N‐myristoylated transmembrane protein, the susceptibility of the N‐terminus of several G‐protein‐coupled receptors (GPCRs) to protein N‐myristoylation was evaluated by in vitro and in vivo metabolic labeling. It was found that the N‐terminal 10 residues of B96Bom, a Bombyx mori GPCR, efficiently directed the protein N‐myristoylation. Analysis of a tumor necrosis factor (TNF) fusion protein with the N‐terminal 90 residues of B96Bom at its N‐terminus revealed that (a) transmembrane domain 1 of B96Bom functioned as a type I signal anchor sequence, (b) the N‐myristoylated N‐terminal domain (58 residues) was translocated across the membrane, and (c) two N‐glycosylation motifs located in this domain were efficiently N‐glycosylated. In addition, when Ala4 in the N‐myristoylation motif of B96Bom90‐TNF, Met‐Gly‐Gln‐Ala‐Ala‐Thr(1–6), was replaced with Asn to generate a new N‐glycosylation motif, Asn‐Ala‐Thr(4–6), efficient N‐glycosylation was observed on this newly introduced N‐glycosylation site in the expressed protein. These results indicate that the N‐myristoylated N‐terminus of B96Bom is translocated across the membrane and exposed to the extracellular surface. To our knowledge, this is the first report showing that a eukaryotic transmembrane protein can be N‐myristoylated and that the N‐myristoylated N‐terminus of the protein can be translocated across the membrane.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2004.04487.x"
        ],
        [
         "18",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "Uptake of denatured collagen into hepatic stellate cells: evidence for the involvement of urokinase plasminogen activator receptor-associated protein/Endo180.",
         "Tissue remodelling is dependent on the integration of signals that control turnover of ECM (extracellular matrix). Breakdown and endocytosis of collagen, a major component of the ECM, is central to this process. Whereas controlled secretion of matrix-degrading enzymes (such as matrix metalloproteinases) has long been known to mediate ECM breakdown, it is becoming clear that uPARAP/Endo180 (where uPARAP stands for urokinase plasminogen activator receptor-associated protein) serves as a receptor that mediates endocytosis of collagen by several types of cells. In the liver, the stellate cells play a major role in turnover of ECM including collagens. These cells synthesize various collagens and also produce matrix metalloproteinases. In the present study, we investigated the capacity of rat hepatic stellate cells to endocytose and degrade 125I-labelled heat-denatured collagen I. It was found that the collagen is efficiently taken up and degraded by these cells. Degradation was inhibited by inhibitors of lysosomal proteases (leupeptin and E-64d) and the vacuolar proton pump (concanamycin A), indicating that it takes place in lysosomes. Furthermore, endocytosed FITC-labelled collagen was shown to reach late endocytic compartments in which it colocalized with LysoTracker (a marker of late endocytic compartments). Competition experiments showed that uPA and unlabelled collagen are capable of inhibiting binding and uptake of [125I]collagen in a dose-dependent manner. Moreover, Western-blot analysis of cell lysate (using a polyclonal rabbit human-Endo180 antiserum) revealed a single band at 180 kDa. In addition, the antiserum was capable of reducing [125I]collagen binding to the cell surface. Finally, using two primers designed from the human uPARAP/Endo180 mRNA sequence, the expression of uPARAP/Endo180 mRNA was detected by reverse transcriptase-PCR. These results together suggest that uPARAP/Endo180 mediates endocytosis of collagen in rat liver stellate cells.",
         "uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV.;Identification and characterization of the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor.;A targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen uptake;Liver extracellular matrix in health and disease;Hepatic stellate cells: unique characteristics in cell biology and phenotype.;uPARAP/Endo180 is essential for cellular uptake of collagen and promotes fibroblast collagen adhesion;Characterization of Sugar Binding by the Mannose Receptor Family Member, Endo180*;The mannose receptor family.;The C-type Lectin Receptor Endo180 Displays Internalization and Recycling Properties Distinct from Other Members of the Mannose Receptor Family*;Collagen-receptor signaling in health and disease.;The Urokinase Plasminogen Activator Receptor–Associated Protein/Endo180 Is Coexpressed with Its Interaction Partners Urokinase Plasminogen Activator Receptor and Matrix Metalloprotease-13 during Osteogenesis;Hepatic Blood Flow Regulation by Stellate Cells in Normal and Injured Liver;Extracellular Matrix Degradation and the Role of Hepatic Stellate Cells;Wortmannin-sensitive trafficking steps in the endocytic pathway in rat liver endothelial cells.;Effects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy.;Endo180, an endocytic recycling glycoprotein related to the macrophage mannose receptor is expressed on fibroblasts, endothelial cells and macrophages and functions as a lectin receptor.;Selective adhesion of macrophages to denatured forms of type I collagen is mediated by scavenger receptors.;A Urokinase Receptor-associated Protein with Specific Collagen Binding Properties*;The C‐type lectin superfamily in the immune system;An orphan receptor tyrosine kinase family whose members serve as nonintegrin collagen receptors.;The discoidin domain receptor tyrosine kinases are activated by collagen.;Vascular Smooth Muscle αvβ3 Integrin Mediates Arteriolar Vasodilation in Response to RGD Peptides;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Cell adhesion receptors for native and denatured type I collagens and fibronectin in rabbit arterial smooth muscle cells in culture.;An endothelial cell receptor for plasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of t-PA-dependent plasminogen activation.;Differential expression of matrix-metalloproteinase-1 and -2 genes in normal and fibrotic human liver.;Transfer of retinol-binding protein from HepG2 human hepatoma cells to cocultured rat stellate cells.;Fat-storing cells as liver-specific pericytes. Spatial dynamics of agonist-stimulated intracellular calcium transients.;Affinity of integrins for damaged extracellular matrix: αvβ3 binds to denatured collagen type I through RGD sites;Identification of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin in collagen.;Internalization of retinol-binding protein in parenchymal and stellate cells of rat liver.;p180, a novel recycling transmembrane glycoprotein with restricted cell type expression;The alpha 2 beta 1 integrin cell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen.;Activation of cultured rat hepatic lipocytes by Kupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation of cell proliferation via induction of platelet-derived growth factor receptors.;Characterization of desmin‐positive rat liver sinusoidal cells;Hepatic lipocytes: the principal collagen-producing cells of normal rat liver.;A radioiodinated, intracellularly trapped ligand for determining the sites of plasma protein degradation in vivo.;The effect of leupeptin on intracellular digestion of asialofetuin in rat hepatocytes.;The structures of the galactose-containing sugar chains of ovalbumin.;HIGH-YIELD PREPARATION OF ISOLATED RAT LIVER PARENCHYMAL CELLS;Distribution of Endo180 receptor and ligand in developing articular cartilage.;Endocytic uptake of advanced glycation end products by mouse liver sinusoidal endothelial cells is mediated by a scavenger receptor distinct from the macrophage scavenger receptor class A.;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-α and TGF-β1;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1.;Vascular smooth muscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD peptides.;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Affinity of integrins for damaged extracellular matrix: alpha v beta 3 binds to denatured collagen type I through RGD sites.;Expression of the gene of the α-smooth muscle-actin isoform in rat liver and in rat fat-storing (ITO) cells;Acidification of the endocytic and exocytic pathways.;Collagen production by rat liver fat-storing cells in primary culture.;Newly administered [3H]retinol is transferred from hepatocytes to stellate cells in liver for storage.;Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes), their related structure in and around the liver sinusoids, and vitamin A-storing cells in extrahepatic organs.;Preparation of isolated rat liver cells.;Hepatic stellate cells: morphology, function, and regulation;Isolation of perisinusoidal stellate cells (vitamin A-storting cells, fat-storting cells) of the rat liver",
         "https://europepmc.org/articles/pmc1134930?pdf=render"
        ],
        [
         "19",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "ISEcp1B-Mediated Transposition of blaCTX-M in Escherichia coli",
         "ABSTRACT Several expanded-spectrum β-lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enterobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional mechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.",
         "",
         "https://europepmc.org/articles/pmc538921?pdf=render"
        ],
        [
         "20",
         "0018ece872bb78b06eb26ea35d5ed27f53702ce6",
         "Protein function prediction using the Protein Link EXplorer (PLEX)",
         "UNLABELLED\nWe introduce the Protein Link EXplorer (PLEX), a web-based environment that allows the construction of a phylogenetic profile for any given amino acid sequence, and its comparison with profiles of approximately 350,000 predicted genes from 89 genomes, as a means of interactively identifying functionally linked genes and predicting protein function. PLEX can be searched iteratively and also enables searches for chromosomal gene neighbors and Rosetta Stone linkages. PLEX search results are accompanied by quantitative estimates of linkage confidence, enabling users to take advantage of coinheritance, operon and gene fusion-based methods for inferring gene function and reconstructing cellular systems and pathways.\n\n\nAVAILABILITY\nhttp://bioinformatics.icmb.utexas.edu/plex",
         "",
         "https://academic.oup.com/bioinformatics/article-pdf/21/10/2558/48972005/bioinformatics_21_10_2558.pdf"
        ],
        [
         "21",
         "001a3ef673ee3e7703c7db597386a2162a2dc402",
         "How and When Selection Experiments Might Actually be Useful1",
         "Abstract Laboratory natural selection and artificial selection are vital tools for addressing specific questions about evolutionary patterns of variation. Laboratory natural selection can illuminate whether a putative selective agent is capable of generating long-term, sustained changes in individual traits and suites of traits. Artificial selection is the essential tool for understanding the general evolvability of traits and the extent to which genetic correlations constrain evolution. We review the contexts in which each type of experiment seems capable of offering key insights into important evolutionary issues. We also discuss theoretical and methodological considerations that play critical roles in designing selection experiments that are relevant to evolutionary patterns of trait variation. In particular, we focus on the critical role of selection intensity and the consequences of experiments with different intensities. While selection experiments are not practical in many cases, sophisticated selection experiments—designed with careful consideration of the theory of selection—should be taken beyond model organisms and used in well-chosen natural systems to understand natural patterns of variation.",
         "",
         "https://academic.oup.com/icb/article-pdf/45/3/391/1674343/i1540-7063-045-03-0391.pdf"
        ],
        [
         "22",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "Phylogenetic relationships of Irkut and West Caucasian bat viruses within the Lyssavirus genus and suggested quantitative criteria based on the N gene sequence for lyssavirus genotype definition.",
         "None",
         "Novel Lyssaviruses Isolated from Bats in Russia;Bat lyssaviruses (Aravan and Khujand) from Central Asia: phylogenetic relationships according to N, P and G gene sequences.;A comparison of DNA vaccines for the rabies-related virus, Mokola.;New Lyssavirus Genotype from the Lesser Mouse-eared Bat (Myotis blythi), Kyrghyzstan;A molecular epidemiological study of Australian bat lyssavirus.;Competition Commission inquiry into POMs.;Characterisation of an Australian bat lyssavirus variant isolated from an insectivorous bat.;Phylogenetic comparison of the genus Lyssavirus using distal coding sequences of the glycoprotein and nucleoprotein genes;Lyssavirus P gene characterisation provides insights into the phylogeny of the genus and identifies structural similarities and diversity within the encoded phosphoprotein.;Increased positive selection pressure in persistent (SSPE) versus acute measles virus infections.;European Bat Lyssavirus Infection in Spanish Bat Populations;Serologic Evidence of Lyssavirus Infections among Bats, the Philippines;Genetic constraints and the adaptive evolution of rabies virus in nature.;Expression of the B-cell and T-cell epitopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG and induction of an humoral response in mice.;MEGA2: molecular evolutionary genetics analysis software;Extensive Attenuation of Rabies Virus by Simultaneously Modifying the Dynein Light Chain Binding Site in the P Protein and Replacing Arg333 in the G Protein;Molecular basis for the interaction between rabies virus phosphoprotein P and the dynein light chain LC8: dissociation of dynein-binding properties and transcriptional functionality of P.;Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains;Host Switching in Lyssavirus History from the Chiroptera to the Carnivora Orders;Evidence of Two Lyssavirus Phylogroups with Distinct Pathogenicity and Immunogenicity;Antigenic and genetic divergence of rabies viruses from bat species indigenous to Canada.;Codon-substitution models for heterogeneous selection pressure at amino acid sites.;Sequencing and position analysis of the glycoprotein gene of four Chinese rabies viruses;Ecology and evolution of rabies virus in Europe.;Phylogeographic patterns exhibited by Ontario rabies virus variants;Chimeric Lyssavirus Glycoproteins with Increased Immunological Potential;Multiple sequence alignment with Clustal X.;Rabies virus quasispecies: implications for pathogenesis.;DNA-based immunization for exploring the enlargement of immunological cross-reactivity against the lyssaviruses.;PAML: a program package for phylogenetic analysis by maximum likelihood;Evolution of European bat lyssaviruses.;The complete Mokola virus genome sequence: structure of the RNA-dependent RNA polymerase.;A Biographical History of Veterinary Pathology;Ten-year survey of British bats for the existence of rabies;Encephalitis caused by a Lyssavirus in fruit bats in Australia.;Characterization of a unique variant of bat rabies virus responsible for newly emerging human cases in North America.;The design of strain-specific polymerase chain reactions for discrimination of the racoon rabies virus strain from indigenous rabies viruses of Ontario.;Genetic polymorphism in the rabies virus nucleoprotein gene.;Molecular epidemiology of lyssaviruses: focus on the glycoprotein and pseudogenes.;Molecular diversity of the Lyssavirus genus.;Isolation and characterization of 115 street rabies virus isolates from Ethiopia by using monoclonal antibodies: identification of 2 isolates as Mokola and Lagos bat viruses.;[A lyssavirus with an unusual antigenic structure isolated from a bat in southern Kyrgyzstan].;Rapid sequence evolution of street rabies glycoprotein is related to the highly heterogeneous nature of the viral population.;PCR technique as an alternative method for diagnosis and molecular epidemiology of rabies virus.;T helper cell epitope of rabies virus nucleoprotein defined by tri‐ and tetrapeptides;Molecular cloning and complete nucleotide sequence of the attenuated rabies virus SAD B19.;Rabies-related Yuli virus, identification with a panel of monoclonal antibodies.;Efficacy of rabies vaccines against Duvenhage virus isolated from European house bats (Eptesicus serotinus), classic rabies and rabies-related viruses.;Immunity against the European bat rabies (Duvenhage) virus induced by rabies vaccines: an experimental study in mice.;Localization and immunological characterization of antigenic domains of the rabies virus internal N and NS proteins.;Walking along the rabies genome: is the large G-L intergenic region a remnant gene?;HUMAN RABIES OF BAT ORIGIN IN EUROPE;Rabies in the Tropics;Rabies-related viruses.;Rabies virus infection of a flying fox bat, Pteropus policephalus in Chandigarh, Northern India.;Giulio Alfredo Maccacaro.;Experimental infection of dogs and monkeys with two rabies serogroup viruses, Lagos bat and Mokola (IbAn 27377): clinical, serologic, virologic, and fluorescent-antibody studies.;An unusual case of human rabies thought to be of chiropteran origin.;Two African Viruses Serologically and Morphologically Related to Rabies Virus;Isolation of Rabies Virus from Fruit Bats in Thailand;Isolation of a virus from Nigerian fruit bats.;New Cases of Mokola Virus Infection in South Africa: A Genotypic Comparison of Southern African Virus Isolates;BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT;MODELTEST: testing the model of DNA substitution;Immunoprotection by rabies virus nucleoprotein.;Nucleotide and deduced amino acid sequences of the nominal nonstructural phosphoprotein of the ERA, PM and CVS-11 strains of rabies virus.;Rabies and Rabies-related viruses in Zimbabwe historical, virological and ecological aspects.;Rabies-Related Viruses;Characterization of an antigenic determinant of the glycoprotein that correlates with pathogenicity of rabies virus.;Antigenic characterization of the bangui strain (ANCB-672d) of lagos bat virus",
         "https://zenodo.org/record/1259441/files/article.pdf"
        ],
        [
         "23",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         "Neuronal Signals in the Monkey Basolateral Amygdala during Reward Schedules",
         "The amygdala is critical for connecting emotional reactions with environmental events. We recorded neurons from the basolateral complex of two monkeys while they performed visually cued schedules of sequential color discrimination trials, with both valid and random cues. When the cues were valid, the visual cue, which was present throughout each trial, indicated how many trials remained to be successfully completed before a reward. Seventy-six percent of recorded neurons showed response selectivity, with the selectivity depending on some aspects of the current schedule. After a reward, when the monkeys knew that the upcoming cue would be valid, 88 of 246 (36%) neurons responded between schedules, seemingly anticipating the receiving information about the upcoming schedule length. When the cue appeared, 102 of 246 (41%) neurons became selective, at this point encoding information about whether the current trial was the only trial required or how many more trials are needed to obtain a reward. These cue-related responses had a median latency of 120 ms (just between the latencies in inferior temporal visual area TE and perirhinal cortex). When the monkey was releasing a touch bar to complete the trial correctly, 71 of 246 (29%) neurons responded, with responses in the rewarded trials being similar no matter which schedule was ending, thus being sensitive to the reward contingency. Finally, 39 of 246 (16%) neurons responded around the reward. We suggest that basolateral amygdala, by anticipating and then delineating the schedule and representing reward contingency, provide contextual information that is important for adjusting motivational level as a function of immediate behavior goals.",
         "",
         "https://www.jneurosci.org/content/jneuro/25/48/11071.full.pdf"
        ],
        [
         "24",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "The role of microRNA genes in papillary thyroid carcinoma.",
         "Apart from alterations in the RET/PTC-RAS-BRAF pathway, comparatively little is known about the genetics of papillary thyroid carcinoma (PTC). We show that numerous microRNAs (miRNAs) are transcriptionally up-regulated in PTC tumors compared with unaffected thyroid tissue. A set of five miRNAs, including the three most up-regulated ones (miR-221, -222, and -146), distinguished unequivocally between PTC and normal thyroid. Additionally, miR-221 was up-regulated in unaffected thyroid tissue in several PTC patients, presumably an early event in carcinogenesis. Tumors in which the up-regulation (11- to 19-fold) of miR-221, -222, and -146 was strongest showed dramatic loss of KIT transcript and Kit protein. In 5 of 10 such cases, this down expression was associated with germline single-nucleotide changes in the two recognition sequences in KIT for these miRNAs. We conclude that up-regulation of several miRs and regulation of KIT are involved in PTC pathogenesis, and that sequence changes in genes targeted by miRNAs can contribute to their regulation.",
         "MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation.;MicroRNA gene expression deregulation in human breast cancer.;MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells.;miRNAs, Cancer, and Stem Cell Division;MicroRNA expression profiles classify human cancers;Cancer genomics: Small RNAs with big impacts;A microRNA polycistron as a potential human oncogene;Small regulatory RNAs in mammals.;Combinatorial microRNA target predictions;The RET/PTC-RAS-BRAF linear signaling cascade mediates the motile and mitogenic phenotype of thyroid cancer cells.;MicroRNAs: a developing story.;RAS Is Regulated by the let-7 MicroRNA Family;Accumulation of miR-155 and BIC RNA in human B cell lymphomas.;A new mechanism of BRAF activation in human thyroid papillary carcinomas.;Detection of Overexpressed and Phosphorylated Wild-Type Kit Receptor in Surgical Specimens of Small Cell Lung Cancer;Kit as a human oncogenic tyrosine kinase.;Oncogenic protein tyrosine kinases;Human MicroRNA Targets;Fast and effective prediction of microRNA/target duplexes.;Genetic analysis of genome-wide variation in human gene expression;MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias.;Annual report to the nation on the status of cancer, 1975–2001, with a special feature regarding survival;An oligonucleotide microchip for genome-wide microRNA profiling in human and mouse tissues.;Reduced Expression of the let-7 MicroRNAs in Human Lung Cancers in Association with Shortened Postoperative Survival;MicroRNAs Genomics, Biogenesis, Mechanism, and Function;Prediction of Mammalian MicroRNA Targets;BRAF mutations in thyroid tumors are restricted to papillary carcinomas and anaplastic or poorly differentiated carcinomas arising from papillary carcinomas.;Reduced accumulation of specific microRNAs in colorectal neoplasia.;Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma;BRAF mutation in papillary thyroid carcinoma.;High prevalence of BRAF mutations in thyroid cancer: genetic evidence for constitutive activation of the RET/PTC-RAS-BRAF signaling pathway in papillary thyroid carcinoma.;Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia;Environmental and heritable causes of cancer among 9.6 million individuals in the Swedish family‐cancer database;Gene expression in papillary thyroid carcinoma reveals highly consistent profiles;Localization of a susceptibility gene for familial nonmedullary thyroid carcinoma to chromosome 2q21.;Papillary thyroid carcinoma associated with papillary renal neoplasia: genetic linkage analysis of a distinct heritable tumor syndrome.;The biology of stem cell factor and its receptor C-kit.;A gene predisposing to familial thyroid tumors with cell oxyphilia maps to chromosome 19p13.2.;Paraffin section detection of the c-kit gene product (CD117) in human tissues: value in the diagnosis of mast cell disorders.;c-Kit proto-oncogene is more likely to lose expression in differentiated thyroid carcinoma than three thyroid-specific genes: thyroid peroxidase, thyroglobulin, and thyroid stimulating hormone receptor.;Transformation of thyroid epithelium is associated with loss of c-kit receptor.;Systematic population-based assessment of cancer risk in first-degree relatives of cancer probands.;Expression of c-kit receptor in normal and transformed human nonlymphoid tissues.;RNA silencing.;M  ED  IC  A  L  SC  IE  N  CE;Loss of heterozygosity in papillary and follicular thyroid carcinoma: a mini review.;de la Chapelle, A.;Age at diagnosis Gender Race Diagnosis",
         "https://europepmc.org/articles/pmc1323209?pdf=render"
        ],
        [
         "25",
         "001ce484190e733e0d1dee181f9ee37fd0b15581",
         "Lethality of chlorine, chlorine dioxide, and a commercial fruit and vegetable sanitizer to vegetative cells and spores of Bacillus cereus and spores of Bacillus thuringiensis",
         "None",
         "Sanitizers: Halogens, Surface-Active Agents, and Peroxides;Mechanisms of killing of Bacillus subtilis spores by hypochlorite and chlorine dioxide;Bacillus anthracis: current knowledge in relation to contamination of food.;Peracetic acid as an alternative wastewater disinfectant to chlorine dioxide;Mechanisms of killing spores of Bacillus subtilis by acid, alkali and ethanol;Differential efficacy of a chlorine dioxide‐containing sanitizer against single species and binary biofilms of a dairy‐associated Bacillus cereus and a Pseudomonas fluorescens isolate;Reduction of listeria monocytogenes on green peppers (Capsicum annuum L.) by gaseous and aqueous chlorine dioxide and water washing and its growth at 7 degrees C.;A study on the effects of high pressure and heat on Bacillus subtilis spores at low pH.;Inactivation of Escherichia coli O157:H7 on surface-uninjured and -injured green pepper (Capsicum annuum L.) by chlorine dioxide gas as demonstrated by confocal laser scanning microscopy;Resistance of Bacillus Endospores to Extreme Terrestrial and Extraterrestrial Environments;Antiseptics and Disinfectants: Activity, Action, and Resistance;Sensitivity of Bacillus coagulans spores to combinations of high hydrostatic pressure, heat, acidity and nisin;The effects of various disinfectants againstListeria monocytogeneson fresh-cut vegetables;Comparative sporicidal effects of liquid chemical agents;Microbiological Control of Cucumber Hydrocooling Water with Chlorine Dioxide †.;Interaction of Bacillus subtilis spores with sodium hypochlorite, sodium dichloroisocyanurate and chloramine-T.;Bacterial spores and chemical sporicidal agents;Bacillus Species;Effects of Organic Acids on Thermal Inactivation of Bacillus stearothermophilus and Bacillus coagulans Spores in Frankfurter Emulsion Slurry.;Sporicidal activity of mixtures of alcohol and hypochlorite.;Ultrastructural and chemical changes in spores of Bacillus cereus after action of disinfectants.;A sporulation medium for Bacillus anthracis.;Sporicidal properties of some halogens.;Kinetics and mechanism of bacterial disinfection by chlorine dioxide.;Resistance of bacterial spores. In: Storz G, Hengge-Aronis R (eds) Bacterial stress;Principles and practices of disinfection, reservation and sterilization;Surface decontamination of fruits and vegetables eaten raw: a review. Food Safety Issues, Food Safety Unit, World Health Organization;These include:;Evicting an unwelcome tenant: anthrax",
         "https://academic.oup.com/jimb/article-pdf/32/7/301/34739283/jimb0301.pdf"
        ],
        [
         "26",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "Recent advances in understanding the cell death pathways activated by anticancer therapy",
         "Over the past two decades, the role of apoptosis in the cytotoxicity of anticancer drugs has become clear. Apoptosis may occur via a death receptor‐dependent (extrinsic) or independent (intrinsic or mitochondrial) pathway. Mitochondria play a central role in cell death in response to DNA damage, and mediate the interaction(s) of various cytoplasmic organelles, including the endoplasmic reticulum, Golgi apparatus, and lysosomes. The mitochondrial pathway of cell death is mediated by Bcl‐2 family proteins, a group of antiapoptotic and proapoptotic proteins that regulate the passage of small molecules, such as cytochrome c, Smac/Diablo, and apoptosis‐inducing factor, which activates caspase cascades, through the mitochondrial transition pore. In addition, apoptosis can induce autophagic cell death via crosstalk between the two pathways upon treatment with anticancer drugs. The current review focused on recent advances surrounding the mechanism(s) of cell death induced by anticancer agents and discussed potential molecular targets for enhancing the chemotherapeutic effect(s) of anticancer agents. Cancer 2005. © 2005 American Cancer Society.",
         "Neutralization of Smac/Diablo by Inhibitors of Apoptosis (IAPs);Aging as a catabolic malfunction.;Apoptosis, autophagy, and more.;Dynamic evolution of the adenine nucleotide translocase interactome during chemotherapy-induced apoptosis;Bax is an important determinant for radiation sensitivity in esophageal carcinoma cells.;Apollon ubiquitinates SMAC and caspase-9, and has an essential cytoprotection function;A Phase I Pharmacokinetic and Biological Correlative Study of Oblimersen Sodium (Genasense, G3139), an Antisense Oligonucleotide to the Bcl-2 mRNA, and of Docetaxel in Patients with Hormone-Refractory Prostate Cancer;DAP-kinase as a target for drug design in cancer and diseases associated with accelerated cell death.;Endogenously released Smac is insufficient to mediate cell death of human lung carcinoma in response to etoposide.;Smac induces cytochrome c release and apoptosis independently from Bax/Bcl-xL in a strictly caspase-3-dependent manner in human carcinoma cells;Nuclear and mitochondrial conversations in cell death: PARP-1 and AIF signaling.;Ceramide-mediated Macroautophagy Involves Inhibition of Protein Kinase B and Up-regulation of Beclin 1*;JNK promotes Bax translocation to mitochondria through phosphorylation of 14‐3‐3 proteins;Multiple cell death pathways as regulators of tumour initiation and progression;Apoptosis defects and chemotherapy resistance: molecular interaction maps and networks;Changes in Therapy for Solid Tumors: Potential for Overcoming Drug Resistance In Vivo with Molecular Targeting Agents;Molecular therapy for peritoneal dissemination of xenotransplanted human MKN-45 gastric cancer cells with adenovirus mediated Bax gene transfer;Mitochondrial protease Omi/HtrA2 enhances caspase activation through multiple pathways;Apoptosis pathways in cancer and cancer therapy;In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker.;Endonuclease G is required for early embryogenesis and normal apoptosis in mice;Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor;Phosphoinositide 3-kinase accelerates autophagic cell death during glucose deprivation in the rat cardiomyocyte-derived cell line H9c2;Endostatin induces autophagic cell death in EAhy926 human endothelial cells.;HSpin1, a transmembrane protein interacting with Bcl-2/Bcl-xL, induces a caspase-independent autophagic cell death;Direct Interaction between Survivin and Smac/DIABLO Is Essential for the Anti-apoptotic Activity of Survivin during Taxol-induced Apoptosis*;Omi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP) irreversibly inactivates IAPs and facilitates caspase activity in apoptosis.;Transformation-selective apoptotic program triggered by farnesyltransferase inhibitors requires Bin1;The CD95 type I/type II model.;Amino Acids Interfere with the ERK1/2-dependent Control of Macroautophagy by Controlling the Activation of Raf-1 in Human Colon Cancer HT-29 Cells*;Induction of autophagic cell death in malignant glioma cells by arsenic trioxide.;Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs.;Chemotherapy enhances TNF-related apoptosis-inducing ligand DISC assembly in HT29 human colon cancer cells;Molecular response of human glioblastoma multiforme cells to ionizing radiation: cell cycle arrest, modulation of the expression of cyclin-dependent kinase inhibitors, and autophagy.;Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?;Current status of the molecular mechanisms of anticancer drug-induced apoptosis;Hypoxia and Nitric Oxide Treatment Confer Tolerance to Glucose Starvation in a 5′-AMP-activated Protein Kinase-dependent Manner*;Mediation of Poly(ADP-Ribose) Polymerase-1-Dependent Cell Death by Apoptosis-Inducing Factor;DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death;Bcl-2 and Bcl-xL Inhibit CD95-mediated Apoptosis by Preventing Mitochondrial Release of Smac/DIABLO and Subsequent Inactivation of X-linked Inhibitor-of-Apoptosis Protein*;Increased Ras expression and caspase-independent neuroblastoma cell death: possible mechanism of spontaneous neuroblastoma regression.;Paclitaxel triggers cell death primarily via caspase-independent routes in the non-small cell lung cancer cell line NCI-H460.;The Apaf-1 apoptosome: a large caspase-activating complex.;Apoptosis-inducing factor (AIF): a novel caspase-independent death effector released from mitochondria.;Identification of Omi/HtrA2 as a Mitochondrial Apoptotic Serine Protease That Disrupts Inhibitor of Apoptosis Protein-Caspase Interaction*;The Tumor Suppressor PTEN Positively Regulates Macroautophagy by Inhibiting the Phosphatidylinositol 3-Kinase/Protein Kinase B Pathway*;Autophagy delays sulindac sulfide-induced apoptosis in the human intestinal colon cancer cell line HT-29.;Bax translocation is crucial for the sensitivity of leukaemic cells to etoposide-induced apoptosis;The target of rapamycin (TOR) proteins;Proapoptotic BAX and BAK: A Requisite Gateway to Mitochondrial Dysfunction and Death;Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function.;PTEN controls tumor-induced angiogenesis;Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis;Bcl-2 down-regulation causes autophagy in a caspase-independent manner in human leukemic HL60 cells;Molecular Determinants of the Caspase-promoting Activity of Smac/DIABLO and Its Role in the Death Receptor Pathway*;BNIP3 and Genetic Control of Necrosis-Like Cell Death through the Mitochondrial Permeability Transition Pore;Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments.;Distinct Classes of Phosphatidylinositol 3′-Kinases Are Involved in Signaling Pathways That Control Macroautophagy in HT-29 Cells*;Induction of autophagy and inhibition of tumorigenesis by beclin 1;Autophagy Is Activated by Apoptotic Signalling in Sympathetic Neurons: An Alternative Mechanism of Death Execution;Cloning and genomic organization of beclin 1, a candidate tumor suppressor gene on chromosome 17q21.;Oncogenic Ras triggers cell suicide through the activation of a caspase-independent cell death program in human cancer cells;BIN1 is a novel MYC–interacting protein with features of a tumour suppressor;Ultrastructural and immunocytochemical characterization of autophagic vacuoles in isolated hepatocytes: effects of vinblastine and asparagine on vacuole distributions.;Studies on the mechanisms of autophagy: formation of the autophagic vacuole;Vinblastine-induced autophagic vacuoles in mouse liver and Ehrlich ascites tumor cells as assessed by freeze-fracture electron microscopy.;The morphology of various types of cell death in prenatal tissues.;Inducing cancer cell death by targeting transcription factors.;Mitochondrial dysfunction is an essential step for killing of non-small cell lung carcinomas resistant to conventional treatment",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/cncr.20947"
        ],
        [
         "27",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "Biochemical and spectroscopic characterization of the bacterial phytochrome of Pseudomonas aeruginosa",
         "Phytochromes are photochromic biliproteins found in plants as well as in some cyanotrophic, photoautotrophic and heterotrophic bacteria. In many bacteria, their function is largely unknown. Here we describe the biochemical and spectroscopic characterization of recombinant bacterial phytochrome from the opportunistic pathogen Pseudomonas aeruginosa (PaBphP). The recombinant protein displays all the characteristic features of a bonafide phytochrome. In contrast with cyanobacteria and plants, the chromophore of this bacterial phytochrome is biliverdin IXα, which is produced by the heme oxygenase BphO in P. aeruginosa. This chromophore was shown to be covalently attached via its A‐ring endo‐vinyl group to a cysteine residue outside the defined bilin lyase domain of plant and cyanobacterial phytochromes. Site‐directed mutagenesis identified Cys12 and His247 as being important for chromophore binding and photoreversibility, respectively. PaBphP is synthesized in the dark in the red‐light‐absorbing Pr form and immediately converted into a far‐red‐light‐absorbing Pfr‐enriched form. It shows the characteristic red/far‐red‐light‐induced photoreversibility of phytochromes. A chromophore analog that lacks the C15/16 double bond was used to show that this photoreversibility is due to a 15Z/15E isomerization of the biliverdin chromophore. Autophosphorylation of PaBphP was demonstrated, confirming its role as a sensor kinase of a bacterial two‐component signaling system.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2005.04623.x"
        ],
        [
         "28",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "Mycobacteria use their surface-exposed glycolipids to infect human macrophages through a receptor-dependent process Published, JLR Papers in Press, December 1, 2004. DOI 10.1194/jlr.M400308-JLR200",
         "Two subfamilies of the polar glycopeptidolipids (GPLs) located on the surface of Mycobacterium smegmatis, along with unknown phospholipids, were recently shown to participate in the nonopsonic phagocytosis of mycobacteria by human macrophages (Villeneuve, C., G. Etienne, V. Abadie, H. Montrozier, C. Bordier, F. Laval, M. Daffe, I. Maridonneau-Parini, and C. Astarie-Dequeker. 2003. Surface-exposed glycopeptidolipids of Mycobacterium smegmatis specifically inhibit the phagocytosis of mycobacteria by human macrophages. Identification of a novel family of glycopeptidolipids. J. Biol. Chem. 278: 51291–51300). As demonstrated herein, a phospholipid mixture that derived from the methanol-insoluble fraction inhibited the phagocytosis of M. smegmatis. Inhibition was essentially attributable to phosphatidylinositol mannosides (PIMs), namely PIM2 and PIM6, because the purified phosphatidylethanolamine, phosphatidylglycerol, and phosphatidylinositol were inactive. This was further confirmed using purified PIM2 and PIM6 from M. bovis BCG that decreased by half the internalization of M. smegmatis. Both compounds also inhibited the uptake of M. tuberculosis and M. avium but had no effect on the internalization of zymosan used as a control particle of the phagocytic process. When coated on latex beads, PIM2 and polar GPL (GPL III) favored the particle entry through complement receptor 3. GPL III, but not PIM2, also directed particle entry through the mannose receptor. Therefore, surface-exposed mycobacterial PIM and polar GPL participate in the receptor-dependent internalization of mycobacteria in human macrophages.",
         "",
         "http://www.jlr.org/article/S0022227520340323/pdf"
        ],
        [
         "29",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "Insights from the developing world: thrifty genotypes and thrifty phenotypes",
         "Few researchers would dispute that the pandemic of obesity is caused by a profound mismatch between humanity's present environmental circumstances and those that have moulded evolutionary selection. This concept was first articulated when gestational diabetes was described as being the result of a ‘thrifty genotype rendered detrimental by progress’. More recently, this hypothesis has been extended to the concept of a ‘thrifty phenotype’ to describe the metabolic adaptations adopted as a survival strategy by a malnourished fetus; changes that may also be inappropriate to deal with a later life of affluence. Both the thrifty genotype and the thrifty phenotype hypotheses would predict that populations in some areas of the developing world would be at greater risk of obesity and its co-morbidities; a proposition to be explored in the present paper. To date thrifty genes remain little more than a nebulous concept propagated by the intuitive logic that man has been selected to survive episodic famine and seasonal hungry periods. Under such conditions those individuals who could lay down extra energy stores and use them most efficiently would have a survival advantage. The search for candidate thrifty genes needs to cover every aspect of human energy balance from food-seeking behaviour to the coupling efficiency of oxidative phosphorylation. The present paper will describe examples of attempts to find thrifty genes in three selected candidate areas: maternally-transmitted mitochondrial genes; the uncoupling proteins; apoE4, whose geographical distribution has been linked to a possible thrifty role in lipoprotein and cholesterol metabolism.",
         "Fetal origins of insulin resistance and obesity;Variation at the insulin gene VNTR (variable number tandem repeat) polymorphism and early growth: studies in a large Finnish birth cohort.;Apolipoprotein E polymorphism and fertility: a study in pre-industrial populations.;Opposing influences of prenatal and postnatal weight gain on adrenarche in normal boys and girls.;Maternal-fetal interactions and birth order influence insulin variable number of tandem repeats allele class associations with head size at birth and childhood weight gain.;The human obesity gene map: the 2003 update.;Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA;Intrauterine factors, adiposity, and hyperinsulinaemia;Global haplotype diversity in the human insulin gene region.;Birthweight and perinatal mortality: paradoxes, social class, and sibling dependencies.;The double puzzle of diabetes;The epidemiology of adverse pregnancy outcomes: an overview.;Neonatal anthropometry: the thin–fat Indian baby. The Pune Maternal Nutrition Study;Adiposity and hyperinsulinemia in Indians are present at birth.;The role of uncoupling proteins in pathophysiological states.;Racial differences in the relation between uncoupling protein genes and resting energy expenditure.;An overview on the nutrition transition and its health implications: the Bellagio meeting;An adipose-specific control of thermogenesis in body weight regulation;Insulin gene variable number of tandem repeat genotype and the low birth weight, precocious pubarche, and hyperinsulinism sequence.;On the importance--and the unimportance--of birthweight.;The thrifty phenotype hypothesis.;Obesity and undernutrition and cardiovascular risk factors in rural and urban Gambian communities.;Glucose, insulin and lipid metabolism in rural Gambians exposed to early malnutrition;Uncoupling proteins: functional characteristics and role in the pathogenesis of obesity and Type II diabetes;Uncoupling proteins: their roles in adaptive thermogenesis and substrate metabolism reconsidered;Fires of life: the struggles of an ancient metabolism in a modern world;The nutrition transition and obesity in the developing world.;Thrifty Genotypes and Phenotypes in the Pathogenesis of Type 2 Diabetes Mellitus;Uncoupling to survive? The role of mitochondrial inefficiency in ageing;The Fetal and Childhood Growth of Persons Who Develop Type 2 Diabetes;Energy adaptations in human pregnancy: limits and long-term consequences.;Interactions of perturbations in intrauterine growth and growth during childhood on the risk of adult-onset disease;Insulin resistance syndrome in 8-year-old Indian children: small at birth, big at 8 years, or both?;Transmission ratio distortion at the INS-IGF2 VNTR;Apolipoprotein E (APOE) allele distribution in the world. Is APOE*4 a ‘thrifty’ allele?;The insulin gene VNTR, type 2 diabetes and birth weight;The thrifty genotype in type 2 diabetes;The INS 5′ Variable Number of Tandem Repeats Is Associated with IGF2 Expression in Humans*;Association of the INS VNTR with size at birth;The Great Famine: Northern Europe in the Early Fourteenth Century;Bergmann's rule and the thrifty genotype.;Imprinted and genotype-specific expression of genes at the IDDM2 locus in pancreas and leucocytes.;Susceptibility to human type 1 diabetes at IDDM2 is determined by tandem repeat variation at the insulin gene minisatellite locus;Biochemical control logic and the metabolism of glutamine.;Type 2 (non-insulin-dependent) diabetes mellitus: the thrifty phenotype hypothesis;The possible role of glutamine substrate cycles in skeletal muscle.;Maternal weight gain, infant birth weight, and diet: causal sequences.;Role of triglyceride-fatty acid cycle in controlling fat metabolism in humans during and after exercise.;A Polymorphic Locus Near the Human Insulin Gene Is Associated with Insulin-dependent Diabetes Melliitus;Poverty and famine: An essay on entitlement and deprivation : , Oxford, New York: Oxford University Press, 1981. xi + 257 pp., index. $17.95;Malnutrition and fertility.;Poverty and famines. An essay on entitlement and deprivation [incl. case studies of famines in Bengal, Ethiopia, Sahel, and Bangladesh].;Famine in Tudor and Stuart England.;The high prevalence of diabetes mellitus on a Central Pacific island;Famine and Human Development: The Dutch Hunger Winter 1944-1945;Faminee of history and of today;Diabetes in American Indians and Other Native Populations of the New World;The determinants of growth and form.;Review lecture - The determinants of growth and form;Diabetes mellitus: a \"thrifty\" genotype rendered detrimental by \"progress\"?;Biology of Human Starvation;A critical evaluation of the fetal origins hypothesis and its implications for developing countries.;Lack of support for a role of the insulin gene variable number of tandem repeats minisatellite (INS-VNTR) locus in fetal growth or type 2 diabetes-related intermediate traits in United Kingdom populations.;Differential reproductive efficiency associated with common apolipoprotein e alleles in postreproductive-aged subjects.;Eating, exercise, and \"thrifty\" genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases.;The insulin gene variable number tandem repeat class I/III polymorphism is in linkage disequilibrium with birth weight but not Type 2 diabetes in the Pima population.;Floods, Famines and Emperors. London: Pimlico;PPARgamma, the ultimate thrifty gene.;Fetal origins of adult disease-the hypothesis revisited.;Genomic imprinting of fetal growth;The Demography of Famine: An Indian Historical Perspective;Biochemical control logic and the metabolism of glutamine.;Agriculture’s mixed blessings;The Third Chimpanzee;Determinants of low birth weight: methodological assessment and meta-analysis.;Origin of Species",
         "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D4334A1A0F9096B08FAE923BAB0CA36D/S0029665105000194a.pdf/div-class-title-insights-from-the-developing-world-thrifty-genotypes-and-thrifty-phenotypes-div.pdf"
        ],
        [
         "30",
         "00206ca6a32791adcca5effdfed6eecd035075f4",
         "Sparse basis selection: new results and application to adaptive prediction of video source traffic",
         "Real-time prediction of video source traffic is an important step in many network management tasks such as dynamic bandwidth allocation and end-to-end quality-of-service (QoS) control strategies. In this paper, an adaptive prediction model for MPEG-coded traffic is developed. A novel technology is used, first developed in the signal processing community, called sparse basis selection. It is based on selecting a small subset of inputs (basis) from among a large dictionary of possible inputs. A new sparse basis selection algorithm is developed that is based on efficiently updating the input selection adaptively. When a new measurement is received, the proposed algorithm updates the selected inputs in a recursive manner. Thus, adaptability is not only in the weight adjustment, but also in the dynamic update of the inputs. The algorithm is applied to the problem of single-step-ahead prediction of MPEG-coded video source traffic, and the developed method achieves improved results, as compared to the published results in the literature. The present analysis indicates that the adaptive feature of the developed algorithm seems to add significant overall value.",
         "",
         "http://www.vision.caltech.edu/malaa/publications/atiya05sparse.pdf"
        ],
        [
         "31",
         "0020d5bdfc5f13026e7a1aca78e94724e8342b62",
         "A mutation in the Icsbp1 gene causes susceptibility to infection and a chronic myeloid leukemia–like syndrome in BXH-2 mice",
         "BXH-2 mice develop a fatal myeloid leukemia by a two-step mutagenic process. First, a BXH-2–specific recessive mutation causes a myeloproliferative syndrome. Second, retroviral insertions alter oncogenes or tumor suppressors, resulting in clonal expansion of leukemic cells. We have identified a recessive locus on chromosome 8 (Myls) that is responsible for myeloproliferation in BXH-2. This Myls interval has been narrowed down to 2 Mb and found to contain several positional candidates, including the interferon consensus sequence–binding protein 1 gene (Icsbp, also known as interferon regulatory factor 8 [IRF8]). We show that BXH-2 mice carry a mutation (915 C to T) resulting in an arginine-to-cysteine substitution at position 294 within the predicted IRF association domain of the protein. Although expression of Icsbp1 mRNA transcripts is normal in BXH-2 splenocytes, these cells are unable to produce interleukin 12 and interferon-γ in response to activating stimuli, confirming that R294C behaves as a loss-of-function mutation. Myeloproliferation in BXH-2 mice is concomitant to increased susceptibility to Mycobacterium bovis (BCG) despite the presence of resistance alleles at the Nramp1 locus. These results suggest a two-step model for chronic myeloid leukemia in BXH-2, in which inactivation of Icsbp1 predisposes to myeloproliferation and immunodeficiency. This event is required for retroviral replication, and subsequent insertional mutagenesis that causes leukemia in BXH-2 mice.",
         "",
         "https://rupress.org/jem/article-pdf/201/6/881/1153938/jem2016881.pdf"
        ],
        [
         "32",
         "0022e70878c2f2a7c3c679a3ef3160a5cb803e20",
         "Binding of estrogen receptor beta to estrogen response element in situ is independent of estradiol and impaired by its amino terminus.",
         "The functions of 17beta-estradiol (E2) are mediated by estrogen receptor (ER) alpha and beta. ERs display similar DNA- and ligand-binding properties in vitro. However, ERbeta shows lower transcriptional activity than ERalpha from the estrogen response element (ERE)-dependent signaling. We predicted that distinct amino termini contribute to differences in transcription efficacies of ERs by affecting in situ ER-ERE interactions. We used chromatin immunoprecipitation and a novel in situ ERE competition assay, which is based on the ability of ER to compete for ERE binding with a designer activator that constitutively induces transcription from an ERE-driven reporter construct. Interference of activator-mediated transcription by unliganded or liganded ERs was taken as an indication of ER-ERE interaction. Results revealed that ERs interacted with ERE similarly in the absence of E2. However, E2 enhanced the ERE binding of ERalpha but not that of ERbeta. The removal of the amino terminus increased the ERbeta-ERE interaction independent of E2. The ERbeta amino terminus also prevented E2-mediated enhancement of the chimeric ERalpha-ERE interaction. Thus, the amino terminus of ERbeta impairs the binding of ERbeta to ERE. The abrogation of ligand-dependent activation function 2 of the amino-terminally truncated ERbeta resulted in the manifestation of E2 effect on ERbeta-ERE interaction. This implies that E2-mediated enhancement of ERbeta-ERE interaction is masked by the activation function 2, whereas the intact amino terminus is a dominant region that decreases the binding of ERbeta to ERE. Thus, ERbeta-ERE interaction is independent of E2 and is impaired by its amino terminus. These findings provide an additional explanation for differences between ERalpha and ERbeta functions that could differentially affect the physiology and pathophysiology of E2 signaling.",
         "",
         "https://academic.oup.com/mend/article-pdf/19/11/2696/10717141/mend2696.pdf"
        ],
        [
         "33",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "Sensor node localization using mobile acoustic beacons",
         "We present a mobile acoustic beacon based sensor node localization method. Our technique is passive in that the sensor nodes themselves do not need to generate an acoustic signal for ranging. This saves cost, power and provides stealthy operation. Furthermore, the beacon can generate much more acoustic energy than a severely resource constrained sensor node, thereby significantly increasing the range. The acoustic ranging method uses a linear frequency modulated signal that can be accurately detected by matched filtering. This provides longer range and higher accuracy than the current state-of-the-art. The localization algorithm was especially designed to work in such acoustically reverberant environment, as urban terrain. The algorithm presented handles non-Gaussian ranging errors caused by echoes. Node locations are computed centrally by solving a global non-linear optimization problem in an iterative and incremental fashion",
         "Resilient Localization for Sensor Networks in Outdoor Environments;Mobile-assisted localization in wireless sensor networks;Sensor network-based countersniper system;Robust distributed network localization with noisy range measurements;The flooding time synchronization protocol;Localization of wireless sensor networks with a mobile beacon;Smart sensors to network the world.;Range-free localization schemes for large scale sensor networks;The n-Hop Multilateration Primitive for Node Localization Problems;Ad hoc positioning system (APS) using AOA;Mica: A Wireless Platform for Deeply Embedded Networks;Middleware challenges for wireless sensor networks;The bits and flops of the n-hop multilateration primitive for node localization problems;Wireless sensor networks for habitat monitoring;A survey on position-based routing in mobile ad hoc networks;Robust range estimation using acoustic and multimodal sensing;Location Systems for Ubiquitous Computing;Smart kindergarten: sensor-based wireless networks for smart developmental problem-solving environments;Exposure in wireless Ad-Hoc sensor networks;Dynamic fine-grained localization in Ad-Hoc networks of sensors;The cricket compass for context-aware mobile applications;GPS-free Positioning in Mobile Ad Hoc Networks;GPS-less low-cost outdoor localization for very small devices;Embedding the Internet: introduction;Acoustic Ranging in Resource-Constrained Sensor Networks;IEEE Computer Graphics & Applications: Projects In VR Pervasive Pose-Aware Applications and Infrastructure;VR Pervasive Pose-Aware Applications and Infrastructure. IEEE Distributed Systems Online;Range-free localization schemes in large scale sensor network;This copyright notice must be included in the reproduced paper. USENIX acknowledges all trademarks herein. Robust Positioning Algorithms for Distributed Ad-Hoc Wireless Sensor Networks;The Design of Calamari : an Ad-hoc Localization System for Sensor Networks;Complex Behavior at Scale: An Experimental Study of Low-Power Wireless Sensor Networks;Convex Optimization Methods for Sensor Node Position Estimation;SpotON: An Indoor 3D Location Sensing Technology Based on RF Signal Strength;The Cricket location-support system;Global Positioning System Overview (last modified 5.1.2000);Numerical Optimization, pages 250–275;Theory and application of digital signal processing;If both | µ i | and σ i are large then again location might be affected by echo and we follow the same procedure as in case 2 above;If µ i is close to zero but σ i is large then either the range deviations are spread around zero or few large deviations caused σ i to be large;If | µ i | is large but σ i is small then all elements of ∆ i are large i.e. the node location is deﬁnitely incorrect",
         "https://ir.vanderbilt.edu/bitstream/1803/13704/1/ms-thesis-manish-kushwaha.pdf"
        ],
        [
         "34",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "Impact of Software Vulnerability Announcements on the Market Value of Software Vendors - an Empirical Investigation",
         "Researchers in the area of information security have mainly been concerned with tools, techniques and policies that firms can use to protect themselves against security breaches. However, information security is as much about security software as it is about secure software. Software is not secure when it has defects or flaws which can be exploited by hackers to cause attacks such as unauthorized intrusion or denial of service attacks. Any public announcement makes about a software defect is termed as 'vulnerability disclosure'. Although research in software economics have studied firms' incentive to improve overall quality, there have been no studies to show that software vendors have an incentive to invest in building more secure software. In this paper, we use the event study methodology to examine software vendors' incentives to build more secure software. We collect data from leading national newspapers and industry sources like CERT by searching for reports on published software vulnerabilities. We show that vulnerability disclosures lead to a negative and significant change in market value for a software vendor. On average, a vendor loses around 0.6% value in stock price when a vulnerability is reported. This is equivalent to a loss in market capitalization values of $0.86 billion per vulnerability announcement. To provide further insight, we use the information content of the disclosure announcement to classify vulnerabilities into various types. We find that the change in stock price is more negative if the vendor fails to provide a patch at the time of disclosure. Moreover, vulnerabilities which cause a confidentiality related breach cause a greater decline in the market value for a vendor than the vulnerabilities which cause non-confidentiality related breaches. Also, more severe flaws have a significantly greater impact than flaws with low or moderate severity. Finally, we find that the markets do not punish a software vendor more severely if a third party discovers a flaw in its product than if the vendor itself discovers the flaw. Our analysis provides many interesting implications for software vendors as well as policy makers.",
         "",
         "http://archive.nyu.edu/bitstream/2451/14996/2/Infosec_Book%2bWattal%2bTelang.pdf"
        ],
        [
         "35",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "Real-Time Polymerase Chain Reaction Quantification of Phytophthora capsici in Different Pepper Genotypes.",
         "ABSTRACT Reliable and sensitive quantification of Phytophthora capsici in pepper plants is of crucial importance in managing the multiple syndromes caused by this pathogen. A real-time polymerase chain reaction (PCR) assay was developed for the determination of P. capsici in pepper tissues. DNA levels of a highly virulent and a less virulent isolate were measured in different pepper genotypes with varying degrees of resistance. Using SYBR Green and specific primers for P. capsici, the minimal amount of pathogen DNA quantified was 10 pg. Pathogen DNA was recorded as early as 8 h postinoculation. Thereafter, the increase was rapid in susceptible cultivars and slower in resistant ones. The amount of pathogen DNA quantified in each pepper genotype correlated with susceptibility to Phytophthora root rot. Likewise, there was a relationship between the virulence of the pathogen and the degree of colonization. Differences also were found in oomycete amount among pepper tissues, with maximal pathogen biomass occurring in stems. The real-time PCR technique developed in this study was sensitive and robust enough to assess both pathogen development and resistance to Phytophthora root rot in different pepper genotypes.",
         "",
         "https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-95-1423"
        ],
        [
         "36",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "Wing-to-Leg Homeosis by Spineless Causes Apoptosis Regulated by Fish-lips, a Novel Leucine-Rich Repeat Transmembrane Protein",
         "ABSTRACT Growth, patterning, and apoptosis are mutually interactive during development. For example, cells that select an abnormal fate in a developing field are frequently removed by apoptosis. An important issue in this process that needs to be resolved is the mechanism used by cells to discern their correct fate from an abnormal fate. In order to examine this issue, we developed an animal model that expresses the dioxin receptor homolog Spineless (Ss) ectopically in the Drosophila wing. The presence of mosaic clones ectopically expressing ss results in a local transformation of organ identity, homeosis, from wing into a leg or antenna. The cells with misspecified fates subsequently activate c-Jun N-terminal kinase to undergo apoptosis in an autonomous or nonautonomous manner depending on their position within the wing, suggesting that a cell-cell interaction is, at least in some cases, involved in the detection of misspecified cells. Similar position dependence is commonly observed when various homeotic genes controlling the body segments are ectopically expressed. The autonomous and nonautonomous apoptosis caused by ss is regulated by a novel leucine-rich repeat family transmembrane protein, Fish-lips (Fili) that interacts with surrounding normal cells. These data support a mechanism in which the lack of some membrane proteins helps to recognize the presence of different cell types and direct these cells to an apoptotic fate in order to exclude them from the normal developing field.",
         "",
         "https://europepmc.org/articles/pmc1069588?pdf=render"
        ],
        [
         "37",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "The sentinel within: exploiting the immune system for cancer biomarkers.",
         "The release of proteins from tumors triggers an immune response in cancer patients. These tumor antigens arise from several mechanisms including tumor-specific alterations in protein expression, mutation, folding, degradation, or intracellular localization. Responses to most tumor antigens are rarely observed in healthy individuals, making the response itself a biomarker that betrays the presence of underlying cancer. Antibody immune responses show promise as clinical biomarkers because antibodies have long half-lives in serum, are easy to measure, and are stable in blood samples. However, our understanding of the specificity and the impact of the immune response in early stages of cancer is limited. The immune response to cancer, whether endogenous or driven by vaccines, involves highly specific T lymphocytes (which target tumor-derived peptides bound to self-MHC proteins) and B lymphocytes (which generate antibodies to tumor-derived proteins). T cell target antigens have been identified either by expression cloning from tumor cDNA libraries, or by prediction based on patterns of antigen expression (\"reverse immunology\"). B cell targets have been similarly identified using the antibodies in patient sera to screen cDNA libraries derived from tumor cell lines. This review focuses on the application of recent advances in proteomics for the identification of tumor antigens. These advances are opening the door for targeted vaccine development, and for using immune response signatures as biomarkers for cancer diagnosis and monitoring.",
         "T Cell–Dependent Antibody Responses against Aberrantly Expressed Cyclin B1 Protein in Patients with Cancer and Premalignant Disease;Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles;Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast Cancer;A phase I trial of DNA vaccination with a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate cancer;Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant induces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans.;Self-Assembling Protein Microarrays;Protein microarray technology;Minor Histocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem Cell Transplantation;An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in association with HLA-DR.;Development of natural protein microarrays for diagnosing cancer based on an antibody response to tumor antigens.;NY-ESO-1 Expression and Immunogenicity in Malignant and Benign Breast Tumors;Clinical proteomics: revolutionizing disease detection and patient tailoring therapy.;The clinical value of intracellular autoantigens B-cell epitopes in systemic rheumatic diseases.;Serological cloning of cancer/testis antigens expressed in prostate cancer using cDNA phage surface display;Complete response to donor lymphocyte infusion in multiple myeloma is associated with antibody responses to highly expressed antigens.;Humoral immune response against melanoma antigens induced by vaccination with cytokine gene‐modified autologous tumor cells;Identification of a shared epitope recognized by melanoma-specific, HLA-A3-restricted cytotoxic T lymphocytes.;Genome-wide Characterization of a Viral Cytotoxic T Lymphocyte Epitope Repertoire*;Microarrays of tumor cell derived proteins uncover a distinct pattern of prostate cancer serum immunoreactivity;Protein arrays for autoantibody profiling and fine‐specificity mapping;Protein microarrays: Promising tools for proteomic research;Recursive partitioning as an approach to selection of immune markers for tumor diagnosis.;Antigen-specific immunity in neuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen.;Expression of cancer/testis (CT) antigens in lung cancer.;Unique conformation of cancer autoantigen B23 in hepatoma: A mechanism for specificity in the autoimmune response;NY-ESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial ovarian cancer.;Identification of tumor‐associated antigens by screening phage‐displayed human cDNA libraries with sera from tumor patients;Tumor-specific immunological recognition of frameshift-mutated peptides in colon cancer with microsatellite instability.;CD1d-restricted T cells regulate dendritic cell function and antitumor immunity in a granulocyte–macrophage colony-stimulating factor-dependent fashion;Melanoma inhibitor of apoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction;The use of HPLC-MS in T-cell epitope identification.;Immunomic analysis of human sarcoma;Enhancement of antibody detection in cancer using panel of recombinant tumor-associated antigens.;Immunoscreening of phage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease.;Integrated functional genomics approach for the design of patient-individual antitumor vaccines.;Proteomics Approaches to Uncover the Repertoire of Circulating Biomarkers for Breast Cancer;CML28 is a broadly immunogenic antigen, which is overexpressed in tumor cells.;Towards patient‐specific tumor antigen selection for vaccination;Cancer/testis antigens: an expanding family of targets for cancer immunotherapy;Serological identification of antigens associated with renal cell carcinoma.;Characterisation of tumour-associated antigens in colon cancer;Solid-Phase Epitope Recovery: A High Throughput Method for Antigen Identification and Epitope Optimization;Enhanced humoral immune response correlates with improved disease-free and overall survival in American Joint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine.;Identification of Breast Cancer-Restricted Antigens by Antibody Screening of SKBR3 cDNA Library Using a Preselected Patient's Serum;Posttranslational protein modifications: new flavors in the menu of autoantigens;ATP6S1 elicits potent humoral responses associated with immune-mediated tumor destruction;Identification of cancer/testis genes by database mining and mRNA expression analysis;Autoantigen microarrays for multiplex characterization of autoantibody responses;Miniaturised multiplexed immunoassays.;Identification of Cyclin B1 as a Shared Human Epithelial Tumor-Associated Antigen Recognized by T Cells;Trends in biomarker research for cancer detection.;Autoimmune diseases: are markers ready for prediction?;Post-translational protein modifications in antigen recognition and autoimmunity.;Identification of a tissue-specific putative transcription factor in breast tissue by serological screening of a breast cancer library.;Profiling the immune response in patients with breast cancer by phage‐displayed cDNA libraries;Tumor-associated antigen TA90 immune complex assay predicts recurrence and survival after surgical treatment of stage I-III melanoma.;Evidence that specific T lymphocytes may participate in the elimination of chronic myelogenous leukemia;Pre-existent immunity to the HER-2/neu oncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer;Profiling the immune responses in patient sera with peptide and cDNA display libraries.;Reactivity of natural and induced human antibodies to MUC1 mucin with MUC1 peptides and n‐acetylgalactosamine (GalNAc) peptides;Rapid degradation of a large fraction of newly synthesized proteins by proteasomes;Serological identification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung cancer.;p53 Antibodies in the sera of patients with various types of cancer: a review.;Expression of cancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a new CT gene, CT9.;CT10: A new cancer‐testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational‐difference analysis;p53 gene mutation, tumor p53 protein overexpression, and serum p53 autoantibody generation in patients with breast cancer.;Cancer-testis antigens and ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-specific ING1 transcripts and a homologue gene.;Antigens recognized by autologous antibody in patients with renal‐cell carcinoma;SYFPEITHI: database for MHC ligands and peptide motifs;Cleavage by Granzyme B Is Strongly Predictive of Autoantigen Status;Humoral immune responses of cancer patients against \"Cancer-Testis\" antigen NY-ESO-1: correlation with clinical events.;The telomerase catalytic subunit is a widely expressed tumor-associated antigen recognized by cytotoxic T lymphocytes.;Isoforms of the human PDZ-73 protein exhibit differential tissue expression.;The 72-kDa Component of Signal Recognition Particle Is Cleaved during Apoptosis*;The Central Role of CD4+ T Cells in the Antitumor Immune Response;Characterization of human colon cancer antigens recognized by autologous antibodies;A Survey of the Humoral Immune Response of Cancer Patients to a Panel of Human Tumor Antigens;Human lung cancer antigens recognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor gene locus on chromosome 3p21.3.;High-titer HER-2/neu protein-specific antibody can be detected in patients with early-stage breast cancer.;SSX: A multigene family with several members transcribed in normal testis and human cancer;Autoantibodies to p53 in ovarian cancer patients and healthy women: a comparison between whole p53 protein and 18-mer peptides for screening purposes.;A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening.;Humoral immune response against p53 protein in patients with colorectal carcinoma;DNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to detect all known p53 alterations in human malignancies.;Anti-p53 antibodies in sera from patients with chronic obstructive pulmonary disease can predate a diagnosis of cancer.;Anti-p53 antibodies in sera of workers occupationally exposed to vinyl chloride.;Absence of p53 autoantibodies in sera from glioma patients.;A motif in human histidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-coil that is essential for enzymatic activity and contains the major autoantigenic epitope.;Clinical utility of the immunocytochemical detection of p53 protein in cytological specimens.;Role of bone marrow-derived cells in presenting MHC class I-restricted tumor antigens.;Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains.;Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-macrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor immunity.;Development of antibodies against p53 in lung cancer patients appears to be dependent on the type of p53 mutation.;HLA-A2 molecules in an antigen-processing mutant cell contain signal sequence-derived peptides;Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry.;Very long charge runs in systemic lupus erythematosus-associated autoantigens.;Characterization of naturally occurring minor histocompatibility peptides including H-4 and H-Y.;The fine specificity of anti-La antibodies induced in mice by immunization with recombinant human La autoantigen.;Post-translational modifications of proteins: some problems left to solve.;CA125 Antigen Levels in Obstetric and Gynecologic Patients;Purification of a human prostate specific antigen.;Detection of a transformation-related antigen in chemically induced sarcomas and other transformed cells of the mouse.;SPECIFIC CARCINOEMBRYONIC ANTIGENS OF THE HUMAN DIGESTIVE SYSTEM;Immunotherapy (APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3 randomized placebo-controlled trial;Efficient purification of unique antibodies using peptide affinity-matrix columns.;Endogenous immune response to early- and intermediate-stage melanoma is correlated with outcomes and is independent of locoregional relapse and standard prognostic factors.;Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem cell transplantation and in healthy female donors.;Recombinant antigen expression on yeast surface (RAYS) for the detection of serological immune responses in cancer patients.;Immunotherapy of hematologic malignancy.;p53 autoantibodies can be indicative of the development of breast cancer relapse.;The autoantibody repertoire: searching for order;MUC1-like tandem repeat proteins are broadly immunogenic in cancer patients.;Epitope identification and vaccine design for cancer immunotherapy.;Immunotherapeutic Targets Cancer : Identification of Potential Diagnostic and Cancer-related Serological Recognition of Human Colon Updated;Identification of tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by serological screening of a testicular library with breast cancer serum.;Humoral immunity to human breast cancer: antigen definition and quantitative analysis of mRNA expression.;Monitoring of p53 autoantibodies after resection of colorectal cancer: relationship to operative curability.;Gure, A. O. et al. CT10: a new cancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-difference analysis. Int. J. Cancer 85, 726-732;MHCPEP, a database of MHC-binding peptides: update 1997;Cancer,Facts and Figures about.",
         "https://europepmc.org/articles/pmc2522321?pdf=render"
        ],
        [
         "38",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "Molecular Mechanisms of Cation Transport by the Renal Na+-K+-Cl- Cotransporter",
         "Two variants of the renal Na+-K+-Cl- cotransporter (NKCC2), called NKCC2A and NKCC2F, display marked differences in Na+, Rb+, and Cl- affinities, yet are identical to one another except for a 23-residue membrane-associated domain that is derived from alternatively spliced exons. The proximal portion of these exons is predicted to encode the second transmembrane domain (tm2) in the form of an α-helix, and the distal portion, part of the following connecting segment (cs1a). In recent studies, we have taken advantage of the A–F differences in kinetic behavior to determine which regions in tm2-cs1a are involved in ion transport. Functional characterizations of chimeras in which tm2 or cs1a were interchanged between the variants showed that both regions are important in specifying ion affinities, but did not allow delineating the contribution of individual residues. Here, we have extended these structure-function analyses by studying additional mutants in which variant residues between A and F were interchanged individually in the tm2-cs1a region (amino acid number 216, 220, 223, 229, or 233 in NKCC2). None of the substitutions were found to affect Km (C1-), suggesting that the affinity difference for anion transport is conveyed by a combination of variant residues in this domain. However, 2 substitutions in the tm2 of F were found to affect cation constants specifically; interestingly, one of these mutations (residue 216) only affected Km (Rb+) while the other (residue 220) only affected Km (Na+). We have thus identified two novel residues in NKCC2 that play a key role in cation transport. Because such residues should be adjacent to one another on the vertical axis of the tm2 α-helix, our results imply, furthermore, that the ion transport sites in NKCC2 could be physically linked.",
         "",
         "http://www.jbc.org/article/S0021925820792379/pdf"
        ],
        [
         "39",
         "00295936451c8a6b8a982d6ba9ab64cb647370ae",
         "Choline deficiency in mice and humans is associated with increased plasma homocysteine concentration after a methionine load.",
         "BACKGROUND\nElevated concentrations of homocysteine in blood may be an independent risk factor for the development of atherosclerosis. Elevated homocysteine concentrations can be caused by decreased methylation of homocysteine to form methionine, as occurs in folate deficiency. A parallel pathway exists for methylation of homocysteine, in which choline, by way of betaine, is the methyl donor.\n\n\nOBJECTIVE\nOur goal was to determine whether choline deficiency results in a decreased capacity to methylate homocysteine.\n\n\nDESIGN\nC57BL/6J mice were fed diets containing 0, 10, or 35 mmol choline/kg diet for 3 wk. We then administered an oral methionine load to the animals and measured plasma homocysteine concentrations. Also, in a pilot study, we examined 8 men who were fed a diet providing 550 mg choline/d per 70 kg body weight for 10 d, followed by a diet providing almost no choline, until the subjects were clinically judged to be choline deficient or for <or=42 d. A methionine load was administered at the end of each dietary phase.\n\n\nRESULTS\nTwo hours after the methionine load, choline-deficient mice had plasma homocysteine concentrations twice those of choline-fed mice. Four hours after the methionine load, clinically choline-depleted men had plasma homocysteine concentrations that were 35% greater than those in men not choline depleted.\n\n\nCONCLUSION\nThese results suggest that choline, like folate, plays an important role in the metabolism of homocysteine in humans and that response to a methionine load may be useful when assessing choline nutriture.",
         "Choline- and betaine-defined diets for use in clinical research and for the management of trimethylaminuria.;Elevated serum creatine phosphokinase in choline-deficient humans: mechanistic studies in C2C12 mouse myoblasts.;Betaine as a Determinant of Postmethionine Load Total Plasma Homocysteine Before and After B-Vitamin Supplementation;Folate status response to controlled folate intake is affected by the methylenetetrahydrofolate reductase 677C-->T polymorphism in young women.;Effect of folic acid on fenofibrate-induced elevation of homocysteine and cysteine.;Betaine supplementation lowers plasma homocysteine in healthy men and women.;Concentrations of choline-containing compounds and betaine in common foods.;Phosphatidylethanolamine N-methyltransferase (PEMT) knockout mice have hepatic steatosis and abnormal hepatic choline metabolite concentrations despite ingesting a recommended dietary intake of choline.;Effects of cytidine 5'-diphosphocholine on plasma homocysteine levels in rat.;Quantitation of choline and its metabolites in tissues and foods by liquid chromatography/electrospray ionization-isotope dilution mass spectrometry.;Hyperhomocysteinemia due to methionine synthase deficiency, cblG: structure of the MTR gene, genotype diversity, and recognition of a common mutation, P1173L.;Choline deficiency causes reversible hepatic abnormalities in patients receiving parenteral nutrition: proof of a human choline requirement: a placebo-controlled trial.;The 1298A-->C polymorphism in methylenetetrahydrofolate reductase (MTHFR): in vitro expression and association with homocysteine.;Determinants of plasma total homocysteine concentration in the Framingham Offspring cohort.;Betaine-homocysteine methyltransferase expression in porcine and human tissues and chromosomal localization of the human gene.;The use of homocysteine and other metabolites in the specific diagnosis of vitamin B-12 deficiency.;Choline deficiency: A cause of hepatic steatosis during parenteral nutrition that can be reversed with intravenous choline supplementation;Evidence that homocysteine is an independent risk factor for atherosclerosis in hyperlipidemic patients.;Vitamin B-6 deficiency vs folate deficiency: comparison of responses to methionine loading in rats.;AIN-93 purified diets for laboratory rodents: final report of the American Institute of Nutrition ad hoc writing committee on the reformulation of the AIN-76A rodent diet.;Low plasma free choline is prevalent in patients receiving long term parenteral nutrition and is associated with hepatic aminotransferase abnormalities.;Accumulation of 1,2-sn-diradylglycerol with increased membrane-associated protein kinase C may be the mechanism for spontaneous hepatocarcinogenesis in choline-deficient rats.;Rapid high-performance liquid chromatographic assay for total homocysteine levels in human serum.;Choline, an essential nutrient for humans;Lactobacillus casei microbiological assay of folic acid derivatives in 96-well microtiter plates.;The active synthesis of phosphatidylcholine is required for very low density lipoprotein secretion from rat hepatocytes.;Plasma choline concentration in humans fed parenterally.;Betaine in the treatment of homocystinuria due to 5,10-methylenetetrahydrofolate reductase deficiency;A rapid method of total lipid extraction and purification.;Effect of methotrexate on homocysteine and other sulfur compounds in tissues of rats fed a normal or a defined, choline-deficient diet;Pathways and Regulation of Homocysteine Metabolism in Mammals;Institute of Medicine, National Academy of Sciences USA. Dietary reference intakes for folate, thiamin, riboflavin, niacin, vitamin B-12, panthothenic acid, biotin, and choline;The biosynthesis of choline and its relation to phospholipid metabolism.",
         "https://academic.oup.com/ajcn/article-pdf/81/2/440/23875105/440.pdf"
        ],
        [
         "40",
         "002a108f32eb157c0b88aea3c6f29d4b614c3b78",
         "Altered gene expression during rat Wolffian duct development following di(n-butyl) phthalate exposure.",
         "Di(n-butyl) phthalate (DBP) is a common plasticizer and solvent that disrupts androgen-dependent male reproductive development in rats. In utero exposure to 500 mg/kg/day DBP on gestation days (GD) 12 to 21 decreases androgen biosynthetic enzymes, resulting in decreased fetal testicular testosterone levels. One consequence of prenatal DBP exposure is malformed epididymides in adult rats. Reduced fetal testosterone levels may be responsible for the malformation, since testosterone is required for Wolffian duct stabilization and their development into epididymides. Currently, little is understood about the molecular mechanisms of Wolffian duct differentiation. The objective of this study was to identify changes in gene expression associated with altered morphology of the proximal Wolffian duct following in utero exposure to DBP. Pregnant Crl:CD(R) (SD) rats were gavaged with corn oil vehicle or 500 mg/kg/day DBP from GD 12 to GD 19 or 21. There were only small morphological differences between control and DBP-exposed Wolffian ducts on GD 19. On GD 21, 89% of male fetuses in the DBP dose group showed marked underdevelopment of Wolffian ducts, characterized by decreased coiling. RNA was isolated from Wolffian ducts on GD 19 and 21. Together with empirical information, cDNA microarrays were used to help identify candidate genes that could be associated with the morphological changes observed on GD 21. These candidate genes were analyzed by real-time RT-PCR. Changes in mRNA expression were observed in genes within the insulin-like growth factor (IGF) pathway, the matrix metalloproteinase (MMP) family, the extracellular matrix, and in other developmentally conserved signaling pathways. On GD 19, immunolocalization of IGF-1 receptor protein demonstrated an increase in cytoplasmic expression in the mesenchymal and epithelial cells. There was also a variable decrease in androgen receptor protein in ductal epithelial cells on GD 19. This study provides insight into the effects of antiandrogens on the molecular mechanisms involved in Wolffian duct development. The altered morphology and changes in gene expression following DBP exposure are suggestive of altered paracrine interactions between ductal epithelial cells and the surrounding mesenchyme during Wolffian duct differentiation due to lowered testosterone production.",
         "",
         "https://academic.oup.com/toxsci/article-pdf/86/1/161/4657319/kfi172.pdf"
        ],
        [
         "41",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         "Prospective grading of neoplastic change in rat esophagus epithelium using angle-resolved low-coherence interferometry.",
         "Angle-resolved low-coherence interferometry (a/LCI) is used to obtain quantitative, depth-resolved nuclear morphology measurements. We compare the average diameter and texture of cell nuclei in rat esophagus epithelial tissue to grading criteria established in a previous a/LCI study to prospectively grade neoplastic progression. We exploit the depth resolution of a/LCI to exclusively examine the basal layer of the epithelium, approximately 50 to 100 microm beneath the tissue surface, without the need for exogenous contrast agents, tissue sectioning, or fixation. The results of two studies are presented that compare the performance of two a/LCI modalities. Overall, the combined studies show 91% sensitivity and 97% specificity for detecting dysplasia, using histopathology as the standard. In addition, the studies enable the effects of dietary chemopreventive agents, difluoromethylornithine (DFMO) and curcumin, to be assessed by observing modulation in the incidence of neoplastic change. We demonstrate that a/LCI is highly effective for monitoring neoplastic change and can be applied to assessing the efficacy of chemopreventive agents in the rat esophagus.",
         "",
         "https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-10/issue-5/051604/Prospective-grading-of-neoplastic-change-in-rat-esophagus-epithelium-using/10.1117/1.2102767.pdf"
        ],
        [
         "42",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "The role of fear-relevant stimuli in visual search: a comparison of phylogenetic and ontogenetic stimuli.",
         "It has been argued that phylogenetic fear-relevant stimuli elicit preattentive capture of attention. To distinguish between fear relevance and time of appearance in evolutionary history, the authors compare phylogenetic and ontogenetic fear-relevant and fear-irrelevant stimuli in a visual search task. The authors found no evidence for a special role of phylogenetic fear-relevant stimuli; it seems that fear relevance in general is more important than is the evolutionary age. The pattern of results indicates that attention toward threatening stimuli is mainly affected by a late component that prolongs the disengagement of attention.",
         "Attentional control of the processing of neural and emotional stimuli.;Searching for threat;Emotion drives attention: detecting the snake in the grass.;The face in the crowd revisited: a threat advantage with schematic stimuli.;Fear and anxiety: animal models and human cognitive psychophysiology.;Facial Expressions of Emotion: Are Angry Faces Detected More Efficiently?;A subcortical pathway to the right amygdala mediating \"unseen\" fear.;It Takes a Confounded Face to Pop Out of a Crowd;On the automatic nature of phobic fear: conditioned electrodermal responses to masked fear-relevant stimuli.;Orienting and schizophrenia: stimulus significance, attention, and distraction in a signaled reaction time task.;A feature-integration theory of attention;The premise of equipotentiality in human classical conditioning: conditioned electrodermal responses to potentially phobic stimuli.;One-trial learning and superior resistance to extinction of autonomic responses conditioned to potentially phobic stimuli.;Phobias and preparedness;The cognitive neuroscience of emotion;Fears, phobias, and preparedness: toward an evolved module of fear and fear learning.;Unconscious emotion : Evolutionary perspectives, psychophysiological data, and neuropsychological mechanisms;A threat imminence approach to human fear responding : direction of threath, aversive contexts, and electrodermal responses;Preparedness and electrodermal fear-conditioning: ontogenetic vs phylogenetic explanations.;Finding the face in the crowd: an anger superiority effect.;Feature analysis in early vision: evidence from search asymmetries.;The emotional brain.;E-Prime;32306-1270), as editor of the Clinician's Research Digest newsletter for",
         "https://access.archive-ouverte.unige.ch/access/metadata/414fbfe1-4481-4240-92c1-e62a3593f0f0/download"
        ],
        [
         "43",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "WBN/Kob-Ht rats spontaneously develop dermatitis under conventional conditions: another possible model for atopic dermatitis.",
         "WBN/Kob-Ht rats (Ht-rats) raised under conventional conditions spontaneously developed dermatitis. In this study, we carried out histopathological analysis to elucidate the pathological features of the dermatitis in Ht-rats. We then tried to detect Staphylococcus species recovered from the skin lesions of Ht-rats. We also measured the serum levels of total IgE, IL-4 and IFN-gamma in these rats. The histopathological data indicated that inflammatory cells had infiltrated the skin lesions. Staphylococcus aureus was recovered from the skin lesions, and the serum levels of total IgE and IL-4 were elevated in Ht-rats with dermatitis. These results suggest that dermatitis in Ht-rats is similar to that in the DS-Nh mice, which has recently been proposed as an animal model for human atopic dermatitis.",
         "Role of the Nh (Non-hair) mutation in the development of dermatitis and hyperproduction of IgE in DS-Nh mice.;DS–Nh as an experimental model of atopic dermatitis induced by Staphylococcus aureus producing staphylococcal enterotoxin C;Characterization of dermatitis arising spontaneously in DS-Nh mice maintained under conventional conditions: another possible model for atopic dermatitis.;Atopic dermatitis: new insights and opportunities for therapeutic intervention.;Locus of dominant hairless gene (Ht) causing abnormal hair and keratinization maps to rat chromosome 10.;Characteristics of spontaneous erythema appeared in hairless rats.;IgE production by normal human lymphocytes is induced by interleukin 4 and suppressed by interferons gamma and alpha and prostaglandin E2.",
         "https://www.jstage.jst.go.jp/article/expanim/54/5/54_5_461/_pdf"
        ],
        [
         "44",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "Structural Analyses Reveal Phosphatidyl Inositols as Ligands for the NR5 Orphan Receptors SF-1 and LRH-1",
         "None",
         "The DEAD-Box Protein DP103 (Ddx20 or Gemin-3) Represses Orphan Nuclear Receptor Activity via SUMO Modification;Amplification of the steroidogenic factor 1 gene in childhood adrenocortical tumors.;Identification of liver receptor homolog-1 as a novel regulator of apolipoprotein AI gene transcription.;Synergy between LRH-1 and beta-catenin induces G1 cyclin-mediated cell proliferation.;Quantitative Proteomics of the Thyroid Hormone Receptor-Coregulator Interactions*[boxs];LRH-1: an orphan nuclear receptor involved in development, metabolism and steroidogenesis.;Sex determination: a ‘window’ of DAX1 activity;Generation and Characterization of Human Steroidogenic Factor 1 LBD Crystals with and without Bound Cofactor Peptide;SHIP-2 and PTEN Are Expressed and Active in Vascular Smooth Muscle Cell Nuclei, but Only SHIP-2 Is Associated with Nuclear Speckles*;Resurrecting the Ancestral Steroid Receptor: Ancient Origin of Estrogen Signaling;MrBayes 3: Bayesian phylogenetic inference under mixed models;Activation of nuclear receptors: a perspective from structural genomics.;StAR-related Lipid Transfer (START) Proteins: Mediators of Intracellular Lipid Metabolism*;The Drosophila Orphan Nuclear Receptor DHR38 Mediates an Atypical Ecdysteroid Signaling Pathway;Structural basis for ligand-independent activation of the orphan nuclear receptor LRH-1.;Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors;Nuclear lipid signalling;Nuclear lipids: New functions for old molecules?;X-Ray Structure of the hRORα LBD at 1.63 Å;Elevation of plasma phospholipid transfer protein in transgenic mice increases VLDL secretion Published, JLR Papers in Press, August 16, 2002. DOI 10.1194/jlr.M200166-JLR200;Crystal Structure of the HNF4α Ligand Binding Domain in Complex with Endogenous Fatty Acid Ligand*;Phosphorylation and Intramolecular Stabilization of the Ligand Binding Domain in the Nuclear Receptor Steroidogenic Factor 1;Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids.;Inhibition of androgen receptor (AR) function by the reproductive orphan nuclear receptor DAX-1.;Control of Cholesterol Turnover in the Mouse*;Phosphatidic Acid-Mediated Mitogenic Activation of mTOR Signaling;The PITP family of phosphatidylinositol transfer proteins;Characterization of a G Protein-activated Phosphoinositide 3-Kinase in Vascular Smooth Muscle Cell Nuclei*;Binding of ligands and activation of transcription by nuclear receptors.;Structure of a Multifunctional Protein;Crystal Structure of the Ligand-binding Domain of the Ultraspiracle Protein USP, the Ortholog of Retinoid X Receptors in Insects*;Haploinsufficiency of steroidogenic factor-1 in mice disrupts adrenal development leading to an impaired stress response.;Nuclear Lipid Signaling;Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors.;Ligand binding and nuclear receptor evolution;A new method to localize and test the significance of incongruence: detecting domain shuffling in the nuclear receptor superfamily.;Inferring the historical patterns of biological evolution;Oxysterol Regulation of Steroidogenic Acute Regulatory Protein Gene Expression;Structure and specificity of nuclear receptor-coactivator interactions.;25-Hydroxycholesterol is not a ligand for the orphan nuclear receptor steroidogenic factor-1 (SF-1).;Wilms' Tumor 1 and Dax-1 Modulate the Orphan Nuclear Receptor SF-1 in Sex-Specific Gene Expression;Activation of the orphan nuclear receptor steroidogenic factor 1 by oxysterols.;DAX-1 inhibits SF-1-mediated transactivation via a carboxy-terminal domain that is deleted in adrenal hypoplasia congenita;A new method of inference of ancestral nucleotide and amino acid sequences.;CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;The nuclear receptor steroidogenic factor 1 acts at multiple levels of the reproductive axis.;A cell-specific nuclear receptor is essential for adrenal and gonadal development and sexual differentiation;Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology;Repression of orphan nuclear recep - or activity via SUMO - modification and the Dead - Box Protein P 103 ( Ddx 20 , Gemin - 3 );Nuclear lipids: key signaling effectors in the nervous system and other tissues;X-ray structure of the hRORalpha LBD at 1.63 A: structural and functional data that cholesterol or a cholesterol derivative is the natural ligand of RORalpha.;Nuclear-receptor ligands and ligand-binding domains.;Standard Fmoc protocols.;Methods Enzymol",
         "http://www.cell.com/article/S009286740500098X/pdf"
        ],
        [
         "45",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "Rat Strain Effects of AT1 Receptor Activation on D1 Dopamine Receptors in Immortalized Renal Proximal Tubule Cells",
         "The dopaminergic and renin-angiotensin systems regulate blood pressure, in part, by affecting sodium transport in renal proximal tubules (RPTs). We have reported that activation of a D1-like receptor decreases AT1 receptor expression in the mouse kidney and in immortalized RPT cells from Wistar-Kyoto (WKY) rats. The current studies were designed to test the hypothesis that activation of the AT1 receptor can also regulate the D1 receptor in RPT cells, and this regulation is aberrant in spontaneously hypertensive rats (SHRs). Long-term (24 hours) stimulation of RPT cells with angiotensin II, via AT1 receptors increased total cellular D1 receptor protein in a time- and concentration-dependent manner in WKY but not in SHR cells. Short-term stimulation (15 minutes) with angiotensin II did not affect total cellular D1 receptor protein in either rat strain. However, in the short-term experiments, angiotensin II decreased cell surface membrane D1 receptor protein in WKY but not in SHR cells. D1 and AT1 receptors colocalized (confocal microscopy) and their coimmunoprecipitation was greater in WKY than in SHRs. However, AT1/D1 receptor coimmunoprecipitation was decreased by angiotensin II (10−8M/24 hours) to a similar extent in WKY (−22±8%) and SHRs (−22±12%). In summary, these studies show that AT1 and D1 receptors interact differently in RPT cells from WKY and SHRs. It is possible that an angiotensin II-mediated increase in D1 receptors and dissociation of AT1 from D1 receptors serve to counter regulate the long-term action of angiotensin II in WKY rats; different effects are seen in SHRs.",
         "",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.HYP.0000184251.01159.72"
        ],
        [
         "46",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         "The value of c-kit mutational analysis in a cytokeratin positive gastrointestinal stromal tumour",
         "The expression of cytokeratins in gastrointestinal stromal tumours (GISTs) is rare and may lead to diagnostic confusion when it occurs. This report describes a metastatic GIST that stained strongly for cytokeratins, CD117, and CD34 in a patient who was previously diagnosed with gastric epithelioid angiosarcoma. A review of both tumours showed the same histological and immunohistochemical profiles, and c-kit molecular analysis revealed an insertional mutation at codon 558 of exon 11 in both tumours. Thus, pathologists should be aware that GISTs can occasionally express cytokeratins, and that c-kit mutational investigations may have a key diagnostic role and may prevent diagnostic mistakes that could have important clinical implications.",
         "",
         "https://jcp.bmj.com/content/jclinpath/58/9/991.full.pdf"
        ],
        [
         "47",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "Contribution of high p34cdc2 kinase activity to premature chromosome condensation of injected somatic cell nuclei in rat oocytes.",
         "The present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated rat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes were placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-dependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15-45 min rather than 46-120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity transiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-phase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed oocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat oocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent level of aged (120 min) or enucleated (+60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2 kinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were treated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated oocytes. A significantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control oocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus cell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage. These results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for their inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemicals such as MG132 is in part effective for rat oocytes to promote PCC and further development.",
         "",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/129/2/1290171.pdf"
        ],
        [
         "48",
         "002cc4c2e457c3cc0fb0f704d0483a003e8a0575",
         "Comparison of automated microbroth dilution and agar dilution for antimicrobial susceptibility of Campylobacter jejuni isolated from dairy sources.",
         "OBJECTIVES\nTo compare the agreement between microbroth dilution and agar dilution for antimicrobial susceptibility testing of Campylobacter jejuni.\n\n\nMETHODS\nUtilizing commercially prepared antimicrobial panels, microbroth dilution was compared with agar dilution for determining antimicrobial susceptibility in C. jejuni isolates. To assess the performance of both techniques for ampicillin, 190 C. jejuni isolates from dairy cattle were utilized. A group of 172 C. jejuni isolates from dairy sources were used to compare the susceptibility to ciprofloxacin, erythromycin, nalidixic acid and tetracycline.\n\n\nRESULTS\nOur results indicate that microbroth dilution and agar dilution agree within +/-1 log2 dilution for 86.7% of the isolates tested. Ciprofloxacin had the highest level of agreement for isolates tested by both techniques, resulting in a kappa of 0.886 and 97.1% agreement +/-1 log2 dilution. The least agreement was observed in determining the susceptibility of isolates to ampicillin and erythromycin (82.1 and 79.7% agreement +/-1 log2 dilution). However, kappa statistics were considered to have good agreement for these antimicrobials. There were no significant differences in the summary statistics for any of the five antimicrobials evaluated for the isolates analysed by the percentage of resistant isolates, MIC50, MIC75 or MIC90 beyond +/-1 log2 dilution. There was no association in the classification of resistance by the testing methods employed. We also demonstrated that the quality control strain of C. jejuni ATCC 33650 performed in a consistent manner for both agar dilution and microbroth dilution.\n\n\nCONCLUSIONS\nMicrobroth dilution may be an acceptable alternative to agar dilution for determining susceptibility of C. jejuni in research or surveillance where flow of samples, labour efficiency and cost may restrict the use of agar dilution.",
         "",
         "https://academic.oup.com/jac/article-pdf/56/4/686/2308205/dki309.pdf"
        ],
        [
         "49",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "Numerical Periodic Normalization for Codim 1 Bifurcations of Limit Cycles",
         "Explicit computational formulas for the coefficients of the periodic normal forms for all codim 1 bifurcations of limit cycles in generic autonomous ODEs are derived. They include second-order coefficients for the fold (limit point) bifurcation, as well as third-order coefficients for the flip (period-doubling) and Neimark--Sacker (torus) bifurcations. The formulas are independent of the dimension of the phase space and involve solutions of certain boundary-value problems on the interval [0,T], where T is the period of the critical cycle, as well as multilinear functions from the Taylor expansion of the right-hand sides near the cycle. The formulas allow us to distinguish between sub- and supercritical bifurcations, in agreement with earlier asymptotic expansions of the bifurcating solutions. Our formulation makes it possible to use robust numerical boundary-value algorithms based on orthogonal collocation, rather than shooting techniques, which greatly expands its applicability. The actual implementation is described in detail. We include three numerical examples, in which codim 2 singularities are detected along branches of codim 1 bifurcations of limit cycles as zeros of the periodic normal form coefficients.",
         "Numerical Methods for Two-Parameter Local Bifurcation Analysis of Maps;Newton's descent method for the determination of invariant tori.;A Newton Method for Locating Invariant Tori of Maps;Sensitivity Analysis of ODES/DAES Using the Taylor Series Method;Numerical Normal Forms for Codim 2 Bifurcations of Fixed Points with at Most Two Critical Eigenvalues;Numerical Methods for the Continuation of Invariant Tori;MATCONT: A MATLAB package for numerical bifurcation analysis of ODEs;Cl_matcont: a continuation toolbox in Matlab;Computation of Periodic Solution Bifurcations in ODEs Using Bordered Systems;Chapter 4 – Numerical Continuation, and Computation of Normal Forms;Computing Periodic Orbits and their Bifurcations with Automatic Differentiation;Numerical Normalization Techniques for All Codim 2 Bifurcations of Equilibria in ODE's;Topics in bifurcation theory and applications;Elements of applied bifurcation theory (2nd ed.);Harmonic balance analysis of period-doubling bifurcations with implications for control of nonlinear dynamics;Algorithm 755: ADOL-C: a package for the automatic differentiation of algorithms written in C/C++;Bifurcation and predictability analysis of a low-order atmospheric circulation model;Models of complex dynamics in nonlinear systems;Bifurcation sequences at 1:4 resonance: an inventory;Harmonic balance methods for the analysis of chaotic dynamics in nonlinear systems;BIFURCATIONS AND CHAOS IN A PERIODIC PREDATOR-PREY MODEL;NUMERICAL ANALYSIS AND CONTROL OF BIFURCATION PROBLEMS (II): BIFURCATION IN INFINITE DIMENSIONS;Numerical Calculation of Invariant Tori;THE QUASIPERIODIC ROUTE TO CHAOS IN A MODEL OF THE PEROXIDASE-OXIDASE REACTION;Global Characterization of the Normal Form for a Vector Field Near a Closed Orbit;A simple global characterization for normal forms of singular vector fields;Normal form reduction for time-periodically driven differential equations;Numerical computation of invariant circles of maps;NONLINEAR OSCILLATIONS, DYNAMICAL SYSTEMS, AND BIFURCATIONS OF VECTOR FIELDS (Applied Mathematical Sciences, 42);Irregularity: a fundamental property of the atmosphere*;Nonlinear Oscillations, Dynamical Systems, and Bifurcations of Vector Fields;Collocation at Gaussian Points;Continuation of Quasi-periodic Invariant Tori;The fold-flip bifurcation;On the computation of characteristic multipliers for predicting limit cycle bifurcations;Elements of applied bifurcation theory;AUTO 2000 : CONTINUATION AND BIFURCATION SOFTWARE FOR ORDINARY DIFFERENTIAL EQUATIONS (with HomCont);CONTENT: A multiplatform environment for analyzing dynamical systems;Normal forms of bifurcating periodic orbits;Numerical computation of periodic solution branches and oscillatory dynamics of the stirred tank reactor with A → B → C reactions;Geometrical Methods in the Theory of Ordinary Differential Equations;Elementary stability and bifurcation theory;Bifurcation of maps and applications;Topics in bifurcation theory;Numerical solution of boundary value problems in ordinary differential equations;Numerical Continuation, And Computation Of Normal Forms;CAPD: Computer assisted proofs in dynamics",
         "http://arxiv.org/pdf/1111.4445.pdf"
        ]
       ],
       "shape": {
        "columns": 5,
        "rows": 69993
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>referenceTitles</th>\n",
       "      <th>openAccessPdf</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>Disruption of Fibroblast Growth Factor Signal ...</td>\n",
       "      <td>Purpose: Synovial sarcoma is a soft tissue sar...</td>\n",
       "      <td></td>\n",
       "      <td>https://aacrjournals.org/clincancerres/article...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>Shuffled iterative decoding</td>\n",
       "      <td>Shuffled versions of iterative decoding of low...</td>\n",
       "      <td>Low-Density Parity-Check Codes;Turbo decoder a...</td>\n",
       "      <td>http://www.merl.com/publications/docs/TR2005-0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>Diphasic effects of Astragalus membranaceus BU...</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.jstage.jst.go.jp/article/bpb/28/8/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>Critical roles of CD146 in zebrafish vascular ...</td>\n",
       "      <td>In this report, we use zebrafish as a model sy...</td>\n",
       "      <td></td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/pdfdirect/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>Differential Contribution of Troponin I Phosph...</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td></td>\n",
       "      <td>http://www.jbc.org/article/S0021925820589394/pdf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69988</th>\n",
       "      <td>fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6</td>\n",
       "      <td>Irregular cycles and steroid hormones in polyc...</td>\n",
       "      <td>BACKGROUND\\nThis cross-sectional study was und...</td>\n",
       "      <td>Predictors for treatment failure after laparos...</td>\n",
       "      <td>https://academic.oup.com/humrep/article-pdf/20...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69989</th>\n",
       "      <td>fffbc9b37f20515de2ff4fe60353d52dc37e65f5</td>\n",
       "      <td>An organizational grid of federated MOSIX clus...</td>\n",
       "      <td>MOSIX is a cluster management system that uses...</td>\n",
       "      <td>From sandbox to playground: dynamic virtual en...</td>\n",
       "      <td>http://www.cs.virginia.edu/~jl3aq/courses/CS85...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69990</th>\n",
       "      <td>fffd2d6d54b4023ece28bfdebffdfb00502a09ec</td>\n",
       "      <td>Mutations in the RNA Polymerase III Subunit Rp...</td>\n",
       "      <td>ABSTRACT Termination by RNA polymerase III (Po...</td>\n",
       "      <td>Nuclear surveillance and degradation of hypomo...</td>\n",
       "      <td>https://europepmc.org/articles/pmc543423?pdf=r...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69991</th>\n",
       "      <td>ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3</td>\n",
       "      <td>The Core Histone N-terminal Tail Domains Funct...</td>\n",
       "      <td>Salt-dependent oligomerization of nucleosomal ...</td>\n",
       "      <td>Histone modifications: combinatorial complexit...</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820789519/pdf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69992</th>\n",
       "      <td>ffffeaedbe7931d161ab532015de3be15f99b180</td>\n",
       "      <td>Of mice and men: should we extrapolate rodent ...</td>\n",
       "      <td>None</td>\n",
       "      <td>Developmental neurotoxicity of ketamine: morph...</td>\n",
       "      <td>https://pubs.asahq.org/anesthesiology/article-...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>69993 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paperId  \\\n",
       "0      0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1      000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2      000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3      0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4      00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "...                                         ...   \n",
       "69988  fffb5a27f191307c8f2fed3a7d1162f93cbb8cb6   \n",
       "69989  fffbc9b37f20515de2ff4fe60353d52dc37e65f5   \n",
       "69990  fffd2d6d54b4023ece28bfdebffdfb00502a09ec   \n",
       "69991  ffff785a92f7cacc7ecb6f3ac3f20ba91d5213b3   \n",
       "69992  ffffeaedbe7931d161ab532015de3be15f99b180   \n",
       "\n",
       "                                                   title  \\\n",
       "0      Disruption of Fibroblast Growth Factor Signal ...   \n",
       "1                            Shuffled iterative decoding   \n",
       "2      Diphasic effects of Astragalus membranaceus BU...   \n",
       "3      Critical roles of CD146 in zebrafish vascular ...   \n",
       "4      Differential Contribution of Troponin I Phosph...   \n",
       "...                                                  ...   \n",
       "69988  Irregular cycles and steroid hormones in polyc...   \n",
       "69989  An organizational grid of federated MOSIX clus...   \n",
       "69990  Mutations in the RNA Polymerase III Subunit Rp...   \n",
       "69991  The Core Histone N-terminal Tail Domains Funct...   \n",
       "69992  Of mice and men: should we extrapolate rodent ...   \n",
       "\n",
       "                                                abstract  \\\n",
       "0      Purpose: Synovial sarcoma is a soft tissue sar...   \n",
       "1      Shuffled versions of iterative decoding of low...   \n",
       "2      This study was designed to investigate the eff...   \n",
       "3      In this report, we use zebrafish as a model sy...   \n",
       "4      Cardiac troponin I is a phosphorylation target...   \n",
       "...                                                  ...   \n",
       "69988  BACKGROUND\\nThis cross-sectional study was und...   \n",
       "69989  MOSIX is a cluster management system that uses...   \n",
       "69990  ABSTRACT Termination by RNA polymerase III (Po...   \n",
       "69991  Salt-dependent oligomerization of nucleosomal ...   \n",
       "69992                                               None   \n",
       "\n",
       "                                         referenceTitles  \\\n",
       "0                                                          \n",
       "1      Low-Density Parity-Check Codes;Turbo decoder a...   \n",
       "2                                                          \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "69988  Predictors for treatment failure after laparos...   \n",
       "69989  From sandbox to playground: dynamic virtual en...   \n",
       "69990  Nuclear surveillance and degradation of hypomo...   \n",
       "69991  Histone modifications: combinatorial complexit...   \n",
       "69992  Developmental neurotoxicity of ketamine: morph...   \n",
       "\n",
       "                                           openAccessPdf  \n",
       "0      https://aacrjournals.org/clincancerres/article...  \n",
       "1      http://www.merl.com/publications/docs/TR2005-0...  \n",
       "2      https://www.jstage.jst.go.jp/article/bpb/28/8/...  \n",
       "3      https://onlinelibrary.wiley.com/doi/pdfdirect/...  \n",
       "4       http://www.jbc.org/article/S0021925820589394/pdf  \n",
       "...                                                  ...  \n",
       "69988  https://academic.oup.com/humrep/article-pdf/20...  \n",
       "69989  http://www.cs.virginia.edu/~jl3aq/courses/CS85...  \n",
       "69990  https://europepmc.org/articles/pmc543423?pdf=r...  \n",
       "69991   http://www.jbc.org/article/S0021925820789519/pdf  \n",
       "69992  https://pubs.asahq.org/anesthesiology/article-...  \n",
       "\n",
       "[69993 rows x 5 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "metadata, textdata = load_dataset(dataset_source='semanticsscholar', years=2005, data_types=['metadata', 'text'])\n",
    "display(metadata)\n",
    "display(textdata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sending incremental file list\n",
      "\n",
      "sent 1,710 bytes  received 33 bytes  3,486.00 bytes/sec\n",
      "total size is 8,750,562,247  speedup is 5,020,402.90\n",
      "Synced /home/jul/DST/asXiv/data/metadata -> /mnt/g/My Drive/DST/MLOps_projects/asXiv/data/metadata (rsync complete)\n"
     ]
    }
   ],
   "source": [
    "push_data_to_drive('metadata')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def load_dataset(path_to_data, dataset_source='semanticsscholar', years=None, data_types=None):\n",
    "    \"\"\"Fetch dataset metadata and text data for given years and data types.\"\"\"\n",
    "    years = (\n",
    "        [str(year) for year in years] if isinstance(years, list) else\n",
    "        [str(years)] if years else\n",
    "        [year for year in os.listdir(os.path.join(path_to_data, 'metadata')) if year.isdigit()]\n",
    "    )\n",
    "    data_types = data_types or ['metadata', 'text']\n",
    "    data_types = [data_types] if not isinstance(data_types, list) else data_types\n",
    "\n",
    "    load_metadata = 'metadata' in data_types\n",
    "    load_textdata = 'text' in data_types or 'pdf' in data_types\n",
    "    load_pdfdata = 'pdf' in data_types\n",
    "\n",
    "    metadata, textdata, pdfdata = [], [], []\n",
    "    metadata_years, textdata_years, pdfdata_years = [], [], []\n",
    "\n",
    "    for year in years:\n",
    "        if load_metadata:\n",
    "            metadata_path = os.path.join(path_to_data, 'metadata', year, f'{dataset_source}_metadata_{year}.parquet')\n",
    "            if os.path.isfile(metadata_path):\n",
    "                metadata.append(pd.read_parquet(metadata_path, engine=\"pyarrow\"))\n",
    "                metadata_years.append(year)\n",
    "            else:\n",
    "                print(f\"No metadata for year {year}\")\n",
    "        \n",
    "        if load_textdata:\n",
    "            textdata_path = os.path.join(path_to_data, 'text', year, f'{dataset_source}_text_{year}.parquet')\n",
    "            if os.path.isfile(textdata_path):\n",
    "                textdata.append(pd.read_parquet(textdata_path, engine=\"pyarrow\"))\n",
    "                textdata_years.append(year)\n",
    "            else:\n",
    "                print(f\"No text data for year {year}\")\n",
    "\n",
    "        if load_pdfdata:\n",
    "            pdfdata_path = os.path.join(path_to_data, 'pdf', year, f'{dataset_source}_textpdf_{year}.parquet')\n",
    "            if not os.path.isfile(pdfdata_path):\n",
    "                pdfdata_path = os.path.join(path_to_data, 'text', year, f'{dataset_source}_text_{year}.parquet')\n",
    "                print(f\"No pdf data for year {year}. Will try to load text data instead\")\n",
    "            if os.path.isfile(pdfdata_path):\n",
    "                pdfdata.append(pd.read_parquet(pdfdata_path, engine=\"pyarrow\"))\n",
    "                pdfdata_years.append(year)\n",
    "            else:\n",
    "                print(f\"No pdf or text data for year {year}\")\n",
    "    \n",
    "    metadata = pd.concat(metadata, axis=0, ignore_index=True).reset_index(drop=True) if metadata else []\n",
    "    textdata = pd.concat(textdata, axis=0, ignore_index=True).reset_index(drop=True) if textdata else []\n",
    "    pdfdata = pd.concat(pdfdata, axis=0, ignore_index=True).reset_index(drop=True) if pdfdata else []\n",
    "    \n",
    "    msg_parts = []\n",
    "    if load_metadata:\n",
    "        msg_parts.append(f'metadata loaded for years: {metadata_years}')\n",
    "    if load_textdata:\n",
    "        msg_parts.append(f'text data loaded for years: {textdata_years}')\n",
    "    if load_pdfdata:\n",
    "        msg_parts.append(f'pdf data loaded for years: {pdfdata_years}')\n",
    "    \n",
    "    if msg_parts:\n",
    "        print(\"; \".join(msg_parts))\n",
    "    if load_pdfdata:\n",
    "        textdata = pdfdata.set_index(\"paperId\").combine_first(textdata.set_index(\"paperId\")).reset_index(drop=False)\n",
    "\n",
    "    if load_metadata and load_textdata and len(metadata) != len(textdata):\n",
    "        raise ValueError(\"Metadata and text data don't have the same length.\")\n",
    "    \n",
    "    output = (data for data in [metadata, textdata] if len(data) > 0)\n",
    "    return output\n",
    "\n",
    "\n",
    "def push_data_to_drive(datatype: str, confirm: bool = True):\n",
    "    valid_types = {\n",
    "        \"textdata\": DATA_PATH / \"textdata\",\n",
    "        \"metadata\": DATA_PATH / \"metadata\",\n",
    "        \"pdfdata\": DATA_PATH / \"pdfdata\"\n",
    "    }\n",
    "\n",
    "    if datatype not in valid_types:\n",
    "        raise ValueError(f\"Invalid datatype '{datatype}'. Choose from: {list(valid_types.keys())}\")\n",
    "\n",
    "    source = valid_types[datatype]\n",
    "    destination = DRIVE_DATA_PATH / datatype\n",
    "\n",
    "    if confirm:\n",
    "        confirm = input(f\"Are you sure you want to sync {source} → {destination}? (Y/n): \").strip().lower()\n",
    "        if confirm not in ['y', 'yes', '']:\n",
    "            print(\"Sync aborted.\")\n",
    "            return\n",
    "    \n",
    "    subprocess.run([\n",
    "        \"rsync\", \"-rv\", \"--delete\", \"--size-only\", #synch recursively and don't check timestamps\n",
    "        str(source) + '/',  # ensures contents are copied, not the folder itself\n",
    "        str(destination)\n",
    "    ])\n",
    "    print(f\"Synced {source} -> {destination} (rsync complete, Google Drive sync in progress...)\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "ename": "FileNotFoundError",
     "evalue": "[Errno 2] No such file or directory: '/home/jul/DST/recoRAG/data/temp/7/0b3b4465396deda32b6c74030afe241c59876063/PIIS0092867406004983.pdf'",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mFileNotFoundError\u001b[39m                         Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[3]\u001b[39m\u001b[32m, line 9\u001b[39m\n\u001b[32m      7\u001b[39m host = \u001b[33m'\u001b[39m\u001b[33mhttp://127.0.0.1\u001b[39m\u001b[33m'\u001b[39m\n\u001b[32m      8\u001b[39m port = \u001b[33m'\u001b[39m\u001b[33m8080\u001b[39m\u001b[33m'\u001b[39m\n\u001b[32m----> \u001b[39m\u001b[32m9\u001b[39m output_dict = \u001b[43mparse_pdf\u001b[49m\u001b[43m(\u001b[49m\u001b[43mhost\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mpdf_path\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mport\u001b[49m\u001b[43m=\u001b[49m\u001b[43mport\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m     11\u001b[39m sections = {\u001b[33m'\u001b[39m\u001b[33mpdf_abstract\u001b[39m\u001b[33m'\u001b[39m: output_dict[\u001b[33m'\u001b[39m\u001b[33mabstractText\u001b[39m\u001b[33m'\u001b[39m],\n\u001b[32m     12\u001b[39m         \u001b[33m'\u001b[39m\u001b[33mpdf_introduction\u001b[39m\u001b[33m'\u001b[39m: \u001b[33m'\u001b[39m\u001b[33m \u001b[39m\u001b[33m'\u001b[39m.join(s[\u001b[33m'\u001b[39m\u001b[33mtext\u001b[39m\u001b[33m'\u001b[39m] \u001b[38;5;28;01mfor\u001b[39;00m s \u001b[38;5;129;01min\u001b[39;00m output_dict[\u001b[33m'\u001b[39m\u001b[33msections\u001b[39m\u001b[33m'\u001b[39m] \u001b[38;5;28;01mif\u001b[39;00m \u001b[33m'\u001b[39m\u001b[33mheading\u001b[39m\u001b[33m'\u001b[39m \u001b[38;5;129;01min\u001b[39;00m s.keys() \u001b[38;5;129;01mand\u001b[39;00m \u001b[33m'\u001b[39m\u001b[33mintroduction\u001b[39m\u001b[33m'\u001b[39m \u001b[38;5;129;01min\u001b[39;00m s[\u001b[33m'\u001b[39m\u001b[33mheading\u001b[39m\u001b[33m'\u001b[39m].lower()),\n\u001b[32m     13\u001b[39m         \u001b[33m'\u001b[39m\u001b[33mpdf_results\u001b[39m\u001b[33m'\u001b[39m: \u001b[33m'\u001b[39m\u001b[33m \u001b[39m\u001b[33m'\u001b[39m.join(s[\u001b[33m'\u001b[39m\u001b[33mtext\u001b[39m\u001b[33m'\u001b[39m] \u001b[38;5;28;01mfor\u001b[39;00m s \u001b[38;5;129;01min\u001b[39;00m output_dict[\u001b[33m'\u001b[39m\u001b[33msections\u001b[39m\u001b[33m'\u001b[39m] \u001b[38;5;28;01mif\u001b[39;00m \u001b[33m'\u001b[39m\u001b[33mheading\u001b[39m\u001b[33m'\u001b[39m \u001b[38;5;129;01min\u001b[39;00m s.keys() \u001b[38;5;129;01mand\u001b[39;00m \u001b[33m'\u001b[39m\u001b[33mresults\u001b[39m\u001b[33m'\u001b[39m \u001b[38;5;129;01min\u001b[39;00m s[\u001b[33m'\u001b[39m\u001b[33mheading\u001b[39m\u001b[33m'\u001b[39m].lower()),\n\u001b[32m   (...)\u001b[39m\u001b[32m     18\u001b[39m         \u001b[33m'\u001b[39m\u001b[33mreference_titles\u001b[39m\u001b[33m'\u001b[39m: \u001b[33m'\u001b[39m\u001b[33m \u001b[39m\u001b[33m'\u001b[39m.join(s[\u001b[33m'\u001b[39m\u001b[33mtitle\u001b[39m\u001b[33m'\u001b[39m] \u001b[38;5;28;01mfor\u001b[39;00m s \u001b[38;5;129;01min\u001b[39;00m output_dict[\u001b[33m'\u001b[39m\u001b[33mreferences\u001b[39m\u001b[33m'\u001b[39m] \u001b[38;5;28;01mif\u001b[39;00m output_dict[\u001b[33m'\u001b[39m\u001b[33mreferences\u001b[39m\u001b[33m'\u001b[39m] \u001b[38;5;129;01mand\u001b[39;00m \u001b[33m'\u001b[39m\u001b[33mtitle\u001b[39m\u001b[33m'\u001b[39m \u001b[38;5;129;01min\u001b[39;00m s.keys())\n\u001b[32m     19\u001b[39m         }\n\u001b[32m     21\u001b[39m pp = pprint.PrettyPrinter(indent=\u001b[32m4\u001b[39m)\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/anaconda3/envs/recoRAG/lib/python3.11/site-packages/science_parse_api/api.py:46\u001b[39m, in \u001b[36mparse_pdf\u001b[39m\u001b[34m(server_address, file_path, port, timeout)\u001b[39m\n\u001b[32m     44\u001b[39m     url = \u001b[33mf\u001b[39m\u001b[33m'\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mserver_address\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;132;01m{\u001b[39;00mendpoint\u001b[38;5;132;01m}\u001b[39;00m\u001b[33m'\u001b[39m\n\u001b[32m     45\u001b[39m file_name = file_path.name\n\u001b[32m---> \u001b[39m\u001b[32m46\u001b[39m files = {\u001b[33m'\u001b[39m\u001b[33mdata-binary\u001b[39m\u001b[33m'\u001b[39m: (file_name, \u001b[43mfile_path\u001b[49m\u001b[43m.\u001b[49m\u001b[43mopen\u001b[49m\u001b[43m(\u001b[49m\u001b[33;43m'\u001b[39;49m\u001b[33;43mrb\u001b[39;49m\u001b[33;43m'\u001b[39;49m\u001b[43m)\u001b[49m, \u001b[33m'\u001b[39m\u001b[33mapplication/pdf\u001b[39m\u001b[33m'\u001b[39m,\n\u001b[32m     47\u001b[39m                          {\u001b[33m'\u001b[39m\u001b[33mExpires\u001b[39m\u001b[33m'\u001b[39m: \u001b[33m'\u001b[39m\u001b[33m0\u001b[39m\u001b[33m'\u001b[39m})}\n\u001b[32m     48\u001b[39m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m     49\u001b[39m     response = requests.post(url, files=files,\n\u001b[32m     50\u001b[39m                              headers={\u001b[33m'\u001b[39m\u001b[33mAccept\u001b[39m\u001b[33m'\u001b[39m: \u001b[33m'\u001b[39m\u001b[33mapplication/json\u001b[39m\u001b[33m'\u001b[39m},\n\u001b[32m     51\u001b[39m                              timeout=timeout)\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/anaconda3/envs/recoRAG/lib/python3.11/pathlib.py:1044\u001b[39m, in \u001b[36mPath.open\u001b[39m\u001b[34m(self, mode, buffering, encoding, errors, newline)\u001b[39m\n\u001b[32m   1042\u001b[39m \u001b[38;5;28;01mif\u001b[39;00m \u001b[33m\"\u001b[39m\u001b[33mb\u001b[39m\u001b[33m\"\u001b[39m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;129;01min\u001b[39;00m mode:\n\u001b[32m   1043\u001b[39m     encoding = io.text_encoding(encoding)\n\u001b[32m-> \u001b[39m\u001b[32m1044\u001b[39m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mio\u001b[49m\u001b[43m.\u001b[49m\u001b[43mopen\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmode\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mbuffering\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mencoding\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merrors\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnewline\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[31mFileNotFoundError\u001b[39m: [Errno 2] No such file or directory: '/home/jul/DST/recoRAG/data/temp/7/0b3b4465396deda32b6c74030afe241c59876063/PIIS0092867406004983.pdf'"
     ]
    }
   ],
   "source": [
    "# pages = extract_pages(\"/home/jul/DST/recoRAG/data/temp/7/0b5101910aca09a90cdbc0281ad8f3f7638e0e61/1471-2105-7-364.pdf\")\n",
    "import pprint\n",
    "\n",
    "pdf_path = Path(\"/home/jul/DST/recoRAG/data/temp/7/0b3b4465396deda32b6c74030afe241c59876063/PIIS0092867406004983.pdf\").resolve()\n",
    "host = 'http://127.0.0.1'\n",
    "port = '8080'\n",
    "output_dict = parse_pdf(host, pdf_path, port=port)\n",
    "\n",
    "sections = {'pdf_abstract': output_dict['abstractText'],\n",
    "        'pdf_introduction': ' '.join(s['text'] for s in output_dict['sections'] if 'heading' in s.keys() and 'introduction' in s['heading'].lower()),\n",
    "        'pdf_results': ' '.join(s['text'] for s in output_dict['sections'] if 'heading' in s.keys() and 'results' in s['heading'].lower()),\n",
    "        'pdf_discussion': ' '.join(s['text'] for s in output_dict['sections'] if 'heading' in s.keys() and 'discussion' in s['heading'].lower()),\n",
    "        'pdf_methods': None,\n",
    "        'pdf_text': ' '.join(s['text'] for s in output_dict['sections'] if 'text' in s.keys()),\n",
    "        'author_list': ' / '.join(s['name'] for s in output_dict['authors'] if 'name' in s.keys()),\n",
    "        'reference_titles': ' '.join(s['title'] for s in output_dict['references'] if output_dict['references'] and 'title' in s.keys())\n",
    "        }\n",
    "\n",
    "pp = pprint.PrettyPrinter(indent=4)\n",
    "pp.pprint(output_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_pdf_num_pages(pdf_path):\n",
    "    # Fast way to get total number of pages\n",
    "    with open(pdf_path, 'rb') as f:\n",
    "        return sum(1 for _ in PDFPage.get_pages(f))\n",
    "\n",
    "def isvalid_page(pdf_path, page_idx):\n",
    "    try:\n",
    "        # extract_pages yields a single item here due to page_numbers=[page_idx]\n",
    "        for _ in extract_pages(pdf_path, page_numbers=[page_idx]):\n",
    "            return True  # Successfully processed page\n",
    "    except Exception:\n",
    "        return False  # Failed due to extract_pages exception\n",
    "    return False\n",
    "\n",
    "def get_valid_pages(pdf_path, timeout=None, num_processes=None):\n",
    "    num_processes = 1 if num_processes is None else num_processes\n",
    "    timeout = 0.1 if timeout is None else timeout\n",
    "    valid_pages = []\n",
    "    try:\n",
    "        n_pages = get_pdf_num_pages(pdf_path)\n",
    "\n",
    "        with multiprocessing.Pool(processes=num_processes) as pool:\n",
    "            async_results = []\n",
    "            # 1) Submit one task per page\n",
    "            for i in range(n_pages):\n",
    "                async_results.append(\n",
    "                    pool.apply_async(isvalid_page, (pdf_path, i))\n",
    "                )\n",
    "\n",
    "            # 2) Collect results (with timeouts)\n",
    "            for i, result in enumerate(async_results):\n",
    "                try:\n",
    "                    success = result.get(timeout=timeout)\n",
    "                    if success:\n",
    "                        valid_pages.append(i)\n",
    "                except multiprocessing.TimeoutError:\n",
    "                    # 3) If any page times out, forcibly kill the whole pool\n",
    "                    pool.terminate()\n",
    "                    break  # exit the for-loop\n",
    "    except Exception as e:\n",
    "        pdfminer_logger.warning(f\"Error parsing PDF num pages for {pdf_path}: {e}\")\n",
    "\n",
    "    return valid_pages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'sections' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mNameError\u001b[39m                                 Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[4]\u001b[39m\u001b[32m, line 12\u001b[39m\n\u001b[32m      9\u001b[39m                 filepath = glob.glob(os.path.join(dir_path, \u001b[33m'\u001b[39m\u001b[33m*.pdf\u001b[39m\u001b[33m'\u001b[39m))[\u001b[32m0\u001b[39m]\n\u001b[32m     10\u001b[39m                 sections,_,_ = extract_pdf_sections(os.path.join(filepath), \u001b[33m'\u001b[39m\u001b[33mBengtsson, Hossjer\u001b[39m\u001b[33m'\u001b[39m, dirname)\n\u001b[32m---> \u001b[39m\u001b[32m12\u001b[39m \u001b[38;5;28mprint\u001b[39m(\u001b[43msections\u001b[49m)\n",
      "\u001b[31mNameError\u001b[39m: name 'sections' is not defined"
     ]
    }
   ],
   "source": [
    "downloads_dir = \"/home/jul/DST/recoRAG/data/tmp/download-clps71gk/\"\n",
    "if os.path.isdir(downloads_dir):\n",
    "        dir_list = os.listdir(downloads_dir)\n",
    "        dir_list.sort(key=lambda name: os.path.getmtime(os.path.join(downloads_dir, name)))\n",
    "        for dirname in dir_list:\n",
    "            print(dirname)\n",
    "            dir_path = os.path.join(downloads_dir, dirname)\n",
    "            if glob.glob(os.path.join(dir_path, '*.pdf')):\n",
    "                filepath = glob.glob(os.path.join(dir_path, '*.pdf'))[0]\n",
    "                sections,_,_ = extract_pdf_sections(os.path.join(filepath), 'Bengtsson, Hossjer', dirname)\n",
    "\n",
    "print(sections)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def pdf_main_fontsize(pdf_path, valid_pages):\n",
    "    try:\n",
    "        sizes = [\n",
    "        character.size\n",
    "        for page_layout in extract_pages(pdf_path, page_numbers=valid_pages)\n",
    "        for element in page_layout if isinstance(element, LTTextContainer)\n",
    "        for text_line in element if isinstance(text_line, LTTextLine)\n",
    "        for character in text_line if isinstance(character, LTChar)\n",
    "    ]\n",
    "        size_mode = stats.mode(sizes, keepdims=True)[0][0] if sizes else 0\n",
    "    except Exception:\n",
    "        size_mode = None\n",
    "    return size_mode\n",
    "\n",
    "def extract_pdf_sections(pdf_path, authorlist, paperId, valid_pages=None, possible_section_headings=None, size_threshold=None):\n",
    "    author_last_names = [name.split()[-1] for name in authorlist.split(',') if name.split()]\n",
    "    last_section_names = {'acknowledgment', 'acknowledgement', 'acknowlegment', 'reference'}\n",
    "    month_markers = {'january', 'february', 'march', 'april', 'may', 'june', 'july', 'august', 'september', 'november', 'december',\n",
    "                     'jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'nov', 'dec'}\n",
    "    year_markers = {str(y) for y in range(1900, 2030)}\n",
    "    section_list = possible_section_headings or {\n",
    "        'introduction', 'results', 'discussion', 'conclusions', 'methods', 'materials', 'experimental',\n",
    "        'materials and methods', 'experimental procedure', 'related work', 'i.', 'ii.', 'iii.', 'iv.', 'v.', 'vi.'\n",
    "    }\n",
    "\n",
    "    size_threshold = size_threshold if size_threshold is not None else 0.9\n",
    "\n",
    "    page_width, page_height = 595, 842\n",
    "    nword_abstract_th, nword_sections_th = 30, 30\n",
    "\n",
    "    if valid_pages == 0:\n",
    "        sections = {'paperId': paperId,\n",
    "            'pdf_abstract': None,\n",
    "            'pdf_introduction': None,\n",
    "            'pdf_results': None,\n",
    "            'pdf_discussion': None,\n",
    "            'pdf_methods': None,\n",
    "            'pdf_text': None,\n",
    "            'author_list': authorlist\n",
    "            }\n",
    "        return\n",
    "\n",
    "    #Determine the most common font size (mode)\n",
    "    size_mode = pdf_main_fontsize(pdf_path, valid_pages)\n",
    "\n",
    "    text_blocks, tag_blocks, nwords_in_blocks = [], [], []\n",
    "    section_to_find, section_heading, tag = 'AUTHORS', 'UNDEFINED', 'UNDEFINED'\n",
    "    reached_end = False\n",
    "    if size_mode is not None:\n",
    "        for p, page_layout in enumerate(extract_pages(pdf_path, page_numbers=valid_pages)):\n",
    "            if reached_end:\n",
    "                break\n",
    "            for element in page_layout:\n",
    "                if not isinstance(element, LTTextBox):\n",
    "                    continue\n",
    "                x0, y0, x1, y1 = element.bbox\n",
    "                if not (y0 > 0.05*page_height and y1 < 0.95*page_height and x0 > 0.05*page_width and x1 < 0.95*page_width):\n",
    "                    continue\n",
    "\n",
    "                filtered_text, sizes = [], []\n",
    "                for text_line in element:\n",
    "                    if isinstance(text_line, LTTextLine):\n",
    "                        line_text = [character.get_text() for character in text_line]\n",
    "                        sizes.extend([character.size for character in text_line if isinstance(character, LTChar)])\n",
    "                        line_text = \"\".join(line_text).strip()\n",
    "                        if not line_text.isdigit(): # Keeps lines that are NOT just numbers\n",
    "                            filtered_text.append(line_text)\n",
    "                        filtered_text.append(\"\\n\") # Preserve line breaks\n",
    "                \n",
    "                #joining characters in a single text while removing weird markers generated by pdfminer\n",
    "                filtered_text = re.sub(r'\\(cid:[^\\)]+\\)', '', \"\".join(filtered_text).strip())\n",
    "                word_list = re.split(r'[\\n\\s]+', filtered_text.lower().strip())\n",
    "                nwords = len(word_list)\n",
    "\n",
    "                if any(end_section in ' '.join(word_list[:3]) for end_section in last_section_names):\n",
    "                    reached_end = True\n",
    "                    continue\n",
    "                \n",
    "                #removing everything before the author block as well as the correspondance fields\n",
    "                if p <= 1:\n",
    "                    nauthors_detected = sum(lastname.lower() in ' '.join(word_list) for lastname in author_last_names)\n",
    "                    if nauthors_detected >= 0.5 * len(author_last_names) and y0 > 0.3 * page_height \\\n",
    "                    and section_to_find == 'AUTHORS':\n",
    "                        text_blocks, tag_blocks, nwords_in_blocks = [], [], []\n",
    "                        section_to_find = 'ABSTRACT'\n",
    "                        filtered_text = []\n",
    "                    if nauthors_detected > 0 and ('@' in filtered_text or \"correspond\" in filtered_text):\n",
    "                        filtered_text = []\n",
    "\n",
    "                #removing blocks likely headers with publication date\n",
    "                if any([m in word_list for m in month_markers]) and any([y in word_list for y in year_markers]) and nwords < 10:\n",
    "                    continue\n",
    "\n",
    "                #removing figure captions\n",
    "                if any([figname in word_list[0] for figname in ['fig', 'figure', 'table', 'image']]):\n",
    "                    continue\n",
    "\n",
    "                #removing previous block if likely a header but not followed by capitalized paragraph\n",
    "                if filtered_text and not filtered_text[0].isupper() and nwords_in_blocks and nwords_in_blocks[-1] <= 3:\n",
    "                    text_blocks.pop()\n",
    "                    tag_blocks.pop()\n",
    "                    nwords_in_blocks.pop()\n",
    "                elif (filtered_text and filtered_text.strip() and filtered_text.strip()[0].isupper() and nwords_in_blocks and \n",
    "                    nwords_in_blocks[-1] <= 3 and text_blocks and any(h in re.sub(r'[\\n\\s]+', ' ', text_blocks[-1].lower()) for h in section_list)):\n",
    "                    section_heading = ''.join([w.upper() for w in re.sub(r'[\\d.]', '', text_blocks[-1])])\n",
    "                    if nwords > nword_sections_th:\n",
    "                        tag_blocks[-1] = section_heading\n",
    "\n",
    "                if not reached_end and filtered_text and (max(sizes, default=0) >= size_threshold * size_mode or nwords > 50):\n",
    "                    if section_to_find == 'ABSTRACT' and nwords > nword_abstract_th:\n",
    "                        tag = 'ABSTRACT'\n",
    "                        if word_list[-1][-1] == '.':\n",
    "                            section_to_find = 'INTRODUCTION'\n",
    "                    elif section_to_find == 'INTRODUCTION':\n",
    "                        if nwords > nword_sections_th:\n",
    "                            tag = 'INTRODUCTION'\n",
    "                            section_heading, section_to_find = 'INTRODUCTION', 'NEXTHEADING'\n",
    "                        else:\n",
    "                            tag = 'UNDEFINED'\n",
    "                    elif section_to_find == 'NEXTHEADING' and nwords > nword_sections_th:\n",
    "                        tag = section_heading\n",
    "                    \n",
    "                    text_blocks.append(filtered_text)\n",
    "                    tag_blocks.append(tag)\n",
    "                    nwords_in_blocks.append(nwords)\n",
    "    else:\n",
    "        pdfminer_logger.warning(f\"PDF {pdf_path} likely corrupted\")\n",
    "\n",
    "    sections = {'paperId': paperId,\n",
    "            'pdf_abstract': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'abstract'})),\n",
    "            'pdf_introduction': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'introduction', 'related work', 'i.', 'ii.'})),\n",
    "            'pdf_results': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'results', 'experiment', 'i.', 'ii.'})),\n",
    "            'pdf_discussion': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'discussion', 'conclusion', 'v.', 'vi.'})),\n",
    "            'pdf_methods': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if any(s in t.lower() for s in {'methods', 'materials', 'experimental', 'materials and methods', 'experimental procedure'})),\n",
    "            'pdf_text': '\\n'.join(text for t, text in zip(tag_blocks, text_blocks) if not any(s in t.lower() for s in {'undefined'})),\n",
    "            'author_list': authorlist\n",
    "            }\n",
    "    \n",
    "    return sections, text_blocks, tag_blocks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'list' object has no attribute 'intersection'",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mAttributeError\u001b[39m                            Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[23]\u001b[39m\u001b[32m, line 1\u001b[39m\n\u001b[32m----> \u001b[39m\u001b[32m1\u001b[39m \u001b[43m[\u001b[49m\u001b[33;43m'\u001b[39;49m\u001b[33;43ma\u001b[39;49m\u001b[33;43m'\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[33;43m'\u001b[39;49m\u001b[33;43mb\u001b[39;49m\u001b[33;43m'\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[33;43m'\u001b[39;49m\u001b[33;43mc\u001b[39;49m\u001b[33;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m.\u001b[49m\u001b[43mintersection\u001b[49m([\u001b[33m'\u001b[39m\u001b[33mb\u001b[39m\u001b[33m'\u001b[39m, \u001b[33m'\u001b[39m\u001b[33mc\u001b[39m\u001b[33m'\u001b[39m, \u001b[33m'\u001b[39m\u001b[33md\u001b[39m\u001b[33m'\u001b[39m, \u001b[33m'\u001b[39m\u001b[33me\u001b[39m\u001b[33m'\u001b[39m])\n",
      "\u001b[31mAttributeError\u001b[39m: 'list' object has no attribute 'intersection'"
     ]
    }
   ],
   "source": [
    "['a', 'b', 'c'].intersection(['b', 'c', 'd', 'e'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# sudo apt-get update\n",
    "# sudo apt-get install chromium-driver\n",
    "\n",
    "def download_PDFs(txtdata):\n",
    "    global driver\n",
    "    #you'll need to have ChromeDriver installed\n",
    "    downloads_dir = \"/home/jul/DST/recoRAG/data/tmp\"\n",
    "    \n",
    "    # service = Service(executable_path=\"/usr/bin/chromedriver\")\n",
    "    # run_headless = False #True #\n",
    "\n",
    "    timeout_enddw = 10\n",
    "    timeout_startdw = 5\n",
    "    timeout_loadpage = 15 #timeout_startdw + timeout_enddw\n",
    "    block_markers = {\"captcha\", \"verify you are human\", \"not a robot\", \"forbidden\", \"error: 500\", \"this site can’t be reached\"}\n",
    "    error_markers = {'just a moment', 'privacy error', 'validate user', 'error 500'}\n",
    "\n",
    "    # options = webdriver.ChromeOptions()\n",
    "    # user_data_dir = tempfile.mkdtemp(prefix=\"chrome-profile-\", dir=\"/tmp\")\n",
    "    # options.add_argument(f\"--user-data-dir={user_data_dir}\")\n",
    "    # options.add_argument(\"--no-sandbox\")\n",
    "    # # options.add_argument(\"--disable-dev-shm-usage\")\n",
    "\n",
    "    # if run_headless:\n",
    "    #     options.add_argument(\"--headless=new\")\n",
    "    #     options.add_argument(\"--disable-blink-features=AutomationControlled\")\n",
    "    # prefs = {\n",
    "    #     \"plugins.always_open_pdf_externally\": True,\n",
    "    #     \"download.prompt_for_download\": False,\n",
    "    #     \"download.default_directory\": downloads_dir,\n",
    "    #     \"download.directory_upgrade\": True\n",
    "    # }\n",
    "    # options.add_experimental_option(\"prefs\", prefs)\n",
    "\n",
    "    # waitforSession = 0\n",
    "    # max_retries = 10\n",
    "    # retry_delay = 3\n",
    "    # while waitforSession < max_retries:\n",
    "    #     try:\n",
    "    #         driver = webdriver.Chrome(service=service, options=options)\n",
    "    #         if run_headless:\n",
    "    #             driver.execute_script(\"Object.defineProperty(navigator, 'webdriver', {get: () => undefined})\")\n",
    "    #         break\n",
    "    #     except Exception as e:\n",
    "    #         print(f\"Session creation failed. Retrying ({waitforSession+1}/{max_retries})...\")\n",
    "    #         time.sleep(retry_delay)  # Wait before retrying\n",
    "    #         waitforSession += 1\n",
    "    # else:\n",
    "    #     raise RuntimeError(f\"Failed to create a Selenium session after multiple attempts. Error: {e}\")\n",
    "\n",
    "    text_pdfs = []\n",
    "    paperIds = []\n",
    "    # fraction_print = 0\n",
    "    # processed_list = []\n",
    "    extensions = ('*.pdf', '*.crdownload')\n",
    "    for k, paper in enumerate(txtdata):\n",
    "        try:\n",
    "            paper_download_dir = os.path.join(downloads_dir, paper['paperId'])\n",
    "            os.makedirs(paper_download_dir, exist_ok=True)\n",
    "            paperIds.append(paper['paperId'])\n",
    "\n",
    "            # Set Chrome's download directory dynamically\n",
    "            driver.execute_cdp_cmd(\n",
    "                    \"Page.setDownloadBehavior\", \n",
    "                    {\"behavior\": \"allow\",\n",
    "                    \"downloadPath\": paper_download_dir}\n",
    "                    )\n",
    "            clean_url = re.sub(r'^http://', 'https://', paper['openAccessPdf'])\n",
    "            filename = None\n",
    "            driver.set_page_load_timeout(timeout_loadpage)\n",
    "            driver.get(clean_url)\n",
    "            end_time_enddw = time.time() + timeout_enddw\n",
    "            end_time_startdw = time.time() + timeout_startdw\n",
    "\n",
    "            pdf_status = \"download too long\"\n",
    "            \n",
    "            if any(marker.lower() in driver.page_source.lower() for marker in block_markers) or \\\n",
    "                'crawlprevention' in driver.current_url or \"europepmc.org/backend\" in driver.current_url or \\\n",
    "                any(marker.lower() in driver.title.lower() for marker in error_markers):\n",
    "                    pdf_status = driver.current_url\n",
    "            else:\n",
    "                while time.time() < end_time_startdw:\n",
    "                    filename = [fname for ext in extensions for fname in glob.glob(os.path.join(paper_download_dir, ext))]\n",
    "                    if filename:\n",
    "                        pdf_status = 'downloading'\n",
    "                        break\n",
    "                    else:\n",
    "                        time.sleep(0.2)\n",
    "\n",
    "            # while time.time() < end_time_enddw:\n",
    "            #     filename = [fname for ext in extensions for fname in glob.glob(os.path.join(paper_download_dir, ext))]\n",
    "            #     if filename:\n",
    "            #         # processed_list = [id for id in processed_list if not any(id in fname for fname in filename)]\n",
    "            #         if len(filename) > 1:\n",
    "            #             raise ImportError('more than one download to process: ', filename)\n",
    "            #         filename = filename[0]\n",
    "            #         break\n",
    "            #     if any(marker.lower() in driver.page_source.lower() for marker in block_markers):\n",
    "            #         filename = None\n",
    "            #         break\n",
    "            #     if time.time() > end_time_startdw and not glob.glob(os.path.join(paper_download_dir, '*.crdownload')):\n",
    "            #         filename = None\n",
    "            #         break\n",
    "            # else:\n",
    "            #     filename = None\n",
    "            \n",
    "        except Exception:\n",
    "            pdf_status = \"broken link\"\n",
    "            \n",
    "        pdf_data = {'paperId': txtdata[k]['paperId'], 'author_list': paper['authorName'], 'pdf_text': None, 'pdf_abstract': None, 'pdf_introduction': None, \n",
    "                        'pdf_discussion': None, 'pdf_results': None, 'pdf_methods': None, 'pdf_status': pdf_status, 'openAccessPdf': paper['openAccessPdf'], 'valid_pages': None}\n",
    "        text_pdfs.append(pdf_data)\n",
    "\n",
    "    while download_inprogress(downloads_dir, paperIds):\n",
    "        time.sleep(0.2)\n",
    "        \n",
    "    # driver.quit()\n",
    "    # try:\n",
    "    #     os.rmdir(user_data_dir)\n",
    "    # except Exception:\n",
    "    #     pass\n",
    "\n",
    "    return text_pdfs\n",
    "\n",
    "def download_inprogress(all_downloads_dir, dir_list):\n",
    "    inprogress = False\n",
    "    if os.path.isdir(all_downloads_dir):\n",
    "        for dirname in dir_list:\n",
    "            dir_path = os.path.join(all_downloads_dir, dirname)\n",
    "            if os.path.isdir(dir_path):\n",
    "                if glob.glob(os.path.join(dir_path, '*.crdownload')):\n",
    "                    inprogress = True\n",
    "                    break\n",
    "    return inprogress\n",
    "                \n",
    "def get_valid_PDFs(text_pdfs, all_downloads_dir, timeout=None, num_processes=None):\n",
    "    #Extracting pdfs from downloads that took too long to be processed immediately\n",
    "    # downloads_dir = \"/tmp/s2tmp\"\n",
    "    num_processes = 1 if num_processes is None else num_processes\n",
    "    timeout = 0.1 if timeout is None else timeout\n",
    "    if os.path.isdir(all_downloads_dir):\n",
    "        dir_list = os.listdir(all_downloads_dir)\n",
    "        dir_list.sort(key=lambda name: os.path.getmtime(os.path.join(all_downloads_dir, name)))\n",
    "        for dirname in dir_list:\n",
    "            dir_path = os.path.join(all_downloads_dir, dirname)\n",
    "            if os.path.isdir(dir_path):\n",
    "                if glob.glob(os.path.join(dir_path, '*.pdf')):\n",
    "                    filepath = glob.glob(os.path.join(dir_path, '*.pdf'))[0]\n",
    "                    paper_data = next((pdf_data for pdf_data in text_pdfs if pdf_data['paperId'] == dirname), None)\n",
    "                    if paper_data:\n",
    "                        valid_pages = get_valid_pages(filepath, timeout=timeout, num_processes=num_processes)\n",
    "                        paper_data['valid_pages'] = valid_pages\n",
    "    return text_pdfs\n",
    "\n",
    "def get_unprocessed_PDFs():\n",
    "    all_downloads_dir = \"/home/jul/DST/recoRAG/data/tmp\"\n",
    "    unprocessed_PDFs = []\n",
    "    if os.path.isdir(all_downloads_dir):\n",
    "        dir_list = os.listdir(all_downloads_dir)\n",
    "        for dirname in dir_list:\n",
    "            try:\n",
    "                dir_path = os.path.join(all_downloads_dir, dirname)\n",
    "                if os.path.isdir(dir_path):\n",
    "                    if glob.glob(os.path.join(dir_path, '*.pdf')):\n",
    "                        unprocessed_PDFs.append(dirname)\n",
    "            except Exception:\n",
    "                pass\n",
    "                    \n",
    "    return unprocessed_PDFs\n",
    "\n",
    "def gather_results(minibatches):\n",
    "    downloads_dir = \"/home/jul/DST/recoRAG/data/tmp\"\n",
    "    final_results = []\n",
    "    for b in minibatches:\n",
    "        mini_results = []\n",
    "        for paper in b:\n",
    "            dic_path = os.path.join(downloads_dir, paper['paperId'] + '.pkl')\n",
    "            if os.path.isfile(dic_path):\n",
    "                try:\n",
    "                    with open(dic_path, 'rb') as f:\n",
    "                        mini_results.append(pickle.load(f))\n",
    "                    os.remove(dic_path)\n",
    "                except Exception as e:\n",
    "                    print(e)\n",
    "                \n",
    "        final_results.append(mini_results)\n",
    "    return final_results\n",
    "    \n",
    "def extract_PDFs(textdata):\n",
    "    #Extracting pdfs from downloads that took too long to be processed immediately\n",
    "    # downloads_dir = \"/tmp/s2tmp\"\n",
    "    timeout = 60\n",
    "    downloads_dir = \"/home/jul/DST/recoRAG/data/tmp\"\n",
    "    # dir_path_orig = [os.path.join(downloads_dir, pdf_data['paperId']) for pdf_data in textdata]\n",
    "    dir_list = [{'id': pdf_data['paperId'], 'time': None} for pdf_data in textdata]\n",
    "    while dir_list:\n",
    "        # if os.path.isdir(downloads_dir):\n",
    "        #     dir_list = os.listdir(downloads_dir) if dir_list is None else dir_list\n",
    "        # dir_list.sort(key=lambda name: os.path.getmtime(os.path.join(downloads_dir, name)))\n",
    "        for dirname in dir_list:\n",
    "            dir_path = os.path.join(downloads_dir, dirname['id'])\n",
    "            if os.path.isdir(dir_path):\n",
    "                if not dirname['time']:\n",
    "                    dirname['time'] = os.path.getmtime(dir_path)\n",
    "                if glob.glob(os.path.join(dir_path, '*.pdf')):\n",
    "                    filepath = glob.glob(os.path.join(dir_path, '*.pdf'))[0]\n",
    "                    paper_data = next((pdf_data for pdf_data in textdata if pdf_data['paperId'] == dirname['id']), None)\n",
    "                    if paper_data:\n",
    "                        if 'valid_pages' not in paper_data.keys():\n",
    "                            paper_data['valid_pages'] = None\n",
    "                        pdf_data,_,_ = extract_pdf_sections(filepath, paper_data['authorName'], paper_data['paperId'], paper_data['valid_pages'])\n",
    "                        for key in pdf_data.keys():\n",
    "                            paper_data[key] = pdf_data[key]\n",
    "                        paper_data['pdf_status'] = \"extracted\"\n",
    "\n",
    "                        dic_path = os.path.join(downloads_dir, dirname['id'] + '.pkl')\n",
    "                        with open(dic_path, 'wb') as f:\n",
    "                            pickle.dump(paper_data, f, protocol=pickle.HIGHEST_PROTOCOL)\n",
    "\n",
    "                    dir_list.remove(dirname)\n",
    "                    shutil.rmtree(dir_path)\n",
    "                elif time.time() - dirname['time'] > timeout:\n",
    "                    crfilepath = glob.glob(os.path.join(dir_path, '*.crdownload'))\n",
    "                    if not crfilepath or time.time() - os.path.getmtime(crfilepath[0]) > timeout:\n",
    "                        dir_list.remove(dirname)\n",
    "                        shutil.rmtree(dir_path)\n",
    "\n",
    "    # for dir_path in dir_path_orig:\n",
    "    #     if os.path.isdir(dir_path):\n",
    "    #         shutil.rmtree(dir_path)\n",
    "\n",
    "    # if not glob.glob(os.path.join(downloads_dir, '*')):\n",
    "    #     shutil.rmtree(downloads_dir)\n",
    "\n",
    "    return textdata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# def close_all_chrome_sessions():\n",
    "#     nkill = 0\n",
    "#     for proc in psutil.process_iter(['pid', 'name', 'cmdline']):\n",
    "#         if proc.info['name'] and ('chrome' in proc.info['name'].lower() or \n",
    "#                                   'chromium' in proc.info['name'].lower()):\n",
    "#             try:\n",
    "#                 proc.kill()\n",
    "#                 nkill += 1\n",
    "#             except Exception as e:\n",
    "#                 print(f\"Could not close process {proc.info['pid']}: {e}\")\n",
    "#     print(f\"Closed {nkill} Chrome processes\")\n",
    "\n",
    "def download_worker_init():\n",
    "    global driver, user_data_dir\n",
    "\n",
    "    worker_id = os.getpid()\n",
    "    user_data_dir = tempfile.mkdtemp(prefix=f\"chrome-profile-{worker_id}\", dir=\"/tmp\")\n",
    "\n",
    "    downloads_dir = \"/home/jul/DST/recoRAG/data/tmp\"\n",
    "    \n",
    "    service = Service(executable_path=\"/usr/bin/chromedriver\")\n",
    "    run_headless = False #True #\n",
    "\n",
    "    options = webdriver.ChromeOptions()\n",
    "    options.add_argument(f\"--user-data-dir={user_data_dir}\")\n",
    "    options.add_argument(\"--no-sandbox\")\n",
    "    options.add_argument(\"--window-size=400,300\")\n",
    "    options.add_argument(\"--window-position=10,10\")\n",
    "    # options.add_argument(\"--disable-dev-shm-usage\")\n",
    "\n",
    "    if run_headless:\n",
    "        options.add_argument(\"--headless=new\")\n",
    "        # options.add_argument(\"--disable-blink-features=AutomationControlled\")\n",
    "    prefs = {\n",
    "        \"plugins.always_open_pdf_externally\": True,\n",
    "        \"download.prompt_for_download\": False,\n",
    "        \"download.default_directory\": downloads_dir,\n",
    "        \"download.directory_upgrade\": True\n",
    "    }\n",
    "    options.add_experimental_option(\"prefs\", prefs)\n",
    "\n",
    "    waitforSession = 0\n",
    "    max_retries = 10\n",
    "    retry_delay = 3\n",
    "    while waitforSession < max_retries:\n",
    "        try:\n",
    "            driver = webdriver.Chrome(service=service, options=options)\n",
    "            # if run_headless:\n",
    "            #     driver.execute_script(\"Object.defineProperty(navigator, 'webdriver', {get: () => undefined})\")\n",
    "            break\n",
    "        except Exception:\n",
    "            print(f\"Session creation failed. Retrying ({waitforSession+1}/{max_retries})...\")\n",
    "            time.sleep(retry_delay)  # Wait before retrying\n",
    "            waitforSession += 1\n",
    "    else:\n",
    "        raise RuntimeError(\"Failed to create a Selenium session after multiple attempts\")\n",
    "\n",
    "    time.sleep(0.5)\n",
    "    atexit.register(download_worker_close)\n",
    "\n",
    "def download_worker_close():\n",
    "    global driver, user_data_dir\n",
    "    worker_id = os.getpid()\n",
    "    \n",
    "    driver.quit()\n",
    "    print(f\"Worker {worker_id} WebDriver closed.\")\n",
    "    \n",
    "    shutil.rmtree(user_data_dir, ignore_errors=True)\n",
    "    print(f\"Worker {worker_id} data dir {user_data_dir} deleted.\")\n",
    "\n",
    "def download_and_extract_PDFs(path_to_data, dataset_source=None, years=None, n_jobs=None, filters=None,\n",
    "                              timeout_per_article=None):\n",
    "    dataset_source = 'semanticsscholar' if dataset_source is None else dataset_source\n",
    "    years = [datetime.now().year] if years is None else years\n",
    "    years = [years] if not isinstance(years, list) else years\n",
    "\n",
    "    output_dir = os.path.join(path_to_data, 'pdfdata')\n",
    "    os.makedirs(output_dir, exist_ok=True)\n",
    "\n",
    "    #Cleaning-up tmp dir\n",
    "    for dirname in os.listdir(\"/tmp\"):\n",
    "        if \"chrome-profile-\" in dirname:\n",
    "            shutil.rmtree(os.path.join(\"/tmp\", dirname))\n",
    "    \n",
    "    downloads_dir = \"/home/jul/DST/recoRAG/data/tmp\"\n",
    "    os.makedirs(downloads_dir, exist_ok=True)\n",
    "    \n",
    "\n",
    "    # filters = [['abstract','==','None']]\n",
    "    filters = [filters] if filters is not None and not isinstance(filters[0], list) else filters\n",
    "\n",
    "    n_jobs = multiprocessing.cpu_count() if n_jobs is None else n_jobs\n",
    "    n_jobs_dwload, n_jobs_extract = (n_jobs // 2, n_jobs // 2) if not isinstance(n_jobs, list) else tuple(n_jobs)\n",
    "    n_jobs_total = n_jobs_dwload + n_jobs_extract\n",
    "\n",
    "    timeout_per_article = 0.4 if timeout_per_article is None else timeout_per_article\n",
    "    timeout_per_worker = timeout_per_article * sum((n_jobs_dwload, n_jobs_extract))\n",
    "\n",
    "    ops = {\n",
    "            \"==\": operator.eq,\n",
    "            \"!=\": operator.ne,\n",
    "            \">\": operator.gt,\n",
    "            \"<\": operator.lt,\n",
    "            \">=\": operator.ge,\n",
    "            \"<=\": operator.le,\n",
    "            \"isna\": pd.isna,\n",
    "            \"notna\": pd.notna\n",
    "        }\n",
    "\n",
    "    for year in years:\n",
    "        print(f\"Downloading pdfs for year {year}\")\n",
    "        metadata, textdata = load_dataset(path_to_data, dataset_source, years=year, data_types=['metadata', 'pdf'])\n",
    "        filtered_indices = textdata.index\n",
    "        if filters:\n",
    "            for col, op, val in filters:\n",
    "                if col in textdata.columns:\n",
    "                    if op in [\"isna\", \"notna\"]:\n",
    "                        idx_list = textdata[ops[op](textdata[col])].index\n",
    "                    else:\n",
    "                        idx_list = textdata[ops[op](textdata[col], val)].index\n",
    "\n",
    "                    filtered_indices = filtered_indices.intersection(idx_list)\n",
    "\n",
    "        data4pdf = pd.merge(textdata[['paperId','openAccessPdf']].loc[filtered_indices], metadata[['paperId','authorName']].loc[filtered_indices], on='paperId', how='inner')\n",
    "        if len(data4pdf) < len(filtered_indices):\n",
    "            raise ValueError(\"List of metadata and text data are notmatching the same articles\")\n",
    "        \n",
    "        # data4pdf = data4pdf.iloc[:400]\n",
    "        minibatch_size = 20 #100\n",
    "        batch_size = 3000 #len(data4pdf) // 10 #2 * n_jobs * minibatch_size\n",
    "        Npapers = 3000 #len(data4pdf)\n",
    "        with tqdm(range(Npapers // batch_size + 1), desc=f'Year {year} / {len(years)} years ({Npapers} papers on that year)', unit='batch') as pbar:\n",
    "            try:\n",
    "                for i in pbar:\n",
    "                    shutil.rmtree(downloads_dir)\n",
    "                    os.makedirs(downloads_dir, exist_ok=True)\n",
    "                    close_all_chrome_sessions()\n",
    "                    \n",
    "                    TimoutRaised = False\n",
    "                    batch = data4pdf.iloc[i*batch_size:min(Npapers, (i+1)*batch_size)]\n",
    "                    Nbatch = len(batch)\n",
    "                    minibatches = [batch.iloc[k*minibatch_size:min(Nbatch, (k+1)*minibatch_size)].to_dict(orient='records') for k in range(Nbatch // minibatch_size + 1)]\n",
    "                    minibatches = [b for b in minibatches if len(b) > 0 and b[0] is not None]\n",
    "                    final_results, dwload_results = [], []\n",
    "                    Nminibatches = len(minibatches)\n",
    "                    if minibatches:\n",
    "                        # with multiprocessing.Pool(processes=n_jobs) as pool:\n",
    "                        #     results = list(pool.map(download_PDFs, minibatches))\n",
    "                        with multiprocessing.Pool(processes=n_jobs_dwload, initializer=download_worker_init) as download_pool, \\\n",
    "                             multiprocessing.Pool(processes= n_jobs_extract) as extract_pool:\n",
    "\n",
    "                            async_dwload = [download_pool.apply_async(download_PDFs, (b,)) for b in minibatches]  \n",
    "                            async_extract = [extract_pool.apply_async(extract_PDFs, (b,)) for b in minibatches]\n",
    "                            \n",
    "                            pending_extract = set(async_extract)\n",
    "                            pending_dwload = set(async_dwload)\n",
    "                            end_time = time.time() + timeout_per_article*batch_size\n",
    "                            old_pending_articles = batch_size\n",
    "                            while pending_extract or pending_dwload:\n",
    "                                #Adjust timeout wall to one many tasks of extraction are still pending still download is done\n",
    "                                if time.time() > end_time:\n",
    "                                    print(f\"{len(pending_extract)} extractions and {len(pending_dwload)} downloads out of {Nminibatches} timed out. Terminating the pool.\")\n",
    "                                    TimoutRaised = True\n",
    "                                    extract_pool.terminate()\n",
    "                                    extract_pool.join()\n",
    "                                    download_pool.terminate()\n",
    "                                    download_pool.join()\n",
    "                                    break\n",
    "\n",
    "                                for task in list(pending_extract):\n",
    "                                    if task.ready():\n",
    "                                        res = task.get()\n",
    "                                        final_results.append(res)\n",
    "                                        pending_extract.remove(task)\n",
    "\n",
    "                                for task in list(pending_dwload):\n",
    "                                    if task.ready():\n",
    "                                        res_dw = task.get()\n",
    "                                        dwload_results.append(res_dw)\n",
    "                                        pending_dwload.remove(task)\n",
    "\n",
    "                                if pending_extract or pending_dwload:\n",
    "                                    if pending_dwload:\n",
    "                                        pbar.set_postfix({'download tasks': f\"{len(pending_dwload)} left\",\n",
    "                                                        'extraction tasks': f\"{len(pending_extract)} left\"})\n",
    "                                    time.sleep(1)\n",
    "\n",
    "                                if not pending_dwload:\n",
    "                                    n_pending_articles = len(get_unprocessed_PDFs())\n",
    "                                    if n_pending_articles != old_pending_articles:\n",
    "                                        extratime = timeout_per_worker / 2 * n_pending_articles / len(pending_extract)\n",
    "                                        pbar.set_postfix({'extraction tasks': f\"{len(pending_extract)} still running, \\\n",
    "                                                          {extratime} seconds left\"})\n",
    "                                        end_time = time.time() + extratime\n",
    "                                        old_pending_articles = n_pending_articles\n",
    "                                    else:\n",
    "                                        end_time = time.time()\n",
    "                        \n",
    "                        pbar.set_postfix({'status': \"gathering results...\"})\n",
    "                        final_results = gather_results(minibatches)\n",
    "                        \n",
    "                        if TimoutRaised:\n",
    "                            #Checking valid pages to avoid pdfminer getting stuck\n",
    "                            #This will change results in place\n",
    "                            n_pending_articles = len(get_unprocessed_PDFs())\n",
    "                            pbar.set_postfix({'status': f\"checking valid pages for {n_pending_articles} articles that timed out...\"})\n",
    "                            get_valid_PDFs([pdf for r in minibatches for pdf in r], downloads_dir, timeout=1)\n",
    "                            lo_ids = get_unprocessed_PDFs()\n",
    "                            lo_papers = [paper for b in minibatches for paper in b if paper['paperId'] in lo_ids]\n",
    "                            N_lo = len(lo_papers)\n",
    "                            minib_size = math.ceil(len(lo_papers) / (n_jobs_total + 1))\n",
    "                            lo_minibatches = [lo_papers[k*minib_size:(k+1)*minib_size] for k in range(n_jobs_total + 1)]\n",
    "                            lo_minibatches = [b for b in lo_minibatches if len(b) > 0 and b[0] is not None]\n",
    "                            Nlo_minibatches = len(lo_minibatches)\n",
    "                            pbar.set_postfix({'status': f\"will process now {(N_lo)} articles...\"})\n",
    "                            print()\n",
    "                            with multiprocessing.Pool(processes=n_jobs_total) as pool:\n",
    "                                async_extract = [pool.apply_async(extract_PDFs, (b,)) for b in lo_minibatches]\n",
    "                                pending = set(async_extract)\n",
    "                                end_time = time.time() + timeout_per_article*N_lo\n",
    "                                while pending:\n",
    "                                    if time.time() > end_time:\n",
    "                                        print(f\"{len(pending)} tasks out of {Nlo_minibatches} timed out after checking valid pdf pages.\")\n",
    "                                        TimoutRaised = True\n",
    "                                        pool.terminate()\n",
    "                                        pool.join()\n",
    "                                        break\n",
    "\n",
    "                                    for task in list(pending):\n",
    "                                        if task.ready():\n",
    "                                            res = task.get()\n",
    "                                            final_results.append(res)\n",
    "                                            pending.remove(task)\n",
    "                                    if pending:\n",
    "                                        time.sleep(0.5)\n",
    "\n",
    "                        if final_results:                \n",
    "                            pdfdata = pd.DataFrame([pdf for r in final_results for pdf in r])\n",
    "                            pbar.set_postfix({'status': f\"{len(pdfdata)} articles extracted out of {batch_size}\"})\n",
    "                            if dwload_results:\n",
    "                                dwdata = pd.DataFrame([{k: d.get(k, None) for k in ['paperId', 'pdf_status']}\n",
    "                                                    for r in dwload_results if r is not None \n",
    "                                                    for d in r if d is not None])\n",
    "                                pdfdata = dwdata.set_index(\"paperId\").combine_first(pdfdata.set_index(\"paperId\")).reset_index(drop=False)\n",
    "                            textdata = pdfdata.set_index(\"paperId\").combine_first(textdata.set_index(\"paperId\")).reset_index(drop=False)\n",
    "                            mask = textdata['abstract'] == 'None'\n",
    "                            textdata.loc[mask, 'abstract'] = textdata.loc[mask, 'pdf_abstract']\n",
    "                            textdata['authorName'] = metadata['authorName']\n",
    "\n",
    "                # Cleaning up download directory\n",
    "                shutil.rmtree(downloads_dir)\n",
    "                os.makedirs(downloads_dir, exist_ok=True)\n",
    "                close_all_chrome_sessions()\n",
    "                            \n",
    "            except Exception:\n",
    "                pbar.close()\n",
    "                raise\n",
    "\n",
    "\n",
    "        output_dir_year = os.path.join(output_dir, str(year))\n",
    "        os.makedirs(output_dir_year, exist_ok=True)\n",
    "        filepath = os.path.join(output_dir_year, f'{dataset_source}_textpdf_{year}.parquet')\n",
    "\n",
    "        # textdata.to_parquet(filepath, engine=\"pyarrow\", compression=\"snappy\", index=True)\n",
    "\n",
    "    return textdata, pdfdata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sending incremental file list\n",
      "2005/\n",
      "2005/semanticsscholar_metadata_2005.parquet\n",
      "\n",
      "sent 162,843,845 bytes  received 59 bytes  36,187,534.22 bytes/sec\n",
      "total size is 8,750,562,247  speedup is 53.74\n",
      "Synced /home/jul/DST/asXiv/data/metadata -> /mnt/g/My Drive/DST/MLOps_projects/asXiv/data/metadata (rsync complete, Google Drive sync in progress...)\n"
     ]
    }
   ],
   "source": [
    "push_data_to_drive('metadata')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Downloading pdfs for year 2006\n",
      "No pdf data for year 2006. Will try to load text data instead\n",
      "metadata loaded for years: ['2006']; text data loaded for years: ['2006']; pdf data loaded for years: ['2006']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Year 2006 / 1 years (3000 papers on that year):   0%|          | 0/2 [00:00<?, ?batch/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Closed 0 Chrome processes\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Year 2006 / 1 years (3000 papers on that year):   0%|          | 0/2 [11:42<?, ?batch/s, extraction tasks=10 still running,                                                           5.0 seconds left]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10 extractions and 0 downloads out of 150 timed out. Terminating the pool.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Year 2006 / 1 years (3000 papers on that year):   0%|          | 0/2 [11:49<?, ?batch/s, status=will process now 5 articles...]                                                                        "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Year 2006 / 1 years (3000 papers on that year):   0%|          | 0/2 [11:53<?, ?batch/s, status=1713 articles extracted out of 3000]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1 tasks out of 5 timed out after checking valid pdf pages.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Year 2006 / 1 years (3000 papers on that year): 100%|██████████| 2/2 [11:54<00:00, 357.31s/batch, status=1713 articles extracted out of 3000]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Closed 369 Chrome processes\n",
      "Closed 168 Chrome processes\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "textdata, pdfdata = download_and_extract_PDFs(DATA_PATH, years=2006, n_jobs=[20, 30])#, filters=[['abstract','==','None']])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "paperId                0\n",
       "abstract             129\n",
       "authorName             0\n",
       "author_list         1225\n",
       "openAccessPdf          0\n",
       "pdf_abstract        1225\n",
       "pdf_discussion      1225\n",
       "pdf_introduction    1225\n",
       "pdf_methods         1225\n",
       "pdf_results         1225\n",
       "pdf_status             0\n",
       "pdf_text            1225\n",
       "referenceTitles        0\n",
       "title                  0\n",
       "valid_pages         2995\n",
       "dtype: int64"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "textdata.iloc[:3000].isna().sum()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No pdf data for year 2006. Will try to load text data instead\n",
      "metadata loaded for years: ['2006']; text data loaded for years: ['2006']; pdf data loaded for years: ['2006']\n"
     ]
    }
   ],
   "source": [
    "metadata0, textdata0 = load_dataset(DATA_PATH, years=[2006], data_types=['metadata', 'pdf'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "ff0c1b6b-f69c-462e-9fd1-47317d999144",
       "rows": [
        [
         "0",
         "00009f98f5ff962beb322e6c0e78e4b77dd2d968",
         "High exposure to Tunga penetrans (Linnaeus, 1758) correlates with intensity of infestation.",
         "Tungiasis is a parasitic skin disease widespread in resource-poor urban and rural communities in Brazil. Inhabitants of an urban slum in Northeast Brazil were examined for the presence of tungiasis lesions and followed-up twice a week for a period of three weeks. Each time the number, stages, and topographic localization of lesions were recorded on a documentation sheet. The infestation rate (number of newly embedded sand fleas per individual and day) remained stable during the observation period. The infestation rate was significantly related to the intensity of infestation (total number of lesions present) (rho = 0.70, p < 0.0001) and the proportion of viable lesions (rho = 0.28, p < 0.0001). The results indicate that in an endemic area the infestation intensity and the proportion of viable lesions can be used as a proxy to assess the exposure of individuals at risk for tungiasis. Persistently high infestation rates during the transmission season favour the use of prevention measures against invading sand fleas (such as a repellent) rather than a drug to kill already embedded parasites.",
         "",
         "https://www.scielo.br/j/mioc/a/csnpsRLq5FqY5MkQLf8GRjG/?lang=en&format=pdf"
        ],
        [
         "1",
         "0000e1833852eb3271356c493d59e83da0c4c7db",
         "Immunoinformatics Comes of Age",
         "With the burgeoning immunological data in the scientific literature, scientists must increasingly rely on Internet resources to inform and enhance their work. Here we provide a brief overview of the adaptive immune response and summaries of immunoinformatics resources, emphasizing those with Web interfaces. These resources include searchable databases of epitopes and immune-related molecules, and analysis tools for T cell and B cell epitope prediction, vaccine design, and protein structure comparisons. There is an agreeable synergy between the growing collections in immune-related databases and the growing sophistication of analysis software; the databases provide the foundation for developing predictive computational tools, which in turn enable more rapid identification of immune responses to populate the databases. Collectively, these resources contribute to improved understanding of immune responses and escape, and evolution of pathogens under immune pressure. The public health implications are vast, including designing vaccines, understanding autoimmune diseases, and defining the correlates of immune protection.",
         "Prediction of the secondary structure of proteins from their amino acid sequence.;How TCRs bind MHCs, peptides, and coreceptors.;Genetic Analysis of Completely Sequenced Disease-Associated MHC Haplotypes Identifies Shuffling of Segments in Recent Human History;Human Immunodeficiency Virus Type 1 Subtype B Ancestral Envelope Protein Is Functional and Elicits Neutralizing Antibodies in Rabbits Similar to Those Elicited by a Circulating Subtype B Envelope;Pcleavage: an SVM based method for prediction of constitutive proteasome and immunoproteasome cleavage sites in antigenic sequences;Generating quantitative models describing the sequence specificity of biological processes with the stabilized matrix method;Bcipep: A database of B-cell epitopes;Automated generation and evaluation of specific MHC binding predictive tools: ARB matrix applications;Modeling the MHC class I pathway by combining predictions of proteasomal cleavage,TAP transport and MHC class I binding;HIV vaccine development by computer assisted design: the GAIA vaccine.;The role of the proteasome in generating cytotoxic T-cell epitopes: insights obtained from improved predictions of proteasomal cleavage;Sensor molecules in intestinal innate immunity against bacterial infections;The Immune Epitope Database and Analysis Resource: From Vision to Blueprint;Antigenicity and Immunogenicity of a Synthetic Human Immunodeficiency Virus Type 1 Group M Consensus Envelope Glycoprotein;Benchmarking B cell epitope prediction: Underperformance of existing methods;IMGT, the international ImMunoGeneTics information system®;Dominant influence of HLA-B in mediating the potential co-evolution of HIV and HLA;Computational methods for prediction of T-cell epitopes--a framework for modelling, testing, and applications.;Prediction of CTL epitopes using QM, SVM and ANN techniques.;Selection, Transmission, and Reversion of an Antigen-Processing Cytotoxic T-Lymphocyte Escape Mutation in Human Immunodeficiency Virus Type 1 Infection;Improved prediction of MHC class I and class II epitopes using a novel Gibbs sampling approach;Immediate Cytotoxicity But Not Degranulation Distinguishes Effector and Memory Subsets of CD8+ T Cells;Analysis and prediction of affinity of TAP binding peptides using cascade SVM;Definition of supertypes for HLA molecules using clustering of specificity matrices;Immune recognition of a human renal cancer antigen through post-translational protein splicing;Sensitive quantitative predictions of peptide-MHC binding by a 'Query by Committee' artificial neural network approach.;Mapping cross-clade HIV-1 vaccine epitopes using a bioinformatics approach.;Identifying MHC Class I Epitopes by Predicting the TAP Transport Efficiency of Epitope Precursors;Prediction of promiscuous and high-affinity mutated MHC binders.;MHCPred: a server for quantitative prediction of peptide-MHC binding;JenPep: A Novel Computational Information Resource for Immunobiology and Vaccinology;ProPred1: Prediction of Promiscuous MHC Class-I Binding Sites;MHCBN: a comprehensive database of MHC binding and non-binding peptides;Diversity Considerations in HIV-1 Vaccine Selection;Large‐scale computational identification of HIV T‐cell epitopes;The MHC haplotype project: a resource for HLA-linked association studies.;Development of a DNA-MVA/HIVA vaccine for Kenya.;Prediction of proteasome cleavage motifs by neural networks.;ProPred: prediction of HLA-DR binding sites;From genome to vaccine: in silico predictions, ex vivo verification.;Crystal Structure of a Neutralizing Human IgG Against HIV-1: A Template for Vaccine Design;Functional Evidence That Conserved TCR CDRα3 Loop Docking Governs the Cross-Recognition of Closely Related Peptide:Class I Complexes1;Quantitative predictions of peptide binding to MHC class I molecules using specificity matrices and anchor-stratified calibrations.;A kinetic model of vertebrate 20S proteasome accounting for the generation of major proteolytic fragments from oligomeric peptide substrates.;Sequence signals for generation of antigenic peptides by the proteasome: implications for proteasomal cleavage mechanism.;The HLA FactsBook;Function of the transport complex TAP in cellular immune recognition.;The specificity of proteasomes: impact on MHC class I processing and presentation of antigens;MHC superfamily structure and the immune system.;Distinct proteolytic processes generate the C and N termini of MHC class I-binding peptides.;Nine major HLA class I supertypes account for the vast preponderance of HLA-A and -B polymorphism;SYFPEITHI: database for MHC ligands and peptide motifs;Description and prediction of peptide-MHC binding: the 'human MHC project'.;A theoretical approach towards the identification of cleavage-determining amino acid motifs of the 20 S proteasome.;The importance of pairwise interactions between peptide residues in the delineation of TCR specificity.;Application of an artificial neural network to predict specific class I MHC binding peptide sequences;Relationship between peptide selectivities of human transporters associated with antigen processing and HLA class I molecules.;MHC Ligands and Peptide Motifs;A structure-based algorithm to predict potential binding peptides to MHC molecules with hydrophobic binding pockets.;Two distinct proteolytic processes in the generation of a major histocompatibility complex class I-presented peptide.;Two complementary methods for predicting peptides binding major histocompatibility complex molecules.;Accessing the Kabat antibody sequence database by computer;An epitope-selective, transporter associated with antigen presentation (TAP)-1/2-independent pathway and a more general TAP-1/2-dependent antigen-processing pathway allow recognition of the HIV-1 envelope glycoprotein by CD8+ CTL.;Cytotoxicity mediated by T cells and natural killer cells is greatly impaired in perforin-deficient mice;Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains.;Identification of peptide sequences that potentially trigger HLA‐A2.1‐restricted cytotoxic T lymphocytes;Emerging principles for the recognition of peptide antigens by MHC class I molecules.;Query by committee;Characterization of a conserved T cell epitope in HIV-1 gp41 recognized by vaccine-induced human cytolytic T cells.;A semi‐empirical method for prediction of antigenic determinants on protein antigens;Prediction of major histocompatibility complex binding regions of protein antigens by sequence pattern analysis.;New hydrophilicity scale derived from high-performance liquid chromatography peptide retention data: correlation of predicted surface residues with antigenicity and X-ray-derived accessible sites.;Induction of hepatitis A virus-neutralizing antibody by a virus-specific synthetic peptide;Prediction of chain flexibility in proteins;Polyvalent vaccines for optimal coverage of potential T-cell epitopes in global HIV-1 variants;PLoS Computational Biology | www.ploscompbiol;HIV Molecular Immunology 2005;Los Alamos Hepatitis C Immunology Database;Identification of immunodominant epitopes of Schistosoma mansoni vaccine candidate antigens using human T cells.;IMGT-ONTOLOGY and IMGT databases, tools and Web resources for immunogenetics and immunoinformatics.;MHCPred: bringing a quantitative dimension to the online prediction of MHC binding.;IMGT/HLA and IMGT/MHC: sequence databases for the study of the major histocompatibility complex;MAPPP: MHC class I antigenic peptide processing prediction.;MMDB: Entrez's 3D-structure database;IMGT , the international ImMunoGeneTics information system;Enhanced motif scan: A tool to scan for HLA anchor residues in proteins. HIV Immunology and HIV/SIV Vaccine Databases Publication number LA-UR 04-8162;SACS-Self-maintaining database of antibody crystal structure information;Reliable prediction of T-cell epitopes using neural networks with novel sequence representations;Web-based Tools for Vaccine Design;Methods for Prediction of Peptide Binding to MHC Molecules: A Comparative Study;ELF: An analysis tool for HIV-1 peptides and HLA types;Structure‐based prediction of binding peptides to MHC class I molecules: Application to a broad range of MHC alleles;PeptGen: Designing Peptides for Immunological Studies and Application to HIV Consensus Sequences;Part 5 – HLA-F;Motifscan: A Web-based tool to find HLA anchor residues in proteins or peptides;MMDB: Entrez's 3D structure database;Mechanisms of MHC class I--restricted antigen processing.;MHCPEP, a database of MHC-binding peptides: update 1997;MHCPEP, a database of MHC-binding peptides: update 1996;Peptide binding to MHC class I molecules: Implications for antigenic peptide prediction;An Artificial Neural Network Approach;Publication number LA-UR 06-0036",
         "https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.0020071&type=printable"
        ],
        [
         "2",
         "00016363b9a128e01a17e9b831249967a0fee15f",
         "The Potassium Channel KAT1 Is Activated by Plant and Animal 14-3-3 Proteins*",
         "14-3-3 proteins modulate the plant inward rectifier K+ channel KAT1 heterologously expressed in Xenopus oocytes. Injection of recombinant plant 14-3-3 proteins into oocytes shifted the activation curve of KAT1 by +11 mV and increased the τon. KAT1 was also modulated by 14-3-3 proteins of Xenopus oocytes. Titration of the endogenous 14-3-3 proteins by injection of the peptide Raf 621p resulted in a strong decrease in KAT1 current (∼70% at –150 mV). The mutation K56E performed on plant protein 14-3-3 in a highly conserved recognition site prevented channel activation. Because the maximal conductance of KAT1 was unaffected by 14-3-3, we can exclude that they act by increasing the number of channels, thus ruling out any effect of these proteins on channel trafficking and/or insertion into the oocyte membrane. 14-3-3 proteins also increased KAT1 current in inside-out patches, suggesting a direct interaction with the channel. Direct interaction was confirmed by overlay experiments with radioactive 14-3-3 on oocyte membranes expressing KAT1.",
         "",
         "http://www.jbc.org/article/S0021925820681212/pdf"
        ],
        [
         "3",
         "00034b38a5273b760824f3223fa0d4fdb349d6df",
         "Competitive recruitment of the periplasmic translocation portal TolB by a natively disordered domain of colicin E9",
         "The natively disordered N-terminal 83-aa translocation (T) domain of E group nuclease colicins recruits OmpF to a colicin-receptor complex in the outer membrane (OM) as well as TolB in the periplasm of Escherichia coli, the latter triggering translocation of the toxin across the OM. We have identified the 16-residue TolB binding epitope in the natively disordered T-domain of the nuclease colicin E9 (ColE9) and solved the crystal structure of the complex. ColE9 folds into a distorted hairpin within a canyon of the six-bladed β-propeller of TolB, using two tryptophans to bolt the toxin to the canyon floor and numerous intramolecular hydrogen bonds to stabilize the bound conformation. This mode of binding enables colicin side chains to hydrogen-bond TolB residues in and around the channel that runs through the β-propeller and that constitutes the binding site of peptidoglycan-associated lipoprotein (Pal). Pal is a globular binding partner of TolB, and their association is known to be important for OM integrity. The structure is therefore consistent with translocation models wherein the colicin disrupts the TolB–Pal complex causing local instability of the OM as a prelude to toxin import. Intriguingly, Ca2+ ions, which bind within the β-propeller channel and switch the surface electrostatics from negative to positive, are needed for the negatively charged T-domain to bind TolB with an affinity equivalent to that of Pal and competitively displace it. Our study demonstrates that natively disordered proteins can compete with globular proteins for binding to folded scaffolds but that this can require cofactors such as metal ions to offset unfavorable interactions.",
         "",
         "https://www.pnas.org/doi/pdf/10.1073/pnas.0603433103"
        ],
        [
         "4",
         "00036a3493967932ac256f38c0c2419415f04feb",
         "Polysaccharides from the root of Angelica sinensis protect bone marrow and gastrointestinal tissues against the cytotoxicity of cyclophosphamide in mice",
         "Cyclophosphamide (CY) is a cytostatic agent that produces systemic toxicity especially on cells with high proliferative capacity, while polysaccharides from Angelica sinensis (AP) have been shown to increase the turnover of gastrointestinal mucosal and hemopoietic stem cells. It is not known whether AP has an effect on CY-induced cytotoxicity on bone marrow and gastrointestinal tract. In this study, we assessed the protective actions of AP on CY-induced leukopenia and proliferative arrest in the gastroduodenal mucosa in mice. Subcutaneous injection of CY (200 mg/kg) provoked dramatic decrease in white blood cell (WBC) count and number of blood vessels and proliferating cells in both the gastric and duodenal mucosae. Subcutaneous injection of AP significantly promoted the recovery from leukopenia and increased number of blood vessels and proliferating cells in both the gastric and duodenal tissues. Western blotting revealed that CY significantly down-regulated the protein expression of vascular endothelial growth factor (VEGF), c-Myc and ornithine decarboxylase (ODC) in gastric mucosae but had no effect on epidermal growth factor (EGF) expression. AP also reversed the dampening effect of CY on VEGF expression in the gastric mucosa. These data suggest that AP is a cytoprotective agent which can protect against the cytotoxicity of CY on hematopoietic and gastrointestinal tissues when the polysaccharide is co-administered with CY in cancer patients during treatment regimen.",
         "",
         "https://www.medsci.org/v03p0001.pdf"
        ],
        [
         "5",
         "0003825e87b4ccccdc75f7014d1d6917804adced",
         "Oxalate accumulation and regulation is independent of glycolate oxidase in rice leaves.",
         "Cellular oxalate, widely distributed in many plants, is implicated to play important roles in various functions and is also known to affect food qualities adversely in fruits and vegetables. How oxalate is regulated in plants is currently not well understood. Glycolate oxidase (GLO) has long been considered as an important player in oxalate accumulation in plants. To gain further insight into the biochemical and molecular mechanisms, the possible roles of GLO in the process were studied. Drastically different levels of oxalate could be achieved by treating rice with various nitrogen forms (nitrate versus ammonium). While nitrate stimulated oxalate accumulation, ammonium reduced its level. Such treatments resulted in similar pattern changes for some other related organic acids, such as glycolate, oxaloacetate, and malate. By feeding plants with exogenous glycolate it was possible almost completely to restore the ammonium-decreased oxalate level. Under the two treatments few differences were observed for GLO mRNA levels, protein levels, and in vitro activities. Both K(m) for glycolate/glyoxylate and K(i) for oxalate remained almost the same for GLO purified from either nitrate- or ammonium-fed leaves. A further in vivo study, with transgenic plants carrying an estradiol-inducible GLO antisense gene, showed that, while the estradiol-induced antisense expression remarkably reduced both GLO protein levels and activities, oxalate levels were not significantly altered in the estradiol-treated transgenic plants. Taken together, it is suggested that oxalate accumulation and regulation is independent of GLO in rice leaves.",
         "",
         "https://academic.oup.com/jxb/article-pdf/57/9/1899/1715826/erj131.pdf"
        ],
        [
         "6",
         "000468a0bca628e7d42ebc516606d7a1874e1491",
         "Embedding Graphs Simultaneously with Fixed Edges",
         "None",
         "",
         "https://link.springer.com/content/pdf/10.1007%2F978-3-540-70904-6_12.pdf"
        ],
        [
         "7",
         "0004f9cbd5224cab85d3da4c93ba7013245647b4",
         "Poly‐gamma‐glutamate in bacteria",
         "Poly‐γ‐glutamate (PGA), a natural polymer, is synthesized by several bacteria (all Gram‐positive), one archaea and one eukaryote. PGA has diverse biochemical properties, enabling it to play different roles, depending on the organism and its environment. Indeed, PGA allows bacteria to survive at high salt concentrations and may also be involved in virulence. The minimal gene sets required for PGA synthesis were recently defined. There are currently two nomenclatures depending on the PGA final status: cap, for ‘capsule’, when PGA is surface associated or pgs, for ‘polyglutamate synthase’, when PGA is released. The minimal gene sets contain four genes termed cap or pgs B, C, A and E. The PGA synthesis complex is membrane‐anchored and uses glutamate and ATP as substrates. Schematically, the reaction may be divided into two steps, PGA synthesis and PGA transport through the membrane. PGA synthesis depends primarily on CapB‐CapC (or PgsB‐PgsC), whereas PGA transport requires the presence, or the addition, of CapA‐CapE (or PgsAA‐PgsE). The synthesis complex is probably responsible for the stereochemical specificity of PGA composition. Finally, PGA may be anchored to the bacterial surface or released. An additional enzyme is involved in this reaction: either CapD, a γ‐glutamyl‐transpeptidase that catalyses anchorage of the PGA, or PgsS, a hydrolase that facilitates release. The anchoring of PGA to the bacterial surface is important for virulence. All cap genes are therefore potential targets for inhibitors specifically blocking PGA synthesis or anchorage.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1365-2958.2006.05179.x"
        ],
        [
         "8",
         "0007428e145f375cd67c1a4cef3aa1c5af931e98",
         "Trypanosoma cruzi strains isolated from human, vector, and animal reservoir in the same endemic region in Mexico and typed as T. cruzi I, discrete typing unit 1 exhibit considerable biological diversity.",
         "In this study, three strains of Trypanosoma cruzi were isolated at the same time and in the same endemic region in Mexico from a human patient with chronic chagasic cardiomyopathy (RyC-H); vector (Triatoma barberi) (RyC-V); and rodent reservoir (Peromyscus peromyscus) (RyC-R). The three strains were characterized by multilocus enzyme electrophoresis, random amplified polymorphic DNA, and by pathological profiles in experimental animals (biodemes). Based on the analysis of genetic markers the three parasite strains were typed as belonging to T. cruzi I major group, discrete typing unit 1. The pathological profile of RyC-H and RyC-V strains indicated medium virulence and low mortality and, accordingly, the strains should be considered as belonging to biodeme Type III. On the other hand, the parasites from RyC-R strain induced more severe inflammatory processes and high mortality (> 40%) and were considered as belonging to biodeme Type II. The relationship between genotypes and biological characteristics in T. cruzi strains is still debated and not clearly understood. An expert committee recommended in 1999 that Biodeme Type III would correspond to T. cruzi I group, whereas Biodeme Type II, to T. cruzi II group. Our findings suggest that, at least for Mexican isolates, this correlation does not stand and that biological characteristics such as pathogenicity and virulence could be determined by factors different from those identified in the genotypic characterization.",
         "",
         "https://www.scielo.br/j/mioc/a/hkBS94wNKVZsJqmD8CxYq3t/?lang=en&format=pdf"
        ],
        [
         "9",
         "0007fad20e444cb908ccea0cb97c2e31ea7212c6",
         "Faithful expression of GFP from the PLCbeta2 promoter in a functional class of taste receptor cells.",
         "Phospholipase C-type beta2 (PLCbeta2) is expressed in a subset of cells within mammalian taste buds. This enzyme is involved in the transduction of sweet, bitter, and umami stimuli and thus is believed to be a marker for gustatory sensory receptor cells. We have developed transgenic mice expressing green fluorescent protein (GFP) under the control of the PLCbeta2 promoter to enable one to identify these cells and record their physiological activity in living preparations. Expression of GFP (especially in lines with more than one copy integrated) is strong enough to be detected in intact tissue preparations using epifluorescence microscopy. By immunohistochemistry, we confirmed that the overwhelming majority of cells expressing GFP are those that endogenously express PLCbeta2. Expression of the GFP transgene in circumvallate papillae occurs at about the same time during development as endogenous PLCbeta2 expression. When loaded with a calcium-sensitive dye in situ, GFP-positive taste cells produce typical Ca2+ responses to a taste stimulus, the bitter compound cycloheximide. These PLCbeta2 promoter-GFP transgenic lines promise to be useful for studying taste transduction, sensory signal processing, and taste bud development.",
         "Liposome-mediated transfection of mature taste cells.;PLCbeta2-independent behavioral avoidance of prototypical bitter-tasting ligands.;Morphologic characterization of rat taste receptor cells that express components of the phospholipase C signaling pathway;Electrophysiological Characterization of Voltage-Gated Currents in Defined Taste Cell Types of Mice;Sour Taste Stimuli Evoke Ca2+ and pH Responses in Mouse Taste Cells;Coding of Sweet, Bitter, and Umami Tastes Different Receptor Cells Sharing Similar Signaling Pathways;Individual mouse taste cells respond to multiple chemical stimuli;Taste receptor cell responses to the bitter stimulus denatonium involve Ca2+ influx via store-operated channels.;An amino-acid taste receptor;“Type III” cells of rat taste buds: Immunohistochemical and ultrastructural studies of neuron‐specific enolase, protein gene product 9.5, and serotonin;Bitter taste transduced by PLC-β2-dependent rise in IP3 and α-gustducin-dependent fall in cyclic nucleotides;IP(3) receptor type 3 and PLCbeta2 are co-expressed with taste receptors T1R and T2R in rat taste bud cells.;Mouse taste cells with glialike membrane properties.;G Protein βγ Complexes in Circumvallate Taste Cells Involved in Bitter Transduction;T2Rs Function as Bitter Taste Receptors;A metabotropic glutamate receptor variant functions as a taste receptor;Gγ13 colocalizes with gustducin in taste receptor cells and mediates IP3 responses to bitter denatonium;Directing Gene Expression to Gustducin-Positive Taste Receptor Cells;Identification of a phospholipase C beta subtype in rat taste cells.;Characterization and solubilization of bitter-responsive receptors that couple to gustducin.;Methylation of an ETS site in the intron enhancer of the keratin 18 gene participates in tissue-specific repression;Roles of phospholipase C beta2 in chemoattractant-elicited responses.;Role of methylation in maintenance of positionally restricted transgene expression in developing muscle.;The 5' flanking region of the rat LAP (C/EBP beta) gene can direct high-level, position-independent, copy number-dependent expression in multiple tissues in transgenic mice.;High-level expression of the rat whey acidic protein gene is mediated by elements in the promoter and 3' untranslated region",
         "https://academic.oup.com/chemse/article-pdf/31/3/213/979519/bjj021.pdf"
        ],
        [
         "10",
         "0007fc19b59d115d1d89d19af7aaffcd72ff4013",
         "Tutorial in biostatistics: the self‐controlled case series method",
         "The self‐controlled case series method was developed to investigate associations between acute outcomes and transient exposures, using only data on cases, that is, on individuals who have experienced the outcome of interest. Inference is within individuals, and hence fixed covariates effects are implicitly controlled for within a proportional incidence framework. We describe the origins, assumptions, limitations, and uses of the method. The rationale for the model and the derivation of the likelihood are explained in detail using a worked example on vaccine safety. Code for fitting the model in the statistical package STATA is described. Two further vaccine safety data sets are used to illustrate a range of modelling issues and extensions of the basic model. Some brief pointers on the design of case series studies are provided. The data sets, STATA code, and further implementation details in SAS, GENSTAT and GLIM are available from an associated website. Copyright © 2005 John Wiley & Sons, Ltd.",
         "Semiparametric analysis of case series data;General population based study of the impact of tricyclic and selective serotonin reuptake inhibitor antidepressants on the risk of acute myocardial infarction;No evidence of an association between MMR vaccine and gait disturbance;Risk of myocardial infarction and stroke after acute infection or vaccination.;Safety of the trivalent inactivated influenza vaccine among children: a population-based study.;Control without separate controls: evaluation of vaccine safety using case-only methods.;The effect of transient exposures on the risk of an acute illness with low hazard rate.;Use of the inactivated intranasal influenza vaccine and the risk of Bell's palsy in Switzerland.;Does influenza vaccination increase consultations, corticosteroid prescriptions, or exacerbations in subjects with asthma or chronic obstructive pulmonary disease?;Exposure to tricyclic and selective serotonin reuptake inhibitor antidepressants and the risk of hip fracture.;Bacterial infections, immune overload, and MMR vaccine;Within‐subject exposure dependency in case‐crossover studies;Statistical assessment of the association between vaccination and rare adverse events post-licensure.;MMR and autism: further evidence against a causal association.;Idiopathic thrombocytopenic purpura and MMR vaccine;Intussusception among infants given an oral rotavirus vaccine.;Bias in the case – crossover design: implications for studies of air pollution;Does influenza vaccination exacerbate asthma? Analysis of a large cohort of children with asthma. Vaccine Safety Datalink Team.;Outbreak of aseptic meningitis associated with mass vaccination with a urabe-containing measles-mumps-rubella vaccine: implications for immunization programs.;Autism and measles, mumps, and rubella vaccine: no epidemiological evidence for a causal association;A unified approach to the analysis of case-distribution (case-only) studies.;Does Influenza Vaccination Exacerbate Asthma?;Bidirectional case-crossover designs for exposures with time trends.;Case series analysis of adverse reactions to vaccines: a comparative evaluation.;A new method for active surveillance of adverse events from diphtheria/tetanus/pertussis and measles/mumps/rubella vaccines;Epidemiologic assessment of risks of adverse reactions associated with intermittent exposure.;Risk of aseptic meningitis after measles, mumps,and rubella vaccine in UK children;Design and analysis of drug safety studies, with special reference to sporadic drug use and acute adverse reactions.;The case-crossover design: a method for studying transient effects on the risk of acute events.;A case-cohort design for epidemiologic cohort studies and disease prevention trials;Copyright Ltd. Statist. Med;Does oral polio vaccine cause intussusception in infants? Evidence from a sequence of three self-controlled cases series studies in the united kingdom;Lack of association between intussusception and oral polio vaccine in Cuban children;GenStat release 7.1. VSN international Ltd;Stata Statistical Software: Release 8.0. College Station, TX: Stata Corporation;The guide to GenStat release;Wheezing lower respiratory disease and vaccination of full‐term infants;Stata Statistical Software: Release 7.;Should we use a case-crossover design?;Relative incidence estimation from case series for vaccine safety evaluation.;GLIM release 4. Royal Statistical Society;Interaction between life history events, Nonparametric analysis for prospective and retrospective data in the presence of censoring",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/sim.2302"
        ],
        [
         "11",
         "00088986c30f3b32724562d84b4e9be87262bc21",
         "Reasoning support for Semantic Web ontology family languages using Alloy",
         "Semantic Web (SW), commonly regarded as the next generation of the Web, is an emerging vision of the new Web from the Knowledge Representation and the Web communities. To realize this vision, a series of techniques has been proposed. Semantic Web Ontology Langauge (OWL) and its extension Semantic Web rule Language (SWRL) and Semantic Web Logic Language (SWRL-FOL) are some of the most important outputs from the SW activities. However the existing reasoning and consistency checking tools for those languages are primitive. This paper proposes using the existing formal modelling tool, in particular Alloy, to provide an automatic reasoning service for the Semantic Web ontology family languages (OWL/SWRL/SWRL-FOL).",
         "",
         "http://www.comp.nus.edu.sg/~dongjs/papers/reasoning-alloy06.pdf"
        ],
        [
         "12",
         "0009cc65e12ef353daf3bcc34fa2e24bbe15c7c8",
         "Vascular endothelial growth factor overexpression by soft tissue sarcoma cells: implications for tumor growth, metastasis, and chemoresistance.",
         "To better elucidate the role of vascular endothelial growth factor (VEGF)(165) in soft tissue sarcoma (STS) growth, metastasis, and chemoresistance, we generated stably transfected human STS cell lines with VEGF(165) to study the effect of VEGF(165) on STS cells in vitro and the effect of culture medium from these cells on human umbilical vascular endothelial cells. Severe combined immunodeficient mice bearing xenografts of transfected cell lines were used to assess the effect of VEGF overexpression and the effect of VEGF receptor (VEGFR) 2 inhibition on STS growth, metastasis, and response to doxorubicin. VEGF(165)-transfected xenografts formed highly vascular tumors with shorter latency, accelerated growth, enhanced chemoresistance, and increased incidence of pulmonary metastases. Blockade of VEGFR2 signaling using DC101 anti-VEGFR2 monoclonal antibody enhanced doxorubicin chemoresponse; this combined biochemotherapy inhibited tumor growth and decreased pulmonary metastases without overt toxicity. Combined therapy reduced microvessel counts while increasing vessel maturation index. VEGF overexpression did not affect on the sarcoma cells per se; however, conditioned medium from VEGF transfectants caused increased endothelial cell proliferation, migration, and chemoresistance. Addition of DC101 induced endothelial cell sensitivity to doxorubicin and suppressed the activity of matrix metalloproteinases secreted by endothelial cells. We therefore conclude that VEGF is a critical determinant of STS growth and metastasis and that STS chemoresistance, in our model, is a process induced by the interplay between STS cells and tumor-associated endothelial cells. STS growth and metastasis can be interrupted by combined low-dose doxorubicin and anti-VEGFR2, a strategy that attacks STS-associated endothelial cells. In the future, such therapeutic approaches may be useful in treating STS before the development of clinically apparent metastases.",
         "Angiogenic profile of soft tissue sarcomas based on analysis of circulating factors and microarray gene expression.;Phase II study of doxorubicin and bevacizumab for patients with metastatic soft-tissue sarcomas.;Expression of vascular endothelial growth factor and its receptor, KDR/Flk-1, in soft tissue sarcomas.;Soft-tissue sarcomas in adults.;Analysis of hypoxia-related gene expression in sarcomas and effect of hypoxia on RNA interference of vascular endothelial cell growth factor A.;Therapeutic targeting of the tumor microenvironment.;Transcriptional Repression of Protein Kinase Cα via Sp1 by Wild Type p53 Is Involved in Inhibition of Multidrug Resistance 1 P-Glycoprotein Phosphorylation*;Antiangiogenic versus cytotoxic therapeutic approaches to human pancreas cancer: an experimental study with a vascular endothelial growth factor receptor-2 tyrosine kinase inhibitor and gemcitabine.;ZD6474, a vascular endothelial growth factor receptor tyrosine kinase inhibitor with additional activity against epidermal growth factor receptor tyrosine kinase, inhibits orthotopic growth and angiogenesis of gastric cancer.;Angiopoietins in tumours: the angiogenic switch;Phase II Study of the Antiangiogenic Agent SU5416 in Patients with Advanced Soft Tissue Sarcomas;Circulating angiogenic factor levels correlate with extent of disease and risk of recurrence in patients with soft tissue sarcoma.;Combined inhibition of VEGF‐ and PDGF‐signaling enforces tumor vessel regression by interfering with pericyte‐mediated endothelial cell survival mechanisms;Blood vessels and nerves: common signals, pathways and diseases;Doxorubicin-based chemotherapy for the palliative treatment of adult patients with locally advanced or metastatic soft tissue sarcoma.;Tumor-derived vascular endothelial growth factor up-regulates angiopoietin-2 in host endothelium and destabilizes host vasculature, supporting angiogenesis in ovarian cancer.;Protracted low-dose effects on human endothelial cell proliferation and survival in vitro reveal a selective antiangiogenic window for various chemotherapeutic drugs.;Vascular and haematopoietic stem cells: novel targets for anti-angiogenesis therapy?;Anti-vascular endothelial growth factor receptor 2 antibody reduces tumorigenicity and metastasis in orthotopic prostate cancer xenografts via induction of endothelial cell apoptosis and reduction of endothelial cell matrix metalloproteinase type 9 production.;Combined anti-fetal liver kinase 1 monoclonal antibody and continuous low-dose doxorubicin inhibits angiogenesis and growth of human soft tissue sarcoma xenografts by induction of endothelial cell apoptosis.;A role for survivin in chemoresistance of endothelial cells mediated by VEGF;VEGF(121), VEGF(165) overexpression enhances tumorigenicity in U251 MG but not in NG-1 glioma cells.;Vascular endothelial growth factor-165 overexpression stimulates angiogenesis and induces cyst formation and macrophage infiltration in human ovarian cancer xenografts.;Vascular endothelial growth factor-A(165) induces progression of melanoma brain metastases without induction of sprouting angiogenesis.;Vascular endothelial growth factor (VEGF) is an autocrine growth factor for VEGF receptor-positive human tumors.;Wild type p53 sensitizes soft tissue sarcoma cells to doxorubicin by down‐regulating multidrug resistance‐1 expression;Normalizing tumor vasculature with anti-angiogenic therapy: A new paradigm for combination therapy;High tumor levels of vascular endothelial growth factor predict poor response to systemic therapy in advanced breast cancer.;Angiogenesis and antiangiogenic approaches to sarcomas;Concentration of vascular endothelial growth factor in the tumour tissue as a prognostic factor of soft tissue sarcomas;Vascular Endothelial Growth Factor and Soft Tissue Sarcomas: Tumor Expression Correlates With Grade;Up-regulation of vascular endothelial growth factor in breast cancer cells by the heregulin-beta1-activated p38 signaling pathway enhances endothelial cell migration.;Wild-type p53 suppresses angiogenesis in human leiomyosarcoma and synovial sarcoma by transcriptional suppression of vascular endothelial growth factor expression.;Blockade of the epidermal growth factor receptor signaling by a novel tyrosine kinase inhibitor leads to apoptosis of endothelial cells and therapy of human pancreatic carcinoma.;Serum levels of vascular endothelial growth factor and basic fibroblast growth factor in patients with soft-tissue sarcoma;Selective ablation of immature blood vessels in established human tumors follows vascular endothelial growth factor withdrawal.;Vascular Endothelial Growth Factor Regulates Endothelial Cell Survival through the Phosphatidylinositol 3′-Kinase/Akt Signal Transduction Pathway;Overexpression of c-erbB-2/neu in breast cancer cells confers increased resistance to Taxol via mdr-1-independent mechanisms.;Patterns and Emerging Mechanisms of the Angiogenic Switch during Tumorigenesis;In vitro neutralization of vascular endothelial growth factor activation of Flk-1 by a monoclonal antibody;c-erbB-2/neu overexpression enhances metastatic potential of human lung cancer cells by induction of metastasis-associated properties.;Effects of verapamil on the acute toxicity of doxorubicin in vivo.;Accurate identification of experimental pulmonary metastases.;Overexpression of Vascular Endothelial Growth Factor-a165 Updated Version Cited Articles Citing Articles E-mail Alerts Overexpression of Vascular Endothelial Growth Factor-a165 Enhances Tumor Angiogenesis but Not Metastasis during ␤-cell Carcinogenesis;Molecular strategies targeting the host component of cancer to enhance tumor response to radiation therapy.;Cancer Research Cancer Res;Patterns of expression and secretion of vascular endothelial growth factor in malignant soft-tissue tumours;Dormancy of micrometastases: Balanced proliferation and apoptosis in the presence of angiogenesis suppression",
         "https://aacrjournals.org/cancerres/article-pdf/66/17/8770/2554504/8770.pdf"
        ],
        [
         "13",
         "000a0424b18c417d0e86d2f7896613f58306a5b3",
         "Cortical representations of pitch in monkeys and humans",
         "None",
         "",
         "https://europepmc.org/articles/pmc4325365?pdf=render"
        ],
        [
         "14",
         "000a0840f6a41c30cd2b60f262f8167be74077da",
         "State Feedback Impulse Elimination for Singular Systems over a Hermite Domain",
         "We reduce the problem of impulse elimination via state feedback in singular differential equations to algebra. Our results are developed for systems over an arbitrary Hermite domain. We show that the established theories for the time-invariant and the real analytic time-varying settings can be unified in this way. Besides the constant and real analytic functions, several other function rings are considered. Our algebraic theory is applied to these cases, providing solutions to the impulse elimination problem for classes of systems not previously studied. In particular, our work allows the restriction of the feedback matrix to certain function rings",
         "Analysis and Numerical Solution of Control Problems in Descriptor Form;Singular Control Systems;Linear Systems over Commutative Rings;A survey of linear singular systems;Controllability, observability, and duality in singular systems;Linear Algebra over Commutative Rings;The Theory of Matrices;CANONICAL FORMS AND SOLVABLE SINGULAR SYSTEMS OF DIFFERENTIAL EQUATIONS;A further interpretation of inconsistent initial conditions in descriptor-variable systems;A generalized state-space for singular systems;Feedback and pole placement in descriptor variable Systems;Introduction to the Theory and Application of the Laplace Transformation;Generalizations of a theorem of doležal;The Theory of Matrices;Elementary divisors and modules;Functions of One Complex Variable;Regularization of Linear Descriptor Systems with Linear Coefficients;State feedback impulse elimination of linear time-varying singular systems;Control System Synthesis : A Factorization Approach;Singular systems of differential equations II;The theory of matrices;Functions of a Complex Variable",
         "http://www.ece.wisc.edu/~cobb/papers/SICON6(2).pdf"
        ],
        [
         "15",
         "000a1c397de7a66ef20202693e14cf9251d471d8",
         "Case‐Control Association Tests Correcting for Population Stratification",
         "In case‐control association studies unobserved population stratification may act as a confounder, leading to an increased number of false positive results. Methods accounting for population structure by using additional genetic markers broadly follow one of two concepts: Genomic Control (GC) and Structured Association (SA). While extending existing methods of Structured Association we show that it is necessary to incorporate phenotypic information when inferring population structure, otherwise a systematic bias is introduced. Moreover, for moderate population stratification a Wald test statistic should be preferred as a Structured Association test statistic in comparison to a likelihood ratio test. The introduced extensions are compared to existing methods of Structured Association, as well as to Genomic Control, in a simulation study which is based on realistic situations of large case‐control studies with moderate population stratification. A disadvantage of Genomic Control turns out to be the large variation in estimating the variance inflation factor, as well as the power loss if population structure increases. We come to the overall conclusion that Structured Association, if applied correctly, is superior to Genomic Control, at least in the case of simple population structure as simulated here.",
         "",
         "https://ediss.uni-goettingen.de/bitstream/11858/00-1735-0000-000D-F273-A/1/koehler.pdf"
        ],
        [
         "16",
         "000a5535c7f5a8b73a8ec93354a41153b87a5ab2",
         "Deficient Major Histocompatibility Complex-Linked Innate Murine Cytomegalovirus Immunity in MA/My.L-H2b Mice and Viral Downregulation of H-2k Class I Proteins",
         "ABSTRACT NK cells are key effectors of innate immunity and host survival during cytomegalovirus (CMV) infection. Innate murine CMV (MCMV) resistance in MA/My mice requires Ly49H/m157-independent H-2k-linked NK cell control. Here we show that replacement of MA/My H-2k with C57L H-2b susceptibility genes led to a remarkable loss of innate virus immunity, though NK gamma interferon was induced in H-2b and H-2k strains shortly after infection. Thus, H-2b genes expressed in C57L or MA/My.L-H2b are sufficient in alerting NK cells to intrusion but fail to support NK restraint of viral infection. In addition, novel H-2 recombinant strains were produced and utilized in a further refinement of a critical genetic interval controlling innate H-2k-linked MCMV resistance. Importantly, this analysis excluded the gene interval from Kk class I through class II. The responsible gene(s) therefore resides in an interval spanning Dk class Ia and more-distal major histocompatibility complex (MHC) nonclassical class Ib genes. Recently, the NK activation receptor Ly49P and MHC class I Dk proteins were genetically implicated in MCMV resistance, in part because Ly49P-expressing reporter T cells could specifically bind Dk molecules on MCMV-infected mouse embryonic fibroblasts (MEFs). However, as we found that H-2k innate resistance differs in the C57L or MA/My backgrounds and because MCMV very efficiently downregulates H-2k class I proteins in L929 cells and primary MEFs shortly after infection, a Ly49P/Dk model should not fully explain H-2k-linked MCMV resistance.",
         "",
         "https://europepmc.org/articles/pmc1797260?pdf=render"
        ],
        [
         "17",
         "000a55c6a88b8c1413e6bb0281ea4e6a8bac4588",
         "Automatic Generation of Detection Algorithms for Design Defects",
         "Maintenance is recognised as the most difficult and expansive activity of the software development process. Numerous techniques and processes have been proposed to ease the maintenance of software. In particular, several authors published design defects formalising \"bad\" solutions to recurring design problems (e.g., anti-patterns, code smells). We propose a language and a framework to express design defects synthetically and to generate detection algorithms automatically. We show that this language is sufficient to describe some design defects and to generate detection algorithms, which have a good precision. We validate the generated algorithms on several programs",
         "",
         "http://www.archipel.uqam.ca/5168/1/Moha06-ASE.pdf"
        ],
        [
         "18",
         "000b0ebf13045810dbd2aeae7423b10ce306dd76",
         "Costimulation via CD55 on Human CD4+ T Cells Mediated by CD97",
         "Decay-accelerating factor (CD55) is a complement regulatory protein, which is expressed by most cells to protect them from complement-mediated attack. CD55 also binds CD97, an EGF-TM7 receptor constitutively expressed on granulocytes and monocytes and rapidly up-regulated on T and B cells upon activation. Early results suggested that CD55 could further enhance T cell proliferation induced by phorbol ester treatment. The present study demonstrates that coengagement of CD55, using either cross-linking mAbs or its natural ligand CD97, and CD3 results in enhanced proliferation of human peripheral blood CD4+ T cells, expression of the activation markers CD69 and CD25, and secretion of IL-10 and GM-CSF. Recently, an increase in T cell responsiveness in CD55−/− mice was shown to be mediated by a lack of complement regulation. In this study, we show that direct stimulation of CD55 on CD4+ T cells with CD97 can modulate T cell activation but does not interfere with CD55-mediated complement regulation. Our results support a multifaceted role for CD55 in human T cell activation, constituting a further link between innate and adaptive immunity.",
         "Holding T cells in check--a new role for complement regulators?;Regulatory T cells: development, function and role in autoimmunity.;Decay-accelerating factor modulates induction of T cell immunity;CD97, an adhesion receptor on inflammatory cells, stimulates angiogenesis through binding integrin counterreceptors on endothelial cells.;The complement inhibitory protein DAF (CD55) suppresses T cell immunity in vivo;CD46 is a cellular receptor for group B adenoviruses;The epidermal growth factor-like domains of the human EMR2 receptor mediate cell attachment through chondroitin sulfate glycosaminoglycans.;Activation of human CD4+ cells with CD3 and CD46 induces a T-regulatory cell 1 phenotype;Cutting Edge: B7/CD28 Interactions Regulate Cell Cycle Progression Independent of the Strength of TCR Signaling1;Expression and regulation of CD97 in colorectal carcinoma cell lines and tumor tissues.;The role of CD55 in protecting the tumour environment from complement attack.;Molecular Analysis of the Epidermal Growth Factor-like Short Consensus Repeat Domain-mediated Protein-Protein Interactions;Tissue distribution of the human CD97 EGF-TM7 receptor.;Family of Escherichia coli Dr adhesins: decay-accelerating factor receptor recognition and invasiveness.;Membrane complement regulatory proteins: insight from animal studies and relevance to human diseases.;Crry/p65, a Membrane Complement Regulatory Protein, Has Costimulatory Properties on Mouse T Cells1;CD46 Is a Cellular Receptor for Human Herpesvirus 6;Mapping the binding domains on decay accelerating factor (DAF) for haemagglutinating enteroviruses: implications for the evolution of a DAF-binding phenotype.;Expression characteristics of the human CD97 antigen.;Membrane cofactor protein (MCP or CD46) is a cellular pilus receptor for pathogenic Neisseria;CD97 is a processed, seven-transmembrane, heterodimeric receptor associated with inflammation.;The seven-span transmembrane receptor CD97 has a cellular ligand (CD55, DAF);EGF-TM7: a novel subfamily of seven-transmembrane-region leukocyte cell-surface molecules.;Membrane cofactor protein (CD46) is a keratinocyte receptor for the M protein of the group A streptococcus.;Coxsackievirus B3 adapted to growth in RD cells binds to decay-accelerating factor (CD55);Decay-accelerating factor (CD55), a glycosylphosphatidylinositol-anchored complement regulatory protein, is a receptor for several echoviruses.;Characterization of an Early Activation‐Dependent Antigen on Lymphocytes Defined by the Monoclonal Antibody BL‐Ac(F2);Human membrane cofactor protein (CD46) acts as a cellular receptor for measles virus;Signal transduction through decay-accelerating factor. Interaction of glycosyl-phosphatidylinositol anchor and protein tyrosine kinases p56lck and p59fyn 1.;Mapping of epitopes, glycosylation sites, and complement regulatory domains in human decay accelerating factor.;Distinct receptor and regulatory properties of recombinant mouse complement receptor 1 (CR1) and Crry, the two genetic homologues of human CR1;GPI-anchored cell-surface molecules complexed to protein tyrosine kinases.;The Dr hemagglutinin, afimbrial adhesins AFA-I and AFA-III, and F1845 fimbriae of uropathogenic and diarrhea-associated Escherichia coli belong to a family of hemagglutinins with Dr receptor recognition;Decay-accelerating factor functions as a signal transducing molecule for human T cells.;Induction of T cell activation by monoclonal anti-Thy-1 antibodies.;Antitumour reactions of monoclonal antibody against a human osteogenic-sarcoma cell line.;Natural versus adaptive regulatory T cells.;Activation of T cells via CD55: recruitment of early components of the CD3-TCR pathway is required for IL-2 secretion.;Decay-accelerating factor: biochemistry, molecular biology, and function.;The epidermal growth factor-seven transmembrane (EGF-TM7) receptor CD97 is required for neutrophil migration and host defense",
         "https://journals.aai.org/jimmunol/article-pdf/177/2/1070/1220551/zim01406001070.pdf"
        ],
        [
         "19",
         "000f0b4a5545457a688f5ab07ac7e179ca881b82",
         "Evolutionary transformations of fetal membrane characters in Eutheria with special reference to Afrotheria.",
         "Analysis of molecular data sets has provided new insights into higher-level relationships of living Eutheria, including the recognition of Afrotheria as a novel taxon. This offers an opportunity to take a fresh look at the evolution of organ systems, including some that are little used in traditional systematics. In the present study, we attempted a reconstruction of the evolution of characters associated with placentation, the fetal membranes and the female reproductive tract. The evolutionary history of 21 characters has been traced, based on a current hypothesis of eutherian relationships, by applying the computer program MacClade. Accordingly, the analysis provides a first comprehensive interpretation of the stem species pattern of Eutheria. Of particular note, this pattern includes an endotheliochorial chorioallantoic placenta. The reconstructed pattern of Eutheria does not change in the basal nodes of the group. Thus, no character transformations occur on the stem lineages of Laurasiatheria or Euarchontoglires, and even Afrotheria has mostly plesiomorphic character conditions. However, two character transformations occur on the common stem lineage of Afrotheria and its sister taxon Xenarthra, i.e., amniogenesis by cavitation instead of folding and the precocial state of the newborn. In addition, we recognized one character transformation on the stem lineage of Afrotheria, i.e., the occurrence of a four-lobed allantoic sac. Thus, contrary to previous assertions, it is possible to identify morphological characters that could be synapomorphic for this novel taxon.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/jez.b.21079"
        ],
        [
         "20",
         "000f70100783b16315ff748f23387076fc91d876",
         "Protein disorder prediction by condensed PSSM considering propensity for order or disorder",
         "None",
         "Accurate Prediction of Protein Disordered Regions by Mining Protein Structure Data;RONN: the bio-basis function neural network technique applied to the detection of natively disordered regions in proteins;IUPred: web server for the prediction of intrinsically unstructured regions of proteins based on estimated energy content;FoldIndex©: a simple tool to predict whether a given protein sequence is intrinsically unfolded",
         "https://bmcbioinformatics.biomedcentral.com/counter/pdf/10.1186/1471-2105-7-319"
        ],
        [
         "21",
         "00117e0d2c89e54b24f9ee6d1b78b479cfc8b357",
         "A Model of Spatial Epidemic Spread When Individuals Move Within Overlapping Home Ranges",
         "None",
         "",
         "https://link.springer.com/content/pdf/10.1007/s11538-005-9027-y.pdf"
        ],
        [
         "22",
         "00122b0f976ecd6b26a8eece8c1d6a89c7cea69b",
         "NKT Cells and IFN-γ Establish the Regulatory Environment for the Control of Diabetogenic T Cells in the Nonobese Diabetic Mouse1",
         "In type 1 diabetes mellitus (T1DM), T cell-mediated destruction of insulin-producing pancreatic β cells leads to the acute onset of hyperglycemia. The nonobese diabetic mouse model of human T1DM reveals that T cells capable of inducing diabetes can escape normal central tolerance, and can cause T1DM if left unchecked. However, several regulatory T cell subsets can temper autoaggressive T cells, although it remains undetermined when and how, and by which subset, homeostatic control of diabetogenic T cells is normally achieved in vivo. Using a cotransfer model, we find that NKT cells efficiently dampen the action of diabetogenic CD4+ T cells, and do so in an indirect manner by modifying the host environment. Moreover, the NKT cell-containing population modifies the host via production of IFN-γ that is necessary for driving the inhibition of diabetogenic T cells in vivo.",
         "",
         "https://www.jimmunol.org/content/jimmunol/176/3/1645.full.pdf"
        ],
        [
         "23",
         "00123890feb5ddd7d2a734aae13f7927aa9e8477",
         "mTOR is out of control in polycystic kidney disease.",
         "Autosomal-dominant polycystic kidney disease (ADPKD) is a common genetic disease caused by mutations in the PKD1 gene. This adult-onset disease results in the accumulation of destructive kidney cysts, leading to progressive loss of renal function and eventually to renal failure. A need for kidney transplantation and dialysis are common outcomes. There are currently no treatment options to prevent or delay the disease onset. Although the PKD1 gene was identified more than a decade ago, the development of treatment strategies has been hampered by a lack of understanding of the function of polycystin-1 (PC1), the protein encoded by the PKD1 gene (1, 2). Work by Shillingford et al. (3) in this issue of PNAS now identifies a new function of PC1, which immediately suggests a possibility for future treatment options.\n\nPC1 is a very large integral membrane protein with a much smaller C-terminal cytoplasmic tail. Previous work had implicated this tail in numerous signaling events, but their relevance to ADPKD had remained largely unclear. Shillingford et al. (3) now report that the PC1 tail interacts with a very interesting player by the name of tuberin, the product of the TSC2 gene. Tuberin mutations lead to the complex disease tuberous sclerosis (TSC), which is ≈10 times less common than ADPKD. Three earlier observations had already suggested that tuberin and PC1 could be functionally linked. First, in addition to benign tumors in multiple organs, TSC patients also exhibit kidney cysts. Second, the TSC2 gene is located only a handful of base pairs away from the PKD1 gene. A subset of patients has larger chromosomal deletions that affect both the TSC2 and the PKD1 genes at the same time. These patients suffer from very severe, early onset polycystic kidney disease. Third, previous results using tuberin null cells had suggested that tuberin may play … \n\n*Correspondence should be addressed: Department of Anatomy, University of California School of Medicine, Genentech Hall, Room N212B, Box 2140, 600 16th Street, San Francisco, CA 94107. E-mail: \n\nmostov{at}itsa.ucsf.edu",
         "The mTOR pathway is regulated by polycystin-1, and its inhibition reverses renal cystogenesis in polycystic kidney disease.;Rapamycin: an anti-cancer immunosuppressant?;Intraflagellar transport and cilia-dependent renal disease: the ciliary hypothesis of polycystic kidney disease.;Autosomal dominant polycystic kidney disease: molecular genetics and pathophysiology.;Polycystins 1 and 2 mediate mechanosensation in the primary cilium of kidney cells;Tuberin, the tuberous sclerosis complex 2 tumor suppressor gene product, regulates Rho activation, cell adhesion and migration;Bending the MDCK Cell Primary Cilium Increases Intracellular Calcium;Tuberin-dependent membrane localization of polycystin-1: a functional link between polycystic kidney disease and the TSC2 tumor suppressor gene.;Polycystin-1, STAT6, and P100 function in a pathway that transduces ciliary mechanosensation and is activated in polycystic kidney disease.;TSC2: filling the GAP in the mTOR signaling pathway.;Dysregulation of the TSC-mTOR pathway in human disease;www.pnas.org cgi doi 10.1073 pnas.0601352103 Mostov D ow nl oa de d by g ue st o n Ja nu ar y 10",
         "https://europepmc.org/articles/pmc1459339?pdf=render"
        ],
        [
         "24",
         "0015c20c3ac2a207674c3bad17bf7487b6e9863d",
         "The F.A.S.T.-Model",
         "None",
         "",
         "http://arxiv.org/pdf/0804.1893"
        ],
        [
         "25",
         "0016d2f869fda0a40404330902d5f4a8c361bad8",
         "Antarctic birds breed later in response to climate change.",
         "In the northern hemisphere, there is compelling evidence for climate-related advances of spring events, but no such long-term biological time series exist for the southern hemisphere. We have studied a unique data set of dates of first arrival and laying of first eggs over a 55-year period for the entire community of Antarctic seabirds in East Antarctica. The records over this long period show a general unexpected tendency toward later arrival and laying, an inverse trend to those observed in the northern hemisphere. Overall, species now arrive at their colonies 9.1 days later, on average, and lay eggs an average of 2.1 days later than in the early 1950s. Furthermore, these delays are linked to a decrease in sea ice extent that has occurred in eastern Antarctica, which underlies the contrasted effects of global climate change on species in Antarctica.",
         "Southern Ocean sea ice and its wider linkages: insights revealed from models and observations;Long-term decline in krill stock and increase in salps within the Southern Ocean;Ice Core Evidence for Antarctic Sea Ice Decline Since the 1950s;The Adélie Penguin: Bellwether of Climate Change;A globally coherent fingerprint of climate change impacts across natural systems;Fingerprints of global warming on wild animals and plants;Environmental Change and Antarctic Seabird Populations;Interpretation of Recent Southern Hemisphere Climate Change;Ecological responses to recent climate change;Brood parasitism: Female ducks can double their reproduction;Devil in the Detail;Emperor penguins and climate change;Ocean circulation off east Antarctica affects ecosystem structure and sea-ice extent;Effects of elevated temperature on multi‐species interactions: the case of Pedunculate Oak, Winter Moth and Tits;First census of breeding seabirds between Cap Bienvenue (Terre Adélie) and Moyes Islands (King George V Land), Antarctica: new records for Antarctic seabird populations;The effect of ice conditions in northern Hudson Bay on breeding by Thick-billed Murres (Uria lomvia);Effects of sea-ice extent and krill or salp dominance on the Antarctic food web;Body Condition and Seabird Reproductive Performance: A Study of Three Petrel Species;Amphibian breeding and climate;The diets of five summer breeding seabirds in Adélie Land, Antarctica;Climate change 2001 : the scientific basis",
         "https://europepmc.org/articles/pmc1458863?pdf=render"
        ],
        [
         "26",
         "0017b6ca03ef7cff14e2bc573673ff91ea8caf24",
         "Soluble N‐cadherin in human biological fluids",
         "Classical cadherins such as E‐, P‐ and N‐cadherin are transmembrane proteins that mediate cell–cell adhesion, and are important in embryogenesis, maintenance of tissue integrity and cancer. Proteolytic shedding of the extracellular domain results in the generation of soluble E‐, P‐ or N‐cadherin ectodomains. Circulating soluble E‐ and P‐cadherin have been described in the serum, and elevated levels were detected in cancer patients when compared with healthy persons. Here we report the presence of soluble N‐cadherin, a 90‐kD protein fragment, in the serum of both healthy persons and cancer patients, using a direct ELISA and immunoprecipitation. A correlation was found between prostate specific antigen and soluble N‐cadherin, and significantly elevated levels were detected in prostate cancer follow‐up patients. The N‐cadherin protein is neo‐expressed by carcinomas of the prostate, and is responsible for epithelial to fibroblastic transition. This is reflected by the higher concentrations of soluble N‐cadherin in prostate cancer patients than in healthy persons. © 2006 Wiley‐Liss, Inc.",
         "Soluble N-cadherin fragment promotes angiogenesis;Membrane Localization of Membrane Type 5 Matrix Metalloproteinase by AMPA Receptor Binding Protein and Cleavage of Cadherins;N‐cadherin switching occurs in high Gleason grade prostate cancer;Differential exoprotease activities confer tumor-specific serum peptidome patterns.;N-cadherin Expression in Breast Cancer: Correlation with an Aggressive Histologic Variant – Invasive Micropapillary Carcinoma;Early prediction of tumor recurrence after curative resection of gastric carcinoma by measuring soluble E‐cadherin;Urinary levels of soluble e-cadherin in the detection of transitional cell carcinoma of the urinary bladder.;ADAM10 mediates E-cadherin shedding and regulates epithelial cell-cell adhesion, migration, and beta-catenin translocation.;Identification of degradome components associated with prostate cancer progression by expression analysis of human prostatic tissues;ADAM10 cleavage of N‐cadherin and regulation of cell–cell adhesion and β‐catenin nuclear signalling;Soluble p-cadherin found in human semen.;Regulation of matrix biology by matrix metalloproteinases.;Critical role of N-cadherin in myofibroblast invasion and migration in vitro stimulated by colon-cancer-cell-derived TGF-β or wounding;N-cadherin in the spotlight of cell-cell adhesion, differentiation, embryogenesis, invasion and signalling.;Soluble E-cadherin is an independent pretherapeutic factor for long-term survival in gastric cancer.;New functions for the matrix metalloproteinases in cancer progression;Plasmin Produces an E-Cadherin Fragment That Stimulates Cancer Cell Invasion;Matrilysin mediates extracellular cleavage of E-cadherin from prostate cancer cells: a key mechanism in hepatocyte growth factor/scatter factor-induced cell-cell dissociation and in vitro invasion.;Soluble E-cadherin is a valid prognostic marker in gastric carcinoma;Lack of correlation between serum levels of E- and P-cadherin fragments and the presence of breast cancer.;Cadherin switching in human prostate cancer progression.;Urinary concentrations of the soluble adhesion molecule E-cadherin and total protein in patients with bladder cancer;Characterization of soluble E-cadherin as a disease marker in gastric cancer patients.;Soluble E-cadherin concentrations in patients with systemic inflammatory response syndrome and multiorgan dysfunction syndrome.;Soluble forms of the adhesion molecule E-cadherin in urine.;Soluble E-cadherin fragments increased in circulation of cancer patients.;Purification and characterization of NCAD90, a Soluble endogenous form of N‐cadherin, which is generated by proteolysis during retinal development and retains adhesive and neurite‐promoting function;Construction of epithelioid sheets by transfection of mouse sarcoma cells with cDNAs for chicken cell adhesion molecules.;Identification and purification of a cell surface glycoprotein mediating intercellular adhesion in embryonic and adult tissue;Ischemia-induced cleavage of cadherins in NRK cells requires MT1-MMP (MMP-14).;Soluble fragment of P‐cadherin adhesion protein found in human milk;Expression of classic cadherins type I in urothelial neoplastic progression.;The E-cadherin/catenin complex in invasion: the role of ectodomain shedding.;Cancer Metastasis, Molecular and Cellular Mechanisms and Clinical Intervention;Plasmin produces an Ecadherin fragment that stimulates cancer cell invasion;Monden M. Characterization of soluble E-cadherin as a disease marker in gastric cancer patients;nary concentrations of the soluble adhesion molecule E - cadherin and total protein in patients with bladder cancer;Identification and purification of a",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/ijc.22219"
        ],
        [
         "27",
         "0018adec76441851263e5f7d8195a50ae353707b",
         "Long-range cooperative interactions modulate dimerization in SARS 3CLpro.",
         "Severe acute respiratory syndrome (SARS) is an infectious disease caused by the human coronavirus, SARS-CoV. The main viral protease, SARS 3CLpro, is a validated target for the development of antiviral therapies. Since the enzyme is a homodimer and the individual monomers are inactive, two approaches are being used to develop inhibitors: enzyme activity inhibitors that target the active site and dimerization inhibitors. Dimerization inhibitors are usually targeted to the dimerization interface and need to compete with the attractive forces between subunits to be effective. In this paper, we show that the dimerization of SARS 3CLpro is also under allosteric control and that additional and energetically more favorable target sites away from the dimerization interface may also lead to subunit dissociation. We previously identified a cluster of conserved serine residues (Ser139, Ser144, and Ser147) located adjacent to the active site of 3CLpro that could effectively be targeted to inactivate the protease [Bacha, U et al. (2004) Biochemistry 43, 4906-4912]. Mutation of any of these serine residues to alanine had a debilitating effect on the catalytic activity of 3CLpro. In particular, the mutation of Ser147, which does not make any contact with the opposing subunit and is located approximately 9 A away from the dimer interface, totally inhibited dimerization and resulted in a complete loss of enzymatic activity. The finding that residues away from the dimer interface are able to control dimerization defines alternative targets for the design of dimerization inhibitors.",
         "",
         "https://europepmc.org/articles/pmc2570436?pdf=render"
        ],
        [
         "28",
         "001a3a8f9bcd143329d679da7b04980946bf1c78",
         "When to start dialysis?",
         "In an attempt to decrease mortality in dialysis patients, the care for end-stage renal disease patients has been a focus of nephrologists for many years. Good pre-dialysis care has been found to improve mortality numbers in several studies. However, it is still not exactly clear, when the best moment has arrived to start renal replacement therapy. By illustrating two totally different patient cases the effect of outcome and quality of life are discussed-demonstrating on the one hand the role of clinical parameters, and on the other hand the influence of biochemical parameters.",
         "",
         "https://academic.oup.com/ndt/article-pdf/21/suppl_2/ii20/6835586/gfl139.pdf"
        ],
        [
         "29",
         "001b4ffaab6cdc560e071cb0dcb2680269480829",
         "Reliability and validity of a computerized neurocognitive test battery, CNS Vital Signs.",
         "None",
         "Efficient Allocation of Attentional Resources in Patients With ADHD;Neurocognitive testing supports a broader concept of mild cognitive impairment;Validity of a novel computerized cognitive battery for mild cognitive impairment;Test-retest reliability, practice effects and reliable change indices for the recognition memory test.;The reliability and stability of verbal working memory measures;Further standardization of some NES3 tests.;Development and validation of a web-based neuropsychological test protocol for sports-related return-to-play decision-making.;The effects of practice on the cognitive test performance of neurologically normal individuals assessed at brief test-retest intervals.;A Verbal Learning and Memory Test for English- and Spanish-speaking Older Mexican-American Adults;Factor Structure, Normative Data and Retest-Reliability of a Test of Fine Motor Functions in Patients With Idiopathic Parkinson's Disease;Note on Comparability of Microcog Test Forms;Development of a general ability index for the Wechsler Adult Intelligence Scale--Third Edition.;Computerized Neurocognitive Scanning: I. Methodology and Validation in Healthy People;Attentional Function as Measured by a Continuous Performance Task in Children with Dyscalculia;Convergent Validity and Neuropsychological Correlates of the Schedule for the Assessment of Negative Symptoms (SANS) Attention Subscale;Effects of stimulants on the continuous performance test (CPT): implications for CPT use and interpretation.;Test–retest bias, reliability, and regression equations for neuropsychological measures repeated over a 12–16-week period;MicroCog: Assessment of Cognitive Functioning;Neuropsychological Assessment of Response Inhibition in Adults With ADHD;Continuous Performance Tests Are Sensitive to ADHD in Adults but Lack Specificity;Effects of chronic nicotine and methylphenidate in adults with attention deficit/hyperactivity disorder.;Effects of moderate and high doses of alcohol on attention, impulsivity, discriminability, and response bias in immediate and delayed memory task performance.;Establishing stable test performance in tests from the Behavioral Assessment and Research System (BARS).;Does the Conners' Continuous Performance Test Aid in ADHD Diagnosis?;Validity of a computer-assisted neurobehavioral test battery in toxicant encephalopathy.;Relationship of CogScreen-AE to flight simulator performance and pilot age.;Recognition Memory for Faces: Reliability and Validity of the Warrington Recognition Memory Test (RMT) in a Neurological Sample;Test–retest reliability and practice effects of Expanded Halstead–Reitan Neuropsychological Test Battery;Handbook of Normative Data for Neuropsychological Assessment;Test\\re-test reliability of the CANTAB and ISPOCD neuropsychological batteries: theoretical and practical issues;Neural activity during attention shifts between object features;Computerized self‐assessment in psychiatric in‐patients: acceptability, feasibility and influence of computer attitude;Clinical utility of the Test of Variables of Attention (TOVA) in the diagnosis of attention-deficit/hyperactivity disorder.;4 T-fMRI study of nonspatial shifting of selective attention: cerebellar and parietal contributions.;Smith's Recognizable Patterns of Human Malformation;A comparison of NES2 and traditional neuropsychological tests in a neurologic patient sample.;Screening for Cognitive Impairment in Older Individuals: Validation Study of a Computer-Based Test;Gender- and Age-Specific Changes in Motor Speed and Eye-Hand Coordination in Adults: Normative Values for the Finger Tapping and Grooved Pegboard Tests;A Compendium of Neuropsychological Tests: Administration, Norms, and Commentary;A methodological review of “method skeptic” reports;Computerized assessment in neuropsychology: A review of tests and test batteries;Concise Guide to Child and Adolescent Psychiatry;Performance of four age groups of normal elderly on the Rey Auditory-Verbal Learning Test.;Neurobehavioral effects of phenytoin prophylaxis of posttraumatic seizures.;The cognitive drug research computerized assessment system for demented patients: A validation study;Test-retest reliability of the halstead-reitan battery and the WAIS in a neuropsychiatric population;Neuropsychological deficit and emotional disturbance in head-injured patients.;Finger Tapping: Effects of Trials and Sessions;A computer-based neurobehavioral evaluation system for occupational and environmental epidemiology: methodology and validation studies.;Use of Memory Tests in Differentiating Organic Disorder from Depression;The computerized continuous performance task: A new measure of inattention;Psychomotor function and psychoactive drugs.;A review of stimulant drug research with hyperactive children.;EFFECTS OF REPEATED ADMINISTRATIONS OF A COMPREHENSIVE NEUROPSYCHOLOGICAL BATTERY AMONG CHRONIC EPILEPTICS;Sustained attention in remitted schizophrenics.;Attention in hyperactive children and the effect of methylphenidate (ritalin).;Perceptual‐Motor Behaviour in Relation to Blood Phenobarbitone Level: A Preliminary Report;ON THE NATURE OF THE \"ABSENCE\" IN CENTRENCEPHALIC EPILEPSY: A STUDY OF SOME BEHAVIORAL, ELECTROENCEPHALOGRAPHIC AND AUTONOMIC FACTORS.;L’examen Clinique en Psychologie;The effect on delayed-alternation test performance of stimulating or destroying electrically structures within the frontal lobes of the monkey's brain.;Self-administered screening for mild cognitive impairment: initial validation of a computerized test battery.;Pointing to an allocentric and egocentric remembered target.;RELIABILITY OF A NOVEL COMPUTERIZED NEUROPSYCHOLOGICAL BATTERY FOR MILD COGNITIVE IMPAIRMENT;Neuropsychological testing of high school athletes. Preliminary norms and test-retest indices.;Relationship of CogScreen-AE to flight simulator performance and pilot age.;Validity of a cognitive computerized assessment system in brain-injured patients.;Domains of behavior from a neuropsychological perspective: the whole story.;Alternate forms of the AVLT: a procedure and test of form equivalency.;An investigation of the test-retest reliability of the Stroop Color-Word Test across two intervals.;Symbol Digit Modalities Test (SDMT). Manual (Revised). Los Angeles;Stroop Color and Word Test: Manual for Clinical and Experimental Uses.;Recognizable patterns of human malformation.;The appraisal of children with cerebral deficits;The effect on delayed-alternation test performance of stimulating or destroying electrically structures within the frontal lobes of the monkey's brain.",
         "https://academic.oup.com/acn/article-pdf/21/7/623/9513/21-7-623.pdf"
        ],
        [
         "30",
         "001b56d111a9c856159f0d08738d8158aabfe621",
         "An auxin-driven polarized transport model for phyllotaxis",
         "Recent studies show that plant organ positioning may be mediated by localized concentrations of the plant hormone auxin. Auxin patterning in the shoot apical meristem is in turn brought about by the subcellular polar distribution of the putative auxin efflux mediator, PIN1. However, the question of what signals determine PIN1 polarization and how this gives rise to regular patterns of auxin concentration remains unknown. Here we address these questions by using mathematical modeling combined with confocal imaging. We propose a model that is based on the assumption that auxin influences the polarization of its own efflux within the meristem epidermis. We show that such a model is sufficient to create regular spatial patterns of auxin concentration on systems with static and dynamic cellular connectivities, the latter governed by a mechanical model. We also optimize parameter values for the PIN1 dynamics by using a detailed auxin transport model, for which parameter values are taken from experimental estimates, together with a template consisting of cell and wall compartments as well as PIN1 concentrations quantitatively extracted from confocal data. The model shows how polarized transport can drive the formation of regular patterns.",
         "Computer simulations reveal properties of the cell-cell signaling network at the shoot apex in Arabidopsis;Patterns of Auxin Transport and Gene Expression during Primordium Development Revealed by Live Imaging of the Arabidopsis Inflorescence Meristem;Functional redundancy of PIN proteins is accompanied by auxin-dependent cross-regulation of PIN expression;Polygonal planforms and phyllotaxis on plants.;Auxin inhibits endocytosis and promotes its own efflux from cells;PIN and AUX/LAX proteins: their role in auxin accumulation.;Microsurgical and laser ablation analysis of leaf positioning and dorsoventral patterning in tomato;A PINOID-Dependent Binary Switch in Apical-Basal PIN Polar Targeting Directs Auxin Efflux;Local, Efflux-Dependent Auxin Gradients as a Common Module for Plant Organ Formation;Regulation of phyllotaxis by polar auxin transport;Microsurgical and laser ablation analysis of interactions between the zones and layers of the tomato shoot apical meristem;Polar auxin transport in the wood-forming tissues of hybrid aspen is under simultaneous control of developmental and environmental signals;Cellerator: extending a computer algebra system to include biochemical arrows for signal transduction simulations;Auxin transport inhibitors block PIN1 cycling and vesicle trafficking;Nonlinear Dynamics and Chaos;Auxin Regulates the Initiation and Radial Position of Plant Lateral Organs;Regulation of polar auxin transport by AtPIN1 in Arabidopsis vascular tissue.;Snakes, shapes, and gradient vector flow;Symmetry in Plants;Arabidopsis AUX1 Gene: A Permease-Like Regulator of Root Gravitropism;Development;Phyllotaxis as a Dynamical Self Organizing Process Part III: The Simulation of the Transient Regimes of Ontogeny;Phyllotaxis as a physical self-organized growth process.;A connectionist model of development.;Numerical Recipes in C: The Art of Scientific Computing;A Diffusion Model of Phyllotaxis;Mathematical analysis of the chemosmotic polar diffusion of auxin through plant tissues.;The dynamics of auxin transport;A model for vein formation in higher plants;Phyllotaxis and the Fibonacci Series;TRANSPORT OF INDOLEACETIC ACID IN PLANT CELLS IN RELATION TO pH AND ELECTRICAL POTENTIAL GRADIENTS, AND ITS SIGNIFICANCE FOR POLAR IAA TRANSPORT;Carrier-mediated auxin transport;The chemical basis of morphogenesis;Studies of nuclear collisions involving 8 MeV deuterons by the photographic method III. Angular distribution of the charged particles produced by the bombardment of hydrogen and deuterium;PHYLLOTAXIS IN THE DICOTYLEDON FROM THE STANDPOINT OF DEVELOPMENTAL ANATOMY;New Phytol;The plant cell;An Explicit Spatial Model of Yeast Microcolony Growth;Modeling the organization of the WUSCHEL expression domain in the shoot apical meristem;Modeling Plant Development with Gene Regulation Networks Including Signaling and Cell Division;Proceedings of the second international conference on systems biology;Nature;Science;J. Theor. Biol;Phys. Rev. Lett;Ann. Bot. (London);Models of biological pattern formation;Proc. Natl. Acad. Sci. USA 78;The Control of the Patterned Differentiation of Vascular Tissues;Adv. Bot. Res;Digital image processing;Proc. R. Soc. London Ser. B;Patterns in plant development: Subject index;Philos. Trans. R. Soc. London B;Biol. Rev. Cambridge Philos. Soc;Ré. Trav. Bot. Néerl",
         "https://europepmc.org/articles/pmc1326488?pdf=render"
        ],
        [
         "31",
         "001bc274047c2f66701d4c2b951054af5560c464",
         "The YLDL Sequence within Sendai Virus M Protein Is Critical for Budding of Virus-Like Particles and Interacts with Alix/AIP1 Independently of C Protein",
         "ABSTRACT For many enveloped viruses, cellular multivesicular body (MVB) sorting machinery has been reported to be utilized for efficient viral budding. Matrix and Gag proteins have been shown to contain one or two L-domain motifs (PPxY, PT/SAP, YPDL, and FPIV), some of which interact specifically with host cellular proteins involved in MVB sorting, which are recruited to the viral budding site. However, for many enveloped viruses, L-domain motifs have not yet been identified and the involvement of MVB sorting machinery in viral budding is still unknown. Here we show that both Sendai virus (SeV) matrix protein M and accessory protein C contribute to virus budding by physically interacting with Alix/AIP1. A YLDL sequence within the M protein showed L-domain activity, and its specific interaction with the N terminus of Alix/AIP1(1-211) was important for the budding of virus-like particles (VLPs) of M protein. In addition, M-VLP budding was inhibited by the overexpression of some deletion mutant forms of Alix/AIP1 and depletion of endogenous Alix/AIP1 with specific small interfering RNAs. The YLDL sequence was not replaceable by other L-domain motifs, such as PPxY and PT/SAP, and even YPxL. C protein was also able to physically interact with the N terminus of Alix/AIP1(212-357) and enhanced M-VLP budding independently of M-Alix/AIP1 interaction, although it was not released from the transfected cells itself. Our results suggest that the interaction of multiple viral proteins with Alix/AIP1 may enhance the efficiency of the utilization of cellular MVB sorting machinery for efficient SeV budding.",
         "",
         "https://europepmc.org/articles/pmc1865917?pdf=render"
        ],
        [
         "32",
         "001c2ec8248cb8b5bc38ece17019ad0e133d7bec",
         "The Two-stage Pathway of Ataxin-3 Fibrillogenesis Involves a Polyglutamine-independent Step*",
         "The aggregation of ataxin-3 is associated with spinocerebellar ataxia type 3, which is characterized by the formation of intraneuronal aggregates. However, the mechanism of aggregation is currently not well understood. Ataxin-3 consists of a folded Josephin domain followed by two ubiquitin-interacting motifs and a C-terminal polyglutamine tract, which in the non-pathological form is less than 45 residues in length. We demonstrate that ataxin-3 with 64 glutamines (at(Q64)) undergoes a two-stage aggregation. The first stage involves formation of SDS-soluble aggregates, and the second stage results in formation of SDS-insoluble aggregates via the poly(Q) region. Both these first and second stage aggregates display typical amyloid-like characteristics. Under the same conditions at(Q15) and at(QHQ) undergo a single step aggregation event resulting in SDS-soluble aggregates, which does not involve the polyglutamine tract. These aggregates do not convert to the SDS-insoluble form. These observations demonstrate that ataxin-3 has an inherent capacity to aggregate through its non-polyglutamine domains. However, the presence of a pathological length polyglutamine tract introduces an additional step resulting in formation of a highly stable amyloid-like aggregate.",
         "",
         "http://www.jbc.org/article/S0021925820557128/pdf"
        ],
        [
         "33",
         "001c4745f180ca22c9ceced3991fe1cc539677bc",
         "Phosphorylation of β-Catenin by Cyclic AMP-dependent Protein Kinase*",
         "β-Catenin is a signaling molecule that promotes cell proliferation by the induction of gene transcription through the activation of T-cell factor (TCF)/lymphoid enhancer factor (LEF) transcription factors. The canonical mechanism of the regulation of β-catenin involves its phosphorylation by casein kinase 1 at the Ser-45 site and by glycogen synthase kinase 3 (GSK3) at the Thr-41, Ser-37, and Ser-33 sites. This phosphorylation targets β-catenin to ubiquitination and degradation by the proteasome system. Mitogenic factors promote β-catenin signaling through the inhibition of GSK3, resulting in reduced β-catenin phosphorylation, its stabilization, and subsequent accumulation in the nucleus, where it stimulates TCF/LEF-dependent gene transcription. In the present study, we have shown that (i) β-catenin can be phosphorylated by protein kinase A (PKA) in vitro and in intact cells at two novel sites, Ser-552 and Ser-675; (ii) phosphorylation by PKA promotes the transcriptional activity (TCF/LEF transactivation) ofβ-catenin; (iii) mutation of Ser-675 attenuates the promoting effect of PKA; (iv) phosphorylation by PKA does not affect the GSK3-dependent phosphorylation ofβ-catenin, its stability, or intracellular localization; and (v) phosphorylation at the Ser-675 site promotes the binding of β-catenin to its transcriptional coactivator, CREB-binding protein. In conclusion, this study identifies a novel, noncanonical mechanism of modulation of β-catenin signaling through direct phosphorylation of β-catenin by PKA, promoting its interaction with CREB-binding protein.",
         "Phosphorylation of β-Catenin by Cyclic AMP-Dependent Protein Kinase Stabilizes β-Catenin through Inhibition of Its Ubiquitination;Prostaglandin E2 Stimulates the β-Catenin/T Cell Factor-dependent Transcription in Colon Cancer*;Proteomic analysis identifies that 14-3-3ζ interacts with β-catenin and facilitates its activation by Akt;Dual role of PKA in phenotypic modulation of vascular smooth muscle cells by extracellular ATP;Functional significance of protein kinase A activation by endothelin-1 and ATP: negative regulation of SRF-dependent gene expression by PKA.;The Wnt signalling pathway;A-Kinase Anchoring Protein AKAP220 Binds to Glycogen Synthase Kinase-3β (GSK-3β) and Mediates Protein Kinase A-dependent Inhibition of GSK-3β*;Glycogen Synthase Kinase 3β Is Activated by cAMP and Plays an Active Role in the Regulation of Melanogenesis*;RGS3 interacts with 14-3-3 via the N-terminal region distinct from the RGS (regulator of G-protein signalling) domain.;Acetylation of β-Catenin by CREB-binding Protein (CBP)*;Crosstalk between cAMP and MAP kinase signaling in the regulation of cell proliferation.;Wnt Signaling Controls the Phosphorylation Status of β-Catenin*;Control of β-Catenin Phosphorylation/Degradation by a Dual-Kinase Mechanism;The Structure of the β-Catenin/E-Cadherin Complex and the Molecular Basis of Diverse Ligand Recognition by β-Catenin;Cyclic AMP Promotes Neuronal Survival by Phosphorylation of Glycogen Synthase Kinase 3β;Linking Colorectal Cancer to Wnt Signaling;Regulator of G Protein Signaling RGS3T Is Localized to the Nucleus and Induces Apoptosis*;The transcriptional coactivator CBP interacts with beta-catenin to activate gene expression.;The p300/CBP acetyltransferases function as transcriptional coactivators of β‐catenin in vertebrates;Mechanism and function of signal transduction by the Wnt/β-catenin and Wnt/Ca2+ pathways;Regulation of E-cadherin/Catenin Association by Tyrosine Phosphorylation*;β-Catenin regulates expression of cyclin D1 in colon carcinoma cells;The oncogenic activation of β-catenin;Identification of c-MYC as a target of the APC pathway.;β-catenin: a key mediator of Wnt signaling;Serine Phosphorylation-regulated Ubiquitination and Degradation of β-Catenin*;β‐catenin is a target for the ubiquitin–proteasome pathway;Activation of β-Catenin-Tcf Signaling in Colon Cancer by Mutations in β-Catenin or APC;Binding of GSK3β to the APC-β-Catenin Complex and Regulation of Complex Assembly;The cytoplasmic domain of the cell adhesion molecule uvomorulin associates with three independent proteins structurally related in different species.;Mutations affecting segment number and polarity in Drosophila;Cyclic AMP‐dependent protein kinase A negatively modulates adherens junction integrity and differentiation of intestinal epithelial cells;14-3-3 proteins: structure, function, and regulation.;hp://w w w .jb.org/  D;Phosphorylation of -Catenin by PKA 9976 JOURNAL OF BIOLOGICAL CHEMISTRY VOLUME",
         "http://www.jbc.org/article/S0021925819563438/pdf"
        ],
        [
         "34",
         "001c80ded266b8d277cf9276003215e75637ceb5",
         "Transcription Regulation of ompF and ompC by a Single Transcription Factor, OmpR*",
         "The ompF and ompC genes of Escherichia coli are reciprocally regulated by a single transcription factor, phosphorylated OmpR (OmpR-P), depending upon medium osmolarity. This regulation involves activation of ompF and its repression with concomitant activation of ompC. This occurs through OmpR-P binding to four (F1, F2, F3, and F4) and three (C1, C2, and C3) sites located upstream of the ompF and ompC promoters, respectively, through a novel mechanism. Here we show that there is a distinct OmpR-P binding hierarchy within F1, F2, and F3 sites as well as within C1, C2, and C3 sites. Each of these sites contains two tandem 10-bp OmpR-P-binding subsites, a-site and b-site (from 5′ to 3′ direction). OmpR-P has higher affinity to the downstream b-site than to the upstream a-site in each case. Six OmpR-P molecules bind to F and C sites two-by-two in a discontinuous “galloping” manner. We propose that this tight hierarchical binding of a transcription factor, OmpR, allows distinct stepwise regulation of ompF and ompC transcription, which minimizes their overlapping expression upon changes in the medium osmolarity to achieve the reciprocal expression of ompF and ompC.",
         "Mechanism of activation for transcription factor PhoB suggested by different modes of dimerization in the inactive and active states.;Structural Analysis and Solution Studies of the Activated Regulatory Domain of the Response Regulator ArcA: A Symmetric Dimer Mediated by the α4-β5-α5 Face;MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli;Structural Analysis of the Domain Interface in DrrB, a Response Regulator of the OmpR/PhoB Subfamily;Interaction of EnvZ, a sensory histidine kinase, with phosphorylated OmpR, the cognate response regulator;Histidine Kinases in Signal Transduction;EnvZ-OmpR Interaction and Osmoregulation in Escherichia coli *;Evidence of intradomain and interdomain flexibility in an OmpR/PhoB homolog from Thermotoga maritima.;The Critical Role of the Conserved Thr247 Residue in the Functioning of the Osmosensor EnvZ, a Histidine Kinase/Phosphatase, in Escherichia coli *;Orientation of OmpR monomers within an OmpR:DNA complex determined by DNA affinity cleaving.;Hierarchical and co‐operative binding of OmpR to a fusion construct containing the ompC and ompF upstream regulatory sequences of Escherichia coli;Relative binding affinities of OmpR and OmpR-phosphate at the ompF and ompC regulatory sites.;The DNA-binding domain of OmpR: crystal structures of a winged helix transcription factor.;Tandem Binding of Six OmpR Proteins to the ompF Upstream Regulatory Sequence of Escherichia coli(*);Fractal patterns for cells in culture;The OmpR protein of Escherichia coli binds to sites in the ompF promoter region in a hierarchical manner determined by its degree of phosphorylation.;A distant upstream site involved in the negative regulation of the Escherichia coli ompF gene;Characterization by deletion and localized mutagenesis in vitro of the promoter region of the Escherichia coli ompC gene and importance of the upstream DNA domain in positive regulation by the OmpR protein;A unique mechanism regulating gene expression: translational inhibition by a complementary RNA transcript (micRNA).;Porin channels in Escherichia coli: studies with liposomes reconstituted from purified proteins;Genetic analysis of the ompB locus in Escherichia coli K-12.;Regulation of Porins in Escherichia coli by the Osmosensing Histidine Kinase/Phosphatase EnvZ;Structure (Camb.) 13, 1353–1363 Hierarchical Binding of OmpR-P Regulates",
         "http://www.jbc.org/article/S0021925820557384/pdf"
        ],
        [
         "35",
         "001cdeaaf9f7141b7a9598b26684c01ab199a311",
         "Electroporation of DNA, RNA, and morpholinos into zebrafish embryos.",
         "None",
         "",
         "http://repositorio.uchile.cl/bitstream/2250/118774/1/Cerda_Gustavo.pdf"
        ],
        [
         "36",
         "001ced5aee9fad2575bb01579ec7e703e8ff7386",
         "Transportin Is a Major Nuclear Import Receptor for c-Fos",
         "c-Fos, a component of the transcription factor AP-1, is rapidly imported into the nucleus after translation. We established an in vitro system using digitonin-permeabilized cells to analyze nuclear import of c-Fos in detail. Two import receptors of the importin β superfamily, importin β itself and transportin, promote import of c-Fos in vitro. Under conditions where importin β-dependent transport was blocked, c-Fos still accumulated in the nucleus in the presence of cytosol. Inhibition of the transportin-dependent pathway, in contrast, abolished import of c-Fos. Furthermore, c-Fos mutants that interact with transportin but not with importin β were efficiently imported in the presence of cytosol. Hence, transportin appears to be the predominant import receptor for c-Fos. A detailed biochemical characterization revealed that the interaction of transportin with c-Fos is distinct from the interaction with its established import cargoes, the M9 sequence of heterogeneous nuclear ribonucleoprotein A1 or the nuclear localization sequence of some basic proteins. Likewise, the binding sites on importin β for its classic import cargo and for c-Fos can be separated. In summary, c-Fos employs a novel mode of receptor-cargo interaction. Hence, transportin may be as versatile as importin β in recognizing different nuclear import cargoes.",
         "",
         "http://www.jbc.org/article/S0021925819766721/pdf"
        ],
        [
         "37",
         "001cf91b762ae802a1fc6fe610759b83b300d1d1",
         "A keratin 18 transgenic zebrafish Tg(k18(2.9):RFP) treated with inorganic arsenite reveals visible overproliferation of epithelial cells.",
         "None",
         "Spatiotemporal expression of zebrafish keratin 18 during early embryogenesis and the establishment of a keratin 18:RFP transgenic line.;Arsenic exposure, urinary arsenic speciation, and peripheral vascular disease in blackfoot disease-hyperendemic villages in Taiwan.;Arsenic contamination in drinking water and skin manifestations in lowland Nepal: the first community-based survey.;Developmental consequences of in utero sodium arsenate exposure in mice with folate transport deficiencies.;Animal models for arsenic carcinogenesis: inorganic arsenic is a transplacental carcinogen in mice.;Mechanism of arsenic carcinogenesis: an integrated approach.;Induction of tumors of the liver, lung, ovary and adrenal in adult mice after brief maternal gestational exposure to inorganic arsenic: promotional effects of postnatal phorbol ester exposure on hepatic and pulmonary, but not dermal cancers.;Acute and chronic arsenic toxicity;A review of animal models for the study of arsenic carcinogenesis.;Arsenite is a cocarcinogen with solar ultraviolet radiation for mouse skin: an animal model for arsenic carcinogenesis.;Genetic toxicology of a paradoxical human carcinogen, arsenic: a review.;K6/ODC transgenic mice as a sensitive model for carcinogen identification.;Risk of internal cancers from arsenic in drinking water.;Arsenic enhancement of skin neoplasia by chronic stimulation of growth factors.;Stages of embryonic development of the zebrafish;Study on the carcinogenicity of lead arsenate and sodium arsenate and on the possible synergistic effect of diethylnitrosamine.;Life term studies on the effect of trace elements on spontaneous tumors in mice and rats.;cDNA sequences of the authentic keratins 8 and 18 in zebrafish.;Teneurin‐2 is expressed in tissues that regulate limb and somite pattern formation and is induced in vitro and in situ by FGF8;Carcinogenicity and co-carcinogenicity of sodium arsenite in P53+/− male mice;nitrosamine. Food Cosmer. Toxicol",
         "http://ntur.lib.ntu.edu.tw/bitstream/246246/162249/1/35.pdf"
        ],
        [
         "38",
         "001d0543c95a930756c172d9000b59ea2d82af78",
         "Mathematical modelling of oxygen concentration in bovine and murine cumulus-oocyte complexes.",
         "Immature oocytes benefit from nutrient modification of the follicular environment by the surrounding cumulus mass. However, the oxygen concentration that the oocyte may be exposed to could be lower than the antral follicular concentration due to the metabolism of surrounding cumulus cells. Using metabolic data previously determined, we have developed a mathematical model of O(2) diffusion across the bovine and murine cumulus-oocyte complex. From this we have determined that across a physiological range of external pO(2), less than 0.25% and 0.5% O(2) is removed by cumulus cells within the bovine and murine cumulus-oocyte complex respectively. Our model differs from others as it: incorporates a term that allows for nonlinear variation of the oxygen consumption rate with oxygen concentration; considers two regions (oocyte and cumulus) sharing a common boundary, both of which consume oxygen at different non linear rates. Cumulus cells therefore remove little O(2), thus sparing this essential gas for the oocyte, which is dependent on ATP generation via oxidative phosphorylation.",
         "High oxygen tension during in vitro oocyte maturation improves in vitro development of porcine oocytes after fertilization.;Effect of hexoses and gonadotrophin supplementation on bovine oocyte nuclear maturation during in vitro maturation in a synthetic follicle fluid medium.;Oocyte control of metabolic cooperativity between oocytes and companion granulosa cells: energy metabolism.;Mitochondria in human oogenesis and preimplantation embryogenesis: engines of metabolism, ionic regulation and developmental competence;Regulation of tissue oxygen levels in the mammalian lens;Cumulus expansion and glucose utilisation by bovine cumulus-oocyte complexes during in vitro maturation: the influence of glucosamine and follicle-stimulating hormone.;An Oxygen Transport Model for Human Bone Marrow Microcirculation;Influence of oocyte-secreted factors and culture duration on the metabolic activity of bovine cumulus cell complexes.;REDOX regulation of early embryo development.;Effects of low O2 and ageing on spindles and chromosomes in mouse oocytes from pre-antral follicle culture.;Low oxygen tension during in vitro maturation is beneficial for supporting the subsequent development of bovine cumulus–oocyte complexes;Perifollicular blood flow Doppler indices, but not follicular pO2, pCO2, or pH, predict oocyte developmental competence in in vitro fertilization.;Low Oxygen Inhibits but Complex High-Glucose Medium Facilitates In Vitro Maturation of Squirrel Monkey Oocyte–Granulosa Cell Complexes;Epigenetic Influences on Oocyte Developmental Competence: Perifollicular Vascularity and Intrafollicular Oxygen;The developmental potential of the human oocyte is related to the dissolved oxygen content of follicular fluid: association with vascular endothelial growth factor levels and perifollicular blood flow characteristics.;Substrate utilization and maturation of cumulus cell-enclosed mouse oocytes: evidence that pyruvate oxidation does not mediate meiotic induction.;Oxygen consumption and energy metabolism of the early mouse embryo;Oxygen uptake and carbohydrate metabolism by in vitro derived bovine embryos.;Factors affecting the developmental competence of mouse oocytes grown in vitro: Oxygen concentration;ATP content of human oocytes and developmental potential and outcome after in-vitro fertilization and embryo transfer.;Laboratory Production of Cattle Embryos;Development of glucose utilization studied in single oocytes and preimplantation embryos from mice.;Oxygen tension in follicular fluid falls with follicle maturation.;Effect of oxygen concentration on in-vitro development of preimplantation sheep and cattle embryos.;Intracellular diffusion gradients of O2 and ATP.;Oxygen concentration gradient across the ovarian follicular epithelium: model, predictions and implications.;Effect of mitochondrial clustering on O2 supply in hepatocytes.;Regulation of mouse oocyte growth: probable nutritional role for intercellular communication between follicle cells and oocytes in oocyte growth.;Intracellular oxygen supply during hypoxia.;Concentrations of hydrogen ions, oxygen, carbon dioxide and bicarbonate in porcine follicular fluid.;The effect of oxygen on the development of preimplantation mouse embryos in vitro.;Oxygen requirements for the maturation of hamster oocytes.;Oxygen transport in heterogeneous tissue.;Volume changes during the preimplantation stages of mouse egg development.;Culture of mouse oocytes in vitro, using a system without oil or protein.;Respiration of tissue as a medium of heterogeneous permeability.;The number and distribution of capillaries in muscles with calculations of the oxygen pressure head necessary for supplying the tissue;Mitochondria in human oogenesis and preimplantation embryogenesis: engines of metabolism, ionic regulation and developmental competence.;Oxygen balance for small organisms: An analytical model;The diffusive transport of gibberellins and abscisic acid through the aleurone layer of germinating barley grain: a mathematical model;Epigenetic Influences on Oocyte Developmental Competence: Perifollicular Vascularity and Intrafollicular Oxygen;An investigation by mathematical modelling of whether mouse and human preimplantation embryos in static culture can satisfy their demands for oxygen by diffusion.;Concentric oxygen diffusion in tissue with heterogeneous permeability and consumption.;Adenylate cyclase activity in zona-free mouse oocytes.;Versuche an uberlebendum carcinomgewebe;Molecular Reproduction and Development",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/131/6/1310999.pdf"
        ],
        [
         "39",
         "001d60f67084573ca4336fc4afdf0b7184aa47e8",
         "A Comprehensive Structure–Function Analysis of Arabidopsis SNI1 Defines Essential Regions and Transcriptional Repressor Activity[W][OA]",
         "The expression of systemic acquired resistance (SAR) in plants involves the upregulation of many Pathogenesis-Related (PR) genes, which work in concert to confer resistance to a broad spectrum of pathogens. Because SAR is a costly process, SAR-associated transcription must be tightly regulated. Arabidopsis thaliana SNI1 (for Suppressor of NPR1, Inducible) is a negative regulator of SAR required to dampen the basal expression of PR genes. Whole genome transcriptional profiling showed that in the sni1 mutant, Nonexpresser of PR genes (NPR1)–dependent benzothiadiazole S-methylester–responsive genes were specifically derepressed. Interestingly, SNI1 also repressed transcription when expressed in yeast, suggesting that it functions as an active transcriptional repressor through a highly conserved mechanism. Chromatin immunoprecipitation indicated that histone modification may be involved in SNI1-mediated repression. Sequence comparison with orthologs in other plant species and a saturating NAAIRS-scanning mutagenesis of SNI1 identified regions in SNI1 that are required for its activity. The structural similarity of SNI1 to Armadillo repeat proteins implies that SNI1 may form a scaffold for interaction with proteins that modulate transcription.",
         "Systemic Acquired Resistance;Whole-Genome Analysis of the SHORT-ROOT Developmental Pathway in Arabidopsis;Faculty Opinions recommendation of A beta-catenin identified by functional rather than sequence criteria and its role in Wnt/MAPK signaling.;A β-Catenin Identified by Functional Rather Than Sequence Criteria and Its Role in Wnt/MAPK Signaling;Fitness Costs of Mutations Affecting the Systemic Acquired Resistance Pathway in Arabidopsis thaliana;Phylogenetic profiling of the Arabidopsis thaliana proteome: what proteins distinguish plants from other organisms?;Development and evaluation of an Arabidopsis whole genome Affymetrix probe array.;Comparison of Li-Wong and loglinear mixed models for the statistical analysis of oligonucleotide arrays;Knockout Analysis of Arabidopsis Transcription Factors TGA2, TGA5, and TGA6 Reveals Their Redundant and Essential Roles in Systemic Acquired Resistance Article, publication date, and citation information can be found at www.plantcell.org/cgi/doi/10.1105/tpc.014894.;Armadillo repeat proteins: beyond the animal kingdom.;Genetic and molecular evidence that the Pseudomonas syringae type III effector protein AvrRpt2 is a cysteine protease;DAVID: Database for Annotation, Visualization, and Integrated Discovery;Statistical significance for genomewide studies;Inducers of Plant Systemic Acquired Resistance Regulate NPR1 Function through Redox Changes;Transcriptional repression in eukaryotes: repressors and repression mechanisms;Arabidopsis : a laboratory manual;C-Terminal Regions of the Human Telomerase Catalytic Subunit Essential for In Vivo Enzyme Activity;Interplay between Two Epigenetic Marks DNA Methylation and Histone H3 Lysine 9 Methylation;Isolation and characterization of broad-spectrum disease-resistant Arabidopsis mutants.;N-Terminal Domains of the Human Telomerase Catalytic Subunit Required for Enzyme Activity in Vivo;Transcription regulation by histone methylation: interplay between different covalent modifications of the core histone tails.;Trans-dominant suppression of plant TGA factors reveals their negative and positive roles in plant defense responses.;The Structure of the β-Catenin/E-Cadherin Complex and the Molecular Basis of Diverse Ligand Recognition by β-Catenin;Constitutive disease resistance requires EDS1 in the Arabidopsis mutants cpr1 and cpr6 and is partially EDS1-dependent in cpr5.;Arabidopsis in planta transformation. Uses, mechanisms, and prospects for transformation of other species.;The transcriptome of Arabidopsis thaliana during systemic acquired resistance;Roles of Salicylic Acid, Jasmonic Acid, and Ethylene in cpr-Induced Resistance in Arabidopsis;Technical Focus:a guide to Agrobacterium binary Ti vectors.;Reduced growth and seed set following chemical induction of pathogen defence: does systemic acquired resistance (SAR) incur allocation costs?;Enhanced genome annotation using structural profiles in the program 3D-PSSM.;The Arabidopsis NPR1/NIM1 Protein Enhances the DNA Binding Activity of a Subgroup of the TGA Family of bZIP Transcription Factors;NPR1 differentially interacts with members of the TGA/OBF family of transcription factors that bind an element of the PR-1 gene required for induction by salicylic acid.;The language of covalent histone modifications;Identification and Cloning of a Negative Regulator of Systemic Acquired Resistance, SNI1, through a Screen for Suppressors of npr1-1;The families of pathogenesis-related proteins, their activities, and comparative analysis of PR-1 type proteins;A mini binary vector series for plant transformation;Topological characteristics of helical repeat proteins.;Interaction of NPR1 with basic leucine zipper protein transcription factors that bind sequences required for salicylic acid induction of the PR-1 gene.;Functional analysis of regulatory sequences controlling PR-1 gene expression in Arabidopsis.;Crystallographic Analysis of the Recognition of a Nuclear Localization Signal by the Nuclear Import Factor Karyopherin α;Uncoupling PR Gene Expression from NPR1 and Bacterial Resistance: Characterization of the Dominant Arabidopsis cpr6-1 Mutant;Regulation of Hypoxia-Inducible mRNAs by the von Hippel-Lindau Tumor Suppressor Protein Requires Binding to Complexes Containing Elongins B/C and Cul2;β-catenin: a key mediator of Wnt signaling;Three-Dimensional Structure of the Armadillo Repeat Region of β-Catenin;The cpr5 mutant of Arabidopsis expresses both NPR1-dependent and NPR1-independent resistance.;Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;Salicylate-independent lesion formation in Arabidopsis lsd mutants.;Arabidopsis signal transduction mutant defective in chemically and biologically induced disease resistance.;A mutation in Arabidopsis that leads to constitutive expression of systemic acquired resistance.;CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;Characterization of an Arabidopsis Mutant That Is Nonresponsive to Inducers of Systemic Acquired Resistance.;Enhanced Protection Against Fungal Attack by Constitutive Co–expression of Chitinase and Glucanase Genes in Transgenic Tobacco;Osmotin overexpression in potato delays development of disease symptoms.;New eukaryotic transcriptional repressers;Transgenic Plants with Enhanced Resistance to the Fungal Pathogen Rhizoctonia solani;The T/t common region of simian virus 40 large T antigen contains a distinct transformation-governing sequence;Basic local alignment search tool.;Basic local alignment search tool. Journal of Molecular Biology;Deletion analysis of GAL4 defines two transcriptional activating segments;Identical short peptide sequences in unrelated proteins can have different conformations: a testing ground for theories of immune recognition.;Structure of thermolysin refined at 1.6 A resolution.;Structure and control of phosphofructokinase from Bacillus stearothermophilus;A structure for thermolysin.;Prediction of Protein Structure;The cpr 5 Mutant of Arabidopsis Expresses Both NPRl-Dependent and NPRl-1 ndependent Resistance;The structure of the beta-catenin/E-cadherin complex and the molecular basis of diverse ligand recognition by beta-catenin.;BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT;Beta-catenin: a key mediator of Wnt signaling.;Stable binding to E2F is not required for the retinoblastoma protein to activate transcription, promote differentiation, and suppress tumor cell growth.;Three-dimensional structure of the armadillo repeat region of beta-catenin.;SWISS‐MODEL and the Swiss‐Pdb Viewer: An environment for comparative protein modeling;Repression versus activation in the control of gene transcription.",
         "https://academic.oup.com/plcell/article-pdf/18/7/1750/36903817/plcell_v18_7_1750.pdf"
        ],
        [
         "40",
         "001e6a0f509456cef8c74afa35cfdc1e9a07daf6",
         "New complexities in the genetics of stuttering: significant sex-specific linkage signals.",
         "Stuttering is a speech disorder long recognized to have a genetic component. Recent linkage studies mapped a susceptibility locus for stuttering to chromosome 12 in 46 highly inbred families ascertained in Pakistan. We report here on linkage studies in 100 families of European descent ascertained in the United States, Sweden, and Israel. These families included 252 individuals exhibiting persistent stuttering, 45 individuals classified as recovered from stuttering, and 19 individuals too young to classify. Primary analyses identified moderate evidence for linkage of the broader diagnosis of \"ever stuttered\" (including both persistent and recovered stuttering) on chromosome 9 (LOD = 2.3 at 60 cM) and of the narrower diagnosis of persistent stuttering on chromosome 15 (LOD = 1.95 at 23 cM). In contrast, sex-specific evidence for linkage on chromosome 7 at 153 cM in the male-only data subset (LOD = 2.99) and on chromosome 21 at 34 cM in the female-only data subset (LOD = 4.5) met genomewide criteria for significance. Secondary analyses revealed a significant increase in the evidence for linkage on chromosome 12, conditional on the evidence for linkage at chromosome 7, with the location of the increased signal congruent with the previously reported signal in families ascertained in Pakistan. In addition, a region on chromosome 2 (193 cM) showed a significant increase in the evidence for linkage conditional on either chromosome 9 (positive) or chromosome 7 (negative); this chromosome 2 region has been implicated elsewhere in studies on autism, with increased evidence for linkage observed when the sample is restricted to those with delayed onset of phrase speech. Our results support the hypothesis that the genetic component to stuttering has significant sex effects.",
         "The sex-specific genetic architecture of quantitative traits in humans;Quantitative genome scan and Ordered-Subsets Analysis of autism endophenotypes support language QTLs;Replication of autism linkage: fine-mapping peak at 17q21.;Variation in ITGB3 has sex-specific associations with plasma lipoprotein(a) and whole blood serotonin levels in a population-based sample;Genomewide significant linkage to stuttering on chromosome 12.;Evidence for sex-specific risk alleles in autism spectrum disorder.;Ignoring linkage disequilibrium among tightly linked markers induces false-positive evidence of linkage for affected sib pair analysis.;Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays;Guidelines for genotyping in genomewide linkage studies: single-nucleotide-polymorphism maps versus microsatellite maps.;Molecular genetics of autism spectrum disorder;Evidence for a Major Gene Influence on Persistent Developmental Stuttering;Whole-genome scan, in a complex disease, using 11,245 single-nucleotide polymorphisms: comparison with microsatellites.;Examination of Potential Overlap in Autism and Language Loci on Chromosomes 2, 7, and 13 in Two Independent Samples Ascertained for Specific Language Impairment;Results of a genome‐wide linkage scan for stuttering;Genome‐wide linkage survey for genetic loci that influence the development of depressive disorders in families with recurrent, early‐onset, major depression;Finding susceptibility genes for developmental disorders of speech: the long and winding road.;Sex stratification of an inflammatory bowel disease genome search shows male-specific linkage to the HLA region of chromosome 6;Probability of detection of genotyping errors and mutations as inheritance inconsistencies in nuclear-family data.;Enhanced Pedigree Error Detection;Genomewide scans of complex human diseases: true linkage is hard to find.;Phenotypic homogeneity provides increased support for linkage on chromosome 2 in autistic disorder.;A forkhead-domain gene is mutated in a severe speech and language disorder;Evidence for a susceptibility gene for autism on chromosome 2 and for genetic heterogeneity.;Improved inference of relationship for pairs of individuals.;A Study of the Genetic and Environmental Etiology of Stuttering in a Selected Twin Sample;Chromosome 7q: where autism meets language disorder?;Erratum to “MSX1 mutation is associated with orofacial clefting and tooth agenesis in humans”;A Unified Approach to Adjusting Association Tests for Population Admixture with Arbitrary Pedigree Structure and Arbitrary Missing Marker Information;Early childhood stuttering I: persistency and recovery rates.;PedCheck: a program for identification of genotype incompatibilities in linkage analysis.;Allele-sharing models: LOD scores and accurate linkage tests.;The use of a genetic map of biallelic markers in linkage studies;The genetic basis of persistence and recovery in stuttering.;Genetics of stuttering: a critical review.;Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results;Genetic aspects of early childhood stuttering.;Early Childhood Stuttering.;Genetic factors in stuttering confirmed.;Concordance for stuttering in monozygotic and dizygotic twin pairs.;Familial stuttering patterns are not related to one measure of severity.;Nature and treatment of stuttering.;Merlin—rapid analysis of dense genetic maps using sparse gene flow trees;Segregation analyses of stuttering;Stuttering as a genetic disorder;A handbook on stuttering",
         "http://www.cell.com/article/S0002929707636957/pdf"
        ],
        [
         "41",
         "001f28be8f0482a67e69907068a20f9960642a8c",
         "Computational and neurobiological mechanisms underlying cognitive flexibility.",
         "The ability to switch between multiple tasks is central to flexible behavior. Although switching between tasks is readily accomplished, a well established consequence of task switching (TS) is behavioral slowing. The source of this switch cost and the contribution of cognitive control to its resolution remain highly controversial. Here, we tested whether proactive interference arising from memory places fundamental constraints on flexible performance, and whether prefrontal control processes contribute to overcoming these constraints. Event-related functional MRI indexed neural responses during TS. The contributions of cognitive control and interference were made theoretically explicit in a computational model of task performance. Model estimates of two levels of proactive interference, \"conceptual conflict\" and \"response conflict,\" produced distinct preparation-related profiles. Left ventrolateral prefrontal cortical activation paralleled model estimates of conceptual conflict, dissociating from that in left inferior parietal cortex, which paralleled model estimates of response conflict. These computationally informed neural measures specify retrieved conceptual representations as a source of conflict during TS and suggest that left ventrolateral prefrontal cortex resolves this conflict to facilitate flexible performance.",
         "Neural evidence for dissociable components of task-switching.;Advance preparation and stimulus-induced interference in cued task switching: further insights from BOLD fMRI;Frontal lobe mechanisms that resolve proactive interference.;Domain-general and domain-sensitive prefrontal mechanisms for recollecting events and detecting novelty.;Dissociable Controlled Retrieval and Generalized Selection Mechanisms in Ventrolateral Prefrontal Cortex;Who Comes First? The Role of the Prefrontal and Parietal Cortex in Cognitive Control;The role of the inferior frontal junction area in cognitive control;Selection for Cognitive Control: A Functional Magnetic Resonance Imaging Study on the Selection of Task-Relevant Information;The effects of recent practice on task switching.;Dissociable neural mechanisms underlying response-based and familiarity-based conflict in working memory;Neural Mechanisms of Transient and Sustained Cognitive Control during Task Switching;Stimulus-related priming during task switching;Task-switching and long-term priming: Role of episodic stimulus–task bindings in task-shift costs;Clever homunculus: is there an endogenous act of control in the explicit task-cuing procedure?;Dissociating the roles of the rostral anterior cingulate and the lateral prefrontal cortices in performing two tasks simultaneously or successively.;The Effect of Cingulate Cortex Lesions on Task Switching and Working Memory;Dissociable Contributions of Prefrontal and Parietal Cortices to Response Selection;Components of Switching Intentional Set;Neural implementation of response selection in humans as revealed by localized effects of stimulus–response compatibility on brain activation;Evidence for Anterior Cingulate Cortex Involvement in Monitoring Preparatory Attentional Set;The role of the frontal cortex in task preparation.;Role of the human medial frontal cortex in task switching: a combined fMRI and TMS study.;Task Switching: A PDP Model;Neural basis of a perceptual decision in the parietal cortex (area LIP) of the rhesus monkey.;Executive control of cognitive processes in task switching.;Conflict monitoring and cognitive control.;The role of prefrontal cortex and posterior parietal cortex in task switching.;Component Processes in Task Switching;Control of Cognitive Processes: Attention and Performance XVIII;Task-set switching and long-term memory retrieval.;Mental fatigue and task control: planning and preparation.;Cortical Surface-Based Analysis II: Inflation, Flattening, and a Surface-Based Coordinate System;Cortical Surface-Based Analysis I. Segmentation and Surface Reconstruction;Inhibition in verbal working memory revealed by brain activation.;Dissociating executive mechanisms of task control following frontal lobe damage and Parkinson's disease.;Role of left inferior prefrontal cortex in retrieval of semantic knowledge: a reevaluation.;Costs of a predictible switch between simple cognitive tasks.;Positron emission tomographic studies of the cortical anatomy of single-word processing;Neural networks and physical systems with emergent collective computational abilities.;Neuropsychologia;Trends Cognit. Sci;Science;Behav. Res. Methods Instr. Comput;Psychol. Rev;Nature;Curr. Direct. Psychol. Sci;Item-and Task-Level Processes in Left Inferior Prefrontal Cortex : Positive and Negative Correlates of Encoding;Neuron;Proc. Natl. Acad. Sci. USA 100;Mem. Cognit;Cereb. Cortex;Brain Mapp;J. Cognit. Neurosci;Attention and performance.;J. Neurophysiol;J. Exp. Psychol. Hum. Percept. Perform;Brain Res. Cognit. Brain Res;Prefrontal cortex activation in task switching: an event-related fMRI study.;Cognit. Psychol;Context, cortex, and dopamine: a connectionist approach to behavior and biology in schizophrenia.",
         "https://europepmc.org/articles/pmc1459038?pdf=render"
        ],
        [
         "42",
         "001f5ce70e459a2f9d909b9c0f15d9dd3e2f6d8c",
         "Herpes Simplex Virus Type 2 (HSV-2) Establishes Latent Infection in a Different Population of Ganglionic Neurons than HSV-1: Role of Latency-Associated Transcripts",
         "ABSTRACT Herpes simplex virus type 1 (HSV-1) and HSV-2 cause very similar acute infections but differ in their abilities to reactivate from trigeminal and dorsal root ganglia. To investigate differences in patterns of viral infection, we colabeled murine sensory ganglia for evidence of HSV infection and for the sensory neuron marker A5 or KH10. During acute infection, 7 to 10% of HSV-1 or HSV-2 antigen-positive neurons were A5 positive and 13 to 16% were KH10 positive, suggesting that both viruses reach each type of neuron in a manner proportional to their representation in uninfected ganglia. In murine trigeminal ganglia harvested during HSV latency, 25% of HSV-1 latency-associated transcript (LAT)- and 4% of HSV-2 LAT-expressing neurons were A5 positive, while 12% of HSV-1 LAT- and 42% of HSV-2 LAT-expressing neurons were KH10 positive. A similar difference was observed in murine dorsal root ganglia. These differences could not be attributed to differences in LAT expression levels in A5- versus KH10-positive neurons. Thus, HSV-1 demonstrated a preference for the establishment of latency in A5-positive neurons, while HSV-2 demonstrated a preference for the establishment of latency in KH10-positive neurons. A chimeric HSV-2 mutant that expresses the HSV-1 LAT exhibited an HSV-1 phenotype, preferentially establishing latency in A5-positive neurons. These data imply that the HSV-1 and HSV-2 LAT regions influence the ability of virus to establish latency in different neuronal subtypes. That the same chimeric virus has a characteristic HSV-1 reactivation phenotype further suggests that LAT-influenced establishment of latency in specific neuronal subtypes could be an important part of the mechanism by which LAT influences viral reactivation phenotypes.",
         "",
         "https://europepmc.org/articles/pmc1797553?pdf=render"
        ],
        [
         "43",
         "001f88f4411f8465f0bd251668a585766474a2b5",
         "Cellular scaling rules for rodent brains",
         "How do cell number and size determine brain size? Here, we show that, in the order Rodentia, increased size of the cerebral cortex, cerebellum, and remaining areas across six species is achieved through greater numbers of neurons of larger size, and much greater numbers of nonneuronal cells of roughly invariant size, such that the ratio between total neuronal and nonneuronal mass remains constant across species. Although relative cerebellar size remains stable among rodents, the number of cerebellar neurons increases with brain size more rapidly than in the cortex, such that the cerebellar fraction of total brain neurons increases with brain size. In contrast, although the relative cortical size increases with total brain size, the cortical fraction of total brain neurons remains constant. We propose that the faster increase in average neuronal size in the cerebral cortex than in the cerebellum as these structures gain neurons and the rapidly increasing glial numbers that generate glial mass to match total neuronal mass at a fixed glia/neuron total mass ratio are fundamental cellular constraints that lead to the relative expansion of cerebral cortical volume across species.",
         "Astrocytes and NG2‐glia: what's in a name?;Signaling between glia and neurons: focus on synaptic plasticity;A novel splice mutation of HERG in a Chinese family with long QT syndrome;Big brains, enhanced cognition, and response of birds to novel environments.;Isotropic Fractionator: A Simple, Rapid Method for the Quantification of Total Cell and Neuron Numbers in the Brain;Neuronal circuits of the neocortex.;Neuronal density, size and shape in the human anterior cingulate cortex: a comparison of Nissl and NeuN staining;The glial identity of neural stem cells;\n New roles for astrocytes: Redefining the functional architecture of the brain;Molecular mechanisms controlling cortical gliogenesis;Social intelligence, innovation, and enhanced brain size in primates;Scaling laws in the mammalian neocortex: Does form provide clues to function?;Brain evolution (Communications arising): How did brains evolve?;Brain evolution (Communication arising): Analysis of mammalian brain architecture;Scalable architecture in mammalian brains;Density-Dependent Feedback Inhibition of Oligodendrocyte Precursor Expansion;Schizophrenia and cognitive dysmetria: a positron-emission tomography study of dysfunctional prefrontal-thalamic-cerebellar circuitry.;Development;The distributions of Purkinje cell perikaryon and nuclear volume in human and rat cerebellum with the nucleator method;A small step for the cell, a giant leap for mankind: a hypothesis of neocortical expansion during evolution;Total numbers of various cell types in rat cerebellar cortex estimated using an unbiased stereological method;A quantitative study of the human cerebellum with unbiased stereological techniques;NeuN, a neuronal specific nuclear protein in vertebrates.;The human cerebro-cerebellar system: its computing, cognitive, and language skills;Brain sizes, surfaces, and neuronal sizes of the cortex cerebri: a stereological investigation of man and his variability and a comparison with some mammals (primates, whales, marsupials, insectivores, and one elephant).;Animal intelligence as encephalization.;Relative brain size and basal metabolic rate in terrestrial vertebrates;Principles of Neural Science;The basic uniformity in structure of the neocortex.;THE ACTIVITIES OF BUTYRYLCHOLINESTERASE AND CARBONIC ANHYDRASE, THE RATE OF ANAEROBIC GLYCOLYSTS, AND THE QUESTION OF A CONSTANT DENSITY OF GLIAL CELLS IN CEREBRAL CORTICES OF VARIOUS MAMMALIAN SPECIES FROM MOUSE TO WHALE;Glia/nerve cell index for cortex of the whale.;Structural and functional organization of mammalian cerebral cortex: The correlation of neurone density with brain size. Cortical neurone density in the fin whale (Balaenoptera Physalus L.) with a note on the cortical neurone density in the Indian elephant;Cell counts in the primate cerebral cortex;Activity of acetylcholine system in cerebral cortex of various unanesthetized mammals.;Curr. Opin. Neurobiol;J. Anat;Proc. Natl. Acad. Sci. USA 102;Cell number and cell density in the cerebellar cortex of man and some other mammals;Nature;Glia:Neuron index: Review and hypothesis to account for different values in various mammals;Size and density of glial and neuronal cells within the cerebral neocortex of various insectivorian species;The control of neuron number.;Annu. Rev. Neurosci;Brain Behav. Evol;Importance de la standardisation dans la surveillance biologique des traitements par les anti-vitamines K.;Philos. Trans. R. Soc. London Ser. B;J. Hirnforsch;Comparison of brain structure volumes in Insectivora and Primates. I. Neocortex.;Folia Primatol;New and revised data on volumes of brain structures in insectivores and primates.;Nat. Rev. Neurosci. Trends Neurosci Curr. Opin. Neurobiol. J. Neurosci. J. Neurocytol;J. Comp. Neurol;Am. J. Physiol. J. Comp. Neurol. Brain Res. Bull. J. Comp. Neurol. Brain Res. Neuroscience Cell Tiss. Res. Annu. Rev. Neurosci. Behav. Brain Res. J. Neurochem",
         "https://europepmc.org/articles/pmc1567708?pdf=render"
        ],
        [
         "44",
         "002010074a1b72f890a62fdae5c3a16296071163",
         "Inactivation of Bacillus anthracis spores in murine primary macrophages",
         "The current model for pathogenesis of inhalation anthrax indicates that the uptake and fate of Bacillus anthracis spores in alveolar macrophages are critical to the infection process. We have employed primary macrophages, which are more efficient for spore uptake than the macrophage‐like cell line RAW264.7, to investigate spore uptake and survival. We found that at a multiplicity of infection (moi) of 5, greater than 80% of the spores of the Sterne strain containing only the pXO1 plasmid were internalized within 1 h. Within 4 h post infection, viability of internalized Sterne spores decreased to approximately 40%. Intracellular vegetative bacteria represented less than 1% of the total spore inoculum throughout the course of infection suggesting effective killing of germinated spores and/or vegetative bacteria. The Sterne spores trafficked quickly to phagolysosomes as indicated by colocalization with lysosome‐associated membrane protein 1 (LAMP1). Expression of a dominant‐negative Rab7 that blocked lysosome fusion enhanced Sterne spore survival. Addition of d‐alanine to the infection resulted in 75% inhibition of spore germination and increased survival of internalized spores of the Sterne strain and a pathogenic strain containing both the pXO1 and pXO2 plasmids. Inhibition was reversed by the addition of l‐alanine, which resumed spore germination and subsequent spore killing. Our data indicate that B. anthracis spores germinate in and are subsequently killed by primary macrophages.",
         "",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1462-5822.2006.00738.x"
        ],
        [
         "45",
         "0020212c1901cc54c623dd2638a2653bd6bfa869",
         "Selective Removal of DNA from Dead Cells of Mixed Bacterial Communities by Use of Ethidium Monoazide",
         "ABSTRACT The distinction between viable and dead bacterial cells poses a major challenge in microbial diagnostics. Due to the persistence of DNA in the environment after cells have lost viability, DNA-based quantification methods overestimate the number of viable cells in mixed populations or even lead to false-positive results in the absence of viable cells. On the other hand, RNA-based diagnostic methods, which circumvent this problem, are technically demanding and suffer from some drawbacks. A promising and easy-to-use alternative utilizing the DNA-intercalating dye ethidium monoazide bromide (EMA) was published recently. This chemical is known to penetrate only into “dead” cells with compromised cell membrane integrity. Subsequent photoinduced cross-linking was reported to inhibit PCR amplification of DNA from dead cells. We provide evidence here that in addition to inhibition of amplification, most of the DNA from dead cells is actually lost during the DNA extraction procedure, probably together with cell debris which goes into the pellet fraction. Exposure of bacteria to increasing stress and higher proportions of dead cells in defined populations led to increasing loss of genomic DNA. Experiments were performed using Escherichia coli O157:H7 and Salmonella enterica serovar Typhimurium as model pathogens and using real-time PCR for their quantification. Results showed that EMA treatment of mixed populations of these two species provides a valuable tool for selective removal of DNA of nonviable cells by using conventional extraction protocols. Furthermore, we provide evidence that prior to denaturing gradient gel electrophoresis, EMA treatment of a mature mixed-population drinking-water biofilm containing a substantial proportion of dead cells can result in community fingerprints dramatically different from those for an untreated biofilm. The interpretation of such fingerprints can have important implications in the field of microbial ecology.",
         "Detection of viable and dead Listeria monocytogenes on gouda‐like cheeses by real‐time PCR;Use of Ethidium Monoazide and PCR in Combination for Quantification of Viable and Dead Cells in Complex Samples;Establishment of a Real-Time PCR-Based Approach for Accurate Quantification of Bacterial RNA Targets in Water, Using Salmonella as a Model Organism;Molecular Biology and DNA Microarray Technology for Microbial Quality Monitoring of Water;Detection of enterohemorrhagic Escherichia coli O157:H7 by using a multiplex real-time PCR assay for genes encoding intimin and Shiga toxins.;Ethidium monoazide for DNA-based differentiation of viable and dead bacteria by 5'-nuclease PCR.;Detection and enumeration of coliforms in drinking water: current methods and emerging approaches.;Assessment of viability of microorganisms employing fluorescence techniques.;β-Tubulin mRNA as a Marker ofCryptosporidium parvum Oocyst Viability;rRNA Stability in Heat-Killed and UV-Irradiated EnterotoxigenicStaphylococcus aureus and Escherichia coliO157:H7;The use of a ribosomal RNA targeted oligonucleotide probe for fluorescent labelling of viable Cryptosporidiumparvum oocysts;Assessment of bacterial viability status by flow cytometry and single cell sorting;Detection of mRNA by Reverse Transcription-PCR as an Indicator of Viability in Escherichia coliCells;Viability and activity in readily culturable bacteria: a review and discussion of the practical issues;Determination of Complement-Mediated Killing of Bacteria by Viability Staining and Bioluminescence;Assessment of E. coli and Salmonella viability and starvation by confocal laser microscopy and flow cytometry using rhodamine 123, DiBAC4(3), propidium iodide, and CTC.;Go with the flow – use of flow cytometry in environmental microbiology;Denaturing gradient gel electrophoresis profiles of 16S rRNA-defined populations inhabiting a hot spring microbial mat community;Comparison of cell viability probes compatible with fixation and permeabilization for combined surface and intracellular staining in flow cytometry.;Effect of stress treatments on the detection of Listeria monocytogenes and enterotoxigenic Escherichia coli by the polymerase chain reaction.;Polymerase chain reaction detection of nonviable bacterial pathogens;Demonstration of specific high affinity binding sites in plasmid DNA by photoaffinity labeling with an ethidium analog.;Rapid Membrane Filtration-Epifluorescent Microscopy Technique for Direct Enumeration of Bacteria in Raw Milk;Spectroscopic properties of ethidium monoazide: a fluorescent photoaffinity label for nucleic acids.;Characterization of azidobenzamidines as photoaffinity labels for trypsin.;Selective covalent binding of an ethidium analog to mitochondrial DNA with production of petite mutants in yeast by photoaffinity labelling.;Complex formation between ethidium bromide and nucleic acids.;Control of mRNA processing and decay in prokaryotes;rRNA stability in heatkilled and UV-irradiated enterotoxigenic Staphylococcus aureus and Escherichia coli O157:H7;Physiological assessment of bacteria using fluorochromes.;using rhodamine 123 , DiBAC 4 ( 3 ) , pro - pidium iodide and CTC;1 – mRNA Degradation in Prokaryotic Cells: An Overview;Control of messenger RNA stability.;Use of a photolabeling technique to identify nonviable cells in fixed homologous or heterologous cell populations.",
         "https://europepmc.org/articles/pmc1393219?pdf=render"
        ],
        [
         "46",
         "002076637c0451e34516f26da871a209b0c201b7",
         "A Thiamin-bound, Pre-decarboxylation Reaction Intermediate Analogue in the Pyruvate Dehydrogenase E1 Subunit Induces Large Scale Disorder-to-Order Transformations in the Enzyme and Reveals Novel Structural Features in the Covalently Bound Adduct*",
         "The crystal structure of the E1 component from the Escherichia coli pyruvate dehydrogenase multienzyme complex (PDHc) has been determined with phosphonolactylthiamin diphosphate (PLThDP) in its active site. PLThDP serves as a structural and electrostatic analogue of the natural intermediate α-lactylthiamin diphosphate (LThDP), in which the carboxylate from the natural substrate pyruvate is replaced by a phosphonate group. This represents the first example of an experimentally determined, three-dimensional structure of a thiamin diphosphate (ThDP)-dependent enzyme containing a covalently bound, pre-decarboxylation reaction intermediate analogue and should serve as a model for the corresponding intermediates in other ThDP-dependent decarboxylases. Regarding the PDHc-specific reaction, the presence of PLThDP induces large scale conformational changes in the enzyme. In conjunction with the E1-PLThDP and E1-ThDP structures, analysis of a H407A E1-PLThDP variant structure shows that an interaction between His-407 and PLThDP is essential for stabilization of two loop regions in the active site that are otherwise disordered in the absence of intermediate analogue. This ordering completes formation of the active site and creates a new ordered surface likely involved in interactions with the lipoyl domains of E2s within the PDHc complex. The tetrahedral intermediate analogue is tightly held in the active site through direct hydrogen bonds to residues His-407, Tyr-599, and His-640 and reveals a new, enzyme-induced, strain-related feature that appears to aid in the decarboxylation process. This feature is almost certainly present in all ThDP-dependent decarboxylases; thus its inclusion in our understanding of general thiamin catalysis is important.",
         "Experimental observation of thiamin diphosphate-bound intermediates on enzymes and mechanistic information derived from these observations.;C2-α-Lactylthiamin Diphosphate Is an Intermediate on the Pathway of Thiamin Diphosphate-dependent Pyruvate Decarboxylation;Molecular mechanism for regulation of the human mitochondrial branched-chain alpha-ketoacid dehydrogenase complex by phosphorylation.;A Molecular Switch and Proton Wire Synchronize the Active Sites in Thiamine Enzymes;Cross-talk between Thiamin Diphosphate Binding and Phosphorylation Loop Conformation in Human Branched-chain α-Keto Acid Decarboxylase/Dehydrogenase*;Tetrahedral intermediates in thiamin diphosphate-dependent decarboxylations exist as a 1',4'-imino tautomeric form of the coenzyme, unlike the michaelis complex or the free coenzyme.;Ligand-induced conformational changes and a reaction intermediate in branched-chain 2-oxo acid dehydrogenase (E1) from Thermus thermophilus HB8, as revealed by X-ray crystallography.;Dual catalytic apparatus of the thiamin diphosphate coenzyme: acid-base via the 1',4'-iminopyrimidine tautomer along with its electrophilic role.;Current mechanistic understanding of thiamin diphosphate-dependent enzymatic reactions.;Histidine 407, a phantom residue in the E1 subunit of the Escherichia coli pyruvate dehydrogenase complex, activates reductive acetylation of lipoamide on the E2 subunit. An explanation for conservation of active sites between the E1 subunit and transketolase.;Structure of the pyruvate dehydrogenase multienzyme complex E1 component from Escherichia coli at 1.85 A resolution.;Snapshot of a key intermediate in enzymatic thiamin catalysis: Crystal structure of the α-carbanion of (α,β-dihydroxyethyl)-thiamin diphosphate in the active site of transketolase from Saccharomyces cerevisiae;Inhibition of the Escherichia coli Pyruvate Dehydrogenase Complex E1 Subunit and Its Tyrosine 177 Variants by Thiamin 2-Thiazolone and Thiamin 2-Thiothiazolone Diphosphates;Recognition of the lipoyl domain is the ultimate determinant of substrate channelling in the pyruvate dehydrogenase multienzyme complex.;Crystal structure of 2-oxoisovalerate and dehydrogenase and the architecture of 2-oxo acid dehydrogenase multienzyme complexes;Crystal structures of the key anaerobic enzyme pyruvate:ferredoxin oxidoreductase, free and in complex with pyruvate;Crystallography & NMR system: A new software suite for macromolecular structure determination.;Crystal structure of the thiamin diphosphate-dependent enzyme pyruvate decarboxylase from the yeast Saccharomyces cerevisiae at 2.3 A resolution.;A thiamin diphosphate binding fold revealed by comparison of the crystal structures of transketolase, pyruvate oxidase and pyruvate decarboxylase.;Catalytic centers in the thiamin diphosphate dependent enzyme pyruvate decarboxylase at 2.4-A resolution.;PROCHECK: a program to check the stereochemical quality of protein structures;Bioassay for carcinogenicity of rotenone in female Wistar rats.;RIBBONS 2.0;Improved methods for building protein models in electron density maps and the location of errors in these models.;Slow-cooling protocols for crystallographic refinement by simulated annealing.;Thiamin diphosphate: a mechanistic update on enzymic and nonenzymic catalysis of decarboxylation;Synthesis and crystal structure of an analog of 2-(.alpha.-lactyl)thiamin, racemic methyl 2-hydroxy-2-(2-thiamin)ethylphosphonate chloride trihydrate. A conformation for a least-motion, maximum-overlap mechanism for thiamin catalysis;Stereochemistry of intermediates in thiamine catalysis. 2. Crystal structure of DL-2-(alpha-hydroxybenzyl)thiamine chloride hydrochloride trihydrate.;NUMBER 22  JOURNAL OF BIOLOGICAL CHEMISTRY 15303  by gest  on July 14;Structure (Camb.) 12, 2185–2196 Pre-decarboxylation Intermediate in PDHc Induces;Thiamine: Catalytic Mechanisms and Role in Normal and Disease States;[20] Processing of X-ray diffraction data collected in oscillation mode.;Conformation of polypeptides and proteins.;Pyruvate Dehydrogenase E1 Subunit Induces Large Scale Disorder-to-Order A Thiamin-bound, Pre-decarboxylation Reaction Intermediate Analogue in the;Access the most updated version of this article at doi",
         "http://www.jbc.org/article/S0021925820559759/pdf"
        ],
        [
         "47",
         "0020f876bc61a0a80a183829e838addcef4a7b8a",
         "Experimental demonstration of the antiherbivore effects of silica in grasses: impacts on foliage digestibility and vole growth rates",
         "The impact of plant-based factors on the population dynamics of mammalian herbivores has been the subject of much debate in ecology, but the role of antiherbivore defences in grasses has received relatively little attention. Silica has been proposed as the primary defence in grasses and is thought to lead to increased abrasiveness of foliage so deterring feeding, as well as reducing foliage digestibility and herbivore performance. However, at present there is little direct experimental evidence to support these ideas. In this study, we tested the effects of manipulating silica levels on the abrasiveness of grasses and on the feeding preference and growth performance of field voles, specialist grass-feeding herbivores. Elevated silica levels did increase the abrasiveness of grasses and deterred feeding by voles. We also demonstrated, for the first time, that silica reduced the growth rates of both juvenile and mature female voles by reducing the nitrogen they could absorb from the foliage. Furthermore, we found that vole feeding leads to increased levels of silica in leaves, suggesting a dynamic feedback between grasses and their herbivores. We propose that silica induction due to vole grazing reduces vole performance and hence could contribute to cyclic dynamics in vole populations.",
         "Dinosaur Coprolites and the Early Evolution of Grasses and Grazers;Fading out of vole and predator cycles?;Long- and short-term induction of defences in seedlings of Shorea leprosula (Dipterocarpaceae): support for the carbon:nutrient balance hypothesis;VOLE‐DRIVEN SUCCESSION IN EXPERIMENTAL WET‐PRAIRIE RESTORATIONS;Optimal Body Size and Energy Expenditure during Winter: Why Are Voles Smaller in Declining Populations?;Does quality of winter food affect spring condition and breeding in female bank voles (Clethrionomys glareolus)?1;Seasonality, density dependence, and population cycles in Hokkaido voles;AUTOCORRELATED EXOGENOUS FACTORS AND THE DETECTION OF DELAYED DENSITY DEPENDENCE;Common mammals drive the evolutionary increase of hypsodonty in the Neogene;Limits to food intake by the Prairie Vole: effects of time for digestion;Symbiosis and the Regulation of Communities1;Life-history traits of voles in a fluctuating population respond to the immediate environment;AVAILABILITY OF FOOD AND THE POPULATION DYNAMICS OF ARVICOLINE RODENTS;Do delayed effects of overgrazing explain population cycles in voles;Experimental tests of predation and food hypotheses for population cycles of voles;Effect of grazing history, experimental defoliation, and genotype on patterns of silicification in Agrostis tenuis Sibth.;EFFECTS OF BIRDS AND RODENTS ON SYNTHETIC TALLGRASS COMMUNITIES;Food selection by two vole species in relation to plant growth strategies and plant chemistry;Handbook of British Mammals;Delayed density-dependence in a small-rodent population;The effect of fertiliser and shading on plant chemical composition and palatability to Orkney voles, Microtus arvalis orcadensis;Grazing by Food-Limited Microtine Rodents on a Productive Experimental Plant Community: Does the \"Green Desert\" Exist?;Population oscillations of boreal rodents: regulation by mustelid predators leads to chaos;Do grasses fight back? The case for antiherbivore defences.;The effect of silica in grasses on the feeding behavior of the prairie vole, Microtus ochrogaster;Silicon uptake and distribution in agropyron smithii as related to grazing history and defoliation.;Influence of Chemical Factors on Palatability of Forage to Voles;Leaf silicification in grasses — A review;ON THE MEANS WHEREBY MAMMALS ACHIEVE INCREASED FUNCTIONAL DURABILITY OF THEIR DENTITIONS, WITH SPECIAL REFERENCE TO LIMITING FACTORS;An Interpretation of Rodent Dynamics as Due to Trophic Interactions;How much do weasels shape microtine cycles in the northern Fennoscandian taiga;Silicon Concentration of Grasses Growing in Sites With Different Grazing Histories;Limiting Resources and Territoriality in Microtine Rodents;A Nutritional Explanation for Body-Size Patterns of Ruminant and Nonruminant Herbivores;Silica as a Defense against Herbivory and a Growth Promotor in African Grasses;Grazing Lawns: Animals in Herds, Plant Form, and Coevolution;Grass leaf silicification: Natural selection for an inducible defense against herbivores.;Nutritional Ecology of Microtine Rodents: Digestibility of Forage;NUTRITION AND POPULATION DYNAMICS OF THE PRAIRIE VOLE, MICROTUS OCHROGASTER, IN CENTRAL ILLINOIS;Chemical Analysis of Ecological Materials.;Cause of Wear in Sheeps' Teeth;Experimental demonstration of the antiherbivore effects of silica in grasses : impacts on foliage digestibility and vole growth rates;Mechanical testing of the abrasiveness of grass leaves. Third plant biomechanics conference proceedings;Interactions between woody plants and browsing mammals mediated by secondary metabolites;COEVOLUTION OF GRASSES AND HERBIVORES;Availability of Food;Population Cycles in Small Mammals;J. Mammal",
         "https://europepmc.org/articles/pmc1636089?pdf=render"
        ],
        [
         "48",
         "00221c6a820c45208f266ee3276397aa16b5c5c4",
         "Revealing network connectivity from response dynamics.",
         "We present a method to infer the complete connectivity of a network from its stable response dynamics. As a paradigmatic example, we consider networks of coupled phase oscillators and explicitly study their long-term stationary response to temporally constant driving. For a given driving condition, measuring the phase differences and the collective frequency reveals information about how the units are interconnected. Sufficiently many repetitions for different driving conditions yield the entire network connectivity (the absence or presence of each connection) from measuring the response dynamics only. For sparsely connected networks, we obtain good predictions of the actual connectivity even for formally underdetermined problems.",
         "Convex Optimization;Does Dynamics Reflect Topology in Directed Networks;Designing complex networks;Entrainment of coupled oscillators on regular networks by pacemakers.;Chemical oscillations;Designing the dynamics of spiking neural networks.;Synchronization reveals topological scales in complex networks.;A method for determining neural connectivity and inferring the underlying network dynamics using extracellular spike recordings;Long range action in networks of chaotic elements;The Kuramoto model: A simple paradigm for synchronization phenomena;Disturbing synchronization: Propagation of perturbations \nin networks of coupled oscillators;Motifs in Brain Networks;Propagation of small perturbations in synchronized oscillator networks;Mechanism of desynchronization in the finite-dimensional Kuramoto model.;Interaction strengths in food webs: issues and opportunities;Entrainment of randomly coupled oscillator networks by a pacemaker.;Breaking synchrony by heterogeneity in complex networks.;Inferring Genetic Networks and Identifying Compound Mode of Action via Expression Profiling;Network motifs: simple building blocks of complex networks.;Wiring Optimization in Cortical Circuits;Modelling Food Webs;Exploring complex networks;From Kuramoto to Crawford: exploring the onset of synchronization in populations of coupled oscillators;Dynamics of neuronal firing correlation: modulation of \"effective connectivity\".;Chemical Oscillations, Waves, and Turbulence;Nonlinearity 19;Physica (Amsterdam) 224D;and O;Rev. Mod. Phys;J. Anim. Ecol;PLoS Biology 2;Phys. Rev. Lett;Europhys. Lett. Eur. Phys. J. B;Nature (London) 427;Science;Neuron 34;Synchronization: A Universal Concept in Nonlinear Sciences;Synchronization: Phase locking and frequency entrainment;Physica D;Phase Resetting in Medicine and Biology;We discard outliers: they may originate from perturbations that create (i) no structurally stable response or (ii) responses that are close to linearly dependent on others;The reconstruction quality depends not only on the degree of inhomogeneity, but also, among other parameters, on the realization of it and of the driving;A directed network is strongly connected if there is a directed path between any ordered pair of nodes;Here and in the following, the driving signals are independent random I i,m ∈ [0, ∆I], where ∆I = 0",
         "http://link.aps.org/pdf/10.1103/PhysRevLett.98.224101"
        ],
        [
         "49",
         "00221da1856a3265b29461d06b3a9054c814f401",
         "Patterns of Regional Brain Hypometabolism Associated with Knowledge of Semantic Features and Categories in Alzheimer's Disease",
         "The study of semantic memory in patients with Alzheimer's disease (AD) has raised important questions about the representation of conceptual knowledge in the human brain. It is still unknown whether semantic memory impairments are caused by localized damage to specialized regions or by diffuse damage to distributed representations within nonspecialized brain areas. To our knowledge, there have been no direct correlations of neuroimaging of in vivo brain function in AD with performance on tasks differentially addressing visual and functional knowledge of living and nonliving concepts. We used a semantic verification task and resting 18-fluorodeoxyglucose positron emission tomography in a group of mild to moderate AD patients to investigate this issue. The four task conditions required semantic knowledge of (1) visual, (2) functional properties of living objects, and (3) visual or (4) functional properties of nonliving objects. Visual property verification of living objects was significantly correlated with left posterior fusiform gyrus metabolism (Brodmann's area [BA] 37/19). Effects of visual and functional property verification for non-living objects largely overlapped in the left anterior temporal (BA 38/20) and bilateral premotor areas (BA 6), with the visual condition extending more into left lateral precentral areas. There were no associations with functional property verification for living concepts. Our results provide strong support for anatomically separable representations of living and nonliving concepts, as well as visual feature knowledge of living objects, and against distributed accounts of semantic memory that view visual and functional features of living and nonliving objects as distributed across a common set of brain areas.",
         "",
         "http://discovery.ucl.ac.uk/6036/1/6036.pdf"
        ]
       ],
       "shape": {
        "columns": 5,
        "rows": 76650
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>referenceTitles</th>\n",
       "      <th>openAccessPdf</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>00009f98f5ff962beb322e6c0e78e4b77dd2d968</td>\n",
       "      <td>High exposure to Tunga penetrans (Linnaeus, 17...</td>\n",
       "      <td>Tungiasis is a parasitic skin disease widespre...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.scielo.br/j/mioc/a/csnpsRLq5FqY5Mk...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0000e1833852eb3271356c493d59e83da0c4c7db</td>\n",
       "      <td>Immunoinformatics Comes of Age</td>\n",
       "      <td>With the burgeoning immunological data in the ...</td>\n",
       "      <td>Prediction of the secondary structure of prote...</td>\n",
       "      <td>https://journals.plos.org/ploscompbiol/article...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>00016363b9a128e01a17e9b831249967a0fee15f</td>\n",
       "      <td>The Potassium Channel KAT1 Is Activated by Pla...</td>\n",
       "      <td>14-3-3 proteins modulate the plant inward rect...</td>\n",
       "      <td></td>\n",
       "      <td>http://www.jbc.org/article/S0021925820681212/pdf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>00034b38a5273b760824f3223fa0d4fdb349d6df</td>\n",
       "      <td>Competitive recruitment of the periplasmic tra...</td>\n",
       "      <td>The natively disordered N-terminal 83-aa trans...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.pnas.org/doi/pdf/10.1073/pnas.0603...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00036a3493967932ac256f38c0c2419415f04feb</td>\n",
       "      <td>Polysaccharides from the root of Angelica sine...</td>\n",
       "      <td>Cyclophosphamide (CY) is a cytostatic agent th...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.medsci.org/v03p0001.pdf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76645</th>\n",
       "      <td>fffabc38e25edd5a95b48e2952322542317ea392</td>\n",
       "      <td>Implicit vs. Explicit Data-Flow Requirements i...</td>\n",
       "      <td>None</td>\n",
       "      <td>Specifying Data-Flow Requirements for the Auto...</td>\n",
       "      <td>https://link.springer.com/content/pdf/10.1007%...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76646</th>\n",
       "      <td>fffae8f127ca823fecd1228e5131aa06ec98696a</td>\n",
       "      <td>Presence of Pathogenicity Island Genes in Ente...</td>\n",
       "      <td>ABSTRACT Enterococcus faecalis isolates of por...</td>\n",
       "      <td>Multilocus Sequence Typing Scheme for Enteroco...</td>\n",
       "      <td>https://europepmc.org/articles/pmc1698369?pdf=...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76647</th>\n",
       "      <td>fffe4813ee529996e47ec6c4f1732f0030d334c9</td>\n",
       "      <td>Variability of Motor Neuron Spike Timing Maint...</td>\n",
       "      <td>The accessory radula closer (ARC) muscle of Ap...</td>\n",
       "      <td>Tight or loose coupling between components of ...</td>\n",
       "      <td>https://www.jneurosci.org/content/jneuro/26/26...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76648</th>\n",
       "      <td>ffff1780c2f0bb4d8ea115fdd820a518a1b33dd6</td>\n",
       "      <td>Relationship between Thermodynamic Driving For...</td>\n",
       "      <td>Chemical reaction systems operating in nonequi...</td>\n",
       "      <td>Generalized Haldane equation and fluctuation t...</td>\n",
       "      <td>https://journals.plos.org/plosone/article/file...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76649</th>\n",
       "      <td>ffffb1b949fe54eb8dda3238df0058b373ec3408</td>\n",
       "      <td>Mutation in a “tesB-Like” Hydroxyacyl-Coenzyme...</td>\n",
       "      <td>ABSTRACT A novel mutant of the marine oil-degr...</td>\n",
       "      <td>Genome sequence of the ubiquitous hydrocarbon-...</td>\n",
       "      <td>https://europepmc.org/articles/pmc1698222?pdf=...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>76650 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paperId  \\\n",
       "0      00009f98f5ff962beb322e6c0e78e4b77dd2d968   \n",
       "1      0000e1833852eb3271356c493d59e83da0c4c7db   \n",
       "2      00016363b9a128e01a17e9b831249967a0fee15f   \n",
       "3      00034b38a5273b760824f3223fa0d4fdb349d6df   \n",
       "4      00036a3493967932ac256f38c0c2419415f04feb   \n",
       "...                                         ...   \n",
       "76645  fffabc38e25edd5a95b48e2952322542317ea392   \n",
       "76646  fffae8f127ca823fecd1228e5131aa06ec98696a   \n",
       "76647  fffe4813ee529996e47ec6c4f1732f0030d334c9   \n",
       "76648  ffff1780c2f0bb4d8ea115fdd820a518a1b33dd6   \n",
       "76649  ffffb1b949fe54eb8dda3238df0058b373ec3408   \n",
       "\n",
       "                                                   title  \\\n",
       "0      High exposure to Tunga penetrans (Linnaeus, 17...   \n",
       "1                         Immunoinformatics Comes of Age   \n",
       "2      The Potassium Channel KAT1 Is Activated by Pla...   \n",
       "3      Competitive recruitment of the periplasmic tra...   \n",
       "4      Polysaccharides from the root of Angelica sine...   \n",
       "...                                                  ...   \n",
       "76645  Implicit vs. Explicit Data-Flow Requirements i...   \n",
       "76646  Presence of Pathogenicity Island Genes in Ente...   \n",
       "76647  Variability of Motor Neuron Spike Timing Maint...   \n",
       "76648  Relationship between Thermodynamic Driving For...   \n",
       "76649  Mutation in a “tesB-Like” Hydroxyacyl-Coenzyme...   \n",
       "\n",
       "                                                abstract  \\\n",
       "0      Tungiasis is a parasitic skin disease widespre...   \n",
       "1      With the burgeoning immunological data in the ...   \n",
       "2      14-3-3 proteins modulate the plant inward rect...   \n",
       "3      The natively disordered N-terminal 83-aa trans...   \n",
       "4      Cyclophosphamide (CY) is a cytostatic agent th...   \n",
       "...                                                  ...   \n",
       "76645                                               None   \n",
       "76646  ABSTRACT Enterococcus faecalis isolates of por...   \n",
       "76647  The accessory radula closer (ARC) muscle of Ap...   \n",
       "76648  Chemical reaction systems operating in nonequi...   \n",
       "76649  ABSTRACT A novel mutant of the marine oil-degr...   \n",
       "\n",
       "                                         referenceTitles  \\\n",
       "0                                                          \n",
       "1      Prediction of the secondary structure of prote...   \n",
       "2                                                          \n",
       "3                                                          \n",
       "4                                                          \n",
       "...                                                  ...   \n",
       "76645  Specifying Data-Flow Requirements for the Auto...   \n",
       "76646  Multilocus Sequence Typing Scheme for Enteroco...   \n",
       "76647  Tight or loose coupling between components of ...   \n",
       "76648  Generalized Haldane equation and fluctuation t...   \n",
       "76649  Genome sequence of the ubiquitous hydrocarbon-...   \n",
       "\n",
       "                                           openAccessPdf  \n",
       "0      https://www.scielo.br/j/mioc/a/csnpsRLq5FqY5Mk...  \n",
       "1      https://journals.plos.org/ploscompbiol/article...  \n",
       "2       http://www.jbc.org/article/S0021925820681212/pdf  \n",
       "3      https://www.pnas.org/doi/pdf/10.1073/pnas.0603...  \n",
       "4                    https://www.medsci.org/v03p0001.pdf  \n",
       "...                                                  ...  \n",
       "76645  https://link.springer.com/content/pdf/10.1007%...  \n",
       "76646  https://europepmc.org/articles/pmc1698369?pdf=...  \n",
       "76647  https://www.jneurosci.org/content/jneuro/26/26...  \n",
       "76648  https://journals.plos.org/plosone/article/file...  \n",
       "76649  https://europepmc.org/articles/pmc1698222?pdf=...  \n",
       "\n",
       "[76650 rows x 5 columns]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "textdata0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   id  age     name  taille\n",
      "0   1   25    Alice     NaN\n",
      "1   2   30      Bob     NaN\n",
      "2   3   36  Charlie   170.0\n",
      "3   4   40    David   180.0\n"
     ]
    }
   ],
   "source": [
    "# Sample DataFrames\n",
    "df1 = pd.DataFrame({\n",
    "    \"id\": [1, 2, 3],\n",
    "    \"name\": [\"Alice\", \"Bob\", \"Charlie\"],\n",
    "    \"age\": [25, 30, 35],\n",
    "})\n",
    "\n",
    "df2 = pd.DataFrame({\n",
    "    \"id\": [3, 4],  # id=2 and id=3 exist, id=4 is new\n",
    "    \"name\": [\"Charlie\", \"David\"],  # id=3 has new data\n",
    "    \"age\": [36, 40],  # Updates for id=2 and id=3\n",
    "    \"taille\": [170, 180]\n",
    "})\n",
    "\n",
    "# 🔥 Merge and update existing rows\n",
    "merged_df = df2.set_index(\"id\").combine_first(df1.set_index(\"id\")).reset_index(drop=False)\n",
    "\n",
    "print(merged_df)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "recoRAG",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
